<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6814934 - Instrument for monitoring nucleic acid amplification - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_4ff636b3d23669b7103f3b3a3a18b4cd/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_4ff636b3d23669b7103f3b3a3a18b4cd__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Instrument for monitoring nucleic acid amplification"><meta name="DC.contributor" content="Russell Gene Higuchi" scheme="inventor"><meta name="DC.contributor" content="Russell Gene Higuchi" scheme="assignee"><meta name="DC.date" content="1997-11-12" scheme="dateSubmitted"><meta name="DC.description" content="This invention relates to improved methods for nucleic acid detection using methods such as the polymerase chain reaction (PCR). More specifically, the invention provides methods for simultaneous amplification and detection to enhance the speed and accuracy of prior methods. The methods involve the introduction of detectable DNA binding agents into the amplification reaction, which agents produce a detectable signal that is enhanced upon binding double-stranded DNA. In a preferred embodiment, the binding agent is a fluorescent dye. The methods also provide means for monitoring the increase in product DNA during an amplification reaction."><meta name="DC.date" content="2004-11-9" scheme="issued"><meta name="DC.relation" content="DE:2651536:A1" scheme="references"><meta name="DC.relation" content="EP:0070685:A2" scheme="references"><meta name="DC.relation" content="EP:0169787:A1" scheme="references"><meta name="DC.relation" content="EP:0200362" scheme="references"><meta name="DC.relation" content="EP:0236069:A2" scheme="references"><meta name="DC.relation" content="EP:0487218:A1" scheme="references"><meta name="DC.relation" content="EP:0488769" scheme="references"><meta name="DC.relation" content="EP:0512334" scheme="references"><meta name="DC.relation" content="EP:0516274:A2" scheme="references"><meta name="DC.relation" content="EP:0524808:A2" scheme="references"><meta name="DC.relation" content="EP:0580362" scheme="references"><meta name="DC.relation" content="EP:0640828" scheme="references"><meta name="DC.relation" content="FR:2250991" scheme="references"><meta name="DC.relation" content="GB:1486210" scheme="references"><meta name="DC.relation" content="JP:H03122552" scheme="references"><meta name="DC.relation" content="JP:H03259099" scheme="references"><meta name="DC.relation" content="JP:H0427399" scheme="references"><meta name="DC.relation" content="JP:H0484751" scheme="references"><meta name="DC.relation" content="JP:H0484752" scheme="references"><meta name="DC.relation" content="JP:S61215948" scheme="references"><meta name="DC.relation" content="JP:S62105031" scheme="references"><meta name="DC.relation" content="SU:1231077" scheme="references"><meta name="DC.relation" content="US:3856470" scheme="references"><meta name="DC.relation" content="US:3992631" scheme="references"><meta name="DC.relation" content="US:3999868" scheme="references"><meta name="DC.relation" content="US:4058370" scheme="references"><meta name="DC.relation" content="US:4119521" scheme="references"><meta name="DC.relation" content="US:4240751" scheme="references"><meta name="DC.relation" content="US:4257774" scheme="references"><meta name="DC.relation" content="US:4563417" scheme="references"><meta name="DC.relation" content="US:4582789" scheme="references"><meta name="DC.relation" content="US:4683195" scheme="references"><meta name="DC.relation" content="US:4683202" scheme="references"><meta name="DC.relation" content="US:4856073" scheme="references"><meta name="DC.relation" content="US:4889818" scheme="references"><meta name="DC.relation" content="US:4902624" scheme="references"><meta name="DC.relation" content="US:5038852" scheme="references"><meta name="DC.relation" content="US:5049490" scheme="references"><meta name="DC.relation" content="US:5096807" scheme="references"><meta name="DC.relation" content="US:5184020" scheme="references"><meta name="DC.relation" content="US:5210015" scheme="references"><meta name="DC.relation" content="US:5219727" scheme="references"><meta name="DC.relation" content="US:5229297" scheme="references"><meta name="DC.relation" content="US:5242797" scheme="references"><meta name="DC.relation" content="US:5269937" scheme="references"><meta name="DC.relation" content="US:5322770" scheme="references"><meta name="DC.relation" content="US:5355215" scheme="references"><meta name="DC.relation" content="US:5374553" scheme="references"><meta name="DC.relation" content="US:5389512" scheme="references"><meta name="DC.relation" content="US:5411876" scheme="references"><meta name="DC.relation" content="US:5415839" scheme="references"><meta name="DC.relation" content="US:5455170" scheme="references"><meta name="DC.relation" content="US:5552580" scheme="references"><meta name="DC.relation" content="US:5618711" scheme="references"><meta name="DC.relation" content="US:5795747" scheme="references"><meta name="DC.relation" content="US:5871908" scheme="references"><meta name="DC.relation" content="US:5994056" scheme="references"><meta name="DC.relation" content="US:6171785" scheme="references"><meta name="DC.relation" content="WO:1990015881:A1" scheme="references"><meta name="DC.relation" content="WO:1992005278:A2" scheme="references"><meta name="citation_reference" content="Advertisement for Perkin Elmer Cetus GeneAmp(R)PCR System 9600, 1991, BioTechniques 10(2)."><meta name="citation_reference" content="Advertisement for Perkin Elmer Cetus GeneAmp®PCR System 9600, 1991, BioTechniques 10(2)."><meta name="citation_reference" content="Annex 2 from cite 46, above."><meta name="citation_reference" content="Appeal Brief submitted by Bio-Rad to the Japanese Patent Office on Oct. 30, 2003 seeking invalidation of counterpart Japanese Patent No. 3136129 (English translation only)."><meta name="citation_reference" content="Applera Answer dated Mar. 29, 2004 filed in Applera v. Bio-rad, Düsseldorf Regional Court Case No. 4a O 44/03 (English language translation only)."><meta name="citation_reference" content="Applera Brief (adding another product) dated Dec. 4, 2003 filed in Applera v. Bio-Rad, Dusseldorf Regional Court Case No. 4a O 44/03 (English language translation only)."><meta name="citation_reference" content="Applera Complaint dated Feb. 7, 2003 filed in Applera v. Bio-Rad, Dusseldorf Regional Court Case No. 4a O 44/03 (English language translation only)."><meta name="citation_reference" content="Applera Corporation&#39;s Claim Construction Brief for U.S. Patent No. 5,552,580 (with copy of Expert Report of Dr. Joseph Jaklevic)."><meta name="citation_reference" content="Applera Corporation&#39;s Corrected Claim Construction Brief for U.S. Pat. No. 5,552,580 (with copy of Notice of Filing of same)."><meta name="citation_reference" content="Applera Extension of Claim (Patent in Suit III) dated Jun. 20, 2003 filed in Applera v. MJ Research, Dusseldorf Regional Court Case No. 4a O 262/02 (English language translation only)."><meta name="citation_reference" content="Applera First Statement dated Jan. 14, 2004 filed with the Tokyo District Court in PE Corp. (NY) vs. Bio-Rad, Tokyo District Court No. H-15 (mo) 15045 (English language translation only)."><meta name="citation_reference" content="Applera Reply Brief dated Feb. 2, 2004 filed Applera v. Bio-Rad, Dusseldorf Regional Court Case No. 4a O 44/03 (English language translation only)."><meta name="citation_reference" content="Applera Response dated February 16, 2004 filed in Applera v. MJ Research, Dusseldorf Regional Court Case No. 4a O 262/02 (English language translation only)."><meta name="citation_reference" content="Applera Response to Oppositions and Intervention against EP 0 872 562 filed in the European Patent Office Feb. 2, 2004."><meta name="citation_reference" content="Applera Summary of Arguments dated Jan. 7, 2004 filed with the Tokyo District Court in PE Corp. (NY) vs. Bio-Rad, Tokyo District Court No. H-15 (mo) 15045 (English language translation only)."><meta name="citation_reference" content="Bauer et al., Traveling waves of in vitro evolving RNA, Proc. Natl. Acad. Sci. 86:7937-7941 (1989)."><meta name="citation_reference" content="Bio-Rad Brief dated Oct. 30, 2003 filed with the Japanese Patent Office in Invalidation Appeal No. 2003-35449 (English language translation only)."><meta name="citation_reference" content="Bio-Rad Protest Brief dated Nov. 19, 2003 filed with the Tokyo District Court in PE Corp. (NY) vs. Bio-Rad, Tokyo District Court No. H-15 (mo) 15045 (English language translation only)."><meta name="citation_reference" content="Bio-Rad Reply Brief dated Mar. 22, 2004 filed in Applera v. Bio-Rad, Düsseldorf Regional Court Case No. 4a O 44/03 (English language translation only)."><meta name="citation_reference" content="Bio-Rad Reply Brief dated Nov. 17, 2003 filed in Applera v. Bio-Rad, Dusseldorf Regional Court Case No. 4a O 44/03 (English language translation only)."><meta name="citation_reference" content="Bio-Rad Second Statement dated Feb. 20, 2004 filed with the Tokyo District Court in PE Corp. (NY) vs. Bio-Rad, Tokyo District Court No. H-15 (mo) 15045 (English language translation only)."><meta name="citation_reference" content="Byrnes, J.J., et al., Biochemistry, 12(22):4378-4384 (1973)."><meta name="citation_reference" content="Cardullo et al., 1988, Proc. Natl. Acad. Sci. USA 85:8790-8794."><meta name="citation_reference" content="Chehab et al., Detection of specific DNA sequences by fluorescence amplification: A color complementation assay, PNAS 86:9178-9182 (1989)."><meta name="citation_reference" content="Decision by Japanese Patent Office dated Dec. 25, 2003 in Invalidation Appeal No. 2002-35399 (English language translation only)."><meta name="citation_reference" content="Decision of Preliminary Injunction from Japanese Litigation H14 (Yo) No. 22065 (English Translation only)."><meta name="citation_reference" content="Declarations of Drs. Manfred Eigen , Ruthild Winkler-Oswatitsch, Björn Lindemann, Andreas Schwienhorst and Rolf Günther from the file history of EP 0 583 265."><meta name="citation_reference" content="Derwent Abstract for Japanese Patent Application JP63119672-A, May 24, 1988 and Patent Abstracts of Japan for Japanese Patent Application JP63119672-A, published May 24, 1988."><meta name="citation_reference" content="Derwent Abstract for SU1702312-A1, 1986."><meta name="citation_reference" content="Dissertation Thesis of Hajo Otten, &quot;Ein Beitrag Zur Durchführung von Kontrollierten Evolutionsexperimenten mit Biologischen Makromolekülen,&quot; pp. 1-17 (publication date unknown), English language translation of the text bridging pp. 16-17, beginning with the heading &quot;2.5 Machinenexperimente&quot;."><meta name="citation_reference" content="Eigen et al., &quot;Report on Evolution Research,&quot; Department of Biochemical Kinetics, Max Planck Institut Fur Biophysikalische Chemie, p. i, 1, 2, 53-56, 65."><meta name="citation_reference" content="Eigen, 1987, &quot;New Concepts for Dealing with the Evolution of Nucleic Acids,&quot; Cold Spring Harbor Symposium on Quantitative Biology, vol. LII, pp. 307-319."><meta name="citation_reference" content="Eigen, allegedly Apr. 18-20, 1991, &quot;Report on Evolution Research,&quot; Department of Biochemical Kinetics, Max Plack Institut für Biophysikalische Chemie, cover page, title page and pp. i-iii and 48-56."><meta name="citation_reference" content="Eigen, M. et al &quot;Sorting single molecules: Application to diagnostics and evolutionary biotechnology&quot; Proc. Natl. Acad. Sci., vol 91, pp. 5740-5747, 1994.*"><meta name="citation_reference" content="Eppendorf Product Literature, Sep. 1973."><meta name="citation_reference" content="Exhibit B11 from Cite No. 62 (Eigen et al., allegedly Apr. 18-20, 1991, &quot;Report on Evolution Research&quot;, pp. i, 1, 2, 53-56 and 65."><meta name="citation_reference" content="Exhibit B11a from Cite No. 62 (undated and unsworn statement by Prof. Manfred Eigen)."><meta name="citation_reference" content="Exhibit Collection B14 from Cite No. 72."><meta name="citation_reference" content="Exhibit Ko No. 8 from Cite No. 53(Eigen et al., allegedly Apr. 18-20, 1991, &quot;Report on Evolution Research,&quot;Department of Biochemical Kinetics, Max Planck Institute Fur Biophysikalische Chemie, complete document)."><meta name="citation_reference" content="Exhibit No. 21 from Cite No. 58 (Eigen et al., &quot;Report on the International Workshop Selection-Natural and Unnatural-In Biotechnology&quot;)."><meta name="citation_reference" content="Exhibit No. 21 from Cite No. 58 (Eigen et al., “Report on the International Workshop Selection—Natural and Unnatural—In Biotechnology”)."><meta name="citation_reference" content="FluoroLog-2 Spectrofluorometer Manual (publication date unknown)."><meta name="citation_reference" content="Full copy of &quot;Report on Evolution Research,&quot; Department of Biochemical Kinetics, Max Planck Institut für Biophysikalische Chemie, allegedly published Apr. 18-20, 1991, submitted as Exhibit 8 in item 51 above, portions of which were cited in this application as item 41 on the Form 1449A submitted to the Patent Office on or about Oct. 24, 2003."><meta name="citation_reference" content="Glazer et al., Proc. Natl. Acad. Sci. USA 87:3851-3855 (1990)."><meta name="citation_reference" content="Haff et al., A high-performance system for automation of the polymerase chain reaction, BioTechniques 10(1):102-112 (Jan. 1991)."><meta name="citation_reference" content="Haff, L.A. &quot;New Developments in Polymerase Chain Reaction&quot; Amplifications, Issue 1, Feb., 1989, pp. 1-12.*"><meta name="citation_reference" content="Henco et al., Determination of in vitro amplified nucleic acid sequences, Chem. Abstr. 119 (15):153368b (1993)."><meta name="citation_reference" content="Higuchi et al., Kinetic PCR analysis: real-time monitoring of DNA amplification reactions, Bio/Technology 11:1026-1030 (1993)."><meta name="citation_reference" content="Higuchi et al., Simultaneous amplification and detection of specific DNA sequences, Bio/Technology 10:413-417 (1992)."><meta name="citation_reference" content="Holland et al., 1991, FASEB 5(4): A261."><meta name="citation_reference" content="Isaacs et al., Nucleic Acids Research, 19(1):109-116 (Jan. 1991)."><meta name="citation_reference" content="Johnson and Hearst, Photochem. and Photobiol., 33:785-791 (1981)."><meta name="citation_reference" content="Kaledin, et al. Biokhimiya 46(9):1576-1584 (1982)."><meta name="citation_reference" content="Kapuseinski and Szer, Nuc. Acids Res. 6(112):3519-3535 (1979)."><meta name="citation_reference" content="Katz et al., Rapid separation, quantitation and purification of products of polymerase chain reaction by liquid chromatography, J. Chromatography 512:433-444 (1990)."><meta name="citation_reference" content="Kellogg et al., Quantitation of HIV-1 Proviral DNA relative to cellular DNA by the polymerase chain reaction, Anal. Biochem. 189:202-208 (1990)."><meta name="citation_reference" content="Khattat et al., JACS, 112:4960 (1990)."><meta name="citation_reference" content="Kirk-Othmer, Encyclopedia of Chemical Technology (Third Edition), 3:906-907."><meta name="citation_reference" content="Kornberg, DNA Replication, 427-441 (1980)."><meta name="citation_reference" content="Kornberg, DNA Replication, 451-455 (1992)."><meta name="citation_reference" content="Kornberg, in DNA Synthesis, W.H. Freman and Co., San Francisco 220-236 (1974)."><meta name="citation_reference" content="Kosarev et al., Determination of bacterial concentration in suspension-involves using di:amino-ethyl-phenyl-phananthridinium bromide as fluorescing substance and measuring fluorescence intensity after excitation, Database WPIL Week 8651 Derwent Publications Ltd., Lond, GB; AN 86-338273 &amp; SU-A-1 231 077 (Appl Microbiol Res) May 15, 1986."><meta name="citation_reference" content="Kosarev et al., Determination of bacterial concentration in suspension—involves using di:amino-ethyl-phenyl-phananthridinium bromide as fluorescing substance and measuring fluorescence intensity after excitation, Database WPIL Week 8651 Derwent Publications Ltd., Lond, GB; AN 86-338273 &amp; SU-A-1 231 077 (Appl Microbiol Res) May 15, 1986."><meta name="citation_reference" content="Kramer et al., Evolution in vitro: sequence phenotype of a mutant RNA resistant to ethidium bromide, J. Mol. Biol. 89:719-736 (1974)."><meta name="citation_reference" content="Literature Express Circle 245, 1990, BioTechniques 9(5): 650."><meta name="citation_reference" content="Lomell, H., et al., Clin. Chem., 35(9):1826-1831 (1989)."><meta name="citation_reference" content="Mabuchi and Nishikawa, Nuc. Acids Res. 18(24):7461-7462 (1990)."><meta name="citation_reference" content="McCleod, 1990 J. Med. Eng. Tech. 14: 60-68."><meta name="citation_reference" content="Millipore Direct Catalogue, p. 393-396 (1994-1995)."><meta name="citation_reference" content="MJ Research Answer dated Mar. 26, 2004 filed in Applera v. MJ Research, Düsseldorf Regional Court Case No. 4a O 235/03 (English language translation only)."><meta name="citation_reference" content="MJ Research Motion (Requesting Stay of Suit Pending Outcome of Opposition) dated Dec. 8, 2003 filed in Applera v. MJ Research, Dusseldorf Regional Court Case No. 4a O 262/02 (English language translation only)."><meta name="citation_reference" content="Morrison et al., Anal. Biochem. 183:231-244 (1989)."><meta name="citation_reference" content="Murakami et al., 1989, Nucl. Acids Res. 17: 5587-5595."><meta name="citation_reference" content="Notice of Intervention submitted by MJ Research, Inc. in corresponding European Patent EP 0 872 562."><meta name="citation_reference" content="Opposition Papers submitted by Bibby Sterilin Ltd. in corresponding European Patent EP 0 872 562."><meta name="citation_reference" content="Opposition Papers submitted by Bio-Rad Inc. in corresponding European Patent EP 0 872 562."><meta name="citation_reference" content="Opposition Papers submitted by Cepheid Inc. in corresponding European Patent EP 0 872 562."><meta name="citation_reference" content="Opposition Papers submitted by Eppendorf AG in corresponding European Patent EP 0 872 562."><meta name="citation_reference" content="Opposition Papers submitted by the Secretary of State for Defence, UK in corresponding European Patent EP 0 872 562."><meta name="citation_reference" content="Oser and Valet, Angew. Chem. Int. Engl. 29(10):1167 (1990)."><meta name="citation_reference" content="Richardson, J. Mol. Biol. 78:703-714 (1973)."><meta name="citation_reference" content="Rolf Gunther, &quot;Construction of a Multichannel Fiberoptic Fluorimeter,&quot; Dissertation prepared at The Max Planck Institute for Biophysical Chemistry at Gottingen, pp. 1-82 (in German, and English translation provided)."><meta name="citation_reference" content="Searle and Embrey, Nuc. Acids Res., 18(13):3653-3762 (1990)."><meta name="citation_reference" content="Sharp et al., Biochemistry 12:3055 (1973)."><meta name="citation_reference" content="Sobel and Jain, J. Mol. Biol. 68:21-34 (1972)."><meta name="citation_reference" content="SPEX Catalog (publication date unknown)."><meta name="citation_reference" content="Stephen J. Morris, &quot;Real-Time Multi-Wavelength Fluorescence Imaging of Living Cells,&quot; BioTechniques 8(3) :296-308 (1990)."><meta name="citation_reference" content="Stufen zum Leben, Piper Verlag, 1987, pp. 5, 193-199."><meta name="citation_reference" content="Wittwer et al., 1990, Anal. Biochem. 186: 328-331."><meta name="citation_patent_number" content="US:6814934"><meta name="citation_patent_application_number" content="US:08/968,208"><link rel="canonical" href="http://www.google.com/patents/US6814934"/><meta property="og:url" content="http://www.google.com/patents/US6814934"/><meta name="title" content="Patent US6814934 - Instrument for monitoring nucleic acid amplification"/><meta name="description" content="This invention relates to improved methods for nucleic acid detection using methods such as the polymerase chain reaction (PCR). More specifically, the invention provides methods for simultaneous amplification and detection to enhance the speed and accuracy of prior methods. The methods involve the introduction of detectable DNA binding agents into the amplification reaction, which agents produce a detectable signal that is enhanced upon binding double-stranded DNA. In a preferred embodiment, the binding agent is a fluorescent dye. The methods also provide means for monitoring the increase in product DNA during an amplification reaction."/><meta property="og:title" content="Patent US6814934 - Instrument for monitoring nucleic acid amplification"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("0KfpU7_KKMWUogTgkIG4Dg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407464522.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("AUS"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("0KfpU7_KKMWUogTgkIG4Dg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407464522.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("AUS"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6814934?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6814934"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=5ZtpBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6814934&amp;usg=AFQjCNHqlkvOZcxmJk6J-SCq6F6PW2iI2A" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6814934.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6814934.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6814934" style="display:none"><span itemprop="description">This invention relates to improved methods for nucleic acid detection using methods such as the polymerase chain reaction (PCR). More specifically, the invention provides methods for simultaneous amplification and detection to enhance the speed and accuracy of prior methods. The methods involve the introduction...</span><span itemprop="url">http://www.google.com/patents/US6814934?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6814934 - Instrument for monitoring nucleic acid amplification</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6814934 - Instrument for monitoring nucleic acid amplification" title="Patent US6814934 - Instrument for monitoring nucleic acid amplification"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6814934 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/968,208</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Nov 9, 2004</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Nov 12, 1997</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">May 2, 1991</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2067909A1">CA2067909A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2067909C">CA2067909C</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2218818A1">CA2218818A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2218818C">CA2218818C</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE1256631T1">DE1256631T1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69229929D1">DE69229929D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69229929T2">DE69229929T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69232773D1">DE69232773D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69232773T2">DE69232773T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69232773T3">DE69232773T3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0512334A2">EP0512334A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0512334A3">EP0512334A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0512334B1">EP0512334B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0872562A1">EP0872562A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0872562B1">EP0872562B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0872562B2">EP0872562B2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1256631A1">EP1256631A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5994056">US5994056</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6171785">US6171785</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08968208, </span><span class="patent-bibdata-value">968208, </span><span class="patent-bibdata-value">US 6814934 B1, </span><span class="patent-bibdata-value">US 6814934B1, </span><span class="patent-bibdata-value">US-B1-6814934, </span><span class="patent-bibdata-value">US6814934 B1, </span><span class="patent-bibdata-value">US6814934B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Russell+Gene+Higuchi%22">Russell Gene Higuchi</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Russell+Gene+Higuchi%22">Russell Gene Higuchi</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6814934.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6814934.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6814934.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (60),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (89),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (66),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (28),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (12)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6814934&usg=AFQjCNGprHpr521MSq0eDJlsXWMsRz0h-g">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6814934&usg=AFQjCNHKZMF7hPC9OyxL9n5WNSOqMOmvhw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6814934B1%26KC%3DB1%26FT%3DD&usg=AFQjCNHyWVeA3wU2GCnExkt1tYhInhY3DQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT55346456" lang="EN" load-source="patent-office">Instrument for monitoring nucleic acid amplification</invention-title></span><br><span class="patent-number">US 6814934 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50747115" lang="EN" load-source="patent-office"> <div class="abstract">This invention relates to improved methods for nucleic acid detection using methods such as the polymerase chain reaction (PCR). More specifically, the invention provides methods for simultaneous amplification and detection to enhance the speed and accuracy of prior methods. The methods involve the introduction of detectable DNA binding agents into the amplification reaction, which agents produce a detectable signal that is enhanced upon binding double-stranded DNA. In a preferred embodiment, the binding agent is a fluorescent dye. The methods also provide means for monitoring the increase in product DNA during an amplification reaction.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(10)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6814934B1/US06814934-20041109-D00000.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6814934B1/US06814934-20041109-D00000.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6814934B1/US06814934-20041109-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6814934B1/US06814934-20041109-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6814934B1/US06814934-20041109-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6814934B1/US06814934-20041109-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6814934B1/US06814934-20041109-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6814934B1/US06814934-20041109-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6814934B1/US06814934-20041109-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6814934B1/US06814934-20041109-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6814934B1/US06814934-20041109-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6814934B1/US06814934-20041109-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6814934B1/US06814934-20041109-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6814934B1/US06814934-20041109-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6814934B1/US06814934-20041109-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6814934B1/US06814934-20041109-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6814934B1/US06814934-20041109-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6814934B1/US06814934-20041109-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6814934B1/US06814934-20041109-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6814934B1/US06814934-20041109-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(12)</span></span></div><div class="patent-text"><div mxw-id="PCLM8761150" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6814934-B1-CLM-00001" class="claim">
      <div class="claim-text">1. An instrument for use in monitoring a nucleic acid amplification reaction comprising multiple thermal cycles, comprising:</div>
      <div class="claim-text">(a) an automated thermal cycler capable of alternately heating and cooling, and adapted to receive, at least one reaction vessel containing an amplification reaction mixture comprising a target nucleic acid, reagents for nucleic acid amplification, and a detectable nucleic acid binding agent; and </div>
      <div class="claim-text">(b) a detector operable to detect a fluorescence optical signal while the amplification reaction is in progress and without opening the at least one reaction vessel, which fluorescence optical signal is related to the amount of amplified nucleic acid in the reaction vessel. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6814934-B1-CLM-00002" class="claim">
      <div class="claim-text">2. The instrument of <claim-ref idref="US-6814934-B1-CLM-00001">claim 1</claim-ref>, wherein the thermal cycler is adapted to receive a plurality of reaction vessels, each containing an amplification reaction mixture.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6814934-B1-CLM-00003" class="claim">
      <div class="claim-text">3. The instrument of <claim-ref idref="US-6814934-B1-CLM-00001">claim 1</claim-ref>, wherein the detector is operable to detect a fluorescence optical signal at a wavelength at or about 570 nm.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6814934-B1-CLM-00004" class="claim">
      <div class="claim-text">4. The instrument of <claim-ref idref="US-6814934-B1-CLM-00001">claim 1</claim-ref>, wherein includes a sealed light transmission path between the reaction vessel and the detector.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6814934-B1-CLM-00005" class="claim">
      <div class="claim-text">5. The instrument of <claim-ref idref="US-6814934-B1-CLM-00004">claim 4</claim-ref>, wherein the sealed light transmission path is a fiber optic cable.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6814934-B1-CLM-00006" class="claim">
      <div class="claim-text">6. The instrument of <claim-ref idref="US-6814934-B1-CLM-00001">claim 1</claim-ref>, wherein the thermal cycler is computer-controlled.</div>
    </div>
    </div> <div class="claim"> <div num="7" id="US-6814934-B1-CLM-00007" class="claim">
      <div class="claim-text">7. A system for use in monitoring a nucleic acid amplification reaction comprising multiple thermal cycles, comprising:</div>
      <div class="claim-text">(a) at least one reaction vessel adapted to contain an amplification reaction mixture comprising a target nucleic acid, reagents for nucleic acid amplification, and a detectable nucleic acid binding agent; </div>
      <div class="claim-text">(b) an automated thermal cycler capable of alternately heating and cooling such a reaction vessel, and </div>
      <div class="claim-text">(c) a detector operable to detect a fluorescence optical signal while the amplification reaction is in progress and without opening the at least one reaction vessel, which fluorescence optical signal is related to the amount of amplified nucleic acid in the reaction vessel. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6814934-B1-CLM-00008" class="claim">
      <div class="claim-text">8. The system of <claim-ref idref="US-6814934-B1-CLM-00007">claim 7</claim-ref>, wherein the system comprises a plurality of reaction vessels, each adapted to contain an amplification reaction mixture.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6814934-B1-CLM-00009" class="claim">
      <div class="claim-text">9. The system of <claim-ref idref="US-6814934-B1-CLM-00007">claim 7</claim-ref>, wherein the detector is operable to detect a fluorescence optical signal at a wavelength at or about 570 nm.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" id="US-6814934-B1-CLM-00010" class="claim">
      <div class="claim-text">10. The system of <claim-ref idref="US-6814934-B1-CLM-00007">claim 7</claim-ref>, wherein the at least one reaction vessel includes a clear or translucent cap optically coupled to the detector by a sealed light transmission path.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" id="US-6814934-B1-CLM-00011" class="claim">
      <div class="claim-text">11. The system of <claim-ref idref="US-6814934-B1-CLM-00010">claim 10</claim-ref>, wherein the sealed light transmission path is a fiber optic cable.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6814934-B1-CLM-00012" class="claim">
      <div class="claim-text">12. The system of <claim-ref idref="US-6814934-B1-CLM-00007">claim 7</claim-ref>, wherein the thermal cycler is computer-controlled.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES54331383" lang="EN" load-source="patent-office" class="description">
    <p>This application is a division of application Ser. No. 07/695,201 filed May 2, 1991, now U.S. Pat. No. 5,994,056.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>Field of the Invention</p>
    <p>The present invention provides improved methods for nucleic acid detection. The novel methods for simultaneous nucleic acid amplification and detection enhance the speed and accuracy of prior detection methods and eliminate the need for sample processing following amplification. In a preferred embodiment, the method provides a modification of the polymerase chain reaction and utilizes agents whose fluorescence is enhanced upon binding double-stranded DNA. The methods provided herein have numerous applications, particularly in the fields of molecular biology, medical diagnostics and forensic sciences.</p>
    <p>Description of Related Art</p>
    <p>The disclosed nucleic acid detection methods offer the advantages of speed and simplicity over prior methods for detecting amplified nucleic acids. Nucleic acid detection techniques in general are particularly useful in medical diagnostic assays. For example, Falkow et al., U.S. Pat. No. 4,358,535 disclose a method for detecting pathogens by spotting a sample (e.g., blood, cells, saliva, etc.) on a filter, lysing the cells and fixing the DNA through chemical denaturation and heating. Then, labeled DNA probes are added and allowed to hybridize with the fixed sample DNA. Hybridization indicates the presence of the pathogen's DNA.</p>
    <p>Nucleic acid detection using oligonucleotide probes has become a standard method for specific target detection. Numerous modifications of the method have been described, including culturing the target cells or organisms in situ on the filter, increasing the amount of target nucleic acid available for detection. Generally, these methods require that the DNA sample is noncovalently bound onto a solid support such as nitrocellulose or nylon and then hybridized to a labeled target-specific probe.</p>
    <p>The sensitivity and specificity of nucleic acid detection methods was greatly improved by the invention of the polymerase chain reaction (PCR). PCR is a process for amplifying nucleic acids and involves the use of two oligonucleotide primers, an agent for polymerization, a target nucleic acid template, and successive cycles of denaturation of nucleic acid and annealing and extension of the primers to produce a large number of copies of a particular nucleic acid segment. With this method, segments of single copy genomic DNA can be amplified more than 10 million fold with very high specificity and fidelity. PCR methods are disclosed in U.S. Pat. No. 4,683,202, which is incorporated herein by reference.</p>
    <p>Methods for detecting PCR products are particularly described in U.S. Pat. No. 4,683,195, which is incorporated herein by reference. Those methods require an oligonucleotide probe capable of hybridizing with the amplified target nucleic acid. European Patent Publication No. 237,362, which is incorporated herein by reference, also describes a PCR-based detection method termed “reverse dot blot”, in which the probe, instead of the amplified DNA, is fixed to the membrane. According to the method, the target, rather than the probe, is labeled for hybridization. These methods require separate steps of amplification, capture, and detection and generally require several hours to complete. In the reverse dot-blot method, storage-stable target-specific reagents are preferred.</p>
    <p>Alterative methods for detecting amplified nucleic acids are described in copending U.S. Ser. No. 076,394, filed Jul. 22, 1987, which is incorporated herein by reference. U.S. Ser. No. 076,394 describes PCR-based methods for simultaneous amplification and labeling of a target nucleic acid. The methods require that at least one amplification primer is labeled. The amplification primer can be labeled with, for example, a radioisotope for direct detection of the amplified product or labeled with a reagent suitable for capturing the product onto a solid support for subsequent detection.</p>
    <p>Other means of detection include the use of fragment length polymorphism hybridization, allele-specific oligonucleotide (ASO) probes (Saiki et al., 1986, <i>Nature </i>324:163), or direct sequencing via the dideoxy method using amplified DNA rather than cloned DNA. The fragment length polymorphism method detects insertions and deletions between PCR primers resulting in PCR products of different lengths, detectable by sizing. ASO methods are useful for detecting allelic sequence variations. In an example of ASO hybridization, the amplified DNA is fixed to a nylon filter (by, for example, UV irradiation) in a series of “dot blots,” then allowed to hybridize with an oligonucleotide probe under stringent conditions. This method is also described in copending U.S. Ser. No. 347,495, filed May 4, 1989, which is incorporated herein by reference. The probe may be labeled with, for example, horseradish peroxidase (HRP) and detected by the presence of a blue precipitate following treatment with suitable oxidation reagents.</p>
    <p>Copending U.S. Ser. No. 563,758, filed Aug. 6, 1990 (now U.S. Pat. No. 5,210,015), and incorporated herein by reference, describes an alterative assay method for detecting amplified nucleic acids. The process employs the 5′ to 3′ nuclease activity of a nucleic acid polymerase to cleave annealed, labeled oligonucleotides from hybridized duplexes and release labeled oligonucleotide fragments for detection. The method is suitable for detecting PCR products and requires a primer pair and a labeled oligonucleotide probe having a blocked 3′—OH terminus to prevent extension by the polymerase.</p>
    <p>Due to the enormous amplification possible with the PCR process, small levels of DNA carryover from samples with high DNA levels, positive control templates, or from previous amplifications, can result in PCR product even in the absence of purposefully added template DNA. Higuchi and Kwok, (1989, <i>Nature </i>339:237-238 and Kwok) and Orrego, (in Innis et al., 1990<i>, PCR Protocols: A Guide to Methods and Applications</i>, Academic Press, Inc., San Diego, Calif.), describe particular methods and precautions for practicing PCR with a minimum of cross contamination. U. S. Ser. No. 609,157, filed Nov. 2, 1990, describes improved methods for reducing the effects of cross contamination by the introduction of unconventional nucleotide bases. These references are incorporated herein by reference. Because the possibility of introducing contaminating DNA to a sample will be increased as the amount of handling steps required for sample preparation, processing, and analysis is increased, it would be preferable to minimize sample handling, particularly after the amplification reaction is complete.</p>
    <p>A number of agents have been described for labeling nucleic acids, whether probe or target, for facilitating detection of target nucleic acid. Suitable labels may provide signals detectable by fluorescence, radioactivity, colorimetry, X-ray diffraction or absorption, magnetism or enzymatic activity and include, for example, fluorophores, chromophores, radioactive isotopes (particularly <sup>32</sup>P and <sup>125</sup>I) electron-dense reagents, enzymes, and ligands having specific binding partners.</p>
    <p>Labeling is achieved by a number of means, such as chemical modification of a primer or probe to incorporate a label or the use of polymerizing agents to incorporate a modified nucleoside triphosphate into an extension product. Intercalating agents non-covalently bind the stacked bases of nucleic acids and as a result the fluorescence of the agent either increases or shifts to a different wavelength. For example, U.S. Pat. No. 4,582,789 describes several intercalating moieties including psoralens. Copending U.S. Ser. No. 076,394 describes methods for amplifying and detecting nucleic acids using psoralen labeled primers. Both the '789 patent and the '394 application are incorporated herein by reference.</p>
    <p>Fluorescent dyes are suitable for detecting nucleic acids. For example, ethidium bromide is an intercalating agent that displays increased fluorescence when bound to double-stranded DNA rather than when in free solution (Sharp et al., 1973, <i>Biochemistry </i>12:3055). Ethidium bromide can be used to detect both single- and double-stranded nucleic acids, although the affinity of ethidium bromide for single-stranded nucleic acid is relatively low. Ethidium bromide is routinely used to detect nucleic acids following gel electrophoresis. Following size fractionation on an appropriate gel matrix, for example, agarose or acrylamide, the gel is soaked in a dilute solution of ethidium bromide. The DNA is then visualized by examining the gel under UV light (see Maniatis et al., 1982 eds., <i>Molecular Cloning: A Laboratory Manual</i>, N.Y., Cold Spring Harbor Laboratory.)</p>
    <p>Alternative fluorescence based methods for detecting DNA have been described. For example, Morrison et al., 1989, <i>Anal. Biochem., </i>183:231-244, which is incorporated herein by reference, describe a two probe method for detecting target DNA. One probe is labeled with fluorescein, the other probe, complementary to the first, is labeled with a quencher for fluorescein emission. The probes are allowed to anneal with denatured DNA containing the target sequence, and the amount of fluorescence is determined. Fluorescence increases to the extent that the fluorescein probe binds to unlabeled, complementary DNA rather than the complementary, quenching probe.</p>
    <p>Mabuchi et a., 1990, <i>Nucl. Acids Res. </i> <b>18(24):7461-7462</b>, which is incorporated herein by reference, describe a method for detecting DNA fragments based on the AT content of the nucleic acid segment. Two fluorochromes are used to stain size fractionated DNA in an agarose gel. The selective binding properties of different fluorochromes for AT rich regions are used to distinguish electrophoresed DNA fragments.</p>
    <p>U.S. Pat. No. 4,257,774, which is incorporated herein by reference, describes the direct binding of fluorescent intercalators to DNA, e.g., ethidium salts, daunomycin, mepacrine and acridine orange, as well as 4′6-diamidino-α-phenylindole to quantitate the DNA. Fluorescence polarization is used for characterization of non-fluorescent DNA binding compounds which complete with the DNA binding dyes.</p>
    <p>Oser and Valet (1990, <i>Angew. Chem. Int. Engl. </i>29(10): 1167) describe a nucleic acid detection scheme that requires two oligonucleotide probes complementary to adjacent sites on a target. The probes are labeled differentially with either a salicylate or a DTPA ligand bearing a fluorescence emitter, Tb<sup>III</sup>. Hybridization of both probes to the target provides steric proximity of the two labels resulting in a measurable increase in Tb<sup>III </sup>fluorescence. The modified probes are prepared specifically for each target to be detected.</p>
    <p>European Patent Publication No. 070,685 describes the use of fluorescent labeled polynucleotide probes in polynucleotide hybridization assays. According to the method, probes are prepared by attaching particular absorber-emitter moieties to the 3′ and 5′ ends of nucleic acid fragments. The fragments are capable of hybridizing to adjacent positions on a target DNA, so that, if both fragments are hybridized, the proximity of the absorber and emitter moieties results in detectable emitter fluorescence.</p>
    <p>According to these methods, the fluorescent dye is introduced to the target DNA after all in vitro nucleic acid polymerization reactions have been completed The inhibitory effects of intercalating agents on nucleic acid polymerases have been described in numerous publications (see for example, Kornberg, 1974, <i>DNA Synthesis</i>, W. H. Freman and Co., San Francisco, and Richardson, 1973, <i>J. Mol Biol. </i>78:703-714, which is incorporated herein by reference).</p>
    <p>DNA binding dyes are useful as antibiotics because of the inhibitory effects on nucleic acid replication processes that result from the agent binding to the template. European Patent Publication No. 169,787 describes the use of intercalating agents for blocking replication of influenza or herpes virus. Kornberg (supra) describes a number of DNA binding agents, both intercalators and non-intercalators, and describes how each compound inhibits nucleic acid replication. At page 227, Kornberg specifically describes that ethidium bromide inhibits DNA replication.</p>
    <p>A method for simultaneous amplification and detection of target nucleic acids would provide advantages over prior detection methods. Such a method would minimize the problems of sample contamination inherent in any process involving a series of manipulative steps for discerning a positive or negative test result By eliminating sample handling and processing steps, a method for simultaneous amplification and detection of target nucleic acids would increase the speed and accuracy of current diagnostic methods. The present invention addresses and solves these needs.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The present invention provides a method for detecting a target nucleic acid in a sample. The method comprises the steps of: (a) providing an amplification reaction mixture that comprises a sample, a DNA binding agent, wherein said agent is characterized as providing a detectable signal when bound to double-stranded nucleic acid which signal is distinguishable from the signal provided by said agent when it is unbound, and reagents for amplification; (b) determining the amount of signal produced by the mixture of step (a); (c) treating said mixture under conditions for amplifying the target nucleic acid; (d) determining the amount of said signal produced by the mixture of step (c); and (e) determining if amplification has occurred.</p>
    <p>The invention is particularly suitable for practice in PCR amplification methods wherein a net increase in double-stranded DNA results in a change in signal strength or type. In a preferred embodiment, the DNA binding agent is a fluorescent DNA binding dye, such as ethidium bromide, and the signal is fluorescence.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 demonstrates the increased fluorescence of the PCR mixture due to amplification of a specific target DNA in the presence of ethidium bromide. The experiment is described in detail in Example II.</p>
    <p>FIG. 2 demonstrates the target specificity of the present detection method and illustrates some of the quantitative aspects of the invention. The details of the experiment are provided at Example IV.</p>
    <p>FIG. 3 demonstrates the use of the present invention for genetic screening. The experiment is described in detail at Example V.</p>
    <p>FIGS. 4A and 4B relate to the quantitative homogeneous assay described in Example VII. FIG. 4A demonstrates that the increase in fluorescence in the positive samples is greater than two standard deviations away from the average of the negative samples. FIG. 4B graphically describes a background substraction method for measuring fluorescence.</p>
    <p>FIGS. 5A and 5B demonstrate the results of an on-line, automated homogeneous amplification and detection system, according to the present invention as demonstrated in Example VIII. FIG. 5A shows a fluorescence print-out from a PCR containing no target DNA and serves as a negative control. FIG. 5B is a fluorescence print out from a PCR containing the appropriate target and demonstrates the continuous monitoring of the PCR and concurrent increase in PCR product.</p>
    <p>FIGS. 6A and 6B schematically show a detector and a thermal cycler according to certain embodiments of the invention.</p>
    <p>FIG. 7 schematically shows certain embodiments of the invention that employ a fiber optic cable.</p>
    <p>FIG. 8 schematically shows certain embodiments of the invention in which a thermal cycler is computer-controlled.</p>
    <heading>DETAILED DESCRIPTION</heading> <p>The present invention provides improved methods for detecting nucleic acids and is especially suited for use in conjunction with amplification processes. The improved methods require neither an oligonucleotide probe nor a labeled amplification primer. The methods enable monitoring the accumulation of product while the amplification reaction is in progress. The invention also allows target specific quantitation. These methods are suitable for use in automated formats.</p>
    <p>The methods disclosed offer vast improvements over prior methods for detecting amplified nucleic acids. According to the invention, amplified nucleic acids are detected without opening the reaction vessel once the amplification reaction is initiated and without any additional handling or manipulative steps subsequent to the reaction. Prior to the present invention, nucleic acid detection methods required a third oligonucleotide reagent as a probe, or a series of manipulative and time consuming steps for target detection, for example, capturing the product on a solid support, which steps are performed after amplification; In some procedures both capture and probe hybridization steps are required. The present invention eliminates the need for using a hybridizing probe reagent or a capture procedure for detecting the amplified target. In a clinical setting the methods of the invention offer speed, simplicity, and decreased opportunity for cross-contamination between samples, particularly between amplified and non-amplified samples. In addition, the present methods offer means for automated detection, monitoring and quantitation of amplification products during the amplification process and after the amplification reaction is complete.</p>
    <p>In various embodiments, the present methods utilize a detectable agent capable of binding double-stranded DNA. The detectable binding agent may be a fluorescent dye or other chromophore, enzyme, or agent capable of producing a signal, directly or indirectly, when bound to double-stranded DNA. The agent may be also characterized as binding to single-stranded DNA or RNA. It is only necessary that the agent is capable of producing a detectable signal when bound to a double-stranded nucleic acid that is distinguishable from the signal produced when that same agent is in solution or bound to a single-stranded nucleic acid.</p>
    <p>In one embodiment, the DNA binding agent is an intercalating agent. As used herein, an intercalating agent is an agent or moiety capable of non-covalent insertion between stacked base pairs in the nucleic acid double helix. Intercalating agents, such as ethidium bromide, fluoresce more intensely when intercalated into double-stranded DNA than when bound to single-stranded DNA, RNA, or in solution. Other intercalating agents exhibit a change in the fluorescence spectra when bound to double-stranded DNA. For example, actinomycin D fluoresces red when bound to single-stranded nucleic acids, and green when bound to a double-stranded template. Whether the detectable signal increases, decreases or is shifted, as is the case with actinomycin D, any intercalating agent that provides a detectable signal that is distinguishable when the agent is bound to double-stranded DNA or unbound is suitable for practicing the disclosed invention. For example, the interaction between DNA and another photoreactive psoralen, 4-aminomethyle-4-5′8-trimethylpsoralen (AMT) has been described (see Johnson et al., 1981, <i>Photochem. &amp; Photobiol., </i>33:785-791, which is incorporated herein by reference). According to the reference, both the absorption at long wavelengths and fluorescence, decline upon intercalation of AMT into the DNA helix.</p>
    <p>Non-intercalating DNA binding agents are also suitable. For example, Hoechst 33258 (Searle &amp; Embrey, 1990, <i>Nuc. Acids Res. </i>18(13):3753-3762) exhibits altered fluorescence with increasing amount of target. Hoechst 33258 is a member of a class of DNA-binding compounds commonly referred to as “groove binders.” This group includes drugs like distamycin, netropsin and others. These compounds recognize and bind the minor groove of duplex DNA.</p>
    <p>According to the present invention, a DNA binding agent produces a detectable signal directly or indirectly. The signal is detectable directly, such as by fluorescence or absorbance, or indirectly via a substituted label moiety or binding ligand attached to the DNA binding agent. For indirect detection any moiety or ligand that is detectably affected by proximity to double-stranded DNA is suitable.</p>
    <p>According to the invention, the detectable binding agent is present in the amplification reaction during the amplification process. As amplification proceeds, the agent produces a detectable signal. Neither the agent nor the signal prevents amplification from proceeding. Consequently, the agent may be added to the reaction mixture prior to amplification or while the reaction is in progress. For example, the agent is included in an amplification buffer comprising appropriate reagents, such as salts and buffering agents. In this manner, it is not necessary to separately add the binding agent to the amplification reaction. For practice of the present invention, any DNA binding agent is suitable, so long as in the presence of that agent a net increase in the amount of double-stranded DNA present is reflected in a change signal intensity that is detectable directly or indirectly. In a preferred embodiment, the detectable signal is fluorescence.</p>
    <p>The term “homogeneous detection assay” is used in the present specification to describe the claimed invention. For ease of understanding, the following definition is provided. Homogeneous detection assay refers to a method for coupled amplification and detection, wherein the process of amplification generates a detectable signal and the need for subsequent sample handling and manipulation to detect the amplified product is minimized or eliminated.</p>
    <p>The present homogeneous assay is suitable for use in conjunction with oligonucleotide probes. For example, in one embodiment the use of an oligonucleotide probe, specific for detecting a particular target sequence, is included in the amplification reaction in addition to the DNA binding agent of the present invention. The probe, labeled with a quencher and fluorophore, hybridizes to the amplified target nucleic acid. In the presence of an agent for polymerization capable of 5′ to 3′ nucleolytic activity, the fluorophore and quencher, when bound to the target, are separated by degradation of the probe by the polymerase. The fluorescence of the unbound probe is detectably distinct from the fluorescence of the bound, and subsequently hydrolyzed probe. Thus, the fluorescence of the DNA binding agent enables detection that amplification has occurred, and the fluorescence of the hybridized probe indicates target specific amplification. So long as amplification is detectable without opening the reaction vessel, or further processing steps once amplification is initiated, the method is within the present definition of a homogeneous assay.</p>
    <p>The term “amplification reaction system” refers to any in vitro means for multiplying the copies of a target sequence of nucleic acid. Such methods include but are not limited to polymerase (PCR), DNA ligase, (LCR), Qβ RNA replicase, and RNA transcription-based (TAS and 3SR) amplification systems.</p>
    <p>The term “amplifying” which typically refers to an “exponential” increase in target nucleic acid is being used herein to describe both linear and exponential increases in the numbers of a select target sequence of nucleic acid.</p>
    <p>The term “amplification reaction mixture” refers to an aqueous solution comprising the various reagents used to amplify a target nucleic acid. These include enzymes, aqueous buffers, salts, amplification primers, target nucleic acid, and nucleoside triphosphates. Depending upon the context, the mixture can be either a complete or incomplete amplification reaction mixture.</p>
    <p>The systems described below are practiced routinely by those of skill in the relevant art. They have been described in detail by others and are summarized below. This invention is not limited to any particular amplification system. As other systems are developed, those systems may benefit by practice of this invention. A recent survey of amplification systems was published in <i>Bio/Technology </i>8:290-293, Apr. 1990, incorporated herein by reference. The following four systems are described below for the convenience of those not familiar with amplification systems and to provide an understanding of the breadth of the present invention.</p>
    <p>Amplification of DNA by PCR is disclosed in U.S. Pat. Nos. 4,683,195 and 4,683,202 (both of which are incorporated herein by reference). Methods for amplifying and detecting nucleic acids by PCR using a thermostable enzyme are disclosed in U.S. Pat. No. 4,965,188, which is incorporated herein by reference.</p>
    <p>PCR amplification of DNA involves repeated cycles of heat-denaturing the DNA, annealing two oligonucleotide primers to sequences that flank the DNA segment to be amplified, and extending the annealed primers with DNA polymerase. The primers hybridize to opposite strands of the target sequence and are oriented so that DNA synthesis by the polymerase proceeds across the region between the primers, effectively doubling the amount of the DNA segment. Moreover, because the extension products are also complementary to and capable of binding primers, each successive cycle essentially doubles the amount of DNA synthesized in the previous cycle. This results in the exponential accumulation of the specific target fragment, at a rate of approximately 2 n per cycle, where n is the number of cycles.</p>
    <p>In the disclosed embodiment, Taq DNA polymerase is preferred although this is not an essential aspect of the invention. Taq polymerase, a thermostable polymerase, is active at high temperatures. Methods for the preparation of Taq are disclosed in U.S. Pat. No. 4,889,818 and incorporated herein by reference. Taq polymerase is available from Perkin Elmer Cetus Instruments (PECI). However, other thermostable DNA polymerases isolated from other <i>Thermus </i>species or non <i>Thermus </i>species(e.g., <i>Thermus thermophilus </i>or <i>Thermotoga maritima</i>), as well as non-thermostable DNA polymerases such as T4 DNA polymerase, T7 DNA polymerase, <i>E. coli </i>DNA polymerase I, or the Klenow fragment of <i>E. coli</i>, can also be used in PCR. Methods for providing thermostable DNA polymerases are provided in copending Ser. Nos. 455,967, filed Dec. 22, 1989 (now U.S. Pat. No. 5,618,711); 567,244, filed Aug. 13, 1990 (now U.S. Pat. No. 5,374,553); and 590,213, 590,466 (now U.S. Pat. No. 5,455,170), and 590,490, filed Sep. 28, 1990, which are all incorporated herein by reference.</p>
    <p>As used herein, the term “primer” refers to an oligonucleotide capable of acting as a point of initiation of DNA synthesis when annealed to a nucleic acid template under conditions in which synthesis of a primer extension product is initiated, i.e., in the presence of four different nucleotide triphosphates and a DNA polymerase in an appropriate buffer (“buffer” includes pH, ionic strength, cofactors, etc.) and at a suitable temperature.</p>
    <p>The nucleoside-5′-triphosphates utilized in the extension process, typically dATP, dCTP, dGTP, and dTTP, are present in total concentration typically ranging from 400 μM to 4.0 mM during the extension reaction, although preferably the concentration is between 500 μM and 1.5 mM.</p>
    <p>The choice of primers for use in PCR determines the specificity of the amplification reaction. Primers used in the present invention are oligonucleotides, usually deoxyribonucleotides several nucleotides in length, that can be extended in a template-specific manner by the polymerase chain reaction. The primer is sufficiently long to prime the synthesis of extension products in the presence of the agent for polymerization and typically contains 10-30 nucleotides, although that exact number is not critical to the successful application of the methods Short primer molecules generally require cooler temperatures to form sufficiently stable hybrid complexes with the template.</p>
    <p>Synthetic oligonucleotides can be prepared using the triester method of Matteucci et al., 1981, <i>J. Am. Chem. Soc. </i>103:3185-3191. Alteratively automated synthesis may be preferred, for example, on a Biosearch 8700 DNA Synthesizer using cyanoethyl phosphoramidite chemistry.</p>
    <p>For primer extension to occur, this primer must anneal to the nucleic acid template. Not every nucleotide of the primer must anneal to the template for extension to occur. The primer sequence need not reflect the exact sequence of template. For example, a non-complementary nucleotide fragment may be attached to the 5′ end of the primer with the remainder of the primer sequence being complementary to the template. Alteratively, non-complementary bases can be interspersed into the primer, provided that the primer sequence has sufficient complementarily with the template for annealing to occur and allow synthesis of a complementary DNA strand.</p>
    <p>Amplification systems such as PCR require a target nucleic acid in a buffer compatible with the enzymes used to amplify the target. The target nucleic acid can be isolated from a variety of biological materials including tissues, body fluids, feces, sputum, saliva, plant cells, bacterial cultures, and the like.</p>
    <p>In general, the nucleic acid in the sample will be a sequence of DNA, most usually genomic DNA. However, the present invention can also be practiced with other nucleic acids, such as messenger RNA, ribosomal RNA, viral RNA, or cloned DNA. Suitable nucleic acid samples include single or double-stranded DNA or RNA for use in the present invention. Those of skill in the art will recognize that whatever the nature of the nucleic acid, the nucleic acid can be amplified merely by making appropriate and well recognized modifications to the method being used.</p>
    <p>To amplify a target nucleic acid sequence in a sample, the sequence must be accessible to the components of the amplification system. In general, this accessibility is ensured by isolating the nucleic acids from a crude biological sample. A variety of techniques for extracting nucleic acids from biological samples are known in the art. For example, see those described in Maniatis et al., <i>Molecular Cloning: A Laboratory Manual </i>(New York, Cold Spring Harbor Laboratory, 1982); Arrand, Preparation of Nucleic Acid Probes, in pp. 18-30<i>, Nucleic Acid Hybridization: A Practical Approach </i>(Ed Hames and Higgins, IRL Press, 1985); or, in <i>PCR Protocols</i>. Chapters 18-20 (Innis et al., ed., Academic Press, 1990).</p>
    <p>Those skilled in the art will know that the PCR process is most usually carried out as an automated process with a thermostable enzyme. In this process, the reaction mixture is cycled through a denaturing temperature range, a primer annealing temperature range, and an extension temperature range. A machine specifically adapted for use with a thermostable enzyme is disclosed more completely in EP No. 236,069, which is incorporated herein by reference, and is commercially available from PECI.</p>
    <p>The ligase chain reaction is described in PCT Patent Publication No. WO 89/09835, which is incorporated herein by reference. The process involves the use of ligase to join oligonucleotide segments that anneal to the target nucleic acid. Ligase chain reaction (LCR) results in amplification of an original target molecule and can provide millions of copies of product DNA. Consequently, the LCR results in a net increase in double-stranded DNA. The present detection methods are applicable to LCR, as well as PCR. LCR requires an oligonucleotide probe for detecting the product DNA. When used in conjunction with the disclosed methods for detecting amplification products, a probe step is unnecessary, and the LCR result is immediately detectable.</p>
    <p>Another amplification scheme exploits the use of the replicase from the RNA bacteriophage Qβ. In this amplification scheme, a modified recombinant bacteriophage genome with a sequence specific for the targeted sequence is initially hybridized with the nucleic acid to be tested. Following enrichment of the duplexes formed between the bacteriophage probe and the nucleic acid in a sample, Qβ replicase is added, which, upon recognizing the retained recombinant genome, begins making large numbers of copies.</p>
    <p>The Qβ system does not require primer sequences and there is no heat denaturation step as with the PCR and LCR amplification systems. The reaction occurs at one temperature, typically 37° C. The preferred template is a substrate for the Qβ replicase, midvariant-1 RNA. A very large increase in the templates is achieved through the use of this system. A review of this amplification system can be found in the International Patent Application Pub. No. WO 87/06270 and in Lizardi et al., 1988, <i>Bio/Technology </i>6:1197-1202.</p>
    <p>The 3SR system is a variation of an in vitro transcription based amplification system. A transcription-based amplification system (TAS) involves the use of primers that encode a promoter to generate DNA copies of a target strand and the production of RNA copies from'the DNA copies with an RNA polymerase. See, e.g., Example 9B of U.S. Pat. No. 4,683,202 and EP No. 310,229. The 3SR System is a system which uses three enzymes to carry out an isothermal replication of target nucleic acids.</p>
    <p>The system begins with a target of single-stranded RNA to which a T7 RNA DNA primer is bound. By extension of the primer with reverse transcriptase, a cDNA is formed, and RNAseH treatment frees the cDNA from the heteroduplex. A second primer is bound to the cDNA and a double stranded cDNA is formed by DNA polymerase (i.e., reverse transcriptase) treatment. One (or both) of the primers encodes a promoter, i.e., the promoter for T7 RNA polymerase, so that the double-stranded cDNA is transcription template for T7 RNA polymerase.</p>
    <p>Transcription competent cDNAs yield antisense RNA copies of the original target. The transcripts are then converted by the reverse transcriptase to double standard cDNA containing double-stranded promoters, optionally on both ends in an inverted repeat orientation. These DNAs can yield RNAs, which can reenter the cycle. A more complete description of the 3SR system can be found in Guatelli et al., 1990, <i>Proc. Natl. Acad. Sci. </i>USA 87:1874-1878, and EP No. 329,822, both of which are incorporated herein by reference. The TAS system is also described in Gingeras et al. in Innis et al. eds., 1990, P Protocols, Academic Press, San Diego, which is incorporated herein by reference.</p>
    <p>The embodiments of the invention exemplified herein disclose the process for detecting amplified nucleic acids in conjunction with PCR. Accordingly, when used in a PCR-based method, the DNA binding agent is characterized as an agent that will not prevent primer annealing, primer extension along a complementary template, or strand separation of a DNA duplex.</p>
    <p>In the process described herein, a sample is provided which contains, or is suspected of containing, a particular oligonucleotide sequence of interest, the “target nucleic acid.” The target may be RNA or DNA or an RNA/DNA hybrid. The target may be single stranded or double stranded. Target preparation will be carried out in a manner appropriate for the particular amplification process to be implemented. For example, in a PCR method where the target nucleic acid is single-stranded DNA, such as mRNA, the target may be first reverse-transcribed into cDNA, prior to amplification.</p>
    <p>Methods for reverse transcribing RNA into cDNA are well known and described in Maniatis et al., supra. Alteratively, preferred methods for reverse transcription utilize thermoactive DNA polymerases. These methods are described in commonly assigned, copending, U.S. Ser. No. 455,611, filed Dec. 22, 1989 (now U.S. Pat. No. 5,322,770), and incorporated herein by reference. U.S. Ser. No. 455,611 describes a procedure for coupled reverse transcription/amplification of an RNA template using a thermostable DNA polymerase. The present specification teaches that intercalating agents do not prevent DNA polymerase activity. Consequently, the present method provides a homogeneous detection assay for RNA targets as well as DNA targets.</p>
    <p>In another embodiment of the present invention, nested primers are used (Mullis et al., 1986<i>, Cold Spring Harbor Symposium on Quantitative Biology </i>51:263, incorporated herein by reference). This method may be preferred when the amount of nucleic acid in a sample is extremely limited, for example, where archival, paraffin embedded samples are used. When nested primers are used, the nucleic acid is first amplified with an outer set of primers. This amplification reaction is followed by a second round of amplification cycles using an inner set of primers. Examples VI and VII describe modifications of nested primer methods that provide superior results without necessitating the additional sample handling required by using nested primers in successive rounds of amplification cycles.</p>
    <p>According to the present invention, the generation of amplification products can be monitored while the reaction is in progress. An apparatus <b>11</b> for detecting the signal generated by the binding agent, can be used to detect, measure, and quantify the signal before, during, and after amplification. Of course, the particular type of signal may dictate the choice of detection method. For example, in a preferred embodiment of the invention, fluorescent DNA binding dyes are used to label PCR products. The dyes intercalate, or bind, the double-stranded PCR products, and consequently, the resulting fluorescence increases as the amount of double-stranded DNA increases. The amount of fluorescence can be quantitated by measurement using a spectra-fluorometer <b>12</b> with or without opening the PCR vessel <b>13</b>. Examples VII and VIII demonstrate this aspect of the invention.</p>
    <p>In Example VIII the fluorescence of the positive control sample was well above background measurements. Because signal generation was measured without having to open the reaction tube, this method of detection is readily adaptable to an automated format in which signal is monitored throughout the amplification process. Example VIII demonstrates an automated, on-line PCR detection method: a fiber optic lead <b>14</b> was used to input excitation light directly to a PCR tube in a heating/cooling block. The same fiber optic was used to return fluorescent emissions back to the spectra-fluorometer, where the value was read.</p>
    <p>In preferred methods for PCR, the amplification reaction is carried out as an automated process. A thermocycler <b>15</b> currently available from Perkin Elmer Cetus Instruments uses a heat block capable of holding up to 48 reaction tubes. Consequently, 48 amplification reactions can be carried out simultaneously. The present invention permits PCR product detection in all 48 samples, without handling the samples, opening tubes, or interrupting the cycling reaction. A suitable optical system moves the excitation light from the source <b>16</b> to the reaction tube and measures the emission light from each tube. For example, multiple fiber optic leads simultaneously read all PCR tubes undergoing thermocycling. However, only a single fluorometer is needed to read fluorescence from the reaction tubes, as each fiber optic can be read rapidly one at a time, for example, during the time frame of a PCR temperature soak. Alternatively, it will be obvious to one skilled in the art that such a detection system is not necessarily limited to a particular thermocycler machine or number of reaction vessels. However, the description of the 48 well PECI thermocycler serves to demonstrate this aspect of the present invention.</p>
    <p>So long as the reaction wells, or tubes, are light sealed to prevent external light sources from influencing fluorescence detection, any over plate, tube cap, or lid apparatus that comprises or can be attached to, a fiber optic lead is suitable. In the embodiment of the invention at Example VIII, the reaction tube lids were removed to accommodate the fiber optic. However, use of a reaction vessel that has a clear or translucent cap <b>17</b> eliminates the need to insert the cable into the tube. It will be apparent that reaction tubes able accommodate a fiber optic cable, without the cable physically contacting the amplification reaction components, is desirable. In a spectrafluorometer capable of heating and cooling a surface, or vessel, an optic fiber is not required. The optic fiber is only necessary where a thermocycler and spectrafluorometer are housed independently.</p>
    <p>An analogous detection scheme is suitable in a 96-well microtiter format. This type of format is frequently desirable in clinical laboratories for large scale sample screening, for example, for genetic analysis such as screening for sickle-cell anemia or the AIDS virus in blood bank screening procedures. The present invention is suitable for this type of analysis and eliminates the need for the numerous washing and extraction procedures that are required with known “in-well” assay procedures such as ELISA type formats or other optical density-based methods. (See Kolber et al., 1988, <i>J. of Immun. Meth. </i>108:255-264, Huschtscha et al., 1989<i>, In Vitro Cell and Dev. Biol. </i>25(1): 105-108, and Voller et al., 1979, <i>The Enzyme Linked Immunosorbent Assay</i>, Dynatech Labs, Alexandria, Va.).</p>
    <p>The present detection methods also allow direct fluorescence measurement using an apparatus similar to ELISA plate reader, but designed to excite and measure fluorescence. For example, the CytoFluor™ 2300 machine manufactured by Millipore is suitable in such a method. It is appropriate to “read” the microtiter plate before and after thermocycling for determining background fluorescence. Alternatively, an apparatus providing a continuous determination of fluorescence is useful for monitoring the increase in PCR product during the amplification reaction.</p>
    <p>In another embodiment of the invention, following amplification, the size of the amplified product is determined without the use of a probe or size fractionation methods such as HPLC or gel electrophoresis. Copending U.S. Ser. No. 601,840, filed Oct. 23, 1990 (now U.S. Pat. No. 5,269,937), which is incorporated herein by reference, describes a method for determining the average molecular weight of a PCR product using light scattering. The method is suitable for use in conjunction with the present invention especially when the homogeneous assay result is detected using a spectra fluorometer. A fluorometer reads emissions at the fluorescence wavelength, according to the present invention, and measures light scattering, for example, at a 180° angle. This aspect of the invention is particularly useful for determining if amplification has occurred and simultaneously distinguishing the amplified target product from, for example, primer-dimer and high molecular weight DNA.</p>
    <p>In a preferred embodiment of the invention, the detectable binding agent is an intercalating fluorescent dye. The present examples describe that ethidium bromide is included in the amplification reaction mixture. Ethidium bromide, like other DNA binding dyes, such as acridines, proflavine, acridine orange, acriflavine, fluorcoumanin, ellipticine, daunomycin, chloroquine, distamycin D, chromomycin, homidium, mithramycin, ruthenium polypyridyls, and anthramycin, exhibit altered fluorescence emissions when bound to double-stranded DNA. In an amplification reaction, a large amount of double-stranded DNA is generated from a starting template. Consequently, when this occurs in the presence of an intercalating fluorescent dye, a net increase in DNA-dependent fluorescence results. The target specificity of PCR and the use of appropriate positive and negative controls ensure that a detectable increase in fluorescence is due to the presence of the amplified target nucleic acid.</p>
    <p>The present specification incudes several examples demonstrating various aspects of the homogeneous detection assay. The particular embodiments describe that ethidium bromide is present in the PCR at a concentration of 0.53 to 1.27 μM although a concentration of 0.08 μM (0.03 μg/ml) to 40.6 μM (16μg/ml) is also suitable; however, an ethidium bromide concentration in the PCR mix in the range of 0.15 μM (0.06 μg/ml) to 20.3 μM (8 μg/ml) is preferred. It will be readily apparent to those of ordinary skill in the art how to determine a suitable concentration of alternative DNA binding agents by empirical adjustment A suitable concentration of agent is any amount that provides a signal that is distinguishable when a net increase in double-stranded DNA has occurred in an amplification reaction. Consequently, the preferred concentration of agent included in the amplification reaction mix may vary depending on the amount of double-stranded non-target DNA (i.e., background genomic DNA), the copy number and amount of target, the quantity and fluorescence of the amplification primers and the particular agent utilized. For example, a standard curve using a known amount of target and varying the amount of binding agents may be appropriate. The fluorescent dye is added to the PCR mixture before starting temperature cycling. A suitable fluorescent dye does not prevent amplification from occurring. It will be obvious to one of ordinary skill in the art to determine the suitability of any novel dye in the method. Example I demonstrates that PCR amplification occurs in the presence of detectable amounts of ethidium bromide.</p>
    <p>The emission spectrum of ethidium bromide peaks at 650 nm for unbound ethidium bromide and at 611 nm when the agent is bound to double-stranded DNA. However, because the emission spectra of bound and unbound ethidium bromide have distinct peaks, with bound ethidium emitting at a lower wavelength, discrimination between bound and unbound ethidium is enhanced by detecting emission at a non-peak wavelength that is less than the peak for bound ethidium. In the disclosed embodiment at Example VII, the fluorescence emission of the sample was detected at 570 nm. At Example VIII, the excitation wavelength was set at 500 nm and emission detection was set at 570 nm.</p>
    <p>It is not essential that the detection or excitation wavelength is optimized for practice of the invention. For example, in Example II, a U.V. light box with an excitation wavelength at 300 nm and detection by photography through a red filter is suitable. A spectra fluorometer, depending on the features of the particular machine utilized, offers the opportunity to set the excitation and emission wavelength, as well as bandwidth. It will be obvious to one of ordinary skill in the art how to determine the wavelength and bandwidth settings for a particular DNA binding agent to be detected. Thus, although each agent has a discrete fluorescence spectrum, a broad range of detection wavelengths are suitable for practicing the invention, as exemplified herein. General guidance is found in, for example, The Merck Index, (eds. Budavari et al., 1989, Merck Co. Inc. Rahway, N.J.) where peak emission wavelengths for particular fluorescent agents bound and unbound to helical DNA are described at each entry. Similarly, the Molecular Probes, Inc. (Eugene, Oreg.) Catalogue, 1990, by Haugland is a suitable reference for describing DNA binding agents useful in the present invention.</p>
    <p>In general, it is preferred but not essential that the DNA polymerase is added to the PCR reaction mixture after both the primer and template are added. Alteratively, for example, the enzyme and primer are added last or the PCR buffer or template plus buffer are added last. It is generally desirable that at least one component that is essential for polymerization not be present until such time as the primer and template are both present, and the enzyme can bind to and extend the desired primer/template substrate (see U.S. patent application Ser. No. 481,501, filed Feb. 16, 1990, now U.S. Pat. No. 5,411,876, which is incorporated herein by reference).</p>
    <p>Commonly assigned, copending application U.S. Ser. No. 609,157, filed Nov. 2, 1990, and incorporated herein by reference, describes improved methods for reducing the effects of cross contamination of amplification reactions. The methods require the introduction of unconventional bases into the amplified product and exposing carryover to enzymatic and/or physical chemical treatment which effectively render the product incapable of serving as a template for subsequent amplifications. The homogeneous detection assay described herein is suitable in conjunction with the sterilization methods described in U.S. Ser. No. 609,157. These methods enhance the accuracy and reliability of amplification results by eliminating steps, and thereby minimizing product handling, which reduces carryover. The sterilization method provides additional assurance that carryover template is eliminated.</p>
    <p>Preferably, the DNA binding agent is storage stable and can be included as a component in a PCR reagent buffer. Thus, the invention provides novel reagents suitable for commercialization in a kit format. Such a reagent may contain a solution of the DNA binding agent in a kit for detecting nucleic acids by PCR. Alteratively, the reagent might contain a DNA binding agent, as well as other PCR buffer components such as Tris-HCl, KCl, and MgCl<sub>2</sub>, each in appropriate concentrations for carrying out PCR. In one embodiment, a kit includes a buffer comprising ethidium bromide at a suitable concentration to provide, in an amplification reaction, a final ethidium bromide concentration in the range of 0.15 μM to 20.3 μM. The buffer may additionally contain any or all of the following reagents: Tris-HCl, pH 8.0-8.3; KCl, and MgCl<sub>2 </sub>each in appropriate concentrations for PCR amplification. Kits for detecting amplified nucleic acids are also envisioned as including any of the following: an agent for polymerization, dNTPs, appropriate primers, and a positive control template.</p>
    <p>The '202 patent describes methods for preparing and using primers for PCR which have non-complementary sequences added to the 5′ end. These “tails” are useful for engineering particular restriction sites or other purposes, because during PCR, the non-complementary tail sequence is incorporated into the double-stranded PCR product. Particular tail sequences provide binding targets for specific dyes. For example, Hoechst 33258 (Searle and Embrey, 1990<i>, Nuc. Acids Res. </i>18:3753-3762) preferentially binds A-T base pairs. A PCR primer synthesized with a long A-T rich 5′ tail provides a relatively A-T rich PCR product in comparison with genomic DNA. Using Hoechst 33358 the A-T rich PCR product has increased fluorescence relative to genomic DNA and, consequently, is useful for increasing signal strength in the presence of genomic DNA.</p>
    <p>Similarly, DAPI forms a fluorescent complex with AT rich DNA (see Kapuseinski &amp; Szer, 1979, <i>Nuc. Acids Res., </i>6(112):3519). In contrast, Actinomycin D forms a fluorescent complex with G-C rich DNA (Jain and Sobell, 1972, <i>J. Mol. Biol. </i>68:21). Thus, the present invention is suitable for monitoring the amount of two distinct target nucleic acids in one reaction vessel, by including two fluorochromes in the reaction mixture during amplification. It is only necessary that one fluorochrome has sequence specificity and that sequence is included in the tail (i.e., 5′ end) of a primer to one of the two target nucleic acids. The emission spectra for each fluorochrome is separately determined during and/or after amplification by a spectra fluorometer. Thus, the invention is particularly useful for quantitative comparisons of two different nucleic acid targets in the same sample and as such may be useful for determining the extent of, for example, an infection or disease if one target is a sequence present in all cells and the other is present in a pathogen.</p>
    <p>Similar agents with distinct binding properties permit multiplex PCR methods for detecting several targets in one sample without ever opening the reaction vessel once the amplification reaction is initiated. Fluorescent DNA binding dyes may each have different emissions and excitation spectra. In a clinical setting, different DNA binding dyes having different DNA sequence specificities are useful for indicating the presence of distinct targets.</p>
    <p>Methods for quantitating nucleic acids are described in commonly assigned, copending U.S. Ser. Nos. 254,889, filed Oct. 7, 1988 (now U.S. Pat. No. 5,389,512), and 413,623, filed Sep.28, 1989 (now U.S. Pat. No. 5,219,727). These applications are incorporated herein by reference. These applications describe PCR-based methods using an internal standard to determine either the relative amount of a target or accurately quantitate the amount of target present prior to amplification, respectively. The present invention is suitable in conjunction with the methods described in the '889 and '623 applications.</p>
    <p>In another embodiment, the present invention provides means for determining the intactness of a DNA sample. For example, forensic analysis often involve samples containing partially degraded target, such as an archival sample. The ability to monitor the generation of PCR product, demonstrated in Example VIII, allows comparison of the amplification profile of a DNA sample with respect to a small PCR product and a large PCR product, in separate reactions. If there is no degradation, monitoring the increase in double-stranded DNA shows that both reactions reach plateau at approximately the same cycle. If there is degradation, the small product would reach plateau sooner than the large fragment due to the presence of more intact target molecules.</p>
    <p>It will be apparent to those of ordinary skill in the art that the homogeneous detection assay provided herein is suitable for a wide variety of applications, including, for example, genetic screening, forensic human identification, pathogen detection, and quantitation, environmental monitoring or tagging and tracing materials with nucleic acid (see, for example, U.S. Ser. No. 355,455, filed May 22, 1989, which is incorporated herein by reference).</p>
    <p>In one embodiment, the detectable signal is fluorescence. Fluorescence is suitable for use in qualitative as well as quantitative methods. Qualitative detection can be made simply and rapidly by visual inspection of the reaction tubes by exposure to UV light This type of rapid, highly sensitive assay is desirable as a rapid screen for the presence of a pathogen or of a particular gene sequence causative of or associated with a disease state. The present invention provides means for cutting costs by a plus/minus pre-screen for the presence of any member of a group of targets. Amplifications for many different such targets would allow this if it may be expected that most samples are negative, e.g., environmental monitoring for pathogen detection systems. Multiplex PCR is a process for including a number of distinct primer pairs in one amplification reaction (see, Gibbs et al., 1989, in <i>PCR Technology </i>ed. Erlich, Stockton Press, N.Y.)). A homogeneous detection assay using multiple primer pairs to assay a wide range of potential targets is then followed by more complex typing procedures only on positive samples. This pre-screen saves the cost of performing the more complex typing procedure on negative samples and/or performing repeated tests on negative samples.</p>
    <p>The present methods for homogeneous detection of target nucleic acids are also suitable for quantitation of a particular target Whether the methods are automated or manually performed, quantitation can be accomplished by a number of means. For example, a serial dilution of the sample, in parallel with a serial dilution of a known standard, provides a series of templates, which, following PCR and signal detection, are suitable for quantitating the amount of starting material in the known sample. Similarly, because fluorescence can be determined between cycles during the course of a PCR, the exponential phase of PCR and the cycle at which the level of product reaches the plateau phase, can be readily determined. The more target DNA present at the start of PCR, the sooner the reaction reaches plateau, i.e., the point when the rate of product accumulation begins to diminish. Because the number of cycles needed to reach plateau is directly related to the amount of target present in the sample, monitoring fluorescence while PCR is in progress serves to quantitate small amounts of DNA.</p>
    <p>The invention is suitable for detecting amplified nucleic acids in the presence of double-stranded genomic DNA. Background levels of DNA as high as 0.5 μg or more will not obscure a positive assay result The target nucleic acid can be a cloned segment, a repeat sequence, such as a multi-copy gene or tandem repeat, a single copy gene, or an infectious agent present in a concentration as low as one copy per 70,000 cells. The starting template is RNA or DNA because PCR provides a net increase in double-stranded nucleic acid starting from either template nucleic acid.</p>
    <p>The target nucleic acid may be a rare sequence, such as a single copy of AIDS virus DNA in a background of human genomic DNA from more than 70,000 cells. In such an instance, procedures for increasing specificity serve to insure that amplification provides a net gain in double-stranded DNA only in response to the presence of the target sequence, and that net gain is detectable in the presence of high background of genomic DNA. U.S. Pat. No. 4,683,195 demonstrates the use of nested primers to decrease the background in the amplification of single copy genes.</p>
    <p>The procedure for nested amplification in U.S. Pat. No. 4,683,195 requires a first primer pair to amplify a target sequence and a second primer pair to amplify a subsegment of the PCR product formed from the first amplification reaction. Following the first PCR, the reaction mixture is diluted 10-fold to reduce the concentration of the first primer pair, and the second primer pair is introduced into the reaction mixture. However, because a particular advantage of the present invention lies in the elimination of steps, the additional steps of stopping an amplification reaction to dilute the sample and adding a second primer pair are not desirable.</p>
    <p>To address this issue, modified nested amplification procedures are provided. The present nested primer methods are vastly improved over prior nested primer procedures for amplifying nucleic acids. These methods provide enhanced specificity and are applicable in any PCR-based amplification scheme. However, in the present disclosure, these procedures are described used in a homogeneous assay.</p>
    <p>In one modified nested primer method, a third primer, internal to a flanking pair of PCR primers is included in a PCR reaction for amplifying a particular target segment. The third primer has an annealing/melting temperature, when it is hybridized to its complementary target strand, that is lower than that of the flanking primer. This property can be imparted to the primer by a shorter length and/or lower G-C content. For the first 15-20 PCR cycles, the temperature during the extension phase of each PCR cycle is maintained sufficiently high, i.e. approximately 65° C. to prevent the short primer from annealing specifically and initiating amplification. The flanking primers anneal sufficiently such that PCR proceeds normally at the high extension temperature. However, the primer flanking the third primer is present at a low concentration. In the method, prior to amplification plateau, when the supply of limiting primer is almost exhausted, the annealing temperature is decreased to approximately 42° C. At this temperature, the third primer “drops in” and proceeds to amplify the target for the remaining 15 or so cycles.</p>
    <p>In an alterative method for nested primer amplification, the need for a low concentration of one primer is eliminated. The flanking primer is synthesized with a G-C rich tail. Because the non-complementary primer tail sequence is incorporated into the PCR product after two cycles of amplification, the G-C tails serves to raise the temperature necessary for denaturing the PCR product because of the increased thermostability of G-C pairs versus A-T pairs. (See Myers et al., 1989, in <i>PCR Technology </i>ed. Erlich, Stockton Press, N.Y., which is incorporated herein by reference). The resulting difference in the denaturing temperature between the nested and flanking PCR product is then exploited to effectively shut down amplification from the tailed flanking primer. Once PCR product is made from the flanking primers, the denaturation temperature is lowered, i.e., from 96° C. to 86° C., so that the temperature is too low for amplification of the flanking PCR product, but is high enough to allow amplification of the nested PCR product. The annealing temperature may also be manipulated, as in the “drop-in” method, to initiate synthesis from the nested primer when desired.</p>
    <p>It would be obvious to one of ordinary skill in the art to empirically determine the appropriate denaturation and annealing temperatures and program a thermocycler accordingly. This “drop-out primer” method provides means for including high concentrations of the flanking primer to maintain PCR efficiency, and allow the amplification initiated by that primer to be terminated during the reaction as desired. This particular method for nested primer amplification is demonstrated in Examples VI and VII.</p>
    <p>The following examples serve to illustrate various aspects of the present invention and are not intended as a limitation.</p>
    <heading>EXAMPLE I</heading> <heading>PCR in the Presence of Ethidium Bromide</heading> <p>This example demonstrates the ability of PCR to proceed in the presence of ethidium bromide.</p>
    <p>Two PCRs were carried out as follows. Each 100 μl PCR contained: 50 ng human DNA, 10 mM Tris-HCl pH 8, 50 mM KCl, 4 mM MgCI<sub>2</sub>, 250 μM each dNTP, 2.5 units Taq polymerase (Perkin-Elmer Cetus Instruments, Norwalk Conn.), 20 picamole each primer GH26 (SEQ ID NO: 1) and GH27 (SEQ ID NO: 2). The human DNA was purified from a human B-cell line. DNA was prepared according to the method described by Maniatis (supra). An oil overlay was added to each reaction to prevent evaporation.</p>
    <p>The two PCRs were conducted under identical conditions except that 0.51 μM of ethidium bromide (Sigma) was included in one reaction. A thermocycler purchased from Perkin-Elmer Cetus Instruments was programmed with the following cycling parameters; denature at 96° C., hold for 1 minute, anneal at 55° C., hold for 1 minute, extend at 72° C., hold for 1 minute. This profile was repeated for 32 cycles. After amplification 5 μl of each PCR was analyzed by gel electrophoresis using a 3% NuSieve agarose gel (FMC). The gel was stained with ethidium bromide by standard methods and the amount of PCR product made in the two reactions was compared. The results demonstrated that the amount of amplified DNA produced in the presence of ethidium bromide was indistinguishable from the amount of PCR product made in the absence of the dye. Additionally, the specificity of the PCR, measured by the ability to produce DNA fragments of the expected size, was unchanged.</p>
    <heading>EXAMPLE II</heading> <heading>The Specificity of PCR is Sufficient for Fluorescence-Based, Homogeneous Detection of Target Nucleic Acids</heading> <p>The experiment described in Example I was expanded upon to determine whether target specific ethidium bromide fluorescence is readily visible using standard PCR conditions. Therefore, five 100 μl reaction mixtures were prepared containing 10 mM Tris-HCl pH8, 50 mM KCl, 2.5 mM MgCl<sub>2 </sub>1.27 μM ethidium bromide, 150 μM each dNTP, 2.5 units Taq polymerase. Primers and target DNA were included as described below. Primer pair GH15 (SEQ ID NO: 3) and GH16 (SEQ ID NO: 4) are specific for amplifying DQα and do not amplify DRβl target DNA. DQα target DNA was prepared by amplifying the human DQα gene as described in Example I. Following amplification, the DQα PCR product was diluted to provide ˜2×10<sup>7 </sup>copies (5 pg) of amplified DNA per amplification. The DQα product was used as a positive control. DRβ1 target DNA was prepared using primer pair GH46 (SEQ ID NO: 5) and GH50 (SEQ ID NO: 6) to amplify the DRβ1 gene in a PCR using human genomic DNA. The PCR product was diluted to provide ˜2×10<sup>7 </sup>copies of DRβ1 DNA and used in Reaction 4, below, as a negative control. The five reaction mixtures contained primers and target as follows:</p>
    <p>Reaction 1—No primers+DQα Target</p>
    <p>Reaction 2—Primers+DQα target</p>
    <p>Reaction 3—Primers+DQα target</p>
    <p>Reaction 4—Primers+DRβ1 target</p>
    <p>Reaction 5—Primers+no target</p>
    <p>Where primers were included 10 pmoles each of GH15 (SEQ ID NO: 3) and GH16 (SEQ ID NO: 4) were added. Reaction mixtures 1 and 2 were not subjected to amplification cycles. Reactions Nos. 3, 4, and 5 were subjected to 20 cycles of amplification. The cycling parameters were 94° C., hold for 1 minute; 45° C., hold for 1 minute; and 72° C., hold for 1 minute.</p>
    <p>The reaction tubes were then placed on a UV light box (300 nm) and photographed for two seconds and one-half second exposures. The results, shown in FIG. 1, demonstrated the increased fluorescence in Reaction No. 3 due to the amplification of the specific target DNA. Reaction Nos. 1 and 2 indicated the level of fluorescence present before any amplification occurred. Reaction Nos. 4 and 5 demonstrated that fluorescence does not increase visibly in a reaction unless, for the particular primer pair present in the reaction, an appropriate template is also present. The different photographic exposures demonstrate the relative differences in fluorescence (FIG. <b>1</b>).</p>
    <heading>EXAMPLE III</heading> <heading>The Simultaneous Ethidium Bromide Detection and Amplification of Specific DNA</heading> <heading>Sequences in the Presence of Genomic DNA</heading> <p>The ability of the present homogeneous assay method for detecting a specific target in the presence of non-target double-stranded DNA was tested. This example demonstrates detection of a specific DNA sequence in a background of genomic DNA. Twenty 100 μl PCR mixtures were prepared as follows. Each contained 10 mM Tris-HCl, pH 8; 50 mM KCl; 2 mM MgCl<sub>2</sub>; 2.9 units Taq polymerase; 180 μm each dNTP; 1.27 μM ethidium bromide; 15 picamole RH191 (SEQ ID NO: 7); and 15 picamole RH192 (SEQ ID NO: 8). The primers RH191 (SEQ ID NO: 7) and RH192 (SEQ ID NO: 8) are derived from primers y1.1 and y1.2, which are described in Kogan et al., 1987<i>, N. Engl. J. Med., </i>317:985-990, which is incorporated herein by reference. These primers are specific for a human male-specific sequence that occurs in several thousand copies per human male cell. Fifteen PCR reaction mixtures were set up as follows. Sample DNA was prepared from human blood taken from a male or female as specified for each reaction. DNA was prepared according to Maniatis supra.</p>
    <p>Reaction Nos. 1-5 contained 2 ng human male DNA</p>
    <p>Reaction Nos. 6-10 contained no DNA</p>
    <p>Reaction Nos. 11-15 contained 60 ng human female DNA</p>
    <p>Reaction Nos. 16-20 contained 60 ng human male DNA The reactions were all placed in a Perkin-Elmer Cetus Instruments thermocycler programmed to cycle at 94° C. for 1 minute; 60° C. for 1 minute, for the indicated number of cycles. As indicated below, tubes were removed from the thermocycler at various cycles to provide a PCR time course for each template. Specifically, for each set of five tubes, 0, 17, 21, 25, and 29 amplification cycles were performed. PCR product was detected by exposing the tubes to UV-light as described above.</p>
    <p>By photography and visual inspection, Tube Nos. 6-15 exhibited no increase in fluorescence. Tube Nos. 1 and 2, the 0 and 17 cycle male DNA samples, also had no increase in fluorescence by visual detection. Only the 21, 25, and 29 cycle male DNA reactions fluoresced under UV light, and the amount of fluorescence increased with increasing cycle number. The results with Tube Nos. 16-20 were similar, except that an increase in fluorescence was noted by 17 cycles. This is consistent with the presence of more copies of the target DNA sequence present in the sample prior to amplification.</p>
    <heading>EXAMPLE IV</heading> <heading>Quantitative Measurement of Target Specific Ethidium Bromide Fluorescence</heading> <p>The reactions of Example III were opened, and the contents transferred to a spectra fluorometer (SPEX Fluorolog-2, purchased from Spex, Edison, N.J.) which was used to get a quantitative fluorescence value for each reaction. The results shown in FIG. 2 not only demonstrate the target specificity of the detection method but illustrate the quantitative aspects of the invention. The effect of increased target in the sample is observable by comparing the time course of fluorescence between the 2 ng and 60 ng male templates. The more target DNA in the sample, the sooner the reaction obtains a measurable increase in fluorescence and finally reaches a plateau level of fluorescence. Exactly when the reaction fluorescence begins to increase measurably is effectively a quantitative measure of how much target was present prior to amplification.</p>
    <heading>EXAMPLE V</heading> <heading>Detection of a Single Copy Gene</heading> <p>This example demonstrates the suitability of the homogeneous assay not only for detecting a single copy gene among total human genomic DNA, but for discriminating among two alleles of that single copy gene present in the sample that differ by a single nucleotide. (Methods for allele specific detection are described in detail in European Patent Publication No. 237,362, which is incorporated herein by reference).</p>
    <p>The particular gene to be detected is the β-globin gene. A single base pair change mutates a wild-type β-globin allele into a sickle cell allele. In this example, the following primers were used: RH187 (SEQ ID NO: 9), RH188 (SEQ ID NO: 10), and RH189 (SEQ ID NO: 11). Primer pair RH187/RH188 (SEQ ID NO: 9/SEQ ID NO: 10) specifically amplified the wild-type allele. Primer pair RH187/RH189 (SEQ ID NO: 9/SEQ ID NO: 11) amplify the sickle cell allele (these primers derive from BGP2, Hβ14A , and Hβ14S described in Wu et al., 1989<i>, PNAS (USA), </i>86:2757-2760, which is incorporated herein by reference). Six PCRs were set up as follows: 10 mM Tris-HCl, pH 8.3; 50 mM KCl; 748 μM total dNTPs; 2.9 units Taq polymerase; 10 pmoles each primer, 1.5 μM MgCl<sub>2</sub>; 1.27 μM ethidium bromide, and 50 ng of human DNA template as follows: one pair of reactions contained human DNA homozygous for sickle allele (SS); one pair of reactions contained wild-type DNA (AA); and one pair contained heterozygous DNA, i.e., one wild-type and one sickle cell allele (AS).</p>
    <p>For each pair of reactions, one tube contained primers specific for the sickle βglobin allele and one tube contained primers for the wild-type sequence. The only difference between the primer sets are the 3′ nucleotides of one of the primer pairs, which match either the sickle cell or the wild-type target sequence. Primer annealing temperature during PCR is set such that amplification will occur only if this 3′ nucleotide matches the template. The cycling parameters were: 94° C. for 60 seconds, 55° C. for 60 seconds.</p>
    <p>The reaction was done in triplicate and after 30 cycles of PCR, the tubes were placed on a UV-light source and photographed (see FIG. <b>3</b>). The results were as follows:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="98pt" align="left"> </colspec> <colspec colname="1" colwidth="49pt" align="center"> </colspec> <colspec colname="2" colwidth="70pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Sickle β-globin</td>
                <td class="description-td">Wild-type</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Primers</td>
                <td class="description-td">Primers</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="77pt" align="left"> </colspec> <colspec colname="2" colwidth="49pt" align="center"> </colspec> <colspec colname="3" colwidth="70pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Homozygous AA</td>
                <td class="description-td">−</td>
                <td class="description-td">+</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Heterozygous AS</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Homozygous SS</td>
                <td class="description-td">+</td>
                <td class="description-td">−</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="3" align="left" class="description-td" colspan="4">A “+” indicates that fluorescence was readily visible under UV light, and when measured on a spectra-fluorometer the “+” tubes had approximately three times greater fluorescence than “−” reactions. The PCR's marked “−” did not change significantly in florescence as a result of the amplification reaction. </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Aliquots of each reaction were analyzed by gel electrophoresis. Each “+” reaction exhibited a specific discrete DNA fragment The “−” reactions had no such DNA fragment by gel analysis.</p>
    <heading>EXAMPLE VI</heading> <heading>Detection of a Rare Target in the Presence of a High</heading> <heading>Background of Double-Stranded DNA</heading> <p>A detection assay was designed to demonstrate the suitability of the present invention for detecting a rare target sequence in a background of DNA from approximately 70,000 human cells. A modified nested primer procedure, as briefly described in the “Detailed Description” section as a primer “drop-out” procedure, was designed to enhance PCR specificity. This assay was done as follows: into PCR reaction vessels 1 through 8 were aliquoted 50 microliters of solution, each containing 50 mM KCl; 10 mM TrisHCl, pH 8.3; 2.5 mM MgCl<sub>2</sub>; 600 μM total dNTPs; 1.25 unit of A DNA polymerase (PECI); 1.27 μM ethidium bromide; 0.5 μg of human cell-line DNA; the primer pair RH171 (SEQ ID NO: 12) and RH176 (SEQ ID NO: 13), each primer at 0.2 μM; and the nested primer RH182 (SEQ ID NO: 14) also at 0.2 μM. A drop of mineral oil was used to cover the eight solutions in order to prevent evaporation. Primer RH176 (SEQ ID NO: 13) carries a GC-rich, non-homologous (to target sequence), 5′ “tail” that raises the denaturation temperature necessary to amplify PCR product made using this primer.</p>
    <p>Reactions 1-4 were made with solutions that were at ambient room temperature and included the three primers before temperature cycling was begun. Reactions 5-8 were made with the addition of the three primers postponed until these reactions were equilibrated to a temperature of 72° C. before beginning thermocycling. For this reason, reactions 5-8 are referred to as being given a “hot-start.” Reactions 2-4 and 6-8 also contained a target, positive control DNA (purchased from PECI) containing HIV sequences to which RH171 (SEQ ID NO: 12), RH176 (SEQ ID NO: 13), and the 3′ portion of RH182 (SEQ ID NO: 14) were homologous. This DNA was diluted such that each reaction containing it had,,on average, four copies of the HIV sequence. Because this average number of copies is small, the actual number of copies in a given reaction can vary considerably. Since no HIV DNA target was added to reactions 1 and 5, these reactions served as negative controls.</p>
    <p>All eight reactions were subjected to thermocycling as follows: denature at 96° C., hold for 1 minute, anneal at 64° C., hold for 1 minute. This profile was repeated for 29 cycles, during which the flanking primer pair, RH171 (SEQ ID NO: 12) and RH176 (SEQ ID NO: 13), efficiently annealed and were used in amplification, while the nested primer RH182 (SEQ ID NO: 14), which does not efficiently anneal at 64° C., was not used in efficient amplification. This was followed by denaturation at 96° C., hold for 1 minute, annealing at 52° C., hold for 1 minute. This profile was repeated for 2 cycles, during which all three primers efficiently annealed and were extended in amplification such that products were made using either RH171 (SEQ ID NO: 12) and RH176 (SEQ ID NO: 13), or RH171 (SEQ ID NO: 12) and RH182 (SEQ ID NO: 14). Because the use of a third, nested primer, increases product specificity, products made using RH171 (SEQ ID NO: 12) and RH182 (SEQ ID NO: 14) were more likely to be HIV specific. These cycles were followed by denaturation at 86° C., hold for 1 minute, anneal at 52° C., hold for 1 minute. This profile was repeated 18 times, during which, products that included the GC-rich primer RH176 (SEQ ID NO: 13), both HIV specific and non-specific, did not efficiently denature at 86° C. and, therefore, did not amplify efficiently, while the amplified HIV sequences made using the nested primer RH182 (SEQ ID NO: 14) and RH171 (SEQ ID NO: 12) did efficiently denature and amplify.</p>
    <p>All eight reactions were analyzed, when completed, by gel electrophoresis. Reactions 2-4 and 5-8 were shown to contain a product of the expected size (approximately 200 bp) as the predominant band on the gel. Reactions 1 and 5, the negative controls, contained no such product. However, reactions 2-4, which were not given a “hot-start,” could be seen to contain DNA fragments of other than the expected size. These other DNA fragments were also visible in reaction 1, indicating that they are not derived from HIV sequences. These other DNA fragments were not visible in reactions 5-8, indicating that use of the “hot-start” had enhanced the specificity of these reactions.</p>
    <p>Eight additional reactions were performed as described above, except that all were given a “hot-start” as described above. The positive control DNA was diluted in these eight reactions, numbered 1 through 8, such that on average, each contains half an HIV target molecule. Since a molecule cannot be divided, this means that some reactions should contain a target molecule and some should not. If this experiment were repeated many times, the fraction of reactions that do contain a target will vary considerably, but should be on average about half. Those that do contain a target molecule are most likely to contain a single target molecule. Upon completion of the reactions, all eight were analyzed by gel electrophoresis. The result was that two of the eight reactions, numbers 1 and 8, displayed a DNA fragment of the expected size (approximately 200 bp) as the predominant band on the gel, with no other bands that migrated into the gel visible except a band corresponding to the primers. Reactions 2-6 displayed no such bands nor any other DNA fragment bands.</p>
    <heading>EXAMPLE VII</heading> <heading>A Method for Quantitative Detection of PCR Product</heading> <p>A spectra-fluorometer Spex fluorolog-2 (Spex, Edison, N.J.) was used to quantitate the net increase in fluorescence generated in response to what is expected to be a single HIV target in the presence of genomic DNA. The spectra-fluorometer was used according to manufacture's specifications as described in Example VIII. The PCR reactions described in Example VI, in which positive control DNA was diluted among eight reactions to contain half an HIV target molecule per reaction, were analyzed for their fluorescence. Twenty μl of each completed reaction was added to 100 μl of 10 mM Tris HCl, pH 8, 0.1 mM EDTA, 1.27 μM ethidium bromide. The fluorescence of these solutions was measured at 570 nm. FIG. 4A graphically demonstrates the significant increase in fluorescence of the two positive samples compared to the two negative samples. In the figure, dotted lines mark the fluorescence values two standard deviations away from the average of the negative samples. The PCR-positive samples are both above this line.</p>
    <p>The results are also shown in FIG. 4B using a background substraction method. The fluorescence up to the second standard deviation below the mean was subtracted from the detected values. Background subtraction is preferably performed by measuring fluorescence in each sample at the beginning of PCR cycling (preferably after the initial denaturation, which reduces the proportion of double-stranded genomic DNA) and subtracting that value from the fluorescence in the sample at the end of PCR cycling.</p>
    <heading>EXAMPLE VIII</heading> <heading>A Method for Automated/On-Line Detection of PCR Product During Amplification</heading> <p>The following experiment demonstrates the suitability of the present method for monitoring a PCR reaction and detecting the net increase in double-stranded DNA due to amplification. The apparatus allows on-line detection of PCR product.</p>
    <p>The apparatus was set up as follows: a Spex-Fluorolog-2 fluorometer with a fiber optic accessory (Spex Catalog No. 1950) was set to emit excitation light at 500 nm with a bandwidth of ˜3.4 nm. A GG 435 nm cut off filter used to exclude second order light (Purchased from Melles Grist Inc.). The emission light was detected at 570 nm with a bandwidth of ˜13.6 nm. A OG530 filter (530 nm cut off) was used to remove excitation light.</p>
    <p>Two PCR reactions were set up as described in Example III using primers RH191 (SEQ ID NO: 7) and RH192 (SEQ ID NO: 8). One reaction tube contained 60 ng human male DNA, the other contained no target DNA. The reactions were set up in 0.5 ml polypropylene tubes; however, the top of the tubes was cut away for attaching the fiber optic cable. The fiber optic was glued to the top of the reaction tube with epoxy. Because this apparatus had one fiber optic, only one PCR was run at a time. The emission light was collected through the oil overlay in the tube. A black “shroud” was built around the tube and the reaction was placed in the thermocycler. The thermocycler was programmed to cycle between 94° C. and 50° C. for 1 minute each, for 30 cycles, followed by continuous incubation at 25° C. The fluorometer and thermocycler were started simultaneously. The parameters of the fluorometer were: time based scan with 5 second integration time; the emission signal was ratioed to that of the excitation light to control for changes in source intensity.</p>
    <p>FIG. 5A shows the results of the PCR reaction containing no DNA. FIG. 5A shows that the thermocycler started at 25° C., the fluorescence dropped as the temperature increased to 94° C. and fluorescence increased again when the temperature decreased to 50° C. This pattern was repeated for the remaining cycles until the thermocycler again reached 25° C. and the fluorescence returned to the approximate starting value.</p>
    <p>FIG. 5B demonstrates the fluorescence profile of a PCR reaction containing: the appropriate target DNA. The fluorescence intensity at 50° C. shows a cycle dependent increase reflecting an increase in the amount of double-stranded DNA. When the thermocycler returned to 25° C., after the 30 cycles were completed, the fluorescence increased to a final value greater than three times the initial fluorescence value at 25° C.</p>
    <p>Following amplification, an aliquot of each reaction mixture was analyzed by agarose gel electrophoresis. The gel analysis demonstrated that no PCR product was visible in the lane containing a sample from the negative control. The electrophoresed sample from the positive control PCR showed a clear and unique band at ˜150 base pairs. The predicted size of the PCR product was 154 bp.</p>
    <p>Thus, the on-line method provided a rapid analysis as to the presence or absence of target DNA without the need for probes or further processing steps. In addition, the continuous detection of fluorescence throughout the amplification provides an amplification profile that reflects the amount of target present at start. If the target DNA is, for example, a human repeat sequence present in millions of copies per cell (e.g., “Alu” sequences; Nelson and Caskey in PCR Technology ed. Erlich [1989], Stockton Press, N.Y.), this method of quantitation could be used to quickly and simply to measure sub-cellular amounts of DNA, which is at present difficult to do without using radioisotopes.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Primer Sequences for Examples I-VIII</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="154pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Cetus</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Nomen-</td>
                <td class="description-td">SEQ</td>
              </tr> <tr class="description-tr"> <td class="description-td">clature</td>
                <td class="description-td">ID NO</td>
                <td class="description-td">Primer Sequence</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="154pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">GH26</td>
                <td class="description-td">1</td>
                <td class="description-td">5′ GTGCTGCAGG TGTAAACTTG TACCAG 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">GH27</td>
                <td class="description-td">2</td>
                <td class="description-td">5′ CACGGATCCG GTAGCAGCGG TAGAGTTG 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">GH15</td>
                <td class="description-td">3</td>
                <td class="description-td">5′ GTGTAAACTT GTACCAG 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">GH16</td>
                <td class="description-td">4</td>
                <td class="description-td">5′ GGTAGCAGCG GTAGAG 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">GH46</td>
                <td class="description-td">5</td>
                <td class="description-td">5′ CCGGATCCTT CGTGTCCCCA CAGCACG 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">GH50</td>
                <td class="description-td">6</td>
                <td class="description-td">5′ CTCCCCAACC CCGTAGTTGT GTCTGCA 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">RH191</td>
                <td class="description-td">7</td>
                <td class="description-td">5′ TCCACTTTAT TCCAGGCCTG T 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">RH192</td>
                <td class="description-td">8</td>
                <td class="description-td">5′ TTGAATGGAA TGGGAACGAA TGG 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">RH187</td>
                <td class="description-td">9</td>
                <td class="description-td">5′ AATAGACCAA TAGGCAGAG 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">RH188</td>
                <td class="description-td">10</td>
                <td class="description-td">5′ CACCTGACTC CTGA 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">RH189</td>
                <td class="description-td">11</td>
                <td class="description-td">5′ CACCTGACTC CTGT 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">RH171</td>
                <td class="description-td">12</td>
                <td class="description-td">5′ CCAGGCCAGA TGAGAGAACC AAGGGG 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">RH176</td>
                <td class="description-td">13</td>
                <td class="description-td">5′ GCGGGCAGGG CGGCGGGGGC GGGGCCGAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">CGGTCTACAT AGTCTCTAAA GGG 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">RH182</td>
                <td class="description-td">14</td>
                <td class="description-td">5′ GGTCCCTGTC TTATGTC 3′</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Although the foregoing invention has been described in some detail for the purpose of illustration, it will be obvious that changes and modifications may be practical within the scope of the appended claims by those of ordinary skill in the art.</p>
    <p>
      <sequence-cwu id="SEQLST-1"> <sequence-list-old-rules> <s-1> <s-1-iii>14</s-1-iii> </s-1> <s-2 seq-no="1"> <s-2-i> <s-2-i-a>26 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>1
GTGCTGCAGG TGTAAACTTG TACCAG                                          26
</s-2-xi> </s-2> <s-2 seq-no="2"> <s-2-i> <s-2-i-a>28 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>2
CACGGATCCG GTAGCAGCGG TAGAGTTG                                        28
</s-2-xi> </s-2> <s-2 seq-no="3"> <s-2-i> <s-2-i-a>17 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>3
GTGTAAACTT GTACCAG                                                    17
</s-2-xi> </s-2> <s-2 seq-no="4"> <s-2-i> <s-2-i-a>16 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>4
GGTAGCAGCG GTAGAG                                                     16
</s-2-xi> </s-2> <s-2 seq-no="5"> <s-2-i> <s-2-i-a>27 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>5
CCGGATCCTT CGTGTCCCCA CAGCACG                                         27
</s-2-xi> </s-2> <s-2 seq-no="6"> <s-2-i> <s-2-i-a>27 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>6
CTCCCCAACC CCGTAGTTGT GTCTGCA                                         27
</s-2-xi> </s-2> <s-2 seq-no="7"> <s-2-i> <s-2-i-a>21 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>7
TCCACTTTAT TCCAGGCCTG T                                               21
</s-2-xi> </s-2> <s-2 seq-no="8"> <s-2-i> <s-2-i-a>23 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>8
TTGAATGGAA TGGGAACGAA TGG                                             23
</s-2-xi> </s-2> <s-2 seq-no="9"> <s-2-i> <s-2-i-a>19 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>9
AATAGACCAA TAGGCAGAG                                                  19
</s-2-xi> </s-2> <s-2 seq-no="10"> <s-2-i> <s-2-i-a>14 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>10
CACCTGACTC CTGA                                                       14
</s-2-xi> </s-2> <s-2 seq-no="11"> <s-2-i> <s-2-i-a>14 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>11
CACCTGACTC CTGT                                                       14
</s-2-xi> </s-2> <s-2 seq-no="12"> <s-2-i> <s-2-i-a>26 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>12
CCAGGCCAGA TGAGAGAACC AAGGGG                                          26
</s-2-xi> </s-2> <s-2 seq-no="13"> <s-2-i> <s-2-i-a>53 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>13
GCGGGCAGGG CGGCGGGGGC GGGGCCGAAC CGGTCTACAT AGTCTCTAAA GGG            53
</s-2-xi> </s-2> <s-2 seq-no="14"> <s-2-i> <s-2-i-a>17 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-xi>14
GGTCCCTGTC TTATGTC                                                    17
</s-2-xi> </s-2> </sequence-list-old-rules> </sequence-cwu> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3856470">US3856470</a></td><td class="patent-data-table-td patent-date-value">Jan 10, 1973</td><td class="patent-data-table-td patent-date-value">Dec 24, 1974</td><td class="patent-data-table-td ">Baxter Laboratories Inc</td><td class="patent-data-table-td ">Rotor apparatus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3992631">US3992631</a></td><td class="patent-data-table-td patent-date-value">Feb 27, 1975</td><td class="patent-data-table-td patent-date-value">Nov 16, 1976</td><td class="patent-data-table-td ">International Diagnostic Technology, Inc.</td><td class="patent-data-table-td ">Fluorometric system, method and test article</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3999868">US3999868</a></td><td class="patent-data-table-td patent-date-value">Nov 17, 1975</td><td class="patent-data-table-td patent-date-value">Dec 28, 1976</td><td class="patent-data-table-td ">Manuel Sanz</td><td class="patent-data-table-td ">Unit for use in optical analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4058370">US4058370</a></td><td class="patent-data-table-td patent-date-value">Jan 27, 1975</td><td class="patent-data-table-td patent-date-value">Nov 15, 1977</td><td class="patent-data-table-td ">Osmo Antero Suovaniemi</td><td class="patent-data-table-td ">Apparatus for accurate pipetting of small liquid volumes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4119521">US4119521</a></td><td class="patent-data-table-td patent-date-value">Apr 25, 1977</td><td class="patent-data-table-td patent-date-value">Oct 10, 1978</td><td class="patent-data-table-td ">Stephen Turner</td><td class="patent-data-table-td ">Dna intercalation agent</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4240751">US4240751</a></td><td class="patent-data-table-td patent-date-value">Nov 9, 1978</td><td class="patent-data-table-td patent-date-value">Dec 23, 1980</td><td class="patent-data-table-td ">Akzona Incorporated</td><td class="patent-data-table-td ">Method and apparatus for specific binding substances</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4257774">US4257774</a></td><td class="patent-data-table-td patent-date-value">Jul 16, 1979</td><td class="patent-data-table-td patent-date-value">Mar 24, 1981</td><td class="patent-data-table-td ">Meloy Laboratories, Inc.</td><td class="patent-data-table-td ">Ethidium bromide, actinomycin d, proflavin</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4563417">US4563417</a></td><td class="patent-data-table-td patent-date-value">Dec 24, 1984</td><td class="patent-data-table-td patent-date-value">Jan 7, 1986</td><td class="patent-data-table-td ">Miles Laboratories, Inc.</td><td class="patent-data-table-td ">Medical diagnosis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4582789">US4582789</a></td><td class="patent-data-table-td patent-date-value">Dec 18, 1984</td><td class="patent-data-table-td patent-date-value">Apr 15, 1986</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">Containing alkylating intercalation moiety</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4683195">US4683195</a></td><td class="patent-data-table-td patent-date-value">Feb 7, 1986</td><td class="patent-data-table-td patent-date-value">Jul 28, 1987</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">One of the first Polymerase Chain Reaction (PCR) patents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4683202">US4683202</a></td><td class="patent-data-table-td patent-date-value">Oct 25, 1985</td><td class="patent-data-table-td patent-date-value">Nov 27, 1990</td><td class="patent-data-table-td ">Cetus Corp</td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4856073">US4856073</a></td><td class="patent-data-table-td patent-date-value">Nov 10, 1987</td><td class="patent-data-table-td patent-date-value">Aug 8, 1989</td><td class="patent-data-table-td ">Sherwood Medical Company</td><td class="patent-data-table-td ">Automated microbiological testing apparatus and method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4889818">US4889818</a></td><td class="patent-data-table-td patent-date-value">Jun 17, 1987</td><td class="patent-data-table-td patent-date-value">Dec 26, 1989</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">Dna polymerase; isolated from thermus aquaticus; use in polymerase chain reaction (pcr)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4902624">US4902624</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 21, 1988</td><td class="patent-data-table-td patent-date-value">Feb 20, 1990</td><td class="patent-data-table-td ">Eastman Kodak Company</td><td class="patent-data-table-td ">Rapid thermal processing of dna strands through a multizonal cycle at controlled temperature</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5038852">US5038852</a></td><td class="patent-data-table-td patent-date-value">Mar 14, 1990</td><td class="patent-data-table-td patent-date-value">Aug 13, 1991</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">Apparatus and method for performing automated amplification of nucleic acid sequences and assays using heating and cooling steps</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5049490">US5049490</a></td><td class="patent-data-table-td patent-date-value">Feb 20, 1990</td><td class="patent-data-table-td patent-date-value">Sep 17, 1991</td><td class="patent-data-table-td ">Eastman Kodak Co.</td><td class="patent-data-table-td ">Quantitative determination of a DNA polymerase and a test kit useful in same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5096807">US5096807</a></td><td class="patent-data-table-td patent-date-value">Dec 1, 1989</td><td class="patent-data-table-td patent-date-value">Mar 17, 1992</td><td class="patent-data-table-td ">Murex Corporation</td><td class="patent-data-table-td ">Imaging immunoassay detection system with background compensation and its use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5184020">US5184020</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 26, 1989</td><td class="patent-data-table-td patent-date-value">Feb 2, 1993</td><td class="patent-data-table-td ">Hearst David P</td><td class="patent-data-table-td ">For treating photoreactive compounds</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5210015">US5210015</a></td><td class="patent-data-table-td patent-date-value">Aug 6, 1990</td><td class="patent-data-table-td patent-date-value">May 11, 1993</td><td class="patent-data-table-td ">Hoffman-La Roche Inc.</td><td class="patent-data-table-td ">Polymerase chain reaction technique</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5219727">US5219727</a></td><td class="patent-data-table-td patent-date-value">Sep 28, 1989</td><td class="patent-data-table-td patent-date-value">Jun 15, 1993</td><td class="patent-data-table-td ">Hoffmann-Laroche Inc.</td><td class="patent-data-table-td ">Quantitation of nucleic acids using the polymerase chain reaction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5229297">US5229297</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 1992</td><td class="patent-data-table-td patent-date-value">Jul 20, 1993</td><td class="patent-data-table-td ">Eastman Kodak Company</td><td class="patent-data-table-td ">Polymerase chain reaction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5242797">US5242797</a></td><td class="patent-data-table-td patent-date-value">Jan 2, 1992</td><td class="patent-data-table-td patent-date-value">Sep 7, 1993</td><td class="patent-data-table-td ">Myron J. Block</td><td class="patent-data-table-td ">Nucleic acid assay method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5269937">US5269937</a></td><td class="patent-data-table-td patent-date-value">Oct 23, 1990</td><td class="patent-data-table-td patent-date-value">Dec 14, 1993</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">HPLC light scattering detector for biopolymers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5322770">US5322770</a></td><td class="patent-data-table-td patent-date-value">Dec 22, 1989</td><td class="patent-data-table-td patent-date-value">Jun 21, 1994</td><td class="patent-data-table-td ">Hoffman-Laroche Inc.</td><td class="patent-data-table-td ">Providing a cDNA by heating a primed RNA with the polymerase in the presence of the four deoxyribonucleotides and a buffer at a temperature to effect hybridizing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5355215">US5355215</a></td><td class="patent-data-table-td patent-date-value">Sep 30, 1992</td><td class="patent-data-table-td patent-date-value">Oct 11, 1994</td><td class="patent-data-table-td ">Environmental Research Institute Of Michigan</td><td class="patent-data-table-td ">Method and apparatus for quantitative fluorescence measurements</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5374553">US5374553</a></td><td class="patent-data-table-td patent-date-value">Aug 13, 1990</td><td class="patent-data-table-td patent-date-value">Dec 20, 1994</td><td class="patent-data-table-td ">Hoffmann-La Roche Inc.</td><td class="patent-data-table-td ">DNA encoding a thermostable nucleic acid polymerase enzyme from thermotoga maritima</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5389512">US5389512</a></td><td class="patent-data-table-td patent-date-value">May 20, 1994</td><td class="patent-data-table-td patent-date-value">Feb 14, 1995</td><td class="patent-data-table-td ">Hoffman-La Roche Inc.</td><td class="patent-data-table-td ">Compared to amount of cellular nucleic acid sequence; primer pairs; simultaneous amplification; useful in detection of infectious agents present in biological samples</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5411876">US5411876</a></td><td class="patent-data-table-td patent-date-value">May 27, 1992</td><td class="patent-data-table-td patent-date-value">May 2, 1995</td><td class="patent-data-table-td ">Hoffmann-La Roche Inc.</td><td class="patent-data-table-td ">Use of grease or wax in the polymerase chain reaction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5415839">US5415839</a></td><td class="patent-data-table-td patent-date-value">Oct 21, 1993</td><td class="patent-data-table-td patent-date-value">May 16, 1995</td><td class="patent-data-table-td ">Abbott Laboratories</td><td class="patent-data-table-td ">Apparatus and method for amplifying and detecting target nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5455170">US5455170</a></td><td class="patent-data-table-td patent-date-value">Aug 27, 1993</td><td class="patent-data-table-td patent-date-value">Oct 3, 1995</td><td class="patent-data-table-td ">Hoffmann-La Roche Inc.</td><td class="patent-data-table-td ">Purified DNA sequence modified to encode thermostable DNA polymerase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5552580">US5552580</a></td><td class="patent-data-table-td patent-date-value">Nov 20, 1991</td><td class="patent-data-table-td patent-date-value">Sep 3, 1996</td><td class="patent-data-table-td ">Beckman Instruments, Inc.</td><td class="patent-data-table-td ">Heated cover device</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5618711">US5618711</a></td><td class="patent-data-table-td patent-date-value">Feb 6, 1995</td><td class="patent-data-table-td patent-date-value">Apr 8, 1997</td><td class="patent-data-table-td ">Hoffmann-La Roche Inc.</td><td class="patent-data-table-td ">Purified recombinant dna polymerase; thermostable, useful in polymerase chain reaction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5795747">US5795747</a></td><td class="patent-data-table-td patent-date-value">Jun 26, 1995</td><td class="patent-data-table-td patent-date-value">Aug 18, 1998</td><td class="patent-data-table-td ">Evotec Biosystems Gmbh</td><td class="patent-data-table-td ">Process for manufacturing new biopolymers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5871908">US5871908</a></td><td class="patent-data-table-td patent-date-value">Feb 4, 1993</td><td class="patent-data-table-td patent-date-value">Feb 16, 1999</td><td class="patent-data-table-td ">Evotec Biosystems Gmbh</td><td class="patent-data-table-td ">Process for the determination of in vitro amplified nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5994056">US5994056</a></td><td class="patent-data-table-td patent-date-value">May 2, 1991</td><td class="patent-data-table-td patent-date-value">Nov 30, 1999</td><td class="patent-data-table-td ">Roche Molecular Systems, Inc.</td><td class="patent-data-table-td ">Homogeneous methods for nucleic acid amplification and detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6171785">US6171785</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 1995</td><td class="patent-data-table-td patent-date-value">Jan 9, 2001</td><td class="patent-data-table-td ">Roche Molecular Systems, Inc.</td><td class="patent-data-table-td ">Methods and devices for hemogeneous nucleic acid amplification and detector</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/DE2651536A1?cl=en">DE2651536A1</a></td><td class="patent-data-table-td patent-date-value">Nov 11, 1976</td><td class="patent-data-table-td patent-date-value">May 18, 1977</td><td class="patent-data-table-td ">Ricoh Kk</td><td class="patent-data-table-td ">Blattzufuehreinrichtung mit kassetten</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0070685A2?cl=en">EP0070685A2</a></td><td class="patent-data-table-td patent-date-value">Jul 14, 1982</td><td class="patent-data-table-td patent-date-value">Jan 26, 1983</td><td class="patent-data-table-td ">Amoco Corporation</td><td class="patent-data-table-td ">Homogeneous nucleic acid hybridization diagnostics by non-radiative energy transfer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0169787A1?cl=en">EP0169787A1</a></td><td class="patent-data-table-td patent-date-value">Jul 22, 1985</td><td class="patent-data-table-td patent-date-value">Jan 29, 1986</td><td class="patent-data-table-td ">Centre National De La Recherche Scientifique (Cnrs)</td><td class="patent-data-table-td ">Use of oligonucleotides bonded to an intercalating agent as drugs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0200362B1?cl=en">EP0200362B1</a></td><td class="patent-data-table-td patent-date-value">Mar 27, 1986</td><td class="patent-data-table-td patent-date-value">Jan 20, 1993</td><td class="patent-data-table-td ">F. Hoffmann-La Roche Ag</td><td class="patent-data-table-td ">Process for amplifying, detecting, and/or cloning nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0236069A2?cl=en">EP0236069A2</a></td><td class="patent-data-table-td patent-date-value">Feb 25, 1987</td><td class="patent-data-table-td patent-date-value">Sep 9, 1987</td><td class="patent-data-table-td ">The Perkin-Elmer Corporation</td><td class="patent-data-table-td ">Apparatus and method for performing automated amplification of nucleic acid sequences and assays using heating and cooling steps</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0487218A1?cl=en">EP0487218A1</a></td><td class="patent-data-table-td patent-date-value">Oct 31, 1991</td><td class="patent-data-table-td patent-date-value">May 27, 1992</td><td class="patent-data-table-td ">Tosoh Corporation</td><td class="patent-data-table-td ">Method for detecting or quantifying target nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0488769B1?cl=en">EP0488769B1</a></td><td class="patent-data-table-td patent-date-value">Nov 29, 1991</td><td class="patent-data-table-td patent-date-value">Apr 29, 1998</td><td class="patent-data-table-td ">The Perkin-Elmer Corporation</td><td class="patent-data-table-td ">Two-piece plastic holder for capped sample tubes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0512334B1?cl=en">EP0512334B1</a></td><td class="patent-data-table-td patent-date-value">Apr 24, 1992</td><td class="patent-data-table-td patent-date-value">Sep 8, 1999</td><td class="patent-data-table-td ">F. Hoffmann-La Roche Ag</td><td class="patent-data-table-td ">Methods for detecting a target nucleic acid in a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0516274A2?cl=en">EP0516274A2</a></td><td class="patent-data-table-td patent-date-value">Apr 16, 1992</td><td class="patent-data-table-td patent-date-value">Dec 2, 1992</td><td class="patent-data-table-td ">Kowa Co. Ltd.</td><td class="patent-data-table-td ">Multichannel optical measuring system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0524808A2?cl=en">EP0524808A2</a></td><td class="patent-data-table-td patent-date-value">Jul 22, 1992</td><td class="patent-data-table-td patent-date-value">Jan 27, 1993</td><td class="patent-data-table-td ">F. Hoffmann-La Roche Ag</td><td class="patent-data-table-td ">In situ PCR preparations, their use, and corresponding thermal cycler equipment</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0580362B1?cl=en">EP0580362B1</a></td><td class="patent-data-table-td patent-date-value">Jul 15, 1993</td><td class="patent-data-table-td patent-date-value">Oct 9, 2002</td><td class="patent-data-table-td ">Tosoh Corporation</td><td class="patent-data-table-td ">Fluorescence detecting apparatus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0640828B1?cl=en">EP0640828B1</a></td><td class="patent-data-table-td patent-date-value">Aug 16, 1994</td><td class="patent-data-table-td patent-date-value">May 10, 2000</td><td class="patent-data-table-td ">F. Hoffmann-La Roche AG</td><td class="patent-data-table-td ">Monitoring multiple reactions simultaneously and analyzing same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DFR%26NR%3D2250991A1%26KC%3DA1%26FT%3DD&amp;usg=AFQjCNF0Of9sX6Da6VknzWvI6ltFUtN2hg">FR2250991A1</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DGB%26NR%3D1486210A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNGVg1bYJBNStpXVw0uMwcs5ViU3yg">GB1486210A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DJP%26NR%3DH0427399A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNEK7fKA-kS7s3_Dyh6JrMMR90DJlQ">JPH0427399A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DJP%26NR%3DH0484751A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNFVnuENWhVGmmSmS9qB0dKJGImW3Q">JPH0484751A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DJP%26NR%3DH0484752A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNHRkKPkZ8pgBeC3T5i8hetTBosolA">JPH0484752A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DJP%26NR%3DH03122552A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNFopxIb6vIAwjWSK9RUBN-WBbiN6A">JPH03122552A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DJP%26NR%3DH03259099A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNGCFRnTOCKib3blDWXT1xlPG-o49w">JPH03259099A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DJP%26NR%3DS61215948A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNEGHFICV2lQrhMfnMg2EH6UXec-lw">JPS61215948A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DJP%26NR%3DS62105031A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNHStliuXTivrKD789cTuxJJ2tT8bw">JPS62105031A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DSU%26NR%3D1231077A1%26KC%3DA1%26FT%3DD&amp;usg=AFQjCNGhwN_c3MeXcTjjCHP86n3rnWq8kw">SU1231077A1</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1990015881A1?cl=en">WO1990015881A1</a></td><td class="patent-data-table-td patent-date-value">Jun 12, 1990</td><td class="patent-data-table-td patent-date-value">Dec 27, 1990</td><td class="patent-data-table-td ">Cis Bio Int</td><td class="patent-data-table-td ">Method for detecting specific nucleic acid sequences and applications of same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1992005278A2?cl=en">WO1992005278A2</a></td><td class="patent-data-table-td patent-date-value">Sep 17, 1991</td><td class="patent-data-table-td patent-date-value">Apr 2, 1992</td><td class="patent-data-table-td ">Fidia Spa</td><td class="patent-data-table-td ">Method for the determination of the differential expression of mrna for the amyloid precursor protein (app)</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Advertisement for Perkin Elmer Cetus GeneAmp(R)PCR System 9600, 1991, BioTechniques 10(2).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Advertisement for Perkin Elmer Cetus GeneAmp®PCR System 9600, 1991, BioTechniques 10(2).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Annex 2 from cite 46, above.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Appeal Brief submitted by Bio-Rad to the Japanese Patent Office on Oct. 30, 2003 seeking invalidation of counterpart Japanese Patent No. 3136129 (English translation only).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Applera Answer dated Mar. 29, 2004 filed in Applera v. Bio-rad, Düsseldorf Regional Court Case No. 4a O 44/03 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Applera Brief (adding another product) dated Dec. 4, 2003 filed in Applera v. Bio-Rad, Dusseldorf Regional Court Case No. 4a O 44/03 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Applera Complaint dated Feb. 7, 2003 filed in Applera v. Bio-Rad, Dusseldorf Regional Court Case No. 4a O 44/03 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Applera Corporation's Claim Construction Brief for U.S. Patent No. 5,552,580 (with copy of Expert Report of Dr. Joseph Jaklevic).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Applera Corporation's Corrected Claim Construction Brief for U.S. Pat. No. 5,552,580 (with copy of Notice of Filing of same).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Applera Extension of Claim (Patent in Suit III) dated Jun. 20, 2003 filed in Applera v. MJ Research, Dusseldorf Regional Court Case No. 4a O 262/02 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Applera First Statement dated Jan. 14, 2004 filed with the Tokyo District Court in PE Corp. (NY) vs. Bio-Rad, Tokyo District Court No. H-15 (mo) 15045 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Applera Reply Brief dated Feb. 2, 2004 filed Applera v. Bio-Rad, Dusseldorf Regional Court Case No. 4a O 44/03 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Applera Response dated February 16, 2004 filed in Applera v. MJ Research, Dusseldorf Regional Court Case No. 4a O 262/02 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Applera Response to Oppositions and Intervention against EP 0 872 562 filed in the European Patent Office Feb. 2, 2004.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Applera Summary of Arguments dated Jan. 7, 2004 filed with the Tokyo District Court in PE Corp. (NY) vs. Bio-Rad, Tokyo District Court No. H-15 (mo) 15045 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bauer et al., Traveling waves of in vitro evolving RNA, Proc. Natl. Acad. Sci. 86:7937-7941 (1989).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bio-Rad Brief dated Oct. 30, 2003 filed with the Japanese Patent Office in Invalidation Appeal No. 2003-35449 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bio-Rad Protest Brief dated Nov. 19, 2003 filed with the Tokyo District Court in PE Corp. (NY) vs. Bio-Rad, Tokyo District Court No. H-15 (mo) 15045 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bio-Rad Reply Brief dated Mar. 22, 2004 filed in Applera v. Bio-Rad, Düsseldorf Regional Court Case No. 4a O 44/03 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bio-Rad Reply Brief dated Nov. 17, 2003 filed in Applera v. Bio-Rad, Dusseldorf Regional Court Case No. 4a O 44/03 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bio-Rad Second Statement dated Feb. 20, 2004 filed with the Tokyo District Court in PE Corp. (NY) vs. Bio-Rad, Tokyo District Court No. H-15 (mo) 15045 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Byrnes, J.J., et al., Biochemistry, 12(22):4378-4384 (1973).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cardullo et al., 1988, Proc. Natl. Acad. Sci. USA 85:8790-8794.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chehab et al., Detection of specific DNA sequences by fluorescence amplification: A color complementation assay, PNAS 86:9178-9182 (1989).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Decision by Japanese Patent Office dated Dec. 25, 2003 in Invalidation Appeal No. 2002-35399 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Decision of Preliminary Injunction from Japanese Litigation H14 (Yo) No. 22065 (English Translation only).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Declarations of Drs. Manfred Eigen , Ruthild Winkler-Oswatitsch, Björn Lindemann, Andreas Schwienhorst and Rolf Günther from the file history of EP 0 583 265.</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Derwent Abstract for Japanese Patent Application JP63119672-A, May 24, 1988 and Patent Abstracts of Japan for Japanese Patent Application JP63119672-A, published May 24, 1988.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Derwent Abstract for SU1702312-A1, 1986.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dissertation Thesis of Hajo Otten, "<a href='http://scholar.google.com/scholar?q="Ein+Beitrag+Zur+Durchf%C3%BChrung+von+Kontrollierten+Evolutionsexperimenten+mit+Biologischen+Makromolek%C3%BClen%2C%22+pp.+1-17+%28publication+date+unknown%29%2C+English+language+translation+of+the+text+bridging+pp.+16-17%2C+beginning+with+the+heading+%222.5+Machinenexperimente"'>Ein Beitrag Zur Durchführung von Kontrollierten Evolutionsexperimenten mit Biologischen Makromolekülen," pp. 1-17 (publication date unknown), English language translation of the text bridging pp. 16-17, beginning with the heading "2.5 Machinenexperimente</a>".</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Eigen et al., "<a href='http://scholar.google.com/scholar?q="Report+on+Evolution+Research%2C"'>Report on Evolution Research,</a>" Department of Biochemical Kinetics, Max Planck Institut Fur Biophysikalische Chemie, p. i, 1, 2, 53-56, 65.</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Eigen, 1987, "<a href='http://scholar.google.com/scholar?q="New+Concepts+for+Dealing+with+the+Evolution+of+Nucleic+Acids%2C"'>New Concepts for Dealing with the Evolution of Nucleic Acids,</a>" Cold Spring Harbor Symposium on Quantitative Biology, vol. LII, pp. 307-319.</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Eigen, allegedly Apr. 18-20, 1991, "<a href='http://scholar.google.com/scholar?q="Report+on+Evolution+Research%2C"'>Report on Evolution Research,</a>" Department of Biochemical Kinetics, Max Plack Institut für Biophysikalische Chemie, cover page, title page and pp. i-iii and 48-56.</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Eigen, M. et al "<a href='http://scholar.google.com/scholar?q="Sorting+single+molecules%3A+Application+to+diagnostics+and+evolutionary+biotechnology"'>Sorting single molecules: Application to diagnostics and evolutionary biotechnology</a>" Proc. Natl. Acad. Sci., vol 91, pp. 5740-5747, 1994.*</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Eppendorf Product Literature, Sep. 1973.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Exhibit B11 from Cite No. 62 (Eigen et al., allegedly Apr. 18-20, 1991, "<a href='http://scholar.google.com/scholar?q="Report+on+Evolution+Research"'>Report on Evolution Research</a>", pp. i, 1, 2, 53-56 and 65.</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Exhibit B11a from Cite No. 62 (undated and unsworn statement by Prof. Manfred Eigen).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Exhibit Collection B14 from Cite No. 72.</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Exhibit Ko No. 8 from Cite No. 53(Eigen et al., allegedly Apr. 18-20, 1991, "<a href='http://scholar.google.com/scholar?q="Report+on+Evolution+Research%2C"'>Report on Evolution Research,</a>"Department of Biochemical Kinetics, Max Planck Institute Fur Biophysikalische Chemie, complete document).</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Exhibit No. 21 from Cite No. 58 (Eigen et al., "<a href='http://scholar.google.com/scholar?q="Report+on+the+International+Workshop+Selection-Natural+and+Unnatural-In+Biotechnology"'>Report on the International Workshop Selection-Natural and Unnatural-In Biotechnology</a>").</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Exhibit No. 21 from Cite No. 58 (Eigen et al., "<a href='http://scholar.google.com/scholar?q="Report+on+the+International+Workshop+Selection%E2%80%94Natural+and+Unnatural%E2%80%94In+Biotechnology"'>Report on the International Workshop Selection—Natural and Unnatural—In Biotechnology</a>").</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">FluoroLog-2 Spectrofluorometer Manual (publication date unknown).</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Full copy of "<a href='http://scholar.google.com/scholar?q="Report+on+Evolution+Research%2C"'>Report on Evolution Research,</a>" Department of Biochemical Kinetics, Max Planck Institut für Biophysikalische Chemie, allegedly published Apr. 18-20, 1991, submitted as Exhibit 8 in item 51 above, portions of which were cited in this application as item 41 on the Form 1449A submitted to the Patent Office on or about Oct. 24, 2003.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Glazer et al., Proc. Natl. Acad. Sci. USA 87:3851-3855 (1990).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Haff et al., A high-performance system for automation of the polymerase chain reaction, BioTechniques 10(1):102-112 (Jan. 1991).</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Haff, L.A. "<a href='http://scholar.google.com/scholar?q="New+Developments+in+Polymerase+Chain+Reaction"'>New Developments in Polymerase Chain Reaction</a>" Amplifications, Issue 1, Feb., 1989, pp. 1-12.*</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Henco et al., Determination of in vitro amplified nucleic acid sequences, Chem. Abstr. 119 (15):153368b (1993).</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Higuchi et al., Kinetic PCR analysis: real-time monitoring of DNA amplification reactions, Bio/Technology 11:1026-1030 (1993).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Higuchi et al., Simultaneous amplification and detection of specific DNA sequences, Bio/Technology 10:413-417 (1992).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holland et al., 1991, FASEB 5(4): A261.</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Isaacs et al., Nucleic Acids Research, 19(1):109-116 (Jan. 1991).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Johnson and Hearst, Photochem. and Photobiol., 33:785-791 (1981).</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kaledin, et al. Biokhimiya 46(9):1576-1584 (1982).</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kapuseinski and Szer, Nuc. Acids Res. 6(112):3519-3535 (1979).</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Katz et al., Rapid separation, quantitation and purification of products of polymerase chain reaction by liquid chromatography, J. Chromatography 512:433-444 (1990).</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kellogg et al., Quantitation of HIV-1 Proviral DNA relative to cellular DNA by the polymerase chain reaction, Anal. Biochem. 189:202-208 (1990).</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Khattat et al., JACS, 112:4960 (1990).</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kirk-Othmer, Encyclopedia of Chemical Technology (Third Edition), 3:906-907.</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kornberg, DNA Replication, 427-441 (1980).</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kornberg, DNA Replication, 451-455 (1992).</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kornberg, in DNA Synthesis, W.H. Freman and Co., San Francisco 220-236 (1974).</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kosarev et al., Determination of bacterial concentration in suspension-involves using di:amino-ethyl-phenyl-phananthridinium bromide as fluorescing substance and measuring fluorescence intensity after excitation, Database WPIL Week 8651 Derwent Publications Ltd., Lond, GB; AN 86-338273 &amp; SU-A-1 231 077 (Appl Microbiol Res) May 15, 1986.</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kosarev et al., Determination of bacterial concentration in suspension—involves using di:amino-ethyl-phenyl-phananthridinium bromide as fluorescing substance and measuring fluorescence intensity after excitation, Database WPIL Week 8651 Derwent Publications Ltd., Lond, GB; AN 86-338273 &amp; SU-A-1 231 077 (Appl Microbiol Res) May 15, 1986.</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kramer et al., Evolution in vitro: sequence phenotype of a mutant RNA resistant to ethidium bromide, J. Mol. Biol. 89:719-736 (1974).</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Literature Express Circle 245, 1990, BioTechniques 9(5): 650.</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lomell, H., et al., Clin. Chem., 35(9):1826-1831 (1989).</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mabuchi and Nishikawa, Nuc. Acids Res. 18(24):7461-7462 (1990).</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McCleod, 1990 J. Med. Eng. Tech. 14: 60-68.</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Millipore Direct Catalogue, p. 393-396 (1994-1995).</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">MJ Research Answer dated Mar. 26, 2004 filed in Applera v. MJ Research, Düsseldorf Regional Court Case No. 4a O 235/03 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">MJ Research Motion (Requesting Stay of Suit Pending Outcome of Opposition) dated Dec. 8, 2003 filed in Applera v. MJ Research, Dusseldorf Regional Court Case No. 4a O 262/02 (English language translation only).</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Morrison et al., Anal. Biochem. 183:231-244 (1989).</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Murakami et al., 1989, Nucl. Acids Res. 17: 5587-5595.</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Notice of Intervention submitted by MJ Research, Inc. in corresponding European Patent EP 0 872 562.</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Opposition Papers submitted by Bibby Sterilin Ltd. in corresponding European Patent EP 0 872 562.</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Opposition Papers submitted by Bio-Rad Inc. in corresponding European Patent EP 0 872 562.</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Opposition Papers submitted by Cepheid Inc. in corresponding European Patent EP 0 872 562.</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Opposition Papers submitted by Eppendorf AG in corresponding European Patent EP 0 872 562.</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Opposition Papers submitted by the Secretary of State for Defence, UK in corresponding European Patent EP 0 872 562.</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Oser and Valet, Angew. Chem. Int. Engl. 29(10):1167 (1990).</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Richardson, J. Mol. Biol. 78:703-714 (1973).</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rolf Gunther, "<a href='http://scholar.google.com/scholar?q="Construction+of+a+Multichannel+Fiberoptic+Fluorimeter%2C"'>Construction of a Multichannel Fiberoptic Fluorimeter,</a>" Dissertation prepared at The Max Planck Institute for Biophysical Chemistry at Gottingen, pp. 1-82 (in German, and English translation provided).</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Searle and Embrey, Nuc. Acids Res., 18(13):3653-3762 (1990).</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sharp et al., Biochemistry 12:3055 (1973).</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sobel and Jain, J. Mol. Biol. 68:21-34 (1972).</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">SPEX Catalog (publication date unknown).</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stephen J. Morris, "<a href='http://scholar.google.com/scholar?q="Real-Time+Multi-Wavelength+Fluorescence+Imaging+of+Living+Cells%2C"'>Real-Time Multi-Wavelength Fluorescence Imaging of Living Cells,</a>" BioTechniques 8(3) :296-308 (1990).</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stufen zum Leben, Piper Verlag, 1987, pp. 5, 193-199.</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wittwer et al., 1990, Anal. Biochem. 186: 328-331.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7315376">US7315376</a></td><td class="patent-data-table-td patent-date-value">Jan 7, 2005</td><td class="patent-data-table-td patent-date-value">Jan 1, 2008</td><td class="patent-data-table-td ">Advanced Molecular Systems, Llc</td><td class="patent-data-table-td ">Fluorescence detection system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7573571">US7573571</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 12, 2007</td><td class="patent-data-table-td patent-date-value">Aug 11, 2009</td><td class="patent-data-table-td ">Los Alamos National Security Llc</td><td class="patent-data-table-td ">Airborne particulate discriminator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7608399">US7608399</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2007</td><td class="patent-data-table-td patent-date-value">Oct 27, 2009</td><td class="patent-data-table-td ">Blood Cell Storage, Inc.</td><td class="patent-data-table-td ">Fluorescence method for determining the quantity of nucleic acids in a sample; provide results within a short amount of time in a closed sterile device that would allow for the clinical determination of the level of microbial contamination within a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7611674">US7611674</a></td><td class="patent-data-table-td patent-date-value">Jan 11, 2007</td><td class="patent-data-table-td patent-date-value">Nov 3, 2009</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Heating/ cooling device for control of a reaction vessel receiver having several recesses arranged in a regular pattern to receive a microtiter plate with severalreaction vessels; thermocycler receiver divided into thermally decoupled segments actuated independentally; automatic; robotics; DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7622296">US7622296</a></td><td class="patent-data-table-td patent-date-value">May 24, 2005</td><td class="patent-data-table-td patent-date-value">Nov 24, 2009</td><td class="patent-data-table-td ">Wafergen, Inc.</td><td class="patent-data-table-td ">Sealed microarrays with transparent cover are placed within indium tin oxide heater; condensation prevention; PCR; high-throughput and low-cost amplification of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7704735">US7704735</a></td><td class="patent-data-table-td patent-date-value">Apr 26, 2007</td><td class="patent-data-table-td patent-date-value">Apr 27, 2010</td><td class="patent-data-table-td ">Fluidigm Corporation</td><td class="patent-data-table-td ">Integrated chip carriers with thermocycler interfaces and methods of using the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7727479">US7727479</a></td><td class="patent-data-table-td patent-date-value">Jun 12, 2006</td><td class="patent-data-table-td patent-date-value">Jun 1, 2010</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Device for the carrying out of chemical or biological reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7777877">US7777877</a></td><td class="patent-data-table-td patent-date-value">Oct 22, 2007</td><td class="patent-data-table-td patent-date-value">Aug 17, 2010</td><td class="patent-data-table-td ">California Institute Of Technology</td><td class="patent-data-table-td ">High efficiency coupling optics for pumping and detection of fluorescence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7833709">US7833709</a></td><td class="patent-data-table-td patent-date-value">May 24, 2005</td><td class="patent-data-table-td patent-date-value">Nov 16, 2010</td><td class="patent-data-table-td ">Wafergen, Inc.</td><td class="patent-data-table-td ">Thermo-controllable chips for multiplex analyses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7939312">US7939312</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 9, 2007</td><td class="patent-data-table-td patent-date-value">May 10, 2011</td><td class="patent-data-table-td ">Dxna Llc</td><td class="patent-data-table-td ">Rapid thermocycler with movable cooling assembly</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7955802">US7955802</a></td><td class="patent-data-table-td patent-date-value">Dec 13, 2007</td><td class="patent-data-table-td patent-date-value">Jun 7, 2011</td><td class="patent-data-table-td ">Luminex Corporation</td><td class="patent-data-table-td ">Systems and methods for multiplex analysis of PCR in real time</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7955864">US7955864</a></td><td class="patent-data-table-td patent-date-value">Aug 22, 2006</td><td class="patent-data-table-td patent-date-value">Jun 7, 2011</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Device and method for making discrete volumes of a first fluid in contact with a second fluid, which are immiscible with each other</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7959929">US7959929</a></td><td class="patent-data-table-td patent-date-value">Oct 19, 2006</td><td class="patent-data-table-td patent-date-value">Jun 14, 2011</td><td class="patent-data-table-td ">University Of Florida Research Foundation, Inc.</td><td class="patent-data-table-td ">isolated influenza virus that is capable of infecting a canid animal; genetic engineered influenza virus; vaccines for inducing an immune response against an influenza virus; veterinary medicine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7972778">US7972778</a></td><td class="patent-data-table-td patent-date-value">Mar 11, 2004</td><td class="patent-data-table-td patent-date-value">Jul 5, 2011</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Using multicompartment apparatus for replicating nucleotide sequences on a miniaturized scale; fertility, immunology, cytology and pharmaceutical screening</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8003370">US8003370</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 2008</td><td class="patent-data-table-td patent-date-value">Aug 23, 2011</td><td class="patent-data-table-td ">California Institute Of Technology</td><td class="patent-data-table-td ">Thermal cycling apparatus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8008046">US8008046</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 2008</td><td class="patent-data-table-td patent-date-value">Aug 30, 2011</td><td class="patent-data-table-td ">California Institute Of Technology</td><td class="patent-data-table-td ">Thermal cycling method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8048626">US8048626</a></td><td class="patent-data-table-td patent-date-value">Jul 27, 2007</td><td class="patent-data-table-td patent-date-value">Nov 1, 2011</td><td class="patent-data-table-td ">California Institute Of Technology</td><td class="patent-data-table-td ">Multiplex Q-PCR arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8048628">US8048628</a></td><td class="patent-data-table-td patent-date-value">May 24, 2007</td><td class="patent-data-table-td patent-date-value">Nov 1, 2011</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods for nucleic acid amplification on a printed circuit board</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8053213">US8053213</a></td><td class="patent-data-table-td patent-date-value">Sep 7, 2010</td><td class="patent-data-table-td patent-date-value">Nov 8, 2011</td><td class="patent-data-table-td ">Roche Diagnostics Operations, Inc.</td><td class="patent-data-table-td ">Detection of PCR products in gel electrophoresis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8067159">US8067159</a></td><td class="patent-data-table-td patent-date-value">Aug 13, 2007</td><td class="patent-data-table-td patent-date-value">Nov 29, 2011</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Microfluidics; electromagnetics; electrophoresis; DNA sequence analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8105553">US8105553</a></td><td class="patent-data-table-td patent-date-value">Jan 25, 2005</td><td class="patent-data-table-td patent-date-value">Jan 31, 2012</td><td class="patent-data-table-td ">Fluidigm Corporation</td><td class="patent-data-table-td ">Apparatus for operating microfluidic device comprising platen having face with fluid ports therein, fluid ports spatially corresponding to inlets on surface of microfluidic device, platform for holding microfluidic device relative to platen, actuator for urging platen against device to change pressure</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8105824">US8105824</a></td><td class="patent-data-table-td patent-date-value">Jan 10, 2011</td><td class="patent-data-table-td patent-date-value">Jan 31, 2012</td><td class="patent-data-table-td ">Fluidigm Corporation</td><td class="patent-data-table-td ">Integrated chip carriers with thermocycler interfaces and methods of using the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8147668">US8147668</a></td><td class="patent-data-table-td patent-date-value">Oct 23, 2006</td><td class="patent-data-table-td patent-date-value">Apr 3, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Apparatus for manipulating droplets</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8148115">US8148115</a></td><td class="patent-data-table-td patent-date-value">Oct 26, 2009</td><td class="patent-data-table-td patent-date-value">Apr 3, 2012</td><td class="patent-data-table-td ">Blood Cell Storage, Inc.</td><td class="patent-data-table-td ">Device and method for extraction and analysis of nucleic acids from biological samples</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8163535">US8163535</a></td><td class="patent-data-table-td patent-date-value">Jan 2, 2009</td><td class="patent-data-table-td patent-date-value">Apr 24, 2012</td><td class="patent-data-table-td ">Blood Cell Storage, Inc.</td><td class="patent-data-table-td ">Devices and processes for nucleic acid extraction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8232091">US8232091</a></td><td class="patent-data-table-td patent-date-value">May 17, 2007</td><td class="patent-data-table-td patent-date-value">Jul 31, 2012</td><td class="patent-data-table-td ">California Institute Of Technology</td><td class="patent-data-table-td ">Thermal cycling system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8252581">US8252581</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2008</td><td class="patent-data-table-td patent-date-value">Aug 28, 2012</td><td class="patent-data-table-td ">Wafergen, Inc.</td><td class="patent-data-table-td ">Apparatus for high throughput chemical reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8278071">US8278071</a></td><td class="patent-data-table-td patent-date-value">Aug 13, 2007</td><td class="patent-data-table-td patent-date-value">Oct 2, 2012</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Method for detecting the presence of a single target nucleic acid in a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8288105">US8288105</a></td><td class="patent-data-table-td patent-date-value">May 25, 2011</td><td class="patent-data-table-td patent-date-value">Oct 16, 2012</td><td class="patent-data-table-td ">Luminex Corporation</td><td class="patent-data-table-td ">Systems and methods for multiplex analysis of PCR in real time</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8343443">US8343443</a></td><td class="patent-data-table-td patent-date-value">Jun 23, 2008</td><td class="patent-data-table-td patent-date-value">Jan 1, 2013</td><td class="patent-data-table-td ">Agency For Science, Technology And Research</td><td class="patent-data-table-td ">Fluid processing and transfer using inter-connected multi-chamber device</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8361807">US8361807</a></td><td class="patent-data-table-td patent-date-value">Mar 22, 2011</td><td class="patent-data-table-td patent-date-value">Jan 29, 2013</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Device and method for making discrete volumes of a first fluid in contact with a second fluid, which are immiscible with each other</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8389288">US8389288</a></td><td class="patent-data-table-td patent-date-value">Jan 18, 2010</td><td class="patent-data-table-td patent-date-value">Mar 5, 2013</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Device for the carrying out of chemical or biological reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8629264">US8629264</a></td><td class="patent-data-table-td patent-date-value">May 18, 2012</td><td class="patent-data-table-td patent-date-value">Jan 14, 2014</td><td class="patent-data-table-td ">Blood Cell Storage, Inc.</td><td class="patent-data-table-td ">Gravity flow fluidic device for nucleic acid extraction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8633015">US8633015</a></td><td class="patent-data-table-td patent-date-value">Sep 24, 2010</td><td class="patent-data-table-td patent-date-value">Jan 21, 2014</td><td class="patent-data-table-td ">Bio-Rad Laboratories, Inc.</td><td class="patent-data-table-td ">Flow-based thermocycling system with thermoelectric cooler</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8652780">US8652780</a></td><td class="patent-data-table-td patent-date-value">Mar 26, 2008</td><td class="patent-data-table-td patent-date-value">Feb 18, 2014</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Restriction endonuclease enhanced polymorphic sequence detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8658099">US8658099</a></td><td class="patent-data-table-td patent-date-value">Nov 6, 2008</td><td class="patent-data-table-td patent-date-value">Feb 25, 2014</td><td class="patent-data-table-td ">Agency For Science, Technology And Research</td><td class="patent-data-table-td ">Integrated apparatus for conducting and monitoring chemical reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8658776">US8658776</a></td><td class="patent-data-table-td patent-date-value">May 26, 2011</td><td class="patent-data-table-td patent-date-value">Feb 25, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Synthesis of 2′,3′-dideoxynucleosides for automated DNA synthesis and pyrophosphorolysis activated polymerization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8663920">US8663920</a></td><td class="patent-data-table-td patent-date-value">Jul 30, 2012</td><td class="patent-data-table-td patent-date-value">Mar 4, 2014</td><td class="patent-data-table-td ">Bio-Rad Laboratories, Inc.</td><td class="patent-data-table-td ">Library characterization by digital assay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8676383">US8676383</a></td><td class="patent-data-table-td patent-date-value">Sep 5, 2007</td><td class="patent-data-table-td patent-date-value">Mar 18, 2014</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Device for carrying out chemical or biological reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8691164">US8691164</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2007</td><td class="patent-data-table-td patent-date-value">Apr 8, 2014</td><td class="patent-data-table-td ">Celula, Inc.</td><td class="patent-data-table-td ">Cell sorting system and methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8709762">US8709762</a></td><td class="patent-data-table-td patent-date-value">Mar 2, 2011</td><td class="patent-data-table-td patent-date-value">Apr 29, 2014</td><td class="patent-data-table-td ">Bio-Rad Laboratories, Inc.</td><td class="patent-data-table-td ">System for hot-start amplification via a multiple emulsion</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8721972">US8721972</a></td><td class="patent-data-table-td patent-date-value">May 14, 2012</td><td class="patent-data-table-td patent-date-value">May 13, 2014</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Device for the carrying out of chemical or biological reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8722336">US8722336</a></td><td class="patent-data-table-td patent-date-value">May 25, 2012</td><td class="patent-data-table-td patent-date-value">May 13, 2014</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Restriction endonuclease enhanced polymorphic sequence detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8730479">US8730479</a></td><td class="patent-data-table-td patent-date-value">Dec 30, 2011</td><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">Bio-Rad Laboratories, Inc.</td><td class="patent-data-table-td ">Detection system for droplet-based assays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8771955">US8771955</a></td><td class="patent-data-table-td patent-date-value">Mar 30, 2012</td><td class="patent-data-table-td patent-date-value">Jul 8, 2014</td><td class="patent-data-table-td ">Blood Cell Storage, Inc.</td><td class="patent-data-table-td ">Device and method for extraction and analysis of nucleic acids from biological samples</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2208796A2?cl=en">EP2208796A2</a></td><td class="patent-data-table-td patent-date-value">Jan 6, 2006</td><td class="patent-data-table-td patent-date-value">Jul 21, 2010</td><td class="patent-data-table-td ">Applied Biosystems, LLC</td><td class="patent-data-table-td ">Polypeptides having nucleic acid binding activity and compositions and methods for nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2407480A1?cl=en">EP2407480A1</a></td><td class="patent-data-table-td patent-date-value">Oct 19, 2006</td><td class="patent-data-table-td patent-date-value">Jan 18, 2012</td><td class="patent-data-table-td ">University of Florida Research Foundation, Incorporated</td><td class="patent-data-table-td ">Materials and methods for respiratory disease control in canines</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2489739A2?cl=en">EP2489739A2</a></td><td class="patent-data-table-td patent-date-value">Apr 21, 2006</td><td class="patent-data-table-td patent-date-value">Aug 22, 2012</td><td class="patent-data-table-td ">University of Florida Research Foundation, Inc.</td><td class="patent-data-table-td ">Materials and methods for respiratory disease control in canines</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2495326A2?cl=en">EP2495326A2</a></td><td class="patent-data-table-td patent-date-value">Apr 21, 2006</td><td class="patent-data-table-td patent-date-value">Sep 5, 2012</td><td class="patent-data-table-td ">University of Florida Research Foundation, Inc.</td><td class="patent-data-table-td ">Materials and methods for respiratory disease control in canines</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2660482A1?cl=en">EP2660482A1</a></td><td class="patent-data-table-td patent-date-value">Aug 22, 2006</td><td class="patent-data-table-td patent-date-value">Nov 6, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Apparatus, system, and method using immiscible-fluid-discrete-volumes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2007024778A2?cl=en">WO2007024778A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 22, 2006</td><td class="patent-data-table-td patent-date-value">Mar 1, 2007</td><td class="patent-data-table-td ">Applera Corp</td><td class="patent-data-table-td ">Device, system and method for depositing processed immiscible-fluid-discrete-volumes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2007147063A2?cl=en">WO2007147063A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 14, 2007</td><td class="patent-data-table-td patent-date-value">Dec 21, 2007</td><td class="patent-data-table-td ">Min Seob Lee</td><td class="patent-data-table-td ">Methods and compositions for the amplification, detection and quantification of nucleic acid from a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2008116941A1?cl=en">WO2008116941A1</a></td><td class="patent-data-table-td patent-date-value">Mar 26, 2007</td><td class="patent-data-table-td patent-date-value">Oct 2, 2008</td><td class="patent-data-table-td ">Fundacion Gaiker</td><td class="patent-data-table-td ">Method and device for detecting genetic material by means of polymerase chain reaction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009054891A2?cl=en">WO2009054891A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 10, 2008</td><td class="patent-data-table-td patent-date-value">Apr 30, 2009</td><td class="patent-data-table-td ">Alonso Castro</td><td class="patent-data-table-td ">Airborne particulate discriminator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009124665A1?cl=en">WO2009124665A1</a></td><td class="patent-data-table-td patent-date-value">Mar 27, 2009</td><td class="patent-data-table-td patent-date-value">Oct 15, 2009</td><td class="patent-data-table-td ">Roche Diagnostics Gmbh</td><td class="patent-data-table-td ">Detection of pcr products in gel electrophoresis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010111509A1?cl=en">WO2010111509A1</a></td><td class="patent-data-table-td patent-date-value">Mar 25, 2010</td><td class="patent-data-table-td patent-date-value">Sep 30, 2010</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Discriminatory positive/extraction control dna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010146339A1?cl=en">WO2010146339A1</a></td><td class="patent-data-table-td patent-date-value">Jun 14, 2010</td><td class="patent-data-table-td patent-date-value">Dec 23, 2010</td><td class="patent-data-table-td ">Bg Research Ltd</td><td class="patent-data-table-td ">Nucleic acid detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011150277A2?cl=en">WO2011150277A2</a></td><td class="patent-data-table-td patent-date-value">May 26, 2011</td><td class="patent-data-table-td patent-date-value">Dec 1, 2011</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Synthesis of 2&#39;, 3&#39;-dideoxynucleosides for automated dna synthesis and pyrophosphorolysis activated polymerization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012099896A2?cl=en">WO2012099896A2</a></td><td class="patent-data-table-td patent-date-value">Jan 17, 2012</td><td class="patent-data-table-td patent-date-value">Jul 26, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Workflow for detection of ligands using nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012154555A1?cl=en">WO2012154555A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 4, 2012</td><td class="patent-data-table-td patent-date-value">Nov 15, 2012</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Nanocompositions for monitoring polymerase chain reaction (pcr)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012170907A2?cl=en">WO2012170907A2</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2012</td><td class="patent-data-table-td patent-date-value">Dec 13, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Polymerization of nucleic acids using proteins having low isoelectric points</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012170908A1?cl=en">WO2012170908A1</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2012</td><td class="patent-data-table-td patent-date-value">Dec 13, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Design and development of novel detergents for use in pcr systems</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013023220A2?cl=en">WO2013023220A2</a></td><td class="patent-data-table-td patent-date-value">Aug 13, 2012</td><td class="patent-data-table-td patent-date-value">Feb 14, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Systems and methods for nucleic acid-based identification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013026027A1?cl=en">WO2013026027A1</a></td><td class="patent-data-table-td patent-date-value">Aug 17, 2012</td><td class="patent-data-table-td patent-date-value">Feb 21, 2013</td><td class="patent-data-table-td ">Nestec S.A.</td><td class="patent-data-table-td ">Compositions and methods for detecting allelic variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013188839A1?cl=en">WO2013188839A1</a></td><td class="patent-data-table-td patent-date-value">Jun 14, 2013</td><td class="patent-data-table-td patent-date-value">Dec 19, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Novel compositions, methods and kits for real time polymerase chain reaction (pcr)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2014071315A1?cl=en">WO2014071315A1</a></td><td class="patent-data-table-td patent-date-value">Nov 4, 2013</td><td class="patent-data-table-td patent-date-value">May 8, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Novel compositions, methods and kits for enhancing pcr specificity</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc422/defs422.htm&usg=AFQjCNHXLT_uXJvUp9ERMJkh6x6moeyOWg#C422S082080">422/82.08</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc250/defs250.htm&usg=AFQjCNE2JFZGWgKce_vsAhza5snqYsKAQg#C250S458100">250/458.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc422/defs422.htm&usg=AFQjCNHXLT_uXJvUp9ERMJkh6x6moeyOWg#C422S307000">422/307</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033500000">G01N33/50</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033580000">G01N33/58</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001700000">C12Q1/70</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12M0001000000">C12M1/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0021640000">G01N21/64</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=Y10S435/81">Y10S435/81</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/703">C12Q1/703</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6879">C12Q1/6879</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6818">C12Q1/6818</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/686">C12Q1/686</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6851">C12Q1/6851</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6816">C12Q1/6816</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N21/6428">G01N21/6428</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6876">C12Q1/6876</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5ZtpBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01L7/52">B01L7/52</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68D2C</span>, <span class="nested-value">C12Q1/68M2</span>, <span class="nested-value">C12Q1/68B2</span>, <span class="nested-value">C12Q1/68B2B</span>, <span class="nested-value">C12Q1/70B2B</span>, <span class="nested-value">C12Q1/68D4</span>, <span class="nested-value">C12Q1/68M</span>, <span class="nested-value">G01N21/64H</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Oct 8, 2013</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-12 IS CONFIRMED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">LIEN RELEASE;ASSIGNOR:BANK OF AMERICA, N.A.;REEL/FRAME:030182/0677</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100528</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS, INC., CALIFORNIA</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 9, 2012</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 18, 2010</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100310</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 16, 2010</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 26, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS INC.,CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100226;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20080701</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS, LLC,CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100226;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20081121</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100302;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100226;REEL/FRAME:23985/801</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100302;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100316;REEL/FRAME:23985/801</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100329;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100413;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100329;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100330;REEL/FRAME:23985/801</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100413;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100420;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100504;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100420;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100504;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;REEL/FRAME:23985/801</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;REEL/FRAME:023994/0587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;REEL/FRAME:023994/0538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;REEL/FRAME:023985/0801</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS, LLC, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS INC., CALIFORNIA</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 5, 2008</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BANK OF AMERICA, N.A, AS COLLATERAL AGENT, WASHING</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;REEL/FRAME:021976/0001</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20081121</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BANK OF AMERICA, N.A, AS COLLATERAL AGENT,WASHINGT</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100216;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100209;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100225;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100302;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100304;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100309;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100316;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100323;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100329;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100330;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100406;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100413;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100415;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100420;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100427;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100504;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100511;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100513;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;REEL/FRAME:21976/1</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 10, 2008</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;REEL/FRAME:021217/0324</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20080630</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 27, 2008</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20080331</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 9, 2008</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 29, 2003</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLERA CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:ROCHE MOLECULAR SYSTEMS, INC.;REEL/FRAME:013838/0508</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20030723</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 14, 2003</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLERA CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:PE CORPORATION (NY);REEL/FRAME:013367/0173</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20020628</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLERA CORPORATION APPLIED BIOSYSTEMS GROUP 850 L</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:PE CORPORATION (NY) /AR;REEL/FRAME:013367/0173</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_4ff636b3d23669b7103f3b3a3a18b4cd.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U3bzX-I8yOGjitPx0xvGEFhl6aNoQ\u0026id=5ZtpBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U3HDlyEXF2ImjpWfBTQ-tLWqWHaTQ\u0026id=5ZtpBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U1xfI4Q-Z17AuZClyYeOUiXKpNAzA","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Instrument_for_monitoring_nucleic_acid_a.pdf?id=5ZtpBAABERAJ\u0026output=pdf\u0026sig=ACfU3U3xYcyaevSa4DDlhpz6VZOhSkocxg"},"sample_url":"http://www.google.com/patents/reader?id=5ZtpBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>