<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6861251 - Translucent solid matrix assay device for microarray analysis - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_4ff636b3d23669b7103f3b3a3a18b4cd/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_4ff636b3d23669b7103f3b3a3a18b4cd__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Translucent solid matrix assay device for microarray analysis"><meta name="DC.contributor" content="Lawrence R. Green" scheme="inventor"><meta name="DC.contributor" content="Pritest, Inc." scheme="assignee"><meta name="DC.date" content="2003-2-24" scheme="dateSubmitted"><meta name="DC.description" content="The present invention concerns methods and compositions relating to matrix arrays. In certain embodiments, the arrays are translucent. In other embodiments, the arrays are reconfigurable. In preferred embodiments, the arrays are translucent and reconfigurable. Reconfigurable arrays may be produced using small linker molecules, such as aptamers or affibodies, attached to the array substrate. Preferably, the small linker molecules bind to an IgG specific portion of antibodies. Such arrays may be used to detect any target that binds selectively or specifically to an IgG, allowing great flexibility of use. Translucent matrix arrays may utilize a translucent, colloidal form of nitrocellulose to coat the array substrate."><meta name="DC.date" content="2005-3-1" scheme="issued"><meta name="DC.relation" content="US:20030112446:A1" scheme="references"><meta name="DC.relation" content="US:20030228637:A1" scheme="references"><meta name="DC.relation" content="US:3985691" scheme="references"><meta name="DC.relation" content="US:4031172" scheme="references"><meta name="DC.relation" content="US:5128528" scheme="references"><meta name="DC.relation" content="US:5270163" scheme="references"><meta name="DC.relation" content="US:5281540" scheme="references"><meta name="DC.relation" content="US:5370842" scheme="references"><meta name="DC.relation" content="US:5401511" scheme="references"><meta name="DC.relation" content="US:5414258" scheme="references"><meta name="DC.relation" content="US:5486452" scheme="references"><meta name="DC.relation" content="US:5512492" scheme="references"><meta name="DC.relation" content="US:5567588" scheme="references"><meta name="DC.relation" content="US:5603872" scheme="references"><meta name="DC.relation" content="US:5608225" scheme="references"><meta name="DC.relation" content="US:5633487" scheme="references"><meta name="DC.relation" content="US:5670637" scheme="references"><meta name="DC.relation" content="US:5696249" scheme="references"><meta name="DC.relation" content="US:5779978" scheme="references"><meta name="DC.relation" content="US:5807522" scheme="references"><meta name="DC.relation" content="US:5827748" scheme="references"><meta name="DC.relation" content="US:5831012" scheme="references"><meta name="DC.relation" content="US:5843653" scheme="references"><meta name="DC.relation" content="US:5880176" scheme="references"><meta name="DC.relation" content="US:5889155" scheme="references"><meta name="DC.relation" content="US:5922593" scheme="references"><meta name="DC.relation" content="US:5956447" scheme="references"><meta name="DC.relation" content="US:6017742" scheme="references"><meta name="DC.relation" content="US:6192168" scheme="references"><meta name="DC.relation" content="US:6242267" scheme="references"><meta name="DC.relation" content="US:6289144" scheme="references"><meta name="DC.relation" content="US:6296189" scheme="references"><meta name="DC.relation" content="US:6331438" scheme="references"><meta name="DC.relation" content="US:6353479" scheme="references"><meta name="DC.relation" content="US:6649416" scheme="references"><meta name="citation_reference" content="Brush, &quot;The Fast Lane,&quot; The Scientist, 14[9]:21, 2000."><meta name="citation_reference" content="Constans, &quot;PIP, PIP, Hooray!,&quot; The Scientist, 16:28, 2002."><meta name="citation_reference" content="Jones, &quot;Troubleshooting Protein Binding in Nitrocellulose Membranes, Part 1: Principles,&quot; IVD Technology. Retrieved Aug. 25, 2003, from the Internet: www.devicelink.com/ivdt/archive/99/03/009.html."><meta name="citation_reference" content="Jones, &quot;Troubleshooting Protein Binding in Nitrocellulose Membranes, Part 2: Common Problems,&quot; IVD Technology. Retrieved Aug. 25, 2003 from the Internet: http://www.devicelink.com/ivdt/archive/99/05/002.html."><meta name="citation_reference" content="Kodak Research and Development, CMOS vs CCD and the future of imaging, Apr. 2001, CMOS, 1-4.*"><meta name="citation_reference" content="PCT Search Report, PCT/US02/32359, dated Oct. 9, 2002."><meta name="citation_reference" content="Stolowitz, et al., &quot;Phenylboronic Acid-Salicyhydroxamic Acid Biconjugates. 1. A novel Boronic Acid Complex for Protein Immobilization,&quot; Bioconjugate Chem.12:229-239, 2001."><meta name="citation_reference" content="Tonkinson, et al., &quot;Nitrocellulose: A Tried and True Polymer Finds Utility as a Post-Genomic Substrate,&quot; Frontiers in Bioscience 7:cl-12, 2002, 22 pages."><meta name="citation_patent_number" content="US:6861251"><meta name="citation_patent_application_number" content="US:10/373,546"><link rel="canonical" href="http://www.google.com/patents/US6861251"/><meta property="og:url" content="http://www.google.com/patents/US6861251"/><meta name="title" content="Patent US6861251 - Translucent solid matrix assay device for microarray analysis"/><meta name="description" content="The present invention concerns methods and compositions relating to matrix arrays. In certain embodiments, the arrays are translucent. In other embodiments, the arrays are reconfigurable. In preferred embodiments, the arrays are translucent and reconfigurable. Reconfigurable arrays may be produced using small linker molecules, such as aptamers or affibodies, attached to the array substrate. Preferably, the small linker molecules bind to an IgG specific portion of antibodies. Such arrays may be used to detect any target that binds selectively or specifically to an IgG, allowing great flexibility of use. Translucent matrix arrays may utilize a translucent, colloidal form of nitrocellulose to coat the array substrate."/><meta property="og:title" content="Patent US6861251 - Translucent solid matrix assay device for microarray analysis"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("mFzpU-PvDIPfsASmsIDwDg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407464522.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("USA"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("mFzpU-PvDIPfsASmsIDwDg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407464522.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("USA"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6861251?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6861251"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=5jhrBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6861251&amp;usg=AFQjCNGg11VPMBo69QG4Wgl8Tfaqlm8DAw" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6861251.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6861251.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20040166494"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US6861251"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6861251" style="display:none"><span itemprop="description">The present invention concerns methods and compositions relating to matrix arrays. In certain embodiments, the arrays are translucent. In other embodiments, the arrays are reconfigurable. In preferred embodiments, the arrays are translucent and reconfigurable. Reconfigurable arrays may be produced using...</span><span itemprop="url">http://www.google.com/patents/US6861251?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6861251 - Translucent solid matrix assay device for microarray analysis</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6861251 - Translucent solid matrix assay device for microarray analysis" title="Patent US6861251 - Translucent solid matrix assay device for microarray analysis"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6861251 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 10/373,546</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Mar 1, 2005</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Feb 24, 2003</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Feb 24, 2003</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US20040166494">US20040166494</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">10373546, </span><span class="patent-bibdata-value">373546, </span><span class="patent-bibdata-value">US 6861251 B2, </span><span class="patent-bibdata-value">US 6861251B2, </span><span class="patent-bibdata-value">US-B2-6861251, </span><span class="patent-bibdata-value">US6861251 B2, </span><span class="patent-bibdata-value">US6861251B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Lawrence+R.+Green%22">Lawrence R. Green</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Pritest,+Inc.%22">Pritest, Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6861251.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6861251.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6861251.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (35),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (8),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (11),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (38),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (10)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6861251&usg=AFQjCNFGLkGo2fruaDOGb8Jk_Wt9u0twCQ">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6861251&usg=AFQjCNEkbXSomZ-9nD3mg4SKyWWm0kKCQw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6861251B2%26KC%3DB2%26FT%3DD&usg=AFQjCNEu73GjHbPi6bGdJoFu8PKzLTF5FQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT55383485" lang="EN" load-source="patent-office">Translucent solid matrix assay device for microarray analysis</invention-title></span><br><span class="patent-number">US 6861251 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50792798" lang="EN" load-source="patent-office"> <div num="P-00001" class="abstract">The present invention concerns methods and compositions relating to matrix arrays. In certain embodiments, the arrays are translucent. In other embodiments, the arrays are reconfigurable. In preferred embodiments, the arrays are translucent and reconfigurable. Reconfigurable arrays may be produced using small linker molecules, such as aptamers or affibodies, attached to the array substrate. Preferably, the small linker molecules bind to an IgG specific portion of antibodies. Such arrays may be used to detect any target that binds selectively or specifically to an IgG, allowing great flexibility of use. Translucent matrix arrays may utilize a translucent, colloidal form of nitrocellulose to coat the array substrate.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(2)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6861251B2/US06861251-20050301-D00000.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6861251B2/US06861251-20050301-D00000.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6861251B2/US06861251-20050301-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6861251B2/US06861251-20050301-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(17)</span></span></div><div class="patent-text"><div mxw-id="PCLM8811076" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A translucent matrix array comprising:
<div class="claim-text">a) a substrate; </div>
<div class="claim-text">b) an optically clear layer of colloidal nitrocellulose coated on the substrate; and </div>
<div class="claim-text">c) one or more first probes attached to the colloidal nitrocellulose. </div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The translucent matrix array of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the first probes are attached to the array in spots of about 300 μm in diameter.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The translucent matrix array of <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein the probes attached to different spots on the array bind to different targets.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The translucent matrix array of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the first probes are selected from the group consisting of antibodies, antibody fragments, FAb fragments, humanized antibodies, single-chain antibodies, chimeric antibodies, oligonucleotides, polynucleotides, nucleic acids, aptamers and affibodies.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. The translucent matrix array of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the substrate is a glass substrate.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. The translucent matrix array of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the substrate is a plastic substrate.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. The translucent matrix array of <claim-ref idref="CLM-00004">claim 4</claim-ref>, wherein the first probes are aptamers or affibodies.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. The translucent matrix array of <claim-ref idref="CLM-00007">claim 7</claim-ref>, wherein the first probes bind to antibodies, antibody fragments, FAb fragments, humanized antibodies, single-chain antibodies or chimeric antibodies.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The translucent matrix array of <claim-ref idref="CLM-00008">claim 8</claim-ref>, wherein the first probes bind to IgG (immunoglobulin type G) antibodies.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The translucent matrix array of <claim-ref idref="CLM-00004">claim 4</claim-ref>, wherein the first probes are antibodies.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. The translucent matrix array of <claim-ref idref="CLM-00010">claim 10</claim-ref>, wherein the first probes are anti-mouse antibodies.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. The translucent matrix array of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the first probes are attached directly to the nitrocellulose.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00013" num="00013" class="claim">
      <div class="claim-text">13. The translucent matrix array of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the one or more first probes are capable of binding to one or more targets.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00014" num="00014" class="claim">
      <div class="claim-text">14. The translucent matrix array of <claim-ref idref="CLM-00013">claim 13</claim-ref>, wherein the one or more targets are selected from the group consisting of a protein, peptide, carbohydrate, polysaccharide, glycoprotein, lipid, hormone, growth factor, cytokine, receptor, antigen, allergen, antibody, substrate, metabolite, cofactor, inhibitor, drug, pharmaceutical, nutrient, toxin, poison, explosive, pesticide, chemical warfare agent, biowarfare agent, biohazardous agent, infectious agent, prion, radioisotope, vitamin, heterocyclic aromatic compound, carcinogen, mutagen, narcotic, amphetamine, barbiturate, hallucinogen, waste product, contaminant, heavy metal, virus, bacterium, <i>Salmonella, Streptococcus, Legionella, E. coli, Giardia, Cryptosporidium, Rickettsia</i>, spore, mold, yeast, algae, amoebae, dinoflagellate, unicellular organism, pathogen and cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00015" num="00015" class="claim">
      <div class="claim-text">15. The translucent matrix array of <claim-ref idref="CLM-00013">claim 13</claim-ref>, further comprising an optical detector aligned to detect fluorescent labels attached to the targets.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00016" num="00016" class="claim">
      <div class="claim-text">16. The translucent matrix array of <claim-ref idref="CLM-00015">claim 15</claim-ref>, wherein the optical detector comprises a CMOS imager.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00017" num="00017" class="claim">
      <div class="claim-text">17. The translucent matrix array of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the optically clear nitrocellulose layer transmits light signals without scatter or interference.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES15833620" lang="EN" load-source="patent-office" class="description">
    <heading>BACKGROUND OF THE INVENTION</heading> <p num="P-00002">1. Field of the Invention</p>
    <p num="P-00003">The present invention relates to the field of microarray analysis. More particularly, the present invention concerns methods, compositions and apparatus relating to translucent, solid, two-dimensional (2D) matrix assay devices for microarray analysis. In certain embodiments of the invention, the microarray may be a reconfigurable microarray.</p>
    <p num="P-00004">2. Description of Related Art</p>
    <p num="P-00005">Microarray analysis involves the attachment of capture molecules, such as antibodies or oligonucleotides, to a solid matrix. Typically, the array is designed so that capture molecules specific for particular target analytes are attached to identifiable locations on the matrix. After exposure to a sample suspected of containing one or more target analytes, the matrix is analyzed to determine if substances in the sample bind to the capture molecules at one or more locations on the array.</p>
    <p num="P-00006">Two-dimensional microarrays have proven useful for a wide range of applications, such as genomic research. Arrays of oligonucleotide probes may be used to determine the match or mismatch for a given sample of DNA or RNA, as in the detection of disease-associated single nucleotide polymorphisms (SNPs). Gene expression-profiling with microarrays containing probes against target gene mRNAs has been used to identify genes that are up- or down-regulated in response to disease, drug treatment, developmental stage and other conditions. Microarrays have also been of use for applications in protein research. However, proteins are more difficult to attach to a solid matrix and far more complex than oligonucleotides. Thus, techniques for use with protein (antibody) microarrays often require modifications compared to the more simple nucleic acid microarrays. (See, e.g., Constans, <i>The Scientist</i>, 16:28, 2002.)</p>
    <p num="P-00007">Many clinical diagnostic devices have been built around microarray platforms incorporating an appropriate solid matrix. These often contain capture molecules that have been printed or otherwise permanently affixed to the matrix. One of the problems with such fixed arrays is that they are static. Once an array has been printed, it cannot be changed or adapted to conduct any tests other than the ones that it was originally designed for. A reconfigurable microarray would be very advantageous in allowing flexibility of use.</p>
    <p num="P-00008">Existing microarrays face additional problems. For example, the type of solid matrix used may affect the results obtained, depending on the method of analysis and the materials used. Most microarrays are produced using covalent, electrostatic or hydrophobic binding to attach capture probes to the surface of a solid matrix. The capture probes remain attached to the surface during sample analysis. Bound target molecules may be detected in a variety of ways. Most commonly, one or more fluorophore tags are attached to the target molecules or cells that are to be bound by a capture molecule. Once binding is complete the tags may be spectrophotometrically detected. Scanners, CCD cameras or similar detectors may be used to determine the location and signal intensity of fluorescent tags bound to matrix arrays.</p>
    <p num="P-00009">The amount of probe material that can be affixed on a matrix surface depends on the composition of the solid matrix. If insufficient amounts of probe are affixed to the matrix, the resulting fluorescent signal will be so weak that it cannot be detected even if the probe captures a tagged target molecule. It is also not sufficient to bind high concentrations of probe molecules to the surface of a solid matrix, if the matrix does not provide sufficient conformational or steric freedom to allow probes to bind to target molecules.</p>
    <p num="P-00010">The solid matrix must also preserve the functional activity of the probe. Proteins, such as antibodies, attached to a solid matrix may undergo denaturation over time, rendering antibodies inactive or enzymes dysfunctional. In such cases, the signal strength (and the amount of target protein identified in a sample) may vary by the length of time following matrix array manufacture. Although such time-dependent processes may be compensated for in part by the use of external standard proteins, the denaturation rates for different antibodies or enzymes affixed to the same matrix may not be identical.</p>
    <p num="P-00011">Other characteristics of the solid matrix used for 2D arrays may also be important. For example, the opacity of the solid matrix may render it useless for certain kinds of analysis. Opaque materials only allow sample analysis to occur on the same side of the solid matrix as the probe array. This prevents the use of see-through optics that detect light from the opposite side of the matrix. For example, a matrix array may be opposed to a fluidic cube or other fluidic device, with probe molecules attached to the array within a cavity formed by the fluidics cube. Detection of real-time binding of target molecules to the probes would be greatly facilitated if emitted light could be detected from the opposite surface of the array. This is not feasible if the array is opaque to the emitted light.</p>
    <p num="P-00012">A need exists for a translucent solid matrix that could be used with a fluidic cube or other flow device. Such translucent matrix materials should also allow for binding of high concentrations of probe molecules, while maintaining probe molecules in an active state.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="P-00013">The present invention resolves a long-standing need in the art by providing compositions for and methods of production and use of translucent solid 2D matrix arrays. In certain embodiments of the invention, the matrix arrays are microarrays, with binding molecule spots of between 1 μm and 999 μm in size. In particular embodiments, the spots may be between 100 and 500 μm in size. More particularly, the spots may be about 300 μm in size. In specific embodiments, the microarrays may be reconfigurable. Exemplary microarrays of potential use in the claimed methods are disclosed, for example in U.S. patent application Ser. No. 10/035,367, filed Dec. 28, 2001, the entire contents of which are incorporated herein by reference.</p>
    <p num="P-00014">In various embodiments of the invention, the solid matrix arrays may exhibit one or more of the following characteristics: [1] inexpensive to produce; [2] long term stability (retain characteristic features over time); [3] ease of manufacture; [4] reproducible target molecule detection and/or quantification between lots; [5] bind high levels of probes; [6] do not interfere with probe functionality (e.g., binding affinity for target molecules); [7] translucent; and [8] do not interfere with probe-target molecule interaction. Such characteristics of the disclosed solid matrix arrays are advantageous compared to previously known solid matrix arrays.</p>
    <p num="P-00015">Certain embodiments of the invention concern efficient methods for producing a translucent (optically clear) nitrocellulose matrix supported on a solid substrate. Such translucent matrices are ideally suited for 2D microarray analysis. The translucent nitrocellulose matrix preserves protein (antibody) functionality and exhibits many of the characteristics listed above, such as high binding capacity. The translucent nitrocellulose solid matrix is suitable for genomic and proteomic work and can be used in a variety of diagnostic formats. Because it is translucent, it is suitable for use with see-through optics that detect light from the opposite side of the matrix from the capture molecules. A non-limiting example of a fluidics cube type biosensor of use with the claimed methods and compositions is disclosed in U.S. patent application Ser. No. 09/974,089, filed Oct. 1, 2001, the entire contents of which are incorporated herein by reference.</p>
    <p num="P-00016">Other embodiments of the invention concern methods for producing a translucent, optically clear signal using small linker molecules bound to the surface of a solid matrix substrate. Aptamers, affibodies (e.g. U.S. Pat. No. 5,831,012), or other linkers that exhibit a high affinity for the Fc portion of certain antibodies may be used to attach antibodies or antibody fragments to a solid matrix. The use of such linkers overcomes the tendency for the solid matrix to display a lower binding capacity for larger molecules (e.g., antibodies). The methods may be used in combination with optically clear nitrocellulose matrix substrates to create reconfigurable microarrays. Alternatively, the methods may be used with traditional translucent substrates such as glass microscope slides.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p num="P-00017">The following drawings form part of the present specification and are included to further demonstrate certain aspects of the present invention. The invention may be better understood by reference to one or more of these drawings in combination with the detailed description of specific embodiments presented herein.</p>
      <p num="P-00018"> <figref idrefs="DRAWINGS">FIG. 1</figref> illustrates an exemplary embodiment of translucent matrix array.</p>
    </description-of-drawings> <heading>DESCRIPTION OF ILLUSTRATIVE EMBODIMENTS</heading> <p num="P-00019">Definitions</p>
    <p num="P-00020">Terms that are not otherwise defined herein are used in accordance with their plain and ordinary meaning.</p>
    <p num="P-00021">As used herein, “a” or “an” may mean one or more than one of an item.</p>
    <p num="P-00022">As used herein, the terms “analyte” and “target” mean any compound, molecule or aggregate of interest for detection. Non-limiting examples of targets include a protein, peptide, carbohydrate, polysaccharide, glycoprotein, lipid, hormone, growth factor, cytokine, receptor, antigen, allergen, antibody, substrate, metabolite, cofactor, inhibitor, drug, pharmaceutical, nutrient, toxin, poison, explosive, pesticide, chemical warfare agent, biowarfare agent, biohazardous agent, infectious agent, prion, radioisotope, vitamin, heterocyclic aromatic compound, carcinogen, mutagen, narcotic, amphetamine, barbiturate, hallucinogen, waste product, contaminant, heavy metal or any other molecule or atom, without limitation as to size. “Targets” are not limited to single molecules or atoms, but may also comprise complex aggregates, such as a virus, bacterium, <i>Salmonella, Streptococcus, Legionella, E. coli, Giardia, Cryptosporidium, Rickettsia</i>, spore, mold, yeast, algae, amoebae, dinoflagellate, unicellular organism, pathogen or cell. In certain embodiments, cells exhibiting a particular characteristic or disease state, such as a cancer cell, may be targets. Virtually any chemical or biological compound, molecule or aggregate could be a target.</p>
    <p num="P-00023">As used herein, “capture molecule” and “probe” refer to a molecule or aggregate that has binding affinity for one or more targets. Within the scope of the present invention virtually any molecule or aggregate that has a binding affinity for some target of interest may be a “capture molecule.” “Capture molecules” include, but are not limited to, polyclonal antibodies, monoclonal antibodies, antibody fragments, FAb fragments, humanized antibodies, single-chain antibodies, chimeric antibodies, affibodies, oligonucleotides, polynucleotides, nucleic acids, aptamers, nucleic acid ligands and any other known ligand that can bind to at least one target molecule. In certain embodiments, the capture molecule is specific for binding to a single target, although in other embodiments the capture molecule may bind to multiple targets that exhibit similar structures or binding domains.</p>
    <p num="P-00024">“Binding” refers to an interaction between a target and a capture molecule, resulting in a sufficiently stable complex so as to permit detection of the target:capture molecule complex. In certain embodiments, binding may also refer to an interaction between a second molecule and a target. For example, in a sandwich ELISA type of detection assay, the capture molecule is an antibody with affinity for a target. After binding of target to capture molecule, a second molecule, typically a tagged antibody with an affinity for a different epitope of the target, is added and the tertiary complex of first antibody:target:second tagged antibody is detected. In alternative embodiments, the first capture molecule may have affinity for a target while the second capture molecule has affinity for the first capture molecule. Although detection may involve the use of a second capture molecule with affinity for a target, in alternative embodiments the binary complex of capture molecule with target may be directly detected. The skilled artisan will be familiar with a variety of techniques by which a target:capture molecule complex may be detected, any of which may be utilized within the scope of the present invention.</p>
    <p num="P-00025">The terms “detection” and “detecting” are used herein to refer to an assay or procedure that is indicative of the presence of one or more specific targets in a sample, or that predicts a disease state or a medical or environmental condition associated with the presence of one or more specific targets in a sample. It will be appreciated by those of skill in the art that all assays exhibit a certain level of false positives and false negatives. Even where a positive result in an assay is not invariably associated with the presence of a target, the result is of use as it indicates the need for more careful monitoring of an individual, a population, or an environmental site. An assay is diagnostic of a disease state or a medical or environmental condition when the assay results show a statistically significant association or correlation with the ultimate manifestation of the disease or condition.</p>
    <p num="P-00026">Traditional Translucent Slides</p>
    <p num="P-00027">Glass or plastic microscope slides have commonly been used as solid matrix supports for microarray analysis. Probe molecules have been attached to glass or plastic surfaces using cross-linking compounds. (See, e.g., Schena, Microarray Analysis, J. Wiley &amp; Sons, New York, N.Y., 648 pp., 2002.) Probes may be printed as 2D arrays of spots, each of about 100 to 500 μm in diameter. The cross-linking compounds and any coating used to attach probes to the glass surface form a solid matrix, on top of the glass substrate. Many different kinds of cross-linkers are known, depending on the type of reactive moieties (e.g., sulfhydryl, amino, carboxyl, phenyl, hydroxyl, aldehyde, etc.) available on the probe molecules that can be cross-linked to the surface without affecting probe functionality (e.g., target molecule binding).</p>
    <p num="P-00028">A problem with previous methods for probe attachment is that the capacity for attachment is limited. As probe size is increased, the number of possible binding sites for prospective target molecules is generally decreased. If the binding sites for the probe are saturated at a level below the threshold for detection, a signal will not be observed even if binding has occurred between probe and target molecule.</p>
    <p num="P-00029">Attempts have been made to attach probes to the glass surface using avidin-coated slides and biotin-conjugated probe molecules. Alternatively, silanes, such as aminosilane or 3-glycidoxypropyltrimethoxysilane, have been coated onto the glass surface, with the silane moiety attached to the glass and the reactive moiety cross-linked to probe molecules. Other approaches have utilized slides coated with reactive substrates with functional aldehyde, carboxyl, epoxy, or amine groups that can form a covalent bond with the probe molecules, affixing them permanently to the glass surface.</p>
    <p num="P-00030">Although these methods work moderately well for small probe molecules, they tend to work poorly for larger probe molecules (e.g., antibodies) where functionality (binding) may depend on probe orientation, flexibility and degree of cross-linking. Covalent attachment methods also tend to bind very little material to the matrix surface. Consequently, probe concentration is low and signal detection is difficult. Because relatively little probe is available on the surface of the 2D array, such systems show a low signal-to-noise ratio for a positive binding reaction between probe and target.</p>
    <p num="P-00031">Protein or peptide target molecules are often detected using antibodies as capture molecules. Two-dimensional arrays used in clinical diagnostics or proteomics frequently utilize antibodies as probes for protein or peptide target molecules. Although antibodies tend to be highly specific for their target antigens, they are not easily attached to glass surfaces with cross-linking agents and standard methods. This is because of the limited amount of material that can be affixed to the matrix with known chemistries, resulting in weak signals generated upon target binding. Another problem is that antibody specific binding cannot be maintained without adequate hydration and support in the matrix. Thus, long term stability of antibody-coupled solid matrix arrays tends to be limited, with inconsistent results obtained depending on the age of the array.</p>
    <p num="P-00032">Attempts have been made to solve this problem by creating an environment that stabilizes the protein and preserves its functional probe features. For example, Prolinx Inc. (Bothell, Wash.) has developed a chemical affinity system using standard glass slides with a polymer brush format affixed to their surface. The system relies upon the interaction between two synthetic small molecules that form a stable complex, phenyldiboronic acid (PDBA) and salicylhydroxamic acid (SHA). (E.g., Stolowitz et al., <i>Bioconjugate Chem</i>. 12:229-239, 2001.) PDBA is first conjugated with protein probes. The conjugated probes then link to SHA attached to the polymer brush to form a 3D functional array. This method is limited by the amount of antibody that can be bound to the surface. More importantly, the target antigen must be sufficiently small to diffuse through the brush border in order to react with antibodies affixed to the matrix. Such methods are not suitable for identifying and/or quantifying larger targets, such as whole cells or bacteria.</p>
    <p num="P-00033">Opaque Slides</p>
    <p num="P-00034">Methods to stabilize and increase the amount of probe attached to matrix arrays are highly desirable. Such methods generally lead to opaque slides, since the matrix materials used to increase probe binding and preserve stability typically involve non-translucent gels, hydro-gels, agars, and other materials coated on the glass surface. Proteins attached to such opaque matrix materials are stabilized by hydrophobic and electrostatic interactions in a three-dimensional array.</p>
    <p num="P-00035">Most scanners in current use for genomic and proteomic microarrays read the slides from the same side as the bound probe and target molecules, using opaque matrix arrays. Opaque matrix-coating materials used to produce microarrays include nylon, PVDF (polyvinylidene fluoride) and nitrocellulose. Nitrocellulose, a traditional polymer substrate in use for more than 50 years, is a substrate with very attractive properties for microarray applications. (E.g., Tonkinson and Stillman, <i>Frontiers in Bioscience </i>7:c1-12, 2002.)</p>
    <p num="P-00036">Opaque nitrocellulose has been extensively used to immobilize proteins and nucleic acids for biomolecular analysis. Nitrocellulose immobilizes molecules of interest in near quantitative fashion and allows for short and long term storage. Nitrocellulose also allows for solution phase target species to efficiently bind to immobilized capture molecules. Diagnostic devices using ELISA methods have employed nitrocellulose membranes with a lateral flow process to bind capture reagents to the membrane (Jones, <i>IVD Technology</i>, 5(2):32, 1999).</p>
    <p num="P-00037">Traditional opaque membrane materials have a number of attractive features. They are inexpensive to construct, bind more than 100 times the amount of protein that can be bound by linker coated glass slides, and are generally easy to work with. This is particularly true for opaque nitrocellulose membranes, which have a long history of use.</p>
    <p num="P-00038">Nitrocellulose is normally produced in a microporous form that may be applied to the surface of glass slides to form an opaque surface. Probes may then be attached to the opaque nitrocellulose membranes in microarrays, using standard nitrocellulose binding methods. Such slides have been used with radioactive, fluorescent and chemiluminescent detection systems (e.g., Brush, The Scientist 14[9]:21, 2000).</p>
    <p num="P-00039">Traditional nitrocellulose membranes are also very brittle in the absence of a supporting structure or foundation, leading to frequent cracking or fragmentation. For this reason, opaque nitrocellulose has been used in a microporous form bound to plastic sheets. Such sheets are always opaque, due to the microporous form, and require a supporting structure (e.g. acetate or cellulose) to avoid damage during handling.</p>
    <p num="P-00040">Although opaque nitrocellulose membranes exhibit high binding capacity, stability and reasonably low interference with target-capture molecule interactions, they are not suitable for use with optical detection systems designed to work from the opposite side of a matrix array.</p>
    <p num="P-00041">Translucent Nitrocellulose Slides</p>
    <p num="P-00042">The methods and compositions disclosed herein may be used to produce translucent surface coatings of colloidal nitrocellulose that retain advantageous binding characteristics of opaque nitrocellulose membranes. The interaction between probe and target molecules can be observed directly on a translucent nitrocellulose solid matrix.</p>
    <p num="P-00043">In some embodiments of the invention, translucent nitrocellulose matrix arrays may be used in combination with a flow cell, fluidics cube or capillary system (e.g., U.S. patent application Ser. No. 09/974,089). In such embodiments, the translucent nitrocellulose matrix may be attached to one side of a glass or plastic slide. Probes may be attached to the nitrocellulose and the interaction between probe and target molecules observed through the glass with a sensor or camera.</p>
    <p num="P-00044">The nitrocellulose material is totally translucent if formed according to the disclosed methods. Light signals may thus be observed without scatter or interference from opaque materials. This allows a greater signal-to-noise ratio and ease of detection of target molecules, compared to opaque microporous nitrocellulose matrix arrays. Such opaque matrix arrays can obscure portions of the light or reaction indicator species (e.g., dye) produced upon binding of target molecules.</p>
    <p num="P-00045">Nitrocellulose in the form of a colloid in an amyl acetate solvent has been used by electron microscopists to make castings for specimens. Colloidal nitrocellulose is formed by casting as an ultra-thin film on a water surface. The film may then be picked up on a transmission electron microscopy (TEM) grid and used as a support film for TEM specimens. Because the film must be very clean and uniform, great care is exercised in its production. Colloidal nitrocellulose is readily soluble in amyl acetate. The amyl acetate is water soluble and evaporates evenly to form uniform films. It is supplied as a 1% solution of very pure nitrocellulose.</p>
    <p num="P-00046">High purity nitrocellulose in EM grade amyl acetate (Collodion) may be purchased from commercial sources. The amyl acetate is purified by refluxing over calcium oxide to remove all moisture. Soluble and suspended material is removed by slow distillation. The removal of all traces of moisture from the solvent permits the formation of very strong colloidal nitrocellulose films with virtually no holes.</p>
    <p num="P-00047">In an exemplary embodiment of the invention, Collodion was obtained in bulk from Ernest F Fullam, Inc. (Latham, N.Y.) and used to manufacture high quality translucent nitrocellulose matrix arrays. An aliquot of 200 μL of 1% Collodion solution was pipetted onto the surface of freshly cleaned standard 25×75 mm glass slides. The Collodion was evenly spread to the edges of the glass slide surface in a dust free area. After drying for 2 hours at room temperature, the slides were heated for an additional hour or more at approximately 60° C. Dried slides were labeled and stored for production of microarrays.</p>
    <p num="P-00048">When using a glass array surface, the edges of each slide were sealed with lacquer (e.g. nail polish) or other adhesive to prevent the ultra-thin nitrocellulose substrate from separating from the glass upon exposure to aqueous solutions. When colloidal nitrocellulose is applied to acetate film or other plastic surfaces, it requires no adhesive and binds avidly. Slides may be composed of almost any translucent material as long as the amyl acetate does not react with the surface to discolor it or alter its properties. Certain types of plastics become opaque when exposed to amyl acetate and are not suitable for use with that solvent system. In alternative embodiments of the invention, the colloidal nitrocellulose may be suspended in other volatile organic solvents besides amyl acetate before application to a glass or other translucent slide.</p>
    <p num="P-00049">The colloidal nitrocellulose slides may be spotted with probes using any known methods for microarray production. Methods for spotting proteins, peptides, oligonucleotides and nucleic acids onto nitrocellulose surfaces are well known in the art. Antibodies and biotinylated bovine serum albumin were used to determine the colloidal nitrocellulose binding capacity. The estimated protein binding capacity for the initial glass matrix arrays was in the range of 100 to 200 μg/cm<sup>2</sup>.</p>
    <p num="P-00050">A CMOS imaging system that detected light emissions through the translucent nitrocellulose matrix arrays was used with a Cy5-streptavidin indicator dye. (See, e.g., U.S. patent application Ser. No. 09/974,089, filed Oct. 1, 2001.) At a loading volume of approximately 5 nanoliters per spot, biotinylated BSA was reproducibly detected at least down to 10 to 20 picograms of protein. Optimal CMOS images were obtained using a protein concentration range of about 20 to 100 μg/mL biotinylated BSA. The CMOS imaging data was confirmed using a 24 hour colloidal gold stain (BioRad, Hercules, Calif.) of the spot arrayed translucent nitrocellulose slides.</p>
    <p num="P-00051">The ability of the colloidal nitrocellulose matrix to maintain antibody binding activity was confirmed by spotting a variety of antibodies on the translucent surface. A concentration range of 20 to 200 μg/mL was used. Nonspecific protein binding sites on the nitrocellulose membrane were blocked with 0.1% BSA in buffer solution. The antibodies were then exposed to solutions containing the appropriate target antigen. Binding was detected using biotinylated second antibodies and Cy5-streptavidin indicator.</p>
    <p num="P-00052">A 2D array spotted with 5 nanoliters per spot of primary goat anti-mouse antibody solution and was developed using a secondary mouse Listeria monoclonal antibody and a secondary biotinylated goat anti-mouse antibody. In this assay, the primary and secondary goat anti-mouse antibodies bound to different epitopes of the secondary mouse antibody. The two secondary antibodies were thus used to detect primary antibody bound to the array. Using a CMOS imaging system, the primary antibody could be detected down to a level of 100 picograms or less of antibody. The CMOS data was confirmed on the same slides using colloidal gold staining.</p>
    <p num="P-00053">An advantage of the translucent nitrocellulose surface is that the progress of the probe binding reaction can be examined by looking through the translucent lower surface of the slide. This allows more effective probe binding to occur. The slide may also be adapted for use with a fluidic cube to mix and deliver samples to the surface. The progress of the probe-target binding reaction may also be monitored in real time through the underside of the slide.</p>
    <p num="P-00054">Reconfigurable Microarrays</p>
    <p num="P-00055">In certain embodiments of the invention, reconfigurable microarrays may be produced by using small linker molecules, such as aptamers or affibodies, bound to the surface of a solid matrix. Aptamers are oligonucleotides derived by an in vitro evolutionary process called SELEX (e.g., Brody and Gold, <i>Molecular Biotechnology </i>74:5-13, 2000). Aptamers may be produced by known methods (e.g., U.S. Pat. Nos. 5,270,163; 5,567,588; 5,670,637; 5,696,249; 5,843,653) or obtained from commercial sources (e.g, Somalogic, Boulder, Colo.). Aptamers are relatively small molecules on the order of 7 to 50 kDa. Because they are small, stable and not as easily damaged as proteins, they may be bound in higher numbers to the surface of a solid matrix. This effectively amplifies the number of probe reactive sites on the surface of an array.</p>
    <p num="P-00056">Affibody® ligands (U.S. Pat. No. 5,831,012) are highly specific affinity proteins that may be designed and used like aptamers. Affibodies may be produced or purchased from commercial sources (Affibody AB, Bromma, Sweden). Aptamers and affibodies may be used in combination with antibodies to increase the functional avidity of translucent or non-translucent solid matrices for probe molecule binding. Increased binding in turn results in an increased signal strength, greater signal-to-noise ratio, more reproducible target molecule detection and greater sensitivity of detection.</p>
    <p num="P-00057">Reconfigurable microarrays may be used in combination with two antibodies and a capture probe. The capture probe may be an affibody, aptamer or any other probe capable of binding one of the antibodies. Both antibodies should selectively bind to a target cell, molecule or antigen.</p>
    <p num="P-00058">The effectiveness of binding is increased if the capture probe binds to a portion of an antibody characteristic of the IgG class. Such probes would only require a small part of the antibody structure to be present in order to react and bind to an antibody-target complex. Larger targets, such as microbes or cells are covered with numerous antigens that may form very large complexes with antibodies. However, truncated IgG antibody fragments could interact with such large targets and still bind to an aptamer or affibody probe on the slide surface.</p>
    <p num="P-00059">Antibodies are most stable in solution and optimal antigen-target molecule binding occurs with antibodies in solution, not attached to a solid matrix. In preferred embodiments of the invention, the two antibodies are allowed to bind to the target in solution. Once target-antibody complexes are formed, the complex can be exposed to aptamer or affibody probe molecules on the reconfigurable matrix array. The probes may bind to a first antibody, while the second antibody may be conjugated to a fluorescent tag or other marker. The tagged complex may then be detected on the surface of the matrix array, using optical detection or any other known detection method.</p>
    <p num="P-00060">For example, an aptamer may be tailored to specifically bind to the Fe portion of mouse IgG with high affinity. Samples containing target molecules of interest may be allowed to interact in solution with a mouse antibody specific for an antigen of interest. The sample is mixed with a different biotinylated or otherwise tagged second (non-mouse) antibody that binds to a different epitope on the same antigen. The target antigen bound to the first and second antibodies is exposed to the aptamer microarray. The anti-mouse aptamer affixes the complex to the solid matrix. After extensive washing to remove unbound tagged antibodies, the complex containing tagged antibody that is attached to the matrix array surface is detected.</p>
    <p num="P-00061">In some embodiments of the invention, multiple analytes may be simultaneously detected on a reconfigurable microarray. Such multiplex assays require that each second antibody specific for a different target antigen be labeled with a distinguishable label. For example, three different second antibodies may be tagged with red, green or blue fluorophores. Using anti-mouse aptamers, mouse antibodies specific for three different targets may be added to a sample and mixed with the target molecules and second antibodies. After exposure to the aptamer array, the presence of each target may be determined by the presence of red, green or blue fluorophores attached to the matrix. The skilled artisan will realize that the invention is not limited to optically distinguishable fluorophore tags and that any known identifiable tag moieties, such as radioactive, fluorescent, luminescent, enzymatic, etc. may be used. The number of target analytes that may be simultaneously detected is limited only by the number of distinguishable tags that may be attached to the second antibody.</p>
    <p num="P-00062">The skilled artisan will realize that many variations on this scheme may be used within the scope of the claimed methods. For example, in alternative embodiments of the invention, a first antibody may be used in conjunction with multiple tagged second antibodies, each of which binds to a different epitope of the target molecule. This may occur, for example, where the available second antibodies are polyclonal antibodies. Alternatively, use of more than one second antibody with affinity for the same antigen may improve the sensitivity of detection. In another alternative, one second antibody may bind to a class of targets (for example, all <i>E. coli </i>bacteria) while a second antibody binds to a specific subclass (e.g., <i>E. coli </i>strain O157:H7).</p>
    <p num="P-00063">In a non-limiting example, the aptamer detection method may be used to detect microbes in a food sample. An aptamer that exhibits high affinity and specific binding for mouse IgG may be obtained. Such aptamers may be rapidly and readily obtained using SELEX. The anti-mouse IgG aptamer may be attached to a slide using standard methods, preferably with a translucent solid matrix. Non-specific binding sites on the matrix may be blocked and the slide washed before testing.</p>
    <p num="P-00064">To detect <i>Listeria monocytogenes</i>, an IgG mouse anti-<i>Listeria m</i>. antibody may be incubated with a food sample of interest at an appropriate concentration (typically 1 to 50 μg/mL). A rabbit (or other non-mouse) biotinylated secondary anti-<i>Listeria m</i>. antibody (1 to 50 μg/mL) and incubated for 5 to 30 minutes. The sample with both antibodies may then be applied to the array containing anti-mouse IgG aptamers. After a short interval (approximately 15 minutes) the array may be washed so that only mouse IgG and rabbit biotinylated antibody complexed with <i>Listeria m</i>. is retained on the array. A solution of Cy5-strepavidin or other indicator applied to the surface may then reveal the presents or absence of an anti-<i>Listeria m</i>. antibody complex affixed to the surface.</p>
    <p num="P-00065">A single aptamer with high selectivity for IgG mouse antibodies may be used as a universal extender to capture and detect a variety of microbes. The only requirement would be that the mouse antibody to the microbe or antigen reacts only with the target and the aptamer, while the biotinylated second antibody reacts only with the target and not with the aptamer.</p>
    <p num="P-00066">Such an aptamer array is reconfigurable and is dependent only upon the nature of the solutions delivered to its surface. For example, an aptamer array that binds to mouse IgG antibodies may in principal be used to detect any target for which a mouse IgG is available. It is far more cost effective to construct such a reconfigurable array, compared to constructing a series of fixed pattern arrays, each of which may only detect a limited number of targets. Thus, the present invention provides significant cost advantages for use in proteomic and genomic work.</p>
    <p num="P-00067">Although the methods disclosed in this section can be used with opaque arrays, it is most effectively used with a translucent matrix where solutions are delivered on one side and examined for reactivity from the opposite side of the array. A fluidic cube attached to the reactive surface can be adjusted to deliver fluids to various parts of the array, allowing the user to determine which test results will be obtained for a given sample.</p>
    <p num="P-00068">Aptamers are more stable than antibodies and large proteins. They are also smaller and bind in higher concentrations to reactive surfaces. Thus, aptamer-based systems extend the avidity of an array for target molecules beyond what could be achieved using larger proteins for array binding. Although the present example is presented with regard to aptamers, the skilled artisan will realize that affibodies or other small ligands could be used in the practice of the claimed methods.</p>
    <p num="P-00069">Fluorescence Detection</p>
    <p num="P-00070">Fluorophores are frequently used to detect the presence or absence of a coupled reaction on a glass surface. Fluorescence detectors measure the intensity of the evanescent wave produced when a fluorophore is excited with a laser or other light source. Typically the laser is used to excite the fluorophore at its absorption peak and the detector is tuned to read the emission signal at a longer emission wavelength, which is characteristic of that particular fluorophore. The shift in wavelength between absorption and emission is referred to as the Stokes shift. Most fluorescence detection methods use fluorophores with a large Stokes shift so that the emission and absorption curves are well separated. With fluorophores that have a small Stokes shift, it is necessary to excite at a shorter wavelength than the optimal peak absorption maximum because of overlap between the emission and absorption curves. The signal emission intensity is reduced and the sensitivity for detecting target molecules is decreased. The need for a large Stokes shift also limits the choices of fluorophores that can be used.</p>
    <p num="P-00071">Because the curves for absorption and emission are frequently very near to one another, accurate reading of the emission signal may be complicated. If the distance between the emission and absorption curves is small, it is difficult to separate the light from an emission spectrum from that of the absorption signal. Lasers with a narrow band at the absorption peak are frequently used with filters to cut out all light up to a certain critical point just below the emission spectral curve. By selecting an appropriate long pass filter, band pass filter, or combination of long pass and band pass filters, the emission signal can be observed in a narrow window, eliminating much of the interference from the excitatory light source. Interference from the excitatory light source is also avoided by aligning the detector and apparatus so that the emission signal can be read at a large incident angle to the excitation beam. Although filters eliminate most of the signal from the excitatory light source, they also cut out a significant portion of the evanescent (emitted) signal. Most band pass filters cut out as much as 40 to 50% of the emission signal. Long pass filters may cut an additional 10% of the emission signal.</p>
    <p num="P-00072">Fluorescent detection is used in a number of common test methods. DNA hybridization is commonly analyzed in this manner, using an appropriate fluorophore coupled to a set of known oligonucleotides that hybridize to capture oligonucleotides affixed to a slide. Sandwich immunoassays also employ this method of analysis, either using a tagged secondary antibody that binds to a primary antibody, or using a secondary biotinylated antibody and an avidin-fluorophore as the tag. Many variations on this method are well known.</p>
    <p num="P-00073">Various other types of light interference may occur in fluorescent detection. Light scatter occurs by reflection of the excitation beam, while light dispersion occurs by reflection and bending of the excitation beam. Scatter and dispersion may represent a large part of the light striking a detector. In general, when a substance (such as a protein, nucleic acid or other biomolecule) is affixed to the surface of a glass slide, it acts as a mirror to reflect and scatter light in a variety of directions. The amount of surface covered and the mass or density of the attached material may greatly affect the amount of scattered light. The chemical composition of proteins, oligonucleotides or polymers attached to the glass surface may also affect the scattered light (see FIG. <b>1</b>). In addition, the material attached to the glass surface material may itself fluoresce. The glass used may also have surface irregularities that can affect the signals received by the detector. The energy absorbed across the glass may vary from one spot to another, making signal analysis very problematic. Such problems require the use of novel methods of fluorescent detection and/or data analysis.</p>
    <p num="P-00074">Evanescent Emission and Scattered Light</p>
    <p num="P-00075">Evanescent signals are generally very weak and light scatter is intense, making accurate quantitative detection of analytes problematic. Light scatter is frequently assumed to be eliminated by filters. However, scattered light is almost always present and can be a significant part of the total signal reaching a detector. Filters used to remove light scatter also remove much of the target emission signal, thereby decreasing detector sensitivity. Filters may also transmit a small amount of scattered light. If the scattered light is relatively large compared to the evanescent emitted light, the detected signal will be a combination from several sources, only one of which represents target molecule binding.</p>
    <p num="P-00076">The components of light scattering are illustrated in FIG. <b>1</b>. Two spots (e.g., different antibodies) are deposited on a glass surface. During a method to detect a target, one of the spots remains totally non-reactive. The other spot reacts with a target, such as a bacterial pathogen and/or other reagents. Target binding to the reactive antibody increases the mass attached to the spot and results in a larger surface area and a change in molecular structure at the spot. A mass effect has occurred. The light scatter from the reactive spot will be different from the light scatter before target molecule binding. A sensitive photon-counting detector could detect this difference in scatter. A variety of instruments, such as certain flow cytometers and turbidity meters take advantage of scatter to quantify the amount of material in a solution. Those instruments measure the angle of scatter for a beam of light impinging on a target material. The change in signal is the difference between the reference signal (S<sub>ref</sub>) and signal 2 (S<sub>2</sub>). In <figref idrefs="DRAWINGS">FIG. 1</figref> the S<sub>2 </sub>signal is shown as having two components, a modified scatter signal plus a mass effect signal of the coupled pathogen. The signal from the reactive spot changes while the signal from the non-reactive spot signal is constant.
<br>Δ<i>S</i>(non-reactive spot)=0<br>Δ<i>S</i>(reactive spot)=Modified(<i>S</i> <sub>p</sub>)+<i>M</i> <sub>1</sub> <i>−S</i> <sub>ref</sub> </p>
    <p num="P-00079">If the mass effect is sufficient to cause a large scatter effect, the fluorophore used for target detection could be eliminated. For example in DNA hybridization experiments, the mass attached to a surface using standard oligonucleotide probes (about 24 nucleotides in length) may be increased by a factor of 2 or more upon binding of target nucleic acids. Such a large change in mass may be detectable by monitoring light scatter instead of evanescent waves. In the case of a sandwich immunoassay with a biotinylated secondary antibody, another mass effect occurs when the biotinylated antibody binds to the pathogen. A third mass effect occurs when avidin-conjugated fluorophore binds to biotin.</p>
    <p num="P-00080">The most sensitive signal may be obtained by subtracting the initial reference signal from the final captured signal, obtained after the fluorophore has been attached and excited. That signal represents the modified accumulated mass effects and the emission signal for the reactive spot.
<br>Δ<i>S</i>(reactive spot)=Modified accumulated mass effects+Emission−<i>S</i> <sub>ref</sub> </p>
    <p num="P-00082">This method of analysis can be used with a CMOS imager or any known digital imaging method that allows storage of pixel images for subsequent processing. The signal obtained from each spot will contain more useful information and will show a more intense change upon target binding if a proper subtraction method is used. The scatter effect may be turned to an advantage in detecting target binding. Moreover, it is unnecessary to have fluorophore emission and absorption curves well separated, since spurious signals are subtracted out of the image. The full intensity of an emission signal may be measured without reducing emitted light by with filters.</p>
    <p num="P-00083">A subtraction method also eliminates artifacts and defects that may derive, for example, from inhomogeneity (chips, flaws) in the glass slide surface. The non-reactive spots completely blank out and do not appear as a signal.</p>
    <p num="P-00084">Because CMOS imagers and pixel capturing devices in general exhibit a random, very low level noise there are limits as to what kinds of signals can be detected. At any given point in time, the baseline reference may exhibit a random number of spikes. A weak signal falling between two spikes would not normally be detected against this background noise.</p>
    <p num="P-00085">The signal-to-noise problem may be improved if numerous images are captured and added one upon the other. Because the random spikes inherent in a detector such as a CMOS imager are constantly shifting about, accumulating the frame images will tend to average out the random noise. However a weak signal from the emission of an excited fluorophore does not change its pixel location. Therefore, an accumulated signal caused by target binding will increase with time. This method is similar to taking a photoimage of a distant star or galaxy, by tracking the object as it moves across the sky. The object of interest appears brighter against the background with time because the signal has accumulated at the same spot on the detector, while the background light averages out.</p>
    <p num="P-00086">Method of Analysis</p>
    <p num="P-00087">In an exemplary embodiment of the invention, a glass slide or other matrix array is secured on a stage. A fluidic cube is attached to the surface of the glass and used to deliver samples, second antibodies and other reagents. Before target molecule binding, an excitatory laser is focused on one end of the glass slide at an inclined angle about 30 to 40 degrees. The glass slide acts as a waveguide to conduct the excitatory light to spots, containing bound primary antibody, on the glass surface. A CMOS imager is used to capture the light signals. The CMOS chip is located beneath the glass slide and is aligned so that spots on the slide are directly above the imager and are sharply focused on the imager surface with optical lenses and apertures. Additional details of a CMOS imaging system and data analysis method of use are disclosed in U.S. patent application Ser. No. 10/373,408, by Lawrence R. Green, entitled “Image Sensor Optimization,” filed on Feb. 24, 2003, the entire text of which is incorporated herein by reference.</p>
    <p num="P-00088">A number of pictures are taken. Each picture represents a single frame. For example 10 frames are taken using a 50 millisecond exposure. The exposure is selected so that the amount of light captured in a single frame is within the sensitive range for the camera. The 10 digital frames are then added to provide a reference set that is used for subtraction of unwanted (background) signals. The accumulated image is referred to as the calibration slide.</p>
    <p num="P-00089">The fluidic cube is used to expose primary antibody to a sample, bind any target molecules to the first antibody, and bind second antibody to the target. The process ends with binding of avidin-fluorophore to the biotinylated second antibody and a final set of washes. The same number of frames used to obtain the reference slide image are taken of the sample slide, using the same exposures. The cumulative set of frames is referred to as the sample slide image. The luminescent signal for each spot is determined by subtracting the reference slide image from the sample slide image. This process essentially eliminates background noise and matrix array artifacts, resulting in very sensitive detection of target molecules.</p>
    <p num="P-00090">In alternative embodiments of the invention, pictures may be obtained in either still frame or video mode. A typical video frame runs at 2000 ms and captures 100 frames each for the reference and sample analysis. This method removes artifacts and non-reactive spots, leaving only those signals that represent target molecule binding to the array.</p>
    <p num="P-00091">Probe Labels</p>
    <p num="P-00092">In various embodiments of the invention, labeled probes may be prepared by any methods known in the art. In certain embodiments, a label moiety may be incorporated into a probe (e.g., peptide, protein, oligonucleotide) during synthesis. In other embodiments of the invention, labels may be attached by covalent, noncovalent, ionic, van der Waals, hydrogen bonding or other forces following probe synthesis. Methods for attaching fluorescent or other labels to probe molecules are known in the art and any such known method may be used to make labeled probes within the scope of the present invention. In particular embodiments of the invention, a probe molecule may be biotinylated and may bind to an avidin or streptavidin-conjugated fluorophore. Fluorophores and conjugated fluorophores may be obtained from commercial sources, such as Molecular Probes, Inc. (Eugene, Oreg.).</p>
    <p num="P-00093">Labels of use may comprise any composition detectable by electrical, optical, spectrophotometric, photochemical, biochemical, immunochemical, or chemical techniques. Labels may include, but are not limited to, conducting, luminescent, fluorescent, chemiluminescent, bioluminescent and phosphorescent labels, chromogens, enzymes or substrates. Fluorescent molecules suitable for use as labels include, but are not limited to, dansyl chloride, rhodamineisothiocyanate, Alexa 350, Alexa 430, AMCA, BODIPY 630/650, BODIPY 650/665, BODIPY-FL, BODIPY-R6G, BODIPY-TMR, BODIPY-TRX, Cascade Blue, Cy2, Cy3, Cy5,6-FAM, fluorescein, HEX, 6-JOE, Oregon Green 488, Oregon Green 500, Oregon Green 514, Pacific Blue, REG, Rhodamine Green, Rhodamine Red, ROX, TAMRA, TET, Tetramethylrhodamine, and Texas Red. A variety of other known fluorescent or luminescent labels may be utilized. (See, e.g., U.S. Pat. No. 5,800,992; U.S. Pat. No. 6,319,668.)</p>
    <p num="P-00094">All of the COMPOSITIONS, METHODS and APPARATUS disclosed and claimed herein can be made and executed without undue experimentation in light of the present disclosure. While the compositions and methods of this invention have been described in terms of preferred embodiments, it will be apparent to those of skill in the art that variations may be applied to the COMPOSITIONS, METHODS and APPARATUS and in the steps or in the sequence of steps of the methods described herein without departing from the concept, spirit and scope of the invention. More specifically, it will be apparent that certain agents that are both chemically and physiologically related may be substituted for the agents described herein while the same or similar results would be achieved. All such similar substitutes and modifications apparent to those skilled in the art are deemed to be within the spirit, scope and concept of the invention as defined by the appended claims.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3985691">US3985691</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 12, 1974</td><td class="patent-data-table-td patent-date-value">Oct 12, 1976</td><td class="patent-data-table-td ">Societe Nationale Des Poudres Et Explosifs</td><td class="patent-data-table-td ">Particulate nitrocellulose composition</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4031172">US4031172</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 28, 1976</td><td class="patent-data-table-td patent-date-value">Jun 21, 1977</td><td class="patent-data-table-td ">The United States Of America As Represented By The Secretary Of The Army</td><td class="patent-data-table-td ">Diethylene glycol monomethyl ether</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5128528">US5128528</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 1990</td><td class="patent-data-table-td patent-date-value">Jul 7, 1992</td><td class="patent-data-table-td ">Dittler Brothers, Inc.</td><td class="patent-data-table-td ">Matrix encoding devices and methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5270163">US5270163</a></td><td class="patent-data-table-td patent-date-value">Aug 17, 1992</td><td class="patent-data-table-td patent-date-value">Dec 14, 1993</td><td class="patent-data-table-td ">University Research Corporation</td><td class="patent-data-table-td ">Methods for identifying nucleic acid ligands</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5281540">US5281540</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 22, 1991</td><td class="patent-data-table-td patent-date-value">Jan 25, 1994</td><td class="patent-data-table-td ">Abbott Laboratories</td><td class="patent-data-table-td ">Apparatus for sampling and analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5370842">US5370842</a></td><td class="patent-data-table-td patent-date-value">Nov 20, 1992</td><td class="patent-data-table-td patent-date-value">Dec 6, 1994</td><td class="patent-data-table-td ">Canon Kabushiki Kaisha</td><td class="patent-data-table-td ">Sample measuring device and sample measuring system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5401511">US5401511</a></td><td class="patent-data-table-td patent-date-value">Oct 9, 1992</td><td class="patent-data-table-td patent-date-value">Mar 28, 1995</td><td class="patent-data-table-td ">Baxter International Inc.</td><td class="patent-data-table-td ">Phosphatidylethanolamine, hyaluronic acid, drug delivery</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5414258">US5414258</a></td><td class="patent-data-table-td patent-date-value">Nov 22, 1993</td><td class="patent-data-table-td patent-date-value">May 9, 1995</td><td class="patent-data-table-td ">Angstrom Technologies, Inc.</td><td class="patent-data-table-td ">Apparatus and method for calibration of fluorescence detectors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5486452">US5486452</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 10, 1987</td><td class="patent-data-table-td patent-date-value">Jan 23, 1996</td><td class="patent-data-table-td ">Ciba-Geigy Corporation</td><td class="patent-data-table-td ">Devices and kits for immunological analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5512492">US5512492</a></td><td class="patent-data-table-td patent-date-value">May 18, 1993</td><td class="patent-data-table-td patent-date-value">Apr 30, 1996</td><td class="patent-data-table-td ">University Of Utah Research Foundation</td><td class="patent-data-table-td ">Waveguide immunosensor with coating chemistry providing enhanced sensitivity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5567588">US5567588</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 1995</td><td class="patent-data-table-td patent-date-value">Oct 22, 1996</td><td class="patent-data-table-td ">University Research Corporation</td><td class="patent-data-table-td ">Forming nucleic acid-target complexes; partitioning increased affinity nucleic acids; amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5603872">US5603872</a></td><td class="patent-data-table-td patent-date-value">Oct 3, 1994</td><td class="patent-data-table-td patent-date-value">Feb 18, 1997</td><td class="patent-data-table-td ">Baxter International Inc.</td><td class="patent-data-table-td ">Activating hyaluronic acid with a crosslinker in acidic ph; buffering to basic ph and incubating</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5608225">US5608225</a></td><td class="patent-data-table-td patent-date-value">Mar 8, 1995</td><td class="patent-data-table-td patent-date-value">Mar 4, 1997</td><td class="patent-data-table-td ">Hitachi Maxell, Ltd.</td><td class="patent-data-table-td ">Fluorescent detecting apparatus and method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5633487">US5633487</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 1995</td><td class="patent-data-table-td patent-date-value">May 27, 1997</td><td class="patent-data-table-td ">Adaptive Optics Associates, Inc.</td><td class="patent-data-table-td ">Multi-focal vision system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5670637">US5670637</a></td><td class="patent-data-table-td patent-date-value">Mar 27, 1995</td><td class="patent-data-table-td patent-date-value">Sep 23, 1997</td><td class="patent-data-table-td ">Nexstar Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Binding to target molecules; antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5696249">US5696249</a></td><td class="patent-data-table-td patent-date-value">Mar 24, 1995</td><td class="patent-data-table-td patent-date-value">Dec 9, 1997</td><td class="patent-data-table-td ">Nexstar Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Nucleic acid ligands</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5779978">US5779978</a></td><td class="patent-data-table-td patent-date-value">Feb 7, 1997</td><td class="patent-data-table-td patent-date-value">Jul 14, 1998</td><td class="patent-data-table-td ">Avl Medical Instruments Ag</td><td class="patent-data-table-td ">Having different sensor elements positioned on boundary face and connected by common sample channel, means for optical and/or temporal separation of light paths of measurement radiation of individual elements</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5807522">US5807522</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Sep 15, 1998</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Tapping capillary dispenser on support with impulse to break meniscus; gene expression studies to diagnose disease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5827748">US5827748</a></td><td class="patent-data-table-td patent-date-value">Jan 24, 1997</td><td class="patent-data-table-td patent-date-value">Oct 27, 1998</td><td class="patent-data-table-td ">The United States Of America As Represented By The Secretary Of The Navy</td><td class="patent-data-table-td ">Chemical sensor using two-dimensional lens array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5831012">US5831012</a></td><td class="patent-data-table-td patent-date-value">Jan 16, 1995</td><td class="patent-data-table-td patent-date-value">Nov 3, 1998</td><td class="patent-data-table-td ">Pharmacia &amp; Upjohn Aktiebolag</td><td class="patent-data-table-td ">Bacterial receptor structures</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5843653">US5843653</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 1995</td><td class="patent-data-table-td patent-date-value">Dec 1, 1998</td><td class="patent-data-table-td ">Nexstar Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Method for detecting a target molecule in a sample using a nucleic acid ligand</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5880176">US5880176</a></td><td class="patent-data-table-td patent-date-value">Oct 20, 1995</td><td class="patent-data-table-td patent-date-value">Mar 9, 1999</td><td class="patent-data-table-td ">Hitachi Maxell, Ltd.</td><td class="patent-data-table-td ">Nonbleeding; for bar codes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5889155">US5889155</a></td><td class="patent-data-table-td patent-date-value">Mar 31, 1995</td><td class="patent-data-table-td patent-date-value">Mar 30, 1999</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Carbohydrate-directed cross-linking reagents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5922593">US5922593</a></td><td class="patent-data-table-td patent-date-value">May 23, 1997</td><td class="patent-data-table-td patent-date-value">Jul 13, 1999</td><td class="patent-data-table-td ">Becton, Dickinson And Company</td><td class="patent-data-table-td ">Microbiological test panel and method therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5956447">US5956447</a></td><td class="patent-data-table-td patent-date-value">May 5, 1997</td><td class="patent-data-table-td patent-date-value">Sep 21, 1999</td><td class="patent-data-table-td ">Univ Central Florida</td><td class="patent-data-table-td ">Device and method for image acquisition through multi-mode fiber</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6017742">US6017742</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 20, 1998</td><td class="patent-data-table-td patent-date-value">Jan 25, 2000</td><td class="patent-data-table-td ">Nisshinbo Industries, Inc.</td><td class="patent-data-table-td ">Immobilization of biologically active substances on a carrier containing a carbodiimide group-containing polymer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6192168">US6192168</a></td><td class="patent-data-table-td patent-date-value">Apr 9, 1999</td><td class="patent-data-table-td patent-date-value">Feb 20, 2001</td><td class="patent-data-table-td ">The United States Of America As Represented By The Secretary Of The Navy</td><td class="patent-data-table-td ">Reflectively coated optical waveguide and fluidics cell integration</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6242267">US6242267</a></td><td class="patent-data-table-td patent-date-value">Mar 19, 1997</td><td class="patent-data-table-td patent-date-value">Jun 5, 2001</td><td class="patent-data-table-td ">University Of Utah Research Foundation</td><td class="patent-data-table-td ">Oscillation apparatus and methods for multi-analyte homogeneous fluoro-immunoassays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6289144">US6289144</a></td><td class="patent-data-table-td patent-date-value">Jan 19, 2000</td><td class="patent-data-table-td patent-date-value">Sep 11, 2001</td><td class="patent-data-table-td ">Novartis Ag</td><td class="patent-data-table-td ">Sensor platform and method for the parallel detection of a plurality of analytes using evanescently excited luminescence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6296189">US6296189</a></td><td class="patent-data-table-td patent-date-value">Aug 23, 1999</td><td class="patent-data-table-td patent-date-value">Oct 2, 2001</td><td class="patent-data-table-td ">Spectra Science Corporation.</td><td class="patent-data-table-td ">Methods and apparatus employing multi-spectral imaging for the remote identification and sorting of objects</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6331438">US6331438</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 24, 1999</td><td class="patent-data-table-td patent-date-value">Dec 18, 2001</td><td class="patent-data-table-td ">Iowa State University Research Foundation, Inc.</td><td class="patent-data-table-td ">Optical sensors and multisensor arrays containing thin film electroluminescent devices</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6353479">US6353479</a></td><td class="patent-data-table-td patent-date-value">Jun 29, 1999</td><td class="patent-data-table-td patent-date-value">Mar 5, 2002</td><td class="patent-data-table-td ">Hewlett-Packard Company</td><td class="patent-data-table-td ">Media-type encoding and print mode selection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6649416">US6649416</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 18, 2000</td><td class="patent-data-table-td patent-date-value">Nov 18, 2003</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">Comprising fluid-permeable, signal enhancing, porous, textured substrate with open pores; generates optical response when subjected to excitation light energy; enhanced responsivity and selectivity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030112446">US20030112446</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 28, 2002</td><td class="patent-data-table-td patent-date-value">Jun 19, 2003</td><td class="patent-data-table-td ">Benjamin Miller</td><td class="patent-data-table-td ">Method for biomolecular sensing and system thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030228637">US20030228637</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 24, 2002</td><td class="patent-data-table-td patent-date-value">Dec 11, 2003</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Nnitrocellulose and/or hydrogel based solid support for detecting tumors, infections, viral, immunological and inflammatory diseases</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Brush, "<a href='http://scholar.google.com/scholar?q="The+Fast+Lane%2C"'>The Fast Lane,</a>" The Scientist, 14[9]:21, 2000.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Constans, "<a href='http://scholar.google.com/scholar?q="PIP%2C+PIP%2C+Hooray%21%2C"'>PIP, PIP, Hooray!,</a>" The Scientist, 16:28, 2002.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jones, "<a href='http://scholar.google.com/scholar?q="Troubleshooting+Protein+Binding+in+Nitrocellulose+Membranes%2C+Part+1%3A+Principles%2C"'>Troubleshooting Protein Binding in Nitrocellulose Membranes, Part 1: Principles,</a>" IVD Technology. Retrieved Aug. 25, 2003, from the Internet: www.devicelink.com/ivdt/archive/99/03/009.html.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jones, "<a href='http://scholar.google.com/scholar?q="Troubleshooting+Protein+Binding+in+Nitrocellulose+Membranes%2C+Part+2%3A+Common+Problems%2C"'>Troubleshooting Protein Binding in Nitrocellulose Membranes, Part 2: Common Problems,</a>" IVD Technology. Retrieved Aug. 25, 2003 from the Internet: http://www.devicelink.com/ivdt/archive/99/05/002.html.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kodak Research and Development, CMOS vs CCD and the future of imaging, Apr. 2001, CMOS, 1-4.*</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">PCT Search Report, PCT/US02/32359, dated Oct. 9, 2002.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stolowitz, et al., "<a href='http://scholar.google.com/scholar?q="Phenylboronic+Acid-Salicyhydroxamic+Acid+Biconjugates.+1.+A+novel+Boronic+Acid+Complex+for+Protein+Immobilization%2C"'>Phenylboronic Acid-Salicyhydroxamic Acid Biconjugates. 1. A novel Boronic Acid Complex for Protein Immobilization,</a>" Bioconjugate Chem.12:229-239, 2001.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tonkinson, et al., "<a href='http://scholar.google.com/scholar?q="Nitrocellulose%3A+A+Tried+and+True+Polymer+Finds+Utility+as+a+Post-Genomic+Substrate%2C"'>Nitrocellulose: A Tried and True Polymer Finds Utility as a Post-Genomic Substrate,</a>" Frontiers in Bioscience 7:cl-12, 2002, 22 pages.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7297497">US7297497</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 8, 2006</td><td class="patent-data-table-td patent-date-value">Nov 20, 2007</td><td class="patent-data-table-td ">Decision Biomarkers, Inc.</td><td class="patent-data-table-td ">Substrates for isolating, reacting and microscopically analyzing materials</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7384742">US7384742</a></td><td class="patent-data-table-td patent-date-value">Aug 18, 2003</td><td class="patent-data-table-td patent-date-value">Jun 10, 2008</td><td class="patent-data-table-td ">Decision Biomarkers, Inc.</td><td class="patent-data-table-td ">Substrates for isolating reacting and microscopically analyzing materials</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7499166">US7499166</a></td><td class="patent-data-table-td patent-date-value">May 20, 2004</td><td class="patent-data-table-td patent-date-value">Mar 3, 2009</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Wide field imager for quantitative analysis of microarrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8300993">US8300993</a></td><td class="patent-data-table-td patent-date-value">Nov 12, 2009</td><td class="patent-data-table-td patent-date-value">Oct 30, 2012</td><td class="patent-data-table-td ">Mbio Diagnostics, Inc.</td><td class="patent-data-table-td ">Waveguide with integrated lens</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8331751">US8331751</a></td><td class="patent-data-table-td patent-date-value">Nov 9, 2010</td><td class="patent-data-table-td patent-date-value">Dec 11, 2012</td><td class="patent-data-table-td ">mBio Diagnositcs, Inc.</td><td class="patent-data-table-td ">Planar optical waveguide with core of low-index-of-refraction interrogation medium</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8349605">US8349605</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 12, 2006</td><td class="patent-data-table-td patent-date-value">Jan 8, 2013</td><td class="patent-data-table-td ">Colorado State University Research Foundation</td><td class="patent-data-table-td ">Optical analyte sensor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8586347">US8586347</a></td><td class="patent-data-table-td patent-date-value">Sep 15, 2011</td><td class="patent-data-table-td patent-date-value">Nov 19, 2013</td><td class="patent-data-table-td ">Mbio Diagnostics, Inc.</td><td class="patent-data-table-td ">System and method for detecting multiple molecules in one assay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8606066">US8606066</a></td><td class="patent-data-table-td patent-date-value">Dec 4, 2012</td><td class="patent-data-table-td patent-date-value">Dec 10, 2013</td><td class="patent-data-table-td ">Mbio Diagnostics, Inc.</td><td class="patent-data-table-td ">Planar optical waveguide with core of low-index-of-refraction interrogation medium</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2294408A2?cl=en">EP2294408A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 20, 2009</td><td class="patent-data-table-td patent-date-value">Mar 16, 2011</td><td class="patent-data-table-td ">Gentel Biosciences, Inc.</td><td class="patent-data-table-td ">Substrates for multiplexed assays and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2006060663A2?cl=en">WO2006060663A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 2, 2005</td><td class="patent-data-table-td patent-date-value">Jun 8, 2006</td><td class="patent-data-table-td ">Green Lawrence R</td><td class="patent-data-table-td ">Reflex supplemental testing - a rapid, efficient and highly accurate method to identify subjects with an infection, disease or other condition</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009149305A2?cl=en">WO2009149305A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 4, 2009</td><td class="patent-data-table-td patent-date-value">Dec 10, 2009</td><td class="patent-data-table-td ">Agnitio Science &amp; Technology</td><td class="patent-data-table-td ">Method for creating distinct nitrocellulose-based pads on a substrate</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc506/defs506.htm&usg=AFQjCNEHFPQDhDZRgfBXPN_PbZKvoq9Ikg#C506S039000">506/39</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S007100">435/7.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc356/defs356.htm&usg=AFQjCNGiEVb0PD41nDL-c_Hc6_cJvkgI_Q#C356S317000">356/317</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc436/defs436.htm&usg=AFQjCNGx2wdEO-lMZYH5-JBwc-X_dDDKIg#C436S530000">436/530</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc436/defs436.htm&usg=AFQjCNGx2wdEO-lMZYH5-JBwc-X_dDDKIg#C436S527000">436/527</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S007200">435/7.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S288700">435/288.7</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S287200">435/287.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S005000">435/5</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006190">435/6.19</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0040100000">C40B40/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0050140000">C40B50/14</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0040060000">C40B40/06</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=B01J0019000000">B01J19/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0060140000">C40B60/14</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00612">B01J2219/00612</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00722">B01J2219/00722</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/0074">B01J2219/0074</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C40B50/14">C40B50/14</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00315">B01J2219/00315</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00527">B01J2219/00527</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/0061">B01J2219/0061</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00536">B01J2219/00536</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00659">B01J2219/00659</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00317">B01J2219/00317</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C40B40/10">C40B40/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00675">B01J2219/00675</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00533">B01J2219/00533</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B82Y30/00">B82Y30/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00725">B01J2219/00725</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00576">B01J2219/00576</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00605">B01J2219/00605</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00387">B01J2219/00387</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00637">B01J2219/00637</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=5jhrBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J19/0046">B01J19/0046</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">B82Y30/00</span>, <span class="nested-value">B01J19/00C</span>, <span class="nested-value">C40B50/14</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Nov 19, 2013</td><td class="patent-data-table-td ">FPB1</td><td class="patent-data-table-td ">Expired due to reexamination which canceled all claims</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 14, 2013</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 14, 2013</td><td class="patent-data-table-td ">SULP</td><td class="patent-data-table-td ">Surcharge for late payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">7</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 15, 2012</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 21, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENTEL BIOSCIENCES, INC., WISCONSIN</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20110126</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:TBFREE, LLC;REEL/FRAME:028418/0794</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 5, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100310</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">THE ADOLPH LAW GROUP, PLLC, WASHINGTON</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:PRITEST, INC.;REEL/FRAME:028324/0194</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 28, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">TBFREE, LLC, WASHINGTON</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:PRITEST, INC.;REEL/FRAME:025543/0173</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20101130</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 9, 2010</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100826</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 19, 2008</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 16, 2003</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">PRITEST, INC., WASHINGTON</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:GREEN, LAWRENCE R.;REEL/FRAME:014171/0890</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20030601</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">PRITEST, INC. 2897 152ND AVENUE NEREDMOND, WASHING</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:GREEN, LAWRENCE R. /AR;REEL/FRAME:014171/0890</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_4ff636b3d23669b7103f3b3a3a18b4cd.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0RTGNWpCij3xiVrAjx-KCm52fduw\u0026id=5jhrBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U2V7-5OHthmok4IGEFdq47_2ZNBAA\u0026id=5jhrBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U1gET9UrXyhRloZGeMHSU0AYQtVYQ","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Translucent_solid_matrix_assay_device_fo.pdf?id=5jhrBAABERAJ\u0026output=pdf\u0026sig=ACfU3U2rKvlSTTVkOBMgpafTHQrhNV-2Jg"},"sample_url":"http://www.google.com/patents/reader?id=5jhrBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>