<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7037687 - Sequencing DNA using polymerase, fluorescence, chemiluminescence, thermopile ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Sequencing DNA using polymerase, fluorescence, chemiluminescence, thermopile, thermistor and refractive index measurements; microcalorimetric detection"><meta name="DC.contributor" content="Peter Williams" scheme="inventor"><meta name="DC.contributor" content="Mark A. Hayes" scheme="inventor"><meta name="DC.contributor" content="Seth D. Rose" scheme="inventor"><meta name="DC.contributor" content="Linda B. Bloom" scheme="inventor"><meta name="DC.contributor" content="Linda J. Reha-Krantz" scheme="inventor"><meta name="DC.contributor" content="Vincent B. Pizziconi" scheme="inventor"><meta name="DC.contributor" content="Arizona Board Of Regents" scheme="assignee"><meta name="DC.contributor" content="University Of Alberta" scheme="assignee"><meta name="DC.date" content="2002-8-28" scheme="dateSubmitted"><meta name="DC.description" content="The present invention relates to a novel method for analyzing nucleic acid sequences based on real-time detection of DNA polymerase-catalyzed incorporation of each of the four nucleotide bases, supplied individually and serially in a microfluidic system, to a reaction cell containing a template system comprising a DNA fragment of unknown sequence and an oligonucleotide primer. Incorporation of a nucleotide base into the template system can be detected by any of a variety of methods including but not limited to fluorescence and chemiluminescence detection. Alternatively, microcalorimetic detection of the heat generated by the incorporation of a nucleotide into the extending template system using thermopile, thermistor and refractive index measurements can be used to detect extension reactions."><meta name="DC.date" content="2006-5-2" scheme="issued"><meta name="DC.relation" content="EP:0223618:A2" scheme="references"><meta name="DC.relation" content="US:5302509" scheme="references"><meta name="DC.relation" content="US:5436149" scheme="references"><meta name="DC.relation" content="US:5547859" scheme="references"><meta name="DC.relation" content="US:5945284" scheme="references"><meta name="DC.relation" content="US:6028190" scheme="references"><meta name="DC.relation" content="US:6087095" scheme="references"><meta name="DC.relation" content="WO:1989009283:A1" scheme="references"><meta name="DC.relation" content="WO:1990013666:A1" scheme="references"><meta name="DC.relation" content="WO:1991006678:A1" scheme="references"><meta name="DC.relation" content="WO:1996027025:A1" scheme="references"><meta name="citation_patent_number" content="US:7037687"><meta name="citation_patent_application_number" content="US:10/229,997"><link rel="canonical" href="http://www.google.com/patents/US7037687"/><meta property="og:url" content="http://www.google.com/patents/US7037687"/><meta name="title" content="Patent US7037687 - Sequencing DNA using polymerase, fluorescence, chemiluminescence, thermopile, thermistor and refractive index measurements; microcalorimetric detection"/><meta name="description" content="The present invention relates to a novel method for analyzing nucleic acid sequences based on real-time detection of DNA polymerase-catalyzed incorporation of each of the four nucleotide bases, supplied individually and serially in a microfluidic system, to a reaction cell containing a template system comprising a DNA fragment of unknown sequence and an oligonucleotide primer. Incorporation of a nucleotide base into the template system can be detected by any of a variety of methods including but not limited to fluorescence and chemiluminescence detection. Alternatively, microcalorimetic detection of the heat generated by the incorporation of a nucleotide into the extending template system using thermopile, thermistor and refractive index measurements can be used to detect extension reactions."/><meta property="og:title" content="Patent US7037687 - Sequencing DNA using polymerase, fluorescence, chemiluminescence, thermopile, thermistor and refractive index measurements; microcalorimetric detection"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("ckzsU6PnNZD_yQTaxIKIDw"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("CAN"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("ckzsU6PnNZD_yQTaxIKIDw"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("CAN"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7037687?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7037687"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=gCVzBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7037687&amp;usg=AFQjCNHrFHJGIaAviB5GjrWCuM5vnfM09g" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7037687.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7037687.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20030138809"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7037687"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7037687" style="display:none"><span itemprop="description">The present invention relates to a novel method for analyzing nucleic acid sequences based on real-time detection of DNA polymerase-catalyzed incorporation of each of the four nucleotide bases, supplied individually and serially in a microfluidic system, to a reaction cell containing a template system...</span><span itemprop="url">http://www.google.com/patents/US7037687?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7037687 - Sequencing DNA using polymerase, fluorescence, chemiluminescence, thermopile, thermistor and refractive index measurements; microcalorimetric detection</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7037687 - Sequencing DNA using polymerase, fluorescence, chemiluminescence, thermopile, thermistor and refractive index measurements; microcalorimetric detection" title="Patent US7037687 - Sequencing DNA using polymerase, fluorescence, chemiluminescence, thermopile, thermistor and refractive index measurements; microcalorimetric detection"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7037687 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 10/229,997</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">May 2, 2006</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Aug 28, 2002</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">May 1, 1998</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2330673A1">CA2330673A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2330673C">CA2330673C</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1082458A1">EP1082458A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030138809">US20030138809</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1999057321A1">WO1999057321A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1999057321A8">WO1999057321A8</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1999057321A9">WO1999057321A9</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">10229997, </span><span class="patent-bibdata-value">229997, </span><span class="patent-bibdata-value">US 7037687 B2, </span><span class="patent-bibdata-value">US 7037687B2, </span><span class="patent-bibdata-value">US-B2-7037687, </span><span class="patent-bibdata-value">US7037687 B2, </span><span class="patent-bibdata-value">US7037687B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Peter+Williams%22">Peter Williams</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Mark+A.+Hayes%22">Mark A. Hayes</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Seth+D.+Rose%22">Seth D. Rose</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Linda+B.+Bloom%22">Linda B. Bloom</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Linda+J.+Reha-Krantz%22">Linda J. Reha-Krantz</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Vincent+B.+Pizziconi%22">Vincent B. Pizziconi</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Arizona+Board+Of+Regents%22">Arizona Board Of Regents</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22University+Of+Alberta%22">University Of Alberta</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7037687.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7037687.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7037687.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (11),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (85),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (17),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (9)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7037687&usg=AFQjCNHRLVfLoyYEkXqwytYmoWHT3x75hw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7037687&usg=AFQjCNHGrH_TftAt6sV_3ijtFdqTp2a_Jg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7037687B2%26KC%3DB2%26FT%3DD&usg=AFQjCNEKzpSoCm8-V59sOp8GCYv5Jp-LXQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129829869" lang="EN" load-source="ifi">Sequencing DNA using polymerase, fluorescence, chemiluminescence, thermopile, thermistor and refractive index measurements; microcalorimetric detection</invention-title></span><br><span class="patent-number">US 7037687 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50970769" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">The present invention relates to a novel method for analyzing nucleic acid sequences based on real-time detection of DNA polymerase-catalyzed incorporation of each of the four nucleotide bases, supplied individually and serially in a microfluidic system, to a reaction cell containing a template system comprising a DNA fragment of unknown sequence and an oligonucleotide primer. Incorporation of a nucleotide base into the template system can be detected by any of a variety of methods including but not limited to fluorescence and chemiluminescence detection. Alternatively, microcalorimetic detection of the heat generated by the incorporation of a nucleotide into the extending template system using thermopile, thermistor and refractive index measurements can be used to detect extension reactions.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(9)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7037687B2/US07037687-20060502-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7037687B2/US07037687-20060502-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7037687B2/US07037687-20060502-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7037687B2/US07037687-20060502-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7037687B2/US07037687-20060502-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7037687B2/US07037687-20060502-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7037687B2/US07037687-20060502-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7037687B2/US07037687-20060502-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7037687B2/US07037687-20060502-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7037687B2/US07037687-20060502-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7037687B2/US07037687-20060502-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7037687B2/US07037687-20060502-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7037687B2/US07037687-20060502-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7037687B2/US07037687-20060502-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7037687B2/US07037687-20060502-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7037687B2/US07037687-20060502-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7037687B2/US07037687-20060502-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7037687B2/US07037687-20060502-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(5)</span></span></div><div class="patent-text"><div mxw-id="PCLM9006665" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A method of nucleic acid sequencing, the method comprising the steps of:
<div class="claim-text">(a) providing a nucleic acid template/primer system comprising individual template/primer duplexes immobilized on a surface;</div>
<div class="claim-text">(b) exposing said individual template/primer duplexes to a polymerase and one or more type of nucleotide comprising an optically-detectable label, wherein said optically-detectable label is not attached to the 3′ position of the sugar moiety of said nucleotide;</div>
<div class="claim-text">(c) removing nucleotide that is not incorporated into a primer;</div>
<div class="claim-text">(d) detecting incorporated nucleotide by the presence of said label;</div>
<div class="claim-text">(e) removing said label from said incorporated nucleotide;</div>
<div class="claim-text">(f) repeating steps (b) through (e) at least once with a different type of nucleotide comprising an optically-detectable label for incorporation into said primer; and</div>
<div class="claim-text">(g) determining a nucleic acid sequence based upon said incorporated nucleotides.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said label is a fluorescent label.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The method of <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein said fluorescent label is selected from the group consisting of fluorescein, rhodamine and BODIPY.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein said detecting step comprises sensing said label.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said detecting step comprises resolving an image of said duplex.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES16029209" lang="EN" load-source="patent-office" class="description">
    <heading>REFERENCE TO RELATED APPLICATIONS</heading> <p num="p-0002">This application is a continuation of U.S. application Ser. No. 09/673,544, filed on Feb. 26, 2001, now abandoned, which is a national stage filing of International Application No. PCT/US99/09616, filed on Apr. 30, 1999, which claimed the benefit of U.S. Provisional Application No. 60/083,840, filed on May 1, 1998, each of the foregoing of which is incorporated by reference herein.</p>
    <p num="p-0003">The present invention relates to a novel method for analyzing nucleic acid sequences based on real-time detection of DNA polymerase-catalyzed incorporation of each of the four deoxynucleoside monophosphates, supplied individually and serially as deoxynucleoside triphosphates in a microfluidic system, to a template system comprising a DNA fragment of unknown sequence and an oligonucleotide primer. Incorporation of a deoxynucleoside monophosphate (dNMP) into the primer can be detected by any of a variety of methods including but not limited to fluorescence and chemiluminescence detection. Alternatively, microcalorimetic detection of the heat generated by the incorporation of a dNMP into the extending primer using thermopile, thermistor and refractive index measurements can be used to detect extension reactions.</p>
    <p num="p-0004">The present invention provides a method for sequencing DNA that avoids electrophoretic separation of DNA fragments thus eliminating the problems associated with anomalous migration of DNA due to repeated base sequences or other self-complementary sequences which can cause single-stranded DNA to self-hybridize into hairpin loops, and also avoids current limitations on the size of fragments that can be read. The method of the invention can be utilized to determine the nucleotide sequence of genomic or cDNA fragments, or alternatively, as a diagnostic tool for sequencing patient derived DNA samples.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p num="p-0005">Currently, two approaches are utilized for DNA sequence determination: the dideoxy chain termination method of Sanger (1977, Proc. Natl. Acad. Sci. 74:5463–5674) and the chemical degradation method of Maxam (1977, Proc. Natl. Acad. Sci. 74:560–564). The Sanger dideoxy chain termination method is the most widely used method and is the method upon which automated DNA sequencing machines rely. In the chain termination method, DNA polymerase enzyme is added to four separate reaction systems to make multiple copies of a template DNA strand in which the growth process has been arrested at each occurrence of an A, in one set of reactions, and a G, C, or T, respectively, in the other sets of reactions, by incorporating in each reaction system one nucleotide type lacking the 3′-OH on the deoxyribose at which chain extension occurs. This procedure produces a series of DNA fragments of different lengths, and it is the length of the extended DNA fragment that signals the position along the template strand at which each of four bases occur. To determine the nucleotide sequence, the DNA fragments are separated by high resolution gel electrophoresis and the order of the four bases is read from the gel.</p>
    <p num="p-0006">A major research goal is to derive the DNA sequence of the entire human genome. To meet this goal the need has developed for new genomic sequencing technology that can dispense with the difficulties of gel electrophoresis, lower the costs of performing sequencing reactions, including reagent costs, increase the speed and accuracy of sequencing, and increase the length of sequence that can be read in a single step. Potential improvements in sequencing speed may be provided by a commercialized capillary gel electrophoresis technique such as that described in Marshall and Pennisis (1998, Science 280:994–995). However, a major problem common to all gel electrophoresis approaches is the occurrence of DNA sequence compressions, usually arising from secondary structures in the DNA fragment, which result in anomalous migration of certain DNA fragments through the gel.</p>
    <p num="p-0007">As genomic information accumulates and the relationships between gene mutations and specific diseases are identified, there will be a growing need for diagnostic methods for identification of mutations. In contrast to the large scale methods needed for sequencing large segments of the human genome, what is needed for diagnostic methods are repetitive, low-cost, highly accurate techniques for resequencing of certain small isolated regions of the genome. In such instances, methods of sequencing based on gel electrophoresis readout become far too slow and expensive.</p>
    <p num="p-0008">When considering novel DNA sequencing techniques, the possibility of reading the sequence directly, much as the cell does, rather than indirectly as in the Sanger dideoxynucleotide approach, is a preferred goal. This was the goal of early unsuccessful attempts to determine the shapes of the individual nucleotide bases with scanning probe microscopes.</p>
    <p num="p-0009">Additionally, another approach for reading a nucleotide sequence directly is to treat the DNA with an exonuclease coupled with a detection scheme for identifying each nucleotide sequentially released as described in Goodwin et al., (1995, Experimental Techniques of Physics 41:279–294). However, researchers using this technology are confronted with the enormous problem of detecting and identifying single nucleotide molecules as they are digested from a single DNA strand. Simultaneous exonuclease digestion of multiple DNA strands to yield larger signals is not feasible because the enzymes rapidly get out of phase, so that nucleotides from different positions on the different strands are released together, and the sequences become unreadable. It would be highly beneficial if some means of external regulation of the exonuclease could be found so that multiple enzyme molecules could be compelled to operate in phase. However, external regulation of an enzyme that remains docked to its polymeric substrate is exceptionally difficult, if not impossible, because after each digestion the next substrate segment is immediately present at the active site. Thus, any controlling signal must be present at the active site at the start of each reaction.</p>
    <p num="p-0010">A variety of methods may be used to detect the polymerase-catalyzed incorporation of deoxynucleoside monophosphates (dNMPs) into a primer at each template site. For example, the pyrophosphate released whenever DNA polymerase adds one of the four dNTPs onto a primer 3′ end may be detected using a chemiluminescent based detection of the pyrophosphate as described in Hyman E. D. (1988, Analytical Biochemistry 174:423–436) and U.S. Pat. No. 4,971,903. This approach has been utilized most recently in a sequencing approach referred to as “sequencing by incorporation” as described in Ronaghi (1996, Analytical Biochem. 242:84) and Ronaghi (1998, Science 281:363–365). However, there exist two key problems associated with this approach, destruction of unincorporated nucleotides and detection of pyrophosphate. The solution to the first problem is to destroy the added, unincorporated nucleotides using a dNTP-digesting enzyme such as apyrase. The solution to the second is the detection of the pyrophosphate using ATP sulfurylase to reconvert the pyrophosphate to ATP which can be detected by a luciferase chemiluminescent reaction as described in U.S. Pat. No. 4,971,903 and Ronaghi (1998, Science 281:363–365). Deoxyadenosine α-thiotriphosphate is used instead of dATP to minimize direct interaction of injected dATP with the luciferase.</p>
    <p num="p-0011">Unfortunately, the requirement for multiple enzyme reactions to be completed in each cycle imposes restrictions on the speed of this approach while the read length is limited by the impossibility of completely destroying unincorporated, non-complementary, nucleotides. If some residual amount of one nucleotide remains in the reaction system at the time when a fresh aliquot of a different nucleotide is added for the next extension reaction, there exists a possibility that some fraction of the primer strands will be extended by two or more nucleotides, the added nucleotide type and the residual impurity type, if these match the template sequence, and so this fraction of the primer strands will then be out of phase with the remainder. This out of phase component produces an erroneous incorporation signal which grows larger with each cycle and ultimately makes the sequence unreadable.</p>
    <p num="p-0012">A different direct sequencing approach uses dNTPs tagged at the 3′ OH position with four different colored fluorescent tags, one for each of the four nucleotides is described in Metzger, M. L., et al. (1994, Nucleic Acids Research 22:4259–4267). In this approach, the primer/template duplex is contacted with all four dNTPs simultaneously. Incorporation of a 3′ tagged NMP blocks further chain extension. The excess and unreacted dNTPs are flushed away and the incorporated nucleotide is identified by the color of the incorporated fluorescent tag. The fluorescent tag must then be removed in order for a subsequent incorporation reaction to occur. Similar to the pyrophosphate detection method, incomplete removal of a blocking fluorescent tag leaves some primer strands unextended on the next reaction cycle, and if these are subsequently unblocked in a later cycle, once again an out-of-phase signal is produced which grows larger with each cycle and ultimately limits the read length. To date, this method has so far been demonstrated to work for only a single base extension. Thus, this method is slow and is likely to be restricted to very short read lengths due to the fact that 99% efficiency in removal of the tag is required to read beyond 50 base pairs. Incomplete removal of the label results in out of phase extended DNA strands.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0013">Accordingly, it is an object of the present invention to provide a novel method for determining the nucleotide sequence of a DNA fragment which eliminates the need for electrophoretic separation of DNA fragments. The inventive method, referred to herein as “reactive sequencing”, is based on detection of DNA polymerase catalyzed incorporation of each of the four nucleotide types, when deoxynucleoside triphosphates (dNTP's) are supplied individually and serially to a DNA primer/template system. The DNA primer/template system comprises a single stranded DNA fragment of unknown sequence, an oligonucleotide primer that forms a matched duplex with a short region of the single stranded DNA, and a DNA polymerase enzyme. The enzyme may either be already present in the template system, or may be supplied together with the dNTP solution.</p>
    <p num="p-0014">Typically a single deoxynucleoside triphosphate (dNTP) is added to the DNA primer template system and allowed to react. As used herein deoxyribonucleotide means and includes, in addition to dGTP, dCTP, dATP, dTTP, chemically modified versions of these deoxyribonucleotides or analogs thereof. Such chemically modified deoxyribonucleotides include but are not limited to those deoxyribonucleotides tagged with a fluorescent or chemiluminescent moiety. Analogs of deoxyribonucleotides that may be used include but are not limited to 7-deazapurine. An extension reaction will occur only when the incoming dNTP base is complementary to the next unpaired base of the DNA template beyond the 3′ end of the primer. While the reaction is occurring, or after a delay of sufficient duration to allow a reaction to occur, the system is tested to determine whether an additional nucleotide derived from the added dNTP has been incorporated into the DNA primer/template system. A correlation between the dNTP added to the reaction cell and detection of an incorporation signal identifies the nucleotide incorporated into the primer/template. The amplitude of the incorporation signal identifies the number of nucleotides incorporated, and thereby quantifies single base repeat lengths where these occur. By repeating this process with each of the four nucleotides individually, the sequence of the template can be directly read in the 5′ to 3′ direction one nucleotide at a time.</p>
    <p num="p-0015">Detection of the polymerase mediated extension reaction and quantification of the extent of reaction can occur by a variety of different techniques, including but not limited to, microcalorimetic detection of the heat generated by the incorporation of a nucleotide into the extending duplex. Optical detection of an extension reaction by fluorescence or chemiluminescence may also be used to detect incorporation of nucleotides tagged with fluorescent or chemiluminescent entities into the extending duplex. Where the incorporated nucleotide is tagged with a fluorophore, excess unincorporated nucleotide is removed, and the template system is illuminated to stimulate fluorescence from the incorporated nucleotide. The fluorescent tag must then be cleaved and removed from the DNA template system before a subsequent incorporation cycle begins. A similar process is followed for chemiluminescent tags, with the chemiluminescent reaction being stimulated by introducing an appropriate reagent into the system, again after excess unreacted tagged dNTP has been removed; however, chemiluminescent tags are typically destroyed in the process of readout and so a separate cleavage and removal step following detection may not be required. For either type of tag, fluorescent or chemiluminescent, the tag may also be cleaved after incorporation and transported to a separate detection chamber for fluorescent or chemiluminescent detection. In this way, fluorescent quenching by adjacent fluorophore tags incorporated in a single base repeat sequence may be avoided. In addition, this may protect the DNA template system from possible radiation damage in the case of fluorescent detection or from possible chemical damage in the case of chemiluminescent detection.</p>
    <p num="p-0016">The present invention further provides a reactive sequencing method that utilizes a two cycle system. An exonuclease-deficient polymerase is used in the first cycle and a mixture of exonuclease-deficient and exonuclease-proficient enzymes are used in the second cycle. In the first cycle, the template-primer system together with an exonuclease-deficient polymerase will be presented sequentially with each of the four possible nucleotides. In the second cycle, after identification of the correct nucleotide, a mixture of exonuclease proficient and deficient polymerases, or a polymerase containing both types of activity will be added in a second cycle together with the correct dNTP identified in the first cycle to complete and proofread the primer extension. In this way, an exonuclease-proficient polymerase is only present in the reaction cell when the correct dNTP is present, so that exonucleolytic degradation of correctly extended strands does not occur, while degradation and correct re-extension of previously incorrectly extended strands does occur, thus achieving extremely accurate strand extension.</p>
    <p num="p-0017">The present invention further provides an apparatus for DNA sequencing comprising: (a) at least one chamber including a DNA primer/template system which produces a detectable signal when a DNA polymerase enzyme incorporates a deoxyribonucleotide monophosphate onto the 3′ end of the primer strand; (b) means for introducing into, and evacuating from, the reaction chamber at least one selected from the group consisting of buffers, electrolytes, DNA template, DNA primer, deoxyribonucleotides, and polymerase enzymes; (c) means for amplifying said signal; and (d) means for converting said signal into an electrical signal.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p num="p-0018">Further objects and advantages of the invention will be apparent from a reading of the following description in conjunction with the accompanying drawings, in which:</p>
      <p num="p-0019"> <figref idrefs="DRAWINGS">FIG. 1</figref> is a schematic diagram illustrating a reactive sequencing device containing a thin film bismuth antimony thermopile in accordance with the invention;</p>
      <p num="p-0020"> <figref idrefs="DRAWINGS">FIG. 2</figref> is a schematic diagram of a reactive sequencing device containing a thermistor in accordance with the invention;</p>
      <p num="p-0021"> <figref idrefs="DRAWINGS">FIG. 3</figref> is a schematic diagram illustrating a representative embodiment of microcalorimetry detection of a DNA polymerase reaction in accordance with the invention;</p>
      <p num="p-0022"> <figref idrefs="DRAWINGS">FIG. 4</figref> is an electrophoretic gel showing a time course for primer extension assays catalyzed by T4 DNA polymerase mutants;</p>
      <p num="p-0023"> <figref idrefs="DRAWINGS">FIG. 5</figref> is a schematic diagram illustrating a nucleotide attached to a fluorophore by a benzoin ester which is a photocleavable linker for use in the invention;</p>
      <p num="p-0024"> <figref idrefs="DRAWINGS">FIG. 6</figref> is a schematic illustration of a nucleotide attached to a chemiluminescent tag for use in the invention;</p>
      <p num="p-0025"> <figref idrefs="DRAWINGS">FIG. 7</figref> is a schematic diagram of a nucleotide attached to a chemiluminescent tag by a cleavable linkage;</p>
      <p num="p-0026"> <figref idrefs="DRAWINGS">FIGS. 8(</figref> <i>a</i>) and <b>8</b>(<i>b</i>) are schematic diagrams of a mechanical fluorescent sequencing method in accordance with the invention in which a DNA template and primer are absorbed on beads captured behind a porous frit; and</p>
      <p num="p-0027"> <figref idrefs="DRAWINGS">FIG. 9</figref> is a schematic diagram of a sequencing method in accordance with the invention utilizing a two cycle system.</p>
    </description-of-drawings> <heading>DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS</heading> <p num="p-0028">The present invention provides a method for determining the nucleic acid sequence of a DNA molecule based on detection of successive single nucleotide DNA polymerase mediated extension reactions. As described in detail below, in one embodiment, a DNA primer/template system comprising a polynucleotide primer complementary to and bound to a region of the DNA to be sequenced is constrained within a reaction cell into which buffer solutions containing various reagents necessary for a DNA polymerase reaction to occur are added. Into the reaction cell, a single type of deoxynucleoside triphosphate (dNTP) is added. Depending on the identity of the next complementary site in the DNA primer/template system, an extension reaction will occur only when the appropriate nucleotide is present in the reaction cell. A correlation between the nucleotide present in the reaction cell and detection of an incorporation signal identifies the next nucleotide of the template. Following each extension reaction, the reaction cell is flushed with dNTP-free buffer, retaining the DNA primer/template system, and the cycle is repeated until the entire nucleotide sequence is identified.</p>
    <p num="p-0029">The present invention is based on the existence of a control signal within the active site of DNA polymerases which distinguish, with high fidelity, complementary and non-complementary fits of incoming deoxynucleotide triphosphates to the base on the template strand at the primer extension site, i.e., to read the sequence, and to incorporate at that site only the one type of deoxynucleotide that is complementary. That is, if the available nucleotide type is not complementary to the next template site, the polymerase is inactive, thus, the template sequence is the DNA polymerase control signal. Therefore, by contacting a DNA polymerase system with a single nucleotide type rather than all four, the next base in the sequence can be identified by detecting whether of not a reaction occurs. Further, single base repeat lengths can be quantified by quantifying the extent of reaction.</p>
    <p num="p-0030">As a first step in the practice of the inventive method, single-stranded template DNA to be sequenced is prepared using any of a variety of different methods known in the art. Two types of DNA can be used as templates in the sequencing reactions. Pure single-stranded DNA such as that obtained from recombinant bacteriophage can be used. The use of bacteriophage provides a method for producing large quantities of pure single stranded template. Alternatively, single-stranded DNA may be derived from double-stranded DNA that has been denatured by heat or alkaline conditions, as described in Chen and Subrung, (1985, DNA 4:165); Huttoi and Skaki (1986, Anal. Biochem. 152:232); and Mierendorf and Pfeffer, (1987, Methods Enzymol. 152:556), may be used. Such double stranded DNA includes, for example, DNA samples derived from patients to be used in diagnostic sequencing reactions.</p>
    <p num="p-0031">The template DNA can be prepared by various techniques well known to those of skill in the art. For example, template DNA can be prepared as vector inserts using any conventional cloning methods, including those used frequently for sequencing. Such methods can be found in Sambrook et al., Molecular Cloning: A Laboratory Manual, Second Edition (Cold Spring Harbor Laboratories, New York, 1989). In a preferred embodiment of the invention, polymerase chain reactions (PCR) may be used to amplify fragments of DNA to be used as template DNA as described in Innis et al., ed. PCR Protocols (Academic Press, New York, 1990).</p>
    <p num="p-0032">The amount of DNA template needed for accurate detection of the polymerase reaction will depend on the detection technique used. For example, for optical detection, e.g., fluorescence or chemiluminescence detection, relatively small quantities of DNA in the femtomole range are needed. For thermal detection quantities approaching one picomole may be required to detect the change in temperature resulting from a DNA polymerase mediated extension reaction.</p>
    <p num="p-0033">In enzymatic sequencing reactions, the priming of DNA synthesis is achieved by the use of an oligonucleotide primer with a base sequence that is complementary to, and therefore capable of binding to, a specific region on the template DNA sequence. In instances where the template DNA is obtained as single stranded DNA from bacteriophage, or as double stranded DNA derived from plasmids, “universal” primers that are complementary to sequences in the vectors, i.e., the bacteriophage, cosmid and plasmid vectors, and that flank the template DNA, can be used.</p>
    <p num="p-0034">Primer oligonucleotides are chosen to form highly stable duplexes that bind to the template DNA sequences and remain intact during any washing steps during the extension cycles. Preferably, the length of the primer oligonucleotide is from 18–30 nucleotides and contains a balanced base composition. The structure of the primer should also be analyzed to confirm that it does not contain regions of dyad symmetry which can fold and self anneal to form secondary structures thereby rendering the primers inefficient. Conditions for selecting appropriate hybridization conditions for binding of the oligonucleotide primers in the template systems will depend on the primer sequence and are well known to those of skill in the art.</p>
    <p num="p-0035">In utilizing the reactive sequencing method of the invention, a variety of different DNA polymerases may be used to incorporate dNTPs onto the 3′ end of the primer which is hybridized to the template DNA molecule. Such DNA polymerases include but are not limited to Taq polymerase, T7 or T4 polymerase, and Klenow polymerase. In a preferred embodiment of the invention, described in detail below, DNA polymerases lacking 5′-3′-exonuclease proofreading activity are used in the sequencing reactions. For the most rapid reaction kinetics, the amount of polymerase is sufficient to ensure that each DNA molecule carries a non-covalently attached polymerase molecule during reaction. For a typical equilibrium constant of ˜50 nM for the dissociation equilibrium:
<br>DNA−Pol⇄DNA+Pol <i>K</i>˜50 nM<br>
the desired condition is: [Pol]≧50 nM+[DNA].
</p>
    <p num="p-0036">In addition, reverse transcriptase which catalyzes the synthesis of single stranded DNA from an RNA template may be utilized in the reactive sequencing method of the invention to sequence messenger RNA (mRNA). Such a method comprises sequentially contacting an RNA template annealed to a primer (RNA primer/template) with dNTPs in the presence of reverse transcriptase enzyme to determine the sequence of the RNA. Because mRNA is produced by RNA polymerase-catalyzed synthesis from a DNA template, and thus contains the sequence information of the DNA template strand, sequencing the mRNA yields the sequence of the DNA gene from which it was transcribed. Eukaryotic mRNAs have poly(A) tails and therefore the primer for reverse transcription can be an oligo(dT). Typically, it will be most convenient to synthesize the oligo(dT) primer with a terminal biotin or amino group through which the primer can be captured on a substrate and subsequently hybridize to and capture the template mRNA strand.</p>
    <p num="p-0037">The extension reactions are carried out in buffer solutions which contain the appropriate concentrations of salts, dNTPs and DNA polymerase required for the DNA polymerase mediated extension to proceed. For guidance regarding such conditions see, for example, Sambrook et al., (1989, Molecular Cloning, A Laboratory Manual. Cold Spring Harbor Press, N.Y.); and Ausubel et al. (1989, Current Protocols in Molecular Biology, Green Publishing Associates and Wiley Interscience, N.Y.).</p>
    <p num="p-0038">Typically, buffer containing one of the four dNTPs is added into a reaction cell. Depending on the identity of the nucleoside base at the next unpaired template site in the primer/template system, a reaction will occur when the reaction cell contains the appropriate dNTP. When the reaction cell contains any one of the other three incorrect dNTPs, no reaction will take place.</p>
    <p num="p-0039">The reaction cell is then flushed with dNTP free buffer and the cycle is repeated until a complete DNA sequence is identified. Detection of a DNA polymerase mediated extension can be made using any of the detection methods described in detail below including optical and thermal detection of an extension reaction.</p>
    <p num="p-0040">In a preferred embodiment of the invention, the primer/template system comprises the template DNA tethered to a solid phase support to permit the sequential addition of sequencing reaction reagents without complicated and time consuming purification steps following each extension reaction. Preferably, the template DNA is covalently attached to a solid phase support, such as the surface of a reaction flow cell, a polymeric microsphere, filter material, or the like, which permits the sequential application of sequencing reaction reagents, i.e., buffers, dNTPs and DNA polymerase, without complicated and time consuming purification steps following each extension reaction. Alternatively, for applications that require sequencing of many samples containing the same vector template or same gene, for example, in diagnostic applications, a universal primer may be tethered to a support, and the template DNA allowed to hybridize to the immobilized primer.</p>
    <p num="p-0041">The DNA may be modified to facilitate covalent or non-covalent tethering of the DNA to a solid phase support. For example, when PCR is used to amplify DNA fragments, the 5′ ends of one set of PCR primer oligonucleotides strands may be modified to carry a linker moiety for tethering one of the two complementary types of DNA strands produced to a solid phase support. Such linker moieties include, for example, biotin. When using biotin, the biotinylated DNA fragments may be bound non-covalently to streptavidin covalently attached to the solid phase support. Alternatively, an amino group (—NH<sub>2</sub>) may be chemically incorporated into one of the PCR primer strands and used to covalently link the DNA template to a solid phase support using standard chemistry, such as reactions with N-hydroxysuccinimide activated agarose surfaces.</p>
    <p num="p-0042">In another embodiment, the 5′ ends of the sequencing oligonucleotide primer may be modified with biotin, for non-covalent capture to a streptavidin-treated support, or with an amino group for chemical linkage to a solid support; the template strands are then captured by the non-covalent binding attraction between the immobilized primer base sequence and the complementary sequence on the template strands. Methods for immobilizing DNA on a solid phase support are well known to those of skill in the art and will vary depending on the solid phase support chosen.</p>
    <p num="p-0043">In the reactive sequencing method of the present invention, DNA polymerase is presented sequentially with each of the 4 dNTPs. In the majority of the reaction cycles, only incorrect dNTPs will be present, thereby increasing the likelihood of misincorporation of incorrect nucleotides into the extending DNA primer/template system.</p>
    <p num="p-0044">Accordingly, the present invention further provides methods for optimizing the reactive sequencing reaction to achieve rapid and complete incorporation of the correct nucleotide into the DNA primer/template system, while limiting the misincorporation of incorrect nucleotides. For example, dNTP concentrations may be lowered to reduce misincorporation of incorrect nucleotides into the DNA primer. K<sub>m </sub>values for incorrect dNTPs can be as much as 1000-fold higher than for correct nucleotides, indicating that a reduction in dNTP concentrations can reduce the rate of misincorporation of nucleotides. Thus, in a preferred embodiment of the invention the concentration of dNTPs in the sequencing reactions are approximately 5–20 μM. At this concentration, incorporation rates are as close to the maximum rate of 400 nucleotides/s for T4 DNA polymerase as possible.</p>
    <p num="p-0045">In addition, relatively short reaction times can be used to reduce the probability of misincorporation. For an incorporation rate approaching the maximum rate of ˜400 nucleotides/s, a reaction time of approximately 25 milliseconds (ms) will be sufficient to ensure extension of 99.99% of primer strands.</p>
    <p num="p-0046">In a specific embodiment of the invention, DNA polymerases lacking 3′ to 5′ exonuclease activity may be used for reactive sequencing to limit exonucleolytic degradation of primers that would occur in the absence of correct dNTPs. In the presence of all four dNTPs, misincorporation frequencies by DNA polymerases possessing exonucleolytic proofreading activity are as low as one error in 10<sup>6 </sup>to 10<sup>8 </sup>nucleotides incorporated as discussed in Echols and Goodman (1991, Annu. Rev. Biochem. 60;477–511); and Goodman et al. (1993, Crit. Rev. Biochem. Molec. Biol. 28:83–126); and Loeb and Kunkel (1982, Annu. Rev. Biochem. 52:429–457). In the absence of proofreading, DNA polymerase error rates are typically on the order of 1 in 10<sup>4 </sup>to 1 in 10<sup>6</sup>. Although exonuclease activity increases the fidelity of a DNA polymerase, the use of DNA polymerases having proofreading activity can pose technical difficulties for the reactive sequencing method of the present invention. Not only will the exonuclease remove any misincorporated nucleotides, but also, in the absence of a correct dNTP complementary to the next template base, the exonuclease will remove correctly-paired nucleotides successively until a point on the template sequence is reached where the base is complementary to the dNTP in the reaction cell. At this point, an idling reaction is established where the polymerase repeatedly incorporates the correct dNMP and then removes it. Only when a correct dNTP is present will the rate of polymerase activity exceed the exonuclease rate so that an idling reaction is established that maintains the incorporation of that correct nucleotide at the 3′ end of the primer.</p>
    <p num="p-0047">A number of T4 DNA polymerase mutants containing specific amino acid substitutions possess reduced exonuclease activity levels up to 10,000-fold less than the wild-type enzyme. For example, Reha-Krantz and Nonay (1993, J. Biol. Chem. 268:27100–17108) report that when Asp 112 was replaced with Ala and Glu 114 was replaced with Ala (D112A/E114A) in T4 polymerase, these two amino acid substitutions reduced the exonuclease activity on double stranded DNA by a factor of about 300 relative to the wild type enzyme. Such mutants may be advantageously used in the practice of the invention for incorporation of nucleotides into the DNA primer/template system.</p>
    <p num="p-0048">In yet another embodiment of the invention, DNA polymerases which are more accurate than wild type polymerases at incorporating the correct nucleotide into a DNA primer/template may be used. For example, in a (D112A/E114A) mutant T4 polymerase with a third mutation where Ile 417 is replaced by Val (I417V/D112A/E114A), the I417V mutation results in an antimutator phenotype for the polymerase (Reha-Krantz and Nonay, 1994, J. Biol. Chem. 269:5635–5643; Stocki et al., 1995, Mol. Biol. 254:15–28). This antimutator phenotype arises because the polymerase tends to move the primer ends from the polymerase site to the exonuclease site more frequently and thus proof read more frequently than the wild type polymerase, and thus increases the accuracy of synthesis.</p>
    <p num="p-0049">In yet another embodiment of the invention, polymerase mutants that are capable of more efficiently incorporating fluorescent-labeled nucleotides into the template DNA system molecule may be used in the practice of the invention. The efficiency of incorporation of fluorescent-labeled nucleotides may be reduced due to the presence of bulky fluorophore labels that may inhibit dNTP interaction at the active site of the polymerase. Polymerase mutants that may be advantageously used for incorporation of fluorescent-labeled dNTPs into DNA include but are not limited to those described in U.S. application Ser. No. 08/632,742 filed Apr. 16, 1996 which is incorporated by reference herein.</p>
    <p num="p-0050">In a preferred embodiment of the invention, the reactive sequencing method utilizes a two cycle system. An exonuclease-deficient polymerase is used in the first cycle and a mixture of exonuclease-deficient and exonuclease-proficient enzymes are used in the second cycle. In the first cycle, the primer/template system together with an exonuclease-deficient polymerase will be presented sequentially with each of the four possible nucleotides. Reaction time and conditions will be such that a sufficient fraction of primers are extended to allow for detection and quantification of nucleotide incorporation, ˜98%, for accurate quantification of multiple single-base repeats. In the second cycle, after identification of the correct nucleotide, a mixture of exonuclease proficient and deficient polymerases, or a polymerase containing both types of activity will be added in a second cycle together with the correct dNTP identified in the first cycle to complete and proofread the primer extension. In this way, an exonuclease-proficient polymerase is only present in the reaction cell when the correct dNTP is present, so that exonucleolytic degradation of correctly extended strands does not occur, while degradation and correct re-extension of previously incorrectly extended strands does occur, thus achieving extremely accurate strand extension.</p>
    <p num="p-0051">The detection of a DNA polymerase mediated extension reaction can be accomplished in a number of ways. For example, the heat generated by the extension reaction can be measured using a variety of different techniques such as those employing thermopile, thermistor and refractive index measurements.</p>
    <p num="p-0052">In an embodiment of the invention, the heat generated by a DNA polymerase mediated extension reaction can be measured. For example, in a reaction cell volume of 100 micrometers<sup>3 </sup>containing 1 μg of water as the sole thermal mass and 2×10<sup>11 </sup>DNA template molecules (300 fmol) tethered within the cell, the temperature of the water increases by 1×10<sup>3</sup>° C. for a polymerase reaction which extends the primer by a single nucleoside monophosphate. This calculation is based on the experimental determination that a one base pair extension in a DNA chain is an exothermic reaction and the enthalpy change associated with this reaction is 3.5 kcal/mole of base. Thus extension of 300 fmol of primer strands by a single base produces 300 fmol×3.5 kcal/mol or 1×10<sup>−9 </sup>cal of heat. This is sufficient to raise the temperature of 1 μg of water by 1×10<sup>−3</sup>° C. Such a temperature change can be readily detectable using thermistors (sensitivity≦10<sup>−4</sup>° C.); thermopiles (sensitivity≦10<sup>−5</sup>° C.); and refractive index measurements (sensitivity≦10<sup>−6</sup>° C.).</p>
    <p num="p-0053">In a specific embodiment of the invention, thermopiles may used to detect temperature changes. Such thermopiles are known to have a high sensitivity to temperature and can make measurements in the tens of micro-degree range in several second time constants. Thermopiles may be fabricated by constructing serial sets of junctions of two dissimilar metals and physically arranging the junctions so that alternating junctions are separated in space. One set of junctions is maintained at a constant reference temperature, while the alternate set of junctions is located in the region whose temperature is to be sensed. A temperature difference between the two sets of junctions produces a potential difference across the junction set which is proportional to the temperature difference, to the thermoelectric coefficient of the junction and to the number of junctions. For optimum response, bimetallic pairs with a large thermoelectric coefficient are desirable, such as bismuth and antimony. Thermopiles may be fabricated using thin film deposition techniques in which evaporated metal vapor is deposited onto insulating substrates through specially fabricated masks. Thermopiles that may be used in the practice of the invention include thermopiles such as those described in U.S. Pat. No. 4,935,345, which is incorporated by reference herein.</p>
    <p num="p-0054">In a specific embodiment of the invention, miniature thin film thermopiles produced by metal evaporation techniques, such as those described in U.S. Pat. No. 4,935,345 incorporated herein by reference, may be used to detect the enthalpy changes. Such devices have been made by vacuum evaporation through masks of about 10 mm square. Using methods of photolithography, sputter etching and reverse lift-off techniques, devices as small as 2 mm square may be constructed without the aid of modem microlithographic techniques. These devices contain 150 thermoelectric junctions and employ 12 micron line widths and can measure the exothermic heat of reaction of enzyme-catalyzed reactions in flow streams where the enzyme is preferably immobilized on the surface of the thermopile.</p>
    <p num="p-0055">To incorporate thermopile detection technology into a reactive sequencing device, thin-film bismuth-antimony thermopiles <b>2</b>, as shown in <figref idrefs="DRAWINGS">FIG. 1</figref>, may be fabricated by successive electron-beam evaporation of bismuth and antimony metals through two different photolithographically-generated masks in order to produce a zigzag array of alternating thin bismuth and antimony wires which are connected to form two sets of bismuth-antimony thermocouple junctions. Modern microlithographic techniques will allow fabrication of devices at least one order of magnitude smaller than those previously made, i.e., with line widths as small as 1 μm and overall dimensions on the order of 100 μm<sup>2</sup>. One set of junctions <b>4</b> (the sensor junctions) is located within the reaction cell <b>6</b>, i.e., deposited on a wall of the reaction cell, while the second reference set of junctions <b>8</b> is located outside the cell at a reference point whose temperature is kept constant. Any difference in temperature between the sensor junctions and the reference junctions results in an electric potential being generated across the device, which can be measured by a high-resolution digital voltmeter <b>10</b> connected to measurement points <b>12</b> at either end of the device. It is not necessary that the temperature of the reaction cell and the reference junctions be the same in the absence of a polymerase reaction event, only that a change in the temperature of the sensor junctions due to a polymerase reaction event be detectable as a change in the voltage generated across the thermopile.</p>
    <p num="p-0056">In addition to thermopiles, as shown in <figref idrefs="DRAWINGS">FIG. 2</figref>, a thermistor <b>14</b> may also be used to detect temperature changes in the reaction cell <b>6</b> resulting from DNA polymerase mediated incorporation of dNMPs into the DNA primer strand. Thermistors are semiconductors composed of a sintered mixture of metallic oxides such as manganese, nickel, and cobalt oxides. This material has a large temperature coefficient of resistance, typically ˜4% per ° C., and so can sense extremely small temperature changes when the resistance is monitored with a stable, high-resolution resistance-measuring device such as a digital voltmeter, e.g., Keithley Instruments Model 2002. A thermistor <b>14</b>, such as that depicted in <figref idrefs="DRAWINGS">FIG. 2</figref>, may be fabricated in the reactive sequencing reaction cell by sputter depositing a thin film of the active thermistor material onto the surface of the reaction cell from a single target consisting of hot pressed nickel, cobalt and manganese oxides. Metal interconnections <b>16</b> which extend out beyond the wall of the reaction cell may also be fabricated in a separate step so that the resistance of the thermistor may be measured using an external measuring device <b>18</b>.</p>
    <p num="p-0057">Temperature changes may also be sensed using a refractive index measurement technique. For example, techniques such as those described in Bornhop (1995, Applied Optics 34:3234–323) and U.S. Pat. No. 5,325,170, may be used to detect refractive index changes for liquids in capillaries. In such a technique, a low-power He—Ne laser is aimed off-center at a right angle to a capillary and undergoes multiple internal reflection. Part of the beam travels through the liquid while the remainder reflects only off the external capillary wall. The two beams undergo different phase shifts depending on the refractive index difference between the liquid and capillary. The result is an interference pattern, with the fringe position extremely sensitive to temperature-induced refractive index changes.</p>
    <p num="p-0058">In a further embodiment of the invention, the thermal response of the system may be increased by the presence of inorganic pyrophosphatase enzyme which is contacted with the template system along with the dNTP solution. Additionally, heat is released as the pyrophosphate released from the dNTPs upon incorporation into the template system is hydrolyzed by inorganic pyrophosphatase enzyme.</p>
    <p num="p-0059">In another embodiment, the pyrophosphate released upon incorporation of dNTP's may be removed from the template system and hydrolyzed, and the resultant heat detected, using thermopile, thermistor or refractive index methods, in a separate reaction cell downstream. In this reaction cell, inorganic pyrophosphatase enzyme may be mixed in solution with the dNTP removed from the DNA template system, or alternatively the inorganic pyrophosphatase enzyme may be covalently tethered to the wall of the reaction cell.</p>
    <p num="p-0060">Alternatively, the polymerase-catalyzed incorporation of a nucleotide base can be detected using fluorescence and chemiluminescence detection schemes. The DNA polymerase mediated extension is detected when a fluorescent or chemiluminescent signal is generated upon incorporation of a fluorescently or chemiluminescently labeled dNMP into the extending DNA primer strand. Such tags are attached to the nucleotide in such a way as to not interfere with the action of the polymerase. For example, the tag may be attached to the nucleotide base by a linker arm sufficiently long to move the bulky fluorophore away from the active site of the enzyme.</p>
    <p num="p-0061">For use of such detection schemes, nucleotide bases are labeled by covalently attaching a compound such that a fluorescent or chemiluminescent signal is generated following incorporation of a dNTP into the extending DNA primer/template. Examples of fluorescent compounds for labeling dNTPs include but are not limited to fluorescein, rhodamine, and BODIPY (4,4-difluoro-4-bora-3a,4a-diaza-s-indacene). See Handbook of Molecular Probes and Fluorescent Chemicals available from Molecular Probes, Inc. (Eugene, Oreg.). Examples of chemiluminescence based compounds that may be used in the sequencing methods of the invention include but are not limited to luminol and dioxetanones (See, Gunderman and McCapra, “Chemiluminescence in Organic Chemistry”, Springer-Verlag, Berlin Heidleberg, 1987)</p>
    <p num="p-0062">Fluorescently or chemiluminescently labeled dNTPs are added individually to a DNA template system containing template DNA annealed to the primer, DNA polymerase and the appropriate buffer conditions. After the reaction interval, the excess dNTP is removed and the system is probed to detect whether a fluorescent or chemiluminescent tagged nucleotide has been incorporated into the DNA template. Detection of the incorporated nucleotide can be accomplished using different methods that will depend on the type of tag utilized.</p>
    <p num="p-0063">For fluorescently-tagged dNTPs the DNA template system may be illuminated with optical radiation at a wavelength which is strongly absorbed by the tag entity. Fluorescence from the tag is detected using for example a photodetector together with an optical filter which excludes any scattered light at the excitation wavelength.</p>
    <p num="p-0064">Since labels on previously incorporated nucleotides would interfere with the signal generated by the most recently incorporated nucleotide, it is essential that the fluorescent tag be removed at the completion of each extension reaction. To facilitate removal of a fluorescent tag, the tag may be attached to the nucleotide via a chemically or photochemically cleavable linker using methods such as those described by Metzger, M. L. et al. (1994, Nucleic Acids Research 22:4259–4267) and Burgess, K. et al., (1997, J. Org. Chem. 62:5165–5168) so that the fluorescent tag may be removed from the DNA template system before a new extension reaction is carried out.</p>
    <p num="p-0065">In a further embodiment utilizing fluorescent detection, the fluorescent tag is attached to the dNTP by a photocleavable or chemically cleavable linker, and the tag is detached following the extension reaction and removed from the template system into a detection cell where the presence, and the amount, of the tag is determined by optical excitation at a suitable wavelength and detection of fluorescence. In this embodiment, the possibility of fluorescence quenching, due to the presence of multiple fluorescent tags immediately adjacent to one another on a primer strand which has been extended complementary to a single base repeat region in the template, is minimized, and the accuracy with which the repeat number can be determined is optimized. In addition, excitation of fluorescence in a separate chamber minimizes the possibility of photolytic damage to the DNA primer/template system.</p>
    <p num="p-0066">In a further embodiment of the technique, the response generated by a DNA polymerase-mediated extension reaction can be amplified. In this embodiment, the dNTP is chemically modified by the covalent attachment of a signaling tag through a linker that can be cleaved either chemically or photolytically. Following exposure of the dNTP to the primer/template system and flushing away any unincorporated chemically modified dNTP, any signaling tag that has been incorporated is detached by a chemical or photolytic reaction and flushed out of the reaction chamber to an amplification chamber in which an amplified signal may be produced and detected.</p>
    <p num="p-0067">A variety of methods may be used to produce an amplified signal. In one such method the signaling tag has a catalytic function. When the catalytic tag is cleaved and allowed to react with its substrate, many cycles of chemical reaction ensue producing many moles of product per mole of catalytic tag, with a corresponding multiplication of reaction enthalpy. Either the reaction product is detected, through some property such as color or absorbency, or the amplified heat product is detected by a thermal sensor. For example, if an enzyme is covalently attached to the dNTP via a cleavable linker arm of sufficient length that the enzyme does not interfere with the active site of the polymerase enzyme. Following incorporation onto the DNA primer strand, that enzyme is detached and transported to a second reactor volume in which it is allowed to interact with its specific substrate, thus an amplified response is obtained as each enzyme molecule carries out many cycles of reaction. For example, the enzyme catalase (CAT) catalyzes the reaction:</p>
    <p num="p-0068"> <chemistry id="CHEM-US-00001" num="00001"> <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7037687B2/US07037687-20060502-C00001.png"> <img id="EMI-C00001" he="4.32mm" wi="60.11mm" file="US07037687-20060502-C00001.TIF" alt="Figure US07037687-20060502-C00001" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US7037687B2/US07037687-20060502-C00001.png" class="patent-full-image" width="240" height="17"> </a> </div> <attachments> <attachment idref="CHEM-US-00001" attachment-type="cdx" file="US07037687-20060502-C00001.CDX"> </attachment> <attachment idref="CHEM-US-00001" attachment-type="mol" file="US07037687-20060502-C00001.MOL"> </attachment> </attachments> </chemistry> <br>
if each dNTP is tagged with a catalase molecule which is detached after dNMP incorporation and allowed to react downstream with hydrogen peroxide, each nucleotide incorporation would generate ˜25 kcal/mol×N of heat where N is the number of hydrogen peroxide molecules decomposed by the catalase. The heat of decomposition of hydrogen peroxide is already ˜6–8 times greater than for nucleotide incorporation, (i.e. 3.5–4 kcal/mol). For decomposition of ˜100–150 hydrogen peroxide molecules the amount of heat generated per base incorporation approaches 1000 times that of the unamplified reaction. Similarly, enzymes which produce colored products, such as those commonly used in enzyme-linked immunosorbent assays (ELISA) could be incorporated as detachable tags. For example the enzyme alkaline phosphatase converts colorless p-nitrophenyl phosphate to a colored product (p-nitrophenol); the enzyme horseradish peroxidase converts colorless o-phenylenediamine hydrochloride to an orange product. Chemistries for linking these enzymes to proteins such as antibodies are well-known to those versed in the art, and could be adapted to link the enzymes to nucleotide bases via linker arms that maintain the enzymes at a distance from the active site of the polymerase enzymes.
</p>
    <p num="p-0069">In a further embodiment, an amplified thermal signal may be produced when the signaling tag is an entity which can stimulate an active response in cells which are attached to, or held in the vicinity of, a thermal sensor such as a thermopile or thermistor. Pizziconi and Page (1997, Biosensors and Bioelectronics 12:457–466) reported that harvested and cultured mast cell populations could be activated by calcium ionophore to undergo exocytosis to release histamine, up to 10–30 μg (100–300 fmol) per cell. The multiple cell reactions leading to exocytosis are themselves exothermic. This process is further amplified using the enzymes diamine oxidase to oxidize the histamine to hydrogen peroxide and imidazoleacetaldehyde, and catalase to disproportionate the hydrogen peroxide. Two reactions together liberate over 100 kJ of heat per mole of histamine. For example, a calcium ionophore is covalently attached to the dNTP base via a linker arm which distances the linked calcium ionophore from the active site of the polymerase enzyme and is chemically or photochemically cleavable. Following the DNA polymerase catalyzed incorporation step, and flushing away unincorporated nucleotides any calcium ionophore remaining bound to an incorporated nucleotide may be cleaved and flushed downstream to a detection chamber containing a mast cell-based sensor such as described by Pizziconi and Page (1997, Biosensors and Bioelectronics 12:457–466). The calcium ionophore would bind to receptors on the mast cells stimulating histamine release with the accompanying generation of heat. The heat production could be further amplified by introducing the enzymes diamine oxidase to oxidize the histamine to hydrogen peroxide and imidazoleacetaldehyde, and catalase to disproportionate the hydrogen peroxide. Thus a significantly amplified heat signal would be produced which could readily be detected by a thermopile or thermistor sensor within, or in contact with, the reaction chamber.</p>
    <p num="p-0070">In a further embodiment utilizing chemiluminescent detection, the chemiluminescent tag is attached to the dNTP by a photocleavable or chemically cleavable linker. The tag is detached following the extension reaction and removed from the template system into a detection cell where the presence, and the amount, of the tag is determined by an appropriate chemical reaction and sensitive optical detection of the light produced. In this embodiment, the possibility of a non-linear optical response due to the presence of multiple chemiluminescent tags immediately adjacent to one another on a primer strand which has been extended complementary to a single base repeat region in the template, is minimized, and the accuracy with which the repeat number can be determined is optimized. In addition, generation of chemiluminescence in a separate chamber minimizes chemical damage to the DNA primer/template system, and allows detection under harsh chemical conditions which otherwise would chemically damage the DNA primer/template. In this way, chemiluminescent tags can be chosen to optimize chemiluminescence reaction speed, or compatibility of the tagged dNTP with the polymerase enzyme, without regard to the compatibility of the chemiluminescence reaction conditions with the DNA primer/template.</p>
    <p num="p-0071">In a further embodiment of the invention, the concentration of the dNTP solution removed from the template system following each extension reaction can be measured by detecting a change in UV absorption due to a change in the concentration of dNTPs, or a change in fluorescence response of fluorescently-tagged dNTPs. The incorporation of nucleotides into the extended template would result in a decreased concentration of nucleotides removed from the template system. Such a change could be detected by measuring the UV absorption of the buffer removed from the template system following each extension cycle.</p>
    <p num="p-0072">In a further embodiment of the invention, extension of the primer strand may be sensed by a device capable of sensing fluorescence from, or resolving an image of, a single DNA molecule. Devices capable of sensing fluorescence from a single molecule include the confocal microscope and the near-field optical microscope. Devices capable of resolving an image of a single molecule include the scanning tunneling microscope (STM) and the atomic force microscope (AFM).</p>
    <p num="p-0073">In this embodiment of the invention, a single DNA template molecule with attached primer is immobilized on a surface and viewed with an optical microscope or an STM or AFM before and after exposure to buffer solution containing a single type of dNTP, together with polymerase enzyme and other necessary electrolytes. When an optical microscope is used, the single molecule is exposed serially to fluorescently-tagged dNTP solutions and as before incorporation is sensed by detecting the fluorescent tag after excess unreacted dNTP is removed. Again as before, the incorporated fluorescent tag must be cleaved and discarded before a subsequent tag can be detected. Using the STM or AFM, the change in length of the primer strand is imaged to detect incorporation of the dNTP. Alternatively the dNTP may be tagged with a physically bulky molecule, more readily visible in the STM or AFM., and this bulky tag is removed and discarded before each fresh incorporation reaction.</p>
    <p num="p-0074">When sequencing a single molecular template in this way, the possibility of incomplete reaction producing erroneous signal and out-of-phase strand extension, does not exist and the consequent limitations on read length do not apply. For a single molecular template, reaction either occurs or it does not, and if it does not, then extension either ceases and is known to cease, or correct extension occurs in a subsequent cycle with the correct dNTP. In the event that an incorrect nucleotide is incorporated, which has the same probability as more the multiple strand processes discussed earlier, for example 1 in 1,000, an error is recorded in the sequence, but this error does not propagate or affect subsequent readout and so the read length is not limited by incorrect incorporation.</p>
    <p num="p-0075">Preparation of specific embodiments in accordance with the present invention will now be described in further detail. These examples are intended to be illustrative and the invention is not limited to the specific materials and methods set forth in these embodiments.</p>
    <heading>EXAMPLE 1</heading> <p num="p-0076">A microcalorimetic experiment was performed which demonstrates for the first time the successful thermal detection of a DNA polymerase reaction. The results are shown in <figref idrefs="DRAWINGS">FIG. 3</figref>. Approximately 20 units of T7 Sequenase was injected into a 3 mL reaction volume containing approximately 20 nmol of DNA template and complementary primer, and an excess of dNTPs. The primer was extended by 52-base pairs, the expected length given the size of the template. Using a commercial microcalorimeter (TAM Model 2273; Thermometrics, Sweden) a reaction enthalpy of 3.5–4 kcal per mole of base was measured (<figref idrefs="DRAWINGS">FIG. 3</figref>). This measurement is well within the value required for thermal detection of DNA polymerase activity. This measurement also demonstrates the sensitivity of thermopile detection as the maximum temperature rise in the reaction cell was 1×10<sup>−3</sup>° C. The lower trace seen in <figref idrefs="DRAWINGS">FIG. 3</figref> is from a reference cell showing the injection artifact for an enzyme-free injection into buffer containing no template system.</p>
    <heading>EXAMPLE 2</heading> <p num="p-0077">To illustrate the utility of mutant T4 polymerases, two primer extension assays were performed with two different mutant T4 polymerases, both of which are exonuclease deficient. In one mutant, Asp112 is replaced with Ala and Glu114 is replaced with Ala (D112A/E114A). The exonuclease activity of this mutant on double-stranded DNA is reduced by a factor of about 300 relative to the wild type enzyme as described by Reha-Krantz and Nonay (1993, J. Biol. Chem. 268:27100–27108). In a second polymerase mutant, in addition to the D112A/E114A amino acid substitutions, a third substitution replaces Ile417 with Val (I417V/D112A/E114A). The I417V mutation increases the accuracy of synthesis by this polymerase (Stocki, S. A. and Reha-Krantz, L. J, 1995, J. Mol. Biol. 245:15–28;Reha-Krantz, L. J. and Nonay, R. L., 1994, J. Biol. Chem. 269:5635–5643)</p>
    <p num="p-0078">Two separate primer extension reactions were carried out using each of the polymerase mutants. In the first, only a single correct nucleotide, dGTP, corresponding to a template C was added. The next unpaired template site is a G so that misincorporation would result in formation of a G•G mispair. A G•G mispair tends to be among the most difficult mispairs for polymerases to make. In the second primer extension reaction, two nucleotides, dGTP and dCTP, complementary to the first three unpaired template sites were added. Following correct incorporation of dGMP and dCMP, the next available template site is a T. Formation of C•T mispairs tend to be very difficult while G•T mispairs tend to be the most frequent mispairs made by polymerases.</p>
    <p num="p-0079">Time courses for primer extension reactions by both mutant T4 polymerases are shown in <figref idrefs="DRAWINGS">FIG. 4</figref>. Low concentrations of T4 polymerase relative to primer/template (p/t) were used so that incorporation reactions could be measured on convenient time scales (60 min). By 64 minutes 98% of the primers were extended. In reactions containing only dGTP, both polymerases nearly completely extended primer ends by dGMP without any detectable incorporation of dGMP opposite G. In reactions containing both dGMP and dCMP, both polymerases nearly completely extended primer ends by addition of one dGMP and two dCMP's. A small percentage (≈1%) of misincorporation was detectable in the reaction catalyzed by the D112A/E114A mutant. Significantly, no detectable misincorporation was seen in the reaction catalyzed by the I417V/D112A/E114A mutant.</p>
    <heading>EXAMPLE 3</heading> <p num="p-0080">In accordance with the invention a fluorescent tag may be attached to the nucleotide base at a site other than the 3′ position of the sugar moiety. Chemistries for such tags which do not interfere with the activity of the DNA polymerase have been developed as described by Goodwin et al. (1995, Experimental Technique of Physics 41:279–294). Generally the tag is attached to the base by a linker arm of sufficient length to move the bulky tag out of the active site of the enzyme during incorporation.</p>
    <p num="p-0081">As illustrated in <figref idrefs="DRAWINGS">FIG. 5</figref>, a nucleotide can be connected to a fluorophore by a photocleavable linker, e.g., a benzoin ester. After the tagged dNMP is incorporated onto the 3′ end of the DNA primer strand, the DNA template system is illuminated by light at a wave length corresponding to the absorption maximum of the fluorophore and the presence of the fluorophore is signaled by detection of fluorescence at the emission maximum of the fluorophore. Following detection of the fluorophore, the linker may be photocleaved to produce compound 2; the result is an elongated DNA molecule with a modified but non-fluorescent nucleotide attached. Many fluorophores, including for example, a dansyl group or acridine, etc., will be employed in the methodology illustrated by <figref idrefs="DRAWINGS">FIG. 5</figref>.</p>
    <p num="p-0082">Alternatively, the DNA template system is not illuminated to stimulate fluorescence. Instead, the photocleavage reaction is carried out to produce compound 2 releasing the fluorophore, which is removed from the template system into a separate detection chamber. There the presence of the fluorophore is detected as before, by illumination at the absorption maximum of the fluorophore and detection of emission near the emission maximum of the fluorophore.</p>
    <heading>EXAMPLE 4</heading> <p num="p-0083">In a specific embodiment of the invention, a linked system consisting of a chemiluminescently tagged dNTP can consist of a chemiluminescent group (the dioxetane portion of compound 4), a chemically cleavable linker (the silyl ether), and an optional photocleavable group (the benzoin ester) as depicted in <figref idrefs="DRAWINGS">FIG. 6</figref>. The cleavage of the silyl ether by a fluoride ion produces detectable chemiluminescence as described in Schaap et al. (1991, “Chemical and Enzymatic Triggering of 1,2-dioxetanes: Structural Effects on Chemiluminescence Efficiency” in Bioluminescence &amp; Chemiluminescence, Stanley, P. E. and Knicha, L. J. (Eds), Wiley, N.Y. 1991, pp. 103–106). In addition, the benzoin ester that links the nucleoside triphosphate to the silyl linker is photocleavable as set forth in Rock and Chan (1996, J. Org. Chem. 61: 1526–1529); and Felder, et al. (1997, First International Electronic Conference on Synthetic Organic Chemistry, September 1–30). Having both a chemiluminescent tag and a photocleavable linker is not always necessary; the silyl ether can be attached directly to the nucleotide base and the chemiluminescent tag is destroyed as it is read.</p>
    <p num="p-0084">As illustrated in <figref idrefs="DRAWINGS">FIG. 6</figref> with respect to compound 3, treatment with fluoride ion liberates the phenolate ion of the adamantyl dioxetane, which is known to chemiluminesce with high efficiency (Bronstein et al., 1991, “Novel Chemiluminescent Adamantyl 1,2-dioxetane Enzyme Substrates,” in Bioluminescence &amp; Chemiluminescence, Stanley, P. E. and Kricka, R. J. (eds), Wiley, N.Y. 1991 pp. 73–82). The other product of the reaction is compound 4, which is no longer chemiluminescent. Compound 4 upon photolysis at 308–366 nm liberates compound 2.</p>
    <p num="p-0085">The synthesis of compound 1 is achieved by attachment of the fluorophore to the carboxyl group of the benzoin, whose α-keto hydroxyl group is protected by 9-fluorenylmethoxycarbonyl (FMOC), followed by removal of the FMOC protecting group and coupling to the nucleotide bearing an activated carbonic acid derivative at its 3′ end. Compound 4 is prepared via coupling of the vinyl ether form of the adamantyl phenol, to chloro(3-cyanopropyl)dimethylsilane, reduction of the cyano group to the amine, generation of the oxetane, and coupling of this chemiluminescence precursor to the nucleotide bearing an activated carbonic acid derivative at its 3′ end.</p>
    <p num="p-0086">The chemiluminescent tag can also be attached to the dNTP by a cleavable linkage and cleaved prior to detection of chemiluminescence. As shown in <figref idrefs="DRAWINGS">FIG. 7</figref>, the benzoin ester linkage in compound 3 may be cleaved photolytically to produce the free chemiluminescent compound 5. Reaction of compound 5 with fluoride ion to generate chemiluminescence may then be carried out after compound 5 has been flushed away from the DNA template primer in the reaction chamber. As an alternative to photolytic cleavage, the tag may be attached by a chemically cleavable linker which is cleaved by chemical processing which does not trigger the chemiluminescent reaction.</p>
    <heading>EXAMPLE 5</heading> <p num="p-0087">In this example, the nucleotide sequence of a template molecule comprising a portion of DNA of unknown sequence is determined. The DNA of unknown sequence is cloned into a single stranded vector such as M13. A primer that is complementary to a single stranded region of the vector immediately upstream of the foreign DNA is annealed to the vector and used to prime synthesis in reactive sequencing. For the annealing reaction, equal molar ratios of primer and template (calculated based on the approximation that one base contributes 330 g/mol to the molecular weight of a DNA polymer) is mixed in a buffer consisting of 67 mM TrisHCl pH 8.8, 16.7 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, and 0.5 mM EDTA. This buffer is suitable both for annealing DNA and subsequent polymerase extension reactions. Annealing is accomplished by heating the DNA sample in buffer to 80° C., and allowing it to slowly cool to room temperature. Samples are briefly spun in a microcentrifuge to remove condensation from the lid and walls of the tube. To the DNA is added 0.2 mol equivalents of T4 polymerase mutant I417V/D112A/E114A and buffer components so that the final reaction cell contains 67 mM TrisHCl pH 8.8, 16.7 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 6.7 mM MgCl<sub>2 </sub>and 0.5 mM dithiothreitol. The polymerase is then queried with one dNTP at a time at a final concentration of 10 μM. The nucleotide is incubated with polymerase at 37° C. for 10 s. Incorporation of dNTPs may be detected by one of the methods described above including measuring fluorescence, chemiluminescence or temperature change. The reaction cycle will be repeated with each of the four dNTPs until the complete sequence of the DNA molecule has been determined.</p>
    <heading>EXAMPLE 6</heading> <p num="p-0088"> <figref idrefs="DRAWINGS">FIG. 7</figref> illustrates a mechanical fluorescent sequencing method in accordance with the invention. A DNA template and primer are captured onto beads 18 using, for example, avidin-biotin or —NH<sub>2</sub>/n-hydroxysuccinimide chemistry and loaded behind a porous frit or filter <b>20</b> at the tip of a micropipette <b>22</b> or other aspiration device as shown in <figref idrefs="DRAWINGS">FIG. 7(</figref> <i>a</i>), step <b>1</b>. Exonuclease deficient polymerase enzyme is added and the pipette tip is lowered into a small reservoir <b>24</b> containing a solution of fluorescently-labelled dNTP. As illustrated in step <b>2</b> of <figref idrefs="DRAWINGS">FIG. 7(</figref> <i>a</i>), a small quantity of dNTP solution is aspirated through the filter and allowed to react with the immobilized DNA. The dNTP solution also contains approximately 100 nM polymerase enzyme, sufficient to replenish rinsing losses. After reaction, as shown in step <b>3</b>, the excess dNTP solution <b>24</b> is forced back out through the frit <b>20</b> into the dNTP reservoir <b>24</b>. In step <b>4</b> of the process the pipette is moved to a reservoir containing buffer solution and several aliquots of buffer solution are aspirated through the frit to rinse excess unbound dNTP from the beads. The buffer inside the pipette is then forced out and discarded to waste <b>26</b>. The pipette is moved to a second buffer reservoir (buffer <b>2</b>), containing the chemicals required to cleave the fluorescent tag from the incorporated dNMP. The reaction is allowed to occur to cleave the tag. As shown in step <b>5</b> the bead/buffer slurry with the detached fluorescent tag in solution is irradiated by a laser or light source <b>28</b> at a wavelength chosen to excite the fluorescent tag, the fluorescence is detected by fluorescence detector 30 and quantified if incorporation has occurred.</p>
    <p num="p-0089">Subsequent steps depend on the enzyme strategy used. If a single-stage strategy with an exonuclease-deficient polymerase is used, as illustrated in <figref idrefs="DRAWINGS">FIG. 7(</figref> <i>b</i>), the solution containing the detached fluorescent tag is discarded to waste (step <b>6</b>) which is expelled, followed by a further rinse step with buffer <b>1</b> (step <b>7</b>) which is thereafter discarded (step <b>8</b>) and the pipette is moved to a second reservoir containing a different dNTP (step <b>9</b>) and the process repeats starting from step <b>3</b>, cycling through all four dNTPs.</p>
    <p num="p-0090">In a two-stage strategy, after the correct dNTP has been identified and the repeat length quantified in step <b>5</b>, the reaction mixture is rinsed as shown in steps <b>6</b>, <b>7</b>, and <b>8</b> of <figref idrefs="DRAWINGS">FIG. 7(</figref> <i>b</i>) and the pipette is returned to a different reservoir containing the same dNTP (e.g., dNTP1) as shown in step (a) of <figref idrefs="DRAWINGS">FIG. 8</figref> to which a quantity of exonuclease-proficient polymerase has been added and the solution is aspirated for a further stage of reaction which proof-reads the prior extension and correctly completes the extension. This second batch of dNTP need not be fluorescently tagged, as the identity of the dNTP is known and no sequence information will be gained in this proof-reading step. If a tagged dNTP is used, the fluorescent tag is preferably cleaved and discarded as in step <b>5</b> of <figref idrefs="DRAWINGS">FIG. 7(</figref> <i>a</i>) using Buffer <b>2</b>. Alternatively, the initial incorporation reaction shown in step <b>2</b> of <figref idrefs="DRAWINGS">FIG. 7(</figref> <i>a</i>) is carried out for long enough, and the initial polymerase is accurate enough, so that the additional amount of fluorescent tag incorporated with dNTP1 at step a of <figref idrefs="DRAWINGS">FIG. 8</figref> is small and does not interfere with quantification of the subsequent dNTP. Following proof-reading in step a of <figref idrefs="DRAWINGS">FIG. 8</figref>, excess dNTP is expelled (step b) and the reaction mixture is rinsed (steps c, d) with a high-salt buffer to dissociate the exo+ polymerase from the DNA primer/template. It is important not to have exonuclease-proficient enzyme present if the DNA primer/template is exposed to an incorrect dNTP. The pipette is then moved to step e, in which the reservoir contains a different dNTP, and the process is repeated, again cycling through all four dNTPs.</p>
    <p num="p-0091">Although the invention has been described herein with reference to specific embodiments, many modifications and variations therein will readily occur to those skilled in the art. Accordingly, all such variations and modifications are included within the intended scope of the invention.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5302509">US5302509</a></td><td class="patent-data-table-td patent-date-value">Feb 27, 1991</td><td class="patent-data-table-td patent-date-value">Apr 12, 1994</td><td class="patent-data-table-td ">Beckman Instruments, Inc.</td><td class="patent-data-table-td ">Method for sequencing polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5436149">US5436149</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 19, 1993</td><td class="patent-data-table-td patent-date-value">Jul 25, 1995</td><td class="patent-data-table-td ">Barnes; Wayne M.</td><td class="patent-data-table-td ">Thermostable DNA polymerase with enhanced thermostability and enhanced length and efficiency of primer extension</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5547859">US5547859</a></td><td class="patent-data-table-td patent-date-value">Aug 2, 1993</td><td class="patent-data-table-td patent-date-value">Aug 20, 1996</td><td class="patent-data-table-td ">Goodman; Myron F.</td><td class="patent-data-table-td ">Chain-terminating nucleotides for DNA sequencing methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5945284">US5945284</a></td><td class="patent-data-table-td patent-date-value">May 27, 1997</td><td class="patent-data-table-td patent-date-value">Aug 31, 1999</td><td class="patent-data-table-td ">The Perkin-Elmer Corporation</td><td class="patent-data-table-td ">Length determination of nucleic acid repeat sequences by discontinuous primer extension</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6028190">US6028190</a></td><td class="patent-data-table-td patent-date-value">May 8, 1996</td><td class="patent-data-table-td patent-date-value">Feb 22, 2000</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Probes labeled with energy transfer coupled dyes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6087095">US6087095</a></td><td class="patent-data-table-td patent-date-value">Apr 22, 1993</td><td class="patent-data-table-td patent-date-value">Jul 11, 2000</td><td class="patent-data-table-td ">Medical Research Council</td><td class="patent-data-table-td ">DNA sequencing method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0223618A2?cl=en">EP0223618A2</a></td><td class="patent-data-table-td patent-date-value">Jul 16, 1986</td><td class="patent-data-table-td patent-date-value">May 27, 1987</td><td class="patent-data-table-td ">New York Medical College</td><td class="patent-data-table-td ">Automatable process for nucleotide sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1989009283A1?cl=en">WO1989009283A1</a></td><td class="patent-data-table-td patent-date-value">Mar 3, 1989</td><td class="patent-data-table-td patent-date-value">Oct 5, 1989</td><td class="patent-data-table-td ">Edward David Hyman</td><td class="patent-data-table-td ">Pyrophosphate-based method and apparatus for sequencing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1990013666A1?cl=en">WO1990013666A1</a></td><td class="patent-data-table-td patent-date-value">May 11, 1990</td><td class="patent-data-table-td patent-date-value">Nov 12, 1990</td><td class="patent-data-table-td ">Amersham Int Plc</td><td class="patent-data-table-td ">Sequencing method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1991006678A1?cl=en">WO1991006678A1</a></td><td class="patent-data-table-td patent-date-value">Oct 26, 1990</td><td class="patent-data-table-td patent-date-value">Apr 27, 1991</td><td class="patent-data-table-td ">Stanford Res Inst Int</td><td class="patent-data-table-td ">Dna sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1996027025A1?cl=en">WO1996027025A1</a></td><td class="patent-data-table-td patent-date-value">Feb 21, 1996</td><td class="patent-data-table-td patent-date-value">Sep 6, 1996</td><td class="patent-data-table-td ">Ely Michael Rabani</td><td class="patent-data-table-td ">Device, compounds, algorithms, and methods of molecular characterization and manipulation with molecular parallelism</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7427673">US7427673</a></td><td class="patent-data-table-td patent-date-value">Dec 4, 2002</td><td class="patent-data-table-td patent-date-value">Sep 23, 2008</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7541444">US7541444</a></td><td class="patent-data-table-td patent-date-value">Aug 22, 2003</td><td class="patent-data-table-td patent-date-value">Jun 2, 2009</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Invention relates to nucleotide or nucleoside molecules modified by reversible covalent attachment of 3&#39;-OH blocking groups thereto, and which molecules may be used in reactions where blocked nucleotide or nucleoside molecules are required, such as in sequencing reactions, polynucleotide synthesis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7566537">US7566537</a></td><td class="patent-data-table-td patent-date-value">Dec 13, 2005</td><td class="patent-data-table-td patent-date-value">Jul 28, 2009</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7592435">US7592435</a></td><td class="patent-data-table-td patent-date-value">Jul 27, 2006</td><td class="patent-data-table-td patent-date-value">Sep 22, 2009</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">increase the intensity of the fluorescent signal after nucleic acid sequencing reactions; a modified fluorescent labelled guanine-containing nucleotides which exhibit enhanced fluorophore intensity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7645596">US7645596</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 5, 2004</td><td class="patent-data-table-td patent-date-value">Jan 12, 2010</td><td class="patent-data-table-td ">Arizona Board Of Regents</td><td class="patent-data-table-td ">Polymerase chain reactions; exonucleases; microfluidics</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7668697">US7668697</a></td><td class="patent-data-table-td patent-date-value">Feb 6, 2007</td><td class="patent-data-table-td patent-date-value">Feb 23, 2010</td><td class="patent-data-table-td ">Andrei Volkov</td><td class="patent-data-table-td ">Method for analyzing dynamic detectable events at the single molecule level</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7713698">US7713698</a></td><td class="patent-data-table-td patent-date-value">Aug 20, 2007</td><td class="patent-data-table-td patent-date-value">May 11, 2010</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Massive parallel method for decoding DNA and RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7771973">US7771973</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 2009</td><td class="patent-data-table-td patent-date-value">Aug 10, 2010</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Modified nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7772384">US7772384</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2008</td><td class="patent-data-table-td patent-date-value">Aug 10, 2010</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">nucleotides having a removable labels, used in polynucleotide sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7785796">US7785796</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2009</td><td class="patent-data-table-td patent-date-value">Aug 31, 2010</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">sequence determination of an immobilized target polynucleotide comprising monitoring the sequential incorporation of nucleotides complementary to the immobilized target polynucleotide; the nucleotides have a base that is linked to a detectable label via a cleavable linker; detection of the labels</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7790869">US7790869</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 2007</td><td class="patent-data-table-td patent-date-value">Sep 7, 2010</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">A nucleotide having a base attached to a detectable label through a cleavable linker; has a deoxyribose comprising a cleavable chemical group capping the 3&#39; OH group of the deoxyribose and wherein both the cleavable linker and the capping cleavable chemical group are cleavable by light</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7816503">US7816503</a></td><td class="patent-data-table-td patent-date-value">Sep 21, 2009</td><td class="patent-data-table-td patent-date-value">Oct 19, 2010</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">increase the intensity of the fluorescent signal after nucleic acid sequencing reactions; a modified fluorescent labelled guanine-containing nucleotides which exhibit enhanced fluorophore intensity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7948015">US7948015</a></td><td class="patent-data-table-td patent-date-value">Dec 14, 2007</td><td class="patent-data-table-td patent-date-value">May 24, 2011</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8043814">US8043814</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2008</td><td class="patent-data-table-td patent-date-value">Oct 25, 2011</td><td class="patent-data-table-td ">Eric Guilbeau</td><td class="patent-data-table-td ">Thermoelectric method of sequencing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8071739">US8071739</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2010</td><td class="patent-data-table-td patent-date-value">Dec 6, 2011</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Modified nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8088575">US8088575</a></td><td class="patent-data-table-td patent-date-value">Jul 19, 2010</td><td class="patent-data-table-td patent-date-value">Jan 3, 2012</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Massive parallel method for decoding DNA and RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8148064">US8148064</a></td><td class="patent-data-table-td patent-date-value">Jul 19, 2010</td><td class="patent-data-table-td patent-date-value">Apr 3, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8158346">US8158346</a></td><td class="patent-data-table-td patent-date-value">Jul 13, 2010</td><td class="patent-data-table-td patent-date-value">Apr 17, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8192961">US8192961</a></td><td class="patent-data-table-td patent-date-value">Oct 24, 2007</td><td class="patent-data-table-td patent-date-value">Jun 5, 2012</td><td class="patent-data-table-td ">Pacific Biosciences Of California, Inc.</td><td class="patent-data-table-td ">System and methods for nucleic acid sequencing of single molecules by polymerase synthesis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8212015">US8212015</a></td><td class="patent-data-table-td patent-date-value">Oct 6, 2010</td><td class="patent-data-table-td patent-date-value">Jul 3, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Modified nucleosides and nucleotides and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8217433">US8217433</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Jul 10, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8247849">US8247849</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Aug 21, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Two-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8262900">US8262900</a></td><td class="patent-data-table-td patent-date-value">Dec 17, 2007</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8263336">US8263336</a></td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8264014">US8264014</a></td><td class="patent-data-table-td patent-date-value">Feb 2, 2011</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8269261">US8269261</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2011</td><td class="patent-data-table-td patent-date-value">Sep 18, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8293082">US8293082</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Oct 23, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8298792">US8298792</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2011</td><td class="patent-data-table-td patent-date-value">Oct 30, 2012</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Four-color DNA sequencing by synthesis using cleavable fluorescent nucleotide reversible terminators</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8306757">US8306757</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2010</td><td class="patent-data-table-td patent-date-value">Nov 6, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8313625">US8313625</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Nov 20, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8313639">US8313639</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Nov 20, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8317999">US8317999</a></td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td patent-date-value">Nov 27, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8349167">US8349167</a></td><td class="patent-data-table-td patent-date-value">Jun 26, 2009</td><td class="patent-data-table-td patent-date-value">Jan 8, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8394586">US8394586</a></td><td class="patent-data-table-td patent-date-value">Mar 28, 2012</td><td class="patent-data-table-td patent-date-value">Mar 12, 2013</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8399188">US8399188</a></td><td class="patent-data-table-td patent-date-value">Sep 4, 2007</td><td class="patent-data-table-td patent-date-value">Mar 19, 2013</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Compositions and methods for nucleotide sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415176">US8415176</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415177">US8415177</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Two-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415716">US8415716</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensors with feedback circuits</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8421437">US8421437</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Apr 16, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Array column integrator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8426898">US8426898</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2010</td><td class="patent-data-table-td patent-date-value">Apr 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8426899">US8426899</a></td><td class="patent-data-table-td patent-date-value">Feb 2, 2011</td><td class="patent-data-table-td patent-date-value">Apr 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8432149">US8432149</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Array column integrator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8432150">US8432150</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for operating an array column integrator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8435395">US8435395</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">May 7, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8441044">US8441044</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">May 14, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for manufacturing low noise chemically-sensitive field effect transistors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8445945">US8445945</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">May 21, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Low noise chemically-sensitive field effect transistors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8450781">US8450781</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">May 28, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8455927">US8455927</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Jun 4, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array with cascoded column circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8470164">US8470164</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2009</td><td class="patent-data-table-td patent-date-value">Jun 25, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8487790">US8487790</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Jul 16, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemical detection circuit including a serializer circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8492799">US8492799</a></td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td patent-date-value">Jul 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8492800">US8492800</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">Jul 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensors with sample and hold capacitors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8496802">US8496802</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Jul 30, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for operating chemically-sensitive sample and hold sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8501412">US8501412</a></td><td class="patent-data-table-td patent-date-value">Sep 21, 2011</td><td class="patent-data-table-td patent-date-value">Aug 6, 2013</td><td class="patent-data-table-td ">Eric Guilbeau</td><td class="patent-data-table-td ">Thermoelectric method of sequencing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8502278">US8502278</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Aug 6, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically-sensitive sample and hold sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8519448">US8519448</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Aug 27, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically-sensitive array with active and reference sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524057">US8524057</a></td><td class="patent-data-table-td patent-date-value">Jul 28, 2011</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524487">US8524487</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array with cascoded column circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8530154">US8530154</a></td><td class="patent-data-table-td patent-date-value">Jul 21, 2006</td><td class="patent-data-table-td patent-date-value">Sep 10, 2013</td><td class="patent-data-table-td ">Pacific Biosciences Of California, Inc.</td><td class="patent-data-table-td ">System and method for nucleic acid sequencing by polymerase synthesis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8530941">US8530941</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2011</td><td class="patent-data-table-td patent-date-value">Sep 10, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8535513">US8535513</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">Sep 17, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540865">US8540865</a></td><td class="patent-data-table-td patent-date-value">Sep 8, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540866">US8540866</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540867">US8540867</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540868">US8540868</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8558288">US8558288</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Oct 15, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8575664">US8575664</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Nov 5, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically-sensitive sensor array calibration circuitry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8592153">US8592153</a></td><td class="patent-data-table-td patent-date-value">Sep 5, 2012</td><td class="patent-data-table-td patent-date-value">Nov 26, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for manufacturing high capacitance microwell structures of chemically-sensitive sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8592154">US8592154</a></td><td class="patent-data-table-td patent-date-value">Sep 7, 2012</td><td class="patent-data-table-td patent-date-value">Nov 26, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for high speed operation of a chemically-sensitive sensor array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8597881">US8597881</a></td><td class="patent-data-table-td patent-date-value">Oct 25, 2011</td><td class="patent-data-table-td patent-date-value">Dec 3, 2013</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Modified nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8653567">US8653567</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Feb 18, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensor with lightly doped drains</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8658017">US8658017</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">Feb 25, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for operating an array of chemically-sensitive sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8666678">US8666678</a></td><td class="patent-data-table-td patent-date-value">Oct 27, 2011</td><td class="patent-data-table-td patent-date-value">Mar 4, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Predictive model for use in sequencing-by-synthesis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8673627">US8673627</a></td><td class="patent-data-table-td patent-date-value">May 24, 2010</td><td class="patent-data-table-td patent-date-value">Mar 18, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Apparatus and methods for performing electrochemical reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8685230">US8685230</a></td><td class="patent-data-table-td patent-date-value">Jul 24, 2012</td><td class="patent-data-table-td patent-date-value">Apr 1, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for high-speed operation of a chemically-sensitive sensor array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8685324">US8685324</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 1, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Matched pair transistor circuits</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8698212">US8698212</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Apr 15, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8742469">US8742469</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors with correlated double sampling</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8747748">US8747748</a></td><td class="patent-data-table-td patent-date-value">Jan 19, 2012</td><td class="patent-data-table-td patent-date-value">Jun 10, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemical sensor with conductive cup-shaped sensor surface</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8748947">US8748947</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Jun 10, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors with reset switch</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8764969">US8764969</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">Jul 1, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for operating chemically sensitive sensors with sample and hold capacitors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8766327">US8766327</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Jul 1, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors with in-sensor current sources</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8772698">US8772698</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Jul 8, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">CCD-based multi-transistor active pixel sensor array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012122484A1?cl=en">WO2012122484A1</a></td><td class="patent-data-table-td patent-date-value">Mar 9, 2012</td><td class="patent-data-table-td patent-date-value">Sep 13, 2012</td><td class="patent-data-table-td ">Roberto Polakiewicz</td><td class="patent-data-table-td ">Methods and reagents for creating monoclonal antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013174432A1?cl=en">WO2013174432A1</a></td><td class="patent-data-table-td patent-date-value">May 24, 2012</td><td class="patent-data-table-td patent-date-value">Nov 28, 2013</td><td class="patent-data-table-td ">Fundació Institut D&#39;investigació Biomèdica De Bellvitge (Idibell)</td><td class="patent-data-table-td ">Method for the identification of the origin of a cancer of unknown primary origin by methylation analysis</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S022100">536/22.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S005000">435/5</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006170">435/6.17</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006120">435/6.12</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0021780000">G01N21/78</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12M0001000000">C12M1/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033580000">G01N33/58</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033530000">G01N33/53</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0019340000">C12P19/34</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021000000">C07H21/00</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=gCVzBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6869">C12Q1/6869</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68E</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Nov 4, 2013</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">LIFE TECHNOLOGIES CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20130823</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:ARIZONA BOARD OF REGENTS FOR AND ON BEHALF OF ARIZONA STATE UNIVERSITY;REEL/FRAME:031159/0626</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 9, 2012</td><td class="patent-data-table-td ">FPB1</td><td class="patent-data-table-td ">Expired due to reexamination which canceled all claims</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 17, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120417</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BROWN RUDNICK LLP, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">NOTICE OF ATTORNEY S LIEN;ASSIGNOR:HELICOS BIOSCIENCES CORPORATION;REEL/FRAME:028060/0898</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 19, 2011</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20110127</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 22, 2009</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 22, 2006</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ARIZONA BOARD OF REGENTS FOR AND ON BEHALF OF ARIZ</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:ASHTON, TOMMY L., II;REEL/FRAME:018670/0841</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20061220</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 29, 2006</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 22, 2003</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ALBERTA, UNIVERSITY OF, CANADA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:REHA-KRANTZ, LINDA J.;REEL/FRAME:014503/0283</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ARIZONA BOARD OF REGENTS, ARIZONA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:WILLIAMS, PETER;HAYES, MARK A.;ROSE, SETH D.;AND OTHERS;REEL/FRAME:014503/0223</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20010126</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U17hwBj9ay2FF8k3Wt1HibeqHaj5A\u0026id=gCVzBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U191HpLVsV5S5vZl81w7-YK7n7fiA\u0026id=gCVzBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U0DcumxcKwoW8930DUTsJ-UFEgJTA","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Sequencing_DNA_using_polymerase_fluoresc.pdf?id=gCVzBAABERAJ\u0026output=pdf\u0026sig=ACfU3U1Za3ttCkCc5aXagoT1UuMM3BPzjA"},"sample_url":"http://www.google.com/patents/reader?id=gCVzBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>