<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7060431 - Method of making and decoding of array sensors with microspheres - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_4ff636b3d23669b7103f3b3a3a18b4cd/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_4ff636b3d23669b7103f3b3a3a18b4cd__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Method of making and decoding of array sensors with microspheres"><meta name="DC.contributor" content="Mark S. Chee" scheme="inventor"><meta name="DC.contributor" content="John R. Stuelpnagel" scheme="inventor"><meta name="DC.contributor" content="Anthony W. Czarnik" scheme="inventor"><meta name="DC.contributor" content="Illumina, Inc." scheme="assignee"><meta name="DC.date" content="1999-6-24" scheme="dateSubmitted"><meta name="DC.description" content="The invention relates to compositions and methods for decoding microsphere array sensors."><meta name="DC.date" content="2006-6-13" scheme="issued"><meta name="DC.relation" content="US:3712535" scheme="references"><meta name="DC.relation" content="US:3748975" scheme="references"><meta name="DC.relation" content="US:4125314" scheme="references"><meta name="DC.relation" content="US:4200110" scheme="references"><meta name="DC.relation" content="US:4224198" scheme="references"><meta name="DC.relation" content="US:4448485" scheme="references"><meta name="DC.relation" content="US:4499052" scheme="references"><meta name="DC.relation" content="US:4582789" scheme="references"><meta name="DC.relation" content="US:4682895" scheme="references"><meta name="DC.relation" content="US:4687732" scheme="references"><meta name="DC.relation" content="US:4729949" scheme="references"><meta name="DC.relation" content="US:4772540" scheme="references"><meta name="DC.relation" content="US:4785814" scheme="references"><meta name="DC.relation" content="US:4822746" scheme="references"><meta name="DC.relation" content="US:4824789" scheme="references"><meta name="DC.relation" content="US:4842783" scheme="references"><meta name="DC.relation" content="US:4868130" scheme="references"><meta name="DC.relation" content="US:4876187" scheme="references"><meta name="DC.relation" content="US:4879097" scheme="references"><meta name="DC.relation" content="US:4894343" scheme="references"><meta name="DC.relation" content="US:4895805" scheme="references"><meta name="DC.relation" content="US:4981783" scheme="references"><meta name="DC.relation" content="US:4999306" scheme="references"><meta name="DC.relation" content="US:5002867" scheme="references"><meta name="DC.relation" content="US:5015843" scheme="references"><meta name="DC.relation" content="US:5019350" scheme="references"><meta name="DC.relation" content="US:5026599" scheme="references"><meta name="DC.relation" content="US:5028545" scheme="references"><meta name="DC.relation" content="US:5061336" scheme="references"><meta name="DC.relation" content="US:5071531" scheme="references"><meta name="DC.relation" content="US:5105305" scheme="references"><meta name="DC.relation" content="US:5110745" scheme="references"><meta name="DC.relation" content="US:5114864" scheme="references"><meta name="DC.relation" content="US:5126022" scheme="references"><meta name="DC.relation" content="US:5132242" scheme="references"><meta name="DC.relation" content="US:5135627" scheme="references"><meta name="DC.relation" content="US:5143853" scheme="references"><meta name="DC.relation" content="US:5176881" scheme="references"><meta name="DC.relation" content="US:5177012" scheme="references"><meta name="DC.relation" content="US:5185178" scheme="references"><meta name="DC.relation" content="US:5185243" scheme="references"><meta name="DC.relation" content="US:5194300" scheme="references"><meta name="DC.relation" content="US:5244636" scheme="references"><meta name="DC.relation" content="US:5244813" scheme="references"><meta name="DC.relation" content="US:5250264" scheme="references"><meta name="DC.relation" content="US:5252494" scheme="references"><meta name="DC.relation" content="US:5254477" scheme="references"><meta name="DC.relation" content="US:5272081" scheme="references"><meta name="DC.relation" content="US:5296375" scheme="references"><meta name="DC.relation" content="US:5298741" scheme="references"><meta name="DC.relation" content="US:5302509" scheme="references"><meta name="DC.relation" content="US:5304487" scheme="references"><meta name="DC.relation" content="US:5308771" scheme="references"><meta name="DC.relation" content="US:5310674" scheme="references"><meta name="DC.relation" content="US:5320814" scheme="references"><meta name="DC.relation" content="US:5326691" scheme="references"><meta name="DC.relation" content="US:5342737" scheme="references"><meta name="DC.relation" content="US:5357590" scheme="references"><meta name="DC.relation" content="US:5380489" scheme="references"><meta name="DC.relation" content="US:5435724" scheme="references"><meta name="DC.relation" content="US:5474895" scheme="references"><meta name="DC.relation" content="US:5481629" scheme="references"><meta name="DC.relation" content="US:5486335" scheme="references"><meta name="DC.relation" content="US:5494798" scheme="references"><meta name="DC.relation" content="US:5494810" scheme="references"><meta name="DC.relation" content="US:5496997" scheme="references"><meta name="DC.relation" content="US:5498392" scheme="references"><meta name="DC.relation" content="US:5506141" scheme="references"><meta name="DC.relation" content="US:5512490" scheme="references"><meta name="DC.relation" content="US:5516635" scheme="references"><meta name="DC.relation" content="US:5518863" scheme="references"><meta name="DC.relation" content="US:5518883" scheme="references"><meta name="DC.relation" content="US:5541311" scheme="references"><meta name="DC.relation" content="US:5545531" scheme="references"><meta name="DC.relation" content="US:5554516" scheme="references"><meta name="DC.relation" content="US:5565324" scheme="references"><meta name="DC.relation" content="US:5573909" scheme="references"><meta name="DC.relation" content="US:5575849" scheme="references"><meta name="DC.relation" content="US:5585069" scheme="references"><meta name="DC.relation" content="US:5587128" scheme="references"><meta name="DC.relation" content="US:5589351" scheme="references"><meta name="DC.relation" content="US:5593838" scheme="references"><meta name="DC.relation" content="US:5595915" scheme="references"><meta name="DC.relation" content="US:5603351" scheme="references"><meta name="DC.relation" content="US:5604097" scheme="references"><meta name="DC.relation" content="US:5610287" scheme="references"><meta name="DC.relation" content="US:5631170" scheme="references"><meta name="DC.relation" content="US:5631337" scheme="references"><meta name="DC.relation" content="US:5632876" scheme="references"><meta name="DC.relation" content="US:5632957" scheme="references"><meta name="DC.relation" content="US:5633972" scheme="references"><meta name="DC.relation" content="US:5635358" scheme="references"><meta name="DC.relation" content="US:5637469" scheme="references"><meta name="DC.relation" content="US:5639603" scheme="references"><meta name="DC.relation" content="US:5640234" scheme="references"><meta name="DC.relation" content="US:5643738" scheme="references"><meta name="DC.relation" content="US:5647030" scheme="references"><meta name="DC.relation" content="US:5649576" scheme="references"><meta name="DC.relation" content="US:5656241" scheme="references"><meta name="DC.relation" content="US:5660988" scheme="references"><meta name="DC.relation" content="US:5671303" scheme="references"><meta name="DC.relation" content="US:5856083" scheme="references"><meta name="DC.relation" content="US:5863722" scheme="references"><meta name="DC.relation" content="US:6197515" scheme="references"><meta name="citation_reference" content="Abel et al., &quot;Fiber-Optic Evanescent Wave Biosensor for the Detection of Oligonucleotides,&quot; Anal. Chem. 68:2905-2912 (1996)."><meta name="citation_reference" content="Angel, S., &quot;Optrodes: Chemically Selective Fiber-Optic Sensors,&quot; Spectroscopy, 2(4):38-47 (1987)."><meta name="citation_reference" content="Anonymous, &quot;Fluorescent Microspheres,&quot; Tech. Note 19, Bang Laboratories, (Fishers, In) Feb. 1997."><meta name="citation_reference" content="Anonymous, &quot;Microsphere Selection Guide,&quot; Bandg Laboratories, (Fisher, In) Sep. 1998."><meta name="citation_reference" content="Baner et al., &quot;Signal amplification of padlock probes by rolling circle replication,&quot; Nucleic Acids Research, 26(22): 5073-5078 (1998)."><meta name="citation_reference" content="Bangs, L.B., &quot;Immunological Applications of Microspheres,&quot; The Latex Course, Bangs Laboratories (Carmel, IN) Apr. 1996."><meta name="citation_reference" content="Barnard et al., &quot;A Fibre-Optic Chemical Sensor with Discrete Sensing Sites,&quot; Nature, 353:338-340 (Sep. 1991)."><meta name="citation_reference" content="Barnard et al., &quot;Fiber-Optic Organic Vapor Sensor,&quot; Environ. Sci. Technol., 25(7): 1301-1304 (1991)."><meta name="citation_reference" content="Ben-Dor, A., et al., &quot;Universal DNA Tag Systems: A Combinatorial Design Scheme.&quot; J. Compu. Biol. 7(3/4): 503-519 (2000)."><meta name="citation_reference" content="Birindelli, S. et al., &quot;Comments on Adaptation of the Cellscan Technique for the SCM Test in Breast Cancer.&quot; European J. Cancer 33(8):1333-1334 (1997)."><meta name="citation_reference" content="Brenner, S., et al. &quot;Gene Expression Analysis by Massively Parallel Signature Sequencing (MPSS) on Microbead Arrays.&quot; Nature Biotechnology 18: 630-634 (2000)."><meta name="citation_reference" content="Bruchez et al., &quot;Semiconductor Nanocrystals as Fluorescent Biological Labels,&quot; Science, 281: p. 2013-; Sep. 25, 1998."><meta name="citation_reference" content="Carey, W.P. et al., &quot;Chemical Piezoelectric Sensor and Sensor Array Characterization,&quot; Anal. Chem. 58(14):3077-3084 (1986)."><meta name="citation_reference" content="Castano, J. et al., &quot;Dynamic Monitoring and Quantification of Gene Expression in Single Living Cells: A Molecular Basis for Secretory Cell Heterogeneity,&quot; Mol. Endo. 10(5):599-605 (1996)."><meta name="citation_reference" content="Chen et al., &quot;A Microsphere-Based Assay for Multiplexed Single Nucleotide Polymorphism Analysis Using Single Base Chain Extension,&quot; Genome Research, 10(4):549-557 (2000)."><meta name="citation_reference" content="Chiavaroli, C. et al., &quot;Simultaneous Monitorinig of Cytosolic Free Calcium and Exocytosis at the Single Cell Leve,&quot; J. Neuroendocrinology 3(3):253-260 (1991)."><meta name="citation_reference" content="Clark, R. et al., &quot;Electrochemical Analysis in Picoliter Microvials,&quot; Anal. Chem. 69:259-263 (1997)."><meta name="citation_reference" content="Czarnik. A &quot;Illuminating the SNP Genomic Code,&quot; Modern Drug Discovery, 49-55 (Nov./Dec. 1998)."><meta name="citation_reference" content="Danielson et al., &quot;A Combinatorial Approach to the Discovery and Optimization of Luminescent Materials,&quot; Nature, 389:944-948 (1997)."><meta name="citation_reference" content="DeForest, W.S., &quot;Photoresist Materials and Processes,&quot; McGraw-Hill Book Company, New York. 89101 (1975)."><meta name="citation_reference" content="Deutsch, M. et al., &quot;Apparatus for high Precision Repetitive Sequential Optical Measurement of Living Dells,&quot; Cytometry 16:214-226 (1994)."><meta name="citation_reference" content="Deutsch, M. et al., &quot;Lymphocyte Fluorescence Polarization Measurements with the Cellscan System: Application to the SCM Cancer Test,&quot; Cytometry 23:159-165 (1996)."><meta name="citation_reference" content="Dickinson et al., &quot;Generating Sensor Diversity Through Combinatorial Polymer Synthesis,&quot; Analytical Chemistry, 69(17): 3413-3418 (1997)."><meta name="citation_reference" content="DiMarco, G. et al., &quot;Luminescent Ru&lt;SUP&gt;11&lt;/SUP&gt;-Polypyridine Complexes in Poly-2-Hydroxyethylmetharcrylate Matrices as Oxygen Sensors,&quot; Adv. Mater., 7(5):(1995)."><meta name="citation_reference" content="Drmanac, R. et al., &quot;Prospects for a Miniaturized, Simplified and Frugal Human Genome Project,&quot; Scientia Yugoslavica, 16(1-2):97-107 (1990)."><meta name="citation_reference" content="Drmanac, R. et al., &quot;Sequencing by Hybridization (SBH) with Oligonucleotide Probes as an Integral Approach for the Analysis of Complex Genomes,&quot; International Journal of Genome Research, 1(1):59-79 (1992)."><meta name="citation_reference" content="Drmanac, R. et al., &quot;Sequencing by Hybridization,&quot; Automated DNA Sequencing and Analysis, ed. M. Adams, C. Fields and J. Venter. (1994)."><meta name="citation_reference" content="Drmanac, R. et al., &quot;Sequencing by Oligonucleotide Hybridization: A Promising Framework in Decoding of the Genome Program,&quot; The First International Conference on Electrophoresis, Supercomputing and the Human Genome, Proceeding os th Apr. 10-13, 1990 Conference at Florida State University. Ed. C. Cantor and H. Lim."><meta name="citation_reference" content="Egner, B.J., et al., &quot;Tagging in Combinatorial Chemistry: The Use of Coloured and Fluorescent Beads.&quot; Chem. Commun. (1997)."><meta name="citation_reference" content="Fan, J.B., et al., &quot;Parallel Genotyping of Human SNPs Using Generic High-density Oligonucleotide Tag Arrays.&quot; Genome Research, 10:853-860 (2000)."><meta name="citation_reference" content="Fathi, et al., &quot;Synthesis and Properties of Combinatorial Libraries of Phosphoramidates,&quot; J. Org. Chem. Soc. 61(16): 5600-5609 (1996)."><meta name="citation_reference" content="Ferguson et al., &quot;A Fiber-Optic DNA Biosensor Microarray for the Analysis of Gene Expression,&quot; Nature Biotechnology, 14:1681-1684 (1996)."><meta name="citation_reference" content="Fodor, S. et al., &quot;Light-Directed, Spatially Addressable Parallel Chemical Synthesis,&quot; Science, 251:767-773 (1991)."><meta name="citation_reference" content="Freeman, T. et al., &quot;Oxygen Probe Based on Tetrakis(alkylamino)ethylene Chemiluminescence,&quot; Anal. Chem., 53:98102 (1981)."><meta name="citation_reference" content="Fuh et al., &quot;Single Fibre Optic Fluorescence pH Probe,&quot; Analyst, 112:1159-1163 (1987)."><meta name="citation_reference" content="Gauci, M. et al., &quot;Observation of Single-Cell Fluorescence Spectra in Laser Flow Cytometry,&quot; Cytometry 25:388-393 (1996)."><meta name="citation_reference" content="Gerry, N.P., et al., &quot;Universal DNA Microarray Method for Multiplex Detection of Low Abundance Point Mutations.&quot; J. Mol. Biol. 292:251-262 (1999)."><meta name="citation_reference" content="Grime, G.W., Non-Silver Photographic Processes, Proceedings of the Symposium on Non-Silver Photographic Processes Held at New College, Oxford, Sep., 1973. &quot;Holographic Diffraction Gratings Recorded in Photoresist,&quot; p. 275-284 (1973)."><meta name="citation_reference" content="Gunderson, K.L., &quot;Mutation Detection by Ligation to Complete n-mer DNA Arrays.&quot; Genome Research 8:1142-1153 (1998)."><meta name="citation_reference" content="Hafeman, D. et al., &quot;Light-Addressable Potentiometric sensor for biochemical systems,&quot; Science, 240:1182-1185 (1988)."><meta name="citation_reference" content="Healey et al., &quot;Development of a Penicillin Biosensor Using a Single Optical Imaging Fiber,&quot; SPIE Proc. 2388:568-573 (1995)."><meta name="citation_reference" content="Healey et al., &quot;Fiberoptic DNA Sensor Array Capable of Detecting Point Mutations,&quot; Analytical Biochemistry, 251:270-279 (1997)."><meta name="citation_reference" content="Healey et al., &quot;Improved Fiber-Optic Chemical Sensor for Penicillin,&quot; Anal. Chem. 67(24):4471-4476 (1995)."><meta name="citation_reference" content="Hirschfeld et al., &quot;Laser-Fiber-Optic &quot;Optrode&quot; for Real Time In Vivo Blood Carbon Level Monitoring,&quot; Journal of Lightwave Technology, LT-5(7):1027-1033 (1987)."><meta name="citation_reference" content="Hirschhorn, J.N., et al., &quot;SBE-TAGS: An Array-Based Method for Efficient Single-Nucleotide Polymorphism Genotyping.&quot; PNAS 97(22): 12164-12169 (2000)."><meta name="citation_reference" content="Hogan, B. et al., &quot;Single-cell Analysis at the Level of a Single Human Erythrocyte,&quot; Trends in Analytical Chemistry 12(1):4-9 (1993)."><meta name="citation_reference" content="Hsuih et al., &quot;Novel, ligation-dependaent PCR assay for detection of Hepatitis C Virus in Serum,&quot; J. of Clinical Microbiology, 34(3): 501-507 (1996)."><meta name="citation_reference" content="Hsuih, T. et al., &quot;Novel, Litigation-Dependent PCR Assay for Detection of Hepatitis C Virus in Serum,&quot; Journal of Clinical Microbiology, Mar. 1996, 34(3): 501-507 (1996)."><meta name="citation_reference" content="Huang, L. et al., &quot;Exploring Single-cell Dynamics Using Chemically-modified Microelectrodes,&quot; Trends in Analytical Chemistry 14(4):158-164 (1995)."><meta name="citation_reference" content="Hubert, C. et al., &quot;Design of a Solvatochromic Polymer-Based Fiber Optics Chemical Sensor for Polar Solvent Detection,&quot; Adv. Mater., 7(11): 914-917 (1995)."><meta name="citation_reference" content="Iannone et al., &quot;Multiplexed Single Nucleotide Polymorphism Genotyping by Oligonucleotide Ligation and Flow Cytometry,&quot; Cytometry, 39:131-140 (2000)."><meta name="citation_reference" content="Illumina Inc. &quot;Emerging,&quot; Windhover&#39;s In Vivo, The Business and Medicine Report, p. 1-2 (Sep. 1998)."><meta name="citation_reference" content="Ince, C. et al., &quot;A Micro-Perfusion Chamber for Single-Cell Fluorescence Measurements,&quot; Journal of Immunological Methods 128:227-234 (1990)."><meta name="citation_reference" content="Jacobs, J. et al., &quot;Combinatorial chemistry-applications of light-directed chemical synthesis,&quot; TIBTECH, 12:19-26 (1994)."><meta name="citation_reference" content="Koop, A. et al., &quot;Continuous Bioluminescent Monitoring of Cytoplasmic ATP in Single Isolated Rat Hepatocytes During Metaboli Poisoning,&quot; Biochem. J. 295:165-170 (1993)."><meta name="citation_reference" content="Lam, K.S., &quot;The &#39;One-Bead-One-Compound&#39; Combinatorial Library Method.&quot; Chem. Rev. 97(2): 411-448 (1997)."><meta name="citation_reference" content="Levy, U. et al., &quot;Direct Picture Transmission in a Single Fiber with Holographic Filters,&quot; Optics Communications, 30(2):163165 (1979)."><meta name="citation_reference" content="Lin et al., &quot;A porous silicon-based optical interferometric biosensor,&quot; Science, 278:840-843 (1997)."><meta name="citation_reference" content="Lin, Z. et al., &quot;Multiplex genotype determination at a large number of gene loci,&quot; Proc. Natl. Acad. Sci. USA, 93: 2582-2587, Mar. 1996."><meta name="citation_reference" content="Lippitsch, M. et al., &quot;Fibre-Optic Oxygen Sensor with the Fluorescence Decay Time as the Information Carrier,&quot; Anal. Chem. Acta., 205:1-6 (1998)."><meta name="citation_reference" content="Lizardi et al., &quot;Mutation detection and single-molecule counting using isothermal rolling-circle amplification,&quot; Nature Genetics, 19: 225-232 (Jul. 1998)."><meta name="citation_reference" content="Lonergan, M.C., &quot;Array-Based Vapor Sensing Using Chemically Sensitive, Polymer Composite Resistors,&quot; IEEE Aerospace Applications Conference Proceedings, 3(8):583-631 (1997)."><meta name="citation_reference" content="Lubbers, D. et al., &quot;Optical Fluorescense Sensors for Continuous Measurement of Chemical Concentrations in Biological Systems,&quot; Sens. Actuators, 4:641-654 (1983)."><meta name="citation_reference" content="Lundstrom, I. et al., &quot;Artificial &#39;olfactory&#39; images from a chemical sensor using a light-impulse technique,&quot; Nature, 352:47-50 (1991)."><meta name="citation_reference" content="Luong, J. et al., &quot;Fluorescence Sensors for Monitoring Bioprocesses,&quot; Practical Fluoro, ed. G.G. Guilbault, 2nd Ed., Marcel Dekker. 775-793 (New York 1990)."><meta name="citation_reference" content="Lyamichev et al., &quot;Polymorphism identification and quantitative detection of genomic DNA by invasive cleavage of oligonucleotide probes,&quot; Nature Biotechnology, 17:292-296 (1999)."><meta name="citation_reference" content="Matthews et al., &quot;Analytical Strategies for the Use of Dna Probes,&quot; Analytical Biochemistry, 169:1-25 (1988)."><meta name="citation_reference" content="McConnell, et al., &quot;The Cytosensor Microphysiometer: Biological Applications of Silicon Technology,&quot; Science 257:1906-1912 (Sep. 1992)."><meta name="citation_reference" content="Metzker, M. et al., &quot;Termination of DNA synthesis by novel 3&#39;-modified-deoxyribonucleoside 5&#39;-triphosphates,&quot; Nucleic Acids Research 22(20):4259-4267 (1994)."><meta name="citation_reference" content="Michael et al., &quot;Fabrication of Micro- and Nanostructures Using Optical Fibers and there Use as Chemical Sensors,&quot; Proc. 3rd Intl. Symp., Microstructures and Microfabricated Systems, ed. P.J. Hesketh, et al., v. 97-5, Electrochem. Soc., 152-157 (Aug. 1997)."><meta name="citation_reference" content="Michael et al., &quot;Making Sensors out of Disarray: Optical Sensor Microarrays,&quot; Proc. SPIE, 3270 34-41 (1998)."><meta name="citation_reference" content="Michael et al., &quot;Randomly Ordered Addressable High-Density Optical Sensor Arrays,&quot; Anal. Chem. 70(7): 1242-1248 (Apr. 1998)."><meta name="citation_reference" content="Mignani, et al, &quot;In-Vivo Biomedical Monitoring by Fiber-Optic Systems,&quot; Journal of Lightwave Technology, 13(7): 1396-1406 (1995)."><meta name="citation_reference" content="Milanovich et al., &quot;Clinical Measurements Using Fiber Optics and Optrodes,&quot; Novel Optical Fiber Techniques for Medical Application, SPIE, 494:1831 (1984)."><meta name="citation_reference" content="Miyawaki, A. et al., &quot;Fluorescent Indicators for Ca&lt;SUP&gt;2+&lt;/SUP&gt; Based on Green Fluorescent Proteins and Calmodulin,&quot; Nature 388:882-887 (Aug. 1997)."><meta name="citation_reference" content="Mrksich, M. et al., &quot;Controlling Cell Attachment of Contoured Surfaces with Self-Assembled Monolayers of Alkanethiolates on Gold,&quot; Proc. Natl. Acad. Sci. USA 93:10775-10778 (Oct. 1996)."><meta name="citation_reference" content="Munkholm, C. et al., &quot;A Fiber-Optic Sensor for CO&lt;SUB&gt;2 &lt;/SUB&gt;Measurement,&quot; Talanta, 35(2):109112 (1988)."><meta name="citation_reference" content="Munkholm, C. et al., &quot;Polymer Modification of Fiber Optic Chemical Sensors as a Method of Enhancing Fluorescense Signal for pH Measurement,&quot; Anal. Chem., 58:14271430 (1986)."><meta name="citation_reference" content="Normie, L. &quot;System Uses Photonics for Early Tumor Detection,&quot; Biophotonics International 24-25 (Sep./Oct. 1996)."><meta name="citation_reference" content="Owicki, J. et al., &quot;Bioassays with a Microphysiometer,&quot; Nature 344:271-272 (Mar. 1990)."><meta name="citation_reference" content="Owicki, J. et al., &quot;Continuous Monitoring of Receptor-Mediated Changes in the Metabolic Rates of Living Cells,&quot; Proc. Natl. Acad. Sci. USA 87:407-4011 (1990)."><meta name="citation_reference" content="Owicki, J. et al., &quot;The Light-Addressable Potentiometric Sensor: Principles and Biological Applications,&quot; Annu. Rev. Biophys, Biomol. Struct. 23:87-113 (1994)."><meta name="citation_reference" content="Pantano et al., &quot;Ordered Nanowell Arrays,&quot; Chem. Mater., 8(12): 2832-2835 (1996)."><meta name="citation_reference" content="Pantano, P. et al., &quot;Analytical Applications of Optical Imaging Fibers,&quot; Anal. Chem., 67:461A-487A (1995)."><meta name="citation_reference" content="Parce, J. et al., &quot;Biosensors for Directly Measuring Cell Affecting Agents,&quot; Ann. Biol. Clin. 48:639-641 (1990)."><meta name="citation_reference" content="Parce, J. et al., &quot;Detection of Cell-Affecting Agents with a Silicon Biosensor,&quot; Science 246:243-247 (Oct. 1989)."><meta name="citation_reference" content="Park, M. et al., &quot;Block Copolymer Lithography: Periodic Arrays of ~10&lt;SUP&gt;11 &lt;/SUP&gt;Holes in 1 Square Centimeter,&quot; Science 276:1401-1404 (May 1997)."><meta name="citation_reference" content="Peterson et al., &quot;Fiber-Optic Sensors for Biomedical Applications,&quot; Science, 13:123-127 (1984)."><meta name="citation_reference" content="Peterson, J. et al., &quot;Fiber Optic pH Probe for Physiological Use,&quot; Anal. Chem., 52:864-869 (1980)."><meta name="citation_reference" content="Piunno et al., &quot;Fiber-Optic DNA Sensor for Fluorometric Nucleic Acid Determination,&quot; Anal. Chem., 67:2635-2643 (1995)."><meta name="citation_reference" content="Plunkett, M. et al., &quot;Combinatorial Chemistry and New Drugs,&quot; Scientific American, Apr. 1997, 69-73."><meta name="citation_reference" content="Pope, E. &quot;Fiber Optic Chemical Microsensors Employing Optically Silica Microspehres,&quot; SPIE, 2388:245-256 (1995)."><meta name="citation_reference" content="Rahmani, et al., &quot;Adaption of the Cellscan Technique for the SCM Test in Breast Cancer,&quot; European J. Cancer 32A(10): 1758-1765 (1996)."><meta name="citation_reference" content="Ronaghi et al., &quot;A Sequencing Method Based on Real-Time Pyrophosphate,&quot; Science, 281:363-365 (1998)."><meta name="citation_reference" content="Shoemaker et al., &quot;Quantitative phenotypic analysis of yeast deletion mutants using a highly parallel molecular bar-coding strategy,&quot; Nature Genetics, 14:450-456 (1996)."><meta name="citation_reference" content="Strachan et al., &quot;A Rapid General Method for the Identification of PCR Products Using a Fibre-Optic Biosensor and its Application to the Detection of Listeria,&quot; Letters in Applied Microbiology, 21:5-9 (1995)."><meta name="citation_reference" content="Vergne et al. Phagosomal pH determination by duel fluorescence flow cytometry. Analytical Biochemistry, vol. 2554(1), pp. 127-132 (Jan. 1, 1998)."><meta name="citation_reference" content="Walt, &quot;Fiber-Optic Sensors for Continuous Clinical Monitoring,&quot; Proc. IEEE, 80(6): 903-911 (1992)."><meta name="citation_reference" content="Walt, D &quot;Fiber Optic Imaging Sensors,&quot; Acc. Chem. Res. 31:267-278 (1998)."><meta name="citation_patent_number" content="US:7060431"><meta name="citation_patent_application_number" content="US:09/344,526"><link rel="canonical" href="http://www.google.com/patents/US7060431"/><meta property="og:url" content="http://www.google.com/patents/US7060431"/><meta name="title" content="Patent US7060431 - Method of making and decoding of array sensors with microspheres"/><meta name="description" content="The invention relates to compositions and methods for decoding microsphere array sensors."/><meta property="og:title" content="Patent US7060431 - Method of making and decoding of array sensors with microspheres"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("VKnpU9XDI9WqsQSTvYCIAg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407464522.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("GBR"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("VKnpU9XDI9WqsQSTvYCIAg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407464522.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("GBR"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7060431?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7060431"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=ZAd0BAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7060431&amp;usg=AFQjCNF8mCTf_IZVtCTNwhOx6l278fAhqw" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7060431.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7060431.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20020132221"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7060431"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7060431" style="display:none"><span itemprop="description">The invention relates to compositions and methods for decoding microsphere array sensors....</span><span itemprop="url">http://www.google.com/patents/US7060431?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7060431 - Method of making and decoding of array sensors with microspheres</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7060431 - Method of making and decoding of array sensors with microspheres" title="Patent US7060431 - Method of making and decoding of array sensors with microspheres"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7060431 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/344,526</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jun 13, 2006</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jun 24, 1999</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Jun 24, 1998</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2335951A1">CA2335951A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2335951C">CA2335951C</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69940430D1">DE69940430D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1090293A2">EP1090293A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1090293B1">EP1090293B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2045334A1">EP2045334A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2360271A1">EP2360271A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7033754">US7033754</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7226734">US7226734</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7455971">US7455971</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8460865">US8460865</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20020132221">US20020132221</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030157504">US20030157504</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20050233318">US20050233318</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20060073513">US20060073513</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20070231824">US20070231824</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20130310280">US20130310280</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1999067641A2">WO1999067641A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1999067641A3">WO1999067641A3</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09344526, </span><span class="patent-bibdata-value">344526, </span><span class="patent-bibdata-value">US 7060431 B2, </span><span class="patent-bibdata-value">US 7060431B2, </span><span class="patent-bibdata-value">US-B2-7060431, </span><span class="patent-bibdata-value">US7060431 B2, </span><span class="patent-bibdata-value">US7060431B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Mark+S.+Chee%22">Mark S. Chee</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22John+R.+Stuelpnagel%22">John R. Stuelpnagel</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Anthony+W.+Czarnik%22">Anthony W. Czarnik</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Illumina,+Inc.%22">Illumina, Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7060431.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7060431.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7060431.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (104),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (99),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (66),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (58),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (7)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7060431&usg=AFQjCNEidsFf_N5iPeunnI5vsRua_-xtew">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7060431&usg=AFQjCNGBuVzlV3sj7-BO_GrDiLTLbthm2w">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7060431B2%26KC%3DB2%26FT%3DD&usg=AFQjCNFVaF1cjHtrYKaK5Epfrro-2ROoWA">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT55580645" lang="EN" load-source="patent-office">Method of making and decoding of array sensors with microspheres</invention-title></span><br><span class="patent-number">US 7060431 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50992782" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">The invention relates to compositions and methods for decoding microsphere array sensors.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(25)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-0.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-0.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-14.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-14.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-15.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-15.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-16.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-16.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-17.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-17.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-18.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-18.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-19.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-19.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-20.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-20.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-21.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-21.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-22.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-22.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-23.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-23.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US7060431-24.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US7060431-24.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(76)</span></span></div><div class="patent-text"><div mxw-id="PCLM9031677" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A method of making a bead array comprising:
<div class="claim-text">a) contacting a substrate with a surface comprising discrete sites at a density of at least 100 sites per 1 mm<sup>2</sup>, with a solution comprising a population of different beads, wherein said beads do not comprise an optical signature; and</div>
<div class="claim-text">b) applying energy to said substrate or said solution, or both, such that at least a subpopulation of said different beads randomly associate onto sites.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. A method according to <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein said discrete sites comprise wells.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. A method according to <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein said energy is in the form of agitation.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. A method according to <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said energy is dipping said substrate into said beads.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. A method according to <claim-ref idref="CLM-00004">claim 4</claim-ref>, wherein said substrate is a fiber optic bundle.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. A method of determining the presence of a target analyte in a sample comprising:
<div class="claim-text">a) contacting said sample with an array comprising:
<div class="claim-text">i) substrate with a surface comprising discrete sites at a density of at least 100 sites per 1 mm<sup>2</sup>, wherein said sites are wells; and</div>
<div class="claim-text">ii) a population of microspheres randomly distributed on said sites, wherein said population comprises at least a first and a second subpopulation each comprising:
<div class="claim-text">(a) a different bioactive agent; and</div>
<div class="claim-text">(b) a different identifier binding ligand;</div>
</div>
</div>
<div class="claim-text">b) determining the presence or absence of said target analyte; and</div>
<div class="claim-text">c) decoding a location of said bioactive agent by correlating said bioactive agent with said location, whereby said decoding comprises contacting said array with at least first and second different decoder binding ligands, whereby said first and second decoder binding ligands bind to a first and second identifier binding ligands, whereby said first and second identifier binding ligands identify said first and second bioactive agents, respectively, to thereby identify a location of said first and second bioactive agents to thereby decode said array.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. The method according to <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein said first and second bioactive agents comprise nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. The method according to <claim-ref idref="CLM-00007">claim 7</claim-ref>, wherein said nucleic acids comprise DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The method according to <claim-ref idref="CLM-00007">claim 7</claim-ref>, wherein said nucleic acids comprise single stranded nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The method according to <claim-ref idref="CLM-00007">claim 7</claim-ref>, wherein said nucleic acids comprise double stranded nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. The method according to <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein said first and second bioactive agents comprise proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. The method according to <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein said substrate comprises a fiber optic bundle.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00013" num="00013" class="claim">
      <div class="claim-text">13. The method according to <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said population of different beads comprises at least a first and second subpopulation.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00014" num="00014" class="claim">
      <div class="claim-text">14. The method according to <claim-ref idref="CLM-00013">claim 13</claim-ref>, wherein said first and second subpopulations each comprise a different bioactive agent.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00015" num="00015" class="claim">
      <div class="claim-text">15. The method according to <claim-ref idref="CLM-00014">claim 14</claim-ref>, wherein said bioactive agent comprises a protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00016" num="00016" class="claim">
      <div class="claim-text">16. The method according to <claim-ref idref="CLM-00014">claim 14</claim-ref>, wherein said bioactive agent comprises a nucleic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00017" num="00017" class="claim">
      <div class="claim-text">17. The method according to <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein said nucleic acid comprises DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00018" num="00018" class="claim">
      <div class="claim-text">18. The method according to <claim-ref idref="CLM-00014">claim 14</claim-ref>, wherein said first and second subpopulations of beads each comprise a different identifier binding ligand.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00019" num="00019" class="claim">
      <div class="claim-text">19. The method according to <claim-ref idref="CLM-00018">claim 18</claim-ref>, wherein said identifier binding ligand comprises a protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00020" num="00020" class="claim">
      <div class="claim-text">20. The method according to <claim-ref idref="CLM-00018">claim 18</claim-ref>, wherein said identifier binding ligand comprises a nucleic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00021" num="00021" class="claim">
      <div class="claim-text">21. The method according to <claim-ref idref="CLM-00020">claim 20</claim-ref>, wherein said bioactive agent and said identifier binding ligand within the same subpopulation each comprise an identical nucleic acid sequence for binding a decoder binding ligand.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00022" num="00022" class="claim">
      <div class="claim-text">22. The method according to <claim-ref idref="CLM-00018">claim 18</claim-ref> further comprising decoding a location of said bioactive agent on said array by correlating said bioactive agent with said location, whereby said decoding comprises contacting said array with at least first and second different decoder binding ligands, whereby said first and second decoder binding ligands bind to first and second identifier binding ligands, whereby said first and second identifier binding ligands identify said first and second bioactive agents, respectively, to thereby identify a location of said first and second bioactive agents to thereby decode said array.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00023" num="00023" class="claim">
      <div class="claim-text">23. The method according to <claim-ref idref="CLM-00022">claim 22</claim-ref>, wherein said decoder binding ligands comprise proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00024" num="00024" class="claim">
      <div class="claim-text">24. The method according to <claim-ref idref="CLM-00022">claim 22</claim-ref>, wherein said decoder binding ligands comprise nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00025" num="00025" class="claim">
      <div class="claim-text">25. The method according to <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said substrate comprises a flat planar substrate.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00026" num="00026" class="claim">
      <div class="claim-text">26. The method according to <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein said identifier binding ligands comprise proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00027" num="00027" class="claim">
      <div class="claim-text">27. The method according to <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein said identifier binding ligands comprise nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00028" num="00028" class="claim">
      <div class="claim-text">28. The method according to <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein comprise decoder binding ligands are proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00029" num="00029" class="claim">
      <div class="claim-text">29. The method according to <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein comprise decoder binding ligands are nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00030" num="00030" class="claim">
      <div class="claim-text">30. The method according to <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein said substrate comprises a flat planar substrate.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00031" num="00031" class="claim">
      <div class="claim-text">31. A method of determining the presence of a target analyte in a sample comprising:
<div class="claim-text">a) contacting said sample with an array comprising:
<div class="claim-text">i) a substrate with a surface comprising sites at a density of at least 100 sites per 1 mm<sup>2</sup>, wherein said sites are present in a depression; and</div>
<div class="claim-text">ii) a population of particles randomly distributed on said sites, wherein said population comprises at least a first and a second subpopulation each comprising:
<div class="claim-text">(a) a different bioactive agent; and</div>
<div class="claim-text">(b) a different identifier binding ligand;</div>
</div>
</div>
<div class="claim-text">b) decoding a location of said bioactive agent by correlating said bioactive agent with said location, whereby said decoding comprises contacting said array with at least first and second different decoder binding ligands, whereby said first and second decoder binding ligands bind to a first and second identifier binding ligands, whereby said first and second identifier binding ligands identify said first and second bioactive agents, respectively, to thereby identify a location of said first and second bioactive agents to thereby decode said array; and</div>
<div class="claim-text">c) determining the presence or absence of said target analyte.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00032" num="00032" class="claim">
      <div class="claim-text">32. The method according to <claim-ref idref="CLM-00031">claim 31</claim-ref>, wherein said bioactive agents comprise nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00033" num="00033" class="claim">
      <div class="claim-text">33. The method according to <claim-ref idref="CLM-00032">claim 32</claim-ref>, wherein said nucleic acids comprise DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00034" num="00034" class="claim">
      <div class="claim-text">34. The method according to <claim-ref idref="CLM-00032">claim 32</claim-ref>, wherein said nucleic acids comprise single stranded nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00035" num="00035" class="claim">
      <div class="claim-text">35. The method according to <claim-ref idref="CLM-00032">claim 32</claim-ref>, wherein said nucleic acids comprise double stranded nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00036" num="00036" class="claim">
      <div class="claim-text">36. The method according to <claim-ref idref="CLM-00031">claim 31</claim-ref>, wherein said bioactive agents comprise proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00037" num="00037" class="claim">
      <div class="claim-text">37. The method according to <claim-ref idref="CLM-00031">claim 31</claim-ref>, wherein said identifier binding ligands comprise proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00038" num="00038" class="claim">
      <div class="claim-text">38. The method according to <claim-ref idref="CLM-00031">claim 31</claim-ref>, wherein said identifier binding ligands comprise nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00039" num="00039" class="claim">
      <div class="claim-text">39. The method according to <claim-ref idref="CLM-00031">claim 31</claim-ref>, wherein said decoder binding ligands comprise proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00040" num="00040" class="claim">
      <div class="claim-text">40. The method according to <claim-ref idref="CLM-00031">claim 31</claim-ref>, wherein said decoder binding ligands comprise nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00041" num="00041" class="claim">
      <div class="claim-text">41. The method according to <claim-ref idref="CLM-00031">claim 31</claim-ref>, wherein said substrate comprises a fiber optic bundle.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00042" num="00042" class="claim">
      <div class="claim-text">42. The method according to <claim-ref idref="CLM-00031">claim 31</claim-ref>, wherein said substrate comprises a flat planar substrate.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00043" num="00043" class="claim">
      <div class="claim-text">43. The method according to <claim-ref idref="CLM-00031">claim 31</claim-ref>, wherein said particles are beads.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00044" num="00044" class="claim">
      <div class="claim-text">44. The method according to <claim-ref idref="CLM-00031">claim 31</claim-ref>, wherein said sites are contiguous.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00045" num="00045" class="claim">
      <div class="claim-text">45. A method of determining the presence of a target analyte in a sample comprising:
<div class="claim-text">a) contacting said sample with an array comprising:
<div class="claim-text">i) a substrate with a surface comprising sites at a density of at least 100 sites per 1 mm<sup>2</sup>, wherein said sites comprise wells; and</div>
<div class="claim-text">ii) a population of particles randomly distributed on said sites, wherein said population comprises at least a first and a second subpopulation each comprising:
<div class="claim-text">(a) a different bioactive agent; and</div>
<div class="claim-text">(b) a different identifier binding ligand;</div>
</div>
</div>
<div class="claim-text">b) decoding a location of said bioactive agent by correlating said bioactive agent with said location, whereby said decoding comprises contacting said array with at least first and second different decoder binding ligands, whereby said first and second decoder binding ligands bind to a first and second identifier binding ligands, whereby said first and second identifier binding ligands identify said first and second bioactive agents, respectively, to thereby identify a location of said first and second bioactive agents to thereby decode said array; and</div>
<div class="claim-text">c) determining the presence or absence of said target analyte.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00046" num="00046" class="claim">
      <div class="claim-text">46. The method according to <claim-ref idref="CLM-00045">claim 45</claim-ref>, wherein said sites are discrete sites.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00047" num="00047" class="claim">
      <div class="claim-text">47. The method according to <claim-ref idref="CLM-00045">claim 45</claim-ref>, wherein said bioactive agents comprise nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00048" num="00048" class="claim">
      <div class="claim-text">48. The method according to <claim-ref idref="CLM-00047">claim 47</claim-ref>, wherein said nucleic acids comprise DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00049" num="00049" class="claim">
      <div class="claim-text">49. The method according to <claim-ref idref="CLM-00047">claim 47</claim-ref>, wherein said nucleic acids comprise single stranded nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00050" num="00050" class="claim">
      <div class="claim-text">50. The method according to <claim-ref idref="CLM-00047">claim 47</claim-ref>, wherein said nucleic acids comprise double stranded nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00051" num="00051" class="claim">
      <div class="claim-text">51. The method according to <claim-ref idref="CLM-00045">claim 45</claim-ref>, wherein said bioactive agents comprise proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00052" num="00052" class="claim">
      <div class="claim-text">52. The method according to <claim-ref idref="CLM-00045">claim 45</claim-ref>, wherein said identifier binding ligands comprise proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00053" num="00053" class="claim">
      <div class="claim-text">53. The method according to <claim-ref idref="CLM-00045">claim 45</claim-ref>, wherein said identifier binding ligands comprise nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00054" num="00054" class="claim">
      <div class="claim-text">54. The method according to <claim-ref idref="CLM-00045">claim 45</claim-ref>, wherein said decoder binding ligands comprise proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00055" num="00055" class="claim">
      <div class="claim-text">55. The method according to <claim-ref idref="CLM-00045">claim 45</claim-ref>, wherein said decoder binding ligands comprise nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00056" num="00056" class="claim">
      <div class="claim-text">56. The method according to <claim-ref idref="CLM-00045">claim 45</claim-ref>, wherein said substrate comprises a fiber optic bundle.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00057" num="00057" class="claim">
      <div class="claim-text">57. The method according to <claim-ref idref="CLM-00045">claim 45</claim-ref>, wherein said substrate comprises a flat planar substrate.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00058" num="00058" class="claim">
      <div class="claim-text">58. The method according to <claim-ref idref="CLM-00045">claim 45</claim-ref>, wherein said particles are beads.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00059" num="00059" class="claim">
      <div class="claim-text">59. A method of making a particle array comprising:
<div class="claim-text">a) contacting a substrate with a surface comprising sites at a density of at least 100 sites per 1 mm<sup>2</sup>, with a solution comprising a population of different particles, wherein said particles do not comprise an optical signature, and wherein said sites comprise wells; and</div>
<div class="claim-text">b) applying energy to said substrate or said solution, or both, such that at least a subpopulation of said different particles randomly associate onto sites.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00060" num="00060" class="claim">
      <div class="claim-text">60. The method according to <claim-ref idref="CLM-00059">claim 59</claim-ref>, wherein said sites are discrete sites.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00061" num="00061" class="claim">
      <div class="claim-text">61. The method according to <claim-ref idref="CLM-00059">claim 59</claim-ref>, wherein said energy is in the form of agitation.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00062" num="00062" class="claim">
      <div class="claim-text">62. The method according to <claim-ref idref="CLM-00059">claim 59</claim-ref>, wherein said energy is dipping said substrate into said particles.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00063" num="00063" class="claim">
      <div class="claim-text">63. The method according to <claim-ref idref="CLM-00059">claim 59</claim-ref>, wherein said substrate is a fiber optic bundle.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00064" num="00064" class="claim">
      <div class="claim-text">64. The method according to <claim-ref idref="CLM-00059">claim 59</claim-ref>, wherein said population of different particles comprises at least a first and second subpopulation.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00065" num="00065" class="claim">
      <div class="claim-text">65. The method according to <claim-ref idref="CLM-00064">claim 64</claim-ref>, wherein said first and second subpopulations each comprise a different bioactive agent.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00066" num="00066" class="claim">
      <div class="claim-text">66. The method according to <claim-ref idref="CLM-00065">claim 65</claim-ref>, wherein said bioactive agent comprises a protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00067" num="00067" class="claim">
      <div class="claim-text">67. The method according to <claim-ref idref="CLM-00065">claim 65</claim-ref>, wherein said bioactive agent comprises a nucleic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00068" num="00068" class="claim">
      <div class="claim-text">68. The method according to <claim-ref idref="CLM-00067">claim 67</claim-ref>, wherein said nucleic acid comprises DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00069" num="00069" class="claim">
      <div class="claim-text">69. The method according to <claim-ref idref="CLM-00065">claim 65</claim-ref>, wherein said first and second subpopulations of particles each comprise a different identifier binding ligand.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00070" num="00070" class="claim">
      <div class="claim-text">70. The method according to <claim-ref idref="CLM-00069">claim 69</claim-ref>, wherein said identifier binding ligand comprises a protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00071" num="00071" class="claim">
      <div class="claim-text">71. The method according to <claim-ref idref="CLM-00069">claim 69</claim-ref>, wherein said identifier binding ligand comprises a nucleic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00072" num="00072" class="claim">
      <div class="claim-text">72. The method according to <claim-ref idref="CLM-00071">claim 71</claim-ref>, wherein said bioactive agent and said identifier binding ligand within the same subpopulation each comprise an identical nucleic acid sequence for binding a decoder binding ligand.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00073" num="00073" class="claim">
      <div class="claim-text">73. The method according to <claim-ref idref="CLM-00069">claim 69</claim-ref> further comprising decoding a location of said bioactive agent on said array by correlating said bioactive agent with said location, whereby said decoding comprises contacting said array with at least first and second different decoder binding ligands, whereby said first and second decoder binding ligands bind to first and second identifier binding ligands, whereby said first and second identifier binding ligands identify said first and second bioactive agents, respectively, to thereby identify a location of said first and second bioactive agents to thereby decode said array.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00074" num="00074" class="claim">
      <div class="claim-text">74. The method according to <claim-ref idref="CLM-00073">claim 73</claim-ref>, wherein said decoder binding ligands comprise proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00075" num="00075" class="claim">
      <div class="claim-text">75. The method according to <claim-ref idref="CLM-00073">claim 73</claim-ref>, wherein said decoder binding ligands comprise nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00076" num="00076" class="claim">
      <div class="claim-text">76. The method according to <claim-ref idref="CLM-00059">claim 59</claim-ref>, wherein said substrate comprises a flat planar substrate. </div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES16054221" lang="EN" load-source="patent-office" class="description">
    <p num="p-0002">This application is a continuing application of U.S. Ser. Nos. 09/189,543, filed Nov. 10, 1998, and of 60/090,473, filed Jun. 24, 1998.</p>
    <heading>FIELD OF THE INVENTION</heading> <p num="p-0003">The invention relates to compositions and methods for decoding microsphere array sensors.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p num="p-0004">There are a number of assays and sensors for the detection of the presence and/or concentration of specific substances in fluids and gases. Many of these rely on specific ligand/antiligand reactions as the mechanism of detection. That is, pairs of substances (i.e. the binding pairs or ligand/antiligands) are known to bind to each other, while binding little or not at all to other substances. This has been the focus of a number of techniques that utilize these binding pairs for the detection of the complexes. These generally are done by labeling one component of the complex in some way, so as to make the entire complex detectable, using, for example, radioisotopes, fluorescent and other optically active molecules, enzymes, etc.</p>
    <p num="p-0005">Of particular use in these sensors are detection mechanisms utilizing luminescence. Recently, the use of optical fibers and optical fiber strands in combination with light absorbing dyes for chemical analytical determinations has undergone rapid development, particularly within the last decade. The use of optical fibers for such purposes and techniques is described by Milanovich et al., “Novel Optical Fiber Techniques For Medical Application”, Proceedings of the SPIE 28th Annual International Technical Symposium On Optics and Electro-Optics, Volume 494, 1980; Seitz, W. R., “Chemical Sensors Based On Immobilized Indicators and Fiber Optics” in <i>C.R.C. Critical Reviews In Analytical Chemistry</i>, Vol. 19, 1988, pp. 135–173; Wolfbeis, O. S., “Fiber Optical Fluorosensors In Analytical Chemistry” in <i>Molecular Luminescence Spectroscopy, Methods and Applications </i>(S. G. Schulman, editor), Wiley &amp; Sons, New York (1988); Angel, S. M., <i>Spectroscopy </i>2(4):38 (1987); Walt, et al., “Chemical Sensors and Microinstrumentation”, <i>ACS Symposium Series</i>, Vol. 403, 1989, p. 252, and Wolfbeis, O. S., <i>Fiber Optic Chemical Sensors</i>, Ed. CRC Press, Boca Raton, Fla., 1991, 2nd Volume.</p>
    <p num="p-0006">When using an optical fiber in an in vitro/in vivo sensor, one or more light absorbing dyes are located near its distal end. Typically, light from an appropriate source is used to illuminate the dyes through the fiber's proximal end. The light propagates along the length of the optical fiber; and a portion of this propagated light exits the distal end and is absorbed by the dyes. The light absorbing dye may or may not be immobilized; may or may not be directly attached to the optical fiber itself; may or may not be suspended in a fluid sample containing one or more analytes of interest; and may or may not be retainable for subsequent use in a second optical determination.</p>
    <p num="p-0007">Once the light has been absorbed by the dye, some light of varying wavelength and intensity returns, conveyed through either the same fiber or collection fiber(s) to a detection system where it is observed and measured. The interactions between the light conveyed by the optical fiber and the properties of the light absorbing dye provide an optical basis for both qualitative and quantitative determinations.</p>
    <p num="p-0008">Of the many different classes of light absorbing dyes which conventionally are employed with bundles of fiber strands and optical fibers for different analytical purposes are those more common compositions that emit light after absorption termed “fluorophores” and those which absorb light and internally convert the absorbed light to heat, rather than emit it as light, termed “chromophores.”</p>
    <p num="p-0009">Fluorescence is a physical phenomenon based upon the ability of some molecules to absorb light (photons) at specified wavelengths and then emit light of a longer wavelength and at a lower energy. Substances able to fluoresce share a number of common characteristics: the ability to absorb light energy at one wavelength λ<sub>ab</sub>; reach an excited energy state; and subsequently emit light at another light wavelength, λ<sub>em</sub>. The absorption and fluorescence emission spectra are individual for each fluorophore and are often graphically represented as two separate curves that are slightly overlapping. The same fluorescence emission spectrum is generally observed irrespective of the wavelength of the exciting light and, accordingly, the wavelength and energy of the exciting light may be varied within limits; but the light emitted by the fluorophore will always provide the same emission spectrum. Finally, the strength of the fluorescence signal may be measured as the quantum yield of light emitted. The fluorescence quantum yield is the ratio of the number of photons emitted in comparison to the number of photons initially absorbed by the fluorophore. For more detailed information regarding each of these characteristics, the following references are recommended: Lakowicz, J. R., <i>Principles of Fluorescence Spectroscopy</i>, Plenum Press, New York, 1983; Freifelder, D., Physical Biochemistry, second edition, W. H. Freeman and Company, New York, 1982; “Molecular Luminescence Spectroscopy Methods and Applications: Part I” (S. G. Schulman, editor) in <i>Chemical Analysis</i>, vol. 77, Wiley &amp; Sons, Inc., 1985; <i>The Theory of Luminescence</i>, Stepanov and Gribkovskii, IIiffe Books, Ltd., London, 1968.</p>
    <p num="p-0010">In comparison, substances which absorb light and do not fluoresce usually convert the light into heat or kinetic energy. The ability to internally convert the absorbed light identifies the dye as a “chromophore.” Dyes which absorb light energy as chromophores do so at individual wavelengths of energy and are characterized by a distinctive molar absorption coefficient at that wavelength. Chemical analysis employing fiber optic strands and absorption spectroscopy using visible and ultraviolet light wavelengths in combination with the absorption coefficient allow for the determination of concentration for specific analyses of interest by spectral measurement. The most common use of absorbance measurement via optical fibers is to determine concentration which is calculated in accordance with Beers' law; accordingly, at a single absorbance wavelength, the greater the quantity of the composition which absorbs light energy at a given wavelength, the greater the optical density for the sample. In this way, the total quantity of light absorbed directly correlates with the quantity of the composition in the sample.</p>
    <p num="p-0011">Many of the recent improvements employing optical fiber sensors in both qualitative and quantitative analytical determinations concern the desirability of depositing and/or immobilizing various light absorbing dyes at the distal end of the optical fiber. In this manner, a variety of different optical fiber chemical sensors and methods have been reported for specific analytical determinations and applications such as pH measurement, oxygen detection, and carbon dioxide analyses. These developments are exemplified by the following publications: Freeman, et al., <i>Anal Chem. </i>53:98 (1983); Lippitsch et al., <i>Anal. Chem. Acta. </i>205:1, (1988); Wolfbeis et al., <i>Anal. Chem. </i>60:2028 (1988); Jordan, et al., <i>Anal. Chem. </i>59:437 (1987); Lubbers et al., <i>Sens. Actuators </i>1983; Munkholm et al., <i>Talanta </i>35:109 (1988); Munkholm et al., <i>Anal. Chem. </i>58:1427 (1986); Seitz, W. R., <i>Anal. Chem. </i>56:16A–34A (1984); Peterson, et al., <i>Anal. Chem. </i>52:864 (1980): Saari, et al., <i>Anal. Chem. </i>54:821 (1982); Saari, et al., <i>Anal. Chem. </i>55:667 (1983); Zhujun et al., <i>Anal. Chem. Acta. </i>160:47 (1984); Schwab, et al., <i>Anal. Chem. </i>56:2199 (1984); Wolfbeis, O. S., “Fiber Optic Chemical Sensors”, Ed. CRC Press, Boca Raton, Fla., 1991, 2nd Volume; and Pantano, P., Walt, D. R., <i>Anal. Chem., </i>481A–487A, Vol. 67, (1995).</p>
    <p num="p-0012">More recently, fiber optic sensors have been constructed that permit the use of multiple dyes with a single, discrete fiber optic bundle. U.S. Pat. Nos. 5,244,636 and 5,250,264 to Walt, et al. disclose systems for affixing multiple, different dyes on the distal end of the bundle, the teachings of each of these patents being incorporated herein by this reference. The disclosed configurations enable separate optical fibers of the bundle to optically access individual dyes. This avoids the problem of deconvolving the separate signals in the returning light from each dye, which arises when the signals from two or more dyes are combined, each dye being sensitive to a different analyte, and there is significant overlap in the dyes' emission spectra.</p>
    <p num="p-0013">U.S. Ser. Nos. 08/818,199 and 09/151,877 describe array compositions that utilize microspheres or beads on a surface of a substrate, for example on a terminal end of a fiber optic bundle, with each individual fiber comprising a bead containing an optical signature. Since the beads go down randomly, a unique optical signature is needed to “decode” the array; i.e. after the array is made, a correlation of the location of an individual site on the array with the bead or bioactive agent at that particular site can be made. This means that the beads may be randomly distributed on the array, a fast and inexpensive process as compared to either the in situ synthesis or spotting techniques of the prior art. Once the array is loaded with the beads, the array can be decoded, or can be used, with full or partial decoding occurring after testing, as is more fully outlined below.</p>
    <p num="p-0014">One drawback with the previous system is that it requires a set of unique optical signatures. While large sets of such signatures are available, for example by using different ratios of different dyes, it would be preferable to use decoding systems that do not rely on the use of sets of optical signatures. Accordingly, it is an object of the invention to provide methods to allow decoding of bead arrays without relying solely on unique optical signatures.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0015">In accordance with the above objects, the present invention provides array compositions comprising a substrate with a surface comprising discrete sites. The composition further comprises a population of microspheres comprising at least a first and a second subpopulation; each subpopulation comprises a bioactive agent; and an identifier binding ligand that will bind a decoder binding ligand such that the identity of the bioactive agent can be elucidated. The microspheres are distributed on the surface.</p>
    <p num="p-0016">In a further aspect, the present invention provides array compositions comprising a substrate with a surface comprising discrete sites, and a population of microspheres comprising at least a first and a second subpopulation. Each subpopulation comprises a bioactive agent and does not comprise an optical signature.</p>
    <p num="p-0017">In an additional aspect, the present invention provides methods of making an array composition as outlined above. The methods comprise forming a surface comprising individual sites on a substrate and distributing microspheres on said surface such that said individual sites contain microspheres. The microspheres comprise at least a first and a second subpopulation each comprising a bioactive agent and do not comprise an optical signature.</p>
    <p num="p-0018">In a further aspect, the invention provides methods of making a composition comprising forming a surface comprising individual sites on a substrate and distributing microspheres on the surface such that the individual sites contain microspheres. The microspheres comprise at least a first and a second subpopulation each comprising a bioactive agent and an identifier binding ligand that will bind a decoder binding ligand such that the identification of the bioactive agent can be elucidated.</p>
    <p num="p-0019">In an additional aspect, the invention provides methods of decoding an array composition comprising providing an array composition as outlined above, and adding a plurality of decoding binding ligands to the array composition to identify the location of at least a plurality of the bioactive agents.</p>
    <p num="p-0020">In a further aspect, the invention provides methods of determining the presence of a target analyte in a sample. The methods comprise contacting the sample with an array composition as outlined herein, and determining the presence or absence of the target analyte.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p num="p-0021">The present invention is generally based on previous work comprising a bead-based analytic chemistry system in which beads, also termed microspheres, carrying different chemical functionalities are distributed on a substrate comprising a patterned surface of discrete sites that can bind the individual microspheres. Since the beads are generally put onto the substrate randomly, the previous work relied on the incorporation of unique optical signatures, generally fluorescent dyes, that could be used to identify the chemical functionality on any particular bead. This allows the synthesis of the candidate agents (i.e. compounds such as nucleic acids and antibodies) to be divorced from their placement on an array, i.e. the candidate agents may be synthesized on the beads, and then the beads are randomly distributed on a patterned surface. Since the beads are first coded with an optical signature, this means that the array can later be “decoded”, i.e. after the array is made, a correlation of the location of an individual site on the array with the bead or candidate agent at that particular site can be made. This means that the beads may be randomly distributed on the array, a fast and inexpensive process as compared to either the in situ synthesis or spotting techniques of the prior art.</p>
    <p num="p-0022">However, the drawback to these methods is that for a large array, the system requires a large number of different optical signatures, which may be difficult or time-consuming to utilize. Accordingly, the present invention provides several improvements over these methods, generally directed to methods of coding and decoding the arrays. That is, as will be appreciated by those in the art, the placement of the bioactive agents is generally random, and thus a coding/decoding system is required to identify the bioactive agent at each location in the array. This may be done in a variety of ways, as is more fully outlined below, and generally includes: a) the use of decoding binding ligands (DBLs), generally directly labeled, that binds to either the bioactive agent or to identifier binding ligands (IBLs) attached to the beads; b) positional decoding, for example by either targeting the placement of beads (for example by using photoactivatible or photocleavable moieties to allow the selective addition of beads to particular locations), or by using either sub-bundles or selective loading of the sites, as are more fully outlined below; c) selective decoding, wherein only those beads that bind to a target are decoded; or d) combinations of any of these. In some cases, as is more fully outlined below, this decoding may occur for all the beads, or only for those that bind a particular target analyte. Similarly, this may occur either prior to or after addition of a target analyte.</p>
    <p num="p-0023">Once the identity of the bioactive agent and its location in the array has been fixed, the array is exposed to samples containing the target analytes, although as outlined below, this can be done prior to or during the analysis as well. The target analytes will bind to the bioactive agents as is more fully outlined below, and results in a change in the optical signal of a particular bead.</p>
    <p num="p-0024">In the present invention, “decoding” does not rely on the use of optical signatures, but rather on the use of decoding binding ligands that are added during a decoding step. The decoding binding ligands will bind either to a distinct identifier binding ligand partner that is placed on the beads, or to the bioactive agent itself, for example when the beads comprise single-stranded nucleic acids as the bioactive agents. The decoding binding ligands are either directly or indirectly labeled, and thus decoding occurs by detecting the presence of the label. By using pools of decoding binding ligands in a sequential fashion, it is possible to greatly minimize the number of required decoding steps.</p>
    <p num="p-0025">Accordingly, the present invention provides array compositions comprising at least a first substrate with a surface comprising individual sites. By “array” herein is meant a plurality of candidate agents in an array format; the size of the array will depend on the composition and end use of the array. Arrays containing from about 2 different bioactive agents (i.e. different beads) to many millions can be made, with very large fiber optic arrays being possible. Generally, the array will comprise from two to as many as a billion or more, depending on the size of the beads and the substrate, as well as the end use of the array, thus very high density, high density, moderate density, low density and very low density arrays may be made. Preferred ranges for very high density arrays are from about 10,000,000 to about 2,000,000,000 (all numbers are per square cm), with from about 100,000,000 to about 1,000,000,000 being preferred. High density arrays range about 100,000 to about 10,000,000, with from about 1,000,000 to about 5,000,000 being particularly preferred. Moderate density arrays range from about 10,000 to about 100,000 being particularly preferred, and from about 20,000 to about 50,000 being especially preferred. Low density arrays are generally less than 10,000, with from about 1,000 to about 5,000 being preferred. Very low density arrays are less than 1,000, with from about 10 to about 1000 being preferred, and from about 100 to about 500 being particularly preferred. In some embodiments, the compositions of the invention may not be in array format; that is, for some embodiments, compositions comprising a single bioactive agent may be made as well. In addition, in some arrays, multiple substrates may be used, either of different or identical compositions. Thus for example, large arrays may comprise a plurality of smaller substrates.</p>
    <p num="p-0026">In addition, one advantage of the present compositions is that particularly through the use of fiber optic technology, extremely high density arrays can be made. Thus for example, because beads of 200 μm or less (with beads of 200 nm possible) can be used, and very small fibers are known, it is possible to have as many as 250,000 or more (in some instances, 1 million) different fibers and beads in a 1 mm<sup>2 </sup>fiber optic bundle, with densities of greater than 15,000,000 individual beads and fibers (again, in some instances as many as 25–50 million) per 0.5 cm<sup>2 </sup>obtainable.</p>
    <p num="p-0027">By “substrate” or “solid support” or other grammatical equivalents herein is meant any material that can be modified to contain discrete individual sites appropriate for the attachment or association of beads and is amenable to at least one detection method. As will be appreciated by those in the art, the number of possible substrates is very large. Possible substrates include, but are not limited to, glass and modified or functionalized glass, plastics (including acrylics, polystyrene and copolymers of styrene and other materials, polypropylene, polyethylene, polybutylene, polyurethanes, Teflon, etc.), polysaccharides, nylon or nitrocellulose, resins, silica or silica-based materials including silicon and modified silicon, carbon, metals, inorganic glasses, plastics, optical fiber bundles, and a variety of other polymers. In general, the substrates allow optical detection and do not themselves appreciably fluoresce.</p>
    <p num="p-0028">Generally the substrate is flat (planar), although as will be appreciated by those in the art, other configurations of substrates may be used as well; for example, three dimensional configurations can be used, for example by embedding the beads in a porous block of plastic that allows sample access to the beads and using a confocal microscope for detection. Similarly, the beads may be placed on the inside surface of a tube, for flow-through sample analysis to minimize sample volume. Preferred substrates include optical fiber bundles as discussed below, and flat planar substrates such as glass, polystyrene and other plastics and acrylics.</p>
    <p num="p-0029">In a preferred embodiment, the substrate is an optical fiber bundle or array, as is generally described in U.S. Ser. Nos. 08/944,850 and 08/519,062, PCT U.S. Pat. No. 98/05025, and PCT U.S. Pat. No. 98/09163, all of which are expressly incorporated herein by reference. Preferred embodiments utilize preformed unitary fiber optic arrays. By “preformed unitary fiber optic array” herein is meant an array of discrete individual fiber optic strands that are co-axially disposed and joined along their lengths. The fiber strands are generally individually clad. However, one thing that distinguished a preformed unitary array from other fiber optic formats is that the fibers are not individually physically manipulatable; that is, one strand generally cannot be physically separated at any point along its length from another fiber strand.</p>
    <p num="p-0030">At least one surface of the substrate is modified to contain discrete, individual sites for later association of microspheres. These sites may comprise physically altered sites, i.e. physical configurations such as wells or small depressions in the substrate that can retain the beads, such that a microsphere can rest in the well, or the use of other forces (magnetic or compressive), or chemically altered or active sites, such as chemically functionalized sites, electrostatically altered sites, hydrophobically/hydrophilically functionalized sites, spots of adhesive, etc.</p>
    <p num="p-0031">The sites may be a pattern, i.e. a regular design or configuration, or randomly distributed. A preferred embodiment utilizes a regular pattern of sites such that the sites may be addressed in the X-Y coordinate plane. “Pattern” in this sense includes a repeating unit cell, preferably one that allows a high density of beads on the substrate. However, it should be noted that these sites may not be discrete sites. That is, it is possible to use a uniform surface of adhesive or chemical functionalities, for example, that allows the association of beads at any position. That is, the surface of the substrate is modified to allow association of the microspheres at individual sites, whether or not those sites are contiguous or non-contiguous with other sites. Thus, the surface of the substrate may be modified such that discrete sites are formed that can only have a single associated bead, or alternatively, the surface of the substrate is modified and beads may go down anywhere, but they end up at discrete sites.</p>
    <p num="p-0032">In a preferred embodiment, the surface of the substrate is modified to contain wells, i.e. depressions in the surface of the substrate. This may be done as is generally known in the art using a variety of techniques, including, but not limited to, photolithography, stamping techniques, molding techniques and microetching techniques. As will be appreciated by those in the art, the technique used will depend on the composition and shape of the substrate.</p>
    <p num="p-0033">In a preferred embodiment, physical alterations are made in a surface of the substrate to produce the sites. In a preferred embodiment, the substrate is a fiber optic bundle and the surface of the substrate is a terminal end of the fiber bundle, as is generally described in U.S. Ser. Nos. 08/818,199 and 09/151,877, both of which are hereby expressly incorporated by reference. In this embodiment, wells are made in a terminal or distal end of a fiber optic bundle comprising individual fibers. In this embodiment, the cores of the individual fibers are etched, with respect to the cladding, such that small wells or depressions are formed at one end of the fibers. The required depth of the wells will depend on the size of the beads to be added to the wells.</p>
    <p num="p-0034">Generally in this embodiment, the microspheres are non-covalently associated in the wells, although the wells may additionally be chemically functionalized as is generally described below, cross-linking agents may be used, or a physical barrier may be used, i.e. a film or membrane over the beads.</p>
    <p num="p-0035">In a preferred embodiment, the surface of the substrate is modified to contain chemically modified sites, that can be used to associate, either covalently or non-covalently, the microspheres of the invention to the discrete sites or locations on the substrate. “Chemically modified sites” in this context includes, but is not limited to, the addition of a pattern of chemical functional groups including amino groups, carboxy groups, oxo groups and thiol groups, that can be used to covalently attach microspheres, which generally also contain corresponding reactive functional groups; the addition of a pattern of adhesive that can be used to bind the microspheres (either by prior chemical functionalization for the addition of the adhesive or direct addition of the adhesive); the addition of a pattern of charged groups (similar to the chemical functionalities) for the electrostatic association of the microspheres, i.e. when the microspheres comprise charged groups opposite to the sites; the addition of a pattern of chemical functional groups that renders the sites differentially hydrophobic or hydrophilic, such that the addition of similarly hydrophobic or hydrophilic microspheres under suitable experimental conditions will result in association of the microspheres to the sites on the basis of hydroaffinity. For example, the use of hydrophobic sites with hydrophobic beads, in an aqueous system, drives the association of the beads preferentially onto the sites. As outlined above, “pattern” in this sense includes the use of a uniform treatment of the surface to allow association of the beads at discrete sites, as well as treatment of the surface resulting in discrete sites. As will be appreciated by those in the art, this may be accomplished in a variety of ways.</p>
    <p num="p-0036">The compositions of the invention further comprise a population of microspheres. By “population” herein is meant a plurality of beads as outlined above for arrays. Within the population are separate subpopulations, which can be a single microsphere or multiple identical microspheres. That is, in some embodiments, as is more fully outlined below, the array may contain only a single bead for each bioactive agent; preferred embodiments utilize a plurality of beads of each type.</p>
    <p num="p-0037">By “microspheres” or “beads” or “particles” or grammatical equivalents herein is meant small discrete particles. The composition of the beads will vary, depending on the class of bioactive agent and the method of synthesis. Suitable bead compositions include those used in peptide, nucleic acid and organic moiety synthesis, including, but not limited to, plastics, ceramics, glass, polystyrene, methylstyrene, acrylic polymers, paramagnetic materials, thoria sol, carbon graphite, titanium dioxide, latex or cross-linked dextrans such as Sepharose, cellulose, nylon, cross-linked micelles and Teflon may all be used. “<i>Microsphere Detection Guide</i>” from Bangs Laboratories, Fishers Ind. is a helpful guide.</p>
    <p num="p-0038">The beads need not be spherical; irregular particles may be used. In addition, the beads may be porous, thus increasing the surface area of the bead available for either bioactive agent attachment or IBL attachment. The bead sizes range from nanometers, i.e. 100 nm, to millimeters, i.e. 1 mm, with beads from about 0.2 micron to about 200 microns being preferred, and from about 0.5 to about 5 micron being particularly preferred, although in some embodiments smaller beads may be used.</p>
    <p num="p-0039">It should be noted that a key component of the invention is the use of a substrate/bead pairing that allows the association or attachment of the beads at discrete sites on the surface of the substrate, such that the beads do not move during the course of the assay.</p>
    <p num="p-0040">Each microsphere comprises a bioactive agent, although as will be appreciated by those in the art, there may be some microspheres which do not contain a bioactive agent, depending on the synthetic methods. By “candidate bioactive agent” or “bioactive agent” or “chemical functionality” or “binding ligand” herein is meant as used herein describes any molecule, e.g., protein, oligopeptide, small organic molecule, coordination complex, polysaccharide, polynucleotide, etc. which can be attached to the microspheres of the invention. It should be understood that the compositions of the invention have two primary uses. In a preferred embodiment, as is more fully outlined below, the compositions are used to detect the presence of a particular target analyte; for example, the presence or absence of a particular nucleotide sequence or a particular protein, such as an enzyme, an antibody or an antigen. In an alternate preferred embodiment, the compositions are used to screen bioactive agents, i.e. drug candidates, for binding to a particular target analyte.</p>
    <p num="p-0041">Bioactive agents encompass numerous chemical classes, though typically they are organic molecules, preferably small organic compounds having a molecular weight of more than 100 and less than about 2,500 Daltons. Bioactive agents comprise functional groups necessary for structural interaction with proteins, particularly hydrogen bonding, and typically include at least an amine, carbonyl, hydroxyl or carboxyl group, preferably at least two of the functional chemical groups. The bioactive agents often comprise cyclical carbon or heterocyclic structures and/or aromatic or polyaromatic structures substituted with one or more of the above functional groups. Bioactive agents are also found among biomolecules including peptides, nucleic acids, saccharides, fatty acids, steroids, purines, pyrimidines, derivatives, structural analogs or combinations thereof. Particularly preferred are nucleic acids and proteins.</p>
    <p num="p-0042">Bioactive agents can be obtained from a wide variety of sources including libraries of synthetic or natural compounds. For example, numerous means are available for random and directed synthesis of a wide variety of organic compounds and biomolecules, including expression of randomized oligonucleotides. Alternatively, libraries of natural compounds in the form of bacterial, fungal, plant and animal extracts are available or readily produced. Additionally, natural or synthetically produced libraries and compounds are readily modified through conventional chemical, physical and biochemical means. Known pharmacological agents may be subjected to directed or random chemical modifications, such as acylation, alkylation, esterification and/or amidification to produce structural analogs.</p>
    <p num="p-0043">In a preferred embodiment, the bioactive agents are proteins. By “protein” herein is meant at least two covalently attached amino acids, which includes proteins, polypeptides, oligopeptides and peptides. The protein may be made up of naturally occurring amino acids and peptide bonds, or synthetic peptidomimetic structures. Thus “amino acid”, or “peptide residue”, as used herein means both naturally occurring and synthetic amino acids. For example, homo-phenylalanine, citrulline and norleucine are considered amino acids for the purposes of the invention. The side chains may be in either the (R) or the (S) configuration. In the preferred embodiment, the amino acids are in the (S) or L-configuration. If non-naturally occurring side chains are used, non-amino acid substituents may be used, for example to prevent or retard in vivo degradations.</p>
    <p num="p-0044">In one preferred embodiment, the bioactive agents are naturally occurring proteins or fragments of naturally occuring proteins. Thus, for example, cellular extracts containing proteins, or random or directed digests of proteinaceous cellular extracts, may be used. In this way libraries of procaryotic and eukaryotic proteins may be made for screening in the systems described herein. Particularly preferred in this embodiment are libraries of bacterial, fungal, viral, and mammalian proteins, with the latter being preferred, and human proteins being especially preferred.</p>
    <p num="p-0045">In a preferred embodiment, the bioactive agents are peptides of from about 5 to about 30 amino acids, with from about 5 to about 20 amino acids being preferred, and from about 7 to about 15 being particularly preferred. The peptides may be digests of naturally occurring proteins as is outlined above, random peptides, or “biased” random peptides. By “randomized” or grammatical equivalents herein is meant that each nucleic acid and peptide consists of essentially random nucleotides and amino acids, respectively. Since generally these random peptides (or nucleic acids, discussed below) are chemically synthesized, they may incorporate any nucleotide or amino acid at any position. The synthetic process can be designed to generate randomized proteins or nucleic acids, to allow the formation of all or most of the possible combinations over the length of the sequence, thus forming a library of randomized bioactive proteinaceous agents.</p>
    <p num="p-0046">In a preferred embodiment, a library of bioactive agents are used. The library should provide a sufficiently structurally diverse population of bioactive agents to effect a probabilistically sufficient range of binding to target analytes. Accordingly, an interaction library must be large enough so that at least one of its members will have a structure that gives it affinity for the target analyte. Although it is difficult to gauge the required absolute size of an interaction library, nature provides a hint with the immune response a diversity of 10<sup>7</sup>–10<sup>8 </sup>different antibodies provides at least one combination with sufficient affinity to interact with most potential antigens faced by an organism. Published in vitro selection techniques have also shown that a library size of 10<sup>7 </sup>to 10<sup>8 </sup>is sufficient to find structures with affinity for the target. Thus, in a preferred embodiment, at least 10<sup>6</sup>, preferably at least 10<sup>7</sup>, more preferably at least 10<sup>3 </sup>and most preferably at least 10<sup>9 </sup>different bioactive agents are simultaneously analyzed in the subject methods. Preferred methods maximize library size and diversity.</p>
    <p num="p-0047">In a preferred embodiment, the library is fully randomized, with no sequence preferences or constants at any position. In a preferred embodiment, the library is biased. That is, some positions within the sequence are either held constant, or are selected from a limited number of possibilities. For example, in a preferred embodiment, the nucleotides or amino acid residues are randomized within a defined class, for example, of hydrophobic amino acids, hydrophilic residues, sterically biased (either small or large) residues, towards the creation of cysteines, for cross-linking, prolines for SH-3 domains, serines, threonines, tyrosines or histidines for phosphorylation sites, etc., or to purines, etc.</p>
    <p num="p-0048">In a preferred embodiment, the bioactive agents are nucleic acids (generally called “probe nucleic acids” or “candidate probes” herein). By “nucleic acid” or “oligonucleotide” or grammatical equivalents herein means at least two nucleotides covalently linked together. A nucleic acid of the present invention will generally contain phosphodiester bonds, although in some cases, as outlined below, nucleic acid analogs are included that may have alternate backbones, comprising, for example, phosphoramide (Beaucage, et al., Tetrahedron, 49(10):1925 (1993) and references therein; Letsinger, <i>J. Org. Chem., </i>35:3800 (1970); Sprinzi, et al., <i>Eur. J. Biochem., </i>81:579 (1977); Letsinger, et al., <i>Nucl. Acids Res., </i>14:3487 (1986); Sawai, et al., <i>Chem. Lett., </i>805 (1984), Letsinger, et al., <i>J. Am. Chem. Soc., </i>110:4470 (1988); and Pauwels, et al., <i>Chemica Scripta, </i>26:141 (1986)), phosphorothioate (Mag, et al., <i>Nucleic Acids Res., </i>19:1437 (1991); and U.S. Pat. No. 5,644,048), phosphorodithioate (Briu, et al., <i>J. Am. Chem. Soc., </i>111:2321 (1989)), O-methylphophoroamidite linkages (see Eckstein, Oligonucleotides and Analogues: A Practical Approach, Oxford University Press), and peptide nucleic acid backbones and linkages (see Egholm, <i>J. Am. Chem. Soc., </i>114:1895 (1992); Meier, et al., <i>Chem. Int. Ed. Engl., </i>31:1008 (1992); Nielsen, <i>Nature, </i>365:566 (1993); Carlsson, et al., <i>Nature, </i>380:207 (1996), all of which are incorporated by reference)). Other analog nucleic acids include those with positive backbones (Denpcy, et al., <i>Proc. Natl. Acad. Sci. USA, </i>92:6097 (1995)); non-ionic backbones (U.S. Pat. Nos. 5,386,023; 5,637,684; 5,602,240; 5,216,141; and 4,469,863; Kiedrowshi, et al., <i>Angew. Chem. Intl. Ed. English, </i>30:423 (1991); Letsinger, et al., <i>J. Am. Chem. Soc., </i>110:4470 (1988); Letsinger, et al., Nucleosides &amp; Nucleotides, 13:1597 (1994); Chapters 2 and 3, ASC Symposium Series 580, “Carbohydrate Modifications in Antisense Research”, Ed. Y. S. Sanghui and P. Dan Cook; Mesmaeker, et al., <i>Bioorganic </i>&amp; <i>Medicinal Chem. Lett., </i>4:395 (1994); Jeffs, et al., <i>J. Biomolecular NMR, </i>34:17 (1994); Tetrahedron Lett., 37:743 (1996)) and non-ribose backbones, including those described in U.S. Pat. Nos. 5,235,033 and 5,034,506, and Chapters 6 and 7, ASC Symposium Series 580, “Carbohydrate Modifications in Antisense Research”, Ed. Y. S. Sanghui and P. Dan Cook. Nucleic acids containing one or more carbocyclic sugars are also included within the definition of nucleic acids (see Jenkins, et al., <i>Chem. Soc. Rev.</i>, (1995) pp. 169–176). Several nucleic acid analogs are described in Rawls, C &amp; E News, Jun. 2, 1997, page 35. All of these references are hereby expressly incorporated by reference. These modifications of the ribose-phosphate backbone may be done to facilitate the addition of additional moieties such as labels, or to increase the stability and half-life of such molecules in physiological environments; for example, PNA is particularly preferred. In addition, mixtures of naturally occurring nucleic acids and analogs can be made. Alternatively, mixtures of different nucleic acid analogs, and mixtures of naturally occurring nucleic acids and analogs may be made. The nucleic acids may be single stranded or double stranded, as specified, or contain portions of both double stranded or single stranded sequence. The nucleic acid may be DNA, both genomic and cDNA, RNA or a hybrid, where the nucleic acid contains any combination of deoxyribo- and ribo-nucleotides, and any combination of bases, including uracil, adenine, thymine, cytosine, guanine, inosine, xanthanine, hypoxanthanine, isocytosine, isoguanine, and base analogs such as nitropyrrole and nitroindole, etc.</p>
    <p num="p-0049">In a preferred embodiment, the bioactive agents are libraries of clonal nucleic acids, including DNA and RNA. In this embodiment, individual nucleic acids are prepared, generally using conventional methods (including, but not limited to, propagation in plasmid or phage vectors, amplification techniques including PCR, etc.). The nucleic acids are preferably arrayed in some format, such as a microtiter plate format, and beads added for attachment of the libraries.</p>
    <p num="p-0050">Attachment of the clonal libraries (or any of the nucleic acids outlined herein) may be done in a variety of ways, as will be appreciated by those in the art, including, but not limited to, chemical or affinity capture (for example, including the incorporation of derivatized nucleotides such as AminoLink or biotinylated nucleotides that can then be used to attach the nucleic acid to a surface, as well as affinity capture by hybridization), cross-linking, and electrostatic attachment, etc.</p>
    <p num="p-0051">In a preferred embodiment, affinity capture is used to attach the clonal nucleic acids to the beads. For example, cloned nucleic acids can be derivatized, for example with one member of a binding pair, and the beads derivatized with the other member of a binding pair. Suitable binding pairs are as described herein for IBL/DBL pairs. For example, the cloned nucleic acids may be biotinylated (for example using enzymatic incorporate of biotinylated nucleotides, for by photoactivated cross-linking of biotin). Biotinylated nucleic acids can then be captured on streptavidin-coated beads, as is known in the art. Similarly, other hapten-receptor combinations can be used, such as digoxigenin and anti-digoxigenin antibodies. Alternatively, chemical groups can be added in the form of derivatized nucleotides, that can them be used to add the nucleic acid to the surface.</p>
    <p num="p-0052">Preferred attachments are covalent, although even relatively weak interactions (i.e. non-covalent) can be sufficient to attach a nucleic acid to a surface, if there are multiple sites of attachment per each nucleic acid. Thus, for example, electrostatic interactions can be used for attachment, for example by having beads carrying the opposite charge to the bioactive agent.</p>
    <p num="p-0053">Similarly, affinity capture utilizing hybridization can be used to attach cloned nucleic acids to beads. For example, as is known in the art, polyA+RNA is routinely captured by hybridization to oligo-dT beads; this may include oligo-dT capture followed by a cross-linking step, such as psoralen crosslinking). If the nucleic acids of interest do not contain a polyA tract, one can be attached by polymerization with terminal transferase, or via ligation of an oligoA linker, as is known in the art.</p>
    <p num="p-0054">Alternatively, chemical crosslinking may be done, for example by photoactivated crosslinking of thymidine to reactive groups, as is known in the art.</p>
    <p num="p-0055">In general, special methods are required to decode clonal arrays, as is more fully outlined below.</p>
    <p num="p-0056">As described above generally for proteins, nucleic acid bioactive agents may be naturally occurring nucleic acids, random nucleic acids, or “biased” random nucleic acids. For example, digests of procaryotic or eukaryotic genomes may be used as is outlined above for proteins.</p>
    <p num="p-0057">In general, probes of the present invention are designed to be complementary to a target sequence (either the target analyte sequence of the sample or to other probe sequences, as is described herein), such that hybridization of the target and the probes of the present invention occurs. This complementarily need not be perfect; there may be any number of base pair mismatches that will interfere with hybridization between the target sequence and the single stranded nucleic acids of the present invention. However, if the number of mutations is so great that no hybridization can occur under even the least stringent of hybridization conditions, the sequence is not a complementary target sequence. Thus, by “substantially complementary” herein is meant that the probes are sufficiently complementary to the target sequences to hybridize under the selected reaction conditions. High stringency conditions are known in the art; see for example Maniatis et al., <i>Molecular Cloning: A Laboratory Manual, </i>2d Edition, 1989, and <i>Short Protocols in Molecular Biology</i>, ed. Ausubel, et al., both of which are hereby incorporated by reference. Stringent conditions are sequence-dependent and will be different in different circumstances. Longer sequences hybridize specifically at higher temperatures. An extensive guide to the hybridization of nucleic acids is found in Tijssen, Techniques in Biochemistry and Molecular Biology—Hybridization with Nucleic Acid Probes, “Overview of principles of hybridization and the strategy of nucleic acid assays” (1993). Generally, stringent conditions are selected to be about 5–10° C. lower than the thermal melting point (T<sub>m</sub>) for the specific sequence at a defined ionic strength pH. The T<sub>m </sub>is the temperature (under defined ionic strength, pH and nucleic acid concentration) at which 50% of the probes complementary to the target hybridize to the target sequence at equilibrium (as the target sequences are present in excess, at T<sub>m</sub>, 50% of the probes are occupied at equilibrium). Stringent conditions will be those in which the salt concentration is less than about 1.0 M sodium ion, typically about 0.01 to 1.0 M sodium ion concentration (or other salts) at pH 7.0 to 8.3 and the temperature is at least about 30° C. for short probes (e.g. 10 to 50 nucleotides) and at least about 60° C. for long probes (e.g. greater than 50 nucleotides). Stringent conditions may also be achieved with the addition of destabilizing agents such as formamide. In another embodiment, less stringent hybridization conditions are used; for example, moderate or low stringency conditions may be used, as are known in the art; see Maniatis and Ausubel, supra, and Tijssen, supra.</p>
    <p num="p-0058">The term “target sequence” or grammatical equivalents herein means a nucleic acid sequence on a single strand of nucleic acid. The target sequence may be a portion of a gene, a regulatory sequence, genomic DNA, cDNA, RNA including mRNA and rRNA, or others. It may be any length, with the understanding that longer sequences are more specific. As will be appreciated by those in the art, the complementary target sequence may take many forms. For example, it may be contained within a larger nucleic acid sequence, i.e. all or part of a gene or mRNA, a restriction fragment of a plasmid or genomic DNA, among others. As is outlined more fully below, probes are made to hybridize to target sequences to determine the presence or absence of the target sequence in a sample. Generally speaking, this term will be understood by those skilled in the art.</p>
    <p num="p-0059">In a preferred embodiment, the bioactive agents are organic chemical moieties, a wide variety of which are available in the literature.</p>
    <p num="p-0060">In a preferred embodiment, each bead comprises a single type of bioactive agent, although a plurality of individual bioactive agents are preferably attached to each bead. Similarly, preferred embodiments utilize more than one microsphere containing a unique bioactive agent; that is, there is redundancy built into the system by the use of subpopulations of microspheres, each microsphere in the subpopulation containing the same bioactive agent.</p>
    <p num="p-0061">As will be appreciated by those in the art, the bioactive agents may either be synthesized directly on the beads, or they may be made and then attached after synthesis. In a preferred embodiment, linkers are used to attach the bioactive agents to the beads, to allow both good attachment, sufficient flexibility to allow good interaction with the target molecule, and to avoid undesirable binding reactions.</p>
    <p num="p-0062">In a preferred embodiment, the bioactive agents are synthesized directly on the beads. As is known in the art, many classes of chemical compounds are currently synthesized on solid supports, including beads, such as peptides, organic moieties, and nucleic acids.</p>
    <p num="p-0063">In a preferred embodiment, the bioactive agents are synthesized first, and then covalently attached to the beads. As will be appreciated by those in the art, this will be done depending on the composition of the bioactive agents and the beads. The functionalization of solid support surfaces such as certain polymers with chemically reactive groups such as thiols, amines, carboxyls, etc. is generally known in the art. Accordingly, “blank” microspheres may be used that have surface chemistries that facilitate the attachment of the desired functionality by the user. Some examples of these surface chemistries for blank microspheres include, but are not limited to, amino groups including aliphatic and aromatic amines, carboxylic acids, aldehydes, amides, chloromethyl groups, hydrazide, hydroxyl groups, sulfonates and sulfates.</p>
    <p num="p-0064">These functional groups can be used to add any number of different candidate agents to the beads, generally using known chemistries. For example, candidate agents containing carbohydrates may be attached to an amino-functionalized support; the aldehyde of the carbohydrate is made using standard techniques, and then the aldehyde is reacted with an amino group on the surface. In an alternative embodiment, a sulfhydryl linker may be used. There are a number of sulfhydryl reactive linkers known in the art such as SPDP, maleimides, α-haloacetyls, and pyridyl disulfides (see for example the 1994 Pierce Chemical Company catalog, technical section on cross-linkers, pages 155–200, incorporated herein by reference) which can be used to attach cysteine containing proteinaceous agents to the support. Alternatively, an amino group on the candidate agent may be used for attachment to an amino group on the surface. For example, a large number of stable bifunctional groups are well known in the art, including homobifunctional and heterobifunctional linkers (see Pierce Catalog and Handbook, pages 155–200). In an additional embodiment, carboxyl groups (either from the surface or from the candidate agent) may be derivatized using well known linkers (see the Pierce catalog). For example, carbodiimides activate carboxyl groups for attack by good nucleophiles such as amines (see Torchilin et al., <i>Critical Rev. Therapeutic Drug Carrier Systems, </i>7(4):275–308 (1991), expressly incorporated herein). Proteinaceous candidate agents may also be attached using other techniques known in the art, for example for the attachment of antibodies to polymers; see Slinkin et al., <i>Bioconj. Chem. </i>2:342–348 (1991); Torchilin et al., supra; Trubetskoy et al., <i>Bioconj. Chem. </i>3:323–327 (1992); King et al., <i>Cancer Res. </i>54:6176–6185 (1994); and Wilbur et al., <i>Bioconjugate Chem. </i>5:220–235 (1994), all of which are hereby expressly incorporated by reference). It should be understood that the candidate agents may be attached in a variety of ways, including those listed above. Preferably, the manner of attachment does not significantly alter the functionality of the candidate agent; that is, the candidate agent should be attached in such a flexible manner as to allow its interaction with a target.</p>
    <p num="p-0065">Specific techniques for immobilizing enzymes on microspheres are known in the prior art. In one case, NH<sub>2 </sub>surface chemistry microspheres are used. Surface activation is achieved with a 2.5% glutaraldehyde in phosphate buffered saline (10 mM) providing a pH of 6.9. (138 mM NaCl, 2.7 mM, KCl). This is stirred on a stir bed for approximately 2 hours at room temperature. The microspheres are then rinsed with ultrapure water plus 0.01% tween 20 (surfactant) −0.02%, and rinsed again with a pH 7.7 PBS plus 0.01% tween 20. Finally, the enzyme is added to the solution, preferably after being prefiltered using a 0.45 μm amicon micropure filter.</p>
    <p num="p-0066">In some embodiments, the microspheres may additionally comprise identifier binding ligands for use in certain decoding systems. By “identifier binding ligands” or “IBLs” herein is meant a compound that will specifically bind a corresponding decoder binding ligand (DBL) to facilitate the elucidation of the identity of the bioactive agent attached to the bead. That is, the IBL and the corresponding DBL form a binding partner pair. By “specifically bind” herein is meant that the IBL binds its DBL with specificity sufficient to differentiate between the corresponding DBL and other DBLs (that is, DBLs for other IBLs), or other components or contaminants of the system. The binding should be sufficient to remain bound under the conditions of the decoding step, including wash steps to remove non-specific binding. In some embodiments, for example when the IBLs and corresponding DBLs are proteins or nucleic acids, the dissociation constants of the IBL to its DBL will be less than about 10<sup>−4</sup>–10<sup>−6 </sup>M<sup>−1</sup>, with less than about 10<sup>−5 </sup>to 10<sup>−9 </sup>M<sup>−1 </sup>being preferred and less than about 10<sup>−7</sup>–10<sup>−9 </sup>M<sup>−1</sup>, being particularly preferred.</p>
    <p num="p-0067">IBL-DBL binding pairs are known or can be readily found using known techniques. For example, when the IBL is a protein, the DBLs include proteins (particularly including antibodies or fragments thereof (FAbs, etc.)) or small molecules, or vice versa (the IBL is an antibody and the DBL is a protein). Metal ion-metal ion ligands or chelators pairs are also useful. Antigen-antibody pairs, enzymes and substrates or inhibitors, other protein-protein interacting pairs, receptor-ligands, complementary nucleic acids (including nucleic acid molecules that form triple helices), and carbohydrates and their binding partners are also suitable binding pairs. Nucleic acid—nucleic acid binding proteins pairs are also useful, including single-stranded or double-stranded nucleic acid binding proteins, and small molecule nucleic acid binding agents. Similarly, as is generally described in U.S. Pat. Nos. 5,270,163, 5,475,096, 5,567,588, 5,595,877, 5,637,459, 5,683,867, 5,705,337, and related patents, hereby incorporated by reference, nucleic acid “aptamers” can be developed for binding to virtually any target; such an aptamer-target pair can be used as the IBL-DBL pair. Similarly, there is a wide body of literature relating to the development of binding pairs based on combinatorial chemistry methods.</p>
    <p num="p-0068">In a preferred embodiment, the IBL is a molecule whose color or luminescence properties change in the presence of a selectively-binding DBL.</p>
    <p num="p-0069">In one embodiment, the DBL may be attached to a bead, i.e. a “decoder bead”, that may carry a label such as a fluorophore.</p>
    <p num="p-0070">In a preferred embodiment, the IBL-DBL pair comprise substantially complementary single-stranded nucleic acids. In this embodiment, the binding ligands can be referred to as “identifier probes” and “decoder probes”. Generally, the identifier and decoder probes range from about 4 basepairs in length to about 1000, with from about 6 to about 100 being preferred, and from about 8 to about 40 being particularly preferred. What is important is that the probes are long enough to be specific, i.e. to distinguish between different IBL-DBL pairs, yet short enough to allow both a) dissociation, if necessary, under suitable experimental conditions, and b) efficient hybridization.</p>
    <p num="p-0071">In a preferred embodiment, as is more fully outlined below, the IBLs do not bind to DBLs. Rather, the IBLs are used as identifier moieties (“IMs”) that are identified directly, for example through the use of mass spectroscopy.</p>
    <p num="p-0072">Alternatively, in a preferred embodiment, the IBL and the bioactive agent are the same moiety; thus, for example, as outlined herein, particularly when no optical signatures are used, the bioactive agent can serve as both the identifier and the agent. For example, in the case of nucleic acids, the bead-bound probe (which serves as the bioactive agent) can also bind decoder probes, to identify the sequence of the probe on the bead. Thus, in this embodiment, the DBLs bind to the bioactive agents. This is particularly useful as this embodiment can give information about the array or the assay in addition to decoding. For example, as is more fully described below, the use of the DBLs allows array calibration and assay development. This may be done even if the DBLs are not used as such; for example in non-random arrays, the use of these probe sets can allow array calibration and assay development even if decoding is not required.</p>
    <p num="p-0073">In a preferred embodiment, the microspheres do not contain an optical signature. That is, as outlined in U.S. Ser. Nos. 08/818,199 and 09/151,877, previous work had each subpopulation of microspheres comprising a unique optical signature or optical tag that is used to identify the unique bioactive agent of that subpopulation of microspheres; that is, decoding utilizes optical properties of the beads such that a bead comprising the unique optical signature may be distinguished from beads at other locations with different optical signatures. Thus the previous work assigned each bioactive agent a unique optical signature such that any microspheres comprising that bioactive agent are identifiable on the basis of the signature. These optical signatures comprised dyes, usually chromophores or fluorophores, that were entrapped or attached to the beads themselves. Diversity of optical signatures utilized different fluorochromes, different ratios of mixtures of fluorochromes, and different concentrations (intensities) of fluorochromes.</p>
    <p num="p-0074">Thus, the present invention does not rely solely on the use of optical properties to decode the arrays. However, as will be appreciated by those in the art, it is possible in some embodiments to utilize optical signatures as an additional coding method, in conjunction with the present system. Thus, for example, as is more fully outlined below, the size of the array may be effectively increased while using a single set of decoding moieties in several ways, one of which is the use in combination with optical signatures one beads. Thus, for example, using one “set” of decoding molecules, the use of two populations of beads, one with an optical signature and one without, allows the effective doubling of the array size. The use of multiple optical signatures similarly increases the possible size of the array.</p>
    <p num="p-0075">In a preferred embodiment, each subpopulation of beads comprises a plurality of different IBLs. By using a plurality of different IBLs to encode each bioactive agent, the number of possible unique codes is substantially increased. That is, by using one unique IBL per bioactive agent, the size of the array will be the number of unique IBLs (assuming no “reuse” occurs, as outlined below). However, by using a plurality of different IBLs per bead, n, the size of the array can be increased to 2<sup>n</sup>, when the presence or absence of each IBL is used as the indicator. For example, the assignment of 10 IBLs per bead generates a 10 bit binary code, where each bit can be designated as “1” (IBL is present) or “0” (IBL is absent). A 10 bit binary code has 2<sup>10 </sup>possible variants However, as is more fully discussed below, the size of the array may be further increased if another parameter is included such as concentration or intensity; thus for example, if two different concentrations of the IBL are used, then the array size increases as 3<sup>n</sup>. Thus, in this embodiment, each individual bioactive agent in the array is assigned a combination of IBLs, which can be added to the beads prior to the addition of the bioactive agent, after, or during the synthesis of the bioactive agent, i.e. simultaneous addition of IBLs and bioactive agent components.</p>
    <p num="p-0076">Alternatively, when the bioactive agent is a polymer of different residues, i.e. when the bioactive agent is a protein or nucleic acid, the combination of different IBLs can be used to elucidate the sequence of the protein or nucleic acid.</p>
    <p num="p-0077">Thus, for example, using two different IBLs (IBL1 and IBL2), the first position of a nucleic acid can be elucidated: for example, adenosine can be represented by the presence of both IBL1 and IBL2; thymidine can be represented by the presence of IBL1 but not IBL2, cytosine can be represented by the presence of IBL2 but not IBL1, and guanosine can be represented by the absence of both. The second position of the nucleic acid can be done in a similar manner using IBL3 and IBL4; thus, the presence of IBL1, IBL2, IBL3 and IBL4 gives a sequence of AA; IBL1, IBL2, and IBL3 shows the sequence AT; IBL1, IBL3 and IBL4 gives the sequence TA, etc. The third position utilizes IBL5 and IBL6, etc. In this way, the use of 20 different identifiers can yield a unique code for every possible 10-mer.</p>
    <p num="p-0078">The system is similar for proteins but requires a larger number of different IBLs to identify each position, depending on the allowed diversity at each position. Thus for example, if every amino acid is allowed at every position, five different IBLs are required for each position. However, as outlined above, for example when using random peptides as the bioactive agents, there may be bias built into the system; not all amino acids may be present at all positions, and some positions may be preset; accordingly, it may be possible to utilize four different IBLs for each amino acid.</p>
    <p num="p-0079">In this way, a sort of “bar code” for each sequence can be constructed; the presence or absence of each distinct IBL will allow the identification of each bioactive agent.</p>
    <p num="p-0080">In addition, the use of different concentrations or densities of IBLs allows a “reuse” of sorts. If, for example, the bead comprising a first agent has a 1× concentration of IBL, and a second bead comprising a second agent has a 10× concentration of IBL, using saturating concentrations of the corresponding labelled DBL allows the user to distinguish between the two beads.</p>
    <p num="p-0081">Once the microspheres comprising the candidate agents and the unique IBLs are generated, they are added to the substrate to form an array. It should be noted that while most of the methods described herein add the beads to the substrate prior to the assay, the order of making, using and decoding the array can vary. For example, the array can be made, decoded, and then the assay done. Alternatively, the array can be made, used in an assay, and then decoded; this may find particular use when only a few beads need be decoded. Alternatively, the beads can be added to the assay mixture, i.e. the sample containing the target analytes, prior to the addition of the beads to the substrate; after addition and assay, the array may be decoded. This is particularly preferred when the sample comprising the beads is agitated or mixed; this can increase the amount of target analyte bound to the beads per unit time, and thus (in the case of nucleic acid assays) increase the hybridization kinetics. This may find particular use in cases where the concentration of target analyte in the sample is low; generally, for low concentrations, long binding times must be used.</p>
    <p num="p-0082">In addition, adding the beads to the assay mixture can allow sorting or selection. For example, a large library of beads may be added to a sample, and only those beads that bind the sample may be added to the substrate. For example, if the target analyte is fluorescently labeled (either directly (for example by the incorporation of labels into nucleic acid amplification reactions) or indirectly (for example via the use of sandwich assays)), beads that exhibit fluorescence as a result of target analyte binding can be sorted via Fluorescence Activated Cell Sorting (FACS) and only these beads added to an array and subsequently decoded. Similarly, the sorting may be accomplished through affinity techniques; affinity columns comprising the target analytes can be made, and only those beads which bind are used on the array. Similarly, two bead systems can be used; for example, magnetic beads comprising the target analytes can be used to “pull out” those beads that will bind to the targets, followed by subsequent release of the magnetic beads (for example via temperature elevation) and addition to an array.</p>
    <p num="p-0083">In general, the methods of making the arrays and of decoding the arrays is done to maximize the number of different candidate agents that can be uniquely encoded. The compositions of the invention may be made in a variety of ways. In general, the arrays are made by adding a solution or slurry comprising the beads to a surface containing the sites for association of the beads. This may be done in a variety of buffers, including aqueous and organic solvents, and mixtures. The solvent can evaporate, and excess beads removed.</p>
    <p num="p-0084">In a preferred embodiment, when non-covalent methods are used to associate the beads to the array, a novel method of loading the beads onto the array is used. This method comprises exposing the array to a solution of particles (including microspheres and cells) and then applying energy, e.g. agitating or vibrating the mixture. This results in an array comprising more tightly associated particles, as the agitation is done with sufficient energy to cause weakly-associated beads to fall off (or out, in the case of wells). These sites are then available to bind a different bead. In this way, beads that exhibit a high affinity for the sites are selected. Arrays made in this way have two main advantages as compared to a more static loading: first of all, a higher percentage of the sites can be filled easily, and secondly, the arrays thus loaded show a substantial decrease in bead loss during assays. Thus, in a preferred embodiment, these methods are used to generate arrays that have at least about 50% of the sites filled, with at least about 75% being preferred, and at least about 90% being particularly preferred. Similarly, arrays generated in this manner preferably lose less than about 20% of the beads during an assay, with less than about 10% being preferred and less than about 5% being particularly preferred.</p>
    <p num="p-0085">In this embodiment, the substrate comprising the surface with the discrete sites is immersed into a solution comprising the particles (beads, cells, etc.). The surface may comprise wells, as is described herein, or other types of sites on a patterned surface such that there is a differential affinity for the sites. This differential affinity results in a competitive process, such that particles that will associate more tightly are selected. Preferably, the entire surface to be “loaded” with beads is in fluid contact with the solution. This solution is generally a slurry ranging from about 10,000:1 beads:solution (vol:vol) to 1:1. Generally, the solution can comprise any number of reagents, including aqueous buffers, organic solvents, salts, other reagent components, etc. In addition, the solution preferably comprises an excess of beads; that is, there are more beads than sites on the array. Preferred embodiments utilize two-fold to billion-fold excess of beads.</p>
    <p num="p-0086">The immersion can mimic the assay conditions; for example, if the array is to be “dipped” from above into a microtiter plate comprising samples, this configuration can be repeated for the loading, thus minimizing the beads that are likely to fall out due to gravity.</p>
    <p num="p-0087">Once the surface has been immersed, the substrate, the solution, or both are subjected to a competitive process, whereby the particles with lower affinity can be disassociated from the substrate and replaced by particles exhibiting a higher affinity to the site. This competitive process is done by the introduction of energy, in the form of heat, sonication, stirring or mixing, vibrating or agitating the solution or substrate, or both.</p>
    <p num="p-0088">A preferred embodiment utilizes agitation or vibration. In general, the amount of manipulation of the substrate is minimized to prevent damage to the array; thus, preferred embodiments utilize the agitation of the solution rather than the array, although either will work. As will be appreciated by those in the art, this agitation can take on any number of forms, with a preferred embodiment utilizing microtiter plates comprising bead solutions being agitated using microtiter plate shakers.</p>
    <p num="p-0089">The agitation proceeds for a period of time sufficient to load the array to a desired fill. Depending on the size and concentration of the beads and the size of the array, this time may range from about 1 second to days, with from about 1 minute to about 24 hours being preferred.</p>
    <p num="p-0090">It should be noted that not all sites of an array may comprise a bead; that is, there may be some sites on the substrate surface which are empty. In addition, there may be some sites that contain more than one bead, although this is not preferred.</p>
    <p num="p-0091">In some embodiments, for example when chemical attachment is done, it is possible to associate the beads in a non-random or ordered way. For example, using photoactivatible attachment linkers or photoactivatible adhesives or masks, selected sites on the array may be sequentially rendered suitable for attachment, such that defined populations of beads are laid down.</p>
    <p num="p-0092">The arrays of the present invention are constructed such that information about the identity of the candidate agent is built into the array, such that the random deposition of the beads in the fiber wells can be “decoded” to allow identification of the candidate agent at all positions. This may be done in a variety of ways, and either before, during or after the use of the array to detect target molecules.</p>
    <p num="p-0093">Thus, after the array is made, it is “decoded” in order to identify the location of one or more of the bioactive agents, i.e. each subpopulation of beads, on the substrate surface.</p>
    <p num="p-0094">In a preferred embodiment, a selective decoding system is used. In this case, only those microspheres exhibiting a change in the optical signal as a result of the binding of a target analyte are decoded. This is commonly done when the number of “hits”, i.e. the number of sites to decode, is generally low. That is, the array is first scanned under experimental conditions in the absence of the target analytes. The sample containing the target analytes is added, and only those locations exhibiting a change in the optical signal are decoded. For example, the beads at either the positive or negative signal locations may be either selectively tagged or released from the array (for example through the use of photocleavable linkers), and subsequently sorted or enriched in a fluorescence-activated cell sorter (FACS). That is, either all the negative beads are released, and then the positive beads are either released or analyzed in situ, or alternatively all the positives are released and analyzed. Alternatively, the labels may comprise halogenated aromatic compounds, and detection of the label is done using for example gas chromatography, chemical tags, isotopic tags, or mass spectral tags.</p>
    <p num="p-0095">As will be appreciated by those in the art, this may also be done in systems where the array is not decoded; i.e. there need not ever be a correlation of bead composition with location. In this embodiment, the beads are loaded on the array, and the assay is run. The “positives”, i.e. those beads displaying a change in the optical signal as is more fully outlined below, are then “marked” to distinguish or separate them from the “negative” beads. This can be done in several ways, preferably using fiber optic arrays. In a preferred embodiment, each bead contains a fluorescent dye. After the assay and the identification of the “positives” or “active beads”, light is shown down either only the positive fibers or only the negative fibers, generally in the presence of a light-activated reagent (typically dissolved oxygen). In the former case, all the active beads are photobleached. Thus, upon non-selective release of all the beads with subsequent sorting, for example using a fluorescence activated cell sorter (FACS) machine, the non-fluorescent active beads can be sorted from the fluorescent negative beads. Alternatively, when light is shown down the negative fibers, all the negatives are non-fluorescent and the the postives are fluorescent, and sorting can proceed. The characterization of the attached bioactive agent may be done directly, for example using mass spectroscopy.</p>
    <p num="p-0096">Alternatively, the identification may occur through the use of identifier moieties (“IMs”), which are similar to IBLs but need not necessarily bind to DBLs. That is, rather than elucidate the structure of the bioactive agent directly, the composition of the IMs may serve as the identifier. Thus, for example, a specific combination of IMs can serve to code the bead, and be used to identify the agent on the bead upon release from the bead followed by subsequent analysis, for example using a gas chromatograph or mass spectroscope.</p>
    <p num="p-0097">Alternatively, rather than having each bead contain a fluorescent dye, each bead comprises a non-fluorescent precursor to a fluorescent dye. For example, using photocleavable protecting groups, such as certain ortho-nitrobenzyl groups, on a fluorescent molecule, photoactivation of the fluorochrome can be done. After the assay, light is shown down again either the “positive” or the “negative” fibers, to distinguish these populations. The illuminated precursors are then chemically converted to a fluorescent dye. All the beads are then released from the array, with sorting, to form populations of fluorescent and non-fluorescent beads (either the positives and the negatives or vice versa).</p>
    <p num="p-0098">In an alternate preferred embodiment, the sites of association of the beads (for example the wells) include a photopolymerizable reagent, or the photopolymerizable agent is added to the assembled array. After the test assay is run, light is shown down again either the “positive” or the “negative” fibers, to distinguish these populations. As a result of the irradiation, either all the positives or all the negatives are polymerized and trapped or bound to the sites, while the other population of beads can be released from the array.</p>
    <p num="p-0099">In a preferred embodiment, the location of every bioactive agent is determined using decoder binding ligands (DBLs). As outlined above, DBLs are binding ligands that will either bind to identifier binding ligands, if present, or to the bioactive agents themselves, preferably when the bioactive agent is a nucleic acid or protein.</p>
    <p num="p-0100">In a preferred embodiment, as outlined above, the DBL binds to the IBL.</p>
    <p num="p-0101">In a preferred embodiment, the bioactive agents are single-stranded nucleic acids and the DBL is a substantially complementary single-stranded nucleic acid that binds (hybridizes) to the bioactive agent, termed a decoder probe herein. A decoder probe that is substantially complementary to each candidate probe is made and used to decode the array. In this embodiment, the candidate probes and the decoder probes should be of sufficient length (and the decoding step run under suitable conditions) to allow specificity; i.e. each candidate probe binds to its corresponding decoder probe with sufficient specificity to allow the distinction of each candidate probe.</p>
    <p num="p-0102">In a preferred embodiment, the DBLs are either directly or indirectly labeled. By “labeled” herein is meant that a compound has at least one element, isotope or chemical compound attached to enable the detection of the compound. In general, labels fall into three classes: a) isotopic labels, which may be radioactive or heavy isotopes; b) magnetic, electrical, thermal; and c) colored or luminescent dyes; although labels include enzymes and particles such as magnetic particles as well. Preferred labels include luminescent labels. In a preferred embodiment, the DBL is directly labeled, that is, the DBL comprises a label. In an alternate embodiment, the DBL is indirectly labeled; that is, a labeling binding ligand (LBL) that will bind to the DBL is used. In this embodiment, the labeling binding ligand-DBL pair can be as described above for IBL-DBL pairs. Suitable labels include, but are not limited to, fluorescent lanthanide complexes, including those of Europium and Terbium, fluorescein, rhodamine, tetramethylrhodamine, eosin, erythrosin, coumarin, methyl-coumarins, pyrene, Malacite green, stilbene, Lucifer Yellow, Cascade Blue™, Texas Red, FITC, PE, cy3, cy5 and others described in the 6th Edition of the Molecular Probes Handbook by Richard P. Haugland, hereby expressly incorporated by reference.</p>
    <p num="p-0103">In one embodiment, the label is a molecule whose color or luminescence properties change in the presence of the IBL, due to a change in the local environment. For example, the label may be: (1) a fluorescent pH indicator whose emission intensity changes with pH; (2) a fluorescent ion indicator, whose emission properties change with ion concentration; or (3) a fluorescent molecule such as an ethidium salt whose fluorescence intensity increases in hydrophobic environments.</p>
    <p num="p-0104">Accordingly, the identification of the location of the individual beads (or subpopulations of beads) is done using one or more decoding steps comprising a binding between the labeled DBL and either the IBL or the bioactive agent (i.e. a hybridization between the candidate probe and the decoder probe when the bioactive agent is a nucleic acid). After decoding, the DBLs can be removed and the array can be used; however, in some circumstances, for example when the DBL binds to an IBL and not to the bioactive agent, the removal of the DBL is not required (although it may be desirable in some circumstances). In addition, as outlined herein, decoding may be done either before the array is used in an assay, during the assay, or after the assay.</p>
    <p num="p-0105">In one embodiment, a single decoding step is done. In this embodiment, each DBL is labeled with a unique label, such that the the number of unique labels is equal to or greater than the number of bioactive agents (although in some cases, “reuse” of the unique labels can be done, as described herein; similarly, minor variants of candidate probes can share the same decoder, if the variants are encoded in another dimension, i.e. in the bead size or label). For each bioactive agent or IBL, a DBL is made that will specifically bind to it and contains a unique label, for example one or more fluorochromes. Thus, the identity of each DBL, both its composition (i.e. its sequence when it is a nucleic acid) and its label, is known. Then, by adding the DBLs to the array containing the bioactive agents under conditions which allow the formation of complexes (termed hybridization complexes when the components are nucleic acids) between the DBLs and either the bioactive agents or the IBLs, the location of each DBL can be elucidated. This allows the identification of the location of each bioactive agent; the random array has been decoded. The DBLs can then be removed, if necessary, and the target sample applied.</p>
    <p num="p-0106">In a preferred embodiment, the number of unique labels is less than the number of unique bioactive agents, and thus a sequential series of decoding steps are used. To facilitate the discussion, this embodiment is explained for nucleic acids, although other types of bioactive agents and DBLs are useful as well. In this embodiment, decoder probes are divided into n sets for decoding. The number of sets corresponds to the number of unique tags. Each decoder probe is labeled in n separate reactions with n distinct tags. All the decoder probes share the same n tags. Each pool of decoders contains only one of the n tag versions of each decoder, and no two decoder probes have the same sequence of tags across all the pools. The number of pools required for this to be true is determined by the number of decoder probes and the n. Hybridization of each pool to the array generates a signal at every address comprising an IBL. The sequential hybridization of each pool in turn will generate a unique, sequence-specific code for each candidate probe. This identifies the candidate probe at each address in the array. For example, if four tags are used, then 4×n sequential hybridizations can ideally distinguish 4<sup>n </sup>sequences, although in some cases more steps may be required. After the hybridization of each pool, the hybrids are denatured and the decoder probes removed, so that the probes are rendered single-stranded for the next hybridization (although it is also possible to hybridize limiting amounts of target so that the available probe is not saturated. Sequential hybridizations can be carried out and analyzed by subtracting pre-existing signal from the previous hybridization).</p>
    <p num="p-0107">An example is illustrative. Assuming an array of 16 probe nucleic acids (numbers 1–16), and four unique tags (four different fluors, for example; labels A–D). Decoder probes 1–16 are made that correspond to the probes on the beads. The first step is to label decoder probes 1–4 with tag A, decoder probes 5–8 with tag B, decoder probes 9–12 with tag C, and decoder probes 13–16 with tag D. The probes are mixed and the pool is contacted with the array containing the beads with the attached candidate probes. The location of each tag (and thus each decoder and candidate probe pair) is then determined. The first set of decoder probes are then removed. A second set is added, but this time, decoder probes 1, 5, 9 and 13 are labeled with tag A, decoder probes 2, 6, 10 and 14 are labeled with tag B, decoder probes 3, 7, 11 and 15 are labeled with tag C, and decoder probes 4, 8, 12 and 16 are labeled with tag D. Thus, those beads that contained tag A in both decoding steps contain candidate probe 1; tag A in the first decoding step and tag B in the second decoding step contain candidate probe 2; tag A in the first decoding step and tag C in the second step contain candidate probe 3; etc. As will be appreciated by those in the art, the decoder probes can be made in any order and added in any order.</p>
    <p num="p-0108">In one embodiment, the decoder probes are labeled in situ; that is, they need not be labeled prior to the decoding reaction. In this embodiment, the incoming decoder probe is shorter than the candidate probe, creating a 5′ “overhang” on the decoding probe. The addition of labeled ddNTPs (each labeled with a unique tag) and a polymerase will allow the addition of the tags in a sequence specific manner, thus creating a sequence-specific pattern of signals. Similarly, other modifications can be done, including ligation, etc.</p>
    <p num="p-0109">In addition, since the size of the array will be set by the number of unique decoding binding ligands, it is possible to “reuse” a set of unique DBLs to allow for a greater number of test sites. This may be done in several ways; for example, by using some subpopulations that comprise optical signatures. Similarly, the use of a positional coding scheme within an array; different sub-bundles may reuse the set of DBLs. Similarly, one embodiment utilizes bead size as a coding modality, thus allowing the reuse of the set of unique DBLs for each bead size. Alternatively, sequential partial loading of arrays with beads can also allow the reuse of DBLs. Furthermore, “code sharing” can occur as well.</p>
    <p num="p-0110">In a preferred embodiment, the DBLs may be reused by having some subpopulations of beads comprise optical signatures. In a preferred embodiment, the optical signature is generally a mixture of reporter dyes, preferably fluorescent. By varying both the composition of the mixture (i.e. the ratio of one dye to another) and the concentration of the dye (leading to differences in signal intensity), matrices of unique optical signatures may be generated. This may be done by covalently attaching the dyes to the surface of the beads, or alternatively, by entrapping the dye within the bead. The dyes may be chromophores or phosphors but are preferably fluorescent dyes, which due to their strong signals provide a good signal-to-noise ratio for decoding. Suitable dyes for use in the invention include those listed for labeling DBLs, above.</p>
    <p num="p-0111">In a preferred embodiment, the encoding can be accomplished in a ratio of at least two dyes, although more encoding dimensions may be added in the size of the beads, for example. In addition, the labels are distinguishable from one another; thus two different labels may comprise different molecules (i.e. two different fluors) or, alternatively, one label at two different concentrations or intensity.</p>
    <p num="p-0112">In a preferred embodiment, the dyes are covalently attached to the surface of the beads. This may be done as is generally outlined for the attachment of the bioactive agents, using functional groups on the surface of the beads. As will be appreciated by those in the art, these attachments are done to minimize the effect on the dye.</p>
    <p num="p-0113">In a preferred embodiment, the dyes are non-covalently associated with the beads, generally by entrapping the dyes in the pores of the beads.</p>
    <p num="p-0114">Additionally, encoding in the ratios of the two or more dyes, rather than single dye concentrations, is preferred since it provides insensitivity to the intensity of light used to interrogate the reporter dye's signature and detector sensitivity.</p>
    <p num="p-0115">In a preferred embodiment, a spatial or positional coding system is done. In this embodiment, there are sub-bundles or subarrays (i.e. portions of the total array) that are utilized. By analogy with the telephone system, each subarray is an “area code”, that can have the same labels (i.e. telephone numbers) of other subarrays, that are separated by virtue of the location of the subarray. Thus, for example, the same unique labels can be reused from bundle to bundle. Thus, the use of 50 unique labels in combination with 100 different subarrays can form an array of 5000 different bioactive agents. In this embodiment, it becomes important to be able to identify one bundle from another; in general, this is done either manually or through the use of marker beads; these can be beads containing unique tags for each subarray, or the use of the same marker bead in differing amounts, or the use of two or more marker beads in different ratios.</p>
    <p num="p-0116">In alternative embodiments, additional encoding parameters can be added, such as microsphere size. For example, the use of different size beads may also allow the reuse of sets of DBLs; that is, it is possible to use microspheres of different sizes to expand the encoding dimensions of the microspheres. Optical fiber arrays can be fabricated containing pixels with different fiber diameters or cross-sections; alternatively, two or more fiber optic bundles, each with different cross-sections of the individual fibers, can be added together to form a larger bundle; or, fiber optic bundles with fiber of the same size cross-sections can be used, but just with different sized beads. With different diameters, the largest wells can be filled with the largest microspheres and then moving onto progressively smaller microspheres in the smaller wells until all size wells are then filled. In this manner, the same dye ratio could be used to encode microspheres of different sizes thereby expanding the number of different oligonucleotide sequences or chemical functionalities present in the array. Although outlined for fiber optic substrates, this as well as the other methods outlined herein can be used with other substrates and with other attachment modalities as well.</p>
    <p num="p-0117">In a preferred embodiment, the coding and decoding is accomplished by sequential loading of the microspheres into the array. As outlined above for spatial coding, in this embodiment, the optical signatures can be “reused”. In this embodiment, the library of microspheres each comprising a different bioactive agent (or the subpopulations each comprise a different bioactive agent), is divided into a plurality of sublibraries; for example, depending on the size of the desired array and the number of unique tags, 10 sublibraries each comprising roughly 10% of the total library may be made, with each sublibrary comprising roughly the same unique tags. Then, the first sublibrary is added to the fiber optic bundle comprising the wells, and the location of each bioactive agent is determined, generally through the use of DBLs. The second sublibrary is then added, and the location of each bioactive agent is again determined. The signal in this case will comprise the signal from the “first” DBL and the “second” DBL; by comparing the two matrices the location of each bead in each sublibrary can be determined. Similarly, adding the third, fourth, etc. sublibraries sequentially will allow the array to be filled.</p>
    <p num="p-0118">In a preferred embodiment, codes can be “shared” in several ways. In a first embodiment, a single code (i.e. IBL/DBL pair) can be assigned to two or more agents if the target analytes different sufficiently in their binding strengths. For example, two nucleic acid probes used in an mRNA quantitation assay can share the same code if the ranges of their hybridization signal intensities do not overlap. This can occur, for example, when one of the target sequences is always present at a much higher concentration than the other. Alternatively, the two target sequences might always be present at a similar concentration, but differ in hybridization efficiency.</p>
    <p num="p-0119">Alternatively, a single code can be assigned to multiple agents if the agents are functionally equivalent. For example, if a set of oligonucleotide probes are designed with the common purpose of detecting the presence of a particular gene, then the probes are functionally equivalent, even though they may differ in sequence. Similarly, if classes or “families” of analytes are desired, all probes for different members of a class such as kinases or G-protein coupled receptors could share a code. Similarly, an array of this type could be used to detect homologs of known genes. In this embodiment, each gene is represented by a heterologous set of probes, hybridizing to different regions of the gene (and therefore differing in sequence). The set of probes share a common code. If a homolog is present, it might hybridize to some but not all of the probes. The level of homology might be indicated by the fraction of probes hybridizing, as well as the average hybridization intensity. Similarly, multiple antibodies to the same protein could all share the same code.</p>
    <p num="p-0120">In a preferred embodiment, decoding of self-assembled random arrays is done on the bases of pH titration. In this embodiment, in addition to bioactive agents, the beads comprise optical signatures, wherein the optical signatures are generated by the use of pH-responsive dyes (sometimes referred to herein as “pH dyes”) such as fluorophores. This embodiment is similar to that outlined in PCT U.S. Pat. No. 98/05025 and U.S. Ser. No. 09/151,877, both of which are expressly incorporated by reference, except that the dyes used in the present invention exhibits changes in fluorescence intensity (or other properties) when the solution pH is adjusted from below the pKa to above the pKa (or vice versa). In a preferred embodiment, a set of pH dyes is used, each with a different pKa, preferably separated by at least 0.5 pH units. Preferred embodiments utilize a pH dye set of pka's of 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 5.5, 6.0, 6.5, 7.0, 7.5, 8.0, 8.5, 9.0, 9.5, 10.0, 10.5, 11, and 11.5. Each bead can contain any subset of the pH dyes, and in this way a unique code for the bioactive agent is generated. Thus, the decoding of an array is achieved by titrating the array from pH 1 to pH 13, and measuring the fluorescence signal from each bead as a function of solution pH.</p>
    <p num="p-0121">Thus, the present invention provides array compositions comprising a substrate with a surface comprising discrete sites. A population of microspheres is distributed on the sites, and the population comprises at least a first and a second subpopulation. Each subpopulation comprises a bioactive agent, and, in addition, at least one optical dye with a given pKa. The pKas of the different optical dyes are different.</p>
    <p num="p-0122">In a preferred embodiment, when for example the array comprises cloned nucleic acids, there are several methods that can be used to decode the arrays. In a preferred embodiment, when some sequence information about the cloned nucleic acids is known, specific decoding probes can be made as is generally outlined herein.</p>
    <p num="p-0123">In a preferred embodiment, “random” decoding probes can be made. By sequential hybridizations or the use of multiple labels, as is outlined above, a unique hybridization pattern can be generated for each sensor element. This allows all the beads representing a given clone to be identified as belonging to the same group. In general, this is done by using random or partially degenerate decoding probes, that bind in a sequence-dependent but not highly sequence-specific manner. The process can be repeated a number of times, each time using a different labeling entity, to generate a different pattern of signals based on quasi-specific interactions. In this way, a unique optical signature is eventually built up for each sensor element. By applying pattern recognition or clustering algorithms to the optical signatures, the beads can be grouped into sets that share the same signature (i.e. carry the same probes).</p>
    <p num="p-0124">In order to identify the actual sequence of the clone itself, additional procedures are required; for example, direct sequencing can be done. By using an ordered array containing the clones, such as a spotted cDNA array, a “key” can be generated that links a hybridization pattern to a specific clone whose position in the set is known. In this way the clone can be recovered and further characterized.</p>
    <p num="p-0125">Alternatively, clone arrays can be decoded using binary decoding with vector tags. For example, partially randomized oligos are cloned into a nucleic acid vector (e.g. plasmid, phage, etc.). Each oligonucleotide sequence consists of a subset of a limited set of sequences. For example, if the limites set comprises 10 sequences, each oligonucleotide may have some subset (or all of the 10) sequences. Thus each of the 10 sequences can be present or absent in the oligonucleotide. Therefore, there are 2<sup>10 </sup>or 1,024 possible combinations. The sequences may overlap, and minor variants can also be represented (e.g. A, C, T and G substitutions) to increase the number of possible combinations. A nucleic acid library is cloned into a vector containing the random code sequences. Alternatively, other methods such as PCR can be used to add the tags. In this way it is possible to use a small number of oligo decoding probes to decode an array of clones.</p>
    <p num="p-0126">Once made, the compositions of the invention find use in a number of applications. In a preferred embodiment, the compositions are used to probe a sample solution for the presence or absence of a target analyte, including the quantification of the amount of target analyte present. By “target analyte” or “analyte” or grammatical equivalents herein is meant any atom, molecule, ion, molecular ion, compound or particle to be either detected or evaluated for binding partners. As will be appreciated by those in the art, a large number of analytes may be used in the present invention; basically, any target analyte can be used which binds a bioactive agent or for which a binding partner (i.e. drug candidate) is sought.</p>
    <p num="p-0127">Suitable analytes include organic and inorganic molecules, including biomolecules. When detection of a target analyte is done, suitable target analytes include, but are not limited to, an environmental pollutant (including pesticides, insecticides, toxins, etc.); a chemical (including solvents, polymers, organic materials, etc.); therapeutic molecules (including therapeutic and abused drugs, antibiotics, etc.); biomolecules (including hormones, cytokines, proteins, nucleic acids, lipids, carbohydrates, cellular membrane antigens and receptors (neural, hormonal, nutrient, and cell surface receptors) or their ligands, etc); whole cells (including procaryotic (such as pathogenic bacteria) and eukaryotic cells, including mammalian tumor cells); viruses (including retroviruses, herpesviruses, adenoviruses, lentiviruses, etc.); and spores; etc. Particularly preferred analytes are nucleic acids and proteins.</p>
    <p num="p-0128">In a preferred embodiment, the target analyte is a protein. As will be appreciated by those in the art, there are a large number of possible proteinaceous target analytes that may be detected or evaluated for binding partners using the present invention. Suitable protein target analytes include, but are not limited to, (1) immunoglobulins; (2) enzymes (and other proteins); (3) hormones and cytokines (many of which serve as ligands for cellular receptors); and (4) other proteins.</p>
    <p num="p-0129">In a preferred embodiment, the target analyte is a nucleic acid. These assays find use in a wide variety of applications.</p>
    <p num="p-0130">In a preferred embodiment, the probes are used in genetic diagnosis. For example, probes can be made using the techniques disclosed herein to detect target sequences such as the gene for nonpolyposis colon cancer, the BRCA1 breast cancer gene, P53, which is a gene associated with a variety of cancers, the Apo E4 gene that indicates a greater risk of Alzheimer's disease, allowing for easy presymptomatic screening of patients, mutations in the cystic fibrosis gene, cytochrome p450s or any of the others well known in the art.</p>
    <p num="p-0131">In an additional embodiment, viral and bacterial detection is done using the complexes of the invention. In this embodiment, probes are designed to detect target sequences from a variety of bacteria and viruses. For example, current blood-screening techniques rely on the detection of anti-HIV antibodies. The methods disclosed herein allow for direct screening of clinical samples to detect HIV nucleic acid sequences, particularly highly conserved HIV sequences. In addition, this allows direct monitoring of circulating virus within a patient as an improved method of assessing the efficacy of anti-viral therapies. Similarly, viruses associated with leukemia, HTLV-I and HTLV-II, may be detected in this way. Bacterial infections such as tuberculosis, chlamydia and other sexually transmitted diseases, may also be detected.</p>
    <p num="p-0132">In a preferred embodiment, the nucleic acids of the invention find use as probes for toxic bacteria in the screening of water and food samples. For example, samples may be treated to lyse the bacteria to release its nucleic acid, and then probes designed to recognize bacterial strains, including, but not limited to, such pathogenic strains as, Salmonella, Campylobacter, <i>Vibrio cholerae</i>, Leishmania, enterotoxic strains of <i>E. coli</i>, and Legionnaire's disease bacteria. Similarly, bioremediation strategies may be evaluated using the compositions of the invention.</p>
    <p num="p-0133">In a further embodiment, the probes are used for forensic “DNA fingerprinting” to match crime-scene DNA against samples taken from victims and suspects.</p>
    <p num="p-0134">In an additional embodiment, the probes in an array are used for sequencing by hybridization.</p>
    <p num="p-0135">The present invention also finds use as a methodology for the detection of mutations or mismatches in target nucleic acid sequences. For example, recent focus has been on the analysis of the relationship between genetic variation and phenotype by making use of polymorphic DNA markers. Previous work utilized short tandem repeats (STRs) as polymorphic positional markers; however, recent focus is on the use of single nucleotide polymorphisms (SNPs). Common SNPs occur at an average frequency of more than 1 per kilobase in human genomic DNA. Some SNPs, particularly those in and around coding sequences, are likely to be the direct cause of therapeutically relevant phenotypic variants. There are a number of well known polymorphisms that cause clinically important phenotypes; for example, the apoE2/3/4 variants are associated with different relative risk of Alzheimer's and other diseases (see Cordor et al., Science 261(1993). Multiplex PCR amplification of SNP loci with subsequent hybridization to oligonucleotide arrays has been shown to be an accurate and reliable method of simultaneously genotyping at least hundreds of SNPs; see Wang et al., Science, 280:1077 (1998); see also Schafer et al., Nature Biotechnology 16:33–39 (1998). The compositions of the present invention may easily be substituted for the arrays of the prior art.</p>
    <p num="p-0136">In a preferred embodiment, the compositions of the invention are used to screen bioactive agents to find an agent that will bind, and preferably modify the function of, a target molecule. As above, a wide variety of different assay formats may be run, as will be appreciated by those in the art. Generally, the target analyte for which a binding partner is desired is labeled; binding of the target analyte by the bioactive agent results in the recruitment of the label to the bead, with subsequent detection.</p>
    <p num="p-0137">In a preferred embodiment, the binding of the bioactive agent and the target analyte is specific; that is, the bioactive agent specifically binds to the target analyte. By “specifically bind” herein is meant that the agent binds the analyte, with specificity sufficient to differentiate between the analyte and other components or contaminants of the test sample. However, as will be appreciated by those in the art, it will be possible to detect analytes using binding which is not highly specific; for example, the systems may use different binding ligands, for example an array of different ligands, and detection of any particular analyte is via its “signature” of binding to a panel of binding ligands, similar to the manner in which “electronic noses” work. This finds particular utility in the detection of chemical analytes. The binding should be sufficient to remain bound under the conditions of the assay, including wash steps to remove non-specific binding, although in some embodiments, wash steps are not desired; i.e. for detecting low affinity binding partners. In some embodiments, for example in the detection of certain biomolecules, the dissociation constants of the analyte to the binding ligand will be less than about 10<sup>−4</sup>–10<sup>−6 </sup>M<sup>−1</sup>, with less than about 10<sup>−5 </sup>to 10<sup>−9 </sup>M<sup>−1 </sup>being preferred and less than about 10<sup>−</sup>–−9 M<sup>−1 </sup>being particularly preferred.</p>
    <p num="p-0138">Generally, a sample containing a target analyte (whether for detection of the target analyte or screening for binding partners of the target analyte) is added to the array, under conditions suitable for binding of the target analyte to at least one of the bioactive agents, i.e. generally physiological conditions. The presence or absence of the target analyte is then detected. As will be appreciated by those in the art, this may be done in a variety of ways, generally through the use of a change in an optical signal. This change can occur via many different mechanisms. A few examples include the binding of a dye-tagged analyte to the bead, the production of a dye species on or near the beads, the destruction of an existing dye species, a change in the optical signature upon analyte interaction with dye on bead, or any other optically interrogatable event.</p>
    <p num="p-0139">In a preferred embodiment, the change in optical signal occurs as a result of the binding of a target analyte that is labeled, either directly or indirectly, with a detectable label, preferably an optical label such as a fluorochrome. Thus, for example, when a proteinaceous target analyte is used, it may be either directly labeled with a fluor, or indirectly, for example through the use of a labeled antibody. Similarly, nucleic acids are easily labeled with fluorochromes, for example during PCR amplification as is known in the art. Alternatively, upon binding of the target sequences, a hybridization indicator may be used as the label. Hybridization indicators preferentially associate with double stranded nucleic acid, usually reversibly. Hybridization indicators include intercalators and minor and/or major groove binding moieties. In a preferred embodiment, intercalators may be used; since intercalation generally only occurs in the presence of double stranded nucleic acid, only in the presence of target hybridization will the label light up. Thus, upon binding of the target analyte to a bioactive agent, there is a new optical signal generated at that site, which then may be detected.</p>
    <p num="p-0140">Alternatively, in some cases, as discussed above, the target analyte such as an enzyme generates a species that is either directly or indirectly optically detectable.</p>
    <p num="p-0141">Furthermore, in some embodiments, a change in the optical signature may be the basis of the optical signal. For example, the interaction of some chemical target analytes with some fluorescent dyes on the beads may alter the optical signature, thus generating a different optical signal.</p>
    <p num="p-0142">As will be appreciated by those in the art, in some embodiments, the presence or absence of the target analyte may be done using changes in other optical or non-optical signals, including, but not limited to, surface enhanced Raman spectroscopy, surface plasmon resonance, radioactivity, etc.</p>
    <p num="p-0143">The assays may be run under a variety of experimental conditions, as will be appreciated by those in the art. A variety of other reagents may be included in the screening assays. These include reagents like salts, neutral proteins, e.g. albumin, detergents, etc which may be used to facilitate optimal protein-protein binding and/or reduce non-specific or background interactions. Also reagents that otherwise improve the efficiency of the assay, such as protease inhibitors, nuclease inhibitors, anti-microbial agents, etc., may be used. The mixture of components may be added in any order that provides for the requisite binding. Various blocking and washing steps may be utilized as is known in the art.</p>
    <p num="p-0144">In a preferred embodiment, two-color competitive hybridization assays are run. These assays can be based on traditional sandwich assays. The beads contain a capture sequence located on one side (upstream or downstream) of the SNP, to capture the target sequence. Two SNP allele-specific probes, each labeled with a different fluorophor, are hybridized to the target sequence. The genotype can be obtained from a ratio of the two signals, with the correct sequence generally exhibiting better binding. This has an advantage in that the target sequence itself need not be labeled. In addition, since the probes are competing, this means that the conditions for binding need not be optimized. Under conditions where a mismatched probe would be stably bound, a matched probe can still displace it. Therefore the competitive assay can provide better discrimination under those conditions. Because many assays are carried out in parallel, conditions cannot be optimzed for every probe simultaneously. Therefore, a competitive assay system can be used to help compensate for non-optimal conditons for mismatch discrimination.</p>
    <p num="p-0145">In a preferred embodiment, dideoxynucleotide chain-termination sequencing is done using the compositions of the invention. In this embodiment, a DNA polymerase is used to extend a primer using fluorescently labeled ddNTPs. The 3′ end of the primer is located adjacent to the SNP site. In this way, the single base extension is complementary to the sequence at the SNP site. By using four different fluorophors, one for each base, the sequence of the SNP can be deduced by comparing the four base-specific signals. This may be done in several ways. In a first embodiment, the capture probe can be extended; in this approach, the probe must either be synthesized 5′-3′ on the bead, or attached at the 5′ end, to provide a free 3′ end for polymerase extension. Alternatively, a sandwich type assay can be used; in this embodiment, the target is captured on the bead by a probe, then a primer is annealed and extended. Again, in the latter case, the target sequence need not be labeled. In addition, since sandwich assays require two specific interactions, this provides increased specificity which is particularly helpful for the analysis of complex samples.</p>
    <p num="p-0146">In addition, when the target analyte and the DBL both bind to the agent, it is also possible to do detection of non-labelled target analytes via competition of decoding.</p>
    <p num="p-0147">In a preferred embodiment, the methods of the invention are useful in array quality control. Prior to this invention, no methods have been described that provide a positive test of the performance of every probe on every array. Decoding of the array not only provides this test, it also does so by making use of the data generated during the decoding process itself. Therefore, no additional experimental work is required. The invention requires only a set of data analysis algorithms that can be encoded in software.</p>
    <p num="p-0148">The quality control procedure can identify a wide variety of systematic and random problems in an array. For example, random specks of dust or other contaminants might cause some sensors to give an incorrect signal—this can be detected during decoding. The omission of one or more agents from multiple arrays can also be detected; that is, since the arrays formulated herein are randomly self-assembled, arrays comprising different statistical distributions of each sensor elements may be made; the invention allows actual QC on the arrays, for “sorting” arrays with good distributions from those with less than ideal distributions. These procedures also allow the determination of sites that do not comprise beads.</p>
    <p num="p-0149">An advantage of this quality control procedure is that it can be implemented immediately prior to the assay itself (or after, in some instances), and is a true functional test of each individual sensor. Therefore any problems that might occur between array assembly and actual use can be detected. In applications where a very high level of confidence is required, and/or there is a significant chance of sensor failure during the experimental procedure, decoding and quality control can be conducted both before and after the actual sample analysis.</p>
    <p num="p-0150">In a preferred embodiment, the arrays can be used to do reagent quality control. In many instances, biological macromolecules are used as reagents and must be quality controlled. For example, large sets of oligonucleotide probes may be provided as reagents. It is typically difficult to perform quality control on large numbers of different biological macromolecules. For example, when large populations of different random oligonucleotides are synthesized, it may be desirable to check that each population is represented. The approach described here can be used to do this by treating the reagents (formulated as the DBLs) as variable instead of the arrays.</p>
    <p num="p-0151">In a preferred embodiment, the methods outlined herein are used in array calibration. For many applications, such as mRNA quantitation, it is desirable to have a signal that is a linear response to the concentration of the target analyte, or, alternatively, if non-linear, to determine a relationship between concentration and signal, so that the concentration of the target analyte can be estimated. For example, different probes can show differences in hybridization efficiencies, etc., and it may be desirable to choose appropriate probes under experimental conditions. Thus, for example, when expression profiling is done, and quantitation is desirable, calibration curves can be done to see how individual probes react, and then the probes that give the best response (i.e. linearity at the concentrations and conditions of interest) can be chosen for further assays. Accordingly, the present invention provides methods of creating calibration curves in parallel for multiple beads in an array. The calibration curves can be created under conditions that simulate the complexity of the sample to be analyzed. Each curve can be constructed independently of the others (e.g. for a different range of concentrations), but at the same time as all the other curves for the array.</p>
    <p num="p-0152">Thus, in these embodiments, different types of experiments can be done. For example, the sequential decoding scheme can be implemented with different concentrations being used as the code “labels”, rather than different fluorophores. In this way, signal as a response to concentration can be measured for each bead. This calibration can be carried out just prior to array use, so that every probe on every array is individually calibrated as needed. Alternatively, different concentrations of the decoding probe can have different labels.</p>
    <p num="p-0153">It should be noted that the assay calibration methods find use in non-random arrays, as well; that is, other types of support-bound nucleic acid arrays can be calibrated using these methods as well. Thus, for example, the sequential addition of different pools of probes, wherein the concentration of the probes is varied, to biochips can allow calibration of any assay system. This type of analysis can also be done on non-random arrays for quality control, to verify the integrity and sequence of the support bound probes, and in assay development to identify good probes.</p>
    <p num="p-0154">In a preferred embodiment, the methods of the invention can be used in assay development as well. Thus, for example, the methods allow the identification of good and bad probes; as is understood by those in the art, some probes do not function well because they do not hybridize well, or because they cross-hybridize with more than one sequence. These problems are easily detected during decoding. The ability to rapidly assess probe performance has the potential to greatly reduce the time and expense of assay development. Thus, probes that respond linearly with concentration, show low non-specific binding, or give signals in a particular range, can be chosen for addition to a new array for assays.</p>
    <p num="p-0155">Similarly, in a preferred embodiment, the methods of the invention are useful in quantitation in assay development. Major challenges of many assays is the ability to detect differences in analyte concentrations between samples, the ability to quantitate these differences, and to measure absolute concentrations of analytes, all in the presence of a complex mixture of related analytes. An example of this problem is the quantitation of a specific mRNA in the presence of total cellular mRNA. One approach that has been developed as a basis of mRNA quantitation makes use of a multiple match and mismatch probe pairs (Lockhart et al., 1996), hereby incorporated by reference in its entirety. While this approach is simple, it requires relatively large numbers of probes. In this approach, a quantitative response to concentration is obtained by averaging the signals from a set of different probes to the gene or sequence of interest. This is necessary because only some probes respond quantitatively, and it is not possible to predict these probes with certainty. In the absence of prior knowledge, only the average response of an appropriately chosen collection of probes is quantitative. However, in the present invention, that can be applied generally to nucleic acid based assays as well as other assays. In essence, the approach is to identify the probes that respond quantitatively in a particular assay, rather than average them with other probes. This is done using the array calibration scheme outlined above, in which concentration-based codes are used. Advantages of this approach include: fewer probes are needed; the accuracy of the measurement is less dependent on the number of probes used; and that the response of the sensors is known with a high level of certainty, since each and every sequence can be tested in an efficient manner. It is important to note that probes that perform well are selected empirically, which avoids the difficulties and uncertainties of predicting probe performance, particularly in complex sequence mixtures. In contrast, in experiments described to date with ordered arrays, relatively small numbers of sequences are checked by performing quantitative spiking experiments, in which a known mRNA is added to a mixture.</p>
    <p num="p-0156">In a preferred embodiment, cDNA arrays are made for RNA expression profiling. In this embodiment, individual cDNA clones are amplified (for example, using PCR) from cDNA libraries propagated in a host-vector system. Each amplified DNA is attached to a population of beads. Different populations are mixed together, to create a collection of beads representing the cDNA library. The beads are arrayed, decoded as outlined above, and used in an assay (although as outlined herein, decoding may occur after assay as well). The assay is done using RNA samples (whole cell or mRNA) that are extracted, labeled if necessary, and hybridized to the array. Comparative analysis allows the detection of differences in the expression levels of individual RNAs. Comparison to an appropriate set of calibration standards allows quantification of absolute amounts of RNA.</p>
    <p num="p-0157">The cDNA array can also be used for mapping, e.g. to map deletions/insertions or copy number changes in the genome, for example from tumors or other tissue samples. This can be done by hybridizing genomic DNA. Instead of cDNAs (or ESTs, etc.), other STS (sequence tagged sites), including random genomic fragments, can also be arrayed for this purpose.</p>
    <p num="p-0158">All references cited herein are incorporated by reference in their entirety.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3712535">US3712535</a></td><td class="patent-data-table-td patent-date-value">Sep 2, 1971</td><td class="patent-data-table-td patent-date-value">Jan 23, 1973</td><td class="patent-data-table-td ">Becton Dickinson Co</td><td class="patent-data-table-td ">Centrifuge rotor and sample holder with agitating means</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3748975">US3748975</a></td><td class="patent-data-table-td patent-date-value">Jan 3, 1972</td><td class="patent-data-table-td patent-date-value">Jul 31, 1973</td><td class="patent-data-table-td ">Rca Corp</td><td class="patent-data-table-td ">Apparatus for and method of correcting a defective photomask</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4125314">US4125314</a></td><td class="patent-data-table-td patent-date-value">May 5, 1977</td><td class="patent-data-table-td patent-date-value">Nov 14, 1978</td><td class="patent-data-table-td ">Dr. Ralph M. Grant Engineering Consultants, Inc.</td><td class="patent-data-table-td ">Beam ratio control for holography</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4200110">US4200110</a></td><td class="patent-data-table-td patent-date-value">Nov 28, 1977</td><td class="patent-data-table-td patent-date-value">Apr 29, 1980</td><td class="patent-data-table-td ">United States Of America</td><td class="patent-data-table-td ">Fiber optic pH probe</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4224198">US4224198</a></td><td class="patent-data-table-td patent-date-value">May 26, 1978</td><td class="patent-data-table-td patent-date-value">Sep 23, 1980</td><td class="patent-data-table-td ">California Institute Of Technology</td><td class="patent-data-table-td ">Protein specific polymeric immunomicrospheres</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4448485">US4448485</a></td><td class="patent-data-table-td patent-date-value">Jul 10, 1981</td><td class="patent-data-table-td patent-date-value">May 15, 1984</td><td class="patent-data-table-td ">Bell Telephone Laboratories, Incorporated</td><td class="patent-data-table-td ">Harmonic generation using a surface of metal particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4499052">US4499052</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 1982</td><td class="patent-data-table-td patent-date-value">Feb 12, 1985</td><td class="patent-data-table-td ">Becton, Dickinson And Company</td><td class="patent-data-table-td ">Apparatus for distinguishing multiple subpopulations of cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4582789">US4582789</a></td><td class="patent-data-table-td patent-date-value">Dec 18, 1984</td><td class="patent-data-table-td patent-date-value">Apr 15, 1986</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">Containing alkylating intercalation moiety</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4682895">US4682895</a></td><td class="patent-data-table-td patent-date-value">Aug 6, 1985</td><td class="patent-data-table-td patent-date-value">Jul 28, 1987</td><td class="patent-data-table-td ">Texas A&amp;M University</td><td class="patent-data-table-td ">Fiber optic probe for quantification of colorimetric reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4687732">US4687732</a></td><td class="patent-data-table-td patent-date-value">Jun 10, 1983</td><td class="patent-data-table-td patent-date-value">Aug 18, 1987</td><td class="patent-data-table-td ">Yale University</td><td class="patent-data-table-td ">Visualization polymers and their application to diagnostic medicine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4729949">US4729949</a></td><td class="patent-data-table-td patent-date-value">Nov 8, 1983</td><td class="patent-data-table-td patent-date-value">Mar 8, 1988</td><td class="patent-data-table-td ">Bar-Ilan University</td><td class="patent-data-table-td ">System and methods for cell selection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4772540">US4772540</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 1985</td><td class="patent-data-table-td patent-date-value">Sep 20, 1988</td><td class="patent-data-table-td ">Bar Ilan University</td><td class="patent-data-table-td ">Applying photoresist, etching, electroplating metal, hardening alloy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4785814">US4785814</a></td><td class="patent-data-table-td patent-date-value">Aug 11, 1987</td><td class="patent-data-table-td patent-date-value">Nov 22, 1988</td><td class="patent-data-table-td ">Cordis Corporation</td><td class="patent-data-table-td ">Optical probe for measuring pH and oxygen in blood and employing a composite membrane</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4822746">US4822746</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 1986</td><td class="patent-data-table-td patent-date-value">Apr 18, 1989</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">Mixing absorber ligand and fluorophor, transferring energy, emission of light of different wavelength</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4824789">US4824789</a></td><td class="patent-data-table-td patent-date-value">Oct 10, 1986</td><td class="patent-data-table-td patent-date-value">Apr 25, 1989</td><td class="patent-data-table-td ">Cardiovascular Devices, Inc.</td><td class="patent-data-table-td ">Gas sensor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4842783">US4842783</a></td><td class="patent-data-table-td patent-date-value">Sep 3, 1987</td><td class="patent-data-table-td patent-date-value">Jun 27, 1989</td><td class="patent-data-table-td ">Cordis Corporation</td><td class="patent-data-table-td ">Adhering the polymer to a waveguide, crosslinking by radiation and absorbing a dye</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4868130">US4868130</a></td><td class="patent-data-table-td patent-date-value">Feb 20, 1987</td><td class="patent-data-table-td patent-date-value">Sep 19, 1989</td><td class="patent-data-table-td ">Biotope, Inc.</td><td class="patent-data-table-td ">Methods for conducting specific binding assays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4876187">US4876187</a></td><td class="patent-data-table-td patent-date-value">Dec 5, 1985</td><td class="patent-data-table-td patent-date-value">Oct 24, 1989</td><td class="patent-data-table-td ">Meiogenics, Inc.</td><td class="patent-data-table-td ">Nucleic acid compositions with scissile linkage useful for detecting nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4879097">US4879097</a></td><td class="patent-data-table-td patent-date-value">Apr 4, 1986</td><td class="patent-data-table-td patent-date-value">Nov 7, 1989</td><td class="patent-data-table-td ">Whitehead Thomas P</td><td class="patent-data-table-td ">Dispensing device and recording apparatus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4894343">US4894343</a></td><td class="patent-data-table-td patent-date-value">Nov 18, 1987</td><td class="patent-data-table-td patent-date-value">Jan 16, 1990</td><td class="patent-data-table-td ">Hitachi, Ltd.</td><td class="patent-data-table-td ">Chamber plate for use in cell fusion and a process for production thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4895805">US4895805</a></td><td class="patent-data-table-td patent-date-value">Aug 24, 1988</td><td class="patent-data-table-td patent-date-value">Jan 23, 1990</td><td class="patent-data-table-td ">Hitachi, Ltd.</td><td class="patent-data-table-td ">Cell manipulating apparatus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4981783">US4981783</a></td><td class="patent-data-table-td patent-date-value">Apr 16, 1986</td><td class="patent-data-table-td patent-date-value">Jan 1, 1991</td><td class="patent-data-table-td ">Montefiore Medical Center</td><td class="patent-data-table-td ">Method for detecting pathological conditions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4999306">US4999306</a></td><td class="patent-data-table-td patent-date-value">Apr 28, 1988</td><td class="patent-data-table-td patent-date-value">Mar 12, 1991</td><td class="patent-data-table-td ">Minnesota Mining And Manufacturing Company</td><td class="patent-data-table-td ">Composition, apparatus and method for sensing ionic components</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5002867">US5002867</a></td><td class="patent-data-table-td patent-date-value">Oct 24, 1988</td><td class="patent-data-table-td patent-date-value">Mar 26, 1991</td><td class="patent-data-table-td ">Macevicz Stephen C</td><td class="patent-data-table-td ">Nucleic acid sequence determination by multiple mixed oligonucleotide probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5015843">US5015843</a></td><td class="patent-data-table-td patent-date-value">Feb 15, 1990</td><td class="patent-data-table-td patent-date-value">May 14, 1991</td><td class="patent-data-table-td ">Polysense, Inc.</td><td class="patent-data-table-td ">Fiber optic chemical sensors based on polymer swelling</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5019350">US5019350</a></td><td class="patent-data-table-td patent-date-value">Feb 13, 1986</td><td class="patent-data-table-td patent-date-value">May 28, 1991</td><td class="patent-data-table-td ">Pfizer Hospital Products, Inc.</td><td class="patent-data-table-td ">Fluorescent polymers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5026599">US5026599</a></td><td class="patent-data-table-td patent-date-value">Mar 27, 1990</td><td class="patent-data-table-td patent-date-value">Jun 25, 1991</td><td class="patent-data-table-td ">Minnesota Mining &amp; Manufacturing</td><td class="patent-data-table-td ">Array of densely packed discrete metal microspheres coated on a substrate</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5028545">US5028545</a></td><td class="patent-data-table-td patent-date-value">Jun 9, 1988</td><td class="patent-data-table-td patent-date-value">Jul 2, 1991</td><td class="patent-data-table-td ">Wallac Oy</td><td class="patent-data-table-td ">Biospecific multianalyte assay method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5061336">US5061336</a></td><td class="patent-data-table-td patent-date-value">May 1, 1989</td><td class="patent-data-table-td patent-date-value">Oct 29, 1991</td><td class="patent-data-table-td ">Soane Technologies, Inc.</td><td class="patent-data-table-td ">Coating walls with adhesion promoter, filling with gelable material, adjustable heating, gelation yield, void-free gel</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5071531">US5071531</a></td><td class="patent-data-table-td patent-date-value">May 1, 1989</td><td class="patent-data-table-td patent-date-value">Dec 10, 1991</td><td class="patent-data-table-td ">Soane Technologies, Inc.</td><td class="patent-data-table-td ">Casting of gradient gels</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5105305">US5105305</a></td><td class="patent-data-table-td patent-date-value">Jan 10, 1991</td><td class="patent-data-table-td patent-date-value">Apr 14, 1992</td><td class="patent-data-table-td ">At&amp;T Bell Laboratories</td><td class="patent-data-table-td ">Near-field scanning optical microscope using a fluorescent probe</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5110745">US5110745</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 1989</td><td class="patent-data-table-td patent-date-value">May 5, 1992</td><td class="patent-data-table-td ">The Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Diagnosis of chronic hyperglycemia</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5114864">US5114864</a></td><td class="patent-data-table-td patent-date-value">Sep 19, 1989</td><td class="patent-data-table-td patent-date-value">May 19, 1992</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">Fiber optic sensors, apparatus, and detection methods using fluid erodible controlled release polymers for delivery of reagent formulations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5126022">US5126022</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 1990</td><td class="patent-data-table-td patent-date-value">Jun 30, 1992</td><td class="patent-data-table-td ">Soane Tecnologies, Inc.</td><td class="patent-data-table-td ">Method and device for moving molecules by the application of a plurality of electrical fields</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5132242">US5132242</a></td><td class="patent-data-table-td patent-date-value">Nov 13, 1989</td><td class="patent-data-table-td patent-date-value">Jul 21, 1992</td><td class="patent-data-table-td ">Cheung Sau W</td><td class="patent-data-table-td ">Fluorescent microspheres and methods of using them</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5135627">US5135627</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 1990</td><td class="patent-data-table-td patent-date-value">Aug 4, 1992</td><td class="patent-data-table-td ">Soane Technologies, Inc.</td><td class="patent-data-table-td ">Mosaic microcolumns, slabs, and separation media for electrophoresis and chromatography</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5143853">US5143853</a></td><td class="patent-data-table-td patent-date-value">Feb 2, 1989</td><td class="patent-data-table-td patent-date-value">Sep 1, 1992</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">Absorbance modulated fluorescence detection methods and sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5176881">US5176881</a></td><td class="patent-data-table-td patent-date-value">Sep 25, 1991</td><td class="patent-data-table-td patent-date-value">Jan 5, 1993</td><td class="patent-data-table-td ">The University Of Tennessee Research Corporation</td><td class="patent-data-table-td ">Fiber optic-based regenerable biosensor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5177012">US5177012</a></td><td class="patent-data-table-td patent-date-value">Aug 27, 1990</td><td class="patent-data-table-td patent-date-value">Jan 5, 1993</td><td class="patent-data-table-td ">Korea Advanced Institute Of Science &amp; Technology</td><td class="patent-data-table-td ">Biosensor containing immobilized Zymomonas mobilis cells for measuring glucose, fructose and sucrose</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5185178">US5185178</a></td><td class="patent-data-table-td patent-date-value">May 29, 1991</td><td class="patent-data-table-td patent-date-value">Feb 9, 1993</td><td class="patent-data-table-td ">Minnesota Mining And Manufacturing Company</td><td class="patent-data-table-td ">Vacuum depositon; surface at least at melting point temperatuer; condensation in form of metal droplets</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5185243">US5185243</a></td><td class="patent-data-table-td patent-date-value">Aug 25, 1988</td><td class="patent-data-table-td patent-date-value">Feb 9, 1993</td><td class="patent-data-table-td ">Syntex (U.S.A.) Inc.</td><td class="patent-data-table-td ">Hybridizing two nucleotide sequences with a target nucleotide, covalently attaching two nucleotide sequences and determining the presence of covalently attached nucleotide sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5194300">US5194300</a></td><td class="patent-data-table-td patent-date-value">Nov 8, 1989</td><td class="patent-data-table-td patent-date-value">Mar 16, 1993</td><td class="patent-data-table-td ">Cheung Sau W</td><td class="patent-data-table-td ">Reacting acrlic latex bead with amine terminated dye and diamine; transacylation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5244636">US5244636</a></td><td class="patent-data-table-td patent-date-value">Jan 25, 1991</td><td class="patent-data-table-td patent-date-value">Sep 14, 1993</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">Analysis of oxygen and carbon dioxide, measurement of pH</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5244813">US5244813</a></td><td class="patent-data-table-td patent-date-value">Apr 20, 1992</td><td class="patent-data-table-td patent-date-value">Sep 14, 1993</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">Fiber optic sensor, apparatus, and methods for detecting an organic analyte in a fluid or vapor sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5250264">US5250264</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 1992</td><td class="patent-data-table-td patent-date-value">Oct 5, 1993</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">Method of making imaging fiber optic sensors to concurrently detect multiple analytes of interest in a fluid sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5252494">US5252494</a></td><td class="patent-data-table-td patent-date-value">Mar 30, 1992</td><td class="patent-data-table-td patent-date-value">Oct 12, 1993</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">Fiber optic sensors, apparatus, and detection methods using controlled release polymers and reagent formulations held within a polymeric reaction matrix</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5254477">US5254477</a></td><td class="patent-data-table-td patent-date-value">Sep 19, 1991</td><td class="patent-data-table-td patent-date-value">Oct 19, 1993</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">Flourescence intramolecular energy transfer conjugate compositions and detection methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5272081">US5272081</a></td><td class="patent-data-table-td patent-date-value">Feb 4, 1991</td><td class="patent-data-table-td patent-date-value">Dec 21, 1993</td><td class="patent-data-table-td ">Bar-Ilan University</td><td class="patent-data-table-td ">System and methods for cell selection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5296375">US5296375</a></td><td class="patent-data-table-td patent-date-value">May 1, 1992</td><td class="patent-data-table-td patent-date-value">Mar 22, 1994</td><td class="patent-data-table-td ">Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Mesoscale sperm handling devices</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5298741">US5298741</a></td><td class="patent-data-table-td patent-date-value">Jan 13, 1993</td><td class="patent-data-table-td patent-date-value">Mar 29, 1994</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">Thin film fiber optic sensor array and apparatus for concurrent viewing and chemical sensing of a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5302509">US5302509</a></td><td class="patent-data-table-td patent-date-value">Feb 27, 1991</td><td class="patent-data-table-td patent-date-value">Apr 12, 1994</td><td class="patent-data-table-td ">Beckman Instruments, Inc.</td><td class="patent-data-table-td ">Method for sequencing polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5304487">US5304487</a></td><td class="patent-data-table-td patent-date-value">May 1, 1992</td><td class="patent-data-table-td patent-date-value">Apr 19, 1994</td><td class="patent-data-table-td ">Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Device for analyzing fluid cell-containing sample consisting of solid substrate with inlet port, flow channel, cell handling region with lysing structure, detection system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5308771">US5308771</a></td><td class="patent-data-table-td patent-date-value">Apr 13, 1992</td><td class="patent-data-table-td patent-date-value">May 3, 1994</td><td class="patent-data-table-td ">Geo-Centers, Inc.</td><td class="patent-data-table-td ">Chemical sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5310674">US5310674</a></td><td class="patent-data-table-td patent-date-value">Jul 29, 1991</td><td class="patent-data-table-td patent-date-value">May 10, 1994</td><td class="patent-data-table-td ">Bar-Ilan University</td><td class="patent-data-table-td ">Apertured cell carrier</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5320814">US5320814</a></td><td class="patent-data-table-td patent-date-value">Nov 25, 1992</td><td class="patent-data-table-td patent-date-value">Jun 14, 1994</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">With photopolymerizable light absorbing dyes; fluorophores, chromophores</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5326691">US5326691</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 21, 1991</td><td class="patent-data-table-td patent-date-value">Jul 5, 1994</td><td class="patent-data-table-td ">John Hozier</td><td class="patent-data-table-td ">Gene cloning</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5342737">US5342737</a></td><td class="patent-data-table-td patent-date-value">Apr 27, 1992</td><td class="patent-data-table-td patent-date-value">Aug 30, 1994</td><td class="patent-data-table-td ">The United States Of America As Represented By The Secretary Of The Navy</td><td class="patent-data-table-td ">Forming photoresist layer on substrate, expoosing to actinic radiation, developing and metallization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5357590">US5357590</a></td><td class="patent-data-table-td patent-date-value">Apr 7, 1993</td><td class="patent-data-table-td patent-date-value">Oct 18, 1994</td><td class="patent-data-table-td ">Siemens Aktiengesellshcaft</td><td class="patent-data-table-td ">Device for optically coupling a plurality of first optical waveguides to a plurality of second optical waveguides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5380489">US5380489</a></td><td class="patent-data-table-td patent-date-value">Feb 18, 1992</td><td class="patent-data-table-td patent-date-value">Jan 10, 1995</td><td class="patent-data-table-td ">Eastman Kodak Company</td><td class="patent-data-table-td ">Element and method for nucleic acid amplification and detection using adhered probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5435724">US5435724</a></td><td class="patent-data-table-td patent-date-value">Jul 6, 1994</td><td class="patent-data-table-td patent-date-value">Jul 25, 1995</td><td class="patent-data-table-td ">International Business Machines Corporation</td><td class="patent-data-table-td ">Dental procedures and apparatus using ultraviolet radiation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5474895">US5474895</a></td><td class="patent-data-table-td patent-date-value">Nov 14, 1991</td><td class="patent-data-table-td patent-date-value">Dec 12, 1995</td><td class="patent-data-table-td ">Siska Diagnostics Inc.</td><td class="patent-data-table-td ">Non-isotopic detection of nucleic acids using a polystyrene support-based sandwich hybridization assay and compositions useful therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5481629">US5481629</a></td><td class="patent-data-table-td patent-date-value">May 31, 1994</td><td class="patent-data-table-td patent-date-value">Jan 2, 1996</td><td class="patent-data-table-td ">Fujitsu Limited</td><td class="patent-data-table-td ">Hybrid optical IC with optical axes at different levels</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5486335">US5486335</a></td><td class="patent-data-table-td patent-date-value">Apr 24, 1995</td><td class="patent-data-table-td patent-date-value">Jan 23, 1996</td><td class="patent-data-table-td ">Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Analytical apparatus in sample flow with substrates and apertures for flow detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5494798">US5494798</a></td><td class="patent-data-table-td patent-date-value">Dec 9, 1993</td><td class="patent-data-table-td patent-date-value">Feb 27, 1996</td><td class="patent-data-table-td ">Gerdt; David W.</td><td class="patent-data-table-td ">Coupler having waist joint region coated with immunoreactant capable of binding to target analyte, means for inserting light into fibers to produce evanescent region, means for measuring light magnitude emitted, comparing from two fibers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5494810">US5494810</a></td><td class="patent-data-table-td patent-date-value">Nov 22, 1994</td><td class="patent-data-table-td patent-date-value">Feb 27, 1996</td><td class="patent-data-table-td ">Cornell Research Foundation, Inc.</td><td class="patent-data-table-td ">Genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5496997">US5496997</a></td><td class="patent-data-table-td patent-date-value">Jan 3, 1994</td><td class="patent-data-table-td patent-date-value">Mar 5, 1996</td><td class="patent-data-table-td ">Pope; Edward J. A.</td><td class="patent-data-table-td ">Sensor incorporating an optical fiber and a solid porous inorganic microsphere</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5498392">US5498392</a></td><td class="patent-data-table-td patent-date-value">Sep 19, 1994</td><td class="patent-data-table-td patent-date-value">Mar 12, 1996</td><td class="patent-data-table-td ">Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Mesoscale polynucleotide amplification device and method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5506141">US5506141</a></td><td class="patent-data-table-td patent-date-value">May 9, 1994</td><td class="patent-data-table-td patent-date-value">Apr 9, 1996</td><td class="patent-data-table-td ">Bar-Ilan University</td><td class="patent-data-table-td ">Apertured cell carrier</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5512490">US5512490</a></td><td class="patent-data-table-td patent-date-value">Aug 11, 1994</td><td class="patent-data-table-td patent-date-value">Apr 30, 1996</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">Support, array of heterogeneous semiselective thin films which function as sensing receptor units</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5516635">US5516635</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 15, 1992</td><td class="patent-data-table-td patent-date-value">May 14, 1996</td><td class="patent-data-table-td ">Ekins; Roger P.</td><td class="patent-data-table-td ">Binding assay employing labelled reagent</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5518863">US5518863</a></td><td class="patent-data-table-td patent-date-value">Jul 5, 1994</td><td class="patent-data-table-td patent-date-value">May 21, 1996</td><td class="patent-data-table-td ">Institut National D&#39;optique</td><td class="patent-data-table-td ">Microlenses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5518883">US5518883</a></td><td class="patent-data-table-td patent-date-value">Jun 15, 1993</td><td class="patent-data-table-td patent-date-value">May 21, 1996</td><td class="patent-data-table-td ">Soini; Erkki J.</td><td class="patent-data-table-td ">Biospecific multiparameter assay method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5541311">US5541311</a></td><td class="patent-data-table-td patent-date-value">Jun 4, 1993</td><td class="patent-data-table-td patent-date-value">Jul 30, 1996</td><td class="patent-data-table-td ">Third Wave Technologies, Inc.</td><td class="patent-data-table-td ">Nucleic acid encoding synthesis-deficient thermostable DNA polymerase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5545531">US5545531</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Aug 13, 1996</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Methods for making a device for concurrently processing multiple biological chip assays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5554516">US5554516</a></td><td class="patent-data-table-td patent-date-value">Dec 2, 1993</td><td class="patent-data-table-td patent-date-value">Sep 10, 1996</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Constant temperature</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5565324">US5565324</a></td><td class="patent-data-table-td patent-date-value">Apr 13, 1994</td><td class="patent-data-table-td patent-date-value">Oct 15, 1996</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Complex combinatorial chemical libraries encoded with tags</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5573909">US5573909</a></td><td class="patent-data-table-td patent-date-value">May 20, 1994</td><td class="patent-data-table-td patent-date-value">Nov 12, 1996</td><td class="patent-data-table-td ">Molecular Probes, Inc.</td><td class="patent-data-table-td ">Fluorescent labeling using microparticles with controllable stokes shift</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5575849">US5575849</a></td><td class="patent-data-table-td patent-date-value">Apr 21, 1994</td><td class="patent-data-table-td patent-date-value">Nov 19, 1996</td><td class="patent-data-table-td ">Canon Kabushiki Kaisha</td><td class="patent-data-table-td ">Electrography</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5585069">US5585069</a></td><td class="patent-data-table-td patent-date-value">Nov 10, 1994</td><td class="patent-data-table-td patent-date-value">Dec 17, 1996</td><td class="patent-data-table-td ">David Sarnoff Research Center, Inc.</td><td class="patent-data-table-td ">Substrates with wells connected by channels controlled by valves, for concurrent analysis of several liquid samples</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5587128">US5587128</a></td><td class="patent-data-table-td patent-date-value">Nov 14, 1994</td><td class="patent-data-table-td patent-date-value">Dec 24, 1996</td><td class="patent-data-table-td ">The Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Polymerase chain reaction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5589351">US5589351</a></td><td class="patent-data-table-td patent-date-value">Dec 6, 1994</td><td class="patent-data-table-td patent-date-value">Dec 31, 1996</td><td class="patent-data-table-td ">Nps Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Detects light transmitted from wells in a sample plate</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5593838">US5593838</a></td><td class="patent-data-table-td patent-date-value">May 31, 1995</td><td class="patent-data-table-td patent-date-value">Jan 14, 1997</td><td class="patent-data-table-td ">David Sarnoff Research Center, Inc.</td><td class="patent-data-table-td ">Analyzing nucleic acids in samples, positioning and isolation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5595915">US5595915</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 1994</td><td class="patent-data-table-td patent-date-value">Jan 21, 1997</td><td class="patent-data-table-td ">Chiron Mimotopes Pty. Ltd</td><td class="patent-data-table-td ">Method of determining antigenically active amino acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5603351">US5603351</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Feb 18, 1997</td><td class="patent-data-table-td ">David Sarnoff Research Center, Inc.</td><td class="patent-data-table-td ">Method and system for inhibiting cross-contamination in fluids of combinatorial chemistry device</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5604097">US5604097</a></td><td class="patent-data-table-td patent-date-value">Dec 19, 1994</td><td class="patent-data-table-td patent-date-value">Feb 18, 1997</td><td class="patent-data-table-td ">Spectragen, Inc.</td><td class="patent-data-table-td ">Methods for sorting polynucleotides using oligonucleotide tags</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5610287">US5610287</a></td><td class="patent-data-table-td patent-date-value">Nov 16, 1994</td><td class="patent-data-table-td patent-date-value">Mar 11, 1997</td><td class="patent-data-table-td ">Molecular Tool, Inc.</td><td class="patent-data-table-td ">Method for immobilizing nucleic acid molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5631170">US5631170</a></td><td class="patent-data-table-td patent-date-value">Jun 9, 1993</td><td class="patent-data-table-td patent-date-value">May 20, 1997</td><td class="patent-data-table-td ">Applied Research Systems Ars Holding N.V.</td><td class="patent-data-table-td ">Method for improving measurement precision in evanescent wave optical biosensor assays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5631337">US5631337</a></td><td class="patent-data-table-td patent-date-value">Jan 19, 1996</td><td class="patent-data-table-td patent-date-value">May 20, 1997</td><td class="patent-data-table-td ">Soane Bioscience</td><td class="patent-data-table-td ">Thermoreversible hydrogels comprising linear copolymers and their use in electrophoresis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5632876">US5632876</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 1995</td><td class="patent-data-table-td patent-date-value">May 27, 1997</td><td class="patent-data-table-td ">David Sarnoff Research Center, Inc.</td><td class="patent-data-table-td ">Apparatus and methods for controlling fluid flow in microchannels</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5632957">US5632957</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 1994</td><td class="patent-data-table-td patent-date-value">May 27, 1997</td><td class="patent-data-table-td ">Nanogen</td><td class="patent-data-table-td ">Molecular biological diagnostic systems including electrodes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5633972">US5633972</a></td><td class="patent-data-table-td patent-date-value">Nov 29, 1995</td><td class="patent-data-table-td patent-date-value">May 27, 1997</td><td class="patent-data-table-td ">Trustees Of Tufts College</td><td class="patent-data-table-td ">Superresolution imaging fiber for subwavelength light energy generation and near-field optical microscopy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5635358">US5635358</a></td><td class="patent-data-table-td patent-date-value">Feb 14, 1994</td><td class="patent-data-table-td patent-date-value">Jun 3, 1997</td><td class="patent-data-table-td ">Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Inducing flow of cells; cell lysis component; cell capture region with reversible binding sites to permit isolation of specific cell population</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5637469">US5637469</a></td><td class="patent-data-table-td patent-date-value">Nov 30, 1994</td><td class="patent-data-table-td patent-date-value">Jun 10, 1997</td><td class="patent-data-table-td ">Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Methods and apparatus for the detection of an analyte utilizing mesoscale flow systems</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5639603">US5639603</a></td><td class="patent-data-table-td patent-date-value">Nov 2, 1993</td><td class="patent-data-table-td patent-date-value">Jun 17, 1997</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Synthesizing and screening molecular diversity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5640234">US5640234</a></td><td class="patent-data-table-td patent-date-value">May 31, 1994</td><td class="patent-data-table-td patent-date-value">Jun 17, 1997</td><td class="patent-data-table-td ">Hoechst Aktiengesellschaft</td><td class="patent-data-table-td ">Optical sensor for detection of chemical species</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5643738">US5643738</a></td><td class="patent-data-table-td patent-date-value">May 31, 1995</td><td class="patent-data-table-td patent-date-value">Jul 1, 1997</td><td class="patent-data-table-td ">David Sarnoff Research Center, Inc.</td><td class="patent-data-table-td ">Method of synthesis of plurality of compounds in parallel using a partitioned solid support</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5647030">US5647030</a></td><td class="patent-data-table-td patent-date-value">Jan 11, 1996</td><td class="patent-data-table-td patent-date-value">Jul 8, 1997</td><td class="patent-data-table-td ">University Of Washington</td><td class="patent-data-table-td ">Fiber optic sensor and methods and apparatus relating thereto</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5649576">US5649576</a></td><td class="patent-data-table-td patent-date-value">Feb 26, 1996</td><td class="patent-data-table-td patent-date-value">Jul 22, 1997</td><td class="patent-data-table-td ">Pharmacopeia, Inc.</td><td class="patent-data-table-td ">Partitioning device</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5656241">US5656241</a></td><td class="patent-data-table-td patent-date-value">Sep 7, 1995</td><td class="patent-data-table-td patent-date-value">Aug 12, 1997</td><td class="patent-data-table-td ">Optical Sensors Incorporated</td><td class="patent-data-table-td ">Detector fixed to elongated optical fiber by acrylic adhesive; overcoating with indicator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5660988">US5660988</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Aug 26, 1997</td><td class="patent-data-table-td ">Id Biomedical Corporation</td><td class="patent-data-table-td ">Cycling probe cleavage detection of nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5671303">US5671303</a></td><td class="patent-data-table-td patent-date-value">Apr 17, 1996</td><td class="patent-data-table-td patent-date-value">Sep 23, 1997</td><td class="patent-data-table-td ">Motorola, Inc.</td><td class="patent-data-table-td ">Molecular detection apparatus and method using optical waveguide detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5856083">US5856083</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 3, 1995</td><td class="patent-data-table-td patent-date-value">Jan 5, 1999</td><td class="patent-data-table-td ">Pharmacopeia, Inc.</td><td class="patent-data-table-td ">Monitoring a photometric change</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5863722">US5863722</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Jan 26, 1999</td><td class="patent-data-table-td ">Lynx Therapeutics, Inc.</td><td class="patent-data-table-td ">Method of sorting polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6197515">US6197515</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 21, 1999</td><td class="patent-data-table-td patent-date-value">Mar 6, 2001</td><td class="patent-data-table-td ">Harvard University</td><td class="patent-data-table-td ">Molecular recognition at surfaces derivatized with self-assembled monolayers</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Abel et al., "<a href='http://scholar.google.com/scholar?q="Fiber-Optic+Evanescent+Wave+Biosensor+for+the+Detection+of+Oligonucleotides%2C"'>Fiber-Optic Evanescent Wave Biosensor for the Detection of Oligonucleotides,</a>" Anal. Chem. 68:2905-2912 (1996).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Angel, S., "<a href='http://scholar.google.com/scholar?q="Optrodes%3A+Chemically+Selective+Fiber-Optic+Sensors%2C"'>Optrodes: Chemically Selective Fiber-Optic Sensors,</a>" Spectroscopy, 2(4):38-47 (1987).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Anonymous, "<a href='http://scholar.google.com/scholar?q="Fluorescent+Microspheres%2C"'>Fluorescent Microspheres,</a>" Tech. Note 19, Bang Laboratories, (Fishers, In) Feb. 1997.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Anonymous, "<a href='http://scholar.google.com/scholar?q="Microsphere+Selection+Guide%2C"'>Microsphere Selection Guide,</a>" Bandg Laboratories, (Fisher, In) Sep. 1998.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baner et al., "<a href='http://scholar.google.com/scholar?q="Signal+amplification+of+padlock+probes+by+rolling+circle+replication%2C"'>Signal amplification of padlock probes by rolling circle replication,</a>" Nucleic Acids Research, 26(22): 5073-5078 (1998).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bangs, L.B., "<a href='http://scholar.google.com/scholar?q="Immunological+Applications+of+Microspheres%2C"'>Immunological Applications of Microspheres,</a>" The Latex Course, Bangs Laboratories (Carmel, IN) Apr. 1996.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Barnard et al., "<a href='http://scholar.google.com/scholar?q="A+Fibre-Optic+Chemical+Sensor+with+Discrete+Sensing+Sites%2C"'>A Fibre-Optic Chemical Sensor with Discrete Sensing Sites,</a>" Nature, 353:338-340 (Sep. 1991).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Barnard et al., "<a href='http://scholar.google.com/scholar?q="Fiber-Optic+Organic+Vapor+Sensor%2C"'>Fiber-Optic Organic Vapor Sensor,</a>" Environ. Sci. Technol., 25(7): 1301-1304 (1991).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ben-Dor, A., et al., "<a href='http://scholar.google.com/scholar?q="Universal+DNA+Tag+Systems%3A+A+Combinatorial+Design+Scheme."'>Universal DNA Tag Systems: A Combinatorial Design Scheme.</a>" J. Compu. Biol. 7(3/4): 503-519 (2000).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Birindelli, S. et al., "<a href='http://scholar.google.com/scholar?q="Comments+on+Adaptation+of+the+Cellscan+Technique+for+the+SCM+Test+in+Breast+Cancer."'>Comments on Adaptation of the Cellscan Technique for the SCM Test in Breast Cancer.</a>" European J. Cancer 33(8):1333-1334 (1997).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Brenner, S., et al. "<a href='http://scholar.google.com/scholar?q="Gene+Expression+Analysis+by+Massively+Parallel+Signature+Sequencing+%28MPSS%29+on+Microbead+Arrays."'>Gene Expression Analysis by Massively Parallel Signature Sequencing (MPSS) on Microbead Arrays.</a>" Nature Biotechnology 18: 630-634 (2000).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bruchez et al., "<a href='http://scholar.google.com/scholar?q="Semiconductor+Nanocrystals+as+Fluorescent+Biological+Labels%2C"'>Semiconductor Nanocrystals as Fluorescent Biological Labels,</a>" Science, 281: p. 2013-; Sep. 25, 1998.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Carey, W.P. et al., "<a href='http://scholar.google.com/scholar?q="Chemical+Piezoelectric+Sensor+and+Sensor+Array+Characterization%2C"'>Chemical Piezoelectric Sensor and Sensor Array Characterization,</a>" Anal. Chem. 58(14):3077-3084 (1986).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Castano, J. et al., "<a href='http://scholar.google.com/scholar?q="Dynamic+Monitoring+and+Quantification+of+Gene+Expression+in+Single+Living+Cells%3A+A+Molecular+Basis+for+Secretory+Cell+Heterogeneity%2C"'>Dynamic Monitoring and Quantification of Gene Expression in Single Living Cells: A Molecular Basis for Secretory Cell Heterogeneity,</a>" Mol. Endo. 10(5):599-605 (1996).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chen et al., "<a href='http://scholar.google.com/scholar?q="A+Microsphere-Based+Assay+for+Multiplexed+Single+Nucleotide+Polymorphism+Analysis+Using+Single+Base+Chain+Extension%2C"'>A Microsphere-Based Assay for Multiplexed Single Nucleotide Polymorphism Analysis Using Single Base Chain Extension,</a>" Genome Research, 10(4):549-557 (2000).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chiavaroli, C. et al., "<a href='http://scholar.google.com/scholar?q="Simultaneous+Monitorinig+of+Cytosolic+Free+Calcium+and+Exocytosis+at+the+Single+Cell+Leve%2C"'>Simultaneous Monitorinig of Cytosolic Free Calcium and Exocytosis at the Single Cell Leve,</a>" J. Neuroendocrinology 3(3):253-260 (1991).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Clark, R. et al., "<a href='http://scholar.google.com/scholar?q="Electrochemical+Analysis+in+Picoliter+Microvials%2C"'>Electrochemical Analysis in Picoliter Microvials,</a>" Anal. Chem. 69:259-263 (1997).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Czarnik. A "<a href='http://scholar.google.com/scholar?q="Illuminating+the+SNP+Genomic+Code%2C"'>Illuminating the SNP Genomic Code,</a>" Modern Drug Discovery, 49-55 (Nov./Dec. 1998).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Danielson et al., "<a href='http://scholar.google.com/scholar?q="A+Combinatorial+Approach+to+the+Discovery+and+Optimization+of+Luminescent+Materials%2C"'>A Combinatorial Approach to the Discovery and Optimization of Luminescent Materials,</a>" Nature, 389:944-948 (1997).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">DeForest, W.S., "<a href='http://scholar.google.com/scholar?q="Photoresist+Materials+and+Processes%2C"'>Photoresist Materials and Processes,</a>" McGraw-Hill Book Company, New York. 89101 (1975).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Deutsch, M. et al., "<a href='http://scholar.google.com/scholar?q="Apparatus+for+high+Precision+Repetitive+Sequential+Optical+Measurement+of+Living+Dells%2C"'>Apparatus for high Precision Repetitive Sequential Optical Measurement of Living Dells,</a>" Cytometry 16:214-226 (1994).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Deutsch, M. et al., "<a href='http://scholar.google.com/scholar?q="Lymphocyte+Fluorescence+Polarization+Measurements+with+the+Cellscan+System%3A+Application+to+the+SCM+Cancer+Test%2C"'>Lymphocyte Fluorescence Polarization Measurements with the Cellscan System: Application to the SCM Cancer Test,</a>" Cytometry 23:159-165 (1996).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dickinson et al., "<a href='http://scholar.google.com/scholar?q="Generating+Sensor+Diversity+Through+Combinatorial+Polymer+Synthesis%2C"'>Generating Sensor Diversity Through Combinatorial Polymer Synthesis,</a>" Analytical Chemistry, 69(17): 3413-3418 (1997).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">DiMarco, G. et al., "<a href='http://scholar.google.com/scholar?q="Luminescent+Ru%26lt%3BSUP%26gt%3B11%26lt%3B%2FSUP%26gt%3B-Polypyridine+Complexes+in+Poly-2-Hydroxyethylmetharcrylate+Matrices+as+Oxygen+Sensors%2C"'>Luminescent Ru&lt;SUP&gt;11&lt;/SUP&gt;-Polypyridine Complexes in Poly-2-Hydroxyethylmetharcrylate Matrices as Oxygen Sensors,</a>" Adv. Mater., 7(5):(1995).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Drmanac, R. et al., "<a href='http://scholar.google.com/scholar?q="Prospects+for+a+Miniaturized%2C+Simplified+and+Frugal+Human+Genome+Project%2C"'>Prospects for a Miniaturized, Simplified and Frugal Human Genome Project,</a>" Scientia Yugoslavica, 16(1-2):97-107 (1990).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Drmanac, R. et al., "<a href='http://scholar.google.com/scholar?q="Sequencing+by+Hybridization+%28SBH%29+with+Oligonucleotide+Probes+as+an+Integral+Approach+for+the+Analysis+of+Complex+Genomes%2C"'>Sequencing by Hybridization (SBH) with Oligonucleotide Probes as an Integral Approach for the Analysis of Complex Genomes,</a>" International Journal of Genome Research, 1(1):59-79 (1992).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Drmanac, R. et al., "<a href='http://scholar.google.com/scholar?q="Sequencing+by+Hybridization%2C"'>Sequencing by Hybridization,</a>" Automated DNA Sequencing and Analysis, ed. M. Adams, C. Fields and J. Venter. (1994).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Drmanac, R. et al., "<a href='http://scholar.google.com/scholar?q="Sequencing+by+Oligonucleotide+Hybridization%3A+A+Promising+Framework+in+Decoding+of+the+Genome+Program%2C"'>Sequencing by Oligonucleotide Hybridization: A Promising Framework in Decoding of the Genome Program,</a>" The First International Conference on Electrophoresis, Supercomputing and the Human Genome, Proceeding os th Apr. 10-13, 1990 Conference at Florida State University. Ed. C. Cantor and H. Lim.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Egner, B.J., et al., "<a href='http://scholar.google.com/scholar?q="Tagging+in+Combinatorial+Chemistry%3A+The+Use+of+Coloured+and+Fluorescent+Beads."'>Tagging in Combinatorial Chemistry: The Use of Coloured and Fluorescent Beads.</a>" Chem. Commun. (1997).</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fan, J.B., et al., "<a href='http://scholar.google.com/scholar?q="Parallel+Genotyping+of+Human+SNPs+Using+Generic+High-density+Oligonucleotide+Tag+Arrays."'>Parallel Genotyping of Human SNPs Using Generic High-density Oligonucleotide Tag Arrays.</a>" Genome Research, 10:853-860 (2000).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fathi, et al., "<a href='http://scholar.google.com/scholar?q="Synthesis+and+Properties+of+Combinatorial+Libraries+of+Phosphoramidates%2C"'>Synthesis and Properties of Combinatorial Libraries of Phosphoramidates,</a>" J. Org. Chem. Soc. 61(16): 5600-5609 (1996).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ferguson et al., "<a href='http://scholar.google.com/scholar?q="A+Fiber-Optic+DNA+Biosensor+Microarray+for+the+Analysis+of+Gene+Expression%2C"'>A Fiber-Optic DNA Biosensor Microarray for the Analysis of Gene Expression,</a>" Nature Biotechnology, 14:1681-1684 (1996).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fodor, S. et al., "<a href='http://scholar.google.com/scholar?q="Light-Directed%2C+Spatially+Addressable+Parallel+Chemical+Synthesis%2C"'>Light-Directed, Spatially Addressable Parallel Chemical Synthesis,</a>" Science, 251:767-773 (1991).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Freeman, T. et al., "<a href='http://scholar.google.com/scholar?q="Oxygen+Probe+Based+on+Tetrakis%28alkylamino%29ethylene+Chemiluminescence%2C"'>Oxygen Probe Based on Tetrakis(alkylamino)ethylene Chemiluminescence,</a>" Anal. Chem., 53:98102 (1981).</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fuh et al., "<a href='http://scholar.google.com/scholar?q="Single+Fibre+Optic+Fluorescence+pH+Probe%2C"'>Single Fibre Optic Fluorescence pH Probe,</a>" Analyst, 112:1159-1163 (1987).</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gauci, M. et al., "<a href='http://scholar.google.com/scholar?q="Observation+of+Single-Cell+Fluorescence+Spectra+in+Laser+Flow+Cytometry%2C"'>Observation of Single-Cell Fluorescence Spectra in Laser Flow Cytometry,</a>" Cytometry 25:388-393 (1996).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gerry, N.P., et al., "<a href='http://scholar.google.com/scholar?q="Universal+DNA+Microarray+Method+for+Multiplex+Detection+of+Low+Abundance+Point+Mutations."'>Universal DNA Microarray Method for Multiplex Detection of Low Abundance Point Mutations.</a>" J. Mol. Biol. 292:251-262 (1999).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grime, G.W., Non-Silver Photographic Processes, Proceedings of the Symposium on Non-Silver Photographic Processes Held at New College, Oxford, Sep., 1973. "<a href='http://scholar.google.com/scholar?q="Holographic+Diffraction+Gratings+Recorded+in+Photoresist%2C"'>Holographic Diffraction Gratings Recorded in Photoresist,</a>" p. 275-284 (1973).</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gunderson, K.L., "<a href='http://scholar.google.com/scholar?q="Mutation+Detection+by+Ligation+to+Complete+n-mer+DNA+Arrays."'>Mutation Detection by Ligation to Complete n-mer DNA Arrays.</a>" Genome Research 8:1142-1153 (1998).</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hafeman, D. et al., "<a href='http://scholar.google.com/scholar?q="Light-Addressable+Potentiometric+sensor+for+biochemical+systems%2C"'>Light-Addressable Potentiometric sensor for biochemical systems,</a>" Science, 240:1182-1185 (1988).</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Healey et al., "<a href='http://scholar.google.com/scholar?q="Development+of+a+Penicillin+Biosensor+Using+a+Single+Optical+Imaging+Fiber%2C"'>Development of a Penicillin Biosensor Using a Single Optical Imaging Fiber,</a>" SPIE Proc. 2388:568-573 (1995).</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Healey et al., "<a href='http://scholar.google.com/scholar?q="Fiberoptic+DNA+Sensor+Array+Capable+of+Detecting+Point+Mutations%2C"'>Fiberoptic DNA Sensor Array Capable of Detecting Point Mutations,</a>" Analytical Biochemistry, 251:270-279 (1997).</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Healey et al., "<a href='http://scholar.google.com/scholar?q="Improved+Fiber-Optic+Chemical+Sensor+for+Penicillin%2C"'>Improved Fiber-Optic Chemical Sensor for Penicillin,</a>" Anal. Chem. 67(24):4471-4476 (1995).</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hirschfeld et al., "<a href='http://scholar.google.com/scholar?q="Laser-Fiber-Optic+%22Optrode%22+for+Real+Time+In+Vivo+Blood+Carbon+Level+Monitoring%2C"'>Laser-Fiber-Optic "Optrode" for Real Time In Vivo Blood Carbon Level Monitoring,</a>" Journal of Lightwave Technology, LT-5(7):1027-1033 (1987).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hirschhorn, J.N., et al., "<a href='http://scholar.google.com/scholar?q="SBE-TAGS%3A+An+Array-Based+Method+for+Efficient+Single-Nucleotide+Polymorphism+Genotyping."'>SBE-TAGS: An Array-Based Method for Efficient Single-Nucleotide Polymorphism Genotyping.</a>" PNAS 97(22): 12164-12169 (2000).</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hogan, B. et al., "<a href='http://scholar.google.com/scholar?q="Single-cell+Analysis+at+the+Level+of+a+Single+Human+Erythrocyte%2C"'>Single-cell Analysis at the Level of a Single Human Erythrocyte,</a>" Trends in Analytical Chemistry 12(1):4-9 (1993).</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hsuih et al., "<a href='http://scholar.google.com/scholar?q="Novel%2C+ligation-dependaent+PCR+assay+for+detection+of+Hepatitis+C+Virus+in+Serum%2C"'>Novel, ligation-dependaent PCR assay for detection of Hepatitis C Virus in Serum,</a>" J. of Clinical Microbiology, 34(3): 501-507 (1996).</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hsuih, T. et al., "<a href='http://scholar.google.com/scholar?q="Novel%2C+Litigation-Dependent+PCR+Assay+for+Detection+of+Hepatitis+C+Virus+in+Serum%2C"'>Novel, Litigation-Dependent PCR Assay for Detection of Hepatitis C Virus in Serum,</a>" Journal of Clinical Microbiology, Mar. 1996, 34(3): 501-507 (1996).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Huang, L. et al., "<a href='http://scholar.google.com/scholar?q="Exploring+Single-cell+Dynamics+Using+Chemically-modified+Microelectrodes%2C"'>Exploring Single-cell Dynamics Using Chemically-modified Microelectrodes,</a>" Trends in Analytical Chemistry 14(4):158-164 (1995).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hubert, C. et al., "<a href='http://scholar.google.com/scholar?q="Design+of+a+Solvatochromic+Polymer-Based+Fiber+Optics+Chemical+Sensor+for+Polar+Solvent+Detection%2C"'>Design of a Solvatochromic Polymer-Based Fiber Optics Chemical Sensor for Polar Solvent Detection,</a>" Adv. Mater., 7(11): 914-917 (1995).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Iannone et al., "<a href='http://scholar.google.com/scholar?q="Multiplexed+Single+Nucleotide+Polymorphism+Genotyping+by+Oligonucleotide+Ligation+and+Flow+Cytometry%2C"'>Multiplexed Single Nucleotide Polymorphism Genotyping by Oligonucleotide Ligation and Flow Cytometry,</a>" Cytometry, 39:131-140 (2000).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Illumina Inc. "<a href='http://scholar.google.com/scholar?q="Emerging%2C"'>Emerging,</a>" Windhover's In Vivo, The Business and Medicine Report, p. 1-2 (Sep. 1998).</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ince, C. et al., "<a href='http://scholar.google.com/scholar?q="A+Micro-Perfusion+Chamber+for+Single-Cell+Fluorescence+Measurements%2C"'>A Micro-Perfusion Chamber for Single-Cell Fluorescence Measurements,</a>" Journal of Immunological Methods 128:227-234 (1990).</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jacobs, J. et al., "<a href='http://scholar.google.com/scholar?q="Combinatorial+chemistry-applications+of+light-directed+chemical+synthesis%2C"'>Combinatorial chemistry-applications of light-directed chemical synthesis,</a>" TIBTECH, 12:19-26 (1994).</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Koop, A. et al., "<a href='http://scholar.google.com/scholar?q="Continuous+Bioluminescent+Monitoring+of+Cytoplasmic+ATP+in+Single+Isolated+Rat+Hepatocytes+During+Metaboli+Poisoning%2C"'>Continuous Bioluminescent Monitoring of Cytoplasmic ATP in Single Isolated Rat Hepatocytes During Metaboli Poisoning,</a>" Biochem. J. 295:165-170 (1993).</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lam, K.S., "<a href='http://scholar.google.com/scholar?q="The+%27One-Bead-One-Compound%27+Combinatorial+Library+Method."'>The 'One-Bead-One-Compound' Combinatorial Library Method.</a>" Chem. Rev. 97(2): 411-448 (1997).</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Levy, U. et al., "<a href='http://scholar.google.com/scholar?q="Direct+Picture+Transmission+in+a+Single+Fiber+with+Holographic+Filters%2C"'>Direct Picture Transmission in a Single Fiber with Holographic Filters,</a>" Optics Communications, 30(2):163165 (1979).</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lin et al., "<a href='http://scholar.google.com/scholar?q="A+porous+silicon-based+optical+interferometric+biosensor%2C"'>A porous silicon-based optical interferometric biosensor,</a>" Science, 278:840-843 (1997).</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lin, Z. et al., "<a href='http://scholar.google.com/scholar?q="Multiplex+genotype+determination+at+a+large+number+of+gene+loci%2C"'>Multiplex genotype determination at a large number of gene loci,</a>" Proc. Natl. Acad. Sci. USA, 93: 2582-2587, Mar. 1996.</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lippitsch, M. et al., "<a href='http://scholar.google.com/scholar?q="Fibre-Optic+Oxygen+Sensor+with+the+Fluorescence+Decay+Time+as+the+Information+Carrier%2C"'>Fibre-Optic Oxygen Sensor with the Fluorescence Decay Time as the Information Carrier,</a>" Anal. Chem. Acta., 205:1-6 (1998).</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lizardi et al., "<a href='http://scholar.google.com/scholar?q="Mutation+detection+and+single-molecule+counting+using+isothermal+rolling-circle+amplification%2C"'>Mutation detection and single-molecule counting using isothermal rolling-circle amplification,</a>" Nature Genetics, 19: 225-232 (Jul. 1998).</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lonergan, M.C., "<a href='http://scholar.google.com/scholar?q="Array-Based+Vapor+Sensing+Using+Chemically+Sensitive%2C+Polymer+Composite+Resistors%2C"'>Array-Based Vapor Sensing Using Chemically Sensitive, Polymer Composite Resistors,</a>" IEEE Aerospace Applications Conference Proceedings, 3(8):583-631 (1997).</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lubbers, D. et al., "<a href='http://scholar.google.com/scholar?q="Optical+Fluorescense+Sensors+for+Continuous+Measurement+of+Chemical+Concentrations+in+Biological+Systems%2C"'>Optical Fluorescense Sensors for Continuous Measurement of Chemical Concentrations in Biological Systems,</a>" Sens. Actuators, 4:641-654 (1983).</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lundstrom, I. et al., "<a href='http://scholar.google.com/scholar?q="Artificial+%27olfactory%27+images+from+a+chemical+sensor+using+a+light-impulse+technique%2C"'>Artificial 'olfactory' images from a chemical sensor using a light-impulse technique,</a>" Nature, 352:47-50 (1991).</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Luong, J. et al., "<a href='http://scholar.google.com/scholar?q="Fluorescence+Sensors+for+Monitoring+Bioprocesses%2C"'>Fluorescence Sensors for Monitoring Bioprocesses,</a>" Practical Fluoro, ed. G.G. Guilbault, 2nd Ed., Marcel Dekker. 775-793 (New York 1990).</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lyamichev et al., "<a href='http://scholar.google.com/scholar?q="Polymorphism+identification+and+quantitative+detection+of+genomic+DNA+by+invasive+cleavage+of+oligonucleotide+probes%2C"'>Polymorphism identification and quantitative detection of genomic DNA by invasive cleavage of oligonucleotide probes,</a>" Nature Biotechnology, 17:292-296 (1999).</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Matthews et al., "<a href='http://scholar.google.com/scholar?q="Analytical+Strategies+for+the+Use+of+Dna+Probes%2C"'>Analytical Strategies for the Use of Dna Probes,</a>" Analytical Biochemistry, 169:1-25 (1988).</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McConnell, et al., "<a href='http://scholar.google.com/scholar?q="The+Cytosensor+Microphysiometer%3A+Biological+Applications+of+Silicon+Technology%2C"'>The Cytosensor Microphysiometer: Biological Applications of Silicon Technology,</a>" Science 257:1906-1912 (Sep. 1992).</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Metzker, M. et al., "<a href='http://scholar.google.com/scholar?q="Termination+of+DNA+synthesis+by+novel+3%27-modified-deoxyribonucleoside+5%27-triphosphates%2C"'>Termination of DNA synthesis by novel 3'-modified-deoxyribonucleoside 5'-triphosphates,</a>" Nucleic Acids Research 22(20):4259-4267 (1994).</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Michael et al., "<a href='http://scholar.google.com/scholar?q="Fabrication+of+Micro-+and+Nanostructures+Using+Optical+Fibers+and+there+Use+as+Chemical+Sensors%2C"'>Fabrication of Micro- and Nanostructures Using Optical Fibers and there Use as Chemical Sensors,</a>" Proc. 3rd Intl. Symp., Microstructures and Microfabricated Systems, ed. P.J. Hesketh, et al., v. 97-5, Electrochem. Soc., 152-157 (Aug. 1997).</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Michael et al., "<a href='http://scholar.google.com/scholar?q="Making+Sensors+out+of+Disarray%3A+Optical+Sensor+Microarrays%2C"'>Making Sensors out of Disarray: Optical Sensor Microarrays,</a>" Proc. SPIE, 3270 34-41 (1998).</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Michael et al., "<a href='http://scholar.google.com/scholar?q="Randomly+Ordered+Addressable+High-Density+Optical+Sensor+Arrays%2C"'>Randomly Ordered Addressable High-Density Optical Sensor Arrays,</a>" Anal. Chem. 70(7): 1242-1248 (Apr. 1998).</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mignani, et al, "<a href='http://scholar.google.com/scholar?q="In-Vivo+Biomedical+Monitoring+by+Fiber-Optic+Systems%2C"'>In-Vivo Biomedical Monitoring by Fiber-Optic Systems,</a>" Journal of Lightwave Technology, 13(7): 1396-1406 (1995).</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Milanovich et al., "<a href='http://scholar.google.com/scholar?q="Clinical+Measurements+Using+Fiber+Optics+and+Optrodes%2C"'>Clinical Measurements Using Fiber Optics and Optrodes,</a>" Novel Optical Fiber Techniques for Medical Application, SPIE, 494:1831 (1984).</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Miyawaki, A. et al., "<a href='http://scholar.google.com/scholar?q="Fluorescent+Indicators+for+Ca%26lt%3BSUP%26gt%3B2%2B%26lt%3B%2FSUP%26gt%3B+Based+on+Green+Fluorescent+Proteins+and+Calmodulin%2C"'>Fluorescent Indicators for Ca&lt;SUP&gt;2+&lt;/SUP&gt; Based on Green Fluorescent Proteins and Calmodulin,</a>" Nature 388:882-887 (Aug. 1997).</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mrksich, M. et al., "<a href='http://scholar.google.com/scholar?q="Controlling+Cell+Attachment+of+Contoured+Surfaces+with+Self-Assembled+Monolayers+of+Alkanethiolates+on+Gold%2C"'>Controlling Cell Attachment of Contoured Surfaces with Self-Assembled Monolayers of Alkanethiolates on Gold,</a>" Proc. Natl. Acad. Sci. USA 93:10775-10778 (Oct. 1996).</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Munkholm, C. et al., "<a href='http://scholar.google.com/scholar?q="A+Fiber-Optic+Sensor+for+CO%26lt%3BSUB%26gt%3B2+%26lt%3B%2FSUB%26gt%3BMeasurement%2C"'>A Fiber-Optic Sensor for CO&lt;SUB&gt;2 &lt;/SUB&gt;Measurement,</a>" Talanta, 35(2):109112 (1988).</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Munkholm, C. et al., "<a href='http://scholar.google.com/scholar?q="Polymer+Modification+of+Fiber+Optic+Chemical+Sensors+as+a+Method+of+Enhancing+Fluorescense+Signal+for+pH+Measurement%2C"'>Polymer Modification of Fiber Optic Chemical Sensors as a Method of Enhancing Fluorescense Signal for pH Measurement,</a>" Anal. Chem., 58:14271430 (1986).</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Normie, L. "<a href='http://scholar.google.com/scholar?q="System+Uses+Photonics+for+Early+Tumor+Detection%2C"'>System Uses Photonics for Early Tumor Detection,</a>" Biophotonics International 24-25 (Sep./Oct. 1996).</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Owicki, J. et al., "<a href='http://scholar.google.com/scholar?q="Bioassays+with+a+Microphysiometer%2C"'>Bioassays with a Microphysiometer,</a>" Nature 344:271-272 (Mar. 1990).</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Owicki, J. et al., "<a href='http://scholar.google.com/scholar?q="Continuous+Monitoring+of+Receptor-Mediated+Changes+in+the+Metabolic+Rates+of+Living+Cells%2C"'>Continuous Monitoring of Receptor-Mediated Changes in the Metabolic Rates of Living Cells,</a>" Proc. Natl. Acad. Sci. USA 87:407-4011 (1990).</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Owicki, J. et al., "<a href='http://scholar.google.com/scholar?q="The+Light-Addressable+Potentiometric+Sensor%3A+Principles+and+Biological+Applications%2C"'>The Light-Addressable Potentiometric Sensor: Principles and Biological Applications,</a>" Annu. Rev. Biophys, Biomol. Struct. 23:87-113 (1994).</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pantano et al., "<a href='http://scholar.google.com/scholar?q="Ordered+Nanowell+Arrays%2C"'>Ordered Nanowell Arrays,</a>" Chem. Mater., 8(12): 2832-2835 (1996).</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pantano, P. et al., "<a href='http://scholar.google.com/scholar?q="Analytical+Applications+of+Optical+Imaging+Fibers%2C"'>Analytical Applications of Optical Imaging Fibers,</a>" Anal. Chem., 67:461A-487A (1995).</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Parce, J. et al., "<a href='http://scholar.google.com/scholar?q="Biosensors+for+Directly+Measuring+Cell+Affecting+Agents%2C"'>Biosensors for Directly Measuring Cell Affecting Agents,</a>" Ann. Biol. Clin. 48:639-641 (1990).</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Parce, J. et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+Cell-Affecting+Agents+with+a+Silicon+Biosensor%2C"'>Detection of Cell-Affecting Agents with a Silicon Biosensor,</a>" Science 246:243-247 (Oct. 1989).</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Park, M. et al., "<a href='http://scholar.google.com/scholar?q="Block+Copolymer+Lithography%3A+Periodic+Arrays+of+%7E10%26lt%3BSUP%26gt%3B11+%26lt%3B%2FSUP%26gt%3BHoles+in+1+Square+Centimeter%2C"'>Block Copolymer Lithography: Periodic Arrays of ~10&lt;SUP&gt;11 &lt;/SUP&gt;Holes in 1 Square Centimeter,</a>" Science 276:1401-1404 (May 1997).</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Peterson et al., "<a href='http://scholar.google.com/scholar?q="Fiber-Optic+Sensors+for+Biomedical+Applications%2C"'>Fiber-Optic Sensors for Biomedical Applications,</a>" Science, 13:123-127 (1984).</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Peterson, J. et al., "<a href='http://scholar.google.com/scholar?q="Fiber+Optic+pH+Probe+for+Physiological+Use%2C"'>Fiber Optic pH Probe for Physiological Use,</a>" Anal. Chem., 52:864-869 (1980).</td></tr><tr><td class="patent-data-table-td ">90</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Piunno et al., "<a href='http://scholar.google.com/scholar?q="Fiber-Optic+DNA+Sensor+for+Fluorometric+Nucleic+Acid+Determination%2C"'>Fiber-Optic DNA Sensor for Fluorometric Nucleic Acid Determination,</a>" Anal. Chem., 67:2635-2643 (1995).</td></tr><tr><td class="patent-data-table-td ">91</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Plunkett, M. et al., "<a href='http://scholar.google.com/scholar?q="Combinatorial+Chemistry+and+New+Drugs%2C"'>Combinatorial Chemistry and New Drugs,</a>" Scientific American, Apr. 1997, 69-73.</td></tr><tr><td class="patent-data-table-td ">92</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pope, E. "<a href='http://scholar.google.com/scholar?q="Fiber+Optic+Chemical+Microsensors+Employing+Optically+Silica+Microspehres%2C"'>Fiber Optic Chemical Microsensors Employing Optically Silica Microspehres,</a>" SPIE, 2388:245-256 (1995).</td></tr><tr><td class="patent-data-table-td ">93</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rahmani, et al., "<a href='http://scholar.google.com/scholar?q="Adaption+of+the+Cellscan+Technique+for+the+SCM+Test+in+Breast+Cancer%2C"'>Adaption of the Cellscan Technique for the SCM Test in Breast Cancer,</a>" European J. Cancer 32A(10): 1758-1765 (1996).</td></tr><tr><td class="patent-data-table-td ">94</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ronaghi et al., "<a href='http://scholar.google.com/scholar?q="A+Sequencing+Method+Based+on+Real-Time+Pyrophosphate%2C"'>A Sequencing Method Based on Real-Time Pyrophosphate,</a>" Science, 281:363-365 (1998).</td></tr><tr><td class="patent-data-table-td ">95</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shoemaker et al., "<a href='http://scholar.google.com/scholar?q="Quantitative+phenotypic+analysis+of+yeast+deletion+mutants+using+a+highly+parallel+molecular+bar-coding+strategy%2C"'>Quantitative phenotypic analysis of yeast deletion mutants using a highly parallel molecular bar-coding strategy,</a>" Nature Genetics, 14:450-456 (1996).</td></tr><tr><td class="patent-data-table-td ">96</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Strachan et al., "<a href='http://scholar.google.com/scholar?q="A+Rapid+General+Method+for+the+Identification+of+PCR+Products+Using+a+Fibre-Optic+Biosensor+and+its+Application+to+the+Detection+of+Listeria%2C"'>A Rapid General Method for the Identification of PCR Products Using a Fibre-Optic Biosensor and its Application to the Detection of Listeria,</a>" Letters in Applied Microbiology, 21:5-9 (1995).</td></tr><tr><td class="patent-data-table-td ">97</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Vergne et al. Phagosomal pH determination by duel fluorescence flow cytometry. Analytical Biochemistry, vol. 2554(1), pp. 127-132 (Jan. 1, 1998).</td></tr><tr><td class="patent-data-table-td ">98</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Walt, "<a href='http://scholar.google.com/scholar?q="Fiber-Optic+Sensors+for+Continuous+Clinical+Monitoring%2C"'>Fiber-Optic Sensors for Continuous Clinical Monitoring,</a>" Proc. IEEE, 80(6): 903-911 (1992).</td></tr><tr><td class="patent-data-table-td ">99</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Walt, D "<a href='http://scholar.google.com/scholar?q="Fiber+Optic+Imaging+Sensors%2C"'>Fiber Optic Imaging Sensors,</a>" Acc. Chem. Res. 31:267-278 (1998).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7510841">US7510841</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 28, 2004</td><td class="patent-data-table-td patent-date-value">Mar 31, 2009</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Methods of making and using composite arrays for the detection of a plurality of target analytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7697807">US7697807</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 1, 2006</td><td class="patent-data-table-td patent-date-value">Apr 13, 2010</td><td class="patent-data-table-td ">Ut-Battelle, Llc</td><td class="patent-data-table-td ">Multi-tipped optical component</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7697808">US7697808</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 16, 2007</td><td class="patent-data-table-td patent-date-value">Apr 13, 2010</td><td class="patent-data-table-td ">Ut-Battelle, Llc</td><td class="patent-data-table-td ">Multi-tipped optical component</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7813013">US7813013</a></td><td class="patent-data-table-td patent-date-value">Nov 21, 2006</td><td class="patent-data-table-td patent-date-value">Oct 12, 2010</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Hexagonal site line scanning method and system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7858301">US7858301</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 7, 2004</td><td class="patent-data-table-td patent-date-value">Dec 28, 2010</td><td class="patent-data-table-td ">Bioarray Solutions, Ltd.</td><td class="patent-data-table-td ">hybridization-mediated multiplexed analysis of polymorphisms under conditions permitting competitive hybridization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7948015">US7948015</a></td><td class="patent-data-table-td patent-date-value">Dec 14, 2007</td><td class="patent-data-table-td patent-date-value">May 24, 2011</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8023162">US8023162</a></td><td class="patent-data-table-td patent-date-value">Sep 3, 2010</td><td class="patent-data-table-td patent-date-value">Sep 20, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Hexagonal site line scanning method and system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8217433">US8217433</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Jul 10, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8222047">US8222047</a></td><td class="patent-data-table-td patent-date-value">Sep 23, 2008</td><td class="patent-data-table-td patent-date-value">Jul 17, 2012</td><td class="patent-data-table-td ">Quanterix Corporation</td><td class="patent-data-table-td ">Ultra-sensitive detection of molecules on single molecule arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8236574">US8236574</a></td><td class="patent-data-table-td patent-date-value">Mar 24, 2010</td><td class="patent-data-table-td patent-date-value">Aug 7, 2012</td><td class="patent-data-table-td ">Quanterix Corporation</td><td class="patent-data-table-td ">Ultra-sensitive detection of molecules or particles using beads or other capture objects</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8247849">US8247849</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Aug 21, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Two-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8262900">US8262900</a></td><td class="patent-data-table-td patent-date-value">Dec 17, 2007</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8263336">US8263336</a></td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8264014">US8264014</a></td><td class="patent-data-table-td patent-date-value">Feb 2, 2011</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8269261">US8269261</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2011</td><td class="patent-data-table-td patent-date-value">Sep 18, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8293082">US8293082</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Oct 23, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8306757">US8306757</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2010</td><td class="patent-data-table-td patent-date-value">Nov 6, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8313625">US8313625</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Nov 20, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8313639">US8313639</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Nov 20, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8317999">US8317999</a></td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td patent-date-value">Nov 27, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8349167">US8349167</a></td><td class="patent-data-table-td patent-date-value">Jun 26, 2009</td><td class="patent-data-table-td patent-date-value">Jan 8, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415171">US8415171</a></td><td class="patent-data-table-td patent-date-value">Mar 24, 2010</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Quanterix Corporation</td><td class="patent-data-table-td ">Methods and systems for extending dynamic range in assays for the detection of molecules or particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415176">US8415176</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415177">US8415177</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Two-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415716">US8415716</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensors with feedback circuits</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8421437">US8421437</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Apr 16, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Array column integrator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8426898">US8426898</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2010</td><td class="patent-data-table-td patent-date-value">Apr 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8426899">US8426899</a></td><td class="patent-data-table-td patent-date-value">Feb 2, 2011</td><td class="patent-data-table-td patent-date-value">Apr 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8432149">US8432149</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Array column integrator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8432150">US8432150</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for operating an array column integrator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8435395">US8435395</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">May 7, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8441044">US8441044</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">May 14, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for manufacturing low noise chemically-sensitive field effect transistors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8445945">US8445945</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">May 21, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Low noise chemically-sensitive field effect transistors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8450781">US8450781</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">May 28, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8455927">US8455927</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Jun 4, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array with cascoded column circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8470164">US8470164</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2009</td><td class="patent-data-table-td patent-date-value">Jun 25, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8483969">US8483969</a></td><td class="patent-data-table-td patent-date-value">Sep 17, 2010</td><td class="patent-data-table-td patent-date-value">Jul 9, 2013</td><td class="patent-data-table-td ">Illuminia, Inc.</td><td class="patent-data-table-td ">Variation analysis for multiple templates on a solid support</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8487790">US8487790</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Jul 16, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemical detection circuit including a serializer circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8492799">US8492799</a></td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td patent-date-value">Jul 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8492800">US8492800</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">Jul 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensors with sample and hold capacitors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8502278">US8502278</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Aug 6, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically-sensitive sample and hold sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524057">US8524057</a></td><td class="patent-data-table-td patent-date-value">Jul 28, 2011</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524487">US8524487</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array with cascoded column circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8530941">US8530941</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2011</td><td class="patent-data-table-td patent-date-value">Sep 10, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8535513">US8535513</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">Sep 17, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540865">US8540865</a></td><td class="patent-data-table-td patent-date-value">Sep 8, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540866">US8540866</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540867">US8540867</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540868">US8540868</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8558288">US8558288</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Oct 15, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8592153">US8592153</a></td><td class="patent-data-table-td patent-date-value">Sep 5, 2012</td><td class="patent-data-table-td patent-date-value">Nov 26, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for manufacturing high capacitance microwell structures of chemically-sensitive sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8592154">US8592154</a></td><td class="patent-data-table-td patent-date-value">Sep 7, 2012</td><td class="patent-data-table-td patent-date-value">Nov 26, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for high speed operation of a chemically-sensitive sensor array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8653567">US8653567</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Feb 18, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensor with lightly doped drains</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8658017">US8658017</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">Feb 25, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for operating an array of chemically-sensitive sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8673627">US8673627</a></td><td class="patent-data-table-td patent-date-value">May 24, 2010</td><td class="patent-data-table-td patent-date-value">Mar 18, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Apparatus and methods for performing electrochemical reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8685324">US8685324</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 1, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Matched pair transistor circuits</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8698212">US8698212</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Apr 15, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8742469">US8742469</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors with correlated double sampling</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8747748">US8747748</a></td><td class="patent-data-table-td patent-date-value">Jan 19, 2012</td><td class="patent-data-table-td patent-date-value">Jun 10, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemical sensor with conductive cup-shaped sensor surface</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8748947">US8748947</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Jun 10, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors with reset switch</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8766327">US8766327</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Jul 1, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors with in-sensor current sources</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8772698">US8772698</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Jul 8, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">CCD-based multi-transistor active pixel sensor array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20090170728">US20090170728</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 13, 2009</td><td class="patent-data-table-td patent-date-value">Jul 2, 2009</td><td class="patent-data-table-td ">Walt David R</td><td class="patent-data-table-td ">Methods for detecting target analytes and enzymatic reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2530616A2?cl=en">EP2530616A2</a></td><td class="patent-data-table-td patent-date-value">May 3, 2012</td><td class="patent-data-table-td patent-date-value">Dec 5, 2012</td><td class="patent-data-table-td ">Agilent Technologies, Inc.</td><td class="patent-data-table-td ">Identification of aberrant microarray features</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013033073A1?cl=en">WO2013033073A1</a></td><td class="patent-data-table-td patent-date-value">Aug 28, 2012</td><td class="patent-data-table-td patent-date-value">Mar 7, 2013</td><td class="patent-data-table-td ">Ventana Medical Systems, Inc.</td><td class="patent-data-table-td ">Expression of ets related gene (erg) and phosphatase and tensin homolog (pten) correlates with prostate cancer capsular penetration</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013085710A2?cl=en">WO2013085710A2</a></td><td class="patent-data-table-td patent-date-value">Nov 19, 2012</td><td class="patent-data-table-td patent-date-value">Jun 13, 2013</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Expanded radix for polymeric tags</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006100">435/6.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc702/defs702.htm&usg=AFQjCNEwDgPZO8lzVvNBu0VAYuT88y7k4g#C702S022000">702/22</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S004000">435/4</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc422/defs422.htm&usg=AFQjCNHXLT_uXJvUp9ERMJkh6x6moeyOWg#C422S050000">422/50</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0001000000">G01N1/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=B01L0003000000">B01L3/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0070000000">C40B70/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033543000">G01N33/543</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0031000000">G01N31/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033566000">G01N33/566</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=B01J0019000000">B01J19/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033580000">G01N33/58</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0040060000">C40B40/06</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033530000">G01N33/53</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0060140000">C40B60/14</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001000000">C12Q1/00</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=Y10S977/924">Y10S977/924</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00572">B01J2219/00572</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00626">B01J2219/00626</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00722">B01J2219/00722</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00641">B01J2219/00641</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00677">B01J2219/00677</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00644">B01J2219/00644</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01L3/5085">B01L3/5085</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/0061">B01J2219/0061</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00619">B01J2219/00619</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00659">B01J2219/00659</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00702">B01J2219/00702</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00605">B01J2219/00605</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C40B70/00">C40B70/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00617">B01J2219/00617</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B82Y30/00">B82Y30/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00596">B01J2219/00596</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00648">B01J2219/00648</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/0063">B01J2219/0063</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6816">C12Q1/6816</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00612">B01J2219/00612</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00621">B01J2219/00621</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6834">C12Q1/6834</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J19/0046">B01J19/0046</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00459">B01J2219/00459</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00317">B01J2219/00317</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C40B40/06">C40B40/06</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00466">B01J2219/00466</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C40B60/14">C40B60/14</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/005">B01J2219/005</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00576">B01J2219/00576</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00585">B01J2219/00585</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00637">B01J2219/00637</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N33/54313">G01N33/54313</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ZAd0BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6837">C12Q1/6837</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">B82Y30/00</span>, <span class="nested-value">C12Q1/68B10A</span>, <span class="nested-value">C12Q1/68B10</span>, <span class="nested-value">C12Q1/68B2</span>, <span class="nested-value">G01N33/543D</span>, <span class="nested-value">B01J19/00C</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Nov 19, 2013</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20130920</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 13, 2013</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 16, 2013</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-5, 13-21, 25, 59-72 AND 76 IS CONFIRMED.CLAIMS 6-12, 22-24, 26-58 AND73-75 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 8, 2012</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120329</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 12, 2009</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 26, 2007</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 1, 1999</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ILLUMINA, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:CHEE, MARK S.;STUELPNAGEL, JOHN R.;CZARNIK, ANTHONY M.;REEL/FRAME:010207/0076</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19990813</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_4ff636b3d23669b7103f3b3a3a18b4cd.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0yl_oOOGNQobbD0DONqXjXDznscQ\u0026id=ZAd0BAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U2A0rR8M39SoVwuE_483_ITH7sGyg\u0026id=ZAd0BAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U24ZeJwIB1VKVZf2UedSujU2hwLJQ","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Method_of_making_and_decoding_of_array_s.pdf?id=ZAd0BAABERAJ\u0026output=pdf\u0026sig=ACfU3U1Xbe9cuw1WqeNcIXM1-5ffSM7A_Q"},"sample_url":"http://www.google.com/patents/reader?id=ZAd0BAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>