<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7160997 - Identifying polymerase chain reaction primer extension products using ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Identifying polymerase chain reaction primer extension products using fluorescence energy transfer (FET); detecting allelic variation/restriction fragment length polymorphisms"><meta name="DC.contributor" content="Quin Chou" scheme="inventor"><meta name="DC.contributor" content="Dragan Spasic" scheme="inventor"><meta name="DC.contributor" content="Quin Chou" scheme="assignee"><meta name="DC.contributor" content="Dragan Spasic" scheme="assignee"><meta name="DC.date" content="2002-8-15" scheme="dateSubmitted"><meta name="DC.description" content="Methods and compositions are provided for detecting a primer extension product in a reaction mixture. In the subject methods, a primer extension reaction is conducted in the presence of a polymerase having 3′→5′ exonuclease activity and at least one FET labeled oligonucleotide probe that includes a 3′→5′ exonuclease resistant quencher domain. Also provided are systems and kits for practicing the subject methods. The subject invention finds use in a variety of different applications, and are particularly suited for use in high fidelity PCR based reactions, including SNP detection applications, allelic variation detection applications, and the like."><meta name="DC.date" content="2007-1-9" scheme="issued"><meta name="DC.relation" content="EP:1275735:A1" scheme="references"><meta name="DC.relation" content="US:4794073" scheme="references"><meta name="DC.relation" content="US:5177196" scheme="references"><meta name="DC.relation" content="US:5538848" scheme="references"><meta name="DC.relation" content="US:5840493" scheme="references"><meta name="DC.relation" content="US:6072046" scheme="references"><meta name="DC.relation" content="US:6117635" scheme="references"><meta name="DC.relation" content="US:6248526" scheme="references"><meta name="DC.relation" content="US:6660845" scheme="references"><meta name="DC.relation" content="WO:2001042505:A2" scheme="references"><meta name="DC.relation" content="WO:2001086001:A1" scheme="references"><meta name="citation_reference" content="Chou, Q. et al. &quot;Use of Dark-Quenched FRET probes in Real-Time PCR&quot;, American Biotechnology Laboratory,19(8):34 (2001)."><meta name="citation_reference" content="EPO Supplemental Search Report, Mailed Mar. 14, 2006."><meta name="citation_reference" content="Kreuzer, K-A, et al. &quot;Simultaneous Absolute Quantifcation of Target and Control Templates by Real-Time Flourescence Reverse Trancription-PCR Using 4-(4&#39;Dimethylaminophenylazo)benzoic Acid as a Dark Quencher Dye&quot; Clinical Chemistry,47(3):488-490 (2001)."><meta name="citation_patent_number" content="US:7160997"><meta name="citation_patent_application_number" content="US:10/222,943"><link rel="canonical" href="http://www.google.com/patents/US7160997"/><meta property="og:url" content="http://www.google.com/patents/US7160997"/><meta name="title" content="Patent US7160997 - Identifying polymerase chain reaction primer extension products using fluorescence energy transfer (FET); detecting allelic variation/restriction fragment length polymorphisms"/><meta name="description" content="Methods and compositions are provided for detecting a primer extension product in a reaction mixture. In the subject methods, a primer extension reaction is conducted in the presence of a polymerase having 3′→5′ exonuclease activity and at least one FET labeled oligonucleotide probe that includes a 3′→5′ exonuclease resistant quencher domain. Also provided are systems and kits for practicing the subject methods. The subject invention finds use in a variety of different applications, and are particularly suited for use in high fidelity PCR based reactions, including SNP detection applications, allelic variation detection applications, and the like."/><meta property="og:title" content="Patent US7160997 - Identifying polymerase chain reaction primer extension products using fluorescence energy transfer (FET); detecting allelic variation/restriction fragment length polymorphisms"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("A0XsU7_bK4LfsAScsoHgDg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("GBR"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("A0XsU7_bK4LfsAScsoHgDg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("GBR"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7160997?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7160997"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=8hN5BAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7160997&amp;usg=AFQjCNGTXds53V5hhTGGFEtgP1bdafRpSQ" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7160997.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7160997.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20030165920"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7160997"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7160997" style="display:none"><span itemprop="description">Methods and compositions are provided for detecting a primer extension product in a reaction mixture. In the subject methods, a primer extension reaction is conducted in the presence of a polymerase having 3′→5′ exonuclease activity and at least one FET labeled oligonucleotide probe that includes...</span><span itemprop="url">http://www.google.com/patents/US7160997?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7160997 - Identifying polymerase chain reaction primer extension products using fluorescence energy transfer (FET); detecting allelic variation/restriction fragment length polymorphisms</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7160997 - Identifying polymerase chain reaction primer extension products using fluorescence energy transfer (FET); detecting allelic variation/restriction fragment length polymorphisms" title="Patent US7160997 - Identifying polymerase chain reaction primer extension products using fluorescence energy transfer (FET); detecting allelic variation/restriction fragment length polymorphisms"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7160997 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 10/222,943</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jan 9, 2007</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Aug 15, 2002</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Feb 27, 2002</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/DE60332292D1">DE60332292D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE60336639D1">DE60336639D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1952792A1">EP1952792A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1952792B1">EP1952792B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6818420">US6818420</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7754453">US7754453</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8088583">US8088583</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030162184">US20030162184</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030165920">US20030165920</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20090081676">US20090081676</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20110020812">US20110020812</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120245334">US20120245334</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">10222943, </span><span class="patent-bibdata-value">222943, </span><span class="patent-bibdata-value">US 7160997 B2, </span><span class="patent-bibdata-value">US 7160997B2, </span><span class="patent-bibdata-value">US-B2-7160997, </span><span class="patent-bibdata-value">US7160997 B2, </span><span class="patent-bibdata-value">US7160997B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Quin+Chou%22">Quin Chou</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Dragan+Spasic%22">Dragan Spasic</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Quin+Chou%22">Quin Chou</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Dragan+Spasic%22">Dragan Spasic</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7160997.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7160997.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7160997.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (11),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (3),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (8),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (17),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (8)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7160997&usg=AFQjCNEhunJBV6hpF4t-mfDBmEnTs6Q1YA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7160997&usg=AFQjCNHdehBVwMJH_KTLznCGZSm9MBdHhQ">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7160997B2%26KC%3DB2%26FT%3DD&usg=AFQjCNGcxGEGJ8UDkvOW2VL9JwiZk9EUIg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129878439" lang="EN" load-source="ifi">Identifying polymerase chain reaction primer extension products using fluorescence energy transfer (FET); detecting allelic variation/restriction fragment length polymorphisms</invention-title></span><br><span class="patent-number">US 7160997 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA51093779" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">Methods and compositions are provided for detecting a primer extension product in a reaction mixture. In the subject methods, a primer extension reaction is conducted in the presence of a polymerase having 3′→5′ exonuclease activity and at least one FET labeled oligonucleotide probe that includes a 3′→5′ exonuclease resistant quencher domain. Also provided are systems and kits for practicing the subject methods. The subject invention finds use in a variety of different applications, and are particularly suited for use in high fidelity PCR based reactions, including SNP detection applications, allelic variation detection applications, and the like.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(9)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7160997B2/US07160997-20070109-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7160997B2/US07160997-20070109-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7160997B2/US07160997-20070109-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7160997B2/US07160997-20070109-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7160997B2/US07160997-20070109-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7160997B2/US07160997-20070109-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7160997B2/US07160997-20070109-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7160997B2/US07160997-20070109-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7160997B2/US07160997-20070109-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7160997B2/US07160997-20070109-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7160997B2/US07160997-20070109-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7160997B2/US07160997-20070109-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7160997B2/US07160997-20070109-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7160997B2/US07160997-20070109-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7160997B2/US07160997-20070109-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7160997B2/US07160997-20070109-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7160997B2/US07160997-20070109-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7160997B2/US07160997-20070109-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(19)</span></span></div><div class="patent-text"><div mxw-id="PCLM9144545" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A FET labeled probe comprising a nucleic acid intercalator bonded to a FET labeled oligonucleotide, wherein said nucleic acid intercalator is covalently bonded to the 3′ end of said FET labeled oligonucleotide.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The FET labeled probe according to <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said nucleic acid intercalator provides increased stability to the hybrid formed from said FET labeled oligonucleotide.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The FET labeled probe according to <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said nucleic acid intercalator provides exonuclease activity resistance to said FET labeled oligonucleotide.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The FET labeled probe according to <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said nucleic acid intercalator comprises a polycyclic compound.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. The FET labeled probe according to <claim-ref idref="CLM-00004">claim 4</claim-ref>, wherein said polycyclic compound comprises an aromatic ring.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. The FET labeled probe according to <claim-ref idref="CLM-00004">claim 4</claim-ref>, wherein said polycyclic compound comprises at least three rings and not more than six rings.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. The FET labeled probe according to <claim-ref idref="CLM-00004">claim 4</claim-ref>, wherein said polycyclic compound comprises at least three rings, wherein at least two of said rings are fused.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. The FET labeled probe according to <claim-ref idref="CLM-00007">claim 7</claim-ref>, wherein said polycyclic compound is an acridine.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The FET labeled probe according to <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said FET labeled oligonucleotide is a nucleic acid detector molecule that includes a single-stranded target binding sequence linked to fluorophore and dark quencher.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The FET labeled probe according to <claim-ref idref="CLM-00009">claim 9</claim-ref>, wherein said FET labeled oligonucleotide is a probe selected from the group consisting of: Taqman probes, scorpion probes, sunrise probes, molecular beacons, conformationally assisted probes, and in situ hybridization probes.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. A kit for use in detecting the production of a primer extension product in a primer extension reaction mixture, said kit comprising:
<div class="claim-text">(a) a FET labeled oligonucleotide that includes a 3′→5′ exonuclease resistant quencher domain and a nucleic acid intercalator covalently bonded to its 3′ end; and </div>
<div class="claim-text">(b) a minor groove binder; and </div>
<div class="claim-text">(c) a polymerase. </div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. The kit according to <claim-ref idref="CLM-00011">claim 11</claim-ref>, wherein said intercalator, said FET labeled oligonucleotide and said minor groove binder are present as a single composition.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00013" num="00013" class="claim">
      <div class="claim-text">13. A kit for use in detecting the production of a primer extension product in a primer extension reaction mixture, said kit comprising:
<div class="claim-text">(a) a FET labeled oligonucleotide that includes a 3′→5′ exonuclease resistant quencher domain and a nucleic acid intercalator covalently bonded to its 3′ end; and </div>
<div class="claim-text">(b) a minor groove binder. </div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00014" num="00014" class="claim">
      <div class="claim-text">14. The kit according to <claim-ref idref="CLM-00013">claim 13</claim-ref>, wherein said minor groove binder provides increased stability to the hybrid formed from said FET labeled oligonucleotide.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00015" num="00015" class="claim">
      <div class="claim-text">15. The kit according to <claim-ref idref="CLM-00013">claim 13</claim-ref>, wherein said minor groove binder comprises a compound capable of binding within the minor groove of double stranded DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00016" num="00016" class="claim">
      <div class="claim-text">16. The kit according to <claim-ref idref="CLM-00013">claim 13</claim-ref>, wherein said minor groove binder comprises a compound capable of binding within the minor groove of double stranded DNA with an association constant of at least 10<sup>3</sup>M<sup>−1</sup>.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00017" num="00017" class="claim">
      <div class="claim-text">17. The kit according to <claim-ref idref="CLM-00013">claim 13</claim-ref>, wherein said minor groove binder is a compound selected from the group consisting of netropsin, distamycin, lexitropsin, mithramycin, chromomycin A<sub>3</sub>, olivomycin, anthramycin, sibiromycin, pentamidine, stilbamidine, berenil, CC-1065, Hoechst 33258, 4′-6-diamidino-2-phenylindole (DAPI), and derivatives thereof.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00018" num="00018" class="claim">
      <div class="claim-text">18. A kit for use in detecting the production of a primer extension product in a primer extension reaction mixture, said kit comprising:
<div class="claim-text">(a) a FET labeled oligonucleotide that includes a 3′→5′ exonuclease resistant quencher domain and a nucleic acid intercalator covalently bonded to its 3′ end. </div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00019" num="00019" class="claim">
      <div class="claim-text">19. The kit according to <claim-ref idref="CLM-00018">claim 18</claim-ref>, wherein said kit further comprises a polymerase.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES16167089" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS-REFERENCE TO RELATED APPLICATIONS</heading> <p num="p-0002">This application is a continuation-in-part of application Ser. No. 10/087,229 filed on Feb. 27, 2002 now U.S. Pat. No. 6,818,420, the disclosure of which is herein incorporated by reference.</p>
    <heading>INTRODUCTION</heading> <p num="p-0003">1. Technical Field</p>
    <p num="p-0004">The technical field of this invention is the polymerase chain reaction (PCR); and particularly high fidelity real-time PCR.</p>
    <p num="p-0005">2. Background of the Invention</p>
    <p num="p-0006">The polymerase chain reaction (PCR) is a powerful method for the rapid and exponential amplification of target nucleic acid sequences. PCR has facilitated the development of gene characterization and molecular cloning techniques including the direct sequencing of PCR amplified DNA, the determination of allelic variation, and the detection of infectious and genetic disease disorders. PCR is performed by repeated cycles of heat denaturation of a DNA template containing the target sequence, annealing of opposing primers to the complementary DNA strands, and extension of the annealed primers with a DNA polymerase. Multiple PCR cycles result in the exponential amplification of the nucleotide sequence delineated by the flanking amplification primers.</p>
    <p num="p-0007">An important modification of the original PCR technique was the substitution of <i>Thermus aquaticus </i>(Taq) DNA polymerase in place of the Klenow fragment of <i>E. coli </i>DNA pol I (Saiki, et al. Science, 230:1350-1354 (1988)). The incorporation of a thermostable DNA polymerase into the PCR protocol obviates the need for repeated enzyme additions and permits elevated annealing and primer extension temperatures which enhance the specificity of primer:template associations. Taq DNA polymerase thus serves to increase the specificity and simplicity of PCR.</p>
    <p num="p-0008">Although Taq DNA polymerase is used in the vast majority of PCR performed today, it has a fundamental drawback: purified Taq DNA polymerase enzyme is devoid of 3′ to 5′ exonuclease activity and thus cannot excise misinserted nucleotides (Tindall, et al., Biochemistry, 29:5226-5231 (1990)). Consistent with these findings, the observed error rate (mutations per nucleotide per cycle) of Taq polymerase is relatively high; estimates range from 2×10<sup>−4 </sup>during PCR (Saiki et al., Science, 239:487-491 (1988); Keohavaong et al. Proc. Natl. Acad. Sci. USA, 86:9253-9257 (1989)) to 2×10<sup>−5 </sup>for base substitution errors produced during a single round of DNA synthesis of the lacZ gene (Eckert et al., Nucl. Acids Res. 18:3739-3744 (1990)).</p>
    <p num="p-0009">Polymerase induced mutations incurred during PCR increase arithmetically as a function of cycle number. For example, if an average of two mutations occur during one cycle of amplification, 20 mutations will occur after 10 cycles and 40 will occur after 20 cycles. Each mutant and wild type template DNA molecule will be amplified exponentially during PCR and thus a large percentage of the resulting amplification products will contain mutations. Mutations introduced by Taq DNA polymerase during DNA amplification have hindered PCR applications that require high fidelity DNA synthesis. Several independent studies suggest that 3′ to 5′ exonuclease-dependent proofreading enhances the fidelity of DNA synthesis (Reyland et al, J. Biol. Chem., 263:6518-6524, 1988; Kunkel et al, J. Biol. Chem., 261:13610-13616, 1986; Bernad et al, Cell, 58:219-228, 1989). As such, it is desirable, where possible, to include a 3′ to 5′ exonuclease-dependent proofreading activity in PCR based reactions. For example, if Taq DNA Polymerase (error rate 2×10<sup>−4</sup>) is used to amplify a 100 bp sequence for 40 cycles by PCR, about 55% of the amplification products will contain one or more errors. In contrast, if a Pwo DNA Polymerase having proof-reading activities is used for the amplification, only 10% of the products will contain an error under the same conditions. The error rate produced by a mixture of Taq DNA Polymerase and a proofreading DNA Polymerase between these two values (Cline et al, Nucleic Acids Res., 24(18):3546-51, 1996).</p>
    <p num="p-0010">In many PCR based reactions, a signal producing system is employed, e.g., to detect the production of amplified product. One type of signal producing system that is attractive for use in PCR based reactions is the fluorescence energy transfer (FET) system, in which a nucleic acid detector includes fluorescence donor and acceptor groups. FET label systems include a number of advantages over other labeling systems, including the ability to perform homogeneous assays in which a separation step of bound vs. unbound labeled nucleic acid detector is not required.</p>
    <p num="p-0011">In such real time detection systems using a FET labeled nucleic acid detector, high fidelity amplification is critical. Any error in sequences where a FET labeled nucleic acid detector binds can cause probes not to bind or wrong probes to bind in the case of allele discrimination, resulting in weak signal or the wrong signal being produced. For example, if a 30 bp PCR fragment which is the target of a FET labeled probe is amplified using Taq DNA Polymerase for 40 cycles, about 22% of the amplification fragments will contain one or more errors. In contrast, if a Pwo DNA, Polymerase having proof-reading activities is used for the amplification, only 3% of the amplification fragments will contain an error under the same conditions. Therefore, the standard low fidelity amplification can cause a decrease in sensitivity or mis-typing in the case of allele discrimination.</p>
    <p num="p-0012">However, as discovered by the current invention a disadvantage of currently available FET labeled nucleic acids having TAMRA or Dabcyl as a quencher is that such nucleic acids are subject to 3′→5′ exonuclease degradation. Accordingly, such FET labeled nucleic acids are not suitable for use in high fidelity PCR applications, where 3′→5′ exonuclease activity, i.e., proofreading activity, is present.</p>
    <p num="p-0013">As such, there is significant interest in the identification and development of FET labeled nucleic acids that can be used in high fidelity PCR applications.</p>
    <p num="h-0003">Relevant Literature</p>
    <p num="p-0014">U.S. Patents of interest include: U.S. Pat. Nos. 5,538,848 and 6,248,526. Also of interest are: WO 01/86001 and WO 01/42505.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0015">Methods and compositions are provided for detecting a primer extension product in a reaction mixture. In certain embodiments of the subject methods, a primer extension reaction is conducted in the presence of a polymerase having 3′→5′ exonuclease activity and at least one FET labeled oligonucleotide probe that includes a 3′→5′ exonuclease resistant quencher domain. In certain embodiments a nucleic acid intercalator is covalently bonded to the FET labeled oligonucleotide. In certain embodiments, a minor groove binder is employed in the reaction. In these latter two embodiments, the polymerase may or may not include a 3′→5′ exonuclease activity. Also provided are systems and kits for practicing the subject methods. The subject invention finds use in a variety of different applications, and is particularly suited for use in high fidelity PCR based reactions, including SNP detection applications, allelic variation detection applications, and the like.</p>
    <p num="h-0005">Definitions</p>
    <p num="p-0016">As used herein, “nucleic acid” means either DNA, RNA, single-stranded or double-stranded, and any chemical modifications thereof. Modifications include, but are not limited to, those which provide other chemical groups that incorporate additional charge, polarizability, hydrogen bonding, electrostatic interaction, and functionality to the nucleic acid. Such modifications include, but are not limited to, 2′-position sugar modifications, 5-position pyrimidine modifications, 8-position purine modifications, modifications at exocyclic amines, substitution of 4-thiouridine, substitution of 5-bromo or 5-iodo-uracil; backbone modifications, methylations, unusual base-pairing combinations such as the isobases isocytidine and isoguanidine and the like. Modifications can also include 3′ and 5′ modifications such as capping.</p>
    <p num="p-0017">As used herein, “fluorescent group” refers to a molecule that, when excited with light having a selected wavelength, emits light of a different wavelength. Fluorescent groups may also be referred to as “fluorophores”.</p>
    <p num="p-0018">As used herein, “fluorescence-modifying group” refers to a molecule that can alter in any way the fluorescence emission from a fluorescent group. A fluorescence-modifying group generally accomplishes this through an energy transfer mechanism. Depending on the identity of the fluorescence-modifying group, the fluorescence emission can undergo a number of alterations, including, but not limited to, attenuation, complete quenching, enhancement, a shift in wavelength, a shift in polarity, a change in fluorescence lifetime. One example of a fluorescence-modifying group is a quenching group.</p>
    <p num="p-0019">As used herein, “energy transfer” refers to the process by which the fluorescence emission of a fluorescent group is altered by a fluorescence-modifying group. If the fluorescence-modifying group is a quenching group, then the fluorescence emission from the fluorescent group is attenuated (quenched). Energy transfer can occur through fluorescence resonance energy transfer, or through direct energy transfer. The exact energy transfer mechanisms in these two cases are different. It is to be understood that any reference to energy transferee in the instant application encompasses all of these mechanistically-distinct phenomena. Energy transfer is also referred to herein as fluorescent energy transfer or FET.</p>
    <p num="p-0020">As used herein, “energy transfer pair” refers to any two, molecules that participate in energy transfer. Typically, one of the molecules acts as a fluorescent group, and the other acts as a fluorescence-modifying group. The preferred energy transfer pair of the instant invention comprises a fluorescent group and a quenching group. In some cases, the distinction between the fluorescent group and the fluorescence-modifying group may be blurred. For example, under certain circumstances, two adjacent fluorescein groups can quench one another's fluorescence emission via direct energy transfer. For this reason, there is no limitation on the identity of the individual members of the energy transfer pair in this application. All that is required is that the spectroscopic properties of the energy transfer pair as a whole change in some measurable way if the distance between the individual members is altered by some critical amount.</p>
    <p num="p-0021">“Energy transfer pair” is used to refer to a group of molecules that form a single complex within which energy transfer occurs. Such complexes may comprise, for example, two fluorescent groups which may be different from one another and one quenching group, two quenching groups and one fluorescent group, or multiple fluorescent groups and multiple quenching groups. In cases where there are multiple fluorescent groups and/or multiple quenching groups, the individual groups may be different from one another.</p>
    <p num="p-0022">As used herein, “quenching group” refers to any fluorescence-modifying group that can attenuate at least partly the light emitted by a fluorescent group. We refer herein to this attenuation as “quenching”. Hence, illumination of the fluorescent group in the presence of the quenching group leads to an emission signal that is less intense than expected, or even completely absent. Quenching occurs through energy transfer between the fluorescent group and the quenching group.</p>
    <p num="p-0023">As used herein, “fluorescence resonance energy transfer” or “FRET” refers to an energy transfer phenomenon in which the light emitted by the excited fluorescent group is absorbed at least partially by a fluorescence-modifying, group. If the fluorescence-modifying group is a quenching group, then that group can either radiate the absorbed light as light of a different wavelength, or it can dissipate it as heat. FRET depends on an overlap between the emission spectrum of the fluorescent group and the absorption spectrum of the quenching group. FRET also depends on the distance between the quenching group and the fluorescent group. Above a certain critical distance, the quenching group is unable to absorb the light emitted by the fluorescent group, or can do so only poorly.</p>
    <p num="p-0024">As used herein “direct energy transfer” refers to an energy transfer mechanism in which passage of a photon between the fluorescent group and the fluorescence-modifying group does not occur. Without being bound by a single mechanism, it is believed that in direct energy transfer, the fluorescent group and the fluorescence-modifying group interfere with each others electronic structure. If the fluorescence-modifying group is a quenching group, this will result in the quenching group preventing the fluorescent group from even emitting light.</p>
    <p num="p-0025">In general, quenching by direct energy transfer is more efficient than quenching by FRET. Indeed, some quenching groups that do not quench particular fluorescent groups by FRET (because they do not have the necessary spectral overlap with the fluorescent group) can do so efficiently by direct energy transfer. Furthermore, some fluorescent groups can act as quenching groups themselves if they are close enough to other fluorescent groups to cause direct energy transfer. For example, under these conditions, two adjacent fluorescein groups can quench one another's fluorescence effectively. For these reasons, there is no limitation on the nature of the fluorescent groups and quenching groups useful for the practice of this invention.</p>
    <p num="p-0026">An example of “stringent hybridization conditions” is hybridization at 50° C. or higher and 6.0×SSC (900 mM NaCl/90 mM sodium citrate). Another example of stringent hybridization conditions is overnight incubation at 42° C. or higher in a solution: 50% formamide, 6×SSC (900 mM NaCl, 90 mM trisodium citrate), 50 mM sodium phosphate (pH 7.6), 10% dextran sulfate, and 20 μg/ml denatured, sheared salmon sperm DNA. Stringent hybridization conditions are hybridization conditions that are at least as stringent as the above representative conditions, where conditions are considered to be at least as stringent if they are at least about 80% as stringent typically at least about 90% as stringent as the above specific stringent conditions. Other stringent hybridization conditions are known in the art and, may also be employed.</p>
    <p num="p-0027">As used herein, the term “nucleic acid intercalator” refers to a molecule that binds to a nucleic acid by inserting itself in between base pairs of adjacent nucleotides without unwinding and without extension of the nucleic acid helix. In general, nucleic acid intercalators are aromatic compounds having a flat configuration, and are preferably polycyclic.</p>
    <p num="p-0028">As used herein, the term “minor groove binder” refers to a molecule that binds to a nucleic acid by inserting itself into the minor groove of a DNA helix. In general, minor groove binders are capable of binding within the minor groove of a DNA helix with an association constant of 10<sup>3</sup>M<sup>−1 </sup>or greater. In addition, minor groove binders generally have crescent shaped three-dimensional structures.</p>
    <p num="p-0029">As used herein, “3′ end” means at any location on the oligonucleotide from and including the 3′ terminus to the center of the oligonucleotide, usually at any location from and including the 3′ terminus to about 10 bp from the 3′ terminus, and more usually at any location from and including the 3′ terminus to about 5 bp from the 3′ terminus.</p>
    <p num="p-0030">As used herein, “5′ end” means at any location on the oligonucleotide from and including the 5′ terminus to the center of the oligonucleotide, usually at any location from and including the 5′ terminus to about 10 bp from the 5′ terminus, and more usually at any location from and including the 5′ terminus to about 5 bp from the 5′ terminus.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p num="p-0031"> <figref idrefs="DRAWINGS">FIGS. 1A</figref> to <b>1</b>E provide graphical results of assays comparing the function of various TET labeled FET probes under high fidelity and standard PCR conditions.</p>
      <p num="p-0032"> <figref idrefs="DRAWINGS">FIGS. 2A</figref> to <b>2</b>B provide graphical results of assays comparing the function of FAM/BHQ1 FET probe and FAM/TAMRA FET probe under high fidelity and standard PCR conditions.</p>
      <p num="p-0033"> <figref idrefs="DRAWINGS">FIG. 3</figref> provides graphical results of a multicomponent analysis of a FAM/TAMRA FET oligo primer and a FAM/BHQ1 FET oligo primer.</p>
      <p num="p-0034"> <figref idrefs="DRAWINGS">FIG. 4</figref> provides graphical results of assays for allele discrimination using FAM and TET labeled FET probes under high fidelity PCR conditions.</p>
    </description-of-drawings> <heading>DESCRIPTION OF THE SPECIFIC EMBODIMENTS</heading> <p num="p-0035">Methods and compositions are provided for detecting a primer extension product in a reaction mixture. In certain embodiments of subject methods, a primer extension reaction is conducted in the presence of a polymerase having 3′→5′ exonuclease activity and at least one FET labeled oligonucleotide probe that includes a 3′→5′ exonuclease resistant quencher domain. In certain embodiments, a nucleic acid intercalator covalently bonded to said FET labeled oligonucleotide. In certain embodiments, a minor groove binder is employed in the reaction. In these latter two embodiments, the polymerase employed may or may not have 3′→5′ exonuclease activity. Also provided are systems and kits for practicing the subject methods. The subject invention finds use in a variety of different applications, and is particularly suited for use in high fidelity PCR based reactions, including SNP detection applications, allelic variation detection applications, and the like.</p>
    <p num="p-0036">Before the subject invention is described further, it is to be understood that the invention is not limited to the particular embodiments of the invention described below, as variations of the particular embodiments may be made and still fall within the scope of the appended claims. It is also to be understood that the terminology employed is for the purpose of describing particular embodiments, and is not intended to be limiting. Instead, the scope of the present invention will be established by the appended claims.</p>
    <p num="p-0037">In this specification and the appended claims, the singular forms “a,” “an” and “the” include plural reference unless the context clearly dictates otherwise. Unless defined otherwise, all technical and scientific terms used herein have the same meaning as commonly understood to one of ordinary skill in the art to which this invention belongs.</p>
    <p num="p-0038">Where a range of values is provided, it is understood that each intervening value, to the tenth of the unit of the lower limit unless the context clearly dictates otherwise, between the upper and lower limit of that range, and any other stated or intervening value in that stated range, is encompassed within the invention. The upper and lower limits of these smaller ranges may independently be included in the smaller ranges, and are also encompassed within the invention, subject to any specifically excluded limit in the stated range. Where the stated range includes one or both of the limits, ranges excluding either or both of those included limits are also included in the invention.</p>
    <p num="p-0039">Unless defined otherwise, all technical and scientific terms used herein have the same meaning as commonly understood to one of ordinary skill in the art to which this invention belongs. Although any methods, devices and materials similar or equivalent to those described herein can be used in the practice or testing of the invention, the preferred methods, devices and materials are now described.</p>
    <p num="p-0040">All publications mentioned herein are incorporated herein by reference for the purpose of describing and disclosing those components that are described in the publications that might be used in connection with the presently described invention.</p>
    <p num="p-0041">As summarized above, the subject invention provides methods of detecting the production of a primer extension product in a primer extension reaction mixture by using a FET labeled oligonucleotide probe. In further describing the subject invention, the methods are described first in greater detail, followed by a review of representative specific applications in which the methods finds use, as well as systems and kits that find use in practicing the subject methods.</p>
    <p num="h-0008">Methods</p>
    <p num="p-0042">As summarized above, the subject invention provides methods for detecting the production of primer extension products in a primer extension reaction mixture. In other words, the subject invention provides methods of determining whether primer extension products are produced in a primer extension reaction. By primer extension product is meant a nucleic acid molecule that results from a template dependent primer extension reaction. Template dependent primer extension reactions are those reactions in which a polymerase extends a nucleic acid primer molecule that is hybridized to a template nucleic acid molecule, where the sequence of bases that is added to the terminus of the primer nucleic acid molecule is determined by the sequence of bases in the template strand. Template dependent primer extension reactions include both amplification and non-amplification primer extension reactions. In many embodiments of the subject invention, the template dependent primer extension reaction in which the production of primer extension products is detected is an amplification reaction, e.g., a polymerase chain reaction (PCR).</p>
    <p num="p-0043">A feature of certain embodiments of the subject methods is that the template dependent primer extension reaction in which the production of primer extension products is detected is a “high fidelity” reaction. By “high fidelity” reaction is meant that the reaction has a low error rate, i.e., a low rate of wrong nucleotide incorporation. As such, the error rate of the subject reactions is typically less than about 2×10<sup>−4</sup>, usually less than about 1×10<sup>−5 </sup>and more usually less than about 1×10<sup>−6</sup>. In other embodiments, the reaction mixture is not a high fidelity reaction mixture.</p>
    <p num="h-0009">Preparation of High Fidelity Reaction Mixture</p>
    <p num="p-0044">In practicing the subject methods of these embodiments, the first step is to produce a “high fidelity” primer extension mixture, e.g., a composition of matter that includes, all of the elements necessary for a high fidelity primer extension reaction to occur, where the primer extension mixture further includes at least one FET, labeled oligonucleotide that includes a 3′→5′ exonuclease resistant quencher domain.</p>
    <p num="p-0045">FET occurs when a suitable fluorescent energy donor and an energy acceptor moiety are in close proximity to one another. The excitation energy absorbed by the donor is transferred to the acceptor which can then further dissipate this energy either by fluorescent emission if a fluorophore, or by non-fluorescent means if a quencher. A donor-acceptor pair comprises two fluorophores having overlapping spectra, where the donor emission overlaps the acceptor absorption, so that there is energy transfer from the excited fluorophore to the other member of the pair. It is not essential that the excited fluorophore actually fluoresce, it being sufficient that the excited fluorophore be able to efficiently absorb the excitation energy and efficiently transfer it to the emitting fluorophore.</p>
    <p num="p-0046">As such, the FET labeled oligonucleotides employed in the subject methods are nucleic acid detectors that include a fluorophore domain where the fluorescent energy donor, i.e., donor, is positioned and an acceptor domain where the fluorescent energy acceptor, i.e., acceptor, is positioned. As mentioned above, the donor domain includes the donor fluorophore. The donor fluorophore may be positioned anywhere in the nucleic acid detector, but is typically present at the 5′ terminus of the detector.</p>
    <p num="p-0047">The acceptor domain includes the fluorescence energy acceptor. The acceptor may be positioned anywhere in the acceptor domain, but is typically present at the 3′ terminus of the nucleic acid detector.</p>
    <p num="p-0048">In addition to the fluorophore and acceptor domains, the FET labeled oligonucleotides also include a target nucleic acid binding domain, which binds to a target nucleic acid sequence, e.g., under stringent hybridization conditions (as defined above). This target binding domain typically ranges in length from about 10 to about 60 nt, usually from about 15 to about 30 nt. Depending on the nature of the oligonucleotide and the assay itself, the target binding domain may hybridize to a region of the template nucleic acid or a region of the primer extension product. For example, where the assay is a 5′ nuclease assay, e.g., in which a Taqman type oligonucleotide probe is employed, the target binding domain hybridizes under stringent conditions to a target binding site of the template nucleic acid, which is downstream or 3′ of the primer binding site. In alternative embodiments, e.g., in molecular beacon type assays, the target binding domain hybridizes to a domain of a primer extension product.</p>
    <p num="p-0049">The overall length of the FET labeled oligonucleotides, which includes all three domains mentioned above, typically ranges from about 10 to about 60 nt, usually from about 15 to about 30 nt.</p>
    <p num="p-0050">The donor fluorophore of the subject probes is typically one that is excited efficiently by a single light source of narrow bandwidth, particularly a laser source. The emitting or accepting fluorophores are selected to be able to receive the energy from the donor fluorophore and emit light. Usually the donor fluorophores will absorb in the range of about 350-800 nm, more usually in the range of about 350-600 nm or 500-750 nm. The transfer of the optical excitation from the donor to the acceptor depends on the distance between the two fluorophores. Thus, the distance must be chosen to provide efficient energy transfer from the donor to the acceptor. The distance between the donor and acceptor moieties on the FET oligonucleotides employed in the subject invention, at least in certain configurations (such as upon intramolecular association) typically ranges from about 10 to about 100 angstroms</p>
    <p num="p-0051">The fluorophores for FET pairs may be selected so as to be from a similar chemical family or a different one, such as cyanine dyes, xanthenes or the like. Fluorophores of interest include, but are not limited to: fluorescein dyes (e.g., 5-carboxyfluorescein (5-FAM), 6-carboxyfluorescein (6-FAM), 2′,4′,1,4,-tetrachlorofluorescein (TET), 2′,4′, 5′,7′,1,4-hexachlorofluorescein (HEX), and 2′,7′-dimethoxy-4′, 5′-dichloro-6-carboxyfluorescein (JOE)), cyanine dyes such as Cy5, dansyl derivatives, rhodamine dyes (e.g., tetramethyl-6-carboxyrhodamine (TAMRA), and tetrapropano-6-carboxyrhodamine (ROX)), DABSYL, DABCYL, cyanine, such as Cy3, anthraquinone, nitrothiazole, and nitroimidazole compounds, and the like. Fluorophores of interest are further described in WO 01/42505 and WO 01/86001, as well as the priority U.S. Applications of these documents the disclosures of the latter of which are herein incorporated by reference.</p>
    <p num="p-0052">A feature of the subject FET labeled oligonucleotides is that they are 3′→5′ exonuclease resistant. As such, they are not degraded by 3′→5′ exonucleases, i.e., enzymes having 3′→5′ exonuclease activity. The 3′→5′ exonuclease resistance of the subject FET labeled oligonucleotides may arise from the presence of the acceptor moiety present in the acceptor domain of the FET labeled oligonucleotide. In many, though not all embodiments, the acceptor moiety is present at the 3′ terminus of the acceptor domain, and in many embodiments at the 3′ terminus of the FET labeled oligonucleotide as a whole.</p>
    <p num="p-0053">Any acceptor or donor that imparts 3′→5′ exonuclease resistance onto the FET labeled oligonucleotides may be employed. In many embodiments, the acceptor moiety is a quencher molecule, e.g., a molecule that absorbs transferred energy but does not emit fluorescence, e.g., a dark quencher. In many embodiments, the dark quencher has maximum absorbance of between about 400 and about 700 nm, and often between about 500 and about 600 nm.</p>
    <p num="p-0054">In certain embodiments, the dark quencher comprises a substituted 4-(phenyldiazenyl)phenylamine structure, often comprising at least two residues selected from aryl, substituted aryl, heteroaryl, substituted heteroaryl and combination thereof, wherein at least two of said residues are covalently linked via an exocyclic diazo bond.</p>
    <p num="p-0055">In certain embodiments, the dark quencher is described by the following formula: 
<chemistry id="CHEM-US-00001" num="00001"> <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7160997B2/US07160997-20070109-C00001.png"> <img id="EMI-C00001" he="19.90mm" wi="75.18mm" file="US07160997-20070109-C00001.TIF" alt="Figure US07160997-20070109-C00001" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US7160997B2/US07160997-20070109-C00001.png" class="patent-full-image" width="300" height="79"> </a> </div> <attachments> <attachment idref="CHEM-US-00001" attachment-type="cdx" file="US07160997-20070109-C00001.CDX"> </attachment> <attachment idref="CHEM-US-00001" attachment-type="mol" file="US07160997-20070109-C00001.MOL"> </attachment> </attachments> </chemistry> <br>
wherein:
</p> <ul> <li id="ul0001-0001" num="0000"> <ul> <li id="ul0002-0001" num="0055">R<sub>0</sub>, R<sub>1</sub>, R<sub>2</sub>, R<sub>3</sub>, R<sub>4</sub>, R<sub>5 </sub>are independently: —H, halogen, —O(CH<sub>2</sub>)<sub>n</sub>CH<sub>3</sub>, —(CH<sub>2</sub>)<sub>n</sub>CH<sub>3</sub>, —</li> <li id="ul0002-0002" num="0056">NO<sub>2</sub>, SO<sub>3</sub>, —N[(CH<sub>2</sub>)<sub>n</sub>CH<sub>3</sub>]<sub>2 </sub>wherein n=0 to 5 or —CN;</li> <li id="ul0002-0003" num="0057">R<sub>6 </sub>is —H or —(CH<sub>2</sub>)<sub>n</sub>CH<sub>3 </sub>where n=0 to 5; and</li> <li id="ul0002-0004" num="0058">v is a number from 0 to 10.</li> </ul> </li> </ul> <p num="p-0056">Dark quenchers of interest are further described in WO 01/42505 and WO 01/86001, as well as the priority U.S. Applications of these documents, the disclosures of the latter of which are herein incorporated by reference.</p>
    <p num="p-0057">The FET labeled oligonucleotide may be structured in a variety of different ways, so long as it includes the above-described donor, acceptor and target nucleic acid binding domains. Typically, the FET labeled oligonucleotide is structured such that energy transfer occurs between the fluorophore and acceptor of the FET labeled oligonucleotide probe upon fluorophore excitation when the FET labeled oligonucleotide is not hybridized to target nucleic acid.</p>
    <p num="p-0058">In certain embodiments, the oligonucleotide is a single stranded molecule that does not form intramolecular structures and in which energy transfer occurs because the spacing of the donor and acceptor provides for energy transfer in the single stranded linear format. In these embodiments, energy transfer also occurs between the fluorophore and acceptor of FET labeled oligonucleotide probe upon fluorophore excitation when the FET labeled oligonucleotide probe is hybridized to a target nucleic acid. Specific examples of such FET labeled oligonucleotide probes include the Taqman™ type probes, as described in U.S. Pat. No. 6,248,526, the disclosure of which is herein incorporated by reference (as well as Held et al., Genome Res. (1996) 6:986-994; Holland et al., Proc. Nat'l Acad. Sci. USA (1991) 88:7276-7280; and Lee et al., Nuc. Acids Res. (1993) 21:3761-3766 (1993)). In many of these embodiments, the target nucleic acid binding domain is one that hybridizes to, i.e., is complementary to, a sequence of the template nucleic acid, i.e., the target nucleic acid of the target nucleic acid binding domain is a sequence present in the template nucleic acid.</p>
    <p num="p-0059">In other embodiments, the probe oligonucleotides are structured such that energy transfer does not occur between the fluorophore and acceptor of said FET labeled oligonucleotide probe upon fluorophore excitation when the FET labeled oligonucleotide probe is hybridized to a target nucleic acid. Examples of these types of probe structures include: Scorpion probes (as described in Whitcombe et al., (Nature Biotechnology (1999) 17:804-807; U.S. Pat of which is herein incorporated by reference), Sunrise probes (as described in Nazarenko et al., Nuc. Acids Res. (1997) 25:2516-2521; U.S. Pat. No. 6,117,635, the disclosure of which is herein incorporated by reference), Molecular Beacons (Tyagi et al., Nature Biotechnology (1996) 14:303-308; U.S. Pat. No. 5,989,823, the disclosure of which is incorporated herein by reference), and conformationally assisted probes (as described in provisional application Ser. No. 60/138,376, the disclosure of which is herein incorporated by reference). In many of these embodiments, the target binding sequence or domain comprises a hybridization domain complementary to a sequence of the primer extension product.</p>
    <p num="p-0060">Since the primer extension reaction mixture produced in the initial step of the subject methods of these particular embodiments is a high fidelity primer extension reaction mixture, it further includes an enzyme having 3′→5′ exonuclease activity. In many embodiments, the 3′→5′ exonuclease is a polymerase that has 3′→5′ exonuclease activity. In many embodiments, the high fidelity nature of the reaction mixture is provided by the presence of a combination of two or more polymerases, at least one of which includes a 3′→5′ exonuclease. In certain embodiments, e.g., in 5′ nuclease applications, care is taken to ensure that a polymerase having 5′→3′ nuclease activity is also included. In many embodiments, the polymerase combination employed includes at least one Family A polymerase and, in many embodiments, a Family A polymerase and a Family B polymerase, where the terms “Family A” and “Family B” correspond to the classification scheme reported in Braithwaite &amp; Ito, Nucleic Acids Res. (1993) 21:787-802. Family A polymerases of interest include <i>Thermus aquaticus </i>polymerases, including the naturally occurring polymerase (Taq) and derivatives and homologues thereof, such as Klentaq (as described in Proc. Natl. Acad. Sci USA (1994) 91:2216-2220); <i>Thermus thermoophilus </i>polymerases, including the naturally occurring polymerase (Tth) and derivatives and homologues thereof, and the like. Family B polymerases of interest include <i>Thermococcus litoralis </i>DNA polymerase (Vent) as described in Perler et al., Proc. Natl. Acad. Sci. USA (1992) 89:5577; <i>Pyrococcus </i>species GB-D (Deep Vent); <i>Pyrococcus furiosus </i>DNA polymerase (Pfu) as described in Lundberg et al., Gene (1991) 108:1-6, <i>Pyrococcus woesei </i>(Pwo) and the like. Of the two types of polymerases employed, the Family A polymerase will typically be present the reaction mixture in an amount greater than the Family B polymerase, where the difference in activity will usually be at least 10-fold, and more usually at least about 100-fold. Accordingly, the reaction mixture will typically comprise from about 0.1 U/μl to 1 U/μl Family A polymerase, usually from about 0.2 to 0.5 U/μl Family A polymerase, while the amount of Family B polymerase will typically range from about 0.01 mU/μl to 10 mU/μl, usually from about 0.05 to 1 mU/μl and more usually from about 0.1 to 0.5 mU/μl, where “U” corresponds to incorporation of 10 nmoles dNTP into acid-insoluble material in 30 min at 74° C.</p>
    <p num="p-0061">In certain embodiments, a nucleic acid intercalator is present in the reaction mixture. In certain of these embodiments, the nucleic acid intercalator is bonded to the FET probe, such that it is stably associated with the FET probe under the conditions of use. In many of these embodiments, the FET probes employed in the subject invention include a nucleic acid intercalator covalently bonded to the FET labeled oligonucleotide.</p>
    <p num="p-0062">The nucleic acid intercalators of interest function to stabilize the nucleic acid helix formed from the FET labeled oligonucleotide. Covalent binding of nucleic acid intercalators of interest also serve to increase hybrid stability by providing additional binding energy. Furthermore, nucleic acid intercalators of interest provide the FET labeled oligonucleotide greater affinity for its complementary sequence. In addition, nucleic acid intercalators provides exonuclease activity resistance when added to the 3′ end of said FET labeled oligonucleotide. The nucleic acid intercalator component of the FET probes of this embodiment may be located at any location of the FET probe, e.g., at the 3′ end or the 5′ end of said FET labeled oligonucleotide, where in many embodiments the intercalator is covalently bonded to the 3′ end, e.g., within at least about 10, such as within about 5 nt residues of the 3′end.</p>
    <p num="p-0063">In general, nucleic acid intercalators of interest are aromatic compounds having a flat configuration. They may be cyclic, or polycyclic, particularly polycyclic aromatic having at least two rings, usually at least three rings and not more than about six rings, more usually not more than about five rings, where at least two of the rings are fused, usually at least three of the rings are fused, and usually not more than four of the rings being fused. The aromatic compound may be carbocyclic or heterocyclic, particularly having from one to three, more usually one to two nitrogen atoms as heteroannular atoms. Other heterannular atoms may include oxygen and sulfur (chalcogen). The rings may be substituted by a wide variety of substituents, which substituents may include alkyl groups of from one to four carbon atoms, usually from one to two carbon atoms, oxy, which includes hydroxy, alkoxy and carboxy ester, generally of from one to four carbon atoms, amino, including mono- and disubstituted amino, particularly mono- and dialkyl amino, of from 0 to 8, usually 0 to 6 carbon atoms, thio, particularly alkylthio from 1 to 4, usually 1 to 2 carbon atoms, cyano, non-oxo-carbonyl, such as carboxy and derivatives thereof, particularly carboxamide or carboxyalkyl, of from 1 to 8 or 1 to 6 carbon atoms, usually 2 to 6 carbon atoms and more usually 2 to 4 carbon atoms, oxo-carbonyl or acyl, generally from 1 to 4 carbon atoms, halo, particularly of atomic number 9 to 35, etc.</p>
    <p num="p-0064">Polycyclic compounds which find use as nucleic acid intercalators include acridines, phenanthridines, porphyrins, phenylindoles, and bisbenzamides.</p>
    <p num="p-0065">The intercalator may be covalently bonded to the FET labeled oligonucleotide using any convenient protocol, which may or may not employ a linker group, where representative protocols include those protocols provided herein for covalent bonding of the fluorescer/quencher moieties to the FET labeled probes.</p>
    <p num="p-0066">DNA intercalators are further described in U.S. Pat. No. 6,280,933; the disclosure of which is herein incorporated by reference.</p>
    <p num="p-0067">In certain embodiments, another component of the reaction mixture produced in the first step of the subject methods is a minor groove binder. Minor groove binders of interest are those capable of binding isohelically into the minor groove of double stranded DNA by fitting edge-on into the minor groove, replacing the spine of hydration and following the natural curvature of the DNA. This type of molecular recognition is at least partially driven by formation of hydrogen bonds between the minor groove binder and the DNA. The minor groove binders of interest form bifurcated hydrogen bonds with nucleotide base pairs, and numerous van der Waals contacts with various atoms in the nucleotide backbone. These atomic interactions stabilize the DNA-minor groove binder structure and, in turn, effectively strengthen the interaction of the two DNA strands of the helix.</p>
    <p num="p-0068">In general, minor groove binders of interest are those capable of binding within the minor groove of a DNA helix with an association constant of 10<sup>3</sup>M<sup>−1 </sup>or greater, as determined by using the assay described in Taquet et al., Biochemistry 1998 Jun. 23;37(25):9119-26. The minor groove binders have a strong preference for A-T (adenine and thymine) rich regions of double stranded DNA. However, minor groove binders which show preference to C-G (cytosine and guanine) rich regions are also of interest. Generally, minor groove binders of interest stabilize A-T hydrogen bonds more than C-G bonds, (thus stabilizing weaker sequences). Minor groove binders form extremely stable duplexes</p>
    <p num="p-0069">The minor groove binders may be free in the aqueous buffer medium of the primer extension mixture, or bound to the FET labeled oligonucleotide (covalently or non-covalently). The minor groove binder may be linked to a covalent structure or chain of atoms that attaches the minor groove binder to the oligonucleotide. This linking chain can and sometimes is considered as part of the minor groove binder, and does not adversely affect the minor groove binding properties. Furthermore, the minor groove binders may be attached to either the 3′ end or the 5′ end of the FET labeled oligonucleotide.</p>
    <p num="p-0070">In general, minor groove binders have crescent shaped three-dimensional structures, and generally have a molecular weight of between approximately 150 to approximately 2000 Daltons. Examples of minor groove binders include netropsin, distamycin, distamycin A, lexitropsin, mithramycin, chromomycin A<sub>3</sub>, olivomycin, anthramycin, sibiromycin, pentamidine, stilbamidine, berenil, CC-1065, Hoechst 33258, 4′-6-diamidino-2-phenylindole (DAPI), and derivatives thereof. Typically, the minor groove binder is a N-methylpyrrole peptide such as netropsin and distamycin A. Minor groove binders are further described in U.S. Pat. Nos. 5,801,155 and 5,955,590; the disclosures of which are herein incorporated by reference.</p>
    <p num="p-0071">In those embodiments where the minor groove binder is present in the reaction mixture, the amount of minor groove binder present in the reaction mixture typically ranges from about 0.02 μM to about 1.5 μM, usually from about 0.1 μM to about 0.75 μM and more usually from about 0.2 μM to about 0.5 μM.</p>
    <p num="p-0072">Another component of the reaction mixture produced in the first step of the subject methods is the template nucleic acid. The nucleic acid that serves as template may be single stranded or double stranded, where the nucleic acid is typically deoxyribonucleic acid (DNA). The length of the template nucleic acid may be as short as 50 bp, but usually be at least about 100 bp long, and more usually at least about 150 bp long, and may be as long as 10,000 bp or longer, e.g., 50,000 bp in length or longer, including a genomic DNA extract, or digest thereof, etc. The nucleic acid may be free in solution, flanked at one or both ends with non-template nucleic acid, present in a vector, e.g. plasmid and the like, with the only criteria being that the nucleic acid be available for participation in the primer extension reaction. The template nucleic acid may be present in purified form, or in a complex mixture with other non-template nucleic acids, e.g., in cellular DNA preparation, etc.</p>
    <p num="p-0073">The template nucleic acid may be derived from a variety of different sources, depending on the application for which the PCR is being performed, where such sources include organisms that comprise nucleic acids, i.e. viruses; prokaryotes, e.g. bacteria, archaea and cyanobacteria; and eukaryotes, e.g. members of the kingdom protista, such as flagellates, amoebas and their relatives, amoeboid parasites, ciliates and the like; members of the kingdom fungi, such as slime molds, acellular slime molds, cellular slime molds, water molds; true molds, conjugating fungi, sac fungi, club fungi, imperfect fungi and the like, plants, such as algae, mosses, liverworts, hornworts, club mosses, horsetails, ferns, gymnosperms and flowering plants, both monocots and dicots, and animals, including sponges, members of the phylum cnidaria, e.g. jelly fish, corals and the like, combjellies, worms, rotifers, roundworms, annelids, molluscs, arthropods, echinoderms, acorn worms, and vertebrates, including reptiles, fishes, birds, snakes, and mammals, e.g. rodents, primates, including humans, and the like. The template nucleic acid may be used directly from its naturally occurring source and/or preprocessed in a number of different ways, as is known in the art. In some embodiments, the template nucleic acid may be from a synthetic source.</p>
    <p num="p-0074">The next component of the reaction mixture produced in the first step of the subject methods is the primers employed in the primer extension reaction, e.g., the PCR primers (such as forward and reverse primers employed in geometric amplification or a single primer employed in a linear amplification). The oligonucleotide primers with which the template nucleic acid (hereinafter referred to as template DNA for convenience) is contacted will be of sufficient length to provide for hybridization to complementary template DNA under annealing conditions (described in greater detail below) but will be of insufficient length to form stable hybrids with template DNA under polymerization conditions. The primers will generally be at least 10 bp in length, usually at least 15 bp in length and more usually at least 16 bp in length and may be as long as 30 bp in length or longer, where the length of the primers will generally range from 18 to 50 bp in length, usually from about 20 to 35 bp in length. The template DNA may be contacted with a single primer or a set of two primers (forward and reverse primers), depending on whether primer extension, linear or exponential amplification of the template DNA is desired. Where a single primer is employed, the primer will typically be complementary to one of the 3′ ends of the template DNA and when two primers are employed, the primers will typically be complementary to the two 3′ ends of the double stranded template DNA.</p>
    <p num="p-0075">In addition to the above components, the reaction mixture produced in the subject methods includes deoxyribonucleoside triphosphates (dNTPs). Usually the reaction mixture will comprise four different types of dNTPs corresponding to the four naturally occurring bases are present, i.e. dATP, dTTP, dCTP and dGTP. In the subject methods, each dNTP will typically be present in an amount ranging from about 10 to 5000 μM, usually from about 20 to 1000 μM.</p>
    <p num="p-0076">The reaction mixture prepared in the first step of the subject methods further includes an aqueous buffer medium that includes a source of monovalent ions, a source of divalent cations and a buffering agent. Any convenient source of monovalent ions, such as KCl, K-acetate, NH<sub>4</sub>-acetate, K-glutamate, NH<sub>4</sub>Cl, ammonium sulfate, and the like may be employed. The divalent cation may be magnesium, manganese, zinc and the like, where the cation will typically be magnesium. Any convenient source of magnesium cation may be employed, including MgCl<sub>2</sub>, Mg-acetate, and the like. The amount of Mg<sup>2+</sup> present in the buffer may range from 0.5 to 10 mM, but will preferably range from about 3 to 6 mM, and will ideally be at about 5 mM. Representative buffering agents or salts that may be present in the buffer include Tris, Tricine, HEPES, MOPS and the like, where the amount of buffering agent will typically range from about 5 to 150 mM, usually from about 10 to 100 mM, and more usually from about 20 to 50 mM, where in certain preferred embodiments the buffering agent will be present in an amount sufficient to provide a pH ranging from about 6.0 to 9.5, where most preferred is pH 7.3 at 72° C. Other agents which may be present in the buffer medium include chelating agents, such as EDTA, EGTA and the like.</p>
    <p num="p-0077">In preparing the reaction mixture, the various constituent components may be combined in any convenient order. For example, the buffer may be combined with primer, polymerase and then template DNA, or all of the various constituent components may be combined at the same time to produce the reaction mixture.</p>
    <p num="h-0010">Preparation of Non-High Fidelity Reaction Mixture</p>
    <p num="p-0078">As indicated above, in certain embodiments, e.g., where a DNA intercalator and/or a minor groove binder is present, the reaction mixture need not be a high fidelity reaction mixture. In these embodiments, the reaction mixture is prepared as described above, but the polymerase having 3′→5′ exonuclease activity, e.g., the Family B polymerases reviewed above, is not included in the reaction mixture.</p>
    <p num="h-0011">Subjecting the Primer Extension Mixture to Primer Extension Reaction Conditions</p>
    <p num="p-0079">Following preparation of the reaction mixture, the reaction mixture is subjected to primer extension reaction conditions, i.e., to conditions that permit for polymerase mediated primer extension by addition of nucleotides to the end of the primer molecule using the template strand as a template. In many embodiments, the primer extension reaction conditions are amplification conditions, which conditions include a plurality of reaction cycles, where each reaction cycle comprises: (1) a denaturation step, (2) an annealing step, and (3) a polymerization step. The number of reaction cycles will vary depending on the application being performed, but will usually be at least 15, more usually at least 20 and may be as high as 60 or higher, where the number of different cycles will typically range from about 20 to 40. For methods where more than about 25, usually more than about 30 cycles are performed, it may be convenient or desirable to introduce additional polymerase into the reaction mixture such that conditions suitable for enzymatic primer extension are maintained.</p>
    <p num="p-0080">The denaturation step comprises heating the reaction mixture to an elevated temperature and maintaining the mixture at the elevated temperature for a period of time sufficient for any double stranded or hybridized nucleic acid present in the reaction mixture to dissociate. For denaturation, the temperature of the reaction mixture will usually be raised to, and maintained at, a temperature ranging from about 85 to 100, usually from about 90 to 98 and more usually from about 93 to 96° C. for a period of time ranging from about 3 to 120 sec, usually from about 5 to 30 sec.</p>
    <p num="p-0081">Following denaturation, the reaction mixture will be subjected to conditions sufficient for primer annealing to template DNA present in the mixture. The temperature to which the reaction mixture is lowered to achieve these conditions will usually be chosen to provide optimal efficiency and specificity and will generally range from about 50 to 75, usually from about 55 to 70 and more usually from about 60 to 68° C. Annealing conditions will be maintained for a period of time ranging from about −15 sec to 30 min, usually from about 30 sec to 5 min.</p>
    <p num="p-0082">Following annealing of primer to template DNA or during annealing of primer to template DNA, the reaction mixture will be subjected to conditions sufficient to provide for polymerization of nucleotides to the primer ends in manner such that the primer is extended in a 5′ to 3′ direction using the DNA to which it is hybridized as a template, i.e., conditions sufficient for enzymatic production of primer extension product. To achieve polymerization conditions, the temperature of the reaction mixture will typically be raised to or maintained at a temperature ranging from about 65 to 75, usually from about 67 to 73° C. and maintained for a period of time ranging from about 15 sec to 20 min, usually from about 30 sec to 5 min.</p>
    <p num="p-0083">The above cycles of denaturation, annealing and polymerization may be performed using an automated device, typically known as a thermal cycler. Thermal cyclers that may be employed are described in U.S. Pat. Nos. 5,612,473; 5,602,756; 5,538,871; and 5,475,610, the disclosures of which are herein incorporated by reference.</p>
    <p num="h-0012">Signal Detection</p>
    <p num="p-0084">The next step in the subject methods is signal detection, i.e., detecting a change in a fluorescent signal from the FET labeled oligonucleotide probe present in the reaction mixture to obtain an assay result. In other words, the next step in the subject methods is to detect any modulation in the fluorescent signal generated by the FET labeled oligonucleotide present in the reaction mixture. The change may be an increase or decrease in fluorescence, depending on the nature of the label employed, but in many embodiments is an increase in fluorescence. The sample may be screened for an increase in fluorescence using any convenient means, e.g., a suitable fluorimeter, such as a thermostable-cuvette or plate-reader fluorimeter. Fluorescence is suitably monitored using a known fluorimeter. The signals from these devices, for instance in the form of photo-multiplier voltages, are sent to a data processor board and converted into a spectrum associated with each sample tube. Multiple tubes, for example 96 tubes, can be assessed at the same time. Data may be collected in this way at frequent intervals, for example once every 10 ms, throughout the reaction. By monitoring the fluorescence of the reactive molecule from the sample during each cycle, the progress of the amplification reaction can be monitored in various ways. For example, the data provided by melting peaks can be analyzed, for example by calculating the area under the melting peaks and this data plotted against the number of cycles.</p>
    <p num="p-0085">The spectra generated in this way can be resolved, for example, using “fits” of pre-selected fluorescent moieties such as dyes, to form peaks representative of each signaling moiety (i.e. fluorophore). The areas under the peaks can be determined which represents the intensity value for each signal, and if required, expressed as quotients of each other. The differential of signal intensities and/or ratios will allow changes in FET to be recorded through the reaction or at different reaction conditions, such as temperatures. The changes are related to the binding phenomenon between the oligonucleotide probe and the target sequence or degradation of the oligonucleotide probe bound to the target sequence. The integral of the area under the differential peaks will allow intensity values for the FET effects to be calculated.</p>
    <p num="p-0086">Screening the mixture for a change in fluorescence provides one or more assay results, depending on whether the sample is screened once at the end of the primer extension reaction, or multiple times, e.g., after each cycle, of an amplification reaction (e.g., as is done in real time PCR monitoring).</p>
    <p num="h-0013">Employing Said Assay Result to Determine Whether a Primer Extension Product is Present in Said Mixture</p>
    <p num="p-0087">The data generated as described above can be interpreted in various ways. In its simplest form, an increase or decrease in fluorescence from the sample in the course of or at the end of the amplification reaction is indicative of an increase in the amount of the target sequence present, i.e., primer extension product present, suggestive of the fact that the amplification reaction has proceeded and therefore the target sequence was in fact present in the sample. Quantitation is also possible by monitoring the amplification reaction throughout the amplification process.</p>
    <p num="p-0088">In this manner, a reaction mixture is readily screened for the presence of primer extension products. The methods are suitable for detection of a single primer extension product as well as multiplex analyses, in which two or more different FET labeled oligonucleotide probes are employed to screen for two or more different primer extension products. In these latter multiplex situations, the number of different types of probes that may be employed typically ranges from about 2 to about 20 or higher, usually from about 2 to about 15.</p>
    <p num="p-0089">The above described methods of detecting the presence of one or more types of primer extension reaction products in a primer extension reaction mixture finds use in a variety of different applications, representative ones of which are now reviewed in greater detail.</p>
    <p num="h-0014">Utility</p>
    <p num="p-0090">The above-described inventive methods find use in a variety of different applications. In general, the subject oligonucleotide probes and methods of using the same find use in any high fidelity primer extension reaction in which a FET probe and proofreading polymerase are employed.</p>
    <p num="p-0091">One type of representative application is in monitoring the progress of nucleic acid amplification reactions, such as polymerase chain reaction applications, including both linear and geometric PCR applications. As used herein, the term monitoring includes a single evaluation at the end of a series of reaction cycles as well as multiple evaluations, e.g., after each reaction cycle, such that the methods can be employed to determine whether a particular amplification reaction series has resulted in the production of primer extension product, e.g., a non-real time evaluation, as well as in a real time evaluation of the progress of the amplification reaction.</p>
    <p num="p-0092">The subject methods find use in both 5′ nuclease methods of monitoring a PCR amplification reaction (e.g., where a Taqman type probe is employed), and non-5′ nuclease methods of monitoring a PCR amplification reaction (e.g., where a molecular beacon type probe is employed). Again, the subject methods find use in evaluating the progress of an amplification reaction at a single time (e.g., non-real time monitoring) and in real-time monitoring.</p>
    <p num="p-0093">Monitoring a PCR reaction according to the subject methods finds use in a variety of specific applications. Representative applications of interest include, but are not limited to: (1) detection of allelic polymorphism; (2) SNP detection; (3) detection of rare mutations; (4) detection of allelic stage of single cells; (5) detection of single or low copy number DNA analyte molecules in a sample; etc. For example, in detection of allelic polymorphism, a nucleic acid sample to be screened, e.g., a genomic DNA cellular extract, is employed as template nucleic acid in the preparation of a primer extension reaction mixture, as described above, where the reaction mixture includes a different and distinguishable FET labeled oligonucleotide probe that is specific for each different allelic sequence to be identified, if present. The assay is then carried out as described above, where the sample is screened for a change in signal from each different oligonucleotide probe. A change in signal from a given probe is indicative of the presence the allelic variant to which that probe is specific in the sample. Likewise, an absence of change in signal is indicative of the absence of the allelic variant in the sample. In this manner, the sample is readily screened for the presence of one or more allelic variants. A similar approach can be used for SNP detection, where a different FET labeled oligonucleotide for each SNP of interest to be screened in a nucleic acid sample is employed.</p>
    <p num="p-0094">A significant benefit of employing the subject methods in PCR screening applications is that the PCR conditions may be “high fidelity,” i.e., they may include a proof reading activity, such that the results obtained from the assays performed according to the subject methods are highly reliable.</p>
    <p num="h-0015">Kits</p>
    <p num="p-0095">Also provided are kits for practicing the subject methods. The kits according to the present invention will comprise at least: (a) a FET labeled oligonucleotide, where the kits may include two or more distinguishable FET labeled oligonucleotides, e.g., that hybridize to different target nucleic acids, e.g., two or more different SNPs; and (b) instructions for using the provide FET labeled oligonucleotide(s) in a high fidelity amplification, e.g., PCR, reaction.</p>
    <p num="p-0096">The subject kits may further comprise additional reagents which are required for or convenient and/or desirable to include in the reaction mixture prepared during the subject methods, where such reagents include: one or more polymerases, including a polymerase mix, where the one or more polymerases at least include a polymerase that exhibits proofreading, i.e., 3′→5′ exonuclease activity in certain embodiments; an aqueous buffer medium (either prepared or present in its constituent components, where one or more of the components may be premixed or all of the components may be separate), and the like.</p>
    <p num="p-0097">The various reagent components of the kits may be present in separate containers, or may all be precombined into a reagent mixture for combination with template DNA. For example, the kit may include an intercalator-FET probe and a minor groove binder, where these two components may be present separately or combined into a single composition for use.</p>
    <p num="p-0098">In addition to the above components, the subject kits will further include instructions for practicing the subject methods. These instructions may be present in the subject kits in a variety of forms, one or more of which may be present in the kit. One form in which these instructions may be present is as printed information on a suitable medium or substrate, e.g., a piece or pieces of paper on which the information is printed, in the packaging of the kit, in a package insert, etc. Yet another means would be a computer readable medium, e.g., diskette, CD, etc., on which the information has been recorded. Yet another means that may be present is a website address which may be used via the internet to access the information at a removed site. Any convenient means may be present in the kits.</p>
    <p num="h-0016">Systems</p>
    <p num="p-0099">Also provided are systems for use in practicing the subject methods. The subject systems at least include one or more FET labeled oligonucleotides and a proofreading activity, as well as any other requisite components for preparing a primer extension reaction mixture, as described above. In addition, the subject systems may include any required devices for practicing the subject methods, e.g., thermal cyclers, fluorimeters, etc.</p>
    <p num="p-0100">The following examples are offered by way of illustration and not by way of limitation.</p>
    <heading>Experimental Procedures</heading> <p num="h-0018">A. General</p>
    <p num="p-0101">The oligonucleotides shown in Table 1 were used for the following example. Standard DNA phosphoramidites, including 6-carboxy-fluorescein (6-FAM) phosphoramidite, 5′-Tetrachloro-Fluorescein (TET) phosphoramidite, and 6-carboxytetramethyl-rhodamine (TAMRA) CPG, 3′-Dabcyl CPG, were obtained from Glen Research. Black Hole Quenchers (BHQ1, BHQ2, BHQ3) CPG were obtained from Biosearch Technology. Eclipse Dark Quencher (eDQ) CPG was obtained from Epoch Bioscience. All primers were purified using Oligo Purification Cartridges (Biosearch Technology). Doubly labeled FET probes were synthesized using CPGs with various quenchers as indicated in Table 1 and with either 6′FAM-labeled or TET-labeled phosphoramidites at the 5′end. The doubly labeled FET probes were purified by preparative HPLC and PAGE using standard protocols. Phosphorothioate modification was prepared by standard procedure.</p>
    <p num="p-0102">
      <tables id="TABLE-US-00001" num="00001"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="329pt" align="center"> </colspec> <colspec colname="2" colwidth="0pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="2" rowsep="1" class="description-td" colspan="2">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Sequence Listing</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="91pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="196pt" align="left"> </colspec> <colspec colname="4" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Name</td>
                <td class="description-td">Type</td>
                <td class="description-td">Sequence</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <colspec colname="4" colwidth="196pt" align="left"> </colspec> <colspec colname="5" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">BCL-F1</td>
                <td class="description-td">SEQ ID NO 1</td>
                <td class="description-td">Primer</td>
                <td class="description-td">5′ GGT GGT GGA GGA GCT CTT CAG 3′</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">BCL-R1</td>
                <td class="description-td">SEQ ID NO 2</td>
                <td class="description-td">Primer</td>
                <td class="description-td">5′ CCA GCC TCC GTT ATC CTG GA 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">BCL-P1</td>
                <td class="description-td">SEQ ID NO 3</td>
                <td class="description-td">Probe</td>
                <td class="description-td">5′ FAM-CCT GTG GAT GAC TGA GTA GGT GAA CCG-</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">BHQ1-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">BCL-P2</td>
                <td class="description-td">SEQ ID NO.4</td>
                <td class="description-td">Probe</td>
                <td class="description-td">5′ FAM-CCT GTG GAT GAC TGA GTA CCT GAA CCG-</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">eDQ-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">BCL-P3</td>
                <td class="description-td">SEQ ID NO: 5</td>
                <td class="description-td">Probe</td>
                <td class="description-td">5′ FAM-GGT GTG GAT GAG TGA GTA CCT GAA CCG-</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">DABCYL-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">BCL-P4</td>
                <td class="description-td">SEQ ID NO: 6</td>
                <td class="description-td">Probe</td>
                <td class="description-td">5′ FAM-OCT GTG GAT GAG TGA GTA GGT GAA CCG-</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">TAMRA-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">BCL-P5</td>
                <td class="description-td">SEQ ID NO: 7</td>
                <td class="description-td">S-oligo</td>
                <td class="description-td">5′ FAM-GGT GTG GAT GAG TGA GTA GGT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Probe</td>
                <td class="description-td">GAA*G*G*G-TAMRA-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACT-F1</td>
                <td class="description-td">SEQ ID NO: 8</td>
                <td class="description-td">Primer</td>
                <td class="description-td">5′ GAG CTA CGA GCT GCC TGA C 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACT-R1</td>
                <td class="description-td">SEQ ID NO: 9</td>
                <td class="description-td">Primer</td>
                <td class="description-td">5′ GAC TCC ATG CCC AGG AAG 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACT-P1</td>
                <td class="description-td">SEQ ID NO 10</td>
                <td class="description-td">Probe</td>
                <td class="description-td">5′ TET-CAT CAC CAT TGG CAA TGA GCG-BHQ1-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACT-P2</td>
                <td class="description-td">SEQ ID NO 11</td>
                <td class="description-td">Probe</td>
                <td class="description-td">5′ TET-CAT CAC CAT TGG CAA TGA GCG-eDQ-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACT-P3</td>
                <td class="description-td">SEQ ID NO. 12</td>
                <td class="description-td">Probe</td>
                <td class="description-td">5′ TET-CAT CAC CAT TGG GAA TGA GCG-DABCYL-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACT-P4</td>
                <td class="description-td">SEQ ID NO: 13</td>
                <td class="description-td">Probe</td>
                <td class="description-td">5′ TET-CAT CAC CAT TGG GAA TGA GCG-TAMRA-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACT-P5</td>
                <td class="description-td">SEQ ID NO: 14</td>
                <td class="description-td">S-oligo</td>
                <td class="description-td">5′ TET-CAT CAC CAT TGG GAA TGA *G*G*G-TAMRA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Probe</td>
                <td class="description-td">3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">ABCG-R1</td>
                <td class="description-td">SEQ ID NO: 15</td>
                <td class="description-td">Primer</td>
                <td class="description-td">5′ CCC AAA AAT TCA TTA TGC TGC AA 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">ABCG-P1</td>
                <td class="description-td">SEQ ID NO: 16</td>
                <td class="description-td">Primer</td>
                <td class="description-td">5′ FAM-CAG CAT TCC ACG ATA TGG ATT TAC GGC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">-BHQ1-3</td>
              </tr> <tr class="description-tr"> <td class="description-td">ABCG-P2</td>
                <td class="description-td">SEQ ID NO: 17</td>
                <td class="description-td">Primer</td>
                <td class="description-td">5′ FAM-CAG CAT TCC ACG ATA TGG ATT TAC GGC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">-TAMRA-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">ABCG-T1</td>
                <td class="description-td">SEQ ID NO: 18</td>
                <td class="description-td">Template</td>
                <td class="description-td">5′ ATC AGC ATT CCA CGA TAT GGA TTT ACG GCA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">TCA GTT GCA GCA TAA TGA ATT TTT GGG A 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">MTHFR-F1</td>
                <td class="description-td">SEQ ID NO: 19</td>
                <td class="description-td">Primer</td>
                <td class="description-td">5′ GGA AGA ATG TGT CAG CCT CAA AG 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">MTHFR-R1</td>
                <td class="description-td">SEQ ID NO: 20</td>
                <td class="description-td">Primer</td>
                <td class="description-td">5′ CTG ACC TGA AGC ACT TGA AGG AG 3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">MTHFR-P1</td>
                <td class="description-td">SEQ ID NO: 21</td>
                <td class="description-td">Wt Probe</td>
                <td class="description-td">5′ TET-TGA AAT CGG CTC CCG CA-BHQ1-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">MTHFR-P2</td>
                <td class="description-td">SEQ ID NO: 22</td>
                <td class="description-td">Mut Probe</td>
                <td class="description-td">5′ FAM-TGA AAT CGA CTC CCG CAG A-BHQ1-3′</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00001" class="description-td" colspan="5">*: phosphorothioate internucleotide linkage </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>EXAMPLE 1</heading> <heading>Properties and use of TET Labeled FET Probes with Various Quenchers</heading> <p num="p-0103">Real Time amplifications and detection were performed in ABI PRISM 7700 (Applied Biosystems) using 30 μl reactions that contained 20 mM Tris-HCl (pH 8.3), 60 mM KCl, 5.3 mM MgCl<sub>2</sub>, 0.2 mM dATP, 0.2 mM dTTP, 0.2 mM dGTP, 0.2 mM dCTP, 0.5 μM of each primer, 0.2 μM FET probe, 4-fold series dilute of Jurkat cDNAs, either 0.6 units Taq Polymerase (Fisher) or a mix of Taq and Pwo DNA Polymerase (Roche Molecular Biochemicals) at unit ratio of 5 to 1.</p>
    <p num="p-0104">A 99 basepair segment of the human beta actin gene was amplified using primers ACT-F1 and ACT-R1 listed in Table 1. Thermal profile was 95° C. 15 sec; 50 cycles of 95° C. 15 sec, 60° C. 30 sec, 72° C. 45 sec. After amplification, fluorescent intensity at each well was measured by post PCR reading.</p>
    <p num="p-0105">Five types of TET labeled FET probes (ACT-P1, ACT-P2, ACT-P3, ACT-P4 and ACT-P5 as shown in Table 1) were tested in Real Time amplification. Post PCR reading result is shown in Table 2. The emission intensity of donor under condition with template is divided by the emission intensity of donor under condition without template to give +/− signal ratio, which indicates whether the probes are degraded or not. For TET labeled FET probes having TAMRA or Dabcyl as quencher, 3′→5′ exonuclease addition caused a dramatic decrease in +/− signal ratio due to cleavage of FET probes.</p>
    <p num="p-0106">
      <tables id="TABLE-US-00002" num="00002"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="322pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 2</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Fluorescent changes in various TET labeled FET probe during PCR</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="126pt" align="center"> </colspec> <colspec colname="3" colwidth="126pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">PCR using Taq pol.</td>
                <td class="description-td">PCR using Taq + Pwo pol.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="49pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <colspec colname="7" colwidth="49pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">donor/quencher</td>
                <td class="description-td">no template</td>
                <td class="description-td">Template</td>
                <td class="description-td">+/− signal ratio</td>
                <td class="description-td">no template</td>
                <td class="description-td">template</td>
                <td class="description-td">+/− signal ratio</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="49pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <colspec colname="7" colwidth="49pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TET/BHQ1</td>
                <td class="description-td">4500.0</td>
                <td class="description-td">20250.0</td>
                <td class="description-td">4.5</td>
                <td class="description-td">4500.0</td>
                <td class="description-td">20000.0</td>
                <td class="description-td">4.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">TET/eDQ</td>
                <td class="description-td">4000.0</td>
                <td class="description-td">20000.0</td>
                <td class="description-td">5.0</td>
                <td class="description-td">4000.0</td>
                <td class="description-td">20000.0</td>
                <td class="description-td">5.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">TET/Dabcyl</td>
                <td class="description-td">6000.0</td>
                <td class="description-td">19000.0</td>
                <td class="description-td">3.2</td>
                <td class="description-td">16000.0</td>
                <td class="description-td">24000.0</td>
                <td class="description-td">1.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">TET/TAMRA</td>
                <td class="description-td">3900.0</td>
                <td class="description-td">19000.0</td>
                <td class="description-td">4.9</td>
                <td class="description-td">12000.0</td>
                <td class="description-td">18000.0</td>
                <td class="description-td">1.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">TET/TAMRA S-oligo</td>
                <td class="description-td">4000.0</td>
                <td class="description-td">20000.0</td>
                <td class="description-td">5.0</td>
                <td class="description-td">3900.0</td>
                <td class="description-td">19000.0</td>
                <td class="description-td">4.9</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0107">Real Time amplification plot is shown in <figref idrefs="DRAWINGS">FIGS. 1A-E</figref>. TET labeled FET probes having either BHQ1 or Eclipse Dark Quencher at 3′-end are resistant to 3′→5′ exonuclease, and are therefore suitable for use in high fidelity PCR (FIGS. <b>1</b>A and <b>1</b>B). By contrast, TET labeled FET probes having either TAMRA or Dabcyl Quencher at 3′-end are degraded by 3′→5′ exonuclease, and no proper real time PCR results can be obtained (FIGS. <b>1</b>C and <b>1</b>D). The TET labeled FET probes having TAMRA Quencher at 3′-end can also be resistant to 3′→5′ exonuclease if three phosphorothioate internucleotide linkages at 3′-end are added (FIG. <b>1</b>E), which confirms the reliability of this whole experiment.</p>
    <heading>EXAMPLE 2</heading> <heading>Properties and use of FAM Labeled FET Probes with Various Quenchers</heading> <p num="p-0108">Real Time amplifications and detection were performed in ABI PRISM 7700 using 30 μl reactions that contained 20 mM Tris-HCl (pH 8.3), 60 mM KCl, 5.3 mM MgCl<sub>2</sub>, 0.2 mM dATP, 0.2 mM dTTP, 0.2 mM dGTP, 0.2 mM dCTP, 0.5 uM of each primers, 0.2 tM FET probe, various copies of a cDNA clone of human bcl-2 gene or Jurkat cDNAs, 0.6 units Taq DNA Polymerase (Fisher Scientific) or a mix of Taq and Pwo DNA Polymerase (Roche Molecular Biochemicals) at unit ratio of 10 to 1.</p>
    <p num="p-0109">A 190 basepair segment of the human bcl-2 gene was amplified using primers BCL-F1 and BCL-R1 listed in Table 1. Thermal profile was 95° C. 15 sec; 50 cycles of 95° C. 15 sec, 60° C. 30 sec, 72° C. 45 sec. After amplification, fluorescent intensity at each well was measured by post PCR reading.</p>
    <p num="p-0110">Five types of FAM-labeled FET probes (BCL-P1, BCL-P2, BCL-P3, BCL-P4 and BCL-P5 as shown in Table 1) were tested in Real Time amplification. Post PCR reading result is shown in Table 3. For FAM labeled FET probes having TAMRA or Dabcyl as quencher, 3′→5′ exonuclease addition caused a dramatic decrease in +/− signal ratio due to cleavage of FET probes.</p>
    <p num="p-0111">
      <tables id="TABLE-US-00003" num="00003"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="322pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 3</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Fluorescent changes in various FAM labeled FET probe during PCR</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="126pt" align="center"> </colspec> <colspec colname="3" colwidth="126pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">PCR using Taq pol.</td>
                <td class="description-td">POR using Taa-i-Pwo ~oI.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="49pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <colspec colname="7" colwidth="49pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">donor/quencher</td>
                <td class="description-td">no template</td>
                <td class="description-td">template</td>
                <td class="description-td">+/− signal ratio</td>
                <td class="description-td">no template</td>
                <td class="description-td">template</td>
                <td class="description-td">+/− signal ratio</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="49pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <colspec colname="7" colwidth="49pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">FAM/BHQ1</td>
                <td class="description-td">5000.0</td>
                <td class="description-td">20000.0</td>
                <td class="description-td">4.0</td>
                <td class="description-td">5000.0</td>
                <td class="description-td">20000.0</td>
                <td class="description-td">4.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">FAM/eDQ</td>
                <td class="description-td">5000.0</td>
                <td class="description-td">20000.0</td>
                <td class="description-td">4.0</td>
                <td class="description-td">5000.0</td>
                <td class="description-td">19000.0</td>
                <td class="description-td">3.8</td>
              </tr> <tr class="description-tr"> <td class="description-td">FAM/Dabcyl</td>
                <td class="description-td">7000.0</td>
                <td class="description-td">20000.0</td>
                <td class="description-td">2.9</td>
                <td class="description-td">18000.0</td>
                <td class="description-td">25000.0</td>
                <td class="description-td">1.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">FAM/TAMRA</td>
                <td class="description-td">5000.0</td>
                <td class="description-td">22500.0</td>
                <td class="description-td">4.5</td>
                <td class="description-td">20000.0</td>
                <td class="description-td">30000.0</td>
                <td class="description-td">1.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">FAM/TAMRA S-oligo</td>
                <td class="description-td">10000.0</td>
                <td class="description-td">30000.0</td>
                <td class="description-td">3.0</td>
                <td class="description-td">10000.0</td>
                <td class="description-td">29000.0</td>
                <td class="description-td">2.9</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0112">Representative Real Time amplification plot is shown in <figref idrefs="DRAWINGS">FIGS. 2A-2B</figref>. FAM labeled FET probes having either BHQ1 or Eclipse Dark Quencher at 3′-end are resistant to 3′→5′ exonuclease, and are therefore suitable for use in high fidelity PCR (FIG. <b>2</b>A). By contrast, FAM, labeled FET probes having either TAMRA or Dabcyl Quencher at 3′-end are degraded by 3′→5′ exonuclease, and no proper real time PCR results can be obtained (FIG. <b>2</b>B). Thus, similar results as Example 1 are repeated in a different system.</p>
    <heading>EXAMPLE 3</heading> <heading>Properties of FET Oligo as Primers</heading> <p num="p-0113">PCR amplifications were performed in ABI PRISM 7700 using 30 μl reactions that contained 10 mM Tris-HCl (pH 8.85), 25 mM KCl, 5 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 5 mM MgCl<sub>2</sub>, 0.2 mM dATP, 0.2 mM dTTP, 0.2 mM dGTP, 0.2 mM dCTP, 0.5 μM of each primer, 0.2 μM of each primer (ABCG-R1 and ABCG-P1 or ABCG-P2), 1 million copies of template (ABCG-T1) or just TE buffer in no template control (NTC), 0.6 units of Pwo DNA Polymerase (Roche Molecular Biochemicals). Thermal profile was 95° C. 10 sec; 40 cycles of 95° C. 15 sec, 60° C. 60 sec.</p>
    <p num="p-0114">Two types of FET primers (ABCG-P1 and ABCG-P2) and a template (ABCG-T1) as listed in Table 1 were tested in Real Time Amplification of human ABC transporter ABCG2. Multicomponent analysis is shown in FIG. <b>3</b>. Thus, the subject FET labeled nucleic acid detectors having a BHQ1 at 3′-end are resistant to 3′→5′ exonuclease activity when used as a primer. By contrast, the FET labeled nucleic acid detectors having a TAMRA at 3′-end are degraded by 3′→5′ exonuclease activity. Similar results were obtained using FET primers having mismatch at 3′-end.</p>
    <heading>EXAMPLE 4</heading> <heading>Allele Discrimination Under High Fidelity PCR</heading> <p num="p-0115">An 89 basepair segment of the human methylenetetrahydrofolate reductase (MTHFR) gene was amplified using primers MTHFR-F1 and MTHFR-R1 listed in Table 1. Real Time amplifications and detection were performed in ABI PRISM 7700 using 30 μl reactions that contained 20 mM Tris-HCl (pH 8.3), 60 mM KCl, 5.3 mM, MgCl<sub>2</sub>, 0.2 mM dATP, 0.2 mM dTTP, 0.2 mM dGTP, 0.2 mM dCTP, 0.1 μM of primer MTHFR-F1 and 1 μM of primer MTHFR-R1, 0.2 μM each of FET probes MTHFR-P1 and MTHFR-P2, templates containing varies number of MTHFR wild type or mutant or both as listed in table 4, a mix of Taq and Pwo DNA Polymerase (Roche Molecular Biochemicals) at unit ratio of 5 to 1. Thermal profile was 95° C. 15 sec; 50 cycles of 95° C. 15 sec, 55° C. 40 sec, 70° C. 40 sec. After amplification, fluorescent intensity at each well was measured by post PCR reading and analyzed by allele discrimination software in ABI PRISM 7700. Graphical result is shown in FIG. <b>4</b> and further summarized in Table 4.</p>
    <p num="p-0116">
      <tables id="TABLE-US-00004" num="00004"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 4</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Summary of template types and allele call by PRISM 7700</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="84pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Template types</td>
                <td class="description-td">Call by</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">(copy num)</td>
                <td class="description-td">PRISM 7700</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 10<sup>6</sup> </td>
                <td class="description-td">Allele 2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 10<sup>6</sup> </td>
                <td class="description-td">Allele 2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 10<sup>6</sup> </td>
                <td class="description-td">Allele 2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 10<sup>6</sup> </td>
                <td class="description-td">Allele 2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 10<sup>6</sup> </td>
                <td class="description-td">Allele 2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 10<sup>6</sup> </td>
                <td class="description-td">Allele 2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 10<sup>6</sup> </td>
                <td class="description-td">Allele 2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 10<sup>6</sup> </td>
                <td class="description-td">Allele 2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 2 × 10<sup>5</sup> </td>
                <td class="description-td">Allele ½</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 2 × 10<sup>5</sup> </td>
                <td class="description-td">Allele ½</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 2 × 10<sup>5</sup> </td>
                <td class="description-td">Allele ½</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 2 × 10<sup>5</sup> </td>
                <td class="description-td">Allele ½</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 4 × 10<sup>4</sup> </td>
                <td class="description-td">Allele ½</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 4 × 10<sup>4</sup> </td>
                <td class="description-td">Allele ½</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 4 × 10<sup>4</sup> </td>
                <td class="description-td">Allele ½</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 4 × 10<sup>4</sup> </td>
                <td class="description-td">Allele ½</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 8 × 10<sup>3</sup> </td>
                <td class="description-td">Allele 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 8 × 10<sup>3</sup> </td>
                <td class="description-td">Allele 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 8 × 10<sup>3</sup> </td>
                <td class="description-td">Allele 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 8 × 10<sup>3</sup> </td>
                <td class="description-td">Allele 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 8 × 10<sup>3</sup> </td>
                <td class="description-td">Allele 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 8 × 10<sup>3</sup> </td>
                <td class="description-td">Allele 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 8 × 10<sup>3</sup> </td>
                <td class="description-td">Allele 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mut: 8 × 10<sup>3</sup> </td>
                <td class="description-td">Allele 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>5</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Wt: 10<sup>6</sup> </td>
                <td class="description-td">Allele 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Wt: 10<sup>6</sup> </td>
                <td class="description-td">Allele 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Wt: 10<sup>6</sup> </td>
                <td class="description-td">Allele 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Wt: 10<sup>6</sup> </td>
                <td class="description-td">Allele 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">No template</td>
                <td class="description-td">No amp</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">No template</td>
                <td class="description-td">No amp</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">No template</td>
                <td class="description-td">No amp</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">No template</td>
                <td class="description-td">No amp</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="left" id="FOO-00002" class="description-td" colspan="3">Allele 1 = wild type (TET labeled probe) </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="left" id="FOO-00003" class="description-td" colspan="3">Allele 2 = mutant type (FAM labeled probe) </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0117">The call by ABI PRISM 7700 matches well with types of template used (Table 4). No template controls give no amplification signal. Thus, two different allele sequences can be detected and distinguished using FAM and TET labeled FET probes under highfidelity PCR.</p>
    <p num="p-0118">B. General</p>
    <p num="p-0119">The oligonucleotides shown in Table 5 were used for the following example. Standard DNA phosphoramidites, including 6-carboxy-fluorescein (6-FAM) phosphoramidite, 5′-Tetrachloro-Fluorescein (TET) phosphoramidite, Acridine (ACR) phosphoramidite and 6-carboxytetramethyl-rhodamine (TAMRA) CPG, 3′-Dabcyl CPG, were obtained from Glen Research. Black Hole Quenchers (BHQ1) CPG were obtained from Biosearch Technology. Eclipse Dark Quencher (eDQ) CPG was obtained from Epoch Bioscience. All primers were purified using Oligo Purification Cartridges (Biosearch Technology). Doubly labeled FET probes were synthesized using CPGs with various quenchers as indicated in Table 5 and with either 6′FAM-labeled or TET-labeled phosphoramidites at the 5′end. The doubly labeled FET probes were purified by preparative HPLC and PAGE using standard protocols. Phosphorothioate modification was prepared by standard procedure. Minor groove binder netropsin and distamycin A were purchased from Sigma.</p>
    <p num="p-0120">
      <tables id="TABLE-US-00005" num="00005"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="336pt" align="center"> </colspec> <colspec colname="2" colwidth="0pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="2" rowsep="1" class="description-td" colspan="2">TABLE 5</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">(continue)</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="105pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="189pt" align="left"> </colspec> <colspec colname="4" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Name</td>
                <td class="description-td">Type</td>
                <td class="description-td">Sequence</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="49pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <colspec colname="4" colwidth="189pt" align="left"> </colspec> <colspec colname="5" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">MTHFR-P1</td>
                <td class="description-td">SEQ ID NO: 21</td>
                <td class="description-td">Wt Probe</td>
                <td class="description-td">5′ TET-TGA AAT CGG CTC CCG CA-BHQ1-3′</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">MTHFR-P2</td>
                <td class="description-td">SEQ ID NO: 22</td>
                <td class="description-td">Mut Probe</td>
                <td class="description-td">5′ FAM-TGA AAT CGA CTC CCG CAG A-BHQ1-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">MTHFR-P13</td>
                <td class="description-td">SEQ ID NO: 23</td>
                <td class="description-td">Wt Probe</td>
                <td class="description-td">5′ TET-TGA AAT CGG CTC CCG C-ACR-BHQ1-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">MTHFR-P14</td>
                <td class="description-td">SEQ ID NO: 24</td>
                <td class="description-td">Wt Probe</td>
                <td class="description-td">5′ TET-ACR TGA AAT CGG CTC CCG C-BHQ1-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">MTHFR-P3</td>
                <td class="description-td">SEQ ID NO: 25</td>
                <td class="description-td">Mut Probe</td>
                <td class="description-td">5′ FAM-TGA AAT CGA CTC CCG CAG-ACR-BHQ1-3′</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>EXAMPLE 6</heading> <heading>Allele Discrimination using Acridine-Labeled FET Probes Under High Fidelity PCR</heading> <p num="p-0121">An 89 basepair segment of the human methylenetetrahydrofolate reductase (MTHFR) gene was amplified using primers MTHFR-F1 and MTHFR-R1 listed in Table 1. Real Time amplifications and detection were performed in ABI PRISM 7700 using 30 μl reactions that contained 20 mM Tris-HCl (pH 8.3), 60 mM KCl, 5.3 mM MgCl<sub>2</sub>, 0.2 mM dATP, 0.2 mM dTTP, 0.2 mM dGTP, 0.2 mM dCTP, 0.15 uM of reference dye ROX, 0.5 uM each of primers MTHFR-F1 and MTHFR-R1, 0.2 μM each of FET probes, templates containing varies number of MTHFR wild type or mutant or both as listed in Table 6, a mix of Taq and Pwo DNA Polymerase (Roche Molecular Biochemicals) at unit ratio of 5 to 1. Five types of FET probes (MTHFR-P1, MTHFR-P2, MTHFR-P3, MTHFR-P13 and MTHFR-P14 as shown in Table 5) were tested in Real Time amplification. Thermal profile was 95° C. 15 sec; 50 cycles of 95° C. 15 sec, 57° C. 40 sec, 70° C. 40 sec.</p>
    <p num="p-0122">Fluorescent intensity at each well was analyzed using real time data and software in ABI PRISM 7700 and summarized in Table 6. There was considerable cross-reactivity (or noise) between dyes FAM and TET (32% and 46% respectively) when normal FET probes were used. Such cross-reactivity was reduced to 2% and 4% when 3′-ACR labeled FET probes were used. However, 5′-ACR labeled FET probes did not show much difference to normal FET probes.</p>
    <p num="p-0123">
      <tables id="TABLE-US-00006" num="00006"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="357pt" align="center"> </colspec> <colspec colname="2" colwidth="0pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="2" rowsep="1" class="description-td" colspan="2">TABLE 6</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Summary of probe and template types and ΔRn changes</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="245pt" align="left"> </colspec> <colspec colname="1" colwidth="112pt" align="center"> </colspec> <colspec colname="2" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Template types (copy num)</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="245pt" align="left"> </colspec> <colspec colname="1" colwidth="35pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>6</sup> </td>
                <td class="description-td">Mut: 10<sup>6</sup> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="245pt" align="left"> </colspec> <colspec colname="1" colwidth="112pt" align="center"> </colspec> <colspec colname="2" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Fluorescent Intensity (ΔRn)</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="245pt" align="center"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Probe Types</td>
                <td class="description-td">TET ΔRn</td>
                <td class="description-td">FAM ΔRn</td>
                <td class="description-td">% noise</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="182pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">5′ FAM-TGA AAT CGA CTC CCG CAG -ACR-BHQ1-3</td>
                <td class="description-td">SEQ. ID No. 25</td>
                <td class="description-td">0.1</td>
                <td class="description-td">4.1</td>
                <td class="description-td">2%</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">5′ FAM-TGA AAT CGA CTC CCG CAG A -BHQ1-3′</td>
                <td class="description-td">SEQ. ID No. 22</td>
                <td class="description-td">1.4</td>
                <td class="description-td">4.4</td>
                <td class="description-td">32%</td>
              </tr> <tr class="description-tr"> <td class="description-td">5′ TET-TGA AAT CGG CTC CCG C -ACR-BHQ1-3′</td>
                <td class="description-td">SEQ. ID No. 23</td>
                <td class="description-td">4.5</td>
                <td class="description-td">0.2</td>
                <td class="description-td">4%</td>
              </tr> <tr class="description-tr"> <td class="description-td">5′ TET-ACR-TGA AAT CGG CTC CCG C -BHQ1-3′</td>
                <td class="description-td">SEQ. ID No. 24</td>
                <td class="description-td">2.7</td>
                <td class="description-td">0.8</td>
                <td class="description-td">30%</td>
              </tr> <tr class="description-tr"> <td class="description-td">5′ TET-TGA AAT CGG CTC CCG CA-BHQ1-3′</td>
                <td class="description-td">SEQ. ID No. 21</td>
                <td class="description-td">5.4</td>
                <td class="description-td">2.5</td>
                <td class="description-td">46%</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0124">Real Time amplifications and detection were further performed under the same conditions as above in ABI PRISM 7700 using 0.2 μM each of either normal or 3′-ACR-labeled FET probes as shown in Table 7 for allele discrimination. Real Time thermal profile was 95° C. 15 sec; 50 cycles of 95° C. 15 sec, 57° C. 40 sec, 70° C. 40 sec.</p>
    <p num="p-0125">Fluorescent intensity of each dye at each well was analyzed using real time data and software in ABI PRISM 7700 and summarized in Table 6. There was considerable cross-reactivity (or noise) between dyes FAM and TET when normal FET probes were used. Such cross-reactivity was reduced to ˜2% when 3′-ACR labeled FET probes were used. Because of much less cross-reactivity (or noise) between wt (TET) and mut (FAM) probes when using 3′-ACR-labeled FET probes, allelic analysis using 3′ACR-labeled probes was much closer to the theoretical prediction, therefore more accurate than using normal FET probe.</p>
    <p num="p-0126">
      <tables id="TABLE-US-00007" num="00007"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="441pt" align="center"> </colspec> <colspec colname="2" colwidth="0pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="2" rowsep="1" class="description-td" colspan="2">TABLE 7</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Summary of probe and template types and allele discrimination results</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="231pt" align="left"> </colspec> <colspec colname="1" colwidth="210pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Template types (copy num)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="offset" colwidth="231pt" align="left"> </colspec> <colspec colname="1" colwidth="42pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <colspec colname="6" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>6 </sup>+</td>
                <td class="description-td">wt: 10<sup>6 </sup>+</td>
                <td class="description-td">wt: 10<sup>6 </sup>+</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">wt: 10<sup>6 </sup> </td>
                <td class="description-td">mut: 10<sup>6 </sup> </td>
                <td class="description-td">mut: 10<sup>6</sup> </td>
                <td class="description-td">mut: 10<sup>5</sup> </td>
                <td class="description-td">mut: 10<sup>6</sup> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="231pt" align="center"> </colspec> <colspec colname="2" colwidth="210pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Probe Types</td>
                <td class="description-td">Fluorescent Intensity (ΔRn)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="189pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="8" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">5′ FAM-TGA AAT CGA CTC CCG CAG A-BHQ1-3′</td>
                <td class="description-td">FAM ΔRn</td>
                <td class="description-td">0.7</td>
                <td class="description-td">5.4</td>
                <td class="description-td">3.8</td>
                <td class="description-td">1.4</td>
                <td class="description-td">5.3</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 22) +</td>
              </tr> <tr class="description-tr"> <td class="description-td">5′ TET-TGA AAT CGG CTC CCG CA-BHQ1-3′</td>
                <td class="description-td">TET ΔRn</td>
                <td class="description-td">5.3</td>
                <td class="description-td">0.4</td>
                <td class="description-td">3.2</td>
                <td class="description-td">5.2</td>
                <td class="description-td">1.3</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 21)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="231pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">% of dominant allele ΔRn to total ΔRn</td>
                <td class="description-td">88%</td>
                <td class="description-td">93%</td>
                <td class="description-td">54%</td>
                <td class="description-td">79%</td>
                <td class="description-td">80%</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Theoretical %</td>
                <td class="description-td">100%</td>
                <td class="description-td">100%</td>
                <td class="description-td">50%</td>
                <td class="description-td">91%</td>
                <td class="description-td">91%</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="189pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="8" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">5′ FAM-TGA AAT CGA CTC CCG CAG-ACR-BHQ1-3′</td>
                <td class="description-td">FAM ΔRn</td>
                <td class="description-td">0.1</td>
                <td class="description-td">4.9</td>
                <td class="description-td">3.1</td>
                <td class="description-td">0.9</td>
                <td class="description-td">5.3</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO:25) +</td>
              </tr> <tr class="description-tr"> <td class="description-td">5′ TET-TGA AAT CGG CTC CCG C-ACR-BHQ1-3</td>
                <td class="description-td">TET ΔRn</td>
                <td class="description-td">5.2</td>
                <td class="description-td">0.1</td>
                <td class="description-td">2.9</td>
                <td class="description-td">5.6</td>
                <td class="description-td">0.8</td>
              </tr> <tr class="description-tr"> <td class="description-td">SEQ ID NO:23</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="231pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">% of dominant allele ΔRn to total ΔRn</td>
                <td class="description-td">98%</td>
                <td class="description-td">98%</td>
                <td class="description-td">52%</td>
                <td class="description-td">86%</td>
                <td class="description-td">87%</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Theoretical %</td>
                <td class="description-td">100%</td>
                <td class="description-td">100%</td>
                <td class="description-td">50%</td>
                <td class="description-td">91%</td>
                <td class="description-td">91%</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0127">The above results and discussion demonstrate that 3′-ACR labeled FET nucleic acid detectors suitable for use in high fidelity PCR for allele discrimination are provided by the subject invention. As such, the subject methods represent a significant contribution to the art.</p>
    <heading>Example 7</heading> <heading>Allele Discrimination Using Free MGB Under High Fidelity Real Time PCR</heading> <p num="p-0128">An 89 basepair segment of the human methylenetetrahydrofolate reductase (MTHFR) gene was amplified using primers MTHFR-F1 and MTHFR-R1 listed in Table 1. Real Time amplifications and detection were performed in ABI PRISM 7700 using 30 μl reactions that contained 20 mM Tris-HCl (pH 8.3), 60 mM KCl, 5.3 mM MgCl<sub>2</sub>, 0.2 mM dATP, 0.2 mM dTTP, 0.2 mM dGTP, 0.2 mM dCTP, 0.15 uM of reference dye ROX, 0.5 uM each of primers MTHFR-F1 and MTHFR-R1, FET probes, plus or minus templates containing varies number of MTHFR wild type or mutant or both as listed in table 7, a mix of Taq and Pwo DNA Polymerase (Roche Molecular Biochemicals) at unit ratio of 5 to 1. In this test 0.2 μM each of either normal or 3′-ACR-labeled FET probes as shown in Table 7, plus or minus 0.4 μM free netropsin or distamycin A, were used for allele discrimination. Thermal profile was 95° C. 15 sec; 50 cycles of 95° C. 15 sec, 63° C. 40 sec, 70° C. 40 sec., or 95° C. 15 sec; 50 cycles of 95° C. 15 sec, 57 ° C. 40 sec, 70° C. 40 sec.</p>
    <p num="p-0129">Similar results as Table 7 can be obtained at 63° C. anneal only when either free netropsin or distamycin A was included in PCR reaction mixture. There was no significant amplification when using normal PCR condition and 63° C. anneal. Similar results as Table 7 were also obtained at 57° C. anneal with or without netropsin or distamycin A. The netropsin or distamycin A acted primarily to enhance oligo's Tm, especially of AT rich sequences, so that shorter oligos/probes can be used in real time PCR to enhance allele discrimination. Such features shall help primers/probes design for AT rich sequences.</p>
    <p num="p-0130">The above results and discussion demonstrate that the inclusion of free netropsin or distamycin A in PCR mixture suitable for use in a variety of different high fidelity PCR applications are provided by the subject invention. As such, the subject methods represent a significant contribution to the art.</p>
    <p num="p-0131">The above results and discussion demonstrate that FET labeled nucleic acid detectors suitable for use in a variety of different high fidelity PCR applications are provide by the subject invention. As such, the subject methods represent a significant contribution to the art.</p>
    <p num="p-0132">All publications and patent applications cited in this specification are herein incorporated by reference as if each individual publication or patent application were specifically and individually indicated to be incorporated by reference. The citation of any publication is for its disclosure prior to the filing date and should not be construed as an admission that the present invention is not entitled to antedate such publication by virtue of prior invention.</p>
    <p num="p-0133">Although the foregoing invention has been described in some detail by way of illustration and example for purposes of clarity of understanding, it is readily apparent to those of ordinary skill in the art in light of the teachings of this invention that certain changes and modifications may be made thereto without departing from the spirit or scope of the appended claims.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4794073">US4794073</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 10, 1985</td><td class="patent-data-table-td patent-date-value">Dec 27, 1988</td><td class="patent-data-table-td ">Molecular Diagnostics, Inc.</td><td class="patent-data-table-td ">Detection of nucleic acid hybrids by prolonged chemiluminescence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5177196">US5177196</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 16, 1990</td><td class="patent-data-table-td patent-date-value">Jan 5, 1993</td><td class="patent-data-table-td ">Microprobe Corporation</td><td class="patent-data-table-td ">Used for diagnostic and chemotherapeutic purposes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5538848">US5538848</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 16, 1994</td><td class="patent-data-table-td patent-date-value">Jul 23, 1996</td><td class="patent-data-table-td ">Applied Biosystems Division, Perkin-Elmer Corp.</td><td class="patent-data-table-td ">Method for detecting nucleic acid amplification using self-quenching fluorescence probe</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5840493">US5840493</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 21, 1996</td><td class="patent-data-table-td patent-date-value">Nov 24, 1998</td><td class="patent-data-table-td ">Mitokor</td><td class="patent-data-table-td ">Diagnosis of late onset diabetes by detection of mutations in adenosine triphosphate synthase or transfer rna (lys) genes in a biological sample; labelled nucleotide sequences as detectors; accuracy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6072046">US6072046</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 26, 1998</td><td class="patent-data-table-td patent-date-value">Jun 6, 2000</td><td class="patent-data-table-td ">Epoch Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Diaziridinyl-aryl and bis-[di(chloroethyl)amino]-aryl oligonucleotide conjugates and reagents for making the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6117635">US6117635</a></td><td class="patent-data-table-td patent-date-value">Apr 11, 1997</td><td class="patent-data-table-td patent-date-value">Sep 12, 2000</td><td class="patent-data-table-td ">Intergen Company</td><td class="patent-data-table-td ">Nucleotide sequence with fluorescein or derivative attached to 5&#39; terminal and 4(4&#39;-dimethylaminophenylazo)benzoic acid attached to nucleotide 22; for ?closed tube? format which decreases possibility of carryover contamination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6248526">US6248526</a></td><td class="patent-data-table-td patent-date-value">Dec 14, 1998</td><td class="patent-data-table-td patent-date-value">Jun 19, 2001</td><td class="patent-data-table-td ">Aventis Behring, Gmbh</td><td class="patent-data-table-td ">Detecting target nucleotide sequences in sample; incubate sample with primers, expose sample to polymerase with accuracy activity, detect and/or measure release of signal</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6660845">US6660845</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 23, 1999</td><td class="patent-data-table-td patent-date-value">Dec 9, 2003</td><td class="patent-data-table-td ">Epoch Biosciences, Inc.</td><td class="patent-data-table-td ">Non-aggregating, non-quenching oligomers comprising nucleotide analogues; methods of synthesis and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1275735A1?cl=en">EP1275735A1</a></td><td class="patent-data-table-td patent-date-value">Jul 11, 2001</td><td class="patent-data-table-td patent-date-value">Jan 15, 2003</td><td class="patent-data-table-td ">F.Hoffmann-La Roche Ag</td><td class="patent-data-table-td ">Composition and method for hot start nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001042505A2?cl=en">WO2001042505A2</a></td><td class="patent-data-table-td patent-date-value">Dec 8, 2000</td><td class="patent-data-table-td patent-date-value">Jun 14, 2001</td><td class="patent-data-table-td ">Epoch Biosciences Inc</td><td class="patent-data-table-td ">Fluorescent quenching detection reagents and methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001086001A1?cl=en">WO2001086001A1</a></td><td class="patent-data-table-td patent-date-value">May 8, 2001</td><td class="patent-data-table-td patent-date-value">Nov 15, 2001</td><td class="patent-data-table-td ">Biosearch Technologies Inc</td><td class="patent-data-table-td ">Dark quenchers for donor-acceptor energy transfer</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chou, Q. et al. "<a href='http://scholar.google.com/scholar?q="Use+of+Dark-Quenched+FRET+probes+in+Real-Time+PCR"'>Use of Dark-Quenched FRET probes in Real-Time PCR</a>", American Biotechnology Laboratory,19(8):34 (2001).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">EPO Supplemental Search Report, Mailed Mar. 14, 2006.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kreuzer, K-A, et al. "<a href='http://scholar.google.com/scholar?q="Simultaneous+Absolute+Quantifcation+of+Target+and+Control+Templates+by+Real-Time+Flourescence+Reverse+Trancription-PCR+Using+4-%284%27Dimethylaminophenylazo%29benzoic+Acid+as+a+Dark+Quencher+Dye"'>Simultaneous Absolute Quantifcation of Target and Control Templates by Real-Time Flourescence Reverse Trancription-PCR Using 4-(4'Dimethylaminophenylazo)benzoic Acid as a Dark Quencher Dye</a>" Clinical Chemistry,47(3):488-490 (2001).</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7365176">US7365176</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 26, 2002</td><td class="patent-data-table-td patent-date-value">Apr 29, 2008</td><td class="patent-data-table-td ">Mayo Foundation For Medical Education And Research</td><td class="patent-data-table-td ">Detecting presence or absence of DNA viruses in biological sample; diagnosis, treatment and prevention of viral disease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7754453">US7754453</a></td><td class="patent-data-table-td patent-date-value">Aug 25, 2008</td><td class="patent-data-table-td patent-date-value">Jul 13, 2010</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Identifying polymerase chain reaction primer extension products using fluorescence energy transfer (FET); detecting allelic variation/restriction fragment length polymorphisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8088583">US8088583</a></td><td class="patent-data-table-td patent-date-value">May 24, 2010</td><td class="patent-data-table-td patent-date-value">Jan 3, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods of using FET labeled oligonucleotides that include a 3′-5′ exonuclease resistant quencher domain and compositions for practicing the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8410255">US8410255</a></td><td class="patent-data-table-td patent-date-value">Aug 6, 2010</td><td class="patent-data-table-td patent-date-value">Apr 2, 2013</td><td class="patent-data-table-td ">Biosearch Technologies, Inc.</td><td class="patent-data-table-td ">Dark quenchers for donor-acceptor energy transfer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8440399">US8440399</a></td><td class="patent-data-table-td patent-date-value">Aug 25, 2009</td><td class="patent-data-table-td patent-date-value">May 14, 2013</td><td class="patent-data-table-td ">Biosearch Technologies, Inc.</td><td class="patent-data-table-td ">Dark quenchers for donor-acceptor energy transfer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8466266">US8466266</a></td><td class="patent-data-table-td patent-date-value">Apr 1, 2009</td><td class="patent-data-table-td patent-date-value">Jun 18, 2013</td><td class="patent-data-table-td ">Biosearch Technologies, Inc.</td><td class="patent-data-table-td ">Stabilized nucleic acid dark quencher-fluorophore probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8633307">US8633307</a></td><td class="patent-data-table-td patent-date-value">Apr 22, 2010</td><td class="patent-data-table-td patent-date-value">Jan 21, 2014</td><td class="patent-data-table-td ">Biosearch Technologies, Inc.</td><td class="patent-data-table-td ">Dark quenchers for donor-acceptor energy transfer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8674094">US8674094</a></td><td class="patent-data-table-td patent-date-value">Jul 28, 2011</td><td class="patent-data-table-td patent-date-value">Mar 18, 2014</td><td class="patent-data-table-td ">Biosearch Technologies, Inc.</td><td class="patent-data-table-td ">Stabilized nucleic acid dark quencher-fluorophore probes</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024330">536/24.33</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006120">435/6.12</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006100">435/6.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0019340000">C12P19/34</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021020000">C07H21/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033533000">G01N33/533</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021040000">C07H21/04</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0006000000">A61K6/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K">A61K</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=8hN5BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/686">C12Q1/686</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68D4</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jul 9, 2014</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 7, 2013</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 11-12 AND 19 ARE CANCELLED.CLAIMS 1-10, 13-14 AND 18 ARE DETERMINED TO BE PATENTABLE AS AMENDED.CLAIMS 15-17, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">LIEN RELEASE;ASSIGNOR:BANK OF AMERICA, N.A.;REEL/FRAME:030182/0461</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100528</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">LIFE TECHNOLOGIES CORPORATION, CALIFORNIA</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 20, 2011</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20110803</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 23, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">INVITROGEN CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:BIOSOURCE INTERNATIONAL, INC.;REEL/FRAME:025563/0615</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20060823</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">LIFE TECHNOLOGIES CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:INVITROGEN CORPORATION;REEL/FRAME:025563/0495</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20081121</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 9, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 5, 2008</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BANK OF AMERICA, N.A., AS COLLATERAL AGENT, WASHIN</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:LIFE TECHNOLOGIES CORPORATION;REEL/FRAME:021975/0467</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20081121</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BANK OF AMERICA, N.A., AS COLLATERAL AGENT,WASHING</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:LIFE TECHNOLOGIES CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100302;REEL/FRAME:21975/467</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:LIFE TECHNOLOGIES CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100318;REEL/FRAME:21975/467</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:LIFE TECHNOLOGIES CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100329;REEL/FRAME:21975/467</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:LIFE TECHNOLOGIES CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100330;REEL/FRAME:21975/467</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:LIFE TECHNOLOGIES CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100413;REEL/FRAME:21975/467</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:LIFE TECHNOLOGIES CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100511;REEL/FRAME:21975/467</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:LIFE TECHNOLOGIES CORPORATION;REEL/FRAME:21975/467</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 9, 2002</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BIOSOURCE INTERNATIONAL, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:CHOU, QUIN;SPASIC, DRAGAN;REEL/FRAME:013561/0658;SIGNINGDATES FROM 20021122 TO 20021125</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2J251GblvTNRZMNGHSJKJwtv8Qhw\u0026id=8hN5BAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U3hn0rmamv5xQiBxvgal_WLgbAcOw\u0026id=8hN5BAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U1LOy1eXIbb3RxX047ooQ6-TSLVjg","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Identifying_polymerase_chain_reaction_pr.pdf?id=8hN5BAABERAJ\u0026output=pdf\u0026sig=ACfU3U0W795t3v-UNUsK_zi4XV1UZp4x2Q"},"sample_url":"http://www.google.com/patents/reader?id=8hN5BAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>