<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7169560 - Short cycle methods for sequencing polynucleotides - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Short cycle methods for sequencing polynucleotides"><meta name="DC.contributor" content="Stanley N Lapidus" scheme="inventor"><meta name="DC.contributor" content="Philip Richard Buzby" scheme="inventor"><meta name="DC.contributor" content="Timothy Harris" scheme="inventor"><meta name="DC.contributor" content="Helicos Biosciences Corporation" scheme="assignee"><meta name="DC.date" content="2004-5-24" scheme="dateSubmitted"><meta name="DC.description" content="The invention provides methods for sequencing a polynucleotide comprising stopping an extension cycle in a sequence by synthesis reaction before the reaction has run to near or full completion."><meta name="DC.date" content="2007-1-30" scheme="issued"><meta name="DC.relation" content="US:3996345" scheme="references"><meta name="DC.relation" content="US:4119368" scheme="references"><meta name="DC.relation" content="US:4153855" scheme="references"><meta name="DC.relation" content="US:4344064" scheme="references"><meta name="DC.relation" content="US:4351760" scheme="references"><meta name="DC.relation" content="US:4683195" scheme="references"><meta name="DC.relation" content="US:4683202" scheme="references"><meta name="DC.relation" content="US:4707237" scheme="references"><meta name="DC.relation" content="US:4711955" scheme="references"><meta name="DC.relation" content="US:4725677" scheme="references"><meta name="DC.relation" content="US:4739044" scheme="references"><meta name="DC.relation" content="US:4757141" scheme="references"><meta name="DC.relation" content="US:4793705" scheme="references"><meta name="DC.relation" content="US:4811218" scheme="references"><meta name="DC.relation" content="US:4863849" scheme="references"><meta name="DC.relation" content="US:4865968" scheme="references"><meta name="DC.relation" content="US:4889818" scheme="references"><meta name="DC.relation" content="US:4942124" scheme="references"><meta name="DC.relation" content="US:4962037" scheme="references"><meta name="DC.relation" content="US:4971903" scheme="references"><meta name="DC.relation" content="US:4979824" scheme="references"><meta name="DC.relation" content="US:4994368" scheme="references"><meta name="DC.relation" content="US:4994372" scheme="references"><meta name="DC.relation" content="US:4994373" scheme="references"><meta name="DC.relation" content="US:5085562" scheme="references"><meta name="DC.relation" content="US:5091652" scheme="references"><meta name="DC.relation" content="US:5096388" scheme="references"><meta name="DC.relation" content="US:5096554" scheme="references"><meta name="DC.relation" content="US:5108892" scheme="references"><meta name="DC.relation" content="US:5112736" scheme="references"><meta name="DC.relation" content="US:5143854" scheme="references"><meta name="DC.relation" content="US:5167784" scheme="references"><meta name="DC.relation" content="US:5171132" scheme="references"><meta name="DC.relation" content="US:5198540" scheme="references"><meta name="DC.relation" content="US:5209834" scheme="references"><meta name="DC.relation" content="US:5224843" scheme="references"><meta name="DC.relation" content="US:5242797" scheme="references"><meta name="DC.relation" content="US:5259737" scheme="references"><meta name="DC.relation" content="US:5260433" scheme="references"><meta name="DC.relation" content="US:5265327" scheme="references"><meta name="DC.relation" content="US:5267152" scheme="references"><meta name="DC.relation" content="US:5302509" scheme="references"><meta name="DC.relation" content="US:5304487" scheme="references"><meta name="DC.relation" content="US:5306403" scheme="references"><meta name="DC.relation" content="US:5336062" scheme="references"><meta name="DC.relation" content="US:5360523" scheme="references"><meta name="DC.relation" content="US:5375979" scheme="references"><meta name="DC.relation" content="US:5376252" scheme="references"><meta name="DC.relation" content="US:5403709" scheme="references"><meta name="DC.relation" content="US:5405747" scheme="references"><meta name="DC.relation" content="US:5405783" scheme="references"><meta name="DC.relation" content="US:5409811" scheme="references"><meta name="DC.relation" content="US:5424186" scheme="references"><meta name="DC.relation" content="US:5449767" scheme="references"><meta name="DC.relation" content="US:5476928" scheme="references"><meta name="DC.relation" content="US:5484701" scheme="references"><meta name="DC.relation" content="US:5492806" scheme="references"><meta name="DC.relation" content="US:5514256" scheme="references"><meta name="DC.relation" content="US:5518900" scheme="references"><meta name="DC.relation" content="US:5525464" scheme="references"><meta name="DC.relation" content="US:5529465" scheme="references"><meta name="DC.relation" content="US:5534125" scheme="references"><meta name="DC.relation" content="US:5547839" scheme="references"><meta name="DC.relation" content="US:5556790" scheme="references"><meta name="DC.relation" content="US:5558991" scheme="references"><meta name="DC.relation" content="US:5599695" scheme="references"><meta name="DC.relation" content="US:5610287" scheme="references"><meta name="DC.relation" content="US:5631734" scheme="references"><meta name="DC.relation" content="US:5632957" scheme="references"><meta name="DC.relation" content="US:5654149" scheme="references"><meta name="DC.relation" content="US:5659171" scheme="references"><meta name="DC.relation" content="US:5670346" scheme="references"><meta name="DC.relation" content="US:5674716" scheme="references"><meta name="DC.relation" content="US:5675155" scheme="references"><meta name="DC.relation" content="US:5688648" scheme="references"><meta name="DC.relation" content="US:5695940" scheme="references"><meta name="DC.relation" content="US:5705018" scheme="references"><meta name="DC.relation" content="US:5707506" scheme="references"><meta name="DC.relation" content="US:5710628" scheme="references"><meta name="DC.relation" content="US:5712476" scheme="references"><meta name="DC.relation" content="US:5733729" scheme="references"><meta name="DC.relation" content="US:5741640" scheme="references"><meta name="DC.relation" content="US:5741644" scheme="references"><meta name="DC.relation" content="US:5744305" scheme="references"><meta name="DC.relation" content="US:5744312" scheme="references"><meta name="DC.relation" content="US:5750341" scheme="references"><meta name="DC.relation" content="US:5753788" scheme="references"><meta name="DC.relation" content="US:5755943" scheme="references"><meta name="DC.relation" content="US:5756285" scheme="references"><meta name="DC.relation" content="US:5759014" scheme="references"><meta name="DC.relation" content="US:5759374" scheme="references"><meta name="DC.relation" content="US:5762876" scheme="references"><meta name="DC.relation" content="US:5776767" scheme="references"><meta name="DC.relation" content="US:5776782" scheme="references"><meta name="DC.relation" content="US:5789168" scheme="references"><meta name="DC.relation" content="US:5795722" scheme="references"><meta name="DC.relation" content="US:5795782" scheme="references"><meta name="DC.relation" content="US:5807679" scheme="references"><meta name="DC.relation" content="US:6383749" scheme="references"><meta name="DC.relation" content="US:RE34069" scheme="references"><meta name="citation_reference" content="Adam et al., &quot;Individual genomes targeted in sequencing revolution&quot;, Nature, vol. 411, p. 402 (May 2001)."><meta name="citation_reference" content="Agrawal, S. et al., &quot;Site Specific Functionalization of Oligodeoxynucleotides for Non-Radioactive Labelling&quot;, Tetrahedron Letters, vol. 31, No. 11, pp. 1543-1546 (1990)."><meta name="citation_reference" content="Ambrose, W. et al., &quot;Single Molecule Detection With Total Internal Reflection Excitation: Comparing Signal-to-Background and Total Signals in Different Geometries&quot;, Cytometry, vol. 36, pp. 224-231 (1999)."><meta name="citation_reference" content="Amit, B. et al., &quot;Photosensitive Protecting Groups of Amino Sugars and Their Use in Glycoside Synthesis . . . Derivatives&quot;, 1. Org. Chem., 39(2):192-6 (1974)."><meta name="citation_reference" content="Arndt-Jovin, D. et al., &quot;Immunofluorescence Localization ofZ-DNA in Chromosomes: Quantitation by Scanning Microphotometry and Computer-assisted Image Analysis&quot;, I. Cell Bio!o?JI, vol. 101, pp. 1422-1433, (Oct. 1985)."><meta name="citation_reference" content="Augustin, M.A., W. Ankenbauer, and B. Angerer, &quot;Progress towards single-molecule sequencing: enzymatic synthesis of nucleotide-specifically labeled DNA.&quot; Journal of Biotechnology, 2001. 8(13): p. 289."><meta name="citation_reference" content="Axelrod, D. et al., &quot;Total internal reflection fluorescent microscopy&quot;, J Microscopy, vol. 129, pp. 19-28, (1983)."><meta name="citation_reference" content="Axelrod, D., &quot;Cell-Substrate Contacts Illuminated by Total Internal Reflection Fluorescence&quot;, I. Cell Bio!o?JI, vol. 89, pp. 141-145, (April 1981)."><meta name="citation_reference" content="Bai, X., et al., &quot;Photocleavage of a 2-nitrobenzyl linker bridging a fluorophore to the 5&#39; end of DNA.&quot; Proc Natl Acad Sci USA, 2003, vol. 100(2). p. 409-13."><meta name="citation_reference" content="Basche, T. et al., &quot;Single Molecule Optical Detection, Imaging and Spectroscopy&quot;, Chs. 2 and 3, Weinheim:VCM, Germany (1997)."><meta name="citation_reference" content="Beaucage, S. et al., &quot;Advances in the Synthesis of Oligonucleotides by the Phosphoramidite Approach&quot; Tetrahedron, 48:2223-2311 (1992)."><meta name="citation_reference" content="Beese, L. et al., &quot;Structure of DNA Polymerase I Klenow Fragment Bound to Duplex DNA&quot;, Science, 260:352-355 (1993)."><meta name="citation_reference" content="Bennett et. al., &quot;Solexa Sequencing chemistry can be applied to different platforms which will have common elements in detection and data processing.&quot; Pharmacogenomics (2004) 5(4)."><meta name="citation_reference" content="Biesalski et al., &quot;Preparation and Characterization of a Polyelectrolyte Monolayer Covalently Attached to a Planar Solid Surface.&quot; Macromolecules 111, 32, 2309-2316. Article was published on the web Mar. 10, 1999."><meta name="citation_reference" content="Black, D.L., Protein diversity from alternating splicing: A challenge for bioinformatics and post genome biology. Cell, 2000. 103(3): p. 367-370."><meta name="citation_reference" content="Blattner, F.R., et al., &quot;The Complete genome sequence of Escherichia coli K-12.&quot; Science, 1997.277(5331):p. 1453-74."><meta name="citation_reference" content="Boles et. al., &quot;High-Resolution Mapping of Carcinogen Binding Sites on DNA&quot; 1986, 25, 3039-3043."><meta name="citation_reference" content="Brackmann et. al, &quot;Optimal Enzymes for Single-Molecule Sequencing&quot; 18, D-04103."><meta name="citation_reference" content="Brakmann, S. and P. Nieckchen, &quot;The large fragment of Escherichia coli DNA polymerase I can synthesize DNA exclusively from fluorescently labeled nucleotides.&quot; Chembiochem, 2001. 2(10):p. 773-777."><meta name="citation_reference" content="Braslavsky, I. et al., &quot;Objective-type dark-field illumination for scattering from microbeads&quot;, Applied Optics, vol. 40, No. 31, pp. 5650-5657, (Nov. 2001)."><meta name="citation_reference" content="Braslavsky, I. et al., &quot;Sequence information can be obtained from single DNA molecules&quot;, PNAS, vol. 100, No. 7, pp. 3960-3964 (Apr. 2003)."><meta name="citation_reference" content="Braslavsky, I. et al., &quot;Single Molecule Measurements of DNA Polymerase Activity: A Step Towards Single Molecule Sequencing&quot;, Biophys. I. Abstracts, p. 507A (2002)."><meta name="citation_reference" content="Brechtel, R. et al., &quot;Control of the electro osmotic flow by metal-salt-containing buffers&quot;, J Chromatoraphy A, vol. 716, pp. 97-105, (1995)."><meta name="citation_reference" content="Bridgman, A. et al., &quot;An improved method for the synthesis ofmercurated Dutp. Enzymatic synthesis of Hg-labelled DNA of high molecular weight suitable for use in an image based DNA sequencing strategy&quot;, DNA Seq., vol. 6, No. 4, pp. 199-209 (1996)."><meta name="citation_reference" content="Bryzek, J. etal., &quot;Micromachines on the march&quot;, IEEE Spectrum, vol. 31, No. 5, pp. 20-31, (1994)."><meta name="citation_reference" content="Buchaillot, L. et al., &quot;Silicon Nitride Thin Films Young&#39;s Modulus Determination by an Optical Non Destructive Method&quot;, Jpn. J. Appl. Phys., vol. 36, pp. L794-L797, (Jun. 1997)."><meta name="citation_reference" content="Burghardt, T. et al., &quot;Total Internal Reflection Fluorescence Study of Energy Transfer in Surface-Adsorbed and Dissolved Bovine Serum Albumin&quot;, Biochemistry, vol. 22, pp. 979-985 (1983)."><meta name="citation_reference" content="Butler, D. et al. &quot;Draft data leave geneticists with a mountain still to climb&quot;, Nature, vol. 405, Issue 6782, pp. 984-985 (May 2000)."><meta name="citation_reference" content="Canard, B., B. Cardona, .and R.S. Sarfati, &quot;Catalytic editing properties of DNA polymerases.&quot; Proc Natl Acad Sci USA, 1995. 92(24): p. 10859-63."><meta name="citation_reference" content="Canard, et al., &quot;DNA polymerase fluorescent substrates with reversible 3&#39;-tags&quot;. Gene, 1994. 148(1): p. 1-6."><meta name="citation_reference" content="Cheng et al., &quot;High-speed DNA sequence analysis,&quot; Prog. in Biochem. and Biophys., vol. 22, pp. 223-227 (1995)."><meta name="citation_reference" content="Chicurel, M., &quot;Faster, better, cheaper genotyping&quot;, Nature, vol. 412, Issue 6847, pp. 580-582, (Aug. 2001)."><meta name="citation_reference" content="Chidgeavadze, Z. et al., &quot;3&#39;-Fluro-2&#39;,3&#39;-dideoxyribonucleoside 5&#39;-triphosphates: terminators of DNA synthesis&quot;, FEBS Letters, 183(2):275-278 (1985)."><meta name="citation_reference" content="Chiu, D. et al., &quot;Patterned deposition of cells and proteins onto surfaces by using three-dimensional microfluidic systems,&quot; PNAS, vol. 97, No. 6, pp. 2408-2413 (2000)."><meta name="citation_reference" content="Chou et al., &quot;A microfabricated device for sizing and sorting DNA molecules&quot;, Applied Sciences, Biophysics: Chou et al., Proc. Natl. Acad. Sci. USA 96, pp. 11-13 (1999)."><meta name="citation_reference" content="Chou et al., &quot;A Microfabricated Rotary Pump&quot;. Biomedical Microdevices. vol. 3: p. 323 (2001)."><meta name="citation_reference" content="Close, D. et al., &quot;Ultraviolet Photobleaching of Free Radicals Created in y-Irradiated Amino Acids&quot;, Radiation Research, vol. 53, pp. 349-357 (1973)."><meta name="citation_reference" content="Cooper, J. et al., &quot;Analysis of Fluorescence Energy Transfer in Duplex and Branched DNA Molecules&quot;, Biochemistry, vol. 29, pp. 9261-9268 (1990)."><meta name="citation_reference" content="Crocker, J.C. and D.G. Grier, &quot;Methods of digital video microscopy for colloidal studies.&quot; Journal of Colloid and Interface Science, 1996. 179(1): p. 298-310."><meta name="citation_reference" content="Dapprich, J., &quot;Single-molecule DNA digestion by lambda-exonuclease.&quot; Cytometry, 1999.36(3): p. 163-168."><meta name="citation_reference" content="Debenham, J.S., et al., &quot;Two New Orthogonal Amine-Protecting Groups that can be Cleaved under Mild or Neutral Conditions.&quot; Journal of the American Chemical Society, 1995. 117(11): p. 3302-3."><meta name="citation_reference" content="Decher et al., &quot;Buildup of ultrathin multilayer films by a self-assembly process,&quot; Thin Solid Films, 210/211:831-835 (1992)."><meta name="citation_reference" content="Decher G.;et al., &quot;Fuzzy nanoassemblies : Toward layered polymeric multicomposites.&quot; Science, 1997.277(5330): p. 1232-1237."><meta name="citation_reference" content="Decher, G. et al. &quot;Buildup of ultrathin multiplayer files by a self-assembly process: III. Consecutively alternating absorption of anionic and cationic polyelectrolytes on charged surfaces&quot;, Thin Solid Films, 210:831-835 (1992)."><meta name="citation_reference" content="Delamarche, E. et al., &quot;Patterned Delivery of Immunoglobulins to Surfaces Using Microfluidic Networks&quot;, Science 276:779-781 (1997)."><meta name="citation_reference" content="Dickson et al., &quot;Simultaneous Imaging of Individual Molecules aligned both parallel and perpendicular to the optic axis&quot; vol. 81, No. 24, 1998."><meta name="citation_reference" content="Doktycz, M. et al., &quot;Genosensors and Model Hybridization Studies&quot;, Automation Technologies for Genome Characterization, Ch. 10 T. Beugelsdijk (Ed), John Wiley &amp; Sons, Inc. (1997), pp. 205-225."><meta name="citation_reference" content="Doublie, S. et al., &quot;Crystal structure of a bacteriophage T7 DNA replication complex at 2.2 A resolution&quot;, Nature; vol. 391, pp. 251-258 (Jan. 1998)."><meta name="citation_reference" content="Driscoll et al., &quot;Atomic-Scale Imaging of DNA Using Scanning Tunneling Microscopy.&quot; Nature, 1990.346(6281): p. 294-296."><meta name="citation_reference" content="Drmanac, R. et al., &quot;Sequencing by hybridization: Towards an automated sequencing of one million M13 clones arrayed on membranes&quot;, Electrophoresis, 13:566-573 (1992)."><meta name="citation_reference" content="Duffy et al., &quot;Patterning Electroluminescence Materials with Feature Sizes as Small as 5 um Using Elastomeric Membrains as Masks for Dry Lift-Off,&quot; Advanced Materials vol. 11, No. 7, pp. 546-552 (1999)."><meta name="citation_reference" content="Duffy et al., &quot;Rapid prototyping of microfludic switches in poly(dimethyl siloxane) and their acuation by electroosmotic flow,&quot; J. Micromech. Microeng., (1999) vol., 9, pp. 211-217."><meta name="citation_reference" content="Duffy et al., &quot;Rapid Prototyping of Microfluidic Systems in Poly(dimethylsiloxane)&quot;, Analytical Chemistry, vol. 70, No. 23, pp. 4974-4984 (1998)."><meta name="citation_reference" content="Effenhauser et al., &quot;Integrated capillary electrophoresis on Flexible silicone microdevices: Analysis of DNA restriction fragments and detection of single DNA molecules on microchips,&quot; Anal. Chem., vol. 69, pp. 3451-3457 (1997)."><meta name="citation_reference" content="Effenhauser et al., &quot;Integrated chip-based capillary electrophoresis,&quot; Electrophoresis, vol. 18, pp. 2203-2213 (1997)."><meta name="citation_reference" content="Eigen, M. et al., &quot;Sorting single molecules: Application to diagnostics and evolutionary biotechnology&quot;, PNAS, vol. 91, pp. 5740-5747, (Jun. 1994)."><meta name="citation_reference" content="Evangelista, R.A., et al. &quot;Characterization of fluorescent nucleoside triphosphates by capillary electrophoresis with laser-induced fluorescence detection: action of alkaline phosphatase and DNA polymerase.&quot; Anal Biochem, 1996.235(1): p. 89-97."><meta name="citation_reference" content="Fahrenberg et al., &quot;A microvalve system fabricated by thermoplastic molding,&quot; J. Micromech. Microeng., vol. 5, pp. 169-171(1995)."><meta name="citation_reference" content="Felicia, Y. et al., &quot;Synthesis and Properties of Adenosine-5&#39;-triphosphoro-y-l-(5-sulfonic acid)naphthyl Ethylamidate: A Fluorescent Nucleotide Substrate for DNA-Dependent RNA Polymerase from E. colt&quot;, Arch. Biochem. Biophys., 246(2):564-571 (1986)."><meta name="citation_reference" content="Ferguson, et al., &quot;A fiber-optic DNA biosensor microarray for the analysis of gene expression,&quot; Nature Biotechnology, vol. 14, pp. 1681-1684 (1996)."><meta name="citation_reference" content="Firtz, I. et al., &quot;Electronic detection of DNA by its intrinsic molecular charge&quot;, PNAS, vol. 99, No. 22, pp. 14142-14146 (Oct. 2002)."><meta name="citation_reference" content="Forster, T., &quot;Delocalized Excitation and Excitation Transfer&quot;, Modem Quantum Chem., Istanbul Lectures, Part TII, pp. 93-137, Academic Press, New York (1965)."><meta name="citation_reference" content="Fu e al., &quot;An integrated microfabricated cell sorter&quot;. Anal Cherm, 2002. 74(11): p. 451-7."><meta name="citation_reference" content="Fu et al., &quot;A microfabricated fluorescence-activated cell sorter,&quot; Nature Biotechnology, vol. 17, pp. 1109-1111 (1999)."><meta name="citation_reference" content="Funatsu, T. et al., &quot;Imaging of single fluorescent molecules and individual ATP turnovers by single myosin molecules in aqueous solution&quot;, Nature, vol. 374, pp. 555-559 (Apr. 1995)."><meta name="citation_reference" content="Garcia, A., &quot;Detennination of Ion Penneability by Fluorescence Quenching&quot;, Meth. in Enzymolgy, 207:501-511 (1992)."><meta name="citation_reference" content="Gardner et al., &quot;Comparative kinetics of nucleotide analog incorporation by Vent DNA polymerase,&quot; J. Biol. Chem., 279, No. 12, Mar. 19, 2004, 11834-11842."><meta name="citation_reference" content="Giller et al., &quot;Incorporation of reporter molecule-labeled nucleotides by DNA polymerases. I. Chemical synthesis of various reporter group-labeled 2&#39;deoxyribonucleoside-5&#39;-triphosphates,&quot; Nucleic Acids Res., 31, No. 10, 2003, 2630-2635."><meta name="citation_reference" content="Giusti, W. et al., &quot;Synthesis and Characterization of 5&#39; -Fluorescent-dye-labeled Oligonucleotides&quot;, PCR Methods and Applications, 2:223-227 (1993)."><meta name="citation_reference" content="Goll et al., &quot;Microvalves with bistable buckled polymer diaphragms,&quot; J. Micromech. Microeng., vol. 6., pp. 77-79 (1996)."><meta name="citation_reference" content="Goodwin, P.M., et al., &quot;Application of single molecule detection to DNA sequencing.&quot; Nucleosides &amp; Nucleotides, 1997. 16(5-6): p. 543-550."><meta name="citation_reference" content="Gravesen et al., &quot;Microfluidics-a review&quot;, J. Micromech. Microeng. 3IOP Publishing Ltd., pp. 168-192 (1993).vol. 3."><meta name="citation_reference" content="Greene, T.W. and P.G.M. Wuts, &quot;Protective Groups in Organic Synthesis.&quot; John Wiley &amp; Sons, Inc.: New York, 1999 3rd Ed."><meta name="citation_reference" content="Gueroui, Z., et al., &quot;Observation by fluorescence microscopy of transcription on single combed DNA.&quot; Proceedings of the National Academy of Sciences of the United States of America, 2002. 99(9): p. 6005-6010."><meta name="citation_reference" content="Guilbault, G., &quot;Practical Fluorescence-Theory, Methods and Techniques,&quot; Chapters 1 and 3, and pp. 521-524, Marcel Dekker, Inc., New York (1973)."><meta name="citation_reference" content="Gyllenstein, U. et al., &quot;Generation of single-stranded DNA by the polymerase chain reaction and its application to direct sequencing of the HLA-DQA locus&quot;, PNAS 85:7652-56."><meta name="citation_reference" content="Ha et al., &quot;Single-molecule fluorescence spectroscopy of enzyme conformational dynamics and cleavage mechanism.&quot; Proceedings of the National Academy of Sciences of the United States of America, 1999.96(3): p. 893-898."><meta name="citation_reference" content="Ha, &quot;Single molecule dynamics studied by polarization modulation,&quot; Phys. Rev. Lett., 77, No. 19, Nov. 4, 1996, 39793982."><meta name="citation_reference" content="Ha, &quot;Single molecule spectroscopy with automated positioning,&quot; Appl. Phys. Lett. 70, No. 6, Feb. 10, 1997, 782-784."><meta name="citation_reference" content="Ha, &quot;Single-molecule fluorescence methods for the study of nucleic acids,&quot; Current Opinion in Struct Bio, 11, 2001, 287-292."><meta name="citation_reference" content="Ha, T., &quot;Single-molecue fluorescence resonance energy transfer.&quot; Methods, 2001. 25(1): p. 78-86."><meta name="citation_reference" content="Hanna, M. et al., &quot;Synthesis and characterization of a new photocrosslinking CTP analog and its use in photoaffmity labeling E. coli and T7 RNA polymerases&quot;, Nucleic Acids Res., 21(9):2073-2079 (1993)."><meta name="citation_reference" content="Hansen, C.J , et al., &quot;A robust and scalable microfluidic metering method that allows Protein crystal growth by free interface diffusion&quot;. Proc Natl Acad Sci U S A, 2002. 99 (26): p. 16531-6."><meta name="citation_reference" content="Harding et al., &quot;Single-molecule detection as an approach to rapid DNA sequencing,&quot; Trends in Biotechnology, vol. 10, 1992."><meta name="citation_reference" content="Harris, J.M., &quot;Introducton to Biochemical and biomedical applications of poly(ethylene glycol).&quot; poly(ethylene glycol) chemistry, Harris, J. M., Ed.; Plenum Press: New York, 1992: p. pp. 1-14."><meta name="citation_reference" content="Harrison et al., &quot;Micromachining a miniaturized capillary electrophoresis-based chemical analysis system on a chip,&quot; Science, vol. 261, pp. 895-897 (1993)."><meta name="citation_reference" content="Hasan, A. et al., &quot;Photolabile Protecting Groups for Nucleosides: Synthesis and Photodeprotection Rates&quot;, Tetrahedron, 53(12):4247-4264 (1997)."><meta name="citation_reference" content="Hornbeck, L. et al., &quot;Bistable Defonnable Mirror Device&quot;, 1988 Techllical Digest Series, vol. 8, Optical Society of America, pp. 107-110, (Jun. 1988)."><meta name="citation_reference" content="Hosokawa et al., &quot;Handling of Picoliter liquid samples in a poly(dimethylsiloxane)-based microfluidic device,&quot; Anal. Chem., vol. 71, No. 20, pp. 4781-4785 (1999). http://biospace.intota.com (query &quot;elastomer&quot;) cited by Examiner E Quan, GAU 1743 in related cases). http://www.chemfinder.com (cited by Examiner E. Quan, GAU 1743, in related case)."><meta name="citation_reference" content="Houseal, T. et al., &quot;Real-time imaging of single DNA molecules with fluorescence microscopy&quot;, Biophys. I., vol. 56, pp. 507-516 (Sep. 1989)."><meta name="citation_reference" content="Howorka, et al., &quot;Sequence-specific detection of individual DNA strands using engineered nanopores.&quot; Nature Biotechnology, 2001.19(7): p. 636-639."><meta name="citation_reference" content="Hubner et al., &quot;Direct observation of the triplet lifetime quenching of single dye molecules by molecular oxygen,&quot; J. Chem. Physics, 115, No. 21, Dec. 1, 2001, 9619-9622."><meta name="citation_reference" content="Hultman, T. et al., &quot;Bidirectional Solid-Phase Sequencing ofln Vitro-Amplified Plasmid DNA&quot;, BioTechniques, vol. 10, No. 1, pp. 84-93 (1991)."><meta name="citation_reference" content="Hyman, E., &quot;A New Method of Sequencing DNA&quot;, Anal. Biochem., 174:423-436 (1988)."><meta name="citation_reference" content="Ishii et al., &quot;Fluorescence resonance energy transfer between single fluorophores attached to a coiled-coil protein in aqueous solution,&quot; Chemical Physics, 247, 1999, 163-173."><meta name="citation_reference" content="Ishikawa, M. et al., &quot;Single-Molecule Detection by Laser-Induced Fluorescence Technique with a Position-Sensitive Photon-Counting Apparatus&quot;, lpn. 1. Appl. Phys., vol. 33, Part 1, No. 3A, pp. 1571-1576 (1994)."><meta name="citation_reference" content="Jacobs et al., &quot;Combinatorial chemistry-applications oflight-directed chemical synthesis&quot;, TIBTech, vol. 12, pp. 19-26 (Jan. 1994)."><meta name="citation_reference" content="Jacobson, et al., &quot;High-speed separations on a microchip,&quot; Anal. Chem., vol. 66, No. 7, pp. 1114-1118 (1994)."><meta name="citation_reference" content="Jacobson, K. et al., &quot;International Workshop on the application of fluoresecence photobleaching techniques to problems in cell biology&quot;, Workshop Summary, Federation Proceedings, vol. 42, pp. 72-79 (1983)."><meta name="citation_patent_number" content="US:7169560"><meta name="citation_patent_application_number" content="US:10/852,482"><link rel="canonical" href="http://www.google.com/patents/US7169560"/><meta property="og:url" content="http://www.google.com/patents/US7169560"/><meta name="title" content="Patent US7169560 - Short cycle methods for sequencing polynucleotides"/><meta name="description" content="The invention provides methods for sequencing a polynucleotide comprising stopping an extension cycle in a sequence by synthesis reaction before the reaction has run to near or full completion."/><meta property="og:title" content="Patent US7169560 - Short cycle methods for sequencing polynucleotides"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("Z0zsU_mBKM2kyAS_jIHQCg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("CAN"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("Z0zsU_mBKM2kyAS_jIHQCg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("CAN"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7169560?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7169560"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=F4l5BAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7169560&amp;usg=AFQjCNFhj9C_PKQH2p7-RtdQSpXvUhf6Rg" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7169560.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7169560.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20050100932"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7169560"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7169560" style="display:none"><span itemprop="description">The invention provides methods for sequencing a polynucleotide comprising stopping an extension cycle in a sequence by synthesis reaction before the reaction has run to near or full completion....</span><span itemprop="url">http://www.google.com/patents/US7169560?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7169560 - Short cycle methods for sequencing polynucleotides</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7169560 - Short cycle methods for sequencing polynucleotides" title="Patent US7169560 - Short cycle methods for sequencing polynucleotides"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7169560 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 10/852,482</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jan 30, 2007</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">May 24, 2004</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Nov 12, 2003</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2545619A1">CA2545619A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1692312A2">EP1692312A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1692312A4">EP1692312A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7491498">US7491498</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7897345">US7897345</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20050100932">US20050100932</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20070122828">US20070122828</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20090191565">US20090191565</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20110151449">US20110151449</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20110152114">US20110152114</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20110245086">US20110245086</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2005047523A2">WO2005047523A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2005047523A3">WO2005047523A3</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">10852482, </span><span class="patent-bibdata-value">852482, </span><span class="patent-bibdata-value">US 7169560 B2, </span><span class="patent-bibdata-value">US 7169560B2, </span><span class="patent-bibdata-value">US-B2-7169560, </span><span class="patent-bibdata-value">US7169560 B2, </span><span class="patent-bibdata-value">US7169560B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Stanley+N+Lapidus%22">Stanley N Lapidus</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Philip+Richard+Buzby%22">Philip Richard Buzby</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Timothy+Harris%22">Timothy Harris</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Helicos+Biosciences+Corporation%22">Helicos Biosciences Corporation</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7169560.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7169560.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7169560.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (100),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (99),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (75),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (11),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (9)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7169560&usg=AFQjCNGkoWOuhoUjofhPb_85bBxWzh7gmA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7169560&usg=AFQjCNEFE6VIZBfXpaefmzBHLVLmIKeOag">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7169560B2%26KC%3DB2%26FT%3DD&usg=AFQjCNGxg5Bzgc9eEhXwLc0UFWIZtia52w">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT55698196" lang="EN" load-source="patent-office">Short cycle methods for sequencing polynucleotides</invention-title></span><br><span class="patent-number">US 7169560 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA51102932" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">The invention provides methods for sequencing a polynucleotide comprising stopping an extension cycle in a sequence by synthesis reaction before the reaction has run to near or full completion.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(9)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7169560B2/US07169560-20070130-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7169560B2/US07169560-20070130-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7169560B2/US07169560-20070130-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7169560B2/US07169560-20070130-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7169560B2/US07169560-20070130-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7169560B2/US07169560-20070130-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7169560B2/US07169560-20070130-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7169560B2/US07169560-20070130-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7169560B2/US07169560-20070130-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7169560B2/US07169560-20070130-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7169560B2/US07169560-20070130-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7169560B2/US07169560-20070130-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7169560B2/US07169560-20070130-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7169560B2/US07169560-20070130-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7169560B2/US07169560-20070130-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7169560B2/US07169560-20070130-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7169560B2/US07169560-20070130-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7169560B2/US07169560-20070130-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(12)</span></span></div><div class="patent-text"><div mxw-id="PCLM9154773" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A method for sequencing a nucleic acid template, the method comprising the steps of:
<div class="claim-text">(a) exposing a nucleic acid template hybridized to a primer to (i) a polymerase capable of catalyzing nucleotide addition to said primer, and (ii) a labeled nucleotide that is not a chain terminating nucleotide, under reaction conditions and for a time such that on average only one nucleotide is added to said primer by said polymerase to produce an extended primer incorporating said labeled nucleotide;</div>
<div class="claim-text">(b) identifying said single labeled nucleotide incorporated in step (a);</div>
<div class="claim-text">(c) neutralizing label in said single labeled nucleotide incorporated in step (a);</div>
<div class="claim-text">(d) repeating steps (a), (b) and (c) at least once; and</div>
<div class="claim-text">(e) determining a sequence of said template based upon the order of incorporation of said labeled nucleotides.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said label is a detectable label.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The method of <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein said detectable label is selected from the group consisting of cyanine, rhodamine, fluorescien, coumarin, BODIPY, alexa, and conjugated multi-dyes.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said template is individually optically resolvable.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said neutralizing step comprises cleaving said label from said nucleotide.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein in said exposing step said polymerase and/or said labeled nucleotide are present at concentrations such that a single labeled nucleotide is added to said primer.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, further comprising the step of washing any unincorporated nucleotide prior to step b.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. The method of <claim-ref idref="CLM-00005">claim 5</claim-ref>, wherein said template is attached to a surface.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said single labeled nucleotide is a deoxynucleotide selected from the group consisting of dATP, dTTP, dUTP, dCTP, and dGTP.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said single-labeled nucleotide is a nucleotide analog.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein in said exposing step, the reaction conditions regulate the rate at which said single labeled nucleotide is added to said primer so that only one labeled nucleotide is added to said primer in each cycle.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. A method for simultaneously sequencing plural nucleic acid templates, the method comprising the steps of:
<div class="claim-text">(a) exposing simultaneously plural single molecule nucleic acid templates immobilized on a surface, each of which is hybridized to a primer, to (i) a polymerase capable of catalyzing nucleotide addition to said primers hybridized to said templates, and (ii) an optically detectable labeled nucleotide that is not a chain terminating nucleotide, under reaction conditions and for a time such that only one nucleotide is added to each said primer by said polymerase to produce plural extended primers incorporating said labeled nucleotide; and</div>
<div class="claim-text">(b) optically identifying which of said templates on said surface incorporated said single labeled nucleotide in step (a).</div>
</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES16177317" lang="EN" load-source="patent-office" class="description">
    <heading>RELATED APPLICATIONS</heading> <p num="p-0002">This application claims the benefit of U.S. Provisional Application Nos. 60/546,277, filed on Feb. 19, 2004, 60/547,611, filed on Feb. 24, 2004, and 60/519,862, filed on Nov. 12, 2003.</p>
    <heading>FIELD OF THE INVENTION</heading> <p num="p-0003">The invention relates to methods for sequencing a polynucleotide, and more particularly, to methods for high throughput single molecule sequencing of target polynucleotides.</p>
    <heading>BACKGROUND</heading> <p num="p-0004">Completion of the human genome has paved the way for important insights into biologic structure and function. Knowledge of the human genome has given rise to inquiry into individual differences, as well as differences within an individual, as the basis for differences in biological function and dysfunction. For example, single nucleotide differences between individuals, called single nucleotide polymorphisms (SNPs), are responsible for dramatic phenotypic differences. Those differences can be outward expressions of phenotype or can involve the likelihood that an individual will get a specific disease or how that individual will respond to treatment. Moreover, subtle genomic changes have been shown to be responsible for the manifestation of genetic diseases, such as cancer. A true understanding of the complexities in either normal or abnormal function will require large amounts of specific sequence information.</p>
    <p num="p-0005">An understanding of cancer also requires an understanding of genomic sequence complexity. Cancer is a disease that is rooted in heterogeneous genomic instability. Most cancers develop from a series of genomic changes, some subtle and some significant, that occur in a small subpopulation of cells. Knowledge of the sequence variations that lead to cancer will lead to an understanding of the etiology of the disease, as well as ways to treat and prevent it. An essential first step in understanding genomic complexity is the ability to perform high-resolution sequencing.</p>
    <p num="p-0006">Various approaches to nucleic acid sequencing exist. One conventional way to do bulk sequencing is by chain termination and gel separation, essentially as described by Sanger et al., Proc Natl Acad Sci U S A, 74(12): 546367 (1977). That method relies on the generation of a mixed population of nucleic acid fragments representing terminations at each base in a sequence. The fragments are then run on an electrophoretic gel and the sequence is revealed by the order of fragments in the gel. Another conventional bulk sequencing method relies on chemical degradation of nucleic acid fragments. See, Maxam et al., Proc. Natl. Acad. Sci., 74: 560564 (1977). Finally, methods have been developed based upon sequencing by hybridization. See, e.g., Drmanac, et al., Nature Biotech., 16: 5458 (1998). Bulk techniques, such as those described above, cannot effectively detect single nucleotide differences between samples, and are not useful for comparative whole genome sequencing. Single molecule techniques are necessary for high-resolution detection of sequence differences.</p>
    <p num="p-0007">There have been several recent reports of sequencing using single molecule techniques. Most conventional techniques have proposed incorporation of fluorescently-labeled nucleotides in a template-dependent manner. A fundamental problem with conventional single molecule techniques is that the sequencing reactions are run to completion. For purposes of single molecule chemistry, this typically means that template is exposed to nucleotides for incorporation for about 10 half lives. This gives rise to problems in the ability to resolve single nucleotides as they incorporate in the growing primer strand. The resolution problem becomes extreme in the situation in which the template comprises a homopolymer region. Such a region is a continuous sequence consisting of the same nucleotide species. When optical signaling is used as the detection means, conventional optics are able to reliably distinguish one from two identical bases, and sometimes two from three, but rarely more than three. Thus, single molecule sequencing using fluorescent labels in a homopolymer region typically results in a signal that does not allow accurate determination of the number of bases in the region.</p>
    <p num="p-0008">One method that has been developed in order to address the homopolymer issue provides for the use of nucleotide analogues that have a modification at the 3 carbon of the sugar that reversibly blocks the hydroxyl group at that position. The added nucleotide is detected by virtue of a label that has been incorporated into the 3 blocking group. Following detection, the blocking group is cleaved, typically, by photochemical means to expose a free hydroxyl group that is available for base addition during the next cycle.</p>
    <p num="p-0009">However, techniques utilizing 3 blocking are prone to errors and inefficiencies. For example, those methods require excessive reagents, including numerous primers complementary to at least a portion of the target nucleic acids and differentially-labeled nucleotide analogues. They also require additional steps, such as cleaving the blocking group and differentiating between the various nucleotide analogues incorporated into the primer. As such, those methods have only limited usefulness.</p>
    <p num="p-0010">Need therefore exists for more effective and efficient methods and devices for single molecule nucleic acid sequencing.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0011">The invention provides methods for high throughput single molecule sequencing. In particular, the invention provides methods for controlling at least one parameter of a nucleotide extension reaction in order to regulate the rate at which nucleotides are added to a primer. The invention provides several ways of controlling nucleic acid sequence-by-synthesis reactions in order to increase the resolution and reliability of single molecule sequencing. Methods of the invention solve the problems that imaging systems have in accurately resolving a sequence at the single-molecule level. In particular, methods of the invention solve the problem of determining the number of nucleotides in a homopolymer stretch.</p>
    <p num="p-0012">Methods of the invention generally contemplate terminating sequence-by-synthesis reactions prior to completion in order to obtain increased resolution of individual nucleotides in a sequence. Fundamentally, this requires exposing nucleotides to a mixture comprising a template, a primer, and a polymerase under conditions sufficient for only limited primer extension. Reactions are conducted under conditions such that it is statistically unlikely that more than 1 or 2 nucleotides are added to a growing primer strand in any given incorporation cycle. An incorporation cycle comprises exposure of a template/primer to nucleotides directed at the base immediately downstream of the primer (this may be all four conventional nucleotides or analogs if the base is not known) and washing unhybridized nucleotide.</p>
    <p num="p-0013">Nucleotide addition in a sequence-by-synthesis reaction is a stochastic process. As in any chemical reaction, nucleotide addition obeys the laws of probability. Methods of the invention are concerned with controlling the rate of nucleotide addition on a per-cycle basis. That is, the invention teaches ways to control the rate of nucleotide addition within an extension cycle given the stochastic nature of the extension reaction itself. Methods of the invention are intended to control reaction rates within the variance that is inherent in a reaction that is fundamentally stochastic. Thus, the ability to control, according to the invention, base addition reactions such that, on average, no more than two bases are added in any cycle takes into account the inherent statistics of the reactions.</p>
    <p num="p-0014">The invention thus teaches polynucleotide sequence analysis using short cycle chemistry. One embodiment of the invention provides methods for slowing or reversibly inhibiting the activity of polymerase during a sequencing-by-synthesis reaction. Other methods teach altering the time of exposure of nucleotides to the template-primer complex. Still other methods teach the use of physical blockers that temporarily halt or slow polymerase activity and/or nucleotide addition. In general, any component of the reaction that permits regulation of the number of labeled nucleotides added to the primer per cycle, or the rate at which the nucleotides are incorporated and detected per cycle is useful in methods of the invention. Additional components include, but are not limited to, the presence or absence of a label on a nucleotide, the type of label and manner of attaching the label; the linker identity and length used to attach the label; the type of nucleotide (including, for example, whether such nucleotide is a dATP, dCTP, dTTP, dGTP or dUTP; a natural or non-natural nucleotide, a nucleotide analogue, or a modified nucleotide); the half-life of the extension cycle (where one half-life is the time taken for at least one incorporation to occur in 50% of the complementary strands); the local sequence immediately 3 to the addition position; whether such base is the first, second, third, etc. base added; the type of polymerase used; the particular batch characteristics of the polymerase; the processivity of the polymerase; the incorporation rate of the polymerase; the number of wash cycles (i.e., the number of times a nucleotide is introduced to the reaction then washed out); the number of target nucleic acids in the reaction; the temperature of the reaction and the reagents used in the reaction.</p>
    <p num="p-0015">In a preferred embodiment of the invention, a nucleic acid template is exposed to a primer capable of hybridizing to the template and a polymerase capable of catalyzing nucleotide addition to the primer. A labeled nucleotide is introduced for a period of time that is statistically insufficient for incorporation of more than about 2 nucleotides per cycle. Nucleotide exposure may also be coordinated with polymerization inhibition such that, on average, 0, 1, or 2 labeled nucleotides are added to the primer, but that 3 labeled nucleotides are almost never added to the primer in each cycle. Ideally, the exposure time, during which labeled nucleotides are exposed to the template-primer complex, is statistically insufficient for incorporation of more nucleotides than are resolvable by a detection system used to detect incorporation.</p>
    <p num="p-0016">The invention also contemplates performing a plurality of base incorporation cycles. Each cycle comprises exposing a template nucleic acid to a labeled nucleotide that is not a chain-terminating nucleotide. The labeled nucleotide is incorporated into a primer hybridized to the template nucleic acid if the nucleotide is capable of hybridizing to the template nucleotide immediately upstream of the primer and there is about a 99% probability that two or fewer of said nucleotides are incorporated into the same primer strand per cycle. Incorporated nucleotides are then identified.</p>
    <p num="p-0017">Methods of the invention also make use of differential base incorporation rates in order to control overall reaction rates. For example, the rate of incorporation is lower for a second nucleotide given incorporation of a prior nucleotide immediately upstream of the second. This effect is magnified if the first nucleotide comprises a label or other group that hinders processivity of the polymerase. By determining an approximate reduction in the rate of incorporation of the second nucleotide, one can regulated the time of exposure of a sample to a second labeled nucleotide such that the time is statistically insufficient for incorporation of more nucleotides than are resolvable by a detection system used to detect incorporation of the nucleotide into the primer.</p>
    <p num="p-0018">The invention may also be conducted using a plurality of primer extension cycles, wherein each cycle comprises exposing a target nucleic acid to a primer capable of hybridizing to the target, thereby forming a primed target; exposing the primed target to a labeled nucleic acid in the presence of a nucleic acid polymerase, coordinating transient inhibition of the polymerase and time of exposure to the labeled nucleotide such that it is statistically likely that at least one of said labeled nucleic acid is incorporated in the primer, but statistically unlikely that more than two of the labeled nucleotide are incorporated in the primer.</p>
    <p num="p-0019">According to another embodiment, methods of the invention comprise conducting a cycle of template-dependent nucleic acid primer extension in the presence of a polymerase and a labeled nucleotide; inhibiting polymerase activity such that it is statistically unlikely that more than about 2 nucleotides are incorporated into the same primer strand in the cycle; washing unincorporated labeled nucleotide away from the template; detecting any incorporation of the labeled nucleotide; neutralizing label in any incorporated labeled nucleotide; removing the inhibition; repeating the foregoing steps; and compiling a sequence based upon the sequence of nucleotides incorporated into the primer.</p>
    <p num="p-0020">In another embodiment, the invention provides a method comprising exposing a nucleic acid template to a primer capable of hybridizing to a portion of the template in order to form a template/primer complex reaction mixture; adding a labeled nucleotide in the presence of a polymerase to the mixture under conditions that promote incorporation of the nucleotide into the primer if the nucleotide is complementary to a nucleotide in the template that is downstream of said primer; coordinating removal of the labeled nucleotide and inhibition of the polymerase so that no more than about 2 nucleotides are incorporated into the same primer; identifying labeled nucleotide that has been incorporated into said primer; repeating the foregoing steps at least once; and determining a sequence of the template based upon the order of the nucleotides incorporated into the primer.</p>
    <p num="p-0021">According to another embodiment, the method comprises exposing a template nucleic acid to a primer capable of hybridizing to a portion of the template upstream of a region of the template to be sequenced; introducing a labeled nucleic acid and a polymerase to the template under conditions wherein the labeled nucleic acid will be incorporated in the primer if the labeled nucleic acid is capable of hybridizing with a base downstream of the primer; and controlling the rate of the incorporation by limiting the time of exposure of the labeled nucleic acid to the template or by inhibiting the polymerase at a predefined time after exposure of the template to the labeled nucleotide; detecting incorporation of the labeled nucleotide into the primer; and identifying the nucleotide in the template as the complement of labeled nucleotide incorporated into the primer.</p>
    <p num="p-0022">In yet another embodiment, methods of the invention comprise exposing a target polynucleotide to a primer capable of hybridizing to the polynucleotide, extending the primer in the presence of a polymerizing agent and one or more extendible nucleotides, each comprising a detectable label. The polymerizing agent is exposed to a cofactor (i.e., any agent that decreases or halts polymerase activity), and the incorporation of label is detected. The steps of extending the primer and exposing the polymerizing agent to a cofactor may be performed simultaneously, or may be performed in separate steps. In one embodiment, the method further comprises inactivating the cofactor, thereby reversing its effect on the polymerizing agent. Modes of inactivation depend on the cofactor. For example, where the cofactor is attached to the nucleotide, inactivation can typically be achieved by cleaving the cofactor from the nucleotide.</p>
    <p num="p-0023">Methods of the invention also address the problem of reduced detection due to a failure of some strands in a given cycle to incorporate labeled nucleotide. In each incorporation cycle, a certain number of strands fail to incorporate a nucleotide that should be incorporated based upon its ability to hybridize to a nucleotide present in the template. The strands that fail to incorporate a nucleotide in a cycle will not be prepared to incorporate a nucleotide in the next cycle (unless it happens to be the same as the unincorporated nucleotide, in which case the strand will still lag behind unless both nucleotides are incorporated in the same cycle). Essentially, this situation results in the strands that failed to incorporate being unavailable for subsequent polymerase-catalyzed additions to the primer. That, in turn, leads to fewer strands available for base addition in each successive cycle (assuming the non-incorporation occurs in all or most cycles). The invention overcomes this problem by exposing a template/primer complex to a labeled nucleotide that is capable of hybridizing to the template nucleotide immediately downstream of the primer. After removing unbound labeled nucleotide, the sample is exposed to unlabeled nucleotide, preferably in excess, of the same species. The unlabeled nucleotide fills in the positions in which hybridization of the labeled nucleotide did not occur. That functions to increase the number of strands that are available for participation in the next round. The effect is to increase resolution in subsequent rounds over background. In a preferred embodiment, the labeled nucleotide comprises a label that impedes the ability of polymerase to add a downstream nucleotide, thus temporarily halting the synthesis reaction until unlabeled nucleotide can be added, at which point polymerase inhibition is removed and t he next incorporation cycle is conducted</p>
    <p num="p-0024">One feature of this embodiment is that a sequence is compiled based upon the incorporation data, while allowing maximum strand participation in each cycle. Thus, methods of the invention are useful for identifying placeholders in some strands in a population of strands being sequenced. As long as there are no more than two consecutive placeholders in any one strand, the invention has a high tolerance for placeholders with little or no effect on the ultimate sequence determination.</p>
    <p num="p-0025">Methods of the invention are also useful for identifying a single nucleotide in a nucleic acid sequence. The method comprises the steps of sequentially exposing a template-bound primer to a labeled nucleotide and an unlabeled nucleotide of the same type in the presence of a polymerase under conditions that allow template-dependent primer extension; determining whether the first nucleotide is incorporated in the primer at a first position; repeating the sequentially exposing step using subsequent labeled and unlabeled nucleotides until a nucleotide is identified at the first position.</p>
    <p num="p-0026">Identification of nucleotides in a sequence can be accomplished according to the invention using fluorescence resonance energy transfer (FRET). Single pair FRET (spFRET) is a good mechanism for increasing signal-to-noise in single molecule sequencing. Generally, a FRET donor (e.g., cyanine-3) is placed on the primer, on the polymerase, or on a previously incorporated nucleotide. The primer/template complex then is exposed to a nucleotide comprising a FRET acceptor (e.g., cyanine-5). If the nucleotide is incorporated, the acceptor is activated and emits detectable radiation, while the donor goes dark. That is the indication that a nucleotide has been incorporated. The nucleotide is identified based upon knowledge of which nucleotide species contained the acceptor. The invention also provides methods for identifying a placeholder in a nucleic acid sequence using FRET. A nucleic acid primer is hybridized to a target nucleic acid at a primer binding site in the target. The primer comprises a donor fluorophore. The hybridized primer is exposed to a first nucleotide comprising an acceptor fluorophore that, when incorporated into the primer, prevents further polymerization of the primer. Whether there is fluorescent emission from the donor and the acceptor is determined, and a placeholder in the nucleic acid sequence is identified as the absence of emission in both the donor and the acceptor.</p>
    <p num="p-0027">In another embodiment, the method comprises hybridizing a nucleic acid primer comprising a donor fluorophore to a target nucleic acid at a primer binding site in the target; exposing the hybridized primer to a first nucleotide comprising an acceptor fluorophore that, when incorporated into the primer, prevents further polymerization of the primer; detecting the presence or absence of fluorescent emission from each of the donor and the acceptor; identifying a nucleotide that has been incorporated into the primer via complementary base pairing with the target as the presence of fluorescent emission from the acceptor; identifying a sequence placeholder as the absence of fluorescent emission from the donor and the acceptor; and repeating the exposing, detecting, and each of the identifying steps, thereby to compile a sequence of the target nucleic acid based upon the sequence of the incorporated nucleotides and the placeholders.</p>
    <p num="p-0028">The invention is useful in sequencing any form of polynucleotides, such as double-stranded DNA, single-stranded DNA, single-stranded DNA hairpins, DNA/RNA hybrids, RNAs with a recognition site for binding of the polymerizing agent, and RNA hairpins. The invention is particularly useful in high throughput sequencing of single molecule polynucleotides in which a plurality of target polynucleotides are attached to a solid support in a spatial arrangement such that each polynucleotides is individually optically resolvable. According to the invention, each detected incorporated label represents a single polynucleotide.</p>
    <p num="p-0029">A detailed description of the certain embodiments of the invention is provided below. Other embodiments of the invention are apparent upon review of the detailed description that follows.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p num="p-0030">The patent or application file contains at least one drawing executed in color. Copies of this patent or patent application publication with color drawings will be provided by the Office upon request and payment of the necessary fee.</p>
      <p num="p-0031"> <figref idrefs="DRAWINGS">FIG. 1</figref> shows asynchronous single molecule sequencing.</p>
      <p num="p-0032"> <figref idrefs="DRAWINGS">FIG. 2</figref> are screenshots showing data from short cycle sequencing with long homopolymer regions. <figref idrefs="DRAWINGS">FIG. 2</figref> <i>a </i>shows full cycle sequencing used to analyze 10 target polynucleotides in a simulated synthesis of their complementary strands using cycle periods of 10 half-lives and repeating the wash cycles 12 times. <figref idrefs="DRAWINGS">FIG. 2</figref> <i>b </i>shows a short cycle sequencing to analyze 10 target polynucleotides by simulating the synthesis of their complementary strands using short cycle periods of 0.8 half-life periods and repeating the wash cycles 60 times.</p>
      <p num="p-0033"> <figref idrefs="DRAWINGS">FIG. 3</figref> shows a short cycle embodiment for analyzing 200 target polynucleotides in a simulated synthesis of their complementary strands using short cycle periods of 0.8 half-life periods and repeating the wash cycles 60 times.</p>
      <p num="p-0034"> <figref idrefs="DRAWINGS">FIG. 4</figref> shows a statistical analysis of incorporation, showing that polymerizing agent may incorporate repeat labeled nucleotides less readily than the first labeled nucleotide.</p>
      <p num="p-0035"> <figref idrefs="DRAWINGS">FIG. 5</figref> shows a simulation showing the effect of decreasing the activity rate of the polymerizing agent and lengthening half-lives on the cycle period.</p>
      <p num="p-0036"> <figref idrefs="DRAWINGS">FIG. 6</figref> shows the number of cycles needed with cycle periods of various half-lives taking into account stalling factors of two (squares), five (triangles) and 10 (crosses), in order to obtain over 25 incorporations in over 80% of target homopolymers, with at least a 97% chance of incorporating two or less nucleotides per cycle (or a smaller than 3% chance of incorporating more than 2 nucleotides per cycle).</p>
      <p num="p-0037"> <figref idrefs="DRAWINGS">FIG. 7</figref> is a series of screenshots showing the effects of altering reaction conditions on the incorporation of nucleotides in a single molecule sequencing by synthesis reaction.</p>
    </description-of-drawings> <heading>DETAILED DESCRIPTION </heading> <p num="p-0038">The invention provides methods for high throughput single molecule sequencing. According to the invention, one or more parameters of a sequencing-by-synthesis reaction are preselected such that the incorporation of, preferably, a single nucleotide on a primed target template is optically detectable. In one embodiment, the preselected parameters regulate the rate at which the nucleotides are incorporated, and the rate at which the incorporated nucleotides are detected. According to this embodiment, the nucleotides are individually detected either as they are incorporated or shortly thereafter, essentially in real-time. In another embodiment, the preselected parameters permit the regulation of the number of nucleotides incorporated during a single extension cycle. In one aspect, the extension cycle is stopped short at a predetermined point at which, on average, only 0, 1, 2, or 3 nucleotides have been incorporated into the primer, rather than permitting the reaction to run to near or full completion in each cycle.</p>
    <p num="p-0039">Short cycle methods according to the invention increase the resolution of individual nucleotides incorporated into the primer, but can decrease the yield of target templates successfully incorporating a nucleotide in a single extension cycle. In traditional full cycle sequencing, nucleotides may be allowed to react in the presence of a polymerizing agent until at least one becomes incorporated into at least 99% of the complementary strands. This would produce a yield of (0.99)<sup>n</sup>100% for a complementary strand extended by n nucleotides. Obtaining incorporation in 99% of the complementary strands, however, requires a period of several half-lives of the incorporation reaction, where one half-life is the time taken for at least one incorporation to occur in 50% of the complementary strands. Typically, the more strands that complete an incorporation during each cycle, the more n-mers obtained after n cycles.</p>
    <p num="p-0040">According to the invention, short cycle methods rely on a period of only a limited number of half-lives of exposure to nucleotides, thus resulting in fewer target templates having incorporated a nucleotide in the short extension cycle. However, the short sequencing cycles provided by methods of the invention allow asynchronous analysis of polynucleotides. Thus, if an incorporation reactions fails to occur on a particular target polynucleotide, it can be completed in a later cycle without producing erroneous information, or interfering with data from other target molecules being analyzed in parallel. As demonstrated in <figref idrefs="DRAWINGS">FIG. 1</figref>, a cytosine (C) incorporates into the extension product of one copy of a target polynucleotide, but fails to incorporate into the other copy. During subsequent cycles of incorporation, however, a C can be incorporated, without adversely affection sequencing information. Thus, in asynchronous incorporation, an incorporation that failed to occur on a particular target in one-cycle can catch up in later cycles, permitting the use of shorter, even if more numerous, cycles.</p>
    <p num="p-0041">Because short cycle methods according the invention permit the detection of, for example, one, two or three individual nucleotides incorporated into a primed template, the invention overcomes the difficulty posed by homopolymer regions of a template sequence. While detection techniques may be able to quantify signal intensity from a smaller number of incorporated nucleotides of the same base-type, for example two or three incorporated nucleotides, longer runs of identical bases may not permit quantification due to increasing signal intensity. That is, it may become difficult to distinguish n bases from n+1 bases, where the fractional increase in signal intensity from the (n+1)h base is small relative to the signal intensity from the already-incorporated n bases.</p>
    <p num="p-0042">In embodiments using short-cycles, it is possible to limit the number of nucleotides that become incorporated in a given cycle. For example, it can be determined by simulation that using a cycle period of about 0.8 half-lives can result in two or less incorporations in nine out of ten homopolymer complementary strands. (See Example 2b). In another simulation, a 0.8 half-life period was shown to allow no more than two incorporations in about 96.0% of 200 homopolymer complementary strands. As detection means can more readily quantify signal intensity from the smaller number of incorporated nucleotides rather than from larger numbers, the use of short-cycles addresses this issue. For example, imaging systems known in the art can reliably distinguish the difference in signal intensity between one versus two fluorescent labeling moieties on consecutively-incorporated nucleotides. Other imaging systems can reliably distinguish the difference in signal intensity between two versus three fluorescent labeling moieties on consecutively-incorporated nucleotides.</p>
    <p num="p-0043">In a further embodiment of the invention, an extension cycle comprising a labeled nucleotide is followed by an extension cycle using an unlabeled nucleotide of the same type so that the position in each of the target template in which a labeled nucleotide failed to incorporated becomes occupied by an unlabeled nucleotide. Methods in accordance with this embodiment provide for continued participation of specific template nucleic acids in which no incorporation of the labeled nucleotide occurred and reduced probability of missing nucleotides in the resulting compiled sequence.</p>
    <p num="p-0044">Further methods of the invention provide for identifying a placeholder in a nucleic acid sequence in the event that an accurate determination of a nucleotide at a particular position is not possible. A placeholder is simply a position of unknown identity. Such a placeholder may be represented in a nucleic acid sequence with, for example, an X, a traditional symbol for an unspecified nucleotide. Slotting a placeholder in a nucleic acid sequence avoids frameshift-type errors in sequence determination.</p>
    <p num="p-0045">Additional aspects of the invention are described in the following sections and illustrated by the Examples.</p>
    <p num="h-0007">Target Nucleic Acids and Nucleotides</p>
    <p num="p-0046">The invention is useful in sequencing any form of polynucleotides, including double-stranded DNA, single-stranded DNA, single-stranded DNA hairpins, DNA/RNA hybrids, RNAs with a recognition site for binding of the polymerizing agent, and RNA hairpins. Further, target polynucleotides may be a specific portion of a genome of a cell, such as an intron, regulatory region, allele, variant or mutation; the whole genome; or any portion therebetween. In other embodiments, the target polynucleotides may be mRNA, tRNA, rRNA, ribozymes, antisense RNA or RNAi. The target polynucleotide may be of any length, such as at least 10 bases, at least 25 bases, at least 50 bases, at least 100 bases, at least 500 bases, at least 1000 bases, or at least 2500 bases. The invention is particularly useful in high throughput sequencing of single molecule polynucleotides in which a plurality of target polynucleotides are attached to a solid support in a spatial arrangement such that each polynucleotides is individually optically resolvable. According to the invention, each detected incorporated label represents a single polynucleotide</p>
    <p num="p-0047">Nucleotides useful in the invention include both naturally-occurring and modified or non-naturally occurring nucleotides, and include nucleotide analogues. A nucleotide according to the invention may be, for example, a ribonucleotide, a deoxyribonucleotide, a modified ribonucleotide, a modified deoxyribonucleotide, a peptide nucleotide, a modified peptide nucleotide or a modified phosphate-sugar backbone nucleotide. Many aspects of nucleotides useful in the methods of the invention are subject to manipulation provide and suitable mechanisms for controlling the reaction. In particular, the species or type of nucleotide (i.e., natural or synthetic dATP, dCTP, dTTP, dGTP or dUTP; a natural or non-natural nucleotide) will affect the rate or efficiency of the reaction and therefore require consideration in preselecting parameters to produce the desire results.</p>
    <p num="p-0048">In addition, certain modifications to the nucleotides, including attaching a label, will affect the reaction. The size, polarity, hydrophobicity, hydrophilicity, charge, and other chemical attributes should be considered in determining parameters that will produce the desired results in the reaction. Labeled nucleotides of the invention include any nucleotide that has been modified to include a label which is directly or indirectly detectable. Such labels include optically-detectable labels such fluorescent labels, including fluorescein, rhodamine, phosphor, polymethadine dye, fluorescent phosphoramidite, texas red, green fluorescent protein, acridine, cyanine, cyanine 5 dye, cyanine 3 dye, 5-(2-aminoethyl)-aminonaphthalene-1-sulfonic acid (EDANS), BODIPY, ALEXA, or a derivative or modification of any of the foregoing. In one embodiment of the invention, fluorescence resonance energy transfer (FRET) technology is employed to produce a detectable, but quenchable, label. FRET may be used in the invention by, for example, modifying the primer to include a FRET donor moiety and using nucleotides labeled with a FRET acceptor moiety.</p>
    <p num="p-0049">The fluorescently labeled nucleotides can be obtained commercially (e.g., from NEN DuPont, Amersham, and BDL). Alternatively, fluorescently labeled nucleotides can also be produced by various techniques, such as those described in Kambara et al., Bio/Techol. (1988) 6: 816821; Smith et al., Nucl. Acid Res. (1985) 13: 23992412, and Smith et al., Nature (1986) 321: 67479.</p>
    <p num="p-0050">The fluorescent dye is preferably linked to the deoxyribose by a linker arm which is easily cleaved by chemical or enzymatic means. The length of the linker between the dye and the nucleotide can impact the incorporation rate and efficiency (see Zhu et al., Cytometry (1997) 28, 206). There are numerous linkers and methods for attaching labels to nucleotides, as shown in Oligonucleotides and Analogues: A Practical Approach (1991) (IRL Press, Oxford); Zuckerman et al., Polynucleotides Research (1987) 15: 530521; Sharma et al., Polynucleotides Research, (1991) 19: 3019; Giusti et al., PCR Methods and Applications (1993) 2: 223227; Fung et al., U.S. Pat. No. 4,757,141; Stabinsky, U.S. Pat. No. 4, 739,044; Agrawal et al., Tetrahedron Letters, (1990) 31: 154346; Sproat et al., Polynucleotides Research (1987) 15: 4837; and Nelson et al., Polynucleotides Research, (1989) 17: 718794.</p>
    <p num="p-0051">While the invention is exemplified herein with fluorescent labels, the invention is not so limited and can be practiced using nucleotides labeled with any form of detectable label, including radioactive labels, chemoluminescent labels, luminescent labels, phosphorescent labels, fluorescence polarization labels, and charge labels.</p>
    <p num="h-0008">Reaction Parameters</p>
    <p num="p-0052">Any parameter that permits the regulation of the number of labeled nucleotides added to the primer, or the rate at which the nucleotides are incorporated and detected can be controlled or exploited in the practice of the invention. Such parameters include, for example, the presence or absence of a label on a nucleotide, the type of label and manner of label attachment; the linker identity and length used to attach the label; the type of nucleotide (including, for example, whether such nucleotide is a dATP, dCTP, dTTP, dGTP or dUTP; a natural or non-natural nucleotide, a nucleotide analogue, or a modified nucleotide); the local sequence immediately 3 to the addition position; whether the base is the first, second, third, etc. base added; the type of polymerase used; the particular batch characteristics of the polymerase; the processivity of the polymerase; the incorporation rate of the polymerase, and use of polymerase cofactors.</p>
    <p num="p-0053">In addition, a variety of the conditions of the reaction provide useful mechanisms for controlling either the number of nucleotides incorporated in a single extension reaction or the rates of nucleotide incorporation and detection. Such conditions include the half-life of the extension cycle (where one half-life is the time taken for at least one incorporation to occur in 50% of the complementary strands); the number of wash cycles (i.e., the number of times a nucleotide is introduced to the reaction then washed out); the number of target nucleic acids in the reaction; and the temperature of the reaction and the reagents used in the reaction.</p>
    <p num="h-0009">Half-Lives and Wash Cycles</p>
    <p num="p-0054">Based on the methods disclosed herein, those of skill in the art will be able to determine the period of half-lives required to limit the number incorporations per cycle for a given number of target polynucleotides. (See Examples 2 and 3,<figref idrefs="DRAWINGS">FIGS. 2 and 3</figref>). Statistical simulations can also provide the number of repeated cycles needed to obtain a given number of incorporations, for example, to sequence a 25 base pair sequence. (See Examples 2 and 3, <figref idrefs="DRAWINGS">FIGS. 2 and 3</figref>). Referring to the simulations above, for example, it can be determined that 60 cycles, each 0.8 half-lives long, would be required for at least 25 incorporations in each of ten complementary strands (Example 2b, <figref idrefs="DRAWINGS">FIG. 2</figref> <i>b</i>). With 200 complementary strands, 60 cycles each 0.8 half-lives long produce at least 20 incorporations in each strand (Example 3, <figref idrefs="DRAWINGS">FIG. 3</figref>). Following the methodologies outlined herein, such as the simulated working examples detailed below, those of skill in the art will be able to make similar determinations for other numbers of targets of varying lengths, and use appropriate cycle periods and numbers of cycles to analyze homopolymer without using blocking moieties or reversible chain termination.</p>
    <p num="p-0055">The cycle period may also be chosen to permit a certain chance of incorporation of a given number of nucleotides in a complementary strand, and the cycle may be repeated a number of times to analyze the sequence of various numbers of target polynucleotides of varying length.</p>
    <p num="p-0056">In some embodiments, nucleotide half-lives for the incorporation reaction are affected by the fact that polymerizing agent may incorporate labeled nucleotides less readily than unlabeled nucleotides. <figref idrefs="DRAWINGS">FIG. 4</figref> illustrates the statistics of incorporation for a certain embodiment using a Klenow exo-minus polymerizing agent and Cy3- or Cy5-labeled nucleotides. The results show that polymerase may incorporate subsequent labeled nucleotides less readily than a prior labeled nucleotide. The graph of <figref idrefs="DRAWINGS">FIG. 4</figref> indicates, for example, that it may take five to ten times longer, resulting in a stalling of the incorporation reaction. In other embodiments, the stalling may vary with the use of other labeled nucleotides, other polymerizing agents and various reaction conditions.</p>
    <p num="p-0057">Polymerase stalling is a useful mechanism for controlling incorporation rates in single molecule reactions. As is shown in the Examples below, polymerase stalling is useful to limit incorporation of nucleotides into any given strand in a fairly precise manner. According to the invention, polymerase stalling is useful to limit incorporation to 1 nucleotide per strand per cycle, on average. Given a priori knowledge of the statistics of incorporation, single molecule reactions are controlled to provide a statistical likelihood that 1, sometimes 2, but rarely 3 nucleotides are incorporated in a strand in any given cycle.</p>
    <p num="p-0058">For example, the rate at which polymerase incorporates labeled nucleotides into a complementary strand may be slowed by a factor of about 2, about 3, about 4, about 5, about 6, about 7, about 8, about 9, about 10, about 11, about 12, or about 15 times compared to that observed with unlabeled nucleotides or compared to that observed for a prior incorporated labeled nucleotide.</p>
    <p num="p-0059">Moreover, this inhibition or delaying and longer half-lives can be taken into account when determining appropriate cycle periods and numbers of cycles to analyze homopolymer targets of a given length. <figref idrefs="DRAWINGS">FIGS. 3 and 4</figref>, for example, illustrate the results of simulations in which various factors affecting incorporation rates are taken into account. The graph of <figref idrefs="DRAWINGS">FIG. 4</figref>, for example, shows the number of cycles needed with cycle periods of various half-lives, taking into account stalling factors of two (squares), five (triangles), and 10 (crosses), in order to obtain 25 incorporations in over 80% of target strands, with at least a 97% chance of incorporating two or fewer nucleotides per cycle (or a smaller than 3% chance of incorporating three or more nucleotides per cycle). As the graph shows, stalling allows longer half-lives, which, in turn, permits the use of fewer cycles to obtain a full sequence with a defined error rate. As <figref idrefs="DRAWINGS">FIG. 5</figref> illustrates, if the use of labeled nucleotides slows down the polymerizing agent by a factor of 5, a cycle period of 2.4 half-lives produces over 80% 25-mers in 30 cycles. Based on the teachings of the invention, one of ordinary skill in the art can determine the cycle period required to limit the number incorporations per cycle for a given number of target polynucleotides of a given length.</p>
    <p num="p-0060">Applying methods disclosed herein, the cycle period may be selected to permit about a 70%, about a 75%, about an 80%, about an 85%, about a 90%, about a 95%, about a 96%, about a 97%, about a 98%, and about a 99% chance of incorporation of two or less nucleotides into the complementary strand. Other cycle periods that may be used in embodiments of the invention include, for example, no more than about 5 half-lives, no more than about 4 half-lives, no more than about 3 half-lives, no more than about 2 half-lives, no more than about 1 half-lives, no more than about 0.9 half-lives, no more than about 0.8 half-lives, no more than about 0.7 half-lives, no more than about 0.6 half-lives, no more than about 0.5 half-lives, no more than about 0.4 half-lives, no more than about 0.3 half-lives, and no more than about 0.2 half-lives of the incorporation reactions.</p>
    <p num="p-0061">In addition to the Examples provided below, various cycle periods and number of times the cycles are repeated may be used with various numbers of targets in certain embodiments of the invention. These include, for example, using about 200 target polynucleotides, a period of no more than about 0.6 half-lives and repeating at least about 50 times; using about 200 target polynucleotides, a period of no more than about 0.6 half-lives and repeating at least about 60 times; using about 200 target polynucleotides, a period of no more than about 0.6 half-lives and repeating at least about 70 times; using about 200 target polynucleotides, a period of no more than about 0.8 half-lives and repeating at least about 50 times; using about 200 target polynucleotides, a period of no more than about 0.8 half-lives and repeating at least about 60 times; using about 200 target polynucleotides, a period of no more than about 0.8 half-lives and repeating at least about 70 times; using about 200 target polynucleotides, a period of no more than about 1 half-life and repeating at least about 50 times; using about 200 target polynucleotides, a period of no more than about I half-life and repeating at least about 60 times; and using about 200 target polynucleotides, a period of no more than about 1 half-life and repeating at least about 70 times. In any of these embodiments, signal from incorporated nucleotides may be reduced after each or a number of cycles.</p>
    <p num="p-0062">The number of times the cycles need to be repeated is also determined based on methods described herein. In general, the number of cycles increases with the length of the sequence to be analyzed and the duration of the half life of nucleotide exposure decreases as the length of sequence to be analyzed becomes longer. Also in general, half lives of nucleotide exposure increase and cycle numbers decrease with greater inhibitory or delaying effects on nucleotide incorporation</p>
    <p num="p-0063">Taking into account various stalling factors, examples of cycle periods and number repeat cycles that may be used in certain embodiments further include a cycle period of no more than about 0.5 half-lives with a stalling factor of about 2, repeated at least about 90 times; a cycle period of no more than about 0.75 half-lives, with a stalling factor of about 2, repeated at least about 75 times; a cycle period of no more than about 1 half-lives, with a stalling factor of about 2, repeated at least about 50 times; a cycle period of no more than about 1.5 half-lives with a stalling factor of about 2 or about 5, repeated at least about 45 times; a cycle period of no more than about 1.75 half-lives, with a stalling factor of about 5, repeated at least about 35 times; a cycle period of no more than about 2 half-lives, with a stalling factor of about 5 or about 10, repeated at least about 35 times; a cycle period of no more than about 2.25 half-lives, with a stalling factor of about 5 or about 10, repeated at least about 30 or at least about 35 times, and a cycle period of about 2.4 half-lives, with a stalling factor of about 5, repeated at least about 30 times.</p>
    <p num="h-0010">Polymerases and Polymerase Cofactors</p>
    <p num="p-0064">Polymerizing agents useful in the invention include DNA polymerases (such as Taq polymerase, T7 mutant DNA polymerase, Klenow and Sequenase, 9N or a variant thereof), RNA polymerases, thermostable polymerases, thermodegradable polymerases, and reverse transcriptases. See e.g., Doublie et al., Nature (1998) 391:25158; Ollis et al. Nature (1985) 313: 76266; Beese et al., Science (1993) 260: 35255; Korolev et al., Proc. Natl. Acad. Sci. USA (1995) 92: 926468; Keifer et al., Structure (1997) 5:95108; and Kim et al., Nature (1995) 376:61216.</p>
    <p num="p-0065">Cofactors of the invention function to inhibit the polymerizing agent, thereby slowing or stopping synthesis activity, permitting the detection of an incorporated labeled nucleotide. Cofactors of the invention include any chemical agent or reaction condition that results in the inhibition of the polymerizing agent. Such inhibition may be in whole or in part and may be permanent, temporary or reversible. For example, a cofactor may be a label, an antibody, an aptamer, an organic or inorganic small molecule, or a polyanion, or it may comprise a chemical modification to a nucleotide (i.e., a nucleotide analogue may comprise a cofactor). A cofactor can be in solution, or it may be attached, either directly or through a linker to a nucleotide, primer, template or polymerase.</p>
    <p num="p-0066">Examples of useful cofactor agents include, among others, light sensitive groups such as 6-nitoveratryloxycarbonyl (NVOC), 2-nitobenzyloxycarbonyl (NBOC), , -dimethyl-dimethoxybenzyloxycarbonyl (DDZ), 5-bromo-7-nitroindolinyl, o-hyrdoxy-2-methyl cinnamoyl, 2-oxymethylene anthraquinone, and t-butyl oxycarbonyl (TBOC). Oligonucleotide Synthesis: A Practical Approach (IRL Press, Oxford). Useful polyanions are described in U.S. Pat. No. 6,667,165 (the disclosure of which is incorporated by reference herein); and useful aptamers are described in U.S. Pat. Nos. 6,020,130 and 6,183,967 (the disclosures of which are incorporated by reference herein). See U.S. Pat. No. 5,338,671 for useful antibodies. Nucleotides possessing various labels and cofactors can be readily synthesized. Labeling moieties are attached at appropriate sites on the nucleotide using chemistry and conditions as described in Gait (1984).</p>
    <p num="p-0067">Further, the cofactor may also be the detectable label. Labels useful as combined labels/cofactors include larger or bulky dyes. For example, the detectable label may comprise a dye having a bulky chemical structure that, once the nucleotide is incorporated into the extending primer, causes a steric hindrance of the polymerizing agent, blocking the polymerizing agent from any further synthesis. Examples of labels that may be useful for this purpose are described in the Example, as well as in Zhu et al., Polynucleotides Res. (1994) 22: 341822. For example, fluorophore labels that may be used to stall the polymerase include Cy3, Cy5, Cy7, ALEXA647, ALEXA 488, BODIPY 576/589, BODIPY 650/665, BODIPY TR, Nile Blue, Sulfo-IRD700, NN382, R6G, Rho123, tetramethylrhodamine and Rhodamine X. In one embodiment, the labels are as bulky as Cy5, with molecular weights at least about 1.5 kDa. In another embodiment, the labels are bulkier than Cy5, having molecular weights of at least about 1.6 kDa, at least about 1.7 kDa, at least about 1.8 kDa, at least about 1.9 kDa, at least about 2.0 kDa at least bout 2.5 kDa, or at least about 3.0 kDa.</p>
    <p num="p-0068">Further examples of such larger dyes include the following, with corresponding formula weights (in g/mol) in parentheses: Cy5 (534.6); Pyrene (535.6); 6-Carboxyfluorescein (FAM) (537.5); 6-Carboxyfluorescein-DMT (FAM-X (537.5); 5(6) Carboxyfluorescein (FAM) (537.5); 5-Fluorescein (FITC) (537.6); Cy3B (543.0); WellRED D4-PA (544.8); BODIPY 630/650 (545.5); 3 6-Carboxyfluorescein (FAM) (569.5); Cy3.5 (576.7); Cascade Blue (580.0); ALEXA Fluor 430 (586.8); Lucifer Yellow (605.5); ALEXA Fluor 532 (608.8); WellRED D2-PA (611.0); Cy5.5 (634.8); DY-630 (634.8); DY-555 (636.2); WellRED D3-PA (645.0); Rhodamine Red-X (654.0); DY-730 (660.9); DY-782 (660.9); DY-550 (667.8); DY-610 (667.8); DY-700 (668.9); 6-Tetrachlorofluorescein (TET) (675.2) ALEXA Fluor 568 (676.8); DY-650 (686.9); 5(6)-Carboxyeosin (689.0); Texas Red-X (702.0); ALEXA Fluor 594 (704.9); DY-675 (706.9); DY-750 (713.0); DY-681 (736.9); Hexachlorofluorescein (HEX) (744.1); DY-633 (751.9); LightCycler Red 705 (753.0); LightCycler Red 640 (758.0); DY-636 (760.9); DY-701 (770.9); FAR-Fuchsia (5-Amidite) (776.0); FAR-Fuchsia (SE) (776.0); DY-676 (808.0); Erythrosin (814); FAR-Blue (5-Amidite) (824.0); FAR-Blue (SE) (824.0); Oyster 556 (850.0); Oyster 656 (900.0); FAR-Green Two (SE) (960.0); ALEXA Fluor 546 (964.4); FAR-Green One (SE), (976.0); ALEXA Fluor 660 (985.0); Oyster 645 (1000.0); ALEXA Fluor 680 (1035.0); ALEXA Fluor 633 (1085.0); ALEXA Fluor 555 (1135.0); ALEXA Fluor 647 (1185.0); ALEXA Fluor 750 (1185.0); ALEXA Fluor 700 (1285.0). These reagents are commercially available from SYNTHEGEN, LLC (Houston, Tex.).</p>
    <p num="p-0069">There is extensive guidance in the literature for derivatizing fluorophore and quencher molecules for covalent attachment via common reactive groups that can be added to a nucleotide (see Haugland, Handbook of Fluorescent Probes and Research Chemicals (1992). There are also many linking moieties and methods for attaching fluorophore moieties to nucleotides, as described in Oligonucleotides and Analogues, supra; Guisti et al., supra; Agrawal et al, Tetrahedron Letters (1990) 31: 154346; and Sproat et al., Polynucleotide Research (1987) 15: 4837.</p>
    <p num="p-0070">In one embodiment, the method further comprises inactivating the cofactor, thereby reversing its effect on the polymerizing agent. Modes of inactivation depend on the cofactor. For example, where the cofactor is attached to the nucleotide, inactivation can typically be achieved by chemical, enzymatic, photochemical or radiation cleavage of the cofactor from the nucleotide. Cleavage of the cofactor can be achieved if a detachable connection between the nucleotide and the cofactor is used. For example, the use of disulfide bonds enables one to disconnect the dye by applying a reducing agent like dithiothreitol (DTT). In a further alternative, where the cofactor is a fluorescent label, it is possible to neutralize the label by bleaching it with radiation.</p>
    <p num="p-0071">In the event that temperature-sensitive cofactors are utilized, inactivation may comprise adjusting the reaction temperature. For example, an antibody that binds to thermostable polymerase at lower temperatures and blocks activity, but is denatured at higher temperatures, thus rendering the polymerase active; or single-stranded aptamers that bind to thermophilic polymerase at lower temperatures but are released at higher temperatures, may be inactivated by increasing the reaction temperature such the cofactor is released but polymerase activity is permitted.</p>
    <p num="p-0072">In one embodiment, transient inhibition of the polymerase and the time of exposure to the labeled nucleotide are coordinated such that it is statistically likely that at least one of the labeled nucleotide is incorporated in the primer, but statistically unlikely that more than two of the labeled nucleotide are incorporated. In another embodiment, the reaction is controlled by inhibiting the polymerase activity such that it is statistically unlikely that more than, for example, one or two nucleotides are incorporated into the same primer strand in the cycle.</p>
    <p num="h-0011">Temperature and Reagents</p>
    <p num="p-0073">Other reaction conditions that are useful in the methods of the invention include reaction temperature and reagents. For example, a temperature above or below the temperature required for optimal activity of the polymerizing agent, such as a temperature of about 2070, would be expected to result in a modulation of the polymerization rate, C. This form of inhibition is typically reversible with correction of the reaction temperature, provided that the delta in temperature was insufficient to cause a permanent damage to the polymerase.</p>
    <p num="p-0074">In another embodiment, buffer reagents useful in the methods of the invention include a detergent or surfactant, such as Triton-X 100, or salt and/or ion concentrations that facilitate or inhibit nucleotide incorporation.</p>
    <p num="h-0012">Predetermined Points for Stopping a Cycle</p>
    <p num="p-0075">The predetermined point at which a short cycle is stopped is defined, for example, by the occurrence of an event (such as the incorporation of a nucleotide comprising a blocking moiety that prevents further extension of the primer), the lapse of a certain amount of time (such as a specific number of half-lives), or the achievement of a statistically-significant datapoint (such as a period at which a statistically significant probability of two or less nucleotides have been incorporated). In one embodiment, the predetermined period of time is coordinated with an amount of polymerization inhibition such that, on average, a certain number of labeled nucleotides are added to the primer. In another embodiment, the number of incorporated labeled nucleotides is, on average, 0, 1 or 2, but almost never more than 3. The time period of exposure is defined in terms of statistical significance. For example, the time period may be that which is statistically insufficient for incorporation of more nucleotides than are resolvable by a detection system used to detect incorporation of the nucleotide into the primer. In another example, the time period that is statistically insufficient for incorporation of a greater number of nucleotides that are individually optically resolvable during a predetermined detection period (i.e., a period of time during which the incorporated nucleotides are detected).</p>
    <p num="p-0076">The reaction may be stopped by washing or flushing out the nucleotides that remain unincorporated and/or washing or flushing out polymerization agent. Further, many aspects of the repeated cycles may be automated, for example, using microfluidics for washing nucleotides to sites of anchored target polynucleotides, and washing out unincorporated nucleotides to halt each cycle.</p>
    <p num="p-0077">The following exemplifications of the invention are useful in understanding certain aspects of the invention but are not intended to limit the scope of the invention in any way.</p>
    <heading>EXAMPLE 1</heading> <p num="p-0078">Primers are synthesized from nucleoside triphosphates by known automated oligonucleotide synthetic techniques, e.g., via standard phosphoramidite technology utilizing a nucleic acid synthesizer, such as the ABI3700 (Applied Biosystems, Foster City, Calif.). The oligonucleotides are prepared as duplexes with a complementary strand, however, only the 5 terminus of the oligonucleotide proper (and not its complement) is biotinylated.</p>
    <p num="h-0014">Ligation of Oligonucleotides and Target Polynucleotides</p>
    <p num="p-0079">Double stranded target nucleic acids are blunt-end ligated to the oligonucleotides in solution using, for example, T4 ligase. The single strand having a 5 biotinylated terminus of the oligonucleotide duplex permits the blunt-end ligation on only on end of the duplex. In a preferred embodiment, the solution-phase reaction is performed in the presence of an excess amount of oligonucleotide to prohibit the formation of concantamers and circular ligation products of the target nucleic acids. Upon ligation, a plurality of chimeric polynucleotide duplexes result. Chimeric polynucleotides are separated from unbound oligonucleotides based upon size and reduced to single strands by subjecting them to a temperature that destabilizes the hydrogen bonds.</p>
    <p num="h-0015">Preparation of Solid Support</p>
    <p num="p-0080">A solid support comprising reaction chambers having a fused silica surface is sonicated in 2% MICRO-90 soap (Cole-Parmer, Vernon Hills, Ill.) for 20 minutes and then cleaned by immersion in boiling RCA solution (6:4:1 high-purity H<sub>2</sub>O/30% NH<sub>4</sub>OH/30% H<sub>2</sub>O<sub>2</sub>) for 1 hour. It is then immersed alternately in polyallylamine (positively charged) and polyacrylic acid (negatively charged; both from Aldrich) at 2 mg/ml and pH 8 for 10 minutes each and washed intensively with distilled water in between. The slides are incubated with 5 mM biotin-amine reagent (Biotin-EZ-Link, Pierce) for 10 minutes in the presence of 1-[3-(dimethylamino)propyl]-3-ethylcarbodiimide hydrochloride (EDC, Sigma) in MES buffer, followed by incubation with Streptavidin Plus (Prozyme, San Leandro, Calif.) at 0.1 mg/ml for 15 minutes in Tris buffer. The biotinylated single-stranded chimeric polynucleotides are deposited via ink-jet printing onto the streptavidin-coated chamber surface at 10 pM for 10 minutes in Tris buffer that contain 100 mM MgCl<sub>2</sub>.</p>
    <p num="h-0016">Equipment</p>
    <p num="p-0081">The experiments are performed on an upright microscope (BH-2, Olympus, Melville, N.Y.) equipped with total internal reflection (TIR) illumination, such as the BH-2 microscope from Olympus (Melville, N.Y.). Two laser beams, 635 (Coherent, Santa Clara, Calif.) and 532 nm (Brimrose, Baltimore), with nominal powers of 8 and 10 mW, respectively, are circularly polarized by quarter-wave plates and undergo TIR in a dove prism (Edmund Scientific, Barrington, N.J.). The prism is optically coupled to the fused silica bottom (Esco, Oak Ridge, N.J.) of the reaction chambers so that evanescent waves illuminated up to 150 nm above the surface of the fused silica. An objective (DPlanApo, 100 UV 1.3oil, Olympus) collects the fluorescence signal through the top plastic cover of the chamber, which is deflected by the objective to 40 m from the silica surface. An image splitter (Optical Insights, Santa Fe, N.M.) directs the light through two bandpass filters (630dcxr, HQ585/80, HQ690/60; Chroma Technology, Brattleboro, Vt.) to an intensified charge-coupled device (I-PentaMAX; Roper Scientific, Trenton, N.J.), which records adjacent images of a 120-60-m section of the surface in two colors.</p>
    <p num="h-0017">Experimental Protocols</p>
    <p num="h-0018">FRET-Based Method Using Nucleotide-Based Donor Fluorophore</p>
    <p num="p-0082">In a first experiment, universal primer is hybridized to a primer attachment site present in support-bound chimeric polynucleotides. Next, a series of incorporation reactions are conducted in which a first nucleotide comprising a cyanine-3 donor fluorophore is incorporated into the primer as the first extended nucleotide. If all the chimeric sequences are the same, then a minimum of one labeled nucleotide must be added as the initial FRET donor because the template nucleotide immediately 3 of the primer is the same on all chimeric polynucleotides. If different chimeric polynucleotides are used (i.e., the polynucleotide portion added to the bound oligonucleotides is different at least one location), then all four labeled dNTPs initially are cycled. The result is the addition of at least one donor fluorophore to each chimeric strand.</p>
    <p num="p-0083">The number of initial incorporations containing the donor fluorophore is limited by either limiting the reaction time (i.e., the time of exposure to donor-labeled nucleotides), by polymerase stalling, or both in combination. The inventors have shown that base-addition reactions are regulated by controlling reaction conditions. For example, incorporations can be limited to 1 or 2 at a time by causing polymerase to stall after the addition of a first base. One way in which this is accomplished is by attaching a dye to the first added base that either chemically or sterically interferes with the efficiency of incorporation of a second base. A computer model was constructed using Visual Basic (v. 6.0, Microsoft Corp.) that replicates the stochastic addition of bases in template-dependent nucleic acid synthesis. The model utilizes several variables that are thought to be the most significant factors affecting the rate of base addition. The number of half-lives until dNTPs are flushed is a measure of the amount of time that a template-dependent system is exposed to dNTPs in solution. The more rapidly dNTPs are removed from the template, the lower will be the incorporation rate. The number of wash cycles does not affect incorporation in any given cycle, but affects the number bases ultimately added to the extending primer. The number of strands to be analyzed is a variable of significance when there is not an excess of dNTPs in the reaction. Finally, the inhibition rate is an approximation of the extent of base addition inhibition, usually due to polymerase stalling. The homopolymer count within any strand can be ignored for purposes of this application. <figref idrefs="DRAWINGS">FIG. 2</figref> is a screenshot showing the inputs used in the model.</p>
    <p num="p-0084">The model demonstrates that, by controlling reaction conditions, one can precisely control the number of bases that are added to an extending primer in any given cycle of incorporation. For example, as shown in <figref idrefs="DRAWINGS">FIG. 7</figref>, at a constant rate of inhibition of second base incorporation (i.e., the inhibitory effect of incorporation of a second base given the presence of a first base), the amount of time that dNTPs are exposed to template in the presence of polymerase determines the number of bases that are statistically likely to be incorporated in any given cycle (a cycle being defined as one round of exposure of template to dNTPs and washing of unbound dNTP from the reaction mixture). As shown in <figref idrefs="DRAWINGS">FIG. 7</figref> <i>a</i>, when time of exposure to dNTPs is limited, the statistical likelihood of incorporation of more than two bases is essentially zero, and the likelihood of incorporation of two bases in a row in the same cycle is very low. If the time of exposure is increased, the likelihood of incorporation of multiple bases in any given cycle is much higher. Thus, the model reflects biological reality. At a constant rate of polymerase inhibition (assuming that complete stalling is avoided), the time of exposure of a template to dNTPs for incorporation is a significant factor in determining the number of bases that will be incorporated in succession in any cycle. Similarly, if time of exposure is held constant, the amount of polymerase stalling will have a predominant effect on the number of successive bases that are incorporated in any given cycle (See, <figref idrefs="DRAWINGS">FIG. 7</figref> <i>b</i>). Thus, it is possible at any point in the sequencing process to add or renew donor fluorophore by simply limiting the statistical likelihood of incorporation of more than one base in a cycle in which the donor fluorophore is added.</p>
    <p num="p-0085">Upon introduction of a donor fluorophore into the extending primer sequence, further nucleotides comprising acceptor fluorophores (here, cyanine-5) are added in a template-dependent manner. It is known that the Foster radius of Cy-3/Cy5 fluorophore pairs is about 5 nm (or about 15 nucleotides, on average). Thus, donor must be refreshed about every 15 bases. This is accomplished under the parameters outlined above. In general, each cycle preferably is regulated to allow incorporation of 1 or 2, but never 3 bases. So, refreshing the donor means simply the addition of all four possible nucleotides in a mixed-sequence population using the donor fluorophore instead of the acceptor fluorophore every approximately 15 bases (or cycles). <figref idrefs="DRAWINGS">FIG. 2</figref> shows schematically the process of FRET-based, template-dependent nucleotide addition as described in this example.</p>
    <p num="p-0086">The methods described above are alternatively conducted with the FRET donor attached to the polymerase molecule. In that embodiment, donor follows the extending primer as new nucleotides bearing acceptor fluorophores are added. Thus, there typically is no requirement to refresh the donor. In another embodiment, the same methods are carried out using a nucleotide binding protein (e.g., DNA binding protein) as the carrier of a donor fluorophore. In that embodiment, the DNA binding protein is spaced at intervals (e.g., about 5 nm or less) to allow FRET. Thus, there are many alternatives for using FRET to conduct single molecule sequencing using the devices and methods taught in the application. However, it is not required that FRET be used as the detection method. Rather, because of the intensities of the FRET signal with respect to background, FRET is an alternative for use when background radiation is relatively high.</p>
    <p num="h-0019">Non-FRET Based Methods</p>
    <p num="p-0087">Methods for detecting single molecule incorporation without FRET are also conducted. In this embodiment, incorporated nucleotides are detected by virtue of their optical emissions after sample washing. Primers are hybridized to the primer attachment site of bound chimeric polynucleotides Reactions are conducted in a solution comprising Klenow fragment Exo-minus polymerase (New England Biolabs) at 10 nM (100 units/ml) and a labeled nucleotide triphosphate in EcoPol reaction buffer (New England Biolabs). Sequencing reactions takes place in a stepwise fashion. First, 0.2 M dUTP-Cy3 and polymerase are introduced to support-bound chimeric polynucleotides, incubated for 6 to 15 minutes, and washed out. Images of the surface are then analyzed for primer-incorporated U-Cy5. Typically, eight exposures of 0.5 seconds each are taken in each field of view in order to compensate for possible intermittency (e.g., blinking) in fluorophore emission. Software is employed to analyze the locations and intensities of fluorescence objects in the intensified charge-coupled device pictures. Fluorescent images acquired in the WinView32 interface (Roper Scientific, Princeton, N.J.) are analyzed using ImagePro Plus software (Media Cybernetics, Silver Springs, Md.). Essentially, the software is programmed to perform spot-finding in a predefined image field using user-defined size and intensity filters. The program then assigns grid coordinates to each identified spot, and normalizes the intensity of spot fluorescence with respect to background across multiple image frames. From those data, specific incorporated nucleotides are identified. Generally, the type of image analysis software employed to analyze fluorescent images is immaterial as long as it is capable of being programmed to discriminate a desired signal over background. The programming of commercial software packages for specific image analysis tasks is known to those of ordinary skill in the art. If U-Cy5 is not incorporated, the substrate is washed, and the process is repeated with dGTP-Cy5, dATP-Cy5, and dCTP-Cy5 until incorporation is observed. The label attached to any incorporated nucleotide is neutralized, and the process is repeated. To reduce bleaching of the fluorescence dyes, an oxygen scavenging system can be used during all green illumination periods, with the exception of the bleaching of the primer tag.</p>
    <p num="p-0088">In order to determine a template sequence, the above protocol is performed sequentially in the presence of a single species of labeled dATP, dGTP, dCTP or dUTP. By so doing, a first sequence can be compiled that is based upon the sequential incorporation of the nucleotides into the extended primer. The first compiled sequence is representative of the complement of the template. As such, the sequence of the template can be easily determined by compiling a second sequence that is complementary to the first sequence. Because the sequence of the oligonucleotide is known, those nucleotides can be excluded from the second sequence to produce a resultant sequence that is representative of the target template.</p>
    <heading>EXAMPLE 2</heading> <p num="p-0089"> <figref idrefs="DRAWINGS">FIG. 2</figref> illustrates the advantage of short-cycle sequencing with respect to avoiding long homopolymer reads. <figref idrefs="DRAWINGS">FIG. 2</figref> <i>a </i>illustrates a simulated analysis of 10 target polynucleotides using non-short-cycle sequencing (Example 2a), whereas <figref idrefs="DRAWINGS">FIG. 2</figref> <i>b </i>illustrates a simulated analysis of the same number of target polynucleotides using short-cycle sequencing (Example 2b).</p>
    <p num="p-0090">The simulations were performed as follows: an Excel spreadsheet was opened and Customize . . .  selected from the Tools menu of the Excel toolbar. The Commands tab was selected and, after scrolling down, Macros was clicked. The smiley face that appeared in the right panel was dragged to the toolbars on top of the spreadsheet. The Customize box was closed and the smiley face clicked once. From the list of subroutines that appeared, ThisWorkbook.Main_Line. was selected. The program was run by clicking again on the smiley face. A copy of the source code for the Excel simulation is provided below.</p>
    <p num="p-0091">Input values were then entered into the tabbed sheet called In Out. There were three input values:</p>
    <p num="p-0092">The first input value corresponded to the period of time allowed for incorporation reactions of provided nucleotides into the growing complementary strands of the polynucleotides to be analyzed. This period was conveniently measured in half-lives of the incorporation reaction itself. Each cycle of incorporation was simulatedly stalled after a period of time, representing, for example, the time when unincorporated nucleotides would be flushed out or the incorporation reactions otherwise stalled.</p>
    <p num="p-0093">The second input value corresponds to the number of times each cycle of incorporation was repeated. That is, the number of times the steps of providing nucleotides, allowing incorporation reactions into the complementary strands in the presence of polymerizing agent, and then stopping the incorporations are repeated. The nucleotides were simulatedly provided as a wash of each of dATPs, dGTPs, dTTPs, and dCTPs. The program then recorded which nucleotides were incorporated, corresponding to a detection step of detecting incorporation.</p>
    <p num="p-0094">The third input value corresponds to number of strands of target polynucleotides to by analyzed in the simulation. The program allowed up to 1100 target polynucleotide molecules to be analyzed in a given simulation.</p>
    <p num="p-0095">After the program was started, as described above, the program first generated the inputted number of strands composed of random sequences. The program then simulated hybridization and polymerization of the correct base of each incorporation reaction, based on the generated sequence of the target polynucleotide templates. The program continued these simulated reactions for the allowed amount of simulated time, determined by the inputted number of half-lives. Statistics of the simulation were then computed and reported, including the longest strand, the shortest strand, and the average length of all strands, as well as the fraction of strands extended by at least 25 nucleotide incorporations, as discussed in more detail below.</p>
    <p num="p-0096">In the first part of this simulation, Example 2 a, the input values used were a cycle period of 10 half-lives, 12 repeats of the cycle, and 10 target polynucleotide strands.</p>
    <p num="p-0097"> <figref idrefs="DRAWINGS">FIG. 2</figref> <i>a </i>illustrates the results obtained. Homopolymers stretches which occurred in the same simulated complementary strand are highlighted in magenta wherever 2 nucleotides of the same base type were incorporated in a row, and in cyan wherever more than two nucleotides of the same base type were incorporated in a row.</p>
    <p num="p-0098"> <figref idrefs="DRAWINGS">FIG. 2</figref> <i>a </i>illustrates that the output values included the longest extended complementary strand obtained during the simulation (Longest extension in the ensemble of molecules); the shorted extended complementary strand obtained during the simulation (Shortest extension in the ensemble of molecules); and the average extension. These numbers represent the greatest number of incorporations into any of the 10 simulatedly growing complementary strands, the smallest number of incorporations for any of the 10, and the average number of incorporations for the 10.<figref idrefs="DRAWINGS">FIG. 2</figref> <i>a </i>indicates that the values obtained for Example 2a were 37 incorporations in the longest extension, 25 in the shortest, and 30.00 as the average number of incorporations.</p>
    <p num="p-0099">The output values also provided information on the number of incorporations that occurred in each of growing complementary strands during each cycle period of the simulation. For example, <figref idrefs="DRAWINGS">FIG. 2</figref> <i>a </i>indicates that for the input values of Example 2a, the percentage of growing stands extended by two or more nucleotides in a homopolymer stretch was 100.0%; and the percentage of growing strands extended by three or more nucleotides in a homopolymer stretch was 60.0%. That is, using a cycle period of 10 half-lives resulted in only 40% of the complementary strands being extended by two or less nucleotides in a homopolymer stretch per cycle of incorporation.</p>
    <p num="p-0100">Further, output values also indicated the total number of incorporations for each of the growing strands for the total number of repeated cycles. This represents the length of the sequence of target polynucleotide analyzed. <figref idrefs="DRAWINGS">FIG. 2</figref> <i>a </i>illustrates that in Example 2 a, 100.0% of the 10 target polynucleotides of the simulation were extended by at least 25 incorporated nucleotides. This illustrates that using a cycle period of 10 half-lives, and repeating the cycles 12 times, allowed analysis of a 25 base sequence of 10 target polynucleotides.</p>
    <p num="p-0101">In the second part of this simulation, Example 2b, the input values used were a cycle period of 0.8 half-lives, 60 repeats of the cycle, and 10 target polynucleotide strands.</p>
    <p num="p-0102"> <figref idrefs="DRAWINGS">FIG. 2</figref> <i>b </i>illustrates the results obtained. Homopolymers stretches which occurred in the same simulated complementary strand are highlighted in magenta wherever 2 nucleotides of the same base type were incorporated in a row, and in cyan wherever more than two nucleotides of the same base type were incorporated in a row.</p>
    <p num="p-0103"> <figref idrefs="DRAWINGS">FIG. 2</figref> <i>b </i>illustrates that the output values included the longest extended complementary strand obtained during the simulation (longest extension in the ensemble of molecules); the shortest extended complementary strand obtained during the simulation (shortest extension in the ensemble of molecules); and the average extension. These numbers represent the greatest number of incorporations into any of the 10 simulatedly growing complementary strands, the smallest number of incorporations for any of the 10, and the average number of incorporations for the 10. <figref idrefs="DRAWINGS">FIG. 2</figref> <i>b </i>indicates that the values obtained for Example 2b were 37 incorporations in the longest extension, 26 in the shortest, and 32.00 as the average number of incorporations.</p>
    <p num="p-0104">The output values also provided information on the number of incorporations that occurred in each of growing complementary strands during each cycle period of the simulation. For example, <figref idrefs="DRAWINGS">FIG. 2</figref> <i>b </i>indicates that for the input values of Example 2b, the percentage of growing stands extended by two or more nucleotides in a homopolymer stretch was 80.0%; and the percentage of growing strands extended by three or more nucleotides in a homopolymer stretch was 10.0%. That is, using a cycle period of 0.8 half-lives resulted in 90% of the complementary strands being extended by two or less nucleotides per cycle of incorporation.</p>
    <p num="p-0105">Output values also indicated the total number of incorporations for each of the growing strands for the total number of repeated cycles. As in Example 2a, this represents the length of the sequence of target polynucleotide analyzed. <figref idrefs="DRAWINGS">FIG. 2</figref> <i>b </i>illustrates that in Example 2b, 100.0% of the 10 target polynucleotides of the simulation were again extended by at least 25 incorporated nucleotides. This illustrates that using a cycle period of 0.8 half-lives, and repeating the cycles 60 times, allowed analysis of a 25 base sequence of 10 target polynucleotides.</p>
    <p num="p-0106">Comparing the two simulations, it will be appreciated by those in the art that the use of short-cycles of sequencing overcame issues of reading long repeats of homopolymer stretches in sequencing by synthesis, without using blocking moieties, as only a few nucleotides were incorporated per cycle. Comparing Examples 2a and 2b, the long cycles in 2a resulted in 40% of the extended complementary strands having two or less homopolymer nucleotide incorporations per cycle. Conversely, the short cycles in 11b resulted in 90% of the extended complementary strands having two or less homopolymer nucleotide incorporations per cycle, facilitating quantification. That is, as explained more thoroughly above, shorter reads can be quantitated to determine the number of nucleotides incorporated, for example, where the nucleotides are of the same</p>
    <p num="p-0107">Comparing Examples 2a and 2b also indicated that a greater number of repeated cycles were needed to analyze a given length of sequence when using shorter cycles. That is, the 10 half-lives cycle was repeated 12 times to result in 100.0% of the 10 complementary strands being extended by at least 25 nucleotides, whereas the 0.8 half-lives cycle was repeated 60 times to obtain this same result and thereby analyze the 25 nucleotides sequence.</p>
    <p num="p-0108">Nonetheless, many aspects of the repeated cycles may be automated, for example, using micro fluidics for washing nucleotides to sites of anchored target polynucleotides, and washing out unincorporated nucleotides to halt each cycle.</p>
    <p num="p-0109">As discussed herein, below is a copy of the source code for the simulation of short-cycle sequencing.</p>
    <p num="p-0110">
      <tables id="TABLE-US-00001" num="00001"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Source Code for Simulation of Short Cycle Sequencing</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Option Explicit all variables must be declared</td>
              </tr> <tr class="description-tr"> <td class="description-td">Option Base 1array pointers start at 1 not 0</td>
              </tr> <tr class="description-tr"> <td class="description-td">-------Constant Declarations-----------------------------------</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const NoColor = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const Black = 1</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const White = 2</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const Red = 3</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const Green = 4</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const Blue = 5</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const Yellow = 6</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const Magenta = 7</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const Cyan = 8</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const A = Red</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const G = Green</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const T = Blue</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const C = Yellow</td>
              </tr> <tr class="description-tr"> <td class="description-td">Const TENTH_HL = 0.93305</td>
              </tr> <tr class="description-tr"> <td class="description-td">-------Variable Declarations-----------------------------------</td>
              </tr> <tr class="description-tr"> <td class="description-td">Note: HL is short for half-life</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim MaxHalfLives As Integer The maximum number of half-lives the experiment will be run</td>
              </tr> <tr class="description-tr"> <td class="description-td">X10 for each wash cycle</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim HalfLives the Half Life variable is stepped in increments 0.1 half lives during every wash</td>
              </tr> <tr class="description-tr"> <td class="description-td">cycle until the max is reached</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim N, I, J, K, L, X, Y, Temp As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim WashCyclesMax, WashCycles A wash cycle is completed after flowing each of AGT&amp;C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim Molecule, Base, BaseType, Position As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim TempReal As Single</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim RandomMoleculesMax</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim HomoPolymersMax</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim MoleculesMax As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">------------the following three variables used to slow things down for second base</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim Longer_HL As Single</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim SecondMoleculeFactor As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">------------The array variables----------</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="133pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Dim TargetStrand(1100, 51) As Integer</td>
                <td class="description-td">--up 1100 molecules, with max length of 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Dim SynthesizedStrand(1100, 51) As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim HL_Tracker(1100, 51) As Integer</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="140pt" align="left"> </colspec> <colspec colname="2" colwidth="168pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Dim PolymerasePointer(1100) As Integer</td>
                <td class="description-td">--contains the next available position on a given</td>
              </tr> <tr class="description-tr"> <td class="description-td">strand</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="133pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Dim StartPointer(1100) As Integer</td>
                <td class="description-td">pointers for determining run-lengths</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim StopPointer(1100) As Integer</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="112pt" align="left"> </colspec> <colspec colname="2" colwidth="196pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Dim Extension(1100)</td>
                <td class="description-td">records how far each molecule has been extended</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Dim TargetStrandFrequencyDist(15) As Integer--for storing frequency distribution of n-mers</td>
              </tr> <tr class="description-tr"> <td class="description-td">of target strand</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim SyntheticStrandFrequencyDist(15) As Integer --for storing frequency distribution of n-mers</td>
              </tr> <tr class="description-tr"> <td class="description-td">of target strand</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim SecondMolecule(1100) As Boolean</td>
              </tr> <tr class="description-tr"> <td class="description-td">--------Code----------------------------------------------------</td>
              </tr> <tr class="description-tr"> <td class="description-td">Sub Initialize( )</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim XX As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">-----clear the array which notes if a molecule is a second molecule</td>
              </tr> <tr class="description-tr"> <td class="description-td">For Molecule = 1 To 1100</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SecondMolecule(Molecule) = False</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next Molecule</td>
              </tr> <tr class="description-tr"> <td class="description-td">Clear the arrays</td>
              </tr> <tr class="description-tr"> <td class="description-td">For Base = 1 To 51</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Molecule = 1 To 1100</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TargetStrand(Molecule, Base) = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SynthesizedStrand(Molecule, Base) = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">HL_Tracker(Molecule, Base) = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">PolymerasePointer(Molecule) = 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Molecule</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next Base</td>
              </tr> <tr class="description-tr"> <td class="description-td">For XX = 1 To 15--clear the frequency distribution list</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TargetStrandFrequencyDist(XX) = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SyntheticStrandFrequencyDist(XX) = 0</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next XX</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For XX = 1 To 9</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(In Out).Cells(5 + XX, 10).Value = </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next XX</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">With Worksheets(In Out)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Get the front panel input values</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TempReal = .Range(D4).Value</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">MaxHalfLives = Int(TempReal * 10)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">WashCyclesMax = .Range(D7).Value</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">RandomMoleculesMax = .Range(D9).Value</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RandomMoleculesMax &gt; 1000 Then RandomMoleculesMax = 1000</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">HomoPolymersMax = .Range(D11).Value</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If HomoPolymersMax &gt; 100 Then HomoPolymersMax = 100</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">MoleculesMax = RandomMoleculesMax + HomoPolymersMax</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SecondMoleculeFactor = .Range(D14).Value</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Longer_HL = Exp(0.0693 / SecondMoleculeFactor)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">----Clear the output values</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Range(D20).Value = </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Range(D21).Value = </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Range(D22).Value = </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Range(E24).Value = </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Range(E25).Value = </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Range(E26).Value = </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End With</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(In Out).Range(E2).Value = Longer_HLDisplay the Longer_HL value</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Clear the output area &amp; Fill Row headings</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">With Worksheets(Molecules)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Range(B2:AY4006).ClearContents</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Range(B2:AY4006).Interior.ColorIndex = NoColor</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For XX = 1 To 1100</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Cells(3 + XX * 4, 1).Value = XX Add the row headings as running numbers</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next XX</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Range(B3).Value = Current Wash Cycle is:</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Range(L3).Value = Current Half-Life is:</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Range(U3).Value = Current Base in the reaction is:</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End With</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Randomize ---Seed the Random Number Generator</td>
              </tr> <tr class="description-tr"> <td class="description-td">End Sub</td>
              </tr> <tr class="description-tr"> <td class="description-td">Sub DrawSynthesizedStrands( )</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim TempMolecule, TempBase As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">With Worksheets(molecules)</td>
              </tr> <tr class="description-tr"> <td class="description-td">For TempBase = 1 To 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For TempMolecule = 1 To MoleculesMax</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If SynthesizedStrand(TempMolecule, TempBase) = Blue Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Cells(TempMolecule * 4 + 2, TempBase + 1).Font.ColorIndex = 2</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Else</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Cells(TempMolecule * 4 + 2, TempBase + 1).Font.ColorIndex = 0</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Cells(TempMolecule * 4 + 2, TempBase + 1).Interior.ColorIndex =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SynthesizedStrand(TempMolecule, TempBase)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If HL_Tracker(TempMolecule, TempBase) &gt; 0 Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">.Cells(TempMolecule * 4 + 2, TempBase + 1).Value = HL_Tracker(TempMolecule,</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TempBase)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next TempMolecule</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next TempBase</td>
              </tr> <tr class="description-tr"> <td class="description-td">End With</td>
              </tr> <tr class="description-tr"> <td class="description-td">End Sub</td>
              </tr> <tr class="description-tr"> <td class="description-td">Sub CreateTargetStrands( )</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim TempRand As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">For Base = 1 To 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Molecule = 1 To RandomMoleculesMax</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TempRand = Int(4 * Rnd + 3) random number of value 3,4,5 or 6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If TempRand = Blue Then TempRand = Cyan turn blue into cyan</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TargetStrand(Molecule, Base) = TempRand</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(Molecule * 4 + 3, Base + 1).Interior.ColorIndex =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TargetStrand(Molecule, Base)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Molecule</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next Base</td>
              </tr> <tr class="description-tr"> <td class="description-td">--now draw molecules with long stretches of homopolymers</td>
              </tr> <tr class="description-tr"> <td class="description-td">For Base = 1 To 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Molecule = RandomMoleculesMax + 1 To MoleculesMax</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TargetStrand(Molecule, Base) = A</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(Molecule * 4 + 3, Base + 1).Interior.ColorIndex =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TargetStrand(Molecule, Base)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Molecule</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next Base</td>
              </tr> <tr class="description-tr"> <td class="description-td">End Sub</td>
              </tr> <tr class="description-tr"> <td class="description-td">Sub Synthesize( )</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim MoleculeSynthesized As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim TempPointer As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim Parameter As Single</td>
              </tr> <tr class="description-tr"> <td class="description-td">For Molecule = 1 To 1100clear array which shows if molecule is a second molecule</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SecondMolecule(Molecule) = False</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next Molecule</td>
              </tr> <tr class="description-tr"> <td class="description-td">For BaseType = A To C Cover each of AGT&amp;C</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If BaseType = A Then Worksheets(Molecules).Range(AD3).Value = A</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If BaseType = G Then Worksheets(Molecules).Range(AD3).Value = G</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If BaseType = T Then Worksheets(Molecules).Range(AD3).Value = T</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If BaseType = C Then Worksheets(Molecules).Range(AD3).Value = C</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For HalfLives = 1 To MaxHalfLives</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Range(R3).Value = HalfLives / 10</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Molecule = 1 To MoleculesMax</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If SecondMolecule(Molecule) = False Then Parameter = TENTH_HL Else Parameter =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Longer_HL</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">-------------If we're flowing in A's, we attempt to polymerize only to T's</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If BaseType = A And TargetStrand(Molecule, PolymerasePointer(Molecule)) = T Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If Rnd &gt; Parameter Then MoleculeSynthesized = 1 Else MoleculeSynthesized = 0 did</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">molecule go?</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If MoleculeSynthesized = 1 Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="70pt" align="left"> </colspec> <colspec colname="1" colwidth="238pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SecondMolecule(Molecule) = True</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SynthesizedStrand(Molecule, PolymerasePointer(Molecule)) = A</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">HL_Tracker(Molecule, PolymerasePointer(Molecule)) = WashCycles</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">PolymerasePointer(Molecule) = PolymerasePointer(Molecule) + 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If PolymerasePointer(Molecule) &gt; 50 Then PolymerasePointer(Molecule) = 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">-------------If we're flowing in T's, we attempt to polymerize only to A's</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If BaseType = T And TargetStrand(Molecule, PolymerasePointer(Molecule)) = A Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If Rnd &gt; Parameter Then MoleculeSynthesized = 1 Else MoleculeSynthesized = 0 did</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">molecule go?</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If MoleculeSynthesized = 1 Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="70pt" align="left"> </colspec> <colspec colname="1" colwidth="238pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SecondMolecule(Molecule) = True</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SynthesizedStrand(Molecule, PolymerasePointer(Molecule)) = T</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">HL_Tracker(Molecule, PolymerasePointer(Molecule)) = WashCycles</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">PolymerasePointer(Molecule) = PolymerasePointer(Molecule) + 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If PolymerasePointer(Molecule) &gt; 50 Then PolymerasePointer(Molecule) = 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">-------------If we're flowing in G's, we attempt to polymerize only to C's</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If BaseType = G And TargetStrand(Molecule, PolymerasePointer(Molecule)) = C Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If Rnd &gt; Parameter Then MoleculeSynthesized = 1 Else MoleculeSynthesized = 0 did</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">molecule go?</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If MoleculeSynthesized = 1 Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="70pt" align="left"> </colspec> <colspec colname="1" colwidth="238pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SecondMolecule(Molecule) = True</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SynthesizedStrand(Molecule, PolymerasePointer(Molecule)) = G</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">HL_Tracker(Molecule, PolymerasePointer(Molecule)) = WashCycles</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">PolymerasePointer(Molecule) = PolymerasePointer(Molecule) + 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If PolymerasePointer(Molecule) &gt; 50 Then PolymerasePointer(Molecule) = 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">-------------If we're flowing in C's, we attempt to polymerize only to G's</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If BaseType = C And TargetStrand(Molecule, PolymerasePointer(Molecule)) = G Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If Rnd &gt; Parameter Then MoleculeSynthesized = 1 Else MoleculeSynthesized = 0 did</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">molecule go?</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If MoleculeSynthesized = 1 Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="70pt" align="left"> </colspec> <colspec colname="1" colwidth="238pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SecondMolecule(Molecule) = True</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SynthesizedStrand(Molecule, PolymerasePointer(Molecule)) = C</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">HL_Tracker(Molecule, PolymerasePointer(Molecule)) = WashCycles</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">PolymerasePointer(Molecule) = PolymerasePointer(Molecule) + 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If PolymerasePointer(Molecule) &gt; 50 Then PolymerasePointer(Molecule) = 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Molecule</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">DrawSynthesizedStrands --for now, display is refreshed after each increment of half life</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">for a given base</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next HalfLives</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next BaseType</td>
              </tr> <tr class="description-tr"> <td class="description-td">End Sub</td>
              </tr> <tr class="description-tr"> <td class="description-td">---Develop an analysis of the distribution of homopolymers in the full-length targets, report as a</td>
              </tr> <tr class="description-tr"> <td class="description-td">frequency</td>
              </tr> <tr class="description-tr"> <td class="description-td">---distribution of n-mers</td>
              </tr> <tr class="description-tr"> <td class="description-td">Sub AnalyzeTargetStrands( )</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim CurrentBase As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim BasesAhead As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim N As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim NumberedBases(50) As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim RunLengths(50) As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">For N = 1 To 15--clear the frequency distribution list</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TargetStrandFrequencyDist(N) = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SyntheticStrandFrequencyDist(N) = 0</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next N</td>
              </tr> <tr class="description-tr"> <td class="description-td">For Molecule = 1 To MoleculesMax Identify Changes among bases</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">NumberedBases(1) = 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(4 + Molecule * 4, 2).Value = NumberedBases(1) take this</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">out. For display only</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Base = 2 To 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If TargetStrand(Molecule, Base  1) &lt;&gt; TargetStrand(Molecule, Base) Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">NumberedBases(Base) = 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Else</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">NumberedBases(Base) = 0</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(4 + Molecule * 4, Base + 1).Value =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">NumberedBases(Base) take this out. For display only</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Base</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">---------------compute run lengths</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">But first we've got a boundary condition problem for the first base--we solve it here!!</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">RunLengths(1) = 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(5 + Molecule * 4, 2).Value = RunLengths(1)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Base = 2 To 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If NumberedBases(Base) = 1 Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">RunLengths(Base) = 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Else</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">RunLengths(Base) = RunLengths(Base - 1) + 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(5 + Molecule * 4, Base + 1).Value = RunLengths(Base)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Base</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">----save only the terminal value of a run length</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Base = 1 To 49</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base + 1) &gt; RunLengths(Base) Then RunLengths(Base) = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(6 + Molecule * 4, Base + 1).Value = RunLengths(Base)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Base</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(6 + Molecule * 4, 50 + 1).Value = RunLengths(50)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">boundary condition</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">-----Now determine the frequency distribution of each N-mer</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Base = 1 To 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base) = 1 Then TargetStrandFrequencyDist(1) =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TargetStrandFrequencyDist(1) + 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base) = 2 Then TargetStrandFrequencyDist(2) =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TargetStrandFrequencyDist(2) + 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base) = 3 Then TargetStrandFrequencyDist(3) =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TargetStrandFrequencyDist(3) + 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base) = 4 Then TargetStrandFrequencyDist(4) =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TargetStrandFrequencyDist(4) + 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base) = 5 Then TargetStrandFrequencyDist(5) =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TargetStrandFrequencyDist(5) + 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base) = 6 Then TargetStrandFrequencyDist(6) =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TargetStrandFrequencyDist(6) + 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base) = 7 Then TargetStrandFrequencyDist(7) =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TargetStrandFrequencyDist(7) + 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base) = 8 Then TargetStrandFrequencyDist(8) =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TargetStrandFrequencyDist(8) + 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base) &gt;= 9 Then TargetStrandFrequencyDist(9) =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TargetStrandFrequencyDist(9) + 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Base</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next Molecule</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For I = 1 To 9</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(In Out).Cells(5 + I, 10).Value = TargetStrandFrequencyDist(I) copy to the</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">spreadsheet</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next I</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">End Sub</td>
              </tr> <tr class="description-tr"> <td class="description-td">Sub AnalyzeResults( )</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim N As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim TwentyFiveMer, TwentyFiveMerAccumulator As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim LongestLength, ShortestLength As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim TempSum, Min, Max As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim AverageLength As Single</td>
              </tr> <tr class="description-tr"> <td class="description-td">----First we analyze the data about the degree of extension</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="77pt" align="left"> </colspec> <colspec colname="2" colwidth="231pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">For N = 1 To 1100</td>
                <td class="description-td">clear the extension array.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Extension(N) = 0</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next N</td>
              </tr> <tr class="description-tr"> <td class="description-td">For Molecule = 1 To MoleculesMax</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">N = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Base = 1 To 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If SynthesizedStrand(Molecule, Base) &lt;&gt; 0 Then N = Base</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">N = 1 debug statement</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Base</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Extension(Molecule) = N</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(In Out).Range(C13).Value = Extension(N) debug statement</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next Molecule ---we now have an array of maximum lengths of each strand in Extension. We</td>
              </tr> <tr class="description-tr"> <td class="description-td">can now compute...</td>
              </tr> <tr class="description-tr"> <td class="description-td">First we do the average:</td>
              </tr> <tr class="description-tr"> <td class="description-td">TempSum = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td">For N = 1 To 1100</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TempSum = Extension(N) + TempSum --grand total</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next N</td>
              </tr> <tr class="description-tr"> <td class="description-td">AverageLength = TempSum / MoleculesMax</td>
              </tr> <tr class="description-tr"> <td class="description-td">Worksheets(In Out).Range(D22).Value = AverageLength</td>
              </tr> <tr class="description-tr"> <td class="description-td">Now we find the Min and Max</td>
              </tr> <tr class="description-tr"> <td class="description-td">Max = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td">Min = 50</td>
              </tr> <tr class="description-tr"> <td class="description-td">For N = 1 To MoleculesMax</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If Max &gt; Extension(N) Then Max = Max Else Max = Extension(N)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If Min &lt; Extension(N) Then Min = Min Else Min = Extension(N)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next N</td>
              </tr> <tr class="description-tr"> <td class="description-td">Worksheets(In Out).Range(D20).Value = Max</td>
              </tr> <tr class="description-tr"> <td class="description-td">Worksheets(In Out).Range(D21).Value = Min</td>
              </tr> <tr class="description-tr"> <td class="description-td">Determine what fraction of molecules are more than 25 bases long</td>
              </tr> <tr class="description-tr"> <td class="description-td">TwentyFiveMerAccumulator = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td">For N = 1 To MoleculesMax</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If Extension(N) &gt; 24 Then TwentyFiveMerAccumulator = TwentyFiveMerAccumulator + 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">NextN</td>
              </tr> <tr class="description-tr"> <td class="description-td">Worksheets(In Out).Range(E26).Value = TwentyFiveMerAccumulator / MoleculesMax</td>
              </tr> <tr class="description-tr"> <td class="description-td">End Sub</td>
              </tr> <tr class="description-tr"> <td class="description-td">Sub AnalyzeSynthesizedStrands( )</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim CurrentBase As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim BasesAhead As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim N As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim NumberedBases(51) As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim RunLengths(51) As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">Dim TwoHitAccumulator, ThreePlusHitAccumulator, TwoHit, ThreeHit As Integer</td>
              </tr> <tr class="description-tr"> <td class="description-td">TwoHitAccumulator = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td">ThreePlusHitAccumulator = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td">For I = 1 To 50</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">NumberedBases(I) = 3</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next I</td>
              </tr> <tr class="description-tr"> <td class="description-td">For Molecule = 1 To MoleculesMax Identify Changes among bases</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">NumberedBases(1) = 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(1 + Molecule * 4, 2).Value = NumberedBases(1) take this</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">out. For display only</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Base = 2 To Extension(Molecule)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If SynthesizedStrand(Molecule, Base  1) &lt;&gt; SynthesizedStrand(Molecule, Base) Or</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">HL_Tracker(Molecule, Base  1) &lt;&gt; HL_Tracker(Molecule, Base) Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">NumberedBases(Base) = 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Else</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">NumberedBases(Base) = 0</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(1 + Molecule * 4, Base + 1).Value =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">NumberedBases(Base) take this out. For display only</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Base</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">---------------compute run lengths</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">But first we've got a boundary condition problem for the first base--we solve it here!!</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">RunLengths(1) = 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(1 + Molecule * 4, 2).Value = RunLengths(1)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Base = 2 To Extension(Molecule)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If NumberedBases(Base) = 1 Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">RunLengths(Base) = 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Else</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">RunLengths(Base) = RunLengths(Base  1) + 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(1 + Molecule * 4, Base + 1).Value = RunLengths(Base)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Base</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">----save only the terminal value of a run length</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Base = 1 To Extension(Molecule)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base + 1) &gt; RunLengths(Base) Then RunLengths(Base) = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(1 + Molecule * 4, Base + 1).Value = RunLengths(Base)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Base</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(1 + Molecule * 4, 50 + 1).Value = RunLengths(Molecule)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">boundary condition</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TwoHit = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ThreeHit = 0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For Base = 1 To Extension(Molecule)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base) = 2 Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(1 + Molecule * 4, Base + 1).Interior.ColorIndex =</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Magenta</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TwoHit = 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If RunLengths(Base) &gt; 2 Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Cells(1 + Molecule * 4, Base + 1).Interior.ColorIndex = Cyan</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ThreeHit = 1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next Base</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">--Now determine what fraction of molecules have either 2 bases or 3+ base hits and report</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">results</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TwoHitAccumulator = TwoHitAccumulator + TwoHit</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ThreePlusHitAccumulator = ThreePlusHitAccumulator + ThreeHit</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Next Molecule</td>
              </tr> <tr class="description-tr"> <td class="description-td">Worksheets(In Out).Range(E24).Value = TwoHitAccumulator / MoleculesMax</td>
              </tr> <tr class="description-tr"> <td class="description-td">Worksheets(In Out).Range(E25).Value = ThreePlusHitAccumulator / MoleculesMax</td>
              </tr> <tr class="description-tr"> <td class="description-td">End Sub</td>
              </tr> <tr class="description-tr"> <td class="description-td">Public Sub Main_Line( )</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Initialize</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">---Creates the new strands based on number of washes for varying degrees of completion per</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">cycle</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">If MoleculesMax &gt; 0 And WashCyclesMax &gt; 0 Then</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CreateTargetStrands</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AnalyzeTargetStrands</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">For WashCycles = 1 To WashCyclesMax Do the desired number of wash cycles</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Worksheets(Molecules).Range(I3).Value = WashCycles</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Synthesize</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Next WashCycles</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">DrawSynthesizedStrands</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AnalyzeResults</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AnalyzeSynthesizedStrands</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="294pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">End If</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">End Sub</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>EXAMPLE 3</heading> <p num="p-0111"> <figref idrefs="DRAWINGS">FIG. 2</figref> illustrates yet another simulated analysis of a number of target polynucleotides using short-cycle sequencing. The simulation was run using the program described in Examples 2a and 2b but using a larger number of target polynucleotides.</p>
    <p num="p-0112">That is, in this simulation, the input values used were a cycle period of 0.8 half-lives, 60 repeats of the cycle, and 200 target polynucleotide strands. <figref idrefs="DRAWINGS">FIG. 2</figref> illustrates the results obtained. Homopolymers stretches which occurred in the same simulated complementary strand are highlighted in magenta wherever nucleotides of the same base type were incorporated in a row, and in cyan wherever more than two nucleotides of the same base type were incorporated in a row.</p>
    <p num="p-0113">The output values obtained were 48 incorporations in the longest extended complementary strand, 20 in the shortest, and 32.00 as the average number of incorporations for the 200 simulatedly extended complementary strands.</p>
    <p num="p-0114">Further, the percentage of growing stands extended by two or more nucleotides in a homopolymer stretch was 78.5%; and the percentage of growing strands extended by three or more nucleotides in a homopolymer stretch was 4.0%. That is, using a cycle period of 0.8 half-lives resulted in 96.0% of the complementary strands being extended by two or less nucleotides in a homopolymer stretch per cycle of incorporation. Moreover, 95.5% of the 200 target polynucleotides of the simulation were extended by at least 25 incorporated nucleotides, while 100% were extended by at least 20 nucleotides. This illustrated that using a cycle period of 0.8 half-lives, and repeating the cycles 60 times, allows analysis of a 20 base sequence of 200 target polynucleotides.</p>
    <heading>EXAMPLE 4 </heading> <p num="p-0115">This example demonstrates a method according to the invention in which a single nucleotide in a position in a nucleic acid sequence is identified. A template-bound primer is sequentially exposed first to a labeled nucleotide and then to an unlabeled nucleotide of the same type under conditions and in the presence of reagents that allow template-dependent primer extension. The template is analyzed in order to determine whether the first nucleotide is incorporated in the primer at the first position or not. If not, then the sequential exposure to labeled and unlabeled nucleotides is repeated using another type of nucleotide until one such nucleotide is determined to have incorporated at the first position. Once an incorporated nucleotide is determined, the identity of the nucleotide in the position in the nucleic acid sequence is identified as the complementary nucleotide.</p>
    <heading>EXAMPLE 5</heading> <p num="p-0116">In this example, a series of reactions are performed as described above in Example 1. A nucleic acid primer is hybridized to a target nucleic acid at a primer binding site in the target. The primer comprises a donor fluorophore. The hybridized primer is exposed to a first nucleotide comprising an acceptor fluorophore comprising a blocking moiety that, when incorporated into the primer, prevents further polymerization of the primer. The presence or absence of fluorescent emission from each of the donor and the acceptor is determined. A nucleotide that has been incorporated into the primer via complementary base pairing with the target is identified by the presence of fluorescent emission from the acceptor, and a sequence placeholder is identified as the absence of fluorescent emission from the donor and the acceptor. A sequence of the target nucleic acid is complied based upon the sequence of the incorporated nucleotides and the placeholders.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3996345">US3996345</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 1975</td><td class="patent-data-table-td patent-date-value">Dec 7, 1976</td><td class="patent-data-table-td ">Syva Company</td><td class="patent-data-table-td ">Fluorescence quenching with immunological pairs in immunoassays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4119368">US4119368</a></td><td class="patent-data-table-td patent-date-value">Dec 22, 1976</td><td class="patent-data-table-td patent-date-value">Oct 10, 1978</td><td class="patent-data-table-td ">Citizen Watch Co. Ltd.</td><td class="patent-data-table-td ">Elastomer display device</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4153855">US4153855</a></td><td class="patent-data-table-td patent-date-value">Dec 16, 1977</td><td class="patent-data-table-td patent-date-value">May 8, 1979</td><td class="patent-data-table-td ">The United States Of America As Represented By The Secretary Of The Army</td><td class="patent-data-table-td ">Method of making a plate having a pattern of microchannels</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4344064">US4344064</a></td><td class="patent-data-table-td patent-date-value">Dec 6, 1979</td><td class="patent-data-table-td patent-date-value">Aug 10, 1982</td><td class="patent-data-table-td ">Western Electric Co., Inc.</td><td class="patent-data-table-td ">Article carrying a distinctive mark</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4351760">US4351760</a></td><td class="patent-data-table-td patent-date-value">Sep 7, 1979</td><td class="patent-data-table-td patent-date-value">Sep 28, 1982</td><td class="patent-data-table-td ">Syva Company</td><td class="patent-data-table-td ">Novel alkyl substituted fluorescent compounds and polyamino acid conjugates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4683195">US4683195</a></td><td class="patent-data-table-td patent-date-value">Feb 7, 1986</td><td class="patent-data-table-td patent-date-value">Jul 28, 1987</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">One of the first Polymerase Chain Reaction (PCR) patents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4683202">US4683202</a></td><td class="patent-data-table-td patent-date-value">Oct 25, 1985</td><td class="patent-data-table-td patent-date-value">Nov 27, 1990</td><td class="patent-data-table-td ">Cetus Corp</td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4707237">US4707237</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 1982</td><td class="patent-data-table-td patent-date-value">Nov 17, 1987</td><td class="patent-data-table-td ">Ciba Corning Diagnostics Corp.</td><td class="patent-data-table-td ">System for identification of cells by electrophoresis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4711955">US4711955</a></td><td class="patent-data-table-td patent-date-value">May 23, 1983</td><td class="patent-data-table-td patent-date-value">Dec 8, 1987</td><td class="patent-data-table-td ">Yale University</td><td class="patent-data-table-td ">Modified nucleotides and methods of preparing and using same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4725677">US4725677</a></td><td class="patent-data-table-td patent-date-value">Aug 10, 1984</td><td class="patent-data-table-td patent-date-value">Feb 16, 1988</td><td class="patent-data-table-td ">Biosyntech Gmbh</td><td class="patent-data-table-td ">Process for the preparation of oligonucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4739044">US4739044</a></td><td class="patent-data-table-td patent-date-value">Jun 13, 1985</td><td class="patent-data-table-td patent-date-value">Apr 19, 1988</td><td class="patent-data-table-td ">Amgen</td><td class="patent-data-table-td ">Method for derivitization of polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4757141">US4757141</a></td><td class="patent-data-table-td patent-date-value">Aug 26, 1985</td><td class="patent-data-table-td patent-date-value">Jul 12, 1988</td><td class="patent-data-table-td ">Applied Biosystems, Incorporated</td><td class="patent-data-table-td ">Conjugates with oligonucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4793705">US4793705</a></td><td class="patent-data-table-td patent-date-value">Oct 7, 1987</td><td class="patent-data-table-td patent-date-value">Dec 27, 1988</td><td class="patent-data-table-td ">The United States Of America As Represented By The United States Department Of Energy</td><td class="patent-data-table-td ">Single molecule tracking</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4811218">US4811218</a></td><td class="patent-data-table-td patent-date-value">Jun 2, 1986</td><td class="patent-data-table-td patent-date-value">Mar 7, 1989</td><td class="patent-data-table-td ">Applied Biosystems, Inc.</td><td class="patent-data-table-td ">Real time scanning electrophoresis apparatus for DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4863849">US4863849</a></td><td class="patent-data-table-td patent-date-value">Jul 18, 1985</td><td class="patent-data-table-td patent-date-value">Sep 5, 1989</td><td class="patent-data-table-td ">New York Medical College</td><td class="patent-data-table-td ">Automatable process for sequencing nucleotide</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4865968">US4865968</a></td><td class="patent-data-table-td patent-date-value">Apr 1, 1985</td><td class="patent-data-table-td patent-date-value">Sep 12, 1989</td><td class="patent-data-table-td ">The Salk Institute For Biological Studies</td><td class="patent-data-table-td ">DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4889818">US4889818</a></td><td class="patent-data-table-td patent-date-value">Jun 17, 1987</td><td class="patent-data-table-td patent-date-value">Dec 26, 1989</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">Dna polymerase; isolated from thermus aquaticus; use in polymerase chain reaction (pcr)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4942124">US4942124</a></td><td class="patent-data-table-td patent-date-value">Aug 4, 1988</td><td class="patent-data-table-td patent-date-value">Jul 17, 1990</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Genetic engineering, vectors and sequencing dna, hhybridizing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4962037">US4962037</a></td><td class="patent-data-table-td patent-date-value">Oct 7, 1987</td><td class="patent-data-table-td patent-date-value">Oct 9, 1990</td><td class="patent-data-table-td ">United States Of America</td><td class="patent-data-table-td ">Method for rapid base sequencing in DNA and RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4971903">US4971903</a></td><td class="patent-data-table-td patent-date-value">Mar 25, 1988</td><td class="patent-data-table-td patent-date-value">Nov 20, 1990</td><td class="patent-data-table-td ">Edward Hyman</td><td class="patent-data-table-td ">Pyrophosphate-based method and apparatus for sequencing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4979824">US4979824</a></td><td class="patent-data-table-td patent-date-value">May 26, 1989</td><td class="patent-data-table-td patent-date-value">Dec 25, 1990</td><td class="patent-data-table-td ">Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">High sensitivity fluorescent single particle and single molecule detection apparatus and method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4994368">US4994368</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 1987</td><td class="patent-data-table-td patent-date-value">Feb 19, 1991</td><td class="patent-data-table-td ">Syntex (U.S.A.) Inc.</td><td class="patent-data-table-td ">Forming extension of hybridized polynucleotide sequence, cleaving, dissociating fragments, hybridizing fragments</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4994372">US4994372</a></td><td class="patent-data-table-td patent-date-value">Dec 6, 1988</td><td class="patent-data-table-td patent-date-value">Feb 19, 1991</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Processive DNA polymerases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4994373">US4994373</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 1989</td><td class="patent-data-table-td patent-date-value">Feb 19, 1991</td><td class="patent-data-table-td ">Enzo Biochem, Inc.</td><td class="patent-data-table-td ">Method and structures employing chemically-labelled polynucleotide probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5085562">US5085562</a></td><td class="patent-data-table-td patent-date-value">Apr 4, 1990</td><td class="patent-data-table-td patent-date-value">Feb 4, 1992</td><td class="patent-data-table-td ">Westonbridge International Limited</td><td class="patent-data-table-td ">Micropump having a constant output</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5091652">US5091652</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 1990</td><td class="patent-data-table-td patent-date-value">Feb 25, 1992</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Laser excited confocal microscope fluorescence scanner and method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5096388">US5096388</a></td><td class="patent-data-table-td patent-date-value">Mar 22, 1990</td><td class="patent-data-table-td patent-date-value">Mar 17, 1992</td><td class="patent-data-table-td ">The Charles Stark Draper Laboratory, Inc.</td><td class="patent-data-table-td ">Microfabricated pump</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5096554">US5096554</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 1989</td><td class="patent-data-table-td patent-date-value">Mar 17, 1992</td><td class="patent-data-table-td ">Applied Biosystems, Inc.</td><td class="patent-data-table-td ">Nucleic acid fractionation by counter-migration capillary electrophoresis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5108892">US5108892</a></td><td class="patent-data-table-td patent-date-value">Aug 3, 1989</td><td class="patent-data-table-td patent-date-value">Apr 28, 1992</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Determination dna sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5112736">US5112736</a></td><td class="patent-data-table-td patent-date-value">Jun 14, 1989</td><td class="patent-data-table-td patent-date-value">May 12, 1992</td><td class="patent-data-table-td ">University Of Utah</td><td class="patent-data-table-td ">Dna sequencing using fluorescence background electroblotting membrane</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5143854">US5143854</a></td><td class="patent-data-table-td patent-date-value">Mar 7, 1990</td><td class="patent-data-table-td patent-date-value">Sep 1, 1992</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Large scale photolithographic solid phase synthesis of polypeptides and receptor binding screening thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5167784">US5167784</a></td><td class="patent-data-table-td patent-date-value">Sep 4, 1991</td><td class="patent-data-table-td patent-date-value">Dec 1, 1992</td><td class="patent-data-table-td ">Xerox Corporation</td><td class="patent-data-table-td ">Sequencing large nucleic acid fragments</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5171132">US5171132</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 1990</td><td class="patent-data-table-td patent-date-value">Dec 15, 1992</td><td class="patent-data-table-td ">Seiko Epson Corporation</td><td class="patent-data-table-td ">Two-valve thin plate micropump</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5198540">US5198540</a></td><td class="patent-data-table-td patent-date-value">Oct 27, 1983</td><td class="patent-data-table-td patent-date-value">Mar 30, 1993</td><td class="patent-data-table-td ">Hubert Koster</td><td class="patent-data-table-td ">Process for the preparation of oligonucleotides in solution</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5209834">US5209834</a></td><td class="patent-data-table-td patent-date-value">Mar 9, 1992</td><td class="patent-data-table-td patent-date-value">May 11, 1993</td><td class="patent-data-table-td ">The United States Of America As Represented By The United States Department Of Energy</td><td class="patent-data-table-td ">Ordered transport and identification of particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5224843">US5224843</a></td><td class="patent-data-table-td patent-date-value">Jun 12, 1990</td><td class="patent-data-table-td patent-date-value">Jul 6, 1993</td><td class="patent-data-table-td ">Westonbridge International Ltd.</td><td class="patent-data-table-td ">Two valve micropump with improved outlet</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5242797">US5242797</a></td><td class="patent-data-table-td patent-date-value">Jan 2, 1992</td><td class="patent-data-table-td patent-date-value">Sep 7, 1993</td><td class="patent-data-table-td ">Myron J. Block</td><td class="patent-data-table-td ">Nucleic acid assay method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5259737">US5259737</a></td><td class="patent-data-table-td patent-date-value">Jul 2, 1991</td><td class="patent-data-table-td patent-date-value">Nov 9, 1993</td><td class="patent-data-table-td ">Seiko Epson Corporation</td><td class="patent-data-table-td ">Silicon semiconductor sandwiched between glass layers; medical equipment</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5260433">US5260433</a></td><td class="patent-data-table-td patent-date-value">Aug 13, 1990</td><td class="patent-data-table-td patent-date-value">Nov 9, 1993</td><td class="patent-data-table-td ">Enzo Diagnostics, Inc.</td><td class="patent-data-table-td ">Saccharide specific binding system labeled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5265327">US5265327</a></td><td class="patent-data-table-td patent-date-value">Sep 13, 1991</td><td class="patent-data-table-td patent-date-value">Nov 30, 1993</td><td class="patent-data-table-td ">Faris Sadeg M</td><td class="patent-data-table-td ">Microchannel plate technology</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5267152">US5267152</a></td><td class="patent-data-table-td patent-date-value">Oct 26, 1990</td><td class="patent-data-table-td patent-date-value">Nov 30, 1993</td><td class="patent-data-table-td ">Yang Won S</td><td class="patent-data-table-td ">Non-invasive method and apparatus for measuring blood glucose concentration</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5302509">US5302509</a></td><td class="patent-data-table-td patent-date-value">Feb 27, 1991</td><td class="patent-data-table-td patent-date-value">Apr 12, 1994</td><td class="patent-data-table-td ">Beckman Instruments, Inc.</td><td class="patent-data-table-td ">Method for sequencing polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5304487">US5304487</a></td><td class="patent-data-table-td patent-date-value">May 1, 1992</td><td class="patent-data-table-td patent-date-value">Apr 19, 1994</td><td class="patent-data-table-td ">Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Device for analyzing fluid cell-containing sample consisting of solid substrate with inlet port, flow channel, cell handling region with lysing structure, detection system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5306403">US5306403</a></td><td class="patent-data-table-td patent-date-value">Aug 24, 1992</td><td class="patent-data-table-td patent-date-value">Apr 26, 1994</td><td class="patent-data-table-td ">Martin Marietta Energy Systems, Inc.</td><td class="patent-data-table-td ">Raman-based system for DNA sequencing-mapping and other separations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5336062">US5336062</a></td><td class="patent-data-table-td patent-date-value">Oct 20, 1992</td><td class="patent-data-table-td patent-date-value">Aug 9, 1994</td><td class="patent-data-table-td ">Fraunhofer-Gesellschaft Zur Forderung Der Angewandten Forschung E.V.</td><td class="patent-data-table-td ">Microminiaturized pump</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5360523">US5360523</a></td><td class="patent-data-table-td patent-date-value">Feb 17, 1993</td><td class="patent-data-table-td patent-date-value">Nov 1, 1994</td><td class="patent-data-table-td ">Li-Cor, Inc.</td><td class="patent-data-table-td ">DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5375979">US5375979</a></td><td class="patent-data-table-td patent-date-value">Jun 16, 1993</td><td class="patent-data-table-td patent-date-value">Dec 27, 1994</td><td class="patent-data-table-td ">Robert Bosch Gmbh</td><td class="patent-data-table-td ">Thermal micropump with values formed from silicon plates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5376252">US5376252</a></td><td class="patent-data-table-td patent-date-value">Nov 10, 1992</td><td class="patent-data-table-td patent-date-value">Dec 27, 1994</td><td class="patent-data-table-td ">Pharmacia Biosensor Ab</td><td class="patent-data-table-td ">Microfluidic structure and process for its manufacture</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5403709">US5403709</a></td><td class="patent-data-table-td patent-date-value">Oct 6, 1992</td><td class="patent-data-table-td patent-date-value">Apr 4, 1995</td><td class="patent-data-table-td ">Hybridon, Inc.</td><td class="patent-data-table-td ">Method for sequencing synthetic oligonucleotides containing non-phosphodiester internucleotide linkages</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5405747">US5405747</a></td><td class="patent-data-table-td patent-date-value">Mar 7, 1994</td><td class="patent-data-table-td patent-date-value">Apr 11, 1995</td><td class="patent-data-table-td ">The Regents Of The University Of California Office Of Technology Transfer</td><td class="patent-data-table-td ">Fluorescent detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5405783">US5405783</a></td><td class="patent-data-table-td patent-date-value">Mar 12, 1992</td><td class="patent-data-table-td patent-date-value">Apr 11, 1995</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Large scale photolithographic solid phase synthesis of an array of polymers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5409811">US5409811</a></td><td class="patent-data-table-td patent-date-value">Apr 16, 1992</td><td class="patent-data-table-td patent-date-value">Apr 25, 1995</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5424186">US5424186</a></td><td class="patent-data-table-td patent-date-value">Dec 6, 1991</td><td class="patent-data-table-td patent-date-value">Jun 13, 1995</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Very large scale immobilized polymer synthesis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5449767">US5449767</a></td><td class="patent-data-table-td patent-date-value">May 20, 1992</td><td class="patent-data-table-td patent-date-value">Sep 12, 1995</td><td class="patent-data-table-td ">Yale University</td><td class="patent-data-table-td ">Modified polynucleotides and methods of preparing same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5476928">US5476928</a></td><td class="patent-data-table-td patent-date-value">Feb 26, 1992</td><td class="patent-data-table-td patent-date-value">Dec 19, 1995</td><td class="patent-data-table-td ">Yale University</td><td class="patent-data-table-td ">Modified nucleotides and polynucleotides and complexes form therefrom</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5484701">US5484701</a></td><td class="patent-data-table-td patent-date-value">Jan 31, 1992</td><td class="patent-data-table-td patent-date-value">Jan 16, 1996</td><td class="patent-data-table-td ">E. I. Du Pont De Nemours And Company</td><td class="patent-data-table-td ">Method for sequencing DNA using biotin-strepavidin conjugates to facilitate the purification of primer extension products</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5492806">US5492806</a></td><td class="patent-data-table-td patent-date-value">Apr 12, 1993</td><td class="patent-data-table-td patent-date-value">Feb 20, 1996</td><td class="patent-data-table-td ">Hyseq, Inc.</td><td class="patent-data-table-td ">Method of determining an ordered sequence of subfragments of a nucleic acid fragment by hybridization of oligonucleotide probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5514256">US5514256</a></td><td class="patent-data-table-td patent-date-value">Oct 28, 1994</td><td class="patent-data-table-td patent-date-value">May 7, 1996</td><td class="patent-data-table-td ">Battelle Memorial Institute</td><td class="patent-data-table-td ">Apparatus for improved DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5518900">US5518900</a></td><td class="patent-data-table-td patent-date-value">Nov 23, 1993</td><td class="patent-data-table-td patent-date-value">May 21, 1996</td><td class="patent-data-table-td ">Molecular Tool, Inc.</td><td class="patent-data-table-td ">Method for generating single-stranded DNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5525464">US5525464</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 1994</td><td class="patent-data-table-td patent-date-value">Jun 11, 1996</td><td class="patent-data-table-td ">Hyseq, Inc.</td><td class="patent-data-table-td ">Method of sequencing by hybridization of oligonucleotide probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5529465">US5529465</a></td><td class="patent-data-table-td patent-date-value">Jul 28, 1992</td><td class="patent-data-table-td patent-date-value">Jun 25, 1996</td><td class="patent-data-table-td ">Fraunhofer-Gesellschaft Zur Forderung Der Angewandten Forschung E.V.</td><td class="patent-data-table-td ">Micro-miniaturized, electrostatically driven diaphragm micropump</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5534125">US5534125</a></td><td class="patent-data-table-td patent-date-value">Aug 10, 1994</td><td class="patent-data-table-td patent-date-value">Jul 9, 1996</td><td class="patent-data-table-td ">Li-Cor, Inc.</td><td class="patent-data-table-td ">DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5547839">US5547839</a></td><td class="patent-data-table-td patent-date-value">Dec 6, 1990</td><td class="patent-data-table-td patent-date-value">Aug 20, 1996</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Contacting polynucleotides with surface of solid substrate, hybridizing using nucleic acid polymerase and labeled nucleotide</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5556790">US5556790</a></td><td class="patent-data-table-td patent-date-value">Dec 5, 1994</td><td class="patent-data-table-td patent-date-value">Sep 17, 1996</td><td class="patent-data-table-td ">Pettit; John W.</td><td class="patent-data-table-td ">Method for Automated DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5558991">US5558991</a></td><td class="patent-data-table-td patent-date-value">Feb 7, 1994</td><td class="patent-data-table-td patent-date-value">Sep 24, 1996</td><td class="patent-data-table-td ">E. I. Du Pont De Nemours And Company</td><td class="patent-data-table-td ">DNA sequencing method using acyclonucleoside triphosphates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5599695">US5599695</a></td><td class="patent-data-table-td patent-date-value">Feb 27, 1995</td><td class="patent-data-table-td patent-date-value">Feb 4, 1997</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Applying a print material onto a substrate for the biosynthesis of an oligonucleotide array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5610287">US5610287</a></td><td class="patent-data-table-td patent-date-value">Nov 16, 1994</td><td class="patent-data-table-td patent-date-value">Mar 11, 1997</td><td class="patent-data-table-td ">Molecular Tool, Inc.</td><td class="patent-data-table-td ">Method for immobilizing nucleic acid molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5631734">US5631734</a></td><td class="patent-data-table-td patent-date-value">Feb 10, 1994</td><td class="patent-data-table-td patent-date-value">May 20, 1997</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Method and apparatus for detection of fluorescently labeled materials</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5632957">US5632957</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 1994</td><td class="patent-data-table-td patent-date-value">May 27, 1997</td><td class="patent-data-table-td ">Nanogen</td><td class="patent-data-table-td ">Molecular biological diagnostic systems including electrodes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5654149">US5654149</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Aug 5, 1997</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Non-radioactive DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5659171">US5659171</a></td><td class="patent-data-table-td patent-date-value">Oct 7, 1994</td><td class="patent-data-table-td patent-date-value">Aug 19, 1997</td><td class="patent-data-table-td ">Northrop Grumman Corporation</td><td class="patent-data-table-td ">Micro-miniature diaphragm pump for the low pressure pumping of gases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5670346">US5670346</a></td><td class="patent-data-table-td patent-date-value">Mar 7, 1995</td><td class="patent-data-table-td patent-date-value">Sep 23, 1997</td><td class="patent-data-table-td ">Amersham International Plc</td><td class="patent-data-table-td ">Modifying nucleotide analogues</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5674716">US5674716</a></td><td class="patent-data-table-td patent-date-value">Apr 13, 1995</td><td class="patent-data-table-td patent-date-value">Oct 7, 1997</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5675155">US5675155</a></td><td class="patent-data-table-td patent-date-value">Apr 26, 1995</td><td class="patent-data-table-td patent-date-value">Oct 7, 1997</td><td class="patent-data-table-td ">Beckman Instruments, Inc.</td><td class="patent-data-table-td ">For detecting electromagnetic radiation from analytical samples</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5688648">US5688648</a></td><td class="patent-data-table-td patent-date-value">Dec 19, 1995</td><td class="patent-data-table-td patent-date-value">Nov 18, 1997</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Identificaiton and detection of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5695940">US5695940</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 1995</td><td class="patent-data-table-td patent-date-value">Dec 9, 1997</td><td class="patent-data-table-td ">Hyseq, Inc.</td><td class="patent-data-table-td ">Method of sequencing by hybridization of oligonucleotide probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5705018">US5705018</a></td><td class="patent-data-table-td patent-date-value">Dec 13, 1995</td><td class="patent-data-table-td patent-date-value">Jan 6, 1998</td><td class="patent-data-table-td ">Hartley; Frank T.</td><td class="patent-data-table-td ">Micromachined peristaltic pump</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5707506">US5707506</a></td><td class="patent-data-table-td patent-date-value">May 6, 1996</td><td class="patent-data-table-td patent-date-value">Jan 13, 1998</td><td class="patent-data-table-td ">Battelle Memorial Institute</td><td class="patent-data-table-td ">Channel plate for DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5710628">US5710628</a></td><td class="patent-data-table-td patent-date-value">Dec 12, 1994</td><td class="patent-data-table-td patent-date-value">Jan 20, 1998</td><td class="patent-data-table-td ">Visible Genetics Inc.</td><td class="patent-data-table-td ">Automated electrophoresis and fluorescence detection apparatus and method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5712476">US5712476</a></td><td class="patent-data-table-td patent-date-value">Mar 27, 1996</td><td class="patent-data-table-td patent-date-value">Jan 27, 1998</td><td class="patent-data-table-td ">Visible Genetics Inc.</td><td class="patent-data-table-td ">Electrophoresis and fluorescence detection apparatus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5733729">US5733729</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 1995</td><td class="patent-data-table-td patent-date-value">Mar 31, 1998</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Analyzing nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5741640">US5741640</a></td><td class="patent-data-table-td patent-date-value">May 18, 1995</td><td class="patent-data-table-td patent-date-value">Apr 21, 1998</td><td class="patent-data-table-td ">Amersham Life Science, Inc.</td><td class="patent-data-table-td ">DNA cycle sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5741644">US5741644</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 1995</td><td class="patent-data-table-td patent-date-value">Apr 21, 1998</td><td class="patent-data-table-td ">Hitachi, Ltd.</td><td class="patent-data-table-td ">Minimizing partial sequence determination, repeated synthesis of new primers, time involved using reduced dna oligomer set having sequence of restriction enzyme cutting site or repetitive sequence in genome</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5744305">US5744305</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 1995</td><td class="patent-data-table-td patent-date-value">Apr 28, 1998</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Arrays of materials attached to a substrate</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5744312">US5744312</a></td><td class="patent-data-table-td patent-date-value">Dec 13, 1996</td><td class="patent-data-table-td patent-date-value">Apr 28, 1998</td><td class="patent-data-table-td ">Amersham Life Science, Inc.</td><td class="patent-data-table-td ">Thermostable DNA polymerase from Thermoanaerobacter thermohydrosulfuricus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5750341">US5750341</a></td><td class="patent-data-table-td patent-date-value">Apr 17, 1995</td><td class="patent-data-table-td patent-date-value">May 12, 1998</td><td class="patent-data-table-td ">Lynx Therapeutics, Inc.</td><td class="patent-data-table-td ">Cycles of duplex extension along a single stranded dna template</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5753788">US5753788</a></td><td class="patent-data-table-td patent-date-value">May 19, 1995</td><td class="patent-data-table-td patent-date-value">May 19, 1998</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Photolabile nucleoside protecting groups</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5755943">US5755943</a></td><td class="patent-data-table-td patent-date-value">Jul 8, 1996</td><td class="patent-data-table-td patent-date-value">May 26, 1998</td><td class="patent-data-table-td ">Li-Cor, Inc.</td><td class="patent-data-table-td ">DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5756285">US5756285</a></td><td class="patent-data-table-td patent-date-value">Apr 14, 1994</td><td class="patent-data-table-td patent-date-value">May 26, 1998</td><td class="patent-data-table-td ">Amersham Life Science, Inc.</td><td class="patent-data-table-td ">DNA cycle sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5759014">US5759014</a></td><td class="patent-data-table-td patent-date-value">Jan 12, 1995</td><td class="patent-data-table-td patent-date-value">Jun 2, 1998</td><td class="patent-data-table-td ">Westonbridge International Limited</td><td class="patent-data-table-td ">Micropump</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5759374">US5759374</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 1996</td><td class="patent-data-table-td patent-date-value">Jun 2, 1998</td><td class="patent-data-table-td ">Hitachi, Ltd.</td><td class="patent-data-table-td ">Used for gel electrophoresis; fluoroscence emission detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5762876">US5762876</a></td><td class="patent-data-table-td patent-date-value">Dec 22, 1994</td><td class="patent-data-table-td patent-date-value">Jun 9, 1998</td><td class="patent-data-table-td ">Molecular Tool, Inc.</td><td class="patent-data-table-td ">Determining genotype at a locus within genetic material using reaction value genetator indicating presence of alleles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5776767">US5776767</a></td><td class="patent-data-table-td patent-date-value">Dec 12, 1995</td><td class="patent-data-table-td patent-date-value">Jul 7, 1998</td><td class="patent-data-table-td ">Visible Genetics Inc.</td><td class="patent-data-table-td ">Combined pluraity of individual seuencers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5776782">US5776782</a></td><td class="patent-data-table-td patent-date-value">Jul 17, 1996</td><td class="patent-data-table-td patent-date-value">Jul 7, 1998</td><td class="patent-data-table-td ">Hamamatsu Photonics K.K.</td><td class="patent-data-table-td ">Radiating with activating light; measuring wavelength over time</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5789168">US5789168</a></td><td class="patent-data-table-td patent-date-value">May 1, 1996</td><td class="patent-data-table-td patent-date-value">Aug 4, 1998</td><td class="patent-data-table-td ">Visible Genetics Inc.</td><td class="patent-data-table-td ">Method for amplification and sequencing of nucleic acid polymers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5795722">US5795722</a></td><td class="patent-data-table-td patent-date-value">Mar 18, 1997</td><td class="patent-data-table-td patent-date-value">Aug 18, 1998</td><td class="patent-data-table-td ">Visible Genetics Inc.</td><td class="patent-data-table-td ">Method and kit for quantitation and nucleic acid sequencing of nucleic acid analytes in a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5795782">US5795782</a></td><td class="patent-data-table-td patent-date-value">Mar 17, 1995</td><td class="patent-data-table-td patent-date-value">Aug 18, 1998</td><td class="patent-data-table-td ">President &amp; Fellows Of Harvard College</td><td class="patent-data-table-td ">Measuring physical properties</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5807679">US5807679</a></td><td class="patent-data-table-td patent-date-value">Jul 7, 1995</td><td class="patent-data-table-td patent-date-value">Sep 15, 1998</td><td class="patent-data-table-td ">Myriad Genetics, Inc.</td><td class="patent-data-table-td ">Island hopping--a method to sequence rapidly very large fragments of DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6383749">US6383749</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 2, 1999</td><td class="patent-data-table-td patent-date-value">May 7, 2002</td><td class="patent-data-table-td ">Clontech Laboratories, Inc.</td><td class="patent-data-table-td ">Solid supports</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/USRE34069">USRE34069</a></td><td class="patent-data-table-td patent-date-value">Feb 16, 1990</td><td class="patent-data-table-td patent-date-value">Sep 15, 1992</td><td class="patent-data-table-td ">Biosyntech Gmbh</td><td class="patent-data-table-td ">Process for the preparation of oligonucleotides</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Adam et al., "<a href='http://scholar.google.com/scholar?q="Individual+genomes+targeted+in+sequencing+revolution"'>Individual genomes targeted in sequencing revolution</a>", Nature, vol. 411, p. 402 (May 2001).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Agrawal, S. et al., "<a href='http://scholar.google.com/scholar?q="Site+Specific+Functionalization+of+Oligodeoxynucleotides+for+Non-Radioactive+Labelling"'>Site Specific Functionalization of Oligodeoxynucleotides for Non-Radioactive Labelling</a>", Tetrahedron Letters, vol. 31, No. 11, pp. 1543-1546 (1990).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ambrose, W. et al., "<a href='http://scholar.google.com/scholar?q="Single+Molecule+Detection+With+Total+Internal+Reflection+Excitation%3A+Comparing+Signal-to-Background+and+Total+Signals+in+Different+Geometries"'>Single Molecule Detection With Total Internal Reflection Excitation: Comparing Signal-to-Background and Total Signals in Different Geometries</a>", Cytometry, vol. 36, pp. 224-231 (1999).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Amit, B. et al., "<a href='http://scholar.google.com/scholar?q="Photosensitive+Protecting+Groups+of+Amino+Sugars+and+Their+Use+in+Glycoside+Synthesis+.+.+.+Derivatives"'>Photosensitive Protecting Groups of Amino Sugars and Their Use in Glycoside Synthesis . . . Derivatives</a>", 1. Org. Chem., 39(2):192-6 (1974).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Arndt-Jovin, D. et al., "<a href='http://scholar.google.com/scholar?q="Immunofluorescence+Localization+ofZ-DNA+in+Chromosomes%3A+Quantitation+by+Scanning+Microphotometry+and+Computer-assisted+Image+Analysis"'>Immunofluorescence Localization ofZ-DNA in Chromosomes: Quantitation by Scanning Microphotometry and Computer-assisted Image Analysis</a>", I. Cell Bio!o?JI, vol. 101, pp. 1422-1433, (Oct. 1985).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Augustin, M.A., W. Ankenbauer, and B. Angerer, "<a href='http://scholar.google.com/scholar?q="Progress+towards+single-molecule+sequencing%3A+enzymatic+synthesis+of+nucleotide-specifically+labeled+DNA."'>Progress towards single-molecule sequencing: enzymatic synthesis of nucleotide-specifically labeled DNA.</a>" Journal of Biotechnology, 2001. 8(13): p. 289.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Axelrod, D. et al., "<a href='http://scholar.google.com/scholar?q="Total+internal+reflection+fluorescent+microscopy"'>Total internal reflection fluorescent microscopy</a>", J Microscopy, vol. 129, pp. 19-28, (1983).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Axelrod, D., "<a href='http://scholar.google.com/scholar?q="Cell-Substrate+Contacts+Illuminated+by+Total+Internal+Reflection+Fluorescence"'>Cell-Substrate Contacts Illuminated by Total Internal Reflection Fluorescence</a>", I. Cell Bio!o?JI, vol. 89, pp. 141-145, (April 1981).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bai, X., et al., "<a href='http://scholar.google.com/scholar?q="Photocleavage+of+a+2-nitrobenzyl+linker+bridging+a+fluorophore+to+the+5%27+end+of+DNA."'>Photocleavage of a 2-nitrobenzyl linker bridging a fluorophore to the 5' end of DNA.</a>" Proc Natl Acad Sci USA, 2003, vol. 100(2). p. 409-13.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Basche, T. et al., "<a href='http://scholar.google.com/scholar?q="Single+Molecule+Optical+Detection%2C+Imaging+and+Spectroscopy"'>Single Molecule Optical Detection, Imaging and Spectroscopy</a>", Chs. 2 and 3, Weinheim:VCM, Germany (1997).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Beaucage, S. et al., "<a href='http://scholar.google.com/scholar?q="Advances+in+the+Synthesis+of+Oligonucleotides+by+the+Phosphoramidite+Approach"'>Advances in the Synthesis of Oligonucleotides by the Phosphoramidite Approach</a>" Tetrahedron, 48:2223-2311 (1992).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Beese, L. et al., "<a href='http://scholar.google.com/scholar?q="Structure+of+DNA+Polymerase+I+Klenow+Fragment+Bound+to+Duplex+DNA"'>Structure of DNA Polymerase I Klenow Fragment Bound to Duplex DNA</a>", Science, 260:352-355 (1993).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bennett et. al., "<a href='http://scholar.google.com/scholar?q="Solexa+Sequencing+chemistry+can+be+applied+to+different+platforms+which+will+have+common+elements+in+detection+and+data+processing."'>Solexa Sequencing chemistry can be applied to different platforms which will have common elements in detection and data processing.</a>" Pharmacogenomics (2004) 5(4).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Biesalski et al., "<a href='http://scholar.google.com/scholar?q="Preparation+and+Characterization+of+a+Polyelectrolyte+Monolayer+Covalently+Attached+to+a+Planar+Solid+Surface."'>Preparation and Characterization of a Polyelectrolyte Monolayer Covalently Attached to a Planar Solid Surface.</a>" Macromolecules 111, 32, 2309-2316. Article was published on the web Mar. 10, 1999.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Black, D.L., Protein diversity from alternating splicing: A challenge for bioinformatics and post genome biology. Cell, 2000. 103(3): p. 367-370.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Blattner, F.R., et al., "<a href='http://scholar.google.com/scholar?q="The+Complete+genome+sequence+of+Escherichia+coli+K-12."'>The Complete genome sequence of Escherichia coli K-12.</a>" Science, 1997.277(5331):p. 1453-74.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boles et. al., "<a href='http://scholar.google.com/scholar?q="High-Resolution+Mapping+of+Carcinogen+Binding+Sites+on+DNA"'>High-Resolution Mapping of Carcinogen Binding Sites on DNA</a>" 1986, 25, 3039-3043.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Brackmann et. al, "<a href='http://scholar.google.com/scholar?q="Optimal+Enzymes+for+Single-Molecule+Sequencing"'>Optimal Enzymes for Single-Molecule Sequencing</a>" 18, D-04103.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Brakmann, S. and P. Nieckchen, "<a href='http://scholar.google.com/scholar?q="The+large+fragment+of+Escherichia+coli+DNA+polymerase+I+can+synthesize+DNA+exclusively+from+fluorescently+labeled+nucleotides."'>The large fragment of Escherichia coli DNA polymerase I can synthesize DNA exclusively from fluorescently labeled nucleotides.</a>" Chembiochem, 2001. 2(10):p. 773-777.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Braslavsky, I. et al., "<a href='http://scholar.google.com/scholar?q="Objective-type+dark-field+illumination+for+scattering+from+microbeads"'>Objective-type dark-field illumination for scattering from microbeads</a>", Applied Optics, vol. 40, No. 31, pp. 5650-5657, (Nov. 2001).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Braslavsky, I. et al., "<a href='http://scholar.google.com/scholar?q="Sequence+information+can+be+obtained+from+single+DNA+molecules"'>Sequence information can be obtained from single DNA molecules</a>", PNAS, vol. 100, No. 7, pp. 3960-3964 (Apr. 2003).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Braslavsky, I. et al., "<a href='http://scholar.google.com/scholar?q="Single+Molecule+Measurements+of+DNA+Polymerase+Activity%3A+A+Step+Towards+Single+Molecule+Sequencing"'>Single Molecule Measurements of DNA Polymerase Activity: A Step Towards Single Molecule Sequencing</a>", Biophys. I. Abstracts, p. 507A (2002).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Brechtel, R. et al., "<a href='http://scholar.google.com/scholar?q="Control+of+the+electro+osmotic+flow+by+metal-salt-containing+buffers"'>Control of the electro osmotic flow by metal-salt-containing buffers</a>", J Chromatoraphy A, vol. 716, pp. 97-105, (1995).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bridgman, A. et al., "<a href='http://scholar.google.com/scholar?q="An+improved+method+for+the+synthesis+ofmercurated+Dutp.+Enzymatic+synthesis+of+Hg-labelled+DNA+of+high+molecular+weight+suitable+for+use+in+an+image+based+DNA+sequencing+strategy"'>An improved method for the synthesis ofmercurated Dutp. Enzymatic synthesis of Hg-labelled DNA of high molecular weight suitable for use in an image based DNA sequencing strategy</a>", DNA Seq., vol. 6, No. 4, pp. 199-209 (1996).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bryzek, J. etal., "<a href='http://scholar.google.com/scholar?q="Micromachines+on+the+march"'>Micromachines on the march</a>", IEEE Spectrum, vol. 31, No. 5, pp. 20-31, (1994).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Buchaillot, L. et al., "<a href='http://scholar.google.com/scholar?q="Silicon+Nitride+Thin+Films+Young%27s+Modulus+Determination+by+an+Optical+Non+Destructive+Method"'>Silicon Nitride Thin Films Young's Modulus Determination by an Optical Non Destructive Method</a>", Jpn. J. Appl. Phys., vol. 36, pp. L794-L797, (Jun. 1997).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Burghardt, T. et al., "<a href='http://scholar.google.com/scholar?q="Total+Internal+Reflection+Fluorescence+Study+of+Energy+Transfer+in+Surface-Adsorbed+and+Dissolved+Bovine+Serum+Albumin"'>Total Internal Reflection Fluorescence Study of Energy Transfer in Surface-Adsorbed and Dissolved Bovine Serum Albumin</a>", Biochemistry, vol. 22, pp. 979-985 (1983).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Butler, D. et al. "<a href='http://scholar.google.com/scholar?q="Draft+data+leave+geneticists+with+a+mountain+still+to+climb"'>Draft data leave geneticists with a mountain still to climb</a>", Nature, vol. 405, Issue 6782, pp. 984-985 (May 2000).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Canard, B., B. Cardona, .and R.S. Sarfati, "<a href='http://scholar.google.com/scholar?q="Catalytic+editing+properties+of+DNA+polymerases."'>Catalytic editing properties of DNA polymerases.</a>" Proc Natl Acad Sci USA, 1995. 92(24): p. 10859-63.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Canard, et al., "<a href='http://scholar.google.com/scholar?q="DNA+polymerase+fluorescent+substrates+with+reversible+3%27-tags"'>DNA polymerase fluorescent substrates with reversible 3'-tags</a>". Gene, 1994. 148(1): p. 1-6.</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cheng et al., "<a href='http://scholar.google.com/scholar?q="High-speed+DNA+sequence+analysis%2C"'>High-speed DNA sequence analysis,</a>" Prog. in Biochem. and Biophys., vol. 22, pp. 223-227 (1995).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chicurel, M., "<a href='http://scholar.google.com/scholar?q="Faster%2C+better%2C+cheaper+genotyping"'>Faster, better, cheaper genotyping</a>", Nature, vol. 412, Issue 6847, pp. 580-582, (Aug. 2001).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chidgeavadze, Z. et al., "<a href='http://scholar.google.com/scholar?q="3%27-Fluro-2%27%2C3%27-dideoxyribonucleoside+5%27-triphosphates%3A+terminators+of+DNA+synthesis"'>3'-Fluro-2',3'-dideoxyribonucleoside 5'-triphosphates: terminators of DNA synthesis</a>", FEBS Letters, 183(2):275-278 (1985).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chiu, D. et al., "<a href='http://scholar.google.com/scholar?q="Patterned+deposition+of+cells+and+proteins+onto+surfaces+by+using+three-dimensional+microfluidic+systems%2C"'>Patterned deposition of cells and proteins onto surfaces by using three-dimensional microfluidic systems,</a>" PNAS, vol. 97, No. 6, pp. 2408-2413 (2000).</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chou et al., "<a href='http://scholar.google.com/scholar?q="A+microfabricated+device+for+sizing+and+sorting+DNA+molecules"'>A microfabricated device for sizing and sorting DNA molecules</a>", Applied Sciences, Biophysics: Chou et al., Proc. Natl. Acad. Sci. USA 96, pp. 11-13 (1999).</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chou et al., "<a href='http://scholar.google.com/scholar?q="A+Microfabricated+Rotary+Pump"'>A Microfabricated Rotary Pump</a>". Biomedical Microdevices. vol. 3: p. 323 (2001).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Close, D. et al., "<a href='http://scholar.google.com/scholar?q="Ultraviolet+Photobleaching+of+Free+Radicals+Created+in+y-Irradiated+Amino+Acids"'>Ultraviolet Photobleaching of Free Radicals Created in y-Irradiated Amino Acids</a>", Radiation Research, vol. 53, pp. 349-357 (1973).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cooper, J. et al., "<a href='http://scholar.google.com/scholar?q="Analysis+of+Fluorescence+Energy+Transfer+in+Duplex+and+Branched+DNA+Molecules"'>Analysis of Fluorescence Energy Transfer in Duplex and Branched DNA Molecules</a>", Biochemistry, vol. 29, pp. 9261-9268 (1990).</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Crocker, J.C. and D.G. Grier, "<a href='http://scholar.google.com/scholar?q="Methods+of+digital+video+microscopy+for+colloidal+studies."'>Methods of digital video microscopy for colloidal studies.</a>" Journal of Colloid and Interface Science, 1996. 179(1): p. 298-310.</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dapprich, J., "<a href='http://scholar.google.com/scholar?q="Single-molecule+DNA+digestion+by+lambda-exonuclease."'>Single-molecule DNA digestion by lambda-exonuclease.</a>" Cytometry, 1999.36(3): p. 163-168.</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Debenham, J.S., et al., "<a href='http://scholar.google.com/scholar?q="Two+New+Orthogonal+Amine-Protecting+Groups+that+can+be+Cleaved+under+Mild+or+Neutral+Conditions."'>Two New Orthogonal Amine-Protecting Groups that can be Cleaved under Mild or Neutral Conditions.</a>" Journal of the American Chemical Society, 1995. 117(11): p. 3302-3.</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Decher et al., "<a href='http://scholar.google.com/scholar?q="Buildup+of+ultrathin+multilayer+films+by+a+self-assembly+process%2C"'>Buildup of ultrathin multilayer films by a self-assembly process,</a>" Thin Solid Films, 210/211:831-835 (1992).</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Decher G.;et al., "<a href='http://scholar.google.com/scholar?q="Fuzzy+nanoassemblies+%3A+Toward+layered+polymeric+multicomposites."'>Fuzzy nanoassemblies : Toward layered polymeric multicomposites.</a>" Science, 1997.277(5330): p. 1232-1237.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Decher, G. et al. "<a href='http://scholar.google.com/scholar?q="Buildup+of+ultrathin+multiplayer+files+by+a+self-assembly+process%3A+III.+Consecutively+alternating+absorption+of+anionic+and+cationic+polyelectrolytes+on+charged+surfaces"'>Buildup of ultrathin multiplayer files by a self-assembly process: III. Consecutively alternating absorption of anionic and cationic polyelectrolytes on charged surfaces</a>", Thin Solid Films, 210:831-835 (1992).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Delamarche, E. et al., "<a href='http://scholar.google.com/scholar?q="Patterned+Delivery+of+Immunoglobulins+to+Surfaces+Using+Microfluidic+Networks"'>Patterned Delivery of Immunoglobulins to Surfaces Using Microfluidic Networks</a>", Science 276:779-781 (1997).</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dickson et al., "<a href='http://scholar.google.com/scholar?q="Simultaneous+Imaging+of+Individual+Molecules+aligned+both+parallel+and+perpendicular+to+the+optic+axis"'>Simultaneous Imaging of Individual Molecules aligned both parallel and perpendicular to the optic axis</a>" vol. 81, No. 24, 1998.</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Doktycz, M. et al., "<a href='http://scholar.google.com/scholar?q="Genosensors+and+Model+Hybridization+Studies"'>Genosensors and Model Hybridization Studies</a>", Automation Technologies for Genome Characterization, Ch. 10 T. Beugelsdijk (Ed), John Wiley &amp; Sons, Inc. (1997), pp. 205-225.</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Doublie, S. et al., "<a href='http://scholar.google.com/scholar?q="Crystal+structure+of+a+bacteriophage+T7+DNA+replication+complex+at+2.2+A+resolution"'>Crystal structure of a bacteriophage T7 DNA replication complex at 2.2 A resolution</a>", Nature; vol. 391, pp. 251-258 (Jan. 1998).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Driscoll et al., "<a href='http://scholar.google.com/scholar?q="Atomic-Scale+Imaging+of+DNA+Using+Scanning+Tunneling+Microscopy."'>Atomic-Scale Imaging of DNA Using Scanning Tunneling Microscopy.</a>" Nature, 1990.346(6281): p. 294-296.</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Drmanac, R. et al., "<a href='http://scholar.google.com/scholar?q="Sequencing+by+hybridization%3A+Towards+an+automated+sequencing+of+one+million+M13+clones+arrayed+on+membranes"'>Sequencing by hybridization: Towards an automated sequencing of one million M13 clones arrayed on membranes</a>", Electrophoresis, 13:566-573 (1992).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Duffy et al., "<a href='http://scholar.google.com/scholar?q="Patterning+Electroluminescence+Materials+with+Feature+Sizes+as+Small+as+5+um+Using+Elastomeric+Membrains+as+Masks+for+Dry+Lift-Off%2C"'>Patterning Electroluminescence Materials with Feature Sizes as Small as 5 um Using Elastomeric Membrains as Masks for Dry Lift-Off,</a>" Advanced Materials vol. 11, No. 7, pp. 546-552 (1999).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Duffy et al., "<a href='http://scholar.google.com/scholar?q="Rapid+prototyping+of+microfludic+switches+in+poly%28dimethyl+siloxane%29+and+their+acuation+by+electroosmotic+flow%2C"'>Rapid prototyping of microfludic switches in poly(dimethyl siloxane) and their acuation by electroosmotic flow,</a>" J. Micromech. Microeng., (1999) vol., 9, pp. 211-217.</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Duffy et al., "<a href='http://scholar.google.com/scholar?q="Rapid+Prototyping+of+Microfluidic+Systems+in+Poly%28dimethylsiloxane%29"'>Rapid Prototyping of Microfluidic Systems in Poly(dimethylsiloxane)</a>", Analytical Chemistry, vol. 70, No. 23, pp. 4974-4984 (1998).</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Effenhauser et al., "<a href='http://scholar.google.com/scholar?q="Integrated+capillary+electrophoresis+on+Flexible+silicone+microdevices%3A+Analysis+of+DNA+restriction+fragments+and+detection+of+single+DNA+molecules+on+microchips%2C"'>Integrated capillary electrophoresis on Flexible silicone microdevices: Analysis of DNA restriction fragments and detection of single DNA molecules on microchips,</a>" Anal. Chem., vol. 69, pp. 3451-3457 (1997).</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Effenhauser et al., "<a href='http://scholar.google.com/scholar?q="Integrated+chip-based+capillary+electrophoresis%2C"'>Integrated chip-based capillary electrophoresis,</a>" Electrophoresis, vol. 18, pp. 2203-2213 (1997).</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Eigen, M. et al., "<a href='http://scholar.google.com/scholar?q="Sorting+single+molecules%3A+Application+to+diagnostics+and+evolutionary+biotechnology"'>Sorting single molecules: Application to diagnostics and evolutionary biotechnology</a>", PNAS, vol. 91, pp. 5740-5747, (Jun. 1994).</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Evangelista, R.A., et al. "<a href='http://scholar.google.com/scholar?q="Characterization+of+fluorescent+nucleoside+triphosphates+by+capillary+electrophoresis+with+laser-induced+fluorescence+detection%3A+action+of+alkaline+phosphatase+and+DNA+polymerase."'>Characterization of fluorescent nucleoside triphosphates by capillary electrophoresis with laser-induced fluorescence detection: action of alkaline phosphatase and DNA polymerase.</a>" Anal Biochem, 1996.235(1): p. 89-97.</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fahrenberg et al., "<a href='http://scholar.google.com/scholar?q="A+microvalve+system+fabricated+by+thermoplastic+molding%2C"'>A microvalve system fabricated by thermoplastic molding,</a>" J. Micromech. Microeng., vol. 5, pp. 169-171(1995).</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Felicia, Y. et al., "<a href='http://scholar.google.com/scholar?q="Synthesis+and+Properties+of+Adenosine-5%27-triphosphoro-y-l-%285-sulfonic+acid%29naphthyl+Ethylamidate%3A+A+Fluorescent+Nucleotide+Substrate+for+DNA-Dependent+RNA+Polymerase+from+E.+colt"'>Synthesis and Properties of Adenosine-5'-triphosphoro-y-l-(5-sulfonic acid)naphthyl Ethylamidate: A Fluorescent Nucleotide Substrate for DNA-Dependent RNA Polymerase from E. colt</a>", Arch. Biochem. Biophys., 246(2):564-571 (1986).</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ferguson, et al., "<a href='http://scholar.google.com/scholar?q="A+fiber-optic+DNA+biosensor+microarray+for+the+analysis+of+gene+expression%2C"'>A fiber-optic DNA biosensor microarray for the analysis of gene expression,</a>" Nature Biotechnology, vol. 14, pp. 1681-1684 (1996).</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Firtz, I. et al., "<a href='http://scholar.google.com/scholar?q="Electronic+detection+of+DNA+by+its+intrinsic+molecular+charge"'>Electronic detection of DNA by its intrinsic molecular charge</a>", PNAS, vol. 99, No. 22, pp. 14142-14146 (Oct. 2002).</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Forster, T., "<a href='http://scholar.google.com/scholar?q="Delocalized+Excitation+and+Excitation+Transfer"'>Delocalized Excitation and Excitation Transfer</a>", Modem Quantum Chem., Istanbul Lectures, Part TII, pp. 93-137, Academic Press, New York (1965).</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fu e al., "<a href='http://scholar.google.com/scholar?q="An+integrated+microfabricated+cell+sorter"'>An integrated microfabricated cell sorter</a>". Anal Cherm, 2002. 74(11): p. 451-7.</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fu et al., "<a href='http://scholar.google.com/scholar?q="A+microfabricated+fluorescence-activated+cell+sorter%2C"'>A microfabricated fluorescence-activated cell sorter,</a>" Nature Biotechnology, vol. 17, pp. 1109-1111 (1999).</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Funatsu, T. et al., "<a href='http://scholar.google.com/scholar?q="Imaging+of+single+fluorescent+molecules+and+individual+ATP+turnovers+by+single+myosin+molecules+in+aqueous+solution"'>Imaging of single fluorescent molecules and individual ATP turnovers by single myosin molecules in aqueous solution</a>", Nature, vol. 374, pp. 555-559 (Apr. 1995).</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Garcia, A., "<a href='http://scholar.google.com/scholar?q="Detennination+of+Ion+Penneability+by+Fluorescence+Quenching"'>Detennination of Ion Penneability by Fluorescence Quenching</a>", Meth. in Enzymolgy, 207:501-511 (1992).</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gardner et al., "<a href='http://scholar.google.com/scholar?q="Comparative+kinetics+of+nucleotide+analog+incorporation+by+Vent+DNA+polymerase%2C"'>Comparative kinetics of nucleotide analog incorporation by Vent DNA polymerase,</a>" J. Biol. Chem., 279, No. 12, Mar. 19, 2004, 11834-11842.</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Giller et al., "<a href='http://scholar.google.com/scholar?q="Incorporation+of+reporter+molecule-labeled+nucleotides+by+DNA+polymerases.+I.+Chemical+synthesis+of+various+reporter+group-labeled+2%27deoxyribonucleoside-5%27-triphosphates%2C"'>Incorporation of reporter molecule-labeled nucleotides by DNA polymerases. I. Chemical synthesis of various reporter group-labeled 2'deoxyribonucleoside-5'-triphosphates,</a>" Nucleic Acids Res., 31, No. 10, 2003, 2630-2635.</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Giusti, W. et al., "<a href='http://scholar.google.com/scholar?q="Synthesis+and+Characterization+of+5%27+-Fluorescent-dye-labeled+Oligonucleotides"'>Synthesis and Characterization of 5' -Fluorescent-dye-labeled Oligonucleotides</a>", PCR Methods and Applications, 2:223-227 (1993).</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Goll et al., "<a href='http://scholar.google.com/scholar?q="Microvalves+with+bistable+buckled+polymer+diaphragms%2C"'>Microvalves with bistable buckled polymer diaphragms,</a>" J. Micromech. Microeng., vol. 6., pp. 77-79 (1996).</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Goodwin, P.M., et al., "<a href='http://scholar.google.com/scholar?q="Application+of+single+molecule+detection+to+DNA+sequencing."'>Application of single molecule detection to DNA sequencing.</a>" Nucleosides &amp; Nucleotides, 1997. 16(5-6): p. 543-550.</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gravesen et al., "<a href='http://scholar.google.com/scholar?q="Microfluidics-a+review"'>Microfluidics-a review</a>", J. Micromech. Microeng. 3IOP Publishing Ltd., pp. 168-192 (1993).vol. 3.</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Greene, T.W. and P.G.M. Wuts, "<a href='http://scholar.google.com/scholar?q="Protective+Groups+in+Organic+Synthesis."'>Protective Groups in Organic Synthesis.</a>" John Wiley &amp; Sons, Inc.: New York, 1999 3rd Ed.</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gueroui, Z., et al., "<a href='http://scholar.google.com/scholar?q="Observation+by+fluorescence+microscopy+of+transcription+on+single+combed+DNA."'>Observation by fluorescence microscopy of transcription on single combed DNA.</a>" Proceedings of the National Academy of Sciences of the United States of America, 2002. 99(9): p. 6005-6010.</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Guilbault, G., "<a href='http://scholar.google.com/scholar?q="Practical+Fluorescence-Theory%2C+Methods+and+Techniques%2C"'>Practical Fluorescence-Theory, Methods and Techniques,</a>" Chapters 1 and 3, and pp. 521-524, Marcel Dekker, Inc., New York (1973).</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gyllenstein, U. et al., "<a href='http://scholar.google.com/scholar?q="Generation+of+single-stranded+DNA+by+the+polymerase+chain+reaction+and+its+application+to+direct+sequencing+of+the+HLA-DQA+locus"'>Generation of single-stranded DNA by the polymerase chain reaction and its application to direct sequencing of the HLA-DQA locus</a>", PNAS 85:7652-56.</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ha et al., "<a href='http://scholar.google.com/scholar?q="Single-molecule+fluorescence+spectroscopy+of+enzyme+conformational+dynamics+and+cleavage+mechanism."'>Single-molecule fluorescence spectroscopy of enzyme conformational dynamics and cleavage mechanism.</a>" Proceedings of the National Academy of Sciences of the United States of America, 1999.96(3): p. 893-898.</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ha, "<a href='http://scholar.google.com/scholar?q="Single+molecule+dynamics+studied+by+polarization+modulation%2C"'>Single molecule dynamics studied by polarization modulation,</a>" Phys. Rev. Lett., 77, No. 19, Nov. 4, 1996, 39793982.</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ha, "<a href='http://scholar.google.com/scholar?q="Single+molecule+spectroscopy+with+automated+positioning%2C"'>Single molecule spectroscopy with automated positioning,</a>" Appl. Phys. Lett. 70, No. 6, Feb. 10, 1997, 782-784.</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ha, "<a href='http://scholar.google.com/scholar?q="Single-molecule+fluorescence+methods+for+the+study+of+nucleic+acids%2C"'>Single-molecule fluorescence methods for the study of nucleic acids,</a>" Current Opinion in Struct Bio, 11, 2001, 287-292.</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ha, T., "<a href='http://scholar.google.com/scholar?q="Single-molecue+fluorescence+resonance+energy+transfer."'>Single-molecue fluorescence resonance energy transfer.</a>" Methods, 2001. 25(1): p. 78-86.</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hanna, M. et al., "<a href='http://scholar.google.com/scholar?q="Synthesis+and+characterization+of+a+new+photocrosslinking+CTP+analog+and+its+use+in+photoaffmity+labeling+E.+coli+and+T7+RNA+polymerases"'>Synthesis and characterization of a new photocrosslinking CTP analog and its use in photoaffmity labeling E. coli and T7 RNA polymerases</a>", Nucleic Acids Res., 21(9):2073-2079 (1993).</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hansen, C.J , et al., "<a href='http://scholar.google.com/scholar?q="A+robust+and+scalable+microfluidic+metering+method+that+allows+Protein+crystal+growth+by+free+interface+diffusion"'>A robust and scalable microfluidic metering method that allows Protein crystal growth by free interface diffusion</a>". Proc Natl Acad Sci U S A, 2002. 99 (26): p. 16531-6.</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Harding et al., "<a href='http://scholar.google.com/scholar?q="Single-molecule+detection+as+an+approach+to+rapid+DNA+sequencing%2C"'>Single-molecule detection as an approach to rapid DNA sequencing,</a>" Trends in Biotechnology, vol. 10, 1992.</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Harris, J.M., "<a href='http://scholar.google.com/scholar?q="Introducton+to+Biochemical+and+biomedical+applications+of+poly%28ethylene+glycol%29."'>Introducton to Biochemical and biomedical applications of poly(ethylene glycol).</a>" poly(ethylene glycol) chemistry, Harris, J. M., Ed.; Plenum Press: New York, 1992: p. pp. 1-14.</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Harrison et al., "<a href='http://scholar.google.com/scholar?q="Micromachining+a+miniaturized+capillary+electrophoresis-based+chemical+analysis+system+on+a+chip%2C"'>Micromachining a miniaturized capillary electrophoresis-based chemical analysis system on a chip,</a>" Science, vol. 261, pp. 895-897 (1993).</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hasan, A. et al., "<a href='http://scholar.google.com/scholar?q="Photolabile+Protecting+Groups+for+Nucleosides%3A+Synthesis+and+Photodeprotection+Rates"'>Photolabile Protecting Groups for Nucleosides: Synthesis and Photodeprotection Rates</a>", Tetrahedron, 53(12):4247-4264 (1997).</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hornbeck, L. et al., "<a href='http://scholar.google.com/scholar?q="Bistable+Defonnable+Mirror+Device"'>Bistable Defonnable Mirror Device</a>", 1988 Techllical Digest Series, vol. 8, Optical Society of America, pp. 107-110, (Jun. 1988).</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hosokawa et al., "<a href='http://scholar.google.com/scholar?q="Handling+of+Picoliter+liquid+samples+in+a+poly%28dimethylsiloxane%29-based+microfluidic+device%2C%22+Anal.+Chem.%2C+vol.+71%2C+No.+20%2C+pp.+4781-4785+%281999%29.+http%3A%2F%2Fbiospace.intota.com+%28query+%22elastomer"'>Handling of Picoliter liquid samples in a poly(dimethylsiloxane)-based microfluidic device," Anal. Chem., vol. 71, No. 20, pp. 4781-4785 (1999). http://biospace.intota.com (query "elastomer</a>") cited by Examiner E Quan, GAU 1743 in related cases). http://www.chemfinder.com (cited by Examiner E. Quan, GAU 1743, in related case).</td></tr><tr><td class="patent-data-table-td ">90</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Houseal, T. et al., "<a href='http://scholar.google.com/scholar?q="Real-time+imaging+of+single+DNA+molecules+with+fluorescence+microscopy"'>Real-time imaging of single DNA molecules with fluorescence microscopy</a>", Biophys. I., vol. 56, pp. 507-516 (Sep. 1989).</td></tr><tr><td class="patent-data-table-td ">91</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Howorka, et al., "<a href='http://scholar.google.com/scholar?q="Sequence-specific+detection+of+individual+DNA+strands+using+engineered+nanopores."'>Sequence-specific detection of individual DNA strands using engineered nanopores.</a>" Nature Biotechnology, 2001.19(7): p. 636-639.</td></tr><tr><td class="patent-data-table-td ">92</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hubner et al., "<a href='http://scholar.google.com/scholar?q="Direct+observation+of+the+triplet+lifetime+quenching+of+single+dye+molecules+by+molecular+oxygen%2C"'>Direct observation of the triplet lifetime quenching of single dye molecules by molecular oxygen,</a>" J. Chem. Physics, 115, No. 21, Dec. 1, 2001, 9619-9622.</td></tr><tr><td class="patent-data-table-td ">93</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hultman, T. et al., "<a href='http://scholar.google.com/scholar?q="Bidirectional+Solid-Phase+Sequencing+ofln+Vitro-Amplified+Plasmid+DNA"'>Bidirectional Solid-Phase Sequencing ofln Vitro-Amplified Plasmid DNA</a>", BioTechniques, vol. 10, No. 1, pp. 84-93 (1991).</td></tr><tr><td class="patent-data-table-td ">94</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hyman, E., "<a href='http://scholar.google.com/scholar?q="A+New+Method+of+Sequencing+DNA"'>A New Method of Sequencing DNA</a>", Anal. Biochem., 174:423-436 (1988).</td></tr><tr><td class="patent-data-table-td ">95</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ishii et al., "<a href='http://scholar.google.com/scholar?q="Fluorescence+resonance+energy+transfer+between+single+fluorophores+attached+to+a+coiled-coil+protein+in+aqueous+solution%2C"'>Fluorescence resonance energy transfer between single fluorophores attached to a coiled-coil protein in aqueous solution,</a>" Chemical Physics, 247, 1999, 163-173.</td></tr><tr><td class="patent-data-table-td ">96</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ishikawa, M. et al., "<a href='http://scholar.google.com/scholar?q="Single-Molecule+Detection+by+Laser-Induced+Fluorescence+Technique+with+a+Position-Sensitive+Photon-Counting+Apparatus"'>Single-Molecule Detection by Laser-Induced Fluorescence Technique with a Position-Sensitive Photon-Counting Apparatus</a>", lpn. 1. Appl. Phys., vol. 33, Part 1, No. 3A, pp. 1571-1576 (1994).</td></tr><tr><td class="patent-data-table-td ">97</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jacobs et al., "<a href='http://scholar.google.com/scholar?q="Combinatorial+chemistry-applications+oflight-directed+chemical+synthesis"'>Combinatorial chemistry-applications oflight-directed chemical synthesis</a>", TIBTech, vol. 12, pp. 19-26 (Jan. 1994).</td></tr><tr><td class="patent-data-table-td ">98</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jacobson, et al., "<a href='http://scholar.google.com/scholar?q="High-speed+separations+on+a+microchip%2C"'>High-speed separations on a microchip,</a>" Anal. Chem., vol. 66, No. 7, pp. 1114-1118 (1994).</td></tr><tr><td class="patent-data-table-td ">99</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jacobson, K. et al., "<a href='http://scholar.google.com/scholar?q="International+Workshop+on+the+application+of+fluoresecence+photobleaching+techniques+to+problems+in+cell+biology"'>International Workshop on the application of fluoresecence photobleaching techniques to problems in cell biology</a>", Workshop Summary, Federation Proceedings, vol. 42, pp. 72-79 (1983).</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7767400">US7767400</a></td><td class="patent-data-table-td patent-date-value">Feb 3, 2008</td><td class="patent-data-table-td patent-date-value">Aug 3, 2010</td><td class="patent-data-table-td ">Helicos Biosciences Corporation</td><td class="patent-data-table-td ">Enhancing resolution of sequence analysis of short DNA stretches via difined length spacers; genetic mapping and genomics</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7767441">US7767441</a></td><td class="patent-data-table-td patent-date-value">Oct 21, 2008</td><td class="patent-data-table-td patent-date-value">Aug 3, 2010</td><td class="patent-data-table-td ">Industrial Technology Research Institute</td><td class="patent-data-table-td ">Bioassay system including optical detection apparatuses, and method for detecting biomolecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7811810">US7811810</a></td><td class="patent-data-table-td patent-date-value">Jul 9, 2009</td><td class="patent-data-table-td patent-date-value">Oct 12, 2010</td><td class="patent-data-table-td ">Industrial Technology Research Institute</td><td class="patent-data-table-td ">Plurality of optical detection device for identifying a single biomolecule; simultaneous sequencing of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7888013">US7888013</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 26, 2005</td><td class="patent-data-table-td patent-date-value">Feb 15, 2011</td><td class="patent-data-table-td ">National Institute For Materials Science</td><td class="patent-data-table-td ">Method of analyzing DNA sequence using field-effect device, and base sequence analyzer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7897345">US7897345</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 13, 2009</td><td class="patent-data-table-td patent-date-value">Mar 1, 2011</td><td class="patent-data-table-td ">Helicos Biosciences Corporation</td><td class="patent-data-table-td ">Short cycle methods for sequencing polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7981604">US7981604</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 9, 2005</td><td class="patent-data-table-td patent-date-value">Jul 19, 2011</td><td class="patent-data-table-td ">California Institute Of Technology</td><td class="patent-data-table-td ">Methods and kits for analyzing polynucleotide sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8137626">US8137626</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 17, 2007</td><td class="patent-data-table-td patent-date-value">Mar 20, 2012</td><td class="patent-data-table-td ">California Institute Of Technology</td><td class="patent-data-table-td ">Fluorescence detector, filter device and related methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8187541">US8187541</a></td><td class="patent-data-table-td patent-date-value">Sep 18, 2007</td><td class="patent-data-table-td patent-date-value">May 29, 2012</td><td class="patent-data-table-td ">California Institute Of Technology</td><td class="patent-data-table-td ">Apparatus for detecting target molecules and related methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8217433">US8217433</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Jul 10, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8222040">US8222040</a></td><td class="patent-data-table-td patent-date-value">Aug 28, 2007</td><td class="patent-data-table-td patent-date-value">Jul 17, 2012</td><td class="patent-data-table-td ">Lightspeed Genomics, Inc.</td><td class="patent-data-table-td ">Nucleic acid sequencing by selective excitation of microparticles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8247849">US8247849</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Aug 21, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Two-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8262900">US8262900</a></td><td class="patent-data-table-td patent-date-value">Dec 17, 2007</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8263336">US8263336</a></td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8264014">US8264014</a></td><td class="patent-data-table-td patent-date-value">Feb 2, 2011</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8269261">US8269261</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2011</td><td class="patent-data-table-td patent-date-value">Sep 18, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8293082">US8293082</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Oct 23, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8306757">US8306757</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2010</td><td class="patent-data-table-td patent-date-value">Nov 6, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8313625">US8313625</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Nov 20, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8313639">US8313639</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Nov 20, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8317999">US8317999</a></td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td patent-date-value">Nov 27, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8349167">US8349167</a></td><td class="patent-data-table-td patent-date-value">Jun 26, 2009</td><td class="patent-data-table-td patent-date-value">Jan 8, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415176">US8415176</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415177">US8415177</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Two-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415716">US8415716</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensors with feedback circuits</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8421437">US8421437</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Apr 16, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Array column integrator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8426898">US8426898</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2010</td><td class="patent-data-table-td patent-date-value">Apr 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8426899">US8426899</a></td><td class="patent-data-table-td patent-date-value">Feb 2, 2011</td><td class="patent-data-table-td patent-date-value">Apr 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8432149">US8432149</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Array column integrator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8432150">US8432150</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for operating an array column integrator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8435395">US8435395</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">May 7, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8441044">US8441044</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">May 14, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for manufacturing low noise chemically-sensitive field effect transistors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8445945">US8445945</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">May 21, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Low noise chemically-sensitive field effect transistors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8450781">US8450781</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">May 28, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8455927">US8455927</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Jun 4, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array with cascoded column circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8470164">US8470164</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2009</td><td class="patent-data-table-td patent-date-value">Jun 25, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8487790">US8487790</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Jul 16, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemical detection circuit including a serializer circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8492799">US8492799</a></td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td patent-date-value">Jul 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8492800">US8492800</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">Jul 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensors with sample and hold capacitors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8502278">US8502278</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Aug 6, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically-sensitive sample and hold sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8502867">US8502867</a></td><td class="patent-data-table-td patent-date-value">Mar 19, 2010</td><td class="patent-data-table-td patent-date-value">Aug 6, 2013</td><td class="patent-data-table-td ">Lightspeed Genomics, Inc.</td><td class="patent-data-table-td ">Synthetic aperture optics imaging method using minimum selective excitation patterns</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524057">US8524057</a></td><td class="patent-data-table-td patent-date-value">Jul 28, 2011</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524487">US8524487</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array with cascoded column circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8530941">US8530941</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2011</td><td class="patent-data-table-td patent-date-value">Sep 10, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8535513">US8535513</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">Sep 17, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540865">US8540865</a></td><td class="patent-data-table-td patent-date-value">Sep 8, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540866">US8540866</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540867">US8540867</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540868">US8540868</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8558288">US8558288</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Oct 15, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8592153">US8592153</a></td><td class="patent-data-table-td patent-date-value">Sep 5, 2012</td><td class="patent-data-table-td patent-date-value">Nov 26, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for manufacturing high capacitance microwell structures of chemically-sensitive sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8632975">US8632975</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 28, 2010</td><td class="patent-data-table-td patent-date-value">Jan 21, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Nucleotide transient binding for sequencing methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8653567">US8653567</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Feb 18, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensor with lightly doped drains</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8656949">US8656949</a></td><td class="patent-data-table-td patent-date-value">Aug 14, 2007</td><td class="patent-data-table-td patent-date-value">Feb 25, 2014</td><td class="patent-data-table-td ">University Of Maryland College Park</td><td class="patent-data-table-td ">Microfluidic devices and methods of fabrication</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8673627">US8673627</a></td><td class="patent-data-table-td patent-date-value">May 24, 2010</td><td class="patent-data-table-td patent-date-value">Mar 18, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Apparatus and methods for performing electrochemical reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8748947">US8748947</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Jun 10, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors with reset switch</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8759077">US8759077</a></td><td class="patent-data-table-td patent-date-value">Aug 28, 2007</td><td class="patent-data-table-td patent-date-value">Jun 24, 2014</td><td class="patent-data-table-td ">Lightspeed Genomics, Inc.</td><td class="patent-data-table-td ">Apparatus for selective excitation of microparticles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8778609">US8778609</a></td><td class="patent-data-table-td patent-date-value">Mar 14, 2014</td><td class="patent-data-table-td patent-date-value">Jul 15, 2014</td><td class="patent-data-table-td ">Good Start Genetics, Inc.</td><td class="patent-data-table-td ">Methods for analyzing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070099212">US20070099212</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 28, 2006</td><td class="patent-data-table-td patent-date-value">May 3, 2007</td><td class="patent-data-table-td ">Timothy Harris</td><td class="patent-data-table-td ">Consecutive base single molecule sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20100330570">US20100330570</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 28, 2010</td><td class="patent-data-table-td patent-date-value">Dec 30, 2010</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Nucleotide transient binding for sequencing methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20110151449">US20110151449</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 18, 2011</td><td class="patent-data-table-td patent-date-value">Jun 23, 2011</td><td class="patent-data-table-td ">Helicos Biosciences Corporation</td><td class="patent-data-table-td ">Short cycle methods for sequencing polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20110245086">US20110245086</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 18, 2011</td><td class="patent-data-table-td patent-date-value">Oct 6, 2011</td><td class="patent-data-table-td ">Helicos Biosciences Corporation</td><td class="patent-data-table-td ">Short cycle methods for sequencing polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2160474A1?cl=en">EP2160474A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 16, 2008</td><td class="patent-data-table-td patent-date-value">Mar 10, 2010</td><td class="patent-data-table-td ">Helicos Biosciences Corporation</td><td class="patent-data-table-td ">Nucleotide analogs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2007120859A2?cl=en">WO2007120859A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 13, 2007</td><td class="patent-data-table-td patent-date-value">Oct 25, 2007</td><td class="patent-data-table-td ">Timothy D Harris</td><td class="patent-data-table-td ">Methods for increasing accuracy of nucleic acid sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011031786A2?cl=en">WO2011031786A2</a></td><td class="patent-data-table-td patent-date-value">Sep 8, 2010</td><td class="patent-data-table-td patent-date-value">Mar 17, 2011</td><td class="patent-data-table-td ">Laboratory Corporation Of America Holdings</td><td class="patent-data-table-td ">Compositions and methods for diagnosing autism spectrum disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011056420A1?cl=en">WO2011056420A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 20, 2010</td><td class="patent-data-table-td patent-date-value">May 12, 2011</td><td class="patent-data-table-td ">Infoscitex Corporation</td><td class="patent-data-table-td ">Methods for identifying nucleic acid ligands</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012012703A2?cl=en">WO2012012703A2</a></td><td class="patent-data-table-td patent-date-value">Jul 22, 2011</td><td class="patent-data-table-td patent-date-value">Jan 26, 2012</td><td class="patent-data-table-td ">Esoterix Genetic Laboratories, Llc</td><td class="patent-data-table-td ">Identification of differentially represented fetal or maternal genomic regions and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012037531A1?cl=en">WO2012037531A1</a></td><td class="patent-data-table-td patent-date-value">Sep 16, 2011</td><td class="patent-data-table-td patent-date-value">Mar 22, 2012</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Capture probes immobilizable via l-nucleotide tail</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012061412A1?cl=en">WO2012061412A1</a></td><td class="patent-data-table-td patent-date-value">Nov 1, 2011</td><td class="patent-data-table-td patent-date-value">May 10, 2012</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Integrated capture and amplification of target nucleic acid for sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012122484A1?cl=en">WO2012122484A1</a></td><td class="patent-data-table-td patent-date-value">Mar 9, 2012</td><td class="patent-data-table-td patent-date-value">Sep 13, 2012</td><td class="patent-data-table-td ">Roberto Polakiewicz</td><td class="patent-data-table-td ">Methods and reagents for creating monoclonal antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013063308A1?cl=en">WO2013063308A1</a></td><td class="patent-data-table-td patent-date-value">Oct 25, 2012</td><td class="patent-data-table-td patent-date-value">May 2, 2013</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">An enzymatic method to enrich for capped rna, kits for performing same, and compositions derived therefrom</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013104990A1?cl=en">WO2013104990A1</a></td><td class="patent-data-table-td patent-date-value">Jan 9, 2013</td><td class="patent-data-table-td patent-date-value">Jul 18, 2013</td><td class="patent-data-table-td ">Oslo Universitetssykehus Hf</td><td class="patent-data-table-td ">Methods and biomarkers for analysis of colorectal cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013116774A1?cl=en">WO2013116774A1</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2013</td><td class="patent-data-table-td patent-date-value">Aug 8, 2013</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Asymmetric hairpin target capture oligomers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013152114A1?cl=en">WO2013152114A1</a></td><td class="patent-data-table-td patent-date-value">Apr 3, 2013</td><td class="patent-data-table-td patent-date-value">Oct 10, 2013</td><td class="patent-data-table-td ">The Regents Of The University Of Michigan</td><td class="patent-data-table-td ">Biomarker associated with irritable bowel syndrome and crohn&#39;s disease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013174432A1?cl=en">WO2013174432A1</a></td><td class="patent-data-table-td patent-date-value">May 24, 2012</td><td class="patent-data-table-td patent-date-value">Nov 28, 2013</td><td class="patent-data-table-td ">Fundaci Institut D&#39;investigaci Biomdica De Bellvitge (Idibell)</td><td class="patent-data-table-td ">Method for the identification of the origin of a cancer of unknown primary origin by methylation analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2014005076A2?cl=en">WO2014005076A2</a></td><td class="patent-data-table-td patent-date-value">Jun 28, 2013</td><td class="patent-data-table-td patent-date-value">Jan 3, 2014</td><td class="patent-data-table-td ">The Regents Of The University Of Michigan</td><td class="patent-data-table-td ">Methods and biomarkers for detection of kidney disorders</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006100">435/6.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006180">435/6.18</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006130">435/6.13</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q">C12Q</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021040000">C07H21/04</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021020000">C07H21/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=F4l5BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6869">C12Q1/6869</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68E</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jun 28, 2013</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">COMPLETE GENOMICS, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">LICENSE;ASSIGNOR:FLUIDIGM CORPORATION;REEL/FRAME:030714/0686</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">FLUIDIGM CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:HELICOS BIOSCIENCES CORPORATION;REEL/FRAME:030714/0546</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20130628</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ILLUMINA, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">LICENSE;ASSIGNOR:FLUIDIGM CORPORATION;REEL/FRAME:030714/0783</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">LIFE TECHNOLOGIES CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">LICENSE;ASSIGNOR:FLUIDIGM CORPORATION;REEL/FRAME:030713/0655</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">PACIFIC BIOSCIENCES OF CALIFORNIA, INC., CALIFORNI</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">LICENSE;ASSIGNOR:FLUIDIGM CORPORATION;REEL/FRAME:030714/0598</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">SEQLL, LLC, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">LICENSE;ASSIGNOR:FLUIDIGM CORPORATION;REEL/FRAME:030714/0633</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 9, 2012</td><td class="patent-data-table-td ">FPB1</td><td class="patent-data-table-td ">Expired due to reexamination which canceled all claims</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 17, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120417</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BROWN RUDNICK LLP, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">NOTICE OF ATTORNEY S LIEN;ASSIGNOR:HELICOS BIOSCIENCES CORPORATION;REEL/FRAME:028060/0898</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 18, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">HELICOS BIOSCIENCES CORPORATION, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">RELEASE BY SECURED PARTY;ASSIGNOR:GENERAL ELECTRIC CAPITAL CORPORATION;REEL/FRAME:027549/0565</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120113</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 22, 2011</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20110127</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 22, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENERAL ELECTRIC CAPITAL CORPORATION, MARYLAND</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20101116</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:HELICOS BIOSCIENCES CORPORATION;REEL/FRAME:025388/0347</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 30, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 26, 2004</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">HELICOS BIOSCIENCES CORPORATION, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:LAPIDUS, STANLEY N.;BUZBY, PHILIP RICHARD;HARRIS, TIMOTHY;REEL/FRAME:015926/0825;SIGNING DATES FROM 20040726 TO 20040809</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 13, 2004</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">HELICOS BIOSCIENCES CORPORATION, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:LAPIDUS, STANLEY N.;BUZBY, PHILIP RICHARD;HARRIS, TIMOTHY;REEL/FRAME:015061/0804;SIGNING DATES FROM 20040726 TO 20040809</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2G7JlM-cVHabjSDrfg3ik9kpKiEA\u0026id=F4l5BAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U1Yq6qQurotR28KYZY4U5KfcfZL5g\u0026id=F4l5BAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U2bAE2brJ62c3uIIeWGD9Ds01x0dg","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Short_cycle_methods_for_sequencing_polyn.pdf?id=F4l5BAABERAJ\u0026output=pdf\u0026sig=ACfU3U2CZRpgfOibDE3fr3pdwo6LZB1qFQ"},"sample_url":"http://www.google.com/patents/reader?id=F4l5BAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>