<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7173124 - Isolation of nucleic acid - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Isolation of nucleic acid"><meta name="DC.contributor" content="Arne Helge Deggerdal" scheme="inventor"><meta name="DC.contributor" content="Frank Larsen" scheme="inventor"><meta name="DC.contributor" content="Invitrogen Dynal As" scheme="assignee"><meta name="DC.date" content="2005-9-23" scheme="dateSubmitted"><meta name="DC.description" content="The present invention provides a method of isolating nucleic acid from a sample, said method comprising contacting said sample with a detergent and a solid support, whereby soluble nucleic acid in said sample is bound to the support, and separating said support with bound nucleic acid from the sample. Where the method of the invention is used to isolate DNA, it may conveniently be couple with a further step to isolate RNA from the same sample."><meta name="DC.date" content="2007-2-6" scheme="issued"><meta name="DC.relation" content="US:5898071" scheme="references"><meta name="citation_patent_number" content="US:7173124"><meta name="citation_patent_application_number" content="US:11/234,001"><link rel="canonical" href="http://www.google.com/patents/US7173124"/><meta property="og:url" content="http://www.google.com/patents/US7173124"/><meta name="title" content="Patent US7173124 - Isolation of nucleic acid"/><meta name="description" content="The present invention provides a method of isolating nucleic acid from a sample, said method comprising contacting said sample with a detergent and a solid support, whereby soluble nucleic acid in said sample is bound to the support, and separating said support with bound nucleic acid from the sample. Where the method of the invention is used to isolate DNA, it may conveniently be couple with a further step to isolate RNA from the same sample."/><meta property="og:title" content="Patent US7173124 - Isolation of nucleic acid"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("-KjtU4XYLMbMsQT724C4DA"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("NOR"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("-KjtU4XYLMbMsQT724C4DA"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("NOR"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7173124?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7173124"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=q3x5BAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7173124&amp;usg=AFQjCNHNa8SeAQpMay5bgC1ApqvqCAN12w" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7173124.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7173124.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20060058519"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7173124"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7173124" style="display:none"><span itemprop="description">The present invention provides a method of isolating nucleic acid from a sample, said method comprising contacting said sample with a detergent and a solid support, whereby soluble nucleic acid in said sample is bound to the support, and separating said support with bound nucleic acid from the sample....</span><span itemprop="url">http://www.google.com/patents/US7173124?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7173124 - Isolation of nucleic acid</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7173124 - Isolation of nucleic acid" title="Patent US7173124 - Isolation of nucleic acid"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7173124 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 11/234,001</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Feb 6, 2007</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Sep 23, 2005</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Dec 12, 1994</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2207608A1">CA2207608A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2207608C">CA2207608C</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69533317D1">DE69533317D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0796327A2">EP0796327A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0796327B1">EP0796327B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7989614">US7989614</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8691969">US8691969</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20040215011">US20040215011</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20060058519">US20060058519</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20070190559">US20070190559</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20080300396">US20080300396</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20090068724">US20090068724</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20090149646">US20090149646</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20110251382">US20110251382</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1996018731A2">WO1996018731A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1996018731A3">WO1996018731A3</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">11234001, </span><span class="patent-bibdata-value">234001, </span><span class="patent-bibdata-value">US 7173124 B2, </span><span class="patent-bibdata-value">US 7173124B2, </span><span class="patent-bibdata-value">US-B2-7173124, </span><span class="patent-bibdata-value">US7173124 B2, </span><span class="patent-bibdata-value">US7173124B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Arne+Helge+Deggerdal%22">Arne Helge Deggerdal</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Frank+Larsen%22">Frank Larsen</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Invitrogen+Dynal+As%22">Invitrogen Dynal As</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7173124.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7173124.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7173124.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (1),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (9),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (13),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (7)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7173124&usg=AFQjCNFWt7o0Znd6bwrG72xlzNT_gg0weA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7173124&usg=AFQjCNG0lm1jYQUioeuBqLSIK_yQH7RUEQ">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7173124B2%26KC%3DB2%26FT%3DD&usg=AFQjCNG5Su6TnDUEwQqX9tXHkvCS-C-knQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT55695026" lang="EN" load-source="patent-office">Isolation of nucleic acid</invention-title></span><br><span class="patent-number">US 7173124 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA51106237" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">The present invention provides a method of isolating nucleic acid from a sample, said method comprising contacting said sample with a detergent and a solid support, whereby soluble nucleic acid in said sample is bound to the support, and separating said support with bound nucleic acid from the sample. Where the method of the invention is used to isolate DNA, it may conveniently be couple with a further step to isolate RNA from the same sample.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(7)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7173124B2/US07173124-20070206-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7173124B2/US07173124-20070206-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7173124B2/US07173124-20070206-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7173124B2/US07173124-20070206-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7173124B2/US07173124-20070206-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7173124B2/US07173124-20070206-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7173124B2/US07173124-20070206-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(29)</span></span></div><div class="patent-text"><div mxw-id="PCLM9158428" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A method of isolating genomic DNA from a sample, said method comprising (a) contacting said sample with a detergent and a solid support in the absence of any chaotropic agent, the solid support comprising an organic polymer, whereby soluble genomic DNA in said sample is bound to the support in a sequence-independent manner in the presence of the detergent and absence of any chaotropic agent, and (b) separating said support with bound genomic DNA from the sample.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, further comprising disrupting or lysing structural components or cells in the sample prior to the contacting step.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the detergent is anionic.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. A method as claimed in <claim-ref idref="CLM-00003">claim 3</claim-ref>, wherein the detergent is sodium dodecyl sulphate, or another alkali metal alkylsulphate salt, or sarkosyl.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the concentration of detergent is 0.2 to 30% (w/v).</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the detergent is contained in a composition additionally comprising one or more monovalent cations, chelating agents or reducing agents.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the detergent is used in alkaline solution.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the solid support is particulate.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. A method as claimed in <claim-ref idref="CLM-00008">claim 8</claim-ref>, wherein the solid support comprises magnetic beads.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the solid support has a hydrophobic surface.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the genomic DNA is eluted from the support, following separation from the sample.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. A method as claimed in <claim-ref idref="CLM-00011">claim 11</claim-ref>, wherein the genomic DNA is eluted by heating.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00013" num="00013" class="claim">
      <div class="claim-text">13. A kit for isolating genomic DNA from a sample, the kit comprising superparamagnetic polystyrene beads and one or more detergents.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00014" num="00014" class="claim">
      <div class="claim-text">14. A kit as claimed in <claim-ref idref="CLM-00013">claim 13</claim-ref>, further comprising one or more buffers, salts, lysis agents, chelating agents and/or reducing agents.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00015" num="00015" class="claim">
      <div class="claim-text">15. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the organic polymer is polyurethane.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00016" num="00016" class="claim">
      <div class="claim-text">16. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the organic polymer is polystyrene.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00017" num="00017" class="claim">
      <div class="claim-text">17. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the organic polymer is latex.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00018" num="00018" class="claim">
      <div class="claim-text">18. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the solid support comprises superparamagnetic polystyrene beads.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00019" num="00019" class="claim">
      <div class="claim-text">19. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the solid support is porous.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00020" num="00020" class="claim">
      <div class="claim-text">20. A method as claimed in <claim-ref idref="CLM-00001">claim 1</claim-ref>, the method further comprising the step of detecting, hybridizing, amplifying or quantifying the bound genomic DNA after the separating step.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00021" num="00021" class="claim">
      <div class="claim-text">21. The method of <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein the disrupting step is effected by one or more of grinding, heating, or alkaline lysis, of the sample.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00022" num="00022" class="claim">
      <div class="claim-text">22. A kit for isolating genomic DNA from a sample, the kit comprising (a) a solid support as defined in <claim-ref idref="CLM-00001">claim 1</claim-ref>; (b) one or more detergents; and (c) instructions for isolating genomic DNA according to the method of <claim-ref idref="CLM-00001">claim 1</claim-ref>.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00023" num="00023" class="claim">
      <div class="claim-text">23. A method of isolating RNA and genomic DNA from a sample, said method comprising (a) contacting said sample with a detergent and a solid support in the absence of any chaotropic agent, the solid support comprising an organic polymer, whereby soluble genomic DNA in said sample is bound to the support in a sequence-independent manner in the presence of the detergent and absence of any chaotropic agent; (b) separating said support with bound genomic DNA from the sample; and (c) isolating RNA from said sample.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00024" num="00024" class="claim">
      <div class="claim-text">24. A kit for isolating RNA and genomic DNA from a sample, the kit comprising (a) superparamagnetic polystyrene beads; (b) oligo dT beads; and (c) one or more detergents.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00025" num="00025" class="claim">
      <div class="claim-text">25. A kit for isolating RNA and genomic DNA from a sample, the kit comprising (a) a solid support comprising an organic polymer; (b) one or more detergents; and (c) instructions for isolating RNA and genomic DNA according to the method of <claim-ref idref="CLM-00023">claim 23</claim-ref>.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00026" num="00026" class="claim">
      <div class="claim-text">26. A method of isolating genomic DNA from cells in a sample, said method comprising (a) obtaining cells from said sample by immunomagnetic separation; (b) producing a lysate by contacting said cells with a detergent and a solid support in the absence of any chaotropic agent, the solid support comprising an organic polymer, whereby soluble genomic DNA in said lysate is bound to the support in a sequence-independent manner in the presence of the detergent and absence of any chaotropic agent; and (c) separating said support with bound genomic DNA from said lysate.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00027" num="00027" class="claim">
      <div class="claim-text">27. A method as claimed in <claim-ref idref="CLM-00026">claim 26</claim-ref>, wherein said cells are in a cell:bead complex.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00028" num="00028" class="claim">
      <div class="claim-text">28. A method of isolating RNA and genomic DNA from cells in a sample, said method comprising (a) obtaining cells from said sample by immunomagnetic separation; (b) producing a lysate by contacting said cells with a detergent and a solid support in the absence of any chaotropic agent, the solid support comprising an organic polymer, whereby soluble genomic DNA in said lysate is bound to the support in a sequence-independent manner in the presence of the detergent and absence of any chaotropic agent; (c) separating said support with bound genomic DNA from said lysate; and (d) isolating RNA from said lysate.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00029" num="00029" class="claim">
      <div class="claim-text">29. A method as claimed in <claim-ref idref="CLM-00028">claim 28</claim-ref>, wherein said cells are in a cell:bead complex.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES16180972" lang="EN" load-source="patent-office" class="description">
    <p num="p-0002">The present invention relates to the isolation of nucleic acid, and especially to the isolation of DNA or RNA from cells.</p>
    <p num="p-0003">The isolation of DNA or RNA is an important step in many biochemical and diagnostic procedures. For example, the separation of nucleic acids from the complex mixtures in which they are often found is frequently necessary before other studies and procedures eg. detection, cloning, sequencing, amplification, hybridisation, cDNA synthesis etc. can be undertaken; the presence of large amounts of cellular or other contaminating material eg. proteins or carbohydrates, in such complex mixtures often impedes many of the reactions and techniques used in molecular biology. In addition, DNA may contaminate RNA preparations and vice versa. Thus, methods for the isolation of nucleic acids from complex mixtures such as cells, tissues etc. are demanded, not only from the preparative point of view, but also in the many methods in use today which rely on the identification of DNA or RNA eg. diagnosis of microbial infections, forensic science, tissue and blood typing, detection of genetic variations etc.</p>
    <p num="p-0004">In RNA identifications it is important for a conclusive diagnosis to be certain that the detected sequence is derived from an RNA molecule and not from genomic DNA contamination in the sample. For this reason, methods for the separation of RNA from DNA are important. Also, for RNA isolation rapid methods are required since RNA molecules usually are very unstable and rapidly degraded by RNases present in cells and body fluids. The quality of the RNA is probably the most important factor in determining the quality of the final results in protocols utilising mRNA, especially for cDNA synthesis. It is important to avoid DNA contamination of RNA preparations for a number of reasons. Firstly, DNA increases viscosity making sample handling difficult leading to poor RNA yield and also RNA of poor quality with the liklihood of DNA contamination. Also, DNA contamination may trap RNase enzymes and make downstream applications such as RT-PCR worthless.</p>
    <p num="p-0005">A range of methods are known for the isolation of nucleic acids, but generally speaking, these rely on a complex series of extraction and washing steps and are time consuming and laborious to perform. Moreover, the use of materials such as alcohols and other organic solvents, chaotropes and proteinases is often involved, which is disadvantageous since such materials tend to interfere with many enzymic reactions and other downstream processing applications.</p>
    <p num="p-0006">Thus, classical methods for the isolation of nucleic acids from complex starting materials such as blood or blood products or tissues involves lysis of the biological material by a detergent or chaotrope, possibly in the presence of protein degrading enzymes, followed by several extractions with organic solvents eg. phenol and/or chloroform, ethanol precipitation, centrifugations and dialysis of the nucleic acids. The purification of RNA from DNA may involve a selective precipitation with LiCl or a selective isolation with acidic guanidinium thiocyanate combined with phenol extractions and ethanol precipitation. Not only are such methods cumbersome and time consuming to perform, but the relatively large number of steps required increases the risk of degradation, sample loss or cross-contamination of samples where several samples are simultaneously processed. In the case of RNA isolation, the risk of DNA contamination is relatively high.</p>
    <p num="p-0007">In purification of RNA, it is commonly desired to specifically isolate mRNA. Most mRNA purification strategies involve isolation of total RNA and fractionation of the isolated RNA. Preparation of high-quality mRNA is an important step in the analysis of gene structure and gene regulation.</p>
    <p num="p-0008">Most eukaryotic mRNAs have a poly(A)tail, typically about 50 to 300 nucleotides long. Such mRNA is referred to as polyadenylated or poly(A)<sup>+</sup> mRNA. In separating this polyadenylated RNA from the non-adenylated RNA which accounts for 95% or more of a cell's total RNA, advantage is taken of this poly(A) tail and some type of affinity separation directed toward the poly(A) tail is performed. The conventional technology has involved purification of total RNA as a first step and selection of poly(A)<sup>+</sup> RNA by affinity chromatography using oligo(dT)-cellulose as the second step. This strategy, is rather time-consuming and labour-intensive. An alternative strategy for mRNA purification is to use oligo(dT) linked to solid supports such as microplates, latex, agarose or magnetic beads.</p>
    <p num="p-0009">Over the past four years it has become increasingly popular to employ a magnetic bead assisted strategy for poly(A)<sup>+</sup> RNA selection since such beads have proven to be favourable in mRNA manipulations. In many approaches, the yield and the quality of the products depends on how rapidly the mRNA can be purified from nucleases and other contaminants. By using the magnetic bead separation technology, pure, intact poly(A)<sup>+</sup> RNA can be obtained rapidly either from total RNA preparations or more importantly, directly from crude lysates of solid tissues, cell or body fluids. The entire procedure can be carried out in a microfuge tube without phenol extractions or ethanol precipitations.</p>
    <p num="p-0010">One approach common in RNA purification, which may be used in conjunction with the solid phase approach is to carry out the lysis of the biological material and the subsequent hybridisation to oligo dT in LiCl and LiDS/SDS buffers, thereby avoiding extra steps such as phenol extraction or proteinase-K digestion. The whole direct mRNA isolation takes approximately 15 minutes and since the mRNA is stable for more than 30 minutes in the lysis buffer, this ensures the high quality of the mRNA purified. However, a disadvantage of this method is that mRNA per weight unit of tissue is affected by the amount of tissue used and above a critical threshold of lysed cells, the yield of mRNA decreases.</p>
    <p num="p-0011">Another common approach for direct mRNA purification is, as mentioned above, to use guanidinium isothiocyanate (GTC) and sarkosyl. A GTC-buffer system is preferred by most researchers due to the ability of this chaotropic salt to inhibit RNases. This may also be used in combination with the magnetic bead approach. However, the viscosity of cell lysates in 4M GTC is high and the beads are not effectively attracted by the magnet, resulting in an increased risk for DNA contamination, both for beads and other solid phases, and lower yields.</p>
    <p num="p-0012">More recently, other methods have been proposed which rely upon the use of a solid phase. In U.S. Pat. No. 5,234,809, for example, is described a method where nucleic acids are bound to a solid phase in the form of silica particles, in the presence of a chaotropic agent such as a guanidinium salt, and thereby separated from the remainder of the sample. WO 91/12079 describes a method whereby nucleic acid is trapped on the surface of a solid phase by precipitation. Generally speaking, alcohols and salts are used as precipitants.</p>
    <p num="p-0013">Although such methods speed up the nucleic acid separation process, there are disadvantages associated with the use of alcohols, chaotropes, and other similar agents. Chaotropes require to be used at high molarity, resulting in viscous solutions which may be difficult to work with, especially in RNA work. Amplification procedures such as PCR, and other enzyme-based reactions, are very sensitive to the inhibitory or otherwise interfering effects of alcohols and other agents. Moreover, the drying of the nucleic acid pellet which is necessary following alcohol precipitation and the problems with dissolving nucleic acids, are also known to lead to artefacts in enzyme-based procedures such as PCR. Since such procedures are now a mainstay of molecular biology, there is a need for improved methods of nucleic acid isolation, and particularly for methods which are quick and simple to perform and which avoid the use of chaotropic agents or alcohol precipitation. There is also a need for a method which allows for differentiation between RNA and DNA and permits a separate isolation of both types of nucleic acid from the same sample. The present invention seeks to provide such methods.</p>
    <p num="p-0014">In particular, it has now been found that nucleic acid may be isolated from a sample in a form suitable for amplification or other downstream processes, by a simple and easy to perform procedure which involves treating the sample with detergent and allowing the nucleic acid to bind to a solid support, whereupon the nucleic acid may be readily separated from the sample, eg. by removal of the support. The binding of the nucleic acid is independent of its sequence.</p>
    <p num="p-0015">In one aspect, the present invention thus provides a method of isolating nucleic acid from a sample, said method comprising contacting said sample with a detergent and a solid support, whereby soluble nucleic acid in said sample is bound to the support, and separating said support with bound nucleic acid from the sample.</p>
    <p num="p-0016">The nucleic acid may be DNA, RNA or any naturally occurring or synthetic modification thereof, and combinations thereof. Preferably however the nucleic acid will be DNA, which may be genomic, or, cDNA, and single or double stranded or in any other form.</p>
    <p num="p-0017">Where the method of the invention is used to isolate DNA, it may conveniently be coupled with a further step to isolate RNA from the same sample. The use of the method in such two-step RNA separations will be described in more detail below.</p>
    <p num="p-0018">The samples may be any material containing nucleic acid, including for example foods and allied products, clinical and environmental samples. However, the sample will generally be a biological sample, which may contain any viral or cellular material, including all prokaryotic or eukaryotic cells, viruses, bacteriophages, mycoplasmas, protoplasts and organelles. Such biological material may thus comprise all types of mammalian and non-mammalian animal cells, plant cells, algae including blue-green algae, fungi, bacteria, protozoa etc. Representative samples thus include whole blood and blood-derived products such as plasma, serum and buffy coat, urine, faeces, cerebrospinal fluid or any other body fluids, tissues, cell cultures, cell suspensions etc.</p>
    <p num="p-0019">The sample may also include relatively pure starting materials such as a PCR product, or semi-pure preparations obtained by other nucleic acid recovery processes.</p>
    <p num="p-0020">The nucleic acid-containing sample may, generally speaking, simply be contacted with the detergent, and a solid phase which may be added to the sample prior to, simultaneously with, or subsequently to the detergent. If necessary, this may be preceded by one or more separate steps to disrupt structural components such as cell walls or to achieve lysis. Procedures for achieving this are well known in the art. Thus, for example, although some cells eg. blood cells, may be lysed by the detergent alone, other cells, eg. plant or fungal cells or solid animal tissues may require more vigorous treatment such as, for example, grinding in liquid nitrogen, heating in the presence of detergent, alkaline lysis in the presence of detergent. For samples in the form of paraffin sections and such like, lysis (and melting of the paraffin) may be effected by heating, for example using a microwave oven (Banerjee, S. K. et al., 1995, Biotechniques 18: 769–773). Also, certain more compact tissues may require enzyme treatment, for example using proteinase K to obtain sufficient release of nucleic acid. The various components are mixed and simply allowed to stand for a suitable interval of time to allow the nucleic acid to bind to the support. Conveniently, if other agents such as enzymes eg. proteinase K are being used, they may be included in with the detergent. The support is then removed from the solution by any convenient means, which will depend of course on the nature of the support, and includes all forms of withdrawing the support away from the sample supernatant, or vice versa, for example centrifugation, decanting, pipetting etc.</p>
    <p num="p-0021">The conditions during this process are not critical, and it has been found convenient, for example, simply to mix the sample with the detergent in the presence of a solid phase, and allow it to stand at room temperature, for 5 to 20 minutes, before separating. As mentioned above, the reaction time is not critical and as little as 5 minutes is often enough. However, if convenient, longer periods may be used, eg. 0.5 to 3 hours, or even overnight. Mixing can be done by any convenient means, including for example simple agitation by stirring or vortexing. Also, if desired, higher or lower temperatures may be used, but are not necessary.</p>
    <p num="p-0022">The detergent may be any detergent, and a vast range are known and described in the literature. Thus, the detergent may be ionic, including anionic and cationic, non-ionic or zwitterionic. The term “ionic detergent” as used herein includes any detergent which is partly or wholly in ionic form when dissolved in water. Anionic detergents have been shown to work particularly well and are preferred. Suitable anionic detergents include for example sodium dodecyl sulphate (SDS) or other alkali metal alkylsulphate salts or similar detergents, sarkosyl, or combinations thereof.</p>
    <p num="p-0023">Conveniently, the detergent may be used in a concentration of 0.2 to 30% (w/v), eg. 0.5 to 30%, preferably 0.5 to 15%, more preferably 1 to 10%. For anionic detergents concentrations of 1.0 to 5% eg. 0.5 to 5% have been shown to work well.</p>
    <p num="p-0024">The detergent may be supplied in simple aqueous solution, which may be alkaline or acidic, or more preferably in a buffer. Any suitable buffer may be used, including for example Tris, Bicine, Tricine, and phosphate buffers. Conveniently, a source of monovalent cations, eg. a salt, may be included to enhance nucleic acid capture, although this is not necessary. Suitable salts include chloride salts, e.g. sodium chloride, lithium chloride etc. at concentrations of 0.1 to 1M, eg. 250 to 500 mM. As mentioned above, other components such as enzymes, may also be included.</p>
    <p num="p-0025">Other optional components in the detergent composition include chelating agents eg. EDTA, EGTA and other polyamino carboxylic acids conveniently at concentrations of 1 to 50 mM etc., reducing agents such as dithiotreitol (DTT) or β-mercaptoethanol, at concentrations of for example 1 to 10 mM.</p>
    <p num="p-0026">Preferred detergent compositions may for example comprise:
</p> <ul> <li id="ul0001-0001" num="0000"> <ul> <li id="ul0002-0001" num="0026">100 mM Tris-HCl pH 7.5</li> <li id="ul0002-0002" num="0027">10 mM EDTA</li> <li id="ul0002-0003" num="0028">2% SDS or:</li> <li id="ul0002-0004" num="0029">100 mM TrisCl pH 7.5</li> <li id="ul0002-0005" num="0030">10 mM EDTA</li> <li id="ul0002-0006" num="0031">5% SDS</li> <li id="ul0002-0007" num="0032">10 mM NaCl or:</li> <li id="ul0002-0008" num="0033">100 mM TrisCl pH 7.5</li> <li id="ul0002-0009" num="0034">500 mM LiCl</li> <li id="ul0002-0010" num="0035">10 mM EDTA</li> <li id="ul0002-0011" num="0036">1% LiDS</li> </ul> </li> </ul> <p num="p-0027">The detergent functions in the method to lyse the nucleic acid containing material, eg. the cells and nuclei to release the nucleic acid. The detergent is also believed to help to disrupt the binding of proteins, eg. DNA-binding proteins, to the nucleic acid and to reduce the problem of contaminants in the sample sticking to the solid support.</p>
    <p num="p-0028">The solid support may be any of the well known supports or matrices which are currently widely used or proposed for immobilisation, separation etc. These may take the form of particles, sheets, gels, filters, membranes, fibres, capillaries, or microtitre strips, tubes, plates or wells etc.</p>
    <p num="p-0029">Conveniently the support may be made of glass, silica, latex or a polymeric material. Preferred are materials presenting a high surface area for binding of the nucleic acid. Although not wishing to be bound by theoretical considerations, it is believed that the nucleic acid binding process may be assisted by the nucleic acid “wrapping around” the support. Such supports will generally have an irregular surface and may be for example be porous or particulate eg. particles, fibres, webs, sinters or sieves. Particulate materials eg. beads are generally preferred due to their greater binding capacity, particularly polymeric beads.</p>
    <p num="p-0030">Conveniently, a particulate solid support used according to the invention will comprise spherical beads. The size of the beads is not critical, but they may for example be of the order of diameter of at least 1 and preferably at least 2 μm, and have a maximum diameter of preferably not more than 10 and more preferably not more than 6 μm. For example, beads of diameter 2.8 μm and 4.5 μm have been shown to work well.</p>
    <p num="p-0031">Monodisperse particles, that is those which are substantially uniform in size (eg. size having a diameter standard deviation of less than 5%) have the advantage that they provide very uniform reproducibility of reaction. Monodisperse polymer particles produced by the technique described in U.S. Pat. No. 4,336,173 are especially suitable.</p>
    <p num="p-0032">Non-magnetic polymer beads suitable for use in the method of the invention are available from Dyno Particles AS (Lillestrøm, Norway) as well as from Qiagen, Pharmacia and Serotec.</p>
    <p num="p-0033">However, to aid manipulation and separation, magnetic beads are preferred. The term “magnetic” as used herein means that the support is capable of having a magnetic moment imparted to it when placed in a magnetic field, and thus is displaceable under the action of that field. In other words, a support comprising magnetic particles may readily be removed by magnetic aggregation, which provides a quick, simple and efficient way of separating the particles following the nucleic acid binding step, and is a far less rigorous method than traditional techniques such as centrifugation which generate shear forces which may degrade nucleic acids.</p>
    <p num="p-0034">Thus, using the method of the invention, the magnetic particles with nucleic acid attached may be removed onto a suitable surface by application of a magnetic field eg. using a permanent magnet. It is usually sufficient to apply a magnet to the side of the vessel containing the sample mixture to aggregate the particles to the wall of the vessel and to pour away the remainder of the sample.</p>
    <p num="p-0035">Especially preferred are superparamagnetic particles for example those described by Sintef in EP-A-106873, as magnetic aggregation and clumping of the particles during reaction can be avoided, thus ensuring uniform and nucleic acid extraction. The well-known magnetic particles sold by Dynal AS (Oslo, Norway) as DYNABEADS, are particularly suited to use in the present invention.</p>
    <p num="p-0036">Functionalised coated particles for use in the present invention may be prepared by modification of the beads according to U.S. Pat. Nos. 4,336,173, 4,459,378 and 4,654,267. Thus, beads, or other supports, may be prepared having different types of functionalised surface, for example positively charged or hydrophobic. Weakly and strongly positively charged surfaces, weakly negatively charged neutral surfaces and hydrophobic surfaces eg. polyurethane-coated have been shown to work well.</p>
    <p num="p-0037">It is also possible to use solid supports which have been modified to permit the selective capture of desired cells, viruses etc. containing the nucleic acid. Thus for example, supports carrying antibodies, or other binding proteins, specific for a desired cell type may be used. This may introduce a degree of selectivity to the isolation of the nucleic acid, since only nucleic acid from a desired target source within a complex mixture may be separated. Thus for example, such a support may be used to separate and remove the desired cell type etc. from the sample, following which, the detergent is added to achieve lysis, release of the nucleic acid, and binding to the support.</p>
    <p num="p-0038">The preparation of such selective cell capture matrices is well known in the art and described in the literature.</p>
    <p num="p-0039">Likewise, the support may be provided with binding partners to assist in the selective capture of nucleic acids. For example, complementary DNA or RNA sequences, or DNA binding proteins may be used, or viral proteins binding to viral nucleic acid. The attachment of such proteins to the solid support may be achieved using techniques well known in the art.</p>
    <p num="p-0040">Although not necessary, it may be convenient to introduce one or more washing steps to the isolation method of the invention, for example following separation of the support from the sample. In the case of magnetic beads, this may conveniently be done before releasing the DNA from the beads. Any conventional washing buffers or other media may be used. Generally speaking, low to moderate ionic strength buffers are preferred eg. 10 mM Tris-HCl at pH 8.0/10 mM NaCl. Other standard washing media, eg. containing alcohols, may also be used, if desired.</p>
    <p num="p-0041">Following the separation step, and any optional washing steps which may be desired, the support carrying the nucleic acid may be transferred eg. resuspended or immersed into any suitable medium eg. water or low ionic strength buffer. Depending on the support and the nature of any subsequent processing desired, it may or may not be desirable to release the nucleic acid from the support.</p>
    <p num="p-0042">In the case of a particulate solid support such as magnetic or non-magnetic beads, this may in many cases be used directly, for example in PCR or other amplifications, without eluting the nucleic acid from the support. Also, for many DNA detection or identification methods elution is not necessary since although the DNA may be randomly in contact with the bead surface and bound at a number of points by hydrogen bonding or ionic or other forces, there will generally be sufficient lengths of DNA available for hybridisation to oligonucleotides and for amplification.</p>
    <p num="p-0043">However, if desired, elution of the nucleic acid may readily be achieved using known means, for example by heating, eg. to 65° C. for 5 to 10 minutes, and following which the support may be removed from the medium leaving the nucleic acid in solution. Such heating is automatically obtained in PCR by the DNA denaturation step preceding the cycling program.</p>
    <p num="p-0044">If it is desired to remove RNA from DNA, this may be achieved by destroying the RNA before the DNA separation step, for example by addition of an RNAase or an alkali such as NaOH.</p>
    <p num="p-0045">Alternatively, as mentioned above, the method of the invention may be used to separate sequentially DNA and RNA from the sample. It may also be used to remove DNA from a sample in an RNA purification procedure.</p>
    <p num="p-0046">Conveniently, the sequential separation may take place using two different solid phases, for example solid supports which can differentiate between DNA and RNA. Thus, such a method may comprise carrying out a first step separation to isolate DNA as described above. A further solid support can then be added to the sample to capture the RNA remaining in the sample, either by using a solid support that can bind the RNA or any remaining nucleic acid, or a solid support that can capture specific RNA molecules (eg. by carrying a complementary nucleic acid probe), or a subset of RNA molecules eg. polyadenylated RNA. In this way it is possible rapidly to isolate and separate DNA and RNA or subsets of both from the same sample. This may be useful, for example by measuring the isolated DNA to estimate the amount of cells used for RNA extraction, which will give a reference between different samples.</p>
    <p num="p-0047">However, the DNA isolation procedure of the invention may also readily be combined, as a preliminary step, with other conventional RNA purification procedures, for example DNA isolation with detergent according to invention may be carried out before a selective RNA precipitation step, for example using LiCl or before RNA separation using GTC and sarkosyl.</p>
    <p num="p-0048">In a representative procedure, the sample is lysed in the presence of detergent and the DNA is allowed to bind to a solid support, whereupon the DNA may readily be separated from the sample by removal of the support. If desired, the DNA can rapidly and easily be further handled for amplification or other downstream processes. The RNA may then be isolated. This can be by a solid phase based system as described above, including a repetition of the method of the invention, or by conventional techniques such as extractions, precipitations or affinity chromatography.</p>
    <p num="p-0049">A particularly advantageous embodiment of the invention is to use the isolation method of the invention to remove DNA from a sample prior to isolation of RNA, such that the viscosity of the lysed sample is reduced and a specific isolation of RNA molecules is favoured which again reduces or avoids the possibility for DNA contamination of the RNA. Such a method also has the advantage of being quick to perform.</p>
    <p num="p-0050">The invention is advantageously amenable to automation, particularly if particles, and especially, magnetic particles are used as the support.</p>
    <p num="p-0051">The various reactants and components required to perform the method of the invention may conveniently be supplied in kit form. Such kits represent a further aspect of the invention.</p>
    <p num="p-0052">At its simplest, this aspect of the invention provides a kit for isolating nucleic acid from a sample comprising a solid support and one or more detergents.</p>
    <p num="p-0053">Optionally included in such a kit may be buffers, salts, lysis agents eg. proteinases, chelating agents and reducing agents.</p>
    <p num="p-0054">For isolation of RNA, the kits may further comprise means for isolating RNA eg. a second solid support for isolating RNA, for example a support provided with probes for capture of RNA eg. oligo dT or probes of complementary sequence to the desired target, or a chaotrope or selective precipitating agent.</p>
    <description-of-drawings> <p num="p-0055">The invention will now be described in more detail in the following non-limiting Examples with reference to the drawings in which:</p>
      <p num="p-0056"> <figref idrefs="DRAWINGS">FIG. 1</figref> shows as optical density scan of DNA isolated as described in Example 1 (ordinate shows absorbance (OD), abscissa shows wavelength (nM)). Maximum absorbance (0.427) at 257.6 nM; minimum absorbance (0.292) at 236.4 nM; at a threshold of 0.100;</p>
      <p num="p-0057"> <figref idrefs="DRAWINGS">FIG. 2</figref> shows gel electrophoresis of a sample of DNA isolated as described in Example 1 (lane 1: isolation; lane 2: λ Hind III (molecular weight marker));</p>
      <p num="p-0058"> <figref idrefs="DRAWINGS">FIG. 3</figref> shows agarose gel electrophoresis of the PCR product of Example 2 (lane 1: PCR product; lane 2: λ Hind III; lane 3: negative PCR control); and</p>
      <p num="p-0059"> <figref idrefs="DRAWINGS">FIG. 4</figref> shows agarose gel electrophoresis of the PCR product of Example 5 (lane 1: λ Hind III; lanes 2 and 3: isolations A and B respectively; lanes 4 and 5: negative control; lane 6: λ Hind III).</p>
      <p num="p-0060"> <figref idrefs="DRAWINGS">FIG. 5</figref> show the comparison between traditionally isolated DNA and DNA isolated with Dynabeads DNA DIRECT. Panel I shows the amount of genomic DNA isolated from 10 μl of whole blood with Dynabeads DNA DIRECT including the optional elution step (lanes 1 and 2), with Dynabeads DNA DIRECT with the elution step omitted (lanes 3 and 4), and with traditional DNA isolation (lanes 5 and 6). The molecular weight marker in lane 7 is λ HindIII. Panel II shows the integrity of DNA isolated by Dynabeads DNA DIRECT. Lanes 1 and 2 show AMXY PCR from 20 ng of DNA isolated with Dynabeads DNA DIRECT from a male and female donor respectively. Lanes 4 and 5 show AMXY PCR from 200 ng DNA isolated by traditional methods from a female and a male donor respectively. Lane 3 is the negative control.</p>
      <p num="p-0061"> <figref idrefs="DRAWINGS">FIG. 6</figref> shows the reproducibility of Dynabeads DNA DIRECT. The figure shows five independent Dynabeads DNA DIRECT isolations from each of two donors. Half of the DNA obtained from 10 μl of blood is shown in the upper part of the figure, 20% of the product from PCR reactions started with 10% of the isolated DNA is shown in the lower part of the figure. Molecular weight markers are λ HindIII (lanes marked M) or 100 bp ladder (lane marked L).</p>
      <p num="p-0062"> <figref idrefs="DRAWINGS">FIG. 7</figref> shows the effect of different anticoagulants. The figure shows Dynabeads DNA DIRECT isolations from whole blood that is not anticoagulated and from blood anticoagulated with EDTA, citrate, or heparin. The two isolations from blood with the same anticoagulant were performed on blood from different donors (A or B). One quarter of the DNA obtained from 10 μl of blood is shown in the upper part of the figure, except for heparin, where half the DNA obtained from 5 μl is shown. 20% of the product from PCR reactions started with 10% of the isolated DNA is shown in the lower part of the figure, except for heparin, where 20% of the isolated DNA was used as starting material.</p>
      <p num="p-0063"> <figref idrefs="DRAWINGS">FIG. 8</figref> shows the effect of sample storage conditions. Panel I shows Dynabeads DNA DIRECT isolations from EDTA blood that has been used fresh, refrigerated for 4 days, or frozen for 4 days. The two isolations from blood with the same storage conditions were performed on blood from different donors. Half of the DNA obtained from 10 μl of blood is shown in the upper part of the figure, 20% of the product from PCR reactions started with 10% of the isolated DNA is shown in the lower part of the figure. Panel II shows Dynabeads DNA DIRECT isolations from citrate blood that has been used fresh or air dried and rehydrated. The two isolations from blood with the same storage conditions were performed on blood from different donors. Half of the DNA obtained from 10. μl of blood is shown in the upper part of the figure, 20% of the product from PCR reactions started with 10% of the isolated DNA is shown in the lower part of the figure.</p>
      <p num="p-0064"> <figref idrefs="DRAWINGS">FIG. 9</figref> shows Dynabeads DNA DIRECT from bone marrow and culture cells. Panel I shows Dynabeads DNA DIRECT isolations from 1, 2 and 5 μl of bone marrow from each of two donors. A or B above the lanes denote the identity of the donor. Half of the DNA obtained is shown in the upper part of the figure, 20% of the product from PCR reactions started with 10% of the isolated DNA is shown in the lower part of the figure. Panel II shows two Dynabeads DNA DIRECT isolations from 4×10<sup>5 </sup>Daudi cells. One tenth of the DNA obtained is shown in the upper part of the figure, 20% of the product from PCR reactions started with 1 μl of a total of 120 μl isolated DNA is shown in the lower part of the figure. The molecular weight marker is λ HindIII for the genomic DNA and 100 bp ladder for the PCR products.</p>
      <p num="p-0065"> <figref idrefs="DRAWINGS">FIG. 10</figref> shows Dynabeads DNA DIRECT from formalin fixed, paraffin embedded material. Lane A is 20% of the PCR product from a reaction started with DNA isolated by DNA DIRECT from a formalin fixed, paraffin embedded section of liver. Lane M is molecular weight marker (100 bp ladder), lane B is positive control (PCR from 20 ng human DNA), and lane C is negative control (PCR from water).</p>
      <p num="p-0066"> <figref idrefs="DRAWINGS">FIG. 11</figref> shows Dynabeads DNA DIRECT for mRNA purification. mRNA was isolated from 1 million Daudi cells per sample with Dynabeads Oligo(dT)<sub>25 </sub>after removal of DNA with Dynabeads DNA DIRECT. Increasing amounts of DNA DIRECT Dynabeads were used to remove genomic DNA; 1 mg in lane 1 and 2; 2 mg in lane 3 and 4; 5 mg in lane 5 and 6 and 10 mg in lane 7 and 8. Lane 9 and 10 are controls where no DNA was removed before direct mRNA purification. The extra bands on the top of the picture show contaminating genomic DNA in lane 9 and 10. The two strong bands in all lanes represent ribsomal RNA.</p>
      <p num="p-0067"> <figref idrefs="DRAWINGS">FIG. 12</figref> shows the results of DNA isolation and PCR amplification from (A) bacteria, (B) fungi, (C) algae and (D) plants. For all samples, DNA was isolated with 200 μl DNA DIRECT (one sample test) and 20% of the isolated DNA and 10% of the PCR products were analysed by agarose electrophoresis. For bacteria, 2.5% of the isolated DNA was used per PCR reaction, for the other samples 5% was used. 16S rRNA regions were amplified from bacterial genomic DNA and from algae chloroplast DNA. From fungi- and algae genomic DNA, 18S rDNA were amplified. Amplification of the group I intron chloroplast trnL and parts of the genomic B15C gene are shown for plants. The negative controls are PCR on samples without DNA prepared in the same way as the other reactions.</p>
    </description-of-drawings> <heading>EXAMPLE 1 </heading> <heading>DNA Isolation from Cell Culture</heading> <p num="p-0068">4×10<sup>6 </sup>HL60 cells were washed twice in PBS and pelleted. The pellet was dissolved in 10 μl PBS, and 1 mg of Dynabeads® M-280* obtainable by autoclaving a suspension of Dynabeads® M-280 tosylactivated (available from DYNAL A/S, Oslo, Norway) in water) resuspended in 0.1 ml lysis buffer [5% SDS/10 mM TrisCl pH 8.0/1 mM EDTA] was added. This was followed immediately by the addition of 1 ml lysisbuffer, and the suspension was incubated for 5 minutes at room temperature, after which the Dynabeads®, with bound DNA was attracted to a magnet and the liquid phase removed. The solid phase was then washed twice with 1 ml washing buffer [50 mM NaCl/10 mM TrisCl pH 8.0/1 mM EDTA]. Finally, the beads, with bound DNA, were resuspended in 0.1 ml water, and incubated for 5 minutes at 65° C. The beads were attracted to a magnet, and the liquid phase withdrawn. The liquid phase was then analyzed for its DNA content. Results from an optical density scan (<figref idrefs="DRAWINGS">FIG. 1</figref>) are in accordance with pure DNA. The OD<sub>260</sub>/OD<sub>280 </sub>ratio is 1.72; pure DNA in water or TE has a ratio of 1.7–1.9. With pure DNA, the concentration can be determined from the OD<sub>260 </sub>of the solution. A 50 μg/ml solution has OD<sub>260</sub>=1.0. From the OD<sub>260 </sub>measurement (Table 1) of 0.436 (0.1 ml total volume, 10 mm lightpath), the yield can be calculated to 2.18 μg DNA, 82% of the 2.67 μg that was the estimated DNA content of the starting material. Gel electrophoresis of a sample of the isolated DNA (<figref idrefs="DRAWINGS">FIG. 2</figref>) shows that most of it is in a high molecular weight form (&gt;20 kb).</p>
    <p num="p-0069">
      <tables id="TABLE-US-00001" num="00001"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">PERKIN-ELMER LAMBDA BIO UV/VIS SPECTROMETER</td>
              </tr> <tr class="description-tr"> <td class="description-td">APPLICATION NO. 3: RATIO 260/280 NM</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="70pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">WAVELENGTH</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">SAMPLE</td>
                <td class="description-td">CYCLE</td>
                <td class="description-td">AUTOZERO</td>
                <td class="description-td">DATA</td>
                <td class="description-td">UNIT</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">004</td>
                <td class="description-td">15:50</td>
                <td class="description-td">260.0 nm</td>
                <td class="description-td">0.436</td>
                <td class="description-td">ABS</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">15:56</td>
                <td class="description-td">280.0 nm</td>
                <td class="description-td">0.253</td>
                <td class="description-td">ABS</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">RATIO</td>
                <td class="description-td">1.723</td>
                <td class="description-td">RAT</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>EXAMPLE 2</heading> <heading>Isolation of DNA from Whole Blood and Enzymatic Application without Elution</heading> <p num="p-0070">5 μl whole blood (EDTA blood) was lysed in 50 μl 5% SDS and 50 μg Dynabeads® M-280* in 5 μl of PBS was added. The lysate was incubated for 1 minute at room temperature before 0.5 ml TrisCl pH 7.5 was added. The lysate was then incubated for 1 minute further at room temperature before the beads with bound DNA were attracted to a magnet and the liquid phase removed. The beads were then washed once with 0.5 ml 10 mM TrisCl pH 7.5, before the beads with bound DNA was resuspended in 40 μl TE (10 mM TrisCl pH 8.0/1 mM EDTA). 4 μl of the isolation was used in starting material for PCR (GAPDH PCR as described in Example 7). The PCR reaction gave large amounts of product, as visualised on agarose gel electrophoresis (<figref idrefs="DRAWINGS">FIG. 3</figref>). 10 μl of a 50 μl PCR reaction was loaded on the gel.</p>
    <heading>EXAMPLE 3</heading> <p num="p-0071">Example 1 was repeated using the following combination of lysisbuffers and washing buffers, and the following results were obtained:</p>
    <p num="h-0006">(where +++ indicates very good DNA isolation)</p>
    <p num="p-0072"> <tables id="TABLE-US-00002" num="00002"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="105pt" align="left"> </colspec> <colspec colname="2" colwidth="77pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> <tr class="description-tr"> <td class="description-td">Lysis buffer</td> <td class="description-td">Washing buffer</td> <td class="description-td">Result</td> </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">2% SDS</td> <td class="description-td"> 50 mM NaCl/1 × TE</td> <td class="description-td">+++</td> </tr> <tr class="description-tr"> <td class="description-td">2% SDS/1 × TE</td> <td class="description-td"> 50 mM NaCl/1 × TE</td> <td class="description-td">+++</td> </tr> <tr class="description-tr"> <td class="description-td">2% SDS/1 × TE/10 mM NaCl</td> <td class="description-td"> 50 mM NaCl/1 × TE</td> <td class="description-td">+++</td> </tr> <tr class="description-tr"> <td class="description-td">5% SDS</td> <td class="description-td"> 50 mM NaCl/1 × TE</td> <td class="description-td">+++</td> </tr> <tr class="description-tr"> <td class="description-td">5% SDS/1 × TE</td> <td class="description-td"> 50 mM NaCl/1 × TE</td> <td class="description-td">+++</td> </tr> <tr class="description-tr"> <td class="description-td">5% SDS/1 × TE/10 mM NaCl</td> <td class="description-td"> 50 mM NaCl/1 × TE</td> <td class="description-td">+++</td> </tr> <tr class="description-tr"> <td class="description-td">1% LiDS/10 × TE/0.5 M LiCl</td> <td class="description-td"> 50 mM NaCl/1 × TE</td> <td class="description-td">+++</td> </tr> <tr class="description-tr"> <td class="description-td">1% LiDS/10 × TE/0.5 M LiCl</td> <td class="description-td">150 mM LiCl/1 × TE</td> <td class="description-td">+++</td> </tr> <tr class="description-tr"> <td class="description-td">5% LiDS</td> <td class="description-td">150 mM LiCl/1 × TE</td> <td class="description-td">+++</td> </tr> <tr class="description-tr"> <td class="description-td">5% SDS</td> <td class="description-td">150 mM LiCl/1 × TE</td> <td class="description-td">+++</td> </tr> <tr class="description-tr"> <td class="description-td">1% Sarcosyl</td> <td class="description-td">150 mM LiCl/1 × TE</td> <td class="description-td">+++</td> </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="left" id="FOO-00001" class="description-td" colspan="3">1 × TE is 10 mM TrisCl pH 8.0/1 mM EDTA,</td> </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="left" id="FOO-00002" class="description-td" colspan="3">10 × TE is 100 mM TrisCl pH 8.0/10 mM EDTA</td> </tr> </tbody> </tgroup> </table> </tables> <br>
1×TE is 10 mM TrisCl pH 8.0/1 mM EDMTA, 10×TE is 100 mM TrisCl pH 8.0/10 mM EDTA
</p>
    <heading>EXAMPLE 4</heading> <p num="p-0073">Following the procedure of Example 1, similar results may be achieved using Dynabeads® M-450 uncoated (Dynal A/S, Oslo, Norway)</p>
    <heading>EXAMPLE 5 </heading> <p num="p-0074">Isolation of DNA from CD2 Positive Cells Obtained from Blood with Immunomagnetic Separation</p>
    <p num="p-0075">This experiment consisted of two identical isolations. 50 μl blood was mixed with 50 μl PBS [150 mM NaCl/10 mM NaH<sub>2</sub>PO<sub>4</sub>/Na<sub>2</sub>HPO<sub>4</sub>, pH 7.4] and 10 μl (4×10<sup>6 </sup>beads) Dynabeads® M-450 Pan-T (CD2) (available from Dynal AS, Oslo, Norway). The mixture was then incubated for 30 minutes at room temperature with gentle tilting and rotation. The cell/beads complex was attracted to a magnet and the fluid withdrawn. The cell/beads complex was then washed four times in 200 μl PBS, before 200 μg Dynabeads® M-280* (as above) and 200 μl lysisbuffer [100 mM Tris-HCl, pH 8.0/500 mM LiCl/10 mM EDTA, pH 8.01/1% LiDS] was added. The mixture was incubated for 5 minutes at room temperature, before the DNA/beads complex was attracted to a magnet, and the supernatant withdrawn. The DNA/beads complex was washed twice with 200 μl washing buffer [10 mM Tris-Hcl, pH 8.0/150 mM LiCl/1 mM EDTA, pH 8.0] and resuspended in 50 μl water. After 5 minutes at 65° C., the beads were attracted to a magnet and the supernatant transferred to a new tube. 5 μl of the supernatant was used as template for polymerase chain reaction (GAPDH PCR as described in Example 7), which gave large amounts of product, as visualised on agarose gel electrophoresis (<figref idrefs="DRAWINGS">FIG. 4</figref>).</p>
    <heading>EXAMPLE 6</heading> <heading>Comparison of Yield and Integrity Between DNA Isolated by a Traditional Method and the Present Method</heading> <p num="p-0076">Using a traditional method based on phenol extraction and ethanol precipitation, genomic DNA was isolated from 5 ml of EDTA anticoagulated blood. Four isolations from 10 μl of the same blood sample were performed using Dynabeads DNA DIRECT (kit, commercially available from Dynal AS, Oslo, Norway, containing beads equivalent to Dynabeads® M-280* as described in Example 1). The DNA from two of the isolations was eluted for 5 minutes at 65° C., while the DNA from the other two isolations was left in the presence of the Dynabeads. All the DNA from the four Dynabeads DNA DIRECT isolations was loaded onto an agarose gel, as was 0.2% of traditionally isolated DNA. The fraction of the traditionally isolated DNA loaded corresponds to the yield from 10 μl of blood (0.2% of 5 ml).</p>
    <p num="p-0077">Traditional DNA isolation was performed according to the method of John and coworkers (John, S. W. M., G. Weitzner, R. Rosen and C. R. Scriver. 1991. A Rapid Procedure for Extracting Genomic DNA from Leukocytes. Nucl. Acid. Res. 19(2):408).</p>
    <p num="p-0078">With Dynabeads DNA DIRECT, lysis of the blood was obtained by mixing 200 μl (one sample test) of Dynabeads DNA DIRECT with 10 μl of blood in a 1.5 ml microcentrifuge tube (200 μg uncoated Dynabeads in Lysis/binding buffer). Lysates were then left on the bench at room temperature for 5 minutes to allow adsorption of genomic DNA to the Dynabeads.</p>
    <p num="p-0079">The DNA/Dynabeads complex was attracted to a magnet (Dynal's Magnetic Particle Collector E (MPC-E)), and the lysate was aspirated and discarded.</p>
    <p num="p-0080">The complex was then washed twice in washing buffer (one of the kit components) by attracting it to a Dynal MPC and discarding the supernatant. Finally, the complex was resuspended in 10 μl of TE pH 8.0 (provided in the kit).</p>
    <p num="p-0081">When elution was performed, it consisted of heating the suspension to 65° C. for five minutes and then attracting the beads to a magnet. The DNA-containing aqueous phase was then withdrawn and used for the experiments.</p>
    <p num="p-0082">The DNA was visualised on ethidium bromide stained 1.5% agarose gels. Electrophoresis was performed in 1×TAE buffer, and the results were documented with a DS34 Polaroid camera and Polaroid 667 film.</p>
    <p num="p-0083">The result of this experiment is shown in panel I of <figref idrefs="DRAWINGS">FIG. 5</figref>. The yield per μl blood is similar with the two methods (lane 1-4 vs lane 5-6), and very little DNA is lost during the elution step (lane 1-2 vs lane 3-4). The molecular weight of the DNA from both methods is more than 20 kb, as it runs slower than the 23.13 kb band of the λ HindIII molecular weight marker.</p>
    <p num="p-0084">DNA DIRECT was used to isolate DNA from ACD blood from two different donors, one male and one female. From each of the isolations, 10% was used as starting material for PCR amplification of an amplicon in the X-Y homologous amelogenin (AMXY) gene (Akane, A., K, Matsubara, H. Nakamura, S. Takahashi and K. Kimura. 1994. Purification of Highly Degraded DNA by Gel Filtration for PCR. BioTechniques 16 (2):235–238), as was 200 ng from each of two traditional DNA isolations.</p>
    <p num="p-0085">DNA DIRECT and traditional DNA isolation was performed as described in the first part of this example.</p>
    <p num="p-0086">All PCR reactions were performed in a 50 μl reaction volume, 10×PCR buffer (Perkin Elmer) was added to a final concentration of 1×, dNTPs (Pharmacia) were added to a final concentration of 0.2 mM, and 1 unit of amplitaq (Perkin Elmer) was used per reaction. 5 pmol each of primers AMXY-1F (5′-CTGATGGTTGGCCTCAAGCCT-GTG-3′) and AMXY-4R (5′-TTCATTGTAAGAGCAAAGCAAACA-3′) were added per reaction. PCR was performed on a Perkin Elmer GeneAmp PCR System 9600. PCR conditions for the AMXY amplicon were 4 min at 94° C., 38×[30 sec at 94° C., 30 sec at 55° C., 1 min at 72° C.], 10 min at 72° C.</p>
    <p num="p-0087">10 μl of the 50 μl PCR reactions were visualised on ethidium bromide stained 1.5% agarose gels. Electrophoresis was performed in 1×TAE buffer, and the results were documented with a DS34 Polaroid camera and Polaroid 667 film.</p>
    <p num="p-0088">The results of this experiment are shown in panel II of <figref idrefs="DRAWINGS">FIG. 5</figref>. The X-Y homologous amelogenin gene is known to be sensitive to DNA degradation (Akane et al 1994, supra). With increasing degradation, the 908 bp long X band gets progressively weaker as compared to the 719 bp long Y band. From panel II of <figref idrefs="DRAWINGS">FIG. 5</figref> it is apparent that the relative strength of the X and Y bands is comparable for DNA isolated with Dynabeads DNA DIRECT and the traditional method, indicating that the degree of degradation is the same with the two methods. The PCR reactions from traditionally isolated DNA gives somewhat more product than does the reactions from DNA DIRECT isolated DNA. The reason for this is that ten times more template is used in the PCR reactions from traditionally isolated DNA than in the PCR reactions from DNA DIRECT isolated DNA.</p>
    <p num="p-0089">
      <tables id="TABLE-US-00003" num="00003"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="left"> </colspec> <colspec colname="2" colwidth="112pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Lysis/binding buffer:</td>
                <td class="description-td"> 0.5 M LiCl</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> 1% LiDS</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> 0.1 M TrisCl pH 7.5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  10 mM EDTA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  5 mM dithiothreitol (DTT)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Washing buffer:</td>
                <td class="description-td">0.15 M LiCl</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  10 mM Tris-HCl pH 8.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  1 mM EDTA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>EXAMPLE 7</heading> <heading>Five Independent DNA Isolations from Blood Samples from each of Two Donors</heading> <p num="p-0090">For this experiment, we used Dynabeads DNA DIRECT kit, which is commercially available from Dynal AS, Oslo, Norway. Buffer compositions are as described in example 6.</p>
    <p num="p-0091">Five DNA isolations were performed from each of two citrate treated blood samples of relatively low white blood cell counts (sample A: 3.6×10<sup>6 </sup>cells/ml, sample B: 2.6×10<sup>6 </sup>cells/ml).</p>
    <p num="p-0092">Lysis of the blood was obtained by mixing 200 μl (one sample test) of Dynabeads DNA DIRECT with blood in a 1.5 ml microcentrifuge tube. Lysates were then left on the bench at room temperature for 5 minutes to allow adsorption of genomic DNA to the Dynabeads.</p>
    <p num="p-0093">The DNA/Dynabeads complex was attracted to a magnet (Dynal's Magnetic Particle Collector E (MPC-E)), and the lysate was aspirated and discarded.</p>
    <p num="p-0094">The complex was then washed twice in washing buffer (one of the kit components) by attracting it to a Dynal MPC and discarding the supernatant. Finally, the complex was resuspended in 40 μl of TE pH 8.0 (provided in the kit). This resuspension was used for PCR and gel electrophoresis without any elution.</p>
    <p num="p-0095">From each isolation, 10% was used as starting material for PCR amplification of an amplicon in the glyceraldehyde phosphate dehydrogenase (GAPDH) gene. All PCR reactions were performed in a 50 μl reaction volume, 10×PCR buffer (Perkin Elmer) was added to a final concentration of 1×, dNTPs (Pharmacia) were added to a final concentration of 0.2 mM, and 1 unit of amplitaq (Perkin Elmer) was used per reaction. 5 pmol each of primers GAPDH-Forward (5′-ACAGTCCATGCCATCACTGCC-3′) and GAPDH-Reverse (5′-GCCTGCTTCACCACCCTTG-3′) were added per reaction. PCR was performed on a Perkin Elmer GeneAmp PCR System 9600. PCR conditions for the GAPDH amplicon were 4 min at 94° C., 34×[30 sec at 94° C., 30 sec at 61° C., 1 min at 72° C.], 10 min at 72° C.</p>
    <p num="p-0096">Both genomic DNA and PCR products were visualised on ethidium bromide stained 1.5% agarose gels. 10 μl of the 50μl reaction was loaded on to an agarose gel, as was 50% of the isolated genomic DNA. Electrophoresis was performed in 1×TAE buffer, and the results were documented with a DS34 Polaroid camera and Polaroid 667 film.</p>
    <p num="p-0097">The results of this experiment are shown in <figref idrefs="DRAWINGS">FIG. 6</figref>. No significant variation between the different isolations can be observed. Similar results were obtained with other coagulants as well as from donors with higher white blood cell counts.</p>
    <heading>EXAMPLE 8</heading> <heading>DNA Isolation from Blood with Different Anticoagulants</heading> <p num="p-0098">Dynabeads DNA DIRECT (kit, commercially available from Dynal AS, Oslo, Norway) was used to isolate DNA from untreated whole blood as well as blood anticoagulated with EDTA, Citrate or Heparin. From each type of starting material, two separate isolations were performed, with blood from different donors. The buffer components in the kit are as described in example 6.</p>
    <p num="p-0099">Lysis of the DNA containing cells from blood was obtained by mixing 200 μl (one sample test) of Dynabeads DNA DIRECT with 5 μl Heparin blood or 10 μl of other blood samples in a 1.5 ml microcentrifuge tube. Lysates were then left on the bench at room temperature for 5 minutes to allow adsorption of genomic DNA to the Dynabeads.</p>
    <p num="p-0100">The DNA/Dynabeads complex was attracted to a magnet (Dynal's Magnetic Particle Collector E (MPC-E)), and the lysate was aspirated and discarded.</p>
    <p num="p-0101">The complex was then washed twice in washing buffer (one of the kit components) by attracting it to a Dynal MPC and discarding the supernatant. Finally, the complex was resuspended in 20–40 μl of TE pH 8.0 (provided in the kit). We used 40 μl as the standard volume, but 20 μl if the starting material was Heparin blood.</p>
    <p num="p-0102">From each isolation, 10% (20% from the Heparin samples) was used as starting material for PCR amplification of an amplicon in the glyceraldehyde phosphate dehydrogenase (GAPDH) gene. PCR was performed directly on the suspension in TE, with the Dynabeads present. All PCR reactions were performed in a 50 μl reaction volume, 10×PCR buffer (Perkin Elmer) was added to a final concentration of 1×, dNTPs (Pharmacia) were added to a final concentration of 0.2 mM, and 1 unit of amplitaq (Perkin Elmer) was used per reaction. 5 pmol each of primers GAPDH-Forward (5′-ACAGTCCATGCCATCACTGCC-3′) and GAPDH-Reverse (5′-GCCTGCTTCACCACCTTCTTG-3′) were added per reaction. PCR was performed on a Perkin Elmer GeneAmp PCR System 9600. PCR conditions for the GAPDH amplicon were 4 min at 94° C., 34×[30 sec at 94° C., 30 sec at 61° C., 1 min at 72° C.], 10 min at 72° C.</p>
    <p num="p-0103">10 μl of the 50 μl reaction was loaded on to an agarose gel, as was 25% (50% from the Heparin samples) of the isolated genomic DNA. Both genomic DNA and PCR products were visualised on ethidium bromide stained 1.5% agarose gels. Electrophoresis was performed in 1×TAE buffer, and the results were documented with a DS34 Polaroid camera and Polaroid 667 film.</p>
    <p num="p-0104">The results of this experiment are shown in <figref idrefs="DRAWINGS">FIG. 7</figref>. As the isolations from Heparinized samples were from only 5 μl of blood, using 20% of the DNA from these isolations as starting material for PCR is comparable to using 10% from the other isolations, that are all from 10 μl blood. When this is taken into consideration, it is apparent that the type of anticoagulant used does not significantly affect the result.</p>
    <p num="p-0105">In the experiment just described, Lithium Heparin was used. In this system, similar results are obtained with Lithium and Sodium Heparin, even though Lithium Heparin has been shown to have inhibitory effects in other systems (Panaccio, M., M. Georgesz and A. M. Lew. 1993. FOLT PCR: A Simple PCR Protocol for Amplifying DNA Directly from Whole Blood. BioTechniques 14(3): 238–243). DNA DIRECT also performs well on blood anticoagulated with ACD (panel II of <figref idrefs="DRAWINGS">FIG. 5</figref>) or CPD (data not shown).</p>
    <heading>EXAMPLE 9</heading> <heading>Isolation of DNA from Blood Samples Stored under Different Conditions</heading> <p num="p-0106">Dynabeads DNA DIRECT (kit, commercially available from Dynal AS, Oslo, Norway) was used to isolate DNA from EDTA blood from two different donors: What was remaining of the blood samples were then divided into two, one part that was stored at +4° C. and one that was stored at −20° C. After 4 days, the frozen samples were thawed, and DNA was isolated from both the frozen samples and the samples that had been kept at +4° C. The buffer components of the kit are as described in example 6.</p>
    <p num="p-0107">Lysis of the blood was obtained by mixing 200 μl (one sample test) of Dynabeads DNA DIRECT with blood in a 1.5 ml microcentrifuge tube. Lysates were then left on the bench at room temperature for 5 minutes to allow adsorption of genomic DNA to the Dynabeads.</p>
    <p num="p-0108">The DNA/Dynabeads complex was attracted to a magnet (Dynal's Magnetic Particle Collector E (MPC-E)), and the lysate was aspirated and discarded.</p>
    <p num="p-0109">The complex was then washed twice in washing buffer (one of the kit components) by attracting it to a Dynal MPC and discarding the supernatant. Finally, the complex was resuspended in 40 μl of TE pH 8.0 (provided in the kit). Both PCR and agarose gel electrophoresis was performed directly on the suspension in TE, with the Dynabeads present.</p>
    <p num="p-0110">From each of the 6 isolations (fresh, refrigerated, and frozen), 10% was used as starting material for PCR amplification of the GAPDR amplicon as described in example 8. Both genomic DNA and PCR products were visualised on ethidium bromide stained 1.5% agarose gels. 10 μl of the 50 μl reaction was loaded on the gel, as was 50% of the isolated genomic DNA. Electrophoresis was performed in 1×TAE buffer, and the results were documented with a DS34 Polaroid camera and Polaroid 667 film. The results of this experiment are shown in panel I of <figref idrefs="DRAWINGS">FIG. 8</figref>. No adverse effect of 4 days storage at +4 or −20° C. was observed in this system.</p>
    <p num="p-0111">Using Dynabeads DNA DIRECT as described earlier in this example, DNA was isolated from two citrate treated blood samples, and from the same two samples 10 μl was spotted on a plastic surface and allowed to air dry at room temperature. The dried blood spots were transferred to 1.5 ml tubes, 40 μl PBS was added, and the tubes were left at room temperature with gentle agitation for 90 min, before DNA was isolated with Dynabeads DNA DIRECT. From each of the 4 isolations (fresh and dried), 10% was used as starting material for PCR amplification of the GAPDH amplicon as described in example 8. Both genomic DNA and PCR products were visualised on ethidium bromide stained 1.5% agarose gels. 10 μl of the 50 μl reaction was loaded on the gel, as was 50% of the isolated genomic DNA. Electrophoresis was performed in 1×TAE buffer, and the results were documented with a DS34 Polaroid camera and Polaroid 667 film. The results of this experiment are shown in panel II of <figref idrefs="DRAWINGS">FIG. 8</figref>. The yield from dried blood is good and the isolated DNA is suitable for PCR.</p>
    <heading>EXAMPLE 10</heading> <heading>DNA Isolation from Bone Marrow and Culture Cells</heading> <p num="h-0019">DNA Isolations from Bone Marrow</p>
    <p num="p-0112">1, 2, and 5 μl of heparinized bone marrow from each of two healthy donors were used as starting material for DNA isolation with DNA DIRECT. The buffer components are as described in example 6. Lysis of the bone marrow was obtained by mixing 200 μl (one sample test) of Dynabeads DNA DIRECT with 1–5 μl of heparinized bone marrow in a 1.5 ml microcentrifuge tube. Lysates were then left on the bench at room temperature for 5 minutes to allow adsorption of genomic DNA to the Dynabeads.</p>
    <p num="p-0113">The DNA/Dynabeads complex was attracted to a magnet (Dynal's Magnetic Particle Collector E(MPC-E)), and the lysate was aspirated and discarded.</p>
    <p num="p-0114">The complex was then washed twice in washing buffer (one of the kit components) by attracting it to a Dynal MPC and discarding the supernatant. Finally, the complex was resuspended in 40 μl of TE pH 8.0 (provided in the kit). Both PCR and agarose gel electrophoresis was performed directly on the suspension in TE, with the Dynabeads present.</p>
    <p num="p-0115">From each of the 6 isolations, 10% was used as starting material for PCR amplification of the GAPDH amplicon as is described in example 8.</p>
    <p num="p-0116">Both genomic DNA and PCR products were visualised on ethidium bromide stained 1.5% agarose gels. 10 μl of the 50 μl reaction was loaded on to an agarose gel, as was 50% of the isolated genomic DNA. Electrophoresis was performed in 1×TAE buffer, and the results were documented with a DS34 Polaroid camera and Polaroid 667 film.</p>
    <p num="p-0117">The results of this experiment are shown in panel I of <figref idrefs="DRAWINGS">FIG. 9</figref>. It is apparent from panel I of <figref idrefs="DRAWINGS">FIG. 9</figref> that the yield per volume starting material is higher from bone marrow than from blood (<figref idrefs="DRAWINGS">FIGS. 5–8</figref>). This is to be expected, since the concentration of DNA containing cells is much higher in bone marrow than in blood. 5 μl of bone marrow is close to the upper limit of what can be handled by 1 sample test of DNA DIRECT. A good correlation between sample size and DNA yield is observed in the 1 to 5 μl sample size interval, but even the yield from 1 μl is sufficient for at least 10 PCR reactions.</p>
    <p num="h-0020">DNA Isolation from Cultured Cells</p>
    <p num="p-0118">Two samples of 4×10<sup>5 </sup>Daudi cells were used as starting material for DNA isolation with DNA DIRECT. DNA isolation from 4×10<sup>5 </sup>cultured cells (cell line Daudi) was performed as described above, except that 1 ml (five sample tests) of Dynabeads DNA DIRECT was used. Accordingly, the washing steps were performed in 1 ml washing buffer. The DNA/Dynabeads complex was resuspended in 120 μl TE, and as for bone marrow, no elution step was performed after the resuspension.</p>
    <p num="p-0119">From each of the isolations, 1 μl of a total of 120 μl was used as starting material for PCR amplification of the GAPDH amplicon, as described in Example 8.</p>
    <p num="p-0120">Both genomic DNA and PCR products were visualised on ethidium bromide stained 1.5% agarose gels. 10 μl of the 50 μl reaction was loaded on to an agarose gel, as was 10% of the isolated genomic DNA. Electrophoresis was performed in 1×TAE buffer, and the results were documented with a DS34 Polaroid camera and Polaroid 667 film.</p>
    <p num="p-0121">The results of this experiment are shown in panel II of <figref idrefs="DRAWINGS">FIG. 9</figref>, demonstrating that at least 120 PCR reactions may be run from an isolation of this scale.</p>
    <heading>EXAMPLE 11</heading> <heading>Isolation of DNA from a Formalin Fixed, Paraffin Embedded Section of Liver</heading> <p num="p-0122">Dynabeads DNA DIRECT (kit, commercially available from Dynal AS, Oslo, Norway) was used to isolate DNA from a formalin fixed, paraffin embedded section of liver. The buffer components of the kit and the bead concentration are as described in example 6.</p>
    <p num="p-0123">Excess paraffin was removed from the edges of the sample with a scalpel blade. Lysis of the sample was obtained by adding 200 μl (one sample test) of Dynabeads DNA DIRECT to the sample in a 1.5 ml microcentrifuge tube. Lysates were then left on the bench at room temperature for 5 minutes to allow adsorption of genomic DNA to the Dynabeads. The lysate, containing DNA and Dynabeads was transferred to a fresh tube, leaving cell debris and paraffin behind.</p>
    <p num="p-0124">The DNA/Dynabeads complex was attracted to a magnet (Dynal's Magnetic Particle Collector E (MPC-E)), and the lysate was aspirated and discarded.</p>
    <p num="p-0125">The complex was then washed twice in washing buffer (one of the kit components) by attracting it to a Dynal MPC and discarding the supernatant. Finally, the complex was resuspended in 10 μl of sterile water. This suspension, with the Dynabeads present, was used as starting material for PCR amplification of the GAPDH amplicon as described in example 8. The PCR product was visualised on an ethidium bromide stained 1.5% agarose gel. 10 μl of the 50 μl reaction was loaded on the gel. Electrophoresis was performed in 1×TAE buffer, and the results were documented with a DS34 Polaroid camera and Polaroid 667 film. The result of this experiment is shown in <figref idrefs="DRAWINGS">FIG. 10</figref>. PCR amplifiable material has clearly been obtained from the formalin fixed, paraffin embedded section of liver.</p>
    <heading>EXAMPLE 12</heading> <heading>Removal of Genomic DNA with DNA DIRECT Prior to mRNA Isolation</heading> <p num="p-0126">mRNA was isolated from 1 million Daudi cells per sample. The cells were lysed in 0.75 ml Lysis/binding buffer with DNA DIRECT Dynabeads present in the buffer. The samples were incubated for 5 minutes and the DNA-Dynabead complexes were collected by applying a Dynal MPC-E magnet for 2 minutes. Different amounts of DNA DIRECT beads were used to remove genomic DNA; 1, 2, 5 and 10 mg per sample (<figref idrefs="DRAWINGS">FIG. 11</figref>).</p>
    <p num="p-0127">The lysate from each sample was transferred to new tubes with 1 mg Dynabeads Oligo(dT)<sub>25 </sub>according to standard procedure (Dynals mRNA DIRECT kit protocol). The Dynabeads were mixed with the lysate to capture the polyadenylated mRNA by hybridisation for 5 minutes at room temperature. The mRNA-Dynabead complexes were collected with the MPC-E magnet by placing the tubes in the magnetic stand for 2 minutes. The solution was removed and discarded. Washing solution with LiDS (0.75 ml) was added and the beads were washed thoroughly by pipetting up and down. The mRNA-Dynabead complexes were collected with the magnet, and the washing procedure was repeated once with washing buffer with LiDS and twice with washing buffer without detergent. Finally, the purified mRNA was eluted from the Dynabeads in 20 μl 5 mM Tris-HCl pH 7.5 buffer, by incubation at 65° C. for 2 minutes. The eluates were analysed by non-denaturing gel electrophoresis in a 1.0 t agarose gel with ethidium bromide. <figref idrefs="DRAWINGS">FIG. 11</figref> shows the results from this experiment.</p>
    <p num="p-0128">The EtBr-staining reveals both double-stranded DNA and rRNA due to secondary structure. The two ribosomal RNA bands are clearly visible in all lanes. In Lane 9–10 in <figref idrefs="DRAWINGS">FIG. 11</figref>, some genomic DNA is present as contamination after the mRNA purification procedure. In the recommended mRNA procedure from DYNAL, a DNA-shearing step is introduced after cell lysis, to reduce the possibility of DNA contamination. By using DNA removal with DNA DIRECT beads this shearing step is not necessary.</p>
    <p num="p-0129">
      <tables id="TABLE-US-00004" num="00004"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="98pt" align="left"> </colspec> <colspec colname="2" colwidth="105pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Lysis/binding buffer:</td>
                <td class="description-td"> 0.5 M LiCl</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> 1% LiDS</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> 0.1 M TrisCl pH 7.5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  10 mM EDTA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  5 mM dithiothreitol (DTT)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Washing buffer with LiDS:</td>
                <td class="description-td">0.15 M LiCl</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">0.1% LiDS</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  10 mM Tris-HCl pH 8.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  1 mM EDTA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Washing buffer:</td>
                <td class="description-td">0.15 M LiCl</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  10 mM Tris-HCl pH 8.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  1 mM EDTA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>EXAMPLE 13 </heading> <heading>Universal Method for DNA Isolation</heading> <p num="h-0027">PCR-Ready DNA from Bacteria, Fungi, Algae, Plants and Vertebrates</p>
    <p num="p-0130"> <i>E. coli </i>and <i>Baceillus cereus </i>were grown overnight at 37° C. in LB medium, <i>Agrobacterium tumefaciens </i>was grown overnight in YEB medium for about 40 hours at 28° C. (Sambrook, J. et al., 1989, Molecular Cloning: A Laboratory Manual, 2nd ed., Cold Spring Harbour Laboratory, NY.). Cyanobacteria and Prochlorthrix were grown in NIVA medium for 14 days at 18° C. using an illumination of 20 micro Einstein (Norwegian Institute of Water Research, 1991, Culture collection of algae). 20–200 million bacteria or 450,000 cyanobacteria were used per DNA isolation.</p>
    <p num="p-0131">Agar plates containing 2% malt extract was used for mycelia growth and incubated for 14 days at room temperature. Mycelia was isolated by scraping the surface of the agar plates with a spatula. Fungi fruitbodies were obtained from natural populations. In the range of 1–3 mg air dried and 3–20 mg fresh fungi fruitbodies were used per DNA isolation.</p>
    <p num="p-0132">Bakers yeast <i>Saccharomyces cerevisiae </i>was obtained from a commercial supplier. Algae were cultured under illumination in IMR-medium for 7 days (Eppley, R et al., 1967, Exp. Mar. Biol. Ecol. 1, 191–208). Fresh leaves from <i>Arabidopsis thaliana </i>and barley (<i>Hordeum vulgare</i>) were picked from young plants (3 weeks old). Epithelia were obtained from perch (<i>Perca fluvatilis</i>) fins. About 1 mg wet weight yeast, 30–100 mg young plant leaves and 100–400 mg perch were used per DNA isolation.</p>
    <p num="p-0133">Multicellular tissues with rigid cell walls were mechanically broken to increase DNA yield. Fungi fruitbodies were ground with foreceps for about 2 minutes. Plant leaves were homogenised for 2 minutes in liquid nitrogen with a pestle (Kontes Scientific Instruments, Vineland, N.J., USA). For all other samples no mechanical work was required for cell breakage.</p>
    <p num="h-0028">DNA Isolation</p>
    <p num="p-0134">DNA isolations were performed using Dynabeads DNA DIRECT (kit, commercially available from Dynal AS, Oslo, Norway). Lysis of the cells and organisms were obtained by mixing 200 μl of Dynabeads DNA DIRECT (200 μg uncoated Dynabeads in Lysis/binding buffer) with the sample in a 1.5 ml microcentrifuge tube. Lysates were then left on the bench at room temperature for 5 to 15 minutes to allow adsorption of genomic DNA to the Dynabeads. For some bacteria and for plants, incubation at 65° C. for 15 minutes was used to improve lysis before the adsorption step.</p>
    <p num="p-0135">The DNA/Dynabeads complex was attracted to a magnet (Dynal's Magnetic Particle Collector E (MPC-E)), and the lysate was aspirated and discarded.</p>
    <p num="p-0136">The complex was then washed twice in washing buffer (one of the kit components) by attracting it to a Dynal MPC and discarding the supernatant. Finally, the complex was resuspended in 40 μl of TE pH 8.0 (provided in the kit) by vigorous pipetting. Elution was performed by heating the suspension to 65° C. for five minutes and then attracting the beads to a magnet. The DNA-containing aqueous phase was then withdrawn and used for the experiments.</p>
    <p num="p-0137">The DNA was visualised on ethidium bromide stained 1.5% agarose gels. Electrophoresis was performed in 1×TAE buffer, and the results were documented with a DS34 Polaroid camera and Polaroid 667 film.</p>
    <p num="p-0138"> <tables id="TABLE-US-00005" num="00005"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="left"> </colspec> <colspec colname="2" colwidth="112pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Lysis/binding buffer:</td> <td class="description-td"> 0.5 M LiCI</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> 1% LiDS</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> 0.1 M TrisCl pH 7.5</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">  10 mM EDTA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">  5 mM dithiothreitol (DTT)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Washing buffer:</td> <td class="description-td">0.15 M LiCl</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">  10 mM Tris-HCl pH 8.0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">  1 mm EDTA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
For an evaluation of the yield from the DNA DIRECT isolation protocol, standard phenol/chloroform based methods were used. Algae, vertebrate and bacterial DNA were isolated with the protocol described by Sambrook, J. et al., 1989, supra. Cyanobacteria were homogenized with alumina type A-5 (Sigma Chemicals Co., St. Louis, USA) before isolation to ensure complete lysis. Plant and fungi DNA were isolated with the protocol described by Scot, O. R. and Bendich, A. J., 1994, in “Plant Molecular Biology Manual”, page D1: 1–8, Kluwer Academic Publisher, Belgium.
<br>
PCR Amplifications
</p>
    <p num="p-0139">For each sample type the reproducibility was tested by using separate DNA isolations, serial DNA dilutions and multiple PCR assays. DNA isolation reagents and PCR reagents were controlled for absence of contamination in each separate experiment. All PCR reactions were performed in a 50 μl reaction volume containing; 15 pmoles primers, 200 μM dNTP, 10 mM Tris-HCl pH 8.8, 1.5 MM MgCl<sub>2</sub>, 50 mM KCl, 0.1% Triton X-100, 1 Unit DynaZyme thermostable polymerase (Finnzymes Oy, Finland) and 0.1–5 μl of isolated DNA. PCR was performed on a Perkin Elmer GeneAmp PCR System 9600.</p>
    <p num="h-0029">Amplicons and Oligonucleotide Primer</p>
    <p num="p-0140">All PCR reactions were started with a DNA denaturation step at 94–97° C. for 3 to 5 minutes and ended with an extension step at 72° C. for 5 minutes.</p>
    <p num="h-0030">Bacteria and Algae:</p>
    <p num="p-0141">The amplicon was a 16S rRNA region corresponding to <i>E. coli </i>base 334 to 939 according to IUD numbering from bacteria and algae chloroplasts (Brosius, J., et al., 1978, Proc. Natl. Acad. Sci., USA, 57, 4801–4805).</p>
    <p num="p-0142">
      <tables id="TABLE-US-00006" num="00006"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Primers:</td>
              </tr> <tr class="description-tr"> <td class="description-td">CC 5′-TGTAAAACGACGGCCAGTCCAGACTCCTACGGGAGGCAGC-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO:7)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">CD 5′-CTTGTGCGGGCCCCCGTCAATTC-3′ (SEQ ID NO:8).</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0143">Primer CC has a 5′ end complementary to −21 M13 universal primer, making it suitable for direct DNA sequencing. Amplification: 30 cycles of 96° C. for 15 seconds and 70° C. for 2 minutes.</p>
    <p num="p-0144">Algae: An 18S rRNA region was amplified with the primers A and B described by Medlin et al., 1990, Gene, 71, 491–499.</p>
    <p num="p-0145">Amplification: 35 cycles of 94° C. for 30 seconds, 50° C. for 1 minute and 72° C. for 1 minute.</p>
    <p num="p-0146">Fungi: A 18S rRNA region (ca. 600 bp.) was amplified with the primers NS3 and NS4 as described by White et al., 1990. In “PCR Products, a Guide to Methods and Applications” by Innis, M. A. et al., page 315–322, Academic Press, New York.</p>
    <p num="p-0147">Amplification: 5 cycles of 94° C. for 30 seconds, 53° C. for 30 seconds and 72° C. for 1 minute; followed by 25 cycles of 94° C. for 30 seconds, 50° C. for 30 seconds increasing with 15 seconds for each cycle, and 72° C. for 1 minute.</p>
    <p num="p-0148">Plants: The tRNL group I intron in chloroplasts were amplified with the primers C and D described by Fangan et al., 1994, BioTechniques 16, 484–494.</p>
    <p num="p-0149">Amplification: 30 cycles of 94° C. for 30 seconds, 55° C. for 30 seconds and 72° C. for 1 minute.</p>
    <p num="p-0150">A part of the <i>Arabidopsis thaliana </i>gene BI5C (800 bp) was amplified with the primers</p>
    <p num="p-0151">
      <tables id="TABLE-US-00007" num="00007"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">5′-CGGGATCCCTAGGAGACACGGTGCCG-3′ (SEQ ID NO:9) and</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">5′-GGAATTCGATCGGCGGTCTTGAAAC-3′ (SEQ ID NO:10)</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0152">Amplification: 35 cycles of 94° C. for 30 seconds, 59° C. for 30 seconds and 72° for 1 minute.</p>
    <p num="p-0153">A part of the Barley gene Bl5C (800 bp) was amplified with the primers</p>
    <p num="p-0154">
      <tables id="TABLE-US-00008" num="00008"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">5′-CGGATCCCGTCATCCTCTTCTCGCACCCC-3′ (SEQ ID NO:11)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">and</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">5′-GGAATTCCCTTCTTGGAGGGCA-GGTCGGCG-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO:12).</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0155">Amplification: 35 cycles of 94° C. for 30 seconds, 60° C. for 30 seconds and 72° C. for 1 minute.</p>
    <p num="p-0156">Perch: Mitochondrial D-loop fragment (800–900 bp) was amplified with the primers HV2 described by Hoelzel et al., 1991, Mol. Biol. Evol., 8, 475–493, and the primer 5′-GGTGACTTGCATGTGTAAGTTCA-3′,</p>
    <p num="p-0157">Amplification: 30 cycles of 96° C. for 1 minute, 52° C. for 2 minutes and 72° C. for 2 minutes.</p>
    <p num="p-0158">The amplified fragments were visualised on ethidium bromide stained 1.5% agarose gels. Electrophoresis was performed in 1×TAE buffer, and the results were documented with a DS34 Polaroid camera and Polaroid 667 film.</p>
    <p num="p-0159">The results of the experiments are shown in <figref idrefs="DRAWINGS">FIG. 12</figref> and Table 2.</p>
    <p num="p-0160">Bacteria: The standard protocol gave DNA yields in the range of 100–1000 ng for the bacteria tested (<figref idrefs="DRAWINGS">FIG. 12A</figref>). For some Cyanobacteria there was a substantial increase in DNA yield (from 500 ng to more than 1 mg) by improving the lysis with an extra initial incubation step at 65° C. for 15 minutes. In all cases, good amplifications were obtained by using 0.25% of the isolated DNA.</p>
    <p num="p-0161">Fungi: The highest DNA yield was obtained from dried fruit-bodies (300–500 ng) compared with fresh fruit bodies (100–200 ng) (<figref idrefs="DRAWINGS">FIG. 12B</figref>). Mycelia gave low DNA recovery probably due to low number of cells per sample. However, in most cases 5% of the isolated DNA was enough to give nice PCR products (Table 2, <figref idrefs="DRAWINGS">FIG. 12B</figref>). For fruit bodies, 0.5–5% of DNA was used for each PCR.</p>
    <p num="p-0162">Algae: All algae tested gave DNA yield in the range of 200–400 ng using the standard protocol and 1 DNA DIRECT sample test (Table 2, <figref idrefs="DRAWINGS">FIG. 12C</figref>). Nice PCR results were obtained both for amplification of genomic DNA and chloroplast DNA by using 5% of the isolated DNA per PCR reaction.</p>
    <p num="p-0163">Plants: To obtain good DNA yield from plant leaves homogenization in liquid nitrogen was necessary. An extra initial incubation step at 65° C. for 15 minutes also improved the results (<figref idrefs="DRAWINGS">FIG. 12D</figref>). Nice PCR results were obtained both for amplification of genomic DNA and chloroplast DNA, when 5% of the isolated DNA was used per PCR reaction.</p>
    <p num="p-0164">Fish: DNA yield of 300–500 ng was routinely obtained from fish epithelia, using the standard protocol and one sample test (Table 2). Mitochondrial DNA was nicely amplified using 5% of the isolated DNA.</p>
    <p num="p-0165">PCR products from all species tested could easily be directly sequenced by solid-phase sequencing (Hultman et al., 1989, Nucleic Acids Res., 17, 4937–4946).</p>
    <p num="p-0166">
      <tables id="TABLE-US-00009" num="00009"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="343pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 2</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">DNA isolation and PCR amplification from different organisms</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="203pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="84pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">DNA</td>
                <td class="description-td">Ampilcon<sup>e</sup> </td>
                <td class="description-td">PCR</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="175pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="63pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="35pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Species<sup>a</sup> </td>
                <td class="description-td">Sample<sup>b</sup> </td>
                <td class="description-td">yield<sup>c</sup> </td>
                <td class="description-td">Gen.</td>
                <td class="description-td">Org.</td>
                <td class="description-td">product<sup>d</sup> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="175pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <colspec colname="4" colwidth="63pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <chemistry id="CHEM-US-00001" num="00001"> <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-C00001.png"> <img id="EMI-C00001" he="41.40mm" wi="60.28mm" file="US07173124-20070206-C00001.TIF" alt="Figure US07173124-20070206-C00001" img-content="table" img-format="tif" src="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-C00001.png" class="patent-full-image" width="241" height="165"> </a> </div>
                    <attachments> <attachment idref="CHEM-US-00001" attachment-type="cdx" file="US07173124-20070206-C00001.CDX"> </attachment> <attachment idref="CHEM-US-00001" attachment-type="mol" file="US07173124-20070206-C00001.MOL"> </attachment> </attachments> </chemistry> </td>
                <td class="description-td">freshfreshfreshfreshfreshfreshfreshfrozenfrozenfrozenfrozen</td>
                <td class="description-td">++++++++++++++++++++++++++++++++</td>
                <td class="description-td">16S16S16S16S16S16S16S16S16S16S16S</td>
                <td class="description-td">+++++++++++++++++++++++++++++++</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <chemistry id="CHEM-US-00002" num="00002"> <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-C00002.png"> <img id="EMI-C00002" he="25.91mm" wi="50.97mm" file="US07173124-20070206-C00002.TIF" alt="Figure US07173124-20070206-C00002" img-content="table" img-format="tif" src="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-C00002.png" class="patent-full-image" width="203" height="103"> </a> </div>
                    <attachments> <attachment idref="CHEM-US-00002" attachment-type="cdx" file="US07173124-20070206-C00002.CDX"> </attachment> <attachment idref="CHEM-US-00002" attachment-type="mol" file="US07173124-20070206-C00002.MOL"> </attachment> </attachments> </chemistry> </td>
                <td class="description-td">d.fruith,d.fruith,f.fruith,f.mycelf.mycelf.mycelf.mycelfresh</td>
                <td class="description-td">+++nt+++ntntnt+</td>
                <td class="description-td">16S16S16S16S16S16S16S16S</td>
                <td class="description-td">++++++++++++++++++++++++</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <chemistry id="CHEM-US-00003" num="00003"> <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-C00003.png"> <img id="EMI-C00003" he="25.23mm" wi="58.17mm" file="US07173124-20070206-C00003.TIF" alt="Figure US07173124-20070206-C00003" img-content="table" img-format="tif" src="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-C00003.png" class="patent-full-image" width="232" height="100"> </a> </div>
                    <attachments> <attachment idref="CHEM-US-00003" attachment-type="cdx" file="US07173124-20070206-C00003.CDX"> </attachment> <attachment idref="CHEM-US-00003" attachment-type="mol" file="US07173124-20070206-C00003.MOL"> </attachment> </attachments> </chemistry> </td>
                <td class="description-td">freshfreshfreshfreshfreshfreshfreshfresh</td>
                <td class="description-td">++++++++++++++++++++++++</td>
                <td class="description-td">16S/16S16S/16S16S/16S16S16S16S16S16S</td>
                <td class="description-td">+++/++++++/++++++/++++++++++++++++++</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <chemistry id="CHEM-US-00004" num="00004"> <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-C00004.png"> <img id="EMI-C00004" he="12.53mm" wi="50.80mm" file="US07173124-20070206-C00004.TIF" alt="Figure US07173124-20070206-C00004" img-content="table" img-format="tif" src="//patentimages.storage.googleapis.com/US7173124B2/US07173124-20070206-C00004.png" class="patent-full-image" width="203" height="50"> </a> </div>
                    <attachments> <attachment idref="CHEM-US-00004" attachment-type="cdx" file="US07173124-20070206-C00004.CDX"> </attachment> <attachment idref="CHEM-US-00004" attachment-type="mol" file="US07173124-20070206-C00004.MOL"> </attachment> </attachments> </chemistry> </td>
                <td class="description-td">leafleaf</td>
                <td class="description-td">++++++</td>
                <td class="description-td">B15C/trnLB15C/trnL</td>
                <td class="description-td">+++/++++++/+++</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Vertbrates</td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>Perca fubvatillis </i>(perch) fish</td>
                <td class="description-td">ep.</td>
                <td class="description-td">+++</td>
                <td class="description-td">D-loop</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00003" class="description-td" colspan="5"> <sup>a</sup> <i>A. rumerifaciens </i>= <i>Aghrobacterium tumefaciens</i>,</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00004" class="description-td" colspan="5"> <i>P. hollandica </i>= <i>Prochlorothrix hollandica</i>.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00005" class="description-td" colspan="5"> <sup>b</sup> <i>d.fruitb </i>= dried fruitbodies, <i>f.mycelia </i>= fresh mycalia, ep. = epithelium.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00006" class="description-td" colspan="5"> <sup>c</sup>Approximate DNA yields relalive to standard phenol/chloroform isolations;</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00007" class="description-td" colspan="5">+++: &gt;80%,</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00008" class="description-td" colspan="5">++: &gt;10%,</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00009" class="description-td" colspan="5">+: &gt;1%,</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00010" class="description-td" colspan="5">nt. = not tested.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00011" class="description-td" colspan="5"> <sup>d</sup>Gen. = genomic DNA, Org. = organelle DNA from chloreplasts (algac and plants) and mitochondria (fish).</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00012" class="description-td" colspan="5"> <sup>e</sup>Amplicons as described in example 13.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5898071">US5898071</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 2, 1998</td><td class="patent-data-table-td patent-date-value">Apr 27, 1999</td><td class="patent-data-table-td ">Whitehead Institute For Biomedical Research</td><td class="patent-data-table-td ">DNA purification and isolation using magnetic particles</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7608399">US7608399</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2007</td><td class="patent-data-table-td patent-date-value">Oct 27, 2009</td><td class="patent-data-table-td ">Blood Cell Storage, Inc.</td><td class="patent-data-table-td ">Fluorescence method for determining the quantity of nucleic acids in a sample; provide results within a short amount of time in a closed sterile device that would allow for the clinical determination of the level of microbial contamination within a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7670768">US7670768</a></td><td class="patent-data-table-td patent-date-value">Feb 2, 1999</td><td class="patent-data-table-td patent-date-value">Mar 2, 2010</td><td class="patent-data-table-td ">Qiagen North American Holdings, Inc.</td><td class="patent-data-table-td ">Isolating DNA from cells using lysing reagent and RNA digesting enzyme on a solid support; purifying DNA from biological samples; e.g. from white blood cells in a whole blood sample; without the use of harmful organic solvents such as urea and guanidine-based solvents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7790865">US7790865</a></td><td class="patent-data-table-td patent-date-value">Feb 2, 1999</td><td class="patent-data-table-td patent-date-value">Sep 7, 2010</td><td class="patent-data-table-td ">Qiagen North American Holdings, Inc</td><td class="patent-data-table-td ">Eluting reagents, methods and kits for isolating DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8148115">US8148115</a></td><td class="patent-data-table-td patent-date-value">Oct 26, 2009</td><td class="patent-data-table-td patent-date-value">Apr 3, 2012</td><td class="patent-data-table-td ">Blood Cell Storage, Inc.</td><td class="patent-data-table-td ">Device and method for extraction and analysis of nucleic acids from biological samples</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8163535">US8163535</a></td><td class="patent-data-table-td patent-date-value">Jan 2, 2009</td><td class="patent-data-table-td patent-date-value">Apr 24, 2012</td><td class="patent-data-table-td ">Blood Cell Storage, Inc.</td><td class="patent-data-table-td ">Devices and processes for nucleic acid extraction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8629264">US8629264</a></td><td class="patent-data-table-td patent-date-value">May 18, 2012</td><td class="patent-data-table-td patent-date-value">Jan 14, 2014</td><td class="patent-data-table-td ">Blood Cell Storage, Inc.</td><td class="patent-data-table-td ">Gravity flow fluidic device for nucleic acid extraction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8747084">US8747084</a></td><td class="patent-data-table-td patent-date-value">Jul 21, 2011</td><td class="patent-data-table-td patent-date-value">Jun 10, 2014</td><td class="patent-data-table-td ">Aperia Technologies, Inc.</td><td class="patent-data-table-td ">Peristaltic pump</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8771955">US8771955</a></td><td class="patent-data-table-td patent-date-value">Mar 30, 2012</td><td class="patent-data-table-td patent-date-value">Jul 8, 2014</td><td class="patent-data-table-td ">Blood Cell Storage, Inc.</td><td class="patent-data-table-td ">Device and method for extraction and analysis of nucleic acids from biological samples</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013126620A1?cl=en">WO2013126620A1</a></td><td class="patent-data-table-td patent-date-value">Feb 22, 2013</td><td class="patent-data-table-td patent-date-value">Aug 29, 2013</td><td class="patent-data-table-td ">Dow Agrosciences Llc</td><td class="patent-data-table-td ">Automation of barrier-based plant nucleic acid and protein extraction</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025400">536/25.4</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025420">536/25.42</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025410">536/25.41</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015100000">C12N15/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021000000">C07H21/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021040000">C07H21/04</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/1006">C12N15/1006</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/1013">C12N15/1013</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=q3x5BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6806">C12Q1/6806</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68A4</span>, <span class="nested-value">C12N15/10A2</span>, <span class="nested-value">C12N15/10A2D</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Nov 13, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">LIFE TECHNOLOGIES AS, NORWAY</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:INVITROGEN DYNAL AS;REEL/FRAME:029284/0916</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120220</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 29, 2011</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">INVITROGEN DYNAL AS, NORWAY</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNORS:DYNAL AS;DYNAL ASA;DYNAL BIOTECH ASA;AND OTHERS;SIGNING DATES FROM 19990922 TO 20060322;REEL/FRAME:026525/0482</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 6, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 28, 2008</td><td class="patent-data-table-td ">FPB1</td><td class="patent-data-table-td ">Expired due to reexamination which canceled all claims</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 16, 2007</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20070821</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 14, 2006</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">DYNAL BIOTECH ASA, NORWAY</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:DYNAL AS;REEL/FRAME:017337/0177</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20060314</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 6, 2006</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">DYNAL AS, NORWAY</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:DEGGERDAL, ARNE HELGE;LARSEN, FRANK;REEL/FRAME:017169/0718</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19971006</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2vcUoEAhO8MeWH1MhMPspdbuZieg\u0026id=q3x5BAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U2ucirITIPa0yaa1wSR8hPMt2tCJQ\u0026id=q3x5BAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U1Cz221Ej_bYMTTb8vMSqwDbA_gcQ","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Isolation_of_nucleic_acid.pdf?id=q3x5BAABERAJ\u0026output=pdf\u0026sig=ACfU3U0DxR2rdxrsq5skiYJyWHcOBq7jHg"},"sample_url":"http://www.google.com/patents/reader?id=q3x5BAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>