<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7217510 - Detecting biological warfare agent; obtain nucleotide sequences, incubate ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Detecting biological warfare agent; obtain nucleotide sequences, incubate with primers, amplify, determine mass of amplification product, compare to control, classify amplification product"><meta name="DC.contributor" content="David J. Ecker" scheme="inventor"><meta name="DC.contributor" content="Richard H. Griffey" scheme="inventor"><meta name="DC.contributor" content="Rangarajan Sampath" scheme="inventor"><meta name="DC.contributor" content="Steven A. Hofstadler" scheme="inventor"><meta name="DC.contributor" content="John McNeil" scheme="inventor"><meta name="DC.contributor" content="Stanley T. Crooke" scheme="inventor"><meta name="DC.contributor" content="Isis Pharmaceuticals, Inc." scheme="assignee"><meta name="DC.date" content="2001-6-26" scheme="dateSubmitted"><meta name="DC.description" content="The present invention relates generally to the field of investigational bioinformatics and more particularly to secondary structure defining databases. The present invention further relates to methods for interrogating a database as a source of molecular masses of known bioagents for comparing against the molecular mass of an unknown or selected bioagent to determine either the identity of the selected bioagent, and/or to determine the origin of the selected bioagent. The identification of the bioagent is important for determining a proper course of treatment and/or irradication of the bioagent in such cases as biological warfare. Furthermore, the determination of the geographic origin of a selected bioagent will facilitate the identification of potential criminal identity."><meta name="DC.date" content="2007-5-15" scheme="issued"><meta name="DC.relation" content="US:20020045178:A1" scheme="references"><meta name="DC.relation" content="US:20020137057:A1" scheme="references"><meta name="DC.relation" content="US:20020150903:A1" scheme="references"><meta name="DC.relation" content="US:20020150927:A1" scheme="references"><meta name="DC.relation" content="US:20030017487:A1" scheme="references"><meta name="DC.relation" content="US:20030039976:A1" scheme="references"><meta name="DC.relation" content="US:20030064483:A1" scheme="references"><meta name="DC.relation" content="US:20030073112:A1" scheme="references"><meta name="DC.relation" content="US:20030113745:A1" scheme="references"><meta name="DC.relation" content="US:5484908" scheme="references"><meta name="DC.relation" content="US:5502177" scheme="references"><meta name="DC.relation" content="US:5503980" scheme="references"><meta name="DC.relation" content="US:5527675" scheme="references"><meta name="DC.relation" content="US:5547835" scheme="references"><meta name="DC.relation" content="US:5580733" scheme="references"><meta name="DC.relation" content="US:5605798" scheme="references"><meta name="DC.relation" content="US:5622824" scheme="references"><meta name="DC.relation" content="US:5625184" scheme="references"><meta name="DC.relation" content="US:5645985" scheme="references"><meta name="DC.relation" content="US:5686242" scheme="references"><meta name="DC.relation" content="US:5691141" scheme="references"><meta name="DC.relation" content="US:5700642" scheme="references"><meta name="DC.relation" content="US:5763588" scheme="references"><meta name="DC.relation" content="US:5770367" scheme="references"><meta name="DC.relation" content="US:5777324" scheme="references"><meta name="DC.relation" content="US:5830653" scheme="references"><meta name="DC.relation" content="US:5830655" scheme="references"><meta name="DC.relation" content="US:5849492" scheme="references"><meta name="DC.relation" content="US:5851765" scheme="references"><meta name="DC.relation" content="US:5864137" scheme="references"><meta name="DC.relation" content="US:5869242" scheme="references"><meta name="DC.relation" content="US:5871697" scheme="references"><meta name="DC.relation" content="US:5872003" scheme="references"><meta name="DC.relation" content="US:5876936" scheme="references"><meta name="DC.relation" content="US:5928906" scheme="references"><meta name="DC.relation" content="US:5965363" scheme="references"><meta name="DC.relation" content="US:5981176" scheme="references"><meta name="DC.relation" content="US:5994066" scheme="references"><meta name="DC.relation" content="US:6001564" scheme="references"><meta name="DC.relation" content="US:6005096" scheme="references"><meta name="DC.relation" content="US:6007992" scheme="references"><meta name="DC.relation" content="US:6018713" scheme="references"><meta name="DC.relation" content="US:6028183" scheme="references"><meta name="DC.relation" content="US:6043031" scheme="references"><meta name="DC.relation" content="US:6046005" scheme="references"><meta name="DC.relation" content="US:6051378" scheme="references"><meta name="DC.relation" content="US:6054278" scheme="references"><meta name="DC.relation" content="US:6055487" scheme="references"><meta name="DC.relation" content="US:6074823" scheme="references"><meta name="DC.relation" content="US:6090558" scheme="references"><meta name="DC.relation" content="US:6104028" scheme="references"><meta name="DC.relation" content="US:6111251" scheme="references"><meta name="DC.relation" content="US:6140053" scheme="references"><meta name="DC.relation" content="US:6146144" scheme="references"><meta name="DC.relation" content="US:6153389" scheme="references"><meta name="DC.relation" content="US:6159681" scheme="references"><meta name="DC.relation" content="US:6197498" scheme="references"><meta name="DC.relation" content="US:6218118" scheme="references"><meta name="DC.relation" content="US:6221587" scheme="references"><meta name="DC.relation" content="US:6221601" scheme="references"><meta name="DC.relation" content="US:6221605" scheme="references"><meta name="DC.relation" content="US:6225450" scheme="references"><meta name="DC.relation" content="US:6235476" scheme="references"><meta name="DC.relation" content="US:6235478" scheme="references"><meta name="DC.relation" content="US:6235480" scheme="references"><meta name="DC.relation" content="US:6238871" scheme="references"><meta name="DC.relation" content="US:6238927" scheme="references"><meta name="DC.relation" content="US:6258538" scheme="references"><meta name="DC.relation" content="US:6265716" scheme="references"><meta name="DC.relation" content="US:6268129" scheme="references"><meta name="DC.relation" content="US:6268131" scheme="references"><meta name="DC.relation" content="US:6268144" scheme="references"><meta name="DC.relation" content="US:6268146" scheme="references"><meta name="DC.relation" content="US:6270973" scheme="references"><meta name="DC.relation" content="US:6270974" scheme="references"><meta name="DC.relation" content="US:6277573" scheme="references"><meta name="DC.relation" content="US:6277578" scheme="references"><meta name="DC.relation" content="US:6300076" scheme="references"><meta name="DC.relation" content="US:6312893" scheme="references"><meta name="DC.relation" content="US:6312902" scheme="references"><meta name="DC.relation" content="US:6361940" scheme="references"><meta name="DC.relation" content="US:6372424" scheme="references"><meta name="DC.relation" content="US:6391551" scheme="references"><meta name="DC.relation" content="US:6393367" scheme="references"><meta name="DC.relation" content="US:6423966" scheme="references"><meta name="DC.relation" content="US:6428955" scheme="references"><meta name="DC.relation" content="US:6432651" scheme="references"><meta name="DC.relation" content="US:6436635" scheme="references"><meta name="DC.relation" content="US:6436640" scheme="references"><meta name="DC.relation" content="US:6458533" scheme="references"><meta name="DC.relation" content="US:6468748" scheme="references"><meta name="DC.relation" content="US:6475143" scheme="references"><meta name="DC.relation" content="US:6475736" scheme="references"><meta name="DC.relation" content="US:6479239" scheme="references"><meta name="DC.relation" content="US:6500621" scheme="references"><meta name="DC.relation" content="US:6558902" scheme="references"><meta name="DC.relation" content="US:6566055" scheme="references"><meta name="DC.relation" content="US:6582916" scheme="references"><meta name="DC.relation" content="US:6589485" scheme="references"><meta name="DC.relation" content="US:6602662" scheme="references"><meta name="DC.relation" content="US:6613509" scheme="references"><meta name="DC.relation" content="US:6623928" scheme="references"><meta name="DC.relation" content="US:6682889" scheme="references"><meta name="DC.relation" content="WO:1995013396:A2" scheme="references"><meta name="citation_reference" content="Aaserud, D. J. et al., &quot;Accurate Base Composition of Double-Strand DNA by Mass Spectrometry,&quot; Amer. Soc. For Mass Spectrometry, 1996, 1266-1269."><meta name="citation_reference" content="Aaserud, D. J., et al., &quot;Accurate base composition of double strand DNA by mass spectrometry,&quot; J. Am. Soc. Mass Spectrom. (1996) 7(12): 1266-1269."><meta name="citation_reference" content="Bahrmand, A. R. et al., &quot;Polymerase chain reaction of bacterial genomes with single universal primer: application to distinguishing mycobacteria species,&quot; Mol. Cell. Probes (1996) 10(2): 117-22."><meta name="citation_reference" content="Bahrmand, A. R. et al., &quot;Use of restriction enzyme analysis of amplified DNA coding for the hsp65 gene and polymerase chain reaction with universal primer for rapid differentiation of mycobacterium species in the clinical laboratory,&quot; Scand. J. Infect. Diseases (1998) 30(5):477-80."><meta name="citation_reference" content="Bastia, T. et al., &quot;Organelle DNA analysis of Solanum and Brassica somatic hybrids by PCR with &#39;universal primers&#39;,&quot; Theoretical and Applied Genetics (2001) 102(8): 1265-1272."><meta name="citation_reference" content="Bowen, J. E. et al., &quot;The native virulence plasmid combination affects the segregational stability of a theta-replicating shuttle vector in Bacilhus anthracis var. New Hampshire,&quot; J Appl Microbiol. (1999) 87(2): 270-8."><meta name="citation_reference" content="Bowen, J. E. et al., &quot;The native virulence plasmid combination affects the segregational stability of a theta-replicating shuttle vector in Bacillus anthracis var. New Hampshire,&quot; J. of Applied Microbiology, 1999, 87, 270-278."><meta name="citation_reference" content="Campbell, W. P. et al., &quot;Detection of California serogroup Bunyaviruses in tissue culture and mosquito pools by PCR,&quot; J. Virol. Methods (1996) 57(2): 175-9."><meta name="citation_reference" content="Cespedes, A. et al., &quot;Polymerase chain reaction restriction fragment length polymorphism analysis of a short fragment of the cytochrome b gene for identification of flatfish species,&quot; J. Food Protection (1998) 61(12): 1684-5."><meta name="citation_reference" content="Chen, C. A. et al., &quot;Universal primers for amplification of mitochondrial small subunit ribosomal RNA-encoding gene in scleractinian corals,&quot; Marine Biotech. (2000) 2(2): 146-153."><meta name="citation_reference" content="Chen, J. et al., &quot;A universal PCR primer to detect members of the Potyviridae and its use to examine the taxonomic status of several members of the family,&quot; Arch. Virol. (2001) 146(4): 757-66."><meta name="citation_reference" content="Cho, M. et al., &quot;Application of the ribonuclease P (RNase P) RNA gene sequence for phylogenetic analysis of the gene Saccharomonospora,&quot; Internat. J. of Sys. Bacteriol. (1998) 48: 1223-1230."><meta name="citation_reference" content="Conrads, G. et al., &quot;16S-23S rDNA internal transcribed spacer sequences for analysis of the phylogenetic relationships among species of the genus Fusobacterium,&quot; International Journal of Systematic and Evolutionary Microbiology (2002) 52(2): 493-499."><meta name="citation_reference" content="Cornel, A. J. et al., &quot;Polymerase chain reaction species diagnostic assay for Anopheles quadrimaculatus cryptic species (Diptera: Culicidae) based on ribosomal DNA ITS2 sequences,&quot; Journal of Medical Entomology (1996) 33(1): 109-16."><meta name="citation_reference" content="Crain, P. F. et al., &quot;Applications of mass spectrometry to the characterization of oligonucleotides and nucleic acids,&quot; Curr Opin Biotechnol (1998) 9(1): 25-34."><meta name="citation_reference" content="Dasen, G. et al., &quot;Classification and identification of Propionibacteria based on 16S ribosomal RNA genes and PCR,&quot; Systematic and Applied Microbiology (1998) 21(2): 251-259."><meta name="citation_reference" content="Deforce, D. L. D. et al., &quot;Characterization of DNA Oligonucleotides by Coupling of Capillary Zone Electrophoresis to Electrospray Ionization Q-TOF Mass Spectrometry,&quot; Anal. Chem. (1998) 70(14): 3060-3068."><meta name="citation_reference" content="Deforce, D. L. et al., &quot;Analysis of oligonucleotides by ESI-MS,&quot; Advances in Chromatography (2000) 40: 539-566."><meta name="citation_reference" content="Demesure, B. et al., &quot;A set of universal primers for amplification of polymorphic non-coding regions of mitochondrial and chloroplast DNA in plants,&quot; Mol. Ecology (1995) 4(1): 129-31."><meta name="citation_reference" content="Dinauer, D. M. et al., &quot;Sequence-based typing of HLA class II DQB1,&quot; Tissue Antigens (2000) 55(4): 364-368."><meta name="citation_reference" content="Dubernet, S. et al., &quot;A PCR-based method for identification of Lactobacilli at the genus level,&quot; FEMS Microbiology Letters (2002) 214(2): 271-275."><meta name="citation_reference" content="Figueiredo, L. T. M. et al., &quot;Identification of Brazilian flaviviruses by a simplified reverse transcription-polymerase chain reaction method using Flavivirus universal primers,&quot; American Journal of Tropical Medicine and Hygiene (1998) 59(3): 357-362."><meta name="citation_reference" content="Flora, J. et al., &quot;Dual-micro-ESI source for precise mass determination on a quadrupole time-of-flight mass spectrometer for genomic and proteomic applications,&quot; Anal. Bioanal. Chem. (2002) 373(7): 538-46."><meta name="citation_reference" content="Fox, A. et al., &quot;Identification and detection of bacteria: electrospray MS-MS versus derivatization/GC-MS,&quot; Proceedings of the ERDEC Scienctific Conference on Chemical and Biological Defense Research (1996) Aberdeen Proving Ground, Md., Nov. 15-18, 1994: p. 39-44."><meta name="citation_reference" content="Fox, A., &quot;Report of the &quot;Bioterrorism Workshop.&quot; Duke University Thomas Center on Apr. 2-4, 2002, organized by US Army Research Office,&quot; J. Microbiol. Methods (2002) 51(3): 247-54."><meta name="citation_reference" content="Fox, K. F. et al., &quot;Identification of Brucella by Ribosomal-spacer-region PCR and differentiation of, Brucella canis from other Brucella spp. pathogenic for humans by carbohydrate profiles,&quot; J. Clin. Microbiol. (1998) 36(11): 3217-3222."><meta name="citation_reference" content="Griffey, R. H. et al., &quot;Detection of base pair mismatches in duplex DNA and RNA oligonucleotides using electrospray mass spectrometry,&quot; Proceeding of SPIE-The International Society for Optical Engineering (1997) 2985(Ultrasensitive Biochemical Diagnostics II): 82-86."><meta name="citation_reference" content="Griffin, T. J. et al., &quot;Direct genetic analysis by matrix-assisted laser desorption/ionization mass spectrometry,&quot; Proc. Natl. Acad. Sci. USA (1999) 96(11): 6301-6306."><meta name="citation_reference" content="Hannis, J. C, et al, &quot;Genotyping short tandem repeats using flow injection and electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,&quot; Rapid Communications in Mass Spectrometry (2001) 15(5): 348-350."><meta name="citation_reference" content="Hannis, J. C. et al., &quot;Accurate characterization of the tyrosine hydroxylase forensic allele 9.3 through development of electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,&quot; Rapid Communications in Mass Spectrometry (1999) 13(10): 954-62."><meta name="citation_reference" content="Hannis, J. C. et al., &quot;Detection of double-stranded PCR amplicons at the attomole level electrosprayed from low nanomolar solutions using FT-ICR mass spectrometry,&quot; Fresenius Journal of Analytical Chemistry (2001) 369(3-4): 246-51."><meta name="citation_reference" content="Hannis, J. C. et al., &quot;Genotyping complex short tandem repeats using electrospray ionization Fourier transform ion cyclotron resonance multistage mass spectrometry,&quot; Proceedings of SPIE-The International Society for Optical Engineering (2000) 3926: 36-47."><meta name="citation_reference" content="Hayashi, H. et al., &quot;Phylogenetic analysis of the human gut microbiota using I 6S rDNA clone libraries and strictly anaerobic culture-based methods,&quot; Microbiol. Immunol. (2002) 46(8): 535-48."><meta name="citation_reference" content="Henchal, E. A. et al., &quot;Sensitivity and specificity of a universal primer set for the rapid diagnosis of dengue virus infections by polymerase chain reaction and nucleic acid hybridization,&quot; American Journal of Tropical Medicine and Hygiene (1991) 45(4): 418-28."><meta name="citation_reference" content="Herrmann et al., J. Clin Microb., 34(8):1897-1902 (1996)."><meta name="citation_reference" content="Herrmann, B. et al., &quot;Differentiation of Chlamydia spp. by Sequence Determination and Restriction Endonuclease Cleavage of RNase P RNA Genes,&quot; J. Clin. Microbiol. (1996) 34(8): 1897-1902."><meta name="citation_reference" content="Higgins, G. S. et al., &quot;Competitive oligonucleotide single-base extension combined with mass spectrometric detection for mutation screening,&quot; BioTechniques (1997) 23(4): 710-714."><meta name="citation_reference" content="Hoffmann, E. et al., &quot;Universal primer set for the full-length amplification of all influenza A viruses,&quot; Archives of Virology (2001) 146(12): 2275-2289."><meta name="citation_reference" content="Honda, K. et al., &quot;Universal method of hypersensitive nested PCR toward forensic DNA typing,&quot; International Congress Series (1998) 7: 28-30."><meta name="citation_reference" content="Hurst, G. B. et al., &quot;Detection of Bacterial DNA Polymerase Chain Reaction Products by Matrix-assisted Laser Desorption/Ionization Mass Spectrometry,&quot; Rapid Communications in Mass Spectrometry, 1996, 10, 377-382."><meta name="citation_reference" content="Hurst, G. B. et al., &quot;Detection of Bacterial DNA polymerase Chain Reation Products by Matrix-assisted Laser Desorption/Ionization Mass Spectrometry,&quot; Rapid Commun. Mass Spectrom. (1996) 10: 377-382."><meta name="citation_reference" content="Hurst, G. B. et al., &quot;MALDI-TOF analysis of polymerase chain reaction products from methanotrophic bacteria,&quot; Anal. Chem, (1998) 70(13): 2693-2698."><meta name="citation_reference" content="Isola, N. R. et al., &quot;MALDI-TOF mass spectrometric method for detection of hybridized DNA oligomers,&quot; Analytical Chemistry (2001) 73(9): 2126-2131."><meta name="citation_reference" content="Jankowski, K. et al., &quot;Mass spectrometry of DNA, Part 2, Quantitative estimation of base composition,&quot; European Journal of Mass Spectrometry in Biochemistry (1980) 1(1): 45-52."><meta name="citation_reference" content="Kageyama, A. et al., &quot;Rapid detection of human fecal Eubacterium species and related genera by nested PCR method,&quot; Microbiology and Immunology (2001) 45(4): 315-318."><meta name="citation_reference" content="Krahmer, M. T. et al., &quot;Electrospray quadrupole mass spectrometry analysis of model oligonucleotides and polymerase chain reaction products: determination of base substitutions, nucleotide additions/deletions and chemical modifications,&quot; Anal. Chem. (1999) 71(14): 2893-900."><meta name="citation_reference" content="Krahmer, M. T. et al., &quot;MS for identification of single nucleotide polymorphisms and MS/MS for discrimination of isomeric PCR products,&quot; Anal. Chem. (2000) 72(17): 4033-4040."><meta name="citation_reference" content="Lacroix, J.-M. et al., &quot;PCR-based technique for the detection of bacteria in semen and urine,&quot; J. Microbiol. Methods (1996) 26: 61-71."><meta name="citation_reference" content="Lebedev et al (Genetic Analysis: Biomolecular Engineering (1996) 13:15-21)."><meta name="citation_reference" content="Leif et al., Eur. J. Biochem., 230:538-48 (1995)."><meta name="citation_reference" content="Leif, H. et al., &quot;Isolation and characterization of the proton-translocating NADH: ubiquinone oxidoreductase from Escherichia coli,&quot; Eur. J. Biochem. (1995) 230(2): 538-548."><meta name="citation_reference" content="Li, J. et al., &quot;Single nucleotide polymorphism determination using primer extension and time-of-flight mass spectrometry,&quot; Electrophoresis (1999) 20(6): 1258-1265."><meta name="citation_reference" content="Little, D. P. et al., &quot;Rapid Sequencing of Oligonucleotides by High-Resolution Mass Spectrometry,&quot; J. Am. Chem. Soc. (1994) 116(11): 4893-4897."><meta name="citation_reference" content="Liu et al., Gene, 172:105-109 (1996)."><meta name="citation_reference" content="Liu, C. et al., &quot;Improving the microdialysis procedure for electrospray ionization mass spectrometry of biological samples,&quot; Journal of Mass Spectrometry (1997) 32(4): 425-431."><meta name="citation_reference" content="Liu, Y. et al., &quot;An unusual gene arrangement for the putative chromosome replication origin and circadian expression dnaN in Synechococcus sp. strain PCC 7942,&quot; Gene (1996) 172(1): 105-109."><meta name="citation_reference" content="Loakes, D. et al., &quot;Nitroindoles as Universal Bases,&quot; Nucleosides &amp; Nucleotides, 1995, 14(3-5), 1001-1003."><meta name="citation_reference" content="Loakes, D. et al., &quot;Nitroindoles as Universal Bases,&quot; Nucleosides Nucleotides (1995) 14:1001-1003."><meta name="citation_reference" content="Love et al., Gene, 166:179-180 (1995)."><meta name="citation_reference" content="Love, B. C. et al., &quot;Cloning and sequence of the groESL heat-shock operon of Pasteurella multocida,&quot; Gene (1995) 166(1): 179-180."><meta name="citation_reference" content="Maiwald, M. et al., &quot;Characterization of contaminating DNA in Taq polymerase which occurs during amplification with a primer set for Legionella 5S ribosomal RNA,&quot; Mol. Cell. Probes (1994) 8(1): 11-14."><meta name="citation_reference" content="Mangrum, J. et al., &quot;Solution composition and thermal denaturation for the production of single-stranded PCR amplicons: piperidine-induced destabilization of the DNA duplex?&quot; J. Am. Soc. Mass Spectrom, (2002) 13(3): 232-40."><meta name="citation_reference" content="Martemyanov et al., Prot. Expr. and Purif., 18:257-61 (2000)."><meta name="citation_reference" content="Martemyanov, K. A. et al., &quot;Extremely Thermostable Elongation Factor G from Aquifex aeolicus: Cloning, Expression, Purification, and Characterization in a Heterologous Translation System,&quot; Protein Expr. Purif. (2000) 18(3): 257-261."><meta name="citation_reference" content="McCabe, K. M. et al., &quot;Bacterial Species Identification after DNA Amplification with a Universal Primer Pair,&quot; Molecular Genetics and Metabolism (1999) 66(3): 205-211."><meta name="citation_reference" content="Meiyu, F. et al., &quot;Detection of flaviviruses by reverse transcriptase-polymerase chain reaction with the universal primer set,&quot; Microbiology and Immunology (1997) 41(3): 209-13."><meta name="citation_reference" content="Messmer, T. O. et al., &quot;Discrimination of Streptococcus pneumoniae from other upper respiratory tract streptococci by arbitrarily primed PCR,&quot; Clin. Biochem. (1995) 28(6):567-72."><meta name="citation_reference" content="Moricca, S. et al., &quot;Detection of Fusarium oxysporum f.sp. vasinfectum in cotton tissue by polymerase chain reaction,&quot; Plant Pathology (1998) 47(4):486-494."><meta name="citation_reference" content="Morse et al., System. Appl. Microbiol., 19:150-157 (1996)."><meta name="citation_reference" content="Morse, R. et al., &quot;Nucleotide Sequence of Part of the ropC Gene Encoding the alpha&#39; Subunit of DNA-Dependent RNA Polymerase from some Gram-Positive Bacteria and Comparative Amino Acid Sequence Analysis,&quot; System Appl. Microbiol. (1996) 19: 150-157."><meta name="citation_reference" content="Muddiman et al (Anal. Chem. (1996) 66:3705-3712)."><meta name="citation_reference" content="Muddiman et al., Analytical Chemistry, vol. 68, pp. 3705-3712 (1996)."><meta name="citation_reference" content="Muddiman, D. C. et al., &quot;Application of secondary ion and matrix-assisted laser desorption-ionization time-of-flight mass spectrometry for the quantitative analysis of biological molecules,&quot; Mass Spectrometry Reviews (1996) 14(6): 383-429."><meta name="citation_reference" content="Muddiman, D. C. et al., &quot;Characterization of PCR products from bacilli using electrospray ioniation FTICR mass spectrometry,&quot; Analytical Chemistry (1996) 68(21); 3705-12."><meta name="citation_reference" content="Muddiman, D. C. et al., &quot;Important aspects concerning the quantification of biomolecules by time-of-flight secondary-ion mass spectrometry,&quot; Applied Spectroscopy (1996) 50(2); 161-166."><meta name="citation_reference" content="Muddiman, D. C. et al., &quot;Length and base composition of PCR-amplified nucleic acids using mass measurements from electrospray ionization mass spectrometry,&quot; Anal. Chem. (1997) 69(8): 1543-1549."><meta name="citation_reference" content="Muddiman, D. C. et al., &quot;Length and Base Composition of PCR-Amplified Nucleic Acids Using Mass Measurements from Electrospray Ionization Mass Spectrometry,&quot; Anal. Chem., 1997, 1543-1549."><meta name="citation_reference" content="Muddiman, D. C. et al., &quot;Precise mass measurement of a double-stranded 500 base-pair (309 kDa) polymerase chain reaction product by negative ion electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,&quot; Rapid Commun. Mass Spectrom. (1999) 13(12): 1201-1204."><meta name="citation_reference" content="Muddiman, D. C. et al., &quot;Precise Mass Measurement of a Double-stranded 500 Base-pair (309 kDa) Polymerase Chain Reaction Product by Negative Ion Electrospray Ionization Fourier Transform Ion Cyclotron Resonance Mass Spectrometry,&quot; Rapid Commun. Mass Spectrom., 1999, 13, 1201-1204."><meta name="citation_reference" content="Muddiman, D. C. et al., &quot;Sequencing and characterization of larger oligonucleotides by electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,&quot; Rev. Anal. Chem. (1998) 17(1): p. 1-68."><meta name="citation_reference" content="Muddiman, D.C., et al., &quot;Sequencing and characterization of larger oligonucleotides by electrospray ionization fourier transform ion cyclotron resonance mass spectrometry,&quot; Rev. Anal. Chem., 1998, 17(1), 1-68."><meta name="citation_reference" content="Muhammad, W. T. et al., &quot;Electrosrpay ionization quadrupole time-of-flight mass spectrometry and quadrupole mass spectrometry for genotyping single nucleotide substitutions in intact polymerase chain reaction products in K-ras and p53,&quot; Rapid Commun. Mass Spectrum. (2002) 16(24): 2278-85."><meta name="citation_reference" content="Mushegian, A. R. et al., &quot;A minimal gene set for cellular life derived by comparison of complete bacterial genomes,&quot; Proc. Natl. Acad. Sci. USA (1996) 93(19): 10268-10273."><meta name="citation_reference" content="Nagpal, M. L. et al., &quot;Utility of 16S-23S rNA spacer region methodology: how similar are interspace regions within a genome and between strains for closely related organisms?&quot; J. Microbiol. Methods (1998) 33(s): 211-219."><meta name="citation_reference" content="Naumov, G. I. et al., &quot;Discrimination of the Soil Yeast Species Williopsis salurnus and Williopsis suaveolens by the Polymerase Chain Reaction with the Universal Primer N21,&quot; Micribiology (Moscow)(Translation of Mikrobiologiya) (2000) 69(2): 280-285."><meta name="citation_reference" content="NCBI Blast results (Mar. 2, 2006)."><meta name="citation_reference" content="Null, A. P. et al., &quot;Evaluation of sample preparation techniques for mass measurements of PCR products using ESI-FT-ICR mass spectrometry,&quot; J. Am. Soc. Mass Spectrom. (2002) 13(4): 338-344."><meta name="citation_reference" content="Null, A. P. et al., &quot;Genotyping of Simple and Compound Short Tandem Repeat Loci Using Electrospray Ionization Fourier Transform Ion Cyclotron Resonance Mass Spectrometry,&quot; Anal. Chem. (2001) 73(18): 4514-4521."><meta name="citation_reference" content="Null, A. P. et al., &quot;Perspectives on the use of electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry for short tandem repeat genotyping in the post-genome era,&quot; Journal of Mass Spectrometry (2001) 36(6): 589-606."><meta name="citation_reference" content="Null, A. P. et al., &quot;Preparation of single-stranded PCR products for electrospray ionization mass spectrometry using the DNA repair enzyme lambda exonuclease,&quot; Analyst (2000) 125(4): 619-626."><meta name="citation_reference" content="Peng, X. et al., &quot;Rapid detection of Shigella species in enviromental sewage by an immunocapture PCR with universal primers,&quot; Appl. Environ. Microbiol. (2002) 68(5): 2580-3."><meta name="citation_reference" content="Pomerantz, S. C. et al., &quot;Determination of oligonucleotide composition from mass spectrometrically measured molecular weight,&quot; J. Am. Soc. Mass Spectrom. (1993) 4(3): 204-9."><meta name="citation_reference" content="Reid, S. M. et al., &quot;Primary diagnosis of foot-and-mouth disease by reverse transcription polymerase chain reaction,&quot; Journal of Virological Methods (2000) 89(1-2): 167-76."><meta name="citation_reference" content="Reilly, K. et al., &quot;Design and use of 16S ribosomal DNA-directed primers in competitive PCRs to enumerate proteolytic bacteria in the rumen,&quot; Microbiol. Ecol. (2002) 43(2): 259-70."><meta name="citation_reference" content="Sala, M. et al., &quot;Ambiguous base pairing of the purine analogue 1-(2-deoxy-alpha-D-ribofuranosyl)-imidazole-4-carboxamide during PCR,&quot; Nucleic Acids Research, 1996, 24(17), 3302-3306."><meta name="citation_reference" content="Seshadri et al., Infection and Immunity, 67(11):6026-6033 (1999)."><meta name="citation_reference" content="Steffens, D. L. et al., &quot;Sequence analysis of mitochondrial DNA hypervariable regions using infrared fluorescence detection,&quot; BioTechniques (1998) 24(6): 1044-1046."><meta name="citation_reference" content="Takahashi et al., J. Antmicrob. Chemotherapy, 41:49-57 (1998)."><meta name="citation_reference" content="Takahashi, H. et al., &quot;Characterization of gyrA, gyrB, grlA and grlB mutations in fluoroquinolone-resistant clinical isolates of Staphylococcus aureus,&quot; J. Antimicrob. Chemother, (1998) 41(1): 49-57."><meta name="citation_reference" content="Tong et al., NAR, 28(6):1447-1454 (2000)."><meta name="citation_reference" content="Tong, J. et al., &quot;Ligation reaction specificities of an NAD&lt;SUP&gt;+&lt;/SUP&gt;-dependent DNA ligase from the hyperthermophile Aquifex aeolicus,&quot; Nucleic Acids Res. (2000) 28(6): 1447-1454."><meta name="citation_reference" content="Van Aerschot, A. et al., &quot;In search of acyclic analogues as universal nucleosides in degenerate probes,&quot; Nucleosides &amp; Nucleotides (1995) 14(3-5): 1053-1056."><meta name="citation_reference" content="Van Aerschot, A. et al., &quot;In Search of Acyclic Analogues as Universal Nucleosides in Degenerate Probes,&quot; Nucleosides &amp; Nucleotides, 1995, 14(3-5), 1053-1056."><meta name="citation_reference" content="Van Baar, B. L., &quot;Characterisation of bacteria by matrix-assisted laser desorption/ionisation and electrospray mass spectrometry,&quot; FEMS Microbiol. Rev. (2000).24(2): 193-219."><meta name="citation_reference" content="Van Camp, G. et al., &quot;Amplification and sequencing of variable regions in bacterial 23S ribosomal RNA genes with conserved primer sequences,&quot; Curr. Microbiol. (1993) 27(3): 147-51."><meta name="citation_reference" content="Walters, J. J. et al., &quot;Genotyping single nucleotide polymorphisms using intact polymerase chain reaction products by electrospray quadrupole mass spectrometry,&quot; Rapid Commun. Mass Spectrom. (2001) 15(18): 1752-1759."><meta name="citation_reference" content="Widjojoatmodjo et al., Journal of Clinical Microbiology, vol. 32, No. 12, pp. 3002-3007 (1994)."><meta name="citation_reference" content="Widjojoatmodjo, M. N. et al., &quot;Rapid identification of bacteria by PCR-single-strand conformation polymorphism,&quot; J. Clin. Microbiol. (1994) 32(12): 3002-3007."><meta name="citation_reference" content="Wolter, A. et al., &quot;Negative Ion FAB Mass Spectrometric Analysis of Non-Charged Key Intermediates in Oligonucleotide Synthesis: Rapid Identification of Partially Protected Dinucleoside Monophosphates,&quot; Biomed. Environ. Mass Spectrom. (1987) 14: 111-116."><meta name="citation_reference" content="Woo, T. H. S. et al., &quot;Identification of Leptospira inadai by continuous monitoring of fluorescence during rapid cycle PCR,&quot; Systematic and Applied Microbiology (1998) 21(1): 89-96."><meta name="citation_reference" content="Wunschel, D. et al., &quot;Discrimination among the B. cereus group, in comparison to B. subtilis, by structural carbohydrate profiles and ribosomal RNA spacer region PCR,&quot; Systematic and Applied Microbiology (1995) 17(4): 625-35."><meta name="citation_reference" content="Wunschel, D. S. et al., &quot;Analysis of double-stranded polymerase chain reaction products from the Bacilus gereus group by electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,&quot; Rapid Communications in Mass Spectromery (1996) 10(1): 29-35."><meta name="citation_reference" content="Wunschel, D. S. et al., &quot;Heterogeneity in Bacillus cereus PCR Products Detected by ESI-FTICR Mass Spectrometry,&quot; Anal Chem., 1988, 70, 1203-1207."><meta name="citation_patent_number" content="US:7217510"><meta name="citation_patent_application_number" content="US:09/891,793"><link rel="canonical" href="http://www.google.com/patents/US7217510"/><meta property="og:url" content="http://www.google.com/patents/US7217510"/><meta name="title" content="Patent US7217510 - Detecting biological warfare agent; obtain nucleotide sequences, incubate with primers, amplify, determine mass of amplification product, compare to control, classify amplification product"/><meta name="description" content="The present invention relates generally to the field of investigational bioinformatics and more particularly to secondary structure defining databases. The present invention further relates to methods for interrogating a database as a source of molecular masses of known bioagents for comparing against the molecular mass of an unknown or selected bioagent to determine either the identity of the selected bioagent, and/or to determine the origin of the selected bioagent. The identification of the bioagent is important for determining a proper course of treatment and/or irradication of the bioagent in such cases as biological warfare. Furthermore, the determination of the geographic origin of a selected bioagent will facilitate the identification of potential criminal identity."/><meta property="og:title" content="Patent US7217510 - Detecting biological warfare agent; obtain nucleotide sequences, incubate with primers, amplify, determine mass of amplification product, compare to control, classify amplification product"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("eqbtU-aeOcK-oQS-_4HADQ"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("NZL"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("eqbtU-aeOcK-oQS-_4HADQ"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("NZL"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7217510?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7217510"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=Xcx7BAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7217510&amp;usg=AFQjCNHceTvbJYB_MtTa-DBN1rkCrRcGpA" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7217510.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7217510.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20030082539"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7217510"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7217510" style="display:none"><span itemprop="description">The present invention relates generally to the field of investigational bioinformatics and more particularly to secondary structure defining databases. The present invention further relates to methods for interrogating a database as a source of molecular masses of known bioagents for comparing against...</span><span itemprop="url">http://www.google.com/patents/US7217510?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7217510 - Detecting biological warfare agent; obtain nucleotide sequences, incubate with primers, amplify, determine mass of amplification product, compare to control, classify amplification product</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7217510 - Detecting biological warfare agent; obtain nucleotide sequences, incubate with primers, amplify, determine mass of amplification product, compare to control, classify amplification product" title="Patent US7217510 - Detecting biological warfare agent; obtain nucleotide sequences, incubate with primers, amplify, determine mass of amplification product, compare to control, classify amplification product"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7217510 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/891,793</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">May 15, 2007</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jun 26, 2001</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Jun 26, 2001</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US8298760">US8298760</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8380442">US8380442</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030082539">US20030082539</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030167133">US20030167133</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030167134">US20030167134</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030187588">US20030187588</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030187593">US20030187593</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030225529">US20030225529</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20040117129">US20040117129</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20050027459">US20050027459</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20110172925">US20110172925</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120123685">US20120123685</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20130124099">US20130124099</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2003001976A2">WO2003001976A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2003001976A3">WO2003001976A3</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09891793, </span><span class="patent-bibdata-value">891793, </span><span class="patent-bibdata-value">US 7217510 B2, </span><span class="patent-bibdata-value">US 7217510B2, </span><span class="patent-bibdata-value">US-B2-7217510, </span><span class="patent-bibdata-value">US7217510 B2, </span><span class="patent-bibdata-value">US7217510B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22David+J.+Ecker%22">David J. Ecker</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Richard+H.+Griffey%22">Richard H. Griffey</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Rangarajan+Sampath%22">Rangarajan Sampath</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Steven+A.+Hofstadler%22">Steven A. Hofstadler</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22John+McNeil%22">John McNeil</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Stanley+T.+Crooke%22">Stanley T. Crooke</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Isis+Pharmaceuticals,+Inc.%22">Isis Pharmaceuticals, Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7217510.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7217510.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7217510.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (104),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (113),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (12),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (21),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (6)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7217510&usg=AFQjCNE5ycNK1aOeLjaUW8gQSbqPHZqMrA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7217510&usg=AFQjCNG429Y3-CtG3amDARzWhO19tp0vYA">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7217510B2%26KC%3DB2%26FT%3DD&usg=AFQjCNHUt1lDd-wl2poYjFx2vKZTKggNhQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129898836" lang="EN" load-source="ifi">Detecting biological warfare agent; obtain nucleotide sequences, incubate with primers, amplify, determine mass of amplification product, compare to control, classify amplification product</invention-title></span><br><span class="patent-number">US 7217510 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA51151110" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">The present invention relates generally to the field of investigational bioinformatics and more particularly to secondary structure defining databases. The present invention further relates to methods for interrogating a database as a source of molecular masses of known bioagents for comparing against the molecular mass of an unknown or selected bioagent to determine either the identity of the selected bioagent, and/or to determine the origin of the selected bioagent. The identification of the bioagent is important for determining a proper course of treatment and/or irradication of the bioagent in such cases as biological warfare. Furthermore, the determination of the geographic origin of a selected bioagent will facilitate the identification of potential criminal identity.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(29)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00013.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00013.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00014.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00014.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00015.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00015.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00016.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00016.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00017.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00017.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00018.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00018.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00019.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00019.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00020.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00020.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00021.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00021.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00022.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00022.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00023.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00023.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00024.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00024.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00025.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00025.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00026.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00026.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00027.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00027.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00028.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00028.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-D00029.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7217510B2/US07217510-20070515-D00029.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(30)</span></span></div><div class="patent-text"><div mxw-id="PCLM9209791" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A method for providing bioagent characterizing information comprising:
<div class="claim-text">a) measuring or calculating with a mass spectrometer a plurality of molecular masses corresponding to a plurality of amplification products, wherein the amplification products are 46 to 166 nucleobases in length, and wherein the amplification products are obtained by amplification of at least one target sequence region of a bioagent nucleic acid gene sequence using a primer pair that hybridizes to the at least one target sequence region of at least nineteen bioagents; said target sequence region comprising two conserved regions that are hybridizable with the primer pair and that flank a variable region that uniquely varies between at least eight bioagents;</div>
<div class="claim-text">b) interrogating a database stored on a computer readable medium with an identification query, wherein the identification query comprises comparison of the measured molecular mass of step a) with the database; said database comprises molecular mass calculated for the target sequence regions for at least nineteen bioagents and each of the calculated molecular masses is indexed to bioagent characterizing information;</div>
<div class="claim-text">c) delivering from the database a response that comprises the bioagent characterization information generated by the comparison of the measured and calculated molecular mass of step b) thereby identifying the bioagent associated with amplification product of step a).</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the nucleic acid gene sequence encodes ribosomal RNA or a protein involved in translation, replication, recombination, repair, transcription, nucleotide metabolism, amino acid metabolism, lipid metabolism, energy generation, uptake, or secretion.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the bioagent characterizing information is a genus name.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="CLM-00003">claim 3</claim-ref> wherein the genus name is <i>Acinetobacter, Aeromonas, Bacillus, Bacteroides, Bartonella, Bordetella, Borrelia, Brucella, Burkholderia, Campylobacter, Chlamydia, Chlamydophila, Clostridium, Coxiella, Enterococcus, Escherichia, Francisella, Fusobacterium, Haemophilus, Helicobacter, Klebsiella, Legionella, Leptospira, Listeria, Moraxella, Mycobacterium, Mycoplasma, Neisseria, Proteus, Pseudomonas, Rhodobacter, Rickettsia, Salmonella, Shigella, Staphylococcus, Streptobacillus, Streptomyces, Treponema, Ureaplasma, Vibrio, </i>or <i>Yersinia. </i> </div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the bioagent characterizing information is a species name.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the bioagent characterizing information is a strain name.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the response is delivered via a network.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. The method of <claim-ref idref="CLM-00007">claim 7</claim-ref> wherein the network is a local area network, a wide area network, or the internet.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein said mass spectrometer is an electrospray Fourier transform ion cyclotron resonance mass spectrometer or an electrospray time-of-flight mass spectrometer.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the primer pair comprises at least one modified nucleobase.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref> wherein the modified nucleobase comprises 2,6-diaminopurine, propyne C, propyne T, phenoxazine, or G-clamp.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein said bioagent is a biological warfare agent.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00013" num="00013" class="claim">
      <div class="claim-text">13. The method of <claim-ref idref="CLM-00012">claim 12</claim-ref> wherein the biological warfare agent comprises <i>Bacillus anthracis, Yersinia pestis, Franciscella tularensis, Brucella suis, Brucella abortus, Brucella melitensis, Burkholderia mallei, Burkholderia pseudomalleii, Salmonella typhi, Rickettsia typhii, Rickettsia prowasekii, Coxiella burnetii, Rhodobacter capsulatus, Chlamydia pneumoniae, Escherichia coli, Shigella dysenteriae, Shigella flexneri, Bacillus cereus, Clostridium botulinum, Coxiella burnetti, Pseudomonas aeruginosa, Legionella pneumophila, </i>or <i>Vibrio cholerae. </i> </div>
    </div>
    </div> <div class="claim"> <div id="CLM-00014" num="00014" class="claim">
      <div class="claim-text">14. A method for providing bioagent characterizing information comprising:
<div class="claim-text">a) measuring or calculating with a mass spectrometer a plurality of molecular masses corresponding to a plurality of amplification products, wherein the amplification products are 46 to 166 nucleobases in length, and wherein the amplification products are obtained by amplification of at least one target sequence of a bioagent nucleic acid gene sequence using a primer pair that hybridizes to the at least one target sequence region of at least nineteen bioagents said target sequence region comprising two conserved regions that are hybridizable with the primer pair and that flank a variable region that uniquely varies between at least eight bioagents;</div>
<div class="claim-text">b) calculating a base composition from said molecular mass measurement, wherein it identifies the number of A residues, C residues, T residues, G residues, U residues, analogues thereof and mass tag residues thereof;</div>
<div class="claim-text">c) interrogating a database stored on a computer readable medium with an identification query, wherein the identification query comprises comparison of the base composition data from step b) with the database; said database comprises base composition data calculated for the target sequence regions for at least nineteen of the bioagents and each of the calculated base compositions is indexed to bioagent characterizing information; and</div>
<div class="claim-text">d) delivering from the database a response comprising bioagent characterization information generated by the comparison of said measured and calculated base composition of step c) thereby identifying the bioagent associated with an amplification product of step a).</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00015" num="00015" class="claim">
      <div class="claim-text">15. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref> wherein the nucleic acid gene sequence encodes ribosomal RNA or a protein involved in translation, replication, recombination, repair, transcription, nucleotide metabolism, amino acid metabolism, lipid metabolism, energy generation, uptake, or secretion.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00016" num="00016" class="claim">
      <div class="claim-text">16. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref> wherein the bioagent characterizing information is a genus name.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00017" num="00017" class="claim">
      <div class="claim-text">17. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref> wherein the genus name is <i>Acinetobacter, Aeromonas, Bacillus, Bacteriodes, Bartonella, Bordetella, Borrelia, Brucella, Burkholderia, Campylobacter, Chlamydia, Chlamydophila, Clostridium, Coxiella, Enterococcus, Escherichia, Francisella, Fusobacterium, Haemophilus, Helicobacter, Klebsiella, Legionella, Leptospira, Listeria, Moraxella, Mycobacterium, Mycoplasma, Neisseria, Proteus, Pseudomonas, Rhodobacter, Rickettsia, Salmonella, Shigella, Staphylococcus, Streptobacillus, Streptomyces, Treponema, Ureaplasma, Vibrio, </i>or <i>Yersinia. </i> </div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00018" num="00018" class="claim">
      <div class="claim-text">18. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref> wherein the bioagent characterizing information is a species name.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00019" num="00019" class="claim">
      <div class="claim-text">19. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref> wherein the bioagent characterizing information is a strain name.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00020" num="00020" class="claim">
      <div class="claim-text">20. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref> wherein the response is delivered via a network.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00021" num="00021" class="claim">
      <div class="claim-text">21. The method of <claim-ref idref="CLM-00020">claim 20</claim-ref> wherein the network is a local area network, a wide area network, or the internet.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00022" num="00022" class="claim">
      <div class="claim-text">22. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref> wherein said mass spectrometer is an electrospray Fourier transform ion cyclotron resonance mass spectrometer or an electrospray time-of-flight mass spectrometer.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00023" num="00023" class="claim">
      <div class="claim-text">23. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref> wherein the primer pair comprises at least one modified nucleobase.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00024" num="00024" class="claim">
      <div class="claim-text">24. The method of <claim-ref idref="CLM-00023">claim 23</claim-ref> wherein the modified nucleobase comprises 2,6-diaminopurine, propyne C, propyne T, phenoxazine, or G-clamp.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00025" num="00025" class="claim">
      <div class="claim-text">25. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref> wherein said bioagent is a biological warfare agent.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00026" num="00026" class="claim">
      <div class="claim-text">26. The method of <claim-ref idref="CLM-00025">claim 25</claim-ref> wherein the biological warfare agent comprises <i>Bacillus anthracis, Yersinia pestis, Franciscella tularensis, Brucella suis, Brucella abortus, Brucella melitensis, Burkholderia mallei, Burkholderia pseudomalleii, Salmonella typhi, Rickettsia typhii, Rickettsia prowasekii, Coxiella burnetti, Rhodobacter capsulatus, Chlamydia pneumoniae, Escherichia coli, Shigella dysenteriae, Shigella flexneri, Bacillus cereus, Clostridium botulinum, Coxiella burnetti, Pseudomonas aeruginosa, Legionella pneumophila, </i>or <i>Vibrio cholerae. </i> </div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00027" num="00027" class="claim">
      <div class="claim-text">27. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein said bioagent is a bacterium, virus, fungus or protozoan.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00028" num="00028" class="claim">
      <div class="claim-text">28. The method of <claim-ref idref="CLM-00027">claim 27</claim-ref> wherein the bioagent is arenavirus, bunyavirus, mononegavirales, picornavirus, astrovirus, calcivirus, nidovirales, flavivirus or togavirus.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00029" num="00029" class="claim">
      <div class="claim-text">29. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref> wherein said bioagent is a bacterium, virus, fungus or protozoan.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00030" num="00030" class="claim">
      <div class="claim-text">30. The method of <claim-ref idref="CLM-00029">claim 29</claim-ref> wherein the bioagent is arenavirus, bunyavirus, mononegavirales, picornavirus, astrovirus, calcivirus, nidovirales, flavivirus or togavirus.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES16232335" lang="EN" load-source="patent-office" class="description">
    <heading>STATEMENT OF GOVERNMENT SUPPORT</heading> <p num="p-0002">This invention was made with United States Government support under DARPA/SPO contract 4400044016. The United States Government has certain rights in the invention.</p>
    <heading>FIELD OF THE INVENTION</heading> <p num="p-0003">The present invention relates generally to the field of investigational bioinformatics and more particularly to secondary structure defining databases. The present invention further relates to methods for interrogating a database as a source of molecular masses of known bioagents for comparing against the molecular mass of an unknown or selected bioagent to determine either the identity of the selected bioagent, and/or to determine the origin of the selected bioagent. The identification of the bioagent is important for determining a proper course of treatment and/or irradication of the bioagent in such cases as biological warfare. Furthermore, the determination of the geographic origin of a selected bioagent will facilitate the identification of potential criminal identity. The present invention also relates to methods for rapid detection and identification of bioagents from environmental, clinical or other samples. The methods provide for detection and characterization of a unique base composition signature (BCS) from any bioagent, including bacteria and viruses. The unique BCS is used to rapidly identify the bioagent.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p num="p-0004">In the United States, hospitals report well over 5 million cases of recognized infectious disease-related illnesses annually. Significantly greater numbers remain undetected, both in the inpatient and community setting, resulting in substantial morbidity and mortality. Critical intervention for infectious disease relies on rapid, sensitive and specific detection of the offending pathogen, and is central to the mission of microbiology laboratories at medical centers. Unfortunately, despite the recognition that outcomes from infectious illnesses are directly associated with time to pathogen recognition, as well as accurate identification of the class and species of microbe, and ability to identify the presence of drug resistance isolates, conventional hospital laboratories often remain encumbered by traditional slow multi-step culture based assays. Other limitations of the conventional laboratory which have become increasingly apparent include: extremely prolonged wait-times for pathogens with long generation time (up to several weeks); requirements for additional testing and wait times for speciation and identification of antimicrobial resistance; diminished test sensitivity for patients who have received antibiotics; and absolute inability to culture certain pathogens in disease states associated with microbial infection.</p>
    <p num="p-0005">For more than a decade, molecular testing has been heralded as the diagnostic tool for the new millennium, whose ultimate potential could include forced obsolescence of traditional hospital laboratories. However, despite the fact that significant advances in clinical application of PCR techniques have occurred, the practicing physician still relies principally on standard techniques. A brief discussion of several existing applications of PCR in the hospital-based setting follows.</p>
    <p num="p-0006">Generally speaking molecular diagnostics have been championed for identifying organisms that cannot be grown in vitro, or in instances where existing culture techniques are insensitive and/or require prolonged incubation times. PCR-based diagnostics have been successfully developed for a wide variety of microbes. Application to the clinical arena has met with variable success, with only a few assays achieving acceptance and utility.</p>
    <p num="p-0007">One of the earliest, and perhaps most widely recognized applications of PCR for clinical practice is in detection of <i>Mycobacterium tuberculosis. </i>Clinical characteristics favoring development of a nonculture-based test for tuberculosis include week to month long delays associated with standard testing, occurrence of drug-resistant isolates and public health imperatives associated with recognition, isolation and treatment. Although frequently used as a diagnostic adjunctive, practical and routine clinical application of PCR remains problematic due to significant inter-laboratory variation in sensitivity, and inadequate specificity for use in low prevalence populations, requiring further development at the technical level. Recent advances in the laboratory suggest that identification of drug resistant isolates by amplification of mutations associated with specific antibiotic resistance (e.g., rpoB gene in rifampin resistant strains) may be forthcoming for clinical use, although widespread application will require extensive clinical validation.</p>
    <p num="p-0008">One diagnostic assay, which has gained widespread acceptance, is for <i>C. trachomatis. </i>Conventional detection systems are limiting due to inadequate sensitivity and specificity (direct immunofluoresence or enzyme immunoassay) or the requirement for specialized culture facilities, due to the fastidious characteristics of this microbe. Laboratory development, followed by widespread clinical validation testing in a variety of acute and nonacute care settings have demonstrated excellent sensitivity (90–100%) and specificity (97%) of the PCR assay leading to its commercial development. Proven efficacy of the PCR assay from both genital and urine sampling, have resulted in its application to a variety of clinical setting, most recently including routine screening of patients considered at risk.</p>
    <p num="p-0009">While the full potential for PCR diagnostics to provide rapid and critical information to physicians faced with difficult clinical-decisions has yet to be realized, one recently developed assay provides an example of the promise of this evolving technology. Distinguishing life-threatening causes of fever from more benign causes in children is a fundamental clinical dilemma faced by clinicians, particularly when infections of the central nervous system are being considered. Bacterial causes of meningitis can be highly aggressive, but generally cannot be differentiated on a clinical basis from aseptic meningitis, which is a relatively benign condition that can be managed on an outpatient basis. Existing blood culture methods often take several days to turn positive, and are often confounded by poor sensitivity or false-negative findings in patients receiving empiric antimicrobials. Testing and application of a PCR assay for enteroviral meningitis has been found to be highly sensitive. With reporting of results within 1 day, preliminary clinical trials have shown significant reductions in hospital costs, due to decreased duration of hospital stays and reduction in antibiotic therapy. Other viral PCR assays, now routinely available include those for herpes simplex virus, cytomegalovirus, hepatitis and HIV. Each has a demonstrated cost savings role in clinical practice, including detection of otherwise difficult to diagnose infections and newly realized capacity to monitor progression of disease and response to therapy, vital in the management of chronic infectious diseases.</p>
    <p num="p-0010">The concept of a universal detection system has been forwarded for identification of bacterial pathogens, and speaks most directly to the possible clinical implications of a broad-based screening tool for clinical use. Exploiting the existence of highly conserved regions of DNA common to all bacterial species in a PCR assay would empower physicians to rapidly identify the presence of bacteremia, which would profoundly impact patient care. Previous empiric decision making could be abandoned in favor of educated practice, allowing appropriate and expeditious decision-making regarding need for antibiotic therapy and hospitalization.</p>
    <p num="p-0011">Experimental work using the conserved features of the 16S rRNA common to almost all bacterial species, is an area of active investigation. Hospital test sites have focused on “high yield” clinical settings where expeditious identification of the presence of systemic bacterial infection has immediate high morbidity and mortality consequences. Notable clinical infections have included evaluation of febrile infants at risk for sepsis, detection of bacteremia in febrile neutropenic cancer patients, and examination of critically ill patients in the intensive care unit. While several of these studies have reported promising results (with sensitivity and specificity well over 90%), significant technical difficulties (described below) remain, and have prevented general acceptance of this assay in clinics and hospitals (which remain dependent on standard blood culture methodologies). Even the revolutionary advances of real-time PCR technique, which offers a quantitative more reproducible and technically simpler system remains encumbered by inherent technical limitations of the PCR assay.</p>
    <p num="p-0012">The principle shortcomings of applying PCR assays to the clinical setting include: inability to eliminate background DNA contamination; interference with the PCR amplification by substrates present in the reaction; and limited capacity to provide rapid reliable speciation, antibiotic resistance and subtype identification. Some laboratories have recently made progress in identifying and removing inhibitors; however background contamination remains problematic, and methods directed towards eliminating exogenous sources of DNA report significant diminution in assay sensitivity. Finally, while product identification and detailed characterization has been achieved using sequencing techniques, these approaches are laborious and time-intensive thus detracting from its clinical applicability.</p>
    <p num="p-0013">Rapid and definitive microbial identification is desirable for a variety of industrial, medical, environmental, quality, and research reasons. Traditionally, the microbiology laboratory has functioned to identify the etiologic agents of infectious diseases through direct examination and culture of specimens. Since the mid-1980s, researchers have repeatedly demonstrated the practical utility of molecular biology techniques, many of which form the basis of clinical diagnostic assays. Some of these techniques include nucleic acid hybridization analysis, restriction enzyme analysis, genetic sequence analysis, and separation and purification of nucleic acids (See, e.g., J. Sambrook, E. F. Fritsch, and T. Maniatis, Molecular Cloning: A Laboratory Manual, 2nd Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989). These procedures, in general, are time-consuming and tedious. Another option is the polymerase chain reaction (PCR) or other amplification procedure which amplifies a specific target DNA sequence based on the flanking primers used. Finally, detection and data analysis convert the hybridization event into an analytical result.</p>
    <p num="p-0014">Other not yet fully realized applications of PCR for clinical medicine is the identification of infectious causes of disease previously described as idiopathic (e.g. <i>Bartonella henselae </i>in bacillary angiomatosis, and <i>Tropheryma whippellii </i>as the uncultured bacillus associated with Whipple's disease). Further, recent epidemiological studies which suggest a strong association between <i>Chlamydia pneumonia </i>and coronary artery disease, serve as example of the possible widespread, yet undiscovered links between pathogen and host which may ultimately allow for new insights into pathogenesis and novel life sustaining or saving therapeutics.</p>
    <p num="p-0015">For the practicing clinician, PCR technology offers a yet unrealized potential for diagnostic omnipotence in the arena of infectious disease. A universal reliable infectious disease detection system would certainly become a fundamental tool in the evolving diagnostic armamentarium of the 21<sup>st </sup>century clinician. For front line emergency physicians, or physicians working in disaster settings, a quick universal detection system, would allow for molecular triage and early aggressive targeted therapy. Preliminary clinical studies using species specific probes suggest that implementing rapid testing in acute care setting is feasible. Resources could thus be appropriately applied, and patients with suspected infections could rapidly be risk stratified to the different treatment settings, depending on the pathogen and virulence. Furthermore, links with data management systems, locally regionally and nationally, would allow for effective epidemiological surveillance, with obvious benefits for antibiotic selection and control of disease outbreaks.</p>
    <p num="p-0016">For the hospitalists, the ability to speciate and subtype would allow for more precise decision-making regarding antimicrobial agents. Patients who are colonized with highly contagious pathogens could be appropriately isolated on entry into the medical setting without delay. Targeted therapy will diminish development of antibiotic resistance. Furthermore, identification of the genetic basis of antibiotic resistant strains would permit precise pharmacologic intervention. Both physician and patient would benefit with less need for repetitive testing and elimination of wait times for test results.</p>
    <p num="p-0017">It is certain that the individual patient will benefit directly from this approach. Patients with unrecognized or difficult to diagnose infections would be identified and treated promptly. There will be reduced need for prolonged inpatient stays, with resultant decreases in iatrogenic events.</p>
    <p num="p-0018">Mass spectrometry provides detailed information about the molecules being analyzed, including high mass accuracy. It is also a process that can be easily automated. Low-resolution MS may be unreliable when used to detect some known agents, if their spectral lines are sufficiently weak or sufficiently close to those from other living organisms in the sample. DNA chips with specific probes can only determine the presence or absence of specifically anticipated organisms. Because there are hundreds of thousands of species of benign bacteria, some very similar in sequence to threat organisms, even arrays with 10,000 probes lack the breadth needed to detect a particular organism.</p>
    <p num="p-0019">Antibodies face more severe diversity limitations than arrays. If antibodies are designed against highly conserved targets to increase diversity, the false alarm problem will dominate, again because threat organisms are very similar to benign ones. Antibodies are only capable of detecting known agents in relatively uncluttered environments.</p>
    <p num="p-0020">Several groups have described detection of PCR products using high resolution electrospray ionization-Fourier transform-ion cyclotron resonance mass spectrometry (ESI-FT-ICR MS). Accurate measurement of exact mass combined with knowledge of the number of at least one nucleotide allowed calculation of the total base composition for PCR duplex products of approximately 100 base pairs. (Aaserud et al., <i>J. Am. Soc. Mass Spec., </i>1996, 7, 1266–1269; Muddiman et al., <i>Anal. Chem., </i>1997, 69, 1543–1549; Wunschel et al., <i>Anal. Chem., </i>1998, 70, 1203–1207; Muddiman et al., Rev. <i>Anal. Chem., </i>1998, 17, 1–68). Electrospray ionization-Fourier transform-ion cyclotron resistance (ESI-FT-ICR) MS may be used to determine the mass of double-stranded, 500 base-pair PCR products via the average molecular mass (Hurst et al., <i>Rapid Commun. Mass Spec. </i>1996, 10, 377–382). The use of matrix-assisted laser desorption ionization-time of flight (MALDI-TOF) mass spectrometry for characterization of PCR products has been described. (Muddiman et al., <i>Rapid Commun. Mass Spec., </i>1999, 13, 1201–1204). However, the degradation of DNAs over about 75 nucleotides observed with MALDI limited the utility of this method.</p>
    <p num="p-0021">U.S. Pat. No. 5,849,492 describes a method for retrieval of phylogenetically informative DNA sequences which comprise searching for a highly divergent segment of genomic DNA surrounded by two highly conserved segments, designing the universal primers for PCR amplification of the highly divergent region, amplifying the genomic DNA by PCR technique using universal primers, and then sequencing the gene to determine the identity of the organism.</p>
    <p num="p-0022">U.S. Pat. No. 5,965,363 discloses methods for screening nucleic acids for polymorphisms by analyzing amplified target nucleic acids using mass spectrometric techniques and to procedures for improving mass resolution and mass accuracy of these methods.</p>
    <p num="p-0023">WO 99/14375 describes methods, PCR primers and kits for use in analyzing preselected DNA tandem nucleotide repeat alleles by mass spectrometry.</p>
    <p num="p-0024">WO 98/12355 discloses methods of determining the mass of a target nucleic acid by mass spectrometric analysis, by cleaving the target nucleic acid to reduce its length, making the target single-stranded and using MS to determine the mass of the single-stranded shortened target. Also disclosed are methods of preparing a double-stranded target nucleic acid for MS analysis comprising amplification of the target nucleic acid, binding one of the strands to a solid support, releasing the second strand and then releasing the first strand which is then analyzed by MS. Kits for target nucleic acid preparation are also provided.</p>
    <p num="p-0025">PCT WO97/33000 discloses methods for detecting mutations in a target nucleic acid by nonrandomly fragmenting the target into a set of single-stranded nonrandom length fragments and determining their masses by MS.</p>
    <p num="p-0026">U.S. Pat. No. 5,605,798 describes a fast and highly accurate mass spectrometer-based process for detecting the presence of a particular nucleic acid in a biological sample for diagnostic purposes.</p>
    <p num="p-0027">WO 98/21066 describes processes for determining the sequence of a particular target nucleic acid by mass spectrometry. Processes for detecting a target nucleic acid present in a biological sample by PCR amplification and mass spectrometry detection are disclosed, as are methods for detecting a target nucleic acid in a sample by amplifying the target with primers that contain restriction sites and tags, extending and cleaving the amplified nucleic acid, and detecting the presence of extended product, wherein the presence of a DNA fragment of a mass different from wild-type is indicative of a mutation. Methods of sequencing a nucleic acid via mass spectrometry methods are also described.</p>
    <p num="p-0028">WO 97/37041, WO 99/31278 and U.S. Pat. No. 5,547,835 describe methods of sequencing nucleic acids using mass spectrometry. U.S. Pat. Nos. 5,622,824, 5,872,003 and 5,691,141 describe methods, systems and kits for exonuclease-mediated mass spectrometric sequencing.</p>
    <p num="p-0029">Thus, there is a need for a method for bioagent detection and identification which is both specific and rapid, and in which no nucleic acid sequencing is required. The present invention addresses this need.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0030">The present invention is directed to method of identifying an unknown bioagent using a database, such as a database stored on, for example, a local computer or perhaps a database accessible over a network or on the internet. This database of molecular masses of known bioagents provides a standard of comparison for determining both identity and geographic origin of the unknown bioagent. The nucleic acid from said bioagent is first contacted with at least one pair of oligonucleotide primers which hybridize to sequences of said nucleic acid that flank a variable nucleic acid sequence of the bioagent. Using PCR technology an amplification product of this variable nucleic acid sequence is made. After standard isolation, the molecular mass of this amplification product is determined using known mass-spec techniques. This molecular mass is compared to the molecular mass of known bioagents within the database, for identifying the unknown bioagent.</p>
    <p num="p-0031">This invention is also directed to databases having cell-data positional significance comprising at least a first table that includes a plurality of data-containing cells. The table is organized into at least a first row and a second row, each row having columns which are aligned relative to each other so that inter-row conserved regions are aligned. This alignment facilitates the analysis of regions, which are highly conserved between species. This alignment further provides insight into secondary polymer structure by this alignment. Although this invention is directed to a database where each row describes any polymer, in a preferred embodiment, the polymer is an RNA. Other alignments that operate in the same manner are also contemplated.</p>
    <p num="p-0032">Another embodiment of this invention is a method for reconciling the content of two databases such that the content of each is a mirror of the other.</p>
    <p num="p-0033">Another embodiment is directed to determining the geographic origin of a bioagent using a database of molecular masses of known bioagents comprising contacting a nucleic acid from the selected bioagent with at least one pair of oligonucleotide primers which hybridize to sequences of the nucleic acid, where the sequences flank a variable nucleic acid sequence of the bioagent. This hybridized region is isolated and amplified through standard PCR techniques known in the art. The molecular mass is determined of this amplified product through any technique known in the art such as, Mass-spectrometry for example. This molecular mass is compared to the molecular masses stored in the database of known bioagents thereby determining a group of probabilistically reasonable geographic origins for the selected bioagent.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p num="p-0034"> <figref idrefs="DRAWINGS">FIGS. 1A–1H</figref> and <figref idrefs="DRAWINGS">FIG. 2</figref> are representative consensus diagrams that show examples of conserved regions from 16S rRNA (<figref idrefs="DRAWINGS">FIGS. 1A</figref>, <b>1</b>A-<b>2</b>, <b>1</b>A-<b>3</b>, and <b>1</b>A-<b>4</b>), 23S rRNA (3′-half, <figref idrefs="DRAWINGS">FIGS. 1B</figref>, <b>1</b>C, and <b>1</b>D; 5′-half, <figref idrefs="DRAWINGS">FIG. 1E–F</figref>), 23S rRNA Domain I (<figref idrefs="DRAWINGS">FIG. 1G</figref>), 23S rRNA Domain IV (<figref idrefs="DRAWINGS">FIG. 1H</figref>) and 16S rRNA Domain III (<figref idrefs="DRAWINGS">FIG. 2</figref>) which are suitable for use in the present invention. Lines with arrows are examples of regions to which intelligent primer pairs for PCR are designed. The label for each primer pair represents the starting and ending base number of the amplified region on the consensus diagram. Bases in capital letters are greater than 95% conserved; bases in lower case letters are 90–95% conserved, filled circles are 80–90% conserved; and open circles are less than 80% conserved. The label for each primer pair represents the starting and ending base number of the amplified region on the consensus diagram. The nucleotide sequence of the 16S rRNA consensus sequence is SEQ ID NO:3 and the nucleotide sequence of the 23S rRNA consensus sequence is SEQ ID NO:4.</p>
      <p num="p-0035"> <figref idrefs="DRAWINGS">FIG. 2</figref> shows a typical primer amplified region from the 16S rRNA Domain III shown in <b>1</b>A and <b>1</b>A-<b>2</b>.</p>
      <p num="p-0036"> <figref idrefs="DRAWINGS">FIG. 3</figref> is a schematic diagram showing conserved regions in RNase P. Bases in capital letters are greater than 90% conserved; bases in lower case letters are 80–90% conserved; filled circles designate bases which are 70–80% conserved; and open circles designate bases that are less than 70% conserved.</p>
      <p num="p-0037"> <figref idrefs="DRAWINGS">FIG. 4</figref> is a schematic diagram of base composition signature determination using nucleotide analog “tags” to determine base composition signatures.</p>
      <p num="p-0038"> <figref idrefs="DRAWINGS">FIG. 5</figref> shows the deconvoluted mass spectra of a <i>Bacillus anthracis </i>region with and without the mass tag phosphorothioate A (A*). The two spectra differ in that the measured molecular weight of the mass tag-containing sequence is greater than the unmodified sequence.</p>
      <p num="p-0039"> <figref idrefs="DRAWINGS">FIG. 6</figref> shows base composition signature (BCS) spectra from PCR products from <i>Staphylococcus aureus </i>(<i>S. aureus </i>16S<sub>—</sub>1337F) and <i>Bacillus anthracus </i>(<i>B. anthr. </i>16S<sub>—</sub>1337F), amplified using the same primers. The two strands differ by only two (AT--&gt;CG) substitutions and are clearly distinguished on the basis of their BCS.</p>
      <p num="p-0040"> <figref idrefs="DRAWINGS">FIG. 7</figref> shows that a single difference between two sequences (A14 in <i>B. anthracis </i>vs. A15 in <i>B. cereus</i>) can be easily detected using ESI-TOF mass spectrometry.</p>
      <p num="p-0041"> <figref idrefs="DRAWINGS">FIG. 8</figref> is an ESI-TOF of <i>Bacillus anthracis </i>spore coat protein sspE 56 mer plus calibrant. The signals unambiguously identify <i>B. anthracis </i>versus other <i>Bacillus </i>species.</p>
      <p num="p-0042"> <figref idrefs="DRAWINGS">FIG. 9</figref> is an ESI-TOF of a <i>B. anthracis </i>synthetic 16S<sub>—</sub>1228 duplex (reverse and forward strands). The technique easily distinguishes between the forward and reverse strands.</p>
      <p num="p-0043"> <figref idrefs="DRAWINGS">FIG. 10</figref> is an ESI-FTICR-MS of a synthetic <i>B. anthracis </i>16S<sub>—</sub>1337 46 base pair duplex.</p>
      <p num="p-0044"> <figref idrefs="DRAWINGS">FIG. 11</figref> is an ESI-TOF-MS of a 56 mer oligonucleotide (3 scans) from the <i>B. anthracis </i>saspB gene with an internal mass standard. The internal mass standards are designated by asterisks.</p>
      <p num="p-0045"> <figref idrefs="DRAWINGS">FIG. 12</figref> is an ESI-TOF-MS of an internal standard with 5 mM TBA-TFA buffer showing that charge stripping with tributylammonium trifluoroacetate reduces the most abundant charge state from [M−8H+]8− to [M−3H+]3−.</p>
      <p num="p-0046"> <figref idrefs="DRAWINGS">FIG. 13</figref> is a portion of a secondary structure defining database according to one embodiment of the present invention, where two examples of selected sequences are displayed graphically thereunder.</p>
      <p num="p-0047"> <figref idrefs="DRAWINGS">FIG. 14</figref> is a three dimensional graph demonstrating the grouping of sample molecular weight according to species.</p>
      <p num="p-0048"> <figref idrefs="DRAWINGS">FIG. 15</figref> is a three dimensional graph demonstrating the grouping of sample molecular weights according to species of virus and mammal infected.</p>
      <p num="p-0049"> <figref idrefs="DRAWINGS">FIG. 16</figref> is a three dimensional graph demonstrating the grouping of sample molecular weights according to species of virus, and animal-origin of infectious agent.</p>
      <p num="p-0050"> <figref idrefs="DRAWINGS">FIG. 17</figref> is a figure depicting how the triangulation method of the present invention provides for the identification of an unknown bioagent without prior knowledge of the unknown agent. The use of different primer sets to distinguish and identify the unknown is also depicted as primer sets I, II and III within this figure. A three dimensional graph depicts all of bioagent space (170), including the unknown bioagent, which after use of primer set I (171) according to a method according to the present invention further differentiates and classifies bioagents according to major classifications (176) which, upon further analysis using primer set II (172) differentiates the unknown agent (177) from other, known agents (173) and finally, the use of a third primer set (175) further specifies subgroups within the family of the unknown (174).</p>
    </description-of-drawings> <heading>DESCRIPTION OF PREFERRED EMBODIMENTS</heading> <p num="p-0051">The present invention provides a combination of a non-PCR biomass detection mode, preferably high-resolution MS, with PCR-based BCS technology using “intelligent primers” which hybridize to conserved sequence regions of nucleic acids derived from a bioagent and which bracket variable sequence regions that uniquely identify the bioagent. The high-resolution MS technique is used to determine the molecular mass and base composition signature (BCS) of the amplified sequence region. This unique “base composition signature” (BCS) is then input to a maximum-likelihood detection algorithm for matching against a database of base composition signatures in the same amplified region. The present method combines PCR-based amplification technology (which provides specificity) and a molecular mass detection mode (which provides speed and does not require nucleic acid sequencing of the amplified target sequence) for bioagent detection and identification.</p>
    <p num="p-0052">The present methods allow extremely rapid and accurate detection and identification of bioagents compared to existing methods. Furthermore, this rapid detection and identification is possible even when sample material is impure. Thus, the method is useful in a wide variety of fields, including, but not limited to, environmental testing (e.g., detection and discrimination of pathogenic vs. non-pathogenic bacteria in water or other samples), germ warfare (allowing immediate identification of the bioagent and appropriate treatment), pharmacogenetic analysis and medical diagnosis (including cancer diagnosis based on mutations and polymorphisms; drug resistance and susceptibility testing; screening for and/or diagnosis of genetic diseases and conditions; and diagnosis of infectious diseases and conditions). The methods leverage ongoing biomedical research in virulence, pathogenicity, drug resistance and genome sequencing into a method which provides greatly improved sensitivity, specificity and reliability compared to existing methods, with lower rates of false positives.</p>
    <p num="p-0053">The present methods can be used, for example, to detect and classify any biological agent, including bacteria, viruses, fungi and toxins. As one example, where the agent is a biological threat, the information obtained is used to determine practical information needed for countermeasures, including toxin genes, pathogenicity islands and antibiotic resistance genes. In addition, the methods can be used to identify natural or deliberate engineering events including chromosome fragment swapping, molecular breeding (gene shuffling) and emerging infectious diseases.</p>
    <p num="p-0054">Bacteria have a common set of absolutely required genes. About 250 genes are present in all bacterial species (<i>Proc. Natl. Acad. Sci. U.S.A., </i>1996, 93, 10268; <i>Science, </i>1995, 270, 397), including tiny genomes like <i>Mycoplasma, Ureaplasma </i>and <i>Rickettsia. </i>These genes encode proteins involved in translation, replication, recombination and repair, transcription, nucleotide metabolism, amino acid metabolism, lipid metabolism, energy generation, uptake, secretion and the like. Examples of these proteins are DNA polymerase III beta, elongation factor TU, heat shock protein groEL, RNA polymerase beta, phosphoglycerate kinase, NADH dehydrogenase, DNA ligase, DNA topoisomerase and elongation factor G. Operons can also be targeted using the present method. One example of an operon is the bfp operon from enteropathogenic <i>E. coli. </i>Multiple core chromosomal genes can be used to classify bacteria at a genus or genus species level to determine if an organism has threat potential. The methods can also be used to detect pathogenicity markers (plasmid or chromosomal) and antibiotic resistance genes to confirm the threat potential of an organism and to direct countermeasures.</p>
    <p num="p-0055">A theoretically ideal bioagent detector would identify, quantify, and report the complete nucleic acid sequence of every bioagent that reached the sensor. The complete sequence of the nucleic acid component of a pathogen would provide all relevant information about the threat, including its identity and the presence of drug-resistance or pathogenicity markers. This ideal has not yet been achieved. However, the present invention provides a straightforward strategy for obtaining information with the same practical value using base composition signatures (BCS). While the base composition of a gene fragment is not as information-rich as the sequence itself, there is no need to analyze the complete sequence of the gene if the short analyte sequence fragment is properly chosen. A database of reference sequences can be prepared in which each sequence is indexed to a unique base composition signature, so that the presence of the sequence can be inferred with accuracy from the presence of the signature. The advantage of base composition signatures is that they can be quantitatively measured in a massively parallel fashion using multiplex PCR (PCR in which two or more primer pairs amplify target sequences simultaneously) and mass spectrometry. These multiple primer amplified regions uniquely identify most threat and ubiquitous background bacteria and viruses. In addition, cluster-specific primer pairs distinguish important local clusters (e.g., anthracis group).</p>
    <p num="p-0056">In the context of this invention, a “bioagent” is any organism, living or dead, or a nucleic acid derived from such an organism. Examples of bioagents include, but are not limited to, cells (including, but not limited to, human clinical samples, bacterial cells and other pathogens), viruses, toxin genes and bioregulating compounds. Samples may be alive or dead or in a vegetative state (for example, vegetative bacteria or spores) and may be encapsulated or bioengineered.</p>
    <p num="p-0057">As used herein, a “base composition signature” (BCS) is the exact base composition from selected fragments of nucleic acid sequences that uniquely identifies the target gene and source organism. BCS can be thought of as unique indexes of specific genes.</p>
    <p num="p-0058">As used herein, “intelligent primers” are primers which bind to sequence regions which flank an intervening variable region. In a preferred embodiment, these sequence regions which flank the variable region are highly conserved among different species of bioagent. For example, the sequence regions may be highly conserved among all Bacillus species. By the term “highly conserved,” it is meant that the sequence regions exhibit between about 80–100%, more preferably between about 90–100% and most preferably between about 95–100% identity. Examples of intelligent primers which amplify regions of the 16S and 23S rRNA are shown in <figref idrefs="DRAWINGS">FIGS. 1A–1H</figref> and <b>2</b>. A typical primer amplified region in 16S rRNA is shown in <figref idrefs="DRAWINGS">FIG. 2</figref>. The arrows represent primers which bind to highly conserved regions which flank a variable region in 16S rRNA domain III. The amplified region is the stem-loop structure under “1100–1188.”</p>
    <p num="p-0059">One main advantage of the detection methods of the present invention is that the primers need not be specific for a particular bacterial species, or even genus, such as <i>Bacillus </i>or <i>Streptomyces. </i>Instead, the primers recognize highly conserved regions across hundreds of bacterial species including, but not limited to, the species described herein. Thus, the same primer pair can be used to identify any desired bacterium because it will bind to the conserved regions which flank a variable region specific to a single species, or common to several bacterial species, allowing nucleic acid amplification of the intervening sequence and determination of its molecular weight and base composition. For example, the 16S<sub>—</sub>971–1062, 16S<sub>—</sub>1228–1310 and 16S<sub>—</sub>1100–1188 regions are 98–99% conserved in about 900 species of bacteria (16S=16S rRNA, numbers indicate nucleotide position). In one embodiment of the present invention, primers used in the present method bind to one or more of these regions or portions thereof.</p>
    <p num="p-0060">The present invention provides a combination of a non-PCR biomass detection mode, preferably high-resolution MS, with nucleic acid amplification-based BCS technology using “intelligent primers” which hybridize to conserved regions and which bracket variable regions that uniquely identify the bioagent(s). Although the use of PCR is preferred, other nucleic acid amplification techniques may also be used, including ligase chain reaction (LCR) and strand displacement amplification (SDA). The high-resolution MS technique allows separation of bioagent spectral lines from background spectral lines in highly cluttered environments. The resolved spectral lines are then translated to BCS which are input to a maximum-likelihood detection algorithm matched against spectra for one or more known BCS. Preferably, the bioagent BCS spectrum is matched against one or more databases of BCS from vast numbers of bioagents. Preferably, the matching is done using a maximum-likelihood detection algorithm.</p>
    <p num="p-0061">In one embodiment, base composition signatures are quantitatively measured in a massively parallel fashion using the polymerase chain reaction (PCR), preferably multiplex PCR, and mass spectrometric (MS) methods. Sufficient quantities of nucleic acids should be present for detection of bioagents by MS. A wide variety of techniques for preparing large amounts of purified nucleic acids or fragments thereof are well known to those of skill in the art. PCR requires one or more pairs of oligonucleotide primers which bind to regions which flank the target sequence(s) to be amplified. These primers prime synthesis of a different strand of DNA, with synthesis occurring in the direction of one primer towards the other primer. The primers, DNA to be amplified, a thermostable DNA polymerase (e.g. Taq polymerase), the four deoxynucleotide triphosphates, and a buffer are combined to initiate DNA synthesis. The solution is denatured by heating, then cooled to allow annealing of newly added primer, followed by another round of DNA synthesis. This process is typically repeated for about 30 cycles, resulting in amplification of the target sequence.</p>
    <p num="p-0062">The “intelligent primers” define the target sequence region to be amplified and analyzed. In one embodiment, the target sequence is a ribosomal RNA (rRNA) gene sequence. With the complete sequences of many of the smallest microbial genomes now available, it is possible to identify a set of genes that defines “minimal life” and identify composition signatures that uniquely identify each gene and organism. Genes that encode core life functions such as DNA replication, transcription, ribosome structure, translation, and transport are distributed broadly in the bacterial genome and are preferred regions for BCS analysis. Ribosomal RNA (rRNA) genes comprise regions that provide useful base composition signatures. Like many genes involved in core life functions, rRNA genes contain sequences that are extraordinarily conserved across bacterial domains interspersed with regions of high variability that are more specific to each species. The variable regions can be utilized to build a database of base composition signatures. The strategy involves creating a structure-based alignment of sequences of the small (16S) and the large (23S) subunits of the rRNA genes. For example, there are currently over 13,000 sequences in the ribosomal RNA database that has been created and maintained by Robin Gutell, University of Texas at Austin, and is publicly available on the Institute for Cellular and Molecular Biology web page on the world wide web of the Internet at, for example, “rna.icmb.utexas.edu/.” There is also a publicly available rRNA database created and maintained by the University of Antwerp, Belgium on the world wide web of the Internet at, for example, “rrna.uia.ac.be.”</p>
    <p num="p-0063">These databases have been analyzed to determine regions that are useful as base composition signatures. The characteristics of such regions include: a) between about 80 and 100%, preferably&gt;about 95% identity among species of the particular bioagent of interest, of upstream and downstream nucleotide sequences which serve as sequence amplification primer sites; b) an intervening variable region which exhibits no greater than about 5% identity among species; and c) a separation of between about 30 and 1000 nucleotides, preferably no more than about 50–250 nucleotides, and more preferably no more than about 60–100 nucleotides, between the conserved regions.</p>
    <p num="p-0064">Due to their overall conservation, the flanking rRNA primer sequences serve as good “universal” primer binding sites to amplify the region of interest for most, if not all, bacterial species. The intervening region between the sets of primers varies in length and/or composition, and thus provides a unique base composition signature.</p>
    <p num="p-0065">It is advantageous to design the “intelligent primers” to be as universal as possible to minimize the number of primers which need to be synthesized, and to allow detection of multiple species using a single pair of primers. These primer pairs can be used to amplify variable regions in these species. Because any variation (due to codon wobble in the 3<sup>rd </sup>position) in these conserved regions among species is likely to occur in the third position of a DNA triplet, oligonucleotide primers can be designed such that the nucleotide corresponding to this position is a base which can bind to more than one nucleotide, referred to herein as a “universal base.” For example, under this “wobble” pairing, inosine (I) binds to U, C or A; guanine (G) binds to U or C, and uridine (U) binds to U or C. Other examples of universal bases include nitroindoles such as 5-nitroindole or 3-nitropyrrole (Loakes et al., <i>Nucleosides and Nucleotides, </i>1995, 14, 1001–1003), the degenerate nucleotides dP or dK (Hill et al.), an acyclic nucleoside analog containing 5-nitroindazole (Van Aerschot et al., <i>Nucleosides and Nucleotides, </i>1995, 14, 1053–1056) or the purine analog 1-(2-deoxy-beta-D-ribofuranosyl)-imidazole-4-carboxamide (Sala et al., <i>Nucl. Acids Res., </i>1996, 24, 3302–3306).</p>
    <p num="p-0066">In another embodiment of the invention, to compensate for the somewhat weaker binding by the “wobble” base, the oligonucleotide primers are designed such that the first and second positions of each triplet are occupied by nucleotide analogs which bind with greater affinity than the unmodified nucleotide. Examples of these analogs include, but are not limited to, 2,6-diaminopurine which binds to thymine, propyne T which binds to adenine and propyne C and phenoxazines, including G-clamp, which binds to G. Propynes are described in U.S. Pat. Nos. 5,645,985, 5,830.653 and 5,484,908, each of which is incorporated herein by reference in its entirety. Phenoxazines are described in U.S. Pat. Nos. 5,502,177, 5,763,588, and 6,005,096, each of which is incorporated herein by reference in its entirety. G-clamps are described in U.S. Pat. Nos. 6,007,992 and 6,028,183, each of which is incorporated herein by reference in its entirety.</p>
    <p num="p-0067">Bacterial biological warfare agents capable of being detected by the present methods include, but are not limited to, <i>Bacillus anthracis </i>(anthrax), <i>Yersinia pestis </i>(pneumonic plague), <i>Franciscella tularensis </i>(tularemia), <i>Brucella suis, Brucella abortus, Brucella melitensis </i>(undulant fever), <i>Burkholderia mallei </i>(glanders), <i>Burkholderia pseudomalleii </i>(melioidosis), <i>Salmonella typhi </i>(typhoid fever), <i>Rickettsia typhii </i>(epidemic typhus), <i>Rickettsia prowasekii </i>(endemic typhus) and <i>Coxiella burnetii </i>(Q fever), <i>Rhodobacter capsulatus, Chlamydia pneumoniae, Escherichia coli, Shigella dysenteriae, Shigella flexneri, Bacillus cereus, Clostridium botulinum, Coxiella burnetti, Pseudomonas aeruginosa, Legionella pneumophila, </i>and <i>Vibrio cholerae. </i> </p>
    <p num="p-0068">Besides 16S and 23S rRNA, other target regions suitable for use in the present invention for detection of bacteria include, but are not limited to, 5S rRNA and RNase P (<figref idrefs="DRAWINGS">FIG. 3</figref>).</p>
    <p num="p-0069">Biological warfare fungus biowarfare agents include, but are not limited to, <i>coccidioides immitis </i>(Coccidioidomycosis).</p>
    <p num="p-0070">Biological warfare toxin genes capable of being detected by the methods of the present invention include, but are not limited to, botulism, T-2 mycotoxins, ricin, staph enterotoxin B, shigatoxin, abrin, aflatoxin, <i>Clostridium perfringens </i>epsilon toxin, conotoxins, diacetoxyscirpenol, tetrodotoxin and saxitoxin.</p>
    <p num="p-0071">Biological warfare viral threat agents are mostly RNA viruses (positive-strand and negative-strand), with the exception of smallpox. Every RNA virus is a family of related viruses (quasispecies). These viruses mutate rapidly and the potential for engineered strains (natural or deliberate) is very high. RNA viruses cluster into families that have conserved RNA structural domains on the viral genome (e.g., virion components, accessory proteins) and conserved housekeeping genes that encode core viral proteins including, for single strand positive strand RNA viruses, RNA-dependent RNA polymerase, double stranded RNA helicase, chymotrypsin-like and papain-like proteases and methyltransferases.</p>
    <p num="p-0072">Examples of (−)-strand RNA viruses include, but are not limited to, arenaviruses (e.g., sabia virus, lassa fever, Machupo, Argentine hemorrhagic fever, flexal virus), bunyaviruses (e.g., hantavirus, nairovirus, phlebovirus, hantaan virus, Congo-crimean hemorrhagic fever, rift valley fever), and mononegavirales (e.g., filovirus, paramyxovirus, ebola virus, Marburg, equine morbillivirus).</p>
    <p num="p-0073">Examples of (+)-strand RNA viruses include, but are not limited to, picornaviruses (e.g., coxsackievirus, echovirus, human coxsackievirus A, human echovirus, human enterovirus, human poliovirus, hepatitis A virus, human parechovirus, human rhinovirus), astroviruses (e.g., human astrovirus), calciviruses (e.g., chiba virus, chitta virus, human calcivirus, norwalk virus), nidovirales (e.g., human coronavirus, human torovirus), flaviviruses (e.g., dengue virus 1–4, Japanese encephalitis virus, Kyanasur forest disease virus, Murray Valley encephalitis virus, Rocio virus, St. Louis encephalitis virus, West Nile virus, yellow fever virus, hepatitis c virus) and togaviruses (e.g., Chikugunya virus, Eastern equine encephalitis virus, Mayaro virus, O'nyong-nyong virus, Ross River virus, Venezuelan equine encephalitis virus, Rubella virus, hepatitis E virus). The hepatitis C virus has a 5′-untranslated region of 340 nucleotides, an open reading frame encoding 9 proteins having 3010 amino acids and a 3′-untranslated region of 240 nucleotides. The 5′-UTR and 3′-UTR are 99% conserved in hepatitis C viruses.</p>
    <p num="p-0074">In one embodiment, the target gene is an RNA-dependent RNA polymerase or a helicase encoded by (+)-strand RNA viruses, or RNA polymerase from a (−)-strand RNA virus. (+)-strand RNA viruses are double stranded RNA and replicate by RNA-directed RNA synthesis using RNA-dependent RNA polymerase and the positive strand as a template. Helicase unwinds the RNA duplex to allow replication of the single stranded RNA. These viruses include viruses from the family picornaviridae (e.g., poliovirus, coxsackievirus, echovirus), togaviridae (e.g., alphavirus, flavivirus, rubivirus), arenaviridae (e.g., lymphocytic choriomeningitis virus, lassa fever virus), cononaviridae (e.g., human respiratory virus) and Hepatitis A virus. The genes encoding these proteins comprise variable and highly conserved regions which flank the variable regions.</p>
    <p num="p-0075">In another embodiment, the detection scheme for the PCR products generated from the bioagent(s) incorporates at least three features. First, the technique simultaneously detects and differentiates multiple (generally about 6–10) PCR products. Second, the technique provides a BCS that uniquely identifies the bioagent from the possible primer sites. Finally, the detection technique is rapid, allowing multiple PCR reactions to be run in parallel.</p>
    <p num="p-0076">In one embodiment, the method can be used to detect the presence of antibiotic resistance and/or toxin genes in a bacterial species. For example, <i>Bacillus anthracis </i>comprising a tetracycline resistance plasmid and plasmids encoding one or both anthracis toxins (px01 and/or px02) can be detected by using antibiotic resistance primer sets and toxin gene primer sets. If the <i>B. anthracis </i>is positive for tetracycline resistance, then a different antibiotic, for example quinalone, is used.</p>
    <p num="p-0077">Mass spectrometry (MS)-based detection of PCR products provides all of these features with additional advantages. MS is intrinsically a parallel detection scheme without the need for radioactive or fluorescent labels, since every amplification product with a unique base composition is identified by its molecular mass. The current state of the art in mass spectrometry is such that less than femtomole quantities of material can be readily analyzed to afford information about the molecular contents of the sample. An accurate assessment of the molecular mass of the material can be quickly obtained, irrespective of whether the molecular weight of the sample is several hundred, or in excess of one hundred thousand atomic mass units (amu) or Daltons. Intact molecular ions can be generated from amplification products using one of a variety of ionization techniques to convert the sample to gas phase. These ionization methods include, but are not limited to, electrospray ionization (ES), matrix-assisted laser desorption ionization (MALDI) and fast atom bombardment (FAB). For example, MALDI of nucleic acids, along with examples of matrices for use in MALDI of nucleic acids, are described in WO 98/54751 (Genetrace, Inc.).</p>
    <p num="p-0078">Upon ionization, several peaks are observed from one sample due to the formation of ions with different charges. Averaging the multiple readings of molecular mass obtained from a single mass spectrum affords an estimate of molecular mass of the bioagent. Electrospray ionization mass spectrometry (ESI-MS) is particularly useful for very high molecular weight polymers such as proteins and nucleic acids having molecular weights greater than 10 kDa, since it yields a distribution of multiply-charged molecules of the sample without causing a significant amount of fragmentation.</p>
    <p num="p-0079">The mass detectors used in the methods of the present invention include, but are not limited to, Fourier transform ion cyclotron resonance mass spectrometry (FT-ICR-MS), ion trap, quadrupole, magnetic sector, time of flight (TOF), Q-TOF, and triple quadrupole.</p>
    <p num="p-0080">In general, the mass spectrometric techniques which can be used in the present invention include, but are not limited to, tandem mass spectrometry, infrared multiphoton dissociation and pyrolytic gas chromatography mass spectrometry (PGC-MS). In one embodiment of the invention, the bioagent detection system operates continually in bioagent detection mode using pyrolytic GC-MS without PCR for rapid detection of increases in biomass (for example, increases in fecal contamination of drinking water or of germ warfare agents). To achieve minimal latency, a continuous sample stream flows directly into the PGC-MS combustion chamber. When an increase in biomass is detected, a PCR process is automatically initiated. Bioagent presence produces elevated levels of large molecular fragments from, for example, about 100–7,000 Da which are observed in the PGC-MS spectrum. The observed mass spectrum is compared to a threshold level and when levels of biomass are determined to exceed a predetermined threshold, the bioagent classification process described hereinabove (combining PCR and MS, preferably FT-ICR MS) is initiated. Optionally, alarms or other processes (halting ventilation flow, physical isolation) are also initiated by this detected biomass level.</p>
    <p num="p-0081">The accurate measurement of molecular mass for large DNAs is limited by the adduction of cations from the PCR reaction to each strand, resolution of the isotopic peaks from natural abundance <sup>13</sup>C and <sup>15</sup>N isotopes, and assignment of the charge state for any ion. The cations are removed by in-line dialysis using a flow-through chip that brings the solution containing the PCR products into contact with a solution containing ammonium acetate in the presence of an electric field gradient orthogonal to the flow. The latter two problems are addressed by operating with a resolving power of&gt;100,000 and by incorporating isotopically depleted nucleotide triphosphates into the DNA. The resolving power of the instrument is also a consideration. At a resolving power of 10,000, the modeled signal from the [M−14H+]<sup>14−</sup> charge state of an 84 mer PCR product is poorly characterized and assignment of the charge state or exact mass is impossible. At a resolving power of 33,000, the peaks from the individual isotopic components are visible. At a resolving power of 100,000, the isotopic peaks are resolved to the baseline and assignment of the charge state for the ion is straightforward. The [<sup>13</sup>C,<sup>15</sup>N]-depleted triphosphates are obtained, for example, by growing microorganisms on depleted media and harvesting the nucleotides (Batey et al., <i>Nucl. Acids Res., </i>1992, 20, 4515–4523).</p>
    <p num="p-0082">While mass measurements of intact nucleic acid regions are believed to be adequate to determine most bioagents, tandem mass spectrometry (MS<sup>n</sup>) techniques may provide more definitive information pertaining to molecular identity or sequence. Tandem MS involves the coupled use of two or more stages of mass analysis where both the separation and detection steps are based on mass spectrometry. The first stage is used to select an ion or component of a sample from which further structural information is to be obtained. The selected ion is then fragmented using, e.g., blackbody irradiation, infrared multiphoton dissociation, or collisional activation. For example, ions generated by electrospray ionization (ESI) can be fragmented using IR multiphoton dissociation. This activation leads to dissociation of glycosidic bonds and the phosphate backbone, producing two series of fragment ions, called the w-series (having an intact 3′ terminus and a 5′ phosphate following internal cleavage) and the α-Base series(having an intact 5′ terminus and a 3′ furan).</p>
    <p num="p-0083">The second stage of mass analysis is then used to detect and measure the mass of these resulting fragments of product ions. Such ion selection followed by fragmentation routines can be performed multiple times so as to essentially completely dissect the molecular sequence of a sample.</p>
    <p num="p-0084">If there are two or more targets of similar base composition or mass, or if a single amplification reaction results in a product which has the same mass as two or more bioagent reference standards, they can be distinguished by using mass-modifying “tags.” In this embodiment of the invention, a nucleotide analog or “tag” is incorporated during amplification (e.g., a 5-(trifluoromethyl) deoxythymidine triphosphate) which has a different molecular weight than the unmodified base so as to improve distinction of masses. Such tags are described in, for example, PCT WO97/33000, which is incorporated herein by reference in its entirety. This further limits the number of possible base compositions consistent with any mass. For example, 5-(trifluoromethyl)deoxythymidine triphosphate can be used in place of dTTP in a separate nucleic acid amplification reaction. Measurement of the mass shift between a conventional amplification product and the tagged product is used to quantitate the number of thymidine nucleotides in each of the single strands. Because the strands are complementary, the number of adenosine nucleotides in each strand is also determined.</p>
    <p num="p-0085">In another amplification reaction, the number of G and C residues in each strand is determined using, for example, the cytidine analog 5-methylcytosine (5-meC) or propyne C. The combination of the A/T reaction and G/C reaction, followed by molecular weight determination, provides a unique base composition. This method is summarized in <figref idrefs="DRAWINGS">FIG. 4</figref> and Table 1.</p>
    <p num="p-0086">
      <tables id="TABLE-US-00001" num="00001"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="49pt" align="left"> </colspec> <colspec colname="3" colwidth="49pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="42pt" align="center"> </colspec> <colspec colname="7" colwidth="42pt" align="center"> </colspec> <colspec colname="8" colwidth="49pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="8" rowsep="1" class="description-td" colspan="8">TABLE 1</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Total</td>
                <td class="description-td">Total</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Double strand</td>
                <td class="description-td">Single strand</td>
                <td class="description-td">Total mass</td>
                <td class="description-td">Base info</td>
                <td class="description-td">Base info</td>
                <td class="description-td">base comp.</td>
                <td class="description-td">base comp.</td>
              </tr> <tr class="description-tr"> <td class="description-td">Mass tag</td>
                <td class="description-td">sequence</td>
                <td class="description-td">sequence</td>
                <td class="description-td">this strand</td>
                <td class="description-td">this strand</td>
                <td class="description-td">other strand</td>
                <td class="description-td">Top strand</td>
                <td class="description-td">Bottom strand</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">T*mass</td>
                <td class="description-td">T*ACGT*AC</td>
                <td class="description-td">T*ACGT*AC</td>
                <td class="description-td">3x</td>
                <td class="description-td">3T</td>
                <td class="description-td">3A</td>
                <td class="description-td">3T</td>
                <td class="description-td">3A</td>
              </tr> <tr class="description-tr"> <td class="description-td">(T* − T) = x</td>
                <td class="description-td">GT*</td>
                <td class="description-td">GT*</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">2A</td>
                <td class="description-td">2T</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AT*GCAT*G</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">2C</td>
                <td class="description-td">2G</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">2G</td>
                <td class="description-td">2C</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">AT*GCAT*G</td>
                <td class="description-td">2x</td>
                <td class="description-td">2T</td>
                <td class="description-td">2A</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">CA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">C*mass</td>
                <td class="description-td">TAC*GTAC*</td>
                <td class="description-td">TAC*GTAC*</td>
                <td class="description-td">2x</td>
                <td class="description-td">2C</td>
                <td class="description-td">2G</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">(C* − C) = y</td>
                <td class="description-td">GT</td>
                <td class="description-td">GT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ATGC*ATGC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">*A</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">ATGC*ATGC</td>
                <td class="description-td">2x</td>
                <td class="description-td">2C</td>
                <td class="description-td">2G</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">*A</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0087">The mass tag phosphorothioate A (A*) was used to distinguish a <i>Bacillus anthracis </i>cluster. The <i>B. anthracis </i>(A<sub>14</sub>G<sub>9</sub>C<sub>14</sub>T<sub>9</sub>) had an average MW of 14072.26, and the <i>B. anthracis </i>(A<sub>1</sub>A*<sub>13</sub>G<sub>9</sub>C<sub>14</sub>T<sub>9</sub>) had an average molecular weight of 14281.11 and the phosphorothioate A had an average molecular weight of +16.06 as determined by ESI-TOF MS. The deconvoluted spectra are shown in <figref idrefs="DRAWINGS">FIG. 5</figref>.</p>
    <p num="p-0088">In another example, assume the measured molecular masses of each strand are 30,000.115 Da and 31,000.115 Da respectively, and the measured number of dT and dA residues are (30,28) and (28,30). If the molecular mass is accurate to 100 ppm, there are 7 possible combinations of dG+dC possible for each strand. However, if the measured molecular mass is accurate to 10 ppm, there are only 2 combinations of dG+dC, and at 1 ppm accuracy there is only one possible base composition for each strand.</p>
    <p num="p-0089">Signals from the mass spectrometer may be input to a maximum-likelihood detection and classification algorithm such as is widely used in radar signal processing. The detection processing uses matched filtering of BCS observed in mass-basecount space and allows for detection and subtraction of signatures from known, harmless organisms, and for detection of unknown bioagent threats. Comparison of newly observed bioagents to known bioagents is also possible, for estimation of threat level, by comparing their BCS to those of known organisms and to known forms of pathogenicity enhancement, such as insertion of antibiotic resistance genes or toxin genes.</p>
    <p num="p-0090">Processing may end with a Bayesian classifier using log likelihood ratios developed from the observed signals and average background levels. The program emphasizes performance predictions culminating in probability-of-detection versus probability-of-false-alarm plots for conditions involving complex backgrounds of naturally occurring organisms and environmental contaminants. Matched filters consist of a priori expectations of signal values given the set of primers used for each of the bioagents. A genomic sequence database (e.g. GenBank) is used to define the mass basecount matched filters. The database contains known threat agents and benign background organisms. The latter is used to estimate and subtract the signature produced by the background organisms. A maximum likelihood detection of known background organisms is implemented using matched filters and a running-sum estimate of the noise covariance. Background signal strengths are estimated and used along with the matched filters to form signatures which are then subtracted. the maximum likelihood process is applied to this “cleaned up” data in a similar manner employing matched filters for the organisms and a running-sum estimate of the noise-covariance for the cleaned up data.</p>
    <p num="p-0091">In one embodiment, a strategy to “triangulate” each organism by measuring signals from multiple core genes is used to reduce false negative and false positive signals, and enable reconstruction of the origin or hybrid or otherwise engineered bioagents. After identification of multiple core genes, alignments are created from nucleic acid sequence databases. The alignments are then analyzed for regions of conservation and variation, and potential primer binding sites flanking variable regions are identified. Next, amplification target regions for signature analysis are selected which distinguishes organisms based on specific genomic differences (i.e., base composition). For example, detection of signatures for the three part toxin genes typical of <i>B. anthracis </i>(Bowen et al., <i>J. Appl. Microbiol., </i>1999, 87, 270–278) in the absence of the expected signatures from the <i>B. anthracis </i>genome would suggest a genetic engineering event.</p>
    <p num="p-0092">The present method can also be used to detect single nucleotide polymorphisms (SNPs), or multiple nucleotide polymorphisms, rapidly and accurately. A SNP is defined as a single base pair site in the genome that is different from one individual to another. The difference can be expressed either as a deletion, an insertion or a substitution, and is frequently linked to a disease state. Because they occur every 100–1000 base pairs, SNPs are the most frequently bound type of genetic marker in the human genome.</p>
    <p num="p-0093">For example, sickle cell anemia results from an A–T transition, which encodes a valine rather than a glutamic acid residue. Oligonucleotide primers may be designed such that they bind to sequences which flank a SNP site, followed by nucleotide amplification and mass determination of the amplified product. Because the molecular masses of the resulting product from an individual who does not have sickle cell anemia is different from that of the product from an individual who has the disease, the method can be used to distinguish the two individuals. Thus, the method can be used to detect any known SNP in an individual and thus diagnose or determine increased susceptibility to a disease or condition.</p>
    <p num="p-0094">In one embodiment, blood is drawn from an individual and peripheral blood mononuclear cells (PBMC) are isolated and simultaneously tested, preferably in a high-throughput screening method, for one or more SNPs using appropriate primers based on the known sequences which flank the SNP region. The National Center for Biotechnology Information maintains a publicly available database of SNPs on the world wide web of the Internet at, for example, “ncbi.nlm.nih.gov/SNP/.”</p>
    <p num="p-0095">The method of the present invention can also be used for blood typing. The gene encoding A, B or O blood type can differ by four single nucleotide polymorphisms. If the gene contains the sequence CGTGGTGACCCTT (SEQ ID NO:5), antigen A results. If the gene contains the sequence CGTCGTCACCGCTA (SEQ ID NO:6) antigen B results. If the gene contains the sequence CGTGGT-ACCCCTT (SEQ ID NO:7), blood group O results (“-” indicates a deletion). These sequences can be distinguished by designing a single primer pair which flanks these regions, followed by amplification and mass determination.</p>
    <p num="p-0096">While the present invention has been described with specificity in accordance with certain of its preferred embodiments, the following examples serve only to illustrate the invention and are not intended to limit the same.</p>
    <heading>EXAMPLES</heading> <heading>Example 1</heading> <heading>Nucleic Acid Isolation and PCR</heading> <p num="p-0097">In one embodiment, nucleic acid is isolated from the organisms and amplified by PCR using standard methods prior to BCS determination by mass spectrometry. Nucleic acid is isolated, for example, by detergent lysis of bacterial cells, centrifugation and ethanol precipitation. Nucleic acid isolation methods are described in, for example, <i>Current Protocols in Molecular Biology </i>(Ausubel et al.) and <i>Molecular Cloning; A Laboratory Manual </i>(Sambrook et al.). The nucleic acid is then amplified using standard methodology, such as PCR, with primers which bind to conserved regions of the nucleic acid which contain an intervening variable sequence as described below.</p>
    <heading>Example 2</heading> <heading>Mass spectrometry</heading> <p num="p-0098">FTICR Instrumentation: The FTICR instrument is based on a 7 tesla actively shielded superconducting magnet and modified Bruker Daltonics Apex II 70e ion optics and vacuum chamber. The spectrometer is interfaced to a LEAP PAL autosampler and a custom fluidics control system for high throughput screening applications. Samples are analyzed directly from 96-well or 384-well microtiter plates at a rate of about 1 sample/minute. The Bruker data-acquisition platform is supplemented with a lab-built ancillary NT datastation which controls the autosampler and contains an arbitrary waveform generator capable of generating complex rf-excite waveforms (frequency sweeps, filtered noise, stored waveform inverse Fourier transform (SWIFT), etc.) for sophisticated tandem MS experiments. For oligonucleotides in the 20–30-mer regime typical performance characteristics include mass resolving power in excess of 100,000 (FWHM), low ppm mass measurement errors, and an operable m/z range between 50 and 5000 m/z.</p>
    <p num="p-0099">Modified ESI Source: In sample-limited analyses, analyte solutions are delivered at 150 nL/minute to a 30 mm i.d. fused-silica ESI emitter mounted on a 3-D micromanipulator. The ESI ion optics consist of a heated metal capillary, an rf-only hexapole, a skimmer cone, and an auxiliary gate electrode. The 6.2 cm rf-only hexapole is comprised of 1 mm diameter rods and is operated at a voltage of 380 Vpp at a frequency of 5 MHz. A lab-built electro-mechanical shutter can be employed to prevent the electrospray plume from entering the inlet capillary unless triggered to the “open” position via a TTL pulse from the data station. When in the “closed” position, a stable electrospray plume is maintained between the ESI emitter and the face of the shutter. The back face of the shutter arm contains an elastomeric seal which can be positioned to form a vacuum seal with the inlet capillary. When the seal is removed, a 1 mm gap between the shutter blade and the capillary inlet allows constant pressure in the external ion reservoir regardless of whether the shutter is in the open or closed position. When the shutter is triggered, a “time slice” of ions is allowed to enter the inlet capillary and is subsequently accumulated in the external ion reservoir. The rapid response time of the ion shutter (&lt;25 ms) provides reproducible, user defined intervals during which ions can be injected into and accumulated in the external ion reservoir.</p>
    <p num="p-0100">Apparatus for Infrared Multiphoton Dissociation: A 25 watt CW CO<sub>2 </sub>laser operating at 10.6 μm has been interfaced to the spectrometer to enable infrared multiphoton dissociation (IRMPD) for oligonucleotide sequencing and other tandem MS applications. An aluminum optical bench is positioned approximately 1.5 m from the actively shielded superconducting magnet such that the laser beam is aligned with the central axis of the magnet. Using standard IR-compatible mirrors and kinematic mirror mounts, the unfocused 3 mm laser beam is aligned to traverse directly through the 3.5 mm holes in the trapping electrodes of the FTICR trapped ion cell and longitudinally traverse the hexapole region of the external ion guide finally impinging on the skimmer cone. This scheme allows IRMPD to be conducted in an m/z selective manner in the trapped ion cell (e.g. following a SWIFT isolation of the species of interest), or in a broadband mode in the high pressure region of the external ion reservoir where collisions with neutral molecules stabilize IRMPD-generated metastable fragment ions resulting in increased fragment ion yield and sequence coverage.</p>
    <heading>Example 3</heading> <heading>Identification of Bioagents</heading> <p num="p-0101">Table 2 shows a small cross section of a database of calculated molecular masses for over 9 primer sets and approximately 30 organisms. The primer sets were derived from rRNA alignment. Examples of regions from rRNA consensus alignments are shown in FIGS <b>1</b>A–<b>1</b>C. Lines with arrows are examples of regions to which intelligent primer pairs for PCR are designed. The primer pairs are &gt;95% conserved in the bacterial sequence database (currently over 10,000 organisms). The intervening regions are variable in length and/or composition, thus providing the base composition “signature” (BCS) for each organism. Primer pairs were chosen so the total length of the amplified region is less than about 80–90 nucleotides. The label for each primer pair represents the starting and ending base number of the amplified region on the consensus diagram.</p>
    <p num="p-0102">Included in the short bacterial database cross-section in Table 2 are many well known pathogens/biowarfare agents (shown in bold/red typeface) such as <i>Bacillus anthracis </i>or <i>Yersinia pestis </i>as well as some of the bacterial organisms found commonly in the natural environment such as <i>Streptomyces. </i>Even closely related organisms can be distinguished from each other by the appropriate choice of primers. For instance, two low G+C organisms, <i>Bacillus anthracis </i>and <i>Staph aureus, </i>can be distinguished from each other by using the primer pair defined by 16S<sub>—</sub>1337 or 23S<sub>—</sub>855 (ΔM of 4 Da).</p>
    <p num="p-0103">
      <tables id="TABLE-US-00002" num="00002"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="441pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 2</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Cross Section Of A Database Of Calculated Molecular Masses<sup>1</sup> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="98pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <colspec colname="7" colwidth="35pt" align="center"> </colspec> <colspec colname="8" colwidth="35pt" align="center"> </colspec> <colspec colname="9" colwidth="42pt" align="center"> </colspec> <colspec colname="10" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Primer Regions →</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Bug Name</td>
                <td class="description-td">16S_971</td>
                <td class="description-td">16S_1100</td>
                <td class="description-td">16S_1337</td>
                <td class="description-td">16S_1294</td>
                <td class="description-td">16S_1228</td>
                <td class="description-td">23S_1021</td>
                <td class="description-td">23S_855</td>
                <td class="description-td">23S_193</td>
                <td class="description-td">23S_115</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="98pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="35pt" align="center"> </colspec> <colspec colname="8" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="9" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="10" colwidth="42pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <i>Acinetobacter calcoaceticus</i>
                </td>
                <td class="description-td">55619.1</td>
                <td class="description-td">55004</td>
                <td class="description-td">28446.7</td>
                <td class="description-td">35854.9</td>
                <td class="description-td">51295.4</td>
                <td class="description-td">30299</td>
                <td class="description-td">42654</td>
                <td class="description-td">39557.5</td>
                <td class="description-td">54999</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00001.png"> <img id="CUSTOM-CHARACTER-00001" he="2.46mm" wi="16.59mm" file="US07217510-20070515-P00001.TIF" alt="Figure US07217510-20070515-P00001" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00001.png" class="patent-full-image" width="66" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55005</b>
                </td>
                <td class="description-td">
                  <b>54388</b>
                </td>
                <td class="description-td">
                  <b>28448</b>
                </td>
                <td class="description-td">
                  <b>35238</b>
                </td>
                <td class="description-td">
                  <b>51296</b>
                </td>
                <td class="description-td">
                  <b>30295</b>
                </td>
                <td class="description-td">
                  <b>42651</b>
                </td>
                <td class="description-td">
                  <b>39560</b>
                </td>
                <td class="description-td">
                  <b>56850</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus cereus</i>
                </td>
                <td class="description-td">55622.1</td>
                <td class="description-td">54387.9</td>
                <td class="description-td">28447.6</td>
                <td class="description-td">35854.9</td>
                <td class="description-td">51296.4</td>
                <td class="description-td">30295</td>
                <td class="description-td">42651</td>
                <td class="description-td">39560.5</td>
                <td class="description-td">56850.3</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bordetella bronchiseptica</i>
                </td>
                <td class="description-td">56857.3</td>
                <td class="description-td">51300.4</td>
                <td class="description-td">28446.7</td>
                <td class="description-td">35857.9</td>
                <td class="description-td">51307.4</td>
                <td class="description-td">30299</td>
                <td class="description-td">42653</td>
                <td class="description-td">39559.5</td>
                <td class="description-td">51920.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Borrelia burgdorferi</i>
                </td>
                <td class="description-td">56231.2</td>
                <td class="description-td">55621.1</td>
                <td class="description-td">28440.7</td>
                <td class="description-td">35852.9</td>
                <td class="description-td">51295.4</td>
                <td class="description-td">30297</td>
                <td class="description-td">42029.9</td>
                <td class="description-td">38941.4</td>
                <td class="description-td">52524.6</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00002.png"> <img id="CUSTOM-CHARACTER-00002" he="2.46mm" wi="15.49mm" file="US07217510-20070515-P00002.TIF" alt="Figure US07217510-20070515-P00002" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00002.png" class="patent-full-image" width="61" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>58098</b>
                </td>
                <td class="description-td">
                  <b>55011</b>
                </td>
                <td class="description-td">
                  <b>28448</b>
                </td>
                <td class="description-td">
                  <b>35854</b>
                </td>
                <td class="description-td">
                  <b>50683</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Campylobacter jejuni</i>
                </td>
                <td class="description-td">58088.5</td>
                <td class="description-td">54386.9</td>
                <td class="description-td">29061.8</td>
                <td class="description-td">35856.9</td>
                <td class="description-td">50674.3</td>
                <td class="description-td">30294</td>
                <td class="description-td">42032.9</td>
                <td class="description-td">39558.5</td>
                <td class="description-td">45732 5</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00003.png"> <img id="CUSTOM-CHARACTER-00003" he="2.46mm" wi="21.51mm" file="US07217510-20070515-P00003.TIF" alt="Figure US07217510-20070515-P00003" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00003.png" class="patent-full-image" width="86" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55000</b>
                </td>
                <td class="description-td">
                  <b>55007</b>
                </td>
                <td class="description-td">
                  <b>29063</b>
                </td>
                <td class="description-td">
                  <b>35855</b>
                </td>
                <td class="description-td">
                  <b>50676</b>
                </td>
                <td class="description-td">
                  <b>30295</b>
                </td>
                <td class="description-td">
                  <b>42036</b>
                </td>
                <td class="description-td">
                  <b>38941</b>
                </td>
                <td class="description-td">
                  <b>56230</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00004.png"> <img id="CUSTOM-CHARACTER-00004" he="2.46mm" wi="20.49mm" file="US07217510-20070515-P00004.TIF" alt="Figure US07217510-20070515-P00004" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00004.png" class="patent-full-image" width="81" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55006</b>
                </td>
                <td class="description-td">
                  <b>53767</b>
                </td>
                <td class="description-td">
                  <b>28445</b>
                </td>
                <td class="description-td">
                  <b>35855</b>
                </td>
                <td class="description-td">
                  <b>51291</b>
                </td>
                <td class="description-td">
                  <b>30300</b>
                </td>
                <td class="description-td">
                  <b>42656</b>
                </td>
                <td class="description-td">
                  <b>39562</b>
                </td>
                <td class="description-td">
                  <b>54999</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Clostridium difficile</i>
                </td>
                <td class="description-td">56855.3</td>
                <td class="description-td">54386.9</td>
                <td class="description-td">28444.7</td>
                <td class="description-td">35853.9</td>
                <td class="description-td">51296.4</td>
                <td class="description-td">30294</td>
                <td class="description-td">41417.8</td>
                <td class="description-td">39556.5</td>
                <td class="description-td">55612.2</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Enterococcus faecalis</i>
                </td>
                <td class="description-td">55620.1</td>
                <td class="description-td">54387.9</td>
                <td class="description-td">28447.6</td>
                <td class="description-td">35858.9</td>
                <td class="description-td">51296.4</td>
                <td class="description-td">30297</td>
                <td class="description-td">42652</td>
                <td class="description-td">39559.5</td>
                <td class="description-td">56849.3</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00005.png"> <img id="CUSTOM-CHARACTER-00005" he="2.46mm" wi="15.16mm" file="US07217510-20070515-P00005.TIF" alt="Figure US07217510-20070515-P00005" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00005.png" class="patent-full-image" width="60" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55622</b>
                </td>
                <td class="description-td">
                  <b>55009</b>
                </td>
                <td class="description-td">
                  <b>28445</b>
                </td>
                <td class="description-td">
                  <b>35857</b>
                </td>
                <td class="description-td">
                  <b>51301</b>
                </td>
                <td class="description-td">
                  <b>30301</b>
                </td>
                <td class="description-td">
                  <b>42656</b>
                </td>
                <td class="description-td">
                  <b>39562</b>
                </td>
                <td class="description-td">
                  <b>54999</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00006.png"> <img id="CUSTOM-CHARACTER-00006" he="2.46mm" wi="19.73mm" file="US07217510-20070515-P00006.TIF" alt="Figure US07217510-20070515-P00006" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00006.png" class="patent-full-image" width="78" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>53769</b>
                </td>
                <td class="description-td">
                  <b>54385</b>
                </td>
                <td class="description-td">
                  <b>28445</b>
                </td>
                <td class="description-td">
                  <b>35856</b>
                </td>
                <td class="description-td">
                  <b>51298</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Haemophilus influenzae</i>
                </td>
                <td class="description-td">55620.1</td>
                <td class="description-td">55006</td>
                <td class="description-td">28444.7</td>
                <td class="description-td">35855.9</td>
                <td class="description-td">51298.4</td>
                <td class="description-td">30298</td>
                <td class="description-td">42656</td>
                <td class="description-td">39560.5</td>
                <td class="description-td">55613.1</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Klebsiella pneumoniae</i>
                </td>
                <td class="description-td">55622.1</td>
                <td class="description-td">55008</td>
                <td class="description-td">28442.7</td>
                <td class="description-td">35856.9</td>
                <td class="description-td">51297.4</td>
                <td class="description-td">30300</td>
                <td class="description-td">42655</td>
                <td class="description-td">39562.5</td>
                <td class="description-td">55000</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00007.png"> <img id="CUSTOM-CHARACTER-00007" he="2.46mm" wi="22.18mm" file="US07217510-20070515-P00007.TIF" alt="Figure US07217510-20070515-P00007" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00007.png" class="patent-full-image" width="88" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55618</b>
                </td>
                <td class="description-td">
                  <b>55626</b>
                </td>
                <td class="description-td">
                  <b>28446</b>
                </td>
                <td class="description-td">
                  <b>35857</b>
                </td>
                <td class="description-td">
                  <b>51303</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycobacterium avium</i>
                </td>
                <td class="description-td">54390.9</td>
                <td class="description-td">55631.1</td>
                <td class="description-td">29064.8</td>
                <td class="description-td">35858.9</td>
                <td class="description-td">51915.5</td>
                <td class="description-td">30298</td>
                <td class="description-td">42656</td>
                <td class="description-td">38942.4</td>
                <td class="description-td">56241.2</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycobacterium leprae</i>
                </td>
                <td class="description-td">54389.9</td>
                <td class="description-td">55629.1</td>
                <td class="description-td">29064.8</td>
                <td class="description-td">35860.9</td>
                <td class="description-td">51917.5</td>
                <td class="description-td">30298</td>
                <td class="description-td">42656</td>
                <td class="description-td">39559.5</td>
                <td class="description-td">56240.2</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycobacterium tuberculosis</i>
                </td>
                <td class="description-td">54390.9</td>
                <td class="description-td">55629.1</td>
                <td class="description-td">29064.8</td>
                <td class="description-td">35860.9</td>
                <td class="description-td">51301.4</td>
                <td class="description-td">30299</td>
                <td class="description-td">42656</td>
                <td class="description-td">39560.5</td>
                <td class="description-td">56243.2</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycoplasma genitalium</i>
                </td>
                <td class="description-td">53143.7</td>
                <td class="description-td">45115.4</td>
                <td class="description-td">29061.8</td>
                <td class="description-td">35854.9</td>
                <td class="description-td">50671.3</td>
                <td class="description-td">30294</td>
                <td class="description-td">43264.1</td>
                <td class="description-td">39558.5</td>
                <td class="description-td">56842.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycoplasma pneumoniae</i>
                </td>
                <td class="description-td">53143.7</td>
                <td class="description-td">45118.4</td>
                <td class="description-td">29061.8</td>
                <td class="description-td">35854.9</td>
                <td class="description-td">50673.3</td>
                <td class="description-td">30294</td>
                <td class="description-td">43264.1</td>
                <td class="description-td">39559.5</td>
                <td class="description-td">56843.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Neisseria gonorrhoeae</i>
                </td>
                <td class="description-td">55627.1</td>
                <td class="description-td">54389.9</td>
                <td class="description-td">28445.7</td>
                <td class="description-td">35855.9</td>
                <td class="description-td">51302.4</td>
                <td class="description-td">30300</td>
                <td class="description-td">42649</td>
                <td class="description-td">39561.5</td>
                <td class="description-td">55000</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00008.png"> <img id="CUSTOM-CHARACTER-00008" he="2.46mm" wi="23.28mm" file="US07217510-20070515-P00008.TIF" alt="Figure US07217510-20070515-P00008" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00008.png" class="patent-full-image" width="93" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55623</b>
                </td>
                <td class="description-td">
                  <b>55010</b>
                </td>
                <td class="description-td">
                  <b>28443</b>
                </td>
                <td class="description-td">
                  <b>35658</b>
                </td>
                <td class="description-td">
                  <b>51301</b>
                </td>
                <td class="description-td">
                  <b>30298</b>
                </td>
                <td class="description-td">
                  <b>43272</b>
                </td>
                <td class="description-td">
                  <b>39558</b>
                </td>
                <td class="description-td">
                  <b>55619</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00009.png"> <img id="CUSTOM-CHARACTER-00009" he="2.46mm" wi="19.73mm" file="US07217510-20070515-P00009.TIF" alt="Figure US07217510-20070515-P00009" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00009.png" class="patent-full-image" width="78" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>58093</b>
                </td>
                <td class="description-td">
                  <b>55621</b>
                </td>
                <td class="description-td">
                  <b>28448</b>
                </td>
                <td class="description-td">
                  <b>35853</b>
                </td>
                <td class="description-td">
                  <b>50677</b>
                </td>
                <td class="description-td">
                  <b>30293</b>
                </td>
                <td class="description-td">
                  <b>42650</b>
                </td>
                <td class="description-td">
                  <b>39559</b>
                </td>
                <td class="description-td">
                  <b>53139</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00010.png"> <img id="CUSTOM-CHARACTER-00010" he="2.46mm" wi="17.27mm" file="US07217510-20070515-P00010.TIF" alt="Figure US07217510-20070515-P00010" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00010.png" class="patent-full-image" width="69" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>58094</b>
                </td>
                <td class="description-td">
                  <b>55623</b>
                </td>
                <td class="description-td">
                  <b>28448</b>
                </td>
                <td class="description-td">
                  <b>35853</b>
                </td>
                <td class="description-td">
                  <b>50679</b>
                </td>
                <td class="description-td">
                  <b>30293</b>
                </td>
                <td class="description-td">
                  <b>42648</b>
                </td>
                <td class="description-td">
                  <b>39559</b>
                </td>
                <td class="description-td">
                  <b>53755</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00011.png"> <img id="CUSTOM-CHARACTER-00011" he="2.46mm" wi="22.61mm" file="US07217510-20070515-P00011.TIF" alt="Figure US07217510-20070515-P00011" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00011.png" class="patent-full-image" width="90" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55622</b>
                </td>
                <td class="description-td">
                  <b>55005</b>
                </td>
                <td class="description-td">
                  <b>28445</b>
                </td>
                <td class="description-td">
                  <b>35857</b>
                </td>
                <td class="description-td">
                  <b>51301</b>
                </td>
                <td class="description-td">
                  <b>30301</b>
                </td>
                <td class="description-td">
                  <b>42658</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00012.png"> <img id="CUSTOM-CHARACTER-00012" he="2.46mm" wi="18.37mm" file="US07217510-20070515-P00012.TIF" alt="Figure US07217510-20070515-P00012" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00012.png" class="patent-full-image" width="73" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55623</b>
                </td>
                <td class="description-td">
                  <b>55009</b>
                </td>
                <td class="description-td">
                  <b>28444</b>
                </td>
                <td class="description-td">
                  <b>35857</b>
                </td>
                <td class="description-td">
                  <b>51301</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Staphylococcus aureus</i>
                </td>
                <td class="description-td">56854.3</td>
                <td class="description-td">54386.9</td>
                <td class="description-td">28443.7</td>
                <td class="description-td">35852.9</td>
                <td class="description-td">51294.4</td>
                <td class="description-td">30298</td>
                <td class="description-td">42655</td>
                <td class="description-td">39559 5</td>
                <td class="description-td">57466.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Streptomyces</i>
                </td>
                <td class="description-td">54389.9</td>
                <td class="description-td">59341.6</td>
                <td class="description-td">29063.8</td>
                <td class="description-td">35858.9</td>
                <td class="description-td">51300.4</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">39563.5</td>
                <td class="description-td">56864.3</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Treponema pallidum</i>
                </td>
                <td class="description-td">56245.2</td>
                <td class="description-td">55631.1</td>
                <td class="description-td">28445.7</td>
                <td class="description-td">35851.9</td>
                <td class="description-td">51297.4</td>
                <td class="description-td">30299</td>
                <td class="description-td">42034.9</td>
                <td class="description-td">38939.4</td>
                <td class="description-td">57473.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00013.png"> <img id="CUSTOM-CHARACTER-00013" he="2.46mm" wi="13.72mm" file="US07217510-20070515-P00013.TIF" alt="Figure US07217510-20070515-P00013" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00013.png" class="patent-full-image" width="54" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55625</b>
                </td>
                <td class="description-td">
                  <b>55626</b>
                </td>
                <td class="description-td">
                  <b>28443</b>
                </td>
                <td class="description-td">
                  <b>35857</b>
                </td>
                <td class="description-td">
                  <b>52536</b>
                </td>
                <td class="description-td">
                  <b>29063</b>
                </td>
                <td class="description-td">
                  <b>30303</b>
                </td>
                <td class="description-td">
                  <b>35241</b>
                </td>
                <td class="description-td">
                  <b>50675</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Vibrio parahaemolyticus</i>
                </td>
                <td class="description-td">54384.9</td>
                <td class="description-td">55626.1</td>
                <td class="description-td">28444.7</td>
                <td class="description-td">34620.7</td>
                <td class="description-td">50064.2</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00014.png"> <img id="CUSTOM-CHARACTER-00014" he="2.46mm" wi="12.70mm" file="US07217510-20070515-P00014.TIF" alt="Figure US07217510-20070515-P00014" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00014.png" class="patent-full-image" width="50" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55620</b>
                </td>
                <td class="description-td">
                  <b>55626</b>
                </td>
                <td class="description-td">
                  <b>28443</b>
                </td>
                <td class="description-td">
                  <b>35857</b>
                </td>
                <td class="description-td">
                  <b>51299</b>
                </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="left" id="FOO-00001" class="description-td" colspan="10"> <sup>1</sup>Molecular mass distribution of PCR amplified regions for a selection of organisms (rows) across various primer pairs (columns). Pathogens are shown in <b>bold</b>. Empty cells indicate presently incomplete or missing data.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0104"> <figref idrefs="DRAWINGS">FIG. 6</figref> shows the use of ESI-FT-ICR MS for measurement of exact mass. The spectra from 46 mer PCR products originating at position 1337 of the 16S rRNA from <i>S. aureus </i>(upper) and <i>B. anthracis </i>(lower) are shown. These data are from the region of the spectrum containing signals from the [M−8H+]<sup>8−</sup> charge states of the respective 5′-3′ strands. The two strands differ by two (AT→CG) substitutions, and have measured masses of 14206.396 and 14208.373±0.010 Da, respectively. The possible base compositions derived from the masses of the forward and reverse strands for the <i>B. anthracis </i>products are listed in Table 3.</p>
    <p num="p-0105"> <tables id="TABLE-US-00003" num="00003"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 3</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Possible base composition for <i>B. anthracis </i>products</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="84pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Calc. Mass</td> <td class="description-td">Error</td> <td class="description-td">Base Comp.</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.2935</td> <td class="description-td">0.079520</td> <td class="description-td">A1 G17 C10 T18</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3160</td> <td class="description-td">0.056980</td> <td class="description-td">A1 G20 C15 T10</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3386</td> <td class="description-td">0.034440</td> <td class="description-td">A1 G23 C20 T2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3074</td> <td class="description-td">0.065560</td> <td class="description-td">A6 G11 C3 T26</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3300</td> <td class="description-td">0.043020</td> <td class="description-td">A6 G14 C8 T18</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3525</td> <td class="description-td">0.020480</td> <td class="description-td">A6 G17 C13 T10</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3751</td> <td class="description-td">0.002060</td> <td class="description-td">A6 G20 C18 T2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3439</td> <td class="description-td">0.029060</td> <td class="description-td">A11 G8 C1 T26</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3665</td> <td class="description-td">0.006520</td> <td class="description-td">A11 G11 C6 T18</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> <b>14208.3890</b> </td> <td class="description-td"> <b>0.016020</b> </td> <td class="description-td"> <b>A11 G14 C11 T10</b> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.4116</td> <td class="description-td">0.038560</td> <td class="description-td">A11 G17 C16 T2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.4030</td> <td class="description-td">0.029980</td> <td class="description-td">A16 G8 C4 T18</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.4255</td> <td class="description-td">0.052520</td> <td class="description-td">A16 G11 C9 T10</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.4481</td> <td class="description-td">0.075060</td> <td class="description-td">A16 G14 C14 T2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.4395</td> <td class="description-td">0.066480</td> <td class="description-td">A21 G5 C2 T18</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.4620</td> <td class="description-td">0.089020</td> <td class="description-td">A21 G8 C7 T10</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.2624</td> <td class="description-td">0.080600</td> <td class="description-td">A0 G14 C13 T19</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.2849</td> <td class="description-td">0.058060</td> <td class="description-td">A0 G17 C18 T11</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3075</td> <td class="description-td">0.035520</td> <td class="description-td">A0 G20 C23 T3</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.2538</td> <td class="description-td">0.089180</td> <td class="description-td">A5 G5 C1 T35</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.2764</td> <td class="description-td">0.066640</td> <td class="description-td">A5 G8 C6 T27</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.2989</td> <td class="description-td">0.044100</td> <td class="description-td">A5 G11 C11 T19</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3214</td> <td class="description-td">0.021560</td> <td class="description-td">A5 G14 C16 T11</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3440</td> <td class="description-td">0.000980</td> <td class="description-td">A5 G17 C21 T3</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3129</td> <td class="description-td">0.030140</td> <td class="description-td">A10 G5 C4 T27</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3354</td> <td class="description-td">0.007600</td> <td class="description-td">A10 G8 C9 T19</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> <b>14079.3579</b> </td> <td class="description-td"> <b>0.014940</b> </td> <td class="description-td"> <b>A10 G11 C14 T11</b> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3805</td> <td class="description-td">0.037480</td> <td class="description-td">A10 G14 C19 T3</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3494</td> <td class="description-td">0.006360</td> <td class="description-td">A15 G2 C2 T27</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3719</td> <td class="description-td">0.028900</td> <td class="description-td">A15 G5 C7 T19</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3944</td> <td class="description-td">0.051440</td> <td class="description-td">A15 G8 C12 T11</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.4170</td> <td class="description-td">0.073980</td> <td class="description-td">A15 G11 C17 T3</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.4084</td> <td class="description-td">0.065400</td> <td class="description-td">A20 G2 C5 T19</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.4309</td> <td class="description-td">0.087940</td> <td class="description-td">A20 G5 C10 T13</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
Among the 16 compositions for the forward strand and the 18 compositions for the reverse strand that were calculated, only one pair (shown in bold) are complementary, corresponding to the actual base compositions of the <i>B. anthracis </i>PCR products.
</p>
    <heading>Example 4</heading> <heading>BCS of Region from <i>Bacillus anthracis </i>and <i>Bacillus cereus </i> </heading> <p num="p-0106">A conserved Bacillus region from <i>B. anthracis </i>(A<sub>14</sub>G<sub>9</sub>C<sub>14</sub>T<sub>9</sub>) and <i>B. cereus </i>(A<sub>15</sub>G<sub>9</sub>C<sub>13</sub>T<sub>9</sub>) having a C to A base change was synthesized and subjected to ESI-TOF MS. The results are shown in <figref idrefs="DRAWINGS">FIG. 7</figref> in which the two regions are clearly distinguished using the method of the present invention (MW=14072.26 vs. 14096.29).</p>
    <heading>Example 5</heading> <heading>Identification of Additional Bioagents</heading> <p num="p-0107">In other examples of the present invention, the pathogen <i>Vibrio cholera </i>can be distinguished from <i>Vibrio parahemolyticus </i>with ΔM&gt;600 Da using one of three 16S primer sets shown in Table 2 (16S<sub>—</sub>971, 16S<sub>—</sub>1228 or 16S<sub>—</sub>1294) as shown in Table 4. The two mycoplasma species in the list (<i>M. genitalium </i>and <i>M. pneumoniae</i>) can also be distinguished from each other, as can the three mycobacteriae. While the direct mass measurements of amplified products can identify and distinguish a large number of organisms, measurement of the base composition signature provides dramatically enhanced resolving power for closely related organisms. In cases such as <i>Bacillus anthracis </i>and <i>Bacillus cereus </i>that are virtually indistinguishable from each other based solely on mass differences, compositional analysis or fragmentation patterns are used to resolve the differences. The single base difference between the two organisms yields different fragmentation patterns, and despite the presence of the ambiguous/unidentified base N at position 20 in <i>B. anthracis, </i>the two organisms can be identified.</p>
    <p num="p-0108">Tables 4a–b show examples of primer pairs from Table 1 which distinguish pathogens from background.</p>
    <p num="p-0109">
      <tables id="TABLE-US-00004" num="00004"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="77pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" rowsep="1" class="description-td" colspan="5">TABLE 4a</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Organism name</td>
                <td class="description-td">23S_855</td>
                <td class="description-td">16S_1337</td>
                <td class="description-td">23S_1021</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Bacillus anthracis</i>
                </td>
                <td class="description-td">42650.98</td>
                <td class="description-td">28447.65</td>
                <td class="description-td">30294.98</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Staphylococcus aureus</i>
                </td>
                <td class="description-td">42654.97</td>
                <td class="description-td">28443.67</td>
                <td class="description-td">30297.96</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0110">
      <tables id="TABLE-US-00005" num="00005"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" rowsep="1" class="description-td" colspan="5">TABLE 4b</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Organism name</td>
                <td class="description-td">16S_971</td>
                <td class="description-td">16S_1294</td>
                <td class="description-td">16S_1228</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Vibrio cholerae</i>
                </td>
                <td class="description-td">55625.09</td>
                <td class="description-td">35856.87</td>
                <td class="description-td">52535.59</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Vibrio parahaemolyticus</i>
                </td>
                <td class="description-td">54384.91</td>
                <td class="description-td">34620.67</td>
                <td class="description-td">50064.19</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0111">Table 5 shows the expected molecular weight and base composition of region 16S<sub>—</sub>1100–1188 in <i>Mycobacterium avium </i>and <i>Streptomyces </i>sp.</p>
    <p num="p-0112">
      <tables id="TABLE-US-00006" num="00006"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <colspec colname="5" colwidth="56pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="5" rowsep="1" class="description-td" colspan="5">TABLE 5</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Molecular</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Region</td>
                <td class="description-td">Organism name</td>
                <td class="description-td">Length</td>
                <td class="description-td">weight</td>
                <td class="description-td">Base comp.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">16S_1100-1188</td>
                <td class="description-td">
                  <i>Mycobacterium avium</i>
                </td>
                <td class="description-td">82</td>
                <td class="description-td">25624.1728</td>
                <td class="description-td">A<sub>16</sub>G<sub>32</sub>C<sub>18</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">16S_1100-1188</td>
                <td class="description-td"> <i>Streptomyces </i>sp.</td>
                <td class="description-td">96</td>
                <td class="description-td">29904.871</td>
                <td class="description-td">A<sub>17</sub>G<sub>38</sub>C<sub>27</sub>T<sub>14</sub> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0113">Table 6 shows base composition (single strand) results for 16S<sub>—</sub>1100–1188 primer amplification reactions from different species of bacteria. Species which are repeated in the table (e.g., <i>Clostridium botulinum</i>) are different strains which have different base compositions in the 16S<sub>—</sub>1100–1188 region.</p>
    <p num="p-0114">
      <tables id="TABLE-US-00007" num="00007"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="left"> </colspec> <colspec colname="2" colwidth="105pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Organism name</td>
                <td class="description-td">Base comp.</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Mycobacterium avium</i>
                </td>
                <td class="description-td">A<sub>16</sub>G<sub>32</sub>C<sub>18</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> <i>Streptomyces </i>sp.</td>
                <td class="description-td">A<sub>17</sub>G<sub>38</sub>C<sub>27</sub>T<sub>14</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Ureaplasma urealyticum</i>
                </td>
                <td class="description-td">A<sub>18</sub>G<sub>30</sub>C<sub>17</sub>T<sub>17</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> <i>Streptomyces </i>sp.</td>
                <td class="description-td">A<sub>19</sub>G<sub>36</sub>C<sub>24</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Mycobacterium leprae</i>
                </td>
                <td class="description-td">A<sub>20</sub>G<sub>32</sub>C<sub>22</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00015.png"> <img id="CUSTOM-CHARACTER-00015" he="2.46mm" wi="13.72mm" file="US07217510-20070515-P00015.TIF" alt="Figure US07217510-20070515-P00015" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00015.png" class="patent-full-image" width="54" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>20</sub> <b>G</b>
                  <sub>33</sub> <b>C</b>
                  <sub>21</sub> <b>T</b>
                  <sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00016.png"> <img id="CUSTOM-CHARACTER-00016" he="2.46mm" wi="17.95mm" file="US07217510-20070515-P00016.TIF" alt="Figure US07217510-20070515-P00016" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00016.png" class="patent-full-image" width="71" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>20</sub> <b>G</b>
                  <sub>33</sub> <b>C</b>
                  <sub>21</sub> <b>T</b>
                  <sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Fusobacterium necroforum</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>26</sub>C<sub>22</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Listeria monocytogenes</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>27</sub>C<sub>19</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Clostridium botulinum</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>27</sub>C<sub>19</sub>T<sub>21</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Neisseria gonorrhoeae</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>28</sub>C<sub>21</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Bartonella quintana</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>30</sub>C<sub>22</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Enterococcus faecalis</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>27</sub>C<sub>20</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Bacillus megaterium</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>28</sub>C<sub>20</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Bacillus subtilis</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>28</sub>C<sub>21</sub>T<sub>17</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Pseudomonas aeruginosa</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>29</sub>C<sub>23</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Legionella pneumophila</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>32</sub>C<sub>20</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Mycoplasma pneumoniae</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>20</sub>C<sub>14</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Clostridium botulinum</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>26</sub>C<sub>20</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Enterococcus faecium</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>26</sub>C<sub>21</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Acinetobacter calcoaceti</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>26</sub>C<sub>21</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00017.png"> <img id="CUSTOM-CHARACTER-00017" he="2.46mm" wi="22.61mm" file="US07217510-20070515-P00017.TIF" alt="Figure US07217510-20070515-P00017" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00017.png" class="patent-full-image" width="90" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>23</sub> <b>G</b>
                  <sub>26</sub> <b>C</b>
                  <sub>24</sub> <b>T</b>
                  <sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00018.png"> <img id="CUSTOM-CHARACTER-00018" he="2.46mm" wi="20.83mm" file="US07217510-20070515-P00018.TIF" alt="Figure US07217510-20070515-P00018" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00018.png" class="patent-full-image" width="83" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>23</sub> <b>G</b>
                  <sub>26</sub> <b>C</b>
                  <sub>24</sub> <b>T</b>
                  <sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Clostridium perfringens</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>27</sub>C<sub>19</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00019.png"> <img id="CUSTOM-CHARACTER-00019" he="2.46mm" wi="16.59mm" file="US07217510-20070515-P00019.TIF" alt="Figure US07217510-20070515-P00019" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00019.png" class="patent-full-image" width="66" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>23</sub> <b>G</b>
                  <sub>27</sub> <b>C</b>
                  <sub>20</sub> <b>T</b>
                  <sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00020.png"> <img id="CUSTOM-CHARACTER-00020" he="2.46mm" wi="14.14mm" file="US07217510-20070515-P00020.TIF" alt="Figure US07217510-20070515-P00020" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00020.png" class="patent-full-image" width="56" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>23</sub> <b>G</b>
                  <sub>27</sub> <b>C</b>
                  <sub>20</sub> <b>T</b>
                  <sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00021.png"> <img id="CUSTOM-CHARACTER-00021" he="2.12mm" wi="20.07mm" file="US07217510-20070515-P00021.TIF" alt="Figure US07217510-20070515-P00021" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00021.png" class="patent-full-image" width="80" height="8"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>23</sub> <b>G</b>
                  <sub>27</sub> <b>C</b>
                  <sub>20</sub> <b>T</b>
                  <sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Aeromonas hydrophila</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>29</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Escherichia coli</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>29</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Pseudomonas putida</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>29</sub>C<sub>21</sub>T<sub>17</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00022.png"> <img id="CUSTOM-CHARACTER-00022" he="2.46mm" wi="15.16mm" file="US07217510-20070515-P00022.TIF" alt="Figure US07217510-20070515-P00022" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00022.png" class="patent-full-image" width="60" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>23</sub> <b>G</b>
                  <sub>29</sub> <b>C</b>
                  <sub>22</sub> <b>T</b>
                  <sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00023.png"> <img id="CUSTOM-CHARACTER-00023" he="2.46mm" wi="18.71mm" file="US07217510-20070515-P00023.TIF" alt="Figure US07217510-20070515-P00023" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00023.png" class="patent-full-image" width="74" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>23</sub> <b>G</b>
                  <sub>29</sub> <b>C</b>
                  <sub>22</sub> <b>T</b>
                  <sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Vibrio cholerae</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>30</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00024.png"> <img id="CUSTOM-CHARACTER-00024" he="2.46mm" wi="21.17mm" file="US07217510-20070515-P00024.TIF" alt="Figure US07217510-20070515-P00024" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00024.png" class="patent-full-image" width="84" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>23</sub> <b>G</b>
                  <sub>31</sub> <b>C</b>
                  <sub>21</sub> <b>T</b>
                  <sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00025.png"> <img id="CUSTOM-CHARACTER-00025" he="2.46mm" wi="22.18mm" file="US07217510-20070515-P00025.TIF" alt="Figure US07217510-20070515-P00025" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00025.png" class="patent-full-image" width="88" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>23</sub> <b>G</b>
                  <sub>31</sub> <b>C</b>
                  <sub>21</sub> <b>T</b>
                  <sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Mycoplasma genitalium</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>19</sub>C<sub>12</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Clostridium botulinum</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>25</sub>C<sub>18</sub>T<sub>20</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Bordetella bronchiseptica</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>26</sub>C<sub>19</sub>T<sub>14</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Francisella tularensis</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>26</sub>C<sub>19</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00026.png"> <img id="CUSTOM-CHARACTER-00026" he="2.46mm" wi="16.93mm" file="US07217510-20070515-P00026.TIF" alt="Figure US07217510-20070515-P00026" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00026.png" class="patent-full-image" width="67" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>26</sub> <b>C</b>
                  <sub>20</sub> <b>T</b>
                  <sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00027.png"> <img id="CUSTOM-CHARACTER-00027" he="2.12mm" wi="19.73mm" file="US07217510-20070515-P00027.TIF" alt="Figure US07217510-20070515-P00027" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00027.png" class="patent-full-image" width="78" height="8"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>26</sub> <b>C</b>
                  <sub>20</sub> <b>T</b>
                  <sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00028.png"> <img id="CUSTOM-CHARACTER-00028" he="2.12mm" wi="20.83mm" file="US07217510-20070515-P00028.TIF" alt="Figure US07217510-20070515-P00028" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00028.png" class="patent-full-image" width="83" height="8"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>26</sub> <b>C</b>
                  <sub>20</sub> <b>T</b>
                  <sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Helicobacter pylori</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>26</sub>C<sub>20</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Helicobacter pylori</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>26</sub>C<sub>21</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Moraxella catarrhalis</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>26</sub>C<sub>23</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> <i>Haemophilus influenzae </i>Rd</td>
                <td class="description-td">A<sub>24</sub>G<sub>28</sub>C<sub>20</sub>T<sub>17</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00029.png"> <img id="CUSTOM-CHARACTER-00029" he="2.12mm" wi="21.51mm" file="US07217510-20070515-P00029.TIF" alt="Figure US07217510-20070515-P00029" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00029.png" class="patent-full-image" width="86" height="8"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>28</sub> <b>C</b>
                  <sub>21</sub> <b>T</b>
                  <sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00030.png"> <img id="CUSTOM-CHARACTER-00030" he="2.12mm" wi="21.84mm" file="US07217510-20070515-P00030.TIF" alt="Figure US07217510-20070515-P00030" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00030.png" class="patent-full-image" width="87" height="8"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>28</sub> <b>C</b>
                  <sub>21</sub> <b>T</b>
                  <sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00031.png"> <img id="CUSTOM-CHARACTER-00031" he="2.46mm" wi="15.83mm" file="US07217510-20070515-P00031.TIF" alt="Figure US07217510-20070515-P00031" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00031.png" class="patent-full-image" width="63" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>28</sub> <b>C</b>
                  <sub>21</sub> <b>T</b>
                  <sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Pseudomonas putida</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>29</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00032.png"> <img id="CUSTOM-CHARACTER-00032" he="2.12mm" wi="15.16mm" file="US07217510-20070515-P00032.TIF" alt="Figure US07217510-20070515-P00032" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00032.png" class="patent-full-image" width="60" height="8"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>30</sub> <b>C</b>
                  <sub>21</sub> <b>T</b>
                  <sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00033.png"> <img id="CUSTOM-CHARACTER-00033" he="2.12mm" wi="13.04mm" file="US07217510-20070515-P00033.TIF" alt="Figure US07217510-20070515-P00033" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00033.png" class="patent-full-image" width="52" height="8"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>30</sub> <b>C</b>
                  <sub>21</sub> <b>T</b>
                  <sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00034.png"> <img id="CUSTOM-CHARACTER-00034" he="2.12mm" wi="23.96mm" file="US07217510-20070515-P00034.TIF" alt="Figure US07217510-20070515-P00034" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7217510B2/US07217510-20070515-P00034.png" class="patent-full-image" width="95" height="8"> </a> </div>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>30</sub> <b>C</b>
                  <sub>21</sub> <b>T</b>
                  <sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Clostridium botulinum</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>24</sub>C<sub>18</sub>T<sub>21</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Clostridium tetani</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>25</sub>C<sub>18</sub>T<sub>20</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Francisella tularensis</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>25</sub>C<sub>19</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Acinetobacter calcoacetic</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>26</sub>C<sub>20</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Bacteriodes fragilis</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>27</sub>C<sub>16</sub>T<sub>22</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Chlamydophila psittaci</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>27</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Borrelia burgdorferi</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>29</sub>C<sub>17</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Streptobacillus monilifor</i>
                </td>
                <td class="description-td">A<sub>26</sub>G<sub>26</sub>C<sub>20</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Rickettsia prowazekii</i>
                </td>
                <td class="description-td">A<sub>26</sub>G<sub>28</sub>C<sub>18</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Rickettsia rickettsii</i>
                </td>
                <td class="description-td">A<sub>26</sub>G<sub>28</sub>C<sub>20</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Mycoplasma mycoides</i>
                </td>
                <td class="description-td">A<sub>28</sub>G<sub>23</sub>C<sub>16</sub>T<sub>20</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0115">The same organism having different base compositions are different strains. Groups of organisms which are in bold have the same base compositions in the amplified region. Some of these organisms can be distinguished using multiple primers. For example, <i>Bacillus anthracis </i>can be distinguished from <i>Bacillus cereus </i>and <i>Bacillus thuringiensis </i>using the primer 16S<sub>—</sub>971–1062 (Table 7). Other primer pairs which produce unique base composition signatures are shown in Table 7 (bold). Clusters containing very similar threat and ubiquitous non-threat organisms (e.g. <i>anthracis </i>cluster) are distinguished at high resolution with focused sets of primer pairs. The known biowarfare agents in Table 7 are <i>Bacillus anthracis, Yersinia pestis, Francisella tularensis </i>and <i>Rickettsia prowazekii.</i> </p>
    <p num="p-0116">
      <tables id="TABLE-US-00008" num="00008"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="56pt" align="left"> </colspec> <colspec colname="4" colwidth="56pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="4" rowsep="1" class="description-td" colspan="4">TABLE 7</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Organism</td>
                <td class="description-td">16S_971-1062</td>
                <td class="description-td">16S_1228-1310</td>
                <td class="description-td">16S_1100-1188</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <i>Aeromonas hydrophila</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>29</sub>C<sub>22</sub>T<sub>20</sub> </td>
                <td class="description-td">A<sub>22</sub>G<sub>27</sub>C<sub>21</sub>T<sub>13</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>31</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Aeromonas salmonicida</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>29</sub>C<sub>22</sub>T<sub>20</sub> </td>
                <td class="description-td">A<sub>22</sub>G<sub>27</sub>C<sub>21</sub>T<sub>13</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>31</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus anthracis</i>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>21</sub> <b>G</b>
                  <sub>27</sub> <b>C</b>
                  <sub>22</sub> <b>T</b>
                  <sub>22</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>22</sub>C<sub>19</sub>T<sub>18</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>27</sub>C<sub>20</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus cereus</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>27</sub>C<sub>21</sub>T<sub>22</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>22</sub>C<sub>19</sub>T<sub>18</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>27</sub>C<sub>20</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus thuringiensis</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>27</sub>C<sub>21</sub>T<sub>22</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>22</sub>C<sub>19</sub>T<sub>18</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>27</sub>C<sub>20</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Chlamydia trachomatis</i>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>22</sub> <b>G</b>
                  <sub>26</sub> <b>C</b>
                  <sub>20</sub> <b>T</b>
                  <sub>23</sub> </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>23</sub> <b>C</b>
                  <sub>19</sub> <b>T</b>
                  <sub>16</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>28</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>Chlamydia pneumoniae </i>AR39</td>
                <td class="description-td">A<sub>26</sub>G<sub>23</sub>C<sub>20</sub>T<sub>22</sub> </td>
                <td class="description-td">A<sub>26</sub>G<sub>22</sub>C<sub>16</sub>T<sub>18</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>28</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Leptospira borgpetersenii</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>26</sub>C<sub>20</sub>T<sub>21</sub> </td>
                <td class="description-td">A<sub>22</sub>G<sub>25</sub>C<sub>21</sub>T<sub>15</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>26</sub>C<sub>24</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Leptospira interrogans</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>26</sub>C<sub>20</sub>T<sub>21</sub> </td>
                <td class="description-td">A<sub>22</sub>G<sub>25</sub>C<sub>21</sub>T<sub>15</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>26</sub>C<sub>24</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycoplasma genitalium</i>
                </td>
                <td class="description-td">A<sub>28</sub>G<sub>23</sub>C<sub>15</sub>T<sub>22</sub> </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>30</sub> <b>G</b>
                  <sub>18</sub> <b>C</b>
                  <sub>15</sub> <b>T</b>
                  <sub>19</sub> </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>19</sub> <b>C</b>
                  <sub>12</sub> <b>T</b>
                  <sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycoplasma pneumoniae</i>
                </td>
                <td class="description-td">A<sub>28</sub>G<sub>23</sub>C<sub>15</sub>T<sub>22</sub> </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>27</sub> <b>G</b>
                  <sub>19</sub> <b>C</b>
                  <sub>16</sub> <b>T</b>
                  <sub>20</sub> </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>23</sub> <b>G</b>
                  <sub>20</sub> <b>C</b>
                  <sub>14</sub> <b>T</b>
                  <sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Escherichia coli</i>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>22</sub> <b>G</b>
                  <sub>28</sub> <b>C</b>
                  <sub>20</sub> <b>T</b>
                  <sub>22</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>25</sub>C<sub>21</sub>T<sub>13</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>29</sub>C<sub>22</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Shigella dysenteriae</i>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>22</sub> <b>G</b>
                  <sub>28</sub> <b>C</b>
                  <sub>21</sub> <b>T</b>
                  <sub>21</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>25</sub>C<sub>21</sub>T<sub>13</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>29</sub>C<sub>22</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Proteus vulgaris</i>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>23</sub> <b>G</b>
                  <sub>26</sub> <b>C</b>
                  <sub>22</sub> <b>T</b>
                  <sub>21</sub> </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>26</sub> <b>G</b>
                  <sub>24</sub> <b>C</b>
                  <sub>19</sub> <b>T</b>
                  <sub>14</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>30</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Yersinia pestis</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>25</sub>C<sub>21</sub>T<sub>22</sub> </td>
                <td class="description-td">A<sub>25</sub>G<sub>24</sub>C<sub>20</sub>T<sub>14</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>30</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Yersinia pseudotuberculosis</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>25</sub>C<sub>21</sub>T<sub>22</sub> </td>
                <td class="description-td">A<sub>25</sub>G<sub>24</sub>C<sub>20</sub>T<sub>14</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>30</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Francisella tularensis</i>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>20</sub> <b>G</b>
                  <sub>25</sub> <b>C</b>
                  <sub>21</sub> <b>T</b>
                  <sub>23</sub> </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>23</sub> <b>G</b>
                  <sub>26</sub> <b>C</b>
                  <sub>17</sub> <b>T</b>
                  <sub>17</sub> </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>26</sub> <b>C</b>
                  <sub>19</sub> <b>T</b>
                  <sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Rickettsia prowazekii</i>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>21</sub> <b>G</b>
                  <sub>26</sub> <b>C</b>
                  <sub>24</sub> <b>T</b>
                  <sub>25</sub> </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>23</sub> <b>C</b>
                  <sub>16</sub> <b>T</b>
                  <sub>19</sub> </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>26</sub> <b>G</b>
                  <sub>28</sub> <b>C</b>
                  <sub>18</sub> <b>T</b>
                  <sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Rickettsia rickettsii</i>
                </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>21</sub> <b>G</b>
                  <sub>26</sub> <b>C</b>
                  <sub>25</sub> <b>T</b>
                  <sub>24</sub> </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>24</sub> <b>G</b>
                  <sub>24</sub> <b>C</b>
                  <sub>17</sub> <b>T</b>
                  <sub>17</sub> </td>
                <td class="description-td">
                  <b>A</b>
                  <sub>26</sub> <b>G</b>
                  <sub>28</sub> <b>C</b>
                  <sub>20</sub> <b>T</b>
                  <sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0117">The sequence of <i>B. anthracis </i>and <i>B. cereus </i>in region 16S<sub>—</sub>971 is shown below. Shown in bold is the single base difference between the two species which can be detected using the methods of the present invention. <i>B. anthracis </i>has an ambiguous base at position 20.</p>
    <p num="p-0118">
      <tables id="TABLE-US-00009" num="00009"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="161pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> <i>B.anthracis</i>_16S_971</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">GCGAAGAACCUUACCAGGU<i>N</i>UUGACAUCCUCUGA<b>C</b>AA</td>
                <td class="description-td">(SEQ ID NO:1)</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCCUAGAGAUAGGGCUUCUCCUUCGGGAGCAGAGUGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAGGUGGUGCAUGGUU </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>B.cereus</i>_16S_971</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCGAAGAACCUUACCAGGUCUUGACAUCCUCUGA<b>A</b>AA</td>
                <td class="description-td">(SEQ ID NO:2)</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCCUAGAGAUAGGGCUUCUCCUUCGGGAGCAGAGUGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAGGUGGUGCAUGGUU </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 6</heading> <heading>ESI-TOF MS of sspE 56-mer Plus Calibrant</heading> <p num="p-0119">The mass measurement accuracy that can be obtained using an internal mass standard in the ESI-MS study of PCR products is shown in <figref idrefs="DRAWINGS">FIG. 8</figref>. The mass standard was a 20-mer phosphorothioate oligonucleotide added to a solution containing a 56-mer PCR product from the <i>B. anthracis </i>spore coat protein sspE. The mass of the expected PCR product distinguishes <i>B. anthracis </i>from other species of <i>Bacillus </i>such as <i>B. thuringiensis </i>and <i>B. cereus. </i> </p>
    <heading>Example 7</heading> <heading> <i>B. anthracis </i>ESI-TOF Synthetic 16S<sub>—</sub>1228 Duplex</heading> <p num="p-0120">An ESI-TOF MS spectrum was obtained from an aqueous solution containing 5 μM each of synthetic analogs of the expected forward and reverse PCR products from the nucleotide 1228 region of the <i>B. anthracis </i>16S rRNA gene. The results (<figref idrefs="DRAWINGS">FIG. 9</figref>) show that the molecular weights of the forward and reverse strands can be accurately determined and easily distinguish the two strands. The [M−21H<sup>+</sup>]<sup>21−</sup> and [M−20H<sup>+</sup>]<sup>20−</sup> charge states are shown.</p>
    <heading>Example 8</heading> <heading>ESI-FTICR-MS of Synthetic <i>B. anthracis </i>16S<sub>—</sub>1337 46 Base Pair Duplex</heading> <p num="p-0121">An ESI-FTICR-MS spectrum was obtained from an aqueous solution containing 5 μM each of synthetic analogs of the expected forward and reverse PCR products from the nucleotide 1337 region of the <i>B. anthracis </i>16S rRNA gene. The results (<figref idrefs="DRAWINGS">FIG. 10</figref>) show that the molecular weights of the strands can be distinguished by this method. The [M−16H<sup>+</sup>]<sup>16−</sup> through [M−10H<sup>+</sup>]<sup>10−</sup> charge states are shown. The insert highlights the resolution that can be realized on the FTICR-MS instrument, which allows the charge state of the ion to be determined from the mass difference between peaks differing by a single 13C substitution.</p>
    <heading>Example 9</heading> <heading>ESI-TOF MS of 56-mer Oligonucleotide from saspB Gene of <i>B. anthracis </i>with Internal Mass Standard</heading> <p num="p-0122">ESI-TOF MS spectra were obtained on a synthetic 56-mer oligonucleotide (5 μM )from the saspB gene of <i>B. anthracis </i>containing an internal mass standard at an ESI of 1.7 μL/min as a function of sample consumption. The results (<figref idrefs="DRAWINGS">FIG. 11</figref>) show that the signal to noise is improved as more scans are summed, and that the standard and the product are visible after only 100 scans.</p>
    <heading>Example 10</heading> <heading>ESI-TOF MS of an Internal Standard with Tributylammonium (TBA)-trifluoroacetate  (TFA) Buffer</heading> <p num="p-0123">An ESI-TOF-MS spectrum of a 20-mer phosphorothioate mass standard was obtained following addition of 5 mM TBA-TFA buffer to the solution. This buffer strips charge from the oligonucleotide and shifts the most abundant charge state from [M−8H<sup>+</sup>]<sup>8−</sup> to [M−3H<sup>+</sup>]<sup>3−</sup> (<figref idrefs="DRAWINGS">FIG. 12</figref>).</p>
    <heading>Example 11</heading> <heading>Master Database Comparison</heading> <p num="p-0124">The molecular masses obtained through Examples 1–10 are compared to molecular masses of known bioagents stored in a master database to obtain a high probability matching molecular mass.</p>
    <heading>Example 12</heading> <heading>Master Data Base Interrogation Over the Internet</heading> <p num="p-0125">The same procedure as in Example 11 is followed except that the local computer did not store the Master database. The Master database is interrogated over an internet connection, searching for a molecular mass match.</p>
    <heading>Example 13</heading> <heading>Master Database Updating</heading> <p num="p-0126">The same procedure as in example 11 is followed except the local computer is connected to the internet and has the ability to store a master database locally. The local computer system periodically, or at the user's discretion, interrogates the Master database, synchronizing the local master database with the global Master database. This provides the current molecular mass information to both the local database as well as to the global Master database. This further provides more of a globalized knowledge base.</p>
    <heading>Example 14</heading> <heading>Global Database Updating</heading> <p num="p-0127">The same procedure as in example 13 is followed except there are numerous such local stations throughout the world. The synchronization of each database adds to the diversity of information and diversity of the molecular masses of known bioagents.</p>
    <p num="p-0128">Various modifications of the invention, in addition to those described herein, will be apparent to those skilled in the art from the foregoing description. Such modifications are also intended to fall within the scope of the appended claims.</p>
    <p num="p-0129">
      <tables id="TABLE-US-00010" num="00010"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="287pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="2" rowsep="1" class="description-td" colspan="2">TABLE 8</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <i>E. coli</i>
                </td>
                <td class="description-td">AUGU-CUGGGA-AACUGCCUGAUGG&lt;--------------&gt;AGG-GGGAUAACUACU-G (GA-AA)</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Hom. Sapien</i>
                </td>
                <td class="description-td">AUC-AGUUAUGGUU-CCUUUGGUCG&lt;CUCGCUCCUCUCCU&gt;ACU-UGGAUAACUGUG-G (UAAUU)</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>mt. H sapien</i>
                </td>
                <td class="description-td"> <i>CAC</i>-<i>CCUCUA</i>-<i>AAUC</i>----------&lt;--------------&gt;--------------------------</td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>Act. Israe</i>5</td>
                <td class="description-td">CAC-GUGAGUAACC-UGCCCCUCAC&lt;--------------&gt;UUC-UGGAUAACCGCU-U (GA-AA)</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Cor. diphth</i>
                </td>
                <td class="description-td">CAC-GUGGGUGAUC-UGCCUCGUAC&lt;--------------&gt;UUC-GGGAUAAGCCUG-G (GA-AA)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>Myb. avium</i>2</td>
                <td class="description-td">CAC-GUGGGCAAUC-UACCCUGCAC&lt;--------------&gt;UUC-GGGAUAAGCCUG-G (GA-AA)</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Myb. leprae</i>
                </td>
                <td class="description-td">CAC-GUGGGUAAUC-UGCCCUGCAC&lt;--------------&gt;UUCAGGGAUAAGCUUG-G (GA-AA)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>Myb. tuber</i>3</td>
                <td class="description-td">CAC-GUGGGUGAUC-UGCCCUGCAC&lt;--------------&gt;UUC-GGGAUAAGCCUG-G (GA-AA)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>Noc. aster</i>4</td>
                <td class="description-td">CAC-GUGGGUGAUC-UGCCUCGUAC&lt;--------------&gt;UUC-GGGAUAAGCCUG-G (GA-AA)</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Stm. acidsc</i>
                </td>
                <td class="description-td">CAC-GUGGGCAAUC-UGCCCUUCAC&lt;--------------&gt;UCU-GGGACAAGCCCU-G (GA-AA)</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Stm. albflv</i>
                </td>
                <td class="description-td">CAC-GUGGGCAAUC-UGCCCUGCAC&lt;--------------&gt;UCU-GGGACAAGCCCU-G (GA-AA)</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Stm. albus</i>
                </td>
                <td class="description-td">CAC-GUGGGCAAUC-UGCCCUGCAC&lt;--------------&gt;UCU-GGGACAAGCCCU-G (GA-AA)</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Stm. ambofa</i>
                </td>
                <td class="description-td">CAC-GUGGGCAAUC-UGCCCUGCAC&lt;--------------&gt;UCU-GGGACAAGCCCU-G (GA-AA)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5484908">US5484908</a></td><td class="patent-data-table-td patent-date-value">Nov 26, 1991</td><td class="patent-data-table-td patent-date-value">Jan 16, 1996</td><td class="patent-data-table-td ">Gilead Sciences, Inc.</td><td class="patent-data-table-td ">Uracil or cytosine derivatives, detection of dna sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5502177">US5502177</a></td><td class="patent-data-table-td patent-date-value">Sep 17, 1993</td><td class="patent-data-table-td patent-date-value">Mar 26, 1996</td><td class="patent-data-table-td ">Gilead Sciences, Inc.</td><td class="patent-data-table-td ">Pyrimidine derivatives for labeled binding partners</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5503980">US5503980</a></td><td class="patent-data-table-td patent-date-value">Oct 17, 1994</td><td class="patent-data-table-td patent-date-value">Apr 2, 1996</td><td class="patent-data-table-td ">Trustees Of Boston University</td><td class="patent-data-table-td ">Determining nucleotide sequence of nucleic acid target using nucleic acid probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5527675">US5527675</a></td><td class="patent-data-table-td patent-date-value">Aug 20, 1993</td><td class="patent-data-table-td patent-date-value">Jun 18, 1996</td><td class="patent-data-table-td ">Millipore Corporation</td><td class="patent-data-table-td ">Method for degradation and sequencing of polymers which sequentially eliminate terminal residues</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5547835">US5547835</a></td><td class="patent-data-table-td patent-date-value">Jan 6, 1994</td><td class="patent-data-table-td patent-date-value">Aug 20, 1996</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Using the sanger sequencing strategy and assembling the sequence information by analysis of the nested fragments obtained by base-specific chain termination via their different molecular masses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5580733">US5580733</a></td><td class="patent-data-table-td patent-date-value">Sep 6, 1994</td><td class="patent-data-table-td patent-date-value">Dec 3, 1996</td><td class="patent-data-table-td ">Wayne State University</td><td class="patent-data-table-td ">Illumination mixture of molecule on matrix using laser</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5605798">US5605798</a></td><td class="patent-data-table-td patent-date-value">Mar 17, 1995</td><td class="patent-data-table-td patent-date-value">Feb 25, 1997</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Detecting target nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5622824">US5622824</a></td><td class="patent-data-table-td patent-date-value">Feb 10, 1995</td><td class="patent-data-table-td patent-date-value">Apr 22, 1997</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA sequencing by mass spectrometry via exonuclease degradation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5625184">US5625184</a></td><td class="patent-data-table-td patent-date-value">May 19, 1995</td><td class="patent-data-table-td patent-date-value">Apr 29, 1997</td><td class="patent-data-table-td ">Perseptive Biosystems, Inc.</td><td class="patent-data-table-td ">Time-of-flight mass spectrometry analysis of biomolecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5645985">US5645985</a></td><td class="patent-data-table-td patent-date-value">Nov 25, 1992</td><td class="patent-data-table-td patent-date-value">Jul 8, 1997</td><td class="patent-data-table-td ">Gilead Sciences, Inc.</td><td class="patent-data-table-td ">Used for diagnosis to detect viruses or diseased conditions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5686242">US5686242</a></td><td class="patent-data-table-td patent-date-value">Oct 27, 1994</td><td class="patent-data-table-td patent-date-value">Nov 11, 1997</td><td class="patent-data-table-td ">Isis Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Determination of oligonucleotides for therapeutics, diagnostics and research reagents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5691141">US5691141</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 1995</td><td class="patent-data-table-td patent-date-value">Nov 25, 1997</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA sequencing by mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5700642">US5700642</a></td><td class="patent-data-table-td patent-date-value">May 22, 1995</td><td class="patent-data-table-td patent-date-value">Dec 23, 1997</td><td class="patent-data-table-td ">Sri International</td><td class="patent-data-table-td ">Oligonucleotide sizing using immobilized cleavable primers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5763588">US5763588</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Jun 9, 1998</td><td class="patent-data-table-td ">Gilead Sciences, Inc.</td><td class="patent-data-table-td ">Pyrimidine derivatives for labeled binding partners</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5770367">US5770367</a></td><td class="patent-data-table-td patent-date-value">Aug 1, 1994</td><td class="patent-data-table-td patent-date-value">Jun 23, 1998</td><td class="patent-data-table-td ">Oxford Gene Technology Limited</td><td class="patent-data-table-td ">Tag reagent and assay method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5777324">US5777324</a></td><td class="patent-data-table-td patent-date-value">Sep 19, 1996</td><td class="patent-data-table-td patent-date-value">Jul 7, 1998</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Method and apparatus for maldi analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5830653">US5830653</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Nov 3, 1998</td><td class="patent-data-table-td ">Gilead Sciences, Inc.</td><td class="patent-data-table-td ">Methods of using oligomers containing modified pyrimidines</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5830655">US5830655</a></td><td class="patent-data-table-td patent-date-value">Apr 26, 1996</td><td class="patent-data-table-td patent-date-value">Nov 3, 1998</td><td class="patent-data-table-td ">Sri International</td><td class="patent-data-table-td ">Oligonucleotide sizing using cleavable primers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5849492">US5849492</a></td><td class="patent-data-table-td patent-date-value">Oct 29, 1996</td><td class="patent-data-table-td patent-date-value">Dec 15, 1998</td><td class="patent-data-table-td ">Phylogenetix Laboratories, Inc.</td><td class="patent-data-table-td ">Method for rapid identification of prokaryotic and eukaryotic organisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5851765">US5851765</a></td><td class="patent-data-table-td patent-date-value">May 30, 1995</td><td class="patent-data-table-td patent-date-value">Dec 22, 1998</td><td class="patent-data-table-td ">Sequenon, Inc.</td><td class="patent-data-table-td ">DNA sequencing by mass spectrometry via exonuclease degradation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5864137">US5864137</a></td><td class="patent-data-table-td patent-date-value">Oct 1, 1996</td><td class="patent-data-table-td patent-date-value">Jan 26, 1999</td><td class="patent-data-table-td ">Genetrace Systems, Inc.</td><td class="patent-data-table-td ">Mass spectrometer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5869242">US5869242</a></td><td class="patent-data-table-td patent-date-value">Sep 18, 1995</td><td class="patent-data-table-td patent-date-value">Feb 9, 1999</td><td class="patent-data-table-td ">Myriad Genetics, Inc.</td><td class="patent-data-table-td ">Rapid nonsequencing analysis of denatured nucleic acid segments for determination of mutations by comparision to wild-type and known variations; for genetic screening and diagnosis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5871697">US5871697</a></td><td class="patent-data-table-td patent-date-value">Oct 24, 1995</td><td class="patent-data-table-td patent-date-value">Feb 16, 1999</td><td class="patent-data-table-td ">Curagen Corporation</td><td class="patent-data-table-td ">Method and apparatus for identifying, classifying, or quantifying DNA sequences in a sample without sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5872003">US5872003</a></td><td class="patent-data-table-td patent-date-value">May 30, 1995</td><td class="patent-data-table-td patent-date-value">Feb 16, 1999</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA sequencing by mass spectrometry via exonuclease degradation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5876936">US5876936</a></td><td class="patent-data-table-td patent-date-value">Jan 15, 1997</td><td class="patent-data-table-td patent-date-value">Mar 2, 1999</td><td class="patent-data-table-td ">Incyte Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Nucleic acid sequencing with solid phase capturable terminators</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5928906">US5928906</a></td><td class="patent-data-table-td patent-date-value">May 9, 1996</td><td class="patent-data-table-td patent-date-value">Jul 27, 1999</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Process for direct sequencing during template amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5965363">US5965363</a></td><td class="patent-data-table-td patent-date-value">Dec 2, 1996</td><td class="patent-data-table-td patent-date-value">Oct 12, 1999</td><td class="patent-data-table-td ">Genetrace Systems Inc.</td><td class="patent-data-table-td ">Methods of preparing nucleic acids for mass spectrometric analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5981176">US5981176</a></td><td class="patent-data-table-td patent-date-value">Jun 17, 1992</td><td class="patent-data-table-td patent-date-value">Nov 9, 1999</td><td class="patent-data-table-td ">City Of Hope</td><td class="patent-data-table-td ">Allele primer complementary to a target dna sequence and immobilized to a solid support; tool for diagnosis and genetic analysis; detection of genetic disorders such as sickle cell anemia and thalasemia</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5994066">US5994066</a></td><td class="patent-data-table-td patent-date-value">Nov 4, 1996</td><td class="patent-data-table-td patent-date-value">Nov 30, 1999</td><td class="patent-data-table-td ">Infectio Diagnostic, Inc.</td><td class="patent-data-table-td ">Species-specific and universal DNA probes and amplification primers to rapidly detect and identify common bacterial pathogens and associated antibiotic resistance genes from clinical specimens for routine diagnosis in microbiology laboratories</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6001564">US6001564</a></td><td class="patent-data-table-td patent-date-value">Sep 11, 1995</td><td class="patent-data-table-td patent-date-value">Dec 14, 1999</td><td class="patent-data-table-td ">Infectio Diagnostic, Inc.</td><td class="patent-data-table-td ">Using oligionucleotides to determine quanity of nucleic acids; contacting sample with oligonucleotides, detecting hybridized complexes or amplified bacterial products with antibiotic resistance gene; for the detection of parasites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6005096">US6005096</a></td><td class="patent-data-table-td patent-date-value">May 8, 1995</td><td class="patent-data-table-td patent-date-value">Dec 21, 1999</td><td class="patent-data-table-td ">Gilead Sciences, Inc.</td><td class="patent-data-table-td ">For oligonucleotides that are modified to enhance the binding affinity and to bear detectable labels; for diagnostic use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6007992">US6007992</a></td><td class="patent-data-table-td patent-date-value">Nov 10, 1997</td><td class="patent-data-table-td patent-date-value">Dec 28, 1999</td><td class="patent-data-table-td ">Gilead Sciences, Inc.</td><td class="patent-data-table-td ">Pyrimidine derivatives for labeled binding partners</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6018713">US6018713</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 9, 1998</td><td class="patent-data-table-td patent-date-value">Jan 25, 2000</td><td class="patent-data-table-td ">Coli; Robert D.</td><td class="patent-data-table-td ">Integrated system and method for ordering and cumulative results reporting of medical tests</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6028183">US6028183</a></td><td class="patent-data-table-td patent-date-value">Nov 7, 1997</td><td class="patent-data-table-td patent-date-value">Feb 22, 2000</td><td class="patent-data-table-td ">Gilead Sciences, Inc.</td><td class="patent-data-table-td ">Pyrimidine derivatives and oligonucleotides containing same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6043031">US6043031</a></td><td class="patent-data-table-td patent-date-value">Mar 18, 1996</td><td class="patent-data-table-td patent-date-value">Mar 28, 2000</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA diagnostics based on mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6046005">US6046005</a></td><td class="patent-data-table-td patent-date-value">Sep 16, 1998</td><td class="patent-data-table-td patent-date-value">Apr 4, 2000</td><td class="patent-data-table-td ">Incyte Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Nucleic acid sequencing with solid phase capturable terminators comprising a cleavable linking group</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6051378">US6051378</a></td><td class="patent-data-table-td patent-date-value">Mar 4, 1997</td><td class="patent-data-table-td patent-date-value">Apr 18, 2000</td><td class="patent-data-table-td ">Genetrace Systems Inc.</td><td class="patent-data-table-td ">Methods of screening nucleic acids using mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6054278">US6054278</a></td><td class="patent-data-table-td patent-date-value">May 5, 1998</td><td class="patent-data-table-td patent-date-value">Apr 25, 2000</td><td class="patent-data-table-td ">The Perkin-Elmer Corporation</td><td class="patent-data-table-td ">Classification of a bacterial isolate based on ribonucleic acid sequences and computer analysis; comparison of ribonucleic acid sequences of bacterial isolate and those stored in a computer database</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6055487">US6055487</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 20, 1997</td><td class="patent-data-table-td patent-date-value">Apr 25, 2000</td><td class="patent-data-table-td ">Margery; Keith S.</td><td class="patent-data-table-td ">Interactive remote sample analysis system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6074823">US6074823</a></td><td class="patent-data-table-td patent-date-value">Nov 6, 1996</td><td class="patent-data-table-td patent-date-value">Jun 13, 2000</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA sequencing by mass spectrometry via exonuclease degradation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6090558">US6090558</a></td><td class="patent-data-table-td patent-date-value">Sep 18, 1998</td><td class="patent-data-table-td patent-date-value">Jul 18, 2000</td><td class="patent-data-table-td ">Genetrace Systems, Inc.</td><td class="patent-data-table-td ">Identification of more than one dna tandem nucleotide repeat region from more than one loci by polymerase chain reaction followed by concurrent determination of mass of the extension products; forensics; paternity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6104028">US6104028</a></td><td class="patent-data-table-td patent-date-value">May 29, 1998</td><td class="patent-data-table-td patent-date-value">Aug 15, 2000</td><td class="patent-data-table-td ">Genetrace Systems Inc.</td><td class="patent-data-table-td ">Volatile matrices for matrix-assisted laser desorption/ionization mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6111251">US6111251</a></td><td class="patent-data-table-td patent-date-value">Sep 19, 1997</td><td class="patent-data-table-td patent-date-value">Aug 29, 2000</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Method and apparatus for MALDI analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6140053">US6140053</a></td><td class="patent-data-table-td patent-date-value">Sep 25, 1998</td><td class="patent-data-table-td patent-date-value">Oct 31, 2000</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Determining a sequence of a nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6146144">US6146144</a></td><td class="patent-data-table-td patent-date-value">Sep 24, 1999</td><td class="patent-data-table-td patent-date-value">Nov 14, 2000</td><td class="patent-data-table-td ">Fowler; Ernest R.</td><td class="patent-data-table-td ">Rug hooking kit and method for handicapped</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6153389">US6153389</a></td><td class="patent-data-table-td patent-date-value">Feb 22, 1999</td><td class="patent-data-table-td patent-date-value">Nov 28, 2000</td><td class="patent-data-table-td ">Haarer; Brian K.</td><td class="patent-data-table-td ">Collecting sample, introducing fragment of dna of known length and sequence into the sample; dna fragment does not interfere with subsequent analysis; label for forensic or medical samples including blood, semen, saliva, urine, tissue</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6159681">US6159681</a></td><td class="patent-data-table-td patent-date-value">May 27, 1999</td><td class="patent-data-table-td patent-date-value">Dec 12, 2000</td><td class="patent-data-table-td ">Syntrix Biochip, Inc.</td><td class="patent-data-table-td ">Light-mediated method and apparatus for the regional analysis of biologic material</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6197498">US6197498</a></td><td class="patent-data-table-td patent-date-value">Apr 6, 1999</td><td class="patent-data-table-td patent-date-value">Mar 6, 2001</td><td class="patent-data-table-td ">Sequenom, Inc</td><td class="patent-data-table-td ">DNA diagnostics based on mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6218118">US6218118</a></td><td class="patent-data-table-td patent-date-value">Jul 9, 1998</td><td class="patent-data-table-td patent-date-value">Apr 17, 2001</td><td class="patent-data-table-td ">Agilent Technologies, Inc.</td><td class="patent-data-table-td ">Method and mixture reagents for analyzing the nucleotide sequence of nucleic acids by mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6221587">US6221587</a></td><td class="patent-data-table-td patent-date-value">May 12, 1998</td><td class="patent-data-table-td patent-date-value">Apr 24, 2001</td><td class="patent-data-table-td ">Isis Pharmceuticals, Inc.</td><td class="patent-data-table-td ">Detecting activation positionin nucleotide sequences; comparing nucleotide sequences and detecting conserved nucleotide sequence structures</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6221601">US6221601</a></td><td class="patent-data-table-td patent-date-value">Nov 2, 1999</td><td class="patent-data-table-td patent-date-value">Apr 24, 2001</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA diagnostics based on mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6221605">US6221605</a></td><td class="patent-data-table-td patent-date-value">Feb 15, 2000</td><td class="patent-data-table-td patent-date-value">Apr 24, 2001</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA diagnostics based on mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6225450">US6225450</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">May 1, 2001</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Intact positively charged ionized mass-modified nucleic acid molecule</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6235476">US6235476</a></td><td class="patent-data-table-td patent-date-value">Aug 18, 1997</td><td class="patent-data-table-td patent-date-value">May 22, 2001</td><td class="patent-data-table-td ">Dako A/S</td><td class="patent-data-table-td ">Process for detecting nucleic acids by mass determination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6235478">US6235478</a></td><td class="patent-data-table-td patent-date-value">Apr 6, 1999</td><td class="patent-data-table-td patent-date-value">May 22, 2001</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA diagnostics based on mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6235480">US6235480</a></td><td class="patent-data-table-td patent-date-value">Jul 21, 1999</td><td class="patent-data-table-td patent-date-value">May 22, 2001</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Screening for the presence of target nucleotide sequences; incubate preferential nucleotide sequences with enzymes and screen for preferentially released nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6238871">US6238871</a></td><td class="patent-data-table-td patent-date-value">May 8, 2000</td><td class="patent-data-table-td patent-date-value">May 29, 2001</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA sequences by mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6238927">US6238927</a></td><td class="patent-data-table-td patent-date-value">Oct 4, 1999</td><td class="patent-data-table-td patent-date-value">May 29, 2001</td><td class="patent-data-table-td ">Mosaic Technologies, Incorporated</td><td class="patent-data-table-td ">Hybridizing probe to tether nucleic acid which has been immobilized to electrophoretic polymeric medium, migrating test nucleic acid through medium and hybridizing with probe causing displacement of tether, detecting probe-target hybrid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6258538">US6258538</a></td><td class="patent-data-table-td patent-date-value">Apr 6, 1999</td><td class="patent-data-table-td patent-date-value">Jul 10, 2001</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA diagnostics based on mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6265716">US6265716</a></td><td class="patent-data-table-td patent-date-value">Feb 29, 2000</td><td class="patent-data-table-td patent-date-value">Jul 24, 2001</td><td class="patent-data-table-td ">Genetrace Systems, Inc.</td><td class="patent-data-table-td ">Volatile matrices for matrix-assisted laser desorption/ionization mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6268129">US6268129</a></td><td class="patent-data-table-td patent-date-value">Mar 4, 1996</td><td class="patent-data-table-td patent-date-value">Jul 31, 2001</td><td class="patent-data-table-td ">Imperial Cancer Research Technology Limited</td><td class="patent-data-table-td ">Method of nucleic acid analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6268131">US6268131</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 1997</td><td class="patent-data-table-td patent-date-value">Jul 31, 2001</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Immobilizing nucleic acid promoter to solid support; hybridization; transcription</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6268144">US6268144</a></td><td class="patent-data-table-td patent-date-value">Sep 15, 1999</td><td class="patent-data-table-td patent-date-value">Jul 31, 2001</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Detecting target nucleotide sequence in sample; obtain nucleotide sequences, immobilize nucleotide sequences on solid support, hybridize probes, ionize and volatize hybridization products, detect sequences by mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6268146">US6268146</a></td><td class="patent-data-table-td patent-date-value">Oct 22, 1999</td><td class="patent-data-table-td patent-date-value">Jul 31, 2001</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Analytical methods and materials for nucleic acid detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6270973">US6270973</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 1999</td><td class="patent-data-table-td patent-date-value">Aug 7, 2001</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Monitoring nucleotide sequences in a sample; mix nucleotide sequences and probes, mix sample with a depolymerizing enzyme, incubate, analyze sample for preferential sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6270974">US6270974</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 1999</td><td class="patent-data-table-td patent-date-value">Aug 7, 2001</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Exogenous nucleic acid detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6277573">US6277573</a></td><td class="patent-data-table-td patent-date-value">Apr 6, 1999</td><td class="patent-data-table-td patent-date-value">Aug 21, 2001</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Detecting nucleic acid molecules and sequences for diagnosis of genetic disease or chromosomal abnormality, a predisposition to a disease, or infection by a pathogenic organism; high speed and accurate</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6277578">US6277578</a></td><td class="patent-data-table-td patent-date-value">Oct 29, 1999</td><td class="patent-data-table-td patent-date-value">Aug 21, 2001</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Deploymerization method for nucleic acid detection of an amplified nucleic acid target</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6300076">US6300076</a></td><td class="patent-data-table-td patent-date-value">Jan 31, 2000</td><td class="patent-data-table-td patent-date-value">Oct 9, 2001</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Diagnostic detecting method of target nucleic acid in biological sample by amplifying nucleic acid, ionizing and volatilizing the product then analyze nucleic acid by mass spectrometry to detect presence of nucleic acid in the sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6312893">US6312893</a></td><td class="patent-data-table-td patent-date-value">Jul 22, 1997</td><td class="patent-data-table-td patent-date-value">Nov 6, 2001</td><td class="patent-data-table-td ">Qiagen Genomics, Inc.</td><td class="patent-data-table-td ">Methods and compositions for determining the sequence of nucleic acid molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6312902">US6312902</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 1999</td><td class="patent-data-table-td patent-date-value">Nov 6, 2001</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Nucleic acid detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6361940">US6361940</a></td><td class="patent-data-table-td patent-date-value">Apr 1, 1998</td><td class="patent-data-table-td patent-date-value">Mar 26, 2002</td><td class="patent-data-table-td ">Qiagen Genomics, Inc.</td><td class="patent-data-table-td ">Compositions and methods for enhancing hybridization and priming specificity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6372424">US6372424</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 1995</td><td class="patent-data-table-td patent-date-value">Apr 16, 2002</td><td class="patent-data-table-td ">Third Wave Technologies, Inc</td><td class="patent-data-table-td ">Rapid detection and identification of pathogens</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6391551">US6391551</a></td><td class="patent-data-table-td patent-date-value">Aug 25, 1999</td><td class="patent-data-table-td patent-date-value">May 21, 2002</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Simple, automatable method of determining presence of endogenous target sequence in sample by mixing target sequence hybridized with 3&#39;-labeled probe with enzyme that releases nucleotides from 3&#39;-end, and analyzing for label</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6393367">US6393367</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 19, 2000</td><td class="patent-data-table-td patent-date-value">May 21, 2002</td><td class="patent-data-table-td ">Proteometrics, Llc</td><td class="patent-data-table-td ">Method for evaluating the quality of comparisons between experimental and theoretical mass data</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6423966">US6423966</a></td><td class="patent-data-table-td patent-date-value">Jan 14, 2000</td><td class="patent-data-table-td patent-date-value">Jul 23, 2002</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Method and apparatus for maldi analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6428955">US6428955</a></td><td class="patent-data-table-td patent-date-value">Nov 6, 1996</td><td class="patent-data-table-td patent-date-value">Aug 6, 2002</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA diagnostics based on mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6432651">US6432651</a></td><td class="patent-data-table-td patent-date-value">Jul 9, 1999</td><td class="patent-data-table-td patent-date-value">Aug 13, 2002</td><td class="patent-data-table-td ">Cetek Corporation</td><td class="patent-data-table-td ">Screening complex material for ligands and binding strength</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6436635">US6436635</a></td><td class="patent-data-table-td patent-date-value">Mar 12, 1996</td><td class="patent-data-table-td patent-date-value">Aug 20, 2002</td><td class="patent-data-table-td ">Boston University</td><td class="patent-data-table-td ">Solid phase sequencing of double-stranded nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6436640">US6436640</a></td><td class="patent-data-table-td patent-date-value">Mar 18, 2000</td><td class="patent-data-table-td patent-date-value">Aug 20, 2002</td><td class="patent-data-table-td ">Exiqon A/S</td><td class="patent-data-table-td ">Use of LNA in mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6458533">US6458533</a></td><td class="patent-data-table-td patent-date-value">Dec 22, 1998</td><td class="patent-data-table-td patent-date-value">Oct 1, 2002</td><td class="patent-data-table-td ">High Throughput Genomics, Inc.</td><td class="patent-data-table-td ">Determining which of a group of expressed sequence tags are complementary to a specific nucleic acid using biochip of immobilized oligonucoleotide probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6468748">US6468748</a></td><td class="patent-data-table-td patent-date-value">Feb 29, 2000</td><td class="patent-data-table-td patent-date-value">Oct 22, 2002</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Methods of screening nucleic acids using volatile salts in mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6475143">US6475143</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 14, 2001</td><td class="patent-data-table-td patent-date-value">Nov 5, 2002</td><td class="patent-data-table-td ">First Opinion Corporation</td><td class="patent-data-table-td ">Automated diagnostic system and method including encoding patient data</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6475736">US6475736</a></td><td class="patent-data-table-td patent-date-value">Oct 25, 2000</td><td class="patent-data-table-td patent-date-value">Nov 5, 2002</td><td class="patent-data-table-td ">Variagenics, Inc.</td><td class="patent-data-table-td ">Methods for genetic analysis of DNA using biased amplification of polymorphic sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6479239">US6479239</a></td><td class="patent-data-table-td patent-date-value">May 16, 2000</td><td class="patent-data-table-td patent-date-value">Nov 12, 2002</td><td class="patent-data-table-td ">Large Scale Biology Corporation</td><td class="patent-data-table-td ">Detection and characterization of microorganisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6500621">US6500621</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2001</td><td class="patent-data-table-td patent-date-value">Dec 31, 2002</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Detection uisng mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6558902">US6558902</a></td><td class="patent-data-table-td patent-date-value">May 7, 1999</td><td class="patent-data-table-td patent-date-value">May 6, 2003</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Infrared matrix-assisted laser desorption/ionization mass spectrometric analysis of macromolecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6566055">US6566055</a></td><td class="patent-data-table-td patent-date-value">Jun 3, 1998</td><td class="patent-data-table-td patent-date-value">May 20, 2003</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Mass spectrometric analysis of target nucleic acids in sample; obtain nucleotide sequences, incubate with primers, amplify, decrease length and determine mass</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6582916">US6582916</a></td><td class="patent-data-table-td patent-date-value">Jul 13, 1999</td><td class="patent-data-table-td patent-date-value">Jun 24, 2003</td><td class="patent-data-table-td ">Aventis Research &amp; Technologies Gmbh &amp; Co. Kg</td><td class="patent-data-table-td ">Compound for use as diagnostic tools in mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6589485">US6589485</a></td><td class="patent-data-table-td patent-date-value">Jun 11, 2001</td><td class="patent-data-table-td patent-date-value">Jul 8, 2003</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Solid support for use in the diagnosis of preferential genetic disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6602662">US6602662</a></td><td class="patent-data-table-td patent-date-value">Nov 28, 2000</td><td class="patent-data-table-td patent-date-value">Aug 5, 2003</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Determining whether a target nucleotide is present in a nucleic acid molecule; contacting hybridized primer with deoxyribonucleoside triphosphates, chain terminating nucleotides and a DNA polymerase; diagnosing genetic diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6613509">US6613509</a></td><td class="patent-data-table-td patent-date-value">Mar 22, 1999</td><td class="patent-data-table-td patent-date-value">Sep 2, 2003</td><td class="patent-data-table-td ">Regents Of The University Of California</td><td class="patent-data-table-td ">DNA sequences; polymorphisms; 13C; 15N; 2H; polymerase chain reactions; mass shift analysis; use in sequence validation, comparison and characterization of sequence polymorphisms without the need to sequence the oligonucleotide</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6623928">US6623928</a></td><td class="patent-data-table-td patent-date-value">May 14, 2001</td><td class="patent-data-table-td patent-date-value">Sep 23, 2003</td><td class="patent-data-table-td ">Qiagen Genomics, Inc.</td><td class="patent-data-table-td ">Methods and compositions for determining the sequence of nucleic acid molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6682889">US6682889</a></td><td class="patent-data-table-td patent-date-value">Nov 8, 2000</td><td class="patent-data-table-td patent-date-value">Jan 27, 2004</td><td class="patent-data-table-td ">Becton, Dickinson And Company</td><td class="patent-data-table-td ">Amplification and detection of organisms of the Chlamydiaceae family</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20020045178">US20020045178</a></td><td class="patent-data-table-td patent-date-value">Jun 13, 2001</td><td class="patent-data-table-td patent-date-value">Apr 18, 2002</td><td class="patent-data-table-td ">The Trustees Of Boston University</td><td class="patent-data-table-td ">Use of nucleotide analogs in the analysis of oligonucleotide mixtures and in highly multiplexed nucleic acid sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20020137057">US20020137057</a></td><td class="patent-data-table-td patent-date-value">Jul 27, 2001</td><td class="patent-data-table-td patent-date-value">Sep 26, 2002</td><td class="patent-data-table-td ">Wold Barbara J.</td><td class="patent-data-table-td ">Detecting genes in biological sample; detector libraries</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20020150903">US20020150903</a></td><td class="patent-data-table-td patent-date-value">Jun 11, 2001</td><td class="patent-data-table-td patent-date-value">Oct 17, 2002</td><td class="patent-data-table-td ">Hubert Koster</td><td class="patent-data-table-td ">Diagnostics based on mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20020150927">US20020150927</a></td><td class="patent-data-table-td patent-date-value">Nov 9, 2001</td><td class="patent-data-table-td patent-date-value">Oct 17, 2002</td><td class="patent-data-table-td ">Matray Tracy J.</td><td class="patent-data-table-td ">Methods for detecting a plurality of analytes by mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030017487">US20030017487</a></td><td class="patent-data-table-td patent-date-value">Jun 4, 2002</td><td class="patent-data-table-td patent-date-value">Jan 23, 2003</td><td class="patent-data-table-td ">Pharmacogenetics, Ltd.</td><td class="patent-data-table-td ">Detection of preferential nucleotide sequences in sample; otain sample, incubate with primers, nucleosides and chain terminating compounds, expose to polymerase, detect incorporation of labeled probe</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030039976">US20030039976</a></td><td class="patent-data-table-td patent-date-value">Aug 14, 2001</td><td class="patent-data-table-td patent-date-value">Feb 27, 2003</td><td class="patent-data-table-td ">Haff Lawrence A.</td><td class="patent-data-table-td ">Methods for base counting</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030064483">US20030064483</a></td><td class="patent-data-table-td patent-date-value">Oct 30, 2001</td><td class="patent-data-table-td patent-date-value">Apr 3, 2003</td><td class="patent-data-table-td ">Duke University.</td><td class="patent-data-table-td ">Biosynthesis of nucleotide sequences fragments; obtain nucleic acid template, incubate with primer, enzyme and nucleotide, amplify sequences, recover chain extension product</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030073112">US20030073112</a></td><td class="patent-data-table-td patent-date-value">Jul 10, 2002</td><td class="patent-data-table-td patent-date-value">Apr 17, 2003</td><td class="patent-data-table-td ">Jing Zhang</td><td class="patent-data-table-td ">Amplificaton of nucleotide sequences present in sample; provide sample containing nucleotide sequnces, incubate with primers, amplify sequences, detect amplified product</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030113745">US20030113745</a></td><td class="patent-data-table-td patent-date-value">May 13, 2002</td><td class="patent-data-table-td patent-date-value">Jun 19, 2003</td><td class="patent-data-table-td ">Monforte Joseph A.</td><td class="patent-data-table-td ">Restriction fragment length polymorphism</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1995013396A2?cl=en">WO1995013396A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 11, 1994</td><td class="patent-data-table-td patent-date-value">May 18, 1995</td><td class="patent-data-table-td ">U Gene Research Bv</td><td class="patent-data-table-td ">A method for identifying microorganisms, and aids useful thereof</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Aaserud, D. J. et al., "<a href='http://scholar.google.com/scholar?q="Accurate+Base+Composition+of+Double-Strand+DNA+by+Mass+Spectrometry%2C"'>Accurate Base Composition of Double-Strand DNA by Mass Spectrometry,</a>" Amer. Soc. For Mass Spectrometry, 1996, 1266-1269.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Aaserud, D. J., et al., "<a href='http://scholar.google.com/scholar?q="Accurate+base+composition+of+double+strand+DNA+by+mass+spectrometry%2C"'>Accurate base composition of double strand DNA by mass spectrometry,</a>" J. Am. Soc. Mass Spectrom. (1996) 7(12): 1266-1269.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bahrmand, A. R. et al., "<a href='http://scholar.google.com/scholar?q="Polymerase+chain+reaction+of+bacterial+genomes+with+single+universal+primer%3A+application+to+distinguishing+mycobacteria+species%2C"'>Polymerase chain reaction of bacterial genomes with single universal primer: application to distinguishing mycobacteria species,</a>" Mol. Cell. Probes (1996) 10(2): 117-22.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bahrmand, A. R. et al., "<a href='http://scholar.google.com/scholar?q="Use+of+restriction+enzyme+analysis+of+amplified+DNA+coding+for+the+hsp65+gene+and+polymerase+chain+reaction+with+universal+primer+for+rapid+differentiation+of+mycobacterium+species+in+the+clinical+laboratory%2C"'>Use of restriction enzyme analysis of amplified DNA coding for the hsp65 gene and polymerase chain reaction with universal primer for rapid differentiation of mycobacterium species in the clinical laboratory,</a>" Scand. J. Infect. Diseases (1998) 30(5):477-80.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bastia, T. et al., "<a href='http://scholar.google.com/scholar?q="Organelle+DNA+analysis+of+Solanum+and+Brassica+somatic+hybrids+by+PCR+with+%27universal+primers%27%2C"'>Organelle DNA analysis of Solanum and Brassica somatic hybrids by PCR with 'universal primers',</a>" Theoretical and Applied Genetics (2001) 102(8): 1265-1272.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bowen, J. E. et al., "<a href='http://scholar.google.com/scholar?q="The+native+virulence+plasmid+combination+affects+the+segregational+stability+of+a+theta-replicating+shuttle+vector+in+Bacilhus+anthracis+var.+New+Hampshire%2C"'>The native virulence plasmid combination affects the segregational stability of a theta-replicating shuttle vector in Bacilhus anthracis var. New Hampshire,</a>" J Appl Microbiol. (1999) 87(2): 270-8.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bowen, J. E. et al., "<a href='http://scholar.google.com/scholar?q="The+native+virulence+plasmid+combination+affects+the+segregational+stability+of+a+theta-replicating+shuttle+vector+in+Bacillus+anthracis+var.+New+Hampshire%2C"'>The native virulence plasmid combination affects the segregational stability of a theta-replicating shuttle vector in Bacillus anthracis var. New Hampshire,</a>" J. of Applied Microbiology, 1999, 87, 270-278.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Campbell, W. P. et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+California+serogroup+Bunyaviruses+in+tissue+culture+and+mosquito+pools+by+PCR%2C"'>Detection of California serogroup Bunyaviruses in tissue culture and mosquito pools by PCR,</a>" J. Virol. Methods (1996) 57(2): 175-9.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cespedes, A. et al., "<a href='http://scholar.google.com/scholar?q="Polymerase+chain+reaction+restriction+fragment+length+polymorphism+analysis+of+a+short+fragment+of+the+cytochrome+b+gene+for+identification+of+flatfish+species%2C"'>Polymerase chain reaction restriction fragment length polymorphism analysis of a short fragment of the cytochrome b gene for identification of flatfish species,</a>" J. Food Protection (1998) 61(12): 1684-5.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chen, C. A. et al., "<a href='http://scholar.google.com/scholar?q="Universal+primers+for+amplification+of+mitochondrial+small+subunit+ribosomal+RNA-encoding+gene+in+scleractinian+corals%2C"'>Universal primers for amplification of mitochondrial small subunit ribosomal RNA-encoding gene in scleractinian corals,</a>" Marine Biotech. (2000) 2(2): 146-153.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chen, J. et al., "<a href='http://scholar.google.com/scholar?q="A+universal+PCR+primer+to+detect+members+of+the+Potyviridae+and+its+use+to+examine+the+taxonomic+status+of+several+members+of+the+family%2C"'>A universal PCR primer to detect members of the Potyviridae and its use to examine the taxonomic status of several members of the family,</a>" Arch. Virol. (2001) 146(4): 757-66.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cho, M. et al., "<a href='http://scholar.google.com/scholar?q="Application+of+the+ribonuclease+P+%28RNase+P%29+RNA+gene+sequence+for+phylogenetic+analysis+of+the+gene+Saccharomonospora%2C"'>Application of the ribonuclease P (RNase P) RNA gene sequence for phylogenetic analysis of the gene Saccharomonospora,</a>" Internat. J. of Sys. Bacteriol. (1998) 48: 1223-1230.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Conrads, G. et al., "<a href='http://scholar.google.com/scholar?q="16S-23S+rDNA+internal+transcribed+spacer+sequences+for+analysis+of+the+phylogenetic+relationships+among+species+of+the+genus+Fusobacterium%2C"'>16S-23S rDNA internal transcribed spacer sequences for analysis of the phylogenetic relationships among species of the genus Fusobacterium,</a>" International Journal of Systematic and Evolutionary Microbiology (2002) 52(2): 493-499.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cornel, A. J. et al., "<a href='http://scholar.google.com/scholar?q="Polymerase+chain+reaction+species+diagnostic+assay+for+Anopheles+quadrimaculatus+cryptic+species+%28Diptera%3A+Culicidae%29+based+on+ribosomal+DNA+ITS2+sequences%2C"'>Polymerase chain reaction species diagnostic assay for Anopheles quadrimaculatus cryptic species (Diptera: Culicidae) based on ribosomal DNA ITS2 sequences,</a>" Journal of Medical Entomology (1996) 33(1): 109-16.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Crain, P. F. et al., "<a href='http://scholar.google.com/scholar?q="Applications+of+mass+spectrometry+to+the+characterization+of+oligonucleotides+and+nucleic+acids%2C"'>Applications of mass spectrometry to the characterization of oligonucleotides and nucleic acids,</a>" Curr Opin Biotechnol (1998) 9(1): 25-34.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dasen, G. et al., "<a href='http://scholar.google.com/scholar?q="Classification+and+identification+of+Propionibacteria+based+on+16S+ribosomal+RNA+genes+and+PCR%2C"'>Classification and identification of Propionibacteria based on 16S ribosomal RNA genes and PCR,</a>" Systematic and Applied Microbiology (1998) 21(2): 251-259.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Deforce, D. L. D. et al., "<a href='http://scholar.google.com/scholar?q="Characterization+of+DNA+Oligonucleotides+by+Coupling+of+Capillary+Zone+Electrophoresis+to+Electrospray+Ionization+Q-TOF+Mass+Spectrometry%2C"'>Characterization of DNA Oligonucleotides by Coupling of Capillary Zone Electrophoresis to Electrospray Ionization Q-TOF Mass Spectrometry,</a>" Anal. Chem. (1998) 70(14): 3060-3068.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Deforce, D. L. et al., "<a href='http://scholar.google.com/scholar?q="Analysis+of+oligonucleotides+by+ESI-MS%2C"'>Analysis of oligonucleotides by ESI-MS,</a>" Advances in Chromatography (2000) 40: 539-566.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Demesure, B. et al., "<a href='http://scholar.google.com/scholar?q="A+set+of+universal+primers+for+amplification+of+polymorphic+non-coding+regions+of+mitochondrial+and+chloroplast+DNA+in+plants%2C"'>A set of universal primers for amplification of polymorphic non-coding regions of mitochondrial and chloroplast DNA in plants,</a>" Mol. Ecology (1995) 4(1): 129-31.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dinauer, D. M. et al., "<a href='http://scholar.google.com/scholar?q="Sequence-based+typing+of+HLA+class+II+DQB1%2C"'>Sequence-based typing of HLA class II DQB1,</a>" Tissue Antigens (2000) 55(4): 364-368.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dubernet, S. et al., "<a href='http://scholar.google.com/scholar?q="A+PCR-based+method+for+identification+of+Lactobacilli+at+the+genus+level%2C"'>A PCR-based method for identification of Lactobacilli at the genus level,</a>" FEMS Microbiology Letters (2002) 214(2): 271-275.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Figueiredo, L. T. M. et al., "<a href='http://scholar.google.com/scholar?q="Identification+of+Brazilian+flaviviruses+by+a+simplified+reverse+transcription-polymerase+chain+reaction+method+using+Flavivirus+universal+primers%2C"'>Identification of Brazilian flaviviruses by a simplified reverse transcription-polymerase chain reaction method using Flavivirus universal primers,</a>" American Journal of Tropical Medicine and Hygiene (1998) 59(3): 357-362.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Flora, J. et al., "<a href='http://scholar.google.com/scholar?q="Dual-micro-ESI+source+for+precise+mass+determination+on+a+quadrupole+time-of-flight+mass+spectrometer+for+genomic+and+proteomic+applications%2C"'>Dual-micro-ESI source for precise mass determination on a quadrupole time-of-flight mass spectrometer for genomic and proteomic applications,</a>" Anal. Bioanal. Chem. (2002) 373(7): 538-46.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fox, A. et al., "<a href='http://scholar.google.com/scholar?q="Identification+and+detection+of+bacteria%3A+electrospray+MS-MS+versus+derivatization%2FGC-MS%2C"'>Identification and detection of bacteria: electrospray MS-MS versus derivatization/GC-MS,</a>" Proceedings of the ERDEC Scienctific Conference on Chemical and Biological Defense Research (1996) Aberdeen Proving Ground, Md., Nov. 15-18, 1994: p. 39-44.</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fox, A., "<a href='http://scholar.google.com/scholar?q="Report+of+the+%22Bioterrorism+Workshop.%22+Duke+University+Thomas+Center+on+Apr.+2-4%2C+2002%2C+organized+by+US+Army+Research+Office%2C"'>Report of the "Bioterrorism Workshop." Duke University Thomas Center on Apr. 2-4, 2002, organized by US Army Research Office,</a>" J. Microbiol. Methods (2002) 51(3): 247-54.</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fox, K. F. et al., "<a href='http://scholar.google.com/scholar?q="Identification+of+Brucella+by+Ribosomal-spacer-region+PCR+and+differentiation+of%2C+Brucella+canis+from+other+Brucella+spp.+pathogenic+for+humans+by+carbohydrate+profiles%2C"'>Identification of Brucella by Ribosomal-spacer-region PCR and differentiation of, Brucella canis from other Brucella spp. pathogenic for humans by carbohydrate profiles,</a>" J. Clin. Microbiol. (1998) 36(11): 3217-3222.</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Griffey, R. H. et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+base+pair+mismatches+in+duplex+DNA+and+RNA+oligonucleotides+using+electrospray+mass+spectrometry%2C"'>Detection of base pair mismatches in duplex DNA and RNA oligonucleotides using electrospray mass spectrometry,</a>" Proceeding of SPIE-The International Society for Optical Engineering (1997) 2985(Ultrasensitive Biochemical Diagnostics II): 82-86.</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Griffin, T. J. et al., "<a href='http://scholar.google.com/scholar?q="Direct+genetic+analysis+by+matrix-assisted+laser+desorption%2Fionization+mass+spectrometry%2C"'>Direct genetic analysis by matrix-assisted laser desorption/ionization mass spectrometry,</a>" Proc. Natl. Acad. Sci. USA (1999) 96(11): 6301-6306.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hannis, J. C, et al, "<a href='http://scholar.google.com/scholar?q="Genotyping+short+tandem+repeats+using+flow+injection+and+electrospray+ionization+Fourier+transform+ion+cyclotron+resonance+mass+spectrometry%2C"'>Genotyping short tandem repeats using flow injection and electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,</a>" Rapid Communications in Mass Spectrometry (2001) 15(5): 348-350.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hannis, J. C. et al., "<a href='http://scholar.google.com/scholar?q="Accurate+characterization+of+the+tyrosine+hydroxylase+forensic+allele+9.3+through+development+of+electrospray+ionization+Fourier+transform+ion+cyclotron+resonance+mass+spectrometry%2C"'>Accurate characterization of the tyrosine hydroxylase forensic allele 9.3 through development of electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,</a>" Rapid Communications in Mass Spectrometry (1999) 13(10): 954-62.</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hannis, J. C. et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+double-stranded+PCR+amplicons+at+the+attomole+level+electrosprayed+from+low+nanomolar+solutions+using+FT-ICR+mass+spectrometry%2C"'>Detection of double-stranded PCR amplicons at the attomole level electrosprayed from low nanomolar solutions using FT-ICR mass spectrometry,</a>" Fresenius Journal of Analytical Chemistry (2001) 369(3-4): 246-51.</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hannis, J. C. et al., "<a href='http://scholar.google.com/scholar?q="Genotyping+complex+short+tandem+repeats+using+electrospray+ionization+Fourier+transform+ion+cyclotron+resonance+multistage+mass+spectrometry%2C"'>Genotyping complex short tandem repeats using electrospray ionization Fourier transform ion cyclotron resonance multistage mass spectrometry,</a>" Proceedings of SPIE-The International Society for Optical Engineering (2000) 3926: 36-47.</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hayashi, H. et al., "<a href='http://scholar.google.com/scholar?q="Phylogenetic+analysis+of+the+human+gut+microbiota+using+I+6S+rDNA+clone+libraries+and+strictly+anaerobic+culture-based+methods%2C"'>Phylogenetic analysis of the human gut microbiota using I 6S rDNA clone libraries and strictly anaerobic culture-based methods,</a>" Microbiol. Immunol. (2002) 46(8): 535-48.</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Henchal, E. A. et al., "<a href='http://scholar.google.com/scholar?q="Sensitivity+and+specificity+of+a+universal+primer+set+for+the+rapid+diagnosis+of+dengue+virus+infections+by+polymerase+chain+reaction+and+nucleic+acid+hybridization%2C"'>Sensitivity and specificity of a universal primer set for the rapid diagnosis of dengue virus infections by polymerase chain reaction and nucleic acid hybridization,</a>" American Journal of Tropical Medicine and Hygiene (1991) 45(4): 418-28.</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Herrmann et al., J. Clin Microb., 34(8):1897-1902 (1996).</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Herrmann, B. et al., "<a href='http://scholar.google.com/scholar?q="Differentiation+of+Chlamydia+spp.+by+Sequence+Determination+and+Restriction+Endonuclease+Cleavage+of+RNase+P+RNA+Genes%2C"'>Differentiation of Chlamydia spp. by Sequence Determination and Restriction Endonuclease Cleavage of RNase P RNA Genes,</a>" J. Clin. Microbiol. (1996) 34(8): 1897-1902.</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Higgins, G. S. et al., "<a href='http://scholar.google.com/scholar?q="Competitive+oligonucleotide+single-base+extension+combined+with+mass+spectrometric+detection+for+mutation+screening%2C"'>Competitive oligonucleotide single-base extension combined with mass spectrometric detection for mutation screening,</a>" BioTechniques (1997) 23(4): 710-714.</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hoffmann, E. et al., "<a href='http://scholar.google.com/scholar?q="Universal+primer+set+for+the+full-length+amplification+of+all+influenza+A+viruses%2C"'>Universal primer set for the full-length amplification of all influenza A viruses,</a>" Archives of Virology (2001) 146(12): 2275-2289.</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Honda, K. et al., "<a href='http://scholar.google.com/scholar?q="Universal+method+of+hypersensitive+nested+PCR+toward+forensic+DNA+typing%2C"'>Universal method of hypersensitive nested PCR toward forensic DNA typing,</a>" International Congress Series (1998) 7: 28-30.</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hurst, G. B. et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+Bacterial+DNA+Polymerase+Chain+Reaction+Products+by+Matrix-assisted+Laser+Desorption%2FIonization+Mass+Spectrometry%2C"'>Detection of Bacterial DNA Polymerase Chain Reaction Products by Matrix-assisted Laser Desorption/Ionization Mass Spectrometry,</a>" Rapid Communications in Mass Spectrometry, 1996, 10, 377-382.</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hurst, G. B. et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+Bacterial+DNA+polymerase+Chain+Reation+Products+by+Matrix-assisted+Laser+Desorption%2FIonization+Mass+Spectrometry%2C"'>Detection of Bacterial DNA polymerase Chain Reation Products by Matrix-assisted Laser Desorption/Ionization Mass Spectrometry,</a>" Rapid Commun. Mass Spectrom. (1996) 10: 377-382.</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hurst, G. B. et al., "<a href='http://scholar.google.com/scholar?q="MALDI-TOF+analysis+of+polymerase+chain+reaction+products+from+methanotrophic+bacteria%2C"'>MALDI-TOF analysis of polymerase chain reaction products from methanotrophic bacteria,</a>" Anal. Chem, (1998) 70(13): 2693-2698.</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Isola, N. R. et al., "<a href='http://scholar.google.com/scholar?q="MALDI-TOF+mass+spectrometric+method+for+detection+of+hybridized+DNA+oligomers%2C"'>MALDI-TOF mass spectrometric method for detection of hybridized DNA oligomers,</a>" Analytical Chemistry (2001) 73(9): 2126-2131.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jankowski, K. et al., "<a href='http://scholar.google.com/scholar?q="Mass+spectrometry+of+DNA%2C+Part+2%2C+Quantitative+estimation+of+base+composition%2C"'>Mass spectrometry of DNA, Part 2, Quantitative estimation of base composition,</a>" European Journal of Mass Spectrometry in Biochemistry (1980) 1(1): 45-52.</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kageyama, A. et al., "<a href='http://scholar.google.com/scholar?q="Rapid+detection+of+human+fecal+Eubacterium+species+and+related+genera+by+nested+PCR+method%2C"'>Rapid detection of human fecal Eubacterium species and related genera by nested PCR method,</a>" Microbiology and Immunology (2001) 45(4): 315-318.</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Krahmer, M. T. et al., "<a href='http://scholar.google.com/scholar?q="Electrospray+quadrupole+mass+spectrometry+analysis+of+model+oligonucleotides+and+polymerase+chain+reaction+products%3A+determination+of+base+substitutions%2C+nucleotide+additions%2Fdeletions+and+chemical+modifications%2C"'>Electrospray quadrupole mass spectrometry analysis of model oligonucleotides and polymerase chain reaction products: determination of base substitutions, nucleotide additions/deletions and chemical modifications,</a>" Anal. Chem. (1999) 71(14): 2893-900.</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Krahmer, M. T. et al., "<a href='http://scholar.google.com/scholar?q="MS+for+identification+of+single+nucleotide+polymorphisms+and+MS%2FMS+for+discrimination+of+isomeric+PCR+products%2C"'>MS for identification of single nucleotide polymorphisms and MS/MS for discrimination of isomeric PCR products,</a>" Anal. Chem. (2000) 72(17): 4033-4040.</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lacroix, J.-M. et al., "<a href='http://scholar.google.com/scholar?q="PCR-based+technique+for+the+detection+of+bacteria+in+semen+and+urine%2C"'>PCR-based technique for the detection of bacteria in semen and urine,</a>" J. Microbiol. Methods (1996) 26: 61-71.</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebedev et al (Genetic Analysis: Biomolecular Engineering (1996) 13:15-21).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Leif et al., Eur. J. Biochem., 230:538-48 (1995).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Leif, H. et al., "<a href='http://scholar.google.com/scholar?q="Isolation+and+characterization+of+the+proton-translocating+NADH%3A+ubiquinone+oxidoreductase+from+Escherichia+coli%2C"'>Isolation and characterization of the proton-translocating NADH: ubiquinone oxidoreductase from Escherichia coli,</a>" Eur. J. Biochem. (1995) 230(2): 538-548.</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Li, J. et al., "<a href='http://scholar.google.com/scholar?q="Single+nucleotide+polymorphism+determination+using+primer+extension+and+time-of-flight+mass+spectrometry%2C"'>Single nucleotide polymorphism determination using primer extension and time-of-flight mass spectrometry,</a>" Electrophoresis (1999) 20(6): 1258-1265.</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Little, D. P. et al., "<a href='http://scholar.google.com/scholar?q="Rapid+Sequencing+of+Oligonucleotides+by+High-Resolution+Mass+Spectrometry%2C"'>Rapid Sequencing of Oligonucleotides by High-Resolution Mass Spectrometry,</a>" J. Am. Chem. Soc. (1994) 116(11): 4893-4897.</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Liu et al., Gene, 172:105-109 (1996).</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Liu, C. et al., "<a href='http://scholar.google.com/scholar?q="Improving+the+microdialysis+procedure+for+electrospray+ionization+mass+spectrometry+of+biological+samples%2C"'>Improving the microdialysis procedure for electrospray ionization mass spectrometry of biological samples,</a>" Journal of Mass Spectrometry (1997) 32(4): 425-431.</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Liu, Y. et al., "<a href='http://scholar.google.com/scholar?q="An+unusual+gene+arrangement+for+the+putative+chromosome+replication+origin+and+circadian+expression+dnaN+in+Synechococcus+sp.+strain+PCC+7942%2C"'>An unusual gene arrangement for the putative chromosome replication origin and circadian expression dnaN in Synechococcus sp. strain PCC 7942,</a>" Gene (1996) 172(1): 105-109.</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Loakes, D. et al., "<a href='http://scholar.google.com/scholar?q="Nitroindoles+as+Universal+Bases%2C"'>Nitroindoles as Universal Bases,</a>" Nucleosides &amp; Nucleotides, 1995, 14(3-5), 1001-1003.</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Loakes, D. et al., "<a href='http://scholar.google.com/scholar?q="Nitroindoles+as+Universal+Bases%2C"'>Nitroindoles as Universal Bases,</a>" Nucleosides Nucleotides (1995) 14:1001-1003.</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Love et al., Gene, 166:179-180 (1995).</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Love, B. C. et al., "<a href='http://scholar.google.com/scholar?q="Cloning+and+sequence+of+the+groESL+heat-shock+operon+of+Pasteurella+multocida%2C"'>Cloning and sequence of the groESL heat-shock operon of Pasteurella multocida,</a>" Gene (1995) 166(1): 179-180.</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Maiwald, M. et al., "<a href='http://scholar.google.com/scholar?q="Characterization+of+contaminating+DNA+in+Taq+polymerase+which+occurs+during+amplification+with+a+primer+set+for+Legionella+5S+ribosomal+RNA%2C"'>Characterization of contaminating DNA in Taq polymerase which occurs during amplification with a primer set for Legionella 5S ribosomal RNA,</a>" Mol. Cell. Probes (1994) 8(1): 11-14.</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mangrum, J. et al., "<a href='http://scholar.google.com/scholar?q="Solution+composition+and+thermal+denaturation+for+the+production+of+single-stranded+PCR+amplicons%3A+piperidine-induced+destabilization+of+the+DNA+duplex%3F"'>Solution composition and thermal denaturation for the production of single-stranded PCR amplicons: piperidine-induced destabilization of the DNA duplex?</a>" J. Am. Soc. Mass Spectrom, (2002) 13(3): 232-40.</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Martemyanov et al., Prot. Expr. and Purif., 18:257-61 (2000).</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Martemyanov, K. A. et al., "<a href='http://scholar.google.com/scholar?q="Extremely+Thermostable+Elongation+Factor+G+from+Aquifex+aeolicus%3A+Cloning%2C+Expression%2C+Purification%2C+and+Characterization+in+a+Heterologous+Translation+System%2C"'>Extremely Thermostable Elongation Factor G from Aquifex aeolicus: Cloning, Expression, Purification, and Characterization in a Heterologous Translation System,</a>" Protein Expr. Purif. (2000) 18(3): 257-261.</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McCabe, K. M. et al., "<a href='http://scholar.google.com/scholar?q="Bacterial+Species+Identification+after+DNA+Amplification+with+a+Universal+Primer+Pair%2C"'>Bacterial Species Identification after DNA Amplification with a Universal Primer Pair,</a>" Molecular Genetics and Metabolism (1999) 66(3): 205-211.</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Meiyu, F. et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+flaviviruses+by+reverse+transcriptase-polymerase+chain+reaction+with+the+universal+primer+set%2C"'>Detection of flaviviruses by reverse transcriptase-polymerase chain reaction with the universal primer set,</a>" Microbiology and Immunology (1997) 41(3): 209-13.</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Messmer, T. O. et al., "<a href='http://scholar.google.com/scholar?q="Discrimination+of+Streptococcus+pneumoniae+from+other+upper+respiratory+tract+streptococci+by+arbitrarily+primed+PCR%2C"'>Discrimination of Streptococcus pneumoniae from other upper respiratory tract streptococci by arbitrarily primed PCR,</a>" Clin. Biochem. (1995) 28(6):567-72.</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Moricca, S. et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+Fusarium+oxysporum+f.sp.+vasinfectum+in+cotton+tissue+by+polymerase+chain+reaction%2C"'>Detection of Fusarium oxysporum f.sp. vasinfectum in cotton tissue by polymerase chain reaction,</a>" Plant Pathology (1998) 47(4):486-494.</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Morse et al., System. Appl. Microbiol., 19:150-157 (1996).</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Morse, R. et al., "<a href='http://scholar.google.com/scholar?q="Nucleotide+Sequence+of+Part+of+the+ropC+Gene+Encoding+the+alpha%27+Subunit+of+DNA-Dependent+RNA+Polymerase+from+some+Gram-Positive+Bacteria+and+Comparative+Amino+Acid+Sequence+Analysis%2C"'>Nucleotide Sequence of Part of the ropC Gene Encoding the alpha' Subunit of DNA-Dependent RNA Polymerase from some Gram-Positive Bacteria and Comparative Amino Acid Sequence Analysis,</a>" System Appl. Microbiol. (1996) 19: 150-157.</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Muddiman et al (Anal. Chem. (1996) 66:3705-3712).</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Muddiman et al., Analytical Chemistry, vol. 68, pp. 3705-3712 (1996).</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, D. C. et al., "<a href='http://scholar.google.com/scholar?q="Application+of+secondary+ion+and+matrix-assisted+laser+desorption-ionization+time-of-flight+mass+spectrometry+for+the+quantitative+analysis+of+biological+molecules%2C"'>Application of secondary ion and matrix-assisted laser desorption-ionization time-of-flight mass spectrometry for the quantitative analysis of biological molecules,</a>" Mass Spectrometry Reviews (1996) 14(6): 383-429.</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, D. C. et al., "<a href='http://scholar.google.com/scholar?q="Characterization+of+PCR+products+from+bacilli+using+electrospray+ioniation+FTICR+mass+spectrometry%2C"'>Characterization of PCR products from bacilli using electrospray ioniation FTICR mass spectrometry,</a>" Analytical Chemistry (1996) 68(21); 3705-12.</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, D. C. et al., "<a href='http://scholar.google.com/scholar?q="Important+aspects+concerning+the+quantification+of+biomolecules+by+time-of-flight+secondary-ion+mass+spectrometry%2C"'>Important aspects concerning the quantification of biomolecules by time-of-flight secondary-ion mass spectrometry,</a>" Applied Spectroscopy (1996) 50(2); 161-166.</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, D. C. et al., "<a href='http://scholar.google.com/scholar?q="Length+and+base+composition+of+PCR-amplified+nucleic+acids+using+mass+measurements+from+electrospray+ionization+mass+spectrometry%2C"'>Length and base composition of PCR-amplified nucleic acids using mass measurements from electrospray ionization mass spectrometry,</a>" Anal. Chem. (1997) 69(8): 1543-1549.</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, D. C. et al., "<a href='http://scholar.google.com/scholar?q="Length+and+Base+Composition+of+PCR-Amplified+Nucleic+Acids+Using+Mass+Measurements+from+Electrospray+Ionization+Mass+Spectrometry%2C"'>Length and Base Composition of PCR-Amplified Nucleic Acids Using Mass Measurements from Electrospray Ionization Mass Spectrometry,</a>" Anal. Chem., 1997, 1543-1549.</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, D. C. et al., "<a href='http://scholar.google.com/scholar?q="Precise+mass+measurement+of+a+double-stranded+500+base-pair+%28309+kDa%29+polymerase+chain+reaction+product+by+negative+ion+electrospray+ionization+Fourier+transform+ion+cyclotron+resonance+mass+spectrometry%2C"'>Precise mass measurement of a double-stranded 500 base-pair (309 kDa) polymerase chain reaction product by negative ion electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,</a>" Rapid Commun. Mass Spectrom. (1999) 13(12): 1201-1204.</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, D. C. et al., "<a href='http://scholar.google.com/scholar?q="Precise+Mass+Measurement+of+a+Double-stranded+500+Base-pair+%28309+kDa%29+Polymerase+Chain+Reaction+Product+by+Negative+Ion+Electrospray+Ionization+Fourier+Transform+Ion+Cyclotron+Resonance+Mass+Spectrometry%2C"'>Precise Mass Measurement of a Double-stranded 500 Base-pair (309 kDa) Polymerase Chain Reaction Product by Negative Ion Electrospray Ionization Fourier Transform Ion Cyclotron Resonance Mass Spectrometry,</a>" Rapid Commun. Mass Spectrom., 1999, 13, 1201-1204.</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, D. C. et al., "<a href='http://scholar.google.com/scholar?q="Sequencing+and+characterization+of+larger+oligonucleotides+by+electrospray+ionization+Fourier+transform+ion+cyclotron+resonance+mass+spectrometry%2C"'>Sequencing and characterization of larger oligonucleotides by electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,</a>" Rev. Anal. Chem. (1998) 17(1): p. 1-68.</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, D.C., et al., "<a href='http://scholar.google.com/scholar?q="Sequencing+and+characterization+of+larger+oligonucleotides+by+electrospray+ionization+fourier+transform+ion+cyclotron+resonance+mass+spectrometry%2C"'>Sequencing and characterization of larger oligonucleotides by electrospray ionization fourier transform ion cyclotron resonance mass spectrometry,</a>" Rev. Anal. Chem., 1998, 17(1), 1-68.</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muhammad, W. T. et al., "<a href='http://scholar.google.com/scholar?q="Electrosrpay+ionization+quadrupole+time-of-flight+mass+spectrometry+and+quadrupole+mass+spectrometry+for+genotyping+single+nucleotide+substitutions+in+intact+polymerase+chain+reaction+products+in+K-ras+and+p53%2C"'>Electrosrpay ionization quadrupole time-of-flight mass spectrometry and quadrupole mass spectrometry for genotyping single nucleotide substitutions in intact polymerase chain reaction products in K-ras and p53,</a>" Rapid Commun. Mass Spectrum. (2002) 16(24): 2278-85.</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mushegian, A. R. et al., "<a href='http://scholar.google.com/scholar?q="A+minimal+gene+set+for+cellular+life+derived+by+comparison+of+complete+bacterial+genomes%2C"'>A minimal gene set for cellular life derived by comparison of complete bacterial genomes,</a>" Proc. Natl. Acad. Sci. USA (1996) 93(19): 10268-10273.</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nagpal, M. L. et al., "<a href='http://scholar.google.com/scholar?q="Utility+of+16S-23S+rNA+spacer+region+methodology%3A+how+similar+are+interspace+regions+within+a+genome+and+between+strains+for+closely+related+organisms%3F"'>Utility of 16S-23S rNA spacer region methodology: how similar are interspace regions within a genome and between strains for closely related organisms?</a>" J. Microbiol. Methods (1998) 33(s): 211-219.</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Naumov, G. I. et al., "<a href='http://scholar.google.com/scholar?q="Discrimination+of+the+Soil+Yeast+Species+Williopsis+salurnus+and+Williopsis+suaveolens+by+the+Polymerase+Chain+Reaction+with+the+Universal+Primer+N21%2C"'>Discrimination of the Soil Yeast Species Williopsis salurnus and Williopsis suaveolens by the Polymerase Chain Reaction with the Universal Primer N21,</a>" Micribiology (Moscow)(Translation of Mikrobiologiya) (2000) 69(2): 280-285.</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">NCBI Blast results (Mar. 2, 2006).</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Null, A. P. et al., "<a href='http://scholar.google.com/scholar?q="Evaluation+of+sample+preparation+techniques+for+mass+measurements+of+PCR+products+using+ESI-FT-ICR+mass+spectrometry%2C"'>Evaluation of sample preparation techniques for mass measurements of PCR products using ESI-FT-ICR mass spectrometry,</a>" J. Am. Soc. Mass Spectrom. (2002) 13(4): 338-344.</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Null, A. P. et al., "<a href='http://scholar.google.com/scholar?q="Genotyping+of+Simple+and+Compound+Short+Tandem+Repeat+Loci+Using+Electrospray+Ionization+Fourier+Transform+Ion+Cyclotron+Resonance+Mass+Spectrometry%2C"'>Genotyping of Simple and Compound Short Tandem Repeat Loci Using Electrospray Ionization Fourier Transform Ion Cyclotron Resonance Mass Spectrometry,</a>" Anal. Chem. (2001) 73(18): 4514-4521.</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Null, A. P. et al., "<a href='http://scholar.google.com/scholar?q="Perspectives+on+the+use+of+electrospray+ionization+Fourier+transform+ion+cyclotron+resonance+mass+spectrometry+for+short+tandem+repeat+genotyping+in+the+post-genome+era%2C"'>Perspectives on the use of electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry for short tandem repeat genotyping in the post-genome era,</a>" Journal of Mass Spectrometry (2001) 36(6): 589-606.</td></tr><tr><td class="patent-data-table-td ">90</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Null, A. P. et al., "<a href='http://scholar.google.com/scholar?q="Preparation+of+single-stranded+PCR+products+for+electrospray+ionization+mass+spectrometry+using+the+DNA+repair+enzyme+lambda+exonuclease%2C"'>Preparation of single-stranded PCR products for electrospray ionization mass spectrometry using the DNA repair enzyme lambda exonuclease,</a>" Analyst (2000) 125(4): 619-626.</td></tr><tr><td class="patent-data-table-td ">91</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Peng, X. et al., "<a href='http://scholar.google.com/scholar?q="Rapid+detection+of+Shigella+species+in+enviromental+sewage+by+an+immunocapture+PCR+with+universal+primers%2C"'>Rapid detection of Shigella species in enviromental sewage by an immunocapture PCR with universal primers,</a>" Appl. Environ. Microbiol. (2002) 68(5): 2580-3.</td></tr><tr><td class="patent-data-table-td ">92</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pomerantz, S. C. et al., "<a href='http://scholar.google.com/scholar?q="Determination+of+oligonucleotide+composition+from+mass+spectrometrically+measured+molecular+weight%2C"'>Determination of oligonucleotide composition from mass spectrometrically measured molecular weight,</a>" J. Am. Soc. Mass Spectrom. (1993) 4(3): 204-9.</td></tr><tr><td class="patent-data-table-td ">93</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Reid, S. M. et al., "<a href='http://scholar.google.com/scholar?q="Primary+diagnosis+of+foot-and-mouth+disease+by+reverse+transcription+polymerase+chain+reaction%2C"'>Primary diagnosis of foot-and-mouth disease by reverse transcription polymerase chain reaction,</a>" Journal of Virological Methods (2000) 89(1-2): 167-76.</td></tr><tr><td class="patent-data-table-td ">94</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Reilly, K. et al., "<a href='http://scholar.google.com/scholar?q="Design+and+use+of+16S+ribosomal+DNA-directed+primers+in+competitive+PCRs+to+enumerate+proteolytic+bacteria+in+the+rumen%2C"'>Design and use of 16S ribosomal DNA-directed primers in competitive PCRs to enumerate proteolytic bacteria in the rumen,</a>" Microbiol. Ecol. (2002) 43(2): 259-70.</td></tr><tr><td class="patent-data-table-td ">95</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sala, M. et al., "<a href='http://scholar.google.com/scholar?q="Ambiguous+base+pairing+of+the+purine+analogue+1-%282-deoxy-alpha-D-ribofuranosyl%29-imidazole-4-carboxamide+during+PCR%2C"'>Ambiguous base pairing of the purine analogue 1-(2-deoxy-alpha-D-ribofuranosyl)-imidazole-4-carboxamide during PCR,</a>" Nucleic Acids Research, 1996, 24(17), 3302-3306.</td></tr><tr><td class="patent-data-table-td ">96</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Seshadri et al., Infection and Immunity, 67(11):6026-6033 (1999).</td></tr><tr><td class="patent-data-table-td ">97</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Steffens, D. L. et al., "<a href='http://scholar.google.com/scholar?q="Sequence+analysis+of+mitochondrial+DNA+hypervariable+regions+using+infrared+fluorescence+detection%2C"'>Sequence analysis of mitochondrial DNA hypervariable regions using infrared fluorescence detection,</a>" BioTechniques (1998) 24(6): 1044-1046.</td></tr><tr><td class="patent-data-table-td ">98</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Takahashi et al., J. Antmicrob. Chemotherapy, 41:49-57 (1998).</td></tr><tr><td class="patent-data-table-td ">99</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Takahashi, H. et al., "<a href='http://scholar.google.com/scholar?q="Characterization+of+gyrA%2C+gyrB%2C+grlA+and+grlB+mutations+in+fluoroquinolone-resistant+clinical+isolates+of+Staphylococcus+aureus%2C"'>Characterization of gyrA, gyrB, grlA and grlB mutations in fluoroquinolone-resistant clinical isolates of Staphylococcus aureus,</a>" J. Antimicrob. Chemother, (1998) 41(1): 49-57.</td></tr><tr><td class="patent-data-table-td ">100</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Tong et al., NAR, 28(6):1447-1454 (2000).</td></tr><tr><td class="patent-data-table-td ">101</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tong, J. et al., "<a href='http://scholar.google.com/scholar?q="Ligation+reaction+specificities+of+an+NAD%26lt%3BSUP%26gt%3B%2B%26lt%3B%2FSUP%26gt%3B-dependent+DNA+ligase+from+the+hyperthermophile+Aquifex+aeolicus%2C"'>Ligation reaction specificities of an NAD&lt;SUP&gt;+&lt;/SUP&gt;-dependent DNA ligase from the hyperthermophile Aquifex aeolicus,</a>" Nucleic Acids Res. (2000) 28(6): 1447-1454.</td></tr><tr><td class="patent-data-table-td ">102</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Van Aerschot, A. et al., "<a href='http://scholar.google.com/scholar?q="In+search+of+acyclic+analogues+as+universal+nucleosides+in+degenerate+probes%2C"'>In search of acyclic analogues as universal nucleosides in degenerate probes,</a>" Nucleosides &amp; Nucleotides (1995) 14(3-5): 1053-1056.</td></tr><tr><td class="patent-data-table-td ">103</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Van Aerschot, A. et al., "<a href='http://scholar.google.com/scholar?q="In+Search+of+Acyclic+Analogues+as+Universal+Nucleosides+in+Degenerate+Probes%2C"'>In Search of Acyclic Analogues as Universal Nucleosides in Degenerate Probes,</a>" Nucleosides &amp; Nucleotides, 1995, 14(3-5), 1053-1056.</td></tr><tr><td class="patent-data-table-td ">104</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Van Baar, B. L., "<a href='http://scholar.google.com/scholar?q="Characterisation+of+bacteria+by+matrix-assisted+laser+desorption%2Fionisation+and+electrospray+mass+spectrometry%2C"'>Characterisation of bacteria by matrix-assisted laser desorption/ionisation and electrospray mass spectrometry,</a>" FEMS Microbiol. Rev. (2000).24(2): 193-219.</td></tr><tr><td class="patent-data-table-td ">105</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Van Camp, G. et al., "<a href='http://scholar.google.com/scholar?q="Amplification+and+sequencing+of+variable+regions+in+bacterial+23S+ribosomal+RNA+genes+with+conserved+primer+sequences%2C"'>Amplification and sequencing of variable regions in bacterial 23S ribosomal RNA genes with conserved primer sequences,</a>" Curr. Microbiol. (1993) 27(3): 147-51.</td></tr><tr><td class="patent-data-table-td ">106</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Walters, J. J. et al., "<a href='http://scholar.google.com/scholar?q="Genotyping+single+nucleotide+polymorphisms+using+intact+polymerase+chain+reaction+products+by+electrospray+quadrupole+mass+spectrometry%2C"'>Genotyping single nucleotide polymorphisms using intact polymerase chain reaction products by electrospray quadrupole mass spectrometry,</a>" Rapid Commun. Mass Spectrom. (2001) 15(18): 1752-1759.</td></tr><tr><td class="patent-data-table-td ">107</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Widjojoatmodjo et al., Journal of Clinical Microbiology, vol. 32, No. 12, pp. 3002-3007 (1994).</td></tr><tr><td class="patent-data-table-td ">108</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Widjojoatmodjo, M. N. et al., "<a href='http://scholar.google.com/scholar?q="Rapid+identification+of+bacteria+by+PCR-single-strand+conformation+polymorphism%2C"'>Rapid identification of bacteria by PCR-single-strand conformation polymorphism,</a>" J. Clin. Microbiol. (1994) 32(12): 3002-3007.</td></tr><tr><td class="patent-data-table-td ">109</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wolter, A. et al., "<a href='http://scholar.google.com/scholar?q="Negative+Ion+FAB+Mass+Spectrometric+Analysis+of+Non-Charged+Key+Intermediates+in+Oligonucleotide+Synthesis%3A+Rapid+Identification+of+Partially+Protected+Dinucleoside+Monophosphates%2C"'>Negative Ion FAB Mass Spectrometric Analysis of Non-Charged Key Intermediates in Oligonucleotide Synthesis: Rapid Identification of Partially Protected Dinucleoside Monophosphates,</a>" Biomed. Environ. Mass Spectrom. (1987) 14: 111-116.</td></tr><tr><td class="patent-data-table-td ">110</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Woo, T. H. S. et al., "<a href='http://scholar.google.com/scholar?q="Identification+of+Leptospira+inadai+by+continuous+monitoring+of+fluorescence+during+rapid+cycle+PCR%2C"'>Identification of Leptospira inadai by continuous monitoring of fluorescence during rapid cycle PCR,</a>" Systematic and Applied Microbiology (1998) 21(1): 89-96.</td></tr><tr><td class="patent-data-table-td ">111</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wunschel, D. et al., "<a href='http://scholar.google.com/scholar?q="Discrimination+among+the+B.+cereus+group%2C+in+comparison+to+B.+subtilis%2C+by+structural+carbohydrate+profiles+and+ribosomal+RNA+spacer+region+PCR%2C"'>Discrimination among the B. cereus group, in comparison to B. subtilis, by structural carbohydrate profiles and ribosomal RNA spacer region PCR,</a>" Systematic and Applied Microbiology (1995) 17(4): 625-35.</td></tr><tr><td class="patent-data-table-td ">112</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wunschel, D. S. et al., "<a href='http://scholar.google.com/scholar?q="Analysis+of+double-stranded+polymerase+chain+reaction+products+from+the+Bacilus+gereus+group+by+electrospray+ionization+Fourier+transform+ion+cyclotron+resonance+mass+spectrometry%2C"'>Analysis of double-stranded polymerase chain reaction products from the Bacilus gereus group by electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,</a>" Rapid Communications in Mass Spectromery (1996) 10(1): 29-35.</td></tr><tr><td class="patent-data-table-td ">113</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wunschel, D. S. et al., "<a href='http://scholar.google.com/scholar?q="Heterogeneity+in+Bacillus+cereus+PCR+Products+Detected+by+ESI-FTICR+Mass+Spectrometry%2C"'>Heterogeneity in Bacillus cereus PCR Products Detected by ESI-FTICR Mass Spectrometry,</a>" Anal Chem., 1988, 70, 1203-1207.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7781162">US7781162</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 11, 2003</td><td class="patent-data-table-td patent-date-value">Aug 24, 2010</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Using replication of intelligent primers (ribosomal RNA or house keeping oligonucleotides) to identify, classify and diagnose infection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7811766">US7811766</a></td><td class="patent-data-table-td patent-date-value">Mar 28, 2008</td><td class="patent-data-table-td patent-date-value">Oct 12, 2010</td><td class="patent-data-table-td ">Thinkvillage, Llc</td><td class="patent-data-table-td ">A pair of oligonucleotide molecules for amplification of ribosomal DNA from a plant material; DNA polymerase chain reaction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8076104">US8076104</a></td><td class="patent-data-table-td patent-date-value">Jan 25, 2008</td><td class="patent-data-table-td patent-date-value">Dec 13, 2011</td><td class="patent-data-table-td ">Rogan Peter K</td><td class="patent-data-table-td ">Rapid and comprehensive identification of prokaryotic organisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8088582">US8088582</a></td><td class="patent-data-table-td patent-date-value">Apr 6, 2007</td><td class="patent-data-table-td patent-date-value">Jan 3, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for the use in identification of fungi</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8242254">US8242254</a></td><td class="patent-data-table-td patent-date-value">Mar 7, 2007</td><td class="patent-data-table-td patent-date-value">Aug 14, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for use in identification of bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8288523">US8288523</a></td><td class="patent-data-table-td patent-date-value">Mar 7, 2007</td><td class="patent-data-table-td patent-date-value">Oct 16, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for use in identification of bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8298760">US8298760</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 13, 2007</td><td class="patent-data-table-td patent-date-value">Oct 30, 2012</td><td class="patent-data-table-td ">Ibis Bioscience, Inc.</td><td class="patent-data-table-td ">Secondary structure defining database and methods for determining identity and geographic origin of an unknown bioagent thereby</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8394945">US8394945</a></td><td class="patent-data-table-td patent-date-value">Mar 7, 2007</td><td class="patent-data-table-td patent-date-value">Mar 12, 2013</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for use in identification of bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8527207">US8527207</a></td><td class="patent-data-table-td patent-date-value">May 15, 2008</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Peter K. Rogan</td><td class="patent-data-table-td ">Accurate identification of organisms based on individual information content</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20130124099">US20130124099</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 29, 2012</td><td class="patent-data-table-td patent-date-value">May 16, 2013</td><td class="patent-data-table-td ">David J. Ecker</td><td class="patent-data-table-td ">Secondary structure defining database and methods for determining identity and geographic origin of an unknown bioagent thereby</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010080616A1?cl=en">WO2010080616A1</a></td><td class="patent-data-table-td patent-date-value">Dec 18, 2009</td><td class="patent-data-table-td patent-date-value">Jul 15, 2010</td><td class="patent-data-table-td ">Abbott Laboratories</td><td class="patent-data-table-td ">Molecular assay for diagnosis of malaria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012159089A1?cl=en">WO2012159089A1</a></td><td class="patent-data-table-td patent-date-value">May 18, 2012</td><td class="patent-data-table-td patent-date-value">Nov 22, 2012</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Products and processes for multiplex nucleic acid identification</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006190">435/6.19</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc710/defs710.htm&usg=AFQjCNEn5ByvpUOuGF3R-upj_HDvCv7LsQ#C710S003000">710/3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S007100">435/7.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc702/defs702.htm&usg=AFQjCNEwDgPZO8lzVvNBu0VAYuT88y7k4g#C702S020000">702/20</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc702/defs702.htm&usg=AFQjCNEwDgPZO8lzVvNBu0VAYuT88y7k4g#C702S027000">702/27</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc702/defs702.htm&usg=AFQjCNEwDgPZO8lzVvNBu0VAYuT88y7k4g#C702S019000">702/19</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006120">435/6.12</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G06F0019180000">G06F19/18</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G06F0019000000">G06F19/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0019340000">C12P19/34</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61B">A61B</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001700000">C12Q1/70</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033480000">G01N33/48</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0031000000">G01N31/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033500000">G01N33/50</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G06F19/18">G06F19/18</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G06F19/10">G06F19/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Xcx7BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G06F19/3493">G06F19/3493</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">G06F19/18</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Oct 25, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 7, 2009</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-30 IS CONFIRMED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 26, 2008</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20080627</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 14, 2007</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">IBIS BIOSCIENCES, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:ISIS PHARMACEUTICALS, INC.;REEL/FRAME:019690/0036</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20070814</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">IBIS BIOSCIENCES, INC.,CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:ISIS PHARMACEUTICALS, INC.;US-ASSIGNMENT DATABASE UPDATED:20100223;REEL/FRAME:19690/36</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 17, 2007</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 11, 2002</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ISIS PHARMACEUTICALS, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:ECKER, DAVID J.;GRIFFEY, RICHARD H.;SAMPATH, RANGARAJAN;AND OTHERS;REEL/FRAME:012706/0435;SIGNING DATES FROM 20011116 TO 20011220</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2jG8qTGA0JPoRAPHWDHnRIBITj3w\u0026id=Xcx7BAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U3-61oYaB5ZthDYQqMq65wfmhZlNg\u0026id=Xcx7BAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U3fd9V7OFlJgLNDgosfXAooCQv7jQ","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Detecting_biological_warfare_agent_obtai.pdf?id=Xcx7BAABERAJ\u0026output=pdf\u0026sig=ACfU3U2TlrhShr0AIe0tNOEX1sdd9RArDA"},"sample_url":"http://www.google.com/patents/reader?id=Xcx7BAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>