<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7226739 - Methods for rapid detection and identification of bioagents in ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Methods for rapid detection and identification of bioagents in epidemiological and forensic investigations"><meta name="DC.contributor" content="David J. Ecker" scheme="inventor"><meta name="DC.contributor" content="Richard H. Griffey" scheme="inventor"><meta name="DC.contributor" content="Rangarajan Sampath" scheme="inventor"><meta name="DC.contributor" content="Steven A. Hofstadler" scheme="inventor"><meta name="DC.contributor" content="John McNeil" scheme="inventor"><meta name="DC.contributor" content="Stanley T. Crooke" scheme="inventor"><meta name="DC.contributor" content="James C. Hannis" scheme="inventor"><meta name="DC.contributor" content="Isis Pharmaceuticals, Inc" scheme="assignee"><meta name="DC.date" content="2003-9-12" scheme="dateSubmitted"><meta name="DC.description" content="The present invention provides methods for rapid forensic investigations by identification of bioagents associated with biowarfare and acts of terrorism or crime. The methods are also useful for epidemiological investigations by genotyping of bioagents."><meta name="DC.date" content="2007-6-5" scheme="issued"><meta name="DC.relation" content="US:20020168630:A1" scheme="references"><meta name="DC.relation" content="US:5547835" scheme="references"><meta name="DC.relation" content="US:5605798" scheme="references"><meta name="DC.relation" content="US:5622824" scheme="references"><meta name="DC.relation" content="US:5691141" scheme="references"><meta name="DC.relation" content="US:5849492" scheme="references"><meta name="DC.relation" content="US:5872003" scheme="references"><meta name="DC.relation" content="US:5965363" scheme="references"><meta name="DC.relation" content="US:6043031" scheme="references"><meta name="DC.relation" content="US:6180372" scheme="references"><meta name="DC.relation" content="US:6221587" scheme="references"><meta name="DC.relation" content="US:6613509" scheme="references"><meta name="DC.relation" content="WO:1997033000:A1" scheme="references"><meta name="DC.relation" content="WO:1997037041:A2" scheme="references"><meta name="DC.relation" content="WO:1998012355:A1" scheme="references"><meta name="DC.relation" content="WO:1998020166:A2" scheme="references"><meta name="DC.relation" content="WO:1998054751:A1" scheme="references"><meta name="DC.relation" content="WO:1999014375:A2" scheme="references"><meta name="DC.relation" content="WO:1999031278:A1" scheme="references"><meta name="citation_reference" content="Aaserud, et al., &quot;Accurate base composition of double-strand DNA by mass spectrometry,&quot; J. Am. Soc. Mass Spec. (1996) 7:1266-1269."><meta name="citation_reference" content="Baker, et al., &quot;Review and re-analysis of domain-specific 16S primers,&quot; J. Microbiol. Methods (2003) 55:541-555."><meta name="citation_reference" content="Benson, et al., &quot;Advantages of Thermococcus kodakaraenis (KOD) DNA polymerase for PCR-mass spectrometry based analyses,&quot; J. Am. Soc. Mass Spectrom. (2003) 14:601-604."><meta name="citation_reference" content="Black. et al., &quot;Detection of trace levels of tricothecene mycotoxins in human urineby gas chromatography-mass spectrometry,&quot; J. Chromatog. (1986) 367:103-115."><meta name="citation_reference" content="Campbell and Huang, &quot;Detection of California serogroup Bunyavirus in tissue culture and mosquito pools by PCR,&quot; J. Virol. Methods (1996) 57:175-179."><meta name="citation_reference" content="Chen, et al., &quot;A universal PCR primer to detect members of the Potyviridae and its use to examine the taxonomic status of several members of the family,&quot; Arch. Virol. (2001) 146:757-766."><meta name="citation_reference" content="Conrads, et al., &quot;16S-23S rDNA internal transcribed spacer sequences for analysis of the phylogenetic relationships among species of the genus Fusobacterium,&quot; Intl. J. System. Evol. Microbiol. (2002) 52:493-499."><meta name="citation_reference" content="Dasen, et al., &quot;Classification and identification of Propioibacteria based on ribosomal RNA genes and PCR,&quot; System. Appl. Microbiol. (1998) 21:251-259."><meta name="citation_reference" content="Deforce, et al., &quot;Characterization of DNA oligonucleotides by coupling of capillary zone electrophoresis to electrospray ionization Q-TOF mass spectrometry,&quot; Anal. Chem. (1998) 70:3060-3068."><meta name="citation_reference" content="Demesure, et al., &quot;A set of universal primers for amplification of polymorphic non-coding regions of mitochondrial and chloroplast DNA in plants,&quot; Mol. Ecol. (1995) 4:129-131."><meta name="citation_reference" content="Flora. et al., &quot;Dual-micro-ESI source for precise mass determination on a quadrupole time-of-flight mass spectrometer for genomic and proteomic applications,&quot; Anal. Bioanal. Chem. (2002) 373:538-546."><meta name="citation_reference" content="Fox et al., &quot;Report of the &#39;Bioterrorism Workshop&#39;&quot;, J. Microbol. Methods (2002) 51:247-254."><meta name="citation_reference" content="Fox, et al., &quot;Identification of Brucella by ribosomal-spacer-region PCR and differentiation of Brucell canis from other Brucella spp. pathogenic for humans by carbohydrate profiles,&quot; J. Clin. Microbiol. (1998) 36:3217-3222."><meta name="citation_reference" content="Griffey and Greig, &quot;Detection of base pair mismatches in duplex DNA and RNA oligonucleotides using electrospray mass spectrometry,&quot; SPIE (1997) 2985:82-86."><meta name="citation_reference" content="Griffin, et al., &quot;Direct genetic analysis by matrix-assisted laseer desorption/ionization mass spectrometry,&quot; proc. Natl. Acad. Sci. USA (1999) 96:6301-6306."><meta name="citation_reference" content="Hannis and Muddiman, &quot;Accurate characterization fo the tyrosine hydroxylase forensic allele 9.3 through development of electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,&quot; Rapid. Comm. Mass Spectrom. (1999) 13:954-962."><meta name="citation_reference" content="Hannis and Muddiman, &quot;Detection of double-stranded PCR amplicons at the attomote level electrosprayed from low nanomolar solutions using FT-ICR mass spectrometry,&quot; Fresenius J. Anal Chem. (2001) 369:246-251."><meta name="citation_reference" content="Hannis and Muddiman, &quot;Genotyping short tandem repeats using flow injection and electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,&quot; Rapid. Comm. Mass Spectrom. (2001) 15:348-350."><meta name="citation_reference" content="Hayashi, et al., &quot;Phylogenetic analysis of the human gut microbiota using 16S rDNA clone libraries and strictly anaerobic culture based methods,&quot; Microbiol. Immunol. (2002) 46:535-548."><meta name="citation_reference" content="Hoffmann, et al., &quot;Universal primer set for the full-length amplification of all influenza A viruses,&quot; Arch. Virol. (2001) 146:2275-2289."><meta name="citation_reference" content="Hurst, et al., &quot;Detection of bacterial DNA polymerase chain reaction products by matrix-assisted laser desorption/ionization mass spectrometry,&quot; Rapid. Comm. Mass. Spec. (1996) 10:377-382."><meta name="citation_reference" content="Isola, et al., &quot;MALDI-TOF mass spectrometric method for detection of hybridized DNA oligomers,&quot; Anal. Chem. (2001) 73:2126-2131."><meta name="citation_reference" content="Jankowski and Soler, &quot;Mass spectrometry of DNA: Part 2* Quantitative estimation of base composition,&quot; Eur. J. Mass Spectrom. Biochem. Med. Environ. Res. (1980) 1:45-52."><meta name="citation_reference" content="Kageyama and Benno, &quot;Rapid detection f human fecal Eubacterium species and related genera by tested PCR method,&quot; Microbiol. Immunol. (2001) 45:315-318."><meta name="citation_reference" content="Little, et al., &quot;Rapid sequencing of oligonucleotides by high-resolution mass spectrometry,&quot; J. Am. Chem. Soc. (1994) 116:4893-4897."><meta name="citation_reference" content="Liu, et al., &quot;Improving the microdialysis procedure for electrospray ionization mass spectrometry of biological samples,&quot; J. Mass Spectrom. (1997) 32:425-431."><meta name="citation_reference" content="Mangrum, et al., &quot;Solution composition and thermal denaturation for the production of single-stranded PCR amplicons: piperidine-induced destabilization of the DNA duplex,&quot; J. Am. Soc. Mass Spectrom. (2002) 13:232-240."><meta name="citation_reference" content="McCabe, et al., &quot;Bacterial species identification after DNA amplification with a universal primer pair,&quot; Mol. Genet. Metab. (1999) 66:205-211."><meta name="citation_reference" content="Meiyu, et al., &quot;Detection of flaviviruses by reverse transcriptase-polymerase chain reaction with the universal primer set,&quot; Microbiol. Immunol. (1997) 41:209-213."><meta name="citation_reference" content="Moricca, et al., &quot;Detection of Fusarium oxysporum f.sp. vasinfectum in cotton tissue by polymerase chain reaction,&quot; Plant Pathol. (1998) 47:486-494."><meta name="citation_reference" content="Muddiman, et al., &quot;Characterization of PCR products from Bacilli using electrospray ionization FTICR mass spectrometry,&quot; Anal Chem. (1996) 68:3705-3712."><meta name="citation_reference" content="Muddiman, et al., &quot;Length and base composition of PCR-amplified nucleic acids using mass measurements from electrospray ionization mass spectrometry,&quot; Anal. Chem. (1997) 69:1543-1549."><meta name="citation_reference" content="Muddiman, et al., &quot;Precise mass measurement of a double-stranded 500 base-pair (309 kDa) polymerase chain reaction product by negative ion electrospray ionization fourier transform ion cyclotron resonance mass spectrometry,&quot; Rapid Comm. Mass Spec. (1999) 13:1201-1204."><meta name="citation_reference" content="Muddiman, et al., &quot;Sequencing and characterization of larger oligonucleotides by electrospray ionization fourier transform ion cyclotron resonance mass spectrometry,&quot; Rev. Anal. Chem. (1998) 17:1-68."><meta name="citation_reference" content="Nagpal, et al., &quot;Utility of 16S-23S rRNA spacer region methodology: how similar are interspace regions within a genome and between strains for closely related organisms?,&quot; J. Microbiol. Methods (1998) 33:211-219."><meta name="citation_reference" content="Nakao et al (J. Clin. Microbiol. (1997) 35(7):1651-1655)."><meta name="citation_reference" content="Nishikawa (FEMS Microbiol. Letters (1999) 178:13-18)."><meta name="citation_reference" content="Null and Muddiman, &quot;Determination of a correction to improve mass measurement accuracy of isotopically unresolved polymerase chain reaction amplicons by electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,&quot; Rapid Comm. Mass Spectrom. (2003) 17:1714-1722."><meta name="citation_reference" content="Null and Muddiman, &quot;Perspectives on the use of electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry for short tandem repeat genotyping in the post genome era,&quot; J. Mass Spectrom. (2001) 36:589-606."><meta name="citation_reference" content="Null, et al., &quot;Evaluation of sample preparation techniques for mass measurements of PCR products using ESI-FT-ICR mass spectrometry,&quot; Am Soc. Mass Spectrom. (2002) 13:338-344."><meta name="citation_reference" content="Null, et al., &quot;Genotyping of simple and compound short tandem repeat loci using electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,&quot; Anal. Chem. (2001) 73:4514-4521."><meta name="citation_reference" content="Null, et al., &quot;Implications of hydrophobicity and free energy of solvation for characterization of nucleic acids by electrospray ionization mass spectrometry,&quot; Anal. Chem. (2003) 75:1331-1339."><meta name="citation_reference" content="Null, et al., &quot;Preparation of single-stranded PCR products for electrospray ionization mass spectrometry using the DNA repair enzyme lambda exonuclease,&quot; Analyst (2000) 125:619-626."><meta name="citation_reference" content="Peng, et al., &quot;Rapid detection of Shigella species in environmental sewage by an immunocapture PCR with universal primers,&quot; App. Environ. Microbiol. (2002) 68:2580-2583."><meta name="citation_reference" content="Pomerantz, et al., &quot;Determination of oligonucleotide composition from mass spectrometrically measured molecular weight,&quot; J. Am. Soc. Mass Spectrom. (1993) 4:204-209."><meta name="citation_reference" content="Ross, et al., &quot;Discrimination of single-nucleotide polymorphisms in human DNA using peptide nucleic acid probes detected by MALDI-TOF mass spectrometry,&quot; Anal. Chem. (1997) 69:4197-4202."><meta name="citation_reference" content="Scaramozzino, et al., &quot;Comparison of Flavivirus universal primer pairs and development of a rapid, highly sensitive heminested reverse transcription-PCR assay for detection of flaviviruses targeted to a conserved region of the NS5 gene sequences,&quot; J. Clin. Microbiol. (2001) 39:1922-1927."><meta name="citation_reference" content="Shaver, et al., &quot;Restriction fragment length polymorphism of rRNA operons for discrimination and intergenic spacer sequences for cataloging of Bacilus subtilis sub-groups,&quot; J. Microbiol. Methods (2002) 50:215-223."><meta name="citation_reference" content="Srinivasan, et al., &quot;Matrix-assisted laser desorption/ionization time-of-flight mass spectrometry as a rapid screening method to detect mutations causing Tay-Sachs disease,&quot; Rapid Comm. Mass Spectrom. (1997) 11:1144-1150."><meta name="citation_reference" content="Steffens and Roy, &quot;Sequence analysis of mitochondrial DNA hypervariable regions using infrared fluorescence detection,&quot; Bio/Techniques (1998) 24:1044-1046."><meta name="citation_reference" content="Wunschel, et al., &quot;Heterogeneity in bacillus cereus PCR products detected by ESI-FTICR mass spectrometry,&quot; Anal. Chem. (1998) 70:1203-1207."><meta name="citation_reference" content="Wunschel, et al., &quot;Mass spectrometric characterization of DNA for molecular biological applications: advances using MALDI and ESI,&quot; Adv. Mass Spectrom., vol. 14, Karjalainen, et al., (eds.) 1998, Elsevier, Amsterdam."><meta name="citation_patent_number" content="US:7226739"><meta name="citation_patent_application_number" content="US:10/660,997"><link rel="canonical" href="http://www.google.com/patents/US7226739"/><meta property="og:url" content="http://www.google.com/patents/US7226739"/><meta name="title" content="Patent US7226739 - Methods for rapid detection and identification of bioagents in epidemiological and forensic investigations"/><meta name="description" content="The present invention provides methods for rapid forensic investigations by identification of bioagents associated with biowarfare and acts of terrorism or crime. The methods are also useful for epidemiological investigations by genotyping of bioagents."/><meta property="og:title" content="Patent US7226739 - Methods for rapid detection and identification of bioagents in epidemiological and forensic investigations"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("BJTtU7TOOtGrsATV1YD4Dw"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("IRL"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("BJTtU7TOOtGrsATV1YD4Dw"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("IRL"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7226739?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7226739"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=2EN8BAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7226739&amp;usg=AFQjCNEVbSBLF6005QjEqOPgoO4-WB306Q" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7226739.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7226739.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20070048735"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7226739"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7226739" style="display:none"><span itemprop="description">The present invention provides methods for rapid forensic investigations by identification of bioagents associated with biowarfare and acts of terrorism or crime. The methods are also useful for epidemiological investigations by genotyping of bioagents....</span><span itemprop="url">http://www.google.com/patents/US7226739?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7226739 - Methods for rapid detection and identification of bioagents in epidemiological and forensic investigations</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7226739 - Methods for rapid detection and identification of bioagents in epidemiological and forensic investigations" title="Patent US7226739 - Methods for rapid detection and identification of bioagents in epidemiological and forensic investigations"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7226739 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 10/660,997</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jun 5, 2007</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Sep 12, 2003</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Mar 2, 2001</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US8563250">US8563250</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20070048735">US20070048735</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120122086">US20120122086</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120171679">US20120171679</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2004053164A1">WO2004053164A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">10660997, </span><span class="patent-bibdata-value">660997, </span><span class="patent-bibdata-value">US 7226739 B2, </span><span class="patent-bibdata-value">US 7226739B2, </span><span class="patent-bibdata-value">US-B2-7226739, </span><span class="patent-bibdata-value">US7226739 B2, </span><span class="patent-bibdata-value">US7226739B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22David+J.+Ecker%22">David J. Ecker</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Richard+H.+Griffey%22">Richard H. Griffey</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Rangarajan+Sampath%22">Rangarajan Sampath</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Steven+A.+Hofstadler%22">Steven A. Hofstadler</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22John+McNeil%22">John McNeil</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Stanley+T.+Crooke%22">Stanley T. Crooke</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22James+C.+Hannis%22">James C. Hannis</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Isis+Pharmaceuticals,+Inc%22">Isis Pharmaceuticals, Inc</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7226739.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7226739.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7226739.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (19),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (52),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (13),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (14),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (6)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=2EN8BAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7226739&usg=AFQjCNFDG5sRxAhjAe1-FnfHuUwzazYlvQ">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=2EN8BAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7226739&usg=AFQjCNEJ336g-ZwP2kHqlOYMdj06Q4JEpQ">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=2EN8BAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7226739B2%26KC%3DB2%26FT%3DD&usg=AFQjCNF4OerMjNyYog0hMRLLMpXQLANpXg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT55753396" lang="EN" load-source="patent-office">Methods for rapid detection and identification of bioagents in epidemiological and forensic investigations</invention-title></span><br><span class="patent-number">US 7226739 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA51160337" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">The present invention provides methods for rapid forensic investigations by identification of bioagents associated with biowarfare and acts of terrorism or crime. The methods are also useful for epidemiological investigations by genotyping of bioagents.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(29)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00013.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00013.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00014.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00014.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00015.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00015.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00016.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00016.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00017.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00017.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00018.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00018.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00019.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00019.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00020.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00020.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00021.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00021.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00022.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00022.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00023.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00023.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00024.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00024.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00025.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00025.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00026.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00026.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00027.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00027.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00028.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00028.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-D00029.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7226739B2/US07226739-20070605-D00029.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(11)</span></span></div><div class="patent-text"><div mxw-id="PCLM9220183" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A method of determining the genotype of a bioagent comprising the steps of:
<div class="claim-text">selecting at least one pair of oligonucleotide primers, wherein one member of said pair of primers hybridizes to a first conserved region of nucleic acid encoding a ribosomal RNA and the other member of said pair of primers hybridizes to a second conserved region of nucleic acid encoding said ribosomal RNA wherein said first and second conserved regions flank a variable nucleic acid region that when amplified creates a unique base composition signature among at least eight bioagents;</div>
<div class="claim-text">amplifying nucleic acid from at least one said bioagent with said pair of oligonucleotide primers to produce an amplification product;</div>
<div class="claim-text">determining the molecular mass of said amplification product by mass spectrometry;</div>
<div class="claim-text">calculating the base composition of said amplification product from said molecular mass;</div>
<div class="claim-text">comparing said base composition of at least one said bioagent to nineteen or more calculated or measured base compositions of amplification products of known bioagents produced by using said at least one pair of oligonucleotide primers, thereby identifying said unknown bioagent at the species level; and</div>
<div class="claim-text">identifying a sub-species characteristic of said bioagent, thereby determining the genotype of said bioagent.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein said sub-species characteristic comprises at least one single nucleotide polymorphism.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the molecular masses of said amplification products are determined by ESI-TOF mass spectrometry.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein said bioagent is a bacterium, mold, fungus or parasite.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. A method of determining the genotype of a bioagent comprising the steps of:
<div class="claim-text">selecting at least one pair of oligonucleotide primers, wherein one member of said pair of primers hybridizes to a first conserved region of a nucleic acid encoding a protein that participates in translation, replication, recombination, repair, transcription, nucleotide metabolism, amino acid metabolism, lipid metabolism, uptake, secretion, antibiotic resistance, virulence, or pathogenicity, and the other member of said pair of primers hybridizes to a second conserved region of said nucleic acid encoding a protein that participates in translation, replication, recombination, repair, transcription, nucleotide metabolism, amino acid metabolism, lipid metabolism, uptake, secretion, antibiotic resistance, virulence, or pathogenicity, wherein said first and second conserved regions flank a variable nucleic acid region that when amplified creates unique base composition signatures among at least eight bioagents;</div>
<div class="claim-text">amplifying nucleic acid from at least one said bioagent with said pair of oligonucleotide primers to produce an amplification product;</div>
<div class="claim-text">determining the molecular mass of said amplification product by mass spectrometry;</div>
<div class="claim-text">calculating the base composition of said amplification product from said molecular mass;</div>
<div class="claim-text">comparing said base composition of at least one said bioagent to nineteen or more calculated or measured base compositions of amplification products of known bioagents produced by using said at least one pair of oligonucleotide primers, thereby identifying said unknown bioagent at the species level; and</div>
<div class="claim-text">identifying a sub-species characteristic of said bioagent, thereby determining the genotype of said bioagent.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. The method of <claim-ref idref="CLM-00005">claim 5</claim-ref> wherein said sub-species characteristic comprises at least one single nucleotide polymorphism.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. The method of <claim-ref idref="CLM-00005">claim 5</claim-ref> wherein said sub-species characteristic comprises a pathogenicity factor.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. The method of <claim-ref idref="CLM-00007">claim 7</claim-ref> wherein said pathogenicity factor is a pathogenicity island, a virulence marker, or a toxin gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The method of <claim-ref idref="CLM-00008">claim 8</claim-ref> wherein said toxin gene has been inserted by genetic engineering.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The method of <claim-ref idref="CLM-00005">claim 5</claim-ref> wherein said molecular masses of said amplification products are determined by ESI-TOF mass spectrometry.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. The method of <claim-ref idref="CLM-00005">claim 5</claim-ref> wherein said bioagent is a bacterium, virus, mold, fungus or parasite.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES16242727" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS-REFERENCE TO RELATED APPLICATIONS</heading> <p num="p-0002">The present application: 1) is a continuation-in-part of U.S. application Ser. No. 10/323,438 filed Dec. 18, 2002 now abandoned; 2) is a continuation-in-part of U.S. application Ser. No. 09/798,007 filed Mar. 2, 2001 now abandoned; 3) claims the benefit of U.S. provisional application Ser. No. 60/431,319 filed Dec. 6, 2002; 4) claims the benefit of U.S. provisional application Ser. No. 60/433,499 filed Dec. 12, 2002; and 5) claims the benefit of U.S. provisional application Ser. No. 60/461,494 filed Apr. 9, 2003, each of which is in incorporated herein by reference in its entirety.</p>
    <heading>STATEMENT OF GOVERNMENT SUPPORT</heading> <p num="p-0003">This invention was made with United States Government support under DARPA/SPO contract 4400044016. The United States Government has certain rights in the invention.</p>
    <heading>FIELD OF THE INVENTION</heading> <p num="p-0004">This invention relates to the field of forensic and epidemiological investigations. The methods provide rapid identification of known or suspected terrorists or criminals based on forensic evidence. Additionally, the methods provide tracking of the geographic locations of the terrorists or criminals by microbial geographic profiling of bioagents associated with the terrorists or criminals. Furthermore, the methods provide genotyping of bioagents such as those associated with acts of biowarfare, terrorism or criminal activity.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p num="p-0005">The ease with which small countries and terrorist groups can now obtain biological warfare agents has escalated the need to provide the war fighter and civilians alike with miniature, easy to use, disposable instruments for detection and identification of potentially hazardous biological agents (Iqbal et al. <i>Biosensors </i>&amp; <i>Bioelectronics </i>2000, 15, 549578; Christel, L. A., et al. <i>J. Biomech. Eng. </i>1999, 121, 2227; Higgins, J. A., et al. <i>Ann. NY Acad. Sci. </i>1999, 894, 130148; and Hood, <i>E. Environ. Health Perspect. </i>1999, 107, 931932). Traditional methods for detection and identification of microorganisms, viruses and/or their products lack the speed and sensitivity to be of field usage since they are not real time or even typically completed in a single day. Microbial and viral identification assays have as their basis, the principle that dates back to the days of Pasteur, i.e. the growth of the organism in culture or replication of virus in a suitable host (Reischl, U., <i>Frontiers Biosci., </i>1996, 1, Application of molecular biology-based methods to the diagnosis of infectious diseases. 1, e72e77). Toxin identification has typically relied on biological assays, which although relatively rapid, usually requires purification of the toxin prior to testing (Feng, <i>P. Mol. Biotechnol. </i>1997, 7, 267278; van der Zee, H. et al. <i>J. AOAC Int. </i>1997, 80, 934940). Depending upon the nature of the agent to be detected, this process can take from days to months (Pillai, S. D. <i>Arch. Virol. </i>1997, 13 Suppl., 6782; van der Zee, H. et al. <i>J. AOAC Int. </i>1997, 80, 934940). Clearly, this is not practical in situations where detection and identification may be required for protecting a population from hazardous biological agents. Molecular recognition systems that can be used for rapid identification can improve response time and thus avert or reduce the number of casualties associated with a potential bioterrorism or biowarfare event.</p>
    <p num="p-0006">Biological threat agents can be either infectious or toxigenic organisms or simply toxins (Hood, E. <i>Environ. Health Perspect. </i>1999, 107, 931932; Haines, J. D. et al <i>J. Okla. State Med. Assoc. </i>1999, 93, 187196). Examples of the former are <i>Bacillus anthracis</i>(anthrax) and <i>Yersinia pestis </i>(plague) while the latter is exemplified by staphylococcal enterotoxin B or botulinum toxin. Detection of toxins has followed a similar track as that observed for detection of chemical threat agents. Typically, toxins are detected on the basis of their respective chemical structures. Detection of mycotoxins has been accomplished using traditional analytical chemistry tools such as gas chromatography-mass spectrometry (GC-MS) (Black, R. M et al. <i>J. Chromatogr. </i>1986, 367, 103116). Although precise and highly sensitive, GC-MS does not lend itself to field applications and cannot be easily applied to complex target analytes such as bacteria. Specific compounds, i.e. signature components might be identified in targeted bacterial agents but this approach tends to be too complex for routine, high throughput analysis (van der Zee, H. et al. <i>J. AOAC Int. </i>1997, 80, 934940; Hood, E. <i>Environ. Health Perspect. </i>1999, 107, 931932).</p>
    <p num="p-0007">A number of technological innovations have provided tools that have made detection and identification of microorganisms, viruses and their products faster and more sensitive. Two significant technologies that have had dramatic impact on potentially rapid detection are: (1) generation of monoclonal antibodies; and (2) collection of individual methodological advances that have formed the basis of recombinant DNA technology. A key event in the latter technology is the development of polymerase chain reaction (PCR) technology which as a singular process, has significantly reshaped the authors' thinking with respect to detection of biological agents.</p>
    <p num="p-0008">Identification of biological threat agents involves recognition of bacteria (vegetative cells and spores), viruses and toxins. Nucleic acid and immunology-based methods for identification of bacteria, viruses and their products (antigens and toxins) have found wide application including testing of food, clinical and environmental samples. Extension of these applications for detection and identification of biological threat agents is reasonable since basic principles involved are identical. Two essential features characterize these applications, i.e. the need to: (1) develop a target specific identification method; and (2) formulate an assay that will work on the requisite sample. However, the majority of the methods developed meet the first criterion, i.e. identify the target organism or toxin, but fail to be sufficiently robust to work on real samples.</p>
    <p num="p-0009">Mass spectrometry provides detailed information about the molecules being analyzed, including high mass accuracy. It is also a process that can be easily automated. Low-resolution MS may be unreliable when used to detect some known agents, if their spectral lines are sufficiently weak or sufficiently close to those from other living organisms in the sample. DNA chips with specific probes can only determine the presence or absence of specifically anticipated organisms. Because there are hundreds of thousands of species of benign bacteria, some very similar in sequence to threat organisms, even arrays with 10,000 probes lack the breadth needed to detect a particular organism.</p>
    <p num="p-0010">Antibodies face more severe diversity limitations than arrays. If antibodies are designed against highly conserved targets to increase diversity, the false alarm problem will dominate, again because threat organisms are very similar to benign ones. Antibodies are only capable of detecting known agents in relatively uncluttered environments.</p>
    <p num="p-0011">Several groups have described detection of PCR products using high resolution electrospray ionization-Fourier transform-ion cyclotron resonance mass spectrometry (ESI-FT-ICR MS). Accurate measurement of exact mass combined with knowledge of the number of at least one nucleotide allowed calculation of the total base composition for PCR duplex products of approximately 100 base pairs. (Aaserud et al., <i>J. Am. Soc. Mass Spec., </i>1996, 7, 12661269; Muddiman et al., <i>Anal. Chem., </i>1997, 69, 15431549; Wunschel et al., <i>Anal. Chem., </i>1998, 70, 12031207; Muddiman et al., <i>Rev. Anal. Chem., </i>1998, 17, 168). Electrospray ionization-Fourier transform-ion cyclotron resistance (ESI-FT-ICR) MS may be used to determine the mass of double-stranded, 500 base-pair PCR products via the average molecular mass (Hurst et al., <i>Rapid Commun. Mass Spec. </i>1996, 10, 377382). The use of matrix-assisted laser desorption ionization-time of flight (MALDI-TOF) mass spectrometry for characterization of PCR products has been described. (Muddiman et al., <i>Rapid Commun. Mass Spec., </i>1999, 13, 12011204). However, the degradation of DNAs over about 75 nucleotides observed with MALDI limited the utility of this method.</p>
    <p num="p-0012">U.S. Pat. No. 5,849,492 describes a method for retrieval of phylogenetically informative DNA sequences which comprise searching for a highly divergent segment of genomic DNA surrounded by two highly conserved segments, designing the universal primers for PCR amplification of the highly divergent region, amplifying the genomic DNA by PCR technique using universal primers, and then sequencing the gene to determine the identity of the organism.</p>
    <p num="p-0013">U.S. Pat. No. 5,965,363 discloses methods for screening nucleic acids for polymorphisms by analyzing amplified target nucleic acids using mass spectrometric techniques and to procedures for improving mass resolution and mass accuracy of these methods.</p>
    <p num="p-0014">WO 99/14375 describes methods, PCR primers and kits for use in analyzing preselected DNA tandem nucleotide repeat alleles by mass spectrometry.</p>
    <p num="p-0015">WO 98/12355 discloses methods of determining the mass of a target nucleic acid by mass spectrometric analysis, by cleaving the target nucleic acid to reduce its length, making the target single-stranded and using MS to determine the mass of the single-stranded shortened target. Also disclosed are methods of preparing a double-stranded target nucleic acid for MS analysis comprising amplification of the target nucleic acid, binding one of the strands to a solid support, releasing the second strand and then releasing the first strand which is then analyzed by MS. Kits for target nucleic acid preparation are also provided.</p>
    <p num="p-0016">PCT WO97/33000 discloses methods for detecting mutations in a target nucleic acid by nonrandomly fragmenting the target into a set of single-stranded nonrandom length fragments and determining their masses by MS.</p>
    <p num="p-0017">U.S. Pat. No. 5,605,798 describes a fast and highly accurate mass spectrometer-based process for detecting the presence of a particular nucleic acid in a biological sample for diagnostic purposes.</p>
    <p num="p-0018">WO 98/21066 describes processes for determining the sequence of a particular target nucleic acid by mass spectrometry. Processes for detecting a target nucleic acid present in a biological sample by PCR amplification and mass spectrometry detection are disclosed, as are methods for detecting a target nucleic acid in a sample by amplifying the target with primers that contain restriction sites and tags, extending and cleaving the amplified nucleic acid, and detecting the presence of extended product, wherein the presence of a DNA fragment of a mass different from wild-type is indicative of a mutation. Methods of sequencing a nucleic acid via mass spectrometry methods are also described.</p>
    <p num="p-0019">WO 97/37041, WO 99/31278 and U.S. Pat. No. 5,547,835 describe methods of sequencing nucleic acids using mass spectrometry. U.S. Pat. Nos. 5,622,824, 5,872,003 and 5,691,141 describe methods, systems and kits for exonuclease-mediated mass spectrometric sequencing.</p>
    <p num="p-0020">Thus, there is a need for methods for bioagent detection and identification which is both specific and rapid, and in which no nucleic acid sequencing is required. The present invention addresses this need as well as other needs.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0021">The present invention is directed to methods of identification of a bioagent associated with an act of biowarfare, terrorism or criminal activity by determining a first molecular mass of a first amplification product of a first bioagent identifying amplicon obtained from a sample taken at the scene of biowarfare, terrorism or criminal activity and comparing the first molecular mass to a second molecular mass of a second bioagent identifying amplicon wherein both first and second bioagent identifying amplicons are correlative.</p>
    <p num="p-0022">The present invention is also directed to forensic methods for tracking the geographic location of a bioagent associated with an act of biowarfare by determining a first molecular mass of a first amplification product of a first bioagent identifying amplicon from a forensic sample obtained from a given geographic location; and comparing the first molecular mass to a second molecular mass of a second bioagent identifying amplicon wherein both first and second bioagent identifying amplicons are correlative, wherein a match between the first molecular mass and the second molecular mass indicates at least transient presence of the bioagent associated with an act of biowarfare at the given geographic location.</p>
    <p num="p-0023">The present invention is also directed to methods of genotyping a bioagent, by determining a first molecular mass of a first amplification product of a first bioagent identifying amplicon that contains genotyping information and comparing the first molecular mass to a second molecular mass of a second bioagent identifying amplicon that contains genotyping information, wherein the first and second bioagent identifying amplicons are correlative, and wherein a match between the first molecular mass and the second molecular mass identifies a genotype of the bioagent.</p>
    <p num="p-0024">The present invention is also directed to methods of tracking a known or suspected terrorist or criminal by determining a first molecular mass of a first amplification product of a first bioagent identifying amplicon containing microbial geographic profiling information from a forensic sample known to be associated with the terrorist or criminal and comparing the first molecular mass to a second molecular mass of a second bioagent identifying amplicon wherein the second bioagent identifying amplicon contains microbial geographic profiling information, wherein both first and second bioagent identifying amplicons are correlative, and wherein a match between the first molecular mass and the second molecular mass indicates at least transient presence of the known or suspected criminal at the geographic location indicated by the microbial geographic profiling information.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p num="p-0025"> <figref idrefs="DRAWINGS">FIGS. 1A1H</figref> and <figref idrefs="DRAWINGS">FIG. 2</figref> are consensus diagrams that show examples of conserved regions from 16S rRNA (<figref idrefs="DRAWINGS">FIGS. 1A-1</figref>, <b>1</b>A-<b>2</b>, <b>1</b>A-<b>3</b>, <b>1</b>A-<b>4</b>, and <b>1</b>A-<b>5</b>), 23S rRNA (<b>3</b>-half, <figref idrefs="DRAWINGS">FIGS. 1B</figref>, <b>1</b>C, and ID; <b>5</b>-half, <figref idrefs="DRAWINGS">FIGS. 1EF</figref>), 23S rRNA Domain I (<figref idrefs="DRAWINGS">FIG. 1G</figref>), 23S rRNA Domain IV (<figref idrefs="DRAWINGS">FIG. 1H</figref>) and 16S rRNA Domain III (<figref idrefs="DRAWINGS">FIG. 2</figref>) which are suitable for use in the present invention. Lines with arrows are examples of regions to which intelligent primer pairs for PCR are designed. The label for each primer pair represents the starting and ending base number of the amplified region on the consensus diagram. Bases in capital letters are greater than 95% conserved; bases in lower case letters are 9095% conserved, filled circles are 8090% conserved; and open circles are less than 80% conserved. The label for each primer pair represents the starting and ending base number of the amplified region on the consensus diagram. The nucleotide sequence of the 16S rRNA consensus sequence is SEQ ID NO:3 and the nucleotide sequence of the 23S rRNA consensus sequence is SEQ ID NO:4.</p>
      <p num="p-0026"> <figref idrefs="DRAWINGS">FIG. 2</figref> shows a typical primer amplified region from the 16S rRNA Domain III shown in <figref idrefs="DRAWINGS">FIG. 1A-1</figref>.</p>
      <p num="p-0027"> <figref idrefs="DRAWINGS">FIG. 3</figref> is a schematic diagram showing conserved regions in RNase P. Bases in capital letters are greater than 90% conserved; bases in lower case letters are 8090% conserved; filled circles designate bases which are 7080% conserved; and open circles designate bases that are less than 70% conserved.</p>
      <p num="p-0028"> <figref idrefs="DRAWINGS">FIG. 4</figref> is a schematic diagram of base composition signature determination using nucleotide analog tags to determine base composition signatures.</p>
      <p num="p-0029"> <figref idrefs="DRAWINGS">FIG. 5</figref> shows the deconvoluted mass spectra of a <i>Bacillus anthracis </i>region with and without the mass tag phosphorothioate A (A*). The two spectra differ in that the measured molecular weight of the mass tag-containing sequence is greater than the unmodified sequence.</p>
      <p num="p-0030"> <figref idrefs="DRAWINGS">FIG. 6</figref> shows base composition signature (BCS) spectra from PCR products from <i>Staphylococcus aureus </i>(<i>S. aureus </i>16S<sub></sub>1337F) and <i>Bacillus anthracis </i>(<i>B. anthr. </i>16S<sub></sub>1337F), amplified using the same primers. The two strands differ by only two (ATCG) substitutions and are clearly distinguished on the basis of their BCS.</p>
      <p num="p-0031"> <figref idrefs="DRAWINGS">FIG. 7</figref> shows that a single difference between two sequences (A14 in <i>B. anthracis </i>vs. A15 in <i>B. cereus</i>) can be easily detected using ESI-TOF mass spectrometry.</p>
      <p num="p-0032"> <figref idrefs="DRAWINGS">FIG. 8</figref> is an ESI-TOF of <i>Bacillus anthracis </i>spore coat protein sspE 56mer plus calibrant. The signals unambiguously identify <i>B. anthracis </i>versus other <i>Bacillus </i>species.</p>
      <p num="p-0033"> <figref idrefs="DRAWINGS">FIG. 9</figref> is an ESI-TOF of a <i>B. anthracis </i>synthetic 16S<sub></sub>1228 duplex (reverse and forward strands). The technique easily distinguishes between the forward and reverse strands.</p>
      <p num="p-0034"> <figref idrefs="DRAWINGS">FIG. 10</figref> is an ESI-FTICR-MS of a synthetic <i>B. anthracis </i>16S<sub></sub>1337 46 base pair duplex.</p>
      <p num="p-0035"> <figref idrefs="DRAWINGS">FIG. 11</figref> is an ESI-TOF-MS of a 56mer oligonucleotide (3 scans) from the <i>B. anthracis </i>saspB gene with an internal mass standard. The internal mass standards are designated by asterisks.</p>
      <p num="p-0036"> <figref idrefs="DRAWINGS">FIG. 12</figref> is an ESI-TOF-MS of an internal standard with 5 mM TBA-TFA buffer showing that charge stripping with tributylammonium trifluoroacetate reduces the most abundant charge state from [M8H+]8 to [M3H+]3.</p>
      <p num="p-0037"> <figref idrefs="DRAWINGS">FIG. 13</figref> is a portion of a secondary structure defining database according to one embodiment of the present invention, where two examples of selected sequences are displayed graphically thereunder.</p>
      <p num="p-0038"> <figref idrefs="DRAWINGS">FIG. 14</figref> is a three dimensional graph demonstrating the grouping of sample molecular weight according to species.</p>
      <p num="p-0039"> <figref idrefs="DRAWINGS">FIG. 15</figref> is a three dimensional graph demonstrating the grouping of sample molecular weights according to species of virus and mammal infected.</p>
      <p num="p-0040"> <figref idrefs="DRAWINGS">FIG. 16</figref> is a three dimensional graph demonstrating the grouping of sample molecular weights according to species of virus, and animal-origin of infectious agent.</p>
      <p num="p-0041"> <figref idrefs="DRAWINGS">FIG. 17</figref> is a figure depicting how the triangulation method of the present invention provides for the identification of an unknown bioagent without prior knowledge of the unknown agent. The use of different primer sets to distinguish and identify the unknown is also depicted as primer sets I, II and III within this figure. A three dimensional graph depicts all of bioagent space (<b>170</b>), including the unknown bioagent, which after use of primer set I (<b>171</b>) according to a method according to the present invention further differentiates and classifies bioagents according to major classifications (<b>176</b>) which, upon further analysis using primer set II (<b>172</b>) differentiates the unknown agent (<b>177</b>) from other, known agents (<b>173</b>) and finally, the use of a third primer set (<b>175</b>) further specifies subgroups within the family of the unknown (<b>174</b>).</p>
      <p num="p-0042"> <figref idrefs="DRAWINGS">FIG. 18</figref> depicts three representative different mass spectral traces of bioagent identifying amplicons, each containing a single nucleotide polymorphism, indicating that molecular mass or base composition signature is capable of distinguishing the single nucleotide polymorphism.</p>
    </description-of-drawings> <heading>DESCRIPTION OF EMBODIMENTS</heading> <heading>A. Introduction</heading> <p num="p-0043">The present invention provides, inter alia, methods for detection and identification of bioagents in an unbiased manner using bioagent identifying amplicons. Intelligent primers are selected to hybridize to conserved sequence regions of nucleic acids derived from a bioagent and which bracket variable sequence regions to yield a bioagent identifying amplicon which can be amplified and which is amenable to molecular mass determination. The molecular mass then provides a means to uniquely identify the bioagent without a requirement for prior knowledge of the possible identity of the bioagent. The molecular mass or corresponding base composition signature (BCS) of the amplification product is then matched against a database of molecular masses or base composition signatures. Furthermore, the method can be applied to rapid parallel multiplex analyses, the results of which can be employed in a triangulation identification strategy. The present method provides rapid throughput and does not require nucleic acid sequencing of the amplified target sequence for bioagent detection and identification.</p>
    <heading>B. Bioagents</heading> <p num="p-0044">In the context of this invention, a bioagent is any organism, cell, or virus, living or dead, or a nucleic acid derived from such an organism, cell or virus. Examples of bioagents include, but are not limited, to cells, including but not limited to, cells, including but not limited to human clinical samples, bacterial cells and other pathogens) viruses, fungi, and protists, parasites, and pathogenicity markers (including but not limited to: pathogenicity islands, antibiotic resistance genes, virulence factors, toxin genes and other bioregulating compounds). Samples may be alive or dead or in a vegetative state (for example, vegetative bacteria or spores) and may be encapsulated or bioengineered. In the context of this invention, a pathogen is a bioagent which causes a disease or disorder.</p>
    <p num="p-0045">Despite enormous biological diversity, all forms of life on earth share sets of essential, common features in their genomes. Bacteria, for example have highly conserved sequences in a variety of locations on their genomes. Most notable is the universally conserved region of the ribosome. but there are also conserved elements in other non-coding RNAs, including RNAse P and the signal recognition particle (SRP) among others. Bacteria have a common set of absolutely required genes. About 250 genes are present in all bacterial species (<i>Proc. Natl. Acad. Sci. U.S.A., </i>1996, 93, 10268<i>; Science, </i>1995, 270, 397), including tiny genomes like <i>Mycoplasma, Ureaplasma </i>and <i>Rickettsia</i>. These genes encode proteins involved in translation, replication, recombination and repair, transcription, nucleotide metabolism, amino acid metabolism, lipid metabolism, energy generation, uptake, secretion and the like. Examples of these proteins are DNA polymerase III beta, elongation factor TU, heat shock protein groEL, RNA polymerase beta, phosphoglycerate kinase, NADH dehydrogenase, DNA ligase, DNA topoisomerase and elongation factor G. Operons can also be targeted using the present method. One example of an operon is the bfp operon from enteropathogenic <i>E. coli</i>. Multiple core chromosomal genes can be used to classify bacteria at a genus or genus species level to determine if an organism has threat potential. The methods can also be used to detect pathogenicity markers (plasmid or chromosomal) and antibiotic resistance genes to confirm the threat potential of an organism and to direct countermeasures.</p>
    <heading>C. Selection of Bioagent Identifying Amplicons</heading> <p num="p-0046">Since genetic data provide the underlying basis for identification of bioagents by the methods of the present invention, it is necessary to select segments of nucleic acids which ideally provide enough variability to distinguish each individual bioagent and whose molecular mass is amenable to molecular mass determination. In one embodiment of the present invention, at least one polynucleotide segment is amplified to facilitate detection and analysis in the process of identifying the bioagent. Thus, the nucleic acid segments which provide enough variability to distinguish each individual bioagent and whose molecular masses are amenable to molecular mass determination are herein described as bioagent identifying amplicons. The term amplicon as used herein, refers to a segment of a polynucleotide which is amplified in an amplification reaction.</p>
    <p num="p-0047">As used herein, intelligent primers are primers that are designed to bind to highly conserved sequence regions of a bioagent identifying amplicon that flank an intervening variable region and yield amplification products which ideally provide enough variability to distinguish each individual bioagent, and which are amenable to molecular mass analysis. By the term highly conserved, it is meant that the sequence regions exhibit between about 80100%, or between about 90100%, or between about 95100% identity. The molecular mass of a given amplification product provides a means of identifying the bioagent from which it was obtained, due to the variability of the variable region. Thus design of intelligent primers requires selection of a variable region with appropriate variability to resolve the identity of a given bioagent. Bioagent identifying amplicons are ideally specific to the identity of the bioagent. A plurality of bioagent identifying amplicons selected in parallel for distinct bioagents which contain the same conserved sequences for hybridization of the same pair of intelligent primers are herein defined as correlative bioagent identifying amplicons.</p>
    <p num="p-0048">In one embodiment, the bioagent identifying amplicon is a portion of a ribosomal RNA (rRNA) gene sequence. With the complete sequences of many of the smallest microbial genomes now available, it is possible to identify a set of genes that defines minimal life and identify composition signatures that uniquely identify each gene and organism. Genes that encode core life functions such as DNA replication, transcription, ribosome structure, translation, and transport are distributed broadly in the bacterial genome and are suitable regions for selection of bioagent identifying amplicons. Ribosomal RNA (rRNA) genes comprise regions that provide useful base composition signatures. Like many genes involved in core life functions, rRNA genes contain sequences that are extraordinarily conserved across bacterial domains interspersed with regions of high variability that are more specific to each species. The variable regions can be utilized to build a database of base composition signatures. The strategy involves creating a structure-based alignment of sequences of the small (16S) and the large (23S) subunits of the rRNA genes. For example, there are currently over 13,000 sequences in the ribosomal RNA database that has been created and maintained by Robin Gutell, University of Texas at Austin, and is publicly available on the Institute for Cellular and Molecular Biology web page on the world wide web of the Internet at, for example, rna.icmb.utexas.edu/. There is also a publicly available rRNA database created and maintained by the University of Antwerp, Belgium on the world wide web of the Internet at, for example, rrna.uia.ac.be.</p>
    <p num="p-0049">These databases have been analyzed to determine regions that are useful as bioagent identifying amplicons. The characteristics of such regions include: a) between about 80 and 100%, or greater than about 95% identity among species of the particular bioagent of interest, of upstream and downstream nucleotide sequences which serve as sequence amplification primer sites; b) an intervening variable region which exhibits no greater than about 5% identity among species; and c) a separation of between about 30 and 1000 nucleotides, or no more than about 50250 nucleotides, or no more than about 60100 nucleotides, between the conserved regions.</p>
    <p num="p-0050">As a non-limiting example, for identification of <i>Bacillus </i>species, the conserved sequence regions of the chosen bioagent identifying amplicon must be highly conserved among all <i>Bacillus </i>species while the variable region of the bioagent identifying amplicon is sufficiently variable such that the molecular masses of the amplification products of all species of <i>Bacillus </i>are distinguishable.</p>
    <p num="p-0051">Bioagent identifying amplicons amenable to molecular mass determination are either of a length, size or mass compatible with the particular mode of molecular mass determination or compatible with a means of providing a predictable fragmentation pattern in order to obtain predictable fragments of a length compatible with the particular mode of molecular mass determination. Such means of providing a predictable fragmentation pattern of an amplification product include, but are not limited to, cleavage with restriction enzymes or cleavage primers, for example.</p>
    <p num="p-0052">Identification of bioagents can be accomplished at different levels using intelligent primers suited to resolution of each individual level of identification. Broad range survey intelligent primers are designed with the objective of identifying a bioagent as a member of a particular division of bioagents. A bioagent division is defined as group of bioagents above the species level and includes but is not limited to: orders, families, classes, clades, genera or other such groupings of bioagents above the species level. As a non-limiting example, members of the <i>Bacillus/Clostridia </i>group or gamma-proteobacteria group may be identified as such by employing broad range survey intelligent primers such as primers which target 16S or 23S ribosomal RNA.</p>
    <p num="p-0053">In some embodiments, broad range survey intelligent primers are capable of identification of bioagents at the species level. One main advantage of the detection methods of the present invention is that the broad range survey intelligent primers need not be specific for a particular bacterial species, or even genus, such as <i>Bacillus </i>or <i>Streptomyces</i>. Instead, the primers recognize highly conserved regions across hundreds of bacterial species including, but not limited to, the species described herein. Thus, the same broad range survey intelligent primer pair can be used to identify any desired bacterium because it will bind to the conserved regions that flank a variable region specific to a single species, or common to several bacterial species, allowing unbiased nucleic acid amplification of the intervening sequence and determination of its molecular weight and base composition. For example, the 16S<sub></sub>9711062, 16S<sub></sub>12281310 and 16S<sub></sub>11001188 regions are 9899% conserved in about 900 species of bacteria (16S=16S rRNA, numbers indicate nucleotide position). In one embodiment of the present invention, primers used in the present method bind to one or more of these regions or portions thereof.</p>
    <p num="p-0054">Due to their overall conservation, the flanking rRNA primer sequences serve as good intelligent primer binding sites to amplify the nucleic acid region of interest for most, if not all, bacterial species. The intervening region between the sets of primers varies in length and/or composition, and thus provides a unique base composition signature. Examples of intelligent primers that amplify regions of the 16S and 23S rRNA are shown in <figref idrefs="DRAWINGS">FIGS. 1A1H</figref>. A typical primer amplified region in 16S rRNA is shown in <figref idrefs="DRAWINGS">FIG. 2</figref>. The arrows represent primers that bind to highly conserved regions which flank a variable region in 16S rRNA domain III. The amplified region is the stem-loop structure under 11001188. It is advantageous to design the broad range survey intelligent primers to minimize the number of primers required for the analysis, and to allow detection of multiple members of a bioagent division using a single pair of primers. The advantage of using broad range survey intelligent primers is that once a bioagent is broadly identified, the process of further identification at species and sub-species levels is facilitated by directing the choice of additional intelligent primers.</p>
    <p num="p-0055">Division-wide intelligent primers are designed with an objective of identifying a bioagent at the species level. As a non-limiting example, a <i>Bacillus anthracis, Bacillus cereus </i>and <i>Bacillus thuringiensis </i>can be distinguished from each other using division-wide intelligent primers. Division-wide intelligent primers are not always required for identification at the species level because broad range survey intelligent primers may provide sufficient identification resolution to accomplishing this identification objective.</p>
    <p num="p-0056">Drill-down intelligent primers are designed with an objective of identifying a sub-species characteristic of a bioagent. A sub-species characteristic is defined as a property imparted to a bioagent at the sub-species level of identification as a result of the presence or absence of a particular segment of nucleic acid. Such sub-species characteristics include, but are not limited to, strains, sub-types, pathogenicity markers such as antibiotic resistance genes, pathogenicity islands, toxin genes and virulence factors. Identification of such sub-species characteristics is often critical for determining proper clinical treatment of pathogen infections.</p>
    <p num="h-0011">Chemical Modifications of Intelligent Primers</p>
    <p num="p-0057">Ideally, intelligent primer hybridization sites are highly conserved in order to facilitate the hybridization of the primer. In cases where primer hybridization is less efficient due to lower levels of conservation of sequence, intelligent primers can be chemically modified to improve the efficiency of hybridization.</p>
    <p num="p-0058">For example, because any variation (due to codon wobble in the 3<sup>rd </sup>position) in these conserved regions among species is likely to occur in the third position of a DNA triplet, oligonucleotide primers can be designed such that the nucleotide corresponding to this position is a base which can bind to more than one nucleotide, referred to herein as a universal base. For example, under this wobble pairing, inosine (I) binds to U, C or A; guanine (G) binds to U or C, and uridine (U) binds to U or C. Other examples of universal bases include nitroindoles such as 5-nitroindole or 3-nitropyrrole (Loakes et al., <i>Nucleosides and Nucleotides, </i>1995, 14, 10011003), the degenerate nucleotides dP or dK (Hill et al.), an acyclic nucleoside analog containing 5-nitroindazole (Van Aerschot et al., <i>Nucleosides and Nucleotides, </i>1995, 14, 10531056) or the purine analog 1-(2-deoxy--D-ribofuranosyl)-imidazole-4-carboxamide (Sala et al., <i>Nucl. Acids Res., </i>1996, 24, 33023306).</p>
    <p num="p-0059">In another embodiment of the invention, to compensate for the somewhat weaker binding by the wobble base, the oligonucleotide primers are designed such that the first and second positions of each triplet are occupied by nucleotide analogs which bind with greater affinity than the unmodified nucleotide. Examples of these analogs include, but are not limited to, 2,6-diaminopurine which binds to thymine, propyne T which binds to adenine and propyne C and phenoxazines, including G-clamp, which binds to G. Propynylated pyrimidines are described in U.S. Pat. Nos. 5,645,985, 5,830,653 and 5,484,908, each of which is commonly owned and incorporated herein by reference in its entirety. Propynylated primers are claimed in U.S. Ser. No. 10/294,203 which is also commonly owned and incorporated herein by reference in entirety. Phenoxazines are described in U.S. Pat. Nos. 5,502,177, 5,763,588, and 6,005,096, each of which is incorporated herein by reference in its entirety. G-clamps are described in U.S. Pat. Nos. 6,007,992 and 6,028,183, each of which is incorporated herein by reference in its entirety.</p>
    <heading>D. Characterization of Bioagent Identifying Amplicons</heading> <p num="p-0060">A theoretically ideal bioagent detector would identify, quantify, and report the complete nucleic acid sequence of every bioagent that reached the sensor. The complete sequence of the nucleic acid component of a pathogen would provide all relevant information about the threat, including its identity and the presence of drug-resistance or pathogenicity markers. This ideal has not yet been achieved. However, the present invention provides a straightforward strategy for obtaining information with the same practical value based on analysis of bioagent identifying amplicons by molecular mass determination.</p>
    <p num="p-0061">In some cases, a molecular mass of a given bioagent identifying amplicon alone does not provide enough resolution to unambiguously identify a given bioagent. For example, the molecular mass of the bioagent identifying amplicon obtained using the intelligent primer pair 16S<sub></sub>971 would be 55622 Da for both <i>E. coli </i>and <i>Salmonella typhimurium</i>. However, if additional intelligent primers are employed to analyze additional bioagent identifying amplicons, a triangulation identification process is enabled. For example, the 16S<sub></sub>1100 intelligent primer pair yields molecular masses of 55009 and 55005 Da for <i>E. coli </i>and <i>Salmonella typhimurium</i>, respectively. Furthermore, the 23S<sub></sub>855 intelligent primer pair yields molecular masses of 42656 and 42698 Da for <i>E. coli </i>and <i>Salmonella typhimurium</i>, respectively. In this basic example, the second and third intelligent primer pairs provided the additional fingerprinting capability or resolution to distinguish between the two bioagents.</p>
    <p num="p-0062">In another embodiment, the triangulation identification process is pursued by measuring signals from a plurality of bioagent identifying amplicons selected within multiple core genes. This process is used to reduce false negative and false positive signals, and enable reconstruction of the origin of hybrid or otherwise engineered bioagents. In this process, after identification of multiple core genes, alignments are created from nucleic acid sequence databases. The alignments are then analyzed for regions of conservation and variation, and bioagent identifying amplicons are selected to distinguish bioagents based on specific genomic differences. For example, identification of the three part toxin genes typical of <i>B. anthracis </i>(Bowen et al., <i>J. Appl. Microbiol., </i>1999, 87, 270278) in the absence of the expected signatures from the <i>B. anthracis </i>genome would suggest a genetic engineering event.</p>
    <p num="p-0063">The triangulation identification process can be pursued by characterization of bioagent identifying amplicons in a massively parallel fashion using the polymerase chain reaction (PCR), such as multiplex PCR, and mass spectrometric (MS) methods. Sufficient quantities of nucleic acids should be present for detection of bioagents by MS. A wide variety of techniques for preparing large amounts of purified nucleic acids or fragments thereof are well known to those of skill in the art. PCR requires one or more pairs of oligonucleotide primers that bind to regions which flank the target sequence(s) to be amplified. These primers prime synthesis of a different strand of DNA, with synthesis occurring in the direction of one primer towards the other primer. The primers, DNA to be amplified, a thermostable DNA polymerase (e.g. Taq polymerase), the four deoxynucleotide triphosphates, and a buffer are combined to initiate DNA synthesis. The solution is denatured by heating, then cooled to allow annealing of newly added primer, followed by another round of DNA synthesis. This process is typically repeated for about 30 cycles, resulting in amplification of the target sequence.</p>
    <p num="p-0064">Although the use of PCR is suitable, other nucleic acid amplification techniques may also be used, including ligase chain reaction (LCR) and strand displacement amplification (SDA). The high-resolution MS technique allows separation of bioagent spectral lines from background spectral lines in highly cluttered environments.</p>
    <p num="p-0065">In another embodiment, the detection scheme for the PCR products generated from the bioagent(s) incorporates at least three features. First, the technique simultaneously detects and differentiates multiple (generally about 610) PCR products. Second, the technique provides a molecular mass that uniquely identifies the bioagent from the possible primer sites. Finally, the detection technique is rapid, allowing multiple PCR reactions to be run in parallel.</p>
    <heading>E. Mass Spectrometric Characterization of Bioagent Identifying Amplicons</heading> <p num="p-0066">Mass spectrometry (MS)-based detection of PCR products provides a means for determination of BCS which has several advantages. MS is intrinsically a parallel detection scheme without the need for radioactive or fluorescent labels, since every amplification product is identified by its molecular mass. The current state of the art in mass spectrometry is such that less than femtomole quantities of material can be readily analyzed to afford information about the molecular contents of the sample. An accurate assessment of the molecular mass of the material can be quickly obtained, irrespective of whether the molecular weight of the sample is several hundred, or in excess of one hundred thousand atomic mass units (amu) or Daltons. Intact molecular ions can be generated from amplification products using one of a variety of ionization techniques to convert the sample to gas phase. These ionization methods include, but are not limited to, electrospray ionization (ES), matrix-assisted laser desorption ionization (MALDI) and fast atom bombardment (FAB). For example, MALDI of nucleic acids, along with examples of matrices for use in MALDI of nucleic acids, are described in WO 98/54751 (Genetrace, Inc.).</p>
    <p num="p-0067">In some embodiments, large DNAs and RNAs, or large amplification products therefrom, can be digested with restriction endonucleases prior to ionization. Thus, for example, an amplification product that was 10 kDa could be digested with a series of restriction endonucleases to produce a panel of, for example, 100 Da fragments. Restriction endonucleases and their sites of action are well known to the skilled artisan. In this manner, mass spectrometry can be performed for the purposes of restriction mapping.</p>
    <p num="p-0068">Upon ionization, several peaks are observed from one sample due to the formation of ions with different charges. Averaging the multiple readings of molecular mass obtained from a single mass spectrum affords an estimate of molecular mass of the bioagent. Electrospray ionization mass spectrometry (ESI-MS) is particularly useful for very high molecular weight polymers such as proteins and nucleic acids having molecular weights greater than 10 kDa, since it yields a distribution of multiply-charged molecules of the sample without causing a significant amount of fragmentation.</p>
    <p num="p-0069">The mass detectors used in the methods of the present invention include, but are not limited to, Fourier transform ion cyclotron resonance mass spectrometry (FT-ICR-MS), ion trap, quadrupole, magnetic sector, time of flight (TOF), Q-TOF, and triple quadrupole.</p>
    <p num="p-0070">In general, the mass spectrometric techniques which can be used in the present invention include, but are not limited to, tandem mass spectrometry, infrared multiphoton dissociation and pyrolytic gas chromatography mass spectrometry (PGC-MS). In one embodiment of the invention, the bioagent detection system operates continually in bioagent detection mode using pyrolytic GC-MS without PCR for rapid detection of increases in biomass (for example, increases in fecal contamination of drinking water or of germ warfare agents). To achieve minimal latency, a continuous sample stream flows directly into the PGC-MS combustion chamber. When an increase in biomass is detected, a PCR process is automatically initiated. Bioagent presence produces elevated levels of large molecular fragments from, for example, about 1007,000 Da which are observed in the PGC-MS spectrum. The observed mass spectrum is compared to a threshold level and when levels of biomass are determined to exceed a predetermined threshold, the bioagent classification process described hereinabove (combining PCR and MS, such as FT-ICR MS) is initiated. Optionally, alarms or other processes (halting ventilation flow, physical isolation) are also initiated by this detected biomass level.</p>
    <p num="p-0071">The accurate measurement of molecular mass for large DNAs is limited by the adduction of cations from the PCR reaction to each strand, resolution of the isotopic peaks from natural abundance <sup>13</sup>C and <sup>15</sup>N isotopes, and assignment of the charge state for any ion. The cations are removed by in-line dialysis using a flow-through chip that brings the solution containing the PCR products into contact with a solution containing ammonium acetate in the presence of an electric field gradient orthogonal to the flow. The latter two problems are addressed by operating with a resolving power of &gt;100,000 and by incorporating isotopically depleted nucleotide triphosphates into the DNA. The resolving power of the instrument is also a consideration. At a resolving power of 10,000, the modeled signal from the [M-14H+]<sup>14</sup> charge state of an 84mer PCR product is poorly characterized and assignment of the charge state or exact mass is impossible. At a resolving power of 33,000, the peaks from the individual isotopic components are visible. At a resolving power of 100,000, the isotopic peaks are resolved to the baseline and assignment of the charge state for the ion is straightforward. The [<sup>13</sup>C, <sup>15</sup>N]-depleted triphosphates are obtained, for example, by growing microorganisms on depleted media and harvesting the nucleotides (Batey et al., <i>Nucl. Acids Res., </i>1992, 20, 45154523).</p>
    <p num="p-0072">While mass measurements of intact nucleic acid regions are believed to be adequate to determine most bioagents, tandem mass spectrometry (MS<sup>n</sup>) techniques may provide more definitive information pertaining to molecular identity or sequence. Tandem MS involves the coupled use of two or more stages of mass analysis where both the separation and detection steps are based on mass spectrometry. The first stage is used to select an ion or component of a sample from which further structural information is to be obtained. The selected ion is then fragmented using, e.g., blackbody irradiation, infrared multiphoton dissociation, or collisional activation. For example, ions generated by electrospray ionization (ESI) can be fragmented using IR multiphoton dissociation. This activation leads to dissociation of glycosidic bonds and the phosphate backbone, producing two series of fragment ions, called the w-series (having an intact 3 terminus and a 5 phosphate following internal cleavage) and the a-Base series(having an intact 5 terminus and a 3 furan).</p>
    <p num="p-0073">The second stage of mass analysis is then used to detect and measure the mass of these resulting fragments of product ions. Such ion selection followed by fragmentation routines can be performed multiple times so as to essentially completely dissect the molecular sequence of a sample.</p>
    <p num="p-0074">If there are two or more targets of similar molecular mass, or if a single amplification reaction results in a product which has the same mass as two or more bioagent reference standards, they can be distinguished by using mass-modifying tags. In this embodiment of the invention, a nucleotide analog or tag is incorporated during amplification (e.g., a 5-(trifluoromethyl)deoxythymidine triphosphate) which has a different molecular weight than the unmodified base so as to improve distinction of masses. Such tags are described in, for example, PCT WO97/33000, which is incorporated herein by reference in its entirety. This further limits the number of possible base compositions consistent with any mass. For example, 5-(trifluoromethyl)deoxythymidine triphosphate can be used in place of dTTP in a separate nucleic acid amplification reaction. Measurement of the mass shift between a conventional amplification product and the tagged product is used to quantitate the number of thymidine nucleotides in each of the single strands. Because the strands are complementary, the number of adenosine nucleotides in each strand is also determined.</p>
    <p num="p-0075">In another amplification reaction, the number of G and C residues in each strand is determined using, for example, the cytidine analog 5-methylcytosine (5-meC) or propyne C. The combination of the A/T reaction and G/C reaction, followed by molecular weight determination, provides a unique base composition. This method is summarized in <figref idrefs="DRAWINGS">FIG. 4</figref> and Table 1.</p>
    <p num="p-0076">
      <tables id="TABLE-US-00001" num="00001"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="63pt" align="left"> </colspec> <colspec colname="4" colwidth="28pt" align="left"> </colspec> <colspec colname="5" colwidth="28pt" align="left"> </colspec> <colspec colname="6" colwidth="28pt" align="left"> </colspec> <colspec colname="7" colwidth="28pt" align="left"> </colspec> <colspec colname="8" colwidth="28pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="8" rowsep="1" class="description-td" colspan="8">TABLE 1</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Total</td>
                <td class="description-td">Total</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Total</td>
                <td class="description-td">Base</td>
                <td class="description-td">Base</td>
                <td class="description-td">base</td>
                <td class="description-td">base</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Double</td>
                <td class="description-td">Single</td>
                <td class="description-td">mass</td>
                <td class="description-td">info</td>
                <td class="description-td">info</td>
                <td class="description-td">comp.</td>
                <td class="description-td">comp.</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">strand</td>
                <td class="description-td">strand</td>
                <td class="description-td">this</td>
                <td class="description-td">this</td>
                <td class="description-td">other</td>
                <td class="description-td">Top</td>
                <td class="description-td">Bottom</td>
              </tr> <tr class="description-tr"> <td class="description-td">Mass tag</td>
                <td class="description-td">sequence</td>
                <td class="description-td">Sequence</td>
                <td class="description-td">strand</td>
                <td class="description-td">strand</td>
                <td class="description-td">strand</td>
                <td class="description-td">strand</td>
                <td class="description-td">strand</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">T*mass</td>
                <td class="description-td">T*ACGT*ACGT</td>
                <td class="description-td">T*ACGT*ACGT*</td>
                <td class="description-td">3x</td>
                <td class="description-td">3T</td>
                <td class="description-td">3A</td>
                <td class="description-td">3T</td>
                <td class="description-td">3A</td>
              </tr> <tr class="description-tr"> <td class="description-td">(T*  T) = x</td>
                <td class="description-td">*</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">2A</td>
                <td class="description-td">2T</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AT*GCAT*GCA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">2C</td>
                <td class="description-td">2G</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">2G</td>
                <td class="description-td">2C</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">AT*GCAT*GCA</td>
                <td class="description-td">2x</td>
                <td class="description-td">2T</td>
                <td class="description-td">2A</td>
              </tr> <tr class="description-tr"> <td class="description-td">C*mass</td>
                <td class="description-td">TAC*GTAC*GT</td>
                <td class="description-td">TAC*GTAC*GT</td>
                <td class="description-td">2x</td>
                <td class="description-td">2C</td>
                <td class="description-td">2G</td>
              </tr> <tr class="description-tr"> <td class="description-td">(C*  C) = y</td>
                <td class="description-td">ATGC*ATGC*A</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">ATGC*ATGC*A</td>
                <td class="description-td">2x</td>
                <td class="description-td">2C</td>
                <td class="description-td">2G</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0077">The mass tag phosphorothioate A (A*) was used to distinguish a <i>Bacillus anthracis </i>cluster. The <i>B. anthracis </i>(A<sub>14</sub>G<sub>9</sub>Cl<sub>4</sub>T<sub>9</sub>) had an average MW of 14072.26, and the <i>B. anthracis</i>(A<sub>1</sub>A*<sub>13</sub>G<sub>9</sub>C<sub>14</sub>T<sub>9</sub>) had an average molecular weight of 14281.11 and the phosphorothioate A had an average molecular weight of +16.06 as determined by ESI-TOF MS. The deconvoluted spectra are shown in <figref idrefs="DRAWINGS">FIG. 5</figref>.</p>
    <p num="p-0078">In another example, assume the measured molecular masses of each strand are 30,000.115Da and 31,000.115 Da respectively, and the measured number of dT and dA residues are (30,28) and (28,30). If the molecular mass is accurate to 100 ppm, there are 7 possible combinations of dG+dC possible for each strand. However, if the measured molecular mass is accurate to 10 ppm, there are only 2 combinations of dG+dC, and at 1 ppm accuracy there is only one possible base composition for each strand.</p>
    <p num="p-0079">Signals from the mass spectrometer may be input to a maximum-likelihood detection and classification algorithm such as is widely used in radar signal processing. The detection processing uses matched filtering of BCS observed in mass-basecount space and allows for detection and subtraction of signatures from known, harmless organisms, and for detection of unknown bioagent threats. Comparison of newly observed bioagents to known bioagents is also possible, for estimation of threat level, by comparing their BCS to those of known organisms and to known forms of pathogenicity enhancement, such as insertion of antibiotic resistance genes or toxin genes.</p>
    <p num="p-0080">Processing may end with a Bayesian classifier using log likelihood ratios developed from the observed signals and average background levels. The program emphasizes performance predictions culminating in probability-of-detection versus probability-of-false-alarm plots for conditions involving complex backgrounds of naturally occurring organisms and environmental contaminants. Matched filters consist of a priori expectations of signal values given the set of primers used for each of the bioagents. A genomic sequence database (e.g. GenBank) is used to define the mass basecount matched filters. The database contains known threat agents and benign background organisms. The latter is used to estimate and subtract the signature produced by the background organisms. A maximum likelihood detection of known background organisms is implemented using matched filters and a running-sum estimate of the noise covariance. Background signal strengths are estimated and used along with the matched filters to form signatures which are then subtracted. the maximum likelihood process is applied to this cleaned up data in a similar manner employing matched filters for the organisms and a running-sum estimate of the noise-covariance for the cleaned up data.</p>
    <heading>F. Base Composition Signatures as Indices of Bioagent Identifying Amplicons</heading> <p num="p-0081">Although the molecular mass of amplification products obtained using intelligent primers provides a means for identification of bioagents, conversion of molecular mass data to a base composition signature is useful for certain analyses. As used herein, a base composition signature (BCS) is the exact base composition determined from the molecular mass of a bioagent identifying amplicon. In one embodiment, a BCS provides an index of a specific gene in a specific organism.</p>
    <p num="p-0082">Base compositions, like sequences, vary slightly from isolate to isolate within species. It is possible to manage this diversity by building base composition probability clouds around the composition constraints for each species. This permits identification of organisms in a fashion similar to sequence analysis. A pseudo four-dimensional plot can be used to visualize the concept of base composition probability clouds (<figref idrefs="DRAWINGS">FIG. 18</figref>). Optimal primer design requires optimal choice of bioagent identifying amplicons and maximizes the separation between the base composition signatures of individual bioagents. Areas where clouds overlap indicate regions that may result in a misclassification, a problem which is overcome by selecting primers that provide information from different bioagent identifying amplicons, ideally maximizing the separation of base compositions. Thus, one aspect of the utility of an analysis of base composition probability clouds is that it provides a means for screening primer sets in order to avoid potential misclassifications of BCS and bioagent identity. Another aspect of the utility of base composition probability clouds is that they provide a means for predicting the identity of a bioagent whose exact measured BCS was not previously observed and/or indexed in a BCS database due to evolutionary transitions in its nucleic acid sequence.</p>
    <p num="p-0083">It is important to note that, in contrast to probe-based techniques, mass spectrometry determination of base composition does not require prior knowledge of the composition in order to make the measurement, only to interpret the results. In this regard, the present invention provides bioagent classifying information similar to DNA sequencing and phylogenetic analysis at a level sufficient to detect and identify a given bioagent. Furthermore, the process of determination of a previously unknown BCS for a given bioagent (for example, in a case where sequence information is unavailable) has downstream utility by providing additional bioagent indexing information with which to populate BCS databases. The process of future bioagent identification is thus greatly improved as more BCS indexes become available in the BCS databases.</p>
    <p num="p-0084">Another embodiment of the present invention is a method of surveying bioagent samples that enables detection and identification of all bacteria for which sequence information is available using a set of twelve broad-range intelligent PCR primers. Six of the twelve primers are broad range survey primers herein defined as primers targeted to broad divisions of bacteria (for example, the <i>Bacillus</i>/Clostridia group or gamma-proteobacteria). The other six primers of the group of twelve primers are division-wide primers herein defined as primers which provide more focused coverage and higher resolution. This method enables identification of nearly 100% of known bacteria at the species level. A further example of this embodiment of the present invention is a method herein designated survey/drill-down wherein a subspecies characteristic for detected bioagents is obtained using additional primers. Examples of such a subspecies characteristic include but are not limited to: antibiotic resistance, pathogenicity island, virulence factor, strain type, sub-species type, and clade group. Using the survey/drill-down method, bioagent detection, confirmation and a subspecies characteristic can be provided within hours. Moreover, the survey/drill-down method can be focused to identify bioengineering events such as the insertion of a toxin gene into a bacterial species that does not normally make the toxin.</p>
    <heading>G. Fields of Application of the Present Invention</heading> <p num="p-0085">The present methods allow extremely rapid and accurate detection and identification of bioagents compared to existing methods. Furthermore, this rapid detection and identification is possible even when sample material is impure. The methods leverage ongoing biomedical research in virulence, pathogenicity, drug resistance and genome sequencing into a method which provides greatly improved sensitivity, specificity and reliability compared to existing methods, with lower rates of false positives. Thus, the methods are useful in a wide variety of fields, including, but not limited to, those fields discussed below.</p>
    <p num="p-0086">1. Forensic Investigations of Biowarfare Agents</p>
    <p num="p-0087">In other embodiments of the invention, the methods disclosed herein can be used for epidemiological and forensics investigations. As used herein, epidemiology refers to an investigative process which attempts to link the effects of exposure to harmful agents to disease or mortality. As used herein, forensics is the study of evidence discovered at a crime investigation or accident scene and which may be used in a court of law. Forensic science is any science used for the purposes of the law, and therefore provides impartial scientific evidence for use in the courts of law, and in a criminal investigation and trial. Forensic science is a multidisciplinary subject, drawing principally from chemistry and biology, but also from physics, geology, psychology and social science, for example.</p>
    <p num="p-0088">Epidemiological and forensic investigations of biowarfare- or bioterrorism-associated events can benefit from rapid and reliable methods of genetic analysis capable of characterizing a variety of genetic marker types. Examples of such genetic marker analyses include, but are not limited to: Multiple-Locus VNTR Analysis (MLVA), Multi-Locus Sequence Typing (MLST) and Single Nucleotide Polymorphism (SNP) analysis. These methods traditionally require independent PCR-based assays followed by electrophoresis on fluorescent sequencing platforms.</p>
    <p num="p-0089">In addition, epidemiologists, for example, can use the present methods to determine the geographic origin of a particular strain of a protist or fungus. For example, a particular strain of bacteria or virus may have a sequence difference that is associated with a particular area of a country or the world and identification of such a sequence difference can lead to the identification of the geographic origin and epidemiological tracking of the spread of the particular disease, disorder or condition associated with the detected protist or fungus. In addition, carriers of particular DNA or diseases, such as mammals, non-mammals, birds, insects, and plants, can be tracked by screening their mtDNA. Diseases, such as malaria, can be tracked by screening the mtDNA of commensals such as mosquitoes.</p>
    <p num="p-0090">In one embodiment the methods of the present invention are employed for identification of bioagent associated with an act of biowarfare, terrorism or criminal activity.</p>
    <p num="p-0091">Examples of bioagents that may be used in acts of biowarfare, or bioterrorism include, but are not limited to: viruses such as: Crimean-Congo hemorrhagic fever virus, Eastern Equine Encephalitis virus, Ebola virus, Equine Morbillivirus, Flexal virus, Guanarito virus, Hantaan or other Hanta viruses, Junin virus, Lassa fever virus, Machupo virus, Marburg virus, Omsk hemorrhagic fever virus, Rift Valley fever virus, Russian Spring-Summer encephalitis virus, Sabia virus, Tick-borne encephalitis complex viruses, Variola major virus (Smallpox virus), Venezuelan Equine Encephalitis virus, Coronavirus and Yellow fever virus; bacteria such as: <i>Bacillus anthracis, Brucella abortus, Brucella melitensis, Brucella suis, Burkholderia </i>(<i>Pseudomonas</i>) <i>mallei, Burkholderia </i>(<i>Pseudomonas</i>) <i>pseudomallei, Clostridium botulinum, Francisella tularensis, Yersinia pestis</i>, Rickettsiae, <i>Coxiella burnetii, Rickettsia prowazekii</i>, and <i>Rickettsia </i>rickettsii; fungi such as: <i>Coccidioides immitis</i>; and toxins such as: abrin, aflatoxins, Botulinum toxins, <i>Clostridium perfringens </i>epsilon toxin, conotoxins, diacetoxyscirpenol, ricin, saxitoxin, shigatoxin, Staphylococcal enterotoxins, Tetrodotoxin and T-2 toxin (see, for example, www.ehs.iastate.edu/publications/factsheets/bioterrorismlaws.pdf).</p>
    <p num="p-0092">As an example, a bioagent such as <i>Bacillus anthracis </i>is identified in a sample obtained from the site of an incident of biowarfare, terrorism or crime by employing intelligent primers to amplify a bioagent identifying amplicon from the bioagent, determining molecular mass (and the base composition signature if desired) of the amplified nucleic acid and matching the molecular mass (and base composition signature, if desired) with a molecular mass or base composition indexed to a bioagent contained in a database of reference base composition signatures.</p>
    <p num="p-0093">2. Epidemiologic Investigations by Genotyping of Bioagents</p>
    <p num="p-0094">In another embodiment, a bioagent is genotyped. Genotyping as used herein, refers to characterization of a nucleic acid representing a gene or a portion of a gene. As a nonlimiting example, genotyping is carried out to investigate the presence or absence of pathogenicity factors defined herein as a segment of nucleic acid which confers pathogenic properties upon a bioagent. Examples of pathogenicity factors include, but are not limited to: pathogenicity islands, virulence markers and toxin components such as: protective antigen, lethal factor and edema factor, all of which are found on the plasmid pX01 and the antiphagocytic capsule found on plasmid pX02 of <i>Bacillus anthracis</i>. Primers targeting nucleic acid regions suspected of containing pathogenicity factors are used to amplify the nucleic acid whose, molecular mass (and base composition signature if desired) is then determined and compared to molecular masses (and base composition signatures if desired) of known pathogenicity factors to identify the pathogenicity factor.</p>
    <p num="p-0095">An example of an advantage conferred by genotyping is that genetic engineering events are detected. Genetic engineering has been used in the past decade to alter the genes of biological weapon agents. Researchers in the USA, UK, Russia, Germany and other countries have introduced genes into hazardous bacteria that are likely to enhance the biowarfare possibilities of these microbes. Strains have been designed that can withstand antibiotics, are undetectable by traditional equipment, can overcome vaccines, or that cause unusual symptoms, thereby hampering diagnosis. In general, gene transfer is used to build more effective biological weapons, it could be used to broaden the military biological warfare spectrum, making it more difficult to fight and control biowarfare agents (www.sunshine-project.org/publications/pr/pr130700.html).</p>
    <p num="p-0096">The present invention is particularly well suited for the task of detecting genetic engineering events since reference databases can be populated with molecular mass data or associated base composition signatures for known pathogenic factors. Primers targeting nucleic acid regions suspected of containing pathogenicity factors are used to amplify the nucleic acid whose molecular mass and base composition signature is then determined and compared to molecular masses and base composition signatures of known pathogenicity factors. A match between a molecular mass and/or base composition signature determined for genetically engineered bioagent and the base composition of a pathogenicity factor provides a means for identifying the pathogenicity factor incorporated into the genetically engineered bioagent so that appropriate countermeasures may be carried out.</p>
    <p num="p-0097">In another embodiment, the present methods are used in forensic investigations which employ microbial geographic profiling information to track a known or suspected terrorist or criminal by obtaining bioagent samples from the site of incidence of an act of terrorism or crime, identifying the bioagent and correlating the identity of the bioagent with the likelihood that the bioagent is associated with the known or suspected terrorist or criminal. As used herein, microbial geographic profiling refers to the process of associating a particular genetic characteristic of a microbial bioagent with the geographic location in which it originates and is typically located.</p>
    <p num="p-0098">In some embodiments of the invention the travels of a known terrorist are tracked. The terrorist can be a member of a known terrorist organization, such as Al Qaeda, or can be a member of an unknown terrorist organization. Alternately, the terrorist can be a single entity operating independently of any organization. Known terrorists include those individuals who are known by or listed by particular governments, such as the United States, or departments or agencies within a government, such as the Federal Bureau of Investigation, the Central Intelligence Agency, the Department of Homeland Security, the State Department, Congress, the National Security Agency, and the like. Suspected terrorists include those individuals not known to be terrorists as described above, but who are likely to be or are suspected of supporting or engaging in terroristic acts.</p>
    <p num="p-0099">Travels of a known or suspected terrorist include all geographic movements of a particular individual terrorist. The travels of the individual terrorist may represent geographic movements of a terrorist organization of which the individual terrorist is a member. Geographic movements or travels of a terrorist include, foot travel, air travel, automobile travel, train travel, bus travel, or any other means of movement from one location in the world to another location in the world. Tracking of such travels or geographic movements includes determining at least one geographic location that the terrorist has occupied, other than the geographic location the terrorist is in when a sample is taken from the terrorist. For example, the travels of a terrorist that is currently located, for instance, in the United Kingdom, at the time a sample is obtained can be tracked to another location such as, for instance, Afghanistan. Thus, the travels of such a terrorist can be tracked from Afghanistan to the United Kingdom.</p>
    <p num="p-0100">Geographic locations of interest to be tracked include every country or state in the world including, but not limited to, the Mideast (e.g., Afghanistan, Iraq, Iran, Syria, Jordan, Palestinian-occupied territory, Lebanon, Kuwait, Yemen, United Arab Emirates, and Saudi Arabia), Northern Africa (e.g., Egypt, Sudan, Somalia, Tunisia, Morocco, and Libya), Asia (e.g., North Korea, China, Pakistan, India, Philippines, and Indonesia), as well as other countries, states, or territories known or suspected of sponsoring or harboring terrorists.</p>
    <p num="p-0101">A nucleic acid from a bioagent obtained from a sample associated with the terrorist is obtained. Samples associated with terrorists can be obtained knowingly from the terrorist or can be obtained covertly from the terrorist. For example, the terrorist can be detained and the sample or samples obtained from the terrorist with the terrorist's knowledge. Alternately, the sample or samples can be obtained covertly from the terrorists without the terrorist's knowledge. The sample from the terrorist can be associated with the terrorist in a direct manner or indirect manner. For example, samples associated with a terrorist can be obtained directly from the terrorist. Such samples include, but are not limited to, a sample of bodily fluid of a sample of tissue from the terrorist. Examples of bodily fluids include, but are not limited to, blood, sweat, tears, urine, saliva, and the like. Examples of bodily tissues include, but are not limited to, hair, skin, bone, and the like. In addition, types of bodily tissues include products derived therefrom such as feces, mucous, and the like. Thus, a food product containing a bioagent that has been consumed by a terrorist can be detected in samples of feces or saliva. Other samples associated with a terrorist include, but are not limited to, a sample of clothing from the terrorist, a sample of environmental material from the terrorist, a sample from a pet travelling with the terrorist, or a sample from a luggage traveling with the terrorist. Examples of environmental material include, but are not limited to, dirt, water, plant material, animal material, and the like that may be present somewhere on the terrorist or with pets, luggage, or companions traveling therewith.</p>
    <p num="p-0102">In one example of the present embodiment, a terrorist from a camp in the Sahara Desert commits an act of terrorism in a European country, leaving forensic evidence at the scene of the terrorist act. Samples of the forensic evidence are analyzed by the methods of the present invention. <i>Bacillus mojavensis </i>is identified in the sample by employing intelligent primers to amplify nucleic acid from the bacterium, determining the base composition signature of the amplified nucleic acid and matching the base composition with a base composition indexed to <i>Bacillus mojavensis </i>contained in a database of reference base composition signatures. This analysis indicates that <i>Bacillus mojavensis </i>is present in the soil sample associated with the terrorist. Further genotyping of the bioagent using primers targeting the pTA-like plasmid rep and mob genes then provides an indication that the strain of <i>Bacillus mojavensis </i>is <i>B. mojavensis </i>IM-E-3, a strain found in the Sahara Desert (Mason, M. P. et al. <i>FEMS Microbiol. Ecol. </i>2002, 42, 235241), thus indicating that the terrorist is associated with a soil sample originating from the Sahara Desert. These analyses are optionally repeated for additional bioagents identified in the forensic sample to ascertain other geographic locations to which the known or suspected terrorist has traveled.</p>
    <p num="p-0103">While the present invention has been described with specificity in accordance with certain of its embodiments, the following examples serve only to illustrate the invention and are not intended to limit the same.</p>
    <heading>EXAMPLES</heading> <heading>Example 1</heading> <heading>Nucleic Acid Isolation and PCR</heading> <p num="p-0104">In one embodiment, nucleic acid is isolated from the organisms and amplified by PCR using standard methods prior to BCS determination by mass spectrometry. Nucleic acid is isolated, for example, by detergent lysis of bacterial cells, centrifugation and ethanol precipitation. Nucleic acid isolation methods are described in, for example, <i>Current Protocols in Molecular Biology </i>(Ausubel et al.) and <i>Molecular Cloning; A Laboratory Manual </i>(Sambrook et al.). The nucleic acid is then amplified using standard methodology, such as PCR, with primers which bind to conserved regions of the nucleic acid which contain an intervening variable sequence as described below.</p>
    <p num="p-0105">General Genomic DNA Sample Prep Protocol: Raw samples are filtered using Supor-200 0.2 m membrane syringe filters (VWR International). Samples are transferred to 1.5 ml eppendorf tubes pre-filled with 0.45 g of 0.7 mm Zirconia beads followed by the addition of 350 l of ATL buffer (Qiagen, Valencia, Calif.). The samples are subjected to bead beating for 10 minutes at a frequency of 19 l/s in a Retsch Vibration Mill (Retsch). After centrifugation, samples are transferred to an S-block plate (Qiagen) and DNA isolation is completed with a BioRobot 8000 nucleic acid isolation robot (Qiagen).</p>
    <p num="p-0106">Swab Sample Protocol. Allegiance S/P brand culture swabs and collection/transport system are used to collect samples. After drying, swabs are placed in 17100 mm culture tubes (VWR International) and the genomic nucleic acid isolation is carried out automatically with a Qiagen Mdx robot and the Qiagen QIAamp DNA Blood BioRobot Mdx genomic preparation kit (Qiagen, Valencia, Calif.).</p>
    <heading>Example 2</heading> <heading>Mass Spectrometry</heading> <p num="p-0107">FTICR Instrumentation: The FTICR instrument is based on a 7 tesla actively shielded superconducting magnet and modified Bruker Daltonics Apex II 70e ion optics and vacuum chamber. The spectrometer is interfaced to a LEAP PAL autosampler and a custom fluidics control system for high throughput screening applications. Samples are analyzed directly from 96-well or 384-well microtiter plates at a rate of about 1 sample/minute. The Bruker data-acquisition platform is supplemented with a lab-built ancillary NT datastation which controls the autosampler and contains an arbitrary waveform generator capable of generating complex rf-excite waveforms (frequency sweeps, filtered noise, stored waveform inverse Fourier transform (SWIFT), etc.) for sophisticated tandem MS experiments. For oligonucleotides in the 2030-mer regime typical performance characteristics include mass resolving power in excess of 100,000 (FWHM), low ppm mass measurement errors, and an operable m/z range between 50 and 5000 m/z.</p>
    <p num="p-0108">Modified ESI Source: In sample-limited analyses, analyte solutions are delivered at 150 nL/minute to a 30 mm i.d. fused-silica ESI emitter mounted on a 3-D micromanipulator. The ESI ion optics consists of a heated metal capillary, an rf-only hexapole, a skimmer cone, and an auxiliary gate electrode. The 6.2 cm rf-only hexapole is comprised of 1 mm diameter rods and is operated at a voltage of 380 Vpp at a frequency of 5 MHz. A lab-built electro-mechanical shutter can be employed to prevent the electrospray plume from entering the inlet capillary unless triggered to the open position via a TTL pulse from the data station. When in the closed position, a stable electrospray plume is maintained between the ESI emitter and the face of the shutter. The back face of the shutter arm contains an elastomeric seal that can be positioned to form a vacuum seal with the inlet capillary. When the seal is removed, a 1 mm gap between the shutter blade and the capillary inlet allows constant pressure in the external ion reservoir regardless of whether the shutter is in the open or closed position. When the shutter is triggered, a time slice of ions is allowed to enter the inlet capillary and is subsequently accumulated in the external ion reservoir. The rapid response time of the ion shutter (&lt;25 ms) provides reproducible, user defined intervals during which ions can be injected into and accumulated in the external ion reservoir.</p>
    <p num="p-0109">Apparatus for Infrared Multiphoton Dissociation: A 25 watt CW CO<sub>2 </sub>laser operating at 10.6 m has been interfaced to the spectrometer to enable infrared multiphoton dissociation (IRMPD) for oligonucleotide sequencing and other tandem MS applications. An aluminum optical bench is positioned approximately 1.5 m from the actively shielded superconducting magnet such that the laser beam is aligned with the central axis of the magnet. Using standard IR-compatible mirrors and kinematic mirror mounts, the unfocused 3 mm laser beam is aligned to traverse directly through the 3.5 mm holes in the trapping electrodes of the FTICR trapped ion cell and longitudinally traverse the hexapole region of the external ion guide finally impinging on the skimmer cone. This scheme allows IRMPD to be conducted in an m/z selective manner in the trapped ion cell (e.g. following a SWIFT isolation of the species of interest), or in a broadband mode in the high pressure region of the external ion reservoir where collisions with neutral molecules stabilize IRMPD-generated metastable fragment ions resulting in increased fragment ion yield and sequence coverage.</p>
    <heading>Example 3</heading> <heading>Identification of Bioagents</heading> <p num="p-0110">Table 2 shows a small cross section of a database of calculated molecular masses for over 9 primer sets and approximately 30 organisms. The primer sets were derived from rRNA alignment. Examples of regions from rRNA consensus alignments are shown in <figref idrefs="DRAWINGS">FIGS. 1A1C</figref>. Lines with arrows are examples of regions to which intelligent primer pairs for PCR are designed. The primer pairs are &gt;95% conserved in the bacterial sequence database (currently over 10,000 organisms). The intervening regions are variable in length and/or composition, thus providing the base composition signature (BCS) for each organism. Primer pairs were chosen so the total length of the amplified region is less than about 8090 nucleotides. The label for each primer pair represents the starting and ending base number of the amplified region on the consensus diagram.</p>
    <p num="p-0111">Included in the short bacterial database cross-section in Table 2 are many well known pathogens/biowarfare agents (shown in bold/red typeface) such as <i>Bacillus anthracis </i>or <i>Yersinia pestis </i>as well as some of the bacterial organisms found commonly in the natural environment such as <i>Streptomyces</i>. Even closely related organisms can be distinguished from each other by the appropriate choice of primers. For instance, two low G+C organisms, <i>Bacillus anthracis </i>and <i>Staph aureus</i>, can be distinguished from each other by using the primer pair defined by 16S<sub></sub>1337 or 23S<sub></sub>855 (M of 4 Da).</p>
    <p num="p-0112">
      <tables id="TABLE-US-00002" num="00002"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="406pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 2</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Cross Section Of A Database Of Calculated Molecular Masses<sup>1</sup> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="91pt" align="left"> </colspec> <colspec colname="1" colwidth="315pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Primer Regions</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <colspec colname="7" colwidth="35pt" align="center"> </colspec> <colspec colname="8" colwidth="35pt" align="center"> </colspec> <colspec colname="9" colwidth="35pt" align="center"> </colspec> <colspec colname="10" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Bug Name</td>
                <td class="description-td">16S_971</td>
                <td class="description-td">16S_1100</td>
                <td class="description-td">16S_1337</td>
                <td class="description-td">16S_1294</td>
                <td class="description-td">16S_1228</td>
                <td class="description-td">23S_1021</td>
                <td class="description-td">23S_855</td>
                <td class="description-td">23S_193</td>
                <td class="description-td">23S_115</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="8" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="9" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="10" colwidth="35pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <i>Acinetobacter calcoaceticus</i>
                </td>
                <td class="description-td">55619.1</td>
                <td class="description-td">55004</td>
                <td class="description-td">28446.7</td>
                <td class="description-td">35854.9</td>
                <td class="description-td">51295.4</td>
                <td class="description-td">30299</td>
                <td class="description-td">42654</td>
                <td class="description-td">39557.5</td>
                <td class="description-td">54999</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00001.png"> <img id="CUSTOM-CHARACTER-00001" he="2.46mm" wi="19.05mm" file="US07226739-20070605-P00001.TIF" alt="Figure US07226739-20070605-P00001" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00001.png" class="patent-full-image" width="76" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55005</b>
                </td>
                <td class="description-td">
                  <b>54388</b>
                </td>
                <td class="description-td">
                  <b>28448</b>
                </td>
                <td class="description-td">
                  <b>35238</b>
                </td>
                <td class="description-td">
                  <b>51296</b>
                </td>
                <td class="description-td">
                  <b>30295</b>
                </td>
                <td class="description-td">
                  <b>42651</b>
                </td>
                <td class="description-td">
                  <b>39560</b>
                </td>
                <td class="description-td">
                  <b>56850</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus cereus</i>
                </td>
                <td class="description-td">55622.1</td>
                <td class="description-td">54387.9</td>
                <td class="description-td">28447.6</td>
                <td class="description-td">35854.9</td>
                <td class="description-td">51296.4</td>
                <td class="description-td">30295</td>
                <td class="description-td">42651</td>
                <td class="description-td">39560.5</td>
                <td class="description-td">56850.3</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bordetella bronchiseptica</i>
                </td>
                <td class="description-td">56857.3</td>
                <td class="description-td">51300.4</td>
                <td class="description-td">28446.7</td>
                <td class="description-td">35857.9</td>
                <td class="description-td">51307.4</td>
                <td class="description-td">30299</td>
                <td class="description-td">42653</td>
                <td class="description-td">39559.5</td>
                <td class="description-td">51920.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Borrelia burgdorferi</i>
                </td>
                <td class="description-td">56231.2</td>
                <td class="description-td">55621.1</td>
                <td class="description-td">28440.7</td>
                <td class="description-td">35852.9</td>
                <td class="description-td">51295.4</td>
                <td class="description-td">30297</td>
                <td class="description-td">42029.9</td>
                <td class="description-td">38941.4</td>
                <td class="description-td">52524.6</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00002.png"> <img id="CUSTOM-CHARACTER-00002" he="2.46mm" wi="17.61mm" file="US07226739-20070605-P00002.TIF" alt="Figure US07226739-20070605-P00002" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00002.png" class="patent-full-image" width="70" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>58098</b>
                </td>
                <td class="description-td">
                  <b>55011</b>
                </td>
                <td class="description-td">
                  <b>28448</b>
                </td>
                <td class="description-td">
                  <b>35854</b>
                </td>
                <td class="description-td">
                  <b>50683</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Campylobacter jejuni</i>
                </td>
                <td class="description-td">58088.5</td>
                <td class="description-td">54386.9</td>
                <td class="description-td">29061.8</td>
                <td class="description-td">35856.9</td>
                <td class="description-td">50674.3</td>
                <td class="description-td">30294</td>
                <td class="description-td">42032.9</td>
                <td class="description-td">39558.5</td>
                <td class="description-td">45732.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00003.png"> <img id="CUSTOM-CHARACTER-00003" he="2.46mm" wi="24.72mm" file="US07226739-20070605-P00003.TIF" alt="Figure US07226739-20070605-P00003" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00003.png" class="patent-full-image" width="98" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55000</b>
                </td>
                <td class="description-td">
                  <b>55007</b>
                </td>
                <td class="description-td">
                  <b>29063</b>
                </td>
                <td class="description-td">
                  <b>35855</b>
                </td>
                <td class="description-td">
                  <b>50676</b>
                </td>
                <td class="description-td">
                  <b>30295</b>
                </td>
                <td class="description-td">
                  <b>42036</b>
                </td>
                <td class="description-td">
                  <b>38941</b>
                </td>
                <td class="description-td">
                  <b>56230</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00004.png"> <img id="CUSTOM-CHARACTER-00004" he="2.46mm" wi="23.28mm" file="US07226739-20070605-P00004.TIF" alt="Figure US07226739-20070605-P00004" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00004.png" class="patent-full-image" width="93" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55006</b>
                </td>
                <td class="description-td">
                  <b>53767</b>
                </td>
                <td class="description-td">
                  <b>28445</b>
                </td>
                <td class="description-td">
                  <b>35855</b>
                </td>
                <td class="description-td">
                  <b>51291</b>
                </td>
                <td class="description-td">
                  <b>30300</b>
                </td>
                <td class="description-td">
                  <b>42656</b>
                </td>
                <td class="description-td">
                  <b>39562</b>
                </td>
                <td class="description-td">
                  <b>54999</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Clostridium difficile</i>
                </td>
                <td class="description-td">56855.3</td>
                <td class="description-td">54386.9</td>
                <td class="description-td">28444.7</td>
                <td class="description-td">35853.9</td>
                <td class="description-td">51296.4</td>
                <td class="description-td">30294</td>
                <td class="description-td">41417.8</td>
                <td class="description-td">39556.5</td>
                <td class="description-td">55612.2</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Enterococcus faecalis</i>
                </td>
                <td class="description-td">55620.1</td>
                <td class="description-td">54387.9</td>
                <td class="description-td">28447.6</td>
                <td class="description-td">35858.9</td>
                <td class="description-td">51296.4</td>
                <td class="description-td">30297</td>
                <td class="description-td">42652</td>
                <td class="description-td">39559.5</td>
                <td class="description-td">56849.3</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00005.png"> <img id="CUSTOM-CHARACTER-00005" he="2.46mm" wi="17.27mm" file="US07226739-20070605-P00005.TIF" alt="Figure US07226739-20070605-P00005" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00005.png" class="patent-full-image" width="69" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55622</b>
                </td>
                <td class="description-td">
                  <b>55009</b>
                </td>
                <td class="description-td">
                  <b>28445</b>
                </td>
                <td class="description-td">
                  <b>35857</b>
                </td>
                <td class="description-td">
                  <b>51301</b>
                </td>
                <td class="description-td">
                  <b>30301</b>
                </td>
                <td class="description-td">
                  <b>42656</b>
                </td>
                <td class="description-td">
                  <b>39562</b>
                </td>
                <td class="description-td">
                  <b>54999</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00006.png"> <img id="CUSTOM-CHARACTER-00006" he="2.46mm" wi="22.61mm" file="US07226739-20070605-P00006.TIF" alt="Figure US07226739-20070605-P00006" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00006.png" class="patent-full-image" width="90" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>53769</b>
                </td>
                <td class="description-td">
                  <b>54385</b>
                </td>
                <td class="description-td">
                  <b>28445</b>
                </td>
                <td class="description-td">
                  <b>35856</b>
                </td>
                <td class="description-td">
                  <b>51298</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Haemophilus influenzae</i>
                </td>
                <td class="description-td">55620.1</td>
                <td class="description-td">55006</td>
                <td class="description-td">28444.7</td>
                <td class="description-td">35855.9</td>
                <td class="description-td">51298.4</td>
                <td class="description-td">30298</td>
                <td class="description-td">42656</td>
                <td class="description-td">39560.5</td>
                <td class="description-td">55613.1</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Klebsiella pneumoniae</i>
                </td>
                <td class="description-td">55622.1</td>
                <td class="description-td">55008</td>
                <td class="description-td">28442.7</td>
                <td class="description-td">35856.9</td>
                <td class="description-td">51297.4</td>
                <td class="description-td">30300</td>
                <td class="description-td">42655</td>
                <td class="description-td">39562.5</td>
                <td class="description-td">55000</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00007.png"> <img id="CUSTOM-CHARACTER-00007" he="2.46mm" wi="25.40mm" file="US07226739-20070605-P00007.TIF" alt="Figure US07226739-20070605-P00007" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00007.png" class="patent-full-image" width="101" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55618</b>
                </td>
                <td class="description-td">
                  <b>55626</b>
                </td>
                <td class="description-td">
                  <b>28446</b>
                </td>
                <td class="description-td">
                  <b>35857</b>
                </td>
                <td class="description-td">
                  <b>51303</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycobacterium avium</i>
                </td>
                <td class="description-td">54390.9</td>
                <td class="description-td">55631.1</td>
                <td class="description-td">29064.8</td>
                <td class="description-td">35858.9</td>
                <td class="description-td">51915.5</td>
                <td class="description-td">30298</td>
                <td class="description-td">42656</td>
                <td class="description-td">38942.4</td>
                <td class="description-td">56241.2</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycobacterium leprae</i>
                </td>
                <td class="description-td">54389.9</td>
                <td class="description-td">55629.1</td>
                <td class="description-td">29064.8</td>
                <td class="description-td">35860.9</td>
                <td class="description-td">51917.5</td>
                <td class="description-td">30298</td>
                <td class="description-td">42656</td>
                <td class="description-td">39559.5</td>
                <td class="description-td">56240.2</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycobacterium tuberculosis</i>
                </td>
                <td class="description-td">54390.9</td>
                <td class="description-td">55629.1</td>
                <td class="description-td">29064.8</td>
                <td class="description-td">35860.9</td>
                <td class="description-td">51301.4</td>
                <td class="description-td">30299</td>
                <td class="description-td">42656</td>
                <td class="description-td">39560.5</td>
                <td class="description-td">56243.2</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycoplasma genitalium</i>
                </td>
                <td class="description-td">53143.7</td>
                <td class="description-td">45115.4</td>
                <td class="description-td">29061.8</td>
                <td class="description-td">35854.9</td>
                <td class="description-td">50671.3</td>
                <td class="description-td">30294</td>
                <td class="description-td">43264.1</td>
                <td class="description-td">39558.5</td>
                <td class="description-td">56842.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycoplasma pneumoniae</i>
                </td>
                <td class="description-td">53143.7</td>
                <td class="description-td">45118.4</td>
                <td class="description-td">29061.8</td>
                <td class="description-td">35854.9</td>
                <td class="description-td">50673.3</td>
                <td class="description-td">30294</td>
                <td class="description-td">43264.1</td>
                <td class="description-td">39559.5</td>
                <td class="description-td">56843.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Neisseria gonorrhoeae</i>
                </td>
                <td class="description-td">55627.1</td>
                <td class="description-td">54389.9</td>
                <td class="description-td">28445.7</td>
                <td class="description-td">35855.9</td>
                <td class="description-td">51302.4</td>
                <td class="description-td">30300</td>
                <td class="description-td">42649</td>
                <td class="description-td">39561.5</td>
                <td class="description-td">55000</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00008.png"> <img id="CUSTOM-CHARACTER-00008" he="2.46mm" wi="26.42mm" file="US07226739-20070605-P00008.TIF" alt="Figure US07226739-20070605-P00008" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00008.png" class="patent-full-image" width="105" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55623</b>
                </td>
                <td class="description-td">
                  <b>55010</b>
                </td>
                <td class="description-td">
                  <b>28443</b>
                </td>
                <td class="description-td">
                  <b>35858</b>
                </td>
                <td class="description-td">
                  <b>51301</b>
                </td>
                <td class="description-td">
                  <b>30298</b>
                </td>
                <td class="description-td">
                  <b>43272</b>
                </td>
                <td class="description-td">
                  <b>39558</b>
                </td>
                <td class="description-td">
                  <b>55619</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00009.png"> <img id="CUSTOM-CHARACTER-00009" he="2.46mm" wi="22.18mm" file="US07226739-20070605-P00009.TIF" alt="Figure US07226739-20070605-P00009" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00009.png" class="patent-full-image" width="88" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>58093</b>
                </td>
                <td class="description-td">
                  <b>55621</b>
                </td>
                <td class="description-td">
                  <b>28448</b>
                </td>
                <td class="description-td">
                  <b>35853</b>
                </td>
                <td class="description-td">
                  <b>50677</b>
                </td>
                <td class="description-td">
                  <b>30293</b>
                </td>
                <td class="description-td">
                  <b>42650</b>
                </td>
                <td class="description-td">
                  <b>39559</b>
                </td>
                <td class="description-td">
                  <b>53139</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00010.png"> <img id="CUSTOM-CHARACTER-00010" he="2.46mm" wi="19.73mm" file="US07226739-20070605-P00010.TIF" alt="Figure US07226739-20070605-P00010" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00010.png" class="patent-full-image" width="78" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>58094</b>
                </td>
                <td class="description-td">
                  <b>55623</b>
                </td>
                <td class="description-td">
                  <b>28448</b>
                </td>
                <td class="description-td">
                  <b>35853</b>
                </td>
                <td class="description-td">
                  <b>50679</b>
                </td>
                <td class="description-td">
                  <b>30293</b>
                </td>
                <td class="description-td">
                  <b>42648</b>
                </td>
                <td class="description-td">
                  <b>39559</b>
                </td>
                <td class="description-td">
                  <b>53755</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00011.png"> <img id="CUSTOM-CHARACTER-00011" he="2.46mm" wi="25.74mm" file="US07226739-20070605-P00011.TIF" alt="Figure US07226739-20070605-P00011" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00011.png" class="patent-full-image" width="102" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55622</b>
                </td>
                <td class="description-td">
                  <b>55005</b>
                </td>
                <td class="description-td">
                  <b>28445</b>
                </td>
                <td class="description-td">
                  <b>35857</b>
                </td>
                <td class="description-td">
                  <b>51301</b>
                </td>
                <td class="description-td">
                  <b>30301</b>
                </td>
                <td class="description-td">
                  <b>42658</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00012.png"> <img id="CUSTOM-CHARACTER-00012" he="2.46mm" wi="21.17mm" file="US07226739-20070605-P00012.TIF" alt="Figure US07226739-20070605-P00012" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00012.png" class="patent-full-image" width="84" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55623</b>
                </td>
                <td class="description-td">
                  <b>55009</b>
                </td>
                <td class="description-td">
                  <b>28444</b>
                </td>
                <td class="description-td">
                  <b>35857</b>
                </td>
                <td class="description-td">
                  <b>51301</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Staphylococcus aureus</i>
                </td>
                <td class="description-td">56854.3</td>
                <td class="description-td">54386.9</td>
                <td class="description-td">28443.7</td>
                <td class="description-td">35852.9</td>
                <td class="description-td">51294.4</td>
                <td class="description-td">30298</td>
                <td class="description-td">42655</td>
                <td class="description-td">39559.5</td>
                <td class="description-td">57466.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Streptomyces</i>
                </td>
                <td class="description-td">54389.9</td>
                <td class="description-td">59341.6</td>
                <td class="description-td">29063.8</td>
                <td class="description-td">35858.9</td>
                <td class="description-td">51300.4</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">39563.5</td>
                <td class="description-td">56864.3</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Treponema pallidum</i>
                </td>
                <td class="description-td">56245.2</td>
                <td class="description-td">55631.1</td>
                <td class="description-td">28445.7</td>
                <td class="description-td">35851.9</td>
                <td class="description-td">51297.4</td>
                <td class="description-td">30299</td>
                <td class="description-td">42034.9</td>
                <td class="description-td">38939.4</td>
                <td class="description-td">57473.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00013.png"> <img id="CUSTOM-CHARACTER-00013" he="2.46mm" wi="15.83mm" file="US07226739-20070605-P00013.TIF" alt="Figure US07226739-20070605-P00013" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00013.png" class="patent-full-image" width="63" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55625</b>
                </td>
                <td class="description-td">
                  <b>55626</b>
                </td>
                <td class="description-td">
                  <b>28443</b>
                </td>
                <td class="description-td">
                  <b>35857</b>
                </td>
                <td class="description-td">
                  <b>52536</b>
                </td>
                <td class="description-td">
                  <b>29063</b>
                </td>
                <td class="description-td">
                  <b>30303</b>
                </td>
                <td class="description-td">
                  <b>35241</b>
                </td>
                <td class="description-td">
                  <b>50675</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Vibrio parahaemolyticus</i>
                </td>
                <td class="description-td">54384.9</td>
                <td class="description-td">55626.1</td>
                <td class="description-td">28444.7</td>
                <td class="description-td">34620.7</td>
                <td class="description-td">50064.2</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00014.png"> <img id="CUSTOM-CHARACTER-00014" he="2.46mm" wi="14.48mm" file="US07226739-20070605-P00014.TIF" alt="Figure US07226739-20070605-P00014" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7226739B2/US07226739-20070605-P00014.png" class="patent-full-image" width="57" height="9"> </a> </div>
                </td>
                <td class="description-td">
                  <b>55620</b>
                </td>
                <td class="description-td">
                  <b>55626</b>
                </td>
                <td class="description-td">
                  <b>28443</b>
                </td>
                <td class="description-td">
                  <b>35857</b>
                </td>
                <td class="description-td">
                  <b>51299</b>
                </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="left" id="FOO-00001" class="description-td" colspan="10"> <sup>1</sup>Molecular mass distribution of PCR amplified regions for a selection of organisms (rows) across various primer pairs (columns). Pathogens are shown in <b>bold</b>. Empty cells indicate presently incomplete or missing data.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0113"> <figref idrefs="DRAWINGS">FIG. 6</figref> shows the use of ESI-FT-ICR MS for measurement of exact mass. The spectra from 46mer PCR products originating at position 1337 of the 16S rRNA from <i>S. aureus </i>(upper) and <i>B. anthracis </i>(lower) are shown. These data are from the region of the spectrum containing signals from the [M8H+]<sup>8</sup> charge states of the respective 53 strands. The two strands differ by two (ATCG) substitutions, and have measured masses of 14206.396 and 14208.373+0.010 Da, respectively. The possible base compositions derived from the masses of the forward and reverse strands for the <i>B. anthracis </i>products are listed in Table 3.</p>
    <p num="p-0114"> <tables id="TABLE-US-00003" num="00003"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 3</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Possible base composition for <i>B. anthracis </i>products</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="84pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Calc. Mass</td> <td class="description-td">Error</td> <td class="description-td">Base Comp.</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.2935</td> <td class="description-td">0.079520</td> <td class="description-td">A1 G17 C10 T18</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3160</td> <td class="description-td">0.056980</td> <td class="description-td">A1 G20 C15 T10</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3386</td> <td class="description-td">0.034440</td> <td class="description-td">A1 G23 C20 T2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3074</td> <td class="description-td">0.065560</td> <td class="description-td">A6 G11 C3 T26</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3300</td> <td class="description-td">0.043020</td> <td class="description-td">A6 G14 C8 T18</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3525</td> <td class="description-td">0.020480</td> <td class="description-td">A6 G17 C13 T10</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3751</td> <td class="description-td">0.002060</td> <td class="description-td">A6 G20 C18 T2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3439</td> <td class="description-td">0.029060</td> <td class="description-td">A11 G8 C1 T26</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.3665</td> <td class="description-td">0.006520</td> <td class="description-td">A11 G11 C6 T18</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> <b>14208.3890</b> </td> <td class="description-td"> <b>0.016020</b> </td> <td class="description-td"> <b>A11 G14 C11 T10</b> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.4116</td> <td class="description-td">0.038560</td> <td class="description-td">A11 G17 C16 T2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.4030</td> <td class="description-td">0.029980</td> <td class="description-td">A16 G8 C4 T18</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.4255</td> <td class="description-td">0.052520</td> <td class="description-td">A16 G11 C9 T10</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.4481</td> <td class="description-td">0.075060</td> <td class="description-td">A16 G14 C14 T2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.4395</td> <td class="description-td">0.066480</td> <td class="description-td">A21 G5 C2 T18</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14208.4620</td> <td class="description-td">0.089020</td> <td class="description-td">A21 G8 C7 T10</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.2624</td> <td class="description-td">0.080600</td> <td class="description-td">A0 G14 C13 T19</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.2849</td> <td class="description-td">0.058060</td> <td class="description-td">A0 G17 C18 T11</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3075</td> <td class="description-td">0.035520</td> <td class="description-td">A0 G20 C23 T3</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.2538</td> <td class="description-td">0.089180</td> <td class="description-td">A5 G5 C1 T35</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.2764</td> <td class="description-td">0.066640</td> <td class="description-td">A5 G8 C6 T27</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.2989</td> <td class="description-td">0.044100</td> <td class="description-td">A5 G11 C11 T19</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3214</td> <td class="description-td">0.021560</td> <td class="description-td">A5 G14 C16 T11</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3440</td> <td class="description-td">0.000980</td> <td class="description-td">A5 G17 C21 T3</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3129</td> <td class="description-td">0.030140</td> <td class="description-td">A10 G5 C4 T27</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3354</td> <td class="description-td">0.007600</td> <td class="description-td">A10 G8 C9 T19</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> <b>14079.3579</b> </td> <td class="description-td"> <b>0.014940</b> </td> <td class="description-td"> <b>A10 G11 C14 T11</b> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3805</td> <td class="description-td">0.037480</td> <td class="description-td">A10 G14 C19 T3</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3494</td> <td class="description-td">0.006360</td> <td class="description-td">A15 G2 C2 T27</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3719</td> <td class="description-td">0.028900</td> <td class="description-td">A15 G5 C7 T19</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.3944</td> <td class="description-td">0.051440</td> <td class="description-td">A15 G8 C12 T11</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.4170</td> <td class="description-td">0.073980</td> <td class="description-td">A15 G11 C17 T3</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.4084</td> <td class="description-td">0.065400</td> <td class="description-td">A20 G2 C5 T19</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">14079.4309</td> <td class="description-td">0.087940</td> <td class="description-td">A20 G5 C10 T13</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
Among the 16 compositions for the forward strand and the 18 compositions for the reverse strand that were calculated, only one pair (shown in bold) are complementary, corresponding to the actual base compositions of the <i>B. anthracis </i>PCR products.
</p>
    <heading>Example 4</heading> <heading>BCS of Region from <i>Bacillus anthracis </i>and <i>Bacillus cereus </i> </heading> <p num="p-0115">A conserved <i>Bacillus </i>region from <i>B. anthracis </i>(A<sub>14</sub>G<sub>9</sub>C<sub>14</sub>T<sub>9</sub>) and <i>B. cereus</i>(A<sub>15</sub>G<sub>9</sub>C<sub>13</sub>T<sub>9</sub>) having a C to A base change was synthesized and subjected to ESI-TOF MS. The results are shown in <figref idrefs="DRAWINGS">FIG. 7</figref> in which the two regions are clearly distinguished using the method of the present invention (MW=14072.26 vs. 14096.29).</p>
    <heading>Example 5</heading> <heading>Identification of Additional Bioagents</heading> <p num="p-0116">In other examples of the present invention, the pathogen <i>Vibrio cholera </i>can be distinguished from <i>Vibrio parahemolyticus </i>with M&gt;600 Da using one of three 16S primer sets shown in Table 2 (16S<sub></sub>971, 16S<sub></sub>1228 or 16s<sup>1294</sup>) as shown in Table 4. The two mycoplasma species in the list (<i>M. genitalium </i>and <i>M. pneumoniae</i>) can also be distinguished from each other, as can the three mycobacteriae. While the direct mass measurements of amplified products can identify and distinguish a large number of organisms, measurement of the base composition signature provides dramatically enhanced resolving power for closely related organisms. In cases such as <i>Bacillus anthracis </i>and <i>Bacillus cereus </i>that are virtually indistinguishable from each other based solely on mass differences, compositional analysis or fragmentation patterns are used to resolve the differences. The single base difference between the two organisms yields different fragmentation patterns, and despite the presence of the ambiguous/unidentified base N at position 20 in <i>B. anthracis</i>, the two organisms can be identified.</p>
    <p num="p-0117">Tables 4ab show examples of primer pairs from Table 1 which distinguish pathogens from background.</p>
    <p num="p-0118">
      <tables id="TABLE-US-00004" num="00004"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="77pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" rowsep="1" class="description-td" colspan="5">TABLE 4a</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Organism name</td>
                <td class="description-td">23S_855</td>
                <td class="description-td">16S_1337</td>
                <td class="description-td">23S_1021</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Bacillus anthracis</i>
                </td>
                <td class="description-td">42650.98</td>
                <td class="description-td">28447.65</td>
                <td class="description-td">30294.98</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>Staphylococcus aureus</i>
                </td>
                <td class="description-td">42654.97</td>
                <td class="description-td">28443.67</td>
                <td class="description-td">30297.96</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0119">
      <tables id="TABLE-US-00005" num="00005"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="4" rowsep="1" class="description-td" colspan="4">TABLE 4b</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Organism name</td>
                <td class="description-td">16S_971</td>
                <td class="description-td">16S_1294</td>
                <td class="description-td">16S_1228</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <i>Vibrio cholerae</i>
                </td>
                <td class="description-td">55625.09</td>
                <td class="description-td">35856.87</td>
                <td class="description-td">52535.59</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Vibrio parahaemolyticus</i>
                </td>
                <td class="description-td">54384.91</td>
                <td class="description-td">34620.67</td>
                <td class="description-td">50064.19</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0120">Table 5 shows the expected molecular weight and base composition of region 16S<sub></sub>11001188 in <i>Mycobacterium avium </i>and <i>Streptomyces </i>sp.</p>
    <p num="p-0121">
      <tables id="TABLE-US-00006" num="00006"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="left"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <colspec colname="5" colwidth="49pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="5" rowsep="1" class="description-td" colspan="5">TABLE 5</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Molecular</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Region</td>
                <td class="description-td">Organism name</td>
                <td class="description-td">Length</td>
                <td class="description-td">weight</td>
                <td class="description-td">Base comp.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">16S<sub></sub> </td>
                <td class="description-td">
                  <i>Mycobacterium</i>
                </td>
                <td class="description-td">82</td>
                <td class="description-td">25624.1728</td>
                <td class="description-td">A<sub>16</sub>G<sub>32</sub>C<sub>18</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">11001188</td>
                <td class="description-td">
                  <i>avium</i>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">16S<sub></sub> </td>
                <td class="description-td"> <i>Streptomyces </i>sp.</td>
                <td class="description-td">96</td>
                <td class="description-td">29904.871</td>
                <td class="description-td">A<sub>17</sub>G<sub>38</sub>C<sub>27</sub>T<sub>14</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">11001188</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0122">Table 6 shows base composition (single strand) results for 16S<sub></sub>11001188 primer amplification reactions different species of bacteria. Species which are repeated in the table (e.g., <i>Clostridium botulinum</i>) are different strains which have different base compositions in the 16S<sub></sub>11001188 region.</p>
    <p num="p-0123">
      <tables id="TABLE-US-00007" num="00007"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="84pt" align="left"> </colspec> <colspec colname="2" colwidth="49pt" align="left"> </colspec> <colspec colname="3" colwidth="84pt" align="left"> </colspec> <colspec colname="4" colwidth="49pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="4" rowsep="1" class="description-td" colspan="4">TABLE 6</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Organism name</td>
                <td class="description-td">Base comp.</td>
                <td class="description-td">Organism name</td>
                <td class="description-td">Base comp.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <i>Mycobacterium avium</i>
                </td>
                <td class="description-td">A<sub>16</sub>G<sub>32</sub>C<sub>18</sub>T<sub>16</sub> </td>
                <td class="description-td">
                  <i>Vibrio cholerae</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>30</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>Streptomyces </i>sp.</td>
                <td class="description-td">A<sub>17</sub>G<sub>38</sub>C<sub>27</sub>T<sub>14</sub> </td>
                <td class="description-td">
                  <i>Aeromonas hydrophila</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>23</sub>G<sub>31</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Ureaplasma urealyticum</i>
                </td>
                <td class="description-td">A<sub>18</sub>G<sub>30</sub>C<sub>17</sub>T<sub>17</sub> </td>
                <td class="description-td">
                  <i>Aeromonas salmonicida</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>23</sub>G<sub>31</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>Streptomyces </i>sp.</td>
                <td class="description-td">A<sub>19</sub>G<sub>36</sub>C<sub>24</sub>T<sub>18</sub> </td>
                <td class="description-td">
                  <i>Mycoplasma genitalium</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>19</sub>C<sub>12</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycobacterium leprae</i>
                </td>
                <td class="description-td">A<sub>20</sub>G<sub>32</sub>C<sub>22</sub>T<sub>16</sub> </td>
                <td class="description-td">
                  <i>Clostridium botulinum</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>25</sub>C<sub>18</sub>T<sub>20</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>M. tuberculosis</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>20</sub>G<sub>33</sub>C<sub>21</sub>T<sub>16</sub> </td>
                <td class="description-td">
                  <i>Bordetella bronchiseptica</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>26</sub>C<sub>19</sub>T<sub>14</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Nocardia asteroides</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>20</sub>G<sub>33</sub>C<sub>21</sub>T<sub>16</sub> </td>
                <td class="description-td">
                  <i>Francisella tularensis</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>26</sub>C<sub>19</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Fusobacterium necroforum</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>26</sub>C<sub>22</sub>T<sub>18</sub> </td>
                <td class="description-td">
                  <i>Bacillus anthracis</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>26</sub>C<sub>20</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Listeria monocytogenes</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>27</sub>C<sub>19</sub>T<sub>19</sub> </td>
                <td class="description-td">
                  <i>Campylobacter jejuni</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>26</sub>C<sub>20</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Clostridium botulinum</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>27</sub>C<sub>19</sub>T<sub>21</sub> </td>
                <td class="description-td">
                  <i>Staphylococcus aureus</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>26</sub>C<sub>20</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Neisseria gonorrhoeae</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>28</sub>C<sub>21</sub>T<sub>18</sub> </td>
                <td class="description-td">
                  <i>Helicobacter pylori</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>26</sub>C<sub>20</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bartonella quintana</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>30</sub>C<sub>22</sub>T<sub>16</sub> </td>
                <td class="description-td">
                  <i>Helicobacter pylori</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>26</sub>C<sub>21</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Enterococcus faecalis</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>27</sub>C<sub>20</sub>T<sub>19</sub> </td>
                <td class="description-td">
                  <i>Moraxella catarrhalis</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>26</sub>C<sub>23</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus megaterium</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>28</sub>C<sub>20</sub>T<sub>18</sub> </td>
                <td class="description-td"> <i>Haemophilus influenzae </i>Rd</td>
                <td class="description-td">A<sub>24</sub>G<sub>28</sub>C<sub>20</sub>T<sub>17</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus subtilis</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>28</sub>C<sub>21</sub>T<sub>17</sub> </td>
                <td class="description-td">
                  <i>Chlamydia trachomatis</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>28</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Pseudomonas aeruginosa</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>29</sub>C<sub>23</sub>T<sub>15</sub> </td>
                <td class="description-td">
                  <i>Chlamydophila pneumoniae</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>28</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Legionella pneumophila</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>32</sub>C<sub>20</sub>T<sub>16</sub> </td>
                <td class="description-td"> <i>C. pneumonia </i>AR39</td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>28</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycoplasma pneumoniae</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>20</sub>C<sub>14</sub>T<sub>16</sub> </td>
                <td class="description-td">
                  <i>Pseudomonas putida</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>29</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Clostridium botulinum</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>26</sub>C<sub>20</sub>T<sub>19</sub> </td>
                <td class="description-td">
                  <i>Proteus vulgaris</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>30</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Enterococcus faecium</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>26</sub>C<sub>21</sub>T<sub>18</sub> </td>
                <td class="description-td">
                  <i>Yersinia pestis</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>30</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Acinetobacter calcoaceti</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>26</sub>C<sub>21</sub>T<sub>19</sub> </td>
                <td class="description-td">
                  <i>Yersinia pseudotuberculos</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>30</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Leptospira borgpeterseni</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>23</sub>G<sub>26</sub>C<sub>24</sub>T<sub>15</sub> </td>
                <td class="description-td">
                  <i>Clostridium botulinum</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>24</sub>C<sub>18</sub>T<sub>21</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Leptospira interrogans</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>23</sub>G<sub>26</sub>C<sub>24</sub>T<sub>15</sub> </td>
                <td class="description-td">
                  <i>Clostridium tetani</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>25</sub>C<sub>18</sub>T<sub>20</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Clostridium perfringens</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>27</sub>C<sub>19</sub>T<sub>19</sub> </td>
                <td class="description-td">
                  <i>Francisella tularensis</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>25</sub>C<sub>19</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus anthracis</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>23</sub>G<sub>27</sub>C<sub>20</sub>T<sub>18</sub> </td>
                <td class="description-td">
                  <i>Acinetobacter calcoacetic</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>26</sub>C<sub>20</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus cereus</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>23</sub>G<sub>27</sub>C<sub>20</sub>T<sub>18</sub> </td>
                <td class="description-td">
                  <i>Bacteriodes fragilis</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>27</sub>C<sub>16</sub>T<sub>22</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus thuringiensis</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>23</sub>G<sub>27</sub>C<sub>20</sub>T<sub>18</sub> </td>
                <td class="description-td">
                  <i>Chlamydophila psittaci</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>27</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Aeromonas hydrophila</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>29</sub>C<sub>21</sub>T<sub>16</sub> </td>
                <td class="description-td">
                  <i>Borrelia burgdorferi</i>
                </td>
                <td class="description-td">A<sub>25</sub>G<sub>29</sub>C<sub>17</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Escherichia coli</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>29</sub>C<sub>21</sub>T<sub>16</sub> </td>
                <td class="description-td">
                  <i>Streptobacillus monilifor</i>
                </td>
                <td class="description-td">A<sub>26</sub>G<sub>26</sub>C<sub>20</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Pseudomonas putida</i>
                </td>
                <td class="description-td">A<sub>23</sub>G<sub>29</sub>C<sub>21</sub>T<sub>17</sub> </td>
                <td class="description-td">
                  <i>Rickettsia prowazekii</i>
                </td>
                <td class="description-td">A<sub>26</sub>G<sub>28</sub>C<sub>18</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Escherichia coli</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>23</sub>G<sub>29</sub>C<sub>22</sub>T<sub>15</sub> </td>
                <td class="description-td">
                  <i>Rickettsia rickettsii</i>
                </td>
                <td class="description-td">A<sub>26</sub>G<sub>28</sub>C<sub>20</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Shigella dysenteriae</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>23</sub>G<sub>29</sub>C<sub>22</sub>T<sub>15</sub> </td>
                <td class="description-td">
                  <i>Mycoplasma mycoides</i>
                </td>
                <td class="description-td">A<sub>28</sub>G<sub>23</sub>C<sub>16</sub>T<sub>20</sub> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0124">The same organism having different base compositions are different strains. Groups of organisms which are highlighted or in italics have the same base compositions in the amplified region. Some of these organisms can be distinguished using multiple primers. For example, <i>Bacillus anthracis </i>can be distinguished from <i>Bacillus cereus </i>and <i>Bacillus thuringiensis </i>using the primer 16S<sub></sub>9711062 (Table 7). Other primer pairs which produce unique base composition signatures are shown in Table 7 (bold). Clusters containing very similar threat and ubiquitous non-threat organisms (e.g. anthracis cluster) are distinguished at high resolution with focused sets of primer pairs. The known biowarfare agents in Table 6 are <i>Bacillus anthracis, Yersinia pestis, Francisella tularensis </i>and <i>Rickettsia prowazekii</i>.</p>
    <p num="p-0125">
      <tables id="TABLE-US-00008" num="00008"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="49pt" align="left"> </colspec> <colspec colname="3" colwidth="49pt" align="left"> </colspec> <colspec colname="4" colwidth="49pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="4" rowsep="1" class="description-td" colspan="4">TABLE 7</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">16S<sub></sub> </td>
                <td class="description-td">16S<sub></sub> </td>
                <td class="description-td">16S<sub></sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Organism</td>
                <td class="description-td">9711062</td>
                <td class="description-td">12281310</td>
                <td class="description-td">11001188</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <i>Aeromonas hydrophila</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>29</sub>C<sub>22</sub>T<sub>20</sub> </td>
                <td class="description-td">A<sub>22</sub>G<sub>27</sub>C<sub>21</sub>T<sub>13</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>31</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Aeromonas</i>
                </td>
                <td class="description-td">A<sub>21</sub>G<sub>29</sub>C<sub>22</sub>T<sub>20</sub> </td>
                <td class="description-td">A<sub>22</sub>G<sub>27</sub>C<sub>21</sub>T<sub>13</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>31</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>salmonicida</i>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus anthracis</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>21</sub>G<sub>27</sub>C<sub>22</sub>T<sub>22</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>22</sub>C<sub>19</sub>T<sub>18</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>27</sub>C<sub>20</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus cereus</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>27</sub>C<sub>21</sub>T<sub>22</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>22</sub>C<sub>19</sub>T<sub>18</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>27</sub>C<sub>20</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Bacillus</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>27</sub>C<sub>21</sub>T<sub>22</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>22</sub>C<sub>19</sub>T<sub>18</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>27</sub>C<sub>20</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>thuringiensis</i>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Chlamydia</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>22</sub>G<sub>26</sub>C<sub>20</sub>T<sub>23</sub> </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>23</sub>C<sub>19</sub>T<sub>16</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>28</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>trachomatis</i>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Chlamydia</i>
                </td>
                <td class="description-td">A<sub>26</sub>G<sub>23</sub>C<sub>20</sub>T<sub>22</sub> </td>
                <td class="description-td">A<sub>26</sub>G<sub>22</sub>C<sub>16</sub>T<sub>18</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>28</sub>C<sub>21</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>pneumoniae </i>AR39</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Leptospira</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>26</sub>C<sub>20</sub>T<sub>21</sub> </td>
                <td class="description-td">A<sub>22</sub>G<sub>25</sub>C<sub>21</sub>T<sub>15</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>26</sub>C<sub>24</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>borgpetersenii</i>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Leptospira interrogans</i>
                </td>
                <td class="description-td">A<sub>22</sub>G<sub>26</sub>C<sub>20</sub>T<sub>21</sub> </td>
                <td class="description-td">A<sub>22</sub>G<sub>25</sub>C<sub>21</sub>T<sub>15</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>26</sub>C<sub>24</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycoplasma</i>
                </td>
                <td class="description-td">A<sub>28</sub>G<sub>23</sub>C<sub>15</sub>T<sub>22</sub> </td>
                <td class="description-td"> <b>A</b> <sub>30</sub>G<sub>18</sub>C<sub>15</sub>T<sub>19</sub> </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>19</sub>C<sub>12</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>genitalium</i>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Mycoplasma</i>
                </td>
                <td class="description-td">A<sub>28</sub>G<sub>23</sub>C<sub>15</sub>T<sub>22</sub> </td>
                <td class="description-td"> <b>A</b> <sub>27</sub>G<sub>19</sub>C<sub>16</sub>T<sub>20</sub> </td>
                <td class="description-td"> <b>A</b> <sub>23</sub>G<sub>20</sub>C<sub>14</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>pneumoniae</i>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Escherichia coli</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>22</sub>G<sub>28</sub>C<sub>20</sub>T<sub>22</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>25</sub>C<sub>21</sub>T<sub>13</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>29</sub>C<sub>22</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Shigella dysenteriae</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>22</sub>G<sub>28</sub>C<sub>21</sub>T<sub>21</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>25</sub>C<sub>21</sub>T<sub>13</sub> </td>
                <td class="description-td">A<sub>23</sub>G<sub>29</sub>C<sub>22</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Proteus vulgaris</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>23</sub>G<sub>26</sub>C<sub>22</sub>T<sub>21</sub> </td>
                <td class="description-td"> <b>A</b> <sub>26</sub>G<sub>24</sub>C<sub>19</sub>T<sub>14</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>30</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Yersinia pestis</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>25</sub>C<sub>21</sub>T<sub>22</sub> </td>
                <td class="description-td">A<sub>25</sub>G<sub>24</sub>C<sub>20</sub>T<sub>14</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>30</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Yersinia</i>
                </td>
                <td class="description-td">A<sub>24</sub>G<sub>25</sub>C<sub>21</sub>T<sub>22</sub> </td>
                <td class="description-td">A<sub>25</sub>G<sub>24</sub>C<sub>20</sub>T<sub>14</sub> </td>
                <td class="description-td">A<sub>24</sub>G<sub>30</sub>C<sub>21</sub>T<sub>15</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>pseudotuberculosis</i>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Francisella tularensis</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>20</sub>G<sub>25</sub>C<sub>21</sub>T<sub>23</sub> </td>
                <td class="description-td"> <b>A</b> <sub>23</sub>G<sub>26</sub>C<sub>17</sub>T<sub>17</sub> </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>26</sub>C<sub>19</sub>T<sub>19</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Rickettsia prowazekii</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>21</sub>G<sub>26</sub>C<sub>24</sub>T<sub>25</sub> </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>23</sub>C<sub>16</sub>T<sub>19</sub> </td>
                <td class="description-td"> <b>A</b> <sub>26</sub>G<sub>28</sub>C<sub>18</sub>T<sub>18</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>Rickettsia rickettsii</i>
                </td>
                <td class="description-td"> <b>A</b> <sub>21</sub>G<sub>26</sub>C<sub>25</sub>T<sub>24</sub> </td>
                <td class="description-td"> <b>A</b> <sub>24</sub>G<sub>24</sub>C<sub>17</sub>T<sub>17</sub> </td>
                <td class="description-td"> <b>A</b> <sub>26</sub>G<sub>28</sub>C<sub>20</sub>T<sub>16</sub> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0126">The sequence of <i>B. anthracis </i>and <i>B. cereus </i>in region 16S<sub></sub>971 is shown below. Shown in bold is the single base difference between the two species which can be detected using the methods of the present invention. <i>B. anthracis </i>has an ambiguous base at position 20.</p>
    <p num="h-0027"> <i>B.anthracis</i> <sub></sub>16S<sub></sub>971</p>
    <p num="h-0028">GCGAAGAACCUUACCAGGUNUUGACAUCCUCUGACAACCCUAGAGAUAGGGCU UCUCCUUCGGGAGCAGAGUGACAGGUGGUGCAUGGUU (SEQ ID NO:1)</p>
    <p num="h-0029">B.cereus<sub></sub>16S<sub></sub>971</p>
    <p num="h-0030">GCGAAGAACCUUACCAGGUCUUGACAUCCUCUGAAAACCCUAGAGAUAGGGCU UCUCCUUCGGGAGCAGAGUGACAGGUGGUGCAUGGUU (SEQ ID NO:2)</p>
    <heading>Example 6</heading> <heading>ESI-TOF MS of sspE 56-mer Plus Calibrant</heading> <p num="p-0127">The mass measurement accuracy that can be obtained using an internal mass standard in the ESI-MS study of PCR products is shown in <figref idrefs="DRAWINGS">FIG. 8</figref>. The mass standard was a 20-mer phosphorothioate oligonucleotide added to a solution containing a 56-mer PCR product from the <i>B. anthracis </i>spore coat protein sspE. The mass of the expected PCR product distinguishes <i>B. anthracis </i>from other species of <i>Bacillus </i>such as <i>B. thuringiensis </i>and <i>B. cereus. </i> </p>
    <heading>Example 7</heading> <heading> <i>B. antiracis </i>ESI-TOF Synthetic 16S<sub></sub>1228 Duplex</heading> <p num="p-0128">An ESI-TOF MS spectrum was obtained from an aqueous solution containing 5 M each of synthetic analogs of the expected forward and reverse PCR products from the nucleotide 1228 region of the <i>B. anthracis </i>16S rRNA gene. The results (<figref idrefs="DRAWINGS">FIG. 9</figref>) show that the molecular weights of the forward and reverse strands can be accurately determined and easily distinguish the two strands. The [M21H<sup>+</sup>]<sup>21</sup> and [M20H<sup>+</sup>]<sup>20</sup> charge states are shown.</p>
    <heading>Example 8</heading> <heading>ESI-FTICR-MS of Synthetic <i>B. anthracis </i>16S<sub></sub>1337 46 Base Pair Duplex</heading> <p num="p-0129">An ESI-FTICR-MS spectrum was obtained from an aqueous solution containing 5 M each of synthetic analogs of the expected forward and reverse PCR products from the nucleotide 1337 region of the <i>B. anthracis </i>16S rRNA gene. The results (<figref idrefs="DRAWINGS">FIG. 10</figref>) show that the molecular weights of the strands can be distinguished by this method. The [M16H<sup>+</sup>]<sup>16</sup> through [M10H<sup>+</sup>]<sup>10</sup> charge states are shown. The insert highlights the resolution that can be realized on the FTICR-MS instrument, which allows the charge state of the ion to be determined from the mass difference between peaks differing by a single 13C substitution.</p>
    <heading>Example 9</heading> <heading>ESI-TOF MS of 56-mer Oligonucleotide from saspB Gene of <i>B. anthracis </i>with Internal Mass Standard</heading> <p num="p-0130">ESI-TOF MS spectra were obtained on a synthetic 56-mer oligonucleotide (5 M) from the saspB gene of <i>B. anthracis </i>containing an internal mass standard at an ESI of 1.7 L/min as a function of sample consumption. The results (<figref idrefs="DRAWINGS">FIG. 11</figref>) show that the signal to noise is improved as more scans are summed, and that the standard and the product are visible after only 100 scans.</p>
    <heading>Example 10</heading> <heading>ESI-TOF MS of an Internal Standard with Tributylammonium (TBA)-trifluoroacetate (TFA) Buffer</heading> <p num="p-0131">An ESI-TOF-MS spectrum of a 20-mer phosphorothioate mass standard was obtained following addition of 5 mM TBA-TFA buffer to the solution. This buffer strips charge from the oligonucleotide and shifts the most abundant charge state from [M8H<sup>+</sup>]<sup>8</sup> to [M3H<sup>+</sup>]<sup>3</sup> (<figref idrefs="DRAWINGS">FIG. 12</figref>).</p>
    <heading>Example 11</heading> <heading>Master Database Comparison</heading> <p num="p-0132">The molecular masses obtained through Examples 110 are compared to molecular masses of known bioagents stored in a master database to obtain a high probability matching molecular mass.</p>
    <heading>Example 12</heading> <heading>Master Data Base Interrogation over the Internet</heading> <p num="p-0133">The same procedure as in Example 11 is followed except that the local computer did not store the Master database. The Master database is interrogated over an internet connection, searching for a molecular mass match.</p>
    <heading>Example 13</heading> <heading>Master Database Updating</heading> <p num="p-0134">The same procedure as in example 11 is followed except the local computer is connected to the internet and has the ability to store a master database locally. The local computer system periodically, or at the user's discretion, interrogates the Master database, synchronizing the local master database with the global Master database. This provides the current molecular mass information to both the local database as well as to the global Master database. This further provides more of a globalized knowledge base.</p>
    <heading>Example 14</heading> <heading>Global Database Updating</heading> <p num="p-0135">The same procedure as in example 13 is followed except there are numerous such local stations throughout the world. The synchronization of each database adds to the diversity of information and diversity of the molecular masses of known bioagents.</p>
    <heading>Example 15</heading> <heading>Genotyping of Diverse Strains of <i>Bacillis anthracis </i> </heading> <p num="p-0136">In accordance with the present invention, the present methods were employed for genotyping 24 globally diverse isolates of <i>B. anthracis </i>using both MLVA and SNP analyses.</p>
    <p num="p-0137">For SNP analysis, bioagent identifying amplicons less than 100 bp long were designed around single or double SNPs from the protective antigen gene (PAG). Amplification products were obtained for the bioagent identifying amplicons. As shown in <figref idrefs="DRAWINGS">FIG. 18</figref>, mass spectral analyses of amplification products of 77 base pair bioagent identifying amplicons containing PAG01 SNP loci clearly indicate the capability of using the molecular mass for distinguish the identity of the SNP.</p>
    <p num="p-0138">For MLVA analysis, amplification products of bioagent identifying amplicons containing portions of pX01, pX02, vrrB2, CG3, Bavntr12, and Bavntr35 loci were examined. Results are shown in Table 8 and indicate that the present methods are capable of distinguishing the molecular masses and base compositions of the MLVA markers at the six different loci.</p>
    <p num="p-0139">
      <tables id="TABLE-US-00009" num="00009"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 8</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">MLVA VNTR loci information and complete base composition</td>
              </tr> <tr class="description-tr"> <td class="description-td">data for observed <i>B. anthracis </i>alleles as detected by ES-FTICR-MS</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="49pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="49pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="49pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Number</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">ESI-FTCR-MS</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">of</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Derived Base</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Observed</td>
                <td class="description-td">Size of Allele</td>
                <td class="description-td">Repeat</td>
                <td class="description-td">Compositions</td>
              </tr> <tr class="description-tr"> <td class="description-td">MLVA Marker</td>
                <td class="description-td">Alleles</td>
                <td class="description-td">(base pairs)</td>
                <td class="description-td">Structure</td>
                <td class="description-td">(A:G:C:T)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">BaVNTR12</td>
                <td class="description-td">1</td>
                <td class="description-td">112</td>
                <td class="description-td">AT</td>
                <td class="description-td">39:20:19:35</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">114</td>
                <td class="description-td"> </td>
                <td class="description-td">40:20:19:36</td>
              </tr> <tr class="description-tr"> <td class="description-td">BaVNTR35</td>
                <td class="description-td">1</td>
                <td class="description-td">103</td>
                <td class="description-td"> </td>
                <td class="description-td">28:13:19:43</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">109</td>
                <td class="description-td"> </td>
                <td class="description-td">29:15:19:46</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">3</td>
                <td class="description-td">115</td>
                <td class="description-td"> </td>
                <td class="description-td">30:17:19:49</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">4</td>
                <td class="description-td">121</td>
                <td class="description-td"> </td>
                <td class="description-td">31:19:19:52</td>
              </tr> <tr class="description-tr"> <td class="description-td">pX01</td>
                <td class="description-td">1</td>
                <td class="description-td">119</td>
                <td class="description-td">AAT</td>
                <td class="description-td">51:14:15:40</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">122</td>
                <td class="description-td"> </td>
                <td class="description-td">53:14:15:41</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">3</td>
                <td class="description-td">125</td>
                <td class="description-td"> </td>
                <td class="description-td">55:14:15:42</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">4</td>
                <td class="description-td">128</td>
                <td class="description-td"> </td>
                <td class="description-td">57:14:15:43</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">5</td>
                <td class="description-td">131</td>
                <td class="description-td"> </td>
                <td class="description-td">59:14:15:44</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">6</td>
                <td class="description-td">134</td>
                <td class="description-td"> </td>
                <td class="description-td">61:14:15:45</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">7</td>
                <td class="description-td">143</td>
                <td class="description-td"> </td>
                <td class="description-td">67:14:15:48</td>
              </tr> <tr class="description-tr"> <td class="description-td">pX02</td>
                <td class="description-td">1</td>
                <td class="description-td">135</td>
                <td class="description-td">AT</td>
                <td class="description-td">37:23:27:48</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">137</td>
                <td class="description-td"> </td>
                <td class="description-td">38:23:27:49</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">3</td>
                <td class="description-td">139</td>
                <td class="description-td"> </td>
                <td class="description-td">39:23:27:50</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">4</td>
                <td class="description-td">141</td>
                <td class="description-td"> </td>
                <td class="description-td">40:23:27:51</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">5</td>
                <td class="description-td">143</td>
                <td class="description-td"> </td>
                <td class="description-td">41:23:27:52</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">6</td>
                <td class="description-td">155</td>
                <td class="description-td"> </td>
                <td class="description-td">47:23:27:58</td>
              </tr> <tr class="description-tr"> <td class="description-td">CG3</td>
                <td class="description-td">1</td>
                <td class="description-td">153</td>
                <td class="description-td"> </td>
                <td class="description-td">61:14:23:55</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">158</td>
                <td class="description-td"> </td>
                <td class="description-td">64:14:23:57</td>
              </tr> <tr class="description-tr"> <td class="description-td">VrrB2</td>
                <td class="description-td">1</td>
                <td class="description-td">150</td>
                <td class="description-td"> </td>
                <td class="description-td">54:21:51:27</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">159</td>
                <td class="description-td"> </td>
                <td class="description-td">59:21:53:29</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">3</td>
                <td class="description-td">168</td>
                <td class="description-td"> </td>
                <td class="description-td">64:21:55:31</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0140">The results presented in this example indicate that the methods of the present invention provide a powerful means for high-throughput genotyping of <i>B. anthracis</i>. These same methods can be applied to similar genotyping analyses for other bioagents.</p>
    <p num="p-0141">Various modifications of the invention, in addition to those described herein, will be apparent to those skilled in the art from the foregoing description. Such modifications are also intended to fall within the scope of the appended claims. Each reference cited in the present application is incorporated herein by reference in its entirety</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5547835">US5547835</a></td><td class="patent-data-table-td patent-date-value">Jan 6, 1994</td><td class="patent-data-table-td patent-date-value">Aug 20, 1996</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Using the sanger sequencing strategy and assembling the sequence information by analysis of the nested fragments obtained by base-specific chain termination via their different molecular masses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5605798">US5605798</a></td><td class="patent-data-table-td patent-date-value">Mar 17, 1995</td><td class="patent-data-table-td patent-date-value">Feb 25, 1997</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Detecting target nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5622824">US5622824</a></td><td class="patent-data-table-td patent-date-value">Feb 10, 1995</td><td class="patent-data-table-td patent-date-value">Apr 22, 1997</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA sequencing by mass spectrometry via exonuclease degradation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5691141">US5691141</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 1995</td><td class="patent-data-table-td patent-date-value">Nov 25, 1997</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA sequencing by mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5849492">US5849492</a></td><td class="patent-data-table-td patent-date-value">Oct 29, 1996</td><td class="patent-data-table-td patent-date-value">Dec 15, 1998</td><td class="patent-data-table-td ">Phylogenetix Laboratories, Inc.</td><td class="patent-data-table-td ">Method for rapid identification of prokaryotic and eukaryotic organisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5872003">US5872003</a></td><td class="patent-data-table-td patent-date-value">May 30, 1995</td><td class="patent-data-table-td patent-date-value">Feb 16, 1999</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA sequencing by mass spectrometry via exonuclease degradation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5965363">US5965363</a></td><td class="patent-data-table-td patent-date-value">Dec 2, 1996</td><td class="patent-data-table-td patent-date-value">Oct 12, 1999</td><td class="patent-data-table-td ">Genetrace Systems Inc.</td><td class="patent-data-table-td ">Methods of preparing nucleic acids for mass spectrometric analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6043031">US6043031</a></td><td class="patent-data-table-td patent-date-value">Mar 18, 1996</td><td class="patent-data-table-td patent-date-value">Mar 28, 2000</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">DNA diagnostics based on mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6180372">US6180372</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 27, 1998</td><td class="patent-data-table-td patent-date-value">Jan 30, 2001</td><td class="patent-data-table-td ">Bruker Daltonik Gmbh</td><td class="patent-data-table-td ">Amplification of preferential nucleotide seuences; transferring reaction solution to reaction chambers, allow sample to flow into cells with variations in heat, amplify and isolate nucleotide sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6221587">US6221587</a></td><td class="patent-data-table-td patent-date-value">May 12, 1998</td><td class="patent-data-table-td patent-date-value">Apr 24, 2001</td><td class="patent-data-table-td ">Isis Pharmceuticals, Inc.</td><td class="patent-data-table-td ">Detecting activation positionin nucleotide sequences; comparing nucleotide sequences and detecting conserved nucleotide sequence structures</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6613509">US6613509</a></td><td class="patent-data-table-td patent-date-value">Mar 22, 1999</td><td class="patent-data-table-td patent-date-value">Sep 2, 2003</td><td class="patent-data-table-td ">Regents Of The University Of California</td><td class="patent-data-table-td ">DNA sequences; polymorphisms; 13C; 15N; 2H; polymerase chain reactions; mass shift analysis; use in sequence validation, comparison and characterization of sequence polymorphisms without the need to sequence the oligonucleotide</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20020168630">US20020168630</a></td><td class="patent-data-table-td patent-date-value">Mar 1, 2002</td><td class="patent-data-table-td patent-date-value">Nov 14, 2002</td><td class="patent-data-table-td ">Fleming James E.</td><td class="patent-data-table-td ">Detecting viable cells in sample; obtain cells, incubate with fluorescent dyes, detect fluorescence, compare to control</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1997033000A1?cl=en">WO1997033000A1</a></td><td class="patent-data-table-td patent-date-value">Mar 4, 1997</td><td class="patent-data-table-td patent-date-value">Sep 12, 1997</td><td class="patent-data-table-td ">Genetrace Systems Inc</td><td class="patent-data-table-td ">Methods of screening nucleic acids using mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1997037041A2?cl=en">WO1997037041A2</a></td><td class="patent-data-table-td patent-date-value">Mar 18, 1997</td><td class="patent-data-table-td patent-date-value">Oct 9, 1997</td><td class="patent-data-table-td ">Sequenom Inc</td><td class="patent-data-table-td ">Dna sequencing by mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998012355A1?cl=en">WO1998012355A1</a></td><td class="patent-data-table-td patent-date-value">Sep 19, 1997</td><td class="patent-data-table-td patent-date-value">Mar 26, 1998</td><td class="patent-data-table-td ">Christopher H Becker</td><td class="patent-data-table-td ">Methods of preparing nucleic acids for mass spectrometric analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998020166A2?cl=en">WO1998020166A2</a></td><td class="patent-data-table-td patent-date-value">Nov 6, 1997</td><td class="patent-data-table-td patent-date-value">May 14, 1998</td><td class="patent-data-table-td ">Andreas Braun</td><td class="patent-data-table-td ">Dna diagnostics based on mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998054751A1?cl=en">WO1998054751A1</a></td><td class="patent-data-table-td patent-date-value">May 29, 1998</td><td class="patent-data-table-td patent-date-value">Dec 3, 1998</td><td class="patent-data-table-td ">Christopher H Becker</td><td class="patent-data-table-td ">Volatile matrices for matrix-assisted laser desorption/ionization mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1999014375A2?cl=en">WO1999014375A2</a></td><td class="patent-data-table-td patent-date-value">Sep 18, 1998</td><td class="patent-data-table-td patent-date-value">Mar 25, 1999</td><td class="patent-data-table-td ">Christopher A Becker</td><td class="patent-data-table-td ">Dna typing by mass spectrometry with polymorphic dna repeat markers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1999031278A1?cl=en">WO1999031278A1</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 1998</td><td class="patent-data-table-td patent-date-value">Jun 24, 1999</td><td class="patent-data-table-td ">Charles Cantor</td><td class="patent-data-table-td ">Mass spectrometric methods for sequencing nucleic acids</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Aaserud, et al., "<a href='http://scholar.google.com/scholar?q="Accurate+base+composition+of+double-strand+DNA+by+mass+spectrometry%2C"'>Accurate base composition of double-strand DNA by mass spectrometry,</a>" J. Am. Soc. Mass Spec. (1996) 7:1266-1269.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baker, et al., "<a href='http://scholar.google.com/scholar?q="Review+and+re-analysis+of+domain-specific+16S+primers%2C"'>Review and re-analysis of domain-specific 16S primers,</a>" J. Microbiol. Methods (2003) 55:541-555.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Benson, et al., "<a href='http://scholar.google.com/scholar?q="Advantages+of+Thermococcus+kodakaraenis+%28KOD%29+DNA+polymerase+for+PCR-mass+spectrometry+based+analyses%2C"'>Advantages of Thermococcus kodakaraenis (KOD) DNA polymerase for PCR-mass spectrometry based analyses,</a>" J. Am. Soc. Mass Spectrom. (2003) 14:601-604.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Black. et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+trace+levels+of+tricothecene+mycotoxins+in+human+urineby+gas+chromatography-mass+spectrometry%2C"'>Detection of trace levels of tricothecene mycotoxins in human urineby gas chromatography-mass spectrometry,</a>" J. Chromatog. (1986) 367:103-115.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Campbell and Huang, "<a href='http://scholar.google.com/scholar?q="Detection+of+California+serogroup+Bunyavirus+in+tissue+culture+and+mosquito+pools+by+PCR%2C"'>Detection of California serogroup Bunyavirus in tissue culture and mosquito pools by PCR,</a>" J. Virol. Methods (1996) 57:175-179.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chen, et al., "<a href='http://scholar.google.com/scholar?q="A+universal+PCR+primer+to+detect+members+of+the+Potyviridae+and+its+use+to+examine+the+taxonomic+status+of+several+members+of+the+family%2C"'>A universal PCR primer to detect members of the Potyviridae and its use to examine the taxonomic status of several members of the family,</a>" Arch. Virol. (2001) 146:757-766.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Conrads, et al., "<a href='http://scholar.google.com/scholar?q="16S-23S+rDNA+internal+transcribed+spacer+sequences+for+analysis+of+the+phylogenetic+relationships+among+species+of+the+genus+Fusobacterium%2C"'>16S-23S rDNA internal transcribed spacer sequences for analysis of the phylogenetic relationships among species of the genus Fusobacterium,</a>" Intl. J. System. Evol. Microbiol. (2002) 52:493-499.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dasen, et al., "<a href='http://scholar.google.com/scholar?q="Classification+and+identification+of+Propioibacteria+based+on+ribosomal+RNA+genes+and+PCR%2C"'>Classification and identification of Propioibacteria based on ribosomal RNA genes and PCR,</a>" System. Appl. Microbiol. (1998) 21:251-259.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Deforce, et al., "<a href='http://scholar.google.com/scholar?q="Characterization+of+DNA+oligonucleotides+by+coupling+of+capillary+zone+electrophoresis+to+electrospray+ionization+Q-TOF+mass+spectrometry%2C"'>Characterization of DNA oligonucleotides by coupling of capillary zone electrophoresis to electrospray ionization Q-TOF mass spectrometry,</a>" Anal. Chem. (1998) 70:3060-3068.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Demesure, et al., "<a href='http://scholar.google.com/scholar?q="A+set+of+universal+primers+for+amplification+of+polymorphic+non-coding+regions+of+mitochondrial+and+chloroplast+DNA+in+plants%2C"'>A set of universal primers for amplification of polymorphic non-coding regions of mitochondrial and chloroplast DNA in plants,</a>" Mol. Ecol. (1995) 4:129-131.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Flora. et al., "<a href='http://scholar.google.com/scholar?q="Dual-micro-ESI+source+for+precise+mass+determination+on+a+quadrupole+time-of-flight+mass+spectrometer+for+genomic+and+proteomic+applications%2C"'>Dual-micro-ESI source for precise mass determination on a quadrupole time-of-flight mass spectrometer for genomic and proteomic applications,</a>" Anal. Bioanal. Chem. (2002) 373:538-546.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fox et al., "<a href='http://scholar.google.com/scholar?q="Report+of+the+%27Bioterrorism+Workshop%27"'>Report of the 'Bioterrorism Workshop'</a>", J. Microbol. Methods (2002) 51:247-254.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fox, et al., "<a href='http://scholar.google.com/scholar?q="Identification+of+Brucella+by+ribosomal-spacer-region+PCR+and+differentiation+of+Brucell+canis+from+other+Brucella+spp.+pathogenic+for+humans+by+carbohydrate+profiles%2C"'>Identification of Brucella by ribosomal-spacer-region PCR and differentiation of Brucell canis from other Brucella spp. pathogenic for humans by carbohydrate profiles,</a>" J. Clin. Microbiol. (1998) 36:3217-3222.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Griffey and Greig, "<a href='http://scholar.google.com/scholar?q="Detection+of+base+pair+mismatches+in+duplex+DNA+and+RNA+oligonucleotides+using+electrospray+mass+spectrometry%2C"'>Detection of base pair mismatches in duplex DNA and RNA oligonucleotides using electrospray mass spectrometry,</a>" SPIE (1997) 2985:82-86.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Griffin, et al., "<a href='http://scholar.google.com/scholar?q="Direct+genetic+analysis+by+matrix-assisted+laseer+desorption%2Fionization+mass+spectrometry%2C"'>Direct genetic analysis by matrix-assisted laseer desorption/ionization mass spectrometry,</a>" proc. Natl. Acad. Sci. USA (1999) 96:6301-6306.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hannis and Muddiman, "<a href='http://scholar.google.com/scholar?q="Accurate+characterization+fo+the+tyrosine+hydroxylase+forensic+allele+9.3+through+development+of+electrospray+ionization+Fourier+transform+ion+cyclotron+resonance+mass+spectrometry%2C"'>Accurate characterization fo the tyrosine hydroxylase forensic allele 9.3 through development of electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,</a>" Rapid. Comm. Mass Spectrom. (1999) 13:954-962.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hannis and Muddiman, "<a href='http://scholar.google.com/scholar?q="Detection+of+double-stranded+PCR+amplicons+at+the+attomote+level+electrosprayed+from+low+nanomolar+solutions+using+FT-ICR+mass+spectrometry%2C"'>Detection of double-stranded PCR amplicons at the attomote level electrosprayed from low nanomolar solutions using FT-ICR mass spectrometry,</a>" Fresenius J. Anal Chem. (2001) 369:246-251.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hannis and Muddiman, "<a href='http://scholar.google.com/scholar?q="Genotyping+short+tandem+repeats+using+flow+injection+and+electrospray+ionization+Fourier+transform+ion+cyclotron+resonance+mass+spectrometry%2C"'>Genotyping short tandem repeats using flow injection and electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,</a>" Rapid. Comm. Mass Spectrom. (2001) 15:348-350.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hayashi, et al., "<a href='http://scholar.google.com/scholar?q="Phylogenetic+analysis+of+the+human+gut+microbiota+using+16S+rDNA+clone+libraries+and+strictly+anaerobic+culture+based+methods%2C"'>Phylogenetic analysis of the human gut microbiota using 16S rDNA clone libraries and strictly anaerobic culture based methods,</a>" Microbiol. Immunol. (2002) 46:535-548.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hoffmann, et al., "<a href='http://scholar.google.com/scholar?q="Universal+primer+set+for+the+full-length+amplification+of+all+influenza+A+viruses%2C"'>Universal primer set for the full-length amplification of all influenza A viruses,</a>" Arch. Virol. (2001) 146:2275-2289.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hurst, et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+bacterial+DNA+polymerase+chain+reaction+products+by+matrix-assisted+laser+desorption%2Fionization+mass+spectrometry%2C"'>Detection of bacterial DNA polymerase chain reaction products by matrix-assisted laser desorption/ionization mass spectrometry,</a>" Rapid. Comm. Mass. Spec. (1996) 10:377-382.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Isola, et al., "<a href='http://scholar.google.com/scholar?q="MALDI-TOF+mass+spectrometric+method+for+detection+of+hybridized+DNA+oligomers%2C"'>MALDI-TOF mass spectrometric method for detection of hybridized DNA oligomers,</a>" Anal. Chem. (2001) 73:2126-2131.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jankowski and Soler, "<a href='http://scholar.google.com/scholar?q="Mass+spectrometry+of+DNA%3A+Part+2%2A+Quantitative+estimation+of+base+composition%2C"'>Mass spectrometry of DNA: Part 2* Quantitative estimation of base composition,</a>" Eur. J. Mass Spectrom. Biochem. Med. Environ. Res. (1980) 1:45-52.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kageyama and Benno, "<a href='http://scholar.google.com/scholar?q="Rapid+detection+f+human+fecal+Eubacterium+species+and+related+genera+by+tested+PCR+method%2C"'>Rapid detection f human fecal Eubacterium species and related genera by tested PCR method,</a>" Microbiol. Immunol. (2001) 45:315-318.</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Little, et al., "<a href='http://scholar.google.com/scholar?q="Rapid+sequencing+of+oligonucleotides+by+high-resolution+mass+spectrometry%2C"'>Rapid sequencing of oligonucleotides by high-resolution mass spectrometry,</a>" J. Am. Chem. Soc. (1994) 116:4893-4897.</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Liu, et al., "<a href='http://scholar.google.com/scholar?q="Improving+the+microdialysis+procedure+for+electrospray+ionization+mass+spectrometry+of+biological+samples%2C"'>Improving the microdialysis procedure for electrospray ionization mass spectrometry of biological samples,</a>" J. Mass Spectrom. (1997) 32:425-431.</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mangrum, et al., "<a href='http://scholar.google.com/scholar?q="Solution+composition+and+thermal+denaturation+for+the+production+of+single-stranded+PCR+amplicons%3A+piperidine-induced+destabilization+of+the+DNA+duplex%2C"'>Solution composition and thermal denaturation for the production of single-stranded PCR amplicons: piperidine-induced destabilization of the DNA duplex,</a>" J. Am. Soc. Mass Spectrom. (2002) 13:232-240.</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McCabe, et al., "<a href='http://scholar.google.com/scholar?q="Bacterial+species+identification+after+DNA+amplification+with+a+universal+primer+pair%2C"'>Bacterial species identification after DNA amplification with a universal primer pair,</a>" Mol. Genet. Metab. (1999) 66:205-211.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Meiyu, et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+flaviviruses+by+reverse+transcriptase-polymerase+chain+reaction+with+the+universal+primer+set%2C"'>Detection of flaviviruses by reverse transcriptase-polymerase chain reaction with the universal primer set,</a>" Microbiol. Immunol. (1997) 41:209-213.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Moricca, et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+Fusarium+oxysporum+f.sp.+vasinfectum+in+cotton+tissue+by+polymerase+chain+reaction%2C"'>Detection of Fusarium oxysporum f.sp. vasinfectum in cotton tissue by polymerase chain reaction,</a>" Plant Pathol. (1998) 47:486-494.</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, et al., "<a href='http://scholar.google.com/scholar?q="Characterization+of+PCR+products+from+Bacilli+using+electrospray+ionization+FTICR+mass+spectrometry%2C"'>Characterization of PCR products from Bacilli using electrospray ionization FTICR mass spectrometry,</a>" Anal Chem. (1996) 68:3705-3712.</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, et al., "<a href='http://scholar.google.com/scholar?q="Length+and+base+composition+of+PCR-amplified+nucleic+acids+using+mass+measurements+from+electrospray+ionization+mass+spectrometry%2C"'>Length and base composition of PCR-amplified nucleic acids using mass measurements from electrospray ionization mass spectrometry,</a>" Anal. Chem. (1997) 69:1543-1549.</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, et al., "<a href='http://scholar.google.com/scholar?q="Precise+mass+measurement+of+a+double-stranded+500+base-pair+%28309+kDa%29+polymerase+chain+reaction+product+by+negative+ion+electrospray+ionization+fourier+transform+ion+cyclotron+resonance+mass+spectrometry%2C"'>Precise mass measurement of a double-stranded 500 base-pair (309 kDa) polymerase chain reaction product by negative ion electrospray ionization fourier transform ion cyclotron resonance mass spectrometry,</a>" Rapid Comm. Mass Spec. (1999) 13:1201-1204.</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Muddiman, et al., "<a href='http://scholar.google.com/scholar?q="Sequencing+and+characterization+of+larger+oligonucleotides+by+electrospray+ionization+fourier+transform+ion+cyclotron+resonance+mass+spectrometry%2C"'>Sequencing and characterization of larger oligonucleotides by electrospray ionization fourier transform ion cyclotron resonance mass spectrometry,</a>" Rev. Anal. Chem. (1998) 17:1-68.</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nagpal, et al., "<a href='http://scholar.google.com/scholar?q="Utility+of+16S-23S+rRNA+spacer+region+methodology%3A+how+similar+are+interspace+regions+within+a+genome+and+between+strains+for+closely+related+organisms%3F%2C"'>Utility of 16S-23S rRNA spacer region methodology: how similar are interspace regions within a genome and between strains for closely related organisms?,</a>" J. Microbiol. Methods (1998) 33:211-219.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Nakao et al (J. Clin. Microbiol. (1997) 35(7):1651-1655).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Nishikawa (FEMS Microbiol. Letters (1999) 178:13-18).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Null and Muddiman, "<a href='http://scholar.google.com/scholar?q="Determination+of+a+correction+to+improve+mass+measurement+accuracy+of+isotopically+unresolved+polymerase+chain+reaction+amplicons+by+electrospray+ionization+Fourier+transform+ion+cyclotron+resonance+mass+spectrometry%2C"'>Determination of a correction to improve mass measurement accuracy of isotopically unresolved polymerase chain reaction amplicons by electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,</a>" Rapid Comm. Mass Spectrom. (2003) 17:1714-1722.</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Null and Muddiman, "<a href='http://scholar.google.com/scholar?q="Perspectives+on+the+use+of+electrospray+ionization+Fourier+transform+ion+cyclotron+resonance+mass+spectrometry+for+short+tandem+repeat+genotyping+in+the+post+genome+era%2C"'>Perspectives on the use of electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry for short tandem repeat genotyping in the post genome era,</a>" J. Mass Spectrom. (2001) 36:589-606.</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Null, et al., "<a href='http://scholar.google.com/scholar?q="Evaluation+of+sample+preparation+techniques+for+mass+measurements+of+PCR+products+using+ESI-FT-ICR+mass+spectrometry%2C"'>Evaluation of sample preparation techniques for mass measurements of PCR products using ESI-FT-ICR mass spectrometry,</a>" Am Soc. Mass Spectrom. (2002) 13:338-344.</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Null, et al., "<a href='http://scholar.google.com/scholar?q="Genotyping+of+simple+and+compound+short+tandem+repeat+loci+using+electrospray+ionization+Fourier+transform+ion+cyclotron+resonance+mass+spectrometry%2C"'>Genotyping of simple and compound short tandem repeat loci using electrospray ionization Fourier transform ion cyclotron resonance mass spectrometry,</a>" Anal. Chem. (2001) 73:4514-4521.</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Null, et al., "<a href='http://scholar.google.com/scholar?q="Implications+of+hydrophobicity+and+free+energy+of+solvation+for+characterization+of+nucleic+acids+by+electrospray+ionization+mass+spectrometry%2C"'>Implications of hydrophobicity and free energy of solvation for characterization of nucleic acids by electrospray ionization mass spectrometry,</a>" Anal. Chem. (2003) 75:1331-1339.</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Null, et al., "<a href='http://scholar.google.com/scholar?q="Preparation+of+single-stranded+PCR+products+for+electrospray+ionization+mass+spectrometry+using+the+DNA+repair+enzyme+lambda+exonuclease%2C"'>Preparation of single-stranded PCR products for electrospray ionization mass spectrometry using the DNA repair enzyme lambda exonuclease,</a>" Analyst (2000) 125:619-626.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Peng, et al., "<a href='http://scholar.google.com/scholar?q="Rapid+detection+of+Shigella+species+in+environmental+sewage+by+an+immunocapture+PCR+with+universal+primers%2C"'>Rapid detection of Shigella species in environmental sewage by an immunocapture PCR with universal primers,</a>" App. Environ. Microbiol. (2002) 68:2580-2583.</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pomerantz, et al., "<a href='http://scholar.google.com/scholar?q="Determination+of+oligonucleotide+composition+from+mass+spectrometrically+measured+molecular+weight%2C"'>Determination of oligonucleotide composition from mass spectrometrically measured molecular weight,</a>" J. Am. Soc. Mass Spectrom. (1993) 4:204-209.</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ross, et al., "<a href='http://scholar.google.com/scholar?q="Discrimination+of+single-nucleotide+polymorphisms+in+human+DNA+using+peptide+nucleic+acid+probes+detected+by+MALDI-TOF+mass+spectrometry%2C"'>Discrimination of single-nucleotide polymorphisms in human DNA using peptide nucleic acid probes detected by MALDI-TOF mass spectrometry,</a>" Anal. Chem. (1997) 69:4197-4202.</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Scaramozzino, et al., "<a href='http://scholar.google.com/scholar?q="Comparison+of+Flavivirus+universal+primer+pairs+and+development+of+a+rapid%2C+highly+sensitive+heminested+reverse+transcription-PCR+assay+for+detection+of+flaviviruses+targeted+to+a+conserved+region+of+the+NS5+gene+sequences%2C"'>Comparison of Flavivirus universal primer pairs and development of a rapid, highly sensitive heminested reverse transcription-PCR assay for detection of flaviviruses targeted to a conserved region of the NS5 gene sequences,</a>" J. Clin. Microbiol. (2001) 39:1922-1927.</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shaver, et al., "<a href='http://scholar.google.com/scholar?q="Restriction+fragment+length+polymorphism+of+rRNA+operons+for+discrimination+and+intergenic+spacer+sequences+for+cataloging+of+Bacilus+subtilis+sub-groups%2C"'>Restriction fragment length polymorphism of rRNA operons for discrimination and intergenic spacer sequences for cataloging of Bacilus subtilis sub-groups,</a>" J. Microbiol. Methods (2002) 50:215-223.</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Srinivasan, et al., "<a href='http://scholar.google.com/scholar?q="Matrix-assisted+laser+desorption%2Fionization+time-of-flight+mass+spectrometry+as+a+rapid+screening+method+to+detect+mutations+causing+Tay-Sachs+disease%2C"'>Matrix-assisted laser desorption/ionization time-of-flight mass spectrometry as a rapid screening method to detect mutations causing Tay-Sachs disease,</a>" Rapid Comm. Mass Spectrom. (1997) 11:1144-1150.</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Steffens and Roy, "<a href='http://scholar.google.com/scholar?q="Sequence+analysis+of+mitochondrial+DNA+hypervariable+regions+using+infrared+fluorescence+detection%2C"'>Sequence analysis of mitochondrial DNA hypervariable regions using infrared fluorescence detection,</a>" Bio/Techniques (1998) 24:1044-1046.</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wunschel, et al., "<a href='http://scholar.google.com/scholar?q="Heterogeneity+in+bacillus+cereus+PCR+products+detected+by+ESI-FTICR+mass+spectrometry%2C"'>Heterogeneity in bacillus cereus PCR products detected by ESI-FTICR mass spectrometry,</a>" Anal. Chem. (1998) 70:1203-1207.</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wunschel, et al., "<a href='http://scholar.google.com/scholar?q="Mass+spectrometric+characterization+of+DNA+for+molecular+biological+applications%3A+advances+using+MALDI+and+ESI%2C"'>Mass spectrometric characterization of DNA for molecular biological applications: advances using MALDI and ESI,</a>" Adv. Mass Spectrom., vol. 14, Karjalainen, et al., (eds.) 1998, Elsevier, Amsterdam.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7781162">US7781162</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 11, 2003</td><td class="patent-data-table-td patent-date-value">Aug 24, 2010</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Using replication of intelligent primers (ribosomal RNA or house keeping oligonucleotides) to identify, classify and diagnose infection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7811766">US7811766</a></td><td class="patent-data-table-td patent-date-value">Mar 28, 2008</td><td class="patent-data-table-td patent-date-value">Oct 12, 2010</td><td class="patent-data-table-td ">Thinkvillage, Llc</td><td class="patent-data-table-td ">A pair of oligonucleotide molecules for amplification of ribosomal DNA from a plant material; DNA polymerase chain reaction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8076104">US8076104</a></td><td class="patent-data-table-td patent-date-value">Jan 25, 2008</td><td class="patent-data-table-td patent-date-value">Dec 13, 2011</td><td class="patent-data-table-td ">Rogan Peter K</td><td class="patent-data-table-td ">Rapid and comprehensive identification of prokaryotic organisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8088582">US8088582</a></td><td class="patent-data-table-td patent-date-value">Apr 6, 2007</td><td class="patent-data-table-td patent-date-value">Jan 3, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for the use in identification of fungi</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8242254">US8242254</a></td><td class="patent-data-table-td patent-date-value">Mar 7, 2007</td><td class="patent-data-table-td patent-date-value">Aug 14, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for use in identification of bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8288523">US8288523</a></td><td class="patent-data-table-td patent-date-value">Mar 7, 2007</td><td class="patent-data-table-td patent-date-value">Oct 16, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for use in identification of bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8394945">US8394945</a></td><td class="patent-data-table-td patent-date-value">Mar 7, 2007</td><td class="patent-data-table-td patent-date-value">Mar 12, 2013</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for use in identification of bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8527207">US8527207</a></td><td class="patent-data-table-td patent-date-value">May 15, 2008</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Peter K. Rogan</td><td class="patent-data-table-td ">Accurate identification of organisms based on individual information content</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8679741">US8679741</a></td><td class="patent-data-table-td patent-date-value">May 30, 2007</td><td class="patent-data-table-td patent-date-value">Mar 25, 2014</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Methods and compositions for the extraction and amplification of nucleic acid from a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8771948">US8771948</a></td><td class="patent-data-table-td patent-date-value">Apr 1, 2010</td><td class="patent-data-table-td patent-date-value">Jul 8, 2014</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Nucleic acid preparation compositions and methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010080616A1?cl=en">WO2010080616A1</a></td><td class="patent-data-table-td patent-date-value">Dec 18, 2009</td><td class="patent-data-table-td patent-date-value">Jul 15, 2010</td><td class="patent-data-table-td ">Abbott Laboratories</td><td class="patent-data-table-td ">Molecular assay for diagnosis of malaria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012159089A1?cl=en">WO2012159089A1</a></td><td class="patent-data-table-td patent-date-value">May 18, 2012</td><td class="patent-data-table-td patent-date-value">Nov 22, 2012</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Products and processes for multiplex nucleic acid identification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013049135A1?cl=en">WO2013049135A1</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2012</td><td class="patent-data-table-td patent-date-value">Apr 4, 2013</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Algorithms for sequence determinations</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=2EN8BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006120">435/6.12</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2EN8BAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=2EN8BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2EN8BAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0019340000">C12P19/34</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=2EN8BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/689">C12Q1/689</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2EN8BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6813">C12Q1/6813</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2EN8BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6888">C12Q1/6888</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2EN8BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6858">C12Q1/6858</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2EN8BAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/686">C12Q1/686</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68D4</span>, <span class="nested-value">C12Q1/68M10B</span>, <span class="nested-value">C12Q1/68D2G</span>, <span class="nested-value">C12Q1/68M10</span>, <span class="nested-value">C12Q1/68B</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Feb 15, 2011</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 1 AND 5 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 2-4 AND 6-11, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 22, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 26, 2009</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20090409</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 14, 2007</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">IBIS BIOSCIENCES, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:ISIS PHARMACEUTICALS, INC.;REEL/FRAME:019690/0036</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20070814</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">IBIS BIOSCIENCES, INC.,CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:ISIS PHARMACEUTICALS, INC.;US-ASSIGNMENT DATABASE UPDATED:20100223;REEL/FRAME:19690/36</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 7, 2007</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 5, 2004</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ISIS PHARMACEUTICALS, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:ECKER, DAVID J.;GRIFFEY, RICHARD H.;SAMPATH, RANGARAJAN;AND OTHERS;REEL/FRAME:014309/0594;SIGNING DATES FROM 20040121 TO 20040204</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2fQoOPkR4z8n2KjdjoBt_sCnch-w\u0026id=2EN8BAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U1T0kKLRMHAMo6V8RLQs2Vdx4FD1A\u0026id=2EN8BAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U3MTe8FmElQ-BpT0hve6GeMJE5OHg","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Methods_for_rapid_detection_and_identifi.pdf?id=2EN8BAABERAJ\u0026output=pdf\u0026sig=ACfU3U2-On8t4edPybs4vB0M3-pGfKeN_w"},"sample_url":"http://www.google.com/patents/reader?id=2EN8BAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>