<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7332304 - Fermentation of genetic engineered microorganisms; gastrointestinal ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_4ff636b3d23669b7103f3b3a3a18b4cd/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_4ff636b3d23669b7103f3b3a3a18b4cd__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Fermentation of genetic engineered microorganisms; gastrointestinal disorders, antiallergens, inflammatory bowel disorders, antiarthritic agents"><meta name="DC.contributor" content="Ming-De Deng" scheme="inventor"><meta name="DC.contributor" content="J. David Angerer" scheme="inventor"><meta name="DC.contributor" content="Don Cyron" scheme="inventor"><meta name="DC.contributor" content="Alan D. Grund" scheme="inventor"><meta name="DC.contributor" content="Thomas A. Jerrell, Jr." scheme="inventor"><meta name="DC.contributor" content="Candice Leanna" scheme="inventor"><meta name="DC.contributor" content="Owen Mathre" scheme="inventor"><meta name="DC.contributor" content="Reinhardt Rosson" scheme="inventor"><meta name="DC.contributor" content="Jeff Running" scheme="inventor"><meta name="DC.contributor" content="Dave Severson" scheme="inventor"><meta name="DC.contributor" content="Linsheng Song" scheme="inventor"><meta name="DC.contributor" content="Sarah Wassink" scheme="inventor"><meta name="DC.contributor" content="Arkion Life Sciences Llc" scheme="assignee"><meta name="DC.date" content="2003-7-1" scheme="dateSubmitted"><meta name="DC.description" content="A biosynthetic method for producing glucosamine and N-acetylglucosamine is disclosed. Such a method includes the fermentation of a genetically modified microorganism to produce glucosamine and/or N-acetylglucosamine. Also disclosed are genetically modified microorganisms that are useful for producing glucosamine and N-acetylglucosamine. In addition, methods of recovering N-acetylglucosamine that has been produced by a fermentation process, including methods that result in N-acetylglucosamine of high purity, are described. Also disclosed is a method to produce glucosamine from N-acetylglucosamine."><meta name="DC.date" content="2008-2-19" scheme="issued"><meta name="DC.relation" content="EP:0614972:A2" scheme="references"><meta name="DC.relation" content="EP:0732400:A2" scheme="references"><meta name="DC.relation" content="GB:1056331" scheme="references"><meta name="DC.relation" content="JP:H09234064" scheme="references"><meta name="DC.relation" content="RU:2141965" scheme="references"><meta name="DC.relation" content="US:20030181419:A1" scheme="references"><meta name="DC.relation" content="US:3008874" scheme="references"><meta name="DC.relation" content="US:3679660" scheme="references"><meta name="DC.relation" content="US:3683076" scheme="references"><meta name="DC.relation" content="US:4642340" scheme="references"><meta name="DC.relation" content="US:5246840" scheme="references"><meta name="DC.relation" content="US:5731184" scheme="references"><meta name="DC.relation" content="US:5739015" scheme="references"><meta name="DC.relation" content="US:5744325" scheme="references"><meta name="DC.relation" content="US:5843923" scheme="references"><meta name="DC.relation" content="US:5876713" scheme="references"><meta name="DC.relation" content="US:5998173" scheme="references"><meta name="DC.relation" content="US:6372457" scheme="references"><meta name="DC.relation" content="US:6486307" scheme="references"><meta name="DC.relation" content="US:6531303" scheme="references"><meta name="DC.relation" content="WO:1997031121:A1" scheme="references"><meta name="DC.relation" content="WO:2002043653:A2" scheme="references"><meta name="DC.relation" content="WO:2002066667:A1" scheme="references"><meta name="citation_reference" content="Araki et al., 1975, Eur. J. Biochem., 55, 71-78."><meta name="citation_reference" content="Bachmann, Escherichia coli and Salmonella typhimurium cellular and molecular biology Mutant Derivatives of E. coli, vol. 2:pp. 1190-1219, 1987."><meta name="citation_reference" content="Badet et al., 1987, Biochemistry (Mosc). 26, 1940-1948."><meta name="citation_reference" content="Balbas et al., 1996, Gene, 172, 65-69."><meta name="citation_reference" content="Bearne, 1996, J. Biol. Chem., 271, 3052-3057."><meta name="citation_reference" content="Bernheim et al., 1970, J. Bacteriol., 101, 384-391."><meta name="citation_reference" content="Blattner et al., 1997, Science, 277, 1453-1474."><meta name="citation_reference" content="Bouffard et al., 1994, J. Mol. Biol., 244, 269-278."><meta name="citation_reference" content="Branden et al., Introduction to Protein Structure, Garland Publishing Inc., New York, p. 247, 1991."><meta name="citation_reference" content="Brown et al., 1989, Biochemical Society Transaction, 18, 317-318."><meta name="citation_reference" content="Chen et al., 1987, Molecular General Genetics, 210, 331-337."><meta name="citation_reference" content="Chen et al., 1998, J. Agric. Food Chem., 46, 1971-1974."><meta name="citation_reference" content="Chen et al., 1998, J. Agric. Food Chem., 46, 3207-3209."><meta name="citation_reference" content="Cochet et al., 1995, Archives of Biochemistry &amp; Biophysics, 324, 391-400."><meta name="citation_reference" content="Curtis et al., 1975, J. Bacteriol., 122, 1189-1199."><meta name="citation_reference" content="Daniels et al., 1993, Mol. Endocrinol., 7, 1041-1048."><meta name="citation_reference" content="Denisot et al., 1991, Arch. Biochem. Biophys., 288, 225-230."><meta name="citation_reference" content="Domard et al., 1991, Int. J. Biol. Macromol., 13, 105-109."><meta name="citation_reference" content="Dutka-Malen et al., 1988, Biochimie, 70, 287-290."><meta name="citation_reference" content="Elliot et al., 1975, Biochem. Biophys. Res. Commun., 64, 277-281."><meta name="citation_reference" content="Fawcett et al., 1995, J. Bacteriol., 177, 1742-1750."><meta name="citation_reference" content="Fernandez-Herrero et al., 1995, Mol. Microbiol., 17, 1-12."><meta name="citation_reference" content="Flores et al., 1996, Nat. Biotechnol., 14, 620-623."><meta name="citation_reference" content="Freese et al., 1970, J. Bacteriol., 101, 1046-1062."><meta name="citation_reference" content="Glaser et al., 1955, J. Biol. Chem., 216, 67-79."><meta name="citation_reference" content="Golinelli-Pimpaneau et al., 1991, Eur. J. Biochem., 201, 175-182."><meta name="citation_reference" content="Holmes et al., 1972, J. Bacteriol., 111, 290-291."><meta name="citation_reference" content="Horlacher et al., 1997, J. Biol. Chem., 272, 13026-13032."><meta name="citation_reference" content="Isaeva et al., 1987, Izvestiya Akademii Nauk SSR, 4, 937-939."><meta name="citation_reference" content="Jones-Mortimer et al., 1980, J. Gen. Microbiol., 117, 369-376."><meta name="citation_reference" content="Joyce et al., 1984, J. Bacteriol., 158, 636-643."><meta name="citation_reference" content="Kashige et al., 1994, Carboyhdrate Research, 257, 285-291."><meta name="citation_reference" content="Kauss et al., 1988, Methods Enzymol., 161, 518-523."><meta name="citation_reference" content="Kimata et al., 1997, Biochemistry (Mosc). 94, 12914-12919."><meta name="citation_reference" content="Knapp et al., 2002, Tetrahedron Lett., 43, 6075-6078."><meta name="citation_reference" content="Konecny et al., 1982, in Enzyme Engineering, vol. 6:pp. 91-96, Plenum Press."><meta name="citation_reference" content="Lengeler, 1980, Mol. Gen. Genet., 179, 49-54."><meta name="citation_reference" content="Leriche et al., 1996, J. Am. Chem. Soc., 118, 1797-1798."><meta name="citation_reference" content="Leriche et al., 1997, Eur. J. Biochem., 245, 418-422."><meta name="citation_reference" content="Marie et al., 1992, Mol. Microbiol., 6, 843-851."><meta name="citation_reference" content="Martinou et al., 1995, Carbohydr. Res., 273, 235-242."><meta name="citation_reference" content="McKnight et al., 1992, J. Biol. Chem., 267, 25208-25212."><meta name="citation_reference" content="Mengin-Lecreulx et al., 1996, The Journal of Biological Chemistry, 271, 32-39."><meta name="citation_reference" content="Miclett et al., 2001, The Journal of Biological Chemistry, 37, 34840-34846."><meta name="citation_reference" content="Milewski, 1993, Biochimistry et. Biophys Acta, 1161, 279-284."><meta name="citation_reference" content="Mio et al., 1999, J. Biol. Chem., 274, 424-429."><meta name="citation_reference" content="Miyazaki et al., 1979, Chem. Pharm. Bull., 27, 532-535."><meta name="citation_reference" content="Mudri et al., 1992, J. Biol. Chem., 267, 25208-25212."><meta name="citation_reference" content="Mukhija et al., 1996, The Journal of Biological Chemistry, 271, 14819-14824."><meta name="citation_reference" content="Novikov et al., 1997, Russian J. Appl. Chem., 70, 1467-1470."><meta name="citation_reference" content="Novikov, 1999, Russian J. Appl. Chem., 72, 156-161."><meta name="citation_reference" content="Obmolova et al., 1994, J. Mol. Biol., 242, 703-705."><meta name="citation_reference" content="Oppon et al., 1998, J. Bacteriol., 180, 3007-3012."><meta name="citation_reference" content="O&#39;Shannessy et al., 1993, Anal. Biochem., 212, 457-468."><meta name="citation_reference" content="O&#39;Shea et al., 1993, Anal. Chem., 65, 948-951."><meta name="citation_reference" content="Oyaizu, 1986, Japanese J. Nutr., 44, 307-315."><meta name="citation_reference" content="Oyaizu, 1988, Nippon Shikuhin Kogyo Gakkaishi, 35, 771-775."><meta name="citation_reference" content="Oyaizu, 1988, Nippon Shikuhin Kogyo Gakkaishi, 35, 846-850."><meta name="citation_reference" content="Penneff et al., 2001, The Journal of Biological Chemistry, 276, 16328-16334."><meta name="citation_reference" content="Peri et al., 1990, Biochemical and Cell Biology, 68, 123-137."><meta name="citation_reference" content="Plumbridge et al., 1993, J. Bacteriol., 175, 4951-4956."><meta name="citation_reference" content="Plumbridge et al., 1999, J. Bacteriol., 181, 47-54."><meta name="citation_reference" content="Plumbridge, 1987, Mol. Gen. Genet., 209, 618-620."><meta name="citation_reference" content="Plumbridge, 1989, Mol. Microbiol., 3, 505-515."><meta name="citation_reference" content="Plumbridge, 1990, J. Bacteriol., 172, 2728-2735."><meta name="citation_reference" content="Plumbridge, 1991, Mol. Microbiol., 5, 2053-2062."><meta name="citation_reference" content="Plumbridge, 1992, J. Gen. Microbiol., 138, 1011-1017."><meta name="citation_reference" content="Plumbridge, 1998, Mol. Microbiol., 27, 369-380."><meta name="citation_reference" content="Plumbridge, 2001, Nucleic Acids Res., 29, 506-514."><meta name="citation_reference" content="Pompeo et al., 2001, J. Biol. Chem., 276, 3833-3839."><meta name="citation_reference" content="Pradel et al., 1988, J. Bacteriol., 170, 4916-4923."><meta name="citation_reference" content="Roseman, 1957, J. Biol. Chem., 226, 115-124."><meta name="citation_reference" content="Rowley et al., 1991, J. Bacteriol., 173, 968-977."><meta name="citation_reference" content="Sarvas, 1971, J. Bacteriol., 105, 467-471."><meta name="citation_reference" content="Sashiwa et al., 2001, Chemistry Letters, 308-309."><meta name="citation_reference" content="Sashiwa et al., 2001, Japanese Soc. Chitin Chitosan, 7, 257-260."><meta name="citation_reference" content="Schuster et al., 1993, Nature, 365, 343-347."><meta name="citation_reference" content="Seffernick et al., J. Bacteriol. 183(8):2405-2410, 2001."><meta name="citation_reference" content="Shi and Xie 2002 Huaxue Shiki 24:93-94."><meta name="citation_reference" content="Shu 1998, Journal Agricultural of Food Chemistry, 46, 1129-1131."><meta name="citation_reference" content="Smith et al., 1996, J. Bacteriol., 178, 2320-2327."><meta name="citation_reference" content="Souza et al., 1997, Archives of Biochemistry &amp; Biophysics, 340, 338-346."><meta name="citation_reference" content="Sumoto et al., 1991, Chem. Pharm. Bull., 39, 792-794."><meta name="citation_reference" content="Teplyakov et al., 1998, Structure, 6, 1047-1055."><meta name="citation_reference" content="Vogler et al., 1989, J. Bacteriol., 171, 6586-6592."><meta name="citation_reference" content="Vogler et al., 1989, Mol. Gen. Genet., 219, 97-105."><meta name="citation_reference" content="Walker et al., 1984, Biochem. J., 224, 799-815."><meta name="citation_reference" content="Wanner, 1990, in The Molecular Basis of Bacterial Metabolism, vol. 41:pp. 152-163, Springer-Verlag."><meta name="citation_reference" content="Wanner, 1993, J. Cell. Biochem., 51, 47-54."><meta name="citation_reference" content="Wanner, 1996, in Escherichia coli and Salmonella: Cellular and Molecular Biology, vol. 1:pp. 1357-1381, ASM Press."><meta name="citation_reference" content="Watanabe et al., 1990, Argric. Biol. Chem., 54, 519-525."><meta name="citation_reference" content="Watzele et al., 1989, J. Biol. Chem., 264, 8753-8758."><meta name="citation_reference" content="White et al., 1967, Biochem. J., 105, 121-125."><meta name="citation_reference" content="White, 1968, Biochem. J., 106, 847-858."><meta name="citation_reference" content="Witkowski et al., Biochemistry 38:11643-11650, 1999."><meta name="citation_reference" content="Wolfe et al., 1956, Arch. Biochem. Biophys., 64, 489-497."><meta name="citation_reference" content="Wu et al., 1971, J. Bacteriol., 105, 455-466."><meta name="citation_reference" content="Xin-Yi et al., 2001, Journal of Changde Teachers University (Natural Science Edition), 13, 63-65."><meta name="citation_reference" content="Yamano et al., 1994, Biosci. Biotechnol. Biochem., 58, 193-195."><meta name="citation_reference" content="Yamano et al., 1996, Biosci. Biotechnol. Biochem., 60, 1320-1323."><meta name="citation_reference" content="Yamano et al., 1997, Biosci. Biotechnol. Biochem., 61, 1349-1353."><meta name="citation_reference" content="Yamano et al., 2000, Mar. Biotechnol., 2, 57-64."><meta name="citation_patent_number" content="US:7332304"><meta name="citation_patent_application_number" content="US:10/612,779"><link rel="canonical" href="http://www.google.com/patents/US7332304"/><meta property="og:url" content="http://www.google.com/patents/US7332304"/><meta name="title" content="Patent US7332304 - Fermentation of genetic engineered microorganisms; gastrointestinal disorders, antiallergens, inflammatory bowel disorders, antiarthritic agents"/><meta name="description" content="A biosynthetic method for producing glucosamine and N-acetylglucosamine is disclosed. Such a method includes the fermentation of a genetically modified microorganism to produce glucosamine and/or N-acetylglucosamine. Also disclosed are genetically modified microorganisms that are useful for producing glucosamine and N-acetylglucosamine. In addition, methods of recovering N-acetylglucosamine that has been produced by a fermentation process, including methods that result in N-acetylglucosamine of high purity, are described. Also disclosed is a method to produce glucosamine from N-acetylglucosamine."/><meta property="og:title" content="Patent US7332304 - Fermentation of genetic engineered microorganisms; gastrointestinal disorders, antiallergens, inflammatory bowel disorders, antiarthritic agents"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("mljpU4rrCsXHsASY_4DYDA"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407464522.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("EGY"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("mljpU4rrCsXHsASY_4DYDA"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407464522.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("EGY"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7332304?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7332304"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=2W2BBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7332304&amp;usg=AFQjCNF_30DHA5elIvwXD9b4UEYDrjZzoA" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7332304.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7332304.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20040091976"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7332304"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7332304" style="display:none"><span itemprop="description">A biosynthetic method for producing glucosamine and N-acetylglucosamine is disclosed. Such a method includes the fermentation of a genetically modified microorganism to produce glucosamine and/or N-acetylglucosamine. Also disclosed are genetically modified microorganisms that are useful for producing...</span><span itemprop="url">http://www.google.com/patents/US7332304?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7332304 - Fermentation of genetic engineered microorganisms; gastrointestinal disorders, antiallergens, inflammatory bowel disorders, antiarthritic agents</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7332304 - Fermentation of genetic engineered microorganisms; gastrointestinal disorders, antiallergens, inflammatory bowel disorders, antiarthritic agents" title="Patent US7332304 - Fermentation of genetic engineered microorganisms; gastrointestinal disorders, antiallergens, inflammatory bowel disorders, antiarthritic agents"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7332304 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 10/612,779</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Feb 19, 2008</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jul 1, 2003</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Jul 1, 2002</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2488853A1">CA2488853A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CN101365785A">CN101365785A</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1576106A2">EP1576106A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1576106A4">EP1576106A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8124381">US8124381</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20040091976">US20040091976</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120009627">US20120009627</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2004003175A2">WO2004003175A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2004003175A3">WO2004003175A3</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">10612779, </span><span class="patent-bibdata-value">612779, </span><span class="patent-bibdata-value">US 7332304 B2, </span><span class="patent-bibdata-value">US 7332304B2, </span><span class="patent-bibdata-value">US-B2-7332304, </span><span class="patent-bibdata-value">US7332304 B2, </span><span class="patent-bibdata-value">US7332304B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Ming-De+Deng%22">Ming-De Deng</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22J.+David+Angerer%22">J. David Angerer</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Don+Cyron%22">Don Cyron</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Alan+D.+Grund%22">Alan D. Grund</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Thomas+A.+Jerrell,+Jr.%22">Thomas A. Jerrell, Jr.</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Candice+Leanna%22">Candice Leanna</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Owen+Mathre%22">Owen Mathre</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Reinhardt+Rosson%22">Reinhardt Rosson</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Jeff+Running%22">Jeff Running</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Dave+Severson%22">Dave Severson</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Linsheng+Song%22">Linsheng Song</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Sarah+Wassink%22">Sarah Wassink</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Arkion+Life+Sciences+Llc%22">Arkion Life Sciences Llc</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7332304.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7332304.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7332304.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (23),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (102),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (2),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (26),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (4)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7332304&usg=AFQjCNEkYU_5uvwg50zOHNRjCtr014QWdQ">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7332304&usg=AFQjCNGlWCAOLQ20rFEoH6Ro2u5OJIkhTA">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7332304B2%26KC%3DB2%26FT%3DD&usg=AFQjCNG4jMx_ilFdulqSQ0xI0ZOvfC0p2w">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129945741" lang="EN" load-source="ifi">Fermentation of genetic engineered microorganisms; gastrointestinal disorders, antiallergens, inflammatory bowel disorders, antiarthritic agents</invention-title></span><br><span class="patent-number">US 7332304 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA51266600" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">A biosynthetic method for producing glucosamine and N-acetylglucosamine is disclosed. Such a method includes the fermentation of a genetically modified microorganism to produce glucosamine and/or N-acetylglucosamine. Also disclosed are genetically modified microorganisms that are useful for producing glucosamine and N-acetylglucosamine. In addition, methods of recovering N-acetylglucosamine that has been produced by a fermentation process, including methods that result in N-acetylglucosamine of high purity, are described. Also disclosed is a method to produce glucosamine from N-acetylglucosamine.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(25)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00013.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00013.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00014.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00014.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00015.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00015.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00016.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00016.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00017.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00017.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00018.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00018.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00019.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00019.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00020.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00020.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00021.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00021.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00022.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00022.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00023.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00023.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00024.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00024.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7332304B2/US07332304-20080219-D00025.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7332304B2/US07332304-20080219-D00025.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(67)</span></span></div><div class="patent-text"><div mxw-id="PCLM9344786" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A method to produce N-acetylglucosamine-6-phosphate or N-acetylglucosamine by fermentation, comprising:
<div class="claim-text">a) culturing in a fermentation medium a microorganism that is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding a glucosamine-6-phosphate acetyltransferase that has an amino acid sequence that is at least 95% identical to SEQ ID NO:30; and</div>
<div class="claim-text">b) collecting a product produced from the step of culturing which is selected from the group consisting of N-acetylglucosamine-6-phosphate and N-acetylglucosamine.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the recombinant nucleic acid molecule further comprises a non-native promoter.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the glucosamine-6-phosphate acetyltransferase has the amino acid sequence of SEQ ID NO:30.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein expression of the recombinant nucleic acid molecule is inducible.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. The method of <claim-ref idref="CLM-00004">claim 4</claim-ref>, wherein expression of the recombinant nucleic acid molecule is inducible by lactose.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. The method of <claim-ref idref="CLM-00005">claim 5</claim-ref>, wherein the microorganism further comprises a partial or complete deletion or inactivation of a gene encoding a Lacl repressor protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the microorganism further comprises a partial or complete deletion of an endogenous gene encoding a glucosamine-6-phosphate deaminase.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the step of culturing includes the step of maintaining the carbon source at a concentration of from about 0.5% to about 5% in the fermentation medium.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the step of culturing is performed in a fermentation medium comprising yeast extract.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the step of culturing is performed in a fermentation medium comprising a carbon source selected from the group consisting of glucose, fructose, a pentose sugar, lactose and gluconic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref>, wherein the pentose sugar is selected from the group consisting of ribose, xylose, and arabinose.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the step of culturing is performed in a fermentation medium comprising glucose and ribose.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00013" num="00013" class="claim">
      <div class="claim-text">13. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the step of culturing is performed in a fermentation medium comprising glucose and gluconic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00014" num="00014" class="claim">
      <div class="claim-text">14. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the step of culturing is performed at a temperature of from about 25° C. to about 45° C.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00015" num="00015" class="claim">
      <div class="claim-text">15. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the step of culturing is performed at about 37° C.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00016" num="00016" class="claim">
      <div class="claim-text">16. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the step of culturing is performed at a pH of from about pH 4 to about pH 7.5.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00017" num="00017" class="claim">
      <div class="claim-text">17. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the step of culturing is performed at a pH of from about pH 6.7 to about pH 7.5.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00018" num="00018" class="claim">
      <div class="claim-text">18. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the step of culturing is performed at a pH of from about pH 4.5 to about pH 5.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00019" num="00019" class="claim">
      <div class="claim-text">19. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the microorganism is a bacterium or yeast.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00020" num="00020" class="claim">
      <div class="claim-text">20. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the microorganism is a bacterium.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00021" num="00021" class="claim">
      <div class="claim-text">21. The method of <claim-ref idref="CLM-00020">claim 20</claim-ref>, wherein the bacterium is a bacterium from a genus selected from the group consisting of: <i>Escherichia, Bacillus, Lactobacillus, Pseudomonas </i>and <i>Streptomyces. </i> </div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00022" num="00022" class="claim">
      <div class="claim-text">22. The method of <claim-ref idref="CLM-00020">claim 20</claim-ref>, wherein the bacterium is a bacterium from a species selected from the group consisting of: <i>Escherichia coil, Bacillus subtilis, Bacillus licheniformis, Lactobacillus brevis, Pseudomonas aeruginosa </i>and <i>Streptomyces lividans. </i> </div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00023" num="00023" class="claim">
      <div class="claim-text">23. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the microorganism is a yeast.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00024" num="00024" class="claim">
      <div class="claim-text">24. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, further comprising the step of contacting the fermentation medium with at least one ion exchange resin.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00025" num="00025" class="claim">
      <div class="claim-text">25. The method of <claim-ref idref="CLM-00023">claim 23</claim-ref>, wherein the yeast is a yeast from a genus selected from the group consisting of: <i>Saccharomyces, Candida, Hansenula, Pichia, Kluveromyces</i>, and <i>Phaffia. </i> </div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00026" num="00026" class="claim">
      <div class="claim-text">26. The method of <claim-ref idref="CLM-00023">claim 23</claim-ref>, wherein the yeast is a yeast from a species selected from the group consisting of: <i>Saccharomyces cerevisiae, Schizosaccharomyces pombe, Candida albicans, Hansenula polymorpha, Pichia pastoris, P. canadensis, Kluyveromyces marxianus </i>and <i>Phaffia phodozyma. </i> </div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00027" num="00027" class="claim">
      <div class="claim-text">27. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the microorganism is transformed with a recombinant nucleic acid molecule comprising a nucleic acid sequence encoding a bacterial or yeast phosphoglucoisomerase.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00028" num="00028" class="claim">
      <div class="claim-text">28. The method of <claim-ref idref="CLM-00027">claim 27</claim-ref>, wherein the phosphoglucoisomerase comprises the amino acid sequence of SEQ ID NO:105.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00029" num="00029" class="claim">
      <div class="claim-text">29. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the microorganism further comprises a partial or complete deletion of an endogenous gene encoding a phosphofructokinase.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00030" num="00030" class="claim">
      <div class="claim-text">30. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the microorganism has been transformed with a recombinant nucleic acid molecule comprising a nucleic acid sequence encoding a bacterial or yeast glutamine synthetase.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00031" num="00031" class="claim">
      <div class="claim-text">31. The method of <claim-ref idref="CLM-00030">claim 30</claim-ref>, wherein the glutamine synthetase comprises the amino acid sequence of SEQ ID NO:89.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00032" num="00032" class="claim">
      <div class="claim-text">32. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the microorganism has been transformed with a recombinant nucleic acid molecule comprising a nucleic acid sequence encoding a bacterial or yeast glucose-6-phosphate dehydrogenase.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00033" num="00033" class="claim">
      <div class="claim-text">33. The method of <claim-ref idref="CLM-00032">claim 32</claim-ref>, wherein the glucose-6-phosphate dehydrogenase comprises the amino acid sequence of SEQ ID NO:95.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00034" num="00034" class="claim">
      <div class="claim-text">34. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the microorganism further comprises a partial or complete deletion of at least one endogenous gene encoding an enzyme involved in glycogen synthesis selected from the group consisting of: ADP-glucose pyrophosphorylase, glycogen synthase and a branching enzyme.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00035" num="00035" class="claim">
      <div class="claim-text">35. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, further comprising recovering an intracellular product from the microorganism selected from the group consisting of: intracellular glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-1-phosphate, and glucosamine.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00036" num="00036" class="claim">
      <div class="claim-text">36. A method to produce N-acetylglucosamine-6-phosphate, N-acetylglucosamine, glucosamine or glucosamine HCl by fermentation, comprising:
<div class="claim-text">culturing in a fermentation medium a microorganism that is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding a glucosamine-6-phosphate acetyltransferase that has an amino acid sequence that is at least 95% identical to SEQ ID NO: 30; and</div>
<div class="claim-text">collecting a product produced from the step of culturing which is selected from the group consisting of N-acetylglucosamine-6-phosphate and N-acetylglucosamine;</div>
<div class="claim-text">and wherein the method further comprises a step selected from the group consisting of:
<div class="claim-text">(a) purifying N-acetylglucosamine from the fermentation medium;</div>
<div class="claim-text">(b) dephosphorylating N-acetylglucosamine-6-phosphate to produce N-acetylglucosamine; and</div>
<div class="claim-text">(c) treating N-acetylglucosamine to produce a glucosamine product selected from the group consisting of: glucosamine and glucosamine HCl.</div>
</div>
</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00037" num="00037" class="claim">
      <div class="claim-text">37. The method of cliam <b>36</b>, wherein step (c) comprises hydrolyzing N-acetylglucosamine under acid and heat conditions or by enzymatic deacetylation.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00038" num="00038" class="claim">
      <div class="claim-text">38. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein N-acetylglucosamine produced by the fermentation method is recovered by precipitating N-acetylglucosamine-containing solids from the fermentation broth.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00039" num="00039" class="claim">
      <div class="claim-text">39. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein N-acetylglucosamine produced by the fermentation method is recovered by crystallizing N-acetylglucosamine-containing solids from the fermentation broth.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00040" num="00040" class="claim">
      <div class="claim-text">40. A method to produce N-acetylalucosamine-6-phosphate or N-acetylglucosamine by fermentation, comprising:
<div class="claim-text">a) culturing in a fermentation medium a microorganism that expresses:
<div class="claim-text">i) a recombinant nucleic acid molecule encoding a glucosamine-6-phosphate acetyltransferase that has an amino acid sequence that is at least 95% identical to SEQ ID NO:30; and</div>
<div class="claim-text">ii) a recombinant nucleic acid molecule encoding a glucosamine-6-phosphate synthase that has an amino acid sequence that is at least 95% identical to the amino acid sequence selected from the group consisting of: SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, SEQ ID NO:16, SEQ ID NO:18, and SEQ ID NO:20; and</div>
</div>
<div class="claim-text">b) collecting a product produced from the step of culturing which is selected from the group consisting of N-acetylglucosamine-6-phosphate and N-acetylglucosamine.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00041" num="00041" class="claim">
      <div class="claim-text">41. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the glucosamine-6-phosphate acetyltransferase has the amino acid sequence of SEQ ID NO:30.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00042" num="00042" class="claim">
      <div class="claim-text">42. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the glucosamine-6-phosphate synthase has the amino acid sequence of SEQ ID NO:6.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00043" num="00043" class="claim">
      <div class="claim-text">43. A method to produce N-acetylalucosamine-6-phosphate or N-acetylglucosamine by fermentation, comprising:
<div class="claim-text">a) culturing in a fermentation medium an <i>E. coli </i>that expresses:
<div class="claim-text">i) a recombinant nucleic acid molecule encoding a glucosamine-6-phosphate acetyltransferase that is at least 95% identical to the amino acid sequence of SEQ ID NO:30; and</div>
<div class="claim-text">ii) a recombinant nucleic acid molecule encoding a glucosamine-6-phosphate synthase that is at least 95% identical to the amino acid sequence of SEQ ID NO:6; and</div>
</div>
<div class="claim-text">b) collecting a product produced from the step of culturing which is selected from the group consisting of N-acetylglucosamine-6-phosphate and N-acetylglucosamine.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00044" num="00044" class="claim">
      <div class="claim-text">44. The method of <claim-ref idref="CLM-00043">claim 43</claim-ref>, wherein the <i>E. coli </i>further comprises a partial or complete deletion of the pfkA gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00045" num="00045" class="claim">
      <div class="claim-text">45. The method of <claim-ref idref="CLM-00044">claim 44</claim-ref>, wherein the <i>E. coli </i>further comprises a partial or complete deletion of the nagA, nagB, and nagE genes.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00046" num="00046" class="claim">
      <div class="claim-text">46. The method of <claim-ref idref="CLM-00044">claim 44</claim-ref>, wherein the <i>E. coli </i>further comprises a partial or complete deletion of the manXYZ gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00047" num="00047" class="claim">
      <div class="claim-text">47. The method of <claim-ref idref="CLM-00044">claim 44</claim-ref>, wherein the recombinant nucleic acid molecules of (a)(i) and (a)(ii) are inducible by lactose or galactose.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00048" num="00048" class="claim">
      <div class="claim-text">48. The method of <claim-ref idref="CLM-00044">claim 44</claim-ref>, wherein the step of culturing is performed in a fermentation medium comprising glucose and fructose.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00049" num="00049" class="claim">
      <div class="claim-text">49. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the microorganism further comprises a partial or complete deletion of an endogenous gene encoding a phosphofructokinase.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00050" num="00050" class="claim">
      <div class="claim-text">50. The method of <claim-ref idref="CLM-00049">claim 49</claim-ref>, wherein the microorganism further comprises a partial or complete deletion of endogenous genes encoding N-acetylglucosamine-6-phosphate deacetylase, glucosamine-6-phosphate deaminase, and N-acetyl-glucosamine-specific enzyme II<sup>Nag</sup>.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00051" num="00051" class="claim">
      <div class="claim-text">51. The method of <claim-ref idref="CLM-00049">claim 49</claim-ref>, wherein the microorganism further comprises a partial or complete deletion of an endogenous gene encoding mannose transporter EIIM,P/III<sup>Man</sup>.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00052" num="00052" class="claim">
      <div class="claim-text">52. The method of <claim-ref idref="CLM-00049">claim 49</claim-ref>, wherein the recombinant nucleic acid molecules of (a)(i) and (a)(ii) are inducible by lactose or galactose.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00053" num="00053" class="claim">
      <div class="claim-text">53. The method of <claim-ref idref="CLM-00049">claim 49</claim-ref>, wherein the step of culturing is performed in a fermentation medium comprising glucose and fructose.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00054" num="00054" class="claim">
      <div class="claim-text">54. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the glucosamine-6-phosphate synthase has an amino acid sequence that is at least 95% identical to SEQ ID NO:4.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00055" num="00055" class="claim">
      <div class="claim-text">55. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the glucosamine-6-phosphate synthase has an amino acid sequence that is at least 95% identical to SEQ ID NO:6.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00056" num="00056" class="claim">
      <div class="claim-text">56. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the glucosamine-6-phosphate synthase has an amino acid sequence that is at least 95% identical to SEQ ID NO:8.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00057" num="00057" class="claim">
      <div class="claim-text">57. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the glucosamine-6-phosphate synthase has an amino acid sequence that is at least 95% identical to SEQ ID NO:10.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00058" num="00058" class="claim">
      <div class="claim-text">58. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the glucosamine-6-phosphate synthase has an amino acid sequence that is at least 95% identical to SEQ ID NO:12.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00059" num="00059" class="claim">
      <div class="claim-text">59. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the glucosamine-6-phosphate synthase has an amino acid sequence that is at least 95% identical to SEQ ID NO:14.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00060" num="00060" class="claim">
      <div class="claim-text">60. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the glucosamine-6-phosphate synthase has an amino acid sequence that is at least 95% identical to SEQ ID NO:16.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00061" num="00061" class="claim">
      <div class="claim-text">61. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the glucosamine-6-phosphate synthase has an amino acid sequence that is at least 95% identical to SEQ ID NO:18.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00062" num="00062" class="claim">
      <div class="claim-text">62. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the glucosamine-6-phosphate synthase has an amino acid sequence that is at least 95% identical to SEQ ID NO:20.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00063" num="00063" class="claim">
      <div class="claim-text">63. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the microorganism is a bacterium or yeast.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00064" num="00064" class="claim">
      <div class="claim-text">64. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the microorganism is a bacterium.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00065" num="00065" class="claim">
      <div class="claim-text">65. The method of <claim-ref idref="CLM-00040">claim 40</claim-ref>, wherein the microorganism is a yeast.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00066" num="00066" class="claim">
      <div class="claim-text">66. The method of <claim-ref idref="CLM-00043">claim 43</claim-ref>, wherein the glucosamine-6-phosphate acetyltransferase has the amino acid sequence of SEQ ID NO:30.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00067" num="00067" class="claim">
      <div class="claim-text">67. The method of <claim-ref idref="CLM-00043">claim 43</claim-ref>, wherein the glucosamine-6-phosphate synthase has the amino acid sequence of SEQ ID NO:6.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES16367330" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS REFERENCE TO RELATED APPLICATIONS</heading> <p num="p-0002">This application claims the benefit of priority under 35 U.S.C. § 119(e) from U.S. Provisional Application Ser. No. 60/393,348, filed Jul. 1, 2002, entitled, “Process and Materials for Production of Glucosamine and N-Acetylglucosamine.” The entire disclosure of U.S. Provisional Application Ser. No. 60/393,348 is incorporated herein by reference.</p>
    <heading>FIELD OF THE INVENTION</heading> <p num="p-0003">The present invention relates to a method for producing glucosamine and N-acetylglucosamine by fermentation. The present invention also relates to genetically modified strains of microorganisms useful for producing glucosamine and N-acetylglucosamine. The present invention also relates to a method to recover N-acetylglucosamine or glucosamine from a fermentation process. The present invention also relates to a method to produce glucosamine from a source of N-acetylglucosamine.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p num="p-0004">Amino sugars are usually found as monomer residues in complex oligosaccharides and polysaccharides. Glucosamine is an amino derivative of the simple sugar, glucose. N-acetylglucosamine is an acetylated derivative of glucosamine. Glucosamine, N-acetylglucosamine and other amino sugars are important constituents of many natural polysaccharides. For example, polysaccharides containing amino sugars can form structural materials for cells, analogous to structural proteins.</p>
    <p num="p-0005">Glucosamine is manufactured as a nutraceutical product with applications in the treatment of osteoarthritic conditions in animals and humans, among other conditions. The market for glucosamine is experiencing tremendous growth. Furthermore, significant erosion of the world market price for glucosamine is not expected. N-acetylglucosamine is also a valuable pharmacological agent in the treatment of a wide variety of ailments. N-acetylglucosamine does not have any established negative side effects. Since N-acetylglucosamine is a valuable and important component of protein synthesis in the animal body it has a positive effect on tissue regeneration, N-acetylglucosamine has therapeutic potential in the prevention and/or treatment of a wide variety of diseases such as gastritis, food allergies, inflammatory bowel disease (IBD), diverticulitis, acute and chronic forms of rheumatoid arthritis and osteoarthritis, as well as the pathological conditions arising from metabolic disorders of the osteoarticular tissues.</p>
    <p num="p-0006">Glucosamine is currently obtained by acid hydrolysis of chitin, a complex carbohydrate derived from N-acetyl-D-glucosamine. Alternatively, glucosamine can also be produced by acid hydrolysis of variously acetylated chitosans. Chitin, a copolymer of N-acetylglucosamine and glucosamine, is a common natural substance, found in arthropods and fungi. It can be obtained from inexpensive sources like arthropod refuse, e.g.: shellfish (lobster, shrimp, krill, crab, and prawn exoskeletons); insects used to biodegrade swine offal like the fly larvae; and more recently from waste fungal biomass used in citric acid production. The final product, salts of glucosamine, are relatively expensive because the relatively low chitin content in refuse sources requires large volumes of waste to be processed in order to obtain relatively small amounts of product, and because the processing itself is relatively low yield and energy and chemically intensive.</p>
    <p num="p-0007">Common industrial practice is to purify the chitin by treating it with combinations of acids and bases to remove minerals, proteins and other impurities accompanying the offal starting material, and to then depolymerize and deacetylate the chitin in a single step to glucosamine through the use of concentrated hydrochloric acid at high temperature, long times and low yields. Glucosamine as a free base is very unstable and subject to degradation. Consequently stable salts such as the hydrochloride are produced. Other blends of salts are offered, usually using the hydrochloride as a base, in order to mimic forms that have been tested for efficacy in clinical settings like Viartril and DONA® 200-S. These compositions take the form of mixed salts with a molecular formula of (glucosamine)<sub>2 </sub>sulfate-(NaCl)<sub>2</sub>, and (glucosamine)<sub>2 </sub>sulfate-(KCl)<sub>2</sub>. More recently, salts of the structure (glucosamine)<sub>2 </sub>sodium bisulfate-(HCl)<sub>2</sub>, (glucosamine)<sub>2</sub>potassium bisulfate-(HCl)<sub>2 </sub>are being investigated as means to provide stable salts of glucosamine, at lower sodium and potassium dosages.</p>
    <p num="p-0008">N-acetylglucosamine is not widely available in the marketplace. It is currently produced by the acetylation of glucosamine using an organic acetylating reagent such as acetic anhydride, an expensive and difficult step. These processes suffer from poor product yields (in the range of 50% conversion of substrate to glucosamine).</p>
    <p num="p-0009">The common forms of glucosamine, being derived from shellfish, carry labels warning consumers of the potential for allergic reactions in persons sensitive to shellfish. Increasingly, consumers are seeking access to material that is free of all animal byproducts. Moreover, the availability of raw material (i.e., a source of chitin, such as crab shells) is becoming increasingly limited. Therefore, there is a need in the industry for a cost-effective method for producing high yields of glucosamine and N-acetylglucosamine for commercial sale and use.</p>
    <p num="p-0010">PCT Publication No. WO 02/66667 disclosed glucosamine and method of making glucosamine from microbial biomass. This method of production overcomes problems associated with shellfish allergy, but it suffers from a major problem of low yield. More particularly, since the method relies on the biomass waste generated in a fermentation that is dedicated to the production of other products such as citric acid, it is not sufficient to produce quantities of glucosamine that meet the increasing market demand for the product. U.S. Pat. No. 6,372,457, incorporated herein by reference in its entirety, disclosed a process and materials for production of glucosamine by microbial fermentation. However, U.S. Pat. No. 6,372,457 does not disclose any method for the production of N-acetylglucosamine.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0011">One embodiment of the present invention relates to a method to produce glucosamine or N-acetylglucosamine by fermentation. The method includes the steps of: (a) culturing in a fermentation medium a microorganism which comprises at least one genetic modification that increases the activity of glucosamine-6-phosphate acetyltransferase; and (b) collecting a product produced from the step of culturing which is selected from the group consisting of glucosamine-6-phosphate, glucosamine, glucosamine-1-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, and N-acetylglucosamine. In one aspect, the genetic modification to increase the activity of glucosamine-6-phosphate acetyltransferase provides a result selected from the group consisting of: increased enzymatic activity of glucosamine-6-phosphate acetyltransferase; overexpression of glucosamine-6-phosphate acetyltransferase by the microorganism; reduced N-acetylglucosamine-6-phosphate product inhibition of the glucosamine-6-phosphate acetyltransferase; and increased affinity of glucosamine-6-phosphate acetyltransferase for glucosamine-6-phosphate. In another aspect, the microorganism is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the glucosamine-6-phosphate acetyltransferase. In one aspect, the nucleic acid sequence encoding a glucosamine-6-phosphate acetyltransferase has at least one genetic modification which increases the enzymatic activity of the glucosamine-6-phosphate acetyltransferase. In another aspect, the glucosamine-6-phosphate acetyltransferase has an amino acid sequence that is at least about 35% identical, or at least about 50% identical, or at least about 70% identical, to an amino acid sequence selected from the group consisting of: SEQ ID NO:30, SEQ ID NO:32 and SEQ ID NO:34, wherein the glucosamine-6-phosphate acetyltransferase has enzymatic activity. In another aspect, the glucosamine-6-phosphate acetyltransferase has an amino acid sequence selected from the group consisting of SEQ ID NO:30, SEQ ID NO:32 and SEQ ID NO:34.</p>
    <p num="p-0012">In another aspect of this embodiment of the invention, the microorganism further comprises at least one genetic modification that decreases the activity of glucosamine-6-phosphate deaminase. The genetic modification to decrease the activity of glucosamine-6-phosphate deaminase can include, but is not limited to, a partial or complete deletion or inactivation of an endogenous gene encoding the glucosamine-6-phosphate deaminase in the microorganism.</p>
    <p num="p-0013">In one aspect of this embodiment of the invention, the microorganism further comprises at least one genetic modification that increases the activity of glucosamine-6-phosphate synthase. In one aspect, the microorganism is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the glucosamine-6-phosphate synthase. Such a glucosamine-6-phosphate synthase can include a glucosamine-6-phosphate synthase comprising an amino acid sequence that is at least about 35% identical, or at least about 50% identical, or at least about 70% identical, to an amino acid sequence selected from: SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, SEQ ID NO:16, SEQ ID NO:18, and SEQ ID NO:20, wherein the glucosamine-6-phosphate synthase has enzymatic activity. In one aspect, the glucosamine-6-phosphate synthase comprises an amino acid sequence selected from the group consisting of: SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, SEQ ID NO:16, SEQ ID NO:18, and SEQ ID NO:20. In one aspect, the glucosamine-6-phosphate synthase has a modification to reduce product inhibition of the glucosamine-6-phosphate synthase as compared to the wild-type glucosamine-6-phosphate synthase. Such a glucosamine-6-phosphate synthase can include, but is not limited to, a protein comprising an amino acid sequence selected from: SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, and SEQ ID NO:14. In a further aspect, this microorganism further comprises at least one genetic modification that decreases the activity of glucosamine-6-phosphate deaminase. The genetic modification to decrease the activity of glucosamine-6-phosphate deaminase can include, but is not limited to, a partial or complete deletion or inactivation of an endogenous gene encoding the glucosamine-6-phosphate deaminase in the microorganism.</p>
    <p num="p-0014">Another embodiment of the present invention relates to a method to produce glucosamine or N-acetylglucosamine by fermentation, comprising: (a) culturing in a fermentation medium a microorganism which comprises at least one genetic modification that increases the activity of glucosamine-6-phosphate deaminase; and (b) collecting a product produced from the step of culturing which is selected from the group consisting of glucosamine-6-phosphate, glucosamine, glucosamine-1-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, and N-acetylglucosamine. In this aspect, the genetic modification preferably provides a results selected from the group consisting of: overexpression of glucosamine-6-phosphate deaminase by the microorganism, increased enzymatic activity of glucosamine-6-phosphate deaminase, increased reverse reaction of glucosamine-6-phosphate deaminase to form increased glucosamine-6-phosphate, reduced forward reaction of glucosamine-6-phosphate deaminase to form reduced fructose-6-phosphate, increased affinity of glucosamine-6-phosphate deaminase for fructose-6-phosphate, reduced affinity of glucosamine-6-phosphate deaminase for glucosamine-6-phosphate, and reduced glucosamine-6-phosphate product inhibition of the glucosamine-6-phosphate deaminase. In one aspect, the microorganism is transformed with a recombinant nucleic acid molecule comprising a nucleic acid sequence encoding a glucosamine-6-phosphate deaminase. In one aspect the nucleic acid sequence encoding a glucosamine-6-phosphate deaminase has at least one genetic modification which increases the enzymatic activity of the glucosamine-6-phosphate deaminase. In one aspect, the glucosamine-6-phosphate deaminase has an amino acid sequence that is at least about 35% identical, or at least about 50% identical, or at least about 70% identical, to an amino acid sequence of SEQ ID NO:42, wherein the glucosamine-6-phosphate deaminase has enzymatic activity. In one aspect, the glucosamine-6-phosphate deaminase has an amino acid sequence of SEQ ID NO:42.</p>
    <p num="p-0015">In one aspect of this embodiment, the microorganism further comprises a genetic modification to decrease the activity of glucosamine-6-phosphate synthase. For example, the genetic modification to decrease the activity of glucosamine-6-phosphate synthase can include, but is not limited to, a partial or complete deletion or inactivation of an endogenous gene encoding glucosamine-6-phosphate synthase in the microorganism.</p>
    <p num="p-0016">In another aspect of this embodiment, the microorganism further comprises a genetic modification to increase the activity of glucosamine-6-phosphate N-acetyltransferase. The genetic modification can provide a result selected from: increased enzymatic activity of glucosamine-6-phosphate acetyltransferase; overexpression of glucosamine-6-phosphate acetyltransferase by the microorganism; reduced N-acetylglucosamine-6-phosphate product inhibition of the glucosamine-6-phosphate acetyltransferase; and increased affinity of glucosamine-6-phosphate acetyltransferase for glucosamine-6-phosphate. In one aspect, the microorganism is transformed with a recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the glucosamine-6-phosphate N-acetyltransferase, as described for the previous embodiment herein. Various aspects of the glucosamine-6-phosphate N-acetyltransferase have been described above in the prior embodiment and are encompassed here. In one aspect, the microorganism further comprises a genetic modification to decrease the activity of glucosamine-6-phosphate synthase. Such a modification can include, but is not limited to, a partial or complete deletion or inactivation of an endogenous gene encoding glucosamine-6-phosphate synthase in the microorganism.</p>
    <p num="p-0017">In yet another aspect of this embodiment, the microorganism further comprises a genetic modification to increase the activity of glucosamine-1-phosphate N-acetyltransferase. The genetic modification can provide a result selected from the group consisting of: increased enzymatic activity of glucosamine-1-phosphate N-acetyltransferase; reduced N-acetylglucosamine-1-phosphate uridyltransferase enzymatic activity; overexpression of an enzyme having glucosamine-1-phosphate N-acetyltransferase activity by the microorganism; increased affinity of glucosamine-1-phosphate N-acetyltransferase for glucosamine-1-phosphate; reduced affinity of an glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase for N-acetylglucosamine-1-phosphate; and reduced N-acetylglucosamine-1-phosphate product inhibition of the glucosamine-1-phosphate N-acetyltransferase. In one aspect, the microorganism comprises a bifunctional glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase, wherein the glucosamine-1-phosphate N-acetyltransferase activity is increased. In another aspect, the microorganism is transformed with a recombinant nucleic acid molecule comprising a nucleic acid sequence encoding a glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase or a nucleic acid sequence encoding a glucosamine-1-phosphate N-acetyltransferase. In one aspect, the nucleic acid sequence encoding a glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase or a glucosamine-1-phosphate N-acetyltransferase has at least one genetic modification which increases the activity of the glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase or the glucosamine-1-phosphate N-acetyltransferase, respectively. In another aspect, the glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase has an amino acid sequence that is at least about 35% identical to an amino acid sequence of SEQ ID NO:56, wherein the glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase has glucosamine-1-phosphate N-acetyltransferase enzymatic activity. In another aspect, the glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase has an amino acid sequence of SEQ ID NO:56. In another aspect, the truncated glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase having glucosamine-1-phosphate N-acetyltransferase activity, and reduced or no N-acetylglucosamine-1-phosphate uridyltransferase activity, including but not limited to, a truncated glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase having an amino acid sequence that is at least about 35% identical to an amino acid sequence of SEQ ID NO:58. In one aspect, the truncated glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate-uridyltransferase has an amino acid sequence of SEQ ID NO:58. In this aspect, the microorganism can further comprise a genetic modification to decrease the activity of glucosamine-6-phosphate synthase, such as is described above.</p>
    <p num="p-0018">Yet another embodiment of the present invention relates to a method to produce glucosamine or N-acetylglucosamine by fermentation, comprising: (a) culturing in a fermentation medium a microorganism which comprises at least one genetic modification that decreases the activity of glucosamine-6-phosphate deaminase and at least one genetic modification that increases the activity of glucosamine-1-phosphate N-acetyltransferase; and (b) collecting a product produced from the step of culturing which is selected from the group consisting of glucosamine-6-phosphate, glucosamine, glucosamine-1-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, and N-acetylglucosamine. In one aspect, the genetic modification to decrease the activity of glucosamine-6-phosphate deaminase comprises a partial or complete deletion or inactivation of an endogenous gene encoding the glucosamine-6-phosphate deaminase in the microorganism. In another aspect, the genetic modification to increase the activity of glucosamine-1-phosphate N-acetyltransferase provides a result selected from the group consisting of: increased enzymatic activity of glucosamine-1-phosphate N-acetyltransferase; reduced N-acetylglucosamine-1-phosphate uridyltransferase enzymatic activity; overexpression of an enzyme having glucosamine-1-phosphate N-acetyltransferase activity by the microorganism; increased affinity of glucosamine-1-phosphate N-acetyltransferase for glucosamine-1-phosphate; reduced affinity of an glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase for N-acetylglucosamine-1-phosphate; and reduced N-acetylglucosamine-1-phosphate product inhibition of the glucosamine-1-phosphate N-acetyltransferase. Various aspects of such a bifunctional glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase have been described in prior embodiments above, and are encompassed here as well.</p>
    <p num="p-0019">In one aspect of this embodiment, the microorganism further comprises at least one genetic modification that increases the activity of glucosamine-6-phosphate synthase. Various aspects of such a modification have been described in prior embodiments above and are encompassed here.</p>
    <p num="p-0020">Yet another embodiment of the invention relates to a method to produce glucosamine or N-acetylglucosamine by fermentation, comprising: (a) culturing in a fermentation medium a microorganism which comprises an endogenous glucosamine-6-phosphate acetyltransferase and at least one genetic modification to increase the activity of glucosamine-6-phosphate synthase; and (b) collecting a product produced from the step of culturing which is selected from the group consisting of glucosamine-6-phosphate, glucosamine, glucosamine-1-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, and N-acetylglucosamine. In one aspect, the microorganism is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the glucosamine-6-phosphate synthase. Various aspects of this modification have been described above and are encompassed herein. In one aspect, the microorganism further comprises at least one genetic modification that decreases the activity of glucosamine-6-phosphate deaminase, as has been described above. In one aspect, the method is a method to produce N-acetylglucosamine by fermentation, and wherein the step of collecting comprises collecting a product produced from the step of culturing which is selected from the group consisting of N-acetylglucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, and N-acetylglucosamine.</p>
    <p num="p-0021">In one aspect of any of the above-described embodiments, the expression of any of the above-referenced recombinant nucleic acid molecules is inducible, such as by, but not limited to, lactose. In one aspect, the microorganism further comprises a genetic modification to reduce inhibition of transcription induction by lactose. Such a genetic modification can include, but is not limited to, a partial or complete deletion or inactivation of a gene encoding a LacI repressor protein.</p>
    <p num="p-0022">In any of the above-identified embodiments of the invention related to a method to produce glucosamine or N-acetylglucosamine by fermentation, the following discussion can apply. In one aspect, the step of culturing includes the step of maintaining the carbon source at a concentration of from about 0.5% to about 5% in the fermentation medium. In another aspect, the step of culturing is performed in a fermentation medium comprising yeast extract. In yet another aspect, the step of culturing is performed in a fermentation medium comprising a carbon source selected from the group consisting of glucose, fructose, a pentose sugar, lactose and gluconic acid. The pentose sugar can include, but is not limited to, ribose, xylose, and arabinose. In another aspect, the step of culturing is performed in a fermentation medium comprising glucose and ribose, and in another aspect, the step of culturing is performed in a fermentation medium comprising glucose and gluconic acid. In one aspect, the step of culturing is performed at a temperature of from about 25° C. to about 45° C., and in another aspect, at about 37° C. In one aspect, the step of culturing is performed at a pH of from about pH 4 to about pH 7.5 and in another aspect, at a pH of from about pH 6.7 to about pH 7.5, and in yet another aspect, at a pH of from about pH 4.5 to about pH 5.</p>
    <p num="p-0023">In any of the above-described embodiments for a method to produce glucosamine or N-acetylglucosamine by a fermentation process, the microorganism can include any one or more of the following modifications. In one aspect, the microorganism further comprises a genetic modification to increase phosphoglucoisomerase activity in the microorganism. For example, the microorganism can be transformed with a recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the phosphoglucoisomerase, which can include a phosphoglucoisomerase comprising an amino acid sequence of SEQ ID NO:105. In another aspect, the microorganism further comprises a partial or complete deletion or inactivation of phosphofructokinase in the microorganism. In another aspect, the microorganism further comprises a genetic modification to increase the activity of glutamine synthetase. For example, the microorganism can be transformed with a recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the glutamine synthetase, which can include a glutamine synthetase comprising an amino acid sequence of SEQ ID NO:89. In another aspect, the microorganism further-comprises a genetic modification to increase the activity of glucose-6-phosphate dehydrogenase. For example, the microorganism can be transformed with a recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the glucose-6-phosphate dehydrogenase, which can include a glucose-6-phosphate dehydrogenase comprising an amino acid sequence of SEQ ID NO:95. In another aspect, the microorganism further comprises a partial or complete deletion or inactivation of genes encoding enzymes responsible for glycogen synthesis in the microorganism, including but not limited to, ADP-glucose pyrophosphorylase, glycogen synthase and a branching enzyme. Preferably, none of the above-described modifications inhibit the ability of the microorganism to metabolize galactose.</p>
    <p num="p-0024">In any of the above-described embodiments of a method to produce glucosamine and/or N-acetylglucosamine by fermentation, the step of collecting can include recovering an intracellular product from the microorganism selected from the group consisting of: intracellular glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine and glucosamine or recovering an extracellular product from the fermentation medium selected from the group consisting of: glucosamine and N-acetylglucosamine. In one aspect, this step includes a step selected from: (a) purifying a product selected from the group consisting of glucosamine and N-acetylglucosamine from the fermentation medium; (b) recovering a product selected from the group consisting of glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and N-acetylglucosamine-1-phosphate from the microorganism; (c) dephosphorylating a product selected from the group consisting of glucosamine-6-phosphate and glucosamine-1-phosphate to produce glucosamine; and (d) dephosphorylating a product selected from the group consisting of N-acetylglucosamine-6-phosphate and N-acetylglucosamine-1-phosphate to produce N-acetylglucosamine; (e) treating a product selected from the group consisting of N-acetylglucosamine, N-acetylglucosamine-6-phosphate and N-acetylglucosamine-1-phosphate to produce a glucosamine product selected from the group consisting of: glucosamine, glucosamine-6-phosphate and glucosamine-1-phosphate. In one aspect, step (e) comprises hydrolyzing the product selected from the group consisting of N-acetylglucosamine, N-acetylglucosamine-6-phosphate and N-acetylglucosamine-1-phosphate, under acid and heat conditions or by enzymatic deacetylation. In one aspect N-acetylglucosamine produced by the fermentation method is recovered by precipitating N-acetylglucosamine-containing solids from the fermentation broth. In another aspect, N-acetylglucosamine produced by the fermentation method is recovered by crystallizing N-acetylglucosamine-containing solids from the fermentation broth.</p>
    <p num="p-0025">Another embodiment of the present invention relates to any of the above-described genetically modified microorganisms.</p>
    <p num="p-0026">The genetically modified microorganism in any of the embodiments described above can include, but is not limited to, bacteria and fungi. In one aspect, the microorganism is selected from the group consisting of bacteria and yeast. Suitable bacteria include, but are not limited to, bacteria from a genus selected from the group consisting of: <i>Escherichia, Bacillus, Lactobacillus, Pseudomonas </i>and <i>Streptomyces</i>. Suitable bacteria further include, but are not limited to, bacteria from a species selected from the group consisting <i>Escherichia coli, Bacillus subtilis, Bacillus licheniformis, Lactobacillus brevis, Pseudomonas aeruginosa </i>and <i>Streptomyces lividans</i>. Suitable yeast include, but are not limited to, yeast from a genus selected from the group consisting of: <i>Saccharomyces, Candida, Hansenula, Pichia, Kluveromyces</i>, and <i>Phaffia</i>. Suitable yeast further include, but are not limited to, yeast from a species selected from the group consisting of: <i>Saccharomyces cerevisiae, Schizosaccharomyces pombe, Candida albicans, Hansenula polymorpha, Pichia pastoris, P. canadensis, Kluyveromyces marxianus </i>and <i>Phaffia rhodozyma</i>. Suitable fungi include, but are not limited to, fungi from a genus selected from the group consisting of: <i>Aspergillus, Absidia, Rhizopus, Chrysosporium, Neurospora </i>and <i>Trichoderma</i>. Suitable fungi further include, but are not limited to, fungi from a species selected from the group consisting of: <i>Aspergillus niger, A. nidulans, Absidia coerulea, Rhizopus oryzae, Chrysosporium lucknowense, Neurospora crassa, N. intermedia </i>and <i>Trichoderm reesei. </i> </p>
    <p num="p-0027">Yet another embodiment of the present invention relates to a method to produce N-acetylglucosamine, comprising: (a) obtaining a fermentation broth containing solubilized N-acetylglucosamine that is a product of a fermentation process; and (b) recovering N-acetylglucosamine-containing solids from the fermentation broth. In one aspect, the method further includes removing cellular material from the fermentation broth. In one aspect, the method further includes decolorizing the fermentation broth. Such a step of decolorizing can include, but is not limited to, multiple N-acetylglucosamine crystallizations, activated carbon treatment, and chromatographic decolorization. In one aspect, the method further includes the step of contacting the fermentation broth with an ion exchange resin. For example, the step of contacting the fermentation broth with an ion exchange resin can include, but is not limited to, contacting the fermentation broth with an anion exchange resin and a cation exchange resin. The step of contacting the fermentation broth with an anion exchange resin and a cation exchange resin can include contacting the fermentation broth with a mixed bed of anion and cation exchange resins.</p>
    <p num="p-0028">In one aspect of this embodiment, the step recovering comprises precipitating N-acetylglucosamine-containing solids from the fermentation broth.</p>
    <p num="p-0029">In one aspect of this embodiment, the step recovering comprises crystallizing N-acetylglucosamine-containing solids from the fermentation broth.</p>
    <p num="p-0030">In one aspect of this embodiment, the step of recovering comprises concentrating the fermentation broth containing solubilized N-acetylglucosamine. In one aspect, the step of concentrating is conducted at less than atmospheric pressure. In another aspect, the step of concentrating is conducted by membrane separation. In another aspect, the step of concentrating is conducted at a temperature of between about 40° C. and about 75° C. In another aspect, the step of concentrating is conducted at a temperature of between about 45° C. and about 55° C. in another aspect, the step of concentrating is conducted to achieve a solids content in the fermentation broth of at least about 30% solids. In another aspect, the step of concentrating is conducted to achieve a solids content in the fermentation broth of at least about 40% solids. In another aspect, the step of concentrating is conducted to achieve a solids content in the fermentation broth of at least about 45% solids. In another aspect, the method further includes cooling the fermentation broth after the step of concentrating. For example, the fermentation broth can be cooled to between about −5° C. and about 45° C., or in another aspect, to between about −5° C. and about room temperature, or in another aspect, to about room temperature.</p>
    <p num="p-0031">Another aspect of the step of concentrating further includes, after the step of cooling, seeding the fermentation broth with crystals of N-acetylglucosamine. In one aspect, the seed crystals of N-acetylglucosamine are selected from the group consisting of N-acetylglucosamine crystals formed by nucleation in the fermentation broth and externally provided N-acetylglucosamine crystals.</p>
    <p num="p-0032">In any of the various aspects of this embodiment, the step of recovering can include a step of contacting N-acetylglucosamine with a water miscible solvent. A water miscible solvent can include, but is not limited to, isopropyl alcohol (IPA), ethanol, methanol, acetone, tetrahydrofuran, dimethylsulfoxide, dimethylformamide, dioxane and acetonitrile.</p>
    <p num="p-0033">In any of the various aspects of this embodiment, the method can further include a step of drying the recovered N-acetylglucosamine-containing solids. The step can include washing the dried N-acetylglucosamine-containing solids with a water miscible solvent.</p>
    <p num="p-0034">In any of the various aspects of this embodiment, the method can include dissolving the recovered N-acetylglucosamine-containing solids to form an N-acetylglucosamine solution and recovering N-acetylglucosamine-containing solids from the solution.</p>
    <p num="p-0035">In any of the various aspects of this embodiment, the method can further include filtration of the fermentation broth to remove bacterial endotoxins.</p>
    <p num="p-0036">Yet another embodiment of the present-invention relates to a method to produce glucosamine from a source of N-acetylglucosamine, comprising: (a) obtaining a source of N-acetylglucosamine selected from the group consisting of: N-acetylglucosamine, N-acetylglucosamine-6-phosphate and N-acetylglucosamine-1-phosphate; and, (b) treating the source of N-acetylglucosamine of (a) to produce a glucosamine product selected from the group consisting of: glucosamine, glucosamine-6-phosphate and glucosamine-1-phosphate, from the source of N-acetylglucosamine. In one aspect, the source of N-acetylglucosamine is at least about 40% N-acetylglucosamine as a percentage of dry solids in the source. In another aspect, the source of N-acetylglucosamine is N-acetylglucosamine that has been produced by a fermentation process. In yet another aspect, the source of N-acetylglucosamine is a fermentation broth containing N-acetylglucosamine that was produced by a fermentation process, wherein the fermentation broth has been treated to substantially remove cellular material. In another aspect, the source of N-acetylglucosamine is provided as a solid or in a solution. In another aspect, the source of N-acetylglucosamine is suspended in an aqueous, low-boiling, primary or secondary alcohol.</p>
    <p num="p-0037">In one aspect of this embodiment, step (b) of treating comprises hydrolyzing the source of N-acetylglucosamine under acid and heat conditions. In one aspect, the step of hydrolyzing is performed at a temperature of from about 60° C. to about 100° C. In another aspect, the step of hydrolyzing is performed at a temperature of from about 70° C. to about 90° C. In another aspect, the step of hydrolyzing is performed using a hydrochloric solution at a concentration of from about 10% by weight to about 40% weight by weight. In another aspect, the ratio of the weight of hydrochloric acid solution to the source of N-acetylglucosamine as a pure dry weight is from about 1:1 by weight to about 5:1 by weight. In another aspect, the step of hydrolyzing is performed for from about 10 minutes to about 24 hours.</p>
    <p num="p-0038">In one aspect, the step of hydrolyzing comprises: (a) hydrolyzing the source of N-acetylglucosamine by combining the source of N-acetylglucosamine with a hydrochloric acid solution or a recycled hydrolysis mother liquor under heat conditions to produce a solution containing glucosamine hydrochloride; (b) cooling the solution of (a) to precipitate the glucosamine hydrochloride; and (c) recovering the precipitated glucosamine hydrochloride-containing solids from (b). In one aspect, the step (a) of hydrolyzing is performed by continuously blending the source of N-acetylglucosamine with a hydrochloric acid solution or a recycled hydrolysis mother liquor to maintain the source of N-acetylglucosamine as a dissolved solution, followed by addition of anhydrous hydrochloric acid under heat conditions to the solution of (a) to initiate hydrolysis and convert the N-acetylglucosamine to glucosamine hydrochloride. In another aspect, the hydrolysis mother liquor is hydrolysis solution that remains after recovering the precipitated glucosamine hydrochloride in step (c), wherein a primary or secondary alcohol is added to the hydrolysis solution prior to, during or after a hydrolysis step is performed. Such a primary or secondary alcohol can include, but is not limited to, methanol, isopropanol, ethanol, n-propanol, n-butanol and sec-butanol. In one aspect, the step of cooling is performed until the solution is from about −5° C. to about 40° C.</p>
    <p num="p-0039">In another aspect, the step of recovering comprises: (i) collecting the precipitated glucosamine hydrochloride-containing solids; (ii) washing the glucosamine hydrochloride-containing solids with a water miscible solvent; and (iii) drying the glucosamine hydrochloride-containing solids.</p>
    <p num="p-0040">In yet another aspect, the step of recovering comprises: (i) collecting the precipitated glucosamine hydrochloride-containing solids; (ii) dissolving the solids from (i) in water to form a solution; (iii) adjusting the pH of the solution of (ii) to between about 2.5 and 4; (iv) contacting the solution of (iii) with activated carbon to decolorize the glucosamine hydrochloride-containing solids; (v) removing the activated carbon from the solution of (iv); (vi) crystallizing glucosamine hydrochloride from the solution of (v). In this aspect, the step of crystallizing comprises concentrating the glucosamine hydrochloride at a temperature of less than about 70° C. In one aspect, the step of crystallizing comprises concentrating the glucosamine hydrochloride at a temperature of less than about 50° C. In another aspect, step of crystallizing comprises concentrating the glucosamine hydrochloride at less than atmospheric pressure. In another aspect, the method further includes recycling solution remaining after the crystallization step (vi) to step (i) of a subsequent recovery process. In another aspect, the method further includes recycling solution remaining after the crystallization step (vi) to a subsequent step of crystallization. In this aspect, the method can include washing the crystallized glucosamine hydrochloride from step (vi) with a water miscible solvent, including, but not limited to, methanol, isopropanol, ethanol, acetonitrile, acetone, tetrahydrofuran, dimethylsulfoxide, dimethylformamide and dioxane. In one aspect, the method further includes drying the crystallized glucosamine hydrochloride after washing at a temperature of less than about 70° C. for less than about 6 hours. The step of drying can be conducted at less than atmospheric pressure. In one aspect, the step of drying is conducted with an air sweep.</p>
    <p num="p-0041">In another aspect of this embodiment, the source of N-acetylglucosamine is suspended in an aqueous, low-boiling, primary or secondary alcohol, and wherein the method comprises an additional step, between steps (a) and (b) of removing the acetic acid ester formed with the alcohol following hydrolysis or prior to recycling the hydrolysis solution for reuse. In one aspect, the acetic acid ester is removed by a process selected from the group consisting of: distillation, flashing, and concentration at less than atmospheric pressure. In another aspect, the step of hydrolyzing is performed at a temperature of between about 60° C. and about 100° C. In another aspect, the step of hydrolyzing is performed at the solution boiling point at one atmosphere. In another aspect, a step of hydrolyzing is performed at a ratio of hydrochloric acid solution to the source of N-acetylglucosamine as a dry weight of from about 3:1 by weight to about 5:1 by weight, and at a temperature of less than about 80° C. In one aspect, the method further includes the steps of washing the glucosamine hydrochloride recovered in step (c) with a water miscible solvent and/or drying the crystallized glucosamine hydrochloride as discussed above.</p>
    <p num="p-0042">In yet another aspect of this embodiment of the invention, step (b) of treating comprises contacting the source of N-acetylglucosamine with a deacetylating enzyme to produce the glucosamine product. Such a deacetylating enzyme can include, but is not limited to, N-acetylglucosamine-6-P deacetylase, N-acetylglucosamine deacetylase, and/or a chitin deacetylase that has been modified to or selected for its ability to deacetylate an N-acetylglucosamine monomer to produce glucosamine. In one aspect, the glucosamine product by crystallization. In one aspect, the method includes recovering the glucosamine product by precipitation. In one aspect, the deacetylating enzyme is immobilized on a substrate. In one aspect, the step of contacting comprises contacting the source of N-acetylglucosamine with the deacetylating enzyme in the presence of an aqueous sodium or calcium chloride solution. In this aspect, the method can further include recovering the glucosamine product by crystallization or precipitation. In one aspect, the step of contacting comprises contacting the source of N-acetylglucosamine with the deacetylating enzyme in the presence of an alcohol to esterify the alcohol. In another aspect, the method further includes mixing a salt with the glucosamine product and contacting the mixture with an ion exchange medium, including, but not limited to, a chloride salt, a phosphate, a sulfate, an iodide and a bisulfate.</p>
    <p num="p-0043">Yet another embodiment of the present invention relates to a method to produce glucosamine by fermentation, comprising: (a) culturing in a fermentation medium a microorganism which has been transformed with a recombinant nucleic acid molecule comprising a nucleic acid sequence encoding glucosamine-6-phosphate synthase, wherein expression of the recombinant nucleic acid molecule is controlled by a lactose induction, and wherein the step of culturing comprises: (i) growing the microorganism in the fermentation medium comprising glucose as a carbon source at a pH of from about pH 4.5 to about pH 7 and at a temperature of from about 25° C. to about 37° C.; (ii) inducing transcription of the nucleic acid sequence by addition of lactose to the fermentation medium in the absence of adding additional glucose to the medium; (iii) fermenting the microorganism after step (ii) in the presence of glucose at a pH of from about 4.5 to about 6.7 and at a temperature of from about 25° C. to about 37° C.; and (b) collecting a product produced from the step of culturing which is selected from the group consisting of glucosamine-6-phosphate and glucosamine. In one aspect, a source of trace elements is added to step (iii) of fermenting, including, but not limited to iron. In one aspect, step (ii) comprises growing the microorganism in the fermentation medium comprising glucose as a carbon source at a pH of about pH 6.9. In one aspect, step (iii) comprises fermenting the microorganism after step (ii) in the presence of glucose at a pH of from about 4.5 to about 5. In one aspect, step (iii) comprises fermenting the microorganism after step (ii) in the presence of glucose at a pH of about 6.7.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE FIGURES OF THE INVENTION</heading> <p num="p-0044"> <figref idrefs="DRAWINGS">FIG. 1</figref> is a schematic representation of the pathways for the biosynthesis and metabolism of glucosamine in recombinant <i>Escherichia coli. </i> </p>
      <p num="p-0045"> <figref idrefs="DRAWINGS">FIG. 2</figref> is a schematic representation of the novel modifications to the pathways for the overproduction of glucosamine and/or N-acetylglucosamine in recombinant <i>E. coli. </i> </p>
      <p num="p-0046"> <figref idrefs="DRAWINGS">FIG. 3</figref> is an outline of pathway from glucose to glucosamine and other competing pathways (PGI=phosphoglucoisomerase; PGM=phosphoglucomutase; PFK=phosphofructokinase; zwf=glucose-6-phosphate dehydrogenase; GlmS=glucosamine-6-P synthase; Phosphatase=phosphatase activities involved in hydrolysis of glucosamine-6-phosphate).</p>
      <p num="p-0047"> <figref idrefs="DRAWINGS">FIG. 4</figref> is a line graph showing effects of glucosamine-6-phosphate on activity of the recombinant <i>Bacillus subtilis </i>GlmS enzyme.</p>
      <p num="p-0048"> <figref idrefs="DRAWINGS">FIG. 5</figref> is a graph showing glucosamine synthase activity of various GlmS proteins at low levels of glucosamine-6-phosphate.</p>
      <p num="p-0049"> <figref idrefs="DRAWINGS">FIG. 6</figref> is a graph showing changes of enzyme activities over time in a shake flask culture.</p>
      <p num="p-0050"> <figref idrefs="DRAWINGS">FIG. 7</figref> is a bar graph showing the levels of glucosamine production induced by lactose under different conditions in shake flasks (the lactose inducible strain is 7107-16 and the control strain is 2123-54).</p>
      <p num="p-0051"> <figref idrefs="DRAWINGS">FIG. 8</figref> is a line graph showing effects of trace elements on glucosamine production in cell cultures induced by lactose after an initial growth phase.</p>
      <p num="p-0052"> <figref idrefs="DRAWINGS">FIG. 9</figref> is a line graph showing effects of phosphate levels and iron feed on glucosamine production in fermentors.</p>
      <p num="p-0053"> <figref idrefs="DRAWINGS">FIG. 10</figref> is a line graph showing the growth curves of 7107-18 with 20 g l<sup>−1 </sup>glucosamine at different pHs.</p>
      <p num="p-0054"> <figref idrefs="DRAWINGS">FIG. 11</figref> is a line graph showing glucosamine production with strain 2123-54 in 1-liter fermentors under different environmental conditions.</p>
      <p num="p-0055"> <figref idrefs="DRAWINGS">FIG. 12</figref> is a line graph illustrating glucosamine degradation at different pHs (60 g l<sup>−1 </sup>in M9A-glucose medium).</p>
      <p num="p-0056"> <figref idrefs="DRAWINGS">FIG. 13</figref> is a line graph illustrating the stability of N-acetylglucosamine in M9A medium (without glucose)—stability was tested at two concentrations of N-acetylglucosamine (40 and 80 g l<sup>−1</sup>) and two pHs (5.5 and 7.0).</p>
      <p num="p-0057"> <figref idrefs="DRAWINGS">FIG. 14</figref> is an outline of the alternative pathway for N-acetylglucosamine production.</p>
      <p num="p-0058"> <figref idrefs="DRAWINGS">FIG. 15</figref> is a line graph showing effect of phosphorylated amino sugars on phosphoglucoisomerase activity (Pgi).</p>
      <p num="p-0059"> <figref idrefs="DRAWINGS">FIG. 16</figref> is a line graph showing effect of phosphorylated amino sugars on glucose-6-phosphate-dehydrogenase.</p>
      <p num="p-0060"> <figref idrefs="DRAWINGS">FIG. 17</figref> is a line graph showing levels of enzyme activities during a N-acetylglucosamine fermentation.</p>
      <p num="p-0061"> <figref idrefs="DRAWINGS">FIG. 18</figref> is a line graph showing effects of pentose and gluconate on N-acetylglucosamine production in shake flask experiments (each sugar added to the medium at 10 g l<sup>−1 </sup>in shake flasks).</p>
      <p num="p-0062"> <figref idrefs="DRAWINGS">FIG. 19</figref> is a line graph showing N-acetylglucosamine production by 7107-87#25 with early and later IPTG induction in fermentors.</p>
      <p num="p-0063"> <figref idrefs="DRAWINGS">FIG. 20</figref> is a line graph showing effects of zinc and iron concentrations on N-acetylglucosamine production, and particularly showing the beneficial effect of removing Zinc.</p>
      <p num="p-0064"> <figref idrefs="DRAWINGS">FIG. 21</figref> is a line graph showing acid/heat hydrolysis of N-acetylglucosamine (10 g/l in M9A medium).</p>
      <p num="p-0065"> <figref idrefs="DRAWINGS">FIG. 22</figref> is a line graph showing glucosamine stability under acid/heat hydrolysis conditions (10 g/l in M9A medium).</p>
      <p num="p-0066"> <figref idrefs="DRAWINGS">FIG. 23</figref> is a line graph showing acid hydrolysis of N-acetylglucosamine (20 g/l in M9A medium) using 0.1N hydrochloric acid at different temperatures.</p>
      <p num="p-0067"> <figref idrefs="DRAWINGS">FIG. 24</figref> is a line graph showing N-acetylglucosamine/glucosamine hydrolysis at 90° C.</p>
      <p num="p-0068"> <figref idrefs="DRAWINGS">FIG. 25</figref> is a line graph showing ammonia formation from N-acetylglucosamine/glucosamine hydrolysis at 90° C.</p>
    </description-of-drawings> <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p num="p-0069">The present invention relates to a biosynthetic method for producing glucosamine and N-acetylglucosamine. Such a method includes the fermentation of a genetically modified microorganism to produce glucosamine and/or N-acetylglucosamine. The present invention also relates to genetically modified microorganisms that are useful for producing glucosamine and N-acetylglucosamine. In addition, the present invention relates to methods of recovering N-acetylglucosamine that has been produced by a fermentation process, including methods that result in N-acetylglucosamine of high purity. The present invention also relates to a method to produce glucosamine from N-acetylglucosamine. Prior to the present invention, N-acetylglucosamine was produced by the acetylation of glucosamine, and therefore, it is not believed that a method to produce commercially useful glucosamine from N-acetylglucosamine has been described or was even commercially feasible. The present invention demonstrated for the first time direct production of N-acetylglucosamine through a totally natural biologic process.</p>
    <p num="p-0070">More specifically, the present invention generally relates to methods to produce glucosamine and N-acetylglucosamine by fermentation of a microorganism. The methods include a first step of culturing in a fermentation medium a microorganism having a genetic modification in an amino sugar metabolic pathway which includes: a pathway for converting glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate or N-acetylglucosamine-1-phosphate into other compounds; a pathway for synthesizing glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate or N-acetylglucosamine-1-phosphate; a pathway for transport of glucosamine, glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine, N-acetylglucosamine-6-phosphate or N-acetylglucosamine-1-phosphate out of the microorganism; a pathway for transport of glucosamine or N-acetylglucosamine into the microorganism, and a pathway which competes for substrates involved in the production of glucosamine-6-phosphate, to produce a product which can include intracellular glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine and/or glucosamine, and/or extracellular glucosamine or N-acetylglucosamine from the microorganism. The methods also include a second step of collecting the product by recovering intracellular glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine and/or glucosamine from the microorganism and/or collecting and/or recovering extracellular glucosamine or N-acetylglucosamine from the fermentation medium, which can include steps of recovery and purification (defined and discussed in detail below). N-acetylglucosamine, N-acetylglucosamine-6-phosphate and N-acetylglucosamine-1-phosphate recovered from the fermentation process of the invention can be deacetylated using the methods described herein to produce glucosamine, glucosamine-6-phosphate and glucosamine-1-phosphate.</p>
    <p num="p-0071">The novel methods of the present invention for production of glucosamine and N-acetylglucosamine by fermentation are inexpensive and can produce a yield of glucosamine and N-acetylglucosamine that exceeds the yield of glucosamine and N-acetylglucosamine produced by current methods. In addition, by using a genetically modified microorganism as described herein, the method of the present invention can be easily modified to adapt to particular problems or changing needs relative to the production of glucosamine and N-acetylglucosamine. Furthermore, the process and materials disclosed in the present invention can be used and/or modified by those of skill in the art to produce other amino sugars such as poly-N-acetylglucosamine, poly-glucosamine, galactosamine, mannosamine, N-acetyl galactosamine, N-acetyl mannosamine and their derivatives.</p>
    <p num="p-0072">The amino sugars, N-acetylglucosamine (GlcNAc) and glucosamine (GlcN), are fundamentally important molecules in microorganisms, because they are the precursors for the biosynthesis of major macromolecules, and in particular, glycoconjugates (i.e., macromolecules containing covalently bound oligosaccharide chains). For example, in <i>Escherichia coli</i>, N-acetylglucosamine and glucosamine are precursors for two macromolecules of the cell envelope, peptidoglycan and lipopolysaccharide. Mutations that block the biosynthesis of peptidoglycan or lipopolysaccharide are lethal, resulting in loss of integrity of the cell envelope and ultimately in cell lysis.</p>
    <p num="p-0073">As used herein, the terms glucosamine, D-glucosamine and N-glucosamine can be used interchangeably. Similarly, the terms glucosamine-6-phosphate and N-glucosamine-6-phosphate can be used interchangeably, and the terms glucosamine-1-phosphate and N-glucosamine-1-phosphate can be used interchangeably. Glucosamine, glucosamine-6-phosphate and glucosamine-1-phosphate can be abbreviated as GlcN, GlcN-6-P and GlcN-1-P, respectively. N-acetylglucosamine can also be called 2-acetamido-2-deoxy-D-glucose. N-acetylglucosamine can also be written as N-acetyl glucosamine. Similarly to glucosamine and derivatives, the terms, N-acetylglucosamine, N-acetylglucosamine-6-phosphate and N-acetylglucosamine-1-phosphate can be abbreviated as GlcNAc (or D-GlcNAc), GlcNAc-6-P and GlcNAc-1-P, respectively. N-acetylglucosamine is also abbreviated as NAG.</p>
    <p num="p-0074">Reference is made herein to the enzymes in an amino sugar metabolic pathway related to the production of the above-identified glucosamine and N-acetylglucosamine products. An amino sugar is an amino derivative of a saccharide (e.g., a saccharide having an amino group in place of a hydroxyl group). According to the present invention, an amino sugar metabolic pathway is any biochemical pathway involved in, or affecting, the biosynthesis, anabolism or catabolism of an amino sugar. As used herein, amino sugar metabolic pathways include pathways involved in the transport of amino sugars and their precursors into and out of a cell, and can also include biochemical pathways which compete for substrates involved in the biosynthesis or catabolism of an amino sugar. For example, the immediate precursor to one of the earliest formed amino sugars is fructose-6-phosphate (F-6-P), which, in a biochemical reaction (catalyzed by glucosamine synthase) or in a biochemical reaction with ammonium (catalyzed by glucosamine deaminase) forms glucosamine-6-phosphate. Fructose-6-phosphate is also an intermediate in the glycolysis pathway. Therefore, the glycolytic pathway competes with the glucosamine-6-phosphate biosynthetic pathway by competing for a substrate, fructose-6-phosphate. In addition, glucosamine-6-phosphate can be converted to other amino sugars and form constituents in various macromolecules by a series of biochemical reactions. As such, the fructose-6-phosphate/glucosamine-6-phosphate pathway, the glycolytic pathway, to the extent that it affects the biosynthesis of glucosamine-6-phosphate, and the glucosamine-6-phosphate/macromolecule biosynthesis pathway are all considered to be amino sugar metabolic pathways in the present invention.</p>
    <p num="p-0075">Pathways for glutamine synthesis and metabolism determine the availability of glutamine (amino donor for glucosamine-6-phosphate synthesis) and thus affect the biosynthesis of glucosamine-6-phosphate. Therefore, all these pathways are all considered to be amino sugar metabolic pathways in the present invention. The synthesis of N-acetylglucosamine-6-phosphate from glucosamine-6-phosphate and the synthesis of N-acetylglucosamine-1-phosphate from glucosamine-1-phosphate require a supply of acetyl-CoA, which is also the substrate for the Krebs Cycle. Therefore, the Krebs Cycle competes with N-acetylglucosamine synthesis pathways for a substrate, acetyl-CoA. As such, the pathways for acetyl-CoA synthesis and metabolism affect N-acetylglucosamine biosynthesis and thus are all considered to be amino sugar metabolic pathways in the present invention.</p>
    <p num="p-0076">For a variety of microorganisms, many of the amino sugar metabolic pathways have been elucidated. In particular, pathways for the biosynthesis and catabolism of glucosamine and N-acetylglucosamine and their phosphorylated derivatives have been elucidated in <i>Escherichia coli</i>. These pathways include the multiple transport systems for the utilization of these amino sugars as carbon sources. Genes encoding the enzymes and proteins directly related to the transport, catabolism and biosynthesis of amino sugars in <i>Escherichia coli </i>have been cloned and sequenced. In addition, mutant strains of <i>Escherichia coli </i>blocked in substantially every step of amino sugar metabolism have been isolated.</p>
    <p num="p-0077">The known pathways for amino sugar metabolism for <i>Escherichia coli </i>are illustrated in <figref idrefs="DRAWINGS">FIG. 1</figref>. U.S. Pat. No. 6,372,457 described for the first time a glucosamine-producing microorganism that has glucosamine production capabilities that far exceed the glucosamine production capability of any known wild-type or mutant microorganism. As disclosed in U.S. Pat. No. 6,372,457, some steps of the catabolic pathway were blocked by gene inactivation (indicated by crosses), and the enzyme GlcN-6-P synthase, which catalyzes the synthesis of GlcN 6-P, was over-expressed (indicated by a heavy line) in a recombinant host cell. The present invention provides novel fermentation processes for the production of glucosamine that were not described in U.S. Pat. No. 6,372,457, including genetic modifications of production microorganisms as well as combinations of genetic modifications that were not described in U.S. Pat. No. 6,372,457 for the production of glucosamine. The present invention is also believed to be the first description of genetic modifications of microorganisms and fermentation processes for the production of N-acetylglucosamine.</p>
    <p num="p-0078">As will be discussed in detail below, even though many of the pathways and genes involved in the amino sugar metabolic pathways have been elucidated, until the present invention, it was not known which of the many possible genetic modifications would be necessary to generate a microorganism that can produce commercially significant amounts of N-acetylglucosamine. Moreover, the novel genetic modifications and combinations thereof for the production of glucosamine as described herein had not previously been appreciated. Indeed, some of the genetic modifications for the production of glucosamine described herein are the opposite of what was described in the prior disclosure of a fermentation method for glucosamine production, U.S. Pat. No. 6,372,457, supra.</p>
    <p num="p-0079">The amino sugar metabolic pathways of the microorganism, <i>Escherichia coli</i>, will be addressed as specific embodiments of the present invention. The major aspects of the genetic modifications in the amino sugar metabolic pathways disclosed in the present invention for the overproduction of glucosamine and/or N-acetylglucosamine are illustrated in <figref idrefs="DRAWINGS">FIG. 2</figref>. Referring to <figref idrefs="DRAWINGS">FIG. 2</figref>, the heavy arrows indicate creation of and/or increase in the metabolic flux by genetic engineering which are contemplated by the present invention. Several different approaches are disclosed for the synthesis of N-acetylglucosamine, including, but not limited to, modifications to GNA1 and/or NagB, and further including combinations of modifications to GlmS and GNA1, GlmS and GlmU, NagB and GNA1, and NagB and GlmU, alone or in further combination with other genetic modifications. By way of example, the present invention contemplates the over-expression or deletion of additional genes to optimize glucosamine/N-acetylglucosamine production. Referring to <figref idrefs="DRAWINGS">FIG. 3</figref> and the table below, this figure illustrates the involvement of various other enzymes in the metabolism of glucose and the formation of N-glucosamine, and the table shows various modifications that could be additionally made to the microorganism hosts of the invention in order to optimize glucosamine and/or N-acetylglucosamine production. All of these embodiments are described in detail below. It will be appreciated that other microorganisms have similar amino sugar metabolic pathways, as well as genes and proteins having similar structure and function within such pathways. As such, the principles discussed below with regard to <i>Escherichia coli </i>are applicable to other microorganisms and are expressly encompassed by the present invention.</p>
    <p num="p-0080">
      <tables id="TABLE-US-00001" num="00001"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="77pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="105pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Modifi-</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Target</td>
                <td class="description-td">cation</td>
                <td class="description-td">Objective</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Glutamine</td>
                <td class="description-td">Over-</td>
                <td class="description-td">Increase the pool of glutamine</td>
              </tr> <tr class="description-tr"> <td class="description-td">synthase (glnA)</td>
                <td class="description-td">expression</td>
              </tr> <tr class="description-tr"> <td class="description-td">Phosphoglucoisomerase</td>
                <td class="description-td">Over-</td>
                <td class="description-td">Increase the pool of fructose-6-P</td>
              </tr> <tr class="description-tr"> <td class="description-td">(pgi)</td>
                <td class="description-td">expression</td>
              </tr> <tr class="description-tr"> <td class="description-td">Lactose transporter</td>
                <td class="description-td">Over-</td>
                <td class="description-td">Increase efficacy in lactose uptake</td>
              </tr> <tr class="description-tr"> <td class="description-td">(lacY)</td>
                <td class="description-td">expression</td>
                <td class="description-td">and reduce glucose repression on</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">lactose induction</td>
              </tr> <tr class="description-tr"> <td class="description-td">Glucose-6-P</td>
                <td class="description-td">Over-</td>
                <td class="description-td">Relieve inhibitory effects of</td>
              </tr> <tr class="description-tr"> <td class="description-td">dehydrogenase (zwf)</td>
                <td class="description-td">expression</td>
                <td class="description-td">phosphorylated amino sugars on</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">pentose phosphate pathway</td>
              </tr> <tr class="description-tr"> <td class="description-td">lac operon repressor</td>
                <td class="description-td">Deletion</td>
                <td class="description-td">Delete one of two lacl genes to</td>
              </tr> <tr class="description-tr"> <td class="description-td">(lacl)</td>
                <td class="description-td"> </td>
                <td class="description-td">reduce glucose repression on</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">lactose induction</td>
              </tr> <tr class="description-tr"> <td class="description-td">lac promoter</td>
                <td class="description-td">Replace-</td>
                <td class="description-td">Replace the lac promoter with the</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ment</td>
                <td class="description-td">lacUV5 promoter to alleviate</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">glucose represssion on lactose</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">induction</td>
              </tr> <tr class="description-tr"> <td class="description-td">Glycogen synthesis</td>
                <td class="description-td">Deletion/</td>
                <td class="description-td">Block glycogen synthesis</td>
              </tr> <tr class="description-tr"> <td class="description-td">enzyme (glg operon)</td>
                <td class="description-td">insertion</td>
              </tr> <tr class="description-tr"> <td class="description-td">Galactose operon</td>
                <td class="description-td">Deletion/</td>
                <td class="description-td">Galactose inducible production</td>
              </tr> <tr class="description-tr"> <td class="description-td">(gal)</td>
                <td class="description-td">insertion</td>
              </tr> <tr class="description-tr"> <td class="description-td">Phosphofructokinase</td>
                <td class="description-td">Deletion</td>
                <td class="description-td">Uncouple glucosamine/N-</td>
              </tr> <tr class="description-tr"> <td class="description-td">A (pfkA)</td>
                <td class="description-td"> </td>
                <td class="description-td">acetylglucosamine synthesis from</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">cell energy metabolism and acetate</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">formation</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0081">It is known in the art that the enzymes having the same biological activity may have different names depending on from what organism the enzyme is derived. The following is a general list of alternate names for many of the enzymes referenced herein and specific names of genes encoding such enzymes from some organisms. The enzyme names can be used interchangeably, or as appropriate for a given sequence or organism, although the invention intends to encompass enzymes of a given function from any organism.</p>
    <p num="p-0082">For example, the enzyme generally referred to herein as “glucosamine-6-phosphate synthase” catalyzes the formation of glucosamine-6-phosphate and glutamate from fructose-6-phosphate and glutamine. The enzyme is also known as glucosamine-fructose-6-phosphate aminotransferase (isomerizing); hexosephosphate aminotransferase; D-fructose-6-phosphate amidotransferase; glucosamine-6-phosphate isomerase (glutamine-forming); L-glutamine-fructose-6-phosphate amidotransferase; and GlcN6P synthase. The glucosamine-6-phosphate synthase from <i>E. coli </i>and other bacteria is generally referred to as GlmS. The glucosamine-6-phosphate synthase from yeast and other sources is generally referred to as GFA or GFAT.</p>
    <p num="p-0083">Glucosamine-6-phosphate synthases from a variety of organisms are known in the art and are contemplated for use in the genetic engineering strategies of the present invention. For example, the glucosamine-6-phosphate synthase from <i>Escherichia coli </i>is described herein. The glucosamine-6-phosphate synthase from <i>E. coli </i>has an amino acid sequence represented herein by SEQ ID NO:2, which is encoded by a nucleic acid sequence represented herein by SEQ ID NO:1. Also described herein is the glucosamine-6-phosphate synthase from <i>Bacillus subtilis</i>, which has an amino acid sequence represented herein by SEQ ID NO:16, encoded by a nucleic acid sequence represented herein by SEQ ID NO:15. Also described herein is the glucosamine-6-phosphate synthase from <i>Saccharomyces cerevisiae</i>, which is known in that organism as glucosamine-fructose-6-phosphate aminotransferase (GFA 1), and which has an amino acid sequence represented herein by SEQ ID NO:18, encoded by a nucleic acid sequence represented herein by SEQ ID NO:17. Also described herein is the glucosamine-6-phosphate synthase from <i>Candida albicans</i>, which is also known in that organism as glucosamine-fructose-6-phosphate aminotransferase (GFA 1), and which has an amino acid sequence represented herein by SEQ ID NO:20, encoded by a nucleic acid sequence represented herein by SEQ ID NO:19. Also included in the invention are glucosamine-6-phosphate synthases which have one or more genetic modifications that produce a result chosen from: increased enzymatic activity of glucosamine-6-phosphate synthase; reduced product inhibition of the glucosamine-6-phosphate synthase; and increased affinity of glucosamine-6-phosphate synthase for its substrates. In general, according to the present invention, an increase or a decrease in a given characteristic of a mutant or modified enzyme is made with reference to the same characteristic of a wild-type (i.e., normal, not modified) enzyme from the same organism which is measured or established under the same or equivalent conditions (discussed in more detail below).</p>
    <p num="p-0084">Also described herein are several glucosamine-6-phosphate synthases with genetic modifications that result in a reduced product inhibition of the enzyme activity. Such modifications to a glucosamine-6-phosphate synthase were also described in detail in U.S. Pat. No. 6,372,457, although at least one additional, distinct GlmS mutant having reduced product inhibition is described herein (SEQ ID NO:14). Glucosamine-6-phosphate synthase enzymes from <i>E. coli </i>that have reduced product inhibition have amino acid sequences including, but not limited to, those represented by SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, and SEQ ID NO:14 (encoded by SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, and SEQ ID NO:13, respectively). Moreover, a table describing amino acid positions in SEQ ID NO:14 relative to the wild-type sequence (SEQ ID NO:2) that result in an enzyme with reduced product inhibition is provided as Table 7. SEQ ID NO:4 has the following mutations with respect to SEQ ID NO:2: Ile to Thr at position 4, Ile to Thr at position 272, and Ser to Pro at position 450. SEQ ID NO:6 has the following mutations with respect to SEQ ID NO:2: Ala to Thr at position 39, Arg to Cys at position 250, and Gly to Ser at position 472. SEQ ID NO:8 has the following mutations with respect to SEQ ID NO:2: Leu to Pro at position 469. SEQ ID NOs:10 and 12 each have at least the following mutation with respect to SEQ ID NO:2: Gly to Ser at position 472. A glucosamine-6-phosphate synthase having a mutation at any one or more of the positions listed in Table 7, including any combination of the mutations shown therein, or a glucosamine-6-phosphate synthase having any of the additional mutations represented in any of SEQ ID NOs:4, 6, 8, 10, 12 or 14, including any combination thereof, is contemplated by the present invention. Moreover, homologous modifications can be made in glucosamine-6-phosphate synthases from other microorganisms. Finally, some naturally occurring glucosamine-6-phosphate synthases have less product inhibition than glucosamine-6-phosphate synthases from other microorganisms. For example, the present inventors have demonstrated that the wild-type glucosamine-6-phosphate synthase from <i>Bacillus subtilis </i>exhibits a product resistance very comparable to the <i>E. coli </i>mutant GlmS enzymes (see Examples section).</p>
    <p num="p-0085">The enzyme generally referred to herein as glucosamine-6-phosphate acetyltransferase converts glucosamine-6-phosphate and acetyl-CoA to N-acetylglucosamine-6-phosphate, releasing CoA. The enzyme is also known as glucosamine-phosphate N-acetyltransferase, phosphoglucosamine transacetylase and phosphoglucosamine acetylase. The yeast enzyme is generally referred to as GNA1. Glucosamine-6-phosphate acetyltransferases from a variety of organisms are known in the art and are contemplated for use in the genetic engineering strategies of the present invention. For example, the glucosamine-6-phosphate acetyltransferase from <i>Saccharomyces cerevisiae </i>is described herein. The glucosamine-6-phosphate acetyltransferase from <i>Saccharomyces cerevisiae </i>has an amino acid sequence represented herein by SEQ ID NO:30, which is encoded by a nucleic acid sequence represented herein by SEQ ID NO:29. Also described herein is the glucosamine-6-phosphate acetyltransferase from <i>Candida albicans</i>, which has an amino acid sequence represented herein by SEQ ID NO:32, encoded by a nucleic acid sequence represented herein by SEQ ID NO:31. Also described herein is the glucosamine-6-phosphate acetyltransferase from <i>Arabidopsis thaliana</i>, which has an amino acid sequence represented herein by SEQ ID NO:34, encoded by a nucleic acid sequence represented herein by SEQ ID NO:33. Also included in the invention are glucosamine-6-phosphate acetyltransferases that have a genetic modification that produces a result selected from: increased enzymatic activity of glucosamine-6-phosphate acetyltransferase; overexpression of glucosamine-6-phosphate acetyltransferase by the microorganism; reduced N-acetylglucosamine-6-phosphate product inhibition of the glucosamine-6-phosphate acetyltransferase; and increased affinity of glucosamine-6-phosphate acetyltransferase for glucosamine-6-phosphate.</p>
    <p num="p-0086">The enzyme generally referred to herein as glucosamine-6-phosphate deaminase catalyzes a reversible reaction of glucosamine-6-phosphate and water to form fructose-6-phosphate and ammonium. The enzyme is also known as glucosamine-6-phosphate isomerase; GlcN6P deaminase; phosphoglucosaminisomerase; phosphoglucosamine isomerase; glucosamine phosphate deaminase; 2-amino-2-deoxy-D-glucose-6-phosphate ketol isomerase (deaminating). In <i>E. coli </i>and other bacteria, the enzyme is generally known as NagB. Glucosamine-6-phosphate deaminases from a variety of organisms are known in the art and are contemplated for use in the genetic engineering strategies of the present invention. For example, the glucosamine-6-phosphate deaminase from <i>Escherichia coli </i>is described herein. The glucosamine-6-phosphate deaminase from <i>E. coli </i>has an amino acid sequence represented herein by SEQ ID NO:42, which is encoded by a nucleic acid sequence represented herein by SEQ ID NO:41. Also included in the invention are glucosamine-6-phosphate deaminases that have a genetic modification that produces a result selected from: increased enzymatic activity of glucosamine-6-phosphate deaminase, increased reverse reaction of glucosamine-6-phosphate deaminase to form increased (more) glucosamine-6-phosphate, reduced forward reaction of glucosamine-6-phosphate deaminase to form reduced (less) fructose-6-phosphate, increased affinity of glucosamine-6-phosphate deaminase for fructose-6-phosphate, reduced affinity of glucosamine-6-phosphate deaminase for glucosamine-6-phosphate, and reduced glucosamine-6-phosphate product inhibition of the glucosamine-6-phosphate deaminase.</p>
    <p num="p-0087">The enzyme generally referred to herein as glucosamine-1-phosphate N-acetyltransferase converts glucosamine-1-phosphate and acetyl-CoA to N-acetylglucosamine-1-phosphate, releasing CoA. The enzyme is known in <i>E. coli </i>and other bacteria as GlmU. The bacterial GlmU enzyme is a bifunctional enzyme, (i.e., it has two enzyme functions it has also the function of N-acetylglucosamine-1-phosphate uridyltransferase, which is also known as UDP-N-acetylglucosamine pyrophosphorylase, UDP-N-acetylglucosamine diphosphorylase). Glucosamine-1-phosphate N-acetyltransferases from a variety of organisms are known in the art and are contemplated for use in the genetic engineering strategies of the present invention. For example, the glucosamine-1-phosphate N-acetyltransferase from <i>Escherichia coli</i>, which is actually the bifunctional glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase, is described herein. The glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase from <i>E. coli </i>has an amino acid sequence represented herein by SEQ ID NO:56, which is encoded by a nucleic acid sequence represented herein by SEQ ID NO:55. Also described herein is a truncated mutant of the <i>E. coli </i>glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase, wherein the portion encoding the N-acetylglucosamine-1-phosphate uridyltransferase has been deleted, effectively leaving an enzyme with only the glucosamine-1-phosphate N-acetyltransferase activity. This truncated glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase has an amino acid sequence represented herein by SEQ ID NO:58, encoded by a nucleic acid sequence represented herein by SEQ ID NO:57. Also included in the invention are glucosamine-1-phosphate N-acetyltransferases that have a genetic modification that produces a result selected from: increased enzymatic activity of glucosamine-1-phosphate N-acetyltransferase; reduced N-acetylglucosamine-1-phosphate uridyltransferase enzymatic activity (if the enzyme is a bifunctional enzyme); increased affinity of glucosamine-1-phosphate N-acetyltransferase for glucosamine-1-phosphate; reduced affinity of an glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase for N-acetylglucosamine-1-phosphate (if the enzyme is a bifunctional enzyme); and reduced N-acetylglucosamine-1-phosphate product inhibition of the glucosamine-1-phosphate N-acetyltransferase.</p>
    <p num="p-0088">The enzyme generally referred to herein as N-acetylglucosamine-6-phosphate deacetylase hydrolyzes N-acetylglucosamine-6-phosphate to glucosamine-6-phosphate and acetate. The enzyme is known in <i>E. coli </i>and other bacteria as NagA. N-acetylglucosamine-6-phosphate deacetylases from a variety of organisms are known in the art and are contemplated for use in the genetic engineering strategies of the present invention. For example, the N-acetylglucosamine-6-phosphate deacetylase from <i>Escherichia coli </i>is described herein. The N-acetylglucosamine-6-phosphate deacetylase from <i>E. coli </i>has an amino acid sequence represented herein by SEQ ID NO:84; which is encoded by a nucleic acid sequence represented herein by SEQ ID NO:85. Also included in the invention are N-acetylglucosamine-6-phosphate deacetylases that have a genetic modification that produces a result selected from: increased activity of glucosamine-6-phosphate deacetylase; increased reverse reaction of glucosamine-6-phosphate deacetylase to form increased N-acetyl glucosamine-6-phosphate; reduced forward reaction of glucosamine-6-phosphate deacetylase to form reduced glucosamine-6-phosphate; increased affinity of glucosamine-6-phosphate deacetylase for glucosamine-6-phosphate; reduced affinity of glucosamine-6-phosphate deacetylase for N-acetyl glucosamine-6-phosphate; reduced N-acetyl glucosamine-6-phosphate product inhibition of the glucosamine-6-phosphate deacetylase.</p>
    <p num="p-0089">The enzyme generally referred to herein as phosphoglucosamine mutase catalyzes the conversion between glucosamine-6-phosphate and glucosamine-1-phosphate. The enzyme is generally known in <i>E. coli </i>and other bacteria as GlmM. Phosphoglucosamine mutases from a variety of organisms are known in the art and are contemplated for use in the genetic engineering strategies of the present invention. For example, the phosphoglucosamine mutase from <i>Escherichia coli </i>is described herein. The phosphoglucosamine mutase from <i>E. coli </i>has an amino acid sequence represented herein by SEQ ID NO:54, which is encoded by a nucleic acid sequence represented herein by SEQ ID NO:53. Also included in the invention are phosphoglucosamine mutases that have a genetic modification that produces a result selected from: increased activity of phosphoglucosamine mutase; increased forward reaction of phosphoglucosamine mutase to form increased glucosamine-1-phosphate; reduced reverse reaction of phosphoglucosamine mutase to form reduced glucosamine-6-phosphate; increased affinity of phosphoglucosamine mutase for glucosamine-6-phosphate; reduced affinity of phosphoglucosamine mutase for glucosamine-1-phosphate; and reduced glucosamine-1-phosphate product inhibition of the phosphoglucosamine mutase.</p>
    <p num="p-0090">The enzyme generally referred to herein as phosphoglucoisomerase catalyzes the interconversion of glucose-6-phosphate to fructose-6-phosphate. The enzyme is generally known in <i>E. coli </i>and other bacteria as phosphoglucoisomerase or Pgi. Phosphoglucoisomerases from a variety of organisms are known in the art and are contemplated for use in the genetic engineering strategies of the present invention. For example, the phosphoglucoisomerase from <i>Escherichia coli </i>is described herein. The phosphoglucoisomerase from <i>E. coli </i>has an amino acid sequence represented herein by SEQ ID NO:105, which is encoded by a nucleic acid sequence represented herein by SEQ ID NO:104.</p>
    <p num="p-0091">The enzyme generally referred to herein as phosphofructokinase catalyzes the formation of fructose-1,6,-biphosphate from fructose 6-phosphate (F-6-P). The major phosphofructokinase in <i>E. coli</i>, encoded by pfkA, provides 90% of the phosphofructokinase activity. The remaining 10% of activity is supplied by the minor phosphofructokinase, encoded by pfkB. The phosphofructokinase enzyme is generally known in <i>E. coli </i>and other bacteria as phosphofructokinase or Pfk. Phosphofructokinases from a variety of organisms are known in the art and are contemplated for use in the genetic engineering strategies of the present invention. For example, the PfkA from <i>Escherichia coli </i>is described herein.</p>
    <p num="p-0092">The enzyme generally referred to herein as glutamine synthetase catalyzes the conversion of L-glutamate to L-glutamine in a reaction requiring NH<sub>3 </sub>and ATP. The enzyme is generally known in <i>E. coli </i>and other bacteria as glutamine synthetase, or GlnA. Glutamine synthetases from a variety of organisms are known in the art and are contemplated for use in the genetic engineering strategies of the present invention. For example, the glutamine synthetase from <i>Escherichia coli </i>is described herein. The glutamine synthetase from <i>E. coli </i>has an amino acid sequence represented herein by SEQ ID NO:89, which is encoded by a nucleic acid sequence represented herein by SEQ ID NO:88.</p>
    <p num="p-0093">The enzyme generally referred to herein as glucose-6-phosphate dehydrogenase catalyzes the first step in the pentose phosphate pathway, converting glucose-6-phosphate into glucono-1,5-lactone. The enzyme is generally known in <i>E. coli </i>and other bacteria as glucose-6-phosphate dehydrogenase, encoded by zwf. Glucose-6-phosphate dehydrogenases from a variety of organisms are known in the art and are contemplated for use in the genetic engineering strategies of the present invention. For example, the glucose-6-phosphate dehydrogenase from <i>Escherichia coli </i>is described herein. The glucose-6-phosphate dehydrogenase from <i>E. coli </i>has an amino acid sequence represented herein by SEQ ID NO:95, which is encoded by a nucleic acid sequence represented herein by SEQ ID NO:94.</p>
    <p num="p-0094">A number of enzymes are responsible for glycogen synthesis in a microorganism. Such enzymes include, but are not limited to, ADP-glucose pyrophosphorylase, glycogen synthase and a branching enzyme.</p>
    <p num="p-0095">Other genes referenced herein include, but are not limited to: N-acetylglucosamine transporter (II<sup>Nag</sup>), mannose transporter (EIIM,P/II<sup>man</sup>), glucose transporter (II<sup>Glc</sup>), and/or a phosphatase. The genes in <i>E. coli </i>correspond to: nagC, nagD, nagE (N-acetylglucosamine transporter (II<sup>Nag</sup>)); manXYZ (mannose transporter (EIIM,P/III<sup>man</sup>)); ptsG (glucose transporter (II<sup>Glc</sup>)) or a phosphatase gene, respectively. The nagC gene encodes a regulatory protein that acts as a repressor of the nag regulon as well as both an activator and repressor of the glmU operon. The function of the nagD gene is not known, but is believed to be related to amino sugar metabolism as it resides within the nag regulon. The nag genes (nagA, nagB, nagC, nagD, nagE) involved in the degradation of glucosamine and N-acetyl-glucosamine exist as a regulon (i.e., the nag regulon) located at 15 min on the <i>Escherichia coli </i>chromosome. The mannose transporter (EIIM,P/II<sup>man</sup>) (manXYZ) is responsible for the transport of glucosamine into the cell. The glucose transporter (II<sup>Glc</sup>) (ptsG) transports glucose into the cell but can serve as a secondary transporter for glucosamine. Phosphatases are well known in the art as catalyzing the dephosphorylation of sugar phosphates and proteins.</p>
    <p num="p-0096">One embodiment of the present invention relates to a method to produce glucosamine and/or N-acetylglucosamine by fermentation. Such a method generally includes the steps of (a) culturing in a fermentation medium a microorganism which comprises at least one specific genetic modification that is disclosed herein as being useful for increasing the production of glucosamine and/or N-acetylglucosamine in the microorganism; and (b) collecting a product produced from the step of culturing which is selected from the group consisting of glucosamine-6-phosphate, glucosamine, glucosamine-1-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, and N-acetylglucosamine. More particularly, the products can include intracellular glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine and/or glucosamine which are collected from the microorganism and/or extracellular glucosamine or N-acetylglucosamine which are collected from the fermentation medium. In one aspect, N-acetylglucosamine, N-acetylglucosamine-6-phosphate and N-acetylglucosamine-1-phosphate are hydrolyzed under acid/heat conditions where the hydrolysis products (glucosamine, glucosamine-6-phosphate and glucosamine-1-phosphate) are stable. In another aspect, N-acetylglucosamine, N-acetylglucosamine-6-phosphate and N-acetylglucosamine-1-phosphate are deacetylated by using a deacetylating enzyme. Recovery and purification methods are discussed in detail below.</p>
    <p num="p-0097">In general, a genetically modified microorganism useful in a method of the present invention typically has at least one modified gene involved in at least one amino sugar metabolic pathway which results in (a) reduced ability to convert glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate into other compounds (i.e., inhibition of glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate catabolic or anabolic pathways), (b) an enhanced ability to produce (i.e., synthesize) glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate, (c) a reduced ability to transport glucosamine and/or N-acetylglucosamine into the cell, (d) an enhanced ability to transport glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate, glucosamine and/or N-acetylglucosamine out of the cell, and/or (e) a reduced ability to use substrates involved in the production of glucosamine-6-P for competing biochemical reactions, and/or (f) a reduced ability to use acetyl-CoA (involved in the production of N-acetylglucosamine-6-phosphate and N-acetylglucosamine-1-phosphate) for competing biochemical reactions.</p>
    <p num="p-0098">In general, a microorganism having a genetically modified amino sugar metabolic pathway has at least one genetic modification, as discussed in detail below, which results in a change in one or more amino sugar metabolic pathways as described above as compared to a wild-type microorganism cultured under the same conditions. Such a modification in an amino sugar metabolic pathway changes the ability of the microorganism to produce an amino sugar. As discussed in detail below, according to the present invention, a genetically modified microorganism preferably has an enhanced ability to produce glucosamine and/or N-acetylglucosamine as compared to a wild-type microorganism of the same species (and preferably the same strain), which is cultured under the same or equivalent conditions. Equivalent conditions are culture conditions which are similar, but not necessarily identical (e.g., some changes in medium composition, temperature, pH and other conditions can be tolerated), and which do not substantially change the effect on microbe growth or production of glucosamine or N-acetylglucosamine by the microbe.</p>
    <p num="p-0099">An amino sugar metabolic pathway which affects the production of glucosamine and/or N-acetylglucosamine can generally be categorized into at least one of the following kinds of pathways: (a) pathways for converting glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and N-acetylglucosamine-1-phosphate into other compounds, (b) pathways for synthesizing glucosamine-6-phosphate, (c) pathways for transporting glucosamine and/or N-acetylglucosamine into a cell, (d) pathways for transporting glucosamine, glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate and/or N-acetylglucosamine out of a cell, and (e) pathways which compete for substrates involved in the production of glucosamine-6-phosphate.</p>
    <p num="p-0100">Development of a microorganism with enhanced ability to produce glucosamine and/or N-acetylglucosamine by genetic modification can be accomplished using both classical strain development and molecular genetic techniques. In general, the strategy for creating a microorganism with enhanced glucosamine and/or N-acetylglucosamine production is to (1) inactivate or delete at least one, and preferably more than one of the amino sugar metabolic pathways in which production of glucosamine-6-phosphate is negatively affected (e.g., inhibited), and (2) amplify at least one, and preferably more than one of the amino sugar metabolic pathways in which glucosamine-6-phosphate production is enhanced.</p>
    <p num="p-0101">In one embodiment of the present invention, enhancement of the ability of a microorganism to synthesize glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate can be accomplished by amplification of the expression (e.g., overexpression) of the glucosamine-6-phosphate synthase gene (glmS), and/or by amplification of the expression of the glucosamine-6-phosphate deaminase gene, which in <i>Escherichia coli </i>is the nagB gene, the product of which is glucosamine-6-phosphate deaminase. Glucosamine-6-phosphate deaminase catalyzes the forward reaction in which glucosamine-6-phosphate is deaminated to form fructose-6-phosphate and ammonium. Glucosamine-6-phosphate deaminase also catalyzes the reverse reaction in which fructose-6-phosphate and ammonium form glucosamine-6-phosphate. The reverse reaction of the glucosamine-6-phosphate deaminase is different from the action of glucosamine-6-phosphate synthase because in the synthase reaction fructose-6-phosphate and glutamine form glucosamine-6-phosphate and glutamic acid. An adequate intracellular supply of glutamine is critical for the glucosamine-6-phosphate synthase reaction. Inspection of the synthetic and degradative pathways for glucosamine-6-phosphate reveals the presence of a potential futile cycle whereby continuous interconversion of fructose-6-phosphate and glucosamine-6-phosphate results in wasteful depletion of glutamine. Therefore, use of the reverse action of the glucosamine-6-phosphate deaminase has an advantage over the glucosamine-6-phosphate synthase since the deaminase uses ammonium as the amino donor rather than an amino acid (glutamine). The glucosamine-6-phosphate deaminase catalyzes the reversible reaction with a kinetic equilibrium in favor of the degradation of glucosamine-6-phosphate into fructose-6-phosphate and ammonium. Therefore, as another embodiment of the present invention, a mutagenized form of glucosamine-6-phosphate deaminase or an over-expressed glucosamine-6-phosphate deaminase, with an increased activity of the reverse action and a decreased activity of the forward reaction is used for production of glucosamine and/or N-acetylglucosamine. Another embodiment of the present invention is to provide a microorganism having a glucosamine-6-phosphate deaminase with increased Vmax, increased specific activity, increased stability, increased affinities to substrates fructose-6-phosphate and ammonium, and reduced product inhibition by glucosamine-6-phoshat. A glucosamine-6-phosphate deaminase with such improvements can be isolated from nature or produced by any suitable method of genetic modification or protein engineering. For example, computer-based protein engineering can be used for this purpose. See for example, Maulik et al., 1997<i>, Molecular Biotechnology: Therapeutic Applications and Strategies</i>, Wiley-Liss, Inc., which is incorporated herein by reference in its entirety. Amplification of the expression of glucosamine-6-phosphate deaminase can be accomplished in <i>Escherichia coli</i>, for example, by introduction of a recombinant nucleic acid molecule encoding the nagB gene. Since the glucosamine-6-phosphate synthase in the host strain can catalyze the synthesis of glucosamine-6-phosphate, the amplification of the expression of glucosamine-6-phosphate deaminase should be analyzed in a mutant <i>Escherichia coli </i>strain which has a inactivated glmS gene. While elimination of glucosamine-6-phosphate synthase activity in this organism is not required, it is one embodiment of the invention.</p>
    <p num="p-0102">By way of example for other enzymes and other proteins described herein, a modified glucosamine-6-phosphate deaminase can be a mutated (i.e., genetically modified) glucosamine-6-phosphate deaminase gene, for example, and can be produced by any suitable method of genetic modification. For example, a recombinant nucleic acid molecule encoding glucosamine-6-phosphate deaminase can be modified by any method for inserting, deleting, and/or substituting nucleotides, such as by error-prone PCR. In this method, the gene is amplified under conditions that lead to a high frequency of misincorporation errors by the DNA polymerase used for the amplification. As a result, a high frequency of mutations are obtained in the PCR products. The resulting glucosamine-6-phosphate deaminase gene mutants can then be screened for by testing the mutant genes for the ability to confer increased glucosamine production onto a test microorganism, as compared to a microorganism carrying the non-mutated recombinant glucosamine-6-phosphate deaminase nucleic acid molecule. Therefore, it is an embodiment of the present invention to provide a microorganism which is transformed with a genetically modified recombinant nucleic acid molecule comprising a nucleic acid sequence encoding a mutant, or homologue, glucosamine-6-phosphate deaminase protein. Such glucosamine-6-phosphate deaminase proteins can be referred to herein as glucosamine-6-phosphate deaminase homologues (described in detail below).</p>
    <p num="p-0103">In one aspect of the invention, overexpression of either glmS or nagB is crucial for the intracellular accumulation of glucosamine-6-phosphate and ultimately for production of glucosamine and/or N-acetylglucosamine, since the level of glucosamine-6-phosphate synthase/or glucosamine-6-phosphate deaminase in the cell will control the redirection of carbon flow away from glycolysis and into glucosamine-6-phosphate synthesis.</p>
    <p num="p-0104">For production of N-acetylglucosamine, glucosamine-6-phosphate must be converted to N-acetylglucosamine-6-phosphate or N-acetylglucosamine-1-phosphate, which is then dephosphorylated and/or secreted into the culture broth. In one embodiment of the present invention, enhancement of the ability of a microorganism to synthesize N-acetylglucosamine-6-phosphate and N-acetylglucosamine is accomplished by amplification of the expression of the glucose-6-phosphate acetyltransferase gene, which, by way of example only, in <i>Saccharomyces cerevisiae </i>is the GNA1 gene, the product of which is glucosamine-6-phosphate acetyltransferase. In another embodiment of the present invention, enhancement of the ability of a microorganism to synthesize N-acetylglucosamine-6-phosphate and N-acetylglucosamine is accomplished by amplification of the expression of the N-acetyl glucose-6-phosphate deacetylase gene, which in <i>Escherichia coli </i>is the nagA gene, the product of which is N-acetylglucosamine-6-phosphate deacetylase. The deacetylase catalyzes the conversion of N-acetylglucosamine-6-phosphate to glucosamine-6-phosphate in the forward reaction. It also catalyzes the reverse reaction to convert glucosamine-6-phosphate to N-acetylglucosamine. N-acetylglucosamine-6-phosphate deacetylase catalyzes the reversible reaction with a kinetic equilibrium in favor of the deacetylation of N-acetylglucosamine-6-phosphate to glucosamine-6-phosphate. Therefore, as another embodiment of the present invention, a mutagenized form of N-acetylglucosamine-6-phosphate deacetylase with an increased activity of the reverse activity of the reverse action and a decreased activity of the forward reaction is used for production of glucosamine and/or N-acetylglucosamine.</p>
    <p num="p-0105">Since the glucosamine-6-phosphate synthase (GlmS) is strongly inhibited by the product glucosamine-6-phosphate, amplification of the expression of the glucosamine-6-phosphate acetyltransferase gene (GNA1) and/or N-acetylglucosamine-6-phosphate deacetylase gene (nagA) can reduce the intracellular level of glucosamine-6-phosphate, reduce the product inhibition of GlmS enzyme, and thus increase production of glucosamine and/or N-acetylglucosamine.</p>
    <p num="p-0106">In another embodiment of the present invention, enhancement of the ability of a microorganism to synthesize glucosamine and N-acetylglucosamine is accomplished by amplification of the expression of the phosphoglucosamine mutase gene, which in <i>Escherichia coli </i>is the glmM gene, the product of which is phosphoglucosamine mutase. The phosphoglucosamine mutase catalyzes the conversion of glucosamine-6-phosphate to glucosamine-1-phosphate. Since the glucosamine-6-phosphate synthase (GlmS) is strongly inhibited by the product glucosamine-6-phosphate, amplification of the expression of the phosphoglucosamine mutase gene can reduce the intracellular level of glucosamine-6-phosphate, reduce the product inhibition of GlmS enzyme, and thus increase production of glucosamine and/or N-acetylglucosamine.</p>
    <p num="p-0107">In another embodiment of the present invention, enhancement of the ability of a microorganism to synthesize glucosamine and/or N-acetylglucosamine is accomplished by amplification of the expression of the bifunctional glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase gene, which in <i>Escherichia coli </i>is the glmU gene, the product of which is glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase. The invention also includes the amplification of expression or increased activity of a protein having glucosamine-1-phosphate N-acetyltransferase activity. The bi-functional enzyme catalyzes the conversion of glucosamine-1-phosphate to N-acetylglucosamine-1-phosphate (as glucosamine-1-phosphate N-acetyltransferase) and converts the product further to UDP-N-acetylglucosamine (as N-acetylglucosamine-1-phosphate uridyltransferase). Therefore, as another embodiment of the present invention, a mutagenized form of glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase with an increased activity of the acetyltransferase action and a decreased activity of the uridyltransferase action is used for production of glucosamine and/or N-acetylglucosamine. The glmU gene is an essential gene for <i>Escherichia coli </i>growth since glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase functions -phosphate within the amino sugar metabolic pathway in which glucosamine-6-phosphate is incorporated, through a series of biochemical reactions, into macromolecules. A mutagenized form of GlmU enzyme with an increased activity of the acetyltransferase action and a decreased activity of the uridyltransferase action needs to be used in a host strain with a wild-type glmU gene so that macro molecules derived from glucosamine can be synthesized to support cell growth.</p>
    <p num="p-0108">Glucosamine and/or N-acetylglucosamine synthesis is closely connected to many different pathways of glucose metabolism as outlined in <figref idrefs="DRAWINGS">FIG. 3</figref>. Glucose is taken up by the cell and is simultaneously converted to glucose-6-P. Glucose is metabolized by number of pathways, including those shown in the figure. In the glucosamine synthesis pathway, glucose-6-P is isomerized to fructose-6-phosphate, followed by the GlmS mediated conversion of fructose-6-phosphate to glucosamine-6-phosphate. Finally, glucosamine-6-phosphate is dephosphorylated and secreted. A major competing alternative route for glucose-6-phosphate is its entry into glycolysis via phosphofructokinase. Another important alternate routes for glucose-6-phosphate is its oxidation to gluconolactone-6-phosphate (the entry into the pentose phosphate pathway). Additionally, glucose-6-phosphate could be converted to glucose-1-phosphate, from which glycogen is made and stored in the cell. It is in the scope of the present invention to modulate other competing pathways to maximize the production of glucosamine and/or N-acetylglucosamine as described in detail below.</p>
    <p num="p-0109">Bacterial cells accumulate glycogen as the major form of stored carbon reserve. Glycogen synthesis involves three enzymes: ADP-glucose pyrophosphorylase, also known as glucose-1-phosphate adenyltransferase), glycogen synthase, and a branching enzyme. These enzymes catalyze, respectively, the synthesis of the monosaccharide donor (ADP-glucose) from glucose-1-phosphate, the polymerization of these monosaccharide units to form a (1,4) polymer of glucose, and the rearrangement of this polymer to generate (1-6) branches in the chain. ADP-glucose pyrophosphorylase is a pivotal enzyme in glycogen synthesis and is strongly modulated by allosteric effectors. For example, 3-phosphoglycerate stimulates enzyme activity, while orthophosphate inhibits the activity. These effectors may play key roles in vivo in the control of glycogen synthesis. The amount of glycogen in the cells accounts for 10 to 60% of their dry weight according to growth conditions. Generally, glycogen accumulates under conditions where the supply of carbon and energy was plentiful and nitrogen was the limiting nutrient. A depletion of phosphate in the growth medium also results in a higher amount of glycogen in the cell. The sole function of ADP-glucose in bacteria is to serve as a precursor for glycogen synthesis. In <i>E. coli </i>mutants with glycogen synthesis blocked cell growth was not detrimentally affected. Blocking glycogen synthesis may make more carbon source available to glucosamine/N-acetylglucosamine production.</p>
    <p num="p-0110">To maximize glucosamine/N-acetylglucosamine production the phosphoglucoisomerase (encoded by the pgi gene) could be manipulated. Overexpression of the pgi gene could increase the conversion of glucose-6-P to fructose-6-P the direct substrate for the synthesis of glucosamine-6-P.</p>
    <p num="p-0111">Glucose is needed to support cell growth and glucosamine synthesis. Phosphofructokinases (PfkA and PfkB, with the former being the major isoenzyme) catalyze the conversion of fructose-6-P to fructose-1,6-biphosphate. The reaction operates in competition to glucosamine-6-P synthesis. In controlling carbon flux, it would be desirable to modulate expression of PfkA and/or PfkB to make more glucose-6-P directed to the glucosamine pathway. To maximize glucosamine synthesis, the supply of glucose should not be limited, but excess glucose usually leads to acetate formation. Since acetate accumulation inhibits cell growth and signals a down turn of the general cellular metabolism, it should be avoided. One approach to managing the carbon supply is to uncouple the carbon flux to glucosamine synthesis from that of cell growth and acetate formation. This could be accomplished by deleting the pfkA gene and supplying cells with fructose for growth. In pfkA knockout mutants the flux of fructose-6-phosphate to glycolysis will be greatly restricted. Therefore, virtually all of the glucose can be used for glucosamine synthesis while cell growth and energy production will be supplied by feeding fructose, which is transported into the cell and phosphorylated to fructose-1-phosphate. The later will be further phosphorylated to fructose-1,6-diphosphate.</p>
    <p num="p-0112">Literature suggests that phosphorylated amino sugars were inhibitory to enzymes of the pentose phosphate pathway. Indeed, N-acetylglucosamine production caused cell growth inhibition that could be relieved by feeding intermediates of the pathway such as gluconate and ribose. Supplementation with gluconate and pentose compounds was also found to increase N-acetylglucosamine production. To overcome inhibition by amino sugars, enzymes of the pentose phosphate pathway, such as glucose-6-P dehydrogenase (encode by the zwf) gene) can be overexpressed.</p>
    <p num="p-0113">Glucosamine/N-acetylglucosamine synthesis consumes glutamine, the amino donor. The synthesis of glutamine involves glutamine synthase encoded by the glnA gene. Overexpression of the glnA gene could increase the pool of glutamine and thus increase glucosamine/N-acetylglucosamine.</p>
    <p num="p-0114">Therefore, having generally described some of the preferred modifications according to the invention, in one embodiment, the microorganism comprises at least one genetic modification that increases the activity of glucosamine-6-phosphate acetyltransferase in the microorganism. Preferably, the genetic modification to increase the activity of glucosamine-6-phosphate acetyltransferase provides a result selected from: increased enzymatic activity of glucosamine-6-phosphate acetyltransferase; overexpression of glucosamine-6-phosphate acetyltransferase by the microorganism; reduced N-acetyl glucosamine-6-phosphate product inhibition of the glucosamine-6-phosphate acetyltransferase; and/or increased affinity of glucosamine-6-phosphate acetyltransferase for glucosamine-6-phosphate. In one aspect, the microorganism is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the glucosamine-6-phosphate acetyltransferase. Such a nucleic acid molecule can include a nucleic acid sequence encoding a glucosamine-6-phosphate acetyltransferase that has at least one genetic modification which increases the enzymatic activity of the glucosamine-6-phosphate acetyltransferase, or produces any of the above-described results. The function and representative sequences for glucosamine-6-phosphate acetyltransferases have been described above.</p>
    <p num="p-0115">In another embodiment, the microorganism comprises at least one genetic modification that increases the activity of glucosamine-6-phosphate synthase. Preferably, the genetic modification to increase the activity of the glucosamine-6-phosphate synthase produces a result selected from: increased enzymatic activity of glucosamine-6-phosphate synthase; overexpression of the glucosamine-6-phosphate synthase; reduced product inhibition of the glucosamine-6-phosphate synthase; and increased affinity of glucosamine-6-phosphate synthase for its substrates. In one aspect, the microorganism is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the glucosamine-6-phosphate synthase. Such a nucleic acid molecule can include a nucleic acid sequence encoding a glucosamine-6-phosphate synthase that has at least one genetic modification which increases the enzymatic activity of the glucosamine-6-phosphate synthase, that reduces the product inhibition of the glucosamine-6-phosphate synthase, or produces any of the above-described results. The function and representative sequences for glucosamine-6-phosphate synthases have been described above.</p>
    <p num="p-0116">In another embodiment, the microorganism comprises at least one genetic modification that decreases the activity of glucosamine-6-phosphate synthase. In one aspect, the genetic modification to decrease the activity of glucosamine-6-phosphate synthase is a partial or complete deletion or inactivation of an endogenous gene encoding glucosamine-6-phosphate synthase in the microorganism.</p>
    <p num="p-0117">In yet another embodiment, the microorganism comprises at least one genetic modification that increases the activity of glucosamine-6-phosphate deaminase. Preferably, the genetic modification to increase the activity of the glucosamine-6-phosphate deaminase produces a result selected from: overexpression of glucosamine-6-phosphate deaminase by the microorganism, increased enzymatic activity of glucosamine-6-phosphate deaminase, increased reverse reaction of glucosamine-6-phosphate deaminase to form increased (more) glucosamine-6-phosphate, reduced forward reaction of glucosamine-6-phosphate deaminase to form reduced (less) fructose-6-phosphate, increased affinity of glucosamine-6-phosphate deaminase for fructose-6-phosphate, reduced affinity of glucosamine-6-phosphate deaminase for glucosamine-6-phosphate, and reduced glucosamine-6-phosphate product inhibition of the glucosamine-6-phosphate deaminase. In one aspect, the microorganism is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the glucosamine-6-phosphate deaminase. Such a nucleic acid molecule can include a nucleic acid sequence encoding a glucosamine-6-phosphate deaminase that has at least one genetic modification which increases the enzymatic activity of the glucosamine-6-phosphate deaminase, or produces any of the above-described results. The function and representative sequences for glucosamine-6-phosphate deaminases have been described above.</p>
    <p num="p-0118">In another embodiment, the microorganism comprises at least one genetic modification that decreases the activity of glucosamine-6-phosphate deaminase. In one aspect, the genetic modification to decrease the activity of glucosamine-6-phosphate deaminase is a partial or complete deletion or inactivation of an endogenous gene encoding glucosamine-6-phosphate deaminase in the microorganism.</p>
    <p num="p-0119">In yet another embodiment, the microorganism comprises at least one genetic modification that increases the activity of glucosamine-1-phosphate N-acetyltransferase. Preferably, the genetic modification provides a result selected from: increased enzymatic activity of glucosamine-1-phosphate N-acetyltransferase; reduced N-acetyl glucosamine-1-phosphate uridyltransferase enzymatic activity; overexpression of an enzyme having glucosamine-1-phosphate N-acetyltransferase activity by the microorganism; increased affinity of glucosamine-1-phosphate N-acetyltransferase for glucosamine-1-phosphate; reduced affinity of an glucosamine-1-phosphate N-acetyltransferase/N-acetyl glucosamine-1-phosphate uridyltransferase for N-acetyl glucosamine-1-phosphate; and/or reduced N-acetyl glucosamine-1-phosphate product inhibition of the glucosamine-1-phosphate N-acetyltransferase. In one aspect, the microorganism is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the glucosamine-1-phosphate N-acetyltransferase or an enzyme comprising the activity of a glucosamine-1-phosphate N-acetyltransferase. Such a nucleic acid molecule can include a nucleic acid sequence encoding a glucosamine-1-phosphate N-acetyltransferase that has at least one genetic modification which increases the enzymatic activity of the glucosamine-1-phosphate N-acetyltransferase, or produces any of the above-described results. The function and representative sequences for glucosamine-1-phosphate N-acetyltransferases have been described above.</p>
    <p num="p-0120">In another embodiment, the genetically modified microorganism comprises at least one genetic modification that increases the activity of N-acetylglucosamine-6-phosphate deacetylase. Preferably, the genetic modification that increases the activity of N-acetylglucosamine-6-phosphate deacetylase produces a result selected from: increased activity of the N-acetylglucosamine-6-phosphate deacetylase; overexpression of the N-acetylglucosamine-6-phosphate deacetylase; increased reverse action of the N-acetylglucosamine-6-phosphate deacetylase to form N-acetylglucosamine-6-phosphate; reduced or more preferably eliminated forward action of the N-acetylglucosamine-6-phosphate deacetylase to form glucosamine-6-phosphate. In one aspect, the microorganism is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the N-acetylglucosamine-6-phosphate deacetylase. Such a nucleic acid molecule can include a nucleic acid sequence encoding a N-acetylglucosamine-6-phosphate deacetylase that has at least one genetic modification which increases the enzymatic activity of the N-acetylglucosamine-6-phosphate deacetylase, or produces any of the above-described results. The function and representative sequences for N-acetylglucosamine-6-phosphate deacetylases have been described above.</p>
    <p num="p-0121">In another embodiment, the genetically modified microorganism comprises at least one genetic modification that increases the activity of phosphoglucosamine mutase. Preferably, the genetic modification that increases the activity of phosphoglucosamine mutase produces a result selected from: increased activity of phosphoglucosamine mutase; overexpression of phosphoglucosamine mutase; increased action of the phosphoglucosamine mutase to form glucosamine-1-phosphate, and/or reduced or more preferably eliminated action of the phosphoglucosamine mutase to form glucosamine-6-phosphate. In one aspect, the microorganism is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the phosphoglucosamine mutase. Such a nucleic acid molecule can include a nucleic acid sequence encoding a phosphoglucosamine mutase that has at least one genetic modification which increases the enzymatic activity of the phosphoglucosamine mutase, or produces any of the above-described results. The function and representative sequences for phosphoglucosamine mutases have been described above.</p>
    <p num="p-0122">In any of the embodiments described herein, the genetically modified microorganism can have at least one additional genetic modification to increase phosphoglucoisomerase activity in the microorganism. Preferably, the genetically modified microorganism can have at least one additional genetic modification to increase phosphoglucoisomerase produces a results selected from: increased activity of phosphoglucoisomerase, overexpression of phosphoglucoisomerase, increased affinity of the phosphoglucoisomerase for its substrate. In one aspect, the microorganism is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the phosphoglucoisomerase. Such a nucleic acid molecule can include a nucleic acid sequence encoding a phosphoglucoisomerase that has at least one genetic modification which increases the enzymatic activity of the phosphoglucoisomerase, or produces any of the above-described results. The function and representative sequences for phosphoglucoisomerase have been described above.</p>
    <p num="p-0123">In any of the embodiments described herein, the genetically modified microorganism can have at least one additional genetic modification to decrease phosphofructokinase activity in the microorganism. In one aspect, the genetic modification to decrease the activity of phosphofructokinase is a partial or complete deletion or inactivation of an endogenous gene encoding phosphofructokinase in the microorganism. The function and representative sequences for phosphofructokinases have been described above.</p>
    <p num="p-0124">In any of the embodiments described herein, the genetically modified microorganism can have at least one additional genetic modification to increase glutamine synthetase activity in the microorganism. Preferably, the genetically modified microorganism can have at least one additional genetic modification to increase glutamine synthetase produces a results selected from: increased activity of glutamine synthetase, overexpression of glutamine synthetase, increased affinity of the glutamine synthetase for its substrate. In one aspect, the microorganism is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the glutamine synthetase. Such a nucleic acid molecule can include a nucleic acid sequence encoding a glutamine synthetase that has at least one genetic modification which increases the enzymatic activity of the glutamine synthetase, or produces any of the above-described results. The function and representative sequences for glutamine synthetase have been described above.</p>
    <p num="p-0125">In any of the embodiments described herein, the genetically modified microorganism can have at least one additional genetic modification to increase glucose-6-phosphate dehydrogenase activity in the microorganism. Preferably, the genetically modified microorganism can have at least one additional genetic modification to increase glucose-6-phosphate dehydrogenase produces a results selected from: increased activity of glucose-6-phosphate dehydrogenase, overexpression of glucose-6-phosphate dehydrogenase, increased affinity of the glucose-6-phosphate dehydrogenase for its substrate. In one aspect, the microorganism is transformed with at least one recombinant nucleic acid molecule comprising a nucleic acid sequence encoding the glucose-6-phosphate dehydrogenase. Such a nucleic acid molecule can include a nucleic acid sequence encoding a glucose-6-phosphate dehydrogenase that has at least one genetic modification which increases the enzymatic activity of the glucose-6-phosphate dehydrogenase, or produces any of the above-described results. The function and representative sequences for glucose-6-phosphate dehydrogenase have been described above.</p>
    <p num="p-0126">In any of the embodiments described herein, the genetically modified microorganism can have at least one additional genetic modification to decrease the activity of one or more enzymes responsible for glycogen synthesis in the microorganism. Such enzymes include, but are not limited to, glycogen synthesis comprise ADP-glucose pyrophosphorylase, glycogen synthase and a branching enzyme. In one aspect, the genetic modification to decrease the activity of enzymes responsible for glycogen synthesis is a partial or complete deletion or inactivation of one or more of the endogenous gene encoding enzymes responsible for glycogen synthesis in the microorganism.</p>
    <p num="p-0127">In any of the embodiments described herein, the genetically modified microorganism can have at least one additional genetic modification to increase the activity of a phosphatase. The initial intracellular products in the genetically modified microorganism described herein are glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine and/or glucosamine. In many microorganisms, including <i>Escherichia coli</i>, an adequate intracellular supply of ammonium is critical for the glucosamine-6-phosphate deaminase reaction. Glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and N-acetylglucosamine-1-phosphate are typically dephosphorylated to glucosamine and/or N-acetylglucosamine prior to transport out of the cell. Nonetheless, it is yet another embodiment of the present invention to provide a microorganism which is genetically modified to have a suitable phosphatase activity for the dephosphorylation of glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate. Such a phosphatase can include, but is not limited to, for example, alkaline phosphatase, acid phosphatase, phospho-sugar phosphatase and phospho-amino sugar phosphatase. In a preferred embodiment, such an <i>Escherichia coli </i>has an enhanced (i.e., increased) level of phosphatase activity (i.e., phosphatase action).</p>
    <p num="p-0128">In any of the embodiments described above, the microorganism can have at least one an additional genetic modification to increase or decrease the activity of an enzyme selected from the group of: N-acetylglucosamine deacetylase, glucosamine-6-phosphate deaminase, N-acetylglucosamine transporter (II<sup>Nag</sup>), glucosamine synthase, phosphoglucosamine mutase, glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase, mannose transporter (EIIM,P/III<sup>man</sup>), phosphofructokinase, glucose transporter (II<sup>Glc</sup>), glucosamine-6-phosphate acetyltransferase and/or a phosphatase. The genes in <i>E. coli </i>correspond to: NagA, nagB, nagC, nagD, nagE, glmS, glmM, glmU, manXYZ, pƒkA, pƒkB, ptsG, GNA1 or a phosphatase gene, respectively.</p>
    <p num="p-0129">Various of the above-identified genetic modifications can be combined to produce microorganisms having more than one modification, as desired to enhance the production of glucosamine and/or N-acetylglucosamine by the microorganism. For example, in one embodiment, the microorganism has the following genetic modifications: (1) a genetic modification to increase the activity of glucosamine-phosphate N-acetyltransferase; and (2) a genetic modification to increase the activity of the glucosamine-6-phosphate synthase. In a more preferred embodiment, the microorganism also has a genetic modification to decrease the activity of the glucosamine-6-phosphate deaminase.</p>
    <p num="p-0130">In another embodiment, the microorganism has the following genetic modifications: (1) a genetic modification to increase the activity of the glucosamine-phosphate N-acetyltransferase; and (2) a genetic modification to increase the activity of the glucosamine-6-phosphate deaminase. In a more preferred embodiment, the microorganism also has a genetic modification to decrease the activity of glucosamine-6-phosphate synthase.</p>
    <p num="p-0131">In another embodiment, the microorganism has the following genetic modifications: (1) a genetic modification to increase the activity of the glucosamine-6-phosphate deaminase; and (2) a genetic modification to increase the activity of glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase, and preferably to increase glucosamine-1-phosphate N-acetyltransferase activity and/or reduce N-acetylglucosamine-1-phosphate uridyltransferase activity. In a more preferred embodiment, the microorganism also has a genetic modification to decrease the activity of glucosamine-6-phosphate synthase.</p>
    <p num="p-0132">In another embodiment, the microorganism has the following genetic modifications: (1) a genetic modification to increase the activity of the glucosamine-6-phosphate synthase; and (2) a genetic modification to increase the activity of glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase, and preferably to increase glucosamine-1-phosphate N-acetyltransferase activity and/or reduce N-acetylglucosamine-1-phosphate uridyltransferase activity. In a more preferred embodiment, the microorganism also has a genetic modification to decrease the activity of glucosamine-6-phosphate deaminase.</p>
    <p num="p-0133">In another embodiment, a genetically modified microorganism useful in a fermentation method of the invention has an endogenous glucosamine-6-phosphate acetyltransferase (e.g., yeast have an endogenouse glucosamine-6-phosphate acetyltransferase) and also at least one genetic modification to increase the activity of glucosamine-6-phosphate synthase. In a more preferred embodiment, the microorganism also has a genetic modification to decrease the activity of glucosamine-6-phosphate deaminase.</p>
    <p num="p-0134">In another embodiment, a genetically modified microorganism useful in a fermentation method of the invention is transformed with a recombinant nucleic acid molecule encoding glucosamine-6-phosphate synthase, glucosamine-6-phosphate deaminase, glucosamine-6-phosphate acetyltransferase, N-acetylglucosamine-6-phosphate deacetylase, phosphoglucosamine mutase or glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase operatively linked to a transcription control sequence. The recombinant nucleic acid molecule can have a genetic modification which affects the action of the enzyme. Expression of the recombinant nucleic acid molecule increases expression and/or biological activity of the glucosamine-6-phosphate synthase, glucosamine-6-phosphate deaminase, glucosamine-6-phosphate acetyltransferase, N-acetylglucosamine-6-phosphate deacetylase, phosphoglucosamine mutase or glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase by the microorganism as compared to the level of expression or biological activity of the protein in the absence of the recombinant nucleic acid molecule. In a preferred embodiment, the recombinant nucleic acid molecule is integrated into the genome of the microorganism. In a further embodiment, the microorganism has at least one additional genetic modification in a gene encoding a protein selected from the group of glucosamine-6-phosphate synthase, glucosamine-6-phosphate deaminase, N-acetylglucosamine-6-phosphate deacetylase, N-acetyl-glucosamine-specific enzyme II<sup>Nag</sup>, phosphoglucosamine mutase, glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase, phosphofructokinase, Enzyme II<sup>Glc </sup>of the PEP:glucose PTS, EIIM,P/III<sup>Man </sup>of the PEP:mannose PTS, glucosamine-6-phosphate acetyl transferase and/or a phosphatase. The genetic modification increase or decreases the action of the protein, except in the case of the phosphatase, in which the action of the phosphatase is preferably increased. In another preferred embodiment, the microorganism has a modification in genes encoding N-acetylglucosamine-6-phosphate deacetylase, glucosamine-6-phosphate deaminase and N-acetyl-glucosamine-specific enzyme II<sup>Nag </sup>wherein the genetic modification decreases or increase action of the protein. In one embodiment, the genetic modification is a deletion of at least a portion of the genes.</p>
    <p num="p-0135">In another embodiment, a genetically modified microorganism of the present invention has a recombinant nucleic acid molecule encoding glucosamine-6-phosphate synthase glucosamine-6-phosphate deaminase, glucosamine-6-phosphate acetyltransferase, N-acetylglucosamine-6-phosphate deacetylase, phosphoglucosamine mutase or glucosamine-1-phosphate N-acetyltransferase/N-acetylglucosamine-1-phosphate uridyltransferase operatively linked to a transcription control sequence; and at least one genetic modification in a gene encoding a protein selected from the group of N-acetylglucosamine-6-phosphate deacetylase, glucosamine-6-phosphate deaminase, N-acetyl-glucosamine-specific enzyme II<sup>Nag</sup>, phosphoglucosamine mutase, glucosamine-1-phosphate N-acetyltransferase-N-acetylglucosamine-1-phosphate uridyltransferase, phosphofructokinase, glucosamine-6-phosphate acetyltransferase, Enzyme II<sup>Glc </sup>of the PEP:glucose PTS, and/or EIIM,P/III<sup>Man </sup>the PEP:mannose PTS. The genetic modification increase or decreases action of the protein and expression of the recombinant nucleic acid molecule increases expression of the enzymes by the microorganism. In another embodiment, the microorganism has at least one genetic modification in a phosphatase gene, such that the phosphatase encoded by such gene has increased action. In a preferred embodiment, the recombinant nucleic acid molecule is integrated into the genome of the microorganism.</p>
    <p num="p-0136">Various other combinations of the mutations described herein are encompassed by the invention and many are described in the Examples section.</p>
    <p num="p-0137">Furthermore, the process and materials disclosed in the present invention can be used and/or modified by those skilled in the art to produce other amino sugars such as poly-N-acetylglucosamine, poly-glucosamine, galactosamine, mannosamine, N-acetyl galactosamine, N-acetyl mannosamine and their derivatives.</p>
    <p num="p-0138">As described above, to produce significantly high yields of glucosamine and/or N-acetylglucosamine by the fermentation method of the present invention, a microorganism is genetically modified to enhance production of glucosamine and/or N-acetylglucosamine. As used herein, a genetically modified microorganism has a genome which is modified (i.e., mutated or changed) from its normal (i.e., wild-type or naturally occurring) form. In one aspect, such an organism can endogenously contain and express a gene encoding the protein of interest, and the genetic modification can be a genetic modification of the gene, whereby the modification has some effect (e.g., increase, decrease, delete) on the expression and/or activity of the gene. In another aspect, such an organism can endogenously contain and express a gene encoding the protein of interest, and the genetic modification can be an introduction of at least one exogenous nucleic acid sequence (e.g., a recombinant nucleic acid molecule), wherein the exogenous nucleic acid sequence encodes the protein of interest and/or a protein that affects the activity of the protein or gene encoding the protein. The exogenous nucleic acid molecule to be introduced into the microorganism can encode a wild-type protein or it can have one or more modifications that affect the expression and/or activity of the encoded protein as compared to the wild-type or normal protein. In yet another aspect, the organism does not necessarily endogenously (naturally) contain the gene encoding the protein of interest, but is genetically modified to introduce at least one recombinant nucleic acid molecule encoding a protein having the biological activity of the protein of interest. Again, the recombinant nucleic acid molecule can encode a wild-type protein or the recombinant nucleic acid sequence can be modified to affect the expression and/or activity of the encoded protein as compared to a wild-type protein. In other embodiments, various expression control sequences (e.g., promoters) can be introduced into the microorganism to effect the expression of an endogenous gene in the microorganism. Various embodiments associated with each of these aspects will be discussed in greater detail below.</p>
    <p num="p-0139">As used herein, a genetically modified microorganism can include any genetically modified microorganism, including a bacterium, a protist, a microalgae, a fungus, or other microbe. Such a genetically modified microorganism has a genome which is modified (i.e., mutated or changed) from its normal (i.e., wild-type or naturally occurring) form and/or is modified to express extrachromosomal genetic material (e.g., a recombinant nucleic acid molecule), such that the desired result is achieved (e.g., increased, decreased, or otherwise modified enzyme expression and/or activity and/or modified production of glucosamine or N-acetylglucosamine as a result of the modification(s)). More particularly, the modification to the microorganism can be achieved by modification of the genome of the microorganism (e.g., endogenous genes) and/or by introducing genetic material (e.g., a recombinant nucleic acid molecule) into the microorganism, which can remain extrachromosomal or can be integrated into the host microbial genome. As such, the genetic modification can include the introduction or modification of regulatory sequences which regulate the expression of endogenous or recombinantly introduced nucleic acid sequences in the microorganism, the introduction of wild-type or modified recombinant nucleic acid molecules (e.g., encoding wild-type or modified proteins), the modification of endogenous genes in the microorganism, or any other modification which results in the microorganism having the specified characteristics with regard to enzyme expression and/or biological activity. Genetic modification of a microorganism can be accomplished using classical strain development and/or molecular genetic techniques. Such techniques known in the art and are generally disclosed for microorganisms, for example, in Sambrook et al., 1989<i>, Molecular Cloning: A Laboratory Manual</i>, Cold Spring Harbor Labs Press. The reference Sambrook et al., ibid., is incorporated by reference herein in its entirety. A genetically modified microorganism can include a microorganism in which nucleic acid molecules have been inserted, deleted and/or modified (i.e., mutated; e.g., by insertion, deletion, substitution, and/or inversion of nucleotides), in such a manner that such modifications provide the desired effect within the microorganism. According to the present invention, a genetically modified microorganism includes a microorganism that has been modified using recombinant technology.</p>
    <p num="p-0140">In one embodiment of the present invention, a genetic modification of a microorganism increases or decreases the activity of a protein involved in an amino sugar metabolic pathway according to the present invention. Such a genetic modification includes any type of modification and specifically includes modifications made by recombinant technology and/or by classical mutagenesis. As used herein, genetic modifications which result in a decrease in gene expression, in the function of the gene, or in the function of the gene product (i.e., the protein encoded by the gene) can be referred to as inactivation (complete or partial), deletion, interruption, blockage, silencing or down-regulation of a gene. For example, a genetic modification in a gene which results in a decrease in the function of the protein encoded by such gene, can be the result of a complete deletion of the gene (i.e., the gene does not exist, and therefore the protein does not exist), a mutation in the gene which results in incomplete or no translation of the protein (e.g., the protein is not expressed), or a mutation in the gene which decreases or abolishes the natural function of the protein (e.g., a protein is expressed which has decreased or no enzymatic activity or action). More specifically, reference to decreasing the action or activity of enzymes discussed herein generally refers to any genetic modification in the microorganism in question which results in decreased expression and/or functionality (biological activity) of the enzymes and includes decreased activity of the enzymes (e.g., specific activity), increased inhibition or degradation of the enzymes as well as a reduction or elimination of expression of the enzymes. For example, the action or activity of an enzyme of the present invention can be decreased by blocking or reducing the production of the enzyme, reducing enzyme activity, or inhibiting the activity of the enzyme. Combinations of some of these modifications are also possible. Blocking or reducing the production of an enzyme can include placing the gene encoding the enzyme under the control of a promoter that requires the presence of an inducing compound in the growth medium. By establishing conditions such that the inducer becomes depleted from the medium, the expression of the gene encoding the enzyme (and therefore, of enzyme synthesis) could be turned off. Blocking or reducing the activity of an enzyme could also include using an excision technology approach similar to that described in U.S. Pat. No. 4,743,546, incorporated herein by reference. To use this approach, the gene encoding the enzyme of interest is cloned between specific genetic sequences that allow specific, controlled excision of the gene from the genome. Excision could be prompted by, for example, a shift in the cultivation temperature of the culture, as in U.S. Pat. No. 4,743,546, or by some other physical or nutritional signal.</p>
    <p num="p-0141">Genetic modifications that result in an increase in gene expression or function can be referred to as amplification, overproduction, overexpression, activation, enhancement, addition, or up-regulation of a gene. More specifically, reference to increasing the action (or activity) of enzymes or other proteins discussed herein generally refers to any genetic modification in the microorganism in question which results in increased expression and/or functionality (biological activity) of the enzymes or proteins and includes higher activity of the enzymes (e.g., specific activity or in vivo enzymatic activity), reduced inhibition or degradation of the enzymes and overexpression of the enzymes. For example, gene copy number can be increased, expression levels can be increased by use of a promoter that gives higher levels of expression than that of the native promoter, or a gene can be altered by genetic engineering or classical mutagenesis to increase the biological activity of an enzyme. Combinations of some of these modifications are also possible.</p>
    <p num="p-0142">In general, according to the present invention, an increase or a decrease in a given characteristic of a mutant or modified enzyme (e.g., enzyme activity) is made with reference to the same characteristic of a wild-type (i.e., normal, not modified) enzyme that is derived from the same organism (from the same source or parent sequence), which is measured or established under the same or equivalent conditions. Similarly, an increase or decrease in a characteristic of a genetically modified microorganism (e.g., expression and/or biological activity of a protein, or production of a product) is made with reference to the same characteristic of a wild-type microorganism of the same species, and preferably the same strain, under the same or equivalent conditions. Such conditions include the assay or culture conditions (e.g., medium components, temperature, pH, etc.) under which the activity of the protein (e.g., expression or biological activity) or other characteristic of the microorganism is measured, as well as the type of assay used, the host microorganism that is evaluated, etc. As discussed above, equivalent conditions are conditions (e.g., culture conditions) which are similar, but not necessarily identical (e.g., some conservative changes in conditions can be tolerated), and which do not substantially change the effect on microbe growth or enzyme expression or biological activity as compared to a comparison made under the same conditions.</p>
    <p num="p-0143">Preferably, a genetically modified microorganism that has a genetic modification that increases or decreases the activity of a given protein (e.g., an enzyme) has an increase or decrease, respectively, in the activity (e.g., expression, production and/or biological activity) of the protein, as compared to the activity of the wild-type protein in a wild-type microorganism, of at least about 5%, and more preferably at least about 10%, and more preferably at least about 15%, and more preferably at least about 20%, and more preferably at least about 25%, and more preferably at least about 30%, and more preferably at least about 35%, and more preferably at least about 40%, and more preferably at least about 45%, and more preferably at least about 50%, and more preferably at least about 55%, and more preferably at least about 60%, and more preferably at least about 65%, and more preferably at least about 70%, and more preferably at least about 75%, and more preferably at least about 80%, and more preferably at least about 85%, and more preferably at least about 90%, and more preferably at least about 95%, or any percentage, in whole integers between 5% and 100% (e.g., 6%, 7%, 8%, etc.). The same differences are preferred when comparing an isolated modified nucleic acid molecule or protein directly to the isolated wild-type nucleic acid molecule or protein (e.g., if the comparison is done in vitro as compared to in vivo).</p>
    <p num="p-0144">In another aspect of the invention, a genetically modified microorganism that has a genetic modification that increases or decreases the activity of a given protein (e.g., an enzyme) has an increase or decrease, respectively, in the activity (e.g., expression, production and/or biological activity) of the protein, as compared to the activity of the wild-type protein in a wild-type microorganism, of at least about 2-fold, and more preferably at least about 5-fold, and more preferably at least about 10-fold, and more preferably about 20-fold, and more preferably at least about 30-fold, and more preferably at least about 40-fold, and more preferably at least about 50-fold, and more preferably at least about 75-fold, and more preferably at least about 100-fold, and more preferably at least about 125-fold, and more preferably at least about 150-fold, or any whole integer increment starting from at least about 2-fold (e.g., 3-fold, 4-fold, 5-fold, 6-fold, etc.).</p>
    <p num="p-0145">The genetic modification of a microorganism to provide increased or decreased activity (including expression, specific activity, in vivo activity, etc.) preferably affects the activity of a glucosamine and/or N-acetylglucosamine biosynthetic pathway or amino sugar metabolic pathway in the microorganism, whether the pathway is endogenous and genetically modified, endogenous with the introduction of one or more recombinant nucleic acid molecules into the organism, or provided completely by recombinant technology. According to the present invention, to “affect the activity of a glucosamine and/or N-acetylglucosamine biosynthetic pathway” includes any genetic modification that causes any detectable or measurable change or modification in the glucosamine and/or N-acetylglucosamine biosynthetic pathway expressed by the organism as compared to in the absence of the genetic modification. A detectable change or modification in the glucosamine and/or N-acetylglucosamine biosynthetic pathway can include, but is not limited to, a detectable change in the production of at least one product in the glucosamine and/or N-acetylglucosamine biosynthetic pathway, or a detectable change in the production of intracellular and/or extracellular glucosamine or N-acetylglucosamine by the microorganism.</p>
    <p num="p-0146">In one embodiment of the present invention, a genetic modification includes a modification of a nucleic acid sequence encoding a particular enzyme or other protein as described herein. Such a modification can be to the endogenous enzyme or protein, whereby a microorganism that naturally contains such a protein is genetically modified by, for example, classical mutagenesis and selection techniques and/or molecular genetic techniques, include genetic engineering techniques. Genetic engineering techniques can include, for example, using a targeting recombinant vector to delete a portion of an endogenous gene or to replace a portion of an endogenous gene with a heterologous sequence, such as a sequence encoding an improved enzyme or other protein or a different promoter that increases the expression of the endogenous enzyme or other protein. Genetic engineering techniques can also include overexpression of a gene using recombinant technology.</p>
    <p num="p-0147">For example, a non-native promoter can be introduced upstream of at least one gene encoding an enzyme or other protein of interest in the amino sugar metabolic pathway described herein. Preferably the 5′ upstream sequence of a endogenous gene is replaced by a constitutive promoter, an inducible promoter, or a promoter with optimal expression under the growth conditions used. This method is especially useful when the endogenous gene is not active or is not sufficiently active under the growth conditions used.</p>
    <p num="p-0148">In another aspect of this embodiment of the invention, the genetic modification can include the introduction of a recombinant nucleic acid molecule encoding a enzyme or protein of interest into a host. The host can include: (1) a host cell that does not express the particular enzyme or protein, or (2) a host cell that does express the particular enzyme or protein, wherein the introduced recombinant nucleic acid molecule changes or enhances the activity of the enzyme or other protein in the microorganism. The present invention intends to encompass any genetically modified microorganism, wherein the microorganism comprises at least one modification suitable for a fermentation process to produce glucosamine or N-acetylglucosamine according to the present invention.</p>
    <p num="p-0149">A genetically modified microorganism can be modified by recombinant technology, such as by introduction of an isolated nucleic acid molecule into a microorganism. For example, a genetically modified microorganism can be transfected with a recombinant nucleic acid molecule encoding a protein of interest, such as a protein for which increased expression is desired. The transfected nucleic acid molecule can remain extrachromosomal or can integrate into one or more sites within a chromosome of the transfected (i.e., recombinant) host cell in such a manner that its ability to be expressed is retained. Preferably, once a host cell of the present invention is transfected with a nucleic acid molecule, the nucleic acid molecule is integrated into the host cell genome. A significant advantage of integration is that the nucleic acid molecule is stably maintained in the cell. In a preferred embodiment, the integrated nucleic acid molecule is operatively linked to a transcription control sequence (described below) which can be induced to control expression of the nucleic acid molecule.</p>
    <p num="p-0150">A nucleic acid molecule can be integrated into the genome of the host cell either by random or targeted integration. Such methods of integration are known in the art. For example, <i>E. coli </i>strain ATCC 47002 contains mutations that confer upon it an inability to maintain plasmids which contain a ColE1 origin of replication. When such plasmids are transferred to this strain, selection for genetic markers contained on the plasmid results in integration of the plasmid into the chromosome. This strain can be transformed, for example, with plasmids containing the gene of interest and a selectable marker flanked by the 5′- and 3′-termini of the <i>E. coli </i>lacZ gene. The lacZ sequences target the incoming DNA to the lacZ gene contained in the chromosome. Integration at the lacZ locus replaces the intact lacZ gene, which encodes the enzyme P-galactosidase, with a partial lacZ gene interrupted by the gene of interest. Successful integrants can be selected for P-galactosidase negativity. A genetically modified microorganism can also be produced by introducing nucleic acid molecules into a recipient cell genome by a method such as by using a transducing bacteriophage. The use of recombinant technology and transducing bacteriophage technology to produce several different genetically modified microorganism of the present invention is known in the art and is described in detail in the Examples section.</p>
    <p num="p-0151">Vectors and methods were described by Hamilton et al. (1989<i>, J. Bacteriol</i>. 171:4617-4622) to make targeted gene deletion and gene integration in <i>E. coli </i>chromosome by temperature shift. The method was adapted to develop different glucosamine production <i>E. coli </i>strains. The protocols for gene integration include the following major steps. The first step is to clone the sequence of the target site and make an internal deletion and/or insert the foreign gene to be integrated at the deletion site. The second step is to subclone the fragment containing these sequences into a temperature sensitive integrative vector containing a temperature sensitive replication origin and an antibiotic selection marker. The third step is to transform the integrative vector into the <i>E. coli </i>host strain and select for clones with the entire plasmid integrated into the chromosome through single crossover recombination event under non-permissive temperature (42° C.). The fourth step is to grow the cells of selected clones in liquid culture at permissive temperature (30° C.). Cells with the integrated plasmid have a tendency to lose the plasmid. Cells that have lost the portion of the replication origin and antibiotic resistance gene or the entire plasmid will outgrow in the culture. Typically, this step was accomplished by inoculating a 50-ml LB medium with cells from a pool of two to ten clones and growing the culture for 24 hrs. The culture was passed to a fresh medium at a 1,000-fold dilution and grown for another period of 24 hrs. Fifth, cells were plated and clones that had lost the antibiotic resistance were selected. Gene specific selection procedures could be used, depending on the nature of integrated gene or deleted gene. Typically for screening clones, PCR was carried out using a primer set that could distinguish the clones with the intended change in the chromosome from its native form by the size of PCR products. Clones were confirmed by Southern Blot analysis using probes specific to the integrated or deleted DNA sequence.</p>
    <p num="p-0152">It is to be understood that the present invention discloses a method comprising the use of a microorganism with an ability to produce commercially useful amounts of glucosamine and/or N-acetylglucosamine in a fermentation process (i.e., preferably an enhanced ability to produce glucosamine and/or N-acetylglucosamine compared to a wild-type microorganism cultured under the same conditions). As used herein, a fermentation process is a process of culturing cells, such as microorganisms, in a container, bioreactor, fermenter, or other suitable culture chamber, in order to produce a product from the cells (i.e., the cells produce a product during the culture process). The product is typically a product useful for experimental or commercial purposes. The fermentation method of the present invention is achieved by the genetic modification of one or more genes encoding a protein involved in an amino sugar metabolic pathway which results in the production (expression) of a protein having an altered (e.g., increased or decreased) function as compared to the corresponding wild-type protein. Such an altered function enhances the ability of the genetically engineered microorganism to produce glucosamine and/or N-acetylglucosamine. It will be appreciated by those of skill in the art that production of genetically modified microorganisms having a particular altered function as described elsewhere herein (e.g., an enhanced ability to produce glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate) such as by the specific selection techniques described in the Examples, can produce many organisms meeting the given functional requirement, albeit by virtue of a variety of different genetic modifications. For example, different random nucleotide deletions and/or substitutions in a given nucleic acid sequence may all give rise to the same phenotypic result (e.g., decreased action of the protein encoded by the sequence). The present invention contemplates any such genetic modification which results in the production of a microorganism having the characteristics set forth herein.</p>
    <p num="p-0153">According to one embodiment of the present invention, a genetically modified microorganism includes a microorganism which has an enhanced ability to synthesize glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate. According to the present invention, “an enhanced ability to synthesize” a product refers to any enhancement, or up-regulation, in an amino sugar metabolic pathway related to the synthesis of the product such that the microorganism produces an increased amount of the product compared to the wild-type microorganism cultured under the same conditions.</p>
    <p num="p-0154">In one aspect of the invention, a genetically modified microorganism useful in a fermentation method produces at least about 1 g/L, and preferably at least about 5 g/L, and more preferably, at least about −10 g/L, and even more preferably, at least about 20 g/L, and even more preferably, at least about 30 g/L, and even more preferably, at least about 40 g/L, and even more preferably, at least about 50 g/L, and even more preferably, at least about 60 g/L, and even more preferably, at least about 70 g/L, and even more preferably, at least about 80 g/L, and even more preferably, at least about 90 g/L, and even more preferably, at least about 100 g/L, and even more preferably, at least about 110 g/L, and even more preferably, at least about 120 g/L, and even more preferably at least about 150 g/L, and even more preferably at least about 180 g/L, or any higher amount, or any amount between at least about 1 g/L and at least about 500 g/L, in whole integers (e.g., 2 g/L, 3 g/L, etc.) of: glucosamine, glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate, when cultured under any suitable culture conditions, and particularly under any of the culture conditions as described herein. In another aspect, a genetically modified microorganism useful in a fermentation method of the present invention produces at least about 2-fold more glucosamine, glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate, and preferably at least about 5-fold, and more preferably at least about 10-fold, and more preferably at least about 25-fold, and more preferably at least about 50-fold, and even more preferably at least about 100-fold, and even more preferably, at least about 200-fold, including any fold increase between at least 2-fold and at least 200-fold, in whole integer increments (i.e., at least 3-fold, at least 4-fold, etc.), more glucosamine, glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate than a wild-type (i.e., non-modified, naturally occurring) microorganism of the same species (and preferably strain) cultured under the same conditions or equivalent conditions as the genetically modified microorganism. A number of specific microorganisms having such characteristics are identified in the Examples section.</p>
    <p num="p-0155">In another aspect, a genetically modified microorganism useful in a fermentation method of the present invention produces at least about 1 g/L of glucosamine when cultured for about 24 hours at 37° C. to a cell density of at least about 8 g/L by dry cell weight, in a pH 7.0 fermentation medium comprising: 14 g/L K<sub>2</sub>HPO<sub>4</sub>, 16 g/L KH<sub>2</sub>PO<sub>4</sub>, 1 g/L Na<sub>3</sub>Citrate.2H<sub>2</sub>O, 5 g/L (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 20 g/L glucose, 10 mM MgSO<sub>4</sub>, 1 mM CaCl<sub>2</sub>, and 1 mM IPTG. In another aspect, a genetically modified microorganism useful in a fermentation method of the present invention produces at least about 1 g/L of glucosamine when cultured for about 10 to about 60 hours at from about 28° C. to about 37° C. to a cell density of at least about 8 g/L by dry cell weight, in a pH 7.0 fermentation medium comprising: 14 g/L K<sub>2</sub>HPO<sub>4</sub>, 16 g/L KH<sub>2</sub>PO<sub>4</sub>, 1 g/L Na<sub>3</sub>Citrate.2H<sub>2</sub>O, 5 g/L (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 20 g/L glucose, 10 mM MgSO<sub>4</sub>, 1 mM CaCl<sub>2</sub>, and from about 0.2 mM to about 1 mM IPTG. In a preferred embodiment, the amount of IPTG is about 0.2 mM.</p>
    <p num="p-0156">A microorganism to be used in the fermentation method of the present invention (e.g., a host cell or production organism) is any microorganism (e.g., a bacterium, a protist, an alga, a fungus, or other microbe), and is most preferably a bacterium, a yeast or a fungus. Suitable bacterial genera include, but are not limited to, <i>Escherichia, Bacillus, Lactobacillus, Pseudomonas </i>and <i>Streptomyces</i>. Suitable bacterial species include, but are not limited to, <i>Escherichia coli, Bacillus subtilis, Bacillus licheniformis, Lactobacillus brevis, Pseudomonas aeruginosa </i>and <i>Streptomyces lividans</i>. Suitable genera of yeast include, but are not limited to, <i>Saccharomyces, Schizosaccharomyces, Candida, Hansenula, Pichia, Kluyveromyces</i>, and <i>Phaffia</i>. Suitable yeast species include, but are not limited to, <i>Saccharomyces cerevisiae, Schizosaccharomyces pombe, Candida albicans, Hansenula polymorpha, Pichia pastoris, P. canadensis, Kluyveromyces marxianus </i>and <i>Phaffia rhodozyma</i>. Suitable fungal genera include, but are not limited to, <i>Aspergillus, Absidia, Rhizopus, Chrysosporium, Neurospora </i>and <i>Trichoderma</i>. Suitable fungal species include, but are not limited to, <i>Aspergillus niger, A. nidulans, Absidia coerulea, Rhizopus oryzae, Chrysosporium lucknowense, Neurospora crassa, N. intermedia </i>and <i>Trichoderm reesei</i>. Particularly preferred strains of <i>Escherichia coli </i>include K-12, B and W, and most preferably, K-12. Although <i>Escherichia coli </i>is one preferred bacteria and is used to exemplify various embodiments of the invention, it is to be understood that any microorganism that produces glucosamine and/or N-acetylglucosamine, and can be genetically modified to enhance production of glucosamine and/or N-acetylglucosamine can be used in the method of the present invention. A microorganism for use in the fermentation method of the present invention can also be referred to as a production organism.</p>
    <p num="p-0157">In a preferred embodiment, the genetically modified microorganism is a bacterium or a yeast, and more preferably, a bacterium of the genus Escherichia, and even more preferably, <i>Escherichia coli</i>. A genetically modified <i>Escherichia coli </i>preferably has a modification in a gene which includes, but is not limited to, nagA, nagB, nagC, nagD, nagE, manXYZ, glmM, pƒkB, pƒkA, glmU, glmS, GNA1, ptsG or a phosphatase gene. In another embodiment, such a genetically modified <i>Escherichia coli </i>has a deletion of nag regulon genes, and in yet another embodiment, a deletion of nag regulon genes and a genetic modification in manXYZ genes such that the proteins encoded by the manXYZ genes have decreased action.</p>
    <p num="p-0158">According to the present invention, reference to a particular enzyme or other protein herein refers to any protein that has at least one biological activity of the wild-type reference protein, including full-length proteins, fusion proteins, or any homologue of a naturally occurring protein (including natural allelic variants, fragments, related proteins from different organisms and synthetically or artificially derived variants (homologues)). A homologue (mutant, variant, modified form) of a reference protein includes proteins which differ from the naturally occurring reference protein in that at least one or a few, but not limited to one or a few, amino acids have been deleted (e.g., a truncated version of the protein, such as a peptide or fragment), inserted, inverted, substituted and/or derivatized (e.g., by glycosylation, phosphorylation, acetylation, myristoylation, prenylation, palmitation, amidation and/or addition of glycosylphosphatidyl inositol). One preferred homologue is a biologically active fragment of a naturally occurring protein. Other preferred homologues of naturally occurring proteins useful in the present invention are described in detail below. Therefore, an isolated nucleic acid molecule of the present invention can encode the translation product of any specified protein open reading frame, domain, biologically active fragment thereof, or any homologue of a naturally occurring protein or domain which has biological activity.</p>
    <p num="p-0159">An isolated protein, according to the present invention, is a protein that has been removed from its natural milieu (i.e., that has been subject to human manipulation) and can include purified proteins, partially purified proteins, recombinantly produced proteins, and synthetically produced proteins, for example. Several recombinantly produced proteins are described in the Examples section. As such, “isolated” does not reflect the extent to which the protein has been purified. In addition, and by way of example by referencing a hypothetical protein called “protein X” (i.e., any enzyme or protein of used in the invention can be substituted for the term), an “<i>E. coli </i>protein X” refers to a protein X (including a homologue of a naturally occurring protein X) from <i>E. coli </i>or to a protein X that has been otherwise produced from the knowledge of the structure (e.g., sequence) and perhaps the function of a naturally occurring protein X from <i>E. coli</i>. In other words, an <i>E. coli </i>protein X includes any protein X that has substantially similar structure and function of a naturally occurring protein X from <i>E. coli </i>or that is a biologically active (i.e., has biological activity) homologue of a naturally occurring protein X from <i>E. coli </i>as described in detail herein. As such, an <i>E. coli </i>protein X can include purified, partially purified, recombinant, mutated/modified and synthetic proteins. This discussion applies similarly to protein X from other microorganisms as disclosed herein.</p>
    <p num="p-0160">Homologues can be the result of natural allelic variation or natural mutation. A naturally occurring allelic variant of a nucleic acid encoding a protein is a gene that occurs at essentially the same locus (or loci) in the genome as the gene which encodes such protein, but which, due to natural variations caused by, for example, mutation or recombination, has a similar but not identical sequence. Allelic variants typically encode proteins having similar activity to that of the protein encoded by the gene to which they are being compared. One class of allelic variants can encode the same protein but have different nucleic acid sequences due to the degeneracy of the genetic code. Allelic variants can also comprise alterations in the 5′ or 3′ untranslated regions of the gene (e.g., in regulatory control regions). Allelic variants are well known to those skilled in the art.</p>
    <p num="p-0161">Homologues can be produced using techniques known in the art for the production of proteins including, but not limited to, direct modifications to the isolated, naturally occurring protein, direct protein synthesis, or modifications to the nucleic acid sequence encoding the protein using, for example, classic or recombinant DNA techniques to effect random or targeted mutagenesis.</p>
    <p num="p-0162">Modifications in homologues, as compared to the wild-type protein, either agonize, antagonize, or do not substantially change, the basic biological activity of the homologue as compared to the naturally occurring protein. In general, the biological activity or biological action of a protein refers to any function(s) exhibited or performed by the protein that is ascribed to the naturally occurring form of the protein as measured or observed in vivo (i.e., in the natural physiological environment of the protein) or in vitro (i.e., under laboratory conditions). The biological activity of the enzymes and proteins used herein have been described in detail above. For example, the enzyme generally referred to herein as “glucosamine-6-phosphate synthase” catalyzes the formation of glucosamine-6-phosphate and glutamate from fructose-6-phosphate and glutamine. Modifications of a protein, such as in a homologue, may result in proteins having the same level of biological activity as the naturally occurring protein, or in proteins having decreased or increased biological activity as compared to the naturally occurring protein. Modifications which result in a decrease in expression or a decrease in the activity of the protein, can be referred to as inactivation (complete or partial), down-regulation, or decreased action of a protein. Similarly, modifications which result in an increase in expression or an increase in the activity of the protein, can be referred to as amplification, overproduction, activation, enhancement, up-regulation or increased action of a protein. A functional subunit, homologue, or fragment of a given protein is preferably capable of performing substantially the same (e.g., at least qualitatively the same) biological function of the native protein (i.e., has biological activity). It is noted that a functional subunit, fragment or other homologue of a protein is not necessarily required to have the same level of biological activity as the reference or wild-type protein in order to be considered to have the biological activity of the reference or wild-type protein (i.e., a qualitative similarity is sufficient). In one embodiment, it is preferred that modifications in homologues as compared to the wild-type protein do not substantially decrease the basic biological activity of the protein as compared to the naturally occurring protein. Increased biological activity (e.g., increased enzyme activity) may be desirable in a homologue. Homologues may also have differences in characteristics other than the functional, or enzymatic, activity of the protein as compared to the naturally occurring form, such as a decreased sensitivity to inhibition by certain compounds as compared to the naturally occurring protein.</p>
    <p num="p-0163">According to the present invention, an isolated protein, including a biologically active homologue or fragment thereof, has at least one characteristic of biological activity of the wild-type, or naturally occurring protein. Methods of detecting and measuring protein expression and biological activity include, but are not limited to, measurement of transcription of the protein, measurement of translation of the protein, measurement of cellular localization of the protein, measurement of binding or association of the protein with another protein, measurement of binding or association of the gene encoding the protein regulatory sequences to a protein or other nucleic acid, measurement of an increase, decrease or induction of biological activity of the protein in a cell that expresses the protein.</p>
    <p num="p-0164">Methods to measure protein expression levels of a protein according to the invention include, but are not limited to: western blotting, immunocytochemistry, flow cytometry or other immunologic-based assays; assays based on a property of the protein including but not limited to, ligand binding, enzyme activity or interaction with other protein partners. Binding assays are also well known in the art. For example, a BIAcore machine can be used to determine the binding constant of a complex between two proteins. The dissociation constant for the complex can be determined by monitoring changes in the refractive index with respect to time as buffer is passed over the chip (O'Shannessy et al. Anal. Biochem. 212:457-468 (1993); Schuster et al., Nature 365:343-347 (1993)). Other suitable assays for measuring the binding of one protein to another include, for example, immunoassays such as enzyme linked immunoabsorbent assays (ELISA) and radioimmunoassays (RIA), or determination of binding by monitoring the change in the spectroscopic or optical properties of the proteins through fluorescence, UV absorption, circular dichrosim, or nuclear magnetic resonance (NMR). Assays for measuring the enzymatic activity of a protein used in the invention are well known in the art and many are described in the Examples section.</p>
    <p num="p-0165">Many of the enzymes and proteins involved in the amino sugar metabolic pathway and which represent desirable targets for modification and use in the fermentation processes described herein have been described above in terms of function and amino acid sequence (and nucleic acid sequence encoding the same) of representative wild-type or mutant proteins. In one embodiment of the invention, homologues of a given protein (which can include related proteins from other organisms or modified forms of the given protein) are encompassed for use in a genetically modified organism of the invention. Homologues of a proteins encompassed by the present invention can comprise an amino acid sequence that is at least about 35% identical, and more preferably at least about 40% identical, and more preferably at least about 45% identical, and more preferably at least about 50% identical, and more preferably at least about 55% identical, and more preferably at least about 60% identical, and more preferably at least about 65% identical, and more preferably at least about 70% identical, and more preferably at least about 75% identical, and more preferably at least about 80% identical, and more preferably at least about 85% identical, and more preferably at least about 90% identical, and more preferably at least about 95% identical, and more preferably at least about 96% identical, and more preferably at least about 97% identical, and more preferably at least about 98% identical, and more preferably at least about 99% identical, or any percent identity between 35% and 99%, in whole integers (i.e., 36%, 37%, etc.) to an amino acid sequence disclosed herein that represents the amino acid sequence of an enzyme or protein that can be modified or overexpressed according to the invention. Preferably, the amino acid sequence of the homologue has a biological activity of the wild-type or reference protein.</p>
    <p num="p-0166">As used herein, unless otherwise specified, reference to a percent (%) identity refers to an evaluation of homology which is performed using: (1) a BLAST 2.0 Basic BLAST homology search using blastp for amino acid searches and blastn for nucleic acid searches with standard default parameters, wherein the query sequence is filtered for low complexity regions by default (described in Altschul, S. F., Madden, T. L., Schäffer; A. A., Zhang, J., Zhang, Z., Miller, W. &amp; Lipman, D. J. (1997) “Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.” Nucleic Acids Res. 25:3389-3402, incorporated herein by reference in its entirety); (2) a BLAST 2 alignment (using the parameters described below); (3) and/or PSI-BLAST with the standard default parameters (Position-Specific Iterated BLAST. It is noted that due to some differences in the standard parameters between BLAST 2.0 Basic BLAST and BLAST 2, two specific sequences might be recognized as having significant homology using the BLAST 2 program, whereas a search performed in BLAST 2.0 Basic BLAST using one of the sequences as the query sequence may not identify the second sequence in the top matches. In addition, PSI-BLAST provides an automated, easy-to-use version of a “profile” search, which is a sensitive way to look for sequence homologues. The program first performs a gapped BLAST database search. The PSI-BLAST program uses the information from any significant alignments returned to construct a position-specific score matrix, which replaces the query sequence for the next round of database searching. Therefore, it is to be understood that percent identity can be determined by using any one of these programs.</p>
    <p num="p-0167">Two specific sequences can be aligned to one another using BLAST 2 sequence as described in Tatusova and Madden, (1999), “Blast 2 sequences—a new tool for comparing protein and nucleotide sequences”, <i>FEMS Microbiol Lett</i>. 174:247-250, incorporated herein by reference in its entirety. BLAST 2 sequence alignment is performed in blastp or blastn using the BLAST 2.0 algorithm to perform a Gapped BLAST search (BLAST 2.0) between the two sequences allowing for the introduction of gaps (deletions and insertions) in the resulting alignment. For purposes of clarity herein, a BLAST 2 sequence alignment is performed using the standard default parameters as follows.</p>
    <p num="h-0007">For blastn, using 0 BLOSUM62 matrix:</p>
    <p num="p-0168">Reward for match=1</p>
    <p num="p-0169">Penalty for mismatch=−2</p>
    <p num="p-0170">Open gap (5) and extension gap (2) penalties</p>
    <p num="p-0171">gap x_dropoff (50) expect (10) word size (11) filter (on)</p>
    <p num="h-0008">For blastp, using 0 BLOSUM62 matrix:</p>
    <p num="p-0172">Open gap (11) and extension gap (1) penalties</p>
    <p num="p-0173">gap x_dropoff (50) expect (10) word size (3) filter (on).</p>
    <p num="p-0174">A protein referenced and/or used in the present invention can also include proteins having an amino acid sequence comprising at least 30 contiguous amino acid residues of the amino acid sequence of the reference protein (i.e., 30 contiguous amino acid residues having 100% identity with 30 contiguous amino acids of either of the above-identified sequences). In a preferred embodiment, a protein referenced and/or used in the present invention includes proteins having amino acid sequences comprising at least 50, and more preferably at least 75, and more preferably at least 100, and more preferably at least 115, and more preferably at least 130, and more preferably at least 150, and more preferably at least 200, and more preferably, at least 250, and more preferably, at least 300, and more preferably, at least 350 contiguous amino acid residues of the amino acid sequence of the reference protein. In one embodiment, such a protein has a biological activity of the reference protein.</p>
    <p num="p-0175">According to the present invention, the term “contiguous” or “consecutive”, with regard to nucleic acid or amino acid sequences described herein, means to be connected in an unbroken sequence. For example, for a first sequence to comprise 30 contiguous (or consecutive) amino acids of a second sequence, means that the first sequence includes an unbroken sequence of 30 amino acid residues that is 100% identical to an unbroken sequence of 30 amino acid residues in the second sequence. Similarly, for a first sequence to have “100% identity” with a second sequence means that the first sequence exactly matches the second sequence with no gaps between nucleotides or amino acids.</p>
    <p num="p-0176">In another embodiment, a protein referenced or used in the present invention, including a homologue, includes a protein having an amino acid sequence that is sufficiently similar to the naturally occurring protein amino acid sequence that a nucleic acid sequence encoding the homologue is capable of hybridizing under moderate, high, or very high stringency conditions (described below) to (i.e., with) a nucleic acid molecule encoding the naturally occurring protein (i.e., to the complement of the nucleic acid strand encoding the naturally occurring protein). Preferably, a given homologue is encoded by a nucleic acid sequence that hybridizes under moderate, high or very high stringency conditions to the complement of a nucleic acid sequence that encodes the wild-type or reference protein.</p>
    <p num="p-0177">A nucleic acid sequence complement of reference nucleic acid sequence refers to the nucleic acid sequence of the nucleic acid strand that is complementary to the strand which encodes a protein. It will be appreciated that a double stranded DNA which encodes a given amino acid sequence comprises a single strand DNA and its complementary strand having a sequence that is a complement to the single strand DNA. As such, nucleic acid molecules of the present invention can be either double-stranded or single-stranded, and include those nucleic acid molecules that form stable hybrids under stringent hybridization conditions with a nucleic acid sequence that encodes an amino acid sequence of a protein, and/or with the complement of the nucleic acid sequence that encodes such protein. Methods to deduce a complementary sequence are known to those skilled in the art. It should be noted that since amino acid sequencing and nucleic acid sequencing technologies are not entirely error-free, the sequences presented herein, at best, represent apparent sequences of the referenced proteins of the present invention.</p>
    <p num="p-0178">As used herein, reference to hybridization conditions refers to standard hybridization conditions under which nucleic acid molecules are used to identify similar nucleic acid molecules. Such standard conditions are disclosed, for example, in Sambrook et al., <i>Molecular Cloning: A Laboratory Manual</i>, Cold Spring Harbor Labs Press, 1989. Sambrook et al., ibid., is incorporated by reference herein in its entirety (see specifically, pages 9.31-9.62). In addition, formulae to calculate the appropriate hybridization and wash conditions to achieve hybridization permitting varying degrees of mismatch of nucleotides are disclosed, for example, in Meinkoth et al., 1984<i>, Anal. Biochem</i>. 138, 267-284; Meinkoth et al., ibid., is incorporated by reference herein in its entirety.</p>
    <p num="p-0179">More particularly, moderate stringency hybridization and washing conditions, as referred to herein, refer to conditions which permit isolation of nucleic acid molecules having at least about 70% nucleic acid sequence identity with the nucleic acid molecule being used to probe in the hybridization reaction (i.e., conditions permitting about 30% or less mismatch of nucleotides). High stringency hybridization and washing conditions, as referred to herein, refer to conditions which permit isolation of nucleic acid molecules having at least about 80% nucleic acid sequence identity with the nucleic acid molecule being used to probe in the hybridization reaction (i.e., conditions permitting about 20% or less mismatch of nucleotides). Very high stringency hybridization and washing conditions, as referred to herein, refer to conditions which permit isolation of nucleic acid molecules having at least about 90% nucleic acid sequence identity with the nucleic acid molecule being used to probe in the hybridization reaction (i.e., conditions permitting about 10% or less mismatch of nucleotides). As discussed above, one of skill in the art can use the formulae in Meinkoth et al., ibid. to calculate the appropriate hybridization and wash conditions to achieve these particular levels of nucleotide mismatch. Such conditions will vary, depending on whether DNA:RNA or DNA:DNA hybrids are being formed. Calculated melting temperatures for DNA:DNA hybrids are 10° C. less than for DNA:RNA hybrids. In particular embodiments, stringent hybridization conditions for DNA:DNA hybrids include hybridization at an ionic strength of 6×SSC (0.9 M Na<sup>+</sup>) at a temperature of between about 20° C. and about 35° C. (lower stringency), more preferably, between about 28° C. and about 40° C. (more stringent), and even more preferably, between about 35° C. and about 45° C. (even more stringent), with appropriate wash conditions. In particular embodiments, stringent hybridization conditions for DNA:RNA hybrids include hybridization at an ionic strength of 6×SSC (0.9 M Na<sup>+</sup>) at a temperature of between about 30° C. and about 45° C., more preferably, between about 38° C. and about 50° C., and even more preferably, between about 45° C. and about 55° C., with similarly stringent wash conditions. These values are based on calculations of a melting temperature for molecules larger than about 100 nucleotides, 0% formamide and a G+C content of about 40%. Alternatively, T<sub>m </sub>can be calculated empirically as set forth in Sambrook et al., supra, pages 9.31 to 9.62. In general, the wash conditions should be as stringent as possible, and should be appropriate for the chosen hybridization conditions. For example, hybridization conditions can include a combination of salt and temperature conditions that are approximately 20-25° C. below the calculated T<sub>m </sub>of a particular hybrid, and wash conditions typically include a combination of salt and temperature conditions that are approximately 12-20° C. below the calculated T<sub>m </sub>of the particular hybrid. One example of hybridization conditions suitable for use with DNA:DNA hybrids includes a 2-24 hour hybridization in 6×SSC (50% formamide) at about 42° C., followed by washing steps that include one or more washes at room temperature in about 2×SSC, followed by additional washes at higher temperatures and lower ionic strength (e.g., at least one wash as about 37° C. in about 0.1×-0.5×SSC, followed by at least one wash at about 68° C. in about 0.1×-0.5×SSC).</p>
    <p num="p-0180">Homologues can, in one embodiment, be the result of natural allelic variation or natural mutation. Homologues of a given protein can also be naturally occurring proteins having substantially the same function from different organisms with at least some structural similarity (e.g., at least about 35% identity) to one another at the nucleic acid or amino acid level as described herein. Homologues of the present invention can also be produced using techniques known in the art including, but not limited to, direct modifications to the protein or modifications to the gene encoding the protein using, for example, classic or recombinant DNA techniques to effect random or targeted mutagenesis. A naturally occurring allelic variant of a nucleic acid encoding a given protein is a gene that occurs at essentially the same locus (or loci) in the genome as the gene which encodes the given protein, but which, due to natural variations caused by, for example, mutation or recombination, has a similar but not identical sequence. Natural allelic variants typically encode proteins having similar activity to that of the protein encoded by the gene to which they are being compared. One class of allelic variants can encode the same protein but have different nucleic acid sequences due to the degeneracy of the genetic code. Allelic variants can also comprise alterations in the 5′ or 3′ untranslated regions of the gene (e.g., in regulatory control regions). Allelic variants are well known to those skilled in the art.</p>
    <p num="p-0181">The minimum size of a protein and/or homologue of the present invention is, in one aspect, a size sufficient to have the desired biological activity of the protein. In another embodiment, a protein of the present invention is at least 30 amino acids long, and more preferably, at least about 50, and more preferably at least 75, and more preferably at least 100, and more preferably at least 115, and more preferably at least 130, and more preferably at least 150, and more preferably at least 200, and more preferably, at least 250, and more preferably, at least 300, and more preferably, at least 350 amino acids long. There is no limit, other than a practical limit, on the maximum size of such a protein in that the protein can include a portion of a given protein or a full-length protein, plus additional sequence (e.g., a fusion protein sequence), if desired. Suitable fusion segments for use with the present invention include, but are not limited to, segments that can: enhance a protein's stability; provide other desirable biological activity (e.g., a second enzyme function); and/or assist with the purification of a protein (e.g., by affinity chromatography).</p>
    <p num="p-0182">In one embodiment of the present invention, any of the amino acid sequences described herein can be produced with from at least one, and up to about 20, additional heterologous amino acids flanking each of the C- and/or N-terminal ends of the specified amino acid sequence. The resulting protein or polypeptide can be referred to as “consisting essentially of” the specified amino acid sequence. According to the present invention, the heterologous amino acids are a sequence of amino acids that are not naturally found (i.e., not found in nature, in vivo) flanking the specified amino acid sequence, or that are not related to the function of the specified amino acid sequence, or that would not be encoded by the nucleotides that flank the naturally occurring nucleic acid sequence encoding the specified amino acid sequence as it occurs in the gene, if such nucleotides in the naturally occurring sequence were translated using standard codon usage for the organism from which the given amino acid sequence is derived. Similarly, the phrase “consisting essentially of”, when used with reference to a nucleic acid sequence herein, refers to a nucleic acid sequence encoding a specified amino acid sequence that can be flanked by from at least one, and up to as many as about 60, additional heterologous nucleotides at each of the 5′ and/or the 3′ end of the nucleic acid sequence encoding the specified amino acid sequence. The heterologous nucleotides are not naturally found (i.e., not found in nature, in vivo) flanking the nucleic acid sequence encoding the specified amino acid sequence as it occurs in the natural gene or do not encode a protein that imparts any additional function to the protein or changes the function of the protein having the specified amino acid sequence.</p>
    <p num="p-0183">Embodiments of the present invention include the use and/or manipulation of nucleic acid molecules that encode enzymes or other proteins in the amino sugar metabolic pathways described herein. A nucleic acid molecule of the present invention includes a nucleic acid molecule comprising, consisting essentially of, or consisting of, a nucleic acid sequence encoding any of the enzymes or other proteins described herein.</p>
    <p num="p-0184">In accordance with the present invention, an isolated nucleic acid molecule is a nucleic acid molecule that has been removed from its natural milieu (i.e., that has been subject to human manipulation), its natural milieu being the genome or chromosome in which the nucleic acid molecule is found in nature. As such, “isolated” does not necessarily reflect the extent to which the nucleic acid-molecule has been purified, but indicates that the molecule does not include an entire genome or an entire chromosome in which the nucleic acid molecule is found in nature. An isolated nucleic acid molecule can include a gene, such as a glucosamine-6-phosphate synthase gene described herein. An isolated nucleic acid molecule that includes a gene is not a fragment of a chromosome that includes such gene, but rather includes the coding region and regulatory regions associated with the gene, but no additional genes naturally found on the same chromosome. An isolated nucleic acid molecule can also include a specified nucleic acid sequence flanked by (i.e., at the 5′ and/or the 3′ end of the sequence) additional nucleic acids that do not normally flank the specified nucleic acid sequence in nature (i.e., are heterologous sequences). Isolated nucleic acid molecules can include DNA, RNA (e.g., mRNA), or derivatives of either DNA or RNA (e.g., cDNA). Although the phrase “nucleic acid molecule” primarily refers to the physical nucleic acid molecule and the phrase “nucleic acid sequence” primarily refers to the sequence of nucleotides on the nucleic acid molecule, the two phrases can be used interchangeably, especially with respect to a nucleic acid molecule, or a nucleic acid sequence, being capable of encoding a protein.</p>
    <p num="p-0185">Preferably, an isolated nucleic acid molecule of the present invention is produced using recombinant DNA technology (e.g., polymerase chain reaction (PCR) amplification, cloning) or chemical synthesis. Isolated nucleic acid molecules include natural nucleic acid molecules and homologues thereof, including, but not limited to, natural allelic variants and modified nucleic acid molecules in which nucleotides have been inserted, deleted, substituted, and/or inverted in such a manner that such modifications provide the desired effect on protein biological activity. Allelic variants and protein homologues (e.g., proteins encoded by nucleic acid homologues) have been discussed in detail above.</p>
    <p num="p-0186">A nucleic acid molecule homologue can be produced using a number of methods known to those skilled in the art (see, for example, Sambrook et al., ibid.). For example, nucleic acid molecules can be modified using a variety of techniques including, but not limited to, classical mutagenesis techniques and recombinant DNA techniques, such as site-directed mutagenesis, chemical treatment of a nucleic acid molecule to induce mutations, restriction enzyme cleavage of a nucleic acid fragment, ligation of nucleic acid fragments, PCR amplification and/or mutagenesis of selected regions of a nucleic acid sequence, synthesis of oligonucleotide mixtures and ligation of mixture groups to “build” a mixture of nucleic acid molecules and combinations thereof. Nucleic acid molecule homologues can be selected from a mixture of modified nucleic acids by screening for the function of the protein encoded by the nucleic acid and/or by hybridization with a wild-type gene.</p>
    <p num="p-0187">The minimum size of a nucleic acid molecule of the present invention is a size sufficient to encode a protein having the desired biological activity, or sufficient to form a probe or oligonucleotide primer that is capable of forming a stable hybrid with the complementary sequence of a nucleic acid molecule encoding the natural protein (e.g., under moderate, high or very high stringency conditions, and preferably under very high stringency conditions). As such, the size of a nucleic acid molecule of the present invention can be dependent on nucleic acid composition and percent homology or identity between the nucleic acid molecule and complementary sequence as well as upon hybridization conditions per se (e.g., temperature, salt concentration, and formamide concentration). The minimal size of a nucleic acid molecule that is used as an oligonucleotide primer or as a probe is typically at least about 12 to about 15 nucleotides in length if the nucleic acid molecules are GC-rich and at least about 15 to about 18 bases in length if they are AT-rich. There is no limit, other than a practical limit, on the maximal size of a nucleic acid molecule of the present invention, in that the nucleic acid molecule can include a portion of a protein encoding sequence, a nucleic acid sequence encoding a full-length protein (including a complete gene).</p>
    <p num="p-0188">Knowing the nucleic acid sequences of certain nucleic acid molecules of the present invention, and particularly any of the nucleic acid molecules described in detail herein, allows one skilled in the art to, for example, (a) make copies of those nucleic acid molecules and/or (b) obtain nucleic acid molecules including at least a portion of such nucleic acid molecules (e.g., nucleic acid molecules including full-length genes, full-length coding regions, regulatory control sequences, truncated coding regions). Such nucleic acid molecules can be obtained in a variety of ways including traditional cloning techniques using oligonucleotide probes of to screen appropriate libraries or DNA and PCR amplification of appropriate libraries or DNA using oligonucleotide primers. Preferred libraries to screen or from which to amplify nucleic acid molecule include bacterial and yeast genomic DNA libraries, and in particular, <i>Escherichia coli </i>genomic DNA libraries. Techniques to clone and amplify genes are disclosed, for example, in Sambrook et al., ibid.</p>
    <p num="p-0189">Another embodiment of the present invention includes a recombinant nucleic acid molecule comprising a recombinant vector and a nucleic acid molecule comprising a nucleic acid sequence encoding an amino acid sequence having a biological activity of any of the enzymes or other proteins in an amino sugar metabolic pathway as described herein. According to the present invention, a recombinant vector is an engineered (i.e., artificially produced) nucleic acid molecule that is used as a tool for manipulating a nucleic acid sequence of choice and for introducing such a nucleic acid sequence into a host cell. The recombinant vector is therefore suitable for use in cloning, sequencing, and/or otherwise manipulating the nucleic acid sequence of choice, such as by expressing and/or delivering the nucleic acid sequence of choice into a host cell to form a recombinant cell. Such a vector typically contains heterologous nucleic acid sequences, that is nucleic acid sequences that are not naturally found adjacent to nucleic acid sequence to be cloned or delivered, although the vector can also contain regulatory nucleic acid sequences (e.g., promoters, untranslated regions) which are naturally found adjacent to nucleic acid molecules of the present invention or which are useful for expression of the nucleic acid molecules of the present invention (discussed in detail below). The vector can be either RNA or DNA, either prokaryotic or eukaryotic, and typically is a plasmid. The vector can be maintained as an extrachromosomal element (e.g., a plasmid) or it can be integrated into the chromosome of a recombinant organism (e.g., a microbe or a plant). The entire vector can remain in place within a host cell, or under certain conditions, the plasmid DNA can be deleted, leaving behind the nucleic acid molecule of the present invention. The integrated nucleic acid molecule can be under chromosomal promoter control, under native or plasmid promoter control, or under a combination of several promoter controls. Single or multiple copies of the nucleic acid molecule can be integrated into the chromosome. A recombinant vector of the present invention can contain at least one selectable marker.</p>
    <p num="p-0190">In one embodiment, a recombinant vector used in a recombinant nucleic acid molecule of the present invention is an expression vector. As used herein, the phrase “expression vector” is used to refer to a vector that is suitable for production of an encoded product (e.g., a protein of interest). In this embodiment, a nucleic acid sequence encoding the product to be produced is inserted into the recombinant vector to produce a recombinant nucleic acid molecule. The nucleic acid sequence encoding the protein to be produced is inserted into the vector in a manner that operatively links the nucleic acid sequence to regulatory sequences in the vector which enable the transcription and translation of the nucleic acid sequence within the recombinant host cell.</p>
    <p num="p-0191">In another embodiment, a recombinant vector used in a recombinant nucleic acid molecule of the present invention is a targeting vector. As used herein, the phrase “targeting vector” is used to refer to a vector that is used to deliver a particular nucleic acid molecule into a recombinant host cell, wherein the nucleic acid molecule is used to delete or inactivate an endogenous gene within the host cell or microorganism (i.e., used for targeted gene disruption or knock-out technology). Such a vector may also be known in the art as a “knock-out” vector. In one aspect of this embodiment, a portion of the vector, but more typically, the nucleic acid molecule inserted into the vector (i.e., the insert), has a nucleic acid sequence that is homologous to a nucleic acid sequence of a target gene in the host cell (i.e., a gene which is targeted to be deleted or inactivated). The nucleic acid sequence of the vector insert is designed to bind to the target gene such that the target gene and the insert undergo homologous recombination, whereby the endogenous target gene is deleted, inactivated or attenuated (i.e., by at least a portion of the endogenous target gene being mutated or deleted).</p>
    <p num="p-0192">Typically, a recombinant nucleic acid molecule includes at least one nucleic acid molecule of the present invention operatively linked to one or more expression control sequences, including transcription control sequences and translation control sequences. As used herein, the phrase “recombinant molecule” or “recombinant nucleic acid molecule” primarily refers to a nucleic acid molecule or nucleic acid sequence operatively linked to an expression control sequence, but can be used interchangeably with the phrase-“nucleic acid molecule”, when such nucleic acid molecule is a recombinant molecule as discussed herein. According to the present invention, the phrase “operatively linked” refers to linking a nucleic acid molecule to an expression control sequence (e.g., a transcription control sequence and/or a translation-control sequence) in a manner such that the molecule is able to be expressed when transfected (i.e., transformed, transduced, transfected, conjugated or conduced) into a host cell. Transcription control sequences are sequences which control the initiation, elongation, or termination of transcription. Particularly important transcription control sequences are those which control transcription initiation, such as promoter, enhancer, operator and repressor sequences. Suitable transcription control sequences include any transcription control sequence that can function in a host cell or organism into which the recombinant nucleic acid molecule is to be introduced.</p>
    <p num="p-0193">Recombinant nucleic acid molecules of the present invention can also contain additional regulatory sequences, such as translation regulatory sequences, origins of replication, and other regulatory sequences that are compatible with the recombinant cell. In one embodiment, a recombinant molecule of the present invention, including those which are integrated into the host cell chromosome, also contains secretory signals (i.e., signal segment nucleic acid sequences) to enable an expressed protein to be secreted from the cell that produces the protein. Suitable signal segments include a signal segment that is naturally associated with the protein to be expressed or any heterologous signal segment capable of directing the secretion of the protein according to the present invention. In another embodiment, a recombinant molecule of the present invention comprises a leader sequence to enable an expressed protein to be delivered to and inserted into the membrane of a host cell. Suitable leader sequences include a leader sequence that is naturally associated with the protein, or any heterologous leader sequence capable of directing the delivery and insertion of the protein to the membrane of a cell.</p>
    <p num="p-0194">According to the present invention, the term “transfection” is used to refer to any method by which an exogenous nucleic acid molecule (i.e., a recombinant nucleic acid molecule) can be inserted into a cell. The term “transformation” can be used interchangeably with the term “transfection” when such term is used to refer to the introduction of nucleic acid molecules into microbial cells, such as algae, bacteria and yeast, or into plant cells. In microbial systems and plant systems, the term “transformation” is used to describe an inherited change due to the acquisition of exogenous nucleic acids by the microorganism or plant and is essentially synonymous with the term “transfection.” Therefore, transfection techniques include, but are not limited to, transformation, chemical treatment of cells, particle bombardment, electroporation, microinjection, lipofection, adsorption, infection and protoplast fusion.</p>
    <p num="p-0195">A recombinant cell is preferably produced by transforming a bacterial or yeast cell (i.e., a host cell) with one or more recombinant molecules, each comprising one or more nucleic acid molecules operatively linked to an expression vector containing one or more transcription control sequences. The phrase, operatively linked, refers to insertion of a nucleic acid molecule into an expression vector in a manner such that the molecule is able to be expressed when transformed into a host cell. As used herein, an expression vector is a DNA or RNA vector that is capable of transforming a host cell and of effecting expression of a specified nucleic acid molecule. Preferably, the expression vector is also capable of replicating within the host cell. In the present invention, expression vectors are typically plasmids. Expression vectors of the present invention include any vectors that function (i.e., direct gene expression) in a yeast host cell or a bacterial host cell, preferably an <i>Escherichia coli </i>host cell. Preferred recombinant cells of the present invention are set forth in the Examples section.</p>
    <p num="p-0196">Nucleic acid molecules of the present invention can be operatively linked to expression vectors containing regulatory sequences such as transcription control sequences translation control sequences, origins of replication, and other regulatory sequences that are compatible with the recombinant cell and that control the expression of nucleic acid molecules of the present invention. In particular, recombinant molecules of the present invention include transcription control sequences. Transcription control sequences are sequences which control the initiation, elongation, and termination of transcription. Particularly important transcription control sequences are those which control transcription initiation, such as promoter, enhancer, operator and repressor sequences. Suitable transcription control sequences include any transcription control sequence that can function in yeast or bacterial cells and preferably, <i>Escherichia coli</i>. A variety of such transcription control sequences are known to those skilled in the art.</p>
    <p num="p-0197">It is preferred that the recombinant nucleic acid molecules comprising nucleic acid sequences encoding various enzymes and proteins described herein (including homologues thereof) be cloned under control of an artificial promoter. The promoter can be any suitable promoter that will provide a level of gene expression required to maintain a sufficient level of the encoded protein in the production organism. Suitable promoters can be promoters inducible by different chemicals (such as lactose, galactose, maltose and salt) or changes of growth conditions (such as temperature). Use of inducible promoter can lead to an optimal performance of gene expression and fermentation process. Preferred promoters can also be constitutive promoters, since the need for addition of expensive inducers is therefore obviated. Such promoters include normally inducible promoter systems that have been made functionally constitutive or “leaky” by genetic modification, such as by using a weaker, mutant repressor gene. Particularly preferred promoters to be used are lac, P<sub>L </sub>and T7. The gene dosage (copy number) can be varied according to the requirements for maximum product formation. In one embodiment, the recombinant genes are integrated into the host genome.</p>
    <p num="p-0198">It may be appreciated by one skilled in the art that use of recombinant DNA technologies can improve expression of transformed nucleic acid molecules by manipulating, for example, the number of copies of the nucleic acid molecules within a host cell, the efficiency with which those nucleic acid molecules are transcribed, the efficiency with which the resultant transcripts are translated, and the efficiency of post-translational modifications. Recombinant techniques useful for increasing the expression of nucleic acid molecules of the present, invention include, but are not limited to, operatively linking nucleic acid molecules to high-copy number plasmids, integration of the nucleic acid molecules into the host cell chromosome, addition of vector stability sequences to plasmids, substitutions or modifications of transcription control signals (e.g., promoters, operators, enhancers), substitutions or modifications of translational control signals, modification of nucleic acid molecules of the present invention to correspond to the codon usage of the host cell, deletion of sequences that destabilize transcripts, and use of control signals that temporally separate recombinant cell growth from recombinant enzyme production during fermentation. The activity of an expressed recombinant protein of the present invention may be improved by fragmenting, modifying, or derivatizing nucleic acid molecules encoding such a protein. Such modifications are described in detail in the Examples section.</p>
    <p num="p-0199">One or more recombinant molecules of the present invention can be used to produce an encoded product of the present invention. In one embodiment, an encoded product is produced by expressing a nucleic acid molecule as described herein under conditions effective to produce the protein. A preferred method to produce an encoded protein is by transfecting a host cell with one or more recombinant molecules to form a recombinant cell. Suitable cells are any microorganisms (e.g., a host cell or production organism) is any microorganism (e.g., a bacterium, a protist, an alga, a fungus, or other microbe), and is most preferably a bacterium, a yeast or a fungus. Suitable bacterial genera include, but are not limited to, <i>Escherichia, Bacillus, Lactobacillus, Pseudomonas </i>and <i>Streptomyces</i>. Suitable bacterial species include, but are not limited to, <i>Escherichia coli, Bacillus subtilis, Bacillus licheniformis, Lactobacillus brevis, Pseudomonas aeruginosa </i>and <i>Streptomyces lividans</i>. Suitable genera of yeast include, but are not limited to, <i>Saccharomyces, Schizosaccharomyces, Candida, Hansenula, Pichia, Kluyveromyces</i>, and <i>Phaffia</i>. Suitable yeast species include, but are not limited to, <i>Saccharomyces cerevisiae, Schizosaccharomyces pombe, Candida albicans, Hansenula polymorpha, Pichia pastoris, P. canadensis, Kluyveromyces marxianus </i>and <i>Phaffia rhodozyma</i>. Suitable fungal genera include, but are not limited to, <i>Aspergillus, Absidia, Rhizopus, Chrysosporium, Neurospora </i>and <i>Trichoderma</i>. Suitable fungal species include, but are not limited to, <i>Aspergillus niger, A. nidulans, Absidia coerulea, Rhizopus oryzae, Chrysosporium lucknowense, Neurospora crassa, N. intermedia </i>and <i>Trichoderm reesei</i>. Host cells can be either untransfected cells or cells that are already transfected with at least one other recombinant nucleic acid molecule.</p>
    <p num="p-0200">Additional embodiments of the present invention include any of the genetically modified microorganisms described herein and microorganisms having the identifying characteristics of the microorganisms specifically identified in the Examples. Such identifying characteristics can include any or all genotypic and/or phenotypic characteristics of the microorganisms in the Examples, including their abilities to produce glucosamine and/or N-acetylglucosamine.</p>
    <p num="p-0201">As noted above, in the method for production of glucosamine and/or N-acetylglucosamine of the present invention, a microorganism having a genetically modified amino sugar metabolic pathway is cultured in a fermentation medium for production of glucosamine and/or N-acetylglucosamine. An appropriate, or effective, fermentation medium refers to any medium in which a genetically modified microorganism of the present invention, when cultured, is capable of producing glucosamine and/or N-acetylglucosamine. Such a medium is typically an aqueous medium comprising assimilable carbon, nitrogen and phosphate sources. Such a medium can also include appropriate salts, minerals, metals and other nutrients. One advantage of the genetic modifications to a microorganism described herein is that although such genetic modifications significantly alter the metabolism of amino sugars, they do not create any nutritional requirements for the production organism. Thus, a minimal-salts medium containing glucose, fructose, lactose, glycerol or a mixture of two or more different compounds as the sole carbon source is preferably used as the fermentation medium. The use of a minimal-salts-glucose medium is the most preferred medium for the glucosamine and/or N-acetylglucosamine fermentation and it will also facilitate recovery and purification of the products. In one aspect, yeast extract is a component of the medium.</p>
    <p num="p-0202">Microorganisms of the present invention can be cultured in conventional fermentation bioreactors. The microorganisms can be cultured by any fermentation process which includes, but is not limited to, batch, fed-batch, cell recycle, and continuous fermentation. Preferably, microorganisms of the present invention are grown by batch or fed-batch fermentation processes.</p>
    <p num="p-0203">In one embodiment of the present invention, before inoculation, the fermentation medium is brought up to the desired temperature, typically from about 20° C. to about 45° C., preferably from about 25° C. to about 45° C., or 25° C. to about 40° C., with temperatures of from about 25° C. to about 37° C., and in some embodiments, about 30° C. or about 37° C. being more preferred. Fermentation conditions can include culturing the microorganisms of the invention at any temperature between about 20° C. and about 40° C., in whole increments (i.e., 21° C., 22° C., etc.). It is noted that the optimum temperature for growth and glucosamine and/or N-acetylglucosamine production by a microorganism of the present invention can vary according to a variety of factors. For example, the selection of a particular promoter for expression of a recombinant nucleic acid molecule in the microorganism can affect the optimum culture temperature. One of ordinary skill in the art can readily determine the optimum growth and glucosamine and/or N-acetylglucosamine production temperature for any microorganism of the present invention using standard techniques, such as those described in the Examples section for one microorganism of the present invention.</p>
    <p num="p-0204">The medium is inoculated with an actively growing culture of the genetically modified microorganism in an amount sufficient to produce, after a reasonable growth period, a high cell density. The cells are grown to a cell density of at least about 10 g/l, preferably between about 10 g/l and about 40 g/l, and more preferably at least about 40 g/l. This process typically requires about 10-60 hours.</p>
    <p num="p-0205">Sufficient oxygen must be added to the medium during the course of the fermentation to maintain cell growth during the initial cell growth and to maintain metabolism, and glucosamine and/or N-acetylglucosamine production. Oxygen is conveniently provided by agitation and aeration of the medium. Conventional methods, such as stirring or shaking, may be used to agitate and aerate the medium. Preferably the oxygen concentration in the medium is greater than about 15% of the saturation value (i.e., the solubility of oxygen in the medium at atmospheric pressure and about 30-40° C.) and more preferably greater than about 20% of the saturation value, although excursions to lower concentrations may occur if fermentation is not adversely affected. It is further understood that the oxygen level can be allowed to reach very low levels for any appropriate amount of time during the fermentation if it enhances stability and formation of glucosamine and/or N-acetylglucosamine during the production process. The oxygen concentration of the medium can be monitored by conventional methods, such as with an oxygen electrode. Other sources of oxygen, such as undiluted oxygen gas and oxygen gas diluted with inert gas other than nitrogen, can be used.</p>
    <p num="p-0206">Since the production of glucosamine and/or N-acetylglucosamine by fermentation is preferably based on using glucose as the sole carbon source, in a preferred embodiment, in <i>Escherichia coli</i>, the PEP:glucose PTS will be induced. Accordingly, even in the absence of a functional EIIM,P/III<sup>Man </sup>of the PEP:mannose PTS (e.g., in an <i>Escherichia coli </i>having a manXYZ mutation), the product, glucosamine, will still be taken up by the cells via the induced glucose transport system. In the presence of excess glucose, however, uptake of glucosamine is severely repressed. Thus, it is one embodiment of the present invention to prevent uptake of the glucosamine product by maintaining an excess of glucose in the fermentation bioreactor. As used herein, “an excess” of glucose refers to an amount of glucose above that which is required to maintain the growth of the microorganism under normal conditions, such as the culturing conditions described above.</p>
    <p num="p-0207">Preferably, the glucose concentration is maintained at a concentration of from about 0.05% to about 15% weight/volume of the fermentation medium. In another embodiment, the glucose concentration is maintained at a concentration of from about 0.5 g/L to about 150 g/L of the fermentation medium, and even more preferably, from about 5 g/L to about 100 g/L of the fermentation medium, and even more preferably from about 5 g/L to about 20 g/L. In one embodiment, the glucose concentration of the fermentation medium is monitored by any suitable method (e.g., by using glucose test strips), and when the glucose concentration is at or near depletion, additional glucose can be added to the medium. In another embodiment, the glucose concentration is maintained by semi-continuous or continuous feeding of the fermentation medium. The parameters disclosed herein for glucose can be applied to any carbon source used in the fermentation medium of the present invention. It is further understood that the carbon source can be allowed to reach undetectable levels for any appropriate amount of time during the fermentation if it enhances the glucosamine and/or N-acetylglucosamine production process. Other carbon sources that can be used in the fermentation method of the present invention, include, but are not limited to, fructose, a pentose sugar, lactose and gluconic acid. Pentose sugars include, but are not limited to, ribose, xylose, and arabinose. In one aspect, the step of culturing is performed in a fermentation medium comprising glucose and ribose.</p>
    <p num="p-0208">It is a further embodiment of the present invention to supplement and/or control other components and parameters of the fermentation medium, as necessary to maintain and/or enhance the production and stability of glucosamine and/or N-acetylglucosamine. For example, in one embodiment, the fermentation medium includes ammonium sulfate, and the ammonium sulfate concentration in the culture medium is supplemented by the addition of excess ammonium sulfate. Preferably, the amount of ammonium sulfate is maintained at a level of from about 0.1% to about 1% (weight/volume) in the fermentation medium, and preferably, at about 0.5%. In yet another embodiment, the pH of the fermentation medium is monitored for fluctuations in pH. In the fermentation method of the present invention, the pH is preferably maintained at a pH of from about pH 4.0 to about pH 8.0, an in one aspect, at a pH of from about pH 4 to about pH 7.5, and in another aspect, at a pH of from about pH 6.7 to about pH 7.5, and in another embodiment, at a pH of from about pH 4.5 to about pH 5. While preferred embodiments are described above, the fermentation process can be conducted at any pH between pH 4 and pH 8, in increments of 0.1 (i.e., pH 4.1, pH 4.2, pH 4.3, etc.). In the method of the present invention, if the starting pH of the fermentation medium is pH 7.0, by way of example, the pH of the fermentation medium is monitored for significant variations from pH 7.0, and is adjusted accordingly, for example, by the addition of sodium hydroxide. Since the optimal pH for stability and formation rates of glucosamine and/or N-acetylglucosamine can be different from the optimal pH for cell growth, a protocol of two phases (or more than two phases) can be used. With such protocols, cells are initially grown at a pH optimal for fast production of biomass, then a different pH is used to maximize synthesis of glucosamine and/or N-acetylglucosamine while minimizing degradation of glucosamine and/or N-acetylglucosamine.</p>
    <p num="p-0209">A further embodiment of the present invention is to redirect carbon flux from acetate production to the production of less toxic byproducts. By such methods, problems of toxicity associated with an excess of glucose in the fermentation medium can be avoided. Methods to redirect carbon flux from acetate production are known in the art.</p>
    <p num="p-0210">In a batch, fed-batch and/or continuous fermentation-process of the present invention, fermentation is continued until the formation of glucosamine and/or N-acetylglucosamine, as evidenced by the accumulation of extracellular glucosamine and/or N-acetylglucosamine, essentially ceases. The total fermentation time is typically from about 40 to about 60 hours, and more preferably, about 48 hours. In a continuous fermentation process, glucosamine and/or N-acetylglucosamine can be removed from the bioreactor as it accumulates in the medium. The method of the present invention results in production of a product which can include intracellular glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine and/or glucosamine; or extracellular glucosamine or N-acetylglucosamine.</p>
    <p num="p-0211">For glucosamine and N-acetylglucosamine synthesis, different genes were overexpressed using the <i>E. coli </i>pET expression system. Gene expression is inducible by IPTG and lactose. The cost of IPTG would make glucosamine manufacture prohibitively expensive. Therefore, the use of IPTG would ideally be eliminated from the fermentation process. Lactose is relatively inexpensive and can be used in large scale production process. For lactose induction, lactose needs first to be converted to allolactose, the true inducer, by β-galactosidase in the cell. With glucosamine production strains such as 2123-54, lactose could not be used as inducer since the strain is negative for β-galactosidase. This was due to the deletion and disruption of the lacZ gene by the insertion of T7-glmS*54 expression cassette at the locus.</p>
    <p num="p-0212">In principle, several different methods could be used to eliminate the dependence on IPTG. One approach illustrated in the present invention is to restore the lacZ gene. This was accomplished by integrating the T7-glmS*54 expression cassette at a different site other than lacZ in the chromosome;
</p> <ul> <li id="ul0001-0001" num="0000"> <ul> <li id="ul0002-0001" num="0212">Integration at the galK site would leave the lacZ gene intact. Lac<sup>+</sup> strains are potentially inducible by lactose, which is much less expensive than IPTG. The galK site was chosen for integration of the T7-glmS*54 expression cassette because the integrant stains would also be Gal<sup>−</sup>. It was reported that in such strains galactose could be used as an inducer for lac promoters. Therefore, in addition to induction by allolactose, galactose generated through lactose hydrolysis could further enhance the induction. This maybe beneficial when a sub-optimal amount of lactose was used in the glucosamine production process. Integration could also be at any other site where an insertion would not cause any negative impact on cell growth and N-acetylglucosamine or/glucosamine production.</li> </ul> </li> </ul> <p num="p-0213">Lactose induction is subject to glucose repression. Glucosamine production did not occur when a high level of glucose was present in the culture. However, once glucose was consumed, lactose was utilized and glucosamine production was induced. The ratio of glucose versus lactose was shown to affect enzyme expression and glucosamine/N-acetylglucosamine production. Glucose repression takes place by repressing the native lac operon. In the presence of glucose, the synthesis of lactose transporter (LacY) and β-galactosidase (LacZ) were repressed, thus few inducer molecules (allolactose and galactose) were produced for the induction of T7 RNA polymerase. The detailed mechanism of glucose repression is still a subject of investigation. It appears that glucose repression acts on two levels. One level of repression is through the cAMP mediated action on the lac promoter sequence. The other is through the inhibition of lactose uptake, which is caused by glucose transporter proteins, i.e. enzyme IIA<sup>Glc </sup>(encoded by the crr gene) and IICB<sup>Glc </sup>(encoded by the ptsG gene). The lacUV5 promoter is not sensitive to glucose repression, but exclusion of lactose (the inducer) from entering the cells prevents induction.</p>
    <p num="p-0214">Glucose repression could be minimized by using optimized growth and induction protocols. Different genetic modifications could also be introduced to minimize glucose repression. One approach was to replace the native lac promoter in the lac operon by the lacUV5 promoter, which is believed to not be repressed by glucose. Glucose repression could also be minimized by genetic modifications of the crr gene. Another approach was to delete one of the lacI repressor genes. In some glucosamine and N-acetylglucosamine production <i>E. coli </i>strains, there are two copies of lacI repressor gene, one in the native lac operon, the other in the DE3 element. Deletion of either lacI gene increased the resistance to glucose repression and glucosamine/N-acetylglucosamine production. Since lactose is transported into the cells through the lactose permease encoded by lacy, lactose induction could be affected by lacY over-expression.</p>
    <p num="p-0215">Glucosamine and or N-acetylglucosamine levels are not only determined by the rate of synthesis but also by the rate of cellular metabolism and degradation if the product is not stable under the fermentation conditions. As disclosed in the present invention, glucosamine was found to be very unstable at the typical pH range used for <i>E. coli </i>growth. Glucosamine and or its degradation products also caused toxic effects on strain 7107-18. Toxicity was observed even when glucosamine at concentration as low as 20 g l<sup>−1 </sup>was pre-incubated in the medium (pH 7.0) for 3.5 hrs prior to cell inoculation. The toxicity was attributed to at least partially to GlcN degradation products in media with a starting pH of 7.0. GlcN is more stable at lower pH, glucosamine is not degraded at pH 4.7 or below.</p>
    <p num="p-0216">For developing an economic process, glucosamine synthesis must be maximized while product degradation minimized. In addition, a GlcN production protocol operated in fermentors at relatively low pH would not only preserve the synthesized GlcN, this process would also protect the cells by reducing the concentration of toxic breakdown products. These benefits must be balanced against the reduced growth rate and metabolic activity of cells grown this way. Continued GlcN synthesis requires the constant generation of energy in the cells, and cells growing slowly at these lower pHs may not be able to generate enough of the energy required.</p>
    <p num="p-0217">Generally, <i>E. coli </i>grows very slowly at pH lower than 6-7. It would be useful to isolate <i>E. coli </i>mutants that exhibit improved growth characteristics at low pH. Alternatively, an enhanced glucosamine synthesis pathway could be engineered in other bacteria and yeast species that normally grow under low pH conditions. For example, <i>Saccharomyces cerevisiae </i>grow optimally at between pH 4 and pH 5.</p>
    <p num="p-0218">A novel strategy was developed to overcome the problem of product degradation. The glucosamine synthesis pathway in <i>E. coli </i>was extended by over-expressing glucosamine-6-P N-acetyltransferase (GNA1) which leads to the synthesis of N-acetylglucosamine-6-P. With this strain, it could be demonstrated that N-acetylglucosamine-6-P was produced and secreted efficiently to the medium as N-acetylglucosamine. N-acetylglucosamine is very stable over wide ranges of pH and temperature. The product can be easily converted back to glucosamine using mild acid conditions. By using this strategy, the titer of N-acetylglucosamine was elevated several-fold higher than in glucosamine production strain. N-acetylglucosamine was also recovered as a final product.</p>
    <p num="p-0219">Several factors may contribute to the high titer of N-acetylglucosamine production in the simple mineral medium. The first factor is the stability of N-acetylglucosamine. This does not only preserve the product but also avoids the toxic effects of glucosamine degradation products. Secondly, the N-acetylation step serves as a strong force to pull the pathway flux to this destination. It is interesting to note that when NagB enzyme was overexpressed in the absence of GlmS and GNA1 enzymes, it only functions to the degree sufficient to provide amino sugar for cell survival and growth. When GNA1 enzyme was co-expressed with NagB, it resulted in multi-gram level production of N-acetylglucosamine, demonstrating the driving power of the acetylation reaction in the pathway. Thirdly, the synthesis of N-acetylglucosamine utilizes acetyl-CoA as the donor of the acetyl group. Although this may be regarded as a kind of metabolic burden to the cells, it actually serves to avoid acetate formation, and thus prevent negative effects caused by acetate accumulation on the cell culture. Acetate formation could also be minimized by genetic modification of the enzymes involved in acetate synthesis.</p>
    <p num="p-0220">The combination of NagB and GNA1 offers an interesting pathway from fructose-6-P to N-acetylglucosamine-6-P. Unlike GlmS which uses glutamine and fructose-6-P as substrates, the NagB enzyme catalyzes the direct assimilation of ammonium into N-acetylglucosamine-6-P. Moreover, NagB protein is about 30 kDa, much smaller than GlmS protein (about 70 kDa). It was found that as compared to the GlmS protein, a larger portion of NagB protein was in the soluble protein fraction when over-expressed in <i>E. coli. </i> </p>
    <p num="p-0221">GlmS enzyme is generally subject to strong product inhibition. The use of a product resistant GlmS enzyme played an important role in elevating the titer of glucosamine and probably also in elevating the titer of N-acetylglucosamine. Product resistant GlmS mutants were created by in vitro mutagenesis. Such mutants could also be isolated from nature. This was illustrated by demonstrating product resistance of a native <i>B. subtilis </i>GlmS.</p>
    <p num="p-0222">The following are exemplary protocols for optimized production of glucosamine and N-acetylglucosamine. These are merely examples of some operable and preferred embodiments of the invention and are not to be construed as the only embodiments of the invention. Several preferred fermentation protocols and parameters of the fermentation process, both for flask cultures and fermentor cultures, are described in detail in the Examples section.</p>
    <p num="p-0223">The following is a description of a typical protocol for lactose-induced glucosamine fermentation process according to the present invention:</p>
    <p num="p-0224"> <tables id="TABLE-US-00002" num="00002"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="182pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain:</td> <td class="description-td">Recombinant <i>E. coli</i> </td> </tr> <tr class="description-tr"> <td class="description-td">Induction:</td> <td class="description-td">30 g/l lactose added (as a 35% feed ramped slowly over a 10</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">hour period) after a cell density of 10 g/l is reached. Glucose</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">feed is suspended during this procedure to prevent glucose</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">repression. After the lactose has all been added, glucose feed</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">is re-instated.</td> </tr> <tr class="description-tr"> <td class="description-td">Feed:</td> <td class="description-td">50% glucose with 5 ug FeSO<sub>4</sub>-7H<sub>2</sub>O/g glucose and 0.33 ug</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">MnSO<sub>4</sub>-H<sub>2</sub>O/g glucose, glucose fed at limiting concentrations.</td> </tr> <tr class="description-tr"> <td class="description-td">Fermen-</td> <td class="description-td">72 hours</td> </tr> <tr class="description-tr"> <td class="description-td">tation</td> </tr> <tr class="description-tr"> <td class="description-td">Time:</td> </tr> <tr class="description-tr"> <td class="description-td">Fermen-</td> <td class="description-td">Fed Batch, with 50% glucose added as required, maintain</td> </tr> <tr class="description-tr"> <td class="description-td">tation</td> <td class="description-td">limiting concentrations of glucose</td> </tr> <tr class="description-tr"> <td class="description-td">Mode:</td> </tr> <tr class="description-tr"> <td class="description-td">Inoculum:</td> <td class="description-td">5% by volume</td> </tr> <tr class="description-tr"> <td class="description-td">pH:</td> <td class="description-td">6.9 during growth, then 6.7 after induction, controlled by 10</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">N NH<sub>4</sub>OH</td> </tr> <tr class="description-tr"> <td class="description-td">Temper-</td> <td class="description-td">30° C., switched to 25° C. after induction</td> </tr> <tr class="description-tr"> <td class="description-td">ature:</td> </tr> <tr class="description-tr"> <td class="description-td">Oxygen:</td> <td class="description-td">Dissolved O<sub>2 </sub>at 20% or greater, controlled by agitation</td> </tr> <tr class="description-tr"> <td class="description-td">Aeration:</td> <td class="description-td">0.5 to 1 vvm</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="98pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Medium:</td> <td class="description-td">Component</td> <td class="description-td">Concentration</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="49pt" align="right"> </colspec> <colspec colname="3" colwidth="49pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">KH<sub>2</sub>PO<sub>4</sub> </td> <td class="description-td">14</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">K<sub>2</sub>HPO<sub>4</sub> </td> <td class="description-td">16</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Na<sub>3</sub>-citrate</td> <td class="description-td">1</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> </td> <td class="description-td">5</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CaCl<sub>2</sub>-H<sub>2</sub>O</td> <td class="description-td">0.05</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">MgSO<sub>4</sub>-7H<sub>2</sub>O</td> <td class="description-td">0.6</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">FeSO<sub>4</sub>-7H<sub>2</sub>O</td> <td class="description-td">3</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">ZnSO<sub>4</sub>-7H<sub>2</sub>O</td> <td class="description-td">3.8</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">MnSO<sub>4</sub>-H<sub>2</sub>O</td> <td class="description-td">0.33</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CuSO<sub>4</sub>-5H<sub>2</sub>O</td> <td class="description-td">0.1</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">NaMoO<sub>4</sub>-2H<sub>2</sub>O</td> <td class="description-td">0.1</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">H<sub>3</sub>BO<sub>3</sub> </td> <td class="description-td">0.1</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CoCl<sub>2</sub>-6H<sub>2</sub>O</td> <td class="description-td">0.1</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="98pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Glucose</td> <td class="description-td">&gt;200 g l<sup>−1</sup>, as needed</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="49pt" align="right"> </colspec> <colspec colname="3" colwidth="49pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Mazu 204 defoamer</td> <td class="description-td">0.25</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
In the above-described medium, all components are added before sterilization except glucose (added incrementally) and Fe, Zn, Mn, Cu, B, Mo, Co trace elements (added after sterilization).
</p>
    <p num="p-0225">The following is a description of a typical protocol for a N-acetylglucosamine fermentation process according to the present invention:</p>
    <p num="p-0226"> <tables id="TABLE-US-00003" num="00003"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="49pt" align="left"> </colspec> <colspec colname="2" colwidth="168pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain:</td> <td class="description-td">Recombinant <i>E. coli</i> </td> </tr> <tr class="description-tr"> <td class="description-td">Induction:</td> <td class="description-td">5 to 10 g l<sup>−1 </sup>lactose added in a single point addition after</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">a cell density of 15 to 20 g l<sup>−1 </sup>is reached. Glucose</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">feed is not suspended during this procedure, but remains</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">steady at 6.5 g l<sup>−1 </sup>hr<sup>−1 </sup>(based on initial volume).</td> </tr> <tr class="description-tr"> <td class="description-td">Feed:</td> <td class="description-td">65% glucose without any amendments, glucose fed at</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">limiting concentrations</td> </tr> <tr class="description-tr"> <td class="description-td">Fermentation</td> <td class="description-td">60 to 72 hours</td> </tr> <tr class="description-tr"> <td class="description-td">Time:</td> </tr> <tr class="description-tr"> <td class="description-td">Fermentation</td> <td class="description-td">Fed Batch, with 65% glucose added as required, maintain</td> </tr> <tr class="description-tr"> <td class="description-td">Mode:</td> <td class="description-td">limiting concentrations of glucose</td> </tr> <tr class="description-tr"> <td class="description-td">Inoculum:</td> <td class="description-td">2.5% to 5% by volume</td> </tr> <tr class="description-tr"> <td class="description-td">pH:</td> <td class="description-td">6.9 throughout, controlled with 12 N NH<sub>4</sub>OH</td> </tr> <tr class="description-tr"> <td class="description-td">Temperature:</td> <td class="description-td">37° C. throughout</td> </tr> <tr class="description-tr"> <td class="description-td">Oxygen:</td> <td class="description-td">Dissolved O<sub>2 </sub>at 20% or greater, controlled by agitation</td> </tr> <tr class="description-tr"> <td class="description-td">Aeration:</td> <td class="description-td">0.5 to 1 vvm</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="133pt" align="left"> </colspec> <colspec colname="2" colwidth="84pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Medium:</td> <td class="description-td">Concentration</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="70pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="84pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Component</td> <td class="description-td">(amount per liter)</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="70pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="right"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">KH<sub>2</sub>PO<sub>4</sub> </td> <td class="description-td">6.67</td> <td class="description-td">g</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Citric acid</td> <td class="description-td">3.25</td> <td class="description-td">g</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CaCl<sub>2</sub>-H<sub>2</sub>O</td> <td class="description-td">0.05</td> <td class="description-td">g</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">MgSO<sub>4</sub>-7H<sub>2</sub>O</td> <td class="description-td">2.5</td> <td class="description-td">g</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">FeSO<sub>4</sub>-7H<sub>2</sub>O</td> <td class="description-td">5</td> <td class="description-td">mg</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">ZnSO<sub>4</sub>-7H<sub>2</sub>O</td> <td class="description-td">3.8</td> <td class="description-td">mg</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">MnSO<sub>4</sub>-H<sub>2</sub>O</td> <td class="description-td">0.33</td> <td class="description-td">mg</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CuSO<sub>4</sub>-5H<sub>2</sub>O</td> <td class="description-td">0.1</td> <td class="description-td">mg</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CoCl<sub>2</sub>-6H<sub>2</sub>O</td> <td class="description-td">0.1</td> <td class="description-td">mg</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="70pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="84pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Glucose</td> <td class="description-td">&gt;200 g, as needed</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="70pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="right"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Mazu 204 defoamer</td> <td class="description-td">0.25</td> <td class="description-td">g</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
In the above-described medium, all components are added before sterilization except glucose (added incrementally). Initial pH is near 3.0 after sterilization and adjusted to 6.9 with NH<sub>4</sub>OH before inoculation.
</p>
    <p num="p-0227">The method of the present invention further includes a step of collecting the product, which can be intracellular glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine and/or glucosamine, and/or extracellular glucosamine and/or extracellular N-acetylglucosamine. The general step of collecting can include steps of recovering the product (defined below) and also purifying the product, as desired. The detailed discussion of the methods for recovery and purification are provided below. To “collect” a product such as glucosamine or N-acetylglucosamine can simply refer to collecting the product from the fermentation bioreactor and need not imply additional steps of separation, recovery, or purification. For example, the step of collecting can refer to removing the entire culture (i.e., the microorganism and the fermentation medium) from the bioreactor, removing the fermentation medium containing extracellular glucosamine and/or N-acetylglucosamine from the bioreactor, and/or removing the microorganism containing intracellular glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine and/or glucosamine from the bioreactor. The term “recovering” or “recover”, as used herein, refers to reducing solubility conditions of the glucosamine or N-acetylglucosamine solution to the point where glucosamine or N-acetylglucosamine, respectively, becomes insoluble and either precipitates out of solution or crystallizes. These steps can be followed by further purification steps. Glucosamine and N-acetylglucosamine are preferably recovered in substantially pure forms. As used herein, “substantially pure” refers to a purity that allows for the effective use of the glucosamine and/or N-acetylglucosamine as a nutraceutical compound for commercial sale. In one embodiment, the glucosamine and/or N-acetylglucosamine products are preferably separated from the production organism and other fermentation medium constituents. Methods to accomplish such separation are described below.</p>
    <p num="p-0228">Preferably, by the method of the present invention, at least about 1 g/L of product (i.e., glucosamine, N-acetylglucosamine, glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and/or N-acetylglocosamine-1-phosphate) are collected or recovered from the microorganism and/or fermentation medium. More preferably, by the method of the present invention, at least about 5 g/L, and even more preferably, at least about 10 g/L, and even more preferably, at least about 20 g/L and even more preferably, at least about 50 g/L, and even more preferably at least about 75 g/L, and even more preferably at least about 100 g/L, and even more preferably at least about 120 g/L of product are recovered, and any whole increment between at least about 1 g/L and at least about 120 g/L (i.e., 2 g/L, 3 g/L, etc.). In one embodiment, the product is recovered in an amount from about 1 g/L to at least about 120 g/L.</p>
    <p num="p-0229">Typically, most of the glucosamine and/or N-acetylglucosamine produced in the present process are extracellular. The microorganism can be removed from the fermentation medium by conventional methods, such as by filtration or centrifugation. In one embodiment, the step of collecting or of recovering the product includes the purification of glucosamine and/or N-acetylglucosamine from the fermentation medium. Glucosamine and/or N-acetylglucosamine can be recovered from the cell-free fermentation medium by conventional methods, such as chromatography, extraction, crystallization (e.g., evaporative crystallization), membrane separation, reverse osmosis and distillation. In a preferred embodiment, glucosamine and/or N-acetylglucosamine are recovered from the cell-free fermentation medium by crystallization. In another embodiment, the step of recovering the product includes the step of concentrating the extracellular glucosamine and/or N-acetylglucosamine.</p>
    <p num="p-0230">In one embodiment, glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine and/or glucosamine accumulate intracellularly, the step of recovering the products includes isolating glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine and/or glucosamine from the microorganism. For example, the products can be collected by lysing the microorganism cells by a method which does not degrade the products (glucosamine, N-acetylglucosamine, glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate), centrifuging the lysate to remove insoluble cellular debris, and then recovering the products glucosamine, N-acetylglucosamine, glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate by a conventional method as described above.</p>
    <p num="p-0231">The initial intracellular products in the genetically modified microorganism described herein are glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate, N-acetylglucosamine-1-phosphate, N-acetylglucosamine and/or glucosamine, It is generally accepted that phosphorylated intermediates are dephosphorylated during export from the microorganism, most likely due to the presence of alkaline phosphatase, acid phosphatase, sugar phosphatase or amino sugar phosphatase in the periplasmic space of the microorganism. In one embodiment of the present invention, glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate are dephosphorylated before or during exp phosphatases in order to facilitate the production of the desired products, glucosamine and/or N-acetylglucosamine. In this embodiment, the need for amplification of a recombinantly provided phosphatase activity in the cell or treatment of the fermentation medium with a phosphatase is obviated. In another embodiment, the level of phosphatase in the production organism is increased by a method including, but not limited to, genetic modification of an endogenous phosphatase gene or by recombinant modification of the microorganism to express a phosphatase gene. In yet another embodiment, the collected fermentation medium is treated with a phosphatase after glucosamine-6-phosphate, glucosamine-1-phosphate, N-acetylglucosamine-6-phosphate and/or N-acetylglucosamine-1-phosphate are released into the medium, such as when cells are lysed as described above.</p>
    <p num="p-0232">As noted above, the process of the present invention produces significant amounts of extracellular glucosamine and/or N-acetylglucosamine. In particular, the process produces extracellular glucosamine and/or N-acetylglucosamine such that greater than about 50% of total glucosamine and/or N-acetylglucosamine are extracellular, more preferably greater than about 75% of total glucosamine and/or N-acetylglucosamine are extracellular, and most preferably greater than about 90% of total glucosamine and/or N-acetylglucosamine are extracellular. By the method of the present invention, production of an extracellular glucosamine and/or N-acetylglucosamine concentration can be achieved which is greater than about 1 g/l, more preferably greater than about 5 g/l, even more preferably greater than about 10 g/l, and even more preferably greater than about 20 g/L, and even more preferably greater than about 50 g/l, and even more preferably greater than about 75 g/l, and even more preferably greater than about 100 g/l, and even more preferably greater than about 120 g/l.</p>
    <p num="p-0233">Another embodiment of the present invention relates to a novel means to produce N-acetylglucosamine. The method includes obtaining a fermentation broth containing solubilized N-acetylglucosamine produced by a fermentation process, such as is discussed above, and recovering N-acetylglucosamine-containing solids from the fermentation broth. According to the present invention, N-acetylglucosamine that is produced by a fermentation process is N-acetylglucosamine that is a product of the fermentation process. In other words, the fermentation process of culturing cells results in production of N-acetylglucosamine by the cells. The term recovering or recover refers to reducing solubility conditions of the N-acetylglucosamine solution to the point where N-acetylglucosamine becomes insoluble and either precipitates out of solution or crystallizes. Fermentor broth containing N-acetylglucosamine, residual media and cellular matter can be treated first by removing the cellular material and bacterial endotoxins. Prior to removal of cellular material, cells in the fermentation broth can be lysed by using different methods, including but not limited to: sonication, osmotic disruption, grinding/beading, French press, homogenization, explosive decompression, solvent treatments, critical point extraction and freeze/thaw. Removal of cellular material and endotoxins can be achieved by micro- or ultrafiltration or a combination thereof. Other techniques known in the art include centrifugation and diafiltration. Following removal of cellular material, the solution can be tested to verify adequate removal of endotoxins.</p>
    <p num="p-0234">Prior to the step of recovering N-acetylglucosamine-containing solids from the fermentation broth, the fermentation broth can be decolored and/or deashed. If both of these processes are conducted, either one can be performed first. However, by performing the deashing step first, the decolorization requirements are reduced because of color removal by the deashing process, such as by the use of ion exchange resins. The step of decolorizing can be conducted by multiple N-acetylglucosamine crystallizations, activated carbon treatment, and chromatographic decolorization. The use of chromatographic decolorization can include the use of ion exchange resins (not for ion exchange, but just for color absorption), Dow Optipore SD-2 resin and classical silica based-chromatographic media.</p>
    <p num="p-0235">The step of deashing can be conducted by contacting the fermentation broth with an ion exchange resin, which can include contacting the fermentation broth with an anion exchange resin and/or a cation exchange resin. In one embodiment, a mixed bed of both an anion and a cation exchange resin is used. Cation removal can be performed on either strong acid or weak acid ion exchange resins, while weak base resins are preferred due to their enhanced ability to remove organic acids produced in the fermentation. Cation removal is typically chosen to precede anion removal because the effluent from the cation exchanger has a reduced pH, while that from the anion exchanger has an elevated pH. N-acetylglucosamine has a demonstrated tendency to epimerize at high pH, producing measurable levels of N-acetylmannosamine. Anion removal can be conducted with either strong or weak base resins. The resultant purified solution carries very little ionic material and consists mostly of water, N-acetylglucosamine, and carbohydrates from the fermentor feed unconsumed during fermentation. The N-acetylglucosamine purity of this intermediate product is typically greater than about 90% of the stream's total dry solids content. If further polishing is desired to complete deashing, then a mixed bed ion exchange step can be applied. In this case, both cationic and anionic resins occupy the same ion exchanger, and the largely deashed fermentor broth is passed through this bed. Although a mixed bed ion exchange can also take the place of the separate cation and anion exchangers, and although this offers the added benefit of minimizing the magnitude of the changes in broth pH, it is more convenient to operate separate beds because of the need to separate the ion exchange resins prior to regeneration, with the concomitant higher risk of resin loss.</p>
    <p num="p-0236">The step of recovering N-acetylglucosamine-containing solids from the fermentation broth can include precipitating and/or crystallizing N-acetylglucosamine-containing solids from the fermentation broth. Typically, prior to the recovering step, the fermentation broth is concentrated to provide a higher concentration of solubilized N-acetylglucosamine to enable precipitation or crystallization. The step of concentrating can be conducted under vacuum (i.e., at less than atmospheric pressure) or by membrane separation. In preferred embodiments, the step of concentrating is conducted at a temperature of between about 40° C. and about 75° C., and more preferably at a temperature of between about 45° C. and about 55° C. The step of concentrating is typically conducted to achieve a solids content in the fermentation broth of at least about 30% solids, more preferably at least about 40% solids, and more preferably at least about 45% solids.</p>
    <p num="p-0237">After concentration, the fermentation broth is typically cooled. Such cooling can be passive, i.e., simply allowing the broth to come to room temperature, or it can be active, such as by the use of freeze dryers, spray chillers, prillers, flakers, and blenders or extruders equipped with cooling jackets. The step of cooling a concentrated broth, alone, can be sufficient for recovery of N-acetylglucosamine-containing solids. For example, the fermentation broth can be cooled to between about −5° C. and about 45° C., between about −5° C. and about room temperature, or to about room temperature.</p>
    <p num="p-0238">Recovering N-acetylglucosamine-containing solids from the fermentation broth can also include the step of seeding the fermentation broth with N-acetylglucosamine crystals to promote recovery by growth of existing crystals. Crystals can be provided by nucleation in the fermentation broth or as externally provided N-acetylglucosamine crystals. In the first instance, the fermentation broth is concentrated and/or cooled until some nucleation occurs by forcing the supersaturated solution into the labile regime but then cooling more slowly to avoid additional nucleation and promote growth of existing crystals. Alternatively, N-acetylglucosamine crystals, produced by any method, can be introduced into the fermentation broth as seed crystals.</p>
    <p num="p-0239">Recovering N-acetylglucosamine-containing solids from the fermentation broth can be enhanced by contacting N-acetylglucosamine in the broth with a water miscible solvent. It has been found that N-acetylglucosamine is very insoluble in water miscible solvents, such as isopropyl alcohol (IPA), ethanol, methanol, acetone, tetrahydrofuran, dimethylsulfoxide, dimethylformamide, dioxane and acetonitrile. Therefore, by introduction of such a water miscible solvent into the broth, the N-acetylglucosamine will become less soluble and will further recovery.</p>
    <p num="p-0240">Recovery processes of the invention can be conducted as either batch or continuous crystallizations of the fermentor broth. Crystals formed during the concentration process can be continuously harvested by filter or centrifuge while the mother liquor is recycled for further concentration, or they can remain with the mother liquor until the solids level mandates removing them.</p>
    <p num="p-0241">After recovery of N-acetylglucosamine-containing solids, the solids can be separated from the fermentation broth, such as by centrifuging or filtering. The resulting solid cake can be dried by any of a variety of techniques known to those skilled in the art, such as vacuum drying, but such step is preferably accomplished at reduced temperature to prevent degradation and color formation. The final dried product will typically be at least about 50% dried solids, more preferably at least about 70% dried solids and more preferably at least about 85% dried solids. Prior to drying, the recovered solids can be washed with a water miscible solvent, such as those discussed above. Such a washing step results in stabilization of the product, for example, to prevent color formation.</p>
    <p num="p-0242">In another embodiment, recovered N-acetylglucosamine-containing solids are further purified by dissolving the solids and recovering the solids in a second recovery step. The second recovery step can include any of the recovery process steps described above. The need for a second cycle of recovery is determined by the desired end product purity and the starting purity.</p>
    <p num="p-0243">Employing the various recovery processes described above, N-acetylglucosamine of high purity can be prepared. In particular, processes of the present invention can produce N-acetylglucosamine that is at least about 70% pure, more preferably at least about 90% pure, and more preferably at least about 99% pure.</p>
    <p num="p-0244">Various recovery process embodiments are preferred. For example, cell-free broth can be used to produce solid N-acetylglucosamine without further ion removal, by decolorizing with activated carbon; concentrating to approximately 45% dry solids or higher in a vacuum evaporator, which can be multi-stage for energy efficiency (e.g., preferred conditions are liquor temperatures between 45° C. and 55° C. at a vacuum of approximately 600 mm Hg.); seeding the warm concentrate with pure N-acetylglucosamine, and allowing it to cool while agitating. The cooled broth is filtered or centrifuged and a solid cake is recovered. The recovered solid can be vacuum-dried and used as an intermediate in the production of pure N-acetylglucosamine or glucosamine salts; and if the N-acetylglucosamine purity is 87% or higher, pure N-acetylglucosamine can be produced from a single crystallization. Alternatively to seeding, cooling, and solids recovery from mother liquor, the supersaturated stream can be forced to solidify by cooling. Suitable cooling devices include freeze dryers, spray chillers, prillers, flakers, and blenders or extruders equipped with cooling jackets. The resultant solid can be further dried.</p>
    <p num="p-0245">In another preferred recovery embodiment, recovered N-acetylglucosamine can be redissolved in water without prior drying, and subjected to a second cycle: concentration, seeding, cooling and solids collection. The collected solids are washed with a water miscible solvent and dried under vacuum. The need for double crystallization is determined by the beginning purity of the N-acetylglucosamine. When purity, as a percentage of the dry solids, is 87% or higher, a single crystallization is required. When the purity is 70%, a double crystallization is required.</p>
    <p num="p-0246">Another preferred recovery embodiment is a single crystallization to generate pure N-acetylglucosamine by using material that has been decolorized and concentrated at temperatures reaching 70° C., or by redissolving a partially purified material at 70° C. and allowing it to cool to room temperature. The recovered solid is washed with a water-miscible solvent and dried.</p>
    <p num="p-0247">In another preferred recovery embodiment, cell-free fermentor broth that has been deashed is further purified by separation on a simulated moving bed chromatographic system, using chromatographic media, including cation exchange resins. Once an about 98% pure stream is obtained, vacuum concentration prepares the material for stabilization treatment described below.</p>
    <p num="p-0248">In another preferred recovery embodiment, in order to achieve high recoveries of N-acetylglucosamine that have high purity and are color-stable, exhibiting less than 1% weight loss due to decomposition on drying at 105° C., a water-miscible solvent is employed. By starting with relatively high purity N-acetylglucosamine, for example 93% achieved by decolorizing and deashing, it is possible to remove water by vacuum concentration, and then by adding a water soluble solvent, to precipitate additional pure product. The process of treating the precipitated product with water-bearing miscible solvents facilitates the subsequent drying step, wherein the residual water is prevented from potentiating a degradation reaction.</p>
    <p num="p-0249">Another embodiment of the invention relates to a method to produce glucosamine from N-acetylglucosamine. For production of glucosamine as a final product, N-acetylglucosamine can be hydrolyzed to produce glucosamine as a final product. The hydrolysis can be done directly with N-acetylglucosamine produced in the fermentation without being first isolated from fermentation or, alternatively, the hydrolysis can be carried on N-acetylglucosamine after being isolated from the fermentation, such as the N-acetylglucosamine recovered using any of the methods described above. Additionally, one can use any other available source of N-acetylglucosamine in this method of the invention. Prior to the present invention, N-acetylglucosamine was produced by the acetylation of glucosamine using an organic acetylating reagent, or by the enzymatic hydrolyzation of N-acetyglucosamine directly from chitin, which parallels the acid hydrolysis process traditionally used to produce glucosamine from chitin. Therefore, there was no need for the reverse reaction to produce glucosamine as an end product from N-acetylglucosamine.</p>
    <p num="p-0250">Accordingly, a source of N-acetylglucosamine as described herein need not refer to a pure source of N-acetylglucosamine, but rather any source (solid or solution) that contains an amount of N-acetylglucosamine for conversion to glucosamine. Glucosamine hydrochloride can be produced both from the N-acetylglucosamine that has been produced is by a fermentation process as described above and then recovered from the fermentation broth as described in detail above, or the process can use N-acetylglucosamine produced by fermentation directly as a fermenter concentrate that has had the cellular material removed, but without removing either the residual color or ionic constituents. Alternatively, any other suitable source of N-acetylglucosamine can be used in the present method, including N-acetylglucosamine produced by other methods, such as the enzymatic hydrolysis of chitin. In a preferred embodiment, glucosamine hydrochloride is produced from a source of N-acetylglucosamine comprising at least about 30% N-acetylglucosamine as a percentage of the total solids in the source, and more preferably, the source comprises at least about 35%, and more preferably at least about 40%, and more preferably at least about 45%, and more preferably at least about 50%, and more preferably at least about 55%, and more preferably at least about 60%, and more preferably at least about 65%, and more preferably at least about 70%, and more preferably at least about 75%, and more preferably at least about 80%, and more preferably at least about 85%, and more preferably at least about 90%, and more preferably at least about 95%, as a percentage of the total solids in the source. In general, glucosamine hydrochloride is produced from a source of N-acetylglucosamine comprising any percentage of N-acetylglucosamine from at least about 1% to 100%, in whole integers (i.e., 1%, 2%, 3%, . . . 98%, 99%, 100%), The N-acetylglucosamine can be used in the reaction as a solid or as a solution in water, or as a solution in an aqueous low boiling primary or secondary alcohol, including, but not limited to, ethanol, methanol, n-propanol, isopropanol, n-butanol, or sec-butanol. In a preferred embodiment, the source of N-acetylglucosamine comprises at least about 40% N-acetylglucosamine as a percentage of the total dry solids in the source.</p>
    <p num="p-0251">In one embodiment, the glucosamine hydrochloride can be produced from a source of N-acetylglucosamine using hydrolysis under acid and heat conditions. Acid hydrolysis techniques are known in the art. The present invention provides a specific adaptation of this chemical reaction to the conversion of N-acetylglucosamine to glucosamine hydrochloride. The hydrolysis reaction releases one mole of acetic acid, and consumes one mole each of hydrochloric acid and water for every mole of N-acetylglucosamine converted to glucosamine hydrochloride. Many combinations of water, hydrochloric acid and N-acetylglucosamine successfully perform the reaction, thus it is possible to use anhydrous hydrochloric acid, dry N-acetylglucosamine and water as reactants, or to bring water in with either or both of the N-acetylglucosamine and hydrochloric acid. The reaction is carried out with excess water and hydrochloric acid. The important parameters are reaction time and temperature, as well as residual hydrochloric acid concentration. The reaction is cooled after conversion is complete, and the residual concentration of excess hydrochloric acid as well as the ending temperature establishes the solubility of the glucosamine hydrochloride product. Excess levels of solubility cause reduced per-pass recovery of glucosamine hydrochloride and reduce yield. Glucosamine can degrade and the reactions accomplishing this occur in solution. The reaction rate is related to temperature, the concentration of N-acetylglucosamine and/or glucosamine in solution, and the concentration of hydrochloric acid in solution. Successful conditions include temperatures from 60° C. to 100° C., with temperatures from 70° C. to 90° C. being more preferred. Acceptable ratios of hydrochloric acid solution to N-acetylglucosamine solid range from 1:1 by weight to 5:1 by weight, with 3:1 being preferred and 2.5:1 being more preferred. Acceptable concentrations of hydrochloric acid solution range from 10-40% w/w, and more preferably from about 10% w/w to 37% w/w. Anhydrous hydrochloric acid can be substituted for aqueous, providing that the heat of dilution is managed and there is adequate solution to fully dissolve the N-acetylglucosamine. The sequence of addition and temperature at addition are of lesser importance, as all combinations are successful. The time to conduct the reaction varies with the temperature of reaction and acid concentration, and ranges from 10 minutes for concentrated acid at high temperatures to 3 hours or more (e.g., up to 24 hours) for dilute acids at low temperatures. The solution is cooled to 4° C. in order to precipitate the glucosamine hydrochloride. Higher and lower temperatures are acceptable (e.g., any temperature between about −5° C. and about 40° C.), but 4° C. was chosen as a convenient temperature to minimize residual glucosamine hydrochloride in the hydrolysis solution at commercially convenient conditions. For more impure hydrolysis solutions, that is solutions where the relative dry solids composition of glucosamine hydrochloride is lower, a slower cooling rate and additional holding time at the ending temperature are required to return the hydrolysis solution to saturation. Solution agitation mitigates, but does not overcome the hindrance posed by impurities to the crystallization process.</p>
    <p num="p-0252">Since the hydrolysis reaction is conducted with excess hydrochloric acid, the hydrolysis solution after cooling and removing glucosamine hydrochloride process has the capability to be used again for hydrolysis. As the acetic acid coproduct increases in concentration and the reactant hydrogen chloride is consumed in each hydrolysis cycle, the hydrolysis solution becomes less active and the reactions take longer to complete. It is possible to replenish the hydrogen chloride by supplying it in its gaseous form to reconstitute this reactant, but the increase of acetic acid coproduct from each reaction cycle poses an increasing obstacle to simple hydrolysate solution recycle. This can be overcome by esterifying the acetic acid through the addition of a primary or secondary alcohol, either prior to, during or following the hydrolysis step. The acetic acid forms an ester with the alcohol which then can be removed by distillation, flashing or vacuum evaporation prior to, during or following N-acetylglucosamine hydrolysis.</p>
    <p num="p-0253">N-acetylglucosamine can be continuously blended with recycled hydrolysis mother liquor in order to ensure it becomes or remains dissolved. Subsequently, replacement anhydrous hydrochloric acid is added, which raises the solution temperature to initiate the hydrolysis reaction and maintain adequate residual acid concentration to minimize glucosamine hydrochloride solubility. Pressure can be controlled above atmospheric, and elevated temperature is selected to permit using minimum reactor residence time. After the complete conversion of N-acetylglucosamine to glucosamine hydrochloride, the reaction is cooled and glucosamine hydrochloride recovered by filtration or centrifugation. The filtrate or centrate are recycled through the cooler until the goal temperature of about 4° C. is reached and the majority of the glucosamine hydrochloride is recovered. The centrate or filtrate is then recycled for reuse. Adequate water to make up reaction losses comes in with the N-acetylglucosamine. The accumulation of reaction coproducts like acetic acid are removed by conducting a continuous purge of the mother liquor, by separating the residual hydrochloric acid from the purge stream and returning it for subsequent reuse, or by esterifying it with a primary or secondary alcohol and drying it as esters.</p>
    <p num="p-0254">Details regarding recovery of glucosamine from the hydrolysis process described herein are similar to those described above for the recovery of N-acetylglucosamine and are incorporated herein for the recovery of glucosamine. Some preferred aspects of glucosamine recovery are discussed below and in the Examples section.</p>
    <p num="p-0255">Filtration or centrifugation can then recover the solid glucosamine salt. Recovered molar yields range from 50% to 90% based on N-acetylglucosamine. Glucosamine purity after alcohol washing and drying typically range from 96% to 100%. The wet cake recovered from the centrifuge or filter is washed with alcohol to minimize the tendency of the product to form lumps during drying. The product is then dried under vacuum in a vacuum dryer or Wyssmont type dryer until loss on drying meets product specifications.</p>
    <p num="p-0256">It is possible to start with recovered N-acetylglucosamine hydrolysis filter- or centrifuge-cake, which can be either washed or not. The cake is dissolved in water to a concentration of approximately 25% (w) at room temperature under agitation. A base, such as sodium hydroxide or potassium hydroxide is added to bring the pH up to between 2.5-4. Once the pH is adjusted, the fully dissolved solution is treated with activated carbon. For example, in the batch mode, 0.02 grams of Darco G-60 or equivalent activated carbon is added per gram glucosamine hydrochloride and mixed for a minimum of 30 minutes. The mixture is filtered to remove the carbon. The carbon is then rinsed with an amount of water approximately equal to the amount of glucosamine hydrochloride being recrystallized to elute any glucosamine hydrochloride entrained in the carbon and filter equipment. Alternatively, the solution can be passed through a granular packed bed of activated carbon, with a subsequent elution step to recover entrained glucosamine hydrochloride prior to regeneration or disposal of the expended activated carbon.</p>
    <p num="p-0257">The clear solution is then heated to 50° C. under vacuum and agitation. A 50-60 cm Hg vacuum has been used. Approximately half of the volume is removed by evaporation and the solids are recovered from the remaining solution by filtration, centrifugation or other appropriate means to recover recrystallized glucosamine hydrochloride. The remaining solution is returned for further recovery. The solids are collected, rinsed with a water-miscible solvent, including, but not limited to: methanol, ethanol, n-propanol, isopropanol, n-butanol, sec-butanol, acetone, tetrahydrofuran, dimethysulfoxide, dimethylformamide, or dioxane, and dried.</p>
    <p num="p-0258">When recrystallizing from single-use hydrolysate, the liquid is further evaporated to a point where solids are visible and the volume is estimated to be around 20-30% of the starting volume. An equal volume of water-miscible solvent is added as a liquid precipitant, the solids are recovered from the resultant solution by filtration, centrifugation or other appropriate means to recover recrystallized glucosamine hydrochloride. The remaining solution is then cooled, for example to 4° C., and filtered to recover the remaining glucosamine hydrochloride.</p>
    <p num="p-0259">When recrystallizing from multiple-use hydrolysate, the activated carbon treated hydrolysate of the subsequent cycle is added to the recovered solution of the current cycle to make up for the volume removed by evaporation and solids recovery. Approximately half volume is then removed by evaporation and the cycle is repeated.</p>
    <p num="p-0260">The purity of the recrystallized glucosamine hydrochloride is dependent on the amount of salts due to the neutralization of the excess acid and other impurities carried forward from the first, hydrolysate, crystallization step. This is controlled by substituting a single-use hydrolysis recrystallization step for the multiple-use hydrolysis crystallization step when the recovered glucosamine hydrochloride ceases to meet purity specifications. The resultant precipitated glucosamine hydrochloride recovered during the miscible-solvent precipitation is redissolved and incorporated as a part of a subsequent series of recrystallizations.</p>
    <p num="p-0261">The recovered glucosamine hydrochloride wet cake should contain alcohol to minimize the tendency of the product to form lumps and darken during drying. It is then dried under vacuum in a vacuum dryer or Wyssmont type dryer until loss on drying meets product specifications.</p>
    <p num="p-0262">In one embodiment, the step of hydrolyzing comprises the steps of: (a) hydrolyzing the source of N-acetylglucosamine by combining the source of N-acetylglucosamine with a hydrochloric acid solution or a recycled hydrolysis mother liquor under heat conditions to produce a solution containing glucosamine hydrochloride; (b) cooling the solution of (a) to precipitate the glucosamine hydrochloride; and (c) recovering the precipitated glucosamine hydrochloride-containing solids from (b). In one aspect, the step of hydrolyzing can be performed by continuously blending the source of N-acetylglucosamine with a hydrochloric acid solution or a recycled hydrolysis mother liquor to maintain the source of N-acetylglucosamine as a dissolved solution, followed by addition of anhydrous hydrochloric acid under heat conditions to the solution of (a) to initiate hydrolysis and convert the N-acetylglucosamine to glucosamine hydrochloride. In another aspect, the recycled hydrolysis mother liquor is hydrolysis solution that remains after recovering the precipitated glucosamine hydrochloride in step (c), wherein a primary or secondary alcohol is added to the hydrolysis solution prior to, during or after a hydrolysis step is performed. In this aspect, the step of cooling can be performed until the solution is from about −5° C. to about 40° C.</p>
    <p num="p-0263">The step of recovering can comprise: (i) collecting the precipitated glucosamine hydrochloride-containing solids; (ii) washing the glucosamine hydrochloride-containing solids with a water miscible solvent (including, but not limited to, methanol, isopropanol, ethanol, acetonitrile, acetone, tetrahydrofuran, dimethylsulfoxide, dimethylformamide and dioxane); and (iii) drying the glucosamine hydrochloride-containing solids.</p>
    <p num="p-0264">In another aspect, the step of recovering can include: (i) collecting the precipitated glucosamine hydrochloride-containing solids; (ii) dissolving the solids from (i) in water to form a solution; (iii) adjusting the pH of the solution of (ii) to between about 2.5 and 4 (e.g., by adding a base, by washing to remove the acid, passing over ion exchange media, or any other suitable method); (iv) contacting the solution of (iii) with activated carbon to decolorize the glucosamine hydrochloride-containing solids; (v) removing the activated carbon from the solution of (iv); and (vi) crystallizing glucosamine hydrochloride from the solution of (v). In this aspect, the step of crystallizing comprises concentrating the glucosamine hydrochloride at a temperature of less than about 70° C., and more preferably at a temperature of less than about 50° C. In one aspect, the step of crystallizing comprises concentrating the glucosamine hydrochloride at less than atmospheric pressure. In another aspect, the process further includes comprising recycling solution remaining after the crystallization step (vi) to step (i) of a subsequent recovery process or to a subsequent step of crystallization (e.g., a recrystallization).</p>
    <p num="p-0265">In another aspect, when the source of N-acetylglucosamine is suspended in an aqueous, low-boiling, primary or secondary alcohol, the hydrolysis method can include an additional step, prior to cooling the solution, of removing the acetic acid ester formed with the alcohol following hydrolysis or prior to recycling the hydrolysis solution for reuse. The acetic acid ester is removed by a process including, but not limited to: distillation, flashing, and concentration at less than atmospheric pressure. In this embodiment, the step of hydrolyzing is performed at a temperature of between about 60° C. and about 100° C., and preferably at the solution boiling point at one atmosphere.</p>
    <p num="p-0266">In another aspect, when the hydrolysis is conducted under relatively high acid to N-acetylglucosamine ratios (e.g., from about 3:1 to about 5:1) and at a relatively low temperature (e.g., less than about 80° C.), the crystal quality of the glucosamine produced can be high enough to avoid the need to recrystallize the glucosamine. In this embodiment, the single crystallization step is followed by a wash in a water miscible solvent as described previously herein and then a step of drying as described above.</p>
    <p num="p-0267">Indeed, any of the methods of hydrolysis and recovery described herein can further include a step of washing the crystallized glucosamine hydrochloride from step (vi) with a water miscible solvent, as described previously herein, followed by a step of drying. In one aspect, the crystallized glucosamine hydrochloride is dried at a temperature of less than about 70° C. for less than about 6 hours, and in another embodiment, at a temperature of less than about 50° C. for less than about 3 hours. The step of drying can be conducted in the presence or absence of a vacuum and in the presence or absence of an air or inert gas sweep.</p>
    <p num="p-0268">Another method for converting N-acetylglucosamine to glucosamine is to use an enzyme hydrolysis procedure. Enzyme processes to hydrolyze N-acetylglucosamine in the fermentation broth or after its recovery are described in the Examples section. Three types of enzymes are candidates: N-acetylglucosamine-6-P deacetylase (EC 3.5.1.25, NagA), N-acetylglucosamine deacetylase (EC 3.5.1.33), and chitin deacetylase.</p>
    <p num="p-0269">The deacetylases described by Fujishima et al. were identified as N-acetylglucosamine-6-P deacetylases (EC 3.5.1.25, NagA). Their affinity and efficacy with N-acetylglucosamine 6-P were much higher than with N-acetylglucosamine. However, N-acetylglucosamine-6-P deacetylase purified from <i>E. coli </i>does not act on N-acetylglucosamine.</p>
    <p num="p-0270">The enzyme N-acetylglucosamine-6-P deacetylase (EC 3.5.1.25, NagA) is well known for its role of converting N-acetylglucosamine-6-P to glucosamine-6-P, a necessary step in the cellular metabolism of N-acetylglucosamine, N-acetylmannosamine and neuraminic acid. Normally, this enzyme such the recombinant <i>E. coli </i>NagA protein is not active on non-phosphorylated N-acetylglucosamine. DNA sequences coding for the N-acetylglucosamine 6-P deacetylase (nagA gene) were determined in many different organisms. It is not know if there exists a deacetylase that is only active on N-acetylglucosamine (thus distinctive from NagA).</p>
    <p num="p-0271">Chitin deacetylase (EC 3.5.1.41) catalyzes deacetylation of the N-acetylglucosamine units in chitin, resulting in chitosan. Chitin deacetylase activity is usually determined by using as substrate glycol chitin (partially O-hydroxyethylated chitin) radiolabeled in N-acetyl groups. The enzyme also acts on mycrocrystalline chitin and carboxymethylchitin (soluble derivative). However, it was reported that chitin deacetylase from <i>Mucor rouxii </i>does not deacetylate N-acetylglucosamine monomer or 2-3 oligomers (Araki and Ito, 1975. Eur. J. Biochem. 55:71-78, which is incorporated herein by reference in its entirety). Although there were no indications that normal chitin deacetylase deacetylate glucosamine monomer, chitin deacetylase variants with such activity could be isolated from nature or created in vitro.</p>
    <p num="p-0272">A number of acyl transferases can remove the acetyl group from a substrate and transfer it to another substrate (Konecny, et al.). Although there were no indications that such enzymes can deacetylate N-acetylglucosamine, acyl transferase variants with such activity could be isolated from nature or generated in vitro.</p>
    <p num="p-0273">Deacetylation of N-acetylglucosamine could be carried out by an acyl transferase contained in or isolated from organisms with a native acyl transferase or organisms with a recombinant acyl transferase. Recombinant acyl transferase could be improved by random or directed mutagenesis and/or by protein engineering.</p>
    <p num="p-0274">In applications of acyl transferase or deacetylase technology to convert N-acetylglucosamine to glucosamine, there is the potential for product degradation through a variety of mechanisms due the well-known instability of glucosamine in solution, especially when outside a low pH environment that would otherwise protonate the nitrogen atom.</p>
    <p num="p-0275">Simple pH reduction, while appropriate to acid hydrolysis, becomes problematic for enzymatic catalysis. Enzymes are more likely to be denatured when they are exposed to an unbuffered or non-neutral pH environment. Moreover, high salt loads are often associated with enzyme denaturation.</p>
    <p num="p-0276">Consequently, it is proposed to employ known enzyme immobilization technology (discussed below) to fix the enzyme conformation and reduce the impact of exposure to high ionic concentrations and reduce expensive enzyme consumption. Further, it is proposed to take advantage of the relatively low solubility of glucosamine hydrochloride in comparison with other salts, such as sodium acetate, calcium acetate, sodium chloride and calcium chloride.</p>
    <p num="p-0277">By presenting an appropriate enzyme with an aqueous sodium or calcium chloride solution and N-acetylglucosamine, the equilibrium reaction will proceed to form highly soluble sodium or calcium acetate, which serves as a pH buffer for the enzyme. The reaction also produces relatively insoluble, but stable, glucosamine hydrochloride, which can be crystallized or precipitated from solution by evaporative concentration or use of liquid precipitants. As the glucosamine separates from solution, it draws the equilibrium reaction forward, consuming additional N-acetylglucosamine. The remaining mother liquor will contain residual dissolved glucosamine hydrochloride, sodium or calcium acetate, sodium or calcium chloride and a minor amount of unreacted N-acetylglucosamine. Treating this mother liquor with alcohol as a liquid precipitant separates the soluble sodium or calcium acetate from the less soluble glucosamine hydrochloride, sodium or calcium chloride and N-acetylglucosamine, which can be recycled to the beginning of the enzymatic conversion process.</p>
    <p num="p-0278">Still another approach for converting N-acetylglucosamine enzymatically to glucosamine hydrochloride is to use the catalytic enzyme to esterify an alcohol by transferring the acetyl group from N-acetylglucosamine and to an added alcohol. Removing the ester formed during hydrolysis drives the reaction forward to consume N-acetylglucosamine, generating glucosamine free base.</p>
    <p num="p-0279">Glucosamine free base formed from enzymatic hydrolysis can be stabilized by passing it as a mixture with a chloride salt solution, e.g. sodium chloride, over a cation exchange resin in the hydrogen form, wherein the salt cation is exchanged for hydrogen ion, and stable glucosamine hydrochloride is formed, while the ion exchange resin retains the salt cations. In this same manner, a selection of any of a variety of salts, including but not limited to phosphates, sulfates, iodides, and bisulfates can be blended with the glucosamine free base, which upon passing over a cation exchange column will be converted to known stable acid salts, including (glucosamine)<sub>2 </sub>sulfate—(NaCl)<sub>2</sub>, (glucosamine)<sub>2 </sub>sulfate—(KCl)<sub>2</sub>, (glucosamine)<sub>2 </sub>sulfate, glucosamine hydrochloride, glucosamine hydroiodide, glucosamine phosphate, (glucosamine)<sub>2</sub>potassium bisulfate—(HCl)<sub>2</sub>, and (glucosamine)<sub>2 </sub>sodium bisulfate—(HCl)<sub>2</sub>.</p>
    <p num="p-0280">Therefore, in one aspect, the recombinant deacetylating enzyme discussed above is bound to a solid support, i.e., an immobilized enzyme. As used herein, an enzyme (e.g., a deacetylase) bound to a solid support includes immobilized isolated enzyme, immobilized cells which contain the enzyme, such as a recombinant enzyme (including immobilized bacterial, fungal (e.g., yeast), microalgal, insect, plant or mammalian cells), stabilized intact cells and stabilized cell/membrane homogenates. Stabilized intact cells and stabilized cell/membrane homogenates include cells and homogenates from naturally occurring microorganisms expressing the enzyme or from genetically modified microorganisms, insect cells or mammalian cells as disclosed elsewhere herein which have been genetically modified to express the enzyme (e.g., by recombinant technology). Thus, although methods for immobilizing enzymes are discussed below, it will be appreciated that such methods are equally applicable to immobilizing bacterial and other cells and in such an embodiment, the cells can be lysed.</p>
    <p num="p-0281">A variety of methods for immobilizing an enzyme are disclosed in Industrial Enzymology 2nd Ed., Godfrey, T. and West, S. Eds., Stockton Press, New York, N.Y., 1996, pp. 267-272; Immobilized Enzymes, Chibata, I. Ed., Halsted Press, New York, N.Y., 1978; Enzymes and Immobilized Cells in Biotechnology, Laskin, A. Ed., Benjamin/Cummings Publishing Co., Inc., Menlo Park, Calif., 1985; and Applied Biochemistry and Bioengineering, Vol. 4, Chibata, I. and Wingard, Jr., L. Eds, Academic Press, New York, N.Y., 1983, which are incorporated herein in their entirety.</p>
    <p num="p-0282">Briefly, a solid support refers to any solid organic supports, artificial membranes, biopolymer supports, or inorganic supports that can form a bond with the enzyme without significantly effecting the activity of the isolated enzyme. Exemplary organic solid supports include polymers such as polystyrene, nylon, phenol-formaldehyde resins, acrylic copolymers (e.g., polyacrylamide), stabilized intact whole cells, and stabilized crude whole cell/membrane homogenates. Exemplary biopolymer supports include cellulose, polydextrans (e.g., Sephadex®), agarose, collagen and chitin. Exemplary inorganic supports include glass beads (porous and nonporous), stainless steel, metal oxides (e.g., porous ceramics such as ZrO<sub>2</sub>, TiO<sub>2</sub>, Al<sub>2</sub>O<sub>3</sub>, and NiO) and sand. Preferably, the solid support is selected from the group consisting of stabilized intact cells and/or crude cell homogenates. Preparation of such supports requires a minimum of handling and cost. Additionally, such supports provide excellent stability of the enzyme.</p>
    <p num="p-0283">Enzymes can be bound to a solid support by a variety of methods including adsorption, cross-linking (including covalent bonding), and entrapment. Adsorption can be through van del Waal's forces, hydrogen bonding, ionic bonding, or hydrophobic binding. Exemplary solid supports for adsorption immobilization include polymeric adsorbents and ion-exchange resins. Solid supports in a bead form are particularly well-suited; The particle size of an adsorption solid support can be selected such that the immobilized enzyme is retained in the reactor by a mesh filter while the substrate (e.g., the oil) is allowed to flow through the reactor at a desired rate. With porous particulate supports it is possible to control the adsorption process to allow enzymes or bacterial cells to be embedded within the cavity of the particle, thus providing protection without an unacceptable loss of activity.</p>
    <p num="p-0284">Each publication and reference cited or described herein is incorporated by reference in its entirety.</p>
    <p num="p-0285">The following experimental results are provided for the purposes of illustration and are not intended to limit the scope of the invention.</p>
    <heading>EXAMPLES</heading> <p num="p-0286">The following is a list of various genetically modified microorganisms referenced and/or described herein. Some of the strains were described in U.S. Pat. No. 6,372,457, supra, and were used as parent strains for the genetic modifications described specifically herein.</p>
    <p num="p-0287">
      <tables id="TABLE-US-00004" num="00004"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="259pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>E. coli Strains</i>
                </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <colspec colname="3" colwidth="49pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain No</td>
                <td class="description-td">Description</td>
                <td class="description-td">Reference</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">W3110</td>
                <td class="description-td">F<sup>− </sup>mcrA mcrB IN(rrnD-rrnE) 1 λ<sup>−</sup> </td>
                <td class="description-td">ATCC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ATCC25947, a derivative of <i>E. coli </i>K-12</td>
              </tr> <tr class="description-tr"> <td class="description-td">7101-17</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3</td>
                <td class="description-td">U.S. Pat. No.</td>
              </tr> <tr class="description-tr"> <td class="description-td">(DE3)</td>
                <td class="description-td"> </td>
                <td class="description-td">6,372,457</td>
              </tr> <tr class="description-tr"> <td class="description-td">2123-4</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7lac-glmS*4::ΔlacZ Cam<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">2123-12</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7lac-glmS::ΔlacZ Cam<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">2123-54</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7lac-glmS*54::ΔlacZ Cam<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">2123-59</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7lac-glmS*59::ΔlacZ Cam<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">2123-64</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7lac-glmS*64::ΔlacZ Cam<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">2123-72</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7lac-glmS*72::ΔlacZ Cam<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">2123-103</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7lac-glmS*103::ΔlacZ Cam<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">2123-124</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7lac-glmS*124::ΔlacZ Cam<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-16</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7-glmS*54::galK</td>
                <td class="description-td">Example 6</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-18</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7-glmS*54::galK</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-22</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 pET24d(+) Kan<sup>R</sup> </td>
                <td class="description-td">Example 2</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-23</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 pET24d(+)/T7-CaGFA1 Kan<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-24</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 pET24(d+)/T7-BsglmS Kan<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-58</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 pET23b(+)/T7-CaGFA1 Kan<sup>R</sup> </td>
                <td class="description-td">Example 2</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-60</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 pET23b(+)/T7-CaGFA1-M Kan<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-84</td>
                <td class="description-td">7107-18, Δlacl(DE3)</td>
                <td class="description-td">Example 28</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-88</td>
                <td class="description-td">7107-18, pET24d(+) Kan<sup>R</sup> </td>
                <td class="description-td">Example 13</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-87</td>
                <td class="description-td">7107-18, pET24d(+)/T7-ScGNA1 Kan<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-90</td>
                <td class="description-td">7107-18, ΔpfkA</td>
                <td class="description-td">Example 22</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-92</td>
                <td class="description-td">7107-18, T7-ScGNA1::ΔmanXYZ</td>
                <td class="description-td">Example 16</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-93</td>
                <td class="description-td">7107-18, pET24d(+)/T7-AtGNA1 Kan<sup>R</sup> </td>
                <td class="description-td">Example 13</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-95</td>
                <td class="description-td">7107-18, T7-ScGNA1::ΔmanXYZ pET24d(+) Kan<sup>R</sup> </td>
                <td class="description-td">Example 24</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-96</td>
                <td class="description-td">7107-18, T7-ScGNA1::ΔmanXYZ pET24d(+)/T7-zwf Kan<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-101</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 pET24(+)/T7-ScGFA1 Kan<sup>R</sup> </td>
                <td class="description-td">Example 2</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-117</td>
                <td class="description-td">7107-18, pET24d(+)/T7-CaGNA1 Kan<sup>R</sup> </td>
                <td class="description-td">Example 13</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-118</td>
                <td class="description-td">7107-18, T7-glnA::ΔpfkB</td>
                <td class="description-td">Example 23</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-119</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-120</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-124</td>
                <td class="description-td">7107-18, T7-ScGNA1::ΔmanXYZ pET24d(+)/T7-pgi Kan<sup>R</sup> </td>
                <td class="description-td">Example 26</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-125</td>
                <td class="description-td">7107-18, T7-glnA::ΔpfkB T7-ScGNA1::ΔmanXYZ</td>
                <td class="description-td">Example 23</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-126</td>
                <td class="description-td">7107-18, T7-glnA::ΔpfkB T7-ScGNA1::ΔmanXYZ</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-133</td>
                <td class="description-td">7107-18, T7-glnA::ΔpfkB, T7-ScGNA1::ΔmanXYZ</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-136</td>
                <td class="description-td">7107-18, T7-ScGNA1::ΔmanXYZ T7-pgi::ΔaraBAD</td>
                <td class="description-td">Example 26</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-141</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-163</td>
                <td class="description-td">7107-18, pET24d(+)/T7-glnA Kan<sup>R</sup> </td>
                <td class="description-td">Example 23</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-214</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 pET24d(+)/T7-glmS Kan<sup>R</sup> </td>
                <td class="description-td">Example 2</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-308</td>
                <td class="description-td">7107-18, ΔglgXCA</td>
                <td class="description-td">Example 27</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-309</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-310</td>
                <td class="description-td">7107-18, lacUV5 promoter replacing the lac promoter</td>
                <td class="description-td">Example 28</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-313</td>
                <td class="description-td">7107-18, lacUV5 replacement Δlacl (lac)</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-314</td>
                <td class="description-td">7107-18, lacUV5 replacement Δlacl (lac)</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-315</td>
                <td class="description-td">7107-18, lacUV5 replacement Δlacl (lac)</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-321</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7-glmS*54::Δnag</td>
                <td class="description-td">Example 29</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-325</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7-glmS*54::Δnag T7-</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ScGNA1::ΔmanXYZ</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-326</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-327</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-328</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-512</td>
                <td class="description-td">UV mutagenized 7107-92 (i.e. 7107-18, T7-</td>
                <td class="description-td">Example 21</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ScGNA1::ΔmanXYZ</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-513</td>
                <td class="description-td">UV mutagenized 7107-92 (i.e. 7107-18, T7-</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ScGNA1::ΔmanXYZ)</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-602</td>
                <td class="description-td">7107-18, ΔpfkA T7-ScGNA1::ΔmanXYZ</td>
                <td class="description-td">Example 22</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-603</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-606</td>
                <td class="description-td">7107-18, T7-ScGNA1::ΔmanXYZ T7-zwf::ΔrhaBAD</td>
                <td class="description-td">Example 24</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-607</td>
                <td class="description-td">7107-18, T7-ScGNA1::ΔmanXYZ T7-ScGNA1::ΔfuclK</td>
                <td class="description-td">Example 16</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">(two copies of ScGNA1)</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-608</td>
                <td class="description-td">7107-18, T7-ScGNA1::ΔmanXYZ T7-ScGNA1::ΔfuclK</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">T7-ScGNA1::ΔtreB (three copies of ScGNA1)</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-609</td>
                <td class="description-td">7107-18, T7-ScGNA1::ΔmanXYZ T7-ScGNA1::ΔfuclK</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">T7-ScGNA1::ΔmelAB (three copies of ScGNA1)</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-610</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-611</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-612</td>
                <td class="description-td">7107-18, T7-ScGNA1::ΔmanXYZ T7-ScGNA1::ΔfuclK</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">T7-ScGNA1::ΔtreB T7-ScGNA1::ΔmelAB (four copies of</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ScGNA1)</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-613</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-633</td>
                <td class="description-td">7107-18, T7-ScGNA1::ΔmanXYZ T7-ScGNA1::ΔfuclK</td>
                <td class="description-td">Example 25</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">zwf::ΔrhaBAD</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-634</td>
                <td class="description-td">7107-18, T7-ScGNA1::ΔmanXYZ zwf::ΔrhaBAD</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-636</td>
                <td class="description-td">7107-18, pET24d(+)/T7-nagB Kan<sup>R</sup> </td>
                <td class="description-td">Example 14</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-637</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-638</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-645</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7-nagB::ΔpfkB</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-646</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7-nagB::ΔpfkB ΔglmS</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-660</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 T7-nagB::ΔpfkB ΔglmS T7-</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ScGNA1::ΔmanXYZ</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-661</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-667</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 pET24d(+)/T7-glmU Kan<sup>R</sup> </td>
                <td class="description-td">Example 15</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-668</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-669</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 pET24d(+)/T7-glmM Kan<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-670</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-671</td>
                <td class="description-td">Tet<sup>R</sup>::Δnag manXYZ DE3 pET24d(+)/T7-glmU-t Kan<sup>R</sup> </td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-672</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-678</td>
                <td class="description-td">7107-18, T7-glmU::Δnag</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-679</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-680</td>
                <td class="description-td">7107-18, T7-glmUt::Δnag</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-681</td>
                <td class="description-td">ib.</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-682</td>
                <td class="description-td">7107-18, T7-glmM::Δglg</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-683</td>
                <td class="description-td">7107-18, T7-glmM::Δglg (glmM oriented opposite to glg)</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-685</td>
                <td class="description-td">7107-18, T7-glmU::Δnag T7-glmM::Δglg (glmM oriented</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">opposite to glg)</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-687</td>
                <td class="description-td">7107-18, T7-glmUt::Δnag T7-glmM::Δglg (glmM oriented</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">opposite to glg)</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-689</td>
                <td class="description-td">7107-18, T7-glmU::Δnag T7-glmM::Δglg</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-692</td>
                <td class="description-td">7107-18, T7-glmUt::Δnag T7-glmM::Δglg</td>
                <td class="description-td">ib.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="left" id="FOO-00001" class="description-td" colspan="3">Note:</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="left" id="FOO-00002" class="description-td" colspan="3">1) All strains listed in the table were derived from the same parent strain W3110.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="left" id="FOO-00003" class="description-td" colspan="3">2) The majority of strains listed in the table were developed from strain 7107-18 (Tet<sup>R</sup>:: nag manXYZ DE3 T7-glmS*54::galK). To simplify, the genotype of these strains are listed as 7107-18 plus new changes.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="left" id="FOO-00004" class="description-td" colspan="3">3) Genes from other sources than <i>E. coli </i>are identified with two letters. Sc: <i>Saccharomyces </i> <i>cerevisiae</i>, Ca: <i>Candita albicans</i>, At: <i>Arabidopsis thaliana</i>, Bs: <i>Bacillus subtilis</i>.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="left" id="FOO-00005" class="description-td" colspan="3">4) For gene integration, the inserted expression cassttte is oriented the same as the gene or operon of the target site except when it is indicated otherwise.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 1</heading> <p num="p-0288">The following example describes mutant screening for better glucosamine producers.</p>
    <p num="p-0289">U.S. Pat. No. 6,372,457, incorporated herein by reference in its entirety, described recombinant <i>E. coli </i>strains that produce glucosamine at high levels. These strains were constructed using a metabolic engineering approach. This approach is consisted of three steps. This first step was to introduce mutations that restrict metabolism and import of glucosamine and its 6-phosphate derivative. The second step was to over-express the <i>E. coli </i>glmS gene coding for glucosamine synthase (GlmS), the key biosynthetic enzyme. The third step was to minimize product inhibition of the GlmS by in vitro mutagenesis of the enzyme. The recombinant strain 2123-54 contained a product-resistant recombinant <i>E. coli </i>glmS mutant gene under T7 promoter control and showed the highest product titer in shake flask culture in a simple mineral salt medium supplemented with glucose. This strain was used as a reference to evaluate new strains for further improvements. It was also used in experiments designed to improve cell growth and IPTG induced glucosamine production.</p>
    <p num="h-0011">Genotype of IPTG-Inducible Glucosamine Production Strain 2123-54:</p>
    <p num="p-0290">2123-54 was derived from a laboratory <i>E. coli </i>K-12 strain designated W3110. The relevant genotype was described in Table 1.</p>
    <p num="p-0291">
      <tables id="TABLE-US-00005" num="00005"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="280pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Genotype of Glucosamine Production Strain 2123-54</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Genetic Change</td>
                <td class="description-td">Description</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Tet<sup>R </sup>at Δnag</td>
                <td class="description-td">Replacement of the nag regulon sequence with a tetracycline resistance</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">marker for eliminating genes for glucosamine-6-phosphate metabolism</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">and glucosamine uptake</td>
              </tr> <tr class="description-tr"> <td class="description-td">manXYZ mutation</td>
                <td class="description-td">Mutation eliminating mannose-specific sugar transport system, which</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">also transports glucosamine</td>
              </tr> <tr class="description-tr"> <td class="description-td">lacZ::T7-glmS*54</td>
                <td class="description-td">An integrated construct for overexpressing the glmS gene encoding for</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GlcN6P synthase. The glmS gene is placed under control by the T7</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">promoter. The construct is integrated into the <i>E. coli </i>chromosome in the</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">lacZ gene. The *54 designation indicates mutations in glmS which result</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">in an enzyme resistant to feedback inhibition by its product</td>
              </tr> <tr class="description-tr"> <td class="description-td">DE3</td>
                <td class="description-td">Genetic element which encodes the gene for T7 RNA polymerase driven</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">by the lacUV promoter which is inducible by IPTG</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0292">The nag and manXYZ mutations were shown to have a positive influence on glucosamine production and continued to be carried into new production strains. By using the T7 expression system the <i>E. coli </i>wild type glmS gene was over-expressed and a few-fold higher level of glucosamine production was produced using IPTG induction. Since the <i>E. coli </i>wild type GlmS is strongly inhibited by its product, glucosamine-6-P, pools of <i>E. coli </i>glmS mutants were created by error-prone PCR and screened by plate feeding assay for increased glucosamine production. The expression constructs expressing glmS* mutants in the improved glucosamine producers were integrated in the chromosome at the lacZ site to generate stable production strains. Many mutant strains synthesized a glucosamine synthase that was product resistant. In shake flask experiments, these mutant strains produced drastically increased levels of glucosamine as compared to the strain with the <i>E. coli </i>wild type glmS expression construct. The strain 2123-54 (with the T7-glmS*54 expression construct) produced over 11 g glucosamine per liter.</p>
    <p num="h-0012">Developing a Simplified Shake Flask Screening Method:</p>
    <p num="p-0293">As described in U.S. Pat. No. 6,372,457, different glucosamine production strains: were grown in a mineral salt medium supplemented with glucose for glucosamine production in shake flask culture. In previous experiments, as they became depleted, glucose and ammonium sulfate were fed to the culture, leading to a continuous glucosamine production. However, this requires frequent monitoring and feeding (every 6-8 hrs) for a period of 3 days. Therefore, it was desirable to develop a shake flask culture protocol that was simpler, yet reliable for evaluating different glucosamine production strains.</p>
    <p num="p-0294">The simple mineral salt medium used in glucosaminee production was M9A (Table 2). A three-step protocol was developed for strain evaluation. First, cells freshly grown on LB plates were used to start a culture in 3-ml LB, which was grown at 37° C. for about 8 hrs. Second, 1.5-ml of the culture was used to inoculate 50 ml of M9A medium in a 250-ml shake flask and the culture was incubated at 37° C. and shaken at 225 rpm for about 16 hrs (overnight). Cell density of the culture was measured at 600 nm. This step was included for cells to adapt to the minimal medium and for reproducible results of glucosamine production. Third, aliquots of cells were added to 50 ml of M9A medium containing IPTG (0.2 mM) in a 250-ml shake flask. The initial cell OD was 0.3. Cells were incubated at 37° C. and shaken at 225 rpm for 72 hrs. Samples (1 ml) were taken at 24, 48 and 72 hrs to determine levels of glucosamine in the culture broth. At the 24 and 48 hr time points, pH was adjusted to 7.0 by addition of small amount of NaOH and glucose was added at 20 g l<sup>−1</sup>. Under these experimental conditions, the control strain 2123-54 produced about 6 g l<sup>−1 </sup>glucosamine at 72 hrs.</p>
    <p num="p-0295"> <tables id="TABLE-US-00006" num="00006"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 2</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">M9A and M9B medium used for glucosamine production*</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="70pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Macroelements (g l<sup>−1</sup>)</td> <td class="description-td">M9A</td> <td class="description-td">M9B</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="70pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">KH<sub>2</sub>PO<sub>4</sub> </td> <td class="description-td">14</td> <td class="description-td">6</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">K<sub>2</sub>HPO<sub>4</sub> </td> <td class="description-td">16</td> <td class="description-td">24</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Na<sub>3</sub>Citrate-2H<sub>2</sub>O</td> <td class="description-td">1</td> <td class="description-td">1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> </td> <td class="description-td">7.5</td> <td class="description-td">7.5</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">MgSO<sub>4</sub>-7H<sub>2</sub>O</td> <td class="description-td">0.25</td> <td class="description-td">0.25</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CaCl<sub>2</sub>-2H<sub>2</sub>O</td> <td class="description-td">0.015</td> <td class="description-td">0.015</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Glucose</td> <td class="description-td">20</td> <td class="description-td">20</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">pH</td> <td class="description-td">7.0</td> <td class="description-td">7.0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="left" id="FOO-00006" class="description-td" colspan="4">*For fermentation the medium is supplemented with antifoam (Mazu 204 and 0.25 ml l<sup>−1</sup>)</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Mutant Screening in Shake Flask Experiments:
</p>
    <p num="p-0296">About 150 recombinant <i>E. coli </i>strains with integrated glmS mutant constructs were re-evaluated using the optimized protocol. Glucosamine was assayed following the colorimetric method using Ehrlich's reagent. The strain 2123-72 and 2123-103 produced glucosamine at levels slightly higher than 2123-54 (Table 3).</p>
    <p num="p-0297"> <tables id="TABLE-US-00007" num="00007"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 3</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">IPTG induced glucosamine production in shake flask culture</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="49pt" align="left"> </colspec> <colspec colname="1" colwidth="168pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Glucosamine (g l<sup>−1</sup>)</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="center"> </colspec> <colspec colname="3" colwidth="49pt" align="center"> </colspec> <colspec colname="4" colwidth="56pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Strains</td> <td class="description-td">24 hr</td> <td class="description-td">48 hr</td> <td class="description-td">72 hr</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">2123-54</td> <td class="description-td"> 2.085 (100%)*</td> <td class="description-td">5.599 (100%)*</td> <td class="description-td"> 6.462 (100%)*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">2123-72</td> <td class="description-td">2.141 (103%)</td> <td class="description-td">6.699 (120%) </td> <td class="description-td">7.791 (120%)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">2123-103</td> <td class="description-td">2.390 (115%)</td> <td class="description-td">7.230 (129%) </td> <td class="description-td">8.712 (135%)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="4" align="left" id="FOO-00007" class="description-td" colspan="5">*The percentage of numbers shown in the parentheses are relative to the values from 2123-54 at different time points.</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Evaluation of High Glucosamine Producing Strains 2123-72 and 2123-103 in 1-Liter Fermentors:
</p>
    <p num="p-0298">The potentially better glucosamine production strains, 2123-72 and 2123-103 were compared to 2123-54 in 1-liter fermentors. Fermenters were set up with an initial volume of 475 ml of M9A medium supplemented with antifoam (Mazu 204 at 0.25 ml l<sup>−1</sup>) and trace elements (Table 4). Fermentations were run using 75% NH<sub>4</sub>OH for pH control to 6.9. Temperature was maintained at 30° C. throughout the fermentation. Aeration and agitation were adjusted to maintain a dissolved oxygen concentration of 20% of air saturation. 65% glucose was fed to the cultures with feed rate controlled by computer program to achieve a growth rate of 0.40 hr<sup>−1 </sup>at inoculation and a maximum rate of 5 ml/hr by 6 hours. Fermentation allowed for precise control of pH, oxygen, and glucose concentration. Higher glucosamine concentration was achieved in the fermentors than in flasks.</p>
    <p num="p-0299">Two sets of each strain were run under two different conditions. One set of fermentors started with a low glucose concentration (10 g l<sup>−1</sup>, glucose limiting) and the other is set started with a higher concentration of glucose (40 g l<sup>−1</sup>, excess glucose). The excess glucose conditions more closely resemble the shake flask growth conditions. The glucose limiting conditions were normally used in fermentation experiments and generally led to higher glucosamine production with 2123-54. The cultures were all induced with IPTG from the start of the fermentation. Under either glucose limiting or excess conditions, both strains 2123-72 and 2123-103 performed better than 2123-54, producing up to 14 g l<sup>−1 </sup>glucosamine by 50 hrs, as compared to production of up to 10 gl<sup>−1 </sup>glucosamine by 50 hours for 2123-54 (data not shown).</p>
    <p num="p-0300">
      <tables id="TABLE-US-00008" num="00008"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 4</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Trace elements supplemented to growth medium used in some</td>
              </tr> <tr class="description-tr"> <td class="description-td">experiments</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="49pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="112pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Microelements</td>
                <td class="description-td">mg l<sup>−1</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="49pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="112pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CoCl<sub>2</sub>-6H<sub>2</sub>O</td>
                <td class="description-td">0.87</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H<sub>3</sub>BO<sub>3</sub> </td>
                <td class="description-td">1.72</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CuCl<sub>2</sub>-2H<sub>2</sub>O</td>
                <td class="description-td">0.60</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">FeCl<sub>3</sub>-6H<sub>2</sub>O</td>
                <td class="description-td">10.50</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">MnCl<sub>2</sub>-4H<sub>2</sub>O</td>
                <td class="description-td">12.00</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ZnCl<sub>2</sub> </td>
                <td class="description-td">1.50</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Na<sub>2</sub>MoO<sub>4</sub>-2H<sub>2</sub>O</td>
                <td class="description-td">1.50</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 2</heading> <p num="p-0301">The following example describes over-expressing different glmS genes for glucosamine production.</p>
    <p num="p-0302">Bacterial glucosamine synthase genes (glmS) and the yeast homologues (GFA genes) were cloned and expressed in <i>E. coli </i>to demonstrate their utility in glucosamine metabolic pathway engineering. Different genes were amplified from <i>Bacillus subtilis, Saccharomyces cerevisiae </i>and <i>Candida albicans </i>by PCR and placed under T7 promoter control in the expression vector pET24d(+). The constructs were transformed into the <i>E. coli </i>strain 7101-17(DE3) and maintained as free replicating plasmids. Additionally, the <i>C. albicans </i>GFA1 gene driven by the T7 promoter was integrated in the chromosome at the lacZ site in 7101-17 (DE3). Strains hosting different glmS and GFA genes were evaluated for gene expression and glucosamine production using IPTG induction.</p>
    <p num="h-0014"> <i>B. subtilis </i>glmS Gene Cloning:</p>
    <p num="p-0303">The <i>B. subtilis </i>glmS gene contains an open reading frame of 1803 bp and encodes a protein of about 65 kDa (599 residues, excluding the initiator methionine which is usually removed in the cells). The nucleotide sequence of the <i>B. subtilis </i>glmS open reading frame is listed in SEQ ID NO:15. The deduced amino acid sequence for the <i>B. subtilis </i>GlmS protein is listed in SEQ ID NO:16. The glmS gene was amplified by PCR from the strains ATCC 23856 and ATCC 23857. The forward primer contained the ATG start codon and a Bsa I site (SEQ ID NO:21): 5′-GAT CGG TCT CGC ATG TGT GGA ATC GTA GGT TAT ATC GGT C-3′. The reverse primer contained the stop codon and a Xho I site (SEQ ID NO:22): 5′-GAT CCT CGA GTT ACT CCA CAG TAA CAC TCT TCGCAA GGT TAC G-3.</p>
    <p num="p-0304">PCR products of expected size were ligated into pET24d(+) (Novagen Inc, Wisconsin). The vector was predigested with the enzymes Nco I and Xho I. The recombinant plasmids pSW 07-15#83 were confirmed by restriction analysis and transformed into 7101-17(DE3), generating <i>E. coli </i>strains 7107-24 (glmS gene from <i>B. subtilis </i>ATCC23856) and 7107-25 (glmS gene from <i>B. subtilis </i>ATCC23857). As a control, the empty vector pET24d(+) was also transformed into 7101-17(DE3), generating the strain 7107-22.</p>
    <p num="h-0015"> <i>S. cerevisiae </i>GFA1 Gene Cloning:</p>
    <p num="p-0305">The <i>S. cerevisiae </i>GFA1 open reading frame has 2154 bp and codes for a peptide of 716 residues (excluding the initiator methionine). The nucleotide sequence of the <i>S. cerevisiae </i>GFA1 open reading frame is listed in SEQ ID NO:17. The deduced amino acid sequence for the <i>S. cerevisiae </i>GFA1 protein is listed in SEQ ID NO:18. The protein size predicted from the sequence is about 80 kDa. No introns are present in the GFA1 gene sequence, therefore, the gene was amplified from genomic DNA prepared from the strain <i>S. cerevisiae </i>S288C (ATCC204508).</p>
    <p num="p-0306">The forward primer, including the ATG start codon and a Bsa I site, had the following sequence (SEQ ID NO:23): 5′-GAT CGG TCT CGC ATG TGT GGT ATC TTT GGT TAC-3′. The reverse primer, including the stop codon and an EcoR I site, had the following sequence (SEQ ID NO:24): 5′-GAT CGA ATT CTT ATT CGA CGG TAA CAG ATT TAG-3′.</p>
    <p num="p-0307">The PCR product of about 2.2 kb was cloned into pPCR-Script Amp SK(+). Recombinant plasmids were confirmed by restriction enzyme digestions. The <i>S. cerevisiae </i>GFA1 fragment was isolated by digestion with EcoR I and Bsa I and ligated into the EcoR I and Nco I sites of pET24d(+). The recombinant plasmid was confirmed by restriction analysis and transformed into 7101-17(DE3), generating the <i>E. coli </i>strain 7107-101.</p>
    <p num="h-0016"> <i>Candida albicans </i>GFA1 Cloning:</p>
    <p num="p-0308">The <i>C. albicans </i>GFA1 gene is free of introns and its 2142-bp open reading frame encodes a peptide of about 80 kDa (712 residues, excluding the initiator methionine). The nucleotide sequence of the <i>C. albicans </i>GFA1 open reading frame is listed in SEQ ID NO:19. The deduced amino acid sequence for the <i>C. albicans </i>GFA1 protein is listed in SEQ ID NO:20. The GFA1 coding sequence was amplified from the strain ATCC10261 by PCR using a forward primer and a reverse primer. The forward primer contained the ATG start codon and a Bsa I site: 5′-GAT CGG TCT CGC ATG TGT GGT ATT TTT GGT TAC GTC-3′ (SEQ ID NO:25). The reverse primer contained the stop codon and a Xho I site: 5′-GAT CCT CGA GTT ACT CAA CAG TAA CTG ATT TAG CC-3′ (SEQ ID NO:26).</p>
    <p num="p-0309">The PCR product was cloned into the vector pMOSBlue (Amersham Pharmacia Biotech, New Jersey) and recombinant plasmids were confirmed by restriction enzyme digestion. The Bsa I-Xho I fragment was isolated and ligated into pET24d(+) prepared by digestion with Nco I and Xho I. The resultant plasmid was transformed into the host 7101-17 (DE3), generating the <i>E. coli </i>strain 7107-23.</p>
    <p num="p-0310">The <i>C. albicans </i>GFA1 gene was also cloned into the expression vector pET23b(+) (Novagen Inc). Unlike pET24d(+), this vector does not contain a lac operator sequence downstream from the T7 promoter. The absence of the lac operator could result in a higher recombinant protein expression level. The <i>C. albicans </i>GFA1 coding sequence was amplified by PCR from the yeast genomic DNA. The forward primer contained the ATG start codon and a Nde I site: 5′-GCG GGT ACC CAT ATG TGT GGT ATT TTT GGT TAC GT-3′ (SEQ ID NO:27). The reverse primer contained a BamHI site: 5′-GCG GGA TCC TTA CTC AAC AGT AAC TGA TTT AGC CA-3′ (SEQ ID NO:28). The PCR products of correct size were ligated into pET23b at the Nde I and BamHI sites. The recombinant plasmid was confirmed by restriction analysis and transformed into the expression host 7101-17(DE3), generating <i>E. coli </i>strains 7107-58 and 7107-59. As a control, the empty vector pET23b was also transformed into 7101-17(DE3), generating the strain 7107-57.</p>
    <p num="p-0311">Overexpression of the <i>Candida albicans </i>GFA1 protein using the <i>E. coli </i>pET expression system was previously described in a publication (P. Sachadyn et al., Protein Expression and Purification 19, 343-349 2000). However, glucosamine production was not demonstrated or discussed at all. In addition, the reported GFA1 gene was cloned from a different <i>C. albicans </i>strain (ATCC13153). Although two GFA1 proteins have the same amino acid sequences, the GFA1 gene from ATCC13153 has the residues Leu 29 and Ala 655 coded by CTA and GCC, respectively, instead of TTA and GCT as in ATCC10261. An attempt was made to test if the use of different codons at the residues Leu 29 and Ala 655 has an impact on GFA1 gene expression in <i>E. coli</i>. Site directed mutagenesis using a strategy based on the Stratagene QuikChange® Site-Directed Mutagenesis Kit (Stratagene, CA) was performed to convert the Leu 29 codon to CTA and the Ala 655 codon to GCC in the plasmid pET23b(+)/<i>C. albicans </i>GFA1, creating the plasmid pET23b(+)/<i>C. albicans </i>GFA1-M. The presence of both mutations in the new plasmid was confirmed by sequencing and the plasmid was transformed into 7101-17(DE3), generating <i>E. coli </i>strains 7107-60 and 7107-61.</p>
    <p num="h-0017">GlmS and GFA Protein Over-Expression:</p>
    <p num="p-0312">Strains transformed with pET vectors containing different glmS and GFA1 genes were grown in LB medium to demonstrate GlmS and GFA1 protein expression. First, cells were grown in LB medium in test tubes at 37° C. overnight. The medium was supplemented with kanamycin (25 mg l<sup>−1</sup>) to maintain plasmids. Then an inoculum of 50-μl overnight culture was used to start a 50-ml culture in a 250-ml baffled flask. Cultures were incubated at 37° C. and shaken at 225 rpm for 3 hrs. At that point, IPTG was added to a final concentration of 1 mM. After an induction period of 3 hrs, cultures were harvested for SDS-PAGE analysis. As a negative control, cells with the empty pET24d(+) vector were also grown and analyzed. For comparison, <i>E. coli </i>cells with the wild-type <i>E. coli </i>glmS gene and mutant glmS*54 gene driven by the T7 promoter and integrated in the chromosome at the lacZ site were also grown as above without antibiotic selection.</p>
    <p num="p-0313">SDS-PAGE was carried out by following the standard methods. When the T7-<i>E. coli </i>glmS expression cassette was carried in pET plasmids or integrated in the chromosome, the GlmS protein was expressed at very high levels (data not shown). Cells hosting the plasmids pET24d(+)/T7-<i>B. subtilis </i>glmS over-expressed a protein of about 65 kDa, the expected size of the GlmS protein (data not shown). The expression level from the integrated cassette was comparable to the cells expressing the <i>E. coli </i>glmS gene contained in pET plasmids.</p>
    <p num="p-0314">Cells hosting the <i>S. cerevisiae </i>GFA1 gene showed a clearly over-expressed protein band of the expected size for the yeast protein (80 kDa, data not shown). In the strain 7107-23, containing the T7-<i>C. albicans </i>GFA1 expression cassette (data not shown), the synthesis of the 80-kDa protein band was not apparent when compared to the strain with the empty vector (data not shown). However, the GFA1 band was over-expressed in the strains 7107-58 and 7107-59 containing the <i>C. albicans </i>GFA1 gene carried by the vector pET23b(+)-based vector (data not shown). The expression level was clearly higher than in the strain 7107-23 with the pET24d(+)-based vector. The use of alternative codons for Leu 29 and Ala 655 did not affect <i>C. albicans </i>GFA 1 protein expression in <i>E. coli. </i> </p>
    <p num="p-0315">All together, the expression levels of the yeast GFA1 genes in <i>E. coli </i>were low as compared to bacterial glmS genes. This is commonly observed when attempting to express eukaryotic genes in <i>E. coli </i>hosts.</p>
    <p num="h-0018">Glucosamine-6-Phosphate Synthase Activity Assay:</p>
    <p num="p-0316">For measurement of enzyme activity and glucosamine production, different strains were grown in M9A medium. A three-step protocol was used for preparing the cultures. First, cells freshly grown on LB plates were used to start a culture in 3-ml LB, which was grown at 37° C. for about 6 hrs. Second, 1.5-ml of the culture was used to inoculate 50 ml of M9A in a 250-ml baffled flask and the culture was incubated at 37° C. and shaken at 225 rpm for about 16 hrs (overnight). This step was included for cells to adapt to the minimal medium and for reproducible results of glucosamine production. Third, aliquots of cells were added to 50 ml of M9A medium containing IPTG (1 mM) in a 250-ml baffled flask. The initial cell density was adjusted to 0.3 OD<sub>600</sub>. Cells were incubated at 37° C. and shaken at 225 rpm for 24 hrs. Following centrifugation, the culture broth was used in glucosamine assays and the cell pellet was used to determine glucosamine synthase activity. Data from representative experiments are shown in Table 5.</p>
    <p num="p-0317">The enzyme activity was readily detectable in <i>E. coli </i>cells expressing the <i>B. subtilis </i>glmS genes (encoding SEQ ID NO:16). The activity level was comparable to the cells with a construct containing the <i>E. coli </i>glmS (encoding SEQ ID NO:2) and <i>E. coli </i>glmS*54 mutant (encoding SEQ ID NO:6) genes. However, only a trace amount of enzyme activity could be detected in cells hosting the yeast GFA1 genes (encoding SEQ ID NO:18 and SEQ ID NO:20). The activity data from cultures in M9A medium were generally consistent with the results of SDS-gel analysis of cells grown in LB medium. Low protein expression levels appeared to be one of the main reasons accounting for poor enzyme activity in cells hosting the yeast GFA1 genes.</p>
    <p num="h-0019">Glucosamine Production by Expressing Different glmS and GFA1 Genes:</p>
    <p num="p-0318">Only a very low level of glucosamine was produced and secreted into the culture medium of 7101-17(DE3) cells transformed with an empty vector pET24d(+) (Table 5). Expression of a bacterial glmS gene (<i>E. coli </i>glmS or <i>B. subtilis </i>glmS) resulted in a greater than 50-fold increase in glucosamine production. A several-fold increase in glucosamine level was also observed in the cultures expressing yeast GFA1 genes. As compared to pET24d(+), the use of pET23b(+) led to a higher level of <i>C. albicans </i>GFA1 protein and a higher level of glucosamine production. Change of the Leu 29 and Ala 655 codons in the <i>C. albicans </i>GFA1 gene did not affect glucosamine production levels. These observations were generally consistent with the results of SDS-PAGE analysis. As observed in enzyme activity assays, integration of the T7-<i>E. coli </i>glmS expression cassette in the chromosome appeared to be beneficial, as a higher glucosamine level was produced in the strain 2123-12 than in 7107-214. Clearly, the <i>E. coli </i>strain with <i>E. coli </i>glmS*54 integrated in the chromosome was superior for glucosamine production when compared to other tested strains.</p>
    <p num="p-0319">
      <tables id="TABLE-US-00009" num="00009"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 5</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Glucosamine synthase activity and glucosamine production in</td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>E. coli </i>strains expressing different glmS and GFA1 homologues</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="112pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Enzyme</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">activity</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(nmol</td>
              </tr> <tr class="description-tr"> <td class="description-td">Strain</td>
                <td class="description-td">Strain</td>
                <td class="description-td">min<sup>1</sup> </td>
                <td class="description-td">Glucosamine</td>
              </tr> <tr class="description-tr"> <td class="description-td">Number</td>
                <td class="description-td">description</td>
                <td class="description-td">mg<sup>−1</sup>)</td>
                <td class="description-td">(mg l<sup>−1</sup>)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="112pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">7107-22</td>
                <td class="description-td">pET24d(+)</td>
                <td class="description-td">trace</td>
                <td class="description-td">5</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-24</td>
                <td class="description-td">pET24d(+)/T7-<i>B. subtilis </i>glmS</td>
                <td class="description-td">637</td>
                <td class="description-td">128</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">23856</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-101</td>
                <td class="description-td">pET24d(+)/T7-<i>S. cerevisea </i>GFA1</td>
                <td class="description-td">trace</td>
                <td class="description-td">47</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-23</td>
                <td class="description-td">pET24d(+)/Ty-<i>C. albicans </i>GFA1</td>
                <td class="description-td">trace</td>
                <td class="description-td">23</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-58</td>
                <td class="description-td">pET23b(+)/T7-<i>C. albicans </i>GFA1</td>
                <td class="description-td">trace</td>
                <td class="description-td">54</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-60</td>
                <td class="description-td">pET23b(+)/T7-<i>C. albicans </i>GFA1-M</td>
                <td class="description-td">trace</td>
                <td class="description-td">58</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-214</td>
                <td class="description-td">pET24d(+)/T7-<i>E. coli </i>glmS</td>
                <td class="description-td">297</td>
                <td class="description-td">37</td>
              </tr> <tr class="description-tr"> <td class="description-td">2123-12</td>
                <td class="description-td">lacZ::T7-<i>E. coli </i>glmS</td>
                <td class="description-td">613</td>
                <td class="description-td">75</td>
              </tr> <tr class="description-tr"> <td class="description-td">2123-54</td>
                <td class="description-td">lacZ::T7-<i>E. coli </i>glmS*54</td>
                <td class="description-td">803</td>
                <td class="description-td">2,029</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="left" id="FOO-00008" class="description-td" colspan="4">Notes:</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="left" id="FOO-00009" class="description-td" colspan="4">1) Host cell: <i>E. coli </i>7101-17 (DE3). Genotype: nagΔ, manXYZ DE3.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="left" id="FOO-00010" class="description-td" colspan="4">2) Cell culture: 30° C. for 26 hrs in shake flasks containing M9A medium supplemented with 7.5 g (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>per liter and 40 g glucose per liter.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="left" id="FOO-00011" class="description-td" colspan="4">3) <i>C. albicans </i>GFA1 (M): Leu 29 and Ala 655 codons changed from TTA and GCT to CTA and GCC, respectively.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 3</heading> <p num="p-0320">The following example shows the characterization of different product-resistant GlmS enzymes: <i>E. coli </i>GlmS mutants and wild type <i>B. subtillus </i>GlmS.</p>
    <p num="p-0321">Different glucosamine synthetase enzymes, including native <i>B. subtillus </i>GlmS, native <i>E. coli </i>GlmS and mutant <i>E. coli </i>GlmS, were studied in vitro. Cells of various <i>E. coli </i>strains were grown in the M9A medium, harvested, and frozen. Cells extracts were prepared and the glucosamine synthetases were characterized and compared. The DNA sequences were determined for two additional <i>E. coli </i>glmS mutants that showed strong product resistance and led to high glucosamine production in recombinant <i>E. coli. </i> </p>
    <p num="h-0021">Sensitivity to Inhibition by Glucosamine-6-P:</p>
    <p num="p-0322">Enzyme sensitivities to the reaction product glucosamine-6-P were examined. Initial velocity was examined at saturating glutamine and fructose-6-phosphate levels over a range of 0-30 mM glucosamine-6-P. Exemplary results are shown in <figref idrefs="DRAWINGS">FIGS. 4 and 5</figref>. The <i>E. coli </i>native GlmS enzyme from the strain 2123-12 loses about 50% activity at 1 mM level of glucosamine-6-P. Activity continues to decrease with increasing levels of glucosamine-6-P. In strain 2123-54, 1 mM glucosamine-6-P essentially has no effect on enzyme activity. Above this level inhibition is fairly linear, with roughly 50% activity remaining at 10 mM glucosamine-6-P. Mutant GlmS enzymes from other strains such as 2123-4, 2123-59, 2123-64, 2123-72,2123-103 and 2123-124 also showed reduced sensitivity to GlcN-6-P inhibition. <figref idrefs="DRAWINGS">FIG. 5</figref> shows activity at relatively “low” [glucosamine-6-P]. This figure highlights the dramatic difference between the wild type GlmS and these mutant GlmS strains. Even fairly low levels of glucosamine-6-P significantly inhibit the native GlmS enzyme.</p>
    <p num="p-0323">When the wild type <i>Bacillus </i>glmS gene was over-expressed in <i>E. coli</i>, it led to a higher level glucosamine production than over-expression of the wild type <i>E. coli </i>glmS gene. Interestingly, the native <i>Bacillus </i>enzyme showed a product resistance very comparable to the <i>E. coli </i>mutant GlmS enzymes (<figref idrefs="DRAWINGS">FIG. 4</figref>). Activity of the <i>B. subtillus </i>enzyme was measured at 0, 2 and 4 mM glucosamine-6-P. The enzyme has a Km (fructose-6-phosphate) of 0.62 mM, and a Ki (glucosamine-6-P) of 1.25 mM (data not shown).</p>
    <p num="p-0324">A secondary plot of nonlinear regression values for Vmax versus [inhibitor] yielded a inhibition constant (Ki) of 0.56 mM for strain 2123-12. The Mutants 2123-54 and the <i>B. subtilis </i>enzyme have higher Ki values (Table 6). Measured Ki constants for several of these mutants are 4 to 8 times the value for the native enzyme in strain 2123-12. From shake flask studies it is apparent that decreased sensitivity to glucosamine-6-P allows higher glucosamine levels to accumulate. This strongly suggests that the intracellular [glucosamine-6-P] is fairly high (multi millimolar) in the recombinant <i>E. coli </i>strains, resulting in decreased glucosamine synthetase activity. Drastically reduced sensitivity to glucosamine-6-P inhibition offers the simplest explanation for the increased glucosamine synthesis in strains over-expressing the mutant <i>E. coli </i>GlmS and wild type <i>Bacillus </i>GlmS enzymes.</p>
    <p num="h-0022">Affinity to Substrates Fructose-6-P and Glutamine:</p>
    <p num="p-0325">Michaelis-Menten constants for glutamine and fructose-6-phosphate were determined using crude extracts (Table 6). With the wild-type <i>E. coli </i>GlmS enzyme (strain 2123-12), nonlinear regression yielded values of 0.20 mM (fructose-6-phosphate) and 0.17 mM (glutamine). Analogous experiments with a mutant GlmS*54 (strain 2123-54) yielded the values of 0.64 mM (fructose-6-phosphate) and 0.73 mM (glutamine). These values are slightly higher than those obtained with the native enzyme. The <i>Bacillus subtilis </i>GlmS enzyme expressed in <i>E. coli </i>(strains 7107-24) showed a Km (fructose-6-P) value very similar to the mutant <i>E. coli </i>enzyme GlmS*54.</p>
    <p num="p-0326"> <tables id="TABLE-US-00010" num="00010"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 6</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Characteristics of different GlmS enzymes</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="84pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="49pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Ki</td> <td class="description-td">Km</td> <td class="description-td">Km</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">(GlcN-6-P)</td> <td class="description-td">(fructose-6-P)</td> <td class="description-td">(glutamine)</td> </tr> <tr class="description-tr"> <td class="description-td">Enzyme sources</td> <td class="description-td">(mM)</td> <td class="description-td">(mM)</td> <td class="description-td">(mM)</td> </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> <tr class="description-tr"> <td class="description-td">WT <i>E. coli </i>GlmS</td> <td class="description-td">0.56</td> <td class="description-td">0.20</td> <td class="description-td">0.17</td> </tr> <tr class="description-tr"> <td class="description-td">Mutant <i>E. coli </i>GlmS*54</td> <td class="description-td">4.00</td> <td class="description-td">0.64</td> <td class="description-td">0.73</td> </tr> <tr class="description-tr"> <td class="description-td">Mutant <i>E. coli </i>GlmS*110</td> <td class="description-td">1.60</td> <td class="description-td">0.41</td> <td class="description-td"> ND*</td> </tr> <tr class="description-tr"> <td class="description-td">Mutant <i>E. coli </i>GlmS*124</td> <td class="description-td">3.50</td> <td class="description-td">1.10</td> <td class="description-td">ND</td> </tr> <tr class="description-tr"> <td class="description-td">Mutant <i>E. coli </i>GlmS*69</td> <td class="description-td">1.50</td> <td class="description-td">0.35</td> <td class="description-td">ND</td> </tr> <tr class="description-tr"> <td class="description-td">Mutant <i>E. coli </i>GlmS*72</td> <td class="description-td">1.40</td> <td class="description-td">0.95</td> <td class="description-td">ND</td> </tr> <tr class="description-tr"> <td class="description-td">Wild type Bacillus GlmS</td> <td class="description-td">1.25</td> <td class="description-td">0.62</td> <td class="description-td">ND</td> </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="left" id="FOO-00012" class="description-td" colspan="4">*ND: Not determined</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Thermal Stability:
</p>
    <p num="p-0327">Denaturation at 50° C. was used to measure possible differences in thermal stability of different GlmS enzymes. Crude extracts were incubated at 50° C. and sampled over a 90-minute period. The samples were assayed for glucosamine synthetase activity at 25° C. with saturating levels of all substrates.</p>
    <p num="p-0328">There does not appear to be any significant correlation between thermal stability at 50° C. and production of glucosamine (data not shown). Strain 2123-124 has the lowest stability but has been shown to be one of the best glucosamine production strains. There is little significant difference in thermal stability between strains 12, 54 and 59, yet from other experiments, the inventors know that strain 54 is a much better strain for glucosamine production.</p>
    <heading>Example 4</heading> <p num="p-0329">The following example describes a GlmS sequence analysis.</p>
    <p num="p-0330">Strains 2123-72 and 2123-103 were derived from the host strain 7101-17(DE3) by transformation and integration experiments with plasmids pK1N23-72 and pKLN23-103, respectively. The glmS regions (glmS*72 and glmS*103) in these plasmids were sequenced. The nucleotide sequence of the <i>E. coli </i>glmS*72 mutant coding sequence is listed as SEQ ID NO:13. The deduced amino acid sequence of the <i>E. coli </i>GlmS*72 protein is listed as SEQ ID NO:14.</p>
    <p num="p-0331">Interestingly, it turned out that both mutants have the same mutations that resulted in amino acid substitutions at positions 15, 387, 450 and 525 (Table 7). The residues at the relevant positions in the <i>E. coli </i>wild type GlmS (SEQ ID NO:2), mutant GlmS*49 (SEQ ID NO:4), GlmS*54 (SEQ ID NO:6) and GlmS* 124 (SEQ ID NO:8) are listed for comparison. The GlmS*72 has no common mutations, with respect to other product resistant GlmS mutants, except for the change of a serine to proline at position 450 which was also found in both GlmS*49 and GlmS*72. Interestingly, three GlmS mutant enzymes (GlmS*49, 72 and 124) have a mutation resulting in a change of one residue to proline in the region 450 to 469. GlmS*54 also has a residue change at position 472: a glycine replaced with a serine. These data suggest that changes in this region of the protein may play an important role in product resistance of the enzyme.</p>
    <p num="p-0332">
      <tables id="TABLE-US-00011" num="00011"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 7</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Base changes in the <i>E. coli </i>mutant</td>
              </tr> <tr class="description-tr"> <td class="description-td">glmS gene coding for a product resistant</td>
              </tr> <tr class="description-tr"> <td class="description-td">glucosamine synthase</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="42pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Position*</td>
                <td class="description-td">15</td>
                <td class="description-td">387</td>
                <td class="description-td">450</td>
                <td class="description-td">525</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">WT glmS</td>
                <td class="description-td">Glu (GAA)</td>
                <td class="description-td">Asp (GAT)</td>
                <td class="description-td">Ser (TCT)</td>
                <td class="description-td">Glu (GAA)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Mutant glmS*72</td>
                <td class="description-td">Lys (AAA)</td>
                <td class="description-td">Val (GTT)</td>
                <td class="description-td">Pro (CCT)</td>
                <td class="description-td">Gly (GGA)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00013" class="description-td" colspan="5">*For simplicity, positions are given according to the numbering of the deduced amino acid sequence.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0333">The nucleotide sequences of different bacterial glmS coding sequences were analyzed using the Megalign Program, J. Hein Method, Lasergene software (DNA Star, Inc, Madison, Wis.) with the standard settings. The amino acid sequences deduced from the nucleotide sequences were also compared using the Megalign Program, Lipman-Pearson Protein Alignment, Lasergene software (DNA Star) with the standard settings. Results are shown in Table 8.</p>
    <p num="p-0334">
      <tables id="TABLE-US-00012" num="00012"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="259pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 8</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Peptide size and homology of glucosamine synthase</td>
              </tr> <tr class="description-tr"> <td class="description-td">enzyme from different microorganisms</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="35pt" align="center"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="147pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Number of</td>
                <td class="description-td"> </td>
                <td class="description-td">Peptide Sequence Homology</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Amin</td>
                <td class="description-td">Peptide</td>
                <td class="description-td">(% identity)**</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="42pt" align="center"> </colspec> <colspec colname="7" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">GlmS</td>
                <td class="description-td">Acids*</td>
                <td class="description-td">Size (kDa)</td>
                <td class="description-td">
                  <i>E. coli</i>
                </td>
                <td class="description-td">
                  <i>B. subtilis</i>
                </td>
                <td class="description-td">
                  <i>S. cerevisiae</i>
                </td>
                <td class="description-td">
                  <i>C. albicans</i>
                </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>E. coli</i>
                </td>
                <td class="description-td">608</td>
                <td class="description-td">67</td>
                <td class="description-td"> </td>
                <td class="description-td">41</td>
                <td class="description-td">42</td>
                <td class="description-td">42</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(50)</td>
                <td class="description-td">(47)</td>
                <td class="description-td">(46)</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>B. subtilis</i>
                </td>
                <td class="description-td">599</td>
                <td class="description-td">65</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">36</td>
                <td class="description-td">36</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(45)</td>
                <td class="description-td">(45)</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>S. cerevisiae</i>
                </td>
                <td class="description-td">716</td>
                <td class="description-td">80</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(72)</td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <i>C. albicans</i>
                </td>
                <td class="description-td">712</td>
                <td class="description-td">79</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00014" class="description-td" colspan="7">*The numbers of amino acid residues does not include the initiator methionine, which is removed enzymatically after translation.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00015" class="description-td" colspan="7">**Homology at the nucleotide level is shown in parentheses.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0335">The <i>Bacillus subtilis </i>glmS gene encodes for a glucosamine synthase of 599 amino acid residues (SEQ ID NO:16) (excluding the initiator methionine, which is normally removed enzymatically after translation), 9 residues short than the <i>E. coli </i>homologue. The amino acid residues of the Bacillus GlmS protein that correspond to the positions where mutations were found in different product resistant <i>E. coli </i>GlmS mutants are listed in Table 9 for comparison. Interestingly, at six out of ten positions, the Bacillus enzyme has a different residue with respect to the <i>E. coli </i>wild type GlmS although none of the changes is the same as in the <i>E. coli </i>mutant enzymes.</p>
    <p num="p-0336">
      <tables id="TABLE-US-00013" num="00013"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="266pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 9</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Amino acid residue changes in produce resistant mutant</td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>E. coli </i>GlmS and wild type <i>B. subtilis </i>GlmS*</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="84pt" align="left"> </colspec> <colspec colname="1" colwidth="140pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">
                  <i>E. coli mutant GlmS</i>
                </td>
                <td class="description-td">
                  <i>B. subtilis</i>
                </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="28pt" align="center"> </colspec> <colspec colname="2" colwidth="56pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <colspec colname="7" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Position</td>
                <td class="description-td"> <i>E. coli </i>wt GlmS</td>
                <td class="description-td">GlmS*49</td>
                <td class="description-td">GlmS*54</td>
                <td class="description-td">GlmS*124</td>
                <td class="description-td">GlmS*72</td>
                <td class="description-td">wt GlmS**</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="56pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <colspec colname="7" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">4</td>
                <td class="description-td">Ile</td>
                <td class="description-td">Thr</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">15</td>
                <td class="description-td">Glu</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Lys</td>
              </tr> <tr class="description-tr"> <td class="description-td">39</td>
                <td class="description-td">Ala</td>
                <td class="description-td"> </td>
                <td class="description-td">Thr</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Gln</td>
              </tr> <tr class="description-tr"> <td class="description-td">250</td>
                <td class="description-td">Arg</td>
                <td class="description-td"> </td>
                <td class="description-td">Cys</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Pro</td>
              </tr> <tr class="description-tr"> <td class="description-td">272</td>
                <td class="description-td">Ile</td>
                <td class="description-td">Thr</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Tyr</td>
              </tr> <tr class="description-tr"> <td class="description-td">387</td>
                <td class="description-td">Asp</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Val</td>
              </tr> <tr class="description-tr"> <td class="description-td">450</td>
                <td class="description-td">Ser</td>
                <td class="description-td">Pro</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Pro</td>
                <td class="description-td">Asp</td>
              </tr> <tr class="description-tr"> <td class="description-td">469</td>
                <td class="description-td">Leu</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Pro</td>
                <td class="description-td"> </td>
                <td class="description-td">Phe</td>
              </tr> <tr class="description-tr"> <td class="description-td">472</td>
                <td class="description-td">Gly</td>
                <td class="description-td"> </td>
                <td class="description-td">Ser</td>
              </tr> <tr class="description-tr"> <td class="description-td">525</td>
                <td class="description-td">Glu</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Gly</td>
                <td class="description-td">His</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00016" class="description-td" colspan="7">*Only the residues that are different from the ones in the <i>E. coli </i>wild type GlmS protein are shown.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00017" class="description-td" colspan="7">**The positions for the Bacillus GlmS sequence are based on the alignment with the <i>E. coli </i>wild type GlmS sequence due to small gaps in the alignment.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 5</heading> <p num="p-0337">The following example demonstrates enzyme activities during glucosamine production in shake flask culture.</p>
    <p num="p-0338">Various enzyme activities relevant to glucosamine production were examined both in shake flasks and in fermentators. The involvement of these enzymes in the metabolism of glucose and the formation of N-glucosamine is outlined in <figref idrefs="DRAWINGS">FIG. 3</figref>. Glucose is taken up by the cell and is simultaneously converted to glucose-6-P. Glucose is metabolized by number of pathways, including those shown in the figure. In the glucosamine synthesis pathway, glucose-6-P is isomerized to fructose-6-phosphate, followed by the GlmS mediated conversion of fructose-6-phosphate to glucosamine-6-phosphate. Finally, glucosamine-6-phosphate is dephosphorylated and secreted. A major competing alternative route for glucose-6-phosphate is its entry into glycolysis via phosphofructokinase. Another important alternate routes for glucose-6-phosphate is its oxidation to gluconolactone-6-phosphate (the entry into the pentose phosphate pathway). Additionally, glucose-6-phosphate could be converted to glucose-1-phosphate, from which glycogen is made and stored in the cell.</p>
    <p num="p-0339">Results of enzyme analysis in a shake flask experiment using strain 2123-54 are shown in <figref idrefs="DRAWINGS">FIG. 6</figref>. Cells were grown in M9A medium and were induced with 0.2 mM IPTG from the start of the culture. Glucosamine synthase (GlmS) activity was high throughout the entire experiment. There was a high level of GlmS activity at 12 hours and that increased further by 24 hours. Afterwards, the GlmS activity appeared to decline. However, this decrease was not dramatic. Activity at 72 hours was still high, and was basically the same as what was observed at 12 hours. Thus, GlmS activity did not appear to decrease significantly during the course of the experiment.</p>
    <p num="p-0340">Phosphoglucoisomerase (Pgi) activity was high at 12 hours, and increased significantly by 24 hours. After this it returned to the level seen at 12 hours and remained at that level for the rest of the experiment. Clearly, formation of fructose-6-phosphate from glucose should not have been limited by low Pgi activity in these cells under this set of experimental conditions.</p>
    <p num="p-0341">The other major route for fructose-6-phosphate is glycolysis. The first committed step here is mediated by phosphofructokinase (Pfk). The highest Pfk activity was observed at 12 hours, and although it decreased over the next 48 hours, the activity remained detectable. This activity pattern is typical for Pfk. Activity is highest during exponential growth, and then decreases as cells enter stationary phase.</p>
    <p num="p-0342">Glucose-6-phosphate dehydrogenase (Glu6P DH) was detected in extracts. This enzyme feeds carbon from glucose into the pentose phosphate pathway for regeneration of NADPH. Activity of this enzyme was quite low relative to others measured, fairly stable, and decreased slightly after the first 24 hours of the experiment. Activity of phosphoglucomutase (Pgm) was found to be very low under the experimental conditions.</p>
    <heading>Example 6</heading> <p num="p-0343">The following Example describes the development of recombinant <i>E. coli </i>strains for lactose inducible glucosamine production.</p>
    <p num="p-0344">In order to develop a commercially viable process for glucosamine production, two factors were necessary. First was to increase glucosamine titer. The second was to eliminate the use of IPTG in the fermentation process, because the cost of IPTG causes glucosamine manufacture to be prohibitively expensive. Strains for glucosamine production that were developed during the early stage of the research program required IPTG in the culture medium to induce the expression of glucosamine synthase.</p>
    <p num="h-0026">A Strategy for Eliminating IPTG from the Fermentation Process</p>
    <p num="p-0345">The requirement for IPTG in the fermentation process stems from the mode of overexpression of the glmS gene, encoding GlcN6P-synthase, in the production strains such as 2123-54. The glmS gene was isolated from <i>E. coli </i>chromosomal DNA and cloned into an expression vector behind a promoter from bacteriophage T7. This promoter is very powerful and very specific. It is not recognized by <i>E. coli </i>RNA polymerase, but rather, by the T7 RNA polymerase. T7 RNA polymerase is provided in a genetic element designated DE3, which contains the gene for T7 RNA polymerase driven by the <i>E. coli </i>lac promoter and lac operator. It is the use of these promoter and operator that necessitates induction with IPTG. The lac promoter is subject to negative control by the lac repressor, encoded by the lacI gene. In the absence of inducer, the lac repressor binds to the lac operator and prevents expression of genes downstream from the operator, in this case the T7 RNA polymerase gene. In the absence of T7 RNA polymerase, the recombinant glmS gene is not expressed. In the presence of IPTG, the lac repressor will not bind to the operator with the result that the T7 RNA polymerase gene is expressed allowing overexpression of glmS.</p>
    <p num="p-0346">Another inducer of the lac promoter is allolactose. This is a byproduct of the action of b-galactosidase on lactose. Glucosamine production strains such as 2123-54 are negative for b-galactosidase due to the deletion and disruption of the lacZ gene. In the presence of a functional lacZ gene, however, lactose could be converted to allolactose, initiating the cascade of reactions described above that would lead to expression of glmS.</p>
    <p num="p-0347">In light of the above, there are several methods for eliminating dependence on IPTG. One approach illustrated in the present example is to restore the lacZ gene. This can be done by integrating the T7-glmS*54 expression cassette at a different site in the chromosome. Integration at the galK site would leave the lacZ gene intact. Lac<sup>+</sup> strains are potentially inducible by lactose, which is much less expensive than IPTG. The galK site was chosen because the integrant stains would also be Gal<sup>−</sup>. It was reported that in such strains galactose could be used as an inducer for lac promoters. Moreover, galactose generated through lactose hydrolysis could enhance lactose induction. This may enhance the induction when a sub-optimal amount of lactose was used in the glucosamine production process. In theory, when the T7-glmS*54 expression cassette is inserted at the galK site in strain 7101-17(DE3) the resulting strain would be analogous to strain 2123-54, but glucosamine production would be inducible with either lactose, galactose or IPTG.</p>
    <p num="h-0027">A General Protocol for Gene Integration or Deletion in Targeted Site on the Chromosome by Temperature Selection</p>
    <p num="p-0348">Vectors and methods were described by Hamilton et al. (1989<i>, J. Bacteriol</i>. 171:4617-4622) to make targeted gene deletion and gene integration in <i>E. coli </i>chromosome by temperature shift. The method was adapted to develop different glucosamine production <i>E. coli </i>strains. The protocols for gene integration include the following major steps. The first step is to clone the sequence of the target site and make an internal deletion and/or insert the foreign gene to be integrated at the deletion site. The second step is to subclone the fragment containing these sequences into a temperature sensitive integrative vector containing a temperature sensitive replication origin and an antibiotic selection marker. The third step is to transform the integrative vector into the <i>E. coli </i>host strain and select for clones with the entire plasmid integrated into the chromosome through single crossover recombination event under non-permissive temperature (42° C.). The fourth step is to grow the cells of selected clones in liquid culture at permissive temperature (30° C.). Cells with the integrated plasmid have a tendency to lose the plasmid. Cells that have lost the portion of the replication origin and antibiotic resistance gene or the entire plasmid will outgrow in the culture. Typically, this step was accomplished by inoculating a 50-ml LB medium with cells from a pool of two to ten clones and growing the culture for 24 hrs. The culture was passed to a fresh medium at a 1,000-fold dilution and grown for another period of 24 hrs. Fifth, cells were plated and clones that had lost the antibiotic resistance were selected. Gene specific selection procedures could be used, depending on the nature of integrated gene or deleted gene. Typically for screening clones, PCR was carried out using a primer set that could distinguish the clones with the intended change in the chromosome from its native form by the size of PCR products. Clones were confirmed by Southern Blot analysis using probes specific to the integrated or deleted DNA sequence.</p>
    <p num="h-0028">Development of Vectors for T7glmS*54 Integration at the galK Site by Temperature Selection</p>
    <p num="p-0349">A vector containing a portion of the <i>E. coli </i>gal operon sequence, the kanamycin resistance selection marker from plasmid pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.) and the temperature sensitive pSC101 replication origin from pMAK705 (Hamilton, et al., 1989<i>, J. Bacteriol</i>. 171:4617-4622) was developed for gene integration at the galK site using a temperature selection protocol. A portion of the <i>E. coli </i>gal operon sequence (3.3 kb) was amplified by PCR from <i>E. coli </i>strain W3110. The PCR product contained the sequence galTKM (starting at 14 bp upstream to the ATG start codon of the galT coding sequence, and ending at 68 bp following the stop codon of the galM coding sequence) and was cloned into vector pCRScript Amp SK(+), generating the recombinant plasmid pKLN23-157. A 0.7-kb deletion was made in the galK sequence (between the restriction sites Sƒo I and Mlu I) and unique restriction sites were added (Sal I, Bgl II, and Mcs I at the site of the deletion, creating plasmid pKLN07-1. A kanamycin resistance cassette was isolated as a Pst I fragment from pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.) by Pst I digestion and a treatment with T4 DNA Polymerase to produce blunt ends. The fragment was ligated into the Not I site (blunted with T4 DNA Polymerase) of the plasmid pKLN07-1 (orientation undetermined). The kan::galTKM fragment was removed from the plasmid as a BamH I/Sac II (ends blunted with T4 DNA Polymerase) fragment and ligated with a Pvu H/Sma I fragment containing the temperature sensitive replicon of pMAK705 (Hamilton, et al., 1989<i>, J. Bacteriol</i>. 171:4617-4622), creating plasmid pSW07-4. The expression cassette T7-glmS*54 was digested from the plasmid pKLN23-54 (disclosed in U.S. Pat. No. 6,372,457) as a Not I fragment and ends made blunt with T4 DNA Polymerase. The fragment was cloned into pSW07-4 at the Msc I site, generating plasmid pSW07-9.</p>
    <p num="h-0029">Selection for Lactose Inducible Strains</p>
    <p num="p-0350">Following transformation of the <i>E. coli </i>strain 7101-17(DE3) with pSW07-9, a protocol adapted from Hamilton et al. (1989) was used for temperature sensitive selection of integration mutants. For plasmids with the temperature sensitive replicon from pMAK705, plasmid replication can take place at 30° C., but plasmid integration is forced at non-permissive temperature (42° C.) under antibiotic selection. Incubating transformed cells at 42° C. thus selected for strains that had integrated the plasmid. This was done by plating cells on plates containing kanamycin, incubating plates at 42° C. and selecting colonies. Usually, the entire plasmid was integrated into the chromosome by homologous recombination. The single crossover event could take place in the galT or galM regions.</p>
    <p num="p-0351">Due to the replication origin, cells with the integrated plasmid have a tendency to loop out the plasmid from the chromosome. While cells with the entire plasmid integrated grow very poorly at 30° C., cells that have lost the plasmid or the portion of the replication origin display normal growth. Therefore, when cells selected at 42° C. were grown at 30° C., cells that have lost the plasmid outgrow the cells retaining the plasmid. In principal, there were two different ways for the cell to lose the plasmid through homologous recombination. They either lose the entire plasmid, resulting in a revertent strain with the native gal operon, or lose only the plasmid part containing the replication origin and selection marker, resulting in a strain with the T7-glmS*54 sequence integrated at the galK site. Cells from colonies isolated from 42° C. selection were incubated in liquid culture at 30° C., plated on plates containing no antibiotic. Since the integration resulted in the inactivation of the galK gene, integrant strains were unable to utilize galactose as the sole carbon sources. Galactose plates were used to screen for such integrant strains, which were then confirmed by Southern Blot hybridization using galK sequence as probe. These lactose-inducible glucosamine production strains were designated 7107-16 and 7107-18.</p>
    <heading>Example 7</heading> <p num="p-0352">This example describes lactose inducible glucosamine production.</p>
    <p num="p-0353">Strains with the T7-glmS*54 expression cassette integrated at the galK site (7107-16 and 7107-18) produce high levels of glucosamine after induction by either IPTG or lactose. The control strain (2123-54) under IPTG induction yielded 4.2 g/l. Glucosamine yields in lactose inducible strains were comparable to IPTG-induced 2123-54. Glucosamine levels in 2123-54 and a lactose inducible strain are shown in <figref idrefs="DRAWINGS">FIG. 7</figref>.</p>
    <p num="p-0354">Glucosamine synthase activity was assayed in samples of lactose induced cultures. Cells were grown in the M9A medium containing different amounts of glucose and/or lactose (numbers indicating grams per liter). Enzyme activity and glucosamine were assayed after 24-hr growth. As shown in Table 10, lactose induces GlmS activity and glucosamine synthesis. Lactose induction was affected by the amount of glucose in the medium. High levels of glucose showed a strong repression on lactose induction. Strain 7107-18 was selected for further development of the lactose induction protocol. The strain was evaluated under various lactose induction schemes in shake flasks and 1-liter fermentors.</p>
    <p num="p-0355"> <tables id="TABLE-US-00014" num="00014"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 10</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Glucosamine synthase activity and</td> </tr> <tr class="description-tr"> <td class="description-td">glucosamine production levels in lactose</td> </tr> <tr class="description-tr"> <td class="description-td">inducible strain 7107-16</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="center"> </colspec> <colspec colname="3" colwidth="49pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">GlmS activity<sup>2)</sup> </td> <td class="description-td">GlcN</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Growth Conditions<sup>1)</sup> </td> <td class="description-td">(μ mol min<sup>−1 </sup>mg<sup>−1</sup>)</td> <td class="description-td">(gl<sup>−1</sup>)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">30 glucose/0.2 mM IPTG</td> <td class="description-td">0.107</td> <td class="description-td">2.386</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">30 glucose/20 lactose</td> <td class="description-td">0.046</td> <td class="description-td">0.202</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">5 glucose/25 lactose</td> <td class="description-td">1.140</td> <td class="description-td">2.042</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">/40 lactose</td> <td class="description-td">0.840</td> <td class="description-td">2.810</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="left" id="FOO-00018" class="description-td" colspan="4">Note:</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="left" id="FOO-00019" class="description-td" colspan="4"> <sup>1)</sup>Cells were grown in the M9A medium containing different amounts of glucose and/or lactose (numbers indicating grams per liter).</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="3" align="left" id="FOO-00020" class="description-td" colspan="4"> <sup>2)</sup>Enzyme activity and glucosamine were assayed after 24-hr growth.</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Lactose Induction and Glucose Repression:
</p>
    <p num="p-0356">In a fermentation experiment, different lactose levels were tried, along with a feed of either lactose or glucose after lactose induction. Glucosamine level was monitored over a 72 hour period. The first protocol for lactose induction was similar to IPTG induction, in that lactose was added before inoculation (with no glucose, since glucose represses the lac operon), followed by glucose feeding to supply carbon for growth, GlcN formation and biomass maintenance. When cells were grown on 40 g l<sup>−1 </sup>lactose and fed continuously with a lactose feed, cells continued to consume lactose. This caused significant levels of galactose to accumulate, up to 40 g l<sup>−1</sup>. Glucosamine production level was similar to that of 2123-54 under IPTG induction (about 10 g l<sup>−1</sup>).</p>
    <p num="p-0357">Cells were grown on 40 g l<sup>−1 </sup>lactose for 24 hrs, and were then switched to a glucose feed. Under these conditions, cells stopped lactose utilization and galactose remained constant at 10 g l<sup>−1 </sup>throughout the remainder of the trial. Glucosamine production continued at a good rate, reaching a level comparable to that of strain 2123-54. This result indicates that glucosamine production can be supported strictly by glucose after lactose utilization is stopped. This also implies that induction requires only small amounts of lactose resulting in accumulation of only low levels of galactose. Reduced need for lactose and reduced accumulation of galactose are both desirable for cost reduction and product recovery.</p>
    <p num="p-0358">Induction schemes using increased lactose of 50 and 60 g/l, followed by glucose feeding showed continued lactose utilization for some time after the glucose addition. This indicates that repression by glucose was incomplete at higher levels of lactose, or that the induced enzyme continued to function at an adequate level to sustain glucosamine production following glucose addition.</p>
    <p num="p-0359">Since it was desirable to use lower levels of lactose, and the experiments showed that the glucose repression could be minimized at the lowest lactose level tested, even lower levels of lactose were tested. Further development work was focused on establishing the minimum level of lactose required for induction, the optimal timing and duration of induction, and the merit of an initial growth phase with glucose prior lactose induction. The latter might be especially important when higher cell densities are desired in controlled fermentation.</p>
    <p num="p-0360">An attempt was made to grow cells on glucose until cells were glucose depleted, followed by induction with lactose. Growth on glucose was allowed to reach a cell density of approximately 17 g l<sup>−1</sup>, and then a slow lactose feed was started. Under these conditions GLcN production reached 10 g l<sup>−1</sup>. This strategy was used in later experiments for GlcN production.</p>
    <p num="h-0031">Effect of Acetate:</p>
    <p num="p-0361">Acetate is a common by-product in <i>E. coli </i>fermentation. It is formed even under aerobic conditions when glucose is in excess, when growth rates exceed a critical level, or when the rates of glycolysis and oxidation of the metabolites formed are unbalanced due to the saturation of the respiration capacity. Addition of acetate to cultures showed a significant negative effect on both growth and glucosamine accumulation. Acetate was also shown to accumulate during glucosamine production. The level of trace elements affected the level of acetate formation (see below).</p>
    <p num="p-0362">Strategies to reduce acetate accumulation through process development included limited growth by slow glucose feeding, such that glucose did not become saturating. This strategy was employed throughout the program. Certain conditions could result in significant levels of acetate, such as high trace elements or potassium limitation. Low pH increased the inhibitory effect of acetate.</p>
    <p num="h-0032">Effect of Temperature:</p>
    <p num="p-0363">As disclosed in U.S. Pat. No. 6,372,457, the preferred temperature for growth and glucosamine production was 30° C. Higher growth temperature (37° C.) lead to higher growth rates, increased glucose uptake, and ultimately inhibitory acetate levels. Higher temperature can also lead to the formation of inclusion bodies that contain insoluble/inactive recombinant GlmS protein. Studies in flasks showed a significant decrease in acetate levels and a comparable GLcN level when the temperature was shifted from 30° C. to 25° C. after induction. Therefore, low temperature (25° C.) to reduce acetate accumulation was evaluated in fermentors. Results showed that at lower temperature, acetate accumulation was reduced, even under conditions of glucose accumulation.</p>
    <p num="h-0033">Effect of Trace Elements:</p>
    <p num="p-0364">In order to grow biomass to higher density, certain trace elements are required, especially iron, zinc, and manganese. An initial titration of the whole trace element package showed that biomass production was limited when the levels of trace elements were too low. It was then determined that iron was the main limiting growth factor, but if in excess it would lead to lower GLcN titers and higher acetate levels. Manganese had similar but weak effects on cell density and GLcN levels in flask culture. Manganese effects appeared to be additive to iron effects. The negative effect of manganese was confirmed in fermentors (<figref idrefs="DRAWINGS">FIG. 8</figref>). It was found that an adequate supply of iron was necessary in order for the cells to become adequately induced by lactose. Therefore, a critical concentration range had to be established to balance growth requirements, induction requirements, and the effects on acetate and GLcN production. The limitation of iron, manganese, and zinc to restrict carbon flow from glycolysis has precedence in the process improvement for citric acid fermentation.</p>
    <p num="p-0365">After several experiments, a final scheme was established: 3 mg l<sup>−1 </sup>iron sulfate in the medium and iron sulfate added to glucose feed at a ratio of 5 μg g<sup>−1 </sup>glucose. At an initial level of 3 mg iron sulfate in the medium, iron addition to the glucose feed solution was required to maintain a low level of biomass growth and extend GLcN production.</p>
    <p num="h-0034">Effect of Phosphate Concentration:</p>
    <p num="p-0366">Phosphorus is a necessary macronutrient for cells, mainly being used for nucleic acid synthesis, phospholipids, and coenzymes. Phosphorylated metabolic intermediates are also necessary for cellular metabolism. High phosphate concentration in the M9A medium serves also as a way to buffer the pH in the culture that is not readily controllable under flask conditions. However, scaling up of this medium to fermentation scale means that phosphate would be in great excess beyond normal biomass requirements. The high salt levels presented a problem for product recovery. Therefore, it was desired to reduce the phosphate level. Several trials with decreasing phosphate levels showed better growth, but much poorer GLcN production. For example, when lowering the potassium phosphate level from 30 g l<sup>−1 </sup>to 6 g l<sup>−1</sup>, growth was improved while GLcN production was greatly reduced (<figref idrefs="DRAWINGS">FIG. 9</figref>).</p>
    <heading>Example 8</heading> <p num="p-0367">This example describes the effects of pH on growth of the lactose-inducible glucosamine production strain 7107-18.</p>
    <p num="p-0368">As shown in Example 11, glucosamine is unstable at the regular pH range used for <i>E. coli </i>growth. Glucosamine also caused toxic effects on strain 7107-18. Toxicity was observed even when glucosamine at concentration as low as 20 g l<sup>−1 </sup>was pre-incubated in the medium (pH 7) for 3.5 hrs prior to cell inoculation. The toxicity was attributed to at least partially to GlcN degradation products in media with a starting pH of 7.0. GlcN is more stable at lower pH, glucosamine is not degraded at or below pH 4.7. However, low pH levels are known to limit cell growth. Therefore, experiments were conducted in which 7107-18 was grown at pH 7.0, 6.0, 5.5, 5.0 and 4.7. The main objectives were to study cell growth at these lower pHs (with GlcN), and to see if the previously observed cell death caused by GlcN degradation could be reduced at lower pHs.</p>
    <p num="p-0369">Batches of M9A medium with 40 g l<sup>−1 </sup>glucose and 20 g l<sup>−1 </sup>GlcN were adjusted to five different pHs. One set of flasks was immediately inoculated with 7107-18 cells. Replicate flasks of the same media were incubated without cells for 10 hours, before inoculation. Seven times over 48 hours, flasks were sampled to measure pH, OD<sub>600</sub>, and GlcN concentration. At each sample point, the pH of the culture was readjusted back to the initial settings.</p>
    <p num="p-0370">Without glucosamine pre-incubation, cell culture at pH 7.0 showed good growth even with 20 g l<sup>−1 </sup>glucosamine in the medium (<figref idrefs="DRAWINGS">FIG. 10</figref>). Between pH 7.0 and 4.7, the growth rate decreased as pH decreased. However, cells continued to show significant growth at pH 4.7.</p>
    <p num="p-0371">Similar to previous observations, cells did not grow well after the medium was pre-incubated with glucosamine at pH 7.0. Cell growth after inoculation into pre-incubated medium was observed. The growth was very poor across all pH levels tested. This appeared to result from a combination of the effects of lower pH and of glucosamine degradation on cell culture.</p>
    <p num="p-0372">Based on these experiments, it looked feasible to grow 7107-18 cells at pH levels lower than 7.0 in order to minimize product loss and toxic effects caused by glucosamine breakdown. However, the optimal pH and culture conditions must be adjusted carefully: lower pH levels could stabilize glucosamine but they also negatively affect cellular metabolism and cell growth. A GlcN production protocol in fermentors run at relatively low pH levels would certainly help preserve any GlcN which is made, protecting the cells in the process by reducing the concentration of breakdown products. These benefits must be balanced against the reduced metabolic activity of cells grown this way. Continued GlcN synthesis requires the constant generation of energy in the cells, and cells growing slowly at these lower pH levels did not appear capable of generating enough of the energy required.</p>
    <heading>Example 9</heading> <p num="p-0373">This example describes fermentation at lower pH to stabilize glucosamine.</p>
    <p num="p-0374">The normal optimal growth pH for <i>E. coli </i>is near neutral (pH 7.0), and a pH of 6.7 to 6.9 was used initially for GLcN fermentations. However, it was found that GLcN is subject to degradation in solution, especially neutral to alkaline pH. The organism is sensitive to pH, and does not grow well as pH decreases. Therefore, tests were conducted to determine the effects of low pH on glucosamine synthesis and accumulation following a growth phase. Effect of dissolved oxygen level was also tested, since glucosamine degradation was believed to be oxidative. Cells were grown at pH 6.7 to reach a high cell density. After the culture was induced, the pH was dropped from 6.7 to 5.5, and the level of dissolved oxygen was also dropped from 20% to 5% saturation. In this example, the best glucosamine accumulation was at low pH, and lower oxygen level also appeared to be beneficial (<figref idrefs="DRAWINGS">FIG. 11</figref>). At least some of the improvement was due to lower degradation, as continued imposition of the pH and oxygen variables showed much lower degradation at low pH in the culture after cessation of glucosamine accumulation at a low but sustaining glucose feed.</p>
    <heading>Example 10</heading> <p num="p-0375">This Example describes a summary of conditions used for the lactose-induced glucosamine fermentation process.</p>
    <p num="p-0376">Fermentation process was developed using recombinant <i>E. coli </i>strain 7107-18 for lactose induced production of glucosamine. The product titer was about 20 g l<sup>−1 </sup>glucosamine at 72 hrs. To those skilled in the art, the process can further be optimized based on the observations disclosed in the present invention. Methods developed for other fermentation processes may also be applied to the glucosamine fermentation process to enhance the performance. The major factors and elements of the process disclosed in the present invention are summarized below.</p>
    <p num="p-0377"> <tables id="TABLE-US-00015" num="00015"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="154pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain:</td> <td class="description-td">Recombinant <i>E. coli</i> </td> </tr> <tr class="description-tr"> <td class="description-td">Induction:</td> <td class="description-td">30 g/l lactose was added (as a 35% feed ramped</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">slowly over a 10 hour period) after a cell density of</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">10 g/l is reached. Glucose feed was suspended</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">during this procedure to prevent glucose</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">repression. After the lactose had all been added,</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">glucose feed was re-instated.</td> </tr> <tr class="description-tr"> <td class="description-td">Feed:</td> <td class="description-td">50% glucose with 5 μg FeSO<sub>4</sub>-7H<sub>2</sub>O/g glucose</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">and 0.33 μg MnSO<sub>4</sub>-H<sub>2</sub>O/g glucose,</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">glucose fed at limiting concentrations.</td> </tr> <tr class="description-tr"> <td class="description-td">Fermentation Time:</td> <td class="description-td">72 hours</td> </tr> <tr class="description-tr"> <td class="description-td">Fermentation Mode:</td> <td class="description-td">Fed Batch, with 50% glucose was added as required,</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">maintained limiting concentrations of glucose.</td> </tr> <tr class="description-tr"> <td class="description-td">Inoculum:</td> <td class="description-td">5% by volume</td> </tr> <tr class="description-tr"> <td class="description-td">pH:</td> <td class="description-td">6.9 during growth, then 6.7 after induction,</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">controlled by 10 N NH<sub>4</sub>OH</td> </tr> <tr class="description-tr"> <td class="description-td">Temperature:</td> <td class="description-td">30 C, switched to 25 C after induction</td> </tr> <tr class="description-tr"> <td class="description-td">Oxygen:</td> <td class="description-td">Dissolved O<sub>2 </sub>at 20% or greater, controlled by</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">agitation</td> </tr> <tr class="description-tr"> <td class="description-td">Aeration:</td> <td class="description-td">0.5 to 1 vvm</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="77pt" align="left"> </colspec> <colspec colname="3" colwidth="77pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Medium:</td> <td class="description-td">Component</td> <td class="description-td">Concentration</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="77pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="right"> </colspec> <colspec colname="4" colwidth="49pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">KH<sub>2</sub>PO<sub>4</sub> </td> <td class="description-td">14</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">K<sub>2</sub>HPO<sub>4</sub> </td> <td class="description-td">16</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Na<sub>3</sub>-citrate</td> <td class="description-td">1</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> </td> <td class="description-td">5</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CaCl<sub>2</sub>-H<sub>2</sub>O</td> <td class="description-td">0.05</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">MgSO<sub>4</sub>-7H<sub>2</sub>O</td> <td class="description-td">0.6</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">FeSO<sub>4</sub>-7H<sub>2</sub>O</td> <td class="description-td">3</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">ZnSO<sub>4</sub>-7H<sub>2</sub>O</td> <td class="description-td">3.8</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">MnSO<sub>4</sub>-H<sub>2</sub>O</td> <td class="description-td">0.33</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CuSO<sub>4</sub>-5H<sub>2</sub>O</td> <td class="description-td">0.1</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">NaMoO<sub>4</sub>-2H<sub>2</sub>O</td> <td class="description-td">0.1</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">H<sub>3</sub>BO<sub>3</sub> </td> <td class="description-td">0.1</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CoCl<sub>2</sub>-6H<sub>2</sub>O</td> <td class="description-td">0.1</td> <td class="description-td">mg l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Glucose</td> <td class="description-td">&gt;200</td> <td class="description-td">g l<sup>−1</sup>, as needed</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Mazu 204 defoamer</td> <td class="description-td">0.25</td> <td class="description-td">g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
All components were added before sterilization except glucose (added incrementally) and Fe, Zn, Mn, Cu, B, Mo, Co trace elements (added after sterilization).
</p>
    <heading>Example 11</heading> <p num="p-0378">This Example describes glucosamine and N-acetylglucosamine stability, and their effects on <i>E. coli. </i> </p>
    <p num="h-0039">Stability of Glucosamine:</p>
    <p num="p-0379">Glucosamine stability was tested in cell-free M9A medium (14 g l<sup>−1 </sup>K<sub>2</sub>HPO<sub>4</sub>, 16 g l<sup>−1 </sup>KH<sub>2</sub>PO<sub>4</sub>, 1 g l<sup>−1 </sup>Na<sub>3</sub>citrate.2H<sub>2</sub>O, 5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 10 mM MgSO<sub>4</sub>, 1 mM CaCl<sub>2</sub>, pH 7.0) supplemented with 40 g l<sup>−1 </sup>glucose. Glucosamine was prepared as a 30% glucosamine-HCl concentrated solution and was added to final concentrations of 0, 4, 13, 26 and 42 g l<sup>−1 </sup>(50 ml total in 250-ml flask). The pH of the medium was adjusted to pH 6.9. Flasks were placed on a shaker and agitated at about 225 rpm. Glucosamine levels and pH were monitored at 30° C. for about 24 hours. Glucosamine was found to be unstable and its degradation was concentration-dependent, with the degradation rates being higher at higher glucosamine concentrations. More than half of the glucosamine degraded in less than a day in medium with 42 g l<sup>−1 </sup>glucosamine. The degradation was only about 25% when the starting glucosamine concentration was 4 g l<sup>−1</sup>. Glucosamine degradation was accompanied by a decline of pH in the medium. In the 42 g l<sup>−1 </sup>glucosamine sample the pH in the medium was lowered by 0.7 units (from 6.9 to 6.2) after a 24-hr period, as compared to a pH change of only about 0.15 units in medium without glucosamine.</p>
    <p num="p-0380">The breakdown of glucosamine (60 g l<sup>−1</sup>) at four starting pH levels was monitored at 30° C. in cell-free M9A-glucose (40 g l<sup>−1</sup>) for 52 hours. At each sampling time, the solutions were readjusted to their original pH. Each condition was run in triplicate. As shown in <figref idrefs="DRAWINGS">FIG. 12</figref>, degradation was strongly pH-dependent, occurring faster at higher pH. The loss of glucosamine was 68% at pH 7.0 as compared to 18% at pH 5.5. Extrapolation of degradation rates versus pH suggests that no breakdown would occur below a pH of about 4.7.</p>
    <p num="p-0381">As glucosamine degrades, the solution developed a yellow-amber color. The extent of color formation can be estimated by absorption at 360-400 nm. Except for an early sampling period during which degradation rates were faster, there was a strong correlation between the amount of degraded glucosamine and the density of amber color in the medium. At the two lowest pH levels, there was a noticeable lag between the disappearance of glucosamine and the appearance of the color, indicating that the colored compound is not the direct degradation product of glucosamine. After the degradation rates slowed at all pH levels tested, the ratio of degraded glucosamine to yellow color was constant.</p>
    <p num="p-0382">Little is known about the molecular events involved in glucosamine degradation and color formation. There are reports about thermal degradation of glucosamine in water and under dry conditions (Shu, 1998, Journal of Agricultural and Food Chemistry, vol. 46, pp. 1129-1131; Chen and Ho, 1998, Journal of Agricultural and Food Chemistry, vol. 46, pp. 1971-1974, both incorporated herein by reference in their entirety). However, it is not known if the same types of chemical events take place under mild conditions used in the present studies. Although workers have observed formation of brown products from glucosamine and studied antioxidative activities of the brown products (Oyaizu and Mankoto, 1988, Nippon Shokuhin Kogyo Gakkaishi, vol. 35, pp. 771-775, incorporated herein by reference in its entirety), the brown products were not chemically identified in their studies. Considering the molecular structure of glucosamine, both the aldehyde and amino groups could be involved in degradation and/or polymerization. Color development indicates formation of conjugated double bands and complex structures.</p>
    <p num="p-0383">To test the involvement of the aldehyde group, degradation was monitored in M9A medium containing 30 g l<sup>−1 </sup>glucosamine with and without 40 g l<sup>−1 </sup>glucose. If the aldehyde group is involved, the presence of glucose would affect the rate of glucosamine degradation/polymerization. No significant difference in glucosamine breakdown was observed. This observation suggests to a role of the amino group in glucosamine degradation.</p>
    <p num="h-0040">Effects of Glucosamine on <i>E. coli </i>7107-18:</p>
    <p num="p-0384">Glucosamine is vital for <i>E. coli </i>growth since it is a precursor for the synthesis of important cell wall components. <i>E. coli </i>is also capable of using Glucosamine and N-acetylglucosamine as the sole carbon source. The catabolism of the amino sugars requires a functional glucosamine deaminase encoded by the nagB gene, which was deleted in strain 7107-18. As expected, 7107-18 could not grow on plates containing glucosamine as the sole carbon source. Experiments were carried out to investigate if glucosamine affects <i>E. coli </i>7107-18 cell growth and survival in medium containing 40 g l<sup>−1 </sup>glucose. Growth of freshly inoculated cultures was slightly inhibited by 10 and 20 g of glucosamine per liter. Glucosamine at 40 g per liter prevented cell growth and began to kill the cells after about 16 hours of incubation, as shown by cell plating. After 52 hrs, the viable count leveled off at about one-fifth the original number.</p>
    <p num="p-0385">More experiments were carried out to investigate the toxic effect of glucosamine on <i>E. coli </i>7107-18. Cells inoculated immediately into M9A-glucose (40 g l<sup>−1</sup>) containing 35 g l<sup>−1</sup>. glucosamine grew reasonably well, while cells died rapidly when inoculated into the identical medium which was pre-incubated for a period as short as 3.5 hours. This shows that the killing is largely due to glucosamine degradation product(s) formed in the medium before inoculation. It is not known why cells can survive and grow if inoculated immediately after the medium is made. One explanation could be that glucosamine is sequentially degraded to products A, B, C and D. Cells could have the ability to tolerate or assimilate earlier products at relatively low levels while the later products and/or higher levels of earlier products could be more toxic. This hypothesis is consistent with the observation that higher glucosamine concentrations and longer pre-incubation times resulted in lower levels of viable counts. With 35 g l<sup>−1 </sup>glucosamine, viable counts leveled off at 10<sup>5</sup>-fold less than inoculation levels. Cells died in the medium with 50 g l<sup>−1 </sup>glucosamine even without medium pre-incubation.</p>
    <p num="h-0041">Stability of N-Acetylylucosamine:</p>
    <p num="p-0386">It appears that the amino group plays an important role in degradation, as suggested by the effects of pH. If this is true, N-acetylglucosamine should be much more stable. In addition, the pH effects may not be as dramatic as they are with glucosamine. The stability of N-acetylglucosamine was tested in an experiment similar to the glucosamine degradation studies. The stability of 80 and 40 g l<sup>−1 </sup>N-acetyl glucosamine at pH 5.5 and 7.0 was monitored in medium M9A-without glucose over a period of two days. No significant degradation occurred (<figref idrefs="DRAWINGS">FIG. 13</figref>). This is in sharp contrast to the degradation seen with glucosamine. When glucosamine breaks down, an amber color develops in the medium. No such color formation was seen in M9A medium containing N-acetyl glucosamine over a period of 48 hours. Additionally, there was no significant drop in pH during incubation. The results confirm that the free amino group is the key functional group involved in glucosamine breakdown and/or polymerization.</p>
    <p num="h-0042">Effects of N-Acetyl Glucosamine on <i>E. coli </i>7107-18:</p>
    <p num="p-0387">Cells were inoculated into M9A-glucose (40 g/L) containing 62 g/L N-acetyl glucosamine. No significant growth inhibition was observed, even when the medium was pre-incubated for more than eight hours.</p>
    <p num="p-0388">In summary, N-acetylglucosamine does not have negative effects on <i>E. coli </i>strain 7107-18 and it is much more stable than glucosamine. Therefore, there is a potential advantage of producing N-acetylglucosamine instead of glucosamine. It is known that glucosamine is transported into the cells and phosphorylated through the mannose transporter whose subunits are encoded by the manXYZ genes and through the glucose transporter encoded by the ptsG gene. Although the manXYZ genes were deleted in the strain 7107-18, glucosamine uptake can still take place by the glucose transporter. With high concentrations in the medium, significant amounts of glucosamine could get into the cells. Due to the deletion of the genes encoding for the mannose transporter (manXYZ) and the N-acetyl glucosamine transporter (nagE) in 7107-18, N-acetyl glucosamine could accumulate to high levels within the medium without being transported back into the cell. This represents another possible advantage of producing N-acetyl glucosamine over glucosamine.</p>
    <heading>Example 12</heading> <p num="p-0389">This Example describes an HPLC method for glucosamine determination.</p>
    <p num="p-0390">It is desirable to develop a simple HPLC method to chromatographically quantitate glucosamine. Characteristics of the desired method should include minimal sample preparation and reasonably accurate determination of glucosamine. The method presented here is based on that described by Way et al. (J. Liq. Chromatography &amp; Related Technol. 23:2861. 2000). Modification of the mobile phase in Way et al.'s procedure allowed for the resolution of the glucosamine peak from other peaks observed in shake flask cell culture samples.</p>
    <p num="p-0391">
      <tables id="TABLE-US-00016" num="00016"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Method description:</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="49pt" align="left"> </colspec> <colspec colname="2" colwidth="168pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Column:</td>
                <td class="description-td">Phenomenex Prodigy ODS(3) C<sub>18 </sub>− 5μ, 150 × 4.6 mm</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">(Phenomenex, Torrance, CA)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Mobile phase:</td>
                <td class="description-td">MeOH: aqueous buffer (1:4 v/v). The aqueous buffer</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">consisted of 10 mM sodium acetate and 10 mM sodium</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">octanesulfonate, pH 5.1. The complete mobile</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">phase was prepared as follows. To 1 liter of</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">deionized water, 0.8 g sodium acetate and 2.16 g sodium</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">octanesulfonate (Sigma-ultrapure) was added. After</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">dissolving the salts, pH was adjusted</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">to 5.1 +/− 0.1 using glacial acetic acid. To this</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1-liter solution 250 ml methanol was added and the</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">solution degassed. Reservoir and column were kept</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">at room temperature.</td>
              </tr> <tr class="description-tr"> <td class="description-td">Flow rate:</td>
                <td class="description-td">0.7 ml/minute</td>
              </tr> <tr class="description-tr"> <td class="description-td">Detector:</td>
                <td class="description-td">refractive index detector at 30° C.</td>
              </tr> <tr class="description-tr"> <td class="description-td">Sample:</td>
                <td class="description-td">10 μl in M9A medium</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0392">Injection of the sterile M9A growth medium resulted in two major peaks on a chromatogram. Additionally, the chromatogram is essentially unchanged if M9A lacking glucose, calcium and magnesium salts was analyzed. Injection of glucosamine in M9A resulted in a single additional distinct peak. Under these conditions glucosamine eluted at about 13 minutes with a very large negative peak immediately following it. Additionally, it was unsuccessful to increase resolution by varying pH, ionic strength, methanol (MeOH) concentration, or octanesulfonate concentration. This negative deflection always follows the glucosamine peak. The usual way to remove such a negative deflection is to dilute the sample in mobile phase. However, even diluting samples twenty fold did not completely remove the negative deflection, and such high dilution is not practical for samples from shake flasks or fermentors.</p>
    <p num="p-0393">It was found that integrating using peak height instead of peak area allowed for fairly accurate quantitation of glucosamine in the approximate range of 500-10,000 parts per million (ppm). To determine the range it was necessary to prepare the standards in M9A rather than in water or mobile phase. Thus, all samples were prepared in the M9A growth medium and any dilutions were made using M9A as well. For each sample the running time was 20 min.</p>
    <p num="h-0044">Method Validation:</p>
    <p num="p-0394">First, several standards of glucosamine in M9A were prepared. Repeated injections gave fairly accurate (+/−5%) results.</p>
    <p num="p-0395">Second, shake flask samples were analyzed by both calorimetric assay and HPLC. Values obtained were generally within 10% of the colorimetric assay. Close agreement between HPLC and the colorimetric method should not necessarily be expected due to the standard deviation inherent to the calorimetric assay. For example, two samples (both 10,000 ppm) assayed by the calorimetric method gave values of 8.9 and 9.3 g l<sup>−1</sup>.</p>
    <p num="p-0396">Third, shake flask samples were spiked with known amounts of glucosamine and assayed. The undiluted shake flask sample produced a value of 1479 ppm. After dilution with an equal volume of a 5000-ppm glucosamine standard, the diluted flask sample produced a value of 3440 (expected 3240). Part of the discrepancy from the expected value is most likely contributed by M9A. In fact M9A alone will give a false glucosamine “value” of around 100 PPM.</p>
    <p num="p-0397">Shake flask culture samples were assayed by both the HPLC method and the calorimetric methods. Results are shown in Table 11. The culture supernatants were obtained by centrifugation and filtration to remove particles, and stored at −20° C. until analysis. HPLC was calibrated using a single point 2500 ppm standard solution prepared in M9A medium. After thawing, samples were immediately analyzed by HPLC. After remaining in the autosampler overnight, the samples were analyzed again. Agreement was very good between the two methods. Samples stored at −20° C. for several weeks showed no decrease in glucosamine concentration. Keeping filtered samples at room temperature overnight did not result in any decrease in glucosamine concentration.</p>
    <p num="p-0398">
      <tables id="TABLE-US-00017" num="00017"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 11</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Comparison of the HPLC method and</td>
              </tr> <tr class="description-tr"> <td class="description-td">colorimetric method for glucosamine</td>
              </tr> <tr class="description-tr"> <td class="description-td">quantification*</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="35pt" align="center"> </colspec> <colspec colname="2" colwidth="70pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="63pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Colorimetric</td>
                <td class="description-td">HPLC</td>
                <td class="description-td">HPLC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Sample ID</td>
                <td class="description-td">Method</td>
                <td class="description-td">(0 hours)</td>
                <td class="description-td">(24 hours)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">D1</td>
                <td class="description-td">4505</td>
                <td class="description-td">4900</td>
                <td class="description-td">4963</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">D2</td>
                <td class="description-td">4138</td>
                <td class="description-td">4641</td>
                <td class="description-td">4618</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">E2</td>
                <td class="description-td"> 451</td>
                <td class="description-td"> 533</td>
                <td class="description-td"> 589</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">F1</td>
                <td class="description-td">1643</td>
                <td class="description-td">1856</td>
                <td class="description-td">1815</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">F2</td>
                <td class="description-td">1938</td>
                <td class="description-td">1875</td>
                <td class="description-td">1968</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H1</td>
                <td class="description-td">1380</td>
                <td class="description-td">1365</td>
                <td class="description-td">1439</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H2</td>
                <td class="description-td">1341</td>
                <td class="description-td">1379</td>
                <td class="description-td">1548</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="left" id="FOO-00021" class="description-td" colspan="5">*Culture samples were centrifuged and filtered to remove particles, and stored at −20° C. until the assays. HPLC analysis was run with the thawed samples (0 hour) and repeated after the samples were left overnight in the autosampler at room temperature (24 hours). Glucosamine concentrations were shown in PPM.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 13</heading> <p num="p-0399">This Example describes over-expression of glucosamine-6-phosphate N-Acetyltransferase 1 genes (GNA1) for N-acetylglucosamine production.</p>
    <p num="p-0400">The following Example describes the cloning and over-expression of different glucosamine-6-phosphate N-Acetyltransferase 1 genes (GNA1) in recombinant <i>E. coli </i>for increased synthesis of N-acetylglucosamine. The feasibility of the strategy was demonstrated with pET-based expression vectors containing the GNA1 genes from yeast <i>Saccharomyces cerevisiae </i>(ScGNA1), yeast <i>Candida albicans </i>(CaGNA1) and higher plant <i>Arabidopsis thaliana </i>(AtGNA1).</p>
    <p num="p-0401">GNA1 cloning and expression were carried out by cloning the coding sequence of the GNA1 gene behind a T7lac promoter in a pET vector and transforming the recombinant pET plasmids into lactose inducible glucosamine production strain <i>E. coli </i>7107-18. Functional expression of the gene was determined by SDS-PAGE and enzyme activity assay. Synthesis of N-acetylglucosamine was monitored in shake flask experiments.</p>
    <p num="h-0046">Cloning the <i>S. cerevisiae </i>GNA1 Gene for Over-Expression in <i>E. coli: </i> </p>
    <p num="p-0402">For cloning and expression of the <i>S. cerevisiae </i>GNA1 gene (ScGNA1), primers were synthesized based on the published sequence of the GNA1 gene (Murakami et al., 1995, Nat. Genet. 10, pp. 261-268, herein by reference in its entirety). Primers were used to amplify the GNA1 coding sequence from genomic DNA isolated from <i>S. cerevisiae </i>strain S288C using polymerase chain reaction (PCR). The primers used for amplification were forward primer 07-83 and reverse primer 07-84 and had the following sequences: 07-83: 5′-GATCGGTCTCGCATGAGCTTACCCGATGGATTTTATATAAGGC-3′ (SEQ ID NO:35); 07-84: 5′-GATCCTCGAGCTATTTTCTAATTTGCATTTCCACGCCTGC-3′ (SEQ ID NO:36). Primer 07-83 contains a Bsa I restriction endonuclease site (GGTCTC, represented in nucleotides 5-10 of SEQ ID NO:35) followed by 31 nucleotides of the GNA1 coding sequence starting from its ATG start codon (represented in nucleotides 13-43 of SEQ ID NO:36). Primer 07-84 contains a Xho I restriction endonuclease site (CTCGAG, represented in nucleotides 5-10 of SEQ ID NO:36) followed by 30 nucleotides of the GNA1 coding sequence starting at its translation stop codon (represented in nucleotides 11-40 of SEQ ID NO:36). PCR amplification was conducted using a standard protocol to generate a fragment of DNA containing the entire ScGNA1 coding sequence flanked by the Bsa I and Xho I sites.</p>
    <p num="p-0403">The PCR product containing the GNA1 sequence was cloned into vector pCR-Script Amp SK(+) (Stratagene, LaJolla, Calif.), generating plasmid pSW07-60. The ScGNA1 fragment was isolated from the plasmid PSW07-60 by Bsa I and Xho I digestion and cloned at the Nco I and Xho I sites of the expression vector pET24d(+) (Novagen, Inc., Madison, Wis., creating plasmid SW07-60. Cloning in this manner places the ScGNA1 sequence behind the T7-lac promoter of pET24d(+), generating an expression cassette of T7-lac-ScGNA1.</p>
    <p num="h-0047">Cloning the <i>C. albicans </i>GNA1 Gene for Over-Expression in <i>E. coli: </i> </p>
    <p num="p-0404">For cloning and expression of <i>C. albicans </i>GNA1 (CaGNA1), primers were synthesized based on published sequence of the GNA1 (Mio et al., 1999, J. Biol. Chem. 274, pp.424-429, herein by reference in its entirety). Primers 07-92 and 07-93 were used to amplify the GNA1 coding sequence from <i>Candida albicans </i>ATCC10261 genomic DNA using PCR. Forward primer 07-92 and reverse primer 07-93 had the following sequences: 07-92: 5′-GATCGGTCTCGCATGATGTTACCACAAGGTTATAC-3′ (SEQ ID NO:37) and 07-93: 5′-GATCCTCGAGCTAGAATCTACATACCATTTCAAC-3′ (SEQ ID NO:38). Primer 07-92 contains a Bsa I restriction endonuclease site (GGTCTC, represented in nucleotides 5-10 of SEQ ID NO:37) followed by 23 nucleotides of the GNA1 coding sequence starting from its ATG start codon (represented in nucleotides 13-35 of SEQ ID NO:37). Primer 07-93 contains a Xho I restriction endonuclease site (CTCGAG, represented in nucleotides 5-10 of SEQ ID NO:38) followed by 24 nucleotides of the GNA1 coding sequence starting at its translation stop codon (represented in nucleotides 11-34 of SEQ ID NO:38). PCR amplification was conducted under standard conditions to generate a fragment of DNA containing the entire CaGNA1 coding sequence flanked by the Bsa I and Xho I sites.</p>
    <p num="p-0405">The PCR product containing the CaGNA1 sequence was ligated into the SrfI site of vector pCR-Script Amp SK(+) (Stratagene, LaJolla, Calif.), generating plasmid pKLN07-33. The CaGNA1 fragment was isolated from plasmid pKLN07-33 with restriction enzymes Bsa I and Xho I and cloned at the Nco I and Xho I sites of the expression vector pET24d(+) (Novagen, Inc., Madison, Wis.), creating plasmids pKLN07-34 and pKLN07-35. Cloning in this manner places the CaGNA1 sequence behind the T7lac promoter of pET24d(+), generating an expression cassette of T7lac-CaGNA1.</p>
    <p num="h-0048">Cloning the Arabidopsis GNA1 Gene for Over-Expression in <i>E. coli: </i> </p>
    <p num="p-0406">For cloning and expression of <i>Arabidopsis thaliana </i>GNA1 (AtGNA1), primers 07-94 and 07-95 were synthesized based on published sequence of the GNA1 (Genebank AL391144). The primers were used to amplify the GNA1 coding sequence from BAC clone F14F8 (Arabidopsis Biological Resource Center DNA Stock Center, Columbus, Ohio) using PCR. Forward primer 07-94 and reverse primer 07-95 had the following sequences: 07-94: 5′-GATGGTCTCGCATGGCTGAGACATTCAAGATC-3′ (SEQ ID NO:39), and 07-95: 5′-GATCCTCGAGTTAATCGAAGTACTTAGACATTTGAATC-3′ (SEQ ID NO:40). Primer 07-94 contains a Bsa I restriction endonuclease site (GGTCTC, represented in nucleotides 4-9 of SEQ ID NO:39) followed by 21 nucleotides of the GNA1 coding sequence starting from its ATG start codon (represented in nucleotides 12-32 of SEQ ID NO:39). Primer 07-95 contains a Xho I site (CTCGAG, represented in nucleotides 5-10 of SEQ ID NO:40) followed by 24 nucleotides of the GNA1 coding sequence starting at its translation stop codon (represented in nucleotides 11-38 of SEQ ID NO:40). PCR amplification was conducted using a standard protocol to generate a fragment of DNA containing the entire GNA1 coding sequence flanked by the Bsa I and Xho I sites. The PCR fragment was digested with restriction endonucleases Bsa I and Xho I and cloned at the Nco I and Xho I sites of the expression vector pET24d(+) (Novagen, Inc., Madison, Wis.) creating plasmid pSW07-70. Cloning in this manner places the AtGNA1 sequence behind the T7lac promoter of pET24d(+), generating an expression cassette of T7lac-AtGNA1.</p>
    <p num="h-0049">Functional Expression of Different Recombinant GNA1 Genes and N-Acetylglucosamine Production in <i>E. coli </i> </p>
    <p num="p-0407">The recombinant plasmids pSW07-62 (containing the <i>S. cerevisiae </i>GNA1 gene), pKLN07-34 (containing the <i>C. albicans </i>GNA1 gene), and pSW07-70 (containing the <i>A. thaliana </i>GNA1 gene) were transformed into <i>E. coli </i>strain 7107-18, generating strains 7107-87, 7107-117 and 7107-93, respectively. Control strain 7107-88 was prepared by transforming the empty vector into the same host. Following a standard protocol, cell cultures of transformants were grown in LB medium and induced with 1 mM IPTG. Samples were taken from induced cultures for SDS-PAGE analysis to confirm GNA1 protein overexpression. The predicted protein sizes of the AtGNA1, CaGNA1 and ScGNA1 are 17 kDa, 16.9 kDa, and 18.1 kDa, respectively. Overexpressed proteins of the predicted sizes were seen in samples from the induced cultures expressing the various GNA1 genes. In all cases, the overexpressed protein appeared to be very soluble. As expected, no overexpressed protein near the predicted size of the GNA1 gene was seen in control strain 7107-88.</p>
    <p num="p-0408">Screening of the different <i>E. coli </i>strains was conducted to confirm functional expression of the recombinant GNA1 genes. Strains hosting the <i>S. cerevisiae, C. albicans </i>and <i>A. thaliana </i>GNA1 expression vectors were grown in M9B production medium [6 g l<sup>−1 </sup>KH<sub>2</sub>PO<sub>4</sub>, 24 g l<sup>−1 </sup>K<sub>2</sub>HPO<sub>4</sub>, 1 g l<sup>−1 </sup>Na<sub>3</sub>Citrate-2H<sub>2</sub>O, 10 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>(phosphate adjusted to pH 7.4) plus trace metals (0.2 mg l<sup>−1 </sup>FeSO<sub>4</sub>-7H<sub>2</sub>O, 0.015 mg l<sup>−1 </sup>ZnSO<sub>4</sub>-7H<sub>2</sub>O, 0.015 mg l<sup>−1 </sup>MnSO<sub>4</sub>—H<sub>2</sub>O, 0.001 mg l<sup>−1 </sup>CuSO4-5H<sub>2</sub>O, 0.001 mg l<sup>−1 </sup>NaMoO<sub>4</sub>-2H<sub>2</sub>O, 0.001 mg l<sup>−1 </sup>H<sub>3</sub>BO<sub>3</sub>, and 0.001 mg l<sup>−1 </sup>COCl<sub>2</sub>-6H<sub>2</sub>O) supplemented with 40 g l<sup>−1 </sup>glucose, 10 g l<sup>−1 </sup>ribose, 5 g l<sup>−1 </sup>yeast extract, 0.6 g l<sup>−1 </sup>MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g l<sup>−1 </sup>CaCl<sub>2</sub>-2H<sub>2</sub>O, 25 mg l<sup>−1 </sup>kanamycin, and 0.2 mM IPTG. At 24 hours, glucose was added to 30 g l<sup>−1 </sup>per day total based on HPLC results, and 5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added to flasks where levels had fallen below 1 g l<sup>−1</sup>. Duplicate flasks were made of each strain so that one flask could be harvested at 24 hours and the other at 48 hours for enzyme analysis and determination of N-acetylglucosamine and acetate levels.</p>
    <p num="p-0409">Glucosamine synthase (GlmS) activity was assayed, and all strains exhibited good activity levels. Samples were also assayed for glucosamine-6-P N-acetyltransferase (GNA1) enzyme activity following the method described by Mio et al. (Journal of Biological Chemistry, 1999, 274, pp. 424-429 (Table 12). As expected, the control strain did not show a significant level of GNA1 activity. Expression of the yeast and higher plant GNA1 genes led to high acetyltransferase activity. These GNA1 transformants also synthesized N-acetylglucosamine at very high levels (Tables 12 and 13). Acetyltransferase activities in strains expressing the GNA1 genes from <i>S. cerevisiae </i>(7107-87) and <i>C. albicans </i>(7107-117) were comparable. However, N-acetylglucosamine production was four times higher in strains expressing the <i>S. cerevisiae </i>gene. Although GNA1 enzyme activity in strains with the <i>A. thaliana </i>gene was lower than the strains with the <i>C. albicans </i>gene, N-acetylglucosamine production was higher in the former than in the latter. Clearly, there is no simple correlation between GNA1 activity levels and N-acetylglucosamine production; the GNA1 enzymes from various sources may have differences in enzyme characteristics. The data demonstrate the utility of different GNA1 genes in metabolic engineering to produce N-acetylglucosamine. Among the three GNA1 genes tested, the <i>S. cerevisiae </i>GNA1 outperformed the others in terms of N-acetylglucosamine production. Therefore, the <i>S. cerevisiae </i>GNA1 gene was selected for later research work disclosed in the present invention.</p>
    <p num="p-0410">
      <tables id="TABLE-US-00018" num="00018"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 12</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Glucosamine synthase and acetyltransferase</td>
              </tr> <tr class="description-tr"> <td class="description-td">activities in stains expressing three</td>
              </tr> <tr class="description-tr"> <td class="description-td">different GNA1 genes.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="119pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="center"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Enzyme activity</td>
                <td class="description-td">GlcNAc</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="49pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Construct</td>
                <td class="description-td">Strain</td>
                <td class="description-td">GlmS</td>
                <td class="description-td">GNA1</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="49pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="35pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Vector</td>
                <td class="description-td">7107-88</td>
                <td class="description-td">0.53</td>
                <td class="description-td">0.06</td>
                <td class="description-td">0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>S. cerevisiae </i>GNA1</td>
                <td class="description-td">7107-87(25)</td>
                <td class="description-td">0.48</td>
                <td class="description-td">19.0</td>
                <td class="description-td">24.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>C. albicans </i>GNA1</td>
                <td class="description-td">7107-117(1)</td>
                <td class="description-td">0.37</td>
                <td class="description-td">21.7</td>
                <td class="description-td">6.3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">7107-117(2)</td>
                <td class="description-td">0.42</td>
                <td class="description-td">19.9</td>
                <td class="description-td">6.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">7107-117(3)</td>
                <td class="description-td">0.29</td>
                <td class="description-td">25.2</td>
                <td class="description-td">6.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>A. thaliana </i>GNA1</td>
                <td class="description-td">7107-93(1)</td>
                <td class="description-td">0.50</td>
                <td class="description-td">6.3</td>
                <td class="description-td">13.3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">7107-93(2)</td>
                <td class="description-td">0.14</td>
                <td class="description-td">5.6</td>
                <td class="description-td">10.7</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00022" class="description-td" colspan="5">(1)Enzyme activity is expressed in μmol min<sup>−1 </sup>mg<sup>−1 </sup>protein.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00023" class="description-td" colspan="5">(2)N-acetylglucosamine levels were determined in samples taken at 23-hr time point.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00024" class="description-td" colspan="5">(3)Numbers in the parentheses indicate different siblings.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0411">Strain 7107-18 produces high levels of glucosamine that can be detected by HPLC. However, little or no free glucosamine (below 0.5 g l<sup>−1</sup>) could be detected in strains that were over-expressing the acetyltransferase gene. This clearly indicated no significant buildup of the intermediate glucosamine-6-P in the GNA1 transformants, confirming that enzyme GNA1 was the main driving force for high level production of N-acetylglucosamine.</p>
    <p num="p-0412">Accumulation of acetate is a recognized obstacle to achieving high levels of recombinant protein and other fermentation products in <i>E. coli</i>. With excess glucose in the medium, <i>E. coli </i>cells tend to synthesize high levels of acetate and other organic acids, usually resulting in growth inhibition. Acetate production has been a problem in glucosamine production <i>E. coli </i>strains. However, N-acetylglucosamine production strains accumulated little or no acetate at the 23-hour timepoint under conditions where the control strain accumulated multi-gram levels of acetate. The synthesis of N-acetylglucosamine consumes acetyl-CoA, the precursor for acetate formation. Although the use of acetyl-CoA will be a metabolic burden imposed on the cell, the re-direction of acetyl-Co A to N-acetylglucosamine production apparently represents a significant benefit by avoiding acetate accumulation. It is important to note that the N-acetylglucosamine producing strains showed higher cell densities than the control strains (Table 13).</p>
    <p num="p-0413">
      <tables id="TABLE-US-00019" num="00019"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="266pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 13</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Cell growth and N-acetylglucosamine production in <i>E. coli </i>strains</td>
              </tr> <tr class="description-tr"> <td class="description-td">transformed with different GNA1 expression constructs.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="98pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="56pt" align="center"> </colspec> <colspec colname="3" colwidth="56pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Growth</td>
                <td class="description-td">Acetate</td>
                <td class="description-td">GlcNAc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">(OD<sub>600</sub>)</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <colspec colname="7" colwidth="28pt" align="center"> </colspec> <colspec colname="8" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain</td>
                <td class="description-td">Construct</td>
                <td class="description-td">23 hrs</td>
                <td class="description-td">48 hrs</td>
                <td class="description-td">23 hrs</td>
                <td class="description-td">48 hrs</td>
                <td class="description-td">23 hrs</td>
                <td class="description-td">48 hrs</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="28pt" align="center"> </colspec> <colspec colname="8" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">7107-88</td>
                <td class="description-td">Vector</td>
                <td class="description-td">3.75</td>
                <td class="description-td">4.2</td>
                <td class="description-td">4.4</td>
                <td class="description-td">5.2</td>
                <td class="description-td">ND*</td>
                <td class="description-td">ND</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-87</td>
                <td class="description-td"> <i>S. cerevisiae </i>GNA1</td>
                <td class="description-td">7.80</td>
                <td class="description-td">12.0</td>
                <td class="description-td">ND</td>
                <td class="description-td">ND</td>
                <td class="description-td">11.7</td>
                <td class="description-td">24.6</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-117</td>
                <td class="description-td"> <i>C. albicans </i>GNA1</td>
                <td class="description-td">10.00</td>
                <td class="description-td">13.8</td>
                <td class="description-td">0.5</td>
                <td class="description-td">5.1</td>
                <td class="description-td">5.1</td>
                <td class="description-td">6.1</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-93</td>
                <td class="description-td"> <i>A. thaliana </i>GNA1</td>
                <td class="description-td">8.70</td>
                <td class="description-td">13.2</td>
                <td class="description-td">ND</td>
                <td class="description-td">4.0</td>
                <td class="description-td">8.0</td>
                <td class="description-td">12.0</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0414">Sequence Analysis of Different GNA1 Enzymes:</p>
    <p num="p-0415">Significant differences were observed in terms of specific activity and NAG production in <i>E. coli </i>strains overexpressing the various GNA1 genes. However, since all enzymes catalyze the same reaction, homology would be expected between the various GNA1 at nucleotide and protein levels. SEQ ID NO:29 contains the coding sequence of the <i>S. cerevisiae </i>GNA1 gene. SEQ ID NO:29 encodes the <i>S. cerevisiae </i>GNA1 amino acid sequence represented here by SEQ ID NO:30. SEQ ID NO:33 contains the coding sequence of the <i>A. thaliana </i>GNA1 gene. SEQ ID NO:33 encodes the <i>A. thaliana </i>GNA1 amino acid sequence represented here by SEQ ID NO:34. SEQ ID NO:31 contains the coding sequence of the <i>C. albicans </i>GNA1 gene. SEQ ID NO:31 encodes the <i>C. albicans </i>GNA1 amino acid sequence represented here by SEQ ID NO:32. When aligned by the J. Hein DNA Alignment method (DNAStar reference), nucleotide sequences exhibited significant homology. The ScGNA1 and At-GNA1 coding sequences share 49.7% identity, while the ScGNA1 and CaGNA1 coding sequences share 53.1% identity. The CaGNA1 and AtGNA1 coding sequences share 47.2% identity.</p>
    <p num="p-0416">Translation of the coding sequences into amino acid sequences revealed significant homology among the various GNA1 proteins when aligned by the Lipman-Pearson Protein Alignment method (DNAStar, Inc., Madison, Wis.). As seen on Table 14, the ScGNA1 sequence (SEQ ID NO:30) shares 44% identity with the CaGNA1 sequence (SEQ ID NO:32) and 38.9% identity with the AtGNA1 (SEQ ID NO:34) sequence. Some regions appeared to be more highly conserved; for example, amino acids GHIED were conserved in all sequences (aligning with amino acids 96-100 of the ScGNA1 sequence (SEQ ID NO:30)). Also, a 20 residue region corresponding to ScGNA1 residues 129-148 of SEQ ID NO:30 was highly conserved. This region had 75% identity to the corresponding region in CaGNA1 sequence and 70% identity to the corresponding region in AtGNA1 sequence.</p>
    <p num="p-0417">
      <tables id="TABLE-US-00020" num="00020"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 14</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Peptide size and homology of</td>
              </tr> <tr class="description-tr"> <td class="description-td">the ScGNA1, AtGNA1, and CaGNA1.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="35pt" align="left"> </colspec> <colspec colname="1" colwidth="35pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="119pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Number of</td>
                <td class="description-td">Peptide</td>
                <td class="description-td">Peptide Sequence Homology</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Amino</td>
                <td class="description-td">Size</td>
                <td class="description-td">(% identity)*</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <colspec colname="6" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">GNA1</td>
                <td class="description-td">Acids</td>
                <td class="description-td">(kDa)</td>
                <td class="description-td">ScGNA1</td>
                <td class="description-td">AtGNA1</td>
                <td class="description-td">CaGNA1</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ScGNA1</td>
                <td class="description-td">158</td>
                <td class="description-td">18</td>
                <td class="description-td"> </td>
                <td class="description-td">38.9 (49.7)</td>
                <td class="description-td">44.0 (53.1)</td>
              </tr> <tr class="description-tr"> <td class="description-td">AtGNA1</td>
                <td class="description-td">148</td>
                <td class="description-td">17</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">37.6 (47.2)</td>
              </tr> <tr class="description-tr"> <td class="description-td">CaGNA1</td>
                <td class="description-td">148</td>
                <td class="description-td">17</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00025" class="description-td" colspan="6">*Homology at the nucleotide level is shown in parentheses.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 14</heading> <p num="p-0418">This Example describes the construction of strains producing N-acetyl glucosamine through the overexpression of the <i>E. coli </i>nagB and the <i>S. cerevisiae </i>GNA1.</p>
    <p num="p-0419">The nagB gene, part of the nag regulon, encodes the glucosamine-6-phosphate deaminase (NagB), is involved in the pathway for catabolism of N-acetyl glucosamine (GlcNAc) as part of the nag regulon. The nag regulon consists of the operon nagBA CD and the divergently transcribed nagE (Plumbridge, JA., 1991, Mol. Microbiol. 8:2053-2062). Exogenous GlcNAc is phosphorylated as it is transported into the cell, forming GlcNAc-6-P. The nagA gene product, encoding the N-acetyl glucosamine-6-phosphate deacetylase, converts the GlcNAc-6-P to GlcN-6-P. NagB then catalyzes the conversion of GlcN-6-P to fructose-6-phosphate (F-6-P).</p>
    <p num="p-0420">The glmS gene product in <i>E. coli </i>catalyzes the synthesis of GlcN-6-P, an essential intermediate in the pathway for the formation of lipopolysaccharide and peptidoglycan. Therefore, mutants with a defective glmS are dependent on exogenous GlcN or GlcNAc. However, NagB has been shown to catalyze the reaction normally performed by GlmS, converting F-6-P to GlcN-6-P. In fact, transformation of glmS mutants with a high-copy plasmid expressing the glucosamine-6-phosphate deaminase (nagB) has been shown to suppress the glmS mutation (J Bac 1989 December; 171(12):6589-6592). Since NagB can catalyze the conversion of F-6-P to GlcN-6-P, it is possible that overexpression of the nagB in place of glmS*54 in our production strains could result in an accumulation of glucosamine. If the T7lac-ScGNA1 cassette was also present in the strain, the GlcN-6-P could be converted to GlcNAc-6-P and accumulate as NAG in the medium.</p>
    <p num="p-0421">To test the efficiency of glucosamine or N-acetyl glucosamine production by overexpression of the nagB gene, the methods and protocols described for GNA1 cloning and integration was adapted to clone and integrate a T7lac-nagB expression cassette at the pfkB site in the chromosome of <i>E. coli </i>7101-17(DE3). The endogenous glmS gene will also need to be inactivated in this strain. The glmS sequence is located downstream from the glmU sequence with no obvious promoter sequence upstream of glmS. It appears that glmU and glmS form an operon glmUS. The glmS sequence and some flanking sequences was amplified by PCR from <i>E. coli </i>genomic DNA, cloned into a plasmid vector. Appropriate restriction digestion was used to remove an internal fragment from the glmS sequence. The glmS sequence with internal deletion was ligated to the temperature sensitive replication origin and the kanamycin selection marker to create an integration vector. The temperature selection protocol was used to select for mutants with the glmS deletion. Since glucosamine is vital for cell wall synthesis, glucosamine needs to be supplied in the culture medium for the growth of the mutants.</p>
    <p num="h-0051">Cloning the nagB Gene for Over-Expression in <i>E. coli </i> </p>
    <p num="p-0422">Using information based on the published sequence of the <i>E. coli </i>nagB gene (Peri et al., 1990, Biochem. Cell Biol. 68, pp.123-137, herein by reference in its entirety), primers were designed to amplify the nagB coding sequence. The nagB coding sequence was amplified by PCR from <i>E. coli </i>W3110 genomic DNA using forward primer 07-141 and reverse primer 07-142, which had the following sequences: 07-141 5′-GATGGTCTCGCATGAGACTGATCCCCCTGAC-3′ (SEQ ID NO:43) and 07-142 5′-GATCCTCGAGTTACAGACCTTTGATATTTTCTGCTTCTAATTC-3′ (SEQ ID NO:44)</p>
    <p num="p-0423">Primer 07-141 contains a Bsa I restriction endonuclease site (GGTCTC, represented in nucleotides 4-9 of SEQ ID NO:43) followed by 20 nucleotides of the nagB coding sequence starting from its ATG start codon (represented in nucleotides 12-31 of SEQ ID NO:43). Primer 07-142 contains a Xho I restriction endonuclease site (CTCGAG, represented in nucleotides 5-10 of SEQ ID NO:44) followed by 33 nucleotides of the nagB coding sequence from its translational stop codon (represented in nucleotides 11-43 of SEQ ID NO:44). The <i>E. coli </i>nagB coding sequence and NagB amino acid sequence are represented by SEQ ID NO:41 and SEQ ID NO:42, respectively.</p>
    <p num="p-0424">The PCR fragment containing the nagB coding sequence was digested with restriction endonucleases Bsa I and Xho I and ligated at the Nco I and Xho I sites of plasmid pET24d(+) (Novagen, Inc., Madison, Wis.), generating plasmid pSW07-93. Cloning in this manner places the nagB sequence behind the T7-lac promoter of pET24d(+), generating an expression cassette of T7-lac-nagB.</p>
    <p num="h-0052">Overexpression of the nagB Gene</p>
    <p num="p-0425">Plasmids pSW07-93#3, #8 and #16 were transformed into strain 7101-17(DE3), generating strains 7107-636,7107-637, and 7107-638, respectively. The pET24d(+) plasmid was also transformed into strain 7101-17(DE3) to generate negative control strain 7107-639. A standard induction experiment was performed in which cell cultures of the transformants were grown in LB medium and induced with 1 mM IPTG. Samples were taken from induced cultures at various timepoints for SDS-PAGE to confirm NagB protein overexpression. The predicted size for the overproduced NagB protein is 29.8 kDa. A protein corresponding to the expected size of NagB was overproduced in strain 7107-636 and 7107-637. No overproduced protein of this size was apparent in control strain 7107-639. Protein samples from 2 hours after induction indicated that most of the overproduced NagB protein was in the soluble fraction. However, at 4 hours after induction, only about 25% of the protein appeared in the soluble fraction.</p>
    <p num="h-0053">Integration of the T7lac-nagB Cassette at the pfkB Locus in the Chromosome</p>
    <p num="p-0426">Having confirmed successful overproduction of NagB, the next step was to integrate the T7lac-nagB expression cassette into the chromosome of <i>E. coli </i>7101-17(DE3). It was decided to target integration to the pfkB gene of the <i>E. coli </i>chromosome. The pfkB gene encodes for a minor phosphofructokinase that supplies 10% of the total phosphofructokinase activity present in <i>E. coli</i>. Therefore, targeting integration to this site should not impair growth or N-acetyl glucosamine production.</p>
    <p num="p-0427">Several cloning steps were required to develop a vector to direct integration of the T7lac-nagB cassette at pfkB of the chromosome in strain 7101-17(DE3). The first step was to clone a region from the <i>E. coli </i>genome containing the pƒkB coding sequencing plus flanking regions. Primers were designed based on published sequences (Blattner et al., 1997, Science 227(5331):1453-1474) to amplify the pƒkB plus flanking genomic regions from <i>E. coli </i>W3110 genomic DNA. Primers 07-16 and 07-17, used to amplify the pfkB region by PCR, had the following sequences: 07-16 5′-GATCGCCGGCTTACATGCTGTAGC CCAGC-3′ (SEQ ID NO:45) and 07-17 5′-GATCCTGCAGTCATGCTGCTAATAATCT ATCC-3′ (SEQ ID NO:46).</p>
    <p num="p-0428">Primer 07-16 contains a Nae I site (GCCGGC, represented in nucleotides 5-10 of SEQ ID NO:45) and amplifies from 1045 nucleotides upstream of the pƒkB coding sequence start codon (represented in nucleotides 11-29 of SEQ ID NO:45). Primer 07-17 adds a Pst I site (CTGCAG, represented in nucleotides 5-10 of SEQ ID NO:46) and amplifies from 1357 basepairs downstream of the pƒkB stop codon (represented in nucleotides 11-32 of SEQ ID NO:46). Ligation of the 3332 basepair PCR product containing the pƒkB plus flanking regions into the SrƒI site of pPCR-Script Amp SK(+) (Stratagene, LaJolla, Calif.) generated plasmid pKLN07-14.</p>
    <p num="p-0429">The T7-lac-nagB cassette was amplified from plasmid pSW07-93#3 (see above) under standard PCR conditions with forward primer 07-145 and reverse primer 07-146. The primers have the following sequences: 07-145 5′-GATCTACGTAAGCAACCGCACCTGT GGC-3′ (SEQ ID NO:47) and 07-146 5′-GATCCAATTGATCCGGATATAGTTCCTCCTT TCAGC-3′ (SEQ ID NO:48).</p>
    <p num="p-0430">Primer 07-145 contains a SnaB I site (TACGTA, represented in nucleotides 5-10 of SEQ ID NO:47) and amplifies from 76 nucleotides upstream of the T7 promoter (represented in nucleotides 11-28 of SEQ ID NO:47). Primer 07-146 contains an Mƒe I site (CAATTG, represented in nucleotides 5-10 of SEQ ID NO:48) and amplifies from 25 basepairs downstream of the T7 terminator (represented in nucleotides 11-36 of SEQ ID NO:48).</p>
    <p num="p-0431">The next cloning step was ligation of the T7lac-nagB cassette into plasmid pKLN07-14. Plasmid pKLN07-14 was digested with restriction endonucleases SnaB I and Mƒe I, removing a 523-bp portion of the pƒkB coding sequence. The PCR fragment containing the T7lac-nagB cassette was digested with restriction endonucleases SnaB I and Mƒe I and ligated at the SnaB I and Mƒe I sites of pKLN07-14, generating plasmid pSW07-97. Plasmid pSW07-97 therefore contains the pƒkB plus flanking genomic regions with a 523 nucleotide region of the pƒkB coding sequence replaced with the T7lac-nagB cassette.</p>
    <p num="p-0432">The fragment containing ORFb1722-DpfkB::T7-lac-nagB-ORFb1725 plus part of the pPCR-Script MCS was digested from plasmid pSW07-97 with restriction endonucleases Not I and Sal I. A fragment containing the temperature sensitive replicon plus kanamycin resistance cassette was excised from plasmid pKLN07-21 (previously described) with restriction endonucleases Not I and Sal I. The two fragments were ligated together, generating plasmid pSW07-98.</p>
    <p num="p-0433">Plasmid pSW07-98 was used to generate <i>E. coli </i>strains with the T7lac-nagB at DpƒkB of the chromosome. Following transformation of <i>E. coli </i>7101-17(DE3) with pSW07-98, the temperature selection protocol was used to select for strains with the T7-lac-nagB integrated at the pƒkB site. The resulting strains were designated 7107-645, and strains were confirmed by standard high stringency Southern hybridization using the nagB coding sequence as probe.</p>
    <p num="h-0054">Deletion of the glmS Gene in the <i>E. coli </i>Strains with an Integrated T7lac-nagB Cassette</p>
    <p num="p-0434">To delete the glmS gene <i>E. coli </i>strain 7107-645, a glmS sequence replacement vector was developed. Several steps were required to construct the vector. The first step was to amplify the region of the genome containing the glmS gene plus flanking sequence. This fragment would next be ligated with the fragment from pKLN07-21 (previously described) containing the temperature sensitive replicon and kanamycin resistance cassette. Finally, a portion of the glmS coding sequence would be excised from the resulting plasmid, generating the integrative vector to target the glmS deletion to the <i>E. coli </i>genome.</p>
    <p num="p-0435">For the first step, primers were synthesized based on the published sequence of the glmS gene plus flanking regions (reference). The primers were used to amplify a fragment containing the glmU, glmS, and pstS genes from <i>E. coli </i>W3110 genomic DNA. The primers used for amplification were designated 07-139 and 07-140 and had the following sequences: 07-139, 5′-GATGCGGCCGCATGTTGAATAATGCTATGAGCGTAGTGATC-3′ (SEQ ID NO:49) and 07-140, 5′-GATCGTCGACTTAGTACAGCGGCTTACCGCTACTGTC-3′ (SEQ ID NO. 50).</p>
    <p num="p-0436">Forward primer 07-139 contains the first 30 nucleotides of the glmU coding sequence from its ATG start codon (represented in nucleotides 12-41 of SEQ ID NO:49) preceded by a Not I restriction endonuclease site (GCGGCCGC, represented in nucleotides 4-11 of SEQ ID NO:49). Reverse primer 07-140 contains the last 27 nucleotides of the pstS coding sequence starting from its translational stop codon (represented in nucleotides 11-37 of SEQ ID NO:50) preceded by a Sal I restriction endonuclease site (GTCGAC, represented in nucleotides 5-10 of SEQ ID NO:50). PCR was conducted under standard conditions to generate a fragment containing the glmU-glmS-pstS coding sequences flanked by Not I and Sal I restriction endonuclease sites.</p>
    <p num="p-0437">The PCR fragment was digested with restriction endonucleases Not I and Sal I. The temperature sensitive replicon plus kanamycin resistance cassette fragment was excised from plasmid pKLN07-21 (previously described) with restriction endonucleases Not I and Sal I. The two fragments were ligated together, generating plasmid pSW07-94#43.</p>
    <p num="p-0438">Plasmid pSW07-94#43 was digested with restriction endonuclease Sac II, removing 980 nucleotides of the glmS coding sequence. The remainder of the plasmid was ligated to itself, generating an integration vector, pSW07-95.</p>
    <p num="p-0439">In an attempt to improve frequency of recombination for generating the glmS mutation on the <i>E. coli </i>chromosome, plasmid pSW07-99 was also constructed by adding 772 nucleotides of the region upstream of glmU to plasmid pSW07-95. Primers were synthesized based on the published sequence of the glmU gene plus flanking regions (reference). The primers were used to amplify a fragment containing 772 nucleotides upstream of the glmU start codon and the first 246 nucleotides of the glmU coding sequence from <i>E. coli </i>W3110 genomic DNA. The primers used for amplification were designated 07-147 and 07-148 and had the following sequences: 07-147,5′-GATGCGGCCGCATGGCAATGACTTACCACC TGGAC-3′ (SEQ ID NO:51) and 07-148, 5′-CGTACCCAGCTGCTCTGCCTGAAGCA CCC-3′ (SEQ ID NO:52).</p>
    <p num="p-0440">Forward primer 07-147 contains the first 24 nucleotides of the atpC coding sequence (represented in nucleotides 12-35 of SEQ ID NO:51) preceded by a Not I restriction endonuclease site (GCGGCCGC, represented in nucleotides 4-11 of SEQ ID NO:51). Reverse primer 07-148 contains 29 nucleotides of the glmU coding sequence (represented in nucleotides 1-29 of SEQ ID NO:52) starting from 246 basepairs downstream of its ATG start codon. PCR was conducted under standard conditions to generate a fragment containing the region of genomic DNA from the atpC start codon to nucleotide 246 of the glmU coding sequence. The PCR product was digested with restriction endonucleases Not I and SexA I and ligated into the Not I and SexA I sites of plasmid pSW07-95#6, generating plasmid pSW07-99.</p>
    <p num="p-0441">Plasmids pSW07-95 and pSW07-99 were used to generate <i>E. coli </i>strains with the glmS deletion on the chromosome. Following transformation of <i>E. coli </i>strains 7107-645(20), 7107-645(30), and 7107-645(43) with plasmid pSW07-95 or pSW07-99, the temperature selection protocol was used to select for strains with the glmS deletion. Glucosamine (2 g l<sup>−1</sup>) was added to all flasks for passaging and all plates following passaging. Kanamycin-sensitive strains were selected and screened using a standard PCR protocol for the presence of the glmS deletion. Potential glmS deletion strains identified by PCR were plated to LB plates without added glucosamine to look for reduced growth. Some of the strains had limited growth on LB plates, while others exhibited no growth. These strains were confirmed as having the glmS deletion by Southern hybridization under high stringency conditions using a 2.0-kb fragment containing the glmS coding sequence. The resulting strains were designated 7107-646.</p>
    <p num="h-0055">Functional Expression of nagB and the Effect on Glucosamine Production</p>
    <p num="p-0442">Shake Flask Screen 61 was conducted to test strains 7107-646 for glucosamine production and enzyme activity. Strains were tested in flasks containing M9B medium [6 g l<sup>−1 </sup>KH<sub>2</sub>PO<sub>4</sub>, 24 g l<sup>−1 </sup>K<sub>2</sub>HPO<sub>4</sub>, 1 g l<sup>−1 </sup>Na<sub>3</sub>Citrate-2H<sub>2</sub>O, 10 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>(phosphate adjusted to pH 7.4) plus trace metals (0.2 mg l<sup>−1 </sup>FeSO<sub>4</sub>-7H<sub>2</sub>O, 0.015 mg l<sup>−1 </sup>ZnSO<sub>4</sub>-7H<sub>2</sub>O, 0.015 mg l<sup>−1 </sup>MnSO<sub>4</sub>—H<sub>2</sub>O, 0.001 m g l<sup>−1 </sup>CuSO4-5H<sub>2</sub>O, 0.001 mg l<sup>−1 </sup>NaMoO<sub>4</sub>-2H<sub>2</sub>O, 0.001 mg l<sup>−1 </sup>H<sub>3</sub>BO<sub>3</sub>, and 0.001 mg l<sup>−1 </sup>CoCl<sub>2</sub>-6H<sub>2</sub>O)] supplemented with 10 g l<sup>−1 </sup>glucose, 0.6 g l<sup>−1 </sup>MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g l<sup>−1 </sup>CaCl<sub>2</sub>-2H<sub>2</sub>O and 0.2 mM IPTG. Cultures were grown at 30° C. for 24 hours and then placed at 25° C. The pH of each culture was adjusted to 7.2 at 24 and 48 hours. At 24 and 48 hours, glucose was added to flasks to approximately 30 g l<sup>−1 </sup>per day total, and 5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added to flasks in which levels had fallen below 1 g l<sup>−1</sup>. Samples were taken at 24 and 48 hours and the glucosamine concentration in the culture supernatant was measured using the modified Elson-Morgan assay as described in U.S. Pat. No. 6,372,457. No glucosamine was detected in any of the samples. 48 hour samples from strain 7107-646#7 and 7107-646#20 were tested for NagB activity. These strains had enzyme activities of 58 and 53 mmol min<sup>−1 </sup>mg<sup>−1</sup>, compared with no detectable activity in the control strain 7101-17(DE3), indicating successful overexpression of nagB. The fact that the 7107-646 strains grew as well or better than control strain 7101-17(DE3) in this experiment indicated that the overexpressed nagB is capable of suppressing the glmS mutation. However, overexpression of nagB did not result in increased glucosamine accumulation.</p>
    <p num="h-0056">Integration of T7lac-ScGNA1 Cassette</p>
    <p num="p-0443">Overexpression of nagB in glmS deletion strains resulted in no accumulation of glucosamine, potentially due to the fact that NagB normally catalyzes the deamination of GlcN-6-P. The small amount of GlcN-6-P produced by the NagB enzyme might be quickly converted back to fructose-6-P, preventing any accumulation of GlcN-6-P. However, if a GNA1 enzyme was introduced, GNA1 would convert GlcN-6-P to GlcNAc-6-P and continuously drive the formation of glucosamine-6-phosphate from fructose-6-phosphate. To test this possibility, the T7lac-ScGNA1 cassette was integrated at manXYZ of strains 7107-646(3) and 7107-646(7). GNA1 integration was carried out with plasmid pSW07-68#25 as detailed in Example 16. Kanamycin-sensitive strains were screened by a standard PCR protocol for the presence of the T7lac-ScGNA1 at the site of the manXYZ deletion. Strains positive by PCR were confirmed by standard high stringency Southern hybridization using a fragment containing the ScGNA1 coding sequence as probe. Resulting strains 7107-660 and 7107-661, derived from strains 7107-646(3) and 7107-646(7), respectively, were tested for NAG accumulation.</p>
    <p num="h-0057">Functional Expression of nagB and N-Acetyl Glucosamine Production in <i>E. coli </i> </p>
    <p num="p-0444">Shake Flask Screen 67 was conducted to test strains with the glmS deletion, T7lac-nagB cassette at pƒkB, and T7lac-ScGNA1 cassette at manXYZ for N-acetyl glucosamine production. Strains 7107-660(1), 7107-660(4), 7107-661(1), 7107-661(2), and 7107-661(3), as well as control strains 7101-17 and 7107-607(2), were grown in shake flasks containing M9B medium (previously described) supplemented with g l<sup>−1 </sup>glucose, 10 g l<sup>−1 </sup>lactose, 0.6 g l<sup>−1 </sup>MgSO<sub>4</sub>-7H<sub>2</sub>O, and 0.05 g l<sup>−1 </sup>CaCl<sub>2</sub>-2H<sub>2</sub>O. Cultures were grown at 37° C. for 8 hours and then placed at 30° C. 8 hours after inoculation, glucose was added to 25 g l<sup>−1 </sup>and pH was adjusted to 7.2. At 24, 31, 48, and 56 hours, glucose was added based on HPLC results to 30-40 g l<sup>−1 </sup>per day total, (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added to 5 g l<sup>−1 </sup>in flasks with levels below 1 g l<sup>−1</sup>, and pH was adjusted to 7.2. At 27 hours, lactose was added to 5 g l<sup>−1</sup>. Samples were removed at 8, 24, 48 and 72 hours for HPLC analysis of NAG levels. Cell cultures were harvested at 72 hours to assay glucosamine-6-phosphate acetyltransferase (GNA1), glucosamine synthase (GlmS), and glucosamine-6-phosphate deaminase (NagB) activities.</p>
    <p num="p-0445">Enzyme assays confirmed that the 7107-660 and 7107-661 strains lacked functional GlmS protein. As expected, NagB activity was elevated in these strains. Glucosamine-6-phosphate acetyltransferase activity was similar to that seen in control strain 7107-607(2), indicating functional expression of the GNA1 protein in these strains.</p>
    <p num="p-0446">The glmS deletion strains overexpressing the nagB and GNA1 were capable of producing N-acetyl glucosamine (Table 15). Strain 7107-661 performed the best, producing 75% of the level of NAG achieved with production strain 7107-607#2. Interestingly, strain 7107-660(1) showed NagB activity 20 fold lower than other siblings and as a consequence produced a much lower level of GlcNAc. It appears that the glucosamine-6-P acetyltransferase helps drive the reaction catalyzed by NagB in the direction of glucosamine-6-phosphate formation. The data demonstrate the functionality of a novel biological pathway for N-acetylglucosamine synthesis. Furthermore, overexpression of nagB in conjunction with GNA1 is an efficient method for producing N-acetyl glucosamine in <i>E. coli</i>.</p>
    <p num="p-0447">
      <tables id="TABLE-US-00021" num="00021"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 15</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Enzyme activities and GlcNAc production in</td>
              </tr> <tr class="description-tr"> <td class="description-td">glmS mutant strains over xpressing nagB.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="105pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Enzyme Activity</td>
                <td class="description-td">GlcNAc</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Construct</td>
                <td class="description-td">Strain</td>
                <td class="description-td">NagB</td>
                <td class="description-td">GlmS</td>
                <td class="description-td">GNA1</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="28pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">control</td>
                <td class="description-td">7101-17</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.08</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">glmS*54, 2 copies</td>
                <td class="description-td">7107-607(2)</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.22</td>
                <td class="description-td">3.8</td>
                <td class="description-td">30.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">of GNA1</td>
              </tr> <tr class="description-tr"> <td class="description-td">glmS deletion, nagB</td>
                <td class="description-td">7107-660(1)</td>
                <td class="description-td">2.0</td>
                <td class="description-td">0.0</td>
                <td class="description-td">3.5</td>
                <td class="description-td">5.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">and GNA1</td>
                <td class="description-td">7107-660(4)</td>
                <td class="description-td">42</td>
                <td class="description-td">0.0</td>
                <td class="description-td">2.9</td>
                <td class="description-td">18.7</td>
              </tr> <tr class="description-tr"> <td class="description-td">glmS deletion, nagB</td>
                <td class="description-td">7107-661(1)</td>
                <td class="description-td">52</td>
                <td class="description-td">0.0</td>
                <td class="description-td">3.5</td>
                <td class="description-td">22.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">and GNA1</td>
                <td class="description-td">7107-661(2)</td>
                <td class="description-td">62</td>
                <td class="description-td">0.0</td>
                <td class="description-td">4.1</td>
                <td class="description-td">22.1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">7107-661(3)</td>
                <td class="description-td">58</td>
                <td class="description-td">0.0</td>
                <td class="description-td">3.9</td>
                <td class="description-td">24.2</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00026" class="description-td" colspan="6">1)Enzyme activities and N-acetyl glucosamine levels were determined in samples taken at the 72 hour timepoint.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00027" class="description-td" colspan="6">2)Enzyme activity is expressed in μmol min<sup>−1 </sup>mg<sup>−1 </sup>protein.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00028" class="description-td" colspan="6">3)Numbers in parentheses indicate different siblings.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 15</heading> <p num="p-0448">This Example describes cloning and overexpression of the glmM and glmU genes for N-acetylglucosamine production in strain 7107-18.</p>
    <p num="p-0449">In <i>E. coli</i>, the glmM and glmU genes encode the enzymes that catalyze the first three steps by which GlcN-6-P is converted to UDP-GlcNAc. UDP-GlcNAc is required for the synthesis of essential cell-envelope components. GlmM catalyzes the interconversion of GlcN-6-P and GlcN-1-P by a two-step ping-pong reaction mechanism in which GlcN-1,6-diphosphate serves as both the first product and the sec active only in a phosphorylated form, although in vivo enzyme activation is unknown. In strains overproducing GlmM at high levels, the total amount of phosphorylated enzyme was not increased, indicating that the level of GlmM phosphorylation may be tightly regulated (Mengin-Lecreulx and van Heifenoort, J. Biol. Chem. 1996 271:32-39). GlmU is a bifunctional enzyme with separate uridyltransferase (N-terminal) and acetyltransferase (C-terminal) domains. The acetyltransferase domain is responsible for catalyzing the conversion of GlcN-1-P to GlcNAc-1-P; the uridyltransferase domain then converts GlcNAc-1-P to UDP-GlcNAc. In a recent publication, truncated versions of GlmU were expressed with N-terminal His<sub>6</sub>tags and assayed for acetyltransferase and uridyltransferase activities (Pompeo et al., J. Biol. Chem. 2001 276:3833-3839). A truncated form of GlmU with uridyltransferase activity decreased by a factor of 1320 relative to full-length GlmU was obtained by deleting the first 78 N-terminal amino acids residues from the protein. This truncated GlmU protein retained 66% of the acetyltransferase activity seen in the full-length GlmU.</p>
    <p num="p-0450"> <figref idrefs="DRAWINGS">FIG. 14</figref> shows the bacterial pathway by which GlcN-6-P is converted to UDP-GlcNAc through the action of the GlmM and GlmU enzymes. Overexpression of glmM, glmU, truncated glmU, or a combination of glmM and glmU may result in an accumulation of N-acetyl glucosamine in the medium. Therefore, strains were constructed for overexpression of these genes.</p>
    <p num="h-0059">Cloning the glmM Gene for Over-Expression in <i>E. coli: </i> </p>
    <p num="p-0451">For cloning and expression of the <i>E. coli </i>glmM gene, primers were synthesized based on the published sequence of the glmM gene (Mengin-Lecreulx, and van Heijenoort, J. Biol. Chem., 1996, 271:32-39). The glmM coding sequence was amplified by PCR under standard conditions from <i>E. coli </i>W3110 genomic DNA using forward primer 07-163 and reverse primer 07-164. The primers have the following sequences: 07-163: 5′ GATCGGTCTC GCATGAGTAATCGTAAATATTTC 3′ (SEQ ID NO:59) and 07-164:5′ GATCCTCGAG TTAAACGGCTTTTACTGCATC3′ (SEQ ID NO:60).</p>
    <p num="p-0452">Primer 07-163 contains a Bsa I restriction endonuclease site (GGTCTC, represented in nucleotides 5-10 of SEQ ID NO:59) followed by 21 nucleotides of the glmM coding sequences starting from its ATG start codon (represented in nucleotides 13-33 of SEQ ID NO:59). Primer 07-164 contains a Xho I restriction endonuclease site (CTCGAG, represented in nucleotides 5-10 of SEQ ID NO:60) followed by 21 nucleotides of the glmM coding sequence starting at its translational stop codon (represented in nucleotides 11-31 of SEQ ID NO:60). <i>E. coli </i>glmM coding sequence and GlmM amino acid sequence are represented as SEQ ID NO:53 and SEQ ID NO:54, respectively. PCR amplification was conducted under standard conditions to generate a fragment of DNA containing the entire glmM coding sequence flanked by Bsa I and Xho I restriction endonuclease sites. The PCR product was digested with restriction enzymes Bsa I and Xho I and ligated at the Nco I and Xho I sites of plasmid pET24d(+) (Novagen, Inc., Madison, Wis.), generating plasmid pSW07-109. Cloning in this manner places the glmM sequence behind the T7lac promoter of pET24d(+), generating the T7lac-glmM expression cassette.</p>
    <p num="h-0060">Cloning the glmU Gene for Over-Expression in <i>E. coli: </i> </p>
    <p num="p-0453">For cloning and expression of the <i>E. coli </i>glmU gene, primers were synthesized based on the published sequence of the glmU gene (Mengin-Lecreulx, and van Heijenoort, J. Bac., 1993, 175:6150-6157). The glmU coding sequence was amplified by PCR under standard conditions from <i>E. coli </i>W3110 genomic DNA using forward primer 07-161 and reverse primer 07-162. The primers have the following sequences: 07-161: 5′GATCGGTCTC GCATGTTGAATAATGCTATGAGC3′ (SEQ ID NO:61) and 07-162: 5′GATCCTCGA GTCACTTTTTCTTTACCGGACGAC3′ (SEQ ID NO:62).</p>
    <p num="p-0454">Primer 07-161 contains a Bsa I restriction endonuclease site (GGTCTC, represented in nucleotides 5-10 of SEQ ID NO:61) followed by 21 nucleotides of the glmU coding sequence starting at its ATG start codon (represented in nucleotides 13-33 of SEQ ID NO:61). Primer 07-162 contains a Xho I site (CTCGAG, represented in nucleotides 5-10 of SEQ ID NO:62) followed by 23 nucleotides of the glmU coding sequence starting at its translational stop codon (represented in nucleotides 11-33 of SEQ ID NO:62). <i>E. coli </i>glmU coding sequence and GlmU amino acid sequence are represented by SEQ ID NO:55 and SEQ ID NO:56, respectively. PCR amplification was conducted under standard conditions to generate a fragment of DNA containing the entire glmU coding sequence flanked by Bsa I and Xho I restriction endonuclease sites. The PCR product was digested with restriction enzymes Bsa I and Xho I and ligated at the Nco I and Xho I sites of plasmid pET24d(+) (Novagen, Inc., Madison, Wis.), generating plasmid pSW07-108. Cloning in this manner places the glmM sequence behind the T7lac promoter of pET24d(+), generating an expression cassette of T7lac-glmU.</p>
    <p num="h-0061">Cloning and Expression of a N-Terminal Truncated GlmU Enzyme (GlmUt)</p>
    <p num="p-0455">The truncated glmU coding sequence was amplified by PCR under standard conditions from <i>E. coli </i>W3110 genomic DNA using forward primer 07-165 and reverse primer 07-162. Primer 07-165 had the following sequence: 07-165: 5′ GATGGTCTCGCAT GGAGCAGCTGGGTACGGGTC 3′ (SEQ ID NO:63).</p>
    <p num="p-0456">Primer 07-165 contains a Bsa I restriction endonuclease site (GGTCTC, represented in nucleotides 4-9 of SEQ ID NO:63) and the glmU CDS sequence starting at 232 bp downstream of the ATG start codon (represented in nucleotides 15-33 of SEQ ID NO:63). This results in a deletion of the first 77 amino acids of the GlmU protein. A start codon was also incorporated into primer 07-165. The PCR product generated with primers 07-165 and 07-162 contains the glmU coding sequence with the first 77 amino acid residues deleted. <i>E. coli </i>N-terminal truncated glmU coding sequence and N-terminal truncated GlmU amino acid sequence are represented by SEQ ID NO:57 and SEQ ID NO:58, respectively. The PCR product was digested with Bsa I and Xho I ligated at the Nco I and Xho I sites of plasmid pET24d(+) (Novagen, Inc., Madison, Wis.), generating plasmid pSW07-110. Cloning in this manner places the truncated glmU (glmUt) sequence behind the T7lac promoter of pET24d(+), generating an expression cassette of T7lac-glmUt.</p>
    <p num="h-0062">Overexpression of GlmM, GlmU and GlmUt Proteins in <i>E. coli </i> </p>
    <p num="p-0457">Plasmids pSW07-108, pSW07-109, and pSW07-110, containing the glmU, glmM or glmUt, were transformed into strain 7101-17(DE3), generating strains listed in Table 16. The empty vector pET24d(+) was transformed into strain 7101-17(DE3), generating control strain 7107-22. A standard induction experiment was performed in which cell cultures of the transformants were grown in LB medium and induced with 1 mM IPTG. Samples were taken from induced cultures at various timepoints for SDS-PAGE to confirm protein overexpression. The predicted size for the GlmU, GlmM and GlmUt proteins were 51 kDa, 50 kDa and 42 kDa, respectively. Protein bands of predicted sizes on SDS-PAGE were seen in samples from strains overexpressing the glmU, glmM and glmUt. No overexpressed proteins near the predicted sizes of GlmU, GlmM, or GlmUt were seen in the control strain. Total and soluble fractions of samples from induced cultures were determined by SDS-PAGE. As judging by visual estimation, about 20% of the GlmU protein was in the soluble fraction. However, little soluble protein was seen with the truncated version of the GlmU protein. Similarly, little GlmM protein was in soluble form.</p>
    <p num="p-0458"> <tables id="TABLE-US-00022" num="00022"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 16</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Strains containing different plasmids</td> </tr> <tr class="description-tr"> <td class="description-td">for GlmU, N-terminal truncated GlmU and</td> </tr> <tr class="description-tr"> <td class="description-td">GlmM proteins.</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="105pt" align="left"> </colspec> <colspec colname="3" colwidth="77pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain</td> <td class="description-td">Description</td> <td class="description-td">Overexpressed Protein</td> </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> <tr class="description-tr"> <td class="description-td">7107-667</td> <td class="description-td">7101-17(DE3)/pSW07-108#1</td> <td class="description-td">GlmU</td> </tr> <tr class="description-tr"> <td class="description-td">7107-668</td> <td class="description-td">7101-17(DE3)/pSW07-108#3</td> <td class="description-td">GlmU</td> </tr> <tr class="description-tr"> <td class="description-td">7107-669</td> <td class="description-td">7101-17(DE3)/pSW07-109#29</td> <td class="description-td">GlmM</td> </tr> <tr class="description-tr"> <td class="description-td">7107-670</td> <td class="description-td">7101-17(DE3)/pSW07-109#30</td> <td class="description-td">GlmM</td> </tr> <tr class="description-tr"> <td class="description-td">7107-671</td> <td class="description-td">7101-17(DE3)/pSW07-110#53</td> <td class="description-td">Truncated GlmU</td> </tr> <tr class="description-tr"> <td class="description-td">7107-672</td> <td class="description-td">7101-17(DE3)/pSW07-110#54</td> <td class="description-td">Truncated GlmU</td> </tr> <tr class="description-tr"> <td class="description-td">7107-22</td> <td class="description-td">7101-17(DE3)/pET24d(+)</td> <td class="description-td">Empty Vector Control</td> </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
Integration of T7lac-glmU and T7lac-glmUt at the Site of the nag Deletion
</p>
    <p num="p-0459">SDS-PAGE gels indicated successful overexpression of the GlmU and GlmUt proteins in <i>E. coli</i>. Therefore, a strategy was developed to integrate the expression cassettes into the chromosome of production strain 7107-18. The target chosen for integration was the site of the nag deletion on the chromosome of strain 7107-18. In the early stages of construction of glucosamine production strains, the nag operon was deleted and a tetracycline resistance cassette inserted at the site of the deletion on the chromosome following P1 transduction with phage prepared from strain IBPC590 (Plumbridge, 1989<i>, Mol. Microbiol</i>. 3:506-515; Plumbridge, 1991<i>, Mol. Microbiol</i>. 5:2053-2062, Plumbridge, 1992<i>, J. Gen. Microbiol</i>. 138:1011-1017). Therefore, targeting integration to this region of the chromosome should not affect growth or glucosamine production in the strain.</p>
    <p num="p-0460">As part of the strategy to develop a vector to target integration of the T7lac-glmU cassette at the site of the nag deletion of the chromosome, the T7lac-glmU fragment was amplified by PCR from plasmid pSW07-108#1. PCR was performed under standard conditions with primers GNT7nagA1-5 and 07-120. The primers have the following sequences: GNT7nagA 1-5: 5′GCGACGCTCTCCCGGGTGCGACTCCTGCATTA3′ (SEQ ID NO:64) and 07-120: 5′ GATCTGTACAATCCGGATATAGTTCCTCCTTTCAGCAAA AAACCCC3′ (SEQ ID NO:65).</p>
    <p num="p-0461">The forward primer GNT7nagA1-5 incorporates a Xma I restriction endonuclease site (CCCGGG, represented in nucleotides 11-16 of SEQ ID NO:64) and amplifies from 296 basepairs upstream of the T7 promoter. Primer 07-120 contains a BsrG I restriction endonuclease site (TGTACA, represented in nucleotides 5-10 of SEQ ID NO:65) and amplifies from 25 basepairs downstream of the T7 terminator.</p>
    <p num="p-0462">Plasmid pCALG43 (described in Example 29) contains sequence flanking the nag deletion of the production strains, the temperature sensitive replicon from pMAK705 (Hamilton et al., 1989, J. Bac. 171(9):4617-4622, herein by reference of its entirety) and the kanamycin resistance cassette of plasmid pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.). The PCR product containing the T7lac-glmU cassette was digested with restriction enzymes Xma I and BsrG I and ligated at the Age I and BsrG I sites of pCALG43. The resulting plasmid, pSW07-112, can be used to direct integration of the T7lac-glmU cassette to the site of the nag deletion on the chromosome of our production strains.</p>
    <p num="p-0463">To generate a plasmid for integration of the T7lac-glmU (truncated) cassette at Dnag, the same strategy and PCR primers were used, except that plasmid pSW07-110#53 served as template for PCR. The resulting plasmid was designated pSW07-113.</p>
    <p num="p-0464">Plasmids pSW07-112 and pSW07-113 were used to generate <i>E. coli </i>strains with the T7lac-glmU cassette or the T7lac-glmUt cassette integrated at the site of the nag deletion on the chromosome. Following transformation of <i>E. coli </i>7107-18 with the plasmids, the temperature selection protocol described in Example 13 was used to select for strains containing the insertions. Kanamycin-sensitive colonies were screened by PCR for the presence of the glmU expression cassettes at the site of the nag deletion. Strains positive by PCR were confirmed by Southern hybridizations conducted under high stringency conditions using the truncated glmU PCR product as probe. Strains 7107-678 and 7107-679 were confirmed as having the T7lac-glmU integrated at the site of the nag deletion. Strains 7107-680 and 7107-681 were confirmed as having the T7lac-glmUt integrated at the site of the nag deletion.</p>
    <p num="h-0063">Integration of the T7lac-glmM at the Site of the glg Deletion</p>
    <p num="p-0465">SDS-PAGE gels indicated successful overexpression of the GlmM protein in <i>E. coli</i>. Therefore, a strategy was developed to integrate the T7lac-glmM cassette into the chromosome of production strain 7107-18. The target chosen for integration was the glg operon. The glg region had previously been targeted for deletion in an effort to increase carbon flow through the glucosamine and N-acetylglucosamine production pathways by blocking the glycogen synthesis pathway. This mutation had no detrimental effect on growth or NAG production. Therefore, integration of the T7lac-glmM cassette at this chromosomal site should not negatively affect production strains.</p>
    <p num="p-0466">As part of the strategy to develop a vector to target integration of the T7lac-glmM cassette at the glg site, the T7lac-glmM fragment was amplified by PCR from plasmid pSW07-109#29. PCR was performed under standard conditions with primers GNT7nagA1-5 and GNT7nagA2-3. The primers have the following sequences: GNT7nagA1-5: 5′GCGACGCTCTCCCGGGTGCGACTCCTGCATTA3′ (SEQ ID NO:66) and GNT7nagA2-3: 5′GCGCTAATCAAGTTTTCCCGGGTCGAGGTGCCGTAA3′ (SEQ ID NO:67).</p>
    <p num="p-0467">Primer GNT7nagA1-5 incorporates an Xma I site (CCCGGG, represented in nucleotides 11-16 of SEQ ID NO:66) and amplifies from 296 basepairs upstream of the T7 promoter. Primer GNT7nagA2-3 also incorporates a Xma I site (CCCGGG, represented in nucleotides 17-22 of SEQ ID NO:67) and amplifies from 254 basepairs downstream of the T7 terminator. The resulting PCR fragment was digested with restriction endonuclease Xma I and ligated into the Age I site of plasmid pCALG28-2 (described in Example 29). This generated plasmids pSW07-111 #3, with the T7lac-glmM cassette ligated in the same orientation with the gig operon, and pSW07-111 #4, with the T7lac-glmM cassette ligated in the opposite orientation of the glg operon. These plasmids can be used to direct integration of the T7lac-glmM to the glg region of the production strain.</p>
    <p num="p-0468">Following transformation of <i>E. coli </i>7107-18 with plasmid pSW07-111#3 or pSW07-111#4, the temperature selection protocol was used to select for strains containing the T7-lac-glmM at the glg site. Southern hybridizations were conducted under high stringency conditions using the glmM coding sequence as probe. Strain 7107-682 was confirmed as having the T7-lac-glmM integrated at the glg site in the chromosome (glmM CDS in the same orientation as the interrupted glg genes). Strain 7107-683 was confirmed as having the T7-lac-glmM integrated at the glg site in the chromosome (glmM CDS in the opposite orientation as the interrupted glg genes).</p>
    <p num="h-0064">GlmM/GlmU Assays</p>
    <p num="p-0469">Various strains containing over-expressed GlmM (mutase), GlmU (acetyltransferase/uridyltransferase), and N-terminal truncated GlmU were examined. Activities of these enzymes were assayed in selected strains from this screen. Phosphoglucosamine mutase (GlmM) was assayed using a coupled reaction in the glucosamine-1-phosphate to glucosamine-6-P direction. The glucosamine-6-P formed was quantitatively converted to 6-phosphogluconate using glucosamine-6-P deaminase (NagB), phosphoglucoisomerease and glucose-6-P dehydrogenase. Formation of NADH allowed monitoring the reaction at 340 nm. Glucosamine-1-phosphate acetyltransferase (GlmU) was assayed using glucosamine-1-phosphate and acetyl-CoA. Formation of free CoA was measured in endpoint assay using the reagent dithiobis (2-nitrobenzoic acid) (DTNB). Formation of free CoA was monitored at 410 nm.</p>
    <p num="p-0470">Levels of enzyme activities and levels of glucosamine and N-acetylglucosamine are summarized in Table 17. Significant amounts of N-acetylglucosamine were produced in strains having overexpressed GlmU, either the native form or the N-terminal truncated version. Activities of this enzyme were generally 30-50 fold higher than the control strain 7017-18 which showed only a very low level of activity. Significant free glucosamine also was formed in these strains. Overexpression of GlmM protein did not lead to higher activity. Under the experimental conditions, overexpression of GlmM alone did not allow formation of significant amount of N-acetylglucosamine. Moreover, overexpression of GlmM plus GlmU did not result in increased levels of N-acetylglucosamine relative to GlmU strains. As literature reported, the GlmM enzyme is subject to regulation of phosphorylation, which was not addressed here. It is anticipated that the creation and use of GlmM mutant enzymes which bypass phosphorylation regulation and or have other improved kinetic features could increase the efficiency of N-acetylglucosamine synthesis by the pathway of GlmS-GlmM-GlmU, or NagB-GlmM-GlmU.</p>
    <p num="p-0471">
      <tables id="TABLE-US-00023" num="00023"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 17</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Analysis of strains over-expressing GlmM, GlmU</td>
              </tr> <tr class="description-tr"> <td class="description-td">and GlmUt (N-terminal truncated GlmU).</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="98pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="63pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Enzyme</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Activities</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">(μmol min<sup>−1</sup> </td>
                <td class="description-td">Metab lites</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mg<sup>−1 </sup>protein)</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain</td>
                <td class="description-td">Genotyp*</td>
                <td class="description-td">GlmM</td>
                <td class="description-td">GlmU</td>
                <td class="description-td">NAG</td>
                <td class="description-td">Glucosamine</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="42pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">7017-18</td>
                <td class="description-td">glmS</td>
                <td class="description-td">0.019</td>
                <td class="description-td">0.008</td>
                <td class="description-td">0.2</td>
                <td class="description-td">4.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">7017-607(2)</td>
                <td class="description-td">glmS GNA1</td>
                <td class="description-td"> </td>
                <td class="description-td">0.083</td>
                <td class="description-td">5.2</td>
                <td class="description-td">ND</td>
              </tr> <tr class="description-tr"> <td class="description-td">7017-678(1)</td>
                <td class="description-td">glmS glmU</td>
                <td class="description-td">0.030</td>
                <td class="description-td">0.480</td>
                <td class="description-td">1.3</td>
                <td class="description-td">2.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">7017-678(2)</td>
                <td class="description-td">glmS glmU</td>
                <td class="description-td"> </td>
                <td class="description-td">0.360</td>
                <td class="description-td">1.2</td>
                <td class="description-td">2.7</td>
              </tr> <tr class="description-tr"> <td class="description-td">7017-680(1)</td>
                <td class="description-td">glmS glmUt</td>
                <td class="description-td">0.032</td>
                <td class="description-td">0.500</td>
                <td class="description-td">1.1</td>
                <td class="description-td">2.1</td>
              </tr> <tr class="description-tr"> <td class="description-td">7017-683(1)</td>
                <td class="description-td">glmS glmM</td>
                <td class="description-td">0.045</td>
                <td class="description-td">0.027</td>
                <td class="description-td">0.3</td>
                <td class="description-td">2.9</td>
              </tr> <tr class="description-tr"> <td class="description-td">7017-689(1)</td>
                <td class="description-td">glmS glmM glmU</td>
                <td class="description-td">0.046</td>
                <td class="description-td">0.310</td>
                <td class="description-td">0.9</td>
                <td class="description-td">1.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">7017-687(1)</td>
                <td class="description-td">glmS glmM</td>
                <td class="description-td">0.055</td>
                <td class="description-td">0.170</td>
                <td class="description-td">0.3</td>
                <td class="description-td">2.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">glmUt</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00029" class="description-td" colspan="6">*All the listed recombinant genes were over-expressed under the T7 promoter control.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00030" class="description-td" colspan="6">glmS = glucosamine synthetase</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00031" class="description-td" colspan="6">glmM = phosphoglucosamine mutase</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00032" class="description-td" colspan="6">glmU = glucosamine-1-phosphate acetyltransferase</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00033" class="description-td" colspan="6">glmUt = N-terminal truncated version of GlmU.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 16</heading> <p num="p-0472">This Example describes integration of one or more copies of the T7lac-ScGNA1 cassette into the chromosome of glucosamine or N-acetylglucosamine production strains.</p>
    <p num="p-0473">Overexpression of the ScGNA1 using vector pET24d(+) resulted in N-acetyl glucosamine production of 24 g l<sup>−1 </sup>after 72 hour culture (Table 1). To avoid the use of 0.25 antibiotics in cultures and to maximize N-acetyl glucosamine production, it was decided to integrate the T7lac-ScGNA1 expression cassette into the chromosome. Since it was not known how many copies of the T7lac-ScGNA1 expression cassette will be optimal for NAG production, it was decided to integrate multiple copies of the cassette into the chromosome. The insertion sites for integration were selected based on the assumption that the targeted genes are not essential to cell growth or N-acetylglucosamine production. Four target sites were selected: manXYZ, ƒucIK, treB and melAB.</p>
    <p num="p-0474">The general strategy and protocols of temperature selection described in 13 were adapted to GNA1 integration in the chromosome. Different temperature sensitive integrative vectors were developed, each containing a temperature sensitive replication origin, an antibiotic selection marker and the T7lac-ScGNA1 expression cassette. For each vector, the T7lac-ScGNA1 expression cassette was isolated from the recombinant plasmid pSW07-62#25 and inserted in a fragment of <i>E. coli </i>DNA sequence cloned from the intended integration site. Integrative vectors were transformed into <i>E. coli </i>host strains and clones with integrated ScGNA1 were selected using the temperature shift procedures.</p>
    <p num="p-0475">The same methods and protocols can be used to integrate other GNA1 homologues from different origins. To those skilled in the art, it is anticipated that modifications and changes may be needed to adapt these methods and protocols to each specific gene. Such modification and changes include, but are not limited to, the use of different restriction sites for cloning and different locations for integration in the chromosome.</p>
    <p num="h-0066">GNA1 Integration at the manXYD Site (One Copy of GNA1):</p>
    <p num="p-0476">The T7lac-ScGNA1 expression cassette was subcloned for integration at the manXYZ site in the chromosome in 7107-18. <i>E. coli </i>manXYD is an operon encoding a complex of three proteins involved in the uptake and phosphorylation of mannose. The operon can be deleted without affecting <i>E. coli </i>growth in medium containing glucose as carbon source. For GNA1 gene integration at the manXYZ site, a plasmid containing the <i>E. coli </i>manXYZ sequence was developed. Primers was synthesized based on the published sequence of <i>E. coli </i>manXYZ (Blattner et al, 1997, Science 277(5331), pp. 1453-1474, herein by reference of its entirety) to amplify the manXYZ operon plus flanking regions from <i>E. coli </i>W3110 genomic DNA using the standard PCR method. The primers used for amplification were forward primer 07-87 and the reverse primer 07-88, which had the following sequences: 07-87: 5′GATGCGGCCGCACTGCAGTAATTACCGCATCCAAC3′ (SEQ ID NO:68) and 07-88: 5′GATGTCGACACCGATTGATGCAGCAAATGCATCC3′ (SEQ ID NO:69).</p>
    <p num="p-0477">Primer 07-87 contains a Not I restriction endonuclease site (GCGGCCGC, represented in nucleotides 4-11 of SEQ ID NO:68) and starts at 905 base pairs upstream of the manX ATG start codon. Primer 07-88 contains a Sal I site (GTCGAC, represented in nucleotides 4-9 of SEQ ID NO:69) and starts from 1010 base pairs downstream from the manZ translational stop codon. PCR was performed using a standard protocol to generate the fragment containing the manXYZ plus flanking regions flanked by Not I and Sal I restriction sites. This fragment was cloned into vector pCR<sup>ò</sup>2.1-TOPO<sup>ò</sup> (Invitrogen, Carlsbad, Calif.), generating plasmid pSW07-65#7.</p>
    <p num="p-0478">To generate a deletion in manXYZ, plasmid pSW07-65#7 was digested with restriction enzyme Hpa I. This released a 2647 bp portion of the plasmid containing most of the coding sequence of manXYZ. Additionally, the T7lac-ScGNA1 fragment was excised from plasmid pSW07-62#25 (previously described in Example 13) using restriction endonuclease Nae I. Plasmid pSW07-62#25 contains Nae I sites at 46 base pairs upstream of the T7 promoter and 164 base pairs downstream of the T7 terminator. The Nae I fragment containing T7lac-ScGNA1 sequence was ligated into the Hpa I sites of pSW07-65#7. As this ligation was a blunt-end ligation, the T7lac-ScGNA1 fragment might ligate into the plasmid in either orientation. Therefore restriction enzyme digestion was used to screen for plasmids with the T7lac-ScGNA1 cassette inserted in the same orientation as manXYZ. The resulting plasmid was designated pSW07-66#25.</p>
    <p num="p-0479">To create a temperature sensitive integrative vector, a fragment containing the temperature sensitive replicon of pMAK705 (Hamilton et al., 1989, J. Bac. 171(9):4617-4622, herein by reference of its entirety) and the kanamycin resistance cassette of plasmid pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.) was excised from plasmid pKLN07-21 using restriction enzymes Not I and Sal I. Plasmid pKLN07-21 was constructed by PCR amplification of the temperature sensitive replicon from plasmid pSW07-4 (described in Example 6), ligation of the PCR product into the vector pPCR-Script<sup>AMP</sup>SK(+) (Stratagene Cloning Systems, La Jolla, Calif.), and addition of the kanamycin resistance cassette from plasmid pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.).</p>
    <p num="p-0480">Restriction enzymes Not I and Sal I were used to excise the fragment containing the T7lac-ScGNA1 plus manXYZ flanking region from plasmid pSW07-66#25. This fragment was then ligated with the Not I/Sal I fragment containing the temperature sensitive replication origin and the kanamycin resistance marker from pKLN07-21, generating plasmid pSW07-68#5. This plasmid had a temperature sensitive replication origin, a kanamycin selection marker and a T7lac-ScGNA1 expression cassette that is flanked by 5′ upstream and 3′ downstream sequences from the manXYZ locus.</p>
    <p num="p-0481">Plasmid pSW07-68#5 was used to generate <i>E. coli </i>strains with T7lac-ScGNA1 integrated at the manXYZ site. As described in U.S. Pat. No. 6,372,457, the manXYZ operon in the strain 7107-18 was mutated by P1 phage transduction. Following transformation of the <i>E. coli </i>strain 7107-18 with pSW07-68#5, the temperature selection protocol described in Example 6 was used to select strains with the T7-lac-ScGNA1 sequence integrated at the manXYZ site. Kanamycin-sensitive strains were screened by PCR for the presence of the T7lac-ScGNA1 cassette at the fuc regulon. Strains were confirmed by Southern hybridization using standard high stringency conditions and the ScGNA1 coding sequence as probe. The resultant strain with one copy of T7lac-ScGNA1 was designated 7107-92.</p>
    <p num="h-0067">Integration of a Second Copy of GNA1 at the ƒucIK Site:</p>
    <p num="p-0482">Integration of a second copy of the T7lac-ScGNA1 cassette was directed to the region of the chromosome encoding for enzymes involved in utilization of L-fucose as an alternative carbon source. The ƒucP (encoding L-fucose permease), ƒucI (encoding L-fucose isomerase), ƒucK (encoding L-fuculose kinase) and ƒucU (unknown protein) genes form an operon involved in L-fucose dissimilation. The ƒucR gene encodes a regulatory protein that activates the L-fucose dissimilation regulon. Integration of the T7lac-ScGNA1 cassette at the fucose operon should not affect the ability of <i>E. coli </i>to grow in medium with glucose as carbon source, nor should it affect its ability to synthesize N-acetyl glucosamine.</p>
    <p num="p-0483">As part of the strategy to develop an integrative vector to target the ƒuc region, primers were synthesized based on published sequence of the fucose regulon (Chen et al., Mol. Gen. Genet. 1987 210:331-337). The ƒucIKU and ƒucR genes were amplified from <i>E. coli </i>W3110 genomic DNA under standard PCR conditions using primers 07-113 and 07-114, which have the following sequences: 07-113: 5′GATGCGGCCGCGCAAGGCAACAGC AAACTGGC-3′ (SEQ ID NO:70) and 07-114: 5′-GATCGGATCCTCAGGCTGTTACCA AAGAAGTTGCAACCTGGC-3′ (SEQ ID NO:71).</p>
    <p num="p-0484">Primer 07-113 contains a Not I restriction endonuclease site (GCGGCCGC, represented in nucleotides 4-11 of SEQ ID NO:70) and amplifies from 824 bp downstream of the ƒucI ATG start codon (represented in nucleotides 12-32 of SEQ ID NO:70). Primer 07-114 contains a BamH I site (GGATCC, represented in nucleotides 5-10 of SEQ ID NO:71) followed by 32 nucleotides of the ƒucR coding sequence starting at its translational stop codon (represented in nucleotides 11-42 of SEQ ID NO:71). PCR was conducted under standard conditions to generate a fragment containing the ƒucIKU and ƒucR sequence flanked by BamH I and Not I restriction endonuclease sites. This fragment was ligated into pPCR-Script Amp SK(+) (Stratagene Cloning Systems, La Jolla, Calif.), generating plasmid pSW07-75. Plasmid pSW07-75 was digested with restriction endonucleases Hpa I and BsrG I, removing a 1239 basepair fragment containing a portion of the ƒucI and ƒucK genes.</p>
    <p num="p-0485">The T7lac-ScGNA1 cassette was amplified by PCR from plasmid pSW07-62#25 using standard conditions. PCR amplification was performed with forward primer 07-115 and reverse primer 07-112, which have the following sequences: 07-115: 5′GATCTGTAC AAGCAACCGCACCTGTGGC3′ (SEQ ID NO:72) and 07-112: 5′GATCAGCGCTA TCCGGATATAGTTCCTCCTTTCAGCAAAAAACCCC3′). (SEQ ID NO:73).</p>
    <p num="p-0486">Primer 07-115 contains a BsrG I site (TGTACA, represented in nucleotides 5-10 of SEQ ID NO:72) and amplifies from 76 basepairs upstream of the T7 promoter sequence of pSW07-62#25. Primer 07-112 contains an Afe I site (AGCGCT, represented in nucleotides 5-10 of SEQ ID NO:73) and amplifies from 25 basepairs downstream of the T7 terminator sequence of pSW07-62#25. The PCR fragment was digested with BsrG I and Afe I and ligated into the BsrG I and Hpa I sites of plasmid pSW07-75, generating plasmid pSW07-76. The recombinant plasmid contained T7lac-ScGNA1 cassette ligated into the site of the ƒucIK deletion.</p>
    <p num="p-0487">To create a temperature sensitive integrative vector, a fragment containing the temperature sensitive replicon of pMAK705 (Hamilton et al., 1989, J. Bac. 171(9):4617-4622, herein by reference of its entirety) and the kanamycin resistance cassette of plasmid pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.) was excised from plasmid pKLN07-21 using restriction enzymes Not I and Kpn I. Plasmid pKLN07-21 was constructed by PCR amplification of the temperature sensitive replicon from plasmid pSW07-4 (described in Example 6), ligation of the PCR product into the vector pPCR-Script™SK(+) (Stratagene Cloning Systems, La Jolla, Calif.), and addition of the kanamycin resistance cassette from plasmid pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.).</p>
    <p num="p-0488">Restriction enzymes Not I and Kpn I were used to excise the fragment containing the T7lacScGNA1 plus ƒuc flanking region from plasmid pSW07-76. This fragment was then ligated with the Not I/Kpn I fragment containing the temperature sensitive replication origin and the kanamycin resistance marker from pKLN07-21, generating plasmid pSW07-77. This plasmid had a temperature sensitive replication origin, a kanamycin selection marker and a T7lac-ScGNA1 expression cassette that is flanked by 5′ upstream and 3′ downstream sequences from the fuc regulon. Plasmid pSW07-77 can be used to direct integration of T7lac-ScGNA1 cassette at the fuc regulon of <i>E. coli. </i> </p>
    <p num="p-0489">Plasmid pSW07-77 was transformed into strain 7107-92#1. The temperature selection protocol was used to select strains with the T7-lac GNA1 sequence integrated at the A ƒucIK site. The resulting strains were designated 7107-607(2), 7107-607(3), and 7107-607(4).</p>
    <p num="h-0068">Integration of a Third Copy of GNA1 at the treB Site:</p>
    <p num="p-0490">Integration of a third copy of the T7-lac-ScGNA1 cassette was directed to the region of the chromosome encoding for enzymes involved in utilization of trehalose as an alternative carbon source. The treB and treC genes, encoding the trehalose transporter and trehalose 6-P hydrolase, respectively form an operon. The treR gene encodes the repressor protein that controls the operon, which is inducible by trehalose-6-phosphate (Horlacher, R., and Boos, W., 1997, J. Biol. Chem., 272(20):13026-13032). As with previous targets, integration of the T7lac-ScGNA1 at the trehalose regulon should not affect the ability of <i>E. coli </i>to grow in medium with glucose as carbon source, nor should it affect its ability to synthesize N-acetyl glucosamine.</p>
    <p num="p-0491">As part of the strategy to develop an integrative vector to target the treB region, primers were synthesized based on published sequence to amplify the treR, treB, and treC genes from <i>E. coli </i>W3110 genomic DNA (Blattner et al., 1997, Science 227(5331):1453-1474). PCR amplification was performed using primers 07-117 and 07-118, which have the following sequences: 07-117: 5′ GAGCGGCCGCATGCAAAATCGGCTGACCATC3′ (SEQ ID NO:74) and 07-118: 5′ GATCGGGCCCTTACTTCTGTAACCACCAGACAG CCTC3′ (SEQ ID NO:75).</p>
    <p num="p-0492">Primer 07-117 contains a Not I restriction endonuclease site (GCGGCCGC, represented in nucleotides 3-10 of SEQ ID NO:74) followed by 21 nucleotides of the treR coding sequence from its ATG start codon (represented in nucleotides 11-31 of SEQ ID NO:74). Primer 07-118 contains an Apa I restriction endonuclease site (GGGCCC, represented in nucleotides 5-10 of SEQ ID NO:75) followed by 27 nucleotides of the treC coding sequence starting at its translational stop codon (represented in nucleotides 11-37 of SEQ ID NO:75). PCR amplification was conducted using a standard protocol to generate a fragment of DNA containing the treR, treB and treC genes flanked by Not I and Apa I restriction sites. The 4.2 kb PCR product was ligated into plasmid pPCR-Script™SK(+) (Stratagene Cloning Systems, La Jolla, Calif.), creating plasmid pSW07-78#20.</p>
    <p num="p-0493">Plasmid pSW07-78#20 was digested with restriction endonuclease Bgl II and treated with T4 DNA polymerase under standard conditions to blunt the ends of the fragment. The blunted fragment was next digested with restriction endonuclease BsrG I. This double digestion with Bgl I and BsrG I removed a 130 basepair region of the treB coding sequence from plasmid pSW07-78#20, leaving the plasmid fragment with one sticky end (BsrG I) and one blunt end.</p>
    <p num="p-0494">The next step was to ligate the T7lac-ScGNA1 cassette into the BsrG I and Bgl II (filled) site of plasmid pSW07-78#20. To accomplish this, the T7lac-ScGNA1 cassette was amplified by PCR from plasmid pSW07-62#25 under standard conditions using forward primer 07-115 (SEQ ID NO:72) and reverse primer 07-112 (SEQ ID NO:73). Primer 07-115 contains a BsrG I site (TGTACA, represented in nucleotides 5-10 of SEQ ID NO:72) and amplifies from 76 basepairs upstream of the T7 promoter sequence of pSW07-62#25. Primer 07-112 contains an Afe I site (AGCGCT, represented in nucleotides 5-10 of SEQ ID NO:73) and amplifies from 25 basepairs downstream of the T7 terminator sequence of pSW07-62#25. The PCR fragment was digested with BsrG I and Afe I and ligated into the BsrG I and Bgl II (filled) site of plasmid pSW07-78#20, generating plasmid pSW07-83.</p>
    <p num="p-0495">To create a temperature sensitive integrative vector, a fragment containing the temperature sensitive replicon of pMAK705 (Hamilton et al., 1989, J. Bac. 171(9):4617-4622, herein by reference of its entirety) and the kanamycin resistance cassette of plasmid pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.) was excised from plasmid pKLN07-21 using restriction enzymes Not I and Apa I.</p>
    <p num="p-0496">Restriction enzymes Not I and Apa I were used to excise the fragment containing the T7lac-ScGNA1 plus tre flanking region from plasmid pSW07-83. This fragment was ligated with the Not I-Apa I fragment from plasmid pKLN07-21 containing the temperature sensitive replication origin and the kanamycin resistance marker, resulting in plasmid pSW07-84. Plasmid pSW07-84 can be used to direct integration of T7-lac-ScGNA1 cassette at the treB of <i>E. coli. </i> </p>
    <p num="p-0497">Plasmid pSW07-84#1 was transformed into strains 7107-607(2), 7107-607(3), and 7107-607(4). The temperature selection protocol was used to select strains with the T7-lac GNA1 sequence integrated at the treB site. Kanamycin-sensitive colonies were screened by PCR for the presence of the T7lac-ScGNA1 cassette at treB of the chromosome. Strains were confirmed by Southern hybridization using standard high stringency conditions and the ScGNA1 coding sequence as probe as having three copies of the T7lac-ScGNA1 cassette integrated in the chromosome. These strains were designated 7107-608(1) and 7107-608(2).</p>
    <p num="h-0069">Integration of a Fourth Copy of GNA1 at the melRAB Site:</p>
    <p num="p-0498">Integration of a fourth copy of the T7lac-ScGNA1 cassette into NAG production strains was targeted to the melR and melAB region of the chromosome. In <i>E. coli</i>, this region encodes proteins involved in the melibiose uptake and hydrolysis. The melA and melB genes, encoding alpha galactosidase and melibiose permease II, respectively, form an operon. The divergently transcribed melR gene encodes the regulator of the melibiose operon. As with previous targets for integration, integration of the T7lac-ScGNA1 cassette at me/AB of the genome should not affect the ability of <i>E. coli </i>to grow in medium with glucose as carbon source, nor should it affect its ability to synthesize N-acetyl glucosamine.</p>
    <p num="p-0499">As part of the strategy to develop an integrative vector to target the melAB region, primers 07-122 and 07-123 were synthesized based on published sequence of the melR, melA and melB genes of <i>E. coli </i>(Blattner et al, 1997, Science 277(5331), pp.1453-1474). PCR was conducted under standard conditions to amplify a fragment containing the melR, melA and melB genes from <i>E. coli </i>W3110 genomic DNA. Forward primer 07-122 and reverse primer 07-123 have the following sequences: 07-122: 5′GATGCGGCCGCTTAGCC GGGAAACGTCTGGCGGC3′ (SEQ ID NO:76) and 07-123: 5′GATCGTCGACTCAGG CTTTCACATCACTCACTGCACC3′ (SEQ ID NO:77).</p>
    <p num="p-0500">Primer 07-122 contains a Not I restriction endonuclease site (GCGGCCGC, represented in nucleotides 4-11 of SEQ ID NO:76) followed by 23 nucleotides of the melR coding sequence starting from the translational stop codon (represented in nucleotides 12-34 of SEQ ID NO:76). Primer 07-123 contains a Sal I restriction endonuclease site (GTCGAC, represented in nucleotides 5-10 of SEQ ID NO:77) followed by 27 nucleotides of the melB coding sequence starting from the translational stop codon (represented in nucleotides 11-37 of SEQ ID NO:77). The PCR fragment containing the melR and melAB coding sequences flanked by Not I and Sal I restriction endonuclease sites was ligated into vector pPCR-Script Amp SK (+) (Stratagene), generating plasmid pSW07-81#5.</p>
    <p num="p-0501">The next step in vector construction was to ligate the T7lac-ScGNA1 cassette into the melAB region of plasmid pSW07-81#5. Plasmid pSW07-81#5 was digested with restriction endonucleases Bgl II and AsiS I, removing a 1676 basepair fragment containing the entire melA coding sequence and the first 199 nucleotides of the melB coding sequence.</p>
    <p num="p-0502">The T7lac-ScGNA1 cassette was amplified by PCR from plasmid pSW07-62#25 under standard conditions with forward primer 07-124 and reverse primer 07-125, which have the following sequences: 07-124: 5′GATGGATCCAGCAACCGCACCTGTGGC3′ (SEQ ID NO:78) and 07-125: 5′GATGCGATCGCTATAGTTCCTCCTTTCAGCAAAA AACCC3′ (SEQ ID NO:79)</p>
    <p num="p-0503">Primer 07-124 contains a BamH I site (GGATCC, represented in nucleotides 4-9 of SEQ ID NO:78) and amplifies from 76 nucleotides upstream of the T7 promoter of plasmid pSW07-62#25. Primer 07-125 contains an AsiS I site (GCGATCGC, represented in nucleotides 4-11 of SEQ ID NO:79) and amplifies from 18 nucleotides downstream of the T7 terminator of plasmid pSW07-62#25. The PCR product containing the T7lac-ScGNA1 was digested with restriction endonucleases BamH I and AsiS I and ligated with the Bgl II and AsiS I fragment from pSW07-81#5, generating plasmid pSW07-82. Plasmid pSW07-82 therefore contained the mel genes in vector pPCR-Script (AmpSK(+), with the 1676 basepair melAB region replaced with the T7lac-ScGNA1 cassette.</p>
    <p num="p-0504">To create a temperature sensitive integrative vector, a fragment containing the temperature sensitive replicon of pMAK705 (Hamilton et al., 1989, J. Bac. 171(9):4617-4622, herein by reference of its entirety) and the kanamycin resistance cassette of plasmid pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.) was excised from plasmid pKLN07-21 using restriction enzymes Not I and Sal I.</p>
    <p num="p-0505">Plasmid pSW07-82 was digested with restriction endonucleases Not I and Sal I isolate the fragment containing the mel genes interrupted with the T7lac-ScGNA1 cassette. This fragment was then ligated with the Not I and Sal I fragment from plasmid pKLN07-21 containing the temperature sensitive replication origin and the kanamycin resistance marker. The resulting plasmid pSW07-84 can be used to direct integration of T7-lac-ScGNA1 cassette at the melAB of <i>E. coli. </i> </p>
    <p num="p-0506">Plasmid pSW07-84 was transformed into strains 7107-608(1) and 7107-608(2). The temperature selection protocol described in Example 6 was used to select strains with the T720 lac GNA1 sequence integrated at the melAB site. Kanamycin-sensitive colonies were screened by PCR for the presence of the T7lac-ScGNA1 cassette at melAB of the chromosome. Strains were confirmed by Southern hybridization using standard high stringency conditions with the ScGNA1 coding sequence as probe as having four copies of the T7lac-ScGNA1 cassette integrated in the chromosome. These resulting strains, derived from 7107-608(1) and 7107-608(2), were designated 7107-612 and 7107-613, respectively.</p>
    <p num="p-0507">Plasmid pSW07-84 was also transformed into strains 7107-607(2), 7107-607(3), and 7107-607(4) to add a third copy of the T7lac-ScGNA1 to the chromosome. Strains were screened as above and confirmed by Southern hybridization using standard high stringency conditions with the ScGNA1 coding sequence as probe as having three copies of the T7lac-ScGNA1 cassette integrated in the chromosome. These strains, derived from strains 7107-607(2), 7107-607(3), and 7107-607(4), were designated as 7107-609, 7107-610 and 611, respectively</p>
    <p num="h-0070">Effects of GNA1 Gene Copy Number on GNA1 Expression Levels and N-Acetylglucosamine Production</p>
    <p num="p-0508">To evaluate the effect of gene copy number on GNA1 expression levels and N-acetylglucosamine production, strains with varying copy numbers of the integrated T7lac-ScGNA1 cassette were tested in shake flask. Samples were taken to analyze GNA1 protein expression by enzyme activity assay and determination of NAG titers.</p>
    <p num="p-0509">Shake flask Screen 53 was conducted to evaluate the effect of copy number of the integrated T7-lac-ScGNA1 cassette on glucosamine synthase (GlmS) activity, glucosamine-6-phosphate acetyltransferase (GNA1) activit were grown in M9B medium (previously described) supplemented with supplemented with 0.6 g l<sup>−1 </sup>MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g l<sup>−1 </sup>CaCl<sub>2</sub>-2H<sub>2</sub>O, 10 g l<sup>−1 </sup>glucose, 40 g l<sup>−1 </sup>lactose, 5 g l<sup>−1 </sup>ribose and 5 g l<sup>−1 </sup>yeast extract. Cultures were grown at 30° C. for the first 24 hours and then placed at 25° C. At 24 and 48-hour timepoints, the pH of the cultures was adjusted to 7.2, glucose was added to each flask to 30 g l<sup>−1 </sup>per day total based on HPLC results, and ammonium sulfate additions of 5 g l<sup>−1 </sup>were made to flasks with ammonium levels below 1 g l<sup>−1</sup>. Samples were removed at 24, 48 and 72 hours to evaluate NAG production and enzyme assays. All strains tested grew to comparable OD<sub>600 </sub>and had comparable glucosamine synthase activity (Table 18). There was a notable correlation between glucosamine-6-phosphate acetyltransferase activity and T7-lac-ScGNA1 copy number, with the strains containing three copies having the highest specific activity. However, this increased specific activity did not result in increased NAG production in the shake flask experiment.</p>
    <p num="p-0510"> <tables id="TABLE-US-00024" num="00024"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 18</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Comparison of growth, enzyme activity, and GlcNAc</td> </tr> <tr class="description-tr"> <td class="description-td">production in strains with multiple copies of the T7-lac-</td> </tr> <tr class="description-tr"> <td class="description-td">ScGNA1 cassette integrated in the chromosome.</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="42pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="70pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">GNA1</td> <td class="description-td"> </td> <td class="description-td">Enzyme Activity</td> <td class="description-td">GlcNAc</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain</td> <td class="description-td">copy number</td> <td class="description-td">A<sub>600</sub> </td> <td class="description-td">GlmS </td> <td class="description-td">GNA1</td> <td class="description-td">(g l<sup>−1</sup>)</td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td> </tr> <tr class="description-tr"> <td class="description-td">7107-92(1)</td> <td class="description-td">1</td> <td class="description-td">12.0</td> <td class="description-td">0.59</td> <td class="description-td">3.0</td> <td class="description-td">30.0</td> </tr> <tr class="description-tr"> <td class="description-td">7107-607(2)</td> <td class="description-td">2</td> <td class="description-td">12.0</td> <td class="description-td">0.59</td> <td class="description-td">5.8</td> <td class="description-td">28.8</td> </tr> <tr class="description-tr"> <td class="description-td">7107-608(1)</td> <td class="description-td">3</td> <td class="description-td">12.8</td> <td class="description-td">0.55</td> <td class="description-td">9.1</td> <td class="description-td">29.3</td> </tr> <tr class="description-tr"> <td class="description-td">7107-608(2)</td> <td class="description-td">3</td> <td class="description-td">13.5</td> <td class="description-td">0.43</td> <td class="description-td">7.9</td> <td class="description-td">29.7</td> </tr> <tr class="description-tr"> <td class="description-td">7107-609(1)</td> <td class="description-td">3</td> <td class="description-td">11.3</td> <td class="description-td">0.37</td> <td class="description-td">8.2</td> <td class="description-td">24.7</td> </tr> <tr class="description-tr"> <td class="description-td">7107-610(1)</td> <td class="description-td">3</td> <td class="description-td">12.0</td> <td class="description-td">0.40</td> <td class="description-td">8.4</td> <td class="description-td">26.3</td> </tr> <tr class="description-tr"> <td class="description-td">7107-611(1)</td> <td class="description-td">3</td> <td class="description-td">12.0</td> <td class="description-td">0.44</td> <td class="description-td">7.4</td> <td class="description-td">26.4</td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00034" class="description-td" colspan="6">1) Enzyme activities and N-acetyl glucosamine levels were determined from the 72 hour timepoint.</td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00035" class="description-td" colspan="6">2) Enzyme activity is expressed in μmol min<sup>−1 </sup>mg<sup>−1 </sup>protein.</td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00036" class="description-td" colspan="6">3) Numbers in parentheses indicate different sibling.</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Evaluation of Strains with Varying Copy Numbers of the Integrated T7lac-ScGNA1 Cassette in 1 Liter Fermentors
</p>
    <p num="p-0511">Strains 7107-92(1), 7107-607(2), 7107-608(2), and 7107-612(1), containing the integrated T7lac-ScGNA1 cassette at copy numbers ranging from 1 to 4, respectively, were evaluated in 1-L fermenters. Fermenters 237-240 were set up with an initial volume of 475 ml. Components of the fermentation medium are listed in Table 19. Fermentations were run using 75% NH<sub>4</sub>OH to control pH at 6.9. Temperature was maintained at 37° C. throughout the fermentation. Aeration and agitation were adjusted to maintain a dissolved oxygen concentration of 20% of air saturation. 65% glucose was fed to the cultures with feed rate controlled by computer program to achieve a growth rate of 0.40 hr<sup>−1 </sup>at inoculation and a maximum rate of 5 ml hr<sup>−1 </sup>by 6 hours. Cultures were induced with food grade lactose added at 5 g l<sup>−1 </sup>at 10 hours, with continued glucose feed.</p>
    <p num="p-0512">
      <tables id="TABLE-US-00025" num="00025"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 19</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Fermentation medium used to test</td>
              </tr> <tr class="description-tr"> <td class="description-td">effects of GNA1 copy number on N-</td>
              </tr> <tr class="description-tr"> <td class="description-td">acetylglucosamine production.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="35pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="119pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Component</td>
                <td class="description-td">Amount (g l<sup>−1</sup>)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="35pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="119pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H<sub>3</sub>PO<sub>4</sub> </td>
                <td class="description-td">4.79</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">KOH</td>
                <td class="description-td">3.15</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Citric acid-H<sub>2</sub>O</td>
                <td class="description-td">3.56</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> </td>
                <td class="description-td">5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">MgSO<sub>4</sub>-7 H<sub>2</sub>O</td>
                <td class="description-td">2.5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CaCl<sub>2</sub>-2 H<sub>2</sub>O</td>
                <td class="description-td">0.05</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Trace Metals</td>
                <td class="description-td">*</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Mazu 204 Antifoam</td>
                <td class="description-td">0.25</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="left" id="FOO-00037" class="description-td" colspan="3">*Trace metal composition is 5 m l<sup>−1 </sup>FeSO<sub>4</sub>-7H<sub>2</sub>O, 3.75 mg l<sup>−1</sup>, ZnSO<sub>4</sub>-7H<sub>2</sub>O, 0.6 mg l<sup>−1 </sup>MnSO<sub>4</sub>-H<sub>2</sub>O, 0.1002 mg l<sup>−1 </sup>CuSO4-5H<sub>2</sub>O and 0.1002 mg l<sup>−1 </sup>CoCl<sub>2</sub>-6H<sub>2</sub>O.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0513">Results of the fermentation are summarized as follows. The copy number of the GNA1 cassette has a slight affect on NAG levels. By the end of the fermentation, strain 7107-608(2), containing three copies of the T7lac-ScGNA1 cassette, produced 16% more NAG than the strain with one copy and 5% more NAG than the strain with two copies of the cassette. In this experiment, addition of the fourth copy of the T7lac-ScGNA1 did not achieve an improvement over strain 7107-608(2). Nonetheless, the utility of increasing the T7-lac-ScGNA1 copy number in production strains has been validated.</p>
    <heading>Example 17</heading> <p num="p-0514">The following Example describes the effects of phosphorylated sugars on glucosamine synthase.</p>
    <p num="p-0515">Glucosamine synthetase activity was examined in the presence of various phosphorylated sugars. Crude enzyme extracts was prepared from cells of 7017-18 grown in shake flasks for 24 hours with lactose induction. Results are summarized in Table 20 below. The data indicates that previously observed glucosamine-6-P showed strong inhibition on glucosamine synthetase at relatively high concentration. Glucosamine-1-P also inhibited the enzyme at 10 mM. No inhibition was observed with the N-acetylglucosamine phosphates.</p>
    <p num="p-0516">
      <tables id="TABLE-US-00026" num="00026"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 20</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Effects of different phosphorylated</td>
              </tr> <tr class="description-tr"> <td class="description-td">sugars on the <i>E. coli </i>glucosamine synthase</td>
              </tr> <tr class="description-tr"> <td class="description-td">GlmS*54</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="105pt" align="left"> </colspec> <colspec colname="2" colwidth="84pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Addition</td>
                <td class="description-td">Activity (%)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="105pt" align="left"> </colspec> <colspec colname="2" colwidth="84pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">control</td>
                <td class="description-td">100</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">glucosamine-6-P (10 mM)</td>
                <td class="description-td">70</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">glucosamine-6-P (20 mM)</td>
                <td class="description-td">46</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">glucosamine-1-P (10 mM)</td>
                <td class="description-td">79</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">N-acetylglucosamine-6-P (10 mM)</td>
                <td class="description-td">100</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">N-acetylglucosamine-6-P (20 mM)</td>
                <td class="description-td">100</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">N-acetylglucosamine-1-P (10 mM)</td>
                <td class="description-td">100</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 18</heading> <p num="p-0517">This Example describes the biochemical effects of phosphorylated sugars on other enzymes relevant to N-acetylglucosamine synthesis.</p>
    <p num="p-0518">Previous discussions on potential toxic effects of phosphorylated sugars (compounds such as glucosamine-6-P) have been limited to enzymes directly involved in metabolism of amino sugars. However, the general phenomenon of sugar toxicity has long been observed in various mutants impaired in sugar metabolism. This has generally been attributed to build up of abnormally high levels of phosphorylated sugar intermediates which inhibit one or more enzyme targets and metabolically poison the cell. Product inhibition of GlmS is one example of this.</p>
    <p num="p-0519"> <figref idrefs="DRAWINGS">FIG. 3</figref> suggests a number of possibilities for other possible targets. A paper (J. Bacteriol. 101:384. 1970) dealing with mutants impaired in amino sugar metabolism describes this phenomenon and shows that pentose sugars can reverse the inhibition. Therefore, effects of glucosamine-6-P and N-acetylglucosamine-6-P on several of the enzymes were examined.</p>
    <p num="p-0520">Glucosamine-6-P is reported as an inhibitor of Pgi in publication (Arch. Biochem. Biophys. 64: 489. 1956). <figref idrefs="DRAWINGS">FIG. 15</figref> shows Pgi (phosphoglucoisomerase inhibition by two amino sugars). Inhibition is observed with both compounds, but significantly less so with N-acetylglucosamine-6-P.</p>
    <p num="p-0521">Glucosamine-6-P is reported as an inhibitor of phosphoglucoisomerase (J. Biol. Chem. 216:67. 1955). <figref idrefs="DRAWINGS">FIG. 16</figref> shows effects of phosphorylated amino sugars on glucose-6-P dehydrogenase (zwf), the entry point for glucose-6-P into the pentose phosphate pathway. Here again glucosamine-6-P appears to be a more potent inhibitor of the enzyme than N-acetylglucosamine-6-P. Similar trends were seen with phosphoglucomutase (Pgm).</p>
    <p num="p-0522">The inhibitory effect of glucosamine-6-P on the above mentioned and possibly other enzymes involved in carbohydrate metabolism may certainly be an explanation for the apparent ceiling on glucosamine productivity observed in 7017-18. Presumably, high concentrations of glucosamine-6-P interfere with activity of several enzymes. Addition of the acetyltransferase (GNA1) to the pathway presumably leads to much lower intracellular levels of glucosamine-6-P. This certainly could be the primary reason for increased productivity along with the enhanced stability of N-acetylglucosamine.</p>
    <p num="p-0523">N-acetylglucosamine-6-P really does not significantly inhibit Zwf or Pgi in the in vitro assays performed. Positive effects of ribose and gluconate on cell growth and N-acetylglucosamine synthesis suggest that one or more steps in the pentose phosphate pathway are affected by phosphorylated amino sugars. On the other hand, glucosamine N-acetyltransferase did not show any significant product inhibition by N-acetylglucosamine-6-P.</p>
    <heading>Example 19</heading> <p num="p-0524">This Example describes enzyme activities during N-acetylglucosamine production in fermentors.</p>
    <p num="p-0525">Various enzyme activities relevant to N-acetylglucosamine production were examined in fermentors. Enzymes assayed were glucosamine synthase (GlmS), glucosamine N-acetyltransferase (GNA1) and glucose-6-phosphate dehydrogenase, which is a key enzyme in pentose phosphate pathway (see <figref idrefs="DRAWINGS">FIG. 3</figref>).</p>
    <p num="h-0074">Strains with GNA1 Plasmids (Fermentor #102):</p>
    <p num="p-0526">Relevant enzyme activity was examined in samples of 7017-87(25) from fermentations 102. This strain contains the acetyltransferase gene construct on plasmids. This run produced 80 g/l N-acetylglucosamine. Results of enzyme activities and N-acetylglucosamine concentrations are shown in <figref idrefs="DRAWINGS">FIG. 17</figref>. High acetyltransferase activity was observed soon after induction and it remained high throughout the run. Interestingly, acetyltransferase activity was present even at 90+hours, while by this time glucosamine synthetase activity had disappeared. N-acetylglucosamine production essentially ceased at around 70 hours. Glucose-6-P dehydrogenase activity was constant throughout the run.</p>
    <p num="h-0075">Strains with Integrated GNA1 Constructs (Fermentors #121-128).</p>
    <p num="p-0527">These runs used <i>E. Coli </i>strain 7017-92(1) containing the integrated acetyltransferase construct. Fermentation variables examined were amounts of iron added in the medium, extra iron feeding, and levels of phosphate buffer (1×=40 g l<sup>−1</sup>). Enzyme activities are summarized in Table 21. Except for the lowest iron level used in fermentation 121, glucosamine synthetase and acetyltransferase activities were very high throughout the experiment in the other seven fermentors. As previously observed, acetyltransferase activity tends to remain at a high level. Glucosamine synthetase activity was clearly adequate at the other iron levels (5-20 PPM) examined with or without additional iron feed. Lower iron gave higher N-acetylglucosamine activity but lower glucosamine synthetase activity. This decreased activity was still adequate for high N-acetylglucosamine production</p>
    <p num="p-0528">
      <tables id="TABLE-US-00027" num="00027"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="259pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 21</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Enzyme activities in N-acetylglucosamine fermentation</td>
              </tr> <tr class="description-tr"> <td class="description-td">(fermentors #121 through 128)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="42pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="63pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GlmS</td>
                <td class="description-td">GNA1</td>
                <td class="description-td">[NAG]</td>
                <td class="description-td">Iron</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="84pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <colspec colname="7" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Sample</td>
                <td class="description-td">Hours</td>
                <td class="description-td">(μmol min<sup>−1</sup>, mg<sup>−1 </sup>protein)</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
                <td class="description-td">(mg l<sup>−1</sup>)</td>
                <td class="description-td">Fe Feed</td>
                <td class="description-td">Phosphate</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="28pt" align="center"> </colspec> <colspec colname="8" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">121-2</td>
                <td class="description-td">24</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.0</td>
                <td class="description-td">7.1</td>
                <td class="description-td">2.5</td>
                <td class="description-td">+</td>
                <td class="description-td">1X</td>
              </tr> <tr class="description-tr"> <td class="description-td">121-5</td>
                <td class="description-td">46</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.5</td>
                <td class="description-td">23</td>
              </tr> <tr class="description-tr"> <td class="description-td">121-7</td>
                <td class="description-td">70</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.6</td>
                <td class="description-td">29</td>
              </tr> <tr class="description-tr"> <td class="description-td">122-2</td>
                <td class="description-td">24</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.0</td>
                <td class="description-td">6.0</td>
                <td class="description-td">5</td>
                <td class="description-td">+</td>
                <td class="description-td">1X</td>
              </tr> <tr class="description-tr"> <td class="description-td">122-5</td>
                <td class="description-td">46</td>
                <td class="description-td">0.43</td>
                <td class="description-td">5.7</td>
                <td class="description-td">56</td>
              </tr> <tr class="description-tr"> <td class="description-td">122-7</td>
                <td class="description-td">70</td>
                <td class="description-td">0.40</td>
                <td class="description-td">6.3</td>
                <td class="description-td">88</td>
              </tr> <tr class="description-tr"> <td class="description-td">122-8</td>
                <td class="description-td">95</td>
                <td class="description-td">0.18</td>
                <td class="description-td">5.5</td>
                <td class="description-td">88</td>
              </tr> <tr class="description-tr"> <td class="description-td">123-2</td>
                <td class="description-td">24</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.0</td>
                <td class="description-td">5.7</td>
                <td class="description-td">10</td>
                <td class="description-td">+</td>
                <td class="description-td">1X</td>
              </tr> <tr class="description-tr"> <td class="description-td">123-5</td>
                <td class="description-td">46</td>
                <td class="description-td">0.81</td>
                <td class="description-td">7.5</td>
                <td class="description-td">49</td>
              </tr> <tr class="description-tr"> <td class="description-td">123-7</td>
                <td class="description-td">70</td>
                <td class="description-td">0.99</td>
                <td class="description-td">10</td>
                <td class="description-td">77</td>
              </tr> <tr class="description-tr"> <td class="description-td">124-2</td>
                <td class="description-td">24</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.0</td>
                <td class="description-td">6.0</td>
                <td class="description-td">20</td>
                <td class="description-td">+</td>
                <td class="description-td">1X</td>
              </tr> <tr class="description-tr"> <td class="description-td">124-5</td>
                <td class="description-td">46</td>
                <td class="description-td">0.81</td>
                <td class="description-td">5.4</td>
                <td class="description-td">50</td>
              </tr> <tr class="description-tr"> <td class="description-td">124-7</td>
                <td class="description-td">70</td>
                <td class="description-td">0.96</td>
                <td class="description-td">6.5</td>
                <td class="description-td">76</td>
              </tr> <tr class="description-tr"> <td class="description-td">125-2</td>
                <td class="description-td">24</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.0</td>
                <td class="description-td">5.8</td>
                <td class="description-td">5</td>
                <td class="description-td">−</td>
                <td class="description-td">1X</td>
              </tr> <tr class="description-tr"> <td class="description-td">125-5</td>
                <td class="description-td">46</td>
                <td class="description-td">0.43</td>
                <td class="description-td">5.2</td>
                <td class="description-td">57</td>
              </tr> <tr class="description-tr"> <td class="description-td">125-7</td>
                <td class="description-td">70</td>
                <td class="description-td">0.30</td>
                <td class="description-td">5.7</td>
                <td class="description-td">92</td>
              </tr> <tr class="description-tr"> <td class="description-td">126-2</td>
                <td class="description-td">24</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.0</td>
                <td class="description-td">7.0</td>
                <td class="description-td">10</td>
                <td class="description-td">−</td>
                <td class="description-td">1X</td>
              </tr> <tr class="description-tr"> <td class="description-td">126-5</td>
                <td class="description-td">46</td>
                <td class="description-td">0.57</td>
                <td class="description-td">3.7</td>
                <td class="description-td">68</td>
              </tr> <tr class="description-tr"> <td class="description-td">126-7</td>
                <td class="description-td">70</td>
                <td class="description-td">1.0</td>
                <td class="description-td">5.5</td>
                <td class="description-td">83</td>
              </tr> <tr class="description-tr"> <td class="description-td">127-2</td>
                <td class="description-td">24</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.0</td>
                <td class="description-td">4.3</td>
                <td class="description-td">20</td>
                <td class="description-td">−</td>
                <td class="description-td">1X</td>
              </tr> <tr class="description-tr"> <td class="description-td">127-5</td>
                <td class="description-td">46</td>
                <td class="description-td">0.61</td>
                <td class="description-td">5.1</td>
                <td class="description-td">49</td>
              </tr> <tr class="description-tr"> <td class="description-td">127-7</td>
                <td class="description-td">70</td>
                <td class="description-td">0.80</td>
                <td class="description-td">5.7</td>
                <td class="description-td">78</td>
              </tr> <tr class="description-tr"> <td class="description-td">128-2</td>
                <td class="description-td">24</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.0</td>
                <td class="description-td">7.0</td>
                <td class="description-td">10</td>
                <td class="description-td">+</td>
                <td class="description-td">  0.2X</td>
              </tr> <tr class="description-tr"> <td class="description-td">128-5</td>
                <td class="description-td">46</td>
                <td class="description-td">0.49</td>
                <td class="description-td">7.7</td>
                <td class="description-td">44</td>
              </tr> <tr class="description-tr"> <td class="description-td">128-7</td>
                <td class="description-td">70</td>
                <td class="description-td">0.63</td>
                <td class="description-td">10</td>
                <td class="description-td">69</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="left" id="FOO-00038" class="description-td" colspan="8">Note:</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="left" id="FOO-00039" class="description-td" colspan="8">1)Levels of iron in the medium are amounts of feSO<sup>4</sup>-7H<sub>2</sub>O (mg 1<sup>−1</sup>).</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="left" id="FOO-00040" class="description-td" colspan="8">2)Iron feed was provided in glucose at 5 μg FeSO<sub>4</sub>-7 H<sub>2</sub>O per g glucose.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="left" id="FOO-00041" class="description-td" colspan="8">3)Phosphate levels: 1X =40 g l<sup>−1 </sup>potassium phosphate.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 20</heading> <p num="p-0529">This example describes metabolic engineering of <i>Saccharomyces cerevisiae </i>for the production of glucosamine and N-acetylglucosamine. Particularly, genes encoding for product-resistant glucosamine synthase, <i>E. coli </i>glmS*54 and <i>Bacillus subtilis </i>glmS, and <i>S. cerevisiae </i>GNA1 gene encoding for glucosamine-6-phosphate N-acetyltransferase were cloned into yeast expression vectors and introduced into the yeast for over-expression.</p>
    <p num="p-0530">The major elements described in previous examples for glucosamine and N-acetylglucosamine production is the over-expression of a product-resistant GlmS enzyme and a GNA1 enzyme. In hosts having a native GNA1 gene such as <i>S. cerevisiae </i>and <i>Candida albicans</i>, over-expressing a product resistant GlmS could lead to an increased level of N-acetylglucosamine production. However, the main products of amino sugars could be glucosamine or N-acetylglucosamine. To produce N-acetylglucosamine as the main product, GNA1 gene needs to be over-expressed.</p>
    <p num="p-0531">Glucosamine degradation is a bottleneck in attempts to produce glucosamine at neutral fermentation pH. Since some yeast and bacteria such <i>S. cerevisiae </i>are adapted to relatively low pH and grow normally in the pH range of 4-5 where glucosamine is stable, over-expressing a product-resistant GlmS enzyme in this type of host could lead to the development of a commercially viable process for direct production of glucosamine. Moreover, since <i>S. cerevisiae </i>is a GRAS organism and it does not produce endotoxins, it maybe a preferred fermentation host to produce glucosamine/N-acetylglucosamine for some applications of the products.</p>
    <p num="h-0077">Cloning of the <i>E. coli </i>Mutant glmS*54 Gene for Expressing in Yeast</p>
    <p num="p-0532">The <i>E. coli </i>mutant glmS*54 gene was cloned into expression vector yEp352-ADH1. This vector is derived from yEp352 (Hill et al., 1986 Yeast 2:163-167) using standard techniques. It replicates to multiple copies per yeast cell and it contains the alcohol dehydrogenase (ADH) promoter and terminator. The vector has an ampicillin resistance marker for selection in <i>E. coli </i>and an URA3 marker for selection in yeast.</p>
    <p num="p-0533">Forward primers nMD7107-021 and reverse primer nMD7107-022 were designed for PCR amplification of the coding sequence of the <i>E. coli </i>mutant glmS*54 gene. Restriction sites were incorporated at the ends to facilitate cloning. The glmS*54 coding sequence was amplified by PCR under standard conditions using forward primer nMD7107-021 (Sac I) and reverse primer nMD7107-022(Hind II) that have the following sequences: nMD7107-021 (Sac I): 5′-AGCTGAGCTCATGTGTGGAATTGTTGGCGCGA-3′ (SEQ ID NO: 80) and nMD7107-022(Hind I): 5′-TACGAAGCTTACTCAACCGTAACCGATTTTGC-3′ (SEQ ID NO:81). Primer nMD7107-021 contains 22 nucleotides of the glmS*54 coding sequence represented in nucleotides 11-32 of SEQ ID NO:80, and primer nMD7107-022 contains 24 nucleotides of the glmS*54 coding sequence represented in nucleotides 9-32 of SEQ ID NO:81.</p>
    <p num="p-0534">Plasmids pKLN23-54 containing the <i>E. coli </i>glmS*54 gene (described in U.S. Pat. No. 6,372,457) were used as the DNA templates in PCR reactions. A single band of PCR products of the expected size was generated under standard PCR conditions using the Taq polymerase. The PCR products were digested with restriction enzymes Sac I and Hind III, purified through agarose-gel and cloned into the yEP352-ADH-1 vector that was predigested with the same enzymes. DNA ligation products were transformed into <i>E. coli </i>Top 10 cells (obtained from Invitrogen Life Technologies, Carlsbad, Calif.) on ampicillin selection. First, pools of colonies (10 per pool) were screened by PCR using the forward and reverse primers. Then, individual clones in positive pools were identified by PCR and confirmed by restriction digestions. Recombinant plasmids MD7107-238 and MD7107-239 contained the <i>E. coli </i>glmS*54 gene in the yeast expression cassette with the ADH promoter and terminator.</p>
    <p num="p-0535">The recombinant plasmids were transformed into cells of <i>S. cerevisiae </i>SWY5 following the LiOAc method described by Geitz et al. (1995). The yeast strain has the ura and his auxotrophic selection markers. Yeast transformants were selected on plates of SCE-minus medium supplemented with L-histidine at 20 mg l<sup>−1 </sup>(Table 22). Transformed yeast cell lines will be analyzed to determine GlmS and GNA1 activities, and levels of glucosamine and N-acetylglucosamine.</p>
    <p num="p-0536"> <tables id="TABLE-US-00028" num="00028"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 22</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC-minus medium for yeast growth</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="105pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Component</td> <td class="description-td">Amount (g l<sup>−1</sup>)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="105pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Yeast Nitrogen base without</td> <td class="description-td">6.7</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">amino acid (YNB)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">glucose</td> <td class="description-td">20</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L-arginine</td> <td class="description-td">0.02</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L-methionine</td> <td class="description-td">0.02</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L-tyrosine</td> <td class="description-td">0.03</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L-isoleucine</td> <td class="description-td">0.03</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L-lysine</td> <td class="description-td">0.03</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L-phenylalanine</td> <td class="description-td">0.05</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L-glutamate</td> <td class="description-td">0.1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L-asparate</td> <td class="description-td">0.1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L-valine</td> <td class="description-td">0.15</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L-threonine</td> <td class="description-td">0.2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L-serine</td> <td class="description-td">0.4</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
Cloning of the <i>B. subtilis </i>glmS Gene for Expressing in Yeast
</p>
    <p num="p-0537">The <i>B. subtilis </i>wild-type glmS was cloned into expression vector yEp352-ADH1. Forward primer nMD7107-023 and reverse primer nMD7107-024 were synthesized to amplify the coding sequence of the <i>B. subtilis </i>wild-type glmS gene. Restriction sites were incorporated at the ends to facilitate cloning. The glmS coding sequence was amplified by PCR under standard conditions using forward primer nMD7107-023 and reverse primer nMD7107-024 that have the following sequences: nMD7107-023(Kpn I): 5′AGCTGGTACCATGTGTGGAATCGTAGGTTATATC-3′ (SEQ ID NO:82) and nMD7107-024 (Sph I): 5′-TACGCATGCTTACTCCACAGTAACACTCTTCGC A-3′ (SEQ ID NO:83). Primer nMD7107-023 contains 24 nucleotides of the glmS coding sequence represented in nucleotides 11-34 of SEQ ID NO:82, and primer nMD7107-024 contains 25 nucleotides of the glmS coding sequence represented in nucleotides 10-34 of SEQ ID NO:83.</p>
    <p num="p-0538">Plasmid pSW07-15#83 containing the Bacillus glmS gene (plasmid described in Example 2) was used as the DNA templates in PCR reactions. A single band of PCR products of the expected size was generated under standard PCR conditions using the Taq polymerase. The PCR products were digested with appropriate restriction enzymes, purified through agarose-gel and cloned into the yEP352-ADH-1 vector predigested with the same enzymes. DNA ligation products were transformed into <i>E. coli </i>Top10 cells (Invitrogen Life Technologies, Carlsbad, Calif.) on ampicillin selection. First, 10 pools of colonies (10 per pool) were screened by PCR using the forward and reverse primers. Then, individual clones in positive pools were identified by PCR and confirmed by restriction digestions. Plasmids MD7107-240 and MD7107-241 contained the Bacillus glmS gene in the yeast expression cassette with the ADH promoter and terminator.</p>
    <p num="p-0539">The recombinant plasmids were transformed into cells of <i>S. cerevisiae </i>SWY5. Yeast transformants were selected on plates of SCE-minus medium supplemented with L-histidine at 20 mg l<sup>−1</sup>. Transformed yeast cell lines are analyzed to determine GlmS and GNA1 activities, and levels of glucosamine and N-acetylglucosamine.</p>
    <p num="h-0078">Cloning of the <i>S. cerevisiae </i>GNA1 for Over-Expression in Yeast</p>
    <p num="p-0540">The fragment containing the ScGNA1 coding sequence was digested from plasmid pSW07-60#3 (previously described in Example 13) using restriction endonucleases EcoR I and Sac I. The resulting fragment was ligated into the EcoR I and Sac I sites of shuttle vector pADH313-956. Cloning in this manner places the ScGNA1 coding sequence between the ADH1 promoter and terminator. This vector contains a histidine selection marker. The resulting plasmids, pSW07-114 (#1 and #18) were transformed into cells of <i>S. cerevisiae </i>SWY5. Yeast transformants were selected on plates of SCE-minus medium supplemented with uracil at 20 mg l<sup>−1</sup>. Transformed yeast cell lines will be analyzed to determine GlmS and GNA1 activities, and levels of glucosamine and N-acetylglucosamine.</p>
    <p num="p-0541">The plasmids with the ScGNA1 expression cassette are also be transformed into yeast cell lines that have already been transformed with the <i>E. coli </i>glmS*54 construct or the <i>Bacillus </i>glmS construct.</p>
    <heading>Example 21</heading> <p num="p-0542">This Example describes random mutagenesis of the <i>E. coli </i>strain 7107-92 for improving N-acetylglucosamine production.</p>
    <p num="p-0543">Strain 7107-92 (with T7-glmS*54 and T7-ScGNA1 integrated into the chromosome, described in Example 16) was mutagenized with UV light and 882 isolated colonies were assayed with the GlcN auxotroph bioassay as described in the U.S. Pat. No. 6,372,457 B 1. Glucosamine auxotroph strain <i>E. coli </i>2123-15 was used as the indicator strain. Based on halo size, 19 mutants were selected and streaked for isolation and five colonies of each were reevaluated. Two mutant strains, 7107-512 and 7107-513, showed the largest halo diameters. They were saved for evaluation in flask culture. Mutant <i>E. coli </i>strains 7107-512 (ATCC No. ______) and 7107-513 (ATCC No. ______) were deposited on Jul. 1, 2003, with the American Type Culture Collection (ATCC), P.O. Box 1549, Manassas, Va. 20108, USA, under the terms of the Budapest Treaty on the International Recognition of the Deposit of Microorganisms for the Purpose of Patent Procedure.</p>
    <p num="p-0544">The mutants were compared to the parent strain. All strains were grown in M9B medium with 5 g l<sup>−1 </sup>ribose and 5 g l<sup>−1 </sup>yeast extract under two different conditions. One set of cultures was grown in a medium with 30 g l<sup>−1 </sup>glucose and 0.2 mM IPTG (IPTG induction). The other set of cultures was grown in the medium with 10 g l<sup>−1 </sup>glucose and 40 g l<sup>−1 </sup>lactose. The culture became lactose-induced once glucose was depleted (lactose induction). Under IPTG induction, N-acetylglucosamine production by mutant 7107-512 was comparable to the parent strain. Mutant 7107-513 produced more glucosamine than the parent strain, 36% more at 71-hr time point. As observed previously, the parent strain produced higher levels of N-acetylglucosamine under lactose induction than under IPTG induction. Two mutants produced the same level of glucosamine, which was about 28% higher than the parent strain at the 71-hr time point. The loci of the mutations were not determined and the mechanisms for the improved N-acetylglucosamine production in the muatnts are not known. Since only about 900 mutant clones were screened, the data clearly demonstrated the potential to further improve the production host by random mutagenesis.</p>
    <p num="p-0545">
      <tables id="TABLE-US-00029" num="00029"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 23</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">N-acetylglucosamine production in UV light mutants</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="105pt" align="left"> </colspec> <colspec colname="1" colwidth="112pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">N-acetyglucosamine (g l<sup>−1</sup>)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Experiment</td>
                <td class="description-td">Strains</td>
                <td class="description-td">23 hrs</td>
                <td class="description-td">47 hrs</td>
                <td class="description-td">71 hrs</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="5" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">IPTG induction</td>
                <td class="description-td">7107-92 </td>
                <td class="description-td">7.2</td>
                <td class="description-td">11.8</td>
                <td class="description-td">13.8</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">7107-512</td>
                <td class="description-td">7.0</td>
                <td class="description-td">12.0</td>
                <td class="description-td">13.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">7107-513</td>
                <td class="description-td">7.2</td>
                <td class="description-td">14.6</td>
                <td class="description-td">18.7</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Lactose induction</td>
                <td class="description-td">7107-92 </td>
                <td class="description-td">4.7</td>
                <td class="description-td">13.3</td>
                <td class="description-td">23.3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">7107-512</td>
                <td class="description-td">8.0</td>
                <td class="description-td">19.0</td>
                <td class="description-td">29.9</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">7107-513</td>
                <td class="description-td">7.1</td>
                <td class="description-td">17.4</td>
                <td class="description-td">29.8</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="5" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 22</heading> <p num="p-0546">The following Example describes the generation of a mutant NAG production strain with a ƒfkA deletion such that NAG production can be uncoupled from growth.</p>
    <p num="p-0547">Phosphofructokinase is a main regulatory enzyme in glycolysis that catalyzes the formation of fructose-1,6-biphosphate from fructose 6-phosphate (F-6-P). The major phosphofructokinase in <i>E. coli</i>, encoded by pƒkA, provides 90% of the phosphofructokinase activity. The remaining 10% of activity is supplied by the minor phosphofructokinase, encoded by pƒkB. In NAG production strains, the overexpressed GlmS*54 catalyzes the conversion of F-6-P to glucosamine-6-phosphate (GlcN-6-P), which can then be converted to GlcNAc-6-P through the action of the overexpressed glucosamine-6-phosphate acetyltransferase (GNA1) from <i>Saccharomyces cerevisiae. </i> </p>
    <p num="p-0548">The rationale for the experiments described in Example 22 is as follows. Growth of a pƒkA mutant strain on a combination of carbon sources (i.e. glucose and fructose) may allow growth to be uncoupled from NAG production. Since pƒkA mutants do not grow well with glucose as carbon source, fructose would be used for cell growth. Imported fructose would be converted to fructose-1,6-biphosphate through the actions of the ƒruA and ƒruK gene products, allowing its entry into the glycolytic pathway. Glucose would be phosphorylated upon its uptake, and the resultant glucose-6-phosphate converted to F-6-P by the pgi gene product, phosphoglucose isomerase. With pƒkA gene deletion, only the minor PƒkB isozyme would be responsible for the conversion of F-6-P to F-1,6-biphosphate. The conversion could become restricted. As a consequence, there might be increased amounts of the F-6-P available for conversion to glucosamine-6-phosphate by the overexpressed GlmS*54. Therefore, deletion of pƒkA may reduce the flow of F-6-P into the glycolytic pathway, potentially allowing more carbon to divert toward glucosamine production. In production strains overexpressing the ScGNA1, this may ultimately result in higher NAG titers.</p>
    <p num="h-0081">Generation of pƒkA Deletion Strains</p>
    <p num="p-0549">The pƒkA deletion was added to the genome of the production strain using the temperature sensitive selection method. This required construction of an integrative vector to target the pƒkA region for deletion. The first step in vector construction was to amplify the sequence containing the pƒkA coding sequence plus flanking regions from <i>E. coli </i>W3110 genomic DNA. Primers were synthesized based on the published sequence of the pƒkA plus flanking regions from the <i>E. coli </i>genome (Blattner et al, 1997, Science 277(5331):14531474). The primers used for PCR amplification were forward primer 07-89 and reverse primer 07-90 and had the following sequences: 07-89: 5′GAGCGGCCGCATGAATCAA TCTTATGGACGGC3′ (SEQ ID NO:86) and 07-90: 5′GAGTCGACTCAGCGTTTGC TGATCTGATCGAACGTAC3′ (SEQ ID NO:87).</p>
    <p num="p-0550">Primer 07-89 contains a Not I site (GCGGCCGC, represented in nucleotides 3-10 of SEQ ID NO:86) and amplifies the ATG start codon of the yiiP coding sequence, located 1083 basepairs upstream of the pƒkA ATG start codon (represented in nucleotides 11-32 of SEQ ID NO:86). Primer 07-90 contains a Sal I site (GTCGAC, represented in nucleotides 3-8 of SEQ ID NO:87) and amplifies from the translational stop codon of the sbp coding sequence, located 1310 basepairs downstream of the pƒkA stop codon (represented in nucleotides 9-37 of SEQ ID NO:87). PCR was performed using a standard protocol to generate the fragment containing the yiiP, pƒkA, and sbp coding sequences flanked by Not I and Sal I restriction endonuclease sites. This fragment was cloned into vector pPCR-Script<sup>AMP</sup>SK(+) (Stratagene Cloning Systems, La Jolla, Calif.) using materials and instructions supplied by the manufacturer. The resulting plasmid was designated pSW07-61.</p>
    <p num="p-0551">To generate a temperature sensitive integrative vector, a fragment containing the temperature sensitive replicon of pMAK705 (Hamilton et al., 1989, J. Bac. 171 (9):4617-462) and the kanamycin resistance cassette of plasmid pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.) were excised from plasmid pKLN07-21 with restriction endonucleases Not I and Sal I. The pƒkA plus flanking regions was digested from plasmid pSW07-61 with restriction enzymes Not I and Sal I. The two fragments were ligated together, generating plasmid pSW07-63.</p>
    <p num="p-0552">To create a deletion in the coding sequence of pƒkA plasmid pSW07-63 was digested to completion with Pvu II, followed by a partial digestion with Ahd L This removed a 781 basepair fragment of the pƒkA coding sequence. The fragment containing the pƒkA deletion was treated with T4 DNA polymerase to fill in the ends and the resulting blunt-ended fragment ligated to itself, resulting in plasmid pSW07-64.</p>
    <p num="p-0553">To generate a strain containing the pfkA deletion, plasmid pSW07-64 was transformed into <i>E. coli </i>7107-18. Following the temperature sensitive selection and passaging protocol, kanamycin-sensitive colonies were screened for slowed growth on defined medium plates containing glucose as carbon source. Strains were confirmed by standard high stringency Southern hybridization using a 1153 basepair fragment containing a portion of the yiiP and the pƒkA sequence. These strains were designated as 7107-90(1) and 7107-90(2).</p>
    <p num="p-0554">Pfk specific activities of 0.054 and 0.035 mmol min<sup>−1 </sup>mg<sup>−1 </sup>protein were observed in strains 7107-90(1) and 7107-90(2), respectively. A specific activity of 0.78 mmol min<sup>−1 </sup>mg<sup>−1 </sup>was detected in the control strain 7107-87(25), which had the wild type pfkA gene. Therefore, the pƒkA mutants have roughly 5-6% Pfk activity observed in the control strain. This residual Pfk activity is undoubtedly contributed by the PfkB isozyme.</p>
    <p num="h-0082">Integration of the T7-lac-ScGNA1 Cassette into the Chromosome of Strains 7107-90(1) and 7107-90(2).</p>
    <p num="p-0555">The 7107-90 strains were derived from glucosamine production strain 0.7107-18; therefore, they do not produce measurable NAG. In order to generate a NAG production strain with the pfkA deletion, the T7-lac-ScGNA1 expression cassette needs to be introduced into the strain. Following the strategy described earlier, an expression cassette of T7-lac-ScGNA1 was integrated at the manXYZ site of the chromosome in strains 7107-90(1) and 7107-90(2), generating strains 7107-602 and 7107-603, respectively. These strains were confirmed by standard high stringency Southern hybridization using a fragment containing the ScGNA1 coding sequence as probe as having an integrated T7lac-ScGNA1 at the site of the manXYZ deletion.</p>
    <p num="h-0083">Shake Flask Analysis of Strain 7107-602 and 7107-603</p>
    <p num="p-0556">Strains 7107-602(1) and 7107-603(1) were tested in Shake Flask Screen 48 using varied mixes of glucose/fructose. Cultures were induced with 0.2 mM IPTG after 24 hours. Interestingly, these strains produced almost no acetate under any of the conditions tested, although they did not produce more NAG than the control strain 7107-92(1) (data not shown). Shake Flask Screen 53 again tested strains 7107-602(1) and 7107-603(1) under conditions for lactose induction. Again, no acetate was produced in these strains and NAG levels were similar to that seen in control strain 7107-92(1).</p>
    <p num="p-0557">To further evaluate acetate formation in strain 7107-602(1), Shake Flask Screen 56 was conducted under conditions which normally increase acetate formation, including addition of yeast extract (YE), ribose, or high trace elements (TE). Cultures were grown in modified M9B medium [6 g/l KH<sub>2</sub>PO<sub>4</sub>, 24 g/l K<sub>2</sub>HPO<sub>4</sub>, 1 g/l Na<sub>3</sub>Citrate.2H<sub>2</sub>O, 10 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>(phosphate adjusted to pH 7.4)]. Low levels of trace metals (0.3 mg/l FeSO<sub>4</sub>-7H<sub>2</sub>O, 0.375 mg/l ZnSO<sub>4</sub>-7H<sub>2</sub>O, 0.02 mg/l MnSO<sub>4</sub>—H<sub>2</sub>O, 0.001 mg/l CuSO4-5H<sub>2</sub>O, 0.001 mg/l NaMoO<sub>4</sub>-2H<sub>2</sub>O, 0.001 mg/l H<sub>3</sub>BO<sub>3</sub>, and 0.001 mg/l COCl<sub>2</sub>-6H<sub>2</sub>O) or high levels of trace metals (12 mg/l FeSO<sub>4</sub>-7H<sub>2</sub>O, 0.375 mg/l ZnSO<sub>4</sub>-7H<sub>2</sub>O, 0.8 mg/l MnSO<sub>4</sub>—H<sub>2</sub>CuSO4-5H<sub>2</sub>O, 0.001 mg/l NaMoO<sub>4</sub>-2H<sub>2</sub>O, 0.001 mg/l H<sub>3</sub>BO<sub>3</sub>, and 0.001 mg/l CoCl<sub>2</sub>-6H<sub>2</sub>O) were added as indicated in Table 24. Cultures were supplemented with 0.6 g/l MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g/l CaCl<sub>2</sub>-2H<sub>2</sub>O, 10 g/l glucose, and 20 g/l lactose. Additionally, 5 g/l ribose and/or 5 g/l yeast extract was added to cultures as indicated in Table 24. Culture were grown at 37° C. for 24 hours and then switched to 25° C. At 12 hours, 20 g/l glucose was added to cultures in which it was depleted and pH was adjusted to 7.2. At 24, 30, 48, and 54 hours, pH of cultures was adjusted to 7.2 and glucose was added to 30 g/l per day total based on HPLC results 5 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added at 24, 30, 48, and 52 hours to flasks in which levels had fallen below 1 g/l.</p>
    <p num="p-0558">
      <tables id="TABLE-US-00030" num="00030"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 24</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Effect of varied levels of trace elements, ribose, and yeast</td>
              </tr> <tr class="description-tr"> <td class="description-td">extract on acetate formulation in strains 7107-602(1) and 7107-92(1).</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="70pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Conditions</td>
                <td class="description-td">Time</td>
                <td class="description-td"> </td>
                <td class="description-td">Acetate</td>
                <td class="description-td">GlcNAc</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <colspec colname="7" colwidth="28pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain</td>
                <td class="description-td">TE</td>
                <td class="description-td">Ribose</td>
                <td class="description-td">YE</td>
                <td class="description-td">(hr)</td>
                <td class="description-td">OD<sub>600</sub> </td>
                <td class="description-td">g/l</td>
                <td class="description-td">g/l</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="8" colwidth="21pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">7107-602(1)</td>
                <td class="description-td">Low</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">24</td>
                <td class="description-td">4.5</td>
                <td class="description-td">0</td>
                <td class="description-td">8.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">48</td>
                <td class="description-td">9.6</td>
                <td class="description-td">0</td>
                <td class="description-td">18.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
                <td class="description-td">13.2</td>
                <td class="description-td">0</td>
                <td class="description-td">14.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">High</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">24</td>
                <td class="description-td">4.75</td>
                <td class="description-td">0</td>
                <td class="description-td">8.7</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">48</td>
                <td class="description-td">9.0</td>
                <td class="description-td">0</td>
                <td class="description-td">22.8</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
                <td class="description-td">12.9</td>
                <td class="description-td">7.2</td>
                <td class="description-td">22.3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Low</td>
                <td class="description-td">None</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">24</td>
                <td class="description-td">4.5</td>
                <td class="description-td">0</td>
                <td class="description-td">6.8</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">48</td>
                <td class="description-td">4.8</td>
                <td class="description-td">0</td>
                <td class="description-td">15.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
                <td class="description-td">10.8</td>
                <td class="description-td">0</td>
                <td class="description-td">28.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">High</td>
                <td class="description-td">None</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">24</td>
                <td class="description-td">4.5</td>
                <td class="description-td">0</td>
                <td class="description-td">6.8</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">48</td>
                <td class="description-td">9.0</td>
                <td class="description-td">0</td>
                <td class="description-td">16.4</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
                <td class="description-td">12.0</td>
                <td class="description-td">9.8</td>
                <td class="description-td">16.3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Low</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">None</td>
                <td class="description-td">24</td>
                <td class="description-td">1.75</td>
                <td class="description-td">0</td>
                <td class="description-td">4.15</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">48</td>
                <td class="description-td">2.4</td>
                <td class="description-td">0</td>
                <td class="description-td">13.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
                <td class="description-td">5.4</td>
                <td class="description-td">0</td>
                <td class="description-td">23.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">High</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">None</td>
                <td class="description-td">24</td>
                <td class="description-td">1.0</td>
                <td class="description-td">0</td>
                <td class="description-td">4.35</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">48</td>
                <td class="description-td">4.8</td>
                <td class="description-td">2.2</td>
                <td class="description-td">13.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
                <td class="description-td">10.5</td>
                <td class="description-td">0</td>
                <td class="description-td">27.8</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-92(1)</td>
                <td class="description-td">Low</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">24</td>
                <td class="description-td">8.5</td>
                <td class="description-td">0</td>
                <td class="description-td">7.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">(Control)</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">48</td>
                <td class="description-td">12.6</td>
                <td class="description-td">6.1</td>
                <td class="description-td">18.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
                <td class="description-td">13.8</td>
                <td class="description-td">13.6</td>
                <td class="description-td">15.8</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">High</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">24</td>
                <td class="description-td">7.5</td>
                <td class="description-td">0</td>
                <td class="description-td">5.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">48</td>
                <td class="description-td">12.6</td>
                <td class="description-td">8.4</td>
                <td class="description-td">13.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
                <td class="description-td">12.9</td>
                <td class="description-td">14.4</td>
                <td class="description-td">11.7</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Low</td>
                <td class="description-td">None</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">24</td>
                <td class="description-td">9.0</td>
                <td class="description-td">0</td>
                <td class="description-td">4.9</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">48</td>
                <td class="description-td">15.0</td>
                <td class="description-td">7.8</td>
                <td class="description-td">9.7</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
                <td class="description-td">15.0</td>
                <td class="description-td">15.0</td>
                <td class="description-td">9.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">High</td>
                <td class="description-td">None</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">24</td>
                <td class="description-td">9.5</td>
                <td class="description-td">0</td>
                <td class="description-td">5.4</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">48</td>
                <td class="description-td">12.6</td>
                <td class="description-td">10.9</td>
                <td class="description-td">7.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
                <td class="description-td">13.2</td>
                <td class="description-td">17.1</td>
                <td class="description-td">7.4</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Low</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">None</td>
                <td class="description-td">24</td>
                <td class="description-td">7.0</td>
                <td class="description-td">0</td>
                <td class="description-td">4.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">48</td>
                <td class="description-td">7.2</td>
                <td class="description-td">1.8</td>
                <td class="description-td">16.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
                <td class="description-td">7.8</td>
                <td class="description-td">3.7</td>
                <td class="description-td">19.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">High</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">None</td>
                <td class="description-td">24</td>
                <td class="description-td">10.0</td>
                <td class="description-td">0</td>
                <td class="description-td">7.17</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">48</td>
                <td class="description-td">13.2</td>
                <td class="description-td">7.4</td>
                <td class="description-td">13.10</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">72</td>
                <td class="description-td">13.2</td>
                <td class="description-td">13.0</td>
                <td class="description-td">11.7</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0559">Even under conditions designed to induce the control strain 7107-92(1) to produce high levels of acetate, acetate production in strain 7107-602(1) is either null or comparatively low (Table 23). Furthermore, although OD measurements tended to be lower for strain 7107-607(2), generally a higher NAG titer was seen in these cultures. Therefore, the pƒkA mutant strain appeared suitable for use as a NAG production host.</p>
    <heading>Example 23</heading> <p num="p-0560">This Example describes cloning and overexpression of the glutamine synthetase (glnA) gene, the integration of a T7lac-glnA cassette into the <i>E. coli </i>chromosome, and the effects of glnA gene over-expression on GlcN/GlcNAc production.</p>
    <p num="p-0561">Glutamine is a primary product of ammonia assimilation that provides nitrogen for amino sugars, as well as for other compounds. The glutamine synthetase, encoded by glnA, catalyzes the conversion of L-glutamate to L-glutamine in a reaction requiring NH<sub>3 </sub>and ATP. L-glutamine is required for the biosynthesis of glucosamine-6-phosphate; GlmS catalyzes the reaction by which L-glutamine and F-6-P are converted to D-glucosamine-6-P and L-glutamine. For maximal levels of GlcN/GlcNAC production, it is essential that adequate levels of glutamine are present in the cell. Overexpression of the glnA gene may increase levels of glutamine, and ultimately increase GlcN and or NAG titer.</p>
    <p num="h-0085">Cloning and Overexpression of the <i>E. coli </i>glnA Gene</p>
    <p num="p-0562">For cloning and overexpression of the <i>E. coli </i>glnA, primers were synthesized based on the published sequence of the glnA gene (Blattner et al, 1997, Science 277(5331):14531474). The nucleotide sequence of the <i>E. coli </i>glnA gene coding sequence is listed in the sequence file identified as SEQ ID NO:88. The deduced amino acid sequence of the <i>E. coli </i>GlnA protein is listed in the sequence file identified as SEQ ID NO:89. The primers were used to amplify the glnA coding sequence from <i>E. coli </i>7101-17(DE3) genomic DNA using PCR. The primers used for amplification were forward primer 07-gln and reverse primer 07-15 and had the following sequences: 07-gln: 5′GATCGGTCTCGCATGTCCGCTGAACAC GTACTGAC3′ (SEQ ID NO:90) and 07-15: 5′GATCCTCGAGTTAGACGCTGTA GTACAGCTC3′ (SEQ ID NO:91).</p>
    <p num="p-0563">Primer 07-gln contains a Bsa I site (GGTCTC, represented in nucleotides 5-10 of SEQ ID NO:90) and 23 nucleotides of the glnA coding sequence from its ATG start codon (represented in nucleotides 13-35 of SEQ ID NO:90). Primer 07-15 contains a Xho I site (CTCGAG, represented in nucleotides 5-10 of SEQ ID NO:91) and 21 nucleotides of the glnA coding sequence from its translational stop codon (represented in nucleotides 11-31 of SEQ ID NO:91). PCR was conducted under standard conditions to generate a fragment containing the glnA coding sequence flanked by Bsa I and Xho I restriction endonuclease sites.</p>
    <p num="p-0564">The PCR fragment containing glnA was digested with restriction endonucleases Bsa I and Xho I and ligated at the Nco I and Xho I sites of vector pET24d(+) (Novagen, Inc., Madison, Wis.), generating plasmid pKLN07-28. Cloning in this manner places the glnA sequence behind the T7lac promoter of pET24d(+), generating an expression cassette of T7lac-glnA.</p>
    <p num="h-0086">Functional Expression of the Recombinant glnA in <i>E. coli </i> </p>
    <p num="p-0565">To test functional expression of the glnA, recombinant plasmid pKLN07-28 was transformed into strain 7107-18, generating strains 7107-163. Control strain 7107-88 was prepared by transforming 7107-18 with the pET24d(+) empty vector. A standard induction protocol was followed in which cell cultures were grown in LB and induced with 1 mM IPTG. Samples were taken from induced cultures for SDS-PAGE to confirm GlnA protein overexpression. The predicted protein size of the GlnA protein is about 52 kDa. An overexpressed protein of about 52 kDa was seen. Most of the overproduced protein appears to be insoluble, with little seen in the soluble fractions. No such overexpressed protein was seen in the control strain, indicating that the overexpressed protein is GlnA enzyme.</p>
    <p num="h-0087">Construction of a Vector to Direct Integration of the T7lac-glnA into the <i>E. coli </i>Chromosome</p>
    <p num="p-0566">Having confirmed successful overexpression of the glnA gene, the next step was to integrate the T7lac-glnA cassette into the genome of production strains. An integrative vector was constructed for this purpose. The <i>E. coli </i>pƒkB gene was chosen as a target site for integration. The pƒkB encodes for the minor isozyme of phosphofructokinase in <i>E. coli</i>, which accounts for only 10% of the total phosphofructokinase activity. Therefore, integration of the cassette at this locus should not significantly affect the performance of the strain.</p>
    <p num="p-0567">As part of the strategy for generating the integrative vector, the pƒkB plus flanking regions was amplified from <i>E. coli </i>W3110 genomic DNA by PCR. Primers were synthesized based on the published sequence of the pƒkB plus its flanking regions (Blattner et al, 1997, Science 277(5331):1453-1474). The primers used to amplify the pƒkB region were forward primer 07-16 and reverse primer 07-17 and had the following sequences: 07-16: 5′GATCGCCGGCTTACATGCTGTAGCCCAGC3′ (SEQ ID NO:92) and 07-17: 5′GATCCTGCAGTCATGCTGCTAATAATCTATCC3′ (SEQ ID NO:93).</p>
    <p num="p-0568">Primer 07-16 contains a Nae I restriction endonuclease site (GCCGGC, represented in nucleotides 5-10 of SEQ ID NO:92) and 19 basepairs of ORFb1722 starting at its putative translational stop codon, located 1042 basepairs upstream of the pfkB start codon (represented in nucleotides 11-29 of SEQ ID NO:92). Primer 07-17 contains a Pst I restriction endonuclease site (CTGCAG, represented in nucleotides 5-10 of SEQ ID NO:93) and 22 basepairs of ORFb1725 starting at its putative translational stop codon, located 1357 basepairs downstream of the translational stop codon of the pfkB coding sequence (represented in nucleotides 11-32 of SEQ ID NO:93). PCR was conducted under standard conditions to generate a fragment containing the ORFb1722, pƒkB, ORFb1724, and ORFb1725 sequences flanked by Nae I and Pst I restriction endonuclease sites. The resulting fragment was ligated into vector pPCR-Script SK(+) (Stratagene Cloning Systems, La Jolla, Calif.), generating plasmid pKLN07-14.</p>
    <p num="p-0569">The next step was to add the kanamycin resistance cassette of plasmid pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.) to plasmid pKLN07-14. The kanamycin resistance cassette was excised from pUC4K with restriction endonuclease Pst I. Plasmid pKLN07-14 was likewise digested with restriction endonuclease Pst I, which removed a 412 basepair fragment of the plasmid including 386 basepairs of the ORFb1725 putative coding sequence. The kanamycin resistance cassette was ligated into the Pst I sites of the backbone of plasmid pKLN07-14, generating plasmids pKLN07-17. Plasmid pKLN07-17 was next digested with restriction endonucleases SnaB I and Btr I, removing 870 basepairs of the pƒkB coding sequence from the plasmid.</p>
    <p num="p-0570">The fragment containing the T7lac-glnA cassette was excised from plasmid pKLN07-28 with restriction endonuclease Nae I, generating a fragment containing the T7lac-glnA cassette flanked by 50 basepairs upstream of the T7 promoter and 164 basepairs downstream of the T7 terminator from vector pET24d(+). The Nae I fragment was ligated into the SnaB I and Btr I sites of plasmid pKLN07-17, generating plasmid pKLN07-29.</p>
    <p num="p-0571">The final step was to add the temperature sensitive replicon from pMAK705 to the fragment from plasmid pKLN07-29 containing the T7lac-glnA cassette in the site of the pƒkB deletion and the kanamycin resistance cassette. Plasmid pKLN07-29 was digested with restriction endonucleases Not I and Kpn I to release the fragment containing the T7lac-glnA and kanamycin resistance cassette from the pPCR-Script backbone. Plasmid pKLN07-20 (previously described) was digested with Not I and Kpn I, excising the fragment containing the temperature sensitive replicon. The two fragments were ligated together, generating plasmid pKLN07-30, containing the temperature sensitive replicon, kanamycin resistance cassette, and <i>E. coli </i>genomic sequence with the T7lac-glnA cassette ligated into the site of the pƒkB deletion. Plasmid pKLN07-30 can be used to direct integration of the T7lac-glnA cassette at the pƒkB of the chromosome following the temperature sensitive selection and passaging protocol.</p>
    <p num="p-0572">Plasmid pKLN07-30 was transformed into glucosamine production strain 7107-18. Following temperature sensitive selection and passaging, kanamycin sensitive colonies were screened for the presence of the T7lac-glnA cassette at the site of the pƒkB deletion using a standard PCR protocol. Several strains identified by PCR were confirmed by high stringency Southern hybridization using a fragment containing the glnA coding sequence as probe. These strains were designated 7107-118 through 7107-123.</p>
    <p num="h-0088">Integration of the T7lac-ScGNA1 Cassette into the Chromosome of Strains 7107-118, 7107-119, and 7107-120</p>
    <p num="p-0573">Strains 7107-118, 7107-119, and 7107-120 were derived from glucosamine production strain 7107-18; therefore, they do not produce measurable NAG. In order to generate a NAG production strain with the glnA expression cassette, the T7lac-ScGNA1 expression cassette needs to be introduced into the strain. As described previously, an expression cassette of T7-lac-ScGNA1 was integrated at the manXYZ of the chromosome in strains 7107-118,7107-119, and 7107-120, generating strains 7107-125 (derived from 7107-119), 7107-126 (derived from 7107-120), 7107-132 (derived from 7107-118), 7107-133 (derived from 7107-118), and 7107-134 (derived from 7107-119). These strains were confirmed by standard high stringency Southern hybridization using a fragment containing the ScGNA1 coding sequence as probe as having an integrated T7lac-ScGNA1 at the site of the manXY deletion.</p>
    <p num="h-0089">Shake Flask Screen 51: Testing Overexpression of the Integrated T7lac-glnA in NAG Production Background</p>
    <p num="p-0574">Shake Flask Screen 51 was conducted to evaluate strains 7107-125, 7107-126, and 7107-133 containing the integrated T7lac-glnA cassette. Cultures were grown in M9B medium supplemented with trace metals and the following substances: 0.6 g/l MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g/l CaCl<sub>2</sub>-2H<sub>2</sub>O, 10 g/l glucose, 40 g/l lactose, 5 g/l ribose, and 5 g/l yeast extract. Cultures were grown at 30° C. for 24 hours and then switched to 25° C. At 24 and 48 hours, the pH of cultures was adjusted to 7.2 and glucose added to 30 g/l, 5 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added at 24 and 48 hours to flasks in which levels had fallen below 1 g/l. Samples were taken at 24, 48, and 72 hours for OD and determination of NAG levels. Cultures were harvested at 72 hours for analysis of enzyme activities.</p>
    <p num="p-0575">As seen in Table 25, the strains overexpressing the glnA performed somewhat better than control strain 7107-92(1), producing about 8% more NAG. None were ever exhausted of ammonia so nitrogen was not limiting. Enzyme activities of GlmS and GNA1 in the glnA overexpression strains are comparable to control strain 7107-92(1). In summary, it appears that glnA overexpression offers a slight improvement in NAG titers which may be accentuated under optimized conditions.</p>
    <p num="p-0576"> <tables id="TABLE-US-00031" num="00031"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 25</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Cell growth, enzyme activity, and GlcNAc production</td> </tr> <tr class="description-tr"> <td class="description-td">in strains overexpressing glnA.</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="119pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="center"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Enzyme Activity</td> <td class="description-td">GlcNAc</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain</td> <td class="description-td">Construct</td> <td class="description-td">OD<sub>600</sub> </td> <td class="description-td">GlmS</td> <td class="description-td">GNA1</td> <td class="description-td">g/l</td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">7107-92(1)</td> <td class="description-td">control</td> <td class="description-td">9.75</td> <td class="description-td">0.34</td> <td class="description-td">3.2</td> <td class="description-td">24.3</td> </tr> <tr class="description-tr"> <td class="description-td">7107-125</td> <td class="description-td">T7lac-glnA</td> <td class="description-td">10.10</td> <td class="description-td">0.36</td> <td class="description-td">3.3</td> <td class="description-td">26.5</td> </tr> <tr class="description-tr"> <td class="description-td">7107-126</td> <td class="description-td">T7lac-glnA</td> <td class="description-td">9.75</td> <td class="description-td">0.25</td> <td class="description-td">3.0</td> <td class="description-td">26.0</td> </tr> <tr class="description-tr"> <td class="description-td">7107-133</td> <td class="description-td">T7lac-glnA</td> <td class="description-td">9.75</td> <td class="description-td">0.31</td> <td class="description-td">2.8</td> <td class="description-td">26.7</td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00042" class="description-td" colspan="6">1) OD, enzyme activities, and GlcNAc levels were measured in the 72-hour timepoint samples.</td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00043" class="description-td" colspan="6">2) Enzyme activities are reported in μmol min<sup>−1 </sup>mg<sup>−1 </sup>protein.</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Fermentation Experiment to Test NAG Production Strains with Integrated T7lac-glnA
</p>
    <p num="p-0577">Strain 7107-133, containing the integrated T7lac-glnA cassette, was next evaluated in a 1-liter fermentor. The fermentor was set up with an initial volume of 475 ml. Components of the fermentation medium are listed in Table 26. Fermentations were run using 75% NH<sub>4</sub>OH for pH control to 6.9. Temperature was maintained at 37° C. throughout the fermentation. Aeration and agitation were adjusted to maintain a dissolved oxygen concentration of 20% of air saturation. 65% glucose was fed to the cultures with feed rate controlled by computer program to achieve a growth rate of 0.40 hr<sup>−1 </sup>at inoculation and a maximum rate of 5 ml/hr by 6 hours. Cultures were induced with food grade lactose added at 5 g/l at 10 hours, with continued glucose feed.</p>
    <p num="p-0578">Fermentation results from strain 7107-133 were compared with those from strain 7107-92(1), previously run in Fermentation 237 under identical conditions. Both of these strains contain one copy of an integrated T7lac-ScGNA1 cassette. Results indicate that the presence of the T7lac-glnA cassette in strain 7107-133 may offer a slight advantage over strain 7107-92(1). Strain 7107-133 achieved 107.1 g/l NAG at 59.6 hrs compared with 96.6 g/l NAG at 59.8 hours in strain 7107-92(1).</p>
    <p num="p-0579">
      <tables id="TABLE-US-00032" num="00032"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 26</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Components of fermentation medium.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="35pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="119pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Component</td>
                <td class="description-td">Amount (g l<sup>−1</sup>)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="35pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="119pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H<sub>3</sub>PO<sub>4</sub> </td>
                <td class="description-td">4.79</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">KOH</td>
                <td class="description-td">3.15</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Citric acid —H<sub>2</sub>O</td>
                <td class="description-td">3.56</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> </td>
                <td class="description-td">5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Mg SO<sub>4</sub>—7H<sub>2</sub>0</td>
                <td class="description-td">2.5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CaCl<sub>2</sub>—2H<sub>2</sub>O</td>
                <td class="description-td">0.05</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Trace Metals</td>
                <td class="description-td">*</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Mazu 204 Antifoam</td>
                <td class="description-td">0.25</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="left" id="FOO-00044" class="description-td" colspan="3">*Trace metal composition is 5 mg/l FeSO<sub>4</sub>—7H<sub>2</sub>O, 3.75 mg/l</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="left" id="FOO-00045" class="description-td" colspan="3">ZnSO<sub>4</sub>—7H<sub>2</sub>O, 0.6 mg/l MnSO<sub>4</sub>—H<sub>2</sub>O, 0.1002 mg/l CuSO4—5H2O, 0.1002 mg/l CoCl<sub>2</sub>—6H<sub>2</sub>O.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 24</heading> <p num="p-0580">This Example describes cloning, overexpression of the glucose-6-phosphate dehydrogenase (zwƒ) gene, integration of a T7lac-zwf cassette into the <i>E. coli </i>chromosome and effect of overexpressing zwƒ under T7 promoter control on NAG production.</p>
    <p num="p-0581">The pentose phosphate pathway provides intermediates for amino acid, nucleotides, and cell wall biosynthesis. Furthermore, the oxidative portion of the pentose phosphate pathway is an important source of NADPH in the cell. The zwf gene of <i>E. coli </i>encodes the glucose-6-phosphate dehydrogenase (G6PDH), which catalyzes the first step in the pentose phosphate pathway, converting glucose-6-phosphate into glucono-1,5-lactone. Expression of the zwƒ gene is coordinated with the cellular growth rate (Rowley, D. and Wolf, R., J. Bac., 1991, 173(3):968-977).</p>
    <p num="p-0582">Literature describes <i>E. coli </i>isolates incapable of growth on NAG or GlcN (J. Bac., 1970, 101:384-391). The authors speculated that the accumulation of amino sugar phosphates may inhibit the reactions catalyzed by phosphohexose isomerase and glucose-6-phosphate dehydrogenase, resulting in pentose starvation. Add reversed the growth inhibition of those strains.</p>
    <p num="p-0583">Recombinant <i>E. coli </i>strains producing GlcN/GlcNAc may accumulate amino sugar phosphates to certain levels. If this is the case, addition of gluconate or pentoses such as ribose should result in increased growth and NAG production. Shake flask experiments conducted with NAG production strain 7107-87#25 demonstrated that addition of ribose or gluconate resulted in both growth and NAG titer increases. Thus it appears that the NAG production strain was experiencing pentose starvation which could be alleviated by addition of pentoses or gluconate.</p>
    <p num="p-0584">As a strategy to alleviate pentose starvation without adding exogenous pentoses, it was decided to overexpress the zwƒ gene in the NAG production strains. It was speculated that overexpression of the zwƒ might reduce the inhibition of phosphorylated amino sugars on the pentose phosphate pathway. If so, this could eliminate the need to supply additional pentoses or gluconate to enhance NAG titer in our strains.</p>
    <p num="h-0091">Cloning and Expression of the <i>E. coli </i>zwƒ</p>
    <p num="p-0585">For cloning and expression of the <i>E. coli </i>zwƒ primers were synthesized based on the published sequence of the zwƒ gene (Blattner et al, 1997, Science 277(5331): 1453-1474). The nucleotide sequence of the <i>E. coli </i>zwƒ gene coding sequence is listed in the sequence file identified as SEQ ID NO:94. The deduced amino acid sequence of the <i>E. coli </i>ZWF protein is listed in the sequence file identified as SEQ ID NO:95. The primers were used to amplify the zwƒ coding sequence from <i>E. coli </i>W3110 genomic DNA using PCR. The primers used for amplification were forward primer 07-101 and reverse primer 07-102 and had the following sequences: 07-101: 5′ GATCGGTCTCGCATGGCGGTAACGCAAACAGC 3′ (SEQ ID NO:96) and 07-102: 5′ GATCCTCGAGTTACTCAAACTCATTCCAGGAACG ACC 3′ (SEQ ID NO:97).</p>
    <p num="p-0586">Primer 07-101 contains a Bsa I restriction endonuclease site (GGTCTC, represented in nucleotides 5-10 of SEQ ID NO:96) and 20 nucleotides of the zwƒ coding sequence from its ATG start codon (represented in nucleotides 13-32 of SEQ ID NO:96). Primer 07-102 contains a Xho I restriction endonuclease site (CTCGAG, represented in nucleotides 5-10 of SEQ ID NO:97) and 27 nucleotides of the zwƒ coding sequence starting from its translational stop codon (represented in nucleotides 11-37 of SEQ ID NO:97). PCR was conducted under standard conditions to generate a fragment containing the zwƒ coding sequence flanked by Bsa I and Xho I restriction endonuclease sites.</p>
    <p num="p-0587">The PCR fragment was digested with restriction endonucleases Bsa I and Xho I and ligated at the Nco I and Xho I sites of vector pET24d(+) (Novagen, Inc., Madison, Wis.), generating plasmid pSW07-71. Cloning in this manner places the zwf sequence behind the T7lac promoter of pET24d(+), generating an expression cassette of T7lac-zwƒ.</p>
    <p num="h-0092">Functional Expression of the Recombinant ZWF Protein in <i>E. coli </i> </p>
    <p num="p-0588">Plasmids pSW07-71 #17, #20, and #33, each containing the T7lac-zwƒ expression cassette in pET24d(+), were transformed into the NAG production strain 7107-92(1), generating strains 7107-96(1) (transformed with pSW07-71 # 17), 7107-96(2) and 7107(transformed with pSW07-71#20), and 7107-96(4) (transformed with pSW07-71#33). Control strain 7107-95 was prepared by transformation of strain 7107-92(1) with pET24d(+). A standard induction protocol was followed in which cell cultures were grown in LB and induced with 1 mM IPTG. Samples were taken from the induced cultures for SDS-PAGE. Results confirmed overproduction of a protein of about 56 kDa, corresponding to the predicted size of the G6PDH protein. However, most of the overproduced protein appeared to be in the insoluble form.</p>
    <p num="p-0589">After 4 hours of IPTG induction, cell cultures were harvested to assay glucose-6-phosphate dehydrogenase activity. Strains 7107-96 (2), 7107-96(3), and 7107-96(4) had G6PDH activity of 69.7, 78.1, and 90.1 mmol min<sup>−1 </sup>mg<sup>−1 </sup>protein, compared with 0.04 mmol min<sup>−1 </sup>mg<sup>−1 </sup>protein for control strain 7107-95. This confirmed that the zwf was successfully overexpressed in the NAG production strain.</p>
    <p num="h-0093">Integration of the T7lac-zwƒ Cassette into the Chromosome of <i>E. coli </i> </p>
    <p num="p-0590">Having confirmed functional expression of the recombinant ZWF protein in <i>E. coli </i>7107-92# 1, the next step was to stably integrate the T7lac-zwf cassette into the chromosome of NAG production strains. An integrative vector was designed to target integration of the cassette to the rha region of the genome. The rhaBAD genes of <i>E. coli </i>form an operon encoding rhamnulokinase, L-rhamnose isomerase, and rhamnulose-1-phosphate aldolase, respectively. These genes are involved in utilization of rhamnose as an alternative carbon source and are considered nonessential genes. Therefore, interruption of this region should not affect growth or NAG production.</p>
    <p num="p-0591">The first step in generating the integrative vector was to clone the rhaBAD region from <i>E. coli </i>W3110 genomic DNA. Primers were synthesized based on the published sequence of the rhaBAD operon plus its flanking regions (Blattner et al, 1997, Science 277(5331):1453-1474). The primers used to amplify the rha region were forward primer 07-107 and reverse primer 07-108 and had the following sequences: 07-107: 5′ CGAATATCACGCGGTGACCAGTTAAAC 3′ (SEQ ID NO:98) and 07-108: 5′ CACAGTGTGCCGATGATTTTGACC 3′ (SEQ ID NO:99).</p>
    <p num="p-0592">Primer 07-107 amplifies from 1096 bases upstream of the rhaB start codon. Primer 07-108 amplifies from 977 bases downstream of the rhaD stop codon. PCR was conducted under standard conditions to generate a fragment containing the rhaBAD operon plus flanking sequences. The PCR fragment was cloned into vector pPCR-Script™SK(+) (Stratagene Cloning Systems, La Jolla, Calif.), generating plasmid pSW07-72.</p>
    <p num="p-0593">The next step in generation of the integrative vector was to add the T7lac-zwƒ cassette into plasmid pSW07-72. The T7-lac-zwƒ cassette was amplified by PCR from plasmid pSw07-71 using standard conditions. PCR amplification was performed with forward primer 07-111 and reverse primer 07-112, which have the following sequences: 07-111: 5′ GACCAATGGCCTAATGGAGCAACCGCACCTGTGGC 3′ (SEQ ID NO:100) and 07-112: 5′ GATCAGCGCTATCCGGATATAGTTCCTCCTTTCAGCAAAAAACCCC 3′ (SEQ ID NO:101).</p>
    <p num="p-0594">Primer 07-111 contains an Xcm I restriction endonuclease site (CCANNNNNNNN TGG, represented in nucleotides 3-17 of SEQ ID NO:100) and amplifies from 80 bp upstream of the T7 promoter sequence of pET24d(+) (represented in nucleotides 18-35 of SEQ ID NO:100). Primer 07-112 contains an Afe I restriction endonuclease site (AGCGCT, represented in nucleotides 5-10 of SEQ ID NO:101) and amplifies from 25 bp downstream of the T7 terminator (represented in nucleotides 11-46 of SEQ ID NO:101). The resulting 1.8-kb PCR product was digested with restriction endonucleases Xcm I and Afe I.</p>
    <p num="p-0595">Plasmid pSW07-72 was digested with restriction enzymes Xcm I and Aƒe I, excising a 3.6 kb fragment from the pSW07-72 backbone. The PCR fragment containing the T7lac-zwƒ with Xcm I and Aƒe I ends was ligated into the site of the excision in plasmid pSW07-72, generating plasmid pSW07-73. Plasmid pSW07-73 therefore has a 3.6 kb deletion of almost the entire rhaBAD operon, with the T7-lac-zwƒ inserted at the site of the deletion.</p>
    <p num="p-0596">The final step in construction of the integrative plasmid was to add the temperature sensitive replicon and kanamycin resistance cassette from plasmid pKLN07-21. Plasmid pKLN07-21 was digested with Not I and Kpn I, excising the fragment containing the temperature sensitive replicon and kanamycin resistance cassette. Plasmid pSW07-73 was digested with restriction enzymes Not I and Kpn I, releasing the fragment containing the T7lac-zwƒ with rhaBAD flanks from the pPCR-Script backbone. The two fragments were ligated together, generating plasmid pSW07-74. Plasmid pSW07-74 can be used to direct integration of the T7lac-zwf at the rha region of the <i>E. coli </i>chromosome.</p>
    <p num="p-0597">Plasmid pSW07-74 was transformed into NAG production strain 7107-92(1). Following temperature sensitive selection and passaging, kanamycin sensitive colonies were screened for the presence of the T7lac-zwf cassette at the site of the rhaBAD deletion using a standard PCR protocol. Several strains identified by PCR were confirmed by high stringency Southern hybridization using a 1.0-kb fragment of the zwƒ coding sequence as probe. These strains were designated 7107-606(1), 7107-606(2), 7107-606(3), and 7107-606(4).</p>
    <p num="h-0094">NAG Production in Strains with an Integrated T7lac-zwƒ Cassette</p>
    <p num="p-0598">Shake Flask Screen 46 was conducted to evaluate the effect of overexpressing zwƒ under T7 promoter control on NAG production in strains 7107-606(1) and 7107-606(3). As discussed above, it was speculated that overexpression of zwƒ could relieve pentose starvation and abolish growth inhibition caused by phosphorylated amino sugars. Cultures were grown in M9B medium (previously described) supplemented with 0.6 g/l MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g/l CaCl<sub>2-—2</sub>H<sub>2</sub>O, 40 g/l glucose, and 0.2 mM IPTG (for flasks with late induction, IPTG was added at 24 hours). 10 g/l ribose and 5 g/l yeast extract was added to cultures as indicated in Table 27. Cultures were incubated at 30° C. for 24 hours and then switched to 25° C. At 24 and 48 hours, the pH of cultures was adjusted to 7.2 and glucose added to 30 g/l per day total. 5 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added at 24 and 48 hours to flasks in which levels had fallen below 1 g/l. Samples were removed at 24, 48, and 72 hours for determination of OD and NAG levels. Cultures were harvested at 72 hours for enzyme analysis.</p>
    <p num="p-0599">
      <tables id="TABLE-US-00033" num="00033"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="259pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 27</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">The effect of varying induction time, ribose addition, and yeast extract addition on</td>
              </tr> <tr class="description-tr"> <td class="description-td">growth, enzyme activity, and GlcNAc production in strains overexpressing zwf.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="77pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Conditions</td>
                <td class="description-td"> </td>
                <td class="description-td">Enzyme Activities<sup>2</sup> </td>
                <td class="description-td">GlcNAc</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="9"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="28pt" align="center"> </colspec> <colspec colname="8" colwidth="28pt" align="center"> </colspec> <colspec colname="9" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain</td>
                <td class="description-td">Ribose</td>
                <td class="description-td">YE</td>
                <td class="description-td">IPTG</td>
                <td class="description-td">OD<sub>600</sub> </td>
                <td class="description-td">GlmS</td>
                <td class="description-td">GNA1</td>
                <td class="description-td">G6PDH</td>
                <td class="description-td">g/l</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="center" rowsep="1" class="description-td" colspan="9"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="9"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="8" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="9" colwidth="28pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">7107-92(1)</td>
                <td class="description-td">10 g/l</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">0.2 mM</td>
                <td class="description-td">13.0</td>
                <td class="description-td">0.21</td>
                <td class="description-td">6.7</td>
                <td class="description-td">0.36</td>
                <td class="description-td">12.9</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">None</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">0.2 mM</td>
                <td class="description-td">10.5</td>
                <td class="description-td">0.15</td>
                <td class="description-td">2.8</td>
                <td class="description-td">ND<sup>3</sup> </td>
                <td class="description-td">7.7</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">None</td>
                <td class="description-td">None</td>
                <td class="description-td">0.2 mM<sup>1</sup> </td>
                <td class="description-td">6.5</td>
                <td class="description-td">0.11</td>
                <td class="description-td">0.0</td>
                <td class="description-td">ND</td>
                <td class="description-td">4.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-606(1)</td>
                <td class="description-td">10 g/l</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">0.2 mM</td>
                <td class="description-td">12.5</td>
                <td class="description-td">0.12</td>
                <td class="description-td">3.0</td>
                <td class="description-td">21.8</td>
                <td class="description-td">6.4</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">None</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">0.2 mM</td>
                <td class="description-td">13.0</td>
                <td class="description-td">0.06</td>
                <td class="description-td">0.4</td>
                <td class="description-td">13.8</td>
                <td class="description-td">5.5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">None</td>
                <td class="description-td">None</td>
                <td class="description-td">0.2 mM<sup>1</sup> </td>
                <td class="description-td">6.75</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.0</td>
                <td class="description-td">29.2</td>
                <td class="description-td">2.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-606(3)</td>
                <td class="description-td">10 g/l</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">0.2 mM</td>
                <td class="description-td">11.5</td>
                <td class="description-td">0.04</td>
                <td class="description-td">1.73</td>
                <td class="description-td">18.8</td>
                <td class="description-td">6.3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">None</td>
                <td class="description-td">5 g/l</td>
                <td class="description-td">0.2 mM</td>
                <td class="description-td">12.5</td>
                <td class="description-td">0.13</td>
                <td class="description-td">1.93</td>
                <td class="description-td">17.2</td>
                <td class="description-td">5.4</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">None</td>
                <td class="description-td">None</td>
                <td class="description-td">0.2 mM<sup>1</sup> </td>
                <td class="description-td">6.75</td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.0</td>
                <td class="description-td">41.8</td>
                <td class="description-td">2.6</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="center" rowsep="1" class="description-td" colspan="9"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="left" id="FOO-00046" class="description-td" colspan="9"> <sup>1</sup>IPTG added at 24 hours for late induction</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="left" id="FOO-00047" class="description-td" colspan="9"> <sup>2</sup>Enzyme activities reported in μmol/min/mg protein</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="left" id="FOO-00048" class="description-td" colspan="9"> <sup>3</sup>ND: not determined</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0600">The results indicate that strains overexpressing the zwƒ driven by the T7 promoter had G6PDH activities 50 to 100 times greater than in the control strains. The zwƒ overexpression did improve growth slightly in the cultures with no added ribose when compared with growth of the control strain. However, GlmS activity tended to be diminished in strains overexpressing the zwƒ, and NAG production levels did not reach the levels seen in control strain 7107-92(1). This indicates that too much carbon may be funneling through the pentose phosphate pathway, away from the glucosamine pathway.</p>
    <heading>Example 25</heading> <p num="p-0601">This Example describes cloning of the glucose-6-phosphate dehydrogenase (zwƒ) gene, the integration of the zwƒ gene with its native promoter into the <i>E. coli </i>chromosome and effects of zwƒ-overexpression under its native promoter control on cell growth and NAG production.</p>
    <p num="h-0096">Cloning of <i>E. coli </i>zwf with its Native Promoter and Regulatory Regions</p>
    <p num="p-0602">As described in Example 24, strains were constructed with an integrated T7lac-zwf cassette as part of the strategy to improve growth and NAG production. It was speculated that overexpression of the zwf might reduce the inhibition of phosphorylated amino sugars on the pentose phosphate pathway. If so, this could eliminate the need to supply additional pentoses or gluconate to enhance NAG titer in our strains.</p>
    <p num="p-0603">The 7107-606 strains containing an integrated T7lac-zwf cassette did grow somewhat better than the control strain, indicating partial alleviation of pentose starvation. However, they did not perform as well as the control strain with regard to NAG production. This may be due to the use of the strong T7 promoter, which might lead to an expression level of ZWF protein that is much too high. Excessive activity of glucose-6-Phosphate dehydrogenase could funnel undesirably high amounts of carbon through the pentose phosphate pathway.</p>
    <p num="p-0604">Therefore, it was decided to overexpress the zwf with its native promoter so that expression of zwf would be regulated by the cell. In <i>E. coli</i>, zwf is subject to growth-rate dependent regulation. Additionally, the zwf is a member of the soxRS regulon and the multiple antibiotic resistance (mar) regulon. Therefore, cloning of the native zwf should include not only the native promoter, but regulatory regions such as the “Soxbox” as well (Fawcett, W. and Wolf, R., 1995, J. Bac. 177(7):1742-1750). The presence of two copies of the zwf on the chromosome could result in increased flow through the pentose phosphate pathway without greatly affecting carbon flow through other pathways, such as that for NAG production.</p>
    <p num="p-0605">For cloning and expression of the <i>E. coli </i>zwf, primers were synthesized based on the published sequence of the zwf gene (Blattner et al, 1997, Science 277(5331): 1453-1474). The primers were used to amplify the zwf coding sequence plus regulatory regions from <i>E. coli </i>W3110 genomic DNA using PCR. The primers used for amplification were reverse primer 07-129 and forward primer 07-130 and had the following sequences: 07-129: 5′GATGCTAGCTAACCGGAGCTCATAGGGC3′ (SEQ ID NO:102) and 07-130: 5′GATTTCGAATGATCAGTGTCAGATTTTTACCC3′ (SEQ ID NO: 103).</p>
    <p num="p-0606">Forward primer 07-130 contains a BstB I site (TTCGAA, represented in nucleotides 4-9 of SEQ ID NO:102) and amplifies from 203 basepairs upstream of the zwf start codon. Reverse primer 07-129 contains a Nhe I site (GCTAGC, represented in nucleotides 4-9 of SEQ ID NO:103) and amplifies from 154 basepairs downstream of the zwf stop codon. PCR was conducted under standard conditions to generate a 1.8-kb fragment flanked by Nhe I and BstB I restriction endonuclease sites.</p>
    <p num="h-0097">Integration of the Recombinant zwf into the Chromosome of <i>E. coli </i> </p>
    <p num="p-0607">An integrative vector was designed to target integration of the zwf cassette to the rha region of the genome. The rhaBAD genes of <i>E. coli </i>form an operon encoding rhamnulokinase, L-rhamnose isomerase, and rhamnulose-1-phosphate aldolase, respectively. These genes are involved in utilization of rhamnose as an alternative carbon source and are considered nonessential genes. Therefore, interruption of this region should not affect the growth or NAG production in our strains.</p>
    <p num="p-0608">Plasmid pSW07-72#45 (described in Example 24) was utilized in the first step for generation of the integrative vector. This plasmid, containing the rhaBAD operon plus flanking sequence, was digested with restriction endonucleases BstB I and Nhe I. This removed a 702 basepair fragment containing a portion of the rhaB and rhaA coding sequences from the plasmid. The PCR fragment containing the zwf coding sequence plus regulatory regions was digested with restriction endonucleases BstB I and Nhe I and ligated into the BstB I and Nhe I sites of pSW07-72#45, generating plasmid pSW07-86.</p>
    <p num="p-0609">Plasmid pSW07-86 was digested with restriction enzymes Kpn I and Not I to release the 6.9-kb fragment containing the zwf flanked by the rha genes. This fragment was ligated with the 4.2 kb Kpn I/Not I fragment from pKLN07-21, containing the temperature sensitive replicon from pMAK705 (Hamilton et al., 1989, J. Bac. 171(9):4617-4622) and the kanamycin resistance cassette from pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.). The resulting plasmid, pSW07-87, can be used for directing integration of the zwf to the rhaBA region of the <i>E. coli </i>chromosome</p>
    <p num="p-0610">Plasmid pSW07-87 was transformed into strains 7107-92(1) and 7107-607(2), which contained one or two copies of the ScGNA1 cassette, respectively. Following temperature sensitive selection and passaging, kanamycin sensitive colonies were screened for the presence of the T7lac-zwf cassette at the site of the rhaBA deletion using a standard PCR protocol. Several strains identified by PCR were confirmed by high stringency Southern hybridization using a fragment containing the zwf coding sequence as probe. Strains derived from 7107-607(2) were designated 7107-633 and strains derived from 7107-92(1) were designated 7107-634.</p>
    <p num="h-0098">Overexpression of the zwf under Native Promoter Control in NAG Production Strains</p>
    <p num="p-0611">Screening included strains 7107-633 and 7107-634 to evaluate the effect of overexpressing the zwf under native promoter control on NAG production. Cultures were grown in M9B medium (previously described) supplemented with 0.6 g/l MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g/l CaCl<sub>2</sub>-2H<sub>2</sub>O, 5 g/l yeast extract, 10 g/l glucose, and 40 g/l lactose. Strains were grown for 24 hours at 30° C. and then switched to 25° C. for the remainder of the experiment. At 24 and 48 hours, the pH of each culture was adjusted to 7.2 and glucose added to 30 g/l per day based on HPLC results. 5 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added at 24 and 48 hours to flasks in which levels had fallen below 1 g/l. Samples were removed at 24, 48 and 72 hours for determination of OD and NAG levels. Results are shown in Table 28.</p>
    <p num="p-0612">
      <tables id="TABLE-US-00034" num="00034"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 28</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Growth and GlcNAc production in strains</td>
              </tr> <tr class="description-tr"> <td class="description-td">overexpressing zwf with its native promoter.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="105pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="56pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">OD<sub>600</sub> </td>
                <td class="description-td">GlcNAc (g/l)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain</td>
                <td class="description-td">Description</td>
                <td class="description-td">48 hr</td>
                <td class="description-td">72 hr</td>
                <td class="description-td">48 hr</td>
                <td class="description-td">72 hr</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="28pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">7107-92(1)</td>
                <td class="description-td">1 GNA1</td>
                <td class="description-td">8.5</td>
                <td class="description-td">12.0</td>
                <td class="description-td">9.6</td>
                <td class="description-td">16.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-607(2)</td>
                <td class="description-td">2 GNA1</td>
                <td class="description-td">10.0</td>
                <td class="description-td">13.8</td>
                <td class="description-td">10.4</td>
                <td class="description-td">16.3</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-606(4)</td>
                <td class="description-td">1 GNA1; T7lac-zwf</td>
                <td class="description-td">11.5</td>
                <td class="description-td">14.4</td>
                <td class="description-td">11.9</td>
                <td class="description-td">19.7</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-634(1)</td>
                <td class="description-td">1 GNA1; zwf</td>
                <td class="description-td">12.0</td>
                <td class="description-td">13.8</td>
                <td class="description-td">11.6</td>
                <td class="description-td">16.9</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-634(2)</td>
                <td class="description-td">1 GNA1; zwf</td>
                <td class="description-td">10.0</td>
                <td class="description-td">12.6</td>
                <td class="description-td">12.7</td>
                <td class="description-td">21.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-633(1)</td>
                <td class="description-td">2 GNA1; zwf</td>
                <td class="description-td">10.0</td>
                <td class="description-td">13.2</td>
                <td class="description-td">10.4</td>
                <td class="description-td">17.8</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-633(5)</td>
                <td class="description-td">2 GNA1; zwf</td>
                <td class="description-td">9.5</td>
                <td class="description-td">12.6</td>
                <td class="description-td">9.8</td>
                <td class="description-td">17.3</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0613">Results indicate that with the zwf overexpressed either with the native or T7 promoter increased cell growth, indicating at least partial alleviation of pentose starvation. Production of NAG was also increased in several of the strains. For example, strain 7107-634(2) produced about 25% more NAG than control strain 7107-92(1). In this experiment, strains with two copies of the T7lac-ScGNA1 cassette with the overexpressed zwf were not improved over those with one copy of the cassette and the overexpressed zwf.</p>
    <p num="h-0099">Evaluation of Strain Overexpressing the zwf Gene in 1-Liter Fermentors</p>
    <p num="p-0614">Strains overexpressing the zwf were next evaluated in 1-L fermentors. These results were also compared with those from strain 7107-92(1), previously run in a separate fermentation under identical conditions. Fermentors were set up with an initial volume of 475 ml. Components of the fermentation medium are listed in Table 26. Fermentations were run using 75% NH<sub>4</sub>OH for pH control to 6.9. Temperature was maintained at 37° C. throughout the fermentation. Aeration and agitation were adjusted to maintain a dissolved oxygen concentration of 20% of air saturation. 65% glucose was fed to the cultures with feed rate controlled by computer program to achieve a growth rate of 0.40 hr<sup>−1 </sup>at inoculation and a maximum rate of 5 ml/hr by 6 hours. Cultures were induced with food grade lactose added at 5 g/l at 10 hours, with continued glucose feed.</p>
    <p num="p-0615">Fermentation results indicated that strain 7107-606(1) achieved a higher OD<sub>600 </sub>than control strains, particularly at earlier timepoints. This may be due to the increased pentose supply in this strain. On the other hand, strains overexpressing the zwf with its native promoter grew to about the same OD<sub>600 </sub>as the control strain. In this experiment, none of the strains overexpressing zwf surpassed control strain 7107-607(2) or 7107-92(1) in GlcNAc production. However, conditions for the fermentation were optimized for the production strains 7107-92(1) and 7107-607(2). It is possible that zwf over-expressing strains may require slightly different fermentation conditions to show their full potentials in improving growth and NAG production.</p>
    <heading>Example 26</heading> <p num="p-0616">This Example describes cloning of the phosphoglucose isomerase (pgi) gene, the integration of a T7lac-pgi cassette into the <i>E. coli </i>chromosome and effects of pgi gene overexpression on NAG production.</p>
    <p num="p-0617">The <i>E. coli </i>pgi gene encodes phosphoglucose isomerase, an enzyme that catalyzes the interconversion of glucose-6-phosphate to fructose-6-phosphate. Overexpression of the pgi may increase the pool of F-6-P in cells, and thus lead to higher GlcN/GlcNAc production. To test this possibility, the pgi gene was cloned and overexpressed in the <i>E. coli </i>NAG production background.</p>
    <p num="h-0101">Cloning and Overexpression of the <i>E. coli </i>pgi</p>
    <p num="p-0618">For cloning and overexpression of the <i>E. coli </i>pgi, primers were synthesized based on the published sequence of the pgi gene (Blattner et al, 1997, Science 277(5331): 1453-1474). The nucleotide sequence of the <i>E. coli </i>pgi gene coding sequence is listed in the sequence identified as SEQ ID NO:104. The deduced amino acid sequence of the <i>E. coli </i>PGI enzyme is listed in the sequence identified as SEQ ID NO:105. The primers were used to amplify the pgi coding sequence from <i>E. coli </i>W3110 genomic DNA by PCR. Forward primer 07-103 and reverse primer 07-104 were used for the amplification and had the following sequences. 07-103: 5′ GATCGGTCTCGCATGAAAAACATCAATCCAACGCAGAC 3′ (SEQ ID NO:106) and 07-104: 5′GATCCTCGAGTTAACCGCGCCACGCTTTATAGC 3′ (SEQ ID NO:107).</p>
    <p num="p-0619">Primer 07-103 contains a Bsa I site (GGTCTC, represented in nucleotides 5-10 of SEQ ID NO:106) and 26 nucleotides of the pgi coding sequence from its ATG start codon (represented in nucleotides 13-38 of SEQ ID NO:106). Primer 07-104 contains a Xho I site (CTCGAG, represented in nucleotides 5-10 of SEQ ID NO:107) and 23 nucleotides of the pgi coding sequence from its translational stop codon (represented in nucleotides 11-33 of SEQ ID NO:107). PCR amplification was conducted under standard conditions to generate a fragment containing the pgi coding sequence flanked by Bsa I and Xho I restriction endonuclease sites.</p>
    <p num="p-0620">The PCR fragment was digested with restriction endonucleases Bsa I and Xho I and ligated at the Nco I and Xho I sites of vector pET24d(+) (Novagen, Inc., Madison, Wis.), generating plasmids PKLNO736 and PKLNO7-37. Cloning in this manner places the pgi sequence behind the T7lac promoter of pET24d(+), generating an expression cassette of T7lac-pgi.</p>
    <p num="h-0102">Functional Expression of the Recombinant gi in <i>E. coli </i> </p>
    <p num="p-0621">Plasmid pKLN07-36 was transformed into NAG production strain 7107-92(1), generating strain 7107-124. Control strain 7107-95 was generated by transforming 7107-92(1) with the pET24d(+) empty vector. A standard induction protocol was followed in which cell cultures were grown in LB and induced with 1 mM IPTG. Samples were taken from the induced and noninduced cultures for SDS-PAGE. Results confirmed overproduction of a protein of about 62 kDa, corresponding to the predicted size of the PGI protein. Total and soluble protein amounts were determined by SDS-PAGE 4 hours after IPTG induction. Samples from strains 7107-124(1) and 7107-124(2) showed overproduction of the 62 kDa protein, indicating successful overexpression of the recombinant PGI protein. The presence of the overproduced protein in both induced and noninduced cultures indicates leaky expression of the pgi gene in the absence of inducer. No such protein band is seen in samples from control stain 7107-95. At least half of the total PGI protein appears to be in the soluble form.</p>
    <p num="p-0622">After four hours of induction, cultures were harvested to determine phosphoglucose isomerase activity. Strains 7107-124(1), 7107-124(2), and 7107-124(3) were found to have Pgi specific activities of 242, 158, and 215 mmol min<sup>−1 </sup>mg<sup>−1 </sup>protein, respectively, compared with 0.94 mmol min<sup>−1 </sup>mg<sup>−1 </sup>protein in control stain 7107-95. This confirms successful overexpression of the Pgi protein, reaching levels of 100 to 200 folds higher compared to the control.</p>
    <p num="h-0103">Construction of a Vector to Direct Integration of the T7lac-pgiA into the <i>E. coli </i>Chromosome</p>
    <p num="p-0623">Having confirmed functional expression of the pgi, the next step was to construct a vector to target integration of the T7lac-pgi cassette into the chromosome of <i>E. coli</i>. The target chosen for integration was the araBAD region of the <i>E. coli </i>genome. The araBAD operon, encoding for the L-ribulokinase, L-arabinose isomerase and L-ribulose-5-P 4-epimerase proteins, is involved in the utilization of L-ara proteins catalyze the conversion of L-arabinose to D-xylulose-5-phosphate, an intermediate in the pentose phosphate shunt. Since L-arabinose is not used as a carbon source in NAG fermentation process, gene integration at this site should not affect cell growth or NAG production.</p>
    <p num="p-0624">The first step in generating the integrative vector was to clone the region of the genome containing the araBAD operon. Primers were synthesized based on the published sequence of the araBAD operon (Blattner et al, 1997, Science 277(5331):1453-1474). The primers used to amplify the araBAD region were forward primer 07-105 and reverse primer 07-106 and had the following sequences: 07-105: 5′ GGATCCTACCTGACGCTTTTTA TCGCAACTC 3′ (SEQ ID NO:108) and 07-106: 5′CGGACGCACATCGGCCTCGTAG AC 3′ (SEQ ID NO:109).</p>
    <p num="p-0625">Primer 07-105 amplifies from 74 basepairs downstream of the ATG start codon of araB and primer 07-106 amplifies from 404 basepairs downstream of the araD translational stop codon. The resulting 4.7-kb PCR fragment containing the araBAD operon was ligated into vector pPCR-Script™SK(+) (Stratagene Cloning Systems, La Jolla, Calif.), generating plasmid pKLN07-38.</p>
    <p num="p-0626">The next step was to add the T7lac-pgi cassette from plasmid pKLN07-37 to plasmid pKLN07-38. The T7lac-pgi cassette was amplified by PCR from plasmid pKLN07-37 with forward primer 07-109 and reverse primer 07-110 that had the following sequences: 07-109: 5′ GATTCCGGAAGCAACCGCACCTGTGGC 3′ (SEQ ID NO:110) and 07-110: 5′ GATCACCTGGTTATAGTTCCTCCTTTCAGCAAAAAACCC 3′ (SEQ ID NO:111)</p>
    <p num="p-0627">Primer 07-109 contains a BspE I restriction endonuclease site (TCCGGA, represented in nucleotides 4-9 of SEQ ID NO:1110) and amplifies from 80 basepairs upstream of the T7 promoter of pET24d(+). Primer 07-110 contains a SexA I restriction endonuclease site (ACCTGGT, represented in nucleotides 5-11 of SEQ ID NO:111) and amplifies from 18 basepairs downstream of the T7 promoter of pET24d(+). PCR was conducted under standard conditions to generate a fragment containing the T7lac-pgi cassette flanked by BspE I and SexA I restriction endonuclease sites. The PCR fragment was subsequently digested with restriction endonuclease BspE I and SexA I.</p>
    <p num="p-0628">Plasmid pKLN07-38 was digested with restriction endonucleases BspE I and SexA I, excising a 2477 basepair fragment containing the last 621 basepairs of the araB coding sequence, the entire araA coding sequence, and the first 59 basepairs of the araD coding sequence. The digested T7lac-pgi PCR fragment (described above) was ligated into the BspE I and SexA I, sites of pKLN07-38, generating plasmid pKLN07-41. Plasmid pKLN07-41, therefore, has a 2.4 kb deletion of a portion of the araBAD operon, with the T7-lac-pgi inserted at the site of the deletion.</p>
    <p num="p-0629">For the final cloning step, the fragment containing the araBAD sequence with the T7lac-pgi insertion was digested from plasmid pKLN07-41 with restriction endonucleases Not I and Sal I. This fragment was ligated with the 4.2 kb Sal 1/Not I fragment from pKLN07-21, containing the temperature sensitive replicon from pMAK705 (Hamilton et al., 1989, J. Bac. 171(9):4617-4622) and the kanamycin resistance cassette from pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.). The resulting plasmid, pKLN07-47, can be used for directing integration of the pgi to the araBAD region of the <i>E. coli </i>chromosome.</p>
    <p num="p-0630">Strain 7107-92(1) was transformed with plasmid pKLN07-47. The temperature selection protocol was performed. Kanamycin sensitive colonies were screened by PCR under standard conditions for the presence of the T7lac-pgi at the site of the araBAD deletion on the chromosome. Several strains identified by PCR were confirmed by high stringency Southern hybridization using a fragment containing the pgi coding sequence as probe. These strains were designated 7107-136 through 7107-141.</p>
    <p num="h-0104">Shake Flask Evaluation of Strain 7107-136 and 7107-141</p>
    <p num="p-0631">Screening of strains included strains 7107-136 and 7107-141 to evaluate the effects of overexpression of the pgi on NAG production. Cultures were grown in M9B medium (previously described) supplemented with 0.6 g/l MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g/l CaCl<sub>2</sub>-2H<sub>2</sub>O, 5 g/l yeast extract, 5 g/l ribose, 10 g/l glucose, and 40 g/l lactose. Strains were grown for 24 hours at 30° C. and then switched to 25° C. for the remainder of the experiment. At 24 and 48 hours, the pH of each culture was adjusted to 7.2 and glucose added to 30 g/l per day based on HPLC results. 5 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added at 24 and 48 hours to flasks in which levels had fallen below 1 g/l. Samples were removed at 24, 48, and 72 hours for determination of OD and NAG levels. Cultures were harvested at 72 hours for enzyme analysis. Results are shown in Table 29.</p>
    <p num="p-0632">
      <tables id="TABLE-US-00035" num="00035"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 29</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Growth and GlcNAc production in</td>
              </tr> <tr class="description-tr"> <td class="description-td">strains overexpressing the pgi.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="105pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="56pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">OD<sub>600</sub> </td>
                <td class="description-td">GlcNAc (g/l)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain</td>
                <td class="description-td">Description</td>
                <td class="description-td">48 hr</td>
                <td class="description-td">72 hr</td>
                <td class="description-td">48 hr</td>
                <td class="description-td">72 hr</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="28pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">7107-92(1)</td>
                <td class="description-td">1 GNA1</td>
                <td class="description-td">12.0</td>
                <td class="description-td">12.0</td>
                <td class="description-td">19.3</td>
                <td class="description-td">30.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-136</td>
                <td class="description-td">1 GNA1; T7lac-pgi</td>
                <td class="description-td">12.75</td>
                <td class="description-td">13.5</td>
                <td class="description-td">16.0</td>
                <td class="description-td">27.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-141</td>
                <td class="description-td">1 GNA1; T7lac-pgi</td>
                <td class="description-td">13.1</td>
                <td class="description-td">12.4</td>
                <td class="description-td">18.5</td>
                <td class="description-td">29.5</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0633">In this experiment, no significant improvement was seen in strains with the overexpressed pgi. However, under optimized shake flask or fermentation conditions, the overexpressed pgi may positively influence growth and/or NAG production.</p>
    <p num="h-0105">Evaluation of Strain 7107-141 in 1 Liter Fermentation</p>
    <p num="p-0634">Strain 7107-141, overexpressing the pgi, was next evaluated in a 1-L fermenter. Results were compared with those from strain 7107-92(1), previously run in a separate fermentation under identical conditions. Fermenters were set up with an initial volume of 475 ml. Components of the fermentation medium are listed in Table 3. Fermentations were run using 75% NH<sub>4</sub>OH for pH control to 6.9. Temperature was maintained at 37° C. throughout the fermentation. Aeration and agitation were adjusted to maintain a dissolved oxygen concentration of 20% of air saturation. 65% glucose was fed to the cultures with feed rate controlled by computer program to achieve a growth rate of 0.40 hr<sup>−1 </sup>at inoculation and a maximum rate of 5 ml/hr by 6 hours. Cultures were induced with food grade lactose added at 5 g/l at 10 hours, with continued glucose feed.</p>
    <p num="p-0635">Similar to the shake flask experiments, no significant improvement in glucosamine production was seen in strains overexpressing the pgi gene. However, as discussed previously, under conditions optimized for these strains, pgi overexpression may positively influence growth and/or NAG production.</p>
    <heading>Example 27</heading> <p num="p-0636">The following Example describes development of glucosamine/N-acetylglucosamine production strains in which the glycogen synthesis is blocked by deletion of glgXCA genes. It also demonstrates the effects of blocking glycogen synthesis on glucosamine/N-acetylglucosamine production.</p>
    <p num="p-0637">Bacteria cells accumulate glycogen as the major form of stored carbon reserve. Glycogen synthesis involves three enzymes: ADP-glucose pyrophosphorylase, glycogen synthase and a branching enzyme. These enzymes catalyze, respectively, the synthesis of the monosaccharide donor (ADP-glucose) from glucose-1-phosphate, the polymerization of these monosaccharide units to form an a (1-4) polymer of glucose, and the rearrangement of this polymer to generate a (1-6) branches in the chain. The ADP-glucose pyrophosphorylase is a pivotal enzyme in glycogen synthesis and is strongly modulated by allosteric effectors. Genes involved in glycogen synthesis and degradation are organized as a glg operon (glgBXCAP), including glgB (1,4-alpha-glucan branching enzyme), glgX (glycosyl hydrolase, debranching enzyme), glgC (ADP-glucose pyrophosphorylase), glgA (glycogen synthase), and glgP (glycogen-maltotetraose phosphorylase). In an effort to increase carbon flow to the glucosamine and N-acetylglucosamine production pathways the glycogen synthesis pathway was blocked by gene deletion in the strains producing glucosamine and N-acetylglucosamine.</p>
    <p num="p-0638">PCR primers of the following sequences were synthesized to clone sequences from the glg operon. GNglgBXCAP1-5: 5′-GAGTCATCCGGATACAGTACGCGA-3′ (SEQ ID NO:112) and GNglgBXCAP2-3: 5′-ATAAACCAGCCGGGCAAATGG-3′ (SEQ ID NO:113).</p>
    <p num="p-0639">PCR amplification with the primers using standard conditions resulted in the generation of a 5737-bp fragment of the glg operon from <i>E. coli </i>strain W3110. The amplified sequence spans from glgB through glgP. The PCR product was ligated into pCR®2.1-TOPO® (Invitrogen TOPO TA Cloning Kit, Catalog # K4500-01), generating the recombinant plasmid pCALG18-1.</p>
    <p num="p-0640">pCALG18-1 was digested with Age I to delete the 3′ portion of glgX, the entire glgC, and the 5′ portion of glgP from the plasmid. The remaining portion of the plasmid was re-circulated to generate pCALG21-1. This recombinant plasmid contains the truncated glg operon (glgXCAD).</p>
    <p num="p-0641">To generate the plasmid needed to integrate glgXCAD into the genomes of glucosamine and/or N-acetylglucosamine producing strains two further procedures were required. First, the Kan<sup>r </sup>gene from pUC4K (Amersham Pharmacia Biotech, Catalog # 27-4958-01, GenBank Accession # X06404) was added to pCALG21-1 to generate pCALG23-1. Second, the temperature sensitive replication origin from pMAK705 (Hamilton, C., et al, 1989, Journal of Bacteriology, 171:9, pp 4617-4622) was added to pCALG23-1 to generate pCALG28-1. To accomplish the first procedure, pCALG21-1 and pUC4K were both digested with BamH I. BamH I digestion linearizes pCALG21-1 upstream of the glgXCA deletion site and releases the Kan<sup>r </sup>fragment from pUC4K. The linearized pCALG21-1 and Kan<sup>r </sup>fragments were ligated to generate pCALG23-1. To generate pCALG28-1, the Kan<sup>r</sup>-glgXCAD fragment from pCALG23-1 was PCR amplified using oligonucleotides GNTOPO2-5 (SEQ ID NO:114) and GNTOPO3-4 (SEQ ID NO:115). Sequence of GNTOPO2-5: 5′-CGCCAAGCTTGGTACCG-3′ (SEQ ID NO:114). The primer sequence is identical to nucleotides 230 to 246 of pCR®2.1-TOP®. Sequence of GNTOPO3-4: 5′-CCCTCTAGATGCATGCTCGAG-3′ (SEQ ID NO:115). The sequence is reverse complimentary to nucleotides 334 to 354 of pCR®2.1-TOPO®.</p>
    <p num="p-0642">The PCR products were ligated to the Sma I fragment containing the temperature sensitive replication origin isolated from pMAK705. The resultant recombinant plasmid pCALG28-1 contains Kan<sup>r</sup>, the glgXCAD sequences and the temperature sensitive replication origin.</p>
    <p num="p-0643">Generation of strains deficient in glycogen synthesis was accomplished by replacement of endogenous glgBXCAP sequences with the recombinant glgXCAD sequence. pCALG28-1 was transformed into <i>E. coli </i>strain 7107-18 and clones in which the endogenous glgBXCAP was replaced with the recombinant glgXCAD sequence from pCALG28-1 were generated using the temperature selection procedure. The desired clones were further identified by PCR screening and confirmed by an iodine vapor test. The PCR screening was conducted using oligonucleotides GNglgBXCAP3-5 (SEQ ID NO:136) and GNglgBXCAP43 (SEQ ID NO:137). The sequence of GNglgBXCAP3-5 is as follows: 5′-GGCGGCTTAAAATGTCCTGAATG-3′ (SEQ ID NO:136). The primer is located further upstream to the 5′ end of the glgBXCAP PCR fragment generated from the oligonucleotides GNglgBXCAP1-5 (SEQ ID NO:112) and GNglgBXCAP2-3 (SEQ ID NO:113). The sequence of GNglgBXCAP4-3 is: 5′-CGAAATCATCGTTGCCAGTAACTTTACG-3′ (SEQ ID NO:137). The primer is located further downstream to the 3′ end of the glgBXCAP PCR fragment generated from the oligonucleotides GNglgBXCAP1-5 (SEQ ID NO:112) and GNglgBXCAP2-3 (SEQ ID NO:113).</p>
    <p num="p-0644">In the PCR screening two strains produced PCR products of the expected size (2295 bp) for the glgXCAD sequences. These two strains were named 7107-308 and 7107-309. The strains were then subjected to an iodine vapor test to demonstrate that that neither of these strains was able to accumulate glycogen. To perform the test, plates with 7107-18, 7107-308, and 7107-309 cells were exposed to the vapors from iodine crystals. Only strain 7107-18 turned a dark brown color, indicating the presence of glycogen.</p>
    <p num="h-0107">Shake Flask Experiments to Test NAG Production in Glycogen Deficient Strains</p>
    <p num="p-0645">A shake flask screen evaluated strains with glycogen synthesis blocked by glg deletion. Strains were grown in M9B medium supplemented with 40 g l<sup>−1 </sup>glucose, 10 g l<sup>−1 </sup>ribose and 5 g l<sup>−1 </sup>yeast extract. There appears to be some interesting differences among strains with glg deletion. Some glg deletion strains (e.g. 7107-604) grew better than the control but produced a lower level of NAG. Strain 7107-605-1 showed a growth comparable to the control but with a 12% improvement in NAG titer.</p>
    <p num="p-0646">
      <tables id="TABLE-US-00036" num="00036"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 30</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Growth and NAG production in glg deletion</td>
              </tr> <tr class="description-tr"> <td class="description-td">strains in shake flask experiments.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="84pt" align="left"> </colspec> <colspec colname="1" colwidth="70pt" align="center"> </colspec> <colspec colname="2" colwidth="63pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">OD</td>
                <td class="description-td">NAG (g l<sup>−1</sup>)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="14pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">47</td>
                <td class="description-td">71</td>
                <td class="description-td">23</td>
                <td class="description-td">47</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Strain</td>
                <td class="description-td"> </td>
                <td class="description-td">23 hrs</td>
                <td class="description-td">hrs</td>
                <td class="description-td">hrs</td>
                <td class="description-td">hrs</td>
                <td class="description-td">hrs</td>
                <td class="description-td">71 hrs</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="14pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="8" colwidth="28pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Control</td>
                <td class="description-td">7107-92(1)</td>
                <td class="description-td">4.2</td>
                <td class="description-td">10</td>
                <td class="description-td">10.5</td>
                <td class="description-td">6.7</td>
                <td class="description-td">10</td>
                <td class="description-td">10</td>
              </tr> <tr class="description-tr"> <td class="description-td">glg deletion</td>
                <td class="description-td">7107-604(1)</td>
                <td class="description-td">8.4</td>
                <td class="description-td">12</td>
                <td class="description-td">12</td>
                <td class="description-td">6.3</td>
                <td class="description-td">7.1</td>
                <td class="description-td">7.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">7107-604-(2)</td>
                <td class="description-td">8.1</td>
                <td class="description-td">12</td>
                <td class="description-td">12.5</td>
                <td class="description-td">6.5</td>
                <td class="description-td">7.7</td>
                <td class="description-td">7.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">7107-605(1)</td>
                <td class="description-td">4.2</td>
                <td class="description-td">10</td>
                <td class="description-td">11.5</td>
                <td class="description-td">6.9</td>
                <td class="description-td">10.4</td>
                <td class="description-td">11.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">7107-605(2)</td>
                <td class="description-td">4.7</td>
                <td class="description-td">10.5</td>
                <td class="description-td">11</td>
                <td class="description-td">6.2</td>
                <td class="description-td">10.3</td>
                <td class="description-td">10.9</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 28</heading> <p num="p-0647">The following Example demonstrates the effects of deletion of one of two lacI genes and replacement of the lac promoter with the lacUV5 promoter on the alleviation of glucose repression and glucosamine/N-acetylglucosamine production.</p>
    <p num="p-0648">It has been known that the presence of glucose in the media represses expression from the lac promoter in <i>E. coli</i>. The lac operon encodes three proteins: LacZ, LacY, and LacA. The lacZ gene encodes the enzyme, b-galactosidase, which cleaves lactose into glucose and galactose and also converts lactose into allolactose, a true inducer of the operon. Allolactose induces the lac operon by interacting with the repressor (encoded by the lacI gene) and to prevent it from binding at the lac operator. The lacY gene encodes the lactose permease that controls the influx of lactose into the cell. The lacA gene encodes thiogalactoside transacetylase (galactoside acetyltransferase), an enzyme that may assist in cellular detoxification of nonmetabolizable pyranosides.</p>
    <p num="p-0649">Transcription of the lac operon requires the binding of the lac promoter sequence by the complex CRP:cAMP, a complex formed between cAMP and its receptor protein (CRP). It has been believed that glucose represses the lac operon by reducing cAMP levels in the cells. The lacUV5 promoter is a mutant version of the lac promoter containing specific nucleotide changes. Due to these changes the lac promoter no longer requires the binding by the CRP:cAMP complex to activate transcription. This suggest that lac operon under lacUV5 control would not be susceptible to glucose repression. Therefore, the lac promoter was replaced by the lacUV5 promoter (represented herein by SEQ ID NO:135) in N-acetylglucosamine and/or glucosamine producing <i>E. coli </i>strains to minimize glucose repression.</p>
    <p num="p-0650">Some N-acetylglucosamine and/or glucosamine producing strains contain two copies of lacI repressor gene. One is a component of the native lac operon and the other is found in the DE3 element. The amount of cellular lacI repressor protein could affect the strength of repression. Therefore, deletion of one of the lacI genes could affect lactose induction of glucosamine/N-acetylglucosamine production.</p>
    <p num="h-0109">Replacement of the lac Promoter with the lacUV5 Promoter</p>
    <p num="p-0651">Several precursor plasmids were generated prior to the construction of the plasmid used to integrate the lacUV5 promoter into N-acetylglucosamine and/or glucosamine producing strains.</p>
    <p num="p-0652">Generation of the first precursor plasmid included the PCR amplification of the mphRlacIlacZ fragment from <i>E. coli </i>strain W3110 was performed using oligonucleotide primers GNmphRlacIlacZ1-5 (SEQ ID NO:116) and GNmphRlacIlacZ2-3 (SEQ ID NO:117). The sequence of GNmphRlacIlacZ1-5 was as follows: 5′-ATTGTGCGCTCAGTATAGGAAGG-3′ (SEQ ID NO:116), and that of GNmphRlacIlacZ2-3: 5′-CGATACTGACGGGCTCCAG-3′ (SEQ ID NO:117). The correct-sized PCR product containing the mphRlacIlacZ sequence was cloned into pCR®2.1-TOPO® to generate pCALG3-1.</p>
    <p num="p-0653">The next precursor plasmid resulted from a site-directed mutagenesis (Stratagene® QuikChange® XL Site-Directed Mutagenesis Kit, Catalog # 200517) of pCALG3-1 using oligonucleotides GNlacdel2-5 (SEQ ID NO:118) and GNlacdel3-5 (SEQ ID NO:119). The sequence of GNlacdel2-5 was: 5′-(phosphorylated) GCAAAACCTTTCGCGGTCACCCATGATAGCGCCCG-3′ (SEQ ID NO:118). The primer sequence is identical to the W3110 lacI sequence except for the ATGG to CACC changes made to add the BstE II site (represented from nucleotide 15-21 of SEQ ID NO:118) 5′ of the lacI start codon. The sequence of GNlacdel3-5: 5′-(phosphorylated) CGGGCGCTATCATGGGTGACCGCGAAA-GGTTTTGC-3′ (SEQ ID NO:119). The oligonucleotide sequence is reverse complementary to the W3110 lacI sequence except for the CCAT to GGTG changes made to add the BstE II site (represented from nucleotide 15-21 of SEQ ID NO:119) 5′ of the lacI start codon.</p>
    <p num="p-0654">Site-directed mutagenesis of pCALG3-1 using oligonucleotides GNlacdel2-5 (SEQ ID NO:118) and GNlacdel3-5 (SEQ ID NO:119) added a BstE II site 5′ of the lac start codon to generate pCALG5-1. This recombinant plasmid consists of the mphRlacIlacZ sequence with a BstE II site 5′ of the lacI start codon in the backbone of pCR®2.1-TOPO®.</p>
    <p num="p-0655">Next, pCALG5-1 was digested with BstE II and the fragment containing 3′ sequences of lacI, a portion of the lacZ gene with the lac promoter, pCR®2.1-TOPO®, and sequences from mphR was isolated. The isolated fragment from pCALG5-1 was ligated to itself to generate pCALG10-1. The recombinant pCALG10-1 plasmid contains mphR sequences, lacID, the lac promoter, lacZ sequences in the backbone of pCR®2.1-TOPO®.</p>
    <p num="p-0656">Next, pCALG10-1 was digested with BstE II and Nde I and the fragment containing pCR®2.1-TOPO® and the mphR sequences was isolated. The isolated pCALG10-1 fragment was then ligated to two PCR products. The first PCR product, generated from pCAL610-1, contained the lacZ sequence with an Nde I site added immediately 5′ of the lacZ start codon. Using PCR under standard conditions and oligonucleotide primers GNLacZ1-5 (SEQ ID NO:120) and GNLacZ2-3 (SEQ ID NO:121), the portion of pCALG10-1 containing the lacZ sequence was amplified. The sequence of GNLacZ1-5 is as follows: 5′-CACAGGAAACACATATGACCATGATTACGG-3′ (SEQ ID NO:120). The primer sequence is identical to nucleotides 1921 to 1950 of pCALG10-1 (the lac promoter/lacZ junction) except for G1932C and C1933A nucleotide changes to add an Nde I site (represented by nucleotides 12-17 of SEQ ID NO:120). The sequence of GNLacZ2-3 is: 5′-CCACCATGATATTCGGCAAGCAG-3′ (SEQ ID NO:121). The sequence is reverse complimentary to nucleotides 6523 to 6545 of pCALG10-1.</p>
    <p num="p-0657">Using PCR under standard conditions and oligonucleotide primers GNLacuv53-5 (SEQ ID NO:122) and GNLacuv54-3 (SEQ ID NO:123), the second PCR product, the lacUV5 promoter from strain 7101-17(DE3), was amplified. The sequence of GNLacuv53-5 is: 5′-CCTTTCGCGGTCACCAGCAAA-3′ (SEQ ID NO:122). The sequence is identical to nucleotides 1253 to 1273 of pCALG10-1 and encompasses the endogenous BstE II site (represented by nucleotide sequences 9 to 15 of SEQ ID NO:122) within lacI. The sequence of GNLacuv54-3 is: 5′-CCGTAATCATGGTCATATGTGTTTCCTGTG-3′ (SEQ ID NO:123). The sequence is reverse complimentary to nucleotides 1921 to 1950 of pCALG10-1 (the lac promoter/lacZ junction) except for C1932G and G1933T nucleotide changes to add an Nde I site (represented by nucleotides 14 through 19 of SEQ ID NO:123).</p>
    <p num="p-0658">The PCR-generated lacUV5 promoter fragment was digested with BstE II plus Nde I, and purified. Similarly, the PCR product containing the lacZ sequence was digested with Nde I and purified. To generate the next precursor plasmid the mphR fragment (Nde I-BstE II) from pCALG10-1, the lacUV5 promoter fragment (BstE II-Nde I), and the lacZ fragment (Nde I-Nde I) were ligated together. The resultant recombinant plasmid, pCALG16-3, contains mphR sequences, lacID, lacUV5 promoter, lacZ sequences (in the opposite order as in the native form of the sequences) in the vector pCR®2.1-TOPO®. The correct lacUV5 promoter sequences of pCALG16-3 were confirmed by sequencing.</p>
    <p num="p-0659">Next, the lacZ fragment in pCALG10-1 was PCR amplified using standard conditions and oligonucleotide primers GNLacZ1-5 (SEQ ID NO:120) and GNlacZ3-3 (SEQ ID NO:124).</p>
    <p num="p-0660">The nucleotide sequence of GNlacZ3-3 is: 5′-GACGAAGCGGCCGCGTAAACG-3′ (SEQ ID NO:124). The sequence is reverse complimentary to nucleotides 3618 to 3638 of pCALG10-1 except for the T3625C, C3626G, and C3630G nucleotides changes to add a Not I site (represented by nucleotides 7 to 14 of SEQ ID NO:124).</p>
    <p num="p-0661">The lacZ PCR fragment was digested with Nde I and Not I and isolated. Additionally, pCALG16-3 was digested with Nde I to Not I and the fragment containing pCR®2.1-TOPO®, sequences from mphR, lacID, and the lacUV5 promoter was isolated. The lacZ PCR fragment and the pCALG16-3 fragment containing pCR®2.1-TOPO®, sequences from mphR, lacID, and the lacUV5 promoter were ligated together to produce pCALG20-1. The recombinant pCALG20-1 plasmid contains pCR®2.1-TOPO®, sequences from mphR, lacID, the lacUV5 promoter (with a Nde I site immediately 5′ of the lacZ start codon), and the lacZ coding region in the 5′ to 3′ orientation.</p>
    <p num="p-0662">The previously constructed pCALG3-1 was used along with pCALG20-1 to generate the next precursor plasmid. Both pCALG3-1 and pCALG20-1 were digested with Apa I. The pCALG20-1 fragment containing the lacUV5 promoter and lacZ sequences and the pCALG3-1 fragment containing pCR®2.1-TOPO®, mphR sequences, and full-length lacI were isolated and ligated together to produce pCALG22-1. The recombinant pCALG22-1 contains mphR sequences, full-length lacI, the lacUV5 promoter (with a Nde I site immediately 5′ of the lacZ start codon), lacZ sequences, and pCR®2.1-TOPO®.</p>
    <p num="p-0663">pCALG22-1 and pUC4K (Amersham Pharmacia Biotech, Catalog # 27-4958-01, GenBank Accession # X06404) were used to generate the next precursor plasmid. pCALG22-1 and pUC4K were both digested with BamH I. The pCALG22-1 fragment containing mphR sequences, full-length lacI, the lacUV5 promoter, lacZ sequences, and pCR®2.1-TOPO® and the pUC4K fragment containing Kan<sup>r </sup>were isolated and ligated together to generate pCALG25-1. The recombinant pCALG25-1 consists of Kan<sup>r</sup>, mphR sequences, full-length lacI, the lacUV5 promoter (with a Nde I site immediately 5′ of the lacZ start codon), lacZ sequences, and pCR®2.1-TOPO®.</p>
    <p num="p-0664">To generate the next precursor plasmid, pCALG25-1 and pKLN07-20 (described elsewhere) were digested with Kpn I and Not I. The pCALG25-1 fragment containing Kan<sup>r</sup>, mphR sequences, full-length lacI, the lacUV5 promoter, and lacZ sequences and the pKLN07-20 fragment containing the temperature sensitive replication origin from pMAK705 (described previously) were isolated and ligated together to generate pCALG29-1. The recombinant pCALG29-1 plasmid contains Kan<sup>r</sup>, mphR sequences, full-length lacI, the lacUV5 promoter (with a Nde I site immediately 5′ of the lacZ start codon), lacZ sequences, and the temperature sensitive region from pMAK705.</p>
    <p num="p-0665">After constructing pCALG29-1 it was thought that this would be the final integrative vector. However, it was subsequently determined out that addition of the Nde I site immediately 5′ of the lacZ start codon inhibited proper expression of lacZ. Therefore, pCALG29-1 was subjected to site-directed mutagenesis (Stratagene® QuikChange® XL Site-Directed Mutagenesis Kit, Catalog # 200517) to destroy the Nde I site and replace the Nde I nucleotide changes with the endogenous lacUV5 nucleotides. To make the necessary changes to pCALG29-1, pCALG29-1 was subjected to site-directed mutagenesis using oligonucleotides GNlacZ-Nde1 (SEQ ID NO:125) and GNlacZ-Nde2 (SEQ ID NO:126) to produce pCALG3′-1.</p>
    <p num="p-0666">The nucleotide sequence of GNlacZ-Nde1 is as follows: 5′-CACACAGGAAA CAGCTATGACCATGATTACGGATTC-ACTGG-3′ (SEQ ID NO:125). The sequence is identical to nucleotides 1919 to 1959 of pCALG10-1. The nucleotide sequence of GNlacZ-Nde2 is as follows: 5′-CCAGTGAATCCGTAATCATGGTCATAGCTGTTTCCTG-TGTG-3′ (SEQ ID NO:126). The sequence is reverse complimentary to nucleotides 1919 to 1959 of pCALG10-1.</p>
    <p num="p-0667">The resultant recombinant pCALG31-1 plasmid contains Kan<sup>r</sup>, mphR sequences, full-length lacI, the lacUV5 promoter (with endogenous nucleotides immediately 5′ of the lacZ start codon), lacZ sequences, and the temperature sensitive replication origin from pMAK705.</p>
    <p num="p-0668">The recombinant pCALG31-1 was transformed into 7107-18 and clones with the lacUV5 promoter replacement were generated using the temperature selection procedure. To identify the correct clones, genomic DNA was prepared and the lacUV5 promoter region was PCR amplified. The PCR products were then sequenced to confirm the presence of the lacUV5 promoter. One strain generated a PCR product of the expected size and had the correct DNA sequence. This strain was named 7107-310.</p>
    <p num="h-0110">LacI Deletion and Replacement of the lac Promoter with the lacUV5 Promoter</p>
    <p num="p-0669">To generate the plasmid needed to delete lacI from the lac operon and replace the lac promoter with the lacUV5 promoter in N-acetylglucosamine and/or glucosamine producing strains two precursor plasmids were developed. To generate the first precursor plasmid, pCALG20-1 (see above) and pUC4K (Amersham Pharmacia Biotech, Catalog #27-4958-01, GenBank Accession # X06404) were digested with BamH I. The pCALG20-1 fragment containing mphR sequences, lacID, the lacUV5 promoter (with a Nde I site immediately 5′ of the lacZ start codon), lacZ sequences, and pCR®2.1-TOPO® and the pUC4K fragment containing Kan<sup>r </sup>were isolated and ligated together to generate pCALG26-1. The recombinant pCALG26-1 consists of Kan<sup>r</sup>, mphR sequences, lacID, the lacUV5 promoter (with a Nde I site immediately 5′ of the lacZ start codon), lacZ sequences, and pCR®2.1-TOPO®.</p>
    <p num="p-0670">To generate the next precursor plasmid, pCALG26-1 and pKLN07-20 (described elsewhere) were digested with Kpn I and Not I. The pCALG26-1 fragment containing Kan<sup>r</sup>, mphR sequences, lacID, the lacUV5 promoter (with a Nde I site immediately 5′ of the lacZ start codon), and lacZ sequences and the pKLN07-20 fragment containing the temperature sensitive replication origin from pMAK705 (described previously) were isolated and ligated together to generate pCALG30-1. The recombinant pCALG30-1 plasmid contains Kan<sup>r</sup>, mphR sequences, lacID, the lacUV5 promoter (with a Nde I site immediately 5′ of the lacZ start codon), lacZ sequences, and the temperature sensitive replication origin from pMAK705.</p>
    <p num="p-0671">After constructing pCALG30-1 it was thought that this would be the final integrative vector. However, it was subsequently determined out that addition of the Nde I site immediately 5′ of the lacZ start codon inhibited proper expression of lacZ. Therefore, pCALG30-1 was subjected to site-directed mutagenesis (Stratagene® QuikChange® XL Site-Directed Mutagenesis Kit, Catalog # 200517) to destroy the Nde I site and replace the Nde I nucleotide changes with the endogenous lacUV5 nucleotides. To make the necessary changes to pCALG30-1, pCALG30-1 was subjected to site-directed mutagenesis using oligonucleotides GNlacZ-Nde1 (SEQ ID NO:125) and GNlacZ-Nde2 (SEQ ID NO:126) to produce pCALG32-2. The resultant recombinant pCALG32-2 plasmid contains Kan<sup>r</sup>, mphR sequences, lacID, the lacUV5 promoter (with endogenous nucleotides immediately 5′ of the lacZ start codon), lacZ sequences, and the temperature sensitive region from pMAK705.</p>
    <p num="p-0672">The recombinant pCALG32-2 was transformed into 7107-18 and clones with the lacUV5 promoter replacement were generated using the temperature selection procedure. To identify the correct clones, genomic DNA was prepared and the lacUV5 promoter region was PCR amplified. The PCR products were then sequenced to confirm the presence of the lacUV5 promoter. Three strains generated the PCR product of the expected size and had the correct DNA sequence. These strains were named 7107-313, 7107-314, and 7107-315.</p>
    <p num="h-0111">LacI Deletion from the DE3 Element</p>
    <p num="p-0673">To delete the lacI gene from the DE3 element in the genome of production strains, the temperature sensitive selection method described in Example 13 was used. This strategy involved construction of an integrative vector to target the DE3 element for the lacI deletion. For the first step of construction, the region containing the lacI of the DE3 element was amplified by PCR from <i>E. coli </i>7107-73 genomic DNA. Primers were synthesized based on the published sequence of the T7 RNAP and the attB region of the <i>E. coli </i>genome (Blattner et al, 1997, Science 277(5331): 1453-1474). Forward primer 07-74 and reverse primer 07-48 were used for the amplification and had the following sequences: 07-74: 5′ GATCCCGGGAACGGACGATTAGAGATCACC 3′ (SEQ ID NO:127) and 07-48: 5′ GTCAGAGAAGTCGTTCTTAGCGATG 3′ (SEQ ID NO:128).</p>
    <p num="p-0674">Forward primer 07-74 added a Sma I site (CCCGGG, represented in nucleotides 4-9 of SEQ ID NO:127) and amplified from 1194 basepairs upstream of the attB site of the <i>E. coli </i>genome. Reverse primer 07-48 amplified from 36 basepairs upstream of the T7 gene 1 ATG start codon. The resulting ˜3.2 kb PCR fragment was ligated into vector pPCR-Script™SK(+) (Stratagene Cloning Systems, La Jolla, Calif.), generating plasmids pSW07-53 #7 and #17. DNA sequencing revealed that the PCR product contained the region of the genome upstream of the attB site plus the lacI lacZ′ fragment from the DE3 element.</p>
    <p num="p-0675">To generate the deletion of the lacI, plasmid pSW07-53#17 was digested with restriction endonucleases Mlu I and Sfo I to remove 640 basepair fragments of the lacI coding sequence. The remainder of the pSW07-53 plasmid was treated with T4 DNAP to produce blunt ends and then ligated onto itself, generating plasmid pSW07-55#13.</p>
    <p num="p-0676">The fragment containing the DE3 sequence with the lacI deletion was digested from plasmid pSW07-55#13 with restriction endonucleases Not I and Sal I. This fragment was ligated with the 4.2 kb Sal I and Not I fragment from pKLN07-21, containing the temperature sensitive replicon from pMAK705 (Hamilton et al., 1989, J. Bac. 171(9):4617-4622) and the kanamycin resistance cassette from pUC4K (Amersham Pharmacia Biotech, Piscataway, N.J.). The resulting plasmid, pKLN07-56#13, was used to generate a deletion of the lacI of the DE3 element on the <i>E. coli </i>chromosome.</p>
    <p num="p-0677">To generate a strain with the lacI deletion of the DE3 element, <i>E. coli </i>7107-18 was transformed with plasmid pSW07-56#13. Following temperature sensitive selection and passaging protocol, strains were screened using a standard PCR protocol for the presence of the deletion. Several strains identified by PCR were confirmed by high stringency Southern hybridization using a fragment containing a portion of the lac and lacZ fragment of the DE3 element as probe. These strains were designated 7107-84(1) through 7107-84(4).</p>
    <p num="h-0112">Glucosamine Production in Strains with lacID and/or the lacUV5 Promoter Replacement in Shake Flasks</p>
    <p num="p-0678">Strain 7107-84(1) contains only one functional lacI gene, as opposed to strain 7107-18 with two functional lacI genes. Shake Flask Screen 42 was conducted to determine the effect of the lacI deletion of strain 7107-84(1) on lactose induction. In the presence of excess glucose and lactose, induction by lactose should be inhibited in strain 7107-18 due to the LacI repressor protein binding to the lac operator, preventing transcription. This inhibition by glucose should be reduced in strain 7107-84(1), as less LacI repressor protein should be present to bind the lac operator, allowing increased transcription from the lac promoters. This should increase pools of the T7 RNAP in the cells, resulting in increased transcription from the T7lac-glmS*54 cassette. Ultimately, higher levels of glucosamine should result in strain 7107-84(1). Likewise, strains containing the lacI deletion is the lac operon (e.g. 7107-313, 7107-314 and 7107-315) should have higher levels of GlcN production upon lactose inductions.</p>
    <p num="p-0679"> <i>E. coli </i>strains with the lacI gene deleted from the lac operon (7107-310), with the lacI gene deleted from the DE3 element (7107-84#1), and with the lacI deletion from the lac operon in which the lac promoter was replaced with the lacUV5 promoter (7107-313, 7107-314, and 7107-315) were compared to strain 7107-18 in shake flask experiments. All strains were grown in flasks containing M9B medium: 6 g/l KH<sub>2</sub>PO<sub>4</sub>, 24 g/l K<sub>2</sub>HPO<sub>4</sub>, 1 g/l Na<sub>3</sub>Citrate-2H<sub>2</sub>O, 10 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>(pH adjusted to 7.4 by phosphate), and trace metals (0.2 mg/l FeSO<sub>4</sub>-7H<sub>2</sub>O, 0.015 mg/l ZnSO<sub>4</sub>-7H<sub>2</sub>O, 0.015 mg/l MnSO<sub>4</sub>—H<sub>2</sub>O, 0.001 mg/l CuSO<sub>4</sub>-5H<sub>2</sub>O, 0.001 mg/l NaMoO<sub>4</sub>-2H<sub>2</sub>O, 0.001 mg/l H<sub>3</sub>BO<sub>3</sub>, and 0.001 mg/l CoCl<sub>2</sub>-6H<sub>2</sub>O), supplemented with 0.6 g/l MgSO<sub>4</sub>-7H<sub>2</sub>O and 0.05 g/l CaCl<sub>2</sub>-2H<sub>2</sub>O. To test glucose repression the flasks contained various amounts of glucose and lactose. The amounts of glucose and lactose used in the flasks are shown in Table 4 and Table 5. The cultures were grown at 30° C., with shaking at 225 rpm, for 24 hours and were then placed at 25° C., with shaking at 225 rpm, for the remainder of the experiment. At 24 and 48 hours, each culture was adjusted to pH 7.0, glucose was added to the flasks to approximately 30 g/l, and 5 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added to the cultures in which the level of ammonia had fallen below 1 g/l. Samples were collected at 24 and 48 hours. At each time point the glucosamine concentration in the culture supernatant was measured using the modified Elson-Morgan assay as described in U.S. Pat. No. 6,372,457 B 1. Glucosamine levels are shown in Table 31.</p>
    <p num="p-0680">As expected, strains 7107-18, 7107-84(1), 7107-310 and 7107-313 performed similarly in the presence of glucose alone or lactose alone. Little GlcN production is seen in either strain when grown in glucose, as no inducer is present. When the strains were grown with lactose as inducer, significant accumulations of glucosamine occurred with both strains. However, when strains were grown in the presence of both glucose and lactose, removal of one copy of the lacI gene from the chromosome significantly impacted lactose induction and, therefore, glucosamine titers. Strain 7107-313 and 7107-84(1) achieved about 6-8 times the level of glucosamine seen in strain 7107-18 at the 48-hour timepoint. This confirms the concept that decreased LacI repressor protein allows increased transcription from lac promoters, resulting in increased levels of glucosamine production.</p>
    <p num="p-0681"> <tables id="TABLE-US-00037" num="00037"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 31</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Glucosamine concentration in different samples</td> </tr> <tr class="description-tr"> <td class="description-td">from Shake Flask Screen 42</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="168pt" align="left"> </colspec> <colspec colname="1" colwidth="49pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Glucosamine</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Concentration</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">(g l<sup>−1</sup>)</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="77pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain</td> <td class="description-td"> </td> <td class="description-td">Glucose</td> <td class="description-td">Lactose</td> <td class="description-td">24 </td> <td class="description-td">48</td> </tr> <tr class="description-tr"> <td class="description-td">Name</td> <td class="description-td">Genotype</td> <td class="description-td">(g l<sup>−1</sup>)</td> <td class="description-td">(g l<sup>−1</sup>)</td> <td class="description-td">hours</td> <td class="description-td">hours</td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="77pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="28pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">7107-18</td> <td class="description-td">galKΔ::T7-lac-glmS*54</td> <td class="description-td">0</td> <td class="description-td">40</td> <td class="description-td">1.5</td> <td class="description-td">4.3</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">30</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0.2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">30</td> <td class="description-td">20</td> <td class="description-td">0.1</td> <td class="description-td">0.4</td> </tr> <tr class="description-tr"> <td class="description-td">7107-84#1</td> <td class="description-td">galKΔ::T7-lac-glmS*54,</td> <td class="description-td">0</td> <td class="description-td">40</td> <td class="description-td">1.7</td> <td class="description-td">4.9</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">laclΔ at DE3 element</td> <td class="description-td">30</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0.2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">30</td> <td class="description-td">20</td> <td class="description-td">0.8</td> <td class="description-td">3.2</td> </tr> <tr class="description-tr"> <td class="description-td">7107-310</td> <td class="description-td">galKΔ::T7-lac-glmS*54,</td> <td class="description-td">0</td> <td class="description-td">40</td> <td class="description-td">1.8</td> <td class="description-td">4.4</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">lacUV5 promoter</td> <td class="description-td">30</td> <td class="description-td">0</td> <td class="description-td">0.3</td> <td class="description-td">0.2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">30</td> <td class="description-td">20</td> <td class="description-td">0.3</td> <td class="description-td">0.4</td> </tr> <tr class="description-tr"> <td class="description-td">7107-313</td> <td class="description-td">galKΔ::T7-lac-glmS*54, </td> <td class="description-td">0</td> <td class="description-td">40</td> <td class="description-td">1.6</td> <td class="description-td">4.2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">lacUV5 promoter,</td> <td class="description-td">30</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0.2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">laclΔat lac operon</td> <td class="description-td">30</td> <td class="description-td">20</td> <td class="description-td">0.7</td> <td class="description-td">2.3</td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
Integration of the T7lac-ScGNA1 Cassette in Strains with lacID and/or the lacUV5 Promoter Replacement
</p>
    <p num="p-0682">To evaluate the effect of the lacI deletion on glucose de-repression in NAG production strains, it was necessary to add the T7lac-ScGNA1 cassette to the chromosome of strain 7107-84(1) and 7107-84(2). The methods and protocols described for GNA1 cloning and integration by temperature sensitive selection with plasmid pSW07-68#5 were employed as detailed elsewhere. Resulting strains 7107-97 and 7107-98 were confirmed by standard high stringency Southern hybridization using the ScGNA1 coding sequence as probe as having the T7lac-ScGNA1 integrated at the site of the manXYZ deletion of the chromosome.</p>
    <p num="p-0683">Similarly, other versions of potentially glucose de-repressed strains were constructed. Strain 7107-310 was constructed by altering the promoter of the lac operon to the lacUV5 version of the promoter. Strains 7107-313, 7107-314, and 7107-315 were constructed by deleting the chromosomal copy of the lacI gene in addition to altering the promoter of the lac operon to the lacUV5 version of the promoter. To evaluate the effect of these mutations on glucose de-repression in a NAG production background, T7lac-ScGNA1 cassette was added to the chromosome of the strains. The methods and protocols described for GNA1 cloning and integration by temperature sensitive selection with plasmid pSW07-68#25 were employed for strains 7107-310 and 7107-313. Resulting strains 7107-129 and 7107-130 (from strain 710-310) and 7107-131 (from strain 7107-313) were confirmed as having the T7lac-ScGNA1 integrated by the site of the manXYZ deletion of the chromosome using high stringency Southern hybridization with the ScGNA1 coding sequence as the probe.</p>
    <p num="h-0113">N-Acetylglucosamine Production in Strains with lacID and/or the lacUV5 Promoter Replacement in Shake Flasks</p>
    <p num="p-0684">Screening was conducted to evaluate the effect of the lacI deletion of the DE3 element on glucose de-repression in a NAG production strain. Strains 7107-97 and 7107-98(2) were tested with varying levels of glucose and lactose, with or without ribose addition. NAG production strain 7107-92(1), with two copies of lacI, was included as a control. Cultures were grown in M9B medium (previously described) supplemented with 0.6 g/l MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g/l CaCl<sub>2</sub>-2H<sub>2</sub>O, varied concentrations of glucose and lactose, and 5 g/l yeast extract. Strains were initially grown on 10 g/l glucose with a switch to lactose utilization once the glucose was depleted. Excess glucose conditions (in the presence of lactose) were also used to determine sensitivity to glucose repression. Each variable was performed with or without ribose addition. Cultures were grown at 30° C. for 24 hours and then switched to 25° C. for the remainder of the experiment. At 24- and 48-hour timpoints, the pH was adjusted to 7.2 and glucose added to a total of 30 g/l. 5 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added at 24 hours and 48 hours to flasks in which levels had fallen below 1 g/l. Samples were analyzed for NAG production at 24, 48, and 72 hours.</p>
    <p num="p-0685">The control strain performed well in non-repressing conditions, yielding over 20 g/l NAG, but produced only about 5 g/l with excess glucose (Table 32). One mutant strain (7107-98) performed similarly to the control, but the other one, (7107-97), showed glucose resistance, producing over 20 g/l NAG even when excess glucose was present. In fact, this strain also appeared to outperform the control under non-repressing conditions in terms of NAG production and acetate accumulation. The addition of ribose to cultures did not significantly increase growth or NAG titers in this experiment. Overall results indicate that deletion of the lacI of the DE3 element does at least partially alleviate glucose repression, resulting in improved NAG titers in the production strain.</p>
    <p num="p-0686">
      <tables id="TABLE-US-00038" num="00038"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 32</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Effect of the lacl deletion of the DE3 element on glucose</td>
              </tr> <tr class="description-tr"> <td class="description-td">de-repression in the presence of varying amounts of</td>
              </tr> <tr class="description-tr"> <td class="description-td">glucose and lactose.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Initial Sugars (g l<sup>−1</sup>)</td>
                <td class="description-td"> </td>
                <td class="description-td">Acetate</td>
                <td class="description-td">NAG</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <colspec colname="7" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain</td>
                <td class="description-td">Glucose</td>
                <td class="description-td">Lactose</td>
                <td class="description-td">Ribose</td>
                <td class="description-td">OD<sub>600</sub> </td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="35pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">7107-92(1)</td>
                <td class="description-td">10</td>
                <td class="description-td">40</td>
                <td class="description-td">5</td>
                <td class="description-td">9.8</td>
                <td class="description-td">0</td>
                <td class="description-td">20.9</td>
              </tr> <tr class="description-tr"> <td class="description-td">(control)</td>
                <td class="description-td">30</td>
                <td class="description-td">20</td>
                <td class="description-td">5</td>
                <td class="description-td">14.0</td>
                <td class="description-td">6.2</td>
                <td class="description-td">5.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">40</td>
                <td class="description-td">20</td>
                <td class="description-td">5</td>
                <td class="description-td">12.0</td>
                <td class="description-td">6.2</td>
                <td class="description-td">4.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-97</td>
                <td class="description-td">10</td>
                <td class="description-td">40</td>
                <td class="description-td">5</td>
                <td class="description-td">15.0</td>
                <td class="description-td">0</td>
                <td class="description-td">26.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">30</td>
                <td class="description-td">20</td>
                <td class="description-td">5</td>
                <td class="description-td">15.5</td>
                <td class="description-td">0</td>
                <td class="description-td">25.4</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">40</td>
                <td class="description-td">20</td>
                <td class="description-td">5</td>
                <td class="description-td">14.0</td>
                <td class="description-td">2.6</td>
                <td class="description-td">21.7</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-98(2)</td>
                <td class="description-td">10</td>
                <td class="description-td">40</td>
                <td class="description-td">5</td>
                <td class="description-td">16.0</td>
                <td class="description-td">4</td>
                <td class="description-td">14.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">30</td>
                <td class="description-td">20</td>
                <td class="description-td">5</td>
                <td class="description-td">15.0</td>
                <td class="description-td">9.4</td>
                <td class="description-td">7.7</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">40</td>
                <td class="description-td">20</td>
                <td class="description-td">5</td>
                <td class="description-td">14.5</td>
                <td class="description-td">10.1</td>
                <td class="description-td">8.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-92(1)</td>
                <td class="description-td">10</td>
                <td class="description-td">40</td>
                <td class="description-td">None</td>
                <td class="description-td">10.5</td>
                <td class="description-td">0</td>
                <td class="description-td">17.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">30</td>
                <td class="description-td">20</td>
                <td class="description-td">None</td>
                <td class="description-td">14.5</td>
                <td class="description-td">6.4</td>
                <td class="description-td">5.7</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">40</td>
                <td class="description-td">20</td>
                <td class="description-td">None</td>
                <td class="description-td">14.5</td>
                <td class="description-td">6.9</td>
                <td class="description-td">5.7</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-97</td>
                <td class="description-td">10</td>
                <td class="description-td">40</td>
                <td class="description-td">None</td>
                <td class="description-td">12.5</td>
                <td class="description-td">3.4</td>
                <td class="description-td">16.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">30</td>
                <td class="description-td">20</td>
                <td class="description-td">None</td>
                <td class="description-td">14.5</td>
                <td class="description-td">0</td>
                <td class="description-td">24.5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">40</td>
                <td class="description-td">20</td>
                <td class="description-td">None</td>
                <td class="description-td">15.5</td>
                <td class="description-td">0</td>
                <td class="description-td">21.9</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-98(2)</td>
                <td class="description-td">10</td>
                <td class="description-td">40</td>
                <td class="description-td">None</td>
                <td class="description-td">14.5</td>
                <td class="description-td">7.3</td>
                <td class="description-td">12.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">30</td>
                <td class="description-td">20</td>
                <td class="description-td">None</td>
                <td class="description-td">14.0</td>
                <td class="description-td">7.8</td>
                <td class="description-td">7.7</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">40</td>
                <td class="description-td">20</td>
                <td class="description-td">None</td>
                <td class="description-td">14.5</td>
                <td class="description-td">9.0</td>
                <td class="description-td">7.3</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00049" class="description-td" colspan="7">(1)Strains: Control 7107-92(1): two lacl genes. 7107-97 and 7107-98(2): only one lacl gene (the one in DE3 was deleted).</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00050" class="description-td" colspan="7">(2)OD<sub>600</sub>, acetate levels, and NAG levels are from the 72-hour timepoint.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0687">Another screening was conducted to evaluate strains 7107-129,7107-130 and 7107-131 for glucose de-repression. NAG production strain 7107-92(1), with two copies of lac, was included as a control. Cultures were grown in M9B medium (previously described) supplemented with 0.6 g/l MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g/l CaCl<sub>2</sub>-2H<sub>2</sub>O, varied concentration of glucose and lactose, 5 g/l ribose, and 5 g/l yeast extract. Strains were initially grown on 10 g/l glucose with a switch to lactose once the glucose was depleted for confirmation of induction by lactose. Excess glucose conditions (in the presence of lactose) in which glucose was always present were also used to test strain 7107-131 to determine sensitivity to glucose with respect to induction. Cultures were grown at 30° C. for 24 hours and then switched to 25° C. for the remainder of the experiment. The pH was adjusted to 7.2 and glucose added to a total of 30 g/l per day at 24 and 48-hour timepoints. 5 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added at 24 hours and 48 hours to flasks in which levels had fallen below 1 g/l. Samples were analyzed for NAG production at 24, 48, and 72 hours.</p>
    <p num="p-0688">As seen in Table 33, one of the lacUV5 mutant strains (7107-130) and the lacUV5 mutant strain with the lacI deletion (7107-131) performed better than control strain 7107-92(1) when initially grown on glucose with a switch to lactose after glucose depletion. In conditions of excess glucose, strain 7107-131 outperformed control strain 7107-92(1), producing about 30% more NAG. This confirms that deletion of either copy of the lac will help alleviate glucose repression, allowing induction to occur in the presence of excess glucose and improving NAG titers.</p>
    <p num="p-0689">
      <tables id="TABLE-US-00039" num="00039"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 33</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">The effect of lacl deletion and/or the lacUV5 promoter</td>
              </tr> <tr class="description-tr"> <td class="description-td">on NAG production under conditions of</td>
              </tr> <tr class="description-tr"> <td class="description-td">glucose limiting or excess</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="70pt" align="center"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Initial Sugars (g l<sup>−1</sup>)</td>
                <td class="description-td"> </td>
                <td class="description-td">Acetate</td>
                <td class="description-td">NAG</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain</td>
                <td class="description-td">Glucose</td>
                <td class="description-td">Lactose</td>
                <td class="description-td">OD<sub>600</sub> </td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Glucose limiting conditions:</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="35pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">7107-92(1)</td>
                <td class="description-td">10</td>
                <td class="description-td">40</td>
                <td class="description-td">9.75</td>
                <td class="description-td">2.4</td>
                <td class="description-td">24.3</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-129</td>
                <td class="description-td">10</td>
                <td class="description-td">40</td>
                <td class="description-td">9.75</td>
                <td class="description-td">2.0</td>
                <td class="description-td">23.3</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-130</td>
                <td class="description-td">10</td>
                <td class="description-td">40</td>
                <td class="description-td">11.25</td>
                <td class="description-td">3.0</td>
                <td class="description-td">30.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-131</td>
                <td class="description-td">10</td>
                <td class="description-td">40</td>
                <td class="description-td">11.25</td>
                <td class="description-td">2.2</td>
                <td class="description-td">31.6</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Glucose excess conditions:</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="35pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">7107-92(1)</td>
                <td class="description-td">40</td>
                <td class="description-td">10</td>
                <td class="description-td">12.75</td>
                <td class="description-td">6.4</td>
                <td class="description-td">5.3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">40</td>
                <td class="description-td">20</td>
                <td class="description-td">13.5</td>
                <td class="description-td">6.5</td>
                <td class="description-td">5.8</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-131</td>
                <td class="description-td">40</td>
                <td class="description-td">10</td>
                <td class="description-td">15.0</td>
                <td class="description-td">5.7</td>
                <td class="description-td">15.4</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">40</td>
                <td class="description-td">10</td>
                <td class="description-td">13.87</td>
                <td class="description-td">6.0</td>
                <td class="description-td">14.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">40</td>
                <td class="description-td">20</td>
                <td class="description-td">15.75</td>
                <td class="description-td">4.0</td>
                <td class="description-td">21.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">40</td>
                <td class="description-td">20</td>
                <td class="description-td">14.25</td>
                <td class="description-td">5.0</td>
                <td class="description-td">18.6</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00051" class="description-td" colspan="6">(1)Strains: Control strain 7107-92(1): lacl(lac), lacl(DE3) 7107-129, 7107-130: lacUV5, lacl(lac), lacl(DE3) 7107-131: lacUV5, laclΔ(lac), lacl(DE3)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" id="FOO-00052" class="description-td" colspan="6">(2)OD<sub>600</sub>, acetate levels, and NAG levels are from the 72-hour timepoint.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 29</heading> <p num="p-0690">The following Example demonstrates the effects of restoration of galactose utilization on N-acetylglucosamine and/or glucosamine accumulation.</p>
    <p num="p-0691">Lactose-induced N-acetylglucosamine and/or glucosamine producing strains containing the galKΔ::T7-lac-glmS*54 construct are unable to use galactose as a carbon source due to the galKΔ. These strains accumulate galactose due to the cleavage of lactose into glucose and galactose by b-galactosidase. In an effort to decrease galactose accumulation in these strains the T7-lac-glmS*54 expression cassette needs to be integrated into a different chromosomal location. The previously constructed N-acetylglucosamine and/or glucosamine producing strains contain nagΔ::Tet<sup>r</sup>. The nagΔ::Tet<sup>r </sup>portion of the genome was transferred from the parent strain IBPC590 (Plumbridge (1991) <i>Mol. Microbiol</i>. 5, 2053-2062) by P1 phage transduction as described in U.S. Pat. No. 6,372,457 B1. The presence of Tet<sup>r </sup>in the production strains is not desirable. Therefore, replacement of Tet<sup>r </sup>with T7-lac-glmS*54 would not only serve as an integration site for T7-lac-glmS*54, it would also remove Tet<sup>r </sup>in the same process.</p>
    <p num="p-0692">Integration of the ΔgalkT7-lac-glmS*54 construct into 7101-17(DE3), to produce strain 7107-18 (described previously in Example 7), has led to a significant increase in glucosamine production. Additionally, integration of the manXYZΔ::T7-lac-Sc GNA1 construct into 7107-18 has led to production of N-acetylglucosamine, described previously in Example 16. To develop N-acetylglucosamine production strains capable of metabolizing galactose, T7-lac-glmS*54 expression cassette was integrated at the nagD::Tet<sup>r </sup>site, followed by the insertion of the T7-lac-Sc GNA1 expression cassette at the manXYZ site.</p>
    <p num="p-0693">The steps involved in the construction of strains containing nagΔ::T7-lac-glmS*54 and manXYZΔ::T7-lac-Sc GNA1 included the generation of three precursor plasmids. To generate the first precursor plasmid the T7-lac-glmS*54 fragment from pKLN23-54 (described in U.S. Pat. No. 6,372,457 B1) was PCR amplified using standard conditions with oligonucleotide primers GNglmSnagE3-5 (SEQ ID NO:129) and GNglmSnagE4-3′ (SEQ ID NO:130).</p>
    <p num="p-0694">GNglmSnagE3-5: 5′-GGATCTAAACCTCAGTAGCGACCGGTCTAGAACTA-GTG-3′ (SEQ ID NO:129) The primer is identical to nucleotides 1109 to 1146 of pKLN23-54 with the following exceptions: C1115A, C 1116A, GI 1120T, G1122A, G1125A, G1129A, and C1133G. The changes at nucleotides 1115 through 1125 of pKLN23-54 (corresponding to nucleotides 8, 12, 14, 17, 21 and 25 of SEQ ID NO:129) were made to increase stability of the primer in the PCR and the changes at 1129 and 1133 of pKLN23-54 add an Age I site to SEQ ID NO:129 (represented from position 21 to position 26 of SEQ ID NO:129).</p>
    <p num="p-0695">GNglmSnagE4-3 5′-CCCTCGCCCCTCTAGAGCATTTAAATTCAGTCAATT-AC-3′ (SEQ ID NO:130) The primer is reverse complimentary to nucleotides 3237 to 3274 of pKLN23-54 with the following exceptions: T3251A, G3253T, G3256A, A3270C, and T3273C (represented by positions 24, 22, 19, 5, and 2, respectively, of SEQ ID NO:130). The changes at 3251 through 3256 were made to add a Swa I site (represented by positions 19 through 26 of SEQ ID NO:130) and the changes at 3270 and 3273 were made to increase stability of the primer in the PCR.</p>
    <p num="p-0696">The resultant PCR product was ligated into pCR®-Blunt II-TOPO® (Invitrogen Zero Blunt® TOPO® PCR Cloning Kit, Catalog # K2800-20). The recombinant plasmid, pCALG38-2, contains the T7-lac-glmS*54 expression cassette flanked 5′ by an Age I site and 3′ by a Swa I site.</p>
    <p num="p-0697">To generate the second precursor plasmid, the nagΔ::Tet<sup>r</sup>::asn fragment was amplified from <i>E. coli </i>strain 7107-18 by PCR using GNnagEtetR1-5 (SEQ ID NO:131) and GNnagEtetR2-3 (SEQ ID NO:132). The nucleotide sequence of GNnagEtetR1-5 is as follows: 5′-CACGCAGGCAGGCTTTACCTTCTTC-3′ (SEQ ID NO:131) and that of GNnagEtetR2-3 is as follows: 5′-CGGAAGAACAAGCGACGGAAGGAC-3′ (SEQ ID NO:132).</p>
    <p num="p-0698">The PCR product was ligated into pCR®-Blunt II-TOPO® to generate the recombinant plasmid pCALG35-1.</p>
    <p num="p-0699">To generate the third precursor plasmid, pCALG38-2 was digested with Age I and Swa I and the T7-lac-glmS*54 fragment was purified. Additionally, pCALG35-1 was digested with Age I and Nru I and the asn-pCR®-Blunt II-TOPO®-nagD fragment was purified. The purified fragments from pCALG38-2 and pCALG35-1 were ligated to generate pCALG40. The recombinant pCALG40 plasmid contains the nagD::T7-lac-glmS*54::asn fragment in pCR®-Blunt II-TOPO®.</p>
    <p num="p-0700">To generate the final plasmid needed to integrate the recombinant nagD::T7-lac-glmS*54::asn fragment into the genomes the following work was performed. pCALG40 was digested with Kpn I and Not I and the fragment containing nagD::T7-lac-glmS*54::asn was isolated. Additionally, pKLN07-21 (described previously) was digested with Kpn I and Not I to isolate the fragment containing Kan<sup>r </sup>(from pUC4K, previously described) and the temperature sensitive replication origin (from pMAK705). The nagD::T7-lac-glmS*54::asn fragment from pCALG40 and the fragment of Kan<sup>r </sup>and temperature sensitive replication origin pKLN07-21 were ligated to generate pCALG43-2.</p>
    <p num="p-0701">The plasmid pCALG43-2 was transformed into 7101-17(DE3) and clones with nagD::T7-lac-glmS*54 in the genome were generated. The correct clones were identified by PCR and confirmed by Southern blot analysis. The clones were also confirmed by the loss of resistance to tetracycline.</p>
    <p num="p-0702">For PCR screening for the strains containing nagD::T7-lac-glmS*54::asn a pair of oligonucleotide primers were synthesized. The forward primer GNnagET7glmS 1-5 has the sequence of 5′-CAC GAT AAA CGG TGA AGC CAT GTC G-3′ (SEQ ID NO:133.) The reverse primer GNnagET7glmS2-3 has the sequence of 5′-CGT CCA TTT TCT TGA ACG CTT CAT CCC-3′ (SEQ ID NO:134.) The forward primer and the reverse primers are located 5′ and 3′, respectively, of the nagD::Tet<sup>r</sup>::asn PCR fragment generated from the oligonucleotides GNnagEtetR1-5 (SEQ ID NO:133) and GNnagEtetR2-3 (SEQ ID NO:134). Four strains were identified by PCR and named 7107-321#1, #2, #3, and #4.</p>
    <p num="p-0703">To confirm the nagΔ::T7-lac-glmS*54 integration in the 7107-321 strains, genomic DNA was isolated and analyzed by Southern blot using a nagE specific probe under standard conditions. The 7107-321 strains were found to be correct by Southern analysis. Additionally, the absence of nagΔ::Tet<sup>r</sup>::asn was confirmed due to the inability of the 7107-321 strains to grow on media that contained tetracycline.</p>
    <p num="p-0704">To add the manXYZΔ::T7-lac-Sc GNA1 construct, the recombinant pSW07-68 plasmid (described previously) was transformed into each of the 7107-321 strains. Clones with the manXYZΔ::T7-lac-Sc GNA1 integrated into the genome were generated using the temperature selection procedure. Clones were identified by PCR and confirmed by Southern blot analysis. Twelve strains generated the PCR products of the expected size. These strains were named 7107-325#1, #2 and #3, (derived from 7107-321#1); 7107-326#1, #2 and #3, (derived from 7107-321#2); 7107-327#1, #2, and #3 (derived from 7107-321#3); and 7107-328#1, #2, and#3 (derived from 7107-321#4). All these strains were confirmed to be correct by Southern blot analysis using a GNA1 specific probe.</p>
    <p num="h-0115">Evaluation of N-Acetylglucosamine Production Strains Capable of Metabolizing Galactose in Shake Flask Experiments</p>
    <p num="p-0705">N-acetylglucosamine production strains capable of metabolizing galactose (7107-325#1, 7107-326#1, 7107-327#1, and 7107-328#1) were tested in shake flask experiments (Table 34). Strains were tested in flasks containing M9B medium with supplementation of 0.6 g l<sup>−1 </sup>MgSO<sub>4</sub>-7H<sub>2</sub>O, and 0.05 g l<sup>−1 </sup>CaCl<sub>2</sub>-2H<sub>2</sub>O. Trace metal supplementation of screens 54 and 55 included 0.2 mg l<sup>−1 </sup>FeSO<sub>4</sub>-7H<sub>2</sub>O, 0.015 mg l<sup>−1 </sup>ZnSO<sub>4</sub>-7H<sub>2</sub>O, 0.015 mg l<sup>−1 </sup>MnSO<sub>4</sub>—H<sub>2</sub>O, 0.001 mg l<sup>−1 </sup>CuSO<sub>4</sub>-5H<sub>2</sub>O, 0.001 mg l<sup>−1 </sup>NaMoO<sub>4</sub>-2H<sub>2</sub>O, 0.001 mg l<sup>−1 </sup>H<sub>3</sub>BO<sub>3</sub>, and 0.001 mg l<sup>−1 </sup>CoCl<sub>2</sub>-6H<sub>2</sub>O while trace metal supplementation of screens 59 and 66 included 0.5 mg l<sup>−1 </sup>FeSO<sub>4</sub>-7H<sub>2</sub>O, 0.38 mg l<sup>−1 </sup>ZnSO<sub>4</sub>-7H<sub>2</sub>O, 0.033 mg l<sup>−1 </sup>MnSO<sub>5</sub>H<sub>2</sub>O, and 0.01 mg l<sup>−1 </sup>CoCl<sub>2</sub>-6H<sub>2</sub>O. As shown in Table 34, various amounts of glucose (Glu), lactose (Lac), yeast extract (YE), and Whey Permeate (WP, Formost Whey, Wisconsin) were used in the flasks. For screens 54 and 55, the cultures were grown at 30° C., with shaking at 225 rpm, for 24 hours and were then placed at 25° C., with shaking at 225 rpm, for the remainder of the experiment. For screens 59 and 66, the cultures were grown at 37° C., with shaking at 225 rpm, for 8 to 10 hours and were then placed at 30° C., with shaking at 225 rpm, for the remainder of the experiment; In screens 59 and 66, at 10 hours pH was adjusted to 7.2 and if the cultures were glucose depleted 20-25 g l<sup>−1 </sup>glucose was added. For all four screens, at 24 and 48 hours, each culture was adjusted to pH 7.2, glucose was added to the flasks to approximately 30 g l<sup>−1</sup>, and 5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added to the cultures in which the level of ammonia had fallen below 1 g l<sup>−1</sup>. In screen 66, at 30 and 54 hours glucose was added if necessary, the pH was adjusted to 7.2, and if the ammonia levels fell below 1 g l<sup>−1</sup>, 2.5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added. For all four screens, samples were collected at 24 and 48 hours and the N-acetylglucosamine and galactose concentrations in the culture supernatant were measured using an HPLC carbohydrate column.</p>
    <p num="p-0706">N-acetylglucosamine concentrations in different samples are shown in Table 34. It can be concluded from these experiments that under some conditions integration of T7lac-glmS*54 at the nagΔ site instead of the galK site improves N-acetylglucosamine production. Table 34 also shows galactose concentrations in different cultures. As expected, integration of T7-lac glmS*54 at the nagΔ site instead of the galK site abolished galactose accumulation.</p>
    <p num="p-0707"> <tables id="TABLE-US-00040" num="00040"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 34</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Levels of N-acetylglucosamine and galactose under different growth conditions</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="35pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="left"> </colspec> <colspec colname="3" colwidth="77pt" align="center"> </colspec> <colspec colname="4" colwidth="84pt" align="center"> </colspec> <colspec colname="5" colwidth="84pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Shake</td> <td class="description-td">Strain*</td> <td class="description-td">Addition to the medium</td> <td class="description-td">GlcNAc</td> <td class="description-td">Galactose</td> </tr> <tr class="description-tr"> <td class="description-td">Flask</td> <td class="description-td">Number</td> <td class="description-td">(g l<sup>−1</sup>)</td> <td class="description-td">(g l<sup>−1</sup>)</td> <td class="description-td">(g l<sup>−1</sup>)</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="12"> <colspec colname="1" colwidth="35pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="14pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="28pt" align="center"> </colspec> <colspec colname="8" colwidth="28pt" align="center"> </colspec> <colspec colname="9" colwidth="28pt" align="center"> </colspec> <colspec colname="10" colwidth="28pt" align="center"> </colspec> <colspec colname="11" colwidth="28pt" align="center"> </colspec> <colspec colname="12" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Screen #</td> <td class="description-td">(7107-)</td> <td class="description-td">Glu</td> <td class="description-td">Lac</td> <td class="description-td">YE</td> <td class="description-td">WP</td> <td class="description-td">24 hrs</td> <td class="description-td">48 hrs</td> <td class="description-td">72 hrs</td> <td class="description-td">24 hrs</td> <td class="description-td">48 hrs</td> <td class="description-td">72 hrs</td> </tr> <tr class="description-tr"> <td namest="1" nameend="12" align="center" rowsep="1" class="description-td" colspan="12"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="12"> <colspec colname="1" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="28pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="14pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="8" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="9" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="10" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="11" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="12" colwidth="28pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">54</td> <td class="description-td">Control</td> <td class="description-td">10</td> <td class="description-td">40</td> <td class="description-td">5</td> <td class="description-td">0</td> <td class="description-td">2.0</td> <td class="description-td">7.0</td> <td class="description-td">13.0</td> <td class="description-td">2.6</td> <td class="description-td">3.4</td> <td class="description-td">4.0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">325#1</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">3.6</td> <td class="description-td">9.9</td> <td class="description-td">14.6</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td">55</td> <td class="description-td">Control</td> <td class="description-td">10</td> <td class="description-td">40</td> <td class="description-td">5</td> <td class="description-td">0</td> <td class="description-td">2.4</td> <td class="description-td">9.6</td> <td class="description-td">16.0</td> <td class="description-td">2.9</td> <td class="description-td">3.3</td> <td class="description-td">3.5</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">325#1</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">4.1</td> <td class="description-td">12.3</td> <td class="description-td">17.0</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">326#1</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">4.1</td> <td class="description-td">11.9</td> <td class="description-td">16.6</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">327#1</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">4.1</td> <td class="description-td">11.4</td> <td class="description-td">15.2</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">328#1</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">4.1</td> <td class="description-td">14.6</td> <td class="description-td">21.5</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td">59</td> <td class="description-td">Control</td> <td class="description-td">8</td> <td class="description-td">20</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">11.3</td> <td class="description-td">20.5</td> <td class="description-td">26.3</td> <td class="description-td">1.6</td> <td class="description-td">2.7</td> <td class="description-td">3.6</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">10</td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td">10.9</td> <td class="description-td">21.3</td> <td class="description-td">24.3</td> <td class="description-td">1.6</td> <td class="description-td">2.3</td> <td class="description-td">2.9</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">5</td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td">11.8</td> <td class="description-td">21.3</td> <td class="description-td">26.2</td> <td class="description-td">1.6</td> <td class="description-td">2.2</td> <td class="description-td">2.0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">2.5</td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td">9.6</td> <td class="description-td">18.4</td> <td class="description-td">21.0</td> <td class="description-td">1.0</td> <td class="description-td">1.1</td> <td class="description-td">0.8</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">1.25</td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td">10.6</td> <td class="description-td">18.7</td> <td class="description-td">22.5</td> <td class="description-td">0.8</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td"> </td> <td class="description-td">10</td> <td class="description-td">12.3</td> <td class="description-td">21.7</td> <td class="description-td">26.3</td> <td class="description-td">1.8</td> <td class="description-td">3.1</td> <td class="description-td">3.5</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td"> </td> <td class="description-td">5</td> <td class="description-td">10.7</td> <td class="description-td">21.4</td> <td class="description-td">27.4</td> <td class="description-td">1.6</td> <td class="description-td">2.4</td> <td class="description-td">2.4</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td"> </td> <td class="description-td">2.5</td> <td class="description-td">12.6</td> <td class="description-td">21.7</td> <td class="description-td">28.3</td> <td class="description-td">1.5</td> <td class="description-td">1.4</td> <td class="description-td">1.2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">325#1</td> <td class="description-td"> </td> <td class="description-td">20</td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td">10.8</td> <td class="description-td">18.6</td> <td class="description-td">21.2</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">10</td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td">11.6</td> <td class="description-td">20.6</td> <td class="description-td">23.6</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">5</td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td">11.7</td> <td class="description-td">20.7</td> <td class="description-td">24.2</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">2.5</td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td">7.4</td> <td class="description-td">14.0</td> <td class="description-td">16.1</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">1.25</td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td">9.7</td> <td class="description-td">16.1</td> <td class="description-td">18.8</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td"> </td> <td class="description-td">10</td> <td class="description-td">11.2</td> <td class="description-td">19.2</td> <td class="description-td">22.2</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td"> </td> <td class="description-td">5</td> <td class="description-td">11.2</td> <td class="description-td">19.6</td> <td class="description-td">22.5</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">0</td> <td class="description-td"> </td> <td class="description-td">2.5</td> <td class="description-td">11.8</td> <td class="description-td">18.9</td> <td class="description-td">21.3</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td class="description-td">66</td> <td class="description-td">Control</td> <td class="description-td">8</td> <td class="description-td">10</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">10.1</td> <td class="description-td">23.9</td> <td class="description-td">26.2</td> <td class="description-td">1.2</td> <td class="description-td">1.6</td> <td class="description-td">1.9</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">328#1</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">12.1</td> <td class="description-td">26.7</td> <td class="description-td">28.8</td> <td class="description-td">0</td> <td class="description-td">0</td> <td class="description-td">0</td> </tr> <tr class="description-tr"> <td namest="1" nameend="12" align="center" rowsep="1" class="description-td" colspan="12"> </td> </tr> <tr class="description-tr"> <td namest="1" nameend="12" align="left" id="FOO-00053" class="description-td" colspan="12">*Control: non-galactose user strain 7107-92</td> </tr> <tr class="description-tr"> <td namest="1" nameend="12" align="left" id="FOO-00054" class="description-td" colspan="12">All other strains are galactose user siblings</td> </tr> <tr class="description-tr"> <td namest="1" nameend="12" align="left" id="FOO-00055" class="description-td" colspan="12">Glu = Glucose, Lac = Lactose, YE = yeast extract, WP = Whey Permeate</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Evaluation of N-Acetylglucosamine Production Strains Capable of Metabolizing Galactose in 1-Liter Fermentors
</p>
    <p num="p-0708">Strains capable of using galactose (7107-325#1 and 7107-328#1) were compared to galactose-non user (7107-92#1) in 1-liter fermentors. The fermentors were set up with an initial volume of 475 ml with fermentation medium: 4.79 g l<sup>−1 </sup>H<sub>3</sub>PO<sub>4</sub>, 3.15 g l<sup>−1 </sup>KOH, 3.56 g l<sup>−1 </sup>citric acid-H<sub>2</sub>O, 5 l<sup>−1 </sup>(NH<sub>2</sub>)<sub>2</sub>SO<sub>4</sub>, 2.5 g l<sup>−1 </sup>MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g l<sup>−1 </sup>CaCl<sub>2</sub>-2H<sub>2</sub>O, trace metals (5 mg l<sup>−1 </sup>FeSO<sub>4</sub>-7H<sub>2</sub>O, 3.75 mg l<sup>−1 </sup>ZnSO<sub>4</sub>-7H<sub>2</sub>O, 0.6 mg l<sup>l</sup> <sup>−1 </sup>CuSO<sub>4</sub>-5H<sub>2</sub>O, and 0.1002 mg l<sup>−1 </sup>CoCl<sub>2</sub>-6H<sub>2</sub>O), and 0.25 g l<sup>−1 </sup>Mazu 204 Antifoam. The pH was adjusted to 7.0 using 45% KOH. The pH (6.9) of the medium was maintained using 75% NH<sub>4</sub>OH, the temperature was maintained at 37° C., and aeration and agitation rates were used to maintain dissolved oxygen at a concentration of 20% saturation. A 65% glucose solution was fed to the cultures with the feed rate controlled by computer program to achieve a growth rate of 0.4 hr<sup>−1 </sup>at inoculation and a maximum rate of 5 ml h<sup>−1 </sup>by 6 hours. Cultures were induced with food grade lactose added at a concentration of 5 g l<sup>−1 </sup>at around 10 hours, with continued glucose feed.</p>
    <p num="p-0709">Seven samples were collected at approximately 10, 21, 28, 35, 45, 52, and 60 hours during the fermentation run. For each sample, the N-acetylglucosamine and galactose concentrations, in addition to the concentrations of several other components for the monitoring of the health of the culture, were determined using an HPLC carbohydrate column. As expected, while galactose was detected in the medium from control strain 7107-92#1 (0.6 g l<sup>−1 </sup>in Sample 7), galactose did not accumulate in the media harboring strains 7107-325#1 and 7107-328#1. Expressing GlmS*54 under these conditions, in a strain that is able to consume galactose, does not improve N-acetylglucosamine production. However, the medium used for this fermentation was optimized for 7107-92#1. The galactose-user strains may require somewhat different fermentation conditions for optimal performance. It is likely that a medium could be generated that would allow the 7107-325#1 and 7107-328#1 strains to produce a higher amount of N-acetylglucosamine than 7107-92#1. Therefore, it can be envisioned that integration of the T7-lac-glmS*54 expression cassette at the nagD site instead of at galK site could prove beneficial for N-acetylglucosamine and/or glucosamine production.</p>
    <p num="h-0116">Fermentation Process Development for N-Acetylglucosamine Production</p>
    <heading>Example 30</heading> <p num="p-0710">This Example describes shake flask experiments for optimizing NAG production with strains containing ScGNA1 plasmids.</p>
    <p num="p-0711">Having determined that overexpression of the ScGNA1 resulted in high levels of NAG in <i>E. coli </i>glucosamine production strains, the next step was to optimize conditions for the production of NAG. The approach taken was to increase cell density and therefore NAG titers without producing excess acetate in shake flask. Strain 7107-87#25, expressing the ScGNA1 from the T7 promoter of plasmid pET24d(+), was used for initial optimization experiments.</p>
    <p num="h-0118">Yeast Extract Addition and Later Induction</p>
    <p num="p-0712">Shake flask screening tested the effect of yeast extract addition on growth, acetate accumulation, and NAG production in strain 7107-87#25. It also tested the effect of earlier versus later induction with 0.2 mM IPTG. Strains were grown in M9B medium (previously described), but with the (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>decreased to 7.5 g l<sup>1</sup>. The M9B medium was supplemented with 30 g l<sup>−1 </sup>glucose, 0.6 g l<sup>−1 </sup>MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g l<sup>−1 </sup>CaCl <sub>2</sub>-2 H<sub>2</sub>O, and 25 mg/ml kanamycin. 0.2 mM IPTG was added to all flasks except those for late induction, to which it was added 24 hours after inoculation. To the control flask no yeast extract was added. For the test flasks, yeast extract (ranging from 0.5 to 4.0 g l<sup>−1</sup>) was added at the start or at 24 hours after inoculation. Cultures were grown at 30° C. until 24 hours and then placed at 25° C. At 24 and 48 hours, 5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>and 20 g l<sup>−1 </sup>glucose were added to each flask and pH was adjusted to 7.2.</p>
    <p num="p-0713">As seen in Table 35, addition of yeast extract resulted in increased growth and NAG production. In fact, the flask with 4 g l<sup>−1 </sup>yeast extract added at the start performed the best, achieving 8.9 g l<sup>−1 </sup>NAG, compared with 2.9 g l<sup>−1 </sup>NAG produced in the control flask. However, 3.0 g l<sup>−1 </sup>acetate also accumulated in these flasks. Flasks with yeast extract added after 24 hours grew nearly as well as those with yeast extract added at the start and accumulated less acetate. However, under these conditions NAG production reached only at 4.9 g l<sup>−1 </sup>at 48 hours in the flask with 4 g l<sup>−1 </sup>yeast extract. Moreover, the flasks with later induction grew better than those initially induced with 0.2 mM IPTG, although NAG production was delayed. This suggests that induction of N-acetylglucosamine synthesis has some negative effects on cell growth.</p>
    <p num="p-0714">
      <tables id="TABLE-US-00041" num="00041"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 35</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Effect of yeast extract addition and later induction on growth,</td>
              </tr> <tr class="description-tr"> <td class="description-td">acetate accumulation, and NAG production in strain</td>
              </tr> <tr class="description-tr"> <td class="description-td">7107-87#25.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="112pt" align="center"> </colspec> <colspec colname="2" colwidth="105pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Conditions</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Yeast Extract</td>
                <td class="description-td">Time of yeast</td>
                <td class="description-td"> </td>
                <td class="description-td">Acetate</td>
                <td class="description-td">GlcNAc</td>
              </tr> <tr class="description-tr"> <td class="description-td">(g l<sup>−1</sup>)</td>
                <td class="description-td">extract addition</td>
                <td class="description-td">OD<sub>600</sub> </td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">IPTG added at the start:</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="35pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">None (control)</td>
                <td class="description-td"> </td>
                <td class="description-td">2.7</td>
                <td class="description-td">0</td>
                <td class="description-td">2.9</td>
              </tr> <tr class="description-tr"> <td class="description-td">0.5</td>
                <td class="description-td">Start (0 hr)</td>
                <td class="description-td">3.2</td>
                <td class="description-td">0</td>
                <td class="description-td">2.1</td>
              </tr> <tr class="description-tr"> <td class="description-td">1.0</td>
                <td class="description-td">Start (0 hr)</td>
                <td class="description-td">4.0</td>
                <td class="description-td">0</td>
                <td class="description-td">2.9</td>
              </tr> <tr class="description-tr"> <td class="description-td">2.0</td>
                <td class="description-td">Start (0 hr)</td>
                <td class="description-td">5.7</td>
                <td class="description-td">1.9</td>
                <td class="description-td">5.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">4.0</td>
                <td class="description-td">Start (0 hr)</td>
                <td class="description-td">8.4</td>
                <td class="description-td">3.0</td>
                <td class="description-td">8.9</td>
              </tr> <tr class="description-tr"> <td class="description-td">0.5</td>
                <td class="description-td">24 hr</td>
                <td class="description-td">2.3</td>
                <td class="description-td">0</td>
                <td class="description-td">2.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">1.0</td>
                <td class="description-td">24 hr</td>
                <td class="description-td">2.7</td>
                <td class="description-td">0</td>
                <td class="description-td">2.8</td>
              </tr> <tr class="description-tr"> <td class="description-td">2.0</td>
                <td class="description-td">24 hr</td>
                <td class="description-td">4.2</td>
                <td class="description-td">0</td>
                <td class="description-td">3.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">4.0</td>
                <td class="description-td">24 hr</td>
                <td class="description-td">6.3</td>
                <td class="description-td">0</td>
                <td class="description-td">4.9</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">IPTG added at 24 hours:</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="35pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">None</td>
                <td class="description-td"> </td>
                <td class="description-td">6.6</td>
                <td class="description-td">2.8</td>
                <td class="description-td">0</td>
              </tr> <tr class="description-tr"> <td class="description-td">2.0</td>
                <td class="description-td">Start (0 hr)</td>
                <td class="description-td">6.9</td>
                <td class="description-td">4.0</td>
                <td class="description-td">3.1</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00056" class="description-td" colspan="5">1) Results taken from 48-hour timepoint.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0715">Addition of Various Sugars and Yeast Extract</p>
    <p num="p-0716">Shake flask screening was conducted to evaluate the effect of various sugars with yeast extract on growth, acetate accumulation, and NAG production in strain 7107-87#25. Strains were grown in M9B medium (previously described), but with the (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>decreased to 7.5 g l<sup>−1</sup>. The M9B medium was supplemented with 0.6 g l<sup>−1 </sup>MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g l<sup>−1 </sup>CaCl<sub>2</sub>-2H<sub>2</sub>O, and 25 mg/ml kanamycin. Due to the positive effects of yeast extract seen in previous experiment, 5 g l<sup>−1 </sup>yeast extract was included in all flasks. 0.2 mM IPTG and varying mixes and concentrations of the sugars lactose, glucose, fructose, and ribose were added to the flasks as indicated in Table 36. It is important to note that strain 7107-87#25 grown in the presence of lactose does not require IPTG for induction, as it is lactose-inducible. Cultures were grown at 30° C. for 24 hours and then shifted to 25° C.</p>
    <p num="p-0717">At about 24 hours, the pH of each culture was adjusted to 7.2. 5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added to each flask and glucose was added to about 30 g l<sup>−1 </sup>per day total based on HPLC results. At 32.5 hours, 15 g l<sup>−1 </sup>glucose, 2.5 g l<sup>−1 </sup>ribose, and 2.5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>were added to flask 7. At 48 hours, 10 g l<sup>−1 </sup>glucose was added to each flask. An additional 2.5 g l<sup>−1 </sup>ribose and 5.0 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added to flask 7 at 49.5 hours. Samples were from the flasks at about 24, 48, and 72 hours to monitor OD<sub>600</sub>, N-acetyl glucosamine levels, and acetate levels.</p>
    <p num="p-0718">
      <tables id="TABLE-US-00042" num="00042"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 36</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Effect of various sugar mixes on growth, acetate</td>
              </tr> <tr class="description-tr"> <td class="description-td">accumulation, and NAG production in strain 7107-87#25.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="91pt" align="center"> </colspec> <colspec colname="2" colwidth="105pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Conditions</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="63pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <colspec colname="7" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Glucose</td>
                <td class="description-td"> </td>
                <td class="description-td">IPTG</td>
                <td class="description-td"> </td>
                <td class="description-td">Acetate</td>
                <td class="description-td">GlcNAc</td>
              </tr> <tr class="description-tr"> <td class="description-td">Flask</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
                <td class="description-td">Other sugars (g l<sup>−1</sup>)</td>
                <td class="description-td">(mM)</td>
                <td class="description-td">OD<sub>600</sub> </td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="63pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="28pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">1</td>
                <td class="description-td">0</td>
                <td class="description-td">lactose (40)</td>
                <td class="description-td">0</td>
                <td class="description-td">10.0</td>
                <td class="description-td">2.5</td>
                <td class="description-td">9.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">2</td>
                <td class="description-td">10</td>
                <td class="description-td">lactose (30)</td>
                <td class="description-td">0</td>
                <td class="description-td">13.5</td>
                <td class="description-td">0</td>
                <td class="description-td">13.7</td>
              </tr> <tr class="description-tr"> <td class="description-td">3</td>
                <td class="description-td">0</td>
                <td class="description-td">None</td>
                <td class="description-td">0.2</td>
                <td class="description-td">11.0</td>
                <td class="description-td">4.7</td>
                <td class="description-td">12.3</td>
              </tr> <tr class="description-tr"> <td class="description-td">4</td>
                <td class="description-td">0</td>
                <td class="description-td">fructose (30)</td>
                <td class="description-td">0.2</td>
                <td class="description-td">5.0</td>
                <td class="description-td">0</td>
                <td class="description-td">9.1</td>
              </tr> <tr class="description-tr"> <td class="description-td">5</td>
                <td class="description-td">0</td>
                <td class="description-td">ribose (20)</td>
                <td class="description-td">0.2</td>
                <td class="description-td">11.0</td>
                <td class="description-td">4.8</td>
                <td class="description-td">10.6</td>
              </tr> <tr class="description-tr"> <td class="description-td">6</td>
                <td class="description-td">15</td>
                <td class="description-td">fructose (15)</td>
                <td class="description-td">0.2</td>
                <td class="description-td">11.0</td>
                <td class="description-td">4.4</td>
                <td class="description-td">14.8</td>
              </tr> <tr class="description-tr"> <td class="description-td">7</td>
                <td class="description-td">20</td>
                <td class="description-td">ribose (10)</td>
                <td class="description-td">0.2</td>
                <td class="description-td">13.5</td>
                <td class="description-td">3.1</td>
                <td class="description-td">27.0</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00057" class="description-td" colspan="7">1) Results taken from 72-hour timepoint.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00058" class="description-td" colspan="7">2) Yeast extract (5.0 g l<sup>−1</sup>) was included in all flasks.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0719">Results shown in Table 36 indicate that three sugars (lactose, glucose and ribose) supported growth well and resulted in significant accumulation of NAG. Flasks containing fructose did not achieve a high OD<sub>600</sub>, although NAG titer reached 9.1 g l<sup>−1</sup>. Initial inclusion of glucose in flasks containing fructose or lactose improved growth as well as NAG titer. However, the best result was achieved in the flask initially containing both glucose and ribose.</p>
    <p num="p-0720">Literature describes <i>E. coli </i>isolates incapable of growing on NAG or GlcN (J. Bac., 1970, 101:384-391). The authors speculated that the accumulation of amino sugar phosphates may inhibit the reactions catalyzed by phosphohexose isomerase and glucose-6-phosphate dehydrogenase, resulting in pentose starvation. Addition of pentoses or gluconate reversed the growth inhibition of the mutants. NAG-producing strain 7107-87#25 may also accumulate amino sugar phosphates (GlcN-6-P and/or GlcNAc-6-P), thereby resulting in pentose starvation. If this is the case, addition of ribose or gluconate should result in improved growth and NAG production. Indeed, growth and N-acetylglucosamine were improved by addition of ribose. In this experiment, the flask containing 20 g l<sup>−1 </sup>glucose plus 10 g l<sup>−1 </sup>ribose produced the highest level of NAG, achieving 27 g l<sup>−1</sup>. This is twice the amount produced in the flask containing 30 g l<sup>−1 </sup>glucose. Growth was also improved in the flask containing both ribose and glucose</p>
    <p num="h-0119">Glucose and Ribose Levels</p>
    <p num="p-0721">Previous experiments indicated that the addition of ribose to flasks containing glucose has a significant positive effect on NAG production in strain 7107-87#25. Shake Flask Screen #35 was conducted to test the effect of different glucose/ribose mixes on growth and NAG production. Strain 7107-87#25 was grown in M9B medium (previously described), supplemented with 0.6 g l<sup>−1 </sup>MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g l<sup>−1 </sup>CaCl<sub>2</sub>-2H<sub>2</sub>kanamycin, 0.2 mM IPTG, and 5.0 g l<sup>−1 </sup>yeast extract. Varying mixes of glucose and ribose were added to each flask as listed in Table 37. Cultures were grown at 30° C. for 24 hours and then shifted to 25° C. At about 24 and 48 hours, the pH of the flasks was adjusted to 7.2. An addition of 5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was made to all flasks at 24 hours. An additional 2.5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added to flasks 4 through 9 and an additional 5.0 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added to flasks 10 through 15 at 29 hours. At about 48 hours, 5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added to flasks 7 through 9, and 2.5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added to flasks 10 through 15. Glucose was added to adjust the glucose level to about 30 g l<sup>−1 </sup>at 24 and 48 hr. Samples were removed from the flasks at about 24, 29, 48, and 72 hours to monitor OD<sub>600</sub>, N-acetyl glucosamine levels, and acetate levels. Selected flasks were harvested at 72 hours for enzyme analysis.</p>
    <p num="p-0722">As seen on Table 37, glucose concentration did not significantly affect NAG production, as long as there was residual glucose. On the other hand, as little as 5 g l<sup>−1 </sup>ribose resulted in a large increase in NAG levels, reaching about 30 g l<sup>−1 </sup>by 72 hours in flasks 4, 5, and 6 compared to about 14 g l<sup>−1 </sup>in flasks without ribose. The best results were seen with 10 g l<sup>−1 </sup>ribose. In these flasks NAG levels reached about 36 g l<sup>−1 </sup>at 72 hours. Growth was also positively affected in cultures containing ribose. Enzyme analysis from selected flasks revealed that ribose addition did not directly affect GlmS or GNA1 activity, as activity levels for both enzymes were the same with or without ribose in the medium. This suggests that the effect of ribose addition was most likely a result of the relief from the shortage in pentose phosphate pathway intermediates.</p>
    <p num="p-0723"> <tables id="TABLE-US-00043" num="00043"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 37</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Effect of glucose and ribose on growth, acetate</td> </tr> <tr class="description-tr"> <td class="description-td">accumulation, and NAG production in strain 7107-87#25.</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="161pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Enzyme</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Activity</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="84pt" align="center"> </colspec> <colspec colname="3" colwidth="56pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Conditions</td> <td class="description-td"> </td> <td class="description-td">(μmol min<sup>−1 </sup> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <colspec colname="7" colwidth="56pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">glucose</td> <td class="description-td">ribose</td> <td class="description-td"> </td> <td class="description-td">Acetate</td> <td class="description-td">GlcNAc</td> <td class="description-td">mg<sup>−1 </sup>protein)</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <colspec colname="7" colwidth="28pt" align="center"> </colspec> <colspec colname="8" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Flask</td> <td class="description-td">(g l<sup>−1</sup>)</td> <td class="description-td">(g l<sup>−1</sup>)</td> <td class="description-td">OD<sub>600</sub> </td> <td class="description-td">(g l<sup>−1</sup>)</td> <td class="description-td">(g l<sup>−1</sup>)</td> <td class="description-td">GlmS</td> <td class="description-td">GNA1</td> </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="8" colwidth="28pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">1</td> <td class="description-td">20</td> <td class="description-td">None</td> <td class="description-td">12.5</td> <td class="description-td">5.4</td> <td class="description-td">14.0</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">2</td> <td class="description-td">30</td> <td class="description-td">None</td> <td class="description-td">13.0</td> <td class="description-td">5.4</td> <td class="description-td">14.7</td> </tr> <tr class="description-tr"> <td class="description-td">3</td> <td class="description-td">40</td> <td class="description-td">None</td> <td class="description-td">12.8</td> <td class="description-td">5.5</td> <td class="description-td">14.5</td> <td class="description-td">0.114</td> <td class="description-td">7.8</td> </tr> <tr class="description-tr"> <td class="description-td">4</td> <td class="description-td">20</td> <td class="description-td">5</td> <td class="description-td">15.6</td> <td class="description-td">0</td> <td class="description-td">28.1</td> </tr> <tr class="description-tr"> <td class="description-td">5</td> <td class="description-td">30</td> <td class="description-td">5</td> <td class="description-td">14.5</td> <td class="description-td">2.9</td> <td class="description-td">30.6</td> </tr> <tr class="description-tr"> <td class="description-td">6</td> <td class="description-td">40</td> <td class="description-td">5</td> <td class="description-td">14.5</td> <td class="description-td">3.1</td> <td class="description-td">29.0</td> <td class="description-td">0.140</td> <td class="description-td">7.9</td> </tr> <tr class="description-tr"> <td class="description-td">7</td> <td class="description-td">20</td> <td class="description-td">10</td> <td class="description-td">16.0</td> <td class="description-td">2.8</td> <td class="description-td">35.2</td> </tr> <tr class="description-tr"> <td class="description-td">8</td> <td class="description-td">30</td> <td class="description-td">10</td> <td class="description-td">14.5</td> <td class="description-td">0</td> <td class="description-td">37.1</td> </tr> <tr class="description-tr"> <td class="description-td">9</td> <td class="description-td">40</td> <td class="description-td">10</td> <td class="description-td">15.0</td> <td class="description-td">0</td> <td class="description-td">36.4</td> <td class="description-td">0.105</td> <td class="description-td">8.0</td> </tr> <tr class="description-tr"> <td class="description-td">10</td> <td class="description-td">20</td> <td class="description-td">15</td> <td class="description-td">15.5</td> <td class="description-td">4.3</td> <td class="description-td">31.3</td> </tr> <tr class="description-tr"> <td class="description-td">11</td> <td class="description-td">30</td> <td class="description-td">15</td> <td class="description-td">16.5</td> <td class="description-td">0</td> <td class="description-td">33.4</td> </tr> <tr class="description-tr"> <td class="description-td">12</td> <td class="description-td">40</td> <td class="description-td">15</td> <td class="description-td">16.0</td> <td class="description-td">0</td> <td class="description-td">34.0</td> <td class="description-td">0.101</td> <td class="description-td">8.2</td> </tr> <tr class="description-tr"> <td class="description-td">13</td> <td class="description-td">20</td> <td class="description-td">20</td> <td class="description-td">15.0</td> <td class="description-td">4.4</td> <td class="description-td">30.0</td> </tr> <tr class="description-tr"> <td class="description-td">14</td> <td class="description-td">30</td> <td class="description-td">20</td> <td class="description-td">15.0</td> <td class="description-td">4.7</td> <td class="description-td">26.3</td> </tr> <tr class="description-tr"> <td class="description-td">15</td> <td class="description-td">40</td> <td class="description-td">20</td> <td class="description-td">16.0</td> <td class="description-td">4.5</td> <td class="description-td">28.4</td> <td class="description-td">0.121</td> <td class="description-td">7.7</td> </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td> </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="left" id="FOO-00059" class="description-td" colspan="8">1) Results obtained from 72-hour timepoint.</td> </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="left" id="FOO-00060" class="description-td" colspan="8">2) Yeast extract (5.0 g l<sup>−1</sup>) was included in all flasks.</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Ribose and Other Intermediates of the Pentose Phosphate Pathway
</p>
    <p num="p-0724">In light of the positive results seen with cultures grown in glucose and ribose, Shake Flask Screen #36 was conducted to determine the effect of several other 5-carbon sugars and gluconate on growth and NAG production. Strain 7107-87#25 was grown in M9B medium (previously described) supplemented with 0.6 g l<sup>−1 </sup>MgSO<sub>4</sub>-7H<sub>2</sub>O, 0.05 g l<sup>−1 </sup>CaCl<sub>2</sub>-2H<sub>2</sub>O, 5.0 g l<sup>−1 </sup>yeast extract, 25 mg/ml kanamycin, and 0.2 mM IPTG. Glucose, ribose, xylose, arabinose, and gluconic acid (potassium salt) were added to flasks as indicated in Table 38. Cultures were grown at 30° C. for 24 hours and then shifted to 25° C. At 24 and 48 hours, pH was adjusted to 7.2. At about 24 hours, glucose was added to about 30 g l<sup>−1 </sup>per day total based on HPLC results from the 24 hours sample. 5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was also added to all flasks. 10 g l<sup>−1 </sup>additional ribose was added to flasks 11 and 12 at both 24 and 48 hours, and 5 g l<sup>−1 </sup>ribose was added to flasks 3 and 4 at 48 hours. An additional 10 g l<sup>−1 </sup>gluconic acid was added to flask 10 at 28.5 hours. At about 48 hours, glucose was added to adjust glucose levels to about 30 g l<sup>−1</sup>. Additionally, 5 g l<sup>−1 </sup>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>was added to flasks 11 and 12. Samples were removed from the each flask at about 24, 30, 48, 54 and 72 hours to monitor OD<sub>600</sub>, N-acetyl glucosamine levels, and acetate levels. Selected flasks were harvested at 72 hours for enzyme analysis.</p>
    <p num="p-0725">
      <tables id="TABLE-US-00044" num="00044"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 38</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Effects of 5-carbon compounds and glucuronic acid</td>
              </tr> <tr class="description-tr"> <td class="description-td">on growth and N-acetylglucosamine production.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="168pt" align="left"> </colspec> <colspec colname="1" colwidth="49pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Enzyme</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="center"> </colspec> <colspec colname="2" colwidth="84pt" align="center"> </colspec> <colspec colname="3" colwidth="49pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Conditions</td>
                <td class="description-td"> </td>
                <td class="description-td">Activity</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <colspec colname="7" colwidth="49pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Other</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(μmol min<sup>−1</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">glucose</td>
                <td class="description-td">sugars</td>
                <td class="description-td"> </td>
                <td class="description-td">Acetate</td>
                <td class="description-td">GlcNAc</td>
                <td class="description-td">mg<sup>−1 </sup>protein)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Flask</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
                <td class="description-td">(10 g l<sup>−1</sup>)</td>
                <td class="description-td">OD<sub>600</sub> </td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
                <td class="description-td">(g l<sup>−1</sup>)</td>
                <td class="description-td">GlmS</td>
                <td class="description-td">GNA1</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="8" colwidth="28pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">1</td>
                <td class="description-td">30</td>
                <td class="description-td">None</td>
                <td class="description-td">11.0</td>
                <td class="description-td">4.5</td>
                <td class="description-td">14.6</td>
                <td class="description-td">.230</td>
                <td class="description-td">11.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">2</td>
                <td class="description-td">40</td>
                <td class="description-td">None</td>
                <td class="description-td">12.5</td>
                <td class="description-td">4.8</td>
                <td class="description-td">15.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">3</td>
                <td class="description-td">30</td>
                <td class="description-td">Ribose</td>
                <td class="description-td">14.0</td>
                <td class="description-td">1.6</td>
                <td class="description-td">29.1</td>
                <td class="description-td">.215</td>
                <td class="description-td">12.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">4</td>
                <td class="description-td">40</td>
                <td class="description-td">Ribose</td>
                <td class="description-td">14.0</td>
                <td class="description-td">1.2</td>
                <td class="description-td">30.8</td>
              </tr> <tr class="description-tr"> <td class="description-td">5</td>
                <td class="description-td">30</td>
                <td class="description-td">Xylose</td>
                <td class="description-td">12.0</td>
                <td class="description-td">4.1</td>
                <td class="description-td">17.1</td>
                <td class="description-td">.211</td>
                <td class="description-td">12.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">6</td>
                <td class="description-td">40</td>
                <td class="description-td">Xylose</td>
                <td class="description-td">11.5</td>
                <td class="description-td">4.1</td>
                <td class="description-td">17.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">7</td>
                <td class="description-td">30</td>
                <td class="description-td">Arabinose</td>
                <td class="description-td">12.5</td>
                <td class="description-td">3.2</td>
                <td class="description-td">20.3</td>
                <td class="description-td">.180</td>
                <td class="description-td">11.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">8</td>
                <td class="description-td">40</td>
                <td class="description-td">Arabinose</td>
                <td class="description-td">12.5</td>
                <td class="description-td">3.3</td>
                <td class="description-td">20.4</td>
              </tr> <tr class="description-tr"> <td class="description-td">9</td>
                <td class="description-td">30</td>
                <td class="description-td">Gluconic</td>
                <td class="description-td">12.0</td>
                <td class="description-td">4.0</td>
                <td class="description-td">24.3</td>
                <td class="description-td">.190</td>
                <td class="description-td">9.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">acid</td>
              </tr> <tr class="description-tr"> <td class="description-td">10</td>
                <td class="description-td">40</td>
                <td class="description-td">Gluconic</td>
                <td class="description-td">13.0</td>
                <td class="description-td">3.8</td>
                <td class="description-td">22.1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">acid</td>
              </tr> <tr class="description-tr"> <td class="description-td">11</td>
                <td class="description-td">30</td>
                <td class="description-td">Ribose +</td>
                <td class="description-td">13.0</td>
                <td class="description-td">2.8</td>
                <td class="description-td">25.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">ribose</td>
              </tr> <tr class="description-tr"> <td class="description-td">12</td>
                <td class="description-td">40</td>
                <td class="description-td">Ribose +</td>
                <td class="description-td">13.0</td>
                <td class="description-td">3.0</td>
                <td class="description-td">24.9</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">ribose</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="left" id="FOO-00061" class="description-td" colspan="8">1) Results are from 72-hour timepoint.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="left" id="FOO-00062" class="description-td" colspan="8">2) Additional ribose (10 g l<sup>−1</sup>) was added to flasks 11 and 12 at 24 and 48 hours.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="left" id="FOO-00063" class="description-td" colspan="8">3) Yeast extract (5.0 g l<sup>−1</sup>) was included in all flasks.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0726">Addition of pentoses and gluconic acid resulted in increased NAG titer and improved growth when compared with the control (<figref idrefs="DRAWINGS">FIG. 18</figref>). These cultures also contained lower acetate levels than the control. However, the addition of ribose resulted in the highest NAG production. Gluconic acid is less expensive than the other sugars, and is therefore attractive for use at industrial scale. Because gluconic acid addition resulted in increased NAG production compared to the control strain, this experiment confirms its potential value in relieving pentose starvation in industrial fermentations. Enzyme activity levels from selected flasks indicate that the dramatic effect of ribose and gluconate on N-acetylglucosamine production can not be attributed to an influence on the activity of GlmS or GNA1. As seen in shake flask screening comparing the effects of different levels of glucose/ribose on growth and production, activity levels are similar with or without ribose in the culture medium.</p>
    <heading>Example 31</heading> <p num="p-0727">This example describes experiments conducted in 1-liter fermentors to optimize NAG production using the strain 7107-87 #25 which contained ScGNA1 plasmids.</p>
    <p num="h-0121">Timing of IPTG Induction:</p>
    <p num="p-0728">A fermentation experiment was conducted to evaluate induction with IPTG from the beginning and after 24 hours of growth (cell mass at about 13 g l<sup>−1</sup>). Growth was greatly inhibited by early induction, which also led to very low NAG titers (&lt;5 g/l) compared to late induction (50 g l<sup>−1 </sup>by 70 hours) as shown in <figref idrefs="DRAWINGS">FIG. 19</figref>. It is believed that the growth was inhibited because amino sugar phosphates accumulated in the cell inhibited specific enzymes of the pentose phosphate pathway. This hypothesis is supported by findings of positive effects of pentose phosphate intermediates on cell growth and NAG production.</p>
    <p num="h-0122">Effect of Pentose Phosphate Pathway (PPP) Intermediates:</p>
    <p num="p-0729">Fermentors were inoculated with cells at three different densities: low (OD=20), intermediate (OD=30) and high (OD=40). The later induction scheme was used (induction at about 24 hrs after inoculation. Higher cell density inoculation led to higher NAG production. Regardless the inoculation cell density, addition of ribose showed improved NAG production, even with a late induction scheme.</p>
    <heading>Example 32</heading> <p num="p-0730">This example describes different fermentation experiments to optimize NAG production using strains containing integrated ScGNA1 constructs. In previous experiments with strains containing ScGNA1 plasmids, IPTG had to be used to induce NAG production since lactose was very inefficient. With integrated ScGNA1 constructs NAG production could be induced effectively by lactose addition.</p>
    <p num="p-0731">Lactose Induction:</p>
    <p num="p-0732">The development of the strain, 7107-92 #1 which contained one copy of the ScGNA1 construct integrated in the chromosome allowed for development of the lactose induction process for NAG production. In light of shake flask results showing growth inhibition by early IPTG induction, a later induction scheme was tested in fermentors. Cells were first inoculated and grown to approximately 15 and −20 g l<sup>−1 </sup>cells. Lactose was then fed to 10, 20, or 30 g l<sup>−1 </sup>for 7 hrs during the period glucose feeding was stoped. Later, glucose feed was re-instated for continued growth and NAG production. It was found that lactose levels as low as 10 g l<sup>−1 </sup>were effective, reaching to 50 g l<sup>−1 </sup>NAG by 78 hours.</p>
    <p num="p-0733">Because the separate lactose feeding method introduces complexity to the process, an experiment was carried out to evaluate different options: lactose feed of 10 g l<sup>−1 </sup>with discontinued glucose feed, IPTG induction with continued glucose feed, and a single point addition of 10 g l<sup>−1 </sup>lactose with and without discontinuation of the glucose feed. Another variable tested was an extended lactose feed to 40 g l<sup>−1</sup>, followed by a glucose feed. The single point lactose addition was as effective as other induction strategy. All achieved a NAG production level between 45-55 g l<sup>−1</sup>. Even though the glucose feed was continuous, its concentration was still limiting to the cells (consumed as quickly as it was added). Because the lac operon is normally repressed by the presence of glucose, the limiting feed rate of glucose apparently allowed for non-repressive conditions for lactose induction. The use of a single point lactose addition greatly simplified the process.</p>
    <p num="p-0734">An experiment evaluated the level of single point lactose induction (1.25 to 10 g l<sup>−1</sup>) after an initial growth period. A two-point induction (5 g l<sup>−1 </sup>each time) was also tried. Lactose as low as 1.25 g l<sup>−1 </sup>was very effective, reaching titers near 100 g l<sup>−1 </sup>NAG. However, the lactose induction at 5 g l<sup>−1 </sup>reached NAG levels over 100 g l<sup>−1</sup>. Therefore, lactose induction at 5 g l<sup>−1 </sup>was applied in further tests as a safe level. The two-point induction was also effective, even resulting in faster initial NAG production rates, but the final titer did not exceed that of single point induction with 5 or 10 g l<sup>−1 </sup>lactose.</p>
    <p num="h-0124">Effect of Higher Operating pH and Temperature:</p>
    <p num="p-0735">The use of lower temperature and pH were key factors in stabilizing the GLcN fermentation, primarily due to the labile nature of GLcN at higher pH and temperature. However, it was shown that NAG was stable at neutral pH. Fermentation for NAG production was operated in the pH range of 6.7 to 7.0 throughout the program. Experimental data showed that NAG was stable for at least 50 hours after the cessation of glucose feed at 37° C. and pH 7.0.</p>
    <p num="p-0736">The relatively low temperatures (30° C. and 25° C.) used in the GLcN process are expensive and difficult to maintain at industrial scale. Moreover, cell growth rate is much lower at 30° C. than at 37° C. The use of 30° C. temperature requires a relatively long period to achieve the biomass required for high NAG production. Therefore, a trial was conducted at 37° C. to evaluate four different growth rates before induction. In this way, the required biomass could be achieved earlier, resulting in an overall more productive process. All fermentors were induced at 10 hours, opposed to the 22 to 24 hours required at 30° C., and all performed well, achieving over 100 g l<sup>−1 </sup>NAG within 50 hours, and some achieving nearly 120 g l<sup>−1 </sup>by 70 hours. From this point forward, 37° C. was used as the growth and production temperature.</p>
    <p num="h-0125">Effect of Glucose Feeding Rate:</p>
    <p num="p-0737">A nominal rate of 6 g l<sup>−1 </sup>hr<sup>−1 </sup>(based on volume after inoculation) was used in most trials. To see if the rate of NAG production was limited by glucose restriction, higher glucose feed rates were tested. Increasing the feed rate to 7.5 g l<sup>−1 </sup>hr<sup>−1 </sup>led to faster initial rates of NAG production, but the final titers were not higher or reached faster. Further increasing the feed rate to 9.5 g l<sup>−1 </sup>hr<sup>−1 </sup>showed even faster initial rates of NAG production. However, these cultures appeared to stop production much earlier and actually had decreased final titer (85 to 65 g l<sup>−1 </sup>NAG). In the same experiment, slow feed rate (6 g l<sup>−1 </sup>hr<sup>−1</sup>) allowed for NAG levels of 100 g l<sup>−1</sup>. From this time point forward, glucose feed rate was kept at 6.5 g l<sup>−1 </sup>hr<sup>−1 </sup>in NAG fermentation.</p>
    <p num="h-0126">Effect of Phosphate Level:</p>
    <p num="p-0738">The fermentation medium contains as much as 30 g l<sup>−1 </sup>total potassium phosphate salts. High phosphate salt level presented a significant problem for recovery as most of the salt is unused by the cells, and has to be removed during product purification. In the NAG process, the high phosphate level was shown not to be critical. The total phosphates were reduced by four times to 7.5 g l<sup>−1 </sup>with little effect on NAG production. This phosphate level was used in subsequent experiments.</p>
    <p num="h-0127">Importance of Magnesium and Iron:</p>
    <p num="p-0739">Magnesium and iron are two required nutrients to achieve higher biomass and stabilize the culture. In the GLcN process, the iron level was seen as a critical factor. A certain level was needed to achieve higher biomass and lactose induction, but too much iron resulted in higher acetate and lower GLcN levels. The iron level determined for the GLcN process was 3 mg l<sup>−1 </sup>FeSO<sub>4</sub>-7H<sub>2</sub>O in the medium with additional iron fed by including FeSO<sub>4</sub>-7H<sub>2</sub>O in glucose feed (5 μg g<sup>−1 </sup>glucose). The same iron level was selected for initial NAG production experiments. Iron addition to the glucose feed complicates the process. Therefore, effort was made to evaluate the effects initial iron concentrations and iron feeding. Effects of magnesium feed were also studied.</p>
    <p num="p-0740">Results of the first experiment showed that levels of 5 to 10 mg l<sup>−1 </sup>FeSO<sub>4</sub>-7H<sub>2</sub>O led to comparable NAG production levels, and that the iron level may not be as critical as in the GLcN process. A follow up study with iron, magnesium showed that iron wasn't absolutely necessary in the feed. Magnesium in the feed may have stabilizing effects. This test was conducted at the high feed rate of 7.5 g l<sup>−1 </sup>hr<sup>−1</sup>, and another follow up test at 6.5 g l<sup>−1 </sup>hr<sup>−1 </sup>showed that both could be removed from the feed without significant reduction of productivity.</p>
    <p num="p-0741">While the importance of both iron and magnesium was recognized, removal of them from the feed was desirable from point of simplifying production process. Therefore iron and magnesium were deleted from the glucose feed and iron held at an initial concentration of 5 mg l<sup>−1 </sup>FeSO<sub>4</sub>-7H<sub>2</sub>O. Magnesium had been added at an initial rate of 0.6 g l<sup>−1 </sup>MgSO<sub>4</sub>-7H<sub>2</sub>O, and if used in the glucose feed at a rate of 5 to 10 mg g<sup>−1 </sup>glucose.</p>
    <p num="p-0742">Magnesium was also recognized as important to the culture and should not be limiting. Magnesium can be sequestered by phosphate under certain sterilization conditions (excessive heat, exposure time, and pH), becoming unavailable to the organism as insoluble magnesium phosphate salt precipitates. A flask study using excess magnesium or limiting magnesium showed that the level was indeed critical to NAG production, and that addition of the magnesium after sterilization was desirable to reduce precipitation effects, if the magnesium level was higher. A fermentation experiment was conducted to compare magnesium addition before and after sterilization. Results showed that cultures with magnesium added before sterilization performed poorly.</p>
    <p num="p-0743">The initial magnesium level was then nearly doubled to 1 g l<sup>−1 </sup>MgSO<sub>4</sub>-7H<sub>2</sub>O, and studies showed that this level could be used if added after sterilization or if increased levels of citric acid was added and the medium acidified before sterilization. This had very positive implications for simplification of the protocol. The acidification also allowed for all trace elements to be added before sterilization. Thus, a medium preparation protocol was developed, in which all components except glucose can be added before sterilization. The acidification of the medium initially was achieved by using phosphoric acid instead of potassium phosphate salts to supply phosphorus. pH was adjusted to required operating pH (7.0) after sterilization with potassium hydroxide, which also supply potassium to the medium. However, this method introduced unnecessarily high amount of potassium. The protocol was further modified by using monobasic potassium phosphate to acidify the medium and adjusting pH with ammonium hydroxide after sterilization. This even further simplified the protocol by allowing for removal of ammonium sulfate from the medium.</p>
    <p num="h-0128">Trace Element Removal and Importance of Zinc:</p>
    <p num="p-0744">The trace element package originally consisted of salts of the following elements, added in μg to mg/l quantities: iron, zinc, manganese, copper, cobalt, molybdenum, and boron. Molybdenum and boron presented toxicity issues if found in the final product, and the effects of some other elements were not known. Therefore single deletion experiments were conducted to determine which if any could be removed. The first trial showed that both molybdenum and boron deletion had no significant negative effect on NAG production, so these were removed from further experiments. Cobalt deletion may have also had a positive effect, but was not deleted because this recognized as a necessary for full function of vitamin B<sub>12</sub>, and a follow up study showed no effect on deletion.</p>
    <p num="p-0745">Results showed a positive effect of removing zinc on NAG production. Further studies confirmed that complete zinc restriction resulted in lower biomass levels, and significantly higher NAG levels, up to 118 g l<sup>−1 </sup>NAG by 60 hours (<figref idrefs="DRAWINGS">FIG. 20</figref>). This was especially interesting because higher initial iron levels, up to 10 mg l<sup>−1 </sup>FeSO<sub>4</sub>-7H<sub>2</sub>O could even be used.</p>
    <heading>Example 33</heading> <p num="p-0746">This Example describes preferred fermentation protocols for NAG production. The fermentation protocol is shown below.</p>
    <p num="p-0747"> <tables id="TABLE-US-00045" num="00045"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="147pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain:</td> <td class="description-td">Recombinant <i>E. coli</i> </td> </tr> <tr class="description-tr"> <td class="description-td">Induction:</td> <td class="description-td">5 to 10 g l<sup>−1 </sup>lactose added in a single point</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">addition after a cell density of 15 to 20 g l<sup>−1 </sup>is</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">reached. Glucose feed is not suspended during</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">this procedure, but remains steady at 6.5 g l<sup>−1</sup> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">hr<sup>−1 </sup>(based on initial volume).</td> </tr> <tr class="description-tr"> <td class="description-td">Feed:</td> <td class="description-td">65% glucose without any amendments, glucose</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">fed at limiting concentrations</td> </tr> <tr class="description-tr"> <td class="description-td">Fermentation Time:</td> <td class="description-td">60 to 72 hours</td> </tr> <tr class="description-tr"> <td class="description-td">Fermentation Mode:</td> <td class="description-td">Fed Batch, with 65% glucose added as required,</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">maintain limiting concentrations of glucose</td> </tr> <tr class="description-tr"> <td class="description-td">Inoculum:</td> <td class="description-td">2.5% to 5% by volume</td> </tr> <tr class="description-tr"> <td class="description-td">pH:</td> <td class="description-td">6.9 throughout, controlled with 12 N NH<sub>4</sub>OH</td> </tr> <tr class="description-tr"> <td class="description-td">Temperature:</td> <td class="description-td">37 C. throughout</td> </tr> <tr class="description-tr"> <td class="description-td">Oxygen:</td> <td class="description-td">Dissolved O<sub>2 </sub>at 20% or greater, controlled</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">by agitation</td> </tr> <tr class="description-tr"> <td class="description-td">Aeration:</td> <td class="description-td">0.5 to 1 vvm</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="56pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Medium:</td> <td class="description-td"> </td> <td class="description-td">Concentration</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Component</td> <td class="description-td">(amount per liter)</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="right"> </colspec> <colspec colname="4" colwidth="63pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">KH<sub>2</sub>PO<sub>4</sub> </td> <td class="description-td">6.67</td> <td class="description-td">g</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Citric acid</td> <td class="description-td">3.25</td> <td class="description-td">g</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CaCl<sub>2</sub>—H<sub>2</sub>O</td> <td class="description-td">0.05</td> <td class="description-td">g</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">MgSO<sub>4</sub>—7H<sub>2</sub>O</td> <td class="description-td">2.5</td> <td class="description-td">g</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">FeSO<sub>4</sub>—7H<sub>2</sub>O</td> <td class="description-td">5</td> <td class="description-td">mg</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">ZnSO<sub>4</sub>—7H<sub>2</sub>O</td> <td class="description-td">3.8</td> <td class="description-td">mg</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">MnSO<sub>4</sub>—H<sub>2</sub>O</td> <td class="description-td">0.33</td> <td class="description-td">mg</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CuSO<sub>4</sub>—5H<sub>2</sub>O</td> <td class="description-td">0.1</td> <td class="description-td">mg</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CoCl<sub>2</sub>—6H<sub>2</sub>O</td> <td class="description-td">0.1</td> <td class="description-td">mg</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="56pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Glucose</td> <td class="description-td">&gt;200 g, as needed</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="right"> </colspec> <colspec colname="4" colwidth="63pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Mazu 204 defoamer</td> <td class="description-td">0.25</td> <td class="description-td">g</td> </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
All components are added before sterilization except glucose (added incrementally). Initial pH is near 3.0 after sterilization and adjusted to 6.9 with NH<sub>4</sub>OH before inoculation.
</p>
    <heading>Example 34</heading> <p num="p-0748">This Example describes purification of N-acetylglucosamine by double crystallization.</p>
    <p num="p-0749">Fermentation broth was subject to cell removal by filtration and micro-filtration. Depending on the fermentation conditions, the percentage of N-acetylglucosamine in the dissolved solid ranged from 70 to 87% (w, on a dry solids basis). Therefore, the crude N-acetylglucosamine product must be purified. This can be done by a combination of cation and anion deionization steps to increase N-acetylglucosamine purity in the crude broth. Different crystallization protocols can also be used. The following Examples describe different experiments to establish protocols and conditions for the purification of N-acetylglucosamine. As demonstrated, N-acetylglucosamine of ≧98% pure was obtained using methods described in Examples 34, 35, 36, 40 and 41. The purity was further increased by the method described in Example 37.</p>
    <p num="p-0750">N-acetylglucosamine was determined by liquid chromatography using a Supelcosil LC-18-DB (250×4.6 mm, 5 μm) column (obtained from Supelco, Bellefonte, Pa.) and a mobile phase of 10% (v) acetonitrile at 1 ml min<sup>−1</sup>. N-acetylglucosamine was measured at 210 nm using a commercial N-acetylglucosamine (Sigma, ≧99%) as external standard. The detector showed a linear response up to 5 g l<sup>−1 </sup>N-acetylglucosamine when the injection volume was 4 μL. Samples and standard were routinely prepared at approximately 3 g l<sup>−1 </sup>and were injected at least twice. The deviation in peak area was less than 2% from the average values.</p>
    <p num="p-0751">Activated carbon (100 mesh G-60, available from Norit Americas, Atlanta, Ga.) was added to a fermentation sample (30 g/L) containing 70% (w) N-acetylglucosamine in the dissolved solid and the mixture was stirred at room temperature for an hour followed by filtration using medium filter paper. The filtrate showed reduced color and was pale yellowish brown. The amount of solid was measured and the percentage of N-acetylglucosamine in the solid was 75% (w).</p>
    <p num="p-0752">The carbon treated sample obtained above containing 77.5 g dissolved solid (58.1 g of which as N-acetylglucosamine) was concentrated at 45° C.-50° C. under vacuum to 51% (w) solid (calculated from sample weight assuming there was no loss of solid during concentrating). It was left at room temperature with occasional mixing for 2 hours. The precipitate was collected by filtration on medium filter paper and redissolved in 100-ml water. The solids content was then determined to be 19.7% (w) solid (total solid 26.2 g).</p>
    <p num="p-0753">The redissolved sample was concentrated again to 44% (w) dissolved solid. Approximately 2 mg pure N-acetylglucosamine powder was added and the sample was mixed on a shaker at room temperature for 2 hours. The precipitate was collected by filtration and washed with ethanol (twice without suspending the solid and twice by suspending it). The white solid was dried at room temperature under vacuum until the weight stayed constant to afford 5.32 g product (98% N-acetylglucosamine, 9% overall recovery).</p>
    <heading>Example 35</heading> <p num="p-0754">This Example describes purification of N-acetylglucosamine by single crystallization from 50° C.</p>
    <p num="p-0755">Another batch of fermentation sample containing 87% (w) N-acetylglucosamine in the dissolved solid was treated with activated carbon as described in Example 34. The amount of dissolved solid was measured and the percentage of N-acetylglucosamine in the solid was 88% (w). The carbon treated sample containing 80.5 g solid (71 g of which as N-acetylglucosamine) was concentrated at 45° C.-50° C. under vacuum to 45% (w) solid (calculated from the weight assuming there was no loss of solid during concentrating) and shaken at room temperature for 16 hours. The precipitate was collected by filtration on medium filter paper and washed with ethanol (twice without suspending the solid and twice by suspending it). After drying under vacuum, 33.2 grams of white solid was obtained (100% pure N-acetylglucosamine, 47% recovery).</p>
    <heading>Example 36</heading> <p num="p-0756">This Example describes purification of N-acetylglucosamine by single crystallization from 70° C.</p>
    <p num="p-0757">Partially purified N-acetylglucosamine could be obtained by one crystallization of a carbon treated fermentation sample (see Example 34). The purity of N-acetylglucosamine ranged from 86% to 90% depending on the degree of concentrating before crystallization and the initial percentage of N-acetylglucosamine in the dissolved solid.</p>
    <p num="p-0758">A solid sample containing 86% or 90% N-acetylglucosamine was mixed with water to 44% (w) solid. The mixture was heated at 70° C. with occasional mixing to dissolve. It was then left at room temperature for approximately 16 hours. The crystals were collected by filtration and washed with ethanol (twice without suspending the solid and twice by suspending it). After drying at room temperature under vacuum, the product contained 99% N-acetylglucosamine (24-30% recovery).</p>
    <heading>Example 37</heading> <p num="p-0759">This Example describes N-acetylglucosamine purification by cation exchange treatment.</p>
    <p num="p-0760">Decolorized fermentation broth was used as the input feed to a cation exchange column. The column was 1.6×15 cm, and contained 20 g DOWEX™ Monosphere 88 resin (available from Dow Corp., Midland, Mich.) in the hydrogen form. The experiments were done using a Pharmacia FPLC™ setup (available from Amersham Biosciences, Piscataway, N.J.) at 4° C. Input feed was about 76% N-acetylglucosamine on a solids basis, conductivity of 10.5 mS/cm, and pH was about 6.5. The material was pumped at a rate of 1 ml/minute. Initial output pH was about 2.3. As the column capacity was reached pH rose and eventually was the same as that of the input feed. Conductivity decreased from around 10.5 mS/cm to around 2.5 mS/cm at about 125 ml. Conductivity then increased with the rise in pH. N-acetylglucosamine purity, measured on a solids basis, also increased rapidly while the pH remained around 2.5. Purity increased to about 85% N-acetylglucosamine. Purity decreased as column capacity became exhausted, and was essentially the same as the input stream by 200 ml. N-acetylglucosamine purity is increased by cation treatment.</p>
    <heading>Example 38</heading> <p num="p-0761">This Example describes N-acetylglucosamine purification by anion exchange treatment. N-acetylglucosamine broth, which has been decolorized and treated with the cation exchange resin, was used as input material for the anion exchange resin-Dow Monosphere 77. Experimental conditions were as described above for the cation exchange resin experiments. Input feed was about 81% N-acetylglucosamine on a solids basis, conductivity of 3.8 mS/cm, and pH was about 3.0. Output pH rose quickly to around 8.0, and later fell to about 3.8 at 250 ml. Conductivity quickly decreased to less than 1 mS/cm, and stayed at that level throughout the run. N-acetylglucosamine purity increased and remained significantly higher as well. N-acetylglucosamine purity increased to about 87% N-acetylglucosamine from an input stream purity of about 81%.</p>
    <p num="p-0762">The combined cation and anion treatments provide a simple method to significantly increase N-acetylglucosamine purity in the crude broth based on a total dry solids basis.</p>
    <heading>Example 39</heading> <p num="p-0763">This Example describes treatment of fermentation broth with activated carbon and mixed-bed ion exchange.</p>
    <p num="p-0764">Fermentation samples were first treated with activated carbon as described in Example 34 and then by mix-bed ion exchange resin AG501-X8(D) (from Bio-Rad, Hercules, Calif.) in a column based on the manufacturer's recommendation. Samples were slowly loaded and flow was driven by gravity. The amount of sample load was to exhaust two thirds of the bed as indicated by the blue indicator changing to gold. Samples containing initially 70% (w) and 87% (w) N-acetylglucosamine in the dissolved solid showed a final N-acetylglucosamine purity of 89% (w) and 93% (w), respectively, in the dissolved solid.</p>
    <heading>Example 40</heading> <p num="p-0765">This Example describes stabilization of purified N-acetylglucosamine.</p>
    <p num="p-0766">A sample N-acetylglucosamine (98% pure) generated by crystallization described above turned light brown in 2 hours at 105° C. and lost 4.7% weight. This color change and weight loss upon heating can be eliminated using one of the following treatments (described below) involving isopropyl alcohol (IPA). It has been demonstrated that IPA (isopropyl alcohol) is a useful reagent in the precipitation of N-acetylglucosamine and in the removal of sample darkening at 105° C. However, it is very likely that a skilled person can choose other organic solvents that are miscible with water such as acetonitrile or ethanol to replace IPA and achieve the same effect.</p>
    <p num="p-0767">Preheat Followed by Water/IPA Precipitation</p>
    <p num="p-0768">A sample (0.30 g) was heated at 105° C. for 2.5 hours. After cooling to room temperature, 0.7 ml water was added to form a yellow suspension. IPA (2.8 ml) was added and the mixture was stirred for 2 hours. The precipitation was collected by filtration and washed twice with IPA by suspending the solid. The mother liquor was pale yellow. The solid was dried under vacuum till the weight was constant (0.18 g, 60% recovery).</p>
    <p num="p-0769">Soak Treatment in 80:20 or 85:15 IPA/Water (v/v)</p>
    <p num="h-0137">A sample was stirred in 80:20 or 85:15 (v/v) IPA/water at 5 or 12 ml solvent mixture/gram solid for 3 hours or overnight. Recovery (same procedure as above) ranged from 56 to 67%.</p>
    <p num="p-0770">Dissolution and Precipitation with IPA</p>
    <p num="p-0771">A sample (0.50 g) was dissolved in 2.27 ml water. PA (12.86 ml, IPA/water=85:15 v/v) was added and the mixture was stirred at room temperature over night. Recovery procedure (56%) was the same as above.</p>
    <p num="p-0772">Dissolution in Water, Followed with Concentration and Precipitation with IPA</p>
    <p num="p-0773">A sample (89.6 g) was dissolved in water to 20% solid (w/w). It was concentrated at 45-50° C. under vacuum. A precipitation started to form when the concentration of solid reached approximately 42% (w/w). It was further concentrated to 55% solid (calculated based on the total sample weight assuming there is no loss of solid during concentrating). IPA was added (IPA/water=85:15 v/v) and it was stirred overnight. The solid was collected by filtration and washed twice with IPA by suspending the solid. The mother liquor was concentrated to dryness to afford 13 g wet solid that was subsequently suspended in 20 ml 85:15 IPA/water (v/v). It was filtered and washed twice with IPA. The wet solid was combined with the solid obtained after the first filtration and dried under vacuum until a constant weight was obtained (82.1 g, 98% N-acetylglucosamine, 92% recovery).</p>
    <heading>Example 41</heading> <p num="p-0774">This Example describes purifying N-acetyl glucosamine with IPA.</p>
    <p num="p-0775">IPA is used to purify N-acetylglucosamine in order to obtain higher recovery. A mixture of PA and water (probably from 70:30 to 85:15 v/v ratio) is used to precipitate N-acetylglucosamine while keeping all the impurities in solution. The purified N-acetylglucosamine thus obtained shows no darkening at 105° C. The required amount of this solvent mixture depends on the amount of impurity in the initial sample and its solubility.</p>
    <heading>Example 42</heading> <p num="p-0776">This Example describes purifying N-acetyl glucosamine with ethanol. A 95% pure NAG sample (40 g) was dissolved in 160-ml water. Ethanol (640 ml) was slowly added while the mixture was stirred. The suspension was stirred at room temperature overnight. The solid was collected by filtration and washed twice by suspending it in ethanol. Drying under vacuum afforded 16.12 g white solid (98% NAG with 42% recovery of NAG). This material showed no darkening at 105° C. in two hours. Any skilled in the art can use other water miscible solvents such as IPA, acetonitrile, or methanol to replace ethanol to achieve the same result.</p>
    <heading>Example 43</heading> <p num="p-0777">This Example demonstrates processes for the chemical hydrolysis of N-acetylglucosamine to glucosamine.</p>
    <p num="p-0778">It is anticipated that N-acetylglucosamine produced by fermentation can be recovered and purified in the form of N-acetylglucosamine as the final product. N-acetylglucosamine produced by fermentation can also be chemically converted to glucosamine before or after being isolated and purified. Simple pilot experiments were conducted to demonstrate chemical conversion of N-acetylglucosamine to glucosamine. In the first experiment N-acetylglucosamine (10 g/l in M9A medium without glucose) was hydrolyzed at 100° C. with various levels of hydrochloric acid. In a parallel experiment, glucosamine was subjected to the same treatments to determine its stability under hydrolysis conditions. Results are shown in <figref idrefs="DRAWINGS">FIGS. 21 and 22</figref>.</p>
    <p num="p-0779">Conversion of N-acetylglucosamine to glucosamine was determined by monitoring the amount of glucosamine generated by the hydrolysis reaction. Glucosamine was determined by an HPLC method based on that described by Way et al. (Journal of Liquid Chromatography and Related Technologies, 23:2861, 2000). Some specific details of the HPLC method are given below: column: Phenomenex prodigy ODS(3) C 18-5 μm (available from Phenomenex, Torrance, Calif.), 150×4.6 mm; mobile phase: methanol: aqueous buffer (4:1, v:v) containing 10 mM sodium acetate, 10 mM sodium octanesulfonate, pH 5.1; flow rate: 0.7 ml per min; detector: refractive index detector at 30° C.</p>
    <p num="p-0780"> <figref idrefs="DRAWINGS">FIG. 21</figref> demonstrates quantitative conversion of N-acetylglucosamine to glucosamine when heated at sufficiently low pH (&lt;1). At higher pH, either no hydrolysis occurred, or the glucosamine formed was degraded. <figref idrefs="DRAWINGS">FIG. 22</figref> shows that glucosamine was stable with heating at a pH of 1.0 or lower. At higher pH a significant amount of glucosamine was lost.</p>
    <p num="p-0781">Acid hydrolysis of N-acetylglucosamine using 1.0 N hydrochloric acid (pH&lt;1) was examined at different temperatures. Both the amount of remaining N-acetylglucosamine and the amount of formed glucosamine were monitored. N-acetylglucosamine was measured by HPLC using a standard carbohydrate system. Some specific details are given below: column: Bio-Rad HPX-87H, 7.8 mm×300 mm; mobile phase: 0.1% HNO<sub>3 </sub>in H<sub>2</sub>O: flow rate: 0.8 ml per min; detector: refractive index detector at 30° C. Glucosamine was measured using the ion pair HPLC column as described above. N-acetylglucosamine (20 g/l) in M9A medium (described previously herein) acidified with 1 N hydrochloric acid to pH&lt;1.0 was incubated at 35° C., 60° C., or 100° C. Results are shown in <figref idrefs="DRAWINGS">FIG. 23</figref>. At 100° C. conversion was complete by 2.5 hours. No significant degradation of the glucosamine formed was observed. At lower temperatures, some hydrolysis was observed, but this was much slower.</p>
    <p num="p-0782">In a parallel experiment, glucosamine (20 g/l) was incubated with 1 N hydrochloric acid at different temperatures as above. No degradation was observed after 24 hours of incubation as measured by HPLC (data not shown).</p>
    <p num="p-0783">In summary, it is clear that N-acetylglucosamine can easily be chemically converted to glucosamine and that glucosamine, the hydrolysis product, is very stable under the hydrolysis conditions used in these experiments.</p>
    <p num="p-0784">Acid hydrolysis of N-acetyl glucosamine to glucosamine is a key step in the whole glucosamine production process. Therefore, hydrolysis experiments were conducted using crystallized N-acetyl glucosamine material from fermentation or purified N-acetyl glucosamine.</p>
    <heading>Example 44</heading> <p num="p-0785">This Example describes deacetylation of N-acetylglucosamine at high hydrochloric acid concentrations and short times.</p>
    <p num="p-0786">Acid hydrolysis of N-acetylglucosamine at 90° C. was performed. N-acetylglucosamine (10% and 20% w/v) in hydrochloric acid (12% and 16%— diluted by volume from 37%) solutions were used. A series of 12-ml glass screw capped tubes with Teflon lined caps were employed. Each tube represented a separate time point (0, 15, 30, 45, 60, 90 minutes) and contained 2 ml of the N-acetylglucosamine/hydrochloric acid solution. Tubes were heated in a heating block equilibrated at 90° C. Tubes were removed at appropriate times and quick cooled in ice water. Appropriate dilutions were made in and samples analyzed by HPLC for glucosamine and N-acetylglucosamine. <figref idrefs="DRAWINGS">FIG. 24</figref> shows the disappearance of N-acetylglucosamine and the formation of glucosamine. Kinetics of N-acetylglucosamine disappearance were essentially the same for all four solutions. Hydrolysis is very rapid at 90° C. By 30 minutes about 95% of the N-acetylglucosamine had been hydrolyzed in all four cases. After 45 minutes less than 1% of the initial concentration of N-acetylglucosamine remained. No N-acetylglucosamine was detected at 60 or 90 minutes. After 1 hour at 90° C., N-acetylglucosamine was no longer detected even with 20% N-acetylglucosamine. <figref idrefs="DRAWINGS">FIG. 24</figref> also shows formation of glucosamine in the tubes containing 10% glucosamine. No large loss of glucosamine was observed after 90 minutes.</p>
    <heading>Example 45</heading> <p num="p-0787">This Example describes deacetylation of N-acetylglucosamine at high hydrochloric acid concentrations for a long period (24 hrs).</p>
    <p num="p-0788">Glucosamine degradation was examined at 90° C. and 100° C. using 5% and 10% glucosamine hydrochloride (w/w) with 12% and 20% hydrochloric acid by weight. Samples were incubated from 1-24 hours and assayed enzymatically for ammonia using glutamate dehydrogenase. Glucosamine refers to glucosamine hydrochloride. All percentages are w/w. Glucosamine degradation is based on ammonia formation. Percent degradation was essentially the same for 5% and 10% solutions. Degradation was significantly higher with higher acid concentration. Similar trends were seen for the 100° C. experiment. The 10 degree temperature increase clearly greatly increases glucosamine degradation.</p>
    <heading>Example 46</heading> <p num="p-0789">This Example describes chemical deacetylation of N-acetylglucosamine at high hydrochloric acid concentrations (30%).</p>
    <p num="p-0790">N-acetylglucosamine was hydrolyzed to glucosamine at 90° C. using the hybridization oven. The reaction was carried out inside a screw capped glass tube. The 30% hydrochloric acid (30 g) was preincubated at 90° C. for 1 hour. After this 10 g solid N-acetylglucosamine was added, quickly dissolved, and a T<sub>o </sub>sample taken. Samples were taken at 30-minute intervals over the next three hours. The solution rapidly turned dark orange/brown within 30 minutes. This darkening was noticeably faster than that previously observed with lower acid concentrations. By 45-60 minutes significant solid glucosamine was observed, converting the mixture into a slurry of glucosamine hydrochloride in hydrochloric acid. Samples were analyzed for ammonia enzymatically. Results are shown in <figref idrefs="DRAWINGS">FIG. 25</figref>. <figref idrefs="DRAWINGS">FIG. 25</figref> also shows data from the glucosamine degradation experiments done at 90° C. using lower concentrations of hydrochloric acid. Degradation of N-acetylglucosamine is significant, and based on ammonia levels is around 3.5% at 3 hours. This is clearly much higher than that observed using glucosamine in 12% or 20% hydrochloric acid.</p>
    <heading>Example 47</heading> <p num="p-0791">This Example describes enzymatic deacetylation of N-acetylglucosamine.</p>
    <p num="p-0792">Enzyme processes to hydrolyze N-acetylglucosamine in the fermentation broth or after its recovery are described below. Three types of enzymes are candidates: N-acetylglucosamine-6-P deacetylase (EC 3.5.1.25, NagA), N-acetylglucosamine deacetylase (EC 3.5.1.33), chitin deacetylase and acyl transferases.</p>
    <p num="p-0793">N-Acetylglucosamine-6-P Deacetylase and N-Acetylglucosamine Deacetylase</p>
    <p num="p-0794">There are enzymes that have been shown to deacetylate N-acetylglucosamine. Roseman reported enzyme activity that catalyzed deacetylation of N-acetylglucosamine (EC 3.5.1.33) in <i>E. coli, Bacillus cadaveris </i>and <i>Streptococcus </i>(Roseman S. 1957. J. Biol. Chem., 226:115-124). A Japanese group (Yamano, Fujishima et al, Osaka National Research Institute, Agency of Industrial Science and Technology) studied the N-acetylglucosamine deacetylases from a chitinase-producing bacterium <i>Vibrio cholerae </i>non-O1 and a marine bacterium <i>Alteromonas</i>. Preparation, properties and use of both N-acetylglucosamine-6-P deacetylase and N-acetylglucosamine deacetylases from specified native organisms were disclosed in the following patents and publications: Fujishima S. et al., 1996, entitled N-acetylglucosamine 6-phosphate deacetylase, JP9234064A2; Fujishima S. et al., 1997, entitled Process for producing N-acetylglucosamine-6-phosphate deacetylase, U.S. Pat. No. 5,744,325; and Fujishima et al., 1996, entitled Process for producing N-acetyl-D-glucosamine deacetylase, EP 0 732 400 B1, which are incorporated herein by reference in their entirety.</p>
    <p num="p-0795">The deacetylases described by Fujishima et al. were identified as N-acetylglucosamine-6-P deacetylases (EC 3.5.1.25, NagA). Their affinity and efficacy with N-acetylglucosamine 6-P were much higher than with N-acetylglucosamine. However, N-acetylglucosamine-6-P deacetylase purified from <i>E. coli </i>does not act on N-acetylglucosamine.</p>
    <p num="p-0796">The enzyme N-acetylglucosamine-6-P deacetylase (EC 3.5.1.25, NagA) is well known for its role of converting N-acetylglucosamine-6-P to glucosamine-6-P, a necessary step in the cellular metabolism of N-acetylglucosamine, N-acetylmannosamine and neuraminic acid. Normally, this enzyme such the recombinant <i>E. coli </i>NagA protein is not active on non-phosphorylated N-acetylglucosamine. DNA sequences coding for the N-acetylglucosamine 6-P deacetylase (nagA gene) were determined in many different organisms. It is not know if there exists a deacetylase that is only active on N-acetylglucosamine (thus distinctive from NagA).</p>
    <p num="p-0797">Chitin Deacetylase</p>
    <p num="p-0798">Chitin deacetylase (EC 3.5.1.41) catalyzes deacetylation of the N-acetylglucosamine units in chitin, resulting in chitosan. Chitin deacetylase activity is usually determined by using as substrate glycol chitin (partially O-hydroxyethylated chitin) radiolabeled in N-acetyl groups. The enzyme also acts on mycrocrystalline chitin and carboxymethylchitin (soluble derivative). However, it was reported that chitin deacetylase from <i>Mucor rouxii </i>does not deacetylate N-acetylglucosamine monomer or 2-3 oligomers (Araki and Ito, 1975. Eur. J. Biochem. 55:71-78, which is incorporated herein by reference in its entirety). Although there were no indications that normal chitin deacetylase deacetylate glucosamine monomer, chitin deacetylase variants with such activity could be isolated from nature or created in vitro.</p>
    <p num="p-0799">Deacetylation of N-acetylglucosamine could be carried out by a deacetylase contained in or isolated from organisms with a native deacetylase or organisms with a recombinant deacetylase. Recombinant deacetylases could be improved by random or directed mutagenesis.</p>
    <p num="p-0800">The following describe experiments of N-acetylglucosamine hydrolysis using N-acetylglucosamine deacetylase and/or N-acetylglucosamine-6-P deacetylase.</p>
    <p num="p-0801">Acyl Transferases</p>
    <p num="p-0802">A number of acyl transferases can remove the acetyl group from a substrate and transfer it to another substrate (Konecny, et al.). Although there were no indications that such enzymes can deacetylate N-acetylglucosamine, acyl trnasferase variants with such activity could be isolated from nature or generated in vitro.</p>
    <p num="p-0803">Deacetylation of N-acetylglucosamine could be carried out by an acyl transferase contained in or isolated from organisms with a native acyl transferase or organisms with a recombinant acyl transferase. Recombinant acyl transferase could be improved by random or directed mutagenesis and/or by protein engineering.</p>
    <p num="p-0804">Defining Enzymatic Hydrolysis Conditions</p>
    <p num="p-0805">Native or recombinant cells expressing a deacetylase are grown under standard conditions or optimized conditions. N-acetylglucosamine hydrolysis is carried out using whole cells, crude enzyme extracts or purified enzymes as catalysts. First, 0.3 ml of 10% solution of N-acetylglucosamine is added as substrate to 0.1 ml of 200 mM phosphate buffer solution (pH 4.5, 5.0, 5.5, 6.0, 6.5, 7.0, 7.5 and 8.0). Secondly, 0.1 ml of an enzyme solution is added. The reaction mixture is incubated at 25, 30, 35, 40, 45 and 50° C. for 30, 60 and 120 min. The formation of the hydrolysis product, glucosamine, is determined by a HPLC method. The remaining substrate, N-acetylglucosamine, is monitored by a different HPLC method. Both HPLC methods were described in previous Examples.</p>
    <p num="p-0806">Hydrolysis of N-Acetylglucosamine Produced in Fermentation</p>
    <p num="p-0807">N-acetylglucosamine in the fermentation broth or after its recovery is hydrolyzed using crude enzyme extracts or purified deacetylase as catalysts under conditions defined above. Glucosamine is recovered by crystallization in a hydrochloric acid solution and washed by ethanol, methanol or isopropyl alcohol.</p>
    <heading>Example 48</heading> <p num="p-0808">This Example describes hydrolysis of high purity N-acetylglucosamine.</p>
    <p num="p-0809">The apparatus for this test consisted of a 1 liter round bottom vessel heated with a heating mantel. The ingredients, 150 grams of water, 21 g of 98% N-acetylglucosamine and 238 grams of 36.7% hydrochloric acid, were mixed together and added to the reaction vessel. The stirred reaction mix was heated up to 70° C. and mixed for 3 hours at which time the mixture was transferred to a beaker and cooled to 20° C. Few crystals were present. The mixture was transferred back into the reaction vessel and 10 grams of N-acetylglucosamine added. The reactor was reheated to 70° C. and reacted for 3 hours, cooled to 4° C. overnight. The crystals present were filtered and washed with ethanol, vacuum dried and assayed (20 g of glucosamine hydrochloride at 99.9%). The filtrate was transferred back into the reaction vessel and 10 g of N-acetylglucosamine were added. The reactor was again heated to 70° C., reacted for 3 hours, cooled overnight to 4° C., filtered, ethanol washed, vacuum dried and assayed (16.1 g of glucosamine hydrochloride at 99.6%). The filtrate was again returned to the reactor, 41.5 grams added and the reaction cycle repeated. Thirty-three grams of glucosamine hydrochloride resulted from the 4<sup>th </sup>cycle with an assay of 99.5%. The total yield was 86% with all the glucosamine hydrochloride produced 99+in assay and white in color. This material did not need recrystallization.</p>
    <heading>Example 49</heading> <p num="p-0810">This Example describes hydrolysis of low purity N-acetylglucosamine.</p>
    <p num="p-0811">This test repeated the conditions of Example 47 using N-acetylglucosamine concentrated and dried from fermentation broth. The solids contained 52% N-acetylglucosamine. Three cycles were performed. The glucosamine hydrochloride produced was darker with each cycle. The first two samples had assays of 99.7% and 99.4% and would not require recrystallization. The third sample had an assay of 97.7% and was darker in color and would require recrystallization. The overall yield was 71%. The final filtrate was brownish-black as opposed to the translucent light brown color of the Example 48 final filtrate.</p>
    <heading>Example 50</heading> <p num="p-0812">This Example describes hydrolysis of fermentation broth concentrated to 21.8% N-acetylglucosamine.</p>
    <p num="p-0813">The apparatus for this test consisted of a 4-liter jacketed glass vessel. Water was used to heat the reaction vessel to the required temperature. Two thousand ml of concentrated fermentation broth, containing 218 g/l N-acetylglucosamine, was added to the reaction vessel. Fifteen hundred ml of 36.7% hydrochloric acid was added. The reaction vessel was stirred under vacuum and heated to 70° C. The reaction proceeded for 90 minutes with 310 ml of condensate being collected. The reaction mixture was cooled to 4° C. and filtered. The solids were washed with 265 ml of ethanol and dried. The conversion based on N-acetylglucosamine analysis in the broth and filtrate was 89.5%. The washed glucosamine hydrochloride was dissolved in water, to give a 1550-ml of solution. Fifteen grams of Darco G-60 activated carbon was added to the solution. After mixing for 30 minutes, the solution was filtered; colorless filtrate resulted. The filtrate was vacuum evaporated at 50° C. with a vacuum of 55-cm Hg. The solids were ethanol washed and dried. The overall yield was 82%.</p>
    <heading>Example 51</heading> <p num="p-0814">This Example describes hydrolysis of low purity N-acetylglucosamine with extra rinsing steps.</p>
    <p num="p-0815">The purpose of this test was to determine if extra washing steps after hydrolysis would produce a high purity product that would not require recrystallization. The apparatus used in Example 49 without the condensate recovery was used. Dried fermentation broth containing 54% N-acetylglucosamine was added to 20% hydrochloric acid. The acid to N-acetylglucosamine ratio was 2.5:1. The reaction conditions for this series were 80° C. for 103 minutes. The wet cake after filtration was washed with 500 g of water, 529 g of 37% acid and 407 g of ethanol. The goal was to determine if extra washing could produce a product that required no re-crystallization. The glucosamine hydrochloride crystals were still slightly tan and still required re-crystallization. The overall yield was 70%, with 7% yield loss due to the first water wash. The acid wash resulted in a 1% yield loss while the ethanol wash attributed 1.6% loss.</p>
    <heading>Example 52</heading> <p num="p-0816">This Example describes carbon treatment during hydrolysis of N-acetylglucosamine.</p>
    <p num="p-0817">Activated carbon is used after hydrolysis and before recrystallization to remove impurities. This test involved adding activated carbon to the hydrolysis solution prior to the reaction. A total of 34 g of Darco G-60 activated carbon was added to the N-acetylglucosamine and hydrochloric acid reaction mixture and mixed for 30 minutes then filtered and added to the reactor. A 1000-g water rinse of the activated carbon was added to the reactor. The resulting glucosamine hydrochloride was lighter, but still required recrystallization. The reaction conditions were 80° C. for 60 minutes. The overall yield of 29% was low due to dilution of the reaction media by the additional water, which reduced the initial hydrochloric acid concentration to 14%.</p>
    <heading>Example 53</heading> <p num="p-0818">This Example describes hydrolysis of low purity N-acetylglucosamine.</p>
    <p num="p-0819">The reaction condition used for this Example was a temperature of 90° C. for 30 minutes. A 3:1 ratio of 30% hydrochloric acid to N-acetylglucosamine was used. The apparatus used in Example 49 was used here. The hydrolysis yield was 86%. The raw glucosamine hydrochloride was re-dissolved in water; treated with activated carbon, filtered and vacuum crystallized at 50° C. and 60 cm HG. The final assay was 99.7% with an overall yield of 58%.</p>
    <heading>Example 54</heading> <p num="p-0820">This Example describes carbon treatment during hydrolysis of N-acetylglucosamine.</p>
    <p num="p-0821">A second test using activated carbon treatment of the hydrolysis solution was attempted. The reaction conditions were those used in Example 52 except that 50 g of activated carbon was added to the hydrolysis mix, filtered and the activated carbon rinsed with 1,000 g of water. The reaction was conducted at 90° C. for 30 minutes with a 3:1 ratio of 20% hydrochloric acid to N-acetylglucosamine. No improvement in color over the results described in Example 52 was apparent. As with the previous test with activated carbon before hydrolysis, acid was added during filtration and cooling to decrease the solubility and improve yield. The final yield was 36% with an assay of 98.4%.</p>
    <heading>Example 55</heading> <p num="p-0822">This Example describes hydrolysis of high-purity N-acetylglucosamine.</p>
    <p num="p-0823">The purpose of this Example was to determine if high quality glucosamine could be produced without requiring recrystallization from high quality N-acetylglucosamine at 90° C. Using the apparatus of Example 49, 320 grams of N-acetylglucosamine were mixed with 941 grams of 30% w/w hydrochloric acid. The mixture was added to the jacketed vessel and was then heated to 90° C. The reaction was allowed to occur during the heating time and for and additional 47 minutes while holding at 90° C. After cooling to 20° C., the wet cake was filtered, washed with ethanol and vacuum dried at 50° C. The overall yield was 71%, which is consistent with a single use of hydrolysis liquor. The glucosamine hydrochloride produced was white with a black tint and required recrystallization.</p>
    <heading>Example 56</heading> <p num="p-0824">This Example describes hydrolysis of impure N-acetylglucosamine.</p>
    <p num="p-0825">The same procedures described in Example 54 were carried out, using solid N-acetylglucosamine isolated from fermentation broth. The purity of the N-acetylglucosamine solids was 48%. A mixture containing 1,411 grams of N-acetylglucosamine were reacted with 4,236 grams of 30% w/w hydrochloric acid at 70° C. for 3 hours and then cooled to 20° C. The wet cake after filtration was washed with ethanol and vacuum dried at 50° C. The overall yield was 90%.</p>
    <heading>Example 57</heading> <p num="p-0826">This Example describes hydrolysis of high purity N-acetylglucosamine.</p>
    <p num="p-0827">Hydrolysis was performed using a 2:1 ratio of 36.7% hydrochloric acid to pure N-acetylglucosamine. Using the apparatus of Example 49, 2,004 grams of N-acetylglucosamine were reacted with 4,009 grams of 36.7% w/w hydrochloric acid at 80° C. for 58 minutes and then cooled to 20° C. The wet cake after filtration was not washed, but left to dry overnight at room temperature. The final cake contained 6% hydrochloric acid and 75.2% glucosamine hydrochloride. The reaction mixture was fairly viscous with black solids on the sides of the reactor. The overall yield was 70%.</p>
    <heading>Example 58</heading> <p num="p-0828">This Example describes hydrolysis of high purity N-acetylglucosamine using recycled hydrochloric acid.</p>
    <p num="p-0829">One of the ways to increase the overall recovered glucosamine hydrochloride yield is to recycle the hydrolysis mother liquor. After the hydrolysis reaction, the mixture is cooled and filtered. The filtrate is weighed and recycled back into the reactor. Fresh 36.7% hydrochloric acid is added to make up the difference in weight between the initial acid weight and the returned filtrate. This series involved reacting a 2.5:1 ratio of 36.7% hydrochloric acid to N-acetylglucosamine at 80° C. for 60 minutes. The hydrochloric acid was added to a 74° C. reactor containing pure, solid N-acetylglucosamine. The reaction mixture was heated to temperature and reacted for 60 minutes, then cooled to 20° C. and filtered. Glucosamine hydrochloride solids were recovered by filtration after this first use of the hydrochloric acid, and the yield was measured at 88%. The filtrate was weighed and returned to the reactor. A second and equal quantity of solid N-acetylglucosamine was added along with enough 36.7% hydrochloric acid to return the acid weight to the level of the first reaction cycle. The reaction was repeated for the same time and temperature for this second use of the acid, and after cooling and filtration, the recovered yield was 105%, indicating some of the glucosamine hydrochloride left in solution from the first acid cycle was recovered during the second cycle filtration. A third cycle was performed in the same manner as the second, with a recovered yield of 60%. The overall recovered yield for the three cycles was 87%. The filtrate from the third cycle was saved for future cycles.</p>
    <heading>Example 59</heading> <p num="p-0830">This Example describes chromatographic purification of N-acetylglucosamine.</p>
    <p num="p-0831">To purify N-acetyl glucosamine, chromatography using the DOWEX™ Monosphere 99/K resin was performed. The resin was used to pack a 2.6×23 cm column. Column bed volume was about 120 ml, with a void volume estimated at 35 ml (measured by draining liquid from the column using a syringe). For these experiments, broth deionized by treatment with cation and anion resins was used. This input material had a N-acetylglucosamine concentration of about 75.7 g/l, total solids of about 83.4 g/l, giving a purity of around 91%. Conductivity was around 0.1 mS/cm.</p>
    <p num="p-0832">In the first experiment, 30 ml of sample was pumped through the column at 2 ml/min. It was clear that N-acetylglucosamine interacted in some manner with the resin. N-acetylglucosamine was not detected until about 60 ml, and eluted in a very broad peak centered on 90 ml, far past the void volume. N-acetylglucosamine purity also varied greatly, with the highest value of about 94.5% N-acetylglucosamine at 90 ml. Purity measured in this narrow region was higher than that of the input material. Variability for the purity values obtained were about 2-3%. In fractions containing lower concentrations of N-acetylglucosamine, purity fell off rapidly above 125 ml and below 75 ml.</p>
    <p num="p-0833">A second experiment using the same column was performed. Here a smaller 5-ml sample was applied at a slower flow rate of 1 ml/min. Results are similar to those seen with the larger sample and faster flow rate. The only difference is the peak of N-acetylglucosamine centers around 80 ml rather than 90 ml.</p>
    <p num="p-0834">N-acetylglucosamine purity measured as percent of total solids varied significantly during the run. This suggested some chromatographic separation or differences in interaction from N-acetylglucosamine between the resin and the other, presumably nonionic compounds present. If the other compounds were interacting with the resin in a manner identical to N-acetylglucosamine, N-acetylglucosamine should have been present at the same level of purity in all fractions containing N-acetylglucosamine, which was not the case.</p>
    <p num="p-0835">The resin used in this Example is then converted to the calcium form using methods known in the art and the same procedure, as outlined above, is used.</p>
    <heading>Example 60</heading> <p num="p-0836">This Example describes simultaneous hydrolysis of high-purity N-acetylglucosamine and acetic acid removal.</p>
    <p num="p-0837">The major coproduct from aqueous hydrochloric acid hydrolysis of N-acetylglucosamine was acetic acid, which has a relatively high boiling point and was not easily removed during the hydrolysis step. Providing an alcohol, such as ethanol or methanol, during the hydrolysis step results in esterification of acetic acid, forming ethyl- or methyl acetate respectively as a coproduct that was removed more readily due to a reduced boiling point. Removing the coproduct, the major impurity present in spent hydrolysate, permitted extending the number of reuses of the hydrochloric acid in the hydrolysis solution.</p>
    <p num="p-0838">A mixture of 173 grams of methanol, 189.4 grams of 36.7 w % aqueous hydrochloric acid, and 201 grams of N-acetylglucosamine was added to a stirred, heated glass vessel and mixed under reflux at 65° C. A sample was taken at 1 hour. The analysis showed 8.3% glucosamine hydrochloride, 11.6% hydrochloric acid and no significant amount of acetic acid by titration. The reaction was stopped after 2 hours and cooled to 20° C. The solids were rinsed with methanol and dried. The initial yield of glucosamine hydrochloride was 25.6%. The hydrolysate was then cooled overnight at 4° C., and filtered. Another 10% glucosamine hydrochloride yield was obtained. The purity of the initial solids was 85%. No acetic acid was present in the filtrate.</p>
    <p num="p-0839">While various embodiments of the present invention have been described in detail, it is apparent that modifications and adaptations of those embodiments will occur to those skilled in the art. It is to be expressly understood, however, that such modifications and adaptations are within the scope of the present invention, as set forth in the following claims.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3008874">US3008874</a></td><td class="patent-data-table-td patent-date-value">May 15, 1958</td><td class="patent-data-table-td patent-date-value">Nov 14, 1961</td><td class="patent-data-table-td ">Pfizer &amp; Co C</td><td class="patent-data-table-td ">Pharmaceutical compositions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3679660">US3679660</a></td><td class="patent-data-table-td patent-date-value">Jun 9, 1969</td><td class="patent-data-table-td patent-date-value">Jul 25, 1972</td><td class="patent-data-table-td ">Magnus Agatha C</td><td class="patent-data-table-td ">Glucosamine-alkylsulfonate and process</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3683076">US3683076</a></td><td class="patent-data-table-td patent-date-value">Oct 14, 1969</td><td class="patent-data-table-td patent-date-value">Aug 8, 1972</td><td class="patent-data-table-td ">Luigi Rovati</td><td class="patent-data-table-td ">Pharmaceutically active glucosamine salts useful in the treatment of osteoarthritis and rheumatoid arthritis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4642340">US4642340</a></td><td class="patent-data-table-td patent-date-value">Apr 30, 1982</td><td class="patent-data-table-td patent-date-value">Feb 10, 1987</td><td class="patent-data-table-td ">Rotta Research Laboratorium S.P.A.</td><td class="patent-data-table-td ">Stable compounds of glucosamine sulphate</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5246840">US5246840</a></td><td class="patent-data-table-td patent-date-value">Mar 22, 1989</td><td class="patent-data-table-td patent-date-value">Sep 21, 1993</td><td class="patent-data-table-td ">Procur Ab</td><td class="patent-data-table-td ">Method for synthesis of oligosaccharides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5731184">US5731184</a></td><td class="patent-data-table-td patent-date-value">Feb 29, 1996</td><td class="patent-data-table-td patent-date-value">Mar 24, 1998</td><td class="patent-data-table-td ">Agency Of Industrial Science &amp; Technology</td><td class="patent-data-table-td ">Incubating alteromonas and recovering enzyme from the culture product</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5739015">US5739015</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 1997</td><td class="patent-data-table-td patent-date-value">Apr 14, 1998</td><td class="patent-data-table-td ">Board Of Supervisors Of Louisiana State University And Agricultural And Mechanical College</td><td class="patent-data-table-td ">Biotransformation of chitin to chitosan</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5744325">US5744325</a></td><td class="patent-data-table-td patent-date-value">Feb 26, 1997</td><td class="patent-data-table-td patent-date-value">Apr 28, 1998</td><td class="patent-data-table-td ">Agency Of Industrial Science And Technology</td><td class="patent-data-table-td ">Process for producing N-acetylglucosamine-6-phosphate deacetylase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5843923">US5843923</a></td><td class="patent-data-table-td patent-date-value">May 22, 1998</td><td class="patent-data-table-td patent-date-value">Dec 1, 1998</td><td class="patent-data-table-td ">Jame Fine Chemicals, Inc.</td><td class="patent-data-table-td ">Glucosamine sulfate potassium chloride and process of preparation thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5876713">US5876713</a></td><td class="patent-data-table-td patent-date-value">Aug 12, 1997</td><td class="patent-data-table-td patent-date-value">Mar 2, 1999</td><td class="patent-data-table-td ">Takeda Chemical Industries Ltd</td><td class="patent-data-table-td ">Glutamine: fructose-6-phosphate amidotransferase, its production and use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5998173">US5998173</a></td><td class="patent-data-table-td patent-date-value">Feb 20, 1996</td><td class="patent-data-table-td patent-date-value">Dec 7, 1999</td><td class="patent-data-table-td ">The University Of Bristish Columbia</td><td class="patent-data-table-td ">Process for producing N-acetyl-D-glucosamine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6372457">US6372457</a></td><td class="patent-data-table-td patent-date-value">Jul 15, 1998</td><td class="patent-data-table-td patent-date-value">Apr 16, 2002</td><td class="patent-data-table-td ">Arkion Life Sciences Llc</td><td class="patent-data-table-td ">Culturing in a fermentation medium a microorganism having a genetic modification in an amino sugar metabolic pathway; recovering a product produced from culturing which is selected from glucosamine-6-phosphate and glucosamine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6486307">US6486307</a></td><td class="patent-data-table-td patent-date-value">Aug 23, 2001</td><td class="patent-data-table-td patent-date-value">Nov 26, 2002</td><td class="patent-data-table-td ">Alberta Research Council Inc.</td><td class="patent-data-table-td ">Preparation of glucosamine hydrochloride</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6531303">US6531303</a></td><td class="patent-data-table-td patent-date-value">Jul 6, 1999</td><td class="patent-data-table-td patent-date-value">Mar 11, 2003</td><td class="patent-data-table-td ">Arkion Life Sciences Llc</td><td class="patent-data-table-td ">Method of producing geranylgeraniol</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030181419">US20030181419</a></td><td class="patent-data-table-td patent-date-value">Mar 5, 2003</td><td class="patent-data-table-td patent-date-value">Sep 25, 2003</td><td class="patent-data-table-td ">Ki-Oh Hwang</td><td class="patent-data-table-td ">Glucosamine and method of making glucosamine from microbial biomass</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0614972A2?cl=en">EP0614972A2</a></td><td class="patent-data-table-td patent-date-value">Mar 11, 1994</td><td class="patent-data-table-td patent-date-value">Sep 14, 1994</td><td class="patent-data-table-td ">Agency Of Industrial Science And Technology</td><td class="patent-data-table-td ">N-Acetyl-D-glucosamine deacetylase and a process for preparing the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0732400A2?cl=en">EP0732400A2</a></td><td class="patent-data-table-td patent-date-value">Mar 11, 1996</td><td class="patent-data-table-td patent-date-value">Sep 18, 1996</td><td class="patent-data-table-td ">Dirctor General Of The Agency Of Industrial Science And Technology</td><td class="patent-data-table-td ">Process for producing N-acetyl-D-glucosamine deacetylase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=2W2BBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DGB%26NR%3D1056331A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNH0Z3AolSUtsBQjeeILNCwUsmml4A">GB1056331A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=2W2BBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DJP%26NR%3DH09234064A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNEG6erAMMPjMN6Pg0eqc83-MHA4ew">JPH09234064A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=2W2BBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DRU%26NR%3D2141965C1%26KC%3DC1%26FT%3DD&amp;usg=AFQjCNFn1mJuE1hZ-1uiqB1MG57yBN8wkA">RU2141965C1</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1997031121A1?cl=en">WO1997031121A1</a></td><td class="patent-data-table-td patent-date-value">Feb 20, 1997</td><td class="patent-data-table-td patent-date-value">Aug 28, 1997</td><td class="patent-data-table-td ">Univ British Columbia</td><td class="patent-data-table-td ">Process for producing n-acetyl-d-glucosamine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002043653A2?cl=en">WO2002043653A2</a></td><td class="patent-data-table-td patent-date-value">Dec 3, 2001</td><td class="patent-data-table-td patent-date-value">Jun 6, 2002</td><td class="patent-data-table-td ">Bhat Ravi Gajanan</td><td class="patent-data-table-td ">Crystalline glucosamine sulphate metal salts</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002066667A1?cl=en">WO2002066667A1</a></td><td class="patent-data-table-td patent-date-value">Feb 15, 2002</td><td class="patent-data-table-td patent-date-value">Aug 29, 2002</td><td class="patent-data-table-td ">Cargill Inc</td><td class="patent-data-table-td ">Glucosamine and method of making glucosamine from microbial biomass</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Araki et al., 1975, Eur. J. Biochem., 55, 71-78.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bachmann, Escherichia coli and Salmonella typhimurium cellular and molecular biology Mutant Derivatives of E. coli, vol. 2:pp. 1190-1219, 1987.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Badet et al., 1987, Biochemistry (Mosc). 26, 1940-1948.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Balbas et al., 1996, Gene, 172, 65-69.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bearne, 1996, J. Biol. Chem., 271, 3052-3057.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bernheim et al., 1970, J. Bacteriol., 101, 384-391.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Blattner et al., 1997, Science, 277, 1453-1474.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bouffard et al., 1994, J. Mol. Biol., 244, 269-278.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Branden et al., Introduction to Protein Structure, Garland Publishing Inc., New York, p. 247, 1991.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Brown et al., 1989, Biochemical Society Transaction, 18, 317-318.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chen et al., 1987, Molecular General Genetics, 210, 331-337.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chen et al., 1998, J. Agric. Food Chem., 46, 1971-1974.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chen et al., 1998, J. Agric. Food Chem., 46, 3207-3209.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cochet et al., 1995, Archives of Biochemistry &amp; Biophysics, 324, 391-400.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Curtis et al., 1975, J. Bacteriol., 122, 1189-1199.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Daniels et al., 1993, Mol. Endocrinol., 7, 1041-1048.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Denisot et al., 1991, Arch. Biochem. Biophys., 288, 225-230.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Domard et al., 1991, Int. J. Biol. Macromol., 13, 105-109.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dutka-Malen et al., 1988, Biochimie, 70, 287-290.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Elliot et al., 1975, Biochem. Biophys. Res. Commun., 64, 277-281.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fawcett et al., 1995, J. Bacteriol., 177, 1742-1750.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fernandez-Herrero et al., 1995, Mol. Microbiol., 17, 1-12.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Flores et al., 1996, Nat. Biotechnol., 14, 620-623.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Freese et al., 1970, J. Bacteriol., 101, 1046-1062.</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Glaser et al., 1955, J. Biol. Chem., 216, 67-79.</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Golinelli-Pimpaneau et al., 1991, Eur. J. Biochem., 201, 175-182.</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holmes et al., 1972, J. Bacteriol., 111, 290-291.</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Horlacher et al., 1997, J. Biol. Chem., 272, 13026-13032.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Isaeva et al., 1987, Izvestiya Akademii Nauk SSR, 4, 937-939.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jones-Mortimer et al., 1980, J. Gen. Microbiol., 117, 369-376.</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Joyce et al., 1984, J. Bacteriol., 158, 636-643.</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kashige et al., 1994, Carboyhdrate Research, 257, 285-291.</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kauss et al., 1988, Methods Enzymol., 161, 518-523.</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kimata et al., 1997, Biochemistry (Mosc). 94, 12914-12919.</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Knapp et al., 2002, Tetrahedron Lett., 43, 6075-6078.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Konecny et al., 1982, in Enzyme Engineering, vol. 6:pp. 91-96, Plenum Press.</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lengeler, 1980, Mol. Gen. Genet., 179, 49-54.</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Leriche et al., 1996, J. Am. Chem. Soc., 118, 1797-1798.</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Leriche et al., 1997, Eur. J. Biochem., 245, 418-422.</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Marie et al., 1992, Mol. Microbiol., 6, 843-851.</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Martinou et al., 1995, Carbohydr. Res., 273, 235-242.</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McKnight et al., 1992, J. Biol. Chem., 267, 25208-25212.</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mengin-Lecreulx et al., 1996, The Journal of Biological Chemistry, 271, 32-39.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Miclett et al., 2001, The Journal of Biological Chemistry, 37, 34840-34846.</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Milewski, 1993, Biochimistry et. Biophys Acta, 1161, 279-284.</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mio et al., 1999, J. Biol. Chem., 274, 424-429.</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Miyazaki et al., 1979, Chem. Pharm. Bull., 27, 532-535.</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mudri et al., 1992, J. Biol. Chem., 267, 25208-25212.</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mukhija et al., 1996, The Journal of Biological Chemistry, 271, 14819-14824.</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Novikov et al., 1997, Russian J. Appl. Chem., 70, 1467-1470.</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Novikov, 1999, Russian J. Appl. Chem., 72, 156-161.</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Obmolova et al., 1994, J. Mol. Biol., 242, 703-705.</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Oppon et al., 1998, J. Bacteriol., 180, 3007-3012.</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">O'Shannessy et al., 1993, Anal. Biochem., 212, 457-468.</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">O'Shea et al., 1993, Anal. Chem., 65, 948-951.</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Oyaizu, 1986, Japanese J. Nutr., 44, 307-315.</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Oyaizu, 1988, Nippon Shikuhin Kogyo Gakkaishi, 35, 771-775.</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Oyaizu, 1988, Nippon Shikuhin Kogyo Gakkaishi, 35, 846-850.</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Penneff et al., 2001, The Journal of Biological Chemistry, 276, 16328-16334.</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Peri et al., 1990, Biochemical and Cell Biology, 68, 123-137.</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Plumbridge et al., 1993, J. Bacteriol., 175, 4951-4956.</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Plumbridge et al., 1999, J. Bacteriol., 181, 47-54.</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Plumbridge, 1987, Mol. Gen. Genet., 209, 618-620.</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Plumbridge, 1989, Mol. Microbiol., 3, 505-515.</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Plumbridge, 1990, J. Bacteriol., 172, 2728-2735.</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Plumbridge, 1991, Mol. Microbiol., 5, 2053-2062.</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Plumbridge, 1992, J. Gen. Microbiol., 138, 1011-1017.</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Plumbridge, 1998, Mol. Microbiol., 27, 369-380.</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Plumbridge, 2001, Nucleic Acids Res., 29, 506-514.</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pompeo et al., 2001, J. Biol. Chem., 276, 3833-3839.</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pradel et al., 1988, J. Bacteriol., 170, 4916-4923.</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Roseman, 1957, J. Biol. Chem., 226, 115-124.</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rowley et al., 1991, J. Bacteriol., 173, 968-977.</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sarvas, 1971, J. Bacteriol., 105, 467-471.</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sashiwa et al., 2001, Chemistry Letters, 308-309.</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sashiwa et al., 2001, Japanese Soc. Chitin Chitosan, 7, 257-260.</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schuster et al., 1993, Nature, 365, 343-347.</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Seffernick et al., J. Bacteriol. 183(8):2405-2410, 2001.</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shi and Xie 2002 Huaxue Shiki 24:93-94.</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shu 1998, Journal Agricultural of Food Chemistry, 46, 1129-1131.</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Smith et al., 1996, J. Bacteriol., 178, 2320-2327.</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Souza et al., 1997, Archives of Biochemistry &amp; Biophysics, 340, 338-346.</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sumoto et al., 1991, Chem. Pharm. Bull., 39, 792-794.</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Teplyakov et al., 1998, Structure, 6, 1047-1055.</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Vogler et al., 1989, J. Bacteriol., 171, 6586-6592.</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Vogler et al., 1989, Mol. Gen. Genet., 219, 97-105.</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Walker et al., 1984, Biochem. J., 224, 799-815.</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wanner, 1990, in The Molecular Basis of Bacterial Metabolism, vol. 41:pp. 152-163, Springer-Verlag.</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wanner, 1993, J. Cell. Biochem., 51, 47-54.</td></tr><tr><td class="patent-data-table-td ">90</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wanner, 1996, in Escherichia coli and Salmonella: Cellular and Molecular Biology, vol. 1:pp. 1357-1381, ASM Press.</td></tr><tr><td class="patent-data-table-td ">91</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Watanabe et al., 1990, Argric. Biol. Chem., 54, 519-525.</td></tr><tr><td class="patent-data-table-td ">92</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Watzele et al., 1989, J. Biol. Chem., 264, 8753-8758.</td></tr><tr><td class="patent-data-table-td ">93</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">White et al., 1967, Biochem. J., 105, 121-125.</td></tr><tr><td class="patent-data-table-td ">94</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">White, 1968, Biochem. J., 106, 847-858.</td></tr><tr><td class="patent-data-table-td ">95</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Witkowski et al., Biochemistry 38:11643-11650, 1999.</td></tr><tr><td class="patent-data-table-td ">96</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wolfe et al., 1956, Arch. Biochem. Biophys., 64, 489-497.</td></tr><tr><td class="patent-data-table-td ">97</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wu et al., 1971, J. Bacteriol., 105, 455-466.</td></tr><tr><td class="patent-data-table-td ">98</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Xin-Yi et al., 2001, Journal of Changde Teachers University (Natural Science Edition), 13, 63-65.</td></tr><tr><td class="patent-data-table-td ">99</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yamano et al., 1994, Biosci. Biotechnol. Biochem., 58, 193-195.</td></tr><tr><td class="patent-data-table-td ">100</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yamano et al., 1996, Biosci. Biotechnol. Biochem., 60, 1320-1323.</td></tr><tr><td class="patent-data-table-td ">101</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yamano et al., 1997, Biosci. Biotechnol. Biochem., 61, 1349-1353.</td></tr><tr><td class="patent-data-table-td ">102</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yamano et al., 2000, Mar. Biotechnol., 2, 57-64.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8124381">US8124381</a></td><td class="patent-data-table-td patent-date-value">Dec 19, 2007</td><td class="patent-data-table-td patent-date-value">Feb 28, 2012</td><td class="patent-data-table-td ">Arkion Life Sciences</td><td class="patent-data-table-td ">Process and materials for production of glucosamine and N-acetylglucosamine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010047516A2?cl=en">WO2010047516A2</a></td><td class="patent-data-table-td patent-date-value">Oct 20, 2009</td><td class="patent-data-table-td patent-date-value">Apr 29, 2010</td><td class="patent-data-table-td ">Cj Cheiljedang Corporation</td><td class="patent-data-table-td ">Corynebacterium genus microorganism with ability to produce n-acetyl glucosamine and method for producing n-acetyl glucosamine or glucosamine using same</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S084000">435/84</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S252300">435/252.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S320100">435/320.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023200">536/23.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S015000">435/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S193000">435/193</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S183000">435/183</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S350000">530/350</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S325000">435/325</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069100">435/69.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S252330">435/252.33</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001480000">C12Q1/48</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0009100000">C12N9/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001200000">C12N1/20</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0021060000">C12P21/06</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001210000">C12N1/21</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015520000">C12N15/52</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015000000">C12N15/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0009000000">C12N9/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0005100000">C12N5/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014000000">C07K14/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0019260000">C12P19/26</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12P19/26">C12P19/26</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=2W2BBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/52">C12N15/52</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12N15/52</span>, <span class="nested-value">C12P19/26</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Dec 10, 2013</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 1, 36, 40 AND 43 ARE DETERMINED TO BE PATENTABLE AS AMENDED.CLAIMS 2-5, 8-26, 35, 37-39, 41-42, 55 AND 63-67, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.CLAIMS 6-7, 27-34, 44-54 AND 56-62 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 4, 2012</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120913</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 19, 2011</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 11, 2003</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ARKION LIFE SCIENCES LLC D/B/A BIO-TECHNICAL RESOU</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:DENG, MING-DE;ANGERER, J. DAVID;CYRON, DON;AND OTHERS;REEL/FRAME:014781/0751;SIGNING DATES FROM 20031121 TO 20031204</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_4ff636b3d23669b7103f3b3a3a18b4cd.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U3R-G7Yy_AOlA4sbZyJYUJYEjcXJg\u0026id=2W2BBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U2gmc-OHcBuykZyT_LiUZif_OERww\u0026id=2W2BBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U22Rj7UmeMM3UDBML-nED6QLS8iYQ","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Fermentation_of_genetic_engineered_micro.pdf?id=2W2BBAABERAJ\u0026output=pdf\u0026sig=ACfU3U0xS_vO4JRgUrxaO9bNnRurj8A0zg"},"sample_url":"http://www.google.com/patents/reader?id=2W2BBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>