<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7462489 - Using proteome profile as diagnostic tool in detection and classification of ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Using proteome profile as diagnostic tool in detection and classification of metabolic disorders"><meta name="DC.contributor" content="Klaus F. Ley" scheme="inventor"><meta name="DC.contributor" content="David M. Smalley" scheme="inventor"><meta name="DC.contributor" content="Ley Klaus F" scheme="assignee"><meta name="DC.contributor" content="Smalley David M" scheme="assignee"><meta name="DC.date" content="2007-11-5" scheme="dateSubmitted"><meta name="DC.description" content="The present invention provides methods for identifying candidate biomarkers in the plasma microparticle proteome. It also provides methods for isolating, identifying, and analyzing microparticles derived from plasma, particularly platelet-poor plasma."><meta name="DC.date" content="2008-12-9" scheme="issued"><meta name="DC.relation" content="US:20070238864:A1" scheme="references"><meta name="DC.relation" content="US:5512657" scheme="references"><meta name="citation_patent_number" content="US:7462489"><meta name="citation_patent_application_number" content="US:11/935,048"><link rel="canonical" href="http://www.google.com/patents/US7462489"/><meta property="og:url" content="http://www.google.com/patents/US7462489"/><meta name="title" content="Patent US7462489 - Using proteome profile as diagnostic tool in detection and classification of metabolic disorders"/><meta name="description" content="The present invention provides methods for identifying candidate biomarkers in the plasma microparticle proteome. It also provides methods for isolating, identifying, and analyzing microparticles derived from plasma, particularly platelet-poor plasma."/><meta property="og:title" content="Patent US7462489 - Using proteome profile as diagnostic tool in detection and classification of metabolic disorders"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("8FTsU7naO6XKsQSHpYLoCQ"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("CZE"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("8FTsU7naO6XKsQSHpYLoCQ"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("CZE"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7462489?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7462489"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=DXpsBQABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7462489&amp;usg=AFQjCNGHt_ZqU81m4s5vT8gbXKRB1JSK3Q" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7462489.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7462489.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20080171312"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7462489"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7462489" style="display:none"><span itemprop="description">The present invention provides methods for identifying candidate biomarkers in the plasma microparticle proteome. It also provides methods for isolating, identifying, and analyzing microparticles derived from plasma, particularly platelet-poor plasma....</span><span itemprop="url">http://www.google.com/patents/US7462489?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7462489 - Using proteome profile as diagnostic tool in detection and classification of metabolic disorders</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7462489 - Using proteome profile as diagnostic tool in detection and classification of metabolic disorders" title="Patent US7462489 - Using proteome profile as diagnostic tool in detection and classification of metabolic disorders"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7462489 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 11/935,048</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Dec 9, 2008</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Nov 5, 2007</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Nov 9, 2006</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2672204A1">CA2672204A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2092325A2">EP2092325A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2092325A4">EP2092325A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20080171312">US20080171312</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2008060896A2">WO2008060896A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2008060896A3">WO2008060896A3</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">11935048, </span><span class="patent-bibdata-value">935048, </span><span class="patent-bibdata-value">US 7462489 B2, </span><span class="patent-bibdata-value">US 7462489B2, </span><span class="patent-bibdata-value">US-B2-7462489, </span><span class="patent-bibdata-value">US7462489 B2, </span><span class="patent-bibdata-value">US7462489B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Klaus+F.+Ley%22">Klaus F. Ley</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22David+M.+Smalley%22">David M. Smalley</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Ley+Klaus+F%22">Ley Klaus F</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Smalley+David+M%22">Smalley David M</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7462489.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7462489.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7462489.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (2),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (2),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (13),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (7)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7462489&usg=AFQjCNGAgA2ClAQJ_nhMh8fUWxN4aspWfA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7462489&usg=AFQjCNFfdwv06O30Xki5oHUk3ZbXt3n7Uw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7462489B2%26KC%3DB2%26FT%3DD&usg=AFQjCNHBb2QpNVMKaBO6SJf6bfQFozcQ2g">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT130023143" lang="EN" load-source="ifi">Using proteome profile as diagnostic tool in detection  and classification of metabolic disorders </invention-title></span><br><span class="patent-number">US 7462489 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA51398047" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">The present invention provides methods for identifying candidate biomarkers in the plasma microparticle proteome. It also provides methods for isolating, identifying, and analyzing microparticles derived from plasma, particularly platelet-poor plasma.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(4)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7462489B2/US07462489-20081209-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7462489B2/US07462489-20081209-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7462489B2/US07462489-20081209-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7462489B2/US07462489-20081209-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7462489B2/US07462489-20081209-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7462489B2/US07462489-20081209-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7462489B2/US07462489-20081209-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7462489B2/US07462489-20081209-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(20)</span></span></div><div class="patent-text"><div mxw-id="PCLM9495826" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A method of identifying candidate biomarkers in the plasma microparticle sub-proteome, comprising:
<div class="claim-text">(a) isolating proteins from blood plasma microparticles obtained from at least one blood sample from a case and at least one blood sample from a control;</div>
<div class="claim-text">(b) identifying the proteins;</div>
<div class="claim-text">(c) applying a statistical test to measure the abundance of each protein identified; and,</div>
<div class="claim-text">(d) determining the presence of positive and negative biomarkers by comparing the abundance of each protein identified in at least one case blood sample with the abundance of the same protein in at least one control blood sample.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein isolating proteins (a), further comprises:
<div class="claim-text">(e) isolating a plasma microparticle-enriched fraction from a blood sample;</div>
<div class="claim-text">(f) purifying plasma microparticles from the enriched fraction; and,</div>
<div class="claim-text">(g) isolating proteins from the purified microparticles.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The method of <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein isolating a plasma microparticle-enriched fraction (e), comprises:
<div class="claim-text">(h) separating platelet-poor plasma (PPP) from a blood sample; and,</div>
<div class="claim-text">(i) fractionating by size to obtain a plasma microparticle-enriched fraction.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="CLM-00003">claim 3</claim-ref>, wherein purifying plasma microparticles (f), comprises:
<div class="claim-text">(j) fractionating the plasma microparticle-enriched fraction to obtain plasma microparticles.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. The method of <claim-ref idref="CLM-00004">claim 4</claim-ref>, wherein isolating proteins (g), comprises:
<div class="claim-text">(k) delipidating the resulting plasma microparticles to obtain proteins.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. The method of <claim-ref idref="CLM-00002">claim 2</claim-ref>, further comprising:
<div class="claim-text">(l) optionally cleaving the resulting proteins to obtain peptides.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. The method of <claim-ref idref="CLM-00003">claim 3</claim-ref>, wherein (h) separating the plasma microparticle-enriched fraction is performed by centrifugation.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. The method of <claim-ref idref="CLM-00003">claim 3</claim-ref>, wherein (i) purifying by fractionating by size is performed by running the fractionated plasma microparticles through a size exclusion column.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The method of <claim-ref idref="CLM-00004">claim 4</claim-ref>, wherein (j) fractionating the PPP is performed by centrifugation.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The method of <claim-ref idref="CLM-00009">claim 9</claim-ref>, wherein the centrifugation is sufficient to form a pellet comprising: plasma microparticles.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. The method of <claim-ref idref="CLM-00005">claim 5</claim-ref>, wherein (k) delipitating is performed by submitting the plasma microparticles to gel electrophoresis.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. The method of <claim-ref idref="CLM-00011">claim 11</claim-ref>, wherein the gel electrophoresis is 1D SDS PAGE.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00013" num="00013" class="claim">
      <div class="claim-text">13. The method of <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein (l) cleaving is performed by digesting the delipidated proteins with an appropriate enzyme.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00014" num="00014" class="claim">
      <div class="claim-text">14. The method of <claim-ref idref="CLM-00013">claim 13</claim-ref>, wherein the enzyme is trypsin.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00015" num="00015" class="claim">
      <div class="claim-text">15. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein (b) identifying is performed by sequencing the resulting proteins or peptides.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00016" num="00016" class="claim">
      <div class="claim-text">16. The method of <claim-ref idref="CLM-00015">claim 15</claim-ref>, wherein (b) identifying is performed by sequencing the resulting peptides.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00017" num="00017" class="claim">
      <div class="claim-text">17. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein the sequencing is performed by mass spectrometry.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00018" num="00018" class="claim">
      <div class="claim-text">18. The method of <claim-ref idref="CLM-00017">claim 17</claim-ref>, wherein the sequencing is performed by a HPLC-MS/MS.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00019" num="00019" class="claim">
      <div class="claim-text">19. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein (c) applying a statistical test is performed by using a method selected from: ion current, spectral count, and coverage.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00020" num="00020" class="claim">
      <div class="claim-text">20. The method of <claim-ref idref="CLM-00019">claim 19</claim-ref>, wherein the statistical method used is comparing spectral counts between cases and controls. </div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES16518370" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS-REFERENCE TO RELATED APPLICATIONS</heading> <p num="p-0002">The present application claims the priority benefits of U.S. Provisional Application No. 60/857,928, filed 9 Nov. 2006, which is expressly incorporated fully herein by reference.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p num="p-0003">Plasma contains one of the most complex and useful human proteomes. Detection of proteins within this type of sample is currently an important tool for evaluating the predisposition, presence, and progression of numerous clinical conditions. However, the current methodology of detecting and measuring individual proteins only begins to scratch the surface of its full potential. Today, tests for only about 120 different protein analytes have been approved by the FDA and approval of tests to detect new protein analytes has averaged only about one test per year over the last decade. This represents only an extremely small fraction of the 40,000 different proteins present in normal plasma and another 500,000 proteins which may be present under a variety of clinical conditions. This has led investigators to attempt to develop methods using high-throughput detection methods to identify many more proteins in the plasma. Current methodology, based on 2D gels, liquid chromatography, and/or mass spectrometry, has lead to the detection of about 500 different plasma proteins. However, this appears to be the limit of detection for the analysis of unfractionated plasma due to the limited dynamic range of these analytical methods. Abundant proteins, such as albumin (35-45 mg/ml), fibrinogen (2-6 mg/ml), immunoglobulins (12-18 mg/ml), and transferrin (2-3 mg/ml), interfere with the detection of proteins that may be present at up to 10 orders of magnitude lower concentrations. Removal of the ten most abundant proteins only increases the sensitivity by one order of magnitude, still leaving most proteins undetected. This has led some to attempts to fractionate the plasma samples to reduce its complexity prior to analysis.</p>
    <p num="p-0004">Microparticles (MPs) are small subcellular membranous vesicles released by essentially all cell types, especially when activated or under stress. They include ectosomes, generated from the ectocytosis (or blebbing) of the plasma membrane, and exosomes, released by fusion of intracellular multivesicular endosomes with the cell surface. In plasma, MPs were discovered as a blood component that promotes coagulation due to the presence of anionic phospholipids on their outer surface. These anionic phospholipids, which are mostly phosphatidylserine, are now widely used to detect MPs from blood samples using flow cytometry based on their affinity for fluorescently labeled annexin V. The cellular source of microparticles can be determined by cell-specific markers detected by flow cytometry. For example, MPs with CD41 (Glycoprotein IIb) expression are believed to be generated by platelets. Using this method, microparticles from platelets, erythrocytes, endothelial cells, neutrophils, lymphocytes and even smooth muscle cells have been detected in plasma. In healthy individuals, over 90% of these MPs originate from platelets, and under a wide range of pathological conditions the total number of microparticles and the number originating from various cell population are altered.</p>
    <p num="p-0005">Traditionally, the presence of these microparticles in the plasma is considered a sign of cellular activation and/or damage generated by the random blebbing of cell membranes. However, significant evidence has accumulated to indicate that MPs contain a unique set of proteins and inflammatory factors that have important biological functions. For example, a major pathway for the rapid secretion of IL-1β from THP-1 monocytes occurs via MP release in ectosomes, and this could be a general mechanism of release for secreted proteins that lack the conventional signaling peptide for secretory proteins. Researchers using MPs isolated either from plasma or the supernatants of stimulated cells have demonstrated a wide variety of effects on cellular function. These effects of MPs include increased expression of adhesion molecules on endothelial cells and monocytes, stimulation of cytokine release, altering vascular reactivity, inducing angiogenesis, decreasing the response to inflammatory mediators, and promoting fibrin formation. The effects vary depending on the cellular source of the microparticles, the method to generate them and the cells (or tissue) affected. While many groups have studied the effects of these MPs on various cells, the protein composition of plasma MPs was largely unknown except for the presence of a few cell-surface markers. Recently, we analyzed the proteome of platelet-derived microparticles and identified 578 proteins. Many microparticles in plasma are derived from platelets, but it is believed that valuable information resides in plasma microparticles derived from other cells as well.</p>
    <p num="p-0006">It is therefore desirable to find methods of identifying biomarkers in plasma microparticles that can be useful for diagnosing various diseases, disorders, and conditions.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0007">The present invention provides methods for isolating, identifying, and analyzing microparticles derived from plasma, particularly platelet-poor plasma. The present invention further provides methods for identifying and analyzing biomarker components of microparticles, particularly proteins and peptides. In one aspect the peptides are components of larger proteins. Plasma-derived microparticles (MPs) were isolated from human blood of three healthy donors using gel filtration and ultracentrifugation. Platelet-derived MPs were produced from human platelets by incubation with ADP, followed by gel filtration and ultracentrifugation. A fraction of each sample was labeled with the ICAT method (U.S. Pat. No. 6,629,040) and processed using the gel-based protocol using manufactures instructions (Applied Biosystems). Another fraction was run on SDS PAGE, digested with trypsin, and extracted. Both sets of samples were analyzed by LC-MS on a ThermoFinnigan LTQ mass spectrometer. Peptide identifications were made using a first-pass filtering of standard criteria of cross correlation values &gt;2.0 (+1 charge), 2.2 (+2 charge) and 3.5 (+3 charge) and requiring that all peptides be fully tryptic. The samples were compared using spectral counts or ICAT. At least twenty-six proteins were identified herein which are found in plasma-derived, but not in platelet-derived MPs, and three more that were much more abundant in plasma-derived MPs.</p>
    <p num="p-0008">The present invention is useful for identifying, measuring, and analyzing additional biomarkers from blood other than those specifically disclosed herein. In one aspect, these biomarkers are useful for identifying disease states in subjects, for monitoring progression of diseases and disorders, and for monitoring response to treatment. The present invention is also useful for identifying a predisposition to a disease or disorder by detecting a biomarker of the invention associated with said disease or disorder. The present invention is also useful for identifying and comparing proteins and peptides from normal and diseased tissues other than blood. The present invention is also useful for identifying biomarkers that have medical prognostic value for patients receiving treatment for a disease to determine how well the treatment is working.</p>
    <p num="p-0009">In normal humans, over 90% of microparticles are from platelets. Therefore, platelet microparticles represent a “background” and are less likely to contain useful information. To this end, the present invention provides in an aspect for subtracting the platelet microparticle proteome from the plasma microparticle proteome.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p num="p-0010"> <figref idrefs="DRAWINGS">FIG. 1</figref>, comprising A and B, provides a schematic representation of a comparative proteomic analysis of plasma MPs and platelet MPs. Two complementary approaches to compare the proteome of plasma and platelet MPs are depicted. A) Comparative analysis of unlabeled proteins based on spectral count. B) Comparative analysis of ICAT-labeled proteins by manual quantitation of labeled peptide ion intensities.</p>
      <p num="p-0011"> <figref idrefs="DRAWINGS">FIG. 2</figref>, comprising parts A and B, illustrates a sample section of a 2D gel and analysis of the data as presented by MSight®. Files generated by the mass spectrometer were converted into .mzXML files and imported into MSight®. A) Sample portion of intensity plot for M/Z between 810 to 1020 and retention time from 90 to 145 min. B) Peptides were annotated for relative quantitation and identification. While most spots appears as pairs of equal intensity, there are several that do not (6, 9, 29, etc.).</p>
      <p num="p-0012"> <figref idrefs="DRAWINGS">FIG. 3</figref> represents a graphic illustration of a spectral count for 186 proteins from plasma-derived MPs and Platelet MPs. Log 2 of spectral count in plasma microparticles divided by spectral count in platelet microparticles for 186 unlabeled proteins. Proteins with peptides only present in plasma MPs are not shown but are included in Table 1. Five differentially expressed proteins are noted.</p>
      <p num="p-0013">FIG. <b>4</b>—a graphic representation of relative ion intensities for 94 peptides from plasma-derived MPs and platelet MPs following ICAT labeling. Normalized Log 2 of adjusted ion intensities for 94 peptides. Peptides from selected proteins are denoted.</p>
    </description-of-drawings> <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p num="p-0014">Thus, in an embodiment, the present invention provides a novel method of identifying candidate biomarkers in the plasma microparticle sub-proteome, comprising:
</p> <ul> <li id="ul0001-0001" num="0000"> <ul> <li id="ul0002-0001" num="0014">(a) isolating proteins from blood plasma microparticles obtained from at least one blood sample from a case and at least one blood sample from a control;</li> <li id="ul0002-0002" num="0015">(b) identifying the proteins;</li> <li id="ul0002-0003" num="0016">(c) applying a statistical test to measure the abundance of each protein identified; and,</li> <li id="ul0002-0004" num="0017">(d) determining the presence of positive and negative biomarkers by comparing the abundance of each protein identified in at least one case blood sample with the abundance of the same protein in at least one control blood sample.</li> </ul> </li> </ul> <p num="p-0015">In another embodiment, the present invention provides a novel method wherein isolating proteins (a), further comprises:
</p> <ul> <li id="ul0003-0001" num="0000"> <ul> <li id="ul0004-0001" num="0019">(e) isolating a plasma microparticle-enriched fraction from a blood sample;</li> <li id="ul0004-0002" num="0020">(f) purifying plasma microparticles from the enriched fraction; and,</li> <li id="ul0004-0003" num="0021">(g) isolating proteins from the purified microparticles.</li> </ul> </li> </ul> <p num="p-0016">In another embodiment, the present invention provides a novel method wherein isolating a plasma microparticle-enriched fraction (e), comprises:
</p> <ul> <li id="ul0005-0001" num="0000"> <ul> <li id="ul0006-0001" num="0023">(h) separating platelet-poor plasma (PPP) from a blood sample; and,</li> <li id="ul0006-0002" num="0024">(i) fractionating by size to obtain a plasma microparticle-enriched fraction.</li> </ul> </li> </ul> <p num="p-0017">In another embodiment, the present invention provides a novel method wherein purifying plasma microparticles (f), comprises:
</p> <ul> <li id="ul0007-0001" num="0000"> <ul> <li id="ul0008-0001" num="0026">(j) fractionating the plasma microparticle-enriched fraction to obtain plasma microparticles.</li> </ul> </li> </ul> <p num="p-0018">In another embodiment, the present invention provides a novel method wherein isolating proteins (g), comprises:
</p> <ul> <li id="ul0009-0001" num="0000"> <ul> <li id="ul0010-0001" num="0028">(k) delipidating the resulting plasma microparticles to obtain proteins.</li> </ul> </li> </ul> <p num="p-0019">In another embodiment, the present invention provides a novel method, further comprising:
</p> <ul> <li id="ul0011-0001" num="0000"> <ul> <li id="ul0012-0001" num="0030">(l) optionally cleaving the resulting proteins to obtain peptides.</li> </ul> </li> </ul> <p num="p-0020">In another embodiment, the present invention provides a novel method wherein (h) separating the plasma microparticle-enriched fraction is performed by centrifugation.</p>
    <p num="p-0021">In another embodiment, the present invention provides a novel method, wherein (i) purifying by fractionating by size is performed by running the fractionated plasma microparticles through a size exclusion column.</p>
    <p num="p-0022">In another embodiment, the present invention provides a novel method wherein (j) fractionating the PPP is performed by centrifugation.</p>
    <p num="p-0023">In another embodiment, the present invention provides a novel method wherein the centrifugation is sufficient to form a pellet comprising: plasma-derived microparticles.</p>
    <p num="p-0024">In another embodiment, the present invention provides a novel method wherein (k) delipitating is performed by submitting the plasma microparticles to gel electrophoresis.</p>
    <p num="p-0025">In another embodiment, the present invention provides a novel method, wherein the gel electrophoresis is SDS PAGE.</p>
    <p num="p-0026">In another embodiment, the present invention provides a novel method wherein (l) cleaving is performed by digesting the delipidated proteins with an appropriate enzyme.</p>
    <p num="p-0027">In another embodiment, the present invention provides a novel method wherein the enzyme is trypsin.</p>
    <p num="p-0028">In another embodiment, the present invention provides a novel method wherein (b) identifying is performed by sequencing the resulting proteins or peptides.</p>
    <p num="p-0029">In another embodiment, the present invention provides a novel method wherein (b) identifying is performed by sequencing the resulting peptides.</p>
    <p num="p-0030">In another embodiment, the present invention provides a novel method wherein the sequencing is performed by mass spectrometry.</p>
    <p num="p-0031">In another embodiment, the present invention provides a novel method wherein the sequencing is performed by a HPLC-MS/MS.</p>
    <p num="p-0032">In another embodiment, the present invention provides a novel method wherein (c) applying a statistical test is performed by using a method selected from: ion current, spectral count, and coverage.</p>
    <p num="p-0033">In another embodiment, the present invention provides a novel method wherein the statistical method used is comparing spectral counts between cases and controls.</p>
    <p num="p-0034">A protein can be considered a positive biomarker when it shows an increased presence in a case when compared with a control and a protein can be considered a negative biomarker when it shows a decreased presence in a case when compared with a control.</p>
    <p num="p-0035">Purifying plasma microparticles is a process wherein abundant plasma proteins (e.g., albumin, fibrinogen, and compliment factors) are suppressed (e.g., either absent or found in low counts in the spectral analysis).</p>
    <p num="p-0036">The process of the present invention can further comprise collecting a blood sample from a case and/or control. These blood samples may be drawn fresh via a variety of methods, examples of which include venipuncture, an indwelling venous or arterial catheter, an intraoperative sample, and blood removed from a wound site during or after surgery. Blood may also be obtained from blood banks. It can be desirable to obtain plasma from the sample immediately (e.g., within 24 hours) and processed to obtain MPs. Plasma may also be obtained from plasma or serum banks. The yield of MP proteins from frozen plasma or serum is typically lower than from fresh plasma. However frozen plasma remains a potential sample source.</p>
    <p num="p-0037">A case is a subject having a disease, condition or disorder. A control is a subject with demonstrated absence of the disease, condition or disorder. For rare diseases, disorders, and conditions, controls are taken from the general population and matched for known risk factors. If the condition is rare (less than 1 in 100), the proposed method is still useful for discovering biomarkers even when the disease, condition or disorder is not demonstrated to be absent in the controls.</p>
    <p num="p-0038">It is expected that the present invention can be used to identify biomarkers in body fluids that include normal tissue samples, diseased tissue samples, biopsies, blood, saliva, feces, semen, tears, cerebrospinal fluid, tears, vaginal lavage fluid, bronchoalveolar lavage fluid, ERCP lavage fluid, and any body fluid other than urine.</p>
    <p num="p-0039">Platelet poor plasma (PPP) can be obtained by removal of platelets from plasma. One procedure that has been used to effect this separation is centrifugation. Typically, the blood sample is subjected to centrifugation until a platelet pellet is formed, with the supernatant being the PPP. Other types of separation technologies could be used. For example, one of ordinary skill in the art could envision using the micromachined filtering device of U.S. Pat. No. 6,637,257 to separate the PPP from the blood sample.</p>
    <p num="p-0040">Microparticle-containing fractions are collected from a size exclusion column, for example by gel filtration. The presence of microparticles is determined by testing fractions for the presence of proteins. A positive test indicates the presence of a plasma microparticle. Fractions are excluded when the protein content exceeds 100 micrograms per milliliter. Typically, this is determined using a MicroBCA protein assay kit from Pierce Biotechnology but other protein assays will also suffice.</p>
    <p num="p-0041">The level of centrifugation necessary to form a pellet comprising plasma microparticles is called ultracentrifugation. Ultracentrifugation is defined as centrifugation generating sedimentation accelerations, also known as g-forces, in access of 10,000 g, where g is the gravitational constant (9.81 m/s<sup>2</sup>). Typically, 100,000 g are used to sediment MPs.</p>
    <p num="p-0042">It can be beneficial to cleave the delipidated proteins in order to simplify their identification. The identification can then be achieved by sampling the peptide fragments of the digested proteins by MS and sequencing them by MS/MS.</p>
    <p num="p-0043">Peptides are identified using a statistical algorithm that compares the top 500 peptides compatible with the masses measured to a database based on the human (or other animal) genome. Examples of suitable software include Sequest® and MASCOT®.</p>
    <p num="p-0044">Any mass spectrometer sensitive enough to sequence proteins can be used. Examples include the ThermoFinnigan LTQ-FT and the ThermoFinnigan LTQ instrument. Time-of-flight mass spectrometers such as Q-TOF can also be used. Electrospray ionization (ESI) has been used as an ion source, but MALDI and related methods may also be used.</p>
    <p num="p-0045">Ion currents are compared by integrating the area under the curve for cases and controls, and forming the ratio. A statistical test is the t test. Spectral counts can be compared with t test followed by Bonferroni correction. Since spectral counts are not normally distributed, non-parametric tests are preferred, such as Wilcoxon rank sum test. One embodiment is contained in a software package called R. Coverage refers to the number of unique peptides per protein, and what percentage of the entire sequence is covered. In general, more unique peptides and higher coverage increase the confidence in protein detection. This is generally used in addition to the XCOR values defined above.</p>
    <p num="p-0046">Alternatively, spectral counts for individual peptides or groups of peptides identifying individual proteins or groups of proteins forming functional groups can be tested for differences between cases and controls by using distribution-free methods. One such method is random permutation. To determine false discovery rate, the labels can be removed from cases and controls, and random permutations of the data can be constructed as known in the art. Labels for cases and controls are then attached randomly to these permutations and tested for significance using the Wilcoxon rank sum test or other test known in the art. If the random permutation produces a result with a probability (p) value smaller than that obtained when the cases are compared with the controls, this constitutes a false discovery. The false discovery rate is then determined in analogy to procedures known in the art.</p>
    <p num="h-0006">Definitions</p>
    <p num="p-0047">In describing and claiming the invention, the following terminology will be used in accordance with the definitions set forth below.</p>
    <p num="p-0048">Abbreviations and Acronyms</p>
    <p num="p-0049">AIM means Apoptosis Inhibitor in Macrophages</p>
    <p num="p-0050">C4BP means Complement Component C4 Binding Protein</p>
    <p num="p-0051">ESI means electrospray ionization</p>
    <p num="p-0052">FCGBP means Fc fragment of IgG binding protein</p>
    <p num="p-0053">ICAT means Isotope-Coded Affinity Tag</p>
    <p num="p-0054">LC/MS means liquid chromatography/mass spectrometry</p>
    <p num="p-0055">LC/MS means liquid chromatography-tandem mass spectrometry</p>
    <p num="p-0056">MALDI means matrix-assisted laser desorption/ionization</p>
    <p num="p-0057">MPs means Microparticles</p>
    <p num="p-0058">MS/MS means tandem mass spectrometry.</p>
    <p num="p-0059">PAGE means Polyacrylamide gel electrophoresis</p>
    <p num="p-0060">PBS means Phosphate buffered saline</p>
    <p num="p-0061">PPP means Platelet-Poor Plasma</p>
    <p num="p-0062">PRP means Platelet-Rich Plasma</p>
    <p num="p-0063">SDS means sodium dodecyl sulfate</p>
    <p num="p-0064">TOF means time-of-flight</p>
    <p num="p-0065">vWF means von Willebrand Factor</p>
    <p num="p-0066">The articles “a” and “an” are used herein to refer to one or to more than one (i.e., to at least one) of the grammatical object of the article. By way of example, “an element” means one element or more than one element.</p>
    <p num="p-0067">As used herein, the term “affected cell” refers to a cell of a subject afflicted with a disease or disorder, which affected cell has an altered phenotype relative to a subject not afflicted with a disease or disorder. Cells or tissue are “affected” by a disease or disorder if the cells or tissue have an altered phenotype relative to the same cells or tissue in a subject not afflicted with a disease or disorder.</p>
    <p num="p-0068">As used herein, the term “biologically active fragments” or “bioactive fragment” of the polypeptides encompasses natural or synthetic portions of the full-length protein that are capable of specific binding to their natural ligand or to a new ligand not bound by the intact protein or of performing the function of the protein.</p>
    <p num="p-0069">A “biomarker” is a specific biochemical in the body which has a particular molecular feature that makes it useful for measuring the progress of disease or the effects of treatment, or for measuring a process of interest.</p>
    <p num="p-0070">A “compound,” as used herein, refers to a polypeptide, isolated nucleic acid, chemical, or other agent used, identified, or isolated in the method of the invention.</p>
    <p num="p-0071">A “control” cell, tissue, sample, or subject is a cell, tissue, sample, or subject of the same type as a test cell, tissue, sample, or subject. The control may, for example, be examined at precisely or nearly the same time the test cell, tissue, sample, or subject is examined. The control may also, for example, be examined at a time distant from the time at which the test cell, tissue, sample, or subject is examined, and the results of the examination of the control may be recorded so that the recorded results may be compared with results obtained by examination of a test cell, tissue, sample, or subject. The control may also be obtained from another source or similar source other than the test group or a test subject, where the test sample is obtained from a subject suspected of having a disease or disorder for which the test is being performed.</p>
    <p num="p-0072">A “case” cell, tissue, sample, or subject is one being examined or treated.</p>
    <p num="p-0073">The use of the word “detect” and its grammatical variants refers to measurement of the species without quantification, whereas use of the word “determine” or “measure” with their grammatical variants are meant to refer to measurement of the species with quantification. The terms “detect” and “identify” are used interchangeably herein.</p>
    <p num="p-0074">As used herein, a “detectable marker” or a “reporter molecule” is an atom or a molecule that permits the specific detection of a compound comprising the marker in the presence of similar compounds without a marker. Detectable markers or reporter molecules include, e.g., radioactive isotopes, antigenic determinants, enzymes, nucleic acids available for hybridization, chromophores, fluorophores, chemiluminescent molecules, electrochemically detectable molecules, and molecules that provide for altered fluorescence-polarization or altered light-scattering.</p>
    <p num="p-0075">A “disease” is a state of health of an animal wherein the animal cannot maintain homeostasis, and wherein if the disease is not ameliorated then the animal's health continues to deteriorate. In contrast, a “disorder” in an animal is a state of health in which the animal is able to maintain homeostasis, but in which the animal's state of health is less favorable than it would be in the absence of the disorder. Left untreated, a disorder does not necessarily cause a further decrease in the animal's state of health.</p>
    <p num="p-0076">A “fragment” or “segment” is a portion of an amino acid sequence, comprising at least one amino acid, or a portion of a nucleic acid sequence comprising at least one nucleotide. The terms “fragment” and “segment” are used interchangeably herein.</p>
    <p num="p-0077">The term “microparticle”, as used herein, refers to any protein containing particle less than 2 micrometers in diameter with a molecular weight of over 100,000 daltons. These include various lipoproteins and membrane vesicles released from cells (see, for example, Piccin, A. et. al, Blood Reviews 2007, 21, 157-171).</p>
    <p num="p-0078">As used herein, a “peptide” encompasses a sequence of 2 or more amino acid residues wherein the amino acids are naturally occurring or synthetic (non-naturally occurring) amino acids covalently linked by peptide bonds. No limitation is placed on the number of amino acid residues which can comprise a protein's or peptide's sequence. Proteins are intact natural or synthetic proteins. Peptides are fragments of proteins (e.g., trypic peptides). Tryptic peptides are peptides resulting from the action of the protease trypsin on natural or synthetic proteins.</p>
    <p num="p-0079">“Plurality” means at least two.</p>
    <p num="p-0080">The term “purified” relates to an enrichment of a molecule or compound relative to other components normally associated with the molecule or compound in a native environment. The term “purified” does not necessarily indicate that complete purity of the particular molecule has been achieved during the process. A “highly purified” compound as used herein refers to a compound that is greater than 90% pure.</p>
    <p num="p-0081">A “sample,” as used herein, refers preferably to a biological sample from a subject, including, but not limited to, normal tissue samples, diseased tissue samples, biopsies, blood, saliva, feces, semen, tears, cerebrospinal fluid, tears, vaginal lavage fluid, bronchoalveolar lavage fluid, ERCP lavage fluid, and any body fluid other than urine. A sample can also be any other source of material obtained from a subject that contains cells, tissues, or fluid of interest. A sample can also be obtained from cell or tissue culture.</p>
    <p num="p-0082">As used herein, the term “secondary antibody” refers to an antibody that binds to the constant region of another antibody (the primary antibody).</p>
    <p num="p-0083">As used herein, the term “solid support” relates to a solvent insoluble substrate that is capable of forming linkages (preferably covalent bonds) with various compounds. The support can be either biological in nature, such as, without limitation, a cell or bacteriophage particle, or synthetic, such as, without limitation, an acrylamide derivative, agarose, cellulose, nylon, silica, or magnetized particles.</p>
    <p num="p-0084">The term “standard,” as used herein, refers to something used for comparison. For example, a standard can be a known standard agent or compound which is administered or added to a control sample and used for comparing results when measuring said compound in a test sample. Standard can also refer to an “internal standard,” such as an agent or compound which is added at known amounts to a sample and is useful in determining such things as purification or recovery rates when a sample is processed or subjected to purification or extraction procedures before a marker of interest is measured.</p>
    <p num="p-0085">A “subject” of analysis, diagnosis, or treatment is an animal. Such animals include mammals, preferably a human.</p>
    <p num="p-0086">In one embodiment, the present invention provides methods for isolating microparticles from plasma which has been depleted of platelets. In one embodiment, the present invention provides a method for identifying and analyzing microparticles isolated from plasma. In one aspect, the method provides for isolating microparticles from platelet-poor plasma. In one aspect, the method provides for identifying and analyzing biomarkers associated with microparticles. In one aspect, the biomarkers are proteins and peptides, or homologs or fragments thereof. In one aspect, the presence of a biomarker identified by the methods of the invention, or a difference in the level of the biomarker relative to a normal control level, is indicative of a disease, disorder, or condition. In one embodiment, the present invention provides diagnostic assays for diseases, disorders, and conditions using biomarkers identified by the methods of the invention.</p>
    <p num="p-0087">The invention is now described with reference to the following Examples. These Examples are provided for the purpose of illustration only and the invention should in no way be construed as being limited to these Examples, but rather should be construed to encompass any and all variations that become evident as a result of the teaching provided herein.</p>
    <heading>EXAMPLES</heading> <p num="p-0088">Materials and Methods</p>
    <p num="p-0089">Isolation of platelets and platelet-derived MPs. Platelets and platelet-derived MPs were isolated as previously described (see Garcia B A, Smalley D M, Cho H, Shabanowitz J, Ley K, Hunt D F. The Platelet Microparticle Proteome. <i>J Proteome Res. </i>2005; 4: 1516-1521). Briefly, human blood was collected by venipuncture into 1/10 volume of an acid-citrate-dextrose (85 mM trisodium citrate, 83 mM dextrose, and 21 mM citric acid) solution. Platelet-rich plasma (PRP) was obtained by centrifugation at 110×g for 15 minutes. Platelets were pelleted by centrifugation at 710×g for 15 minutes, and the supernatant, platelet-poor plasma (PPP), was retained for isolation of plasma MPs (see below). The platelet pellet was washed three times, resuspended in 10 mL of Tyrode's buffer, and centrifuged one additional time at 110×g to remove remaining red blood cells or debris. To generate platelet-derived microparticles, ADP (10 μM final concentration) was added to the platelet suspension for 10 minutes. Platelets were removed by centrifugation (710×g for 15 minutes) and platelet-derived MPs were pelleted by centrifugation at 150,000×g for 90 minutes at 10° C.</p>
    <p num="p-0090">Isolation of plasma-derived MPs. Plasma-derived MPs were isolated by gel filtration chromatography followed by ultracentrifugation. Briefly, the platelet-poor plasma (PPP) generated above was centrifuged an additional two times to remove residual cells and cell debris at 710×g and 25° C. for 15 minutes. This plasma was then applied to a Sephacryl® S-500 HR (GE Healthcare, Piscataway, N.J.) gel filtration column and MP-containing fractions were concentrated by ultracentrifugation at 150,000×g for 90 minutes at 10° C.</p>
    <p num="p-0091">Sample preparation for unlabeled protein analysis. Platelet- and plasma-derived microparticle pellets were processed as depicted in <figref idrefs="DRAWINGS">FIG. 1A</figref>. MPs were resuspended in a minimal volume of PBS (phosphate buffered saline, pH 7.4) and a small aliquot was taken for protein analysis using the Micro BCA Protein Assay (Pierce Biotechnology, Inc., Rockford, Ill.). Forty microliters of plasma microparticles and an equivalent amount of protein from the platelet MPs, resuspended with PBS to 40 μL, were mixed with 10 μL of 5× SDS-PAGE loading buffer (0.5 M Tris, pH 6.8, 10% SDS, 38% glycerol, 0.1% bromophenol blue). The separate samples (50 μL each) were heated to 95° C. for 5 minutes, allowed to cool to room temperature, and centrifuged for 2 minutes at 14,000 rpm prior to loading onto the gel. Microparticle proteins were electrophoresed approximately 1 cm into a 7.5% acrylamide SDS-PAGE using a Mini-gel system (Biorad, Hercules, Calif.) at 150 V. The acrylamide gel section containing the proteins was cut out and placed in fixative (50% methanol, 12% acetic acid, 0.05% formalin) for 2 hours. The in-gel tryptic digestion of the lanes and the peptide extraction was performed as previously described (see Shevchenko A, Wilm M, Vorm O, Mann M. Mass spectrometric sequencing of proteins silver-stained polyacrylamide gels. <i>Anal Chem. </i>1996; 68: 850-858.). The extracted peptide solutions were lyophilized and reconstituted to 20 μL with 0.1% acetic acid for mass spectrometry analysis. Three sets of platelet- and plasma-derived MP peptides were generated, and each of these samples was analyzed by LC/MS twice.</p>
    <p num="p-0092">ICAT-Labeling, electrophoresis, digestion, and peptide enrichment Relative quantitation of proteins following ICAT labeling was performed as depicted in <figref idrefs="DRAWINGS">FIG. 1B</figref>. Briefly, platelet- and plasma-derived microparticle pellets were resuspended in PBS (Phosphate buffered saline, pH 7.4), and protein concentration was determined. Solutions containing equivalent protein amounts of paired samples (plasma MP and platelet-derived MP) were lyophilized and resuspended in 1% SDS in denaturing buffer. Samples were labeled and processed using an ICAT labeling kit (Applied Biosystems, Foster City, Calif.) as instructed with the following modifications.</p>
    <p num="p-0093">The initial labeling reaction was done at ½ of the recommended volume, protein amount, and ICAT reagent because of the low amount of protein obtained from each plasma MP preparation. The differentially-labeled proteins were mixed, applied to the gel, electrophoresed, and cut from the gel as described above except the loading buffer did not contain the reducing agent or SDS. The proteins were digested with trypsin, extracted from the gel and processed through the avidin column as recommended by manufacturer. The samples were lyophilized and reconstituted to 20 μL with 0.1% acetic acid for mass spectrometry analysis. This procedure was repeated three times with plasma MPs labeled with the light ICAT reagent for two of these samples and labeled with the heavy ICAT reagent in the third.</p>
    <p num="p-0094">Liquid chromatography/mass spectrometry (LC/MS) and protein identification. Samples were loaded onto a 360 μm o.d.×75 μm i.d. microcapillary fused silica precolumn packed with irregular 5-20 μm C18 resin. After sample loading, the precolumn was washed with 0.1% acetic acid for 15 minutes to remove any buffer salts or gel contaminants. The precolumn was then connected to a 360 μm o.d.×50 μm i.d. analytical column packed with regular 5 μm C18 resin constructed with an integrated electrospray emitter tip. Samples were gradient eluted with an 1100 series binary HPLC solvent delivery system (Agilent, Palo Alto, Calif.) directly into a ThermoFinnigan LTQ ion trap mass spectrometer (Thermo Electron Corp, San Jose, Calif.) at a flow rate of 60 nL/min. In another example, the samples were analyzed using a ThermoFinnigan LTQ-FT instrument, which provides superior mass resolution. The HPLC stepwise gradient used was initially 100% A, 5% B at 5 minutes, 50% B at 220 minutes, 100% B at 240 minutes, and restored to 100% A at 280 minutes (solvent A=0.1 M acetic acid, solvent B=70% acetonitrile in 0.1 M acetic acid). The LTQ mass spectrometer was operated in the data-dependent mode in which first an initial MS scan recorded the mass to charge (m/z) ratios of ions over the mass range 300-2000 Da, and then the 10 most abundant ions were automatically selected for subsequent collisionally-activated dissociation and an MS/MS spectrum recorded.</p>
    <p num="p-0095">All MS/MS data were searched against a human protein database downloaded from the NCBI website using the SEQUEST® program (Thermo Electron Corp.). For unlabeled peptides, a static modification of 57 Da for cysteine residues was employed in the search parameters. For ICAT-labeled peptides, a static modification of 227.127 was used for the light isotope label and an additional 9 Daltons for the heavy ICAT-labeled peptides. Peptide identifications were made using a first-pass filtering of standard criteria as previously described (see Washburn M P, Wolters D, Yates J R, 3rd. Large-scale analysis of the yeast proteome by multidimensional protein identification technology. <i>Nat Biotechnol. </i>2001; 19: 242-247), including cross correlation values ≧2.0 (+1 charge), 2.2 (+2 charge) and 3.5 (+3 charge) and all peptides must be fully tryptic. Protein assignments required at least 2 MS/MS spectra matches that passed the above criteria. Manual validation of at least one MS/MS spectrum-peptide sequence match per protein was performed for all proteins that were determined to be differentially expressed.</p>
    <p num="p-0096">Comparative analysis of unlabeled peptides using Spectral Count. For the unlabeled scheme (<figref idrefs="DRAWINGS">FIG. 1A</figref>), all search results not passing the quality criteria described above were eliminated. The number of spectra for each peptide was determined, and the number of total proteins detected were calculated. If any protein had a spectral count of less than 2 for either the plasma MPs or the platelet MPs, it was eliminated from that group. Only proteins with an overall spectral count of 10 or greater were analyzed further by this method. The ratio of spectra from the plasma MP versus the platelet MP was calculated, log 2 transformed, and then adjusted for an overall ratio score of 0.00 excluding peptides derived from vWF. The standard deviation (SD) of the log score was calculated, and all proteins that were over 3 SD above (or below) the mean were considered to be enriched (or depleted) with “high confidence”. Proteins with log 2 scores between 2 and 3 SD greater than the mean were classified as possible candidates that should be examined further.</p>
    <p num="p-0097">Comparative analysis of ICAT-labeled peptides using MSight®. Comparative quantitation was performed using MSight®, freely available from the Swiss Institute of Bioinformatics Expasy website. Data files (.RAW) generated using XCalibur Software (Thermo Electron Corp.) were converted to mzXML files using ReaDW (Institute for Systems Biology, Seattle, Wash.). These files were imported into MSight®, and the peptides were manually quantified. The peptides were identified using SEQUEST® as described above. A sample representation of the MSight® display for a small portion of one of these runs is shown in <figref idrefs="DRAWINGS">FIG. 2</figref> (ion intensities not shown). Peptides that were detected in all three ICAT analyses based on SEQUEST® results were quantified, if possible. Then, peptides with high ion intensities were quantified and linked backed to SEQUEST® results. Attempts to quantify peaks sometimes failed due to poor signal-to-noise ratios, overlapping peptides, and ambiguous identification of a given peak.</p>
    <p num="p-0098">Unless otherwise noted, ICAT quantitation results are only reported if good peak quantitation was possible in at least two of the three ICAT runs and labeled alternatively in these two runs. This led to the quantitation of 94 peptides. Utilizing the differential labeling, the ratios of the relative quantitation in plasma MPs versus platelet MPs were calculated, a log 2 transformation of these ratios was performed, and adjusted to generate an overall log 2 score of 0.00, excluding vWF. VWF was excluded because it was evident that there was significant enrichment in the plasma MPs and due to the large number of peptides examined for this protein and the extent of enrichment. To determine significance, the paired t-test was used to examine differences in peptide expression intensities between plasma MPs and platelet MPs. P&lt;0.05 was used to determine differentially expressed peptides.</p>
    <p num="h-0008">Results</p>
    <p num="p-0099">Relative Quantitation of Plasma MPs and Platelet MPs using Spectral Counts. Plasma MPs and platelet MPs were isolated from three individuals, the proteins processed and digested into peptides, and peptide analysis of each was performed in duplicate. The MS/MS spectrum was searched using SEQUEST® and any spectrum not passing first-pass filtering was discarded. From the plasma MP, 1343±589 spectra per run (total of 8058 spectra in 6 runs) were obtained, identifying 3726 unique peptides and 550 unique proteins with 2 or more spectra (not shown). From platelet MPs, we obtained 2087±812 spectra per run (total of 12,522 spectra in 6 runs), identifying 6975 unique peptides and 1004 unique proteins with 2 or more spectra (results not shown).</p>
    <p num="p-0100">For proteins with a cumulative spectral count of 10 or more, the relative spectral counts were compared between plasma MPs and platelet MPs (results not shown). The log 2 of the ratio of the spectral counts was normalized to an average of 0.00 (excluding vWF) which represents the null hypothesis of equal representation of proteins in plasma and platelet MPs. <figref idrefs="DRAWINGS">FIG. 3</figref> illustrates the normalized log 2 ratios for all proteins whose peptides were detected in both platelet and plasma MPs. In addition, there were 23 proteins for which peptides were detected in plasma MPs but not platelet MPs (see Table 1 below), all with high confidence (over 3 standard deviations away from mean). One of these, von Willebrand Factor (vWF), is clearly overexpressed in plasma MPs (over 3 standard deviations above platelet microparticles, 745 scans in plasma MP vs 23 scans in platelet MPs). The other two appeared to be overexpressed (with lower confidence—between 2 and 3 standard deviations away from mean) and we attempted to examine these further using ICAT-label (see section entitled “ICAT quantitation of Differentially Expressed Peptides based on Scan Number”). We showed with high confidence that a total of 24 proteins are enriched in plasma MPs versus platelet MPs. Complete description of these proteins and the peptide sequences detected were determined (results not shown). No proteins enriched in platelet MPs with high confidence were found.</p>
    <p num="p-0101">
      <tables id="TABLE-US-00001" num="00001"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="77pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <colspec colname="7" colwidth="28pt" align="center"> </colspec> <colspec colname="8" colwidth="28pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Plasma</td>
                <td class="description-td"> </td>
                <td class="description-td">Platelet</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">plasma</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">MP</td>
                <td class="description-td">Plasma</td>
                <td class="description-td">MP</td>
                <td class="description-td">Platelet</td>
                <td class="description-td">Spec-</td>
                <td class="description-td">MP</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Accession</td>
                <td class="description-td">total</td>
                <td class="description-td">MP</td>
                <td class="description-td">total</td>
                <td class="description-td">MP</td>
                <td class="description-td">tral</td>
                <td class="description-td">platelet</td>
              </tr> <tr class="description-tr"> <td class="description-td">Protein Name</td>
                <td class="description-td">No.</td>
                <td class="description-td">scans</td>
                <td class="description-td">Peptides</td>
                <td class="description-td">scans</td>
                <td class="description-td">Peptides</td>
                <td class="description-td">Count</td>
                <td class="description-td">MP</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="77pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="8" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Fc fragment of IgG</td>
                <td class="description-td">4503681</td>
                <td class="description-td">167</td>
                <td class="description-td">54</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">167</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">binding protein</td>
              </tr> <tr class="description-tr"> <td class="description-td">complement</td>
                <td class="description-td">4502503</td>
                <td class="description-td">114</td>
                <td class="description-td">23</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">114</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">component 4 binding</td>
              </tr> <tr class="description-tr"> <td class="description-td">protein, alpha</td>
              </tr> <tr class="description-tr"> <td class="description-td">haptoglobin-related</td>
                <td class="description-td">45580723</td>
                <td class="description-td">86</td>
                <td class="description-td">19</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">86</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">protein</td>
              </tr> <tr class="description-tr"> <td class="description-td">complement</td>
                <td class="description-td">4502501</td>
                <td class="description-td">86</td>
                <td class="description-td">37</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">86</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">component 4B</td>
              </tr> <tr class="description-tr"> <td class="description-td">galectin 3 binding</td>
                <td class="description-td">5031863</td>
                <td class="description-td">68</td>
                <td class="description-td">18</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">68</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">protein</td>
              </tr> <tr class="description-tr"> <td class="description-td">apolipoprotein B-100</td>
                <td class="description-td">4502153</td>
                <td class="description-td">37</td>
                <td class="description-td">25</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">37</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">alpha-2-macroglobulin</td>
                <td class="description-td">4557225</td>
                <td class="description-td">36</td>
                <td class="description-td">21</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">36</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">Haptoglobin</td>
                <td class="description-td">4826762</td>
                <td class="description-td">32</td>
                <td class="description-td">12</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">32</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">apolipoprotein A-I</td>
                <td class="description-td">4557321</td>
                <td class="description-td">28</td>
                <td class="description-td">10</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">28</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">CD5 antigen-like</td>
                <td class="description-td">5174411</td>
                <td class="description-td">24</td>
                <td class="description-td">14</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">24</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">protein</td>
              </tr> <tr class="description-tr"> <td class="description-td">Protein S, alpha</td>
                <td class="description-td">4506117</td>
                <td class="description-td">24</td>
                <td class="description-td">15</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">24</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">transferrin receptor</td>
                <td class="description-td">4507457</td>
                <td class="description-td">20</td>
                <td class="description-td">14</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">20</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">propionyl-Coenzyme A</td>
                <td class="description-td">4557833</td>
                <td class="description-td">18</td>
                <td class="description-td">1</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">18</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">carboxylase, alpha</td>
              </tr> <tr class="description-tr"> <td class="description-td">apolipoprotein E3</td>
                <td class="description-td">4557325</td>
                <td class="description-td">17</td>
                <td class="description-td">10</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">17</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">immunoglobulin J</td>
                <td class="description-td">21489959</td>
                <td class="description-td">15</td>
                <td class="description-td">5</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">15</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">chain</td>
              </tr> <tr class="description-tr"> <td class="description-td">complement</td>
                <td class="description-td">4502505</td>
                <td class="description-td">14</td>
                <td class="description-td">5</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">14</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">component 4 binding</td>
              </tr> <tr class="description-tr"> <td class="description-td">protein, beta</td>
              </tr> <tr class="description-tr"> <td class="description-td">brain adenylate cyclase</td>
                <td class="description-td">31083193</td>
                <td class="description-td">12</td>
                <td class="description-td">1</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">12</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">transferrin</td>
                <td class="description-td">4557871</td>
                <td class="description-td">11</td>
                <td class="description-td">8</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">11</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">complement</td>
                <td class="description-td">4557385</td>
                <td class="description-td">11</td>
                <td class="description-td">10</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">11</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">component 3</td>
              </tr> <tr class="description-tr"> <td class="description-td">Ig kappa chain</td>
                <td class="description-td">51475407</td>
                <td class="description-td">10</td>
                <td class="description-td">2</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">10</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">apolipoprotein C-III</td>
                <td class="description-td">4557323</td>
                <td class="description-td">10</td>
                <td class="description-td">2</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">10</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">tenascin C</td>
                <td class="description-td">4504549</td>
                <td class="description-td">10</td>
                <td class="description-td">9</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">10</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">coagulation factor VIII</td>
                <td class="description-td">4503647</td>
                <td class="description-td">10</td>
                <td class="description-td">10</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">10</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">von Willebrand factor*</td>
                <td class="description-td">4507907</td>
                <td class="description-td">745</td>
                <td class="description-td">102</td>
                <td class="description-td">23</td>
                <td class="description-td">15</td>
                <td class="description-td">768</td>
                <td class="description-td">32.39</td>
              </tr> <tr class="description-tr"> <td class="description-td">solute carrier family 4,</td>
                <td class="description-td">4507021</td>
                <td class="description-td">12</td>
                <td class="description-td">7</td>
                <td class="description-td">3</td>
                <td class="description-td">3</td>
                <td class="description-td">15</td>
                <td class="description-td">4.00</td>
              </tr> <tr class="description-tr"> <td class="description-td">anion exchanger,</td>
              </tr> <tr class="description-tr"> <td class="description-td">member 1*</td>
              </tr> <tr class="description-tr"> <td class="description-td">four and a half LIM</td>
                <td class="description-td">21361122</td>
                <td class="description-td">11</td>
                <td class="description-td">4</td>
                <td class="description-td">3</td>
                <td class="description-td">1</td>
                <td class="description-td">14</td>
                <td class="description-td">3.67</td>
              </tr> <tr class="description-tr"> <td class="description-td">domains 1*</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="left" id="FOO-00001" class="description-td" colspan="8">*in plasma-derived compared to platelet-derived microparticles.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0102">Quantitative Analysis of Plasma MPs and Platelet MPs using ICAT As a complementary approach to examine differences in the protein composition of platelet MPs versus plasma MPs, a differential analysis using isotope labeling was performed. MP were differentially labeled and mixed prior to PAGE, trypsin digestion, extraction, and LC/MS (<figref idrefs="DRAWINGS">FIG. 1B</figref>). This was performed three times, and 94 of the more abundant peptides were quantified (results not shown). The log 2 of the ratios of the peptide abundances in plasma MPs to platelet MPs was calculated, and the average adjusted to 0.00, excluding the peptides from vWF (<figref idrefs="DRAWINGS">FIG. 4</figref>). Expression of peptides from three proteins (albumin, beta globin, and vWF) were consistently enriched in plasma MPs versus platelet MPs. The results for vWF are consistent with those obtained from spectral count analysis. While the spectral counts for albumin and beta globin (and several other hemoglobin proteins) were all about 2-fold higher in the plasma MPs versus the platelet MPs, the prior method was not quantitative enough to demonstrate an enrichment.</p>
    <p num="p-0103">Peptides from three proteins, thrombospondin, and integrins alpha 2b and beta 3, that are normally associated with platelet function, appear to be enriched in the platelet MP. Peptides from more ubiquitous proteins, such as the cytoskeletal proteins, do not show such enrichment. One possible explanation could be that when platelet MPs are in the plasma, they could lose certain proteins either by degradation, cleavage, or binding to surfaces. It seems unlikely that it is caused simply by the presence of MPs from other sources because even if 20% of the MPs were not of platelet origin, that would not lead to a sufficient dilution of signal to be detected by this method.</p>
    <p num="p-0104">ICAT quantitation of Differentially Expressed Peptides based on Scan Number In order to confirm the results of the spectral count analysis of unlabelled proteins, the peptides from all proteins (except vWF) which were determined to be enriched in one fraction based on spectral count analysis were examined (results not shown). Of these 27 proteins, spectra for peptides from 16 of these were detected in at least one of the three ICAT analysis. Similar to the spectral count of the unlabelled samples, there was a dramatic difference in spectral counts of the ICAT-labeled proteins between the two samples for these “enriched proteins” with 63 spectra for peptides from these proteins in the plasma MPs while only 2 were detected in the platelet MPs (results not shown). Of the 63 Peptides identified by Sequest, the ion intensities of 33 (representing 27 different) peptides could be quantified. It is important to note in many cases, the peptide from the platelet microparticle sample was not visually detected and the background ion intensities was used. This likely led to an underestimation of the fold enrichment in plasma MPs. This is particularly true for a number of peptides that had poor signal-to-noise ratios due to low abundance. Peptides from all of the proteins believed to be enriched in the plasma MP had an adjusted log 2 score of over 1.58, representing at least a three fold increase in concentration. Dynamin 1-like protein, which was enriched in the platelet MP had a log 2 score less than 0.00, suggesting that it truly is enriched in the fraction.</p>
    <p num="p-0105">Discussion: Comparing the results from the two approaches (spectral counts versus ICAT-label) provides us with a wealth of information about the relative strengths and weaknesses of each. The most obvious strength of the spectral count approach is its ability to generate a large number of protein differences without extensive and time consuming manual quantitation. The main strength of the ICAT based approach is its superior quantitation; being able to detect 2-fold differences in abundance. While it is currently extremely time consuming to determine these differences, the development and availability of better software should alleviate this problem. It is interesting to note that spectral count appears to work for lower abundant peptides/proteins even when the ion current does not appear above the background. These features demonstrate the complementary nature of the two approaches.</p>
    <p num="p-0106">Analysis of microparticles isolated from healthy volunteers shows 24 gene products enriched in plasma MPs compared to platelet microparticles isolated from the same individuals. Peptides from twenty three of these, represented by a total of 860 mass spectra and 325 peptides, were not detected in platelet MPs. Beyond statistical analysis, confidence in the detected differences is further enhanced by confirmation using direct ICAT-based comparison. The present study, for the first time, defines the protein composition of plasma microparticles as compared with platelet-derived microparticles using LC/MS as an unbiased approach. As expected, 186 of 209 gene products identified in plasma MPs are also found in platelet microparticles and platelets, confirming that most plasma MPs are derived from platelets. However, we found significant and potentially biologically important differences. Almost all (23 of 24) differentially expressed gene products as determined by spectral count were found exclusively in plasma-derived and not in platelet-derived MPs. Several lines of reasoning suggest that contamination with plasma proteins is not a major factor. First, of the most abundant plasma proteins, albumin, IgG, transferrin, fibrinogen, and IgA, only one, transferrin, was enriched in the plasma MPs versus the platelet MPs based on spectral counts and this protein had a relatively low number of these counts (109th most abundant protein). Its receptor, a membrane protein, presumably in exosomes (see below) had almost double the spectral counts. Second, the plasma-derived proteins detected in the plasma MPs are not a random collection as would be expected if it was a contaminant. For example, seven of these proteins are associated with apoptosis of cells. Four of them, complement 3, complement 4B, complement 4 binding proteins α and β, are from the complement pathway.</p>
    <p num="p-0107">One mechanism by which nucleated cells are resistant to complement-induced cell death is by shedding the complement components off of the membrane. The complement components detected in our study could represent MPs generated by subclinical autoimmune processes. Complement C3 is the most important complement component that can be activated by any of the three distinct pathways, classical (antibody-initiated), lectin (microbe-initiated) and alternative pathways. The most likely explanation is subclinical activation of the classical pathway, which includes C3 and C4, by autoantibodies. Subclinical levels of autoantibodies are widely found in healthy individuals. Some of them include so-called natural antibodies, for example, to blood group antigens. Others are against certain pathogens such as <i>Streptococcus pneumoniae </i>and cross-react with endogenous oxidized lipids such as oxLDL. However, C4 is involved in both the lectin and classical pathways, so we cannot exclude that intestinal or other commensal flora may provide a stimulus for subclinical complement activation that leads to expulsion of the complement-bound membranes into microparticles. C4BP (C4 Binding protein) is a potent circulating soluble inhibitor that inhibits both the classical and alternative pathway with a role in localizing complement regulatory activity to the surface of apoptotic cells, which undergo ectocytosis to release MPs. Part of this complex includes Protein S, another protein enriched in the plasma MPs which has been independently shown to stimulate the phagocytosis of apoptotic cells.</p>
    <p num="p-0108">Two other proteins enriched in plasma MPs, CD5-like antigen and galectin 3-binding protein also are associated with inhibition of apoptosis. CD5-like antigen, also called SP-α, has not been extensively studied, but its mouse homolog, AIM (apoptosis inhibitor in macrophages) has several informative studies (see Joseph S B, et. al., <i>Cell. </i>2004; 119: 299-309; and Miyazaki T, et. al., <i>J Exp Med. </i>1999; 189: 413-422). Cells from mice deficient in liver X receptors (LXRs) undergo accelerated apoptosis when challenged with <i>Listeria monocytogenes</i>, which is highly dependent on the AIM, a direct target for regulation by LXRα. Similarly, thymocytes have been shown to have increased susceptibility to apoptosis in mice lacking AIM. Galectin 3-binding protein, also called Mac-2 binding protein, associates with galectin 3, which increases cellular resistance to apoptosis. The presence of all these proteins in plasma MPs suggests that a larger percentage of these particles are either derived from apoptotic cells or an indication that microparticles are shed to prevent apoptosis.</p>
    <p num="p-0109">Another set of proteins associated with plasma MPs are related to iron transport and hemoglobin clearance: transferrin, transferrin receptor, haptoglobin, and haptoglobin-related protein. Therefore, its presence in the plasma microparticles, along with its ligand, transferrin, is logical. Haptoglobin is an abundant protein that binds free hemoglobin which would otherwise be toxic to cells, especially in the kidney. Hemoglobin was detected by our analysis in both plasma MPs and platelet MPs, suggesting that haptoglobin-hemoglobin complexes may associate with MPs. It is not known why haptoglobin and haptoglobin-related proteins are overexpressed in plasma MPs.</p>
    <p num="p-0110">The third category of plasma proteins overexpressed in plasma MPs are immunoglobulins, represented by immunoglobulin (Ig) J chain and kappa chain. This is not a random selection of immunoglobulins, again making plasma contamination a less likely explanation. IgJ is associated with IgM, which includes all natural antibodies. Also enriched are several complement components, as discussed above. These findings are consistent with our idea that the classical complement pathway may be activated at a low level through IgM auto-antibodies, leading to the formation of microparticles from unknown cells (not platelets) that remove IgJ and associated complement components from the affected cell membrane.</p>
    <p num="p-0111">Four proteins found herein to be overexpressed in plasma MPs compared to platelet MPs, are associated with lipoproteins (results not shown). Unlike the others described above, these most likely represent contamination of plasma MPs with the larger plasma lipoproteins of the chylomicron (apo A-I and C-III), chylomicron remnant (apo E), VLDL (apo A-I, B100, and C-III) and IDL (apo B100 and E) fractions. Plasma lipoproteins appear in overlapping fractions with microparticles, because their sizes and therefore, their gel filtration properties are similar to MPs (results not shown). While they should remain in the supernatant after the final centrifugation step, their contamination with the pellet is difficult to avoid. In addition to the four lipoproteins found overexpressed in plasma microparticles, apolipoproteins A-II, C-II, L-I, and Lp(a), were found in only the plasma MPs but the number of spectra for each was below 10, and were not examined further. Three proteins associated with blood coagulation, protein S, factor VIII, and vWF (data not shown), were enriched in plasma MPs compared to platelet MPs.</p>
    <p num="p-0112">vWF in platelet MPs being highly enriched in plasma MP as disclosed herein was unexpected, suggesting that endothelial vWF associates with plasma MPs or even promotes their formation. Endothelial vWF is secreted from Weibel-Palade bodies and previously has been shown to be present in endothelial-derived MPs under disease conditions. It exists as a very large multimer and is cleaved by MMP-13. Our proteomic analysis does not distinguish between monomers and multimers. The association of coagulation factors and platelet-binding factors is entirely consistent with the idea that plasma MPs are a pro-coagulant surface and associate with nascent thrombi.</p>
    <p num="p-0113">FCGBP is a protein with unknown function produced by intestinal and colonic goblet cells. Due to its cellular source and its ability to bind IGG, it has been hypothesized to assist in preventing antigen invasion into the mucosa. Kobayashi et al. found that FCGBP could inhibit a complement-mediated reaction and suggested that it may play an important role in immunological defense of mucosal surfaces (see <i>Gut. </i>2002; 51: 169-176). While this goes along with our theories of ectosome production discussed above, there is little evidence to support the connection between FCGBP and the prevention of cell death in vascular tissue. This is particularly true considering the location of FCGBP production. However, the detection of this protein in plasma MPs is very intriguing especially when one considers its prevalence (spectral count of 167 including 54 unique peptides).</p>
    <p num="p-0114">Among the remaining three proteins, propionyl-coenzyme A carboxylase alpha and brain adenylate cyclase are intracellular proteins, while tenascin is an extracellular matrix protein. Propionyl-coenzyme A carboxylase is a mitochondrial enzyme involved in the catabolism of amino acids, odd-chained fatty acids and other metabolites. Tenascin is an extracellular matrix protein associated with provisional and tumor matrices but not found in most mature tissues. Tenascin may be secreted at sites of small injuries and wound healing, and cells with rapid turnover such as leukocytes and wound fibroblasts may shed tenascin associated with MPs. Alternatively, tenascin may form aggregates that migrate in the MP fraction. All three of these three proteins were unexpected in the plasma MP fraction. Further studies are needed to examine their implications. However, they are clearly not a random assortment of plasma protein contaminants.</p>
    <p num="p-0115">In conclusion, the present disclosure establishes significant differences between plasma- and platelet-derived microparticle proteins. A large majority of these differences are proteins involved with apoptosis or iron transport. It is reassuring that very few proteins were detected in plasma MPs that could be classified as plasma contaminants. In studying the biology of complement, coagulation and other processes, microparticles will now have to be considered as important contributors for removing such components from cells. It remains to be seen if there are dramatic differences under pathological conditions and is the next step in determining their importance in a wide variety of diseases.</p>
    <p num="p-0116">Candidate Biomarkers for Arthritis</p>
    <p num="p-0117">The above-described procedures were used to identify candidate arthritis biomarkers. Blood samples were drawn from five patients matched for age, smoking, gender, and risk factors. One patient, the case, had arthritis. This patient's sample was analyzed in duplicate and compared to the other four, also analyzed in duplicate. The proteins shown in Table 2 below are significantly overexpressed in the case as determined by a stringent statistical test (Student's T test followed with Bonferroni correction for multiple comparisons) compared to the controls (i.e., other patients). Proteins that are associated with known problems of blood drawing such as hemolysis were removed. An example of a removed protein is SPTB spectrin beta isoform a, a protein mainly found in red blood cells and known to be associated with hemolysis, a common occurrence in blood sampling. The column “Bonferroni” contains the Bonferroni-corrected p-values (cutoff at 0.01). All of the shown proteins are positive biomarkers (i.e., they are higher when arthritis is present). Negative biomarkers, i.e., proteins that are detected less in the diseased sample, were removed for the purposes of this analysis. An example of a negative biomarker that was removed is STXBP2 Syntaxin-binding protein 2, which was found 0 times in the case and 9 times in controls. Negative biomarkers may still be useful, but positive biomarkers are often more desirable.</p>
    <p num="p-0118">
      <tables id="TABLE-US-00002" num="00002"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="140pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="49pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="4" rowsep="1" class="description-td" colspan="4">TABLE 2</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Bonferroni-</td>
                <td class="description-td">ratio</td>
              </tr> <tr class="description-tr"> <td class="description-td">Protein</td>
                <td class="description-td">Accession</td>
                <td class="description-td">corrected p</td>
                <td class="description-td">disease/control</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="140pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="49pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Isoform Gamma-B of Fibrinogen gamma chain</td>
                <td class="description-td">21891</td>
                <td class="description-td">0.0000002</td>
                <td class="description-td">4</td>
              </tr> <tr class="description-tr"> <td class="description-td">precursor</td>
              </tr> <tr class="description-tr"> <td class="description-td">CATSPER3 cation channel, sperm associated 3</td>
                <td class="description-td">256795</td>
                <td class="description-td">0.0005068</td>
                <td class="description-td">16</td>
              </tr> <tr class="description-tr"> <td class="description-td">Alpha-1-acid glycoprotein 1 precursor</td>
                <td class="description-td">22429</td>
                <td class="description-td">0.0006064</td>
                <td class="description-td">16</td>
              </tr> <tr class="description-tr"> <td class="description-td">Isoform 2 of WD repeat protein 1</td>
                <td class="description-td">216256</td>
                <td class="description-td">0.0009691</td>
                <td class="description-td">7</td>
              </tr> <tr class="description-tr"> <td class="description-td">MARK3 Isoform 1 of MAP/microtubule</td>
                <td class="description-td">183118</td>
                <td class="description-td">0.0012696</td>
                <td class="description-td">14</td>
              </tr> <tr class="description-tr"> <td class="description-td">affinity-regulating kinase 3</td>
              </tr> <tr class="description-tr"> <td class="description-td">SPTB spectrin beta isoform a</td>
                <td class="description-td">784382</td>
                <td class="description-td">0.0013030</td>
                <td class="description-td">14</td>
              </tr> <tr class="description-tr"> <td class="description-td">Zyxin</td>
                <td class="description-td">20513</td>
                <td class="description-td">0.0013883</td>
                <td class="description-td">9</td>
              </tr> <tr class="description-tr"> <td class="description-td">PDZ and LIM domain protein 1</td>
                <td class="description-td">10414</td>
                <td class="description-td">0.0013967</td>
                <td class="description-td">4</td>
              </tr> <tr class="description-tr"> <td class="description-td">RHOC Rho-related GTP-binding protein RhoC</td>
                <td class="description-td">27434</td>
                <td class="description-td">0.0015415</td>
                <td class="description-td">8</td>
              </tr> <tr class="description-tr"> <td class="description-td">precursor</td>
              </tr> <tr class="description-tr"> <td class="description-td">DEFA1; LOC653600; LOC728358 Neutrophil</td>
                <td class="description-td">5721</td>
                <td class="description-td">0.0015895</td>
                <td class="description-td">14</td>
              </tr> <tr class="description-tr"> <td class="description-td">defensin 1 precursor</td>
              </tr> <tr class="description-tr"> <td class="description-td">EPHB2 Isoform 1 of Ephrin type-B receptor 2</td>
                <td class="description-td">21275</td>
                <td class="description-td">0.0015895</td>
                <td class="description-td">14</td>
              </tr> <tr class="description-tr"> <td class="description-td">precursor</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACTR3B actin-related protein 3-beta isoform 1</td>
                <td class="description-td">7068</td>
                <td class="description-td">0.0096585</td>
                <td class="description-td">8</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0119">Other methods which were used but not described herein are well known and within the competence of one of ordinary skill in the art of cell biology, molecular biology, biochemistry, analytical chemistry, bioinformatics and statistics. The invention should not be construed to be limited solely to the assays and methods described herein, but should be construed to include other methods and assays as well. One of skill in the art will know that other assays and methods are available to perform the procedures described herein. Headings are included herein for reference and to aid in locating certain sections. These headings are not intended to limit the scope of the concepts described therein under, and these concepts may have applicability in other sections throughout the entire specification. The disclosures of each and every patent, patent application, and publication cited herein are hereby incorporated herein by reference in their entirety. While this invention has been disclosed with reference to specific embodiments, it is apparent that other embodiments and variations of this invention may be devised by the previous description of the disclosed embodiments is provided to enable any person skilled in the art to make or use the present invention. Various modifications to these embodiments will be readily apparent to those skilled in the art, and the generic principles defined herein may be applied to other embodiments without departing from the spirit or scope of the invention. Accordingly, the present invention is not intended to be limited to the embodiments shown herein but is to be accorded the widest scope consistent with the principles and novel features disclosed herein.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5512657">US5512657</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 6, 1994</td><td class="patent-data-table-td patent-date-value">Apr 30, 1996</td><td class="patent-data-table-td ">Bainbridge Sciences, Inc.</td><td class="patent-data-table-td ">Detection of complexes which include basement membrane components as diagnostic of cancer and other diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070238864">US20070238864</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 19, 2006</td><td class="patent-data-table-td patent-date-value">Oct 11, 2007</td><td class="patent-data-table-td ">University Of Florida Research Foundation, Inc.</td><td class="patent-data-table-td ">Multidimensional protein separation</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7888035">US7888035</a></td><td class="patent-data-table-td patent-date-value">Oct 30, 2009</td><td class="patent-data-table-td patent-date-value">Feb 15, 2011</td><td class="patent-data-table-td ">Caris Mpi, Inc.</td><td class="patent-data-table-td ">Methods for assessing RNA patterns</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7897356">US7897356</a></td><td class="patent-data-table-td patent-date-value">Nov 12, 2009</td><td class="patent-data-table-td patent-date-value">Mar 1, 2011</td><td class="patent-data-table-td ">Caris Life Sciences</td><td class="patent-data-table-td ">Methods and systems of using exosomes for determining phenotypes</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc436/defs436.htm&usg=AFQjCNGx2wdEO-lMZYH5-JBwc-X_dDDKIg#C436S016000">436/16</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc436/defs436.htm&usg=AFQjCNGx2wdEO-lMZYH5-JBwc-X_dDDKIg#C436S089000">436/89</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S002000">435/2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc436/defs436.htm&usg=AFQjCNGx2wdEO-lMZYH5-JBwc-X_dDDKIg#C436S086000">436/86</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc514/defs514.htm&usg=AFQjCNG3QPK1Vhbpe6jO1mfvD3pMcfCuxA#C514S001100">514/1.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A01N0001020000">A01N1/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A01N0033000000">A01N33/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A01N0031000000">A01N31/00</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N33/6803">G01N33/6803</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N33/6848">G01N33/6848</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DXpsBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N2550/00">G01N2550/00</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">G01N33/68A</span>, <span class="nested-value">G01N33/68A12</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jul 16, 2013</td><td class="patent-data-table-td ">ERR</td><td class="patent-data-table-td ">Erratum</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">&quot;ALL REFERENCE TO REEXAMINATION CERTIFICATE NO. C1 7,462,489 TO KLAUS F.LEY, ET AL, OF LA JOLLA, CAFOR METHODS FOR IDENTIFYING AND ANALYZING BIOMARKERS FROM PLASMA-DERIVED MICROPARTICLES, APPEARING IN THE OFFICIAL GAZETTE OF JULY 03, 2012, SHOULD BE DELETED, SINCE THE REEXAMINATION CERTIFICATE HAS BEEN VACATED.&quot;</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 7, 2012</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 7, 2012</td><td class="patent-data-table-td ">SULP</td><td class="patent-data-table-td ">Surcharge for late payment</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 11, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:LEY, KLAUS F;SMALLEY, DAVID M;SIGNING DATES FROM 20120619 TO 20120623;REEL/FRAME:028531/0420</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">MICROPARTICLES PROTEOMICS LLC, CALIFORNIA</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 3, 2012</td><td class="patent-data-table-td ">FPB1</td><td class="patent-data-table-td ">Expired due to reexamination which canceled all claims</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 28, 2011</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20110502</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0bh8EIYYs1CgGdoeLc5FVA8U58_A\u0026id=DXpsBQABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U3VHBDIF0dWjKoULL8sLTRuho6aJA\u0026id=DXpsBQABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U1__ge_J0oIDiXu5YIEru68DFzknw","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Using_proteome_profile_as_diagnostic_too.pdf?id=DXpsBQABERAJ\u0026output=pdf\u0026sig=ACfU3U3JkZtlmSoFw-jxHSSSH0nZR3GS-Q"},"sample_url":"http://www.google.com/patents/reader?id=DXpsBQABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>