<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7534879 - Modulation of exon recognition in pre-mRNA by interfering with the secondary ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Modulation of exon recognition in pre-mRNA by interfering with the secondary RNA structure"><meta name="DC.contributor" content="Judith C. T. van Deutekom" scheme="inventor"><meta name="DC.contributor" content="Academisch Ziekenhuis Leiden" scheme="assignee"><meta name="DC.date" content="2008-8-25" scheme="dateSubmitted"><meta name="DC.description" content="The invention provides a method for generating an oligonucleotide with which an exon may be skipped in a pre-mRNA and thus excluded from a produced mRNA thereof. Further provided are methods for altering the secondary structure of an mRNA to interfere with splicing processes and uses of the oligonucleotides and methods in the treatment of disease. Further provided are pharmaceutical compositions and methods and means for inducing skipping of several exons in a pre-mRNA."><meta name="DC.date" content="2009-5-19" scheme="issued"><meta name="DC.relation" content="EP:1054058:A1" scheme="references"><meta name="DC.relation" content="EP:1133993:A1" scheme="references"><meta name="DC.relation" content="EP:1191097:A1" scheme="references"><meta name="DC.relation" content="WO:2004083432:A1" scheme="references"><meta name="citation_reference" content="Aartsma-Rus et al., Targeted exon skipping as a potential gene correction therapy for Duchenne muscular dystrophy, Neuromuscular Disorders, 2002, S71-S77, vol. 12."><meta name="citation_reference" content="De Angelis et al., Chimeric snRNA molecules carrying antisense sequences against the splice junctions of exon 51 of the dystrophin pre-mRNA induce exon skipping and restoration of a dystrophin synthesis in delta 48-50 DMD cells, PNAS, Jul. 9, 2002, pp. 9456-9461, vol. 99, No. 14."><meta name="citation_reference" content="Dickson et al., Screening for antisense modulation of dystrophin pre-mRNA splicing, Neuromuscul. Disord., 2002, S67-70, Suppl. 1."><meta name="citation_reference" content="Dunckley et al., Modifications of splicing in the dystrophin gene in cultured Mdx muscle cells by antisense oligoribonucleotides, Human Molecular Genetics, 1995, pp. 1083-1090, vol. 5, No. 1."><meta name="citation_reference" content="Lu et al., Massive Idiosyncratic Exon Skipping Corrects the Nonsense Mutation in Dystrophic Mouse Muscle and Produces Functional Revertant Fibers by Clonal Expansion, The Journal of Cell Biology, Mar. 6, 2000, pp. 985-995, vol. 148, No. 5."><meta name="citation_reference" content="Mann et al., Antisense-induced exon skipping and synthesis of dystrophin in the mdx mouse, PNAS, Jan. 2, 2001, pp. 42-47, vol. 98, No. 1."><meta name="citation_reference" content="Mann et al., Improved antisense oligonucleotide induced exon skipping in the mdx mouse model of muscular dystrophy, The Journal of Gene Medicine, 2002, pp. 644-654, vol. 4."><meta name="citation_reference" content="Suter et al., Double-target antisense U7 snRNAs promote efficient skipping of an aberrant exon in three human B-thalassemic mutations, Human Molecular Genetics, 1999, pp. 2415-2423, vol. 8, No. 13."><meta name="citation_reference" content="Van Deutekom et al., Antisense-induced exon skipping restores dystrophin expression in DMD patient derived muscle cells, Human Molecular Genetics, 2001, pp. 1547-1554, vol. 10, No. 15."><meta name="citation_patent_number" content="US:7534879"><meta name="citation_patent_application_number" content="US:12/198,007"><link rel="canonical" href="http://www.google.com/patents/US7534879"/><meta property="og:url" content="http://www.google.com/patents/US7534879"/><meta name="title" content="Patent US7534879 - Modulation of exon recognition in pre-mRNA by interfering with the secondary RNA structure"/><meta name="description" content="The invention provides a method for generating an oligonucleotide with which an exon may be skipped in a pre-mRNA and thus excluded from a produced mRNA thereof. Further provided are methods for altering the secondary structure of an mRNA to interfere with splicing processes and uses of the oligonucleotides and methods in the treatment of disease. Further provided are pharmaceutical compositions and methods and means for inducing skipping of several exons in a pre-mRNA."/><meta property="og:title" content="Patent US7534879 - Modulation of exon recognition in pre-mRNA by interfering with the secondary RNA structure"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("wlfsU8RRo7qxBLOcgbAK"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("USA"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("wlfsU8RRo7qxBLOcgbAK"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("USA"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7534879?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7534879"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=zfa4BQABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7534879&amp;usg=AFQjCNGUuCC2zmU6729fsgYo4Iae7_ycwA" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7534879.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7534879.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20090076246"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7534879"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7534879" style="display:none"><span itemprop="description">The invention provides a method for generating an oligonucleotide with which an exon may be skipped in a pre-mRNA and thus excluded from a produced mRNA thereof. Further provided are methods for altering the secondary structure of an mRNA to interfere with splicing processes and uses of the oligonucleotides...</span><span itemprop="url">http://www.google.com/patents/US7534879?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7534879 - Modulation of exon recognition in pre-mRNA by interfering with the secondary RNA structure</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7534879 - Modulation of exon recognition in pre-mRNA by interfering with the secondary RNA structure" title="Patent US7534879 - Modulation of exon recognition in pre-mRNA by interfering with the secondary RNA structure"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7534879 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 12/198,007</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">May 19, 2009</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Aug 25, 2008</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Mar 21, 2003</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2519863A1">CA2519863A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2524255A1">CA2524255A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2524255C">CA2524255C</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1606407A2">EP1606407A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1606407B1">EP1606407B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8759507">US8759507</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20060099616">US20060099616</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20060147952">US20060147952</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20090076246">US20090076246</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20110312086">US20110312086</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20130072671">US20130072671</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20130302806">US20130302806</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2004083432A1">WO2004083432A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2004083446A2">WO2004083446A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2004083446A3">WO2004083446A3</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">12198007, </span><span class="patent-bibdata-value">198007, </span><span class="patent-bibdata-value">US 7534879 B2, </span><span class="patent-bibdata-value">US 7534879B2, </span><span class="patent-bibdata-value">US-B2-7534879, </span><span class="patent-bibdata-value">US7534879 B2, </span><span class="patent-bibdata-value">US7534879B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Judith+C.+T.+van+Deutekom%22">Judith C. T. van Deutekom</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Academisch+Ziekenhuis+Leiden%22">Academisch Ziekenhuis Leiden</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7534879.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7534879.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7534879.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (4),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (9),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (13),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (24),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (4)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7534879&usg=AFQjCNHt7MI9Mb0bDO7D_juavZ9hqMEwEg">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7534879&usg=AFQjCNHJ9esFvglzTz31Nht5wLdbACyjFg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7534879B2%26KC%3DB2%26FT%3DD&usg=AFQjCNErQhBrnmIXdVAphG4sti0XOS7SwQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT72010081" lang="EN" load-source="patent-office">Modulation of exon recognition in pre-mRNA by interfering with the secondary RNA structure</invention-title></span><br><span class="patent-number">US 7534879 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA53244225" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">The invention provides a method for generating an oligonucleotide with which an exon may be skipped in a pre-mRNA and thus excluded from a produced mRNA thereof. Further provided are methods for altering the secondary structure of an mRNA to interfere with splicing processes and uses of the oligonucleotides and methods in the treatment of disease. Further provided are pharmaceutical compositions and methods and means for inducing skipping of several exons in a pre-mRNA.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(7)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7534879B2/US07534879-20090519-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7534879B2/US07534879-20090519-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7534879B2/US07534879-20090519-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7534879B2/US07534879-20090519-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7534879B2/US07534879-20090519-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7534879B2/US07534879-20090519-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7534879B2/US07534879-20090519-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7534879B2/US07534879-20090519-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7534879B2/US07534879-20090519-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7534879B2/US07534879-20090519-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7534879B2/US07534879-20090519-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7534879B2/US07534879-20090519-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7534879B2/US07534879-20090519-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7534879B2/US07534879-20090519-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(3)</span></span></div><div class="patent-text"><div mxw-id="PCLM13147078" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. An isolated oligonucleotide of between 20 to 50 nucleotides comprising a sequence consisting of SEQ ID: 27.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The isolated oligonucleotide of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the oligonucleotide comprises a 2′-O-methyl-phosphorothioate oligoribonucleotide modification.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The isolated oligonucleotide of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the oligonucleotide is a peptide nucleic acid, locked nucleic acid, and/or morpholino phosphorodiamidate oligonucleotide.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES18534116" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS-REFERENCE TO RELATED APPLICATIONS</heading> <p num="p-0002">This application is a continuation of patent application Ser. No. 11/233,495, filed Sep. 21, 2005, pending, which is a continuation of PCT International Patent Application No. PCT/NL2003/000214, filed on Mar. 21, 2003, designating the United States of America, and published, in English, as PCT International Publication No. WO 2004/083432 A1 on Sep. 30, 2004, the contents of the entirety of each of which is incorporated by this reference.</p>
    <heading>TECHNICAL FIELD</heading> <p num="p-0003">The invention relates to the fields of molecular biology and medicine. More in particular, the invention relates to the restructuring of mRNA produced from pre-mRNA, and therapeutic uses thereof.</p>
    <heading>BACKGROUND</heading> <p num="p-0004">The central dogma of biology is that genetic information resides in the DNA of a cell and is expressed upon transcription of this information, where after production of the encoded protein follows by the translation machinery of the cell. This view of the flow of genetic information has prompted the predominantly DNA-based approach for interfering with the protein content of a cell. This view is slowly changing and alternatives for interfering at the DNA level are being pursued.</p>
    <p num="p-0005">In higher eukaryotes, the genetic information for proteins in the DNA of the cell is encoded in exons that are separated from each other by intronic sequences. These introns are in some cases very long. The transcription machinery generates a pre-mRNA that contains both exons and introns, while the splicing machinery, often already during the production of the pre-mRNA, generates the actual coding region for the protein by splicing together the exons present in the pre-mRNA.</p>
    <p num="p-0006">Although much is known about the actual processes involved in the generation of an mRNA from a pre-mRNA, much also remains hidden. In the invention it has been shown possible to influence the splicing process such that a different mRNA is produced. The process allows for the predictable and reproducible restructuring of mRNA produced by a splicing machinery. An oligonucleotide capable of hybridizing to pre-mRNA at a location of an exon that is normally included in the mature mRNA can direct the exclusion of the thus targeted exon or a part thereof.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0007">In the invention, means and methods are provided for the design of appropriate complementary oligonucleotides. To this end, the invention provides a method for generating an oligonucleotide comprising determining, from a (predicted) secondary structure of RNA from an exon, a region that assumes a structure that is hybridized to another part of the RNA (closed structure) and a region that is not hybridized in the structure (open structure), and subsequently generating an oligonucleotide, which at least in part is complementary to the closed structure and which at least in part is complementary to the open structure. RNA molecules exhibit strong secondary structures, mostly due to base pairing of complementary or partly complementary stretches within the same RNA. It has long since been thought that structures in the RNA play a role in the function of the RNA. Without being bound by theory, it is believed that the secondary structure of the RNA of an exon plays a role in structuring the splicing process. Through its structure, an exon is recognized as a part that needs to be included in the pre-mRNA. Herein, this signaling function is referred to as an “exon inclusion signal.” A complementary oligonucleotide of the invention is capable of interfering with the structure of the exon and thereby capable of interfering with the exon inclusion signal of the exon. It has been found that many complementary oligonucleotides indeed comprise this capacity, some more efficiently than others. Oligonucleotides of the invention, i.e., those with the overlap directed toward open and closed structures in the native exon RNA, are a selection from all possible oligonucleotides. The selection encompasses oligonucleotides that can efficiently interfere with an exon inclusion signal.</p>
    <p num="p-0008">Without being bound by theory, it is thought that the overlap with an open structure improves the invasion efficiency of the oligonucleotide (i.e., increases the efficiency with which the oligonucleotide can enter the structure), whereas the overlap with the closed structure subsequently increases the efficiency of interfering with the secondary structure of the RNA of the exon, and thereby interfere with the exon inclusion signal. It is found that the length of the partial complementarity to both the closed and the open structure is not extremely restricted. We have observed high efficiencies with oligonucleotides with variable lengths of complementarity in either structure. The term “complementarity” is used herein to refer to a stretch of nucleic acids that can hybridize to another stretch of nucleic acids under physiological conditions. It is thus not absolutely required that all the bases in the region of complementarity are capable of pairing with bases in the opposing strand. For instance, when designing the oligonucleotide, one may want to incorporate, for example, a residue that does not base pair with the base on the complementary strand. Mismatches may to some extent be allowed, if under the circumstances in the cell, the stretch of nucleotides is capable of hybridizing to the complementary part. In a preferred embodiment, a complementary part (either to the open or to the closed structure) comprises at least three and, more preferably, at least four, consecutive nucleotides. The complementary regions are preferably designed such that, when combined, they are specific for the exon in the pre-mRNA. Such specificity may be created with various lengths of complementary regions as this depends on the actual sequences in other mRNA or pre-mRNA in the system. The risk that also one or more other pre-RNA will be able to hybridize to the oligonucleotide decreases with increasing size of the oligonucleotide. It is clear that oligonucleotides comprising mismatches in the region of complementarity but that retain the capacity to hybridize to the targeted region(s) in the pre-mRNA, can be used in the invention. However, preferably at least the complementary parts do not comprise such mismatches as these typically have a higher efficiency and a higher specificity, than oligonucleotides having such mismatches in one or more complementary regions. It is thought that higher hybridization strengths (i.e., increasing number of interactions with the opposing strand), are favorable in increasing the efficiency of the process of interfering with the splicing machinery of the system.</p>
    <p num="p-0009">The secondary structure is best analyzed in the context of the pre-mRNA wherein the exon resides. Such structure may be analyzed in the actual RNA. However, it is currently possible to predict the secondary structure of an RNA molecule (at lowest energy costs) quite well using structure-modeling programs. A non-limiting example of a suitable program is RNA mfold version 3.1 server (Mathews et al., 1999<i>, J. Mol. Biol. </i>288:911-940). A person skilled in the art will be able to predict, with suitable reproducibility, a likely structure of the exon, given the nucleotide sequence. Best predictions are obtained when providing such modeling programs with both the exon and flanking intron sequences. It is typically not necessary to model the structure of the entire pre-mRNA.</p>
    <p num="p-0010">The open and closed structure to which the oligonucleotide is directed, are preferably adjacent to one another. It is thought that in this way, the annealing of the oligonucleotide to the open structure induces opening of the closed structure, and annealing progresses into this closed structure. Through this action, the previously closed structure assumes a different conformation. The different conformation may result in the disruption of the exon inclusion signal. However, when potential (cryptic) splice acceptor and/or donor sequences are present within the targeted exon, occasionally a new exon inclusion signal is generated defining a different (neo) exon, e.g., with a different 5′ end, a different 3′ end, or both. This type of activity is within the scope of the invention as the targeted exon is excluded from the mRNA. The presence of a new exon, containing part of the targeted exon, in the mRNA does not alter the fact that the targeted exon, as such, is excluded. The inclusion of a neo-exon can be seen as a side effect which occurs only occasionally. There are two possibilities when exon skipping is used to restore (part of) an open reading frame that was disrupted as a result of a mutation. One is that the neo-exon is functional in the restoration of the reading frame, whereas in the other case the reading frame is not restored. When selecting oligonucleotides for restoring reading frames by means of exon-skipping, it is, of course, clear that under these conditions, only those oligonucleotides are selected that indeed result in exon-skipping that restores the open reading frame, with or without a neo-exon.</p>
    <p num="p-0011">Pre-mRNA can be subject to various splicing events, for instance, through alternative splicing. Such events may be induced or catalyzed by the environment of a cell or artificial splicing system. Thus, from the same pre-mRNA, several different mRNAs may be produced. The different mRNAs all included exonic sequences, as that is the definition of an exon. However, the fluidity of the mRNA content necessitates a definition of the term “exon” in the invention. An “exon,” according to the invention, is a sequence present in both the pre-mRNA and mRNA produced thereof, wherein the sequence included in the mRNA is, in the pre-mRNA, flanked on one side (first and last exon) or both sides (any exon other than the first and the last exon) by sequences not present in the mRNA. In principle, any mRNA produced from the pre-mRNA qualifies for this definition. However, for the invention, so-called dominant mRNAs are preferred, i.e., mRNA that makes up at least 5% of the mRNA produced from the pre-mRNA under the set conditions. Human immuno-deficiency virus, in particular, uses alternative splicing to an extreme. Some very important protein products are produced from mRNA making up even less than 5% of the total mRNA produced from the virus. The genomic RNA of retroviruses can be seen as pre-mRNA for any spliced product derived from it. As alternative splicing may vary in different cell types, the exons are defined as exons in the context of the splicing conditions used in that system. As a hypothetical example, an mRNA in a muscle cell may contain an exon that is absent in an mRNA produced from the same pre-mRNA in a nerve cell. Similarly, mRNA in a cancer cell may contain an exon not present in mRNA produced from the same mRNA in a normal cell.</p>
    <p num="p-0012">Alternative splicing may occur by splicing from the same pre-mRNA. However, alternative splicing may also occur through a mutation in the pre-mRNA, for instance, generating an additional splice acceptor and/or splice donor sequence. Such alternative splice sequences are often referred to as cryptic splice acceptor/donor sequences. Such cryptic splice sites can result in new exons (neo-exons). Inclusion of neo-exons into produced mRNA can be prevented, at least in part, using a method of the invention. In case a neo-exon is flanked by a cryptic and a “normal” splice donor/acceptor sequence, the neo-exon encompasses the old (paleo) exon. If in this case the original splice donor/acceptor sequence, for which the cryptic splice donor/acceptor has taken its place, is still present in the pre-mRNA, it is possible to enhance the production of mRNA containing the paleo-exon by interfering with the exon-recognition signal of the neo-exon. This interference can be both in the part of the neo-exon corresponding to the paleo-exon, or the additional part of such neo-exons. This type of exon skipping can be seen as splice correction.</p>
    <p num="p-0013">The exon skipping technique can be used for many different purposes. Preferably, however, exon skipping is used for restructuring mRNA that is produced from pre-mRNA exhibiting undesired splicing in a subject. The restructuring may be used to decrease the amount of protein produced by the cell. This is useful when the cell produces a particular undesired protein. In a preferred embodiment, however, restructuring is used to promote the production of a functional protein in a cell, i.e., restructuring leads to the generation of a coding region for a functional protein. The latter embodiment is preferably used to restore an open reading frame that was lost as a result of a mutation. Preferred genes comprise a Duchenne muscular dystrophy gene, a collagen VI alpha 1 gene (COL6A1), a myotubular myopathy 1 gene (MTM1), a dysferlin gene (DYSF), a laminin-alpha 2 gene (LAMA2), an emery-dreyfuss muscular dystrophy gene (EMD), and/or a calpain 3 gene (CAPN3). The invention is further delineated by means of examples drawn from the Duchenne muscular dystrophy gene. Although this gene constitutes a particularly preferred gene in the invention, the invention is not limited to this gene.</p>
    <p num="p-0014">Duchenne muscular dystrophy (DMD) and Becker muscular dystrophy (BMD) are both caused by mutations in the DMD gene that is located on the X chromosome and codes for dystrophin (1-6). DMD has an incidence of 1:3500 newborn males. Patients suffer from progressive muscle weakness, are wheelchair bound before the age of 13 and often die before the third decade of their life (7). The generally milder BMD has an incidence of 1:20,000. BMD patients often remain ambulant for over 40 years and have longer life expectancies when compared to DMD patients (8).</p>
    <p num="p-0015">Dystrophin is an essential component of the dystrophin-glycoprotein complex (DGC), which, amongst others, maintains the membrane stability of muscle fibers (9, 10). Frame-shifting mutations in the DMD gene result in dystrophin deficiency in muscle cells. This is accompanied by reduced levels of other DGC proteins and results in the severe phenotype found in DMD patients (11, 12). Mutations in the DMD gene that keep the reading frame intact, generate shorter, but partly functional dystrophins, associated with the less severe BMD (13, 14).</p>
    <p num="p-0016">Despite extensive efforts, no clinically applicable and effective therapy for DMD patients has yet been developed (15), although a delay of the onset and/or progression of disease manifestations can be achieved by glucocorticoid therapy (16). Promising results have recently been reported by us and others on a genetic therapy aimed at restoring the reading frame of the dystrophin pre-mRNA in cells from the mdx mouse model and DMD patients (17-23). By the targeted skipping of a specific exon, a DMD phenotype can be converted into a milder BMD phenotype. The skipping of an exon can be induced by the binding of antisense oligoribonucleotides (“AONs”), targeting either one or both of the splice sites or exon-internal sequences. Since an exon will only be included in the mRNA when both the splice sites are recognized by the spliceosome complex, splice sites are obvious targets for AONs. This was shown to be successful, albeit with variable efficacy and efficiency (17, 18, 20, 21).</p>
    <p num="p-0017">We hypothesized that targeting exon-internal sequences might increase specificity and reduce interference with the splicing machinery itself. Some exons have weak splice sites and appear to require binding of an SR protein to an exon recognition sequence (ERS) or an exonic splicing enhancer (ESE) to be properly recognized by the splicing machinery (24). SR proteins are a highly conserved family of arginine-/serine-rich, spliceosome-associated phosphoproteins essential for pre-mRNA splicing (50, 51). SR proteins appear to act early in splicing by promoting splice site recognition and spliceosome assembly. SR proteins also play a regulatory role, because they can determine alternative splice site usage in vivo and in vitro. SR proteins appear to be recruited from nuclear “speckles” in which they are concentrated, to sites of transcription in order to spatially coordinate transcription and pre-mRNA splicing within the cell nucleus (49, 52). Disruptive point mutations or AONs that block these sequences have been found to result in exon skipping (19, 22, 24-28). Using exon-internal AONs specific for an ERS-like sequence in exon 46, we were previously able to modulate the splicing pattern in cultured myotubes from two different DMD patients with an exon 45 deletion (19). Following AON treatment, exon 46 was skipped, which resulted in a restored reading frame and the induction of dystrophin synthesis in at least 75% of the cells. We have recently shown that exon skipping can also efficiently be induced in human control muscle cells for 15 different DMD exons using exon-internal AONs (23, unpublished results).</p>
    <p num="p-0018">In contrast to the previous opinion that skipping can only be achieved with weak splice sites or exons containing ERS-like sequences, we have seen that of the exons that were skipped in the invention, most do not have weak splice sites nor do they contain ERS-like sequences. Thus, binding of the AONs to the targeted exon per se is sufficient to cause exon skipping, either by interfering with one or more components of the splicing machinery or by altering the secondary structure of the RNA in such a manner that the splicing machinery no longer recognizes the exon. In a preferred embodiment, the exon to be skipped comprises exons 2, 8, 9, 17, 19, 29, 40-46, 49-53, 55 or 59; more preferably, exons 2, 8, 9, 17, 40, 41, 42, 44, 49-52 or 59. In yet another embodiment, the exon to be skipped comprises exons 2, 29, 40, 41, 42, 43, 44, 45, 46, 49, 50, 51 or 53.</p>
    <p num="p-0019">Any oligonucleotide fulfilling the requirements of the invention may be used to induce exon skipping in the DMD gene. In a preferred embodiment, an oligonucleotide comprises a sequence as depicted as active in exon-skipping in Table 2, or a functional equivalent thereof comprising a similar, preferably the same, hybridization capacity in kind, not necessarily in amount. Preferably, an oligonucleotide comprising a sequence as depicted in Table 2, derived from the exons 2, 40, 41, 42, 43, 44, 45, 49, 50, 51 or 53, demonstrably active in exon skipping.</p>
    <p num="p-0020">Reading frame correction can be achieved by skipping one or two exons flanking a deletion, by skipping in-frame exons containing a nonsense mutation, or by skipping duplicated exons. This results in proteins similar to those found in various BMD patients (2, 29). A survey of the Leiden DMD mutation database (WorldWideWeb.dmd.nl; (30)) evinces that we can thus correct over 75% of DMD-causing mutations (see Table 4). We show the actual therapeutic effect of exon skipping for seven different mutations. In all patient muscle cell cultures, we were able to restore dystrophin synthesis in 75% to 80% of treated cells.</p>
    <p num="p-0021">The complementary oligonucleotide generated through a method of the invention is preferably complementary to a consecutive part of between 16 and 50 nucleotides of the exon RNA. Different types of nucleic acid may be used to generate the oligonucleotide. Preferably, the oligonucleotide comprises RNA, as RNA/RNA hybrids are very stable. Since one of the aims of the exon skipping technique is to direct splicing in subjects, it is preferred that the oligonucleotide RNA comprises a modification providing the RNA with an additional property, for instance, resistance to endonucleases and RNaseH, additional hybridization strength, increased stability (for instance, in a bodily fluid), increased or decreased flexibility, reduced toxicity, increased intracellular transport, and/or tissue-specificity, etc. Preferably, the modification comprises a 2′-O-methyl-phosphorothioate oligoribonucleotide modification.</p>
    <p num="p-0022">With the advent of nucleic acid-mimicking technology, it has become possible to generate molecules that have a similar, preferably the same, hybridization characteristics, in kind, not necessarily in amount, as nucleic acid itself. Such equivalents are, of course, also part of the invention. Examples of such mimics equivalents are peptide nucleic acid, locked nucleic acid and/or a morpholino phosphorodiamidate. Suitable but non-limiting examples of equivalents of oligonucleotides of the invention can be found in C. Wahlestedt et al., Potent and non-toxic antisense oligonucleotides containing locked nucleic acids, <i>Proc. Natl. Acad. Sci. U.S.A. </i>97:5633-8 (2000); A. N. Elayadi and D. R. Corey, Application of PNA and LNA oligomers to chemotherapy, <i>Curr. Opin. Investig. Drugs </i>2:558-61 (2001); H. J. Larsen, T. Bentin, and P. E. Nielsen, Antisense properties of peptide nucleic acid, <i>Biochim. Biophys, Acta </i>1489:159-66 (1999); D. A. Braasch and D. R. Corey, Novel antisense and peptide nucleic acid strategies for controlling gene expression, <i>Biochemistry </i>41:4503-10 (2002); J. Summerton and D. Weller, Morpholino antisense oligomers: design, preparation, and properties, <i>Antisense Nucleic Acid Drug Dev. </i>7:187-95 (1997). Hybrids between one or more of the equivalents among each other and/or together with nucleic acid are, of course, also part of the invention. In a preferred embodiment, an equivalent comprises locked nucleic acid, as locked nucleic acid displays a higher target affinity and reduced toxicity and, therefore, shows a higher efficiency of exon skipping.</p>
    <p num="p-0023">An oligonucleotide of the invention typically does not have to overlap with a splice donor or splice acceptor of the exon.</p>
    <p num="p-0024">An oligonucleotide of the invention, or equivalent thereof, may, of course, be combined with other methods for interfering with the structure of an mRNA. It is, for instance, possible to include in a method at least one other oligonucleotide that is complementary to at least one other exon in the pre-mRNA. This can be used to prevent inclusion of two or more exons of a pre-mRNA in mRNA produced from this pre-mRNA. In a preferred embodiment, at least one other oligonucleotide is an oligonucleotide, or equivalent thereof, generated through a method of the invention. This part of the invention is further referred to as double- or multi-exon skipping. In most cases, double-exon skipping results in the exclusion of only the two targeted (complementary) exons from the pre-mRNA. However, in other cases, it was found that the targeted exons and the entire region in between the exons in the pre-mRNA were not present in the produced mRNA even when other exons (intervening exons) were present in such region. This multi-skipping was notably so for the combination of oligonucleotides derived from the DMD gene, wherein one oligonucleotide for exon 45 and one oligonucleotide for exon 51 was added to a cell transcribing the DMD gene. Such a set-up resulted in mRNA being produced that did not contain exons 45 to 51. Apparently, the structure of the pre-mRNA in the presence of the mentioned oligonucleotides was such that the splicing machinery was stimulated to connect exons 44 and 52 to each other.</p>
    <p num="p-0025">It has now also been found possible to specifically promote the skipping of the intervening exons by providing a linkage between the two complementary oligonucleotides. To this end, provided is a compound capable of hybridizing to at least two exons in a pre-mRNA encoded by a gene, the compound comprising at least two parts, wherein a first part comprises an oligonucleotide having at least eight consecutive nucleotides that are complementary to a first of at least two exons, and wherein a second part comprises an oligonucleotide having at least eight consecutive nucleotides that are complementary to a second exon in the pre-mRNA. The at least two parts are linked in the compound so as to form a single molecule. The linkage may be through any means but is preferably accomplished through a nucleotide linkage. In the latter case, the number of nucleotides that do not contain an overlap between one or the other complementary exon can be zero, but is preferably between 4 to 40 nucleotides. The linking moiety can be any type of moiety capable of linking oligonucleotides. Currently, many different compounds are available that mimic hybridization characteristics of oligonucleotides. Such a compound is also suitable for the invention if such equivalent comprises similar hybridization characteristics in kind, not necessarily in amount. Suitable equivalents were mentioned earlier in this description. One, or preferably more, of the oligonucleotides in the compound are generated by a method for generating an oligonucleotide of the invention. As mentioned, oligonucleotides of the invention do not have to consist of only oligonucleotides that contribute to hybridization to the targeted exon. There may be additional material and/or nucleotides added.</p>
    <p num="p-0026">As mentioned, a preferred gene for restructuring mRNA is the DMD gene. The DMD gene is a large gene, with many different exons. Considering that the gene is located on the X-chromosome, it is mostly boys that are affected, although girls can also be affected by the disease, as they may receive a bad copy of the gene from both parents, or are suffering from a particularly biased inactivation of the functional allele due to a particularly biased X chromosome inactivation in their muscle cells. The protein is encoded by a plurality of exons (79) over a range of at least 2.6 Mb. Defects may occur in any part of the DMD gene. Skipping of a particular exon or particular exons can very often result in a restructured mRNA that encodes a shorter than normal, but at least partially functional, dystrophin protein. A practical problem in the development of a medicament based on exon-skipping technology is the plurality of mutations that may result in a deficiency in functional dystrophin protein in the cell. Despite the fact that already multiple different mutations can be corrected for by the skipping of a single exon, this plurality of mutations requires the generation of a large number of different pharmaceuticals; as for different mutations, different exons need to be skipped.</p>
    <p num="p-0027">An advantage of a compound capable of inducing skipping of two or more exons, is that more than one exon can be skipped with a single pharmaceutical. This property is practical and very useful in that only a limited number of pharmaceuticals need to be generated for treating many different Duchenne or Becker mutations. Another option now open to the person skilled in the art is to select particularly functional restructured dystrophin proteins and produce compounds capable of generating these preferred dystrophin proteins. Such preferred end results are further referred to as mild phenotype dystrophins. The structure of the normal dystrophin protein can be schematically represented as two endpoints having structural function (the beads), which are connected to each other by a long, at least partly flexible, rod. This rod is shortened in many Becker patients.</p>
    <p num="p-0028">This led the field to the conclusion that not so much the length of the rod, but the presence of a rod and the composition thereof (with respect to particular hinge regions in the protein), is crucial to the function per se of the dystrophin protein. Though the size of the rod may have an impact on the amount of functionality of the resulting (Becker) protein, there are many notable exceptions. These exceptions will be detailed below. There are especially benign mutations that can have a very short rod. It was noted by the inventors that many more different types of Becker patients should have been detected in the patient population. However, some types of shortened dystrophin proteins, that according to this hypothesis should have a Becker phenotype, are not detected in human population. For some of these “theoretical” Becker forms, this could just be a matter of chance. However, in the invention, it has been found that at least some of these “potential” Becker patients have such a benign phenotype that subjects having these types of mutations do not present themselves to a doctor or are not diagnosed as suffering from Becker's disease. With a compound of the invention, it is possible to restructure DMD pre-mRNA of many different Duchenne and even Becker patients such that a mild phenotype dystrophin is generated after translation of the restructured mRNA.</p>
    <p num="p-0029">Thus provided is a particularly preferred compound, wherein the parts of the compounds at least comprise a first part comprising an oligonucleotide or equivalent thereof, complementary to exon 17, and a second part comprising an oligonucleotide or equivalent thereof, complementary to exon 48. The resulting restructured mRNA encodes an in-frame shortened dystrophin protein, lacking all exons from 17 to 48. This shortened dystrophin protein mimics a mild phenotype dystrophin as mentioned above. The compound (referred to as the 17-48 compound) should, according to current databases, be able to deal with as much as 20% of the patients having a DMD mutation currently characterized. Another preferred compound is the 45-55 compound. This compound should, according to the same calculations, be able to deal with 38% of the patients having a DMD mutation thus far characterized.</p>
    <p num="p-0030">In yet another embodiment, the compound comprises a 42-55 compound or a 49-59 compound capable of dealing with, respectively, 65% and 18% of the currently characterized DMD patients. Preferred are a 45-49 compound and a 45-51 compound, preferably in the form as disclosed in the experimental part, having the potential to treat, respectively, 4% and 8% of the DMD patients characterized thus far.</p>
    <p num="p-0031">Another aspect of the invention is a compound capable of hybridizing to one exon in a pre-mRNA encoded by a gene, the compound comprising at least two parts, wherein a first part comprises an oligonucleotide of which at least a part of the oligonucleotide is complementary to the closed structure and wherein a second part comprises an oligonucleotide of which at least part is complementary to the open structure. The open and closed structures are, of course, determined from a secondary structure of RNA from the exon. A compound having two distinguishable parts complementary to a single exon may comprise an oligonucleotide, or equivalent thereof, or combination thereof, as mentioned herein in the method for generating the oligonucleotide.</p>
    <p num="p-0032">A transcription system containing a splicing system can be generated in vitro. The art has suitable systems available. However, the need for mRNA restructuring is, of course, predominantly felt for the manipulation of living cells, preferably, cells in which a desired effect can be achieved through the restructuring of an mRNA. Preferred mRNAs that are restructured are listed hereinabove. Preferably, genes active in muscle cells are used in the invention. Muscle cells (e.g., myotubes) are multinucleated cells in which many, but not all, muscle cell-specific genes are transcribed via long pre-mRNA. Such long pre-mRNAs are preferred for the invention, as restructuring of mRNAs produced from such long mRNAs is particularly efficient. It is thought, though it need not necessarily be so, that the relatively long time needed to generate the full pre-mRNA aids the efficiency of restructuring using a method or means of the invention, as more time is allowed for the process to proceed. The preferred group of genes of which the mRNA is preferably restructured in a method of the invention comprises: COL6A1 causing Bethlem myopathy, MTM1 causing myotubular myopathy, DYSF (dysferlin causing Miyoshi myopathy), and LGMD, LAMA2 (laminin alpha 2) causing Merosin-deficient muscular dystrophy, EMD (emerin) causing Emery-Dreyfuss muscular dystrophy, the DMD gene causing Duchenne muscular dystrophy and Becker muscular dystrophy, and CAPN3 (calpain) causing LGMD2A. Any cell may be used, however, as mentioned, a preferred cell is a cell derived from a DMD patient. Cells can be manipulated in vitro, i.e., outside the subject's body. However, ideally, the cells are provided with a restructuring capacity in vivo. Suitable means for providing cells with an oligonucleotide, equivalent or compound of the invention are present in the art. Improvements in these techniques are anticipated considering the progress that has already thus far been achieved. Such future improvements may, of course, be incorporated to achieve the mentioned effect on restructuring of mRNA using a method of the invention. At present, suitable means for delivering an oligonucleotide, equivalent or compound of the invention to a cell in vivo comprises polyethylenimine (PEI) or synthetic amphiphils (SAINT-18) suitable for nucleic acid transfections. The amphiphils show increased delivery and reduced toxicity, also when used for in vivo delivery. Preferably, compounds mentioned in J. {hacek over (S)}misterová, A. Wagenaar, M. C. A. Stuart, E. Polushkin, G. ten Brinke, R. Hulst, J. B. F. N. Engberts, and D. Hoekstra, “Molecular shape of the Cationic Lipid Controls the Structure of the Cationic Lipid/Dioleylphosphatidylethanolamine-DNA Complexes and the Efficiency of Gene Delivery,” <i>J. Biol. Chem. </i>2001, 276:47615. The synthetic amphiphils preferably used are based upon the easily synthetically available “long-tailed” pyridinium head group-based materials. Within the large group of amphiphils synthesized, several show a remarkable transfection potential combined with a low toxicity in terms of overall cell survival. The ease of structural modification can be used to allow further modifications and the analysis of their further (in vivo) nucleic acid transfer characteristics and toxicity.</p>
    <p num="p-0033">An oligonucleotide, equivalent thereof, or a compound according to the invention may be used for, at least in part, altering recognition of the exon in a pre-mRNA. In this embodiment, the splicing machinery is at least in part prevented from linking the exon boundaries to the mRNA. The oligonucleotide, equivalent or compound of the invention is at least in part capable of altering exon recognition in a pre-mRNA. This use is thus also provided in the invention. The prevention of inclusion of a targeted exon in an mRNA is also provided as a use for, at least in part, stimulating exon skipping in a pre-mRNA. As mentioned above, the targeted exon is not included in the resulting mRNA. However, part of the exon (a neo-exon) may occasionally be retained in the produced mRNA. This sometimes occurs when the targeted exon contains a potential splice acceptor and/or splice donor sequence. In this embodiment, the splicing machinery is redirected to utilize a previously unused (or underused) splice acceptor/donor sequence, thereby creating a new exon (neo-exon). The neo-exon may have one end in common with the paleo-exon, although this does not always have to be the case. Thus, in one aspect, an oligonucleotide, equivalent or compound of the invention is used for altering the efficiency with which a splice donor or splice acceptor is used by a splicing machinery.</p>
    <p num="p-0034">In view of the foregoing, further provided is the use of an oligonucleotide, an equivalent thereof, or a compound of the invention for the preparation of a medicament. Further provided is a pharmaceutical preparation comprising an oligonucleotide, equivalent thereof or a compound according to the invention. An oligonucleotide, an equivalent thereof or a compound of the invention can be used for the preparation of a medicament for the treatment of an inherited disease. Similarly provided is a method for altering the efficiency with which an exon in a pre-mRNA is recognized by a splicing machinery, the pre-mRNA being encoded by a gene comprising at least two exons and at least one intron, the method comprising providing a transcription system comprising the splicing machinery and the gene, with an oligonucleotide, equivalent thereof or a compound according to the invention, wherein the oligonucleotide, equivalent thereof or compound is capable of hybridizing to at least one of the exons, and allowing for transcription and splicing to occur in the transcription system. The gene may comprise at least three exons.</p>
    <p num="p-0035">An oligonucleotide of the invention may be provided to a cell in the form of an expression vector, wherein the expression vector encodes a transcript comprising the oligonucleotide. The expression vector is preferably introduced into the cell via a gene delivery vehicle. A delivery vehicle may be a viral vector such as an adenoviral vector and, more preferably, an adeno-associated virus vector. Also provided are such expression vectors and delivery vehicles. It is within the skill of the artisan to design suitable transcripts. Preferred for the invention are PolIII-driven transcripts, preferably in the form of a fusion transcript with a U1 or U7 transcript. Such fusions may be generated as described in references 53 and 54.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p num="p-0036"> <figref idrefs="DRAWINGS">FIG. 1</figref> shows RT-PCR and sequence analysis of dystrophin mRNA fragments of the AON-treated DMD patient myotube cultures, focusing on the regions encompassing the exons targeted for skipping. Shorter novel transcripts were observed when compared to the untransfected myotube cultures (NT). Sequence analysis confirmed the precise skipping of the targeted exons. An alternatively spliced product, detected for patient 50685.1 (Panel C) was sequenced and found to be derived from activation of a cryptic splice site in exon 51. Shorter fragments, detected in untransfected myotube cultures from DL 363.2 (Panel B), DL 589.2 (Panel D) and 53914.1 (Panel E), were sequenced and found to be the result of the spontaneous skipping of exons 44, 50 and 53, respectively. Note that in some analyses, additional fragments, slightly shorter than the wild-type products, were present. This was due to heteroduplex formation. 100 bp: size marker, —RT-PCR: negative control.</p>
      <p num="p-0037"> <figref idrefs="DRAWINGS">FIG. 2</figref> illustrates immuno-histochemical analysis of the AON-treated myotube cultures from the six different DMD patients. Cells were stained for myosin to identify fully differentiated myotubes (not shown). Monoclonal antibodies MANDYS1 (middle panel) and Dys2 (right panel) were used to detect dystrophin one to four days post-transfection. No dystrophin signals could be detected in untreated cells stained with MANDYS1 (left panel) nor Dys2 (not shown), whereas clear, mainly cytoplasmatic dystrophin signals could be detected for each patient upon the induced exon skipping. In patients DL 363.2 (row B), DL 589.2 (row D) and 53914.1 (row E), dystrophin membrane signals could be observed. Note that membrane signals were more often found for Dys2, which recognizes the full-length dystrophin. MANDYS1 recognizes an internal part of dystrophin and is more prone to generate cytoplasmatic signals, since it also detects dystrophin in the first stages of synthesis. Magnification 63×.</p>
      <p num="p-0038"> <figref idrefs="DRAWINGS">FIG. 3</figref> is Western blot analysis of the AON-treated myotube cultures. Monoclonal antibody DY4 was used to detect dystrophin. Protein extracts isolated from human control myotube cultures (HC) were used as a positive control (Panels C and F). To avoid over-exposure, this sample was 1 to 10 diluted. To demonstrate equal loading of protein samples, blots were additionally stained with an antibody against myosin. No, or, as a result of spontaneous exon skipping, very low (Panels B and C) levels of dystrophin were detected in non-transfected myotube cultures (NT). Clear dystrophin signals were observed in AON-treated myotube cultures for each of the patients. For 50685.1 and DL 363.2, a time-course experiment was performed. Dystrophin could be detected 16 hours post-transfection and was found at increasing levels at 24 hours and 48 hours post-transfection for 50685.1 (Panel D). For DL 363.2, dystrophin could be detected in increasing levels up to seven days post-transfection (Panel B). For patients DL 515.2 (Panel A), DL 363.2 (Panel B) and DL 589.2 (Panel E), the detected dystrophin was significantly shorter than the control dystrophin. This is due to the size of the deletions in these patients.</p>
      <p num="p-0039"> <figref idrefs="DRAWINGS">FIG. 4</figref> shows immuno-histochemical analysis of four DGC proteins from treated myotube cultures from patient DL 363.2. Cells were stained for myosin to identify sufficiently differentiated myotubes (not shown). Monoclonal antibodies NCL-a-SARC, NCL-b-SARC, NCL-g-SARC and NCL-b-DG were used to detect α-sarcoglycan, β-sarcoglycan, γ-sarcoglycan and β-dystroglycan, respectively. These proteins were detected in reduced percentages (˜40%) in untreated myotubes, and were mainly located in the cytoplasm (row A). Following AON treatment, however, α-sarcoglycan was detected in 70%, β-sarcoglycan was detected in 90%, γ-sarcoglycan was detected in 90% and β-dystroglycan was detected in 80% of the myotubes, and the proteins were mostly membrane-bound (row B). Magnification 63×.</p>
      <p num="p-0040"> <figref idrefs="DRAWINGS">FIG. 5</figref> is RT-PCR analysis of human dystrophin mRNA in the regions encompassing the exons targeted for skipping. Shorter novel transcript fragments were observed following transfection with the different AONs when compared to non-transfected myotube cultures (NT). Sequence analysis (not shown) confirmed the skipping of the targeted exons, as indicated by the labels adjacent to the images. Alternatively spliced products, detected in the regions around exon 2 (b), exon 29 (c), and exon 51 (h), were sequenced and found to be derived from either co-skipping of adjacent exons or usage of a cryptic splice site. No specific (RT-) PCR products were obtained. In some analyses, additional fragments, lightly shorter than the wild-type products, were present. This was due to heteroduplex formation.</p>
      <p num="p-0041"> <figref idrefs="DRAWINGS">FIG. 6</figref> illustrates double-exon skipping in DMD patient DL90.3 carrying a nonsense mutation in the out-of-frame exon 43. RT-PCR analysis of dystrophin mRNA fragments of AON-treated myotubes from this patient showed a shorter, novel transcript not present in untransfected myotubes (NT). Sequence analysis confirmed the precise skipping of the targeted exons 43 and 44. Besides this double-skip, we also detected a single-exon 44 skip. Note that the additional fragment, slightly shorter than the wild-type product, is due to heteroduplex formation. 100 bp: size marker, −RT-PCR: negative control.</p>
      <p num="p-0042"> <figref idrefs="DRAWINGS">FIG. 7</figref> illustrates double- and multi-exon skipping in human control myotubes (KM 109), DMD patient DL 470.2, carrying a deletion of exons 46 to 50, and DMD patient 50685.1, carrying a deletion of exons 48 to 50. Panel A is RT-PCR analysis of dystrophin mRNA fragments in the myotube cultures treated with either a mixture of h45AON5 and h51AON2 (mix) or with a U-linked combination of AONs (U: h45AON5 linked to h51AON2 by ten uracil nucleotides). In all samples treated with either the mix of AONs or the U-linker AON, a shorter transcript fragment was detected that contained exon 44 spliced to exon 52, and that was not present in untreated myotubes (NT). This novel, in-frame transcript arose from double-exon skipping in patient DL 470.2 (the targeted exons 45 and 51 are directly flanking the deletion), but from multi-exon skipping in both the human control and patient 50685.1. In the treated patient myotube cultures, additional shorter fragments were observed due to single-exon 45 and single-exon 51 skipping. Note that in some lanes, other fragments, slightly shorter than the wild-type products, were present. This was due to heteroduplex formation. 100 bp: size marker, −RT-PCR: negative control. Panel B shows that all fragments were quantified using the DNA 7500 LABCHIP® and the BIOANALYZER™ (Agilent). The percentage of double- or multi-exon 45 to 51 skipping was determined by the ratio of this fragment to the total of transcript fragments. The U-combined AON seems less efficient in DL 470.2, but more efficient in KM 109 and 50685.1, when compared to the mixture of AONs.</p>
    </description-of-drawings> <heading>DETAILED DESCRIPTION</heading> <heading>Examples</heading> <heading>Example 1</heading> <heading>Results</heading> <p num="p-0043">This study includes six DMD patients affected by different mutations (Table 1). Patient DL 515.2 carries an exon 45-50 deletion; hence exon 51 skipping would be frame correcting. Patient DL 363.2 has a deletion of exon 45-54; the reading frame for this patient would be corrected by an exon 44 skip. For patient 50685.1, who is affected by an exon 48-50 deletion, reading frame correction requires an exon 51 skip. Patient DL 589.2 has an exon 51-55 deletion; the reading frame would be corrected by an exon 50 skip. Patient 53914.1 carries a single-exon 52 deletion. Notably, in this case, both the skipping of exon 51 or exon 53 would be frame correcting. Finally, patient 50423.1 has a deletion of a single base pair in exon 49, at position 7389 on cDNA level, resulting in a frame-shift and a premature stop codon in exon 49. Since exon 49 is an in-frame exon, skipping of this exon would correct the reading frame for this patient.</p>
    <p num="p-0044">We have previously identified AONs with which the skipping of the mentioned target exons 44, 49, 50, 51 and 53 can be induced at concentrations of 1 μM (23). In subsequent dose-response experiments, however, we have obtained substantial skipping efficiencies with lower concentrations of 500 nM or 200 nM, and even 100 nM for most AONs (data not shown). This had the extra advantageous effect of lower doses of PEI required for transfection, which significantly reduced the levels of cytotoxicity as found in our earlier transfection experiments. Myotube cultures from the six DMD patients were transfected with the relevant AONs. On average, 70% to 90% of cells showed specific nuclear uptake of fluorescent AONs. RNA was isolated 24 hours post-transfection and analyzed by RT-PCR (<figref idrefs="DRAWINGS">FIG. 1</figref>). In all patients, the targeted exons were skipped at high efficiencies, and precisely at the exon boundaries, as confirmed by sequence analysis of the novel shorter transcripts (<figref idrefs="DRAWINGS">FIG. 1</figref>). For patient 50685.1, an additional transcript fragment was found (<figref idrefs="DRAWINGS">FIG. 1</figref>, Panel C). Sequence analysis showed that this was generated by the activation of a cryptic splice site in exon 51. This was previously also observed in human control cells treated with the same AON (23). Remarkably, low levels of spontaneous exon skipping were observed in untreated cells derived from patients DL 363.2 (exon 44 skip), DL 589.2 (exon 50 skip), and 53914.1 (exon 53 skip). RT-PCR analysis on several larger areas of the DMD gene transcript did not reveal additional, unexpected, aberrant splicing patterns induced by the AON-treatment.</p>
    <p num="p-0045">The resulting in-frame transcripts should restore dystrophin synthesis. Indeed, immuno-histochemical analysis of transfected myotube cultures detected dystrophin in the majority of myotubes for each patient (<figref idrefs="DRAWINGS">FIG. 2</figref>). The therapeutic efficiency was determined by double staining, using antibodies against myosin, to identify sufficiently differentiated myotubes and dystrophin. On average, 75% to 80% of myosin-positive myotubes showed dystrophin expression. We observed clear membrane-bound dystrophin for patients DL 363.2, DL 589.2 and 53914.1 two days post-transfection (<figref idrefs="DRAWINGS">FIG. 2</figref>, rows B, D, and E). The presence of dystrophin was confirmed for each patient by Western blot analysis (<figref idrefs="DRAWINGS">FIG. 3</figref>). For patients 50685.1 and DL 363.2, we performed time course experiments, which indicated that dystrophin can be detected as soon as 16 hours post-transfection (<figref idrefs="DRAWINGS">FIG. 3</figref>, Panel D) and at increasing levels up to seven days post-transfection (<figref idrefs="DRAWINGS">FIG. 3</figref>, Panel B). The dystrophin proteins from patients DL515.2, DL 363.2 and DL 589.2 are significantly shorter than the human control, which is due to the size of the deletion.</p>
    <p num="p-0046">For one patient, DL 363.2, we also assessed whether the induction of the dystrophin synthesis resulted in the restoration of the DGC (<figref idrefs="DRAWINGS">FIG. 4</figref>). Prior to AON treatment, we found reduced, mainly cytoplasmatic alpha, beta, gamma sarcoglycan and beta-dystroglycan signals (30%, 30%, 40% and 80%, respectively) (<figref idrefs="DRAWINGS">FIG. 4</figref>, row A). Following AON transfection, increased levels of mainly membrane-bound alpha-, beta- and gamma-sarcoglycans and beta-dystroglycan were detected in 70%, 90%, 90% and 80% of the treated myotube cultures, respectively (<figref idrefs="DRAWINGS">FIG. 4</figref>, row B).</p>
    <p num="h-0010">Discussion</p>
    <p num="p-0047">The reading frame correction strategy for DMD patients is aimed at antisense-induced, targeted exon skipping. This would convert a severe DMD phenotype into a mostly milder BMD phenotype. We determined the broad applicability in six patients, carrying five different deletions and a point mutation in an exon 49 (Table 1). Following AON treatment, we show for each patient the precise skipping of the targeted exon on the RNA level, and a dystrophin protein in 75% to 80% of the treated myotubes. In particular, we here report, for the first time, the application of a single AON treatment (i.e., the induced skipping of exon 51) to correct the reading frame for several different deletions.</p>
    <p num="p-0048">Interestingly, the levels of exon skipping observed in the DMD patient cells are significantly higher than those previously obtained in human control cells (23). Typically, the novel skip transcript is the major product. This can be explained by the action of the nonsense-mediated decay (NMD) process (25, 32). In control cells, the skip of an out-of-frame exon results in an out-of-frame transcript, which will be susceptible to NMD. In patient cells, the skip of a target exon results in an in-frame transcript that would be resistant to NMD and thus more stable than the out-of-frame transcript originally present.</p>
    <p num="p-0049">For three of the patients (DL 363.2, DL 589.2 and 53914.1), we detected low levels of spontaneous skipping of exons 44, 50 and 53 in untreated cells. This phenomenon has previously also been described for so-called revertant muscle fibers (33-35). These dystrophin-positive fibers are present in low amounts (2% to 10%) in DMD muscles and are considered to be the result of secondary somatic mutations and/or alternative splicing that restore the reading frame. The existence of revertant fibers has been suggested to correlate with the severity of the disease (36, 37).</p>
    <p num="p-0050">Restoration of the dystrophin synthesis could be detected as soon as 16 hours post-transfection. At two days post-transfection, dystrophin was detected at the membrane, indicating that these novel BMD-like proteins are likely in part functional. Furthermore, we show that restoration of the dystrophin synthesis appears to re-establish the formation of the dystrophin-glycoprotein complex.</p>
    <p num="p-0051">In patients DL 363.2 and DL 589.2, the targeted exon skipping enlarged the deletions to span exons 44-54 and 50-55, respectively. So far, these deletions have not been reported in DMD or BMD patients. This means that they either do not exist or generate a very mild phenotype not diagnosed as BMD. Considering both the large variety of BMD mutations and the markedly lower incidence of BMD observed, we consider the last explanation more plausible than the first. The out-of-frame deletions from patients DL 515.1, 50685.1 and 50423.1 were converted into in-frame deletions as observed in BMD patients carrying deletions of exon 45-51, exon 48-51 and exon 49 (30, 38-40); noteworthy, the exon 48-51 deletion has even been described in an asymptomatic person (40). On the other hand, however, there are also DMD patients carrying such deletions (38, 41-43). Since most of these theoretical in-frame deletions have been detected on the DNA level only, we hypothesize that the dystrophin deficiency in these DMD patients may be caused by additional aberrant splicing patterns on the RNA level, resulting in an out-of-frame transcript.</p>
    <p num="p-0052">It is feasible to correct over 75% of the mutations reported in the Leiden DMD-mutation database (30). Our results indicate that antisense-induced reading frame correction will be a promising therapeutic approach for many DMD patients carrying different deletions and point mutations. Towards the establishment of clinical trials, we are currently investigating and optimizing delivery methods in muscle tissue of mice in vivo.</p>
    <p num="h-0011">Material and Methods</p>
    <p num="h-0012">AONs and Primers</p>
    <p num="p-0053">The AONs applied (Table 1) were previously described (23). They contain a 5′ fluorescein group (6-FAM), a full-length phosphorothioate backbone and 2′-O-methyl modified ribose molecules (Eurogentec, BE). To avoid interference with the fluorescent signals of the secondary antibodies, unlabelled AONs were used for immuno-histochemical analyses. Primers for RT-PCR analysis (sequences available upon request) were synthesized by Eurogentec (BE) or by Isogen Bioscience BV (NL).</p>
    <p num="h-0013">Myogenic Cell Cultures and AON Transfections</p>
    <p num="p-0054">Primary human myoblasts from patients DL 515.2 (deletion exon 45-50), DL 363.2 (deletion exon 45-54), 50685.1 (deletion exon 48-50), DL 589.2 (deletion exon 51-55) and 53914.1 (deletion exon 52) were isolated from a muscle biopsy and cultured as described (44). Cultures were seeded in collagen pre-coated flasks and plates (Vitrogen 100, Cohesion). Myotubes were obtained from confluent myoblast cultures, following 7 to 14 days of serum deprivation. They were subsequently transfected using polyethylenimine (PEI) for three hours in low-serum medium, according to the manufacturer's instructions (ExGen500; MBI Fermentas), and with 3.5 μl PEI applied per μg of transfected AON. For RT-PCR analysis, concentrations of 500 nM AON were used. At this concentration, the highest skipping levels can be obtained, albeit with moderate levels of cell death. Because more viable myotubes are required for immunohistochemical and western blot analysis, concentrations of 200 nM were applied.</p>
    <p num="p-0055">For patient 50423.1, who carries a point mutation in exon 49, only fibroblasts were available. Following infection (MOI 50-100) with an adenoviral vector containing the MyoD gene (Ad50MyoD), the fibroblasts were forced into myogenesis according to protocols described previously (45-47). Two hours post-infection, the medium was replaced by low-serum medium, and cells were incubated for eight to ten days until myotubes were formed. Transfection conditions were identical to those described above.</p>
    <p num="h-0014">RNA Isolation and RT-PCR Analysis</p>
    <p num="p-0056">At 24 hours post-transfection, total RNA was isolated from the myotube cultures (RNA-Bee RNA isolation solvent, Campro Scientific, NL). 300 ng of total RNA was used for RT-PCR analysis using C.therm polymerase (Roche Diagnostics, NL) in a 20 μl reaction at 60° C. for 30 minutes, primed with different DMD gene-specific reverse primers (Table 1). Primary PCRs were performed by 20 cycles of 94° C. (40 seconds), 60° C. (40 seconds) and 72° C. (60 seconds). One μl of these reactions was then reamplified in nested PCRs by 32 cycles of 94° C. (40 seconds), 60° C. (40 seconds) and 72° C. (60 seconds). PCR products were analyzed on 1.5% or 2% agarose gels. Noteworthy, no evidence for a significant preference for the amplification of shorter fragments was obtained in PCR analyses on a defined series of mixtures of known quantities of the normal and shorter transcript fragments (data not shown).</p>
    <p num="h-0015">Sequence Analysis</p>
    <p num="p-0057">RT-PCR products were isolated from agarose gels using the QIAQUICK® Gel Extraction Kit (Qiagen). Direct DNA sequencing was carried out by the Leiden Genome Technology Center (LGTC) using the BigDye Terminator Cycle Sequencing Ready Reaction kit (PE Applied Biosystems) and analyzed on an ABI 3700 Sequencer (PE Applied Biosystems).</p>
    <p num="h-0016">Protein Isolation and Western Blot Analysis</p>
    <p num="p-0058">Protein extracts were isolated from treated myotube cultures (25 cm<sup>2 </sup>flasks), using 150 μl of treatment buffer (75 mM Tris-HCl pH 6.8, 15% SDS, 5% b-mercaptoethanol, 2% glycerol, 0.001% bromophenol blue), at two to four days post-transfection, depending on the survival rate of the myotubes. For the time course experiments, protein extracts were isolated 4 hours, 8 hours, 16 hours, 24 hours and 48 hours post-transfection (for patient 50685.1) or at 2 days, 4 days and 7 days post-transfection (for patient DL 363.2).</p>
    <p num="p-0059">Polyacrylamide gel electrophoresis and Western blotting were performed as described by Anderson et al., with some minor adjustments (48). Briefly, samples (75 μl) were run overnight at 4° C. on a 4% to 7% polyacrylamide gradient gel. Gels were blotted to nitrocellulose for five to six hours at 4° C. Blots were blocked for one hour with 5% non-fat dried milk in TBST buffer (10 mM Tris-HCl, 0.15 M NaCl, 0.5% Tween 20, pH 8), followed by an overnight incubation with NCL-DYS2 (which recognizes dystrophin) diluted 1:50. HRP-conjugated anti-mouse (Santa Cruz) diluted 1:10,000 was used as a secondary antibody. Immuno-reactive bands were visualized using Lumi-Lightplus Western Blotting Substrate and scanned with a Lumi-Imager (Roche Diagnostics, NL).</p>
    <p num="h-0017">Immunohistochemical Analysis</p>
    <p num="p-0060">Treated myotube cultures were fixed in −20° C. methanol at one to four days post-transfection, depending of the survival rate of the myotubes. Prior to reaction with the different antibodies, the cells were incubated for one hour in a blocking solution containing 5% horse serum (Gibco BRL) and 0.05% Tween-20 (Sigma) in PBS (Gibco BRL). All antibodies used were diluted in this blocking solution. The following antibodies were applied: desmin polyclonal antibody (ICN Biomedicals) diluted 1:100, myosin monoclonal antibody diluted 1:100 (MF20; Developmental Studies Hybridoma Bank, University of Iowa), myosin polyclonal antibody L53 diluted 1:100 (a gift from Dr. M. van den Hoff, AMC, NL), MANDYS1 (a gift from Dr. G. Morris, North East Wales Institute, UK) diluted 1:10 and NCL-DYS2 (Novacastra Laboratories Ltd) diluted 1:10 to detect dystrophin, NCL-a-SARC (Novacastra Laboratories Ltd) diluted 1:75, NCL-b-SARC (Novacastra Laboratories Ltd) diluted 1:50, NCL-g-SARC (Novacastra Laboratories Ltd) diluted 1:50 and NCL-b-DG (Novacastra Laboratories Ltd) diluted 1:50 to detect α-sarcoglycan, β-sarcoglycan, γ-sarcoglycan and β-dystroglycan, respectively. After one hour incubation, slides were rinsed and incubated for one hour with the secondary antibodies Alexa Fluor 594 goat anti-rabbit conjugate diluted 1:1000 or Alexa Fluor 488 goat anti-mouse conjugate diluted 1:250 (Molecular Probes Inc). The slides were analyzed using a Leica confocal microscope equipped with epifluorescence optics. Digital images were captured using a CCD camera (Photometrics).</p>
    <heading>Example 2</heading> <heading>Materials and methods</heading> <p num="h-0020">AONs and Primers</p>
    <p num="p-0061">A series of AONs (two per exon, see Table 2) was designed to bind to exon-internal target sequences showing a relatively high purine-content and, preferably, an open secondary pre-mRNA structure (at 37° C.), as predicted by the RNA mfold version 3.1 server [22]. The AONs varied in length between 15 and 24 bp, with G/C contents between 26 and 67%. They were synthesized with the following chemical modifications: a 5′-fluorescein group (6-FAM), a full-length phosphorothioate backbone and 2′-O-methyl-modified ribose molecules (Eurogentec, BE). The primers used for reverse transcription-polymerase chain reaction (RT-PCR) analysis (Table 3) were synthesized by Eurogentec (BE) or by Isogen Bioscience BV (NL).</p>
    <p num="h-0021">In Vitro Experiments</p>
    <p num="p-0062">Primary human myoblasts were isolated from a muscle biopsy from a non-affected individual (KM108) by enzymatic dissociation. Briefly, the tissue was homogenized in a solution containing 5 mg/ml collagenase type VIII (Sigma), 5 mg/ml bovine albumin fraction V (Sigma), 1% trypsin (Gibco BRL) in PBS (Gibco BRL). Following serial incubation steps of 15 minutes at 37° C., suspensions containing the dissociated cells were added to, and pooled in, an equal volume of proliferation medium (Nut.Mix F-10 (HAM) with GlutaMax-1, Gibco BRL) supplemented with 20% fetal bovine serum (Gibco BRL) and 1% penicillin/streptomycin solution (Gibco BRL). After centrifugation, the cells were plated and further cultured in proliferation medium, using flasks that were pre-coated with purified bovine dermal collagen (Vitrogen 100; Cohesion).</p>
    <p num="p-0063">The myogenic cell content of the culture, as determined by the percentage of desmin-positive cells in an immunohistochemical assay, was improved to 58% by repetitive pre-plating [23]. Myotubes were obtained from confluent myoblast cultures following 7 to 14 days of incubation in low-serum medium (DMEM (Gibco BRL), supplemented with 2% GlutaMax-1, 1% glucose, 2% fetal bovine serum and 1% penicillinlstreptomycin solution). For transfection of the myotube cultures, we used polyethylenimine (PEI; ExGen 500) according to the manufacturer's instructions (MBI Fermentas). The cultures were transfected for three hours in low-serum medium with 1 mM of each AON linked to PEI at a ratio-equivalent of 3.5.</p>
    <p num="p-0064">RNA isolation and RT-PCR analysis at 24 hours post-transfection, total RNA was isolated from the myotube cultures using RNAzol B according to the manufacturer's instructions (Campro Scientific, NL). One microgram of RNA was then used for RT-PCR analysis using C. therm. polymerase (Roche Diagnostics) in a 20 μl reaction at 60° C. for 30 minutes, primed with different DMD gene-specific reverse (RT) primers (Table 3). Primary PCRs were carried out with outer primer sets (see Table 3), for 20 cycles of 94° C. (40 seconds), 60° C. (40 seconds), and 72° C. (90 seconds). One microliter of this reaction was then reamplified in nested PCRs using the appropriate primer combinations (Table 3) for 32 cycles of 94° C. (40 seconds), 60° C. (40 seconds), and 72° C. (60 seconds). PCR products were analyzed on 1.5 or 2% agarose gels.</p>
    <p num="p-0065">Sequence analysis RT-PCR products were isolated from agarose gels using the QIAQUICK® Gel Extraction kit (Qiagen). Direct DNA sequencing was carried out by the Leiden Genome Technology Center (LGTC) using the BIGDYE® Terminator Cycle Sequencing Ready Reaction kit (PE Applied Biosystems), and analyzed on an ABI 3700 Sequencer (PE Applied Biosystems).</p>
    <p num="h-0022">Results</p>
    <p num="h-0023">In Vitro Exon Skipping</p>
    <p num="p-0066">AONs were empirically analyzed for the induction of exon skipping following transfection into human control myotube cultures, using the cationic polymer polyethylenimine (PEI). As determined by the nuclear uptake of the fluorescent AONs, average transfection efficiencies of 60-80% were obtained. At 24 hours post-transfection, transcripts were analyzed by RT-PCR using different primer combinations encompassing the targeted exons (Table 3). Of the 30 AONs tested, a total of 21 (70%) reproducibly generated shorter transcript fragments with sizes corresponding to the specific skipping of the targeted exons (<figref idrefs="DRAWINGS">FIG. 5</figref> and Table 2). In fact, as confirmed by sequence analysis of the shorter transcripts (data not shown), we could induce the specific skipping of 13 out of the 15 exons targeted (five out of the seven in-frame exons, and eight out of the eight out-of-frame exons). No skipping of exons 47 and 48 was detected (<figref idrefs="DRAWINGS">FIG. 5</figref>, Panels e and g).</p>
    <p num="p-0067">In the specific transcript regions that were screened in these experiments, we observed in the non-transfected control myotubes, alternative splicing patterns around exons 2 and 29 (<figref idrefs="DRAWINGS">FIG. 5</figref>, Panels b and c). The alternative products were sequenced and found to be due to the skipping of exons 2-7 (in-frame), exons 3-7 (out-of-frame), exons 28-29 (in-frame), and exons 27-29 (in-frame). This genuinely occurring exon skipping was also detected previously in human skeletal muscle [24, 25]. Remarkably, the level of the alternative splicing was significantly enhanced by the AON treatment of the transfected myotube cultures. Also noteworthy is the observation that h2AON1 not only induced exon 2 skipping in the normal transcript, but also in one of the alternative transcripts consisting of exons 1 and 2 spliced to exon 8 (<figref idrefs="DRAWINGS">FIG. 5</figref>, Panel b).</p>
    <p num="p-0068">The majority of AONs induced the precise skipping of the targeted exons, using the original splice sites of the adjacent exons. However, in response to h51AON2, an in-frame cryptic splice site was used in exon 51 (<figref idrefs="DRAWINGS">FIG. 5</figref>, Panel h). The level of this alternatively spliced product was variable in serial transfection experiments. Finally, in some of the transfection experiments, additional aberrant splicing fragments were detected due to the co-skipping of adjacent exons. Their incidence, however, was inconsistent, and at very low levels.</p>
    <p num="p-0069">References to Example 2 (numbering in this part refers strictly to numbering used in Example 2)
</p> <ul> <li id="ul0001-0001" num="0069">[1] Hoffman E. P., R. H. Brown Jr., and L. M. Kunkel. Dystrophin: the protein product of the Duchenne muscular dystrophy locus. <i>Cell </i>1987, 51:919-928.</li> <li id="ul0001-0002" num="0070">[2] Monaco A. P., C. J. Bertelson, S. Liechti-Gallati, H. Moser, and L. M. Kunkel. An explanation for the phenotypic differences between patients bearing partial deletions of the DMD locus. <i>Genomics </i>1988, 2:90-95.</li> <li id="ul0001-0003" num="0071">[3] Koenig M., A. H. Beggs, and M. Moyer, et al. The molecular basis for Duchenne versus Becker muscular dystrophy: correlation of severity with type of deletion. <i>Am. J. Hum. Genet. </i>1989, 45:498-506.</li> <li id="ul0001-0004" num="0072">[4] Zubrzycka-Gaarn E. E., D. E. Bulman, and G. Karpati, et al. The Duchenne muscular dystrophy gene product is localized in sarcolemma of human skeletal muscle. <i>Nature </i>1988, 333:466-469.</li> <li id="ul0001-0005" num="0073">[5] Yoshida M. and E. Ozawa. Glycoprotein complex anchoring dystrophin to sarcolemma. <i>J. Biochem</i>. (Tokyo) 1990, 108:748-752.</li> <li id="ul0001-0006" num="0074">[6] Ervasti J. M. and K. P. Campbell. Membrane organization of the dystrophin-glycoprotein complex. <i>Cell </i>1991, 66:1121-1131.</li> <li id="ul0001-0007" num="0075">[7] Koenig M., A. P. Monaco and L. M. Kunkel. The complete sequence of dystrophin predicts a rod-shaped cytoskeletal protein. <i>Cell </i>1988, 53:219-226.</li> <li id="ul0001-0008" num="0076">[8] van Deutekom J. C., S. S. Floyd and D. K. Booth, et al. Implications of maturation for viral gene delivery to skeletal muscle. <i>Neuromuscul. Disord. </i>1998, 8:135-148.</li> <li id="ul0001-0009" num="0077">[9] Mayeda A., Y. Hayase, H. Inoue, E. Ohtsuka and Y. Ohshima. Surveying cis-acting sequences of pre-mRNA by adding antisense 20-O-methyl oligoribonucleotides to a splicing reaction. <i>J. Biochem</i>. (Tokyo) 1990, 108:399-405.</li> <li id="ul0001-0010" num="0078">[10] Galderisi U., A. Cascino and A. Giordano. Antisense oligonucleotides as therapeutic agents. <i>J. Cell. Physiol. </i>1999, 181:251-257.</li> <li id="ul0001-0011" num="0079">[11] Baker B. F. and B. P. Monia. Novel mechanisms for antisense-mediated regulation of gene expression. <i>Biochim. Biophys. Acta </i>1999, 1489:3-18.</li> <li id="ul0001-0012" num="0080">[12] Kole R. and P. Sazani. Antisense effects in the cell nucleus: modification of splicing. <i>Curr. Opin. Mol. Ther. </i>2001, 3:229-234.</li> <li id="ul0001-0013" num="0081">[13] Sicinski P., Y. Geng, A. S. Ryder-Cook, E. A. Barnard, M. G. Darlison and P. J. Barnard. The molecular basis of muscular dystrophy in the mdx mouse: a point mutation. <i>Science </i>1989, 244:1578-1580.</li> <li id="ul0001-0014" num="0082">[14] Dunckley M. G., M. Manoharan, P. Villiet, I. C. Eperon and G. Dickson. Modification of splicing in the dystrophin gene in cultured Mdx muscle cells by antisense oligoribonucleotides. <i>Hum. Mol. Genet. </i>1998, 7:1083-1090.</li> <li id="ul0001-0015" num="0083">[15] Mann C. J., K. Honeyman and A. J. Cheng, et al. Antisense-induced exon skipping and synthesis of dystrophin in the mdx mouse. <i>Proc. Natl. Acad. Sci. U.S.A. </i>2001, 98:42-47.</li> <li id="ul0001-0016" num="0084">[16] Wilton S. D., F. Lloyd and K. Carville, et al. Specific removal of the nonsense mutation from the mdx dystrophin mRNA using anti-sense oligonucleotides. <i>Neuromuscul. Disord. </i>1999, 9:330-338.</li> <li id="ul0001-0017" num="0085">[17] Takeshima Y., H. Wada, M. Yagi, et al. Oligonucleotides against a splicing enhancer sequence led to dystrophin production in muscle cells from a Duchenne muscular dystrophy patient. <i>Brain Dev. </i>2001, 23:788-790.</li> <li id="ul0001-0018" num="0086">[18] Pramono Z. A., Y. Takeshima, H. Alimsardjono, A. Ishii, S. Takeda and M. Matsuo. Induction of exon skipping of the dystrophin transcript in lymphoblastoid cells by transfecting an antisense oligodeoxynucleotide complementary to an exon recognition sequence. <i>Biochem. Biophys. Res. Commun. </i>1996, 226:445-449.</li> <li id="ul0001-0019" num="0087">[19] Watakabe A., K. Tanaka and Y. Shimura. The role of exon sequences in splice site selection. <i>Genes Dev. </i>1993, 7:407-418.</li> <li id="ul0001-0020" num="0088">[20] Tanaka K., A. Watakabe and Y. Shimura. Polypurine sequences within a downstream exon function as a splicing enhancer. <i>Mol. Cell. Biol. </i>1994, 14:1347-1354.</li> <li id="ul0001-0021" num="0089">[21] van Deutekom J. C., M. Bremmer-Bout, A. A. Janson, et al. Antisense-induced exon skipping restores dystrophin expression in DMD patient-derived muscle cells. <i>Hum. Mol. Genet. </i>2001, 10:1547-1554.</li> <li id="ul0001-0022" num="0090">[22] Mathews D. H., J. Sabina, M. Zuker and D. H. Turner. Expanded sequence dependence of thermodynamic parameters improves prediction of RNA secondary structure. <i>J. Mol. Biol. </i>1999, 288:911-940.</li> <li id="ul0001-0023" num="0091">[23] Richler C. and D. Yaffe. The in vitro cultivation and differentiation capacities of myogenic cell lines. <i>Dev. Biol. </i>1970, 23:1-22.</li> <li id="ul0001-0024" num="0092">[24] Surono A., Y. Takeshima, T. Wibawa, Z. A. Pramono and M. Matsuo. Six novel transcripts that remove a huge intron ranging from 250 to 800 kb are produced by alternative splicing of the 50 region of the dystrophin gene in human skeletal muscle. <i>Biochem. Biophys. Res. Commun. </i>1997, 239:895-899.</li> <li id="ul0001-0025" num="0093">[25] Shiga N., Y. Takeshima, H. Sakamoto, et al. Disruption of the splicing enhancer sequence within exon 27 of the dystrophin gene by a nonsense mutation induces partial skipping of the exon and is responsible for Becker muscular dystrophy. <i>J. Clin. Invest. </i>1997, 100:2204-2210.</li> <li id="ul0001-0026" num="0094">[26] Wells D. J., K. E. Wells, E. A. Asante, et al. Expression of human full-length and minidystrophin in transgenic mdx mice: implications for gene therapy of Duchenne muscular dystrophy. <i>Hum. Mol. Genet. </i>1995, 4:1245-1250.</li> <li id="ul0001-0027" num="0095">[27] Sironi M., U. Pozzoli, R. Cagliani, G. P. Comi, A. Bardoni and N. Bresolin. Analysis of splicing parameters in the dystrophin gene: relevance for physiological and pathogenetic splicing mechanisms. <i>Hum. Genet. </i>2001, 109:73-84.</li> <li id="ul0001-0028" num="0096">A. Aartsma-Rus et al., <i>Neuromuscular Disorders </i>12 (2002) S71-S77.</li> </ul> <heading>Example 3</heading> <heading>Results</heading> <p num="h-0026">Double-Exon Skipping in Two DMD Patients</p>
    <p num="p-0070">This study includes two DMD patients affected by different frame-disrupting mutations in the DMD gene that require the skip of two exons for correction of the reading frame (Table 5). Patient DL 90.3 carries a nonsense mutation in exon 43. Considering that this single exon is out-of-frame, the skipping of exon 43 would remove the nonsense mutation but not restore the reading frame.</p>
    <p num="p-0071">Since the combination with exon 44 is in-frame, in this patient, we aimed at double-exon skipping, targeting both these exons. Patient DL 470.2 is affected by a deletion of exons 46 to 50. Frame restoration would require a double-exon skipping of both exons flanking the deletion. Myotube cultures from both patients were transfected with a mixture of exon 43- and 44-specific AONs (DL90.3) or exon 45- and 51-specific AONs (DL470.2). The individual AONs (Table 5) were previously highly effective in single-exon skipping. Transfection efficiencies were typically over 80%, as indicated by the number of cells with specific nuclear uptake of the fluorescent AONs. RT-PCR analysis at 24 to 48 hours post-transfection, indeed demonstrated the feasibility of specific double-exon skipping in both samples (<figref idrefs="DRAWINGS">FIGS. 6 and 7</figref>). This was confirmed by sequence analysis (data not shown). Additional shorter transcript fragments were obtained due to single-exon skipping: in patient DL90.3, exon 44 skipping (<figref idrefs="DRAWINGS">FIG. 6</figref>), and in patient DL470.2, exon 51 skipping (<figref idrefs="DRAWINGS">FIG. 7</figref>).</p>
    <p num="h-0027">Multi-Exon Skipping</p>
    <p num="p-0072">The splicing of exon 44 directly to exon 52 (as induced in DL470.2) generates an in-frame transcript. We hypothesized that by inducing the skipping of the entire stretch of exons in between, i.e., multi-exon skipping, we would induce a BMD-like deletion (45-51) that covers and restores several known, smaller, DMD mutations. This would further enlarge the group of DMD patients that would benefit from one type of frame correction. The feasibility of multi-exon skipping was first shown in human control myotubes that were treated with a mixture of the exon 45- and 51-specific AONs (<figref idrefs="DRAWINGS">FIG. 7</figref>; KM 109). We then applied it to myotubes from a third DMD patient carrying an exon 48-50 deletion (50685.1). By the AON-induced skipping of the (remaining) stretch of exons in between and including exons 45 and 51, we obtained the anticipated smaller in-frame transcript with exon 44 spliced to exon 52 (<figref idrefs="DRAWINGS">FIG. 7</figref>).</p>
    <p num="h-0028">Double- and Multi-Exon Skipping Using a U-Linked AON Combination</p>
    <p num="p-0073">The skipping of more than one exon from one pre-mRNA molecule requires that both AONs are present in the same nucleus, targeting the same molecule. To enlarge this chance, we here studied the feasibility of one combined AON carrying both AONs specific for exons 45 and 51 (h45AON5 and h51AON2) linked by ten uracil nucleotides (Table 5). Following transfection of this “U-linker AON” into myotubes from human control and the DMD patients D1470.2 and 50685.1, RT-PCR analysis demonstrated its efficacy to generate the anticipated in-frame transcript with exon 44 spliced to exon 52 (<figref idrefs="DRAWINGS">FIG. 7</figref>). This multi-exon skipping occurred specifically and precisely at the exon boundaries as confirmed by sequence analysis (data not shown). In contrast to patient DL 470.2, the U-linker AON was a slightly more efficient than the mixture of AONs in the human control and in patient 50685.1.</p>
    <p num="h-0029">Materials and Methods</p>
    <p num="h-0030">AONs and Primers</p>
    <p num="p-0074">AONs (Table 5) targeting exons 43, 44 and 51 were previously described (Aartsma-Rus, 2002). AONs targeting exon 45 were newly designed (sequences upon request). All AONs contain a 5′ fluorescein group (6-FAM), a full-length phosphorothioate backbone and 2′-O-methyl modified ribose molecules (Eurogentec, BE). To avoid interference with the fluorescent signals of the secondary antibodies, unlabelled AONs were used for immuno-histochemical analyses. Primers for RT-PCR analysis (Table 5, sequences available upon request) were synthesized by Eurogentec (BE).</p>
    <p num="h-0031">RNA Isolation and RT-PCR Analysis</p>
    <p num="p-0075">At 24 to 48 hours post-transfection, total RNA was isolated from the myotube cultures (RNA-Bee RNA isolation solvent, Campro Scientific, NL). 300 ng of total RNA were used for RT-PCR analysis using C. therm. polymerase (Roche Diagnostics, NL) in a 20 μl reaction at 60° C. for 30 minutes, primed with different DMD gene-specific reverse primers (Table 5). Primary PCRs were performed by 20 cycles of 94° C. (40 seconds), 60° C. (40 seconds) and 72° C. (60 seconds). One μl of these reactions was then re-amplified in nested PCRs by 32 cycles of 94° C. (40 seconds), 60° C. (40 seconds) and 72° C. (60 seconds). PCR products were analyzed on 1.5% or 2% agarose gels. For quantification of the transcript products, nested PCRs were performed using 24 cycles. PCR products were analyzed using the DNA 7500 LabChip® Kit and the Agilent 2100 Bioanalyzer (Agilent Technologies, NL).</p>
    <p num="h-0032">Sequence Analysis</p>
    <p num="p-0076">RT-PCR products were isolated from agarose gels using the QIAquick Gel Extraction Kit (Qiagen). Direct DNA sequencing was carried out by the Leiden Genome Technology Center (LGTC) using the BigDye Terminator Cycle Sequencing Ready Reaction kit (PE Applied Biosystems) and analyzed on an ABI 3700 Sequencer (PE Applied Biosystems).</p>
    <heading>Example 4</heading> <heading>Expression Vectors Encoding a Transcript Comprising an Oligonucleotide of the Invention</heading> <p num="p-0077">Due to the defined turnover rate of both the dystrophin pre-mRNA and the AONs, our DMD frame-correction therapy would require repetitive administrations of AONs. In addition, relatively high levels of antisense RNA will be necessary within the nucleus, where transcription and splicing of the dystrophin pre-mRNA occur. Therefore, we have set up a vector system in which specific AON sequences are incorporated into a modified gene. In this example, this embodiment is described for U7 small nuclear RNA (U7snRNA). U7snRNA is the RNA component of the U7 ribonucleoprotein particle (U7snRNP) that is involved in the processing of the 3′ end of histone pre-mRNAs. Inherent to its function, U7snRNA is efficiently transported back from the cytoplasm to the nucleus in which it gets subsequently incorporated into very stable U7snRNP complexes. A similar approach was successfully applied in AON-based gene therapy studies on β-thalassemia (53, 54). In these studies, different plasmids were engineered containing a modified U7snRNA gene from which the natural antisense sequence directed to the histone pre-mRNA was replaced with antisense sequences targeted to different β-thalassemia-associated aberrant splicing sites in the β-globin gene. Following transfection of these plasmids, correct splicing and expression of the full-length β-globin protein could be restored with an efficiency of up to 65% in cultured cells expressing the different mutant β-globin genes.</p>
    <p num="p-0078">Various U7snRNA gene constructs were engineered as described in reference 53 with the modification that the β-globin sequences were exactly replaced by the antisense sequences derived from the different AONs. In this Example, the sequences were replaced by the antisense sequences of m46AON4, 6, 9, or 11 that were effective in inducing the skipping of mouse exon 46. A sense construct was included as negative control (m46SON6). Following construct validation by sequencing, the plasmids were tested in vitro by transfection into cultured C2C12 mouse myoblasts. The U7snRNA-m46AON6 construct was most efficient.</p>
    <p num="p-0079">To enhance delivery of the AON-U7snRNA gene constructs, we have cloned them into recombinant adeno-associated viral (rAAV) vectors. AAV is a single-stranded DNA parvovirus that is non-pathogenic and shows a helper-dependent life cycle. In contrast to other viruses (adenovirus, retrovirus, and herpes simplex virus), rAAV vectors have demonstrated to be very efficient in transducing mature skeletal muscle. Because application of rAAV in classical DMD “gene addition” studies has been hindered by its restricted packaging limits (&lt;5 kb), we applied rAAV for the efficient delivery of the much smaller U7snRNA antisense constructs (&lt;600 bp) to mature murine skeletal muscle. The rAAV-U7-AON vectors also contain the gene for green fluorescence protein (GFP-cDNA), which allows analysis of transduction efficiencies in muscle post-injection. High titer virus productions were effective in inducing exon skipping.</p>
    <p num="h-0035">References (to the general part, excluding Example 2)</p>
    <p num="p-0080">
      </p> <ul> <li id="ul0002-0001" num="0107">1. Hoffman E. P., R. H. Brown Jr., and L. M. Kunkel (1987) Dystrophin: the protein product of the Duchenne muscular dystrophy locus. <i>Cell </i>51:919-928.</li>
        <li id="ul0002-0002" num="0108">2. Hoffman E. P., K. H. Fischbeck, R. H. Brown, M. Johnson, R. Medori, J. D. Loike, J. B. Harris, R. Waterston, M. Brooke, L. Specht, et al. (1988) Characterization of dystrophin in muscle-biopsy specimens from patients with Duchenne's or Becker's muscular dystrophy. <i>N. Engl. J. Med. </i>318:1363-1368.</li>
        <li id="ul0002-0003" num="0109">3. Den Dunnen J. T., P. M. Grootscholten, E. Bakker, L. A. Blonden, H. B. Ginjaar, M. C. Wapenaar, H. M. van Paassen, C. van Broeckhoven, P. L. Pearson, and G. J. van Ommen (1989) Topography of the Duchenne muscular dystrophy (DMD) gene: FIGE and cDNA analysis of 194 cases reveals 115 deletions and 13 duplications. <i>Am. J. Hum. Genet. </i>45:835-847.</li>
        <li id="ul0002-0004" num="0110">4. Koenig M., A. H. Beggs, M. Moyer, S. Scherpf, K. Heindrich, T. Bettecken, G. Meng, C. R. Muller, M. Lindlof, H. Kaariainen, et al. (1989) The molecular basis for Duchenne versus Becker muscular dystrophy: correlation of severity with type of deletion. <i>Am. J. Hum. Genet. </i>45:498-506.</li>
        <li id="ul0002-0005" num="0111">5. Tuffery-Giraud S., S. Chambert, J. Demaille, and M. Claustres (1999) Point mutations in the dystrophin gene: evidence for frequent use of cryptic splice sites as a result of splicing defects. <i>Hum. Mutat. </i>14:359-368.</li>
        <li id="ul0002-0006" num="0112">6. Prior T. W., C. Bartolo, D. K. Pearl, A. C. Papp, P. J. Snyder, M. S. Sedra, A. H. Burghes, and J. R. Mendell (1995) Spectrum of small mutations in the dystrophin coding region. <i>Am. J. Hum. Genet. </i>57:22-33.</li>
        <li id="ul0002-0007" num="0113">7. Moser H. (1984) Duchenne muscular dystrophy: pathogenetic aspects and genetic prevention. <i>Hum. Genet. </i>66:17-40.</li>
        <li id="ul0002-0008" num="0114">8. Emery A. E. (2002) The muscular dystrophies. <i>Lancet </i>359:687-695.</li>
        <li id="ul0002-0009" num="0115">9. Yoshida M. and E. Ozawa (1990) Glycoprotein complex anchoring dystrophin to sarcolemma. <i>J. Biochem</i>. (Tokyo), 108:748-752.</li>
        <li id="ul0002-0010" num="0116">10. Ervasti J. M. and K. P. Campbell (1991) Membrane organization of the dystrophin-glycoprotein complex. <i>Cell </i>66:1121-1131.</li>
        <li id="ul0002-0011" num="0117">11. Di Blasi C., L. Morandi, R. Barresi, F. Blasevich, F. Comelio, and M. Mora (1996) Dystrophin-associated protein abnormalities in dystrophin-deficient muscle fibers from symptomatic and asymptomatic Duchenne/Becker muscular dystrophy carriers. <i>Acta Neuropathol</i>. (Berl), 92:369-377.</li>
        <li id="ul0002-0012" num="0118">12. Ervasti J. M., K. Ohlendieck, S. D. Kahl, M. G. Gaver, and K. P. Campbell (1990) Deficiency of a glycoprotein component of the dystrophin complex in dystrophic muscle. <i>Nature </i>345:315-319.</li>
        <li id="ul0002-0013" num="0119">13. Matsumura K., A. H. Burghes, M. Mora, F. M. Tome, L. Morandi, F. Cornello, F. Leturcq, M. Jeanpierre, J. C. Kaplan, P. Reinert, et al. (1994) Immunohistochemical analysis of dystrophin-associated proteins in Becker/Duchenne muscular dystrophy with huge in-frame deletions in the NH2-terminal and rod domains of dystrophin. <i>J. Clin. Invest. </i>93:99-105.</li>
        <li id="ul0002-0014" num="0120">14. Monaco A. P., C. J. Bertelson, S. Liechti-Gallati, H. Moser, L. M. Kunkel (1988) An explanation for the phenotypic differences between patients bearing partial deletions of the DMD locus. <i>Genomics </i>2:90-95.</li>
        <li id="ul0002-0015" num="0121">15. Clemens P. R. and F. J. Duncan (2001) Progress in gene therapy for Duchenne muscular dystrophy. <i>Curr. Neurol. Neurosci. Rep. </i>1:89-96.</li>
        <li id="ul0002-0016" num="0122">16. Khan M. A. (1993) Corticosteroid therapy in Duchenne muscular dystrophy. <i>J. Neurol. Sci. </i>120:8-14.</li>
        <li id="ul0002-0017" num="0123">17. De Angelis F. G., O. Sthandier, B. Berarducci, S. Toso, G. Galluzzi, E. Ricci, G. Cossu and I. Bozzoni (2002) Chimeric snRNA molecules carrying antisense sequences against the splice junctions of exon 51 of the dystrophin pre-mRNA induce exon skipping and restoration of a dystrophin synthesis in Delta 48-50 DMD cells. <i>Proc. Natl. Acad. Sci. USA </i>99:9456-9461.</li>
        <li id="ul0002-0018" num="0124">18. Mann C. J., K. Honeyman, A. J. Cheng, T. Ly, F. Lloyd, S. Fletcher, J. E. Morgan, T. A. Partridge and S. D. Wilton (2001) Antisense-induced exon skipping and synthesis of dystrophin in the mdx mouse. <i>Proc. Natl. Acad. Sci. USA, </i>98:42-47.</li>
        <li id="ul0002-0019" num="0125">19. van Deutekom J. C., M. Bremmer-Bout, A. A. Janson, I. B. Ginjaar, F. Baas, J. T. den Dunnen and G. J. van Ommen (2001) Antisense-induced exon skipping restores dystrophin expression in DMD patient-derived muscle cells. <i>Hum. Mol. Genet. </i>10:1547-1554.</li>
        <li id="ul0002-0020" num="0126">20. Wilton S. D., F. Lloyd, K. Carville, S. Fletcher, K. Honeyman, S. Agrawal and R. Kole (1999) Specific removal of the nonsense mutation from the mdx dystrophin mRNA using antisense oligonucleotides. <i>Neuromuscul. Disord. </i>9:330-338.</li>
        <li id="ul0002-0021" num="0127">21. Dunckley M. G., M. Manoharan, P. Villiet, I. C. Eperon and G. Dickson (1998) Modification of splicing in the dystrophin gene in cultured Mdx muscle cells by antisense oligoribonucleotides. <i>Hum. Mol. Genet. </i>7:1083-1090.</li>
        <li id="ul0002-0022" num="0128">22. Takeshima Y., H. Wada, M. Yagi, Y. Ishikawa, R. Minami, H. Nakamura and M. Matsuo (2001) Oligonucleotides against a splicing enhancer sequence led to dystrophin production in muscle cells from a Duchenne muscular dystrophy patient. <i>Brain Dev. </i>23:788-790.</li>
        <li id="ul0002-0023" num="0129">23. Aartsma-Rus A., M. Bremmer-Bout, A. Janson, J. den Dunnen, G. van Ommen and J. van Deutekom (2002) Targeted exon skipping as a potential gene correction therapy for Duchenne muscular dystrophy. <i>Neuromuscul. Disord. </i>12, S71.</li>
        <li id="ul0002-0024" num="0130">24. Shiga N., Y. Takeshima, H. Sakamoto, K. Inoue, Y. Yokota, M. Yokoyama and M. Matsuo (1997) Disruption of the splicing enhancer sequence within exon 27 of the dystrophin gene by a nonsense mutation induces partial skipping of the exon and is responsible for Becker muscular dystrophy. <i>J. Clin. Invest. </i>100:2204-2210.</li>
        <li id="ul0002-0025" num="0131">25. Cartegni L., S. L. Chew and A. R. Krainer (2002) Listening to silence and understanding nonsense: exonic mutations that affect splicing. <i>Nat. Rev. Genet. </i>3:285-298.</li>
        <li id="ul0002-0026" num="0132">26. Schaal T. D. and T. Maniatis (1999) Multiple distinct splicing enhancers in the protein-coding sequences of a constitutively spliced pre-mRNA. <i>Mol. Cell. Biol. </i>19:261-273.</li>
        <li id="ul0002-0027" num="0133">27. Takeshima Y., H. Nishio, H. Sakamoto, H. Nakamura and M. Matsuo (1995) Modulation of in vitro splicing of the upstream intron by modifying an intra-exon sequence which is deleted from the dystrophin gene in dystrophin Kobe. <i>J. Clin. Invest. </i>95:515-520.</li>
        <li id="ul0002-0028" num="0134">28. Pramono Z. A., Y. Takeshima, H. Alimsardjono, A. Ishii, S. Takeda, and M. Matsuo (1996) Induction of exon skipping of the dystrophin transcript in lymphoblastoid cells by transfecting an antisense oligodeoxynucleotide complementary to an exon recognition sequence. <i>Biochem. Biophys. Res. Commun. </i>226:445-449.</li>
        <li id="ul0002-0029" num="0135">29. Koenig M., A. P. Monaco and L. M. Kunkel (1988) The complete sequence of dystrophin predicts a rod-shaped cytoskeletal protein. <i>Cell </i>53:219-226.</li>
        <li id="ul0002-0030" num="0136">30. Den Dunnen J. (1996) The Leiden Muscular Dystrophy pages; http://www.dmd.nl.</li>
        <li id="ul0002-0031" num="0137">31. Mann C. J., K. Honeyman, G. McClorey, S. Fletcher and S. D. Wilton (2002) Improved antisense oligonucleotide-induced exon skipping in the mdx mouse model of muscular dystrophy. <i>J. Gene Med. </i>4:644-654.</li>
        <li id="ul0002-0032" num="0138">32. Kerr T. P., C. A. Sewry, S. A. Robb and R. G. Roberts (2001) Long mutant dystrophins and variable phenotypes: evasion of nonsense-mediated decay? <i>Hum. Genet. </i>109:402-407.</li>
        <li id="ul0002-0033" num="0139">33. Klein C. J., D. D. Coovert, D. E. Bulman, P. N. Ray, J. R. Mendell and A. H. Burghes (1992) Somatic reversion/suppression in Duchenne muscular dystrophy (DMD): evidence supporting a frame-restoring mechanism in rare dystrophin-positive fibers. <i>Am. J. Hum. Genet. </i>50:950-959.</li>
        <li id="ul0002-0034" num="0140">34. Sherratt T. G., T. Vulliamy, V. Dubowitz, C. A. Sewry and P. N. Strong (1993) Exon skipping and translation in patients with frameshift deletions in the dystrophin gene. <i>Am. J. Hum. Genet. </i>53:1007-1015.</li>
        <li id="ul0002-0035" num="0141">35. Lu Q. L., G. E. Morris, S. D. Wilton, T. Ly, O. V. Artem'yeva, P. Strong and T. A. Partridge (2000) Massive idiosyncratic exon skipping corrects the nonsense mutation in dystrophic mouse muscle and produces functional revertant fibers by clonal expansion. <i>J. Cell Biol. </i>148:985-996.</li>
        <li id="ul0002-0036" num="0142">36. Nicholson L. V., M. A. Johnson, K. M. Bushby and D. Gardner-Medwin (1993) Functional significance of dystrophin-positive fibers in Duchenne muscular dystrophy. <i>Arch. Dis. Child </i>68:632-636.</li>
        <li id="ul0002-0037" num="0143">37. Vainzof M., M. R. Passos-Bueno, R. I. Takata, C. Pavanello Rde and M. Zatz (1993) Intrafamilial variability in dystrophin abundance correlated with difference in the severity of the phenotype. <i>J. Neurol. Sci. </i>119:38-42.</li>
        <li id="ul0002-0038" num="0144">38. Singh V., S. Sinha, S. Mishra, L. S. Chaturvedi, S. Pradhan, R. D. Mittal and B. Mittal (1997) Proportion and pattern of dystrophin gene deletions in north Indian Duchenne and Becker muscular dystrophy patients. <i>Hum. Genet. </i>99:206-208.</li>
        <li id="ul0002-0039" num="0145">39. Melacini P., M. Fanin, G. A. Danieli, G. Fasoli, C. Villanova, C. Angelini, L. Vitiello, M. Miorelli, G. F. Buja, M. L. Mostacciuolo, et al. (1993) Cardiac involvement in Becker muscular dystrophy. <i>J. Am. Coll. Cardiol. </i>22:1927-1934.</li>
        <li id="ul0002-0040" num="0146">40. Melis M. A., M. Cau, F. Muntoni, A. Mateddu, R. Galanello, L. Boccone, F. Deidda, D. Loi and A. Cao (1998) Elevation of serum creatine kinase as the only manifestation of an intragenic deletion of the dystrophin gene in three unrelated families. <i>Europ. J. Paediatr. Neurol. </i>2:255-261.</li>
        <li id="ul0002-0041" num="0147">41. Onengut, S., G. N. Kavaslar, E. Battaloglu, P. Serdaroglu, F. Deymeer, C. Ozdemir, F. Calafell and A. Tolun (2000) Deletion pattern in the dystrophin gene in Turks and a comparison with Europeans and Indians. <i>Ann. Hum. Genet. </i>64:33-40.</li>
        <li id="ul0002-0042" num="0148">42. Rosenberg C., L. Navajas, D. F. Vagenas, E. Bakker, M. Vainzof, M. R. Passos-Bueno, R. I. Takata, G. J. Van Ommen, M. Zatz and J. T. Den Dunnen (1998) Clinical diagnosis of heterozygous dystrophin gene deletions by fluorescence in situ hybridization. <i>Neuromuscul. Disord. </i>8:447-452.</li>
        <li id="ul0002-0043" num="0149">43. Sertic J., N. Barisic, M. Sostarko, Z. Brzovic and A. Stavljenic-Rukavina (1997) Deletion screening of the Duchenne/Becker muscular dystrophy gene in Croatian population. <i>Coll. Anthropol. </i>21:151-156.</li>
        <li id="ul0002-0044" num="0150">44. Rando T. A. and H. M. Blau (1994) Primary mouse myoblast purification, characterization, and transplantation for cell-mediated gene therapy. <i>J. Cell Biol. </i>125:1275-1287.</li>
        <li id="ul0002-0045" num="0151">45. Murry C. E., M. A. Kay, T. Bartosek, S. D. Hauschka and S. M. Schwartz (1996) Muscle differentiation during repair of myocardial necrosis in rats via gene transfer with MyoD. <i>J. Clin. Invest. </i>98:2209-2217.</li>
        <li id="ul0002-0046" num="0152">46. Roest P. A., A. C. van der Tuijn, H. B. Ginjaar, R. C. Hoeben, F. B. Hoger-Vorst, E. Bakker, J. T. den Dunnen and G. J. van Ommen (1996) Application of in vitro Myo-differentiation of non-muscle cells to enhance gene expression and facilitate analysis of muscle proteins. <i>Neuromuscul. Disord. </i>6:195-202.</li>
        <li id="ul0002-0047" num="0153">47. Havenga M. J., A. A. Lemckert, O. J. Ophorst, M. van Meijer, W. T. Germeraad, J. Grimbergen, M. A. van Den Doel, R. Vogels, J. van Deutekom and A. A. Janson, et al. (2002) Exploiting the natural diversity in adenovirus tropism for therapy and prevention of disease. <i>J. Virol. </i>76:4612-4620.</li>
        <li id="ul0002-0048" num="0154">48. Anderson L. V. and K. Davison (1999) Multiplex Western blotting system for the analysis of muscular dystrophy proteins. <i>Am. J. Pathol. </i>154:1017-1022.</li>
        <li id="ul0002-0049" num="0155">49. Neugebauer K. M. et al., <i>J. Cell. Biol. </i>129:899-908 (1995).</li>
        <li id="ul0002-0050" num="0156">50. Tacke R. and J. L. Manley, <i>Proc. Soc. Exp. Biol. Med. </i>220:59-63 (1999).</li>
        <li id="ul0002-0051" num="0157">51. Graveley B. R. et al., <i>Curr. Biol. </i>9:R6-7 (1999).</li>
        <li id="ul0002-0052" num="0158">52. Misteli T et al., <i>Nature </i>387:523-527 (1997).</li>
        <li id="ul0002-0053" num="0159">53. Gorman L, Suter D, Emerick V, et al. Stable alteration of pre mRNA splicing patterns by modified U7 small nuclear RNAs. <i>Proc. Natl. Acad. Sci. USA </i>1998, 4929-4934.</li>
        <li id="ul0002-0054" num="0160">54. Suter D, Tomasini R, Reber U, et al. Double target antisense U7 snRNAs promote efficient skipping of an aberrant exon in three human beta thalassemic mutations. <i>Hum. Mol. Genet. </i>1999, 8:2415-2423.</li>
      </ul> <p num="p-0081">
      <tables id="TABLE-US-00001" num="00001"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="357pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Overview of the patients, the AONs and the primer sets used in this study</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="77pt" align="left"> </colspec> <colspec colname="3" colwidth="49pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="42pt" align="left"> </colspec> <colspec colname="6" colwidth="63pt" align="left"> </colspec> <colspec colname="7" colwidth="56pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Patients</td>
                <td class="description-td">Mutations</td>
                <td class="description-td">Targeted exons</td>
                <td class="description-td">AONs<sup>a</sup> </td>
                <td class="description-td">RT-primers<sup>b</sup> </td>
                <td class="description-td">Primary PCR sets<sup>b</sup> </td>
                <td class="description-td">Nested PCR sets<sup>b</sup> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">DL 515.2</td>
                <td class="description-td">Deletion exon 45-50</td>
                <td class="description-td">Exon 51</td>
                <td class="description-td">h51AON1</td>
                <td class="description-td">h53r</td>
                <td class="description-td">h41f-h53r</td>
                <td class="description-td">h42f-h52r</td>
              </tr> <tr class="description-tr"> <td class="description-td">DL 363.2</td>
                <td class="description-td">Deletion exon 45-54</td>
                <td class="description-td">Exon 44</td>
                <td class="description-td">h44AON1</td>
                <td class="description-td">h55r2</td>
                <td class="description-td">h42f-h55r2</td>
                <td class="description-td">h44f-h55r</td>
              </tr> <tr class="description-tr"> <td class="description-td">50685.1</td>
                <td class="description-td">Deletion exon 48-50</td>
                <td class="description-td">Exon 51</td>
                <td class="description-td">h51AON1</td>
                <td class="description-td">h53r</td>
                <td class="description-td">h46f-h53r</td>
                <td class="description-td">h47f-h52r</td>
              </tr> <tr class="description-tr"> <td class="description-td">DL 589.2</td>
                <td class="description-td">Deletion exon 51-55</td>
                <td class="description-td">Exon 50</td>
                <td class="description-td">h50AON1</td>
                <td class="description-td">h58r</td>
                <td class="description-td">h47f-h58r</td>
                <td class="description-td">h49f-h57r</td>
              </tr> <tr class="description-tr"> <td class="description-td">53914.1</td>
                <td class="description-td">Deletion exon 52</td>
                <td class="description-td">Exon 51</td>
                <td class="description-td">h51AON1</td>
                <td class="description-td">h55r</td>
                <td class="description-td">h49f-h55r</td>
                <td class="description-td">h50f-h54r</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">″</td>
                <td class="description-td">Exon 53</td>
                <td class="description-td">h53AON1</td>
                <td class="description-td">″</td>
                <td class="description-td">″</td>
                <td class="description-td">″</td>
              </tr> <tr class="description-tr"> <td class="description-td">50423.1</td>
                <td class="description-td">Point mutation exon 49</td>
                <td class="description-td">Exon 49</td>
                <td class="description-td">h49AON1</td>
                <td class="description-td">h52r</td>
                <td class="description-td">h46f-h52r</td>
                <td class="description-td">h47f-h51r</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00001" class="description-td" colspan="7"> <sup>a</sup>AON sequences were published previously (23).</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00002" class="description-td" colspan="7"> <sup>b</sup>Primer sequences available upon request.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0082">
      <tables id="TABLE-US-00002" num="00002"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="315pt" align="center"> </colspec> <colspec colname="2" colwidth="0pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="2" rowsep="1" class="description-td" colspan="2">TABLE 2</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Characteristics of the AONs used to study the</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">targeted skipping of 15 different DMD exons<sup>a</sup> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="9"> <colspec colname="1" colwidth="28pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="105pt" align="left"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="49pt" align="center"> </colspec> <colspec colname="9" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SEQ ID</td>
                <td class="description-td"> </td>
                <td class="description-td">Antisense</td>
                <td class="description-td">Length</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Exon</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">NO:</td>
                <td class="description-td">Name</td>
                <td class="description-td">sequence (5′-3′)</td>
                <td class="description-td">(bp)</td>
                <td class="description-td">G/C%</td>
                <td class="description-td">U/C%</td>
                <td class="description-td">skip</td>
                <td class="description-td">Transcript</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="center" rowsep="1" class="description-td" colspan="9"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="9"> <colspec colname="1" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="105pt" align="left"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="49pt" align="center"> </colspec> <colspec colname="9" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">1</td>
                <td class="description-td">h2AON 1</td>
                <td class="description-td">cccauuuugugaauguuuucuuuu</td>
                <td class="description-td">24</td>
                <td class="description-td">29</td>
                <td class="description-td">75</td>
                <td class="description-td">+</td>
                <td class="description-td">OF</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">2</td>
                <td class="description-td">h2AON 2</td>
                <td class="description-td">uugugcauuuacccauuuugug</td>
                <td class="description-td">22</td>
                <td class="description-td">36</td>
                <td class="description-td">68</td>
                <td class="description-td">−</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">3</td>
                <td class="description-td">h29AON 1</td>
                <td class="description-td">Uauccucugaaugucgcauc</td>
                <td class="description-td">20</td>
                <td class="description-td">45</td>
                <td class="description-td">65</td>
                <td class="description-td">+</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">4</td>
                <td class="description-td">h29AON 2</td>
                <td class="description-td">gguuauccucugaaugucgc</td>
                <td class="description-td">20</td>
                <td class="description-td">50</td>
                <td class="description-td">60</td>
                <td class="description-td">+</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">5</td>
                <td class="description-td">h40AON 1</td>
                <td class="description-td">Gagccuuuuuucuucuuug</td>
                <td class="description-td">19</td>
                <td class="description-td">37</td>
                <td class="description-td">79</td>
                <td class="description-td">+</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">6</td>
                <td class="description-td">h40AON 2</td>
                <td class="description-td">Uccuuucgucucugggcuc</td>
                <td class="description-td">19</td>
                <td class="description-td">58</td>
                <td class="description-td">79</td>
                <td class="description-td">+</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">7</td>
                <td class="description-td">h41AON 1</td>
                <td class="description-td">Cuccucuuucuucuucugc</td>
                <td class="description-td">19</td>
                <td class="description-td">47</td>
                <td class="description-td">95</td>
                <td class="description-td">+</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">8</td>
                <td class="description-td">h41AON 2</td>
                <td class="description-td">Cuucgaaacugagcaaauuu</td>
                <td class="description-td">20</td>
                <td class="description-td">35</td>
                <td class="description-td">50</td>
                <td class="description-td">+</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">9</td>
                <td class="description-td">h42AON 1</td>
                <td class="description-td">cuugugagacaugagug</td>
                <td class="description-td">17</td>
                <td class="description-td">47</td>
                <td class="description-td">41</td>
                <td class="description-td">+</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">10</td>
                <td class="description-td">h42AON 2</td>
                <td class="description-td">cagagacuccucuugcuu</td>
                <td class="description-td">18</td>
                <td class="description-td">50</td>
                <td class="description-td">67</td>
                <td class="description-td">+</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">11</td>
                <td class="description-td">h43AON 1</td>
                <td class="description-td">ugcugcugucuucuugcu</td>
                <td class="description-td">18</td>
                <td class="description-td">50</td>
                <td class="description-td">78</td>
                <td class="description-td">−</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">12</td>
                <td class="description-td">h43AON 2</td>
                <td class="description-td">Uuguuaacuuuuucccauu</td>
                <td class="description-td">19</td>
                <td class="description-td">26</td>
                <td class="description-td">79</td>
                <td class="description-td">+</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">13</td>
                <td class="description-td">h44AON 1</td>
                <td class="description-td">cgccgccauuucucaacag</td>
                <td class="description-td">19</td>
                <td class="description-td">58</td>
                <td class="description-td">63</td>
                <td class="description-td">+</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">14</td>
                <td class="description-td">h44AON 2</td>
                <td class="description-td">uuuguauuuagcauguuccc</td>
                <td class="description-td">20</td>
                <td class="description-td">35</td>
                <td class="description-td">70</td>
                <td class="description-td">+</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">15</td>
                <td class="description-td">h45AON 1</td>
                <td class="description-td">gcugaauuauuucuucccc</td>
                <td class="description-td">19</td>
                <td class="description-td">42</td>
                <td class="description-td">74</td>
                <td class="description-td">−</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">16</td>
                <td class="description-td">h45AON 5</td>
                <td class="description-td">gcccaaugccauccugg</td>
                <td class="description-td">17</td>
                <td class="description-td">65</td>
                <td class="description-td">58</td>
                <td class="description-td">+</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">17</td>
                <td class="description-td">h46AON 4b</td>
                <td class="description-td">cugcuuccuccaacc</td>
                <td class="description-td">15</td>
                <td class="description-td">60</td>
                <td class="description-td">80</td>
                <td class="description-td">+</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">18</td>
                <td class="description-td">h46AON 8b</td>
                <td class="description-td">gcuuuucuuuuaguugcugc</td>
                <td class="description-td">20</td>
                <td class="description-td">40</td>
                <td class="description-td">75</td>
                <td class="description-td">+</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">19</td>
                <td class="description-td">h47AON 1</td>
                <td class="description-td">ucuugcucuucugggcuu</td>
                <td class="description-td">18</td>
                <td class="description-td">50</td>
                <td class="description-td">78</td>
                <td class="description-td">−</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">20</td>
                <td class="description-td">h47AON 2</td>
                <td class="description-td">cuugagcuuauuuucaaguuu</td>
                <td class="description-td">21</td>
                <td class="description-td">29</td>
                <td class="description-td">67</td>
                <td class="description-td">−</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">21</td>
                <td class="description-td">h48AON 1</td>
                <td class="description-td">uuucuccuuguuucuc</td>
                <td class="description-td">16</td>
                <td class="description-td">38</td>
                <td class="description-td">94</td>
                <td class="description-td">−</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">22</td>
                <td class="description-td">h48AON 2</td>
                <td class="description-td">ccauaaauuuccaacugauuc</td>
                <td class="description-td">21</td>
                <td class="description-td">33</td>
                <td class="description-td">62</td>
                <td class="description-td">−</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">23</td>
                <td class="description-td">h49AON 1</td>
                <td class="description-td">Cuuccacauccgguuguuu</td>
                <td class="description-td">19</td>
                <td class="description-td">47</td>
                <td class="description-td">74</td>
                <td class="description-td">+</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">24</td>
                <td class="description-td">h49AON 2</td>
                <td class="description-td">Guggcugguuuuuccuugu</td>
                <td class="description-td">19</td>
                <td class="description-td">47</td>
                <td class="description-td">68</td>
                <td class="description-td">+</td>
                <td class="description-td">IF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">25</td>
                <td class="description-td">h50AON 1</td>
                <td class="description-td">cucagagcucagaucuu</td>
                <td class="description-td">17</td>
                <td class="description-td">47</td>
                <td class="description-td">59</td>
                <td class="description-td">+</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">26</td>
                <td class="description-td">h50AON 2</td>
                <td class="description-td">ggcugcuuugcccuc</td>
                <td class="description-td">15</td>
                <td class="description-td">67</td>
                <td class="description-td">73</td>
                <td class="description-td">−</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">27</td>
                <td class="description-td">h51AON 1</td>
                <td class="description-td">Ucaaggaagauggcauuucu</td>
                <td class="description-td">20</td>
                <td class="description-td">40</td>
                <td class="description-td">45</td>
                <td class="description-td">+</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">28</td>
                <td class="description-td">h51AON 2</td>
                <td class="description-td">ccucugugauuuuauaacuugau</td>
                <td class="description-td">23</td>
                <td class="description-td">30</td>
                <td class="description-td">65</td>
                <td class="description-td">+</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">29</td>
                <td class="description-td">h53AON 1</td>
                <td class="description-td">cuguugccuccgguucug</td>
                <td class="description-td">18</td>
                <td class="description-td">61</td>
                <td class="description-td">72</td>
                <td class="description-td">+</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">30</td>
                <td class="description-td">h53AON 2</td>
                <td class="description-td">uuggcucuggccuguccu</td>
                <td class="description-td">18</td>
                <td class="description-td">61</td>
                <td class="description-td">72</td>
                <td class="description-td">−</td>
                <td class="description-td">OF</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="center" rowsep="1" class="description-td" colspan="9"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="left" id="FOO-00003" class="description-td" colspan="9"> <sup>a</sup>Two AONs were tested per exon. Their different lengths and G/C contents (%) did not correlate to their effectivity in exon skipping (1, induced skipping, 2, no skipping). The AONs were directed to purine (A/G)-rich sequences as indicated by their (antisense) U/C content (%). Skipping of the target exons resulted in either an in-frame (IF) or an out-of-frame (OF) transcript.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="left" id="FOO-00004" class="description-td" colspan="9"> <sup>b</sup>van Deutekom et al., 2001 [21].</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0083">
      <tables id="TABLE-US-00003" num="00003"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 3</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Primer sets used for the RT-PCR analyses to</td>
              </tr> <tr class="description-tr"> <td class="description-td">detect the skipping of the targeted exons<sup>a</sup> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="63pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="56pt" align="left"> </colspec> <colspec colname="4" colwidth="56pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Target</td>
                <td class="description-td">RT-</td>
                <td class="description-td">Primary PCR</td>
                <td class="description-td">Nested PCR</td>
              </tr> <tr class="description-tr"> <td class="description-td">exon</td>
                <td class="description-td">primer</td>
                <td class="description-td">primer set</td>
                <td class="description-td">primer set</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="63pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="56pt" align="left"> </colspec> <colspec colname="4" colwidth="56pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">2</td>
                <td class="description-td">h4r</td>
                <td class="description-td">h1f1-h4r</td>
                <td class="description-td">h1f2-h3r</td>
              </tr> <tr class="description-tr"> <td class="description-td">2</td>
                <td class="description-td">h9r</td>
                <td class="description-td">h1f1-h9r</td>
                <td class="description-td">h1f2-h8r</td>
              </tr> <tr class="description-tr"> <td class="description-td">29</td>
                <td class="description-td">h31r</td>
                <td class="description-td">h25f-h31r</td>
                <td class="description-td">h26f-h30r</td>
              </tr> <tr class="description-tr"> <td class="description-td">40</td>
                <td class="description-td">h44r</td>
                <td class="description-td">h38f-h44r</td>
                <td class="description-td">h39f-h43r</td>
              </tr> <tr class="description-tr"> <td class="description-td">41</td>
                <td class="description-td">h44r</td>
                <td class="description-td">h38f-h44r</td>
                <td class="description-td">h39f-h43r</td>
              </tr> <tr class="description-tr"> <td class="description-td">42</td>
                <td class="description-td">h44r</td>
                <td class="description-td">h38f-h44r</td>
                <td class="description-td">h39f-h43r</td>
              </tr> <tr class="description-tr"> <td class="description-td">43</td>
                <td class="description-td">h47r</td>
                <td class="description-td">h41f-h47r</td>
                <td class="description-td">h42f-h46r</td>
              </tr> <tr class="description-tr"> <td class="description-td">44</td>
                <td class="description-td">h47r</td>
                <td class="description-td">h41f-h47r</td>
                <td class="description-td">h42f-h46r</td>
              </tr> <tr class="description-tr"> <td class="description-td">45</td>
                <td class="description-td">h47r</td>
                <td class="description-td">h41f-h47r</td>
                <td class="description-td">h42f-h46r</td>
              </tr> <tr class="description-tr"> <td class="description-td">46</td>
                <td class="description-td">h48r</td>
                <td class="description-td">h44f-h48r</td>
                <td class="description-td">h45f-h47r</td>
              </tr> <tr class="description-tr"> <td class="description-td">47</td>
                <td class="description-td">h52r</td>
                <td class="description-td">h44f-h52r</td>
                <td class="description-td">h46f-h51r</td>
              </tr> <tr class="description-tr"> <td class="description-td">48</td>
                <td class="description-td">h52r</td>
                <td class="description-td">h44f-h52r</td>
                <td class="description-td">h46f-h51r</td>
              </tr> <tr class="description-tr"> <td class="description-td">49</td>
                <td class="description-td">h52r</td>
                <td class="description-td">h44f-h52r</td>
                <td class="description-td">h46f-h51r</td>
              </tr> <tr class="description-tr"> <td class="description-td">50</td>
                <td class="description-td">h52r</td>
                <td class="description-td">h44f-h52r</td>
                <td class="description-td">h46f-h51r</td>
              </tr> <tr class="description-tr"> <td class="description-td">51</td>
                <td class="description-td">h53r</td>
                <td class="description-td">h47f-h53r</td>
                <td class="description-td">h49f-h52r</td>
              </tr> <tr class="description-tr"> <td class="description-td">53</td>
                <td class="description-td">h55r</td>
                <td class="description-td">h50f-h55r</td>
                <td class="description-td">h51f-h54r</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="left" id="FOO-00005" class="description-td" colspan="4"> <sup>a</sup>Primer sequences are available upon request.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0084">
      <tables id="TABLE-US-00004" num="00004"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="336pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 4</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Overview and frequency of the DMD-causing mutations in the Leiden DMD (LDMD) Database, theoretically</td>
              </tr> <tr class="description-tr"> <td class="description-td">correctable by skipping one of the 12 exons successfully targeted in this study</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="49pt" align="left"> </colspec> <colspec colname="1" colwidth="287pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Therapeutic for DMD-mutations:</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="49pt" align="center"> </colspec> <colspec colname="2" colwidth="63pt" align="center"> </colspec> <colspec colname="3" colwidth="63pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="center"> </colspec> <colspec colname="5" colwidth="63pt" align="center"> </colspec> <colspec colname="6" colwidth="56pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Deletions</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">No. of nonsense</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">(exons)</td>
                <td class="description-td">% of deletions</td>
                <td class="description-td">Duplications</td>
                <td class="description-td">% of duplications</td>
                <td class="description-td">mutations in</td>
              </tr> <tr class="description-tr"> <td class="description-td">Skippable exon</td>
                <td class="description-td">(exons)</td>
                <td class="description-td">in LDMD Database</td>
                <td class="description-td">(exons)</td>
                <td class="description-td">in LDMD Database</td>
                <td class="description-td">LDMD Database</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="49pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="63pt" align="center"> </colspec> <colspec colname="3" colwidth="63pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="63pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="56pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">2</td>
                <td class="description-td">3-7, 3-19, 3-21</td>
                <td class="description-td">2.9</td>
                <td class="description-td">2</td>
                <td class="description-td">9.0</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">29</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">5</td>
              </tr> <tr class="description-tr"> <td class="description-td">40</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">41</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">4</td>
              </tr> <tr class="description-tr"> <td class="description-td">42</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">0</td>
              </tr> <tr class="description-tr"> <td class="description-td">43</td>
                <td class="description-td">44, 44-47, 44-49,</td>
                <td class="description-td">3.7</td>
                <td class="description-td">43</td>
                <td class="description-td">3.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">44-51</td>
              </tr> <tr class="description-tr"> <td class="description-td">44</td>
                <td class="description-td">5-43, 14-43, 19-43,</td>
                <td class="description-td">7.8</td>
                <td class="description-td">44</td>
                <td class="description-td">3.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">30-43, 35-43,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">36-43, 40-43, 42-43,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">45, 45-54</td>
              </tr> <tr class="description-tr"> <td class="description-td">46</td>
                <td class="description-td">21-45, 45, 47-54,</td>
                <td class="description-td">5.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">47-56</td>
              </tr> <tr class="description-tr"> <td class="description-td">49</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">50</td>
                <td class="description-td">51, 51-53, 51-55</td>
                <td class="description-td">5.2</td>
                <td class="description-td">50</td>
                <td class="description-td">3.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">51</td>
                <td class="description-td">45-50, 47-50, 48-50,</td>
                <td class="description-td">17.5</td>
                <td class="description-td">51</td>
                <td class="description-td">1.5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">49-50, 50, 52,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">52-63</td>
              </tr> <tr class="description-tr"> <td class="description-td">53</td>
                <td class="description-td">10-52, 45-52, 46-52,</td>
                <td class="description-td">7.5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">47-52, 48-52, 49-52,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">50-52, 52</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0085">
      <tables id="TABLE-US-00005" num="00005"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="322pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 5</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Overview of the patients, the AONs and the primer sets used in Example 3</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="84pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <colspec colname="5" colwidth="42pt" align="left"> </colspec> <colspec colname="6" colwidth="42pt" align="left"> </colspec> <colspec colname="7" colwidth="42pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Targeted</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Primary PCR</td>
                <td class="description-td">Nested PCR</td>
              </tr> <tr class="description-tr"> <td class="description-td">Patients</td>
                <td class="description-td">Mutations</td>
                <td class="description-td">exons</td>
                <td class="description-td">AONs</td>
                <td class="description-td">RT-primers<sup>b</sup> </td>
                <td class="description-td">primer sets<sup>b</sup> </td>
                <td class="description-td">primer sets<sup>b</sup> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">DL 90.3</td>
                <td class="description-td">Nonsense mutation exon 43</td>
                <td class="description-td">Exon 43</td>
                <td class="description-td">h43AON2<sup>a</sup> </td>
                <td class="description-td">h48r</td>
                <td class="description-td">h41f-h48r</td>
                <td class="description-td">h42f-h47r</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Exon 44</td>
                <td class="description-td">h44AON1<sup>a</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td">DL470.2</td>
                <td class="description-td">Deletion exon 46-50</td>
                <td class="description-td">Exon 45</td>
                <td class="description-td">h45AON5</td>
                <td class="description-td">h53r</td>
                <td class="description-td">h42f-h53r</td>
                <td class="description-td">h43-h52r</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Exon 51</td>
                <td class="description-td">h51AON2<sup>a</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Exon 45</td>
                <td class="description-td">U-linker</td>
                <td class="description-td">h53r</td>
                <td class="description-td">h42f-h53r</td>
                <td class="description-td">h43f-h52r</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Exon 51</td>
                <td class="description-td">AON<sup>c</sup> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00006" class="description-td" colspan="7"> <sup>a</sup>Separate AON sequences were published previously (Aartsma-Rus, 2002).</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00007" class="description-td" colspan="7"> <sup>b</sup>Primer sequences available upon request.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" id="FOO-00008" class="description-td" colspan="7"> <sup>c</sup>U linker AON consists of h45AON5 linked to h51AON2 by ten uracils.</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1054058A1?cl=en">EP1054058A1</a></td><td class="patent-data-table-td patent-date-value">May 18, 2000</td><td class="patent-data-table-td patent-date-value">Nov 22, 2000</td><td class="patent-data-table-td ">Jcr Pharmaceuticals Co., Ltd.</td><td class="patent-data-table-td ">Pharmaceutical composition for treatment of Duchenne muscular dystrophy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1133993A1?cl=en">EP1133993A1</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2000</td><td class="patent-data-table-td patent-date-value">Sep 19, 2001</td><td class="patent-data-table-td ">Yvonne Hofmann</td><td class="patent-data-table-td ">Substances for the treatment of spinal muscular atrophy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1191097A1?cl=en">EP1191097A1</a></td><td class="patent-data-table-td patent-date-value">Sep 21, 2000</td><td class="patent-data-table-td patent-date-value">Mar 27, 2002</td><td class="patent-data-table-td ">Leids Universitair Medisch Centrum</td><td class="patent-data-table-td ">Induction of exon skipping in eukaryotic cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004083432A1?cl=en">WO2004083432A1</a></td><td class="patent-data-table-td patent-date-value">Mar 21, 2003</td><td class="patent-data-table-td patent-date-value">Sep 30, 2004</td><td class="patent-data-table-td ">Academisch Ziekenhuis Leiden</td><td class="patent-data-table-td ">Modulation of exon recognition in pre-mrna by interfering with the secondary rna structure</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Aartsma-Rus et al., Targeted exon skipping as a potential gene correction therapy for Duchenne muscular dystrophy, Neuromuscular Disorders, 2002, S71-S77, vol. 12.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">De Angelis et al., Chimeric snRNA molecules carrying antisense sequences against the splice junctions of exon 51 of the dystrophin pre-mRNA induce exon skipping and restoration of a dystrophin synthesis in delta 48-50 DMD cells, PNAS, Jul. 9, 2002, pp. 9456-9461, vol. 99, No. 14.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dickson et al., Screening for antisense modulation of dystrophin pre-mRNA splicing, Neuromuscul. Disord., 2002, S67-70, Suppl. 1.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dunckley et al., Modifications of splicing in the dystrophin gene in cultured Mdx muscle cells by antisense oligoribonucleotides, Human Molecular Genetics, 1995, pp. 1083-1090, vol. 5, No. 1.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lu et al., Massive Idiosyncratic Exon Skipping Corrects the Nonsense Mutation in Dystrophic Mouse Muscle and Produces Functional Revertant Fibers by Clonal Expansion, The Journal of Cell Biology, Mar. 6, 2000, pp. 985-995, vol. 148, No. 5.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mann et al., Antisense-induced exon skipping and synthesis of dystrophin in the mdx mouse, PNAS, Jan. 2, 2001, pp. 42-47, vol. 98, No. 1.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mann et al., Improved antisense oligonucleotide induced exon skipping in the mdx mouse model of muscular dystrophy, The Journal of Gene Medicine, 2002, pp. 644-654, vol. 4.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Suter et al., Double-target antisense U7 snRNAs promote efficient skipping of an aberrant exon in three human B-thalassemic mutations, Human Molecular Genetics, 1999, pp. 2415-2423, vol. 8, No. 13.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Van Deutekom et al., Antisense-induced exon skipping restores dystrophin expression in DMD patient derived muscle cells, Human Molecular Genetics, 2001, pp. 1547-1554, vol. 10, No. 15.</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7973015">US7973015</a></td><td class="patent-data-table-td patent-date-value">Mar 21, 2003</td><td class="patent-data-table-td patent-date-value">Jul 5, 2011</td><td class="patent-data-table-td ">Academisch Ziekenhuis Leiden</td><td class="patent-data-table-td ">Directing splicing of messenger RNA associated with dystrophin protein for diagnosis and treatment of muscular dystrophy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8450474">US8450474</a></td><td class="patent-data-table-td patent-date-value">Oct 11, 2011</td><td class="patent-data-table-td patent-date-value">May 28, 2013</td><td class="patent-data-table-td ">The University Of Western Australia</td><td class="patent-data-table-td ">Antisense oligonucleotides for inducing exon skipping and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8455634">US8455634</a></td><td class="patent-data-table-td patent-date-value">Jun 24, 2011</td><td class="patent-data-table-td patent-date-value">Jun 4, 2013</td><td class="patent-data-table-td ">The University Of Western Australia</td><td class="patent-data-table-td ">Antisense oligonucleotides for inducing exon skipping and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8455635">US8455635</a></td><td class="patent-data-table-td patent-date-value">Oct 11, 2011</td><td class="patent-data-table-td patent-date-value">Jun 4, 2013</td><td class="patent-data-table-td ">The University Of Western Australia</td><td class="patent-data-table-td ">Antisense oligonucleotides for inducing exon skipping and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8455636">US8455636</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 11, 2011</td><td class="patent-data-table-td patent-date-value">Jun 4, 2013</td><td class="patent-data-table-td ">The University Of Western Australia</td><td class="patent-data-table-td ">Antisense oligonucleotides for inducing exon skipping and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8476423">US8476423</a></td><td class="patent-data-table-td patent-date-value">Oct 11, 2011</td><td class="patent-data-table-td patent-date-value">Jul 2, 2013</td><td class="patent-data-table-td ">The University of Western Austrailia</td><td class="patent-data-table-td ">Antisense oligonucleotides for inducing exon skipping and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8486907">US8486907</a></td><td class="patent-data-table-td patent-date-value">Oct 11, 2011</td><td class="patent-data-table-td patent-date-value">Jul 16, 2013</td><td class="patent-data-table-td ">The University Of Western Australia</td><td class="patent-data-table-td ">Antisense oligonucleotides for inducing exon skipping and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524880">US8524880</a></td><td class="patent-data-table-td patent-date-value">Jul 15, 2010</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">The University Of Western Australia</td><td class="patent-data-table-td ">Antisense oligonucleotides for inducing exon skipping and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8609065">US8609065</a></td><td class="patent-data-table-td patent-date-value">Jan 8, 2010</td><td class="patent-data-table-td patent-date-value">Dec 17, 2013</td><td class="patent-data-table-td ">Prosensa Technologies B.V.</td><td class="patent-data-table-td ">Molecules for targeting compounds to various selected organs, tissues or tumor cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8637483">US8637483</a></td><td class="patent-data-table-td patent-date-value">Nov 12, 2010</td><td class="patent-data-table-td patent-date-value">Jan 28, 2014</td><td class="patent-data-table-td ">The University Of Western Australia</td><td class="patent-data-table-td ">Antisense molecules and methods for treating pathologies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20120022145">US20120022145</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 11, 2011</td><td class="patent-data-table-td patent-date-value">Jan 26, 2012</td><td class="patent-data-table-td ">The University Of Western Australia</td><td class="patent-data-table-td ">Antisense Oligonucleotides for Inducing Exon Skipping and Methods of Use Thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20120029058">US20120029058</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 11, 2011</td><td class="patent-data-table-td patent-date-value">Feb 2, 2012</td><td class="patent-data-table-td ">The University Of Western Australia</td><td class="patent-data-table-td ">Antisense Oligonucleotides for Inducing Exon Skipping and Methods of Use Thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20130085139">US20130085139</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 4, 2012</td><td class="patent-data-table-td patent-date-value">Apr 4, 2013</td><td class="patent-data-table-td ">Royal Holloway And Bedford New College</td><td class="patent-data-table-td ">Oligomers</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024500">536/24.5</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024310">536/24.31</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S325000">435/325</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S375000">435/375</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006150">435/6.15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006110">435/6.11</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0048000000">A61K48/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0038000000">A61K38/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021040000">C07H21/04</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015113000">C12N15/113</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/315">C12N2310/315</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K48/0016">A61K48/0016</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/3233">C12N2310/3233</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/3231">C12N2310/3231</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/321">C12N2310/321</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/111">C12N2310/111</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/3181">C12N2310/3181</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07H21/02">C07H21/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K48/00">A61K48/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K38/00">A61K38/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/346">C12N2310/346</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=zfa4BQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/113">C12N15/113</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">A61K48/00B2</span>, <span class="nested-value">C12N15/113</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Nov 12, 2012</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 27, 2012</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-3 IS CONFIRMED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 15, 2011</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20101109</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 3, 2008</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ACADEMISCH ZIEKENHUIS LEIDEN, NETHERLANDS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:VAN DEUTEKOM, JUDITH C.T.;REEL/FRAME:021477/0126</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20051126</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0OywlDho_zUzSVgz_ULpcYuT5Bkg\u0026id=zfa4BQABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U1KH0O-SSDYzzj6aI88_gkCUHRSlQ\u0026id=zfa4BQABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U0GJT9RrviZl9I6SInmFA7vp3Ne0w","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Modulation_of_exon_recognition_in_pre_mR.pdf?id=zfa4BQABERAJ\u0026output=pdf\u0026sig=ACfU3U0RcH9eMF1RWOC_3Mh5mUHXHZWiRA"},"sample_url":"http://www.google.com/patents/reader?id=zfa4BQABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>