<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7598035 - Ordering pairs of sequence tags via providing a population of pairs ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_4ff636b3d23669b7103f3b3a3a18b4cd/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_4ff636b3d23669b7103f3b3a3a18b4cd__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Ordering pairs of sequence tags via providing a population of pairs, removing duplicates, comparing sequence tags, ordering sequence tag of pair matches; physical maps and resriction fragment digestion"><meta name="DC.contributor" content="Stephen C. Macevicz" scheme="inventor"><meta name="DC.contributor" content="Solexa, Inc." scheme="assignee"><meta name="DC.date" content="2004-10-8" scheme="dateSubmitted"><meta name="DC.description" content="The invention provides a method for constructing a high resolution physical map of a polynucleotide. In accordance with the invention, nucleotide sequences are determined at the ends of restriction fragments produced by a plurality of digestions with a plurality of combinations of restriction endonucleases so that a pair of nucleotide sequences is obtained for each restriction fragment. A physical map of the polynucleotide is constructed by ordering the pairs of sequences by matching the identical sequences among the pairs."><meta name="DC.date" content="2009-10-6" scheme="issued"><meta name="DC.relation" content="EP:0593095:A1" scheme="references"><meta name="DC.relation" content="EP:0761822:A2" scheme="references"><meta name="DC.relation" content="US:4293652" scheme="references"><meta name="DC.relation" content="US:5102785" scheme="references"><meta name="DC.relation" content="US:5196328" scheme="references"><meta name="DC.relation" content="US:5409897" scheme="references"><meta name="DC.relation" content="US:5508169" scheme="references"><meta name="DC.relation" content="US:5604097" scheme="references"><meta name="DC.relation" content="US:5658736" scheme="references"><meta name="DC.relation" content="US:5667970" scheme="references"><meta name="DC.relation" content="US:5695937" scheme="references"><meta name="DC.relation" content="US:5710000" scheme="references"><meta name="DC.relation" content="US:5728524" scheme="references"><meta name="DC.relation" content="US:5804382" scheme="references"><meta name="DC.relation" content="US:5834201" scheme="references"><meta name="DC.relation" content="US:5846719" scheme="references"><meta name="DC.relation" content="US:5861252" scheme="references"><meta name="DC.relation" content="US:5968784" scheme="references"><meta name="DC.relation" content="US:6054276" scheme="references"><meta name="DC.relation" content="US:6136537" scheme="references"><meta name="DC.relation" content="WO:1996017082:A2" scheme="references"><meta name="DC.relation" content="WO:1998010095:A1" scheme="references"><meta name="DC.relation" content="WO:1998031838:A1" scheme="references"><meta name="citation_reference" content="Chen et al., &quot;Ordered shotgun sequencing, a strategy for integrated mapping and sequencing of YAC clones,&quot; Genomics, 17:651-656 (1993)."><meta name="citation_reference" content="Evans, &quot;Combinatoric strategies for genome mapping,&quot; BioEssays, 13:39-44 (1991)."><meta name="citation_reference" content="Green et al., &quot;Systematic generation of sequence-tagged sites for physical mapping of human chromosomes: application to the mapping of human chromosome 7 using yeast artificial chromosomes,&quot; Genomics, 11:548-564 (1991)."><meta name="citation_reference" content="Hasan et al., &quot;A novel multistep method for generating precise unidirectional deletions using Bsp MI, a class-IIS restriction enzyme,&quot; Gene, 50:55-62 (1986)."><meta name="citation_reference" content="Hasan et al., &quot;An Mbo II/Fok I trimming plasmid allowing consecutive cycles of precise 1- to 12-base-pair deletions in cloned DNA,&quot; Gene, 82:305-311 (1989)."><meta name="citation_reference" content="Howard et al., Biotechniques, 7:940-942 (1989)."><meta name="citation_reference" content="Hudson et al., &quot;An STS-based map of the human genome,&quot; Science, 270:1945-1954 (1995)."><meta name="citation_reference" content="Kato, &quot;Description of the entire mRNA population by a 3&#39; end cDNA fragment generated by class IIs restriction enzymes,&quot; Nucleic Acids Research, 23:3685-3690 (1995)."><meta name="citation_reference" content="Kato, &quot;RNA fingerprinting by molecular indexing,&quot; Nucleic Acids Research, 24:394-395 (1996)."><meta name="citation_reference" content="Michiels et al., &quot;Molecular approaches to genome analysis: a strategy for the construction of ordered overlapping clone libraries,&quot; CABIOS, 3: 203-210 (1987)."><meta name="citation_reference" content="New England Biolabs 96/97 Catalog, p. 115. Published by New England Biolabs. Inc., 32 Tozer Road, Beverly, MA 01915-5599."><meta name="citation_reference" content="New England Biolabs 96/97 Catalog, pp. 15, 18, 22, 23 and 35. Published by New England Biolabs. Inc., 32 Tozer Road, Beverly, MA 01915-5599."><meta name="citation_reference" content="New England Biolabs 96/97 Catalog, pp. 19, 37, 108 and 109. Published by New England Biolabs. Inc., 32 Tozer Road, Beverly, MA 01915-5599."><meta name="citation_reference" content="New England Biolabs 96/97 Catalog, pp. 36 and 50. Published by New England Biolabs. Inc., 32 Tozer Road, Beverly, MA 01915-5599."><meta name="citation_reference" content="Olson et al., &quot;Random-clone strategy for genomic restriction mapping in yeast,&quot; Proc.-Natl. Acad. Sci., 83:7826-7830 (1986)."><meta name="citation_reference" content="Poustka and Lehrach, &quot;Chromosome jumping: a long range cloning technique,&quot; in Genetic Engineering: Principles and Methods, J.K. Setlow, Editor, 10:169-193 (1988)."><meta name="citation_reference" content="Poustka and Lehrach, &quot;Jumping libraries and linking libraries: the next generation of molecular tools in mammalian genetics,&quot; Trends in Genetics, 2:174-179 (1986)."><meta name="citation_reference" content="Roach et al., &quot;Pairwise end sequencing: A unified approach to genomic mapping and sequencing,&quot; Genomics, 26:345-353 (1995)."><meta name="citation_reference" content="Sapolsky et al., &quot;Mapping genomic library clones using oligonucleotide arrays,&quot; Genomics, 33:445-456 (1996)."><meta name="citation_reference" content="Smith et al., &quot;A simple method for DNA restriction site mapping,&quot; Nucleic Acids Research, 3: 2387-2398 (1976)."><meta name="citation_reference" content="Smith et al., &quot;Genomic sequence sampling: a strategy for high resolution sequence-based physical mapping of complex genomes,&quot; Nature Genetics, 7:40-47 (1994)."><meta name="citation_reference" content="Velculescu et al., &quot;Serial analysis of gene expression,&quot; Science, 270:484-487 (1995)."><meta name="citation_reference" content="Wong et al., &quot;Multiple-complete-digest restriction fragment mapping: Generating sequence-ready maps for large-scale DNA sequencing,&quot; Proc. Natl. Acad. Sci., 94:5225-5230 (1997)."><meta name="citation_reference" content="Yi et al., &quot;Construction of restriction fragment maps of 50- to 100-kilobase DNA&quot;, Methods in Enzymology, 218:651-671 (1993)."><meta name="citation_patent_number" content="US:7598035"><meta name="citation_patent_application_number" content="US:10/962,337"><link rel="canonical" href="http://www.google.com/patents/US7598035"/><meta property="og:url" content="http://www.google.com/patents/US7598035"/><meta name="title" content="Patent US7598035 - Ordering pairs of sequence tags via providing a population of pairs, removing duplicates, comparing sequence tags, ordering sequence tag of pair matches; physical maps and resriction fragment digestion"/><meta name="description" content="The invention provides a method for constructing a high resolution physical map of a polynucleotide. In accordance with the invention, nucleotide sequences are determined at the ends of restriction fragments produced by a plurality of digestions with a plurality of combinations of restriction endonucleases so that a pair of nucleotide sequences is obtained for each restriction fragment. A physical map of the polynucleotide is constructed by ordering the pairs of sequences by matching the identical sequences among the pairs."/><meta property="og:title" content="Patent US7598035 - Ordering pairs of sequence tags via providing a population of pairs, removing duplicates, comparing sequence tags, ordering sequence tag of pair matches; physical maps and resriction fragment digestion"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("wePoU9zcCo2YyATdwYD4Ag"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407291699.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("LUX"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("wePoU9zcCo2YyATdwYD4Ag"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407291699.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("LUX"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7598035?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7598035"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=oyTnBQABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7598035&amp;usg=AFQjCNFApSLi89hMxwc2A3OpI97f8fqfcw" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7598035.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7598035.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20050042673"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7598035"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7598035" style="display:none"><span itemprop="description">The invention provides a method for constructing a high resolution physical map of a polynucleotide. In accordance with the invention, nucleotide sequences are determined at the ends of restriction fragments produced by a plurality of digestions with a plurality of combinations of restriction endonucleases...</span><span itemprop="url">http://www.google.com/patents/US7598035?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7598035 - Ordering pairs of sequence tags via providing a population of pairs, removing duplicates, comparing sequence tags, ordering sequence tag of pair matches; physical maps and resriction fragment digestion</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7598035 - Ordering pairs of sequence tags via providing a population of pairs, removing duplicates, comparing sequence tags, ordering sequence tag of pair matches; physical maps and resriction fragment digestion" title="Patent US7598035 - Ordering pairs of sequence tags via providing a population of pairs, removing duplicates, comparing sequence tags, ordering sequence tag of pair matches; physical maps and resriction fragment digestion"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7598035 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 10/962,337</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Oct 6, 2009</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Oct 8, 2004</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Feb 23, 1998</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6054276">US6054276</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6720179">US6720179</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8288095">US8288095</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8557521">US8557521</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20040082001">US20040082001</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20050042673">US20050042673</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20100136548">US20100136548</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20130052651">US20130052651</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20140057274">US20140057274</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">10962337, </span><span class="patent-bibdata-value">962337, </span><span class="patent-bibdata-value">US 7598035 B2, </span><span class="patent-bibdata-value">US 7598035B2, </span><span class="patent-bibdata-value">US-B2-7598035, </span><span class="patent-bibdata-value">US7598035 B2, </span><span class="patent-bibdata-value">US7598035B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Stephen+C.+Macevicz%22">Stephen C. Macevicz</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Solexa,+Inc.%22">Solexa, Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7598035.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7598035.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7598035.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (23),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (24),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (6),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (34),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (5)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7598035&usg=AFQjCNGyvy85aaszZpv9j24fLLdsnL83sg">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7598035&usg=AFQjCNGGCRcn5DALlVUQ8WlJM7FE-aj0cw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7598035B2%26KC%3DB2%26FT%3DD&usg=AFQjCNE1qbdJVs6zlWzLFAvi8P-F1a9p4A">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT130070143" lang="EN" load-source="ifi">Ordering pairs of sequence tags via providing a population of pairs, removing duplicates, comparing sequence tags, ordering sequence tag of pair matches; physical maps and resriction fragment digestion </invention-title></span><br><span class="patent-number">US 7598035 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA62624654" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">The invention provides a method for constructing a high resolution physical map of a polynucleotide. In accordance with the invention, nucleotide sequences are determined at the ends of restriction fragments produced by a plurality of digestions with a plurality of combinations of restriction endonucleases so that a pair of nucleotide sequences is obtained for each restriction fragment. A physical map of the polynucleotide is constructed by ordering the pairs of sequences by matching the identical sequences among the pairs.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(4)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7598035B2/US07598035-20091006-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7598035B2/US07598035-20091006-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7598035B2/US07598035-20091006-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7598035B2/US07598035-20091006-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7598035B2/US07598035-20091006-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7598035B2/US07598035-20091006-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7598035B2/US07598035-20091006-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7598035B2/US07598035-20091006-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(5)</span></span></div><div class="patent-text"><div mxw-id="PCLM23096159" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A method of forming a linked pair of segments from each fragment of a population of polynucleotide fragments, wherein each said fragment is in a vector, said vector comprises two type IIs restriction endonuclease recognition sites, one of said type IIs restriction endonuclease recognition sites is adjacent to one end of the fragment and another of said type IIs restriction endonuclease recognition sites is adjacent to another end of the fragment, and one or more type IIs restriction endonucleases recognizing said type IIs restriction endonuclease recognition sites have two cleavage sites within the interior of the fragment, the method comprising:
<div class="claim-text">treating each said vector and fragment with said one or more type IIs restriction endonucleases recognizing said type IIs restriction recognition sites, such that each fragment is cleaved at said cleavage sites, thereby removing the region of the fragment between said cleavage sites, whereby a pair of segments from said fragment remain connected to each other via said vector after said removing and each of said segments has a cleaved end; and</div>
<div class="claim-text">ligating together the cleaved ends produced by said removing thereby forming a linked pair of segments from each said fragment.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the two type IIs recognition sites are recognized by the same type IIs restriction endonuclease.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. A method of forming linearized vectors having pairs of segments from a population of polynucleotide fragments, the method comprising the steps of:
<div class="claim-text">i) inserting each of said polynucleotide fragments from said population into a vector having two type IIs restriction endonuclease recognition sites and forming vectors containing said polynucleotide fragments such that each of the vectors containing said polynucleotide fragments comprises a fragment from said population of polynucleotide fragments, one of said type IIs restriction endonuclease recognition sites in each of the vectors containing said polynucleotide fragments is adjacent to one end of the fragment and another of said type IIs endonuclease recognition sites in each of the vectors containing said polynucleotide fragments is adjacent to another end of the fragment, and one or more type IIs restriction endonucleases recognizing said type IIs restriction endonuclease recognition sites have two cleavage sites within the interior of the fragment; and</div>
<div class="claim-text">ii) treating each of the vectors containing said polynucleotide fragments with said one or more type IIs restriction endonucleases recognizing the type IIs restriction endonuclease recognition sites and producing linearized vectors having pairs of segments wherein each of said linearized vectors having pairs of segments has a pair of segments of the fragment, one of said pair of segments is located at one end of each of the linearized vectors, and another of said pair of segments is located at another end of each of the linearized vectors.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="CLM-00003">claim 3</claim-ref>, wherein said type IIs recognition sites are recognized by the same type IIs restriction endonuclease.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. The method of <claim-ref idref="CLM-00003">claim 3</claim-ref>, further comprising the step of re-circularizing each of the linearized vectors, by ligating the pair of segments together. </div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES28074891" lang="EN" load-source="patent-office" class="description">
    <p num="p-0002">This application is a divisional of application Ser. No. 10/706,118 filed on Nov. 12, 2003, now abandoned, which is a divisional of application Ser. No. 09/549,748 filed on Apr. 14, 2000, now U.S. Pat. No. 6,720,179, which is a divisional of application Ser. No. 09/028,128 filed on Feb. 23, 1998, now U.S. Pat. No. 6,054,276, all of which are incorporated herein by reference in their entirety.</p>
    <heading>FIELD OF THE INVENTION</heading> <p num="p-0003">The invention relates generally to methods for construction physical maps of DNA, especially genomic DNA, and more particularly, to a method of providing high resolution physical maps by sequence analysis of concatenations of segments of restriction fragment ends.</p>
    <heading>BACKGROUND</heading> <p num="p-0004">Physical maps of one or more large pieces of DNA, such as a genome or chromosome, consist of an ordered collection of molecular landmarks that may be used to position, or map, a smaller fragment, such as clone containing a gene of interest, within the larger structure, e.g. U.S. Department of Energy, “Primer on Molecular Genetics,” from Human Genome 1991-92 Program Report; and Los Alamos Science, 20: 112-122 (1992). An important goal of the Human Genome Project has been to provide a series of genetic and physical maps of the human genome with increasing resolution, i.e. with reduced distances in basepairs between molecular landmarks, e.g. Murray et al, Science, 265: 2049-2054 (1994); Hudson et al, Science, 270: 1945-1954 (1995); Schuler et al, Science, 274: 540-546 (1996); and so on. Such maps have great value not only in furthering our understanding of genome organization, but also as tools for helping to fill contig gaps in large-scale sequencing projects and as tools for helping to isolate disease-related genes in positional cloning projects, e.g. Rowen et al, pages 167-174, in Adams et al, editors, Automated DNA Sequencing and Analysis (Academic Press, New York, 1994); Collins, Nature Genetics, 9: 347-350 (1995); Rossiter and Caskey, Annals of Surgical Oncology, 2: 14-25 (1995); and Schuler et al (cited above). In both cases, the ability to rapidly construct high-resolution physical maps of large pieces of genomic DNA is highly desirable.</p>
    <p num="p-0005">Two important approaches to genomic mapping include the identification and use of sequence tagged sites (STS's), e.g. Olson et al, Science, 245: 1434-1435 (1989); and Green et al, PCR Methods and Applications, 1: 77-90 (1991), and the construction and use of jumping and linking libraries, e.g. Collins et al, Proc. Natl. Acad. Sci., 81: 6812-6816 (1984); and Poustka and Lehrach, Trends in Genetics, 2: 174-179 (1986). The former approach makes maps highly portable and convenient, as maps consist of ordered collections of nucleotide sequences that allow application without having to acquire scarce or specialized reagents and libraries. The latter approach provides a systematic means for identifying molecular landmarks spanning large genetic distances and for ordering such landmarks via hybridization assays with members of a linking library.</p>
    <p num="p-0006">Unfortunately, these approaches to mapping genomic DNA are difficult and laborious to implement. It would be highly desirable if there was an approach for constructing physical maps that combined the systematic quality of the jumping and linking libraries with the convenience and portability of the STS approach.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0007">Accordingly, an object of my invention is to provide methods and materials for constructing high resolution physical maps of genomic DNA.</p>
    <p num="p-0008">Another object of my invention is to provide a method of ordering restriction fragments from multiple enzyme digests by aligning matching sequences of their ends.</p>
    <p num="p-0009">Still another object of my invention is to provide a high resolution physical map of a target polynucleotide that permits directed sequencing of the target polynucleotide with the sequences of the map.</p>
    <p num="p-0010">Another object of my invention is to provide vectors for excising ends of restriction fragments for concatenation and sequencing.</p>
    <p num="p-0011">Still another object of my invent is to provide a method monitoring the expression of genes.</p>
    <p num="p-0012">A further object of my invention is to provide physical maps of genomic DNA that consist of an ordered collection of nucleotide sequences spaced at an average distance of a few hundred to a few thousand bases.</p>
    <p num="p-0013">My invention achieves these and other objects by providing methods and materials for determining the nucleotide sequences of both ends of restriction fragments obtained from multiple enzymatic digests of a target polynucleotide, such as a fragment of a genome, or chromosome, or an insert of a cosmid, BAC, YAC, or the like. In accordance with the invention, a polynucleotide is separately digested with different combinations of restriction endonucleases and the ends of the restriction fragments are sequenced so that pairs of sequences from each fragment are produced. A physical map of the polynucleotide is constructed by ordering the pairs of sequences by matching the identical sequences among such pairs resulting from all of the digestions.</p>
    <p num="p-0014">In the preferred embodiment, a polynucleotide is mapped by the following steps: (a) providing a plurality of populations of restriction fragments, the restriction fragments of each population having ends defined by digesting the polynucleotide with a plurality of combinations of restriction endonucleases; (b) determining the nucleotide sequence of a portion of each end of each restriction fragment of each population so that a pair of nucleotide sequences is obtained for each restriction fragment of each population; and (c) ordering the pairs of nucleotide sequences by matching the nucleotide sequences between pairs to form a map of the polynucleotide.</p>
    <p num="p-0015">Another aspect of the invention is the monitoring gene expression by providing pairs of segments excised from cDNAs. In this embodiment, segments from each end of each cDNA of a population of cDNAs are ligated together to form pairs, which serve to identify their associated cDNAs. Concatenations of such pairs are sequenced by conventional techniques to provide information on the relative frequencies of expression in the population.</p>
    <p num="p-0016">The invention provides a means for generating a high density physical map of target polynucleotides based on the positions of the restriction sites of predetermined restriction endonucleases. Such physical maps provide many advantages, including a more efficient means for directed sequencing of large DNA fragments, the positioning of expression sequence tags and cDNA sequences on large genomic fragments, such as BAC library inserts, thereby making positional candidate mapping easier; and the like.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p num="p-0017"> <figref idrefs="DRAWINGS">FIG. 1</figref> graphically illustrates the concept of a preferred embodiment of the invention.</p>
      <p num="p-0018"> <figref idrefs="DRAWINGS">FIG. 2</figref> provides a diagram of a vector for forming pairs of nucleotide sequences in accordance with a preferred embodiment of the invention.</p>
      <p num="p-0019"> <figref idrefs="DRAWINGS">FIG. 3</figref> illustrates a scheme for carrying out the steps of a preferred embodiment of the invention.</p>
      <p num="p-0020"> <figref idrefs="DRAWINGS">FIG. 4</figref> illustrates locations on yeast chromosome 1 where sequence information is provided in a physical map based on digestions with Hind III, Eco RI, and Xba I in accordance with the invention.</p>
    </description-of-drawings> <heading>DEFINITIONS</heading> <p num="p-0021">As used herein, the process of “mapping” a polynucleotide means providing a ordering, or series, of sequenced segments of the polynucleotide that correspond to the actual ordering of the segments in the polynucleotide. For example, the following set of five-base sequences is a map of the polynucleotide below (SEQ ID NO: 8), which has the ordered set of sequences making up the map underlined:</p>
    <p num="p-0022">
      <tables id="TABLE-US-00001" num="00001"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <colspec colname="2" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">              (gggtc, ttatt, aacct, catta, ccgga)</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT<u>GGGTC</u>AACAAATTACC<u>TTATT</u>GT<u>AACCT</u>TCG<u>CATTA</u>G<u>CCGGA</u>GCCT</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0023">The term “oligonucleotide” as used herein includes linear oligomers of natural or modified monomers or linkages, including deoxyribonucleosides, ribonucleosides, and the like, capable of specifically binding to a target polynucleotide by way of a regular pattern of monomer-to-monomer interactions, such as Watson-Crick type of base pairing, base stacking, Hoogsteen or reverse Hoogsteen types of base pairing, or the like. Usually monomers are linked by phosphodiester bonds or analogs thereof to form oligonucleotides ranging in size from a few monomeric units, e.g. 3-4, to several tens of monomeric units, e.g. 40-60. Whenever an oligonucleotide is represented by a sequence of letters, such as “ATGCCTG,” it will be understood that the nucleotides are in 5′→3′ order from left to right and that “A” denotes deoxyadenosine, “C” denotes deoxycytidine, “G” denotes deoxyguanosine, and “T” denotes thymidine, unless otherwise noted. Usually oligonucleotides comprise the four natural nucleotides; however, they may also comprise non-natural nucleotide analogs. It is clear to those skilled in the art when oligonucleotides having natural or non-natural nucleotides may be employed, e.g. where processing by enzymes is called for, usually oligonucleotides consisting of natural nucleotides are required.</p>
    <p num="p-0024">“Perfectly matched” in reference to a duplex means that the poly- or oligonucleotide strands making up the duplex form a double stranded structure with one other such that every nucleotide in each strand undergoes Watson-Crick basepairing with a nucleotide in the other strand. The term also comprehends the pairing of nucleoside analogs, such as deoxyinosine, nucleosides with 2-aminopurine bases, and the like, that may be employed. In reference to a triplex, the term means that the triplex consists of a perfectly matched duplex and a third strand in which every nucleotide undergoes Hoogsteen or reverse Hoogsteen association with a basepair of the perfectly matched duplex.</p>
    <p num="p-0025">As used herein, “nucleoside” includes the natural nucleosides, including 2′-deoxy and 2′-hydroxyl forms, e.g. as described in Kornberg and Baker, DNA Replication, 2nd Ed. (Freeman, San Francisco, 1992). “Analogs” in reference to nucleosides includes synthetic nucleosides having modified base moieties and/or modified sugar moieties, e.g. described by Scheit, Nucleotide Analogs (John Wiley, New York, 1980); Uhlman and Peyman, Chemical Reviews, 90: 543-584 (1990), or the like, with the only proviso that they are capable of specific hybridization. Such analogs include synthetic nucleosides designed to enhance binding properties, reduce complexity, increase specificity, and the like.</p>
    <p num="p-0026">As used herein, the term “complexity” in reference to a population of polynucleotides means the number of different species of polynucleotide present in the population.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p num="p-0027">In accordance with the present invention, segments of nucleotides at each end of restriction fragments produced from multiple digestions of a polynucleotide are sequenced and used to arrange the fragments into a physical map. Such a physical map consists of an ordered collection of the nucleotide sequences of the segments immediately adjacent to the cleavage sites of the endonucleases used in the digestions. Preferably, after each digestion, segments are removed from the ends of each restriction fragment by cleavage with a type IIs restriction endonuclease. Excised segments from the same fragment are ligated together to form a pair of segments. Preferably, collections of such pairs are concatenated by ligation, cloned, and sequenced using conventional techniques.</p>
    <p num="p-0028">The concept of the invention is illustrated in <figref idrefs="DRAWINGS">FIG. 1</figref> for an embodiment which employs three restriction endonucleases: r, q, and s. Polynucleotide (<b>50</b>) has recognition sites (r<sub>1</sub>, r<sub>2</sub>, r<sub>3</sub>, and r<sub>4</sub>) for restriction endonucleases r, recognition sites (q<sub>1 </sub>through q<sub>4</sub>) for restriction endonuclease q, and recognition sites (s<sub>1 </sub>through s<sub>5</sub>) for restriction endonuclease s. In accordance with the preferred embodiment, polynucleotide (<b>50</b>) is separately digested with r and s, q and s, and r and q to produce three populations of restriction fragments (<b>58</b>), (<b>60</b>), and (<b>62</b>), respectively. Segments adjacent to the ends of each restriction fragment are sequenced to form sets of pairs (<b>52</b>), (<b>54</b>), and (<b>56</b>) of nucleotide sequences, which for sake of illustration are shown directly beneath their corresponding restriction fragments in the correct order. Pairs of sequences from all three sets are ordered by matching sequences between pairs as shown (<b>70</b>). A nucleotide sequence (<b>72</b>) from a first pair is matched with a sequence (<b>74</b>) of a second pair whose other sequence (<b>76</b>), in turn, is matched with a sequence (<b>78</b>) of a third pair. The matching continues, as (<b>80</b>) is matched with (<b>82</b>), (<b>84</b>) with (<b>86</b>), (<b>88</b>) with (<b>90</b>), and so on, until the maximum number of pairs are included. It is noted that some pairs (<b>92</b>) do not contribute to the map. These correspond to fragments having the same restriction site at both ends. In other word, they correspond to situations where there are two (or more) consecutive restriction sites of the same type without other sites in between, e.g. s<sub>3 </sub>and s<sub>4 </sub>in this example. Preferably, algorithms used for assembling a physical map from the pairs of sequences can eliminate pairs having identical sequences.</p>
    <p num="p-0029">Generally, a plurality of enzymes is employed in each digestion. Preferably, at least three distinct recognition sites are used. This can be accomplished by using three or more restriction endonucleases, such as Hind III, Eco RI, and Xba I, which recognize different nucleotide sequences, or by using restriction endonucleases recognizing the same nucleotide sequence, but which have different methylation sensitivities. That is, it is understood that a different “recognition site” may be different solely by virtue of a different methylation state. Preferably, a set of at least three recognition endonucleases is employed in the method of the invention. From this set a plurality of combinations of restriction endonucleases is formed for separate digestion of a target polynucleotide. Preferably, the combinations are “n−1” combinations of the set. In other words, for a set of n restriction endonucleases, the preferred combinations are all the combinations of n−1 restriction endonucleases. For example, as illustrated in <figref idrefs="DRAWINGS">FIG. 1</figref> where a set of three restriction endonucleases (r, q, and s) are employed, the n−1 combinations are (r, q), (r, s), and (q, s). Likewise, if four restriction endonucleases (r, q, s, and w) are employed, the n−1 combinations are (r, q, s), (r, q, w), (r, s, w), and (q, s, w). It is readily seen that where a set of n restriction endonucleases are employed the plurality of n−1 combinations is n.</p>
    <p num="p-0030">Preferably, the method of the invention is carried out using a vector, such as that illustrated in <figref idrefs="DRAWINGS">FIG. 2</figref>. The vector is readily constructed from commercially available materials using conventional recombinant DNA techniques, e.g. as disclosed in Sambrook et al, Molecular Cloning, Second Edition (Cold Spring Harbor Laboratory, New York, 1989). Preferably, pUC-based plasmids, such as pUC19, or λ-based phages, such as λ ZAP Express (Stratagene Cloning Systems, La Jolla, Calif.), or like vectors are employed. Important features of the vector are recognition sites (<b>204</b>) and (<b>212</b>) for two type IIs restriction endonucleases that flank restriction fragment (<b>208</b>). For convenience, the two type IIs restriction enzymes are referred to herein as “IIs<sub>1</sub>” and “IIs<sub>2</sub>”, respectively. IIs<sub>1 </sub>and IIs<sub>2 </sub>may be the same or different. Recognition sites (<b>204</b>) and (<b>212</b>) are oriented so that the cleavage sites of IIs<sub>1 </sub>and IIs<sub>2 </sub>are located in the interior of restriction fragment (<b>208</b>). In other words, taking the 5′ direction as “upstream” and the 3′ direction as “downstream,” the cleavage site of IIs<sub>1 </sub>is downstream of its recognition site and the cleavage site of IIs<sub>2 </sub>is upstream of its recognition site. Thus, when the vector is cleaved with IIs<sub>1 </sub>and IIs<sub>2 </sub>two segments (<b>218</b>) and (<b>220</b>) of restriction fragment (<b>208</b>) remain attached to the vector. The vector is then re-circularized by ligating the two ends together, thereby forming a pair of segments. If such cleavage results in one or more single stranded overhangs, i.e. one or more non-blunt ends, then the ends are preferably rendered blunt prior to re-circularization, for example, by digesting the protruding strand with a nuclease such as Mung bean nuclease, or by extending a 3′ recessed strand, if one is produced in the digestion. The ligation reaction for re-circularization is carried out under conditions that favor the formation of covalent circles rather than concatemers of the vector. Preferably, the vector concentration for the ligation is between about 0.4 and about 4.0 μg/ml of vector DNA, e.g. as disclosed in Collins et al, Proc. Natl. Acad. Sci., 81: 6812-6812 (1984), for α-based vectors. For vectors of different molecular weight, the concentration range is adjusted appropriately.</p>
    <p num="p-0031">In the preferred embodiments, the number of nucleotides identified depends on the “reach” of the type IIs restriction endonucleases employed. “Reach” is the amount of separation between a recognition site of a type IIs restriction endonuclease and its cleavage site, e.g. Brenner, U.S. Pat. No. 5,559,675. The conventional measure of reach is given as a ratio of integers, such as “(16/14)”, where the numerator is the number of nucleotides from the recognition site in the 5′→3′ direction that cleavage of one strand occurs and the denominator is the number of nucleotides from the recognition site in the 3′→5′ direction that cleavage of the other strand occurs. Preferred type IIs restriction endonucleases for use as IIs<sub>1 </sub>and IIs<sub>2 </sub>in the preferred embodiment include the following: Bbv I, Bce 83 I, Bcef I, Bpm I, Bsg I, BspLU II III, Bst 71 I, Eco 57 I, Fok I, Gsu I, Hga I, Mme I, and the like. In the preferred embodiment, a vector is selected which does not contain a recognition site, other than (<b>204</b>) and (<b>212</b>), for the type IIs enzyme(s) used to generate pairs of segments; otherwise, re-circularization cannot be carried out.</p>
    <p num="p-0032">Preferably, a type IIs restriction endonuclease for generating pairs of segments has as great a reach as possible to maximize the probability that the nucleotide sequences of the segments are unique. This in turn maximizes the probability that a unique physical map can be assembled. If the target polynucleotide is a bacterial genome of 1 megabase, for a restriction endonuclease with a six basepair recognition site, about 250 fragments are generated (or about 500 ends) and the number of nucleotides determined could be as low as five or six, and still have a significant probability that each end sequence would be unique. Preferably, for polynucleotides less than or equal to 10 megabases, at least 8 nucleotides are determined in the regions adjacent to restriction sites, when a restriction endonuclease having a six basepair recognition site is employed. Generally for polynucleotides less than or equal to 10 megabases, 9-12 nucleotides are preferably determined to ensure that the end sequences are unique. In the preferred embodiment, type IIs enzymes having a (16/14) reach effectively provide 9 bases of unique sequence (since blunting reduces the number of bases to 14 and 5 bases are part of the recognition sites (<b>206</b>) or (<b>210</b>)). In a polynucleotide having a random sequence of nucleotides, a 9-mer appears on average about once every 262,000 bases. Thus, 9-mer sequences are quite suitable for uniquely labeling restriction fragments of a target polynucleotide corresponding to a typical yeast artificial chromosome (YACs) insert, i.e. 100-1000 kilobases, bacterial artificial chromosome (BAC) insert, i.e. 50-250 kilobases, and the like.</p>
    <p num="p-0033">Immediately adjacent to IIs sites (<b>204</b>) and (<b>212</b>) are restriction sites (<b>206</b>) and (<b>210</b>), respectively that permit restriction fragment (<b>208</b>) to be inserted into the vector. That is, restriction site (<b>206</b>) is immediately downstream of (<b>204</b>) and (<b>210</b>) is immediately upstream of (<b>212</b>). Preferably, sites (<b>204</b>) and (<b>206</b>) are as close together as possible, even overlapping, provided type IIs site (<b>206</b>) is not destroyed upon cleavage with the enzymes for inserting restriction fragment (<b>208</b>). This is desirable because the recognition site of the restriction endonuclease used for generating the fragments occurs between the recognition site and cleavage site of type IIs enzyme used to remove a segment for sequencing, i.e. it occurs within the “reach” of the type IIs enzyme. Thus, the closer the recognition sites, the larger the piece of unique sequence can be removed from the fragment. The same of course holds for restriction sites (<b>210</b>) and (<b>212</b>). Preferably, whenever the vector employed is based on a pUC plasmid, restriction sites (<b>206</b>) and (<b>210</b>) are selected from either the restriction sites of polylinker region of the pUC plasmid or from the set of sites which do not appeal in the pUC. Such sites include Eco RI, Apo I, Ban II, Sac I, Kpn I, Acc65 I, Ava I, Xma I, Sma I, Bam HI, Xba I, Sal I, Hinc II, Acc I, BspMI, Pst I, Sse8387 I, Sph I, Hind III, Afl II, Age I, Bsp120 I, Asc I, Bbs I, Bcl I, Bgl II, Blp I, BsaA I, Bsa BI, Bse RI, Bsm I, Cla I, Bsp EI, BssH II, Bst BI, BstXI, Dra III, Eag I, Eco RV, Fse I, Hpa I, Mfe I, Nae I, Nco I, Nhe I, Not I, Nru I, Pac I, Xho I, Pme I, Sac II, Spe I, Stu I, and the like. Preferably, six-nucleotide recognition sites (i.e. “6-cutters”) are used, and more preferably, 6-cutters leaving four-nucleotide protruding strands are used.</p>
    <p num="p-0034">Preferably, the vectors contain primer binding sites (<b>200</b>) and (<b>216</b>) for primers p<sub>1 </sub>and p<sub>2</sub>, respectively, which may be used to amplify the pair of segments by PCR after re-circularization. Recognition sites (<b>202</b>) and (<b>214</b>) are for restriction endonucleases w<sub>1 </sub>and w<sub>2</sub>, which are used to cleave the pair of segments from the vector after amplification. Preferably, w<sub>1 </sub>and w<sub>2</sub>, which may be the same or different, are type IIs restriction endonucleases whose cleavage sites correspond to those of (<b>206</b>) and (<b>210</b>), thereby removing surplus, or non-informative, sequence (such as the recognition sites (<b>204</b>) and (<b>212</b>)) and generating protruding ends that permit concatenation of the pairs of segments.</p>
    <p num="p-0035"> <figref idrefs="DRAWINGS">FIG. 3</figref> illustrates steps in a preferred method using vectors of <figref idrefs="DRAWINGS">FIG. 2</figref>. Genomic or other DNA (<b>400</b>) is obtained using conventional techniques, e.g. Herrmann and Frischauf, Methods in Enzymology, 152: 180-183 (1987); Frischauf, Methods in Enzymology, 152: 183-199 (1987), or the like, after which it is divided (302) into aliquots that are separately digested (<b>310</b>) with combinations restriction endonucleases, as shown in <figref idrefs="DRAWINGS">FIG. 3</figref> for the n−1 combinations of the set of enzymes r, s, and q. Preferably, the resulting fragments are treated with a phosphatase to prevent ligation of the genomic fragments with one another before or during insertion into a vector. Restriction fragments are inserted (<b>312</b>) into vectors designed with cloning sites to specifically accept the fragments. That is, fragments digested with r and s are inserted into a vector that accepts r-s fragments. Fragments having the same ends, e.g. r-r and s-s, are not cloned since information derived from them does not contribute to the map. r-s fragments are of course inserted into the vector in both orientations. Thus, for a set of three restriction endonucleases, only three vectors are required, e.g. one each for accepting r-s, r-q, and s-q fragments. Likewise, for a set of four restriction endonucleases, e.g. r, s, q, and t, only six vectors are required, one each for accepting r-s, r-q, r-t, s-q, s-t, and q-t fragments.</p>
    <p num="p-0036">After insertion, a suitable host is transformed with the vectors and cultured, i.e. expanded (<b>314</b>), using conventional techniques. Transformed host cells are then selected, e.g. by plating and picking colonies using a standard marker, e.g. β-glactosidase/X-gal. A large enough sample of transformed host cells is taken to ensure that every restriction fragment is present for analysis with a reasonably large probability. This is similar to the problem of ensuring representation of a clone of a rare mRNA in a cDNA library, as discussed in Sambrook et al, Molecular Cloning, Second Edition (Cold Spring Harbor Laboratory, New York, 1989), and like references. Briefly, the number of fragments, N, that must be in a sample to achieve a given probability, P, of including a given fragment is the following: N=ln(1−P)/ln(1−f), where f is the frequency of the fragment in the population. Thus, for a population of 500 restriction fragments, a sample containing 3454 vectors will include at least one copy of each fragment (i.e. a complete set) with a probability of 99.9%; and a sample containing 2300 vectors will include at least one copy of each fragment with a probability of 99%. The table below provides the results of similar calculations for target polynucleotides of different sizes:</p>
    <p num="p-0037">
      <tables id="TABLE-US-00002" num="00002"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="49pt" align="center"> </colspec> <colspec colname="2" colwidth="84pt" align="left"> </colspec> <colspec colname="3" colwidth="84pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="3" rowsep="1" class="description-td" colspan="3">TABLE I</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Average fragment size</td>
                <td class="description-td">Average fragment size</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">after cleavage with</td>
                <td class="description-td">after cleavage with</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2 six-cutters</td>
                <td class="description-td">3 six-cutters</td>
              </tr> <tr class="description-tr"> <td class="description-td">Size of Target</td>
                <td class="description-td">(No. of fragments)</td>
                <td class="description-td">(No. of fragments)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Polynucleotide</td>
                <td class="description-td">[Sample size for complete</td>
                <td class="description-td">[Sample size for complete</td>
              </tr> <tr class="description-tr"> <td class="description-td">(basepairs)</td>
                <td class="description-td">set with 99% probability]</td>
                <td class="description-td">set with 99% probability]</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">2.5 × 10<sup>5</sup> </td>
                <td class="description-td">2048 (124) [576]</td>
                <td class="description-td">1365 (250) [1050]</td>
              </tr> <tr class="description-tr"> <td class="description-td">  5 × 10<sup>5</sup> </td>
                <td class="description-td">2048 (250) [1050]</td>
                <td class="description-td">1365 (500) [2300]</td>
              </tr> <tr class="description-tr"> <td class="description-td">  1 × 10<sup>6</sup> </td>
                <td class="description-td">2048 (500) [2300]</td>
                <td class="description-td">1365 (1000) [4605]</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0038">After selection, the vector-containing hosts are combined and expanded in cultured. The vectors are then isolated, e.g. by a conventional mini-prep, or the like, and cleaved with IIs<sub>1 </sub>and IIs<sub>2 </sub>(<b>316</b>). The fragments comprising the vector and ends (i.e. segments) of the restriction fragment insert are isolated, e.g. by gel electrophoresis, blunted (<b>316</b>), and re-circularized (<b>320</b>). The resulting pairs of segments in the re-circularized vectors are then amplified (<b>322</b>), e.g. by polymerase chain reaction (PCR), after which the amplified pairs are cleaved with w (<b>324</b>) to free the pairs of segments, which are isolated (<b>326</b>), e.g. by gel electrophoresis. The isolated pairs are concatenated (<b>328</b>) in a conventional ligation reaction to produce concatemers of various sizes, which are separated, e.g. by gel electrophoresis. Concatemers greater than about 200-300 basepairs are isolated and cloned (<b>330</b>) into a standard sequencing vector, such as M13. The sequences of the cloned concatenated pairs are analyzed on a conventional DNA sequencer, such as a model 377 DNA sequencer from Perkin-Elmer Applied Biosystems Division (Foster City, Calif.).</p>
    <p num="p-0039">In the above embodiment, the sequences of the pairs of segments are readily identified between sequences for the recognition site of the enzymes used in the digestions. For example, when pairs are concatenated from fragments of the r and s digestion after cleavage with a type IIs restriction endonuclease of reach (16/14), the following pattern is observed (SEQ ID NO: 1):</p>
    <p num="p-0040"> <tables id="TABLE-US-00003" num="00003"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <colspec colname="2" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">. . . NNNNrrrrrrNNNNNNNNNNNNNNNNNNqqqqqqNNNNN</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">N . . .</td> </tr> </tbody> </tgroup> </table> </tables> <br>
where “r” and “q” represent the nucleotides of the recognition sites of restriction endonuclease r and q, respectively, and where the N's are the nucleotides of the pairs of segments. Thus, the pairs are recognized by their length and their spacing between known recognition sites.
</p>
    <p num="p-0041">Pairs of segments are ordered by matching the sequences of segments between pairs. That is, a candidate map is built by selecting pairs that have one identical and one different sequence. The identical sequences are matched to form a candidate map, or ordering, as illustrated below for pairs (s<sub>1</sub>, s<sub>2</sub>), (s<sub>3</sub>, s<sub>2</sub>), (s<sub>3</sub>, s<sub>4</sub>), (s<sub>5</sub>, s<sub>4</sub>), and (s<sub>5</sub>, s<sub>6</sub>), where the “s<sub>k</sub>'s” represent the nucleotide sequences of the segments:</p>
    <p num="p-0042"> <tables id="TABLE-US-00004" num="00004"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">  . . .   s<sub>1</sub>---- s<sub>2</sub> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">s<sub>3</sub>----------------- s<sub>2</sub> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">s<sub>3</sub>---------------------------------- --- s<sub>4</sub> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">                                   s<sub>5</sub>------- s<sub>4</sub> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">                                   s<sub>5</sub>---------------- s<sub>6 </sub>. . .</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Sequence matching and candidate map construction is readily carried out by computer algorithms, such as the Fortran code provided in Appendix A. Preferably, a map construction algorithm initially sorts the pairs to remove identical pairs prior to map construction. That is, preferably only one pair of each kind is used in the reconstruction. If for two pairs, (s<sub>i</sub>, s<sub>j</sub>) and (s<sub>m</sub>, s<sub>n</sub>), s<sub>i</sub>=s<sub>m </sub>and s<sub>j</sub>=s<sub>n</sub>, then one of the two can be eliminated prior to map construction. As pointed out above, such additional pairs either correspond to restriction fragments such as (<b>92</b>) of <figref idrefs="DRAWINGS">FIG. 1</figref> (no sites of a second or third restriction endonuclease in its interior) or they are additional copies of pairs (because of sampling) that can be used in the analysis. Preferably, an algorithm selects the largest candidate map as a solution, i.e. the candidate map that uses the maximal number of pairs.
</p>
    <p num="p-0043">The vector of <figref idrefs="DRAWINGS">FIG. 2</figref> can also be used for determining the frequency of expression of particular cDNAs in a cDNA library. Preferably, cDNAs whose frequencies are to be determined are cloned into a vector by way of flanking restriction sites that correspond to those of (<b>206</b>) and (<b>210</b>). Thus, cDNAs may be cleaved from the library vectors and directionally inserted into the vector of <figref idrefs="DRAWINGS">FIG. 2</figref>. After insertion, analysis is carried out as described for the mapping embodiment, except that a larger number of concatemers are sequenced in order to obtain a large enough sample of cDNAs for reliable data on frequencies.</p>
    <heading>Example 1</heading> <heading>Constructing a Physical Map of Yeast Chromosome I with Hind III, Eco RI, and Xba I</heading> <p num="p-0044">In this example, a physical map of the 230 kilobase yeast chromosome 1 is constructed using pUC19 plasmids modified in accordance with <figref idrefs="DRAWINGS">FIG. 2</figref>. The chromosome is separately digested to completion with the following combinations of enzymes: Hind III and Eco RI, Hind III and Xba I, and Eco RI and Xba Ito generate three populations of restriction fragments. Fragments from each population are inserted into separate pUC19 plasmids, one for each restriction fragment having different ends. That is, restriction fragments from the Hind III-Eco RI digestion are present in three types, ones with a Hind III-digested end and an Eco RI-digested end (“H-E” fragments), one with only Hind III-digested ends (“H-H” fragments), and ones with only Eco RI-digested fragments (“E-E” fragments). Likewise, restriction fragments from the Hind III-Kba I digestion are present in three types, ones with a Hind III-digested end and an Xba I-digested end (“H-X” fragments), one with only Hind III-digested ends (“H-H” fragments), and ones with only Xba I-digested fragments (“X-X” fragments). Finally, restriction fragments from the Xba I-Leo RI digestion are present in three types, ones with a Xba I-digested end and an Leo RI-digested end (“X-E” fragments), one with only Xba I-digested ends (“X-X” fragments), and ones with only Leo RI-digested fragments (“E-E” fragments). Thus, the plasmid for the Hind III-Eco RI digestion accepts H-E fragments; the plasmid for the Hind III-Xba I digestion accepts H-X fragments; and the plasmid for the Xba I-Leo RI digestion accepts X-E fragments. The construction of the plasmid for accepting H-E fragments is described below. The other plasmids are constructed in a similar manner. Synthetic oligonucleotides (i) through (iv) are combined with a Eco RI- and Hind III-digested pUC19 in a ligation reaction so that they assemble into the double stranded insert of Formula I.</p>
    <p num="p-0045"> <tables id="TABLE-US-00005" num="00005"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="168pt" align="left"> </colspec> <colspec colname="3" colwidth="63pt" align="left"> </colspec> <colspec colname="4" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">(i)</td> <td class="description-td">5′-AATTAGCCGTACCTGCAGCAGTGCAGAAGCTTGCGT</td> <td class="description-td">(SEQ ID NO: 2)</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">(ii)</td> <td class="description-td">5′-AAACCTCAGAATTCCTGCACAGCTGCGAATCATTCG</td> <td class="description-td">(SEQ ID NO: 3)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">(iii)</td> <td class="description-td">5′-AGCTCGAATGATTCGCAGCTGTGCAGGAATTCTGAG</td> <td class="description-td">(SEQ ID NO: 4)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">(iv)</td> <td class="description-td">5′-GTTTACGCAAGCTTCTGCACTGCTGCAGGTACGGCT</td> <td class="description-td">(SEQ ID NO: 5)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="231pt" align="left"> </colspec> <colspec colname="2" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">                      →    →</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">                      Bbv I   Bsg I   Hind III</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">                       ↓     ↓     ↓</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">5′-AATTAGCCGTACCTGCAGCAGTGCAGAAGCTTGCGTAAACCTCA-</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">       <u>TCGGCATGGACGTCGTCA</u>CGTCTTCGAACGCATTTGGAGT-</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">         P<sub>1 </sub>primer binding site</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">                       p2 primer binding site</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">         -GAATTCCTGCACAGCTGCGAATCATTCG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">         -CTTAAGGACG<u>TGTCGACGCTTAGTAAG</u>CTCGA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">              ↑     ↑     ↑</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">           Eco RI   Bsg I   Bbv I</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">                         <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US7598035B2/US07598035-20091006-P00001.png"> <img id="CUSTOM-CHARACTER-00001" he="2.12mm" wi="2.46mm" file="US07598035-20091006-P00001.TIF" alt="Figure US07598035-20091006-P00001" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7598035B2/US07598035-20091006-P00001.png" class="patent-full-image" width="9" height="8"> </a> </div>      <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US7598035B2/US07598035-20091006-P00001.png"> <img id="CUSTOM-CHARACTER-00002" he="2.12mm" wi="2.46mm" file="US07598035-20091006-P00001.TIF" alt="Figure US07598035-20091006-P00001" img-content="character" img-format="tif" src="//patentimages.storage.googleapis.com/US7598035B2/US07598035-20091006-P00001.png" class="patent-full-image" width="9" height="8"> </a> </div> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">            Formula I (SEQ ID NO: 6; SEQ ID NO: 7)</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Note that the insert has compatible ends to the Eco RI-Hind III-digested plasmid, but that the original Eco RI and Hind III sites are destroyed upon ligation. The horizontal arrows above and below the Bsg I and Bbv I sites indicate the direction of the cleavage site relative to the recognition site of the enzymes. After ligation, transformation of a suitable host, and expansion, the modified pUC19 is isolated and the insert is sequenced to confirm its identity.
</p>
    <p num="p-0046">Yeast chromosome 1 DNA is separated into three aliquots of about 5 μg DNA (0.033 pmol) each, which are then separately digested to completion with Hind III and Eco RI, Hind III and Xba I, and Eco RI and Xba I, respectively. For each of the three populations, the same procedure is followed, which is described as follows for the pUC19 designed for H-E fragments.</p>
    <p num="p-0047">Since each enzyme recognizes a six basepair recognition sequence, about 100-140 fragments are produced for a total of about 3.3 pmol of fragments, about fifty percent of which are H-E fragments. 5.26 μg (3 pmol) of plasmid DNA is digested with Eco RI and Hind III in Eco RI buffer as recommended by the manufacturer (New England Biolabs, Beverly, Mass.), purified by phenol extraction and ethanol precipitation, and ligated to the H-E fragments of the mixture in a standard ligation reaction. A bacterial host is transformed, e.g. by electroporation, and plated so that hosts containing recombinant plasmids are identified by white colonies. The digestion of the yeast chromosome I generates about 124 fragments of the three types, about fifty percent of which are H-E fragments and about twenty-five percent each are H-H or E-E fragments. About 290 colonies are picked for H-E fragments, and about 145 each are picked for H-H and E-E fragments. The same procedure is carried out for all the other types of fragments, so that six populations of transformed hosts are obtained, one each for H-E, H-X, X-E, H-H, E-E, and X-X fragments. Each of the populations is treated separately as follows: About 10 μg of plasmid DNA is digested to completion with Bsg I using the manufacturer's protocol (New England Biolabs, Beverly, Mass.) and after phenol extraction the vector/segment-containing fragment is isolated, e.g. by gel electrophoresis. The ends of the isolated fragment are then blunted by Mung bean nuclease (using the manufacturer's recommended protocol, New England Biolabs), after which the blunted fragments are purified by phenol extraction and ethanol precipitation. The fragments are then resuspended in a ligation buffer at a concentration of about 0.05 μg/ml in 20 1-ml reaction volumes. The dilution is designed to promote self-ligation of the fragments, following the protocol of Collins et al, Proc. Natl. Acad. Sci., 81: 6812-6816 (1984). After ligation and concentration by ethanol precipitation, phages from the 20 reactions are combined. The pairs of segments carried by the plasmids are then amplified by PCR using primers p<sub>1 </sub>and p<sub>2</sub>. The amplified product is purified by phenol extraction and ethanol precipitation, after which it is cleaved with Bbv I using the manufacturer's recommended protocol (New England Biolabs). After isolation by polyacrylamide gel electrophoresis, the pairs are concatenated by carrying out a conventional ligation reaction. The concatenated fragments are then separated by polyacrylamide gel electrophoresis and concatemers greater than about 200 basepairs are isolated and ligated into an equimolar mixture of three Phagescript SK sequencing vectors (Stratagene Cloning Systems, La Jolla, Calif.), separately digested with Hind III, Eco RI, and Hind III and Eco RI, respectively. (Other appropriate mixtures and digestions are employed when different combinations of enzymes are used). Preferably, a number of clones are expanded and sequenced that ensure with a probability of at least 99% that all of the pairs of the aliquot are sequenced. A “lane” of sequence data (about 600 bases) obtained with conventional sequencing provides the sequences of about 25 pairs of segments. Thus, after transfection, about 13 individual clones are expanded and sequenced on a commercially available DNA sequencer, e.g. PE Applied Biosystems model 377, to give the identities of about 325 pairs of segments. The other sets of fragments require an additional 26 lanes of sequencing (13 each for the H-X and X-E fragments).</p>
    <p num="p-0048"> <figref idrefs="DRAWINGS">FIG. 4</figref> illustrates the positions on yeast chromosome 1 of pairs of segments ordered in accordance with the algorithm of Appendix A. The relative spacing of the segments along the chromosome is only provided to show the distribution of sequence information along the chromosome.</p>
    <heading>Example 2</heading> <heading>Directed Sequencing of Yeast Chromosome 1 Using Restriction Map Sequences as Spaced PCR Primers</heading> <p num="p-0049">In this example, the 14-mer segments making up the physical map of Example 1 are used to separately amplify by PCR fragments that collectively cover yeast 1 chromosome. The PCR products are inserted into standard M13mp19, or like, sequencing vectors and sequenced in both the forward and reverse directions using conventional protocols. For fragments greater than about 800 basepairs, the sequence information obtained in the first round of sequencing is used to synthesized new sets of primers for the next round of sequencing. Such directed sequencing continues until each fragment is completely sequenced. Based on the map of Example 1, 174 primers are synthesized for 173 PCRs. The total number of sequencing reactions required to cover yeast chromosome 1 depends on the distribution of fragment sizes, and particularly, how many rounds of sequencing are required to cover each fragment: the larger the fragment, the more rounds of sequencing that are required for full coverage. Full coverage of a fragment is obtained when inspection of the sequence information shows that complementary sequences are being identified. Below, it is assumed that conventional sequencing will produce about 400 bases at each end of a fragment in each round. Inspection shows that the distribution of fragment sizes from the Example 1 map of yeast chromosome I are shown below together with reaction and primer requirements:</p>
    <p num="p-0050"> <tables id="TABLE-US-00006" num="00006"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="42pt" align="center"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="70pt" align="center"> </colspec> <colspec colname="4" colwidth="70pt" align="center"> </colspec> <colspec colname="5" colwidth="70pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td> </tr> <tr class="description-tr"> <td class="description-td">Round of</td> <td class="description-td">Fragment</td> <td class="description-td">Number of</td> <td class="description-td">Number of</td> <td class="description-td">Number of</td> </tr> <tr class="description-tr"> <td class="description-td">Sequencing</td> <td class="description-td">size range</td> <td class="description-td">Fragments</td> <td class="description-td">Seq. or PCR Primers</td> <td class="description-td">Sequencing Reactions</td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="42pt" align="center"> </colspec> <colspec colname="2" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="70pt" align="center"> </colspec> <colspec colname="4" colwidth="70pt" align="center"> </colspec> <colspec colname="5" colwidth="70pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">1</td> <td class="description-td">&gt;0</td> <td class="description-td">174 </td> <td class="description-td">174</td> <td class="description-td">348</td> </tr> <tr class="description-tr"> <td class="description-td">2</td> <td class="description-td">&gt;800</td> <td class="description-td">92</td> <td class="description-td">184</td> <td class="description-td">184</td> </tr> <tr class="description-tr"> <td class="description-td">3</td> <td class="description-td">&gt;1600</td> <td class="description-td">53</td> <td class="description-td">106</td> <td class="description-td">106</td> </tr> <tr class="description-tr"> <td class="description-td">4</td> <td class="description-td">&gt;2400</td> <td class="description-td">28</td> <td class="description-td"> 56</td> <td class="description-td">56</td> </tr> <tr class="description-tr"> <td class="description-td">5</td> <td class="description-td">&gt;3200</td> <td class="description-td">16</td> <td class="description-td"> 32</td> <td class="description-td">32</td> </tr> <tr class="description-tr"> <td class="description-td">6</td> <td class="description-td">&gt;4000</td> <td class="description-td"> 7</td> <td class="description-td"> 14</td> <td class="description-td">14</td> </tr> <tr class="description-tr"> <td class="description-td">7</td> <td class="description-td">&gt;4800</td> <td class="description-td"> 5</td> <td class="description-td"> 10</td> <td class="description-td">10</td> </tr> <tr class="description-tr"> <td class="description-td">8</td> <td class="description-td">&gt;5600</td> <td class="description-td"> 1</td> <td class="description-td"> 2</td> <td class="description-td">2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Total No. of Primers:</td> <td class="description-td">578</td> <td class="description-td">752</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Seq. reactions for map:</td> <td class="description-td">39</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Total No. of</td> <td class="description-td">791</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Reactions:</td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
This compares to about 2500-3000 sequencing reactions that are required for full coverage using shotgun sequencing.
</p>
    <p num="p-0051">
      <tables id="TABLE-US-00007" num="00007"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">APPENDIX A</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Computer Code for Ordering Pairs into a Physical Map</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">program opsort</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
                <td class="description-td">opsort reads ordered pairs from disk files</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
                <td class="description-td">p1.dat, p2.dat, and p3.dat. and sorts</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
                <td class="description-td">them into a physical map.</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">character*1 op(1000,2,14),w(14),x(14)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">character*1 fp(1000,2,14),test(14)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">open(1,file=‘p1.dat’,status=‘old’)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">open(5,file=‘olist.dat’,status=‘replace’)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">nop=0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">read(1,100)nop1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">nop=nop + nop1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 101 j=1,nop</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="14pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> read(1,102)(w(i),i=1,14),</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">+</td>
                <td class="description-td">      <sup> </sup>(x(k),k=1,14)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 121 kk=1,14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> op(j,1,kk)=w(kk)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> op(j,2,kk)=x(kk)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">121</td>
                <td class="description-td"> continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">101</td>
                <td class="description-td">continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">read(1,100)nop2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">nop=nop + nop2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 1011 j=nop1+1,nop</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="14pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> read(1,102)(w(i),i=1,14),</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">+</td>
                <td class="description-td">      <sup> </sup>(x(k),k=1,14)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 1211 kk=1,14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> op(j,1,kk)=w(kk)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> op(j,2,kk)=x(kk)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1211</td>
                <td class="description-td">  continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1011</td>
                <td class="description-td"> continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">close(1)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">write(5,110)nop1,nop2,nop</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">110</td>
                <td class="description-td">format(3(2x,i4))</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">open(1,file=‘p2.dat’,status=‘old’)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">read(1,100)nop3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">nop=nop + nop3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 104 j=nop1+nop2+1,nop</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="14pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> read(1,102)(w(i),i=1,14),</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">+</td>
                <td class="description-td">      <sup> </sup>(x(k),k=1,14)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 122 kk=1,14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> op(j,1,kk)=w(kk)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> op(j,2,kk)=x(kk)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">122</td>
                <td class="description-td"> continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">104</td>
                <td class="description-td">continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">read(1,100)nop4</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">nop=nop + nop4</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 1041 j=nop1+nop2+nop3+1,nop</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="14pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> read(1,102)(w(i),i=1,14),</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">+</td>
                <td class="description-td">      <sup> </sup>(x(k),k=1,14)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 1221 kk=1,14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> op(j,1,kk)=w(kk)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> op(j,2,kk)=x(kk)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1221</td>
                <td class="description-td">  continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1041</td>
                <td class="description-td"> continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">close(1)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">write(5,1108)nop1,nop2,nop3,nop4,nop</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1108</td>
                <td class="description-td">format(5(2x,i4))</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">open(1,file=‘p3.dat’,status=‘old’)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">read(1,100)nop5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">nop=nop + nop5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 105 j=nop1+nop2+nop3+nop4+1,nop</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="14pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> read(1,102)(w(i),i=1,14),</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">+</td>
                <td class="description-td">      <sup> </sup>(x(k),k=1,14)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 123 kk=1,14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> op(j,1,kk)=w(kk)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> op(j,2,kk)=x(kk)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">123</td>
                <td class="description-td"> continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">105</td>
                <td class="description-td">continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">read(1,100)nop6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">nop=nop + nop6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 1051 j=nop1+nop2+nop3+nop4+nop5+1,nop</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="14pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> read(1,102)(w(i),i=1,14),</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">+</td>
                <td class="description-td">      <sup> </sup>(x(k),k=1,14)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 1231 kk=1,14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> op(j,1,kk)=w(kk)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> op(j,2,kk)=x(kk)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1231</td>
                <td class="description-td">  continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1051</td>
                <td class="description-td"> continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">close(1)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">write(5,1109)nop1,nop2,nop3,nop4,nop5,nop6,nop</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1109</td>
                <td class="description-td">format(7(2x,i4))</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">100</td>
                <td class="description-td">format(i4)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">102</td>
                <td class="description-td">format(2(2x,14a1))</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">111</td>
                <td class="description-td">format(/)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">write(5,111)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 120 m=1,nop</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="14pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> write(5,102)(op(m,1,i),i=1,14),</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">+</td>
                <td class="description-td">      (op(m,2,k),k=1,14)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> write(*,102)(op(m,1,i),i=1,14),</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">+</td>
                <td class="description-td">      (op(m,2,k),k=1,14)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">120</td>
                <td class="description-td"> continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">write(5,111)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 1100 i=1,14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> test(i)=op(1,2,i)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">   fp(1,1,i)=op(1,1,i)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">   fp(1,2,i)=op(1,2,i)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1100</td>
                <td class="description-td"> continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">nxx=nop</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">ns=1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1000</td>
                <td class="description-td">continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">ne=0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 2000 ix=2,nxx</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> nt=0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> do 2100 jx=1,14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> if(test(jx).ne.op(ix,1,jx)) then</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  nt=nt+1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  endif</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2100</td>
                <td class="description-td">continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> if(nt.eq.0) then</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  ns=ns+1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  ne=ne+1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  if(ne.gt.1) then</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">   write(*,1003)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1003</td>
                <td class="description-td">format(1x,‘ne is gt 1’)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  endif</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  do 2200 kx=1,14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">   fp(ns,1,kx)=op(ix,1,kx)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">   fp(ns,2,kx)=op(ix,2,kx)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">   test(kx)=op(ix,2,kx)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2200</td>
                <td class="description-td">   continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">    mm=0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">   do 2300 mx=1,nxx</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">    if(mx.eq.ix) then</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">     goto 2300</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">    else</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">     mm=mm+1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">     do 2400 ma=1,14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">     op(mm,1,ma)=op(mx,1,ma)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">     op(mm,2,ma)=op(mx,2,ma)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2400</td>
                <td class="description-td">     continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">    endif</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2300</td>
                <td class="description-td">    continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">   endif</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2000</td>
                <td class="description-td">  continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  nxx=nxx−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">  if(ne.ne.0) then</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">    goto 1000</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">    endif</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">do 1220 m=1,ns</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="14pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> write(5,102)(fp(m,1,i),i=1,14),</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">+</td>
                <td class="description-td">      (fp(m,2,k),k=1,14)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> write(*,102)(fp(m,1,i),i=1,14),</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">+</td>
                <td class="description-td">      (fp(m,2,k),k=1,14)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="175pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1220</td>
                <td class="description-td">  continue</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">write(*,100)ns</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">close(5)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">c</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">end</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4293652">US4293652</a></td><td class="patent-data-table-td patent-date-value">May 25, 1979</td><td class="patent-data-table-td patent-date-value">Oct 6, 1981</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">Method for synthesizing DNA sequentially</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5102785">US5102785</a></td><td class="patent-data-table-td patent-date-value">Apr 25, 1988</td><td class="patent-data-table-td patent-date-value">Apr 7, 1992</td><td class="patent-data-table-td ">E. I. Du Pont De Nemours And Company</td><td class="patent-data-table-td ">Method of gene mapping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5196328">US5196328</a></td><td class="patent-data-table-td patent-date-value">May 21, 1987</td><td class="patent-data-table-td patent-date-value">Mar 23, 1993</td><td class="patent-data-table-td ">Institute For Cancer Research</td><td class="patent-data-table-td ">Modified cloning vectors for restriction mapping, their preparation and use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5409897">US5409897</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 31, 1994</td><td class="patent-data-table-td patent-date-value">Apr 25, 1995</td><td class="patent-data-table-td ">Merck &amp; Co., Inc.</td><td class="patent-data-table-td ">Cysteine-modified acidic fibroblast growth factor and methods of use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5508169">US5508169</a></td><td class="patent-data-table-td patent-date-value">Oct 28, 1994</td><td class="patent-data-table-td patent-date-value">Apr 16, 1996</td><td class="patent-data-table-td ">Queen&#39;s University At Kingston</td><td class="patent-data-table-td ">Treating dna with restriction endonuclease yields synthetic nucleic acids which can be selectively joined to end groups of unknown dna fragments for detection and analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5604097">US5604097</a></td><td class="patent-data-table-td patent-date-value">Dec 19, 1994</td><td class="patent-data-table-td patent-date-value">Feb 18, 1997</td><td class="patent-data-table-td ">Spectragen, Inc.</td><td class="patent-data-table-td ">Methods for sorting polynucleotides using oligonucleotide tags</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5658736">US5658736</a></td><td class="patent-data-table-td patent-date-value">Jan 16, 1996</td><td class="patent-data-table-td patent-date-value">Aug 19, 1997</td><td class="patent-data-table-td ">Genetics Institute, Inc.</td><td class="patent-data-table-td ">Oligonucleotide population preparation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5667970">US5667970</a></td><td class="patent-data-table-td patent-date-value">May 10, 1994</td><td class="patent-data-table-td patent-date-value">Sep 16, 1997</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Cloning and hybridization with overlapping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5695937">US5695937</a></td><td class="patent-data-table-td patent-date-value">Sep 12, 1995</td><td class="patent-data-table-td patent-date-value">Dec 9, 1997</td><td class="patent-data-table-td ">The Johns Hopkins University School Of Medicine</td><td class="patent-data-table-td ">Producing complementary dna oligonucleotides, isolating first and second defined nucleotide sequence tags, linking tags to first and second oligonucleotide linker, cleaving linked dnas and ligating the tags to form ditag</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5710000">US5710000</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Jan 20, 1998</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Polynucleotide sequences, cleavage of seqences from sequences of polynucleotide sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5728524">US5728524</a></td><td class="patent-data-table-td patent-date-value">Feb 16, 1995</td><td class="patent-data-table-td patent-date-value">Mar 17, 1998</td><td class="patent-data-table-td ">Medical Research Counsil</td><td class="patent-data-table-td ">Process for categorizing nucleotide sequence populations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5804382">US5804382</a></td><td class="patent-data-table-td patent-date-value">May 10, 1996</td><td class="patent-data-table-td patent-date-value">Sep 8, 1998</td><td class="patent-data-table-td ">Beth Israel Deaconess Medical Center, Inc.</td><td class="patent-data-table-td ">Hybridization of amplification-tagged tester dna and non-tagged driver dna in reaction mixture, removal of single-stranded dna and amplification tag prior to amplification of tester-tester homoduplex</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5834201">US5834201</a></td><td class="patent-data-table-td patent-date-value">Jul 11, 1997</td><td class="patent-data-table-td patent-date-value">Nov 10, 1998</td><td class="patent-data-table-td ">Life Technologies, Inc.</td><td class="patent-data-table-td ">Nucleic acid marker ladder for estimating mass</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5846719">US5846719</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 1996</td><td class="patent-data-table-td patent-date-value">Dec 8, 1998</td><td class="patent-data-table-td ">Lynx Therapeutics, Inc.</td><td class="patent-data-table-td ">Oligonucleotide tags for sorting and identification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5861252">US5861252</a></td><td class="patent-data-table-td patent-date-value">Nov 27, 1996</td><td class="patent-data-table-td patent-date-value">Jan 19, 1999</td><td class="patent-data-table-td ">Hitachi, Ltd.</td><td class="patent-data-table-td ">Method of analysis or assay for polynucleotides and analyzer or instrument for polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5968784">US5968784</a></td><td class="patent-data-table-td patent-date-value">Jan 15, 1997</td><td class="patent-data-table-td patent-date-value">Oct 19, 1999</td><td class="patent-data-table-td ">Chugai Pharmaceutical Co., Ltd.</td><td class="patent-data-table-td ">Detecting gene transcription patterns in mrna population; preparing and cleaving complementary dnas, insertion of in vector, replicating, isolating, digesting dna, ligating tags, sequencing; determining pattern expression of disease mrna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6054276">US6054276</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 23, 1998</td><td class="patent-data-table-td patent-date-value">Apr 25, 2000</td><td class="patent-data-table-td ">Macevicz; Stephen C.</td><td class="patent-data-table-td ">DNA restriction site mapping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6136537">US6136537</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 6, 1998</td><td class="patent-data-table-td patent-date-value">Oct 24, 2000</td><td class="patent-data-table-td ">Macevicz; Stephen C.</td><td class="patent-data-table-td ">Gene expression analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0593095A1?cl=en">EP0593095A1</a></td><td class="patent-data-table-td patent-date-value">Feb 15, 1990</td><td class="patent-data-table-td patent-date-value">Apr 20, 1994</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Genomic cloning and mapping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0761822A2?cl=en">EP0761822A2</a></td><td class="patent-data-table-td patent-date-value">Sep 12, 1996</td><td class="patent-data-table-td patent-date-value">Mar 12, 1997</td><td class="patent-data-table-td ">The Johns Hopkins University School Of Medicine</td><td class="patent-data-table-td ">Method for serial analysis of gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1996017082A2?cl=en">WO1996017082A2</a></td><td class="patent-data-table-td patent-date-value">Nov 21, 1995</td><td class="patent-data-table-td patent-date-value">Jun 6, 1996</td><td class="patent-data-table-td ">Du Pont</td><td class="patent-data-table-td ">Compound microsatellite primers for the detection of genetic polymorphisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998010095A1?cl=en">WO1998010095A1</a></td><td class="patent-data-table-td patent-date-value">Sep 5, 1997</td><td class="patent-data-table-td patent-date-value">Mar 12, 1998</td><td class="patent-data-table-td ">Brax Genomics Ltd</td><td class="patent-data-table-td ">Characterising dna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998031838A1?cl=en">WO1998031838A1</a></td><td class="patent-data-table-td patent-date-value">Jan 15, 1998</td><td class="patent-data-table-td patent-date-value">Jul 23, 1998</td><td class="patent-data-table-td ">Chugai Pharmaceutical Co Ltd</td><td class="patent-data-table-td ">Method for analyzing quantitative expression of genes</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chen et al., "<a href='http://scholar.google.com/scholar?q="Ordered+shotgun+sequencing%2C+a+strategy+for+integrated+mapping+and+sequencing+of+YAC+clones%2C"'>Ordered shotgun sequencing, a strategy for integrated mapping and sequencing of YAC clones,</a>" Genomics, 17:651-656 (1993).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Evans, "<a href='http://scholar.google.com/scholar?q="Combinatoric+strategies+for+genome+mapping%2C"'>Combinatoric strategies for genome mapping,</a>" BioEssays, 13:39-44 (1991).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Green et al., "<a href='http://scholar.google.com/scholar?q="Systematic+generation+of+sequence-tagged+sites+for+physical+mapping+of+human+chromosomes%3A+application+to+the+mapping+of+human+chromosome+7+using+yeast+artificial+chromosomes%2C"'>Systematic generation of sequence-tagged sites for physical mapping of human chromosomes: application to the mapping of human chromosome 7 using yeast artificial chromosomes,</a>" Genomics, 11:548-564 (1991).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hasan et al., "<a href='http://scholar.google.com/scholar?q="A+novel+multistep+method+for+generating+precise+unidirectional+deletions+using+Bsp+MI%2C+a+class-IIS+restriction+enzyme%2C"'>A novel multistep method for generating precise unidirectional deletions using Bsp MI, a class-IIS restriction enzyme,</a>" Gene, 50:55-62 (1986).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hasan et al., "<a href='http://scholar.google.com/scholar?q="An+Mbo+II%2FFok+I+trimming+plasmid+allowing+consecutive+cycles+of+precise+1-+to+12-base-pair+deletions+in+cloned+DNA%2C"'>An Mbo II/Fok I trimming plasmid allowing consecutive cycles of precise 1- to 12-base-pair deletions in cloned DNA,</a>" Gene, 82:305-311 (1989).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Howard et al., Biotechniques, 7:940-942 (1989).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hudson et al., "<a href='http://scholar.google.com/scholar?q="An+STS-based+map+of+the+human+genome%2C"'>An STS-based map of the human genome,</a>" Science, 270:1945-1954 (1995).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kato, "<a href='http://scholar.google.com/scholar?q="Description+of+the+entire+mRNA+population+by+a+3%27+end+cDNA+fragment+generated+by+class+IIs+restriction+enzymes%2C"'>Description of the entire mRNA population by a 3' end cDNA fragment generated by class IIs restriction enzymes,</a>" Nucleic Acids Research, 23:3685-3690 (1995).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kato, "<a href='http://scholar.google.com/scholar?q="RNA+fingerprinting+by+molecular+indexing%2C"'>RNA fingerprinting by molecular indexing,</a>" Nucleic Acids Research, 24:394-395 (1996).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Michiels et al., "<a href='http://scholar.google.com/scholar?q="Molecular+approaches+to+genome+analysis%3A+a+strategy+for+the+construction+of+ordered+overlapping+clone+libraries%2C"'>Molecular approaches to genome analysis: a strategy for the construction of ordered overlapping clone libraries,</a>" CABIOS, 3: 203-210 (1987).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">New England Biolabs 96/97 Catalog, p. 115. Published by New England Biolabs. Inc., 32 Tozer Road, Beverly, MA 01915-5599.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">New England Biolabs 96/97 Catalog, pp. 15, 18, 22, 23 and 35. Published by New England Biolabs. Inc., 32 Tozer Road, Beverly, MA 01915-5599.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">New England Biolabs 96/97 Catalog, pp. 19, 37, 108 and 109. Published by New England Biolabs. Inc., 32 Tozer Road, Beverly, MA 01915-5599.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">New England Biolabs 96/97 Catalog, pp. 36 and 50. Published by New England Biolabs. Inc., 32 Tozer Road, Beverly, MA 01915-5599.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Olson et al., "<a href='http://scholar.google.com/scholar?q="Random-clone+strategy+for+genomic+restriction+mapping+in+yeast%2C"'>Random-clone strategy for genomic restriction mapping in yeast,</a>" Proc.-Natl. Acad. Sci., 83:7826-7830 (1986).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Poustka and Lehrach, "<a href='http://scholar.google.com/scholar?q="Chromosome+jumping%3A+a+long+range+cloning+technique%2C"'>Chromosome jumping: a long range cloning technique,</a>" in Genetic Engineering: Principles and Methods, J.K. Setlow, Editor, 10:169-193 (1988).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Poustka and Lehrach, "<a href='http://scholar.google.com/scholar?q="Jumping+libraries+and+linking+libraries%3A+the+next+generation+of+molecular+tools+in+mammalian+genetics%2C"'>Jumping libraries and linking libraries: the next generation of molecular tools in mammalian genetics,</a>" Trends in Genetics, 2:174-179 (1986).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Roach et al., "<a href='http://scholar.google.com/scholar?q="Pairwise+end+sequencing%3A+A+unified+approach+to+genomic+mapping+and+sequencing%2C"'>Pairwise end sequencing: A unified approach to genomic mapping and sequencing,</a>" Genomics, 26:345-353 (1995).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sapolsky et al., "<a href='http://scholar.google.com/scholar?q="Mapping+genomic+library+clones+using+oligonucleotide+arrays%2C"'>Mapping genomic library clones using oligonucleotide arrays,</a>" Genomics, 33:445-456 (1996).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Smith et al., "<a href='http://scholar.google.com/scholar?q="A+simple+method+for+DNA+restriction+site+mapping%2C"'>A simple method for DNA restriction site mapping,</a>" Nucleic Acids Research, 3: 2387-2398 (1976).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Smith et al., "<a href='http://scholar.google.com/scholar?q="Genomic+sequence+sampling%3A+a+strategy+for+high+resolution+sequence-based+physical+mapping+of+complex+genomes%2C"'>Genomic sequence sampling: a strategy for high resolution sequence-based physical mapping of complex genomes,</a>" Nature Genetics, 7:40-47 (1994).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Velculescu et al., "<a href='http://scholar.google.com/scholar?q="Serial+analysis+of+gene+expression%2C"'>Serial analysis of gene expression,</a>" Science, 270:484-487 (1995).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wong et al., "<a href='http://scholar.google.com/scholar?q="Multiple-complete-digest+restriction+fragment+mapping%3A+Generating+sequence-ready+maps+for+large-scale+DNA+sequencing%2C"'>Multiple-complete-digest restriction fragment mapping: Generating sequence-ready maps for large-scale DNA sequencing,</a>" Proc. Natl. Acad. Sci., 94:5225-5230 (1997).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yi et al., "<a href='http://scholar.google.com/scholar?q="Construction+of+restriction+fragment+maps+of+50-+to+100-kilobase+DNA"'>Construction of restriction fragment maps of 50- to 100-kilobase DNA</a>", Methods in Enzymology, 218:651-671 (1993).</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8288095">US8288095</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 11, 2009</td><td class="patent-data-table-td patent-date-value">Oct 16, 2012</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Methods and compositions for ordering restriction fragments</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8541207">US8541207</a></td><td class="patent-data-table-td patent-date-value">Oct 21, 2009</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Preservation of information related to genomic DNA methylation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8557521">US8557521</a></td><td class="patent-data-table-td patent-date-value">Sep 18, 2012</td><td class="patent-data-table-td patent-date-value">Oct 15, 2013</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Methods for sequencing polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8738300">US8738300</a></td><td class="patent-data-table-td patent-date-value">Jun 12, 2012</td><td class="patent-data-table-td patent-date-value">May 27, 2014</td><td class="patent-data-table-td ">Good Start Genetics, Inc.</td><td class="patent-data-table-td ">Sequence assembly</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013174432A1?cl=en">WO2013174432A1</a></td><td class="patent-data-table-td patent-date-value">May 24, 2012</td><td class="patent-data-table-td patent-date-value">Nov 28, 2013</td><td class="patent-data-table-td ">Fundació Institut D&#39;investigació Biomèdica De Bellvitge (Idibell)</td><td class="patent-data-table-td ">Method for the identification of the origin of a cancer of unknown primary origin by methylation analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2014074611A1?cl=en">WO2014074611A1</a></td><td class="patent-data-table-td patent-date-value">Nov 6, 2013</td><td class="patent-data-table-td patent-date-value">May 15, 2014</td><td class="patent-data-table-td ">Good Start Genetics, Inc.</td><td class="patent-data-table-td ">Methods and systems for identifying contamination in samples</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006110">435/6.11</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024330">536/24.33</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025300">536/25.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006120">435/6.12</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015640000">C12N15/64</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0019340000">C12P19/34</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015100000">C12N15/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021020000">C07H21/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015660000">C12N15/66</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021040000">C07H21/04</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021000000">C07H21/00</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6809">C12Q1/6809</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/10">C12N15/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6841">C12Q1/6841</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/683">C12Q1/683</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/66">C12N15/66</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6869">C12Q1/6869</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/64">C12N15/64</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G06F19/22">G06F19/22</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6872">C12Q1/6872</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=oyTnBQABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6874">C12Q1/6874</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68E2</span>, <span class="nested-value">C12Q1/68B6A</span>, <span class="nested-value">C12Q1/68B14</span>, <span class="nested-value">C12Q1/68E4</span>, <span class="nested-value">C12N15/66</span>, <span class="nested-value">C12Q1/68A6</span>, <span class="nested-value">C12N15/64</span>, <span class="nested-value">C12N15/10</span>, <span class="nested-value">G06F19/22</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jan 21, 2014</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-2 IS CONFIRMED.CLAIMS 3-5 ARE CANCELLED.NEW CLAIMS 6-9 ARE ADDED AND DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 6, 2013</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 11, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ILLUMINA, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:SOLEXA, INC.;REEL/FRAME:024812/0913</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100208</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:MACEVICZ, STEPHEN C.;REEL/FRAME:024812/0910</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20010330</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">LYNX THERAPEUTICS, INC., CALIFORNIA</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 16, 2010</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100113</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 29, 2007</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">SOLEXA, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:LYNX THERAPEUTICS, INC.;REEL/FRAME:020031/0061</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20050304</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_4ff636b3d23669b7103f3b3a3a18b4cd.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U1peSOzYeKk_M74Koh_nAjpqTNGTA\u0026id=oyTnBQABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U0tJbt4Bjph460JuKPgnAJrjkxkgA\u0026id=oyTnBQABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U24XngpsZ80VYHGR9kNqE5_ozFLyw","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Ordering_pairs_of_sequence_tags_via_prov.pdf?id=oyTnBQABERAJ\u0026output=pdf\u0026sig=ACfU3U2Sftrq6EOBILxIUDKW103EqnwE3A"},"sample_url":"http://www.google.com/patents/reader?id=oyTnBQABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>