<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7732593 - Useful in the design of improved RNAi-mediating agents which allow for more ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Useful in the design of improved RNAi-mediating agents which allow for more exact control of the efficacy of RNA silencing; enhancing silencing of a target mRNA"><meta name="DC.contributor" content="Phillip D. Zamore" scheme="inventor"><meta name="DC.contributor" content="Guiliang Tang" scheme="inventor"><meta name="DC.contributor" content="University Of Massachusetts" scheme="assignee"><meta name="DC.date" content="2008-6-13" scheme="dateSubmitted"><meta name="DC.description" content="Based at least in part on an understanding of the mechanisms by which small RNAs (e.g., naturally-occurring miRNAs) mediate RNA silencing in plants, rules have been established for determining, for example, the degree of complementarity required between an RNAi-mediating agent and its target, i.e., whether mismatches are tolerated, the number of mismatches tolerated, the effect of the position of the mismatches, etc. Such rules are useful, in particular, in the design of improved RNAi-mediating agents which allow for more exact control of the efficacy of RNA silencing."><meta name="DC.date" content="2010-6-8" scheme="issued"><meta name="DC.relation" content="US:20020086356:A1" scheme="references"><meta name="DC.relation" content="US:20030108923:A1" scheme="references"><meta name="DC.relation" content="US:20040002153:A1" scheme="references"><meta name="DC.relation" content="US:20040137471:A1" scheme="references"><meta name="DC.relation" content="US:20040192629:A1" scheme="references"><meta name="DC.relation" content="US:20040203145:A1" scheme="references"><meta name="DC.relation" content="US:20040224328:A1" scheme="references"><meta name="DC.relation" content="US:20040229266:A1" scheme="references"><meta name="DC.relation" content="US:20040259247:A1" scheme="references"><meta name="DC.relation" content="US:20040259248:A1" scheme="references"><meta name="DC.relation" content="US:20050026278:A1" scheme="references"><meta name="DC.relation" content="US:20050037988:A1" scheme="references"><meta name="DC.relation" content="US:20050059005:A1" scheme="references"><meta name="DC.relation" content="US:20050181382:A1" scheme="references"><meta name="DC.relation" content="US:20050182005:A1" scheme="references"><meta name="DC.relation" content="US:20050186586:A1" scheme="references"><meta name="DC.relation" content="US:20050227256:A1" scheme="references"><meta name="DC.relation" content="US:20050234007:A1" scheme="references"><meta name="DC.relation" content="US:20050246794:A1" scheme="references"><meta name="DC.relation" content="US:20050256072:A1" scheme="references"><meta name="DC.relation" content="US:20060009402:A1" scheme="references"><meta name="DC.relation" content="US:20060134787:A1" scheme="references"><meta name="DC.relation" content="US:20060166910:A1" scheme="references"><meta name="DC.relation" content="US:20060212950:A1" scheme="references"><meta name="DC.relation" content="US:20070003960:A1" scheme="references"><meta name="DC.relation" content="US:20070003961:A1" scheme="references"><meta name="DC.relation" content="US:20070003962:A1" scheme="references"><meta name="DC.relation" content="US:20070003963:A1" scheme="references"><meta name="DC.relation" content="US:20070031844:A1" scheme="references"><meta name="DC.relation" content="US:20070039072:A1" scheme="references"><meta name="DC.relation" content="US:20070093445:A1" scheme="references"><meta name="DC.relation" content="US:20070207974:A1" scheme="references"><meta name="DC.relation" content="US:6506559" scheme="references"><meta name="DC.relation" content="US:7459547" scheme="references"><meta name="DC.relation" content="WO:2001075164:A2" scheme="references"><meta name="DC.relation" content="WO:2002044321:A2" scheme="references"><meta name="DC.relation" content="WO:2003020931:A2" scheme="references"><meta name="DC.relation" content="WO:2004027030:A2" scheme="references"><meta name="DC.relation" content="WO:2004045543:A2" scheme="references"><meta name="DC.relation" content="WO:2004063375:A1" scheme="references"><meta name="DC.relation" content="WO:2005001043:A2" scheme="references"><meta name="DC.relation" content="WO:2005062937:A2" scheme="references"><meta name="DC.relation" content="WO:2006015389:A2" scheme="references"><meta name="citation_reference" content="Amarzguioui, Mohammed et al., &quot;Tolerance for mutations and chemical modifications in a siRNA,&quot; Nucleic Acids Research, vol. 31(2):589-595 (2003)."><meta name="citation_reference" content="Ameres et al., &quot;Molecular basis for target RNA recognition and cleavage by human RISC,&quot; Cell, 130: 101-112 (2007)."><meta name="citation_reference" content="Bartel, David P., &quot;MicroRNAs: Genomics, Biogenesis, Mechanism, and Function,&quot; Cell, vol. 116:281-297 (2004)."><meta name="citation_reference" content="Beclin, Christophe et al., &quot;A Branched Pathway for Transgene-Induced RNA Silencing in Plants,&quot; Current Biology, vol. 12:684-688, (2002)."><meta name="citation_reference" content="Boden, D. et al, &quot;Efficient gene transfer of HIV-1-specific short hairpin RNA into human lymphocytic cells using recombinant adeno-associated virus vectors,&quot; Mol. Ther., vol. 9(3):396-402 (2004)."><meta name="citation_reference" content="Boden, Daniel et al, &quot;Enhanced gene silencing of HIV-1 specific siRNA using microRNA designed hairpins,&quot; Nucleic Acids Research, vol. 32(3):1154-1158 (2004)."><meta name="citation_reference" content="Bonnet, Eric et al, &quot;Evidence that microRNA precursors, unlike other non-coding RNAs, have lower folding free energies than random sequences,&quot; Bioinformatics, vol. 20(17):2911-2917 (2004)."><meta name="citation_reference" content="Chi, Jen-Tsan et al, &quot;Genomewide view of gene silencing by small interfering RNAs,&quot; PNAS, vol. 100(11):6343-6346 (2003)."><meta name="citation_reference" content="Chiu, Ya-Lin et al, &quot;RNAi in Human Cells: Basic Structural and Functional Features of Small Interfering RNA,&quot; Molecular Cell, vol. 10:549-561 (2002)."><meta name="citation_reference" content="Chiu, Ya-Lin et al, &quot;siRNA function in RNAi: A chemical modification analysis,&quot; RNA, vol. 9:1034-1048 (2003)."><meta name="citation_reference" content="Cogoni, Carlo et al., &quot;Gene silencing in Neurospora crassa requires a protein homologous to RNA-dependent RNA polymerase,&quot; Nature . vol. 399:166-169. (1999)."><meta name="citation_reference" content="Dalmay, Talmas et al., &quot;An RNA-Dependent RNA Polymerase Gene in Arabidopsis Is Required for Posttranscriptional Gene Silencing Mediated by a Transgene but Not by a Virus,&quot; Cell. vol. 101:543-553, (2000)."><meta name="citation_reference" content="Dharmacon RNA Technologies: On-Target siRNA, Company Brochure, 2003."><meta name="citation_reference" content="Dharmacon RNA Technologies: Products for RNA Interference, Company Brochure, 2003."><meta name="citation_reference" content="Doench, John G. et al, &quot;siRNAs can function as miRNAs,&quot; Genes &amp; Development, vol. 17:438-442 (2003)."><meta name="citation_reference" content="Du, Quan et al, &quot;A sytematic analysis of the silencing effects of an active siRNA at all single-nucleotide mismatched target sites,&quot; Nucleic Acids Research, vol. 33(5):1671-1677 (2005)."><meta name="citation_reference" content="Elbashir et al (EMBO J. 20(23): 6877-6888, 2001)."><meta name="citation_reference" content="Elbashir et al (Nature 411: 494-498, 2001)."><meta name="citation_reference" content="Elbashir, Sayda M. et al, &quot;Analysis of gene function in somatic mammalian cells using small interfering RNAs,&quot; Methods, vol. 26:199-213 (2002)."><meta name="citation_reference" content="Elbashir, Sayda M. et al, &quot;Functional anatomy of siRNAs for mediating efficient RNAi in Drosophila melanogaster embryo lysate,&quot; The EMBO Journal, vol. 20(23):6877-6888 (2001)."><meta name="citation_reference" content="Elbashir, Sayda M. et al, &quot;RNA interference is mediated by 21- and 22-nucleotide RNAs,&quot; Genes &amp; Development, vol. 15:188-200 (2001)."><meta name="citation_reference" content="Gong, Delquin et al, &quot;Picking a winner: new mechanistic insights into the design of effective siRNAs,&quot; Trends in Biotechnology, vol. 22(9):451-454 (2004)."><meta name="citation_reference" content="Haley et al (Nature Struct. &amp; Mol. Biol 11(7): 599-606, 2004."><meta name="citation_reference" content="Haley, Benjamin et al, &quot;In vitro analysis of RNA interference in Drosophila melanogaster,&quot; Methods, vol. 30:330-336 (2003)."><meta name="citation_reference" content="Haley, Benjamin et al., &quot;Kinetic analysis of the RNAi enzyme complex,&quot; Nature Structural &amp; Molecular Biology, vol. 11(7):599-606 (2004)."><meta name="citation_reference" content="Hamada, Makiko et al, &quot;Effects on RNA Interference in Gene Expression (RNAi) in Cultured Mammalian Cells of Mismatches and the Introduction of Chemical Modifications at the 3&#39;-Ends of siRNAs,&quot; Antisense and Nucleic Acid Drug Development, vol. 12:301-309 (2002)."><meta name="citation_reference" content="Hamada, Makiko et al, “Effects on RNA Interference in Gene Expression (RNAi) in Cultured Mammalian Cells of Mismatches and the Introduction of Chemical Modifications at the 3′-Ends of siRNAs,” Antisense and Nucleic Acid Drug Development, vol. 12:301-309 (2002)."><meta name="citation_reference" content="Holen, Torgeir et al, &quot;Positional effects of short interfering RNAs targeting the human coagulation trigger Tissue Factor,&quot; Nucleic Acids Research, vol. 30(8):1757-1766 (2002)."><meta name="citation_reference" content="Holen, Torgeir et al, &quot;Similar behaviour of single-strand and double-strand siRNAs suggests they act through a common RNAi pathway,&quot; Nucleic Acids Research, vol. 31(9):2401-2407 (2003)."><meta name="citation_reference" content="Holen, Torgeir et al., &quot;Tolerated wobble mutations in siRNAs decrease specificity, but can enhance activity in vivo,&quot; Nucleic Acids Research, vol. 33(15):4707-4710 (2005)."><meta name="citation_reference" content="Hutvágner, György et al, &quot;A microRNA in a Multiple-Turnover RNAi Enzyme Complex,&quot; Science, vol. 297:2056-2060 (2002)."><meta name="citation_reference" content="International Preliminary Report on Patentability for Application No. PCT/US2004/017130, dated Dec. 8, 2005."><meta name="citation_reference" content="Invitation to Pay Additional Fees Application No. PCT/USO4/17130, dated Apr. 1, 2005."><meta name="citation_reference" content="Jackson, Aimee L. et al, &quot;Expression profiling reveals off-target gene regulation by RNAi,&quot; Nature Biotechnology, vol. 21(6):635-638 (2003)."><meta name="citation_reference" content="Ketting, René F. et al, &quot;Dicer functions in RNA interference and in synthesis of small RNA involved in development timing in C. elegans,&quot; Genes &amp; Development, vol. 15:2654-2659 (2001)."><meta name="citation_reference" content="Khvorova, Anastasia et al, &quot;Functional siRNAs and miRNAs Exhibit Strand Bias,&quot; Cell, vol. 115:209-216 (2003)."><meta name="citation_reference" content="Krol, Jacek et al, &quot;Structural Features of MicroRNA (miRNA) Precursors and Their Relevance to miRNA Biogenesis and Small Interfering RNA/Short Hairpin RNA Design,&quot; The Journal of Biological Chemistry, vol. 279(40):42230-42239 (2004)."><meta name="citation_reference" content="Lagos-Quintana, Mariana et al., &quot;Identification of Novel Genes Coding for Small Expressed RNAs,&quot; Science.. vol. 294:853-858, (2001)."><meta name="citation_reference" content="Lagos-Quintana, Mariana et al., &quot;Identification of Tissue-Specific MicroRNAs from Mouse,&quot; Current Biology. vol. 12:735-739, (2002)."><meta name="citation_reference" content="Lai, Eric C., &quot;Micro RNAs are complementary to 3&#39; UTR sequence motifs that mediate negative post-transcriptional regulation,&quot; Nature Genetics, vol. 30:363-364 (2002)."><meta name="citation_reference" content="Lai, Eric C., “Micro RNAs are complementary to 3′ UTR sequence motifs that mediate negative post-transcriptional regulation,” Nature Genetics, vol. 30:363-364 (2002)."><meta name="citation_reference" content="Lee, Rosalind C. et al., &quot;The C. elegans Heterochronic Gene lin-4 Encodes Small RNAs with Antisense Complementarity to lin-14,&quot; Cell. vol. 75:843-854, (1993)."><meta name="citation_reference" content="Li, Wan Xiang et al., &quot;Viral suppressors of RNA silencing,&quot; Current Opinion in Biotechnology. vol. 12:150-154, (2001)."><meta name="citation_reference" content="Lim, Lee P. et al, &quot;Microarray analysis shows that some microRNAs downregulate large numbers of target mRNAs,&quot; Nature, vol. 433:769-773 (2005)."><meta name="citation_reference" content="Llave, Cesar et al., &quot;Cleavage of Scarecrow-like mRNA Targets Directed by a Class of Arabidopsis miRNA,&quot; Science . vol. 297:2053-2056, (2002)."><meta name="citation_reference" content="Mallory, Allison C. et al, &quot;MicroRNA control of Phabulosa in leaf development: importance of pairing to the microRNA 5&#39; region,&quot; The EMBO Journal, vol. 23:3356-3364 (2004)."><meta name="citation_reference" content="Mallory, Allison C. et al, “MicroRNA control of Phabulosa in leaf development: importance of pairing to the microRNA 5′ region,” The EMBO Journal, vol. 23:3356-3364 (2004)."><meta name="citation_reference" content="Martens, Henrik et al., &quot;RNAi in Dictyostelium : The Role of RNA-directed RNA Polymerases and Double-stranded RNase,&quot; Molecular Biology of the Cell . vol. 13:445-453, (2002)."><meta name="citation_reference" content="Martinez, Javier et al, &quot;Single-Stranded Antisense siRNAs Guide Target RNA Cleavage in RNAi,&quot; Cell, vol. 110:563-574 (2002)."><meta name="citation_reference" content="Martinez, Luis Alfonso et al, &quot;Synthetic small inhibiting RNAs: Efficient tolls to inactivate oncogenic mutations and restore p53 pathways,&quot; PNAS, vol. 99(23):14849-14854 (2002)."><meta name="citation_reference" content="McConnell, Jane R. et al., &quot;Role of Phabulosa and Phavoluta in determining radial patterning in shoots,&quot; Nature . vol. 411:709-713, (2001)."><meta name="citation_reference" content="McManus, Michael T. et al, &quot;Gene silencing using micro-RNA designed hairpins,&quot; RNA, Vol. 8:842-850 (2002)."><meta name="citation_reference" content="Meister, Gunter et al, &quot;Sequence-specific inhibition of microRNA- and siRNA-induced RNA silencing,&quot; RNA, vol. 10:544-550 (2004)."><meta name="citation_reference" content="Mourelatos, Zissimos et al, &quot;miRNPs: a novel class of ribonucleoproteins containing numerous microRNAs,&quot; Genes &amp; Development, vol. 16:720-728 (2002)."><meta name="citation_reference" content="Mourrain, Philippe et al., &quot;Arabidopsis SGS2 and SGS3 Genes Are Required for Posttranscriptional Gnee Silencing and Natural Virus Resistance,&quot; Cell . vol. 101:533-542, (2000)."><meta name="citation_reference" content="Murchison, E.P. et al, &quot;miRNAs on the move: miRNA biogenesis and the RNAi machinery,&quot; Curr. Opin. Cell. Biol., vol. 16(3):223-229 (2004)."><meta name="citation_reference" content="Nykänen, Antti et al, &quot;ATP Requirements and Small Interfering RNA Structure in the RNA Interference Pathway,&quot; Cell, vol. 107:309-321 (2001)."><meta name="citation_reference" content="Ohnishi, Yusuke et al, &quot;Influence of assembly of siRNA elements into RNA-induced silencing complex by fork-siRNA duplex carrying nucleotide mismatches at the 3&#39;- or 5&#39;-end of the sense-stranded siRNA element,&quot; Biochemical and Biophysical Research Communications, vol. 329:516-521 (2005)."><meta name="citation_reference" content="Ohnishi, Yusuke et al, “Influence of assembly of siRNA elements into RNA-induced silencing complex by fork-siRNA duplex carrying nucleotide mismatches at the 3′- or 5′-end of the sense-stranded siRNA element,” Biochemical and Biophysical Research Communications, vol. 329:516-521 (2005)."><meta name="citation_reference" content="Pal-Bhadra, Manika et al., &quot;RNAi Related Mechanisms Affect both Transcriptional and Posttranscriptional Transgene Silencing in Drosophila,&quot; Molecular Cell. vol. 9:315-327, (2002)."><meta name="citation_reference" content="Park, Wonkeun et al., &quot;Carpel Factory, a Dicer Homolog, and HEN1, a Novel Protein, Act in microRNa Metabolism in Arabidopsis thaliana,&quot; Current Biology, vol. 12:1484-1495, (2002)."><meta name="citation_reference" content="Patzel, Volker et al, &quot;Design of siRNAs producing unstructured guide-RNAs results in improved RNA interference efficiency,&quot; Nature Biotechnology, vol. 23(11):1440-1444 (2005)."><meta name="citation_reference" content="Pusch, Oliver et al, &quot;Nucleotide sequence homology requirements of HIV-1-specific short hairpin RNA,&quot; Nucleic Acids Research, vol. 31(22):6444-6449 (2003)."><meta name="citation_reference" content="Reinhart, Brenda J. et al., &quot;MicroRNAs in plants,&quot; Genes &amp; Development, vol. 16:1616-1626 (2002)."><meta name="citation_reference" content="Reynolds, Angela et al, &quot;Rational siRNA design for RNA interference,&quot; Nature Biotechnology, vol. 22(3):326-330 (2004)."><meta name="citation_reference" content="Rhoades, Matthew W. et al., &quot;Prediction of Plant MicroRNA Targets,&quot; Cell, vol. 110:513-520 (2002)."><meta name="citation_reference" content="Ruvkun, Gary, &quot;Glimpses of a Tiny RNA World,&quot; Science, vol. 294:797-799, (2001)."><meta name="citation_reference" content="Saxena, Sandeep et al, &quot;Small RNAs with Imperfect Match to Endogenous mRNA Repress Translation,&quot; The Journal of Biological Chemistry, vol. 278(45):44312-44319 (2003)."><meta name="citation_reference" content="Schubert, Steffen et al., &quot;Local RNA Target Structure Influence siRNA Efficacy: Systematic Analysis of Intentionally Designed Binding Regions,&quot; J. Mol. Biol., vol. 348:883-893 (2005)."><meta name="citation_reference" content="Schwarz, Dianne S. et al, &quot;Asymmetry in the Assembly of the RNAi Enzyme Complex,&quot; Cell, vol. 115:199-208 (2003)."><meta name="citation_reference" content="Schwarz, Dianne S. et al, &quot;The RNA-Induced Silencing Complex Is a Mg2+-Dependent Endonuclease,&quot; Current Biology, vol. 14:787-791 (2004)."><meta name="citation_reference" content="Schwarz, Dianne S. et al, &quot;Why do miRNAs live in the miRNP?&quot; Genes &amp; Development, vol. 16:1025-1031 (2002)."><meta name="citation_reference" content="Schwarz, Dianne S. et al., &quot;Designing siRNA That Distinguish between Genes That Differ by a Single Nucleotide,&quot; PLoS Genetics, vol. 2(9):1-12 (2006)."><meta name="citation_reference" content="Semizarov, Dimitri et al, &quot;Specificity of short interfering RNA determined through gene expression signatures,&quot; PNAS, vol. 100(11):6347-6352 (2003)."><meta name="citation_reference" content="Smart, Nicola et al., &quot;A rapid and sensitive assay for quantification of siRNA efficiency and specificity,&quot; Biol. Proced. Online, vol. 7(1):1-7 (2005)."><meta name="citation_reference" content="Supplementary European Search Report for Application No. EP04753864, dated Mar. 5, 2007."><meta name="citation_reference" content="Tang, G. et al, &quot;Biochemical dissection of RNA silencing in plants,&quot; Methods Mol. Biol., vol. 257:223-244 (2004)."><meta name="citation_reference" content="Tang, Guiliang et al, &quot;A biochemical framework for RNA silencing in plants,&quot; Genes &amp; Development, vol. 17:49-63 (2003)."><meta name="citation_reference" content="Tomari, Yukihide et al, &quot;RISC assembly defects in the Drosophila RNAi mutant armitage,&quot; Department of Biochemistry and Molecular Pharmacology, 2004."><meta name="citation_reference" content="Vargason, Jeffrey M. et al, &quot;Size Selective Recognition of siRNA by an RNA Silencing Suppressor,&quot; Cell, vol. 115:799-811 (2003)."><meta name="citation_reference" content="Vaucheret, Herve et al., &quot;Post-transcriptional gene silencing in plants,&quot; Journal of Cell Science, vol. 114:3083-3091, (2001)."><meta name="citation_reference" content="Waterhouse, Peter M. et al., &quot;Gene silencing as an adaptive defense against viruses,&quot; Nature, vol. 411:834-842, (2001)."><meta name="citation_reference" content="Xu et al (Virology 171(2): 331-341, 1989)."><meta name="citation_reference" content="Zeng et al (RNA 9: 112-123, 2003)."><meta name="citation_patent_number" content="US:7732593"><meta name="citation_patent_application_number" content="US:12/139,072"><link rel="canonical" href="http://www.google.com/patents/US7732593"/><meta property="og:url" content="http://www.google.com/patents/US7732593"/><meta name="title" content="Patent US7732593 - Useful in the design of improved RNAi-mediating agents which allow for more exact control of the efficacy of RNA silencing; enhancing silencing of a target mRNA"/><meta name="description" content="Based at least in part on an understanding of the mechanisms by which small RNAs (e.g., naturally-occurring miRNAs) mediate RNA silencing in plants, rules have been established for determining, for example, the degree of complementarity required between an RNAi-mediating agent and its target, i.e., whether mismatches are tolerated, the number of mismatches tolerated, the effect of the position of the mismatches, etc. Such rules are useful, in particular, in the design of improved RNAi-mediating agents which allow for more exact control of the efficacy of RNA silencing."/><meta property="og:title" content="Patent US7732593 - Useful in the design of improved RNAi-mediating agents which allow for more exact control of the efficacy of RNA silencing; enhancing silencing of a target mRNA"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("PIftU_D_IoPEsATn6IHQCg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("GBR"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("PIftU_D_IoPEsATn6IHQCg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("GBR"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7732593?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7732593"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=15l3BgABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7732593&amp;usg=AFQjCNEEmCv6xK5GLQ_4wAzRyDCxBruyUA" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7732593.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7732593.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20090098614"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7732593"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7732593" style="display:none"><span itemprop="description">Based at least in part on an understanding of the mechanisms by which small RNAs (e.g., naturally-occurring miRNAs) mediate RNA silencing in plants, rules have been established for determining, for example, the degree of complementarity required between an RNAi-mediating agent and its target, i.e., whether...</span><span itemprop="url">http://www.google.com/patents/US7732593?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7732593 - Useful in the design of improved RNAi-mediating agents which allow for more exact control of the efficacy of RNA silencing; enhancing silencing of a target mRNA</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7732593 - Useful in the design of improved RNAi-mediating agents which allow for more exact control of the efficacy of RNA silencing; enhancing silencing of a target mRNA" title="Patent US7732593 - Useful in the design of improved RNAi-mediating agents which allow for more exact control of the efficacy of RNA silencing; enhancing silencing of a target mRNA"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7732593 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 12/139,072</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jun 8, 2010</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jun 13, 2008</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Jun 2, 2003</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2528012A1">CA2528012A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1633767A2">EP1633767A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1633767A4">EP1633767A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7459547">US7459547</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7772203">US7772203</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20050037988">US20050037988</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20080318896">US20080318896</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20090098614">US20090098614</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20100317105">US20100317105</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20110152347">US20110152347</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20140163084">US20140163084</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2004111191A2">WO2004111191A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2004111191A3">WO2004111191A3</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">12139072, </span><span class="patent-bibdata-value">139072, </span><span class="patent-bibdata-value">US 7732593 B2, </span><span class="patent-bibdata-value">US 7732593B2, </span><span class="patent-bibdata-value">US-B2-7732593, </span><span class="patent-bibdata-value">US7732593 B2, </span><span class="patent-bibdata-value">US7732593B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Phillip+D.+Zamore%22">Phillip D. Zamore</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Guiliang+Tang%22">Guiliang Tang</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22University+Of+Massachusetts%22">University Of Massachusetts</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7732593.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7732593.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7732593.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (43),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (84),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (4),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (18),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (5)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7732593&usg=AFQjCNEWrqr5NSG-ydlU5at9j_GwQYZY4Q">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7732593&usg=AFQjCNGuk5k9ENr4sQmNwZv9rrp-XAphTw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7732593B2%26KC%3DB2%26FT%3DD&usg=AFQjCNHtYzv2LglenGrKHf6US-_4ResnVg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT130111393" lang="EN" load-source="ifi">Useful in the design of improved RNAi-mediating agents which allow for more exact control of the efficacy of RNA silencing; enhancing silencing of a target mRNA</invention-title></span><br><span class="patent-number">US 7732593 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA78601176" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">Based at least in part on an understanding of the mechanisms by which small RNAs (e.g., naturally-occurring miRNAs) mediate RNA silencing in plants, rules have been established for determining, for example, the degree of complementarity required between an RNAi-mediating agent and its target, i.e., whether mismatches are tolerated, the number of mismatches tolerated, the effect of the position of the mismatches, etc. Such rules are useful, in particular, in the design of improved RNAi-mediating agents which allow for more exact control of the efficacy of RNA silencing.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(12)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7732593B2/US07732593-20100608-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7732593B2/US07732593-20100608-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7732593B2/US07732593-20100608-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7732593B2/US07732593-20100608-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7732593B2/US07732593-20100608-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7732593B2/US07732593-20100608-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7732593B2/US07732593-20100608-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7732593B2/US07732593-20100608-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7732593B2/US07732593-20100608-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7732593B2/US07732593-20100608-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7732593B2/US07732593-20100608-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7732593B2/US07732593-20100608-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7732593B2/US07732593-20100608-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7732593B2/US07732593-20100608-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7732593B2/US07732593-20100608-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7732593B2/US07732593-20100608-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7732593B2/US07732593-20100608-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7732593B2/US07732593-20100608-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7732593B2/US07732593-20100608-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7732593B2/US07732593-20100608-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7732593B2/US07732593-20100608-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7732593B2/US07732593-20100608-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7732593B2/US07732593-20100608-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7732593B2/US07732593-20100608-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(30)</span></span></div><div class="patent-text"><div mxw-id="PCLM31980478" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A method of enhancing RNA silencing activity of an RNAi agent in a mammalian cell or a plant cell comprising:
<div class="claim-text">(i) selecting a target sequence in an mRNA expressed in the mammalian or plant cell; and</div>
<div class="claim-text">(ii) synthesizing an RNAi agent comprising an antisense strand that is complementary to the target sequence, wherein three, four or five nucleotides within 5 or fewer nucleotides from the 3′ end of the antisense strand are substituted with a nucleotide which does not form a Watson-Crick base pair when the antisense strand is base paired with the target sequence, such that the RNA silencing activity of the RNAi agent is enhanced.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the RNAi agent is small interfering RNA (siRNA).</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the RNAi agent is a micro RNA (miRNA).</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="CLM-00003">claim 3</claim-ref>, wherein the RNAi agent is a plant miRNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. The method of <claim-ref idref="CLM-00003">claim 3</claim-ref>, wherein the RNAi agent is an animal miRNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the RNAi agent is chemically synthesized.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the RNAi agent is enzymatically synthesized.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the RNAi agent is formulated to facilitate entry of the agent into the cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the RNAi agent is derived from an engineered precursor.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The method of <claim-ref idref="CLM-00009">claim 9</claim-ref>, wherein the engineered precursor is a miRNA precursor (pre-miRNA).</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref>, wherein the pre-miRNA is encoded by a vector.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. The method of <claim-ref idref="CLM-00009">claim 9</claim-ref>, wherein the engineered precursor is a primary miRNA transcript (pri-miRNA).</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00013" num="00013" class="claim">
      <div class="claim-text">13. The method of <claim-ref idref="CLM-00012">claim 12</claim-ref>, wherein the pri-miRNA is encoded by a vector.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00014" num="00014" class="claim">
      <div class="claim-text">14. The method of <claim-ref idref="CLM-00009">claim 9</claim-ref>, wherein the engineered precursor is a small hairpin RNA (shRNA) comprising a stem portion comprising the antisense strand.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00015" num="00015" class="claim">
      <div class="claim-text">15. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref>, wherein the shRNA is encoded by a vector.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00016" num="00016" class="claim">
      <div class="claim-text">16. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the cell is a mammalian cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00017" num="00017" class="claim">
      <div class="claim-text">17. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the cell is a human cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00018" num="00018" class="claim">
      <div class="claim-text">18. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the cell is a plant cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00019" num="00019" class="claim">
      <div class="claim-text">19. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the mRNA specifies the amino acid sequence of a cellular protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00020" num="00020" class="claim">
      <div class="claim-text">20. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the mRNA specifies the amino acid sequence of an extracellular protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00021" num="00021" class="claim">
      <div class="claim-text">21. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the mRNA specifies the amino acid sequence of a pathogen-associated protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00022" num="00022" class="claim">
      <div class="claim-text">22. The method of <claim-ref idref="CLM-00021">claim 21</claim-ref>, wherein the pathogen-associated protein is a viral protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00023" num="00023" class="claim">
      <div class="claim-text">23. The method of <claim-ref idref="CLM-00021">claim 21</claim-ref>, wherein the pathogen-associated protein is expressed by a host of a pathogen.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00024" num="00024" class="claim">
      <div class="claim-text">24. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the mRNA specifies the amino acid sequence of an endogenous protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00025" num="00025" class="claim">
      <div class="claim-text">25. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the mRNA specifies the amino acid sequence of a heterologous protein expressed in a recombinant cell or a genetically altered organism.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00026" num="00026" class="claim">
      <div class="claim-text">26. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the mRNA specifies the amino acid sequence of a protein encoded by a transgene.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00027" num="00027" class="claim">
      <div class="claim-text">27. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the mRNA specifies the amino acid sequence of a protein encoded by a pathogen which is capable of infecting a cell or an organism from which the cell is derived.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00028" num="00028" class="claim">
      <div class="claim-text">28. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein each of three nucleotides within 5 or fewer nucleotides from the 3′ end of the antisense strand are substituted with a nucleotide which does not form a Watson-Crick base pair.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00029" num="00029" class="claim">
      <div class="claim-text">29. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein each of four nucleotides within 5 or fewer nucleotides from the 3′ end of the antisense strand are substituted with a nucleotide which does not form a Watson-Crick base pair.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00030" num="00030" class="claim">
      <div class="claim-text">30. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein each of five nucleotides within 5 or fewer nucleotides from the 3′ end of the antisense strand are substituted with a nucleotide which does not form a Watson-Crick base pair.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES37349717" lang="EN" load-source="patent-office" class="description">
    <heading>RELATED APPLICATIONS</heading> <p num="p-0002">This application is a Continuation of U.S. patent application Ser. No. 10/859,337 (now U.S. Pat. No. 7,459,547), entitled, “Methods and Compositions for Controlling the Efficacy of RNA silencing”, filed Jun. 2, 2004, which claims the benefit of U.S. Provisional Patent Application Ser. No. 60/475,386, entitled “Methods and Compositions for Controlling Efficacy of RNA Silencing”, filed Jun. 2, 2003. The entire contents of the above-referenced applications are incorporated herein by this reference.</p>
    <heading>STATEMENT OF FEDERALLY SPONSORED RESEARCH</heading> <p num="p-0003">This invention was made with government support under Grant no. GM062862 awarded by the National Institutes of Health. The government has certain rights in the invention.</p>
    <heading>GOVERNMENT RIGHTS</heading> <p num="p-0004">This invention was made at least in part with government support under grant no. GM62862-01 awarded by the National Institutes of Health. The government may have certain rights in this invention.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p num="p-0005">RNA interference (RNAi) in animals and basal eukaryotes, quelling in fungi, and posttranscriptional gene silencing (PTGS) in plants are examples of a broad family of phenomena collectively called RNA silencing (Kooter et al. 1999; Li and Ding 2001; Matzke et al. 2001; Vaucheret et al. 2001; Waterhouse et al. 2001; Hannon 2002; Plasterk 2002). The unifying features of RNA silencing phenomena are the production of small (21-26 nt) RNAs that act as specificity determinants for down-regulating gene expression (Hamilton and Baulcombe 1999; Hammond et al. 2000; Parrish et al. 2000; Zamore et al. 2000; Djikeng et al. 2001; Parrish and Fire 2001; Tijsterman et al. 2002) and the requirement for one or more members of the Argonaute family of proteins (or PPD proteins, named for their characteristic PAZ and Piwi domains) (Tabara et al. 1999; Fagard et al. 2000; Hammond et al. 2001; Hutvágner and Zamore 2002; Kennerdell et al. 2002; Martinez et al. 2002a; Pal-Bhadra et al. 2002; Williams and Rubin 2002).</p>
    <p num="p-0006">Small RNAs are generated in animals by members of the Dicer family of double-stranded RNA (dsRNA)-specific endonucleases (Bernstein et al. 2001; Billy et al. 2001; Grishok et al. 2001; Ketting et al. 2001). Dicer family members are large, multidomain proteins that contain putative RNA helicase, PAZ, two tandem ribonuclease III (RNase III), and one or two dsRNA-binding domains. The tandem RNase III domains are believed to mediate endonucleolytic cleavage of dsRNA into small interfering RNAs (siRNAs), the mediators of RNAi. In <i>Drosophila </i>and mammals, siRNAs, together with one or more Argonaute proteins, form a protein-RNA complex, the RNA-induced silencing complex (RISC), which mediates the cleavage of target RNAs at sequences with extensive complementarity to the siRNA (Hammond et al. 2000, 2001; Zamore et al. 2000; Elbashir et al. 2001a, b, c; Nykänen et al. 2001; Hutvágner and Zamore 2002; Martinez et al. 2002a).</p>
    <p num="p-0007">In addition to Dicer and Argonaute proteins, RNA-dependent RNA polymerase (RdRP) genes are required for RNA silencing in <i>Caenorhabditis elegans </i>(Smardon et al. 2000; Sijen et al. 2001), <i>Neurospora crassa </i>(Cogoni and Macino 1999), and <i>Dictyostelium discoideum </i>(Martens et al. 2002), but likely not for RNAi in <i>Drosophila </i>or mammals (Celotto and Graveley 2002; Chiu and Rana 2002; Holen et al. 2002; Martinez et al. 2002b; Schwarz et al. 2002; Roignant et al. 2003). In plants, PTGS initiated by transgenes that overexpress an endogenous mRNA also requires a putative RdRP, SGS2 (SDE1; Dalmay et al. 2000; Mourrain et al. 2000), although transgenes designed to generate dsRNA bypass this requirement (Beclin et al. 2002). Similarly, silencing induced by viruses replicating through a dsRNA intermediate (virus-induced gene silencing, VIGS) does not require SGS2 (Dalmay et al. 2000).</p>
    <p num="p-0008">Dicer in animals and CARPEL FACTORY (CAF, a Dicer homolog) in plants also generate microRNAs (miRNAs), 20-24-nt, single-stranded noncoding RNAs thought to regulate endogenous mRNA expression (Lee et al. 1993; Reinhart et al. 2000, 2002; Grishok et al. 2001; Hutvágner et al. 2001; Ketting et al. 2001; Lagos-Quintana et al. 2001, 2002; Lau et al. 2001; Lee and Ambros 2001; Mourelatos et al. 2002; Park et al. 2002). miRNAs are produced by Dicer cleavage of stem-loop precursor RNA transcripts (pre-miRNAs); the miRNA can reside on either the 5′ or 3′ side of the double-stranded stem (Lee et al. 1993; Pasquinelli et al. 2000; Lagos-Quintana et al. 2001; Lau et al. 2001; Lee and Ambros 2001). In animals, pre-miRNAs are transcribed as longer primary transcripts (pri-miRNAs) that are processed in the nucleus into compact, folded structures (pre-miRNAs), then exported to the cytoplasm, where they are cleaved by Dicer to yield mature miRNAs (Lee et al. 2002). Animal miRNAs are only partially complementary to their target mRNAs; this partial complementarity has been proposed to cause miRNAs to repress translation of their targets, rather than direct target cleavage by the RNAi pathway (for review, see Ruvkun 2001; Hutvágner and Zamore 2002). Plant miRNAs have far greater complementarity to cellular mRNAs and have been proposed to mediate target RNA cleavage via an RNAi-like mechanism (Llave et al. 2002b; Rhoades et al. 2002).</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0009">The present invention is based, at least in part, on the finding that extracts of wheat germ recapitulate many of the key features of RNA silencing in plants. Using this in vitro system, it is shown that in plants, ATP-dependent, Dicer-like enzymes cleave dsRNA into small RNAs that have the structure of siRNAs. Unlike <i>Drosophila </i>embryos or mammalian cells, plants convert dsRNA into two distinct classes of siRNAs, long and short siRNAs. Inhibitor studies indicate that a different Dicer-like enzyme generates each siRNA class. Furthermore, a wheat RdRP activity can synthesize dsRNA using exogenous single-stranded RNA as a template without an exogenous primer, and that this dsRNA is preferentially converted into long siRNAs.</p>
    <p num="p-0010">Wheat germ extracts also contain an endogenous RISC programmed with a miRNA which can direct efficient cleavage of the wild-type <i>Arabidopsis </i>PHAVOLUTA (PHV) mRNA sequence, but not that of a previously described dominant PHV mutant. Interestingly, exact complementarity between the miRNA and target mRNA is not necessary for the miRNA to direct efficient target cleavage. An siRNA containing three mismatches with its target mRNA, was found to be at least as potent as an siRNA with perfect complementarity to the same target sequence, demonstrating that mismatches per se do not block target cleavage. Rather, the specific position and sequence of siRNA:target RNA mismatches determine if they permit or disrupt RNAi. It is proposed that three or four mismatches between an miRNA (or the guide strand of an siRNA duplex) and its target RNA, properly placed so as to still permit mRNA cleavage, facilitates the release of cleaved target RNA from the RISC complex, thereby increasing the rate of enzyme turnover. In particular, the efficiency of cleavage is greater when a G:U base pair, referred to also as a G:U wobble, is present near the 5′ or 3′ end of the complex formed between the miRNA and the target. Understanding the natural mechanism by which miRNAs efficienty mediate RNAi in plants allows for the design of improved RNAi agents for use in mediating RNAi not only in plants, but in eukaryotes (in particular, in mammals).</p>
    <p num="p-0011">Accordingly, the present invention features methods of enhancing the efficacy of an RNAi agent comprising substituting a at least one terminal nucleotide with a nucleotide that does not form a Watson-Crick base pair with the corresponding nucleotide in a target mRNA. The invention also provides compositions comprising RNAi agents, e.g., siRNAs, pre-miRNA, shRNAs, having nucleotide substitutions for enhanced efficacy of RNAi, as well as vectors, transgenes and cells comprising the RNAi agents. Further featured is a Dicer-like enzyme, an extract comprising the enzyme, and methods for their use. Kits for use in mediating RNAi comprising the compositions of the invention are provided. Therapeutic methods and pharmaceutical compositions are also provided.</p>
    <p num="p-0012">Other features and advantages of the invention will be apparent from the following detailed description and claims.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p num="p-0013"> <figref idrefs="DRAWINGS">FIG. 1</figref>. <i>Arabidopsis thaliana </i>small RNAs form two distinct size classes. (A) Size distribution of small RNA clones. (B) Sequence composition of the 5′ ends of cloned small RNA as a function of length.</p>
      <p num="p-0014"> <figref idrefs="DRAWINGS">FIG. 2</figref>. dsRNA is cleaved into two discrete classes of bona fide siRNAs in plant extracts. (A) Upon incubation in wheat germ extract, 32P-dsRNA was cleaved into small RNAs in a highly processive reaction, as in fly embryo lysate. (B) 32P-dsRNA was cleaved in wheat germ extract into two sizes of small RNAs, ˜21-nt and ˜24-25-nt long, relative to synthetic 5′-32P-radiolabeled RNA markers. (C) 32P-dsRNA was cleaved in cauliflower extract into two sizes of small RNAs. (D) Efficient production of small RNAs in wheat germ extract required ATP. ATP, creatine phosphate, and creatine kinase were included (+ATP) or omitted ([−] ATP) from the reaction, (E) Small RNAs produced in vitro in wheat germ extract are double-stranded. 32P-dsRNA was incubated in wheat germ extract or <i>Drosophila </i>embryo lysate, deproteinized at room temperature without organic extraction, then analyzed by gel filtration on a Superdex 200 HR column. The peak positions of double- and single-stranded synthetic siRNA standards are indicated. (F) Scheme for detecting 3′ overhanging ends on small RNAs by nuclease protection. (G) Small RNAs produced by incubation of 32P-dsRNA in wheat germ extract have ˜2-nt 3′ overhanging ends and a central double-stranded body, characteristics of the products of Dicer cleavage. Brackets indicate the nuclease digestion products. The positions of 5′-32P-radiolabeled size markers are indicated at left. 3′-phosphorylated markers were generated by reacting synthetic RNAs one base longer than indicated with periodate, followed by [beta]-elimination, yielding an RNA one base shorter, but bearing a 3′-phosphate in place of a hydroxyl.</p>
      <p num="p-0015"> <figref idrefs="DRAWINGS">FIG. 3</figref>. The two classes of plant siRNAs are produced by different enzymes. (A,B) 32P-dsRNA was incubated in either <i>Drosophila </i>embryo lysate or wheat germ extract for 3 hours in the presence of increasing concentrations of 21-nt or 25-nt siRNA duplexes, then analyzed by denaturing gel electrophoresis and quantified. The siRNA concentration is presented in micromoles nucleotide per liter to permit comparison of the different lengths of siRNA duplex used. The relative efficiency of the reactions was determined by fitting the data to a single exponential and comparing the rate constant. (A) 21-nt siRNA duplexes (filled circles) are more efficient inhibitors of <i>Drosophila </i>Dicer than 25-nt siRNA duplexes (open circles). (B) Production of 25-nt siRNAs in wheat germ extract (squares) was inhibited more efficiently by a 25-nt synthetic siRNA duplex (red symbols) than by a 21-nt siRNA duplex (black symbols), but production of the 21-nt siRNAs (circles) was not inhibited by either synthetic siRNA duplex. (C) dsRNA competitor inhibited the production of both 25-nt (black squares) and 21-nt (red circles) siRNAs in wheat germ extract. Production of siRNAs in <i>Drosophila </i>embryo lysate (blue circles) was also inhibited by dsRNA competitor, but to a lower extent, perhaps reflecting a higher concentration of Dicer in <i>Drosophila </i>embryo lysate than in wheat germ extract.</p>
      <p num="p-0016"> <figref idrefs="DRAWINGS">FIG. 4</figref>. Wheat germ extract contains an RdRP activity. Single-stranded RNA of the indicated size and cap structure was incubated in wheat germ extract for 3 hours in the presence of ATP, CTP, GTP, and α-32P-UTP. The products of the reaction were analyzed by denaturing polyacrylamide gel electrophoresis.</p>
      <p num="p-0017"> <figref idrefs="DRAWINGS">FIG. 5</figref>. Characterization of the wheat RdRP activity. (A) Wheat germ extract, but not <i>Drosophila </i>embryo lysate, contains an RdRP activity that can extend a primer. The arrowhead indicates the primer extension product produced when an antisense 21-nt RNA primer, but not a sense primer, was incubated in the wheat germ extract with a 592-nt single-stranded RNA. The primers correspond to nucleotides 511-532 of the RNA template. (B) RdRP-dependent production of small RNAs in wheat germ extract. Increasing concentrations of a 2.7-kb <i>Photinus pyralis </i>(Pp) luciferase mRNA triggered production of 32P-radiolabeled small RNAs in wheat germ extract when ribonucleotide triphosphates (including α-32P-UTP), but not when 3′-deoxy GTP and 3′-deoxy CTP were included in the reaction. (C) Production of newly synthesized small RNAs was more efficiently inhibited by a 25-nt synthetic siRNA duplex (open circles) than by a 21-nt synthetic siRNA duplex (open squares).</p>
      <p num="p-0018"> <figref idrefs="DRAWINGS">FIG. 6</figref>. miR165/166 in wheat germ extract. (A) A wheat ortholog of miR165 or miR166 is present in wheat germ extract. Quantitative Northern hybridization analysis using synthetic miR165 RNA concentration standards, antisense miR165 RNA, and total RNA prepared from 30 μL of wheat germ extract or <i>Drosophila </i>embryo lysate. (B) Quantitation of the data in A. Closed circles, synthetic miR165 standards; open circle, RNA extracted from 30 μL of wheat germ extract. The line shows a linear fit of the four highest concentration standards. (C) Schematic of the RNA targets, indicating the sequences (SEQ ID NOS: 1, 17, 15, 23, 3, 24, and 4, respectively) of the miR165/166-complementary regions of wild-type PHV and mutant phv mRNAs, miR165, miR166, and the siRNA antisense strands used in <figref idrefs="DRAWINGS">FIG. 7C</figref>.</p>
      <p num="p-0019"> <figref idrefs="DRAWINGS">FIG. 7</figref>. An endogenous wheat nuclease efficiently cleaves wild-type but not mutant PHV target RNAs. (A) When incubated in wheat germ extract, 5′-radiolabeled target RNA containing wild-type PHV sequences was cleaved within the PHV sequences complementary to miR165 and miR166. In contrast, a dominant G→A mutant target RNA was cleaved inefficiently. (B) Quantification of the data in A. (Circles) Wild-type PHV sequences; (squares) mutant sequences; (filled symbols) full-length target RNA; (open symbols) 5′ cleavage product. The difference in cleavage rates is ˜14-fold. (C) Analysis of PHV cleavage in an in vitro RNAi reaction programmed with siRNA duplexes and <i>Drosophila </i>embryo lysate. The identity of the antisense strand of the siRNA duplex and the RNA target used is indicated above the gel and described in <figref idrefs="DRAWINGS">FIG. 6C</figref>.</p>
      <p num="p-0020"> <figref idrefs="DRAWINGS">FIG. 8</figref>. Quantification of the fraction of target mRNA cleaved by siRNA having perfect complementarity to target versus siRNA having two mismatched with target (miR 165 siRNA).</p>
    </description-of-drawings> <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p num="p-0021">The present invention is based, at least in part, on the discovery that extracts of wheat germ, introduced for the study of translation and protein translocation in the 1970s (Roberts and Paterson 1973), recapitulate many of the key features of RNA silencing in plants. Using this in vitro system, the instant inventors have shown that in plants, ATP-dependent, Dicer-like enzymes cleave dsRNA into small RNAs that have the structure of siRNAs. Unlike <i>Drosophila </i>embryos or mammalian cells, plants convert dsRNA into two distinct classes of siRNAs, long (e.g., 21-22 nucleotides) and short (e.g., 24-25 nucleotides) siRNAs. Inhibitor studies indicate that a second Dicer-like enzyme functions in plants to generate each siRNA class. The instant inventors have also shown that a wheat RdRP activity can synthesize dsRNA using exogenous single-stranded RNA as a template without an exogenous primer, and that this dsRNA is preferentially converted into long siRNAs.</p>
    <p num="p-0022">Finally, it is demonstrated that wheat germ extracts contain an endogenous RISC programmed with a miRNA. This endogenous miRNA complex has sufficient sequence information to direct efficient cleavage of the wild-type <i>Arabidopsis </i>PHA VOLUTA (PHV) mRNA sequence, but not that of a previously described dominant PHV mutant that perturbs leaf development. Based on an understanding of the mechanism by which miRNAs direct RNAi in plants, new siRNAs can be designed for regulating RNAi in plants. More importantly, siRNAs can be designed, for example, based on the sequence of various eukaryotic miRNAs, such siRNAs having utility in mediating RNAi in mammals, and particularly, in humans.</p>
    <p num="p-0023">Accordingly, in one aspect, the instant invention provides a method of enhancing the efficacy of an RNAi agent, involving substituting at least one terminal nucleotide of the RNAi agent with a nucleotide which does not form a Watson-Crick base pair with the corresponding nucleotide in a target mRNA, such that efficacy is enhanced.</p>
    <p num="p-0024">In one embodiment, the substituted nucleotide forms a G:U wobble base pair with the target mRNA. In one preferred embodiment, the substitution is an A→G substitution, the G forming a G:U wobble base pair with a U in the corresponding target mRNA. In another preferred embodiment, the substitution is a C→U substitution, the U forming a G:U wobble base pair with a G in the corresponding target mRNA.</p>
    <p num="p-0025">In one embodiment, the terminal nucleotide is within 5 or fewer nucleotides from the 5′ end of the RNAi agent. In a related embodiment, the terminal nucleotide is within 5 or fewer nucleotides from the 3′ end of the RNAi agent.</p>
    <p num="p-0026">In one embodiment, at least two terminal nucleotides are substituted. In preferred embodiments, the two terminal nucleotides substituted are at the 5′ end of the RNAi agent or at the 3′ end of the RNAi agent. In another preferred embodiment, a first terminal nucleotide substituted is at the 5′ end of the RNAi agent and a second terminal nucleotide substituted is at the 3′ end of the RNAi agent.</p>
    <p num="p-0027">In one embodiment, at least three, four or five terminal nucleotides are substituted.</p>
    <p num="p-0028">In another aspect, the instant invention provides a RNAi agent having at least one terminal nucleotide of the RNAi agent substituted with a nucleotide which forms a G:U wobble base pair with the corresponding nucleotide in a target mRNA.</p>
    <p num="p-0029">In one embodiment of this aspect of the invention, the substitution is an A→G substitution, the G forming a G:U wobble base pair with a U in the corresponding target mRNA. In another embodiment, the substitution is a C→U substitution, the U forming a G:U wobble base pair with a G in the corresponding target mRNA.</p>
    <p num="p-0030">In other embodiments, the terminal nucleotide is within 5 or fewer nucleotides from the 5′ end of the RNAi agent or from the 3′ end of the RNAi agent.</p>
    <p num="p-0031">In yet other embodiments, at least two terminal nucleotides are substituted. In preferred embodiments, the two terminal nucleotides substituted are at the 5′ end of the RNAi agent or at the 3′ end of the RNAi agent. In other preferred embodiments, a first terminal nucleotide substituted is at the 5′ end of the RNAi agent and a second terminal nucleotide substituted is at the 3′ end of the RNAi agent.</p>
    <p num="p-0032">In other embodiments, at least three, four or five terminal nucleotides are substituted.</p>
    <p num="p-0033">In various embodiments of this aspect of the invention, the RNAi agent is chemically synthesized, enzymatically synthesized, or derived from an engineered precursor.</p>
    <p num="p-0034">In another aspect, the instant invention provides a method of enhancing silencing of a target mRNA, comprising contacting a cell having an RNAi pathway with the RNAi agent of any one of the preceding claims under conditions such that silencing is enhanced.</p>
    <p num="p-0035">In yet another aspect, the instant invention provides a method of enhancing silencing of a target mRNA in a subject, comprising administering to the subject a pharmaceutical composition comprising the RNAi agent of any one of the preceding claims such that silencing is enhanced.</p>
    <p num="p-0036">In certain embodiments of the invention, compositions are provided comprising the RNAi agents of the invention formulated to facilitate entry of the agent into a cell. Pharmaceutical compositions comprising the RNAi agents of the invention are also provided.</p>
    <p num="p-0037">In other embodiments, the instant invention provides engineered pre-miRNA comprising the RNAi agent of any one of the preceding claims, and vectors encoding the pre-miRNA.</p>
    <p num="p-0038">In related embodiments, the instant invention provides a pri-miRNA comprising the pre-miRNA of the invention, and a vector encoding the pri-miRNA.</p>
    <p num="p-0039">In yet other embodiments, the invention provides a small hairpin RNA (shRNA) comprising nucleotide sequence identical to any of the RNAi agents of the instant invention, a vector encoding the shRNA, and a transgene encoding the shRNA.</p>
    <p num="p-0040">The instant invention further provides a cell, e.g., a mammalian cell, preferably a human cell, comprising the vectors of the invention. 35.</p>
    <p num="p-0041">In another aspect, the instant invention provides an isolated <i>Arabidopsis thaliana </i>Dicer-like enzyme capable of cleaving a long dsRNA substrate into short, 24-25 nucleotide dsRNA products, the activity of said enzyme being inhibited in the presence of said dsRNA products. In a related aspect, the instant invention provides a method of generating a RNAi agent 24-25 nucleotides in length, comprising incubating a dsRNA substrate with the enzyme of the invention, such that the agent is generated. Also provided is an <i>Arabidopsis thaliana </i>cell-free extract comprising the enzyme of the invention. In a related aspect, a method is provided of generating a RNAi agent 24-25 nucleotides in length, comprising incubating a dsRNA substrate with the extract of the invention, such that the agent is generated.</p>
    <p num="p-0042">In a certain embodiment of the instant invention, a kit is provided for use in mediating RNAi, comprising the enzyme or the extract of the invention, and instructions for use.</p>
    <p num="p-0043">So that the invention may be more readily understood, certain terms are first defined.</p>
    <p num="p-0044">The term “nucleoside” refers to a molecule having a purine or pyrimidine base covalently linked to a ribose or deoxyribose sugar. Exemplary nucleosides include adenosine, guanosine, cytidine, uridine and thymidine. Additional exemplary nucleosides include inosine, 1-methyl inosine, pseudouridine, 5,6-dihydrouridine, ribothymidine, <sup>2</sup>N-methylguanosine and <sup>2,2</sup>N,N-dimethylguanosine (also referred to as “rare” nucleosides). The term “nucleotide” refers to a nucleoside having one or more phosphate groups joined in ester linkages to the sugar moiety. Exemplary nucleotides include nucleoside monophosphates, diphosphates and triphosphates. The terms “polynucleotide” and “nucleic acid molecule” are used interchangeably herein and refer to a polymer of nucleotides joined together by a phosphodiester linkage between 5′ and 3′ carbon atoms.</p>
    <p num="p-0045">The term “RNA” or “RNA molecule” or “ribonucleic acid molecule” refers to a polymer of ribonucleotides. The term “DNA” or “DNA molecule” or deoxyribonucleic acid molecule” refers to a polymer of deoxyribonucleotides. DNA and RNA can be synthesized naturally (e.g., by DNA replication or transcription of DNA, respectively). RNA can be post-transcriptionally modified. DNA and RNA can also be chemically synthesized. DNA and RNA can be single-stranded (i.e., ssRNA and ssDNA, respectively) or multi-stranded (e.g., double stranded, i.e., dsRNA and dsDNA, respectively). “mRNA” or “messenger RNA” is single-stranded RNA that specifies the amino acid sequence of one or more polypeptide chains. This information is translated during protein synthesis when ribosomes bind to the mRNA.</p>
    <p num="p-0046">As used herein, the term “small interfering RNA” (“siRNA”) (also referred to in the art as “short interfering RNAs”) refers to an RNA (or RNA analog) comprising between about 10-50 nucleotides (or nucleotide analogs) which is capable of directing or mediating RNA interference. Preferably, an siRNA comprises between about 15-30 nucleotides or nucleotide analogs, more preferably between about 16-25 nucleotides (or nucleotide analogs), even more preferably between about 18-23 nucleotides (or nucleotide analogs), and even more preferably between about 19-22 nucleotides (or nucleotide analogs) (e.g., 19, 20, 21 or 22 nucleotides or nucleotide analogs). The term “short” siRNA refers to a siRNA comprising ˜21 nucleotides (or nucleotide analogs), for example, 19, 20, 21 or 22 nucleotides. The term “long” siRNA refers to a siRNA comprising ˜24-25 nucleotides, for example, 23, 24, 25 or 26 nucleotides. Short siRNAs may, in some instances, include fewer than 19 nucleotides, e.g., 16, 17 or 18 nucleotides, provided that the shorter siRNA retains the ability to mediate RNAi. Likewise, long siRNAs may, in some instances, include more than 26 nucleotides, provided that the longer siRNA retains the ability to mediate RNAi absent further processing, e.g., enzymatic processing, to a short siRNA.</p>
    <p num="p-0047">The term “nucleotide analog” or “altered nucleotide” or “modified nucleotide” refers to a non-standard nucleotide, including non-naturally occurring ribonucleotides or deoxyribonucleotides. Preferred nucleotide analogs are modified at any position so as to alter certain chemical properties of the nucleotide yet retain the ability of the nucleotide analog to perform its intended function. Examples of positions of the nucleotide which may be derivitized include the 5 position, e.g., 5-(2-amino)propyl uridine, 5-bromo uridine, 5-propyne uridine, 5-propenyl uridine, etc.; the 6 position, e.g., 6-(2-amino)propyl uridine; the 8-position for adenosine and/or guanosines, e.g., 8-bromo guanosine, 8-chloro guanosine, 8-fluoroguanosine, etc. Nucleotide analogs also include deaza nucleotides, e.g., 7-deaza-adenosine; O- and N-modified (e.g., alkylated, e.g., N6-methyl adenosine, or as otherwise known in the art) nucleotides; and other heterocyclically modified nucleotide analogs such as those described in Herdewijn, <i>Antisense Nucleic Acid Drug Dev., </i>2000 Aug. 10(4):297-310.</p>
    <p num="p-0048">Nucleotide analogs may also comprise modifications to the sugar portion of the nucleotides. For example the 2′ OH-group may be replaced by a group selected from H, OR, R, F, Cl, Br, I, SH, SR, NH<sub>2</sub>, NHR, NR<sub>2</sub>, COOR, or OR, wherein R is substituted or unsubstituted C<sub>1</sub>-C<sub>6 </sub>alkyl, alkenyl, alkynyl, aryl, etc. Other possible modifications include those described in U.S. Pat. Nos. 5,858,988, and 6,291,438.</p>
    <p num="p-0049">The phosphate group of the nucleotide may also be modified, e.g., by substituting one or more of the oxygens of the phosphate group with sulfur (e.g., phosphorothioates), or by making other substitutions which allow the nucleotide to perform its intended function such as described in, for example, Eckstein, <i>Antisense Nucleic Acid Drug Dev. </i>2000 Apr. 10(2):117-21, Rusckowski et al. <i>Antisense Nucleic Acid Drug Dev. </i>2000 Oct. 10(5):333-45, Stein, <i>Antisense Nucleic Acid Drug Dev. </i>2001 Oct. 11(5): 317-25, Vorobjev et al. <i>Antisense Nucleic Acid Drug Dev. </i>2001 Apr. 11(2):77-85, and U.S. Pat. No. 5,684,143. Certain of the above-referenced modifications (e.g., phosphate group modifications) preferably decrease the rate of hydrolysis of, for example, polynucleotides comprising said analogs in vivo or in vitro.</p>
    <p num="p-0050">The term “oligonucleotide” refers to a short polymer of nucleotides and/or nucleotide analogs. The term “RNA analog” refers to an polynucleotide (e.g., a chemically synthesized polynucleotide) having at least one altered or modified nucleotide as compared to a corresponding unaltered or unmodified RNA but retaining the same or similar nature or function as the corresponding unaltered or unmodified RNA. As discussed above, the oligonucleotides may be linked with linkages which result in a lower rate of hydrolysis of the RNA analog as compared to an RNA molecule with phosphodiester linkages. For example, the nucleotides of the analog may comprise methylenediol, ethylene diol, oxymethylthio, oxyethylthio, oxycarbonyloxy, phosphorodiamidate, phosphoroamidate, and/or phosphorothioate linkages. Preferred RNA analogues include sugar- and/or backbone-modified ribonucleotides and/or deoxyribonucleotides. Such alterations or modifications can further include addition of non-nucleotide material, such as to the end(s) of the RNA or internally (at one or more nucleotides of the RNA). An RNA analog need only be sufficiently similar to natural RNA that it has the ability to mediate (mediates) RNA interference.</p>
    <p num="p-0051">As used herein, the term “RNA interference” (“RNAi”) refers to a selective intracellular degradation of RNA. RNAi occurs in cells naturally to remove foreign RNAs (e.g., viral RNAs). Natural RNAi proceeds via fragments cleaved from free dsRNA which direct the degradative mechanism to other similar RNA sequences. Alternatively, RNAi can be initiated by the hand of man, for example, to silence the expression of target genes.</p>
    <p num="p-0052">A RNAi agent having a strand which is “sequence sufficiently complementary to a target mRNA sequence to direct target-specific RNA interference (RNAi)” means that the strand has a sequence sufficient to trigger the destruction of the target mRNA by the RNAi machinery or process.</p>
    <p num="p-0053">The term “phosphorylated” means that at least one phosphate group is attached to a chemical (e.g., organic) compound. Phosphate groups can be attached, for example, to proteins or to sugar moieties via the following reaction: free hydroxyl group+phosphate donor→phosphate ester linkage. The term “5′ phosphorylated” is used to describe, for example, polynucleotides or oligonucleotides having a phosphate group attached via ester linkage to the C5 hydroxyl of the 5′ sugar (e.g., the 5′ ribose or deoxyribose, or an analog of same). Mono-, di-, and triphosphates are common. Also intended to be included within the scope of the instant invention are phosphate group analogs which function in the same or similar manner as the mono-, di-, or triphosphate groups found in nature (see e.g., exemplified analogs.)</p>
    <p num="p-0054">As used herein, the term “isolated RNA” (e.g., “isolated siRNA” or “isolated siRNA precursor”) refers to RNA molecules which are substantially free of other cellular material, or culture medium when produced by recombinant techniques, or substantially free of chemical precursors or other chemicals when chemically synthesized.</p>
    <p num="p-0055">The term “in vitro” has its art recognized meaning, e.g., involving purified reagents or extracts, e.g., cell extracts. The term “in vivo” also has its art recognized meaning, e.g., involving living cells, e.g., immortalized cells, primary cells, cell lines, and/or cells in an organism.</p>
    <p num="p-0056">As used herein, the term “transgene” refers to any nucleic acid molecule, which is inserted by artifice into a cell, and becomes part of the genome of the organism that develops from the cell. Such a transgene may include a gene that is partly or entirely heterologous (i.e., foreign) to the transgenic organism, or may represent a gene homologous to an endogenous gene of the organism. The term “transgene” also means a nucleic acid molecule that includes one or more selected nucleic acid sequences, e.g., DNAs, that encode one or more engineered RNA precursors, to be expressed in a transgenic organism, e.g., animal, which is partly or entirely heterologous, i.e., foreign, to the transgenic animal, or homologous to an endogenous gene of the transgenic animal, but which is designed to be inserted into the animal's genome at a location which differs from that of the natural gene. A transgene includes one or more promoters and any other DNA, such as introns, necessary for expression of the selected nucleic acid sequence, all operably linked to the selected sequence, and may include an enhancer sequence.</p>
    <p num="p-0057">A gene “involved” in a disorder includes a gene, the normal or aberrant expression or function of which effects or causes a disease or disorder or at least one symptom of said disease or disorder</p>
    <p num="p-0058">The phrase “examining the function of a gene in a cell or organism” refers to examining or studying the expression, activity, function or phenotype arising therefrom.</p>
    <p num="p-0059">Various methodologies of the instant invention include step that involves comparing a value, level, feature, characteristic, property, etc., to a “suitable control”, referred to interchangeably herein as an “appropriate control”. A “suitable control” or “appropriate control” is any control or standard familiar to one of ordinary skill in the art useful for comparison purposes. In one embodiment, a “suitable control” or “appropriate control” is a value, level, feature, characteristic, property, etc., determined prior to performing an RNAi methodology, as described herein. For example, a transcription rate, mRNA level, translation rate, protein level, biological activity, cellular characteristic or property, genotype, phenotype, etc., can be determined prior to introducing a RNAi agent of the invention into a cell or organism. In another embodiment, a “suitable control” or “appropriate control” is a value, level, feature, characteristic, property, etc. determined in a cell or organism, e.g., a control or normal cell or organism, exhibiting, for example, normal traits. In yet another embodiment, a “suitable control” or “appropriate control” is a predefined value, level, feature, characteristic, property, etc.</p>
    <p num="p-0060">Various aspects of the invention are described in further detail in the following subsections.</p>
    <p num="p-0061">I. RNA Molecules</p>
    <p num="p-0062">The present invention features “RNAi agents”, methods of making said RNAi agents and methods (e.g., research and/or therapeutic methods) for using said RNAi agents. The RNAi agents can be siRNA molecules, precursor molecules (e.g., engineered precursor molecules) that are processed into siRNA molecules, or molecules (e.g., DNA molecules) that encode, for example, precursor molecules (e.g., engineered precursor molecules)</p>
    <p num="p-0063">Exemplary siRNA molecules have a length from about 10-50 or more nucleotides. Preferably, siRNA molecule has a length from about 15-45 or 15-30 nucleotides. More preferably, the siRNA molecule has a length from about 16-25 or 18-23 nucleotides. The siRNA molecules of the invention further comprise at least one strand that has a sequence that is “sufficiently complementary” to a target mRNA sequence to direct target-specific RNA interference (RNAi), as defined herein, i.e., the strand has a sequence sufficient to trigger the destruction of the target mRNA by the RNAi machinery or process. Such a strand can be referred to as an antisense strand in the context of a ds-siRNA molecule. The siRNA molecule can be designed such that every residue is complementary to a residue in the target molecule. Preferably, however, the siRNA molecule is designed such that modified base pairing, in particular, G:U base pairing (i.e., G:U “wobble” base pairing) occurs between the strand of the siRNA molecule mediating RNAi and the target mRNA.</p>
    <p num="p-0064">In further embodiments, substitutions can be made within the molecule to increase stability and/or enhance processing activity of said molecule. Substitutions can be made within the strand or can be made to residues a the ends of the strand. Preferably, however, substitutions are not made in the central portion of the strand as the sequence of this portion of the strand has been determined to be essential to effecting cleavage of the corresponding target mRNA. The 5′-terminus is, most preferably, phosphorylated (i.e., comprises a phosphate, diphosphate, or triphosphate group). The 3′ end of an siRNA can be a hydroxyl group although there is no requirement for a 3′ hydroxyl group when the active agent is a ss-siRNA molecule.</p>
    <p num="p-0065">The target RNA cleavage reaction guided by siRNAs is highly sequence specific. In general, siRNA containing a nucleotide sequences identical to a portion of the target gene are preferred for inhibition. However, 100% sequence identity between the siRNA and the target gene is not required to practice the present invention. Thus the invention has the advantage of being able to tolerate sequence variations that might be expected due to genetic mutation, strain polymorphism, or evolutionary divergence. For example, siRNA sequences with insertions, deletions, and single point mutations relative to the target sequence have also been found to be effective for inhibition. Alternatively, siRNA sequences with nucleotide analog substitutions or insertions can be effective for inhibition.</p>
    <p num="p-0066">Moreover, not all positions of a siRNA contribute equally to target recognition. Mismatches in the center of the siRNA are most critical and essentially abolish target RNA cleavage. In contrast, the 3′ nucleotides of the siRNA do not contribute significantly to specificity of the target recognition. In particular, residues 3′ of the siRNA sequence which is complementary to the target RNA (e.g., the guide sequence) are not critical for target RNA cleavage.</p>
    <p num="p-0067">Sequence identity may determined by sequence comparison and alignment algorithms known in the art. To determine the percent identity of two nucleic acid sequences (or of two amino acid sequences), the sequences are aligned for optimal comparison purposes (e.g., gaps can be introduced in the first sequence or second sequence for optimal alignment). The nucleotides (or amino acid residues) at corresponding nucleotide (or amino acid) positions are then compared. When a position in the first sequence is occupied by the same residue as the corresponding position in the second sequence, then the molecules are identical at that position. The percent identity between the two sequences is a function of the number of identical positions shared by the sequences (i.e., % homology= # of identical positions/total # of positions×100), optionally penalizing the score for the number of gaps introduced and/or length of gaps introduced.</p>
    <p num="p-0068">The comparison of sequences and determination of percent identity between two sequences can be accomplished using a mathematical algorithm. In one embodiment, the alignment generated over a certain portion of the sequence aligned having sufficient identity but not over portions having low degree of identity (i.e., a local alignment). A preferred, non-limiting example of a local alignment algorithm utilized for the comparison of sequences is the algorithm of Karlin and Altschul (1990) <i>Proc. Natl. Acad. Sci. USA </i>87:2264-68, modified as in Karlin and Altschul (1993) <i>Proc. Natl. Acad. Sci. USA </i>90:5873-77. Such an algorithm is incorporated into the BLAST programs (version 2.0) of Altschul, et al. (1990) <i>J. Mol. Biol. </i>215:403-10.</p>
    <p num="p-0069">In another embodiment, the alignment is optimized by introducing appropriate gaps and percent identity is determined over the length of the aligned sequences (i.e., a gapped alignment). To obtain gapped alignments for comparison purposes, Gapped BLAST can be utilized as described in Altschul et al., (1997) <i>Nucleic Acids Res. </i>25(17):3389-3402. In another embodiment, the alignment is optimized by introducing appropriate gaps and percent identity is determined over the entire length of the sequences aligned (i.e., a global alignment). A preferred, non-limiting example of a mathematical algorithm utilized for the global comparison of sequences is the algorithm of Myers and Miller, <i>CABIOS </i>(1989). Such an algorithm is incorporated into the ALIGN program (version 2.0) which is part of the GCG sequence alignment software package. When utilizing the ALIGN program for comparing amino acid sequences, a PAM120 weight residue table, a gap length penalty of 12, and a gap penalty of 4 can be used.</p>
    <p num="p-0070">Greater than 90% sequence identity, e.g., 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99% or even 100% sequence identity, between a strand of the RNAi agent and the portion of the target gene is preferred. Alternatively, the RNAi agent may be defined functionally as a nucleotide sequence (or oligonucleotide sequence) that is capable of hybridizing with a portion of the target gene transcript (e.g., 400 mM NaCl, 40 mM PIPES pH 6.4, 1 mM EDTA, 50° C. or 70° C. hybridization for 12-16 hours; followed by washing). Additional preferred hybridization conditions include hybridization at 70° C. in 1×SSC or 50° C. in 1×SSC, 50% formamide followed by washing at 70° C. in 0.3×SSC or hybridization at 70° C. in 4×SSC or 50° C. in 4×SSC, 50% formamide followed by washing at 67° C. in 1×SSC. The hybridization temperature for hybrids anticipated to be less than 50 base pairs in length should be 5-10° C. less than the melting temperature (Tm) of the hybrid, where Tm is determined according to the following equations. For hybrids less than 18 base pairs in length, Tm(° C.)=2(# of A+T bases)+4(# of G+C bases). For hybrids between 18 and 49 base pairs in length, Tm(° C.)=81.5+16.6(log10[Na+])+0.41(% G+C)— (600/N), where N is the number of bases in the hybrid, and [Na+] is the concentration of sodium ions in the hybridization buffer ([Na+] for 1×SSC=0.165 M). Additional examples of stringency conditions for polynucleotide hybridization are provided in Sambrook, J., E. F. Fritsch, and T. Maniatis, 1989<i>, Molecular Cloning: A Laboratory Manual</i>, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., chapters 9 and 11, and <i>Current Protocols in Molecular Biology, </i>1995, F. M. Ausubel et al., eds., John Wiley &amp; Sons, Inc., sections 2.10 and 6.3-6.4, incorporated herein by reference. The length of the identical nucleotide sequences may be at least about 10, 12, 15, 17, 20, 22, 25, 27, 30, 32, 35, 37, 40, 42, 45, 47 or 50 bases.</p>
    <p num="p-0071">In a preferred aspect, the RNA molecules of the present invention are modified to improve stability in serum or in growth medium for cell cultures. In order to enhance the stability, the 3′-residues may be stabilized against degradation, e.g., they may be selected such that they consist of purine nucleotides, particularly adenosine or guanosine nucleotides. Alternatively, substitution of pyrimidine nucleotides by modified analogues, e.g., substitution of uridine by 2′-deoxythymidine is tolerated and does not affect the efficiency of RNA interference. For example, the absence of a 2′ hydroxyl may significantly enhance the nuclease resistance of the RNA agents in tissue culture medium.</p>
    <p num="p-0072">In an especially preferred embodiment of the present invention the RNA molecule may contain at least one modified nucleotide analogue. The nucleotide analogues may be located at positions where the target-specific activity, e.g., the RNAi mediating activity is not substantially effected, e.g., in a region at the 5′-end and/or the 3′-end of the RNA molecule. Particularly, the ends may be stabilized by incorporating modified nucleotide analogues.</p>
    <p num="p-0073">Preferred nucleotide analogues include sugar- and/or backbone-modified ribonucleotides (i.e., include modifications to the phosphate-sugar backbone). For example, the phosphodiester linkages of natural RNA may be modified to include at least one of a nitrogen or sulfur heteroatom. In preferred backbone-modified ribonucleotides the phosphoester group connecting to adjacent ribonucleotides is replaced by a modified group, e.g., of phosphothioate group. In preferred sugar-modified ribonucleotides, the 2′ OH-group is replaced by a group selected from H, OR, R, halo, SH, SR, NH<sub>2</sub>, NHR, NR<sub>2 </sub>or ON, wherein R is C<sub>1</sub>-C<sub>6 </sub>alkyl, alkenyl or alkynyl and halo is F, Cl, Br or I.</p>
    <p num="p-0074">Also preferred are nucleobase-modified ribonucleotides, i.e., ribonucleotides, containing at least one non-naturally occurring nucleobase instead of a naturally occurring nucleobase. Bases may be modified to block the activity of adenosine deaminase. Exemplary modified nucleobases include, but are not limited to, uridine and/or cytidine modified at the 5-position, e.g., 5-(2-amino)propyl uridine, 5-bromo uridine; adenosine and/or guanosines modified at the 8 position, e.g., 8-bromo guanosine; deaza nucleotides, e.g., 7-deaza-adenosine; O- and N-alkylated nucleotides, e.g., N6-methyl adenosine are suitable. It should be noted that the above modifications may be combined.</p>
    <p num="p-0075">RNA may be produced enzymatically or by partial/total organic synthesis, any modified nibonucleotide can be introduced by in vitro enzymatic or organic synthesis. In one embodiment, a RNAi agent is prepared chemically. Methods of synthesizing RNA molecules are known in the art, in particular, the chemical synthesis methods as de scribed in Verma and Eckstein (1998) <i>Annul Rev. Biochem. </i>67:99-134. In another embodiment, a RNAi agent is prepared enzymatically. For example, a ds-siRNA can be prepared by enzymatic processing of a long ds RNA having sufficient complementarity to the desired target mRNA. Processing of long ds RNA can be accomplished in vitro, for example, using appropriate cellular lysates and ds-siRNAs can be subsequently purified by gel electrophoresis or gel filtration. ds-siRNA can then be denatured according to art-recognized methodologies. In an exemplary embodiment, RNA can be purified from a mixture by extraction with a solvent or resin, precipitation, electrophoresis, chromatography, or a combination thereof. Alternatively, the RNA may be used with no or a minimum of purification to avoid losses due to sample processing. Alternatively, the single-stranded RNAs can also be prepared by enzymatic transcription from synthetic DNA templates or from DNA plasmids isolated from recombinant bacteria. Typically, phage RNA polymerases are used such as T7, T3 or SP6 RNA polymerase (Milligan and Uhlenbeck (1989) <i>Methods Enzymol. </i>180:51-62). The RNA may be dried for storage or dissolved in an aqueous solution. The solution may contain buffers or salts to inhibit annealing, and/or promote stabilization of the single strands.</p>
    <p num="p-0076">In one embodiment, the target mRNA of the invention specifies the amino acid sequence of a cellular protein (e.g., a nuclear, cytoplasmic, transmembrane, or membrane-associated protein). In another embodiment, the target mRNA of the invention specifies the amino acid sequence of an extracellular protein (e.g., an extracellular matrix protein or secreted protein). As used herein, the phrase “specifies the amino acid sequence” of a protein means that the mRNA sequence is translated into the amino acid sequence according to the rules of the genetic code. The following classes of proteins are listed for illustrative purposes: developmental proteins (e.g., adhesion molecules, cyclin kinase inhibitors, Wnt family members, Pax family members, Winged helix family members, Hox family members, cytokines/lymphokines and their receptors, growth/differentiation factors and their receptors, neurotransmitters and their receptors); oncogene-encoded proteins (e.g., ABLI, BCLI, BCL2, BCL6, CBFA2, CBL, CSFIR, ERBA, ERBB, EBRB2, ETSI, ETSI, ETV6, FGR, FOS, FYN, HCR, HRAS, JUN, KRAS, LCK, LYN, MDM2, MLL, MYB, MYC, MYCLI, MYCN, NRAS, PIM I, PML, RET, SRC, TALI, TCL3, and YES); tumor suppressor proteins (e.g., APC, BRCA1, BRCA2, MADH4, MCC, NF I, NF2, RB I, TP53, and WTI); and enzymes (e.g., ACC synthases and oxidases, ACP desaturases and hydroxylases, ADP-glucose pyrophorylases, ATPases, alcohol dehydrogenases, amylases, amyloglucosidases, catalases, cellulases, chalcone synthases, chitinases, cyclooxygenases, decarboxylases, dextriinases, DNA and RNA polymerases, galactosidases, glucanases, glucose oxidases, granule-bound starch synthases, GTPases, helicases, hernicellulases, integrases, inulinases, invertases, isomerases, kinases, lactases, lipases, lipoxygenases, lysozymes, nopaline synthases, octopine synthases, pectinesterases, peroxidases, phosphatases, phospholipases, phosphorylases, phytases, plant growth regulator synthases, polygalacturonases, proteinases and peptidases, pullanases, recombinases, reverse transcriptases, RUBISCOs, topoisomerases, and xylanases).</p>
    <p num="p-0077">In a preferred aspect of the invention, the target mRNA molecule of the invention specifies the amino acid sequence of a protein associated with a pathological condition. For example, the protein may be a pathogen-associated protein (e.g., a viral protein involved in immunosuppression of the host, replication of the pathogen, transmission of the pathogen, or maintenance of the infection), or a host protein which facilitates entry of the pathogen into the host, drug metabolism by the pathogen or host, replication or integration of the pathogen's genome, establishment or spread of infection in the host, or assembly of the next generation of pathogen. Alternatively, the protein may be a tumor-associated protein or an autoimmune disease-associated protein.</p>
    <p num="p-0078">In one embodiment, the target mRNA molecule of the invention specifies the amino acid sequence of an endogenous protein (i.e., a protein present in the genome of a cell or organism). In another embodiment, the target mRNA molecule of the invention specified the amino acid sequence of a heterologous protein expressed in a recombinant cell or a genetically altered organism. In another embodiment, the target mRNA molecule of the invention specified the amino acid sequence of a protein encoded by a transgene (i.e., a gene construct inserted at an ectopic site in the genome of the cell). In yet another embodiment, the target mRNA molecule of the invention specifies the amino acid sequence of a protein encoded by a pathogen genome which is capable of infecting a cell or an organism from which the cell is derived.</p>
    <p num="p-0079">By inhibiting the expression of such proteins, valuable information regarding the function of said proteins and therapeutic benefits which may be obtained from said inhibition may be obtained.</p>
    <p num="p-0080">In one embodiment, RNAi agents are synthesized either in vivo, in situ, or in vitro. Endogenous RNA polymerase of the cell may mediate transcription in vivo or in situ, or cloned RNA polymerase can be used for transcription in vivo or in vitro. For transcription from a transgene in vivo or an expression construct, a regulatory region (e.g., promoter, enhancer, silencer, splice donor and acceptor, polyadenylation) may be used to transcribe the RNAi agent. Inhibition may be targeted by specific transcription in an organ, tissue, or cell type; stimulation of an environmental condition (e.g., infection, stress, temperature, chemical inducers); and/or engineering transcription at a developmental stage or age. A transgenic organism that expresses an RNAi agent from a recombinant construct may be produced by introducing the construct into a zygote, an embryonic stem cell, or another multipotent cell derived from the appropriate organism.</p>
    <p num="p-0081">II. Short Hairpin RNAs (shRNAs)</p>
    <p num="p-0082">In certain featured embodiments, the instant invention features shRNAs which can be processed into siRNAs, for example, by a cell's endogenous RNAi machinery. In contrast to short siRNA duplexes, short hairpin RNAs (shRNAs) mimics the natural precursors of miRNAs and enters at the top of the RNAi pathway. For this reason, shRNAs are believed to mediate RNAi more efficiently by being fed through the entire natural RNAi pathway.</p>
    <p num="p-0083">1. Engineered RNA Precursors that Generate siRNAs</p>
    <p num="p-0084">Naturally-occurring miRNA precursors (pre-miRNA) have a single strand that forms a duplex stem including two portions that are generally complementary, and a loop, that connects the two portions of the stem. In typical pre-miRNAs, the stem includes one or more bulges, e.g., extra nucleotides that create a single nucleotide “loop” in one portion of the stem, and/or one or more unpaired nucleotides that create a gap in the hybridization of the two portions of the stem to each other. Short hairpin RNAs, or engineered RNA precursors, of the invention are artificial constructs based on these naturally occurring pre-miRNAs, but which are engineered to deliver desired siRNAs.</p>
    <p num="p-0085">In shRNAs, or engineered precursor RNAs, of the instant invention, one portion of the duplex stem is a nucleic acid sequence that is complementary (or anti-sense) to the target mRNA. Thus, engineered RNA precursors include a duplex stem with two portions and a loop connecting the two stem portions. The two stem portions are about 18 or 19 to about 25, 30, 35, 37, 38, 39, or 40 or more nucleotides in length. When used in mammalian cells, the length of the stem portions should be less than about 30 nucleotides to avoid provoking non-specific responses like the interferon pathway. In non-mammalian cells, the stem can be longer than 30 nucleotides. In fact, the stem can include much larger sections complementary to the target mRNA (up to, and including the entire mRNA). The two portions of the duplex stem must be sufficiently complementary to hybridize to form the duplex stem. Thus, the two portions can be, but need not be, fully or perfectly complementary. In addition, the two stem portions can be the same length, or one portion can include an overhang of 1, 2, 3, or 4 micleotides. The overhanging nucleotides can include, for example, uracils (Us), e.g., all Us. The loop in the shRNAs or engineered RNA precursors may differ from natural pre-miRNA sequences by modifying the loop sequence to increase or decrease the number of paired nucleotides, or replacing all or part of the loop sequence with a tetraloop or other loop sequences. Thus, the loop in the shRNAs or engineered RNA precursors can be 2, 3, 4, 5, 6, 7, 8, 9, or more, e.g., 15 or 20, or more nucleotides in length.</p>
    <p num="p-0086">shRNAs of the invention include the sequences of the desired siRNA duplex. The desired siRNA duplex, and thus both of the two stem portions in the engineered RNA precursor, are selected by methods known in the art. These include, but are not limited to, selecting an 18, 19, 20, 21 nucleotide, or longer, sequence from the target gene mRNA sequence from a region 100 to 200 or 300 nucleotides on the 3′ side of the start of translation. In general, the sequence can be selected from any portion of the mRNA from the target gene, such as the 5′ UTR (untranslated region), coding sequence, or 3′ UTR. This sequence can optionally follow immediately after a region of the target gene containing two adjacent AA nucleotides. The last two nucleotides of the 21 or so nucleotide sequence can be selected to be UU (so that the anti-sense strand of the siRNA begins with UU). This 21 or so nucleotide sequence is used to create one portion of a duplex stem in the engineered RNA precursor. This sequence can replace a stem portion of a wild-type pre-stRNA sequence, e.g., enzymatically, or is included in a complete sequence that is synthesized. For example, one can synthesize DNA oligonucleotides that encode the entire stem-loop engineered RNA precursor, or that encode just the portion to be inserted into the duplex stem of the precursor, and using restriction enzymes to build the engineered RNA precursor construct, e.g., from a wild-type pre-stRNA.</p>
    <p num="p-0087">Engineered RNA precursors include in the duplex stem the 21-22 or so nucleotide sequences of the siRNA desired to be produced in vivo. Thus, the stem portion of the engineered RNA precursor includes at least 18 or 19 nucleotide pairs corresponding to the sequence of an exonic portion of the gene whose expression is to be reduced or inhibited. The two 3′ nucleotides flanking this region of the stem are chosen so as to maximize the production of the siRNA from the engineered RNA precursor, and to maximize the efficacy of the resulting siRNA in targeting the corresponding mRNA for destruction by RNAi in vivo and in vitro.</p>
    <p num="p-0088">Another defining feature of these engineered RNA precursors is that as a consequence of their length, sequence, and/or structure, they do not induce sequence non-specific responses, such as induction of the interferon response or apoptosis, or that they induce a lower level of such sequence non-specific responses than long, double-stranded RNA (&gt;150 bp) that has been used to induce RNAi. For example, the interferon response is triggered by dsRNA longer than 30 base pairs.</p>
    <p num="p-0089">2. Transgenes Encoding Engineered RNA Precursors</p>
    <p num="p-0090">The new engineered RNA precursors can be synthesized by standard methods known in the art, e.g., by use of an automated DNA synthesizer (such as are commercially available from Biosearch, Applied Biosystems, etc.). These synthetic, engineered RNA precursors can be used directly as described below or cloned into expression vectors by methods known in the field. The engineered RNA precursors should be delivered to cells in vitro or in vivo in which it is desired to target a specific mRNA for destruction. A number of methods have been developed for delivering DNA or RNA to cells. For example, for in vivo delivery, molecules can be injected directly into a tissue site or administered systemically. In vitro delivery includes methods known in the art such as electroporation and lipofection.</p>
    <p num="p-0091">To achieve intracellular concentrations of the nucleic acid molecule sufficient to suppress expression of endogenous mRNAs, one can use, for example, a recombinant DNA construct in which the oligonucleotide is placed under the control of a strong Pol III (e.g., U6 or Pol III H1-RNA promoter) or Pol II promoter. The use of such a construct to transfect target cells in vitro or in vivo will result in the transcription of sufficient amounts of the engineered RNA precursor to lead to the production of an siRNA that can target a corresponding mRNA sequence for cleavage by RNAi to decrease the expression of the gene encoding that mRNA. For example, a vector can be introduced in vivo such that it is taken up by a cell and directs the transcription of an engineered RNA precursor. Such a vector can remain episomal or become chromosomally integrated, as long as it can be transcribed to produce the desired stRNA precursor.</p>
    <p num="p-0092">Such vectors can be constructed by recombinant DNA technology methods known in the art. Vectors can be plasmid, viral, or other vectors known in the art such as those described herein, used for replication and expression in mammalian cells or other targeted cell types. The nucleic acid sequences encoding the engineered RNA precursors can be prepared using known techniques. For example, two synthetic DNA oligonucleotides can be synthesized to create a novel gene encoding the entire engineered RNA precursor. The DNA oligonucleotides, which will pair, leaving appropriate ‘sticky ends’ for cloning, can be inserted into a restriction site in a plasmid that contains a promoter sequence (e.g., a Pol II or a Pol III promoter) and appropriate terminator sequences 3′ to the engineered RNA precursor sequences (e.g., a cleavage and polyadenylation signal sequence from SV40 or a Pol III terminator sequence).</p>
    <p num="p-0093">The invention also encompasses genetically engineered host cells that contain any of the foregoing expression vectors and thereby express the nucleic acid molecules of the invention in the host cell. The host cells can be cultured using known techniques and methods (see, e.g., Culture of Animal Cells (R. I. Freshney, Alan R. Liss, Inc. 1987); Molecular Cloning, Sambrook et al. (Cold Spring Harbor Laboratory Press, 1989)).</p>
    <p num="p-0094">Successful introduction of the vectors of the invention into host cells can be monitored using various known methods. For example, transient transfection can be signaled with a reporter, such as a fluorescent marker, such as Green Fluorescent Protein (GFP). Stable transfection can be indicated using markers that provide the transfected cell with resistance to specific environmental factors (e.g., antibiotics and drugs), such as hygromycin B resistance, e.g., in insect cells and in mammalian cells.</p>
    <p num="p-0095">3. Regulatory Sequences</p>
    <p num="p-0096">The expression of the engineered RNA precursors is driven by regulatory sequences, and the vectors of the invention can include any regulatory sequences known in the art to act in mammalian cells, e.g., human or murine cells; in insect cells; in plant cells; or other cells. The term regulatory sequence includes promoters, enhancers, and other expression control elements. It will be appreciated that the appropriate regulatory sequence depends on such factors as the future use of the cell or transgenic animal into which a sequence encoding an engineered RNA precursor is being introduced, and the level of expression of the desired RNA precursor. A person skilled in the art would be able to choose the appropriate regulatory sequence. For example, the transgenic animals described herein can be used to determine the role of a test polypeptide or the engineered RNA precursors in a particular cell type, e.g., a hematopoietic cell. In this case, a regulatory sequence that drives expression of the transgene ubiquitously, or a hematopoietic-specific regulatory sequence that expresses the transgene only in hematopoietic cells, can be used. Expression of the engineered RNA precursors in a hematopoietic cell means that the cell is now susceptible to specific, targeted RNAi of a particular gene. Examples of various regulatory sequences are described below.</p>
    <p num="p-0097">The regulatory sequences can be inducible or constitutive. Suitable constitutive regulatory sequences include the regulatory sequence of a housekeeping gene such as the α-actin regulatory sequence, or may be of viral origin such as regulatory sequences derived from mouse mammary tumor virus (MMTV) or cytomegalovirus (CMV).</p>
    <p num="p-0098">Alternatively, the regulatory sequence can direct transgene expression in specific organs or cell types (see, e.g., Lasko et al., 1992<i>, Proc. Natl. Acad. Sci. USA </i>89:6232). Several tissue-specific regulatory sequences are known in the art including the albumin regulatory sequence for liver (Pinkert et al., 1987<i>, Genes Dev. </i>1:268276); the endothelin regulatory sequence for endothelial cells (Lee, 1990<i>, J. Biol. Chem. </i>265:10446-50); the keratin regulatory sequence for epidermis; the myosin light chain-2 regulatory sequence for heart (Lee et al., 1992<i>, J. Biol. Chem. </i>267:15875-85), and the insulin regulatory sequence for pancreas (Bucchini et al., 1986<i>, Proc. Natl. Acad. Sci. USA </i>83:2511-2515), or the vav regulatory sequence for hematopoietic cells (Oligvy et al., 1999<i>, Proc. Natl. Acad. Sci. USA </i>96:14943-14948). Another suitable regulatory sequence, which directs constitutive expression of transgenes in cells of hematopoietic origin, is the murine MHC class I regulatory sequence (Morello et al., 1986<i>, EMBO J. </i>5:1877-1882). Since NMC expression is induced by cytokines, expression of a test gene operably linked to this regulatory sequence can be upregulated in the presence of cytokines.</p>
    <p num="p-0099">In addition, expression of the transgene can be precisely regulated, for example, by using an inducible regulatory sequence and expression systems such as a regulatory sequence that is sensitive to certain physiological regulators, e.g., circulating glucose levels, or hormones (Docherty et al., 1994, FASEB J. 8:20-24). Such inducible expression systems, suitable for the control of transgene expression in cells or in mammals such as mice, include regulation by ecdysone, by estrogen, progesterone, tetracycline, chemical inducers of dimerization, and isopropyl-beta-D1-thiogalactopyranoside (IPTG) (collectively referred to as “the regulatory molecule”). Each of these expression systems is well described in the literature and permits expression of the transgene throughout the animal in a manner controlled by the presence or absence of the regulatory molecule. For a review of inducible expression systems, see, e.g., Mills, 2001<i>, Genes Devel. </i>15:1461-1467, and references cited therein.</p>
    <p num="p-0100">The regulatory elements referred to above include, but are not limited to, the cytomegalovirus hCMV immediate early gene, the early or late promoters of SV40 adenovirus (Bemoist et al., <i>Nature, </i>290:304, 1981), the tet system, the lac system, the trp system, the TAC system, the TRC system, the major operator and promoter regions of phage A, the control regions of fd coat protein, the promoter for 3-phosphoglycerate kinase, the promoters of acid phosphatase, and the promoters of the yeast α-mating factors. Additional promoters include the promoter contained in the 3′ long terminal repeat of Rous sarcoma virus (Yamamoto et al., <i>Cell </i>22:787-797, 1988); the herpes thymidine kinase promoter (Wagner et al., <i>Proc. Natl. Acad. Sci. USA </i>78:1441, 1981); or the regulatory sequences of the metallothionein gene (Brinster et al., <i>Nature </i>296:39, 1988).</p>
    <p num="p-0101">4. Assay for Testing Engineered RNA Precursors</p>
    <p num="p-0102"> <i>Drosophila </i>embryo lysates can be used to determine if an engineered RNA precursor was, in fact, the direct precursor of a mature stRNA or siRNA. This lysate assay is described in Tuschl et al., 1999, supra, Zamore et al., 2000, supra, and Hutvdgner et al. 2001, supra. These lysates recapitulate RNAi in vitro, thus permitting investigation into whether the proposed precursor RNA was cleaved into a mature stRNA or siRNA by an RNAi-like mechanism. Briefly, the precursor RNA is incubated with <i>Drosophila </i>embryo lysate for various times, and then assayed for the production of the mature siRNA or stRNA by primer extension or Northern hybridization. As in the in vivo setting, mature RNA accumulates in the cell-free reaction. Thus, an RNA corresponding to the proposed precursor can be shown to be converted into a mature stRNA or siRNA duplex in the <i>Drosophila </i>embryo lysate.</p>
    <p num="p-0103">Furthermore, an engineered RNA precursor can be functionally tested in the <i>Drosophila </i>embryo lysates. In this case, the engineered RNA precursor is incubated in the lysate in the presence of a 5′ radiolabeled target mRNA in a standard in vitro RNAi reaction for various lengths of time. The target mRNA can be 5′ radiolabeled using guanylyl transferase (as described in Tuschl et al., 1999, supra and references therein) or other suitable methods. The products of the in vitro reaction are then isolated and analyzed on a denaturing acrylamide or agarose gel to determine if the target mRNA has been cleaved in response to the presence of the engineered RNA precursor in the reaction. The extent and position of such cleavage of the mRNA target will indicate if the engineering of the precursor created a pre-siRNA capable of mediating sequence-specific RNAi.</p>
    <p num="p-0104">III. Methods of Introducing RNAs, Vectors, and Host Cells</p>
    <p num="p-0105">Physical methods of introducing nucleic acids include injection of a solution containing the RNA, bombardment by particles covered by the RNA, soaking the cell or organism in a solution of the RNA, or electroporation of cell membranes in the presence of the RNA. A viral construct packaged into a viral particle would accomplish both efficient introduction of an expression construct into the cell and transcription of RNA encoded by the expression construct. Other methods known in the art for introducing nucleic acids to cells may be used, such as lipid-mediated carrier transport, chemical-mediated transport, such as calcium phosphate, and the like. Thus the RNA may be introduced along with components that perform one or more of the following activities: enhance RNA uptake by the cell, inhibit annealing of single strands, stabilize the single strands, or other-wise increase inhibition of the target gene.</p>
    <p num="p-0106">RNA may be directly introduced into the cell (i.e., intracellularly); or introduced extracellularly into a cavity, interstitial space, into the circulation of an organism, introduced orally, or may be introduced by bathing a cell or organism in a solution containing the RNA. Vascular or extravascular circulation, the blood or lymph system, and the cerebrospinal fluid are sites where the RNA may be introduced.</p>
    <p num="p-0107">The cell with the target gene may be derived from or contained in any organism. The organism may a plant, animal, protozoan, bacterium, virus, or fungus. The plant may be a monocot, dicot or gymnosperm; the animal may be a vertebrate or invertebrate. Preferred microbes are those used in agriculture or by industry, and those that are pathogenic for plants or animals. Fungi include organisms in both the mold and yeast morphologies. Plants include <i>arabidopsis</i>; field crops (e.g., alfalfa, barley, bean, corn, cotton, flax, pea, rape, nice, rye, safflower, sorghum, soybean, sunflower, tobacco, and wheat); vegetable crops (e.g., asparagus, beet, broccoli, cabbage, carrot, cauliflower, celery, cucumber, eggplant, lettuce, onion, pepper, potato, pumpkin, radish, spinach, squash, taro, tomato, and zucchini); fruit and nut crops (e.g., almond, apple, apricot, banana, black-berry, blueberry, cacao, cherry, coconut, cranberry, date, faJoa, filbert, grape, grapefruit, guava, kiwi, lemon, lime, mango, melon, nectarine, orange, papaya, passion fruit, peach, peanut, pear, pineapple, pistachio, plum, raspberry, strawberry, tangerine, walnut, and watermelon); and ornamentals (e.g., alder, ash, aspen, azalea, birch, boxwood, camellia, carnation, chrysanthemum, elm, fir, ivy, jasmine, juniper, oak, palm, poplar, pine, redwood, rhododendron, rose, and rubber). Examples of vertebrate animals include fish, mammal, cattle, goat, pig, sheep, rodent, hamster, mouse, rat, primate, and human; invertebrate animals include nematodes, other worms, <i>drosophila</i>, and other insects.</p>
    <p num="p-0108">The cell having the target gene may be from the germ line or somatic, totipotent or pluripotent, dividing or non-dividing, parenchyma or epithelium, immortalized or transformed, or the like. The cell may be a stem cell or a differentiated cell. Cell types that are differentiated include adipocytes, fibroblasts, myocytes, cardiomyocytes, endothelium, neurons, glia, blood cells, megakaryocytes, lymphocytes, macrophages, neutrophils, eosinophils, basophils, mast cells, leukocytes, granulocytes, keratinocytes, chondrocytes, osteoblasts, osteoclasts, hepatocytes, and cells of the endocrine or exocrine glands.</p>
    <p num="p-0109">Depending on the particular target gene and the dose of double stranded RNA material delivered, this process may provide partial or complete loss of function for the target gene. A reduction or loss of gene expression in at least 50%, 60%, 70%, 80%, 90%, 95% or 99% or more of targeted cells is exemplary. Inhibition of gene expression refers to the absence (or observable decrease) in the level of protein and/or mRNA product from a target gene. Specificity refers to the ability to inhibit the target gene without manifest effects on other genes of the cell. The consequences of inhibition can be confirmed by examination of the outward properties of the cell or organism (as presented below in the examples) or by biochemical techniques such as RNA solution hybridization, nuclease protection, Northern hybridization, reverse transcription, gene expression monitoring with a microarray, antibody binding, enzyme linked immunosorbent assay (ELISA), Western blotting, radioimmunoassay (RIA), other immunoassays, and fluorescence activated cell analysis (FACS).</p>
    <p num="p-0110">For RNA-mediated inhibition in a cell line or whole organism, gene expression is conveniently assayed by use of a reporter or drug resistance gene whose protein product is easily assayed. Such reporter genes include acetohydroxyacid synthase (AHAS), alkaline phosphatase (AP), beta galactosidase (LacZ), beta glucoronidase (GUS), chloramphenicol acetyltransferase (CAT), green fluorescent protein (GFP), horseradish peroxidase (HRP), luciferase (Luc), nopaline synthase (NOS), octopine synthase (OCS), and derivatives thereof. Multiple selectable markers are available that confer resistance to ampicillin, bleomycin, chloramphenicol, gentarnycin, hygromycin, kanamycin, lincomycin, methotrexate, phosphinothricin, puromycin, and tetracyclin. Depending on the assay, quantitation of the amount of gene expression allows one to determine a degree of inhibition which is greater than 10%, 33%, 50%, 90%, 95% or 99% as compared to a cell not treated according to the present invention. Lower doses of injected material and longer times after administration of RNAi agent may result in inhibition in a smaller fraction of cells (e.g., at least 10%, 20%, 50%, 75%, 90%, or 95% of targeted cells). Quantitation of gene expression in a cell may show similar amounts of inhibition at the level of accumulation of target mRNA or translation of target protein. As an example, the efficiency of inhibition may be determined by assessing the amount of gene product in the cell; mRNA may be detected with a hybridization probe having a nucleotide sequence outside the region used for the inhibitory double-stranded RNA, or translated polypeptide may be detected with an antibody raised against the polypeptide sequence of that region.</p>
    <p num="p-0111">The RNA may be introduced in an amount which allows delivery of at least one copy per cell. Higher doses (e.g., at least 5, 10, 100, 500 or 1000 copies per cell) of material may yield more effective inhibition; lower doses may also be useful for specific applications.</p>
    <p num="p-0112">IV. Methods of Treatment:</p>
    <p num="p-0113">The present invention provides for both prophylactic and therapeutic methods of treating a subject at risk of (or susceptible to) a disorder or having a disorder associated with aberrant or unwanted target gene expression or activity. “Treatment”, or “treating” as used herein, is defined as the application or administration of a therapeutic agent (e.g., a RNA agent or vector or transgene encoding same) to a patient, or application or administration of a therapeutic agent to an isolated tissue or cell line from a patient, who has a disease or disorder, a symptom of disease or disorder or a predisposition toward a disease or disorder, with the purpose to cure, heal, alleviate, relieve, alter, remedy, ameliorate, improve or affect the disease or disorder, the symptoms of the disease or disorder, or the predisposition toward disease.</p>
    <p num="p-0114">With regards to both prophylactic and therapeutic methods of treatment, such treatments may be specifically tailored or modified, based on knowledge obtained from the field of pharmacogenomics. “Pharmacogenomics”, as used herein, refers to the application of genomics technologies such as gene sequencing, statistical genetics, and gene expression analysis to drugs in clinical development and on the market. More specifically, the term refers the study of how a patient's genes determine his or her response to a drug (e.g., a patient's “drug response phenotype”, or “drug response genotype”). Thus, another aspect of the invention provides methods for tailoring an individual's prophylactic or therapeutic treatment with either the target gene molecules of the present invention or target gene modulators according to that individual's drug response genotype. Pharmacogenomics allows a clinician or physician to target prophylactic or therapeutic treatments to patients who will most benefit from the treatment and to avoid treatment of patients who will experience toxic drug-related side effects.</p>
    <p num="p-0115">1. Prophylactic Methods</p>
    <p num="p-0116">In one aspect, the invention provides a method for preventing in a subject, a disease or condition associated with an aberrant or unwanted target gene expression or activity, by administering to the subject a therapeutic agent (e.g., an RNAi agent or vector or transgene encoding same). Subjects at risk for a disease which is caused or contributed to by aberrant or unwanted target gene expression or activity can be identified by, for example, any or a combination of diagnostic or prognostic assays as described herein. Administration of a prophylactic agent can occur prior to the manifestation of symptoms characteristic of the target gene aberrancy, such that a disease or disorder is prevented or, alternatively, delayed in its progression. Depending on the type of target gene aberrancy, for example, a target gene, target gene agonist or target gene antagonist agent can be used for treating the subject. The appropriate agent can be determined based on screening assays described herein.</p>
    <p num="p-0117">2. Therapeutic Methods</p>
    <p num="p-0118">Another aspect of the invention pertains to methods of modulating target gene expression, protein expression or activity for therapeutic purposes. Accordingly, in an exemplary embodiment, the modulatory method of the invention involves contacting a cell capable of expressing target gene with a therapeutic agent (e.g., an RNAi agent or vector or transgene encoding same) that is specific for the target gene or protein (e.g., is specific for the mRNA encoded by said gene or specifying the amino acid sequence of said protein) such that expression or one or more of the activities of target protein is modulated. These modulatory methods can be performed in vitro (e.g., by culturing the cell with the agent) or, alternatively, in vivo (e.g., by administering the agent to a subject). As such, the present invention provides methods of treating an individual afflicted with a disease or disorder characterized by aberrant or unwanted expression or activity of a target gene polypeptide or nucleic acid molecule. Inhibition of target gene activity is desirable in situations in which target gene is abnormally unregulated and/or in which decreased target gene activity is likely to have a beneficial effect.</p>
    <p num="p-0119">3. Pharmacogenomics</p>
    <p num="p-0120">The therapeutic agents (e.g., an RNAi agent or vector or transgene encoding same) of the invention can be administered to individuals to treat (prophylactically or therapeutically) disorders associated with aberrant or unwanted target gene activity. In conjunction with such treatment, pharmacogenomics (i.e., the study of the relationship between an individual's genotype and that individual's response to a foreign compound or drug) may be considered. Differences in metabolism of therapeutics can lead to severe toxicity or therapeutic failure by altering the relation between dose and blood concentration of the pharmacologically active drug. Thus, a physician or clinician may consider applying knowledge obtained in relevant pharmacogenomics studies in determining whether to administer a therapeutic agent as well as tailoring the dosage and/or therapeutic regimen of treatment with a therapeutic agent.</p>
    <p num="p-0121">Pharmacogenomics deals with clinically significant hereditary variations in the response to drugs due to altered drug disposition and abnormal action in affected persons. See, for example, Eichelbaum, M. et al. (1996) <i>Clin. Exp. Pharmacol. Physiol. </i>23(10-11): 983-985 and Linder, M. W. et al. (1997) <i>Clin. Chem. </i>43(2):254-266. In general, two types of pharmacogenetic conditions can be differentiated. Genetic conditions transmitted as a single factor altering the way drugs act on the body (altered drug action) or genetic conditions transmitted as single factors altering the way the body acts on drugs (altered drug metabolism). These pharmacogenetic conditions can occur either as rare genetic defects or as naturally-occurring polymorphisms. For example, glucose-6-phosphate dehydrogenase deficiency (G6PD) is a common inherited enzymopathy in which the main clinical complication is haemolysis after ingestion of oxidant drugs (anti-malarials, sulfonamides, analgesics, nitrofurans) and consumption of fava beans.</p>
    <p num="p-0122">One pharmacogenomics approach to identifying genes that predict drug response, known as “a genome-wide association”, relies primarily on a high-resolution map of the human genome consisting of already known gene-related markers (e.g., a “bi-allelic” gene marker map which consists of 60,000-100,000 polymorphic or variable sites on the human genome, each of which has two variants.). Such a high-resolution genetic map can be compared to a map of the genome of each of a statistically significant number of patients taking part in a Phase II/III drug trial to identify markers associated with a particular observed drug response or side effect. Alternatively, such a high resolution map can be generated from a combination of some ten-million known single nucleotide polymorphisms (SNPs) in the human genome. As used herein, a “SNP” is a common alteration that occurs in a single nucleotide base in a stretch of DNA. For example, a SNP may occur once per every 1000 bases of DNA. A SNP may be involved in a disease process, however, the vast majority may not be disease-associated. Given a genetic map based on the occurrence of such SNPs, individuals can be grouped into genetic categories depending on a particular pattern of SNPs in their individual genome. In such a manner, treatment regimens can be tailored to groups of genetically similar individuals, taking into account traits that may be common among such genetically similar individuals.</p>
    <p num="p-0123">Alternatively, a method termed the “candidate gene approach” can be utilized to identify genes that predict drug response. According to this method, if a gene that encodes a drugs target is known (e.g., a target gene polypeptide of the present invention), all common variants of that gene can be fairly easily identified in the population and it can be determined if having one version of the gene versus another is associated with a particular drug response.</p>
    <p num="p-0124">As an illustrative embodiment, the activity of drug metabolizing enzymes is a major determinant of both the intensity and duration of drug action. The discovery of genetic polymorphisms of drug metabolizing enzymes (e.g., N-acetyltransferase 2 (NAT 2) and cytochrome P450 enzymes CYP2D6 and CYP2C19) has provided an explanation as to why some patients do not obtain the expected drug effects or show exaggerated drug response and serious toxicity after taking the standard and safe dose of a drug. These polymorphisms are expressed in two phenotypes in the population, the extensive metabolizer (EM) and poor metabolizer (PM). The prevalence of PM is different among different populations. For example, the gene coding for CYP2D6 is highly polymorphic and several mutations have been identified in PM, which all lead to the absence of functional CYP2D6. Poor metabolizers of CYP2D6 and CYP2C19 quite frequently experience exaggerated drug response and side effects when they receive standard doses. If a metabolite is the active therapeutic moiety, PM show no therapeutic response, as demonstrated for the analgesic effect of codeine mediated by its CYP2D6 formed metabolite morphine. The other extreme are the so called ultra-rapid metabolizers who do not respond to standard doses. Recently, the molecular basis of ultra-rapid metabolism has been identified to be due to CYP2D6 gene amplification.</p>
    <p num="p-0125">Alternatively, a method termed the “gene expression profiling” can be utilized to identify genes that predict drug response. For example, the gene expression of an animal dosed with a therapeutic agent of the present invention can give an indication whether gene pathways related to toxicity have been turned on.</p>
    <p num="p-0126">Information generated from more than one of the above pharmacogenomics approaches can be used to determine appropriate dosage and treatment regimens for prophylactic or therapeutic treatment an individual. This knowledge, when applied to dosing or drug selection, can avoid adverse reactions or therapeutic failure and thus enhance therapeutic or prophylactic efficiency when treating a subject with a therapeutic agent, as described herein.</p>
    <p num="p-0127">Therapeutic agents can be tested in an appropriate animal model. For example, an RNAi agent (or expression vector or transgene encoding same) as described herein can be used in an animal model to determine the efficacy, toxicity, or side effects of treatment with said agent. Alternatively, a therapeutic agent can be used in an animal model to determine the mechanism of action of such an agent. For example, an agent can be used in an animal model to determine the efficacy, toxicity, or side effects of treatment with such an agent. Alternatively, an agent can be used in an animal model to determine the mechanism of action of such an agent.</p>
    <p num="p-0128">V. Pharmaceutical Compositions</p>
    <p num="p-0129">The invention pertains to uses of the above-described agents for therapeutic treatments as described infra. Accordingly, the modulators of the present invention can be incorporated into pharmaceutical compositions suitable for administration. Such compositions typically comprise the nucleic acid molecule, protein, antibody, or modulatory compound and a pharmaceutically acceptable carrier. As used herein the language “pharmaceutically acceptable carrier” is intended to include any and all solvents, dispersion media, coatings, antibacterial and antifungal agents, isotonic and absorption delaying agents, and the like, compatible with pharmaceutical administration. The use of such media and agents for pharmaceutically active substances is well known in the art. Except insofar as any conventional media or agent is incompatible with the active compound, use thereof in the compositions is contemplated. Supplementary active compounds can also be incorporated into the compositions.</p>
    <p num="p-0130">A pharmaceutical composition of the invention is formulated to be compatible with its intended route of administration. Examples of routes of administration include parenteral, e.g., intravenous, intradermal, subcutaneous, intraperitoneal, intramuscular, oral (e.g., inhalation), transdermal (topical), and transmucosal administration. Solutions or suspensions used for parenteral, intradermal, or subcutaneous application can include the following components: a sterile diluent such as water for injection, saline solution, fixed oils, polyethylene glycols, glycerine, propylene glycol or other synthetic solvents; antibacterial agents such as benzyl alcohol or methyl parabens; antioxidants such as ascorbic acid or sodium bisulfite; chelating agents such as ethylenediaminetetraacetic acid; buffers such as acetates, citrates or phosphates and agents for the adjustment of tonicity such as sodium chloride or dextrose. pH can be adjusted with acids or bases, such as hydrochloric acid or sodium hydroxide. The parenteral preparation can be enclosed in ampoules, disposable syringes or multiple dose vials made of glass or plastic.</p>
    <p num="p-0131">Pharmaceutical compositions suitable for injectable use include sterile aqueous solutions (where water soluble) or dispersions and sterile powders for the extemporaneous preparation of sterile injectable solutions or dispersion. For intravenous administration, suitable carriers include physiological saline, bacteriostatic water, Cremophor EL™ (BASF, Parsippany, N.J.) or phosphate buffered saline (PBS). In all cases, the composition must be sterile and should be fluid to the extent that easy syringability exists. It must be stable under the conditions of manufacture and storage and must be preserved against the contaminating action of microorganisms such as bacteria and fungi. The carrier can be a solvent or dispersion medium containing, for example, water, ethanol, polyol (for example, glycerol, propylene glycol, and liquid polyetheylene glycol, and the like), and suitable mixtures thereof. The proper fluidity can be maintained, for example, by the use of a coating such as lecithin, by the maintenance of the required particle size in the case of dispersion and by the use of surfactants. Prevention of the action of microorganisms can be achieved by various antibacterial and antifungal agents, for example, parabens, chlorobutanol, phenol, ascorbic acid, thimerosal, and the like. In many cases, it will be preferable to include isotonic agents, for example, sugars, polyalcohols such as manitol, sorbitol, sodium chloride in the composition. Prolonged absorption of the injectable compositions can be brought about by including in the composition an agent which delays absorption, for example, aluminum monostearate and gelatin.</p>
    <p num="p-0132">Sterile injectable solutions can be prepared by incorporating the active compound in the required amount in an appropriate solvent with one or a combination of ingredients enumerated above, as required, followed by filtered sterilization. Generally, dispersions are prepared by incorporating the active compound into a sterile vehicle which contains a basic dispersion medium and the required other ingredients from those enumerated above. In the case of sterile powders for the preparation of sterile injectable solutions, the preferred methods of preparation are vacuum drying and freeze-drying which yields a powder of the active ingredient plus any additional desired ingredient from a previously sterile-filtered solution thereof.</p>
    <p num="p-0133">Oral compositions generally include an inert diluent or an edible carrier. They can be enclosed in gelatin capsules or compressed into tablets. For the purpose of oral therapeutic administration, the active compound can be incorporated with excipients and used in the form of tablets, troches, or capsules. Oral compositions can also be prepared using a fluid carrier for use as a mouthwash, wherein the compound in the fluid carrier is applied orally and swished and expectorated or swallowed. Pharmaceutically compatible binding agents, and/or adjuvant materials can be included as part of the composition. The tablets, pills, capsules, troches and the like can contain any of the following ingredients, or compounds of a similar nature: a binder such as microcrystalline cellulose, gum tragacanth or gelatin; an excipient such as starch or lactose, a disintegrating agent such as alginic acid, Primogel, or corn starch; a lubricant such as magnesium stearate or Sterotes; a glidant such as colloidal silicon dioxide; a sweetening agent such as sucrose or saccharin; or a flavoring agent such as peppermint, methyl salicylate, or orange flavoring.</p>
    <p num="p-0134">For administration by inhalation, the compounds are delivered in the form of an aerosol spray from pressured container or dispenser which contains a suitable propellant, e.g., a gas such as carbon dioxide, or a nebulizer.</p>
    <p num="p-0135">Systemic administration can also be by transmucosal or transdermal means. For transmucosal or transdermal administration, penetrants appropriate to the barrier to be permeated are used in the formulation. Such penetrants are generally known in the art, and include, for example, for transmucosal administration, detergents, bile salts, and fusidic acid derivatives. Transmucosal administration can be accomplished through the use of nasal sprays or suppositories. For transdermal administration, the active compounds are formulated into ointments, salves, gels, or creams as generally known in the art.</p>
    <p num="p-0136">The compounds can also be prepared in the form of suppositories (e.g., with conventional suppository bases such as cocoa butter and other glycerides) or retention enemas for rectal delivery.</p>
    <p num="p-0137">In one embodiment, the active compounds are prepared with carriers that will protect the compound against rapid elimination from the body, such as a controlled release formulation, including implants and microencapsulated delivery systems. Biodegradable, biocompatible polymers can be used, such as ethylene vinyl acetate, polyanhydrides, polyglycolic acid, collagen, polyorthoesters, and polylactic acid. Methods for preparation of such formulations will be apparent to those skilled in the art. The materials can also be obtained commercially from Alza Corporation and Nova Pharmaceuticals, Inc. Liposomal suspensions (including liposomes targeted to infected cells with monoclonal antibodies to viral antigens) can also be used as pharmaceutically acceptable carriers. These can be prepared according to methods known to those skilled in the art, for example, as described in U.S. Pat. No. 4,522,811.</p>
    <p num="p-0138">It is especially advantageous to formulate oral or parenteral compositions in dosage unit form for ease of administration and uniformity of dosage. Dosage unit form as used herein refers to physically discrete units suited as unitary dosages for the subject to be treated; each unit containing a predetermined quantity of active compound calculated to produce the desired therapeutic effect in association with the required pharmaceutical carrier. The specification for the dosage unit forms of the invention are dictated by and directly dependent on the unique characteristics of the active compound and the particular therapeutic effect to be achieved, and the limitations inherent in the art of compounding such an active compound for the treatment of individuals.</p>
    <p num="p-0139">Toxicity and therapeutic efficacy of such compounds can be determined by standard pharmaceutical procedures in cell cultures or experimental animals, e.g., for determining the LD50 (the dose lethal to 50% of the population) and the ED50 (the dose therapeutically effective in 50% of the population). The dose ratio between toxic and therapeutic effects is the therapeutic index and it can be expressed as the ratio LD50/ED50. Compounds that exhibit large therapeutic indices are preferred. Although compounds that exhibit toxic side effects may be used, care should be taken to design a delivery system that targets such compounds to the site of affected tissue in order to minimize potential damage to uninfected cells and, thereby, reduce side effects.</p>
    <p num="p-0140">The data obtained from the cell culture assays and animal studies can be used in formulating a range of dosage for use in humans. The dosage of such compounds lies preferably within a range of circulating concentrations that include the ED50 with little or no toxicity. The dosage may vary within this range depending upon the dosage form employed and the route of administration utilized. For any compound used in the method of the invention, the therapeutically effective dose can be estimated initially from cell culture assays. A dose may be formulated in animal models to achieve a circulating plasma concentration range that includes the EC50 (i.e., the concentration of the test compound which achieves a half-maximal response) as determined in cell culture. Such information can be used to more accurately determine useful doses in humans. Levels in plasma may be measured, for example, by high performance liquid chromatography.</p>
    <p num="p-0141">The pharmaceutical compositions can be included in a container, pack, or dispenser together with instructions for administration.</p>
    <p num="p-0142">VI. Transgenic Organisms</p>
    <p num="p-0143">Engineered RNA precursors of the invention can be expressed in transgenic animals. These animals represent a model system for the study of disorders that are caused by, or exacerbated by, overexpression or underexpression (as compared to wildtype or normal) of nucleic acids (and their encoded polypeptides) targeted for destruction by the RNAi agents, e.g., siRNAs and shRNAs, and for the development of therapeutic agents that modulate the expression or activity of nucleic acids or polypeptides targeted for destruction.</p>
    <p num="p-0144">Transgenic animals can be farm animals (pigs, goats, sheep, cows, horses, rabbits, and the like), rodents (such as rats, guinea pigs, and mice), non-human primates (for example, baboons, monkeys, and chimpanzees), and domestic animals (for example, dogs and cats). Invertebrates such as <i>Caenorhabditis elegans </i>or <i>Drosophila </i>can be used as well as non-mammalian vertebrates such as fish (e.g., zebrafish) or birds (e.g., chickens).</p>
    <p num="p-0145">Engineered RNA precursors with stems of 18 to 30 nucleotides in length are preferred for use in mammals, such as mice. A transgenic founder animal can be identified based upon the presence of a transgene that encodes the new RNA precursors in its genome, and/or expression of the transgene in tissues or cells of the animals, for example, using PCR or Northern analysis. Expression is confirmed by a decrease in the expression (RNA or protein) of the target sequence.</p>
    <p num="p-0146">A transgenic founder animal can be used to breed additional animals carrying the transgene. Moreover, transgenic animals carrying a transgene encoding the RNA precursors can further be bred to other transgenic animals carrying other transgenes. In addition, cells obtained from the transgenic founder animal or its offspring can be cultured to establish primary, secondary, or immortal cell lines containing the transgene.</p>
    <p num="h-0008">1. Procedures for Making Transgenic, Non-Human Animals</p>
    <p num="p-0147">A number of methods have been used to obtain transgenic, non-human animals, which are animals that have gained an additional gene by the introduction of a transgene into their cells (e.g., both the somatic and genn cells), or into an ancestor's genn line. In some cases, transgenic animals can be generated by commercial facilities (e.g., The Transgenic <i>Drosophila </i>Facility at Michigan State University, The Transgenic Zebrafish Core Facility at the Medical College of Georgia (Augusta, Ga.), and Xenogen Biosciences (St. Louis, Mo.). In general, the construct containing the transgene is supplied to the facility for generating a transgenic animal.</p>
    <p num="p-0148">Methods for generating transgenic animals include introducing the transgene into the germ line of the animal. One method is by microinjection of a gene construct into the pronucleus of an early stage embryo (e.g., before the four-cell stage; Wagner et al., 1981, Proc. Natl. Acad. Sci. USA 78:5016; Brinster et al., 1985, Proc. Natl. Acad. Sci. USA 82:4438). Alternatively, the transgene can be introduced into the pronucleus by retroviral infection. A detailed procedure for producing such transgenic mice has been described (see e.g., Hogan et al., MP1 ulating the Mouse ErnbnLo. Cold Spring Harbour Laboratory, Cold Spring Harbour, N.Y. (1986); U.S. Pat. No. 5,175,383 (1992)). This procedure has also been adapted for other animal species (e.g., Hammer et al., 1985, Nature 315:680; Murray et al., 1989, Reprod. Fert. Devl. 1:147; Pursel et al., 1987, Vet. hnmunol. Histopath. 17:303; Rexroad et al., 1990, J. Reprod. Fert. 41 (suppl): 1 19; Rexroad et al., 1989, Molec. Reprod. Devl. 1:164; Simons et al., 1988, BioTechnology 6:179; Vize et al., 1988, J. Cell. Sci. 90:295; and Wagner, 1989, J. Cell. Biochem. 13B (suppl): 164).</p>
    <p num="p-0149">In brief, the procedure involves introducing the transgene into an animal by microinjecting the construct into the pronuclei of the fertilized mammalian egg(s) to cause one or more copies of the transgene to be retained in the cells of the developing mammal(s). Following introduction of the transgene construct into the fertilized egg, the egg may be incubated in vitro for varying amounts of time, or reimplanted a in surrogate host, or both. One common method is to incubate the embryos in vitro for about 1-7 days, depending on the species, and then reimplant them into the surrogate host. The presence of the transgene in the progeny of the transgenically manipulated embryos can be tested by Southern blot analysis of a segment of tissue.</p>
    <p num="p-0150">Another method for producing germ-line transgenic animals is through the use of embryonic stem (ES) cells. The gene construct can be introduced into embryonic stem cells by homologous recombination (Thomas et al., 1987, Cell 51:503; Capecchi, Science 1989, 244:1288; Joyner et al., 1989, Nature 338:153) in a transcriptionally active region of the genome. A suitable construct can also be introduced into embryonic stem cells by DNA-mediated transfection, such as by 17 electroporation (Ausubel et al., Current Protocols in Molecular Biology, John Wiley &amp; Sons, 1987). Detailed procedures for culturing embryonic stem cells (e.g., ES-D3@ ATCC# CCL-1934, ES-E14TG2a, ATCC# CCL-1821, American Type Culture Collection, Rockville, AM) and methods of making transgenic animals from embryonic stem cells can be found in Teratocarcinomas and Embi3Lonic Stem Cells, A Practical Approach, ed. E. J. Robertson (IRL Press, 1987). In brief, the ES cells are obtained from pre-implantation embryos cultured in vitro (Evans et al., 1981, Nature 292:154-156). Transgenes can be efficiently introduced into ES cells by DNA transfection or by retrovirus-mediated transduction. The resulting transformed ES cells can thereafter be combined with blastocysts from a non-human animal. The ES cells colonize the embryo and contribute to the genn line of the resulting chimeric animal.</p>
    <p num="p-0151">In the above methods, the transgene can be introduced as a linear construct, a circular plasmid, or a viral vector, which can be incorporated and inherited as a transgene integrated into the host genome. The transgene can also be constructed to permit it to be inherited as an extrachromosomal plasmid (Gassmann et al., 1995, Proc. Natl. Acad. Sci. USA 92:1292). A plasmid is a DNA molecule that can replicate autonomously in a host.</p>
    <p num="p-0152">The transgenic, non-human animals can also be obtained by infecting or transfecting cells either in vivo (e.g., direct injection), ex vivo (e.g., infecting the cells outside the host and later reimplanting), or in vitro (e.g., infecting the cells outside host), for example, with a recombinant viral vector carrying a gene encoding the engineered RNA precursors. Examples of suitable viral vectors include recombinant retroviral vectors (Valerio et al., 1989, Gene 84:419; Scharfinan et al., 1991, Proc. Natl. Acad. Sci. USA 88:462; Miller and Buttimore, 1986, Mol. Cell. Biol. 6:2895), recombinant adenoviral vectors (Freidman et al., 1986, Mol. Cell. Biol. 6:3791; Levrero et al., 1991, Gene 101: 195), and recombinant Herpes simplex viral vectors (Fink et al., 1992, Human Gene Therapy 3:11). Such methods are also useful for introducing constructs into cells for uses other than generation of transgenic animals.</p>
    <p num="p-0153">Other approaches include insertion of transgenes encoding the new engineered RNA precursors into viral vectors including recombinant adenovirus, adenoassociated virus, and herpes simplex virus-1, or recombinant bacterial or eukaryotic plasmids. Viral vectors transfect cells directly. Other approaches include delivering the transgenes, in the form of plasmid DNA, with the help of, for example, cationic liposomes (lipofectin) or derivatized (e.g., antibody conjugated) polylysine conjugates, gramacidin S, artificial viral envelopes, or other such intracellular carriers, as well as direct injection of the transgene construct or CaPO<sub>4 </sub>precipitation carried out in vivo. Such methods can also be used in vitro to introduce constructs into cells for uses other than generation of transgenic animals.</p>
    <p num="p-0154">Retrovirus vectors and adeno-associated virus vectors can be used as a recombinant gone delivery system for the transfer of exogenous genes in vivo or in vitro. These vectors provide efficient delivery of genes into cells, and the transferred nucleic acids are stably integrated into the chromosomal DNA of the host. The development of specialized cell lines (termed “packaging cells”) which produce only replication-defective retroviruses has increased the utility of retroviruses for gene therapy, and defective retroviruses are characterized for use in gene transfer for gene therapy purposes (for a review see Miller, 1990, Blood 76:271). A replication defective retrovirus can be packaged into virions which can be used to infect a target cell through the use of a helper virus by standard techniques. Protocols for producing recombinant retroviruses and for infecting cells in vitro or in vivo with such viruses can be found in Current Protocols in Molecular Biology, Ausubel, F. M. et al., (eds.) Greene Publishing Associates, (1989), Sections 9 9.14 and other standard laboratory manuals.</p>
    <p num="p-0155">Examples of suitable retroviruses include pLJ, pZIP, pWE and pEM which are known to those skilled in the art. Examples of suitable packaging virus lines for preparing both ecotropic and amphotropic retroviral systems include Psi-Crip, PsiCre, Psi-2 and Psi-Am. Retroviruses have been used to introduce a variety of genes into many different cell types, including epithelial cells, in vitro and/or in vivo (see for example Eglitis, et al., 1985, Science 230:1395-1398; Danos and Mulligan, 1988, Proc. Natl. Acad. Sci. USA 85:6460-6464; Wilson et al., 1988, Proc. Natl. Acad. Sci. USA 85:3014-3018; Armentano et al., 1990, Proc. Natl. Acad. Sci. USA 87:61416145; Huber et al., 1991, Proc. Natl. Acad. Sci. USA 88:8039-8043; Ferry et al., 1991, Proc. Natl. Acad. Sci. USA 88:8377-8381; Chowdhury et al., 1991, Science 254:1802-1805; van Beusechem. et al., 1992, Proc. Nad. Acad. Sci. USA 89:7640-19; Kay et al., 1992, Human Gene Therapy 3:641-647; Dai et al., 1992, Proc. Natl. Acad. Sci. USA 89:10892-10895; Hwu et al., 1993, J. Immunol. 150:4104-4115; U.S. Pat. No. 4,868,116; U.S. Pat. No. 4,980,286; PCT Application WO 89/07136; PCT Application WO 89/02468; PCT Application WO 89/05345; and PCT Application WO 92/07573).</p>
    <p num="p-0156">In another example, recombinant retroviral vectors capable of transducing and expressing genes inserted into the genome of a cell can be produced by transfecting the recombinant retroviral genome into suitable packaging cell lines such as PA317 and Psi-CRIP (Comette et al., 1991, Human Gene Therapy 2:5-10; Cone et al., 1984, Proc. Natl. Acad. Sci. USA 81:6349). Recombinant adenoviral vectors can be used to infect a wide variety of cells and tissues in susceptible hosts (e.g., rat, hamster, dog, and chimpanzee) (Hsu et al., 1992, J. Infectious Disease, 166:769), and also have the advantage of not requiring mitotically active cells for infection. Another viral gene delivery system useful in the present invention also utilizes adenovirus-derived vectors. The genome of an adenovirus can be manipulated such that it encodes and expresses a gene product of interest but is inactivated in terms of its ability to replicate in a normal lytic viral life cycle. See, for example, Berkner et al. (1988, BioTechniques 6:616), Rosenfeld et al. (1991, Science 252:431-434), and Rosenfeld et al. (1992, Cell 68:143-155). Suitable adenoviral vectors derived from the adenovirus strain Ad type 5 d1324 or other strains of adenovirus (e.g., Ad2, AO, Ad7 etc.) are known to those skilled in the art. Recombinant adenoviruses can be advantageous in certain circumstances in that they are not capable of infecting nondividing cells and can be used to infect a wide variety of cell types, including epithelial cells (Rosenfeld et al., 1992, cited supra). Furthermore, the virus particle is relatively stable and amenable to purification and concentration, and as above, can be modified to affect the spectrum of infectivity. Additionally, introduced adenoviral DNA (and foreign DNA contained therein) is not integrated into the genome of a host cell but remains episomal, thereby avoiding potential problems that can occur as a result of insertional mutagenesis hz situ where introduced DNA becomes integrated into the host genome (e.g., retroviral DNA). Moreover, the carrying capacity of the adenoviral genome for foreign DNA is large (up to 8 kilobases) relative to other gene delivery vectors (Berkner et al. cited supra; Haj-Ahmand and Graham, 1986, J. Virol. 57:267).</p>
    <p num="p-0157">Yet another viral vector system useful for delivery of the subject transgenes is the adeno-associated virus (AAV). Adeno-associated virus is a naturally occurring defective virus that requires another virus, such as an adenovirus or a herpes virus, as a helper virus for efficient replication and a productive life cycle. For a review, see Muzyczka et al. (1992, Curr. Topics in Micro. and Immunol. 158:97-129). It is also one of the few viruses that may integrate its DNA into non-dividing cells, and exhibits a high frequency of stable integration (see for example Flotte et al. (1992, Am. J. Respir. Cell. Mol. Biol. 7:349-356; Samulski et al., 1989, J. Virol. 63:3822-3828; and McLaughlin et al. (1989, J. Virol. 62:1963-1973). Vectors containing as little as 300 base pairs of AAV can be packaged and can integrate. Space for exogenous DNA is limited to about 4.5 kb. An AAV vector such as that described in Tratschin et al. (1985) Mol. Cell. Biol. 5:3251-3260 can be used to introduce DNA into cells. A variety of nucleic acids have been introduced into different cell types using AAV vectors (see for example Hennonat et al. (1984) Proc. Nad. Acad. Sci. USA 8 1:64666470; Tratschin et al. (1985) Mol. Cell. Biol. 4:2072-2081; Wondisford et al. (1988) Mol. Endocrinol. 2:32-39; Tratschin et al. (1984) J. Virol. 51:611-619; and Flotte et al. (1993) J. Biol. Chem. 268:3781-3790).</p>
    <p num="p-0158">In addition to viral transfer methods, such as those illustrated above, non-viral methods can also be employed to cause expression of an shRNA or engineered RNA precursor of the invention in the tissue of an animal. Most non-viral methods of gene transfer rely on nominal mechanisms used by mammalian cells for the uptake and intracellular transport of macromolecules. In preferred embodiments, non-viral gene delivery systems of the present invention rely on endocytic pathways for the uptake of the subject gene of the invention by the targeted cell. Exemplary gene delivery systems of this type include liposomal derived systems, poly-lysine conjugates, and artificial viral envelopes. Other embodiments include plasmid injection systems such as are described in Meuli et al., (2001) J. Invest. DerinatoL, 116(1):131-135; Cohen et al., (2000) Gene Ther., 7(22):1896-905; and Tam et al., (2000) Gene Ther., 7(21):186774.</p>
    <p num="p-0159">In a representative embodiment, a gene encoding an shRNA or engineered RNA precursor of the invention can be entrapped in liposomes bearing positive charges on their surface (e.g., lipofectins) and (optionally) which are tagged with antibodies against cell surface antigens of the target tissue (Mizuno et al., (1992) No Shinkei Geka, 20:547-55 1; PCT publication WO91/06309; Japanese patent application 10473 8 1; and European patent publication EP-A-43 075).</p>
    <p num="p-0160">Animals harboring the transgene can be identified by detecting the presence of the transgene in genomic DNA (e.g., using Southern analysis). In addition, expression of the shRNA or engineered RNA precursor can be detected directly (e.g., by Northern analysis). Expression of the transgene can also be confirmed by detecting a decrease in the amount of protein corresponding to the targeted sequence. When the transgene is under the control of an inducible or developmentally regulated promoter, expression of the target protein is decreased when the transgene is induced or at the developmental stage when the transgene is expressed, respectively.</p>
    <p num="h-0009">2. Clones of Transgenic Animals</p>
    <p num="p-0161">Clones of the non-human transgenic animals described herein can be produced according to the methods described in Wilmut et al. ((1997) Nature, 385:810-813) and PCT publication Nos. WO 97/07668 and WO 97/07669. In brief, a cell, e.g., a somatic cell from the transgenic animal, can be isolated and induced to exit the growth cycle and enter the GO phase to become quiescent. The quiescent cell can then be fused, e.g., through the use of electrical pulses, to an enucleated oocyte from an animal of the same species from which the quiescent cell is isolated. The reconstructed oocyte is then cultured such that it develops into a morula or blastocyte and is then transferred to a pseudopregnant female foster animal. Offspring borne of this female foster animal will be clones of the animal from which the cell, e.g., the somatic cell, was isolated.</p>
    <p num="p-0162">Once the transgenic animal is produced, cells of the transgenic animal and cells from a control animal are screened to determine the presence of an RNA precursor nucleic acid sequence, e.g., using polymerase chain reaction (PCR). Alternatively, the cells can be screened to determine if the RNA precursor is expressed (e.g., by standard procedures such as Northern blot analysis or reverse transcriptase-polymerase chain reaction (RT-PCR); Sambrook et al., Molecular Cloning—A Laboratory Manual, (Cold Spring Harbor Laboratory, 1989)).</p>
    <p num="p-0163">The transgenic animals of the present invention can be homozygous or heterozygous, and one of the benefits of the invention is that the target mRNA is effectively degraded even in heterozygotes. The present invention provides for transgenic animals that carry a transgene of the invention in all their cells, as well as animals that carry a transgene in some, but not all of their cells. That is, the invention provides for mosaic animals. The transgene can be integrated as a single transgene or in concatatners, e.g., head-to-head tandems or head-to-tail tandems.</p>
    <p num="p-0164">For a review of techniques that can be used to generate and assess transgenic animals, skilled artisans can consult Gordon (IwL Rev. CytoL 1 1 5:171-229, 1989), and may obtain additional guidance from, for example: Hogan et al. “Manipulating the Mouse Embryo” (Cold Spring Harbor Press, Cold Spring Harbor, N.Y., 1986; Krimpenfort et al., Bio/Technology 9:86, 1991; Palmiter et al., Cell 41:343, 1985; Kraemer et al., “Genetic Manipulation of the Early Mammalian Embryo,” Cold Spring Harbor Press, Cold Spring Harbor, N.Y., 1985; Hammer et al., Nature 315:680, 1985; Purcel et al., Science, 244:1281, 1986; Wagner et al., U.S. Pat. No. 5,175,385; and Krimpenfort et al., U.S. Pat. No. 5,175,384.</p>
    <p num="h-0010">3. Transgenic Plants</p>
    <p num="p-0165">Among the eukaryotic organisms featured in the invention are plants containing an exogenous nucleic acid that encodes an engineered RNA precursor of the invention.</p>
    <p num="p-0166">Accordingly, a method according to the invention comprises making a plant having a nucleic acid molecule or construct, e.g., a transgene, described herein. Techniques for introducing exogenous micleic acids into monocotyledonous and dicotyledonous plants are known in the art, and include, without limitation, <i>Agrobacterium</i>-mediated transformation, viral vector-mediated transformation, electroporation and particle gun transformation, see, e.g., U.S. Pat. Nos. 5,204,253 and 6,013,863. If a cell or tissue culture is used as the recipient tissue for transformation, plants can be regenerated from transformed cultures by techniques known to those skilled in the art. Transgenic plants can be entered into a breeding program, e.g., to introduce a nucleic acid encoding a polypeptide into other lines, to transfer the nucleic acid to other species or for further selection of other desirable traits. Alternatively, transgenic plants can be propagated vegetatively for those species amenable to such techniques. Progeny includes descendants of a particular plant or plant line. Progeny of a plant include seeds formed on F1, F2, F3, and subsequent generation plants, or seeds formed on BQ, BC2, BC3, and subsequent generation plants. Seeds produced by a transgenic plant can be grown and then selfed (or outcrossed and selfed) to obtain seeds homozygous for the nucleic acid encoding a novel polypeptide.</p>
    <p num="p-0167">A suitable group of plants with which to practice the invention include dicots, such as safflower, alfalfa, soybean, rapeseed (high erucic acid and canola), or sunflower. Also suitable are monocots such as corn, wheat, rye, barley, oat, rice, millet, amaranth or sorghum. Also suitable are vegetable crops or root crops such as potato, broccoli, peas, sweet corn, popcorn, tomato, beans (including kidney beans, lima beans, dry beans, green beans) and the like. Also suitable are fruit crops such as peach, pear, apple, cherry, orange, lemon, grapefruit, plum, mango and palm. Thus, the invention has use over a broad range of plants, including species from the genera <i>Anacardium, Arachis, Asparagus, Atropa, Avena, Brassica, Citrus, Citrullus, Capsicum, Carthamus, Cocos, Coffea, Cucumis, Cucurbita, Daucus, Elaeis, Fragaria, Glycine, Gossypium, Helianthus, Heterocallis, Hordeum, Hyoscyalnus, Lactuca, Linum, Lolium, Lupinus, Lycopersicon, Malus, Manihot, Majorana, Medicago, Nicotiana, Olea, Oryza, Panicum, Pannesetum, Persea, Phaseolus, Pistachia, Pisum, Pyrus, Prunus, Raphanus, Ricinus, Secale, Senecio, Sinapis, Solanum, Sorghum, Theobromus, Trigonella, Triticum, Vicia, Vitis, Vigna </i>and <i>Zea. </i> </p>
    <p num="p-0168">The skilled artisan will appreciate that the enumerated organisms are also useful for practicing other aspects of the invention, e.g., as host cells, as described supra.</p>
    <p num="p-0169">The nucleic acid molecules of the invention can be expressed in plants in a cell- or tissue-specific manner according to the regulatory elements chosen to include in a particular nucleic acid construct present in the plant. Suitable cells, tissues, and organs in which to express a chimeric polypeptide of the invention include, without limitation, egg cell, central cell, synergid cell, zygote, ovule primordia, nucellus, integuments, endothelium, female garnetophyte cells, embryo, axis, cotyledons, suspensor, endosperm, seed coat, ground meristem, vascular bundle, cambium, phloem, cortex, shoot or root apical meristems, lateral shoot or root meristems, floral meristem, leaf primordia, leaf mesophyll cells, and leaf epidermal cells, e.g., epidermal cells involved in fortning the cuticular layer. Also suitable are cells and tissues grown in liquid media or on semi-solid media.</p>
    <p num="h-0011">4. Transgenic Fungi</p>
    <p num="p-0170">Other eukaryotic organisms featured in the invention are fungi containing an exogenous nucleic acid molecule that encodes an engineered RNA precursor of the invention. Accordingly, a method according to the invention comprises introducing a nucleic acid molecule or construct as described herein into a fungus. Techniques for introducing exogenous nucleic acids into many fungi are known in the art, see, e.g., U.S. Pat. Nos. 5,252,726 and 5,070,020. Transformed fungi can be cultured by techniques known to those skilled in the art. Such fungi can be used to introduce a nucleic acid encoding a polypeptide into other fungal strains, to transfer the nucleic acid to other species or for further selection of other desirable traits.</p>
    <p num="p-0171">A suitable group of fungi with which to practice the invention include fission yeast and budding yeast, such as <i>Saccharomyces cereviseae, S. pombe</i>, S. carlsbergeris and <i>Candida albicans</i>. Filamentous fungi such as <i>Aspergillus </i>spp. and <i>Penicillium </i>spp. are also useful.</p>
    <p num="p-0172">VII. Knockout and/or Knockdown Cells or Organisms</p>
    <p num="p-0173">A further preferred use for the RNAi agents of the present invention (or vectors or transgenes encoding same) is a functional analysis to be carried out in eukaryotic cells, or eukaryotic non-human organisms, preferably mammalian cells or organisms and most preferably human cells, e.g., cell lines such as HeLa or 293 or rodents, e.g., rats and mice. By administering a suitable RNAi agent which is sufficiently complementary to a target mRNA sequence to direct target-specific RNA interference, a specific knockout or knockdown phenotype can be obtained in a target cell, e.g., in cell culture or in a target organism.</p>
    <p num="p-0174">Thus, a further subject matter of the invention is a eukaryotic cell or a eukaryotic non-human organism exhibiting a target gene-specific knockout or knockdown phenotype comprising a fully or at least partially deficient expression of at least one endogeneous target gene wherein said cell or organism is transfected with at least one vector comprising DNA encoding a RNAi agent capable of inhibiting the expression of the target gene. It should be noted that the present invention allows a target-specific knockout or knockdown of several different endogeneous genes due to the specificity of the RNAi agent.</p>
    <p num="p-0175">Gene-specific knockout or knockdown phenotypes of cells or non-human organisms, particularly of human cells or non-human mammals may be used in analytic to procedures, e.g. in the functional and/or phenotypical analysis of complex physiological processes such as analysis of gene expression profiles and/or proteomes. Preferably the analysis is carried out by high throughput methods using oligonucleotide based chips.</p>
    <p num="p-0176">Using RNAi based knockout or knockdown technologies, the expression of an endogeneous target gene may be inhibited in a target cell or a target organism. The endogeneous gene may be complemented by an exogenous target nucleic acid coding for the target protein or a variant or mutated form of the target protein, e.g., a gene or a DNA, which may optionally be fused to a further nucleic acid sequence encoding a detectable peptide or polypeptide, e.g. an affinity tag, particularly a multiple affinity tag.</p>
    <p num="p-0177">Variants or mutated forms of the target gene differ from the endogeneous target gene in that they encode a gene product which differs from the endogeneous gene product on the amino acid level by substitutions, insertions and/or deletions of single or multiple amino acids. The variants or mutated forms may have the same biological activity as the endogeneous target gene. On the other hand, the variant or mutated target gene may also have a biological activity, which differs from the biological activity of the endogeneous target gene, e.g. a partially deleted activity, a completely deleted activity, an enhanced activity etc. The complementation may be accomplished by compressing the polypeptide encoded by the endogeneous nucleic acid, e.g., a fusion protein comprising the target protein and the affinity tag and the double stranded RNA molecule for knocking out the endogeneous gene in the target cell. This compression may be accomplished by using a suitable expression vector expressing both the polypeptide encoded by the endogenous nucleic acid, e.g., the tag-modified target protein and the double stranded RNA molecule or alternatively by using a combination of expression vectors. Proteins and protein complexes which are synthesized de novo in the target cell will contain the exogenous gene product, e.g., the modified fusion protein. In order to avoid suppression of the exogenous gene product by the RNAi agent, the nucleotide sequence encoding the exogenous nucleic acid may be altered at the DNA level (with or without causing mutations on the amino acid level) in the part of the sequence which so is homologous to the RNAi agent. Alternatively, the endogeneous target gene may be complemented by corresponding nucleotide sequences from other species, e.g. from mouse.</p>
    <p num="p-0178">VIII. Functional Genomics and/or Proteomics</p>
    <p num="p-0179">Preferred applications for the cell or organism of the invention is the analysis of gene expression profiles and/or proteomes. In an especially preferred embodiment an analysis of a variant or mutant form of one or several target proteins is carried out, wherein said variant or mutant forms are reintroduced into the cell or organism by an exogenous target nucleic acid as described above. The combination of knockout of an endogeneous gene and rescue by using mutated, e.g. partially deleted exogenous target has advantages compared to the use of a knockout cell. Further, this method is particularly suitable for identifying functional domains of the targeted protein. In a further preferred embodiment a comparison, e.g. of gene expression profiles and/or proteomes and/or phenotypic characteristics of at least two cells or organisms is carried out. These organisms are selected from: (i) a control cell or control organism without target gene inhibition, (ii) a cell or organism with target gene inhibition and (iii) a cell or organism with target gene inhibition plus target gene complementation by an exogenous target nucleic acid.</p>
    <p num="p-0180">Furthermore, the RNA knockout complementation method may be used for is preparative purposes, e.g. for the affinity purification of proteins or protein complexes from eukaryotic cells, particularly mammalian cells and more particularly human cells. In this embodiment of the invention, the exogenous target nucleic acid preferably codes for a target protein which is fused to art affinity tag. This method is suitable for functional proteome analysis in mammalian cells, particularly human cells.</p>
    <p num="p-0181">Another utility of the present invention could be a method of identifying gene function in an organism comprising the use of an RNAi agent to inhibit the activity of a target gene of previously unknown function. Instead of the time consuming and laborious isolation of mutants by traditional genetic screening, functional genomics would envision determining the function of uncharacterized genes by employing the invention to reduce the amount and/or alter the timing of target gene activity. The invention could be used in determining potential targets for pharmaceutics, understanding normal and pathological events associated with development, determining signaling pathways responsible for postnatal development/aging, and the like. The increasing speed of acquiring nucleotide sequence information from genomic and expressed gene sources, including total sequences for the yeast, <i>D. melanogaster</i>, and <i>C. elegans </i>genomes, can be coupled with the invention to determine gene function in an organism (e.g., nematode). The preference of different organisms to use particular codons, searching sequence databases for related gene products, correlating the linkage map of genetic traits with the physical map from which the nucleotide sequences are derived, and artificial intelligence methods may be used to define putative open reading frames from the nucleotide sequences acquired in such sequencing projects. A simple assay would be to inhibit gene expression according to the partial sequence available from an expressed sequence tag (EST). Functional alterations in growth, development, metabolism, disease resistance, or other biological processes would be indicative of the normal role of the EST's gene product.</p>
    <p num="p-0182">The ease with which RNA can be introduced into an intact cell/organism containing the target gene allows the present invention to be used in high throughput screening (HTS). Solutions containing RNAi agents that are capable of inhibiting the different expressed genes can be placed into individual wells positioned on a microtiter plate as an ordered array, and intact cells/organisms in each well can be assayed for any changes or modifications in behavior or development due to inhibition of target gene activity. The amplified RNA can be fed directly to, injected into, the cell/organism containing the target gene. Alternatively, the RNAi agent can be produced from a vector, as described herein. Vectors can be injected into, the cell/organism containing the target gene. The function of the target gene can be assayed from the effects it has on the cell/organism when gene activity is inhibited. This screening could be amenable to small subjects that can be processed in large number, for example: <i>arabidopsis</i>, bacteria, <i>drosophila</i>, fungi, nematodes, viruses, zebrafish, and tissue culture cells derived from mammals. A nematode or other organism that produces a calorimetric, fluorogenic, or luminescent signal in response to a regulated promoter (e.g., transfected with a reporter gene construct) can be assayed in an HTS format.</p>
    <p num="p-0183">The present invention may be useful in allowing the inhibition of essential genes. Such genes may be required for cell or organism viability at only particular stages of development or cellular compartments. The functional equivalent of conditional mutations may be produced by inhibiting activity of the target gene when or where it is not required for viability. The invention allows addition of RNAi agents at specific times of development and locations in the organism without introducing permanent mutations into the target genome.</p>
    <p num="p-0184">IX. Screening Assays</p>
    <p num="p-0185">The methods of the invention are also suitable for use in methods to identify and/or characterize potential pharmacological agents, e.g. identifying new pharmacological agents from a collection of test substances and/or characterizing mechanisms of action and/or side effects of known pharmacological agents.</p>
    <p num="p-0186">Thus, the present invention also relates to a system for identifying and/or characterizing pharmacological agents acting on at least one target protein comprising: (a) a eukaryotic cell or a eukaryotic non-human organism capable of expressing at least one endogeneous target gene coding for said so target protein, (b) at least one RNA agent capable of inhibiting the expression of said at least one endogeneous target gene, and (c) a test substance or a collection of test substances wherein pharmacological properties of said test substance or said collection are to be identified and/or characterized. Further, the system as described above preferably comprises: (d) at least one exogenous target nucleic acid coding for the target protein or a variant or mutated form of the target protein wherein said exogenous target nucleic acid differs from the endogeneous target gene on the nucleic acid level such that the expression of the exogenous target nucleic acid is substantially less inhibited by the RNA agent than the expression of the endogeneous target gene.</p>
    <p num="p-0187">The test compounds of the present invention can be obtained using any of the numerous approaches in combinatorial library methods known in the art, including: biological libraries; spatially addressable parallel solid phase or solution phase libraries; synthetic library methods requiring deconvolution; the ‘one-bead one-compound’ library method; and synthetic library methods using affinity chromatography selection. The biological library approach is limited to peptide libraries, while the other four approaches are applicable to peptide, non-peptide oligomer or small molecule libraries of compounds (Lam, K. S. (1997) <i>Anticancer Drug Des. </i>12:145).</p>
    <p num="p-0188">Examples of methods for the synthesis of molecular libraries can be found in the art, for example in: DeWitt et al. (1993) <i>Proc. Natl. Acad. Sci. U.S.A. </i>90:6909; Erb et al. (1994) <i>Proc. Natl. Acad. Sci. USA </i>91:11422; Zuckermann et al. (1994). <i>J. Med. Chem. </i>37:2678; Cho et al. (1993) <i>Science </i>261:1303; Carrell et al. (1994) <i>Angew. Chem. Int. Ed. Engl. </i>33:2059; Carell et al. (1994) <i>Angew. Chem. Int. Ed. Engl. </i>33:2061; and in Gallop et al. (1994) <i>J. Med. Chem. </i>37:1233.</p>
    <p num="p-0189">Libraries of compounds may be presented in solution (e.g., Houghten (1992) <i>Biotechniques </i>13:412-421), or on beads (Lam (1991) <i>Nature </i>354:82-84), chips (Fodor (1993) Nature 364:555-556), bacteria (Ladner U.S. Pat. No. 5,223,409), spores (Ladner USP 409), plasmids (Cull et al. (1992) <i>Proc Natl Acad Sci USA </i>89:1865-1869) or on phage (Scott and Smith (1990) <i>Science </i>249:386-390); (Devlin (1990) <i>Science </i>249:404-406); (Cwirla et al. (1990) <i>Proc. Natl. Acad. Sci. </i>87:6378-6382); (Felici (1991) <i>J. Mol. Biol. </i>222:301-310); (Ladner supra.)).</p>
    <p num="p-0190">In a preferred embodiment, the library is a natural product library, e.g., a library produced by a bacterial, fungal, or yeast culture. In another preferred embodiment, the library is a synthetic compound library.</p>
    <p num="p-0191">This invention is further illustrated by the following examples which should not be construed as limiting. The contents of all references, patents and published patent applications cited throughout this application are incorporated herein by reference.</p>
    <heading>EXAMPLES</heading> <heading>Materials and Methods</heading> <p num="h-0014">Lysate Preparation</p>
    <p num="p-0192">Fly embryo lysates were prepared as previously described (Tuschl et al. 1999). Wheat germ extracts were prepared from frozen or vacuum-packed raw wheat germ (e.g., Fearn Nature Fresh Raw Wheat Germ, Bread and Circus) as described (Erickson and Blobel 1983). The extract was centrifuged at 14,500 g at 4° C. for 25 min; the supernatant was then frozen in aliquots in liquid nitrogen and stored at −80° C. For cauliflower extract, the outer layer of fresh cauliflower (Shaws Supermarket) was harvested with a razor blade and ground to a powder under liquid nitrogen in a mortar and pestle, then homogenized with 3 mL of 1× lysis buffer (100 mM potassium acetate, 30 mM HEPES-KOH at pH 7.4, 2 mM magnesium acetate) containing 5 mM dithiothreitol (DTT) and 1 mg/mL Pefabloc SC (Boehringer Mannheim) per gram of plant tissue. The extract was centrifuged, and the supernatant was stored as described for the <i>Drosophila </i>embryo lysate.</p>
    <p num="h-0015">Analysis of dsRNA Processing</p>
    <p num="p-0193">For analysis of dsRNA processing, 5 nM internally α-32P-UTP-labeled dsRNA was incubated in a 10 μL reaction containing 5 μL of <i>Drosophila </i>embryo lysate (Tuschl et al. 1999) or wheat germ extract, 100 μM GTP, 500 μM ATP, 10 mM creatine phosphate, 10 μg/mL creatine phosphokinase, 5 mM DTT, and 0.1 U/μL RNasin (Promega) at 25° C. for 3 h. Reactions were stopped by the addition of 2× proteinase K buffer [200 mM Tris-HCl at pH 7.5, 25 mM EDTA, 300 mM NaCl, 2% (w/v) sodium dodecyl sulfate] and deproteinized with ˜2 mg/mL proteinase K at 65° C. for 15 min. Products were precipitated with 3 volumes cold ethanol and analyzed by electrophoresis in a 15% polyacrylamide sequencing gel.</p>
    <p num="h-0016">Gel Filtration and RNAse Protection</p>
    <p num="p-0194">Internally α-32P-UTP-labeled dsRNAs were incubated in wheat germ extract, then deproteinized at room temperature with proteinase K (1 h) and RNA-precipitated with 3 volumes of cold ethanol. The RNA was resuspended in 1× lysis buffer and analyzed by gel filtration as described (Nykanen et al. 2001). For RNase protection, the RNA products of a 10 μL wheat germ extract reaction were deproteinized at room temperature and analyzed by RNAse protection essentially as described (Sambrook et al. 1989). Briefly, the siRNA pellets were dissolved in 10 μL of RNAse digestion buffer (300 mM NaCl, 10 mM Tris-HCl at pH 7.4, and 5 mM EDTA at pH 7.5) containing 10 mM [beta]-glycerophosphate, 5 mM ATP, O-6.6 U of RNAse A, and 0-1.1 U of RNAse Ti. For control experiments, 5′-32P-radiolabeled synthetic, double-stranded siRNAs were mixed with the products of a wheat germ reaction performed with unlabeled dsRNA and coprecipitated with 3 volumes of cold ethanol. RNAse protection was at 25° C. for 1 h, stopped by adding 0.6 μL of 10% SDS and 0.3 μL of 20 mg/mL proteinase K, then incubated at 25° C. for 1 h. The reactions were then adjusted to 200 μL with 2×PK buffer containing 0.2 mg/mL Glycogen (Roche), extracted with an equal volume of phenol/chloroform/isoamylalcohol (25:24:1; v/v/v), precipitated with 3 volumes of cold ethanol, and analyzed in a 15% sequencing polyacrylamide gel.</p>
    <p num="h-0017">Synthetic siRNAs Used as Inhibitors</p>
    <p num="p-0195">The 21-nt siRNA inhibitor comprised CGUACGCGGAAUAC UUCGA(5-Iodo-U)U (SEQ ID NO: 25) annealed with UCGAAGUAUUCCGCGUACGUG (SEQ ID NO: 26); the 25-mer comprised AUCACGUACGCGGAAUACUUCGA(5-Iodo-U)U (SEQ ID NO: 27) annealed with UCGAAGUAUUCCGCGUACGUGAUUG (SEQ ID NO: 28). The 5-Iodo-U nucleotides were included to facilitate studies not presented here, and we have no evidence they enhance the effectiveness of the siRNAs as inhibitors.</p>
    <p num="h-0018">Analysis of RdRP Activity</p>
    <p num="p-0196">Assays were performed in a final volume of 10 μL containing 5 μL of lysate, 100 μM GTP, 100 μM CTP, 500 μM ATP, 20 μM UTP, 5 μCi of α-32P-UTP (25 Ci/mmole), 10 mM creatine phosphate, 10 μg/mL creatine phosphokinase, 5 mM DTT, 0.2 U/μL Super-RNasin (Ambion), and 7-methyl-G- or A-capped RNAs. After incubation at 25° C. for 3 h, the reaction was deproteinized with proteinase K in 200 μL of 2× proteinase K buffer at 65° C. for 15 min. After phenol/chloroform/isoamylalcohol extraction, the aqueous phase was precipitated with 3 volumes of cold ethanol, resuspended in 10 μL of 2× formamide loading buffer as described (Sambrook et al. 1989), and resolved on 10% or 15% polyacrylamide sequencing gels. For primed assays, capped RNAs were preincubated with single-stranded 21-nt RNA primers or siRNA duplexes at room temperature for 10 min before the remaining reaction components were added.</p>
    <p num="h-0019"> <i>Arabidopsis </i>PHV, PHB, and Mutant PHV Target RNAs</p>
    <p num="p-0197"> <i>Arabidopsis </i>PHV and PHB cDNA sequences containing the miR165/166 complementary sequences were amplified from an <i>Arabidopsis </i>flower cDNA library (CD4-6) by polymerase chain reaction (PCR) using the following primer pairs: 5′-PHV primer, GCGTAATACGACTCACTATAGGCGCCGGAACAAGTTG AAG (SEQ ID NO:9), and 3′-PHV primer, GACAGTCACGGAACCAAGATG (SEQ ID NO:10); or 5′-PHB primer, GCGTAATACGACTCACTATAGGTGAGTCTGTGGTCGTGAGTG (SEQ ID NO:11), and 3′-PHB primer, GCTGCTGCTAAAGTCGTAGGA (SEQ ID NO:12). The Arabidopsis G [right-arrow]. A mutant phv template was initially amplified using the 5′-PHV primer and CCACTGCAGTTGCGTGAAACAGCTACGATACCAATAGAATCCGGATCAGGCTTCAT CCC (SEQ ID NO:13). This PCR product was diluted 100-fold, then reamplified with the 5′-PHV primer and GACAGTCACGGAACCAAGATGGACGATCTTTGAGGATTTCAGCGACCTTCATGGGT TCTAAACTCACGAGGCCACAGGCACGTGCTGCTATTCCACTGCAGTTGCGTGAAAC AGC (SEQ ID NO:14). In vitro RNA transcription and cap labeling were as described (Tuschl et al. 1999; Zamore et al. 2000).</p>
    <p num="h-0020">In vitro RNAi in Fly Embryo Lysate and Wheat Germ Extract</p>
    <p num="p-0198">For RNAi in <i>Drosophila </i>embryo lysate, four siRNA duplexes were chemically synthesized (Dharmacon), annealed, and incubated in a standard RNAi reaction (Zamore et al. 2000). The sequences of siRNAs (sense and antisense strands) corresponding to miR165, miR166, PHV, and mutant phv target positions were miR165, UCGGACCAGGCUUCAUCCCCC (SEQ ID NO:15) and GGGAUGAAGCCUGGUCCGAGG (SEQ ID NO:16); miR166, UCGGACCAGGCUUCAUUCCCC (SEQ ID NO:17) and GGAAUGAAGCCUGGUCCGAGA (SEQ ID NO:18); PHV, CCGGACCAGGCUUCAUCCCAA (SEQ ID NO:19) and GGGAUGAAGCCUGGUCCGGAU (SEQ ID NO:20); and mutant phv, CCGGAUCAGGCUUCAUCCCAA (SEQ ID NO:21) and GGGAUGAAGCCUGAUCCGGAU (SEQ ID NO:22). Wheat germ extract target cleavage reactions were as standard <i>Drosophila </i>in vitro RNAi reactions, except that no exogenous siRNAs were added.</p>
    <p num="h-0021">Total RNA Isolation and Northern Analysis</p>
    <p num="p-0199">Total RNA was isolated from lysates, and Northern analysis was performed as described (Hutvágner and Zamore 2002). 5′-32P-radioalabeled synthetic miR165 antisense siRNA (above) was used as probe.</p>
    <heading>Overview of Examples I-VI</heading> <p num="p-0200">The data presented in Examples I-VI demonstrate that extracts of wheat germ, introduced for the study of translation and protein translocation in the 1970s (Roberts and Paterson 1973), recapitulate many of the key features of RNA silencing in plants. Using this in vitro system, it is shown that in plants, ATP-dependent, Dicer-like enzymes cleave dsRNA into small RNAs that have the structure of siRNAs. Unlike <i>Drosophila </i>embryos or mammalian cells, plants convert dsRNA into two distinct classes of siRNAs, long and short siRNAs. Inhibitor studies indicate that a different Dicer-like enzyme generates each siRNA class.</p>
    <p num="p-0201">The data further demonstrate that a wheat RdRP activity can synthesize dsRNA using exogenous single-stranded RNA as a template without an exogenous primer, and that this dsRNA is preferentially converted into long siRNAs.</p>
    <p num="p-0202">Finally, it is demonstrated that wheat germ extracts contain an endogenous RISC programmed with a miRNA. This endogenous miRNA complex can direct efficient cleavage of the wild-type <i>Arabidopsis </i>PHAVOLUTA (PHV) mRNA sequence, but not that of a previously described dominant PHV mutant that perturbs leaf development. This finding supports the view that in plants miRNAs direct RNAi and explains the molecular basis for the dominant PHV mutation in <i>Arabidopsis</i>. Interestingly, exact complementarity between the miRNA and target mRNA is not necessary for the miRNA to direct efficient target cleavage. In fact, it is demonstrated that the efficiency of cleavage is greater when a G:U base pair, referred to also as a G:U wobble, is present near the 5′ or 3′ end of the complex formed between the miRNA and the target. Understanding the natural mechanism by which miRNAs efficiency mediate RNAi in plants allows for the design of improved RNAi agents for use in mediating RNAi not only in plants, but in eukaryotes (in particular, in mammals).</p>
    <heading>Example I</heading> <heading>Two Distinct Classes of Small RNAs Derived from dsRNA in Plant Extracts</heading> <p num="p-0203">Two distinct classes of small RNAs are produced in transgenic plants bearing silenced transgenes (Hamilton et al. 2002; Mallory et al. 2002). To test if the production of these two classes of small RNAs was a normal feature of plant biology or a specialized response to foreign DNA, the length distribution of a nonredundant set of 423 endogenous small RNAs cloned from <i>Arabidopsis thaliana </i>was examined. (For the sequences of 143 of the small RNAs see Llave et al. 2002a; Reinhart et al. 2002). Excluded from this analysis are cloned fragments of tRNA and rRNA. Included in the set are known and predicted miRNAs, as well as small RNAs of unknown function corresponding to intragenic regions or to mRNA sequences in either the sense or antisense orientation. The distribution of lengths within this set was bimodal, with peaks at 21 and 24 nt (<figref idrefs="DRAWINGS">FIG. 1A</figref>). In contrast, the length distribution of cloned small RNAs from <i>C. elegans </i>forms a single broad peak (Lau et al. 2001). The two classes of green fluorescent protein (GFP)-derived small RNAs were proposed to be siRNAs with distinct RNA silencing functions: the ˜21-mers to direct posttranscriptional silencing via mRNA degradation and the ˜24-mers to trigger systemic silencing and the methylation of homologous DNA (Hamilton et al. 2002). Analysis of the two classes of endogenous small RNAs indicates that each class has a distinct sequence bias, with a 5′-uridine predominating in the shorter class and a 5′-adenosine in the longer class (<figref idrefs="DRAWINGS">FIG. 1B</figref>). The 5′ sequence bias of the short class is produced by the inclusion in the data set of miRNAs, which in plants and animals typically begin with uridine (Lagos-Quintana et al. 2001, 2002; Lau et al. 2001; Lee and Ambros 2001; Reinhart et al. 2002). Thus, the non-miRNA small RNAs in the shorter class display no 5′ sequence bias, whereas a 5′-adenosine is overrepresented in the longer class. The two classes are either generated by different enzymes, function in separate effector complexes, or both.</p>
    <heading>Example II</heading> <heading>Plant Small RNAs are Bona Fide siRNAs</heading> <p num="p-0204">Although the small RNAs that correlate with the posttranscriptional silencing of homologous target mRNAs were first discovered in plants (Hamilton and Baulcombe 1999), they have not yet been shown to be the direct products of endonucleolytic cleavage of long dsRNA. To begin to test if small RNAs are, in fact, siRNAs, we prepared plant extracts and monitored them for Dicer-like activity. When uniformly 32P-radiolabeled dsRNA was incubated in wheat germ extract, it was efficiently cleaved into small RNAs (<figref idrefs="DRAWINGS">FIG. 2A</figref>). As reported previously for extracts of <i>Drosophila </i>(Zamore et al. 2000) and for purified <i>Drosophila </i>(Bernstein et al. 2001) and human Dicer (Billy et al. 2001), no intermediate products were detected in the conversion of dsRNA into small RNAs. Unlike the fly and human Dicer reactions, two discrete size classes, one ˜21-nt and the other 24-25-nt long, were produced from the dsRNA upon incubation in wheat germ extract (<figref idrefs="DRAWINGS">FIG. 2B</figref>). The ratio of wheat 24-25-mers to ˜21-mers in 14 separate reactions was 4±1.7, similar to the roughly 2.5-fold excess of longer small RNA sequences cloned from <i>Arabidopsis</i>. (The 2.5-fold excess of long to short, cloned endogenous small RNAs underestimates the ratio, because it includes miRNAs, which are predominantly short.). Silencing-related small RNAs have thus far only been demonstrated in vivo for dicots, and wheat is a monocot. Extracts of the dicot cauliflower, a member of the mustard family like <i>Arabidopsis</i>, also converted dsRNA into two discrete sizes of small RNAs, ˜21 and ˜24 nt (<figref idrefs="DRAWINGS">FIG. 2C</figref>). In both <i>Drosophila </i>and <i>C. elegans</i>, Dicer requires ATP for efficient production of both siRNAs (Zamore et al. 2000; Bernstein et al. 2001; Nykanen et al. 2001) and miRNAs (Hutvágner et al. 2001; Ketting et al. 2001). Consistent with the idea that both classes of small RNAs are produced by plant orthologs of Dicer, efficient production of both the ˜21-nt and the ˜24-nt small RNAs in wheat germ extract required ATP (<figref idrefs="DRAWINGS">FIG. 2D</figref>).</p>
    <p num="p-0205">Although small, silencing-associated RNAs in plants are commonly called siRNAs, and synthetic siRNA duplexes initiate plant RNA silencing (Klahre et al. 2002), plant small RNAs have not been demonstrated to be double-stranded RNAs with 2-nt, 3′ overhanging ends and 3′-hydroxyl termini. Such attributes reflect the unique production of siRNAs by members of the Dicer family of ribonuclease III enzymes. To determine if the small RNAs generated from dsRNA in wheat germ extracts were bona fide siRNAs, we analyzed their structure. Uniformly 32P-radiolabeled dsRNA was incubated in wheat germ extract, deproteinized, and fractionated by gel filtration to resolve single-stranded from double-stranded siRNA (Nykanen et al. 2001). Both classes of small RNA products of the in vitro wheat germ reaction comigrated with a synthetic siRNA duplex and with <i>Drosophila </i>siRNA duplexes generated by processing dsRNA in <i>Drosophila </i>embryo lysate (<figref idrefs="DRAWINGS">FIG. 2E</figref>). Therefore, the small RNAs generated by incubating dsRNA in wheat germ extract are double-stranded.</p>
    <p num="p-0206">Next, we examined the end structure of the small RNAs. Treatment of 5′-32P-radiolabeled, synthetic siRNA duplexes with the single-stranded RNA-specific nucleases Ti and RNase A removes the 2-nt, 3′ overhanging ends typical of siRNAs, generating 1-nt and 2-nt shorter RNAs. In a denaturing polyacrylamide gel, such nuclease products of siRNAs migrate faster, because they contain 3′-phosphates (diagramed in <figref idrefs="DRAWINGS">FIG. 2F</figref>). When synthetic 25-nt duplexes with 2-nt, 3′ overhangs were digested with Ti and RNase A, the expected 24-nt and 23-nt, 3′ phosphorylated products were generated (<figref idrefs="DRAWINGS">FIG. 2G</figref>). The small RNAs produced by incubation of dsRNA in the wheat germ extract are a mixture of ˜21-nt and 24-25-nt species. Digestion of this mixture with single-stranded nucleases produced a faster-migrating population of RNA species whose length distribution is consistent with the original mixture having the single-stranded overhangs and double-stranded body characteristic of siRNAs (<figref idrefs="DRAWINGS">FIG. 2G</figref>). Both size classes of small RNAs produced upon incubation of dsRNA in wheat germ extract have 2′,3′-hydroxyl and 5′ monophosphate termini (data not shown). In sum, the small RNAs have all the hallmarks of the products of Dicer-mediated cleavage of dsRNA. It is concluded that they are bona fide siRNAs.</p>
    <heading>Example III</heading> <heading>Different Dicer-like Enzymes Produce each Class of siRNA</heading> <p num="p-0207">There are at least two mechanisms by which long dsRNA could be converted in plants into distinct size classes of small RNAs. Local dsRNA sequence might determine siRNA length, irrespective of which Dicer ortholog cleaves the dsRNA. In this case, we anticipate that the two classes of small RNAs would have distinct sequence compositions. Instead, only the 5′ ends of the two classes show sequence bias (<figref idrefs="DRAWINGS">FIG. 1B</figref>). An alternative explanation is that different Dicer orthologs produce each class. Both the <i>Arabidopsis </i>and rice genomes encode at least four different Dicer-like proteins, including the <i>Arabidopsis </i>protein CARPEL FACTORY/SHORT INTEGUMENTS-1 (CAF). The number of wheat Dicer orthologs is presently unknown, because the hexaploid wheat genome remains to be sequenced.</p>
    <p num="p-0208"> <i>Drosophila </i>Dicer binds tightly to siRNAs (P. D. Zamore and B. Haley, unpubl.). Therefore, we reasoned that different Dicer orthologs might be differentially inhibited by their products, siRNAs. We tested the ability of 21-nt and 25-nt synthetic siRNA duplexes to inhibit the production of siRNAs in <i>Drosophila </i>embryo lysates and the production of the two distinct classes of siRNA in wheat germ extract. <i>Drosophila </i>Dicer produces siRNAs 21-22 nt long. <i>Drosophila </i>Dicer was inhibited more strongly by a 21-nt siRNA duplex than by a 25-mer (<figref idrefs="DRAWINGS">FIG. 3A</figref>). Conversely, production of 24-25-nt siRNAs by wheat germ extract was inhibited more strongly by an ˜25-nt synthetic siRNA duplex competitor than a 21-mer (<figref idrefs="DRAWINGS">FIG. 3B</figref>). These results are consistent with the idea that the authentic siRNA product of Dicer should bind more strongly to its active site than an siRNA of an inauthentic length. Surprisingly, production of the ˜21-nt siRNAs was completely refractory to inhibition by either 21-nt or 25-nt synthetic siRNA duplexes, at siRNA concentrations as high as 800 nM (<figref idrefs="DRAWINGS">FIG. 3B</figref>). The simplest explanation for these data is that a different Dicer-like enzyme generates each class of siRNA and that the enzyme responsible for producing the 24-25-nt siRNAs is strongly inhibited by its siRNA product, whereas the enzyme that produces the ˜21-nt siRNAs is not inhibited by siRNA product at the concentrations tested. An alternative explanation is that the concentration of the enzyme that produces the ˜21-mers is higher than the highest concentration of inhibitor we tested, 800 nM. For this to be true, the enzyme would need to be present at micromolar concentration in the extract, which seems unlikely, as it would then correspond to ˜1% of total protein. The finding that production of both classes of siRNAs were equally and strongly inhibited by long dsRNA competitor (<figref idrefs="DRAWINGS">FIG. 3C</figref>) also supports an argument against this view. If the enzyme that generates the 21-mers were present in the extract at very high concentration, its activity should not have been competed by the same concentrations of long dsRNA competitor that saturate the enzyme that produces the 24-25-nt products. It is concluded that each class of siRNA is produced by the ATP-dependent, endonucleolytic cleavage of dsRNA by a different Dicer ortholog.</p>
    <heading>Example IV</heading> <heading>An RNA-dependent RNA Polymerase Activity in Wheat Germ Extracts</heading> <p num="p-0209">Genetic evidence implicates an RNA-dependent RNA polymerase (RdRP) in PTGS triggered by transgenes expressing sense mRNA (S-PTGS; Dalmay et al. 2000; Mourrain et al. 2000). Plant RdRPs have been proposed to generate dsRNA from aberrantly expressed single-stranded RNA, thereby leading to the production of siRNAs that silence that RNA (Vaucheret et al. 2001). No direct biochemical evidence has yet been presented demonstrating that such a pathway is plausible.</p>
    <p num="p-0210">Wheat germ extracts contain an RdRP activity (<figref idrefs="DRAWINGS">FIG. 4</figref>). Increasing concentrations of single-stranded RNA were incubated with the extract and ribonucleotide triphosphates, including [alpha]-32P-UTP. Single-stranded RNA ranging from 77 to 501 nt, either bearing a 7-methyl-G(5′)ppp(5′)G or an A(5′)ppp(5′) cap structure, all led to the incorporation of 32P into RNA with approximately the same length as the exogenous, nonradioactive single-stranded RNA (<figref idrefs="DRAWINGS">FIG. 4</figref>). These radioactive RNAs correspond to bona fide complementary RNA (cRNA) generated by an RdRP that copied the single-stranded RNA by initiating RNA synthesis at the extreme 3′ end of the exogenous template RNA (data not shown). In theory, these newly radioactive RNAs could have arisen by transfer of radiolabel to the input RNA itself. This type of label transfer had previously been observed when similar experiments were performed using <i>Drosophila </i>embryo lysates, but not with wheat germ extract. Instead, the 32P-RNA represents newly synthesized cRNA produced by a wheat enzyme using exogenous single-stranded RNA as a template in the absence of an exogenous nucleic acid primer</p>
    <p num="p-0211">In addition to copying single-stranded RNA into approximately full-length cRNA, RdRPs have also been reported to extend primers, using single-stranded RNA as a template (e.g., Schiebel et al. 1998). The RdRP activity or activities in wheat germ extract could similarly extend a 32P-radiolabeled primer (<figref idrefs="DRAWINGS">FIG. 5A</figref>), but only when the RNA primer was complementary (antisense) to the template RNA. Under identical conditions, no such primer-extension activity was detected in lysates of syncitial blastoderm <i>Drosophila </i>embryos, despite earlier reports to the contrary (Lipardi et al. 2001). RNA-dependent, RNA-primer-extension activity was detected, however, when wheat and fly extracts are mixed (<figref idrefs="DRAWINGS">FIG. 5A</figref>). In neither <i>Drosophila </i>embryo lysate nor wheat germ extract can we detect primer extension of a single-stranded RNA template using a 21-nt siRNA duplex rather than a 21-nt antisense primer.</p>
    <p num="p-0212">It is proposed that aberrant single-stranded RNA triggers silencing in plants when it serves as a template for the production of cRNA, generating dsRNA, which can then be cleaved by Dicer into siRNA duplexes. These data suggest that such copying does not require primers, but is triggered merely by an exceptionally high concentration of single-stranded RNA. To test if high concentrations of single-stranded mRNA could lead to the production of siRNAs, the RdRP reactions were repeated using a 2.7-kb single-stranded firefly luciferase mRNA. Increasing concentrations of the mRNA were incubated in either wheat germ extract or <i>Drosophila </i>embryo lysate in the presence of ATP, CTP, GTP, and [alpha]-32P-UTP, and examined for the production of 21-25-nt radioactive RNAs. <figref idrefs="DRAWINGS">FIG. 5B</figref> (left) shows that when the incubations were performed in wheat germ lysates, a single class of small RNA, ˜24 nt long, was produced with increasing concentrations of the exogenous, single-stranded template RNA. No such radioactive product was observed in <i>Drosophila </i>embryo lysates, but it is noted that these lysates contain endogenous UTP, which may preclude detection of 32P small RNAs. To test if the radiolabeled ˜24-nt products were generated by the de novo synthesis of RNA, the experiment was repeated, replacing CTP and GTP with 3′-deoxy CTP and 3′-deoxy GTP, inhibitors of RNA synthesis. In the presence of these inhibitors, no radioactive small RNAs were observed in the wheat reaction (<figref idrefs="DRAWINGS">FIG. 5B</figref>, right). Thus, single-stranded RNA can trigger in wheat germ extract the de novo synthesis of ˜24-nt small RNAs.</p>
    <p num="p-0213">Notably, the production of 21-nt RNAs was not detected in this assay. The assay should have detected such 21-nt small RNAs if they were present at 1/10 the concentration of the ˜24-mers, but we would be unlikely to detect them far below this threshold. Experiments with double-stranded RNA suggest that the 21-mers are produced in wheat at about ¼ the rate of the 24-25-nt small RNAs (<figref idrefs="DRAWINGS">FIG. 2</figref>). Thus, the production of dsRNA by the RdRP activity may be coupled to the production of the longer class of small RNAs. It is noted that such coupling does not imply that production of ˜24-nt siRNAs from exogenous dsRNA requires the participation of an RdRP. It is proposed that dsRNA generated by RdRP copying of single-stranded RNA is preferentially processed by a wheat Dicer ortholog that produces long siRNAs, perhaps because the two proteins are physically linked.</p>
    <p num="p-0214">The question of whether the ˜24-nt RNAs synthesized in the RdRP reactions are actual products of Dicer cleavage of dsRNA was next addressed. Production of wheat 24-25-nt siRNAs from 32P-radiolabeled dsRNA is efficiently inhibited by synthetic siRNA duplexes; 25-nt synthetic siRNA duplexes are more potent inhibitors than 21-nt duplexes (<figref idrefs="DRAWINGS">FIG. 3B</figref>). Therefore, an experiment was conducted to determine if production of the ˜24-nt small RNAs in the RdRP reactions was similarly inhibited by synthetic siRNA duplexes. <figref idrefs="DRAWINGS">FIG. 5C</figref> shows that the production of ˜24-nt small RNAs in the RdRP reactions programmed with a 2.7-kb single-stranded RNA template was inhibited by synthetic siRNA duplexes. Like the production of 24-25-nt siRNAs from exogenous dsRNA, production of the de novo synthesized ˜24-mers was inhibited to a greater extent by 25-nt synthetic siRNA duplexes than by 21-nt duplexes (<figref idrefs="DRAWINGS">FIG. 5C</figref>). Half-maximal inhibition of small RNA production in the RdRP-dependent reactions occurred at roughly the same concentration of synthetic siRNA duplex as inhibition of the processing of 32P dsRNA (cf. <figref idrefs="DRAWINGS">FIGS. 5C and 3B</figref>). It is concluded that in wheat germ extract, exogenous single-stranded RNA provides the template for the synthesis of cRNA by an RdRP and that the resulting template-RNA:cRNA hybrid is then preferentially cleaved into ˜24-nt siRNAs by a Dicer-like enzyme.</p>
    <heading>Example V</heading> <heading>miRNAs Act as siRNAs in Plants</heading> <p num="p-0215">In addition to siRNAs, another class of small RNAs, microRNAs (miRNAs), has been detected in plants (Llave et al. 2002a; Park et al. 2002; Reinhart et al. 2002). Like their animal counterparts, plant miRNAs are generated by a Dicer family member, CAF. miRNAs are encoded in stem-loop precursor RNAs that are cleaved by CAF into 21-24-nt single-stranded small RNAs (Park et al. 2002; Reinhart et al. 2002). Exogenous miRNA precursors were not faithfully processed into mature miRNAs in wheat germ extract (data not shown). Instead, in vitro transcribed pre-miRNAs were cleaved into small RNAs too long to correspond to authentic, mature miRNAs. Perhaps the Dicer ortholog responsible for miRNA maturation in wheat—presumably wheat CAF—is absent from wheat germ extracts. In <i>Arabidopsis</i>, CAF transcripts that encode a protein with a nuclear localization signal have been reported, suggesting that CAF protein may be nuclear (Jacobsen et al. 1999). Because wheat germ extracts are essentially cytoplasm, nuclear CAF might not be present in the extract.</p>
    <p num="p-0216">Plant miRNAs differ from animal miRNAs in that there are corresponding mRNA sequences in the <i>Arabidopsis </i>and rice genomes with significant complementarity to miRNA sequences (Llave et al. 2002a, b; Reinhart et al. 2002; Rhoades et al. 2002). The high degree of complementarity between 14 recently analyzed plant miRNAs and specific families of developmentally important plant mRNAs led to the proposal that plant miRNAs direct developmentally controlled mRNA destruction (Rhoades et al. 2002). That is, after the plant miRNAs are generated by the cleavage of pre-miRNAs by CAF, they enter the RNAi pathway and function as siRNAs. In contrast, animal miRNAs are thought to act as translational repressors (for review, see Ruvkun 2001). An untested feature of this proposal is that an RNAi-like pathway in plants tolerates the three to four mismatches sometimes observed between an miRNA and its predicted mRNA target.</p>
    <p num="p-0217">If plant miRNAs are endogenous mediators of RNAi, then wheat germ extracts should contain miRNA-programmed complexes that specify endonucleolytic cleavage of corresponding target RNAs. In particular, miR165 has been proposed to down-regulate PHV and PHABULOSA (PHB) mRNA expression in <i>Arabidopsis </i>by an RNAi-like mechanism (Rhoades et al. 2002). PHV and PHB encode homeodomain-leucine zipper transcription factors implicated in the perception of radial position in the shoot tissues that give rise to leaves (McConnell and Barton 1998; McConnell et al. 2001). Dominant phv and phb mutations alter a single amino acid (glycine→glutamic acid) in the sterol/lipid-binding domain of the proteins, suggesting that the mutant phenotype results from a change in the function of PHV and PHB (McConnell and Barton 1998; McConnell et al. 2001). However, the discovery of plant miRNAs complementary to this site in PHV led to the suggestion that the molecular basis of the dominance is the persistence of PHV and PHB expression at developmental stages when these mRNAs are normally destroyed (Rhoades et al. 2002). This hypothesis is consistent with both the increased overall levels of PHB mRNA in the dominant mutant and the increased activity of a dominant mutant phb mRNA on the abaxial, rather than the adaxial, domain of the leaf primordium (McConnell and Barton 1998; McConnell et al. 2001).</p>
    <p num="p-0218">miR165 or miR166 is present in wheat germ extracts (<figref idrefs="DRAWINGS">FIG. 6A</figref>). miR165 and miR166 differ by a single C-to-U transition that decreases the complementarity of miR166 to PHV and PHB by changing a G:C base pair to a G:U wobble. Rice (Oryza) is the sequenced genome most closely related to wheat. Although the rice genome encodes no miR165 homolog, it encodes six copies of miR166 (Reinhart et al. 2002). Because the Northern hybridization conditions used herein cannot distinguish between miR165 and miR166, the endogenous wheat miRNA is referred to as miR165/166.</p>
    <p num="p-0219">To begin to test the hypothesis that plant miRNAs function to regulate target gene expression by an RNAi-like mechanism, target RNAs were prepared encoding a portion of the wild-type sequence of <i>Arabidopsis </i>PHV or the dominant G→A point mutation, which falls within the PHV sequences proposed to pair with miR165/166. The target RNAs and relevant miRNAs are shown in <figref idrefs="DRAWINGS">FIG. 6C</figref>. 5′-radiolabeled target RNAs were incubated with wheat germ extract, then analyzed on a denaturing sequencing gel. In the absence of any other exogenous RNA, the wild-type PHV target RNA, but not the dominant G→A mutant, was efficiently cleaved within the region complementary to miR165/166 (FIG. <b>7</b>A,B). This 21-nt region is identical in PHV and PHB, and a target RNA that contained sequence from the <i>Arabidopsis </i>PHB mRNA was also cleaved within the sequences complementary to miR165/166 upon incubation in the wheat germ extract (data not shown). In the RNAi pathway, a key feature of small RNA-directed target destruction is that pretreatment with the single-stranded nucleic acid-specific enzyme, micrococcal nuclease, abolishes RISC activity (Hammond et al. 2000). Cleavage of the PHV target RNA was likewise abolished by pretreatment of the extract with micrococcal nuclease (data not shown), consistent with the view that miR165/166 acts as a guide to direct target cleavage. The difference in cleavage rate between wild-type and mutant target RNAs, which differ only at a single nucleotide, was &gt;14-fold (<figref idrefs="DRAWINGS">FIG. 7B</figref>). Thus, the resistance of the mutant phv RNA to cleavage by an endogenous RNAi-like nuclease can explain why the mutation is dominant.</p>
    <p num="p-0220">Next, cleavage of the PHV target RNA by various siRNAs was analuzed in <i>Drosophila </i>embryo lysate (<figref idrefs="DRAWINGS">FIG. 6C</figref>). An siRNA with perfect complementarity to the site predicted to pair with miR165/166 and an siRNA duplex in which one strand had the sequence of miR165 or miR166 directed cleavage of the PHV target RNA, yielding the predicted 514-nt 5′ cleavage product (<figref idrefs="DRAWINGS">FIG. 7C</figref>). None of these three siRNAs efficiently cleaved the PHV mutant target (<figref idrefs="DRAWINGS">FIG. 7C</figref>). Quantification of the cleavage mediated by the siRNA with perfect complementarity as compared to that mediated by miR165 demonstrates that miR165 more efficiently mediates target cleavage. This is presumed to be due to the fact that miR165 forms two G:U wobble base pairs with the target mRNA, one at position 1 and one at position 17 (with respect to the 5′ end of the antisense siRNA strand) (<figref idrefs="DRAWINGS">FIG. 8</figref>).</p>
    <p num="p-0221">The failure of the miR165-siRNA duplex to cleave mutant PHV was a direct consequence of its reduced complementarity to the target RNA at position 6 (with respect to the 5′ end of the antisense siRNA strand), because an siRNA with perfect complementarity to the mutant sequence (<figref idrefs="DRAWINGS">FIG. 6B</figref>) efficiently cleaved the mutant RNA (<figref idrefs="DRAWINGS">FIG. 7C</figref>). The 5′ cleavage product produced in the siRNA-programmed RNAi reactions comigrated with that produced when the PHV target RNA was incubated in wheat germ extract without exogenous siRNA (<figref idrefs="DRAWINGS">FIG. 7C</figref>).</p>
    <p num="p-0222">The simplest explanation for the sequence-specificity of the nuclease is that it is guided by miR165/166: cleavage requires a nucleic acid component, occurs at the same site on the PHV target RNA as directed by an siRNA duplex with the sequence of miR165 or miR166 in <i>Drosophila </i>embryo lysate, and, like the siRNA, is inefficient with the G→A mutant phv RNA. In the RNAi pathway, an siRNA-programmed endonuclease complex is called an RISC (Hammond et al. 2000). These data suggest that wheat miR165/166 is in an RISC, supporting the proposal that plant miRNAs regulate expression of their mRNA targets by endogenous RNAi.</p>
    <heading>Example VI</heading> <heading>miR165/166 Directs Multiple Rounds of Target Cleavage</heading> <p num="p-0223">The next question addressed was whether the miR165/166-programmed RISC acts as an enzyme. Quantitative Northern hybridization demonstrates that the wheat germ extract reactions contained 0.083 nM miR165/166 (<figref idrefs="DRAWINGS">FIG. 6B</figref>). The target RNA concentration in these reactions was 5 nM, and more than half the target RNA was destroyed in 80 min (<figref idrefs="DRAWINGS">FIG. 7A</figref>). Thus, each miR165/166 RNA directed cleavage of ˜30 target RNA molecules. Therefore, the miR165/166-programmed RISC is a multiple-turnover enzyme.</p>
    <p num="p-0224">Discussion</p>
    <p num="p-0225">The above data show that wheat germ extracts recapitulate in vitro many aspects of RNA silencing in plants. Wheat germ extracts convert exogenous dsRNA into two distinct classes of small RNAs. Detailed analysis of these small RNAs indicates that they are bona fide siRNAs. Thus, plant siRNAs are derived directly from longer dsRNA, just as in animals. The data indicate that distinct Dicer-like enzymes generate the two functionally distinct classes of siRNAs. Cloned endogenous small RNAs from <i>Arabidopsis </i>likewise form two distinct length classes, whose 5′ ends indicate that they are made by distinct enzymes. An alternative view, that one or more Dicer-like enzymes may generate both classes of small RNAs, with the different lengths a byproduct of local sequence context, is not consistent with the above observation that production of 24-25-nt RNAs in wheat germ extract was inhibited by synthetic siRNA duplexes, whereas ˜21-nt siRNA production was not. If the production of siRNAs is tightly coupled to the assembly of downstream effector complexes, then their production by different Dicer orthologs may ensure that the two classes of siRNAs function in different cellular pathways (see also Hamilton et al. 2002).</p>
    <p num="p-0226">A hallmark of PTGS in plants and RNAi in nematodes is the spreading of silencing signals along the length of the mRNA target. In plants, spreading occurs in both the 5′ and 3′ directions and requires the putative RdRP gene, SGS2. Spreading is observed even when silencing is initiated by a single siRNA sequence (Klahre et al. 2002). One hypothesis is that 5′ spreading is initiated by the antisense siRNA strand priming copying of the target mRNA by an RdRP, thereby producing dsRNA. 3′ spreading cannot be explained by such a mechanism. Both 5′ and 3′ spreading might instead be catalyzed by the conversion of mRNA fragments into dsRNA by an RdRP that initiates synthesis at the 3′ end of the two fragments generated when an RISC cleaves the target RNA. This dsRNA would then be cleaved by a Dicer-like enzyme to produce secondary siRNAs (Lipardi et al. 2001; Sijen et al. 2001). Such RNA synthesis would occur without the involvement of a primer. The above data demonstrate that exogenous single-stranded RNA is copied into cRNA in the extract by a wheat RdRP that acts without the aid of an exogenous primer. The resulting dsRNA is cleaved preferentially into the longer class of siRNAs, suggesting the RdRP is physically linked to a specific Dicer ortholog. The specific biochemical function of the 24-25-nt siRNAs generated in this reaction remains to be determined.</p>
    <p num="p-0227">miRNAs Function as siRNAs in Plants</p>
    <p num="p-0228">The above data further show that miRNAs in plants function in much the same way that siRNA duplexes function in <i>Drosophila </i>and humans: as guides for an endonuclease complex. Each endonuclease complex can catalyze multiple rounds of target cleavage, indicating that the miRNA is not consumed in the reaction. Entry of a miRNA into a multiple-turnover RNAi enzyme complex is not unprecedented; in human cells, the miRNA let-7 is a component of an RISC, although the human genome does not appear to contain any mRNA sequences with sufficient complementarity to be cleaved by this RISC (Hutvágner and Zamore 2002). Like the plant miR165/166-programmed RISC, the human let-7-programmed RISC can catalyze multiple rounds of target cleavage.</p>
    <p num="p-0229">Additional support for the idea that plant miRNAs direct cleavage of complementary mRNA targets comes from the work of Carrington and colleagues, who recently showed that a family of <i>Arabidopsis </i>mRNAs encoding SCARECROW-LIKE (SCL) transcription factors is cleaved by an RNAi-like process directed by miR171, an miRNA that is fully complementary to its mRNA targets, unlike miR165/166 (Llave et al. 2002b). Like wheat miR165/166, <i>Arabidopsis </i>miR171 appears to direct the endonucleolytic cleavage of its target mRNAs. In this respect, miR171 functions as if it were a single-stranded siRNA. Single-stranded siRNAs can trigger RNAi in both <i>Drosophila </i>and mammalian cell extracts and in vivo in HeLa cells (Martinez et al. 2002a; Schwarz et al. 2002), although much higher concentrations of single-stranded siRNA is required than for duplex (Schwarz et al. 2002). Furthermore, an individual human RISC contains only one strand of the exogenous siRNA duplex used to trigger RNAi (Martinez et al. 2002a).</p>
    <p num="p-0230">The observation that, in <i>Drosophila </i>embryo lysate, an siRNA with the sequence of miR165, which contains three mismatches with its target mRNA, is at least as potent as an siRNA with perfect complementarity to the same target sequence, demonstrates that mismatches per se do not block target cleavage. Rather, the specific position and sequence of siRNA:target RNA mismatches determine if they permit or disrupt RNAi. The data also suggest that miRNAs in plants evolved to optimize cleavage efficiency rather than maximize complementarity to their targets. It is predicted that three or four mismatches between an miRNA (or the guide strand of an siRNA duplex) and its target RNA, properly placed so as to still permit mRNA cleavage, will facilitate the release of cleaved target RNA from the RISC complex, thereby increasing the rate of enzyme turnover.</p>
    <p num="p-0231">miRNA Function and the Spread of Silencing Signals Along a Silenced Sequence</p>
    <p num="p-0232">Spreading of silencing signals along the length of a silenced mRNA sequence is a common feature of plant RNA silencing. Because miRNAs act as siRNAs, one might anticipate that they would also elicit spreading. However, miRNA-induced spreading is not consistent with the genetics of the PHV and PHB mutants; the very existence of a dominant PHV mutant excludes both 5′ and 3′ spreading. Spreading of the silencing signal—that is, the generation of new siRNAs 5′ or 3′ to the site of initial target cleavage—would produce siRNAs containing sequences common to both the wild-type and mutant PHV mRNAs. If such siRNAs were generated, they would direct destruction of the mutant PHV mRNA. In such a case, the PHV mutant could only have been recovered as a recessive, not a dominant allele. Genetic studies (McConnell et al. 2001) show that endonucleolytic cleavage of target RNAs by miRNA-directed RISC complexes does not trigger spreading in plants. This remains true even when the miRNA is the perfect complement of its mRNA target (Llave et al. 2002b).</p>
    <p num="p-0233">How, then, can the well-documented spreading phenomenon observed for S-PTGS be reconciled with the absence of spreading in miRNA-directed target cleavage? It is proposed that plants contain two separate mechanisms for target mRNA destruction—endogenous mRNAs are regulated by endonucleolytic cleavage directed by miRNA-programmed RISC complexes, whereas exogenous silencing triggers, such as transgenes or viruses, might initiate successive cycles of siRNA-primed, RdRP-catalyzed dsRNA synthesis, followed by cleavage of the dsRNA into siRNAs by Dicer-like enzymes, a mechanism termed random degradative PCR (Lipardi et al. 2001). RISC complexes would play no role in the execution of target RNAs in this cycle. The observation that a single siRNA sequence can trigger 3′ spreading (Klahre et al. 2002) is difficult to reconcile with a priming mechanism. Intriguingly, VIGS-mediated RNA silencing of endogenous genes is not associated with spreading of silencing into regions of the target sequence 5′ or 3′ to the initial silencing trigger (Vaistij et al. 2002), although such silencing clearly must involve siRNAs derived from viral dsRNA, not endogenous miRNAs.</p>
    <p num="p-0234">An alternative hypothesis is that the absolute concentration of an RNA target might determine if the 5′ and 3′ cleavage fragments generated by target cleavage are converted into dsRNA by an RdRP. Only when the products of RISC-mediated target cleavage accumulate to a sufficiently high concentration would they serve as substrates for the RdRP and consequently trigger spreading. Experiments with polygalacturonase-silenced tomatoes support this view (Han and Grierson 2002). In these plants, siRNAs were produced from the silencing-inducing transgene but not the corresponding silenced endogene. The siRNAs were preferentially produced from the 3′ end of the transgene, consistent with the idea that plant RdRPs act without aid of a primer. Furthermore, these authors detected mRNA degradation products consistent with endonucleolytic cleavage of the targeted polygalacturonase endogene. Thus, RISC-mediated cleavage per se does not appear to trigger spreading along the target RNA sequence. More likely, the endonucleolytic cleavage of transgenic mRNA produces a sufficiently high concentration of mRNA fragments to recruit an RdRP, resulting in the production of siRNAs from the 3′ cleavage product. miRNA-directed cleavage of natural plant regulatory targets would not lead to spreading, because endogenous mRNA targets are not present at sufficiently high concentrations to recruit the RdRP. This model predicts that the putative RdRP SGS2 (SDE1) required for PTGS, will not be required for miRNA-directed destruction of endogenous mRNA targets. In fact, no developmental abnormalities have been reported for SGS2 mutants (Mourrain et al. 2000), including mutations likely to be strongly hypomorphic or functionally null (Dalmay et al. 2000), suggesting that plants lacking SGS2 protein have normal miRNA biogenesis and function.</p>
    <heading>REFERENCES</heading> <p num="p-0235">
      </p> <ul> <li id="ul0001-0001" num="0234">Beclin, C., Boutet, S., Waterhouse, P., and Vaucheret, H. 2002. A branched pathway for transgene-induced RNA silencing in plants. Curr. Biol. 12: 684-688.</li>
        <li id="ul0001-0002" num="0235">Bernstein, E., Caudy, A. A., Hammond, S. M., and Hannon, G. J. 2001. Role for a bidentate ribonuclease in the initiation step of RNA interference. Nature 409: 363-366.</li>
        <li id="ul0001-0003" num="0236">Billy, E., Brondani, V., Zhang, H., Muller, U., and Filipowicz, W. 2001. Specific interference with gene expression induced by long, double-stranded RNA in mouse embryonal teratocarcinoma cell lines. Proc. Natl. Acad. Sci. 98: 14428-14433.</li>
        <li id="ul0001-0004" num="0237">Celotto, A. M. and Graveley, B. R. 2002. Exon-specific RNAi: A tool for dissecting the functional relevance of alternative splicing. RNA 8: 8-24.</li>
        <li id="ul0001-0005" num="0238">Chiu, Y.-L. and Rana, T. M. 2002. RNAi in human cells: Basic structural and functional features of small interfering RNA. Mol. Cell. 10: 549-561.</li>
        <li id="ul0001-0006" num="0239">Cogoni, C. and Macino, G. 1999. Gene silencing in <i>Neurospora crassa </i>requires a protein homologous to RNA-dependent RNA polymerase. Nature 399: 166-169.</li>
        <li id="ul0001-0007" num="0240">Dalmay, T., Hamilton, A., Rudd, S., Angell, S., and Baulcombe, D.C. 2000. An RNA-dependent RNA polymerase gene in <i>Arabidopsis </i>is required for posttranscriptional gene silencing mediated by a transgene but not by a virus. Cell 101: 543-553.</li>
        <li id="ul0001-0008" num="0241">Djikeng, A., Shi, H., Tschudi, C., and Ullu, E. 2001. RNA interference in <i>Trypanosoma brucei</i>: Cloning of small interfering RNAs provides evidence for retroposon-derived 24-26-nucleotide RNAs. RNA 7: 1522-1530.</li>
        <li id="ul0001-0009" num="0242">Elbashir, S. M., Harborth, J., Lendeckel, W., Yalcin, A., Weber, K., and Tuschl, T. 2001a. Duplexes of 21-nucleotide RNAs mediate RNA interference in mammalian cell culture. Nature 411: 494-498.</li>
        <li id="ul0001-0010" num="0243">Elbashir, S. M., Lendeckel, W., and Tuschl, T. 2001b. RNA interference is mediated by 21- and 22-nucleotide RNAs. Genes &amp; Dev. 15: 188-200.</li>
        <li id="ul0001-0011" num="0244">Elbashir, S. M., Martinez, J., Patkaniowska, A., Lendeckel, W., and Tuschl, T. 2001c. Functional anatomy of siRNAs for mediating efficient RNAi in <i>Drosophila melanogaster </i>embryo lysate. EMBO J. 20: 6877-6888.</li>
        <li id="ul0001-0012" num="0245">Erickson, A. H. and Blobel, G. 1983. Cell-free translation of messenger RNA in a wheat germ system. Methods Enzymol. 96: 38-50.</li>
        <li id="ul0001-0013" num="0246">Fagard, M., Boutet, S., Morel, J.-B., Bellini, C., and Vaucheret, H.2000. AGO1, QDE-2, and RDE-1 are related proteins required for post-transcriptional gene silencing in plants, quelling in fungi, and RNA interference in animals. Proc. Natl. Acad. Sci. 97: 11650-11654.</li>
        <li id="ul0001-0014" num="0247">Grishok, A., Pasquinelli, A. E., Conte, D., Li, N., Parrish, S., Ha, I., Baillie, D. L., Fire, A., Ruvkun, G., and Mello, C. C. 2001. Genes and mechanisms related to RNA interference regulate expression of the small temporal RNAs that control <i>C. elegans </i>developmental timing. Cell 106: 23-34.</li>
        <li id="ul0001-0015" num="0248">Hamilton, A. J. and Baulcombe, D. C. 1999. A species of small antisense RNA in posttranscriptional gene silencing in plants. Science 286: 950-952.</li>
        <li id="ul0001-0016" num="0249">Hamilton, A., Voinnet, O., Chappell, L., and Baulcombe, D. 2002. Two classes of short interfering RNA in RNA silencing. EMBO J. 21: 4671-4679.</li>
        <li id="ul0001-0017" num="0250">Hammond, S. M., Bernstein, E., Beach, D., and Hannon, G. J. 2000. An RNA-directed nuclease mediates post-transcriptional gene silencing in <i>Drosophila </i>cells. Nature 404: 293-296.</li>
        <li id="ul0001-0018" num="0251">Hammond, S. M., Boettcher, S., Caudy, A. A., Kobayashi, R., and Hannon, G. J. 2001. Argonaute2, a link between genetic and biochemical analyses of RNAi. Science 293: 1146-1150.</li>
        <li id="ul0001-0019" num="0252">Han, Y. and Grierson, D. 2002. Relationship between small antisense RNAs and aberrant RNAs associated with sense transgene mediated gene silencing in tomato. Plant J. 29: 509-519.</li>
        <li id="ul0001-0020" num="0253">Hannon, G. J. 2002. RNA interference. Nature 418: 244-251.</li>
        <li id="ul0001-0021" num="0254">Holen, T., Amarzguioui, M., Wiiger, M. T., Babaie, E., and Prydz, H.2002. Positional effects of short interfering RNAs targeting the human coagulation trigger Tissue Factor. Nucleic Acids Res. 30: 1757-1766.</li>
        <li id="ul0001-0022" num="0255">Hutvágner, G. and Zamore, P. D. 2002. A microRNA in a multiple-turnover RNAi enzyme complex. Science 297: 2056-2060.</li>
        <li id="ul0001-0023" num="0256">Hutvágner, G., McLachlan, J., Pasquinelli, A. E., Balint, É., Tuschl, T., and Zamore, P. D. 2001. A cellular function for the RNA-interference enzyme Dicer in the maturation of the let-7 small temporal RNA. Science 293: 834-838.</li>
        <li id="ul0001-0024" num="0257">Jacobsen, S. E., Running, M. P., and Meyerowitz, E. M. 1999. Disruption of an RNA helicase/RNase III gene in <i>Arabidopsis </i>causes unregulated cell division in floral meristems. Development 126: 5231-5243.</li>
        <li id="ul0001-0025" num="0258">Kennerdell, J. R., Yamaguchi, S., and Carthew, R. W. 2002. RNAi is activated during <i>Drosophila </i>oocyte maturation in a manner dependent on aubergine and spindle-E. Genes &amp; Dev. 16: 1884-1889.</li>
        <li id="ul0001-0026" num="0259">Ketting, R. F., Fischer, S. E., Bernstein, E., Sijen, T., Hannon, G. J., and Plasterk, R. H. 2001. Dicer functions in RNA interference and in synthesis of small RNA involved in developmental timing in <i>C. elegans</i>. Genes &amp; Dev. 15: 2654-2659.</li>
        <li id="ul0001-0027" num="0260">Klahre, U., Crete, P., Leuenberger, S. A., Iglesias, V. A., and Meins, F., Jr. 2002. High molecular weight RNAs and small interfering RNAs induce systemic posttranscriptional gene silencing in plants. Proc. Natl. Acad. Sci. 99: 11981-11986.</li>
        <li id="ul0001-0028" num="0261">Kooter, J. M., Matzke, M. A., and Meyer, P. 1999. Listening to the silent genes: Transgene silencing, gene regulation and pathogen control. Trends Plant Sci. 4: 340-347.</li>
        <li id="ul0001-0029" num="0262">Lagos-Quintana, M., Rauhut, R., Lendeckel, W., and Tuschl, T. 2001. Identification of novel genes coding for small expressed RNAs. Science 294: 853-858.</li>
        <li id="ul0001-0030" num="0263">Lagos-Quintana, M., Rauhut, R., Yalcin, A., Meyer, J., Lendeckel, W., and Tuschl, T. 2002. Identification of tissue-specific microRNAs from mouse. Curr. Biol. 12: 735-739.</li>
        <li id="ul0001-0031" num="0264">Lau, N. C., Lim, L. P., Weinstein, E. G., and Bartel, D. P. 2001. An abundant class of tiny RNAs with probable regulatory roles in <i>Caenorhabditis elegans</i>. Science 294: 858-862.</li>
        <li id="ul0001-0032" num="0265">Lee, R. C. and Ambros, V. 2001. An extensive class of small RNAs in <i>Caenorhabditis elegans</i>. Science 294: 862-864.</li>
        <li id="ul0001-0033" num="0266">Lee, R. C., Feinbaum, R. C., and Ambros, V. 1993. The <i>C. elegans </i>heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14. Cell 75: 843-854.</li>
        <li id="ul0001-0034" num="0267">Lee, Y., Jeon, K., Lee, J. T., Kim, S., and Kim, V. N. 2002. MicroRNA maturation: Stepwise processing and subcellular localization. EMBO J. 21: 4663-4670.</li>
        <li id="ul0001-0035" num="0268">Li, W. X. and Ding, S. W. 2001. Viral suppressors of RNA silencing. Curr. Opin. Biotechnol. 12: 150-154.</li>
        <li id="ul0001-0036" num="0269">Lipardi, C., Wei, Q., and Paterson, B. M. 2001. RNAi as random degradative PCR. siRNA primers convert mRNA into dsRNAs that are degraded to generate new siRNAs. Cell 107: 297-307.</li>
        <li id="ul0001-0037" num="0270">Llave, C., Kasschau, K. D., Rector, M. A., and Carrington, J. C. 2002a. Endogenous and silencing-associated small RNAs in plants. Plant Cell 14: 1605-1619.</li>
        <li id="ul0001-0038" num="0271">Llave, C., Xie, Z., Kasschau, K. D., and Carrington, J. C. 2002b. Cleavage of Scarecrow-Like mRNA targets directed by a class of <i>Arabidopsis </i>miRNA. Science 297: 2053-2056.</li>
        <li id="ul0001-0039" num="0272">Mallory, A. C., Reinhart, B. J., Bartel, D., Vance, V. B., and Bowman, L. H. 2002. A viral suppressor of RNA silencing differentially regulates the accumulation of short interfering RNAs and micro-RNAs in tobacco. Proc. Natl. Acad. Sci. 99: 15228-15233.</li>
        <li id="ul0001-0040" num="0273">Martens, H., Novotny, J., Oberstrass, J., Steck, T. L., Postlethwait, P., and Nellen, W. 2002. RNAi in Dictyostelium: The role of RNA-directed RNA polymerases and double-stranded RNase. Mol. Biol. Cell 13: 445-453.</li>
        <li id="ul0001-0041" num="0274">Martinez, J., Patkaniowska, A., Urlaub, H., Lührmann, R., and Tuschl, T. 2002a. Single-stranded antisense siRNAs guide target RNA cleavage in RNAi. Cell 110:563.</li>
        <li id="ul0001-0042" num="0275">Martinez, L. A., Naguibneva, I., Lehrmann, H., Vervisch, A., Tchenio, T., Lozano, G., and Harel-Bellan, A. 2002b. Synthetic small inhibiting RNAs: Efficient tools to inactivate oncogenic mutations and restore p53 pathways. Proc. Natl. Acad. Sci. 99: 14849-14854.</li>
        <li id="ul0001-0043" num="0276">Matzke, M. A., Matzke, A. J., Pruss, G. J., and Vance, V. B. 2001. RNA-based silencing strategies in plants. Curr. Opin. Genet. Dev. 11: 221-227.</li>
        <li id="ul0001-0044" num="0277">McConnell, J. R. and Barton, M. K. 1998. Leaf polarity and meristem formation in <i>Arabidopsis</i>. Development 125: 2935-2942.</li>
        <li id="ul0001-0045" num="0278">McConnell, J. R., Emery, J., Eshed, Y., Bao, N., Bowman, J., and Barton, M. K. 2001. Role of PHABULOSA and PHAVOLUTA in determining radial patterning in shoots. Nature 411: 709-713.</li>
        <li id="ul0001-0046" num="0279">Mourelatos, Z., Dostie, J., Paushkin, S., Sharma, A. K., Charroux, B., Abel, L., Rappsilber, J., Mann, M., and Dreyfuss, G. 2002. miRNPs: A novel class of ribonucleoproteins containing numerous microRNAs. Genes &amp; Dev. 16: 720-728.</li>
        <li id="ul0001-0047" num="0280">Mourrain, P., Beclin, C., Elmayan, T., Feuerbach, F., Godon, C., Morel, J. B., Jouette, D., Lacombe, A. M., Nikic, S., Picault, N. et al. 2000. <i>Arabidopsis </i>SGS2 and SGS3 genes are required for posttranscriptional gene silencing and natural virus resistance. Cell 101: 533-542.</li>
        <li id="ul0001-0048" num="0281">Nykanen, A., Haley, B., and Zamore, P. D. 2001. ATP requirements and small interfering RNA structure in the RNA interference pathway. Cell 107: 309-321.</li>
        <li id="ul0001-0049" num="0282">Pal-Bhadra, M., Bhadra, U., and Birchler, J. A. 2002. RNAi related mechanisms affect both transcriptional and posttranscriptional transgene silencing in <i>Drosophila</i>. Mol. Cell. 9: 315-327.</li>
        <li id="ul0001-0050" num="0283">Park, W., Li, J., Song, R., Messing, J., and Chen, X. 2002. CARPEL FACTORY, a Dicer homolog, and HEN1, a novel protein, act in microRNA metabolism in <i>Arabidopsis thaliana</i>. Curr. Biol. 12: 1484-1495.</li>
        <li id="ul0001-0051" num="0284">Parrish, S, and Fire, A. 2001. Distinct roles for RDE-1 and RDE-4 during RNA interference in <i>Caenorhabditis elegans</i>. RNA 7: 1397-1402.</li>
        <li id="ul0001-0052" num="0285">Parrish, S., Fleenor, J., Xu, S., Mello, C., and Fire, A. 2000. Functional anatomy of a dsRNA trigger. Differential requirement for the two trigger strands in RNA interference. Mol. Cell. 6: 1077-1087.</li>
        <li id="ul0001-0053" num="0286">Pasquinelli, A. E., Reinhart, B. J., Slack, F., Martindale, M. Q., Kuroda, M. I., Maller, B., Hayward, D. C., Ball, E. E., Degnan, B., Muller, P. et al. 2000. Conservation of the sequence and temporal expression of let-7 heterochronic regulatory RNA. Nature 408: 86-89.</li>
        <li id="ul0001-0054" num="0287">Plasterk, R. H. 2002. RNA silencing: The genome's immune system. Science 296: 1263-1265.</li>
        <li id="ul0001-0055" num="0288">Reinhart, B. J., Slack, F. J., Basson, M., Pasquinelli, A. E., Bettinger, J. C., Rougvie, A. E., Horvitz, H. R., and Ruvkun, G. 2000. The 21-nucleotide let-7 RNA regulates developmental timing in <i>Caenorhabditis elegans</i>. Nature 403: 901-906.</li>
        <li id="ul0001-0056" num="0289">Reinhart, B. J., Weinstein, E. G., Rhoades, M. W., Bartel, B., and Bartel, D. P. 2002. MicroRNAs in plants. Genes &amp; Dev. 16: 1616-1626.</li>
        <li id="ul0001-0057" num="0290">Rhoades, M. W., Reinhart, B. J., Lim, L. P., Burge, C. B., Bartel, B., and Bartel, D. P. 2002. Prediction of plant microRNA targets. Cell 110: 513-520.</li>
        <li id="ul0001-0058" num="0291">Roberts, B. E. and Paterson, B. M. 1973. Efficient translation of tobacco mosaic virus RNA and rabbit globin 9S RNA in a cell-free system from commercial wheat germ. Proc. Natl. Acad. Sci. 70: 2330-2334.</li>
        <li id="ul0001-0059" num="0292">Roignant, J.-Y., Carre, C., Mugat, B., Szymczak, D., Lepesant, J.-A., and Antoniewski, C. 2003. Absence of transitive and systemic pathways allows cell-specific and isoform-specific RNAi in <i>Drosophila</i>. RNA (In press).</li>
        <li id="ul0001-0060" num="0293">Ruvkun, G. 2001. Molecular biology. Glimpses of a tiny RNA world. Science 294: 797-799.</li>
        <li id="ul0001-0061" num="0294">Sambrook, J., Fritsch, E., and Maniatis, T. 1989. Molecular cloning: A laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.</li>
        <li id="ul0001-0062" num="0295">Schiebel, W., Pelissier, T., Riedel, L., Thalmeir, S., Schiebel, R., Kempe, D., Lottspeich, F., Sanger, H. L., and Wassenegger, M. 1998. Isolation of an RNA-directed RNA polymerase-specific cDNA clone from tomato. Plant Cell 10: 2087-2101.</li>
        <li id="ul0001-0063" num="0296">Schwarz, D. S., Hutvágner, G., Haley, B., and Zamore, P. D. 2002. Evidence that siRNAs function as guides, not primers, in the <i>Drosophila </i>and human RNAi pathways. Mol. Cell. 10: 537-548.</li>
        <li id="ul0001-0064" num="0297">Sijen, T., Fleenor, J., Simmer, F., Thijssen, K. L., Parrish, S., Timmons, L., Plasterk, R. H., and Fire, A. 2001. On the role of RNA amplification in dsRNA-triggered gene silencing. Cell 107: 465-476.</li>
        <li id="ul0001-0065" num="0298">Smardon, A., Spoerke, J., Stacey, S., Klein, M., Mackin, N., and Maine, E. 2000. EGO-1 is related to RNA-directed RNA polymerase and functions in germ-line development and RNA interference in <i>C. elegans</i>. Curr. Biol. 10: 169-178.</li>
        <li id="ul0001-0066" num="0299">Tabara, H., Sarkissian, M., Kelly, W. G., Fleenor, J., Grishok, A., Timmons, L., Fire, A., and Mello, C. C. 1999. The rde-1 gene, RNA interference, and transposon silencing in <i>C. elegans</i>. Cell 99: 123-132.</li>
        <li id="ul0001-0067" num="0300">Tijsterman, M., Ketting, R. F., Okihara, K. L., Sijen, T., and Plasterk, R. H. 2002. RNA helicase MUT-14-dependent gene silencing triggered in <i>C. elegans </i>by short antisense RNAs. Science 295: 694-697.</li>
        <li id="ul0001-0068" num="0301">Tuschl, T., Zamore, P. D., Lehmann, R., Bartel, D. P., and Sharp, P. A. 1999. Targeted mRNA degradation by double-stranded RNA in vitro. Genes &amp; Dev. 13: 3191-3197.</li>
        <li id="ul0001-0069" num="0302">Vaistij, F. E., Jones, L., and Baulcombe, D. C. 2002. Spreading of RNA targeting and DNA methylation in RNA silencing requires transcription of the target gene and a putative RNA-dependent RNA polymerase. Plant Cell 14: 857-867.</li>
        <li id="ul0001-0070" num="0303">Vaucheret, H., Beclin, C., and Fagard, M. 2001. Post-transcriptional gene silencing in plants. J. Cell Sci. 114: 3083-3091.</li>
        <li id="ul0001-0071" num="0304">Waterhouse, P. M., Wang, M. B., and Lough, T. 2001. Gene silencing as an adaptive defence against viruses. Nature 411: 834-842.</li>
        <li id="ul0001-0072" num="0305">Williams, R. W. and Rubin, G. M. 2002. ARGONAUTE1 is required for efficient RNA interference in <i>Drosophila </i>embryos. Proc. Natl. Acad. Sci. 99: 6889-6894.</li>
        <li id="ul0001-0073" num="0306">Zamore, P. D., Tuschl, T., Sharp, P. A., and Bartel, D. P. 2000. RNAi: Double-stranded RNA directs the ATP-dependent cleavage of mRNA at 21 to 23 nucleotide intervals. Cell 101: 25-33.</li>
      </ul> <heading>Equivalents</heading> <p num="p-0236">Those skilled in the art will recognize, or be able to ascertain using no more than routine experimentation, many equivalents to the specific embodiments of the invention described herein. Such equivalents are intended to be encompassed by the following claims.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6506559">US6506559</a></td><td class="patent-data-table-td patent-date-value">Dec 18, 1998</td><td class="patent-data-table-td patent-date-value">Jan 14, 2003</td><td class="patent-data-table-td ">Carnegie Institute Of Washington</td><td class="patent-data-table-td ">Molecular hybridization of complementary strands</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7459547">US7459547</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 2, 2004</td><td class="patent-data-table-td patent-date-value">Dec 2, 2008</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Determination competitive binder between Rna and target compound in plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20020086356">US20020086356</a></td><td class="patent-data-table-td patent-date-value">Mar 30, 2001</td><td class="patent-data-table-td patent-date-value">Jul 4, 2002</td><td class="patent-data-table-td ">Whitehead Institute For Biomedical Research</td><td class="patent-data-table-td ">Preferential nucleotide sequences for controlling preferential ribonucleic acids; for use in therapeutic genomics; for gene therapy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030108923">US20030108923</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2002</td><td class="patent-data-table-td patent-date-value">Jun 12, 2003</td><td class="patent-data-table-td ">Whitehead Institute For Biomedical Research</td><td class="patent-data-table-td ">RNA sequence-specific mediators of RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040002153">US20040002153</a></td><td class="patent-data-table-td patent-date-value">Jan 3, 2003</td><td class="patent-data-table-td patent-date-value">Jan 1, 2004</td><td class="patent-data-table-td ">Monia Brett P.</td><td class="patent-data-table-td ">Modulation of PTEN expression via oligomeric compounds</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040137471">US20040137471</a></td><td class="patent-data-table-td patent-date-value">Sep 18, 2003</td><td class="patent-data-table-td patent-date-value">Jul 15, 2004</td><td class="patent-data-table-td ">Timothy Vickers</td><td class="patent-data-table-td ">selecting single-stranded oligomeric compounds for inhibiting RNA expression; RNA interference; modifications include 2&#39;-O alkyl, 2&#39;-O-alkoxyalkyl, 2&#39;-alkoxy, 2&#39;-dimethylaminooxyethoxy, 2&#39;-dimethylaminoethoxyethoxy for example</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040192629">US20040192629</a></td><td class="patent-data-table-td patent-date-value">Nov 4, 2003</td><td class="patent-data-table-td patent-date-value">Sep 30, 2004</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Selectively inhibit the mutant while retaining wild-type gene expression to treat diseases caused by dominant mutants such as amyotrophic lateral sclerosis, Huntington&#39;s disease, Alzheimer&#39;s disease, and Parkinson&#39;s disease; administering to the cell an siRNA specific for the target allele</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040203145">US20040203145</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 2003</td><td class="patent-data-table-td patent-date-value">Oct 14, 2004</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Compositions for RNA interference and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040224328">US20040224328</a></td><td class="patent-data-table-td patent-date-value">Jan 12, 2004</td><td class="patent-data-table-td patent-date-value">Nov 11, 2004</td><td class="patent-data-table-td ">Hans Prydz</td><td class="patent-data-table-td ">Identifying regulatory binding sites of short interfering RNA; antisense agents and gene expression inhibition</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040229266">US20040229266</a></td><td class="patent-data-table-td patent-date-value">Apr 27, 2004</td><td class="patent-data-table-td patent-date-value">Nov 18, 2004</td><td class="patent-data-table-td ">Max-Planck-Gesellschaft Zur Forderung Der Wissenschaften E.V.</td><td class="patent-data-table-td ">RNA interference mediating small RNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040259247">US20040259247</a></td><td class="patent-data-table-td patent-date-value">Nov 29, 2001</td><td class="patent-data-table-td patent-date-value">Dec 23, 2004</td><td class="patent-data-table-td ">Thomas Tuschl</td><td class="patent-data-table-td ">short interfering RNAs for use treating diseases, in protein separation or purification, in drug screening; cell with target gene-specific knockout phenotype; double stranded RNA; RNA fragments are the sequence-specific mediators of RNAi (RNA interference)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040259248">US20040259248</a></td><td class="patent-data-table-td patent-date-value">Apr 27, 2004</td><td class="patent-data-table-td patent-date-value">Dec 23, 2004</td><td class="patent-data-table-td ">Max-Planck-Gesellschaft Zur Forderung Der Wissenschaften E.V.</td><td class="patent-data-table-td ">RNA interference mediating small RNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050026278">US20050026278</a></td><td class="patent-data-table-td patent-date-value">Apr 27, 2004</td><td class="patent-data-table-td patent-date-value">Feb 3, 2005</td><td class="patent-data-table-td ">Max-Planck-Gesellschaft Zur Forderung Der Wissenschaften E.V.</td><td class="patent-data-table-td ">RNA interference mediating small RNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050037988">US20050037988</a></td><td class="patent-data-table-td patent-date-value">Jun 2, 2004</td><td class="patent-data-table-td patent-date-value">Feb 17, 2005</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Determination competitive binder between Rna and target compound in plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050059005">US20050059005</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2002</td><td class="patent-data-table-td patent-date-value">Mar 17, 2005</td><td class="patent-data-table-td ">Thomas Tuschl</td><td class="patent-data-table-td ">Microrna molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050181382">US20050181382</a></td><td class="patent-data-table-td patent-date-value">Jun 2, 2004</td><td class="patent-data-table-td patent-date-value">Aug 18, 2005</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Mediates silencing of a target gene; lessening the base pair strength between the 5&#39; end of the first strand and the 3&#39; end of a second strand of the duplex as compared to the base pair strength</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050182005">US20050182005</a></td><td class="patent-data-table-td patent-date-value">May 13, 2004</td><td class="patent-data-table-td patent-date-value">Aug 18, 2005</td><td class="patent-data-table-td ">Tuschl Thomas H.</td><td class="patent-data-table-td ">Anti-microRNA oligonucleotide molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050186586">US20050186586</a></td><td class="patent-data-table-td patent-date-value">Aug 4, 2004</td><td class="patent-data-table-td patent-date-value">Aug 25, 2005</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Methods and compositions for enhancing the efficacy and specificity of RNAi</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050227256">US20050227256</a></td><td class="patent-data-table-td patent-date-value">Nov 26, 2004</td><td class="patent-data-table-td patent-date-value">Oct 13, 2005</td><td class="patent-data-table-td ">Gyorgy Hutvagner</td><td class="patent-data-table-td ">Sequence-specific inhibition of small RNA function</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050234007">US20050234007</a></td><td class="patent-data-table-td patent-date-value">Jun 2, 2005</td><td class="patent-data-table-td patent-date-value">Oct 20, 2005</td><td class="patent-data-table-td ">Max-Planck-Gesellschaft Zur Forderung Der Wissenschaften E.V.</td><td class="patent-data-table-td ">RNA interference mediating small RNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050246794">US20050246794</a></td><td class="patent-data-table-td patent-date-value">Apr 7, 2005</td><td class="patent-data-table-td patent-date-value">Nov 3, 2005</td><td class="patent-data-table-td ">Dharmacon Inc.</td><td class="patent-data-table-td ">Functional and hyperfunctional siRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050256072">US20050256072</a></td><td class="patent-data-table-td patent-date-value">Feb 9, 2005</td><td class="patent-data-table-td patent-date-value">Nov 17, 2005</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Dual functional oligonucleotides for use in repressing mutant gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20060009402">US20060009402</a></td><td class="patent-data-table-td patent-date-value">Jul 12, 2002</td><td class="patent-data-table-td patent-date-value">Jan 12, 2006</td><td class="patent-data-table-td ">Zamore Phillip D</td><td class="patent-data-table-td ">Engineered RNA precursors encoded in plasmids and/or other vectors; simple, low cost technique utilizing DNA molecule production; host cells, transgenes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20060134787">US20060134787</a></td><td class="patent-data-table-td patent-date-value">Dec 22, 2004</td><td class="patent-data-table-td patent-date-value">Jun 22, 2006</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Methods and compositions for enhancing the efficacy and specificity of single and double blunt-ended siRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20060166910">US20060166910</a></td><td class="patent-data-table-td patent-date-value">Jul 10, 2003</td><td class="patent-data-table-td patent-date-value">Jul 27, 2006</td><td class="patent-data-table-td ">Max-Planck-Gesellschaft Zur Foerderung Der Wissenschaften E.V.</td><td class="patent-data-table-td ">Rna-interference by single-stranded rna molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20060212950">US20060212950</a></td><td class="patent-data-table-td patent-date-value">Jan 15, 2004</td><td class="patent-data-table-td patent-date-value">Sep 21, 2006</td><td class="patent-data-table-td ">Max-Planck-Gesellschaft Zur Forderung Der Wisseneschaften E.V.</td><td class="patent-data-table-td ">Inducible small rna expression constructs for targeted gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070003960">US20070003960</a></td><td class="patent-data-table-td patent-date-value">Jun 26, 2006</td><td class="patent-data-table-td patent-date-value">Jan 4, 2007</td><td class="patent-data-table-td ">Whitehead Institute For Biomedical Research</td><td class="patent-data-table-td ">RNA sequence-specific mediators of RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070003961">US20070003961</a></td><td class="patent-data-table-td patent-date-value">Jun 26, 2006</td><td class="patent-data-table-td patent-date-value">Jan 4, 2007</td><td class="patent-data-table-td ">Whitehead Institute For Biomedical Research</td><td class="patent-data-table-td ">RNA sequence-specific mediators of RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070003962">US20070003962</a></td><td class="patent-data-table-td patent-date-value">Jun 26, 2006</td><td class="patent-data-table-td patent-date-value">Jan 4, 2007</td><td class="patent-data-table-td ">Whitehead Institute For Biomedical Research</td><td class="patent-data-table-td ">Used in targeting specific mRNAs for degradation in mammalian cells where use of long dsRNAs to elicit RNAi is usually not practical, presumably because of the deleterious effects of the interferon response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070003963">US20070003963</a></td><td class="patent-data-table-td patent-date-value">Jun 26, 2006</td><td class="patent-data-table-td patent-date-value">Jan 4, 2007</td><td class="patent-data-table-td ">Whitehead Institute For Biomedical Research</td><td class="patent-data-table-td ">Used in targeting specific mRNAs for degradation in mammalian cells where use of long dsRNAs to elicit RNAi is usually not practical, presumably because of the deleterious effects of the interferon response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070031844">US20070031844</a></td><td class="patent-data-table-td patent-date-value">Nov 14, 2003</td><td class="patent-data-table-td patent-date-value">Feb 8, 2007</td><td class="patent-data-table-td ">Anastasia Khvorova</td><td class="patent-data-table-td ">Functional and hyperfunctional siRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070039072">US20070039072</a></td><td class="patent-data-table-td patent-date-value">Mar 29, 2005</td><td class="patent-data-table-td patent-date-value">Feb 15, 2007</td><td class="patent-data-table-td ">Dharmacon Inc.</td><td class="patent-data-table-td ">Functional and hyperfunctional siRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070093445">US20070093445</a></td><td class="patent-data-table-td patent-date-value">Dec 6, 2006</td><td class="patent-data-table-td patent-date-value">Apr 26, 2007</td><td class="patent-data-table-td ">Max-Planck-Gesellschaft Zur Forderung Der Wissenschaften E. V.</td><td class="patent-data-table-td ">RNA interference mediating small RNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070207974">US20070207974</a></td><td class="patent-data-table-td patent-date-value">Mar 30, 2005</td><td class="patent-data-table-td patent-date-value">Sep 6, 2007</td><td class="patent-data-table-td ">Dharmacon Inc.</td><td class="patent-data-table-td ">Short interference RNA duplexes, when introduced into mammalian cells in culture, perform sequence-specific inhibition of target mRNA without inducing an interferon response; specific gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001075164A2?cl=en">WO2001075164A2</a></td><td class="patent-data-table-td patent-date-value">Mar 30, 2001</td><td class="patent-data-table-td patent-date-value">Oct 11, 2001</td><td class="patent-data-table-td ">Whitehead Biomedical Inst</td><td class="patent-data-table-td ">Rna sequence-specific mediators of rna interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002044321A2?cl=en">WO2002044321A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 29, 2001</td><td class="patent-data-table-td patent-date-value">Jun 6, 2002</td><td class="patent-data-table-td ">Max Planck Gesellschaft</td><td class="patent-data-table-td ">Rna interference mediating small rna molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2003020931A2?cl=en">WO2003020931A2</a></td><td class="patent-data-table-td patent-date-value">Aug 29, 2002</td><td class="patent-data-table-td patent-date-value">Mar 13, 2003</td><td class="patent-data-table-td ">Galapagos Genomics N V</td><td class="patent-data-table-td ">Sirna knockout assay method and constructs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004027030A2?cl=en">WO2004027030A2</a></td><td class="patent-data-table-td patent-date-value">Sep 18, 2003</td><td class="patent-data-table-td patent-date-value">Apr 1, 2004</td><td class="patent-data-table-td ">Brenda F Baker</td><td class="patent-data-table-td ">Efficient reduction of target rna’s by single- and double-stranded oligomeric compounds</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004045543A2?cl=en">WO2004045543A2</a></td><td class="patent-data-table-td patent-date-value">Nov 14, 2003</td><td class="patent-data-table-td patent-date-value">Jun 3, 2004</td><td class="patent-data-table-td ">Dharmacon Inc</td><td class="patent-data-table-td ">Functional and hyperfunctional sirna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004063375A1?cl=en">WO2004063375A1</a></td><td class="patent-data-table-td patent-date-value">Jan 14, 2004</td><td class="patent-data-table-td patent-date-value">Jul 29, 2004</td><td class="patent-data-table-td ">Mohammed Amarzguioui</td><td class="patent-data-table-td ">OPTIMIZING siRNA BY RNAi ANTISENSE</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005001043A2?cl=en">WO2005001043A2</a></td><td class="patent-data-table-td patent-date-value">Jun 2, 2004</td><td class="patent-data-table-td patent-date-value">Jan 6, 2005</td><td class="patent-data-table-td ">Univ Massachusetts</td><td class="patent-data-table-td ">METHODS AND COMPOSITIONS FOR ENHANCING THE EFFICACY AND SPECIFICITY OF FNAi</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005062937A2?cl=en">WO2005062937A2</a></td><td class="patent-data-table-td patent-date-value">Dec 22, 2004</td><td class="patent-data-table-td patent-date-value">Jul 14, 2005</td><td class="patent-data-table-td ">Univ Massachusetts</td><td class="patent-data-table-td ">Methods and compositions for enhancing the efficacy and specificity of single and double blunt-ended sirna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2006015389A2?cl=en">WO2006015389A2</a></td><td class="patent-data-table-td patent-date-value">Aug 4, 2005</td><td class="patent-data-table-td patent-date-value">Feb 9, 2006</td><td class="patent-data-table-td ">Univ Massachusetts</td><td class="patent-data-table-td ">Methods and compositions for enhancing the efficacy and specificity of rna silencing</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Amarzguioui, Mohammed et al., "<a href='http://scholar.google.com/scholar?q="Tolerance+for+mutations+and+chemical+modifications+in+a+siRNA%2C"'>Tolerance for mutations and chemical modifications in a siRNA,</a>" Nucleic Acids Research, vol. 31(2):589-595 (2003).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ameres et al., "<a href='http://scholar.google.com/scholar?q="Molecular+basis+for+target+RNA+recognition+and+cleavage+by+human+RISC%2C"'>Molecular basis for target RNA recognition and cleavage by human RISC,</a>" Cell, 130: 101-112 (2007).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bartel, David P., "<a href='http://scholar.google.com/scholar?q="MicroRNAs%3A+Genomics%2C+Biogenesis%2C+Mechanism%2C+and+Function%2C"'>MicroRNAs: Genomics, Biogenesis, Mechanism, and Function,</a>" Cell, vol. 116:281-297 (2004).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Beclin, Christophe et al., "<a href='http://scholar.google.com/scholar?q="A+Branched+Pathway+for+Transgene-Induced+RNA+Silencing+in+Plants%2C"'>A Branched Pathway for Transgene-Induced RNA Silencing in Plants,</a>" Current Biology, vol. 12:684-688, (2002).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boden, D. et al, "<a href='http://scholar.google.com/scholar?q="Efficient+gene+transfer+of+HIV-1-specific+short+hairpin+RNA+into+human+lymphocytic+cells+using+recombinant+adeno-associated+virus+vectors%2C"'>Efficient gene transfer of HIV-1-specific short hairpin RNA into human lymphocytic cells using recombinant adeno-associated virus vectors,</a>" Mol. Ther., vol. 9(3):396-402 (2004).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boden, Daniel et al, "<a href='http://scholar.google.com/scholar?q="Enhanced+gene+silencing+of+HIV-1+specific+siRNA+using+microRNA+designed+hairpins%2C"'>Enhanced gene silencing of HIV-1 specific siRNA using microRNA designed hairpins,</a>" Nucleic Acids Research, vol. 32(3):1154-1158 (2004).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bonnet, Eric et al, "<a href='http://scholar.google.com/scholar?q="Evidence+that+microRNA+precursors%2C+unlike+other+non-coding+RNAs%2C+have+lower+folding+free+energies+than+random+sequences%2C"'>Evidence that microRNA precursors, unlike other non-coding RNAs, have lower folding free energies than random sequences,</a>" Bioinformatics, vol. 20(17):2911-2917 (2004).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chi, Jen-Tsan et al, "<a href='http://scholar.google.com/scholar?q="Genomewide+view+of+gene+silencing+by+small+interfering+RNAs%2C"'>Genomewide view of gene silencing by small interfering RNAs,</a>" PNAS, vol. 100(11):6343-6346 (2003).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chiu, Ya-Lin et al, "<a href='http://scholar.google.com/scholar?q="RNAi+in+Human+Cells%3A+Basic+Structural+and+Functional+Features+of+Small+Interfering+RNA%2C"'>RNAi in Human Cells: Basic Structural and Functional Features of Small Interfering RNA,</a>" Molecular Cell, vol. 10:549-561 (2002).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chiu, Ya-Lin et al, "<a href='http://scholar.google.com/scholar?q="siRNA+function+in+RNAi%3A+A+chemical+modification+analysis%2C"'>siRNA function in RNAi: A chemical modification analysis,</a>" RNA, vol. 9:1034-1048 (2003).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cogoni, Carlo et al., "<a href='http://scholar.google.com/scholar?q="Gene+silencing+in+Neurospora+crassa+requires+a+protein+homologous+to+RNA-dependent+RNA+polymerase%2C"'>Gene silencing in Neurospora crassa requires a protein homologous to RNA-dependent RNA polymerase,</a>" Nature . vol. 399:166-169. (1999).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dalmay, Talmas et al., "<a href='http://scholar.google.com/scholar?q="An+RNA-Dependent+RNA+Polymerase+Gene+in+Arabidopsis+Is+Required+for+Posttranscriptional+Gene+Silencing+Mediated+by+a+Transgene+but+Not+by+a+Virus%2C"'>An RNA-Dependent RNA Polymerase Gene in Arabidopsis Is Required for Posttranscriptional Gene Silencing Mediated by a Transgene but Not by a Virus,</a>" Cell. vol. 101:543-553, (2000).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dharmacon RNA Technologies: On-Target siRNA, Company Brochure, 2003.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dharmacon RNA Technologies: Products for RNA Interference, Company Brochure, 2003.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Doench, John G. et al, "<a href='http://scholar.google.com/scholar?q="siRNAs+can+function+as+miRNAs%2C"'>siRNAs can function as miRNAs,</a>" Genes &amp; Development, vol. 17:438-442 (2003).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Du, Quan et al, "<a href='http://scholar.google.com/scholar?q="A+sytematic+analysis+of+the+silencing+effects+of+an+active+siRNA+at+all+single-nucleotide+mismatched+target+sites%2C"'>A sytematic analysis of the silencing effects of an active siRNA at all single-nucleotide mismatched target sites,</a>" Nucleic Acids Research, vol. 33(5):1671-1677 (2005).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Elbashir et al (EMBO J. 20(23): 6877-6888, 2001).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Elbashir et al (Nature 411: 494-498, 2001).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Elbashir, Sayda M. et al, "<a href='http://scholar.google.com/scholar?q="Analysis+of+gene+function+in+somatic+mammalian+cells+using+small+interfering+RNAs%2C"'>Analysis of gene function in somatic mammalian cells using small interfering RNAs,</a>" Methods, vol. 26:199-213 (2002).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Elbashir, Sayda M. et al, "<a href='http://scholar.google.com/scholar?q="Functional+anatomy+of+siRNAs+for+mediating+efficient+RNAi+in+Drosophila+melanogaster+embryo+lysate%2C"'>Functional anatomy of siRNAs for mediating efficient RNAi in Drosophila melanogaster embryo lysate,</a>" The EMBO Journal, vol. 20(23):6877-6888 (2001).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Elbashir, Sayda M. et al, "<a href='http://scholar.google.com/scholar?q="RNA+interference+is+mediated+by+21-+and+22-nucleotide+RNAs%2C"'>RNA interference is mediated by 21- and 22-nucleotide RNAs,</a>" Genes &amp; Development, vol. 15:188-200 (2001).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gong, Delquin et al, "<a href='http://scholar.google.com/scholar?q="Picking+a+winner%3A+new+mechanistic+insights+into+the+design+of+effective+siRNAs%2C"'>Picking a winner: new mechanistic insights into the design of effective siRNAs,</a>" Trends in Biotechnology, vol. 22(9):451-454 (2004).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Haley et al (Nature Struct. &amp; Mol. Biol 11(7): 599-606, 2004.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Haley, Benjamin et al, "<a href='http://scholar.google.com/scholar?q="In+vitro+analysis+of+RNA+interference+in+Drosophila+melanogaster%2C"'>In vitro analysis of RNA interference in Drosophila melanogaster,</a>" Methods, vol. 30:330-336 (2003).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Haley, Benjamin et al., "<a href='http://scholar.google.com/scholar?q="Kinetic+analysis+of+the+RNAi+enzyme+complex%2C"'>Kinetic analysis of the RNAi enzyme complex,</a>" Nature Structural &amp; Molecular Biology, vol. 11(7):599-606 (2004).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hamada, Makiko et al, "<a href='http://scholar.google.com/scholar?q="Effects+on+RNA+Interference+in+Gene+Expression+%28RNAi%29+in+Cultured+Mammalian+Cells+of+Mismatches+and+the+Introduction+of+Chemical+Modifications+at+the+3%27-Ends+of+siRNAs%2C"'>Effects on RNA Interference in Gene Expression (RNAi) in Cultured Mammalian Cells of Mismatches and the Introduction of Chemical Modifications at the 3'-Ends of siRNAs,</a>" Antisense and Nucleic Acid Drug Development, vol. 12:301-309 (2002).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hamada, Makiko et al, "<a href='http://scholar.google.com/scholar?q="Effects+on+RNA+Interference+in+Gene+Expression+%28RNAi%29+in+Cultured+Mammalian+Cells+of+Mismatches+and+the+Introduction+of+Chemical+Modifications+at+the+3%E2%80%B2-Ends+of+siRNAs%2C"'>Effects on RNA Interference in Gene Expression (RNAi) in Cultured Mammalian Cells of Mismatches and the Introduction of Chemical Modifications at the 3′-Ends of siRNAs,</a>" Antisense and Nucleic Acid Drug Development, vol. 12:301-309 (2002).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holen, Torgeir et al, "<a href='http://scholar.google.com/scholar?q="Positional+effects+of+short+interfering+RNAs+targeting+the+human+coagulation+trigger+Tissue+Factor%2C"'>Positional effects of short interfering RNAs targeting the human coagulation trigger Tissue Factor,</a>" Nucleic Acids Research, vol. 30(8):1757-1766 (2002).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holen, Torgeir et al, "<a href='http://scholar.google.com/scholar?q="Similar+behaviour+of+single-strand+and+double-strand+siRNAs+suggests+they+act+through+a+common+RNAi+pathway%2C"'>Similar behaviour of single-strand and double-strand siRNAs suggests they act through a common RNAi pathway,</a>" Nucleic Acids Research, vol. 31(9):2401-2407 (2003).</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holen, Torgeir et al., "<a href='http://scholar.google.com/scholar?q="Tolerated+wobble+mutations+in+siRNAs+decrease+specificity%2C+but+can+enhance+activity+in+vivo%2C"'>Tolerated wobble mutations in siRNAs decrease specificity, but can enhance activity in vivo,</a>" Nucleic Acids Research, vol. 33(15):4707-4710 (2005).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hutvágner, György et al, "<a href='http://scholar.google.com/scholar?q="A+microRNA+in+a+Multiple-Turnover+RNAi+Enzyme+Complex%2C"'>A microRNA in a Multiple-Turnover RNAi Enzyme Complex,</a>" Science, vol. 297:2056-2060 (2002).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">International Preliminary Report on Patentability for Application No. PCT/US2004/017130, dated Dec. 8, 2005.</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Invitation to Pay Additional Fees Application No. PCT/USO4/17130, dated Apr. 1, 2005.</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jackson, Aimee L. et al, "<a href='http://scholar.google.com/scholar?q="Expression+profiling+reveals+off-target+gene+regulation+by+RNAi%2C"'>Expression profiling reveals off-target gene regulation by RNAi,</a>" Nature Biotechnology, vol. 21(6):635-638 (2003).</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ketting, René F. et al, "<a href='http://scholar.google.com/scholar?q="Dicer+functions+in+RNA+interference+and+in+synthesis+of+small+RNA+involved+in+development+timing+in+C.+elegans%2C"'>Dicer functions in RNA interference and in synthesis of small RNA involved in development timing in C. elegans,</a>" Genes &amp; Development, vol. 15:2654-2659 (2001).</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Khvorova, Anastasia et al, "<a href='http://scholar.google.com/scholar?q="Functional+siRNAs+and+miRNAs+Exhibit+Strand+Bias%2C"'>Functional siRNAs and miRNAs Exhibit Strand Bias,</a>" Cell, vol. 115:209-216 (2003).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Krol, Jacek et al, "<a href='http://scholar.google.com/scholar?q="Structural+Features+of+MicroRNA+%28miRNA%29+Precursors+and+Their+Relevance+to+miRNA+Biogenesis+and+Small+Interfering+RNA%2FShort+Hairpin+RNA+Design%2C"'>Structural Features of MicroRNA (miRNA) Precursors and Their Relevance to miRNA Biogenesis and Small Interfering RNA/Short Hairpin RNA Design,</a>" The Journal of Biological Chemistry, vol. 279(40):42230-42239 (2004).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lagos-Quintana, Mariana et al., "<a href='http://scholar.google.com/scholar?q="Identification+of+Novel+Genes+Coding+for+Small+Expressed+RNAs%2C"'>Identification of Novel Genes Coding for Small Expressed RNAs,</a>" Science.. vol. 294:853-858, (2001).</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lagos-Quintana, Mariana et al., "<a href='http://scholar.google.com/scholar?q="Identification+of+Tissue-Specific+MicroRNAs+from+Mouse%2C"'>Identification of Tissue-Specific MicroRNAs from Mouse,</a>" Current Biology. vol. 12:735-739, (2002).</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lai, Eric C., "<a href='http://scholar.google.com/scholar?q="Micro+RNAs+are+complementary+to+3%27+UTR+sequence+motifs+that+mediate+negative+post-transcriptional+regulation%2C"'>Micro RNAs are complementary to 3' UTR sequence motifs that mediate negative post-transcriptional regulation,</a>" Nature Genetics, vol. 30:363-364 (2002).</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lai, Eric C., "<a href='http://scholar.google.com/scholar?q="Micro+RNAs+are+complementary+to+3%E2%80%B2+UTR+sequence+motifs+that+mediate+negative+post-transcriptional+regulation%2C"'>Micro RNAs are complementary to 3′ UTR sequence motifs that mediate negative post-transcriptional regulation,</a>" Nature Genetics, vol. 30:363-364 (2002).</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lee, Rosalind C. et al., "<a href='http://scholar.google.com/scholar?q="The+C.+elegans+Heterochronic+Gene+lin-4+Encodes+Small+RNAs+with+Antisense+Complementarity+to+lin-14%2C"'>The C. elegans Heterochronic Gene lin-4 Encodes Small RNAs with Antisense Complementarity to lin-14,</a>" Cell. vol. 75:843-854, (1993).</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Li, Wan Xiang et al., "<a href='http://scholar.google.com/scholar?q="Viral+suppressors+of+RNA+silencing%2C"'>Viral suppressors of RNA silencing,</a>" Current Opinion in Biotechnology. vol. 12:150-154, (2001).</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lim, Lee P. et al, "<a href='http://scholar.google.com/scholar?q="Microarray+analysis+shows+that+some+microRNAs+downregulate+large+numbers+of+target+mRNAs%2C"'>Microarray analysis shows that some microRNAs downregulate large numbers of target mRNAs,</a>" Nature, vol. 433:769-773 (2005).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Llave, Cesar et al., "<a href='http://scholar.google.com/scholar?q="Cleavage+of+Scarecrow-like+mRNA+Targets+Directed+by+a+Class+of+Arabidopsis+miRNA%2C"'>Cleavage of Scarecrow-like mRNA Targets Directed by a Class of Arabidopsis miRNA,</a>" Science . vol. 297:2053-2056, (2002).</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mallory, Allison C. et al, "<a href='http://scholar.google.com/scholar?q="MicroRNA+control+of+Phabulosa+in+leaf+development%3A+importance+of+pairing+to+the+microRNA+5%27+region%2C"'>MicroRNA control of Phabulosa in leaf development: importance of pairing to the microRNA 5' region,</a>" The EMBO Journal, vol. 23:3356-3364 (2004).</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mallory, Allison C. et al, "<a href='http://scholar.google.com/scholar?q="MicroRNA+control+of+Phabulosa+in+leaf+development%3A+importance+of+pairing+to+the+microRNA+5%E2%80%B2+region%2C"'>MicroRNA control of Phabulosa in leaf development: importance of pairing to the microRNA 5′ region,</a>" The EMBO Journal, vol. 23:3356-3364 (2004).</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Martens, Henrik et al., "<a href='http://scholar.google.com/scholar?q="RNAi+in+Dictyostelium+%3A+The+Role+of+RNA-directed+RNA+Polymerases+and+Double-stranded+RNase%2C"'>RNAi in Dictyostelium : The Role of RNA-directed RNA Polymerases and Double-stranded RNase,</a>" Molecular Biology of the Cell . vol. 13:445-453, (2002).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Martinez, Javier et al, "<a href='http://scholar.google.com/scholar?q="Single-Stranded+Antisense+siRNAs+Guide+Target+RNA+Cleavage+in+RNAi%2C"'>Single-Stranded Antisense siRNAs Guide Target RNA Cleavage in RNAi,</a>" Cell, vol. 110:563-574 (2002).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Martinez, Luis Alfonso et al, "<a href='http://scholar.google.com/scholar?q="Synthetic+small+inhibiting+RNAs%3A+Efficient+tolls+to+inactivate+oncogenic+mutations+and+restore+p53+pathways%2C"'>Synthetic small inhibiting RNAs: Efficient tolls to inactivate oncogenic mutations and restore p53 pathways,</a>" PNAS, vol. 99(23):14849-14854 (2002).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McConnell, Jane R. et al., "<a href='http://scholar.google.com/scholar?q="Role+of+Phabulosa+and+Phavoluta+in+determining+radial+patterning+in+shoots%2C"'>Role of Phabulosa and Phavoluta in determining radial patterning in shoots,</a>" Nature . vol. 411:709-713, (2001).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McManus, Michael T. et al, "<a href='http://scholar.google.com/scholar?q="Gene+silencing+using+micro-RNA+designed+hairpins%2C"'>Gene silencing using micro-RNA designed hairpins,</a>" RNA, Vol. 8:842-850 (2002).</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Meister, Gunter et al, "<a href='http://scholar.google.com/scholar?q="Sequence-specific+inhibition+of+microRNA-+and+siRNA-induced+RNA+silencing%2C"'>Sequence-specific inhibition of microRNA- and siRNA-induced RNA silencing,</a>" RNA, vol. 10:544-550 (2004).</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mourelatos, Zissimos et al, "<a href='http://scholar.google.com/scholar?q="miRNPs%3A+a+novel+class+of+ribonucleoproteins+containing+numerous+microRNAs%2C"'>miRNPs: a novel class of ribonucleoproteins containing numerous microRNAs,</a>" Genes &amp; Development, vol. 16:720-728 (2002).</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mourrain, Philippe et al., "<a href='http://scholar.google.com/scholar?q="Arabidopsis+SGS2+and+SGS3+Genes+Are+Required+for+Posttranscriptional+Gnee+Silencing+and+Natural+Virus+Resistance%2C"'>Arabidopsis SGS2 and SGS3 Genes Are Required for Posttranscriptional Gnee Silencing and Natural Virus Resistance,</a>" Cell . vol. 101:533-542, (2000).</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Murchison, E.P. et al, "<a href='http://scholar.google.com/scholar?q="miRNAs+on+the+move%3A+miRNA+biogenesis+and+the+RNAi+machinery%2C"'>miRNAs on the move: miRNA biogenesis and the RNAi machinery,</a>" Curr. Opin. Cell. Biol., vol. 16(3):223-229 (2004).</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nykänen, Antti et al, "<a href='http://scholar.google.com/scholar?q="ATP+Requirements+and+Small+Interfering+RNA+Structure+in+the+RNA+Interference+Pathway%2C"'>ATP Requirements and Small Interfering RNA Structure in the RNA Interference Pathway,</a>" Cell, vol. 107:309-321 (2001).</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ohnishi, Yusuke et al, "<a href='http://scholar.google.com/scholar?q="Influence+of+assembly+of+siRNA+elements+into+RNA-induced+silencing+complex+by+fork-siRNA+duplex+carrying+nucleotide+mismatches+at+the+3%27-+or+5%27-end+of+the+sense-stranded+siRNA+element%2C"'>Influence of assembly of siRNA elements into RNA-induced silencing complex by fork-siRNA duplex carrying nucleotide mismatches at the 3'- or 5'-end of the sense-stranded siRNA element,</a>" Biochemical and Biophysical Research Communications, vol. 329:516-521 (2005).</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ohnishi, Yusuke et al, "<a href='http://scholar.google.com/scholar?q="Influence+of+assembly+of+siRNA+elements+into+RNA-induced+silencing+complex+by+fork-siRNA+duplex+carrying+nucleotide+mismatches+at+the+3%E2%80%B2-+or+5%E2%80%B2-end+of+the+sense-stranded+siRNA+element%2C"'>Influence of assembly of siRNA elements into RNA-induced silencing complex by fork-siRNA duplex carrying nucleotide mismatches at the 3′- or 5′-end of the sense-stranded siRNA element,</a>" Biochemical and Biophysical Research Communications, vol. 329:516-521 (2005).</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pal-Bhadra, Manika et al., "<a href='http://scholar.google.com/scholar?q="RNAi+Related+Mechanisms+Affect+both+Transcriptional+and+Posttranscriptional+Transgene+Silencing+in+Drosophila%2C"'>RNAi Related Mechanisms Affect both Transcriptional and Posttranscriptional Transgene Silencing in Drosophila,</a>" Molecular Cell. vol. 9:315-327, (2002).</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Park, Wonkeun et al., "<a href='http://scholar.google.com/scholar?q="Carpel+Factory%2C+a+Dicer+Homolog%2C+and+HEN1%2C+a+Novel+Protein%2C+Act+in+microRNa+Metabolism+in+Arabidopsis+thaliana%2C"'>Carpel Factory, a Dicer Homolog, and HEN1, a Novel Protein, Act in microRNa Metabolism in Arabidopsis thaliana,</a>" Current Biology, vol. 12:1484-1495, (2002).</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Patzel, Volker et al, "<a href='http://scholar.google.com/scholar?q="Design+of+siRNAs+producing+unstructured+guide-RNAs+results+in+improved+RNA+interference+efficiency%2C"'>Design of siRNAs producing unstructured guide-RNAs results in improved RNA interference efficiency,</a>" Nature Biotechnology, vol. 23(11):1440-1444 (2005).</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pusch, Oliver et al, "<a href='http://scholar.google.com/scholar?q="Nucleotide+sequence+homology+requirements+of+HIV-1-specific+short+hairpin+RNA%2C"'>Nucleotide sequence homology requirements of HIV-1-specific short hairpin RNA,</a>" Nucleic Acids Research, vol. 31(22):6444-6449 (2003).</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Reinhart, Brenda J. et al., "<a href='http://scholar.google.com/scholar?q="MicroRNAs+in+plants%2C"'>MicroRNAs in plants,</a>" Genes &amp; Development, vol. 16:1616-1626 (2002).</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Reynolds, Angela et al, "<a href='http://scholar.google.com/scholar?q="Rational+siRNA+design+for+RNA+interference%2C"'>Rational siRNA design for RNA interference,</a>" Nature Biotechnology, vol. 22(3):326-330 (2004).</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rhoades, Matthew W. et al., "<a href='http://scholar.google.com/scholar?q="Prediction+of+Plant+MicroRNA+Targets%2C"'>Prediction of Plant MicroRNA Targets,</a>" Cell, vol. 110:513-520 (2002).</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ruvkun, Gary, "<a href='http://scholar.google.com/scholar?q="Glimpses+of+a+Tiny+RNA+World%2C"'>Glimpses of a Tiny RNA World,</a>" Science, vol. 294:797-799, (2001).</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Saxena, Sandeep et al, "<a href='http://scholar.google.com/scholar?q="Small+RNAs+with+Imperfect+Match+to+Endogenous+mRNA+Repress+Translation%2C"'>Small RNAs with Imperfect Match to Endogenous mRNA Repress Translation,</a>" The Journal of Biological Chemistry, vol. 278(45):44312-44319 (2003).</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schubert, Steffen et al., "<a href='http://scholar.google.com/scholar?q="Local+RNA+Target+Structure+Influence+siRNA+Efficacy%3A+Systematic+Analysis+of+Intentionally+Designed+Binding+Regions%2C"'>Local RNA Target Structure Influence siRNA Efficacy: Systematic Analysis of Intentionally Designed Binding Regions,</a>" J. Mol. Biol., vol. 348:883-893 (2005).</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schwarz, Dianne S. et al, "<a href='http://scholar.google.com/scholar?q="Asymmetry+in+the+Assembly+of+the+RNAi+Enzyme+Complex%2C"'>Asymmetry in the Assembly of the RNAi Enzyme Complex,</a>" Cell, vol. 115:199-208 (2003).</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schwarz, Dianne S. et al, "<a href='http://scholar.google.com/scholar?q="The+RNA-Induced+Silencing+Complex+Is+a+Mg2%2B-Dependent+Endonuclease%2C"'>The RNA-Induced Silencing Complex Is a Mg2+-Dependent Endonuclease,</a>" Current Biology, vol. 14:787-791 (2004).</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schwarz, Dianne S. et al, "<a href='http://scholar.google.com/scholar?q="Why+do+miRNAs+live+in+the+miRNP%3F"'>Why do miRNAs live in the miRNP?</a>" Genes &amp; Development, vol. 16:1025-1031 (2002).</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schwarz, Dianne S. et al., "<a href='http://scholar.google.com/scholar?q="Designing+siRNA+That+Distinguish+between+Genes+That+Differ+by+a+Single+Nucleotide%2C"'>Designing siRNA That Distinguish between Genes That Differ by a Single Nucleotide,</a>" PLoS Genetics, vol. 2(9):1-12 (2006).</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Semizarov, Dimitri et al, "<a href='http://scholar.google.com/scholar?q="Specificity+of+short+interfering+RNA+determined+through+gene+expression+signatures%2C"'>Specificity of short interfering RNA determined through gene expression signatures,</a>" PNAS, vol. 100(11):6347-6352 (2003).</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Smart, Nicola et al., "<a href='http://scholar.google.com/scholar?q="A+rapid+and+sensitive+assay+for+quantification+of+siRNA+efficiency+and+specificity%2C"'>A rapid and sensitive assay for quantification of siRNA efficiency and specificity,</a>" Biol. Proced. Online, vol. 7(1):1-7 (2005).</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Supplementary European Search Report for Application No. EP04753864, dated Mar. 5, 2007.</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tang, G. et al, "<a href='http://scholar.google.com/scholar?q="Biochemical+dissection+of+RNA+silencing+in+plants%2C"'>Biochemical dissection of RNA silencing in plants,</a>" Methods Mol. Biol., vol. 257:223-244 (2004).</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tang, Guiliang et al, "<a href='http://scholar.google.com/scholar?q="A+biochemical+framework+for+RNA+silencing+in+plants%2C"'>A biochemical framework for RNA silencing in plants,</a>" Genes &amp; Development, vol. 17:49-63 (2003).</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tomari, Yukihide et al, "<a href='http://scholar.google.com/scholar?q="RISC+assembly+defects+in+the+Drosophila+RNAi+mutant+armitage%2C"'>RISC assembly defects in the Drosophila RNAi mutant armitage,</a>" Department of Biochemistry and Molecular Pharmacology, 2004.</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Vargason, Jeffrey M. et al, "<a href='http://scholar.google.com/scholar?q="Size+Selective+Recognition+of+siRNA+by+an+RNA+Silencing+Suppressor%2C"'>Size Selective Recognition of siRNA by an RNA Silencing Suppressor,</a>" Cell, vol. 115:799-811 (2003).</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Vaucheret, Herve et al., "<a href='http://scholar.google.com/scholar?q="Post-transcriptional+gene+silencing+in+plants%2C"'>Post-transcriptional gene silencing in plants,</a>" Journal of Cell Science, vol. 114:3083-3091, (2001).</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Waterhouse, Peter M. et al., "<a href='http://scholar.google.com/scholar?q="Gene+silencing+as+an+adaptive+defense+against+viruses%2C"'>Gene silencing as an adaptive defense against viruses,</a>" Nature, vol. 411:834-842, (2001).</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Xu et al (Virology 171(2): 331-341, 1989).</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Zeng et al (RNA 9: 112-123, 2003).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8304530">US8304530</a></td><td class="patent-data-table-td patent-date-value">Mar 23, 2010</td><td class="patent-data-table-td patent-date-value">Nov 6, 2012</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Methods and compositions for enhancing the efficacy and specificity of RNA silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8309704">US8309704</a></td><td class="patent-data-table-td patent-date-value">Jun 2, 2004</td><td class="patent-data-table-td patent-date-value">Nov 13, 2012</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Mediates silencing of a target gene; lessening the base pair strength between the 5&#39; end of the first strand and the 3&#39; end of a second strand of the duplex as compared to the base pair strength</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8309705">US8309705</a></td><td class="patent-data-table-td patent-date-value">Mar 29, 2010</td><td class="patent-data-table-td patent-date-value">Nov 13, 2012</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Methods and compositions for enhancing the efficacy and specificity of RNA silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8329892">US8329892</a></td><td class="patent-data-table-td patent-date-value">Mar 29, 2010</td><td class="patent-data-table-td patent-date-value">Dec 11, 2012</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Methods and compositions for enhancing the efficacy and specificity of RNA silencing</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024500">536/24.5</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025300">536/25.3</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N">C12N</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015820000">C12N15/82</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015110000">C12N15/11</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021000000">C07H21/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021040000">C07H21/04</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0048000000">A61K48/00</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/8218">C12N15/8218</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/111">C12N15/111</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/113">C12N15/113</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/335">C12N2310/335</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2320/50">C12N2320/50</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/14">C12N2310/14</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=15l3BgABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/8261">C12N15/8261</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12N15/11M</span>, <span class="nested-value">C12N15/82B4</span>, <span class="nested-value">C12N15/82C8</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Dec 9, 2013</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 13, 2012</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 27, 2011</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIM 1 IS DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 2-30, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 9, 2010</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100802</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 3, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">UNIVERSITY OF MASSACHUSETTS,MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:ZAMORE, PHILLIP D.;TANG, GUILIANG;SIGNED BETWEEN 20040724 AND 20040804;US-ASSIGNMENT DATABASE UPDATED:20100303;REEL/FRAME:24020/805</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:ZAMORE, PHILLIP D.;TANG, GUILIANG;SIGNED BETWEEN 20040724 AND 20040804;REEL/FRAME:24020/805</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:ZAMORE, PHILLIP D.;TANG, GUILIANG;SIGNING DATES FROM 20040724 TO 20040804;REEL/FRAME:024020/0805</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U1DspP-9WXpwd0VQrhMAm-WIYGwUg\u0026id=15l3BgABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U09mACyObb7j69iaGSTmQpxxsOD0w\u0026id=15l3BgABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U1sy3zUsdMUcPr6oo5YMLuSdA4NlQ","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Useful_in_the_design_of_improved_RNAi_me.pdf?id=15l3BgABERAJ\u0026output=pdf\u0026sig=ACfU3U0QgDftMQ3txeE3X9K5gS6tDWPMag"},"sample_url":"http://www.google.com/patents/reader?id=15l3BgABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>