<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7767400 - Enhancing resolution of sequence analysis of short DNA stretches via difined ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Enhancing resolution of sequence analysis of short DNA stretches via difined length spacers; genetic mapping and genomics"><meta name="DC.contributor" content="Timothy D. Harris" scheme="inventor"><meta name="DC.contributor" content="Helicos Biosciences Corporation" scheme="assignee"><meta name="DC.date" content="2008-2-3" scheme="dateSubmitted"><meta name="DC.description" content="The disclosure provides methods of generating paired reads in sequencing-by-synthesis process, particularly, in systems with relatively short read lengths (e.g., 15-35bases), such as for example, in single molecule sequencing by synthesis. Several implementations of the methods are provided. Of particular advantage are the methods that permit re-sequencing of the template, which yields lower error rates. The invention further provides methods of using paired reads, for example, for positioning them over repeats or for assembly into large sequences, including whole genome assembly."><meta name="DC.date" content="2010-8-3" scheme="issued"><meta name="DC.relation" content="US:20060292611:A1" scheme="references"><meta name="DC.relation" content="US:20070070349:A1" scheme="references"><meta name="DC.relation" content="US:20070087362:A1" scheme="references"><meta name="DC.relation" content="US:6210891" scheme="references"><meta name="DC.relation" content="US:6258568" scheme="references"><meta name="DC.relation" content="US:6274320" scheme="references"><meta name="DC.relation" content="US:7169560" scheme="references"><meta name="DC.relation" content="US:7282337" scheme="references"><meta name="DC.relation" content="US:7575865" scheme="references"><meta name="DC.relation" content="US:7601499" scheme="references"><meta name="DC.relation" content="WO:2005003375:A2" scheme="references"><meta name="DC.relation" content="WO:2007010252:A1" scheme="references"><meta name="DC.relation" content="WO:2007120859:A2" scheme="references"><meta name="citation_reference" content="Bennett et al.; &quot;Toward the 1,000 Dollars Human Genome&quot;; Pharmacogenomics, 6:373-382 (2005)."><meta name="citation_reference" content="Edwards et al.; &quot;Automated DNA Sequencing of the Human HPRT Locus&quot;; Genomics; 6:593-608 (1990)."><meta name="citation_reference" content="Karow, Julia; &quot;As Users Demand Paired-End Sequencing, 454, Illumina, and ABI Work on New Kits&quot;; GenomeWeb LLC; print-out (3 pp.) downloaded Oct. 19, 2008 from http://www.in-sequence.com/issues/1-9/features/138789-1. html (2008)."><meta name="citation_reference" content="Karow, Julia; “As Users Demand Paired-End Sequencing, 454, Illumina, and ABI Work on New Kits”; GenomeWeb LLC; print-out (3 pp.) downloaded Oct. 19, 2008 from http://www.in-sequence.com/issues/1—9/features/138789-1. html (2008)."><meta name="citation_reference" content="Print-out (1 page) downloaded Oct. 19, 2008 from http://yahoo.brand.edgar-online.com/EFX-dll/EDGARpro.dll? FetchFilingHTML1?SessionID=M6WLWdR4tzhwJch&amp;ID=537360; &quot;The Current Genetics and Genome Market-Ample Room for Multiple Players &amp; Applications&quot; marked as Appendix A."><meta name="citation_reference" content="Print-out (1 page) downloaded Oct. 19, 2008 from http://yahoo.brand.edgar-online.com/EFX—dll/EDGARpro.dll? FetchFilingHTML1?SessionID=M6WLWdR4tzhwJch&amp;ID=537360; “The Current Genetics and Genome Market—Ample Room for Multiple Players &amp; Applications” marked as Appendix A."><meta name="citation_reference" content="Skiena et al.; Presentation Entitled &quot;Sequence Assembly for Short Paired-Read Technologies&quot;; Department of Computer Science, State University of New York, Stony Brook, NY; (45 pp.); downloaded on Oct. 24, 2008 from http://www.cs.sunysb.edu/."><meta name="citation_reference" content="Specification and pending claims from co-pending co-owned U.S. Appl. No. 11/928,695."><meta name="citation_patent_number" content="US:7767400"><meta name="citation_patent_application_number" content="US:12/025,033"><link rel="canonical" href="http://www.google.com/patents/US7767400"/><meta property="og:url" content="http://www.google.com/patents/US7767400"/><meta name="title" content="Patent US7767400 - Enhancing resolution of sequence analysis of short DNA stretches via difined length spacers; genetic mapping and genomics"/><meta name="description" content="The disclosure provides methods of generating paired reads in sequencing-by-synthesis process, particularly, in systems with relatively short read lengths (e.g., 15-35bases), such as for example, in single molecule sequencing by synthesis. Several implementations of the methods are provided. Of particular advantage are the methods that permit re-sequencing of the template, which yields lower error rates. The invention further provides methods of using paired reads, for example, for positioning them over repeats or for assembly into large sequences, including whole genome assembly."/><meta property="og:title" content="Patent US7767400 - Enhancing resolution of sequence analysis of short DNA stretches via difined length spacers; genetic mapping and genomics"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("xUzsU6GyCML5yQTqg4KYDQ"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("CAN"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("xUzsU6GyCML5yQTqg4KYDQ"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("CAN"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7767400?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7767400"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=97GGBgABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7767400&amp;usg=AFQjCNH7LsnGW76IBkUAG08qpVz1eNs-Lw" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7767400.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7767400.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20090197257"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7767400"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7767400" style="display:none"><span itemprop="description">The disclosure provides methods of generating paired reads in sequencing-by-synthesis process, particularly, in systems with relatively short read lengths (e.g., 15-35bases), such as for example, in single molecule sequencing by synthesis. Several implementations of the methods are provided. Of particular...</span><span itemprop="url">http://www.google.com/patents/US7767400?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7767400 - Enhancing resolution of sequence analysis of short DNA stretches via difined length spacers; genetic mapping and genomics</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7767400 - Enhancing resolution of sequence analysis of short DNA stretches via difined length spacers; genetic mapping and genomics" title="Patent US7767400 - Enhancing resolution of sequence analysis of short DNA stretches via difined length spacers; genetic mapping and genomics"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7767400 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 12/025,033</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Aug 3, 2010</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Feb 3, 2008</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Feb 3, 2008</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US20090197257">US20090197257</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20100304447">US20100304447</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">025033, </span><span class="patent-bibdata-value">12025033, </span><span class="patent-bibdata-value">US 7767400 B2, </span><span class="patent-bibdata-value">US 7767400B2, </span><span class="patent-bibdata-value">US-B2-7767400, </span><span class="patent-bibdata-value">US7767400 B2, </span><span class="patent-bibdata-value">US7767400B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Timothy+D.+Harris%22">Timothy D. Harris</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Helicos+Biosciences+Corporation%22">Helicos Biosciences Corporation</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7767400.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7767400.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7767400.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (13),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (8),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (14),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (14),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (8)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7767400&usg=AFQjCNHGj1nScmMEdji0_QZlvmmju8tifw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7767400&usg=AFQjCNHatZz6G_lZ6FOSYzmEnTs7P3vMvw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7767400B2%26KC%3DB2%26FT%3DD&usg=AFQjCNHBKtHr8qZDxMG-nybeinOkxrZ6Tg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT130121768" lang="EN" load-source="ifi">Enhancing resolution of sequence analysis of short DNA stretches via difined length spacers; genetic mapping and genomics</invention-title></span><br><span class="patent-number">US 7767400 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA79543729" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">The disclosure provides methods of generating paired reads in sequencing-by-synthesis process, particularly, in systems with relatively short read lengths (e.g., 15-35bases), such as for example, in single molecule sequencing by synthesis. Several implementations of the methods are provided. Of particular advantage are the methods that permit re-sequencing of the template, which yields lower error rates. The invention further provides methods of using paired reads, for example, for positioning them over repeats or for assembly into large sequences, including whole genome assembly.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(7)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7767400B2/US07767400-20100803-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7767400B2/US07767400-20100803-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7767400B2/US07767400-20100803-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7767400B2/US07767400-20100803-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7767400B2/US07767400-20100803-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7767400B2/US07767400-20100803-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7767400B2/US07767400-20100803-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7767400B2/US07767400-20100803-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7767400B2/US07767400-20100803-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7767400B2/US07767400-20100803-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7767400B2/US07767400-20100803-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7767400B2/US07767400-20100803-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7767400B2/US07767400-20100803-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7767400B2/US07767400-20100803-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(32)</span></span></div><div class="patent-text"><div mxw-id="PCLM32672168" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A method for generating paired reads from a strand of a nucleic acid duplex, the method comprising:
<div class="claim-text">a) providing a nucleic acid template attached, either directly or indirectly, to a solid support;</div>
<div class="claim-text">b) conducting a sequencing-by-synthesis reaction using detectably labeled nucleotides to obtain a first read of the template;</div>
<div class="claim-text">c) synthesizing a spacer of a defined length; and</div>
<div class="claim-text">d) conducting a sequencing-by-synthesis reaction using detectably labeled nucleotides to obtain a second read of the template, said second read being separated from the first read by the spacer.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the template is individually optically resolvable.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, further comprising mapping the first and second reads onto a reference sequence.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the first and/or the second reads are 5-100 nucleotides long.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the spacer length is greater than the length of the first or the second read.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the spacer length is 20-1000 nucleotides.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the spacer length is fixed by the length of the template.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the spacer length is dynamically controlled.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the first and/or the second reads are resequenced by a multiple pass sequencing.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein a copy of the template comprises a phosphothioate nucleotide or another nucleotide analog resistant to enzymatic digestion.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the first and the second reads are obtained from the same copy of the template.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the first and the second reads are obtained from different copies of the template.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00013" num="00013" class="claim">
      <div class="claim-text">13. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the template is attached to a solid support at the 5′ end that is hybridized to a universal primer covalently attached to the support.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00014" num="00014" class="claim">
      <div class="claim-text">14. The method of <claim-ref idref="CLM-00013">claim 13</claim-ref>, wherein the universal primer is covalently attached to the support.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00015" num="00015" class="claim">
      <div class="claim-text">15. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the template is attached to a solid support at the 3′ end.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00016" num="00016" class="claim">
      <div class="claim-text">16. The method of <claim-ref idref="CLM-00015">claim 15</claim-ref>, wherein the template is attached to the support at the 5<b>40</b> end.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00017" num="00017" class="claim">
      <div class="claim-text">17. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein the template comprises a universal primer site at the 3′ end.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00018" num="00018" class="claim">
      <div class="claim-text">18. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the method comprises:
<div class="claim-text">i) hybridizing the template to a first universal primer which is attached to a solid support;</div>
<div class="claim-text">ii) obtaining the first read by copying a portion of the template proximal to the support;</div>
<div class="claim-text">iii) further extending the copy of the template thereby creating the spacer; and</div>
<div class="claim-text">iv) obtaining the second read by copying a distal portion of the template.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00019" num="00019" class="claim">
      <div class="claim-text">19. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the method comprises:
<div class="claim-text">i) covalently attaching the template to a solid support, said template comprising a complement of a universal primer;</div>
<div class="claim-text">ii) hybridizing the universal primer to the template;</div>
<div class="claim-text">iii) obtaining the first read by copying a first portion of the template;</div>
<div class="claim-text">iv) further extending the copy of the template thereby creating the spacer; and</div>
<div class="claim-text">v) obtaining the second read by copying a second portion of the template.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00020" num="00020" class="claim">
      <div class="claim-text">20. The method of <claim-ref idref="CLM-00019">claim 19</claim-ref>, wherein the method further comprises melting off the copy of the template and, optionally, repeating steps ii) - v).</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00021" num="00021" class="claim">
      <div class="claim-text">21. The method of <claim-ref idref="CLM-00019">claim 19</claim-ref>, wherein the method further comprises incorporating a phosphothioate nucleotide or another nucleotide analog resistant to enzymatic digestion, performing enzymatic digestion of the copy, and optionally, repeating steps ii) - v).</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00022" num="00022" class="claim">
      <div class="claim-text">22. The method of <claim-ref idref="CLM-00019">claim 19</claim-ref>, wherein the template is covalently attached to the support at the 3′ end and the universal primer is proximal to the support.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00023" num="00023" class="claim">
      <div class="claim-text">23. The method of <claim-ref idref="CLM-00019">claim 19</claim-ref>, wherein the template is covalently attached to the support at the 5′ end and the universal primer is distal to the support.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00024" num="00024" class="claim">
      <div class="claim-text">24. The method of <claim-ref idref="CLM-00019">claim 19</claim-ref>, wherein the length of the spacer is dynamically controlled.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00025" num="00025" class="claim">
      <div class="claim-text">25. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, further comprising resequencing the first and/or the second reads.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00026" num="00026" class="claim">
      <div class="claim-text">26. A method of using paired reads, the method comprising: mapping the paired reads generated by the method of <claim-ref idref="CLM-00001">claim 1</claim-ref> onto a reference sequence.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00027" num="00027" class="claim">
      <div class="claim-text">27. The method according to <claim-ref idref="CLM-00026">claim 26</claim-ref>, wherein using comprises positioning the paired reads over repeats in a reference sequence.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00028" num="00028" class="claim">
      <div class="claim-text">28. The method according to <claim-ref idref="CLM-00026">claim 26</claim-ref>, wherein using comprises assembling a portion of a genome.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00029" num="00029" class="claim">
      <div class="claim-text">29. The method according to <claim-ref idref="CLM-00026">claim 26</claim-ref>, wherein using comprises assembling a whole genome.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00030" num="00030" class="claim">
      <div class="claim-text">30. A method for generating paired reads from a strand of a nucleic acid duplex, the method comprising:
<div class="claim-text">a) providing individually optically resolvable nucleic acid templates anchored, directly or indirectly, to a solid support;</div>
<div class="claim-text">b) conducting a sequencing-by-synthesis reaction using detectably labeled nucleotides to obtain a first read of a template, said first read consisting of 20-150 bases;</div>
<div class="claim-text">c) synthesizing a spacer of 20-1000 bases; and</div>
<div class="claim-text">d) conducting a sequencing-by-synthesis reaction using detectably labeled nucleotides to obtain a second read of the template, said second read consisting of 20-150 bases and separated from the first read by the spacer.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00031" num="00031" class="claim">
      <div class="claim-text">31. The method of <claim-ref idref="CLM-00030">claim 30</claim-ref>, further comprising resequencing the first and/or the second reads prior to the mapping step.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00032" num="00032" class="claim">
      <div class="claim-text">32. A method of using paired reads, the method comprising: mapping the paired reads generated by the method of <claim-ref idref="CLM-00030">claim 30</claim-ref> onto a reference sequence.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES38004517" lang="EN" load-source="patent-office" class="description">
    <heading>TECHNICAL FIELD</heading> <p num="p-0002">The invention is in the field of molecular biology and relates to methods for nucleic acid analysis. In particular, the invention relates to methods of obtaining paired-end reads on nucleic acid sequencing-by-synthesis platforms.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p num="p-0003">A number of initiatives are currently underway to obtain sequence information directly from millions of individual molecules of DNA in parallel.</p>
    <p num="p-0004">The real-time single molecule sequencing-by-synthesis technologies rely on the detection of fluorescent nucleotides as they are incorporated into a nascent strand of DNA that is complementary to the template being sequenced. An example of asynchronous single molecule sequencing by synthesis is illustrated in <figref idrefs="DRAWINGS">FIG. 1</figref>. As shown, oligonucleotides 30-50 bases in length are covalently anchored at the 5′ end to glass cover slips. These anchored strands perform two functions. First, they act as capture sites for the target template strands, if the templates are configured with capture tails complementary to the surface bound oligonucleotides. They also act as primers for the template-directed primer extension that forms the basis of the sequence reading. The capture primers are a fixed position site for sequence determination. Each cycle consists of adding the polymerase-labeled nucleotide analog mixture, rinsing, optically imaging the field containing millions of active primer template duplexes, and chemically cleaving the dye-linker to remove the dye. The cycle (synthesis, detection, and dye removal) is repeated up to 100 times and, possibly, more.</p>
    <p num="p-0005">Four major high-throughput sequencing platforms are currently available: the Genome Sequencers from Roche/454 Life Sciences (Margulies et al. (2005) Nature, 437:376-380; U.S. Pat. Nos. 6,274,320; 6,258,568; 6,210,891), the 1G Analyzer from Illumina/Solexa (Bennett et al. (2005) Pharmacogenomics, 6:373-382), the SOLiD system from Applied Biosystems (solid.appliedbiosystems.com), and the Heliscope™ system from Helicos Biosciences (see, e.g., U.S. Patent App. Pub. No. 2007/0070349 and the illustration in <figref idrefs="DRAWINGS">FIG. 1</figref>). Although these new technologies are significantly cheaper compared to the traditional methods, such as gel/capillary Gilbert-Sanger sequencing, the sequence reads produced by the new technologies are generally much shorter (˜25-40 vs. ˜500-700 bases). For example, the average read lengths on the four major platforms are currently as follows: Roche/454, 250 bases (depending on the organism); Illumina/Solexa, 25 bases; SOLiD, 35 bases; Heliscope, 25 bases. While such short reads (also referred to as “microreads”) are sufficient for the resequencing ˜80% of normal human genomes, for which there is a reliable reference sequence, microreads are limiting for a number of other applications. First, short reads are not optimal for the de novo assembly of genomes. Second, the detection and proper placement of amplifications, inversions, and translocations using short reads are severely limited. The proper detection and placement of short indels are also difficult. Short reads may therefore be problematic for the resequencing of highly polymorphic or highly aberrant genomes. For example, the occurrence of Large-scale Copy-number Variations (LCVs) in normal (non-disease) individuals is an indication that acquiring an accurate description of human genetic variation may require more than the detection of single-nucleotide polymorphisms. Genetic rearrangements are even more heterogeneous and prevalent in cancer genomes, underscoring the importance of their proper detection and characterization.</p>
    <p num="p-0006">To mitigate the drawbacks of short-read sequencing methods, several groups have proposed to use paired reads, the approach originally developed by Edwards et al. (Genomics (1990) 6: 593-608) for traditional sequencing methods. By linking two reads positioned a certain known distance apart from each other (thus also referred to as “paired,” “paired-end,” “mate,” or “matched” reads), a large informatic leverage is achieved to resolve repeats, insertions, deletions, and inversions, which are important mutation types, for example, in tumors.</p>
    <p num="p-0007">In addition to the above limitations of short reads, another inherent disadvantage of single molecule sequencing technologies is a high per-read error rate. This is due to the all-or-none signal detection during an incorporation event and the increased susceptibility to contaminating nucleotides. For instance, the incorporation of an unlabeled nucleotide contaminant in a single nascent strand of complementary DNA will produce a failed detection event or a deletion in the read relative to the reference. Sequencing errors in short reads are especially problematic as they complicate proper alignment of the reads onto a reference sequence. Thus, techniques that allow re-sequencing the original DNA template are preferable because they drastically reduce sequencing errors. See, e.g., WO 2007/12089 and U.S. patent application Ser. No. 11/404,675 for “melt-and-resequence” methods.</p>
    <p num="p-0008">Therefore, there is a need for methods of obtaining paired reads in sequencing-by-synthesis technologies, particularly, those with short read lengths, such as produced by single molecule sequencing.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0009">The invention provides methods of generating paired reads in a sequencing-by-synthesis process, particularly, in systems with relatively short read lengths (e.g., 15-35 bases), such as for example, in single molecule sequencing by synthesis. In such single molecule sequencing methods, the template is individually-optically resolvable. Additional advantages are provided by those methods that permit re-sequencing of the original template, thereby yielding fewer sequencing errors.</p>
    <p num="p-0010">In general, the methods of the invention include:
</p> <ul> <li id="ul0001-0001" num="0000"> <ul> <li id="ul0002-0001" num="0010">a) providing a template nucleic acid;</li> <li id="ul0002-0002" num="0011">b) conducting a sequencing-by-synthesis reaction to obtain a first read from the template; and</li> <li id="ul0002-0003" num="0012">c) conducting a sequencing-by-synthesis reaction to obtain a second read from the template.
<br>
Both reads (the first and second reads) are separated in the template nucleic acid by a spacer of a defined length (e.g., 20-250). The spacer length may be fixed by the size of the template or it may be dynamically controlled, (e.g., by kinetic control of the polymerization reaction or other methods).
</li> </ul> </li> </ul> <p num="p-0011">In some embodiments, one or both reads are generated by copying the original template, while in other embodiments one or both reads are generated by copying a copy of the original template. In some embodiments, the reads are generated from the same strand of a nucleic acid duplex, while in other embodiments, the first and second reads are generated from different strands.</p>
    <p num="p-0012">In certain embodiments (including Method A as illustrated in <figref idrefs="DRAWINGS">FIG. 2</figref>), the methods of the invention include:
</p> <ul> <li id="ul0003-0001" num="0000"> <ul> <li id="ul0004-0001" num="0015">i) hybridizing a template to a first universal primer which is attached to a solid support;</li> <li id="ul0004-0002" num="0016">ii) obtaining a first read by copying a portion of the template proximal to the support;</li> <li id="ul0004-0003" num="0017">iii) extending the copy of the template so as to copy the complement of a second universal priming site located on the distal end of the template;</li> <li id="ul0004-0004" num="0018">iv) melting off the template;</li> <li id="ul0004-0005" num="0019">v) hybridizing a second universal primer to the copy of the template; and</li> <li id="ul0004-0006" num="0020">vi) obtaining the second read by copying a portion of the copy of the template, said portion of the copy distal to the support.</li> </ul> </li> </ul> <p num="p-0013">In certain other embodiments (including Method B as illustrated in <figref idrefs="DRAWINGS">FIG. 3</figref>), the methods of the invention include:
</p> <ul> <li id="ul0005-0001" num="0000"> <ul> <li id="ul0006-0001" num="0022">i) hybridizing a template to a first universal primer which is attached to a solid support;</li> <li id="ul0006-0002" num="0023">ii) obtaining a first read by copying a portion of the template proximal to the support;</li> <li id="ul0006-0003" num="0024">iii) further extending the copy of the template thereby creating a spacer; and</li> <li id="ul0006-0004" num="0025">iv) obtaining the second read by copying a portion of the template distal to the support.</li> </ul> </li> </ul> <p num="p-0014">Yet in certain other embodiments (including Methods C and D as illustrated in <figref idrefs="DRAWINGS">FIGS. 4 and 5</figref>, respectively), the methods of the invention include:
</p> <ul> <li id="ul0007-0001" num="0000"> <ul> <li id="ul0008-0001" num="0027">i) covalently attaching a template to a solid support, said template comprising a complement of a universal primer;</li> <li id="ul0008-0002" num="0028">ii) hybridizing the universal primer to the template;</li> <li id="ul0008-0003" num="0029">iii) obtaining a first read by copying a first portion of the template;</li> <li id="ul0008-0004" num="0030">iv) further extending the copy of the template thereby creating a spacer; and</li> <li id="ul0008-0005" num="0031">v) obtaining a second read by copying a second portion of the template.
<br>
In some such embodiments, the template is covalently attached to the support at the 3′ end and the universal primer is proximal to the support, whereas in other embodiments, the template is covalently attached to the support at the 5′ end and the universal primer is distal to the support.
</li> </ul> </li> </ul> <p num="p-0015">The invention also provides methods of using the paired reads, for example, for positioning them over repeats or for assembly into large sequences, including whole genome assembly.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE FIGURES</heading> <p num="p-0016"> <figref idrefs="DRAWINGS">FIGS. 1A and 1B</figref> illustrate a typical process of single molecule sequencing by synthesis. 1) “Capture probes” (T(50) oligonucleotides also functioning as primers) are covalently bound with 5′ “down” to a surface. 2) Genomic DNA is fragmented, and a polyA tail and a Cy3 label are added at 3′ each fragment. These DNA templates are then hybridized to the capture probes. 3) The captured templates are imaged to establish their location. 4) The captured templates are incubated with a Cy5-labeled nucleotide and a polymerase mixture to allow the polymerization reaction to proceed. 5) The surface is rinsed to wash out unincorporated nucleotides and other reagents. 6) The incorporated nucleotides are imaged and associated with each template by their location. 7) The Cy5 label is chemically cleaved off. 8) The process is repeated with another type of nucleotide.</p>
      <p num="p-0017"> <figref idrefs="DRAWINGS">FIG. 2</figref> illustrates a method for generating paired-end reads (Method A), which includes a modified copy step, wherein the spacer length is determined by the size the template. This method allows resequencing of the distal read with a possibility of resequencing the surface proximal read given the proper length of the spacer.</p>
      <p num="p-0018"> <figref idrefs="DRAWINGS">FIG. 3</figref> illustrates a method for generating paired-end reads (Method B) with a 5′ non-covalent attachment of the template and a dynamically controlled spacer length. This method does not allow resequencing of the original template.</p>
      <p num="p-0019"> <figref idrefs="DRAWINGS">FIG. 4</figref> illustrates a method for generating paired-end reads (Method C) with a 3′-covalently attached template, which allows resequencing.</p>
      <p num="p-0020"> <figref idrefs="DRAWINGS">FIG. 5</figref> illustrates a method for generating paired-end reads (Method D) which includes a modified copy step, dynamically controlled spacer length. This method allows re-sequencing of the distal read with a possibility of resequencing the surface-proximal end given the proper length of the spacer.</p>
      <p num="p-0021"> <figref idrefs="DRAWINGS">FIG. 6</figref> provides length distribution of paired-end reads obtained using Method A as described in the Examples.</p>
    </description-of-drawings> <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p num="p-0022">The invention provides methods of generating paired reads from a nucleic acid template in a sequencing-by-synthesis process, wherein the first and second reads are separated in the template nucleic acid by a spacer of a defined length. The invention is suitable for various sequencing-by-synthesis technologies, in particular, the sequencing platforms with relatively short read lengths. In illustrative embodiments, the methods are used in the context of single molecule sequencing as described, for example, in U.S. Pat. No. 7,283,337.</p>
    <p num="p-0023">Generally, the methods of the invention include:
</p> <ul> <li id="ul0009-0001" num="0000"> <ul> <li id="ul0010-0001" num="0041">a) providing a template nucleic acid;</li> <li id="ul0010-0002" num="0042">b) conducting a sequencing-by-synthesis reaction to obtain a first read from the template; and</li> <li id="ul0010-0003" num="0043">c) conducting a sequencing-by-synthesis reaction to obtain a second read from the template.</li> </ul> </li> </ul> <p num="p-0024">Nonlimiting illustrative embodiments of the invention are shown in <figref idrefs="DRAWINGS">FIGS. 2-5</figref> and are described in detail below.</p>
    <p num="p-0025">The invention also provides methods of using the paired reads, for example, for positioning them over repeats or for assembly into large sequences, including whole genome assembly. Such methods include mapping the first and second paired reads onto a reference sequence.</p>
    <p num="p-0026">In some embodiments, the invention provides additional advantages because it allows resequencing of the reads, thereby reducing the overall error-rate for respective read(s) (“multiple pass reads”). For example, the first and/or the second reads are resequenced multiple times, e.g., once, twice, or more times. In some embodiments, the error rates for single pass sequencing are 3-5% per base, whereas the error rate for two-pass sequencing of the same single molecule is reduced to 0.1-0.25%.</p>
    <p num="h-0006">Reads and Spacers</p>
    <p num="p-0027">Reads are typically represented by a sequence of detectably labeled nucleotides incorporated into a growing chain of a nucleic acid duplex. Once the nucleotides are incorporated, their labels may remain attached or they may be released. Read lengths (average available read lengths) in the methods of the invention may vary depending on the system used. The first or second read length may be, for example, 5-150, 5-100, 10-75, 15-50, 15-25, 20-35 nucleotides, 50 or fewer, 40 or fewer, 30 or fewer, 25 or fewer, and 20 or fewer nucleotides. (The terms “nucleotide” and “base” in reference to sequence lengths are used interchangeably herein.)</p>
    <p num="p-0028">In some embodiments, one or both reads are generated by copying the original template, while in other embodiments, one or both reads are generated by copying a copy of the original template. In some embodiments, the reads are generated from the same strand of a nucleic acid duplex, while in other embodiments, the first and second reads are generated from different strands.</p>
    <p num="p-0029">A spacer is typically generated by incorporating nonlabeled (e.g., native) nucleotides into the growing chain. The spacer lengths may vary. As used herein the length of the spacer is defined as the length of sequence between the first and second reads, regardless of whether both reads are generated from the same strand of a nucleic acid or from two different (complementary) strands. In some embodiments, the spacer length is greater than the length of either the first or the second read. The length is “defined” in that it is chosen before obtaining the read(s) and/or is determined after the reads are obtained. For example, the spacer length may be 20-1000, 20-750, 20-500, 30-300, 50-250, 50-150, at least 20 at least 50, at least 75, at least 100, or at least 150 bases long. In various embodiments of the invention, the spacer length can be controlled to ±50%, ±40%, ±30%, ±25% or a lower percent.</p>
    <p num="p-0030">In some embodiments, the spacer length is fixed by the size of the template, for example, when two priming sites are located at each end of the template (see, e.g., <figref idrefs="DRAWINGS">FIG. 2</figref>). In other embodiments, the spacer length is dynamically controlled (see, e.g., <figref idrefs="DRAWINGS">FIGS. 3</figref>, <b>4</b>, and <b>5</b>). Several methods may be used for the dynamic control of the spacer length. For example, in one method, only one species of the labeled nucleotide is added at a time per incorporation cycles, whereupon its location in the growing chain is detected. (The sequential addition of all four species of labeled nucleotides is referred to as a “quad.”) The spacer of desired length may be synthesized by the addition of a pre-determined number of quads coupled with kinetic control of the nucleotide addition by the polymerase, e.g., as described in U.S. Pat. No. 7,169,560. The statistics of spacer length vs. cycle count can be predetermined for a given system and target. Taking M13 genome as an example of genomic sequence, the average distance between G bases is 6 nucleotides with a maximum of 34 bases. If, for example, a long run of AT repeats, sometimes seen in tumors may be encountered, thus a very long spacer would result. If misincorporation can be controlled, variation of the limiting base would allow a useful variety of spacer control statistic to be used. For the G spacer limit chosen above, GC rich regions grow more slowly than AT rich regions, but useful spacer length control is maintained. Under controlled conditions, each incorporation cycle need take no more than a few seconds. Spacer growth may take a minute or two and can be easily automated. An alternative way to dynamically control the spacer length is to use a polymerase in place of the terminal deoxynucleotidyl transferase (TdT), all 4 dNTPs in place of dATP, and duplex templates in place of single-stranded DNA “buffer.” A comparable process can be used for control of spacer length. A mixture of nucleotides reflecting the sample composition with the total concentration a factor greater than the template concentration, equal to the desired spacer length may be preferred, or an excess concentration of 3 nucleotides and 1 limiting base may give preferred results. Initiation of synthesis can be controlled with temperature. Other suitable methods for the dynamic control of spacer length may be used.</p>
    <p num="p-0031">Specific exemplary embodiments (Methods A-D) are described below.</p>
    <p num="h-0007">Method A: Single Pass Reads with Copy to End</p>
    <p num="p-0032">One method for generating paired-end reads according to the invention is illustrated in <figref idrefs="DRAWINGS">FIG. 2</figref>. The first read is obtained using a sequencing-by-synthesis process, and the copy is completed using unlabeled nucleotides. The priming at the 3′ end of the anchored copy strand allows the top primer to provide the second read of the pair. Spacer length is determined by the template size. Using size filtration columns, efficient sample handling with sizes 250±100 bases is feasible. However, the approach illustrated in <figref idrefs="DRAWINGS">FIG. 2</figref> does not allow both ends of the template to be read more than once. The surface proximal end is read only once.</p>
    <p num="p-0033">Accordingly, in some embodiments, the invention provides a method that includes:
</p> <ul> <li id="ul0011-0001" num="0000"> <ul> <li id="ul0012-0001" num="0054">i) hybridizing a template to a first universal primer which is attached to a solid support;</li> <li id="ul0012-0002" num="0055">ii) obtaining a first read by copying a portion of the template proximal to the support;</li> <li id="ul0012-0003" num="0056">iii) extending the copy of the template so as to copy the complement of a second universal priming site located on the distal end of the template;</li> <li id="ul0012-0004" num="0057">iv) melting off the template;</li> <li id="ul0012-0005" num="0058">v) hybridizing a second universal primer to the copy of the template; and</li> <li id="ul0012-0006" num="0059">vi) obtaining a second read by copying a portion of the copy of the template, said portion of the copy distal to the support.
<br>
Method B: Single Pass Reads with Variable Spacers
</li> </ul> </li> </ul> <p num="p-0034">In the embodiments, such as the one shown in <figref idrefs="DRAWINGS">FIG. 3</figref>, the template is anchored to the surface and the first read (e.g., the first 15-25 bases) is obtained by a sequencing-by-synthesis process, following which a series of unlabeled bases are added to form a spacer. Then, a second read is obtained by extending the spacer in a second round of sequencing-by-synthesis (with labeled nucleotides). The length of the spacer is dynamically controlled. In some such embodiments, the method of the invention includes the following steps:
</p> <ul> <li id="ul0013-0001" num="0000"> <ul> <li id="ul0014-0001" num="0061">i) hybridizing a template to a first universal primer which is attached to a solid support;</li> <li id="ul0014-0002" num="0062">ii) obtaining a first read by copying a portion of the template proximal to the support;</li> <li id="ul0014-0003" num="0063">iii) further extending the copy of the template thereby creating a spacer; and</li> <li id="ul0014-0004" num="0064">iv) obtaining a second read by copying a portion of the template distal to the support.
<br>
Method C: Multiple Pass Reads with A 3′ Covalent Attachment of Template
</li> </ul> </li> </ul> <p num="p-0035">As illustrated in <figref idrefs="DRAWINGS">FIG. 4</figref>, the template nucleic can be attached at the 3′ end, using click chemistry. This template attachment strategy uses the highly specific reactivity between terminal azide and terminal alkyne groups (see, e.g., Chidsey et al. (2005 JACS, 127:8600 and references cited therein). A first step is to provide a surface with either terminal azide or terminal alkyne groups. Second, the template must be provided with the other member of the click pair, for example, using a nucleotide analog with that group on a suitable linker. One such example is deoxy-adenine triphosphate with a 6-carbon linker terminated with an alkyne. This nucleotide analog can be added to the 3′ terminus using d Terminal transferase (dTT) as is done to add labeled diodeoxy 3′ terminal bases to these templates. These templates can then be covalently attached to an azide functional surface simply by exposing the surface to the template mixture with the appropriate catalyst.</p>
    <p num="p-0036">Accordingly, in some embodiments, the invention provides a method that includes:
</p> <ul> <li id="ul0015-0001" num="0000"> <ul> <li id="ul0016-0001" num="0067">i) covalently attaching the template to a solid support at the 3′ end (the template contains a universal priming site which is proximal to the support);</li> <li id="ul0016-0002" num="0068">ii) hybridizing the universal primer to the template;</li> <li id="ul0016-0003" num="0069">iii) obtaining a first read by copying a first portion of the template;</li> <li id="ul0016-0004" num="0070">iv) further extending the copy of the template thereby creating a spacer; and</li> <li id="ul0016-0005" num="0071">v) obtaining a second read by copying a second portion of the template.</li> </ul> </li> </ul> <p num="p-0037">If resequencing is desired, all or a part of the template copy can be removed and steps ii)-v) can be repeated. For example, a copy of the template is melted off. Alternatively, a phosphothioate nucleotide or another nucleotide analog resistant to enzymatic digestion is incorporated in the copy and the respective portion of the copy is removed by enzymatic digestion (e.g., exonuclease digestion). the universal primer may be first capped with a phosphotioate nucleotide. For example, the addition of a phosphothioate terminal nucleotide to the primer allows exonuclease digestion of the copy strand with exception of the primer. Upon digestion, both reads may be resequenced. Yet if only the distal read needs to be resequenced, the phosphotioate may be incorporated immediately before the distal read so that upon exonuclease digestion, solely the distal read may be resequenced. It will be understood that if only a part of the template copy is removed, then only steps iii) through v) or iv) through v) need to be repeated, depending on the position of phosphothioate.</p>
    <p num="h-0008">Method D: Multiple Pass Reads with A 5′ Covalent Attachment of Template</p>
    <p num="p-0038">As illustrated in <figref idrefs="DRAWINGS">FIG. 5</figref>, the template nucleic can be attached at the 5′ end, for example, by an amine linkage.</p>
    <p num="p-0039">Accordingly, in some embodiments, the invention provides a method that includes:
</p> <ul> <li id="ul0017-0001" num="0000"> <ul> <li id="ul0018-0001" num="0075">i) covalently attaching a template to a solid support at the 5′ end (the template contains a universal priming site which is distal to the support);</li> <li id="ul0018-0002" num="0076">ii) hybridizing the universal primer to the template;</li> <li id="ul0018-0003" num="0077">iii) obtaining a first read by copying a first portion of the template;</li> <li id="ul0018-0004" num="0078">iv) further extending the copy of the template thereby creating a spacer; and</li> <li id="ul0018-0005" num="0079">v) obtaining a second read by copying a second portion of the template.</li> </ul> </li> </ul> <p num="p-0040">Similarly to Method C, all or a part of the template copy can be removed, and the first and/or the second reads may be resequenced.</p>
    <p num="h-0009">Nucleic Acid Templates</p>
    <p num="p-0041">The length of the target nucleic acid may vary. The average length of the target nucleic acid may be, for example, at least 300, 350, 400, 450, 500, 550, 600, 700, 800, 900, 1000, 2000, 3000, 4000, 5000 nts or longer. In some embodiments, the length of the target is between 300 and 5000 nts, 400 and 4000 nts, or 500 and 3000 nts.</p>
    <p num="p-0042">Template nucleic acid can come from a variety of sources. For example, nucleic acids can be naturally occurring DNA or RNA (e.g., mRNA or non-coding RNA) isolated from any source, recombinant molecules, cDNA, or synthetic analogs. For example, the template nucleic acid may include whole genes, gene fragments, exons, introns, regulatory elements (such as promoters, enhancers, initiation and termination regions, expression regulatory factors, expression controls, and other control regions), DNA comprising one or more single-nucleotide polymorphisms (SNPs), allelic variants, and other mutations. The template nucleic acid may also be tRNA, rRNA, ribozymes, splice variants, or antisense RNA.</p>
    <p num="p-0043">Template nucleic acid may be obtained from whole organisms, organs, tissues, or cells from different stages of development, differentiation, or disease state, and from different species (human and non-human, including bacteria and virus). Various methods for extraction of nucleic acids from biological samples are known (see, e.g., Nucleic Acids Isolation Methods, Bowein (ed.), American Scientific Publishers, 2002). Typically, genomic DNA is obtained from nuclear extracts that are subjected to mechanical shearing to generate random long fragments. For example, genomic DNA may be extracted from tissue or cells using a Qiagen DNeasy Blood &amp; Tissue Kit following the manufacturer's protocols.</p>
    <p num="h-0010">Sequencing platforms</p>
    <p num="p-0044">The invention can be used on any suitable sequencing-by-synthesis platform. As described above, four major sequencing-by-synthesis platforms are currently available: the Genome Sequencers from Roche/454 Life Sciences, the 1 G Analyzer from Illumina/Solexa, the SOLiD system from Applied BioSystems, and the Heliscope system from Helicos Biosciences. Each of these platforms can be used in the methods of the invention. In some embodiments, the sequencing platforms used in the methods of the present invention have one or more of the following features:
</p> <ul> <li id="ul0019-0001" num="0000"> <ul> <li id="ul0020-0001" num="0085">1) four differently optically labeled nucleotides are utilized (e.g., 1 G Analyzer);</li> <li id="ul0020-0002" num="0086">2) sequencing-by-ligation is utilized (e.g., SOLiD);</li> <li id="ul0020-0003" num="0087">3) pyrophosphate detection is utilized (e.g., Roche/454); and</li> <li id="ul0020-0004" num="0088">4) four identically optically labeled nucleotides are utilized (e.g., Helicos).</li> </ul> </li> </ul> <p num="p-0045">Heliscope is the only one of the four systems that provides true single-molecule sequencing (tSMS™), thus eliminating amplification artifacts such as errors or bias. Thus, in some embodiments, the methods of the invention are practiced on a tSMS™ system.</p>
    <p num="p-0046">In some embodiments, a plurality of nucleic acid molecules being sequenced is bound to a solid support. To immobilize the nucleic acid on a solid support, a capture sequence/universal priming site can be added at the 3′ and/or 5′ end of the template. The nucleic acids may be bound to the solid support by hybridizing the capture sequence to a complementary sequence covalently attached to the solid support. The capture sequence (also referred to as a universal capture sequence) is a nucleic acid sequence complimentary to a sequence attached to a solid support that may dually serve as a universal primer. In some embodiments, the capture sequence is polyN<sub>n</sub>, wherein N is U, A, T, G, or C, n≧5, e g., 20-70, 40-60, e.g., about 50. For example, the capture sequence could be polyT<sub>40-50 </sub>or its complement.</p>
    <p num="p-0047">As an alternative to a capture sequence, a member of a coupling pair (such as, e.g., antibody/antigen, receptor/ligand, or the avidin-biotin pair as described in, e.g., U.S. Patent Application No. 2006/0252077) may be linked to each fragment to be captured on a surface coated with a respective second member of that coupling pair.</p>
    <p num="p-0048">The solid support may be, for example, a glass surface such as described in, e.g., U.S. Patent App. Pub. No. 2007/0070349. The surface may be coated with an epoxide, polyelectrolyte multilayer, or other coating suitable to bind nucleic acids. In preferred embodiments, the surface is coated with epoxide and a complement of the capture sequence is attached via an amine linkage. The surface may be derivatized with avidin or streptavidin, which can be used to attach to a biotin-bearing target nucleic acid. Alternatively, other coupling pairs, such as antigen/antibody or receptor/ligand pairs, may be used. The surface may be passivated in order to reduce background. Passivation of the epoxide surface can be accomplished by exposing the surface to a molecule that attaches to the open epoxide ring, e.g., amines, phosphates, and detergents.</p>
    <p num="p-0049">Subsequent to the capture, the sequence may be analyzed, for example, by single molecule detection/sequencing, e.g., as described in the Example and in U.S. Pat. No. 7,283,337, including template-dependent sequencing-by-synthesis. In sequencing-by-synthesis, the surface-bound molecule is exposed to a plurality of labeled nucleotide triphosphates in the presence of polymerase. The sequence of the template is determined by the order of labeled nucleotides incorporated into the 3′ end of the growing chain. This can be done in real time or can be done in a step-and-repeat mode. For real-time analysis, different optical labels to each nucleotide may be incorporated and multiple lasers may be utilized for stimulation of incorporated nucleotides.</p>
    <p num="p-0050">Other details and variations of the sequencing methods are provided below.</p>
    <p num="h-0011">Other General Considerations</p>
    <p num="p-0051">A. Nucleotides—Nucleotides useful in the invention include any nucleotide or nucleotide analog, whether naturally occurring or synthetic. For example, preferred nucleotides include phosphate esters of deoxyadenosine, deoxycytidine, deoxyguanosine, deoxythymidine, adenosine, cytidine, guanosine, and uridine. Other nucleotides useful in the invention comprise an adenine, cytosine, guanine, thymine base, a xanthine or hypoxanthine; 5-bromouracil, 2-aminopurine, deoxyinosine, or methylated cytosine, such as 5-methylcytosine, and N4-methoxydeoxycytosine. Also included are bases of polynucleotide mimetics, such as methylated nucleic acids, e.g., 2′-O-methRNA, peptide nucleic acids, modified peptide nucleic acids, locked nucleic acids and any other structural moiety that can act substantially like a nucleotide or base, for example, by exhibiting base-complementarity with one or more bases that occur in DNA or RNA and/or being capable of base-complementary incorporation, and includes chain-terminating analogs. A nucleotide corresponds to a specific nucleotide species if they share base-complementarity with respect to at least one base.</p>
    <p num="p-0052">Nucleotides for nucleic acid sequencing according to the invention preferably comprise a detectable label that is directly or indirectly detectable. Preferred labels include optically-detectable labels, such as fluorescent labels. Examples of fluorescent labels include, but are not limited to, 4-acetamido-4′-isothiocyanatostilbene-2,2′disulfonic acid; acridine and derivatives: acridine, acridine isothiocyanate; 5-(2′-aminoethyl)aminonaphthalene-1-sulfonic acid (EDANS); 4-amino-N-[3-vinylsulfonyl)phenyl]naphthalimide-3,5 disulfonate; N-(4-anilino-1-naphthyl)maleimide; anthranilamide; BODIPY; Brilliant Yellow; coumarin and derivatives; coumarin, 7-amino-4-methylcoumarin (AMC, Coumarin 120), 7-amino-4-trifluoromethylcouluarin (Coumaran 151); cyanine dyes, cyanosine; 4′,6-diaminidino-2-phenylindole (DAPI); 5′5″-dibromopyrogallol-sulfonaphthalein (Bromopyrogallol Red); 7-diethylamino-3-(4′-isothiocyanatophenyl)-4-methylcoumarin; diethylenetriamine pentaacetate; 4,4′-diisothiocyanatodihydro-stilbene-2,2′-disulfonic acid; 4,4′-diisothiocyanatostilbene-2,2′-disulfonic acid; 5-[dimethylamino]naphthalene-1-sulfonyl chloride (DNS, dansylchloride); 4-dimethylaminophenylazophenyl-4′-isothiocyanate (DABITC); eosin and derivatives; eosin, eosin isothiocyanate, erythrosin and derivatives; erythrosin B, erythrosin, isothiocyanate; ethidium; fluorescein and derivatives; 5-carboxyfluorescein (FAM), 5-(4,6-dichlorotriazin-2-yl)aminofluorescein (DTAF), 2′,7′-dimethoxy-4′5′-dichloro-6-carboxyfluorescein, fluorescein, fluorescein isothiocyanate, QFITC, (XRITC); fluorescamine; IR144; IR1446; Malachite Green isothiocyanate; 4-methylumbelliferoneortho cresolphthalein; nitrotyrosine; pararosaniline; Phenol Red; B-phycoerythrin; o-phthaldialdehyde; pyrene and derivatives: pyrene, pyrene butyrate, succinimidyl 1-pyrene; butyrate quantum dots; Reactive Red 4 (Cibacron® Brilliant Red 3B-A) rhodamine and derivatives: 6-carboxy-X-rhodamine (ROX), 6-carboxyrhodamine (R6G), lissamine rhodamine B sulfonyl chloride rhodamine (Rhod), rhodamine B, rhodamine 123, rhodamine X isothiocyanate, sulforhodamine B, sulforhodamine 101, sulfonyl chloride derivative of sulforhodamine 101 (Texas Red); N,N,N′,N′tetramethyl-6-carboxyrhodamine (TAMRA); tetramethyl rhodamine; tetramethyl rhodamine isothiocyanate (TRITC); riboflavin; rosolic acid; terbium chelate derivatives; Cy3; Cy5; Cy5.5; Cy7; IRD 700; IRD 800; La Jolta Blue; phthalo cyanine; and naphthalo cyanine. Preferred fluorescent labels are cyanine-3 and cyanine-5. Labels other than fluorescent labels are contemplated by the invention, including other optically-detectable labels.</p>
    <p num="p-0053">B. Nucleic Acid Polymerases—Nucleic acid polymerases generally useful in the invention include DNA polymerases, RNA polymerases, reverse transcriptases, and mutant or altered forms of any of the foregoing. DNA polymerases and their properties are described in detail in, among other places, DNA Replication 2nd edition, Komberg and Baker, W. H. Freeman, New York, N.Y. (1991). Known conventional DNA polymerases useful in the invention include, but are not limited to, <i>Pyrococcus furiosus </i>(Pfu) DNA polymerase (Lundberg et al. (1991) Gene, 108:1, Stratagene), <i>Pyrococcus woesei </i>(Pwo) DNA polymerase (Hinnisdaels et al., 1996, Biotechniques, 20:186-8, Boehringer Mannheim), <i>Thermus thermophilus </i>(Tth) DNA polymerase (Myers and Gelfand 1991, Biochemistry 30:7661), <i>Bacillus stearothermophilus </i>DNA polymerase (Stenesh et al. (1977) Biochim. Biophys. Acta, 475:32), <i>Thermococcus litoralis </i>(Tli) DNA polymerase (also referred to as Vent® DNA polymerase, Cariello et al. (1991) Polynucleotides Res., 19:4193; New England Biolabs), 9° Nm® DNA polymerase (New England Biolabs), Stoffel fragment, ThermoSequenase® (Amersham Pharmacia Biotech UK), Therminator® (New England Biolabs), <i>Thermotoga maritima </i>(Tma) DNA polymerase (Diaz et al. (1998) Braz. J. Med. Res., 31:1239), <i>Thermus aquaticus </i>(Taq) DNA polymerase (Chien et al. (1976) J. Bacteoriol., 127:1550), DNA polymerase, <i>Pyrococcus kodakaraensis </i>KOD DNA polymerase (Takagi et al. (1997) Appl. Environ. Microbiol., 63:4504), JDF-3 DNA polymerase (from <i>thermococcus </i>sp. JDF-3, PCT Patent Application Publication WO 01/32887), <i>Pyrococcus </i>GB-D (PGB-D) DNA polymerase (also referred as Deep Vent® DNA polymerase, Juncosa-Ginesta et al. (1994) Biotechniques, 16:820; New England Biolabs), UITma DNA polymerase (from thermophile <i>Thermotoga maritima; </i>Diaz et al. (1998) Braz. J. Med. Res., 31:1239; PE Applied Biosystems), Tgo DNA polymerase (from <i>thermococcus gorgonarius</i>, Roche Molecular Biochemicals), <i>E. coli </i>DNA polymerase I (Lecomte et al. (1983) Polynucleotides Res., 11:7505), T7 DNA polymerase (Nordstrom et al. (1981) J. Biol. Chem., 256:3112), and archaeal DP11/DP2 DNA polymerase II (Cann et al. (1998) Proc. Natl. Acad. Sci. USA, 95:14250-5).</p>
    <p num="p-0054">While mesophilic polymerases are contemplated by the invention, preferred polymerases are thermophilic. Thermophilic DNA polymerases include, but are not limited to, ThermoSequenase®, 9°N®, Therminator®, Taq, Tne, Tma, Pfu, Tfl, Tth, Tli, Stoffel fragment, Vent® and Deep Vent® DNA polymerase, KOD DNA polymerase, Tgo, JDF-3, and mutants, variants and derivatives thereof.</p>
    <p num="p-0055">Reverse transcriptases useful in the invention include, but are not limited to, reverse transcriptases from HIV, HTLV-1, HTLV-II, FeLV, FIV, SIV, AMV, MMTV, MoMuLV and other retroviruses (see Levin (1997) Cell, 88:5-8; Verma (1977) Biochim. Biophys. Acta, 473:1-38; Wu et al. (1975) CRC Crit. Rev. Biochem, 3:289-347).</p>
    <p num="p-0056">C. Surfaces—In a preferred embodiment, nucleic acid template molecules are attached to a solid support (“substrate”). Substrates for use in the invention can be two-or three-dimensional and can comprise a planar surface (e.g., a glass slide) or can be shaped. A substrate can include glass (e.g., controlled pore glass (CPG)), quartz, plastic (such as polystyrene (low cross-linked and high cross-linked polystyrene), polycarbonate, polypropylene and poly(methymethacrylate)), acrylic copolymer, polyamide, silicon, metal (e.g., alkanethiolate-derivatized gold), cellulose, nylon, latex, dextran, gel matrix (e.g., silica gel), polyacrolein, or composites.</p>
    <p num="p-0057">Suitable three-dimensional substrates include, for example, spheres, microparticles, beads, membranes, slides, plates, micromachined chips, tubes (e.g., capillary tubes), microwells, microfluidic devices, channels, filters, or any other structure suitable for anchoring a nucleic acid. Substrates can include planar arrays or matrices capable of having regions that include populations of template nucleic acids or primers. Examples include nucleoside-derivatized CPG and polystyrene slides; derivatized magnetic slides; polystyrene grafted with polyethylene glycol, and the like.</p>
    <p num="p-0058">In one embodiment, a substrate is coated to allow optimum optical processing and nucleic acid attachment. Substrates for use in the invention can also be treated to reduce background. Exemplary coatings include epoxides, and derivatized epoxides (e.g., with a binding molecule, such as streptavidin). The surface can also be treated to improve the positioning of attached nucleic acids (e.g., nucleic acid template molecules, primers, or template molecule/primer duplexes) for analysis. As such, a surface according to the invention can be treated with one or more charge layers (e.g., a negative charge) to repel a charged molecule (e.g., a negatively charged labeled nucleotide). For example, a substrate according to the invention can be treated with polyallylamine followed by polyacrylic acid to form a polyelectrolyte multilayer. The carboxyl groups of the polyacrylic acid layer are negatively charged and thus repel negatively charged labeled nucleotides, improving the positioning of the label for detection. Coatings or films applied to the substrate should be able to withstand subsequent treatment steps (e.g., photoexposure, boiling, baking, soaking in warm detergent-containing liquids, and the like) without substantial degradation or disassociation from the substrate.</p>
    <p num="p-0059">Examples of substrate coatings include, vapor phase coatings of 3-aminopropyltrimethoxysilane, as applied to glass slide products, for example, from Erie Glass (Portsmouth, N.H.). In addition, generally, hydrophobic substrate coatings and films aid in the uniform distribution of hydrophilic molecules on the substrate surfaces. Importantly, in those embodiments of the invention that employ substrate coatings or films, the coatings or films that are substantially non-interfering with primer extension and detection steps are preferred. Additionally, it is preferable that any coatings or films applied to the substrates either increase template molecule binding to the substrate or, at least, do not substantially impair template binding.</p>
    <p num="p-0060">Various methods can be used to anchor or immobilize the primer to the surface of the substrate. The immobilization can be achieved through direct or indirect bonding to the surface. The bonding can be by covalent linkage. See, Joos et al. (1997) Analytical Biochemistry, 247:96-101; Oroskar et al. (1996) Clin. Chem., 42:1547-1555; and Khandjian (1986) Mol. Bio. Rep., 11:107-11. A preferred attachment is direct amine bonding of a terminal nucleotide of the template or the primer to an epoxide integrated on the surface. The bonding also can be through non-covalent linkage. For example, biotin-streptavidin (Taylor et al. (1991) J. Phys. D: Appl. Phys., 24:1443) and digoxigenin with anti-digoxigenin (Smith et al. (1992) Science, 253:11220, are common tools for anchoring nucleic acids to surfaces and parallels. Alternatively, the attachment can be achieved by anchoring a hydrophobic chain into a lipid monolayer or bilayer. Other methods known in the art for attaching nucleic acid molecules to substrates can also be used.</p>
    <p num="p-0061">D. Detection—Any detection method may be used that is suitable for the type of label employed. Thus, exemplary detection methods include radioactive detection, optical absorbance detection, e.g., UV-visible absorbance detection, optical emission detection, e.g., fluorescence or chemiluminescence. For example, extended primers can be detected on a substrate by scanning all or portions of each substrate simultaneously or serially, depending on the scanning method used. For fluorescence labeling, selected regions on a substrate may be serially scanned one-by-one or row-by-row using a fluorescence microscope apparatus, such as described in Fodor (U.S. Pat. No. 5,445,934) and Mathies et al. (U.S. Pat. No. 5,091,652). Devices capable of sensing fluorescence from a single molecule include the scanning tunneling microscope (siM) and the atomic force microscope (AFM). Hybridization patterns may also be scanned using a CCD camera (e.g., Model TE/CCD512SF, Princeton Instruments, Trenton, N.J.) with suitable optics (Ploem, in Fluorescent and Luminescent Probes for Biological Activity, Mason (ed.), Academic Press, Landon, pp. 1-11 (1993), such as described in Yershov et al. (1996) Proc. Natl. Acad. Sci., 93:4913, or may be imaged by TV monitoring. For radioactive signals, a Phosphorlmager™ device can be used (Johnston et al. (1990) Electrophoresis, 13:566; Drmanac et al. (1992) Electrophoresis, 13:566). Other commercial suppliers of imaging instruments include General Scanning Inc., (Watertown, Mass.; genscan.com), Genix Technologies (Waterloo, Ontario, Canada; confocal.com), and Applied Precision Inc. Such detection methods are particularly useful to achieve simultaneous scanning of multiple attached template nucleic acids.</p>
    <p num="p-0062">A number of approaches can be used to detect incorporation of fluorescently-labeled nucleotides into a single nucleic acid molecule. Optical setups include near-field scanning microscopy, far-field confocal microscopy, wide-field epi-illumination, light scattering, dark field microscopy, photoconversion, single and/or multiphoton excitation, spectral wavelength discrimination, fluorophore identification, evanescent wave illumination, and total internal reflection fluorescence (TIRF) microscopy. In general, certain methods involve detection of laser-activated fluorescence using a microscope equipped with a camera. Suitable photon detection systems include, but are not limited to, photodiodes and intensified CCD cameras. For example, an intensified charge couple device (ICCD) camera can be used. The use of an ICCD camera to image individual fluorescent dye molecules in a fluid near a surface provides numerous advantages. For example, with an ICCD optical setup, it is possible to acquire a sequence of images (“movies”) of fluorophores.</p>
    <p num="p-0063">Some embodiments of the present invention use TIRF microscopy for two-dimensional imaging. TIRF microscopy uses totally internally reflected excitation light and is well known in the art. See, e.g., nikon-instruments.jp/eng/page/products/tirf.aspx. In certain embodiments, detection is carried out using evanescent wave illumination and total internal reflection fluorescence microscopy. An evanescent light field can be set up at the surface, for example, to image fluorescently-labeled nucleic acid molecules. When a laser beam is totally reflected at the interface between a liquid and a solid substrate (e.g., glass), the excitation light beam penetrates only a short distance into the liquid. The optical field does not end abruptly at the reflective interface, but its intensity falls off exponentially with distance. This surface electromagnetic field, called the “evanescent wave”, can selectively excite fluorescent molecules in the liquid near the interface. The thin evanescent optical field at the interface provides low background and facilitates the detection of single molecules with high signal-to-noise ratio at visible wavelengths.</p>
    <p num="p-0064">The evanescent field also can image fluorescently-labeled nucleotides upon their incorporation into the attached template/primer complex in the presence of a polymerase. Total internal reflectance fluorescence microscopy is then used to visualize the attached template/primer duplex and/or the incorporated nucleotides with single molecule resolution.</p>
    <p num="p-0065">E. Data Analysis, Mapping, and Sequence Assembly—Data analysis of two reads on a single molecule template presents a variety of options. If the two-pass reads differ by more than, for example, 1 or 2 bases the reads should be discarded as arising from unresolved templates or surface contamination near a single template. A composite tag can be constructed of the two or more originating reads, applying high “quality” scores to those bases in agreement. Positioning of the composite tag over the reference can be done with high confidence, assigning any disagreement to a mutation of the sample if a long enough fraction of the tag aligns to give unambiguous alignment. With this exceptional accuracy, ˜99.7% or greater, virtually every tag covering a mutation should give the same answer for haploid samples, 50% for diploid samples SNP positions, and an excellent rare mutation detection with coverage as low as 10×. Alternatively, each read of a tag could be aligned separately. These results can be compared to a separate alignment of each individual read over the reference. Positions of agreement within a small error budget, 1 or 2 errors, are scored for best agreements between the tags. A modified Smith-Waterman alignment strategy using the composite tag with base scores may be used. For example paired-end read alignments and assembly, see, e.g., Batzoglou et al. (2002) Genome Res., 12(1):177-189.</p>
    <p num="p-0066">The following Example provides illustrative embodiments of the invention and does not in any way limit the invention.</p>
    <heading>EXAMPLES</heading> <heading>Example 1</heading> <heading>Single Molecule Sequencing of the M13 Genome</heading> <p num="p-0067">The 7249 nucleotide genome of the bacteriophage M13mp18 was sequenced using a single molecule system of the invention. Purified, single-stranded viral M13mp18 genomic DNA was obtained from New England Biolabs. Approximately 25 μg of M13DNA was digested to an average fragment size of 250 bp with 0.1 U Dnase I (New England Biolabs) for 10 minutes at 37° C. Digested DNA fragment sizes were estimated by running an aliquot of the digestion mixture on a precast denaturing (TBE-Urea) 10% polyacrylamide gel (Novagen) and staining with SYBR Gold (Invitrogen/Molecular Probes). The DNase I-digested genomic DNA was filtered through a YM10 ultrafiltration spin column (Millipore) to remove small digestion products less than about 30 nt. Approximately 20 pmol of the filtered DNase I digest was then polyadenylated with terminal transferase according to known methods (Roychoudhury, R and Wu, R. 1980, Terminal transferase-catalyzed addition of nucleotides to the 3′ termini of DNA. Methods Enzymol. 65(1):43-62). The average dA tail length was 50±5 nucleotides. Terminal transferase was then used to label the fragments with Cy3-ddUTP. The resulting fragments were again filtered with a YM10 ultrafiltration spin column to remove free nucleotides and stored in ddH<sub>2</sub>O at −20° C.</p>
    <p num="p-0068">Epoxide-coated glass slides were prepared for oligo attachment. Epoxide-functionalized 40 mm diameter #1.5 glass cover slips (slides) were obtained from Erie Scientific (Salem, N.H.). The slides were preconditioned by soaking in 3×SSC for 15 minutes at 37° C. Next, a 500 pM aliquot of 5′ aminated polydT(50) (polythymidine of 50 bp in length with a 5′ terminal amine) was incubated with each slide for 30 minutes at room temperature in a volume of 80 ml. The resulting slides had polydT(50) primer attached by direct amine linkage to the epoxide. The slides were then treated with phosphate (1 M) for 4 hours at room temperature in order to passivate the surface. Slides were then stored in a polymerase rinse buffer (20 mM Tris, 100 mM NaCl, 0.001% Triton X-100 pH 8.0) until they were used for sequencing.</p>
    <p num="p-0069">For sequencing, the slides were placed in a modified FCS2 flow cell (Bioptechs, Butler, Pa.) using a 50 μm thick gasket. The flow cell was placed on a movable stage that is part of a high-efficiency fluorescence imaging system built around a Nikon TE-2000 inverted microscope equipped with a total internal reflection (TIR) objective. The slide was then rinsed with HEPES buffer with 100 mM NaCl and equilibrated to a temperature of 50° C. An aliquot of the M13 template fragments described above was diluted in 3×SSC to a final concentration of 1.2 nM. A 100 μl aliquot was placed in the flow cell and incubated on the slide for 15 minutes. After incubation, the flow cell was rinsed with 1×SSC/HEPES/0.1% SDS followed by HEPES/NaCl. A passive vacuum apparatus was used to pull fluid across the flow cell. The resulting slide contained M13 template/oligo(dT) primer duplex. The temperature of the flow cell was then reduced to 37° C. for sequencing and the objective was brought into contact with the flow cell.</p>
    <p num="p-0070">For sequencing, cytosine triphosphate, guanidine triphosphate, adenine triphosphate, and uracil triphosphate, each having a cyanine-5 label (at the 7-deaza position for ATP and GTP and at the C5 position for CTP and UTP (PerkinElmer)) were stored separately in buffer containing 20 mM Tris-HCl, pH 8.8, 10 mM MgSO<sub>4</sub>, 10 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 10 mM HCl, and 0.1% Triton X-100, and 100U Klenow exo<sup>−</sup> polymerase (NEN). Sequencing proceeded as follows.</p>
    <p num="p-0071">First, initial imaging was used to determine the positions of duplex on the epoxide surface. The Cy3 label attached to the M13 templates was imaged by excitation using a laser tuned to 532 nm radiation (Verdi V-2 Laser, Coherent, Inc., Santa Clara, Calif.) in order to establish duplex position. For each slide only single fluorescent molecules that were imaged in this step were counted. Imaging of incorporated nucleotides as described below was accomplished by excitation of a cyanine-5 dye using a 635 nm radiation laser (Coherent). 5 μM Cy5CTP was placed into the flow cell and exposed to the slide for 2 minutes. After incubation, the slide was rinsed in 1×SSC/15 mM HEPES/0.1% SDS/pH 7.0 (“SSC/HEPES/SDS”) (15 times in 60 μl volumes each, followed by 150 mM HEPES/150 mM NaCl/pH 7.0 (“HEPES/NaCl”) (10 times at 60 μl volumes). An oxygen scavenger containing 30% acetonitrile and scavenger buffer (134 μl HEPES/NaCl, 24 μl 100 mM Trolox in MES, pH6.1, 10 μl DABCO in MES, pH6.1, 8 μl 2M glucose, 20 μl NaI (50 mM stock in water), and 4 μl glucose oxidase) was next added. The slide was then imaged (500 frames) for 0.2 seconds using an Inova 301K laser (Coherent) at 647 nm, followed by green imaging with a Verdi V-2 laser (Coherent) at 532 nm for 2 seconds to confirm duplex position. The positions having detectable fluorescence were recorded. After imaging, the flow cell was rinsed 5 times each with SSC/HEPES/SDS (60 μl) and HEPES/NaCl (60 μl). Next, the cyanine-5 label was cleaved off incorporated CTP by introduction into the flow cell of 50 mM TCEP for 5 minutes, after which the flow cell was rinsed 5 times each with SSC/HEPES/SDS (60 μl) and HEPES/NaCl (60 μl). The remaining nucleotide was capped with 50 mM iodoacetamide for 5 minutes followed by rinsing 5 times each with SSC/HEPES/SDS (60 μl) and HEPES/NaCl (60 μl ). The scavenger was applied again in the manner described above, and the slide was again imaged to determine the effectiveness of the cleave/cap steps and to identify non-incorporated fluorescent objects.</p>
    <p num="p-0072">The procedure described above was then conducted with the following addition of nucleotides: 100 nM Cy5dATP, followed by 100 nM Cy5dGTP, and finally 500 nM Cy5dUTP. By far the most common substitution error is G for A in some sequence contexts. This type of misincorporation may be minimized by base addition order, i.e., always adding dATP-Cy5 just before dGTP-Cy5. In this way, the vast majority of “A” sites (T on the template) are filled, thus avoiding such a misincorporation. The procedure (expose to nucleotide, polymerase, rinse, scavenger, image, rinse, cleave, rinse, cap, rinse, scavenger, final image) was repeated as described for ATP, GTP, and UTP except that Cy5dUTP was incubated for 5 minutes instead of 2 minutes. Uridine was used instead of Thymidine due to the fact that the Cy5 label was incorporated at the position normally occupied by the methyl group in Thymidine triphosphate, thus turning the dTTP into dUTP. All 64 cycles (C, A, G, U) were conducted as described in this and the preceding paragraph.</p>
    <p num="p-0073">Once 64 cycles were completed, the image stack data (i.e., the single molecule sequences obtained from the various surface-bound duplex) were aligned to the M13 reference sequence. The image data obtained was compressed to collapse homopolymeric regions. Thus, the sequence “TCAAAGC” would be represented as “TCAGC” in the data tags used for alignment. Similarly, homopolymeric regions in the reference sequence were collapsed for alignment. The sequencing protocol described above resulted in an aligned M13 sequence with an accuracy of between 98.8% and 99.96% (depending on depth of coverage). The individual single molecule sequence read lengths obtained ranged from 2 to 33 consecutive nucleotides with about 12.6 consecutive nucleotides being the average length.</p>
    <p num="p-0074">The alignment algorithm matched sequences obtained as described above with the actual M13 linear sequence. Placement of obtained sequence on M13 was based upon the best match between the obtained sequence and a portion of M13 of the same length, taking into consideration 0, 1, or 2 possible errors. All obtained 9-mers with 0 errors (meaning that they exactly matched a 9-mer in the M13 reference sequence) were first aligned with M13. Then 10-, 11-, and 12-mers with 0 or 1 error were aligned. Finally, all 11-mers or greater with 0, 1, or 2 errors were aligned. At a coverage depth of greater than or equal to one, 5,001 bases of the 5,066 base M13 collapsed genome were covered at an accuracy of 98.8%. Similarly, at a coverage depth of greater than or equal to five, 83.6% of the genome was covered at an accuracy of 99.3%, at a depth ofgreater than or equal to ten, 51.9% of the genome was covered at an accuracy of 99.96%. The average coverage depth was 12.6 nucleotides.</p>
    <heading>Example 2</heading> <heading>Length distribution of Strands with Matched Reads</heading> <p num="p-0075">Paired-end reads were obtained using experimental procedures essentially as described in Example 1. The results, representing read length distribution, are shown in <figref idrefs="DRAWINGS">FIG. 6</figref>, This distribution arises from the spontaneous cessation of sequence accumulation. Two origins are responsible, primer and primer-template loss, dominated by low level nuclease activity, and spontaneous termination of synthesis activity. The loss process is relatively easy to characterize, does not seem to depend on strand growth, and is likely to become insignificant as reagent processing it perfected. The mechanisms of spontaneous termination are less understood at this time. Template damage, either before or caused by the sequencing process does not seem to be a dominant cause. There is some evidence that the residual linker arm after dye cleavage (so-called “scars”) may be a substantial factor. The informatic leverage of paired reads, each longer than 30 bases is marginal until those reads exceed ˜150 bases. The advantages of these much longer reads for de novo assembly may not be sufficient to justify the system time.</p>
    <p num="p-0076">All publications, patents, patent applications, and biological sequences cited in this disclosure are incorporated by reference in their entirety.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6210891">US6210891</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 1997</td><td class="patent-data-table-td patent-date-value">Apr 3, 2001</td><td class="patent-data-table-td ">Pyrosequencing Ab</td><td class="patent-data-table-td ">Method of sequencing DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6258568">US6258568</a></td><td class="patent-data-table-td patent-date-value">Dec 22, 1997</td><td class="patent-data-table-td patent-date-value">Jul 10, 2001</td><td class="patent-data-table-td ">Pyrosequencing Ab</td><td class="patent-data-table-td ">Using polymerase chain reaction to extend a primer and release inorganic pyrophosphate, then detecting release of inorganic phosphate to identify base complementary to target position; removing unincorporated nucleotides using enzyme</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6274320">US6274320</a></td><td class="patent-data-table-td patent-date-value">Sep 16, 1999</td><td class="patent-data-table-td patent-date-value">Aug 14, 2001</td><td class="patent-data-table-td ">Curagen Corporation</td><td class="patent-data-table-td ">Method of sequencing a nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7169560">US7169560</a></td><td class="patent-data-table-td patent-date-value">May 24, 2004</td><td class="patent-data-table-td patent-date-value">Jan 30, 2007</td><td class="patent-data-table-td ">Helicos Biosciences Corporation</td><td class="patent-data-table-td ">Short cycle methods for sequencing polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7282337">US7282337</a></td><td class="patent-data-table-td patent-date-value">Apr 14, 2006</td><td class="patent-data-table-td patent-date-value">Oct 16, 2007</td><td class="patent-data-table-td ">Helicos Biosciences Corporation</td><td class="patent-data-table-td ">Sequencing a portion of a template and a portion of template complementary sequence.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7575865">US7575865</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 1, 2005</td><td class="patent-data-table-td patent-date-value">Aug 18, 2009</td><td class="patent-data-table-td ">454 Life Sciences Corporation</td><td class="patent-data-table-td ">Methods of amplifying and sequencing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7601499">US7601499</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 6, 2006</td><td class="patent-data-table-td patent-date-value">Oct 13, 2009</td><td class="patent-data-table-td ">454 Life Sciences Corporation</td><td class="patent-data-table-td ">Paired end sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20060292611">US20060292611</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 2006</td><td class="patent-data-table-td patent-date-value">Dec 28, 2006</td><td class="patent-data-table-td ">Jan Berka</td><td class="patent-data-table-td ">Preparing a target nucleic acid fragments to produce a smaller nucleic acid which comprises the two ends of the target nucleic acid; for rapid cloning, sequencing, or amplification; library may be constructed in vitro without the use of prokaryotic or eukaryotic host cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070070349">US20070070349</a></td><td class="patent-data-table-td patent-date-value">Sep 23, 2005</td><td class="patent-data-table-td patent-date-value">Mar 29, 2007</td><td class="patent-data-table-td ">Helicos Biosciences Corporation</td><td class="patent-data-table-td ">Optical train and method for TIRF single molecule detection and analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070087362">US20070087362</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 16, 2006</td><td class="patent-data-table-td patent-date-value">Apr 19, 2007</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Plurality of beads immobilized in a semi-solid medium to form an array; miniaturized, high density, clonal bead-based arrays; polymerase colony (polony) technology; hybridization, replication, genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005003375A2?cl=en">WO2005003375A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 28, 2004</td><td class="patent-data-table-td patent-date-value">Jan 13, 2005</td><td class="patent-data-table-td ">454 Corp</td><td class="patent-data-table-td ">Methods of amplifying and sequencing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2007010252A1?cl=en">WO2007010252A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 20, 2006</td><td class="patent-data-table-td patent-date-value">Jan 25, 2007</td><td class="patent-data-table-td ">Solexa Ltd</td><td class="patent-data-table-td ">Method for sequencing a polynucleotide template</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2007120859A2?cl=en">WO2007120859A2</a></td><td class="patent-data-table-td patent-date-value">Apr 13, 2007</td><td class="patent-data-table-td patent-date-value">Oct 25, 2007</td><td class="patent-data-table-td ">Timothy D Harris</td><td class="patent-data-table-td ">Methods for increasing accuracy of nucleic acid sequencing</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bennett et al.; "<a href='http://scholar.google.com/scholar?q="Toward+the+1%2C000+Dollars+Human+Genome"'>Toward the 1,000 Dollars Human Genome</a>"; Pharmacogenomics, 6:373-382 (2005).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Edwards et al.; "<a href='http://scholar.google.com/scholar?q="Automated+DNA+Sequencing+of+the+Human+HPRT+Locus"'>Automated DNA Sequencing of the Human HPRT Locus</a>"; Genomics; 6:593-608 (1990).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Karow, Julia; "<a href='http://scholar.google.com/scholar?q="As+Users+Demand+Paired-End+Sequencing%2C+454%2C+Illumina%2C+and+ABI+Work+on+New+Kits"'>As Users Demand Paired-End Sequencing, 454, Illumina, and ABI Work on New Kits</a>"; GenomeWeb LLC; print-out (3 pp.) downloaded Oct. 19, 2008 from http://www.in-sequence.com/issues/1-9/features/138789-1. html (2008).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Karow, Julia; "<a href='http://scholar.google.com/scholar?q="As+Users+Demand+Paired-End+Sequencing%2C+454%2C+Illumina%2C+and+ABI+Work+on+New+Kits"'>As Users Demand Paired-End Sequencing, 454, Illumina, and ABI Work on New Kits</a>"; GenomeWeb LLC; print-out (3 pp.) downloaded Oct. 19, 2008 from http://www.in-sequence.com/issues/1—9/features/138789-1. html (2008).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Print-out (1 page) downloaded Oct. 19, 2008 from http://yahoo.brand.edgar-online.com/EFX-dll/EDGARpro.dll? FetchFilingHTML1?SessionID=M6WLWdR4tzhwJch&amp;ID=537360; "<a href='http://scholar.google.com/scholar?q="The+Current+Genetics+and+Genome+Market-Ample+Room+for+Multiple+Players+%26amp%3B+Applications"'>The Current Genetics and Genome Market-Ample Room for Multiple Players &amp; Applications</a>" marked as Appendix A.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Print-out (1 page) downloaded Oct. 19, 2008 from http://yahoo.brand.edgar-online.com/EFX—dll/EDGARpro.dll? FetchFilingHTML1?SessionID=M6WLWdR4tzhwJch&amp;ID=537360; "<a href='http://scholar.google.com/scholar?q="The+Current+Genetics+and+Genome+Market%E2%80%94Ample+Room+for+Multiple+Players+%26amp%3B+Applications"'>The Current Genetics and Genome Market—Ample Room for Multiple Players &amp; Applications</a>" marked as Appendix A.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Skiena et al.; Presentation Entitled "<a href='http://scholar.google.com/scholar?q="Sequence+Assembly+for+Short+Paired-Read+Technologies"'>Sequence Assembly for Short Paired-Read Technologies</a>"; Department of Computer Science, State University of New York, Stony Brook, NY; (45 pp.); downloaded on Oct. 24, 2008 from http://www.cs.sunysb.edu/.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Specification and pending claims from co-pending co-owned U.S. Appl. No. 11/928,695.</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8143030">US8143030</a></td><td class="patent-data-table-td patent-date-value">Mar 27, 2009</td><td class="patent-data-table-td patent-date-value">Mar 27, 2012</td><td class="patent-data-table-td ">Pacific Biosciences Of California, Inc.</td><td class="patent-data-table-td ">Intermittent detection during analytical reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8153375">US8153375</a></td><td class="patent-data-table-td patent-date-value">Mar 27, 2009</td><td class="patent-data-table-td patent-date-value">Apr 10, 2012</td><td class="patent-data-table-td ">Pacific Biosciences Of California, Inc.</td><td class="patent-data-table-td ">Compositions and methods for nucleic acid sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8236499">US8236499</a></td><td class="patent-data-table-td patent-date-value">Mar 27, 2009</td><td class="patent-data-table-td patent-date-value">Aug 7, 2012</td><td class="patent-data-table-td ">Pacific Biosciences Of California, Inc.</td><td class="patent-data-table-td ">Methods and compositions for nucleic acid sample preparation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8236532">US8236532</a></td><td class="patent-data-table-td patent-date-value">Dec 17, 2009</td><td class="patent-data-table-td patent-date-value">Aug 7, 2012</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Multibase delivery for long reads in sequencing by synthesis protocols</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8309330">US8309330</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2011</td><td class="patent-data-table-td patent-date-value">Nov 13, 2012</td><td class="patent-data-table-td ">Pacific Biosciences Of California, Inc.</td><td class="patent-data-table-td ">Diagnostic sequencing with small nucleic acid circles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8383369">US8383369</a></td><td class="patent-data-table-td patent-date-value">Sep 16, 2009</td><td class="patent-data-table-td patent-date-value">Feb 26, 2013</td><td class="patent-data-table-td ">Pacific Biosciences Of California, Inc.</td><td class="patent-data-table-td ">Intermittent detection during analytical reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8455193">US8455193</a></td><td class="patent-data-table-td patent-date-value">Feb 23, 2012</td><td class="patent-data-table-td patent-date-value">Jun 4, 2013</td><td class="patent-data-table-td ">Pacific Biosciences Of California, Inc.</td><td class="patent-data-table-td ">Compositions and methods for nucleic acid sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8476022">US8476022</a></td><td class="patent-data-table-td patent-date-value">Jul 10, 2012</td><td class="patent-data-table-td patent-date-value">Jul 2, 2013</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Method of making an array of nucleic acid colonies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8535886">US8535886</a></td><td class="patent-data-table-td patent-date-value">Jun 12, 2012</td><td class="patent-data-table-td patent-date-value">Sep 17, 2013</td><td class="patent-data-table-td ">Pacific Biosciences Of California, Inc.</td><td class="patent-data-table-td ">Methods and compositions for nucleic acid sample preparation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8628940">US8628940</a></td><td class="patent-data-table-td patent-date-value">Dec 30, 2010</td><td class="patent-data-table-td patent-date-value">Jan 14, 2014</td><td class="patent-data-table-td ">Pacific Biosciences Of California, Inc.</td><td class="patent-data-table-td ">Intermittent detection during analytical reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8709729">US8709729</a></td><td class="patent-data-table-td patent-date-value">May 29, 2013</td><td class="patent-data-table-td patent-date-value">Apr 29, 2014</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Method of making an array of nucleic acid colonies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012037531A1?cl=en">WO2012037531A1</a></td><td class="patent-data-table-td patent-date-value">Sep 16, 2011</td><td class="patent-data-table-td patent-date-value">Mar 22, 2012</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Capture probes immobilizable via l-nucleotide tail</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012061412A1?cl=en">WO2012061412A1</a></td><td class="patent-data-table-td patent-date-value">Nov 1, 2011</td><td class="patent-data-table-td patent-date-value">May 10, 2012</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Integrated capture and amplification of target nucleic acid for sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013116774A1?cl=en">WO2013116774A1</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2013</td><td class="patent-data-table-td patent-date-value">Aug 8, 2013</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Asymmetric hairpin target capture oligomers</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006110">435/6.11</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024330">536/24.33</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc702/defs702.htm&usg=AFQjCNEwDgPZO8lzVvNBu0VAYuT88y7k4g#C702S020000">702/20</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025300">536/25.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025320">536/25.32</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S287200">435/287.2</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12M0001340000">C12M1/34</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021040000">C07H21/04</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021000000">C07H21/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0031000000">G01N31/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=97GGBgABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6869">C12Q1/6869</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68E</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Feb 3, 2014</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 28, 2013</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">COMPLETE GENOMICS, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">LICENSE;ASSIGNOR:FLUIDIGM CORPORATION;REEL/FRAME:030714/0686</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">FLUIDIGM CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:HELICOS BIOSCIENCES CORPORATION;REEL/FRAME:030714/0546</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20130628</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ILLUMINA, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">LICENSE;ASSIGNOR:FLUIDIGM CORPORATION;REEL/FRAME:030714/0783</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">PACIFIC BIOSCIENCES OF CALIFORNIA, INC., CALIFORNI</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">LICENSE;ASSIGNOR:FLUIDIGM CORPORATION;REEL/FRAME:030714/0598</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">SEQLL, LLC, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">LICENSE;ASSIGNOR:FLUIDIGM CORPORATION;REEL/FRAME:030714/0633</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 23, 2012</td><td class="patent-data-table-td ">FPB1</td><td class="patent-data-table-td ">Expired due to reexamination which canceled all claims</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 17, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BROWN RUDNICK LLP, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">NOTICE OF ATTORNEY S LIEN;ASSIGNOR:HELICOS BIOSCIENCES CORPORATION;REEL/FRAME:028060/0898</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120417</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 18, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120113</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">HELICOS BIOSCIENCES CORPORATION, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">RELEASE BY SECURED PARTY;ASSIGNOR:GENERAL ELECTRIC CAPITAL CORPORATION;REEL/FRAME:027549/0565</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 22, 2011</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20110127</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 22, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENERAL ELECTRIC CAPITAL CORPORATION, MARYLAND</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:HELICOS BIOSCIENCES CORPORATION;REEL/FRAME:025388/0347</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20101116</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 21, 2008</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">HELICOS BIOSCIENCES CORPORATION, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:HARRIS, TIMOTHY D.;REEL/FRAME:020543/0592</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20080220</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U1e1VowU5SLMfVw089nMSMfxZfj-A\u0026id=97GGBgABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U0SVyA0ooEcAlFIw3zHRm3kCMV_Mw\u0026id=97GGBgABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U2_qfIG43FuEkLUPelXAwxd6urgkw","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Enhancing_resolution_of_sequence_analysi.pdf?id=97GGBgABERAJ\u0026output=pdf\u0026sig=ACfU3U3v7IJGhKeWAlYx_YsuacwCA-Mf2Q"},"sample_url":"http://www.google.com/patents/reader?id=97GGBgABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>