<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US7875440 - Using polymerase with reduced exonuclease activity as replicative agent in ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_4ff636b3d23669b7103f3b3a3a18b4cd/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_4ff636b3d23669b7103f3b3a3a18b4cd__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Using polymerase with reduced exonuclease activity as replicative agent in DNA sequencing"><meta name="DC.contributor" content="Peter Williams" scheme="inventor"><meta name="DC.contributor" content="Thomas J. Taylor" scheme="inventor"><meta name="DC.contributor" content="Daniel J. B. Williams" scheme="inventor"><meta name="DC.contributor" content="Ian Gould" scheme="inventor"><meta name="DC.contributor" content="Mark A. Hayes" scheme="inventor"><meta name="DC.contributor" content="Arizona Board Of Regents" scheme="assignee"><meta name="DC.date" content="2007-10-30" scheme="dateSubmitted"><meta name="DC.description" content="The present invention relates to a novel method for analyzing nucleic acid sequences based on real-time detection of DNA polymerase-catalyzed incorporation of each of the four nucleotide bases, supplied individually and serially in a microfluidic system, to a reaction cell containing a template system comprising a DNA fragment of unknown sequence and an oligonucleotide primer. Incorporation of a nucleotide base into the template system can be detected by any of a variety of methods including but not limited to fluorescence and chemiluminescence detection. Alternatively, microcalorimetic detection of the heat generated by the incorporation of a nucleotide into the extending template system using thermopile, thermistor and refractive index measurements can be used to detect extension reactions."><meta name="DC.date" content="2011-1-25" scheme="issued"><meta name="DC.relation" content="EP:0223618:A2" scheme="references"><meta name="DC.relation" content="US:5436149" scheme="references"><meta name="DC.relation" content="WO:1989009283:A1" scheme="references"><meta name="DC.relation" content="WO:1991006678:A1" scheme="references"><meta name="citation_patent_number" content="US:7875440"><meta name="citation_patent_application_number" content="US:11/929,141"><link rel="canonical" href="http://www.google.com/patents/US7875440"/><meta property="og:url" content="http://www.google.com/patents/US7875440"/><meta name="title" content="Patent US7875440 - Using polymerase with reduced exonuclease activity as replicative agent in DNA sequencing"/><meta name="description" content="The present invention relates to a novel method for analyzing nucleic acid sequences based on real-time detection of DNA polymerase-catalyzed incorporation of each of the four nucleotide bases, supplied individually and serially in a microfluidic system, to a reaction cell containing a template system comprising a DNA fragment of unknown sequence and an oligonucleotide primer. Incorporation of a nucleotide base into the template system can be detected by any of a variety of methods including but not limited to fluorescence and chemiluminescence detection. Alternatively, microcalorimetic detection of the heat generated by the incorporation of a nucleotide into the extending template system using thermopile, thermistor and refractive index measurements can be used to detect extension reactions."/><meta property="og:title" content="Patent US7875440 - Using polymerase with reduced exonuclease activity as replicative agent in DNA sequencing"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("fuLoU838Msq0yATr2ICwCg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407291699.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("LUX"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("fuLoU838Msq0yATr2ICwCg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407291699.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("LUX"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us7875440?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US7875440"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=3hO8BgABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS7875440&amp;usg=AFQjCNH1lPWrcPxf5OsHzpCx3OzhqJa9rg" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US7875440.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US7875440.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20080213770"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US7875440"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US7875440" style="display:none"><span itemprop="description">The present invention relates to a novel method for analyzing nucleic acid sequences based on real-time detection of DNA polymerase-catalyzed incorporation of each of the four nucleotide bases, supplied individually and serially in a microfluidic system, to a reaction cell containing a template system...</span><span itemprop="url">http://www.google.com/patents/US7875440?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US7875440 - Using polymerase with reduced exonuclease activity as replicative agent in DNA sequencing</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US7875440 - Using polymerase with reduced exonuclease activity as replicative agent in DNA sequencing" title="Patent US7875440 - Using polymerase with reduced exonuclease activity as replicative agent in DNA sequencing"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US7875440 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 11/929,141</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jan 25, 2011</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Oct 30, 2007</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">May 1, 1998</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US8216514">US8216514</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8263364">US8263364</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8263365">US8263365</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20080213770">US20080213770</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20110183321">US20110183321</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20110183322">US20110183322</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20110183323">US20110183323</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20110294115">US20110294115</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120015353">US20120015353</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120258449">US20120258449</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20130288235">US20130288235</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20130295560">US20130295560</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">11929141, </span><span class="patent-bibdata-value">929141, </span><span class="patent-bibdata-value">US 7875440 B2, </span><span class="patent-bibdata-value">US 7875440B2, </span><span class="patent-bibdata-value">US-B2-7875440, </span><span class="patent-bibdata-value">US7875440 B2, </span><span class="patent-bibdata-value">US7875440B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Peter+Williams%22">Peter Williams</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Thomas+J.+Taylor%22">Thomas J. Taylor</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Daniel+J.+B.+Williams%22">Daniel J. B. Williams</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Ian+Gould%22">Ian Gould</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Mark+A.+Hayes%22">Mark A. Hayes</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Arizona+Board+Of+Regents%22">Arizona Board Of Regents</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7875440.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7875440.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7875440.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (4),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (4),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (13),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (5)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/7875440&usg=AFQjCNFP7jeI-jxmlbQNgFrRDME_0ZogKA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D7875440&usg=AFQjCNErhAba1nKDOwRYoBX09G0fww8IjA">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D7875440B2%26KC%3DB2%26FT%3DD&usg=AFQjCNER2Si8VHm7yfsO4EVo6fPAaSVCZw">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT130149782" lang="EN" load-source="ifi">Using polymerase with reduced exonuclease activity as replicative agent in DNA sequencing</invention-title></span><br><span class="patent-number">US 7875440 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA82679288" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">The present invention relates to a novel method for analyzing nucleic acid sequences based on real-time detection of DNA polymerase-catalyzed incorporation of each of the four nucleotide bases, supplied individually and serially in a microfluidic system, to a reaction cell containing a template system comprising a DNA fragment of unknown sequence and an oligonucleotide primer. Incorporation of a nucleotide base into the template system can be detected by any of a variety of methods including but not limited to fluorescence and chemiluminescence detection. Alternatively, microcalorimetic detection of the heat generated by the incorporation of a nucleotide into the extending template system using thermopile, thermistor and refractive index measurements can be used to detect extension reactions.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(16)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00000.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00000.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00013.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00013.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00014.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00014.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-D00015.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US7875440B2/US07875440-20110125-D00015.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(30)</span></span></div><div class="patent-text"><div mxw-id="PCLM34552719" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A method of DNA sequencing comprising the steps of:
<div class="claim-text">(a) providing a template system comprising at least one nucleic acid molecule of unknown sequence hybridized to a primer oligonucleotide in the presence of a DNA polymerase with reduced exonuclease activity;</div>
<div class="claim-text">(b) contacting the template system with a single type of deoxyribonucleotide under conditions which allow extension of the primer by incorporation of at least one deoxyribonucleotide to the 3′ end of the primer to form an extended primer;</div>
<div class="claim-text">(c) observing the template system in order to detect whether extension of the primer has occurred;</div>
<div class="claim-text">(d) detecting the number of deoxyribonucleotides incorporated into the primer;</div>
<div class="claim-text">(e) removing unincorporated deoxyribonucleotide; and</div>
<div class="claim-text">(f) repeating steps (a) through (e) to determine the nucleotide sequence of the nucleic acid molecule.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the at least one deoxyribonucleotide includes a chemiluminescent moiety comprising detecting whether extension of the primer has occurred by detecting a chemiluminescent signal emitted by the chemiluminescent moiety, and further comprising removing the chemiluminescent moiety from the template system.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the at least one deoxyribonucleotide includes a fluorescent moiety comprising detecting whether extension of the primer has occurred by detecting a fluorescent signal emitted by the fluorescent moiety, and further comprising removing the fluorescent moiety from the template system.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the at least one deoxyribonucleotide includes a fluorescent moiety comprising detecting whether extension of the primer has occurred by detecting a fluorescent signal emitted by the fluorescent moiety, and further comprising destroying the fluorescent signal without removal of the fluorescent moiety.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. The method of <claim-ref idref="CLM-00004">claim 4</claim-ref> wherein the fluorescent moiety is destroyed by reaction with compounds capable of extracting an electron from the excited state of the fluorescent moiety.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. The method of <claim-ref idref="CLM-00005">claim 5</claim-ref> wherein the compound is a diphenyliodonium salt.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> comprising detecting whether extension of the primer has occurred by detecting a change in the concentration of unincorporated deoxyribonucleotide.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein incorporation of the at least one deoxyribonucleotide generates heat, comprising detecting whether extension of the primer has occurred by detecting the heat generated by said incorporation.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The method of <claim-ref idref="CLM-00008">claim 8</claim-ref> wherein a thermopile is used to detect the generated heat.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The method of <claim-ref idref="CLM-00008">claim 8</claim-ref> wherein a thermistor is used to detect the generated heat.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the template system further includes a buffer wherein incorporation of the at least one deoxyribonucleotide generates heat which is absorbed by said buffer and further comprising measuring the refractive index of the buffer.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> comprising detecting whether extension of the primer has occurred by detecting the concentration of pyrophosphate released by addition of a deoxyribonucleotide to the 3′ end of the primer.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00013" num="00013" class="claim">
      <div class="claim-text">13. The method of <claim-ref idref="CLM-00012">claim 12</claim-ref> wherein the concentration of pyrophosphate is detected by hydrolyzing the pyrophosphate and measuring heat generated by hydrolysis of the pyrophosphate.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00014" num="00014" class="claim">
      <div class="claim-text">14. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the DNA polymerase is a T4 DNA polymerase with a substitution of amino acid residue Asp112 by Ala and Glu114 by Ala.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00015" num="00015" class="claim">
      <div class="claim-text">15. The method of <claim-ref idref="CLM-00011">claim 11</claim-ref> wherein the DNA polymerase further comprises a T4 DNA polymerase with a substitution of amino acid residue Ile417 by Val.</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00016" num="00016" class="claim">
      <div class="claim-text">16. A method of DNA sequencing comprising the steps of:
<div class="claim-text">(a) providing a template system comprising at least one nucleic acid molecule of unknown sequence hybridized to a primer oligonucleotide in the presence of a exonuclease deficient DNA polymerase;</div>
<div class="claim-text">(b) contacting the template system with a single type of deoxyribonucleotide under conditions which allow extension of the primer by incorporation of at least one deoxyribonucleotide to the 3′ end of the primer to form an extended primer;</div>
<div class="claim-text">(c) observing the template system in order to detect whether extension of the primer has occurred thereby identifying the deoxyribonucleotide added to the 3′ end of the primer;</div>
<div class="claim-text">(d) detecting the number of deoxyribonucleotides incorporated into the primer;</div>
<div class="claim-text">(e) removing unincorporated deoxyribonucleotide;</div>
<div class="claim-text">(f) contacting the template system with a mixture including an exonuclease proficient DNA polymerase, an exonuclease deficient DNA polymerase and the identified deoxyribonucleotide of step (b);</div>
<div class="claim-text">(g) removing the mixture of step (f); and</div>
<div class="claim-text">(h) repeating steps (a) through (g) to determine the nucleotide sequence of the nucleic acid molecule.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00017" num="00017" class="claim">
      <div class="claim-text">17. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref> wherein the at least one deoxyribonucleotide includes a fluorescent moiety comprising detecting whether extension of the primer has occurred by detecting a fluorescent signal emitted by the fluorescent moiety.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00018" num="00018" class="claim">
      <div class="claim-text">18. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref> wherein the at least one deoxyribonucleotide includes a fluorescent moiety comprising detecting whether extension of the primer has occurred by detecting a fluorescent signal emitted by the fluorescent moiety, and further comprising destroying the fluorescent signal without removal of the fluorescent moiety.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00019" num="00019" class="claim">
      <div class="claim-text">19. The method of <claim-ref idref="CLM-00018">claim 18</claim-ref> wherein the fluorescent moiety is destroyed by reaction with compounds capable of extracting an electron from the excited state of the fluorescent moiety.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00020" num="00020" class="claim">
      <div class="claim-text">20. The method of <claim-ref idref="CLM-00019">claim 19</claim-ref> wherein the compound is a diphenyliodonium salt.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00021" num="00021" class="claim">
      <div class="claim-text">21. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref> wherein the at least one deoxyribonucleotide includes a chemiluminescent moiety comprising detecting whether extension of the primer has occurred by detecting chemiluminescent signal emitted by the chemiluminescent moiety.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00022" num="00022" class="claim">
      <div class="claim-text">22. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref> comprising detecting whether extension of the primer has occurred by detecting a change in the concentration of unincorporated deoxyribonucleotide.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00023" num="00023" class="claim">
      <div class="claim-text">23. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref> wherein incorporation of the at least one deoxyribonucleotide generates heat comprising detecting whether extension of the primer has occurred by detecting heat generated by said incorporation.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00024" num="00024" class="claim">
      <div class="claim-text">24. The method of <claim-ref idref="CLM-00023">claim 23</claim-ref> wherein a thermopile is used to detect the generated heat.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00025" num="00025" class="claim">
      <div class="claim-text">25. The method of <claim-ref idref="CLM-00023">claim 23</claim-ref> wherein a thermistor is used to detect the generated heat.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00026" num="00026" class="claim">
      <div class="claim-text">26. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref> wherein the template system further includes a buffer wherein incorporation of the at least one deoxyribonucleotide generates heat which is absorbed by said buffer and further comprising measuring the refractive index of the buffer.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00027" num="00027" class="claim">
      <div class="claim-text">27. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref> comprising detecting whether extension of the primer has occurred by detecting the concentration of pyrophosphate released by incorporation of a deoxyribonucleotide to the 3′ end of the primer.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00028" num="00028" class="claim">
      <div class="claim-text">28. The method of <claim-ref idref="CLM-00027">claim 27</claim-ref> wherein the concentration of pyrophosphate is detected by hydrolyzing the pyrophosphate and measuring the heat generated by hydrolysis of the pyrophosphate.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00029" num="00029" class="claim">
      <div class="claim-text">29. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref> wherein the exonuclease deficient DNA polymerase is a T4 DNA polymerase with a substitution of amino acid residue Asp112 by Ala and Glu114 by Ala.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00030" num="00030" class="claim">
      <div class="claim-text">30. The method of <claim-ref idref="CLM-00026">claim 26</claim-ref> wherein the exonuclease deficient DNA polymerase further comprises a T4 DNA polymerase with a substitution of amino acid residue Ile417 by Val.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES40424251" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS REFERENCE TO RELATED APPLICATIONS</heading> <p num="p-0002">This patent application is a continuation of Ser. No. 09/941,882 filed Aug. 28, 2001, which is a continuation-in-part of Ser. No. 09/673,544 filed Feb. 26, 2001 and now abandoned, which is a continuation-in-part of PCT/US99/09616 filed Apr. 30, 1999 and claims the benefit of provisional application Ser. No. 60/083,840 filed May 1, 1998.</p>
    <heading>INTRODUCTION</heading> <p num="p-0003">The present invention relates to a novel method for analyzing nucleic acid sequences based on real-time detection of DNA polymerase-catalyzed incorporation of each of the four deoxynucleoside monophosphates, supplied individually and serially as deoxynucleoside triphosphates in a micro fluidic system, to a template system comprising a DNA fragment of unknown sequence and an oligonucleotide primer. Incorporation of a deoxynucleosidemonophosphate (dNMP) into the primer can be detected by any of a variety of methods including but not limited to fluorescence and chemiluminescence detection. Alternatively, microcalorimetic detection of the heat generated by the incorporation of a dNMP into the extending primer using thermopile, thermistor and refractive index measurements can be used to detect extension reactions. The present invention further provides a method for monitoring and correction of sequencing errors due to misincorporation or extension failure.</p>
    <p num="p-0004">The present invention provides a method for sequencing DNA that avoids electrophoretic separation of DNA fragments thus eliminating the problems associated with anomalous migration of DNA due to repeated base sequences or other self-complementary sequences which can cause single-stranded DNA to self-hybridize into hairpin loops, and also avoids current limitations on the size of fragments that can be read. The method of the invention can be utilized to determine the nucleotide sequence of genomic or cDNA fragments, or alternatively, as a diagnostic tool for sequencing patient derived DNA samples.</p>
    <heading>BACKGROUND OF INVENTION</heading> <p num="p-0005">Currently, two approaches are utilized for DNA sequence determination: the dideoxy chain termination method of Sanger (1977, Proc. Natl. Acad. Sci. 74:5463-5674) and the chemical degradation method of Maxam (1977, Proc. Natl. Acad. Sci. 74:560-564). The Sanger dideoxy chain termination method is the most widely used method and is the method upon which automated DNA sequencing machines rely. In the chain termination method, DNA polymerase enzyme is added to four separate reaction systems to make multiple copies of a template DNA strand in which the growth process has been arrested at each occurrence of an A, in one set of reactions, and a G, C, or T, respectively, in the other sets of reactions, by incorporating in each reaction system one nucleotide type lacking the 3′-OH on the deoxyribose at which chain extension occurs. This procedure produces a series of DNA fragments of different lengths, and it is the length of the extended DNA fragment that signals the position along the template strand at which each of four bases occur. To determine the nucleotide sequence, the DNA fragments are separated by high resolution gel electrophoresis and the order of the four bases is read from the gel.</p>
    <p num="p-0006">A major research goal is to derive the DNA sequence of the entire human genome. To meet this goal the need has developed for new genomic sequencing technology that can dispense with the difficulties of gel electrophoresis, lower the costs of performing sequencing reactions, including reagent costs, increase the speed and accuracy of sequencing, and increase the length of sequence that can be read in a single step. Potential improvements in sequencing speed may be provided by a commercialized capillary gel electrophoresis technique such as that described in Marshall and Pennisis (1998, Science 280:994-995). However, a major problem common to all gel electrophoresis approaches is the occurrence of DNA sequence compressions, usually arising from secondary structures in the DNA fragment, which result in anomalous migration of certain DNA fragments through the gel.</p>
    <p num="p-0007">As genomic information accumulates and the relationships between gene mutations and specific diseases are identified, there will be a growing need for diagnostic methods for identification of mutations. In contrast to the large scale methods needed for sequencing large segments of the human genome, what is needed for diagnostic methods are repetitive, low-cost, highly accurate techniques for resequencing of certain small isolated regions of the genome. In such instances, methods of sequencing based on gel electrophoresis readout become far too slow and expensive.</p>
    <p num="p-0008">When considering novel DNA sequencing techniques, the possibility of reading the sequence directly, much as the cell does, rather than indirectly as in the Sanger dideoxynucleotide approach, is a preferred goal. This was the goal of early unsuccessful attempts to determine the shapes of the individual nucleotide bases with scanning probe microscopes.</p>
    <p num="p-0009">Additionally, another approach for reading a nucleotide sequence directly is to treat the DNA with an exonuclease coupled with a detection scheme for identifying each nucleotide sequentially released as described in Goodwin, et al., (1995, Experimental Techniques of Physics 41:279-294). However, researchers using this technology are confronted with the enormous problem of detecting and identifying single nucleotide molecules as they are digested from a single DNA strand. Simultaneous exonuclease digestion of multiple DNA strands to yield larger signals is not feasible because the enzymes rapidly get out of phase, so that nucleotides from different positions on the different strands are released together, and the sequences become unreadable. It would be highly beneficial if some means of external regulation of the exonuclease could be found so that multiple enzyme molecules could be compelled to operate in phase. However, external regulation of an enzyme that remains docked to its polymeric substrate is exceptionally difficult, if not impossible, because after each digestion the next substrate segment is immediately present at the active site. Thus, any controlling signal must be present at the active site at the start of each reaction.</p>
    <p num="p-0010">A variety of methods may be used to detect the polymerase-catalyzed incorporation of deoxynucleoside monophosphates (dNMPs) into a primer at each template site. For example, the pyrophosphate released whenever DNA polymerase adds one of the four dNTPs onto a primer 3′ end may be detected using a chemiluminescent based detection of the pyrophosphate as described in Hyman E. D. (1988, Analytical Biochemistry 174:423-436) and U.S. Pat. No. 4,971,903. This approach has been utilized most recently in a sequencing approach referred to as “sequencing by incorporation” as described in Ronaghi (1996, Analytical Biochem. 242:84) and Ronaghi (1998, Science 281:363-365). However, there exist two key problems associated with this approach, destruction of unincorporated nucleotides and detection of pyrophosphate. The solution to the first problem is to destroy the added, unincorporated nucleotides using a dNTP-digesting enzyme such as apyrase. The solution to the second is the detection of the pyrophosphate using ATP sulfurylase to reconvert the pyrophosphate to ATP which can be detected by a luciferase chemiluminescent reaction as described in U.S. Pat. No. 4,971,903 and Ronaghi (1998, Science 281:363-365). Deoxyadenosine α-thiotriphosphate is used instead of dATP to minimize direct interaction of injected dATP with the luciferase.</p>
    <p num="p-0011">Unfortunately, the requirement for multiple enzyme reactions to be completed in each cycle imposes restrictions on the speed of this approach while the read length is limited by the impossibility of completely destroying unincorporated, non-complementary, nucleotides. If some residual amount of one nucleotide remains in the reaction system at the time when a fresh aliquot of a different nucleotide is added for the next extension reaction, there exists a possibility that some fraction of the primer strands will be extended by two or more nucleotides, the added nucleotide type and the residual impurity type, if these match the template sequence, and so this fraction of the primer strands will then be out of phase with the remainder. This out of phase component produces an erroneous incorporation signal which grows larger with each cycle and ultimately makes the sequence unreadable.</p>
    <p num="p-0012">A different direct sequencing approach uses dNTPs tagged at the 3′ OH position with four different colored fluorescent tags, one for each of the four nucleotides is described in Metzger, M. L., et al. (1994, Nucleic Acids Research 22:4259-4267). In this approach, the primer/template duplex is contacted with all four dNTPs simultaneously. Incorporation of a 3′ tagged NMP blocks further chain extension. The excess and unreacted dNTPs are flushed away and the incorporated nucleotide is identified by the color of the incorporated fluorescent tag. The fluorescent tag must then be removed in order for a subsequent incorporation reaction to occur. Similar to the pyrophosphate detection method, incomplete removal of a blocking fluorescent tag leaves some primer strands unextended on the next reaction cycle, and if these are subsequently unblocked in a later cycle, once again an out-of-phase signal is produced which grows larger with each cycle and ultimately limits the read length. To date, this method has so far been demonstrated to work for only a single base extension. Thus, this method is slow and is likely to be restricted to very short read lengths due to the fact that 99% efficiency in removal of the tag is required to read beyond 50 base pairs. Incomplete removal of the label results in out of phase extended DNA strands.</p>
    <heading>SUMMARY OF INVENTION</heading> <p num="p-0013">Accordingly, it is an object of the present invention to provide a novel method for determining the nucleotide sequence of a DNA fragment which eliminates the need for electrophoretic separation of DNA fragments. The inventive method, referred to herein as “reactive sequencing”, is based on detection of DNA polymerase catalyzed incorporation of each of the four nucleotide types, when deoxynucleoside triphosphates (dNTP's) are supplied individually and serially to a DNA primer/template system. The DNA primer/template system comprises a single stranded DNA fragment of unknown sequence, an oligonucleotide primer that forms a matched duplex with a short region of the single stranded DNA, and a DNA polymerase enzyme. The enzyme may either be already present in the template system, or may be supplied together with the dNTP solution.</p>
    <p num="p-0014">Typically a single deoxynucleoside triphosphate (dNTP) is added to the DNA primer template system and allowed to react. As used herein deoxyribonucleotide means and includes, in addition to dGTP, dCTP, dATP, dTTP, chemically modified versions of these deoxyribonucleotides or analogs thereof. Such chemically modified deoxyribonucleotides include but are not limited to those deoxyribonucleotides tagged with a fluorescent or chemiluminescent moiety. Analogs of deoxyribonucleotides that may be used include but are not limited to 7-deazapurine. The present invention additionally provides a method for improving the purity of deoxynucleotides used in the polymerase reaction.</p>
    <p num="p-0015">An extension reaction will occur only when the incoming dNTP base is complementary to the next unpaired base of the DNA template beyond the 3′ end of the primer. While the reaction is occurring, or after a delay of sufficient duration to allow a reaction to occur, the system is tested to determine whether an additional nucleotide derived from the added dNTP has been incorporated into the DNA primer/template system. A correlation between the dNTP added to the reaction cell and detection of an incorporation signal identifies the nucleotide incorporated into the primer/template. The amplitude of the incorporation signal identifies the number of nucleotides incorporated, and thereby quantifies single base repeat lengths where these occur. By repeating this process with each of the four nucleotides individually, the sequence of the template can be directly read in the 5′ to 3′ direction one nucleotide at a time.</p>
    <p num="p-0016">Detection of the polymerase mediated extension reaction and quantification of the extent of reaction can occur by a variety of different techniques, including but not limited to, microcalorimetic detection of the heat generated by the incorporation of a nucleotide into the extending duplex. Optical detection of an extension reaction by fluorescence or chemiluminescence may also be used to detect incorporation of nucleotides tagged with fluorescent or chemiluminescent entities into the extending duplex. Where the incorporated nucleotide is tagged with a fluorophore, excess unincorporated nucleotide is removed, and the template system is illuminated to stimulate fluorescence from the incorporated nucleotide. The fluorescent tag may then be cleaved and removed from the DNA template system before a subsequent incorporation cycle begins. A similar process is followed for chemiluminescent tags, with the chemiluminescent reaction being stimulated by introducing an appropriate reagent into the system, again after excess unreacted tagged dNTP has been removed; however, chemiluminescent tags are typically destroyed in the process of readout and so a separate cleavage and removal step following detection may not be required. For either type of tag, fluorescent or chemiluminescent, the tag may also be cleaved after incorporation and transported to a separate detection chamber for fluorescent or chemiluminescent detection. In this way, fluorescent quenching by adjacent fluorophore tags incorporated in a single base repeat sequence may be avoided. In addition, this may protect the DNA template system from possible radiation damage in the case of fluorescent detection or from possible chemical damage in the case of chemiluminescent detection. Alternatively the fluorescent tag may be selectively destroyed by a chemical or photochemical reaction. This process eliminates the need to cleave the tag after each readout, or to detach and transport the tag from the reaction chamber to a separate detection chamber for fluorescent detection. The present invention provides a method for selective destruction of a fluorescent tag by a photochemical reaction with diphenyliodonium ions or related species.</p>
    <p num="p-0017">The present invention further provides a reactive sequencing method that utilizes a two cycle system. An exonuclease-deficient polymerase is used in the first cycle and a mixture of exonuclease-deficient and exonuclease-proficient enzymes are used in the second cycle. In the first cycle, the template-primer system together with an exonuclease-deficient polymerase will be presented sequentially with each of the four possible nucleotides. In the second cycle, after identification of the correct nucleotide, a mixture of exonuclease proficient and deficient polymerases, or a polymerase containing both types of activity will be added in a second cycle together with the correct dNTP identified in the first cycle to complete and proofread the primer extension. In this way, an exonuclease-proficient polymerase is only present in the reaction cell when the correct dNTP is present, so that exonucleolytic degradation of correctly extended strands does not occur, while degradation and correct re-extension of previously incorrectly extended strands does occur, thus achieving extremely accurate strand extension.</p>
    <p num="p-0018">The present invention also provides a method for monitoring reactive sequencing reactions to detect and correct sequencing reaction errors resulting from misincorporation, i.e., incorrectly incorporating a non-complementary base, and extension failure, i.e., failure to extend a fraction of the DNA primer strands. The method is based on the ability to (i) determine the size of the trailing strand population (trailing strands are those primer strands which have undergone an extension failure at any extension prior to the current reaction step); (ii) determine the downstream sequence of the trailing strand population between the 3′ terminus of the trailing strands and the 3′ terminus of the corresponding leading strands (“downstream” refers to the template sequence beyond the current 3′ terminus of a primer strand; correspondingly, “upstream” refers to the known template and complementary primer sequence towards the 5′ end of the primer strand; “leading strands” are those primer strands which have not previously undergone extension failure); and (iii) predict at each extension step the signal to be expected from the extension of the trailing strands through simulation of the occurrence of an extension failure at any point upstream from the 3′ terminus of the leading strand. Subtraction of the predicted signal from the measured signal yields a signal due only to valid extension of the leading strand population.</p>
    <p num="p-0019">In a preferred embodiment of the invention, the monitoring for reactive sequencing reaction errors is computer-aided. The ability to monitor extension failures permits determination of the point to which the trailing strands for a given template sequence have advanced and the sequence in the 1, 2 or 3 base gap between these strands and the leading strands. Knowing this information the dNTP probe cycle can be altered to selectively extend the trailing strands for a given template sequence while not extending the leading strands, thereby resynchronizing the populations.</p>
    <p num="p-0020">The present invention further provides an apparatus for DNA sequencing comprising: (a) at least one chamber including a DNA primer/template system which produces a detectable signal when a DNA polymerase enzyme incorporates a deoxyribonucleotide monophosphate onto the 3′ end of the primer strand; (b) means for introducing into, and evacuating from, the reaction chamber at least one selected from the group consisting of buffers, electrolytes, DNA template, DNA primer, deoxyribonucleotides, and polymerase enzymes; (c) means for amplifying said signal; and (d) means for converting said signal into an electrical signal.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF DRAWINGS</heading> <p num="p-0021">Further objects and advantages of the invention will be apparent from a reading of the following description in conjunction with the accompanying drawings, in which:</p>
      <p num="p-0022"> <figref idrefs="DRAWINGS">FIG. 1</figref> is a schematic diagram illustrating a reactive sequencing device containing a thin film bismuth antimony thermopile in accordance with the invention;</p>
      <p num="p-0023"> <figref idrefs="DRAWINGS">FIG. 2</figref> is a schematic diagram of a reactive sequencing device containing a thermistor in accordance with the invention;</p>
      <p num="p-0024"> <figref idrefs="DRAWINGS">FIG. 3</figref> is a schematic diagram illustrating a representative embodiment of micro calorimetry detection of a DNA polymerase reaction in accordance with the invention;</p>
      <p num="p-0025"> <figref idrefs="DRAWINGS">FIG. 4</figref> is an electrophoretic gel showing a time course for primer extension assays catalyzed by T4 DNA polymerase mutants;</p>
      <p num="p-0026"> <figref idrefs="DRAWINGS">FIG. 5</figref> is a schematic diagram illustrating a nucleotide attached to a fluorophore by a benzoin ester which is a photocleavable linker for use in the invention;</p>
      <p num="p-0027"> <figref idrefs="DRAWINGS">FIG. 6</figref> is a schematic illustration of a nucleotide attached to a chemiluminescent tag for use in the invention;</p>
      <p num="p-0028"> <figref idrefs="DRAWINGS">FIG. 7</figref> is a schematic diagram of a nucleotide attached to a chemiluminescent tag by a cleavable linkage;</p>
      <p num="p-0029"> <figref idrefs="DRAWINGS">FIGS. 8(</figref> <i>a</i>) and <b>8</b>(<i>b</i>) are schematic diagrams of a mechanical fluorescent sequencing method in accordance with the invention in which a DNA template and primer are absorbed on beads captured behind a porous frit; and</p>
      <p num="p-0030"> <figref idrefs="DRAWINGS">FIG. 9</figref> is a schematic diagram of a sequencing method in accordance with the invention utilizing a two cycle system.</p>
      <p num="p-0031"> <figref idrefs="DRAWINGS">FIG. 10</figref> is a diagram of the mechanism of photochemical degradation of fluorescein by diphenyliodonium ion (DPI).</p>
      <p num="p-0032"> <figref idrefs="DRAWINGS">FIG. 11</figref> shows fluorescence spectra of equimolar concentrations of fluorescein and tetramethylrhodamine dyes before and after addition of a solution of diphenyliodonium chloride.</p>
      <p num="p-0033"> <figref idrefs="DRAWINGS">FIG. 12</figref> is the UV absorption spectra obtained from (1) fluorescein and (2) fluorescein+DPI after a single flash from a xenon camera strobe.</p>
      <p num="p-0034"> <figref idrefs="DRAWINGS">FIG. 13</figref> displays the fluorescence spectra from single nucleotide polymerase reactions with DPI photobleaching between incorporation reactions.</p>
      <p num="p-0035"> <figref idrefs="DRAWINGS">FIG. 14A-D</figref>. Simulation of Reactive Sequencing of [CTGA] GAA ACC AGA MG TCC [T], probed with a dNTP cycle. <b>14</b>A. Sequence readout close to the primer where no extension failure has occurred. <b>14</b>B. Sequence readout downstream of primer where 60% of the strands have undergone extension failure and are producing out of phase signals and misincorporation has prevented extension on 75% of all strands. <b>14</b>C. Downstream readout with error signals from trailing strands (dark shading) distinguished from correct readout signals from leading strands (light shading) using knowledge of the downstream sequence of the trailing strands. <b>14</b>D. Corrected sequence readout following subtraction of error signals from trailing strands. Note the similarity to the data of <figref idrefs="DRAWINGS">FIG. 1A</figref>.</p>
      <p num="p-0036"> <figref idrefs="DRAWINGS">FIG. 15</figref>. Effect of a leading strand population on extension signals.</p>
    </description-of-drawings> <heading>DETAILED DESCRIPTION</heading> <p num="p-0037">The present invention provides a method for determining the nucleic acid sequence of a DNA molecule based on detection of successive single nucleotide DNA polymerase mediated extension reactions. As described in detail below, in one embodiment, a DNA primer/template system comprising a polynucleotide primer complementary to and bound to a region of the DNA to be sequenced is constrained within a reaction cell into which buffer solutions containing various reagents necessary for a DNA polymerase reaction to occur are added. Into the reaction cell, a single type of deoxynucleoside triphosphate (dNTP) is added. Depending on the identity of the next complementary site in the DNA primer/template system, an extension reaction will occur only when the appropriate nucleotide is present in the reaction cell. A correlation between the nucleotide present in the reaction cell and detection of an incorporation signal identifies the next nucleotide of the template. Following each extension reaction, the reaction cell is flushed with dNTP-free buffer, retaining the DNA primer/template system, and the cycle is repeated until the entire nucleotide sequence is identified.</p>
    <p num="p-0038">The present invention is based on the existence of a control signal within the active site of DNA polymerases which distinguish, with high fidelity, complementary and non-complementary fits of incoming deoxynucleotide triphosphates to the base on the template strand at the primer extension site, i.e., to read the sequence, and to incorporate at that site only the one type of deoxynucleotide that is complementary. That is, if the available nucleotide type is not complementary to the next template site, the polymerase is inactive, thus, the template sequence is the DNA polymerase control signal. Therefore, by contacting a DNA polymerase system with a single nucleotide type rather than all four, the next base in the sequence can be identified by detecting whether or not a reaction occurs. Further, single base repeat lengths can be quantified by quantifying the extent of reaction.</p>
    <p num="p-0039">As a first step in the practice of the inventive method, single-stranded template DNA to be sequenced is prepared using any of a variety of different methods known in the art. Two types of DNA can be used as templates in the sequencing reactions. Pure single-stranded DNA such as that obtained from recombinant bacteriophage can be used. The use of bacteriophage provides a method for producing large quantities of pure single stranded template. Alternatively, single-stranded DNA may be derived from double-stranded DNA that has been denatured by heat or alkaline conditions, as described in Chen and Subrung, (1985, DNA 4:165); Huttoi and Skaki (1986, Anal. Biochem. 152:232); and Mierendorf and Pfeffer, (1987, Methods Enzymol. 152:556), may be used. Such double stranded DNA includes, for example, DNA samples derived from patients to be used in diagnostic sequencing reactions.</p>
    <p num="p-0040">The template DNA can be prepared by various techniques well known to those of skill in the art. For example, template DNA can be prepared as vector inserts using any conventional cloning methods, including those used frequently for sequencing. Such methods can be found in Sambrook et al., Molecular Cloning: A Laboratory Manual, Second Edition (Cold Spring Harbor Laboratories, New York, 1989). In a preferred embodiment of the invention, polymerase chain reactions (PCR) may be used to amplify fragments of DNA to be used as template DNA as described in Innis et al., ed. PCR Protocols (Academic Press, New York, 1990).</p>
    <p num="p-0041">The amount of DNA template needed for accurate detection of the polymerase reaction will depend on the detection technique used. For example, for optical detection, e.g., fluorescence or chemiluminescence detection, relatively small quantities of DNA in the femtomole range are needed. For thermal detection quantities approaching one picomole may be required to detect the change in temperature resulting from a DNA polymerase mediated extension reaction.</p>
    <p num="p-0042">In enzymatic sequencing reactions, the priming of DNA synthesis is achieved by the use of an oligonucleotide primer with a base sequence that is complementary to, and therefore capable of binding to, a specific region on the template DNA sequence. In instances where the template DNA is obtained as single stranded DNA from bacteriophage, or as double stranded DNA derived from plasmids, “universal” primers that are complementary to sequences in the vectors, i.e., the bacteriophage, cosmid and plasmid vectors, and that flank the template DNA, can be used.</p>
    <p num="p-0043">Primer oligonucleotides are chosen to form highly stable duplexes that bind to the template DNA sequences and remain intact during any washing steps during the extension cycles. Preferably, the length of the primer oligonucleotide is from 18-30 nucleotides and contains a balanced base composition. The structure of the primer should also be analyzed to confirm that it does not contain regions of dyad symmetry which can fold and self anneal to form secondary structures thereby rendering the primers inefficient. Conditions for selecting appropriate hybridization conditions for binding of the oligonucleotide primers in the template systems will depend on the primer sequence and are well known to those of skill in the art.</p>
    <p num="p-0044">In utilizing the reactive sequencing method of the invention, a variety of different DNA to incorporate dNTPs onto the 3′ end of the primer which is hybridized to the polymerases include but are not limited to Taq polymerase, T7 or T4 polymerase, and Klenow polymerases. In a preferred embodiment of the invention, described in detail below, DNA polymerases lacking 5″-3″-exonuclease proofreading activity are used in the sequencing reactions. For the most rapid reaction kinetics, the amount of polymerase is sufficient to ensure that each DNA molecule carries a non-covalently attached polymerase molecule during reaction. For a typical equilibrium constant of −50 nM for the equilibrium:
<br>DNA−Pol≈DNA+Pol K˜50 nM</p> <ul> <li id="ul0001-0001" num="0000"> <ul> <li id="ul0002-0001" num="0044">the desired condition is: [Pol]≧50 nM+[DNA].</li> </ul> </li> </ul> <p num="p-0045">In addition, reverse transcriptase which catalyzes the synthesis of single stranded DNA from an RNA template may be utilized in the reactive sequencing method of the invention to sequence messenger RNA (mRNA). Such a method comprises sequentially contacting an RNA template annealed to a primer (RNA primer/template) with dNTPs in the presence of reverse transcriptase enzyme to determine the sequence of the RNA. Because mRNA is produced by RNA polymerase-catalyzed synthesis from a DNA template, and thus contains the sequence information of the DNA template strand, sequencing the mRNA yields the sequence of the DNA gene from which it was transcribed. Eukaryotic mRNAs have poly(A) tails and therefore the primer for reverse transcription can be an oligo(dT). Typically, it will be most convenient to synthesize the oligo(dT) primer with a terminal biotin or amino group through which the primer can be captured on a substrate and subsequently hybridize to and capture the template mRNA strand.</p>
    <p num="p-0046">The extension reactions are carried out in buffer solutions which contain the appropriate concentrations of salts, dNTPs and DNA polymerase required for the DNA polymerase mediated extension to proceed. For guidance regarding such conditions see, for example, Sambrook, et al., (1989, Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Press, N.Y.); and Ausubel, et al. (1989, Current Protocols in Molecular Biology, Green Publishing Associates and Wiley Interscience, N.Y.).</p>
    <p num="p-0047">Typically, buffer containing one of the four dNTPs is added into a reaction cell. Depending on the identity of the nucleoside base at the next unpaired template site in the primer/template system, a reaction will occur when the reaction cell contains the appropriate dNTP. When the reaction cell contains any one of the other three incorrect dNTPs, no reaction will take place.</p>
    <p num="p-0048">The reaction cell is then flushed with dNTP free buffer and the cycle is repeated until a complete DNA sequence is identified. Detection of a DNA polymerase mediated extension can be made using any of the detection methods described in detail below including optical and thermal detection of an extension reaction.</p>
    <p num="p-0049">In some instances, a nucleotide solution is found to be contaminated with any of the other three nucleotides. In such instances a small fraction of strands may be extended by incorporation of an impurity dNTP when the dNTP type supplied is incorrect for extension, producing a population of strands which are subsequently extended ahead of the main strand population. Thus, in an embodiment of the invention, each nucleotide solution can be treated to remove any contaminated nucleotides. Treatment of each nucleotide solution involves reaction of the solution prior to use with immobilized DNA complementary to each the possibly contaminating nucleotides. For example, a dATP solution will be allowed to react with immobilized poly (dA), poly (dG) or poly (dC), with appropriate primers and polymerase, for a time sufficient to incorporate any contaminating dTTP, dCTP and dGTP nucleotides into DNA.</p>
    <p num="p-0050">In a preferred embodiment of the invention, the primer/template system comprises the template DNA tethered to a solid phase support to permit the sequential addition of sequencing reaction reagents without complicated and time consuming purification steps following each extension reaction. Preferably, the template DNA is covalently attached to a solid phase support, such as the surface of a reaction flow cell, a polymeric microsphere, filter material, or the like, which permits the sequential application of sequencing reaction reagents, i.e., buffers, dNTPs and DNA polymerase, without complicated and time consuming purification steps following each extension reaction. Alternatively, for applications that require sequencing of many samples containing the same vector template or same gene, for example, in diagnostic applications, a universal primer may be tethered to a support, and the template DNA allowed to hybridize to the immobilized primer.</p>
    <p num="p-0051">The DNA may be modified to facilitate covalent or non-covalent tethering of the DNA to a solid phase support. For example, when PCR is used to amplify DNA fragments, the 5′ ends of one set of PCR primer oligonucleotides strands may be modified to carry a linker moiety for tethering one of the two complementary types of DNA strands produced to a solid phase support. Such linker moieties include, for example, biotin. When using biotin, the biotinylated DNA fragments may be bound non-covalently to streptavidin covalently attached to the solid phase support. Alternatively, an amino group (—NH<sub>2</sub>) may be chemically incorporated into one of the PCR primer strands and used to covalently link the DNA template to a solid phase support using standard chemistry, such as reactions with N-hydroxysuccinimide activated agarose surfaces.</p>
    <p num="p-0052">In another embodiment, the 5′ ends of the sequencing oligonucleotide primer may be modified with biotin, for non-covalent capture to a streptavidin-treated support, or with an amino group for chemical linkage to a solid support; the template strands are then captured by the non-covalent binding attraction between the immobilized primer base sequence and the complementary sequence on the template strands. Methods for immobilizing DNA on a solid phase support are well known to those of skill in the art and will vary depending on the solid phase support chosen.</p>
    <p num="p-0053">In the reactive sequencing method of the present invention, DNA polymerase is presented sequentially with each of the 4 dNTPs. In the majority of the reaction cycles, only incorrect dNTPs will be present, thereby increasing the likelihood of misincorporation of incorrect nucleotides into the extending DNA primer/template system.</p>
    <p num="p-0054">Accordingly, the present invention further provides methods for optimizing the reactive sequencing reaction to achieve rapid and complete incorporation of the correct nucleotide into the DNA primer/template system, while limiting the misincorporation of incorrect nucleotides. For example, dNTP concentrations may be lowered to reduce misincorporation of incorrect nucleotides into the DNA primer. K<sub>m </sub>values for incorrect dNTPs can be as much as 1000-fold higher than for correct nucleotides, indicating that a reduction in dNTP concentrations can reduce the rate of misincorporation of nucleotides. Thus, in a preferred embodiment of the invention the concentration of dNTPs in the sequencing reactions are approximately 5-20 μM. At this concentration, incorporation rates are as close to the maximum rate of 400 nucleotides/s for T4 DNA polymerase as possible.</p>
    <p num="p-0055">In addition, relatively short reaction times can be used to reduce the probability of misincorporation. For an incorporation rate approaching the maximum rate of ˜400 nucleotides/s, a reaction time of approximately 25 milliseconds (ms) will be sufficient to ensure extension of 99.99% of primer strands.</p>
    <p num="p-0056">In a specific embodiment of the invention, DNA polymerases lacking 3′ to 5′ exonuclease activity may be used for reactive sequencing to limit exonucleolytic degradation of primers that would occur in the absence of correct dNTPs. In the presence of all four dNTPs, misincorporation frequencies by DNA polymerases possessing exonucleolytic proofreading activity are as low as one error in 106 to 108 nucleotides incorporated as discussed in Echols and Goodman (1991, Annu. Rev. Biochem 60; 477-511); and Goodman, et al. (1993, Crit. Rev. Biochem. Molec. Biol. 28:83-126); and Loeb and Kunkel (1982, Annu. Rev. Biochem. 52:429-457). In the absence of proofreading, DNA polymerase error rates are typically on the order of 1 in 10<sup>4 </sup>to 1 in 10<sup>6</sup>. Although exonuclease activity increases the fidelity of a DNA polymerase, the use of DNA polymerases having proofreading activity can pose technical difficulties for the reactive sequencing method of the present invention. Not only will the exonuclease remove any misincorporated nucleotides, but also, in the absence of a correct dNTP complementary to the next template base, the exonuclease will remove correctly-paired nucleotides successively until a point on the template sequence is reached where the base is complementary to the dNTP in the reaction cell. At this point, an idling reaction is established where the polymerase repeatedly incorporates the correct dNMP and then removes it. Only when a correct dNTP is present will the rate of polymerase activity exceed the exonuclease rate so that an idling reaction is established that maintains the incorporation of that correct nucleotide at the 3′ end of the primer.</p>
    <p num="p-0057">A number of T4 DNA polymerase mutants containing specific amino acid substitutions possess reduced exonuclease activity levels up to 10,000-fold less than the wild-type enzyme. For example, Reha-Krantz and Nonay (1993, J. Biol. Chem. 268:27100-17108) report that when Asp 112 was replaced with Ala and Glu 114 was replaced with Ala (D112A/E114A) in T4 polymerase, these two amino acid substitutions reduced the exonuclease activity on double stranded DNA by a factor of about 300 relative to the wild type enzyme. Such mutants may be advantageously used in the practice of the invention for incorporation of nucleotides into the DNA primer/template system.</p>
    <p num="p-0058">In yet another embodiment of the invention, DNA polymerases which are more accurate than wild type polymerases at incorporating the correct nucleotide into a DNA primer/template may be used. For example, in a (D112A/E114A) mutant T4 polymerase with a third mutation where Ile 417 is replaced by Val (I417V/D112A/E114A), the 1417V mutation results in an antimutator phenotype for the polymerase (Reha-Krantz and Nonay, 1994, J. Biol. Chem. 269:5635-5643; Stocki et al., 1995, Mol. Biol. 254:15-28). This antimutator phenotype arises because the polymerase tends to move the primer ends from the polymerase site to the exonuclease site more frequently and thus proof read more frequently than the wild type polymerase, and thus increases the accuracy of synthesis.</p>
    <p num="p-0059">In yet another embodiment of the invention, polymerase mutants that are capable of more efficiently incorporating fluorescent-labeled nucleotides into the template DNA system molecule may be used in the practice of the invention. The efficiency of incorporation of fluorescent-labeled nucleotides may be reduced due to the presence of bulky fluorophore labels that may inhibit dNTP interaction at the active site of the polymerase. Polymerase mutants that may be advantageously used for incorporation of fluorescent-labeled dNTPs into DNA include but are not limited to those described in U.S. application Ser. No. 08/632,742 filed Apr. 16, 1996 which is incorporated by reference herein.</p>
    <p num="p-0060">In a preferred embodiment of the invention, the reactive sequencing method utilizes a two cycle system. An exonuclease-deficient polymerase is used in the first cycle and a mixture of exonuclease-deficient and exonuclease-proficient enzymes are used in the second cycle. In the first cycle, the primer/template system together with an exonuclease-deficient polymerase will be presented sequentially with each of the four possible nucleotides. Reaction time and conditions will be such that a sufficient fraction of primers are extended to allow for detection and quantification of nucleotide incorporation, ˜98%, for accurate quantification of multiple single-base repeats. In the second cycle, after identification of the correct nucleotide, a mixture of exonuclease proficient and deficient polymerases, or a polymerase containing both types of activity will be added in a second cycle together with the correct dNTP identified in the first cycle to complete and proofread the primer extension. In this way, an exonuclease-proficient polymerase is only present in the reaction cell when the correct dNTP is present, so that exonucleolytic degradation of correctly extended strands does not occur, while degradation and correct re-extension of previously incorrectly extended strands does occur, thus achieving extremely accurate strand extension.</p>
    <p num="p-0061">The detection of a DNA polymerase mediated extension reaction can be accomplished in a number of ways. For example, the heat generated by the extension reaction can be measured using a variety of different techniques such as those employing thermopile, thermistor and refractive index measurements.</p>
    <p num="p-0062">In an embodiment of the invention, the heat generated by a DNA polymerase mediated extension reaction can be measured. For example, in a reaction cell volume of 100 micrometers<sup>3 </sup>containing 1 μg of water as the sole thermal mass and 2×10<sup>11 </sup>DNA template molecules (300 fmol) tethered within the cell, the temperature of the water increases by 1×10<sup>3</sup>° C. for a polymerase reaction which extends the primer by a single nucleoside monophosphate. This calculation is based on the experimental determination that a one base pair extension in a DNA chain is an exothermic reaction and the enthalpy change associated with this reaction is 3.5 kcal/mole of base. Thus extension of 300 fmol of primer strands by a single base produces 300 fmol×3.5 kcal/mol or 1×10<sup>−9 </sup>cal of heat. This is sufficient to raise the temperature of 1 μg of water by 1×10<sup>−3</sup>° C. Such a temperature change can be readily detectable using thermistors (sensitivity ≦10° C.); thermopiles (sensitivity ≦10<sup>−5</sup>° C.); and refractive index measurements (sensitivity ≦10° C.).</p>
    <p num="p-0063">In a specific embodiment of the invention, thermopiles may be used to detect temperature changes. Such thermopiles are known to have a high sensitivity to temperature and can make measurements in the tens of micro-degree range in several second time constants. Thermopiles may be fabricated by constructing serial sets of junctions of two dissimilar metals and physically arranging the junctions so that alternating junctions are separated in space. One set of junctions is maintained at a constant reference temperature, while the alternate set of junctions is located in the region whose temperature is to be sensed. A temperature difference between the two sets of junctions produces a potential difference across the junction set which is proportional to the temperature difference, to the thermoelectric coefficient of the junction and to the number of junctions. For optimum response, bimetallic pairs with a large thermoelectric coefficient are desirable, such as bismuth and antimony. Thermopiles may be fabricated using thin film deposition techniques in which evaporated metal vapor is deposited onto insulating substrates through specially fabricated masks. Thermopiles that may be used in the practice of the invention include thermopiles such as those described in U.S. Pat. No. 4,935,345, which is incorporated by reference herein.</p>
    <p num="p-0064">In a specific embodiment of the invention, miniature thin film thermopiles produced by metal evaporation techniques, such as those described in U.S. Pat. No. 4,935,345 incorporated herein by reference, may be used to detect the enthalpy changes. Such devices have been made by vacuum evaporation through masks of about 10 mm square. Using methods of photolithography, sputter etching and reverse lift-off techniques, devices as small as 2 mm square may be constructed without the aid of modern microlithographic techniques. These devices contain 150 thermoelectric junctions and employ 12 micron line widths and can measure the exothermic heat of reaction of enzyme-catalyzed reactions in flow streams where the enzyme is preferably immobilized on the surface of the thermopile.</p>
    <p num="p-0065">To incorporate thermopile detection technology into a reactive sequencing device, thin-film bismuth-antimony thermopiles <b>2</b>, as shown in <figref idrefs="DRAWINGS">FIG. 1</figref>, may be fabricated by successive electron-beam evaporation of bismuth and antimony metals through two different photolithographically-generated masks in order to produce a zigzag array of alternating thin bismuth and antimony wires which are connected to form two sets of bismuth-antimony thermocouple junctions. Modern microlithographic techniques will allow fabrication of devices at least one order of magnitude smaller than those previously made, i.e., with line widths as small as 1 ÿm and overall dimensions on the order of 100 μm<sup>2</sup>. One set of junctions <b>4</b> (the sensor junctions) is located within the reaction cell <b>6</b>, i.e., deposited on a wall of the reaction cell, while the second reference set of junctions <b>8</b> is located outside the cell at a reference point whose temperature is kept constant. Any difference in temperature between the sensor junctions and the reference junctions results in an electric potential being generated across the device, which can be measured by a high-resolution digital voltmeter <b>10</b> connected to measurement points <b>12</b> at either end of the device. It is not necessary that the temperature of the reaction cell and the reference junctions be the same in the absence of a polymerase reaction event, only that a change in the temperature of the sensor junctions due to a polymerase reaction event be detectable as a change in the voltage generated across the thermopile.</p>
    <p num="p-0066">In addition to thermopiles, as shown in <figref idrefs="DRAWINGS">FIG. 2</figref>, a thermistor <b>14</b> may also be used to detect temperature changes in the reaction cell <b>6</b> resulting from DNA polymerase mediated incorporation of dNMPs into the DNA primer strand. Thermistors are semiconductors composed of a sintered mixture of metallic oxides such as manganese, nickel, and cobalt oxides. This material has a large temperature coefficient of resistance, typically ˜4% per ° C., and so can sense extremely small temperature changes when the resistance is monitored with a stable, high-resolution resistance-measuring device such as a digital voltmeter, e.g., Keithley Instruments Model 2002. A thermistor <b>14</b>, such as that depicted in <figref idrefs="DRAWINGS">FIG. 2</figref>, may be fabricated in the reactive sequencing reaction cell by sputter depositing a thin film of the active thermistor material onto the surface of the reaction cell from a single target consisting of hot pressed nickel, cobalt and manganese oxides. Metal interconnections <b>16</b> which extend out beyond the wall of the reaction cell may also be fabricated in a separate step so that the resistance of the thermistor may be measured using an external measuring device <b>18</b>.</p>
    <p num="p-0067">Temperature changes may also be sensed using a refractive index measurement technique. For example, techniques such as those described in Bornhop (1995, Applied Optics 34:3234-323) and U.S. Pat. No. 5,325,170, may be used to detect refractive index changes for liquids in capillaries. In such a technique, a low-power He—Ne laser is aimed off-center at a right angle to a capillary and undergoes multiple internal reflection. Part of the beam travels through the liquid while the remainder reflects only off the external capillary wall. The two beams undergo different phase shifts depending on the refractive index difference between the liquid and capillary. The result is an interference pattern, with the fringe position extremely sensitive to temperature-induced refractive index changes.</p>
    <p num="p-0068">In a further embodiment of the invention, the thermal response of the system may be increased by the presence of inorganic pyrophosphatase enzyme which is contacted with the template system along with the dNTP solution. Additionally, heat is released as the pyrophosphate released from the dNTPs upon incorporation into the template system is hydrolyzed by inorganic pyrophosphatase enzyme.</p>
    <p num="p-0069">In another embodiment, the pyrophosphate released upon incorporation of dNTP's may be removed from the template system and hydrolyzed, and the resultant heat detected, using thermopile, thermistor or refractive index methods, in a separate reaction cell downstream. In this reaction cell, inorganic pyrophosphatase enzyme may be mixed in solution with the dNTP removed from the DNA template system, or alternatively the inorganic pyrophosphatase enzyme may be covalently tethered to the wall of the reaction cell.</p>
    <p num="p-0070">Alternatively, the polymerase-catalyzed incorporation of a nucleotide base can be detected using fluorescence and chemiluminescence detection schemes. The DNA polymerase mediated extension is detected when a fluorescent or chemiluminescent signal is generated upon incorporation of a fluorescently or chemiluminescently labeled dNMP into the extending DNA primer strand. Such tags are attached to the nucleotide in such a way as to not interfere with the action of the polymerase. For example, the tag may be attached to the nucleotide base by a linker arm sufficiently long to move the bulky fluorophore away from the active site of the enzyme.</p>
    <p num="p-0071">For use of such detection schemes, nucleotide bases are labeled by covalently attaching a compound such that a fluorescent or chemiluminescent signal is generated following incorporation of a dNTP into the extending DNA primer/template. Examples of fluorescent compounds for labeling dNTPs include but are not limited to fluorescein, rhodamine, and BODIPY (4,4-difluoro-4-bora-3a,4a-diaza-s-indacene). See “Handbook of Molecular Probes and Fluorescent Chemicals”, available from Molecular Probes, Inc. (Eugene, Oreg.). Examples of chemiluminescence based compounds that may be used in the sequencing methods of the invention include but are not limited to luminol and dioxetanones (See, Gundennan and McCapra, “Chemiluminescence in Organic Chemistry”, Springer-Verlag, Berlin Heidleberg, 1987)</p>
    <p num="p-0072">Fluorescently or chemiluminescently labeled dNTPs are added individually to a DNA template system containing template DNA annealed to the primer, DNA polymerase and the appropriate buffer conditions. After the reaction interval, the excess dNTP is removed and the system is probed to detect whether a fluorescent or chemiluminescent tagged nucleotide has been incorporated into the DNA template. Detection of the incorporated nucleotide can be accomplished using different methods that will depend on the type of tag utilized.</p>
    <p num="p-0073">For fluorescently-tagged dNTPs the DNA template system may be illuminated with optical radiation at a wavelength which is strongly absorbed by the tag entity. Fluorescence from the tag is detected using for example a photodetector together with an optical filter which excludes any scattered light at the excitation wavelength.</p>
    <p num="p-0074">Since labels on previously incorporated nucleotides would interfere with the signal generated by the most recently incorporated nucleotide, it is essential that the fluorescent tag be removed at the completion of each extension reaction. To facilitate removal of a fluorescent tag, the tag may be attached to the nucleotide via a chemically or photochemically cleavable linker using methods such as those described by Metzger, M. L., et al. (1994, Nucleic Acids Research 22:4259-4267) and Burgess, K., et al., (1997, J. Org. Chem. 62:5165-5168) so that the fluorescent tag may be removed from the DNA template system before a new extension reaction is carried out.</p>
    <p num="p-0075">In a further embodiment utilizing fluorescent detection, the fluorescent tag is attached to the dNTP by a photocleavable or chemically cleavable linker, and the tag is detached following the extension reaction and removed from the template system into a detection cell where the presence, and the amount, of the tag is determined by optical excitation at a suitable wavelength and detection of fluorescence. In this embodiment, the possibility of fluorescence quenching, due to the presence of multiple fluorescent tags immediately adjacent to one another on a primer strand which has been extended complementary to a single base repeat region in the template, is minimized, and the accuracy with which the repeat number can be determined is optimized. In addition, excitation of fluorescence in a separate chamber minimizes the possibility of photolytic damage to the DNA primer/template system.</p>
    <p num="p-0076">In an additional embodiment utilizing fluorescent detection, the signal from the fluorescent tag can be destroyed using a chemical reaction which specifically targets the fluorescent moiety and reacts to form a final product which is no longer fluorescent. In this embodiment, the fluorescent tag attached to the nucleotide base is destroyed following extension and detection of the fluorescence signal, without the removal of the tag. In a specific embodiment, fluorophores attached to dNTP bases may be selectively destroyed by reaction with compounds capable of extracting an electron from the excited state of the fluorescent moiety thereby producing a radical ion of the fluorescent moiety which then reacts to form a final product which is no longer fluorescent. In a further specific embodiment, the signal from a fluorescent tag is destroyed by photochemical reaction with the cation of a diphenyliodonium salt following extension and detection of the fluorescence label. The fluorescent tag attached to the incorporated nucleotide base is destroyed, without removal of the tag, by the addition of a solution of a diphenyliodonium salt to the reaction cell and subsequent UV light exposure. The diphenyliodonium salt solution is removed and the reactive sequencing is continued. This embodiment does not require dNTP's with chemically or photochemically cleavable linkers, since the fluorescent tag need not be removed.</p>
    <p num="p-0077">In a further embodiment of the technique, the response generated by a DNA polymerase-mediated extension reaction can be amplified. In this embodiment, the dNTP is chemically modified by the covalent attachment of a signaling tag through a linker that can be cleaved either chemically or photolytically. Following exposure of the dNTP to the primer/template system and flushing away any unincorporated chemically modified dNTP, any signaling tag that has been incorporated is detached by a chemical or photolytic reaction and flushed out of the reaction chamber to an amplification chamber in which an amplified signal may be produced and detected.</p>
    <p num="p-0078">A variety of methods may be used to produce an amplified signal. In one such method the signaling tag has a catalytic function. When the catalytic tag is cleaved and allowed to react with its substrate, many cycles of chemical reaction ensue producing many moles of product per mole of catalytic tag, with a corresponding multiplication of reaction enthalpy. Either the reaction product is detected, through some property such as color or absorbency, or the amplified heat product is detected by a thermal sensor. For example, if an enzyme is covalently attached to the dNTP via a cleavable linker arm of sufficient length that the enzyme does not interfere with the active site of the polymerase enzyme. Following incorporation onto the DNA primer strand, that enzyme is detached and transported to a second reactor volume in which it is allowed to interact with its specific substrate, thus an amplified response is obtained as each enzyme molecule carries out many cycles of reaction. For example, the enzyme catalase (CAT) catalyzes the reaction:</p>
    <p num="p-0079">
      <chemistry id="CHEM-US-00001" num="00001"> <div class="patent-image"> <a href="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-C00001.png"> <img id="EMI-C00001" he="8.64mm" wi="76.20mm" file="US07875440-20110125-C00001.TIF" alt="Figure US07875440-20110125-C00001" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US7875440B2/US07875440-20110125-C00001.png" class="patent-full-image" width="304" height="34"> </a> </div>
        <attachments> <attachment idref="CHEM-US-00001" attachment-type="cdx" file="US07875440-20110125-C00001.CDX"> </attachment> <attachment idref="CHEM-US-00001" attachment-type="mol" file="US07875440-20110125-C00001.MOL"> </attachment> </attachments> </chemistry> </p>
    <p num="p-0080">If each dNTP is tagged with a catalase molecule which is detached after dNMP incorporation and allowed to react downstream with hydrogen peroxide, each nucleotide incorporation would generate ˜25 kcal/mol×N of heat where N is the number of hydrogen peroxide molecules decomposed by the catalase. The heat of decomposition of hydrogen peroxide is already ˜6-8 times greater than for nucleotide incorporation, (i.e. 3.5-4 kcal/mol). For decomposition of −150 hydrogen peroxide molecules the amount of heat generated per base incorporation approaches 1000 times that of the unamplified reaction. Similarly, enzymes which produce colored products, such as those commonly used in enzyme-linked immunosorbent assays (ELISA) could be incorporated as detachable tags. For example the enzyme alkaline phosphatase converts colorless p-nitrophenyl phosphate to a colored product (p-nitrophenol); the enzyme horseradish peroxidase converts colorless o-phenylenediamine hydrochloride to an orange product. Chemistries for linking these enzymes to proteins such as antibodies are well-known to those versed in the art, and could be adapted to link the enzymes to nucleotide bases via linker arms that maintain the enzymes at a distance from the active site of the polymerase enzymes.</p>
    <p num="p-0081">In a further embodiment, an amplified thermal signal may be produced when the signaling tag is an entity which can stimulate an active response in cells which are attached to, or held in the vicinity of, a thermal sensor such as a thermopile or thermistor. Pizziconi and Page (1997, Biosensors and Bioelectronics 12:457-466) reported that harvested and cultured mast cell populations could be activated by calcium ionophore to undergo exocytosis to release histamine, up to 10-30 pg (100-300 fmol) per cell. The multiple cell reactions leading to exocytosis are themselves exothermic. This process is further amplified using the enzymes diamine oxidase to oxidize the histamine to hydrogen peroxide and imidazoleacetaldehyde, and catalase to disproportionate the hydrogen peroxide. Two reactions together liberate over 100 kJ of heat per mole of histamine. For example, a calcium ionophore is covalently attached to the dNTP base via a linker arm which distances the linked calcium ionophore from the active site of the polymerase enzyme and is chemically or photochemically cleavable. Following the DNA polymerase catalyzed incorporation step, and flushing away unincorporated nucleotides any calcium ionophore remaining bound to an incorporated nucleotide may be cleaved and flushed downstream to a detection chamber containing a mast cell-based sensor such as described by Pizziconi and Page (1997, Biosensors and Bioelectronics 12:457-466). The calcium ionophore would bind to receptors on the mast cells stimulating histamine release with the accompanying generation of heat. The heat production could be further amplified by introducing the enzymes diamine oxidase to oxidize the histamine to hydrogen peroxide and imidazoleacetaldehyde, and catalase to disproportionate the hydrogen peroxide. Thus a significantly amplified heat signal would be produced which could readily be detected by a thermopile or thermistor sensor within, or in contact with, the reaction chamber.</p>
    <p num="p-0082">In a further embodiment utilizing chemiluminescent detection, the chemiluminescent tag is attached to the dNTP by a photocleavable or chemically cleavable linker. The tag is detached following the extension reaction and removed from the template system into a detection cell where the presence, and the amount, of the tag is determined by an appropriate chemical reaction and sensitive optical detection of the light produced. In this embodiment, the possibility of a non-linear optical response due to the presence of multiple chemiluminescent tags immediately adjacent to one another on a primer strand which has been extended complementary to a single base repeat region in the template, is minimized, and the accuracy with which the repeat number can be determined is optimized. In addition, generation of chemiluminescence in a separate chamber minimizes chemical damage to the DNA primer/template system, and allows detection under harsh chemical conditions which otherwise would chemically damage the DNA primer/template. In this way, chemiluminescent tags can be chosen to optimize chemiluminescence reaction speed, or compatibility of the tagged dNTP with the polymerase enzyme, without regard to the compatibility of the chemiluminescence reaction conditions with the DNA primer/template.</p>
    <p num="p-0083">In a further embodiment of the invention, the concentration of the dNTP solution removed from the template system following each extension reaction can be measured by detecting a change in UV absorption due to a change in the concentration of dNTPs, or a change in fluorescence response of fluorescently-tagged dNTPs. The incorporation of nucleotides into the extended template would result in a decreased concentration of nucleotides removed from the template system. Such a change could be detected by measuring the UV absorption of the buffer removed from the template system following each extension cycle.</p>
    <p num="p-0084">In a further embodiment of the invention, extension of the primer strand may be sensed by a device capable of sensing fluorescence from, or resolving an image of, a single DNA molecule. Devices capable of sensing fluorescence from a single molecule include the confocal microscope and the near-field optical microscope. Devices capable of resolving an image of a single molecule include the scanning tunneling microscope (STM) and the atomic force microscope (AFM).</p>
    <p num="p-0085">In this embodiment of the invention, a single DNA template molecule with attached primer is immobilized on a surface and viewed with an optical microscope or an STM or AFM before and after exposure to buffer solution containing a single type of dNTP, together with polymerase enzyme and other necessary electrolytes. When an optical microscope is used, the single molecule is exposed serially to fluorescently-tagged dNTP solutions and as before incorporation is sensed by detecting the fluorescent tag after excess unreacted dNTP is removed. Again as before, the incorporated fluorescent tag must be cleaved and discarded before a subsequent tag can be detected. Using the STM or AFM, the change in length of the primer strand is imaged to detect incorporation of the dNTP. Alternatively the dNTP may be tagged with a physically bulky molecule, more readily visible in the STM or AFM, and this bulky tag is removed and discarded before each fresh incorporation reaction.</p>
    <p num="p-0086">When sequencing a single molecular template in this way, the possibility of incomplete reaction producing erroneous signal and out-of-phase strand extension, does not exist and the consequent limitations on read length do not apply. For a single molecular template, reaction either occurs or it does not, and if it does not, then extension either ceases and is known to cease, or correct extension occurs in a subsequent cycle with the correct dNTP. In the event that an incorrect nucleotide is incorporated, which has the same probability as more the multiple strand processes discussed earlier, for example 1 in 1,000, an error is recorded in the sequence, but this error does not propagate or affect subsequent readout and so the read length is not limited by incorrect incorporation.</p>
    <p num="h-0007">Detection and Compensation for DNA Polymerase Errors</p>
    <p num="p-0087">In the reactive sequencing process, extension failures will typically arise due to the kinetics of the extension reaction and limitations on the amount of time allotted for each extension trial with the single deoxynucleotide triphosphates (dNTP's). When reaction is terminated by flushing away the dNTP supply, some small fraction of the primer strands may remain unextended. These strands on subsequent dNTP reaction cycles will continue to extend but will be out of phase with the majority strands, giving rise to small out-of-phase signals (i.e. signaling a positive incorporation for an added dNTP which is incorrect for extension of the majority strands). Because extension failure can occur, statistically, on any extension event, these out-of-phase signals will increase as the population of strands with extension failures grows. Ultimately the out-of-phase signal becomes comparable in amplitude with the signal due to correct extension of the majority strands and the sequence may be unreadable. The length by which the primer has been extended when the sequence becomes unreadable is known as the sequencing read length.</p>
    <p num="p-0088">The present invention relates to a method that can extend the sequencing read length in two ways, first, by discriminating between the in-phase and out-of-phase signals, and second by calculating where, and how, a dNTP probe sequence can be altered so as selectively to extend the out-of-phase strands to bring them back into phase with the majority strands.</p>
    <p num="p-0089">Specifically, a method is provided for discriminating between the in-phase and out-of-phase sequencing signals comprising: (i) detecting and measuring error signals thereby determining the size of the trailing strand population; (ii) between the 3′ terminus of the trailing strand primers and the 3′ terminus of the leading strand primers; (iii) simulating the occurrence of an extension failure at a point upstream from the 3′ terminus of the leading strands thereby predicting at each extension step the exact point in the sequence previously traversed by the leading strands to which the 3′ termini of the trailing strands have been extended; (iv) predicting for each dNTP introduced the signal to be expected from correct extension of the trailing strands; and (v) subtracting the predicted signal from the measured signal to yield a signal due only to correct extension of the leading strand population.</p>
    <p num="p-0090">“Upstream” refers to the known sequence of bases correctly incorporated onto the primer strands. “Downstream” refers to the sequence beyond the 3′ terminus. Thus for the leading strand population the downstream sequence is unknown but is predetermined by the sequence of the template strand that has not yet been read; for the trailing strand population, the downstream sequence is known for the gap between the 3′ termini of the trailing and leading strands.</p>
    <p num="p-0091">The gap between the leading and trailing primer strands may be 1, 2 or 3 bases (where a single base repeat of any length, e.g. AAAA, is counted as a single base because the entire repeat will be traversed in a single reaction cycle if the correct dNTP is introduced), but can never exceed 3 bases nor shrink spontaneously to zero if the reaction cycle of the four dNTP's is unchanged and no other reaction errors occur, for example a second extension failure on the same primer strand. If the reaction cycle of the four dNTP's is unchanged, it may readily be understood that a primer strand which has failed to extend when the correct dNTP, for example dATP, is in the reaction chamber cannot trail the leading (majority) strands (which did extend) by more than 3 bases, because the fourth base in the dNTP reaction cycle will always once again be the correct base (dATP) for the strand which failed to extend previously. Similarly, a trailing strand resulting from an extension failure can never re-synchronize with the leading strands if extension subsequently proceeds correctly, because the leading strands will always have extended by at least one more nucleotide—G, T, or C in the example discussion of an A extension failure—before the trailing strand can add the missing A. The effect is that after each complete dNTP cycle the trailing strands always follow the leading strands by an extension amount that represents the bases added in one complete dNTP cycle at a given point in the sequence. A further consequence is that all trailing strands that have undergone a single failure are in phase with each other regardless of the point at which the extension failure occurred.</p>
    <p num="p-0092">The methods described herein may be utilized to significantly extend the read length that can be achieved by the technique of reactive sequencing by providing a high level of immunity to erroneous signals arising from extension failure. In a preferred embodiment of the invention, the discrimination method of the invention is computer based.</p>
    <p num="p-0093">First, determination of the readout signals allows real-time discrimination between the signals due to correct extension of the leading strand population and error signals arising from extension of the population of trailing strands resulting from extension failure. Using this information, accurate sequence readout can be obtained significantly beyond the point at which the trailing strand signals would begin to mask the correct leading strand signals. In fact, because the trailing strand signals can always be distinguished from the leading strand signals, it is possible to allow the trailing strand population to continue to grow, at the expense of the leading strands, to the point where the sequence is read from the signals generated on the trailing strand population, and the leading strand signals are treated as error signals to be corrected for. Ultimately, as the probability that a primer strand will have undergone at least one extension failure approaches unity, the signals from the leading strand population will disappear. Correspondingly the probability will increase that a trailing strand will undergo a second extension failure; the signals from this second population of double failure strands can be monitored and the single failure strand signals corrected in just the same way as the zero failure strand signals were corrected for signals due to single failure strands.</p>
    <p num="p-0094">Second, because knowledge of the leading strand sequence permits one to know the point to which the trailing strands have advanced, by simulating the effect of an extension failure on that known sequence in a computer, and also to know the sequence in the 1, 2 or 3 base gap between these strands and the leading strands, then for a given template sequence the dNTP probe cycle can be altered at any point to selectively extend the trailing strands while not extending the leading strands, thereby resynchronizing the populations. Alternatively the gap between leading and trailing strands can be simulated in the computer and the gap can be eliminated by reversing the dNTP cycle whenever the gap shrinks to a single base. These processes are referred to as “healing.” If a large number of different sequences are being read in parallel with the same dNTP reagents, an altered dNTP probe cycle that is correct for healing extension failure strands on a given sequence may not be correct for healing other sequences. However, with a large enough number of parallel sequence readouts, roughly one-third of the sequences will have trailing strands with a 1-base gap at any point, and so reversal of the dNTP probe cycle at arbitrary intervals will heal roughly one-third of the readouts with extension failure gaps. Repeated arbitrary reversal of the dNTP probe cycle eventually heals roughly two-thirds of all the readouts. The overall effect of these error correction and error elimination processes is to reduce, or eliminate any limitation on read length arising from extension failure.</p>
    <p num="p-0095">The ability to overcome the read length limitations imposed by extension failure provides significant additional flexibility in experimental design. For example, it may be that read length is not limited by extension failure, but rather by misincorporation of incorrect nucleotides, which shuts down extension on the affected strands and steadily reduces the signal, ultimately to the point where it is not detectable with the desired accuracy. In this case, the ability to eliminate the effects of extension failure allows the experimenter great flexibility to alter the reaction conditions in such a way that misincorporation is minimized, at the expense of an increased incidence of extension failure. Misincorporation frequency depends in part on the concentration of the probing dNTP's and the reaction time allowed. Longer reaction times, or higher dNTP concentrations result in an increased probability of misincorporation, but a reduced incidence of extension failure. Therefore, if a higher level of extension failure can be tolerated due to, for example, the computer-aided signal discrimination and dNTP cycle-reversal healing methods, then reaction times and/or dNTP reagent concentrations can be reduced to minimize misincorporation, with the resulting increase in extension failure being countered by the computer-aided signal discrimination and/or dNTP cycle-reversal healing techniques described above.</p>
    <p num="p-0096">If the deoxyribonucleotides used for the polymerase reaction are impure a small fraction of strands will extend when the main nucleotide is incorrect and produce a population of leading, rather than trailing, error strands. As with the trailing strands, the leading strand population is never more than three bases, nor less than one base, ahead of the main population, unless a second error occurs on the same strand, and also, regardless of where an incorrect extension by an impurity dNTP occurs, the leading strands are all in phase with each other. A given base site can be probed either 1, 2 or 3 times with an incorrect dNTP before it must be extended by the correct dNTP, so on the average twice. If each of the incorrect dNTP's is assumed to carry the same percentage of dNTP impurity, then the probability of incorrect extension by, e.g. 99% pure dNTP containing the correct complementary base as an impurity is 1%÷3 (only ⅓ of the impurity will be the correct complementary base)×2 (average 2 incorrect trials between each correct extension), that is, 0.67%.</p>
    <p num="p-0097">As with trailing strands, the leading strand population can produce out-of-phase extension signals that complicate the readout of the majority strand sequence, as shown in <figref idrefs="DRAWINGS">FIG. 15</figref>. Because the sequence downstream of the 3′ terminus of the majority strands is not known at the time of extension of those strands, the signal due to leading strand extension can not immediately be corrected for, nor can an altered dNTP cycle be calculated which would automatically heal the gap between majority and leading strands for a given template sequence. However similar methods can be used to ameliorate the effects of a leading strand population. First, as with trailing strands, reversal of the dNTP probe cycle automatically heals the gap between leading and majority strand populations whenever the gap shrinks to a single base. Therefore, arbitrary reversal of the dNTP probe cycle has a ⅓ probability of healing the gap for a given sequence, or will heal ⅓ of the sequences in a large population of sequences probed in parallel. Continued arbitrary reversal eventually heals roughly two-thirds of such gaps. Second, although the sequence downstream of the 3′ terminus of the majority strands is not immediately known, information about this sequence becomes available as soon as the majority strands traverse the gap region. Therefore, for each extension of the majority strands it is possible, ideally using a computer simulation, to calculate when the leading strand population would have traversed that base and thus the signal by which a prior extension of the majority strands would have been contaminated. In this way the majority strand extension signals can retrospectively be corrected for leading strand signals.</p>
    <p num="p-0098">There are important aspects to leading strand creation that reduce the frequency of occurrence of leading strand events. First, if the concentration of impurity dNTP's is sufficiently low, a leading strand population cannot be created by impurity extension of the first base of a repeat. This is because the probability of incorrect incorporation of two impurity bases on the same strand in the same reaction cycle is the square of the probability for a single incorporation, and therefore vanishingly small for small impurity levels. Therefore, whenever the correct dNTP for extension of the repeat length is supplied, all strands will be extended to completion when the correct nucleotide is supplied, regardless of whether some fraction of the strands were already partially extended by one base of the repeat. Second, not all incorrect extensions result in a permanent phase difference. For a permanent phase difference to result, a second extension (by a correct base) must occur on the leading strand before the main strands extend to catch up to the leading strand. Labeling the next four sites along the template sequence: 1, 2, 3, 4, then, by definition, if a leading strand is created by incorporation of an impurity base on site 1 while the majority of the strands do not extend, the main nucleotide supplied is incorrect for extension at site 1. If the main nucleotide supplied is correct for extension at site 2, a 2-base lead is created. There is 1 chance in 4 that the reaction chamber contains the correct nucleotide for site 2, so the probability of creating a 2-base extension in a single step (with an impurity extension followed by a correct extension) is ¼ the probability of the impurity extension alone. For the 0.67% impurity extension probability cited above, this means a 0.16% probability of creating a 2-base extension in a single cycle.</p>
    <p num="p-0099">However, if the main nucleotide supplied is incorrect for further extension at site 2, and, by definition incorrect for extension at site 1, then for the lead to become fixed, the correct nucleotide for site 2 must be supplied before the correct nucleotide to extend at site 1. The probability that site 2 will extend before site 1 is therefore 50%; for a 0.67% impurity extension probability, the probability that this creates a fixed lead due to a second extension by a correct nucleotide is 0.33%. Overall, a 1% impurity level results in ˜0.5% probability of creating a leading strand in any given reaction trial.</p>
    <p num="p-0100">Preparation of specific embodiments in accordance with the present invention will now be described in further detail. These examples are intended to be illustrative and the invention is not limited to the specific materials and methods set forth in these embodiments.</p>
    <heading>Example 1</heading> <p num="p-0101">A microcalorimetic experiment was performed which demonstrates for the first time the successful thermal detection of a DNA polymerase reaction. The results are shown in <figref idrefs="DRAWINGS">FIG. 3</figref>. Approximately 20 units of T7 Sequenase was injected into a 3 mL reaction volume containing approximately 20 nmol of DNA template and complementary primer, and an excess of dNTPs. The primer was extended by 52-base pairs, the expected length given the size of the template. Using a commercial microcalorimeter (TAM Model 2273; Thermometrics, Sweden) a reaction enthalpy of 3.5-4 kcal per mole of base was measured (<figref idrefs="DRAWINGS">FIG. 3</figref>). This measurement is well within the value required for thermal detection of DNA polymerase activity. This measurement also demonstrates the sensitivity of thermopile detection as the maximum temperature rise in the reaction cell was 1×10<sup>−3 </sup>C. The lower trace seen in <figref idrefs="DRAWINGS">FIG. 3</figref> is from a reference cell showing the injection artifact for an enzyme-free injection into buffer containing no template system.</p>
    <heading>Example 2</heading> <p num="p-0102">To illustrate the utility of mutant T4 polymerases, two primer extension assays were performed with two different mutant T4 polymerases, both of which are exonuclease deficient. In one mutant, Asp112 is replaced with Ala and Glu114 is replaced with Ala (D112A/E114A). The exonuclease activity of this mutant on double-stranded DNA is reduced by a factor of about 300 relative to the wild type enzyme as described by Reha-Krantz and Nonay (1993, J. Biol. Chem. 268:27100-27108). In a second polymerase mutant, in addition to the DI12A/E114A amino acid substitutions, a third substitution replaces Ile417 with Val (I417V/D112A/E114A). The I417V mutation increases the accuracy of synthesis by this polymerase (Stocki, S. A. and Reha-Krantz, L. J, 1995, J. Mol. Biol. 245:15-28; Reha-Krantz, L. J. and Nonay, R. L., 1994, J. Biol. Chem. 269:5635-5643)</p>
    <p num="p-0103">Two separate primer extension reactions were carried out using each of the polymerase mutants. In the first, only a single correct nucleotide, dGTP, corresponding to a template C was added; The next unpaired template site is a G so that misincorporation would result in formation of a G-G mispair. A G-G mispair tends to be among the most difficult mispairs for polymerases to make. In the second primer extension reaction, two nucleotides, dGTP and dCTP, complementary to the first three unpaired template sites were added. Following correct incorporation of dGMP and dCMP, the next available template site is a T. Formation of C-T mispairs tend to be very difficult while G-T mispairs tend to be the most frequent mispairs made by polymerases.</p>
    <p num="p-0104">Time courses for primer extension reactions by both mutant T4 polymerases are shown in <figref idrefs="DRAWINGS">FIG. 4</figref>. Low concentrations of T4 polymerase relative to primer/template (p/t) were used so that incorporation reactions could be measured on convenient time scales (60 min). By 64 minutes 98% of the primers were extended. In reactions containing only dGTP, both polymerases nearly completely extended primer ends by dGMP without any detectable incorporation of dGMP opposite G. In reactions containing both dGMP and dCMP, both polymerases nearly completely extended primer ends by addition of one dGMP and two dCMP's. A small percentage (=1%) of misincorporation was detectable in the reaction catalyzed by the D112A/E114A mutant. Significantly, no detectable misincorporation was seen in the reaction catalyzed by the I417V/D112A/E114A mutant.</p>
    <heading>Example 3</heading> <p num="p-0105">In accordance with the invention a fluorescent tag may be attached to the nucleotide base at a site other than the 3′ position of the sugar moiety. Chemistries for such tags which do not interfere with the activity of the DNA polymerase have been developed as described by Goodwin et al. (1995, Experimental Technique of Physics 41:279-294). Generally the tag is attached to the base by a linker arm of sufficient length to move the bulky tag out of the active site of the enzyme during incorporation.</p>
    <p num="p-0106">As illustrated in <figref idrefs="DRAWINGS">FIG. 5</figref>, a nucleotide can be connected to a fluorophore by a photocleavable linker, e.g., a benzoin ester. After the tagged dNMP is incorporated onto the 3′ end of the DNA primer strand, the DNA template system is illuminated by light at a wave length corresponding to the absorption maximum of the fluorophore and the presence of the fluorophore is signaled by detection of fluorescence at the emission maximum of the fluorophore. Following detection of the fluorophore, the linker may be photocleaved to produce compound 2; the result is an elongated DNA molecule with a modified but non-fluorescent nucleotide attached. Many fluorophores, including for example, a dansyl group or acridine, etc., will be employed in the methodology illustrated by <figref idrefs="DRAWINGS">FIG. 5</figref>.</p>
    <p num="p-0107">Alternatively, the DNA template system is not illuminated to stimulate fluorescence. Instead, the photocleavage reaction is carried out to produce compound 2 releasing the fluorophore, which is removed from the template system into a separate detection chamber. There the presence of the fluorophore is detected as before, by illumination at the absorption maximum of the fluorophore and detection of emission near the emission maximum of the fluorophore.</p>
    <heading>Example 4</heading> <p num="p-0108">In a specific embodiment of the invention, a linked system consisting of a chemiluminescently tagged dNTP can consist of a chemiluminescent group (the dioxetane portion of compound 4), a chemically cleavable linker (the silyl ether), and an optional photocleavable group (the benzoin ester) as depicted in <figref idrefs="DRAWINGS">FIG. 6</figref>. The cleavage of the silyl ether by a fluoride ion produces detectable chemiluminescence as described in Schaap, et al. (1991, “Chemical and Enzymatic Triggering of 1,2-dioxetanes: Structural Effects on Chemiluminescence Efficiency” in Bioluminescence &amp; Chemiluminescence, Stanley, P. E. and Knicha, L. J. (Eds), Wiley, N.Y. 1991, pp. 103-106). In addition, the benzoin ester that links the nucleoside triphosphate to the silyl linker is photocleavable as set forth in Rock and Chan (1996, J. Org. Chem. 61: 1526-1529); and Felder, et al. (1997, First International Electronic Conference on Synthetic Organic Chemistry, September. 1-30). Having both a chemiluminescent tag and a photocleavable linker is not always necessary; the silyl ether can be attached directly to the nucleotide base and the chemiluminescent tag is destroyed as it is read.</p>
    <p num="p-0109">As illustrated in <figref idrefs="DRAWINGS">FIG. 6</figref> with respect to compound 3, treatment with fluoride ion liberates the phenolate ion of the adamantyl dioxetane, which is known to chemiluminesce with high efficiency (Bronstein et al., 1991, “Novel Chemiluminescent Adamantyl 1,2-dioxetane Enzyme Substrates,” in Bioluminescence &amp; Chemiluminescence, Stanley, P. E. and Kricka, R. J. (eds), Wiley, N.Y. 1991 pp. 73-82). The other product of the reaction is compound 4, which is no longer chemiluminescent. Compound 4 upon photolysis at 308-366 nm liberates compound 2.</p>
    <p num="p-0110">The synthesis of compound 1 is achieved by attachment of the fluorophore to the carboxyl group of the benzoin, whose α-keto hydroxyl group is protected by 9(FMOC), followed by removal of the FMOC protecting group and coupling to the nucleotide bearing an activated carbonic acid derivative at its 3′ end. Compound 4 is prepared via coupling of the vinyl ether form of the adamantyl phenol, to chloro(3-cyanopropyl)dimethylsilane, reduction of the cyano group to the amine, generation of the oxetane, and coupling of this chemiluminescence precursor to the nucleotide bearing an activated carbonic acid derivative at its 3′ end.</p>
    <p num="p-0111">The chemiluminescent tag can also be attached to the dNTP by a cleavable linkage and cleaved prior to detection of chemiluminescence. As shown in <figref idrefs="DRAWINGS">FIG. 7</figref>, the benzoin ester linkage in compound 3 may be cleaved photolytically to produce the free chemiluminescent compound 5. Reaction of compound 5 with fluoride ion to generate chemiluminescence may then be carried out after compound 5 has been flushed away from the DNA template primer in the reaction chamber. As an alternative to photolytic cleavage, the tag may be attached by a chemically cleavable linker which is cleaved by chemical processing which does not trigger the chemiluminescent reaction.</p>
    <heading>Example 5</heading> <p num="p-0112">In this example, the nucleotide sequence of a template molecule comprising a portion of DNA of unknown sequence is determined. The DNA of unknown sequence is cloned into a single stranded vector such as M13. A primer that is complementary to a single stranded region of the vector immediately upstream of the foreign DNA is annealed to the vector and used to prime synthesis in reactive sequencing. For the annealing reaction, equal molar ratios of primer and template (calculated based on the approximation that one base contributes 330 g/mol to the molecular weight of a DNA polymer) is mixed in a buffer consisting of 67 mM Tris HCl pH 8.8, 16.7 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, and 0.5 mM EDTA. This buffer is suitable both for annealing DNA and subsequent polymerase extension reactions. Annealing is accomplished by heating the DNA sample in buffer to 80° C. and allowing it to slowly cool to room temperature. Samples are briefly spun in a microcentrifuge to remove condensation from the lid and walls of the tube. To the DNA is added 0.2 mol equivalents of T4 polymerase mutant I417V/D112A/E114A and buffer components so that the final reaction cell contains 67 mM Tris HCl pH 8.8, 16.7 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>6.7 mM MgCl<sub>2 </sub>and 0.5 mM dithiothreitol. The polymerase is then queried with one dNTP at a time at a final concentration of 10 ÿM. The nucleotide is incubated with polymerase at 37° C. for 10 s. Incorporation of dNTPs may be detected by one of the methods described above including measuring fluorescence, chemiluminescence or temperature change. The reaction cycle will be repeated with each of the four dNTPs until the complete sequence of the DNA molecule has been determined.</p>
    <heading>Example 6</heading> <p num="p-0113"> <figref idrefs="DRAWINGS">FIG. 7</figref> illustrates a mechanical fluorescent sequencing method in accordance with the invention. A DNA template and primer are captured onto beads <b>18</b> using, for example, avidin-biotin or —NH<sub>2</sub>/n-hydroxysuccinimide chemistry and loaded behind a porous frit or filter <b>20</b> at the tip of a micropipette <b>22</b> or other aspiration device as shown in <figref idrefs="DRAWINGS">FIG. 7(</figref> <i>a</i>), step <b>1</b>. Exonuclease deficient polymerase enzyme is added and the pipette tip is lowered into a small reservoir <b>24</b> containing a solution of fluorescently-labeled dNTP. As illustrated in step <b>2</b> of <figref idrefs="DRAWINGS">FIG. 7(</figref> <i>a</i>), a small quantity of dNTP solution is aspirated through the filter and allowed to react with the immobilized DNA. The dNTP solution also contains approximately 100 nM polymerase enzyme, sufficient to replenish rinsing losses. After reaction, as shown in step <b>3</b>, the excess dNTP solution <b>24</b> is forced back out through the frit <b>20</b> into the dNTP reservoir <b>24</b>. In step <b>4</b> of the process the pipette is moved to a reservoir containing buffer solution and several aliquots of buffer solution are aspirated through the frit to rinse excess unbound dNTP from the beads. The buffer inside the pipette is then forced out and discarded to waste <b>26</b>. The pipette is moved to a second buffer reservoir (buffer <b>2</b>), containing the chemicals required to cleave the fluorescent tag from the incorporated dNMP. The reaction is allowed to occur to cleave the tag. As shown in step <b>5</b> the bead/buffer slurry with the detached fluorescent tag in solution is irradiated by a laser or light source <b>28</b> at a wavelength chosen to excite the fluorescent tag, the fluorescence is detected by fluorescence detector <b>30</b> and quantified if incorporation has occurred.</p>
    <p num="p-0114">Subsequent steps depend on the enzyme strategy used. If a single-stage strategy with an exonuclease-deficient polymerase is used, as illustrated in <figref idrefs="DRAWINGS">FIG. 7(</figref> <i>b</i>), the solution containing the detached fluorescent tag is discarded to waste (step <b>6</b>) which is expelled, followed by a further rinse step with buffer <b>1</b> (step <b>7</b>) which is thereafter discarded (step <b>8</b>) and the pipette is moved to a second reservoir containing a different dNTP (step <b>9</b>) and the process repeats starting from step <b>3</b>, cycling through all four dNTPs.</p>
    <p num="p-0115">In a two-stage strategy, after the correct dNTP has been identified and the repeat length quantified in step <b>5</b>, the reaction mixture is rinsed as shown in steps <b>6</b>, <b>7</b>, and <b>8</b> of <figref idrefs="DRAWINGS">FIG. 7(</figref> <i>b</i>) and the pipette is returned to a different reservoir containing the same dNTP (e.g., dNTPI) as shown in step (a) of <figref idrefs="DRAWINGS">FIG. 8</figref> to which a quantity of exonuclease-proficient polymerase has been added and the solution is aspirated for a further stage of reaction which proof-reads the prior extension and correctly completes the extension. This second batch of dNTP need not be fluorescently tagged, as the identity of the dNTP is known and no sequence information will be gained in this proof-reading step. If a tagged dNTP is used, the fluorescent tag is preferably cleaved and discarded as in step <b>5</b> of <figref idrefs="DRAWINGS">FIG. 7(</figref> <i>a</i>) using Buffer <b>2</b>. Alternatively, the initial incorporation reaction shown in step <b>2</b> of <figref idrefs="DRAWINGS">FIG. 7(</figref> <i>a</i>) is carried out for long enough, and the initial polymerase is accurate enough, so that the additional amount of fluorescent tag incorporated with dNTP1 at step a of <figref idrefs="DRAWINGS">FIG. 8</figref> is small and does not interfere with quantification of the subsequent dNTP. Following proof-reading in step a of <figref idrefs="DRAWINGS">FIG. 8</figref>, excess dNTP is expelled (step b) and the reaction mixture is rinsed (steps c, d) with a high-salt buffer to dissociate the exo+ polymerase from the DNA primer/template. It is important not to have exonuclease-proficient enzyme present if the DNA primer/template is exposed to an incorrect dNTP. The pipette is then moved to step e, in which the reservoir contains a different dNTP, and the process is repeated, again cycling through all four dNTPs.</p>
    <heading>Example 7</heading> <p num="p-0116">A new process for destruction of a fluorophore signal which involves reaction of the electronically excited fluorophore with an electron-abstracting species, such as diphenyliodonium salts, is described.</p>
    <p num="p-0117">The reaction of a diphenyliodonium ion with an electronically excited fluorescein molecule is illustrated in <figref idrefs="DRAWINGS">FIG. 10</figref>. The diphenyliodonium ion extracts an electron from the excited state of the fluorescein molecule producing a radical ion of the fluorescein molecule and a neutral diphenyliodonium free radical. The diphenyliodonium free radical rapidly decomposes to iodobenzene and a phenyl radical. The fluorescein radical ion then either reacts with the phenyl radical or undergoes an internal arrangement to produce a final product which is no longer fluorescent.</p>
    <p num="p-0118"> <figref idrefs="DRAWINGS">FIGS. 11 and 12</figref> demonstrate evidence for the specific destruction of fluorescein by diphenylionium ion. In <figref idrefs="DRAWINGS">FIG. 11</figref>, fluorescence spectra are presented for a mixture of fluorescein and tetramethylrhodamine dyes, before and after addition of a solution of diphenyliodonium chloride. It is seen that the fluorescence from the fluorescein dye is immediately quenched, demonstrating electron abstraction from the excited state of the molecule while the fluorescence from the rhodamine is unaffected, apart from a small decrease due to the dilution of the dye solution by the added diphenyliodonium chloride solution.</p>
    <p num="p-0119">Elimination of the fluorescent signal from the fluorescein dye by diphenyliodonium chloride is not in itself proof that the fluorescein molecule has been destroyed, because electron abstraction from the excited state of fluorescein effectively quenches the fluorescence, and quenching need not result in destruction of the fluorescein molecule. However, <figref idrefs="DRAWINGS">FIG. 12</figref> demonstrates that the fluorescein molecule is destroyed by reaction with the diphenyliodonium and not simply quenched. <figref idrefs="DRAWINGS">FIG. 12</figref> demonstrates the ultraviolet (UV) absorption spectra for a fluorescein solution before and after addition of a solution of diphenyliodonium chloride. Spectrum <b>1</b> is the UV absorption spectrum of a pure fluorescein solution. Spectrum <b>2</b> is the UV absorption of the fluorescein solution following the addition of a solution containing a molar excess of diphenyliodonium (DPI) chloride and exposure to a single flash from a xenon camera strobe. The data show that fluorescein is essentially destroyed by the photochemical reaction with the DPI ion. <figref idrefs="DRAWINGS">FIG. 12</figref> provides clear evidence that diphenyliodonium chloride not only quenches the fluorescence from the fluorescein dye but destroys the molecule to such an extent that it can no longer act as a fluorophore.</p>
    <p num="p-0120">An experiment was performed to demonstrate efficient fluorescent detection fluorophore using a template sequence. The template, synthesized with a alkylamino linker at the 5′ terminus, was:</p>
    <p num="p-0121">
      <tables id="TABLE-US-00001" num="00001"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <colspec colname="2" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">3′-H<sub>2</sub>N—(CH<sub>2</sub>)<sub>7</sub>-GAC CAT TAT AGG TCT TGT TA<u>G GGA AAG(</u> </td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0122">The trial sequence to be determined is: G GGA AAG GAA GA.</p>
    <p num="p-0123">A tetramethyrhodamine-labeled primer sequence was synthesized to be complementary to the template as follows:</p>
    <p num="p-0124">
      <tables id="TABLE-US-00002" num="00002"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <colspec colname="2" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">5′-[Rhodamine]-(CH<sub>2</sub>)<sub>6</sub>-CTG GTA ATA TCC AGA ACA AT-</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0125">The alkyl amino-terminated template molecules were chemically linked to Sepharose beads derivatized with N-hydroxysuccinimide and the rhodamine-labeled primer was annealed to the template. The beads with attached DNA template and annealed primer were loaded behind a B-100 disposable filter in a 5-ml syringe. A volume containing a mixture of fluorescein-labeled and unlabelled dCTP in a ratio of 1:2 and exonuclease-deficient polymerase enzyme in a reaction buffer as specified by the manufacturer was drawn into the syringe. Reaction was allowed to proceed for 20 minutes, at 35° C. After the reaction, the fluid was forced out of the syringe, retaining the beads with the reacted DNA behind the filter, and three washes with double-distilled water were performed by drawing water through the filter into the syringe and expelling it. The beads were resuspended in phosphate buffer, the filter was removed and the suspension was dispensed into a cuvette for fluorescence analysis. Following fluorescence analysis, the bead suspension was loaded back into the syringe which was then fitted with a filter tip, and the phosphate buffer was dispensed. A solution of DPI was drawn up into the syringe with a concentration calculated to be in 1:1 molar equivalence to the theoretical amount of DNA template, the filter was removed and the bead suspension was dispensed into a cuvette for UV light exposure for 15 minutes. The suspension was recollected into a syringe, the filter was reattached, the DPI solution was expelled, and the beads were resuspended by drawing up 0.7 mL of phosphate buffer. After removal of the filter the bead suspension was dispensed into a clean cuvette for fluorescence analysis to check the completeness of destruction of the fluorescein by the reaction with the DPI. A subsequent polymerase reaction was performed using the same protocol with labeled dTTP and similarly measured for fluorescence.</p>
    <p num="p-0126"> <figref idrefs="DRAWINGS">FIG. 13</figref> demonstrates the results of the polymerase reactions, with photochemical destruction of the fluorescein label by DPI following each nucleotide incorporation reaction. Curve <b>1</b> shows rhodamine fluorescence following annealing of the rhodamine labeled primer to the beads, demonstrating covalent attachment of the template strands to the beads and capture of the rhodamine-labeled primer strands. Curve <b>2</b> demonstrates detection of fluorescein following polymerase-catalyzed incorporation of three partially fluorescein-labeled dCMPs onto the 3′ terminus of the primer strands. Curve <b>3</b> shows complete destruction of the incorporated fluorescein label by photo-induced reaction with diphenyliodonium chloride. Loss of rhodamine signal here is attributed to loss of a significant fraction of the beads which stuck to the filter during washes. Curve <b>4</b> shows detection of a new fluorescein label following photochemical destruction of the fluorescein attached to the dCMP's and subsequent polymerase-catalyzed incorporation of three partially fluorescein-labeled dTMPs onto the 3′ terminus of the primer strands.</p>
    <p num="p-0127">The following methods were utilized to demonstrate successful destruction of a fluorescein-labeled dTMP.</p>
    <p num="p-0128">Sepharose beads were purchased from Amersham with surfaces derivatized with N-hydroxysuccinimide for reaction with primary amine groups. The alkyl amino-terminated templates were chemically linked to the Sepharose beads using the standard procedure recommended by the manufacturer.</p>
    <p num="p-0129">The beads with attached template were suspended in 250 mM Tris buffer containing 250 mM NaCl and 40 nM MgCl<sub>2</sub>. The solution containing the primer strands was added and the mixture heated to 80° C. and cooled over ˜2 hours to anneal the primers to the surface-immobilized DNA template strands.</p>
    <p num="p-0130">Fluorescein-labeled dUTP and dCTP were purchased from NEN Life Science Products. Unlabeled dTTP and dCTP were purchased from Amersham.</p>
    <p num="p-0131">Prior to any reaction, the annealed primer/template was subjected to fluorescence analysis to ensure that annealing had occurred. The excitation wavelength used was 320 nm and fluorescence from fluorescein and rhodamine was detected at ˜520 nm and ˜580 nm respectively.</p>
    <p num="p-0132">Reagent volumes were calculated on the assumption that the DNA template was attached to the beads with 100% efficiency.</p>
    <p num="p-0133">The 5× reaction buffer contained:
</p> <ul> <li id="ul0003-0001" num="0000"> <ul> <li id="ul0004-0001" num="0134">1) 250 mM Tris buffer, pH 7.5</li> <li id="ul0004-0002" num="0135">2) 250 mM NaCl</li> <li id="ul0004-0003" num="0136">3) 40 mM MgCl<sub>2 </sub> </li> <li id="ul0004-0004" num="0137">4) 1 mg/mL BSA</li> <li id="ul0004-0005" num="0138">5) 25 mM dithiothreitol (DTT)
        <ul> <li id="ul0005-0001" num="0139">mixed and brought to volume with double-distilled H<sub>2</sub>O</li> </ul> </li> </ul> </li> </ul> <p num="p-0134">T4 DNA polymerase was obtained from Worthington Biochemical Corp. The polymerase was dissolved in the polymerase buffer according to the manufacturer's protocols.</p>
    <p num="p-0135">Fluorescein-labeled and unlabeled dCTP's were mixed in a ratio of 1:2.</p>
    <p num="p-0136">The reaction was run in a 5 mL syringe (Becton Dickinson) fitted with a B-10 (Upchurch Scientific). This limits the reaction volume to 5 mL total:</p>
    <p num="p-0137">
      <tables id="TABLE-US-00003" num="00003"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="91pt" align="left"> </colspec> <colspec colname="3" colwidth="49pt" align="right"> </colspec> <colspec colname="4" colwidth="49pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Primer template suspension</td>
                <td class="description-td">0.7</td>
                <td class="description-td">mL</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">T4 DNA Polymerase</td>
                <td class="description-td">1.0</td>
                <td class="description-td">mL</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">FdCTP/dCTP</td>
                <td class="description-td">0.040</td>
                <td class="description-td">mL</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">5× reaction buffer</td>
                <td class="description-td">2.0</td>
                <td class="description-td">mL</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">double-dist. H<sub>2</sub>O</td>
                <td class="description-td">1.0</td>
                <td class="description-td">mL</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0138">The reaction was allowed to proceed in a 35° C. oven for 20 minutes. Following reaction, the fluid was forced out of the syringe allowing the filter to retain the beads with the reacted DNA. Three washes with double-distilled water were performed. All waste was collected and saved for future reuse. The beads were resuspended in 0.7 mL of phosphate buffer, the filter was removed and the suspension was dispensed into a cuvette for fluorescence analysis.</p>
    <p num="p-0139">Following fluorescence analysis the bead suspension was collected into a 1 mL syringe (Becton Dickinson) which was then fitted with a filter tip. The phosphate buffer was dispensed and the waste collected. A solution of diphenyliodonium chloride (DPI) was drawn up with a concentration calculated to be in 1:1 molar equivalence to the theoretical amount of DNA template (i.e. DPI was present in excess of the incorporated fluorescein-labeled dCTP). The filter was removed and the bead suspension with added DPI was dispensed into a cuvette and exposed to UV light for 15 minutes. The suspension was recollected into a syringe, the filter reattached, the DPI solution was dispensed and the beads were resuspended in 0.7 mL of phosphate buffer. The bead suspension was dispensed into a clean cuvette for fluorescence analysis.</p>
    <p num="p-0140">It should be noted that a significant fraction of the beads used in this procedure appeared to become stuck in the filter on the syringe. This resulted in a significant increase in the pressure needed to force fluids through the filter as it became clogged by the beads, and more importantly reduced the amount of DNA available for fluorescent detection of incorporated nucleotides and reduced the weak rhodamine signal from the labeled primer to the point where it was no longer detectable.</p>
    <p num="p-0141">Following the successful incorporation reaction with dCTP, a subsequent polymerase reaction was run to incorporate dTTP. The incorporated fluorescein-labeled dTMP was detected, but with significantly lower intensity due to the losses of the beads in the filter in the multiple transfer steps between the reaction syringe and the analysis cuvette. The lowered signal could also result in part from a different labeling efficiency of the dTTP and a different incorporation efficiency for the labeled nucleotide in the polymerase reaction. Because the rhodamine signal was no longer detectable following the second incorporation reaction it was not possible to correct for bead losses.</p>
    <p num="p-0142">The results are shown in <figref idrefs="DRAWINGS">FIG. 13</figref>. The data represented by the curves were obtained sequentially as follows:</p>
    <p num="p-0143">Curve <b>1</b> shows the rhodamine fluorescence following annealing of the rhodamine-labeled primer to the bead-immobilized DNA template.</p>
    <p num="p-0144">Curve <b>2</b> demonstrates detection of the fluorescein-labeled dCTP following polymerase-catalyzed incorporation of three dCMP's onto the 3′ terminus of the primer strands.</p>
    <p num="p-0145">Curve <b>3</b> demonstrates complete destruction of the incorporated fluorescein label on the dCMP's by photo-induced reaction with dipenyliodonium chloride. In this instance, the rhodamine label also has vanished; this is primarily because a significant fraction of the beads were lost by sticking in the filter used in the reagent flushing operation. It is possible that the rhodamine also was destroyed by the DPI photochemical reaction.</p>
    <p num="p-0146">Curve <b>4</b> demonstrates detection of a new fluorescein label following photochemical destruction of the fluorescein label on the dCMP's and polymerase-catalyzed incorporation of three fluorescein-tagged dTMP's onto the 3′ terminus of the primer strands. The lower signal compared to curve <b>2</b> results mainly from the bead losses in the syringe, but may also reflect a lower incorporation efficiency of the dTMP and/or a lower labeling efficiency. Because the rhodamine signal from the labeled primer is no longer detectable, the bead losses cannot be calibrated.</p>
    <p num="p-0147">The results shown here demonstrate the concept of reactive sequencing by fluorescent detection of DNA extension followed by photochemical destruction of the fluorophore, which allows further extension and detection of a subsequent added fluorophore. This cycle can be repeated a large number of times if sample losses are avoided. In practical applications of this approach, such losses will be avoided by attaching the primer or template strands to the fixed surface of an array device, for example a microscope slide, and transferring the entire array device between a reaction vessel and the fluorescent reader.</p>
    <heading>Example 8</heading> <p num="p-0148">Read length is defined as the maximum length of DNA sequence that can be read before uncertainties in the identities of the DNA bases exceed some defined level. In the reactive sequencing approach, read length is limited by two types of polymerase failures: misincorporation, i.e., incorrectly incorporating a noncomplementary base, and extension failure, i.e., failure to extend some fraction of the DNA primer strands on a given cycle in the presence of the correct complementary base. Example 2 demonstrated that reaction conditions can be optimized such that neither type of failure affects more than ˜1% of the arrayed strands for any given incorporation reaction. Neither type of failure directly produces an error signal in the sequence readout, because neither a 1% positive signal, for a misincorporation, nor a 1% decrease in the signal for a correct incorporation, in the case of extension failure, will be significant compared to the signals anticipated for a correct incorporation. However, accumulated failures limit the read length in a variety of different ways.</p>
    <p num="p-0149">For example, misincorporation inhibits any further extension on the affected strand resulting in a reduction in subsequent signals. It is estimated that the probability of continuing to extend a given strand following a misincorporation is no greater than 0.1%, so that any contribution to the fluorescent signal resulting from misincorporation followed by subsequent extension of the error strand will be negligible. Instead, the accumulation of misincorporations resulting in inhibition of strand extension ultimately reduces the overall signal amplitude for correct base incorporation to a level at which noise signals in the detection system begin to have a significant probability of producing a false signal that is read as a true base incorporation.</p>
    <p num="p-0150">Extension failures typically arise due to the kinetics of the extension reaction and limitations on the amount of time allotted for each extension trial with the single deoxynucleotide triphosphates (dNTP's). When reaction is terminated by flushing away the dNTP supply, a small fraction of the primer strands may remain unextended. These strands on subsequent dNTP reaction cycles will continue to extend but will be out of phase with the majority strands, giving rise to small out-of-phase signals, i.e., signaling a positive incorporation for an added dNTP which is incorrect for extension of the majority strands. Because extension failure can occur, statistically, on any extension event, the out-of-phase signals will increase as the population of strands with extension failures grows. If reaction conditions are chosen so that the reaction is 99.9% complete on a given reaction cycle, for example, after a further number, N, of successful extension reactions, the out-of-phase signal will be approximately (1-0.999<sup>N</sup>). The number N at which the out-of-phase signal becomes large enough to be incorrectly read as a correct extension signal is the read length. For example, after extension by 200 bases with 99.9% completion, the out-of-phase signal is approximately 18% of the in-phase signal, for a single base extension in either case. After extension by 400 bases the out-of-phase signal grows to 33%. The point at which the read must terminate is dictated by the ability to distinguish the in-phase signals from the out-of-phase signals.</p>
    <p num="p-0151">In what follows, a length of single base repeats, e.g. AAAAA, is treated as a single base for the purposes of discussing the phase difference between strands. If the reaction cycle of the four dNTP's is unchanged, then a primer strand which has failed to extend when the correct dNTP, for example dATP, is in the reaction cell cannot trail the leading, i.e., majority strands, which did extend correctly, by more than 3 bases because the fourth base in the dNTP reaction cycle will always once again be the correct base (dATP) for the strand which failed to extend previously. It is assumed that extension failure is purely statistical, and that any strand which fails to extend has an equal chance of subsequent extension when the correct dNTP is supplied, and that this extension probability is sufficiently high that the chance of repeated extension failures on the same strand is vanishingly small. For example, if the probability of extension failure on a single strand is 0.1%, the probability of two extension failures on the same strand is (0.001)<sup>2 </sup>or 10<sup>−6</sup>. Similarly, the trailing strand can never resynchronize with the leading strands if extension subsequently proceeds correctly, because the leading strands will always have extended by at least one more nucleotide—G, T, or C in the example discussion of an A extension failure—before the trailing strand can add the missing A. The effect is that after each complete dNTP cycle the trailing strands always follow the leading strands by an extension amount that represents the bases added in one complete dNTP cycle at a given point in the sequence. These observations predict that: (i) the gap between the leading and trailing strands perpetually oscillates between 1 and 3 bases and can never increase unless a second extension failure occurs on the same strand; and (ii) the gap between the leading and trailing strands is independent of the position along the trailing strand at which the extension failure occurs. This gap at any given point in the extension of the leading strands is solely a function of the sequence of the leading strand population up to that point and the dNTP probe cycle. In other words, a population of trailing strands is produced due to random extension failure at different points in the sequence, but these trailing strands themselves are all exactly in phase with each other.</p>
    <p num="p-0152">Because the result of an extension failure is to produce a trailing strand population that trails the leading strands perpetually by an amount that oscillates between one and three nucleotides, assuming that a second extension failure does not occur on the trailing strand and that the probing dNTP cycle remains unchanged, therefore the gap between the leading and trailing strand populations can always be known by tracking the leading strand sequence by, for example, computer simulation and simulating an extension failure event at any point along the sequence.</p>
    <p num="p-0153">Thus the present invention provides, first, a general method of computer tracking of the sequence information which allows the out-of-phase error signals due to extension of trailing strands to be recognized and subtracted from the correct signals, and, second, methods of altering the probing dNTP cycle to selectively extend the trailing strands so that they move back into phase with the leading strands, thus completely eliminating sequence uncertainty due to out-of-phase signals arising from the trailing strands that result from extension failure.</p>
    <p num="p-0154">The statistics which govern the ability to distinguish an incorrect signal from out-of-phase strands from a correct signal depend upon the noise level and statistical variation of the fluorescence signal. Assuming that the signal for a correct 1-base extension has a standard deviation of ±5%, then statistically 99.75% of the signals will have an amplitude between 0.85 and 1.15 (±3 standard deviations from the average value) when the average value is 1.0 and the standard deviation is 0.05. If the extension signal must be at least 85% of the average single extension signal to register a correct extension, then statistically a correct extension will be missed only 0.13% of the time, i.e. the readout accuracy would be 99.87%. Another 0.13% of the signals for a correct extension will be greater than 1.15, but the concern is only with signals that are lower than average and so are more difficult to distinguish from a growing signal from out-of-phase strands. The statistics for errors arising from out-of-phase extension of a trailing strand are similar. If the standard deviation of the trailing strand signals is also ±5% of the mean extension signal which will be true whenever the trailing strand intensity approaches the leading strand intensity, then if the trailing strand intensity does not grow beyond 0.7, the fraction of trailing strand extensions that give rise to a signal of 0.85 or greater 4 standard deviations beyond the mean is less than 0.01%. Thus an out-of phase signal arising from a single-base extension on one of the three sets of trailing strands should be distinguishable from the in-phase signal with accuracy so long as the out-of-phase signal does not grow beyond −70% of the in-phase signal.</p>
    <p num="p-0155">The above discussion assumes that all the extension events correspond to single base extensions. However, multiple single-base repeats are common in DNA sequences, thus one must consider the situation where the out-of-phase signal can be M times larger than that for a single base extension, where M is the repeat number. For example, if the population of one of the three sets of out-of-phase strands has grown to 20% of the leading strand population, at which level the in-phase and out-of-phase signals can readily be distinguished for a single base extension, then if this set of out-of-phase strands encounters a 5-base repeat, e.g. AAAAA, the signal for that repeat becomes identical in magnitude to that for a single base extension on the in-phase strands. Real-time computer monitoring of the extension signals permits discrimination against such repeat-enhanced out-of-phase signals, for example, by implementing linear and/or nonlinear auto-regressive moving average (ARMA) schemes. The essential points here are as follows (i) the out-of-phase strands are those that are trailing the majority strands as a result of extension failure; misincorporation events which could produce leading error strands have the effect of shutting down further extension on the affected strands and so do not give rise to significant out-of-phase error signals; (ii) there is always only one population of trailing strands regardless of where the extension failure occurred; all the primer strands in this population have been extended to the same point which trails the leading strand sequence by 1, 2 or 3 bases; and (iii) because the leading strands have always previously traversed the sequence subsequently encountered by the trailing strands, the sequence at least 1 base beyond the 3′ terminus of the trailing strands is always known and allows prediction of exactly whether, and by how much, these trailing strands will extend for any nucleotide supplied, by simulating, in a computer for example, the effect of an extension failure at any point in the known sequence upstream of the position to which the leading strands have advanced.</p>
    <p num="p-0156">On each incorporation trial, in addition to any possible correct extension signal for the leading strands, there may also be an error signal corresponding to extension of the trailing strands. For example, let us assume that the trailing strand population has grown as large as 20% of the leading strand population. The size of this population can be monitored by detecting the incorporation signal when the trailing strands extend and the leading strands do not. Assume that the leading strand population has just traversed a single base repeat region on the template, for example AAAAA, and incorporated onto the primer the complementary T repeat: TTTTT. The trailing strands will not traverse this same AAAAA repeat for at least a complete cycle of the four probing nucleotides, until the next time the strands are probed with dTTP. Knowing the size of the trailing strand population from the amplitude of its incorporation signals, determined at any point where the leading strands do not extend but the trailing strands do, the signal to be expected from the trailing strand population due to the TTTTT incorporation can be calculated precisely. If the trailing strand population is ⅕ as large as the leading strand population, for example, this signal will mimic incorporation of a single T on the leading strand population. In the absence of the computer-aided monitoring method discussed here, such a false signal would give rise to a drastic sequence error.</p>
    <p num="p-0157"> <figref idrefs="DRAWINGS">FIGS. 14A and 14B</figref> demonstrate how data would appear for a sequence: [CTGA] GM ACC AGA MG TCC [T], probed with a dNTP cycle: CAGT, close to the primer where no extension failure has occurred (<figref idrefs="DRAWINGS">FIG. 14A</figref>) and well downstream (<figref idrefs="DRAWINGS">FIG. 14B</figref>) at a point where 60% of the strands have undergone extension failure and are producing out-of-phase signals, and misincorporation has shut down extension on 75% of all strands. The readouts shown start at the second G in the sequence (beyond the [CTGA] sequence in parentheses) and end at the last C (before the [T] in parentheses). The digital nature of the signal in <figref idrefs="DRAWINGS">FIG. 14A</figref> and also the amplitude scale should be noted. In <figref idrefs="DRAWINGS">FIG. 14B</figref>, the signal for a single base extension has been reduced by 60%, from 1.0 to 0.4 due to the extension failure strands, and by a further factor of 4 to 0.1 due to misincorporation and the resulting 75% signal loss. However, added to the correct extension signals are signals due to the out-of-phase extension of the trailing strands. At first sight, the readout is completely different from the correct readout shown in <figref idrefs="DRAWINGS">FIG. 14A</figref>, due to the superposition of signals produced when the trailing strands encounter the sequence previously traversed by the leading strands. Particularly large errors arise whenever the trailing strand population encounters the AAA repeats. For example, the second T probe yields a signal amplitude corresponding to an AAAAA repeat instead of the correct single A, the third G probe gives a signal corresponding to CCC when in fact there is no C at this point in the leading strand sequence, the fourth T probe reads 4 A's when the correct sequence has none (the trailing strands encounter the second AAA repeat). However, because the sequence from the leading strands is known, the false signals arising from the trailing strands can be predicted and subtracted from the total signal to obtain the correct sequence readout. This is shown in <figref idrefs="DRAWINGS">FIG. 14C</figref>, where the signals arising from the trailing strands are coded by different shading from the leading strand signal. Because the signals due to the trailing strands can be predicted, the error signals can be subtracted to obtain the correct digital sequence readout shown in <figref idrefs="DRAWINGS">FIG. 14D</figref>. It should be noted that the data in <figref idrefs="DRAWINGS">FIG. 14D</figref> are now identical to those in <figref idrefs="DRAWINGS">FIG. 14A</figref>, and yield the correct sequence readout for the leading strands, the only difference being that the overall intensity is reduced due to the assumed loss of signal due to misincorporation and extension failure, the latter populating the trailing strands. In other words, by keeping track of the sequence in a computer the effect is as though one could directly visualize the different contributions as depicted on the plot in <figref idrefs="DRAWINGS">FIG. 14C</figref>. Therefore, it is possible to predict for any probe nucleotide event exactly what the signal from the trailing strand population should be, and subtract this error signal from the measured signal to arrive at a true digital signal representative of the sequence of the leading strand population, which is the desired result.</p>
    <p num="p-0158">Given the ability to compute and subtract any trailing strand signals as discussed, the accuracy with which nucleotide incorporation or non-incorporation on the leading strands can be sensed is limited, not by the absolute size of the trailing strand signal, but instead by the noise on those signals. For example, assume that the signal for a single-base extension of a trailing strand population equal to 20% of the leading strand population is 0.2±0.05. If the trailing strands encounter a 5-base repeat, the resulting signal would be identical in amplitude to that produced by a single-base extension of the leading strands, but this signal could be subtracted from the observed signal to yield either a signal resulting from a leading strand extension, or a null signal corresponding to no extension of the leading strands. Assuming that the noise is purely statistical and therefore is reduced in proportion to the square root of the signal amplitude, for a 5-base extension of the trailing strands or a single extension of the leading strands the signal would be 1±(0.05×√5), i.e. 1±0.11, because the statistical noise on a set of added signals grows as the square root of the number of signals. One can subtract from this value a correction signal which is much more accurately known because the trailing strand signal has been repeatedly measured yielding better statistics on this value. It is assumed that the uncertainty in the correction signal is negligible. For no extension of the leading strands, the resulting difference signal would be 0±0.11, whereas a single extension of the leading strands would yield a difference signal of 1±0.11; the two signals are distinguishable with better than 99.9% accuracy.</p>
    <p num="p-0159">The example given here is an extreme case: in fact, the extension failure can be corrected at any point, so that it will be possible to minimize the trailing strand population below a level where it would produce signals that make the leading strand sequence uncertain.</p>
    <p num="p-0160">There are additional advantages to the computer-aided monitoring method proposed. First, the signals from the trailing strands serve as an additional check on the leading strand sequence. Second, the trailing strand population could be allowed to surpass the leading strand population in magnitude. Without computer-aided monitoring, readout would have to cease well before this point, however, with computer-aided monitoring, readout can continue, now using the trailing strands rather than the leading strands to reveal the sequence. Thus, the strand population that trails due to only one extension failure now becomes the leading strand population for the purposes of computer aided monitoring. This allows readout to continue until further complications arise from the occurrence of 2 extension failures on the same strand, producing a new trailing strand population which can be tracked in the same way as the single failure strands, while the population of strands that have undergone no error failure diminishes to the point where it contributes no detectable signal.</p>
    <p num="p-0161">Optimization of reagents, enzyme and reaction conditions should allow misincorporation probabilities below 1%, and extension failure probabilities as low as 0.1%. The computer aided monitoring method of the present invention additionally provides a means for healing the trailing strand population by selectively extending this population so that it is again synchronous with the leading strands. For example, given a dNTP probe cycle of GCTA, and a template sequence (beyond the 3′ end of the primer) of:</p>
    <p num="p-0162"> <tables id="TABLE-US-00004" num="00004"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="161pt" align="left"> </colspec> <colspec colname="2" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">. . . GTGCAGATCTG . . .</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
and assuming that when dCTP is in the reaction chamber, the polymerase fails to incorporate a C in some fraction of the primer strands, the following results:
</p>
    <p num="p-0163"> <tables id="TABLE-US-00005" num="00005"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="77pt" align="left"> </colspec> <colspec colname="2" colwidth="126pt" align="left"> </colspec> <colspec colname="3" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Template</td> <td class="description-td">. . . GTG CAG ATC TG . . .</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Main strands</td> <td class="description-td">. . . C</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Template</td> <td class="description-td">. . . GTG CAG ATC TG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Failure strands</td> <td class="description-td">. . .</td> </tr> </tbody> </tgroup> </table> </tables> <br>
At the end of the first cycle, the main strands have extended by . . . CA, while the failure strand has not advanced. After one more complete cycle, the main strand extension is . . . C and the failure strand now reads . . . CA, i.e. now just one base out of phase.
</p>
    <p num="p-0164"> <tables id="TABLE-US-00006" num="00006"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="77pt" align="left"> </colspec> <colspec colname="2" colwidth="126pt" align="left"> </colspec> <colspec colname="3" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Template</td> <td class="description-td">. . . GTG CAG ATC TG . . .</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Main strands</td> <td class="description-td">. . . CAC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Template</td> <td class="description-td">. . . GTG CAG ATC TG . . .</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Failure strands</td> <td class="description-td">. . . CA</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Because the phase lag arises from the repeating interaction of the probe cycle sequence with the template sequence, the unchanged probe cycle can never have the correct sequence to resynchronise the strands. Instead, if the probe cycle is unchanged, and if no further extension failures occur, the phaase lag for a given failure strand oscillates perpetually between 1 and 3 bases, counting single base repeats as one base for this purpose. However because the leading strand sequence up to the last extension is always known, one can determine the effect of introducing an extension failure at some upstream position. It should be noted that an extension failure introduced at any arbitrary upstream position, or any base type, always produces the same phase lag because the effect of an extension failure is to cause extension of the afflicted strand to lag by one complete dNTP cycle. Thus, it is possible to alter the probe cycle sequence, for example to probe with a C, instead of a G, after the last A in the sequence discussed above. The failure strand would advance while the main strands did not and the phase lag would heal. In yet another embodiment the dNTP probe cycle may be reversed whenever the phase lag shrinks to only 1 base. Whenever the phase difference declines to a single base, or repeats of a single base, then simply reversing the probe cycle sequence always resynchronizes the strands.
</p>
    <p num="p-0165"> <figref idrefs="DRAWINGS">FIG. 15</figref> shows how a leading strand population arising from incorrect extension of a fraction of primer strands due to nucleotide impurities can adversely affect the signals from the main population. Using the same template sequence as before: [CTGA] GM ACC AGA AA GTC C [TC AGT] and the same probe cycle: CAGT, the effect of a leading strand population which is 20% of the main strand population can be simulated and 2 bases ahead of the main strands at the time the main strand sequence begins to be read. The leading strands have already extended by -C TTT. The first C probe extends the main primer strands by one base complementary to the first G in the sequence giving a single base extension signal of 1. The first G extends the leading strands by -GG- complementary to the -CC- repeat, giving a signal of 0.4. Greater ambiguity arises when the leading strands encounter the second AAA-repeat at the second T probe, increasing the main strand signal from the correct value for a single base extension to 1.6. In the absence of further information, this value will be ambiguous or may be interpreted as a 2-base repeat.</p>
    <p num="p-0166">Correction for these ambiguities comes from the fact that the correct sequence of the main strands is read following the leading strand read. In general, a large multiple repeat which can give an error signal when encountered by the leading strands will subsequently give a larger signal when encountered by the main strands, and superimposed on this correct signal will be a leading strand signal for which there are three possibilities: (i) zero signal: the leading strands do not extend; (ii) small signal that does not create ambiguity—the leading strands extend by a single base or a repeat number small enough not to simulate an additional base extension of the main strands; (iii) large signal; the leading strands encounter a second large repeat. By monitoring the main strand sequence, it is possible at each extension to retroactively estimate the effects of a leading strand population and subtract such signals from the main strand signals to arrive at a correct sequence. In the case where the leading strands encounter a repeat large enough to create ambiguity in the sequence, even if the leading strands subsequently encounter a second or third large repeat when the main strands encounter the first repeat, the main strands will eventually traverse the same region to give sufficient information to derive the correct sequence. In other words, the sequence information at any point is always overdetermined—the signal for any given extension is always read twice, by the leading strands and the main strands, and so yields sufficient information to determine both the correct sequence and the fractional population of the leading strands, which are the two pieces of information required.</p>
    <p num="p-0167">Because the sequence of the leading strand population produced by impure nucleotides cannot be known until it is subsequently traversed by the main strands, one cannot know what dNTP probe cycle would act to extend the main strands while not extending the leading strands, as was the case for a trailing strand population produced by extension failure. However, as with trailing strands, the gap between the leading and main strands oscillates perpetually between one and three bases, and can be reconnected by reversing the dNTP probe sequence whenever the gap between the leading and main strands shrinks to a single base. Although it cannot be known when this single base gap occurs, the dNTP probe sequence can be reversed at regular intervals. Trials indicate that such a process ultimately reconnects approximately ⅔ of the leading strands. The statistics for this process are as follows.</p>
    <p num="p-0168">Statistically, because the gap between the main and leading strands can be 1, 2 or 3 bases, there is a ⅓ probability that the leading strand population will have only a 1-base phase lag at any time the cycle is reversed. The 1-base phase difference will always be healed by a cycle reversal. Another ⅓ of the time the leading strands are 2 bases ahead at the time the cycle is reversed. For the next probing base the following possibilities exist:</p>
    <p num="p-0169">
      <tables id="TABLE-US-00007" num="00007"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="28pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="161pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Lead</td>
                <td class="description-td">Main</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">strand</td>
                <td class="description-td">strand</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="161pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">0</td>
                <td class="description-td">0</td>
                <td class="description-td">No extension on either strand: Prob. ¾ × ¾ = 9/16</td>
              </tr> <tr class="description-tr"> <td class="description-td">+1</td>
                <td class="description-td">0</td>
                <td class="description-td">Phase lag increases: Prob. ¼ × ¾ = 3/12</td>
              </tr> <tr class="description-tr"> <td class="description-td">+1</td>
                <td class="description-td">+1</td>
                <td class="description-td">Both strands advance: Prob. ¼ × ¼ = 1/16</td>
              </tr> <tr class="description-tr"> <td class="description-td">0</td>
                <td class="description-td">+1</td>
                <td class="description-td">Phase lag decreases: Prob. ¾ × ¼ = 3/12</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0170"> <tables id="TABLE-US-00008" num="00008"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="91pt" align="left"> </colspec> <colspec colname="3" colwidth="105pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Phase lag stays at 2:</td> <td class="description-td">Number of chances = 10/16</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Phase lag decreases</td> <td class="description-td">Number of chances = 3/12</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Phase lag increases</td> <td class="description-td">Number of chances = 3/12</td> </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
So the chance of making a 2-base gap worse is (3/12)/(10/16+3/12)=28% three gap sizes: 1-base gap heals (33% of population); 2-base gap gets worse 28% of the time: only ⅓ of gaps are 2 base, so 9% total get worse; 3-base gap also gets worse 28 of the time, again 9% overall effect. In sum, 33% heal at a given reversal, 18% lose at a given reversal and the remaining 50% are unchanged. Even assuming the 18% are permanently lost (and a 2 base gap increased to a 3 base gap can still rejoin), at each subsequent reversal ⅓ of the 50% of strands are healed, which are unchanged by the previous reversal, as follow
</p>
    <p num="p-0171"> <tables id="TABLE-US-00009" num="00009"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="147pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Reversal #</td> <td class="description-td">Fraction of gaps healed</td> </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="84pt" align="right"> </colspec> <colspec colname="4" colwidth="63pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">1</td> <td class="description-td">33%</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">2</td> <td class="description-td">17%</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">3</td> <td class="description-td">9%</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">4</td> <td class="description-td">4.5%</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">5</td> <td class="description-td">2.5%</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">6</td> <td class="description-td">1%</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Total</td> <td class="description-td">~67%</td> </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
Therefore, repeated reversal of the dNTP probe cycle can reduce by ⅔ the phase signals due to incorrect extension by nucleotide impurities, or random effectively increasing the read length when limited by either effect by a factor of 3.
</p>
    <p num="p-0172">Although the invention has been described herein with reference to specific embodiments, many modifications and variations therein will readily occur to those skilled in the art. Accordingly, all such variations and modifications are included within the intended scope of the invention.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5436149">US5436149</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 19, 1993</td><td class="patent-data-table-td patent-date-value">Jul 25, 1995</td><td class="patent-data-table-td ">Barnes; Wayne M.</td><td class="patent-data-table-td ">Thermostable DNA polymerase with enhanced thermostability and enhanced length and efficiency of primer extension</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0223618A2?cl=en">EP0223618A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 16, 1986</td><td class="patent-data-table-td patent-date-value">May 27, 1987</td><td class="patent-data-table-td ">New York Medical College</td><td class="patent-data-table-td ">Automatable process for nucleotide sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1989009283A1?cl=en">WO1989009283A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 3, 1989</td><td class="patent-data-table-td patent-date-value">Oct 5, 1989</td><td class="patent-data-table-td ">Edward David Hyman</td><td class="patent-data-table-td ">Pyrophosphate-based method and apparatus for sequencing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1991006678A1?cl=en">WO1991006678A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 26, 1990</td><td class="patent-data-table-td patent-date-value">Apr 27, 1991</td><td class="patent-data-table-td ">Stanford Res Inst Int</td><td class="patent-data-table-td ">Dna sequencing</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8216514">US8216514</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 8, 2011</td><td class="patent-data-table-td patent-date-value">Jul 10, 2012</td><td class="patent-data-table-td ">Arizona Board Of Regents</td><td class="patent-data-table-td ">Method of determining the nucleotide sequence of Oligonucleotides and DNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8666678">US8666678</a></td><td class="patent-data-table-td patent-date-value">Oct 27, 2011</td><td class="patent-data-table-td patent-date-value">Mar 4, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Predictive model for use in sequencing-by-synthesis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20120258449">US20120258449</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 29, 2012</td><td class="patent-data-table-td patent-date-value">Oct 11, 2012</td><td class="patent-data-table-td ">Arizona Board Of Regents</td><td class="patent-data-table-td ">Method of determining the nucleotide sequence of oligonucleotides and dna molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013186306A1?cl=en">WO2013186306A1</a></td><td class="patent-data-table-td patent-date-value">Jun 13, 2013</td><td class="patent-data-table-td patent-date-value">Dec 19, 2013</td><td class="patent-data-table-td ">Boehringer Ingelheim International Gmbh</td><td class="patent-data-table-td ">Method for identifying transcriptional regulatory elements</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc422/defs422.htm&usg=AFQjCNHXLT_uXJvUp9ERMJkh6x6moeyOWg#C422S050000">422/50</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006100">435/6.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006180">435/6.18</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0019340000">C12P19/34</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021000000">C07H21/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6816">C12Q1/6816</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=3hO8BgABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6869">C12Q1/6869</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68E</span>, <span class="nested-value">C12Q1/68B2</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Mar 25, 2014</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 1-7, 12 AND 13 ARE CANCELLED.CLAIMS 8-11 AND 14-30 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">LIFE TECHNOLOGIES CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20130823</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:ARIZONA BOARD OF REGENTS FOR AND ON BEHALF OF ARIZONA STATE UNIVERSITY;REEL/FRAME:031159/0626</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 17, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120417</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BROWN RUDNICK LLP, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">NOTICE OF ATTORNEY S LIEN;ASSIGNOR:HELICOS BIOSCIENCES CORPORATION;REEL/FRAME:028060/0898</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 2, 2011</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20110607</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 28, 2008</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ARIZONA BOARD OF REGENTS, ARIZONA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:WILLIAMS, PETER;TAYLOR, THOMAS J.;WILLIAMS, DANIEL J.B.;AND OTHERS;REEL/FRAME:021457/0949;SIGNING DATES FROM 20020208 TO 20020218</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:WILLIAMS, PETER;TAYLOR, THOMAS J.;WILLIAMS, DANIEL J.B.;AND OTHERS;SIGNING DATES FROM 20020208 TO 20020218;REEL/FRAME:021457/0949</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_4ff636b3d23669b7103f3b3a3a18b4cd.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0b7afrcD30-mfZGkaMsmd_1sGoHg\u0026id=3hO8BgABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U1atv4Ui9V3anSWbo0N3Uo3PJABVw\u0026id=3hO8BgABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U37Dwn85Yp1rEtA7Il98OOPQQWgew","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Using_polymerase_with_reduced_exonucleas.pdf?id=3hO8BgABERAJ\u0026output=pdf\u0026sig=ACfU3U0o30QFdWOfnb2A8lL6lCpXCKqtgw"},"sample_url":"http://www.google.com/patents/reader?id=3hO8BgABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>