<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US8021867 - Rationally-designed meganucleases with altered sequence specificity and DNA ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_4ff636b3d23669b7103f3b3a3a18b4cd/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_4ff636b3d23669b7103f3b3a3a18b4cd__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Rationally-designed meganucleases with altered sequence specificity and DNA-binding affinity"><meta name="DC.contributor" content="James J. Smith" scheme="inventor"><meta name="DC.contributor" content="Derek Jantz" scheme="inventor"><meta name="DC.contributor" content="Homme W. Hellinga" scheme="inventor"><meta name="DC.contributor" content="Duke University" scheme="assignee"><meta name="DC.date" content="2006-10-18" scheme="dateSubmitted"><meta name="DC.description" content="Rationally-designed LAGLIDADG (SEQ ID NO: 37) meganucleases and methods of making such meganucleases are provided. In addition, methods are provided for using the meganucleases to generate recombinant cells and organisms having a desired DNA sequence inserted into a limited number of loci within the genome, as well as methods of gene therapy, for treatment of pathogenic infections, and for in vitro applications in diagnostics and research."><meta name="DC.date" content="2011-9-20" scheme="issued"><meta name="DC.relation" content="EP:1485475:A1" scheme="references"><meta name="DC.relation" content="US:20020107214:A1" scheme="references"><meta name="DC.relation" content="US:20020110898:A1" scheme="references"><meta name="DC.relation" content="US:20030113887:A1" scheme="references"><meta name="DC.relation" content="US:20030175968:A1" scheme="references"><meta name="DC.relation" content="US:20030182670:A1" scheme="references"><meta name="DC.relation" content="US:20030229039:A1" scheme="references"><meta name="DC.relation" content="US:20040002092:A1" scheme="references"><meta name="DC.relation" content="US:20040019002:A1" scheme="references"><meta name="DC.relation" content="US:20040068761:A1" scheme="references"><meta name="DC.relation" content="US:20040171154:A1" scheme="references"><meta name="DC.relation" content="US:20050026157:A1" scheme="references"><meta name="DC.relation" content="US:20050032223:A1" scheme="references"><meta name="DC.relation" content="US:20050064474:A1" scheme="references"><meta name="DC.relation" content="US:20050172365:A1" scheme="references"><meta name="DC.relation" content="US:20050208489:A1" scheme="references"><meta name="DC.relation" content="US:20060078552:A1" scheme="references"><meta name="DC.relation" content="US:20060153826:A1" scheme="references"><meta name="DC.relation" content="US:20060206949:A1" scheme="references"><meta name="DC.relation" content="US:20060282914:A1" scheme="references"><meta name="DC.relation" content="US:5436150" scheme="references"><meta name="DC.relation" content="US:5474896" scheme="references"><meta name="DC.relation" content="US:5789215" scheme="references"><meta name="DC.relation" content="US:5792632" scheme="references"><meta name="DC.relation" content="US:5792640" scheme="references"><meta name="DC.relation" content="US:5830729" scheme="references"><meta name="DC.relation" content="US:5866361" scheme="references"><meta name="DC.relation" content="US:5948678" scheme="references"><meta name="DC.relation" content="US:5962327" scheme="references"><meta name="DC.relation" content="US:6238924" scheme="references"><meta name="DC.relation" content="US:6265196" scheme="references"><meta name="DC.relation" content="US:6395959" scheme="references"><meta name="DC.relation" content="US:6566579" scheme="references"><meta name="DC.relation" content="US:6610545" scheme="references"><meta name="DC.relation" content="US:6794136" scheme="references"><meta name="DC.relation" content="US:6822137" scheme="references"><meta name="DC.relation" content="US:6833252" scheme="references"><meta name="DC.relation" content="US:7214536" scheme="references"><meta name="DC.relation" content="US:7271000" scheme="references"><meta name="DC.relation" content="US:7309605" scheme="references"><meta name="DC.relation" content="WO:1996014408:A2" scheme="references"><meta name="DC.relation" content="WO:2000046386:A2" scheme="references"><meta name="DC.relation" content="WO:2003078619:A1" scheme="references"><meta name="DC.relation" content="WO:2004031346:A2" scheme="references"><meta name="DC.relation" content="WO:2004067736:A2" scheme="references"><meta name="DC.relation" content="WO:2004067753:A2" scheme="references"><meta name="DC.relation" content="WO:2005049842:A2" scheme="references"><meta name="DC.relation" content="WO:2005105989:A1" scheme="references"><meta name="DC.relation" content="WO:2006097784:A1" scheme="references"><meta name="DC.relation" content="WO:2006097853:A1" scheme="references"><meta name="DC.relation" content="WO:2006097854:A1" scheme="references"><meta name="DC.relation" content="WO:2007034262:A1" scheme="references"><meta name="citation_reference" content="Aagaard et al (1997), Profile of the DNA recognition site of the archaeal homing endonuclease I-Dmol, Nucleic Acids Res. 25(8):1523-1530."><meta name="citation_reference" content="Adler, David A. et al. &quot;Bioinformatics.&quot; Encyclopedia of Life Sciences. No month listed-2001. John Wiley &amp; Sons Ltd. pp. 1-8."><meta name="citation_reference" content="Adler, David A. et al. “Bioinformatics.” Encyclopedia of Life Sciences. No month listed—2001. John Wiley &amp; Sons Ltd. pp. 1-8."><meta name="citation_reference" content="Aggarwal et al., Novel site-specific DNA endonucleases, Structural Biology, 1998, pp. 19-25, vol. 8."><meta name="citation_reference" content="Argast et al., I-Ppol and I-Crel homing site sequence degeneracy determined by random mutagenesis and sequential in vitro enrichment, J. Mol. Biol., 1998, pp. 345-353, vol. 280."><meta name="citation_reference" content="Arnould et al., Engineering of large numbers of highly specific homing endonucleases that induce recombination of novel DNA targets, J. Mol. Biol., 2006, pp. 443-458, vol. 355."><meta name="citation_reference" content="Ashworth et al. Jun. 2006. Computational redesign of endonucleases DNA binding and cleavage specificity. Nature, 441, 656-659."><meta name="citation_reference" content="Belfort et al., Homing enconucleases: keeping the house in order, Nucleic Acids Research, 1997, vol. 25, No. 17, pp. 3379-3388."><meta name="citation_reference" content="Beumer et al., Efficient gene targeting in Drosophila with zinc finger nucleases, Genetics, Feb. 1, 2006, pp. 1-48, 10.1534/genetics.105.052829."><meta name="citation_reference" content="Bibikova et al., Enhancing gene targeting with designed zinc finger nucleases, Science, May 2, 2003, p. 764, vol. 300."><meta name="citation_reference" content="Bibikova et al., Stimulation of homologous recombination through targeted cleavage by chimeric nucleases, Molecular and Cellular Biology, Jan. 2001, pp. 289-297, vol. 21 No. 1."><meta name="citation_reference" content="Bibikova et al., Targeted chromosomal cleavage and mutagenesis in Drosophila using zinc-finger nucleases, Genetics, Jul. 2002, pp. 1169-1175, vol. 161."><meta name="citation_reference" content="Bolduc et al. Revised Sep. 2003. Structural and biochemical analyses of DNA and RNA binding by a bifunctional homing endonuclease and group I intron splicing factor. Genes Dev, 17, 2875-2888."><meta name="citation_reference" content="Chames et al., In vivo selection of engineered homing endonucleases using double-strand break induced homologous recombination, Nucleic Acids Research, 2005, pp. 1-10, vol. 33 No. 20 e178."><meta name="citation_reference" content="Chen et al., A highly sensitive selection method for directed evolution of homing endonucleases, Nucleic Acids Research, 2005, pp. 1-7, vol. 33 No. 18 e154."><meta name="citation_reference" content="Chen, R. Jan. 1, 2001. Enzyme engineering: rational redesign versus directed evolution. Trends Biotechnol, 19, 13-14."><meta name="citation_reference" content="Chevalier B. S. et al. &quot;Homing Endonucleases: Structural and Functional Insight Into the Catalysts if Intron/Intein Mobility.&quot; Nucleic Acids Research. Oxford University Press. Surrey, Great Britain. vol. 29, No. 18, Sep. 15, 2001. pp. 3757-3774."><meta name="citation_reference" content="Chevalier et al. (2002), Design, Activity, and Structure of a Highly Specific Artificial Endonuclease, Mol. Cell 10:895-905."><meta name="citation_reference" content="Chevalier et al., Design, activity, and structure of a highly specific artificial endonuclease, Molecular Cell, Oct. 2002, pp. 895-905, vol. 10."><meta name="citation_reference" content="Chevalier et al., Flexible DNA target site recognition by divergent homing endoculease isoschizomers I-Crel and I-Msol, J. Mol. Biol., 2003, pp. 253-269, vol. 329."><meta name="citation_reference" content="Chevalier et al., Metal-dependent DNA cleavage mechanism of the I-Crel LAGLIDADG homing endonuclease, Biochemistry, 2004, pp. 14015-14026, vol. 43."><meta name="citation_reference" content="Chevalier et al., The homing endonuclease I-Crel uses three metals, one of which is shared between the two active sites, Nature Structural Biology, Apr. 2001, pp: 312-316, vol. 8 No. 4."><meta name="citation_reference" content="Chilton et al., Targeted integration of T-DNA into the tobacco genome at double-stranded breaks: new insights on the mechanism of T-DNA integration, Plant Physiology, Nov. 2003, pp. 956-965, vol. 133."><meta name="citation_reference" content="Cozzone, Alain J. &quot;Proteins: Fundamental Chemical Properties.&quot; Encyclopedia of Life Sciences. No month listed-2002. John Wiley &amp; Sons Ltd. pp. 1-10."><meta name="citation_reference" content="Cozzone, Alain J. “Proteins: Fundamental Chemical Properties.” Encyclopedia of Life Sciences. No month listed—2002. John Wiley &amp; Sons Ltd. pp. 1-10."><meta name="citation_reference" content="Dalgaard, Jacob Z. et al. &quot;A Site-Specific Endonuclease Encoded by a Typical Archaeal Intron.&quot; Proc. Natl. Acad. Sci. Jun. 1993. vol. 90. pp. 5414-5417."><meta name="citation_reference" content="Desjarlais et al., Use of a zinc-finger consensus sequence framework and specificity rules to design specific DNA binding proteins, Proc. Natl. Acad. Sci. USA, Mar. 1993, pp. 2256-2260, vol. 90."><meta name="citation_reference" content="Dhanasekaran et al., Designer zinc finger proteins: tools for creating artificial DNA-binding functional proteins, Acc. Chem. Res., 2006, pp. 45-52, vol. 30 No. 1."><meta name="citation_reference" content="Doyon et al., Directed evolution and substrate specificity profile of homing endonuclease I-Scel, J. Am. Chem. Soc., 2006, pp. 2477-2484, vol. 128."><meta name="citation_reference" content="Duan et al. May 16, 1997. Crystal structure of PI-Scel, a homing endonuclease with protein splicing activity. Cell, 89, 555-564."><meta name="citation_reference" content="Durai et al. Oct. 26, 2005. Zinc finger nucleases: custom-designed molecular scissors for genome engineering of plant and mammalian cells. Nucleic Acids Res, 33, 5978-5990."><meta name="citation_reference" content="Epinat et al., A novel engineered meganuclease induces homologous recombination in yeast and mammalian cells, Nucleic Acids Research, 2003, pp. 2952-2962, vol. 31 No. 11."><meta name="citation_reference" content="European Office Action, European Patent Application No. 06 826 293.0, dated Nov. 20, 2008 (3 pages)."><meta name="citation_reference" content="European Search Report for European Patent Application No. 10191885.2 mailed Apr. 26, 2011. 12 pages."><meta name="citation_reference" content="European Search Report for European Patent Application No. 10191888.6 mailed May 18, 2011. 5 pages."><meta name="citation_reference" content="European Search Report for European Patent Application No. 10191904.1 mailed May 13, 2011. 9 pages."><meta name="citation_reference" content="Fitzsimons Hall et al., Creation of an artificial bifunctional intein by grafting a homing endonuclease into a mini-intein, J. Mol. Biol., 2002, pp. 173-179, vol. 323."><meta name="citation_reference" content="Gimble et al. Oct. 2003. Assessing the plasticity of DNA target site recognition of the PI-Scel homing endonuclease using a bacterial two-hybrid selection system. J Mol Biol, 334, 993-1008."><meta name="citation_reference" content="Gimble, F.S. Engineering Homing Endonulceases for Genomic Applications. In Homing Endonucleases and Inteins. Belfort, Derbyshire, Stoddard and Woods, Eds. Springer-Verlag, Berlin Heidelberg. No month listed-2005. 16 pages."><meta name="citation_reference" content="Gimble, F.S. Engineering Homing Endonulceases for Genomic Applications. In Homing Endonucleases and Inteins. Belfort, Derbyshire, Stoddard and Woods, Eds. Springer-Verlag, Berlin Heidelberg. No month listed—2005. 16 pages."><meta name="citation_reference" content="Grindl et al., The protein splicing domain of the homing endonuclease PI-Scel is responsible for specific DNA binding, Nucleic Acids Research, 1998, vol. 26, No. 8, pp. 1857-1862."><meta name="citation_reference" content="Guhan et al., Structural and functional characteristics of homing endonucleases, Critical Reviews in Biochemistry and Molecular Biology, 2003, pp. 199-248, vol. 38 No. 3."><meta name="citation_reference" content="Harris, J.L. and Craik, C.S. Engineering enzyme specificity. Curr Opin Chem Biol, 2, 127-132. No month listed-1998."><meta name="citation_reference" content="Harris, J.L. and Craik, C.S. Engineering enzyme specificity. Curr Opin Chem Biol, 2, 127-132. No month listed—1998."><meta name="citation_reference" content="Heath, Patrick J. et al. &quot;The Structure of I-Crel, A Group I Intron-encoded Homing Endonuclease.&quot; Nature Structural Biology. vol. 4, No. 6. Jun. 1997. pp. 468-476."><meta name="citation_reference" content="Hu et al. &quot;Probing the Structure of the PI-Seel-DNA Complex by Affinity Cleavage and Affinity Photcross-linking.&quot; The Journal of Biological Chemistry. Jan. 28, 2000. vol. 275, No. 4. pp. 2705-2712."><meta name="citation_reference" content="Ichiyanagi et al., Crystal structure of an archaeal intein-encoded homing endonuclease PI-Pful, J. Mol. Biol., 2000, pp. 889-901, vol. 300."><meta name="citation_reference" content="International Search Report from PCT/US2006/040919, mailed Aug. 29, 2007."><meta name="citation_reference" content="Jurica et al. (1998), DNA Recognition and Cleavage by the LAGLIDADG Homing Endonuclease I-Crel, Mol. Cell 2:469-476."><meta name="citation_reference" content="Jurica et al., DNA recognication and cleavage by the LAGLIDADG homing endonuclease I-Crel, Molecular Cell, Oct. 1998, pp. 469-476, vol. 2."><meta name="citation_reference" content="Jurica, M.S. et al. &quot;Homing Endonucleases: Structure, Function and Evolution.&quot; CMLS: Cellular and Molecular Life Sciences. Feb. 1999. vol. 55. pp. 1304-1326."><meta name="citation_reference" content="Liang et al. &quot;Genetic Fusion of Subunits of a Dimeric Protein Substantially Enhances its Stability and Rate of Folding.&quot; Proc. Natl. Acad. Sci. vol. 90. Aug. 1993. pp. 7010-7014."><meta name="citation_reference" content="Lloyd et al., Targeted mutagenesis using zinc-finger nucleases in Arabidopsis, PNAS, Feb. 8, 2005, pp. 2232-2237, vol. 102 No. 6."><meta name="citation_reference" content="Looger, L. et al., Computational Design of Receptor and Sensor Proteins with Novel Functions, Nature, 2003, pp. 185-190, vol. 423 No. 6936."><meta name="citation_reference" content="Lucas et al., Rapid evolution of the DNA-binding site in LAGLIDADG homing endonucleases, Nucleic Acids Research, 2001, pp. 960-969, vol. 29 No. 4."><meta name="citation_reference" content="Maggert et al., Highly efficient sex chromosome interchanges produced by I-Crel expression in Drosophila, Genetics, Nov. 2005, pp. 1103-1114, vol. 171."><meta name="citation_reference" content="Mani et al., Binding of two zinc finger nuclease monomers to two specific sites is required for effective double-strand DNA cleavage, Biochemical and Biophysical Research Communications, 2005, pp. 1191-1197, vol. 334."><meta name="citation_reference" content="Mani et al., Design, engineering, and characterization of zinc finger nucleases, Biochemical and Biophysical Research Communications, 2005, pp. 447-457, vol. 335."><meta name="citation_reference" content="Matsumura et al. Feb. 21, 2006. Crystal structure of intein homing endonuclease II encoded in DNA polymerase gene from hyperthermophilic archaeon Thermococcus kodakaraensis strain KOD1. Proteins, 63, 711-715."><meta name="citation_reference" content="McDaniel et al., Advances in synthetic biology: on the path from prototypes to applications, Current Opinion in Biotechnology, 2005, pp. 476-483, vol. 16."><meta name="citation_reference" content="Monnat et al., Generation of highly site-specific DNA double-strand breaks in human cells by the homing endonucleases I-Ppol and I Crel, Biochemical and Biophysical Research Communications, 1999, pp. 88-93, vol. 255."><meta name="citation_reference" content="Moure et al. Sep. 2003. The crystal structure of the gene targeting homing endonuclease I-Scel reveals the origins of its target site specificity. J Mol Biol, 334, 685-695."><meta name="citation_reference" content="Moure et al., Crystal structure of the intein homing endonuclease PI-Scel bound to its recognition sequence, Nature Structural Biology, Oct. 2002, pp. 764-770, vol. 9 No. 10."><meta name="citation_reference" content="Moure, Carmen M. et al. &quot;Crystal Structure of the Intein Homing Endonuclease PI-Scel Bound to its Recognition Sequence.&quot; Howard Hughes Medical Institute, Baylor College of Medicine, Houston Texas. Nature Publishing Group. Sep. 3, 2002. 7 pages."><meta name="citation_reference" content="Pabo et al., Transcription factors: structural families and principles of DNA recognition, Annu. Rev. Biochem., 1992, pp. 1053-1095, vol. 61."><meta name="citation_reference" content="Pace, C. Nick et al. &quot;Protein Stability.&quot; Encyclopedia of Life Sciences. No month listed-2001. John Wiley &amp; Sons Ltd. pp. 1-4."><meta name="citation_reference" content="Pace, C. Nick et al. “Protein Stability.” Encyclopedia of Life Sciences. No month listed—2001. John Wiley &amp; Sons Ltd. pp. 1-4."><meta name="citation_reference" content="Poland, Bradley W. et al. &quot;Structural Insights into the Protein Splicing Mechanism of PI-Scel.&quot; The Journal of Biological Chemistry. Jun. 2000. vol. 275, No. 22. pp. 16408-16413."><meta name="citation_reference" content="Porteus et al., Gene targeting using zinc finger nucleases, Nature Biotechnology, Aug. 2005, pp. 967-973, vol. 23 No. 8."><meta name="citation_reference" content="Porteus, Mammalian gene targeting with designed zinc finger nucleases, Molecular Therapy, Feb. 2006, pp. 438-446, vol. 13 No. 2."><meta name="citation_reference" content="Prieto et al., Nucleic Acids Research 35(10):3262-3271, 2007."><meta name="citation_reference" content="Puchta et al., Two different but related mechanisms are used in plants for the repair of genomic double-strand breaks by homologous recombination, Proc. Natl. Acad. Sci. USA, May 1996, pp. 5055-5060, vol. 93."><meta name="citation_reference" content="Rong et al., Targeted mutagenesis by homologous recombination in D. melanogaster, Genes &amp; Dev., 2002, pp. 1568-1581, vol. 16."><meta name="citation_reference" content="Rosen et al. Sep. 13, 2006. Homing endonuclease I-Crel derivatives with novel DNA target specificities. Nucleic Acids Res, 34, 4791-4800."><meta name="citation_reference" content="Rouet et al., Introduction of double-strand breaks into the genome of mouse cells by expression of a rare-cutting endonuclease, Molecular and Cellular Biology, Dec. 1994, pp. 8096-8106, vol. 14 No. 12."><meta name="citation_reference" content="Sali, Andrej et al. &quot;Comparative Protein Modelling by Satisfaction of Spatial Restraints.&quot; J. Mol. Biol. Jul. 1993. pp. 779-815."><meta name="citation_reference" content="Seligman et al. (2002), Mutations altering the cleavage specificity of a homing endonuclease, Nucleic Acids Res. 30(17): 3870-3879."><meta name="citation_reference" content="Seligman et al., Genetic Analysis of the Chlamydomonas reinhardtii I-Crel mobile intron homing system in Escherichia coli, Genetics, Dec. 1997, pp. 1653-1664, vol. 147."><meta name="citation_reference" content="Seligman et al., Mutations altering the cleavage specificity of a homing endonuclease, Nucleic Acids Research, 2002, pp. 3870-3879, vol. 30 No. 17."><meta name="citation_reference" content="Shen et al. Jul. 2004. DNA binding and cleavage by the HNH homing endonuclease I-Hmul. J Mol Biol, 342, 43-56."><meta name="citation_reference" content="Silva et al., Analysis of the LAGLIDADG interface of the monomeric homing endonuclease I-Dmol, Nucleic Acids Research, 2004, pp. 3156-3168, vol. 32 No. 10."><meta name="citation_reference" content="Silva et al., Crystal structure of the thermostable archaeal intron-encoded endonuclease I-Dmol, J. Mol. Biol., 1999, pp. 1123-1136, vol. 286."><meta name="citation_reference" content="Silva, George H. et al. &quot;From Monomeric to Homodimeric Endonucleases and Back: Engineering Novel Specificity of LAGLIDADG Enzymes.&quot; J. Mol. Biol. Jul. 2006. pp. 744-754."><meta name="citation_reference" content="Smith et al., A combinatorial approach to create artificial homing endonucleases cleaving chosen sequences, Nucleic Acids Research, vol. 00, No. 00, Nov. 27, 2006, pp. 1-12."><meta name="citation_reference" content="Smith et al., A detailed study of the substrate specificity of a chimeric restriction enzyme, Nucleic Acids Research, 1999, pp. 674-681, vol. 27 No. 2."><meta name="citation_reference" content="Smith et al., Requirements for double-strand cleavage by chimeric restriction enzymes with zinc finger DNA-recognition domains, Nucleic Acids Research, 2000, pp. 3361-3369, vol. 28 No. 17."><meta name="citation_reference" content="Spiegel et al., The structure of I-Ceul homing endonuclease: evolving asymmetric DNA recognition from a symmetric protein scaffold, Structure, May 2006, pp. 869-880, vol. 14."><meta name="citation_reference" content="Stoddard, Homing endonuclease structure and function, Quarterly Reviews of Biophysics, 2005, pp. 1-47."><meta name="citation_reference" content="Sussman et al., Isolation and characterization of new homing endonuclease specificities at individual target site positions, J. Mol. Biol., 2004, pp. 31-41, vol. 342."><meta name="citation_reference" content="Turmel, Monique et al. &quot;Evolutionary Conserved and Functionally Important Residues in the I-Ceul Homing Endonuclease.&quot; Nucleic Acid Research. Apr. 1997. vol. 25, No. 13. pp. 2610-2619."><meta name="citation_reference" content="Tzfira et al., Towards targeted mutagenesis and gene replacement in plants, Trends in Biotechnology, Dec. 2005, pp. 567-569, vol. 23 No. 12."><meta name="citation_reference" content="Urnov et al., Designed transcription factors as structural, functional and therapeutic probes of chromatin in vivo, EMBO Reports, 2002, pp. 610-615, vol. 3 No. 7."><meta name="citation_reference" content="Urnov et al., Highly efficient endogenous human gene correction using designed zinc-finger nucleases, Nature, Jun. 2, 2005, pp. 646-651, vol. 435."><meta name="citation_reference" content="Wang et al., Purification, biochemical characterization and protein-DNA interactions of the I-Crel endonuclease produced in Escherichia coli, Nucleic Acids Research, 1997, pp. 3767-3776, vol. 25 No. 19."><meta name="citation_reference" content="Wende et al. (1996), Binding, bending and cleavage of DNA substrates by the homing endonuclease PI-Scel, Nucleic Acid Res. 24(21): 4123-4132."><meta name="citation_reference" content="Werner, Erik et al. &quot;High Resolution Crystal Structure of Domain I of the Saccharomyces cerevisiae Homing Endonuclease PI-Scel.&quot; Nucelic Acid Research. Jul. 2002. vol. 30, No. 18. pp. 3962-3971."><meta name="citation_reference" content="Wright et al., High-frequency homologous recombination in plants mediated by zinc-finger nucleases, The Plant Journal, 2005, pp. 693-705, vol. 44."><meta name="citation_patent_number" content="US:8021867"><meta name="citation_patent_application_number" content="US:11/583,368"><link rel="canonical" href="http://www.google.com/patents/US8021867"/><meta property="og:url" content="http://www.google.com/patents/US8021867"/><meta name="title" content="Patent US8021867 - Rationally-designed meganucleases with altered sequence specificity and DNA-binding affinity"/><meta name="description" content="Rationally-designed LAGLIDADG (SEQ ID NO: 37) meganucleases and methods of making such meganucleases are provided. In addition, methods are provided for using the meganucleases to generate recombinant cells and organisms having a desired DNA sequence inserted into a limited number of loci within the genome, as well as methods of gene therapy, for treatment of pathogenic infections, and for in vitro applications in diagnostics and research."/><meta property="og:title" content="Patent US8021867 - Rationally-designed meganucleases with altered sequence specificity and DNA-binding affinity"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("OuHoU-eoGsW4sASg3YDIBA"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407291699.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("CZE"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("OuHoU-eoGsW4sASg3YDIBA"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407291699.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("CZE"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us8021867?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US8021867"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=b3MABwABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS8021867&amp;usg=AFQjCNEKFYn2JFx6ag0rJQDQO_nzyebpng" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US8021867.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US8021867.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20070117128"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US8021867"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US8021867" style="display:none"><span itemprop="description">Rationally-designed LAGLIDADG (SEQ ID NO: 37) meganucleases and methods of making such meganucleases are provided. In addition, methods are provided for using the meganucleases to generate recombinant cells and organisms having a desired DNA sequence inserted into a limited number of loci within the...</span><span itemprop="url">http://www.google.com/patents/US8021867?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US8021867 - Rationally-designed meganucleases with altered sequence specificity and DNA-binding affinity</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US8021867 - Rationally-designed meganucleases with altered sequence specificity and DNA-binding affinity" title="Patent US8021867 - Rationally-designed meganucleases with altered sequence specificity and DNA-binding affinity"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US8021867 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 11/583,368</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Sep 20, 2011</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Oct 18, 2006</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Oct 18, 2005</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2626262A1">CA2626262A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1945762A2">EP1945762A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1945762B1">EP1945762B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2327774A2">EP2327774A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2327774A3">EP2327774A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2341135A2">EP2341135A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2341135A3">EP2341135A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2357226A1">EP2357226A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2357226B1">EP2357226B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2360244A1">EP2360244A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2368981A2">EP2368981A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2368981A3">EP2368981A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2484758A1">EP2484758A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2484758B1">EP2484758B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2628794A2">EP2628794A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2628794A3">EP2628794A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2650365A1">EP2650365A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2650366A2">EP2650366A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2650366A3">EP2650366A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2662442A1">EP2662442A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8119361">US8119361</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8119381">US8119381</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8124369">US8124369</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8129134">US8129134</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8133697">US8133697</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8143015">US8143015</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8143016">US8143016</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8148098">US8148098</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8163514">US8163514</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8304222">US8304222</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8377674">US8377674</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20070117128">US20070117128</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120015406">US20120015406</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120015421">US20120015421</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120021465">US20120021465</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120021520">US20120021520</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120021521">US20120021521</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120028314">US20120028314</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120028315">US20120028315</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120030783">US20120030783</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120040405">US20120040405</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120040406">US20120040406</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120070877">US20120070877</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120264189">US20120264189</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20120266266">US20120266266</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20130227720">US20130227720</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2007047859A2">WO2007047859A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2007047859A3">WO2007047859A3</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">11583368, </span><span class="patent-bibdata-value">583368, </span><span class="patent-bibdata-value">US 8021867 B2, </span><span class="patent-bibdata-value">US 8021867B2, </span><span class="patent-bibdata-value">US-B2-8021867, </span><span class="patent-bibdata-value">US8021867 B2, </span><span class="patent-bibdata-value">US8021867B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22James+J.+Smith%22">James J. Smith</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Derek+Jantz%22">Derek Jantz</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Homme+W.+Hellinga%22">Homme W. Hellinga</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Duke+University%22">Duke University</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US8021867.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8021867.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8021867.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (52),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (97),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (12),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (11),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (3)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/8021867&usg=AFQjCNGgOFnYzDsFtljYNkUfYMkxF70SJw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D8021867&usg=AFQjCNFh-tuum0e_n9YyVsDvjlwkzGTirA">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D8021867B2%26KC%3DB2%26FT%3DD&usg=AFQjCNEuzewiFSZtNUx_G-BOwygg7sqPMA">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT106239866" lang="EN" load-source="patent-office">Rationally-designed meganucleases with altered sequence specificity and DNA-binding affinity</invention-title></span><br><span class="patent-number">US 8021867 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA88021765" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">Rationally-designed LAGLIDADG (SEQ ID NO: 37) meganucleases and methods of making such meganucleases are provided. In addition, methods are provided for using the meganucleases to generate recombinant cells and organisms having a desired DNA sequence inserted into a limited number of loci within the genome, as well as methods of gene therapy, for treatment of pathogenic infections, and for in vitro applications in diagnostics and research.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(3)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8021867B2/US08021867-20110920-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8021867B2/US08021867-20110920-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8021867B2/US08021867-20110920-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8021867B2/US08021867-20110920-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8021867B2/US08021867-20110920-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8021867B2/US08021867-20110920-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(19)</span></span></div><div class="patent-text"><div mxw-id="PCLM38540883" lang="EN" load-source="patent-office" class="claims">
    <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A recombinant meganuclease having altered cleavage specificity relative to the wild-type I-CreI meganuclease of SEQ ID NO: 1, comprising a polypeptide having at least 85% sequence identity to residues 2-153 of the I-CreI meganuclease of SEQ ID NO: 1, which differs from residues 2-153 of the I-CreI meganuclease of SEQ ID NO: 1 by:
<div class="claim-text">(a) at least a first specificity-altering amino acid modification at a position corresponding to a position of SEQ ID NO: 1 selected from the group consisting of positions 24, 26, 28, 30, 32, 33, 38, 40, 42, 44, 46, 66, 68, 70, 75, 77, 79 and 139; and</div>
<div class="claim-text">(b) at least a second specificity-altering amino acid modification at a position corresponding to position 26 or 77 of SEQ ID NO: 1, wherein said second specificity-altering amino acid modification is a substitution selected from the group consisting of Q26E, Q26K, Q26S, I77E, I77Q, I77R and I77S; and</div>
<div class="claim-text">(c) at least a third amino acid modification affecting heterodimer formation selected from the group consisting of K7D, K7E, E8K, E8R, K57D, K57E, E61K, E61R, K96D and K96E.</div>
</div>
    </div>
    </div> <div class="claim"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. A recombinant meganuclease having altered cleavage specificity relative to the wild-type I-CreI meganuclease of SEQ ID NO:1, comprising a polypeptide having at least 85% sequence identity to residues 2-153 of the I-CreI meganuclease of SEQ ID NO:1, which differs from residues 2-153 of the I-CreI meganuclease of SEQ ID NO:1 by:
<div class="claim-text">(a) at least a first specificity-altering amino acid modification at a position corresponding to a position of SEQ ID NO:1 selected from the group consisting of positions 24, 26, 28, 30, 32, 33, 38, 40, 42, 44, 46, 66, 68, 70, 75, 77, 79 and 139; and</div>
<div class="claim-text">(b) at least a second specificity-altering amino acid modification at a position corresponding to position 26 or 77 of SEQ ID NO: 1, wherein said second specificity-altering amino acid modification is a substitution selected from the group consisting of Q26E, Q26K, Q26S, I77E, I77Q, I77R and I77S; and</div>
<div class="claim-text">(c) at least a third amino acid modification affecting DNA binding affinity selected from the group consisting of P29D, P29E, P29H, P29K, P29N, P29Q, P29R, P29S, P29T, K34D, K34E, K34H, K34N, K34Q, K34S, K34T, K48D, K48E, K48H, K48N, K48Q, K48S, K48T, R51D, R51E, R51H, R51N, R51Q, R51S, R51T, V64D, V64E, V64H, V64K, V64N, V64Q, V64R, V64S, V64T, E80H, E80K, E80N, E80Q, E80R, E80S, E80T, I81D, I81E, I81H, I81K, I81N, I81Q, I81R, I81S, I81T, K82D, K82E, K82H, K82N, K82Q, K82S, K82T, L112D, L112E, L112K, L112H, L112N, L112Q, L112R, L112S, L112T, K116D, Kl116E, K116H, K116N, K116Q, K116S, K116T, D137H, D137K, D137N, D137Q, D137R, D137S, D137T, K139D, K139E, K139H, K139N, K139Q, K139S, K139T, T143D, T143E, T143K and T143R.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The recombinant meganuclease of <claim-ref idref="CLM-00001">claim 1</claim-ref> or <claim-ref idref="CLM-00002">2</claim-ref> wherein: said second amino acid modification is Q26E.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The recombinant meganuclease of <claim-ref idref="CLM-00001">claim 1</claim-ref> or <claim-ref idref="CLM-00002">2</claim-ref> wherein: said second amino acid modification is Q26K.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. The recombinant meganuclease of <claim-ref idref="CLM-00001">claim 1</claim-ref> or <claim-ref idref="CLM-00002">2</claim-ref> wherein: said second amino acid modification is Q26S.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. The recombinant meganuclease of <claim-ref idref="CLM-00001">claim 1</claim-ref> or <claim-ref idref="CLM-00002">2</claim-ref> wherein: said second amino acid modification is I77E.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. The recombinant meganuclease of <claim-ref idref="CLM-00001">claim 1</claim-ref> or <claim-ref idref="CLM-00002">2</claim-ref> wherein: said second amino acid modification is I77Q.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. The recombinant meganuclease of <claim-ref idref="CLM-00001">claim 1</claim-ref> or <claim-ref idref="CLM-00002">2</claim-ref> wherein: said second amino acid modification is I77R.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The recombinant meganuclease of <claim-ref idref="CLM-00001">claim 1</claim-ref> or <claim-ref idref="CLM-00002">2</claim-ref> wherein: said second amino acid modification is I77S.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The recombinant meganuclease of <claim-ref idref="CLM-00001">claim 1</claim-ref> or <claim-ref idref="CLM-00002">2</claim-ref> wherein:
<div class="claim-text">said first amino acid modification is selected from the group consisting of I24C, I24K, I24R, K28C, K28Q, K28R, N30E, N30K, N30Q, N30R, S32A, S32C, S32D, S32E, S32H, S32I, S32K, S32L, S32N, S32Q, S32R, S32T, S32V, Y33C, Y33D, Y33E, Y33F, Y33I, Y33K, Y33L, Y33R, Y33V, Q38C, Q38E, Q38H, Q38I, Q38K, Q38L, Q38N, Q38R, S40C, S40E, S40I, S40R, S40V, A42E, A42Q, A42R, Q44A, Q44C, Q44D, Q44E, Q44I, Q44K, Q44L, Q44N, Q44R, Q44T, Q44V, T46A, T46C, T46D, T46E, T46G, T46H, T46K, T46Q, T46R, Y66K, Y66M, R68C, R68E, R68F, R68H, R68K, R68L, R68M, R68Q, R68Y, R70A, R70C, R70D, R70E, R70G, R70H, R70K, R70L, R70Q, R70S, D75C, D75E, D75H, D75L, D75Q, D75R, D75Y, S79C, S79I, S79V, K139H, and K139Y.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. The recombinant meganuclease of <claim-ref idref="CLM-00002">claim 2</claim-ref> wherein: said third amino acid modification is E80R.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. The recombinant meganuclease of <claim-ref idref="CLM-00002">claim 2</claim-ref> wherein: said third amino acid modification is E80K.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00013" num="00013" class="claim">
      <div class="claim-text">13. The recombinant meganuclease of <claim-ref idref="CLM-00002">claim 2</claim-ref> wherein: said third amino acid modification is E80Q.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00014" num="00014" class="claim">
      <div class="claim-text">14. The recombinant meganuclease of <claim-ref idref="CLM-00002">claim 2</claim-ref> wherein: said third amino acid modification is E80H.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00015" num="00015" class="claim">
      <div class="claim-text">15. The recombinant meganuclease of <claim-ref idref="CLM-00002">claim 2</claim-ref> wherein: said third amino acid modification is E80N.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00016" num="00016" class="claim">
      <div class="claim-text">16. The recombinant meganuclease of <claim-ref idref="CLM-00002">claim 2</claim-ref> wherein: said third amino acid modification is E80S.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00017" num="00017" class="claim">
      <div class="claim-text">17. The recombinant meganuclease of <claim-ref idref="CLM-00002">claim 2</claim-ref> wherein: said third amino acid modification is E80T.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00018" num="00018" class="claim">
      <div class="claim-text">18. The recombinant meganuclease of <claim-ref idref="CLM-00002">claim 2</claim-ref> wherein: said third amino acid modification is I81K.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00019" num="00019" class="claim">
      <div class="claim-text">19. The recombinant meganuclease of <claim-ref idref="CLM-00002">claim 2</claim-ref> wherein: said third amino acid modification is I81R.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES44074013" lang="EN" load-source="patent-office" class="description">
    <heading>RELATED APPLICATION</heading> <p num="p-0002">This application claims benefit of priority to U.S. Provisional Patent Application No. 60/727,512, filed Oct. 18, 2005, the entire disclosure of which is hereby incorporated by reference.</p>
    <heading>GOVERNMENT SUPPORT</heading> <p num="p-0003">The invention was supported in part by grants 2R01-GM-0498712, 5F32-GM072322 and 5 DP1 OD000122 from the National Institute of General Medical Sciences of National Institutes of Health of the United States of America. Therefore, the U.S. government may have certain rights in the invention.</p>
    <heading>FIELD OF THE INVENTION</heading> <p num="p-0004">The invention relates to the field of molecular biology and recombinant nucleic acid technology. In particular, the invention relates to rationally-designed, non-naturally-occurring meganucleases with altered DNA recognition sequence specificity and/or altered affinity. The invention also relates to methods of producing such meganucleases, and methods of producing recombinant nucleic acids and organisms using such meganucleases.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p num="p-0005">Genome engineering requires the ability to insert, delete, substitute and otherwise manipulate specific genetic sequences within a genome, and has numerous therapeutic and biotechnological applications. The development of effective means for genome modification remains a major goal in gene therapy, agrotechnology, and synthetic biology (Porteus et al. (2005), <i>Nat. Biotechnol. </i>23: 967-73; Tzfira et al. (2005), <i>Trends Biotechnol. </i>23: 567-9; McDaniel et al. (2005), <i>Curr. Opin. Biotechnol. </i>16: 476-83). A common method for inserting or modifying a DNA sequence involves introducing a transgenic DNA sequence flanked by sequences homologous to the genomic target and selecting or screening for a successful homologous recombination event. Recombination with the transgenic DNA occurs rarely but can be stimulated by a double-stranded break in the genomic DNA at the target site. Numerous methods have been employed to create DNA double-stranded breaks, including irradiation and chemical treatments. Although these methods efficiently stimulate recombination, the double-stranded breaks are randomly dispersed in the genome, which can be highly mutagenic and toxic. At present, the inability to target gene modifications to unique sites within a chromosomal background is a major impediment to successful genome engineering.</p>
    <p num="p-0006">One approach to achieving this goal is stimulating homologous recombination at a double-stranded break in a target locus using a nuclease with specificity for a sequence that is sufficiently large to be present at only a single site within the genome (see, e.g., Porteus et al. (2005), <i>Nat. Biotechnol. </i>23: 967-73). The effectiveness of this strategy has been demonstrated in a variety of organisms using chimeric fusions between an engineered zinc finger DNA-binding domain and the non-specific nuclease domain of the FokI restriction enzyme (Porteus (2006), <i>Mol Ther </i>13: 438-46; Wright et al. (2005), <i>Plant J. </i>44: 693-705; Urnov et al. (2005), <i>Nature </i>435: 646-51). Although these artificial zinc finger nucleases stimulate site-specific recombination, they retain residual non-specific cleavage activity resulting from under-regulation of the nuclease domain and frequently cleave at unintended sites (Smith et al. (2000), <i>Nucleic Acids Res. </i>28: 3361-9). Such unintended cleavage can cause mutations and toxicity in the treated organism (Porteus et al. (2005), <i>Nat. Biotechnol. </i>23: 967-73).</p>
    <p num="p-0007">A group of naturally-occurring nucleases which recognize 15-40 base-pair cleavage sites commonly found in the genomes of plants and fungi may provide a less toxic genome engineering alternative. Such “meganucleases” or “homing endonucleases” are frequently associated with parasitic DNA elements, such as group 1 self-splicing introns and inteins. They naturally promote homologous recombination or gene insertion at specific locations in the host genome by producing a double-stranded break in the chromosome, which recruits the cellular DNA-repair machinery (Stoddard (2006), <i>Q. Rev. Biophys. </i>38: 49-95). Meganucleases are commonly grouped into four families: the LAGLIDADG family, the GIY-YIG family, the His-Cys box family and the HNH family. These families are characterized by structural motifs, which affect catalytic activity and recognition sequence. For instance, members of the LAGLIDADG family are characterized by having either one or two copies of the conserved LAGLIDADG motif (SEQ ID NO: 37)(see Chevalier et al. (2001), <i>Nucleic Acids Res. </i>29(18): 3757-3774). The LAGLIDADG meganucleases with a single copy of the LAGLIDADG motif (SEQ ID NO: 37) form homodimers, whereas members with two copies of the LAGLIDADG motif (SEQ ID NO: 37) are found as monomers. Similarly, the GIY-YIG family members have a GIY-YIG module, which is 70-100 residues long and includes four or five conserved sequence motifs with four invariant residues, two of which are required for activity (see Van Roey et al. (2002), <i>Nature Struct. Biol. </i>9: 806-811). The His-Cys box meganucleases are characterized by a highly conserved series of histidines and cysteines over a region encompassing several hundred amino acid residues (see Chevalier et al. (2001), <i>Nucleic Acids Res. </i>29(18): 3757-3774). In the case of the NHN family, the members are defined by motifs containing two pairs of conserved histidines surrounded by asparagine residues (see Chevalier et al. (2001), <i>Nucleic Acids Res. </i>29(18): 3757-3774). The four families of meganucleases are widely separated from one another with respect to conserved structural elements and, consequently, DNA recognition sequence specificity and catalytic activity.</p>
    <p num="p-0008">Natural meganucleases, primarily from the LAGLIDADG family, have been used to effectively promote site-specific genome modification in plants, yeast, <i>Drosophila</i>, mammalian cells and mice, but this approach has been limited to the modification of either homologous genes that conserve the meganuclease recognition sequence (Monnat et al. (1999), <i>Biochem. Biophys. Res. Commun. </i>255: 88-93) or to pre-engineered genomes into which a recognition sequence has been introduced (Rouet et al. (1994), <i>Mol. Cell. Biol. </i>14: 8096-106; Chilton et al. (2003), <i>Plant Physiol. </i>133: 956-65; Puchta et al. (1996), <i>Proc. Natl. Acad. Sci. USA </i>93: 5055-60; Rong et al. (2002), <i>Genes Dev. </i>16: 1568-81; Gouble et al. (2006), <i>J. Gene Med. </i>8(5):616-622).</p>
    <p num="p-0009">Systematic implementation of nuclease-stimulated gene modification requires the use of engineered enzymes with customized specificities to target DNA breaks to existing sites in a genome and, therefore, there has been great interest in adapting meganucleases to promote gene modifications at medically or biotechnologically relevant sites (Porteus et al. (2005), <i>Nat. Biotechnol. </i>23: 967-73; Sussman et al. (2004), <i>J. Mol. Biol. </i>342: 31-41; Epinat et al. (2003), <i>Nucleic Acids Res. </i>31: 2952-62).</p>
    <p num="p-0010">The meganuclease I-CreI from <i>Chlamydomonas reinhardtii </i>is a member of the LAGLIDADG family which recognizes and cuts a 22 base-pair recognition sequence in the chloroplast chromosome, and which presents an attractive target for meganuclease redesign. The wild-type enzyme is a homodimer in which each monomer makes direct contacts with 9 base pairs in the full-length recognition sequence. Genetic selection techniques have been used to identify mutations in I-CreI that alter base preference at a single position in this recognition sequence (Sussman et al. (2004), <i>J. Mol. Biol. </i>342: 31-41; Chames et al. (2005), <i>Nucleic Acids Res. </i>33: e178; Seligman et al. (2002), <i>Nucleic Acids Res. </i>30: 3870-9) or, more recently, at three positions in the recognition sequence (Arnould et al. (2006), <i>J. Mol. Biol. </i>355: 443-58). The I-CreI protein-DNA interface contains nine amino acids that contact the DNA bases directly and at least an additional five positions that can form potential contacts in modified interfaces. The size of this interface imposes a combinatorial complexity that is unlikely to be sampled adequately in sequence libraries constructed to select for enzymes with drastically altered cleavage sites.</p>
    <p num="p-0011">There remains a need for nucleases that will facilitate precise modification of a genome. In addition, there remains a need for techniques for generating nucleases with pre-determined, rationally-designed recognition sequences that will allow manipulation of genetic sequences at specific genetic loci and for techniques utilizing such nucleases to genetically engineer organisms with precise sequence modifications.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p num="p-0012">The present invention is based, in part, upon the identification and characterization of specific amino acid residues in the LAGLIDADG family of meganucleases that make contacts with DNA bases and the DNA backbone when the meganucleases associate with a double-stranded DNA recognition sequence, and thereby affect the specificity and activity of the enzymes. This discovery has been used, as described in detail below, to identify amino acid substitutions which can alter the recognition sequence specificity and/or DNA-binding affinity of the meganucleases, and to rationally design and develop meganucleases that can recognize a desired DNA sequence that naturally-occurring meganucleases do not recognize. The invention also provides methods that use such meganucleases to produce recombinant nucleic acids and organisms by utilizing the meganucleases to cause recombination of a desired genetic sequence at a limited number of loci within the genome of the organism, for gene therapy, for treatment of pathogenic infections, and for in vitro applications in diagnostics and research.</p>
    <p num="p-0013">Thus, in some embodiments, the invention provides recombinant meganucleases having altered specificity for at least one recognition sequence half-site relative to a wild-type I-CreI meganuclease, in which the meganuclease includes a polypeptide having at least 85% sequence similarity to residues 2-153 of the wild-type I-CreI meganuclease of SEQ ID NO: 1, but in which the recombinant meganuclease has specificity for a recognition sequence half-site which differs by at least one base pair from a half-site within an I-CreI meganuclease recognition sequence selected from SEQ ID NO: 2, SEQ ID NO: 3, SEQ ID NO: 4 and SEQ ID NO: 5, and in which the recombinant meganuclease includes at least one modification listed in Table 1 which is not an excluded modification found in the prior art.</p>
    <p num="p-0014">In other embodiments, the invention provides recombinant meganucleases having altered specificity for at least one recognition sequence half-site relative to a wild-type I-MsoI meganuclease, in which the meganuclease includes a polypeptide having at least 85% sequence similarity to residues 6-160 of the I-MsoI meganuclease of SEQ ID NO: 6, but in which the recombinant meganuclease has specificity for a recognition sequence half-site which differs by at least one base pair from a half-site within an I-MsoI meganuclease recognition sequence selected from SEQ ID NO: 7 and SEQ ID NO: 8, and in which the recombinant meganuclease includes at least one modification listed in Table 2 which is not an excluded modification found in the prior art.</p>
    <p num="p-0015">In other embodiments, the invention provides recombinant meganucleases having altered specificity for a recognition sequence relative to a wild-type I-SceI meganuclease, in which the meganuclease includes a polypeptide having at least 85% sequence similarity to residues 3-186 of the I-SceI meganuclease of SEQ ID NO: 9, but in which the recombinant meganuclease has specificity for a recognition sequence which differs by at least one base pair from an I-SceI meganuclease recognition sequence of SEQ ID NO: 10 and SEQ ID NO: 11, and in which the recombinant meganuclease includes at least one modification listed in Table 3 which is not an excluded modification found in the prior art.</p>
    <p num="p-0016">In other embodiments, the invention provides recombinant meganucleases having altered specificity for at least one recognition sequence half-site relative to a wild-type I-CeuI meganuclease, in which the meganuclease includes a polypeptide having at least 85% sequence similarity to residues 5-211 of the I-CeuI meganuclease of SEQ ID NO: 12, but in which the recombinant meganuclease has specificity for a recognition sequence half-site which differs by at least one base pair from a half-site within an I-CeuI meganuclease recognition sequence selected from SEQ ID NO: 13 and SEQ ID NO: 14, and in which the recombinant meganuclease includes at least one modification listed in Table 4 which is not an excluded modification found in the prior art.</p>
    <p num="p-0017">The meganucleases of the invention can include one, two, three or more of the modifications which have been disclosed herein in order to affect the sequence specificity of the recombinant meganucleases at one, two, three or more positions within the recognition sequence. The meganucleases can include only the novel modifications disclosed herein, or can include the novel modifications disclosed herein in combination with modifications found in the prior art. Specifically excluded, however, are recombinant meganucleases comprising only the modifications of the prior art.</p>
    <p num="p-0018">In another aspect, the invention provides for recombinant meganucleases with altered binding affinity for double-stranded DNA which is not sequence-specific. This is accomplished by modifications of the meganuclease residues which make contacts with the backbone of the double-stranded DNA recognition sequence. The modifications can increase or decrease the binding affinity and, consequently, can increase or decrease the overall activity of the enzyme. Moreover, increases/decreases in binding and activity have been found to causes decreases/increases in sequence specificity. Thus, the invention provides a means for altering sequence specificity generally by altering DNA-binding affinity.</p>
    <p num="p-0019">Thus, in some embodiments, the invention provides for recombinant meganucleases having altered binding affinity for double-stranded DNA relative to a wild-type I-CreI meganuclease, in which the meganuclease includes a polypeptide having at least 85% sequence similarity to residues 2-153 of the I-CreI meganuclease of SEQ ID NO: 1, and in which the DNA-binding affinity has been either (1) increased by at least one modification corresponding to a substitution selected from (a) substitution of E80, D137, I81, L112, P29, V64 or Y66 with H, N, Q, S, T, K or R, or (b) substitution of T46, T140 or T143 with K or R; or, conversely, (2) decreased by at least one modification corresponding to a substitution selected from (a) substitution of K34, K48, R51, K82, K116 or K139 with H, N, Q, S, T, D or E, or (b) substitution of I81, L112, P29, V64, Y66, T46, T140 or T143 with D or E.</p>
    <p num="p-0020">In other embodiments, the invention provides for recombinant meganucleases having altered binding affinity for double-stranded DNA relative to a wild-type I-MsoI meganuclease, in which the meganuclease includes a polypeptide having at least 85% sequence similarity to residues 6-160 of the I-MsoI meganuclease of SEQ ID NO: 6, and in which the DNA-binding affinity has been either (1) increased by at least one modification corresponding to a substitution selected from (a) substitution of E147, 185, G86 or Y118 with H, N, Q, S, T, K or R, or (b) substitution of Q41, N70, S87, T88, H89, Q122, Q139, S150 or N152 with K or R; or, conversely, (2) decreased by at least one modification corresponding to a substitution selected from (a) substitution of K36, R51, K123, K143 or R144 with H, N, Q, S, T, D or E, or (b) substitution of I85, G86, Y118, Q41, N70, S87, T88, H89, Q122, Q139, S150 or N152 with D or E.</p>
    <p num="p-0021">In other embodiments, the invention provides for recombinant meganucleases having altered binding affinity for double-stranded DNA relative to a wild-type I-SceI meganuclease, in which the meganuclease includes a polypeptide having at least 85% sequence similarity to residues 3-186 of the I-SceI meganuclease of SEQ ID NO: 9, and in which the DNA-binding affinity has been either (I) increased by at least one modification corresponding to a substitution selected from (a) substitution of D201, L19, L80, L92, Y151, Y188, I191, Y199 or Y222 with H, N, Q, S, T, K or R, or (b) substitution of N15, N17, S81, H84, N94, N120, T156, N157, S159, N163, Q165, S166, N194 or S202 with K or R; or, conversely, (2) decreased by at least one modification corresponding to a substitution selected from (a) substitution of K20, K23, K63, K122, K148, K153, K190, K193, K195 or K223 with H, N, Q, S, T, D or E, or (b) substitution of L19, L80, L92, Y151, Y188, I191, Y199, Y222, N15, N17, S81, H84, N94, N120, T156, N157, S159, N163, Q165, S166, N194 or S202 with D or E.</p>
    <p num="p-0022">In other embodiments, the invention provides for recombinant meganucleases having altered binding affinity for double-stranded DNA relative to a wild-type I-CeuI meganuclease, in which the meganuclease includes a polypeptide having at least 85% sequence similarity to residues 5-211 of the I-CeuI meganuclease of SEQ ID NO: 12, and in which the DNA-binding affinity has been either (1) increased by at least one modification corresponding to a substitution selected from (a) substitution of D25 or D128 with H, N, Q, S, T, K or R, or (b) substitution of S68, N70, H94, S117, N120, N129 or H172 with K or R; or, conversely, (2) decreased by at least one modification corresponding to a substitution selected from (a) substitution of K21, K28, K31, R112, R114 or R130 with H, N, Q, S, T, D or E, or (b) substitution of S68, N70, H94, S117, N120, N129 or H172 with D or E.</p>
    <p num="p-0023">The meganucleases of the invention can include one, two, three or more of the modifications of backbone contact residues which have been disclosed herein in order to affect DNA-binding affinity. In addition, these modifications affecting DNA-binding affinity can be combined with one or more of the novel modifications of the base contact residues described above which alter the sequence specificity of the recombinant meganucleases at specific positions within the recognition sequence, or with the prior art modifications described above, or with a combination of the novel modifications and prior art modifications. In particular, by combining backbone contact modifications and base contact modifications, recombinant meganucleases can be rationally-designed with desired specificity and activity. For example, increases in DNA-binding affinity can be designed which may offset losses in affinity resulting from designed changes to base contact residues, or decreases in affinity can be designed which may also decrease sequence specificity and broaden the set of recognition sequences for an enzyme.</p>
    <p num="p-0024">In another aspect, the invention provides for rationally-designed meganuclease monomers with altered affinity for homo- or heterodimer formation. The affinity for dimer formation can be measured with the same monomer (i.e., homodimer formation) or with a different monomer (i.e., heterodimer formation) such as a reference wild-type meganuclease. These recombinant meganucleases have modifications to the amino acid residues which are present at the protein-protein interface between monomers in a meganuclease dimer. The modifications can be used to promote heterodimer formation and create meganucleases with non-palindromic recognition sequences.</p>
    <p num="p-0025">Thus, in some embodiments, the invention provides recombinant meganuclease monomers having altered affinity for dimer formation with a reference meganuclease monomer, in which the recombinant monomer includes a polypeptide having at least 85% sequence similarity to residues 2-153 of the I-CreI meganuclease of SEQ ID NO: 1, but in which affinity for dimer formation has been altered by at least one modification corresponding to a substitution selected from (a) substitution of K7, K57 or K96 with D or E, or (b) substitution of E8 or E61 with K or R. Based upon such recombinant monomers, the invention also provides recombinant meganuclease heterodimers including (1) a first polypeptide having at least 85% sequence similarity to residues 2-153 of the I-CreI meganuclease of SEQ ID NO: 1, but in which affinity for dimer formation has been altered by at least one modification corresponding to a substitution selected from (a) substitution of K7, K57 or K96 with D or E, and (2) a second polypeptide having at least 85% sequence similarity to residues 2-153 of the I-CreI meganuclease of SEQ ID NO: 1, but i which affinity for dimer formation has been altered by at least one modification corresponding to a substitution selected from (b) substitution of E8 or E61 with K or R.</p>
    <p num="p-0026">In other embodiments, the invention provides recombinant meganuclease monomers having altered affinity for dimer formation with a reference meganuclease monomer, in which the recombinant monomer includes a polypeptide having at least 85% sequence similarity to residues 6-160 of the I-MsoI meganuclease of SEQ ID NO: 6, but in which affinity for dimer formation has been altered by at least one modification corresponding to a substitution selected from (a) substitution of R302 with D or E, or (b) substitution of D20, E11 or Q64 with K or R. Based upon such recombinant monomers, the invention also provides recombinant meganuclease heterodimers including (1) a first polypeptide having at least 85% sequence similarity to residues 6-160 of the I-MsoI meganuclease of SEQ ID NO: 6, but in which affinity for dimer formation has been altered by at least one modification corresponding to a substitution selected from (a) substitution of R302 with D or E, and (2) a second polypeptide having at least 85% sequence similarity to residues 6-160 of the I-MsoI meganuclease of SEQ ID NO: 6, but in which affinity for dimer formation has been altered by at least one modification corresponding to a substitution selected from (b) substitution of D20, E11 or Q64 with K or R.</p>
    <p num="p-0027">In other embodiments, the invention provides recombinant meganuclease monomers having altered affinity for dimer formation with a reference meganuclease monomer, in which the recombinant monomer includes a polypeptide having at least 85% sequence similarity to residues 5-211 of the I-CeuI meganuclease of SEQ ID NO: 12, but in which affinity for dimer formation has been altered by at least one modification corresponding to a substitution selected from (a) substitution of R93 with D or E, or (b) substitution of E152 with K or R. Based upon such recombinant monomers, the invention also provides recombinant meganuclease heterodimers including (1) a first polypeptide having at least 85% sequence similarity to residues 5-211 of the I-CeuI meganuclease of SEQ ID NO: 12, but in which affinity for dimer formation has been altered by at least one modification corresponding to a substitution selected from (a) substitution of R93 with D or E, and (2) a second polypeptide having at least 85% sequence similarity to residues 5-211 of the I-CeuI meganuclease of SEQ ID NO: 12, but in which affinity for dimer formation has been altered by at least one modification corresponding to a substitution selected from (b) substitution of E152 with K or R.</p>
    <p num="p-0028">The recombinant meganuclease monomers or heterodimers with altered affinity for dimer formation can also include one, two, three or more of the modifications of base contact residues described above; one, two, three or more of the modifications of backbone contact residues described above; or combinations of both. Thus, for example, the base contacts of a monomer can be modified to alter sequence specificity, the backbone contacts of a monomer can be modified to alter DNA-binding affinity, and the protein-protein interface can be modified to affect dimer formation. Such a recombinant monomer can be combined with a similarly modified monomer to produce a rationally-designed meganuclease heterodimer with desired sequence specificity and activity.</p>
    <p num="p-0029">In another aspect, the invention provides for various methods of use for the rationally-designed meganucleases described and enabled herein. These methods include producing genetically-modified cells and organisms, treating diseases by gene therapy, treating pathogen infections, and using the recombinant meganucleases for in vitro applications for diagnostics and research.</p>
    <p num="p-0030">Thus, in one aspect, the invention provides methods for producing a genetically-modified eukaryotic cell including an exogenous sequence of interest inserted in a chromosome, by transfecting the cell with (i) a first nucleic acid sequence encoding a meganuclease of the invention, and (ii) a second nucleic acid sequence including said sequence of interest, wherein the meganuclease produces a cleavage site in the chromosome and the sequence of interest is inserted into the chromosome at the cleavage site either by homologous recombination or non-homologous end-joining.</p>
    <p num="p-0031">Alternatively, in another aspect, the invention provides methods for producing a genetically-modified eukaryotic cell including an exogenous sequence of interest inserted in a chromosome, by introducing a meganuclease protein of the invention into the cell, and transfecting the cell with a nucleic acid including the sequence of interest, wherein the meganuclease produces a cleavage site in the chromosome and the sequence of interest is inserted into the chromosome at the cleavage site either by homologous recombination or non-homologous end-joining.</p>
    <p num="p-0032">In another aspect, the invention provides methods for producing a genetically-modified eukaryotic cell by disrupting a target sequence in a chromosome, by transfecting the cell with a nucleic acid encoding a meganuclease of the invention, wherein the meganuclease produces a cleavage site in the chromosome and the target sequence is disrupted by non-homologous end-joining at the cleavage site.</p>
    <p num="p-0033">In another aspect, the invention provides methods of producing a genetically-modified organism by producing a genetically-modified eukaryotic cell according to the methods described above, and growing the genetically-modified eukaryotic cell to produce the genetically-modified organism. In these embodiments, the eukaryotic cell can be selected from a gamete, a zygote, a blastocyst cell, an embryonic stem cell, and a protoplast cell.</p>
    <p num="p-0034">In another aspect, the invention provides methods for treating a disease by gene therapy in a eukaryote, by transfecting at least one cell of the eukaryote with one or more nucleic acids including (i) a first nucleic acid sequence encoding a meganuclease of the invention, and (ii) a second nucleic acid sequence including a sequence of interest, wherein the meganuclease produces a cleavage site in the chromosome and the sequence of interest is inserted into the chromosome by homologous recombination or non-homologous end-joining, and insertion of the sequence of interest provides gene therapy for the disease.</p>
    <p num="p-0035">Alternatively, in another aspect, the invention provides methods for treating a disease by gene therapy in a eukaryote, by introducing a meganuclease protein of the invention into at least one cell of the eukaryote, and transfecting the cell with a nucleic acid including a sequence of interest, wherein the meganuclease produces a cleavage site in the chromosome and the sequence of interest is inserted into the chromosome at the cleavage site by homologous recombination or non-homologous end-joining, and insertion of the sequence of interest provides gene therapy for the disease.</p>
    <p num="p-0036">In another aspect, the invention provides methods for treating a disease by gene therapy in a eukaryote by disrupting a target sequence in a chromosome of the eukaryotic, by transfecting at least one cell of the eukaryote with a nucleic acid encoding a meganuclease of the invention, wherein the meganuclease produces a cleavage site in the chromosome and the target sequence is disrupted by non-homologous end-joining at the cleavage site, wherein disruption of the target sequence provides the gene therapy for the disease.</p>
    <p num="p-0037">In another aspect, the invention provides methods for treating a viral or prokaryotic pathogen infection in a eukaryotic host by disrupting a target sequence in a genome of the pathogen, by transfecting at least one infected cell of the host with a nucleic acid encoding a meganuclease of the invention, wherein the meganuclease produces a cleavage site in the genome and the target sequence is disrupted by either (1) non-homologous end-joining at the cleavage site or (2) by homologous recombination with a second nucleic acid, and wherein disruption of the target sequence provides treatment for the infection.</p>
    <p num="p-0038">More generally, in another aspect, the invention provides methods for rationally-designing recombinant meganucleases having altered specificity for at least one base position of a recognition sequence, by (1) determining at least a portion of a three-dimensional structure of a reference meganuclease-DNA complex; (2) identifying amino acid residues forming a base contact surface at the base position; (3) determining a distance between a β-carbon of at least a first residue of the contact surface and at least a first base at the base position; and (4) identifying an amino acid substitution to promote the desired change by either (a) for a first residue which is &lt;6 Å from the first base, selecting a substitution from Group 1 and/or Group 2 which is a member of an appropriate one of Group G, Group C, Group T or Group A; or (b) for a first residue which is &gt;6 Å from said first base, selecting a substitution from Group 2 and/or Group 3 which is a member of an appropriate one of Group G, Group C, Group T or Group A, where each of the Groups is defined herein. This method may be repeated for additional contact residues for the same base, and for contact residues for the other base at the same position, as well as for additional positions.</p>
    <p num="p-0039">In addition, in another general aspect, the invention provides methods for rationally-designing a recombinant meganuclease having increased DNA-binding affinity, by (1) determining at least a portion of a three-dimensional structure of a reference meganuclease-DNA complex; (2) identifying amino acid contact residues forming a backbone contact surface; and (3) identifying an amino acid substitution to increase the DNA-binding affinity by (a) for a contact residue having a negatively-charged or hydrophobic side chain, selecting a substitution having an uncharged/polar or positively-charged side chain; or (b) for a contact residue having an uncharged/polar side chain, selecting a substitution having a positively-charged side chain. Conversely, the invention also provides methods for rationally-designing a recombinant meganuclease having decreased DNA-binding affinity, by (1) determining at least a portion of a three-dimensional structure of a reference meganuclease-DNA complex; (2) identifying amino acid contact residues forming a backbone contact surface; (3) identifying an amino acid substitution to decrease the DNA-binding affinity by (a) for a contact residue having a positively-charged side chain, selecting a substitution having an uncharged/polar or negatively-charged side chain; or (b) for a contact residue having an hydrophobic or uncharged/polar side chain, selecting a substitution having a negatively-charged side chain.</p>
    <p num="p-0040">These and other aspects and embodiments of the invention will be apparent to one of ordinary skill in the art based upon the following detailed description of the invention.</p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE FIGURES</heading> <p num="p-0041"> <figref idrefs="DRAWINGS">FIG. 1(A)</figref> illustrates the interactions between the I-CreI homodimer and its naturally-occurring double-stranded recognition sequence, based upon crystallographic data. This schematic representation depicts the recognition sequence (SEQ ID NO: 2 and SEQ ID NO: 3), shown as unwound for illustration purposes only, bound by the homodimer, shown as two ovals. The bases of each DNA half-site are numbered −1 through −9, and the amino acid residues of I-CreI which form the recognition surface are indicated by one-letter amino acid designations and numbers indicating residue position. Solid black lines: hydrogen bonds to DNA bases. Dashed lines: amino acid positions that form additional contacts in enzyme designs but do not contact the DNA in the wild-type complex. Arrows: residues that interact with the DNA backbone and influence cleavage activity.</p>
      <p num="p-0042"> <figref idrefs="DRAWINGS">FIG. 1(B)</figref> illustrates the wild-type contacts between the A-T base pair at position −4 of the cleavage half-site on the right side of <figref idrefs="DRAWINGS">FIG. 1(A)</figref>. Specifically, the residue Q26 is shown to interact with the A base. Residue 177 is in proximity to the base pair but not specifically interacting.</p>
      <p num="p-0043"> <figref idrefs="DRAWINGS">FIG. 1(C)</figref> illustrates the interactions between a rationally-designed variant of the I-CreI meganuclease in which residue I77 has been modified to E77. As a result of this change, a G-C base pair is preferred at position −4. The interaction between Q26 and the G base is mediated by a water molecule, as has been observed crystallographically for the cleavage half-site on the left side of <figref idrefs="DRAWINGS">FIG. 1(A)</figref>.</p>
      <p num="p-0044"> <figref idrefs="DRAWINGS">FIG. 1(D)</figref> illustrates the interactions between a rationally-designed variant of the I-CreI meganuclease in which residue Q26 has been modified to E26 and residue I77 has been modified to R77. As a result of this change, a C-G base pair is preferred at position −4.</p>
      <p num="p-0045"> <figref idrefs="DRAWINGS">FIG. 1(E)</figref> illustrates the interactions between a rationally-designed variant of the I-CreI meganuclease in which residue Q26 has been modified to A26 and residue I77 has been modified to Q77. As a result of this change, a T-A base pair is preferred at position −4.</p>
      <p num="p-0046"> <figref idrefs="DRAWINGS">FIG. 2(A)</figref> shows a comparison of one recognition sequence for each of the wild type I-CreI meganuclease (WT) and 11 rationally-designed meganuclease heterodimers of the invention. Bases that are conserved relative to the WT recognition sequence are shaded. The 9 bp half-sites are bolded. WT: wild-type (SEQ ID NO: 4); CF: ΔF508 allele of the human CFTR gene responsible for most cases of cystic fibrosis (SEQ ID NO: 25); MYD: the human DM kinase gene associated with myotonic dystrophy (SEQ ID NO: 27); CCR: the human CCR5 gene (a major HIV co-receptor) (SEQ ID NO: 26); ACH: the human FGFR3 gene correlated with achondroplasia (SEQ ID NO: 23); TAT: the HIV-1 TAT/REV gene (SEQ ID NO: 15); HSV: the HSV-1 UL36 gene (SEQ ID NO: 28); LAM: the bacteriophage λ p05 gene (SEQ ID NO: 22); POX: the Variola (smallpox) virus gp009 gene (SEQ ID NO: 30); URA: the <i>Saccharomyces cerevisiae </i>URA3 gene (SEQ ID NO: 36); GLA: the <i>Arabidopsis thaliana </i>GL2 gene (SEQ ID NO: 32); BRP: the <i>Arabidopsis thaliana </i>BP-1 gene (SEQ ID NO: 33).</p>
      <p num="p-0047"> <figref idrefs="DRAWINGS">FIG. 2(B)</figref> illustrates the results of incubation of each of wild-type I-CreI (WT) and 11 rationally-designed meganuclease heterodimers with plasmids harboring the recognition sites for all 12 enzymes for 6 hours at 37° C. Percent cleavage is indicated in each box.</p>
      <p num="p-0048"> <figref idrefs="DRAWINGS">FIG. 3</figref> illustrates cleavage patterns of wild-type and rationally-designed I-CreI homodimers. (A) wild type I-CreI. (B) I-CreI K116D. (C-L) rationally-designed meganucleases of the invention. Enzymes were incubated with a set of plasmids harboring palindromes of the intended cleavage half-site the 27 corresponding single-base pair variations. Bar graphs show fractional cleavage (F) in 4 hours at 37° C. Black bars: expected cleavage patterns based on Table 1. Gray bars: DNA sites that deviate from expected cleavage patterns. White circles indicate bases in the intended recognition site. Also shown are cleavage time-courses over two hours. The open circle time-course plots in C and L correspond to cleavage by the CCR1 and BRP2 enzymes lacking the E80Q mutation. The cleavage sites correspond to the 5′ (left column) and 3′ (right column) half-sites for the heterodimeric enzymes described in <figref idrefs="DRAWINGS">FIG. 2(A)</figref>.</p>
    </description-of-drawings> <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p num="h-0008">1.1 Introduction</p>
    <p num="p-0049">The present invention is based, in part, upon the identification and characterization of specific amino acids in the LAGLIDADG family of meganucleases that make specific contacts with DNA bases and non-specific contacts with the DNA backbone when the meganucleases associate with a double-stranded DNA recognition sequence, and which thereby affect the recognition sequence specificity and DNA-binding affinity of the enzymes. This discovery has been used, as described in detail below, to identify amino acid substitutions in the meganucleases that can alter the specificity and/or affinity of the enzymes, and to rationally design and develop meganucleases that can recognize a desired DNA sequence that naturally-occurring meganucleases do not recognize, and/or that have increased or decreased specificity and/or affinity relative to the naturally-occurring meganucleases. Furthermore, because DNA-binding affinity affects enzyme activity as well as sequence-specificity, the invention provides rationally-designed meganucleases with altered activity relative to naturally-occurring meganucleases. In addition, the invention provides rationally-designed meganucleases in which residues at the interface between the monomers associated to form a dimer have been modified in order to promote heterodimer formation. Finally, the invention provides uses for the rationally-designed meganucleases in the production of recombinant cells and organisms, as well as in gene therapy, anti-pathogen, anti-cancer, and in vitro applications, as disclosed herein.</p>
    <p num="p-0050">As a general matter, the invention provides methods for generating rationally-designed LAGLIDADG meganucleases containing altered amino acid residues at sites within the meganuclease that are responsible for (1) sequence-specific binding to individual bases in the double-stranded DNA recognition sequence, or (2) non-sequence-specific binding to the phosphodiester backbone of a double-stranded DNA molecule. Because enzyme activity is correlated to DNA-binding affinity, however, altering the amino acids involved in binding to the DNA recognition sequence can alter not only the specificity of the meganuclease through specific base pair interactions, but also the activity of the meganuclease by increasing or decreasing overall binding affinity for the double-stranded DNA. Similarly, altering the amino acids involved in binding to the DNA backbone can alter not only the activity of the enzyme, but also the degree of specificity or degeneracy of binding to the recognition sequence by increasing or decreasing overall binding affinity for the double-stranded DNA.</p>
    <p num="p-0051">As described in detail below, the methods of rationally-designing meganucleases include the identification of the amino acids responsible for DNA recognition/binding, and the application of a series of rules for selecting appropriate amino acid changes. With respect to meganuclease sequence specificity, the rules include both steric considerations relating to the distances in a meganuclease-DNA complex between the amino acid side chains of the meganuclease and the bases in the sense and anti-sense strands of the DNA, and considerations relating to the non-covalent chemical interactions between functional groups of the amino acid side chains and the desired DNA base at the relevant position.</p>
    <p num="p-0052">Finally, a majority of natural meganucleases that bind DNA as homodimers recognize pseudo- or completely palindromic recognition sequences. Because lengthy palindromes are expected to be rare, the likelihood of encountering a palindromic sequence at a genomic site of interest is exceedingly low. Consequently, if these enzymes are to be redesigned to recognize genomic sites of interest, it is necessary to design two enzyme monomers recognizing different half-sites that can heterodimerize to cleave the non-palindromic hybrid recognition sequence. Therefore, in some aspects, the invention provides rationally-designed meganucleases in which monomers differing by at least one amino acid position are dimerized to form heterodimers. In some cases, both monomers are rationally-designed to form a heterodimer which recognizes a non-palindromic recognition sequence. A mixture of two different monomers can result in up to three active forms of meganuclease dimer: the two homodimers and the heterodimer. In addition or alternatively, in some cases, amino acid residues are altered at the interfaces at which monomers can interact to form dimers, in order to increase or decrease the likelihood of formation of homodimers or heterodimers.</p>
    <p num="p-0053">Thus, in one aspect, the invention provide methods for rationally designing LAGLIDADG meganucleases containing amino acid changes that alter the specificity and/or activity of the enzymes. In another aspect, the invention provides the rationally-designed meganucleases resulting from these methods. In another aspect, the invention provides methods that use such rationally-designed meganucleases to produce recombinant nucleic acids and organisms in which a desired DNA sequence or genetic locus within the genome of an organism is modified by the insertion, deletion, substitution or other manipulation of DNA sequences. In another aspect, the invention provides methods for reducing the survival of pathogens or cancer cells using rationally-designed meganucleases which have pathogen-specific or cancer-specific recognition sequences.</p>
    <p num="h-0009">1.2 References and Definitions</p>
    <p num="p-0054">The patent and scientific literature referred to herein establishes knowledge that is available to those of skill in the art. The issued U.S. patents, allowed applications, published foreign applications, and references, including GenBank database sequences, that are cited herein are hereby incorporated by reference to the same extent as if each was specifically and individually indicated to be incorporated by reference.</p>
    <p num="p-0055">As used herein, the term “meganuclease” refers to an endonuclease that binds double-stranded DNA at a recognition sequence that is greater than 12 base pairs. Naturally-occurring meganucleases can be monomeric (e.g., I-SceI) or dimeric (e.g., I-CreI). The term meganuclease, as used herein, can be used to refer to monomeric meganucleases, dimeric meganucleases, or to the monomers which associate to form a dimeric meganuclease. The term “homing endonuclease” is synonymous with the term “meganuclease.”</p>
    <p num="p-0056">As used herein, the term “LAGLIDADG meganuclease” refers either to meganucleases including a single LAGLIDADG motif (SEQ ID NO: 37), which are naturally dimeric, or to meganucleases including two LAGLIDADG motifs (SEQ ID NO: 37), which are naturally monomeric. The term “mono-LAGLIDADG meganuclease” is used herein to refer to meganucleases including a single LAGLIDADG motif (SEQ ID NO: 37), and the term “di-LAGLIDADG meganuclease” is used herein to refer to meganucleases including two LAGLIDADG motifs (SEQ ID NO: 37), when it is necessary to distinguish between the two. Each of the two structural domains of a di-LAGLIDADG meganuclease which includes a LAGLIDADG motif (SEQ ID NO: 37) can be referred to as a LAGLIDADG subunit.</p>
    <p num="p-0057">As used herein, the term “rationally-designed” means non-naturally occurring and/or genetically engineered. The rationally-designed meganucleases of the invention differ from wild-type or naturally-occurring meganucleases in their amino acid sequence or primary structure, and may also differ in their secondary, tertiary or quaternary structure. In addition, the rationally-designed meganucleases of the invention also differ from wild-type or naturally-occurring meganucleases in recognition sequence-specificity and/or activity.</p>
    <p num="p-0058">As used herein, with respect to a protein, the term “recombinant” means having an altered amino acid sequence as a result of the application of genetic engineering techniques to nucleic acids which encode the protein, and cells or organisms which express the protein. With respect to a nucleic acid, the term “recombinant” means having an altered nucleic acid sequence as a result of the application of genetic engineering techniques. Genetic engineering techniques include, but are not limited to, PCR and DNA cloning technologies; transfection, transformation and other gene transfer technologies; homologous recombination; site-directed mutagenesis; and gene fusion. In accordance with this definition, a protein having an amino acid sequence identical to a naturally-occurring protein, but produced by cloning and expression in a heterologous host, is not considered recombinant.</p>
    <p num="p-0059">As used herein with respect to recombinant proteins, the term “modification” means any insertion, deletion or substitution of an amino acid residue in the recombinant sequence relative to a reference sequence (e.g., a wild-type).</p>
    <p num="p-0060">As used herein, the term “genetically-modified” refers to a cell or organism in which, or in an ancestor of which, a genomic DNA sequence has been deliberately modified by recombinant technology. As used herein, the term “genetically-modified” encompasses the term “transgenic.”</p>
    <p num="p-0061">As used herein, the term “wild-type” refers to any naturally-occurring form of a meganuclease. The term “wild-type” is not intended to mean the most common allelic variant of the enzyme in nature but, rather, any allelic variant found in nature. Wild-type meganucleases are distinguished from recombinant or non-naturally-occurring meganucleases.</p>
    <p num="p-0062">As used herein, the term “recognition sequence half-site” or simply “half site” means a nucleic acid sequence in a double-stranded DNA molecule which is recognized by a monomer of a mono-LAGLIDADG meganuclease or by one LAGLIDADG subunit of a di-LAGLIDADG meganuclease.</p>
    <p num="p-0063">As used herein, the term “recognition sequence” refers to a pair of half-sites which is bound and cleaved by either a mono-LAGLIDADG meganuclease dimer or a di-LAGLIDADG meganuclease monomer. The two half-sites may or may not be separated by base pairs that are not specifically recognized by the enzyme. In the cases of I-CreI, I-MsoI and I-CeuI, the recognition sequence half-site of each monomer spans 9 base pairs, and the two half-sites are separated by four base pairs which are not recognized specifically but which constitute the actual cleavage site (which has a 4 base pair overhang). Thus, the combined recognition sequences of the I-CreI, I-MsoI and I-CeuI meganuclease dimers normally span 22 base pairs, including two 9 base pair half-sites flanking a 4 base pair cleavage site. The base pairs of each half-site are designated −9 through −1, with the −9 position being most distal from the cleavage site and the −1 position being adjacent to the 4 central base pairs, which are designated N<sub>1</sub>-N<sub>4</sub>. The strand of each half-site which is oriented 5′ to 3′ in the direction from −9 to −1 (i.e., towards the cleavage site), is designated the “sense” strand and the opposite strand is designated the “antisense strand”, although neither strand may encode protein. Thus, the “sense” strand of one half-site is the antisense strand of the other half-site. See, for example, <figref idrefs="DRAWINGS">FIG. 1(A)</figref>. In the case of the I-SceI meganuclease, which is a di-LAGLIDADG meganuclease monomer, the recognition sequence is an approximately 18 bp non-palindromic sequence, and there are no central base pairs which are not specifically recognized. By convention, one of the two strands is referred to as the “sense” strand and the other the “antisense” strand, although neither strand may encode protein.</p>
    <p num="p-0064">As used herein, the term “specificity” means the ability of a meganuclease to recognize and cleave double-stranded DNA molecules only at a particular sequence of base pairs referred to as the recognition sequence, or only at a particular set of recognition sequences. The set of recognition sequences will share certain conserved positions or sequence motifs, but may be degenerate at one or more positions. A highly-specific meganuclease is capable of cleaving only one or a very few recognition sequences. Specificity can be determined in a cleavage assay as described in Example 1. As used herein, a meganuclease has “altered” specificity if it binds to and cleaves a recognition sequence which is not bound to and cleaved by a reference meganuclease (e.g., a wild-type) or if the rate of cleavage of a recognition sequence is increased or decreased by a statistically significant (p&lt;0.05) amount relative to a reference meganuclease.</p>
    <p num="p-0065">As used herein, the term “degeneracy” means the opposite of “specificity.” A highly-degenerate meganuclease is capable of cleaving a large number of divergent recognition sequences. A meganuclease can have sequence degeneracy at a single position within a half-site or at multiple, even all, positions within a half-site. Such sequence degeneracy can result from (i) the inability of any amino acid in the DNA-binding domain of a meganuclease to make a specific contact with any base at one or more positions in the recognition sequence, (ii) the ability of one or more amino acids in the DNA-binding domain of a meganuclease to make specific contacts with more than one base at one or more positions in the recognition sequence, and/or (iii) sufficient non-specific DNA binding affinity for activity. A “completely” degenerate position can be occupied by any of the four bases and can be designated with an “N” in a half-site. A “partially” degenerate position can be occupied by two or three of the four bases (e.g., either purine (Pu), either pyrimidine (Py), or not G).</p>
    <p num="p-0066">As used herein with respect to meganucleases, the term “DNA-binding affinity” or “binding affinity” means the tendency of a meganuclease to non-covalently associate with a reference DNA molecule (e.g., a recognition sequence or an arbitrary sequence). Binding affinity is measured by a dissociation constant, K<sub>D </sub>(e.g., the K<sub>D </sub>of I-CreI for the WT recognition sequence is approximately 0.1 nM). As used herein, a meganuclease has “altered” binding affinity if the K<sub>D </sub>of the recombinant meganuclease for a reference recognition sequence is increased or decreased by a statistically significant (p&lt;0.05) amount relative to a reference meganuclease.</p>
    <p num="p-0067">As used herein with respect to meganuclease monomers, the term “affinity for dimer formation” means the tendency of a meganuclease monomer to non-covalently associate with a reference meganuclease monomer. The affinity for dimer formation can be measured with the same monomer (i.e., homodimer formation) or with a different monomer (i.e., heterodimer formation) such as a reference wild-type meganuclease. Binding affinity is measured by a dissociation constant, K<sub>D</sub>. As used herein, a meganuclease has “altered” affinity for dimer formation if the K<sub>D </sub>of the recombinant meganuclease monomer for a reference meganuclease monomer is increased or decreased by a statistically significant (p&lt;0.05) amount relative to a reference meganuclease monomer.</p>
    <p num="p-0068">As used herein, the term “palindromic” refers to a recognition sequence consisting of inverted repeats of identical half-sites. In this case, however, the palindromic sequence need not be palindromic with respect to the four central base pairs, which are not contacted by the enzyme. In the case of dimeric meganucleases, palindromic DNA sequences are recognized by homodimers in which the two monomers make contacts with identical half-sites.</p>
    <p num="p-0069">As used herein, the term “pseudo-palindromic” refers to a recognition sequence consisting of inverted repeats of non-identical or imperfectly palindromic half-sites. In this case, the pseudo-palindromic sequence not only need not be palindromic with respect to the four central base pairs, but also can deviate from a palindromic sequence between the two half-sites. Pseudo-palindromic DNA sequences are typical of the natural DNA sites recognized by wild-type homodimeric meganucleases in which two identical enzyme monomers make contacts with different half-sites.</p>
    <p num="p-0070">As used herein, the term “non-palindromic” refers to a recognition sequence composed of two unrelated half-sites of a meganuclease. In this case, the non-palindromic sequence need not be palindromic with respect to either the four central base pairs or the two monomer half-sites. Non-palindromic DNA sequences are recognized by either di-LAGLIDADG meganucleases, highly degenerate mono-LAGLIDADG meganucleases (e.g., I-CeuI) or by heterodimers of mono-LAGLIDADG meganuclease monomers that recognize non-identical half-sites.</p>
    <p num="p-0071">As used herein, the term “activity” refers to the rate at which a meganuclease of the invention cleaves a particular recognition sequence. Such activity is a measurable enzymatic reaction, involving the hydrolysis of phosphodiester bonds of double-stranded DNA. The activity of a meganuclease acting on a particular DNA substrate is affected by the affinity or avidity of the meganuclease for that particular DNA substrate which is, in turn, affected by both sequence-specific and non-sequence-specific interactions with the DNA.</p>
    <p num="p-0072">As used herein, the term “homologous recombination” refers to the natural, cellular process in which a double-stranded DNA-break is repaired using a homologous DNA sequence as the repair template (see, e.g. Cahill et al. (2006), <i>Front. Biosci. </i>11: 1958-1976). The homologous DNA sequence may be an endogenous chromosomal sequence or an exogenous nucleic acid that was delivered to the cell. Thus, in some embodiments, a rationally-designed meganuclease is used to cleave a recognition sequence within a target sequence and an exogenous nucleic acid with homology to or substantial sequence similarity with the target sequence is delivered into the cell and used as a template for repair by homologous recombination. The DNA sequence of the exogenous nucleic acid, which may differ significantly from the target sequence, is thereby incorporated into the chromosomal sequence. The process of homologous recombination occurs primarily in eukaryotic organisms. The term “homology” is used herein as equivalent to “sequence similarity” and is not intended to require identity by descent or phylogenetic relatedness.</p>
    <p num="p-0073">As used herein, the term “non-homologous end-joining” refers to the natural, cellular process in which a double-stranded DNA-break is repaired by the direct joining of two non-homologous DNA segments (see, e.g. Cahill et al. (2006), <i>Front. Biosci. </i>11: 1958-1976). DNA repair by non-homologous end-joining is error-prone and frequently results in the untemplated addition or deletion of DNA sequences at the site of repair. Thus, in certain embodiments, a rationally-designed meganuclease can be used to produce a double-stranded break at a meganuclease recognition sequence within a target sequence to disrupt a gene (e.g., by introducing base insertions, base deletions, or frameshift mutations) by non-homologous end-joining. In other embodiments, an exogenous nucleic acid lacking homology to or substantial sequence similarity with the target sequence may be captured at the site of a meganuclease-stimulated double-stranded DNA break by non-homologous end-joining (see, e.g. Salomon, et al. (1998), <i>EMBO J. </i>17:6086-6095). The process of non-homologous end-joining occurs in both eukaryotes and prokaryotes such as bacteria.</p>
    <p num="p-0074">As used herein, the term “sequence of interest” means any nucleic acid sequence, whether it codes for a protein, RNA, or regulatory element (e.g., an enhancer, silencer, or promoter sequence), that can be inserted into a genome or used to replace a genomic DNA sequence using a meganuclease protein. Sequences of interest can have heterologous DNA sequences that allow for tagging a protein or RNA that is expressed from the sequence of interest. For instance, a protein can be tagged with tags including, but not limited to, an epitope (e.g., c-myc, FLAG) or other ligand (e.g., poly-His). Furthermore, a sequence of interest can encode a fusion protein, according to techniques known in the art (see, e.g., Ausubel et al., <i>Current Protocols in Molecular Biology</i>, Wiley 1999). In some embodiments, the sequence of interest is flanked by a DNA sequence that is recognized by the recombinant meganuclease for cleavage. Thus, the flanking sequences are cleaved allowing for proper insertion of the sequence of interest into genomic recognition sequences cleaved by the recombinant meganuclease. In some embodiments, the entire sequence of interest is homologous to or has substantial sequence similarity with the a target sequence in the genome such that homologous recombination effectively replaces the target sequence with the sequence of interest. In other embodiments, the sequence of interest is flanked by DNA sequences with homology to or substantial sequence similarity with the target sequence such that homologous recombination inserts the sequence of interest within the genome at the locus of the target sequence. In some embodiments, the sequence of interest is substantially identical to the target sequence except for mutations or other modifications in the meganuclease recognition sequence such that the meganuclease can not cleave the target sequence after it has been modified by the sequence of interest.</p>
    <p num="p-0075">As used herein with respect to both amino acid sequences and nucleic acid sequences, the terms “percentage similarity” and “sequence similarity” refer to a measure of the degree of similarity of two sequences based upon an alignment of the sequences which maximizes similarity between aligned amino acid residues or nucleotides, and which is a function of the number of identical or similar residues or nucleotides, the number of total residues or nucleotides, and the presence and length of gaps in the sequence alignment. A variety of algorithms and computer programs are available for determining sequence similarity using standard parameters. As used herein, sequence similarity is measured using the BLASTp program for amino acid sequences and the BLASTn program for nucleic acid sequences, both of which are available through the National Center for Biotechnology Information, and are described in, for example, Altschul et al. (1990), <i>J. Mol. Biol. </i>215:403-410; Gish and States (1993), <i>Nature Genet. </i>3:266-272; Madden et al. (1996), <i>Meth. Enzymol. </i>266:131-141; Altschul et al. (1997), <i>Nucleic Acids Res. </i>25:33 89-3402); Zhang et al. (2000), <i>J. Comput. Biol. </i>7(1-2):203-14. As used herein, percent similarity of two amino acid sequences is the score based upon the following parameters for the BLASTp algorithm: word size=3; gap opening penalty=−11; gap extension penalty=−1; and scoring matrix=BLOSUM62. As used herein, percent similarity of two nucleic acid sequences is the score based upon the following parameters for the BLASTn algorithm: word size=11; gap opening penalty=−5; gap extension penalty=−2; match reward=1; and mismatch penalty=−3.</p>
    <p num="p-0076">As used herein with respect to modifications of two proteins or amino acid sequences, the term “corresponding to” is used to indicate that a specified modification in the first protein is a substitution of the same amino acid residue as in the modification in the second protein, and that the amino acid position of the modification in the first proteins corresponds to or aligns with the amino acid position of the modification in the second protein when the two proteins are subjected to standard sequence alignments (e.g., using the BLASTp program). Thus, the modification of residue “X” to amino acid “A” in the first protein will correspond to the modification of residue “Y” to amino acid “A” in the second protein if residues X and Y correspond to each other in a sequence alignment, and despite the fact that X and Y may be different numbers.</p>
    <p num="p-0077">As used herein, the recitation of a numerical range for a variable is intended to convey that the invention may be practiced with the variable equal to any of the values within that range. Thus, for a variable which is inherently discrete, the variable can be equal to any integer value within the numerical range, including the end-points of the range. Similarly, for a variable which is inherently continuous, the variable can be equal to any real value within the numerical range, including the end-points of the range. As an example, and without limitation, a variable which is described as having values between 0 and 2 can take the values 0, 1 or 2 if the variable is inherently discrete, and can take the values 0.0, 0.1, 0.01, 0.001, or any other real values ≧0 and ≦2 if the variable is inherently continuous.</p>
    <p num="p-0078">As used herein, unless specifically indicated otherwise, the word “or” is used in the inclusive sense of “and/or” and not the exclusive sense of “either/or.”</p>
    <p num="h-0010">2.1 Rationally-Designed Meganucleases with Altered Sequence-Specificity</p>
    <p num="p-0079">In one aspect of the invention, methods for rationally designing recombinant LAGLIDADG family meganucleases are provided. In this aspect, recombinant meganucleases are rationally-designed by first predicting amino acid substitutions that can alter base preference at each position in the half-site. These substitutions can be experimentally validated individually or in combinations to produce meganucleases with the desired cleavage specificity.</p>
    <p num="p-0080">In accordance with the invention, amino acid substitutions that can cause a desired change in base preference are predicted by determining the amino acid side chains of a reference meganuclease (e.g., a wild-type meganuclease, or a non-naturally-occurring reference meganuclease) that are able to participate in making contacts with the nucleic acid bases of the meganuclease's DNA recognition sequence and the DNA phosphodiester backbone, and the spatial and chemical nature of those contacts. These amino acids include but are not limited to side chains involved in contacting the reference DNA half-site. Generally, this determination requires having knowledge of the structure of the complex between the meganuclease and its double-stranded DNA recognition sequence, or knowledge of the structure of a highly similar complex (e.g., between the same meganuclease and an alternative DNA recognition sequence, or between an allelic or phylogenetic variant of the meganuclease and its DNA recognition sequence).</p>
    <p num="p-0081">Three-dimensional structures, as described by atomic coordinates data, of a polypeptide or complex of two or more polypeptides can be obtained in several ways. For example, protein structure determinations can be made using techniques including, but not limited to, X-ray crystallography, NMR, and mass spectrometry. Another approach is to analyze databases of existing structural co-ordinates for the meganuclease of interest or a related meganuclease. Such structural data is often available from databases in the form of three-dimensional coordinates. Often this data is accessible through online databases (e.g., the RCSB Protein Data Bank).</p>
    <p num="p-0082">Structural information can be obtained experimentally by analyzing the diffraction patterns of, for example, X-rays or electrons, created by regular two- or three-dimensional arrays (e.g., crystals) of proteins or protein complexes. Computational methods are used to transform the diffraction data into three-dimensional atomic co-ordinates in space. For example, the field of X-ray crystallography has been used to generate three-dimensional structural information on many protein-DNA complexes, including meganucleases (see, e.g., Chevalier et al. (2001), <i>Nucleic Acids Res. </i>29(18): 3757-3774).</p>
    <p num="p-0083">Nuclear Magnetic Resonance (NMR) also has been used to determine inter-atomic distances of molecules in solution. Multi-dimensional NMR methods combined with computational methods have succeeded in determining the atomic co-ordinates of polypeptides of increasing size (see, e.g., Tzakos et al. (2006), <i>Annu. Rev. Biophys. Biomol. Struct. </i>35:19-42.).</p>
    <p num="p-0084">Alternatively, computational modeling can be used by applying algorithms based on the known primary structures and, when available, secondary, tertiary and/or quaternary structures of the protein/DNA, as well as the known physiochemical nature of the amino acid side chains, nucleic acid bases, and bond interactions. Such methods can optionally include iterative approaches, or experimentally-derived constraints. An example of such computational software is the CNS program described in Adams et al. (1999), <i>Acta Crystallogr. D. Biol. Crystallogr. </i>55 (Pt 1): 181-90. A variety of other computational programs have been developed that predict the spatial arrangement of amino acids in a protein structure and predict the interaction of the amino acid side chains of the protein with various target molecules (see, e.g., U.S. Pat. No. 6,988,041).</p>
    <p num="p-0085">Thus, in some embodiments of the invention, computational models are used to identify specific amino acid residues that specifically interact with DNA nucleic acid bases and/or facilitate non-specific phosphodiester backbone interactions. For instance, computer models of the totality of the potential meganuclease-DNA interaction can be produced using a suitable software program, including, but not limited to, MOLSCRIPT™ 2.0 (Avatar Software AB, Stockholm, Sweden), the graphical display program O (Jones et. al. (1991), <i>Acta Crystallography</i>, A47: 110), the graphical display program GRASP™ (Nicholls et al. (1991), <i>PROTEINS, Structure, Function and Genetics </i>11(4): 281ff), or the graphical display program INSIGH™ (TSI, Inc., Shoreview, Minn.). Computer hardware suitable for producing, viewing and manipulating three-dimensional structural representations of protein-DNA complexes are commercially available and well known in the art (e.g., Silicon Graphics Workstation, Silicon Graphics, Inc., Mountainview, Calif.).</p>
    <p num="p-0086">Specifically, interactions between a meganuclease and its double-stranded DNA recognition sequences can be resolved using methods known in the art. For example, a representation, or model, of the three dimensional structure of a multi-component complex structure, for which a crystal has been produced, can be determined using techniques which include molecular replacement or SIR/MIR (single/multiple isomorphous replacement) (see, e.g., Brunger (1997), <i>Meth. Enzym. </i>276: 558-580; Navaza and Saludjian (1997), <i>Meth. Enzym. </i>276: 581-594; Tong and Rossmann (1997), <i>Meth. Enzym. </i>276: 594-611; and Bentley (1997), <i>Meth. Enzym. </i>276: 611-619) and can be performed using a software program, such as AMoRe/Mosflm (Navaza (1994), <i>Acta Cryst. A</i>50: 157-163; CCP4 (1994), <i>Acta Cryst. D</i>50: 760-763) or XPLOR (see, Brünger et al. (1992), <i>X</i>-<i>PLOR Version </i>3.1<i>. A System for X</i>-<i>ray Crystallography and NMR</i>, Yale University Press, New Haven, Conn.).</p>
    <p num="p-0087">The determination of protein structure and potential meganuclease-DNA interaction allows for rational choices concerning the amino acids that can be changed to affect enzyme activity and specificity. Decisions are based on several factors regarding amino acid side chain interactions with a particular base or DNA phosphodiester backbone. Chemical interactions used to determine appropriate amino acid substitutions include, but are not limited to, van der Waals forces, steric hindrance, ionic bonding, hydrogen bonding, and hydrophobic interactions. Amino acid substitutions can be selected which either favor or disfavor specific interactions of the meganuclease with a particular base in a potential recognition sequence half-site in order to increase or decrease specificity for that sequence and, to some degree, overall binding affinity and activity. In addition, amino acid substitutions can be selected which either increase or decrease binding affinity for the phosphodiester backbone of double-stranded DNA in order to increase or decrease overall activity and, to some degree, to decrease or increase specificity.</p>
    <p num="p-0088">Thus, in specific embodiments, a three-dimensional structure of a meganuclease-DNA complex is determined and a “contact surface” is defined for each base-pair in a DNA recognition sequence half-site. In some embodiments, the contact surface comprises those amino acids in the enzyme with β-carbons less than 9.0 Å from a major groove hydrogen-bond donor or acceptor on either base in the pair, and with side chains oriented toward the DNA, irrespective of whether the residues make base contacts in the wild-type meganuclease-DNA complex. In other embodiments, residues can be excluded if the residues do not make contact in the wild-type meganuclease-DNA complex, or residues can be included or excluded at the discretion of the designer to alter the number or identity of the residues considered. In one example, as described below, for base positions −2, −7, −8, and −9 of the wild-type I-CreI half-site, the contact surfaces were limited to the amino acid positions that actually interact in the wild-type enzyme-DNA complex. For positions −1, −3, −4, −5, and −6, however, the contact surfaces were defined to contain additional amino acid positions that are not involved in wild-type contacts but which could potentially contact a base if substituted with a different amino acid.</p>
    <p num="p-0089">It should be noted that, although a recognition sequence half-site is typically represented with respect to only one strand of DNA, meganucleases bind in the major groove of double-stranded DNA, and make contact with nucleic acid bases on both strands. In addition, the designations of “sense” and “antisense” strands are completely arbitrary with respect to meganuclease binding and recognition. Sequence specificity at a position can be achieved either through interactions with one member of a base pair, or by a combination of interactions with both members of a base-pair. Thus, for example, in order to favor the presence of an A/T base pair at position X, where the A base is on the “sense” strand and the T base is on the “antisense” strand, residues are selected which are sufficiently close to contact the sense strand at position X and which favor the presence of an A, and/or residues are selected which are sufficiently close to contact the antisense strand at position X and which favor the presence of a T. In accordance with the invention, a residue is considered sufficiently close if the β-carbon of the residue is within 9 Å of the closest atom of the relevant base.</p>
    <p num="p-0090">Thus, for example, an amino acid with a β-carbon within 9 Å of the DNA sense strand but greater than 9 Å from the antisense strand is considered for potential interactions with only the sense strand. Similarly, an amino acid with a β-carbon within 9 Å of the DNA antisense strand but greater than 9 Å from the sense strand is considered for potential interactions with only the antisense strand. Amino acids with β-carbons that are within 9 Å of both DNA strands are considered for potential interactions with either strand.</p>
    <p num="p-0091">For each contact surface, potential amino acid substitutions are selected based on their predicted ability to interact favorably with one or more of the four DNA bases. The selection process is based upon two primary criteria: (i) the size of the amino acid side chains, which will affect their steric interactions with different nucleic acid bases, and (ii) the chemical nature of the amino acid side chains, which will affect their electrostatic and bonding interactions with the different nucleic acid bases.</p>
    <p num="p-0092">With respect to the size of side chains, amino acids with shorter and/or smaller side chains can be selected if an amino acid β-carbon in a contact surface is &lt;6 Å from a base, and amino acids with longer and/or larger side chains can be selected if an amino acid β-carbon in a contact surface is &gt;6 Å from a base. Amino acids with side chains that are intermediate in size can be selected if an amino acid β-carbon in a contact surface is 5-8 Å from a base.</p>
    <p num="p-0093">The amino acids with relatively shorter and smaller side chains can be assigned to Group 1, including glycine (G), alanine (A), serine (S), threonine (T), cysteine (C), valine (V), leucine (L), isoleucine (I), aspartate (D), asparagine (N) and proline (P). Proline, however, is expected to be used less frequently because of its relative inflexibility. In addition, glycine is expected to be used less frequently because it introduces unwanted flexibility in the peptide backbone and its very small size reduces the likelihood of effective contacts when it replaces a larger residue. On the other hand, glycine can be used in some instances for promoting a degenerate position. The amino acids with side chains of relatively intermediate length and size can be assigned to Group 2, including lysine (K), methionine (M), arginine (R), glutamate (E) and glutamine (Q). The amino acids with relatively longer and/or larger side chains can be assigned to Group 3, including lysine (K), methionine (M), arginine (R), histidine (H), phenylalanine (F), tyrosine (Y), and tryptophan (W). Tryptophan, however, is expected to be used less frequently because of its relative inflexibility. In addition, the side chain flexibility of lysine, arginine, and methionine allow these amino acids to make base contacts from long or intermediate distances, warranting their inclusion in both Groups 2 and 3. These groups are also shown in tabular form below:</p>
    <p num="p-0094">
      <tables id="TABLE-US-00001" num="00001"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="77pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Group 1</td>
                <td class="description-td">Group 2</td>
                <td class="description-td">Group 3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">glycine (G)</td>
                <td class="description-td">glutamine (Q)</td>
                <td class="description-td">arginine (R)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">alanine (A)</td>
                <td class="description-td">glutamate (E)</td>
                <td class="description-td">histidine (H)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">serine (S)</td>
                <td class="description-td">lysine (K)</td>
                <td class="description-td">phenylalanine (F)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">threonine (T)</td>
                <td class="description-td">methionine (M)</td>
                <td class="description-td">tyrosine (Y)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cysteine (C)</td>
                <td class="description-td">arginine (R)</td>
                <td class="description-td">tryptophan (W)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">valine (V)</td>
                <td class="description-td"> </td>
                <td class="description-td">lysine (K)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">leucine (L)</td>
                <td class="description-td"> </td>
                <td class="description-td">methionine (M)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">isoleucine (I)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">aspartate (D)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">asparagine (N)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">proline (P)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0095">With respect to the chemical nature of the side chains, the different amino acids are evaluated for their potential interactions with the different nucleic acid bases (e.g., van der Waals forces, ionic bonding, hydrogen bonding, and hydrophobic interactions) and residues are selected which either favor or disfavor specific interactions of the meganuclease with a particular base at a particular position in the double-stranded DNA recognition sequence half-site. In some instances, it may be desired to create a half-site with one or more complete or partial degenerate positions. In such cases, one may choose residues which favor the presence of two or more bases, or residues which disfavor one or more bases. For example, partial degenerate base recognition can be achieved by sterically hindering a pyrimidine at a sense or antisense position.</p>
    <p num="p-0096">Recognition of guanine (G) bases is achieved using amino acids with basic side chains that form hydrogen bonds to N7 and O6 of the base. Cytosine (C) specificity is conferred by negatively-charged side chains which interact unfavorably with the major groove electronegative groups present on all bases except C. Thymine (T) recognition is rationally-designed using hydrophobic and van der Waals interactions between hydrophobic side chains and the major groove methyl group on the base. Finally, adenine (A) bases are recognized using the carboxamide side chains Asn and Gln or the hydroxyl side chain of Tyr through a pair of hydrogen bonds to N7 and N6 of the base. Lastly, His can be used to confer specificity for a purine base (A or G) by donating a hydrogen bond to N7. These straightforward rules for DNA recognition can be applied to predict contact surfaces in which one or both of the bases at a particular base-pair position are recognized through a rationally-designed contact.</p>
    <p num="p-0097">Thus, based on their binding interactions with the different nucleic acid bases, and the bases which they favor at a position with which they make contact, each amino acid residue can be assigned to one or more different groups corresponding to the different bases they favor (i.e., G, C, T or A). Thus, Group G includes arginine (R), lysine (K) and histidine (H); Group C includes aspartate (D) and glutamate (E); Group T includes alanine (A), valine (V), leucine (L), isoleucine (I), cysteine (C), threonine (T), methionine (M) and phenylalanine (F); and Group A includes asparagine (N), glutamine (N), tyrosine (Y) and histidine (H). Note that histidine appears in both Group G and Group A; that serine (S) is not included in any group but may be used to favor a degenerate position; and that proline, glycine, and tryptophan are not included in any particular group because of predominant steric considerations. These groups are also shown in tabular form below:</p>
    <p num="p-0098">
      <tables id="TABLE-US-00002" num="00002"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="49pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="56pt" align="left"> </colspec> <colspec colname="4" colwidth="56pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Group G</td>
                <td class="description-td">Group C</td>
                <td class="description-td">Group T</td>
                <td class="description-td">Group A</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">arginine (R)</td>
                <td class="description-td">aspartate (D)</td>
                <td class="description-td">alanine (A)</td>
                <td class="description-td">asparagine (N)</td>
              </tr> <tr class="description-tr"> <td class="description-td">lysine (K)</td>
                <td class="description-td">glutamate (E)</td>
                <td class="description-td">valine (V)</td>
                <td class="description-td">glutamine (Q)</td>
              </tr> <tr class="description-tr"> <td class="description-td">histidine (H)</td>
                <td class="description-td"> </td>
                <td class="description-td">leucine (L)</td>
                <td class="description-td">tyrosine (Y)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">isoleucine (I)</td>
                <td class="description-td">histidine (H)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">cysteine (C)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">threonine (T)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">methionine (M)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">phenylalanine (F)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0099">Thus, in accordance with the invention, in order to effect a desired change in the recognition sequence half-site of a meganuclease at a given position X, (1) determine at least the relevant portion of the three-dimensional structure of the wild-type or reference meganuclease-DNA complex and the amino acid residue side chains which define the contact surface at position X; (2) determine the distance between the β-carbon of at least one residue comprising the contact surface and at least one base of the base pair at position X; and (3)(a) for a residue which is &lt;6 Å from the base, select a residue from Group 1 and/or Group 2 which is a member of the appropriate one of Group G, Group C, Group T or Group A to promote the desired change, and/or (b) for a residue which is &gt;6 Å from the base, select a residue from Group 2 and/or Group 3 which is a member of the appropriate one of Group G, Group C, Group T or Group A to promote the desired change. More than one such residue comprising the contact surface can be selected for analysis and modification and, in some embodiments, each such residue is analyzed and multiple residues are modified. Similarly, the distance between the β-carbon of a residue included in the contact surface and each of the two bases of the base pair at position X can be determined and, if the residue is within 9 Å of both bases, then different substitutions can be made to affect the two bases of the pair (e.g., a residue from Group 1 to affect a proximal base on one strand, or a residue from Group 3 to affect a distal base on the other strand). Alternatively, a combination of residue substitutions capable of interacting with both bases in a pair can affect the specificity (e.g., a residue from the T Group contacting the sense strand combined with a residue from the A Group contacting the antisense strand to select for T/A). Finally, multiple alternative modifications of the residues can be validated either empirically (e.g., by producing the recombinant meganuclease and testing its sequence recognition) or computationally (e.g., by computer modeling of the meganuclease-DNA complex of the modified enzyme) to choose amongst alternatives.</p>
    <p num="p-0100">Once one or more desired amino acid modifications of the wild-type or reference meganuclease are selected, the rationally-designed meganuclease can be produced by recombinant methods and techniques well known in the art. In some embodiments, non-random or site-directed mutagenesis techniques are used to create specific sequence modifications. Non-limiting examples of non-random mutagenesis techniques include overlapping primer PCR (see, e.g., Wang et al. (2006), <i>Nucleic Acids Res. </i>34(2): 517-527), site-directed mutagenesis (see, e.g., U.S. Pat. No. 7,041,814), cassette mutagenesis (see, e.g., U.S. Pat. No. 7,041,814), and the manufacturer's protocol for the Altered Sites® II Mutagenesis Systems kit commercially available from Promega Biosciences, Inc. (San Luis Obispo, Calif.).</p>
    <p num="p-0101">The recognition and cleavage of a specific DNA sequence by a rationally-designed meganuclease can be assayed by any method known by one skilled in the art (see, e.g., U.S. Pat. Pub. No. 2006/0078552). In certain embodiments, the determination of meganuclease cleavage is determined by in vitro cleavage assays. Such assays use in vitro cleavage of a polynucleotide substrate comprising the intended recognition sequence of the assayed meganuclease and, in certain embodiments, variations of the intended recognition sequence in which one or more bases in one or both half-sites have been changed to a different base. Typically, the polynucleotide substrate is a double-stranded DNA molecule comprising a target site which has been synthesized and cloned into a vector. The polynucleotide substrate can be linear or circular, and typically comprises only one recognition sequence. The meganuclease is incubated with the polynucleotide substrate under appropriate conditions, and the resulting polynucleotides are analyzed by known methods for identifying cleavage products (e.g., electrophoresis or chromatography). If there is a single recognition sequence in a linear, double-strand DNA substrate, the meganuclease activity is detected by the appearance of two bands (products) and the disappearance of the initial full-length substrate band. In one embodiment, meganuclease activity can be assayed as described in, for example, Wang et al. (1997), <i>Nucleic Acid Res., </i>25: 3767-3776.</p>
    <p num="p-0102">In other embodiments, the cleavage pattern of the meganuclease is determined using in vivo cleavage assays (see, e.g., U.S. Pat. Pub. No. 2006/0078552). In particular embodiments, the in vivo test is a single-strand annealing recombination test (SSA). This kind of test is known to those of skill in the art (Rudin et al. (1989), <i>Genetics </i>122: 519-534; Fishman-Lobell et al. (1992), <i>Science </i>258: 480-4).</p>
    <p num="p-0103">As will be apparent to one of skill in the art, additional amino acid substitutions, insertions or deletions can be made to domains of the meganuclease enzymes other than those involved in DNA recognition and binding without complete loss of activity. Substitutions can be conservative substitutions of similar amino acid residues at structurally or functionally constrained positions, or can be non-conservative substitutions at positions which are less structurally or functionally constrained. Such substitutions, insertions and deletions can be identified by one of ordinary skill in the art by routine experimentation without undue effort. Thus, in some embodiments, the recombinant meganucleases of the invention include proteins having anywhere from 85% to 99% sequence similarity (e.g., 85%, 87.5%, 90%, 92.5%, 95%, 97.5%, 99%) to a reference meganuclease sequence. With respect to each of the wild-type I-CreI, I-MsoI, I-SceI and I-CeuI proteins, the most N-terminal and C-terminal sequences are not clearly visible in X-ray crystallography studies, suggesting that these positions are not structurally or functionally constrained. Therefore, these residues can be excluded from calculation of sequence similarity, and the following reference meganuclease sequences can be used: residues 2-153 of SEQ ID NO: 1 for I-CreI, residues 6-160 of SEQ ID NO: 6 for I-MsoI, residues 3-186 of SEQ ID NO: 9 for I-SceI, and residues 5-211 of SEQ ID NO: 12 for I-CeuI.</p>
    <p num="h-0011">2.2 LAGLIDADG Family Meganucleases</p>
    <p num="p-0104">The LAGLIDADG meganuclease family is composed of more than 200 members from a diverse phylogenetic group of host organisms. All members of this family have one or two copies of a highly conserved LAGLIDADG motif (SEQ ID NO: 37) along with other structural motifs involved in cleavage of specific DNA sequences. Enzymes that have a single copy of the LAGLIDADG motif (SEQ ID NO: 37) (i.e., mono-LAGLIDADG meganucleases) function as dimers, whereas the enzymes that have two copies of this motif (i.e., di-LAGLIDADG meganucleases) function as monomers.</p>
    <p num="p-0105">All LAGLIDADG family members recognize and cleave relatively long sequences (&gt;12 bp), leaving four nucleotide 3′ overhangs. These enzymes also share a number of structural motifs in addition to the LAGLIDADG motif (SEQ ID NO: 37), including a similar arrangement of anti-parallel β-strands at the protein-DNA interface. Amino acids within these conserved structural motifs are responsible for interacting with the DNA bases to confer recognition sequence specificity. The overall structural similarity between some members of the family (e.g., I-CreI, I-MsoI, I-SceI and I-CeuI) has been elucidated by X-ray crystallography. Accordingly, the members of this family can be modified at particular amino acids within such structural motifs to change the over-all activity or sequence-specificity of the enzymes, and corresponding modifications can reasonable be expected to have similar results in other family members. See, generally, Chevalier et al. (2001), <i>Nucleic Acid Res. </i>29(18): 3757-3774).</p>
    <p num="h-0012">2.2.1 Meganucleases Derived from I-CreI</p>
    <p num="p-0106">In one aspect, the present invention relates to rationally-designed meganucleases which are based upon or derived from the I-CreI meganuclease of <i>Chlamydomonas reinhardtii</i>. The wild-type amino acid sequence of the I-CreI meganuclease is shown in SEQ ID NO: 1, which corresponds to Genbank Accession # PO5725. Two recognition sequence half sites of the wild-type I-CreI meganuclease from crystal structure PDB # 1BP7 are shown below:</p>
    <p num="p-0107"> <tables id="TABLE-US-00003" num="00003"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Position</td> <td class="description-td">  -9-8-7-6-5-4-3-2-1</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="259pt" align="right"> </colspec> <colspec colname="2" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SEQ ID NO: 2</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">5′-<b>G A A A C T G T C </b>T C A C G A C G T T T T G-3′</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="259pt" align="right"> </colspec> <colspec colname="2" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SEQ ID NO: 3</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">3′-C T T T G A C A G A G T G <b>C T G C A A A A C</b>-5′</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Position</td> <td class="description-td">                            -1-2-3-4-5-6-7-8-9</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Note that this natural recognition sequence is not perfectly palindromic, even outside the central four base pairs. The two recognition sequence half-sites are shown in bold on their respective sense strands.
</p>
    <p num="p-0108">Wild-type I-CreI also recognizes and cuts the following perfectly palindromic (except for the central N<sub>1</sub>—N<sub>4 </sub>bases) sequence:</p>
    <p num="p-0109">
      <tables id="TABLE-US-00004" num="00004"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Position</td>
                <td class="description-td">  -9-8-7-6-5-4-3-2-1</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="259pt" align="right"> </colspec> <colspec colname="2" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SEQ ID NO: 4</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">5′-<b>C A A A C T G T C </b>G T G A G A C A G T T T G-3′</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="259pt" align="right"> </colspec> <colspec colname="2" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SEQ ID NO: 5</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">3′-G T T T G A C A G C A C T <b>C T G T C A A A C</b>-5′</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Position</td>
                <td class="description-td">                            -1-2-3-4-5-6-7-8-9</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0110">The palindromic sequence of SEQ ID NO: 4 and SEQ ID NO: 5 is considered to be a better substrate for the wild-type I-CreI because the enzyme binds this site with higher affinity and cleaves it more efficiently than the natural DNA sequence. For the purposes of the following disclosure, and with particular regard to the experimental results presented herein, this palindromic sequence cleaved by wild-type I-CreI is referred to as “WT” (see, e.g., <figref idrefs="DRAWINGS">FIG. 2(A)</figref>). The two recognition sequence half-sites are shown in bold on their respective sense strands.</p>
    <p num="p-0111"> <figref idrefs="DRAWINGS">FIG. 1(A)</figref> depicts the interactions of a wild-type I-CreI meganuclease homodimer with a double-stranded DNA recognition sequence, <figref idrefs="DRAWINGS">FIG. 1(B)</figref> shows the specific interactions between amino acid residues of the enzyme and bases at the −4 position of one half-site for a wild-type enzyme and one wild-type recognition sequence, and FIGS. <b>1</b>(C)-(E) show the specific interactions between amino acid residues of the enzyme and bases at the −4 position of one half-site for three rationally-designed meganucleases of the invention with altered specificity at position −4 of the half-site.</p>
    <p num="p-0112">Thus, the base preference at any specified base position of the half-site can be rationally altered to each of the other three base pairs using the methods disclosed herein. First, the wild type recognition surface at the specified base position is determined (e.g., by analyzing meganuclease-DNA complex co-crystal structures; or by computer modeling of the meganuclease-DNA complexes). Second, existing and potential contact residues are determined based on the distances between the β-carbons of the surrounding amino acid positions and the nucleic acid bases on each DNA strand at the specified base position. For example, and without limitation, as shown in <figref idrefs="DRAWINGS">FIG. 1(A)</figref>, the I-CreI wild type meganuclease-DNA contact residues at position −4 involve a glutamine at position 26 which hydrogen bonds to an A base on the antisense DNA strand. Residue 77 was also identified as potentially being able to contact the −4 base on the DNA sense strand. The β-carbon of residue 26 is 5.9 Å away from N7 of the A base on the antisense DNA strand, and the β-carbon of residue 77 is 7.15 Å away from the C5-methyl of the T on the sense strand. According to the distance and base chemistry rules described herein, a C on the sense strand could hydrogen bond with a glutamic acid at position 77 and a G on the antisense strand could bond with glutamine at position 26 (mediated by a water molecule, as observed in the wild-type I-CreI crystal structure) (see FIG. <b>1</b>(C)); a G on the sense strand could hydrogen bond with an arginine at position 77 and a C on the antisense strand could hydrogen bond with a glutamic acid at position 26 (see FIG. <b>1</b>(D)); an A on the sense strand could hydrogen bond with a glutamine at position 77 and a T on the antisense strand could form hydrophobic contacts with an alanine at position 26 (see <figref idrefs="DRAWINGS">FIG. 1(E)</figref>). If the base specific contact is provided by position 77, then the wild-type contact, Q26, can be substituted (e.g., with a serine residue) to reduce or remove its influence on specificity. Alternatively, complementary mutations at positions 26 and 77 can be combined to specify a particular base pair (e.g., A26 specifies a T on the antisense strand and Q77 specifies an A on the sense strand (<figref idrefs="DRAWINGS">FIG. 1(E)</figref>). These predicted residue substitutions have all been validated experimentally.</p>
    <p num="p-0113">Thus, in accordance with the invention, a substantial number of amino acid modifications to the DNA recognition domain of the I-CreI meganuclease have been identified which, singly or in combination, result in recombinant meganucleases with specificities altered at individual bases within the DNA recognition sequence half-site, such that these rationally-designed meganucleases have half-sites different from the wild-type enzyme. The amino acid modifications of I-CreI and the resulting change in recognition sequence half-site specificity are shown in Table 1:</p>
    <p num="p-0114"> <tables id="TABLE-US-00005" num="00005"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="259pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" rowsep="1" class="description-td" colspan="2">TABLE 1</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Favored Sense-Strand Base</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="12"> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <colspec colname="11" colwidth="28pt" align="left"> </colspec> <colspec colname="12" colwidth="35pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Posn.</td> <td class="description-td">A</td> <td class="description-td">C</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">A/T</td> <td class="description-td">A/C</td> <td class="description-td">A/G</td> <td class="description-td">C/T</td> <td class="description-td">G/T</td> <td class="description-td">A/G/T</td> <td class="description-td">A/C/G/T</td> </tr> <tr class="description-tr"> <td namest="1" nameend="12" align="center" rowsep="1" class="description-td" colspan="12"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="12"> <colspec colname="1" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="28pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <colspec colname="11" colwidth="28pt" align="left"> </colspec> <colspec colname="12" colwidth="35pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">−1</td> <td class="description-td">Y75</td> <td class="description-td"> <b>R70*</b> </td> <td class="description-td">K70</td> <td class="description-td">Q70*</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">T46*</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">G70</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L75*</td> <td class="description-td">H75*</td> <td class="description-td">E70*</td> <td class="description-td">C70</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">A70</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C75*</td> <td class="description-td">R75*</td> <td class="description-td">E75*</td> <td class="description-td">L70</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">S70</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Y139*</td> <td class="description-td">H46*</td> <td class="description-td">E46*</td> <td class="description-td">Y75*</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">G46*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C46*</td> <td class="description-td">K46*</td> <td class="description-td">D46*</td> <td class="description-td">Q75*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">A46*</td> <td class="description-td">R46*</td> <td class="description-td"> </td> <td class="description-td">H75*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">H139</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q46*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">H46*</td> </tr> <tr class="description-tr"> <td class="description-td">−2</td> <td class="description-td">Q70</td> <td class="description-td">E70</td> <td class="description-td">H70</td> <td class="description-td"> <b>Q44*</b> </td> <td class="description-td">C44*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">T44*</td> <td class="description-td">D70</td> <td class="description-td">D44*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">A44*</td> <td class="description-td">K44*</td> <td class="description-td">E44*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V44*</td> <td class="description-td">R44*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">I44*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L44*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">N44*</td> </tr> <tr class="description-tr"> <td class="description-td">−3</td> <td class="description-td">Q68</td> <td class="description-td">E68</td> <td class="description-td">R<b>68</b> </td> <td class="description-td">M68</td> <td class="description-td"> </td> <td class="description-td">H68</td> <td class="description-td"> </td> <td class="description-td">Y68</td> <td class="description-td">K68</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C24*</td> <td class="description-td">F68</td> <td class="description-td"> </td> <td class="description-td">C68</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> <b>I24*</b> </td> <td class="description-td">K24*</td> <td class="description-td"> </td> <td class="description-td">L68</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">R24*</td> <td class="description-td"> </td> <td class="description-td">F68</td> </tr> <tr class="description-tr"> <td class="description-td">−4</td> <td class="description-td">A26*</td> <td class="description-td">E77</td> <td class="description-td">R77</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">S77</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">S26*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Q77</td> <td class="description-td">K26*</td> <td class="description-td">E26*</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>Q26*</b> </td> </tr> <tr class="description-tr"> <td class="description-td">−5</td> <td class="description-td"> </td> <td class="description-td">E42</td> <td class="description-td">R42</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>K28*</b> </td> <td class="description-td">C28*</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">M66</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q42</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K66</td> </tr> <tr class="description-tr"> <td class="description-td">−6</td> <td class="description-td">Q40</td> <td class="description-td">E40</td> <td class="description-td">R40</td> <td class="description-td">C40</td> <td class="description-td">A40</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>S40</b> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C28*</td> <td class="description-td">R28*</td> <td class="description-td"> </td> <td class="description-td">I40</td> <td class="description-td">A79</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">S28*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">V40</td> <td class="description-td">A28*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">C79</td> <td class="description-td">H28*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">I79</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">V79</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q28*</td> </tr> <tr class="description-tr"> <td class="description-td">−7</td> <td class="description-td"> <b>N30*</b> </td> <td class="description-td">E38</td> <td class="description-td">K38</td> <td class="description-td">I38</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">C38</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">H38</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> <b>Q38</b> </td> <td class="description-td">K30*</td> <td class="description-td">R38</td> <td class="description-td">L38</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">N38</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">R30*</td> <td class="description-td">E30*</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q30*</td> </tr> <tr class="description-tr"> <td class="description-td">−8</td> <td class="description-td">F33</td> <td class="description-td">E33</td> <td class="description-td">F33</td> <td class="description-td">L33</td> <td class="description-td"> </td> <td class="description-td">R32*</td> <td class="description-td">R33</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> <b>Y33</b> </td> <td class="description-td">D33</td> <td class="description-td">H33</td> <td class="description-td">V33</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">I33</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">F33</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">C33</td> </tr> <tr class="description-tr"> <td class="description-td">−9</td> <td class="description-td"> </td> <td class="description-td">E32</td> <td class="description-td">R32</td> <td class="description-td">L32</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">D32</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>S32</b> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K32</td> <td class="description-td">V32</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">I32</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">N32</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">A32</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">H32</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">C32</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q32</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">T32</td> </tr> <tr class="description-tr"> <td namest="1" nameend="12" align="center" rowsep="1" class="description-td" colspan="12"> </td> </tr> <tr class="description-tr"> <td namest="1" nameend="12" align="left" id="FOO-00001" class="description-td" colspan="12">Bold entries are wild-type contact residues and do not constitute “modifications” as used herein.</td> </tr> <tr class="description-tr"> <td namest="1" nameend="12" align="left" id="FOO-00002" class="description-td" colspan="12">An asterisk indicates that the residue contacts the base on the antisense strand.</td> </tr> </tbody> </tgroup> </table> </tables> <br>
2.2.2 Meganucleases Derived from I-MsoI
</p>
    <p num="p-0115">In another aspect, the present invention relates to rationally-designed meganucleases which are based upon or derived from the I-MsoI meganuclease of <i>Monomastix </i>sp. The wild-type amino acid sequence of the I-MsoI meganuclease is shown in SEQ ID NO: 6, which corresponds to Genbank Accession # AAL34387. Two recognition sequence half-sites of the wild-type I-MsoI meganuclease from crystal structure PDB # 1M5X are shown below:</p>
    <p num="p-0116"> <tables id="TABLE-US-00006" num="00006"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Position</td> <td class="description-td">  -9-8-7-6-5-4-3-2-1</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="259pt" align="right"> </colspec> <colspec colname="2" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SEQ ID NO: 7</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">5′-<b>C A G A A C G T C </b>G T G A G A C A G T T C C-3′</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="259pt" align="right"> </colspec> <colspec colname="2" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SEQ ID NO: 8</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">3′-G T C T T G C A G C A C T <b>C T G T C A A G G</b>-5′</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Position</td> <td class="description-td">                            -1-2-3-4-5-6-7-8-9</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Note that the recognition sequence is not perfectly palindromic, even outside the central four base pairs. The two recognition sequence half-sites are shown in bold on their respective sense strands.
</p>
    <p num="p-0117">In accordance with the invention, a substantial number of amino acid modifications to the DNA recognition domain of the I-MsoI meganuclease have been identified which, singly or in combination, can result in recombinant meganucleases with specificities altered at individual bases within the DNA recognition sequence half-sites, such that these rationally-designed meganucleases have recognition sequences different from the wild-type enzyme. Amino acid modifications of I-MsoI and the predicted change in recognition sequence half-site specificity are shown in Table 2:</p>
    <p num="p-0118"> <tables id="TABLE-US-00007" num="00007"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="140pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 2</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Favored Sense-Strand Base</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <colspec colname="5" colwidth="42pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Position</td> <td class="description-td">A</td> <td class="description-td">C</td> <td class="description-td">G</td> <td class="description-td">T</td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td> </tr> <tr class="description-tr"> <td class="description-td">−1</td> <td class="description-td">K75*</td> <td class="description-td"> <b>D77</b> </td> <td class="description-td">K77</td> <td class="description-td">C77</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Q77</td> <td class="description-td">E77</td> <td class="description-td">R77</td> <td class="description-td">L77</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">A49*</td> <td class="description-td">K49*</td> <td class="description-td">E49*</td> <td class="description-td">Q79*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C49*</td> <td class="description-td"> <b>R75*</b> </td> <td class="description-td">E79*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">K79*</td> <td class="description-td">K75*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">R79*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K79*</td> </tr> <tr class="description-tr"> <td class="description-td">−2</td> <td class="description-td">Q75</td> <td class="description-td">E75</td> <td class="description-td">K75</td> <td class="description-td">A75</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">K81</td> <td class="description-td">D75</td> <td class="description-td">E47*</td> <td class="description-td">C75</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C47*</td> <td class="description-td">R47*</td> <td class="description-td">E81*</td> <td class="description-td">V75</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">I47*</td> <td class="description-td">K47*</td> <td class="description-td"> </td> <td class="description-td">I75</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L47*</td> <td class="description-td">K81*</td> <td class="description-td"> </td> <td class="description-td">T75</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">R81*</td> <td class="description-td"> </td> <td class="description-td">Q47*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q81*</td> </tr> <tr class="description-tr"> <td class="description-td">−3</td> <td class="description-td">Q72</td> <td class="description-td">E72</td> <td class="description-td"> <b>R72</b> </td> <td class="description-td">K72</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C26*</td> <td class="description-td">Y72</td> <td class="description-td">K72</td> <td class="description-td">Y72</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L26*</td> <td class="description-td">H26*</td> <td class="description-td"> <b>Y26*</b> </td> <td class="description-td">H26*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V26*</td> <td class="description-td">K26*</td> <td class="description-td">F26*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">A26*</td> <td class="description-td">R26*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">I26*</td> </tr> <tr class="description-tr"> <td class="description-td">−4</td> <td class="description-td"> <b>K28</b> </td> <td class="description-td"> <b>K28*</b> </td> <td class="description-td">R83</td> <td class="description-td"> <b>K28</b> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Q83</td> <td class="description-td">R28*</td> <td class="description-td">K83</td> <td class="description-td">K83</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">E83</td> <td class="description-td"> </td> <td class="description-td">Q28*</td> </tr> <tr class="description-tr"> <td class="description-td">−5</td> <td class="description-td"> <b>K28</b> </td> <td class="description-td"> <b>K28*</b> </td> <td class="description-td">R45</td> <td class="description-td">Q28*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C28*</td> <td class="description-td">R28*</td> <td class="description-td">E28*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L28*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">I28*</td> </tr> <tr class="description-tr"> <td class="description-td">−6</td> <td class="description-td">I30*</td> <td class="description-td">E43</td> <td class="description-td">R43</td> <td class="description-td">K43</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V30*</td> <td class="description-td">E85</td> <td class="description-td">K43</td> <td class="description-td"> <b>I85</b> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">S30*</td> <td class="description-td">K30*</td> <td class="description-td">K85</td> <td class="description-td">V85</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L30*</td> <td class="description-td">R30*</td> <td class="description-td">R85</td> <td class="description-td">L85</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Q43</td> <td class="description-td"> </td> <td class="description-td">E30*</td> <td class="description-td">Q30*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">D30*</td> </tr> <tr class="description-tr"> <td class="description-td">−7</td> <td class="description-td"> <b>Q41</b> </td> <td class="description-td">E32</td> <td class="description-td"> <b>R32</b> </td> <td class="description-td">K32</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">E41</td> <td class="description-td">R41</td> <td class="description-td">M41</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K41</td> <td class="description-td">L41</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">I41</td> </tr> <tr class="description-tr"> <td class="description-td">−8</td> <td class="description-td"> <b>Y35</b> </td> <td class="description-td">E32</td> <td class="description-td"> <b>R32</b> </td> <td class="description-td">K32</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">K35</td> <td class="description-td"> </td> <td class="description-td">K32</td> <td class="description-td">K35</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K35</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">R35</td> </tr> <tr class="description-tr"> <td class="description-td">−9</td> <td class="description-td">N34</td> <td class="description-td"> <b>D34</b> </td> <td class="description-td">K34</td> <td class="description-td">S34</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">H34</td> <td class="description-td">E34</td> <td class="description-td">R34</td> <td class="description-td">C34</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">S34</td> <td class="description-td">H34</td> <td class="description-td">V34</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">T34</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">A34</td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00003" class="description-td" colspan="5">Bold entries are represent wild-type contact residues and do not constitute “modifications” as used herein.</td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00004" class="description-td" colspan="5">An asterisk indicates that the residue contacts the base on the antisense strand.</td> </tr> </tbody> </tgroup> </table> </tables> <br>
2.2.3 Meganucleases Derived from I-SceI
</p>
    <p num="p-0119">In another aspect, the present invention relates to rationally-designed meganucleases which are based upon or derived from the I-SceI meganuclease of <i>Saccharomyces cerevisiae</i>. The wild-type amino acid sequence of the I-SceI meganuclease is shown in SEQ ID NO: 9, which corresponds to Genbank Accession # CAA09843. The recognition sequence of the wild-type I-SceI meganuclease from crystal structure PDB # 1R7M is shown below:</p>
    <p num="p-0120"> <tables id="TABLE-US-00008" num="00008"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="259pt" align="right"> </colspec> <colspec colname="2" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SEQ ID NO: 10</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Sense</td> <td class="description-td">5′-T T A C C C T G T T A T C C C T A G-3′</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="259pt" align="right"> </colspec> <colspec colname="2" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SEQ ID NO: 11</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Antisense</td> <td class="description-td">3′-A A T G G G A C A  A  T  A  C  G  G  A  T  C-5′</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Position</td> <td class="description-td">   1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Note that the recognition sequence is non-palindromic and there are not four base pairs separating half-sites.
</p>
    <p num="p-0121">In accordance with the invention, a substantial number of amino acid modifications to the DNA recognition domain of the I-SceI meganuclease have been identified which, singly or in combination, can result in recombinant meganucleases with specificities altered at individual bases within the DNA recognition sequence, such that these rationally-designed meganucleases have recognition sequences different from the wild-type enzyme. The amino acid modifications of I-SceI and the predicted change in recognition sequence specificity are shown in Table 3:</p>
    <p num="p-0122"> <tables id="TABLE-US-00009" num="00009"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="154pt" align="center"> </colspec> <colspec colname="2" colwidth="7pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 3</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Favored Sense-Strand Base</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Position</td> <td class="description-td">A</td> <td class="description-td">C</td> <td class="description-td">G</td> <td class="description-td">T</td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="56pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">4</td> <td class="description-td">K50</td> <td class="description-td"> <b>R50*</b> </td> <td class="description-td">E50*</td> <td class="description-td">K57</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K50*</td> <td class="description-td">R57</td> <td class="description-td">M57</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">E57</td> <td class="description-td">K57</td> <td class="description-td">Q50*</td> </tr> <tr class="description-tr"> <td class="description-td">5</td> <td class="description-td">K48</td> <td class="description-td"> <b>R48*</b> </td> <td class="description-td">E48*</td> <td class="description-td">Q48*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> <b>Q102</b> </td> <td class="description-td">K48*</td> <td class="description-td">K102</td> <td class="description-td">C102</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">E102</td> <td class="description-td">R102</td> <td class="description-td">L102</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">E59</td> <td class="description-td"> </td> <td class="description-td">V102</td> </tr> <tr class="description-tr"> <td class="description-td">6</td> <td class="description-td">K59</td> <td class="description-td">R59*</td> <td class="description-td">K84</td> <td class="description-td"> <b>Q59*</b> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K59*</td> <td class="description-td">E59*</td> <td class="description-td">Y46</td> </tr> <tr class="description-tr"> <td class="description-td">7</td> <td class="description-td">C46*</td> <td class="description-td">R46*</td> <td class="description-td"> <b>K86</b> </td> <td class="description-td">K68</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L46*</td> <td class="description-td">K46*</td> <td class="description-td">R86</td> <td class="description-td">C86</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V46*</td> <td class="description-td">E86</td> <td class="description-td">E46*</td> <td class="description-td">L86</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q46*</td> </tr> <tr class="description-tr"> <td class="description-td">8</td> <td class="description-td">K61*</td> <td class="description-td">E88</td> <td class="description-td"> <b>E61*</b> </td> <td class="description-td">K88</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">S61*</td> <td class="description-td">R61*</td> <td class="description-td"> <b>R88</b> </td> <td class="description-td">Q61*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V61*</td> <td class="description-td">H61*</td> <td class="description-td">K88</td> <td class="description-td">H61*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">A61*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L61*</td> </tr> <tr class="description-tr"> <td class="description-td">9</td> <td class="description-td"> <b>T98*</b> </td> <td class="description-td">R98*</td> <td class="description-td">E98*</td> <td class="description-td">Q98*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C98*</td> <td class="description-td">K98*</td> <td class="description-td">D98*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V98*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L98*</td> </tr> <tr class="description-tr"> <td class="description-td">10</td> <td class="description-td"> <b>V96*</b> </td> <td class="description-td">K96*</td> <td class="description-td">D96*</td> <td class="description-td">Q96*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C96*</td> <td class="description-td">R96*</td> <td class="description-td">E96*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">A96*</td> </tr> <tr class="description-tr"> <td class="description-td">11</td> <td class="description-td">C90*</td> <td class="description-td">K90*</td> <td class="description-td">E90*</td> <td class="description-td">Q90*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L90*</td> <td class="description-td">R90*</td> </tr> <tr class="description-tr"> <td class="description-td">12</td> <td class="description-td">Q193</td> <td class="description-td">E165</td> <td class="description-td">K165</td> <td class="description-td">C165</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">E193</td> <td class="description-td">R165</td> <td class="description-td">L165</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">D193</td> <td class="description-td"> </td> <td class="description-td">C193</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">V193</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">A193</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">T193</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">S193</td> </tr> <tr class="description-tr"> <td class="description-td">13</td> <td class="description-td">C193*</td> <td class="description-td"> <b>K193*</b> </td> <td class="description-td">E193*</td> <td class="description-td">Q193*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L193*</td> <td class="description-td">R193*</td> <td class="description-td">D193*</td> <td class="description-td">C163</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">D192</td> <td class="description-td">K163</td> <td class="description-td">L163</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">R192</td> </tr> <tr class="description-tr"> <td class="description-td">14</td> <td class="description-td">L192*</td> <td class="description-td">E161</td> <td class="description-td">K147</td> <td class="description-td">K161</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C192*</td> <td class="description-td">R192*</td> <td class="description-td">K161</td> <td class="description-td">Q192*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K192*</td> <td class="description-td">R161</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">R197</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">D192*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">E192*</td> </tr> <tr class="description-tr"> <td class="description-td">15</td> <td class="description-td"> </td> <td class="description-td">E151</td> <td class="description-td">K151</td> <td class="description-td">C151</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">L151</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K151</td> </tr> <tr class="description-tr"> <td class="description-td">17</td> <td class="description-td"> <b>N152*</b> </td> <td class="description-td">K152*</td> <td class="description-td">N152*</td> <td class="description-td">Q152*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">S152*</td> <td class="description-td">K150*</td> <td class="description-td">S152*</td> <td class="description-td">Q150*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C150*</td> <td class="description-td"> </td> <td class="description-td">D152*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L150*</td> <td class="description-td"> </td> <td class="description-td"> <b>D150*</b> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V150*</td> <td class="description-td"> </td> <td class="description-td">E150*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">T150*</td> </tr> <tr class="description-tr"> <td class="description-td">18</td> <td class="description-td">K155*</td> <td class="description-td">R155*</td> <td class="description-td">E155*</td> <td class="description-td">H155*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C155*</td> <td class="description-td">K155*</td> <td class="description-td"> </td> <td class="description-td">Y155*</td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00005" class="description-td" colspan="5">Bold entries are wild-type contact residues and do not constitute “modifications” as used herein.</td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00006" class="description-td" colspan="5">An asterisk indicates that the residue contacts the base on the antisense strand.</td> </tr> </tbody> </tgroup> </table> </tables> <br>
2.2.4 Meganucleases Derived from I-CeuI
</p>
    <p num="p-0123">In another aspect, the present invention relates to rationally-designed meganucleases which are based upon or derived from the I-CeuI meganuclease of <i>Chlamydomonas eugametos</i>. The wild-type amino acid sequence of the I-CeuI meganuclease is shown in SEQ ID NO: 12, which corresponds to Genbank Accession # P32761. Two recognition sequence half sites of the wild-type I-CeuI meganuclease from crystal structure PDB # 2EX5 are shown below:</p>
    <p num="p-0124"> <tables id="TABLE-US-00010" num="00010"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Position</td> <td class="description-td">  -9-8-7-6-5-4-3-2-1</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="259pt" align="right"> </colspec> <colspec colname="2" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SEQ ID NO: 13</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">5′-A T A A C G G T C C T A A G G T A G C G A A-3′</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="259pt" align="right"> </colspec> <colspec colname="2" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SEQ ID NO: 14</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="217pt" align="left"> </colspec> <colspec colname="3" colwidth="182pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">3′-T A T T G C C A G G A T T C C A T C G C T T-5′</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Position</td> <td class="description-td">                            -1-2-3-4-5-6-7-8-9</td> </tr> </tbody> </tgroup> </table> </tables> <br>
Note that the recognition sequence is non-palindromic, even outside the central four base pairs, despite the fact that I-CeuI is a homodimer, due to the natural degeneracy in the I-CeuI recognition interface (Spiegel et al. (2006), <i>Structure </i>14:869-80). The two recognition sequence half-sites are shown in bold on their respective sense strands.
</p>
    <p num="p-0125">In accordance with the invention, a substantial number of amino acid modifications to the DNA recognition domain of the I-CeuI meganuclease have been identified which, singly or in combination, result in recombinant meganucleases with specificities altered at individual bases within the DNA recognition sequence half-site, such that these rationally-designed meganucleases can have recognition sequences different from the wild-type enzyme. The amino acid modifications of I-CeuI and the predicted change in recognition sequence specificity are shown in Table 4:</p>
    <p num="p-0126"> <tables id="TABLE-US-00011" num="00011"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="147pt" align="center"> </colspec> <colspec colname="2" colwidth="14pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 4</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Favored Sense-Strand Base</td> <td class="description-td"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="left"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <colspec colname="5" colwidth="42pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Position</td> <td class="description-td">A</td> <td class="description-td">C</td> <td class="description-td">G</td> <td class="description-td">T</td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td> </tr> <tr class="description-tr"> <td class="description-td">−1</td> <td class="description-td">C92*</td> <td class="description-td"> <b>K116*</b> </td> <td class="description-td">E116*</td> <td class="description-td">Q116*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">A92*</td> <td class="description-td">R116*</td> <td class="description-td">E92*</td> <td class="description-td">Q92*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V92*</td> <td class="description-td">D116*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K92*</td> </tr> <tr class="description-tr"> <td class="description-td">−2</td> <td class="description-td">Q117</td> <td class="description-td">E117</td> <td class="description-td">K117</td> <td class="description-td">C117</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C90*</td> <td class="description-td">D117</td> <td class="description-td">R124</td> <td class="description-td">V117</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L90*</td> <td class="description-td">R174*</td> <td class="description-td">K124</td> <td class="description-td">T117</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V90*</td> <td class="description-td">K124*</td> <td class="description-td">E124*</td> <td class="description-td">Q90*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K90*</td> <td class="description-td">E90*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">R90*</td> <td class="description-td">D90*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K68*</td> </tr> <tr class="description-tr"> <td class="description-td">−3</td> <td class="description-td">C70*</td> <td class="description-td">K70*</td> <td class="description-td">E70*</td> <td class="description-td">Q70*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V70*</td> <td class="description-td"> </td> <td class="description-td"> <b>E88*</b> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">T70*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L70*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">K70*</td> </tr> <tr class="description-tr"> <td class="description-td">−4</td> <td class="description-td">Q126</td> <td class="description-td">E126</td> <td class="description-td">R126</td> <td class="description-td">K126</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">N126</td> <td class="description-td">D126</td> <td class="description-td">K126</td> <td class="description-td">L126</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">K88*</td> <td class="description-td">R88*</td> <td class="description-td"> <b>E88*</b> </td> <td class="description-td">Q88*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L88*</td> <td class="description-td">K88*</td> <td class="description-td">D88*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C88*</td> <td class="description-td">K72*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C72*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L72*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V72*</td> </tr> <tr class="description-tr"> <td class="description-td">−5</td> <td class="description-td">C74*</td> <td class="description-td"> <b>K74*</b> </td> <td class="description-td">E74*</td> <td class="description-td">C128</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L74*</td> <td class="description-td"> </td> <td class="description-td"> <b>K128</b> </td> <td class="description-td">L128</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V74*</td> <td class="description-td"> </td> <td class="description-td">R128</td> <td class="description-td">V128</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">T74*</td> <td class="description-td"> </td> <td class="description-td">E128</td> <td class="description-td">T128</td> </tr> <tr class="description-tr"> <td class="description-td">−6</td> <td class="description-td">Q86</td> <td class="description-td"> <b>D86</b> </td> <td class="description-td"> <b>K128</b> </td> <td class="description-td">K86</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">E86</td> <td class="description-td">R128</td> <td class="description-td">C86</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">R84*</td> <td class="description-td">R86</td> <td class="description-td">L86</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">K84*</td> <td class="description-td">K86</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">E84*</td> </tr> <tr class="description-tr"> <td class="description-td">−7</td> <td class="description-td"> <b>L76*</b> </td> <td class="description-td">R76*</td> <td class="description-td">E76*</td> <td class="description-td">H76*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">C76*</td> <td class="description-td">K76*</td> <td class="description-td">R84</td> <td class="description-td">Q76*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">K76*</td> <td class="description-td">H76*</td> </tr> <tr class="description-tr"> <td class="description-td">−8</td> <td class="description-td">Y79</td> <td class="description-td">D79</td> <td class="description-td">R79</td> <td class="description-td">C79</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">R79</td> <td class="description-td">E79</td> <td class="description-td">K79</td> <td class="description-td">L79</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Q76</td> <td class="description-td">D76</td> <td class="description-td">K76</td> <td class="description-td">V79</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">E76</td> <td class="description-td">R76</td> <td class="description-td">L76</td> </tr> <tr class="description-tr"> <td class="description-td">−9</td> <td class="description-td">Q78</td> <td class="description-td">D78</td> <td class="description-td">R78</td> <td class="description-td">K78</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">N78</td> <td class="description-td">E78</td> <td class="description-td">K78</td> <td class="description-td">V78</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">H78</td> <td class="description-td"> </td> <td class="description-td">H78</td> <td class="description-td">L78</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">K78</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">C78</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">T78</td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00007" class="description-td" colspan="5">Bold entries are wild-type contact residues and do not constitute “modifications” as used herein.</td> </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" id="FOO-00008" class="description-td" colspan="5">An asterisk indicates that the residue contacts the base on the antisense strand.</td> </tr> </tbody> </tgroup> </table> </tables> <br>
2.2.5 Specifically-Excluded Recombinant Meganucleases
</p>
    <p num="p-0127">The present invention is not intended to embrace certain recombinant meganucleases which have been described in the prior art, and which have been developed by alternative methods. These excluded meganucleases include those described by Arnould et al. (2006), <i>J. Mol. Biol. </i>355: 443-58; Sussman et al. (2004), <i>J. Mol. Biol. </i>342: 31-41; Chames et al. (2005), <i>Nucleic Acids Res. </i>33: e178; Seligman et al. (2002), <i>Nucleic Acids Res. </i>30: 3870-9; and Ashworth et al. (2006), <i>Nature </i>441 (7093):656-659; the entire disclosures of which are hereby incorporated by reference, including recombinant meganucleases based on I-CreI with single substitutions selected from C33, R33, A44, H33, K32, F33, R32, A28, A70, E33, V33, A26, and R66. Also excluded are recombinant meganucleases based on I-CreI with three substitutions selected from A68/N70/N75 and D44/D70/N75, or with four substitutions selected from K44/T68/G60/N75 and R44/A68/T70/N75. Lastly, specifically excluded is the recombinant meganuclease based on I-MsoI with the pair of substitutions L28 and R83. These substitutions or combinations of substitutions are referred to herein as the “excluded modifications.”</p>
    <p num="h-0013">2.2.6 Meganucleases with Multiple Changes in the Recognition Sequence Half-Site</p>
    <p num="p-0128">In another aspect, the present invention relates to rationally-designed meganucleases which are produced by combining two or more amino acid modifications as described in sections 2.2.1-2.2.4 above, in order to alter half-site preference at two or more positions in a DNA recognition sequence half-site. For example, without limitation, and as more fully described below, the enzyme DJ1 was derived from I-CreI by incorporating the modifications R30/E38 (which favor C at position −7), R40 (which favors G at position −6), R42 (which favors at G at position −5), and N32 (which favors complete degeneracy at position −9). The rationally-designed DJ1 meganuclease invariantly recognizes C<sub>−7 </sub>G<sub>−6 </sub>G<sub>−5 </sub>compared to the wild-type preference for A<sub>−7 </sub>A<sub>−6 </sub>C<sub>−5</sub>, and has increased tolerance for A at position −9.</p>
    <p num="p-0129">The ability to combine residue substitutions that affect different base positions is due in part to the modular nature of the LAGLIDADG meganucleases. A majority of the base contacts in the LAGLIDADG (SEQ ID NO: 37) recognition interfaces are made by individual amino acid side chains, and the interface is relatively free of interconnectivity or hydrogen bonding networks between side chains that interact with adjacent bases. This generally allows manipulation of residues that interact with one base position without affecting side chain interactions at adjacent bases. The additive nature of the mutations listed in sections 2.2.1-2.2.4 above is also a direct result of the method used to identify these mutations. The method predicts side chain substitutions that interact directly with a single base. Interconnectivity or hydrogen bonding networks between side chains is generally avoided to maintain the independence of the substitutions within the recognition interface.</p>
    <p num="p-0130">Certain combinations of side chain substitutions are completely or partially incompatible with one another. When an incompatible pair or set of amino acids are incorporated into a rationally-designed meganuclease, the resulting enzyme will have reduced or eliminated catalytic activity. Typically, these incompatibilities are due to steric interference between the side chains of the introduced amino acids and activity can be restored by identifying and removing this interference. Specifically, when two amino acids with large side chains (e.g., amino acids from group 2 or 3) are incorporated at amino acid positions that are adjacent to one another in the meganuclease structure (e.g., positions 32 and 33, 28 and 40, 28 and 42, 42 and 77, or 68 and 77 in the case of meganucleases derived from I-CreI), it is likely that these two amino acids will interfere with one another and reduce enzyme activity. This interference be eliminated by substituting one or both incompatible amino acids to an amino acid with a smaller side chain (e.g., group 1 or group 2). For example, in rationally-designed meganucleases derived from I-CreI, K28 interferes with both R40 and R42. To maximize enzyme activity, R40 and R42 can be combined with a serine or aspartic acid at position 28.</p>
    <p num="p-0131">Combinations of amino substitutions, identified as described herein, can be used to rationally alter the specificity of a wild-type meganuclease (or a previously modified meganuclease) from an original recognition sequence to a desired recognition sequence which may be present in a nucleic acid of interest (e.g., a genome). <figref idrefs="DRAWINGS">FIG. 2A</figref>, for example, shows the “sense” strand of the I-CreI meganuclease recognition sequence WT (SEQ ID NO: 4) as well as a number of other sequences for which a rationally-designed meganuclease would be useful. Conserved bases between the WT recognition sequence and the desired recognition sequence are shaded. In accordance with the invention, recombinant meganucleases based on the I-CreI meganuclease can be rationally-designed for each of these desired recognition sequences, as well as any others, by suitable amino acid substitutions as described herein.</p>
    <p num="h-0014">3. Rationally-Designed Meganucleases with Altered DNA-Binding Affinity</p>
    <p num="p-0132">As described above, the DNA-binding affinity of the recombinant meganucleases of the invention can be modulated by altering certain amino acids that form the contact surface with the phosphodiester backbone of DNA. The contact surface comprises those amino acids in the enzyme with β-carbons less than 9 Å from the DNA backbone, and with side chains oriented toward the DNA, irrespective of whether the residues make contacts with the DNA backbone in the wild-type meganuclease-DNA complex. Because DNA-binding is a necessary precursor to enzyme activity, increases/decreases in DNA-binding affinity have been shown to cause increases/decreases, respectively, in enzyme activity. However, increases/decreases in DNA-binding affinity also have been shown to cause decreases/increases in the meganuclease sequence-specificity. Therefore, both activity and specificity can be modulated by modifying the phosphodiester backbone contacts.</p>
    <p num="p-0133">Specifically, to increase enzyme activity/decrease enzyme specificity:</p>
    <p num="p-0134">(i) Remove electrostatic repulsion between the enzyme and DNA backbone. If an identified amino acid has a negatively-charged side chain (e.g., aspartic acid, glutamic acid) which would be expected to repulse the negatively-charged DNA backbone, the repulsion can be eliminated by substituting an amino acid with an uncharged or positively-charged side chain, subject to effects of steric interference. An experimentally verified example is the mutation of glutamic acid 80 in I-CreI to glutamine.</p>
    <p num="p-0135">(ii) Introduce electrostatic attraction interaction between the enzyme and the DNA backbone. At any of the positions of the contact surface, the introduction of an amino acid with a positively-charged side chain (e.g., lysine or arginine) is expected to increase binding affinity, subject to effects of steric interference.</p>
    <p num="p-0136">(iii) Introduce a hydrogen-bond between the enzyme and the DNA backbone. If an amino acid of the contact surface does not make a hydrogen bond with the DNA backbone because it lacks an appropriate hydrogen-bonding functionality or has a side chain that is too short, too long, and/or too inflexible to interact with the DNA backbone, a polar amino acid capable of donating a hydrogen bond (e.g., serine, threonine, tyrosine, histidine, glutamine, asparagine, lysine, cysteine, or arginine) with the appropriate length and flexibility can be introduced, subject to effects of steric interference.</p>
    <p num="p-0137">Specifically, to decrease enzyme activity/increase enzyme specificity:</p>
    <p num="p-0138">(i) Introduce electrostatic repulsion between the enzyme and the DNA backbone. At any of the positions of the contact surface, the introduction of an amino acid with a negatively-charged side chain (e.g., glutamic acid, aspartic acid) is expected to decrease binding affinity, subject to effects of steric interference.</p>
    <p num="p-0139">(ii) Remove electrostatic attraction between the enzyme and DNA. If any amino acid of the contact surface has a positively-charged side chain (e.g., lysine or arginine) that interacts with the negatively-charged DNA backbone, this favorable interaction can be eliminated by substituting an amino acid with an uncharged or negatively-charged side chain, subject to effects of steric interference. An experimentally verified example is the mutation of lysine 116 in I-CreI to aspartic acid.</p>
    <p num="p-0140">(iii) Remove a hydrogen-bond between the enzyme and the DNA backbone. If any amino acid of the contact surface makes a hydrogen bond with the DNA backbone, it can be substituted to an amino acid that would not be expected to make a similar hydrogen bond because its side chain is not appropriately functionalized or it lacks the necessary length/flexibility characteristics.</p>
    <p num="p-0141">For example, in some recombinant meganucleases based on I-CreI, the glutamic acid at position 80 in the I-CreI meganuclease is altered to either a lysine or a glutamine to increase activity. In another embodiment, the tyrosine at position 66 of I-CreI is changed to arginine or lysine, which increases the activity of the meganuclease. In yet another embodiment, enzyme activity is decreased by changing the lysine at position 34 of I-CreI to aspartic acid, changing the tyrosine at position 66 to aspartic acid, and/or changing the lysine at position 116 to aspartic acid.</p>
    <p num="p-0142">The activities of the recombinant meganucleases can be modulated such that the recombinant enzyme has anywhere from no activity to very high activity with respect to a particular recognition sequence. For example, the DJ1 recombinant meganuclease when carrying glutamic acid mutation at position 26 loses activity completely. However, the combination of the glutamic acid substitution at position 26 and a glutamine substitution at position 80 creates a recombinant meganuclease with high specificity and activity toward a guanine at −4 within the recognition sequence half-site (see <figref idrefs="DRAWINGS">FIG. 1(D)</figref>).</p>
    <p num="p-0143">In accordance with the invention, amino acids at various positions in proximity to the phosphodiester DNA backbone can be changed to simultaneously affect both meganuclease activity and specificity. This “tuning” of the enzyme specificity and activity is accomplished by increasing or decreasing the number of contacts made by amino acids with the phosphodiester backbone. A variety of contacts with the phosphodiester backbone can be facilitated by amino acid side chains. In some embodiments, ionic bonds, salt bridges, hydrogen bonds, and steric hindrance affect the association of amino acid side chains with the phosphodiester backbone. For example, for the I-CreI meganuclease, alteration of the lysine at position 116 to an aspartic acid removes a salt bridge between nucleic acid base pairs at positions −8 and −9, reducing the rate of enzyme cleavage but increasing the specificity.</p>
    <p num="p-0144">The residues forming the backbone contact surface of each of the wild-type I-CreI (SEQ ID NO: 1), I-MsoI (SEQ ID NO: 6), I-SceI (SEQ ID NO: 9) and I-CeuI (SEQ ID NO: 12) meganucleases are identified in Table 5 below:</p>
    <p num="p-0145">
      <tables id="TABLE-US-00012" num="00012"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="70pt" align="left"> </colspec> <colspec colname="4" colwidth="70pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="4" rowsep="1" class="description-td" colspan="4">TABLE 5</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">I-CreI</td>
                <td class="description-td">I-MsoI</td>
                <td class="description-td">I-SceI</td>
                <td class="description-td">I-CeuI</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">P29, K34, T46, K48,</td>
                <td class="description-td">K36, Q41, R51, N70,</td>
                <td class="description-td">N15, N17, L19, K20,</td>
                <td class="description-td">K21, D25, K28, K31,</td>
              </tr> <tr class="description-tr"> <td class="description-td">R51, V64, Y66, E80,</td>
                <td class="description-td">I85, G86, S87, T88,</td>
                <td class="description-td">K23, K63, L80, S81,</td>
                <td class="description-td">S68, N70, H94, R112,</td>
              </tr> <tr class="description-tr"> <td class="description-td">I81, K82, L112,</td>
                <td class="description-td">H89, Y118, Q122,</td>
                <td class="description-td">H84, L92, N94, N120,</td>
                <td class="description-td">R114, S117, N120,</td>
              </tr> <tr class="description-tr"> <td class="description-td">K116, D137, K139,</td>
                <td class="description-td">K123, Q139, K143,</td>
                <td class="description-td">K122, K148, Y151,</td>
                <td class="description-td">D128, N129, R130,</td>
              </tr> <tr class="description-tr"> <td class="description-td">T140, T143</td>
                <td class="description-td">R144, E147, S150,</td>
                <td class="description-td">K153, T156, N157,</td>
                <td class="description-td">H172</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">N152</td>
                <td class="description-td">S159, N163, Q165,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">S166, Y188, K190,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">I191, K193, N194,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">K195, Y199, D201,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">S202, Y222, K223</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0146">To increase the affinity of an enzyme and thereby make it more active/less specific:
</p> <ul> <li id="ul0001-0001" num="0146">(1) Select an amino acid from Table 5 for the corresponding enzyme that is either negatively-charged (D or E), hydrophobic (A, C, F, G, I, L, M, P, V, W, Y), or uncharged/polar (H, N, Q, S, T).</li> <li id="ul0001-0002" num="0147">(2) If the amino acid is negatively-charged or hydrophobic, mutate it to uncharged/polar (less effect) or positively-charged (K or R, more effect).</li> <li id="ul0001-0003" num="0148">(3) If the amino acid is uncharged/polar, mutate it to positively-charged.</li> </ul> <p num="p-0147">To decrease the affinity of an enzyme and thereby make it less active/more specific:
</p> <ul> <li id="ul0002-0001" num="0150">(1) Select an amino acid from Table 5 for the corresponding enzyme that is either positively-charged (K or R), hydrophobic (A, C, F, G, I, L, M, P, V, W, Y), or uncharged/polar (H, N, Q, S, T).</li> <li id="ul0002-0002" num="0151">(2) If the amino acid is positively-charged, mutate it to uncharged/polar (less effect) or negatively-charged (more effect).</li> <li id="ul0002-0003" num="0152">(3) If the amino acid is hydrophobic or uncharged/polar, mutate it to negatively-charged.
<br>
4. Heterodimeric Meganucleases
</li> </ul> <p num="p-0148">In another aspect, the invention provides meganucleases which are heterodimers formed by the association of two monomers, one of which may be a wild-type and one or both of which may be a non-naturally-occurring or recombinant form. For example, wild-type I-CreI meganuclease is normally a homodimer composed of two monomers that each bind to one half-site in the pseudo-palindromic recognition sequence. A heterodimeric recombinant meganuclease can be produced by combining two meganucleases that recognize different half-sites, for example by co-expressing the two meganucleases in a cell or by mixing two meganucleases in solution. The formation of heterodimers can be favored over the formation of homodimers by altering amino acids on each of the two monomers that affect their association into dimers. In particular embodiments, certain amino acids at the interface of the two monomers are altered from negatively-charged amino acids (D or E) to positively-charged amino acids (K or R) on a first monomer and from positively-charged amino acids to negatively-charged amino acids on a second monomer (Table 6). For example, in the case of meganucleases derived from I-CreI, lysines at positions 7 and 57 are mutated to glutamic acids in the first monomer and glutamic acids at positions 8 and 61 are mutated to lysines in the second monomer. The result of this process is a pair of monomers in which the first monomer has an excess of positively-charged residues at the dimer interface and the second monomer has an excess of negatively-charged residues at the dimer interface. The first and second monomer will, therefore, associate preferentially over their identical monomer pairs due to the electrostatic interactions between the altered amino acids at the interface.</p>
    <p num="p-0149">
      <tables id="TABLE-US-00013" num="00013"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="105pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">I-CreI: First Monomer</td>
                <td class="description-td">I-CreI: Second Monomer</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Substitutions</td>
                <td class="description-td">Substitutions</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">K7 to E7 or D7</td>
                <td class="description-td">E8 to K8 or R8</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">K57 to E57 or D57</td>
                <td class="description-td">E61 to K61 or R61</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">K96 to E96 or D96</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">I-MsoI: First Monomer</td>
                <td class="description-td">I-MsoI: Second Monomer</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Substitutions</td>
                <td class="description-td">Substitutions</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">R302 to E302 or D302</td>
                <td class="description-td">D20 to K60 or R60</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">E11 to K11 or R11</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Q64 to K64 or R64</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">I-CeuI: First Monomer</td>
                <td class="description-td">I-CeuI: Second Monomer</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Substitutions</td>
                <td class="description-td">Substitutions</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">R93 to E93 or D93</td>
                <td class="description-td">E152 to K152 or R152</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0150">Alternatively, or in addition, certain amino acids at the interface of the two monomers can be altered to sterically hinder homodimer formation. Specifically, amino acids in the dimer interface of one monomer are substituted with larger or bulkier residues that will sterically prevent the homodimer. Amino acids in the dimer interface of the second monomer optionally can be substituted with smaller residues to compensate for the bulkier residues in the first monomer and remove any clashes in the heterodimer, or can be unmodified.</p>
    <p num="p-0151">In another alternative or additional embodiment, an ionic bridge or hydrogen bond can be buried in the hydrophobic core of a heterodimeric interface. Specifically, a hydrophobic residue on one monomer at the core of the interface can be substituted with a positively charged residue. In addition, a hydrophobic residue on the second monomer, that interacts in the wild type homodimer with the hydrophobic residue substituted in the first monomer, can be substituted with a negatively charged residue. Thus, the two substituted residues can form an ionic bridge or hydrogen bond. At the same time, the electrostatic repulsion of an unsatisfied charge buried in a hydrophobic interface should disfavor homodimer formation.</p>
    <p num="p-0152">Finally, as noted above, each monomer of the heterodimer can have different amino acids substituted in the DNA recognition region such that each has a different DNA half-site and the combined dimeric DNA recognition sequence is non-palindromic.</p>
    <p num="h-0015">5. Methods of Producing Recombinant Cells and Organisms</p>
    <p num="p-0153">Aspects of the present invention further provide methods for producing recombinant, transgenic or otherwise genetically-modified cells and organisms using rationally-designed meganucleases. Thus, in certain embodiments, recombinant meganucleases are developed to specifically cause a double-stranded break at a single site or at relatively few sites in the genomic DNA of a cell or an organism to allow for precise insertion(s) of a sequence of interest by homologous recombination. In other embodiments, recombinant meganucleases are developed to specifically cause a double-stranded break at a single site or at relatively few sites in the genomic DNA of a cell or an organism to either (a) allow for rare insertion(s) of a sequence of interest by non-homologous end-joining or (b) allow for the disruption of the target sequence by non-homologous end-joining. As used herein with respect to homologous recombination or non-homologous end-joining of sequences of interest, the term “insertion” means the ligation of a sequence of interest into a chromosome such that the sequence of interest is integrated into the chromosome. In the case of homologous recombination, an inserted sequence can replace an endogenous sequence, such that the original DNA is replaced by exogenous DNA of equal length, but with an altered nucleotide sequence. Alternatively, an inserted sequence can include more or fewer bases than the sequence it replaces.</p>
    <p num="p-0154">Therefore, in accordance with this aspect of the invention, the recombinant organisms include, but are not limited to, monocot plant species such as rice, wheat, corn (maize) and rye, and dicot species such as legumes (e.g., kidney beans, soybeans, lentils, peanuts, peas), alfalfa, clover, tobacco and <i>Arabidopsis </i>species. In addition, the recombinant organisms can include, but are not limited to, animals such as humans and non-human primates, horses, cows, goats, pigs, sheep, dogs, cats, guinea pigs, rats, mice, lizards, fish and insects such as <i>Drosophila </i>species. In other embodiments, the organism is a fungus such as a <i>Candida, Neurospora </i>or <i>Saccharomyces </i>species.</p>
    <p num="p-0155">In some embodiments, the methods of the invention involve the introduction of a sequence of interest into a cell such as a germ cell or stem cell that can become a mature recombinant organism or allow the resultant genetically-modified organism to give rise to progeny carrying the inserted sequence of interest in its genome.</p>
    <p num="p-0156">Meganuclease proteins can be delivered into cells to cleave genomic DNA, which allows for homologous recombination or non-homologous end-joining at the cleavage site with a sequence of interest, by a variety of different mechanisms known in the art. For example, the recombinant meganuclease protein can introduced into a cell by techniques including, but not limited to, microinjection or liposome transfections (see, e.g., Lipofectamine™, Invitrogen Corp., Carlsbad, Calif.). The liposome formulation can be used to facilitate lipid bilayer fusion with a target cell, thereby allowing the contents of the liposome or proteins associated with its surface to be brought into the cell. Alternatively, the enzyme can be fused to an appropriate uptake peptide such as that from the HIV TAT protein to direct cellular uptake (see, e.g., Hudecz et al. (2005), <i>Med. Res. Rev. </i>25: 679-736).</p>
    <p num="p-0157">Alternatively, gene sequences encoding the meganuclease protein are inserted into a vector and transfected into a eukaryotic cell using techniques known in the art (see, e.g., Ausubel et. al., <i>Current Protocols in Molecular Biology</i>, Wiley 1999). The sequence of interest can be introduced in the same vector, a different vector, or by other means known in the art.</p>
    <p num="p-0158">Non-limiting examples of vectors for DNA transfection include virus vectors, plasmids, cosmids, and YAC vectors. Transfection of DNA sequences can be accomplished by a variety of methods known to those of skill in the art. For instance, liposomes and immunoliposomes are used to deliver DNA sequences to cells (see, e.g., Lasic et al. (1995), <i>Science </i>267: 1275-76). In addition, viruses can be utilized to introduce vectors into cells (see, e.g., U.S. Pat. No. 7,037,492). Alternatively, transfection strategies can be utilized such that the vectors are introduced as naked DNA (see, e.g., Rui et al. (2002), <i>Life Sci. </i>71(15): 1771-8).</p>
    <p num="p-0159">General methods for delivering nucleic acids into cells include: (1) chemical methods (Graham et al. (1973), <i>Virology </i>54(2):536-539; Zatloukal et al. (1992), <i>Ann. N.Y. Acad. Sci., </i>660:136-153; (2) physical methods such as microinjection (Capecchi (1980), <i>Cell </i>22(2):479-488, electroporation (Wong et al. (1982), <i>Biochim. Biophys. Res. Commun. </i>107(2):584-587; Fromm et al. (1985), <i>Proc. Nat'l Acad. Sci. USA </i>82(17):5824-5828; U.S. Pat. No. 5,384,253) and ballistic injection (Johnston et al. (1994), <i>Methods Cell. Biol. </i>43(A): 353-365; Fynan et al. (1993), <i>Proc. Nat'l Acad. Sci. USA </i>90(24): 11478-11482); (3) viral vectors (Clapp (1993), <i>Clin. Perinatol. </i>20(1): 155-168; Lu et al. (1993), <i>J. Exp. Med. </i>178(6):2089-2096; Eglitis et al. (1988), <i>Avd. Exp. Med. Biol. </i>241:19-27; Eglitis et al. (1988), <i>Biotechniques </i>6(7):608-614); and (4) receptor-mediated mechanisms (Curiel et al. (1991), <i>Proc. Nat'l Acad. Sci. USA </i>88(19):8850-8854; Curiel et al. (1992), <i>Hum. Gen. Ther. </i>3(2):147-154; Wagner et al. (1992), <i>Proc. Nat'l Acad. Sci. USA </i>89 (13):6099-6103).</p>
    <p num="p-0160">In certain embodiments, a genetically-modified plant is produced, which contains the sequence of interest inserted into the genome. In certain embodiments, the genetically-modified plant is produced by transfecting the plant cell with DNA sequences corresponding to the recombinant meganuclease and the sequence of interest, which may or may not be flanked by the meganuclease recognition sequences and/or sequences substantially identical to the target sequence. In other embodiments, the genetically-modified plant is produced by transfecting the plant cell with DNA sequences corresponding to the recombinant meganuclease only, such that cleavage promotes non-homologous end-joining and disrupts the target sequence containing the recognition sequence. In such embodiments, the meganuclease sequences are under the control of regulatory sequences that allow for expression of the meganuclease in the host plant cells. These regulatory sequences include, but are not limited to, constitutive plant promoters such as the NOS promoter, chemically-inducible gene promoters such as the dexamethasone-inducible promoter (see, e.g., Gremillon et al. (2004), <i>Plant J. </i>37:218-228), and plant tissue specific promoters such as the LGC1 promoter (see, e.g., Singh et al. (2003), <i>FEBS Lett. </i>542:47-52).</p>
    <p num="p-0161">Suitable methods for introducing DNA into plant cells include virtually any method by which DNA can be introduced into a cell, including but not limited to <i>Agrobacterium </i>infection, PEG-mediated transformation of protoplasts (Omirulleh et al. (1993), <i>Plant Molecular Biology, </i>21:415-428), desiccation/inhibition-mediated DNA uptake, electroporation, agitation with silicon carbide fibers, ballistic injection or microprojectile bombardment, and the like.</p>
    <p num="p-0162">In other embodiments, a genetically-modified animal is produced using a recombinant meganuclease. As with plant cells, the nucleic acid sequences can be introduced into a germ cell or a cell that will eventually become a transgenic organism. In some embodiments, the cell is a fertilized egg, and exogenous DNA molecules can be injected into the pro-nucleus of the fertilized egg. The micro-injected eggs are then transferred into the oviducts of pseudopregnant foster mothers and allowed to develop. The recombinant meganuclease is expressed in the fertilized egg (e.g., under the control of a constitutive promoter, such as 3-phosphoglycerate kinase), and facilitates homologous recombination of the sequence of interest into one or a few discrete sites in the genome. Alternatively, the genetically-modified animals can be obtained by utilizing recombinant embryonic stem (“ES”) cells for the generation of the transgenics, as described by Gossler et al. (1986), <i>Proc. Natl. Acad. Sci. USA </i>83:9065 9069.</p>
    <p num="p-0163">In certain embodiments, a recombinant mammalian expression vector is capable of directing tissue-specific expression of the nucleic acid preferentially in a particular cell type. Tissue-specific regulatory elements are known in the art. Non-limiting examples of suitable tissue-specific promoters include the albumin promoter (liver-specific; Pinkert et al. (1987), <i>Genes Dev. </i>1: 268-277), lymphoid-specific promoters (Calame and Eaton (1988), <i>Adv. Immunol. </i>43: 235-275), in particular promoters of T cell receptors (Winoto and Baltimore (1989), <i>EMBO J. </i>8: 729-733) and immunoglobulins (Banerji et al. (1983), <i>Cell </i>33: 729-740; Queen and Baltimore (1983), <i>Cell </i>33: 741-748), neuron-specific promoters (e.g., the neurofilament promoter; Byrne and Ruddle (1989), <i>Proc. Natl. Acad. Sci. USA </i>86: 5473-5477), pancreas-specific promoters (Edlund et al. (1985), <i>Science </i>230: 912-916), and mammary gland-specific promoters (e.g., milk whey promoter; U.S. Pat. No. 4,873,316 and European Pat. Pub. EP 0 264 166). Developmentally-regulated promoters are also encompassed, e.g., the murine hox promoters (Kessel and Gruss (1990), <i>Science </i>249: 374-379) and the α-fetoprotein promoter (Campes and Tilghman (1989), <i>Genes Dev. </i>3: 537-546).</p>
    <p num="p-0164">In certain embodiments, a rationally-designed meganuclease may be tagged with a peptide epitope (e.g., an HA, FLAG, or Myc epitope) to monitor expression levels or localization. In some embodiments, the meganuclease may be fused to a sub-cellular localization signal such as a nuclear-localization signal (e.g., the nuclear localization signal from SV40) or chloroplast or mitochondrial localization signals. In other embodiments, the meganuclease may be fused to a nuclear export signal to localize it to the cytoplasm. The meganuclease may also be fused to an unrelated protein or protein domain such as a protein that stimulates DNA-repair or homologous recombination (e.g., recA, RAD51, RAD52, RAD54, RAD57 or BRCA2).</p>
    <p num="h-0016">6. Methods for Gene Therapy</p>
    <p num="p-0165">Aspects of the invention allow for the use of recombinant meganuclease for gene therapy. As used herein, “gene therapy” means therapeutic treatments that comprise introducing into a patient a functional copy of at least one gene, or gene regulatory sequence such as a promoter, enhancer, or silencer to replace a gene or gene regulatory region that is defective in its structure and/or function. The term “gene therapy” can also refer to modifications made to a deleterious gene or regulatory element (e.g., oncogenes) that reduce or eliminate expression of the gene. Gene therapy can be performed to treat congenital conditions, conditions resulting from mutations or damage to specific genetic loci over the life of the patient, or conditions resulting from infectious organisms.</p>
    <p num="p-0166">In some aspects of the invention, dysfunctional genes are replaced or disabled by the insertion of exogenous nucleic acid sequences into a region of the genome affecting gene expression. In certain embodiments, the recombinant meganuclease is targeted to a particular sequence in the region of the genome to be modified so as to alleviate the condition. The sequence can be a region within an exon, intron, promoter, or other regulatory region that is causing dysfunctional expression of the gene. As used herein, the term “dysfunctional expression” means aberrant expression of a gene product either by the cell producing too little of the gene product, too much of the gene product, or producing a gene product that has a different function such as lacking the necessary function or having more than the necessary function.</p>
    <p num="p-0167">Exogenous nucleic acid sequences inserted into the modified region can be used to provide “repaired” sequences that normalize the gene. Gene repair can be accomplished by the introduction of proper gene sequences into the gene allowing for proper function to be reestablished. In these embodiments, the nucleic acid sequence to be inserted can be the entire coding sequence for a protein or, in certain embodiments, a fragment of the gene comprising only the region to be repaired. In other embodiments the nucleic acid sequence to be inserted comprises a promoter sequence or other regulatory elements such that mutations causing abnormal expression or regulation are repaired. In other embodiments, the nucleic acid sequence to be inserted contains the appropriate translation stop codon lacking in a mutated gene. The nucleic acid sequence can also have sequences for stopping transcription in a recombinant gene lacking appropriate transcriptional stop signals.</p>
    <p num="p-0168">Alternatively, the nucleic acid sequences can eliminate gene function altogether by disrupting the regulatory sequence of the gene or providing a silencer to eliminate gene function. In some embodiments, the exogenous nucleic acid sequence provides a translation stop codon to prevent expression of the gene product. In other embodiments, the exogenous nucleic acid sequences provide transcription stop element to prevent expression of a full length RNA molecule. In still other embodiments, gene function is disrupted directly by the meganuclease by introducing base insertions, base deletions, and/or frameshift mutations through non-homologous end-joining.</p>
    <p num="p-0169">In many instances, it is desirable to direct the proper genetic sequences to a target cell or population of cells that is the cause of the disease condition. Such targeting of therapeutics prevents healthy cells from being targeted by the therapeutics. This increases the efficacy of the treatment, while decreasing the potentially adverse effects that the treatment could have on healthy cells.</p>
    <p num="p-0170">Delivery of recombinant meganuclease genes and the sequence of interest to be inserted into the genome to the cells of interest can be accomplished by a variety of mechanisms. In some embodiments, the nucleic acids are delivered to the cells by way of viruses with particular viral genes inactivated to prevent reproduction of the virus. Thus, a virus can be altered so that it is capable only of delivery and maintenance within a target cell, but does not retain the ability to replicate within the target cell or tissue. One or more DNA sequences can be introduced to the altered viral genome, so as to produce a viral genome that acts like a vector, and may or may not be inserted into a host genome and subsequently expressed. More specifically, certain embodiments include employing a retroviral vector such as, but not limited to, the MFG or pLJ vectors. An MFG vector is a simplified Moloney murine leukemia virus vector (MoMLV) in which the DNA sequences encoding the pol and env proteins have been deleted to render it replication defective. A pLJ retroviral vector is also a form of the MoMLV (see, e.g., Korman et al. (1987), <i>Proc. Nat'l Acad. Sci., </i>84:2150-2154). In other embodiments, a recombinant adenovirus or adeno-associated virus can be used as a delivery vector.</p>
    <p num="p-0171">In other embodiments, the delivery of recombinant meganuclease protein and/or recombinant meganuclease gene sequences to a target cell is accomplished by the use of liposomes. The production of liposomes containing nucleic acid and/or protein cargo is known in the art (see, e.g., Lasic et al. (1995), <i>Science </i>267: 1275-76). Immunoliposomes incorporate antibodies against cell-associated antigens into liposomes, and can delivery DNA sequences for the meganuclease or the meganuclease itself to specific cell types (see, e.g., Lasic et al. (1995), <i>Science </i>267: 1275-76; Young et al. (2005), <i>J. Calif. Dent. Assoc. </i>33(12): 967-71; Pfeiffer et al. (2006), <i>J. Vasc. Surg. </i>43(5): 1021-7). Methods for producing and using liposome formulations are well known in the art, (see, e.g., U.S. Pat. Nos. 6,316,024, 6,379,699, 6,387,397, 6,511,676 and 6,593,308, and references cited therein). In some embodiments, liposomes are used to deliver the sequences of interest as well as the recombinant meganuclease protein or recombinant meganuclease gene sequences.</p>
    <p num="h-0017">7. Methods for Treating Pathogen Infection.</p>
    <p num="p-0172">Aspects of the invention also provide methods of treating infection by a pathogen. Pathogenic organisms include viruses such as, but not limited to, herpes simplex virus 1, herpes simplex virus 2, human immunodeficiency virus 1, human immunodeficiency virus 2, variola virus, polio virus, Epstein-Barr virus, and human papilloma virus and bacterial organisms such as, but not limited to, <i>Bacillus anthracis, Haemophilus </i>species, <i>Pneumococcus </i>species, <i>Staphylococcus aureus, Streptococcus </i>species, methicillin-resistant <i>Staphylococcus aureus</i>, and <i>Mycoplasma tuberculosis</i>. Pathogenic organisms also include fungal organisms such as, but not limited to, <i>Candida, Blastomyces, Cryptococcus</i>, and <i>Histoplasma </i>species.</p>
    <p num="p-0173">In some embodiments, a rationally-designed meganuclease can be targeted to a recognition sequence within the pathogen genome, e.g., to a gene or regulatory element that is essential for growth, reproduction, or toxicity of the pathogen. In certain embodiments, the recognition sequence may be in a bacterial plasmid. Meganuclease-mediated cleavage of a recognition sequence in a pathogen genome can stimulate mutation within a targeted, essential gene in the form of an insertion, deletion or frameshift, by stimulating non-homologous end-joining. Alternatively, cleavage of a bacterial plasmid can result in loss of the plasmid along with any genes encoded on it, such as toxin genes (e.g., <i>B. anthracis </i>Lethal Factor gene) or antibiotic resistance genes. As noted above, the meganuclease may be delivered to the infected patient, animal, or plant in either protein or nucleic acid form using techniques that are common in the art. In certain embodiments, the meganuclease gene may be incorporated into a bacteriophage genome for delivery to pathogenic bacteria.</p>
    <p num="p-0174">Aspects of the invention also provide therapeutics for the treatment of certain forms of cancer. Because human viruses are often associated with tumor formation (e.g., Epstein-Barr Virus and nasopharyngeal carcinomas; Human Papilloma Virus and cervical cancer) inactivation of these viral pathogens may prevent cancer development or progression. Alternatively, double-stranded breaks targeted to the genomes of these tumor-associated viruses using rationally-designed meganucleases may be used to trigger apoptosis through the DNA damage response pathway. In this manner, it may be possible to selectively induce apoptosis in tumor cells harboring the viral genome.</p>
    <p num="h-0018">8. Methods for Genotyping and Pathogen Identification</p>
    <p num="p-0175">Aspects of the invention also provide tools for in vitro molecular biology research and development. It is common in the art to use site-specific endonucleases (e.g., restriction enzymes) for the isolation, cloning, and manipulation of nucleic acids such as plasmids, PCR products, BAC sequences, YAC sequences, viruses, and genomic sequences from eukaryotic and prokaryotic organisms (see, e.g., Ausubel et al., <i>Current Protocols in Molecular Biology</i>, Wiley 1999). Thus, in some embodiments, a rationally-designed meganuclease may be used to manipulate nucleic acid sequences in vitro. For example, rationally-designed meganucleases recognizing a pair of recognition sequences within the same DNA molecule can be used to isolate the intervening DNA segment for subsequent manipulation such as ligation into a bacterial plasmid, BAC, or YAC.</p>
    <p num="p-0176">In another aspect, this invention provides tools for the identification of pathogenic genes and organisms. In one embodiment, rationally-designed meganucleases can be used to cleave recognition sites corresponding to polymorphic genetic regions correlated to disease to distinguish disease-causing alleles from healthy alleles (e.g., a rationally-designed meganuclease which recognizes the ΔF-508 allele of the human CFTR gene, see example 4). In this embodiment, DNA sequences isolated from a human patient or other organism are digested with a rationally-designed meganuclease, possibly in conjunction with additional site-specific nucleases, and the resulting DNA fragment pattern is analyzed by gel electrophoresis, capillary electrophoresis, mass spectrometry, or other methods known in the art. This fragmentation pattern and, specifically, the presence or absence of cleavage by the rationally-designed meganuclease, indicates the genotype of the organism by revealing whether or not the recognition sequence is present in the genome. In another embodiment, a rationally-designed meganuclease is targeted to a polymorphic region in the genome of a pathogenic virus, fungus, or bacterium and used to identify the organism. In this embodiment, the rationally-designed meganuclease cleaves a recognition sequence that is unique to the pathogen (e.g., the spacer region between the 16S and 23S rRNA genes in a bacterium; see, e.g., van der Giessen et al. (1994), <i>Microbiology </i>140:1103-1108) and can be used to distinguish the pathogen from other closely-related organisms following endonuclease digest of the genome and subsequent analysis of the fragmentation pattern by electrophoresis, mass spectrometry, or other methods known in the art.</p>
    <p num="h-0019">9. Methods for the Production of Custom DNA-Binding Domains.</p>
    <p num="p-0177">In another aspect, the invention provides rationally-designed DNA-binding proteins that lack endonuclease cleavage activity. The catalytic activity of a rationally-designed meganuclease can be eliminated by mutating amino acids involved in catalysis (e.g., the mutation of Q47 to E in I-CreI, see Chevalier et al. (2001), <i>Biochemistry. </i>43:14015-14026); the mutation of D44 or D145 to N in I-SceI; the mutation of E66 to Q in I-CeuI; the mutation of D22 to N in I-MsoI). The inactivated meganuclease can then be fused to an effector domain from another protein including, but not limited to, a transcription activator (e.g., the GAL4 transactivation domain or the VP16 transactivation domain), a transcription repressor (e.g., the KRAB domain from the Kruppel protein), a DNA methylase domain (e.g., M.CviPI or M.SssI), or a histone acetyltransferase domain (e.g., HDAC1 or HDAC2). Chimeric proteins consisting of an engineered DNA-binding domain, most notably an engineered zinc finger domain, and an effector domain are known in the art (see, e.g., Papworth et al. (2006), <i>Gene </i>366:27-38).</p>
    <heading>EXAMPLES</heading> <p num="p-0178">This invention is further illustrated by the following examples, which should not be construed as limiting. Those skilled in the art will recognize, or be able to ascertain, using no more than routine experimentation, numerous equivalents to the specific substances and procedures described herein. Such equivalents are intended to be encompassed in the scope of the claims that follow the examples below. Examples 1-4 below refer specifically to rationally-designed meganucleases based on I-CreI, but rationally-designed meganucleases based on I-SceI, I-MsoI, I-CeuI, and other LAGLIDADG meganucleases can be similarly produced and used, as described herein.</p>
    <heading>Example 1</heading> <heading>Rational Design of Meganucleases Recognizing the HIV-1 TAT Gene</heading> <p num="h-0023">1. Meganuclease Design.</p>
    <p num="p-0179">A pair of meganucleases were designed to recognize and cleave the DNA site 5′-GAAGAGCTCATCAGAACAGTCA-3′ (SEQ ID NO: 15) found in the HIV-1 TAT Gene. In accordance with Table 1, two meganucleases, TAT1 and TAT2, were designed to bind the half-sites 5′-GAAGAGCTC-3′ (SEQ ID NO: 16) and 5′-TGACTGTTC-3′ (SEQ ID NO: 17), respectively, using the following base contacts (non-WT contacts are in bold):</p>
    <p num="h-0024">TAT1:</p>
    <p num="p-0180"> <tables id="TABLE-US-00014" num="00014"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">G</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">G</td> <td class="description-td">A</td> <td class="description-td">G</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td">S32</td> <td class="description-td">Y33</td> <td class="description-td">N30/</td> <td class="description-td"> <b>R40</b> </td> <td class="description-td">K28</td> <td class="description-td"> <b>S26/</b> </td> <td class="description-td"> <b>K24/</b> </td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Resi-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q38</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>R77</b> </td> <td class="description-td"> <b>Y68</b> </td> </tr> <tr class="description-tr"> <td class="description-td">dues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
TAT2:
</p>
    <p num="p-0181">
      <tables id="TABLE-US-00015" num="00015"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">T</td>
                <td class="description-td">G</td>
                <td class="description-td">A</td>
                <td class="description-td">C</td>
                <td class="description-td">T</td>
                <td class="description-td">G</td>
                <td class="description-td">T</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact</td>
                <td class="description-td">
                  <b>C32</b>
                </td>
                <td class="description-td">
                  <b>R33</b>
                </td>
                <td class="description-td">N30/</td>
                <td class="description-td">
                  <b>R28/</b>
                </td>
                <td class="description-td">
                  <b>M66</b>
                </td>
                <td class="description-td">
                  <b>S26/</b>
                </td>
                <td class="description-td">
                  <b>Y68</b>
                </td>
                <td class="description-td">Q44</td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td class="description-td">Resi-</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Q38</td>
                <td class="description-td">
                  <b>E40</b>
                </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>R77</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">dues</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0182">The two enzymes were cloned, expressed in <i>E. coli</i>, and assayed for enzyme activity against the corresponding DNA recognition sequence as described below. In both cases, the rationally-designed meganucleases were found to be inactive. A second generation of each was then produced in which E80 was mutated to Q to improve contacts with the DNA backbone. The second generation TAT2 enzyme was found to be active against its intended recognition sequence while the second generation TAT1 enzyme remained inactive. Visual inspection of the wild-type I-CreI co-crystal structure suggested that TAT1 was inactive due to a steric clash between R40 and K28. To alleviate this clash, TAT1 variants were produced in which K28 was mutated to an amino acid with a smaller side chain (A, S, T, or C) while maintaining the Q80 mutation. When these enzymes were produced in <i>E. coli </i>and assayed, the TAT1 variants with S28 and T28 were both found to be active against the intended recognition sequence while maintaining the desired base preference at position −7.</p>
    <p num="h-0025">2. Construction of Recombinant Meganucleases.</p>
    <p num="p-0183">Mutations for the redesigned I-CreI enzymes were introduced using mutagenic primers in an overlapping PCR strategy. Recombinant DNA fragments of I-CreI generated in a primary PCR were joined in a secondary PCR to produce full-length recombinant nucleic acids. All recombinant I-CreI constructs were cloned into pET21a vectors with a six histidine tag fused at the 3′ end of the gene for purification (Novagen Corp., San Diego, Calif.). All nucleic acid sequences were confirmed using Sanger Dideoxynucleotide sequencing (see Sanger et al. (1977), <i>Proc. Natl. Acad. Sci. USA. </i>74(12): 5463-7).</p>
    <p num="p-0184">Wild-type I-CreI and all engineered meganucleases were expressed and purified using the following method. The constructs cloned into a pET21a vector were transformed into chemically competent BL21 (DE3) pLysS, and plated on standard 2xYT plates containing 200 μg/ml carbanicillin. Following overnight growth, transformed bacterial colonies were scraped from the plates and used to inoculate 50 ml of 2XYT broth. Cells were grown at 37° C. with shaking until they reached an optical density of 0.9 at a wavelength of 600 nm. The growth temperature was then reduced from 37° C. to 22° C. Protein expression was induced by the addition of 1 mM IPTG, and the cells were incubated with agitation for two and a half hours. Cells were then pelleted by centrifugation for 10 min. at 6000×g. Pellets were resuspended in 1 ml binding buffer (20 mM Tris-HCL, pH 8.0, 500 mM NaCl, 10 mM imidazole) by vortexing. The cells were then disrupted with 12 pulses of sonication at 50% power and the cell debris was pelleted by centrifugation for 15 min. at 14,000×g. Cell supernatants were diluted in 4 ml binding buffer and loaded onto a 200 μl nickel-charged metal-chelating Sepharose column (Pharmacia).</p>
    <p num="p-0185">The column was subsequently washed with 4 ml wash buffer (20 mM Tris-HCl, pH 8.0, 500 mM NaCl, 60 mM imidazole) and with 0.2 ml elution buffer (20 mM Tris-HCl, pH 8.0, 500 mM NaCl, 400 mM imidazole). Meganuclease enzymes were eluted with an additional 0.6 ml of elution buffer and concentrated to 50-130 μl using Vivospin disposable concentrators (ISC, Inc., Kaysville, Utah). The enzymes were exchanged into SA buffer (25 mM Tris-HCL, pH 8.0, 100 mM NaCl, 5 mM MgCl<sub>2</sub>, 5 mM EDTA) for assays and storage using Zeba spin desalting columns (Pierce Biotechnology, Inc., Rockford, Ill.). The enzyme concentration was determined by absorbance at 280 nm using an extinction coefficient of 23,590 M<sup>−1</sup>cm<sup>−1</sup>. Purity and molecular weight of the enzymes was then confirmed by MALDI-TOF mass spectrometry.</p>
    <p num="p-0186">Heterodimeric enzymes were produced either by purifying the two proteins independently, and mixing them in vitro or by constructing an artificial operon for tandem expression of the two proteins in <i>E. coli</i>. In the former case, the purified meganucleases were mixed 1:1 in solution and pre-incubated at 42° C. for 20 minutes prior to the addition of DNA substrate. In the latter case, the two genes were cloned sequentially into the pET-21a expression vector using NdeI/EcoRI and EcoRI/HindIII. The first gene in the operon ends with two stop codons to prevent read-through errors during transcription. A 12-base pair nucleic acid spacer and a Shine-Dalgarno sequence from the pET21 vector separated the first and second genes in the artificial operon.</p>
    <p num="h-0026">3. Cleavage Assays.</p>
    <p num="p-0187">All enzymes purified as described above were assayed for activity by incubation with linear, double-stranded DNA substrates containing the meganuclease recognition sequence. Synthetic oligonucleotides corresponding to both sense and antisense strands of the recognition sequence were annealed and were cloned into the SmaI site of the pUC19 plasmid by blunt-end ligation. The sequences of the cloned binding sites were confirmed by Sanger dideoxynucleotide sequencing. All plasmid substrates were linearized with XmnI, ScaI or BpmI concurrently with the meganuclease digest. The enzyme digests contained 5 μl 0.05 μM DNA substrate, 2.5 μl 5 μM recombinant I-CreI meganuclease, 9.5 μl SA buffer, and 0.5 μl XmnI, ScaI, or BpmI. Digests were incubated at either 37° C., or 42° C. for certain meganuclease enzymes, for four hours. Digests were stopped by adding 0.3 mg/ml Proteinase K and 0.5% SDS, and incubated for one hour at 37° C. Digests were analyzed on 1.5% agarose and visualized by ethidium bromide staining.</p>
    <p num="p-0188">To evaluate meganuclease half-site preference, rationally-designed meganucleases were incubated with a set of DNA substrates corresponding to a perfect palindrome of the intended half-site as well as each of the 27 possible single-base-pair substitutions in the half-site. In this manner, it was possible to determine how tolerant each enzyme is to deviations from its intended half-site.</p>
    <p num="h-0027">4. Recognition Sequence-Specificity.</p>
    <p num="p-0189">Purified recombinant TAT1 and TAT2 meganucleases recognized DNA sequences that were distinct from the wild-type meganuclease recognition sequence (<figref idrefs="DRAWINGS">FIG. 2(B)</figref>). The wild-type I-CreI meganuclease cleaves the WT recognition sequence, but cuts neither the intended sequence for TAT1 nor the intended sequence for TAT2. TAT1 and TAT2, likewise, cut their intended recognition sequences but not the wild-type sequence. The meganucleases were then evaluated for half-site preference and overall specificity (<figref idrefs="DRAWINGS">FIG. 3</figref>). Wild-type I-CreI was found to be highly tolerant of single-base-pair substitutions in its natural half-site. In contrast, TAT1 and TAT2 were found to be highly-specific and completely intolerant of base substitutions at positions −1, −2, −3, −6, and −8 in the case of TAT1, and positions −1, −2, and −6 in the case of TAT2.</p>
    <heading>Example 2</heading> <heading>Rational Design of Meganucleases with Altered DNA-Binding Affinity</heading> <p num="h-0030">1. Meganucleases with Increased Affinity and Increased Activity.</p>
    <p num="p-0190">The meganucleases CCR1 and BRP2 were designed to cleave the half-sites 5′-AACCCTCTC-3′ (SEQ ID NO: 18) and 5′-CTCCGGGTC-3′ (SEQ ID NO: 19), respectively. These enzymes were produced in accordance with Table 1 as in Example 1:</p>
    <p num="h-0031">CCR1:</p>
    <p num="p-0191"> <tables id="TABLE-US-00016" num="00016"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">C</td> <td class="description-td">C</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td"> <b>N32</b> </td> <td class="description-td">Y33</td> <td class="description-td"> <b>R30/</b> </td> <td class="description-td"> <b>R28/</b> </td> <td class="description-td"> <b>E42</b> </td> <td class="description-td">Q26</td> <td class="description-td"> <b>K24/</b> </td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Resi-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>E38</b> </td> <td class="description-td"> <b>E40</b> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>Y68</b> </td> </tr> <tr class="description-tr"> <td class="description-td">dues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
BRP2:
</p>
    <p num="p-0192">
      <tables id="TABLE-US-00017" num="00017"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">C</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
                <td class="description-td">C</td>
                <td class="description-td">G</td>
                <td class="description-td">G</td>
                <td class="description-td">G</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact</td>
                <td class="description-td">
                  <b>S32</b>
                </td>
                <td class="description-td">
                  <b>C33</b>
                </td>
                <td class="description-td">R30/</td>
                <td class="description-td">
                  <b>R28/</b>
                </td>
                <td class="description-td">
                  <b>R42</b>
                </td>
                <td class="description-td">
                  <b>S26/</b>
                </td>
                <td class="description-td">
                  <b>R68</b>
                </td>
                <td class="description-td">Q44</td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td class="description-td">Resi-</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">E38</td>
                <td class="description-td">
                  <b>E40</b>
                </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>R77</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">dues</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0193">Both enzymes were expressed in <i>E. coli</i>, purified, and assayed as in Example 1. Both first generation enzymes were found to cleave their intended recognition sequences with rates that were considerably below that of wild-type I-CreI with its natural recognition sequence. To alleviate this loss in activity, the DNA-binding affinity of CCR1 and BRP2 was increased by mutating E80 to Q in both enzymes. These second-generation versions of CCR1 and BRP2 were found to cleave their intended recognition sequences with substantially increased catalytic rates.</p>
    <p num="h-0032">2. Meganucleases with Decreased DNA-Binding Affinity and Decreased Activity But Increased Specificity.</p>
    <p num="p-0194">Wild-type I-CreI was found to be highly-tolerant of substitutions to its half-site (<figref idrefs="DRAWINGS">FIG. 3(A)</figref>). In an effort to make the enzyme more specific, the lysine at position 116 of the enzyme, which normally makes a salt-bridge with a phosphate in the DNA backbone, was mutated to aspartic acid to reduce DNA-binding affinity. This rationally-designed enzyme was found to cleave the wild-type recognition sequence with substantially reduced activity but the recombinant enzyme was considerably more specific than wild-type. The half-site preference of the K116D variant was evaluated as in Example 1 and the enzyme was found to be entirely intolerant of deviation from its natural half-site at positions −1, −2, and −3, and displayed at least partial base preference at the remaining 6 positions in the half-site (<figref idrefs="DRAWINGS">FIG. 3(B)</figref>).</p>
    <heading>Example 3</heading> <heading>Rationally-Designed Meganuclease Heterodimers</heading> <p num="h-0035">1. Cleavage of Non-Palindromic DNA Sites by Meganuclease Heterodimers Formed in Solution.</p>
    <p num="p-0195">Two meganucleases, LAM1 and LAM2, were designed to cleave the half-sites 5′-TGCGGTGTC-3′ (SEQ ID NO: 20) and 5′-CAGGCTGTC-3′ (SEQ ID NO: 21), respectively. The heterodimer of these two enzymes was expected to recognize the DNA sequence 5′-TGCGGTGTCCGGCGACAGCCTG-3′ (SEQ ID NO: 22) found in the bacteriophage λ p05 gene.</p>
    <p num="h-0036">LAM1:</p>
    <p num="p-0196"> <tables id="TABLE-US-00018" num="00018"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">T</td> <td class="description-td">G</td> <td class="description-td">C</td> <td class="description-td">G</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td"> <b>C32</b> </td> <td class="description-td"> <b>R33</b> </td> <td class="description-td"> <b>R30/</b> </td> <td class="description-td"> <b>D28/</b> </td> <td class="description-td"> <b>R42</b> </td> <td class="description-td">Q26</td> <td class="description-td">R68</td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Resi-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>E38</b> </td> <td class="description-td"> <b>R40</b> </td> </tr> <tr class="description-tr"> <td class="description-td">dues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
LAM2:
</p>
    <p num="p-0197">
      <tables id="TABLE-US-00019" num="00019"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">C</td>
                <td class="description-td">A</td>
                <td class="description-td">G</td>
                <td class="description-td">G</td>
                <td class="description-td">C</td>
                <td class="description-td">T</td>
                <td class="description-td">G</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact</td>
                <td class="description-td">S32</td>
                <td class="description-td">Y33</td>
                <td class="description-td">
                  <b>E30/</b>
                </td>
                <td class="description-td">
                  <b>R40</b>
                </td>
                <td class="description-td">K28/</td>
                <td class="description-td">Q26</td>
                <td class="description-td">R68</td>
                <td class="description-td">Q44</td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td class="description-td">Resi-</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>R38</b>
                </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>E42</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">dues</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0198">LAM1 and LAM 2 were cloned, expressed in <i>E. coli</i>, and purified individually as described in Example 1. The two enzymes were then mixed 1:1 and incubated at 42° C. for 20 minutes to allow them to exchange subunits and re-equilibrate. The resulting enzyme solution, expected to be a mixture of LAM1 homodimer, LAM2 homodimer, and LAM1/LAM2 heterodimer, was incubated with three different recognition sequences corresponding to the perfect palindrome of the LAM1 half-site, the perfect palindrome of the LAM2 half-site, and the non-palindromic hybrid site found in the bacteriophage λ genome. The purified LAM1 enzyme alone cuts the LAM1 palindromic site, but neither the LAM2 palindromic site, nor the LAM1/LAM2 hybrid site. Likewise, the purified LAM2 enzyme alone cuts the LAM2 palindromic site but neither the LAM1 palindromic site nor the LAM1/LAM2 hybrid site. The 1:1 mixture of LAM1 and LAM2, however, cleaves all three DNA sites. Cleavage of the LAM1/LAM2 hybrid site indicates that two distinct redesigned meganucleases can be mixed in solution to form a heterodimeric enzyme capable of cleaving a non-palindromic DNA site.</p>
    <p num="h-0037">2. Cleavage of Non-Palindromic DNA Sites by Meganuclease Heterodimers Formed by Co-Expression.</p>
    <p num="p-0199">Genes encoding the LAM1 and LAM2 enzymes described above were arranged into an operon for simultaneous expression in <i>E. coli </i>as described in Example 1. The co-expressed enzymes were purified as in Example 1 and the enzyme mixture incubated with the three potential recognition sequences described above. The co-expressed enzyme mixture was found to cleave all three sites, including the LAM1/LAM2 hybrid site, indicating that two distinct rationally-designed meganucleases can be co-expressed to form a heterodimeric enzyme capable of cleaving a non-palindromic DNA site.</p>
    <p num="h-0038">3. Preferential Cleavage of Non-Palindromic DNA Sites by Meganuclease Heterodimers with Modified Protein-Protein Interfaces.</p>
    <p num="p-0200">For applications requiring the cleavage of non-palindromic DNA sites, it is desirable to promote the formation of enzyme heterodimers while minimizing the formation of homodimers that recognize and cleave different (palindromic) DNA sites. To this end, variants of the LAM1 enzyme were produced in which lysines at positions 7, 57, and 96 were changed to glutamic acids. This enzyme was then co-expressed and purified as in above with a variant of LAM2 in which glutamic acids at positions 8 and 61 were changed to lysine. In this case, formation of the LAM1 homodimer was expected to be reduced due to electrostatic repulsion between E7, E57, and E96 in one monomer and E8 and E61 in the other monomer. Likewise, formation of the LAM2 homodimer was expected to be reduced due to electrostatic repulsion between K7, K57, and K96 on one monomer and K8 and K61 on the other monomer. Conversely, the LAM1/LAM2 heterodimer was expected to be favored due to electrostatic attraction between E7, E57, and E96 in LAM1 and K8 and K61 in LAM2. When the two meganucleases with modified interfaces were co-expressed and assayed as described above, the LAM1/LAM2 hybrid site was found to be cleaved preferentially over the two palindromic sites, indicating that substitutions in the meganuclease protein-protein interface can drive the preferential formation of heterodimers.</p>
    <heading>Example 4</heading> <heading>Additional Meganuclease Heterodimers which Cleave Physiologic DNA Sequences</heading> <p num="h-0041">1. Meganuclease Heterodimers which Cleave DNA Sequences Relevant to Gene Therapy.</p>
    <p num="p-0201">A rationally-designed meganuclease heterodimer (ACH1/ACH2) can be produced that cleaves the sequence 5′-CTGGGAGTCTCAGGACAGCCTG-3′ (SEQ ID NO: 23) in the human FGFR3 gene, mutations in which cause achondroplasia. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0042">ACH1:</p>
    <p num="p-0202"> <tables id="TABLE-US-00020" num="00020"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">G</td> <td class="description-td">G</td> <td class="description-td">G</td> <td class="description-td">A</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td"> <b>D32</b> </td> <td class="description-td"> <b>C33</b> </td> <td class="description-td"> <b>E30/</b> </td> <td class="description-td"> <b>R40/</b> </td> <td class="description-td"> <b>R42</b> </td> <td class="description-td"> <b>A26/</b> </td> <td class="description-td">R68</td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Resi-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>R38</b> </td> <td class="description-td"> <b>D28</b> </td> <td class="description-td"> </td> <td class="description-td"> <b>Q77</b> </td> </tr> <tr class="description-tr"> <td class="description-td">dues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
ACH2:
</p>
    <p num="p-0203">
      <tables id="TABLE-US-00021" num="00021"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">C</td>
                <td class="description-td">A</td>
                <td class="description-td">G</td>
                <td class="description-td">G</td>
                <td class="description-td">C</td>
                <td class="description-td">T</td>
                <td class="description-td">G</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact</td>
                <td class="description-td">
                  <b>D32</b>
                </td>
                <td class="description-td">Y33</td>
                <td class="description-td">
                  <b>E30/</b>
                </td>
                <td class="description-td">
                  <b>R40</b>
                </td>
                <td class="description-td">
                  <b>K28/</b>
                </td>
                <td class="description-td">Q26</td>
                <td class="description-td">R68</td>
                <td class="description-td">Q44</td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td class="description-td">Resi-</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>R38</b>
                </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>E42</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">dues</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0204">A rationally-designed meganuclease heterodimer (HGH1/HGH2) can be produced that cleaves the sequence 5′-CCAGGTGTCTCTGGACTCCTCC-3′ (SEQ ID NO: 24) in the promoter of the Human Growth Hormone gene. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0043">HGH1:</p>
    <p num="p-0205"> <tables id="TABLE-US-00022" num="00022"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">C</td> <td class="description-td">C</td> <td class="description-td">A</td> <td class="description-td">G</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td"> <b>D32</b> </td> <td class="description-td"> <b>C33</b> </td> <td class="description-td">N30/</td> <td class="description-td"> <b>R40/</b> </td> <td class="description-td"> <b>R42</b> </td> <td class="description-td">Q26</td> <td class="description-td">R68</td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Resi-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q38</td> <td class="description-td"> <b>D28</b> </td> </tr> <tr class="description-tr"> <td class="description-td">dues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
HGH2:
</p>
    <p num="p-0206">
      <tables id="TABLE-US-00023" num="00023"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">G</td>
                <td class="description-td">G</td>
                <td class="description-td">A</td>
                <td class="description-td">G</td>
                <td class="description-td">G</td>
                <td class="description-td">A</td>
                <td class="description-td">G</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact Residues</td>
                <td class="description-td">
                  <b>K32</b>
                </td>
                <td class="description-td">
                  <b>R33</b>
                </td>
                <td class="description-td">N30/Q38</td>
                <td class="description-td">
                  <b>R40/D28</b>
                </td>
                <td class="description-td">
                  <b>R42</b>
                </td>
                <td class="description-td">
                  <b>A26</b>
                </td>
                <td class="description-td">R68</td>
                <td class="description-td">
                  <b>Q44</b>
                </td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0207">A rationally-designed meganuclease heterodimer (CF1/CF2) can be produced that cleaves the sequence 5′-GAAAATATCATTGGTGTTTCCT-3′ (SEQ ID NO: 25) in the ΔF508 allele of the human CFTR gene. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0044">CF1:</p>
    <p num="p-0208"> <tables id="TABLE-US-00024" num="00024"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="35pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="35pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">G</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">T</td> <td class="description-td">A</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact Residues</td> <td class="description-td">S32</td> <td class="description-td">Y33</td> <td class="description-td">N30/Q38</td> <td class="description-td"> <b>Q40</b> </td> <td class="description-td">K28</td> <td class="description-td">Q26</td> <td class="description-td"> <b>H68/C24</b> </td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
CF2:
</p>
    <p num="p-0209">
      <tables id="TABLE-US-00025" num="00025"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="35pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="35pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">A</td>
                <td class="description-td">G</td>
                <td class="description-td">G</td>
                <td class="description-td">A</td>
                <td class="description-td">A</td>
                <td class="description-td">A</td>
                <td class="description-td">C</td>
                <td class="description-td">A</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact Residues</td>
                <td class="description-td">N32</td>
                <td class="description-td">R33</td>
                <td class="description-td">
                  <b>E30/R38</b>
                </td>
                <td class="description-td">
                  <b>Q40</b>
                </td>
                <td class="description-td">
                  <b>K28</b>
                </td>
                <td class="description-td">
                  <b>A26</b>
                </td>
                <td class="description-td">
                  <b>Y68/C24</b>
                </td>
                <td class="description-td">T44</td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0210">A rationally-designed meganuclease heterodimer (CCR1/CCR2) can be produced that cleaves the sequence 5′-AACCCTCTCCAGTGAGATGCCT-3′ (SEQ ID NO: 26) in the human CCR5 gene (an HIV co-receptor). For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0045">CCR1:</p>
    <p num="p-0211"> <tables id="TABLE-US-00026" num="00026"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="231pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="35pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="35pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">C</td> <td class="description-td">C</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact Residues</td> <td class="description-td">N32</td> <td class="description-td">Y33</td> <td class="description-td"> <b>R30/E38</b> </td> <td class="description-td"> <b>E40/R28</b> </td> <td class="description-td"> <b>E42</b> </td> <td class="description-td">Q26</td> <td class="description-td"> <b>Y68/K24</b> </td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
CCR2:
</p>
    <p num="p-0212">
      <tables id="TABLE-US-00027" num="00027"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="35pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="35pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">A</td>
                <td class="description-td">G</td>
                <td class="description-td">G</td>
                <td class="description-td">C</td>
                <td class="description-td">A</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact Residues</td>
                <td class="description-td">
                  <b>N32</b>
                </td>
                <td class="description-td">
                  <b>R33</b>
                </td>
                <td class="description-td">
                  <b>E30/R38</b>
                </td>
                <td class="description-td">
                  <b>E40</b>
                </td>
                <td class="description-td">K28</td>
                <td class="description-td">Q26</td>
                <td class="description-td">
                  <b>Y68/K24</b>
                </td>
                <td class="description-td">Q44</td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0213">A rationally-designed meganuclease heterodimer (MYD1/MYD2) can be produced that cleaves the sequence 5′-GACCTCGTCCTCCGACTCGCTG-3′ (SEQ ID NO: 27) in the 3′ untranslated region of the human DM kinase gene. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0046">MYD1:</p>
    <p num="p-0214"> <tables id="TABLE-US-00028" num="00028"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="231pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="35pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="35pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">G</td> <td class="description-td">A</td> <td class="description-td">C</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact Residues</td> <td class="description-td">S32</td> <td class="description-td">Y33</td> <td class="description-td"> <b>R30/E38</b> </td> <td class="description-td"> <b>E40/R28</b> </td> <td class="description-td"> <b>K66</b> </td> <td class="description-td">Q26/E77</td> <td class="description-td">R68</td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
MYD1:
</p>
    <p num="p-0215"> <tables id="TABLE-US-00029" num="00029"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">C</td> <td class="description-td">A</td> <td class="description-td">G</td> <td class="description-td">C</td> <td class="description-td">G</td> <td class="description-td">A</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact Residues</td> <td class="description-td">S32</td> <td class="description-td">Y33</td> <td class="description-td"> <b>E30/R38</b> </td> <td class="description-td"> <b>E40/R28</b> </td> <td class="description-td"> <b>R42</b> </td> <td class="description-td"> <b>A26</b> </td> <td class="description-td">R68</td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>Q77</b> </td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
2. Meganuclease Heterodimers which Cleave DNA Sequences in Pathogen Genomes.
</p>
    <p num="p-0216">A rationally-designed meganuclease heterodimer (HSV1/HSV2) can be produced that cleaves the sequence 5′-CTCGATGTCGGACGACACGGCA-3′ (SEQ ID NO: 28) in the UL36 gene of Herpes Simplex Virus-1 and Herpes Simplex Virus-2. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0047">HSV1:</p>
    <p num="p-0217"> <tables id="TABLE-US-00030" num="00030"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> <td class="description-td">G</td> <td class="description-td">A</td> <td class="description-td">T</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact Residues</td> <td class="description-td">S32</td> <td class="description-td"> <b>C33</b> </td> <td class="description-td">R30/E38</td> <td class="description-td"> <b>R40/K28</b> </td> <td class="description-td"> <b>Q42/</b> </td> <td class="description-td">Q26</td> <td class="description-td">R68</td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
HSV2:
</p>
    <p num="p-0218">
      <tables id="TABLE-US-00031" num="00031"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">T</td>
                <td class="description-td">G</td>
                <td class="description-td">C</td>
                <td class="description-td">C</td>
                <td class="description-td">G</td>
                <td class="description-td">T</td>
                <td class="description-td">G</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact Residues</td>
                <td class="description-td">C32</td>
                <td class="description-td">R33</td>
                <td class="description-td">
                  <b>R30/E38</b>
                </td>
                <td class="description-td">
                  <b>E40/R28</b>
                </td>
                <td class="description-td">
                  <b>R42</b>
                </td>
                <td class="description-td">Q26</td>
                <td class="description-td">R68</td>
                <td class="description-td">Q44</td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0219">A rationally-designed meganuclease heterodimer (ANT1/ANT2) can be produced that cleaves the sequence 5′-ACAAGTGTCTATGGACAGTTTA-3′ (SEQ ID NO: 29) in the <i>Bacillus anthracis </i>genome. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0048">ANT1:</p>
    <p num="p-0220"> <tables id="TABLE-US-00032" num="00032"> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="56pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="left"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <colspec colname="9" colwidth="21pt" align="left"> </colspec> <colspec colname="10" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">A</td> <td class="description-td">C</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact Residues</td> <td class="description-td"> <b>N32</b> </td> <td class="description-td"> <b>C33</b> </td> <td class="description-td">N30/Q38</td> <td class="description-td"> <b>Q40/A28</b> </td> <td class="description-td"> <b>R42</b> </td> <td class="description-td">Q26</td> <td class="description-td">R68</td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
ANT2:
</p>
    <p num="p-0221">
      <tables id="TABLE-US-00033" num="00033"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">T</td>
                <td class="description-td">A</td>
                <td class="description-td">A</td>
                <td class="description-td">A</td>
                <td class="description-td">C</td>
                <td class="description-td">T</td>
                <td class="description-td">G</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact</td>
                <td class="description-td">
                  <b>C32</b>
                </td>
                <td class="description-td">Y33</td>
                <td class="description-td">N30/</td>
                <td class="description-td">
                  <b>Q40</b>
                </td>
                <td class="description-td">
                  <b>E42</b>
                </td>
                <td class="description-td">Q26</td>
                <td class="description-td">R68</td>
                <td class="description-td">
                  <b>Q44</b>
                </td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td class="description-td">Res-</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Q38</td>
              </tr> <tr class="description-tr"> <td class="description-td">idues</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0222">A rationally-designed meganuclease heterodimer (POX1/POX2) can be produced that cleaves the sequence 5′-AAAACTGTCAAATGACATCGCA-3′ (SEQ ID NO: 30) in the Variola (smallpox) virus gp009 gene. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0049">POX1:</p>
    <p num="p-0223"> <tables id="TABLE-US-00034" num="00034"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td"> <b>N32</b> </td> <td class="description-td">C33</td> <td class="description-td">N30/</td> <td class="description-td"> <b>Q40</b> </td> <td class="description-td">K28</td> <td class="description-td">Q26</td> <td class="description-td">R68</td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Res-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q38</td> </tr> <tr class="description-tr"> <td class="description-td">idues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
POX2:
</p>
    <p num="p-0224">
      <tables id="TABLE-US-00035" num="00035"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">T</td>
                <td class="description-td">G</td>
                <td class="description-td">C</td>
                <td class="description-td">G</td>
                <td class="description-td">A</td>
                <td class="description-td">T</td>
                <td class="description-td">G</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact</td>
                <td class="description-td">
                  <b>C32</b>
                </td>
                <td class="description-td">
                  <b>R33</b>
                </td>
                <td class="description-td">
                  <b>R30/</b>
                </td>
                <td class="description-td">
                  <b>R40</b>
                </td>
                <td class="description-td">
                  <b>C28/</b>
                </td>
                <td class="description-td">Q26</td>
                <td class="description-td">R68</td>
                <td class="description-td">Q44</td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td class="description-td">Res-</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>E38</b>
                </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>Q42</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">idues</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0225">A rationally-designed meganuclease homodimer (EBB1/EBB1) can be produced that cleaves the pseudo-palindromic sequence 5′-CGGGGTCTCGTGCGAGGCCTCC-3′ (SEQ ID NO: 31) in the Epstein-Barr Virus BALF2 gene. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0050">EBB1:</p>
    <p num="p-0226"> <tables id="TABLE-US-00036" num="00036"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">C</td> <td class="description-td">G</td> <td class="description-td">G</td> <td class="description-td">G</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td">S32</td> <td class="description-td"> <b>R33</b> </td> <td class="description-td"> <b>D30/</b> </td> <td class="description-td"> <b>R40/</b> </td> <td class="description-td"> <b>R42</b> </td> <td class="description-td">Q26</td> <td class="description-td"> <b>Y68/</b> </td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Res-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>Q38</b> </td> <td class="description-td"> <b>D28</b> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>K24</b> </td> </tr> <tr class="description-tr"> <td class="description-td">idues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
EBB1:
</p>
    <p num="p-0227"> <tables id="TABLE-US-00037" num="00037"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">G</td> <td class="description-td">G</td> <td class="description-td">A</td> <td class="description-td">G</td> <td class="description-td">G</td> <td class="description-td">C</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td">S32</td> <td class="description-td"> <b>R33</b> </td> <td class="description-td"> <b>D30/</b> </td> <td class="description-td"> <b>R40/</b> </td> <td class="description-td"> <b>R42</b> </td> <td class="description-td">Q26</td> <td class="description-td"> <b>Y68/</b> </td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Res-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>Q38</b> </td> <td class="description-td"> <b>D28</b> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>K24</b> </td> </tr> <tr class="description-tr"> <td class="description-td">idues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
3. Meganuclease Heterodimers which Cleave DNA Sequences in Plant Genomes.
</p>
    <p num="p-0228">A rationally-designed meganuclease heterodimer (GLA1/GLA2) can be produced that cleaves the sequence 5′-CACTAACTCGTATGAGTCGGTG-3′ (SEQ ID NO: 32) in the <i>Arabidopsis thalianna </i>GL2 gene. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0051">GLA1:</p>
    <p num="p-0229"> <tables id="TABLE-US-00038" num="00038"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">C</td> <td class="description-td">A</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td">S32</td> <td class="description-td">Y33</td> <td class="description-td"> <b>R30/</b> </td> <td class="description-td"> <b>S40/</b> </td> <td class="description-td">K28</td> <td class="description-td"> <b>A26/</b> </td> <td class="description-td"> <b>Y68/</b> </td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Res-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>E38</b> </td> <td class="description-td"> <b>C79</b> </td> <td class="description-td"> </td> <td class="description-td"> <b>Q77</b> </td> <td class="description-td"> <b>K24</b> </td> </tr> <tr class="description-tr"> <td class="description-td">idues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
GLA2:
</p>
    <p num="p-0230">
      <tables id="TABLE-US-00039" num="00039"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">C</td>
                <td class="description-td">A</td>
                <td class="description-td">C</td>
                <td class="description-td">C</td>
                <td class="description-td">G</td>
                <td class="description-td">A</td>
                <td class="description-td">C</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact</td>
                <td class="description-td">S32</td>
                <td class="description-td">Y33</td>
                <td class="description-td">
                  <b>R30/</b>
                </td>
                <td class="description-td">
                  <b>E40/</b>
                </td>
                <td class="description-td">
                  <b>R42</b>
                </td>
                <td class="description-td">
                  <b>A26</b>
                </td>
                <td class="description-td">
                  <b>Y68/</b>
                </td>
                <td class="description-td">Q44</td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td class="description-td">Res-</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>E38</b>
                </td>
                <td class="description-td">
                  <b>R28</b>
                </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>Q77</b>
                </td>
                <td class="description-td">
                  <b>K24</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">idues</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0231">A rationally-designed meganuclease heterodimer (BRP1/BRP2) can be produced that cleaves the sequence 5′-TGCCTCCTCTAGAGACCCGGAG-3′ (SEQ ID NO: 33) in the <i>Arabidopsis thalianna </i>BP1 gene. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0052">BRP1:</p>
    <p num="p-0232"> <tables id="TABLE-US-00040" num="00040"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">T</td> <td class="description-td">G</td> <td class="description-td">C</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td"> <b>C32</b> </td> <td class="description-td"> <b>R33</b> </td> <td class="description-td"> <b>R30/</b> </td> <td class="description-td"> <b>R28/</b> </td> <td class="description-td"> <b>K66</b> </td> <td class="description-td"> <b>Q26/</b> </td> <td class="description-td"> <b>Y68/</b> </td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Res-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>E38</b> </td> <td class="description-td"> <b>E40</b> </td> <td class="description-td"> </td> <td class="description-td"> <b>E77</b> </td> <td class="description-td"> <b>K24</b> </td> </tr> <tr class="description-tr"> <td class="description-td">idues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
BRP2:
</p>
    <p num="p-0233">
      <tables id="TABLE-US-00041" num="00041"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">C</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
                <td class="description-td">C</td>
                <td class="description-td">G</td>
                <td class="description-td">G</td>
                <td class="description-td">G</td>
                <td class="description-td">T</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact</td>
                <td class="description-td">S32</td>
                <td class="description-td">
                  <b>C33</b>
                </td>
                <td class="description-td">
                  <b>R30/</b>
                </td>
                <td class="description-td">
                  <b>E40/</b>
                </td>
                <td class="description-td">
                  <b>R42</b>
                </td>
                <td class="description-td">
                  <b>S26</b>
                </td>
                <td class="description-td">R68</td>
                <td class="description-td">Q44</td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td class="description-td">Res-</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>E38</b>
                </td>
                <td class="description-td">
                  <b>R28</b>
                </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>R77</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">idues</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0234">A rationally-designed meganuclease heterodimer (MGC1/MGC2) can be produced that cleaves the sequence 5′-TAAAATCTCTAAGGTCTGTGCA-3′ (SEQ ID NO: 34) in the <i>Nicotiana tabacum </i>Magnesium Chelatase gene. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0053">MGC1:</p>
    <p num="p-0235"> <tables id="TABLE-US-00042" num="00042"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">T</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">T</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td"> <b>C32</b> </td> <td class="description-td">Y33</td> <td class="description-td">N30/</td> <td class="description-td"> <b>Q40/</b> </td> <td class="description-td">K28</td> <td class="description-td">Q26</td> <td class="description-td"> <b>Y68/</b> </td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Res-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q38</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>K24</b> </td> </tr> <tr class="description-tr"> <td class="description-td">idues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
MGC2:
</p>
    <p num="p-0236">
      <tables id="TABLE-US-00043" num="00043"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Position</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">−9</td>
                <td class="description-td">−8</td>
                <td class="description-td">−7</td>
                <td class="description-td">−6</td>
                <td class="description-td">−5</td>
                <td class="description-td">−4</td>
                <td class="description-td">−3</td>
                <td class="description-td">−2</td>
                <td class="description-td">−1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td>
                <td class="description-td">T</td>
                <td class="description-td">G</td>
                <td class="description-td">C</td>
                <td class="description-td">A</td>
                <td class="description-td">C</td>
                <td class="description-td">A</td>
                <td class="description-td">G</td>
                <td class="description-td">A</td>
                <td class="description-td">C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Contact</td>
                <td class="description-td">S32</td>
                <td class="description-td">
                  <b>R33</b>
                </td>
                <td class="description-td">
                  <b>R30/</b>
                </td>
                <td class="description-td">
                  <b>Q40</b>
                </td>
                <td class="description-td">K28</td>
                <td class="description-td">
                  <b>A26</b>
                </td>
                <td class="description-td">R68</td>
                <td class="description-td">
                  <b>T44</b>
                </td>
                <td class="description-td">R70</td>
              </tr> <tr class="description-tr"> <td class="description-td">Resi-</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>E38</b>
                </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">
                  <b>Q77</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td">dues</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0237">A rationally-designed meganuclease heterodimer (CYP/HGH2) can be produced that cleaves the sequence 5′-CAAGAATTCAAGCGAGCATTAA-3′ (SEQ ID NO: 35) in the <i>Nicotiana tabacum </i>CYP82E4 gene. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0054">CYP:</p>
    <p num="p-0238"> <tables id="TABLE-US-00044" num="00044"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">C</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">G</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">T</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td"> <b>D32</b> </td> <td class="description-td">Y33</td> <td class="description-td">N30/</td> <td class="description-td"> <b>R40/</b> </td> <td class="description-td">K28</td> <td class="description-td"> <b>Q77/</b> </td> <td class="description-td"> <b>Y68</b> </td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Resi-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q38</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>A26</b> </td> </tr> <tr class="description-tr"> <td class="description-td">dues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
HGH2:
</p>
    <p num="p-0239"> <tables id="TABLE-US-00045" num="00045"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">T</td> <td class="description-td">T</td> <td class="description-td">A</td> <td class="description-td">A</td> <td class="description-td">T</td> <td class="description-td">G</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td">S32</td> <td class="description-td"> <b>C33</b> </td> <td class="description-td">N30/</td> <td class="description-td"> <b>Q40</b> </td> <td class="description-td"> <b>K66</b> </td> <td class="description-td"> <b>R77/</b> </td> <td class="description-td"> <b>Y68</b> </td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Resi-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q38</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>S26</b> </td> <td class="description-td"> <b>K24</b> </td> </tr> <tr class="description-tr"> <td class="description-td">dues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
4. Meganuclease Heterodimers which Cleave DNA Sequences in Yeast Genomes.
</p>
    <p num="p-0240">A rationally-designed meganuclease heterodimer (URA1/URA2) can be produced that cleaves the sequence 5′-TTAGATGACAAGGGAGACGCAT-3′ (SEQ ID NO: 36) in the <i>Saccharomyces cerevisiae </i>URA3 gene. For example, a meganuclease was designed based on the I-CreI meganuclease, as described above, with the following contact residues and recognition sequence half-sites:</p>
    <p num="h-0055">URA1:</p>
    <p num="p-0241"> <tables id="TABLE-US-00046" num="00046"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">T</td> <td class="description-td">T</td> <td class="description-td">A</td> <td class="description-td">G</td> <td class="description-td">A</td> <td class="description-td">T</td> <td class="description-td">G</td> <td class="description-td">A</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td">S32</td> <td class="description-td"> <b>C33</b> </td> <td class="description-td">N30/</td> <td class="description-td"> <b>R40</b> </td> <td class="description-td">K28</td> <td class="description-td">Q26</td> <td class="description-td">R68</td> <td class="description-td"> <b>T44</b> </td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Resi-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Q38</td> </tr> <tr class="description-tr"> <td class="description-td">dues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
URA2:
</p>
    <p num="p-0242"> <tables id="TABLE-US-00047" num="00047"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Position</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">−9</td> <td class="description-td">−8</td> <td class="description-td">−7</td> <td class="description-td">−6</td> <td class="description-td">−5</td> <td class="description-td">−4</td> <td class="description-td">−3</td> <td class="description-td">−2</td> <td class="description-td">−1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td namest="offset" nameend="9" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Base</td> <td class="description-td">A</td> <td class="description-td">T</td> <td class="description-td">G</td> <td class="description-td">C</td> <td class="description-td">G</td> <td class="description-td">T</td> <td class="description-td">C</td> <td class="description-td">T</td> <td class="description-td">C</td> </tr> <tr class="description-tr"> <td class="description-td">Contact</td> <td class="description-td">N32</td> <td class="description-td"> <b>C33</b> </td> <td class="description-td"> <b>E30/</b> </td> <td class="description-td"> <b>E40/</b> </td> <td class="description-td"> <b>R42</b> </td> <td class="description-td">Q26</td> <td class="description-td"> <b>Y68/</b> </td> <td class="description-td">Q44</td> <td class="description-td">R70</td> </tr> <tr class="description-tr"> <td class="description-td">Resi-</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>R38</b> </td> <td class="description-td"> <b>R28</b> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> <b>K24</b> </td> </tr> <tr class="description-tr"> <td class="description-td">dues</td> </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
5. Recognition Sequence Specificity.
</p>
    <p num="p-0243">The rationally-designed meganucleases outlined above in this Example were cloned, expressed in <i>E. coli</i>, and purified as in Example 1. Each purified meganuclease was then mixed 1:1 with its corresponding heterodimerization partner (e.g., ACH1 with ACH2, HGH1 with HGH2, etc.) and incubated with a linearized DNA substrate containing the intended non-palindromic DNA recognition sequence for each meganuclease heterodimer. As shown in <figref idrefs="DRAWINGS">FIG. 3</figref>, each rationally-designed meganuclease heterodimer cleaves its intended DNA site.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5436150">US5436150</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 1993</td><td class="patent-data-table-td patent-date-value">Jul 25, 1995</td><td class="patent-data-table-td ">The Johns Hopkins University</td><td class="patent-data-table-td ">Inserted mutants into Dna sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5474896">US5474896</a></td><td class="patent-data-table-td patent-date-value">Nov 5, 1992</td><td class="patent-data-table-td patent-date-value">Dec 12, 1995</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Nucleotide sequence encoding the enzyme I-SceI and the uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5789215">US5789215</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 1997</td><td class="patent-data-table-td patent-date-value">Aug 4, 1998</td><td class="patent-data-table-td ">Genpharm International</td><td class="patent-data-table-td ">Gene targeting in animal cells using isogenic DNA constructs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5792632">US5792632</a></td><td class="patent-data-table-td patent-date-value">Nov 7, 1994</td><td class="patent-data-table-td patent-date-value">Aug 11, 1998</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Nucleotide sequence encoding the enzyme I-SceI and the uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5792640">US5792640</a></td><td class="patent-data-table-td patent-date-value">Dec 20, 1995</td><td class="patent-data-table-td patent-date-value">Aug 11, 1998</td><td class="patent-data-table-td ">The Johns Hopkins University</td><td class="patent-data-table-td ">General method to clone hybrid restriction endonucleases using lig gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5830729">US5830729</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 1996</td><td class="patent-data-table-td patent-date-value">Nov 3, 1998</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">I Sce I-induced gene replacement and gene conversion in embryonic stem cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5866361">US5866361</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 1995</td><td class="patent-data-table-td patent-date-value">Feb 2, 1999</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Vectors are useful in gene mapping and site-directed insertion of genes.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5948678">US5948678</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 1998</td><td class="patent-data-table-td patent-date-value">Sep 7, 1999</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">This invention also relates to the use of i-scei for mapping eukaryotic genomes and for in vivo site directed genetic recombination.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5962327">US5962327</a></td><td class="patent-data-table-td patent-date-value">Apr 5, 1995</td><td class="patent-data-table-td patent-date-value">Oct 5, 1999</td><td class="patent-data-table-td ">Institut Pasteur Universite Paris-Vi</td><td class="patent-data-table-td ">Nucleotide sequence encoding the enzyme I-SceI and the uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6238924">US6238924</a></td><td class="patent-data-table-td patent-date-value">Nov 20, 1998</td><td class="patent-data-table-td patent-date-value">May 29, 2001</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">In situ genetic recombination in yeast, bacteria or mammals; insert synthetic gene coding endonuclease into expression vector, transform cells of yeast, bacteria, or mammals and detect insertion into chromosome</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6265196">US6265196</a></td><td class="patent-data-table-td patent-date-value">Mar 31, 1999</td><td class="patent-data-table-td patent-date-value">Jul 24, 2001</td><td class="patent-data-table-td ">Johns Hopkins University</td><td class="patent-data-table-td ">Preparing delivery vehicle containing nuclease having a dna binding domain or gene encoding the nuclease, delivering into cells, then deactivating target</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6395959">US6395959</a></td><td class="patent-data-table-td patent-date-value">May 6, 1996</td><td class="patent-data-table-td patent-date-value">May 28, 2002</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Nucleotide sequence encoding the enzyme I SceI and the use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6566579">US6566579</a></td><td class="patent-data-table-td patent-date-value">Jul 17, 1998</td><td class="patent-data-table-td patent-date-value">May 20, 2003</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Generation of transgenic rodent; obtain embryonic stem cells, incubate with vector, transform cells, allow homologous recombination, propagate cells, recover animal, monitor gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6610545">US6610545</a></td><td class="patent-data-table-td patent-date-value">Apr 18, 2001</td><td class="patent-data-table-td patent-date-value">Aug 26, 2003</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Nucleotide sequence encoding the enzyme I-SceI and the uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6794136">US6794136</a></td><td class="patent-data-table-td patent-date-value">Nov 20, 2000</td><td class="patent-data-table-td patent-date-value">Sep 21, 2004</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Comprises iterative cycles of rational design, site selection, redesign and site selection of the redesigned molecule</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6822137">US6822137</a></td><td class="patent-data-table-td patent-date-value">Feb 4, 1999</td><td class="patent-data-table-td patent-date-value">Nov 23, 2004</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Nucleotide sequence encoding the enzyme I-SCEI and the uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6833252">US6833252</a></td><td class="patent-data-table-td patent-date-value">Jan 27, 2000</td><td class="patent-data-table-td patent-date-value">Dec 21, 2004</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">A DNA sequence that can be incorporated in cloning and expression vectors, transformed cell lines and transgenic animals; the vectors are useful in gene mapping and site-directed insertion of genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7214536">US7214536</a></td><td class="patent-data-table-td patent-date-value">Sep 1, 2004</td><td class="patent-data-table-td patent-date-value">May 8, 2007</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Generating site directed DNA breaks in transformed cells via expression of heterologous endonuclease; bioreactor for generating exogenous proteins; gene therapy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7271000">US7271000</a></td><td class="patent-data-table-td patent-date-value">May 23, 2002</td><td class="patent-data-table-td patent-date-value">Sep 18, 2007</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Providing cells containing double-stranded DNA, wherein such DNA comprising atleast I-Sce I meganucleases and selecting the cells in which double strand break has been induced; useful for mapping eukaryotic genomes; genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7309605">US7309605</a></td><td class="patent-data-table-td patent-date-value">Apr 9, 2004</td><td class="patent-data-table-td patent-date-value">Dec 18, 2007</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Nucleotide sequence encoding the enzyme I-SceI and the uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20020107214">US20020107214</a></td><td class="patent-data-table-td patent-date-value">Jul 27, 2001</td><td class="patent-data-table-td patent-date-value">Aug 8, 2002</td><td class="patent-data-table-td ">The Children&#39;s Medical Center</td><td class="patent-data-table-td ">Gene repair involving the induction of double-stranded DNA cleavage at a chromosomal target site</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20020110898">US20020110898</a></td><td class="patent-data-table-td patent-date-value">Aug 3, 2001</td><td class="patent-data-table-td patent-date-value">Aug 15, 2002</td><td class="patent-data-table-td ">The Children&#39;s Medical Center Corporation</td><td class="patent-data-table-td ">Attenuating endogenous genes in cell; obtain cell, transform with vectors conding restriction enzymes, recover and propagate transformed cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030113887">US20030113887</a></td><td class="patent-data-table-td patent-date-value">Apr 18, 2001</td><td class="patent-data-table-td patent-date-value">Jun 19, 2003</td><td class="patent-data-table-td ">Institut Pasteur And Universite Paris Vi</td><td class="patent-data-table-td ">Use of restriction endonuclease I-SceI for mapping eukaryotic genomes and for in vivo site directed genetic recombination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030175968">US20030175968</a></td><td class="patent-data-table-td patent-date-value">Mar 1, 2001</td><td class="patent-data-table-td patent-date-value">Sep 18, 2003</td><td class="patent-data-table-td ">Golic Kent G.</td><td class="patent-data-table-td ">Endogenous cellular process; genetic engineering; using endonuclease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030182670">US20030182670</a></td><td class="patent-data-table-td patent-date-value">May 23, 2002</td><td class="patent-data-table-td patent-date-value">Sep 25, 2003</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Nucleotide sequence encoding the enzyme I-SceI and the uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030229039">US20030229039</a></td><td class="patent-data-table-td patent-date-value">Jan 2, 2003</td><td class="patent-data-table-td patent-date-value">Dec 11, 2003</td><td class="patent-data-table-td ">The Children&#39;s Medical Center Corporation</td><td class="patent-data-table-td ">Gene therapy; mutant Dna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040002092">US20040002092</a></td><td class="patent-data-table-td patent-date-value">Mar 14, 2003</td><td class="patent-data-table-td patent-date-value">Jan 1, 2004</td><td class="patent-data-table-td ">Sylvain Arnould</td><td class="patent-data-table-td ">Hybrid and single chain meganucleases and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040019002">US20040019002</a></td><td class="patent-data-table-td patent-date-value">Jan 6, 2003</td><td class="patent-data-table-td patent-date-value">Jan 29, 2004</td><td class="patent-data-table-td ">The Children&#39;s Medical Center Corporation</td><td class="patent-data-table-td ">Modulation, deactivation genes in Dna ; chimera restriction endonuclease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040068761">US20040068761</a></td><td class="patent-data-table-td patent-date-value">May 9, 2003</td><td class="patent-data-table-td patent-date-value">Apr 8, 2004</td><td class="patent-data-table-td ">Golic Kent G.</td><td class="patent-data-table-td ">Ends-out gene targeting method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040171154">US20040171154</a></td><td class="patent-data-table-td patent-date-value">Jul 26, 2002</td><td class="patent-data-table-td patent-date-value">Sep 2, 2004</td><td class="patent-data-table-td ">Francesca Storici</td><td class="patent-data-table-td ">in such a way that no heterologous sequence is retained after the mutagenesis is complete; employ integrative recombinant oligonucleotides (IROs); kits; genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050026157">US20050026157</a></td><td class="patent-data-table-td patent-date-value">Sep 5, 2003</td><td class="patent-data-table-td patent-date-value">Feb 3, 2005</td><td class="patent-data-table-td ">David Baltimore</td><td class="patent-data-table-td ">Enzymatic fusion protein comprising cleavage and nuclear localization domain for use gene targeting tool in gene therapeutics</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050032223">US20050032223</a></td><td class="patent-data-table-td patent-date-value">Sep 1, 2004</td><td class="patent-data-table-td patent-date-value">Feb 10, 2005</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Generating site directed DNA breaks in transformed cells via expression of heterologous endonuclease; bioreactor for generating exogenous proteins; gene therapy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050064474">US20050064474</a></td><td class="patent-data-table-td patent-date-value">Aug 6, 2004</td><td class="patent-data-table-td patent-date-value">Mar 24, 2005</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Cleaving cellular chromatin and inserting expression vector comprising nucleotide sequences coding zinc finger-cleavage half domain genetic exchange; genome engineering and homologous recombination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050172365">US20050172365</a></td><td class="patent-data-table-td patent-date-value">Jan 5, 2004</td><td class="patent-data-table-td patent-date-value">Aug 4, 2005</td><td class="patent-data-table-td ">Holger Puchta</td><td class="patent-data-table-td ">Recombination systems and methods for eliminating nucleic acid sequences from the genome of eukaryotic organisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050208489">US20050208489</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2003</td><td class="patent-data-table-td patent-date-value">Sep 22, 2005</td><td class="patent-data-table-td ">Dana Carroll</td><td class="patent-data-table-td ">Targeted chromosomal mutagenasis using zinc finger nucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20060078552">US20060078552</a></td><td class="patent-data-table-td patent-date-value">Mar 14, 2003</td><td class="patent-data-table-td patent-date-value">Apr 13, 2006</td><td class="patent-data-table-td ">Sylvain Arnould</td><td class="patent-data-table-td ">Hybrid and single chain meganucleases and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20060153826">US20060153826</a></td><td class="patent-data-table-td patent-date-value">Jan 28, 2004</td><td class="patent-data-table-td patent-date-value">Jul 13, 2006</td><td class="patent-data-table-td ">Sylvain Arnould</td><td class="patent-data-table-td ">Use of meganucleases for inducing homologous recombination ex vivo and in toto in vertebrate somatic tissues and application thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20060206949">US20060206949</a></td><td class="patent-data-table-td patent-date-value">Jan 28, 2004</td><td class="patent-data-table-td patent-date-value">Sep 14, 2006</td><td class="patent-data-table-td ">Sylvain Arnould</td><td class="patent-data-table-td ">Custom-made meganuclease and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20060282914">US20060282914</a></td><td class="patent-data-table-td patent-date-value">Nov 17, 2004</td><td class="patent-data-table-td patent-date-value">Dec 14, 2006</td><td class="patent-data-table-td ">D Halluin Kathleen</td><td class="patent-data-table-td ">Targeted dna insertion in plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1485475A1?cl=en">EP1485475A1</a></td><td class="patent-data-table-td patent-date-value">Mar 14, 2003</td><td class="patent-data-table-td patent-date-value">Dec 15, 2004</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">Hybrid and single chain meganucleases and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1996014408A2?cl=en">WO1996014408A2</a></td><td class="patent-data-table-td patent-date-value">Nov 6, 1995</td><td class="patent-data-table-td patent-date-value">May 17, 1996</td><td class="patent-data-table-td ">Pasteur Institut</td><td class="patent-data-table-td ">Nucleotide sequence encoding the enzyme i-scei and the uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2000046386A2?cl=en">WO2000046386A2</a></td><td class="patent-data-table-td patent-date-value">Feb 3, 2000</td><td class="patent-data-table-td patent-date-value">Aug 10, 2000</td><td class="patent-data-table-td ">Childrens Medical Center</td><td class="patent-data-table-td ">Gene repair involving the induction of double-stranded dna cleavage at a chromosomal target site</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2003078619A1?cl=en">WO2003078619A1</a></td><td class="patent-data-table-td patent-date-value">Mar 14, 2003</td><td class="patent-data-table-td patent-date-value">Sep 25, 2003</td><td class="patent-data-table-td ">Sylvain Arnould</td><td class="patent-data-table-td ">Hybrid and single chain meganucleases and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004031346A2?cl=en">WO2004031346A2</a></td><td class="patent-data-table-td patent-date-value">Sep 5, 2003</td><td class="patent-data-table-td patent-date-value">Apr 15, 2004</td><td class="patent-data-table-td ">Hutchinson Fred Cancer Res</td><td class="patent-data-table-td ">Methods and compositions concerning designed highly-specific nucleic acid binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004067736A2?cl=en">WO2004067736A2</a></td><td class="patent-data-table-td patent-date-value">Jan 28, 2004</td><td class="patent-data-table-td patent-date-value">Aug 12, 2004</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">Custom-made meganuclease and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004067753A2?cl=en">WO2004067753A2</a></td><td class="patent-data-table-td patent-date-value">Jan 28, 2004</td><td class="patent-data-table-td patent-date-value">Aug 12, 2004</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">Use of meganucleases for inducing homologous recombination ex vivo and in toto in vertebrate somatic tissues and application thereof.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005049842A2?cl=en">WO2005049842A2</a></td><td class="patent-data-table-td patent-date-value">Nov 17, 2004</td><td class="patent-data-table-td patent-date-value">Jun 2, 2005</td><td class="patent-data-table-td ">Bayer Bioscience Nv</td><td class="patent-data-table-td ">Improved targeted dna insertion in plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005105989A1?cl=en">WO2005105989A1</a></td><td class="patent-data-table-td patent-date-value">Apr 27, 2005</td><td class="patent-data-table-td patent-date-value">Nov 10, 2005</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">I-dmoi derivatives with enhanced activity at 37°c and use thereof.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2006097784A1?cl=en">WO2006097784A1</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2005</td><td class="patent-data-table-td patent-date-value">Sep 21, 2006</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">I-crei meganuclease variants with modified specificity, method of preparation and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2006097853A1?cl=en">WO2006097853A1</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2006</td><td class="patent-data-table-td patent-date-value">Sep 21, 2006</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">I-crei meganuclease variants with modified specificity, method of preparation and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2006097854A1?cl=en">WO2006097854A1</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2006</td><td class="patent-data-table-td patent-date-value">Sep 21, 2006</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">Heterodimeric meganucleases and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2007034262A1?cl=en">WO2007034262A1</a></td><td class="patent-data-table-td patent-date-value">Sep 19, 2005</td><td class="patent-data-table-td patent-date-value">Mar 29, 2007</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">Heterodimeric meganucleases and use thereof</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Aagaard et al (1997), Profile of the DNA recognition site of the archaeal homing endonuclease I-Dmol, Nucleic Acids Res. 25(8):1523-1530.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Adler, David A. et al. "<a href='http://scholar.google.com/scholar?q="Bioinformatics."'>Bioinformatics.</a>" Encyclopedia of Life Sciences. No month listed-2001. John Wiley &amp; Sons Ltd. pp. 1-8.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Adler, David A. et al. "<a href='http://scholar.google.com/scholar?q="Bioinformatics."'>Bioinformatics.</a>" Encyclopedia of Life Sciences. No month listed—2001. John Wiley &amp; Sons Ltd. pp. 1-8.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Aggarwal et al., Novel site-specific DNA endonucleases, Structural Biology, 1998, pp. 19-25, vol. 8.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Argast et al., I-Ppol and I-Crel homing site sequence degeneracy determined by random mutagenesis and sequential in vitro enrichment, J. Mol. Biol., 1998, pp. 345-353, vol. 280.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Arnould et al., Engineering of large numbers of highly specific homing endonucleases that induce recombination of novel DNA targets, J. Mol. Biol., 2006, pp. 443-458, vol. 355.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ashworth et al. Jun. 2006. Computational redesign of endonucleases DNA binding and cleavage specificity. Nature, 441, 656-659.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Belfort et al., Homing enconucleases: keeping the house in order, Nucleic Acids Research, 1997, vol. 25, No. 17, pp. 3379-3388.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Beumer et al., Efficient gene targeting in Drosophila with zinc finger nucleases, Genetics, Feb. 1, 2006, pp. 1-48, 10.1534/genetics.105.052829.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bibikova et al., Enhancing gene targeting with designed zinc finger nucleases, Science, May 2, 2003, p. 764, vol. 300.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bibikova et al., Stimulation of homologous recombination through targeted cleavage by chimeric nucleases, Molecular and Cellular Biology, Jan. 2001, pp. 289-297, vol. 21 No. 1.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bibikova et al., Targeted chromosomal cleavage and mutagenesis in Drosophila using zinc-finger nucleases, Genetics, Jul. 2002, pp. 1169-1175, vol. 161.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bolduc et al. Revised Sep. 2003. Structural and biochemical analyses of DNA and RNA binding by a bifunctional homing endonuclease and group I intron splicing factor. Genes Dev, 17, 2875-2888.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chames et al., In vivo selection of engineered homing endonucleases using double-strand break induced homologous recombination, Nucleic Acids Research, 2005, pp. 1-10, vol. 33 No. 20 e178.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chen et al., A highly sensitive selection method for directed evolution of homing endonucleases, Nucleic Acids Research, 2005, pp. 1-7, vol. 33 No. 18 e154.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chen, R. Jan. 1, 2001. Enzyme engineering: rational redesign versus directed evolution. Trends Biotechnol, 19, 13-14.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chevalier B. S. et al. "<a href='http://scholar.google.com/scholar?q="Homing+Endonucleases%3A+Structural+and+Functional+Insight+Into+the+Catalysts+if+Intron%2FIntein+Mobility."'>Homing Endonucleases: Structural and Functional Insight Into the Catalysts if Intron/Intein Mobility.</a>" Nucleic Acids Research. Oxford University Press. Surrey, Great Britain. vol. 29, No. 18, Sep. 15, 2001. pp. 3757-3774.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chevalier et al. (2002), Design, Activity, and Structure of a Highly Specific Artificial Endonuclease, Mol. Cell 10:895-905.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chevalier et al., Design, activity, and structure of a highly specific artificial endonuclease, Molecular Cell, Oct. 2002, pp. 895-905, vol. 10.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chevalier et al., Flexible DNA target site recognition by divergent homing endoculease isoschizomers I-Crel and I-Msol, J. Mol. Biol., 2003, pp. 253-269, vol. 329.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chevalier et al., Metal-dependent DNA cleavage mechanism of the I-Crel LAGLIDADG homing endonuclease, Biochemistry, 2004, pp. 14015-14026, vol. 43.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chevalier et al., The homing endonuclease I-Crel uses three metals, one of which is shared between the two active sites, Nature Structural Biology, Apr. 2001, pp: 312-316, vol. 8 No. 4.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chilton et al., Targeted integration of T-DNA into the tobacco genome at double-stranded breaks: new insights on the mechanism of T-DNA integration, Plant Physiology, Nov. 2003, pp. 956-965, vol. 133.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cozzone, Alain J. "<a href='http://scholar.google.com/scholar?q="Proteins%3A+Fundamental+Chemical+Properties."'>Proteins: Fundamental Chemical Properties.</a>" Encyclopedia of Life Sciences. No month listed-2002. John Wiley &amp; Sons Ltd. pp. 1-10.</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cozzone, Alain J. "<a href='http://scholar.google.com/scholar?q="Proteins%3A+Fundamental+Chemical+Properties."'>Proteins: Fundamental Chemical Properties.</a>" Encyclopedia of Life Sciences. No month listed—2002. John Wiley &amp; Sons Ltd. pp. 1-10.</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dalgaard, Jacob Z. et al. "<a href='http://scholar.google.com/scholar?q="A+Site-Specific+Endonuclease+Encoded+by+a+Typical+Archaeal+Intron."'>A Site-Specific Endonuclease Encoded by a Typical Archaeal Intron.</a>" Proc. Natl. Acad. Sci. Jun. 1993. vol. 90. pp. 5414-5417.</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Desjarlais et al., Use of a zinc-finger consensus sequence framework and specificity rules to design specific DNA binding proteins, Proc. Natl. Acad. Sci. USA, Mar. 1993, pp. 2256-2260, vol. 90.</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dhanasekaran et al., Designer zinc finger proteins: tools for creating artificial DNA-binding functional proteins, Acc. Chem. Res., 2006, pp. 45-52, vol. 30 No. 1.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Doyon et al., Directed evolution and substrate specificity profile of homing endonuclease I-Scel, J. Am. Chem. Soc., 2006, pp. 2477-2484, vol. 128.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Duan et al. May 16, 1997. Crystal structure of PI-Scel, a homing endonuclease with protein splicing activity. Cell, 89, 555-564.</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Durai et al. Oct. 26, 2005. Zinc finger nucleases: custom-designed molecular scissors for genome engineering of plant and mammalian cells. Nucleic Acids Res, 33, 5978-5990.</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Epinat et al., A novel engineered meganuclease induces homologous recombination in yeast and mammalian cells, Nucleic Acids Research, 2003, pp. 2952-2962, vol. 31 No. 11.</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">European Office Action, European Patent Application No. 06 826 293.0, dated Nov. 20, 2008 (3 pages).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">European Search Report for European Patent Application No. 10191885.2 mailed Apr. 26, 2011. 12 pages.</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">European Search Report for European Patent Application No. 10191888.6 mailed May 18, 2011. 5 pages.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">European Search Report for European Patent Application No. 10191904.1 mailed May 13, 2011. 9 pages.</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fitzsimons Hall et al., Creation of an artificial bifunctional intein by grafting a homing endonuclease into a mini-intein, J. Mol. Biol., 2002, pp. 173-179, vol. 323.</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gimble et al. Oct. 2003. Assessing the plasticity of DNA target site recognition of the PI-Scel homing endonuclease using a bacterial two-hybrid selection system. J Mol Biol, 334, 993-1008.</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gimble, F.S. Engineering Homing Endonulceases for Genomic Applications. In Homing Endonucleases and Inteins. Belfort, Derbyshire, Stoddard and Woods, Eds. Springer-Verlag, Berlin Heidelberg. No month listed-2005. 16 pages.</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gimble, F.S. Engineering Homing Endonulceases for Genomic Applications. In Homing Endonucleases and Inteins. Belfort, Derbyshire, Stoddard and Woods, Eds. Springer-Verlag, Berlin Heidelberg. No month listed—2005. 16 pages.</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grindl et al., The protein splicing domain of the homing endonuclease PI-Scel is responsible for specific DNA binding, Nucleic Acids Research, 1998, vol. 26, No. 8, pp. 1857-1862.</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Guhan et al., Structural and functional characteristics of homing endonucleases, Critical Reviews in Biochemistry and Molecular Biology, 2003, pp. 199-248, vol. 38 No. 3.</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Harris, J.L. and Craik, C.S. Engineering enzyme specificity. Curr Opin Chem Biol, 2, 127-132. No month listed-1998.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Harris, J.L. and Craik, C.S. Engineering enzyme specificity. Curr Opin Chem Biol, 2, 127-132. No month listed—1998.</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Heath, Patrick J. et al. "<a href='http://scholar.google.com/scholar?q="The+Structure+of+I-Crel%2C+A+Group+I+Intron-encoded+Homing+Endonuclease."'>The Structure of I-Crel, A Group I Intron-encoded Homing Endonuclease.</a>" Nature Structural Biology. vol. 4, No. 6. Jun. 1997. pp. 468-476.</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hu et al. "<a href='http://scholar.google.com/scholar?q="Probing+the+Structure+of+the+PI-Seel-DNA+Complex+by+Affinity+Cleavage+and+Affinity+Photcross-linking."'>Probing the Structure of the PI-Seel-DNA Complex by Affinity Cleavage and Affinity Photcross-linking.</a>" The Journal of Biological Chemistry. Jan. 28, 2000. vol. 275, No. 4. pp. 2705-2712.</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ichiyanagi et al., Crystal structure of an archaeal intein-encoded homing endonuclease PI-Pful, J. Mol. Biol., 2000, pp. 889-901, vol. 300.</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">International Search Report from PCT/US2006/040919, mailed Aug. 29, 2007.</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jurica et al. (1998), DNA Recognition and Cleavage by the LAGLIDADG Homing Endonuclease I-Crel, Mol. Cell 2:469-476.</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jurica et al., DNA recognication and cleavage by the LAGLIDADG homing endonuclease I-Crel, Molecular Cell, Oct. 1998, pp. 469-476, vol. 2.</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jurica, M.S. et al. "<a href='http://scholar.google.com/scholar?q="Homing+Endonucleases%3A+Structure%2C+Function+and+Evolution."'>Homing Endonucleases: Structure, Function and Evolution.</a>" CMLS: Cellular and Molecular Life Sciences. Feb. 1999. vol. 55. pp. 1304-1326.</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Liang et al. "<a href='http://scholar.google.com/scholar?q="Genetic+Fusion+of+Subunits+of+a+Dimeric+Protein+Substantially+Enhances+its+Stability+and+Rate+of+Folding."'>Genetic Fusion of Subunits of a Dimeric Protein Substantially Enhances its Stability and Rate of Folding.</a>" Proc. Natl. Acad. Sci. vol. 90. Aug. 1993. pp. 7010-7014.</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lloyd et al., Targeted mutagenesis using zinc-finger nucleases in Arabidopsis, PNAS, Feb. 8, 2005, pp. 2232-2237, vol. 102 No. 6.</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Looger, L. et al., Computational Design of Receptor and Sensor Proteins with Novel Functions, Nature, 2003, pp. 185-190, vol. 423 No. 6936.</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lucas et al., Rapid evolution of the DNA-binding site in LAGLIDADG homing endonucleases, Nucleic Acids Research, 2001, pp. 960-969, vol. 29 No. 4.</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Maggert et al., Highly efficient sex chromosome interchanges produced by I-Crel expression in Drosophila, Genetics, Nov. 2005, pp. 1103-1114, vol. 171.</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mani et al., Binding of two zinc finger nuclease monomers to two specific sites is required for effective double-strand DNA cleavage, Biochemical and Biophysical Research Communications, 2005, pp. 1191-1197, vol. 334.</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mani et al., Design, engineering, and characterization of zinc finger nucleases, Biochemical and Biophysical Research Communications, 2005, pp. 447-457, vol. 335.</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Matsumura et al. Feb. 21, 2006. Crystal structure of intein homing endonuclease II encoded in DNA polymerase gene from hyperthermophilic archaeon Thermococcus kodakaraensis strain KOD1. Proteins, 63, 711-715.</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McDaniel et al., Advances in synthetic biology: on the path from prototypes to applications, Current Opinion in Biotechnology, 2005, pp. 476-483, vol. 16.</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Monnat et al., Generation of highly site-specific DNA double-strand breaks in human cells by the homing endonucleases I-Ppol and I Crel, Biochemical and Biophysical Research Communications, 1999, pp. 88-93, vol. 255.</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Moure et al. Sep. 2003. The crystal structure of the gene targeting homing endonuclease I-Scel reveals the origins of its target site specificity. J Mol Biol, 334, 685-695.</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Moure et al., Crystal structure of the intein homing endonuclease PI-Scel bound to its recognition sequence, Nature Structural Biology, Oct. 2002, pp. 764-770, vol. 9 No. 10.</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Moure, Carmen M. et al. "<a href='http://scholar.google.com/scholar?q="Crystal+Structure+of+the+Intein+Homing+Endonuclease+PI-Scel+Bound+to+its+Recognition+Sequence."'>Crystal Structure of the Intein Homing Endonuclease PI-Scel Bound to its Recognition Sequence.</a>" Howard Hughes Medical Institute, Baylor College of Medicine, Houston Texas. Nature Publishing Group. Sep. 3, 2002. 7 pages.</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pabo et al., Transcription factors: structural families and principles of DNA recognition, Annu. Rev. Biochem., 1992, pp. 1053-1095, vol. 61.</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pace, C. Nick et al. "<a href='http://scholar.google.com/scholar?q="Protein+Stability."'>Protein Stability.</a>" Encyclopedia of Life Sciences. No month listed-2001. John Wiley &amp; Sons Ltd. pp. 1-4.</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pace, C. Nick et al. "<a href='http://scholar.google.com/scholar?q="Protein+Stability."'>Protein Stability.</a>" Encyclopedia of Life Sciences. No month listed—2001. John Wiley &amp; Sons Ltd. pp. 1-4.</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Poland, Bradley W. et al. "<a href='http://scholar.google.com/scholar?q="Structural+Insights+into+the+Protein+Splicing+Mechanism+of+PI-Scel."'>Structural Insights into the Protein Splicing Mechanism of PI-Scel.</a>" The Journal of Biological Chemistry. Jun. 2000. vol. 275, No. 22. pp. 16408-16413.</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Porteus et al., Gene targeting using zinc finger nucleases, Nature Biotechnology, Aug. 2005, pp. 967-973, vol. 23 No. 8.</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Porteus, Mammalian gene targeting with designed zinc finger nucleases, Molecular Therapy, Feb. 2006, pp. 438-446, vol. 13 No. 2.</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Prieto et al., Nucleic Acids Research 35(10):3262-3271, 2007.</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Puchta et al., Two different but related mechanisms are used in plants for the repair of genomic double-strand breaks by homologous recombination, Proc. Natl. Acad. Sci. USA, May 1996, pp. 5055-5060, vol. 93.</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rong et al., Targeted mutagenesis by homologous recombination in D. melanogaster, Genes &amp; Dev., 2002, pp. 1568-1581, vol. 16.</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rosen et al. Sep. 13, 2006. Homing endonuclease I-Crel derivatives with novel DNA target specificities. Nucleic Acids Res, 34, 4791-4800.</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rouet et al., Introduction of double-strand breaks into the genome of mouse cells by expression of a rare-cutting endonuclease, Molecular and Cellular Biology, Dec. 1994, pp. 8096-8106, vol. 14 No. 12.</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sali, Andrej et al. "<a href='http://scholar.google.com/scholar?q="Comparative+Protein+Modelling+by+Satisfaction+of+Spatial+Restraints."'>Comparative Protein Modelling by Satisfaction of Spatial Restraints.</a>" J. Mol. Biol. Jul. 1993. pp. 779-815.</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Seligman et al. (2002), Mutations altering the cleavage specificity of a homing endonuclease, Nucleic Acids Res. 30(17): 3870-3879.</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Seligman et al., Genetic Analysis of the Chlamydomonas reinhardtii I-Crel mobile intron homing system in Escherichia coli, Genetics, Dec. 1997, pp. 1653-1664, vol. 147.</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Seligman et al., Mutations altering the cleavage specificity of a homing endonuclease, Nucleic Acids Research, 2002, pp. 3870-3879, vol. 30 No. 17.</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shen et al. Jul. 2004. DNA binding and cleavage by the HNH homing endonuclease I-Hmul. J Mol Biol, 342, 43-56.</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Silva et al., Analysis of the LAGLIDADG interface of the monomeric homing endonuclease I-Dmol, Nucleic Acids Research, 2004, pp. 3156-3168, vol. 32 No. 10.</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Silva et al., Crystal structure of the thermostable archaeal intron-encoded endonuclease I-Dmol, J. Mol. Biol., 1999, pp. 1123-1136, vol. 286.</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Silva, George H. et al. "<a href='http://scholar.google.com/scholar?q="From+Monomeric+to+Homodimeric+Endonucleases+and+Back%3A+Engineering+Novel+Specificity+of+LAGLIDADG+Enzymes."'>From Monomeric to Homodimeric Endonucleases and Back: Engineering Novel Specificity of LAGLIDADG Enzymes.</a>" J. Mol. Biol. Jul. 2006. pp. 744-754.</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Smith et al., A combinatorial approach to create artificial homing endonucleases cleaving chosen sequences, Nucleic Acids Research, vol. 00, No. 00, Nov. 27, 2006, pp. 1-12.</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Smith et al., A detailed study of the substrate specificity of a chimeric restriction enzyme, Nucleic Acids Research, 1999, pp. 674-681, vol. 27 No. 2.</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Smith et al., Requirements for double-strand cleavage by chimeric restriction enzymes with zinc finger DNA-recognition domains, Nucleic Acids Research, 2000, pp. 3361-3369, vol. 28 No. 17.</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Spiegel et al., The structure of I-Ceul homing endonuclease: evolving asymmetric DNA recognition from a symmetric protein scaffold, Structure, May 2006, pp. 869-880, vol. 14.</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stoddard, Homing endonuclease structure and function, Quarterly Reviews of Biophysics, 2005, pp. 1-47.</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sussman et al., Isolation and characterization of new homing endonuclease specificities at individual target site positions, J. Mol. Biol., 2004, pp. 31-41, vol. 342.</td></tr><tr><td class="patent-data-table-td ">90</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Turmel, Monique et al. "<a href='http://scholar.google.com/scholar?q="Evolutionary+Conserved+and+Functionally+Important+Residues+in+the+I-Ceul+Homing+Endonuclease."'>Evolutionary Conserved and Functionally Important Residues in the I-Ceul Homing Endonuclease.</a>" Nucleic Acid Research. Apr. 1997. vol. 25, No. 13. pp. 2610-2619.</td></tr><tr><td class="patent-data-table-td ">91</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tzfira et al., Towards targeted mutagenesis and gene replacement in plants, Trends in Biotechnology, Dec. 2005, pp. 567-569, vol. 23 No. 12.</td></tr><tr><td class="patent-data-table-td ">92</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Urnov et al., Designed transcription factors as structural, functional and therapeutic probes of chromatin in vivo, EMBO Reports, 2002, pp. 610-615, vol. 3 No. 7.</td></tr><tr><td class="patent-data-table-td ">93</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Urnov et al., Highly efficient endogenous human gene correction using designed zinc-finger nucleases, Nature, Jun. 2, 2005, pp. 646-651, vol. 435.</td></tr><tr><td class="patent-data-table-td ">94</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wang et al., Purification, biochemical characterization and protein-DNA interactions of the I-Crel endonuclease produced in Escherichia coli, Nucleic Acids Research, 1997, pp. 3767-3776, vol. 25 No. 19.</td></tr><tr><td class="patent-data-table-td ">95</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wende et al. (1996), Binding, bending and cleavage of DNA substrates by the homing endonuclease PI-Scel, Nucleic Acid Res. 24(21): 4123-4132.</td></tr><tr><td class="patent-data-table-td ">96</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Werner, Erik et al. "<a href='http://scholar.google.com/scholar?q="High+Resolution+Crystal+Structure+of+Domain+I+of+the+Saccharomyces+cerevisiae+Homing+Endonuclease+PI-Scel."'>High Resolution Crystal Structure of Domain I of the Saccharomyces cerevisiae Homing Endonuclease PI-Scel.</a>" Nucelic Acid Research. Jul. 2002. vol. 30, No. 18. pp. 3962-3971.</td></tr><tr><td class="patent-data-table-td ">97</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wright et al., High-frequency homologous recombination in plants mediated by zinc-finger nucleases, The Plant Journal, 2005, pp. 693-705, vol. 44.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8338157">US8338157</a></td><td class="patent-data-table-td patent-date-value">Sep 10, 2010</td><td class="patent-data-table-td patent-date-value">Dec 25, 2012</td><td class="patent-data-table-td ">Precision Biosciences, Inc.</td><td class="patent-data-table-td ">Rationally-designed meganuclease variants of lig-34 and I-crei for maize genome engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8624000">US8624000</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 31, 2012</td><td class="patent-data-table-td patent-date-value">Jan 7, 2014</td><td class="patent-data-table-td ">Cellectis S.A.</td><td class="patent-data-table-td ">Use of meganucleases for inducing homologous recombination ex vivo and in toto in vertebrate somatic tissues and application thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8697395">US8697395</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 24, 2012</td><td class="patent-data-table-td patent-date-value">Apr 15, 2014</td><td class="patent-data-table-td ">Cellectis S.A.</td><td class="patent-data-table-td ">Use of meganucleases for inducing homologous recombination ex vivo and in toto in vertebrate somatic tissues and application thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20100086533">US20100086533</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 19, 2007</td><td class="patent-data-table-td patent-date-value">Apr 8, 2010</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">Laglidadg homing endonuclease variants having novel substrate specificity and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20120171191">US20120171191</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 26, 2010</td><td class="patent-data-table-td patent-date-value">Jul 5, 2012</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">Meganuclease variants cleaving the genome of a pathogenic non-integrating virus and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20120288941">US20120288941</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 24, 2012</td><td class="patent-data-table-td patent-date-value">Nov 15, 2012</td><td class="patent-data-table-td ">Cellectis S.A.</td><td class="patent-data-table-td ">Use of meganucleases for inducing homologous recombination ex vivo and in toto in vertebrate somatic tissues and application thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20120288942">US20120288942</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 24, 2012</td><td class="patent-data-table-td patent-date-value">Nov 15, 2012</td><td class="patent-data-table-td ">Cellectis S.A.</td><td class="patent-data-table-td ">Use of meganucleases for inducing homologous recombination ex vivo and in toto in vertebrate somatic tissues and application thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20120288943">US20120288943</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 24, 2012</td><td class="patent-data-table-td patent-date-value">Nov 15, 2012</td><td class="patent-data-table-td ">Cellectis S.A.</td><td class="patent-data-table-td ">Use of meganucleases for inducing homologous recombination ex vivo and in toto in vertebrate somatic tissues and application thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20120304321">US20120304321</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 31, 2012</td><td class="patent-data-table-td patent-date-value">Nov 29, 2012</td><td class="patent-data-table-td ">Cellectis S.A.</td><td class="patent-data-table-td ">Use of meganucleases for inducing homologous recombination ex vivo and in toto in vertebrate somatic tissues and application thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20120331574">US20120331574</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 31, 2012</td><td class="patent-data-table-td patent-date-value">Dec 27, 2012</td><td class="patent-data-table-td ">Cellectis S.A.</td><td class="patent-data-table-td ">Use of meganucleases for inducing homologous recombination ex vivo and in toto in vertebrate somatic tissues and application thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013136273A2?cl=en">WO2013136273A2</a></td><td class="patent-data-table-td patent-date-value">Mar 13, 2013</td><td class="patent-data-table-td patent-date-value">Sep 19, 2013</td><td class="patent-data-table-td ">University Of Guelph</td><td class="patent-data-table-td ">Methods of increasing tolerance to heat stress and amino acid content of plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013136274A1?cl=en">WO2013136274A1</a></td><td class="patent-data-table-td patent-date-value">Mar 13, 2013</td><td class="patent-data-table-td patent-date-value">Sep 19, 2013</td><td class="patent-data-table-td ">University Of Guelph</td><td class="patent-data-table-td ">Myb55 promoter and use thereof</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S196000">435/196</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S350000">530/350</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069100">435/69.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006180">435/6.18</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0021000000">C12P21/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014000000">C07K14/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0009160000">C12N9/16</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/22">C12N9/22</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=b3MABwABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/902">C12N15/902</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12N9/22</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">May 6, 2014</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-19 IS CONFIRMED.NEW CLAIMS 20-32 ARE ADDED AND DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 11, 2012</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20121005</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 16, 2007</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">DUKE UNIVERSITY, NORTH CAROLINA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:SMITH, JAMES J.;JANTZ, DEREK;HELLINGA, HOMME W.;REEL/FRAME:018760/0216</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20061214</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_4ff636b3d23669b7103f3b3a3a18b4cd.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U1XcQTsxrNAU-izv1bF-_x6q1exLw\u0026id=b3MABwABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U0Cg0kuK-vlNlmqdeMWueUTcByjHA\u0026id=b3MABwABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U2izmcMsNaJIynN1s7_v5ieFy15YA","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Rationally_designed_meganucleases_with_a.pdf?id=b3MABwABERAJ\u0026output=pdf\u0026sig=ACfU3U1F6sPbK4wGDiRZmGZPTYC-K6HBWA"},"sample_url":"http://www.google.com/patents/reader?id=b3MABwABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>