<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US8153399 - Methods and compositions for inactivating glutamine synthetase gene expression - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_4ff636b3d23669b7103f3b3a3a18b4cd/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_4ff636b3d23669b7103f3b3a3a18b4cd__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Methods and compositions for inactivating glutamine synthetase gene expression"><meta name="DC.contributor" content="Pei-Qi Liu" scheme="inventor"><meta name="DC.contributor" content="Jeffrey C. Miller" scheme="inventor"><meta name="DC.contributor" content="Sangamo Biosciences, Inc." scheme="assignee"><meta name="DC.date" content="2009-10-29" scheme="dateSubmitted"><meta name="DC.description" content="Disclosed herein are methods and compositions for inactivating a glutamine synthetase (GS) gene, using fusion proteins comprising a zinc finger protein and a cleavage domain or cleavage half-domain. Polynucleotides encoding said fusion proteins are also provided, as are cells comprising said polynucleotides and fusion proteins."><meta name="DC.date" content="2012-4-10" scheme="issued"><meta name="DC.relation" content="GB:2338237" scheme="references"><meta name="DC.relation" content="US:20030232410:A1" scheme="references"><meta name="DC.relation" content="US:20040096452:A1" scheme="references"><meta name="DC.relation" content="US:20050026157:A1" scheme="references"><meta name="DC.relation" content="US:20050064474:A1" scheme="references"><meta name="DC.relation" content="US:20050208489:A1" scheme="references"><meta name="DC.relation" content="US:20050277132:A1" scheme="references"><meta name="DC.relation" content="US:20060063231:A1" scheme="references"><meta name="DC.relation" content="US:20060188987:A1" scheme="references"><meta name="DC.relation" content="US:20070134796:A1" scheme="references"><meta name="DC.relation" content="US:20070218528:A1" scheme="references"><meta name="DC.relation" content="US:20080015164:A1" scheme="references"><meta name="DC.relation" content="US:20080131962:A1" scheme="references"><meta name="DC.relation" content="US:20080159996:A1" scheme="references"><meta name="DC.relation" content="US:20080188000:A1" scheme="references"><meta name="DC.relation" content="US:20080299580:A1" scheme="references"><meta name="DC.relation" content="US:20090042250:A1" scheme="references"><meta name="DC.relation" content="US:20090305419:A1" scheme="references"><meta name="DC.relation" content="US:5356802" scheme="references"><meta name="DC.relation" content="US:5487994" scheme="references"><meta name="DC.relation" content="US:5789538" scheme="references"><meta name="DC.relation" content="US:5925523" scheme="references"><meta name="DC.relation" content="US:6007988" scheme="references"><meta name="DC.relation" content="US:6013453" scheme="references"><meta name="DC.relation" content="US:6140081" scheme="references"><meta name="DC.relation" content="US:6140466" scheme="references"><meta name="DC.relation" content="US:6200759" scheme="references"><meta name="DC.relation" content="US:6242568" scheme="references"><meta name="DC.relation" content="US:6410248" scheme="references"><meta name="DC.relation" content="US:6453242" scheme="references"><meta name="DC.relation" content="US:6479626" scheme="references"><meta name="DC.relation" content="US:6534261" scheme="references"><meta name="DC.relation" content="US:6607882" scheme="references"><meta name="DC.relation" content="US:6824978" scheme="references"><meta name="DC.relation" content="US:6903185" scheme="references"><meta name="DC.relation" content="US:6933113" scheme="references"><meta name="DC.relation" content="US:6979539" scheme="references"><meta name="DC.relation" content="US:7013219" scheme="references"><meta name="DC.relation" content="US:7153949" scheme="references"><meta name="DC.relation" content="US:7163824" scheme="references"><meta name="DC.relation" content="WO:1995019431:A1" scheme="references"><meta name="DC.relation" content="WO:1996006166:A1" scheme="references"><meta name="DC.relation" content="WO:1998037186:A1" scheme="references"><meta name="DC.relation" content="WO:1998053057:A1" scheme="references"><meta name="DC.relation" content="WO:1998053058:A1" scheme="references"><meta name="DC.relation" content="WO:1998053059:A1" scheme="references"><meta name="DC.relation" content="WO:1998053060:A1" scheme="references"><meta name="DC.relation" content="WO:1998054311:A1" scheme="references"><meta name="DC.relation" content="WO:2000027878:A1" scheme="references"><meta name="DC.relation" content="WO:2001060970:A2" scheme="references"><meta name="DC.relation" content="WO:2001088197:A2" scheme="references"><meta name="DC.relation" content="WO:2002016536:A1" scheme="references"><meta name="DC.relation" content="WO:2002077227:A2" scheme="references"><meta name="DC.relation" content="WO:2002099084:A2" scheme="references"><meta name="DC.relation" content="WO:2003016496:A2" scheme="references"><meta name="DC.relation" content="WO:2007014275:A2" scheme="references"><meta name="citation_reference" content="Almassy, et al., &quot;Novel Subunit-Subunit Interactions in the Structure of Glutamine Synthetase,&quot; Nature 323:304-309 (1986)."><meta name="citation_reference" content="Almassy, et al., “Novel Subunit—Subunit Interactions in the Structure of Glutamine Synthetase,” Nature 323:304-309 (1986)."><meta name="citation_reference" content="Ashworth, et al., &quot;Computational Redesign of Endonuclease DNA Binding and Cleavage Specificity,&quot; Nature 441:656-659 (2006)."><meta name="citation_reference" content="Beerli, et al., &quot;Engineering Polydactyl Zinc-Finger Transcription Factors,&quot; Nature Biotechnology 20:135-141 (2002)."><meta name="citation_reference" content="Bibikova, et al., &quot;Enhancing Gene Targeting With Designed Zinc Finger Nucleases,&quot; Science 300:764 (2003)."><meta name="citation_reference" content="Bibikova, et al., &quot;Stimulation of Homologous Recombination Through Targeted Cleavage by Chimeric Nucleases,&quot; Mol Cell Biol 21:289-297 (2001)."><meta name="citation_reference" content="Bibikova, et al., &quot;Targeted Chromosomal Cleavage and Mutagenesis in Drosophila Using Zinc-Finger Nucleases,&quot; Genetics 161:1169-1175 (2002)."><meta name="citation_reference" content="Bitinate, et al., &quot;Foki Dimerization Is Required for DNA Cleavage,&quot; PNAS USA 95:10570-10575 (1998)."><meta name="citation_reference" content="Chevalier, et al., &quot;Design, Activity, and Structure of a Highly Specific Artificial Endonuclease,&quot; Molecular Cell 10:895-905 (2002)."><meta name="citation_reference" content="Choo, et al., &quot;Advances in Zinc Finger Engineering,&quot; Curr. Opin. Struct. Biol. 10:411-416 (2000)."><meta name="citation_reference" content="Epinat, et al., &quot;A Novel Engineered Meganuclease Induces Homologous Recombination in Yeast and Mammalian Cells,&quot; Nucleic Acids Research 31:2952-2962 (2003)."><meta name="citation_reference" content="Gill, et al., &quot;Multicopy Crystallographic Refinement of a Relaxed Glutamine Synthetase From Mycobacterium tuberculosis Highlights Flexible Loops in the Enzymatic Mechanism and Its Regulation,&quot; Biochem 41:9863-9872 (2002)."><meta name="citation_reference" content="Isalan et al., &quot;A Rapid, Generally Applicable Method to Engineer Zinc Fingers Illustrated by Targeting the HIV-1 Promoter,&quot; Nat Biotechnol 19:656-660 (2001)."><meta name="citation_reference" content="Kim, et al., &quot;Chimeric Restriction Endonuclease,&quot; PNAS USA 91:883-887 (1994)."><meta name="citation_reference" content="Kim, et al., &quot;Insertion and Deletion Mutants of FOKI Restriction Endonuclease,&quot; J. Biol. Chem. 269:31978-31981 (1994)."><meta name="citation_reference" content="Kohli, et al., &quot;Facile Methods for Generating Human Somatic Cell Gene Knockouts Using Recombinant Adeno-Associated Viruses,&quot; Nucleic Acids Research 32:E3 (2004)."><meta name="citation_reference" content="Li, et al., &quot;Alteration of the Cleavage Distance of FOK I Restriction Endonuclease by Insertion Mutagenesis,&quot; PNAS USA 90:2764-2768 (1993)."><meta name="citation_reference" content="Li, et al., &quot;Functional Domains in FOK I Restriction Endonuclease,&quot; PNAS USA 89:4275-4279 (1992)."><meta name="citation_reference" content="Liaw, et al., &quot;Structural Model for the Reaction Mechanism of Glutamine Synthetase, Based on Five Crystal Structures of Enzyme-Substrate Complexes,&quot; Biochem 33:675-681 (1994)."><meta name="citation_reference" content="Lloyd, et al., &quot;Targeted Mutagenesis Using Zinc-Finger Nucleases in Arabidopsis&quot; PNAS USA 102:2232-2237 (2005)."><meta name="citation_reference" content="Lombardo, et al., &quot;Gene Editing in Human Stem Cells Using Zinc Finger Nucleases and Integrase-Defective Lentiviral Vector Delivery,&quot; Nature Biotechnology 25:1298-1306 (2007)."><meta name="citation_reference" content="Mansour, et al., &quot;Disruption of the Proto-Oncogene Int-2 in Mouse Embryo-Derived Stem Cells: A General Strategy for Targeting Mutations to Non-Selectable Genes,&quot; Nature 336:348-352 (1988)."><meta name="citation_reference" content="Miller, et al., &quot;An Improved Zinc-Finger Nuclease Architecture for Highly Specific Genome Editing,&quot; Nature Biotech 25:778-785 (2007)."><meta name="citation_reference" content="Moehle, et al., &quot;Targeted Gene Addition Into a Specified Location in the Human Genome Using Designed Zinc Finger Nucleases,&quot; PNAS USA 104:3055-3060 (2007)."><meta name="citation_reference" content="Morton, et al., &quot;Induction and Repair of Zinc-Finger Nuclease-Targeted Double-Strand Breaks in Caenorhabditis Elegans Somatic Cells,&quot; PNAS USA 103:16370-16375 (2006)."><meta name="citation_reference" content="Pabo, et al., &quot;Design and Selection of Novel CYS2-HIS2 Zinc Finger Proteins,&quot; Ann. Rev. Biochem. 70:313-340 (2001)."><meta name="citation_reference" content="Paques, et al., &quot;Meganucleases and DNA Double-Strand Break-Induced Recombination: Perspectives for Gene Therapy,&quot; Current Gene Therapy 7:49-66 (2007)."><meta name="citation_reference" content="Perez, et al., &quot;Establishment of HIV-1 Resistance in CD4+ T Cells by Genome Editing Using Zinc-Finger Nucleases,&quot; Nature Biotechnology 26:808-816 (2008)."><meta name="citation_reference" content="Porteus, et al., &quot;Gene Targeting Using Zinc Finger Nucleases,&quot; Nature Biotechnology 23:967-973 (2005)."><meta name="citation_reference" content="Rago, et al., &quot;Genetic Knockouts and Knockins in Human Somatic Cells,&quot; Nature Protocols 2:2734- 2746 (2007)."><meta name="citation_reference" content="Santiago, et al., &quot;Targeted Gene Knockout in Mammalian Cells by Using Engineered Zinc Finger Nucleases,&quot; PNAS USA 105:5809-5814 (2008)."><meta name="citation_reference" content="Segal, et al., &quot;Custom DNA-Binding Proteins Come of Age: Polydactyl Zinc-Finger Proteins,&quot; Curr. Opin. Biotechnol. 12:632-637 (2001)."><meta name="citation_reference" content="Umov, et al., &quot;Highly Efficient Endogenous Human Gene Correction Using Designed Zinc-Finger Nucleases,,&quot; Nature 435:646-651 (2005)."><meta name="citation_reference" content="Vasquez, et al., &quot;Manipulating the Mammalian Genome by Homologous Recombination,&quot; PNAS USA 98:8403-8410 (2001)."><meta name="citation_reference" content="Wurm, et al., &quot;Production of Recombinant Protein Therapeutics in Cultivated Mammalian Cells,&quot; Nature Biotechnology 22:1393-1398 (2004)."><meta name="citation_patent_number" content="US:8153399"><meta name="citation_patent_application_number" content="US:12/589,884"><link rel="canonical" href="http://www.google.com/patents/US8153399"/><meta property="og:url" content="http://www.google.com/patents/US8153399"/><meta name="title" content="Patent US8153399 - Methods and compositions for inactivating glutamine synthetase gene expression"/><meta name="description" content="Disclosed herein are methods and compositions for inactivating a glutamine synthetase (GS) gene, using fusion proteins comprising a zinc finger protein and a cleavage domain or cleavage half-domain. Polynucleotides encoding said fusion proteins are also provided, as are cells comprising said polynucleotides and fusion proteins."/><meta property="og:title" content="Patent US8153399 - Methods and compositions for inactivating glutamine synthetase gene expression"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("OGroU9OpO8HKsQSeo4DQCA"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407291699.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("NLD"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("OGroU9OpO8HKsQSeo4DQCA"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407291699.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("NLD"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us8153399?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US8153399"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=sy04BwABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS8153399&amp;usg=AFQjCNFnlKlNtrCgEB9Lz-gQbUssLvJq9Q" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US8153399.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US8153399.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20100129869"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US8153399"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US8153399" style="display:none"><span itemprop="description">Disclosed herein are methods and compositions for inactivating a glutamine synthetase (GS) gene, using fusion proteins comprising a zinc finger protein and a cleavage domain or cleavage half-domain. Polynucleotides encoding said fusion proteins are also provided, as are cells comprising said polynucleotides...</span><span itemprop="url">http://www.google.com/patents/US8153399?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US8153399 - Methods and compositions for inactivating glutamine synthetase gene expression</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US8153399 - Methods and compositions for inactivating glutamine synthetase gene expression" title="Patent US8153399 - Methods and compositions for inactivating glutamine synthetase gene expression"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US8153399 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 12/589,884</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Apr 10, 2012</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Oct 29, 2009</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Oct 29, 2008</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2741119A1">CA2741119A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CN102264915A">CN102264915A</a>, </span><span class="patent-bibdata-value"><a href="/patents/CN102264915B">CN102264915B</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2344660A2">EP2344660A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2344660A4">EP2344660A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20100129869">US20100129869</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20100311124">US20100311124</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20130164785">US20130164785</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2010053518A2">WO2010053518A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2010053518A3">WO2010053518A3</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">12589884, </span><span class="patent-bibdata-value">589884, </span><span class="patent-bibdata-value">US 8153399 B2, </span><span class="patent-bibdata-value">US 8153399B2, </span><span class="patent-bibdata-value">US-B2-8153399, </span><span class="patent-bibdata-value">US8153399 B2, </span><span class="patent-bibdata-value">US8153399B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Pei-Qi+Liu%22">Pei-Qi Liu</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Jeffrey+C.+Miller%22">Jeffrey C. Miller</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Sangamo+Biosciences,+Inc.%22">Sangamo Biosciences, Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US8153399.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8153399.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8153399.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (56),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (35),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (16),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (3)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/8153399&usg=AFQjCNFrH0S18YEA1PVZHkwjK90FYFZ3SA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D8153399&usg=AFQjCNG4DPafgg9UlUQU35j2eYhBI7JrfQ">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D8153399B2%26KC%3DB2%26FT%3DD&usg=AFQjCNFLVbqpg0fuob__2c4cnh6v_3EnPg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT110903605" lang="EN" load-source="patent-office">Methods and compositions for inactivating glutamine synthetase gene expression</invention-title></span><br><span class="patent-number">US 8153399 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA93930426" lang="EN" load-source="patent-office"> <div num="p-0001" class="abstract">Disclosed herein are methods and compositions for inactivating a glutamine synthetase (GS) gene, using fusion proteins comprising a zinc finger protein and a cleavage domain or cleavage half-domain. Polynucleotides encoding said fusion proteins are also provided, as are cells comprising said polynucleotides and fusion proteins.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(26)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00013.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00013.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00014.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00014.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00015.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00015.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00016.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00016.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00017.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00017.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00018.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00018.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00019.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00019.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00020.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00020.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00021.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00021.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00022.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00022.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00023.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00023.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00024.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00024.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00025.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00025.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US8153399B2/US08153399-20120410-D00026.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US8153399B2/US08153399-20120410-D00026.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(18)</span></span></div><div class="patent-text"><div mxw-id="PCLM57507209" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is:</claim-statement> <div class="claim"> <div id="CLM-00001" num="00001" class="claim">
      <div class="claim-text">1. A zinc finger DNA-binding domain comprising four, five or six zinc finger recognition regions designated F1 to F4, F1-F5 or F1 to F-6 as shown in a single row of Table 1.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00002" num="00002" class="claim">
      <div class="claim-text">2. A fusion protein comprising a zinc finger DNA-binding domain according to <claim-ref idref="CLM-00001">claim 1</claim-ref> and at least one cleavage domain or at least one cleavage half-domain.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00003" num="00003" class="claim">
      <div class="claim-text">3. The fusion protein of <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein the cleavage half-domain is a wild-type FokI cleavage half-domain.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00004" num="00004" class="claim">
      <div class="claim-text">4. The fusion protein of <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein the cleavage half-domain is an engineered FokI cleavage half-domain.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00005" num="00005" class="claim">
      <div class="claim-text">5. A polynucleotide encoding the zinc finger DNA-binding domain according to <claim-ref idref="CLM-00001">claim 1</claim-ref>.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00006" num="00006" class="claim">
      <div class="claim-text">6. An isolated cell comprising a protein according to <claim-ref idref="CLM-00001">claim 1</claim-ref>.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00007" num="00007" class="claim">
      <div class="claim-text">7. A cell line in which glutamine synthetase (GS) is partially or fully inactivated by a fusion protein according to <claim-ref idref="CLM-00002">claim 2</claim-ref>.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00008" num="00008" class="claim">
      <div class="claim-text">8. A method of inactivating an endogenous cellular GS gene in a cell, the method comprising:
<div class="claim-text">introducing, into a cell, a first nucleic acid encoding a first polypeptide, wherein the first polypeptide comprises a fusion protein according to <claim-ref idref="CLM-00002">claim 2</claim-ref>:
<div class="claim-text">such that the polypeptide is expressed in the cell, whereby the polypeptide binds to its target site and cleaves the GS gene.</div>
</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00009" num="00009" class="claim">
      <div class="claim-text">9. The method of <claim-ref idref="CLM-00008">claim 8</claim-ref>, further comprising introducing a nucleic acid encoding a second polypeptide, wherein the second polypeptide comprises:
<div class="claim-text">(i) a zinc finger DNA-binding domain that is engineered to bind to a second target site in the GS gene; and</div>
<div class="claim-text">(ii) a cleavage domain; such that the second polypeptide is expressed in the cell, whereby the first and second polypeptides bind to their respective target sites and cleave the GS gene.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00010" num="00010" class="claim">
      <div class="claim-text">10. The method of <claim-ref idref="CLM-00009">claim 9</claim-ref>, wherein the first and second polypeptides are encoded by the same nucleic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00011" num="00011" class="claim">
      <div class="claim-text">11. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref>, wherein the first and second polypeptides are encoded by different nucleic acids.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00012" num="00012" class="claim">
      <div class="claim-text">12. The method of <claim-ref idref="CLM-00008">claim 8</claim-ref>, further comprising inactivating a DHFR gene in the cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00013" num="00013" class="claim">
      <div class="claim-text">13. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref>, further comprising inactivating a FUT8 gene in the cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00014" num="00014" class="claim">
      <div class="claim-text">14. A method of producing a recombinant protein of interest in a host cell, the method comprising the steps of:
<div class="claim-text">(a) providing a host cell comprising an endogenous GS gene;</div>
<div class="claim-text">(b) inactivating the endogenous GS gene of the host cell by the method of <claim-ref idref="CLM-00008">claim 8</claim-ref>; and</div>
<div class="claim-text">(c) introducing an expression vector comprising a transgene, the transgene comprising a sequence encoding a protein of interest into the host cell, thereby producing the recombinant protein.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00015" num="00015" class="claim">
      <div class="claim-text">15. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref>, wherein the protein of interest comprises α1-antitrypsin.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00016" num="00016" class="claim">
      <div class="claim-text">16. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref>, wherein the protein of interest is a monoclonal antibody.</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00017" num="00017" class="claim">
      <div class="claim-text">17. A cell line in which a GS gene is partially or fully inactivated, wherein the cell line is produced by
<div class="claim-text">(a) inactivating the GS gene in a cell according to the method of <claim-ref idref="CLM-00008">claim 8</claim-ref>; and</div>
<div class="claim-text">(b) culturing the cell under conditions suitable for generating a cell line in which the GS gene is partially or fully inactivated.</div>
</div>
    </div>
    </div> <div class="claim-dependent"> <div id="CLM-00018" num="00018" class="claim">
      <div class="claim-text">18. The cell line of <claim-ref idref="CLM-00015">claim 15</claim-ref>, wherein the cell is a mammalian cell selected from the group consisting of a COS cell, a CHO cell, a VERO cell, a MDCK cell, a WI38 cell, a V79 cell, a B14AF28-G3 cell, a BHK cell, a HaK cell, a NS0 cell, a SP2/0-Ag14 cell, a HeLa cell, an HEK293 cell, and a perC6 cell.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES45502821" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS-REFERENCE TO RELATED APPLICATIONS</heading> <p num="p-0002">The present application claims the benefit of U.S. Provisional Application No. 61/197,600, filed Oct. 29, 2008, the disclosure of which is hereby incorporated by reference in its entirety.</p>
    <heading>STATEMENT OF RIGHTS TO INVENTIONS MADE UNDER FEDERALLY SPONSORED RESEARCH</heading> <p num="p-0003">Not applicable.</p>
    <heading>TECHNICAL FIELD</heading> <p num="p-0004">The present disclosure is in the fields of genome engineering, cell culture, generation of cell lines and protein production.</p>
    <heading>BACKGROUND</heading> <p num="p-0005">Glutamine synthetase (GS) is a critical enzyme in the synthesis of the amino acid L-glutamine. See, Meister, A. in <i>Glutamine Metabolism, Enzymology and Regulation </i>(eds. J. Mora &amp; R. Palacios) 1-40 (Academic Press, N.Y.; 1980). A GS-negative cell line is therefore auxotrophic for L-glutamine. GS is frequently used as a selection marker gene in CHO cell based recombinant protein expression systems (Wurm et al. (2004)<i>Nature Biotechnology </i>22: 1393-1398), though the absence of a GS-negative CHO line requires the use of the GS inhibitor methionine sulfoximine to achieve selection.</p>
    <p num="p-0006">In addition, dihydrofolate reductase (DHFR, 5,6,7,8-tetrahydrofolate:NADP+oxidoreductase) is an essential enzyme in both eukaryotes and prokaryotes and catalyzes the NADPH-dependent reduction of dihydrofolate to tetrahydrofolate, an essential carrier of one-carbon units in the biosynthesis of thymidylate, purine nucleotides, glycine and methyl compounds.</p>
    <p num="p-0007">DHFR-deficient cells have long been used for production of recombinant proteins. DHFR-deficient cells will only grow in medium supplemented by certain factors involved in folate metabolism or if DHFR is provided to the cell, for example as a transgene. Cells into which a DHFR transgene has been stably integrated can be selected for by growing the cells in unsupplemented medium. Moreover, exogenous sequences are typically co-integrated when introduced into a cell using a single polynucleotide. Accordingly, when the DHFR transgene also includes a sequence encoding a protein of interest, selected cells will express both DHFR and the protein of interest. Furthermore, in response to inhibitors such as methotrexate (MTX), the DHFR gene copy number can be amplified. Accordingly, sequences encoding a protein of interest that are co-integrated with exogenous DHFR can be amplified by gradually exposing the cells to increasing concentrations of methotrexate, resulting in overexpression of the recombinant protein of interest. However, despite the wide use of DHFR-deficient cell systems for recombinant protein expression, currently available DHFR-deficient cell lines do not grow as well as the parental DHFR-competent cells from which they are derived.</p>
    <p num="p-0008">Thus, mammalian cells with single and multi-gene knockouts have enormous utility in research, drug discovery, and cell-based therapeutics. However, conventional methods for the targeted elimination of an investigator-specified gene rely upon the process of homologous recombination or gene targeting. Mansour et al. (1988) <i>Nature </i>336:348-352; Vasquez et al. (2001) <i>Proc Natl Acad Sci USA </i>98:8403-8410; Rago et al. (2007) <i>Nature Protocols </i>2:2734-2746; Kohli et al. (2004) <i>Nucleic Acids Research </i>32, e3. While capable of generating a defined biallelic knockout, for many cell types this technique has proven too inefficient and thus too laborious for routine application. See, e.g, Yamane-Ohnuki et al. (2004) <i>Biotechnol Bioeng </i>87:614-622. These methods for targeted gene deletion require sequential rounds of homologous recombination and drug selection to isolate rare desired events—a process sufficiently laborious to limit application to individual loci. Consequently, the generation of mammalian cell lines modified at multiple target loci has been largely unexplored.</p>
    <p num="p-0009">Zinc-finger nucleases (ZFNs) have been used for targeted cleavage and gene inactivation. See, for example, United States Patent Publications 20030232410; 20050208489; 20050026157; 20050064474; 20060188987; 20060063231; 2008/0015164 and U.S. Ser. No. 12/218,035 and International Publication WO 07/014,275, the disclosures of which are incorporated by reference in their entireties for all purposes. Formed via the fusion of an engineered zinc-finger DNA binding domain specific for a designated target sequence and the catalytic domain of Fok I (a restriction endonuclease from <i>Flavobacterium okeanokoites</i>), ZFNs provide the ability to place a double-strand DNA break (DSB) at a chosen genomic address. The removal of this site-specific DSB is carried out by the cell's own DNA repair machinery either via a homology-directed repair process when donor DNA is provided, or via non-homologous end joining (NHEJ). See, e.g., Urnov et al. (2005) <i>Nature </i>435:646-651 (2005); Moehle et al. (2007) <i>Proc Natl Acad Sci USA </i>104:3055-3060 (2007); Bibikova, et al. (2001) <i>Mol Cell Biol </i>21:289-297; Bibikova et al. (2003) <i>Science </i>300:764; Porteus et al. (2005) <i>Nature Biotechnology </i>23:967-973; Lombardo et al. (2007) <i>Nature Biotechnology </i>25:1298-1306; Perez et al. (2008) <i>Nature Biotechnology </i>26:808-816; Bibikova et al. (2002) <i>Genetics </i>161:1169-1175; Lloyd et al. (2005) <i>Proc Natl Acad Sci USA </i>102:2232-2237; Morton et al. (2006) <i>Proc Natl Acad Sci USA </i>103:16370-16375.</p>
    <p num="p-0010">While both homology-directed repair and NHEJ processes result in the modification of the target locus, the NHEJ-driven approach obviates the need for donor DNA design and synthesis yet results in a high frequency of disrupted alleles and the error-prone nature of the NHEJ-mediated DSB repair can be exploited to achieve the knockout of a targeted gene in mammalian cells following simple transient transfection of a DNA construct encoding the ZFNs. See, e.g., Santiago et al. (2008) <i>Proc Natl Acad Sci USA </i>105:5809-5814. ZFN technology has allowed the isolation of several independent knockout cell lines from a screening effort of less than one 96-well plate of single-cell derived clones. As no donor DNA or selection strategy was employed, the resultant single-gene knock out line is a suitable starting cell line for subsequent genetic modification.</p>
    <heading>SUMMARY</heading> <p num="p-0011">Disclosed herein are compositions for the partial or complete inactivation of a GS gene. Also disclosed herein are methods of making and using these compositions (reagents), for example to inactivate GS in a cell to produce cell lines in which a GS gene is inactivated. GS disrupted cell lines are useful, for example, in production of recombinant proteins.</p>
    <p num="p-0012">In one aspect, zinc finger proteins, engineered to bind in a GS gene, are provided. Any of the zinc finger proteins described herein may include 1, 2, 3, 4, 5, 6 or more zinc fingers, each zinc finger having a recognition helix that binds to a target subsite in a GS gene. In certain embodiments, the zinc finger proteins comprise 4, 5 or 6 fingers (wherein the individual zinc fingers are designated F1, F2, F3, F4, F5 and F6) and comprise the amino acid sequence of the recognition helices shown in Table 1.</p>
    <p num="p-0013">In another aspect, zinc finger proteins, engineered to bind in a DHFR gene, are provided. Any of the zinc finger proteins described herein may include 1, 2, 3, 4, 5, 6 or more zinc fingers, each zinc finger having a recognition helix that binds to a target subsite in a DHFR gene. In certain embodiments, the zinc finger proteins comprise 4, 5 or 6 fingers (wherein the individual zinc fingers are designated F1, F2, F3, F4, F5 and F6) and comprise the amino acid sequence of the recognition helices shown in Table 2.</p>
    <p num="p-0014">In another aspect, fusion proteins comprising any of the zinc finger proteins described herein and at least one cleavage domain or at least one cleavage half-domain, are also provided. In certain embodiments; the cleavage half-domain is a wild-type FokI cleavage half-domain. In other embodiments, the cleavage half-domain is an engineered FokI cleavage half-domain.</p>
    <p num="p-0015">In yet another aspect, a polynucleotide encoding any of the proteins described herein is provided.</p>
    <p num="p-0016">In yet another aspect, also provided is an isolated cell comprising any of the proteins and/or polynucleotides as described herein. In certain embodiments, GS is inactivated (partially or fully) in the cell. Any of the cells described herein may include additional genes that have been inactivated, for example, using zinc finger nucleases designed to bind to a target site in the selected gene. In certain embodiments, provided herein are cells or cell lines in which FUT8, dihydrofolate reductase (DHFR) and glutamine synthetase (GS) have been inactivated.</p>
    <p num="p-0017">In addition, methods of using the zinc finger proteins and fusions thereof in methods of inactivating GS in a cell or cell line are provided.</p>
    <p num="p-0018">Thus, in another aspect, provided herein is a method for inactivating a cellular GS gene (e.g., an endogenous GS gene) in a cell, the method comprising: (a) introducing, into a cell, a first nucleic acid encoding a first polypeptide, wherein the first polypeptide comprises: (i) a zinc finger DNA-binding domain that is engineered to bind to a first target site in an endogenous GS gene; and (ii) a cleavage domain; such that the polypeptide is expressed in the cell, whereby the polypeptide binds to the target site and cleaves the GS gene. In certain embodiments, the methods further comprise introducing a nucleic acid encoding a second polypeptide, wherein the second polypeptide comprises: (i) a zinc finger DNA-binding domain that is engineered to bind to a second target site in the GS gene; and (ii) a cleavage domain; such that the second polypeptide is expressed in the cell, whereby the first and second polypeptides bind to their respective target sites and cleave the GS gene. The first and second polypeptides may be encoded by the first nucleic acid or by different nucleic acids. In certain embodiments, one or more additional polynucleotides or polypeptides are introduced into the cells, for example polynucleotides encoding additional zinc finger proteins.</p>
    <p num="p-0019">In yet another aspect, the disclosure provides a method of producing a recombinant protein of interest in a host cell, the method comprising the steps of: (a) providing a host cell comprising an endogenous GS gene; (b) inactivating the endogenous GS gene of the host cell by any of the methods described herein; and (c) introducing an expression vector comprising a transgene, the transgene comprising a sequence encoding a protein of interest, into the host cell, thereby producing the recombinant protein. In certain embodiments, the protein of interest comprises an antibody, e.g., a monoclonal antibody.</p>
    <p num="p-0020">In any of the cells and methods described herein, the cell or cell line can be for example, but not limited to a COS, CHO (e.g., CHO-S, CHO-K1, CHO-DG44, CHO-DUXB11, CHO-DUKX, CHOK1SV), VERO, MDCK, WI38, V79, B14AF28-G3, BHK, HaK, NS0, SP2/0-Ag14, HeLa, HEK293 (e.g., HEK293-F, HEK293-H, HEK293-T), NIH3T3, perC6, insect cell such as <i>Spodoptera fugiperda </i>(Sf), or fungal cell such as <i>Saccharomyces, Pichia </i>and <i>Schizosaccharomyces. </i> </p>
    <description-of-drawings> <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p num="p-0021"> <figref idrefs="DRAWINGS">FIG. 1</figref>, panels A to E, show ZFN-mediated disruption of the glutamine synthetase gene in CHO cells and generation of single knockout GS−/− cell lines. <figref idrefs="DRAWINGS">FIG. 1A</figref> is a schematic depicting the actively-transcribed GS gene, which contains 7 exons. The start codon ATG located in exon 2. <figref idrefs="DRAWINGS">FIG. 1A</figref> also shows that the target sequence for ZFN pair ZFN9372/ZFN9075 is located in exon 6 as indicated and that the target sequence of ZFN pair 8361/8365 is located in exon 2. Additional ZFNs (9076, 9179, 7858, 7889, 9373) targeted to exon 6 are also shown. <figref idrefs="DRAWINGS">FIG. 1B</figref> are gels showing ZFN-mediated disruption of the endogenous CHO GS gene using ZFN pair 9372/9075 linked to either the wild-type or an obligate heterodimer EL/KK variant of the catalytic domain of Fok I in CHO-K1 cells (left panel) or CHO-S cells (right panel), as determined by the Surveyor™ Nuclease Assay. <figref idrefs="DRAWINGS">FIG. 1C</figref> (SEQ ID NOS: 58-73, respectively) depicts exemplary DNA sequences of the target GS locus for the indicated cell lines along with their growth properties in the absence of exogenously added L-glutamine (“L-Glu Growth”; ‘+’: growth in the absence of L-glutamine indistinguishable from wild-type CHO; ‘−’: no growth in the absence of L-glutamine). The ZFN target sequences are underlined. The protein translation is shown under the wild-type sequence. Capital letters indicate exonic sequences, small letters indicate intronic sequences, ‘−’ indicate deletions, bold letters indicate insertions. <figref idrefs="DRAWINGS">FIG. 1D</figref> depicts Western blot analysis of selected CHO-S (left panel) and CHO-K1 (right panel) cell lines using an anti-GS monoclonal antibody (top panels). As a loading control, blots were re-probed with an anti-DHFR antibody (bottom panels). <figref idrefs="DRAWINGS">FIG. 1E</figref> shows graphs depicting growth and viability of selected CHO-K1 GS−/− cell lines that had been grown for period of approximately 3 months. The viability (smooth line on the top of the graph) and viable cell density (jagged line) of these cell lines is shown for a 30 day period following the initial 3 month period, where cells were grown in L-glutamine-supplemented medium, splitting the cells every 2-3 days. L-glutamine was withdrawn at the time indicated by the arrow.</p>
      <p num="p-0022"> <figref idrefs="DRAWINGS">FIG. 2</figref>, panels A to F, depict ZFN-mediated disruption of the DHFR gene in CHO cells and generation of double knockout DHFR−/−GS−/− cell lines. <figref idrefs="DRAWINGS">FIG. 2A</figref> shows the genomic organization of the DHFR gene in CHO cells and the location of the target sites for ZFN pairs 9461/7844 (in exon 1, see, U.S. Patent Application No. 20080015164) and 9476/9477 (in intron 1 located 240-bp 3′ of the exon 1 ZFN 9461/7844 cleavage site). <figref idrefs="DRAWINGS">FIG. 2B</figref> depicts the level of gene modification using ZFN pair 9461/7844 (left panel) and ZFN pair 9476/9477 (right panel), as measured using the Surveyor™ Nuclease Assay. <figref idrefs="DRAWINGS">FIG. 2</figref> C depicts PCR analysis of GS−/− clone B3 genomic DNA 2 days after concurrent transfection of plasmids encoding both ZFN pairs, which resulted in a deletion of approximately 240 bp in the DHFR locus (small arrow in lane 7). Serving as controls were cells transfected with empty vector (Lane 1), a GFP control vector (Lane 2), the exonic ZFN pair only (ZFN9461/ZFN7844, Lane 3), the intronic ZFN pair only (ZFN9476/ZFN9477, Lane 4), the “inner” ZFNs relative to the deletion (ZFN7844 and ZFN9477, LaneS), and the “outer” ZFNs relative to the deletion (ZFN9461 and ZFN9476, Lane 6). <figref idrefs="DRAWINGS">FIG. 2D</figref> (SEQ ID NOS: 74-81, respectively) depicts results of DHFR genotyping analysis of single-cell lines. For homozygous clones, the common sequence for both alleles is shown. For compound heterozygous clones the sequence of each unique allele is shown. <figref idrefs="DRAWINGS">FIG. 2E</figref> depicts Western blot analysis with the indicated antibodies (DHFR, GS and β-tubulin) of whole cell lysates from the cell lines indicated above each lane. Cell lines 1F1.6 and 2B12.8 (DHFR/GS knockout) were analyzed while the parental GS−/− cell line B3, wild-type CHO cells (WT), and the DHFR-deficient CHO cell line DG44 (see Urlab et al. (1983) Cell 33:405-412) served as controls. <figref idrefs="DRAWINGS">FIG. 2F</figref> depicts growth of the 1F1.6 and 2B12.8 cell lines and the dependence of these lines on exogenously provided hypoxanthine, thymidine and glutamine.</p>
      <p num="p-0023"> <figref idrefs="DRAWINGS">FIG. 3</figref>, panels A to D, show ZFN-mediated disruption of the FUT8 gene in CHO cells and generation of triple knockout FUT8−/−DHFR−/−GS−/− cell lines. <figref idrefs="DRAWINGS">FIG. 3A</figref> is a schematic depicting ZFNs targeted to the critical and highly conserved region encoding the FUT8 Fut motif II located in exon 10. See, also, U.S. patent application Ser. No. 12/218,0135. <figref idrefs="DRAWINGS">FIG. 3B</figref> depicts the level of gene modification 2 days after transient transfection of the ZFN pair 12176/12172 into the DHFR−/−GS−/− clone 1F1.6, as measured using the Surveyor™ Nuclease Assay. <figref idrefs="DRAWINGS">FIG. 3C</figref> depicts. fluorescent-LCA binding activity of the triple-KO clones 35F2 and 14C1 (lines on left side of panel) by FACS analysis. F-LCA stained wild-type CHO-S cells (line on right side of panel) served as positive controls, and unstained cells (dotted line) served as negative controls. <figref idrefs="DRAWINGS">FIG. 3D</figref> (SEQ ID NOS: 82-87, respectively) shows the genotype (for both alleles) of the triple-knockout cell lines 35F2 and 14C1 at the FUT8 locus where the sequence shown is for both alleles. Capital letters indicate exon 10 sequences, small letters indicates intron sequences, italic letters indicate Fut motif II sequences, bold letters indicate sequence insertions, ‘−’ indicates deletions. The protein translation is also shown under the wild-type sequence. Underline indicates the ZFN binding sites.</p>
      <p num="p-0024"> <figref idrefs="DRAWINGS">FIG. 4</figref>, panels A to C, depict an exemplary ZFN design and its target site in the CHO GS gene. <figref idrefs="DRAWINGS">FIG. 4A</figref> is a schematic representation of the functional intron-containing glutamine synthetase gene in CHO. cells. It contains 7 exons, the start codon ATG is located in exon 2, and the target sequence of ZFN pair ZFN9372/ZFN9075, located in exon 6, is also indicated. The nucleotide sequence (SEQ ID NO: 88) of the region of interest around exon 6 is shown. Capital letters indicate exon 6 sequences, small letters indicate intron sequences. The target sequences of ZFN9372 and ZFN9075 are underlined. <figref idrefs="DRAWINGS">FIG. 4B</figref> (SEQ ID NOS: 89 and 90 is a schematic representation of ZFN9372 and ZFN9075 binding to their double-stranded target sequence (underlined). <figref idrefs="DRAWINGS">FIG. 4C</figref> depicts the target sequences and zinc-finger designs of ZFN9372 (SEQ ID NOS: 6-12) and ZFN9075 (SEQ ID NOS: 1-5). The core DNA target sequences (capital letters) and 2 flanking bases (small letters) are shown. ZFN9075 contains 4 zinc-finger DNA-binding domains, and ZFN9372 contains 6 zinc-finger domains. The amino acid residues at positions ‘−1’ to ‘+6’ of the recognition a-helix of each of the zinc-finger DNA-binding domains for the indicated target DNA triplets are shown.</p>
      <p num="p-0025"> <figref idrefs="DRAWINGS">FIG. 5</figref> (SEQ ID NOS: 91-132, respectively) shows exemplary results of sequencing of the genomic GS locus from ZFN transfected cells. “C” refers to the count (number of times the indicated sequence was observed) and “G” refers to the genotype. The ZFN target sequences are underlined. Bold letters indicate sequence insertions, ‘−’ indicates deletions.</p>
      <p num="p-0026"> <figref idrefs="DRAWINGS">FIG. 6</figref> is a graph depicting L-glutamine-dependent growth of a homozygous GS<sup>−/−</sup> cell line (Clone B3). In the presence of L-glutamine, CHO-S GS<sup>−/−</sup> clone B3 (triangle) grows as well as wild-type CHO-S (diamond). In the absence of exogenous L-glutamine, the Clone B3 (circle) stopped growing and all cells died within 4 days, whereas the wild-type CHO cells (square) continued to grow at a reduced rate.</p>
      <p num="p-0027"> <figref idrefs="DRAWINGS">FIG. 7</figref>, panels A to C, depict exemplary ZFN designs and their target sites in the CHO DHFR gene. <figref idrefs="DRAWINGS">FIG. 7A</figref> (SEQ ID NO: 133) shows the nucleotide sequence of the DHFR region targeted by the indicated ZFN pairs in CHO cells. Capital letters indicate exon sequences, small letters indicate intron sequences. <figref idrefs="DRAWINGS">FIG. 7B</figref> (SEQ ID NOS: 134-137, respectively) is a schematic representation of the ZFNs targeting DHFR when the ZFNs are bound to their double-stranded target sequences (underlined). The two ZFNs labeled “inner” target sequences are internal to the region to be deleted, while the two ZFNs labeled “outer” target sequences are outside of the expected deletion junction. <figref idrefs="DRAWINGS">FIG. 7C</figref> shows the target sequences (SEQ ID NOS: 33, 32, 38, and 41, respectively) and zinc-finger designs (SEQ ID NOS: 37, 36, 35, 34, 25, 2, 3, 14, 40, 14, 39, 2, 43, 42, 17, and 7, respectively) of exemplary ZFNs targeting the CHO DHFR gene. The core DNA target sequences (capital letters) and 2 flanking bases (small letters) are shown. All ZFNs contain 4 zinc-finger DNA-binding domains. The amino acid residues at positions ‘−1’ to ‘−6’ of the recognition α-helix for each of the zinc-finger DNA-binding domains for the indicated target DNA triplet are shown.</p>
      <p num="p-0028"> <figref idrefs="DRAWINGS">FIG. 8</figref> (SEQ ID NOS: 138-150, respectively) shows exemplary results of sequencing of the genomic DHFR locus from ZFN transfected cells. “C” refers to the count (number of times the indicated sequence was observed) and “G” refers to the genotype. The ZFN target sequences are underlined. Bold letters indicate sequence insertions, ‘−’ indicates deletions. Bold letters indicate sequence insertions. Italics indicate sequence changes. Capital letters indicate exonic sequence, small letters indicate intronic sequence.</p>
      <p num="p-0029"> <figref idrefs="DRAWINGS">FIG. 9</figref>, panels A to C, depict ZFN target sites and finger designs targeted to the CHO FUT8 gene. <figref idrefs="DRAWINGS">FIG. 9A</figref> (SEQ ID NO: 151) depicts a schematic of the ZFN binding sites within exon 10 of the FUT8 gene as well as the nucleotide sequence of the target region within exon 10. Capital letters indicate exon 10 sequences, small letters indicate intronic sequences. The target sequences of ZFN12176 and ZFN12172 are underlined. The nucleotides comprising the fucosyltransferase motif II are shown in italics. <figref idrefs="DRAWINGS">FIG. 9B</figref> is a schematic representation of ZFN12176 (SEQ ID NO: 153) and ZFN12172 (SEQ ID NO: 152) binding to their double-stranded target sequence (underlined). <figref idrefs="DRAWINGS">FIG. 9C</figref> depicts the target sequences (SEQ ID NOS: 154 and 155) and zinc-finger designs (SEQ ID NOS: 54, 53, 52, 51, 3, 49, 48, 47, 9, 46, and 45, respectively) of ZFN12176 and ZFN12172. The core DNA target sequences (capital letters) and 2 flanking bases (small letters) are shown. ZFN12176 contains 5 zinc-finger DNA-binding domains, and ZFN12172 contains 6 zinc-finger domains. The amino acid residues at positions ‘−1’ to ‘+6’ of the recognition α-helix for each of the zinc-finger DNA-binding domains for the indicated target DNA triplet are shown.</p>
      <p num="p-0030"> <figref idrefs="DRAWINGS">FIG. 10</figref>, panels A to C, show ZFN targeting of CCR5, GR and AAVS1 loci. <figref idrefs="DRAWINGS">FIG. 10A</figref> is a schematic showing the location of ZFN targeting sites in C-C chemokine receptor 5 (CCR5), glucocorticoid receptor (GR), and adeno-associated virus integration site (AAVS1) loci, respectively. <figref idrefs="DRAWINGS">FIG. 10B</figref> depicts ZFN-mediated simultaneous disruption, as measured by Surveyor™ Nuclease Assay, of CCR5 (left panel), GR (middle panel), and AAVS1 (right panel) loci as measured 20 days after transient co-transfection of pairs of ZFNs linked to either the wild-type, ZFN-Fok I (wt) or an obligate heterodimer EL/KK variant, ZFN-Fok I(EL/KK), of the catalytic domain of Fok into K562 cells. Treatments for each lane are as following: lanel, CCR5 ZFN-FokI (wt); lane 2, GR ZFN-Fok I(wt); lane 3, AAVS1 ZFN-Fok I(wt); lane 4, CCR5 ZFN-Fok I(EL/KK); lane 5, GR ZFN-Fok I(EL/KK); lane 6, AAVS1 ZFN-Fok I(EL/KK); lane 7, CCR5 ZFN-FokI(wt)+GR ZFN-FokI(wt)+AAVS1 ZFN-FokI(wt); lane 8, CCR5 ZFN-FokI(EL/KK)+GR ZFN-FokI(EL/KK)+AAVS1 ZFN-FokI(EL/KK). <figref idrefs="DRAWINGS">FIG. 10C</figref> (SEQ lD NOS: 156-165, respectively) depicts the genotype of a single triple knockout clone wherein the target CCR5, GR, AAVS1 loci for the indicated cell lines are shown. The ZFN target sequences are underlined. ‘−’ indicate deletions, bold letters indicate insertions.</p>
      <p num="p-0031"> <figref idrefs="DRAWINGS">FIG. 11</figref>, panels A and B, show sequence alignments of CHO GS genomic sequences as determined herein with the indicated species. <figref idrefs="DRAWINGS">FIG. 11A</figref> (SEQ ID NOS: 166-174, respectively) shows alignment of exon 2 sequence. The target sites for ZFN8361 and ZFN8365 are underlined. <figref idrefs="DRAWINGS">FIG. 11B</figref> (SEQ ID NOS: 175-185, respectively) shows alignment of exon 6 sequence. The target sites for ZFN9075 and ZFN9372 are underlined.</p>
      <p num="p-0032"> <figref idrefs="DRAWINGS">FIG. 12</figref>, panels A to D, show ZFN-mediated disruption in human and mouse cells. <figref idrefs="DRAWINGS">FIGS. 12A</figref> and B show disruption of GS in human K562 mediated by ZFNs targeted to exon 2 of CHO GS (<figref idrefs="DRAWINGS">FIG. 12A</figref>) and exon 6 of CHO GS (<figref idrefs="DRAWINGS">FIG. 12B</figref>). <figref idrefs="DRAWINGS">FIGS. 12C</figref> and D show disruption of GS in mouse Neuro2a cells mediated by ZFNs targeted to exon 2 of CHO GS (<figref idrefs="DRAWINGS">FIG. 12C</figref>) and exon 6 of CHO GS (<figref idrefs="DRAWINGS">FIG. 12D</figref>).</p>
      <p num="p-0033"> <figref idrefs="DRAWINGS">FIG. 13</figref>, panels A to C, show the complete genomic sequence of CHO GS locus (SEQ ID NO:55). Introns and exons are designated as shown.</p>
      <p num="p-0034"> <figref idrefs="DRAWINGS">FIG. 14</figref>, panels A to C, show the results of GS-specific gene targeting in HEK293 cells. <figref idrefs="DRAWINGS">FIG. 14A</figref> depicts the percent of NHEJ activity in cells treated with GS-specific ZFNs (labeled GS) in comparison with cells transfected with a GFP donor molecule (labeled GFP). <figref idrefs="DRAWINGS">FIG. 14B</figref> shows that 2 clones, g17 and g52, derived from a pool of cells treated with the GS-specific ZFNs, do not express GS as assayed by Western blot. <figref idrefs="DRAWINGS">FIG. 14C</figref> is a graph showing growth over 7 days of the GS knock out clones and demonstrates the requirement for glutamine supplementation for growth.</p>
    </description-of-drawings> <heading>DETAILED DESCRIPTION</heading> <p num="p-0035">Described herein are compositions and methods for partial or complete inactivation of a GS gene. Also disclosed are methods of making and using these compositions (reagents), for example to inactivate a GS gene in a target cell. Inactivation of GS, alone or in combination with other genes such as DHFR and FUT8 in a target cell can be used to produce cell lines for expression of recombinant proteins, for example monoclonal antibodies that elicit enhanced antibody-dependent cellular cytotoxicity (ADCC).</p>
    <p num="p-0036">General</p>
    <p num="p-0037">Practice of the methods, as well as preparation and use of the compositions disclosed herein employ, unless otherwise indicated, conventional techniques in molecular biology, biochemistry, chromatin structure and analysis, computational chemistry, cell culture, recombinant DNA and related fields as are within the skill of the art. These techniques are fully explained in the literature. See, for example, Sambrook et al. <smallcaps>MOLECULAR CLONING: A LABORATORY MANUAL</smallcaps>, Second edition, Cold Spring Harbor Laboratory Press, 1989 and Third edition, 2001; Ausubel et al., <smallcaps>CURRENT PROTOCOLS IN MOLECULAR BIOLOGY</smallcaps>, John Wiley &amp; Sons, New York, 1987 and periodic updates; the series <smallcaps>METHODS IN ENZYMOLOGY</smallcaps>, Academic Press, San Diego; Wolffe, <smallcaps>CHROMATIN STRUCTURE AND FUNCTION</smallcaps>, Third edition, Academic Press, San Diego, 1998; <smallcaps>METHODS IN ENZYMOLOGY</smallcaps>, Vol. 304, “Chromatin” (P. M. Wassarman and A. P. Wolffe, eds.), Academic Press, San Diego, 1999; and <smallcaps>METHODS IN MOLECULAR BIOLOGY</smallcaps>, Vol. 119, “Chromatin Protocols” (P. B. Becker, ed.) Humana Press, Totowa, 1999.</p>
    <heading>DEFINITIONS</heading> <p num="p-0038">The terms “nucleic acid,” “polynucleotide,” and “oligonucleotide” are used interchangeably and refer to a deoxyribonucleotide or ribonucleotide polymer, in linear or circular conformation, and in either single- or double-stranded form. For the purposes of the present disclosure, these terms are not to be construed as limiting with respect to the length of a polymer. The terms can encompass known analogues of natural nucleotides, as well as nucleotides that are modified in the base, sugar and/or phosphate moieties (e.g., phosphorothioate backbones). In general, an analogue of a particular nucleotide has the same base-pairing specificity; i.e., an analogue of A will base-pair with T.</p>
    <p num="p-0039">The terms “polypeptide,” “peptide” and “protein” are used interchangeably to refer to a polymer of amino acid residues. The term also applies to amino acid polymers in which one or more amino acids are chemical analogues or modified derivatives of a corresponding naturally-occurring amino acids.</p>
    <p num="p-0040">“Binding” refers to a sequence-specific, non-covalent interaction between macromolecules (e.g., between a protein and a nucleic acid). Not all components of a binding interaction need be sequence-specific (e.g., contacts with phosphate residues in a DNA backbone), as long as the interaction as a whole is sequence-specific. Such interactions are generally characterized by a dissociation constant (IQ of 10<sup>−6 </sup>M<sup>−1 </sup>or lower. “Affinity” refers to the strength of binding: increased binding affinity being correlated with a lower K<sub>d</sub>.</p>
    <p num="p-0041">A “binding protein” is a protein that is able to bind non-covalently to another molecule. A binding protein can bind to, for example, a DNA molecule (a DNA-binding protein), an RNA molecule (an RNA-binding protein) and/or a protein molecule (a protein-binding protein). In the case of a protein-binding protein, it can bind to itself (to form homodimers, homotrimers, etc.) and/or it can bind to one or more molecules of a different protein or proteins. A binding protein can have more than one type of binding activity. For example, zinc finger proteins have DNA-binding, RNA-binding and protein-binding activity.</p>
    <p num="p-0042">A “zinc finger DNA binding protein” (or binding domain) is a protein, or a domain within a larger protein, that binds DNA in a sequence-specific manner through one or more zinc fingers, which are regions of amino acid sequence within the binding domain whose structure is stabilized through coordination of a zinc ion. The term zinc finger DNA binding protein is often abbreviated as zinc finger protein or ZFP.</p>
    <p num="p-0043">Zinc finger binding domains can be “engineered” to bind to a predetermined nucleotide sequence. Non-limiting examples of methods for engineering zinc finger proteins are design and selection. A designed zinc finger protein is a protein not occurring in nature whose design/composition results principally from rational criteria. Rational criteria for design include application of substitution rules and computerized algorithms for processing information in a database storing information of existing ZFP designs and binding data. See, for example, U.S. Pat. Nos. 6,140,081; 6,453,242; and 6,534,261; see also WO 98/53058; WO 98/53059; WO 98/53060; WO 02/016536 and WO 03/016496.</p>
    <p num="p-0044">A “selected” zinc finger protein is a protein not found in nature whose production results primarily from an empirical process such as phage display, interaction trap or hybrid selection. See e.g., U.S. Pat. No. 5,789,538; U.S. Pat. No. 5,925,523; U.S. Pat. No. 6,007,988; U.S. Pat. No. 6,013,453; U.S. Pat. No. 6,200,759; WO 95/19431; WO 96/06166; WO 98/53057; WO 98/54311; WO 00/27878; WO 01/60970; WO 01/88197 and WO 02/099084.</p>
    <p num="p-0045">The term “sequence” refers to a nucleotide sequence of any length, which can be DNA or RNA; can be linear, circular or branched and can be either single-stranded or double stranded. The term “donor sequence” refers to a nucleotide sequence that is inserted into a genome. A donor sequence can be of any length, for example between 2 and 10,000 nucleotides in length (or any integer value therebetween or thereabove), preferably between about 100 and 1,000 nucleotides in length (or any integer therebetween), more preferably between about 200 and 500 nucleotides in length.</p>
    <p num="p-0046">A “homologous, non-identical sequence” refers to a first sequence which shares a degree of sequence identity with a second sequence, but whose sequence is not identical to that of the second sequence. For example, a polynucleotide comprising the wild-type sequence of a′ mutant gene is homologous and non-identical to the sequence of the mutant gene. In certain embodiments, the degree of homology between the two sequences is sufficient to allow homologous recombination therebetween, utilizing normal cellular mechanisms. Two homologous non-identical sequences can be any length and their degree of non-homology can be as small as a single nucleotide (e.g., for correction of a genomic point mutation by targeted homologous recombination) or as large as 10 or more kilobases (e.g., for insertion of a gene at a predetermined ectopic site in a chromosome). Two polynucleotides comprising the homologous non-identical sequences need not be the same length. For example, an exogenous polynucleotide (i.e., donor polynucleotide) of between 20 and 10,000 nucleotides or nucleotide pairs can be used.</p>
    <p num="p-0047">Techniques for determining nucleic acid and amino acid sequence identity are known in the art. Typically, such techniques include determining the nucleotide sequence of the mRNA for a gene and/or determining the amino acid sequence encoded thereby, and comparing these sequences to a second nucleotide or amino acid sequence. Genomic sequences can also be determined and compared in this fashion. In general, identity refers to an exact nucleotide-to-nucleotide or amino acid-to-amino acid correspondence of two polynucleotides or polypeptide sequences, respectively. Two or more sequences (polynucleotide or amino acid) can be compared by determining their percent identity. The percent identity of two sequences, whether nucleic acid or amino acid sequences, is the number of exact matches between two aligned sequences divided by the length of the shorter sequences and multiplied by 100. An approximate alignment for nucleic acid sequences is provided by the local homology algorithm of Smith and Waterman, <i>Advances in Applied Mathematics </i>2:482-489 (1981). This algorithm can be applied to amino acid sequences by using the scoring matrix developed by Dayhoff, <i>Atlas of Protein Sequences and Structure</i>, M. O. Dayhoff ed., 5 suppl. 3:353-358, National Biomedical Research Foundation, Washington, D.C., USA, and normalized by Gribskov, <i>Nucl. Acids Res. </i>14(6):6745-6763 (1986). An exemplary implementation of this algorithm to determine percent identity of a sequence is provided by the Genetics Computer Group (Madison, Wis.) in the “BestFit” utility application. Other suitable programs for calculating the percent identity or similarity between sequences are generally known in the art, for example, another alignment program is BLAST, used with default parameters. For example, BLASTN and BLASTP can be used using the following default parameters: genetic code=standard; filter=none; strand=both; cutoff=60; expect=10; Matrix=BLOSUM62; Descriptions=50 sequences; sort by=HIGH SCORE; Databases=non-redundant, GenBank+EMBL+DDBJ+PDB+GenBank CDS translations+Swiss protein+Spupdate+PIR. Details of these programs can be found on the GenBank website. With respect to sequences described herein, the range of desired degrees of sequence identity is approximately 80% to 100% and any integer value therebetween. Typically the percent identities between sequences are at least 70-75%, preferably 80-82%, more preferably 85-90%, even more preferably 92%, still more preferably 95%, and most preferably 98% sequence identity.</p>
    <p num="p-0048">Alternatively, the degree of sequence similarity between polynucleotides can be determined by hybridization of polynucleotides under conditions that allow formation of stable duplexes between homologous regions, followed by digestion with single-stranded-specific nuclease(s), and size determination of the digested fragments. Two nucleic acid, or two polypeptide sequences are substantially homologous to each other when the sequences exhibit at least about 70%-75%, preferably 80%-82%, more-preferably 85%-90%, even more preferably 92%, still more preferably 95%, and most preferably 98% sequence identity over a defined length of the molecules, as determined using the methods above. As used herein, substantially homologous also refers to sequences showing complete identity to a specified DNA or polypeptide sequence. DNA sequences that are substantially homologous can be identified in a Southern hybridization experiment under, for example, stringent conditions, as defined for that particular system. Defining appropriate hybridization conditions is within the skill of the art. See, e.g., Sambrook et al., supra; <i>Nucleic Acid Hybridization: A Practical Approach</i>, editors B. D. Hames and S. J. Higgins, (1985) Oxford; Washington, D.C.; IRL Press).</p>
    <p num="p-0049">Selective hybridization of two nucleic acid fragments can be determined as follows. The degree of sequence identity between two nucleic acid molecules affects the efficiency and strength of hybridization events between such molecules. A partially identical nucleic acid sequence will at least partially inhibit the hybridization of a completely identical sequence to a target molecule. Inhibition of hybridization of the completely identical sequence can be assessed using hybridization assays that are well known in the art (e.g., Southern (DNA) blot, Northern (RNA) blot, solution hybridization, or the like, see Sambrook, et al., <i>Molecular Cloning: A Laboratory Manual</i>, Second Edition, (1989) Cold Spring Harbor, N.Y.). Such assays can be conducted using varying degrees of selectivity, for example, using conditions varying from low to high stringency. If conditions of low stringency are employed, the absence of non-specific binding can be assessed using a secondary probe that lacks even a partial degree of sequence identity (for example, a probe having less than about 30% sequence identity with the target molecule), such that, in the absence of non-specific binding events, the secondary probe will not hybridize to the target.</p>
    <p num="p-0050">When utilizing a hybridization-based detection system, a nucleic acid probe is chosen that is complementary to a reference nucleic acid sequence, and then by selection of appropriate conditions the probe and the reference sequence selectively hybridize, or bind, to each other to form a duplex molecule. A nucleic acid molecule that is capable of hybridizing selectively to a reference sequence under moderately stringent hybridization conditions typically hybridizes under conditions that allow detection of a target nucleic acid sequence of at least about 10-14 nucleotides in length having at least approximately 70% sequence identity with the sequence of the selected nucleic acid probe. Stringent hybridization conditions typically allow detection of target nucleic acid sequences of at least about 10-14 nucleotides in length having a sequence identity of greater than about 90-95% with the sequence of the selected nucleic acid probe. Hybridization conditions useful for probe/reference sequence hybridization, where the probe and reference sequence have a specific degree of sequence identity, can be determined as is known in the art (see, for example, <i>Nucleic Acid Hybridization: A Practical Approach</i>, editors B. D. Hames and S. J. Higgins, (1985) Oxford; Washington, D.C.; IRL Press).</p>
    <p num="p-0051">Conditions for hybridization are well-known to those of skill in the art.</p>
    <p num="p-0052">Hybridization stringency refers to the degree to which hybridization conditions disfavor the formation of hybrids containing mismatched nucleotides, with higher stringency correlated with a lower tolerance for mismatched hybrids. Factors that affect the stringency of hybridization are well-known to those of skill in the art and include, but are not limited to, temperature, pH, ionic strength, and concentration of organic solvents such as, for example, formamide and dimethylsulfoxide. As is known to those of skill in the art, hybridization stringency is increased by higher temperatures, lower ionic strength and lower solvent concentrations.</p>
    <p num="p-0053">With respect to stringency conditions for hybridization, it is well known in the art that numerous equivalent conditions can be employed to establish a particular stringency by varying, for example, the following factors: the length and nature of the sequences, base composition of the various sequences, concentrations of salts and other hybridization solution components, the presence or absence of blocking agents in the hybridization solutions (e.g., dextran sulfate, and polyethylene glycol), hybridization reaction temperature and time parameters, as well as, varying wash conditions. The selection of a particular set of hybridization conditions is selected following standard methods in the art (see, for example, Sambrook, et al., <i>Molecular Cloning: A Laboratory Manual</i>, Second Edition, (1989) Cold Spring Harbor, N.Y.).</p>
    <p num="p-0054">“Recombination” refers to a process of exchange of genetic information between two polynucleotides. For the purposes of this disclosure, “homologous recombination (HR)” refers to the specialized form of such exchange that takes place, for example, during repair of double-strand breaks in cells. This process requires nucleotide sequence homology, uses a “donor” molecule to template repair of a “target” molecule (i.e., the one that experienced the double-strand break), and is variously known as “non-crossover gene conversion” or “short tract gene conversion,” because it leads to the transfer of genetic information from the donor to the target. Without wishing to be bound by any particular theory, such transfer can involve mismatch correction of heteroduplex DNA that forms between the broken target and the donor, and/or “synthesis-dependent strand annealing,” in which the donor is used to resynthesize genetic information that will become part of the target, and/or related processes. Such specialized HR often results in an alteration of the sequence of the target molecule such that part or all of the sequence of the donor polynucleotide is incorporated into the target polynucleotide.</p>
    <p num="p-0055">“Cleavage” refers to the breakage of the covalent backbone of a DNA molecule. Cleavage can be initiated by a variety of methods including, but not limited to, enzymatic or chemical hydrolysis of a phosphodiester bond. Both single-stranded cleavage and double-stranded cleavage are possible, and double-stranded cleavage can occur as a result of two distinct single-stranded cleavage events. DNA cleavage can result in the production of either blunt ends or staggered ends. In certain embodiments, fusion polypeptides are used for targeted double-stranded DNA cleavage.</p>
    <p num="p-0056">An “cleavage half-domain” is a polypeptide sequence which, in conjunction with a second polypeptide (either identical or different) forms a complex having cleavage activity (preferably double-strand cleavage activity). The terms “first and second cleavage half-domains;” “+ and − cleavage half-domains” and “right and left cleavage half-domains” are used interchangeably to refer to pairs of cleavage half-domains that dimerize.</p>
    <p num="p-0057">An “engineered cleavage half-domain” is a cleavage half-domain that has been modified so as to form obligate heterodimers with another cleavage half-domain (e.g., another engineered cleavage half-domain). See, also, U.S. Patent Publication Nos. 2005/0064474; 2007/0218528 and 2008/0131962, incorporated herein by reference in their entireties.</p>
    <p num="p-0058">“Chromatin” is the nucleoprotein structure comprising the cellular genome. Cellular chromatin comprises nucleic acid, primarily DNA, and protein, including histones and non-histone chromosomal proteins. The majority of eukaryotic cellular chromatin exists in the form of nucleosomes, wherein a nucleosome core comprises approximately 150 base pairs of DNA associated with an octamer comprising two each of histones H2A, H2B, H3 and H4; and linker DNA (of variable length depending on the organism) extends between nucleosome cores. A molecule of histone H1 is generally associated with the linker DNA. For the purposes of the present disclosure, the term “chromatin” is meant to encompass all types of cellular nucleoprotein, both prokaryotic and eukaryotic. Cellular chromatin includes both chromosomal and episomal chromatin.</p>
    <p num="p-0059">A “chromosome,” is a chromatin complex comprising all or a portion of the genome of a cell. The genome of a cell is often characterized by its karyotype, which is the collection of all the chromosomes that comprise the genome of the cell. The genome of a cell can comprise one or more chromosomes.</p>
    <p num="p-0060">An “episome” is a replicating nucleic acid, nucleoprotein complex or other structure comprising a nucleic acid that is not part of the chromosomal karyotype of a cell. Examples of episomes include plasmids and certain viral genomes.</p>
    <p num="p-0061">An “accessible region” is a site in cellular chromatin in which a target site present in the nucleic acid can be bound by an exogenous molecule which recognizes the target site. Without wishing to be bound by any particular theory, it is believed that an accessible region is one that is not packaged into a nucleosomal structure. The distinct structure of an accessible region can often be detected by its sensitivity to chemical and enzymatic probes, for example, nucleases.</p>
    <p num="p-0062">A “target site” or “target sequence” is a nucleic acid sequence that defines a portion of a nucleic acid to which a binding molecule will bind, provided sufficient conditions for binding exist. For example, the sequence 5′-GAATTC-3′ is a target site for the Eco RI restriction endonuclease.</p>
    <p num="p-0063">An “exogenous” molecule is a molecule that is not normally present in a cell, but can be introduced into a cell by one or more genetic, biochemical or other methods. “Normal presence in the cell” is determined with respect to the particular developmental stage and environmental conditions of the cell. Thus, for example, a molecule that is present only during embryonic development of muscle is an exogenous molecule with respect to an adult muscle cell. Similarly, a molecule induced by heat shock is an exogenous molecule with respect to a non-heat-shocked cell. An exogenous molecule can comprise, for example, a functioning version of a malfunctioning endogenous molecule or a malfunctioning version of a normally-functioning endogenous molecule.</p>
    <p num="p-0064">An exogenous molecule can be, among other things, a small molecule, such as is generated by a combinatorial chemistry process, or a macromolecule such as a protein, nucleic acid, carbohydrate, lipid, glycoprotein, lipoprotein, polysaccharide, any modified derivative of the above molecules, or any complex comprising one or more of the above molecules. Nucleic acids include DNA and RNA, can be single- or double-stranded; can be linear, branched or circular; and can be of any length. Nucleic acids include those capable of forming duplexes, as well as triplex-forming nucleic acids. See, for example, U.S. Pat. Nos. 5,176,996 and 5,422,251. Proteins include, but are not limited to, DNA-binding proteins, transcription factors, chromatin remodeling factors, methylated DNA binding proteins, polymerases, methylases, demethylases, acetylases, deacetylases, kinases, phosphatases, integrases, recombinases, ligases, topoisomerases, gyrases and helicases. An exogeneous molecule can also be the same type of molecule as an endogenous molecule but derived from a different species than the cell is derived from. For example, a human nucleic acid sequenced may be introduced into a cell line originally derived from a mouse or hamster.</p>
    <p num="p-0065">An exogenous molecule can be the same type of molecule as an endogenous molecule, e.g., an exogenous protein or nucleic acid. For example, an exogenous nucleic acid can comprise an infecting viral genome, a plasmid or episome introduced into a cell, or a chromosome that is not normally present in the cell. Methods for the introduction of exogenous molecules into cells are known to those of skill in the art and include, but are not limited to, lipid-mediated transfer (i.e., liposomes, including neutral and cationic lipids), electroporation, direct injection, cell fusion, particle bombardment, calcium phosphate co-precipitation, DEAE-dextran-mediated transfer and viral vector-mediated transfer. An exogenous molecule can also refer to a nucleic acid from a different species, for example, a human gene inserted into a hamster genome.</p>
    <p num="p-0066">By contrast, an “endogenous” molecule is one that is normally present in a particular cell at a particular developmental stage under particular environmental conditions. For example, an endogenous nucleic acid can comprise a chromosome, the genome of a mitochondrion, chloroplast or other organelle, or a naturally-occurring episomal nucleic acid. Additional endogenous molecules can include proteins, for example, transcription factors and enzymes.</p>
    <p num="p-0067">A “fusion” molecule is a molecule in which two or more subunit molecules are linked, preferably covalently. The subunit molecules can be the same chemical type of molecule, or can be different chemical types of molecules. Examples of the first type of fusion molecule include, but are not limited to, fusion proteins (for example, a fusion between a ZFP DNA-binding domain and a cleavage domain) and fusion nucleic acids (for example, a nucleic acid encoding the fusion protein described supra). Examples of the second type of fusion molecule include, but are not limited to, a fusion between a triplex-forming nucleic acid and a polypeptide, and a fusion between a minor groove binder and a nucleic acid.</p>
    <p num="p-0068">Expression of a fusion protein in a cell can result from delivery of the fusion protein to the cell or by delivery of a polynucleotide encoding the fusion protein to a cell, wherein the polynucleotide is transcribed, and the transcript is translated, to generate the fusion protein. Trans-splicing, polypeptide cleavage and polypeptide ligation can also be involved in expression of a protein in a cell. Methods for polynucleotide and polypeptide delivery to cells are presented elsewhere in this disclosure.</p>
    <p num="p-0069">A “gene,” for the purposes of the present disclosure, includes a DNA region encoding a gene product (see infra), as well as all DNA regions which regulate the production of the gene product, whether or not such regulatory sequences are adjacent to coding and/or transcribed sequences. Accordingly, a gene includes, but is not necessarily limited to, promoter sequences, terminators, translational regulatory sequences such as ribosome binding sites and internal ribosome entry sites, enhancers, silencers, insulators, boundary elements, replication origins, matrix attachment sites and locus control regions.</p>
    <p num="p-0070">“Gene expression” refers to the conversion of the information, contained in a gene, into a gene product. A gene product can be the direct transcriptional product of a gene (e.g., mRNA, tRNA, rRNA, antisense RNA, shRNAs, micro RNAs (miRNAs) ribozyme, structural RNA or any other type of RNA) or a protein produced by translation of a mRNA. Gene products also include RNAs which are modified, by processes such as capping, polyadenylation, methylation, and editing, and proteins modified by, for example, methylation, acetylation, phosphorylation, ubiquitination, ADP-ribosylation, myristilation, and glycosylation.</p>
    <p num="p-0071">“Modulation” of gene expression refers to a change in the activity of a gene. Modulation of expression can include, but is not limited to, gene activation and gene repression. Gene inactivation refers to any reduction in gene expression as compared to a cell that does not include a ZFP as described herein. Thus, gene inactivation may be complete (knock-out) or partial (e.g., a hypomorph in which a gene exhibits less than normal expression levels or a product of a mutant gene that shows partial reduction in the activity it influences).</p>
    <p num="p-0072">“Eucaryotic” cells include, but are not limited to, fungal cells (such as yeast), plant cells, animal cells, mammalian cells and human cells (e.g., T-cells).</p>
    <p num="p-0073">A “region of interest” is any region of cellular chromatin, such as, for example, a gene or a non-coding sequence within or adjacent to a gene, in which it is desirable to bind an exogenous molecule. Binding can be for the purposes of targeted DNA cleavage and/or targeted recombination. A region of interest can be present in a chromosome, an episome, an organellar genome (e.g., mitochondrial, chloroplast), or an infecting viral genome, for example. A region of interest can be within the coding region of a gene, within transcribed non-coding regions such as, for example, leader sequences, trailer sequences or introns, or within non-transcribed regions, either upstream or downstream of the coding region. A region of interest can be as small as a single nucleotide pair or up to 2,000 nucleotide pairs in length, or any integral value of nucleotide pairs.</p>
    <p num="p-0074">The terms “operative linkage” and “operatively linked” (or “operably linked”) are used interchangeably with reference to a juxtaposition of two or more components (such as sequence elements), in which the components are arranged such that both components function normally and allow the possibility that at least one of the components can mediate a function that is exerted upon at least one of the other components. By way of illustration, a transcriptional regulatory sequence, such as a promoter, is operatively linked to a coding sequence if the transcriptional regulatory sequence controls the level of transcription of the coding sequence in response to the presence or absence of one or more transcriptional regulatory factors. A transcriptional regulatory sequence is generally operatively linked in cis with a coding sequence, but need not be directly adjacent to it. For example, an enhancer is a transcriptional regulatory sequence that is operatively linked to a coding sequence, even though they are not contiguous.</p>
    <p num="p-0075">With respect to fusion polypeptides, the term “operatively linked” can refer to the fact that each of the components performs the same function in linkage to the other component as it would if it were not so linked. For example, with respect to a fusion polypeptide in which a ZFP DNA-binding domain is fused to a cleavage domain, the ZFP DNA-binding domain and the cleavage domain are in operative linkage if, in the fusion polypeptide, the ZFP DNA-binding domain portion is able to bind its target site and/or its binding site, while the cleavage domain is able to cleave DNA in the vicinity of the target site.</p>
    <p num="p-0076">A “functional fragment” of a protein, polypeptide or nucleic acid is a protein, polypeptide or nucleic acid whose sequence is not identical to the full-length protein, polypeptide or nucleic acid, yet retains the same function as the full-length protein, polypeptide or nucleic acid. A functional fragment can possess more, fewer, or the same number of residues as the corresponding native molecule, and/or can contain one or more amino acid or nucleotide substitutions. Methods for determining the function of a nucleic acid (e.g., coding function, ability to hybridize to another nucleic acid) are well-known in the art. Similarly, methods for determining protein function are well-known. For example, the DNA-binding function of a polypeptide can be determined, for example, by filter-binding, electrophoretic mobility-shift, or immunoprecipitation assays. DNA cleavage can be assayed by gel electrophoresis. See Ausubel et al., supra. The ability of a protein to interact with another protein can be determined, for example, by co-immunoprecipitation, two-hybrid assays or complementation, both genetic and biochemical. See, for example, Fields et al. (1989) <i>Nature </i>340:245-246; U.S. Pat. No. 5,585,245 and PCT WO 98/44350.</p>
    <p num="p-0077">The term “antibody” as used herein includes antibodies obtained from both polyclonal and monoclonal preparations, as well as, the following: hybrid (chimeric) antibody molecules (see, for example, Winter et al., Nature (1991) 349:293-299; and U.S. Pat. No. 4,816,567); F(ab′)<sub>2 </sub>and F(ab) fragments; Fv molecules (non-covalent heterodimers, see, for example, Inbar et al., Proc Natl Acad Sci USA (1972) 69:2659-2662; and Ehrlich et al., Biochem (1980) 19:4091-4096); single-chain Fv molecules (sFv) (see, for example, Huston et al., Proc Natl Acad Sci USA (1988) 85:5879-5883); dimeric and trimeric antibody fragment constructs; minibodies (see, e.g., Pack et al., Biochem (1992) 31:1579-1584; Cumber et al., J Immunology (1992) 149B: 120-126); humanized antibody molecules (see, for example, Riechmann et al., Nature (1988) 332:323-327; Verhoeyan et al., Science (1988) 239:1534-1536; and U.K. Patent Publication No. GB 2,276,169, published 21 Sep. 1994); and, any functional fragments obtained from such molecules, wherein such fragments retain immunological binding properties of the parent antibody molecule.</p>
    <p num="p-0078">As used herein, the term “monoclonal antibody” refers to an antibody composition having a homogeneous antibody population. The term is not limited regarding the species or source of the antibody, nor is it intended to be limited by the manner in which it is made. The term encompasses whole immunoglobulins as well as fragments such as Fab, F(ab′)2, Fv, and other fragments, as well as chimeric and humanized homogeneous antibody populations that exhibit immunological binding properties of the parent monoclonal antibody molecule.</p>
    <p num="p-0079">Zinc Finger Nucleases</p>
    <p num="p-0080">Described herein are zinc finger nucleases (ZFNs) that can be used for inactivation of a GS gene. ZFNs comprise a zinc finger protein (ZFP) and a nuclease (cleavage) domain.</p>
    <p num="p-0081">A. Zinc Finger Proteins</p>
    <p num="p-0082">Zinc finger binding domains can be engineered to bind to a sequence of choice. See, for example, Beerli et al. (2002)<i>Nature Biotechnol. </i>20:135-141; Pabo et al. (2001) <i>Ann. Rev. Biochem. </i>70:313-340; Isalan et al. (2001) <i>Nature Biotechnol. </i>19:656-660; Segal et al. (2001) <i>Curr. Opin. Biotechnol. </i>12:632-637; Choo et al. (2000) <i>Curr. Opin. Struct. Biol. </i>10:411-416. An engineered zinc finger binding domain can have a novel binding specificity, compared to a naturally-occurring zinc finger protein. Engineering methods include, but are not limited to, rational design and various types of selection. Rational design includes, for example, using databases comprising triplet (or quadruplet) nucleotide sequences and individual zinc finger amino acid sequences, in which each triplet or quadruplet nucleotide sequence is associated with one or more amino acid sequences of zinc fingers which bind the particular triplet or quadruplet sequence. See, for example, co-owned U.S. Pat. Nos. 6,453,242 and 6,534,261, incorporated by reference herein in their entireties.</p>
    <p num="p-0083">Exemplary selection methods, including phage display and two-hybrid systems, are disclosed in U.S. Pat. Nos. 5,789,538; 5,925,523; 6,007,988; 6,013,453; 6,410,248; 6,140,466; 6,200,759; and 6,242,568; as well as WO 98/37186; WO 98/53057; WO 00/27878; WO 01/88197 and GB 2,338,237. In addition, enhancement of binding specificity for zinc finger binding domains has been described, for example, in co-owned WO 02/077227.</p>
    <p num="p-0084">Selection of target sites; ZFPs and methods for design and construction of fusion proteins (and polynucleotides encoding same) are known to those of skill in the art and described in detail in U.S. Patent Application Publication Nos. 20050064474 and 20060188987, incorporated by reference in their entireties herein.</p>
    <p num="p-0085">In addition, as disclosed in these and other references, zinc finger domains and/or multi-fingered zinc finger proteins may be linked together using any suitable linker sequences, including for example, linkers of 5 or more amino acids in length. See, also, U.S. Pat. Nos. 6,479,626; 6,903,185; and 7,153,949 for exemplary linker sequences 6 or more amino acids in length. The proteins described herein may include any combination of suitable linkers between the individual zinc fingers of the protein. Examples of additional linker structures are found in U.S. Provisional Application No. 61/130,099, filed May 28, 2008 and entitled Compositions For Linking DNA-Binding Domains And Cleavage Domains.</p>
    <p num="p-0086">Table 1 describes a number of zinc finger binding domains that have been engineered to bind to nucleotide sequences in the GS gene. See, also, <figref idrefs="DRAWINGS">FIG. 1</figref> and <figref idrefs="DRAWINGS">FIG. 3</figref>. Each row describes a separate zinc finger DNA-binding domain. The DNA target sequence for each domain is shown in the first column and the second through fifth columns show the amino acid sequence of the recognition region (amino acids −1 through +6, with respect to the start of the helix) of each of the zinc fingers (F1 through F4, F5 or F6) in the protein. Nucleotides shown in uppercase denote triplets that are directly targeted by a zinc finger and nucleotides shown in lower case denotes bases that are not directly targeted by a zinc finger (for example due to longer linkers between fingers which can result in “skipping” a base). Also provided in the first column is an identification number for the proteins.</p>
    <p num="p-0087">
      <tables id="TABLE-US-00001" num="00001"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="287pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Zinc finger nucleases targeted to GS</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="77pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <colspec colname="6" colwidth="35pt" align="left"> </colspec> <colspec colname="7" colwidth="35pt" align="left"> </colspec> <colspec colname="8" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">ZFN name</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Target sequence</td>
                <td class="description-td">F1</td>
                <td class="description-td">F2</td>
                <td class="description-td">F3</td>
                <td class="description-td">F4</td>
                <td class="description-td">F5</td>
                <td class="description-td">F6</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ZFN 9075</td>
                <td class="description-td">QSSDLSR</td>
                <td class="description-td">RSDNLRE</td>
                <td class="description-td">RSDTLSN</td>
                <td class="description-td">RKDVRIT</td>
                <td class="description-td">N/A</td>
                <td class="description-td">N/A</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">gaATGGTGCAGGCTgc</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 1)</td>
                <td class="description-td">NO: 2)</td>
                <td class="description-td">NO: 3)</td>
                <td class="description-td">NO: 4)</td>
                <td class="description-td">NO: 5)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ZFN 9372</td>
                <td class="description-td">RSDHLST</td>
                <td class="description-td">QSSDLRR</td>
                <td class="description-td">RSDHLSQ</td>
                <td class="description-td">QSANRTT</td>
                <td class="description-td">RSDNLSQ</td>
                <td class="description-td">ASNDRKK</td>
              </tr> <tr class="description-tr"> <td class="description-td">gtTCCCAGGAATGGG</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTTGGgg</td>
                <td class="description-td">NO: 7)</td>
                <td class="description-td">NO: 8)</td>
                <td class="description-td">NO: 9)</td>
                <td class="description-td">NO: 10)</td>
                <td class="description-td">NO: 11)</td>
                <td class="description-td">NO: 12)</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 6)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ZFN 8361</td>
                <td class="description-td">QSGALAR</td>
                <td class="description-td">RSDALTQ</td>
                <td class="description-td">RSDSLSA</td>
                <td class="description-td">RSAHLSR</td>
                <td class="description-td">N/A</td>
                <td class="description-td">N/A</td>
              </tr> <tr class="description-tr"> <td class="description-td">ctGGGTTGATGGTActg</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
              </tr> <tr class="description-tr"> <td class="description-td">gagaaggactg</td>
                <td class="description-td">NO: 14)</td>
                <td class="description-td">NO: 15)</td>
                <td class="description-td">NO: 16)</td>
                <td class="description-td">NO: 17)</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 13)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ZFN 8365</td>
                <td class="description-td">RSDHLST</td>
                <td class="description-td">QSSDLRR</td>
                <td class="description-td">RSDSLSV</td>
                <td class="description-td">DNANRTK</td>
                <td class="description-td">N/A</td>
                <td class="description-td">N/A</td>
              </tr> <tr class="description-tr"> <td class="description-td">taTACATGGCTTGGact</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
              </tr> <tr class="description-tr"> <td class="description-td">ttctcaccctg</td>
                <td class="description-td">NO: 7)</td>
                <td class="description-td">NO: 8)</td>
                <td class="description-td">NO: 19)</td>
                <td class="description-td">NO: 20)</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 18)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ZFN 9076</td>
                <td class="description-td">QSSDLSR</td>
                <td class="description-td">RSDNLRE</td>
                <td class="description-td">RSSALTR</td>
                <td class="description-td">RSDALTQ</td>
                <td class="description-td">N/A</td>
                <td class="description-td">N/A</td>
              </tr> <tr class="description-tr"> <td class="description-td">gaATGGTGCAGGCTgc</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
              </tr> <tr class="description-tr"> <td class="description-td">cataccaacttt</td>
                <td class="description-td">NO: 2)</td>
                <td class="description-td">NO: 3)</td>
                <td class="description-td">NO: 22)</td>
                <td class="description-td">NO: 15)</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 21)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ZFN 9179</td>
                <td class="description-td">QSSDLSR</td>
                <td class="description-td">RSDNLRE</td>
                <td class="description-td">QSSHLTR</td>
                <td class="description-td">TSSNRKT</td>
                <td class="description-td">N/A</td>
                <td class="description-td">N/A</td>
              </tr> <tr class="description-tr"> <td class="description-td">ggAATGGTgCAGGCTg</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
              </tr> <tr class="description-tr"> <td class="description-td">ccataccaactt</td>
                <td class="description-td">NO: 2)</td>
                <td class="description-td">NO: 3)</td>
                <td class="description-td">NO: 24)</td>
                <td class="description-td">NO: 25)</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 23)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ZFN 7858</td>
                <td class="description-td">QSSDLSR</td>
                <td class="description-td">RSDNLRE</td>
                <td class="description-td">RSDSLLR</td>
                <td class="description-td">RSDALTQ</td>
                <td class="description-td">N/A</td>
                <td class="description-td">N/A</td>
              </tr> <tr class="description-tr"> <td class="description-td">gaATGGTGCAGGCTgc</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 1)</td>
                <td class="description-td">NO: 2)</td>
                <td class="description-td">NO: 3)</td>
                <td class="description-td">NO: 26)</td>
                <td class="description-td">NO: 15)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ZFN 7889</td>
                <td class="description-td">RSDHLSQ</td>
                <td class="description-td">QSANRTT</td>
                <td class="description-td">RSDNLSQ</td>
                <td class="description-td">ASNDRKK</td>
                <td class="description-td">N/A</td>
                <td class="description-td">N/A</td>
              </tr> <tr class="description-tr"> <td class="description-td">gtTCCCAGGAATGGgc</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 27)</td>
                <td class="description-td">NO: 9)</td>
                <td class="description-td">NO: 10)</td>
                <td class="description-td">NO: 11)</td>
                <td class="description-td">NO: 12)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ZFN 9373</td>
                <td class="description-td">RSDHLSQ</td>
                <td class="description-td">RNADRIT</td>
                <td class="description-td">RSDHLSQ</td>
                <td class="description-td">QSANRTT</td>
                <td class="description-td">RSDNLSQ</td>
                <td class="description-td">ASNDRKK</td>
              </tr> <tr class="description-tr"> <td class="description-td">gtTCCCAGGAATGGgC</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTGGGgtcaaag</td>
                <td class="description-td">NO: 9)</td>
                <td class="description-td">NO: 29)</td>
                <td class="description-td">NO: 9)</td>
                <td class="description-td">NO: 10)</td>
                <td class="description-td">NO: 11)</td>
                <td class="description-td">NO: 12)</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 28)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0088">As described below, in certain embodiments, a four-, five-, or six-finger binding domain as shown in Table 1 is fused to a cleavage half-domain, such as, for example, the cleavage domain of a Type IIs restriction endonuclease such as FokI. One or more pairs of such zinc finger/nuclease half-domain fusions are used for targeted cleavage, as disclosed, for example, in U.S. Patent Publication Nos. 20050064474 and 20070218528.</p>
    <p num="p-0089">For targeted cleavage, the near edges of the binding sites can separated by 5 or more nucleotide pairs, and each of the fusion proteins can bind to an opposite strand of the DNA target. All pairwise combinations of the proteins shown in Table 1 can be used for targeted cleavage of a GS gene. Following the present disclosure, ZFNs can be targeted to any sequence in a GS gene.</p>
    <p num="p-0090">B. Cleavage Domains</p>
    <p num="p-0091">The ZFNs also comprise a nuclease (cleavage domain, cleavage half-domain). The cleavage domain portion of the fusion proteins disclosed herein can be obtained from any endonuclease or exonuclease. Exemplary endonucleases from which a cleavage domain can be derived include, but are not limited to, restriction endonucleases and homing endonucleases. See, for example, 2002-2003 Catalogue, New England Biolabs, Beverly, Mass.; and Belfort et al. (1997) <i>Nucleic Acids Res. </i>25:3379-3388. Additional enzymes which cleave DNA are known (e.g., Si Nuclease; mung bean nuclease; pancreatic DNase I; micrococcal nuclease; yeast HO endonuclease; see also Linn et al. (eds.) <i>Nucleases</i>, Cold Spring Harbor Laboratory Press, 1993). One or more of these enzymes (or functional fragments thereof) can be used as a source of cleavage domains and cleavage half-domains.</p>
    <p num="p-0092">Similarly, a cleavage half-domain can be derived from any nuclease or portion thereof, as set forth above, that requires dimerization for cleavage activity. In general, two fusion proteins are required for cleavage if the fusion proteins comprise cleavage half-domains. Alternatively, a single protein comprising two cleavage half-domains can be used. The two cleavage half-domains can be derived from the same endonuclease (or functional fragments thereof), or each cleavage half-domain can be derived from a different endonuclease (or functional fragments thereof). In addition, the target sites for the two fusion proteins are preferably disposed, with respect to each other, such that binding of the two fusion proteins to their respective target sites places the cleavage half-domains in a spatial orientation to each other that allows the cleavage half-domains to form a functional cleavage domain, e.g., by dimerizing. Thus, in certain embodiments, the near edges of the target sites are separated by 5-8 nucleotides or by 15-18 nucleotides. However any integral number of nucleotides or nucleotide pairs can intervene between two target sites (e.g., from 2 to 50 nucleotide pairs or more). In general, the site of cleavage lies between the target sites.</p>
    <p num="p-0093">Restriction endonucleases (restriction enzymes) are present in many species and are capable of sequence-specific binding to DNA (at a recognition site), and cleaving DNA at or near the site of binding. Certain restriction enzymes (e.g., Type IIS) cleave DNA at sites removed from the recognition site and have separable binding and cleavage domains. For example, the Type IIS enzyme Fok I catalyzes double-stranded cleavage of DNA, at 9 nucleotides from its recognition site on one strand and 13 nucleotides from its recognition site on the other. See, for example, U.S. Pat. Nos. 5,356,802; 5,436,150 and 5,487,994; as well as Li et al. (1992) <i>Proc. Natl. Acad. Sci. USA </i>89:4275-4279; Li et al. (1993) <i>Proc. Natl. Acad. Sci. USA </i>90:2764-2768; Kim et al. (1994a) <i>Proc. Natl. Acad. Sci. USA </i>91:883-887; Kim et al. (1994b) <i>J. Biol. Chem. </i>269:31,978-31,982. Thus, in one embodiment, fusion proteins comprise the cleavage domain (or cleavage half-domain) from at least one Type IIS restriction enzyme and one or more zinc finger binding domains, which may or may not be engineered.</p>
    <p num="p-0094">An exemplary Type IIS restriction enzyme, whose cleavage domain is separable from the binding domain, is Fok I. This particular enzyme is active as a dimer. Bitinaite et al. (1998) <i>Proc. Natl. Acad. Sci. USA </i>95: 10,570-10,575. Accordingly, for the purposes of the present disclosure, the portion of the Fok I enzyme used in the disclosed fusion proteins is considered a cleavage half-domain. Thus, for targeted double-stranded cleavage and/or targeted replacement of cellular sequences using zinc finger-Fok fusions, two fusion proteins, each comprising a FokI cleavage half-domain, can be used to reconstitute a catalytically active cleavage domain. Alternatively, a single polypeptide molecule containing a zinc finger binding domain and two Fok I cleavage half-domains can also be used. Parameters for targeted cleavage and targeted sequence alteration using zinc finger-Fok I fusions are provided elsewhere in this disclosure.</p>
    <p num="p-0095">A cleavage domain or cleavage half-domain can be any portion of a protein that retains cleavage activity, or that retains the ability to multimerize (e.g., dimerize) to form a functional cleavage domain.</p>
    <p num="p-0096">Exemplary Type IIS restriction enzymes are described in International Publication WO 07/014,275, incorporated herein in its entirety. Additional restriction enzymes also contain separable binding and cleavage domains, and these are contemplated by the present disclosure. See, for example, Roberts et al. (2003) <i>Nucleic Acids Res. </i>31:418-420.</p>
    <p num="p-0097">In certain embodiments, the cleavage domain comprises one or more engineered cleavage half-domain (also referred to as dimerization domain mutants) that minimize or prevent homodimerization, as described, for example, in U.S. Patent Publication Nos. 20050064474; 20060188987 and 20080131962, the disclosures of all of which are incorporated by reference in their entireties herein. Amino acid residues at positions 446, 447, 479, 483, 484, 486, 487, 490, 491, 496, 498, 499, 500, 531, 534, 537, and 538 of Fok are all targets for influencing dimerization of the Fok I cleavage half-domains.</p>
    <p num="p-0098">Exemplary engineered cleavage half-domains of Fok I that form obligate heterodimers include a pair in which a first cleavage half-domain includes mutations at amino acid residues at positions 490 and 538 of Fok and a second cleavage half-domain includes mutations at amino acid residues 486 and 499.</p>
    <p num="p-0099">Thus, in one embodiment, a mutation at 490 replaces Glu (E) with Lys (K); the mutation at 538 replaces Iso (I) with Lys (K); the mutation at 486 replaced Gln (O) with Glu (E); and the mutation at position 499 replaces Iso (I) with Lys (K). Specifically, the engineered cleavage half-domains described herein were prepared by mutating positions 490 (E→K) and 538 (I→K) in one cleavage half-domain to produce an engineered cleavage half-domain designated “E490K:I538K” and by mutating positions 486 (Q→E) and 499 (I→L) in another cleavage half-domain to produce an engineered cleavage half-domain designated “Q486E:I499L”. As described in the examples a pair of ZFNs in which one ZFN comprises the “E490K:I538K” cleavage domain and other comprises “Q486E:I499L” cleavage domain is also referred to as a “EL/KK” ZFN pair. The engineered cleavage half-domains described herein are obligate heterodimer mutants in which aberrant cleavage is minimized or abolished when one or more pairs of nucleases containing these cleavage half-domains are used for cleavage. See, e.g., U.S. Patent Publication No. 20080131962, the disclosure of which is incorporated by reference in its entirety for all purposes.</p>
    <p num="p-0100">Engineered cleavage half-domains described herein can be prepared using any suitable method, for example, by site-directed mutagenesis of wild-type cleavage half-domains (Fok I) as described in U.S. Patent Publication Nos. 20050064474 (Example 5) and 20070134796 (Example 38).</p>
    <p num="p-0101">C. Additional Methods for Targeted Cleavage in GS</p>
    <p num="p-0102">Any nuclease having a target site in a GS gene can be used in the methods disclosed herein. For example, homing endonucleases and meganucleases have very long recognition sequences, some of which are likely to be present, on a statistical basis, once in a human-sized genome. Any such nuclease having a unique target site in a GS gene can be used instead of, or in addition to, a zinc finger nuclease, for targeted cleavage in a GS gene.</p>
    <p num="p-0103">Exemplary homing endonucleases include I-SceI, I-CeuI, PI-PspI, PI-Sce, I-SceIV, I-CsmI, I-PanI, I-SceI, I-PpoI, I-SceIII, I-CreI, I-TevI, I-TevII and I-TevIII. Their recognition sequences are known. See also U.S. Pat. No. 5,420,032; U.S. Pat. No. 6,833,252; Belfort et al. (1997) <i>Nucleic Acids Res. </i>25:3379-3388; Dujon et al. (1989) <i>Gene </i>82:115-118; Perler et al. (1994) Nucleic Acids Res. 22, 1125-1127; Jasin (1996) <i>Trends Genet. </i>12:224-228; Gimble et al. (1996) <i>J. Mol. Biol. </i>263:163-180; Argast et al. (1998) <i>J. Mol. Biol. </i>280:345-353 and the New England Biolabs catalogue.</p>
    <p num="p-0104">Although the cleavage specificity of most homing endonucleases is not absolute with respect to their recognition sites, the sites are of sufficient length that a single cleavage event per mammalian-sized genome can be obtained by expressing a homing endonuclease in a cell containing a single copy of its recognition site. It has also been reported that the specificity of homing endonucleases and meganucleases can be engineered to bind non-natural target sites. See, for example, Chevalier et al. (2002) <i>Molec. Cell </i>10:895-905; Epinat et al. (2003) <i>Nucleic Acids Res. </i>31:2952-2962; Ashworth et al. (2006) <i>Nature </i>441:656-659; Paques et al. (2007) <i>Current Gene Therapy </i>7:49-66. Thus, engineered homing endonucleases and/or meganucelases which are designed to specifically bind a desired target locus, such as GS, may also be used.</p>
    <p num="p-0105">Delivery</p>
    <p num="p-0106">The ZFNs described herein may be delivered to a target cell by any suitable means. Suitable cells include but not limited to eukaryotic and prokaryotic cells and/or cell lines. Non-limiting examples of such cells or cell lines include COS, CHO (e.g., CHO-S, CHO-K1, CHO-DG44, CHO-DUXB11, CHO-DUKX, CHOK1SV), VERO, MDCK, WI38, V79, B14AF28-G3, BHK, HaK, NS0, SP2/0-Ag14, HeLa, HEK293 (e.g., HEK293-F, HEK293-H, HEK293-T), and perC6 cells as well as insect cells such as <i>Spodoptera fugiperda </i>(Sf), or fungal cells such as <i>Saccharomyces, Pichia </i>and <i>Schizosaccharomyces. </i> </p>
    <p num="p-0107">Methods of delivering proteins comprising zinc fingers are described, for example, in U.S. Pat. Nos. 6,453,242; 6,503,717; 6,534,261; 6,599,692; 6,607,882; 6,689,558; 6,824,978; 6,933,113; 6,979,539; 7,013,219; and 7,163,824, the disclosures of all of which are incorporated by reference herein in their entireties.</p>
    <p num="p-0108">ZFNs as described herein may also be delivered using vectors containing sequences encoding one or more of the ZFNs. Any vector systems may be used including, but not limited to, plasmid vectors, retroviral vectors, lentiviral vectors, adenovirus vectors, poxvirus vectors; herpesvirus vectors and adeno-associated virus vectors, etc. See, also, U.S. Pat. Nos. 6,534,261; 6,607,882; 6,824,978; 6,933,113; 6,979,539; 7,013,219; and 7,163,824, incorporated by reference herein in their entireties. Furthermore, it will be apparent that any of these vectors may comprise one or more ZFN encoding sequences. Thus, when one or more pairs of ZFNs are introduced into the cell, the ZFNs may be carried on the same vector or on different vectors. When multiple vectors are used, each vector may comprise a sequence encoding one or multiple ZFNs.</p>
    <p num="p-0109">Conventional viral and non-viral based gene transfer methods can be used to introduce nucleic acids encoding engineered ZFPs in cells (e.g., mammalian cells) and target tissues. Such methods can also be used to administer nucleic acids encoding ZFPs to cells in vitro. In certain embodiments, nucleic acids encoding ZFPs are administered for in vivo or ex vivo gene therapy uses. Non-viral vector delivery systems include DNA plasmids, naked nucleic acid, and nucleic acid complexed with a delivery vehicle such as a liposome or poloxamer. Viral vector delivery systems include DNA and RNA viruses, which have either episomal or integrated genomes after delivery to the cell. For a review of gene therapy procedures, see Anderson, <i>Science </i>256:808-813 (1992); Nabel &amp; Felgner, <i>TIBTECH </i>11:211-217 (1993); Mitani &amp; Caskey, <i>TIBTECH </i>11:162-166 (1993); Dillon, <i>TIBTECH </i>11:167-175 (1993); Miller, <i>Nature </i>357:455-460 (1992); Van Brunt, <i>Biotechnology </i>6(10):1149-1154 (1988); Vigne, <i>Restorative Neurology and Neuroscience </i>8:35-36 (1995); Kremer &amp; Perricaudet, <i>British Medical Bulletin </i>51(1):31-44 (1995); Haddada et al., in <i>Current Topics in Microbiology and Immunology </i>Doerfler and Böhm (eds.) (1995); and Yu et al., <i>Gene Therapy </i>1:13-26 (1994).</p>
    <p num="p-0110">Methods of non-viral delivery of nucleic acids encoding engineered ZFPs include electroporation, lipofection, microinjection, biolistics, virosomes, liposomes, immunoliposomes, polycation or lipid:nucleic acid conjugates, naked DNA, artificial virions, and agent-enhanced uptake of DNA. Sonoporation using, e.g., the Sonitron 2000 system (Rich-March) can also be used for delivery of nucleic acids.</p>
    <p num="p-0111">Additional exemplary nucleic acid delivery systems include those provided by Amaxa Biosystems (Cologne, Germany), Maxcyte, Inc. (Rockville, Md.) and BTX Molecular Delivery Systems (Holliston, Mass.) and Copernicus Therapeutics Inc., (see for example U.S. Pat. No. 6,008,336).</p>
    <p num="p-0112">Lipofection is described in e.g., U.S. Pat. No. 5,049,386, U.S. Pat. No. 4,946,787; and U.S. Pat. No. 4,897,355) and lipofection reagents are sold commercially (e.g., Transfectam™ and Lipofectin™). Cationic and neutral lipids that are suitable for efficient receptor-recognition lipofection of polynucleotides include those of Felgner, WO 91/17424, WO 91/16024. Delivery can be to cells (ex vivo administration) or target tissues (in vivo administration).</p>
    <p num="p-0113">The preparation of lipid:nucleic acid complexes, including targeted liposomes such as immunolipid complexes, is well known to one of skill in the art (see, e.g., Crystal, <i>Science </i>270:404-410 (1995); Blaese et al., <i>Cancer Gene Ther. </i>2:291-297 (1995); Behr et al., <i>Bioconjugate Chem. </i>5:382-389 (1994); Remy et al., <i>Bioconjugate Chem. </i>5:647-654 (1994); Gao et al., <i>Gene Therapy </i>2:710-722 (1995); Ahmad et al., <i>Cancer Res. </i>52:4817-4820 (1992); U.S. Pat. Nos. 4,186,183, 4,217,344, 4,235,871, 4,261,975, 4,485,054, 4,501,728, 4,774,085, 4,837,028, and 4,946,787).</p>
    <p num="p-0114">The use of RNA or DNA viral based systems for the delivery of nucleic acids encoding engineered ZFPs take advantage of highly evolved processes for targeting a virus to specific cells in the body and trafficking the viral payload to the nucleus. Viral vectors can be administered directly to patients (in vivo) or they can be used to treat cells in vitro and the modified cells are administered to patients (ex vivo). Conventional viral based systems for the delivery of ZFPs include, but are not limited to, retroviral, lentivirus, adenoviral, adeno-associated, vaccinia and herpes simplex virus vectors for gene transfer. Integration in the host genome is possible with the retrovirus, lentivirus, and adeno-associated virus gene transfer methods, often resulting in long term expression of the inserted transgene. Additionally, high transduction efficiencies have been observed in many different cell types and target tissues.</p>
    <p num="p-0115">The tropism of a retrovirus can be altered by incorporating foreign envelope proteins, expanding the potential target population of target cells. Lentiviral vectors are retroviral vectors that are able to transduce or infect non-dividing cells and typically produce high viral titers. Selection of a retroviral gene transfer system depends on the target tissue. Retroviral vectors are comprised of cis-acting long terminal repeats with packaging capacity for up to 6-10 kb of foreign sequence. The minimum cis-acting LTRs are sufficient for replication and packaging of the vectors, which are then used to integrate the therapeutic gene into the target cell to provide permanent transgene expression. Widely used retroviral vectors include those based upon murine leukemia virus (MuLV), gibbon ape leukemia virus (GaLV), Simian Immunodeficiency virus (SW), human immunodeficiency virus (HIV), and combinations thereof (see, e.g., Buchscher et al., <i>J. Virol. </i>66:2731-2739 (1992); Johann et al., <i>J. Virol. </i>66:1635-1640 (1992); Sommerfelt et al., <i>Virol. </i>176:58-59 (1990); Wilson et al., <i>J. Virol. </i>63:2374-2378 (1989); Miller et al., <i>J. Virol. </i>65:2220-2224 (1991); PCT/US94/05700).</p>
    <p num="p-0116">In applications in which transient expression of a ZFP fusion protein is preferred, adenoviral based systems can be used. Adenoviral based vectors are capable of very high transduction efficiency in many cell types and do not require cell division. With such vectors, high titer and high levels of expression have been obtained. This vector can be produced in large quantities in a relatively simple system. Adeno-associated virus (“AAV”) vectors are also used to transduce cells with target nucleic acids, e.g., in the in vitro production of nucleic acids and peptides, and for in vivo and ex vivo gene therapy procedures (see, e.g., West et al., <i>Virology </i>160:38-47 (1987); U.S. Pat. No. 4,797,368; WO 93/24641; Kotin, <i>Human Gene Therapy </i>5:793-801 (1994); Muzyczka, <i>J. Clin. Invest. </i>94:1351 (1994). Construction of recombinant AAV vectors are described in a number of publications, including U.S. Pat. No. 5,173,414; Tratschin et al., <i>Mol. Cell. Biol. </i>5:3251-3260 (1985); Tratschin, et al., <i>Mol. Cell. Biol. </i>4:2072-2081 (1984); Hermonat &amp; Muzyczka, <i>PNAS </i>81:6466-6470 (1984); and Samulski et al., <i>J. Virol. </i>63:03822-3828 (1989).</p>
    <p num="p-0117">At least six viral vector approaches are currently available for gene transfer in clinical trials, which utilize approaches that involve complementation of defective vectors by genes inserted into helper cell lines to generate the transducing agent.</p>
    <p num="p-0118">pLASN and MFG-S are examples of retroviral vectors that have been used in clinical trials (Dunbar et al., <i>Blood </i>85:3048-305 (1995); Kohn et al., <i>Nat. Med. </i>1:1017-102 (1995); Malech et al., <i>PNAS </i>94:22 12133-12138 (1997)). PA317/pLASN was the first therapeutic vector used in a gene therapy trial. (Blaese et al., <i>Science </i>270:475-480 (1995)). Transduction efficiencies of 50% or greater have been observed for MFG-S packaged vectors. (Ellem et al., <i>Immunol Immunother. </i>44(1):10-20 (1997); Dranoff et al., <i>Hum. Gene Ther. </i>1:111-2 (1997).</p>
    <p num="p-0119">Recombinant adeno-associated virus vectors (rAAV) are a promising alternative gene delivery systems based on the defective and nonpathogenic parvovirus adeno-associated type 2 virus. All vectors are derived from a plasmid that retains only the AAV 145 bp inverted terminal repeats flanking the transgene expression cassette. Efficient gene transfer and stable transgene delivery due to integration into the genomes of the transduced cell are key features for this vector system. (Wagner et al., <i>Lancet </i>351:9117 1702-3 (1998), Kearns et al., <i>Gene Ther. </i>9:748-55 (1996)).</p>
    <p num="p-0120">Replication-deficient recombinant adenoviral vectors (Ad) can be produced at high titer and readily infect a number of different cell types. Most adenovirus vectors are engineered such that a transgene replaces the Ad E1a, E1b, and/or E3 genes; subsequently the replication defective vector is propagated in human 293 cells that supply deleted gene function in trans. Ad vectors can transduce multiple types of tissues in vivo, including nondividing, differentiated cells such as those found in liver, kidney and muscle. Conventional Ad vectors have a large carrying capacity. An example of the use of an Ad vector in a clinical trial involved polynucleotide therapy for antitumor immunization with intramuscular injection (Sterman et al., <i>Hum. Gene Ther. </i>7:1083-9 (1998)). Additional examples of the use of adenovirus vectors for gene transfer in clinical trials include Rosenecker et al., <i>Infection </i>24:1 5-10 (1996); Sterman et al., <i>Hum. Gene Ther. </i>9:7 1083-1089 (1998); Welsh et al., <i>Hum. Gene Ther. </i>2:205-18 (1995); Alvarez et al., <i>Hum. Gene Ther. </i>5:597-613 (1997); Topf et al., <i>Gene Ther. </i>5:507-513 (1998); Sterman et al., <i>Hum. Gene Ther. </i>7:1083-1089 (1998).</p>
    <p num="p-0121">Packaging cells are used to form virus particles that are capable of infecting a host cell. Such cells include 293 cells, which package adenovirus, and ψ2 cells or PA317 cells, which package retrovirus. Viral vectors used in gene therapy are usually generated by a producer cell line that packages a nucleic acid vector into a viral particle. The vectors typically contain the minimal viral sequences required for packaging and subsequent integration into a host (if applicable), other viral sequences being replaced by an expression cassette encoding the protein to be expressed. The missing viral functions are supplied in trans by the packaging cell line. For example, AAV vectors used in gene therapy typically only possess inverted terminal repeat (ITR) sequences from the AAV genome which are required for packaging and integration into the host genome. Viral DNA is packaged in a cell line, which contains a helper plasmid encoding the other AAV genes, namely rep and cap, but lacking ITR sequences. The cell line is also infected with adenovirus as a helper. The helper virus promotes replication of the AAV vector and expression of AAV genes from the helper plasmid. The helper plasmid is not packaged in significant amounts due to a lack of ITR sequences. Contamination with adenovirus can be reduced by, e.g., heat treatment to which adenovirus is more sensitive than AAV.</p>
    <p num="p-0122">In many gene therapy applications, it is desirable that the gene therapy vector be delivered with a high degree of specificity to a particular tissue type. Accordingly, a viral vector can be modified to have specificity for a given cell type by expressing a ligand as a fusion protein with a viral coat protein on the outer surface of the virus. The ligand is chosen to have affinity for a receptor known to be present on the cell type of interest. For example, Han et al., <i>Proc. Natl. Acad. Sci. USA </i>92:9747-9751 (1995), reported that Moloney murine leukemia virus can be modified to express human heregulin fused to gp70, and the recombinant virus infects certain human breast cancer cells expressing human epidermal growth factor receptor. This principle can be extended to other virus-target cell pairs, in which the target cell expresses a receptor and the virus expresses a fusion protein comprising a ligand for the cell-surface receptor. For example, filamentous phage can be engineered to display antibody fragments (e.g., FAB or Fv) having specific binding affinity for virtually any chosen cellular receptor. Although the above description applies primarily to viral vectors, the same principles can be applied to nonviral vectors. Such vectors can be engineered to contain specific uptake sequences which favor uptake by specific target cells.</p>
    <p num="p-0123">Gene therapy vectors can be delivered in vivo by administration to an individual patient, typically by systemic administration (e.g., intravenous, intraperitoneal, intramuscular, subdermal, or intracranial infusion) or topical application, as described below. Alternatively, vectors can be delivered to cells ex vivo, such as cells explanted from an individual patient (e.g., lymphocytes, bone marrow aspirates, tissue biopsy) or universal donor hematopoietic stem cells, followed by reimplantation of the cells into a patient, usually after selection for cells which have incorporated the vector.</p>
    <p num="p-0124">Ex vivo cell transfection for diagnostics, research, or for gene therapy (e.g., via re-infusion of the transfected cells into the host organism) is well known to those of skill in the art. In a preferred embodiment, cells are isolated from the subject organism, transfected with a ZFP nucleic acid (gene or cDNA), and re-infused back into the subject organism (e.g., patient). Various cell types suitable for ex vivo transfection are well known to those of skill in the art (see, e.g., Freshney et al., <i>Culture of Animal Cells, A Manual of Basic Technique </i>(3rd ed. 1994)) and the references cited therein for a discussion of how to isolate and culture cells from patients).</p>
    <p num="p-0125">In one embodiment, stem cells are used in ex vivo procedures for cell transfection and gene therapy. The advantage to using stem cells is that they can be differentiated into other cell types in vitro, or can be introduced into a mammal (such as the donor of the cells) where they will engraft in the bone marrow. Methods for differentiating CD34+ cells in vitro into clinically important immune cell types using cytokines such a GM-CSF, IFN-γ and TNF-α are known (see Inaba et al., <i>J. Exp. Med. </i>176:1693-1702 (1992)).</p>
    <p num="p-0126">Stem cells are isolated for transduction and differentiation using known methods. For example, stem cells are isolated from bone marrow cells by panning the bone marrow cells with antibodies which bind unwanted cells, such as CD4+ and CD8+(T cells), CD45+ (panB cells), GR-1 (granulocytes), and Iad (differentiated antigen presenting cells) (see Inaba et al., <i>J. Exp. Med. </i>176:1693-1702 (1992)).</p>
    <p num="p-0127">Vectors (e.g., retroviruses, adenoviruses, liposomes, etc.) containing therapeutic ZFP nucleic acids can also be administered directly to an organism for transduction of cells in vivo. Alternatively, naked DNA can be administered. Administration is by any of the routes normally used for introducing a molecule into ultimate contact with blood or tissue cells including, but not limited to, injection, infusion, topical application and electroporation. Suitable methods of administering such nucleic acids are available and well known to those of skill in the art, and, although more than one route can be used to administer a particular composition, a particular route can often provide a more immediate and more effective reaction than another route.</p>
    <p num="p-0128">Methods for introduction of DNA into hematopoietic stem cells are disclosed, for example, in U.S. Pat. No. 5,928,638. Vectors useful for introduction of transgenes into hematopoietic stem cells, e.g., CD34<sup>+</sup> cells, include adenovirus Type 35.</p>
    <p num="p-0129">Vectors suitable for introduction of transgenes into immune cells (e.g., T-cells) include non-integrating lentivirus vectors. See, for example, Ory et al. (1996) <i>Proc. Natl. Acad. Sci. USA </i>93:11382-11388; Dull et al. (1998) <i>J. Virol. </i>72:8463-8471; Zuffery et al. (1998) <i>J. Virol. </i>72:9873-9880; Follenzi et al. (2000) <i>Nature Genetics </i>25:217-222.</p>
    <p num="p-0130">Pharmaceutically acceptable carriers are determined in part by the particular composition being administered, as well as by the particular method used to administer the composition. Accordingly, there is a wide variety of suitable formulations of pharmaceutical compositions available, as described below (see, e.g., <i>Remington's Pharmaceutical Sciences, </i>17th ed., 1989).</p>
    <p num="p-0131">As noted above, the disclosed methods and compositions can be used in any type of cell including, but not limited to, prokaryotic cells, fungal cells, Archaeal cells, plant cells, insect cells, animal cells, vertebrate cells, mammalian cells and human cells. Suitable cell lines for protein expression are known to those of skill in the art and include, but are not limited to COS, CHO (e.g., CHO-S, CHO-K1, CHO-DG44, CHO-DUXB11), VERO, MDCK, WI38, V72, B14AF28-G3, BHK, HaK, NS0, SP2/0-Ag14, HeLa, HEK293 (e.g., HEK293-F, HEK293-H, HEK293-T), perC6, insect cells such as <i>Spodoptera fugiperda </i>(Sf), and fungal cells such as <i>Saccharomyces, Pichia </i>and <i>Schizosaccharomyces</i>. Progeny, variants and derivatives of these cell lines can also be used.</p>
    <p num="p-0132">Applications</p>
    <p num="p-0133">The disclosed methods and compositions can be used for inactivation of a GS genomic sequence. As noted above, inactivation includes partial or complete repression of GS gene expression in a cell. Inactivation of a GS gene can be achieved, for example, by a single cleavage event, by cleavage followed by non-homologous end joining, by cleavage at two sites followed by joining so as to delete the sequence between the two cleavage sites, by targeted recombination of a missense or nonsense codon into the coding region, by targeted recombination of an irrelevant sequence (i.e., a “stuffer” sequence) into the gene or its regulatory region, so as to disrupt the gene or regulatory region, or by targeting recombination of a splice acceptor sequence into an intron to cause mis-splicing of the transcript.</p>
    <p num="p-0134">Thus, the methods and compositions described herein allow for the generation of GS-deficient cell lines for use in recombinant protein production, for example α1-antitrypsin and/or monoclonal antibody production. Additional genes, such as FUT8 may also be inactivated as cells in which FUT8 is inactivated produce antibodies that exhibit greater effector function, particularly in the induction of ADCC.</p>
    <heading>EXAMPLES</heading> <heading>Example 1</heading> <heading>Design and Construction of ZFNs</heading> <p num="h-0012">A. Glutamine Synthetase (GS) ZFNs</p>
    <p num="p-0135">As the full sequence of the CHO genome is not available, the CHO GS gene was cloned and sequenced to generate the target DNA sequences for ZFN designs. The complete CHO GS sequence is shown in <figref idrefs="DRAWINGS">FIG. 13</figref>, which also designates introns and exons.</p>
    <p num="p-0136">ZFN target sites in exon 6 of the CHO GS gene were selected because this region encodes amino acids critical for the catalytic function of the GS protein based on available crystal structures (see, e.g., Almassy et al. (1986) <i>Nature </i>323:304-309; Gill et al. (2002) <i>Biochem. </i>41:9863-9872; Liaw et al. (1994) <i>Biochem. </i>33:675-681) and, as such, ZFN-mediated mutation of this site was expected to result in loss of functional GS activity. Partial sequence for CHO GS exon 6 is shown below. Capital letters indicate exon sequence; small letters indicate intron sequence; and ZFN target sites 9372/9075 (Table 1, <figref idrefs="DRAWINGS">FIG. 1A</figref> and <figref idrefs="DRAWINGS">FIG. 4A</figref>) are underlined:</p>
    <p num="p-0137">
      <tables id="TABLE-US-00002" num="00002"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="right"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 30)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">5′-atggcactattctgttccttttcctcccctctgaagacttggcacat</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ggggactttggttaacaagggtgatgacttaaaagtggttcagggtagag</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">gtaagtagaacaagctaggagcttgagttggcctgaacagttagttggcc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ttattctaaaggtcaacatgttctttctagTGGGAATTCCAAATAGGACC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">CTGTGAAGGAATCCGCATGGGAGATCATCTCTGGGTGGCCCGTTTCATCT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">TGCATCGAGTATGTGAAGACTTTGGGGTAATAGCAACCTTTGACC<u>CCAAG</u> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <u>CCCATTCCTGGGA</u>ACTGGA<u>ATGGTGCAGGCT</u>GCCATACCAACTTTAGCAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">CAAGGCCATGCGGGAGGAGAATGGTCTGAAgtaagtagcttcctctggag</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ccatctttattctcatggggtggaagggctttgtgttagggttgggaaag</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ttggacttctcacaaactacatgccatgctcttcgtgtttgtcataagcc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tatcgttttgtacccgttggagaagtgacagtactctaggaatagaatta</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">cagctgtgatatgggaaagttgtcacgtaggttcaagcatttaaaggtct</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ttagtaagaactaaatacacatacaagcaagtgggtgacttaattcttac</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tgatgggaagaggccagtgatgggggtcttcccatccaaaagataattgg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tattacatgttgaggactggtctgaagcacttgagacataggtcacaagg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">cagacacagcctgcatcaagtatttattggtttcttatggaactcatgcc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tgctcctgcccttgaaggacag-3′</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0138">Binding sites for ZFN9372 and ZFN9075 are shown in <figref idrefs="DRAWINGS">FIG. 4B</figref>, while <figref idrefs="DRAWINGS">FIG. 4C</figref> depicts the target and finger designs of these ZFNs.</p>
    <p num="p-0139">In addition, ZFNs targeted to sites in exon 2 were also designed. Partial sequence for CHO exon 2 is shown below. Capital letters indicate exon sequence; small letters indicate intron sequence; and ZFN target sites 8365/8361 (Table 1, and <figref idrefs="DRAWINGS">FIG. 1A</figref>) are underlined.</p>
    <p num="p-0140">
      <tables id="TABLE-US-00003" num="00003"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="right"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 31)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">5′-ggctggcagatctccgagttcgaggctgacctggtctgaatagcaag</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">gaaattaaggggtgaggcgtatgtctgttaaagcaagaataaaaggcaaa</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ggaacactccacagtcaattattcaagtcttgatggcagtaatgtagttg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tattgggtggattaagacattctaataatgaatttttttgtctttttgtt</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ccctcttttcagctttctcaaaattaatggatattaaaaatccccttagc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">cgggcgttggtggcacacacctttaatcccagcactcgggaggcagaggc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">aggcagatctctgtgagttcgaggccagcctggtctccagattgagtgcc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">aggataggctccaaagctacacagaaaccgtgtctcgaaaaacaaaacaa</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">aaaaataaaaaaaaaaatcccttaactagcccaacctacaagggatgatc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tttgtctaactatgaactttaaacctcttgaaagcagagtgaataatgca</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">cttcaataatgttgacttccaaaggagagaccaccacaccgttccctgtg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">cctcttacgcaattcctgcaggggacccccttcagagtagatgttaatga</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">aatgacttttgtctctcCAGAGCACCTTCCACCATGGCCACCTCAGCAAG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">TTCCCACTTGAACAAAAACATCAAGCAAATGTACTTGTGCCTGCCCCAGG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">GTGAGAAAGT<u>CCAAGCCATGTA</u>TATCT<u>GGGTTGATGGTA</u>CTGGAGAAGGA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">CTGCGCTGCAAAACCCGCACCCTGGACTGTGAGCCCAAGTGTGTAGAAGg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tgagcatgggcaggagcaggacatgtgcctggaagtgggcaagcagcctg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">agatttgaccttccttctgttttgtttgcaaagtctttcaaaagcaggtc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tcttcaggcctcagtcagtcacccgtaagctgccgagtagtctggaggca</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tagaaaacaatggaggcctttatttagatggaatcttgtgtgtgctggta</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">cactgaagaaaaatattgggtcatatttgtagggggtgggaggttggagt</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">attgctaacctagccaaccccaggaacctagtttgaaagacctgtaacta</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">gaatatgctatcaagtttatagagcagtggttctcaacctttcaaatgct</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ttacacttgaatacaactcctcatgttctggtgattacccccatcccaac</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">cattgctaacttcttaactgaaatttcactactgctacgaatcataatgt</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">atctgtgtttttggatggtcttaggtgacccctgtgaaagggttgtgaga</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ccatcctcaaaggggttgtgacctacaggttgagacccttttgagtgctg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tgtttattagtatttatacagtggaattctgggtgcaaagcacatgctcc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">aaagtagtttctctgggactggccatttgttttcgatggggatcttttaa</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">aacttgcaaaggaaccaaaaaaaaaaaaatgcagaaaaaaggaggtgggg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">gagtgcacgcctttaatcccagtacttgggaggcagaggcaggcggatct</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ctgtgagtttgagaccagcctggtctacaagagctagttccaggacagcc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tccaaagccacagagaaaccctgtctcaaaacaaacaaacaaacaaaaaa</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">attaaaaaaaaaaaaaactttcaaaggagacctgttttattttagttgtg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">gcctttgttttggtaggaagggcagctagtttaggatgagtttttattat</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tctaagatgttgccgtttg-3′</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0141">ZFNs targeting the CHO GS sequence (for example as shown in Table 1), were assembled into mammalian expression vectors as described in U.S. patent application Ser. No. 12/218,035 and tested by transient transfection into CHO cells.</p>
    <p num="h-0013">B. DHFR ZFNs</p>
    <p num="p-0142">ZFNs targeted to DHFR were designed and produced as described in U.S. Patent Publication No 2008/0015164. For experiments described herein, DHFR ZFN pairs designated 9461/7844 and 9476/9477 (individual ZFNs shown in Table 2) were used.</p>
    <p num="p-0143"> <tables id="TABLE-US-00004" num="00004"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="350pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 2</td> </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Exemplary zinc finger nucleases targeted to DHFR</td> </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="70pt" align="left"> </colspec> <colspec colname="4" colwidth="70pt" align="left"> </colspec> <colspec colname="5" colwidth="70pt" align="left"> </colspec> <colspec colname="6" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">ZFN name</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Target sequence</td> <td class="description-td">F1</td> <td class="description-td">F2</td> <td class="description-td">F3</td> <td class="description-td">F4</td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td> </tr> <tr class="description-tr"> <td class="description-td">ZFN 7844</td> <td class="description-td">QSGALAR</td> <td class="description-td">RSDNLRE</td> <td class="description-td">QSSDLSR</td> <td class="description-td">TSSNRKT</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">ccAATGCTCAGGTAct</td> <td class="description-td">(SEQ ID NO: 14)</td> <td class="description-td">(SEQ ID NO: 3)</td> <td class="description-td">(SEQ ID NO: 2)</td> <td class="description-td">(SEQ ID NO: 25)</td> </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 32)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">ZFN 9461</td> <td class="description-td">RSDTLSE</td> <td class="description-td">NNRDRTK</td> <td class="description-td">RSDHLSA</td> <td class="description-td">QSGHLSR</td> </tr> <tr class="description-tr"> <td class="description-td">agGGAAGGTCTCCGtt</td> <td class="description-td">(SEQ ID NO: 34)</td> <td class="description-td">(SEQ ID NO: 35)</td> <td class="description-td">(SEQ ID NO: 36)</td> <td class="description-td">(SEQ ID NO: 37)</td> </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 33)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">ZFN 9476</td> <td class="description-td">QSSDLSR</td> <td class="description-td">DRSDLSR</td> <td class="description-td">QSGALAR</td> <td class="description-td">RSDHLTT</td> </tr> <tr class="description-tr"> <td class="description-td">caTGGGTAGCCGCTga</td> <td class="description-td">(SEQ ID NO: 2)</td> <td class="description-td">(SEQ ID NO: 39)</td> <td class="description-td">(SEQ ID NO: 14)</td> <td class="description-td">(SEQ ID NO: 40)</td> </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 38)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">ZFN 9477</td> <td class="description-td">RSDHLST</td> <td class="description-td">RSAHLSR</td> <td class="description-td">RSDHLSR</td> <td class="description-td">DSSDRKK</td> </tr> <tr class="description-tr"> <td class="description-td">agTCCGGGGGGTGGtg</td> <td class="description-td">(SEQ ID NO: 7)</td> <td class="description-td">(SEQ ID NO: 17) </td> <td class="description-td">(SEQ ID NO: 42)</td> <td class="description-td">(SEQ ID NO: 43)</td> </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 41)</td> </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td> </tr> </tbody> </tgroup> </table> </tables> <br>
C. FUT8 ZFNs
</p>
    <p num="p-0144">ZFNs targeted to FUT8 were designed and produced as described in U.S. Ser. No. 12/218,035. For experiments described herein, FUT8 ZFNs designated 12176 and 12172 and shown in Table 3 were used.</p>
    <p num="p-0145">
      <tables id="TABLE-US-00005" num="00005"> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="280pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 3</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Exemplary zinc finger nucleases targeted to FUT8</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <colspec colname="6" colwidth="35pt" align="left"> </colspec> <colspec colname="7" colwidth="35pt" align="left"> </colspec> <colspec colname="8" colwidth="0pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">ZFN name</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Target sequence</td>
                <td class="description-td">F1</td>
                <td class="description-td">F2</td>
                <td class="description-td">F3</td>
                <td class="description-td">F4</td>
                <td class="description-td">F5</td>
                <td class="description-td">F6</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ZFN 12172</td>
                <td class="description-td">RSDNLSV</td>
                <td class="description-td">QNATRIN</td>
                <td class="description-td">RSDHLSQ</td>
                <td class="description-td">QSATRTK</td>
                <td class="description-td">RSDNLSR</td>
                <td class="description-td">RNDNRKT</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">AAGGAGGCAAAGA</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAAAG</td>
                <td class="description-td">NO: 45)</td>
                <td class="description-td">NO: 46)</td>
                <td class="description-td">NO: 9)</td>
                <td class="description-td">NO: 47)</td>
                <td class="description-td">NO: 48)</td>
                <td class="description-td">NO: 49)</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 44)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ZFN 12176</td>
                <td class="description-td">RSDNLRE</td>
                <td class="description-td">NNTQLIE</td>
                <td class="description-td">TSSILSR</td>
                <td class="description-td">RSDNLSA</td>
                <td class="description-td">RKDTRIT</td>
                <td class="description-td">N/A</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAGAAGGGTCATC</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
              </tr> <tr class="description-tr"> <td class="description-td">AG</td>
                <td class="description-td">NO: 3)</td>
                <td class="description-td">NO: 51)</td>
                <td class="description-td">NO: 52)</td>
                <td class="description-td">NO: 53)</td>
                <td class="description-td">NO: 54)</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 50)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ZFN 12170</td>
                <td class="description-td">RSDNLSV</td>
                <td class="description-td">QNATRIN</td>
                <td class="description-td">RSDNLST</td>
                <td class="description-td">QSATRTK</td>
                <td class="description-td">RSDNLSR</td>
                <td class="description-td">RNDNRKT</td>
              </tr> <tr class="description-tr"> <td class="description-td">taAAGGAGGCAAAG</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
                <td class="description-td">(SEQ ID</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACAAAGt</td>
                <td class="description-td">NO: 45)</td>
                <td class="description-td">NO: 46)</td>
                <td class="description-td">NO: 57)</td>
                <td class="description-td">NO: 47)</td>
                <td class="description-td">NO: 48)</td>
                <td class="description-td">NO: 49)</td>
              </tr> <tr class="description-tr"> <td class="description-td">(SEQ ID NO: 56)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0146">Plasmids comprising sequences encoding the above ZFNs were constructed essentially as described in Urnov et al. (2005)<i>Nature </i>435(7042):646-651 via fusion to wild-type FokI cleavage domains or obligate heterodimer FokI cleavage domains described in U.S. Patent Publication No. 2008/0131962 and Miller et al. (2007) <i>Nature Biotech. </i>25:778-785.</p>
    <heading>Example 2</heading> <heading>GS-ZFN Modification of Endogenous GS</heading> <p num="p-0147">To determine whether GS-targeted ZFNs modified the endogenous GS locus as expected, CEL-1 mismatch assays were performed essentially as per the manufacturer's instructions (Trangenomic SURVEYOR™). Briefly, the appropriate ZFN plasmid pairs were transfected into CHO K-I cells growing adherently in serum-containing medium or CHO-S cells growing in suspension in serum-free chemically defined medium.</p>
    <p num="p-0148">CHO K-I cells were obtained from the American Type Culture Collection and grown as recommended in F-12 medium (Invitrogen) supplemented with 10% qualified fetal calf serum (FCS, Hyclone). CHO-S cells were purchased from Invitrogen (Carlsbad, Calif.) and were grown and maintained as suspension cultures in chemically-defined protein-free and animal component-free CD-CHO medium supplemented with 8 mM L-glutamine and HT supplement as necessary (100 μM sodium hypoxanthine and 16 μM thymidine) (all from Invitrogen, Carlsbad, Calif.) in a humidity-controlled shaker incubator (ATR, inc., Laurel, Md.) at 125 rpm with 5% CO<sub>2 </sub>at 37° C.</p>
    <p num="p-0149">Adherent cells were disassociated from plasticware using TrypLE Select™ protease (Invitrogen). For transfection, one million CHO K-I cells were mixed with 1 μg each zinc finger nuclease and 100 μL Amaxa Solution T. Cells were transfected in an Amaxa Nucleofector II™ using program U-23 and recovered into 1.4 mL warm F-12 medium+10% FCS.</p>
    <p num="p-0150">Genomic DNA was extracted from the ZFN-treated cells using the Qiagen DNeasy™ kit (Qiagen, Inc.), the target locus amplified by PCR using the appropriate primers for the region of the GS locus targeted by the ZFNs, the PCR products were subjected to a melting/annealing step resulting in the formation of distorted duplex DNA through random re-annealing of mutant and wild-type DNA. CEL-I enzyme (Surveyor™ mutation detection kit, Transgenomic, Inc.) was then added to specifically cleave the DNA duplexes at the sites of mismatches. The CEL-I cleaved samples were resolved on a 10% TBE polyacrylamide gel, stained with ethidium bromide, and the DNA bands were quantified using densitometry. The frequency of NHEJ was calculated as described essentially in Miller et al. (2007), supra.</p>
    <p num="p-0151">As shown in <figref idrefs="DRAWINGS">FIGS. 1A and 1B</figref>, GS ZFNs modified the endogenous GS locus. In particular, the ZFN pair including ZFN 9372 and ZFN 9075 resulted in modification of 26% (with wild-type cleavage domains) and 24% (with engineered obligate heterodimer forming cleavage domains) of chromosomes in CHO-KI cells and 25% of chromosomes in CHO-S cells (<figref idrefs="DRAWINGS">FIG. 1B</figref>). Similarly, ZFN pair including ZFN 8361 and 8365 (targeted to exon 2) resulted in modification of 7% of chromosomes.</p>
    <p num="p-0152">In agreement with the results of the Surveyor™ Nuclease Assay (<figref idrefs="DRAWINGS">FIG. 1A</figref>), direct sequencing confirmed 34% (91/266) of the alleles harbored mutations at the ZFN target site typical of NHEJ-mediated DNA repair (see, e.g., Weterings et al. (2004) <i>DNA repair </i>(<i>Amst</i>) 3:1425-1435). Importantly, 81% of the sequenced mutations resulted in a reading frame shift (<figref idrefs="DRAWINGS">FIG. 5</figref>).</p>
    <heading>Example 3</heading> <heading>Generation of GS-Negative Cell Lines</heading> <p num="p-0153">To generate CHO cell lines lacking GS, single-cell derived lines were isolated by limiting dilution from both the CHO-K1 and CHO-S transfected pools described in Example 2. Sequencing of the ZFN target region revealed that 17 of 54 CHO-S derived cell lines (31%) had at least one disrupted GS gene, 8 of 54 (15%) were homozygous for a mutant allele, 2 were compound heterozygous, and the remaining 7 were heterozygous (with one wild-type allele). For CHO-K1-derived cell lines, sequencing analysis identified 18 of 50 (36%) had at least one disrupted GS allele, with 5 (10%) homozygous for a given mutation. See, also, <figref idrefs="DRAWINGS">FIG. 1C</figref> showing the genotypes of the homozygous mutant lines.</p>
    <p num="p-0154">All homozygous mutant lines harbor GS mutations that result in a shift in the open reading frame or an in-frame deletion of critical amino acids (e.g. lines B4, KA2, and KA4), and were therefore expected to produce no active GS enzyme. As predicted, none of the homozygous mutant cell lines grew in the absence of exogenous L-glutamine, in contrast to wild-type CHO cells (<figref idrefs="DRAWINGS">FIG. 1C</figref>).</p>
    <p num="p-0155">Western blot analysis using an anti-GS monoclonal antibody (BD Biosciences) confirmed that all cell lines with frame shifts or large deletions expressed no detectable GS protein (for example, clone B3, <figref idrefs="DRAWINGS">FIG. 1D</figref>). Lines harboring small in-frame deletions (B4 and KA2) did express GS proteins that were detectable by Western blot but, consistent with the elimination of amino acids critical for GS catalytic activity, did not support the growth of the cells in the absence of L-glutamine (<figref idrefs="DRAWINGS">FIG. 1C</figref>). Growth of cell line B3 was also shown to be dependent on glutamine supplementation (see <figref idrefs="DRAWINGS">FIG. 6</figref>).</p>
    <p num="p-0156">To further validate their GS<sup>−/−</sup> phenotype, three of the CHO-K1-derived clones were adapted for growth in suspension in a chemically-defined serum-free medium over a 3 month period. Growth and viability of the cells was then monitored intensively for a further 4 weeks (<figref idrefs="DRAWINGS">FIG. 1E</figref>).</p>
    <p num="p-0157">The ZFN-generated GS<sup>−/−</sup> lines grew normally in serum-free suspension. cultures in the presence of L-glutamine (24-27 hour doubling time similar to the industry standard CHOK1 SV grown under similar conditions). Subsequent removal of L-glutamine resulted in immediate cessation of cell growth and a rapid loss of viability (see arrows in <figref idrefs="DRAWINGS">FIG. 1E</figref>). Transient transfection of a human IgG antibody expression construct confirmed all three GS<sup>−/−</sup> lines supported comparable transgene expression levels to those obtained with the CHOK1SV line.</p>
    <p num="p-0158">Taken together, these data confirm the successful generation of genetically and phenotypically validated GS knockout CHO cell lines.</p>
    <heading>Example 4</heading> <heading>Generation of GS-DHFR Double Knockout Cell Lines</heading> <p num="p-0159">To create a double-knockout cell line, ZFNs targeted to the DHFR gene were used on the background of the CHO-S GS<sup>−/−</sup> cell line B3. ZFN-mediated knockout of DHFR using the strategy described for GS above is described in U.S. Patent Publication No. 2008/0015164:</p>
    <p num="p-0160">In this Example, two pairs of ZFNs targeting two distinct regions of the DHFR gene were delivered simultaneously with the goal of eliminating the intervening genomic fragment through the repair of the cleaved chromosomal ends via NHEJ. Deletion of a ZFN-specified intergenic sequence was expected to result in a larger and better defined mutation that eliminates the potential for a smaller in-frame mutation to recover protein expression e.g. <figref idrefs="DRAWINGS">FIG. 1D</figref> (Left Panel labeled “B4”).</p>
    <p num="p-0161">In addition to the previously described ZFN pair 9461/7844 that target exon 1 of the CHO DHFR gene (U.S. Patent Publication No. 2008/0015164), a second ZFN pair was generated which targets a site ˜240 bp away in the intron immediately following exon 1 (<figref idrefs="DRAWINGS">FIG. 2A</figref> and <figref idrefs="DRAWINGS">FIG. 7A</figref>).</p>
    <p num="p-0162">Transient transfection was performed as described in Example 3 and transfection with either the ZFN9461/ZFN7844 pair targeting exon 1, or the ZFN9476/ZFN9477 (See <figref idrefs="DRAWINGS">FIGS. 7B and 7C</figref>) pair targeting the intronic site alone resulted in allelic mutation frequencies at the endogenous DHFR locus of 15% and 18%, respectively in the GS<sup>−/−</sup> cell line B3 (<figref idrefs="DRAWINGS">FIG. 2B</figref>). Co-transfection of both ZFN pairs into the GS<sup>−/−</sup> cell line B3 resulted in the appearance of a shorter PCR amplification product consistent with the expected ˜240 bp deletion of the sequence between the ZFN binding sites (<figref idrefs="DRAWINGS">FIG. 2C</figref>). The frequency of this deletion event was estimated at ˜8% of all alleles and was observed only when both pairs of ZFNs were introduced into the cells (<figref idrefs="DRAWINGS">FIG. 2C</figref>).</p>
    <p num="p-0163">Cloning and sequencing of the presumptive deletion PCR fragment revealed the expected excision of the sequence between the ZFN target sites (<figref idrefs="DRAWINGS">FIG. 8</figref>). The observed minor variations in the sequence at the junction of the rejoined chromosomal ends are consistent with the expected small insertions and deletions characteristic of the NHEJ repair process.</p>
    <p num="p-0164">Single-cell derived lines were isolated from the ZFN-treated pool in <figref idrefs="DRAWINGS">FIG. 2C</figref> Lane 7 by limiting dilution. PCR-amplification and sequencing revealed 9% (18 of 200) of the cell lines possessed the expected ˜240-bp deletion. Four of these cell lines (2% of all lines screened, 22% of all lines containing the deletion) were found to harbor a biallelic mutation in the DHFR gene (<figref idrefs="DRAWINGS">FIG. 2D</figref>). Cell line 2B12.8 is homozygous for a 244-bp deletion of the intervening region between the exon1 and intron 1 ZFN cleavage sites. The remainder of the lines had just one allele with the 240-bp deletion while the other presented classical smaller NHEJ driven mutations at the exonic and/or intronic ZFN target sites. For example, cell line 1F1.6 contained a ˜240-bp deletion in one allele (consistent with a dual ZFN cleavage and deletion event), but the other allele contained a 4-bp deletion in exon 1 resulting in a frame shift.</p>
    <p num="p-0165">Cell lines 1F1.6 and 2B12.8 had genotypes consistent with complete gene knockout and thus were selected for further characterization. Western blot analysis revealed no full-length DHFR or GS protein in either of these single-cell derived lines (<figref idrefs="DRAWINGS">FIG. 2E</figref>). Furthermore, neither line was stainable with fluorescein-labeled methotrexate which binds to the active site of the DHFR protein, in contrast to the parental GS<sup>−/−</sup> line B3 or wild-type CHO cells. Most importantly, growth of the 1F1.6 and 2B12.8 cell lines was dependent upon the addition of exogenous hypoxanthine and thymidine (HT) and L-glutamine to the culture medium. In contrast, the parental GS<sup>−/−</sup> line B3 required L-glutamine but not HT, while the DHFR-negative CHO cell line DG44 (GS<sup>WT</sup>) required only HT but not L-glutamine and the wild-type CHO-S required neither supplement for growth (<figref idrefs="DRAWINGS">FIG. 2F</figref>).</p>
    <p num="p-0166">The dependence on exogenous HT and L-glutamine confirms the functional loss of both GS and DHFR activity in cell lines 1F1.6 and 2B12.8.</p>
    <p num="p-0167">Thus, taken together these data demonstrate the successful generation of genetically and phenotypically validated GS<sup>−/−</sup>/DHFR<sup>−/−</sup> double knockout CHO cell lines.</p>
    <heading>Example 5</heading> <heading>Generation of GS-DHFR-FUT8 Triple Knockout Cell Lines</heading> <p num="p-0168">A triple knockout of GS, DHFR and FUT8 was also generated using the GS<sup>−/−</sup> DHFR<sup>−/−</sup> CHO cell line IF1.6. As described in U.S. Ser. No. 12/218,035, aberrant glycosylation resulting from the use of non-human cells e.g. CHO in the expression of protein therapeutics can alter the potency, half-life and even the immunogenicity of the protein product. Thus methods for humanizing the glycosylation of proteins in non-human expression systems are highly desirable. The CHO FUT8 gene is such a target encoding an α1,6-fucosyltransferase that catalyzes the transfer of fucose from GDP-fucose to the core GlcNAc of N-linked oligosaccharides. Point mutations of amino acid residues in the highly conserved Fut motif II encoded in exon 10 result in the inactivation of the FUT8 enzyme.</p>
    <p num="p-0169">Thus, we cloned and sequenced the region spanning FUT8 exon 10 and flanking introns from CHO-K1 genome, and designed ZFNs to target exon 10 (U.S. patent application Ser. No. 12/218,035; <figref idrefs="DRAWINGS">FIG. 3A</figref> and <figref idrefs="DRAWINGS">FIG. 9</figref>). Transient delivery of ZFN12172/ZFN12176 into cell line 1F1.6 resulted in modification of 7.5% of the FUT8 alleles (<figref idrefs="DRAWINGS">FIG. 3B</figref>), and single-cell derived lines were obtained by limiting dilution from this pool.</p>
    <p num="p-0170"> <i>C. griseus </i>FUT8 was not detectable using commercially available antibodies, so we further assayed α1,6-fucosyltransferase activity using a FACS-based fluorescent Lens culinaris agglutin (F-LCA) binding assay, as described in Yamane-Ohnuki et al. (2004) <i>Biotechnol Bioeng </i>87:614-622. LCA selectively binds to cells that present core fucosylated oligosaccharides on their cell surface. Cells lacking core fucosylation due to a complete loss of FUT8 activity do not bind to LCA. Screening of the ZFN-generated cell lines for their ability to bind F-LCA showed no F-LCA binding in 2% of the line (4 of 200) indicating a complete absence of FUT8 enzymatic activity (<figref idrefs="DRAWINGS">FIG. 3C</figref>).</p>
    <p num="p-0171">Sequencing of PCR amplicons of genomic DNA confirmed that both clones possess biallelic compound heterozygous mutations at the FUT8 target locus (<figref idrefs="DRAWINGS">FIG. 3D</figref>), which encode a shift in reading frame in this functionally critical region. Thus the genotype of both cell lines is consistent with the lack of F-LCA binding observed (<figref idrefs="DRAWINGS">FIG. 3C</figref>) and demonstrates the ZFN-driven genetic and functional knockout of FUT8 in cell lines 35F2 and 14C1. Both of the FUT8<sup>−/−</sup> clones also stably maintained their GS<sup>−/−</sup> and DHFR<sup>−/−</sup> genotypes and phenotypes, throughout the multiple rounds of ZFN transfection and single-cell cloning. Indeed, the generation of a triple knockout cell line was well-tolerated as measured by growth rate studies in serum-containing medium supplemented with HT and L-glutamine which gave mean population doubling times of 21.8 and 21.6 hours for clones 35F2 and 14C1, respectively. Taken together these data demonstrate the successful generation of single, double and triple gene knockouts in CHO cells using engineered ZFNs.</p>
    <heading>Example 6</heading> <heading>Generation of GR-CCR5-PPP1R12C Triple Knockout Cell Lines</heading> <p num="p-0172">A triple knockout cell line was also generated by concurrent administration of inactivating ZFNs. In particular, K562 cell lines in which CCR5, glucocorticoid receptor (GR) and PPP1R12C (also known as the adeno-associated virus integration site or “AAVS1”) were inactivated were also generated by simultaneous application of ZFNs targeted to these loci.</p>
    <p num="p-0173">ZFNs targeted to CCR5, GR and AAVS1 were generated as described in U.S. Patent Publication Nos. 20080159996 (CCR5); 20080188000 (GR) and U.S. application Ser. No. 12/150,103 (PPP1R12C/AAVS1). Schematics showing the ZFN binding sites in portions of these genes are shown in <figref idrefs="DRAWINGS">FIG. 10A</figref>. The CCR5 gene contains 3 exons, the coding sequence (CDS) is located within exon 3, and the CCR5 ZFN target sequence is located in the CDS as indicated. The GR gene contains 9 exons, and the GR ZFN target sequence is located in Exon3. The AAVS1 ZFN target sequence is located in the middle of the AAVS1 region.</p>
    <p num="p-0174">Plasmids encoding pairs of ZFNs linked to either the wild-type, ZFN-Fok I(wt) or an obligate heterodimer EL/KK variant, ZFN-Fok I(EL/KK) of the catalytic domain of Fok I were transiently co-transfected into K562 cells, and the modification frequency at CCR5 (left), GR (middle), AAVS1 (right) loci was determined by the Surveyor™ Nuclease Assay 10 days after transfection, respectively. As shown in <figref idrefs="DRAWINGS">FIG. 10B</figref>, the target genes were subject to NHEJ following ZFN treatment.</p>
    <p num="p-0175">Single-cell derived cell lines of these triple-knockout cells were also evaluated by PCR and sequencing of genomic DNA described above. In particular, the lane 8 samples in <figref idrefs="DRAWINGS">FIG. 10B</figref>, treated with CCR5 ZFN-Fok I(EL/KK)+GR ZFN-Fok I(EL/KK)+AAVS1 ZFN-Fok I(EL/KK) were examined. The PCR primers were designed to specifically amplify unmodified wild-type (wt) sequences but not sequences with deletions.</p>
    <p num="p-0176">Results are shown below in the following Table 4. All clones containing wt sequences (wt or heterozygous) had visible PCR bands at predicted sizes, whereas knock-out (KO) clones had no visible PCR bands. Among 144 single cell clones screened by CCR5 PCR, 5 clones contain CCR5 KO. Nine clones were identified as GR KO by GR PCR. Two of the clones contain both CCR5 KO and GR KO. All 12 clones, which are KO clones based on either CCR5 PCR or GR PCR, were screened by AAVS1 PCR. Among the 144 clones screened, one CCR5 single KO clone was identified, seven GR single KO clones were identified, whereas no attempt was tried to identify AAVS1 single KO clones. One CCR5/GR dual KO clone was identified. Two CCR5/AAVS1 dual KO clones were identified. No GR/AAVS1 dual KO clone was identified among the 144 clones, though it is very likely that a GR/AAVS1 dual KO clone might have been identified if more clones were screened. One CCR5/GR/AAVS1 triple KO clone (B17) was identified.</p>
    <p num="p-0177">
      <tables id="TABLE-US-00006" num="00006"> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 4</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Evaluation of single cell clones of putative triple knockout cells</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="105pt" align="left"> </colspec> <colspec colname="1" colwidth="42pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CCR5</td>
                <td class="description-td">GR</td>
                <td class="description-td">AAVS1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Total Clones screened</td>
                <td class="description-td">144 </td>
                <td class="description-td">144 </td>
                <td class="description-td">12*</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Total KO clones</td>
                <td class="description-td">5</td>
                <td class="description-td">9</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Single KO clone</td>
                <td class="description-td">1</td>
                <td class="description-td">7</td>
                <td class="description-td">N/A</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CCR5/GR dual KO clone</td>
                <td class="description-td">1</td>
                <td class="description-td">1</td>
                <td class="description-td">N/A</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CCR5/AAVS1 dual KO clone</td>
                <td class="description-td">2</td>
                <td class="description-td">N/A</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GR/AAVS1 dual KO clone</td>
                <td class="description-td">N/A</td>
                <td class="description-td">0</td>
                <td class="description-td">0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CCR5/GR/AAVS1 triple</td>
                <td class="description-td">1</td>
                <td class="description-td">1</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">KO clone</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="left" id="FOO-00001" class="description-td" colspan="5">*12 clones that were identified as CCR5 or GR KO clones</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p num="p-0178">Exemplary sequencing data of cells from triple knockout clone B17 is shown in <figref idrefs="DRAWINGS">FIG. 10C</figref>. The ZFN target sequences are underlined. ‘−’ indicate deletions, bold letters indicate insertions.</p>
    <heading>Example 7</heading> <heading>GS-Targeted ZFNs are Active in Multiple Species and Cell Types</heading> <p num="p-0179">GS-specific ZFNs as described herein (designed to human CHO cell GS sequence) were also evaluated in mouse cells. As shown in <figref idrefs="DRAWINGS">FIGS. 11A and 11B</figref>, ZFN target sites in GS were well conserved across species.</p>
    <p num="p-0180">ZFN pairs 8361/8365 (targeted to exon 2) and 9075/9372 (targeted to exon 6) were evaluated for their ability to cleave GS in human K562 and mouse Neuro2a cells as described in Example 2 above.</p>
    <p num="p-0181">As shown in <figref idrefs="DRAWINGS">FIG. 12</figref>, ZFNs designed to CHO GS sequences were functional in other human cell types and in other species.</p>
    <p num="p-0182">GS-specific ZFNs (ZFN 9075 and ZFN 9372) were also tested in human embryonic kidney HEK293 cells as describe in Example 2. As shown in <figref idrefs="DRAWINGS">FIG. 14A</figref>, the CEL-I enzyme (Surveyor™ mutation detection kit, Transgenomic, Inc.) indicated an NHEJ occurance rate of 21%.</p>
    <p num="p-0183">GS knockout cell lines were made in a HEK293 cell background using the same approach as in CHO cells using the same GS ZFNs (see Example 3). Two knockout clones, clone g17 and g52, were further characterized. Clone g52 is homozygous for an 11 bp-deletion at the GS locus. Clone 17 has a 169 bp-deletion at one GS allele, and a 4 bp-insertion at the other allele. As shown in <figref idrefs="DRAWINGS">FIG. 14B</figref>, neither clone expresses GS protein as determined by western blotting. Importantly, both clones required exogenously supplemented L-glutamine for growth (see <figref idrefs="DRAWINGS">FIG. 14C</figref>).</p>
    <p num="p-0184">Thus, ZFNs can be employed to rapidly knock out multiple genes in a single mammalian cell line by sequential or simultaneous inactivation of the target genes. The results presented here are not dependent on the cell line or the choice of ZFNs. The frequency of ZFN-driven biallelic knockout events at each individual gene (&gt;1%), combined with the lack of dependence on selection markers results in the ability to rapidly “stack” such genetic lesions and generate complex multi-gene knockout cell lines previously considered impractical. The high efficiency of gene knock out following only transient ZFN expression and the ability to stack multiple traits in eukaryotic cells makes it feasible to perform custom genetic engineering of essentially any cell type.</p>
    <p num="p-0185">All patents, patent applications and publications mentioned herein are hereby incorporated by reference in their entirety.</p>
    <p num="p-0186">Although disclosure has been provided in some detail by way of illustration and example for the purposes of clarity of understanding, it will be apparent to those skilled in the art that various changes and modifications can be practiced without departing from the spirit or scope of the disclosure. Accordingly, the foregoing descriptions and examples should not be construed as limiting.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5356802">US5356802</a></td><td class="patent-data-table-td patent-date-value">Apr 3, 1992</td><td class="patent-data-table-td patent-date-value">Oct 18, 1994</td><td class="patent-data-table-td ">The Johns Hopkins University</td><td class="patent-data-table-td ">Functional domains in flavobacterium okeanokoites (FokI) restriction endonuclease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5487994">US5487994</a></td><td class="patent-data-table-td patent-date-value">Nov 23, 1994</td><td class="patent-data-table-td patent-date-value">Jan 30, 1996</td><td class="patent-data-table-td ">The Johns Hopkins University</td><td class="patent-data-table-td ">Insertion and deletion mutants of FokI restriction endonuclease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5789538">US5789538</a></td><td class="patent-data-table-td patent-date-value">Apr 18, 1997</td><td class="patent-data-table-td patent-date-value">Aug 4, 1998</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Zinc finger proteins with high affinity new DNA binding specificities</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5925523">US5925523</a></td><td class="patent-data-table-td patent-date-value">Aug 26, 1997</td><td class="patent-data-table-td patent-date-value">Jul 20, 1999</td><td class="patent-data-table-td ">President &amp; Fellows Of Harvard College</td><td class="patent-data-table-td ">Intraction trap assay, reagents and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6007988">US6007988</a></td><td class="patent-data-table-td patent-date-value">Aug 17, 1995</td><td class="patent-data-table-td patent-date-value">Dec 28, 1999</td><td class="patent-data-table-td ">Medical Research Council</td><td class="patent-data-table-td ">A library of dna sequences which encode zinc finger polypeptide binding motifs for display on viral particles; the dna sequences having random allocation of amino acids; for producing zinc finger molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6013453">US6013453</a></td><td class="patent-data-table-td patent-date-value">Aug 25, 1998</td><td class="patent-data-table-td patent-date-value">Jan 11, 2000</td><td class="patent-data-table-td ">Medical Research Council</td><td class="patent-data-table-td ">Libraries of dna sequences encoding zinc finger binding motifs for display on a viral particle, together with methods of designing zinc finger binding polypeptides for binding to a target sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6140081">US6140081</a></td><td class="patent-data-table-td patent-date-value">Oct 16, 1998</td><td class="patent-data-table-td patent-date-value">Oct 31, 2000</td><td class="patent-data-table-td ">The Scripps Research Institute</td><td class="patent-data-table-td ">Nucleotide binding polypeptide; for efficient use as diagnostic tools in gene therapy and transgenic cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6140466">US6140466</a></td><td class="patent-data-table-td patent-date-value">May 27, 1997</td><td class="patent-data-table-td patent-date-value">Oct 31, 2000</td><td class="patent-data-table-td ">The Scripps Research Institute</td><td class="patent-data-table-td ">Comprises at least three zinc finger domains that bind to target cellular nucleotide sequence and modulate its transcription; genome-specific transcriptional switches in gene therapy; transgenic plants and animals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6200759">US6200759</a></td><td class="patent-data-table-td patent-date-value">Apr 21, 1999</td><td class="patent-data-table-td patent-date-value">Mar 13, 2001</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Interaction trap assay, reagents and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6242568">US6242568</a></td><td class="patent-data-table-td patent-date-value">Jan 18, 1995</td><td class="patent-data-table-td patent-date-value">Jun 5, 2001</td><td class="patent-data-table-td ">The Scripps Research Institute</td><td class="patent-data-table-td ">Zinc finger protein derivatives and methods therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6410248">US6410248</a></td><td class="patent-data-table-td patent-date-value">Jan 29, 1999</td><td class="patent-data-table-td patent-date-value">Jun 25, 2002</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">General strategy for selecting high-affinity zinc finger proteins for diverse DNA target sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6453242">US6453242</a></td><td class="patent-data-table-td patent-date-value">Jan 12, 1999</td><td class="patent-data-table-td patent-date-value">Sep 17, 2002</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Selection of sites for targeting by zinc finger proteins and methods of designing zinc finger proteins to bind to preselected sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6479626">US6479626</a></td><td class="patent-data-table-td patent-date-value">Mar 1, 1999</td><td class="patent-data-table-td patent-date-value">Nov 12, 2002</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Poly zinc finger proteins with improved linkers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6534261">US6534261</a></td><td class="patent-data-table-td patent-date-value">Jan 12, 1999</td><td class="patent-data-table-td patent-date-value">Mar 18, 2003</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Regulation of endogenous gene expression in cells using zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6607882">US6607882</a></td><td class="patent-data-table-td patent-date-value">Jan 6, 2000</td><td class="patent-data-table-td patent-date-value">Aug 19, 2003</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Regulation of endogenous gene expression in cells using zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6824978">US6824978</a></td><td class="patent-data-table-td patent-date-value">Nov 3, 2000</td><td class="patent-data-table-td patent-date-value">Nov 30, 2004</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Regulation of endogenous gene expression in cells using zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6903185">US6903185</a></td><td class="patent-data-table-td patent-date-value">May 13, 2002</td><td class="patent-data-table-td patent-date-value">Jun 7, 2005</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Generating chimeric zinc finger protein; obtain nucleotide sequences coding chimeric protein, transform cells, express nucleotide sequences, recover protein</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6933113">US6933113</a></td><td class="patent-data-table-td patent-date-value">Aug 28, 2001</td><td class="patent-data-table-td patent-date-value">Aug 23, 2005</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Using recombinant zinc finger proteins; use in human therapeutic applications such as cancer therapy and altering plant phenotypes, including disease resistance, fruit ripening</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6979539">US6979539</a></td><td class="patent-data-table-td patent-date-value">Jul 2, 2001</td><td class="patent-data-table-td patent-date-value">Dec 27, 2005</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Genetic engineering; antiinflammatory agents, rheumatic diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7013219">US7013219</a></td><td class="patent-data-table-td patent-date-value">Sep 16, 2002</td><td class="patent-data-table-td patent-date-value">Mar 14, 2006</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Regulation of endogenous gene expression in cells using zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7153949">US7153949</a></td><td class="patent-data-table-td patent-date-value">Apr 20, 2005</td><td class="patent-data-table-td patent-date-value">Dec 26, 2006</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Nucleic acid encoding poly-zinc finger proteins with improved linkers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7163824">US7163824</a></td><td class="patent-data-table-td patent-date-value">Aug 15, 2002</td><td class="patent-data-table-td patent-date-value">Jan 16, 2007</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Cell containing nucleic acid molecule comprising polynucleotide sequence encoding engineered zinc finger protein operably linked to promoter, contacting target site in endogenous cellular gene with protein to inhibit transcription</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030232410">US20030232410</a></td><td class="patent-data-table-td patent-date-value">Mar 20, 2003</td><td class="patent-data-table-td patent-date-value">Dec 18, 2003</td><td class="patent-data-table-td ">Monika Liljedahl</td><td class="patent-data-table-td ">Methods and compositions for using zinc finger endonucleases to enhance homologous recombination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040096452">US20040096452</a></td><td class="patent-data-table-td patent-date-value">Aug 9, 2001</td><td class="patent-data-table-td patent-date-value">May 20, 2004</td><td class="patent-data-table-td ">Denney Dan W.</td><td class="patent-data-table-td ">Multivalent vaccines for the treatment of B-cell malignancies (e.g., lymphomas and leukemias) . The present invention also provides methods for the production of custom vaccines, including multivalent vaccines for the treatment of immune</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050026157">US20050026157</a></td><td class="patent-data-table-td patent-date-value">Sep 5, 2003</td><td class="patent-data-table-td patent-date-value">Feb 3, 2005</td><td class="patent-data-table-td ">David Baltimore</td><td class="patent-data-table-td ">Enzymatic fusion protein comprising cleavage and nuclear localization domain for use gene targeting tool in gene therapeutics</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050064474">US20050064474</a></td><td class="patent-data-table-td patent-date-value">Aug 6, 2004</td><td class="patent-data-table-td patent-date-value">Mar 24, 2005</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Cleaving cellular chromatin and inserting expression vector comprising nucleotide sequences coding zinc finger-cleavage half domain genetic exchange; genome engineering and homologous recombination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050208489">US20050208489</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2003</td><td class="patent-data-table-td patent-date-value">Sep 22, 2005</td><td class="patent-data-table-td ">Dana Carroll</td><td class="patent-data-table-td ">Targeted chromosomal mutagenasis using zinc finger nucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20050277132">US20050277132</a></td><td class="patent-data-table-td patent-date-value">Feb 17, 2005</td><td class="patent-data-table-td patent-date-value">Dec 15, 2005</td><td class="patent-data-table-td ">Divergence Inc., A Delaware Corporation</td><td class="patent-data-table-td ">Nematode GS-like sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20060063231">US20060063231</a></td><td class="patent-data-table-td patent-date-value">Sep 8, 2005</td><td class="patent-data-table-td patent-date-value">Mar 23, 2006</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions and methods for protein production</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20060188987">US20060188987</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 2005</td><td class="patent-data-table-td patent-date-value">Aug 24, 2006</td><td class="patent-data-table-td ">Dmitry Guschin</td><td class="patent-data-table-td ">Targeted deletion of cellular DNA sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070134796">US20070134796</a></td><td class="patent-data-table-td patent-date-value">Jul 26, 2006</td><td class="patent-data-table-td patent-date-value">Jun 14, 2007</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Targeted integration and expression of exogenous nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070218528">US20070218528</a></td><td class="patent-data-table-td patent-date-value">Feb 3, 2005</td><td class="patent-data-table-td patent-date-value">Sep 20, 2007</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Methods and compositions for targeted cleavage and recombination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20080015164">US20080015164</a></td><td class="patent-data-table-td patent-date-value">May 17, 2007</td><td class="patent-data-table-td patent-date-value">Jan 17, 2008</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Methods and compositions for inactivation of dihydrofolate reductase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20080131962">US20080131962</a></td><td class="patent-data-table-td patent-date-value">May 23, 2007</td><td class="patent-data-table-td patent-date-value">Jun 5, 2008</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Engineered cleavage half-domains</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20080159996">US20080159996</a></td><td class="patent-data-table-td patent-date-value">May 23, 2007</td><td class="patent-data-table-td patent-date-value">Jul 3, 2008</td><td class="patent-data-table-td ">Dale Ando</td><td class="patent-data-table-td ">Using zinc finger protein nuclease as therapeutic tool in treatment and prevention of viral infection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20080188000">US20080188000</a></td><td class="patent-data-table-td patent-date-value">Nov 13, 2007</td><td class="patent-data-table-td patent-date-value">Aug 7, 2008</td><td class="patent-data-table-td ">Andreas Reik</td><td class="patent-data-table-td ">Using targeted cleavage of genomic DNA within glucocorticoid receptor (GR) gene to treat and prevent asthma, chronic airway and cushing&#39;s diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20080299580">US20080299580</a></td><td class="patent-data-table-td patent-date-value">Apr 24, 2008</td><td class="patent-data-table-td patent-date-value">Dec 4, 2008</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Targeted integration into the PPP1R12C locus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20090042250">US20090042250</a></td><td class="patent-data-table-td patent-date-value">Jul 10, 2008</td><td class="patent-data-table-td patent-date-value">Feb 12, 2009</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Methods and compositions for inactivating alpha 1,6 fucosyltransferase (FUT8) gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20090305419">US20090305419</a></td><td class="patent-data-table-td patent-date-value">May 28, 2009</td><td class="patent-data-table-td patent-date-value">Dec 10, 2009</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for linking DNA-binding domains and cleavage domains</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=sy04BwABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DGB%26NR%3D2338237A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNFtNZlu_WZNNVOjMgEvKYvAHqTx3w">GB2338237A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1995019431A1?cl=en">WO1995019431A1</a></td><td class="patent-data-table-td patent-date-value">Jan 18, 1995</td><td class="patent-data-table-td patent-date-value">Jul 20, 1995</td><td class="patent-data-table-td ">Carlos F Barbas Iii</td><td class="patent-data-table-td ">Zinc finger protein derivatives and methods therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1996006166A1?cl=en">WO1996006166A1</a></td><td class="patent-data-table-td patent-date-value">Aug 17, 1995</td><td class="patent-data-table-td patent-date-value">Feb 29, 1996</td><td class="patent-data-table-td ">Yen Choo</td><td class="patent-data-table-td ">Improvements in or relating to binding proteins for recognition of dna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998037186A1?cl=en">WO1998037186A1</a></td><td class="patent-data-table-td patent-date-value">Feb 18, 1998</td><td class="patent-data-table-td patent-date-value">Aug 27, 1998</td><td class="patent-data-table-td ">Actinova Ltd</td><td class="patent-data-table-td ">In vitro peptide or protein expression library</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998053057A1?cl=en">WO1998053057A1</a></td><td class="patent-data-table-td patent-date-value">May 26, 1998</td><td class="patent-data-table-td patent-date-value">Nov 26, 1998</td><td class="patent-data-table-td ">Medical Res Council</td><td class="patent-data-table-td ">Nucleic acid binding polypeptide library</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998053058A1?cl=en">WO1998053058A1</a></td><td class="patent-data-table-td patent-date-value">May 26, 1998</td><td class="patent-data-table-td patent-date-value">Nov 26, 1998</td><td class="patent-data-table-td ">Medical Res Council</td><td class="patent-data-table-td ">Nucleic acid binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998053059A1?cl=en">WO1998053059A1</a></td><td class="patent-data-table-td patent-date-value">May 26, 1998</td><td class="patent-data-table-td patent-date-value">Nov 26, 1998</td><td class="patent-data-table-td ">Choo Yen</td><td class="patent-data-table-td ">Nucleic acid binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998053060A1?cl=en">WO1998053060A1</a></td><td class="patent-data-table-td patent-date-value">May 26, 1998</td><td class="patent-data-table-td patent-date-value">Nov 26, 1998</td><td class="patent-data-table-td ">Medical Res Council</td><td class="patent-data-table-td ">Nucleic acid binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998054311A1?cl=en">WO1998054311A1</a></td><td class="patent-data-table-td patent-date-value">May 27, 1998</td><td class="patent-data-table-td patent-date-value">Dec 3, 1998</td><td class="patent-data-table-td ">Carlos F Barbas Iii</td><td class="patent-data-table-td ">Zinc finger protein derivatives and methods therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2000027878A1?cl=en">WO2000027878A1</a></td><td class="patent-data-table-td patent-date-value">Nov 9, 1999</td><td class="patent-data-table-td patent-date-value">May 18, 2000</td><td class="patent-data-table-td ">Choo Yen</td><td class="patent-data-table-td ">Screening system for zinc finger polypeptides for a desired binding ability</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001060970A2?cl=en">WO2001060970A2</a></td><td class="patent-data-table-td patent-date-value">Feb 17, 2001</td><td class="patent-data-table-td patent-date-value">Aug 23, 2001</td><td class="patent-data-table-td ">Toolgen Inc</td><td class="patent-data-table-td ">Zinc finger domains and methods of identifying same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001088197A2?cl=en">WO2001088197A2</a></td><td class="patent-data-table-td patent-date-value">May 16, 2001</td><td class="patent-data-table-td patent-date-value">Nov 22, 2001</td><td class="patent-data-table-td ">Massachusetts Inst Technology</td><td class="patent-data-table-td ">Methods and compositions for interaction trap assays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002016536A1?cl=en">WO2002016536A1</a></td><td class="patent-data-table-td patent-date-value">Aug 9, 2001</td><td class="patent-data-table-td patent-date-value">Feb 28, 2002</td><td class="patent-data-table-td ">Aihara Shin</td><td class="patent-data-table-td ">Bactericidal antifouling detergent for hard surface</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002077227A2?cl=en">WO2002077227A2</a></td><td class="patent-data-table-td patent-date-value">Nov 20, 2001</td><td class="patent-data-table-td patent-date-value">Oct 3, 2002</td><td class="patent-data-table-td ">Eisenberg Stephen P</td><td class="patent-data-table-td ">Iterative optimization in the design of binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002099084A2?cl=en">WO2002099084A2</a></td><td class="patent-data-table-td patent-date-value">Apr 4, 2002</td><td class="patent-data-table-td patent-date-value">Dec 12, 2002</td><td class="patent-data-table-td ">Choo Yen</td><td class="patent-data-table-td ">Composite binding polypeptides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2003016496A2?cl=en">WO2003016496A2</a></td><td class="patent-data-table-td patent-date-value">Aug 20, 2002</td><td class="patent-data-table-td patent-date-value">Feb 27, 2003</td><td class="patent-data-table-td ">Carlos F Barbas Iii</td><td class="patent-data-table-td ">Zinc finger binding domains for cnn</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2007014275A2?cl=en">WO2007014275A2</a></td><td class="patent-data-table-td patent-date-value">Jul 26, 2006</td><td class="patent-data-table-td patent-date-value">Feb 1, 2007</td><td class="patent-data-table-td ">Sangamo Biosciences Inc</td><td class="patent-data-table-td ">Targeted integration and expression of exogenous nucleic acid sequences</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Almassy, et al., "<a href='http://scholar.google.com/scholar?q="Novel+Subunit-Subunit+Interactions+in+the+Structure+of+Glutamine+Synthetase%2C"'>Novel Subunit-Subunit Interactions in the Structure of Glutamine Synthetase,</a>" Nature 323:304-309 (1986).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Almassy, et al., "<a href='http://scholar.google.com/scholar?q="Novel+Subunit%E2%80%94Subunit+Interactions+in+the+Structure+of+Glutamine+Synthetase%2C"'>Novel Subunit—Subunit Interactions in the Structure of Glutamine Synthetase,</a>" Nature 323:304-309 (1986).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ashworth, et al., "<a href='http://scholar.google.com/scholar?q="Computational+Redesign+of+Endonuclease+DNA+Binding+and+Cleavage+Specificity%2C"'>Computational Redesign of Endonuclease DNA Binding and Cleavage Specificity,</a>" Nature 441:656-659 (2006).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Beerli, et al., "<a href='http://scholar.google.com/scholar?q="Engineering+Polydactyl+Zinc-Finger+Transcription+Factors%2C"'>Engineering Polydactyl Zinc-Finger Transcription Factors,</a>" Nature Biotechnology 20:135-141 (2002).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bibikova, et al., "<a href='http://scholar.google.com/scholar?q="Enhancing+Gene+Targeting+With+Designed+Zinc+Finger+Nucleases%2C"'>Enhancing Gene Targeting With Designed Zinc Finger Nucleases,</a>" Science 300:764 (2003).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bibikova, et al., "<a href='http://scholar.google.com/scholar?q="Stimulation+of+Homologous+Recombination+Through+Targeted+Cleavage+by+Chimeric+Nucleases%2C"'>Stimulation of Homologous Recombination Through Targeted Cleavage by Chimeric Nucleases,</a>" Mol Cell Biol 21:289-297 (2001).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bibikova, et al., "<a href='http://scholar.google.com/scholar?q="Targeted+Chromosomal+Cleavage+and+Mutagenesis+in+Drosophila+Using+Zinc-Finger+Nucleases%2C"'>Targeted Chromosomal Cleavage and Mutagenesis in Drosophila Using Zinc-Finger Nucleases,</a>" Genetics 161:1169-1175 (2002).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bitinate, et al., "<a href='http://scholar.google.com/scholar?q="Foki+Dimerization+Is+Required+for+DNA+Cleavage%2C"'>Foki Dimerization Is Required for DNA Cleavage,</a>" PNAS USA 95:10570-10575 (1998).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chevalier, et al., "<a href='http://scholar.google.com/scholar?q="Design%2C+Activity%2C+and+Structure+of+a+Highly+Specific+Artificial+Endonuclease%2C"'>Design, Activity, and Structure of a Highly Specific Artificial Endonuclease,</a>" Molecular Cell 10:895-905 (2002).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Choo, et al., "<a href='http://scholar.google.com/scholar?q="Advances+in+Zinc+Finger+Engineering%2C"'>Advances in Zinc Finger Engineering,</a>" Curr. Opin. Struct. Biol. 10:411-416 (2000).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Epinat, et al., "<a href='http://scholar.google.com/scholar?q="A+Novel+Engineered+Meganuclease+Induces+Homologous+Recombination+in+Yeast+and+Mammalian+Cells%2C"'>A Novel Engineered Meganuclease Induces Homologous Recombination in Yeast and Mammalian Cells,</a>" Nucleic Acids Research 31:2952-2962 (2003).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gill, et al., "<a href='http://scholar.google.com/scholar?q="Multicopy+Crystallographic+Refinement+of+a+Relaxed+Glutamine+Synthetase+From+Mycobacterium+tuberculosis+Highlights+Flexible+Loops+in+the+Enzymatic+Mechanism+and+Its+Regulation%2C"'>Multicopy Crystallographic Refinement of a Relaxed Glutamine Synthetase From Mycobacterium tuberculosis Highlights Flexible Loops in the Enzymatic Mechanism and Its Regulation,</a>" Biochem 41:9863-9872 (2002).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Isalan et al., "<a href='http://scholar.google.com/scholar?q="A+Rapid%2C+Generally+Applicable+Method+to+Engineer+Zinc+Fingers+Illustrated+by+Targeting+the+HIV-1+Promoter%2C"'>A Rapid, Generally Applicable Method to Engineer Zinc Fingers Illustrated by Targeting the HIV-1 Promoter,</a>" Nat Biotechnol 19:656-660 (2001).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kim, et al., "<a href='http://scholar.google.com/scholar?q="Chimeric+Restriction+Endonuclease%2C"'>Chimeric Restriction Endonuclease,</a>" PNAS USA 91:883-887 (1994).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kim, et al., "<a href='http://scholar.google.com/scholar?q="Insertion+and+Deletion+Mutants+of+FOKI+Restriction+Endonuclease%2C"'>Insertion and Deletion Mutants of FOKI Restriction Endonuclease,</a>" J. Biol. Chem. 269:31978-31981 (1994).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kohli, et al., "<a href='http://scholar.google.com/scholar?q="Facile+Methods+for+Generating+Human+Somatic+Cell+Gene+Knockouts+Using+Recombinant+Adeno-Associated+Viruses%2C"'>Facile Methods for Generating Human Somatic Cell Gene Knockouts Using Recombinant Adeno-Associated Viruses,</a>" Nucleic Acids Research 32:E3 (2004).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Li, et al., "<a href='http://scholar.google.com/scholar?q="Alteration+of+the+Cleavage+Distance+of+FOK+I+Restriction+Endonuclease+by+Insertion+Mutagenesis%2C"'>Alteration of the Cleavage Distance of FOK I Restriction Endonuclease by Insertion Mutagenesis,</a>" PNAS USA 90:2764-2768 (1993).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Li, et al., "<a href='http://scholar.google.com/scholar?q="Functional+Domains+in+FOK+I+Restriction+Endonuclease%2C"'>Functional Domains in FOK I Restriction Endonuclease,</a>" PNAS USA 89:4275-4279 (1992).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Liaw, et al., "<a href='http://scholar.google.com/scholar?q="Structural+Model+for+the+Reaction+Mechanism+of+Glutamine+Synthetase%2C+Based+on+Five+Crystal+Structures+of+Enzyme-Substrate+Complexes%2C"'>Structural Model for the Reaction Mechanism of Glutamine Synthetase, Based on Five Crystal Structures of Enzyme-Substrate Complexes,</a>" Biochem 33:675-681 (1994).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lloyd, et al., "<a href='http://scholar.google.com/scholar?q="Targeted+Mutagenesis+Using+Zinc-Finger+Nucleases+in+Arabidopsis"'>Targeted Mutagenesis Using Zinc-Finger Nucleases in Arabidopsis</a>" PNAS USA 102:2232-2237 (2005).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lombardo, et al., "<a href='http://scholar.google.com/scholar?q="Gene+Editing+in+Human+Stem+Cells+Using+Zinc+Finger+Nucleases+and+Integrase-Defective+Lentiviral+Vector+Delivery%2C"'>Gene Editing in Human Stem Cells Using Zinc Finger Nucleases and Integrase-Defective Lentiviral Vector Delivery,</a>" Nature Biotechnology 25:1298-1306 (2007).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mansour, et al., "<a href='http://scholar.google.com/scholar?q="Disruption+of+the+Proto-Oncogene+Int-2+in+Mouse+Embryo-Derived+Stem+Cells%3A+A+General+Strategy+for+Targeting+Mutations+to+Non-Selectable+Genes%2C"'>Disruption of the Proto-Oncogene Int-2 in Mouse Embryo-Derived Stem Cells: A General Strategy for Targeting Mutations to Non-Selectable Genes,</a>" Nature 336:348-352 (1988).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Miller, et al., "<a href='http://scholar.google.com/scholar?q="An+Improved+Zinc-Finger+Nuclease+Architecture+for+Highly+Specific+Genome+Editing%2C"'>An Improved Zinc-Finger Nuclease Architecture for Highly Specific Genome Editing,</a>" Nature Biotech 25:778-785 (2007).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Moehle, et al., "<a href='http://scholar.google.com/scholar?q="Targeted+Gene+Addition+Into+a+Specified+Location+in+the+Human+Genome+Using+Designed+Zinc+Finger+Nucleases%2C"'>Targeted Gene Addition Into a Specified Location in the Human Genome Using Designed Zinc Finger Nucleases,</a>" PNAS USA 104:3055-3060 (2007).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Morton, et al., "<a href='http://scholar.google.com/scholar?q="Induction+and+Repair+of+Zinc-Finger+Nuclease-Targeted+Double-Strand+Breaks+in+Caenorhabditis+Elegans+Somatic+Cells%2C"'>Induction and Repair of Zinc-Finger Nuclease-Targeted Double-Strand Breaks in Caenorhabditis Elegans Somatic Cells,</a>" PNAS USA 103:16370-16375 (2006).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pabo, et al., "<a href='http://scholar.google.com/scholar?q="Design+and+Selection+of+Novel+CYS2-HIS2+Zinc+Finger+Proteins%2C"'>Design and Selection of Novel CYS2-HIS2 Zinc Finger Proteins,</a>" Ann. Rev. Biochem. 70:313-340 (2001).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Paques, et al., "<a href='http://scholar.google.com/scholar?q="Meganucleases+and+DNA+Double-Strand+Break-Induced+Recombination%3A+Perspectives+for+Gene+Therapy%2C"'>Meganucleases and DNA Double-Strand Break-Induced Recombination: Perspectives for Gene Therapy,</a>" Current Gene Therapy 7:49-66 (2007).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Perez, et al., "<a href='http://scholar.google.com/scholar?q="Establishment+of+HIV-1+Resistance+in+CD4%2B+T+Cells+by+Genome+Editing+Using+Zinc-Finger+Nucleases%2C"'>Establishment of HIV-1 Resistance in CD4+ T Cells by Genome Editing Using Zinc-Finger Nucleases,</a>" Nature Biotechnology 26:808-816 (2008).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Porteus, et al., "<a href='http://scholar.google.com/scholar?q="Gene+Targeting+Using+Zinc+Finger+Nucleases%2C"'>Gene Targeting Using Zinc Finger Nucleases,</a>" Nature Biotechnology 23:967-973 (2005).</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rago, et al., "<a href='http://scholar.google.com/scholar?q="Genetic+Knockouts+and+Knockins+in+Human+Somatic+Cells%2C"'>Genetic Knockouts and Knockins in Human Somatic Cells,</a>" Nature Protocols 2:2734- 2746 (2007).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Santiago, et al., "<a href='http://scholar.google.com/scholar?q="Targeted+Gene+Knockout+in+Mammalian+Cells+by+Using+Engineered+Zinc+Finger+Nucleases%2C"'>Targeted Gene Knockout in Mammalian Cells by Using Engineered Zinc Finger Nucleases,</a>" PNAS USA 105:5809-5814 (2008).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Segal, et al., "<a href='http://scholar.google.com/scholar?q="Custom+DNA-Binding+Proteins+Come+of+Age%3A+Polydactyl+Zinc-Finger+Proteins%2C"'>Custom DNA-Binding Proteins Come of Age: Polydactyl Zinc-Finger Proteins,</a>" Curr. Opin. Biotechnol. 12:632-637 (2001).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Umov, et al., "<a href='http://scholar.google.com/scholar?q="Highly+Efficient+Endogenous+Human+Gene+Correction+Using+Designed+Zinc-Finger+Nucleases%2C%2C"'>Highly Efficient Endogenous Human Gene Correction Using Designed Zinc-Finger Nucleases,,</a>" Nature 435:646-651 (2005).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Vasquez, et al., "<a href='http://scholar.google.com/scholar?q="Manipulating+the+Mammalian+Genome+by+Homologous+Recombination%2C"'>Manipulating the Mammalian Genome by Homologous Recombination,</a>" PNAS USA 98:8403-8410 (2001).</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wurm, et al., "<a href='http://scholar.google.com/scholar?q="Production+of+Recombinant+Protein+Therapeutics+in+Cultivated+Mammalian+Cells%2C"'>Production of Recombinant Protein Therapeutics in Cultivated Mammalian Cells,</a>" Nature Biotechnology 22:1393-1398 (2004).</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069100">435/69.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S015000">435/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S252300">435/252.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006100">435/6.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001200000">C12N1/20</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0021060000">C12P21/06</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001480000">C12Q1/48</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Y603/01002">C12Y603/01002</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2510/00">C12N2510/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/22">C12N9/22</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/93">C12N9/93</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sy04BwABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12P21/02">C12P21/02</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12P21/02</span>, <span class="nested-value">C12N9/22</span>, <span class="nested-value">C12N9/93</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jun 17, 2014</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 1 AND 18 ARE DETERMINED TO BE PATENTABLE AS AMENDED.CLAIMS 2-17, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 18, 2012</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120803</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 13, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">SANGAMO BIOSCIENCES, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:LIU, PEI-QI;MILLER, JEFFREY C.;SIGNING DATES FROM 20091202 TO 20091203;REEL/FRAME:023777/0824</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_4ff636b3d23669b7103f3b3a3a18b4cd.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2c069d8a5BXmbbQVRYlH8X0v9drw\u0026id=sy04BwABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U1E6Ffy59Y5fP4icVXz0Q9OLi0UWQ\u0026id=sy04BwABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U3qO3JrgD1JYujRFe6nsc_Lh2y0Sw","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Methods_and_compositions_for_inactivatin.pdf?id=sy04BwABERAJ\u0026output=pdf\u0026sig=ACfU3U2orI3UWwxTR-00yAawQHprgwxuPw"},"sample_url":"http://www.google.com/patents/reader?id=sy04BwABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>